
<html lang="en"     class="pb-page"  data-request-id="7419d839-a367-4b98-9237-c532123241fb"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b00765;issue:issue:10.1021/jmcmar.2018.61.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors" /></meta><meta name="dc.Creator" content="Shuai  Liu" /></meta><meta name="dc.Creator" content="Hailemichael O.  Yosief" /></meta><meta name="dc.Creator" content="Lingling  Dai" /></meta><meta name="dc.Creator" content="He  Huang" /></meta><meta name="dc.Creator" content="Gagan  Dhawan" /></meta><meta name="dc.Creator" content="Xiaofeng  Zhang" /></meta><meta name="dc.Creator" content="Alex M.  Muthengi" /></meta><meta name="dc.Creator" content="Justin  Roberts" /></meta><meta name="dc.Creator" content="Dennis L.  Buckley" /></meta><meta name="dc.Creator" content="Jennifer A.  Perry" /></meta><meta name="dc.Creator" content="Lei  Wu" /></meta><meta name="dc.Creator" content="James E.  Bradner" /></meta><meta name="dc.Creator" content="Jun  Qi" /></meta><meta name="dc.Creator" content="Wei  Zhang" /></meta><meta name="dc.Description" content="The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single molecule could lead to the development of an effective therapeutic strategy for a variety of diseases in wh..." /></meta><meta name="Description" content="The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single molecule could lead to the development of an effective therapeutic strategy for a variety of diseases in wh..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 20, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00765" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00765" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00765" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00765" /></link>
        
    
    

<title>Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00765" /></meta><meta property="og:title" content="Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0006.jpeg" /></meta><meta property="og:description" content="The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single molecule could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated. Compound 23 has been found to be a potent dual kinase–bromodomain inhibitor (BRD4-BD1 IC50 = 28 nM, PLK1 IC50 = 40 nM). Compound 6 was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC50 = 2579 nM, PLK1 IC50 = 9.9 nM). Molecular docking studies with 23 and BRD4-BD1/PLK1 as well as with 6 corroborate the biochemical assay results." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00765"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00765">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00765&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00765&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00765&amp;href=/doi/10.1021/acs.jmedchem.8b00765" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 7785-7795</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00743" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00783" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Shuai Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuai Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuai++Liu">Shuai Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hailemichael O. Yosief</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hailemichael O. Yosief</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hailemichael+O.++Yosief">Hailemichael O. Yosief</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lingling Dai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lingling Dai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Phase I Clinical Trial Center & Department of Clinical Pharmacology, Xiangya Hospital, Central South University, Changsha, Hunan 410008, P.R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lingling++Dai">Lingling Dai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">He Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">He Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Stony Brook University, Stony Brook, New York 11794-3400, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=He++Huang">He Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gagan Dhawan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gagan Dhawan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</div><div class="loa-info-affiliations-info">Department of Biomedical Science, Acharya Narendra Dev College, University of Delhi, New Delhi 110019, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gagan++Dhawan">Gagan Dhawan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaofeng Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaofeng Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaofeng++Zhang">Xiaofeng Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alex M. Muthengi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex M. Muthengi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex+M.++Muthengi">Alex M. Muthengi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Justin Roberts</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Justin Roberts</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Justin++Roberts">Justin Roberts</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dennis L. Buckley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dennis L. Buckley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dennis+L.++Buckley">Dennis L. Buckley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer A. Perry</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer A. Perry</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+A.++Perry">Jennifer A. Perry</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lei Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lei Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lei++Wu">Lei Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James E. Bradner</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James E. Bradner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Novartis Institutes for Biomedical Research, Cambridge, Massachusetts 02139, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#fc969d91998fd29e8e9d9892998ebc92938a9d8e88958fd29f9391"><span class="__cf_email__" data-cfemail="e78d868a8294c985958683898295a78988918695938e94c984888a">[email protected]</span></a>. Tel: +1 (617) 871-8000. Fax: +1 (617) 582-7370.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+E.++Bradner">James E. Bradner</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2718-4415" title="Orcid link">http://orcid.org/0000-0002-2718-4415</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jun Qi</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jun Qi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#b5dfc0dbeac4dcf5d1d3d6dc9bddd4c7c3d4c7d19bd0d1c0"><span class="__cf_email__" data-cfemail="39534c57664850795d5f5a501751584b4f584b5d175c5d4c">[email protected]</span></a>. Tel: +1 (617) 632-6629. Fax: +1 (617) 582-7370.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jun++Qi">Jun Qi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1461-3356" title="Orcid link">http://orcid.org/0000-0002-1461-3356</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Wei Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#0770626e35297d6f66696047726a6529626372"><span class="__cf_email__" data-cfemail="9fe8faf6adb1e5f7fef1f8dfeaf2fdb1fafbea">[email protected]</span></a>. Tel: +1 (617) 287-6147. Fax: +1 (617) 287-6030.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Zhang">Wei Zhang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6097-2763" title="Orcid link">http://orcid.org/0000-0002-6097-2763</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00765&amp;href=/doi/10.1021%2Facs.jmedchem.8b00765" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 7785–7795</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 20, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 May 2018</li><li><span class="item_label"><b>Published</b> online</span>20 August 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 September 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00765" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00765</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7785%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DShuai%2BLiu%252C%2BHailemichael%2BO.%2BYosief%252C%2BLingling%2BDai%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D17%26contentID%3Dacs.jmedchem.8b00765%26title%3DStructure-Guided%2BDesign%2Band%2BDevelopment%2Bof%2BPotent%2Band%2BSelective%2BDual%2BBromodomain%2B4%2B%2528BRD4%2529%252FPolo-like%2BKinase%2B1%2B%2528PLK1%2529%2BInhibitors%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7795%26publicationDate%3DSeptember%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00765"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3356</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">21</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00765" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Shuai&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Hailemichael&quot;,&quot;last_name&quot;:&quot;O. Yosief&quot;},{&quot;first_name&quot;:&quot;Lingling&quot;,&quot;last_name&quot;:&quot;Dai&quot;},{&quot;first_name&quot;:&quot;He&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Gagan&quot;,&quot;last_name&quot;:&quot;Dhawan&quot;},{&quot;first_name&quot;:&quot;Xiaofeng&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;M. Muthengi&quot;},{&quot;first_name&quot;:&quot;Justin&quot;,&quot;last_name&quot;:&quot;Roberts&quot;},{&quot;first_name&quot;:&quot;Dennis&quot;,&quot;last_name&quot;:&quot;L. Buckley&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;A. Perry&quot;},{&quot;first_name&quot;:&quot;Lei&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;E. Bradner&quot;},{&quot;first_name&quot;:&quot;Jun&quot;,&quot;last_name&quot;:&quot;Qi&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;7785-7795&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00765&quot;},&quot;abstract&quot;:&quot;The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single molecule could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated. Compound 23 has been found to be a potent dual kinase–bromodomain inhibitor (BRD4-BD1 IC50 = 28 nM, PLK1 IC50 = 40 nM). Compound 6 was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC50 = 2579 nM, PLK1 IC50 = 9.9 nM). Molecular docking studies with 23 and BRD4-BD1/PLK1 as well as with 6 corroborate the biochemical assay results.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00765&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00765" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00765&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00765" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00765&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00765" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00765&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00765&amp;href=/doi/10.1021/acs.jmedchem.8b00765" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00765" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00765" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00765%26sid%3Dliteratum%253Aachs%26pmid%3D30125504%26genre%3Darticle%26aulast%3DLiu%26date%3D2018%26atitle%3DStructure-Guided%2BDesign%2Band%2BDevelopment%2Bof%2BPotent%2Band%2BSelective%2BDual%2BBromodomain%2B4%2B%2528BRD4%2529%252FPolo-like%2BKinase%2B1%2B%2528PLK1%2529%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D17%26spage%3D7785%26epage%3D7795%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/jmcmar.2018.61.issue-17/20180913/jmcmar.2018.61.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single molecule could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated. Compound <b>23</b> has been found to be a potent dual kinase–bromodomain inhibitor (BRD4-BD1 IC<sub>50</sub> = 28 nM, PLK1 IC<sub>50</sub> = 40 nM). Compound <b>6</b> was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC<sub>50</sub> = 2579 nM, PLK1 IC<sub>50</sub> = 9.9 nM). Molecular docking studies with <b>23</b> and BRD4-BD1/PLK1 as well as with <b>6</b> corroborate the biochemical assay results.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17867" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17867" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The bromodomain and extra-terminal domain (BET) family of proteins is essential for recognizing acetylated lysines (KAc) on chromatin and has emerged as a potential therapeutic target for the treatment of cancer, inflammation, and viral infectious diseases.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8">(1−8)</a> BET proteins, including BRD2, BRD3, BRD4, and BRDT, are characterized by dual bromodomains (BD1 and BD2) that bind to KAc on histones to facilitate the recruitment and stabilization of protein complexes, which plays an active role in gene expression.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12 ref13">(9−13)</a> With the development of the first small molecule inhibitor of BET bromodomains, (+)-JQ1, we were able to demonstrate that binding to BRD4 displaces BRD4 from chromatin and interrupts the biological function of the protein.<a onclick="showRef(event, 'ref3 ref14'); return false;" href="javascript:void(0);" class="ref ref3 ref14">(3,14)</a> Since our initial reports, several potent and selective BET inhibitors with different chemotypes have been reported to possess binding affinities, selectivity profiles, and cellular activities that are similar to those of (+)-JQ1.<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19">(15−19)</a> More importantly, more than 12 BET bromodomain inhibitors have progressed into clinical trials, including a derivative of JQ1,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> for the treatment of hematologic malignancies, solid tumors, and cardiovascular disease.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> With multiple cancer studies demonstrating the development of resistance to single agents, including BET inhibitors, strategies for combination therapies are being intently pursued.<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22−24)</a></div><div class="NLM_p">Recent reports highlight that the inhibition of polo-like kinase 1 (PLK1), a crucial cell-cycle regulator, synergizes with BET inhibition in multiple cancer types, including prostate cancer and acute myeloid leukemia (AML).<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> While combinatorial inhibition of PLK1 and BET bromodomains with two molecules appears to be synergistic,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> achieving this with a single agent may increase potency for two disease-relevant targets/pathways while decreasing toxicity. The concept has recently been demonstrated with dual PI3K-BRD4 inhibitors that have recently been reported to block the expression of the MYC oncogene, while markedly inhibiting cancer cell growth and metastasis in hepatocellular carcinoma and neuroblastoma in vivo.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> The recent discovery that multiple kinase inhibitors, including the PLK1 inhibitor, BI-2536, and the janus kinase 2 (JAK2) inhibitor, TG101209, (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) exhibit moderate to excellent inhibitory activity against the BET family of proteins opens up the possibility of creating polypharmacological compounds that target both kinases and BET proteins equivalently.<a onclick="showRef(event, 'ref7 ref23 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref7 ref23 ref29 ref30 ref31 ref32">(7,23,29−32)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Kinase inhibitors with nanomolar inhibitory activity against BRD4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In this study, we utilized the scaffold of BI-2536 to develop inhibitors with varied inhibitory activities against BRD4 and PLK1 in order to perform structure–activity relationship (SAR) studies, as well as understand the effect of inhibiting both proteins (PLK1 and BRD4) with a single agent in cancer, as both the BET family of proteins and PLK1 are intently pursued targets for the treatment of cancer.</div><div class="NLM_p">First, we inspected the binding modes of BI-2536 with BRD4-BD1 and PLK1, respectively, through cocrystallization studies (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). In the binding complex of BI-2536 with BRD4-BD1, the methylated amide functions as an acetylated lysine mimic, where the carbonyl oxygen of the dihydropteridinone ring forms a water-mediated hydrogen bond with the conserved acetyl lysine recognition motif, asparagine (N140), in the binding pocket. The methyl group is orientated into a hydrophobic subpocket formed by F83, I146, and C136. The NH group on the aniline and one of the pyrimidine nitrogen atoms interact with the Q85 backbone amide through an intermediary water molecule. The ethyl group that branches off of the asymmetric dihydropteridinone carbon atom is projected into a small hydrophobic subpocket formed by V87, L92, L94, and Y97, while the cyclopentyl moiety and the <i>N</i>-methylpiperidine are both solvent exposed.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of BI-2536 (dark red) bound to (A) BRD4-BD1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a>) and (B) PLK1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a>). The protein residues in contact with the ligand are represented by yellow sticks; hydrogen bonding is represented by red dashed lines, and the remaining residues near the binding pocked are shown in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the basis of structural information from the cocrystals, we envisioned several strategies to improve the BRD4-BD1 binding affinity while reducing the PLK1 inhibition to achieve balanced dual PLK1/BRD4-BD1 inhibition (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). First, we hypothesize that the replacement of the amide methyl group at site 1 with a slightly larger ethyl or isopropyl might enhance binding affinity to BRD4-BD1. Second, to decrease the affinity for PLK1, the elaboration of the 3-methoxy group on the aromatic ring (site 2) with a bulkier group might modulate binding affinity. This methyl ether group is involved in critical hydrogen bonds with the PLK1 active site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The interruption of this interaction is expected to reduce the PLK1 binding affinity of BI-2536 but render its activity to BRD4-BD1 unchanged. Third, modification on the ethyl group might have more impact on PLK1 activity as the pocket in PLK1 is narrower compared to the binding pocket in BRD4-BD1. In the end, we will modify the solvent exposed sites (sites 4 and 5) to fully investigate the effect of each group on the binding affinity of the parent compound toward BRD4-BD1 and PLK1 based on the chemistry we established previously developing fluorescently tagged kinase inhibitors.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Thus, we conducted a medicinal chemistry campaign around the BI-2536 core to develop small molecules with fine-tuned activities against PLK1 and BET bromodomains.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Available modification sites on BI-2536.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Utilizing the structural information from the cocrystal structures of BI-2535 with BRD4-BD1 and PLK1, a series of BI-2536 analogues were synthesized using the general synthesis route shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. In brief, amino acids with varying alkyl side chains (different R<sup>1</sup> group) were esterified with thionyl chloride in MeOH followed by alkylation using reductive amination with cyclic or acyclic ketones. The resulting secondary amine derivative, <b>E1</b>, underwent a regioselective nucleophilic aromatic substitution reaction with 2,6-dichloro-5-nitropyrimidine to produce tertiary anilines. The nitro group was reduced using zinc dust in hot acetic acid, generating the dihydropteridinone ring in <b>E2</b>. Methylation of the amide produced scaffold <b>E3</b>, which was used in a nucleophilic substitution reaction with the aminobenzoic acid derivative and led to the formation of intermediate <b>E4</b> with an acid motif, which was coupled with different amines to produce the final BI-2536 analogues. As planned in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, the resulting analogues include derivatives with different substituent groups on the amide bond (site 1); different ethers at site 2 with cyclic or acyclic motifs; varying substituent groups, such as hydrogen, methyl, propyl, and isobutyl on site 3; and the solvent-exposed site 4 or 5. All compounds in this focused library were then screened for their inhibitory activity against BRD4-BD1 and BRDT-BD1 using AlphaScreen assay technology to obtain IC<sub>50</sub> values (all IC<sub>50</sub> values are for BD1 of all of the bromodomains tested unless otherwise noted).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0002.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of BI-2536 Analogues <b>1–20</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The modification of the cyclopentyl moiety at site 4 with different cyclic and acyclic groups, cyclohexyl (compound <b>1</b>) and isopropyl (compound <b>2</b>), created compounds with nanomolar inhibitory activity against BRD4 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These analogues, however, were less potent than parent compound BI-2536, indicating that the cyclopentyl group is crucial for the potent inhibitory activity against BRD4. These compounds, however, maintained low nanomolar inhibitory activity against PLK1, suggesting that modification at site 4 is tolerated for PLK1 binding. On the other hand, unmethylated amide derivatives of compounds <b>1</b> and <b>2</b> (compounds <b>3</b> and <b>4</b>, respectively) resulted in a more than 1000-fold decrease in binding affinity for BRD4. Interestingly, the inhibitors with unmethylated amide, compounds <b>3</b> and <b>4</b>, displayed a more than 250-fold selectivity for PLK1 over BRD4, indicating the importance of the methylated amide as an <i>N</i>-acetylated lysine mimic for BRD4 binding. Thus, it is critical to have a methylated amide at site 1 to serve as an <i>N</i>-acetylated lysine mimic for bromodomain inhibition. Taken together, the replacement of the cyclopentyl group at site 4 together with an unmethylated amide reduces the BRD4 activity of the inhibitors while maintaining PLK1 selectivity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure–Activity Relationships on BI-2536 Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0004.gif" alt="" id="GRAPHIC-d7e642-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">BRD4-BD1<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">PLK1<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">BRDT-BD1<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BI-2536</td><td class="colsep0 rowsep0" align="left">Cp</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Et</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">0.205 ± 0.008</td><td class="colsep0 rowsep0" align="left">0.004</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">Cy</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Et</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">0.65 ± 0.024</td><td class="colsep0 rowsep0" align="left">0.004</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left"><i>i</i>-Pr</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Et</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">0.186 ± 0.013</td><td class="colsep0 rowsep0" align="left">0.004</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left"><i>i</i>-Pr</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Et</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">2.62</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">Cy</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Et</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">0.0372</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">Cp</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">0.084 ± 0.006</td><td class="colsep0 rowsep0" align="left">0.0011</td><td class="colsep0 rowsep0" align="left">0.342 ± 0.015</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">Cp</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Pr</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">2.579 ± 0.180</td><td class="colsep0 rowsep0" align="left">0.0099</td><td class="colsep0 rowsep0" align="left">8.468 ± 0.076</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">Cp</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">6.633 ± 0.530</td><td class="colsep0 rowsep0" align="left">0.0541</td><td class="colsep0 rowsep0" align="left">25.99 ± 9.70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">Cp</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-iBu</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">18.8 ± 3.4</td><td class="colsep0 rowsep0" align="left">0.078</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">Cp</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">0.107 ± 0.002</td><td class="colsep0 rowsep0" align="left">0.0087</td><td class="colsep0 rowsep0" align="left">0.229 ± 0.010</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">Cp</td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-Et</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">0.489 ± 0.017</td><td class="colsep0 rowsep0" align="left">0.0325</td><td class="colsep0 rowsep0" align="left">3.513 ± 0.351</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">Cy</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">0.31 ± 0.01</td><td class="colsep0 rowsep0" align="left">0.0086</td><td class="colsep0 rowsep0" align="left">0.75 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left"><i>i</i>-Pr</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">0.421 ± 0.016</td><td class="colsep0 rowsep0" align="left">0.0252</td><td class="colsep0 rowsep0" align="left">0.978 ± 0.043</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">tetrahydro-2<i>H</i>-pyran</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-iBu</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">0.322</td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left"><i>i</i>-Pr</td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-iBu</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">41.9</td><td class="colsep0 rowsep0" align="left">0.218</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>S</i>)-JQ1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.032 ± 0.001</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.103 ± 0.004</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were measured by an AlphaScreen binding assay and reported as the average of three replicates ± SE.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values were measured using the Adapta assay format (ThermoFisher Scientific).</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Not determined.</p></div></div></div><div class="NLM_p">Next, we examined the modification of site 3 with methyl (compound <b>5</b>), propyl (compound <b>6</b>), hydrogen (compound <b>7</b>), and isobutyl (compound <b>8</b>) substituents. The analogue with a methyl substitution at the asymmetric dihydropteridinone carbon atom, compound <b>5</b>, exhibited good inhibitory activity against both BRD4 (IC<sub>50</sub> = 84 nM) and PLK1 (IC<sub>50</sub> = 11 nM). Selectivity profiling against 32 bromodomains (BROMOscan) and 468 kinases (KINOMEscan)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> confirmed selective activity for the BET subfamily and PLK kinases (PLK1–3) at 1 μM through the profiling platform at DiscoverX. Specifically, a 1 μM solution of compound <b>5</b> exhibited selective activity against BRD4-BD1 (12% of control) and PLK kinases (0.3–0.5% of control) (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information, Figure 1">Supporting Information, Figure 1</a>). No significant difference in binding activity was observed for the (<i>S</i>) and (<i>R</i>) isomers of the methyl substituent, which is similar to the observation made on the (<i>S</i>) and (<i>R</i>) isomers of BI-2536 by the Fletcher group.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> However, replacing the asymmetric ethyl group with bulky alkyl groups such as propyl (compound <b>6</b>) and isobutyl (compound <b>8</b>) groups resulted in a significant decrease in binding affinity toward BRD4. While compound <b>6</b> displayed excellent inhibitory activity against PLK1, with more than 250-fold selectivity over BRD4, further increasing the size of site 3 with an isobutyl group or deleting the methyl group resulted in a reduction of PLK1 inhibitory activity.</div><div class="NLM_p">With the improvement in BRD4 inhibitory activity with compound <b>5</b>, we further examined this compound through the replacement of the cyclopentyl group with cyclohexyl (compound <b>11</b>), tetrahydro-2<i>H</i>-pyran (compound <b>13</b>), and isopropyl (compound <b>12</b>) groups. None of these substitutions improved the potency toward BRD4 above that seen for compound <b>5</b>, further indicating that substituting the cyclopentyl moiety for the methyl group at the dihydropteridinone ring is optimal for BRD4 inhibition. On the other hand, the inhibitory activity of those analogues against PLK1 remained in the low nanomolar range. Similarly, replacing the cyclopentyl group of compound <b>8</b> with tetrahydro-2<i>H</i>-pyran (compound <b>13</b>) and isopropyl (compound <b>14</b>) groups did not improve the potency toward BRD4 but showed modest inhibitory activity against PLK1, suggesting that PLK1 is tolerant of substitutions at the asymmetric carbon and anilide nitrogen (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Taken together, the adjustment of substituent groups at site 3 tunes down BRD4 activity with little impact on PLK1 activity, leading us to discover the most PLK1 selective inhibitor in the series, compound <b>6</b> (>260-fold).</div><div class="NLM_p">To increase the BRD4 selectivity of BI-2536 analogues, we utilized compound <b>5</b> to further explore the SAR around the terminal amine (site 5) and the methoxy group at site 2 to investigate their impact on potency and selectivity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and the synthesis route shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Replacement of the terminal cyclic amine (amino-piperidine, site 5) with different alkyl and aromatic amines (compounds <b>15</b>–<b>20</b>) did not grossly impact the potency and selectivity for BRD4, implying that the various amine replacements are not engaged in any significant interaction with the amino acid residues in the binding pocket of BRD4. On the other hand, the modification of the methoxy moiety (site 2) with a cyclopentoxyl group improved the potency toward bromodomains while reducing the potency against PLK1. These substitutions created a molecule that has a relatively balanced potency toward both PLK1 and BRD4 with a less than 2-fold selectivity difference (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Notably, compound <b>23</b> exhibited improved potency for BRD4 compared to that of our reference BET inhibitor, (+)-JQ1, but reduced activity against PLK1 compared to that of BI-2536, making it one of the most potent BET bromodomain inhibitors with reduced PLK1 inhibition in the series. This improvement can be overridden by introducing a larger propyl group (compound <b>24</b>) at site 1. Also, the (<i>S</i>)-isomer of compound <b>23</b>, <b>21</b>, retains good inhibitory activity against BRD4 but is less potent toward PLK1, suggesting that the chirality change has more impact on PLK1 activity than on BRD4 activity, which is consistent with the SAR we established in the previous round. Finally, we examined the three compounds with the most balanced BRD4 and PLK1 activities (compounds <b>5</b>, <b>6</b>, and <b>23</b>) in a human acute myeloid leukemia (AML) cell line, MOLM13. In general, BRD4 bromodomain inhibition causes a G1 arrest in the cell cycle as demonstrated by JQ1, while dual PLK1/BRD4 inhibitors, such as BI-2536, cause a G1/G2 arrest in the cell cycle. A 24 h treatment with compound <b>23</b> caused cell cycle arrest in the G1/G2 stage of the cell cycle, suggesting that it is functionally inhibiting both BRD4 and PLK1 (<a class="ref internalNav" href="#notes1" aria-label="Supporting Information, Figure 2">Supporting Information, Figure 2</a>).</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0003.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>21</b> and <b>23–25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure–Activity Relationship on Compound <b>5</b> Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0005.gif" alt="" id="GRAPHIC-d7e1216-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="±">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>5</sup></th><th class="colsep0 rowsep0" align="center" char="±">BRD4-BD1<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">PLK1<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±">BRDT-BD1<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">4-amine, 1-methylpiperidine</td><td class="colsep0 rowsep0" align="char" char="±">0.084 ± 0.006</td><td class="colsep0 rowsep0" align="left">0.0011</td><td class="colsep0 rowsep0" align="char" char="±">0.342 ± 0.015</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">hexamethylene amine</td><td class="colsep0 rowsep0" align="char" char="±">0.107 ± 0.003</td><td class="colsep0 rowsep0" align="left">0.16</td><td class="colsep0 rowsep0" align="char" char="±">0.378 ± 0.014</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">aminomethylpyridine</td><td class="colsep0 rowsep0" align="char" char="±">0.095 ± 0.002</td><td class="colsep0 rowsep0" align="left">0.0084</td><td class="colsep0 rowsep0" align="char" char="±">0.289 ± 0.008</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">phenylethylamine</td><td class="colsep0 rowsep0" align="char" char="±">0.183 ± 0.009</td><td class="colsep0 rowsep0" align="left">0.027</td><td class="colsep0 rowsep0" align="char" char="±">0.558 ± 0.015</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left"><i>N,N</i>-dimethylethylamine</td><td class="colsep0 rowsep0" align="char" char="±">0.104 ± 0.050</td><td class="colsep0 rowsep0" align="left">0.021</td><td class="colsep0 rowsep0" align="char" char="±">0.392 ± 0.013</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">aminopiperidine</td><td class="colsep0 rowsep0" align="char" char="±">0.109 ± 0.003</td><td class="colsep0 rowsep0" align="left">0.009</td><td class="colsep0 rowsep0" align="char" char="±">0.56 ± 0.025</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">piperidine</td><td class="colsep0 rowsep0" align="char" char="±">0.094 ± 0.002</td><td class="colsep0 rowsep0" align="left">0.095</td><td class="colsep0 rowsep0" align="char" char="±">0.369 ± 0.011</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">(<i>S</i>)-Me</td><td class="colsep0 rowsep0" align="left">O-Cp</td><td class="colsep0 rowsep0" align="left">4-amine, 1-methylpiperidine</td><td class="colsep0 rowsep0" align="char" char="±">0.059 ± 0.001</td><td class="colsep0 rowsep0" align="left">0.127</td><td class="colsep0 rowsep0" align="char" char="±">0.245 ± 0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">O-Cp</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="char" char="±">0.253 ± 0.007</td><td class="colsep0 rowsep0" align="left">0.457</td><td class="colsep0 rowsep0" align="char" char="±">0.933 ± 0.032</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Me</td><td class="colsep0 rowsep0" align="left">O-Cp</td><td class="colsep0 rowsep0" align="left">4-amine, 1-methylpiperidine</td><td class="colsep0 rowsep0" align="char" char="±">0.028 ± 0.001</td><td class="colsep0 rowsep0" align="left">0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.102 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">(<i>R</i>)-Pr</td><td class="colsep0 rowsep0" align="left">O-Cp</td><td class="colsep0 rowsep0" align="left">4-amine, 1-methylpiperidine</td><td class="colsep0 rowsep0" align="char" char="±">0.596 ± 0.014</td><td class="colsep0 rowsep0" align="left">0.047</td><td class="colsep0 rowsep0" align="char" char="±">1.951 ± 0.080</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">O-Cp</td><td class="colsep0 rowsep0" align="left">4-amine, 1-methylpiperidine</td><td class="colsep0 rowsep0" align="char" char="±">0.69 ± 0.018</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">3.543 ± 0.198</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were measured by an AlphaScreen binding assay and reported as the average of three replicates ± SE.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values were measured using the Adapta assay format (ThermoFisher Scientific).</p></div></div></div><div class="NLM_p">To gain a better understanding of the potency and predict the binding modes of our BI-2536 analogues, we utilized molecular modeling with PLK1 and BRD4-BD1 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). As described above, compound <b>23</b> shows equivalent potency for both BRD4 and PLK1, while compound <b>6</b> demonstrates the most selectivity for PLK1 over BRD4 in the series. To gain a better understanding of how compound <b>23</b> interacts with BRD4, we further modeled the docking of <b>23</b> with BRD4-BD1. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, all of the interactions between residues on BRD4 and BI-2536 are conserved when <b>23</b> resides in the active site of BRD4. The conserved phenyl ring in the center of compound <b>23</b> develops π–π stacking interactions with the aromatic rings in the WPF shelf (W81, P82, F83) of BRD4 as seen with the parental compound BI-2536. Furthermore, almost all of the original binding patterns established in our cocrystallography study of BI-2536 and BRD4 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) are strengthened as indicated by the van der Waals (vdw) interaction energy decomposition, where stronger interactions formed with P82, Q85, V87, C136, N140, I146, and compound <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Although the L92, L94, and Y97 hydrophobic subpocket of BRD4 is moved further from the dihydropteridinone, the deep pocket formed by F83, I146, and M149 made closer contact with the newly embedded cyclopentane to compensate or even gain back the hydrophobic interaction loss (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Overall, the docking analysis of compound <b>23</b> with BRD4 is consistent with the binding affinity defined in our biochemical assays.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Docking Results of Some BI-2536 Analogues on BRD4 and PLK1<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">docking score (kcal/mol)</th><th class="rowsep1 colsep0" colspan="2" align="center">IC<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" colspan="2" align="center">ligand–receptor interactions (vdw energy < −4 kcal/mol)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">molecule</th><th class="colsep0 rowsep0" align="center" char=".">BRD4</th><th class="colsep0 rowsep0" align="center" char=".">PLK1</th><th class="colsep0 rowsep0" align="center">BRD4</th><th class="colsep0 rowsep0" align="center" char=".">PLK1</th><th class="colsep0 rowsep0" align="center">BRD4</th><th class="colsep0 rowsep0" align="center">PLK1</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char=".">–7.34</td><td class="colsep0 rowsep0" align="char" char=".">–8.46</td><td class="colsep0 rowsep0" align="left">0.028</td><td class="colsep0 rowsep0" align="char" char=".">0.0400</td><td class="colsep0 rowsep0" align="left">Pro82, Gln85, Val87, Ile146</td><td class="colsep0 rowsep0" align="left">Leu59, Cys133, Arg136, Phe183</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">–6.54</td><td class="colsep0 rowsep0" align="char" char=".">–8.68</td><td class="colsep0 rowsep0" align="left">2.579</td><td class="colsep0 rowsep0" align="char" char=".">0.0099</td><td class="colsep0 rowsep0" align="left">Trp81, Pro82, Gln85, Val87, Leu92</td><td class="colsep0 rowsep0" align="left">Leu59, Arg136, Phe183</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">–6.63</td><td class="colsep0 rowsep0" align="char" char=".">–7.62</td><td class="colsep0 rowsep0" align="left">18.80</td><td class="colsep0 rowsep0" align="char" char=".">0.0780</td><td class="colsep0 rowsep0" align="left">Gln85, Val87, Asp88</td><td class="colsep0 rowsep0" align="left">Gly63, Asp194</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">–6.50</td><td class="colsep0 rowsep0" align="char" char=".">–8.23</td><td class="colsep0 rowsep0" align="left">0.084</td><td class="colsep0 rowsep0" align="char" char=".">0.0011</td><td class="colsep0 rowsep0" align="left">Val87, Asp88, Ile146</td><td class="colsep0 rowsep0" align="left">Leu59, Arg136, Phe183</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BI-2536</td><td class="colsep0 rowsep0" align="char" char=".">–7.02</td><td class="colsep0 rowsep0" align="char" char=".">–8.60</td><td class="colsep0 rowsep0" align="left">0.025</td><td class="colsep0 rowsep0" align="char" char=".">0.00083</td><td class="colsep0 rowsep0" align="left">Leu92, Ile146</td><td class="colsep0 rowsep0" align="left">Arg57, Leu59, Arg136, Phe183</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup><sup>a</sup></sup><p class="last">Details in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a>.</p></div></div></div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Comparison of the per-residue decomposition of two BRD4-BD1 ligands, <b>23</b> in blue and BI-2536 in red, with intermolecular vdw interactions as a function of the receptor sequence. (B) Docking of <b>23</b> (cyan) with BRD4-BD1 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a>); protein residues in contact with the ligand are represented by yellow sticks, and the rest are in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In modeling the docking between compound <b>23</b> and PLK1, we found that the C133 backbone amide nitrogen of PLK1 conserves its hydrogen bonding to the nitrogen of dihydropteridinone in <b>23</b>, as indicated by unchanged dips at residue C133 from the vdw energy decomposition. The π–π stacking interaction between F83 and the dihydropteridinone in <b>23</b> is also conserved (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). The resulting docking pose of <b>23</b> with PLK1 retains interactions similar to those of BI-2536 with PLK1 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). In addition, the methyl group in <b>23</b>, which replaced the original ethyl group in BI-2536, does not reduce interactions with A80 and K82 significantly. However, a notable difference upon the substitution of cyclopentane for the methyl group in <b>23</b> is that the cyclopentane extends deeper into the hydrophobic subpocket between R57, F58, and L59 and creates an additional interaction with R136 and E140 that could further stabilize the overall binding.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0011.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Comparison of the per-residue decomposition of two PLK1 ligands, <b>23</b> in blue and BI-2536 in red, with intermolecular vdw interactions as a function of the receptor sequence. (B) Docking of <b>23</b> (cyan) in PLK1 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a>); protein residues in contact with the ligand are represented by yellow sticks, and the rest are in gray. Hydrogen bonding is represented by a red dashed line.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Molecular modeling of compound <b>6</b> with BRD4-BD1 demonstrates that its docking pose demonstrates drastic variation from the cocrystal structure of BRD4-BD1 bound to BI-2536. When BI-2536 binds to BRD4-BD1, the dihydropteridinone and cyclopentane rings rest in between the WPF shelf (W81, P82, F83) and the BC loop (particularly N140 through the bridging water) of BRD4-BD1, with the ethyl group of BI-2536 pointing away to make additional contacts with V87, L92, L94, and Y97 (sharing the bridging water with N140) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). With BI-2536, Q85 interacts with the inhibitor through a water-mediated hydrogen bonding effect. However, the longer alkyl chain of compound <b>6</b> breaks this interaction due to steric clashes with L94 and Y97 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Therefore, compound <b>6</b> has to adapt a novel orientation in the BRD4-BD1 active site, which demonstrates much weakened interactions by a striking positive vdw energy contribution from Y97. Although the phenyl ring of compound <b>6</b> shifts into the pocket thereby enhancing binding with the WPF shelf, it still cannot overcome the repulsive effect of the positive vdw effect. We believe that these adjustments contributed significantly toward the reduced activity of <b>6</b> against BRD4-BD1.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0012.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Comparison of the per-residue decomposition of two BRD4-BD1 ligands, <b>6</b> in gray and BI-2536 in dark red, with intermolecular vdw interactions as a function of the receptor sequence. (B) Docking of <b>6</b> (gray) in BRD4 (1) (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a>); protein residues in contact with the ligand are represented by yellow sticks, and the rest are in gray. (C) Steric clashes caused by the propyl group if <b>6</b> is docked to BRD4 (1) in the same pose as BI-2536 (bumps are shown in red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Alternatively, the molecular docking of compound <b>6</b> with PLK1 shows great similarity to BI-2536 cocrystallized with PLK1. The C133 residue is conserved and acts as a hydrogen bonding receptor to the nitrogen in the dihydropteridinone in <b>6</b>, and the π–π stacking interaction between F83 and the same aromatic ring is also kept (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B). Furthermore, the propyl group in <b>6</b>, which replaced the original ethyl group in BI-2536 at site 3, does not reduce interactions with A80 and K82 much. The docking pose suggests that the carbonyl oxygen moved away from the active site but may form hydrogen bonds with either R136, R57, or crystal water, as was observed in the BI-2536–PLK1 complex structure, to reduce the entropy penalty when the new complex is forming. Again, the molecular docking analysis is in agreement with the binding affinity assays that illustrate a potent interaction between PLK1 and <b>6</b>. The detailed ligand interaction diagrams (LID) are listed in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information, Figures 3 and 4">Supporting Information, Figures 3 and 4</a>.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0001.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Comparison of the per-residue decomposition of two PLK1-ligands, <b>6</b> in gray and BI-2536 in dark red, with intermolecular vdw interactions as a function of the receptor sequence. (B) Docking of <b>6</b> (gray) in PLK1 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a>); protein residues in contact with the ligand are represented by yellow sticks with the rest in gray, and hydrogen bonding is represented by red dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62342" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62342" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our focused SAR study of the dual kinase–bromodomain inhibitor, BI-2536, has led to the identification of potent, balanced inhibitors of BRD4 and PLK1, as well as a more PLK1 selective inhibitor. Through these studies, we determined the requirements for both PLK1 and BET protein binding. We determined that the methylated amide and the cyclopentyl group of the BI-2536 scaffold are important for maintaining bromodomain binding activity but less important for PLK1 inhibitory activity. Moreover, substituting the asymmetric ethyl moiety at the dihydropteridine ring with a methyl group improved the inhibitory activity of the resulting compounds against both BRD4 and PLK1, as seen with compound <b>5</b>. Installing bulky alkyl groups at the asymmetric carbon resulted in inhibitors with potent and selective PLK1 activity but largely limited BRD4 activity, as seen with compound <b>6</b>. Further modification on the methoxy group of compound <b>5</b> led to the identification of compound <b>23</b>, a compound with increased potency for BRD4 and much reduced PLK1 activity. Our molecular docking studies suggested possible binding orientations with BRD4-BD1 and PLK1, and the binding energies were in agreement with the observed SAR established through our medicinal chemistry exercise.</div><div class="NLM_p last">In summary, we have demonstrated that fine-tuning the selectivity for inhibitors demonstrating polypharmacology can be achieved through structural guidance. We initiated an SAR campaign of BI-2536 in order to alter the PLK1 and BRD4 potency/selectivity of the compound and confirmed our biochemical findings through molecular modeling. In this study, we discovered a set of compounds with balanced selectivity against BRD4 and PLK1 that will entitle us to further explore the biological outcome of dual BRD4/PLK1 inhibition. In particular, compound <b>23</b> is a well-balanced BRD4 and PLK1 dual inhibitor when assayed biochemically and in cells. Compound <b>23</b> demonstrates an increased potency for BRD4 and a decreased potency for PLK1 as compared to parent molecule BI-2536. We envision that this set of molecules will generate sufficient PLK1 and BRD4 inhibition in order to study the simultaneous targeting of PLK1 and BRD4 in cancer.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01480" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01480" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> General Synthetic Information</h3><div class="NLM_p last">All chemicals and solvents were purchased from commercial suppliers and used as received. <sup>1</sup>H NMR (300 MHz) and <sup>13</sup>C NMR (75 MHz) spectra were recorded on a Varian NMR spectrometer, and <sup>1</sup>H NMR (400 MHz) and <sup>13</sup>C NMR spectra (101 MHz) were recorded on Agilent NMR spectrometers. LCMS analyses were performed on an Agilent 2100 LC with a 6130 quadrupole MS spectrometer. Accurate mass measurements were performed on a Fusion Lumos Orbitrap mass spectrometer (Thermo Fisher Scientific) at a mass resolution setting of 500 000 immediately after calibration or under lock mass calibration using solvent ions. A C<sub>18</sub> column (5.0 μm, 6.0 × 50 mm<sup>2</sup>) was used for separation. The mobile phases were MeOH and H<sub>2</sub>O, both containing 0.05% CF<sub>3</sub>CO<sub>2</sub>H. A linear gradient from 25/75 (v/v) MeOH/H<sub>2</sub>O to 100% MeOH over 7.0 min at a flow rate of 0.7 mL/min was used as a mobile phase. UV detections were conducted at 210, 254, and 365 nm. Low-resolution mass spectra were recorded by APCI (atmospheric pressure chemical ionization) unless otherwise specified. Flash chromatography separation was performed on a YAMAZEN AI-580 system with Agela silica gel (12 or 20 g, 230–400 mesh) cartridges. Final products were purified on an Agela HP 100 preparative LC system with a Venusil PrepG C<sub>18</sub> column (10 μm, 120 Å, 21.2 × 250 mm<sup>2</sup>). All biologically evaluated compounds were found to be >95% pure as determined by NMR and LCMS.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Synthesis of BI-2536 Analogues</h3><div class="NLM_p">The synthesis of compounds <b>1</b>–<b>20</b> shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> was carried out following the reported procedures (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The synthesis of analogues of compound <b>5</b>, including compounds <b>21</b> and <b>23</b>–<b>25</b> listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, was performed following the route shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Bromocyclopentane (1.3 equiv) was added slowly to a stirred suspension of potassium carbonate (1.5 equiv) in DMF containing methyl-3-hydroxy-4-nitrobenzoate (1 equiv) at 65 °C for 4 h, and then the product was treated by SnCl<sub>2</sub>·2H<sub>2</sub>O in EtOAc at 50 °C overnight. A mixture of Ar–Cl (1 equiv), X-Phos (28 mol %), Pd<sub>2</sub>(dba)<sub>3</sub> (7 mol %), methyl 4-amino-3-(cyclopentyloxy)benzoate (1.3 equiv), and K<sub>2</sub>CO<sub>3</sub>(5 equiv) in <i>t</i>-BuOH was heated at 100 °C in a sealed tube for 8 h. The mixture was then diluted with EtOAc and washed three times with aqueous NaHCO<sub>3</sub>. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and condensed. The crude product was purified by a preparative LC system to obtain the final product.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> (<i>R</i>)-4-((8-Cyclopentyl-5,7-dimethyl-6-oxo-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>5</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.41 (1H, d, <i>J</i> = 9.0 Hz), 7.72 (1H, s), 7.41–7.31 (3H, m), 4.50–4.45 (1H, m), 4.31–4.24 (1H, q, <i>J</i> = 6.7 Hz), 4.02–3.82 (6H, s, N–CH<sub>3</sub> and O–CH<sub>3</sub>), 3.27–3.26 (6H, m), 3.22 (3H, s), 2.57–2.47 (6H, m), 2.06–1.86 (4H, m), 1.25 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ 170.9, 166.6, 156.3, 153.0, 139.44, 138.7, 127.3, 121.2, 117.3, 117.3, 109.9, 59.5, 59.3, 56.4, 55.6, 55.02, 47.1, 44.9, 31.2, 30.5, 29.9, 28.5, 24.3, 23.8, 19.0; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>37</sub>N<sub>7</sub>O<sub>3</sub> 536.3349, found 508.3031 (M<sup>+</sup> + 1).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> (<i>R</i>)-4-((8-Cyclopentyl-5-methyl-6-oxo-7-propyl-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>6</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.54 (1H, d, <i>J</i> = 8.4 Hz), 7.68 (1H, s), 7.59 (1H, s), 7.51 (1H, s), 7.42 (1H, s), 7.42 (1H, d, <i>J</i> = 1.8 Hz), 4.56–4.46 (1H, m), 4.24 (1H, dd, <i>J</i> = 8.0, 3.7 Hz), 3.97 (3H, s), 3.31 (3H, s), 2.86–2.84 (2H, m), 2.31 (3H, s), 2.25–2.10 (4H, m), 2.07–2.04 (4H, m), 1.81–1.71 (14H, m), 1.37–1.21 (4H, m), 0.87 (3H, t, <i>J</i> = 7.3 Hz); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.9, 155.0, 152.1, 147.2, 137.9, 133.2, 126.2, 126.0, 118.8, 116.4, 115.8, 109.0, 58.8, 58.1, 54.4, 35.8, 32.4, 29.6, 29.1, 28.2, 23.5, 23.0, 17.8, 13.9; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>29</sub>H<sub>41</sub>N<sub>7</sub>O<sub>3</sub> 536.3349, found 536.3342 (M<sup>+</sup> + 1).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> (<i>R</i>)-4-((8-Cyclopentyl-7-isobutyl-5-methyl-6-oxo-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>8</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.40 (1H, d, <i>J</i> = 9.0 Hz), 7.74 (1H, s), 7.42–7.38 (3H, m), 4.21–4.17 (1H, m), 3.91 (3H, s), 3.87–3.83 (1H, m), 3.23 (3H, s), 2.91–2.88 (1H, m), 2.27 (3H, s), 2.21–2.13 (2H, m), 2.02–1.97 (2H, m), 1.92–1.83 (2H, m), 1.74–1.54 (8H, m), 1.41–1.36 (1H, m), 0.92 (3H, d, <i>J</i> = 6.3 Hz), 0.78 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ 169.1, 165.2, 156.2, 153.6, 148.4, 139.6, 134.0, 127.4, 121.2, 117.4, 117.3, 109.9, 59.6, 58.4, 56.4, 55.5, 45.8, 42.5, 32.0, 30.49, 30.4, 28.6, 25.7, 24.2, 23.9, 23.7, 21.7; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>43</sub>N<sub>7</sub>O<sub>3</sub> 550.3506, found 550.3501 (M<sup>+</sup> + 1).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> (<i>R</i>)-4-((8-Isopropyl-5,7-dimethyl-6-oxo-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>12</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.42 (1H, d, <i>J</i> = 9.3 Hz), 7.69 (1H, s), 7.38 (3H, m), 3.99–3.90 (5H, m), 3.90 (3H, s), 2.72–2.65 (4H, m), 2.55 (3H, s), 1.86–1.76 (4H, m), 1.30–1.23 (9H, m); <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ 169.2, 166.4, 156.3, 152.4, 148.3, 139.5, 134.1, 127.1, 121.3, 117.2, 109.8, 56.4, 54.7, 53.6, 46.6, 44.4, 30.7, 28.5, 21.5, 19.9; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>25</sub>H<sub>35</sub>N<sub>7</sub>O<sub>3</sub> 482.2880, found 482.2884 (M<sup>+</sup> + 1).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (<i>R</i>)-4-((7-Isobutyl-5-methyl-6-oxo-8-(tetrahydro-2<i>H</i>-pyran-4-yl)-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>13</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.42 (1H, d, <i>J</i> = 9.0 Hz), 7.78 (1H, s), 7.45–7.42 (3H, m), 3.92 (3H, s), 3.56–3.53 (2H, m), 3.22 (3H, s), 3.03–2.99 (2H, m), 2.37 (3H, m), 1.96–1.92 (3H, m), 1.70–1.65 (4H, m), 1.55–1.38 (5H, m), 0.94 (3H, d, <i>J</i> = 6.3 Hz), 0.79 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ 171.5, 171.4, 170.9, 169.1, 164.86, 153.0, 150.0, 140.1, 127.5, 121.2, 117.4, 110.0, 68.6, 68.5, 57.3, 56.4, 55.3, 55.2, 45.4, 43.4, 32.6, 31.6, 28.7, 25.8, 23.9, 21.7; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>43</sub>N<sub>7</sub>O<sub>4</sub> 566.3455, found 566.3448 (M<sup>+</sup> + 1).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (<i>S</i>)-4-((8-Cyclopentyl-5,7-dimethyl-6-oxo-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>21</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.43 (1H, d, <i>J</i> = 8.7 Hz), 7.75 (1H, s), 7.43–7.41 (3H, m), 4.31–4.29 (1H, q, <i>J</i> = 6.0 Hz), 3.94 (3H, s), 3.24 (3H, s), 3.12–3.07 (3H, m), 2.90–2.87 (3H, m), 2.12–2.09 (3H, m), 1.91–1.89 (4H, m), 1.79–1.67 (6H, m), 1.26 (3H, d, <i>J</i> = 6.9 Hz); <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ 171.1, 169.1, 166.6, 156.2, 153.0, 148.3, 139.4, 134.0, 127.4, 121.2, 117.2, 109.9, 59.3, 56.4, 55.6, 55.4, 47.8, 45.7, 31.9, 30.5, 29.9, 28.5, 24.2, 23.8, 19.0; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>43</sub>N<sub>7</sub>O<sub>3</sub> 562.3506, found 562.3501 (M<sup>+</sup> + 1).</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (<i>R</i>)-4-((8-Cyclopentyl-5,7-dimethyl-6-oxo-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-(cyclopentyloxy)-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>2</b><b>3</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.53 (1H, d, <i>J</i> = 8.0 Hz), 7.71 (1H, s), 7.60 (1H, s), 7.41 (1H, d, <i>J</i> = 1.9 Hz), 4.62–4.51 (1H, m), 4.32–4.30 (1H, m), 4.14–4.06 (1H, m), 3.97 (3H, s), 3.31 (3H, s), 2.83–2.81 (3H, m), 2.29 (3H, s), 2.20–2.10 (3H, m), 2.08–1.93 (4H, m), 1.32 (3H, d, <i>J</i> = 6.8 Hz); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.6, 165.0, 155.2, 151.8, 147.2, 138.3, 133.1, 126.6, 118.8, 116.3, 116.0, 109.1, 57.6, 55.9, 54.4, 46.2, 32.5, 30.0, 29.5, 28.3, 23.7, 23.2, 19.0. HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>43</sub>N<sub>7</sub>O<sub>3</sub> 562.3506, found 562.3508 (M<sup>+</sup> + 1).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (<i>R</i>)-4-((8-Cyclopentyl-5-methyl-6-oxo-7-propyl-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-(cyclopentyloxy)-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>2</b><b>4</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.44 (1H, d, <i>J</i> = 8.1 Hz), 7.72 (1H, s), 7.42–7.39 (3H, m), 4.39–4.34 (1H, m), 4.27–4.24 (1H, m), 3.85–3.80 (1H, m), 3.25 (3H, s), 2.86–2.60 (2H, m), 2.26 (3H, s), 2.16–2.09 (3H, m), 2.01–1.80 (14H, m), 1.76–1.59 (8H, m), 1.26–1.17 (2H, m), 0.83 (3H, t, <i>J</i> = 7.2 Hz); <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ 169.12, 165.4, 156.0, 153.4, 146.5, 139.0, 134.8, 127.3, 120.8, 117.2, 112.4, 81.8, 60.67, 60.5, 55.6, 49.7, 46.0, 36.8, 33.6, 32.2, 30.2, 29.9, 28.4, 24.8, 24.4, 24.2, 18.7, 14.0; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>33</sub>H<sub>47</sub>N<sub>7</sub>O<sub>3</sub> 590.3819, found 590.3815 (M<sup>+</sup> + 1).</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 4-((8-Cyclopentyl-5-methyl-6-oxo-5,6,7,8-tetrahydropteridin-2-yl)amino)-3-(cyclopentyloxy)-<i>N</i>-(1-methylpiperidin-4-yl)benzamide (<b>2</b><b>5</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.43 (1H, d, <i>J</i> = 8.4 Hz), 7.64 (1H, s), 7.42–7.39 (3H, m), 4.06 (3H, s), 3.28–3.25 (4H, m), 3.23 (3H, s), 3.02–2.97 (3H, m), 2.37 (3H, s), 2.30–2.26 (3H, m), 1.98–1.93 (7H, m), 1.74–1.68 (9H, m); <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ 169.1, 163.5, 156.1, 153.1, 146.4, 138.5, 134.8, 127.1, 120.8, 117.0, 116.8, 112.3, 81.8, 56.9, 55.37, 47.6, 45.9, 45.5, 33.6, 31.7, 28.0, 27.9, 24.8, 24.8; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>41</sub>N<sub>7</sub>O<sub>3</sub> 548.3350, found 548.3346 (M<sup>+</sup> + 1).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Molecular Docking Software</h3><div class="NLM_p last">For protein preparation, UCSF Chimera Dock Prep tool<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and UCSF DOCK6<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> were used.For compound structures and the charge assignment method, ChemDraw 15.0, Chem3D 14.0, GAFF, and AM1-BCC were used. For docking and energy decomposition analysis, the UCSF DOCK6 suite was used. For dock poses, visualization, and illustration, UCSF Chimera ViewDock and PyMOL were used.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Compounds and Protein Preparation</h3><div class="NLM_p last">The synthesized compounds were built in ChemDraw, and then their Cartesian coordinates’ construction and energy minimization using MM2 force field<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> were generated in Chem3D. The crystal structures of BI-2536 binding with BRD4-BD1 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a>) and PLK1 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a>) were downloaded from the Protein Data Bank (PDB) archive. Protein targets were prepared using the Chimera DockPrep tool, in which AMBER force field ff14SB<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> was employed to provide accurate descriptions for proteins. GAFF<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a>and AM1-BCC<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> were used to generate parameters and assign charges for all inhibitor molecules.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Docking and Energy Decomposition Analysis</h3><div class="NLM_p last">The DOCK6 suite was used for docking compounds to their protein targets. Flexible ligand docking was performed under the guidance of Anchor-and-Grow algorithm. Dock Score was used as the criterion for ranking the binding poses. We visually examined the top 10 ranked poses, and the highest ranked poses are reported in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. In the case of PLK1 and BRD4-BD1, DOCK6 successfully sampled and top-scored the BI-2536 docking poses to reproduce the corresponding crystal binding poses with an rmsd <2 Å. DOCK Score is composed of van der Waals (vdw) and electrostatic interactions. The vdw composition of Dock Score was further decomposed onto every residue of its binder protein by using DOCK write_footprint function, which generates both the vdw and electrostatic parts. Only the vdw composition was noticeably diversified and thus referred to represent the contribution of the protein target per residue in protein–ligand interaction energy profiles as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>–<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Specific Favorable and Unfavorable Interactions Recognition</h3><div class="NLM_p last">As listed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, the specific residual information of the protein receptors interacting with BI-2536 analogues was recognized by referring to vdw energy and by the visualization of binding poses. A threshold of <−4 kcal/mol in decomposed vdw energy was used to include the first shell of residues around the molecules (also see <a class="ref internalNav" href="#notes1" aria-label="Supporting Information Figure S-3">Supporting Information Figure S-3</a>). The positive peak observed in the per-residue decomposition of BRD4-DB1 compound <b>6</b>, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, was ascribed to the steric clashes of residues Y97, L94 with the propyl group on <b>6</b>. The disc was scripted in PyMOL software to show vdw overlaps or steric clashing (also see <a class="ref internalNav" href="#notes1" aria-label="Supporting Information Figure S-4">Supporting Information Figure S-4</a>).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> BRD4-BD1 and BRDT-BD1 Activity Assays</h3><div class="NLM_p last">Compound potency was assessed by relative IC<sub>50</sub> potency determined by an AlphaScreen biotin–JQ1 competition assay as reported previously by our group.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> All reagents were diluted in 50 mM HEPES, 150 mM NaCl, 0.1% w/v BSA, and 0.01% w/v Tween20, pH 7.5, and allowed to equilibrate to room temperature prior to the addition to plates. After the addition of Alpha beads to the master solutions, all subsequent steps were performed in low-light conditions. A 2× solution of components with final concentrations of BRD at 40 nM, Ni-coated acceptor bead at 25 μg/mL, and 20 nM biotin–JQ1 was added in 10 μL to 384-well plates (AlphaPlate-384, PerkinElmer, USA). After a 1 min 1000 rpm spin down, 100 nL portions of compounds in DMSO from stock plates were added by pin transfer using a Janus Workstation (PerkinElmer, USA). The 2×, 10 μL of streptavidin-coated donor beads (25 μg/mL) were added to the previous solution. Following this addition, the plates were sealed with foil to block light exposure and to prevent evaporation. The plates were spun down again at 1000 rpm for 1 min. Next, the plates were incubated at room temperature with the plate reader (for temperature equilibration) for 1 h prior to reading the assay. The signal is stable for up to 3 h after donor bead addition. AlphaScreen measurements were performed on an Envision 2104 (PerkinElmer, USA) utilizing the manufacturer’s protocol.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> PLK1 Activity Assay</h3><div class="NLM_p">The enzymatic activities against PLK1 were tested in Z-Lyte assays with an ATP concentration of <i>K</i><sub>M</sub> for each kinase per manufacturer’s protocol (Thermo Fisher).</div><div class="NLM_p last">All the compounds were tested in parallel with the cross-linker assay to rule out the assay binder/interrupter.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Selectivity Profiling</h3><div class="NLM_p last">Compound <b>5</b> was assayed at 1 μM in DiscoverX’s BROMOscan (<i>bromo</i>MAX) and KINOMEscan (<i>scan</i>MAX) assay platforms. The results for single-concentration binding interactions for compound <b>5</b> are reported as the % of control (DMSO).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Cell Cycle Study</h3><div class="NLM_p last">MOLM13 cells were treated with 1 μM JQ1, BI2536, TAE684, compound <b>5</b>, compound <b>6</b>, or compound <b>23</b> for 24 h. Cells were subsequently harvested and stained with propidium iodide (Calbiochem); this was followed by the measurement of propidium-iodide-mediated fluorescence with a Guava H6T (Millipore Sigma).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00765" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44286" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44286" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00765" class="ext-link">10.1021/acs.jmedchem.8b00765</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">BROMOscan and KINOMEscan data for <b>5</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00765/suppl_file/jm8b00765_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00765/suppl_file/jm8b00765_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00765/suppl_file/jm8b00765_si_001.pdf">jm8b00765_si_001.pdf (3.47 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00765/suppl_file/jm8b00765_si_002.csv">jm8b00765_si_002.csv (2.23 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are the following: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a> for BI2536, BRD4-BD1; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a> for PLK1-BI2536.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b00765" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85835" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85835" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James E. Bradner</span> - <span class="hlFld-Affiliation affiliation">Department of Medical Oncology, 
     and , Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Novartis
Institutes for Biomedical Research, Cambridge, Massachusetts 02139, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2718-4415" title="Orcid link">http://orcid.org/0000-0002-2718-4415</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#85efe4e8e0f6abe7f7e4e1ebe0f7c5ebeaf3e4f7f1ecf6abe6eae8"><span class="__cf_email__" data-cfemail="f399929e9680dd918192979d9681b39d9c859281879a80dd909c9e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jun Qi</span> - <span class="hlFld-Affiliation affiliation">Department of Cancer Biology, 
     and , Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medicine, Harvard Medical School, Boston, Massachusetts 02115, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1461-3356" title="Orcid link">http://orcid.org/0000-0002-1461-3356</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b2d8c7dcedc3dbf2d6d4d1db9cdad3c0c4d3c0d69cd7d6c7"><span class="__cf_email__" data-cfemail="771d021928061e371311141e591f16050116051359121302">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6097-2763" title="Orcid link">http://orcid.org/0000-0002-6097-2763</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0d7a68643f2377656c636a4d78606f23686978"><span class="__cf_email__" data-cfemail="8cfbe9e5bea2f6e4ede2ebccf9e1eea2e9e8f9">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuai Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hailemichael O. Yosief</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lingling Dai</span> - <span class="hlFld-Affiliation affiliation">Department of Cancer Biology, 
     and , Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Phase
I Clinical Trial Center & Department of Clinical Pharmacology,
Xiangya Hospital, Central South University, Changsha, Hunan 410008, P.R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">He Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Stony Brook University, Stony Brook, New York 11794-3400, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gagan Dhawan</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biomedical Science, Acharya Narendra Dev College, University of Delhi, New Delhi 110019, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaofeng Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex M. Muthengi</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Massachusetts—Boston, Boston, Massachusetts 02125, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justin Roberts</span> - <span class="hlFld-Affiliation affiliation">Department of Medical Oncology and , Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dennis L. Buckley</span> - <span class="hlFld-Affiliation affiliation">Department of Medical Oncology and , Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer A. Perry</span> - <span class="hlFld-Affiliation affiliation">Department of Cancer Biology and , Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei Wu</span> - <span class="hlFld-Affiliation affiliation">Department of Cancer Biology and , Dana-Farber Cancer Institute, Boston, Massachusetts 02215, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81324" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81324" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was partially supported by National Institutes of Health U54 grant CA156732 (J.E.B. and W.Z.), U54 grant (HD093540-01 for J.Q.), and P01 grant (CA066996 for J.Q.). We thank Jason Evans and Dennis Zeh for performing the HRMS analysis of some synthetic compounds.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal domain</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain 4</p></td></tr><tr><td class="NLM_term">PLK1</td><td class="NLM_def"><p class="first last">polo-like kinase</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">JAK2</td><td class="NLM_def"><p class="first last">janus kinase 2</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">LID</td><td class="NLM_def"><p class="first last">ligand interaction diagrams</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30804" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30804" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Beating the odds: BETs in disease</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">468</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.tibs.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=26145250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyiurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2015&pages=468-479&author=C.+Y.+Wangauthor=P.+Filippakopoulos&title=Beating+the+odds%3A+BETs+in+disease&doi=10.1016%2Fj.tibs.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Beating the odds: BETs in disease</span></div><div class="casAuthors">Wang, Chen-Yi; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">468-479</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are evolutionarily conserved protein interaction modules that specifically recognize acetyl-lysine on histones and other proteins, facilitating roles in regulating gene transcription.  BRD-contg. proteins bound to chromatin loci such as enhancers are often deregulated in disease leading to aberrant expression of proinflammatory cytokines and growth-promoting genes.  Recent developments targeting the bromo and extraterminal (BET) subset of BRD proteins demonstrated remarkable efficacy in murine models providing a compelling rationale for drug development and translation to the clinic.  Here we summarise recent advances in our understanding of the roles of BETs in regulating gene transcription in normal and diseased tissue as well as the current status of their clin. translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhNr2U06Dv6bVg90H21EOLACvtfcHk0lhfwTfxQ-KcNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyiurfN&md5=5d3c4c0db326118f6b89792cee30a8a0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DBeating%2520the%2520odds%253A%2520BETs%2520in%2520disease%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2015%26volume%3D40%26spage%3D468%26epage%3D479%26doi%3D10.1016%2Fj.tibs.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvanese, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dar, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aull, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planelles, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siliciano, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain-targeting compounds reactivate HIV from latency via a Tat-independent mechanism</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">452</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.4161/cc.23309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.4161%2Fcc.23309" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=452-462&author=D.+Boehmauthor=V.+Calvaneseauthor=R.+D.+Darauthor=S.+Xingauthor=S.+Schroederauthor=L.+Martinsauthor=K.+Aullauthor=P.+C.+Liauthor=V.+Planellesauthor=J.+E.+Bradnerauthor=M.+M.+Zhouauthor=R.+F.+Silicianoauthor=L.+Weinbergerauthor=E.+Verdinauthor=M.+Ott&title=BET+bromodomain-targeting+compounds+reactivate+HIV+from+latency+via+a+Tat-independent+mechanism&doi=10.4161%2Fcc.23309"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.4161%2Fcc.23309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.23309%26sid%3Dliteratum%253Aachs%26aulast%3DBoehm%26aufirst%3DD.%26aulast%3DCalvanese%26aufirst%3DV.%26aulast%3DDar%26aufirst%3DR.%2BD.%26aulast%3DXing%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DS.%26aulast%3DMartins%26aufirst%3DL.%26aulast%3DAull%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DP.%2BC.%26aulast%3DPlanelles%26aufirst%3DV.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DSiliciano%26aufirst%3DR.%2BF.%26aulast%3DWeinberger%26aufirst%3DL.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DBET%2520bromodomain-targeting%2520compounds%2520reactivate%2520HIV%2520from%2520latency%2520via%2520a%2520Tat-independent%2520mechanism%26jtitle%3DCell%2520Cycle%26date%3D2013%26volume%3D12%26spage%3D452%26epage%3D462%26doi%3D10.4161%2Fcc.23309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delmore, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilpatrick, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinzel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsagel, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0litvFaxXuPW8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14532</span>– <span class="NLM_lpage">14537</span>, <span class="refDoi"> DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0litvFaxXuPW8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallenkamp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelato, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span> <span> </span><span class="NLM_article-title">Bromodomains and their pharmacological inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fcmdc.201300434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24497428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=438-464&author=D.+Gallenkampauthor=K.+A.+Gelatoauthor=B.+Haendlerauthor=H.+Weinmann&title=Bromodomains+and+their+pharmacological+inhibitors&doi=10.1002%2Fcmdc.201300434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains and Their Pharmacological Inhibitors</span></div><div class="casAuthors">Gallenkamp, Daniel; Gelato, Kathy A.; Haendler, Bernard; Weinmann, Hilmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">438-464</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Over 60 bromodomains belonging to proteins with very different functions have been identified in humans.  Several of them interact with acetylated lysine residues, leading to the recruitment and stabilization of protein complexes.  The bromodomain and extra-terminal domain (BET) proteins contain tandem bromodomains which bind to acetylated histones and are thereby implicated in a no. of DNA-centered processes, including the regulation of gene expression.  The recent identification of inhibitors of BET and non-BET bromodomains is one of the few examples in which effective blockade of a protein-protein interaction can be achieved with a small mol.  This has led to major strides in the understanding of the function of bromodomain-contg. proteins and their involvement in diseases such as cancer and inflammation.  Indeed, BET bromodomain inhibitors are now being clin. evaluated for the treatment of hematol. tumors and have also been tested in clin. trials for the relatively rare BRD-NUT midline carcinoma.  This review gives an overview of the newest developments in the field, with a focus on the biol. of selected bromodomain proteins on the one hand, and on reported pharmacol. inhibitors on the other, including recent examples from the patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-8_ymytCPbVg90H21EOLACvtfcHk0litvFaxXuPW8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D&md5=2ce7c6b5c37001fb8fb901fe04aeb606</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300434%26sid%3Dliteratum%253Aachs%26aulast%3DGallenkamp%26aufirst%3DD.%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DBromodomains%2520and%2520their%2520pharmacological%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D438%26epage%3D464%26doi%3D10.1002%2Fcmdc.201300434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0ljs9nZjOcHErA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0ljs9nZjOcHErA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding Sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+Sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0ljs9nZjOcHErA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520Sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The bromodomain interaction module</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>586</i></span>,  <span class="NLM_fpage">2692</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2012.04.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.febslet.2012.04.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22710155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=2692-2704&author=P.+Filippakopoulosauthor=S.+Knapp&title=The+bromodomain+interaction+module&doi=10.1016%2Fj.febslet.2012.04.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain interaction module</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2692-2704</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ε-N-acetylation of protein Lys residues (Kac) is one of the most abundant post-translation modifications (PTMs) in the human proteome.  In the nucleus, the acetylation of histones has been linked to transcriptional activation of genes but the functional consequences of most acetylation events and proteins recruited to these sites remains largely unknown.  Bromodomains (BRDs) are small helical interaction modules that specifically recognize acetylation sites in proteins.  BRDs have recently emerged as interesting targets for the development of specific protein interaction inhibitors, enabling a novel exiting strategy for the development of new therapies.  Here, the author provides an overview over sequence requirements of BRDs, known substrates, and the structural mechanisms of specific Kac recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7LaThygz_HrVg90H21EOLACvtfcHk0lgyodUUcdsVcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D&md5=c02dee22c330b1d5c1ba39579360ebc5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520bromodomain%2520interaction%2520module%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D2692%26epage%3D2704%26doi%3D10.1016%2Fj.febslet.2012.04.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Progress in the development and application of small molecule inhibitors of bromodomain–acetyl lysine interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9393</span>– <span class="NLM_lpage">9413</span>, <span class="refDoi"> DOI: 10.1021/jm300915b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300915b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9393-9413&author=D.+S.+Hewingsauthor=T.+P.+Rooneyauthor=L.+E.+Jenningsauthor=D.+A.+Hayauthor=C.+J.+Schofieldauthor=P.+E.+Brennanauthor=S.+Knappauthor=S.+J.+Conway&title=Progress+in+the+development+and+application+of+small+molecule+inhibitors+of+bromodomain%E2%80%93acetyl+lysine+interactions&doi=10.1021%2Fjm300915b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm300915b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300915b%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DJennings%26aufirst%3DL.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DProgress%2520in%2520the%2520development%2520and%2520application%2520of%2520small%2520molecule%2520inhibitors%2520of%2520bromodomain%25E2%2580%2593acetyl%2520lysine%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9393%26epage%3D9413%26doi%3D10.1021%2Fjm300915b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span> <span> </span><span class="NLM_article-title">Small molecule bromodomain inhibitors: extending the druggable genome</span>. <i>Prog. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-396493-9.00001-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2FB978-0-12-396493-9.00001-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22520470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChtr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1-55&author=C.+W.+Chung&title=Small+molecule+bromodomain+inhibitors%3A+extending+the+druggable+genome&doi=10.1016%2FB978-0-12-396493-9.00001-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule bromodomain inhibitors: extending the druggable genome</span></div><div class="casAuthors">Chung, Chun-Wa</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-55</span>CODEN:
                <span class="NLM_cas:coden">PMDCAY</span>;
        ISSN:<span class="NLM_cas:issn">0079-6468</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review on some of the existing knowledge for the young and rapidly evolving field of bromodomain inhibition.  Topics covered include structure and function of bromodomain, bromodomain-contg. proteins, therapeutic potential of bromodomains, and bromodomain inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI9QR6f10R97Vg90H21EOLACvtfcHk0lgyodUUcdsVcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChtr7K&md5=6a947139e3939e752fb09a749938b3a9</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-396493-9.00001-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-396493-9.00001-7%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BW.%26atitle%3DSmall%2520molecule%2520bromodomain%2520inhibitors%253A%2520extending%2520the%2520druggable%2520genome%26jtitle%3DProg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D51%26spage%3D1%26epage%3D55%26doi%3D10.1016%2FB978-0-12-396493-9.00001-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yik, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">Recruitment of P-TEFb for stimulation of transcriptional elongation by the bromodomain protein Brd4</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.06.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.molcel.2005.06.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=16109377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=535-545&author=Z.+Yangauthor=J.+H.+Yikauthor=R.+Chenauthor=N.+Heauthor=M.+K.+Jangauthor=K.+Ozatoauthor=Q.+Zhou&title=Recruitment+of+P-TEFb+for+stimulation+of+transcriptional+elongation+by+the+bromodomain+protein+Brd4&doi=10.1016%2Fj.molcel.2005.06.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Recruitment of P-TEFb for stimulation of transcriptional elongation by the bromodomain protein Brd4</span></div><div class="casAuthors">Yang, Zhiyuan; Yik, Jasper H. N.; Chen, Ruichuan; He, Nanhai; Jang, Moon Kyoo; Ozato, Keiko; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">535-545</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cyclinT1/Cdk9 heterodimer that constitutes core P-TEFb is generally presumed to be the transcriptionally active form for stimulating RNA polymerase II elongation.  About half of cellular P-TEFb also exists in an inactive complex with the 7SK snRNA and the HEXIM1 protein.  Here, we show that the remaining half assocs. with the bromodomain protein Brd4.  In stress-induced cells, the 7SK/HEXIM1-bound P-TEFb is quant. converted into the Brd4-assocd. form.  The assocn. with Brd4 is necessary to form the transcriptionally active P-TEFb, recruits P-TEFb to a promoter, and enables P-TEFb to contact the Mediator complex, a potential target for the Brd4-mediated recruitment.  Although generally required for transcription, the P-TEFb-recruitment function of Brd4 can be substituted by that of HIV-1 Tat, which recruits P-TEFb directly for activated HIV-1 transcription.  Brd4, HEXIM1, and 7SK are all implicated in regulating cell growth, which may result from their dynamic control of the general transcription factor P-TEFb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptFELcZRBnvLVg90H21EOLACvtfcHk0lgyodUUcdsVcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur8%253D&md5=50325264e1eb27789eb46999ca3f799d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.06.029%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DYik%26aufirst%3DJ.%2BH.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DRecruitment%2520of%2520P-TEFb%2520for%2520stimulation%2520of%2520transcriptional%2520elongation%2520by%2520the%2520bromodomain%2520protein%2520Brd4%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D535%26epage%3D545%26doi%3D10.1016%2Fj.molcel.2005.06.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein Brd4 is a positive regulatory component of P-TEFb and stimulates RNA polymerase II-dependent transcription</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.06.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.molcel.2005.06.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=16109376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=523-534&author=M.+K.+Jangauthor=K.+Mochizukiauthor=M.+Zhouauthor=H.+S.+Jeongauthor=J.+N.+Bradyauthor=K.+Ozato&title=The+bromodomain+protein+Brd4+is+a+positive+regulatory+component+of+P-TEFb+and+stimulates+RNA+polymerase+II-dependent+transcription&doi=10.1016%2Fj.molcel.2005.06.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein Brd4 is a positive regulatory component of P-TEFb and stimulates RNA polymerase II-dependent transcription</span></div><div class="casAuthors">Jang, Moon Kyoo; Mochizuki, Kazuki; Zhou, Meisheng; Jeong, Ho-Sang; Brady, John N.; Ozato, Keiko</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">523-534</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Brd4 is a mammalian bromodomain protein that binds to acetylated chromatin.  Proteomic anal. revealed that Brd4 interacts with cyclinT1 and Cdk9 that constitutes core pos. transcription elongation factor b (P-TEFb).  Brd4 interacted with P-TEFb in the living nucleus through its bromodomain.  About half of P-TEFb was bound to the inhibitory subunit and functionally inactive.  Brd4 interacted with P-TEFb that was free of the inhibitory subunit.  An increase in Brd4 expression led to increased P-TEFb-dependent phosphorylation of RNA polymerase II (RNAPII) CTD and stimulation of transcription from promoters in vivo.  Conversely, a redn. in Brd4 expression by siRNA reduced CTD phosphorylation and transcription, revealing that Brd4 is a pos. regulatory component of P-TEFb.  In chromatin immunopptn. (ChlP) assays, the recruitment of P-TEFb to a promoter was dependent on Brd4 and was enhanced by an increase in chromatin acetylation.  Together, P-TEFb alternately interacts with Brd4 and the inhibitory subunit to maintain functional equil. in the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeaKLJgZobzrVg90H21EOLACvtfcHk0ljs-HfOaC55bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur4%253D&md5=90a7c10d7e7dd0e47d28ac708f9965ae</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.06.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.06.027%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DMochizuki%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DJeong%26aufirst%3DH.%2BS.%26aulast%3DBrady%26aufirst%3DJ.%2BN.%26aulast%3DOzato%26aufirst%3DK.%26atitle%3DThe%2520bromodomain%2520protein%2520Brd4%2520is%2520a%2520positive%2520regulatory%2520component%2520of%2520P-TEFb%2520and%2520stimulates%2520RNA%2520polymerase%2520II-dependent%2520transcription%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D523%26epage%3D534%26doi%3D10.1016%2Fj.molcel.2005.06.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljs-HfOaC55bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0ljs-HfOaC55bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delannee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tellier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudit, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">From ApoA1 upregulation to BET family bromodomain inhibition: Discovery of I-BET151</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2963</span>– <span class="NLM_lpage">2967</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.01.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.bmcl.2012.01.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22386529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1elsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2963-2967&author=O.+Mirguetauthor=Y.+Lamotteauthor=F.+Doncheauthor=J.+Toumauthor=F.+Gellibertauthor=A.+Bouillotauthor=R.+Gosminiauthor=V.+L.+Nguyenauthor=D.+Delanneeauthor=J.+Sealauthor=F.+Blandelauthor=A.+B.+Boullayauthor=E.+Boursierauthor=S.+Martinauthor=J.+M.+Brusqauthor=G.+Krysaauthor=A.+Riouauthor=R.+Tellierauthor=A.+Costazauthor=P.+Huetauthor=Y.+Duditauthor=L.+Trottetauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=From+ApoA1+upregulation+to+BET+family+bromodomain+inhibition%3A+Discovery+of+I-BET151&doi=10.1016%2Fj.bmcl.2012.01.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">From ApoA1 upregulation to BET family bromodomain inhibition: Discovery of I-BET151</span></div><div class="casAuthors">Mirguet, Olivier; Lamotte, Yann; Donche, Frederic; Toum, Jerome; Gellibert, Francoise; Bouillot, Anne; Gosmini, Romain; Nguyen, Van-Loc; Delannee, Delphine; Seal, Jonathan; Blandel, Florence; Boullay, Anne-Benedicte; Boursier, Eric; Martin, Sandrine; Brusq, Jean-Marie; Krysa, Gael; Riou, Alizon; Tellier, Remi; Costaz, Agnes; Huet, Pascal; Dudit, Yann; Trottet, Lionel; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2963-2967</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery, synthesis, and biol. evaluation of a novel series of 7-isoxazoloquinolines is described.  Several analogs are shown to increase ApoA1 expression within the nanomolar range in the human hepatic cell line HepG2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIu6f80tbgE7Vg90H21EOLACvtfcHk0lj9U6r-y282Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1elsbY%253D&md5=48c9a8dc4b70bea4a86807f7a1cec5f1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.01.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.01.125%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DDelannee%26aufirst%3DD.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DBlandel%26aufirst%3DF.%26aulast%3DBoullay%26aufirst%3DA.%2BB.%26aulast%3DBoursier%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DRiou%26aufirst%3DA.%26aulast%3DTellier%26aufirst%3DR.%26aulast%3DCostaz%26aufirst%3DA.%26aulast%3DHuet%26aufirst%3DP.%26aulast%3DDudit%26aufirst%3DY.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DFrom%2520ApoA1%2520upregulation%2520to%2520BET%2520family%2520bromodomain%2520inhibition%253A%2520Discovery%2520of%2520I-BET151%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2963%26epage%3D2967%26doi%3D10.1016%2Fj.bmcl.2012.01.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bish, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nocka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primiano, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralph, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciammetta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span> <span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra C-terminal bromodomain inhibition through optimization of a fragment-derived hit</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9831</span>– <span class="NLM_lpage">9837</span>, <span class="refDoi"> DOI: 10.1021/jm3010515</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+C-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit&doi=10.1021%2Fjm3010515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0lj9U6r-y282Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520C-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837%26doi%3D10.1021%2Fjm3010515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of emerging epigenetic targets for cancer therapy: A patent review (2010–2014)</span>. <i>Pharm. Pat. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.4155/ppa.15.16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.4155%2Fppa.15.16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=26174566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKhtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=261-284&author=M.+Tanakaauthor=J.+M.+Robertsauthor=J.+Qiauthor=J.+E.+Bradner&title=Inhibitors+of+emerging+epigenetic+targets+for+cancer+therapy%3A+A+patent+review+%282010%E2%80%932014%29&doi=10.4155%2Fppa.15.16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of emerging epigenetic targets for cancer therapy: a patent review (2010-2014)</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Qi, Jun; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Patent Analyst</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-284</span>CODEN:
                <span class="NLM_cas:coden">PPAHCN</span>;
        ISSN:<span class="NLM_cas:issn">2046-8954</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Gene regulatory pathways comprise an emerging and active area of chem. probe discovery and investigational drug development.  Emerging insights from cancer genome sequencing and chromatin biol. have identified leveraged opportunities for development of chromatin-directed small mols. as cancer therapies.  At present, only six agents in two epigenetic target classes have been approved by the US FDA, limited to treatment of hematol. malignancies.  Recently, new classes of epigenetic inhibitors have appeared in literatures.  First-in-class compds. have successfully transitioned to clin. investigation, importantly also in solid tumors and pediatric malignancies.  This review considers patent applications for small-mol. inhibitors of selected epigenetic targets from 2010 to 2014.  Included are exemplary classes of chromatin-assocd. epigenomic writers (DOT1L and EZH2), erasers (LSD1) and readers (BRD4).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk2bznZTVgDLVg90H21EOLACvtfcHk0lgGyvSCaLagWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKhtr7I&md5=e72c0204edadeac9382a26256bbbbc6c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4155%2Fppa.15.16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Fppa.15.16%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DInhibitors%2520of%2520emerging%2520epigenetic%2520targets%2520for%2520cancer%2520therapy%253A%2520A%2520patent%2520review%2520%25282010%25E2%2580%25932014%2529%26jtitle%3DPharm.%2520Pat.%2520Anal.%26date%3D2015%26volume%3D4%26spage%3D261%26epage%3D284%26doi%3D10.4155%2Fppa.15.16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garnier, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibitors: a patent review</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1517/13543776.2014.859244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1517%2F13543776.2014.859244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24261714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptlWgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=185-199&author=J.+M.+Garnierauthor=P.+P.+Sharpauthor=C.+J.+Burns&title=BET+bromodomain+inhibitors%3A+a+patent+review&doi=10.1517%2F13543776.2014.859244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibitors: a patent review</span></div><div class="casAuthors">Garnier, Jean-Marc; Sharp, Phillip P.; Burns, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">185-199</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The bromodomain (BRD) and extra-C terminal domain (BET) protein family consists of four members (BRD2, BRD3, BRD4 and BRDT).  These "epigenetic readers" bind to acetyllysine (KAc) residues on the tails of histones H3 and H4, and regulate chromatin structure and gene expression.  There is increasing evidence of their role in human disease, and recently a no. of small-mol. inhibitors have been reported.  There is increasing interest in the inhibition of BET proteins for a variety of therapeutic applications that have resulted in considerable patent activity from academia and biotechnol. and pharmaceutical companies.  Areas covered: Data supporting the use of BET inhibitors in treating disease are outlined, and the current patent literature is discussed.  The survey is focused on patents claiming compds. as BET inhibitors and addnl. patents covering compds. now reported as BET inhibitors have been included.  Expert opinion: There is now compelling preclin. data demonstrating BET inhibition as a strategy to target processes known to be involved in disease development and progression with clin. trials of two bona fide BET inhibitors now underway.  Patent activity in this area is increasing with initial activity focused on variations to reported BET inhibitors and more recent patents disclosing novel chemotypes as BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6PVpCrReabVg90H21EOLACvtfcHk0lgGyvSCaLagWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptlWgtg%253D%253D&md5=bb33008eefa34f3689e0ee4a49a2539e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.859244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.859244%26sid%3Dliteratum%253Aachs%26aulast%3DGarnier%26aufirst%3DJ.%2BM.%26aulast%3DSharp%26aufirst%3DP.%2BP.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibitors%253A%2520a%2520patent%2520review%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2014%26volume%3D24%26spage%3D185%26epage%3D199%26doi%3D10.1517%2F13543776.2014.859244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meslamani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain: from epigenome reader to druggable target</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1839</i></span>,  <span class="NLM_fpage">676</span>– <span class="NLM_lpage">685</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.bbagrm.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24686119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=676-685&author=R.+Sanchezauthor=J.+Meslamaniauthor=M.+M.+Zhou&title=The+bromodomain%3A+from+epigenome+reader+to+druggable+target&doi=10.1016%2Fj.bbagrm.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain: From epigenome reader to druggable target</span></div><div class="casAuthors">Sanchez, Roberto; Meslamani, Jamel; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">676-685</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a fundamental post-translational modification that plays an important role in the control of gene transcription in chromatin in an ordered fashion.  The bromodomain, the conserved structural module present in transcription-assocd. proteins, functions exclusively to recognize acetyl-lysine on histones and non-histone proteins.  The structural analyses of bromodomains' recognition of lysine-acetylated peptides derived from histones and cellular proteins provide detailed insights into the differences and unifying features of biol. ligand binding selectivity by the bromodomains.  Newly developed small-mol. inhibitors targeting bromodomain proteins further highlight the functional importance of bromodomain/acetyl-lysine binding as a key mechanism in orchestrating mol. interactions and regulation in chromatin biol. and gene transcription.  These new studies argue that modulating bromodomain/acetyl-lysine interactions with small-mol. chems. offer new opportunities to control gene expression in a wide array of human diseases including cancer and inflammation.  This article is part of a Special Issue entitled: Mol. mechanisms of histone modification function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO3Au7jfK0ibVg90H21EOLACvtfcHk0lgGyvSCaLagWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D&md5=95ec21cf13231d9e63bce9b3ccdd6d6f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMeslamani%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DThe%2520bromodomain%253A%2520from%2520epigenome%2520reader%2520to%2520druggable%2520target%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D676%26epage%3D685%26doi%3D10.1016%2Fj.bbagrm.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shortt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">A chemical probe toolbox for dissecting the cancer epigenome</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1038/nrc.2016.148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnrc.2016.148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=28228643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjt1ags70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=160-183&author=J.+Shorttauthor=C.+J.+Ottauthor=R.+W.+Johnstoneauthor=J.+E.+Bradner&title=A+chemical+probe+toolbox+for+dissecting+the+cancer+epigenome&doi=10.1038%2Fnrc.2016.148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe toolbox for dissecting the cancer epigenome</span></div><div class="casAuthors">Shortt, Jake; Ott, Christopher J.; Johnstone, Ricky W.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">160-183</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer cell hallmarks are underpinned by transcriptional programs operating in the context of a dynamic and complicit epigenomic environment.  Somatic alterations of chromatin modifiers are among the most prevalent cancer perturbations.  There is a pressing need for targeted chem. probes to dissect these complex, interconnected gene regulatory circuits.  Validated chem. probes empower mechanistic research while providing the pharmacol. proof of concept that is required to translate drug-like derivs. into therapy for cancer patients.  In this review, we describe chem. probe development for epigenomic effector proteins that are linked to cancer pathogenesis.  By annotating these reagents, we aim to share our perspectives on an informative 'epigenomic toolbox' of broad utility to the research community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRUui_EZyjwrVg90H21EOLACvtfcHk0lgGyvSCaLagWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjt1ags70%253D&md5=8f5f05608a3a035deb494d08445b249f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2016.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2016.148%26sid%3Dliteratum%253Aachs%26aulast%3DShortt%26aufirst%3DJ.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DA%2520chemical%2520probe%2520toolbox%2520for%2520dissecting%2520the%2520cancer%2520epigenome%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D160%26epage%3D183%26doi%3D10.1038%2Fnrc.2016.148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witwicki, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huh, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekram, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1038/nature16508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnature16508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=26735014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=413-417&author=S.+Shuauthor=C.+Y.+Linauthor=H.+H.+Heauthor=R.+M.+Witwickiauthor=D.+P.+Tabassumauthor=J.+M.+Robertsauthor=M.+Janiszewskaauthor=S.+J.+Huhauthor=Y.+Liangauthor=J.+Ryanauthor=E.+Dohertyauthor=H.+Mohammedauthor=H.+Guoauthor=D.+G.+Stoverauthor=M.+B.+Ekramauthor=J.+Brownauthor=C.+D%E2%80%99Santosauthor=I.+E.+Kropauthor=D.+Dillonauthor=M.+McKeownauthor=C.+Ottauthor=J.+Qiauthor=M.+Niauthor=P.+K.+Raoauthor=M.+Duarteauthor=S.+Y.+Wuauthor=C.+M.+Chiangauthor=L.+Andersauthor=R.+A.+Youngauthor=E.+Winerauthor=A.+Letaiauthor=W.+T.+Barryauthor=J.+S.+Carrollauthor=H.+Longauthor=M.+Brownauthor=X.+S.+Liuauthor=C.+A.+Meyerauthor=J.+E.+Bradnerauthor=K.+Polyak&title=Response+and+resistance+to+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1038%2Fnature16508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span></div><div class="casAuthors">Shu, Shaokun; Lin, Charles Y.; He, Housheng Hansen; Witwicki, Robert M.; Tabassum, Doris P.; Roberts, Justin M.; Janiszewska, Michalina; Huh, Sung Jin; Liang, Yi; Ryan, Jeremy; Doherty, Ernest; Mohammed, Hisham; Guo, Hao; Stover, Daniel G.; Ekram, Muhammad B.; Peluffo, Guillermo; Brown, Jonathan; D'Santos, Clive; Krop, Ian E.; Dillon, Deborah; McKeown, Michael; Ott, Christopher; Qi, Jun; Ni, Min; Rao, Prakash K.; Duarte, Melissa; Wu, Shwu-Yuan; Chiang, Cheng-Ming; Anders, Lars; Young, Richard A.; Winer, Eric P.; Letai, Antony; Barry, William T.; Carroll, Jason S.; Long, Henry W.; Brown, Myles; Shirley Liu, X.; Meyer, Clifford A.; Bradner, James E.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">413-417</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous and clin. aggressive disease for which there is no targeted therapy.  BET bromodomain inhibitors, which have shown efficacy in several models of cancer, have not been evaluated in TNBC.  These inhibitors displace BET bromodomain proteins such as BRD4 from chromatin by competing with their acetyl-lysine recognition modules, leading to inhibition of oncogenic transcriptional programs.  Here we report the preferential sensitivity of TNBCs to BET bromodomain inhibition in vitro and in vivo, establishing a rationale for clin. investigation and further motivation to understand mechanisms of resistance.  In paired cell lines selected for acquired resistance to BET inhibition from previously sensitive TNBCs, we failed to identify gatekeeper mutations, new driver events or drug pump activation.  BET-resistant TNBC cells remain dependent on wild-type BRD4, which supports transcription and cell proliferation in a bromodomain-independent manner.  Proteomic studies of resistant TNBC identify strong assocn. with MED1 and hyper-phosphorylation of BRD4 attributable to decreased activity of PP2A, identified here as a principal BRD4 serine phosphatase.  Together, these studies provide a rationale for BET inhibition in TNBC and present mechanism-based combination strategies to anticipate clin. drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZu5A85pdQDbVg90H21EOLACvtfcHk0ljevoXgmaQL3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D&md5=50a03861d9156ee6e344e95c5f5361b9</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature16508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16508%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DWitwicki%26aufirst%3DR.%2BM.%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DJaniszewska%26aufirst%3DM.%26aulast%3DHuh%26aufirst%3DS.%2BJ.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DJ.%26aulast%3DDoherty%26aufirst%3DE.%26aulast%3DMohammed%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DEkram%26aufirst%3DM.%2BB.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DDillon%26aufirst%3DD.%26aulast%3DMcKeown%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DP.%2BK.%26aulast%3DDuarte%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DAnders%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DWiner%26aufirst%3DE.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DBarry%26aufirst%3DW.%2BT.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DResponse%2520and%2520resistance%2520to%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D413%26epage%3D417%26doi%3D10.1038%2Fnature16508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gutteridge, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndiaye, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, N.</span></span> <span> </span><span class="NLM_article-title">PLK1 inhibitors in cancer therapy: From laboratory to clinics</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1427</span>– <span class="NLM_lpage">1435</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-15-0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1158%2F1535-7163.MCT-15-0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=27330107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSrtbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=1427-1435&author=R.+E.+Gutteridgeauthor=M.+A.+Ndiayeauthor=X.+Liuauthor=N.+Ahmad&title=PLK1+inhibitors+in+cancer+therapy%3A+From+laboratory+to+clinics&doi=10.1158%2F1535-7163.MCT-15-0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Plk1 Inhibitors in Cancer Therapy: From Laboratory to Clinics</span></div><div class="casAuthors">Gutteridge, Rosie Elizabeth Ann; Ndiaye, Mary Ann; Liu, Xiaoqi; Ahmad, Nihal</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1427-1435</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Polo-like kinase 1 (Plk1) overexpression has been shown to occur in a wide range of tumors, prompting research and development of Plk1 inhibitors as a means of cancer treatment.  This review discusses recent advances in the development of Plk1 inhibitors for cancer management.  Plk1 inhibition has been shown to cause mitotic block and apoptosis of cells with higher mitotic index and therefore higher Plk1 expression.  The potential of Plk1 inhibitors as cancer therapeutics has been widely investigated.  However, a complete understanding of Plk1 biol./mechanism is yet to be fully achieved.  Resistance to certain chemotherapeutic drugs has been linked to Plk1 overexpression, and Plk1-mediated mitotic events such as microtubule rearrangement have been found to reduce the efficacy of chemotherapeutic agents.  The Plk1 inhibitor volasertib has shown considerable promise in clin. studies, having reached phase III trials.  However, preclin. success with Plk1 inhibitors has not translated well into clin. success.  In our view, combined therapies targeting other relevant pathways together with Plk1 may be vital to combat issues obsd. with monotherapy, esp. resistance.  In addn., research should also be directed toward understanding the mechanisms of Plk1 and designing addnl. next generations of specific, potent Plk1 inhibitors to target cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0TZb-4-s_ULVg90H21EOLACvtfcHk0ljevoXgmaQL3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSrtbbM&md5=dacdc56c51e8b16c484ac44cdb61b9fb</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-15-0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-15-0897%26sid%3Dliteratum%253Aachs%26aulast%3DGutteridge%26aufirst%3DR.%2BE.%26aulast%3DNdiaye%26aufirst%3DM.%2BA.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DAhmad%26aufirst%3DN.%26atitle%3DPLK1%2520inhibitors%2520in%2520cancer%2520therapy%253A%2520From%2520laboratory%2520to%2520clinics%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D1427%26epage%3D1435%26doi%3D10.1158%2F1535-7163.MCT-15-0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seton-Rogers, S.</span></span> <span> </span><span class="NLM_article-title">Epigenetics: Place your BETs</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">638</span>, <span class="refDoi"> DOI: 10.1038/nrc4035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnrc4035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=26469143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Kqt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=638&author=S.+Seton-Rogers&title=Epigenetics%3A+Place+your+BETs&doi=10.1038%2Fnrc4035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics Place your BETs</span></div><div class="casAuthors">Seton-Rogers, Sarah</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">638</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Three papers have reported new data on resistance to bromodomain and extraterminal (BET) inhibitors and how best to use these inhibitors in combination therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRNwJQ_NIQ1rVg90H21EOLACvtfcHk0ljevoXgmaQL3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Kqt7vK&md5=f7b4ad27f8e7285cd914aada2e240620</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnrc4035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc4035%26sid%3Dliteratum%253Aachs%26aulast%3DSeton-Rogers%26aufirst%3DS.%26atitle%3DEpigenetics%253A%2520Place%2520your%2520BETs%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D638%26doi%3D10.1038%2Fnrc4035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elzey, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">PLK1 inhibition enhances the efficacy of bet epigenetic reader blockade in castration-resistant prostate cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1554</span>– <span class="NLM_lpage">1565</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-17-0945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1158%2F1535-7163.MCT-17-0945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=29716963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1alur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1554-1565&author=F.+Maoauthor=J.+Liauthor=Q.+Luoauthor=R.+Wangauthor=Y.+Kongauthor=C.+Carlockauthor=Z.+Liuauthor=B.+D.+Elzeyauthor=X.+Liu&title=PLK1+inhibition+enhances+the+efficacy+of+bet+epigenetic+reader+blockade+in+castration-resistant+prostate+cancer&doi=10.1158%2F1535-7163.MCT-17-0945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Plk1 Inhibition Enhances the Efficacy of BET Epigenetic Reader Blockade in Castration-Resistant Prostate Cancer</span></div><div class="casAuthors">Mao, Fengyi; Li, Jie; Luo, Qian; Wang, Ruixin; Kong, Yifan; Carlock, Colin; Liu, Zian; Elzey, Bennet D.; Liu, Xiaoqi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1554-1565</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Polo-like kinase 1 (Plk1), a crucial regulator of cell-cycle progression, is overexpressed in multiple types of cancers and has been proven to be a potent and promising target for cancer treatment.  In case of prostate cancer, we once showed that antineoplastic activity of Plk1 inhibitor is largely due to inhibition of androgen receptor (AR) signaling.  However, we also discovered that Plk1 inhibition causes activation of the β-catenin pathway and increased expression of c-MYC, eventually resulting in resistance to Plk1 inhibition.  JQ1, a selective small-mol. inhibitor targeting the amino-terminal bromodomains of BRD4, has been shown to dramatically inhibit c-MYC expression and AR signaling, exhibiting antiproliferative effects in a range of cancers.  Because c-MYC and AR signaling are essential for prostate cancer initiation and progression, we aim to test whether targeting Plk1 and BRD4 at the same time is an effective approach to treat prostate cancer.  Herein, we show that a combination of Plk1 inhibitor GSK461364A and BRD4 inhibitor JQ1 had a strong synergistic effect on castration-resistant prostate cancer (CRPC) cell lines, as well as in CRPC xenograft tumors.  Mechanistically, the synergistic effect is likely due to two reasons: (i) Plk1 inhibition results in the accumulation of β-catenin in the nucleus, thus elevation of c-MYC expression, whereas JQ1 treatment directly suppresses c-MYC transcription; (ii) Plk1 and BRD4 dual inhibition acts synergistically in inhibition of AR signaling.  Mol Cancer Ther; 17(7); 1554-65. ©2018 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfot-k6rAvLVg90H21EOLACvtfcHk0liYQneLtvZOeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1alur7F&md5=9e5aec6def8df63a1c2e6fadabef26ad</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-17-0945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-17-0945%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DCarlock%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DElzey%26aufirst%3DB.%2BD.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DPLK1%2520inhibition%2520enhances%2520the%2520efficacy%2520of%2520bet%2520epigenetic%2520reader%2520blockade%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2018%26volume%3D17%26spage%3D1554%26epage%3D1565%26doi%3D10.1158%2F1535-7163.MCT-17-0945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tontsch-Grunt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waizenegger, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savarese, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span> <span> </span><span class="NLM_article-title">Synergistic activity of BET inhibitor BI 894999 with PLK inhibitor volasertib in AML in vitro and in vivo</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2018.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.canlet.2018.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=29454094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsVGisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2018&pages=112-120&author=U.+Tontsch-Gruntauthor=D.+Rudolphauthor=I.+Waizeneggerauthor=A.+Baumauthor=D.+Gerlachauthor=H.+Engelhardtauthor=M.+Wurmauthor=F.+Savareseauthor=N.+Schweiferauthor=N.+Kraut&title=Synergistic+activity+of+BET+inhibitor+BI+894999+with+PLK+inhibitor+volasertib+in+AML+in+vitro+and+in+vivo&doi=10.1016%2Fj.canlet.2018.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic activity of BET inhibitor BI 894999 with PLK inhibitor volasertib in AML in vitro and in vivo</span></div><div class="casAuthors">Tontsch-Grunt, Ulrike; Rudolph, Dorothea; Waizenegger, Irene; Baum, Anke; Gerlach, Daniel; Engelhardt, Harald; Wurm, Melanie; Savarese, Fabio; Schweifer, Norbert; Kraut, Norbert</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112-120</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Interactions between a new potent Bromodomain and extraterminal domain (BET) inhibitor BI 894999 and the polo-like kinase (PLK) inhibitor volasertib were studied in acute myeloid leukemia cell lines in vitro and in vivo.  We provide data for the distinct mechanisms of action of these two compds. with a potential utility in AML based on gene expression, cell cycle profile and modulation of PD biomarkers such as MYC and HEXIM1.  In contrast to BI 894999, volasertib treatment neither affects MYC nor HEXIM1 expression, but augments and prolongs the decrease of MYC expression caused by BI 894999 treatment.  In vitro combination of both compds. leads to a decrease in S-Phase and to increased apoptosis.  In vitro scheduling expts. guided in vivo expts. in disseminated AML mouse models.  Co-administration of BI 894999 and volasertib dramatically reduces tumor burden accompanied by long-term survival of tumor-bearing mice and eradication of AML cells in mouse bone marrow.  Together, these preclin. findings provide evidence for the strong synergistic effect of BI 894999 and volasertib, warranting future clin. studies in patients with AML to investigate this paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyqA7Dr13AlrVg90H21EOLACvtfcHk0liYQneLtvZOeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsVGisb8%253D&md5=7ed409793589b9ca3e8b1053614881ea</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2018.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2018.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DTontsch-Grunt%26aufirst%3DU.%26aulast%3DRudolph%26aufirst%3DD.%26aulast%3DWaizenegger%26aufirst%3DI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DGerlach%26aufirst%3DD.%26aulast%3DEngelhardt%26aufirst%3DH.%26aulast%3DWurm%26aufirst%3DM.%26aulast%3DSavarese%26aufirst%3DF.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DKraut%26aufirst%3DN.%26atitle%3DSynergistic%2520activity%2520of%2520BET%2520inhibitor%2520BI%2520894999%2520with%2520PLK%2520inhibitor%2520volasertib%2520in%2520AML%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Lett.%26date%3D2018%26volume%3D421%26spage%3D112%26epage%3D120%26doi%3D10.1016%2Fj.canlet.2018.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlich, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durden, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Dual-activity PI3K–BRD4 inhibitor for the orthogonal inhibition of MYC to block tumor growth and metastasis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E1072</span>– <span class="NLM_lpage">E1080</span>, <span class="refDoi"> DOI: 10.1073/pnas.1613091114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1073%2Fpnas.1613091114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E1072-E1080&author=F.+H.+Andrewsauthor=A.+R.+Singhauthor=S.+Joshiauthor=C.+A.+Smithauthor=G.+A.+Moralesauthor=J.+R.+Garlichauthor=D.+L.+Durdenauthor=T.+G.+Kutateladze&title=Dual-activity+PI3K%E2%80%93BRD4+inhibitor+for+the+orthogonal+inhibition+of+MYC+to+block+tumor+growth+and+metastasis&doi=10.1073%2Fpnas.1613091114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1613091114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1613091114%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DF.%2BH.%26aulast%3DSingh%26aufirst%3DA.%2BR.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DC.%2BA.%26aulast%3DMorales%26aufirst%3DG.%2BA.%26aulast%3DGarlich%26aufirst%3DJ.%2BR.%26aulast%3DDurden%26aufirst%3DD.%2BL.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DDual-activity%2520PI3K%25E2%2580%2593BRD4%2520inhibitor%2520for%2520the%2520orthogonal%2520inhibition%2520of%2520MYC%2520to%2520block%2520tumor%2520growth%2520and%2520metastasis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE1072%26epage%3DE1080%26doi%3D10.1073%2Fpnas.1613091114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgoyne, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicklick, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kono, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlich, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durden, D. L.</span></span> <span> </span><span class="NLM_article-title">Single agent and synergistic activity of the “first-in-class” Dual PI3K/BRD4 inhibitor SF1126 with sorafenib in hepatocellular carcinoma</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2553</span>– <span class="NLM_lpage">2562</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-15-0976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1158%2F1535-7163.MCT-15-0976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=27496136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2553-2562&author=A.+R.+Singhauthor=S.+Joshiauthor=A.+M.+Burgoyneauthor=J.+K.+Sicklickauthor=S.+Ikedaauthor=Y.+Konoauthor=J.+R.+Garlichauthor=G.+A.+Moralesauthor=D.+L.+Durden&title=Single+agent+and+synergistic+activity+of+the+%E2%80%9Cfirst-in-class%E2%80%9D+Dual+PI3K%2FBRD4+inhibitor+SF1126+with+sorafenib+in+hepatocellular+carcinoma&doi=10.1158%2F1535-7163.MCT-15-0976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Single agent and synergistic activity of the "first-in-class" dual PI3K/BRD4 inhibitor SF1126 with sorafenib in hepatocellular carcinoma</span></div><div class="casAuthors">Singh, Alok R.; Joshi, Shweta; Burgoyne, Adam M.; Sicklick, Jason K.; Ikeda, Sadakatsu; Kono, Yuko; Garlich, Joseph R.; Morales, Guillermo A.; Durden, Donald L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2553-2562</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Deregulated PI3K/AKT/mTOR, Ras/Raf/MAPK, and c-Myc signaling pathways are of prognostic significance in hepatocellular carcinoma (HCC).  Sorafenib, the only drug clin. approved for patients with advanced HCC, blocks the Ras/Raf/MAPK pathway but it does not inhibit the PI3K/AKT/mTOR pathway or c-Myc activation.  Hence, there is an unmet medical need to identify potent PI3K/BRD4 inhibitors, which can be used either alone or in combination with sorafenib to treat patients with advanced HCC.  Herein, we show that SF1126 (pan PI3K/BRD4 inhibitor) as single agent or in combination with sorafenib inhibited proliferation, cell cycle, apoptosis, and multiple key enzymes in PI3K/AKT/mTOR and Ras/Raf/MAPK pathway in Hep3B, HepG2, SK-Hep1, and Huh7 HCC cell lines.  We demonstrate that the active moiety of the SF1126 prodrug LY294002 binds to and blocks BRD4 interaction with the acetylated histone-H4 chromatin mark protein and displaced BRD4 coactivator protein from the transcriptional start site of MYC in Huh7 and SK-Hep1 HCC cell lines.  Moreover, SF1126 blocked expression levels of c-Myc in HCC cells.  Treatment of SF1126 either alone or in combination with sorafenib showed significant antitumor activity in vivo.  Our results establish that SF1126 is a dual PI3K/BRD4 inhibitor.  This agent has completed a phase I clin. trial in humans with good safety profile.  Our data support the potential future consideration of a phase II clin. trial of SF1126, a clin. relevant dual "first-in-class" PI3K/BRD4 inhibitor in advanced HCC, and a potential combination with sorafenib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1PYR8DNGoU7Vg90H21EOLACvtfcHk0lhdrjH5AhfX8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqs7Y%253D&md5=e0133b38f398c81afca96d078f54db86</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-15-0976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-15-0976%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DA.%2BR.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DBurgoyne%26aufirst%3DA.%2BM.%26aulast%3DSicklick%26aufirst%3DJ.%2BK.%26aulast%3DIkeda%26aufirst%3DS.%26aulast%3DKono%26aufirst%3DY.%26aulast%3DGarlich%26aufirst%3DJ.%2BR.%26aulast%3DMorales%26aufirst%3DG.%2BA.%26aulast%3DDurden%26aufirst%3DD.%2BL.%26atitle%3DSingle%2520agent%2520and%2520synergistic%2520activity%2520of%2520the%2520%25E2%2580%259Cfirst-in-class%25E2%2580%259D%2520Dual%2520PI3K%252FBRD4%2520inhibitor%2520SF1126%2520with%2520sorafenib%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2553%26epage%3D2562%26doi%3D10.1158%2F1535-7163.MCT-15-0976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-lysine binding site of bromodomain-containing protein 4(BRD4) interacts with diverse kinase inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+Emberauthor=J.+Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Acetyl-lysine+binding+site+of+bromodomain-containing+protein+4%28BRD4%29+interacts+with+diverse+kinase+inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DAcetyl-lysine%2520binding%2520site%2520of%2520bromodomain-containing%2520protein%25204%2528BRD4%2529%2520interacts%2520with%2520diverse%2520kinase%2520inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahony&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lgsOib2iUYESw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koblan, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Assessment of bromodomain target engagement by a series of BI2536 analogues with miniaturized BET-BRET</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2581</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fcmdc.201600502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=27862999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVChsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2575-2581&author=L.+W.+Koblanauthor=D.+L.+Buckleyauthor=C.+J.+Ottauthor=M.+E.+Fitzgeraldauthor=S.+W.+Emberauthor=J.+Y.+Zhuauthor=S.+Liuauthor=J.+M.+Robertsauthor=D.+Remillardauthor=S.+Vittoriauthor=W.+Zhangauthor=E.+Schonbrunnauthor=J.+E.+Bradner&title=Assessment+of+bromodomain+target+engagement+by+a+series+of+BI2536+analogues+with+miniaturized+BET-BRET&doi=10.1002%2Fcmdc.201600502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Bromodomain Target Engagement by a Series of BI2536 Analogues with Miniaturized BET-BRET</span></div><div class="casAuthors">Koblan, Luke W.; Buckley, Dennis L.; Ott, Christopher J.; Fitzgerald, Mark E.; Ember, Stuart W. J.; Zhu, Jin-Yi; Liu, Shuai; Roberts, Justin M.; Remillard, David; Vittori, Sarah; Zhang, Wei; Schonbrunn, Ernst; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2575-2581</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Evaluating the engagement of a small mol. ligand with a protein target in cells provides useful information for chem. probe optimization and pharmaceutical development.  While several techniques exist that can be performed in a low-throughput manner, systematic evaluation of large compd. libraries remains a challenge.  In-cell engagement measurements are esp. useful when evaluating compd. classes suspected to target multiple cellular factors.  In this study we used a bioluminescent resonant energy transfer assay to assess bromodomain engagement by a compd. series contg. bromodomain- and kinase-biasing polypharmacophores based on the known dual BRD4 bromodomain/PLK1 kinase inhibitor BI2536.  With this assay, we discovered several novel agents with bromodomain-selective specificity profiles and cellular activity.  Thus, this platform aids in distinguishing mols. whose cellular activity is difficult to assess due to polypharmacol. effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiVVtnPQvEQrVg90H21EOLACvtfcHk0lgsOib2iUYESw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVChsbfK&md5=a6d7c6a58870eb6208e18eb32068d821</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600502%26sid%3Dliteratum%253Aachs%26aulast%3DKoblan%26aufirst%3DL.%2BW.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DEmber%26aufirst%3DS.%2BW.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DAssessment%2520of%2520bromodomain%2520target%2520engagement%2520by%2520a%2520series%2520of%2520BI2536%2520analogues%2520with%2520miniaturized%2520BET-BRET%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2575%26epage%3D2581%26doi%3D10.1002%2Fcmdc.201600502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fountain, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheenstra, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strovel, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span> <span> </span><span class="NLM_article-title">BRD4 structure–activity relationships of dual PLK1 Kinase/BRD4 bromodomain inhibitor BI-2536</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">769</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00084</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00084" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=764-769&author=L.+Chenauthor=J.+L.+Yapauthor=M.+Yoshiokaauthor=M.+E.+Lanningauthor=R.+N.+Fountainauthor=M.+Rajeauthor=J.+A.+Scheenstraauthor=J.+W.+Strovelauthor=S.+Fletcher&title=BRD4+structure%E2%80%93activity+relationships+of+dual+PLK1+Kinase%2FBRD4+bromodomain+inhibitor+BI-2536&doi=10.1021%2Facsmedchemlett.5b00084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Structure-Activity Relationships of Dual PLK1 Kinase/BRD4 Bromodomain Inhibitor BI-2536</span></div><div class="casAuthors">Chen, Lijia; Yap, Jeremy L.; Yoshioka, Makoto; Lanning, Maryanna E.; Fountain, Rachel N.; Raje, Mithun; Scheenstra, Jacob A.; Strovel, Jeffrey W.; Fletcher, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">764-769</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused library of analogs of the dual PLK1 kinase/BRD4 bromodomain inhibitor BI-2536 was prepd. and then analyzed for BRD4 and PLK1 inhibitory activities.  Particularly, replacement of the cyclopentyl group with a 3-bromobenzyl moiety afforded the most potent BRD4 inhibitor of the series (39j) with a Ki = 8.7 nM, which was equipotent against PLK1.  The superior affinity of 39j over the parental compd. to BRD4 possibly derives from improved interactions with the WPF shelf.  Meanwhile, substitution of the pyrimidine NH with an oxygen atom reversed the PLK1/BRD4 selectivity to convert BI-2536 into a BRD4-selective inhibitor, likely owing to the loss of a crit. hydrogen bond in PLK1.  We believe further fine-tuning will furnish a BRD4 "magic bullet" or an even more potent PLK1/BRD4 dual inhibitor toward the expansion and improved efficacy of the chemotherapy arsenal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5UnNSmf1L17Vg90H21EOLACvtfcHk0li2KWhxscJROA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D&md5=f39070e652bca20c03a3bc5fe070a7fc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00084%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DYoshioka%26aufirst%3DM.%26aulast%3DLanning%26aufirst%3DM.%2BE.%26aulast%3DFountain%26aufirst%3DR.%2BN.%26aulast%3DRaje%26aufirst%3DM.%26aulast%3DScheenstra%26aufirst%3DJ.%2BA.%26aulast%3DStrovel%26aufirst%3DJ.%2BW.%26aulast%3DFletcher%26aufirst%3DS.%26atitle%3DBRD4%2520structure%25E2%2580%2593activity%2520relationships%2520of%2520dual%2520PLK1%2520Kinase%252FBRD4%2520bromodomain%2520inhibitor%2520BI-2536%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D764%26epage%3D769%26doi%3D10.1021%2Facsmedchemlett.5b00084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelias, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0li2KWhxscJROA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span> <span> </span><span class="NLM_article-title">Leveraging kinase inhibitors to develop small molecule tools for imaging kinases by fluorescence microscopy</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2523</span>– <span class="NLM_lpage">2526</span>, <span class="refDoi"> DOI: 10.1039/c2mb25099c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1039%2Fc2mb25099c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22673640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2523-2526&author=Z.+Zhangauthor=N.+Kwiatkowskiauthor=H.+Zengauthor=S.+M.+Limauthor=N.+S.+Grayauthor=W.+Zhangauthor=P.+L.+Yang&title=Leveraging+kinase+inhibitors+to+develop+small+molecule+tools+for+imaging+kinases+by+fluorescence+microscopy&doi=10.1039%2Fc2mb25099c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging kinase inhibitors to develop small molecule tools for imaging kinases by fluorescence microscopy</span></div><div class="casAuthors">Zhang, Zijuan; Kwiatkowski, Nicholas; Zeng, Hong; Lim, Sang Min; Gray, Nathanael S.; Zhang, Wei; Yang, Priscilla L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2523-2526</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As the usage of fluorescence microscopy as a tool to study biol. systems continues to grow, so does the need for addnl. tools that permit the selective detection of proteins of interest.  Existing selective and well-characterized kinase inhibitors may be exploited to develop novel small mol. probes useful in imaging kinases by fluorescence microscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkTi2Qtkh07Vg90H21EOLACvtfcHk0lgx8yyPCJtrJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbnF&md5=e59fd8a1bbd080bcfa2653fd7250328e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2Fc2mb25099c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2mb25099c%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DP.%2BL.%26atitle%3DLeveraging%2520kinase%2520inhibitors%2520to%2520develop%2520small%2520molecule%2520tools%2520for%2520imaging%2520kinases%2520by%2520fluorescence%2520microscopy%26jtitle%3DMol.%2520BioSyst.%26date%3D2012%26volume%3D8%26spage%3D2523%26epage%3D2526%26doi%3D10.1039%2Fc2mb25099c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera - A visualization system for exploratory research and analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera+-+A+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lgx8yyPCJtrJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%2520-%2520A%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brozell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moustakas, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">DOCK 6: Impact of new features and current docking performance</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1156</span>, <span class="refDoi"> DOI: 10.1002/jcc.23905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.23905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=25914306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1132-1156&author=W.+J.+Allenauthor=T.+E.+Baliusauthor=S.+Mukherjeeauthor=S.+R.+Brozellauthor=D.+T.+Moustakasauthor=P.+T.+Langauthor=D.+A.+Caseauthor=I.+D.+Kuntzauthor=R.+C.+Rizzo&title=DOCK+6%3A+Impact+of+new+features+and+current+docking+performance&doi=10.1002%2Fjcc.23905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">DOCK 6: Impact of new features and current docking performance</span></div><div class="casAuthors">Allen, William J.; Balius, Trent E.; Mukherjee, Sudipto; Brozell, Scott R.; Moustakas, Demetri T.; Lang, P. Therese; Case, David A.; Kuntz, Irwin D.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1132-1156</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This manuscript presents the latest algorithmic and methodol. developments to the structure-based design program DOCK 6.7 focused on an updated internal energy function, new anchor selection control, enhanced minimization options, a footprint similarity scoring function, a symmetry-cor. root-mean-square deviation algorithm, a database filter, and docking forensic tools.  An important strategy during development involved use of three orthogonal metrics for assessment and validation: pose reprodn. over a large database of 1043 protein-ligand complexes (SB2012 test set), cross-docking to 24 drug-target protein families, and database enrichment using large active and decoy datasets (Directory of Useful Decoys [DUD]-E test set) for five important proteins including HIV protease and IGF-1R.  Relative to earlier versions, a key outcome of the work is a significant increase in pose reprodn. success in going from DOCK 4.0.2 (51.4%) → 5.4 (65.2%) → 6.7 (73.3%) as a result of significant decreases in failure arising from both sampling 24.1% → 13.6% → 9.1% and scoring 24.4% → 21.1% → 17.5%.  Companion cross-docking and enrichment studies with the new version highlight other strengths and remaining areas for improvement, esp. for systems contg. metal ions.  The source code for DOCK 6.7 is available for download and free for academic users at. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNgMYHDmTu7Vg90H21EOLACvtfcHk0lhu1vczvdMD1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D&md5=8b5fc6cc7533f975e1e740acb3688807</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23905%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDOCK%25206%253A%2520Impact%2520of%2520new%2520features%2520and%2520current%2520docking%2520performance%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D1132%26epage%3D1156%26doi%3D10.1002%2Fjcc.23905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allinger, N. L.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis. 130. MM2. A hydrocarbon force field utilizing V1 and V2 torsional terms</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">8127</span>– <span class="NLM_lpage">8134</span>, <span class="refDoi"> DOI: 10.1021/ja00467a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00467a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADyaE1cXmtVOm" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1977&pages=8127-8134&author=N.+L.+Allinger&title=Conformational+analysis.+130.+MM2.+A+hydrocarbon+force+field+utilizing+V1+and+V2+torsional+terms&doi=10.1021%2Fja00467a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis.  130.  MM2.  A hydrocarbon force field utilizing V1 and V2 torsional terms</span></div><div class="casAuthors">Allinger, Norman L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8127-34</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">An improved force field for mol.-mechanics calcns. of the structures and energies of hydrocarbons is presented.  The problem of simultaneously obtaining a sufficiently large gauche butane interaction energy while keeping the H atoms small enough for good structural predictions was solved with 1- and 2-fold rotational barriers.  For 42 selected diverse types of hydrocarbons, the std. deviation between the calcd. and exptl. heats of formation is 0.42 kcal/mol, compared with an av. reported exptl. error for the same group of compds. of 0.40 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7SvCTV-ylebVg90H21EOLACvtfcHk0lhu1vczvdMD1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXmtVOm&md5=17d96eafa46ab1f9dc6ea672e700aa2d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja00467a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00467a001%26sid%3Dliteratum%253Aachs%26aulast%3DAllinger%26aufirst%3DN.%2BL.%26atitle%3DConformational%2520analysis.%2520130.%2520MM2.%2520A%2520hydrocarbon%2520force%2520field%2520utilizing%2520V1%2520and%2520V2%2520torsional%2520terms%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1977%26volume%3D99%26spage%3D8127%26epage%3D8134%26doi%3D10.1021%2Fja00467a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ponder, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, F. M.</span></span> <span> </span><span class="NLM_article-title">An efficient newton-like method for molecular mechanics energy minimization of large molecules</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1016</span>– <span class="NLM_lpage">1024</span>, <span class="refDoi"> DOI: 10.1002/jcc.540080710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.540080710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADyaL1cXhtFGhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1987&pages=1016-1024&author=J.+W.+Ponderauthor=F.+M.+Richards&title=An+efficient+newton-like+method+for+molecular+mechanics+energy+minimization+of+large+molecules&doi=10.1002%2Fjcc.540080710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient Newton-like method for molecular mechanics energy minimization of large molecules</span></div><div class="casAuthors">Ponder, Jay W.; Richards, Frederic M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1016-24</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">Techniques from numerical anal. and crystallog. refinement are combined to produce a variant of the Truncated Newton nonlinear optimization procedure.  The new algorithm shows particular promise for potential energy minimization of large mol. systems.  Usual implementations of Newton's method require storage space proportional to the no. of atoms squared (i.e., O(N2)) and computer time of O(N3).  Suggested implementation of the Truncated Newton technique requires storage of less than O(N1.5) and CPU time of less than O(N2) for structures contg. several hundred to a few thousand atoms.  The algorithm exhibits quadratic convergence near the min. and is also very tolerant of poor initial structures.  A comparison with existing optimization procedures is detailed for cyclohexane, arachidonic acid, and the small protein crambin.  In particular, a structure for crambin (662 atoms) is refined to an RMS gradient of 3.6 × 10-6 kcal/mol/Å per atom on the MM2 potential energy surface.  Several suggestions are made which may lead to further improvement of the new method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlXX33qM5gJbVg90H21EOLACvtfcHk0ljTWiy6Ux5S-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhtFGhsg%253D%253D&md5=aace9382993efb8da18b17f8869c1871</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540080710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540080710%26sid%3Dliteratum%253Aachs%26aulast%3DPonder%26aufirst%3DJ.%2BW.%26aulast%3DRichards%26aufirst%3DF.%2BM.%26atitle%3DAn%2520efficient%2520newton-like%2520method%2520for%2520molecular%2520mechanics%2520energy%2520minimization%2520of%2520large%2520molecules%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1987%26volume%3D8%26spage%3D1016%26epage%3D1024%26doi%3D10.1002%2Fjcc.540080710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maier, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasavajhala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">ff14SB: Improving the accuracy of protein side chain and backbone parameters from FF99SB</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3696</span>– <span class="NLM_lpage">3713</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+Improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+FF99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0ljTWiy6Ux5S-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520Improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520FF99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0ljKb8bazxWBCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span> <span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0ljKb8bazxWBCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marineau, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Biased multicomponent reactions to develop novel bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9019</span>– <span class="NLM_lpage">9027</span>, <span class="refDoi"> DOI: 10.1021/jm501120z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501120z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9019-9027&author=M.+R.+McKeownauthor=D.+L.+Shawauthor=H.+Fuauthor=S.+Liuauthor=X.+Xuauthor=J.+J.+Marineauauthor=Y.+Huangauthor=X.+Zhangauthor=D.+L.+Buckleyauthor=A.+Kadamauthor=Z.+Zhangauthor=S.+C.+Blacklowauthor=J.+Qiauthor=W.+Zhangauthor=J.+E.+Bradner&title=Biased+multicomponent+reactions+to+develop+novel+bromodomain+inhibitors&doi=10.1021%2Fjm501120z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Multicomponent Reactions to Develop Novel Bromodomain Inhibitors</span></div><div class="casAuthors">McKeown, Michael R.; Shaw, Daniel L.; Fu, Harry; Liu, Shuai; Xu, Xiang; Marineau, Jason J.; Huang, Yibo; Zhang, Xiaofeng; Buckley, Dennis L.; Kadam, Asha; Zhang, Zijuan; Blacklow, Stephen C.; Qi, Jun; Zhang, Wei; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9019-9027</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET bromodomain inhibition has contributed new insights into gene regulation and emerged as a promising therapeutic strategy in cancer.  Structural analogy of early methyl-triazolo BET inhibitors has prompted a need for structurally dissimilar ligands as probes of bromodomain function.  Using fluorous-tagged multicomponent reactions, we developed a focused chem. library of bromodomain inhibitors around a 3,5-dimethylisoxazole biasing element with micromolar biochem. IC50.  Iterative synthesis and biochem. assessment allowed optimization of novel BET bromodomain inhibitors based on an imidazo[1,2-a]pyrazine scaffold.  Lead compd. I (UMB-32) binds BRD4 with a Kd of 550 nM and 724 nM cellular potency in BRD4-dependent lines.  Addnl., compd. I shows potency against TAF1, a bromodomain-contg. transcription factor previously unapproached by discovery chem.  Compd. I was cocrystd. with BRD4, yielding a 1.56 Å resoln. crystal structure.  This research showcases new applications of fluorous and multicomponent chem. synthesis for the development of novel epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocWEf4LoDJTrVg90H21EOLACvtfcHk0ljKb8bazxWBCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN&md5=a4e20101f5a0b547451814af73c0418f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm501120z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501120z%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DShaw%26aufirst%3DD.%2BL.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKadam%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBiased%2520multicomponent%2520reactions%2520to%2520develop%2520novel%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9019%26epage%3D9027%26doi%3D10.1021%2Fjm501120z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 21 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Pan Tang, Jifa Zhang, Jie Liu, Cheng-Ming Chiang, <span class="NLM_string-name hlFld-ContribAuthor">Liang Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2419-2435. <a href="https://doi.org/10.1021/acs.jmedchem.0c01487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01487%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BBromodomain%252Band%252BExtraterminal%252BProteins%252Bfor%252BDrug%252BDiscovery%25253A%252BFrom%252BCurrent%252BProgress%252Bto%252BTechnological%252BDevelopment%26aulast%3DTang%26aufirst%3DPan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2419%26epage%3D2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martin Hügle, Pierre Regenass, Robin Warstat, Mirjam Hau, Karin Schmidtkunz, Xavier Lucas, Daniel Wohlwend, Oliver Einsle, Manfred Jung, Bernhard Breit, <span class="NLM_string-name hlFld-ContribAuthor">Stefan Günther</span>. </span><span class="cited-content_cbyCitation_article-title">4-Acyl Pyrroles as Dual BET-BRD7/9 Bromodomain Inhibitors Address BETi Insensitive Human Cancer Cell Lines. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15603-15620. <a href="https://doi.org/10.1021/acs.jmedchem.0c00478" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00478</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00478%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D4-Acyl%252BPyrroles%252Bas%252BDual%252BBET-BRD7%25252F9%252BBromodomain%252BInhibitors%252BAddress%252BBETi%252BInsensitive%252BHuman%252BCancer%252BCell%252BLines%26aulast%3DH%25C3%25BCgle%26aufirst%3DMartin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D21032020%26date%3D04122020%26volume%3D63%26issue%3D24%26spage%3D15603%26epage%3D15620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kaikai Lv, Weicong Chen, Danqi Chen, Jie Mou, Huijie Zhang, Tiantian Fan, Yanlian Li, Danyan Cao, Xin Wang, Lin Chen, Jingkang Shen, Dongsheng Pei, <span class="NLM_string-name hlFld-ContribAuthor">Bing Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9787-9802. <a href="https://doi.org/10.1021/acs.jmedchem.0c00962" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00962%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRational%252BDesign%252Band%252BEvaluation%252Bof%252B6-%252528Pyrimidin-2-ylamino%252529-3%25252C4-dihydroquinoxalin-2%2525281H%252529-ones%252Bas%252BPolypharmacological%252BInhibitors%252Bof%252BBET%252Band%252BKinases%26aulast%3DLv%26aufirst%3DKaikai%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05062020%26date%3D26082020%26date%3D11082020%26volume%3D63%26issue%3D17%26spage%3D9787%26epage%3D9802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianping Hu, Chang-Qing Tian, Mohammadali Soleimani Damaneh, Yanlian Li, Danyan Cao, Kaikai Lv, Ting Yu, Tao Meng, Danqi Chen, Xin Wang, Lin Chen, Jian Li, Shan-Shan Song, Xia-Juan Huan, Lihuai Qin, Jingkang Shen, Ying-Qing Wang, Ze-Hong Miao, <span class="NLM_string-name hlFld-ContribAuthor">Bing Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Discovery and Development of a Series of Potent and Selective Bromodomain and Extra-Terminal Protein Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (18)
                                     , 8642-8663. <a href="https://doi.org/10.1021/acs.jmedchem.9b01094" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01094%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDiscovery%252Band%252BDevelopment%252Bof%252Ba%252BSeries%252Bof%252BPotent%252Band%252BSelective%252BBromodomain%252Band%252BExtra-Terminal%252BProtein%252BInhibitors%26aulast%3DHu%26aufirst%3DJianping%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07072019%26date%3D16092019%26date%3D06092019%26volume%3D62%26issue%3D18%26spage%3D8642%26epage%3D8663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ellen Watts, David Heidenreich, Elizabeth Tucker, Monika Raab, Klaus Strebhardt, Louis Chesler, Stefan Knapp, Benjamin Bellenie, <span class="NLM_string-name hlFld-ContribAuthor">Swen Hoelder</span>. </span><span class="cited-content_cbyCitation_article-title">Designing Dual Inhibitors of Anaplastic Lymphoma Kinase (ALK) and Bromodomain-4 (BRD4) by Tuning Kinase Selectivity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (5)
                                     , 2618-2637. <a href="https://doi.org/10.1021/acs.jmedchem.8b01947" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01947</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01947%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesigning%252BDual%252BInhibitors%252Bof%252BAnaplastic%252BLymphoma%252BKinase%252B%252528ALK%252529%252Band%252BBromodomain-4%252B%252528BRD4%252529%252Bby%252BTuning%252BKinase%252BSelectivity%26aulast%3DWatts%26aufirst%3DEllen%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D13122018%26date%3D21022019%26volume%3D62%26issue%3D5%26spage%3D2618%26epage%3D2637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian-Bo  Tong</span>, <span class="hlFld-ContribAuthor ">Ding  Luo</span>, <span class="hlFld-ContribAuthor ">Shuai  Bian</span>, <span class="hlFld-ContribAuthor ">Xing  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Structural investigation of tetrahydropteridin analogues as selective PLK1 inhibitors for treating cancer through combined QSAR techniques, molecular docking, and molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Liquids</span><span> <strong>2021,</strong> <em>335 </em>, 116235. <a href="https://doi.org/10.1016/j.molliq.2021.116235" title="DOI URL">https://doi.org/10.1016/j.molliq.2021.116235</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molliq.2021.116235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molliq.2021.116235%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Liquids%26atitle%3DStructural%252Binvestigation%252Bof%252Btetrahydropteridin%252Banalogues%252Bas%252Bselective%252BPLK1%252Binhibitors%252Bfor%252Btreating%252Bcancer%252Bthrough%252Bcombined%252BQSAR%252Btechniques%25252C%252Bmolecular%252Bdocking%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DTong%26aufirst%3DJian-Bo%26date%3D2021%26volume%3D335%26spage%3D116235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jan  Chasák</span>, <span class="hlFld-ContribAuthor ">Lucie  Brulíková</span>. </span><span class="cited-content_cbyCitation_article-title">Polymer-Supported Synthesis of Various Pteridinones and Pyrimidodiazepinones. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (6)
                                     , 1603. <a href="https://doi.org/10.3390/molecules26061603" title="DOI URL">https://doi.org/10.3390/molecules26061603</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26061603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26061603%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPolymer-Supported%252BSynthesis%252Bof%252BVarious%252BPteridinones%252Band%252BPyrimidodiazepinones%26aulast%3DChas%25C3%25A1k%26aufirst%3DJan%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D6%26spage%3D1603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sharon  Rossiter</span>, <span class="hlFld-ContribAuthor ">Mehrnoosh  Ostovar</span>. </span><span class="cited-content_cbyCitation_article-title">Bicyclic 6-6 Systems: Pteridines. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/B978-0-12-818655-8.00040-8" title="DOI URL">https://doi.org/10.1016/B978-0-12-818655-8.00040-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-818655-8.00040-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-818655-8.00040-8%26sid%3Dliteratum%253Aachs%26atitle%3DBicyclic%252B6-6%252BSystems%25253A%252BPteridines%26aulast%3DRossiter%26aufirst%3DSharon%26date%3D2021%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2021%26volume%3Dvol.%252010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaosa  Chang</span>, <span class="hlFld-ContribAuthor ">Dejuan  Sun</span>, <span class="hlFld-ContribAuthor ">Danfeng  Shi</span>, <span class="hlFld-ContribAuthor ">Guan  Wang</span>, <span class="hlFld-ContribAuthor ">Yanmei  Chen</span>, <span class="hlFld-ContribAuthor ">Kai  Zhang</span>, <span class="hlFld-ContribAuthor ">Huidan  Tan</span>, <span class="hlFld-ContribAuthor ">Jie  Liu</span>, <span class="hlFld-ContribAuthor ">Bo  Liu</span>, <span class="hlFld-ContribAuthor ">Liang  Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and biological evaluation of quinazolin-4(3H)-one derivatives co-targeting poly(ADP-ribose) polymerase-1 and bromodomain containing protein 4 for breast cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     , 156-180. <a href="https://doi.org/10.1016/j.apsb.2020.06.003" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.06.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.06.003%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bbiological%252Bevaluation%252Bof%252Bquinazolin-4%2525283H%252529-one%252Bderivatives%252Bco-targeting%252Bpoly%252528ADP-ribose%252529%252Bpolymerase-1%252Band%252Bbromodomain%252Bcontaining%252Bprotein%252B4%252Bfor%252Bbreast%252Bcancer%252Btherapy%26aulast%3DChang%26aufirst%3DXiaosa%26date%3D2021%26volume%3D11%26issue%3D1%26spage%3D156%26epage%3D180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Tim  Zegar</span>, <span class="hlFld-ContribAuthor ">Tim  Weiser</span>, <span class="hlFld-ContribAuthor ">Feda H.  Hamdan</span>, <span class="hlFld-ContribAuthor ">Benedict‐Tilman  Berger</span>, <span class="hlFld-ContribAuthor ">Romain  Lucas</span>, <span class="hlFld-ContribAuthor ">Dimitrios‐IIias  Balourdas</span>, <span class="hlFld-ContribAuthor ">Swetlana  Ladigan</span>, <span class="hlFld-ContribAuthor ">Phyllis F.  Cheung</span>, <span class="hlFld-ContribAuthor ">Sven‐Thorsten  Liffers</span>, <span class="hlFld-ContribAuthor ">Marija  Trajkovic‐Arsic</span>, <span class="hlFld-ContribAuthor ">Bjoern  Scheffler</span>, <span class="hlFld-ContribAuthor ">Andreas C.  Joerger</span>, <span class="hlFld-ContribAuthor ">Stephan A.  Hahn</span>, <span class="hlFld-ContribAuthor ">Steven A.  Johnsen</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Jens T.  Siveke</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of a dual
              BET
              /
              HDAC
              inhibitor for treatment of pancreatic ductal adenocarcinoma. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Cancer</span><span> <strong>2020,</strong> <em>147 </em>
                                    (10)
                                     , 2847-2861. <a href="https://doi.org/10.1002/ijc.33137" title="DOI URL">https://doi.org/10.1002/ijc.33137</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ijc.33137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fijc.33137%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Cancer%26atitle%3DCharacterization%252Bof%252Ba%252Bdual%252BBET%252B%25252F%252BHDAC%252Binhibitor%252Bfor%252Btreatment%252Bof%252Bpancreatic%252Bductal%252Badenocarcinoma%26aulast%3DZhang%26aufirst%3DXin%26date%3D2020%26date%3D2020%26volume%3D147%26issue%3D10%26spage%3D2847%26epage%3D2861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yinliang  Qi</span>, <span class="hlFld-ContribAuthor ">Le  Xu</span>, <span class="hlFld-ContribAuthor ">Zhiwei  Li</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>, <span class="hlFld-ContribAuthor ">Tao  Hu</span>, <span class="hlFld-ContribAuthor ">Bixi  Yin</span>, <span class="hlFld-ContribAuthor ">Mingze  Qin</span>, <span class="hlFld-ContribAuthor ">Yajing  Liu</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zhao</span>, <span class="hlFld-ContribAuthor ">Yunlei  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel pteridinone derivatives as potent dual inhibitors of PLK1 and BRD4. </span><span class="cited-content_cbyCitation_journal-name">New Journal of Chemistry</span><span> <strong>2020,</strong> <em>44 </em>
                                    (38)
                                     , 16477-16490. <a href="https://doi.org/10.1039/D0NJ03477K" title="DOI URL">https://doi.org/10.1039/D0NJ03477K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NJ03477K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NJ03477K%26sid%3Dliteratum%253Aachs%26jtitle%3DNew%2520Journal%2520of%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bpteridinone%252Bderivatives%252Bas%252Bpotent%252Bdual%252Binhibitors%252Bof%252BPLK1%252Band%252BBRD4%26aulast%3DQi%26aufirst%3DYinliang%26date%3D2020%26date%3D2020%26volume%3D44%26issue%3D38%26spage%3D16477%26epage%3D16490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angel R.  de Lera</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Two-hit wonders: The expanding universe of multitargeting epigenetic agents. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 135-154. <a href="https://doi.org/10.1016/j.cbpa.2020.05.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTwo-hit%252Bwonders%25253A%252BThe%252Bexpanding%252Buniverse%252Bof%252Bmultitargeting%252Bepigenetic%252Bagents%26aulast%3Dde%2BLera%26aufirst%3DAngel%2BR.%26date%3D2020%26volume%3D57%26spage%3D135%26epage%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ehab  Ghazy</span>, <span class="hlFld-ContribAuthor ">Patrik  Zeyen</span>, <span class="hlFld-ContribAuthor ">Daniel  Herp</span>, <span class="hlFld-ContribAuthor ">Martin  Hügle</span>, <span class="hlFld-ContribAuthor ">Karin  Schmidtkunz</span>, <span class="hlFld-ContribAuthor ">Frank  Erdmann</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Matthias  Schmidt</span>, <span class="hlFld-ContribAuthor ">Elizabeth R.  Morales</span>, <span class="hlFld-ContribAuthor ">Christophe  Romier</span>, <span class="hlFld-ContribAuthor ">Stefan  Günther</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and biological evaluation of dual targeting inhibitors of histone deacetylase 6/8 and bromodomain BRPF1. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112338. <a href="https://doi.org/10.1016/j.ejmech.2020.112338" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112338</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112338%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bbiological%252Bevaluation%252Bof%252Bdual%252Btargeting%252Binhibitors%252Bof%252Bhistone%252Bdeacetylase%252B6%25252F8%252Band%252Bbromodomain%252BBRPF1%26aulast%3DGhazy%26aufirst%3DEhab%26date%3D2020%26volume%3D200%26spage%3D112338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dawei  Zhou</span>, <span class="hlFld-ContribAuthor ">Tsuyoshi  Hayashi</span>, <span class="hlFld-ContribAuthor ">Maxime  Jean</span>, <span class="hlFld-ContribAuthor ">Weili  Kong</span>, <span class="hlFld-ContribAuthor ">Guillaume  Fiches</span>, <span class="hlFld-ContribAuthor ">Ayan  Biswas</span>, <span class="hlFld-ContribAuthor ">Shuai  Liu</span>, <span class="hlFld-ContribAuthor ">Hailemichael O.  Yosief</span>, <span class="hlFld-ContribAuthor ">Xiaofeng  Zhang</span>, <span class="hlFld-ContribAuthor ">Jay  Bradner</span>, <span class="hlFld-ContribAuthor ">Jun  Qi</span>, <span class="hlFld-ContribAuthor ">Wei  Zhang</span>, <span class="hlFld-ContribAuthor ">Netty  Santoso</span>, <span class="hlFld-ContribAuthor ">Jian  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Polo-like kinase 1 (PLK1) facilitates the elimination of HIV-1 viral reservoirs in CD4
              +
              T cells ex vivo. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2020,</strong> <em>6 </em>
                                    (29)
                                     , eaba1941. <a href="https://doi.org/10.1126/sciadv.aba1941" title="DOI URL">https://doi.org/10.1126/sciadv.aba1941</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.aba1941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.aba1941%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DInhibition%252Bof%252BPolo-like%252Bkinase%252B1%252B%252528PLK1%252529%252Bfacilitates%252Bthe%252Belimination%252Bof%252BHIV-1%252Bviral%252Breservoirs%252Bin%252BCD4%252B%25252B%252BT%252Bcells%252Bex%252Bvivo%26aulast%3DZhou%26aufirst%3DDawei%26date%3D2020%26date%3D2020%26volume%3D6%26issue%3D29%26spage%3Deaba1941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fumiaki  Yokokawa</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress on phenotype-based discovery of dengue inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 541-551. <a href="https://doi.org/10.1039/D0MD00052C" title="DOI URL">https://doi.org/10.1039/D0MD00052C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00052C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00052C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bon%252Bphenotype-based%252Bdiscovery%252Bof%252Bdengue%252Binhibitors%26aulast%3DYokokawa%26aufirst%3DFumiaki%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D5%26spage%3D541%26epage%3D551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ning-Yu  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Xu</span>, <span class="hlFld-ContribAuthor ">Kun-Jie  Xiao</span>, <span class="hlFld-ContribAuthor ">Wei-Qiong  Zuo</span>, <span class="hlFld-ContribAuthor ">Yong-Xia  Zhu</span>, <span class="hlFld-ContribAuthor ">Rong  Hu</span>, <span class="hlFld-ContribAuthor ">Wan-Li  Wang</span>, <span class="hlFld-ContribAuthor ">Yao-Jie  Shi</span>, <span class="hlFld-ContribAuthor ">Luo-Ting  Yu</span>, <span class="hlFld-ContribAuthor ">Zhi-Hao  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and biological evaluation of 4,5-dihydro-[1,2,4]triazolo[4,3-f]pteridine derivatives as novel dual-PLK1/BRD4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>191 </em>, 112152. <a href="https://doi.org/10.1016/j.ejmech.2020.112152" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112152</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112152%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bbiological%252Bevaluation%252Bof%252B4%25252C5-dihydro-%25255B1%25252C2%25252C4%25255Dtriazolo%25255B4%25252C3-f%25255Dpteridine%252Bderivatives%252Bas%252Bnovel%252Bdual-PLK1%25252FBRD4%252Binhibitors%26aulast%3DWang%26aufirst%3DNing-Yu%26date%3D2020%26volume%3D191%26spage%3D112152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dejuan  Sun</span>, <span class="hlFld-ContribAuthor ">Yuqian  Zhao</span>, <span class="hlFld-ContribAuthor ">Shouyue  Zhang</span>, <span class="hlFld-ContribAuthor ">Lan  Zhang</span>, <span class="hlFld-ContribAuthor ">Bo  Liu</span>, <span class="hlFld-ContribAuthor ">Liang  Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Dual-target kinase drug design: Current strategies and future directions in cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>188 </em>, 112025. <a href="https://doi.org/10.1016/j.ejmech.2019.112025" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.112025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.112025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.112025%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDual-target%252Bkinase%252Bdrug%252Bdesign%25253A%252BCurrent%252Bstrategies%252Band%252Bfuture%252Bdirections%252Bin%252Bcancer%252Btherapy%26aulast%3DSun%26aufirst%3DDejuan%26date%3D2020%26volume%3D188%26spage%3D112025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalie  Timme</span>, <span class="hlFld-ContribAuthor ">Youjia  Han</span>, <span class="hlFld-ContribAuthor ">Shuai  Liu</span>, <span class="hlFld-ContribAuthor ">Hailemichael O.  Yosief</span>, <span class="hlFld-ContribAuthor ">Heathcliff Dorado  García</span>, <span class="hlFld-ContribAuthor ">Yi  Bei</span>, <span class="hlFld-ContribAuthor ">Filippos  Klironomos</span>, <span class="hlFld-ContribAuthor ">Ian C.  MacArthur</span>, <span class="hlFld-ContribAuthor ">Annabell  Szymansky</span>, <span class="hlFld-ContribAuthor ">Jennifer  von Stebut</span>, <span class="hlFld-ContribAuthor ">Victor  Bardinet</span>, <span class="hlFld-ContribAuthor ">Constantin  Dohna</span>, <span class="hlFld-ContribAuthor ">Annette  Künkele</span>, <span class="hlFld-ContribAuthor ">Jana  Rolff</span>, <span class="hlFld-ContribAuthor ">Patrick  Hundsdörfer</span>, <span class="hlFld-ContribAuthor ">Andrej  Lissat</span>, <span class="hlFld-ContribAuthor ">Georg  Seifert</span>, <span class="hlFld-ContribAuthor ">Angelika  Eggert</span>, <span class="hlFld-ContribAuthor ">Johannes H.  Schulte</span>, <span class="hlFld-ContribAuthor ">Wei  Zhang</span>, <span class="hlFld-ContribAuthor ">Anton G.  Henssen</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Dual PLK1 and BRD4 Inhibitors are Active Against Preclinical Models of Pediatric Solid Tumors. </span><span class="cited-content_cbyCitation_journal-name">Translational Oncology</span><span> <strong>2020,</strong> <em>13 </em>
                                    (2)
                                     , 221-232. <a href="https://doi.org/10.1016/j.tranon.2019.09.013" title="DOI URL">https://doi.org/10.1016/j.tranon.2019.09.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tranon.2019.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tranon.2019.09.013%26sid%3Dliteratum%253Aachs%26jtitle%3DTranslational%2520Oncology%26atitle%3DSmall-Molecule%252BDual%252BPLK1%252Band%252BBRD4%252BInhibitors%252Bare%252BActive%252BAgainst%252BPreclinical%252BModels%252Bof%252BPediatric%252BSolid%252BTumors%26aulast%3DTimme%26aufirst%3DNatalie%26date%3D2020%26volume%3D13%26issue%3D2%26spage%3D221%26epage%3D232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elizabeth  Henderson</span>, <span class="hlFld-ContribAuthor ">Panagis  Filippakopoulos</span>. </span><span class="cited-content_cbyCitation_article-title">BET mechanisms in cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 101-142. <a href="https://doi.org/10.1016/B978-0-12-816422-8.00006-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-816422-8.00006-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816422-8.00006-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816422-8.00006-4%26sid%3Dliteratum%253Aachs%26atitle%3DBET%252Bmechanisms%252Bin%252Bcancer%26aulast%3DHenderson%26aufirst%3DElizabeth%26date%3D2020%26spage%3D101%26epage%3D142%26pub%3DElsevier%26atitle%3DHistone%252BModifications%252Bin%252BTherapy%26date%3D2020%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Valeur</span>, <span class="hlFld-ContribAuthor ">Frank  Narjes</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>, <span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging modes-of-action in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 1550-1568. <a href="https://doi.org/10.1039/C9MD00263D" title="DOI URL">https://doi.org/10.1039/C9MD00263D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00263D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00263D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DEmerging%252Bmodes-of-action%252Bin%252Bdrug%252Bdiscovery%26aulast%3DValeur%26aufirst%3DEric%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D9%26spage%3D1550%26epage%3D1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea G.  Cochran</span>, <span class="hlFld-ContribAuthor ">Andrew R.  Conery</span>, <span class="hlFld-ContribAuthor ">Robert J.  Sims</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomains: a new target class for drug development. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2019,</strong> <em>18 </em>
                                    (8)
                                     , 609-628. <a href="https://doi.org/10.1038/s41573-019-0030-7" title="DOI URL">https://doi.org/10.1038/s41573-019-0030-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DBromodomains%25253A%252Ba%252Bnew%252Btarget%252Bclass%252Bfor%2525C2%2525A0drug%252Bdevelopment%26aulast%3DCochran%26aufirst%3DAndrea%2BG.%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D8%26spage%3D609%26epage%3D628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Kinase inhibitors with nanomolar inhibitory activity against BRD4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of BI-2536 (dark red) bound to (A) BRD4-BD1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a>) and (B) PLK1 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a>). The protein residues in contact with the ligand are represented by yellow sticks; hydrogen bonding is represented by red dashed lines, and the remaining residues near the binding pocked are shown in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Available modification sites on BI-2536.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0002.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of BI-2536 Analogues <b>1–20</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0003.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>21</b> and <b>23–25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Comparison of the per-residue decomposition of two BRD4-BD1 ligands, <b>23</b> in blue and BI-2536 in red, with intermolecular vdw interactions as a function of the receptor sequence. (B) Docking of <b>23</b> (cyan) with BRD4-BD1 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a>); protein residues in contact with the ligand are represented by yellow sticks, and the rest are in gray.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0011.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Comparison of the per-residue decomposition of two PLK1 ligands, <b>23</b> in blue and BI-2536 in red, with intermolecular vdw interactions as a function of the receptor sequence. (B) Docking of <b>23</b> (cyan) in PLK1 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a>); protein residues in contact with the ligand are represented by yellow sticks, and the rest are in gray. Hydrogen bonding is represented by a red dashed line.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0012.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Comparison of the per-residue decomposition of two BRD4-BD1 ligands, <b>6</b> in gray and BI-2536 in dark red, with intermolecular vdw interactions as a function of the receptor sequence. (B) Docking of <b>6</b> (gray) in BRD4 (1) (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a>); protein residues in contact with the ligand are represented by yellow sticks, and the rest are in gray. (C) Steric clashes caused by the propyl group if <b>6</b> is docked to BRD4 (1) in the same pose as BI-2536 (bumps are shown in red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/medium/jm-2018-00765d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0001.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Comparison of the per-residue decomposition of two PLK1-ligands, <b>6</b> in gray and BI-2536 in dark red, with intermolecular vdw interactions as a function of the receptor sequence. (B) Docking of <b>6</b> (gray) in PLK1 (PDBID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a>); protein residues in contact with the ligand are represented by yellow sticks with the rest in gray, and hydrogen bonding is represented by red dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00765/20180907/images/large/jm-2018-00765d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00765&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71880" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71880" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Beating the odds: BETs in disease</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">468</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.tibs.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=26145250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVyiurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2015&pages=468-479&author=C.+Y.+Wangauthor=P.+Filippakopoulos&title=Beating+the+odds%3A+BETs+in+disease&doi=10.1016%2Fj.tibs.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Beating the odds: BETs in disease</span></div><div class="casAuthors">Wang, Chen-Yi; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">468-479</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are evolutionarily conserved protein interaction modules that specifically recognize acetyl-lysine on histones and other proteins, facilitating roles in regulating gene transcription.  BRD-contg. proteins bound to chromatin loci such as enhancers are often deregulated in disease leading to aberrant expression of proinflammatory cytokines and growth-promoting genes.  Recent developments targeting the bromo and extraterminal (BET) subset of BRD proteins demonstrated remarkable efficacy in murine models providing a compelling rationale for drug development and translation to the clinic.  Here we summarise recent advances in our understanding of the roles of BETs in regulating gene transcription in normal and diseased tissue as well as the current status of their clin. translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhNr2U06Dv6bVg90H21EOLACvtfcHk0livR_D2QLHHAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVyiurfN&md5=5d3c4c0db326118f6b89792cee30a8a0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DBeating%2520the%2520odds%253A%2520BETs%2520in%2520disease%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2015%26volume%3D40%26spage%3D468%26epage%3D479%26doi%3D10.1016%2Fj.tibs.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvanese, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dar, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aull, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planelles, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siliciano, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain-targeting compounds reactivate HIV from latency via a Tat-independent mechanism</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">452</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.4161/cc.23309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.4161%2Fcc.23309" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=452-462&author=D.+Boehmauthor=V.+Calvaneseauthor=R.+D.+Darauthor=S.+Xingauthor=S.+Schroederauthor=L.+Martinsauthor=K.+Aullauthor=P.+C.+Liauthor=V.+Planellesauthor=J.+E.+Bradnerauthor=M.+M.+Zhouauthor=R.+F.+Silicianoauthor=L.+Weinbergerauthor=E.+Verdinauthor=M.+Ott&title=BET+bromodomain-targeting+compounds+reactivate+HIV+from+latency+via+a+Tat-independent+mechanism&doi=10.4161%2Fcc.23309"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.4161%2Fcc.23309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.23309%26sid%3Dliteratum%253Aachs%26aulast%3DBoehm%26aufirst%3DD.%26aulast%3DCalvanese%26aufirst%3DV.%26aulast%3DDar%26aufirst%3DR.%2BD.%26aulast%3DXing%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DS.%26aulast%3DMartins%26aufirst%3DL.%26aulast%3DAull%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DP.%2BC.%26aulast%3DPlanelles%26aufirst%3DV.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DSiliciano%26aufirst%3DR.%2BF.%26aulast%3DWeinberger%26aufirst%3DL.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DBET%2520bromodomain-targeting%2520compounds%2520reactivate%2520HIV%2520from%2520latency%2520via%2520a%2520Tat-independent%2520mechanism%26jtitle%3DCell%2520Cycle%26date%3D2013%26volume%3D12%26spage%3D452%26epage%3D462%26doi%3D10.4161%2Fcc.23309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delmore, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issa, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilpatrick, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinzel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffernan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsagel, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghobrial, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">904</span>– <span class="NLM_lpage">917</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lgElI3D3SikwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14532</span>– <span class="NLM_lpage">14537</span>, <span class="refDoi"> DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0lhkUhfNpEKORg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallenkamp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelato, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span> <span> </span><span class="NLM_article-title">Bromodomains and their pharmacological inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fcmdc.201300434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24497428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=438-464&author=D.+Gallenkampauthor=K.+A.+Gelatoauthor=B.+Haendlerauthor=H.+Weinmann&title=Bromodomains+and+their+pharmacological+inhibitors&doi=10.1002%2Fcmdc.201300434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains and Their Pharmacological Inhibitors</span></div><div class="casAuthors">Gallenkamp, Daniel; Gelato, Kathy A.; Haendler, Bernard; Weinmann, Hilmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">438-464</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Over 60 bromodomains belonging to proteins with very different functions have been identified in humans.  Several of them interact with acetylated lysine residues, leading to the recruitment and stabilization of protein complexes.  The bromodomain and extra-terminal domain (BET) proteins contain tandem bromodomains which bind to acetylated histones and are thereby implicated in a no. of DNA-centered processes, including the regulation of gene expression.  The recent identification of inhibitors of BET and non-BET bromodomains is one of the few examples in which effective blockade of a protein-protein interaction can be achieved with a small mol.  This has led to major strides in the understanding of the function of bromodomain-contg. proteins and their involvement in diseases such as cancer and inflammation.  Indeed, BET bromodomain inhibitors are now being clin. evaluated for the treatment of hematol. tumors and have also been tested in clin. trials for the relatively rare BRD-NUT midline carcinoma.  This review gives an overview of the newest developments in the field, with a focus on the biol. of selected bromodomain proteins on the one hand, and on reported pharmacol. inhibitors on the other, including recent examples from the patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-8_ymytCPbVg90H21EOLACvtfcHk0lhkUhfNpEKORg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D&md5=2ce7c6b5c37001fb8fb901fe04aeb606</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300434%26sid%3Dliteratum%253Aachs%26aulast%3DGallenkamp%26aufirst%3DD.%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DBromodomains%2520and%2520their%2520pharmacological%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D438%26epage%3D464%26doi%3D10.1002%2Fcmdc.201300434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lhkUhfNpEKORg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lhkUhfNpEKORg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding Sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+Sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lhwYRaxCo3LHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520Sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The bromodomain interaction module</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>586</i></span>,  <span class="NLM_fpage">2692</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2012.04.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.febslet.2012.04.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22710155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=2692-2704&author=P.+Filippakopoulosauthor=S.+Knapp&title=The+bromodomain+interaction+module&doi=10.1016%2Fj.febslet.2012.04.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain interaction module</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2692-2704</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ε-N-acetylation of protein Lys residues (Kac) is one of the most abundant post-translation modifications (PTMs) in the human proteome.  In the nucleus, the acetylation of histones has been linked to transcriptional activation of genes but the functional consequences of most acetylation events and proteins recruited to these sites remains largely unknown.  Bromodomains (BRDs) are small helical interaction modules that specifically recognize acetylation sites in proteins.  BRDs have recently emerged as interesting targets for the development of specific protein interaction inhibitors, enabling a novel exiting strategy for the development of new therapies.  Here, the author provides an overview over sequence requirements of BRDs, known substrates, and the structural mechanisms of specific Kac recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7LaThygz_HrVg90H21EOLACvtfcHk0lhwYRaxCo3LHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D&md5=c02dee22c330b1d5c1ba39579360ebc5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520bromodomain%2520interaction%2520module%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D2692%26epage%3D2704%26doi%3D10.1016%2Fj.febslet.2012.04.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Progress in the development and application of small molecule inhibitors of bromodomain–acetyl lysine interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9393</span>– <span class="NLM_lpage">9413</span>, <span class="refDoi"> DOI: 10.1021/jm300915b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300915b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9393-9413&author=D.+S.+Hewingsauthor=T.+P.+Rooneyauthor=L.+E.+Jenningsauthor=D.+A.+Hayauthor=C.+J.+Schofieldauthor=P.+E.+Brennanauthor=S.+Knappauthor=S.+J.+Conway&title=Progress+in+the+development+and+application+of+small+molecule+inhibitors+of+bromodomain%E2%80%93acetyl+lysine+interactions&doi=10.1021%2Fjm300915b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm300915b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300915b%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DJennings%26aufirst%3DL.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DProgress%2520in%2520the%2520development%2520and%2520application%2520of%2520small%2520molecule%2520inhibitors%2520of%2520bromodomain%25E2%2580%2593acetyl%2520lysine%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9393%26epage%3D9413%26doi%3D10.1021%2Fjm300915b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span> <span> </span><span class="NLM_article-title">Small molecule bromodomain inhibitors: extending the druggable genome</span>. <i>Prog. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-396493-9.00001-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2FB978-0-12-396493-9.00001-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22520470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChtr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1-55&author=C.+W.+Chung&title=Small+molecule+bromodomain+inhibitors%3A+extending+the+druggable+genome&doi=10.1016%2FB978-0-12-396493-9.00001-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule bromodomain inhibitors: extending the druggable genome</span></div><div class="casAuthors">Chung, Chun-Wa</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-55</span>CODEN:
                <span class="NLM_cas:coden">PMDCAY</span>;
        ISSN:<span class="NLM_cas:issn">0079-6468</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review on some of the existing knowledge for the young and rapidly evolving field of bromodomain inhibition.  Topics covered include structure and function of bromodomain, bromodomain-contg. proteins, therapeutic potential of bromodomains, and bromodomain inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI9QR6f10R97Vg90H21EOLACvtfcHk0lhwYRaxCo3LHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChtr7K&md5=6a947139e3939e752fb09a749938b3a9</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-396493-9.00001-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-396493-9.00001-7%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BW.%26atitle%3DSmall%2520molecule%2520bromodomain%2520inhibitors%253A%2520extending%2520the%2520druggable%2520genome%26jtitle%3DProg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D51%26spage%3D1%26epage%3D55%26doi%3D10.1016%2FB978-0-12-396493-9.00001-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yik, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">Recruitment of P-TEFb for stimulation of transcriptional elongation by the bromodomain protein Brd4</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.06.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.molcel.2005.06.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=16109377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=535-545&author=Z.+Yangauthor=J.+H.+Yikauthor=R.+Chenauthor=N.+Heauthor=M.+K.+Jangauthor=K.+Ozatoauthor=Q.+Zhou&title=Recruitment+of+P-TEFb+for+stimulation+of+transcriptional+elongation+by+the+bromodomain+protein+Brd4&doi=10.1016%2Fj.molcel.2005.06.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Recruitment of P-TEFb for stimulation of transcriptional elongation by the bromodomain protein Brd4</span></div><div class="casAuthors">Yang, Zhiyuan; Yik, Jasper H. N.; Chen, Ruichuan; He, Nanhai; Jang, Moon Kyoo; Ozato, Keiko; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">535-545</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cyclinT1/Cdk9 heterodimer that constitutes core P-TEFb is generally presumed to be the transcriptionally active form for stimulating RNA polymerase II elongation.  About half of cellular P-TEFb also exists in an inactive complex with the 7SK snRNA and the HEXIM1 protein.  Here, we show that the remaining half assocs. with the bromodomain protein Brd4.  In stress-induced cells, the 7SK/HEXIM1-bound P-TEFb is quant. converted into the Brd4-assocd. form.  The assocn. with Brd4 is necessary to form the transcriptionally active P-TEFb, recruits P-TEFb to a promoter, and enables P-TEFb to contact the Mediator complex, a potential target for the Brd4-mediated recruitment.  Although generally required for transcription, the P-TEFb-recruitment function of Brd4 can be substituted by that of HIV-1 Tat, which recruits P-TEFb directly for activated HIV-1 transcription.  Brd4, HEXIM1, and 7SK are all implicated in regulating cell growth, which may result from their dynamic control of the general transcription factor P-TEFb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptFELcZRBnvLVg90H21EOLACvtfcHk0lgHsZC04XfxXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur8%253D&md5=50325264e1eb27789eb46999ca3f799d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.06.029%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DYik%26aufirst%3DJ.%2BH.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DRecruitment%2520of%2520P-TEFb%2520for%2520stimulation%2520of%2520transcriptional%2520elongation%2520by%2520the%2520bromodomain%2520protein%2520Brd4%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D535%26epage%3D545%26doi%3D10.1016%2Fj.molcel.2005.06.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein Brd4 is a positive regulatory component of P-TEFb and stimulates RNA polymerase II-dependent transcription</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.06.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.molcel.2005.06.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=16109376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=523-534&author=M.+K.+Jangauthor=K.+Mochizukiauthor=M.+Zhouauthor=H.+S.+Jeongauthor=J.+N.+Bradyauthor=K.+Ozato&title=The+bromodomain+protein+Brd4+is+a+positive+regulatory+component+of+P-TEFb+and+stimulates+RNA+polymerase+II-dependent+transcription&doi=10.1016%2Fj.molcel.2005.06.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein Brd4 is a positive regulatory component of P-TEFb and stimulates RNA polymerase II-dependent transcription</span></div><div class="casAuthors">Jang, Moon Kyoo; Mochizuki, Kazuki; Zhou, Meisheng; Jeong, Ho-Sang; Brady, John N.; Ozato, Keiko</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">523-534</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Brd4 is a mammalian bromodomain protein that binds to acetylated chromatin.  Proteomic anal. revealed that Brd4 interacts with cyclinT1 and Cdk9 that constitutes core pos. transcription elongation factor b (P-TEFb).  Brd4 interacted with P-TEFb in the living nucleus through its bromodomain.  About half of P-TEFb was bound to the inhibitory subunit and functionally inactive.  Brd4 interacted with P-TEFb that was free of the inhibitory subunit.  An increase in Brd4 expression led to increased P-TEFb-dependent phosphorylation of RNA polymerase II (RNAPII) CTD and stimulation of transcription from promoters in vivo.  Conversely, a redn. in Brd4 expression by siRNA reduced CTD phosphorylation and transcription, revealing that Brd4 is a pos. regulatory component of P-TEFb.  In chromatin immunopptn. (ChlP) assays, the recruitment of P-TEFb to a promoter was dependent on Brd4 and was enhanced by an increase in chromatin acetylation.  Together, P-TEFb alternately interacts with Brd4 and the inhibitory subunit to maintain functional equil. in the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeaKLJgZobzrVg90H21EOLACvtfcHk0lgHsZC04XfxXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur4%253D&md5=90a7c10d7e7dd0e47d28ac708f9965ae</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.06.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.06.027%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DMochizuki%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DJeong%26aufirst%3DH.%2BS.%26aulast%3DBrady%26aufirst%3DJ.%2BN.%26aulast%3DOzato%26aufirst%3DK.%26atitle%3DThe%2520bromodomain%2520protein%2520Brd4%2520is%2520a%2520positive%2520regulatory%2520component%2520of%2520P-TEFb%2520and%2520stimulates%2520RNA%2520polymerase%2520II-dependent%2520transcription%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D523%26epage%3D534%26doi%3D10.1016%2Fj.molcel.2005.06.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgHsZC04XfxXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lgKKxah-dG7jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delannee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tellier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudit, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">From ApoA1 upregulation to BET family bromodomain inhibition: Discovery of I-BET151</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2963</span>– <span class="NLM_lpage">2967</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.01.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.bmcl.2012.01.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22386529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1elsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2963-2967&author=O.+Mirguetauthor=Y.+Lamotteauthor=F.+Doncheauthor=J.+Toumauthor=F.+Gellibertauthor=A.+Bouillotauthor=R.+Gosminiauthor=V.+L.+Nguyenauthor=D.+Delanneeauthor=J.+Sealauthor=F.+Blandelauthor=A.+B.+Boullayauthor=E.+Boursierauthor=S.+Martinauthor=J.+M.+Brusqauthor=G.+Krysaauthor=A.+Riouauthor=R.+Tellierauthor=A.+Costazauthor=P.+Huetauthor=Y.+Duditauthor=L.+Trottetauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=From+ApoA1+upregulation+to+BET+family+bromodomain+inhibition%3A+Discovery+of+I-BET151&doi=10.1016%2Fj.bmcl.2012.01.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">From ApoA1 upregulation to BET family bromodomain inhibition: Discovery of I-BET151</span></div><div class="casAuthors">Mirguet, Olivier; Lamotte, Yann; Donche, Frederic; Toum, Jerome; Gellibert, Francoise; Bouillot, Anne; Gosmini, Romain; Nguyen, Van-Loc; Delannee, Delphine; Seal, Jonathan; Blandel, Florence; Boullay, Anne-Benedicte; Boursier, Eric; Martin, Sandrine; Brusq, Jean-Marie; Krysa, Gael; Riou, Alizon; Tellier, Remi; Costaz, Agnes; Huet, Pascal; Dudit, Yann; Trottet, Lionel; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2963-2967</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery, synthesis, and biol. evaluation of a novel series of 7-isoxazoloquinolines is described.  Several analogs are shown to increase ApoA1 expression within the nanomolar range in the human hepatic cell line HepG2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIu6f80tbgE7Vg90H21EOLACvtfcHk0lgKKxah-dG7jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1elsbY%253D&md5=48c9a8dc4b70bea4a86807f7a1cec5f1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.01.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.01.125%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DDelannee%26aufirst%3DD.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DBlandel%26aufirst%3DF.%26aulast%3DBoullay%26aufirst%3DA.%2BB.%26aulast%3DBoursier%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DRiou%26aufirst%3DA.%26aulast%3DTellier%26aufirst%3DR.%26aulast%3DCostaz%26aufirst%3DA.%26aulast%3DHuet%26aufirst%3DP.%26aulast%3DDudit%26aufirst%3DY.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DFrom%2520ApoA1%2520upregulation%2520to%2520BET%2520family%2520bromodomain%2520inhibition%253A%2520Discovery%2520of%2520I-BET151%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2963%26epage%3D2967%26doi%3D10.1016%2Fj.bmcl.2012.01.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bish, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nocka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primiano, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralph, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciammetta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span> <span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra C-terminal bromodomain inhibition through optimization of a fragment-derived hit</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9831</span>– <span class="NLM_lpage">9837</span>, <span class="refDoi"> DOI: 10.1021/jm3010515</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+C-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit&doi=10.1021%2Fjm3010515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0lja2L4L9dyXow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520C-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837%26doi%3D10.1021%2Fjm3010515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of emerging epigenetic targets for cancer therapy: A patent review (2010–2014)</span>. <i>Pharm. Pat. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.4155/ppa.15.16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.4155%2Fppa.15.16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=26174566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKhtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=261-284&author=M.+Tanakaauthor=J.+M.+Robertsauthor=J.+Qiauthor=J.+E.+Bradner&title=Inhibitors+of+emerging+epigenetic+targets+for+cancer+therapy%3A+A+patent+review+%282010%E2%80%932014%29&doi=10.4155%2Fppa.15.16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of emerging epigenetic targets for cancer therapy: a patent review (2010-2014)</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Qi, Jun; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Patent Analyst</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-284</span>CODEN:
                <span class="NLM_cas:coden">PPAHCN</span>;
        ISSN:<span class="NLM_cas:issn">2046-8954</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Gene regulatory pathways comprise an emerging and active area of chem. probe discovery and investigational drug development.  Emerging insights from cancer genome sequencing and chromatin biol. have identified leveraged opportunities for development of chromatin-directed small mols. as cancer therapies.  At present, only six agents in two epigenetic target classes have been approved by the US FDA, limited to treatment of hematol. malignancies.  Recently, new classes of epigenetic inhibitors have appeared in literatures.  First-in-class compds. have successfully transitioned to clin. investigation, importantly also in solid tumors and pediatric malignancies.  This review considers patent applications for small-mol. inhibitors of selected epigenetic targets from 2010 to 2014.  Included are exemplary classes of chromatin-assocd. epigenomic writers (DOT1L and EZH2), erasers (LSD1) and readers (BRD4).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk2bznZTVgDLVg90H21EOLACvtfcHk0lja2L4L9dyXow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKhtr7I&md5=e72c0204edadeac9382a26256bbbbc6c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4155%2Fppa.15.16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Fppa.15.16%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DInhibitors%2520of%2520emerging%2520epigenetic%2520targets%2520for%2520cancer%2520therapy%253A%2520A%2520patent%2520review%2520%25282010%25E2%2580%25932014%2529%26jtitle%3DPharm.%2520Pat.%2520Anal.%26date%3D2015%26volume%3D4%26spage%3D261%26epage%3D284%26doi%3D10.4155%2Fppa.15.16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garnier, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibitors: a patent review</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1517/13543776.2014.859244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1517%2F13543776.2014.859244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24261714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptlWgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=185-199&author=J.+M.+Garnierauthor=P.+P.+Sharpauthor=C.+J.+Burns&title=BET+bromodomain+inhibitors%3A+a+patent+review&doi=10.1517%2F13543776.2014.859244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibitors: a patent review</span></div><div class="casAuthors">Garnier, Jean-Marc; Sharp, Phillip P.; Burns, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">185-199</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The bromodomain (BRD) and extra-C terminal domain (BET) protein family consists of four members (BRD2, BRD3, BRD4 and BRDT).  These "epigenetic readers" bind to acetyllysine (KAc) residues on the tails of histones H3 and H4, and regulate chromatin structure and gene expression.  There is increasing evidence of their role in human disease, and recently a no. of small-mol. inhibitors have been reported.  There is increasing interest in the inhibition of BET proteins for a variety of therapeutic applications that have resulted in considerable patent activity from academia and biotechnol. and pharmaceutical companies.  Areas covered: Data supporting the use of BET inhibitors in treating disease are outlined, and the current patent literature is discussed.  The survey is focused on patents claiming compds. as BET inhibitors and addnl. patents covering compds. now reported as BET inhibitors have been included.  Expert opinion: There is now compelling preclin. data demonstrating BET inhibition as a strategy to target processes known to be involved in disease development and progression with clin. trials of two bona fide BET inhibitors now underway.  Patent activity in this area is increasing with initial activity focused on variations to reported BET inhibitors and more recent patents disclosing novel chemotypes as BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6PVpCrReabVg90H21EOLACvtfcHk0lja2L4L9dyXow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptlWgtg%253D%253D&md5=bb33008eefa34f3689e0ee4a49a2539e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.859244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.859244%26sid%3Dliteratum%253Aachs%26aulast%3DGarnier%26aufirst%3DJ.%2BM.%26aulast%3DSharp%26aufirst%3DP.%2BP.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibitors%253A%2520a%2520patent%2520review%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2014%26volume%3D24%26spage%3D185%26epage%3D199%26doi%3D10.1517%2F13543776.2014.859244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meslamani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain: from epigenome reader to druggable target</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1839</i></span>,  <span class="NLM_fpage">676</span>– <span class="NLM_lpage">685</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.bbagrm.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24686119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=676-685&author=R.+Sanchezauthor=J.+Meslamaniauthor=M.+M.+Zhou&title=The+bromodomain%3A+from+epigenome+reader+to+druggable+target&doi=10.1016%2Fj.bbagrm.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain: From epigenome reader to druggable target</span></div><div class="casAuthors">Sanchez, Roberto; Meslamani, Jamel; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">676-685</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a fundamental post-translational modification that plays an important role in the control of gene transcription in chromatin in an ordered fashion.  The bromodomain, the conserved structural module present in transcription-assocd. proteins, functions exclusively to recognize acetyl-lysine on histones and non-histone proteins.  The structural analyses of bromodomains' recognition of lysine-acetylated peptides derived from histones and cellular proteins provide detailed insights into the differences and unifying features of biol. ligand binding selectivity by the bromodomains.  Newly developed small-mol. inhibitors targeting bromodomain proteins further highlight the functional importance of bromodomain/acetyl-lysine binding as a key mechanism in orchestrating mol. interactions and regulation in chromatin biol. and gene transcription.  These new studies argue that modulating bromodomain/acetyl-lysine interactions with small-mol. chems. offer new opportunities to control gene expression in a wide array of human diseases including cancer and inflammation.  This article is part of a Special Issue entitled: Mol. mechanisms of histone modification function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO3Au7jfK0ibVg90H21EOLACvtfcHk0lja2L4L9dyXow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtFagtrw%253D&md5=95ec21cf13231d9e63bce9b3ccdd6d6f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMeslamani%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DThe%2520bromodomain%253A%2520from%2520epigenome%2520reader%2520to%2520druggable%2520target%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D676%26epage%3D685%26doi%3D10.1016%2Fj.bbagrm.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shortt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">A chemical probe toolbox for dissecting the cancer epigenome</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1038/nrc.2016.148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnrc.2016.148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=28228643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjt1ags70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=160-183&author=J.+Shorttauthor=C.+J.+Ottauthor=R.+W.+Johnstoneauthor=J.+E.+Bradner&title=A+chemical+probe+toolbox+for+dissecting+the+cancer+epigenome&doi=10.1038%2Fnrc.2016.148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe toolbox for dissecting the cancer epigenome</span></div><div class="casAuthors">Shortt, Jake; Ott, Christopher J.; Johnstone, Ricky W.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">160-183</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer cell hallmarks are underpinned by transcriptional programs operating in the context of a dynamic and complicit epigenomic environment.  Somatic alterations of chromatin modifiers are among the most prevalent cancer perturbations.  There is a pressing need for targeted chem. probes to dissect these complex, interconnected gene regulatory circuits.  Validated chem. probes empower mechanistic research while providing the pharmacol. proof of concept that is required to translate drug-like derivs. into therapy for cancer patients.  In this review, we describe chem. probe development for epigenomic effector proteins that are linked to cancer pathogenesis.  By annotating these reagents, we aim to share our perspectives on an informative 'epigenomic toolbox' of broad utility to the research community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRUui_EZyjwrVg90H21EOLACvtfcHk0lh1sY9_K1CFmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjt1ags70%253D&md5=8f5f05608a3a035deb494d08445b249f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2016.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2016.148%26sid%3Dliteratum%253Aachs%26aulast%3DShortt%26aufirst%3DJ.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DA%2520chemical%2520probe%2520toolbox%2520for%2520dissecting%2520the%2520cancer%2520epigenome%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D160%26epage%3D183%26doi%3D10.1038%2Fnrc.2016.148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witwicki, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huh, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekram, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1038/nature16508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnature16508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=26735014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=413-417&author=S.+Shuauthor=C.+Y.+Linauthor=H.+H.+Heauthor=R.+M.+Witwickiauthor=D.+P.+Tabassumauthor=J.+M.+Robertsauthor=M.+Janiszewskaauthor=S.+J.+Huhauthor=Y.+Liangauthor=J.+Ryanauthor=E.+Dohertyauthor=H.+Mohammedauthor=H.+Guoauthor=D.+G.+Stoverauthor=M.+B.+Ekramauthor=J.+Brownauthor=C.+D%E2%80%99Santosauthor=I.+E.+Kropauthor=D.+Dillonauthor=M.+McKeownauthor=C.+Ottauthor=J.+Qiauthor=M.+Niauthor=P.+K.+Raoauthor=M.+Duarteauthor=S.+Y.+Wuauthor=C.+M.+Chiangauthor=L.+Andersauthor=R.+A.+Youngauthor=E.+Winerauthor=A.+Letaiauthor=W.+T.+Barryauthor=J.+S.+Carrollauthor=H.+Longauthor=M.+Brownauthor=X.+S.+Liuauthor=C.+A.+Meyerauthor=J.+E.+Bradnerauthor=K.+Polyak&title=Response+and+resistance+to+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1038%2Fnature16508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span></div><div class="casAuthors">Shu, Shaokun; Lin, Charles Y.; He, Housheng Hansen; Witwicki, Robert M.; Tabassum, Doris P.; Roberts, Justin M.; Janiszewska, Michalina; Huh, Sung Jin; Liang, Yi; Ryan, Jeremy; Doherty, Ernest; Mohammed, Hisham; Guo, Hao; Stover, Daniel G.; Ekram, Muhammad B.; Peluffo, Guillermo; Brown, Jonathan; D'Santos, Clive; Krop, Ian E.; Dillon, Deborah; McKeown, Michael; Ott, Christopher; Qi, Jun; Ni, Min; Rao, Prakash K.; Duarte, Melissa; Wu, Shwu-Yuan; Chiang, Cheng-Ming; Anders, Lars; Young, Richard A.; Winer, Eric P.; Letai, Antony; Barry, William T.; Carroll, Jason S.; Long, Henry W.; Brown, Myles; Shirley Liu, X.; Meyer, Clifford A.; Bradner, James E.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">413-417</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous and clin. aggressive disease for which there is no targeted therapy.  BET bromodomain inhibitors, which have shown efficacy in several models of cancer, have not been evaluated in TNBC.  These inhibitors displace BET bromodomain proteins such as BRD4 from chromatin by competing with their acetyl-lysine recognition modules, leading to inhibition of oncogenic transcriptional programs.  Here we report the preferential sensitivity of TNBCs to BET bromodomain inhibition in vitro and in vivo, establishing a rationale for clin. investigation and further motivation to understand mechanisms of resistance.  In paired cell lines selected for acquired resistance to BET inhibition from previously sensitive TNBCs, we failed to identify gatekeeper mutations, new driver events or drug pump activation.  BET-resistant TNBC cells remain dependent on wild-type BRD4, which supports transcription and cell proliferation in a bromodomain-independent manner.  Proteomic studies of resistant TNBC identify strong assocn. with MED1 and hyper-phosphorylation of BRD4 attributable to decreased activity of PP2A, identified here as a principal BRD4 serine phosphatase.  Together, these studies provide a rationale for BET inhibition in TNBC and present mechanism-based combination strategies to anticipate clin. drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZu5A85pdQDbVg90H21EOLACvtfcHk0lh1sY9_K1CFmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D&md5=50a03861d9156ee6e344e95c5f5361b9</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature16508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16508%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DWitwicki%26aufirst%3DR.%2BM.%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DJaniszewska%26aufirst%3DM.%26aulast%3DHuh%26aufirst%3DS.%2BJ.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DJ.%26aulast%3DDoherty%26aufirst%3DE.%26aulast%3DMohammed%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DEkram%26aufirst%3DM.%2BB.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DDillon%26aufirst%3DD.%26aulast%3DMcKeown%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DP.%2BK.%26aulast%3DDuarte%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DAnders%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DWiner%26aufirst%3DE.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DBarry%26aufirst%3DW.%2BT.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DResponse%2520and%2520resistance%2520to%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D413%26epage%3D417%26doi%3D10.1038%2Fnature16508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gutteridge, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndiaye, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, N.</span></span> <span> </span><span class="NLM_article-title">PLK1 inhibitors in cancer therapy: From laboratory to clinics</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1427</span>– <span class="NLM_lpage">1435</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-15-0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1158%2F1535-7163.MCT-15-0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=27330107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSrtbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=1427-1435&author=R.+E.+Gutteridgeauthor=M.+A.+Ndiayeauthor=X.+Liuauthor=N.+Ahmad&title=PLK1+inhibitors+in+cancer+therapy%3A+From+laboratory+to+clinics&doi=10.1158%2F1535-7163.MCT-15-0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Plk1 Inhibitors in Cancer Therapy: From Laboratory to Clinics</span></div><div class="casAuthors">Gutteridge, Rosie Elizabeth Ann; Ndiaye, Mary Ann; Liu, Xiaoqi; Ahmad, Nihal</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1427-1435</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Polo-like kinase 1 (Plk1) overexpression has been shown to occur in a wide range of tumors, prompting research and development of Plk1 inhibitors as a means of cancer treatment.  This review discusses recent advances in the development of Plk1 inhibitors for cancer management.  Plk1 inhibition has been shown to cause mitotic block and apoptosis of cells with higher mitotic index and therefore higher Plk1 expression.  The potential of Plk1 inhibitors as cancer therapeutics has been widely investigated.  However, a complete understanding of Plk1 biol./mechanism is yet to be fully achieved.  Resistance to certain chemotherapeutic drugs has been linked to Plk1 overexpression, and Plk1-mediated mitotic events such as microtubule rearrangement have been found to reduce the efficacy of chemotherapeutic agents.  The Plk1 inhibitor volasertib has shown considerable promise in clin. studies, having reached phase III trials.  However, preclin. success with Plk1 inhibitors has not translated well into clin. success.  In our view, combined therapies targeting other relevant pathways together with Plk1 may be vital to combat issues obsd. with monotherapy, esp. resistance.  In addn., research should also be directed toward understanding the mechanisms of Plk1 and designing addnl. next generations of specific, potent Plk1 inhibitors to target cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0TZb-4-s_ULVg90H21EOLACvtfcHk0ljNwu6OfapiLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSrtbbM&md5=dacdc56c51e8b16c484ac44cdb61b9fb</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-15-0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-15-0897%26sid%3Dliteratum%253Aachs%26aulast%3DGutteridge%26aufirst%3DR.%2BE.%26aulast%3DNdiaye%26aufirst%3DM.%2BA.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DAhmad%26aufirst%3DN.%26atitle%3DPLK1%2520inhibitors%2520in%2520cancer%2520therapy%253A%2520From%2520laboratory%2520to%2520clinics%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D1427%26epage%3D1435%26doi%3D10.1158%2F1535-7163.MCT-15-0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seton-Rogers, S.</span></span> <span> </span><span class="NLM_article-title">Epigenetics: Place your BETs</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">638</span>, <span class="refDoi"> DOI: 10.1038/nrc4035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnrc4035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=26469143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Kqt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=638&author=S.+Seton-Rogers&title=Epigenetics%3A+Place+your+BETs&doi=10.1038%2Fnrc4035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics Place your BETs</span></div><div class="casAuthors">Seton-Rogers, Sarah</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">638</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Three papers have reported new data on resistance to bromodomain and extraterminal (BET) inhibitors and how best to use these inhibitors in combination therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRNwJQ_NIQ1rVg90H21EOLACvtfcHk0ljNwu6OfapiLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Kqt7vK&md5=f7b4ad27f8e7285cd914aada2e240620</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnrc4035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc4035%26sid%3Dliteratum%253Aachs%26aulast%3DSeton-Rogers%26aufirst%3DS.%26atitle%3DEpigenetics%253A%2520Place%2520your%2520BETs%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D638%26doi%3D10.1038%2Fnrc4035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elzey, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">PLK1 inhibition enhances the efficacy of bet epigenetic reader blockade in castration-resistant prostate cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1554</span>– <span class="NLM_lpage">1565</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-17-0945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1158%2F1535-7163.MCT-17-0945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=29716963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1alur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1554-1565&author=F.+Maoauthor=J.+Liauthor=Q.+Luoauthor=R.+Wangauthor=Y.+Kongauthor=C.+Carlockauthor=Z.+Liuauthor=B.+D.+Elzeyauthor=X.+Liu&title=PLK1+inhibition+enhances+the+efficacy+of+bet+epigenetic+reader+blockade+in+castration-resistant+prostate+cancer&doi=10.1158%2F1535-7163.MCT-17-0945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Plk1 Inhibition Enhances the Efficacy of BET Epigenetic Reader Blockade in Castration-Resistant Prostate Cancer</span></div><div class="casAuthors">Mao, Fengyi; Li, Jie; Luo, Qian; Wang, Ruixin; Kong, Yifan; Carlock, Colin; Liu, Zian; Elzey, Bennet D.; Liu, Xiaoqi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1554-1565</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Polo-like kinase 1 (Plk1), a crucial regulator of cell-cycle progression, is overexpressed in multiple types of cancers and has been proven to be a potent and promising target for cancer treatment.  In case of prostate cancer, we once showed that antineoplastic activity of Plk1 inhibitor is largely due to inhibition of androgen receptor (AR) signaling.  However, we also discovered that Plk1 inhibition causes activation of the β-catenin pathway and increased expression of c-MYC, eventually resulting in resistance to Plk1 inhibition.  JQ1, a selective small-mol. inhibitor targeting the amino-terminal bromodomains of BRD4, has been shown to dramatically inhibit c-MYC expression and AR signaling, exhibiting antiproliferative effects in a range of cancers.  Because c-MYC and AR signaling are essential for prostate cancer initiation and progression, we aim to test whether targeting Plk1 and BRD4 at the same time is an effective approach to treat prostate cancer.  Herein, we show that a combination of Plk1 inhibitor GSK461364A and BRD4 inhibitor JQ1 had a strong synergistic effect on castration-resistant prostate cancer (CRPC) cell lines, as well as in CRPC xenograft tumors.  Mechanistically, the synergistic effect is likely due to two reasons: (i) Plk1 inhibition results in the accumulation of β-catenin in the nucleus, thus elevation of c-MYC expression, whereas JQ1 treatment directly suppresses c-MYC transcription; (ii) Plk1 and BRD4 dual inhibition acts synergistically in inhibition of AR signaling.  Mol Cancer Ther; 17(7); 1554-65. ©2018 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWfot-k6rAvLVg90H21EOLACvtfcHk0ljw4jJG4G06CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1alur7F&md5=9e5aec6def8df63a1c2e6fadabef26ad</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-17-0945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-17-0945%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DCarlock%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DElzey%26aufirst%3DB.%2BD.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DPLK1%2520inhibition%2520enhances%2520the%2520efficacy%2520of%2520bet%2520epigenetic%2520reader%2520blockade%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2018%26volume%3D17%26spage%3D1554%26epage%3D1565%26doi%3D10.1158%2F1535-7163.MCT-17-0945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tontsch-Grunt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waizenegger, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savarese, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span> <span> </span><span class="NLM_article-title">Synergistic activity of BET inhibitor BI 894999 with PLK inhibitor volasertib in AML in vitro and in vivo</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2018.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1016%2Fj.canlet.2018.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=29454094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsVGisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2018&pages=112-120&author=U.+Tontsch-Gruntauthor=D.+Rudolphauthor=I.+Waizeneggerauthor=A.+Baumauthor=D.+Gerlachauthor=H.+Engelhardtauthor=M.+Wurmauthor=F.+Savareseauthor=N.+Schweiferauthor=N.+Kraut&title=Synergistic+activity+of+BET+inhibitor+BI+894999+with+PLK+inhibitor+volasertib+in+AML+in+vitro+and+in+vivo&doi=10.1016%2Fj.canlet.2018.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic activity of BET inhibitor BI 894999 with PLK inhibitor volasertib in AML in vitro and in vivo</span></div><div class="casAuthors">Tontsch-Grunt, Ulrike; Rudolph, Dorothea; Waizenegger, Irene; Baum, Anke; Gerlach, Daniel; Engelhardt, Harald; Wurm, Melanie; Savarese, Fabio; Schweifer, Norbert; Kraut, Norbert</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112-120</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Interactions between a new potent Bromodomain and extraterminal domain (BET) inhibitor BI 894999 and the polo-like kinase (PLK) inhibitor volasertib were studied in acute myeloid leukemia cell lines in vitro and in vivo.  We provide data for the distinct mechanisms of action of these two compds. with a potential utility in AML based on gene expression, cell cycle profile and modulation of PD biomarkers such as MYC and HEXIM1.  In contrast to BI 894999, volasertib treatment neither affects MYC nor HEXIM1 expression, but augments and prolongs the decrease of MYC expression caused by BI 894999 treatment.  In vitro combination of both compds. leads to a decrease in S-Phase and to increased apoptosis.  In vitro scheduling expts. guided in vivo expts. in disseminated AML mouse models.  Co-administration of BI 894999 and volasertib dramatically reduces tumor burden accompanied by long-term survival of tumor-bearing mice and eradication of AML cells in mouse bone marrow.  Together, these preclin. findings provide evidence for the strong synergistic effect of BI 894999 and volasertib, warranting future clin. studies in patients with AML to investigate this paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyqA7Dr13AlrVg90H21EOLACvtfcHk0ljw4jJG4G06CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsVGisb8%253D&md5=7ed409793589b9ca3e8b1053614881ea</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2018.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2018.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DTontsch-Grunt%26aufirst%3DU.%26aulast%3DRudolph%26aufirst%3DD.%26aulast%3DWaizenegger%26aufirst%3DI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DGerlach%26aufirst%3DD.%26aulast%3DEngelhardt%26aufirst%3DH.%26aulast%3DWurm%26aufirst%3DM.%26aulast%3DSavarese%26aufirst%3DF.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DKraut%26aufirst%3DN.%26atitle%3DSynergistic%2520activity%2520of%2520BET%2520inhibitor%2520BI%2520894999%2520with%2520PLK%2520inhibitor%2520volasertib%2520in%2520AML%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Lett.%26date%3D2018%26volume%3D421%26spage%3D112%26epage%3D120%26doi%3D10.1016%2Fj.canlet.2018.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlich, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durden, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Dual-activity PI3K–BRD4 inhibitor for the orthogonal inhibition of MYC to block tumor growth and metastasis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E1072</span>– <span class="NLM_lpage">E1080</span>, <span class="refDoi"> DOI: 10.1073/pnas.1613091114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1073%2Fpnas.1613091114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E1072-E1080&author=F.+H.+Andrewsauthor=A.+R.+Singhauthor=S.+Joshiauthor=C.+A.+Smithauthor=G.+A.+Moralesauthor=J.+R.+Garlichauthor=D.+L.+Durdenauthor=T.+G.+Kutateladze&title=Dual-activity+PI3K%E2%80%93BRD4+inhibitor+for+the+orthogonal+inhibition+of+MYC+to+block+tumor+growth+and+metastasis&doi=10.1073%2Fpnas.1613091114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1613091114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1613091114%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DF.%2BH.%26aulast%3DSingh%26aufirst%3DA.%2BR.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DC.%2BA.%26aulast%3DMorales%26aufirst%3DG.%2BA.%26aulast%3DGarlich%26aufirst%3DJ.%2BR.%26aulast%3DDurden%26aufirst%3DD.%2BL.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DDual-activity%2520PI3K%25E2%2580%2593BRD4%2520inhibitor%2520for%2520the%2520orthogonal%2520inhibition%2520of%2520MYC%2520to%2520block%2520tumor%2520growth%2520and%2520metastasis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE1072%26epage%3DE1080%26doi%3D10.1073%2Fpnas.1613091114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgoyne, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicklick, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kono, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlich, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durden, D. L.</span></span> <span> </span><span class="NLM_article-title">Single agent and synergistic activity of the “first-in-class” Dual PI3K/BRD4 inhibitor SF1126 with sorafenib in hepatocellular carcinoma</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2553</span>– <span class="NLM_lpage">2562</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-15-0976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1158%2F1535-7163.MCT-15-0976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=27496136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2553-2562&author=A.+R.+Singhauthor=S.+Joshiauthor=A.+M.+Burgoyneauthor=J.+K.+Sicklickauthor=S.+Ikedaauthor=Y.+Konoauthor=J.+R.+Garlichauthor=G.+A.+Moralesauthor=D.+L.+Durden&title=Single+agent+and+synergistic+activity+of+the+%E2%80%9Cfirst-in-class%E2%80%9D+Dual+PI3K%2FBRD4+inhibitor+SF1126+with+sorafenib+in+hepatocellular+carcinoma&doi=10.1158%2F1535-7163.MCT-15-0976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Single agent and synergistic activity of the "first-in-class" dual PI3K/BRD4 inhibitor SF1126 with sorafenib in hepatocellular carcinoma</span></div><div class="casAuthors">Singh, Alok R.; Joshi, Shweta; Burgoyne, Adam M.; Sicklick, Jason K.; Ikeda, Sadakatsu; Kono, Yuko; Garlich, Joseph R.; Morales, Guillermo A.; Durden, Donald L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2553-2562</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Deregulated PI3K/AKT/mTOR, Ras/Raf/MAPK, and c-Myc signaling pathways are of prognostic significance in hepatocellular carcinoma (HCC).  Sorafenib, the only drug clin. approved for patients with advanced HCC, blocks the Ras/Raf/MAPK pathway but it does not inhibit the PI3K/AKT/mTOR pathway or c-Myc activation.  Hence, there is an unmet medical need to identify potent PI3K/BRD4 inhibitors, which can be used either alone or in combination with sorafenib to treat patients with advanced HCC.  Herein, we show that SF1126 (pan PI3K/BRD4 inhibitor) as single agent or in combination with sorafenib inhibited proliferation, cell cycle, apoptosis, and multiple key enzymes in PI3K/AKT/mTOR and Ras/Raf/MAPK pathway in Hep3B, HepG2, SK-Hep1, and Huh7 HCC cell lines.  We demonstrate that the active moiety of the SF1126 prodrug LY294002 binds to and blocks BRD4 interaction with the acetylated histone-H4 chromatin mark protein and displaced BRD4 coactivator protein from the transcriptional start site of MYC in Huh7 and SK-Hep1 HCC cell lines.  Moreover, SF1126 blocked expression levels of c-Myc in HCC cells.  Treatment of SF1126 either alone or in combination with sorafenib showed significant antitumor activity in vivo.  Our results establish that SF1126 is a dual PI3K/BRD4 inhibitor.  This agent has completed a phase I clin. trial in humans with good safety profile.  Our data support the potential future consideration of a phase II clin. trial of SF1126, a clin. relevant dual "first-in-class" PI3K/BRD4 inhibitor in advanced HCC, and a potential combination with sorafenib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1PYR8DNGoU7Vg90H21EOLACvtfcHk0lirHgRxIRzetA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqs7Y%253D&md5=e0133b38f398c81afca96d078f54db86</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-15-0976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-15-0976%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DA.%2BR.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DBurgoyne%26aufirst%3DA.%2BM.%26aulast%3DSicklick%26aufirst%3DJ.%2BK.%26aulast%3DIkeda%26aufirst%3DS.%26aulast%3DKono%26aufirst%3DY.%26aulast%3DGarlich%26aufirst%3DJ.%2BR.%26aulast%3DMorales%26aufirst%3DG.%2BA.%26aulast%3DDurden%26aufirst%3DD.%2BL.%26atitle%3DSingle%2520agent%2520and%2520synergistic%2520activity%2520of%2520the%2520%25E2%2580%259Cfirst-in-class%25E2%2580%259D%2520Dual%2520PI3K%252FBRD4%2520inhibitor%2520SF1126%2520with%2520sorafenib%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2553%26epage%3D2562%26doi%3D10.1158%2F1535-7163.MCT-15-0976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-lysine binding site of bromodomain-containing protein 4(BRD4) interacts with diverse kinase inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+Emberauthor=J.+Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Acetyl-lysine+binding+site+of+bromodomain-containing+protein+4%28BRD4%29+interacts+with+diverse+kinase+inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DAcetyl-lysine%2520binding%2520site%2520of%2520bromodomain-containing%2520protein%25204%2528BRD4%2529%2520interacts%2520with%2520diverse%2520kinase%2520inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahony&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lirHgRxIRzetA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koblan, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Assessment of bromodomain target engagement by a series of BI2536 analogues with miniaturized BET-BRET</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2581</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fcmdc.201600502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=27862999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVChsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2575-2581&author=L.+W.+Koblanauthor=D.+L.+Buckleyauthor=C.+J.+Ottauthor=M.+E.+Fitzgeraldauthor=S.+W.+Emberauthor=J.+Y.+Zhuauthor=S.+Liuauthor=J.+M.+Robertsauthor=D.+Remillardauthor=S.+Vittoriauthor=W.+Zhangauthor=E.+Schonbrunnauthor=J.+E.+Bradner&title=Assessment+of+bromodomain+target+engagement+by+a+series+of+BI2536+analogues+with+miniaturized+BET-BRET&doi=10.1002%2Fcmdc.201600502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Bromodomain Target Engagement by a Series of BI2536 Analogues with Miniaturized BET-BRET</span></div><div class="casAuthors">Koblan, Luke W.; Buckley, Dennis L.; Ott, Christopher J.; Fitzgerald, Mark E.; Ember, Stuart W. J.; Zhu, Jin-Yi; Liu, Shuai; Roberts, Justin M.; Remillard, David; Vittori, Sarah; Zhang, Wei; Schonbrunn, Ernst; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2575-2581</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Evaluating the engagement of a small mol. ligand with a protein target in cells provides useful information for chem. probe optimization and pharmaceutical development.  While several techniques exist that can be performed in a low-throughput manner, systematic evaluation of large compd. libraries remains a challenge.  In-cell engagement measurements are esp. useful when evaluating compd. classes suspected to target multiple cellular factors.  In this study we used a bioluminescent resonant energy transfer assay to assess bromodomain engagement by a compd. series contg. bromodomain- and kinase-biasing polypharmacophores based on the known dual BRD4 bromodomain/PLK1 kinase inhibitor BI2536.  With this assay, we discovered several novel agents with bromodomain-selective specificity profiles and cellular activity.  Thus, this platform aids in distinguishing mols. whose cellular activity is difficult to assess due to polypharmacol. effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiVVtnPQvEQrVg90H21EOLACvtfcHk0lirHgRxIRzetA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVChsbfK&md5=a6d7c6a58870eb6208e18eb32068d821</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600502%26sid%3Dliteratum%253Aachs%26aulast%3DKoblan%26aufirst%3DL.%2BW.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DEmber%26aufirst%3DS.%2BW.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DAssessment%2520of%2520bromodomain%2520target%2520engagement%2520by%2520a%2520series%2520of%2520BI2536%2520analogues%2520with%2520miniaturized%2520BET-BRET%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2575%26epage%3D2581%26doi%3D10.1002%2Fcmdc.201600502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fountain, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheenstra, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strovel, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span> <span> </span><span class="NLM_article-title">BRD4 structure–activity relationships of dual PLK1 Kinase/BRD4 bromodomain inhibitor BI-2536</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">769</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00084</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00084" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=764-769&author=L.+Chenauthor=J.+L.+Yapauthor=M.+Yoshiokaauthor=M.+E.+Lanningauthor=R.+N.+Fountainauthor=M.+Rajeauthor=J.+A.+Scheenstraauthor=J.+W.+Strovelauthor=S.+Fletcher&title=BRD4+structure%E2%80%93activity+relationships+of+dual+PLK1+Kinase%2FBRD4+bromodomain+inhibitor+BI-2536&doi=10.1021%2Facsmedchemlett.5b00084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Structure-Activity Relationships of Dual PLK1 Kinase/BRD4 Bromodomain Inhibitor BI-2536</span></div><div class="casAuthors">Chen, Lijia; Yap, Jeremy L.; Yoshioka, Makoto; Lanning, Maryanna E.; Fountain, Rachel N.; Raje, Mithun; Scheenstra, Jacob A.; Strovel, Jeffrey W.; Fletcher, Steven</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">764-769</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused library of analogs of the dual PLK1 kinase/BRD4 bromodomain inhibitor BI-2536 was prepd. and then analyzed for BRD4 and PLK1 inhibitory activities.  Particularly, replacement of the cyclopentyl group with a 3-bromobenzyl moiety afforded the most potent BRD4 inhibitor of the series (39j) with a Ki = 8.7 nM, which was equipotent against PLK1.  The superior affinity of 39j over the parental compd. to BRD4 possibly derives from improved interactions with the WPF shelf.  Meanwhile, substitution of the pyrimidine NH with an oxygen atom reversed the PLK1/BRD4 selectivity to convert BI-2536 into a BRD4-selective inhibitor, likely owing to the loss of a crit. hydrogen bond in PLK1.  We believe further fine-tuning will furnish a BRD4 "magic bullet" or an even more potent PLK1/BRD4 dual inhibitor toward the expansion and improved efficacy of the chemotherapy arsenal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5UnNSmf1L17Vg90H21EOLACvtfcHk0lgy9NoicxBbaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVChtb0%253D&md5=f39070e652bca20c03a3bc5fe070a7fc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00084%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYap%26aufirst%3DJ.%2BL.%26aulast%3DYoshioka%26aufirst%3DM.%26aulast%3DLanning%26aufirst%3DM.%2BE.%26aulast%3DFountain%26aufirst%3DR.%2BN.%26aulast%3DRaje%26aufirst%3DM.%26aulast%3DScheenstra%26aufirst%3DJ.%2BA.%26aulast%3DStrovel%26aufirst%3DJ.%2BW.%26aulast%3DFletcher%26aufirst%3DS.%26atitle%3DBRD4%2520structure%25E2%2580%2593activity%2520relationships%2520of%2520dual%2520PLK1%2520Kinase%252FBRD4%2520bromodomain%2520inhibitor%2520BI-2536%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D764%26epage%3D769%26doi%3D10.1021%2Facsmedchemlett.5b00084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelias, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lgy9NoicxBbaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span> <span> </span><span class="NLM_article-title">Leveraging kinase inhibitors to develop small molecule tools for imaging kinases by fluorescence microscopy</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2523</span>– <span class="NLM_lpage">2526</span>, <span class="refDoi"> DOI: 10.1039/c2mb25099c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1039%2Fc2mb25099c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=22673640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2523-2526&author=Z.+Zhangauthor=N.+Kwiatkowskiauthor=H.+Zengauthor=S.+M.+Limauthor=N.+S.+Grayauthor=W.+Zhangauthor=P.+L.+Yang&title=Leveraging+kinase+inhibitors+to+develop+small+molecule+tools+for+imaging+kinases+by+fluorescence+microscopy&doi=10.1039%2Fc2mb25099c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging kinase inhibitors to develop small molecule tools for imaging kinases by fluorescence microscopy</span></div><div class="casAuthors">Zhang, Zijuan; Kwiatkowski, Nicholas; Zeng, Hong; Lim, Sang Min; Gray, Nathanael S.; Zhang, Wei; Yang, Priscilla L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2523-2526</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As the usage of fluorescence microscopy as a tool to study biol. systems continues to grow, so does the need for addnl. tools that permit the selective detection of proteins of interest.  Existing selective and well-characterized kinase inhibitors may be exploited to develop novel small mol. probes useful in imaging kinases by fluorescence microscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkTi2Qtkh07Vg90H21EOLACvtfcHk0ljVI49M1hR6TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbnF&md5=e59fd8a1bbd080bcfa2653fd7250328e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2Fc2mb25099c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2mb25099c%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DP.%2BL.%26atitle%3DLeveraging%2520kinase%2520inhibitors%2520to%2520develop%2520small%2520molecule%2520tools%2520for%2520imaging%2520kinases%2520by%2520fluorescence%2520microscopy%26jtitle%3DMol.%2520BioSyst.%26date%3D2012%26volume%3D8%26spage%3D2523%26epage%3D2526%26doi%3D10.1039%2Fc2mb25099c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera - A visualization system for exploratory research and analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera+-+A+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0ljVI49M1hR6TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%2520-%2520A%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balius, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brozell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moustakas, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, R. C.</span></span> <span> </span><span class="NLM_article-title">DOCK 6: Impact of new features and current docking performance</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1156</span>, <span class="refDoi"> DOI: 10.1002/jcc.23905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.23905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=25914306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1132-1156&author=W.+J.+Allenauthor=T.+E.+Baliusauthor=S.+Mukherjeeauthor=S.+R.+Brozellauthor=D.+T.+Moustakasauthor=P.+T.+Langauthor=D.+A.+Caseauthor=I.+D.+Kuntzauthor=R.+C.+Rizzo&title=DOCK+6%3A+Impact+of+new+features+and+current+docking+performance&doi=10.1002%2Fjcc.23905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">DOCK 6: Impact of new features and current docking performance</span></div><div class="casAuthors">Allen, William J.; Balius, Trent E.; Mukherjee, Sudipto; Brozell, Scott R.; Moustakas, Demetri T.; Lang, P. Therese; Case, David A.; Kuntz, Irwin D.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1132-1156</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This manuscript presents the latest algorithmic and methodol. developments to the structure-based design program DOCK 6.7 focused on an updated internal energy function, new anchor selection control, enhanced minimization options, a footprint similarity scoring function, a symmetry-cor. root-mean-square deviation algorithm, a database filter, and docking forensic tools.  An important strategy during development involved use of three orthogonal metrics for assessment and validation: pose reprodn. over a large database of 1043 protein-ligand complexes (SB2012 test set), cross-docking to 24 drug-target protein families, and database enrichment using large active and decoy datasets (Directory of Useful Decoys [DUD]-E test set) for five important proteins including HIV protease and IGF-1R.  Relative to earlier versions, a key outcome of the work is a significant increase in pose reprodn. success in going from DOCK 4.0.2 (51.4%) → 5.4 (65.2%) → 6.7 (73.3%) as a result of significant decreases in failure arising from both sampling 24.1% → 13.6% → 9.1% and scoring 24.4% → 21.1% → 17.5%.  Companion cross-docking and enrichment studies with the new version highlight other strengths and remaining areas for improvement, esp. for systems contg. metal ions.  The source code for DOCK 6.7 is available for download and free for academic users at. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNgMYHDmTu7Vg90H21EOLACvtfcHk0ljVI49M1hR6TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D&md5=8b5fc6cc7533f975e1e740acb3688807</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23905%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDOCK%25206%253A%2520Impact%2520of%2520new%2520features%2520and%2520current%2520docking%2520performance%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D1132%26epage%3D1156%26doi%3D10.1002%2Fjcc.23905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allinger, N. L.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis. 130. MM2. A hydrocarbon force field utilizing V1 and V2 torsional terms</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">8127</span>– <span class="NLM_lpage">8134</span>, <span class="refDoi"> DOI: 10.1021/ja00467a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00467a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADyaE1cXmtVOm" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1977&pages=8127-8134&author=N.+L.+Allinger&title=Conformational+analysis.+130.+MM2.+A+hydrocarbon+force+field+utilizing+V1+and+V2+torsional+terms&doi=10.1021%2Fja00467a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis.  130.  MM2.  A hydrocarbon force field utilizing V1 and V2 torsional terms</span></div><div class="casAuthors">Allinger, Norman L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8127-34</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">An improved force field for mol.-mechanics calcns. of the structures and energies of hydrocarbons is presented.  The problem of simultaneously obtaining a sufficiently large gauche butane interaction energy while keeping the H atoms small enough for good structural predictions was solved with 1- and 2-fold rotational barriers.  For 42 selected diverse types of hydrocarbons, the std. deviation between the calcd. and exptl. heats of formation is 0.42 kcal/mol, compared with an av. reported exptl. error for the same group of compds. of 0.40 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7SvCTV-ylebVg90H21EOLACvtfcHk0ljVI49M1hR6TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXmtVOm&md5=17d96eafa46ab1f9dc6ea672e700aa2d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja00467a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00467a001%26sid%3Dliteratum%253Aachs%26aulast%3DAllinger%26aufirst%3DN.%2BL.%26atitle%3DConformational%2520analysis.%2520130.%2520MM2.%2520A%2520hydrocarbon%2520force%2520field%2520utilizing%2520V1%2520and%2520V2%2520torsional%2520terms%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1977%26volume%3D99%26spage%3D8127%26epage%3D8134%26doi%3D10.1021%2Fja00467a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ponder, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, F. M.</span></span> <span> </span><span class="NLM_article-title">An efficient newton-like method for molecular mechanics energy minimization of large molecules</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1016</span>– <span class="NLM_lpage">1024</span>, <span class="refDoi"> DOI: 10.1002/jcc.540080710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.540080710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADyaL1cXhtFGhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1987&pages=1016-1024&author=J.+W.+Ponderauthor=F.+M.+Richards&title=An+efficient+newton-like+method+for+molecular+mechanics+energy+minimization+of+large+molecules&doi=10.1002%2Fjcc.540080710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient Newton-like method for molecular mechanics energy minimization of large molecules</span></div><div class="casAuthors">Ponder, Jay W.; Richards, Frederic M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1016-24</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">Techniques from numerical anal. and crystallog. refinement are combined to produce a variant of the Truncated Newton nonlinear optimization procedure.  The new algorithm shows particular promise for potential energy minimization of large mol. systems.  Usual implementations of Newton's method require storage space proportional to the no. of atoms squared (i.e., O(N2)) and computer time of O(N3).  Suggested implementation of the Truncated Newton technique requires storage of less than O(N1.5) and CPU time of less than O(N2) for structures contg. several hundred to a few thousand atoms.  The algorithm exhibits quadratic convergence near the min. and is also very tolerant of poor initial structures.  A comparison with existing optimization procedures is detailed for cyclohexane, arachidonic acid, and the small protein crambin.  In particular, a structure for crambin (662 atoms) is refined to an RMS gradient of 3.6 × 10-6 kcal/mol/Å per atom on the MM2 potential energy surface.  Several suggestions are made which may lead to further improvement of the new method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlXX33qM5gJbVg90H21EOLACvtfcHk0ljZzIsZH2l9ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhtFGhsg%253D%253D&md5=aace9382993efb8da18b17f8869c1871</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540080710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540080710%26sid%3Dliteratum%253Aachs%26aulast%3DPonder%26aufirst%3DJ.%2BW.%26aulast%3DRichards%26aufirst%3DF.%2BM.%26atitle%3DAn%2520efficient%2520newton-like%2520method%2520for%2520molecular%2520mechanics%2520energy%2520minimization%2520of%2520large%2520molecules%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1987%26volume%3D8%26spage%3D1016%26epage%3D1024%26doi%3D10.1002%2Fjcc.540080710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maier, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasavajhala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">ff14SB: Improving the accuracy of protein side chain and backbone parameters from FF99SB</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3696</span>– <span class="NLM_lpage">3713</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3696-3713&author=J.+A.+Maierauthor=C.+Martinezauthor=K.+Kasavajhalaauthor=L.+Wickstromauthor=K.+E.+Hauserauthor=C.+Simmerling&title=ff14SB%3A+Improving+the+accuracy+of+protein+side+chain+and+backbone+parameters+from+FF99SB&doi=10.1021%2Facs.jctc.5b00255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</span></div><div class="casAuthors">Maier, James A.; Martinez, Carmenza; Kasavajhala, Koushik; Wickstrom, Lauren; Hauser, Kevin E.; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3696-3713</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics is powerful for its speed in atomistic simulations, but an accurate force field is required.  The Amber ff99SB force field improved protein secondary structure balance and dynamics from earlier force fields like ff99, but weaknesses in side chain rotamer and backbone secondary structure preferences have been identified.  Here, we performed a complete refit of all amino acid side chain dihedral parameters, which had been carried over from ff94.  The training set of conformations included multidimensional dihedral scans designed to improve transferability of the parameters.  Improvement in all amino acids was obtained as compared to ff99SB.  Parameters were also generated for alternate protonation states of ionizable side chains.  Av. errors in relative energies of pairs of conformations were under 1.0 kcal/mol as compared to QM, reduced 35% from ff99SB.  We also took the opportunity to make empirical adjustments to the protein backbone dihedral parameters as compared to ff99SB.  Multiple small adjustments of φ and ψ parameters were tested against NMR scalar coupling data and secondary structure content for short peptides.  The best results were obtained from a phys. motivated adjustment to the φ rotational profile that compensates for lack of ff99SB QM training data in the β-ppII transition region.  Together, these backbone and side chain modifications (hereafter called ff14SB) not only better reproduced their benchmarks, but also improved secondary structure content in small peptides and reprodn. of NMR χ1 scalar coupling measurements for proteins in soln.  We also discuss the Amber ff12SB parameter set, a preliminary version of ff14SB that includes most of its improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgntnN9ES97rVg90H21EOLACvtfcHk0ljZzIsZH2l9ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN&md5=7b803577b3b6912cc6750cfbd356596e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00255%26sid%3Dliteratum%253Aachs%26aulast%3DMaier%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DKasavajhala%26aufirst%3DK.%26aulast%3DWickstrom%26aufirst%3DL.%26aulast%3DHauser%26aufirst%3DK.%2BE.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3Dff14SB%253A%2520Improving%2520the%2520accuracy%2520of%2520protein%2520side%2520chain%2520and%2520backbone%2520parameters%2520from%2520FF99SB%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3696%26epage%3D3713%26doi%3D10.1021%2Facs.jctc.5b00255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0ljZzIsZH2l9ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span> <span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lh-XPKYUTzlDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marineau, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Biased multicomponent reactions to develop novel bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9019</span>– <span class="NLM_lpage">9027</span>, <span class="refDoi"> DOI: 10.1021/jm501120z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501120z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9019-9027&author=M.+R.+McKeownauthor=D.+L.+Shawauthor=H.+Fuauthor=S.+Liuauthor=X.+Xuauthor=J.+J.+Marineauauthor=Y.+Huangauthor=X.+Zhangauthor=D.+L.+Buckleyauthor=A.+Kadamauthor=Z.+Zhangauthor=S.+C.+Blacklowauthor=J.+Qiauthor=W.+Zhangauthor=J.+E.+Bradner&title=Biased+multicomponent+reactions+to+develop+novel+bromodomain+inhibitors&doi=10.1021%2Fjm501120z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Multicomponent Reactions to Develop Novel Bromodomain Inhibitors</span></div><div class="casAuthors">McKeown, Michael R.; Shaw, Daniel L.; Fu, Harry; Liu, Shuai; Xu, Xiang; Marineau, Jason J.; Huang, Yibo; Zhang, Xiaofeng; Buckley, Dennis L.; Kadam, Asha; Zhang, Zijuan; Blacklow, Stephen C.; Qi, Jun; Zhang, Wei; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9019-9027</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET bromodomain inhibition has contributed new insights into gene regulation and emerged as a promising therapeutic strategy in cancer.  Structural analogy of early methyl-triazolo BET inhibitors has prompted a need for structurally dissimilar ligands as probes of bromodomain function.  Using fluorous-tagged multicomponent reactions, we developed a focused chem. library of bromodomain inhibitors around a 3,5-dimethylisoxazole biasing element with micromolar biochem. IC50.  Iterative synthesis and biochem. assessment allowed optimization of novel BET bromodomain inhibitors based on an imidazo[1,2-a]pyrazine scaffold.  Lead compd. I (UMB-32) binds BRD4 with a Kd of 550 nM and 724 nM cellular potency in BRD4-dependent lines.  Addnl., compd. I shows potency against TAF1, a bromodomain-contg. transcription factor previously unapproached by discovery chem.  Compd. I was cocrystd. with BRD4, yielding a 1.56 Å resoln. crystal structure.  This research showcases new applications of fluorous and multicomponent chem. synthesis for the development of novel epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocWEf4LoDJTrVg90H21EOLACvtfcHk0lh-XPKYUTzlDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN&md5=a4e20101f5a0b547451814af73c0418f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm501120z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501120z%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DShaw%26aufirst%3DD.%2BL.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKadam%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBiased%2520multicomponent%2520reactions%2520to%2520develop%2520novel%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9019%26epage%3D9027%26doi%3D10.1021%2Fjm501120z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74','PDB','4O74'); return false;">PDB: 4O74</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU','PDB','2RKU'); return false;">PDB: 2RKU</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i30"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00765">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_48166"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00765">10.1021/acs.jmedchem.8b00765</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">BROMOscan and KINOMEscan data for <b>5</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00765/suppl_file/jm8b00765_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00765/suppl_file/jm8b00765_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00765/suppl_file/jm8b00765_si_001.pdf">jm8b00765_si_001.pdf (3.47 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00765/suppl_file/jm8b00765_si_002.csv">jm8b00765_si_002.csv (2.23 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are the following: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O74">4O74</a> for BI2536, BRD4-BD1; <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RKU">2RKU</a> for PLK1-BI2536.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00765%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00765" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996d89ad5a3c52","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
