
<html lang="en"     class="pb-page"  data-request-id="ce0c0390-f776-4ce3-b66c-2beb5ece4211"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2018.61.issue-20;requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.8b01248;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase–Bromodomain Inhibitor" /></meta><meta name="dc.Creator" content="Anand  Divakaran" /></meta><meta name="dc.Creator" content="Siva K.  Talluri" /></meta><meta name="dc.Creator" content="Alex M.  Ayoub" /></meta><meta name="dc.Creator" content="Neeraj K.  Mishra" /></meta><meta name="dc.Creator" content="Huarui  Cui" /></meta><meta name="dc.Creator" content="John C.  Widen" /></meta><meta name="dc.Creator" content="Norbert  Berndt" /></meta><meta name="dc.Creator" content="Jin-Yi  Zhu" /></meta><meta name="dc.Creator" content="Angela S.  Carlson" /></meta><meta name="dc.Creator" content="Joseph J.  Topczewski" /></meta><meta name="dc.Creator" content="Ernst K.  Schonbrunn" /></meta><meta name="dc.Creator" content="Daniel A.  Harki" /></meta><meta name="dc.Creator" content="William C. K.  Pomerantz" /></meta><meta name="dc.Description" content="As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis. When dysregulated..." /></meta><meta name="Description" content="As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis. When dysregulated..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 25, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01248" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01248" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01248" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01248" /></link>
        
    
    

<title>Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase–Bromodomain Inhibitor | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01248" /></meta><meta property="og:title" content="Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase–Bromodomain Inhibitor" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0013.jpeg" /></meta><meta property="og:description" content="As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis. When dysregulated, “reader” proteins become drivers of disease. In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging. We describe the &gt;55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase–bromodomain inhibitors. Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1). Cellular efficacy was demonstrated via reduction of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 production through potential synergistic inhibition of BRD4(1) and p38α. These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01248"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01248">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01248&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01248&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01248&amp;href=/doi/10.1021/acs.jmedchem.8b01248" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 9316-9334</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01225" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01249" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase–Bromodomain Inhibitor</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Anand Divakaran</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anand Divakaran</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, University of Minnesota, 2231 6th Street SE, Minneapolis, Minnesota 55455, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anand++Divakaran">Anand Divakaran</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1782-6234" title="Orcid link">http://orcid.org/0000-0002-1782-6234</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Siva K. Talluri</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Siva K. Talluri</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Siva+K.++Talluri">Siva K. Talluri</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alex M. Ayoub</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex M. Ayoub</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex+M.++Ayoub">Alex M. Ayoub</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Neeraj K. Mishra</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Neeraj K. Mishra</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Neeraj+K.++Mishra">Neeraj K. Mishra</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huarui Cui</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huarui Cui</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huarui++Cui">Huarui Cui</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John C. Widen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John C. Widen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, University of Minnesota, 2231 6th Street SE, Minneapolis, Minnesota 55455, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+C.++Widen">John C. Widen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Norbert Berndt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Norbert Berndt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery Department, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Norbert++Berndt">Norbert Berndt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jin-Yi Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jin-Yi Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery Department, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jin-Yi++Zhu">Jin-Yi Zhu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Angela S. Carlson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Angela S. Carlson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Angela+S.++Carlson">Angela S. Carlson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph J. Topczewski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph J. Topczewski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+J.++Topczewski">Joseph J. Topczewski</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9921-5102" title="Orcid link">http://orcid.org/0000-0002-9921-5102</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ernst K. Schonbrunn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ernst K. Schonbrunn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery Department, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ernst+K.++Schonbrunn">Ernst K. Schonbrunn</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3589-3510" title="Orcid link">http://orcid.org/0000-0002-3589-3510</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel A. Harki</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel A. Harki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, University of Minnesota, 2231 6th Street SE, Minneapolis, Minnesota 55455, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#5430353c35263f3d1421393a7a313021"><span class="__cf_email__" data-cfemail="4622272e27342d2f06332b2868232233">[email protected]</span></a> (D.A.H.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+A.++Harki">Daniel A. Harki</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5950-931X" title="Orcid link">http://orcid.org/0000-0001-5950-931X</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">William C. K. Pomerantz</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William C. K. Pomerantz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, University of Minnesota, 2231 6th Street SE, Minneapolis, Minnesota 55455, United States</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#8ff8ecffcffae2e1a1eaebfa"><span class="__cf_email__" data-cfemail="ec9b8f9cac998182c2898899">[email protected]</span></a> (W.C.K.P.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+C.+K.++Pomerantz">William C. K. Pomerantz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0163-4078" title="Orcid link">http://orcid.org/0000-0002-0163-4078</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01248&amp;href=/doi/10.1021%2Facs.jmedchem.8b01248" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 9316–9334</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 25, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 August 2018</li><li><span class="item_label"><b>Published</b> online</span>25 September 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 October 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01248" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01248</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9316%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAnand%2BDivakaran%252C%2BSiva%2BK.%2BTalluri%252C%2BAlex%2BM.%2BAyoub%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D20%26contentID%3Dacs.jmedchem.8b01248%26title%3DMolecular%2BBasis%2Bfor%2Bthe%2BN-Terminal%2BBromodomain-and-Extra-Terminal-Family%2BSelectivity%2Bof%2Ba%2BDual%2BKinase%25E2%2580%2593Bromodomain%2BInhibitor%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9334%26publicationDate%3DOctober%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01248"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2912</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">34</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01248" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase–Bromodomain Inhibitor&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Anand&quot;,&quot;last_name&quot;:&quot;Divakaran&quot;},{&quot;first_name&quot;:&quot;Siva&quot;,&quot;last_name&quot;:&quot;K. Talluri&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;M. Ayoub&quot;},{&quot;first_name&quot;:&quot;Neeraj&quot;,&quot;last_name&quot;:&quot;K. Mishra&quot;},{&quot;first_name&quot;:&quot;Huarui&quot;,&quot;last_name&quot;:&quot;Cui&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;C. Widen&quot;},{&quot;first_name&quot;:&quot;Norbert&quot;,&quot;last_name&quot;:&quot;Berndt&quot;},{&quot;first_name&quot;:&quot;Jin-Yi&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Angela&quot;,&quot;last_name&quot;:&quot;S. Carlson&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;J. Topczewski&quot;},{&quot;first_name&quot;:&quot;Ernst&quot;,&quot;last_name&quot;:&quot;K. Schonbrunn&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;A. Harki&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;C. K. Pomerantz&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;25&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;9316-9334&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01248&quot;},&quot;abstract&quot;:&quot;As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis. When dysregulated, “reader” proteins become drivers of disease. In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging. We describe the &gt;55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase–bromodomain inhibitors. Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1). Cellular efficacy was demonstrated via reduction of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 production through potential synergistic inhibition of BRD4(1) and p38α. These dual inhibitors provide a new scaffold for domain-sele&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01248&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01248" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01248&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01248" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01248&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01248" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01248&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01248&amp;href=/doi/10.1021/acs.jmedchem.8b01248" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01248" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01248" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DDivakaran%26date%3D2018%26atitle%3DMolecular%2BBasis%2Bfor%2Bthe%2BN-Terminal%2BBromodomain-and-Extra-Terminal-Family%2BSelectivity%2Bof%2Ba%2BDual%2BKinase%25E2%2580%2593Bromodomain%2BInhibitor%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D20%26spage%3D9316%26epage%3D9334%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/jmcmar.2018.61.issue-20/20181025/jmcmar.2018.61.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis. When dysregulated, “reader” proteins become drivers of disease. In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging. We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase–bromodomain inhibitors. Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1). Cellular efficacy was demonstrated via reduction of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 production through potential synergistic inhibition of BRD4(1) and p38α. These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81531" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81531" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">How our genetic information is manipulated and ultimately expressed as a heritable phenotype is a central question in the field of epigenetics. N-terminal modifications on conserved histone proteins are key regulators of transcription.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> As such, proteins that interpret these complex modifications through molecular recognition are known to be essential for maintaining cellular homeostasis, and when dysregulated, they become drivers of disease.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Bromodomains are a subset of epigenetic “reader” or effector-domain-containing proteins, which function via binding to distinct lysine-acylation states, most commonly N-ε-acetylated lysines (K<sub>ac</sub>) on histones.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> This similar mode of molecular recognition for all 61 human bromodomains presents a significant hurdle for finding selective inhibitors as both therapeutic agents and chemical probes.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">One family of bromodomains with clinical relevance are the bromodomain-and-extra-terminal (BET) proteins BRD2, BRD3, BRD4, and testis-specific BRDT, each of which contains a pair of bromodomains. Recent studies have shown the divergent roles of these domains for targeting both acetylated histones and acetylated transcription factors through the N- and C-terminal bromodomains (D1 and D2, respectively).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Prominent roles of the BRD4 BET bromodomains include regulation of c-Myc expression at superenhancers<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and enhancement of NF-κB-mediated transcription through interactions with acetylated K310 of the κB RelA subunit.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">Because of the biological significance of bromodomain functions in cellular proliferation and inflammatory pathways, over 20 clinical trials are underway for studying the effects of BET-family inhibition as anticancer and anti-inflammatory therapies. Because of the high sequence identity among BET proteins, most reported inhibitors act as pan-inhibitors of BET bromodomains, such as (+)-JQ1 (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), and do not discern the functions of individual bromodomains.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Ouyang et al. reported FL-411 (<b>2</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which is an exception to the norm in being selective for BRD4 over other BET proteins.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Nevertheless, the potential off-target effects, dose-limiting thrombocytopenia, and concerns over latent-viral reactivation associated with pan-BET inhibition highlight the need for novel scaffolds with improved selectivity and potency.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0001.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Reported BET bromodomain inhibitors and a new 1,4,5-trisubstituted imidazole, <b>V</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In lieu of isoform-specific inhibitors, domain selectivity among the BET family of proteins has been explored. D2-selective molecules RVX-208 (<b>3</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) and RVX-297 have allowed the study of D2-dependent processes, but they have reduced efficacy mitigating BET-related transcription in comparison with that of pan-BET inhibition.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Alternatively, specific deletion of only BRDT(1) in mice was sufficient to impair spermatogenesis.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In the case of BRD4, fluorescence-recovery-after-photobleaching experiments identified similar levels of BRD4 displacement from chromatin between a D1-defective mutant and (+)-JQ1 (<b>1</b>)-treated cells, whereas chromatin binding by a D2-defective mutant was less affected.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In relation to NF-κB signaling, interactions of BRD4(1) are likely more relevant for targeting transcriptional activity.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Although Olinone and MS-436 (<b>4</b> and <b>5</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) have been reported to possess some level of D1 selectivity,<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> BET bromodomain research continues to be dominated by pan-BET inhibitors. Despite being potent BRD4(1) inhibitors, <b>5</b> and structural analogue MS-611 are not selective for D1 over D2 in other BET proteins. In contrast, D1-selective molecule <b>4</b> is a modest inhibitor of BRD4(1) (<i>K</i><sub>d</sub> = 3.4 μM) but represents the highest level of selectivity. Alternative chemical-biology strategies like the “bump-and-hole” approach<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> have considerably advanced our understanding of individual BET bromodomain and -protein function. However, improved D1-selective inhibitors are still required to further dissect the relevant interactions of BRD4 with chromatin and acetylated transcription factors. The development of such small molecules may have significant implications for studying the roles of relevant BET proteins in transcription and the treatment of disease.</div><div class="NLM_p last">In this report, we describe dual kinase–bromodomain inhibitors with the strongest affinity for BRD4(1) among BET bromodomains. Although other BET bromodomain inhibitors have been reported in the literature, a notable feature of this molecule is its selectivity for the first bromodomains of BET proteins with simultaneous kinase-inhibitory activity against mitogen-activated-protein (MAP) kinase p38α. The atomic-level insights from structural analysis and structure–activity relationships will inform the future development of BET-D1-selective inhibitors, and optimization of this dual inhibitor should afford a useful tool compound to test synergistic effects in treating inflammation and cancer.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72322" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72322" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The 1,4,5-trisubstituted imidazole, SB-284851-BT (<b>I</b>), and related analogues are known p38α inhibitors and were recently discovered to bind to BET bromodomains in three separate reports using diverse libraries, including the published kinase-inhibitor-set library from GlaxoSmithKline.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> Our study tested for BET bromodomain selectivity by using a <sup>19</sup>F NMR assay, which simultaneously assessed binding to BRD4(1) and the bromodomain of BPTF.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a><b>I</b> was selective for BRD4(1) over BPTF in the assay and was further validated as a ligand in a thermal-stability assay, through differential-scanning fluorimetry (DSF), and in a fluorescence-anisotropy (FA) assay, displacing fluorescently labeled pan-BET ligand BI-6727, referred to here as BI-BODIPY. Although our previous study identified selectivity for the BET family, we did not further evaluate selectivity within the BET family of bromodomains.</div><div class="NLM_p">We first turned our attention to two BET bromodomains, BRDT(1) and BRD4(1). On the basis of significant differences in affinity arising from methyl-substitution patterns on the phenyl ring of <b>I</b> for BRD4(1),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> we synthesized several new analogues and determined their affinities to these two domains. The synthesis for <b>I</b> has already been established,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and we took advantage of a late-stage intermediate (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) for analogue synthesis using a nucleophilic aromatic-substitution reaction with various phenolic and anilinic derivatives (<a class="ref internalNav" href="#sch2" aria-label="Schemes 2">Schemes 2</a>–<named-content content-type="anchor" rid="sch3" type="simple"></named-content><a class="ref internalNav" href="#sch4" aria-label="4">4</a>), of which several are described here (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Using FA to quantify the activities of these newly synthesized compounds against both BRDT(1) and BRD4(1), we found that compound <b>I</b> binds to both domains. The observed affinity for BRD4(1) was 3-fold weaker than the previously reported data of the compound used directly from the library stock, indicating a possible discrepancy in concentration or purity. Consistent with our prior report,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> the para-substituted analogue, <b>II</b>, bound substantially more weakly. However, a 3,4-dimethyl-substituted analogue with an O-substituted pyrimidine, compound <b>III</b>, bound with increased affinity (IC<sub>50</sub> = 2.9 ± 0.2 μM). Finally, when the pyrimidine oxygen was changed to a nitrogen, we observed an increase in potency, with IC<sub>50</sub> values of 1.3 ± 0.2 and 1.7 ± 0.4 μM for the 3,4- and 3,5-dimethyl analogues, compounds <b>IV</b> and <b>V</b>, respectively, against BRD4(1). Molecule <b>VI</b>, with a 3,5-difluoro-substituted aromatic ring (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) versus methyl groups did not interact with high affinity (IC<sub>50</sub> > 250 μM), supporting the need for an electron-rich ring. A direct-binding titration of compounds <b>IV</b> and <b>V</b> by isothermal-titration calorimetry (ITC) determined entropically favorable binding events, leading to <i>K</i><sub>d</sub> values of 1.8 and 1.2 μM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S7</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0002.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Competitive-inhibition experiments via fluorescence anisotropy of SB-284851-BT (<b>I</b>) and analogues against BET bromodomains indicating selectivity for BET D1s. (A) IC<sub>50</sub> values of analogues (means ± SEM) determined by competitive-fluorescence-anisotropy assays. Asterisks (*) indicate incomplete inhibition observed; ND indicates not determined. (B) Competition-binding isotherms of compounds <b>IV</b> and <b>V</b> against the first and second bromodomains of BRD4 and BRDT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Common Intermediate <b>17</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Procedure adapted from Boehm et al. and Adams et al.<a onclick="showRef(event, 'ref27 ref51'); return false;" href="javascript:void(0);" class="ref ref27 ref51">(27,51)</a></p></p></figure><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>I</b>, <b>II</b>, and <b>III</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0008.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>IV</b> and <b>V</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0009.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>VI</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To evaluate if these molecules would function as pan-BET inhibitors on the basis of binding to both BRDT(1) and BRD4(1), we next tested D1 versus D2 selectivity, as our fluorescent tracer binds to D2s of BRDT and BRD4 with high affinity (<i>K</i><sub>d</sub> = 270 and 183 nM). Surprisingly, in all cases, the analogues tested were unable to fully displace BI-BODIPY from either D2 construct (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Alternatively, pan-BET inhibitor BI2536<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> displaced the fluorescent tracer from the domains of both BRDT and BRD4. A D2-selective inhibitor, RVX-208, was tested in a similar fashion and validated the experimental outcomes seen with our protein constructs (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figures S2 and S3</a>). Additionally, a dual bromodomain construct of BRD4 was tested in our FA assay, and only partial displacement was observed with compound <b>V</b>, consistent with it only fully displacing BI-BODIPY from the first bromodomain (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S4</a>). Molecules <b>I</b>–<b>III</b> also maintained their selectivity for D1 over D2 when tested against BRDT(2), indicating that neither the substitution pattern on the aromatic ring nor the O to N substitution on the pyrimidine ring was responsible for the observed selectivity.</div><div class="NLM_p">The observed preference toward BRD4(1) led us to evaluate selectivity against the two other BET proteins, BRD2 and BRD3, using an orthogonal assay. Compound <b>V</b> was assayed commercially against seven of the eight BET bromodomains in an ALPHAScreen format.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Although BRDT(2) was not available for testing by ALPHAScreen, D1 over D2 selectivity was maintained for compound <b>V</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S8</a>). Additionally, compound <b>V</b> displayed 6- and 16-fold selectivity for BRD4 over ubiquitously expressed BET proteins BRD2 and BRD3, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S8</a>). Together, these results support a valuable scaffold for developing D1-selective inhibitors of BET bromodomains, albeit with the highest affinity for BRD4(1) and BRDT(1).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. ALPHAScreen Analysis of Compounds <b>V</b>, <b>VII</b>, <b>VIII</b>, and <b>IX</b> against 7 of 8 BET Bromodomains</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0012.gif" alt="" id="fx1" /></img><div></div></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Structural Analysis of Binding</h3><div class="NLM_p">Because of the paucity of D1-selective inhibitors and the generality of these results within the trisubstituted-imidazole scaffold, we sought to structurally characterize their interactions. Protein-observed fluorine NMR (PrOF NMR) is an emerging method for ligand discovery that has been used to characterize the binding interfaces of small molecules with bromodomains<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> as well as to quantify the affinities of weakly binding molecules via chemical-shift-perturbation experiments over a range of ligand concentrations.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> PrOF NMR was used to characterize the binding interactions of compound <b>V</b> with both BRD4(1) and BRDT(1) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) and to further test selectivity against additional bromodomains at higher small-molecule concentrations than were tolerated in earlier experiments. Our focus in these experiments was primarily on the WPF-shelf tryptophan residue adjacent to the K<sub>ac</sub>-binding site, W81 for BRD4(1) and W50 for BRDT(1), to see if an unusual effect on resonance perturbation was observed relative to those of our prior reports.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> With both BRD4(1) and BRDT(1) and using compound <b>V</b>, an intermediate to slow exchange perturbation of the WPF-shelf resonance was observed, which reached the fully bound state at approximately 1 equiv. of small molecule. Such NMR behavior is indicative of a residence time on the protein that is sufficiently long on the NMR time scale to partially resolve both the bound and unbound states and is consistent with the binding site and single-digit-micromolar-<i>K</i><sub>d</sub> affinities exhibited by compounds <b>IV</b> and <b>V</b> for these domains.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> W44 of BRDT(1) is outside the binding site and was also minimally perturbed, indicating a conformational change of the protein. However, these data did not support binding to a new binding site and were similar to observations in our prior reports.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> We further tested non-BET bromodomains of BPTF and a yeast GCN5-like malarial protein that also contains a WPF-shelf. In these cases, we did not detect binding to either protein, consistent with selectivity for the BET family of bromodomains (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S9</a>).</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0003.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Confirmation of compound <b>V</b> binding BRD4(1) and BRDT(1) and possible structural basis for the D1 over D2 selectivity of the inhibitors. (A) PrOF NMR titrations with bromodomains of BRD4 and BRDT. Samples contain 50 μM <sup>19</sup>F-labeled BRD4(1) or BRDT(1) with increasing concentrations of ligand <b>V</b>. (B) Electron-density maps of ligands (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub>, 1σ) from X-ray structures of BRD4(1) liganded with <b>IV</b> (yellow, 1.74 Å, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH7">6MH7</a>) and <b>V</b> (orange, 1.60 Å, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH1">6MH1</a>). (C) Superposition of compounds <b>IV</b> and <b>V</b> in the K<sub>ac</sub> site of BRD4(1) revealing subtle conformational changes of the dimethylphenyl moieties nested in the WPF shelf. (D) Unliganded state of BRD4(1) (gray, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IOR">4IOR</a>), with the K<sub>ac</sub> site around Asn140 in a relaxed state, and BRD4(1) bound to inhibitor <b>V</b> (yellow ligand and orange structure), with Asn140 giving way (Δ = 0.8 Å) as a result of steric hindrance with the imidazole moiety of the inhibitor (red dotted line) and positioning itself for optimal H-bonding interaction. As a result, the adjacent Lys141 undergoes a large conformational change (Δ = 2.2 Å). (E) Unliganded BRD4(2) (green, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OUO">2OUO</a>), with the region around Asn433 being highly similar to that of BRD4(1) (gray) except for the presence of Pro434 instead of lysine. The geometric constraints of the Asn–Pro–Pro sequence in BRD4(2) likely renders the peptide more rigid and less compatible with the inhibitor-induced conformational changes observed in BRD4(1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We turned to X-ray crystallography to obtain higher-resolution structural information to understand the mode of molecular recognition and to rationalize the domain selectivity. Trisubstituted imidazoles SB-251527 and SB-284847-BT have previously been crystallized with BRD4(1) (PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7F">4O7F</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7B">4O7B</a>, respectively).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In addition, we crystallized compounds <b>IV</b> and <b>V</b>, with BRD4(1) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). All four compounds engage the protein in a similar fashion, with the imidazole making direct hydrogen-bonding interactions with N140, which is displaced by 0.8 Å relative to the apo structure, along with a displacement of the Cα of the adjacent K141 by 2.2 Å (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D,E and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">S11</a>). However, unlike pan-BET inhibitors, these molecules failed to hydrogen bond to Y97 via a bridging-structured water. The O–N substitution pattern did alter the orientation of the aromatic ring slightly, which may favor edge-to-face π–π interactions with W81 of BRD4(1) in a lipophilic region adjacent to the WPF shelf (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S12</a>), impacting affinity but not selectivity. The methyl groups were necessary for binding affinity, as a 3,5-difluoro-substituted analogue, <b>VI</b>, only bound weakly to both bromodomains of BRD4. Overlay of a BRD4(2) crystal structure shows potential steric incompatibilities at multiple regions of the small molecule, including a steric clash of the C2 imidazole portions of the rings of compounds <b>IV</b> and <b>V</b> with N433 in the binding site, which may not be as conformationally flexible as the BRD4(1) N140 because of the adjacent P434. These incompatibilities may preclude a high-affinity interaction with BRD4(2); however, such a steric effect could not be directly proven from our structures. Together, <sup>19</sup>F NMR and X-ray structural analysis could be used to rationalize binding interactions but were not sufficient to describe selectivity.</div><div class="NLM_p last">As an alternative possibility for selectivity, we considered the sequence similarity between BET D1 and D2. Of the three nonconserved amino acids between the binding sites, we hypothesized the proximity of the piperdyl group to D144 in our crystal structures was potentially responsible for the observed selectivity trends. This BET-D1-conserved aspartic acid is replaced by a histidine (H437 of BRD4) in BET D2s. At physiological pH, H437 will be weakly protonated and may result in potential electrostatic repulsions in addition to steric effects that prevent binding to D2. To determine whether the origins of D1 versus D2 selectivity are due to H437 in D2, we expressed a H437D-mutant BRD4(2) construct to restore binding of compound <b>V</b>. For this experiment, we used a new fluorescent tracer for the FA assay based on a fluorescein-labeled (+)-JQ1 molecule. (+)-JQ1 bound to H437D BRD4(2) with comparable affinity to the WT D2 protein in direct-binding assays (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S5</a>; <i>K</i><sub>d</sub> BRD4(2) = 120 nM, H437D BRD4(2) = 200 nM) and with a slightly attenuated IC<sub>50</sub> in competition assays (IC<sub>50</sub> BRD4(2) = 490 nM, H437D BRD4(2) = 590 nM). Binding of <b>V</b> to H437D BRD4(2), however, was not observed (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S6</a>), suggesting the observed selectivity trends are not entirely attributable to H437 of BRD4(2).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Displacement of Structured Waters</h3><div class="NLM_p">Recent computational studies suggested a second possible explanation for the observed selectivity on the basis of differential affinities for the four structured waters within the BET bromodomain binding sites. In these reports, the stabilities of bromodomain water networks have identified the water molecules most likely to be displaced in BRD4(1) and BRDT(1) among the other BET bromodomains,<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> whose water molecules are more tightly bound to the protein. Such an effect would also explain the observed selectivity of compound <b>V</b> for BRD4(1) and BRDT(1) over the other BETs measured in the ALPHAScreen assay. We compared the cocrystal structures of compound <b>V</b> with BRD4(1) and of a DMSO molecule bound to the same protein (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Whereas in the DMSO costructure, six water molecules are present, including the four conserved structured waters, three water molecules are displaced in the cocrystal structure with compound <b>V</b>, and the water network is reorganized by the fluorophenyl group of compound <b>V</b>. Similar displacements were observed in the cocrystal structure with compound <b>IV</b> (not shown). Displacement of water in the binding site is also consistent with the positive entropy of binding in ITC experiments with compounds <b>IV</b> and <b>V</b> (5.0 and 4.2 cal/mol/°C, respectively). Importantly, these modes of binding and selectivity are not observed in BET-D1-selective molecules <b>4</b> (olinone) and <b>5</b> (MS-436); however, a remarkably similar binding mode has been observed with a selective TRIM24-bromodomain inhibitor.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> These results may in part give rise to our observed selectivity, but the generality of such a finding will be useful to study in future investigations.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0016.jpeg" id="GRAPHIC-d7e904-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Displacement of structured waters from BRD4(1). Crystal structure of BRD4(1) with DMSO (left, yellow, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IOR">4IOR</a>) and compound <b>V</b> (right, yellow, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH1">6MH1</a>), with structurally conserved water molecules shown in cyan.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further evaluate the displacement of structured waters by the fluorophenyl group and additional effects around the imidazole ring, which removes steric effects from a C–H at position C2, we synthesized three new molecules (<b>VII</b>, <b>VIII</b>, and <b>XI</b>) based on a 1,4,5-trisubsititued triazole scaffold (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). These molecules either maintain the fluorophenyl group for a direct comparison with <b>V</b> or replace this phenyl group with a smaller hydroxymethyl or methyl group to maintain the structured waters. According to PrOF NMR, compound <b>VII</b> bound BRD4(1) in a manner similar to that of compound <b>V</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S10</a>), but because of solubility challenges at high concentrations and the low sensitivity of FA, ALPHAScreen was used to test for BET selectivity. Replacement of the imidazole ring with a triazole in compound <b>VII</b> resulted in a moderate loss of potency to BRD4(1) (IC<sub>50</sub> = 1.8 μM for <b>V</b> vs 27 μM for <b>VII</b>), likely because of the weakened hydrogen bonding with the conserved N140 of BRD4, and a significant loss of selectivity (27 μM for D1 vs 75 μM for D2). Replacement of the fluorophenyl group with a hydroxymethyl group (<b>VIII</b>) resulted in further reduced binding (IC<sub>50</sub> > 100 μM), possibly because of the hydrophobic nature of the binding pocket. Finally, in contrast, truncation of the fluorophenyl group to a methyl in compound <b>IX</b> enhanced binding 20-fold relative to that of <b>VII</b> (IC<sub>50</sub> = 1.3 μM for <b>IX</b> vs 27 μM for <b>VII</b>). The large gain in potency indicates an energetic penalty must be overcome for displacing the structured waters with a large aromatic group. In the case of compound <b>IX</b>, which lacks a large aromatic group displacing structured waters, an inversion of BRD4-domain selectivity favoring BRD4(2) (IC<sub>50</sub> = 0.56 μM) and overall pan-BET inhibition were observed.</div><figure id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0010.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Triazoles <b>VII</b>, <b>VIII</b>, and <b>IX</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 100 <sup>ο</sup>C, 3 h for the synthesis of <b>34</b> and <b>36</b>; (b) NaHMDS, −78 <sup>ο</sup>C, 0.5 h for the synthesis of <b>35</b>.</p></p></figure><div class="NLM_p last">From these data, we conclude that D1 over D2 selectivity cannot be attributed to the methyl-substitution pattern of the pyrimidine-substituted aromatic ring, the phenoxy-pyrimidine to anilino-pyrimidine substitution, or even steric incompatibilities with the piperdyl group. Rather, the observed selectivity is due to a combination of effects including the imidazole ring itself and from the fluorophenyl ring displacing structured waters.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Bromodomain and Kinase-Selectivity Profiling</h3><div class="NLM_p">To more broadly test the bromodomain selectivity of compound <b>V</b>, we used a commercial-assay platform to characterize the activity against 32 of the 61 bromodomains provided by DiscoverX. On the basis of a measured <i>K</i><sub>d</sub> of 1.2 μM, we tested the selectivity of <b>V</b> at 10 μM (approximately 10-fold above the <i>K</i><sub>d</sub>). Consistent with our FA and ALPHAScreen results, the two most significant interactions with <b>V</b> of the eight BET bromodomains were those of BRD4(1) and BRDT(1), with 86 and 50% inhibition at that concentration, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Table S3</a>). The BET D2s were the most unaffected (7–27% inhibition) relative to D1s. Overall, only bromodomains SMARCA2, PCAF, p300, and SMARCA4 had inhibitory activity between 40 and 50%. These results demonstrate the selectivity of compound <b>V</b> toward BET D1s and, given the modular synthesis of the 1,4,5-trisubstituted-imidazole scaffold, represent a useful starting point for domain-selective-inhibitor development.</div><div class="NLM_p last">As compound <b>I</b> is a reported MAP-kinase inhibitor, we were interested in verifying the selectivity of these analogues against other kinases. Similar analogues of <b>I</b>, either maintaining pyrimidyl-aniline substitutions but lacking dimethyl groups or containing methylated piperidinyl groups, have been previously reported and are highly potent and selective for p38α and β at 100 nM.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> As such, we expected a similar affinity for p38α and a similar kinase-selectivity profile. Indeed, the <i>K</i><sub>d</sub> of analogue <b>V</b> against p38α was determined to be 0.47 nM (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S13</a>). Because of the weaker D1 potency, we tested for selectivity among kinases at a concentration of 1 μM using the DiscoverX platform. In this case, we measured high inhibitory activity against p38α and β (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Table S4</a>). However, additional kinases were inhibited at this concentration, giving a selectivity score, S(1), of 0.037, similar to that of the p38α chemical probe SB203580 (0.024 at 100 nM). Because of the available crystal-structure data of related compounds bound to p38α and the wealth of data available on this inhibitor class, kinase selectivity may be further improved and potentially even abolished through rational design.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Cellular Evaluation of Lead Compounds</h3><div class="NLM_p">Given the BET-D1 selectivity exhibited by compounds <b>IV</b> and <b>V</b>, we proceeded by testing these molecules in cells. First, we verified target engagement in the native chemical environment by using a cellular thermal-shift assay (CETSA) in A549 cells, where the thermal stability of the target protein is altered upon binding to small molecules.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Here, an isothermal-denaturation experiment indicated compound <b>V</b> stabilized BRD4 and p38α in a dose-dependent manner at concentrations above 3 μM and 1 nM, respectively. Protein-melt temperatures correlated well with previous CETSA experiments performed on BRD4 and p38α.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> Importantly, as this experiment does not measure cellular outcomes downstream of target inhibition, these results confirm unambiguously that compound <b>V</b> enters cells and maintains interactions with BRD4 and p38α in a cellular context (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).</div><figure id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0004.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cellular evaluation of lead compounds confirming target engagement. (A) Cellular thermal-shift assay with compound <b>V</b> indicating thermal stabilization of BRD4 and p38α in A549 cells, with β-actin used as a loading control. (B) Western blots of expressed c-Myc levels in MM.1S cells, with GAPDH used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">After verifying protein-target engagement, we proceeded to test the efficacy of our D1-selective inhibitors in MM.1S cells by measuring reduction of c-Myc, an oncogene highly sensitive to BET inhibition, in several leukemia cell lines. Similar to (+)-JQ1, used as a positive control, compounds <b>IV</b> and <b>V</b> inhibited expression of the c-Myc oncoprotein in MM.1S cells at concentrations approaching their respective BRD4(1) <i>K</i><sub>d</sub> values, as determined by Western blot (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). These results also mirrored cell-viability data in the cell line studied (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S14</a>). Together, these are characteristic hallmarks of BET inhibitors, and suggest effective cellular bromodomain-inhibitory activity. To verify inhibition of the p38α kinase, we determined the phosphorylation states of its downstream phosphorylation target, MSK1. Here we observed a decrease in levels of MSK1 phosphorylation at T581 upon treatment with compounds <b>IV</b> and <b>V</b>, although at levels significantly higher than their affinity for p38α. This effect was not observed with (+)-JQ1 treatment (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S16</a>). Together, these results are consistent with BET-D1 inhibition using our 1,4,5-trisubstituted-imidazole scaffold being sufficient to regulate transcription in cells in a BRD4-specific manner.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Inhibition of Inflammation</h3><div class="NLM_p">The role BRD4 plays in coupling NF-κB signaling to gene transcription via K<sub>ac</sub> recognition has been disputed. In certain systems, transcriptional activation of RelA-target genes has been observed independent of BRD4,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> whereas Chen and co-workers have reported on BRD4 maintaining activity of NF-κB signaling via recognition of K310 acetylation at the RelA subunit.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In the case of NF-κB-dependent activity, it remains unclear whether D1 or D2 plays the dominant role in driving transcription; the two bromodomains of BRD4 maintain interactions with chromatin, and the presence of both domains enhances transcription.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In contrast, whereas Zou and co-workers showed a 10-fold stronger interaction of RelA with D2 by a fluorescence-competition assay,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Jung et al. did not detect any binding interactions in a TR-FRET assay,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> leading to uncertainty of a direct interaction between RelA and BRD4. Together, these data highlight a need for confirmation of the relevant interactions of each bromodomain of BRD4.</div><div class="NLM_p">To further assess cellular efficacy of our D1-selective inhibitors, we employed a luminescence-based NF-κB reporter assay in A549-NF-κB-luc cells. Compound <b>V</b> (51% inhibition at 10 μM) produced the most effective knockdown of TNFα-induced transcription in this assay, compared to inhibition of BRD4(2) using RVX-208 (25% inhibition at 10 μM, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). As other p38 MAPK inhibitors (SB-203580 and VX-702) showed limited inhibitory effects in this assay (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S17</a>), and compounds <b>IV</b> and <b>V</b> exhibited minimal cytotoxicity toward A549 cells up to concentrations of 25 and 10 μM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S14</a>), these experiments suggest BRD4(1) inhibition may be crucial to NF-κB signaling in A549 cells and support earlier results using BET-selective inhibitor MS-611.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><figure id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0005.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of NF-κB signaling and an NF-κB target gene. (A) NF-κB–luciferase-reporter assay with A549 cells with 20, 5, and 1 μM (+)-JQ1 and 10, 5, and 1 μM <b>IV</b>, <b>V</b>, and RVX-208. (B) Sandwich IL-8 ELISA with 10, 1, and 0.1 μM (+)-JQ1, <b>V</b>, SB-203580, and RVX-208. Data reported are means ± SEM values (<i>n</i> = 3 biological replicates) normalized to compound-induced (I) control values. All samples except the not-induced (NI) control were induced with 10 ng/mL TNFα. ANOVA comparisons to the three RVX-208 treatments; ns, not statistically significant; ***<i>p</i> < 0.001; ****<i>p</i> < 0.0001. See <a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S3</a> for compound-mediated-cellular-cytotoxicity data.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Inhibition of BET bromodomains has been shown to decrease production of cytokines; however, the roles of individual BET bromodomains are also yet to be deconvoluted. To assess the effect of compound <b>V</b> on inflammation at the protein level, we quantitated the suppression of cytokine production using an immunosorbent assay (ELISA). We chose to measure the concentrations of secreted cytokine IL-8, because its role as an inflammatory stimulus for neutrophil recruitment has been well characterized.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> At concentrations of 1 μM in this assay, BRD4(1) inhibition using compound <b>V</b> reduced levels of cytokine production (62% inhibition at 1 μM) more effectively than BRD4(2) inhibition by RVX-208 (40% inhibition at 1 μM). Moreover, the effects of compound <b>V</b> were comparable to those of pan-BET inhibition by (+)-JQ1 (63% inhibition at 1 μM, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). This increased potency of <b>V</b> is likely due to the additional inhibition of p38α. However, other off-targets cannot be ruled out at this point. Interestingly, complete inhibition of cytokine production was not observed with any of these molecules, whereas an irreversible NF-κB inhibitor, parthenolide, decreased IL-8 production to baseline at 20 μM.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> These results, coupled with confirmation of target occupancy by CETSA, establish a potential mode of synergy via dual targeting of p38α and BET bromodomains.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83376" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83376" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Through a late-stage structure–activity-relationship (SAR) investigation of 1,4,5-trisubstituted imidazoles, we identified a small series of BET inhibitors. Biophysical characterization of lead compounds against the entire BET family revealed an unexpected but high selectivity toward BET D1s over D2s, with the highest affinity for BRD4(1). Our experimental data ruled out selectivity toward N-terminal BET bromodomains because of steric hindrance by a histidine present in C-terminal BET bromodomains, where the corresponding position is occupied by aspartic acid in BET D1s (D144 in the case of BRD4). Although H437 appears within 3 Å of the piperidyl group in overlays of BRD4(2) crystal structures (e.g., PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEM">2YEM</a>), this amino acid adopts alternate conformations that may accommodate the ligand (e.g., PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEU">5UEU</a>).</div><div class="NLM_p">Rather, our data with compounds <b>VII</b> to <b>IX</b> indicate selectivity was largely due to the fluorophenyl moiety and the imidazole ring itself. Structural analysis indicates a binding mode of suitably substituted imidazole rings similar to that of acetyl groups on acetylated histones, with hydrogen bonding to a conserved asparagine residue (N140 in the case of BRD4(1)) by the imidazole-ring nitrogen, although N140 and K141 are slightly displaced to accommodate the imidazole ring. Additionally, the fluorophenyl group sits much deeper in the binding pocket than those of previously reported BET inhibitors and is able to displace three water molecules present in the cocrystal structure of BRD4(1) with DMSO, in which one of these is a structurally conserved water. Our data and previous computational studies on the stability of bromodomain water networks<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> lead us to believe our observed selectivity profile also arises in part from the displacement of a conserved water from BRD4(1) and BRDT(1) but not from other BET bromodomains.</div><div class="NLM_p">Cellular data indicate our compounds are highly cell-permeable and capable of engaging BRD4 in various cell lines. However, whereas effects on the MM.1S cell line appear to be largely BRD4-driven because of the c-Myc sensitivity of this cell line, the A549 lung-cancer cell line appears to be increasingly sensitive to both p38α and BRD4 inhibition, on the basis of decreases in cytokine secretion upon treatment with our molecules. Although such cooperative effects can only be cautiously assigned because of additional off-target kinase engagement, our preliminary evaluations suggest selective inhibition of BRD4(1) may be more efficacious than BRD4(2) inhibition. These cellular data confirm previous chromatin-binding studies and work targeting inhibition of BET bromodomains in inflammation;<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> however, bromodomain inhibition alone failed to completely inhibit signaling pathways in the system studied here.</div><div class="NLM_p last">Although kinase activity was secondary to our aim of developing selective BET inhibitors, we endeavored to fully characterize the kinase activity of our lead molecules. Previous reports on substituted imidazoles and pyrazoles of the “DFG-out” class of hinge-binding kinase inhibitors suggest that modest additions to the periphery of the scaffold, such as through N-alkylation, may improve selectivity.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> Additionally, by replacing the <i>p</i>-fluorophenyl moiety from similar trisubstituted imidazoles with ethyl or chloro groups, Gallagher et al. observed a loss of p38α-kinase inhibition.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> We thus envision the selectivity profile of these molecules may be readily improved by further analogue synthesis and SAR studies.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04520" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Through a systematic investigation of a recently characterized 1,4,5-substituted-imidazole scaffold and a newly reported 1,4,5-substituted-triazole scaffold, we have described a new BET-inhibitor class with high selectivity for D1 over D2 of BRD4 and BRDT. From a therapeutic standpoint, this class of 1,4,5-trisubstituted imidazoles and triazoles may be a valuable starting point for dual kinase–bromodomain inhibition, as simultaneous BRD4 and MAP–PI3-kinase inhibition has recently been suggested for synergistic treatment of colon cancer, avoiding the development of resistance to BET- or kinase-inhibitor monotherapy<a onclick="showRef(event, 'ref28 ref49'); return false;" href="javascript:void(0);" class="ref ref28 ref49">(28,49)</a> and resulting in decreased toxicity relative to a cocktail of (+)-JQ1 and PI3K inhibitors.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> As demonstrated here, the optimization of further inhibitors of individual BET D1s may enable selective control of BRD4-associated inflammatory signaling. Finally, the displacement of structured waters may also be a general strategy that can be applied to other known pan-BET inhibitors, which we are actively investigating.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38847" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38847" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Synthetic Methods</h3><div class="NLM_p">All chemical reagents were purchased from commercial vendors and used without further purification. Flash chromatography of compounds was performed on a Teledyne-Isco Rf-plus CombiFlash instrument across RediSep Rf columns. <sup>1</sup>H (400 MHz) and <sup>13</sup>C (125 MHz) NMR spectra were recorded in chloroform-<i>d</i> or methanol-<i>d</i><sub>4</sub> and chemical shifts (δ) are reported in parts per million (ppm). All coupling constants (<i>J</i>) are reported in hertz (Hz), with s, singlet; d, doublet; t, triplet; and m, multiplet. Purities of final compounds diluted in DMSO were determined by reverse-phase-high-performance-liquid-chromatography (HPLC) analysis using an Agilent 1200 Series HPLC (215 and 254 nm DAD detector; Agilent Zorbax SBC18 column, 4.6 × 150 mm, 5.0 μm; mobile phase gradient of 90:10 to 15:85 0.1% TFA in H<sub>2</sub>O/acetonitrile over 22 min; flow rate of 1.0 mL/min), and all compounds used were verified to be >95% pure by peak area. High-resolution mass spectrometry was performed using positive-mode-electrospray-ionization methods (ESI-MS) with a Bruker BioTOF II spectrometer.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Synthesis of Common Intermediate <b>12</b></h4><div id="sec5_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Synthesis of 4-Methylbenzenesulfinic Acid (<b>7</b>)</h5><div class="NLM_p last">To a suspended solution of <i>p</i>-toluenesulfinic acid sodium salt, <b>6</b> (3.0 g, 16 mmol), in water (10.0 mL) was added methyl <i>tert</i>-butylether (5.0 mL) followed by dropwise addition of concd HCl (1.5 mL). After the reaction was stirred for 5 min, the organic phase was separated, and the aqueous phase was extracted with methyl <i>tert</i>-butylether (2 × 25 mL). The combined organic phase was dried over anhydrous magnesium sulfate and concentrated under reduced pressure. The solid obtained was resuspended in hexanes (50 mL), filtered, and dried to give the desired compound, <b>7</b>, as a white solid (2.4 g, 91%).</div></div><div id="sec5_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Synthesis of <i>N</i>-((4-Fluorophenyl)(tosyl)methyl)formamide (<b>8</b>)</h5><div class="NLM_p last">A mixture of 4-methylbenzenesulfinic acid, <b>7</b> (0.50 g, 3.2 mmol); 4-fluorobenzaldehyde (0.58 g, 4.7 mmol); formamide (0.52 g, 12 mmol); and camphorsulfonic acid (0.09 g, 0.4 mmol) was stirred at 60 °C for 18 h. The resulting solid was stirred with a mixture of methanol (0.8 mL) and hexanes (1.9 mL). The solid was filtered, resuspended in methanol/hexanes (1:3, 4.5 mL), and stirred to obtain a fine suspension. The suspension was filtered and dried to give the pure compound, <b>8</b>, as a white solid (0.64 g, 65%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.06 (s, 1H), 7.69 (d, 2H, <i>J</i> = 8.3 Hz), 7.43–7.4 (m, 2H), 7.31 (d, 2H, <i>J</i> = 7.8 Hz), 7.06 (t, 2H, <i>J</i> = 8.3 Hz), 6.29 (s, 1H), 2.43 (s, 3H).</div></div><div id="sec5_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> Synthesis of 1-Fluoro-4-(isocyano(tosyl)methyl)benzene (<b>9</b>)</h5><div class="NLM_p last">To a solution of <i>N</i>-((4-fluorophenyl)(tosyl)methyl)formamide <b>8</b> (0.20 g, 0.65 mmol) in 1,2-dimethoxyethane (3.5 mL) at −10 °C was added POCl<sub>3</sub> (0.15 mL, 1.6 mmol) followed by Et<sub>3</sub>N (0.46 mL, 3.3 mmol), keeping the internal temperature below −5 °C. The reaction was warmed to room temperature and stirred for 2 h. The reaction mixture was quenched by addition of ice-cold water, and the aqueous phase was extracted with methyl <i>tert</i>-butylether (2 × 15 mL). The combined organic phase was washed with saturated aqueous NaHCO<sub>3</sub> (1 × 10 mL) and brine (1 × 10 mL) and concentrated under reduced pressure. The residue was triturated with hexanes to give the desired product as a brown solid (0.11 g, 55%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.65 (d, 2H, <i>J</i> = 8.0 Hz), 7.38–7.37 (m, 4H), 7.12 (t, 2H, <i>J</i> = 8.3 Hz), 5.61 (s, 1H), 2.5 (s, 3H).</div></div><div id="sec5_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Synthesis of 4-(Dimethoxymethyl)-2-(methylthio)pyrimidine (<b>13</b>)</h5><div class="NLM_p last">A mixture of pyruvic aldehyde dimethylacetal, <b>10</b> (6.0 mL, 50 mmol), and <i>N</i>,<i>N</i>-dimethylformamide dimethylacetal (6.0 mL, 45 mmol) was heated at 100 °C for 18 h. After the mixture was cooled to room temperature, methanol (30.0 mL), thiourea (6.96 g, 91.4 mmol), and sodium methoxide (25% in MeOH, 23.1 mL, 106 mmol) were added. The reaction mixture was then stirred at 70 °C for 2 h. The reaction mixture was then cooled, iodomethane (14.4 mL, 231 mmol) was added dropwise, and the reaction was stirred at room temperature for 3 h. The reaction mixture was then diluted with ethyl acetate (200 mL) and water (100 mL). The organic phase was separated, and the aqueous phase was extracted with ethyl acetate (2 × 100 mL). The combined organic phase was dried over anhydrous magnesium sulfate and concentrated under reduced pressure to give the corresponding product, <b>13</b>, as a brown oil (7.5 g, 75%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.56 (d, 1H, <i>J</i> = 5.1 Hz), 7.19 (d, 1H, <i>J</i> = 4.9 Hz), 5.19 (s, 1H), 3.42 (s, 6H), 2.58 (s, 3H).</div></div><div id="sec5_1_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Synthesis of 2-(Methylthio)pyrimidine-4-carbaldehyde (<b>14</b>)</h5><div class="NLM_p last">A solution of 4-(dimethoxymethyl)-2-(methylthio)pyrimidine <b>13</b> (2.0 g, 10 mmol) in 3 N HCl (8.4 mL, 25 mmol) was stirred at 48 °C for 16 h. The reaction was cooled and quenched by addition of solid Na<sub>2</sub>CO<sub>3</sub>. The residue was washed with ethyl acetate (3 × 50 mL). The combined organic phase was dried and concentrated to give the crude product, which was purified by filtration over a pad of silica gel eluting with dichloromethane to give the pure product as a pale-yellow solid (0.6 g, 40%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.96 (d, 1H, 0.6 Hz), 8.77 (dd, 1H, <i>J</i> = 4.8, 0.5 Hz), 7.45 (d, 1H, <i>J</i> = 4.8 Hz), 2.64 (s, 3H).</div></div><div id="sec5_1_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> Synthesis of <i>tert</i>-Butyl 4-(((2-(Methylthio)pyrimidin-4-yl)methylene)amino)piperidine-1-carboxylate (<b>15</b>)</h5><div class="NLM_p last">To a solution of 2-(methylthio)pyrimidine-4-carbaldehyde, <b>9</b> (0.20 g, 1.3 mmol), and <i>tert</i>-butyl 4-aminopiperidine-1-carboxylate (0.26 g, 1.3 mmol) in dichloromethane (3 mL) was added anhydrous magnesium sulfate (0.31 g, 2.6 mmol), and the reaction was stirred at room temperature for 24 h. After confirmation of the completion of reaction by TLC, the reaction mixture was filtered, and the solvent was evaporated to give the desired product, which was taken directly to the next step.</div></div><div id="sec5_1_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> Synthesis of <i>tert</i>-Butyl 4-(4-(4-Fluorophenyl)-5-(2-(methylthio)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate (<b>16</b>)</h5><div class="NLM_p last">A mixture of <i>tert</i>-butyl 4-(((2-(methylthio)pyrimidin-4-yl)methylene)amino)piperidine-1-carboxylate, <b>15</b> (0.44 g, 1.3 mmol); 1-fluoro-4-(isocyano(tosyl)methyl)benzene, <b>4</b> (0.38 g, 1.3 mmol); and potassium carbonate (0.18 g, 1.3 mmol) in DMF at room temperature was stirred for 48 h. The reaction mixture was quenched by the addition of water and extracted with ethyl acetate (3 × 25 mL). The combined organic phase was washed with 5% aqueous LiCl and concentrated under reduced pressure to give the crude product, which was purified by column chromatography over silica gel (60–120 mesh) using ethyl acetate and hexanes as eluent (0–60%) to give the desired product as a yellow solid (0.17 g, 28%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.35 (d, 1H, <i>J</i> = 5.2 Hz), 7.76 (s, 1H), 7.45–7.41 (m, 2H), 7.03 (t, 2H, <i>J</i> = 8.6 Hz), 6.79 (d, 1H, <i>J</i> = 5.2 Hz), 4.85 (tt, 1H, <i>J</i> = 12.0, 3.6 Hz), 4.31 (br s, 2H), 2.84–2.78 (m, 2H), 2.6 (s, 3H), 2.19–2.16 (m, 2H), 1.86 (dq, 2H, <i>J</i> = 12.3, 4.1 Hz), 1.49 (s, 9H).</div></div><div id="sec5_1_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Synthesis of <i>tert</i>-Butyl 4-(4-(4-Fluorophenyl)-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate (<b>17</b>)</h5><div class="NLM_p last">A solution of OXONE (160 mg, 0.25 mmol) in water (1.4 mL) was added dropwise to a solution of <i>tert</i>-butyl 4-(4-(4-fluorophenyl)-5-(2-(methylthio)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate, <b>16</b> (50 mg, 0.11 mmol), in THF (2.0 mL) at −10 °C. The reaction mixture was then stirred at room temperature for 24 h. The reaction mixture was quenched by addition of ice-cold water and extracted with dichloromethane (3 × 20 mL). The combined organic phase was washed with brine and concentrated under reduced pressure to give the desired product as a pale-yellow foam (45 mg, 84%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.50 (d, <i>J</i> = 5.4 Hz, 1H), 7.78 (s, 1H), 7.42–7.35 (m, 2H), 7.22 (d, <i>J</i> = 5.4 Hz, 1H), 7.03 (t, <i>J</i> = 8.6 Hz, 2H), 5.00 (tt, <i>J</i> = 12.0, 3.6 Hz, 1H), 3.72–3.64 (m, 4H), 3.32 (s, 3H), 2.28–2.11 (m, 2H), 2.00–1.81 (m, 2H).</div></div><div id="sec5_1_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Synthesis of <i>tert</i>-Butyl 4-(5-(2-(3,5-Dimethylphenoxy)pyrimidin-4-yl)-4-(4-fluorophenyl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate (<b>18</b>)</h5><div class="NLM_p last">To a suspension of NaH (60% in mineral oil, 9 mg, 0.04 mmol) in anhydrous THF (2 mL) was added 3,5-difluorophenol (50 mg, 0.40 mmol) in anhydrous THF (0.5 mL) dropwise at room temperature. The mixture was stirred for 10 min; this was followed by the addition of <i>tert</i>-butyl 4-(4-(4-fluorophenyl)-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate, <b>17</b> (45 mg, 0.091 mmol), in anhydrous THF (0.5 mL). The reaction mixture was stirred for 0.5 h, quenched by the addition of water, and extracted with ethyl acetate (3 × 10 mL). The combined organic phase was dried and concentrated to give the product, which was taken to the next step without further purification (29 mg, 59%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.37 (d, <i>J</i> = 5.2 Hz, 1H), 7.71 (s, 1H), 7.48–7.41 (m, 2H), 7.05 (t, <i>J</i> = 8.7 Hz, 2H), 6.91 (dt, <i>J</i> = 1.5, 0.8 Hz, 1H), 6.87–6.79 (m, 3H), 4.69 (tt, <i>J</i> = 12.0, 3.7 Hz, 1H), 4.22–4.01 (m, 2H), 2.48 (d, <i>J</i> = 13.0 Hz, 2H), 2.34 (s, 6H), 1.95–1.88 (m, 2H), 1.70 (dd, <i>J</i> = 11.7, 4.2 Hz, 2H), 1.48 (s, 9H). ESI-MS: calcd: 543.3, found: 544.3 (M+H).</div></div><div id="sec5_1_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Synthesis of 2-(3,5-Dimethylphenoxy)-4-(4-(4-fluorophenyl)-1-(piperidin-4-yl)-1<i>H</i>-imidazol-5-yl)pyrimidine (<b>I</b>)</h5><div class="NLM_p last">To a solution of compound <b>18</b> (29 mg, 0.053 mmol) in dichloromethane (1 mL) was added trifluoroacetic acid (0.1 mL), and the reaction mixture was stirred at room temperature for 15 min. The solvent was evaporated, and the crude sample was triturated with diethyl ether to give the desired product, <b>I</b>, as a white solid (2TFA salt; 25 mg, 70%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.41 (d, <i>J</i> = 5.2 Hz, 1H), 8.37 (s, 1H), 7.40–7.33 (m, 2H), 7.10 (t, <i>J</i> = 8.8 Hz, 2H), 6.87 (s, 1H), 6.85 (d, <i>J</i> = 5.2 Hz, 1H), 6.83 (s, 2H), 4.64 (ddd, <i>J</i> = 11.9, 7.9, 3.9 Hz, 1H), 3.29–3.25 (m, 2H), 2.61 (td, <i>J</i> = 13.1, 3.0 Hz, 2H), 2.26 (s, 6H), 2.15–2.08 (m, 2H), 1.99 (qd, <i>J</i> = 13.1, 4.1 Hz, 2H). <sup>13</sup>C NMR (125 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.1, 164.5 (d, <i>J</i> = 248.7 Hz), 162.5, 160.9 (d, <i>J</i> = 36.4 Hz), 160.6, 157.6, 152.9, 139.7, 139.0, 136.2, 130.9 (d, <i>J</i> = 8.9 Hz), 127.0, 126.4, 125.0, 119.0, 116.7, 115.9 (d, <i>J</i> = 22.7 Hz), 115.7, 115.4, 52.4, 43.1, 29.5, 19.9. HRMS: calcd: 444.2200, found: 444.2207. HPLC Purity: 97.2%.</div></div><div id="sec5_1_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Synthesis of <i>tert</i>-Butyl 4-(4-(4-Fluorophenyl)-5-(2-(<i>p</i>-tolyloxy)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate (<b>19</b>)</h5><div class="NLM_p last">To a suspension of NaH (60% in mineral oil, 13 mg, 0.56 mmol) in anhydrous THF (2 mL) was added <i>p</i>-cresol (66 mg, 0.62 mmol) in anhydrous THF (0.5 mL) dropwise at room temperature. The mixture was stirred for 10 min, which was followed by the addition of <i>tert</i>-butyl 4-(4-(4-fluorophenyl)-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate, <b>17</b> (70 mg, 0.14 mmol), in anhydrous THF (0.5 mL). The reaction mixture was stirred for 0.5 h, quenched by the addition of water, and extracted with ethyl acetate (3 × 10 mL). The combined organic phase was dried and concentrated to give the product, which was taken to the next step without further purification (65 mg, 88%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.38 (d, <i>J</i> = 5.2 Hz, 1H), 7.70 (s, 1H), 7.47–7.39 (m, 2H), 7.26–7.21 (m, 2H), 7.13 (d, <i>J</i> = 8.5 Hz, 2H), 7.08–7.00 (m, 2H), 6.84 (d, <i>J</i> = 5.2 Hz, 1H), 4.66 (tt, <i>J</i> = 11.9, 3.8 Hz, 1H), 4.21–3.99 (m, 2H), 2.46 (t, <i>J</i> = 12.8 Hz, 2H), 2.38 (s, 3H), 1.91–1.84 (m, 2H), 1.70 (dt, <i>J</i> = 12.3, 6.1 Hz, 2H), 1.48 (s, 9H).</div></div><div id="sec5_1_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Synthesis of 4-(4-(4-Fluorophenyl)-1-(piperidin-4-yl)-1<i>H</i>-imidazol-5-yl)-2-(<i>p</i>-tolyloxy)pyrimidine (<b>II</b>)</h5><div class="NLM_p last">To a solution of compound <b>19</b> (65 mg, 0.12 mmol) in dichloromethane (1 mL) was added trifluoroacetic acid (0.1 mL), and the reaction mixture was stirred at room temperature for 15 min. The solvent was evaporated, and the crude was triturated with diethyl ether to give the desired product, <b>II</b>, as a white solid (2TFA salt; 35 mg, 50%). <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.55 (d, <i>J</i> = 5.1 Hz, 1H), 7.50–7.46 (m, 2H), 7.32 (d, <i>J</i> = 8.2 Hz, 2H), 7.25–7.19 (m, 4H), 6.98 (d, <i>J</i> = 5.1 Hz, 1H), 4.77 (tt, <i>J</i> = 11.6, 4.1 Hz, 1H), 3.44–3.36 (m, 2H), 2.76 (td, <i>J</i> = 13.0, 3.2 Hz, 2H), 2.42 (s, 3H), 2.23–2.09 (m, 4H). <sup>13</sup>C NMR (125 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.3, 164.6, (d, <i>J</i> = 249.3 Hz), 162.6, 161.2, 161.0, (d, <i>J</i> = 35.5 Hz), 157.7, 150.7, 138.2, 136.1, 135.4, 131.0 (d, <i>J</i> = 9.0 Hz), 130.9, 130.0, 125.8, 125.0, 121.3, 116.8, 115.9 (d, <i>J</i> = 21.8 Hz), 115.8, 52.5, 43.0, 29.4, 19.5. HRMS: calcd: 430.2043, found: 430.2041. HPLC Purity: 99.5%.</div></div><div id="sec5_1_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Synthesis of <i>tert</i>-Butyl 4-(5-(2-(3,4-Dimethylphenoxy)pyrimidin-4-yl)-4-(4-fluorophenyl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate (<b>20</b>)</h5><div class="NLM_p last">To a suspension of NaH (60% in mineral oil, 9.0 mg, 0.36 mmol) in anhydrous THF (2 mL) was added 3,4-dimethylphenol (50 mg, 0.40 mmol) in anhydrous THF (0.5 mL) dropwise at room temperature. The mixture was stirred for 10 min; this was followed by the addition of <i>tert</i>-butyl 4-(4-(4-fluorophenyl)-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate, <b>17</b> (45 mg, 0.091 mmol), in anhydrous THF (0.5 mL). The reaction mixture was stirred for 0.5 h, quenched by the addition of water, and extracted with ethyl acetate (3 × 10 mL). The combined organic phase was dried and concentrated to give the product, which was taken to the next step without further purification (47 mg, 95%). ESI-MS: calcd: 543.3, found: 544.3 (M+H).</div></div><div id="sec5_1_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Synthesis of 2-(3,4-Dimethylphenoxy)-4-(4-(4-fluorophenyl)-1-(piperidin-4-yl)-1<i>H</i>-imidazol-5-yl)pyrimidine (<b>III</b>)</h5><div class="NLM_p last">To a solution of compound <b>20</b> (47 mg, 0.86 mmol) in dichloromethane (1 mL) was added trifluoroacetic acid (0.1 mL), and the reaction mixture was stirred at room temperature for 15 min. The solvent was evaporated, and the crude was triturated with diethyl ether to give the desired product, <b>III</b>, as a white solid (2TFA salt; 30 mg, 52%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.53 (d, <i>J</i> = 5.1 Hz, 1H), 8.51 (s, 1H), 7.51–7.45 (m, 2H), 7.26 (d, <i>J</i> = 8.2 Hz, 1H), 7.24–7.19 (m, 2H), 7.12 (d, <i>J</i> = 2.5 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 2.5 Hz, 1H), 6.96 (d, <i>J</i> = 5.1 Hz, 1H), 4.78 (dq, <i>J</i> = 11.9, 4.0 Hz, 1H), 3.42–3.33 (m, 2H), 2.73 (td, <i>J</i> = 13.0, 3.1 Hz, 2H), 2.33 (s, 6H), 2.25–2.16 (m, 2H), 2.10 (qd, <i>J</i> = 12.9, 3.9 Hz, 2H). <sup>13</sup>C NMR (125 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.3, 164.6 (d, <i>J</i> = 248.9 Hz), 162.6, 160.8, (d, <i>J</i> = 35.1 Hz) 157.1, 150.8, 138.3, 137.9, 136.0, 134.0, 131.1, 131.0 (d, <i>J</i> = 8.6 Hz), 130.3, 129.8, 125.5, 125.1, 122.3, 118.6, 116.8, 115.9 (d, <i>J</i> = 22.1 Hz), 115.8, 52.6, 43.0, 29.4, 18.5, 17.8. HRMS: calcd: 444.2200, found: 444.2226. HPLC Purity: 99.7%.</div></div><div id="sec5_1_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Synthesis of <i>tert</i>-Butyl 4-(5-(2-((3,4-Dimethylphenyl)amino)pyrimidin-4-yl)-4-(4-fluorophenyl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate (<b>21</b>)</h5><div class="NLM_p last">A mixture of 3,4-dimethylaniline (150 mg, 1.25 mmol) and <i>tert</i>-butyl 4-(4-(4-fluorophenyl)-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate, <b>17</b> (65 mg, 0.13 mmol), in 1,4-dioxane (1 mL) were heated in a sealed tube at 140 °C for 16 h. The reaction mixture was quenched by the addition of water and extracted with ethyl acetate (3 × 5 mL). The combined organic phase was dried and concentrated to give the product, which was purified by column chromatography over silica gel (60–120 mesh) using ethyl acetate and hexanes as eluent (70–100%, 32 mg, 46%). The product was taken to the next step without further purification.</div></div><div id="sec5_1_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Synthesis of <i>N</i>-(3,4-Dimethylphenyl)-4-(4-(4-fluorophenyl)-1-(piperidin-4-yl)-1<i>H</i>-imidazol-5-yl)pyrimidin-2-amine (<b>IV</b>)</h5><div class="NLM_p last">To a solution of compound <b>21</b> (32 mg, 0.059 mmol) in dichloromethane (1 mL) was added trifluoroacetic acid (0.1 mL), and the reaction mixture was stirred at room temperature for 15 min. The solvent was evaporated, and the crude was triturated with diethyl ether to give the desired product, <b>IV</b>, as a pale-yellow solid (3TFA salt; 23 mg, 50%). <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.28 (d, <i>J</i> = 5.0 Hz, 1H), 8.23 (s, 0.5H), 7.55–7.46 (m, 2H), 7.35 (dq, <i>J</i> = 4.7, 2.4 Hz, 2H), 7.22–7.09 (m, 3H), 6.52 (d, <i>J</i> = 5.1 Hz, 1H), 5.00 (ddt, <i>J</i> = 12.1, 8.3, 3.8 Hz, 1H), 3.42–3.34 (m, 2H), 2.76–2.67 (m, 2H), 2.39 (dt, <i>J</i> = 13.3, 2.7 Hz, 2H), 2.28 (d, <i>J</i> = 3.6 Hz, 6H), 2.15 (qd, <i>J</i> = 13.2, 4.2 Hz, 2H). <sup>13</sup>C NMR (125 MHz, methanol-<i>d</i><sub>4</sub>) δ 163.9, 161.9, 161.7, 161.4, 160.8, 158.5, 157.0, 139.9, 137.0, 136.6, 135.8, 131.6, 130.5, 130.4, 129.4, 125.4, 122.8, 119.0, 117.9, 115.4, 115.2, 112.4, 51.6, 43.1, 29.7, 18.6, 17.8. HRMS: calcd: 443.2359, found: 443.2345. HPLC Purity: 97.5%.</div></div><div id="sec5_1_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Synthesis of <i>tert</i>-Butyl 4-(5-(2-((3,5-Dimethylphenyl)amino)pyrimidin-4-yl)-4-(4-fluorophenyl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate (<b>22</b>)</h5><div class="NLM_p last">A mixture of 3,5-dimethylaniline (100 mg, 0.825 mmol) and <i>tert</i>-butyl 4-(4-(4-fluorophenyl)-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate, <b>17</b> (45 mg, 0.091 mmol), was heated in a sealed tube at 140 °C for 16 h. The reaction mixture was quenched by the addition of water and extracted with ethyl acetate (3 × 5 mL). The combined organic phase was dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure to give the product, which was purified by column chromatography over silica gel (60–120 mesh) using ethyl acetate and hexanes as eluent (70–100%, 27 mg, 55%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.25 (d, <i>J</i> = 5.1 Hz, 1H), 7.73 (s, 1H), 7.53–7.45 (m, 2H), 7.22–7.18 (m, 3H), 7.06–6.98 (m, 2H), 6.75 (s, 1H), 6.56 (d, <i>J</i> = 5.1 Hz, 1H), 4.86–4.74 (m, 1H), 4.12 (q, <i>J</i> = 7.1 Hz, 2H), 2.44 (dd, <i>J</i> = 15.7, 9.9 Hz, 2H), 2.32 (s, 6H), 1.84–1.69 (m, 4H), 1.46 (s, 9H).</div></div><div id="sec5_1_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Synthesis of <i>N</i>-(3,5-Dimethylphenyl)-4-(4-(4-fluorophenyl)-1-(piperidin-4-yl)-1<i>H</i>-imidazol-5-yl)pyrimidin-2-amine (<b>V</b>)</h5><div class="NLM_p last">To a solution of compound <b>22</b> (30 mg, 0.05 mmol) in dichloromethane (1 mL) was added trifluoroacetic acid (0.1 mL), and the reaction mixture was stirred at room temperature for 15 min. The solvent was evaporated, and the crude was triturated with diethyl ether to give the desired product, <b>V</b>, as a pale-yellow solid (3TFA salt; 37 mg, 96%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.29 (d, <i>J</i> = 5.1 Hz, 1H), 7.80 (s, 1H), 7.51 (dd, <i>J</i> = 8.4, 5.4 Hz, 2H), 7.24 (s, 2H), 7.19 (s, 1H), 7.04 (t, <i>J</i> = 8.6 Hz, 2H), 6.76 (s, 1H), 6.58 (d, <i>J</i> = 5.1 Hz, 1H), 4.78 (tt, <i>J</i> = 12.0, 3.9 Hz, 1H), 3.15–3.08 (m, 2H), 2.49–2.40 (m, 2H), 2.34 (s, 6H), 2.12–2.05 (m, 2H), 1.84 (qd, <i>J</i> = 12.2, 3.9 Hz, 2H). <sup>13</sup>C NMR (125 MHz, methanol-<i>d</i><sub>4</sub>) δ 164.4, (d, <i>J</i> = 247.6 Hz) 162.4, 160.7, 158.8, 156.0, 152.6, 148.2, 139.1, 138.2, 135.4, 130.7, (d, <i>J</i> = 8.4 Hz) 130.6, 124.8, 119.0, 115.8, (d, <i>J</i> = 22.1 Hz) 115.6, 112.6, 52.3, 48.1, 43.0, 29.5, 20.1. HRMS: calcd: 443.2359, found: 443.2388. HPLC Purity: 95.7%.</div></div><div id="sec5_1_1_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Synthesis of <i>tert</i>-Butyl 4-(5-(2-(3,5-Difluorophenoxy)pyrimidin-4-yl)-4-(4-fluorophenyl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate (<b>23</b>)</h5><div class="NLM_p last">To a suspension of NaH (60% in mineral oil, 9 mg, 0.04 mmol) in anhydrous THF (2 mL) was added 3,5-dimethylphenol (50 mg, 0.40 mmol) in anhydrous THF (0.5 mL) dropwise at room temperature. The mixture was stirred for 10 min; this was followed by the addition of <i>tert</i>-butyl 4-(4-(4-fluorophenyl)-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-imidazol-1-yl)piperidine-1-carboxylate, <b>17</b> (45 mg, 0.091 mmol), in anhydrous THF (0.5 mL). The reaction mixture was stirred for 0.5 h, quenched by the addition of water and extracted with ethyl acetate (3 × 10 mL). The combined organic phase was dried and concentrated to give the product, which was taken to the next step without further purification (29 mg, 59%). <sup>1</sup>H NMR (400 MHz, methanol-d4) δ 8.46 (d, <i>J</i> = 4.9 Hz, 2H), 7.5–7.3 (m, 2H), 7.09 (t, 2H), 6.95 (dt, 3H), 6.87 (tt, <i>J</i> = 9.2, 2.3 Hz, 1H), 4.7–4.6 (m, 1H), 3.5–3.4 (m, 3H), 2.88 (td, <i>J</i> = 13.1, 3.0 Hz, 2H), 2.21 (d, <i>J</i> = 13.4 Hz, 2H), 2.10 (qd, <i>J</i> = 13.0, 4.1 Hz, 2H), 1.45 (s, 9H). 1.22 (s, 1H), 1.19 (s, 2H), 0.8–0.7 (m, 1H). ESI-MS: calcd: 551.6, found: 552.1 (M+H).</div></div><div id="sec5_1_1_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Synthesis of 2-(3,5-Difluorophenoxy)-4-(4-(4-fluorophenyl)-1-(piperidin-4-yl)-1<i>H</i>-imidazol-5-yl)pyrimidine (<b>VI</b>)</h5><div class="NLM_p last">To a solution of compound <b>23</b> (45 mg, 0.82 mmol) in dichloromethane (1 mL) was added trifluoroacetic acid (0.1 mL), and the reaction mixture was stirred at room temperature for 15 min. The solvent was evaporated, and the crude was triturated with diethyl ether to give the desired product, <b>VI</b>, as a white solid (2TFA salt; 31 mg, 56%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.46 (d, <i>J</i> = 4.9 Hz, 1H), 7.40–7.32 (m, 1H), 7.14–7.03 (m, 1H), 6.99–6.82 (m, 2H), 3.44–3.36 (m, 2H), 2.88 (td, <i>J</i> = 13.1, 3.0 Hz, 1H), 2.21 (d, <i>J</i> = 13.4 Hz, 1H), 2.10 (qd, <i>J</i> = 13.0, 4.1 Hz, 2H), 1.19 (s, 1H), 0.84–0.73 (m, 1H). <sup>13</sup>C NMR (125 MHz, methanol-<i>d</i><sub>4</sub>) δ 163.3 (dd, <i>J</i><sub>C–F</sub> = 248.8, 15.2 Hz), 161.1 (d, <i>J</i><sub>C–F</sub> = 35.2 Hz), 160.6, 158.6, 154.7 (d, <i>J</i><sub>C–F</sub> = 13.4 Hz), 136.5, 130.7 (d, <i>J</i><sub>C–F</sub> = 8.5 Hz), 117.7, 115.6 (d, <i>J</i><sub>C–F</sub> = 22.1 Hz), 105.8 (t, <i>J</i><sub>C–F</sub> = 22.1, 21.5 Hz), 100.9 (t, <i>J</i><sub>C–F</sub> = 26.1 Hz), 51.9, 43.2, 29.7. HRMS: calcd: 451.1680, found: 452.2210. HPLC Purity: 97.9%.</div></div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Synthesis of Triazoles <b>VII</b>–<b>IX</b></h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Synthesis of Sulfones <b>31</b>, <b>32</b>, and <b>33</b></h4><div id="sec5_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> Synthesis of <i>tert</i>-Butyl 4-Azidopiperidine-1-carboxylate (<b>24</b>)</h5><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-((methylsulfonyl)oxy)piperidine-1-carboxylate (1.1 g, 3.9 mmol) in DMF (10 mL) was added solid NaN<sub>3</sub> (0.48 g, 7.3 mmol). The reaction was sealed under air and heated to 75 °C. After 20 h, the reaction was cooled to room temperature. The reaction mixture was diluted with water and extracted with ethyl acetate. The combined organic phases were washed with water and brine, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure. This afforded azide <b>24</b> quantitatively, which was used without further purification. Characterization data for this compound have been previously reported.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div></div><div id="sec5_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> General Click Procedure and Synthesis of <i>tert</i>-Butyl 4-(4-(4-Fluorophenyl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>25</b>)</h5><div class="NLM_p last">The crude oil of azide <b>24</b> (3.9 mmol, assuming quantitative yield from previous reaction) was dissolved in <i>t</i>-BuOH/H<sub>2</sub>O (1:1, 10 mL). A separate vial was charged with alkyne (660 mg, 4.8 mmol), copper sulfate pentahydrate (40 mg, 0.16 mmol), and sodium ascorbate (170 mg, 0.84 mmol). The solution of azide was transferred into the vial containing alkyne by pipet and sealed under air. After 18 h at room temperature, the reaction mixture was diluted with water and extracted with ethyl acetate. The combined organic phases were washed with brine, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure to afford the triazole (1.10 g, 77%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.81 (apparent dd, <i>J</i> = 8.8, 5.3 Hz, 2H), 7.75 (s, 1H), 7.13 (apparent t, <i>J</i> = 8.7 Hz, 2H), 4.67 (tt, <i>J</i> = 11.6, 4.1 Hz, 1H), 4.30 (br, 2H), 2.97 (br, 2H), 2.28–2.22 (m, 2H), 2.00 (qd, <i>J</i> = 12.2, 4.4 Hz, 2H), 1.50 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 162.7 (d, <i>J</i><sub>C–F</sub> = 247.4 Hz), 154.5, 146.8, 127.4 (d, <i>J</i><sub>C–F</sub> = 8.1 Hz), 126.8 (d, <i>J</i><sub>C–F</sub> = 3.2 Hz), 117.1, 115.9 (d, <i>J</i><sub>C–F</sub> = 21.6 Hz), 80.2, 58.3, 42.7 (br), 32.5, 28.4. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ −113.5. IR (NaCl, thin film, cm<sup>–1</sup>): 2974, 1689, 1496, 1424, 1244, 1166. HRMS (ESI): calcd for C<sub>18</sub>H<sub>23</sub>FN<sub>4</sub>NaO<sub>2</sub><sup>+</sup>, (M+Na)<sup>+</sup> 369.1697, found 369.1682.</div></div><div id="sec5_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> Synthesis of <i>tert</i>-Butyl 4-(4-(((<i>tert</i>-Butyldimethylsilyl)oxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>26</b>)</h5><div class="NLM_p last">The <a class="ref internalNav" href="#sec5_2_1_2" aria-label="General Click Procedure">General Click Procedure</a> was used, and the product, <b>26</b>, was isolated as a white solid in 83% yield. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.47 (s, 1H), 4.84 (s, 2H), 4.59 (tt, <i>J</i> = 11.6, 4.0 Hz, 1H), 4.26 (br, 2H), 2.93 (t, <i>J</i> = 13.1 Hz, 2H), 2.26–2.12 (m, 2H), 1.94 (apparent qd, <i>J</i> = 12.3, 4.4 Hz, 2H), 1.47 (s, 9H), 0.91 (s, 9H), 0.10 (s, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 154.7, 148.7, 119.4, 80.3, 58.2, 58.2, 42.9 (br) 32.6, 28.6, 26.1, 18.5, −5.1. IR (NaCl, thin film, cm<sup>–1</sup>): 2930, 2857, 1696, 1423, 1366, 1244, 1168, 838, 778. HRMS (ESI): calcd for C<sub>19</sub>H<sub>36</sub>N<sub>4</sub>NaO<sub>3</sub>Si<sup>+</sup>, (M+Na)<sup>+</sup> 419.2449, found 419.2452.</div></div><div id="sec5_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> Synthesis of <i>tert</i>-Butyl 4-(4-Methyl-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>27</b>)</h5><div class="NLM_p last">The crude oil of the azide (3.6 mmol, assuming quantitative yield from previous reaction) was dissolved in <i>t</i>-BuOH/H<sub>2</sub>O (1:1, 20 mL). A separate three-neck round-bottom flask was charged with copper sulfate pentahydrate (290 mg, 1.2 mmol) and sodium ascorbate (710 mg, 3.6 mmol). The solution of azide was transferred into this flask by pipet. The center neck of the flask was fitted with a condensing dewar which was filled with a dry ice/acetone bath. The other two necks were sealed with septa. The top of the condensing dewar was vented by needle, and this vent remained open to avoid over pressurization. Propyne was bubbled into the solution of azide by needle until the propyne started to reflux in the condensing dewar. After 1 h at room temperature, the flask was placed on a 40 °C heating block to melt the ice that formed in the flask as a result of propyne condensation. After 4 h at this temperature, the reaction was cooled to room temperature, and the mixture was diluted with water and brine and extracted with ethyl acetate. The combined organic phases were washed with brine, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure to afford the triazole quantitatively as a pale-green solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.96 (s, 1H), 4.64 (tt, <i>J</i> = 11.4, 3.7 Hz, 1H), 4.04 (d, <i>J</i> = 11.0 Hz, 2H), 2.94 (br, 2H), 2.21 (s, 3H), 2.02 (d, <i>J</i> = 12.2 Hz, 2H), 1.80 (qd, <i>J</i> = 12.2, 4.3 Hz, 2H), 1.42 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 154.3, 142.9 (br), 121.3, 79.4, 57.3, 43.1 (br), 32.4, 28.5, 11.1. IR (NaCl, thin film, cm<sup>–1</sup>): 2974, 1689, 1424, 1366, 1250, 1169, 1004. HRMS (ESI): calcd for C<sub>13</sub>H<sub>22</sub>N<sub>4</sub>NaO<sub>2</sub><sup>+</sup>, (M+Na)<sup>+</sup> 289.1635, found 289.1632.</div></div><div id="sec5_2_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> General Coupling Procedure and Synthesis of <i>tert</i>-Butyl 4-(4-(4-Fluorophenyl)-5-(2-(methylthio)pyrimidin-4-yl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>28</b>)</h5><div class="NLM_p last">A solution of triazole <b>25</b> (170 mg, 0.50 mmol) in THF (8 mL) was cooled in a dry ice/acetone bath. Then <i>n</i>-BuLi (0.26 mL, 2.5 M in hexanes, 0.65 mmol) was added dropwise. After 10 min, a freshly prepared solution of ZnCl<sub>2</sub> (110 mg, 0.78 mmol) in THF (1 mL) was added dropwise. After 10 min, a freshly prepared solution of SPhos Pd G3 (17 mg, 22 μmol) in THF (1 mL) was added followed by chloro-pyrimidine (0.12 mL, 1.0 mmol). The reaction was heated to 60 °C. After 1.5 h, the reaction mixture was diluted with water and extracted with ethyl acetate. The combined organic phases were washed with brine, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure. Final purification by column chromatography (0–50% EtOAc in hexanes) yielded the product as a white solid (210 mg, 88%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.50 (d, <i>J</i> = 5.1 Hz, 1H), 7.50 (apparent dd, <i>J</i> = 8.6, 5.4 Hz, 2H), 7.09 (apparent t, <i>J</i> = 8.7 Hz, 2H), 6.86 (d, <i>J</i> = 5.1 Hz, 1H), 4.96 (tt, <i>J</i> = 11.4, 4.0 Hz, 1H), 4.30 (br, 2H), 2.89 (br, 2H), 2.61 (s, 3H), 2.42–2.28 (m, 2H), 2.23–2.06 (m, 2H), 1.50 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 173.9, 163.1 (d, <i>J</i><sub>C–F</sub> = 248.9 Hz), 158.0, 155.3, 154.6, 146.3, 130.3 (d, <i>J</i><sub>C–F</sub> = 8.3 Hz), 129.3, 126.4 (d, <i>J</i><sub>C–F</sub> = 3.3 Hz), 116.5, 116.0 (d, <i>J</i><sub>C–F</sub> = 21.8 Hz), 80.0, 57.8, 43.0 (br), 32.3, 28.4, 14.1. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ −112.2. IR (NaCl, thin film, cm<sup>–1</sup>): 2976, 2930, 2863, 1692, 1550, 1506, 1420, 1349, 1243, 1158, 997, 913, 841, 732. HRMS (ESI): calcd for C<sub>23</sub>H<sub>27</sub>FN<sub>6</sub>NaO<sub>2</sub>S<sup>+</sup>, (M+Na)<sup>+</sup> 493.1792, found 493.1774.</div></div><div id="sec5_2_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> Synthesis of <i>tert</i>-Butyl 4-(4-(((<i>tert</i>-Butyldimethylsilyl)oxy)methyl)-5-(2-(methylthio)pyrimidin-4-yl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>29</b>)</h5><div class="NLM_p last">A solution of triazole <b>26</b> (830 mg, 2.1 mmol) in THF (15 mL) was cooled in a dry ice/acetone bath. Then <i>n</i>-BuLi (1.1 mL, 2.5 M in hexanes, 2.7 mmol) was added dropwise. After 10 min, a freshly prepared solution of ZnCl<sub>2</sub> (370 mg, 2.7 mmol) in THF (5 mL) was added dropwise. After 10 min, a freshly prepared solution of SPhos Pd G3 (71 mg, 91 μmol) in THF (5 mL) was added followed by chloro-pyrimidine (0.36 mL, 3.1 mmol). The reaction was heated to 60 °C. After 18 h, the reaction mixture was diluted with water and extracted with ethyl acetate. The combined organic phases were washed with brine, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure. Final purification by column chromatography (0–80% EtOAc in hexanes) yielded the product as a white solid (990 mg, 92%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.64 (d, <i>J</i> = 5.2 Hz, 1H), 7.71 (d, <i>J</i> = 5.1 Hz, 1H), 5.24 (tt, <i>J</i> = 11.3, 4.0 Hz, 1H), 4.83 (s, 2H), 4.28 (br, 2H), 2.88 (br, 2H), 2.58 (s, 3H), 2.28 (qd, <i>J</i> = 11.8, 4.1 Hz, 2H), 2.18–2.11 (m, 2H), 1.47 (s, 9H), 0.87 (d, <i>J</i> = 2.6 Hz, 9H), 0.09 (s, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 173.2, 158.4, 154.7, 154.6, 146.9, 131.6, 116.3, 80.0, 58.1, 57.1, 42.9, 32.4, 28.5, 25.9, 18.3, 14.2, −5.1. IR (NaCl, thin film, cm<sup>–1</sup>): 2929, 2856, 1697, 1542, 1419, 1157, 837. HRMS (ESI): calcd for C<sub>24</sub>H<sub>40</sub>N<sub>6</sub>NaO<sub>3</sub>SSi<sup>+</sup>, (M+Na)<sup>+</sup> 543.2544, found 543.2529.</div></div><div id="sec5_2_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> Synthesis of <i>tert</i>-Butyl 4-(4-(((<i>tert</i>-Butyldimethylsilyl)oxy)methyl)-5-(2-(methylthio)pyrimidin-4-yl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>30</b>)</h5><div class="NLM_p last">A variation of the general coupling procedure was used. After the addition of SPhos Pd G3 and electrophile <b>27</b>, the reaction was heated to 60 °C for 3.5 h. Final purification by column chromatography (0–60% IPA in hexanes) yielded the product (210 mg, 52%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.65 (d, <i>J</i> = 5.1 Hz, 1H), 7.11 (d, <i>J</i> = 5.1 Hz, 1H), 5.11 (tt, <i>J</i> = 11.3, 3.9 Hz, 1H), 4.27 (br, 2H), 2.86 (br, 2H), 2.58 (s, 3H), 2.49 (s, 3H), 2.26 (d, <i>J</i> = 10.2 Hz, 2H), 2.10 (d, <i>J</i> = 12.4 Hz, 2H), 1.47 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 173.5, 158.1, 155.0, 154.6, 143.8, 129.6, 115.1, 79.9, 57.8, 42.6 (br), 32.2, 28.4, 14.1, 12.1. IR (NaCl, thin film, cm<sup>–1</sup>): 2974, 2930, 2864, 1691, 1548, 1422, 1276, 1162, 1006. HRMS (ESI): calcd for C<sub>18</sub>H<sub>26</sub>N<sub>6</sub>NaO<sub>2</sub>S<sup>+</sup>, (M+Na)<sup>+</sup> 413.1730, found 413.1732.</div></div><div id="sec5_2_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> General Oxidation Procedure and Synthesis of <i>tert</i>-Butyl 4-(4-(4-Fluorophenyl)-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>31</b>)</h5><div class="NLM_p last">To a solution of thioether <b>28</b> (350 mg, 0.74 mmol) in THF (10 mL) cooled in an ice bath, a solution of oxone (690 mg, 2.3 mmol) in water (3 mL) was added dropwise. The reaction was sealed under air and warmed to rt. After 20 h, the reaction was diluted with ice water and extracted with ethyl acetate. The combined organic phase was washed with brine, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure. This afforded sulfone <b>31</b> (330 mg, 66%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.80 (d, <i>J</i> = 5.3 Hz, 1H), 7.50 (apparent dd, <i>J</i> = 8.8, 5.2 Hz, 2H), 7.41 (d, <i>J</i> = 5.3 Hz, 1H), 7.15 (apparent t, <i>J</i> = 8.6 Hz, 2H), 5.15 (tt, <i>J</i> = 10.6, 4.6 Hz, 1H), 4.33 (br, 2H), 3.41 (s, 3H), 2.97 (br, 2H), 2.44–2.16 (m, 4H), 1.48 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 166.6, 163.4 (d, <i>J</i><sub>C–F</sub> = 250.2 Hz), 158.9, 157.0, 154.6, 147.7, 130.6 (d, <i>J</i><sub>C–F</sub> = 8.3 Hz), 128.0, 126.0 (d, <i>J</i><sub>C–F</sub> = 3.3 Hz), 122.8, 116.4 (d, <i>J</i><sub>C–F</sub> = 21.8 Hz), 79.9, 59.2, 43.3 (br), 39.1, 32.4, 28.4. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ −110.9. IR (NaCl, thin film, cm<sup>–1</sup>): 2977, 2929, 1685, 1578, 1507, 1420, 1324, 1244, 1159, 1135, 843. HRMS (ESI): calcd for C<sub>23</sub>H<sub>27</sub>FN<sub>6</sub>NaO<sub>4</sub>S<sup>+</sup>, (M+Na)<sup>+</sup> 525.1691, found 525.1688.</div></div><div id="sec5_2_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> Synthesis of <i>tert</i>-Butyl 4-(4-(((<i>tert</i>-Butyldimethylsilyl)oxy)methyl)-5-(2-(methylthio)pyrimidin-4-yl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>32</b>)</h5><div class="NLM_p last">To a solution of thioether <b>29</b> (290 mg, 0.56 mmol) in THF (10 mL) cooled in an ice bath, a solution of oxone (400 mg, 1.3 mmol) in H<sub>2</sub>O (4 mL) was added dropwise. The reaction was sealed under air and warmed to room temperature. After 20 h, the reaction was diluted with ice water and extracted with ethyl acetate. The combined organic phases were washed with brine, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure. The crude material was dissolved in DCM (5 mL). To this solution, imidazole (65 mg, 0.95 mmol) was added followed by TBSCl (130 mg, 0.85 mmol). The reaction was sealed under air at room temperature. After 3 h, the reaction was quenched by the addition of water and extracted with DCM. The combined organic phases were dried over anhydrous sodium sulfate, filtered, and concentrated under reduced pressure. This afforded sulfone <b>32</b> (275 mg, 89%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.07 (d, <i>J</i> = 5.2 Hz, 1H), 8.35 (d, <i>J</i> = 5.3, 1H), 5.40–5.27 (m, 1H), 4.91 (br, 2H), 4.31 (br, 2H), 3.42 (s, 3H), 3.01 (br, 2H), 2.26 (br, 4H), 1.49 (s, 9H), 0.92 (s, 9H), 0.10 (s, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 166.1, 159.7, 156.0, 154.6, 148.0, 130.4, 123.2, 80.0, 59.4, 55.9, 43.3, 39.2, 32.4, 30.3, 28.4, 25.7, −3.6. IR (NaCl, thin film, cm<sup>–1</sup>): 2977, 2932, 1684, 1582, 1429, 1320, 1161, 1006, 736. HRMS (ESI): calcd for C<sub>24</sub>H<sub>40</sub>N<sub>6</sub>NaO<sub>5</sub>SSi<sup>+</sup>, (M+Na)<sup>+</sup> 575.2442, found 575.2430.</div></div><div id="sec5_2_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> Synthesis of <i>tert</i>-Butyl 4-(4-Methyl-5-(2-(methylsulfonyl)pyrimidin-4-yl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>33</b>)</h5><div class="NLM_p last">The <a class="ref internalNav" href="#sec5_2_1_8" aria-label="General Oxidation Procedure">General Oxidation Procedure</a> was used with thioether <b>30</b>, and the sulfone was isolated in quantitative yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.02 (d, <i>J</i> = 5.3 Hz, 1H), 7.72 (d, <i>J</i> = 5.3 Hz, 1H), 5.35–5.06 (m, 1H), 4.26 (br, 2H), 3.37 (s, 3H), 2.94 (br, 2H), 2.58 (s, 3H), 2.36–2.09 (m, 4H), 1.45 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.3, 159.3, 156.7, 154.6, 145.1, 128.4, 121.5, 79.8, 59.1, 42.8 (br), 39.2, 32.3, 28.4, 12.6. IR (NaCl, thin film, cm<sup>–1</sup>): 2976, 2931, 1687, 1580, 1452, 1426, 1321, 1249, 1163, 1007. HRMS (ESI): calcd for C<sub>18</sub>H<sub>26</sub>N<sub>6</sub>NaO<sub>4</sub>S<sup>+</sup>, (M+Na)<sup>+</sup> 445.1628, found 445.1623.</div></div><div id="sec5_2_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> General Nucleophilic-Aromatic-Substitution Procedure and Synthesis of <i>tert</i>-Butyl 4-(5-(2-((3,5-Dimethylphenyl)amino)pyrimidin-4-yl)-4-(4-fluorophenyl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>34</b>)</h5><div class="NLM_p last">To a vial containing sulfone <b>31</b> (85 mg, 0.170 mmol), 2,5-dimethylaniline (0.21 mL, 1.7 mmol) was added neat. The vial was sealed under air and heated to 100 °C. After 3 h, the reaction mixture was cooled to room temperature and diluted with THF (0.5 mL), and then Boc<sub>2</sub>O (0.10 mL, 0.44 mmol) was added. After an additional 2 h at room temperature, the reaction was concentrated under reduced pressure. Purification by column chromatography (0–60% IPA in hexanes with 1% TEA) yielded semipure material. The purified material was recrystallized in DCM and hexanes to afford the desired product, <b>34</b> (54 mg, 59%), as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.41 (d, <i>J</i> = 5.0 Hz, 1H), 7.59 (apparent dd, <i>J</i> = 8.7, 5.4 Hz, 2H), 7.36 (s, 1H), 7.20 (s, 2H), 7.10 (apparent t, <i>J</i> = 8.7 Hz, 2H), 6.80 (s, 1H), 6.61 (d, <i>J</i> = 5.0 Hz, 1H), 4.93 (tt, <i>J</i> = 11.4, 4.0 Hz, 1H), 4.16 (br, 2H), 2.58 (br, 2H), 2.33 (s, 6H), 2.32–2.19 (m, 2H), 2.09–2.02 (m, 2H), 1.49 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 163.0 (d, <i>J</i><sub>C–F</sub> = 248.4 Hz), 160.7, 159.3, 156.0, 154.5, 145.7, 138.7, 138.3, 130.2 (d, <i>J</i><sub>C–F</sub> = 8.2 Hz), 130.0, 126.6 (d, <i>J</i><sub>C–F</sub> = 3.1 Hz), 125.7, 118.4, 115.8 (d, <i>J</i><sub>C–F</sub> = 21.7 Hz), 112.9, 79.9, 57.3, 42.7 (br), 32.2, 28.4, 21.4. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ −112.7. IR (NaCl, thin film, cm<sup>–1</sup>): 2974, 1691, 1536, 1430, 1160. HRMS (ESI): calcd for C<sub>30</sub>H<sub>34</sub>FN<sub>7</sub>NaO<sub>2</sub><sup>+</sup>, (M+Na)<sup>+</sup> 566.2650, found 566.2643.</div></div><div id="sec5_2_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> General Boc-Deprotection Procedure and Synthesis of <i>N</i>-(3,5-Dimethylphenyl)-4-(4-(4-fluorophenyl)-1-(piperidin-4-yl)-1<i>H</i>-1,2,3-triazol-5-yl)pyrimidin-2-amine (<b>VII</b>)</h5><div class="NLM_p last">To a solution of Boc-protected amine <b>34</b> (18 mg, 32 μmol) in DCM (0.5 mL) at room temperature was added TFA (0.5 mL). After 1 h, the product was concentrated under reduced pressure. This afforded the product quantitatively as a yellow solid. The ratio of TFA and amine was determined by <sup>19</sup>F NMR. <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.44 (d, <i>J</i> = 5.0 Hz, 1H), 7.60 (apparent dd, <i>J</i> = 8.8, 5.3 Hz, 2H), 7.27–7.18 (m, 4H), 6.79 (s, 1H), 6.68 (d, <i>J</i> = 5.0 Hz, 1H), 5.16 (tt, <i>J</i> = 10.7, 4.3 Hz, 1H), 3.52–3.45 (m, 2H), 2.96–2.86 (m, 2H), 2.55–2.36 (m, 4H), 2.31 (s, 6H). <sup>13</sup>C NMR (126 MHz, MeOD) δ 163.2 (d, <i>J</i><sub>C–F</sub> = 247.6 Hz), 160.8, 159.0, 155.2, 145.4, 139.0, 138.2, 130.9, 130.1 (d, <i>J</i><sub>C–F</sub> = 8.4 Hz), 126.2, 124.9, 118.9, 115.5 (d, <i>J</i><sub>C–F</sub> = 22.1 Hz), 112.0, 54.0, 42.7, 28.8, 20.1. <sup>19</sup>F NMR (376 MHz, MeOD) δ −77.5 (9H), −114.3 (1H). IR (NaCl, thin film, cm<sup>–1</sup>): 2916, 2850, 1676, 1624, 1592, 1510, 1457, 1198, 1186, 1160, 1143. HRMS (ESI): calcd for C<sub>25</sub>H<sub>26</sub>FN<sub>7</sub>Na<sup>+</sup>, (M+Na)<sup>+</sup> 466.2126, found 466.2117. HPLC Purity: 96.3%.</div></div><div id="sec5_2_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> Synthesis of <i>tert</i>-Butyl 4-(4-(((<i>tert</i>-Butyldimethylsilyl)oxy)methyl)-5-(2-((3,5-dimethylphenyl)amino)pyrimidin-4-yl)-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>35</b>)</h5><div class="NLM_p last">To a solution of 3,5-dimethylaniline (0.02 mL, 0.2 mmol) in THF (2 mL) cooled in a dry ice/acetone bath was added NaHMDS (0.28 mL, 1 M in THF, 0.28 mmol). After 5 min, a solution of sulfone <b>32</b> (62 mg, 0.11 mmol) in THF (2 mL) was added dropwise. After an additional 30 min, the reaction was quenched with water and extracted with ethyl acetate. The combined organic phases were washed with brine, dried over anhydrous sodium sulfate, filtered, and concentrated under reduced pressure. Purification by column chromatography (0–55% EtOAc in hexanes) yielded the desired product, <b>35</b> (39.1 mg, 59%), as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.55 (d, <i>J</i> = 5.0 Hz, 1H), 7.40 (d, <i>J</i> = 5.0 Hz, 1H), 7.29 (s, 1H), 7.14 (s, 2H), 6.79 (s, 1H), 5.16 (tt, <i>J</i> = 11.3, 4.0 Hz, 1H), 4.87 (s, 2H), 4.11 (br, 2H), 2.51 (br, 2H), 2.32 (s, 6H), 2.24–2.15 (m, 2H), 2.07–1.96 (m, 2H), 1.48 (s, 9H), 0.91 (s, 9H), 0.13 (s, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 160.3, 159.4, 155.2, 154.5, 146.3, 138.6, 138.3, 132.3, 125.8, 118.9, 112.6, 79.8, 57.4, 56.9, 42.2, 32.1, 28.4, 25.9, 21.4, 18.3, −5.1. IR (NaCl, thin film, cm<sup>–1</sup>): 2927, 2854, 1688, 1596, 1453, 1330, 1250, 1158. HRMS (ESI): calcd for C<sub>31</sub>H<sub>47</sub>N<sub>7</sub>NaO<sub>3</sub>Si<sup>+</sup>, (M+Na)<sup>+</sup> 616.3402, found 616.3394.</div></div><div id="sec5_2_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> Synthesis of (5-(2-((3,5-Dimethylphenyl)amino)pyrimidin-4-yl)-1-(piperidin-4-yl)-1<i>H</i>-1,2,3-triazol-4-yl)methanol (<b>VIII</b>)</h5><div class="NLM_p last">To a solution of Boc-protected amine <b>35</b> (39 mg, 66 μmol) in DCM (0.5 mL) at room temperature was added TFA (0.5 mL). After 3 h, the solution was heated to 40 °C. After 1 h, the reaction was concentrated under reduced pressure. Analysis by <sup>19</sup>F NMR with 2,2,2-trifluoroethanol as the standard indicated the formation of a mono-TFA salt in quantitative yield. <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.60 (d, <i>J</i> = 5.0 Hz, 1H), 7.31 (d, <i>J</i> = 5.0 Hz, 1H), 7.19 (s, 1H), 6.79 (s, 2H), 5.36 (tt, <i>J</i> = 10.8, 4.2 Hz, 1H), 4.79 (s, 2H), 3.44 (d, <i>J</i> = 13.2 Hz, 2H), 2.91–2.75 (m, 2H), 2.48–2.32 (m, 4H), 2.31 (s, 6H). <sup>13</sup>C NMR (126 MHz, MeOD) δ 160.8, 159.6, 154.3, 146.1, 139.1, 138.1, 132.6, 124.9, 119.3, 111.6, 54.4, 54.1, 42.7, 28.8, 20.1. IR (NaCl, thin film, cm<sup>–1</sup>): 2932, 2859, 1679, 1627, 1595, 1463, 1403, 1190, 1139, 1076. HRMS (ESI): calcd for C<sub>20</sub>H<sub>25</sub>N<sub>7</sub>NaO<sup>+</sup>, (M+Na)<sup>+</sup> 402.2013, found 402.2022. HPLC Purity: 96.9%.</div></div><div id="sec5_2_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> Synthesis of <i>tert</i>-Butyl 4-(5-(2-((3,5-Dimethylphenyl)amino)pyrimidin-4-yl)-4-methyl-1<i>H</i>-1,2,3-triazol-1-yl)piperidine-1-carboxylate (<b>36</b>)</h5><div class="NLM_p last">A variation of the <a class="ref internalNav" href="#sec5_2_1_11" aria-label="General SNAr Procedure">General SNAr Procedure</a> was used with sulfone <b>33</b>. The product was purified by prep TLC (20% IPA in hexanes), and the product was isolated in 68% yield. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.54 (d, <i>J</i> = 5.0 Hz, 1H), 7.57 (s, 1H), 7.15 (s, 2H), 6.83 (d, <i>J</i> = 5.0 Hz, 1H), 6.78 (s, 1H), 5.06 (tt, <i>J</i> = 11.4, 4.0 Hz, 1H), 4.11 (s, 2H), 2.64–2.38 (m, 5H), 2.31 (s, 6H), 2.24–2.13 (m, 2H), 2.06–1.91 (m, 2H), 1.47 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 160.5, 159.3, 155.7, 154.5, 143.3, 138.6, 138.4, 130.4, 125.7, 118.9, 111.5, 79.8, 57.2, 42.3 (br), 32.1, 28.4, 21.4, 12.0. IR (NaCl, thin film, cm<sup>–1</sup>): 2974, 2929, 1693, 1567, 1537, 1427, 1366, 1164. HRMS (ESI): calcd for C<sub>25</sub>H<sub>33</sub>N<sub>7</sub>NaO<sub>2</sub><sup>+</sup>, (M+Na)<sup>+</sup> 486.2588, found 486.2574.</div></div><div id="sec5_2_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> Synthesis of <i>N</i>-(3,5-Dimethylphenyl)-4-(4-methyl-1-(piperidin-4-yl)-1<i>H</i>-1,2,3-triazol-5-yl)pyrimidin-2-amine (<b>IX</b>)</h5><div class="NLM_p last">The <a class="ref internalNav" href="#sec5_2_1_12" aria-label="General Boc-Deprotection Procedure">General Boc-Deprotection Procedure</a> was used with Boc-protected amine <b>36</b>, and the product was isolated in quantitative yield. Analysis by <sup>19</sup>F NMR with 2,2,2-trifluoroethanol as the standard indicated the formation of a bis-TFA salt in quantitative yield. <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.57 (d, <i>J</i> = 5.3 Hz, 1H), 7.18 (s, 2H), 7.07 (d, <i>J</i> = 5.3 Hz, 1H), 6.83 (s, 1H), 5.30 (tt, <i>J</i> = 10.7, 4.2 Hz, 1H), 3.43 (d, <i>J</i> = 13.3 Hz, 2H), 2.85–2.75 (m, 2H), 2.53 (s, 3H), 2.35–2.32 (m, 2H), 2.31 (s, 6H). <sup>13</sup>C NMR (126 MHz, MeOD) δ 159.8, 157.8, 155.8, 143.4, 138.5, 138.4, 130.8, 125.5, 119.7, 110.8, 54.2, 42.7, 28.7, 20.1, 10.4. IR (NaCl, thin film, cm<sup>–1</sup>): 2925, 2852, 1681, 1628, 1593, 1513, 1460, 1198, 1141. HRMS (ESI): calcd for C<sub>20</sub>H<sub>25</sub>N<sub>7</sub>Na<sup>+</sup>, (M+Na)<sup>+</sup> 386.2064, found 386.2053. HPLC Purity: 96.1%.</div></div><div id="sec5_2_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> Synthesis of (<i>S</i>)-<i>N</i>-(4-Aminobutyl)-2-(4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamide (<b>39</b>)</h5><div class="NLM_p last">Commercially acquired (+)-JQ1 (Sigma, SML0974) was hydrolyzed to acid <b>37</b>; conjugated to the <i>N</i>-Boc-1,4-butanediamine linker (Sigma, 15404); and deprotected to yield the free amine, <b>39</b>, according to previously reported procedures.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div></div><div id="sec5_2_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> Synthesis of (<i>S</i>)-5-((4-(2-(4-(4-Chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetamido)butyl)carbamoyl)-2-(6-hydroxy-3-oxo-3<i>H</i>-xanthen-9-yl)benzoic Acid (<b>40</b>)</h5><div class="NLM_p">To a solution of <b>39</b> (8 mg, 12 μmol) in DCM (1 mL) was added NHS–fluorescein (6 mg, 13 mmol; Thermo Scientific, 46409). The reaction mixture was stirred overnight at room temperature and purified using reverse-phase chromatography (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). HRMS (ESI): calcd for C<sub>44</sub>H<sub>37</sub>ClN<sub>6</sub>O<sub>7</sub>S (M+H)<sup>+</sup> 829.3250, found 829.3762; C<sub>44</sub>H<sub>37</sub>ClN<sub>6</sub>NaO<sub>7</sub>S (M+Na)<sup>+</sup> 851.2031, found 851.3621.</div><figure id="sch6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0011.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Fluorescein–JQ1 Tracer (<b>40</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Protein Expression</h3><div class="NLM_p last">Expression of BRD4(1), BRD4(2), H437D BRD4(2), BRD4(1 + 2), BRDT(1), BRDT(2), BPTF, and 5FW-pfGCN5 in either unlabeled or 5FW-labeled forms was based on established methods using <i>Escherichia coli</i> Bl21(DE3) + pRARE strains<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and is briefly described here. To express the labeled protein, a secondary culture in LB media was grown to an OD at 600 nm of 0.6 was reached; this was followed by harvesting. Cells were resuspended in defined media containing 5-fluoroindole (60 mg/L) in place of tryptophan. The resuspended <i>E. coli</i> were incubated at 37 °C with shaking for 1.5 h. This was followed by cooling to 20 °C and media temperature equilibration for 30 min. Protein expression was induced with 1 mM IPTG overnight (14–16 h) at 20 °C. The cells were harvested and stored at −20 °C. Cell pellets were thawed at room temperature; this was followed by the addition of lysis buffer (50 mM potassium phosphate, pH 7.4, 300 mM NaCl) containing PMSF (5 mM) and Halt protease-inhibitor cocktail, and proteins were purified using Ni-affinity chromatography. Purity of proteins was assessed by SDS-PAGE. Fluorinated amino acid incorporation efficiency in proteins was measured by LC-MS on a Waters Synapt G2 UPLC/QTOF-MS. Protein concentrations were determined via absorbance at 280 nm.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Fluorescence Anisotropy</h3><div class="NLM_p">Fluorescence-anisotropy experiments were carried out in 50 mM HEPES, 100 mM NaCl, and 4 mM CHAPS (pH 7.4) in 384-well plates (Corning 4511). Stock solutions (25 μM) of fluorescent tracer in DMSO were diluted to 15 nM for these experiments. Plates were read on a Tecan Infinity 500 with an excitation wavelength at 485 nm and emission at 535 nm. For direct-binding experiments, the protein was serially diluted across the plate, and the resulting anisotropy values were fit using <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a> in GraphPad Prism, where <i>b</i> and <i>c</i> are the maximum and minimum anisotropy values, respectively; <i>a</i> is the concentration of fluorescent tracer; <i>x</i> is the concentration of protein; and <i>y</i> is the observed anisotropy value.<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_m001.gif" alt="" id="_i63" /></img><span class="labelSpan">(1)</span></span></div><div class="NLM_p last">The protein concentrations for the competition experiments were determined from protein concentrations in the direct-binding experiments at which the fluorescent tracer is 80% bound. Stock solutions of inhibitor molecules, which were 50 mM in DMSO, were serially diluted from 250 μM to subnanomolar concentrations, keeping the concentrations of protein, tracer, and other components constant. Anisotropy values were fit using GraphPad Prism’s log(inhibitor) versus response (four parameters) function. The IC<sub>50</sub> values are reported as the means ± SEM, as determined from three or more independent experiments. Direct binding with fluorescent tracer and self-competition experiments with JQ1 or BI-2536 were carried out prior to each set of competition experiments to assess protein quality and assay stability and for determination of percent bound for those compounds that did not show complete inhibition.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Isothermal-Titration Calorimetry</h3><div class="NLM_p last">To measure the interaction parameters with BRD4(1), we used a GE Auto ITC200; 35 μM BRD4(1) and varying concentrations of compounds <b>IV</b> and <b>V</b> were used for all the experiments. BRD4(1) solution was loaded in the sample cell of the calorimeter and titrated with a solution of <b>IV</b> or <b>V</b>. Potassium phosphate (50 mM) at pH 7.5 containing 150 mM NaCl was used to dilute the stock solutions of both compounds and protein. Diluted solutions of protein were concentrated in a 3 kDa cutoff centrifuge filter. Compound <b>IV</b> and <b>V</b> were diluted from 50 mM stock solutions in DMSO into the same buffer as protein to the required concentrations, briefly sonicated, and used for the titration. Titrations were carried out by injecting 1 μL followed by 20 identical injections of 2 μL with a releasing period of 5 s per injection every 300 s. The control heat of dilution was measured by independent ligand titration into buffer and was subtracted from the protein–ligand-binding-experimental data. Data were analyzed by Origin software to calculate the enthalpy of binding (Δ<i>H</i>) and association constants (<i>K</i><sub>a</sub>). Heat liberated by each injection was integrated (Δ<i>H</i>) and plotted against the molar ratios of ligand and protein.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Protein Observed Fluorine NMR</h3><div class="NLM_p last"><sup>19</sup>F NMR spectra were acquired at 470 MHz on a Bruker 500 spectrometer with a 5 mm Prodigy TCI Cryoprobe without proton decoupling. Samples containing 40–50 μM bromodomains were dosed in 50 mM TRIS, 100 mM NaCl, and 5% D<sub>2</sub>O (pH 7.4) with varying ligand concentrations for binding assays and normalized to 0.05% DMSO. Spectra were referenced to trifluoroacetate (−76.55 ppm). Measurement parameters included a relaxation-delay time of 0.7 s and a 90° flip angle. An acquisition time of 0.05 s was used for all experiments. A spectral width of at least 10 ppm was used for all spectra, and the screening experiments used 400 transients.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Site-Directed Mutagenesis</h3><div class="NLM_p last">Site-directed mutagenesis of H437D on BRD4(2) was conducted using previously reported transfer-PCR procedures in 50 μL reaction mixtures.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Reaction mixtures contained Phusion High-Fidelity Master Mix (NEB), template DNA, 50 nM concentrations of each of the primers: the T7-forward primer (5′-TAATACGACTCACTATAGGG-3′) and the reverse-complement primer (5′-CAACCCTCCTGACG*ATGAGGTGGTGGC-3′, where * indicates the site of mutation). Successful mutagenesis was confirmed by DNA sequencing.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> X-ray Crystallography</h3><div class="NLM_p last">Crystals of BRD4(1) were grown in the presence of 1 mM ligand and 10% (v/v) DMSO from vapor-diffusion hanging drops with 0.1 M (NH4)<sub>2</sub>SO<sub>4</sub>, 0.05 M Tris (pH 8.5), and 12.5% (w/v) PEG 3350 as described previously.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Crystals were flash-frozen in a stream of nitrogen gas, and X-ray-diffraction data were recorded at −180 °C at beamline X11 of the Advanced Photon Source, Argonne National Laboratories. Data were reduced and scaled with XDS;<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> PHENIX<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> was employed for phasing and refinement, and model building was performed using Coot.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> All structures were solved by molecular replacement using PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7B">4O7B</a><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> as the search model. Initial models for the small-molecule ligands were generated using MarvinSketch (ChemAxon) with ligand restraints from eLBOW of the PHENIX suite. Figures were prepared using PyMOL (Schrödinger, LLC).</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Cell Culture</h3><div class="NLM_p last">MM.1S and A549-NF-κB-luc cells were grown in a humidified 5% CO<sub>2</sub> environment at 37 °C. A549-NF-κB-luc cells<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> were cultured in DMEM media (Gibco) supplemented with 10% fetal-bovine serum (FBS, Cellgro), penicillin (50 IU/mL, Cellgro), streptomycin (50 μg/mL, Cellgro), and hygromycin (100 μg/mL, Roche). Cells were maintained at a density below 2 × 10<sup>6</sup> cells per milliliter and were dissociated from adherent plates in 0.25% trypsin/EDTA (Gibco) with 2 min incubation times. MM.1S cells were grown in RPMI-1640 containing 10% FBS. Cell-line authenticity was verified using the short-tandem-repeat (STR)-profiling service provided by ATCC.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Western Blotting</h3><div class="NLM_p last">MM.1S cells were seeded in six-well plates at a density of 10<sup>6</sup> cells per well and exposed for 6 h to increasing compound concentrations ranging from 0.1 to 30 μM. Cells were harvested by low-speed centrifugation at 300<i>g</i> for 5 min and washed once with PBS. Cells were lysed in 75 μL of CelLytic M (Sigma-Aldrich) supplemented with 1× Halt Protease and Phosphatase Inhibitor Cocktail and 5 mM EDTA (ThermoFisher Scientific). Protein concentrations were assessed with the Bio-Rad Protein Assay Dye Reagent, and samples were mixed with 1/3 vol of 4× SDS sample buffer (Bio-Rad) containing 10% β-mercaptoethanol; this was followed by 12.5% SDS-PAGE. Proteins were transferred to PVDF membranes for 3 h on ice using Bio-Rad Mini Trans-Blot Cells. Membranes were incubated with TBS-T containing 5% nonfat dry milk for 1 h at room temperature and then incubated for 16 h with primary antibodies (c-Myc, Cell Signaling Technology, #5605, diluted 1:4000 in TBS-T containing 5% BSA; GAPDH, BioLegend #919501, diluted 1:50 000 in TBS-T containing 5% nonfat dry milk) at 4 °C. After the membranes were washed three times with TBS-T, they were incubated with 1000-fold-diluted HRP-conjugated secondary antibodies from Jackson ImmunoResearch (goat anti-rabbit-IgG, #111-035-003 and goat anti-mouse-IgG, #115-035-003). Membranes were again washed three times with TBS-T and then exposed to SignalFire ECL Reagent (Cell Signaling Technology) for 1 min; this was followed by exposure to X-ray film.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Viability Assays</h3><div class="NLM_p last">MM.1S cells were seeded in 96-well plates at approximately 20 000 cells per well (0.1 mL) and incubated with increasing compound concentrations in the presence of 0.1% DMSO for 72 h with six replicates per concentration. After drug treatment, 15 μL of CellTiter Blue reagent (Promega) was added to each well; this was followed by vigorous orbital shaking for 5 min and incubation for 3 h at 37 °C. Plates were placed in a Wallac EnVision 2103 Multilabel Reader (PerkinElmer), and fluorescence was determined using excitation and emission filters of 570 and 615 nm, respectively. Dose–response data were analyzed with GraphPad Prism.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Cellular Thermal-Shift Assay</h3><div class="NLM_p">Assays were performed as previously reported by Jafari et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> in both the isothermal-dose–response- and melt-curve formats. Cells (1 × 10<sup>6</sup>) were treated with the desired amounts of compound in serum-supplemented DMEM, with DMSO concentrations normalized to 0.1% for all samples. Dosed cells in microcentrifuge tubes were incubated at 37 °C for 1 h with mild intermittent agitation. Upon completion of the incubation period, cells were thermally denatured for 3 min in a heated water bath and subsequently equilibrated at room temperature for a further 3 min. Following centrifugation and rinsing with PBS, cells were lysed over three freeze thaw cycles before soluble protein concentrations of supernatant were determined using the BCA protein assay kit (Pierce). Samples were normalized to the lowest total soluble-protein concentration among the grouped samples.</div><div class="NLM_p last">NuPAGE 4× LDS sample buffer and NuPAGE 10× sample reducing agent (Invitrogen) were added to normalized protein samples and denatured by heating at 90 °C for 5 min. Protein samples were separated on a gradient 4–12% SDS-PAGE gel (Invitrogen), transferred to a polyvinylidene difluoride membrane (Immobilon), and blocked by incubation in blocking buffer (0.05 g/mL BioRad nonfat milk in PBS) at 4 °C overnight. Proteins were detected by incubation with primary antibodies (BRD4 rabbit mAb, Cell Signaling Technologies, 13440; p38α rabbit mAb, Cell Signaling Technologies, 9212; β-actin mouse mAb, Santa Cruz Biotechnology, SC-47778) diluted 1:1000 in blocking buffer for 4 h at room temperature. The membrane was washed in MQ-water and incubated for 1 h at room temperature with HRP-conjugated antimouse (Thermo, A16066) or antirabbit secondary antibodies (Thermo, 65-6120) similarly diluted in blocking buffer. The membrane was again washed in MQ-water before the immunocomplexes were visualized upon addition of HRP substrate (Thermo) using the Odyssey FC imaging system (LICOR Biotech).</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> NF-κB-Reporter Assay</h3><div class="NLM_p">Reporter assays were performed as previously described by Hexum et al.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Briefly, compounds were serially diluted and dosed to A549-NF-κB-luc cells (final volume per well of 100 μL; final DMSO concentration of 0.5%) in 96-well plates. NF-κB was induced by the addition of TNF-α (15 ng/mL final concentration, diluted in PBS; Invitrogen) 30 min after dosing. Bright-Glo luciferase reagent (Promega) was added to each well (100 μL) after 8 h, and luminescence measurements were obtained on a Synergy plate reader. Cellular viability was measured in parallel by colorimetric-viability staining (Alamar Blue, Invitrogen). Cell viabilities of compound-treated cells are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S14</a>.</div><div class="NLM_p last">Data are reported as means ± SEM from three replicate biological experiments with three technical replicates per experiment. Mean and uncertainty in each percent-NF-κB-activity value were calculated by propagating the calculated means from individual biological replicates. Statistical analyses and plotting of results were performed on GraphPad Prism (v. 7.0).</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> IL-8 ELISA</h3><div class="NLM_p">A549-NF-κB-luc cells were seeded into 24-well plates at a cell density of 2 × 10<sup>5</sup> cells per well. Cells were incubated with the compounds for 24 h and induced with TNF-α (10 ng/mL) 6 h after the initial dosing, resulting in an 18 h induction. Upon completion of the incubation period, cell media (500 μL) was harvested and centrifuged (10 000<i>g</i>, 5 min) before secreted-IL-8-protein levels were measured using a human IL-8 ELISA kit (Thermo Scientific, EH2IL8) following the vendor’s instructions. On a Synergy plate reader, the absorbance of each sample at 450 and 550 nm was collected following addition of the substrate stop solution. A standard curve was plotted in Microsoft Excel and <i>A</i><sub>450–550</sub> values of unknown samples were plotted against the standard curve to yield IL-8 protein concentrations in picograms per milliliter.</div><div class="NLM_p last">Data are reported as means ± SEM from three replicate biological experiments with three technical replicates per experiment. Mean and uncertainty in each IL-8-concentration value were calculated by propagating the calculated means from individual biological replicates. Statistical analyses and plotting of results were performed on GraphPad Prism (v. 7.0).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01248" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35171" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35171" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01248" class="ext-link">10.1021/acs.jmedchem.8b01248</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular-string files for the final compounds synthesized (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Compound-purity analysis, fluorescence anisotropy, affinity determination, bromodomain ALPHAScreen, PrOF NMR titrations, binding interactions of IV and V, bromodomain-selectivity profiling, kinase-affinity determination, kinase-selectivity profiling, cellular toxicity, inhibition of p38-MAPK activity, reporter assay on p38-MAPK inhibitors, refinement statistics, NMR spectra of final compounds, and NMR spectra of novel intermediates (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_001.csv">jm8b01248_si_001.csv (1.7 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf">jm8b01248_si_002.pdf (9.66 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">BRD4(1) bound to compound <b>IV</b>, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH7">6MH7</a>; BRD4(1) bound to compound <b>V</b>, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH1">6MH1</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b01248" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12068" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12068" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel A. Harki</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, University of Minnesota, 2231 6th Street SE, Minneapolis, Minnesota 55455, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5950-931X" title="Orcid link">http://orcid.org/0000-0001-5950-931X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#adc9ccc5ccdfc6c4edd8c0c383c8c9d8"><span class="__cf_email__" data-cfemail="6307020b0211080a23160e0d4d060716">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William C. K. Pomerantz</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, University of Minnesota, 2231 6th Street SE, Minneapolis, Minnesota 55455, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0163-4078" title="Orcid link">http://orcid.org/0000-0002-0163-4078</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e3948093a3968e8dcd868796"><span class="__cf_email__" data-cfemail="e1968291a1948c8fcf848594">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anand Divakaran</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, University of Minnesota, 2231 6th Street SE, Minneapolis, Minnesota 55455, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1782-6234" title="Orcid link">http://orcid.org/0000-0002-1782-6234</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Siva K. Talluri</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex M. Ayoub</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Neeraj K. Mishra</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huarui Cui</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John C. Widen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, University of Minnesota, 2231 6th Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Norbert Berndt</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Department, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jin-Yi Zhu</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Department, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Angela S. Carlson</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph J. Topczewski</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9921-5102" title="Orcid link">http://orcid.org/0000-0002-9921-5102</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ernst K. Schonbrunn</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Department, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3589-3510" title="Orcid link">http://orcid.org/0000-0002-3589-3510</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.D. and S.K.T. contributed equally. S.K.T. designed and synthesized the molecules with H.C., A.S.C.; A.M.A. and N.K.M. performed biophysical experiments; A.D., J.C.W., and N.B. designed and performed cellular experiments; J.Y.Z. performed crystallography; W.C.K.P., D.A.H, J.J.T., and E.K.S. oversaw the experimental and interpretation of data; and A.D. and W.C.K.P. wrote the manuscript. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3829-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the NSF (Grant CHE-1352019, W.C.K.P.), AHA (Grant 15SDG25710427, W.C.K.P.), NIGMS (Grant R35GM124718), and the University of Minnesota Masonic Cancer Center (Pre-R01 pilot grant, D.A.H. and W.C.K.P.). A.D. was supported by the NIH Chemistry–Biology Interface Training Grant at the University of Minnesota (5T32GM008700-18). Isothermal-titration calorimetry and nuclear-magnetic-resonance spectroscopy were carried out using instruments funded by NIH Shared Instrumentation Grants (S10-OD017982 and S10-OD011952, respectively). We also thank the Moffitt Chemical Biology Core for use of the protein crystallography facility (NCI Grant P30-CA076292). BI-BODIPY was a generous gift from Wei Zhang at the University of Massachusetts, Boston. We thank Prof. Brian Smith for providing the BRD4 Bromodomain construct BRD4 (1 + 2) spanning residues 38-460.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra terminal</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">BRD2</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 2</p></td></tr><tr><td class="NLM_term">BRD3</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 3</p></td></tr><tr><td class="NLM_term">BRDT</td><td class="NLM_def"><p class="first last">bromodomain testis-specific protein</p></td></tr><tr><td class="NLM_term">c-Myc</td><td class="NLM_def"><p class="first last">Myc proto-oncogene protein</p></td></tr><tr><td class="NLM_term">NF-κB</td><td class="NLM_def"><p class="first last">nuclear-factor κ-light-chain enhancer of activated B cells</p></td></tr><tr><td class="NLM_term">IL-8</td><td class="NLM_def"><p class="first last">interleukin-8</p></td></tr><tr><td class="NLM_term">K<sub>ac</sub></td><td class="NLM_def"><p class="first last">N-ε-acetylated lysine</p></td></tr><tr><td class="NLM_term">D1</td><td class="NLM_def"><p class="first last">N-terminal BET bromodomain</p></td></tr><tr><td class="NLM_term">D2</td><td class="NLM_def"><p class="first last">C-terminal BET bromodomain</p></td></tr><tr><td class="NLM_term">dTAG</td><td class="NLM_def"><p class="first last">degradation tag</p></td></tr><tr><td class="NLM_term">p38α</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase 14</p></td></tr><tr><td class="NLM_term">BPTF</td><td class="NLM_def"><p class="first last">nucleosome-remodeling-factor subunit BPTF</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential-scanning fluorimetry</p></td></tr><tr><td class="NLM_term">FA</td><td class="NLM_def"><p class="first last">fluorescence anisotropy</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal-titration calorimetry</p></td></tr><tr><td class="NLM_term">ALPHAScreen</td><td class="NLM_def"><p class="first last">amplified-luminescent-proximity-homogeneous-assay screen</p></td></tr><tr><td class="NLM_term">PrOF NMR</td><td class="NLM_def"><p class="first last">protein-observed fluorine NMR</p></td></tr><tr><td class="NLM_term">GCN5</td><td class="NLM_def"><p class="first last">histone acetyltransferase GCN5</p></td></tr><tr><td class="NLM_term">TRIM24</td><td class="NLM_def"><p class="first last">Transcription intermediary factor 1α</p></td></tr><tr><td class="NLM_term">SMARCA2</td><td class="NLM_def"><p class="first last">probable global transcription activator SNF2L2</p></td></tr><tr><td class="NLM_term">PCAF</td><td class="NLM_def"><p class="first last">histone acetyltransferase KAT2B</p></td></tr><tr><td class="NLM_term">p300</td><td class="NLM_def"><p class="first last">histone acetyltransferase p300</p></td></tr><tr><td class="NLM_term">SMARCA4</td><td class="NLM_def"><p class="first last">transcription activator BRG1</p></td></tr><tr><td class="NLM_term">MAP</td><td class="NLM_def"><p class="first last">mitogen-activated protein</p></td></tr><tr><td class="NLM_term">GAPDH</td><td class="NLM_def"><p class="first last">glyceraldehyde-3-phosphate dehydrogenase</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">MSK1</td><td class="NLM_def"><p class="first last">ribosomal protein S6 kinase α5</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved Förster resonance-energy transfer</p></td></tr><tr><td class="NLM_term">CSBP</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase 14</p></td></tr><tr><td class="NLM_term">DAD</td><td class="NLM_def"><p class="first last">diode-array detector</p></td></tr><tr><td class="NLM_term">CSA</td><td class="NLM_def"><p class="first last">camphorulfonic acid</p></td></tr><tr><td class="NLM_term">DMF-DMA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide-dimethylacetal</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">IPA</td><td class="NLM_def"><p class="first last">isopropyl alcohol</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">NaHMDS</td><td class="NLM_def"><p class="first last">sodium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term">NHS</td><td class="NLM_def"><p class="first last"><i>N</i>-hydroxysuccinimide</p></td></tr><tr><td class="NLM_term">IPTG</td><td class="NLM_def"><p class="first last">isopropyl β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside</p></td></tr><tr><td class="NLM_term">PMSF</td><td class="NLM_def"><p class="first last">phenylmethylsulfonyl fluoride</p></td></tr><tr><td class="NLM_term">SDS-PAGE</td><td class="NLM_def"><p class="first last">sodium dodecyl sulfate–polyacrylamide-gel electrophoresis</p></td></tr><tr><td class="NLM_term">QTOF-MS</td><td class="NLM_def"><p class="first last">quadrupole time-of-flight mass spectrometry</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">CHAPS</td><td class="NLM_def"><p class="first last">3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal-bovine serum</p></td></tr><tr><td class="NLM_term">EDT</td><td class="NLM_def"><p class="first last">ethanedithiol</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86461" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86461" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 58 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allfrey, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faulkner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirsky, A. E.</span></span> <span> </span><span class="NLM_article-title">Acetylation and Methylation of Histones and Their Possible Role in the Regulation of Rna Synthesis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">1938</span>),  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1073/pnas.51.5.786</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1073%2Fpnas.51.5.786" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1964&pages=786-794&issue=1938&author=V.+G.+Allfreyauthor=R.+Faulknerauthor=A.+E.+Mirsky&title=Acetylation+and+Methylation+of+Histones+and+Their+Possible+Role+in+the+Regulation+of+Rna+Synthesis&doi=10.1073%2Fpnas.51.5.786"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.51.5.786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.51.5.786%26sid%3Dliteratum%253Aachs%26aulast%3DAllfrey%26aufirst%3DV.%2BG.%26aulast%3DFaulkner%26aufirst%3DR.%26aulast%3DMirsky%26aufirst%3DA.%2BE.%26atitle%3DAcetylation%2520and%2520Methylation%2520of%2520Histones%2520and%2520Their%2520Possible%2520Role%2520in%2520the%2520Regulation%2520of%2520Rna%2520Synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1964%26volume%3D51%26issue%3D1938%26spage%3D786%26epage%3D794%26doi%3D10.1073%2Fpnas.51.5.786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of Chromatin by Histone Modifications</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">395</span>, <span class="refDoi"> DOI: 10.1038/cr.2011.22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fcr.2011.22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21321607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=381-395&issue=3&author=A.+J.+Bannisterauthor=T.+Kouzarides&title=Regulation+of+Chromatin+by+Histone+Modifications&doi=10.1038%2Fcr.2011.22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin by histone modifications</span></div><div class="casAuthors">Bannister, Andrew J.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-395</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Chromatin is not an inert structure, but rather an instructive DNA scaffold that can respond to external cues to regulate the many uses of DNA.  A principal component of chromatin that plays a key role in this regulation is the modification of histones.  There is an ever-growing list of these modifications and the complexity of their action is only just beginning to be understood.  However, it is clear that histone modifications play fundamental roles in most biol. processes that are involved in the manipulation and expression of DNA.  Here, we describe the known histone modifications, define where they are found genomically and discuss some of their functional consequences, concg. mostly on transcription where the majority of characterization has taken place.  Cell Research (2011) 21:381-395. doi:10.1038/cr.2011.22; published online 15 Feb. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ObWaVBHPorVg90H21EOLACvtfcHk0liMOdSmUPZu0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D&md5=bbdf55be25985810b90ba2efe6399f1b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fcr.2011.22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2011.22%26sid%3Dliteratum%253Aachs%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520Chromatin%2520by%2520Histone%2520Modifications%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26issue%3D3%26spage%3D381%26epage%3D395%26doi%3D10.1038%2Fcr.2011.22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Bromodomains as Therapeutic Targets</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e29</span>, <span class="refDoi"> DOI: 10.1017/S1462399411001992</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1017%2FS1462399411001992" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21933453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=e29&author=S.+Mullerauthor=P.+Filippakopoulosauthor=S.+Knapp&title=Bromodomains+as+Therapeutic+Targets&doi=10.1017%2FS1462399411001992"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains as therapeutic targets</span></div><div class="casAuthors">Muller, Susanne; Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Reviews in Molecular Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e29/1-e29/21</span>CODEN:
                <span class="NLM_cas:coden">ERMMFS</span>;
        ISSN:<span class="NLM_cas:issn">1462-3994</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Acetylation of lysine residues is a post-translational modification with broad relevance to cellular signalling and disease biol.  Enzymes that 'write' (histone acetyltransferases, HATs) and 'erase' (histone deacetylases, HDACs) acetylation sites are an area of extensive research in current drug development, but very few potent inhibitors that modulate the 'reading process' mediated by acetyl lysines have been described.  The principal readers of ε-N-acetyl lysine (Kac) marks are bromodomains (BRDs), which are a diverse family of evolutionary conserved protein-interaction modules.  The conserved BRD fold contains a deep, largely hydrophobic acetyl lysine binding site, which represents an attractive pocket for the development of small, pharmaceutically active mols.  Proteins that contain BRDs have been implicated in the development of a large variety of diseases.  Recently, two highly potent and selective inhibitors that target BRDs of the BET (bromodomains and extra-terminal) family provided compelling data supporting targeting of these BRDs in inflammation and in an aggressive type of squamous cell carcinoma.  It is likely that BRDs will emerge alongside HATs and HDACs as interesting targets for drug development for the large no. of diseases that are caused by aberrant acetylation of lysine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSuKacpsr6xLVg90H21EOLACvtfcHk0liMOdSmUPZu0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI&md5=dc9383afcbe6aedc265041c21213aa0b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1017%2FS1462399411001992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399411001992%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DBromodomains%2520as%2520Therapeutic%2520Targets%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2011%26volume%3D13%26spage%3De29%26doi%3D10.1017%2FS1462399411001992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Functions of Bromodomain-Containing Proteins and Their Roles in Homeostasis and Cancer</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nrm.2016.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnrm.2016.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=28053347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=246-262&issue=4&author=T.+Fujisawaauthor=P.+Filippakopoulos&title=Functions+of+Bromodomain-Containing+Proteins+and+Their+Roles+in+Homeostasis+and+Cancer&doi=10.1038%2Fnrm.2016.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span></div><div class="casAuthors">Fujisawa, Takao; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">246-262</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are evolutionarily conserved protein-protein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions and are present in most tissues.  BRDs selectively recognize and bind to acetylated Lys residues - particularly in histones - and thereby have important roles in the regulation of gene expression.  BRD-contg. proteins are frequently dysregulated in cancer, they participate in gene fusions that generate diverse, frequently oncogenic proteins, and many cancer-causing mutations have been mapped to the BRDs themselves.  Importantly, BRDs can be targeted by small-mol. inhibitors, which has stimulated many translational research projects that seek to attenuate the aberrant functions of BRD-contg. proteins in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-drLvQK1rvLVg90H21EOLACvtfcHk0lgiU0wwKV96Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D&md5=cbccdee8423bad7a426c9cea6566151f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.143%26sid%3Dliteratum%253Aachs%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DFunctions%2520of%2520Bromodomain-Containing%2520Proteins%2520and%2520Their%2520Roles%2520in%2520Homeostasis%2520and%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26issue%3D4%26spage%3D246%26epage%3D262%26doi%3D10.1038%2Fnrm.2016.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Bromodomain Interaction Module</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>586</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">2692</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2012.04.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.febslet.2012.04.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=22710155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=2692-2704&issue=17&author=P.+Filippakopoulosauthor=S.+Knapp&title=The+Bromodomain+Interaction+Module&doi=10.1016%2Fj.febslet.2012.04.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain interaction module</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2692-2704</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ε-N-acetylation of protein Lys residues (Kac) is one of the most abundant post-translation modifications (PTMs) in the human proteome.  In the nucleus, the acetylation of histones has been linked to transcriptional activation of genes but the functional consequences of most acetylation events and proteins recruited to these sites remains largely unknown.  Bromodomains (BRDs) are small helical interaction modules that specifically recognize acetylation sites in proteins.  BRDs have recently emerged as interesting targets for the development of specific protein interaction inhibitors, enabling a novel exiting strategy for the development of new therapies.  Here, the author provides an overview over sequence requirements of BRDs, known substrates, and the structural mechanisms of specific Kac recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7LaThygz_HrVg90H21EOLACvtfcHk0lgiU0wwKV96Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D&md5=c02dee22c330b1d5c1ba39579360ebc5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Bromodomain%2520Interaction%2520Module%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26issue%3D17%26spage%3D2692%26epage%3D2704%26doi%3D10.1016%2Fj.febslet.2012.04.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of BET Bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span> (<span class="NLM_issue">7327</span>),  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&issue=7327&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+Inhibition+of+BET+Bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lglVO5iBxgmfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520Inhibition%2520of%2520BET%2520Bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26issue%3D7327%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-F.</span></span> <span> </span><span class="NLM_article-title">Brd4 Coactivates Transcriptional Activation of NF-KappaB via Specific Binding to Acetylated RelA</span>. <i>Mol. Cel. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1375</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1128/MCB.01365-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=19103749" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&issue=5&author=B.+Huangauthor=X.-D.+Yangauthor=M.-M.+Zhouauthor=K.+Ozatoauthor=L.-F.+Chen&title=Brd4+Coactivates+Transcriptional+Activation+of+NF-KappaB+via+Specific+Binding+to+Acetylated+RelA&doi=10.1128%2FMCB.01365-08"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.-D.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DBrd4%2520Coactivates%2520Transcriptional%2520Activation%2520of%2520NF-KappaB%2520via%2520Specific%2520Binding%2520to%2520Acetylated%2520RelA%26jtitle%3DMol.%2520Cel.%2520Biol.%26date%3D2009%26volume%3D29%26issue%3D5%26spage%3D1375%26epage%3D1387%26doi%3D10.1128%2FMCB.01365-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovén, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.cell.2013.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=23582323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=320-334&issue=2&author=J.+Lov%C3%A9nauthor=H.+A.+Hokeauthor=C.+Y.+Linauthor=A.+Lauauthor=D.+A.+Orlandoauthor=C.+R.+Vakocauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Selective+Inhibition+of+Tumor+Oncogenes+by+Disruption+of+Super-Enhancers&doi=10.1016%2Fj.cell.2013.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span></div><div class="casAuthors">Loven, Jakob; Hoke, Heather A.; Lin, Charles Y.; Lau, Ashley; Orlando, David A.; Vakoc, Christopher R.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-334</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells.  Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM).  BRD4 and Mediator were found to co-occupy thousands of enhancers assocd. with active genes.  They also co-occupied a small set of exceptionally large super-enhancers assocd. with genes that feature prominently in MM biol., including the MYC oncogene.  Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC.  Super-enhancers were found at key oncogenic drivers in many other tumor cells.  These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-NqFYuGMqrVg90H21EOLACvtfcHk0lglVO5iBxgmfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D&md5=66b6ccb5b8a6039832910443fd8b4664</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DLov%25C3%25A9n%26aufirst%3DJ.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520Inhibition%2520of%2520Tumor%2520Oncogenes%2520by%2520Disruption%2520of%2520Super-Enhancers%26jtitle%3DCell%26date%3D2013%26volume%3D153%26issue%3D2%26spage%3D320%26epage%3D334%26doi%3D10.1016%2Fj.cell.2013.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&issue=1&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lglVO5iBxgmfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26issue%3D1%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Small-Molecule Bromodomain-Containing Protein 4 (BRD4) Inhibitor That Induces AMP-Activated Protein Kinase-Modulated Autophagy-Associated Cell Death in Breast Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">9990</span>– <span class="NLM_lpage">10012</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00275</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00275" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9990-10012&issue=24&author=L.+Ouyangauthor=L.+Zhangauthor=J.+Liuauthor=L.+Fuauthor=D.+Yaoauthor=Y.+Zhaoauthor=S.+Zhangauthor=G.+Wangauthor=G.+Heauthor=B.+Liu&title=Discovery+of+a+Small-Molecule+Bromodomain-Containing+Protein+4+%28BRD4%29+Inhibitor+That+Induces+AMP-Activated+Protein+Kinase-Modulated+Autophagy-Associated+Cell+Death+in+Breast+Cancer&doi=10.1021%2Facs.jmedchem.7b00275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Bromodomain-Containing Protein 4 (BRD4) Inhibitor That Induces AMP-Activated Protein Kinase-Modulated Autophagy-Associated Cell Death in Breast Cancer</span></div><div class="casAuthors">Ouyang, Liang; Zhang, Lan; Liu, Jie; Fu, Leilei; Yao, Dahong; Zhao, Yuqian; Zhang, Shouyue; Wang, Guan; He, Gu; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9990-10012</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Upon the basis of The Cancer Genome Atlas (TCGA) data set, the authors identified that several autophagy-related proteins such as AMP-activated protein kinase (AMPK) were remarkably downregulated in breast cancer.  Combined with coimmunopptn. assay, the authors demonstrated that BRD4 might interact with AMPK.  After analyses of the pharmacophore and WPF interaction optimization, the authors designed a small-mol. inhibitor of BRD4, 9f (FL-411 (2-(4-hydroxy-3,5-dimethylphenyl)-7-methyl-5,6,7,8-tetrahydropyrido[4',3':4,5]thieno[2,3-d]pyrimidin-4(3H)-one)) which was validated by cocrystal structure with BD1 of BRD4.  Subsequently, 9f was discovered to induce ATG5-dependent autophagy-assocd. cell death (ACD) by blocking BRD4-AMPK interaction and thus activating AMPK-mTOR-ULK1-modulated autophagic pathway in breast cancer cells.  Interestingly, the iTRAQ-based proteomics analyses revealed that 9f induced ACD pathways involved in HMGB1, VDAC1/2, and eEF2.  Moreover, 9f displayed a therapeutic potential on both breast cancer xenograft mouse and zebrafish models.  Together, these results demonstrate that a novel small-mol. inhibitor of BRD4 induces BRD4-AMPK-modulated ACD in breast cancer, which may provide a candidate drug for future cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Ocrq2wyy6bVg90H21EOLACvtfcHk0liX0DwPtMP5LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrvF&md5=ac15f7e108511ea207f2c2fcd299b4e5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00275%26sid%3Dliteratum%253Aachs%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520Small-Molecule%2520Bromodomain-Containing%2520Protein%25204%2520%2528BRD4%2529%2520Inhibitor%2520That%2520Induces%2520AMP-Activated%2520Protein%2520Kinase-Modulated%2520Autophagy-Associated%2520Cell%2520Death%2520in%2520Breast%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D24%26spage%3D9990%26epage%3D10012%26doi%3D10.1021%2Facs.jmedchem.7b00275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Measures, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of Bromodomain–Acetyl-Lysine Interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1021/cb500996u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500996u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=22-39&issue=1&author=M.+Brandauthor=A.+M.+Measuresauthor=B.+G.+Wilsonauthor=W.+A.+Cortopassiauthor=R.+Alexanderauthor=M.+H%C3%B6ssauthor=D.+S.+Hewingsauthor=T.+P.+C.+Rooneyauthor=R.+S.+Patonauthor=S.+J.+Conway&title=Small+Molecule+Inhibitors+of+Bromodomain%E2%80%93Acetyl-Lysine+Interactions&doi=10.1021%2Fcb500996u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Bromodomain-Acetyl-lysine Interactions</span></div><div class="casAuthors">Brand, Michael; Measures, Angelina R.; Wilson, Brian G.; Cortopassi, Wilian A.; Alexander, Rikki; Hoss, Matthias; Hewings, David S.; Rooney, Timothy P. C.; Paton, Robert S.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-39</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Bromodomains are protein modules that bind to acetylated lysine residues.  Their interaction with histone proteins suggests that they function as "readers" of histone lysine acetylation, a component of the proposed "histone code".  Bromodomain-contg. proteins are often found as components of larger protein complexes with roles in fundamental cellular process including transcription.  The publication of two potent ligands for the BET bromodomains in 2010 demonstrated that small mols. can inhibit the bromodomain-acetyl-lysine protein-protein interaction.  These mols. display strong phenotypic effects in a no. of cell lines and affect a range of cancers in vivo.  This work stimulated intense interest in developing further ligands for the BET bromodomains and the design of ligands for non-BET bromodomains.  Here we review the recent progress in the field with particular attention paid to ligand design, the assays employed in early ligand discovery, and the use of computational approaches to inform ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1JZtWpBpqS7Vg90H21EOLACvtfcHk0liX0DwPtMP5LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D&md5=e40e85fc300109654712a00de400c8ba</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb500996u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500996u%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DMeasures%26aufirst%3DA.%2BM.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DAlexander%26aufirst%3DR.%26aulast%3DH%25C3%25B6ss%26aufirst%3DM.%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520Bromodomain%25E2%2580%2593Acetyl-Lysine%2520Interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D1%26spage%3D22%26epage%3D39%26doi%3D10.1021%2Fcb500996u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrieu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">Clinical Trials for BET Inhibitors Run Ahead of the Science</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2016.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.ddtec.2016.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=27769357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=45-50&author=G.+Andrieuauthor=A.+C.+Belkinaauthor=G.+V.+Denis&title=Clinical+Trials+for+BET+Inhibitors+Run+Ahead+of+the+Science&doi=10.1016%2Fj.ddtec.2016.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical trials for BET inhibitors run ahead of the science</span></div><div class="casAuthors">Andrieu Guillaume; Belkina Anna C; Denis Gerald V</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several cancer clinical trials for small molecule inhibitors of BET bromodomain proteins have been initiated.  There is enthusiasm for the anti-proliferative effect of inhibiting BRD4, one of the targets of these inhibitors, which is thought to cooperate with MYC, a long-desired target for cancer therapeutics.  However, no current inhibitor is selective for BRD4 among the three somatic BET proteins, which include BRD2 and BRD3; their respective functions are partially overlapping and none are functionally redundant with BRD4.  Each BET protein controls distinct transcriptional pathways that are important for functions beyond cancer cell proliferation, including insulin production, cytokine gene transcription, T cell differentiation, adipogenesis and most seriously, active repression of dangerous latent viruses like HIV.  BET inhibitors have been shown to reactivate HIV in human cells.  Failure to appreciate that at concentrations used, no available BET inhibitor is member-selective, or to develop a sound biological basis to understand the diverse functions of BET proteins before undertaking for these clinical trials is reckless and likely to lead to adverse events.  More mechanistic information from new basic science studies should enable proper focus on the most relevant cancers and define the expected side effect profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEothR-V9Y-Lt7lRqEFUJVfW6udTcc2eZXEjQedQ4TRbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D&md5=5fcc9e96fa590221fb07c5e98ae91d59</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2016.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DAndrieu%26aufirst%3DG.%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DClinical%2520Trials%2520for%2520BET%2520Inhibitors%2520Run%2520Ahead%2520of%2520the%2520Science%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2016%26volume%3D19%26spage%3D45%26epage%3D50%26doi%3D10.1016%2Fj.ddtec.2016.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amorim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyengar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarotto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leleu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morschhauser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broussais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahatt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokiec, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salles, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palumbo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thieblemont, C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Inhibitor OTX015 in Patients with Lymphoma or Multiple Myeloma: A Dose-Escalation, Open-Label, Pharmacokinetic, Phase 1 Study</span>. <i>Lancet Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">e196</span>– <span class="NLM_lpage">e204</span>, <span class="refDoi"> DOI: 10.1016/S2352-3026(16)00021-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2FS2352-3026%2816%2900021-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=27063978" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=e196-e204&issue=4&author=S.+Amorimauthor=A.+Stathisauthor=M.+Gleesonauthor=S.+Iyengarauthor=V.+Magarottoauthor=X.+Leleuauthor=F.+Morschhauserauthor=L.+Karlinauthor=F.+Broussaisauthor=K.+Rezaiauthor=P.+Heraitauthor=C.+Kahattauthor=F.+Lokiecauthor=G.+Sallesauthor=T.+Faconauthor=A.+Palumboauthor=D.+Cunninghamauthor=E.+Zuccaauthor=C.+Thieblemont&title=Bromodomain+Inhibitor+OTX015+in+Patients+with+Lymphoma+or+Multiple+Myeloma%3A+A+Dose-Escalation%2C+Open-Label%2C+Pharmacokinetic%2C+Phase+1+Study&doi=10.1016%2FS2352-3026%2816%2900021-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS2352-3026%2816%2900021-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2352-3026%252816%252900021-1%26sid%3Dliteratum%253Aachs%26aulast%3DAmorim%26aufirst%3DS.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DGleeson%26aufirst%3DM.%26aulast%3DIyengar%26aufirst%3DS.%26aulast%3DMagarotto%26aufirst%3DV.%26aulast%3DLeleu%26aufirst%3DX.%26aulast%3DMorschhauser%26aufirst%3DF.%26aulast%3DKarlin%26aufirst%3DL.%26aulast%3DBroussais%26aufirst%3DF.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DKahatt%26aufirst%3DC.%26aulast%3DLokiec%26aufirst%3DF.%26aulast%3DSalles%26aufirst%3DG.%26aulast%3DFacon%26aufirst%3DT.%26aulast%3DPalumbo%26aufirst%3DA.%26aulast%3DCunningham%26aufirst%3DD.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DThieblemont%26aufirst%3DC.%26atitle%3DBromodomain%2520Inhibitor%2520OTX015%2520in%2520Patients%2520with%2520Lymphoma%2520or%2520Multiple%2520Myeloma%253A%2520A%2520Dose-Escalation%252C%2520Open-Label%252C%2520Pharmacokinetic%252C%2520Phase%25201%2520Study%26jtitle%3DLancet%2520Haematol.%26date%3D2016%26volume%3D3%26issue%3D4%26spage%3De196%26epage%3De204%26doi%3D10.1016%2FS2352-3026%2816%2900021-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an Inhibitor of BET Transcriptional Regulators with Selectivity for the Second Bromodomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">49</span>),  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&issue=49&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+Mullerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+Inhibitor+of+BET+Transcriptional+Regulators+with+Selectivity+for+the+Second+Bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lj2_iodUyxShg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520Inhibitor%2520of%2520BET%2520Transcriptional%2520Regulators%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26issue%3D49%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span> <span> </span><span class="NLM_article-title">RVX-297- a Novel BD2 Selective Inhibitor of BET Bromodomains</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>477</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2016.06.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.bbrc.2016.06.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=27282480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2016&pages=62-67&issue=1&author=O.+A.+Kharenkoauthor=E.+M.+Gesnerauthor=R.+G.+Patelauthor=K.+Norekauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=P.+R.+Youngauthor=K.+G.+McLureauthor=H.+C.+Hansen&title=RVX-297-+a+Novel+BD2+Selective+Inhibitor+of+BET+Bromodomains&doi=10.1016%2Fj.bbrc.2016.06.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span></div><div class="casAuthors">Kharenko, Olesya A.; Gesner, Emily M.; Patel, Reena G.; Norek, Karen; White, Andre; Fontano, Eric; Suto, Robert K.; Young, Peter R.; McLure, Kevin G.; Hansen, Henrik C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-67</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers that specifically bind to the acetyl lysine residues of histones and transcription factors.  Small mol. BET bromodomain inhibitors can disrupt this interaction which leads to potential modulation of several disease states.  Here we describe the binding properties of a novel BET inhibitor RVX-297 that is structurally related to the clin. compd. RVX-208, currently undergoing phase III clin. trials for the treatment of cardiovascular diseases, but is distinctly different in its biol. and pharmacokinetic profiles.  We report that RVX-297 preferentially binds to the BD2 domains of the BET bromodomain and Extra Terminal (BET) family of protein.  We demonstrate the differential binding modes of RVX-297 in BD1 and BD2 domains of BRD4 and BRD2 using X-ray crystallog., and describe the structural differences driving the BD2 selective binding of RVX-297.  The isothermal titrn. calorimetry (ITC) data illustrate the related differential thermodn. of binding of RVX-297 to single as well as dual BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLUg96I7HObVg90H21EOLACvtfcHk0lj2_iodUyxShg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D&md5=a3d064e2245bf14ae525eac2490808cb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.06.021%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26atitle%3DRVX-297-%2520a%2520Novel%2520BD2%2520Selective%2520Inhibitor%2520of%2520BET%2520Bromodomains%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D477%26issue%3D1%26spage%3D62%26epage%3D67%26doi%3D10.1016%2Fj.bbrc.2016.06.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berkovits, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolgemuth, D. J.</span></span> <span> </span><span class="NLM_article-title">The First Bromodomain of the Testis-Specific Double Bromodomain Protein Brdt Is Required for Chromocenter Organization That Is Modulated by Genetic Background</span>. <i>Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>360</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1016/j.ydbio.2011.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.ydbio.2011.10.005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=360&publication_year=2011&pages=358-368&issue=2&author=B.+D.+Berkovitsauthor=D.+J.+Wolgemuth&title=The+First+Bromodomain+of+the+Testis-Specific+Double+Bromodomain+Protein+Brdt+Is+Required+for+Chromocenter+Organization+That+Is+Modulated+by+Genetic+Background&doi=10.1016%2Fj.ydbio.2011.10.005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ydbio.2011.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ydbio.2011.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DBerkovits%26aufirst%3DB.%2BD.%26aulast%3DWolgemuth%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520First%2520Bromodomain%2520of%2520the%2520Testis-Specific%2520Double%2520Bromodomain%2520Protein%2520Brdt%2520Is%2520Required%2520for%2520Chromocenter%2520Organization%2520That%2520Is%2520Modulated%2520by%2520Genetic%2520Background%26jtitle%3DDev.%2520Biol.%26date%3D2011%26volume%3D360%26issue%3D2%26spage%3D358%26epage%3D368%26doi%3D10.1016%2Fj.ydbio.2011.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Assessing Cellular Efficacy of Bromodomain Inhibitors Using Fluorescence Recovery after Photobleaching</span>. <i>Epigenet. Chromatin</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">14</span>, <span class="refDoi"> DOI: 10.1186/1756-8935-7-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1186%2F1756-8935-7-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=25097667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVaqtrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=14&issue=1&author=M.+Philpottauthor=C.+M.+Rogersauthor=C.+Yappauthor=C.+Wellsauthor=J.+P.+Lambertauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=A.+C.+Gingrasauthor=S.+Knappauthor=S.+M%C3%BCller&title=Assessing+Cellular+Efficacy+of+Bromodomain+Inhibitors+Using+Fluorescence+Recovery+after+Photobleaching&doi=10.1186%2F1756-8935-7-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing cellular efficacy of bromodomain inhibitors using fluorescence recovery after photobleaching</span></div><div class="casAuthors">Philpott, Martin; Rogers, Catherine M.; Yapp, Clarence; Wells, Chris; Lambert, Jean-Philippe; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Gingras, Anne-Claude; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14/1-14/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Acetylation of lysine residues in histone tails plays an important role in the regulation of gene transcription.  Bromdomains are the readers of acetylated histone marks, and, consequently, bromodomain-contg. proteins have a variety of chromatin-related functions.  Moreover, they are increasingly being recognized as important mediators of a wide range of diseases.  The first potent and selective bromodomain inhibitors are beginning to be described, but the diverse or unknown functions of bromodomain-contg. proteins present challenges to systematically demonstrating cellular efficacy and selectivity for these inhibitors.  Here we assess the viability of fluorescence recovery after photobleaching (FRAP) assays as a target agnostic method for the direct visualisation of an on-target effect of bromodomain inhibitors in living cells.  Results: Mutation of a conserved asparagine crucial for binding to acetylated lysines in the bromodomains of BRD3, BRD4 and TRIM24 all resulted in redn. of FRAP recovery times, indicating loss of or significantly reduced binding to acetylated chromatin, as did the addn. of known inhibitors.  Significant differences between wild type and bromodomain mutants for ATAD2, BAZ2A, BRD1, BRD7, GCN5L2, SMARCA2 and ZMYND11 required the addn. of the histone deacetylase inhibitor suberoylanilide hydroxamic acid (SAHA) to amplify the binding contribution of the bromodomain.  Under these conditions, known inhibitors decreased FRAP recovery times back to mutant control levels.  Mutation of the bromodomain did not alter FRAP recovery times for full-length CREBBP, even in the presence of SAHA, indicating that other domains are primarily responsible for anchoring CREBBP to chromatin.  However, FRAP assays with multimerised CREBBP bromodomains resulted in a good assay to assess the efficacy of bromodomain inhibitors to this target.  The bromodomain and extraterminal protein inhibitor PFI-1 was inactive against other bromodomain targets, demonstrating the specificity of the method.  Conclusions: Viable FRAP assays were established for 11 representative bromodomain-contg. proteins that broadly cover the bromodomain phylogenetic tree.  Addn. of SAHA can overcome weak binding to chromatin, and the use of tandem bromodomain constructs can eliminate masking effects of other chromatin binding domains.  Together, these results demonstrate that FRAP assays offer a potentially pan-bromodomain method for generating cell-based assays, allowing the testing of compds. with respect to cell permeability, on-target efficacy and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0eRoqpM6LTbVg90H21EOLACvtfcHk0ljJuRDD4EwFTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVaqtrzE&md5=966744a9e7b3cba00776b76d65594fc6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-7-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-7-14%26sid%3Dliteratum%253Aachs%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DAssessing%2520Cellular%2520Efficacy%2520of%2520Bromodomain%2520Inhibitors%2520Using%2520Fluorescence%2520Recovery%2520after%2520Photobleaching%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2014%26volume%3D7%26issue%3D1%26spage%3D14%26doi%3D10.1186%2F1756-8935-7-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. F.</span></span> <span> </span><span class="NLM_article-title">Brd4 Maintains Constitutively Active NF-κB in Cancer Cells by Binding to Acetylated RelA</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">2395</span>– <span class="NLM_lpage">2404</span>, <span class="refDoi"> DOI: 10.1038/onc.2013.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fonc.2013.179" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=2395-2404&issue=18&author=Z.+Zouauthor=B.+Huangauthor=X.+Wuauthor=H.+Zhangauthor=J.+Qiauthor=J.+Bradnerauthor=S.+Nairauthor=L.+F.+Chen&title=Brd4+Maintains+Constitutively+Active+NF-%CE%BAB+in+Cancer+Cells+by+Binding+to+Acetylated+RelA&doi=10.1038%2Fonc.2013.179"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fonc.2013.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2013.179%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DNair%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBrd4%2520Maintains%2520Constitutively%2520Active%2520NF-%25CE%25BAB%2520in%2520Cancer%2520Cells%2520by%2520Binding%2520to%2520Acetylated%2520RelA%26jtitle%3DOncogene%26date%3D2014%26volume%3D33%26issue%3D18%26spage%3D2395%26epage%3D2404%26doi%3D10.1038%2Fonc.2013.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">9251</span>– <span class="NLM_lpage">9264</span>, <span class="refDoi"> DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&issue=22&author=G.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.+M.+Zhou&title=Structure-Guided+Design+of+Potent+Diazobenzene+Inhibitors+for+the+BET+Bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0lhImmo0JdUNHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure-Guided%2520Design%2520of%2520Potent%2520Diazobenzene%2520Inhibitors%2520for%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D22%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuppusetty, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">3,5-Dimethylisoxazoles Act as Acetyl-Lysine-Mimetic Bromodomain Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">6761</span>– <span class="NLM_lpage">6770</span>, <span class="refDoi"> DOI: 10.1021/jm200640v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&issue=19&author=D.%0AS.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knappauthor=S.+J.+Conwayauthor=T.+D.+Heightman&title=3%2C5-Dimethylisoxazoles+Act+as+Acetyl-Lysine-Mimetic+Bromodomain+Ligands&doi=10.1021%2Fjm200640v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Dimethylisoxazoles Act As Acetyl-lysine-mimetic Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Wang, Minghua; Philpott, Martin; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Vuppusetty, Chaitanya; Marsden, Brian; Knapp, Stefan; Conway, Stuart J.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6761-6770</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription.  Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code.  Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties.  With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance.  We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains.  Using X-ray crystallog. anal., we have detd. the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains.  By exploiting these interactions, we have developed compd. 4d, which has IC50 values of <5 μM for the bromodomain-contg. proteins BRD2(1) and BRD4(1).  These compds. are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEzK4xMvJhXrVg90H21EOLACvtfcHk0lhImmo0JdUNHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO&md5=7d6f9eff2735c8a7189dec4fb5c5d13d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3D3%252C5-Dimethylisoxazoles%2520Act%2520as%2520Acetyl-Lysine-Mimetic%2520Bromodomain%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D19%26spage%3D6761%26epage%3D6770%26doi%3D10.1021%2Fjm200640v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gacias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matikainen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casaccia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.chembiol.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=24954007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=841-854&issue=7&author=M.+Gaciasauthor=G.+Gerona-Navarroauthor=A.+N.+Plotnikovauthor=G.+Zhangauthor=L.+Zengauthor=J.+Kaurauthor=G.+Moyauthor=E.+Rusinovaauthor=Y.+Rodriguezauthor=B.+Matikainenauthor=A.+Vincekauthor=J.+Joshuaauthor=P.+Casacciaauthor=M.+M.+Zhou&title=Selective+Chemical+Modulation+of+Gene+Transcription+Favors+Oligodendrocyte+Lineage+Progression&doi=10.1016%2Fj.chembiol.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span></div><div class="casAuthors">Gacias, Mar; Gerona-Navarro, Guillermo; Plotnikov, Alexander N.; Zhang, Guangtao; Zeng, Lei; Kaur, Jasbir; Moy, Gregory; Rusinova, Elena; Rodriguez, Yoel; Matikainen, Bridget; Vincek, Adam; Joshua, Jennifer; Casaccia, Patrizia; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">841-854</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lysine acetylation regulates gene expression through modulating protein-protein interactions in chromatin.  Chem. inhibition of acetyl-lysine binding bromodomains of the major chromatin regulators BET (bromodomain and extraterminal domain) proteins has been shown to effectively block cell proliferation in cancer and inflammation.  However, whether selective inhibition of individual BET bromodomains has distinctive functional consequences remains only partially understood.  In this study, we show that selective chem. inhibition of the first bromodomain of BET proteins using our small-mol. inhibitor, Olinone, accelerated the progression of mouse primary oligodendrocyte progenitors toward differentiation, whereas inhibition of both bromodomains of BET proteins hindered differentiation.  This effect was target specific, as it was not detected in cells treated with inactive analogs and independent of any effect on proliferation.  Therefore, selective chem. modulation of individual bromodomains, rather than use of broad-based inhibitors, may enhance regenerative strategies in disorders characterized by myelin loss such as aging and neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1fCvOR18yE7Vg90H21EOLACvtfcHk0lhImmo0JdUNHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK&md5=6b377663a8d0d07441ab4f8a5d8191b3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DGacias%26aufirst%3DM.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DKaur%26aufirst%3DJ.%26aulast%3DMoy%26aufirst%3DG.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DY.%26aulast%3DMatikainen%26aufirst%3DB.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DCasaccia%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DSelective%2520Chemical%2520Modulation%2520of%2520Gene%2520Transcription%2520Favors%2520Oligodendrocyte%2520Lineage%2520Progression%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26issue%3D7%26spage%3D841%26epage%3D854%26doi%3D10.1016%2Fj.chembiol.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardote, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pschibul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, T. T.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology. A Bump-and-Hole Approach to Engineer Controlled Selectivity of BET Bromodomain Chemical Probes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span> (<span class="NLM_issue">6209</span>),  <span class="NLM_fpage">638</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1126/science.1249830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1126%2Fscience.1249830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=25323695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=638-641&issue=6209&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=T.+Cardoteauthor=C.+Tallantauthor=A.+Pschibulauthor=K.-H.+Chanauthor=M.+Zengerleauthor=J.+R.+Garciaauthor=T.+T.-L.+Kwanauthor=F.+M.+Fergusonauthor=A.+Ciulli&title=Chemical+Biology.+A+Bump-and-Hole+Approach+to+Engineer+Controlled+Selectivity+of+BET+Bromodomain+Chemical+Probes&doi=10.1126%2Fscience.1249830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Cardote, Teresa; Tallant, Cynthia; Pschibul, Annica; Chan, Kwok-Ho; Zengerle, Michael; Garcia, Jordi R.; Kwan, Terence T.-L.; Ferguson, Fleur M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6209</span>),
    <span class="NLM_cas:pages">638-641</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Small mols. are useful tools for probing the biol. function and therapeutic potential of individual proteins, but achieving selectivity is challenging when the target protein shares structural domains with other proteins.  The bromodomain and extra-terminal (BET) proteins have attracted interest because of their roles in transcriptional regulation, epigenetics, and cancer.  The BET bromodomains (protein interaction modules that bind acetyl-lysine) have been targeted by potent small-mol. inhibitors, but these inhibitors lack selectivity for individual family members.  We developed an Et deriv. of an existing small-mol. inhibitor, I-BET/JQ1, and showed that it binds leucine/alanine mutant bromodomains with nanomolar affinity and achieves up to 540-fold selectivity relative to wild-type bromodomains.  Cell culture studies showed that blockade of the first bromodomain alone is sufficient to displace a specific BET protein, Brd4, from chromatin.  Expansion of this approach could help identify the individual roles of single BET proteins in human physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9EnzLtvUKwbVg90H21EOLACvtfcHk0liYskZhO57_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL&md5=04fe967e4681fb4b2b3168f4d5f8ff81</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.1249830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1249830%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DCardote%26aufirst%3DT.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DJ.%2BR.%26aulast%3DKwan%26aufirst%3DT.%2BT.-L.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DChemical%2520Biology.%2520A%2520Bump-and-Hole%2520Approach%2520to%2520Engineer%2520Controlled%2520Selectivity%2520of%2520BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DScience%26date%3D2014%26volume%3D346%26issue%3D6209%26spage%3D638%26epage%3D641%26doi%3D10.1126%2Fscience.1249830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Runcie, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Beurden, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulthard, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of a “Bump-and-Hole” Approach to Allele-Selective BET Bromodomain Inhibition</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2452</span>– <span class="NLM_lpage">2468</span>, <span class="refDoi"> DOI: 10.1039/C7SC02536J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1039%2FC7SC02536J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=29732121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2452-2468&issue=9&author=A.+C.+Runcieauthor=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Testaauthor=L.+Van+Beurdenauthor=M.+G.+J.+Baudauthor=O.+Epemoluauthor=L.+C.+J.+Ellisauthor=K.+D.+Readauthor=V.+Coulthardauthor=A.+Brienauthor=A.+Ciulli&title=Optimization+of+a+%E2%80%9CBump-and-Hole%E2%80%9D+Approach+to+Allele-Selective+BET+Bromodomain+Inhibition&doi=10.1039%2FC7SC02536J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a "bump-and-hole" approach to allele-selective BET bromodomain inhibition</span></div><div class="casAuthors">Runcie, A. C.; Zengerle, M.; Chan, K.-H.; Testa, A.; van Beurden, L.; Baud, M. G. J.; Epemolu, O.; Ellis, L. C. J.; Read, K. D.; Coulthard, V.; Brien, A.; Ciulli, A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2452-2468</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Allele-specific chem. genetics enables selective inhibition within families of highly-conserved proteins.  The four BET (bromodomain & extra-terminal domain) proteins - BRD2, BRD3, BRD4 and BRDT bind acetylated chromatin via their bromodomains and regulate processes such as cell proliferation and inflammation.  BET bromodomains are of particular interest, as they are attractive therapeutic targets but existing inhibitors are pan-selective.  We previously established a bump-&-hole system for the BET bromodomains, pairing a leucine/alanine mutation with an ethyl-derived analog of an established benzodiazepine scaffold.  Here we optimize upon this system with the introduction of a more conservative and less disruptive leucine/valine mutation.  Extensive structure-activity-relationships of diverse benzodiazepine analogs guided the development of potent, mutant-selective inhibitors with desirable physiochem. properties.  The active enantiomer of our best compd. - 9-ME-1 - shows ~ 200 nM potency, >100-fold selectivity for the L/V mutant over wild-type and excellent DMPK properties.  Through a variety of in vitro and cellular assays we validate the capabilities of our optimized system, and then utilize it to compare the relative importance of the first and second bromodomains to chromatin binding.  These expts. confirm the primacy of the first bromodomain in all BET proteins, but also significant variation in the importance of the second bromodomain.  We also show that, despite having a minor role in chromatin recognition, BRD4 BD2 is still essential for gene expression, likely through the recruitment of non-histone proteins.  The disclosed inhibitor:mutant pair provides a powerful tool for future cellular and in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq47qHG4-zFurVg90H21EOLACvtfcHk0liYskZhO57_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D&md5=9a44bdccabca9f1d4dfdc983250d9aa3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC7SC02536J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC02536J%26sid%3Dliteratum%253Aachs%26aulast%3DRuncie%26aufirst%3DA.%2BC.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DVan%2BBeurden%26aufirst%3DL.%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DEllis%26aufirst%3DL.%2BC.%2BJ.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCoulthard%26aufirst%3DV.%26aulast%3DBrien%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DOptimization%2520of%2520a%2520%25E2%2580%259CBump-and-Hole%25E2%2580%259D%2520Approach%2520to%2520Allele-Selective%2520BET%2520Bromodomain%2520Inhibition%26jtitle%3DChem.%2520Sci.%26date%3D2018%26volume%3D9%26issue%3D9%26spage%3D2452%26epage%3D2468%26doi%3D10.1039%2FC7SC02536J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szklarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul Azeez, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikolajczyk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepetov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Haj Zen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teicher, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Characterization of the Published Kinase Inhibitor Set</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1038/nbt.3374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnbt.3374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=26501955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslantL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=95-103&issue=1&author=J.+M.+Elkinsauthor=V.+Fedeleauthor=M.+Szklarzauthor=K.+R.+Abdul+Azeezauthor=E.+Salahauthor=J.+Mikolajczykauthor=S.+Romanovauthor=N.+Sepetovauthor=X.-P.+Huangauthor=B.+L.+Rothauthor=A.+Al+Haj+Zenauthor=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropshaauthor=J.+Morrisauthor=B.+A.+Teicherauthor=M.+Kunkelauthor=E.+Polleyauthor=K.+E.+Lackeyauthor=F.+L.+Atkinsonauthor=J.+P.+Overingtonauthor=P.+Bamboroughauthor=S.+M%C3%BCllerauthor=D.+J.+Priceauthor=T.+M.+Willsonauthor=D.+H.+Drewryauthor=S.+Knappauthor=W.+J.+Zuercher&title=Comprehensive+Characterization+of+the+Published+Kinase+Inhibitor+Set&doi=10.1038%2Fnbt.3374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive characterization of the Published Kinase Inhibitor Set</span></div><div class="casAuthors">Elkins, Jonathan M.; Fedele, Vita; Szklarz, Marta; Abdul Azeez, Kamal R.; Salah, Eidarus; Mikolajczyk, Jowita; Romanov, Sergei; Sepetov, Nikolai; Huang, Xi-Ping; Roth, Bryan L.; Al Haj Zen, Ayman; Fourches, Denis; Muratov, Eugene; Tropsha, Alex; Morris, Joel; Teicher, Beverly A.; Kunkel, Mark; Polley, Eric; Lackey, Karen E.; Atkinson, Francis L.; Overington, John P.; Bamborough, Paul; Muller, Susanne; Price, Daniel J.; Willson, Timothy M.; Drewry, David H.; Knapp, Stefan; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-103</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite the success of protein kinase inhibitors as approved therapeutics, drug discovery has focused on a small subset of kinase targets.  Here we provide a thorough characterization of the Published Kinase Inhibitor Set (PKIS), a set of 367 small-mol. ATP-competitive kinase inhibitors that was recently made freely available with the aim of expanding research in this field and as an expt. in open-source target validation.  We screen the set in activity assays with 224 recombinant kinases and 24 G protein-coupled receptors and in cellular assays of cancer cell proliferation and angiogenesis.  We identify chem. starting points for designing new chem. probes of orphan kinases and illustrate the utility of these leads by developing a selective inhibitor for the previously untargeted kinases LOK and SLK.  Our cellular screens reveal compds. that modulate cancer cell growth and angiogenesis in vitro.  These reagents and assocd. data illustrate an efficient way forward to increasing understanding of the historically untargeted kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4P1WN3gdiXbVg90H21EOLACvtfcHk0liYskZhO57_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslantL%252FM&md5=3f15096e79632dbaaadf4460a6848e60</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3374%26sid%3Dliteratum%253Aachs%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR.%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DMikolajczyk%26aufirst%3DJ.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DSepetov%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DAl%2BHaj%2BZen%26aufirst%3DA.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DTeicher%26aufirst%3DB.%2BA.%26aulast%3DKunkel%26aufirst%3DM.%26aulast%3DPolley%26aufirst%3DE.%26aulast%3DLackey%26aufirst%3DK.%2BE.%26aulast%3DAtkinson%26aufirst%3DF.%2BL.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DComprehensive%2520Characterization%2520of%2520the%2520Published%2520Kinase%2520Inhibitor%2520Set%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26issue%3D1%26spage%3D95%26epage%3D103%26doi%3D10.1038%2Fnbt.3374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.
J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-Lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&issue=5&author=S.+W.%0AJ.+Emberauthor=J.+Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Acetyl-Lysine+Binding+Site+of+Bromodomain-Containing+Protein+4+%28BRD4%29+Interacts+with+Diverse+Kinase+Inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0lhO-QH2zJE4Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DAcetyl-Lysine%2520Binding%2520Site%2520of%2520Bromodomain-Containing%2520Protein%25204%2520%2528BRD4%2529%2520Interacts%2520with%2520Diverse%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26issue%3D5%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawk, L. M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassel, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dos Santos, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Dual Screening of BPTF and Brd4 Using Protein-Observed Fluorine NMR Uncovers New Bromodomain Probe Molecules</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2256</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2246-2256&issue=10&author=A.+K.+Urickauthor=L.+M.+L.+Hawkauthor=M.+K.+Casselauthor=N.+K.+Mishraauthor=S.+Liuauthor=N.+Adhikariauthor=W.+Zhangauthor=C.+O.+Dos+Santosauthor=J.+L.+Hallauthor=W.+C.+K.+Pomerantz&title=Dual+Screening+of+BPTF+and+Brd4+Using+Protein-Observed+Fluorine+NMR+Uncovers+New+Bromodomain+Probe+Molecules&doi=10.1021%2Facschembio.5b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Screening of BPTF and Brd4 Using Protein-Observed Fluorine NMR Uncovers New Bromodomain Probe Molecules</span></div><div class="casAuthors">Urick, Andrew K.; Hawk, Laura M. L.; Cassel, Melissa K.; Mishra, Neeraj K.; Liu, Shuai; Adhikari, Neeta; Zhang, Wei; dos Santos, Camila O.; Hall, Jennifer L.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2246-2256</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein dysregulation is linked to cancer, diabetes, and inflammation.  Selective inhibition of bromodomain function is a newly proposed therapeutic strategy.  The authors describe a 19F NMR dual screening method for small mol. discovery using fluorinated tryptophan resonances on two bromodomain-contg. proteins.  The chem. shift dispersion of 19F resonances within fluorine-labeled proteins enables the simultaneous anal. of two fluorinated bromodomains by NMR.  A library of 229 small mols. was screened against the first bromodomain of Brd4 and the BPTF bromodomain.  The authors report the first small mol. selective for BPTF over Brd4, termed AU1.  The Kd = 2.8 μM for AU1, which is active in a cell-based reporter assay.  No binding was detected with Brd4.  Three new Brd4 inhibitors with submicromolar affinity were also discovered.  Brd4 hits were validated in a thermal stability assay and potency detd. via fluorescence anisotropy.  The speed, ease of interpretation, and low protein concn. needed for protein-obsd. 19F NMR expts. in a multiprotein format offers a new method to discover and characterize selective ligands for bromodomain-contg. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjhnKasUdNL7Vg90H21EOLACvtfcHk0lhO-QH2zJE4Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfI&md5=584fa6b83f8bfa661f55edb1078e570c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00483%26sid%3Dliteratum%253Aachs%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DCassel%26aufirst%3DM.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DAdhikari%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDos%2BSantos%26aufirst%3DC.%2BO.%26aulast%3DHall%26aufirst%3DJ.%2BL.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DDual%2520Screening%2520of%2520BPTF%2520and%2520Brd4%2520Using%2520Protein-Observed%2520Fluorine%2520NMR%2520Uncovers%2520New%2520Bromodomain%2520Probe%2520Molecules%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D10%26spage%3D2246%26epage%3D2256%26doi%3D10.1021%2Facschembio.5b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bower, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. L.</span></span> <span> </span><span class="NLM_article-title">Phenoxypyrimidine Inhibitors of P38α Kinase: Synthesis and Statistical Evaluation of the P38 Inhibitory Potencies of a Series of 1-(Piperidin-4-Yl)-4-(4-Fluorophenyl)-5-(2-Phenoxypyrimidin-4-Yl) Imidazoles</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1123</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(01)00163-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2FS0960-894X%2801%2900163-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1123-1126&issue=9&author=J.+C.+Boehmauthor=M.+J.+Bowerauthor=T.+F.+Gallagherauthor=S.+Kassisauthor=S.+R.+Johnsonauthor=J.+L.+Adams&title=Phenoxypyrimidine+Inhibitors+of+P38%CE%B1+Kinase%3A+Synthesis+and+Statistical+Evaluation+of+the+P38+Inhibitory+Potencies+of+a+Series+of+1-%28Piperidin-4-Yl%29-4-%284-Fluorophenyl%29-5-%282-Phenoxypyrimidin-4-Yl%29+Imidazoles&doi=10.1016%2FS0960-894X%2801%2900163-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2801%2900163-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252801%252900163-9%26sid%3Dliteratum%253Aachs%26aulast%3DBoehm%26aufirst%3DJ.%2BC.%26aulast%3DBower%26aufirst%3DM.%2BJ.%26aulast%3DGallagher%26aufirst%3DT.%2BF.%26aulast%3DKassis%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DAdams%26aufirst%3DJ.%2BL.%26atitle%3DPhenoxypyrimidine%2520Inhibitors%2520of%2520P38%25CE%25B1%2520Kinase%253A%2520Synthesis%2520and%2520Statistical%2520Evaluation%2520of%2520the%2520P38%2520Inhibitory%2520Potencies%2520of%2520a%2520Series%2520of%25201-%2528Piperidin-4-Yl%2529-4-%25284-Fluorophenyl%2529-5-%25282-Phenoxypyrimidin-4-Yl%2529%2520Imidazoles%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2001%26volume%3D11%26issue%3D9%26spage%3D1123%26epage%3D1126%26doi%3D10.1016%2FS0960-894X%2801%2900163-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors for Rationally Designed Polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&issue=4&author=P.+Ciceriauthor=S.+M%C3%BCllerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+Kinase-Bromodomain+Inhibitors+for+Rationally+Designed+Polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lhO-QH2zJE4Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520for%2520Rationally%2520Designed%2520Polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26issue%3D4%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-Peptide Displacement Assays for Interactome Mapping and Inhibitor Discovery</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2899</span>, <span class="refDoi"> DOI: 10.1039/c1mb05099k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1039%2Fc1mb05099k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21804994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFakt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=2899&issue=10&author=M.+Philpottauthor=J.+Yangauthor=T.+Tumberauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=T.+Keatesauthor=I.+Felletarauthor=A.+Ciulliauthor=S.+Knappauthor=T.+D.+Heightman&title=Bromodomain-Peptide+Displacement+Assays+for+Interactome+Mapping+and+Inhibitor+Discovery&doi=10.1039%2Fc1mb05099k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-peptide displacement assays for interactome mapping and inhibitor discovery</span></div><div class="casAuthors">Philpott, Martin; Yang, Jing; Tumber, Tony; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Keates, Tracy; Felletar, Ildiko; Ciulli, Alessio; Knapp, Stefan; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2899-2908</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone lysine acetylation is a key component of epigenetic regulation of gene transcription.  Bromodomains, found in histone acetyl transferases and other chromatin-assocd. proteins, bind selectively to acetylated lysines, acting as "readers" of the histone code, and have recently been shown to contain a druggable binding pocket.  Here we report the development of high-throughput assays that quantify the binding of bromodomains to acetylated histone peptides.  We have used these assays to screen for histone binding partners of as yet uncharacterized bromodomains, adding to current knowledge of the histone code and expanding the repertoire of assays for chem. probe discovery.  We have also demonstrated that these assays can be used to detect small mol. binding from the very weak to the nanomolar range.  This assay methodol. is thereby anticipated to provide the basis both for broader interactome profiling and for small mol. inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5mWM_E4IUzbVg90H21EOLACvtfcHk0ljttdIO19Bn2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFakt77P&md5=0f9eccb2d2761e1707c9b04bccc76050</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2Fc1mb05099k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1mb05099k%26sid%3Dliteratum%253Aachs%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTumber%26aufirst%3DT.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DBromodomain-Peptide%2520Displacement%2520Assays%2520for%2520Interactome%2520Mapping%2520and%2520Inhibitor%2520Discovery%26jtitle%3DMol.%2520BioSyst.%26date%3D2011%26volume%3D7%26issue%3D10%26spage%3D2899%26doi%3D10.1039%2Fc1mb05099k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C.</span></span> <span> </span><span class="NLM_article-title">Fluorinated Aromatic Amino Acids Are Sensitive 19F NMR Probes for Bromodomain-Ligand Interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2755</span>– <span class="NLM_lpage">2760</span>, <span class="refDoi"> DOI: 10.1021/cb5007344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5007344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs12gtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2755-2760&issue=12&author=N.+K.+Mishraauthor=A.+K.+Urickauthor=S.+W.+J.+Emberauthor=E.+Schonbrunnauthor=W.+C.+Pomerantz&title=Fluorinated+Aromatic+Amino+Acids+Are+Sensitive+19F+NMR+Probes+for+Bromodomain-Ligand+Interactions&doi=10.1021%2Fcb5007344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorinated Aromatic Amino Acids Are Sensitive 19F NMR Probes for Bromodomain-Ligand Interactions</span></div><div class="casAuthors">Mishra, Neeraj K.; Urick, Andrew K.; Ember, Stuart W. J.; Schonbrunn, Ernst; Pomerantz, William C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2755-2760</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors describe a 19F NMR method for detecting bromodomain-ligand interactions using fluorine-labeled arom. amino acids due to the conservation of arom. residues in the bromodomain binding site.  The authors test the sensitivity, accuracy, and speed of this method with small mol. ligands (+)-JQ1, BI2536, Dinaciclib, TG101348, and acetaminophen using three bromodomains Brd4, BrdT, and BPTF.  Simplified 19F NMR spectra allowed for simultaneous testing of multiple bromodomains to assess selectivity and identification of a new BPTF ligand.  Fluorine labeling only modestly affected the Brd4 structure and function assessed by isothermal titrn. calorimetry, CD, and x-ray crystallog.  The speed, ease of interpretation, and low concn. of protein needed for binding expts. affords a new method to discover and characterize both native and new ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoejxq3KKVddrVg90H21EOLACvtfcHk0ljttdIO19Bn2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs12gtLfP&md5=97f9edadc91cba8670f54bcca57fa03f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fcb5007344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5007344%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%26atitle%3DFluorinated%2520Aromatic%2520Amino%2520Acids%2520Are%2520Sensitive%252019F%2520NMR%2520Probes%2520for%2520Bromodomain-Ligand%2520Interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26issue%3D12%26spage%3D2755%26epage%3D2760%26doi%3D10.1021%2Fcb5007344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arntson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Protein-Observed Fluorine NMR: A Bioorthogonal Approach for Small Molecule Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">5158</span>– <span class="NLM_lpage">5171</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01447</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01447" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFSmtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5158-5171&issue=11&author=K.+E.+Arntsonauthor=W.+C.+K.+Pomerantz&title=Protein-Observed+Fluorine+NMR%3A+A+Bioorthogonal+Approach+for+Small+Molecule+Discovery&doi=10.1021%2Facs.jmedchem.5b01447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Observed Fluorine NMR: A Bioorthogonal Approach for Small Molecule Discovery</span></div><div class="casAuthors">Arntson, Keith E.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5158-5171</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 19F isotope is 100% naturally abundant and is the second most sensitive and stable NMR-active nucleus.  Unlike the ubiquitous hydrogen atom, fluorine is nearly absent in biol. systems, making it a unique bioorthogonal atom for probing mol. interactions in biol.  Over 73 fluorinated proteins have been studied by 19F NMR since the seminal studies of Hull and Sykes in 1974.  With advances in cryoprobe prodn. and fluorinated amino acid incorporation strategies, protein-based 19F NMR offers opportunities to the medicinal chemist for characterizing and ultimately discovering new small mol. protein ligands.  This review will highlight new advances using 19F NMR for characterizing small mol. interactions with both small and large proteins as well as detailing NMR resonance assignment challenges and amino acid incorporation approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8Heq0upzHCbVg90H21EOLACvtfcHk0ljttdIO19Bn2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFSmtb%252FI&md5=1900310c924bce5b79d71008e6e786fb</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01447%26sid%3Dliteratum%253Aachs%26aulast%3DArntson%26aufirst%3DK.%2BE.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DProtein-Observed%2520Fluorine%2520NMR%253A%2520A%2520Bioorthogonal%2520Approach%2520for%2520Small%2520Molecule%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D11%26spage%3D5158%26epage%3D5171%26doi%3D10.1021%2Facs.jmedchem.5b01447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishima, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torchia, D. A.</span></span> <span> </span><span class="NLM_article-title">Protein Dynamics from NMR</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">743</span>, <span class="refDoi"> DOI: 10.1038/78963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2F78963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10966641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFWlu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=740-743&issue=9&author=R.+Ishimaauthor=D.+A.+Torchia&title=Protein+Dynamics+from+NMR&doi=10.1038%2F78963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics from NMR</span></div><div class="casAuthors">Ishima, Rieko; Torchia, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">740-743</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">A review with 66 refs.  This review surveys recent investigations of conformational fluctuations of proteins in soln. using NMR techniques.  Advances in exptl. methods have provided more accurate means of characterizing fast and slow internal motions as well as overall diffusion.  The information obtained from NMR dynamics expts. provides insights into specific structural changes or configurational energetics assocd. with function.  A variety of applications illustrate that studies of protein dynamics provide insights into protein-protein interactions, target recognition, ligand binding, and enzyme function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9AJMPUp5LpbVg90H21EOLACvtfcHk0lg3OIL8icBqdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFWlu7o%253D&md5=5c05fbb0bbc273789c5e45e1adc9497c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2F78963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F78963%26sid%3Dliteratum%253Aachs%26aulast%3DIshima%26aufirst%3DR.%26aulast%3DTorchia%26aufirst%3DD.%2BA.%26atitle%3DProtein%2520Dynamics%2520from%2520NMR%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D7%26issue%3D9%26spage%3D740%26epage%3D743%26doi%3D10.1038%2F78963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Analysis of Water Stability in Bromodomain Binding Pockets with Grand Canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=29863194" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=19&issue=1&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-Scale+Analysis+of+Water+Stability+in+Bromodomain+Binding+Pockets+with+Grand+Canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-Scale%2520Analysis%2520of%2520Water%2520Stability%2520in%2520Bromodomain%2520Binding%2520Pockets%2520with%2520Grand%2520Canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26issue%3D1%26spage%3D19%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bharatham, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavish, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shadrick, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelat, A. A.</span></span> <span> </span><span class="NLM_article-title">The Role of ZA Channel Water-Mediated Interactions in the Design of Bromodomain-Selective BET Inhibitors</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2018.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.jmgm.2018.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=29605436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1Gks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=197-210&author=N.+Bharathamauthor=P.+J.+Slavishauthor=W.+R.+Shadrickauthor=B.+M.+Youngauthor=A.+A.+Shelat&title=The+Role+of+ZA+Channel+Water-Mediated+Interactions+in+the+Design+of+Bromodomain-Selective+BET+Inhibitors&doi=10.1016%2Fj.jmgm.2018.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ZA channel water-mediated interactions in the design of bromodomain-selective BET inhibitors</span></div><div class="casAuthors">Bharatham, Nagakumar; Slavish, Peter J.; Young, Brandon M.; Shelat, Anang A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">197-210</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Bromodomain and Extra-Terminal domain (BET) family of proteins are involved in the regulation of gene transcription, and their dysregulation is implicated in several diseases including cancer.  BET proteins contain two tandem bromodomains (BD1 and BD2) that independently recognize acetylated-lysine residues and appear to have distinct biol. roles.  We compared several published co-crystal structures and found five positions near the substrate binding pocket that vary between BET bromodomains.  One position located in the ZA loop has unique properties.  In BRD2-4, this residue is glutamine in BD1 and lysine in BD2; in BRDT, this residue is arginine in BD1 and asparagine in BD2.  Using mol. modeling, we identified differences in the water-mediated network at this position between bromodomains.  Mol. dynamics simulations helped rationalize the obsd. bromodomain selectivity for exemplar BET inhibitors and a congeneric series of tetrahydroquinolines (THQ) that differed by a single heteroatom near the ZA channel.  The 2-furan SJ830599, the most BD2-selective THQ analog, did not disrupt the water-mediated networks in either domain, but was electrostatically-repulsed by the specific arrangement of the W5 water dipole in BD1.  Our work underscores the value of exploring water-mediated interactions to study ligand binding, and highlights the difficulty of optimizing polar interactions due to high desolvation penalties.  Finally, we suggest further modifications to THQ-based BET inhibitors that would increase BD2-selectivity in BRD2-4, while minimizing affinity for one or both bromodomains of BRDT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptIhxYNtOqIbVg90H21EOLACvtfcHk0lg3OIL8icBqdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1Gks7w%253D&md5=8db9f397aa72ae776017ffe829aa30ce</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2018.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2018.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DBharatham%26aufirst%3DN.%26aulast%3DSlavish%26aufirst%3DP.%2BJ.%26aulast%3DShadrick%26aufirst%3DW.%2BR.%26aulast%3DYoung%26aufirst%3DB.%2BM.%26aulast%3DShelat%26aufirst%3DA.%2BA.%26atitle%3DThe%2520Role%2520of%2520ZA%2520Channel%2520Water-Mediated%2520Interactions%2520in%2520the%2520Design%2520of%2520Bromodomain-Selective%2520BET%2520Inhibitors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2018%26volume%3D81%26spage%3D197%26epage%3D210%26doi%3D10.1016%2Fj.jmgm.2018.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, K.</span></span> <span> </span><span class="NLM_article-title">The Polar Warhead of a TRIM24 Bromodomain Inhibitor Rearranges a Water-Mediated Interaction Network</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>284</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1082</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1111/febs.14041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1111%2Ffebs.14041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2017&pages=1082-1095&issue=7&author=J.+Liuauthor=F.+Liauthor=H.+Baoauthor=Y.+Jiangauthor=S.+Zhangauthor=R.+Maauthor=J.+Gaoauthor=J.+Wuauthor=K.+Ruan&title=The+Polar+Warhead+of+a+TRIM24+Bromodomain+Inhibitor+Rearranges+a+Water-Mediated+Interaction+Network&doi=10.1111%2Ffebs.14041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1111%2Ffebs.14041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.14041%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBao%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRuan%26aufirst%3DK.%26atitle%3DThe%2520Polar%2520Warhead%2520of%2520a%2520TRIM24%2520Bromodomain%2520Inhibitor%2520Rearranges%2520a%2520Water-Mediated%2520Interaction%2520Network%26jtitle%3DFEBS%2520J.%26date%3D2017%26volume%3D284%26issue%3D7%26spage%3D1082%26epage%3D1095%26doi%3D10.1111%2Ffebs.14041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canagarajah, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassisà, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobb, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Meguid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsmith, E. J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Inhibitor Selectivity in MAP Kinases</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1117</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(98)00113-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2FS0969-2126%2898%2900113-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=1117-1128&issue=9&author=Z.+Wangauthor=B.+J.+Canagarajahauthor=J.+C.+Boehmauthor=S.+Kassis%C3%A0author=M.+H.+Cobbauthor=P.+R.+Youngauthor=S.+Abdel-Meguidauthor=J.+L.+Adamsauthor=E.+J.+Goldsmith&title=Structural+Basis+of+Inhibitor+Selectivity+in+MAP+Kinases&doi=10.1016%2FS0969-2126%2898%2900113-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2898%2900113-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252898%252900113-0%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DCanagarajah%26aufirst%3DB.%2BJ.%26aulast%3DBoehm%26aufirst%3DJ.%2BC.%26aulast%3DKassis%25C3%25A0%26aufirst%3DS.%26aulast%3DCobb%26aufirst%3DM.%2BH.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DAbdel-Meguid%26aufirst%3DS.%26aulast%3DAdams%26aufirst%3DJ.%2BL.%26aulast%3DGoldsmith%26aufirst%3DE.%2BJ.%26atitle%3DStructural%2520Basis%2520of%2520Inhibitor%2520Selectivity%2520in%2520MAP%2520Kinases%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26issue%3D9%26spage%3D1117%26epage%3D1128%26doi%3D10.1016%2FS0969-2126%2898%2900113-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span> <span> </span><span class="NLM_article-title">The Cellular Thermal Shift Assay for Evaluating Drug Target Interactions in Cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&issue=9&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundb%C3%A4ckauthor=P.+Nordlundauthor=D.+M.+Molina&title=The+Cellular+Thermal+Shift+Assay+for+Evaluating+Drug+Target+Interactions+in+Cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0liMOn0oi7YZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26atitle%3DThe%2520Cellular%2520Thermal%2520Shift%2520Assay%2520for%2520Evaluating%2520Drug%2520Target%2520Interactions%2520in%2520Cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Design and Characterization of Bivalent BET Inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1096</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnchembio.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=27775715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1089-1096&issue=12&author=M.+Tanakaauthor=J.+M.+Robertsauthor=H.-S.+Seoauthor=A.+Souzaauthor=J.+Paulkauthor=T.+G.+Scottauthor=S.+L.+DeAngeloauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Design+and+Characterization+of+Bivalent+BET+Inhibitors&doi=10.1038%2Fnchembio.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of bivalent BET inhibitors</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Seo, Hyuk-Soo; Souza, Amanda; Paulk, Joshiawa; Scott, Thomas G.; DeAngelo, Stephen L.; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1089-1096</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cellular signaling is often propagated by multivalent interactions.  Multivalency creates avidity, allowing stable biophys. recognition.  Multivalency is an attractive strategy for achieving potent binding to protein targets, as the affinity of bivalent ligands is often greater than the sum of monovalent affinities.  The bromodomain and extraterminal domain (BET) family of transcriptional coactivators features tandem bromodomains through which BET proteins bind acetylated histones and transcription factors.  All reported antagonists of the BET protein BRD4 bind in a monovalent fashion.  Here the authors describe, to the knowledge for the first time, a bivalent BET bromodomain inhibitor-MT1-which has unprecedented potency.  Biophys. and biochem. studies suggest MT1 is an intramol. bivalent BRD4 binder that is more than 100-fold more potent, in cellular assays, than the corresponding monovalent antagonist, JQ1.  MT1 significantly (P < 0.05) delayed leukemia progression in mice, as compared to JQ1.  These data qualify a powerful chem. probe for BET bromodomains and a rationale for further development of multidomain inhibitors of epigenetic reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQwc6X0VYYbVg90H21EOLACvtfcHk0liMOn0oi7YZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurrM&md5=7319203dac409d5dee3c580b49fcf185</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2209%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDesign%2520and%2520Characterization%2520of%2520Bivalent%2520BET%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26issue%3D12%26spage%3D1089%26epage%3D1096%26doi%3D10.1038%2Fnchembio.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span> (<span class="NLM_issue">6141</span>),  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&issue=6141&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+Drug+Target+Engagement+in+Cells+and+Tissues+Using+the+Cellular+Thermal+Shift+Assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0liMOn0oi7YZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520Drug%2520Target%2520Engagement%2520in%2520Cells%2520and%2520Tissues%2520Using%2520the%2520Cellular%2520Thermal%2520Shift%2520Assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26issue%3D6141%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin, B. M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-Containing Protein 4 – Independent Transcriptional Activation by Autoimmune Regulator (AIRE) and NF-κB</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">4993</span>– <span class="NLM_lpage">5004</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA117.001518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1074%2Fjbc.RA117.001518" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=4993-5004&issue=14&author=F.+Huangauthor=W.+Shaoauthor=K.+Fujinagaauthor=B.+M.+Peterlin&title=Bromodomain-Containing+Protein+4+%E2%80%93+Independent+Transcriptional+Activation+by+Autoimmune+Regulator+%28AIRE%29+and+NF-%CE%BAB&doi=10.1074%2Fjbc.RA117.001518"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA117.001518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA117.001518%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DF.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DFujinaga%26aufirst%3DK.%26aulast%3DPeterlin%26aufirst%3DB.%2BM.%26atitle%3DBromodomain-Containing%2520Protein%25204%2520%25E2%2580%2593%2520Independent%2520Transcriptional%2520Activation%2520by%2520Autoimmune%2520Regulator%2520%2528AIRE%2529%2520and%2520NF-%25CE%25BAB%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26issue%3D14%26spage%3D4993%26epage%3D5004%26doi%3D10.1074%2Fjbc.RA117.001518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberspächer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmees, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span> <span> </span><span class="NLM_article-title">Affinity Map of Bromodomain Protein 4 (BRD4) Interactions with the Histone H4 Tail and the Small Molecule Inhibitor JQ1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">9304</span>– <span class="NLM_lpage">9319</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.523019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1074%2Fjbc.M113.523019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=24497639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVWjs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=9304-9319&issue=13&author=M.+Jungauthor=M.+Philpottauthor=S.+M%C3%BCllerauthor=J.+Schulzeauthor=V.+Badockauthor=U.+Ebersp%C3%A4cherauthor=D.+Moosmayerauthor=B.+Baderauthor=N.+Schmeesauthor=A.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=B.+Haendler&title=Affinity+Map+of+Bromodomain+Protein+4+%28BRD4%29+Interactions+with+the+Histone+H4+Tail+and+the+Small+Molecule+Inhibitor+JQ1&doi=10.1074%2Fjbc.M113.523019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Affinity Map of Bromodomain Protein 4 (BRD4) Interactions with the Histone H4 Tail and the Small Molecule Inhibitor JQ1</span></div><div class="casAuthors">Jung, Marie; Philpott, Martin; Mueller, Susanne; Schulze, Jessica; Badock, Volker; Eberspaecher, Uwe; Moosmayer, Dieter; Bader, Benjamin; Schmees, Norbert; Fernandez-Montalvan, Amaury; Haendler, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9304-9319</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Bromodomain protein 4 (BRD4) is a member of the bromodomain and extra-terminal domain (BET) protein family.  It binds to acetylated histone tails via its tandem bromodomains BD1 and BD2 and forms a complex with the pos. transcription elongation factor b, which controls phosphorylation of RNA polymerase II, ultimately leading to stimulation of transcription elongation.  An essential role of BRD4 in cell proliferation and cancer growth has been reported in several recent studies.  We analyzed the binding of BRD4 BD1 and BD2 to different partners and showed that the strongest interactions took place with di- and tetra-acetylated peptides derived from the histone 4 N-terminal tail.  We also found that several histone 4 residues neighboring the acetylated lysines significantly influenced binding.  We generated 10 different BRD4 BD1 mutants and analyzed their affinities to acetylated histone tails and to the BET inhibitor JQ1 using several complementary biochem. and biophys. methods.  The impact of these mutations was confirmed in a cellular environment.  Altogether, the results show that Trp-81, Tyr-97, Asn-140, and Met-149 play similarly important roles in the recognition of acetylated histones and JQ1.  Pro-82, Leu-94, Asp-145, and Ile-146 have a more differentiated role, suggesting that different kinds of interactions take place and that resistance mutations compatible with BRD4 function are possible.  Our study extends the knowledge on the contribution of individual BRD4 amino acids to histone and JQ1 binding and may help in the design of new BET antagonists with improved pharmacol. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8wAGYLccT1bVg90H21EOLACvtfcHk0liOJbjQpYwXgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVWjs70%253D&md5=bf1d882e680e82660065b3e7b0a0f0f9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.523019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.523019%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchulze%26aufirst%3DJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DEbersp%25C3%25A4cher%26aufirst%3DU.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DSchmees%26aufirst%3DN.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%26aulast%3DHaendler%26aufirst%3DB.%26atitle%3DAffinity%2520Map%2520of%2520Bromodomain%2520Protein%25204%2520%2528BRD4%2529%2520Interactions%2520with%2520the%2520Histone%2520H4%2520Tail%2520and%2520the%2520Small%2520Molecule%2520Inhibitor%2520JQ1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26issue%3D13%26spage%3D9304%26epage%3D9319%26doi%3D10.1074%2Fjbc.M113.523019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekido, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akahoshi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukaida, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, K.</span></span> <span> </span><span class="NLM_article-title">Essential Involvement of Interleukin-8 (IL-8) in Acute Inflammation</span>. <i>J. Leukocyte Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">564</span>, <span class="refDoi"> DOI: 10.1002/jlb.56.5.559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1002%2Fjlb.56.5.559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=7964163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADyaK2cXmvFymtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1994&pages=559-564&issue=5&author=A.+Haradaauthor=N.+Sekidoauthor=T.+Akahoshiauthor=T.+Wadaauthor=N.+Mukaidaauthor=K.+Matsushima&title=Essential+Involvement+of+Interleukin-8+%28IL-8%29+in+Acute+Inflammation&doi=10.1002%2Fjlb.56.5.559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Essential involvement of interleukin-8 (IL-8) in acute inflammation</span></div><div class="casAuthors">Harada, Akihisa; Sekido, Nobuaki; Akahoshi, Tohru; Wada, Takashi; Mukaida, Naofumi; Matsushima, Kouji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">559-64</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A review with 48 refs. summarizing the effects of anti-interleukin-8 antibody on rabbit exptl. models of acute inflammation, including lipopolysaccharide-induced dermatitis, lipopolysaccharide/interleukin-1-induced arthritis, ischemia-reperfusion injury in lung, and acute immune complex-type glomerulonephritis.  The authors also discuss the potential use of anti-interleukin-8 antibody for the treatment of inflammatory reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaVYZTnD1V9rVg90H21EOLACvtfcHk0liOJbjQpYwXgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmvFymtbo%253D&md5=03b803d7d0d920fdbe6de0a8f7b42260</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjlb.56.5.559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjlb.56.5.559%26sid%3Dliteratum%253Aachs%26aulast%3DHarada%26aufirst%3DA.%26aulast%3DSekido%26aufirst%3DN.%26aulast%3DAkahoshi%26aufirst%3DT.%26aulast%3DWada%26aufirst%3DT.%26aulast%3DMukaida%26aufirst%3DN.%26aulast%3DMatsushima%26aufirst%3DK.%26atitle%3DEssential%2520Involvement%2520of%2520Interleukin-8%2520%2528IL-8%2529%2520in%2520Acute%2520Inflammation%26jtitle%3DJ.%2520Leukocyte%2520Biol.%26date%3D1994%26volume%3D56%26issue%3D5%26spage%3D559%26epage%3D564%26doi%3D10.1002%2Fjlb.56.5.559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hexum, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummond, K. M.</span></span> <span> </span><span class="NLM_article-title">A Redox Economical Synthesis of Bioactive 6,12-Guaianolides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2644</span>– <span class="NLM_lpage">2647</span>, <span class="refDoi"> DOI: 10.1021/ol400904y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol400904y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=2644-2647&issue=11&author=B.+Wenauthor=J.+K.+Hexumauthor=J.+C.+Widenauthor=D.+A.+Harkiauthor=K.+M.+Brummond&title=A+Redox+Economical+Synthesis+of+Bioactive+6%2C12-Guaianolides&doi=10.1021%2Fol400904y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fol400904y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol400904y%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DB.%26aulast%3DHexum%26aufirst%3DJ.%2BK.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DBrummond%26aufirst%3DK.%2BM.%26atitle%3DA%2520Redox%2520Economical%2520Synthesis%2520of%2520Bioactive%25206%252C12-Guaianolides%26jtitle%3DOrg.%2520Lett.%26date%3D2013%26volume%3D15%26issue%3D11%26spage%3D2644%26epage%3D2647%26doi%3D10.1021%2Fol400904y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of Inflammation by a Synthetic Histone Mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span> (<span class="NLM_issue">7327</span>),  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&issue=7327&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+Inflammation+by+a+Synthetic+Histone+Mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0liOJbjQpYwXgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520Inflammation%2520by%2520a%2520Synthetic%2520Histone%2520Mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26issue%3D7327%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolajczyk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>190</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">3670</span>– <span class="NLM_lpage">3678</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1202838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670-3678&issue=7&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+Protein+Function+Is+Required+for+Inflammation%3A+Brd2+Genetic+Disruption+and+BET+Inhibitor+JQ1+Impair+Mouse+Macrophage+Inflammatory+Responses&doi=10.4049%2Fjimmunol.1202838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-α while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0ljNBoM8mZZHQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520Protein%2520Function%2520Is%2520Required%2520for%2520Inflammation%253A%2520Brd2%2520Genetic%2520Disruption%2520and%2520BET%2520Inhibitor%2520JQ1%2520Impair%2520Mouse%2520Macrophage%2520Inflammatory%2520Responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26issue%3D7%26spage%3D3670%26epage%3D3678%26doi%3D10.4049%2Fjimmunol.1202838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milletti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermann, J. C.</span></span> <span> </span><span class="NLM_article-title">Targeted Kinase Selectivity from Kinase Profiling Data</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1021/ml300012r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300012r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=383-386&issue=5&author=F.+Millettiauthor=J.+C.+Hermann&title=Targeted+Kinase+Selectivity+from+Kinase+Profiling+Data&doi=10.1021%2Fml300012r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fml300012r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300012r%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26atitle%3DTargeted%2520Kinase%2520Selectivity%2520from%2520Kinase%2520Profiling%2520Data%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26issue%3D5%26spage%3D383%26epage%3D386%26doi%3D10.1021%2Fml300012r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&issue=11&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0ljNBoM8mZZHQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26issue%3D11%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibel, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laydon, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenthal, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fier-Thompson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abt, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soreson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smietana, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garigipati, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erhard, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krog, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheldrake, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. L.</span></span> <span> </span><span class="NLM_article-title">Regulation of Stress-Induced Cytokine Production by Pyridinylimidazoles; Inhibition of CSBP Kinase</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(96)00212-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2FS0968-0896%2896%2900212-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=49-64&issue=1&author=T.+F.+Gallagherauthor=G.+L.+Seibelauthor=S.+Kassisauthor=J.+T.+Laydonauthor=M.+J.+Blumenthalauthor=J.+C.+Leeauthor=D.+Leeauthor=J.+C.+Boehmauthor=S.+M.+Fier-Thompsonauthor=J.+W.+Abtauthor=M.+E.+Soresonauthor=J.+M.+Smietanaauthor=R.+F.+Hallauthor=R.+S.+Garigipatiauthor=P.+E.+Benderauthor=K.+F.+Erhardauthor=A.+J.+Krogauthor=G.+A.+Hofmannauthor=P.+L.+Sheldrakeauthor=P.+C.+McDonnellauthor=S.+Kumarauthor=P.+R.+Youngauthor=J.+L.+Adams&title=Regulation+of+Stress-Induced+Cytokine+Production+by+Pyridinylimidazoles%3B+Inhibition+of+CSBP+Kinase&doi=10.1016%2FS0968-0896%2896%2900212-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2896%2900212-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252896%252900212-X%26sid%3Dliteratum%253Aachs%26aulast%3DGallagher%26aufirst%3DT.%2BF.%26aulast%3DSeibel%26aufirst%3DG.%2BL.%26aulast%3DKassis%26aufirst%3DS.%26aulast%3DLaydon%26aufirst%3DJ.%2BT.%26aulast%3DBlumenthal%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DBoehm%26aufirst%3DJ.%2BC.%26aulast%3DFier-Thompson%26aufirst%3DS.%2BM.%26aulast%3DAbt%26aufirst%3DJ.%2BW.%26aulast%3DSoreson%26aufirst%3DM.%2BE.%26aulast%3DSmietana%26aufirst%3DJ.%2BM.%26aulast%3DHall%26aufirst%3DR.%2BF.%26aulast%3DGarigipati%26aufirst%3DR.%2BS.%26aulast%3DBender%26aufirst%3DP.%2BE.%26aulast%3DErhard%26aufirst%3DK.%2BF.%26aulast%3DKrog%26aufirst%3DA.%2BJ.%26aulast%3DHofmann%26aufirst%3DG.%2BA.%26aulast%3DSheldrake%26aufirst%3DP.%2BL.%26aulast%3DMcDonnell%26aufirst%3DP.%2BC.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DAdams%26aufirst%3DJ.%2BL.%26atitle%3DRegulation%2520of%2520Stress-Induced%2520Cytokine%2520Production%2520by%2520Pyridinylimidazoles%253B%2520Inhibition%2520of%2520CSBP%2520Kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1997%26volume%3D5%26issue%3D1%26spage%3D49%26epage%3D64%26doi%3D10.1016%2FS0968-0896%2896%2900212-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9305</span>– <span class="NLM_lpage">9320</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00438</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00438" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9305-9320&issue=20&author=L.+Carlinoauthor=G.+Rastelli&title=Dual+Kinase-Bromodomain+Inhibitors+in+Anticancer+Drug+Discovery%3A+A+Structural+and+Pharmacological+Perspective&doi=10.1021%2Facs.jmedchem.6b00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span></div><div class="casAuthors">Carlino, Luca; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9305-9320</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases play crucial roles in several cell transformation processes and are validated drug targets for many human diseases, including cancer.  Nevertheless, most tumors have eluded the effects of inhibition of a single kinase by activating resistance mechanisms and/or alternative pathways and escape mechanisms.  In recent years, multitarget approaches directed toward inhibition of kinases and targets of different families have received increasing attention.  In particular, co-targeting kinases and bromodomain epigenetic reader proteins has rapidly emerged as a promising approach to cancer drug development.  In this manuscript, we will review the recent discoveries that led to the identification and optimization of dual kinase/bromodomain inhibitors.  We will analyze and compare the structural features required for dual inhibition and comment on the potential of this approach in anticancer drug discovery.  Moreover, we will introduce computational approaches useful for the identification of dual kinase/bromodomain inhibitors and generate ad hoc pharmacophore and docking models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkHDSBWCGdbVg90H21EOLACvtfcHk0lhAVLLcxjK4jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP&md5=1351a235bba09782160a86ae3f904833</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00438%26sid%3Dliteratum%253Aachs%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520in%2520Anticancer%2520Drug%2520Discovery%253A%2520A%2520Structural%2520and%2520Pharmacological%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D20%26spage%3D9305%26epage%3D9320%26doi%3D10.1021%2Facs.jmedchem.6b00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlich, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durden, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Dual-Activity PI3K–BRD4 Inhibitor for the Orthogonal Inhibition of MYC to Block Tumor Growth and Metastasis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">E1072</span>– <span class="NLM_lpage">E1080</span>, <span class="refDoi"> DOI: 10.1073/pnas.1613091114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1073%2Fpnas.1613091114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E1072-E1080&issue=7&author=F.+H.+Andrewsauthor=A.+R.+Singhauthor=S.+Joshiauthor=C.+A.+Smithauthor=G.+A.+Moralesauthor=J.+R.+Garlichauthor=D.+L.+Durdenauthor=T.+G.+Kutateladze&title=Dual-Activity+PI3K%E2%80%93BRD4+Inhibitor+for+the+Orthogonal+Inhibition+of+MYC+to+Block+Tumor+Growth+and+Metastasis&doi=10.1073%2Fpnas.1613091114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1613091114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1613091114%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DF.%2BH.%26aulast%3DSingh%26aufirst%3DA.%2BR.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DC.%2BA.%26aulast%3DMorales%26aufirst%3DG.%2BA.%26aulast%3DGarlich%26aufirst%3DJ.%2BR.%26aulast%3DDurden%26aufirst%3DD.%2BL.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DDual-Activity%2520PI3K%25E2%2580%2593BRD4%2520Inhibitor%2520for%2520the%2520Orthogonal%2520Inhibition%2520of%2520MYC%2520to%2520Block%2520Tumor%2520Growth%2520and%2520Metastasis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26issue%3D7%26spage%3DE1072%26epage%3DE1080%26doi%3D10.1073%2Fpnas.1613091114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. L.</span>; <span class="NLM_string-name">Sheldrake, P. W.</span>; <span class="NLM_string-name">Gallagher, T. F.</span>; <span class="NLM_string-name">Garigipati, R.</span></span> <span> </span><span class="NLM_article-title">Novel 1,4,5-Substituted Imidazole Compounds and Compositions for Use in Therapy as Cytokine Inhibitors</span>. U.S. Patent <span class="NLM_patent">US5593992A</span>, <span class="NLM_year">1995</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&author=J.+L.+Adams&author=P.+W.+Sheldrake&author=T.+F.+Gallagher&author=R.+Garigipati&title=Novel+1%2C4%2C5-Substituted+Imidazole+Compounds+and+Compositions+for+Use+in+Therapy+as+Cytokine+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%2BL.%26atitle%3DNovel%25201%252C4%252C5-Substituted%2520Imidazole%2520Compounds%2520and%2520Compositions%2520for%2520Use%2520in%2520Therapy%2520as%2520Cytokine%2520Inhibitors%26date%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryzhakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarret, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouklovsky, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, G.</span></span> <span> </span><span class="NLM_article-title">Radical-Mediated Dearomatization of Indoles with Sulfinate Reagents for the Synthesis of Fluorinated Spirocyclic Indolines</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">6336</span>– <span class="NLM_lpage">6339</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.7b03155</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.7b03155" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=6336-6339&issue=23&author=D.+Ryzhakovauthor=M.+Jarretauthor=R.+Guillotauthor=C.+Kouklovskyauthor=G.+Vincent&title=Radical-Mediated+Dearomatization+of+Indoles+with+Sulfinate+Reagents+for+the+Synthesis+of+Fluorinated+Spirocyclic+Indolines&doi=10.1021%2Facs.orglett.7b03155"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.7b03155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.7b03155%26sid%3Dliteratum%253Aachs%26aulast%3DRyzhakov%26aufirst%3DD.%26aulast%3DJarret%26aufirst%3DM.%26aulast%3DGuillot%26aufirst%3DR.%26aulast%3DKouklovsky%26aufirst%3DC.%26aulast%3DVincent%26aufirst%3DG.%26atitle%3DRadical-Mediated%2520Dearomatization%2520of%2520Indoles%2520with%2520Sulfinate%2520Reagents%2520for%2520the%2520Synthesis%2520of%2520Fluorinated%2520Spirocyclic%2520Indolines%26jtitle%3DOrg.%2520Lett.%26date%3D2017%26volume%3D19%26issue%3D23%26spage%3D6336%26epage%3D6339%26doi%3D10.1021%2Facs.orglett.7b03155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.</span>; <span class="NLM_string-name">Erdman, P. E.</span>; <span class="NLM_string-name">Fung, L.</span>; <span class="NLM_string-name">Mercurio, F.</span>; <span class="NLM_string-name">Sullivan, R.</span>; <span class="NLM_string-name">Torres, E.</span></span> <span> </span><span class="NLM_article-title">Compounds Targeting Proteins, Compositions, Methods, and Uses Thereof</span>. U.S. Patent <span class="NLM_patent">20180170948</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=K.+Chan&author=P.+E.+Erdman&author=L.+Fung&author=F.+Mercurio&author=R.+Sullivan&author=E.+Torres&title=Compounds+Targeting+Proteins%2C+Compositions%2C+Methods%2C+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.%26atitle%3DCompounds%2520Targeting%2520Proteins%252C%2520Compositions%252C%2520Methods%252C%2520and%2520Uses%2520Thereof%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erijman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernheim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shifman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peleg, Y.</span></span> <span> </span><span class="NLM_article-title">Transfer-PCR (TPCR): A Highway for DNA Cloning and Protein Engineering</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>175</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2011.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.jsb.2011.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21515384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslWnu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2011&pages=171-177&issue=2&author=A.+Erijmanauthor=A.+Dantesauthor=R.+Bernheimauthor=J.+M.+Shifmanauthor=Y.+Peleg&title=Transfer-PCR+%28TPCR%29%3A+A+Highway+for+DNA+Cloning+and+Protein+Engineering&doi=10.1016%2Fj.jsb.2011.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Transfer-PCR (TPCR): A highway for DNA cloning and protein engineering</span></div><div class="casAuthors">Erijman, Ariel; Dantes, Ada; Bernheim, Reut; Shifman, Julia M.; Peleg, Yoav</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-177</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNA cloning and protein engineering are basic methodologies employed for various applications in all life-science disciplines.  Manipulations of DNA however, could be a lengthy process that slows down subsequent expts.  To facilitate both DNA cloning and protein engineering, Transfer-PCR (TPCR) is presented, a novel approach that integrates in a single tube, PCR amplification of the target DNA from an origin vector and its subsequent integration into the destination vector.  TPCR can be applied for incorporation of DNA fragments into any desired position within a circular plasmid without the need for purifn. of the intermediate PCR product and without the use of any com. kit.  Several examples demonstrate the applicability of the TPCR platform for both DNA cloning and for multiple-site targeted mutagenesis.  In both cases, the TPCR reaction was most efficient within a narrow range of primer concns.  In mutagenesis, TPCR was primarily advantageous for generation of combinatorial libraries of targeted mutants but could be also applied to generation of variants with specific multiple mutations throughout the target gene.  Adaptation of the TPCR platform should facilitate, simplify and significantly reduce time and costs for diverse protein structure and functional studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojVuamZPNVCrVg90H21EOLACvtfcHk0lhAVLLcxjK4jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslWnu70%253D&md5=8235dc3a1b08a4c5eb2e5d6f1054aab1</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2011.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2011.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DErijman%26aufirst%3DA.%26aulast%3DDantes%26aufirst%3DA.%26aulast%3DBernheim%26aufirst%3DR.%26aulast%3DShifman%26aufirst%3DJ.%2BM.%26aulast%3DPeleg%26aufirst%3DY.%26atitle%3DTransfer-PCR%2520%2528TPCR%2529%253A%2520A%2520Highway%2520for%2520DNA%2520Cloning%2520and%2520Protein%2520Engineering%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D175%26issue%3D2%26spage%3D171%26epage%3D177%26doi%3D10.1016%2Fj.jsb.2011.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&issue=2&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liTlJNKcxXspg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&issue=2&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liTlJNKcxXspg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liTlJNKcxXspg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hexum, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tello-Aburto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struntz, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harned, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span> <span> </span><span class="NLM_article-title">Bicyclic Cyclohexenones as Inhibitors of NF-κB Signaling</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1021/ml300034a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300034a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=459-464&issue=6&author=J.+K.+Hexumauthor=R.+Tello-Aburtoauthor=N.+B.+Struntzauthor=A.+M.+Harnedauthor=D.+A.+Harki&title=Bicyclic+Cyclohexenones+as+Inhibitors+of+NF-%CE%BAB+Signaling&doi=10.1021%2Fml300034a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fml300034a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300034a%26sid%3Dliteratum%253Aachs%26aulast%3DHexum%26aufirst%3DJ.%2BK.%26aulast%3DTello-Aburto%26aufirst%3DR.%26aulast%3DStruntz%26aufirst%3DN.%2BB.%26aulast%3DHarned%26aufirst%3DA.%2BM.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26atitle%3DBicyclic%2520Cyclohexenones%2520as%2520Inhibitors%2520of%2520NF-%25CE%25BAB%2520Signaling%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26issue%3D6%26spage%3D459%26epage%3D464%26doi%3D10.1021%2Fml300034a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 34 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Huarui Cui, Angela S. Carlson, Mary A. Schleiff, Anand Divakaran, Jorden A. Johnson, Caroline R. Buchholz, Huda Zahid, Nora R. Vail, Ke Shi, Hideki Aihara, Daniel A. Harki, Grover P. Miller, Joseph J. Topczewski, <span class="NLM_string-name hlFld-ContribAuthor">William C. K. Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">4-Methyl-1,2,3-Triazoles as N-Acetyl-Lysine Mimics Afford Potent BET Bromodomain Inhibitors with Improved Selectivity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 10497-10511. <a href="https://doi.org/10.1021/acs.jmedchem.1c00933" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00933</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00933%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D4-Methyl-1%25252C2%25252C3-Triazoles%252Bas%252BN-Acetyl-Lysine%252BMimics%252BAfford%252BPotent%252BBET%252BBromodomain%252BInhibitors%252Bwith%252BImproved%252BSelectivity%26aulast%3DCui%26aufirst%3DHuarui%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D24052021%26date%3D08072021%26volume%3D64%26issue%3D14%26spage%3D10497%26epage%3D10511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lee A. Harrison, Stephen J. Atkinson, Anna Bassil, Chun-wa Chung, Paola Grandi, James R. J. Gray, Etienne Levernier, Antonia Lewis, David Lugo, Cassie Messenger, Anne-Marie Michon, Darren J. Mitchell, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Jonathan T. Seal, Simon Taylor, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a Series of N-Methylpyridine-2-carboxamides as Potent and Selective Inhibitors of the Second Bromodomain (BD2) of the Bromo and Extra Terminal Domain (BET) Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02155%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Ba%252BSeries%252Bof%252BN-Methylpyridine-2-carboxamides%252Bas%252BPotent%252Band%252BSelective%252BInhibitors%252Bof%252Bthe%252BSecond%252BBromodomain%252B%252528BD2%252529%252Bof%252Bthe%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BProteins%26aulast%3DHarrison%26aufirst%3DLee%2BA.%26date%3D2021%26date%3D2020%26date%3D14122020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ping Bai, Yu Lan, Hao Wang, Yan Liu, Robin Striar, Gengyang Yuan, Sepideh Afshar, Julia S. Zagaroli, Darcy R. Tocci, Amelia G. Langan, <span class="NLM_string-name hlFld-ContribAuthor">Changning Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Characterization of a Positron Emission Tomography Imaging Probe Selectively Targeting the Second Bromodomain of Bromodomain Protein BRD4. </span><span class="cited-content_cbyCitation_journal-name">Bioconjugate Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.bioconjchem.1c00245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.bioconjchem.1c00245%26sid%3Dliteratum%253Aachs%26jtitle%3DBioconjugate%2520Chemistry%26atitle%3DSynthesis%252Band%252BCharacterization%252Bof%252Ba%252BPositron%252BEmission%252BTomography%252BImaging%252BProbe%252BSelectively%252BTargeting%252Bthe%252BSecond%252BBromodomain%252Bof%252BBromodomain%252BProtein%252BBRD4%26aulast%3DBai%26aufirst%3DPing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D09052021%26date%3D10062021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Juliana R. Alexander, Paul V. Kevorkian, <span class="NLM_string-name hlFld-ContribAuthor">Joseph J. Topczewski</span>. </span><span class="cited-content_cbyCitation_article-title">Silver Mediated Banert Cascade with Carbon Nucleophiles. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2021,</strong> <em>23 </em>
                                    (8)
                                     , 3227-3230. <a href="https://doi.org/10.1021/acs.orglett.1c01032" title="DOI URL">https://doi.org/10.1021/acs.orglett.1c01032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.1c01032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.1c01032%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DSilver%252BMediated%252BBanert%252BCascade%252Bwith%252BCarbon%252BNucleophiles%26aulast%3DAlexander%26aufirst%3DJuliana%2BR.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D25032021%26date%3D02042021%26volume%3D23%26issue%3D8%26spage%3D3227%26epage%3D3230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">En-Chih Liu, <span class="NLM_string-name hlFld-ContribAuthor">Joseph J. Topczewski</span>. </span><span class="cited-content_cbyCitation_article-title">Enantioselective Nickel-Catalyzed Alkyne–Azide Cycloaddition by Dynamic Kinetic Resolution. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (14)
                                     , 5308-5313. <a href="https://doi.org/10.1021/jacs.1c01354" title="DOI URL">https://doi.org/10.1021/jacs.1c01354</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c01354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c01354%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DEnantioselective%252BNickel-Catalyzed%252BAlkyne%2525E2%252580%252593Azide%252BCycloaddition%252Bby%252BDynamic%252BKinetic%252BResolution%26aulast%3DLiu%26aufirst%3DEn-Chih%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D03022021%26date%3D02042021%26volume%3D143%26issue%3D14%26spage%3D5308%26epage%3D5313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Helen E. Aylott, Stephen J. Atkinson, Paul Bamborough, Anna Bassil, Chun-wa Chung, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Cassie Messenger, Darren Mitchell, Alexander Phillipou, Alex Preston, Rab K. Prinjha, Francesco Rianjongdee, Inmaculada Rioja, Jonathan T. Seal, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Template-Hopping Approach Leads to Potent, Selective, and Highly Soluble Bromo and Extraterminal Domain (BET) Second Bromodomain (BD2) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3249-3281. <a href="https://doi.org/10.1021/acs.jmedchem.0c02156" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02156%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTemplate-Hopping%252BApproach%252BLeads%252Bto%252BPotent%25252C%252BSelective%25252C%252Band%252BHighly%252BSoluble%252BBromo%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitors%26aulast%3DAylott%26aufirst%3DHelen%2BE.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D04032021%26volume%3D64%26issue%3D6%26spage%3D3249%26epage%3D3281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pan Tang, Jifa Zhang, Jie Liu, Cheng-Ming Chiang, <span class="NLM_string-name hlFld-ContribAuthor">Liang Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2419-2435. <a href="https://doi.org/10.1021/acs.jmedchem.0c01487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01487%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BBromodomain%252Band%252BExtraterminal%252BProteins%252Bfor%252BDrug%252BDiscovery%25253A%252BFrom%252BCurrent%252BProgress%252Bto%252BTechnological%252BDevelopment%26aulast%3DTang%26aufirst%3DPan%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26082020%26date%3D22022021%26volume%3D64%26issue%3D5%26spage%3D2419%26epage%3D2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martin Hügle, Pierre Regenass, Robin Warstat, Mirjam Hau, Karin Schmidtkunz, Xavier Lucas, Daniel Wohlwend, Oliver Einsle, Manfred Jung, Bernhard Breit, <span class="NLM_string-name hlFld-ContribAuthor">Stefan Günther</span>. </span><span class="cited-content_cbyCitation_article-title">4-Acyl Pyrroles as Dual BET-BRD7/9 Bromodomain Inhibitors Address BETi Insensitive Human Cancer Cell Lines. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15603-15620. <a href="https://doi.org/10.1021/acs.jmedchem.0c00478" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00478</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00478%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D4-Acyl%252BPyrroles%252Bas%252BDual%252BBET-BRD7%25252F9%252BBromodomain%252BInhibitors%252BAddress%252BBETi%252BInsensitive%252BHuman%252BCancer%252BCell%252BLines%26aulast%3DH%25C3%25BCgle%26aufirst%3DMartin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D21032020%26date%3D04122020%26volume%3D63%26issue%3D24%26spage%3D15603%26epage%3D15620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Prakriti Kalra, Logan McGraw, Jennifer R. Kimbrough, Anil K. Pandey, Jonathan Solberg, Huarui Cui, Anand Divakaran, Kristen John, Jon E. Hawkinson, <span class="NLM_string-name hlFld-ContribAuthor">William C. K. Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">Quantifying the Selectivity of Protein–Protein and Small Molecule Interactions with Fluorinated Tandem Bromodomain Reader Proteins. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (11)
                                     , 3038-3049. <a href="https://doi.org/10.1021/acschembio.0c00720" title="DOI URL">https://doi.org/10.1021/acschembio.0c00720</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00720%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DQuantifying%252Bthe%252BSelectivity%252Bof%252BProtein%2525E2%252580%252593Protein%252Band%252BSmall%252BMolecule%252BInteractions%252Bwith%252BFluorinated%252BTandem%252BBromodomain%252BReader%252BProteins%26aulast%3DKalra%26aufirst%3DPrakriti%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07092020%26date%3D19102020%26date%3D03112020%26volume%3D15%26issue%3D11%26spage%3D3038%26epage%3D3049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher R. Wellaway, Paul Bamborough, Sharon G. Bernard, Chun-wa Chung, Peter D. Craggs, Leanne Cutler, Emmanuel H. Demont, John P. Evans, Laurie Gordon, Bhumika Karamshi, Antonia J. Lewis, Matthew J. Lindon, Darren J. Mitchell, Inmaculada Rioja, Peter E. Soden, Simon Taylor, Robert J. Watson, Rob Willis, James M. Woolven, Beata S. Wyspiańska, William J. Kerr, <span class="NLM_string-name hlFld-ContribAuthor">Rab K. Prinjha</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9020-9044. <a href="https://doi.org/10.1021/acs.jmedchem.0c00566" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00566</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ba%252BBromodomain%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BInhibitor%252BSelective%252Bfor%252Bthe%252BN-Terminal%252BBromodomains%252BThat%252BRetains%252Ban%252BAnti-inflammatory%252Band%252BAntiproliferative%252BPhenotype%26aulast%3DWellaway%26aufirst%3DChristopher%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06042020%26date%3D03082020%26volume%3D63%26issue%3D17%26spage%3D9020%26epage%3D9044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert J. Watson, Paul Bamborough, Heather Barnett, Chun-wa Chung, Rob Davis, Laurie Gordon, Paola Grandi, Massimo Petretich, Alex Phillipou, Rab K. Prinjha, Inmaculada Rioja, Peter Soden, Thilo Werner, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9045-9069. <a href="https://doi.org/10.1021/acs.jmedchem.0c00614" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGSK789%25253A%252BA%252BSelective%252BInhibitor%252Bof%252Bthe%252BFirst%252BBromodomains%252B%252528BD1%252529%252Bof%252Bthe%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BProteins%26aulast%3DWatson%26aufirst%3DRobert%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14042020%26date%3D04082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9045%26epage%3D9069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan T. Seal, Stephen J. Atkinson, Helen Aylott, Paul Bamborough, Chun-wa Chung, Royston C. B. Copley, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Matthew Lindon, Cassie Messenger, Anne-Marie Michon, Darren Mitchell, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Simon Taylor, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9093-9126. <a href="https://doi.org/10.1021/acs.jmedchem.0c00796" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00796</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BOptimization%252Bof%252Ba%252BNovel%25252C%252BWeak%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BBromodomain%252BFragment%252BLigand%252Bto%252Ba%252BPotent%252Band%252BSelective%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitor%26aulast%3DSeal%26aufirst%3DJonathan%2BT.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11052020%26date%3D30082020%26date%3D23072020%26volume%3D63%26issue%3D17%26spage%3D9093%26epage%3D9126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kaikai Lv, Weicong Chen, Danqi Chen, Jie Mou, Huijie Zhang, Tiantian Fan, Yanlian Li, Danyan Cao, Xin Wang, Lin Chen, Jingkang Shen, Dongsheng Pei, <span class="NLM_string-name hlFld-ContribAuthor">Bing Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9787-9802. <a href="https://doi.org/10.1021/acs.jmedchem.0c00962" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00962%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRational%252BDesign%252Band%252BEvaluation%252Bof%252B6-%252528Pyrimidin-2-ylamino%252529-3%25252C4-dihydroquinoxalin-2%2525281H%252529-ones%252Bas%252BPolypharmacological%252BInhibitors%252Bof%252BBET%252Band%252BKinases%26aulast%3DLv%26aufirst%3DKaikai%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05062020%26date%3D26082020%26date%3D11082020%26volume%3D63%26issue%3D17%26spage%3D9787%26epage%3D9802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">George S. Sheppard, Le Wang, Steven D. Fidanze, Lisa A. Hasvold, Dachun Liu, John K. Pratt, Chang H. Park, Kenton Longenecker, Wei Qiu, Maricel Torrent, Peter J. Kovar, Mai Bui, Emily Faivre, Xiaoli Huang, Xiaoyu Lin, Denise Wilcox, Lu Zhang, Yu Shen, Daniel H. Albert, Terrance J. Magoc, Ganesh Rajaraman, Warren M. Kati, <span class="NLM_string-name hlFld-ContribAuthor">Keith F. McDaniel</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5585-5623. <a href="https://doi.org/10.1021/acs.jmedchem.0c00628" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00628</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BN-Ethyl-4-%25255B2-%2525284-fluoro-2%25252C6-dimethyl-phenoxy%252529-5-%2525281-hydroxy-1-methyl-ethyl%252529phenyl%25255D-6-methyl-7-oxo-1H-pyrrolo%25255B2%25252C3-c%25255Dpyridine-2-carboxamide%252B%252528ABBV-744%252529%25252C%252Ba%252BBET%252BBromodomain%252BInhibitor%252Bwith%252BSelectivity%252Bfor%252Bthe%252BSecond%252BBromodomain%26aulast%3DSheppard%26aufirst%3DGeorge%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16042020%26date%3D07052020%26date%3D23042020%26volume%3D63%26issue%3D10%26spage%3D5585%26epage%3D5623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael D. Olp, Daniel J. Sprague, Christopher J. Goetz, Stefan G. Kathman, Sarah L. Wynia-Smith, Shifali Shishodia, Steven B. Summers, Ziyang Xu, Alexander V. Statsyuk, <span class="NLM_string-name hlFld-ContribAuthor">Brian C. Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent-Fragment Screening of BRD4 Identifies a Ligandable Site Orthogonal to the Acetyl-Lysine Binding Sites. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (4)
                                     , 1036-1049. <a href="https://doi.org/10.1021/acschembio.0c00058" title="DOI URL">https://doi.org/10.1021/acschembio.0c00058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00058%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DCovalent-Fragment%252BScreening%252Bof%252BBRD4%252BIdentifies%252Ba%252BLigandable%252BSite%252BOrthogonal%252Bto%252Bthe%252BAcetyl-Lysine%252BBinding%252BSites%26aulast%3DOlp%26aufirst%3DMichael%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22012020%26date%3D09032020%26date%3D23032020%26date%3D09032020%26volume%3D15%26issue%3D4%26spage%3D1036%26epage%3D1049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Juliana R. Alexander, Mary H. Packard, Alanna M. Hildebrandt, Amy A. Ott, <span class="NLM_string-name hlFld-ContribAuthor">Joseph J. Topczewski</span>. </span><span class="cited-content_cbyCitation_article-title">Divergent Mechanisms of the Banert Cascade with Propargyl Azides. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>85 </em>
                                    (5)
                                     , 3174-3181. <a href="https://doi.org/10.1021/acs.joc.9b03061" title="DOI URL">https://doi.org/10.1021/acs.joc.9b03061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b03061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b03061%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DDivergent%252BMechanisms%252Bof%252Bthe%252BBanert%252BCascade%252Bwith%252BPropargyl%252BAzides%26aulast%3DAlexander%26aufirst%3DJuliana%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D13112019%26date%3D24012020%26date%3D16012020%26volume%3D85%26issue%3D5%26spage%3D3174%26epage%3D3181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anand Divakaran, Steven E. Kirberger, <span class="NLM_string-name hlFld-ContribAuthor">William C. K. Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">SAR by (Protein-Observed) 19F NMR. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2019,</strong> <em>52 </em>
                                    (12)
                                     , 3407-3418. <a href="https://doi.org/10.1021/acs.accounts.9b00377" title="DOI URL">https://doi.org/10.1021/acs.accounts.9b00377</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.9b00377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.9b00377%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DSAR%252Bby%252B%252528Protein-Observed%252529%252B19F%252BNMR%26aulast%3DDivakaran%26aufirst%3DAnand%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D11072019%26date%3D13112019%26volume%3D52%26issue%3D12%26spage%3D3407%26epage%3D3418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">David Remillard, Dennis L. Buckley, Hyuk-Soo Seo, Fleur M. Ferguson, Sirano Dhe-Paganon, James E. Bradner, <span class="NLM_string-name hlFld-ContribAuthor">Nathanael S. Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Dual Inhibition of TAF1 and BET Bromodomains from the BI-2536 Kinase Inhibitor Scaffold. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (10)
                                     , 1443-1449. <a href="https://doi.org/10.1021/acsmedchemlett.9b00243" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00243</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00243%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDual%252BInhibition%252Bof%252BTAF1%252Band%252BBET%252BBromodomains%252Bfrom%252Bthe%252BBI-2536%252BKinase%252BInhibitor%252BScaffold%26aulast%3DRemillard%26aufirst%3DDavid%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D03062019%26date%3D13092019%26date%3D18092019%26date%3D13092019%26volume%3D10%26issue%3D10%26spage%3D1443%26epage%3D1449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Angela
S. Carlson, Huarui Cui, Anand Divakaran, Jorden A. Johnson, Ryan M. Brunner, William C. K. Pomerantz, <span class="NLM_string-name hlFld-ContribAuthor">Joseph J. Topczewski</span>. </span><span class="cited-content_cbyCitation_article-title">Systematically Mitigating the p38α Activity of Triazole-based BET Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 1296-1301. <a href="https://doi.org/10.1021/acsmedchemlett.9b00227" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00227</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00227%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSystematically%252BMitigating%252Bthe%252Bp38%2525CE%2525B1%252BActivity%252Bof%252BTriazole-based%252BBET%252BInhibitors%26aulast%3DCarlson%26aufirst%3DAngela%2BS.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D20052019%26date%3D02082019%26date%3D07082019%26date%3D02082019%26volume%3D10%26issue%3D9%26spage%3D1296%26epage%3D1301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul Bamborough, Chun-wa Chung, Emmanuel H. Demont, Angela M. Bridges, Peter D. Craggs, David P. Dixon, Peter Francis, Rebecca C. Furze, Paola Grandi, Emma J. Jones, Bhumika Karamshi, Kelly Locke, Simon C. C. Lucas, Anne-Marie Michon, Darren J. Mitchell, Peter Pogány, Rab K. Prinjha, Christina Rau, Ana Maria Roa, Andrew D. Roberts, Robert J. Sheppard, <span class="NLM_string-name hlFld-ContribAuthor">Robert J. Watson</span>. </span><span class="cited-content_cbyCitation_article-title">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (16)
                                     , 7506-7525. <a href="https://doi.org/10.1021/acs.jmedchem.9b00673" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00673</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00673%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BQualified%252BSuccess%25253A%252BDiscovery%252Bof%252Ba%252BNew%252BSeries%252Bof%252BATAD2%252BBromodomain%252BInhibitors%252Bwith%252Ba%252BNovel%252BBinding%252BMode%252BUsing%252BHigh-Throughput%252BScreening%252Band%252BHit%252BQualification%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23042019%26date%3D09082019%26volume%3D62%26issue%3D16%26spage%3D7506%26epage%3D7525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">En-Chih Liu, <span class="NLM_string-name hlFld-ContribAuthor">Joseph J. Topczewski</span>. </span><span class="cited-content_cbyCitation_article-title">Enantioselective Copper Catalyzed Alkyne–Azide Cycloaddition by Dynamic Kinetic Resolution. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2019,</strong> <em>141 </em>
                                    (13)
                                     , 5135-5138. <a href="https://doi.org/10.1021/jacs.9b01091" title="DOI URL">https://doi.org/10.1021/jacs.9b01091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b01091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b01091%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DEnantioselective%252BCopper%252BCatalyzed%252BAlkyne%2525E2%252580%252593Azide%252BCycloaddition%252Bby%252BDynamic%252BKinetic%252BResolution%26aulast%3DLiu%26aufirst%3DEn-Chih%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D29012019%26date%3D21032019%26date%3D19032019%26volume%3D141%26issue%3D13%26spage%3D5135%26epage%3D5138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Choon Kiat  Lim</span>, <span class="hlFld-ContribAuthor ">Motakis  Efthymios</span>, <span class="hlFld-ContribAuthor ">Wilson  Tan</span>, <span class="hlFld-ContribAuthor ">Matias Ilmari  Autio</span>, <span class="hlFld-ContribAuthor ">Zenia  Tiang</span>, <span class="hlFld-ContribAuthor ">Peter Yiqing  Li</span>, <span class="hlFld-ContribAuthor ">Roger Sik Yin  Foo</span>. </span><span class="cited-content_cbyCitation_article-title">Dimethyl sulfoxide (DMSO) enhances direct cardiac reprogramming by inhibiting the bromodomain of coactivators CBP/p300. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular and Cellular Cardiology</span><span> <strong>2021,</strong> <em>160 </em>, 15-26. <a href="https://doi.org/10.1016/j.yjmcc.2021.06.008" title="DOI URL">https://doi.org/10.1016/j.yjmcc.2021.06.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.yjmcc.2021.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.yjmcc.2021.06.008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520and%2520Cellular%2520Cardiology%26atitle%3DDimethyl%252Bsulfoxide%252B%252528DMSO%252529%252Benhances%252Bdirect%252Bcardiac%252Breprogramming%252Bby%252Binhibiting%252Bthe%252Bbromodomain%252Bof%252Bcoactivators%252BCBP%25252Fp300%26aulast%3DLim%26aufirst%3DChoon%2BKiat%26date%3D2021%26volume%3D160%26spage%3D15%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eclair Venturini  Filho</span>, <span class="hlFld-ContribAuthor ">Erick M.C.  Pinheiro</span>, <span class="hlFld-ContribAuthor ">Sergio  Pinheiro</span>, <span class="hlFld-ContribAuthor ">Sandro J.  Greco</span>. </span><span class="cited-content_cbyCitation_article-title">Aminopyrimidines: Recent synthetic procedures and anticancer activities. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2021,</strong> <em>92 </em>, 132256. <a href="https://doi.org/10.1016/j.tet.2021.132256" title="DOI URL">https://doi.org/10.1016/j.tet.2021.132256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2021.132256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2021.132256%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DAminopyrimidines%25253A%252BRecent%252Bsynthetic%252Bprocedures%252Band%252Banticancer%252Bactivities%26aulast%3DFilho%26aufirst%3DEclair%2BVenturini%26date%3D2021%26volume%3D92%26spage%3D132256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shaista  Amin</span>, <span class="hlFld-ContribAuthor ">M. Mumtaz  Alam</span>, <span class="hlFld-ContribAuthor ">Mymoona  Akhter</span>, <span class="hlFld-ContribAuthor ">A. K.  Najmi</span>, <span class="hlFld-ContribAuthor ">Nadeem  Siddiqui</span>, <span class="hlFld-ContribAuthor ">Asif  Husain</span>, <span class="hlFld-ContribAuthor ">M.  Shaquiquzzaman</span>. </span><span class="cited-content_cbyCitation_article-title">A review on synthetic procedures and applications of phosphorus oxychloride (POCl
              3
              ) in the last biennial period (2018–19). </span><span class="cited-content_cbyCitation_journal-name">Phosphorus, Sulfur, and Silicon and the Related Elements</span><span> <strong>2021,</strong> <em>196 </em>
                                    (3)
                                     , 211-274. <a href="https://doi.org/10.1080/10426507.2020.1831499" title="DOI URL">https://doi.org/10.1080/10426507.2020.1831499</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10426507.2020.1831499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10426507.2020.1831499%26sid%3Dliteratum%253Aachs%26jtitle%3DPhosphorus%252C%2520Sulfur%252C%2520and%2520Silicon%2520and%2520the%2520Related%2520Elements%26atitle%3DA%252Breview%252Bon%252Bsynthetic%252Bprocedures%252Band%252Bapplications%252Bof%252Bphosphorus%252Boxychloride%252B%252528POCl%252B3%252B%252529%252Bin%252Bthe%252Blast%252Bbiennial%252Bperiod%252B%2525282018%2525E2%252580%25259319%252529%26aulast%3DAmin%26aufirst%3DShaista%26date%3D2021%26date%3D2020%26volume%3D196%26issue%3D3%26spage%3D211%26epage%3D274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhifeng  Yu</span>, <span class="hlFld-ContribAuthor ">Angela F.  Ku</span>, <span class="hlFld-ContribAuthor ">Justin L.  Anglin</span>, <span class="hlFld-ContribAuthor ">Rajesh  Sharma</span>, <span class="hlFld-ContribAuthor ">Melek Nihan  Ucisik</span>, <span class="hlFld-ContribAuthor ">John C.  Faver</span>, <span class="hlFld-ContribAuthor ">Feng  Li</span>, <span class="hlFld-ContribAuthor ">Pranavanand  Nyshadham</span>, <span class="hlFld-ContribAuthor ">Nicholas  Simmons</span>, <span class="hlFld-ContribAuthor ">Kiran L.  Sharma</span>, <span class="hlFld-ContribAuthor ">Sureshbabu  Nagarajan</span>, <span class="hlFld-ContribAuthor ">Kevin  Riehle</span>, <span class="hlFld-ContribAuthor ">Gundeep  Kaur</span>, <span class="hlFld-ContribAuthor ">Banumathi  Sankaran</span>, <span class="hlFld-ContribAuthor ">Marta  Storl-Desmond</span>, <span class="hlFld-ContribAuthor ">Stephen S.  Palmer</span>, <span class="hlFld-ContribAuthor ">Damian W.  Young</span>, <span class="hlFld-ContribAuthor ">Choel  Kim</span>, <span class="hlFld-ContribAuthor ">Martin M.  Matzuk</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and characterization of bromodomain 2–specific inhibitors of BRDT. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2021,</strong> <em>118 </em>
                                    (9)
                                     , e2021102118. <a href="https://doi.org/10.1073/pnas.2021102118" title="DOI URL">https://doi.org/10.1073/pnas.2021102118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2021102118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2021102118%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DDiscovery%252Band%252Bcharacterization%252Bof%252Bbromodomain%252B2%2525E2%252580%252593specific%252Binhibitors%252Bof%252BBRDT%26aulast%3DYu%26aufirst%3DZhifeng%26date%3D2021%26date%3D2021%26volume%3D118%26issue%3D9%26spage%3De2021102118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huarui  Cui</span>, <span class="hlFld-ContribAuthor ">Anand  Divakaran</span>, <span class="hlFld-ContribAuthor ">Anil K.  Pandey</span>, <span class="hlFld-ContribAuthor ">Jorden A.  Johnson</span>, <span class="hlFld-ContribAuthor ">Huda  Zahid</span>, <span class="hlFld-ContribAuthor ">Zachariah J.  Hoell</span>, <span class="hlFld-ContribAuthor ">Mikael O.  Ellingson</span>, <span class="hlFld-ContribAuthor ">Ke  Shi</span>, <span class="hlFld-ContribAuthor ">Hideki  Aihara</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Harki</span>, <span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">Selective N‐Terminal BET Bromodomain Inhibitors by Targeting Non‐Conserved Residues and Structured Water Displacement**. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (3)
                                     , 1240-1246. <a href="https://doi.org/10.1002/ange.202008625" title="DOI URL">https://doi.org/10.1002/ange.202008625</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202008625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202008625%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DSelective%252BN%2525E2%252580%252590Terminal%252BBET%252BBromodomain%252BInhibitors%252Bby%252BTargeting%252BNon%2525E2%252580%252590Conserved%252BResidues%252Band%252BStructured%252BWater%252BDisplacement%252A%252A%26aulast%3DCui%26aufirst%3DHuarui%26date%3D2021%26date%3D2020%26volume%3D133%26issue%3D3%26spage%3D1240%26epage%3D1246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huarui  Cui</span>, <span class="hlFld-ContribAuthor ">Anand  Divakaran</span>, <span class="hlFld-ContribAuthor ">Anil K.  Pandey</span>, <span class="hlFld-ContribAuthor ">Jorden A.  Johnson</span>, <span class="hlFld-ContribAuthor ">Huda  Zahid</span>, <span class="hlFld-ContribAuthor ">Zachariah J.  Hoell</span>, <span class="hlFld-ContribAuthor ">Mikael O.  Ellingson</span>, <span class="hlFld-ContribAuthor ">Ke  Shi</span>, <span class="hlFld-ContribAuthor ">Hideki  Aihara</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Harki</span>, <span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">Selective N‐Terminal BET Bromodomain Inhibitors by Targeting Non‐Conserved Residues and Structured Water Displacement**. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (3)
                                     , 1220-1226. <a href="https://doi.org/10.1002/anie.202008625" title="DOI URL">https://doi.org/10.1002/anie.202008625</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202008625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202008625%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DSelective%252BN%2525E2%252580%252590Terminal%252BBET%252BBromodomain%252BInhibitors%252Bby%252BTargeting%252BNon%2525E2%252580%252590Conserved%252BResidues%252Band%252BStructured%252BWater%252BDisplacement%252A%252A%26aulast%3DCui%26aufirst%3DHuarui%26date%3D2021%26date%3D2020%26volume%3D60%26issue%3D3%26spage%3D1220%26epage%3D1226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ellen E.  Guest</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Pickett</span>, <span class="hlFld-ContribAuthor ">Jonathan D.  Hirst</span>. </span><span class="cited-content_cbyCitation_article-title">Structural variation of protein–ligand complexes of the first bromodomain of BRD4. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>1 </em><a href="https://doi.org/10.1039/D1OB00658D" title="DOI URL">https://doi.org/10.1039/D1OB00658D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1OB00658D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1OB00658D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DStructural%252Bvariation%252Bof%252Bprotein%2525E2%252580%252593ligand%252Bcomplexes%252Bof%252Bthe%252Bfirst%252Bbromodomain%252Bof%252BBRD4%26aulast%3DGuest%26aufirst%3DEllen%2BE.%26date%3D2021%26date%3D2021%26volume%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caroline R.  Buchholz</span>, <span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">19
              F NMR viewed through two different lenses: ligand-observed and protein-observed
              19
              F NMR applications for fragment-based drug discovery. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2021,</strong> <em>68 </em><a href="https://doi.org/10.1039/D1CB00085C" title="DOI URL">https://doi.org/10.1039/D1CB00085C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CB00085C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CB00085C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3D19%252BF%252BNMR%252Bviewed%252Bthrough%252Btwo%252Bdifferent%252Blenses%25253A%252Bligand-observed%252Band%252Bprotein-observed%252B19%252BF%252BNMR%252Bapplications%252Bfor%252Bfragment-based%252Bdrug%252Bdiscovery%26aulast%3DBuchholz%26aufirst%3DCaroline%2BR.%26date%3D2021%26date%3D2021%26volume%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dailin  Liang</span>, <span class="hlFld-ContribAuthor ">Yifan  Yu</span>, <span class="hlFld-ContribAuthor ">Zonghui  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Novel strategies targeting bromodomain-containing protein 4 (BRD4) for cancer drug discovery. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112426. <a href="https://doi.org/10.1016/j.ejmech.2020.112426" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112426</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112426%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252Bstrategies%252Btargeting%252Bbromodomain-containing%252Bprotein%252B4%252B%252528BRD4%252529%252Bfor%252Bcancer%252Bdrug%252Bdiscovery%26aulast%3DLiang%26aufirst%3DDailin%26date%3D2020%26volume%3D200%26spage%3D112426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea  Wisniewski</span>, <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span>. </span><span class="cited-content_cbyCitation_article-title">BET proteins: Investigating BRDT as a potential target for male contraception. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (6)
                                     , 126958. <a href="https://doi.org/10.1016/j.bmcl.2020.126958" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.126958</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.126958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.126958%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DBET%252Bproteins%25253A%252BInvestigating%252BBRDT%252Bas%252Ba%252Bpotential%252Btarget%252Bfor%252Bmale%252Bcontraception%26aulast%3DWisniewski%26aufirst%3DAndrea%26date%3D2020%26volume%3D30%26issue%3D6%26spage%3D126958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Alessia  Lucidi</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic polypharmacology: A new frontier for epi‐drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2020,</strong> <em>40 </em>
                                    (1)
                                     , 190-244. <a href="https://doi.org/10.1002/med.21600" title="DOI URL">https://doi.org/10.1002/med.21600</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21600%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DEpigenetic%252Bpolypharmacology%25253A%252BA%252Bnew%252Bfrontier%252Bfor%252Bepi%2525E2%252580%252590drug%252Bdiscovery%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26date%3D2019%26volume%3D40%26issue%3D1%26spage%3D190%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deheng  Chen</span>, <span class="hlFld-ContribAuthor ">Tian  Lu</span>, <span class="hlFld-ContribAuthor ">Ziqin  Yan</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Feilong  Zhou</span>, <span class="hlFld-ContribAuthor ">Xilin  Lyu</span>, <span class="hlFld-ContribAuthor ">Biling  Xu</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Yujun  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>182 </em>, 111633. <a href="https://doi.org/10.1016/j.ejmech.2019.111633" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111633</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111633%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%25252C%252Bstructural%252Binsight%25252C%252Band%252Bbioactivities%252Bof%252BBY27%252Bas%252Ba%252Bselective%252Binhibitor%252Bof%252Bthe%252Bsecond%252Bbromodomains%252Bof%252BBET%252Bproteins%26aulast%3DChen%26aufirst%3DDeheng%26date%3D2019%26volume%3D182%26spage%3D111633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>, <span class="hlFld-ContribAuthor ">Jorden A.  Johnson</span>, <span class="hlFld-ContribAuthor ">Peter D.  Ycas</span>. </span><span class="cited-content_cbyCitation_article-title">Applied Biophysics for Bromodomain Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 287-337. <a href="https://doi.org/10.1007/7355_2019_79" title="DOI URL">https://doi.org/10.1007/7355_2019_79</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_79%26sid%3Dliteratum%253Aachs%26atitle%3DApplied%252BBiophysics%252Bfor%252BBromodomain%252BDrug%252BDiscovery%26aulast%3DPomerantz%26aufirst%3DWilliam%2BC.%2BK.%26date%3D2019%26date%3D2019%26spage%3D287%26epage%3D337%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0001.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Reported BET bromodomain inhibitors and a new 1,4,5-trisubstituted imidazole, <b>V</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0002.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Competitive-inhibition experiments via fluorescence anisotropy of SB-284851-BT (<b>I</b>) and analogues against BET bromodomains indicating selectivity for BET D1s. (A) IC<sub>50</sub> values of analogues (means ± SEM) determined by competitive-fluorescence-anisotropy assays. Asterisks (*) indicate incomplete inhibition observed; ND indicates not determined. (B) Competition-binding isotherms of compounds <b>IV</b> and <b>V</b> against the first and second bromodomains of BRD4 and BRDT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Common Intermediate <b>17</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Procedure adapted from Boehm et al. and Adams et al.<a onclick="showRef(event, 'ref27 ref51'); return false;" href="javascript:void(0);" class="ref ref27 ref51">(27,51)</a></p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>I</b>, <b>II</b>, and <b>III</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0008.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>IV</b> and <b>V</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0009.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>VI</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0003.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Confirmation of compound <b>V</b> binding BRD4(1) and BRDT(1) and possible structural basis for the D1 over D2 selectivity of the inhibitors. (A) PrOF NMR titrations with bromodomains of BRD4 and BRDT. Samples contain 50 μM <sup>19</sup>F-labeled BRD4(1) or BRDT(1) with increasing concentrations of ligand <b>V</b>. (B) Electron-density maps of ligands (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub>, 1σ) from X-ray structures of BRD4(1) liganded with <b>IV</b> (yellow, 1.74 Å, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH7">6MH7</a>) and <b>V</b> (orange, 1.60 Å, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH1">6MH1</a>). (C) Superposition of compounds <b>IV</b> and <b>V</b> in the K<sub>ac</sub> site of BRD4(1) revealing subtle conformational changes of the dimethylphenyl moieties nested in the WPF shelf. (D) Unliganded state of BRD4(1) (gray, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IOR">4IOR</a>), with the K<sub>ac</sub> site around Asn140 in a relaxed state, and BRD4(1) bound to inhibitor <b>V</b> (yellow ligand and orange structure), with Asn140 giving way (Δ = 0.8 Å) as a result of steric hindrance with the imidazole moiety of the inhibitor (red dotted line) and positioning itself for optimal H-bonding interaction. As a result, the adjacent Lys141 undergoes a large conformational change (Δ = 2.2 Å). (E) Unliganded BRD4(2) (green, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OUO">2OUO</a>), with the region around Asn433 being highly similar to that of BRD4(1) (gray) except for the presence of Pro434 instead of lysine. The geometric constraints of the Asn–Pro–Pro sequence in BRD4(2) likely renders the peptide more rigid and less compatible with the inhibitor-induced conformational changes observed in BRD4(1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0016.jpeg" id="rightTab-GRAPHIC-d7e904-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Displacement of structured waters from BRD4(1). Crystal structure of BRD4(1) with DMSO (left, yellow, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IOR">4IOR</a>) and compound <b>V</b> (right, yellow, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH1">6MH1</a>), with structurally conserved water molecules shown in cyan.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0010.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Triazoles <b>VII</b>, <b>VIII</b>, and <b>IX</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 100 <sup>ο</sup>C, 3 h for the synthesis of <b>34</b> and <b>36</b>; (b) NaHMDS, −78 <sup>ο</sup>C, 0.5 h for the synthesis of <b>35</b>.</p></p></figure><figure data-id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0004.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cellular evaluation of lead compounds confirming target engagement. (A) Cellular thermal-shift assay with compound <b>V</b> indicating thermal stabilization of BRD4 and p38α in A549 cells, with β-actin used as a loading control. (B) Western blots of expressed c-Myc levels in MM.1S cells, with GAPDH used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0005.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of NF-κB signaling and an NF-κB target gene. (A) NF-κB–luciferase-reporter assay with A549 cells with 20, 5, and 1 μM (+)-JQ1 and 10, 5, and 1 μM <b>IV</b>, <b>V</b>, and RVX-208. (B) Sandwich IL-8 ELISA with 10, 1, and 0.1 μM (+)-JQ1, <b>V</b>, SB-203580, and RVX-208. Data reported are means ± SEM values (<i>n</i> = 3 biological replicates) normalized to compound-induced (I) control values. All samples except the not-induced (NI) control were induced with 10 ng/mL TNFα. ANOVA comparisons to the three RVX-208 treatments; ns, not statistically significant; ***<i>p</i> < 0.001; ****<i>p</i> < 0.0001. See <a href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf" class="ext-link">Figure S3</a> for compound-mediated-cellular-cytotoxicity data.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/medium/jm-2018-01248k_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0011.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Fluorescein–JQ1 Tracer (<b>40</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b01248/20181019/images/large/jm-2018-01248k_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01248&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57452" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57452" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 58 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allfrey, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faulkner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirsky, A. E.</span></span> <span> </span><span class="NLM_article-title">Acetylation and Methylation of Histones and Their Possible Role in the Regulation of Rna Synthesis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">1938</span>),  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1073/pnas.51.5.786</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1073%2Fpnas.51.5.786" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1964&pages=786-794&issue=1938&author=V.+G.+Allfreyauthor=R.+Faulknerauthor=A.+E.+Mirsky&title=Acetylation+and+Methylation+of+Histones+and+Their+Possible+Role+in+the+Regulation+of+Rna+Synthesis&doi=10.1073%2Fpnas.51.5.786"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.51.5.786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.51.5.786%26sid%3Dliteratum%253Aachs%26aulast%3DAllfrey%26aufirst%3DV.%2BG.%26aulast%3DFaulkner%26aufirst%3DR.%26aulast%3DMirsky%26aufirst%3DA.%2BE.%26atitle%3DAcetylation%2520and%2520Methylation%2520of%2520Histones%2520and%2520Their%2520Possible%2520Role%2520in%2520the%2520Regulation%2520of%2520Rna%2520Synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1964%26volume%3D51%26issue%3D1938%26spage%3D786%26epage%3D794%26doi%3D10.1073%2Fpnas.51.5.786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of Chromatin by Histone Modifications</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">395</span>, <span class="refDoi"> DOI: 10.1038/cr.2011.22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fcr.2011.22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21321607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=381-395&issue=3&author=A.+J.+Bannisterauthor=T.+Kouzarides&title=Regulation+of+Chromatin+by+Histone+Modifications&doi=10.1038%2Fcr.2011.22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin by histone modifications</span></div><div class="casAuthors">Bannister, Andrew J.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-395</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Chromatin is not an inert structure, but rather an instructive DNA scaffold that can respond to external cues to regulate the many uses of DNA.  A principal component of chromatin that plays a key role in this regulation is the modification of histones.  There is an ever-growing list of these modifications and the complexity of their action is only just beginning to be understood.  However, it is clear that histone modifications play fundamental roles in most biol. processes that are involved in the manipulation and expression of DNA.  Here, we describe the known histone modifications, define where they are found genomically and discuss some of their functional consequences, concg. mostly on transcription where the majority of characterization has taken place.  Cell Research (2011) 21:381-395. doi:10.1038/cr.2011.22; published online 15 Feb. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ObWaVBHPorVg90H21EOLACvtfcHk0ljhtVjtkhVNRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D&md5=bbdf55be25985810b90ba2efe6399f1b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fcr.2011.22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2011.22%26sid%3Dliteratum%253Aachs%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520Chromatin%2520by%2520Histone%2520Modifications%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26issue%3D3%26spage%3D381%26epage%3D395%26doi%3D10.1038%2Fcr.2011.22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Bromodomains as Therapeutic Targets</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e29</span>, <span class="refDoi"> DOI: 10.1017/S1462399411001992</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1017%2FS1462399411001992" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21933453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=e29&author=S.+Mullerauthor=P.+Filippakopoulosauthor=S.+Knapp&title=Bromodomains+as+Therapeutic+Targets&doi=10.1017%2FS1462399411001992"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains as therapeutic targets</span></div><div class="casAuthors">Muller, Susanne; Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Reviews in Molecular Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e29/1-e29/21</span>CODEN:
                <span class="NLM_cas:coden">ERMMFS</span>;
        ISSN:<span class="NLM_cas:issn">1462-3994</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Acetylation of lysine residues is a post-translational modification with broad relevance to cellular signalling and disease biol.  Enzymes that 'write' (histone acetyltransferases, HATs) and 'erase' (histone deacetylases, HDACs) acetylation sites are an area of extensive research in current drug development, but very few potent inhibitors that modulate the 'reading process' mediated by acetyl lysines have been described.  The principal readers of ε-N-acetyl lysine (Kac) marks are bromodomains (BRDs), which are a diverse family of evolutionary conserved protein-interaction modules.  The conserved BRD fold contains a deep, largely hydrophobic acetyl lysine binding site, which represents an attractive pocket for the development of small, pharmaceutically active mols.  Proteins that contain BRDs have been implicated in the development of a large variety of diseases.  Recently, two highly potent and selective inhibitors that target BRDs of the BET (bromodomains and extra-terminal) family provided compelling data supporting targeting of these BRDs in inflammation and in an aggressive type of squamous cell carcinoma.  It is likely that BRDs will emerge alongside HATs and HDACs as interesting targets for drug development for the large no. of diseases that are caused by aberrant acetylation of lysine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSuKacpsr6xLVg90H21EOLACvtfcHk0liFYl3IkhSg-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OhurjI&md5=dc9383afcbe6aedc265041c21213aa0b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1017%2FS1462399411001992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399411001992%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DBromodomains%2520as%2520Therapeutic%2520Targets%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2011%26volume%3D13%26spage%3De29%26doi%3D10.1017%2FS1462399411001992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Functions of Bromodomain-Containing Proteins and Their Roles in Homeostasis and Cancer</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nrm.2016.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnrm.2016.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=28053347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=246-262&issue=4&author=T.+Fujisawaauthor=P.+Filippakopoulos&title=Functions+of+Bromodomain-Containing+Proteins+and+Their+Roles+in+Homeostasis+and+Cancer&doi=10.1038%2Fnrm.2016.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span></div><div class="casAuthors">Fujisawa, Takao; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">246-262</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are evolutionarily conserved protein-protein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions and are present in most tissues.  BRDs selectively recognize and bind to acetylated Lys residues - particularly in histones - and thereby have important roles in the regulation of gene expression.  BRD-contg. proteins are frequently dysregulated in cancer, they participate in gene fusions that generate diverse, frequently oncogenic proteins, and many cancer-causing mutations have been mapped to the BRDs themselves.  Importantly, BRDs can be targeted by small-mol. inhibitors, which has stimulated many translational research projects that seek to attenuate the aberrant functions of BRD-contg. proteins in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-drLvQK1rvLVg90H21EOLACvtfcHk0liFYl3IkhSg-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D&md5=cbccdee8423bad7a426c9cea6566151f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.143%26sid%3Dliteratum%253Aachs%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DFunctions%2520of%2520Bromodomain-Containing%2520Proteins%2520and%2520Their%2520Roles%2520in%2520Homeostasis%2520and%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26issue%3D4%26spage%3D246%26epage%3D262%26doi%3D10.1038%2Fnrm.2016.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Bromodomain Interaction Module</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>586</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">2692</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2012.04.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.febslet.2012.04.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=22710155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=2692-2704&issue=17&author=P.+Filippakopoulosauthor=S.+Knapp&title=The+Bromodomain+Interaction+Module&doi=10.1016%2Fj.febslet.2012.04.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain interaction module</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2692-2704</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ε-N-acetylation of protein Lys residues (Kac) is one of the most abundant post-translation modifications (PTMs) in the human proteome.  In the nucleus, the acetylation of histones has been linked to transcriptional activation of genes but the functional consequences of most acetylation events and proteins recruited to these sites remains largely unknown.  Bromodomains (BRDs) are small helical interaction modules that specifically recognize acetylation sites in proteins.  BRDs have recently emerged as interesting targets for the development of specific protein interaction inhibitors, enabling a novel exiting strategy for the development of new therapies.  Here, the author provides an overview over sequence requirements of BRDs, known substrates, and the structural mechanisms of specific Kac recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7LaThygz_HrVg90H21EOLACvtfcHk0liFYl3IkhSg-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D&md5=c02dee22c330b1d5c1ba39579360ebc5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Bromodomain%2520Interaction%2520Module%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26issue%3D17%26spage%3D2692%26epage%3D2704%26doi%3D10.1016%2Fj.febslet.2012.04.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of BET Bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span> (<span class="NLM_issue">7327</span>),  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&issue=7327&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+Inhibition+of+BET+Bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lh5WXIUWwJDBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520Inhibition%2520of%2520BET%2520Bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26issue%3D7327%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-F.</span></span> <span> </span><span class="NLM_article-title">Brd4 Coactivates Transcriptional Activation of NF-KappaB via Specific Binding to Acetylated RelA</span>. <i>Mol. Cel. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1375</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1128/MCB.01365-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=19103749" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&issue=5&author=B.+Huangauthor=X.-D.+Yangauthor=M.-M.+Zhouauthor=K.+Ozatoauthor=L.-F.+Chen&title=Brd4+Coactivates+Transcriptional+Activation+of+NF-KappaB+via+Specific+Binding+to+Acetylated+RelA&doi=10.1128%2FMCB.01365-08"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.-D.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DBrd4%2520Coactivates%2520Transcriptional%2520Activation%2520of%2520NF-KappaB%2520via%2520Specific%2520Binding%2520to%2520Acetylated%2520RelA%26jtitle%3DMol.%2520Cel.%2520Biol.%26date%3D2009%26volume%3D29%26issue%3D5%26spage%3D1375%26epage%3D1387%26doi%3D10.1128%2FMCB.01365-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovén, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.cell.2013.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=23582323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=320-334&issue=2&author=J.+Lov%C3%A9nauthor=H.+A.+Hokeauthor=C.+Y.+Linauthor=A.+Lauauthor=D.+A.+Orlandoauthor=C.+R.+Vakocauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Selective+Inhibition+of+Tumor+Oncogenes+by+Disruption+of+Super-Enhancers&doi=10.1016%2Fj.cell.2013.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span></div><div class="casAuthors">Loven, Jakob; Hoke, Heather A.; Lin, Charles Y.; Lau, Ashley; Orlando, David A.; Vakoc, Christopher R.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-334</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells.  Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM).  BRD4 and Mediator were found to co-occupy thousands of enhancers assocd. with active genes.  They also co-occupied a small set of exceptionally large super-enhancers assocd. with genes that feature prominently in MM biol., including the MYC oncogene.  Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC.  Super-enhancers were found at key oncogenic drivers in many other tumor cells.  These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-NqFYuGMqrVg90H21EOLACvtfcHk0ljeeQQWI6sO2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D&md5=66b6ccb5b8a6039832910443fd8b4664</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DLov%25C3%25A9n%26aufirst%3DJ.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520Inhibition%2520of%2520Tumor%2520Oncogenes%2520by%2520Disruption%2520of%2520Super-Enhancers%26jtitle%3DCell%26date%3D2013%26volume%3D153%26issue%3D2%26spage%3D320%26epage%3D334%26doi%3D10.1016%2Fj.cell.2013.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&issue=1&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0ljeeQQWI6sO2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26issue%3D1%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Small-Molecule Bromodomain-Containing Protein 4 (BRD4) Inhibitor That Induces AMP-Activated Protein Kinase-Modulated Autophagy-Associated Cell Death in Breast Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">9990</span>– <span class="NLM_lpage">10012</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00275</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00275" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9990-10012&issue=24&author=L.+Ouyangauthor=L.+Zhangauthor=J.+Liuauthor=L.+Fuauthor=D.+Yaoauthor=Y.+Zhaoauthor=S.+Zhangauthor=G.+Wangauthor=G.+Heauthor=B.+Liu&title=Discovery+of+a+Small-Molecule+Bromodomain-Containing+Protein+4+%28BRD4%29+Inhibitor+That+Induces+AMP-Activated+Protein+Kinase-Modulated+Autophagy-Associated+Cell+Death+in+Breast+Cancer&doi=10.1021%2Facs.jmedchem.7b00275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Bromodomain-Containing Protein 4 (BRD4) Inhibitor That Induces AMP-Activated Protein Kinase-Modulated Autophagy-Associated Cell Death in Breast Cancer</span></div><div class="casAuthors">Ouyang, Liang; Zhang, Lan; Liu, Jie; Fu, Leilei; Yao, Dahong; Zhao, Yuqian; Zhang, Shouyue; Wang, Guan; He, Gu; Liu, Bo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9990-10012</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Upon the basis of The Cancer Genome Atlas (TCGA) data set, the authors identified that several autophagy-related proteins such as AMP-activated protein kinase (AMPK) were remarkably downregulated in breast cancer.  Combined with coimmunopptn. assay, the authors demonstrated that BRD4 might interact with AMPK.  After analyses of the pharmacophore and WPF interaction optimization, the authors designed a small-mol. inhibitor of BRD4, 9f (FL-411 (2-(4-hydroxy-3,5-dimethylphenyl)-7-methyl-5,6,7,8-tetrahydropyrido[4',3':4,5]thieno[2,3-d]pyrimidin-4(3H)-one)) which was validated by cocrystal structure with BD1 of BRD4.  Subsequently, 9f was discovered to induce ATG5-dependent autophagy-assocd. cell death (ACD) by blocking BRD4-AMPK interaction and thus activating AMPK-mTOR-ULK1-modulated autophagic pathway in breast cancer cells.  Interestingly, the iTRAQ-based proteomics analyses revealed that 9f induced ACD pathways involved in HMGB1, VDAC1/2, and eEF2.  Moreover, 9f displayed a therapeutic potential on both breast cancer xenograft mouse and zebrafish models.  Together, these results demonstrate that a novel small-mol. inhibitor of BRD4 induces BRD4-AMPK-modulated ACD in breast cancer, which may provide a candidate drug for future cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Ocrq2wyy6bVg90H21EOLACvtfcHk0likqlgIrZaFlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitrvF&md5=ac15f7e108511ea207f2c2fcd299b4e5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00275%26sid%3Dliteratum%253Aachs%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520Small-Molecule%2520Bromodomain-Containing%2520Protein%25204%2520%2528BRD4%2529%2520Inhibitor%2520That%2520Induces%2520AMP-Activated%2520Protein%2520Kinase-Modulated%2520Autophagy-Associated%2520Cell%2520Death%2520in%2520Breast%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D24%26spage%3D9990%26epage%3D10012%26doi%3D10.1021%2Facs.jmedchem.7b00275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Measures, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of Bromodomain–Acetyl-Lysine Interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1021/cb500996u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500996u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=22-39&issue=1&author=M.+Brandauthor=A.+M.+Measuresauthor=B.+G.+Wilsonauthor=W.+A.+Cortopassiauthor=R.+Alexanderauthor=M.+H%C3%B6ssauthor=D.+S.+Hewingsauthor=T.+P.+C.+Rooneyauthor=R.+S.+Patonauthor=S.+J.+Conway&title=Small+Molecule+Inhibitors+of+Bromodomain%E2%80%93Acetyl-Lysine+Interactions&doi=10.1021%2Fcb500996u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Bromodomain-Acetyl-lysine Interactions</span></div><div class="casAuthors">Brand, Michael; Measures, Angelina R.; Wilson, Brian G.; Cortopassi, Wilian A.; Alexander, Rikki; Hoss, Matthias; Hewings, David S.; Rooney, Timothy P. C.; Paton, Robert S.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-39</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Bromodomains are protein modules that bind to acetylated lysine residues.  Their interaction with histone proteins suggests that they function as "readers" of histone lysine acetylation, a component of the proposed "histone code".  Bromodomain-contg. proteins are often found as components of larger protein complexes with roles in fundamental cellular process including transcription.  The publication of two potent ligands for the BET bromodomains in 2010 demonstrated that small mols. can inhibit the bromodomain-acetyl-lysine protein-protein interaction.  These mols. display strong phenotypic effects in a no. of cell lines and affect a range of cancers in vivo.  This work stimulated intense interest in developing further ligands for the BET bromodomains and the design of ligands for non-BET bromodomains.  Here we review the recent progress in the field with particular attention paid to ligand design, the assays employed in early ligand discovery, and the use of computational approaches to inform ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1JZtWpBpqS7Vg90H21EOLACvtfcHk0likqlgIrZaFlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D&md5=e40e85fc300109654712a00de400c8ba</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb500996u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500996u%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DMeasures%26aufirst%3DA.%2BM.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DAlexander%26aufirst%3DR.%26aulast%3DH%25C3%25B6ss%26aufirst%3DM.%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520Bromodomain%25E2%2580%2593Acetyl-Lysine%2520Interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D1%26spage%3D22%26epage%3D39%26doi%3D10.1021%2Fcb500996u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrieu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">Clinical Trials for BET Inhibitors Run Ahead of the Science</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2016.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.ddtec.2016.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=27769357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=45-50&author=G.+Andrieuauthor=A.+C.+Belkinaauthor=G.+V.+Denis&title=Clinical+Trials+for+BET+Inhibitors+Run+Ahead+of+the+Science&doi=10.1016%2Fj.ddtec.2016.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical trials for BET inhibitors run ahead of the science</span></div><div class="casAuthors">Andrieu Guillaume; Belkina Anna C; Denis Gerald V</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several cancer clinical trials for small molecule inhibitors of BET bromodomain proteins have been initiated.  There is enthusiasm for the anti-proliferative effect of inhibiting BRD4, one of the targets of these inhibitors, which is thought to cooperate with MYC, a long-desired target for cancer therapeutics.  However, no current inhibitor is selective for BRD4 among the three somatic BET proteins, which include BRD2 and BRD3; their respective functions are partially overlapping and none are functionally redundant with BRD4.  Each BET protein controls distinct transcriptional pathways that are important for functions beyond cancer cell proliferation, including insulin production, cytokine gene transcription, T cell differentiation, adipogenesis and most seriously, active repression of dangerous latent viruses like HIV.  BET inhibitors have been shown to reactivate HIV in human cells.  Failure to appreciate that at concentrations used, no available BET inhibitor is member-selective, or to develop a sound biological basis to understand the diverse functions of BET proteins before undertaking for these clinical trials is reckless and likely to lead to adverse events.  More mechanistic information from new basic science studies should enable proper focus on the most relevant cancers and define the expected side effect profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEothR-V9Y-Lt7lRqEFUJVfW6udTcc2ebledcLc1Fz27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srkvFOjtQ%253D%253D&md5=5fcc9e96fa590221fb07c5e98ae91d59</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2016.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DAndrieu%26aufirst%3DG.%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DClinical%2520Trials%2520for%2520BET%2520Inhibitors%2520Run%2520Ahead%2520of%2520the%2520Science%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2016%26volume%3D19%26spage%3D45%26epage%3D50%26doi%3D10.1016%2Fj.ddtec.2016.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amorim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyengar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarotto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leleu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morschhauser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broussais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahatt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokiec, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salles, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palumbo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thieblemont, C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Inhibitor OTX015 in Patients with Lymphoma or Multiple Myeloma: A Dose-Escalation, Open-Label, Pharmacokinetic, Phase 1 Study</span>. <i>Lancet Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">e196</span>– <span class="NLM_lpage">e204</span>, <span class="refDoi"> DOI: 10.1016/S2352-3026(16)00021-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2FS2352-3026%2816%2900021-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=27063978" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=e196-e204&issue=4&author=S.+Amorimauthor=A.+Stathisauthor=M.+Gleesonauthor=S.+Iyengarauthor=V.+Magarottoauthor=X.+Leleuauthor=F.+Morschhauserauthor=L.+Karlinauthor=F.+Broussaisauthor=K.+Rezaiauthor=P.+Heraitauthor=C.+Kahattauthor=F.+Lokiecauthor=G.+Sallesauthor=T.+Faconauthor=A.+Palumboauthor=D.+Cunninghamauthor=E.+Zuccaauthor=C.+Thieblemont&title=Bromodomain+Inhibitor+OTX015+in+Patients+with+Lymphoma+or+Multiple+Myeloma%3A+A+Dose-Escalation%2C+Open-Label%2C+Pharmacokinetic%2C+Phase+1+Study&doi=10.1016%2FS2352-3026%2816%2900021-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS2352-3026%2816%2900021-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2352-3026%252816%252900021-1%26sid%3Dliteratum%253Aachs%26aulast%3DAmorim%26aufirst%3DS.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DGleeson%26aufirst%3DM.%26aulast%3DIyengar%26aufirst%3DS.%26aulast%3DMagarotto%26aufirst%3DV.%26aulast%3DLeleu%26aufirst%3DX.%26aulast%3DMorschhauser%26aufirst%3DF.%26aulast%3DKarlin%26aufirst%3DL.%26aulast%3DBroussais%26aufirst%3DF.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DKahatt%26aufirst%3DC.%26aulast%3DLokiec%26aufirst%3DF.%26aulast%3DSalles%26aufirst%3DG.%26aulast%3DFacon%26aufirst%3DT.%26aulast%3DPalumbo%26aufirst%3DA.%26aulast%3DCunningham%26aufirst%3DD.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DThieblemont%26aufirst%3DC.%26atitle%3DBromodomain%2520Inhibitor%2520OTX015%2520in%2520Patients%2520with%2520Lymphoma%2520or%2520Multiple%2520Myeloma%253A%2520A%2520Dose-Escalation%252C%2520Open-Label%252C%2520Pharmacokinetic%252C%2520Phase%25201%2520Study%26jtitle%3DLancet%2520Haematol.%26date%3D2016%26volume%3D3%26issue%3D4%26spage%3De196%26epage%3De204%26doi%3D10.1016%2FS2352-3026%2816%2900021-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an Inhibitor of BET Transcriptional Regulators with Selectivity for the Second Bromodomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">49</span>),  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&issue=49&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+Mullerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+Inhibitor+of+BET+Transcriptional+Regulators+with+Selectivity+for+the+Second+Bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lhb7xm815YlAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520Inhibitor%2520of%2520BET%2520Transcriptional%2520Regulators%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26issue%3D49%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span> <span> </span><span class="NLM_article-title">RVX-297- a Novel BD2 Selective Inhibitor of BET Bromodomains</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>477</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2016.06.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.bbrc.2016.06.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=27282480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2016&pages=62-67&issue=1&author=O.+A.+Kharenkoauthor=E.+M.+Gesnerauthor=R.+G.+Patelauthor=K.+Norekauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=P.+R.+Youngauthor=K.+G.+McLureauthor=H.+C.+Hansen&title=RVX-297-+a+Novel+BD2+Selective+Inhibitor+of+BET+Bromodomains&doi=10.1016%2Fj.bbrc.2016.06.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-297- a novel BD2 selective inhibitor of BET bromodomains</span></div><div class="casAuthors">Kharenko, Olesya A.; Gesner, Emily M.; Patel, Reena G.; Norek, Karen; White, Andre; Fontano, Eric; Suto, Robert K.; Young, Peter R.; McLure, Kevin G.; Hansen, Henrik C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-67</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers that specifically bind to the acetyl lysine residues of histones and transcription factors.  Small mol. BET bromodomain inhibitors can disrupt this interaction which leads to potential modulation of several disease states.  Here we describe the binding properties of a novel BET inhibitor RVX-297 that is structurally related to the clin. compd. RVX-208, currently undergoing phase III clin. trials for the treatment of cardiovascular diseases, but is distinctly different in its biol. and pharmacokinetic profiles.  We report that RVX-297 preferentially binds to the BD2 domains of the BET bromodomain and Extra Terminal (BET) family of protein.  We demonstrate the differential binding modes of RVX-297 in BD1 and BD2 domains of BRD4 and BRD2 using X-ray crystallog., and describe the structural differences driving the BD2 selective binding of RVX-297.  The isothermal titrn. calorimetry (ITC) data illustrate the related differential thermodn. of binding of RVX-297 to single as well as dual BET bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDLUg96I7HObVg90H21EOLACvtfcHk0lhb7xm815YlAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpslajtrw%253D&md5=a3d064e2245bf14ae525eac2490808cb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.06.021%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26atitle%3DRVX-297-%2520a%2520Novel%2520BD2%2520Selective%2520Inhibitor%2520of%2520BET%2520Bromodomains%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D477%26issue%3D1%26spage%3D62%26epage%3D67%26doi%3D10.1016%2Fj.bbrc.2016.06.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berkovits, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolgemuth, D. J.</span></span> <span> </span><span class="NLM_article-title">The First Bromodomain of the Testis-Specific Double Bromodomain Protein Brdt Is Required for Chromocenter Organization That Is Modulated by Genetic Background</span>. <i>Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>360</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1016/j.ydbio.2011.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.ydbio.2011.10.005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=360&publication_year=2011&pages=358-368&issue=2&author=B.+D.+Berkovitsauthor=D.+J.+Wolgemuth&title=The+First+Bromodomain+of+the+Testis-Specific+Double+Bromodomain+Protein+Brdt+Is+Required+for+Chromocenter+Organization+That+Is+Modulated+by+Genetic+Background&doi=10.1016%2Fj.ydbio.2011.10.005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ydbio.2011.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ydbio.2011.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DBerkovits%26aufirst%3DB.%2BD.%26aulast%3DWolgemuth%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520First%2520Bromodomain%2520of%2520the%2520Testis-Specific%2520Double%2520Bromodomain%2520Protein%2520Brdt%2520Is%2520Required%2520for%2520Chromocenter%2520Organization%2520That%2520Is%2520Modulated%2520by%2520Genetic%2520Background%26jtitle%3DDev.%2520Biol.%26date%3D2011%26volume%3D360%26issue%3D2%26spage%3D358%26epage%3D368%26doi%3D10.1016%2Fj.ydbio.2011.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Assessing Cellular Efficacy of Bromodomain Inhibitors Using Fluorescence Recovery after Photobleaching</span>. <i>Epigenet. Chromatin</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">14</span>, <span class="refDoi"> DOI: 10.1186/1756-8935-7-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1186%2F1756-8935-7-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=25097667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVaqtrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=14&issue=1&author=M.+Philpottauthor=C.+M.+Rogersauthor=C.+Yappauthor=C.+Wellsauthor=J.+P.+Lambertauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=A.+C.+Gingrasauthor=S.+Knappauthor=S.+M%C3%BCller&title=Assessing+Cellular+Efficacy+of+Bromodomain+Inhibitors+Using+Fluorescence+Recovery+after+Photobleaching&doi=10.1186%2F1756-8935-7-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing cellular efficacy of bromodomain inhibitors using fluorescence recovery after photobleaching</span></div><div class="casAuthors">Philpott, Martin; Rogers, Catherine M.; Yapp, Clarence; Wells, Chris; Lambert, Jean-Philippe; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Gingras, Anne-Claude; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14/1-14/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Acetylation of lysine residues in histone tails plays an important role in the regulation of gene transcription.  Bromdomains are the readers of acetylated histone marks, and, consequently, bromodomain-contg. proteins have a variety of chromatin-related functions.  Moreover, they are increasingly being recognized as important mediators of a wide range of diseases.  The first potent and selective bromodomain inhibitors are beginning to be described, but the diverse or unknown functions of bromodomain-contg. proteins present challenges to systematically demonstrating cellular efficacy and selectivity for these inhibitors.  Here we assess the viability of fluorescence recovery after photobleaching (FRAP) assays as a target agnostic method for the direct visualisation of an on-target effect of bromodomain inhibitors in living cells.  Results: Mutation of a conserved asparagine crucial for binding to acetylated lysines in the bromodomains of BRD3, BRD4 and TRIM24 all resulted in redn. of FRAP recovery times, indicating loss of or significantly reduced binding to acetylated chromatin, as did the addn. of known inhibitors.  Significant differences between wild type and bromodomain mutants for ATAD2, BAZ2A, BRD1, BRD7, GCN5L2, SMARCA2 and ZMYND11 required the addn. of the histone deacetylase inhibitor suberoylanilide hydroxamic acid (SAHA) to amplify the binding contribution of the bromodomain.  Under these conditions, known inhibitors decreased FRAP recovery times back to mutant control levels.  Mutation of the bromodomain did not alter FRAP recovery times for full-length CREBBP, even in the presence of SAHA, indicating that other domains are primarily responsible for anchoring CREBBP to chromatin.  However, FRAP assays with multimerised CREBBP bromodomains resulted in a good assay to assess the efficacy of bromodomain inhibitors to this target.  The bromodomain and extraterminal protein inhibitor PFI-1 was inactive against other bromodomain targets, demonstrating the specificity of the method.  Conclusions: Viable FRAP assays were established for 11 representative bromodomain-contg. proteins that broadly cover the bromodomain phylogenetic tree.  Addn. of SAHA can overcome weak binding to chromatin, and the use of tandem bromodomain constructs can eliminate masking effects of other chromatin binding domains.  Together, these results demonstrate that FRAP assays offer a potentially pan-bromodomain method for generating cell-based assays, allowing the testing of compds. with respect to cell permeability, on-target efficacy and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0eRoqpM6LTbVg90H21EOLACvtfcHk0lgjoHwemLlx0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVaqtrzE&md5=966744a9e7b3cba00776b76d65594fc6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-7-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-7-14%26sid%3Dliteratum%253Aachs%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DAssessing%2520Cellular%2520Efficacy%2520of%2520Bromodomain%2520Inhibitors%2520Using%2520Fluorescence%2520Recovery%2520after%2520Photobleaching%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2014%26volume%3D7%26issue%3D1%26spage%3D14%26doi%3D10.1186%2F1756-8935-7-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. F.</span></span> <span> </span><span class="NLM_article-title">Brd4 Maintains Constitutively Active NF-κB in Cancer Cells by Binding to Acetylated RelA</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">2395</span>– <span class="NLM_lpage">2404</span>, <span class="refDoi"> DOI: 10.1038/onc.2013.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fonc.2013.179" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=2395-2404&issue=18&author=Z.+Zouauthor=B.+Huangauthor=X.+Wuauthor=H.+Zhangauthor=J.+Qiauthor=J.+Bradnerauthor=S.+Nairauthor=L.+F.+Chen&title=Brd4+Maintains+Constitutively+Active+NF-%CE%BAB+in+Cancer+Cells+by+Binding+to+Acetylated+RelA&doi=10.1038%2Fonc.2013.179"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fonc.2013.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2013.179%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DNair%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBrd4%2520Maintains%2520Constitutively%2520Active%2520NF-%25CE%25BAB%2520in%2520Cancer%2520Cells%2520by%2520Binding%2520to%2520Acetylated%2520RelA%26jtitle%3DOncogene%26date%3D2014%26volume%3D33%26issue%3D18%26spage%3D2395%26epage%3D2404%26doi%3D10.1038%2Fonc.2013.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">9251</span>– <span class="NLM_lpage">9264</span>, <span class="refDoi"> DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&issue=22&author=G.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.+M.+Zhou&title=Structure-Guided+Design+of+Potent+Diazobenzene+Inhibitors+for+the+BET+Bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0lgjoHwemLlx0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure-Guided%2520Design%2520of%2520Potent%2520Diazobenzene%2520Inhibitors%2520for%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D22%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuppusetty, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">3,5-Dimethylisoxazoles Act as Acetyl-Lysine-Mimetic Bromodomain Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">6761</span>– <span class="NLM_lpage">6770</span>, <span class="refDoi"> DOI: 10.1021/jm200640v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200640v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6761-6770&issue=19&author=D.%0AS.+Hewingsauthor=M.+Wangauthor=M.+Philpottauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=C.+Vuppusettyauthor=B.+Marsdenauthor=S.+Knappauthor=S.+J.+Conwayauthor=T.+D.+Heightman&title=3%2C5-Dimethylisoxazoles+Act+as+Acetyl-Lysine-Mimetic+Bromodomain+Ligands&doi=10.1021%2Fjm200640v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Dimethylisoxazoles Act As Acetyl-lysine-mimetic Bromodomain Ligands</span></div><div class="casAuthors">Hewings, David S.; Wang, Minghua; Philpott, Martin; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Vuppusetty, Chaitanya; Marsden, Brian; Knapp, Stefan; Conway, Stuart J.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6761-6770</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-lysine acetylation is a vital chromatin post-translational modification involved in the epigenetic regulation of gene transcription.  Bromodomains bind acetylated lysines, acting as readers of the histone-acetylation code.  Competitive inhibitors of this interaction have antiproliferative and anti-inflammatory properties.  With 57 distinct bromodomains known, the discovery of subtype-selective inhibitors of the histone-bromodomain interaction is of great importance.  We have identified the 3,5-dimethylisoxazole moiety as a novel acetyl-lysine bioisostere, which displaces acetylated histone-mimicking peptides from bromodomains.  Using X-ray crystallog. anal., we have detd. the interactions responsible for the activity and selectivity of 4-substituted 3,5-dimethylisoxazoles against a selection of phylogenetically diverse bromodomains.  By exploiting these interactions, we have developed compd. 4d, which has IC50 values of <5 μM for the bromodomain-contg. proteins BRD2(1) and BRD4(1).  These compds. are promising leads for the further development of selective probes for the bromodomain and extra C-terminal domain (BET) family and CREBBP bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEzK4xMvJhXrVg90H21EOLACvtfcHk0lh3YFr0aN4x4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaks7rO&md5=7d6f9eff2735c8a7189dec4fb5c5d13d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm200640v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200640v%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DVuppusetty%26aufirst%3DC.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3D3%252C5-Dimethylisoxazoles%2520Act%2520as%2520Acetyl-Lysine-Mimetic%2520Bromodomain%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D19%26spage%3D6761%26epage%3D6770%26doi%3D10.1021%2Fjm200640v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gacias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matikainen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casaccia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.chembiol.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=24954007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=841-854&issue=7&author=M.+Gaciasauthor=G.+Gerona-Navarroauthor=A.+N.+Plotnikovauthor=G.+Zhangauthor=L.+Zengauthor=J.+Kaurauthor=G.+Moyauthor=E.+Rusinovaauthor=Y.+Rodriguezauthor=B.+Matikainenauthor=A.+Vincekauthor=J.+Joshuaauthor=P.+Casacciaauthor=M.+M.+Zhou&title=Selective+Chemical+Modulation+of+Gene+Transcription+Favors+Oligodendrocyte+Lineage+Progression&doi=10.1016%2Fj.chembiol.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span></div><div class="casAuthors">Gacias, Mar; Gerona-Navarro, Guillermo; Plotnikov, Alexander N.; Zhang, Guangtao; Zeng, Lei; Kaur, Jasbir; Moy, Gregory; Rusinova, Elena; Rodriguez, Yoel; Matikainen, Bridget; Vincek, Adam; Joshua, Jennifer; Casaccia, Patrizia; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">841-854</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lysine acetylation regulates gene expression through modulating protein-protein interactions in chromatin.  Chem. inhibition of acetyl-lysine binding bromodomains of the major chromatin regulators BET (bromodomain and extraterminal domain) proteins has been shown to effectively block cell proliferation in cancer and inflammation.  However, whether selective inhibition of individual BET bromodomains has distinctive functional consequences remains only partially understood.  In this study, we show that selective chem. inhibition of the first bromodomain of BET proteins using our small-mol. inhibitor, Olinone, accelerated the progression of mouse primary oligodendrocyte progenitors toward differentiation, whereas inhibition of both bromodomains of BET proteins hindered differentiation.  This effect was target specific, as it was not detected in cells treated with inactive analogs and independent of any effect on proliferation.  Therefore, selective chem. modulation of individual bromodomains, rather than use of broad-based inhibitors, may enhance regenerative strategies in disorders characterized by myelin loss such as aging and neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1fCvOR18yE7Vg90H21EOLACvtfcHk0lglQ_cEyYRRRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK&md5=6b377663a8d0d07441ab4f8a5d8191b3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DGacias%26aufirst%3DM.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DKaur%26aufirst%3DJ.%26aulast%3DMoy%26aufirst%3DG.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DY.%26aulast%3DMatikainen%26aufirst%3DB.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DCasaccia%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DSelective%2520Chemical%2520Modulation%2520of%2520Gene%2520Transcription%2520Favors%2520Oligodendrocyte%2520Lineage%2520Progression%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26issue%3D7%26spage%3D841%26epage%3D854%26doi%3D10.1016%2Fj.chembiol.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Shiao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardote, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pschibul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, T. T.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology. A Bump-and-Hole Approach to Engineer Controlled Selectivity of BET Bromodomain Chemical Probes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span> (<span class="NLM_issue">6209</span>),  <span class="NLM_fpage">638</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1126/science.1249830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1126%2Fscience.1249830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=25323695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=638-641&issue=6209&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=T.+Cardoteauthor=C.+Tallantauthor=A.+Pschibulauthor=K.-H.+Chanauthor=M.+Zengerleauthor=J.+R.+Garciaauthor=T.+T.-L.+Kwanauthor=F.+M.+Fergusonauthor=A.+Ciulli&title=Chemical+Biology.+A+Bump-and-Hole+Approach+to+Engineer+Controlled+Selectivity+of+BET+Bromodomain+Chemical+Probes&doi=10.1126%2Fscience.1249830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Cardote, Teresa; Tallant, Cynthia; Pschibul, Annica; Chan, Kwok-Ho; Zengerle, Michael; Garcia, Jordi R.; Kwan, Terence T.-L.; Ferguson, Fleur M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6209</span>),
    <span class="NLM_cas:pages">638-641</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Small mols. are useful tools for probing the biol. function and therapeutic potential of individual proteins, but achieving selectivity is challenging when the target protein shares structural domains with other proteins.  The bromodomain and extra-terminal (BET) proteins have attracted interest because of their roles in transcriptional regulation, epigenetics, and cancer.  The BET bromodomains (protein interaction modules that bind acetyl-lysine) have been targeted by potent small-mol. inhibitors, but these inhibitors lack selectivity for individual family members.  We developed an Et deriv. of an existing small-mol. inhibitor, I-BET/JQ1, and showed that it binds leucine/alanine mutant bromodomains with nanomolar affinity and achieves up to 540-fold selectivity relative to wild-type bromodomains.  Cell culture studies showed that blockade of the first bromodomain alone is sufficient to displace a specific BET protein, Brd4, from chromatin.  Expansion of this approach could help identify the individual roles of single BET proteins in human physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9EnzLtvUKwbVg90H21EOLACvtfcHk0lglQ_cEyYRRRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL&md5=04fe967e4681fb4b2b3168f4d5f8ff81</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.1249830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1249830%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DCardote%26aufirst%3DT.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DJ.%2BR.%26aulast%3DKwan%26aufirst%3DT.%2BT.-L.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DChemical%2520Biology.%2520A%2520Bump-and-Hole%2520Approach%2520to%2520Engineer%2520Controlled%2520Selectivity%2520of%2520BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DScience%26date%3D2014%26volume%3D346%26issue%3D6209%26spage%3D638%26epage%3D641%26doi%3D10.1126%2Fscience.1249830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Runcie, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Beurden, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulthard, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of a “Bump-and-Hole” Approach to Allele-Selective BET Bromodomain Inhibition</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2452</span>– <span class="NLM_lpage">2468</span>, <span class="refDoi"> DOI: 10.1039/C7SC02536J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1039%2FC7SC02536J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=29732121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2452-2468&issue=9&author=A.+C.+Runcieauthor=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Testaauthor=L.+Van+Beurdenauthor=M.+G.+J.+Baudauthor=O.+Epemoluauthor=L.+C.+J.+Ellisauthor=K.+D.+Readauthor=V.+Coulthardauthor=A.+Brienauthor=A.+Ciulli&title=Optimization+of+a+%E2%80%9CBump-and-Hole%E2%80%9D+Approach+to+Allele-Selective+BET+Bromodomain+Inhibition&doi=10.1039%2FC7SC02536J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a "bump-and-hole" approach to allele-selective BET bromodomain inhibition</span></div><div class="casAuthors">Runcie, A. C.; Zengerle, M.; Chan, K.-H.; Testa, A.; van Beurden, L.; Baud, M. G. J.; Epemolu, O.; Ellis, L. C. J.; Read, K. D.; Coulthard, V.; Brien, A.; Ciulli, A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2452-2468</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Allele-specific chem. genetics enables selective inhibition within families of highly-conserved proteins.  The four BET (bromodomain & extra-terminal domain) proteins - BRD2, BRD3, BRD4 and BRDT bind acetylated chromatin via their bromodomains and regulate processes such as cell proliferation and inflammation.  BET bromodomains are of particular interest, as they are attractive therapeutic targets but existing inhibitors are pan-selective.  We previously established a bump-&-hole system for the BET bromodomains, pairing a leucine/alanine mutation with an ethyl-derived analog of an established benzodiazepine scaffold.  Here we optimize upon this system with the introduction of a more conservative and less disruptive leucine/valine mutation.  Extensive structure-activity-relationships of diverse benzodiazepine analogs guided the development of potent, mutant-selective inhibitors with desirable physiochem. properties.  The active enantiomer of our best compd. - 9-ME-1 - shows ~ 200 nM potency, >100-fold selectivity for the L/V mutant over wild-type and excellent DMPK properties.  Through a variety of in vitro and cellular assays we validate the capabilities of our optimized system, and then utilize it to compare the relative importance of the first and second bromodomains to chromatin binding.  These expts. confirm the primacy of the first bromodomain in all BET proteins, but also significant variation in the importance of the second bromodomain.  We also show that, despite having a minor role in chromatin recognition, BRD4 BD2 is still essential for gene expression, likely through the recruitment of non-histone proteins.  The disclosed inhibitor:mutant pair provides a powerful tool for future cellular and in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq47qHG4-zFurVg90H21EOLACvtfcHk0liGRVX18j_AwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKku70%253D&md5=9a44bdccabca9f1d4dfdc983250d9aa3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC7SC02536J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC02536J%26sid%3Dliteratum%253Aachs%26aulast%3DRuncie%26aufirst%3DA.%2BC.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DVan%2BBeurden%26aufirst%3DL.%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DEllis%26aufirst%3DL.%2BC.%2BJ.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCoulthard%26aufirst%3DV.%26aulast%3DBrien%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DOptimization%2520of%2520a%2520%25E2%2580%259CBump-and-Hole%25E2%2580%259D%2520Approach%2520to%2520Allele-Selective%2520BET%2520Bromodomain%2520Inhibition%26jtitle%3DChem.%2520Sci.%26date%3D2018%26volume%3D9%26issue%3D9%26spage%3D2452%26epage%3D2468%26doi%3D10.1039%2FC7SC02536J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szklarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul Azeez, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikolajczyk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepetov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Haj Zen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teicher, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Characterization of the Published Kinase Inhibitor Set</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1038/nbt.3374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnbt.3374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=26501955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslantL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=95-103&issue=1&author=J.+M.+Elkinsauthor=V.+Fedeleauthor=M.+Szklarzauthor=K.+R.+Abdul+Azeezauthor=E.+Salahauthor=J.+Mikolajczykauthor=S.+Romanovauthor=N.+Sepetovauthor=X.-P.+Huangauthor=B.+L.+Rothauthor=A.+Al+Haj+Zenauthor=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropshaauthor=J.+Morrisauthor=B.+A.+Teicherauthor=M.+Kunkelauthor=E.+Polleyauthor=K.+E.+Lackeyauthor=F.+L.+Atkinsonauthor=J.+P.+Overingtonauthor=P.+Bamboroughauthor=S.+M%C3%BCllerauthor=D.+J.+Priceauthor=T.+M.+Willsonauthor=D.+H.+Drewryauthor=S.+Knappauthor=W.+J.+Zuercher&title=Comprehensive+Characterization+of+the+Published+Kinase+Inhibitor+Set&doi=10.1038%2Fnbt.3374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive characterization of the Published Kinase Inhibitor Set</span></div><div class="casAuthors">Elkins, Jonathan M.; Fedele, Vita; Szklarz, Marta; Abdul Azeez, Kamal R.; Salah, Eidarus; Mikolajczyk, Jowita; Romanov, Sergei; Sepetov, Nikolai; Huang, Xi-Ping; Roth, Bryan L.; Al Haj Zen, Ayman; Fourches, Denis; Muratov, Eugene; Tropsha, Alex; Morris, Joel; Teicher, Beverly A.; Kunkel, Mark; Polley, Eric; Lackey, Karen E.; Atkinson, Francis L.; Overington, John P.; Bamborough, Paul; Muller, Susanne; Price, Daniel J.; Willson, Timothy M.; Drewry, David H.; Knapp, Stefan; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-103</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite the success of protein kinase inhibitors as approved therapeutics, drug discovery has focused on a small subset of kinase targets.  Here we provide a thorough characterization of the Published Kinase Inhibitor Set (PKIS), a set of 367 small-mol. ATP-competitive kinase inhibitors that was recently made freely available with the aim of expanding research in this field and as an expt. in open-source target validation.  We screen the set in activity assays with 224 recombinant kinases and 24 G protein-coupled receptors and in cellular assays of cancer cell proliferation and angiogenesis.  We identify chem. starting points for designing new chem. probes of orphan kinases and illustrate the utility of these leads by developing a selective inhibitor for the previously untargeted kinases LOK and SLK.  Our cellular screens reveal compds. that modulate cancer cell growth and angiogenesis in vitro.  These reagents and assocd. data illustrate an efficient way forward to increasing understanding of the historically untargeted kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4P1WN3gdiXbVg90H21EOLACvtfcHk0liGRVX18j_AwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslantL%252FM&md5=3f15096e79632dbaaadf4460a6848e60</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3374%26sid%3Dliteratum%253Aachs%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR.%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DMikolajczyk%26aufirst%3DJ.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DSepetov%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DAl%2BHaj%2BZen%26aufirst%3DA.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DTeicher%26aufirst%3DB.%2BA.%26aulast%3DKunkel%26aufirst%3DM.%26aulast%3DPolley%26aufirst%3DE.%26aulast%3DLackey%26aufirst%3DK.%2BE.%26aulast%3DAtkinson%26aufirst%3DF.%2BL.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DComprehensive%2520Characterization%2520of%2520the%2520Published%2520Kinase%2520Inhibitor%2520Set%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26issue%3D1%26spage%3D95%26epage%3D103%26doi%3D10.1038%2Fnbt.3374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.
J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-Lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&issue=5&author=S.+W.%0AJ.+Emberauthor=J.+Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Acetyl-Lysine+Binding+Site+of+Bromodomain-Containing+Protein+4+%28BRD4%29+Interacts+with+Diverse+Kinase+Inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0liy4goOJEBjzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DAcetyl-Lysine%2520Binding%2520Site%2520of%2520Bromodomain-Containing%2520Protein%25204%2520%2528BRD4%2529%2520Interacts%2520with%2520Diverse%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26issue%3D5%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawk, L. M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassel, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dos Santos, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Dual Screening of BPTF and Brd4 Using Protein-Observed Fluorine NMR Uncovers New Bromodomain Probe Molecules</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2256</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2246-2256&issue=10&author=A.+K.+Urickauthor=L.+M.+L.+Hawkauthor=M.+K.+Casselauthor=N.+K.+Mishraauthor=S.+Liuauthor=N.+Adhikariauthor=W.+Zhangauthor=C.+O.+Dos+Santosauthor=J.+L.+Hallauthor=W.+C.+K.+Pomerantz&title=Dual+Screening+of+BPTF+and+Brd4+Using+Protein-Observed+Fluorine+NMR+Uncovers+New+Bromodomain+Probe+Molecules&doi=10.1021%2Facschembio.5b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Screening of BPTF and Brd4 Using Protein-Observed Fluorine NMR Uncovers New Bromodomain Probe Molecules</span></div><div class="casAuthors">Urick, Andrew K.; Hawk, Laura M. L.; Cassel, Melissa K.; Mishra, Neeraj K.; Liu, Shuai; Adhikari, Neeta; Zhang, Wei; dos Santos, Camila O.; Hall, Jennifer L.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2246-2256</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein dysregulation is linked to cancer, diabetes, and inflammation.  Selective inhibition of bromodomain function is a newly proposed therapeutic strategy.  The authors describe a 19F NMR dual screening method for small mol. discovery using fluorinated tryptophan resonances on two bromodomain-contg. proteins.  The chem. shift dispersion of 19F resonances within fluorine-labeled proteins enables the simultaneous anal. of two fluorinated bromodomains by NMR.  A library of 229 small mols. was screened against the first bromodomain of Brd4 and the BPTF bromodomain.  The authors report the first small mol. selective for BPTF over Brd4, termed AU1.  The Kd = 2.8 μM for AU1, which is active in a cell-based reporter assay.  No binding was detected with Brd4.  Three new Brd4 inhibitors with submicromolar affinity were also discovered.  Brd4 hits were validated in a thermal stability assay and potency detd. via fluorescence anisotropy.  The speed, ease of interpretation, and low protein concn. needed for protein-obsd. 19F NMR expts. in a multiprotein format offers a new method to discover and characterize selective ligands for bromodomain-contg. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjhnKasUdNL7Vg90H21EOLACvtfcHk0li_-tEgbs-l8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfI&md5=584fa6b83f8bfa661f55edb1078e570c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00483%26sid%3Dliteratum%253Aachs%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DCassel%26aufirst%3DM.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DAdhikari%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDos%2BSantos%26aufirst%3DC.%2BO.%26aulast%3DHall%26aufirst%3DJ.%2BL.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DDual%2520Screening%2520of%2520BPTF%2520and%2520Brd4%2520Using%2520Protein-Observed%2520Fluorine%2520NMR%2520Uncovers%2520New%2520Bromodomain%2520Probe%2520Molecules%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D10%26spage%3D2246%26epage%3D2256%26doi%3D10.1021%2Facschembio.5b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bower, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. L.</span></span> <span> </span><span class="NLM_article-title">Phenoxypyrimidine Inhibitors of P38α Kinase: Synthesis and Statistical Evaluation of the P38 Inhibitory Potencies of a Series of 1-(Piperidin-4-Yl)-4-(4-Fluorophenyl)-5-(2-Phenoxypyrimidin-4-Yl) Imidazoles</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1123</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(01)00163-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2FS0960-894X%2801%2900163-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1123-1126&issue=9&author=J.+C.+Boehmauthor=M.+J.+Bowerauthor=T.+F.+Gallagherauthor=S.+Kassisauthor=S.+R.+Johnsonauthor=J.+L.+Adams&title=Phenoxypyrimidine+Inhibitors+of+P38%CE%B1+Kinase%3A+Synthesis+and+Statistical+Evaluation+of+the+P38+Inhibitory+Potencies+of+a+Series+of+1-%28Piperidin-4-Yl%29-4-%284-Fluorophenyl%29-5-%282-Phenoxypyrimidin-4-Yl%29+Imidazoles&doi=10.1016%2FS0960-894X%2801%2900163-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2801%2900163-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252801%252900163-9%26sid%3Dliteratum%253Aachs%26aulast%3DBoehm%26aufirst%3DJ.%2BC.%26aulast%3DBower%26aufirst%3DM.%2BJ.%26aulast%3DGallagher%26aufirst%3DT.%2BF.%26aulast%3DKassis%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DAdams%26aufirst%3DJ.%2BL.%26atitle%3DPhenoxypyrimidine%2520Inhibitors%2520of%2520P38%25CE%25B1%2520Kinase%253A%2520Synthesis%2520and%2520Statistical%2520Evaluation%2520of%2520the%2520P38%2520Inhibitory%2520Potencies%2520of%2520a%2520Series%2520of%25201-%2528Piperidin-4-Yl%2529-4-%25284-Fluorophenyl%2529-5-%25282-Phenoxypyrimidin-4-Yl%2529%2520Imidazoles%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2001%26volume%3D11%26issue%3D9%26spage%3D1123%26epage%3D1126%26doi%3D10.1016%2FS0960-894X%2801%2900163-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors for Rationally Designed Polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&issue=4&author=P.+Ciceriauthor=S.+M%C3%BCllerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+Kinase-Bromodomain+Inhibitors+for+Rationally+Designed+Polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lgq0qU490fm0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520for%2520Rationally%2520Designed%2520Polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26issue%3D4%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-Peptide Displacement Assays for Interactome Mapping and Inhibitor Discovery</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2899</span>, <span class="refDoi"> DOI: 10.1039/c1mb05099k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1039%2Fc1mb05099k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21804994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFakt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=2899&issue=10&author=M.+Philpottauthor=J.+Yangauthor=T.+Tumberauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=T.+Keatesauthor=I.+Felletarauthor=A.+Ciulliauthor=S.+Knappauthor=T.+D.+Heightman&title=Bromodomain-Peptide+Displacement+Assays+for+Interactome+Mapping+and+Inhibitor+Discovery&doi=10.1039%2Fc1mb05099k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-peptide displacement assays for interactome mapping and inhibitor discovery</span></div><div class="casAuthors">Philpott, Martin; Yang, Jing; Tumber, Tony; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Keates, Tracy; Felletar, Ildiko; Ciulli, Alessio; Knapp, Stefan; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2899-2908</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone lysine acetylation is a key component of epigenetic regulation of gene transcription.  Bromodomains, found in histone acetyl transferases and other chromatin-assocd. proteins, bind selectively to acetylated lysines, acting as "readers" of the histone code, and have recently been shown to contain a druggable binding pocket.  Here we report the development of high-throughput assays that quantify the binding of bromodomains to acetylated histone peptides.  We have used these assays to screen for histone binding partners of as yet uncharacterized bromodomains, adding to current knowledge of the histone code and expanding the repertoire of assays for chem. probe discovery.  We have also demonstrated that these assays can be used to detect small mol. binding from the very weak to the nanomolar range.  This assay methodol. is thereby anticipated to provide the basis both for broader interactome profiling and for small mol. inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5mWM_E4IUzbVg90H21EOLACvtfcHk0lgO82bTOJU24w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFakt77P&md5=0f9eccb2d2761e1707c9b04bccc76050</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2Fc1mb05099k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1mb05099k%26sid%3Dliteratum%253Aachs%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTumber%26aufirst%3DT.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DBromodomain-Peptide%2520Displacement%2520Assays%2520for%2520Interactome%2520Mapping%2520and%2520Inhibitor%2520Discovery%26jtitle%3DMol.%2520BioSyst.%26date%3D2011%26volume%3D7%26issue%3D10%26spage%3D2899%26doi%3D10.1039%2Fc1mb05099k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C.</span></span> <span> </span><span class="NLM_article-title">Fluorinated Aromatic Amino Acids Are Sensitive 19F NMR Probes for Bromodomain-Ligand Interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2755</span>– <span class="NLM_lpage">2760</span>, <span class="refDoi"> DOI: 10.1021/cb5007344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5007344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs12gtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2755-2760&issue=12&author=N.+K.+Mishraauthor=A.+K.+Urickauthor=S.+W.+J.+Emberauthor=E.+Schonbrunnauthor=W.+C.+Pomerantz&title=Fluorinated+Aromatic+Amino+Acids+Are+Sensitive+19F+NMR+Probes+for+Bromodomain-Ligand+Interactions&doi=10.1021%2Fcb5007344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorinated Aromatic Amino Acids Are Sensitive 19F NMR Probes for Bromodomain-Ligand Interactions</span></div><div class="casAuthors">Mishra, Neeraj K.; Urick, Andrew K.; Ember, Stuart W. J.; Schonbrunn, Ernst; Pomerantz, William C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2755-2760</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors describe a 19F NMR method for detecting bromodomain-ligand interactions using fluorine-labeled arom. amino acids due to the conservation of arom. residues in the bromodomain binding site.  The authors test the sensitivity, accuracy, and speed of this method with small mol. ligands (+)-JQ1, BI2536, Dinaciclib, TG101348, and acetaminophen using three bromodomains Brd4, BrdT, and BPTF.  Simplified 19F NMR spectra allowed for simultaneous testing of multiple bromodomains to assess selectivity and identification of a new BPTF ligand.  Fluorine labeling only modestly affected the Brd4 structure and function assessed by isothermal titrn. calorimetry, CD, and x-ray crystallog.  The speed, ease of interpretation, and low concn. of protein needed for binding expts. affords a new method to discover and characterize both native and new ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoejxq3KKVddrVg90H21EOLACvtfcHk0lhCUP5DHNQUkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs12gtLfP&md5=97f9edadc91cba8670f54bcca57fa03f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fcb5007344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5007344%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%26atitle%3DFluorinated%2520Aromatic%2520Amino%2520Acids%2520Are%2520Sensitive%252019F%2520NMR%2520Probes%2520for%2520Bromodomain-Ligand%2520Interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26issue%3D12%26spage%3D2755%26epage%3D2760%26doi%3D10.1021%2Fcb5007344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arntson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Protein-Observed Fluorine NMR: A Bioorthogonal Approach for Small Molecule Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">5158</span>– <span class="NLM_lpage">5171</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01447</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01447" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFSmtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5158-5171&issue=11&author=K.+E.+Arntsonauthor=W.+C.+K.+Pomerantz&title=Protein-Observed+Fluorine+NMR%3A+A+Bioorthogonal+Approach+for+Small+Molecule+Discovery&doi=10.1021%2Facs.jmedchem.5b01447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Observed Fluorine NMR: A Bioorthogonal Approach for Small Molecule Discovery</span></div><div class="casAuthors">Arntson, Keith E.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5158-5171</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 19F isotope is 100% naturally abundant and is the second most sensitive and stable NMR-active nucleus.  Unlike the ubiquitous hydrogen atom, fluorine is nearly absent in biol. systems, making it a unique bioorthogonal atom for probing mol. interactions in biol.  Over 73 fluorinated proteins have been studied by 19F NMR since the seminal studies of Hull and Sykes in 1974.  With advances in cryoprobe prodn. and fluorinated amino acid incorporation strategies, protein-based 19F NMR offers opportunities to the medicinal chemist for characterizing and ultimately discovering new small mol. protein ligands.  This review will highlight new advances using 19F NMR for characterizing small mol. interactions with both small and large proteins as well as detailing NMR resonance assignment challenges and amino acid incorporation approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8Heq0upzHCbVg90H21EOLACvtfcHk0lhMDGChlEbCkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFSmtb%252FI&md5=1900310c924bce5b79d71008e6e786fb</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01447%26sid%3Dliteratum%253Aachs%26aulast%3DArntson%26aufirst%3DK.%2BE.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DProtein-Observed%2520Fluorine%2520NMR%253A%2520A%2520Bioorthogonal%2520Approach%2520for%2520Small%2520Molecule%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D11%26spage%3D5158%26epage%3D5171%26doi%3D10.1021%2Facs.jmedchem.5b01447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishima, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torchia, D. A.</span></span> <span> </span><span class="NLM_article-title">Protein Dynamics from NMR</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">743</span>, <span class="refDoi"> DOI: 10.1038/78963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2F78963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10966641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFWlu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=740-743&issue=9&author=R.+Ishimaauthor=D.+A.+Torchia&title=Protein+Dynamics+from+NMR&doi=10.1038%2F78963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics from NMR</span></div><div class="casAuthors">Ishima, Rieko; Torchia, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">740-743</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">A review with 66 refs.  This review surveys recent investigations of conformational fluctuations of proteins in soln. using NMR techniques.  Advances in exptl. methods have provided more accurate means of characterizing fast and slow internal motions as well as overall diffusion.  The information obtained from NMR dynamics expts. provides insights into specific structural changes or configurational energetics assocd. with function.  A variety of applications illustrate that studies of protein dynamics provide insights into protein-protein interactions, target recognition, ligand binding, and enzyme function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9AJMPUp5LpbVg90H21EOLACvtfcHk0lhMDGChlEbCkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFWlu7o%253D&md5=5c05fbb0bbc273789c5e45e1adc9497c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2F78963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F78963%26sid%3Dliteratum%253Aachs%26aulast%3DIshima%26aufirst%3DR.%26aulast%3DTorchia%26aufirst%3DD.%2BA.%26atitle%3DProtein%2520Dynamics%2520from%2520NMR%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D7%26issue%3D9%26spage%3D740%26epage%3D743%26doi%3D10.1038%2F78963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Analysis of Water Stability in Bromodomain Binding Pockets with Grand Canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=29863194" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=19&issue=1&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-Scale+Analysis+of+Water+Stability+in+Bromodomain+Binding+Pockets+with+Grand+Canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-Scale%2520Analysis%2520of%2520Water%2520Stability%2520in%2520Bromodomain%2520Binding%2520Pockets%2520with%2520Grand%2520Canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26issue%3D1%26spage%3D19%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bharatham, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavish, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shadrick, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelat, A. A.</span></span> <span> </span><span class="NLM_article-title">The Role of ZA Channel Water-Mediated Interactions in the Design of Bromodomain-Selective BET Inhibitors</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2018.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.jmgm.2018.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=29605436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1Gks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=197-210&author=N.+Bharathamauthor=P.+J.+Slavishauthor=W.+R.+Shadrickauthor=B.+M.+Youngauthor=A.+A.+Shelat&title=The+Role+of+ZA+Channel+Water-Mediated+Interactions+in+the+Design+of+Bromodomain-Selective+BET+Inhibitors&doi=10.1016%2Fj.jmgm.2018.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ZA channel water-mediated interactions in the design of bromodomain-selective BET inhibitors</span></div><div class="casAuthors">Bharatham, Nagakumar; Slavish, Peter J.; Young, Brandon M.; Shelat, Anang A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">197-210</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Bromodomain and Extra-Terminal domain (BET) family of proteins are involved in the regulation of gene transcription, and their dysregulation is implicated in several diseases including cancer.  BET proteins contain two tandem bromodomains (BD1 and BD2) that independently recognize acetylated-lysine residues and appear to have distinct biol. roles.  We compared several published co-crystal structures and found five positions near the substrate binding pocket that vary between BET bromodomains.  One position located in the ZA loop has unique properties.  In BRD2-4, this residue is glutamine in BD1 and lysine in BD2; in BRDT, this residue is arginine in BD1 and asparagine in BD2.  Using mol. modeling, we identified differences in the water-mediated network at this position between bromodomains.  Mol. dynamics simulations helped rationalize the obsd. bromodomain selectivity for exemplar BET inhibitors and a congeneric series of tetrahydroquinolines (THQ) that differed by a single heteroatom near the ZA channel.  The 2-furan SJ830599, the most BD2-selective THQ analog, did not disrupt the water-mediated networks in either domain, but was electrostatically-repulsed by the specific arrangement of the W5 water dipole in BD1.  Our work underscores the value of exploring water-mediated interactions to study ligand binding, and highlights the difficulty of optimizing polar interactions due to high desolvation penalties.  Finally, we suggest further modifications to THQ-based BET inhibitors that would increase BD2-selectivity in BRD2-4, while minimizing affinity for one or both bromodomains of BRDT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptIhxYNtOqIbVg90H21EOLACvtfcHk0lhbcpk6C6OzcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1Gks7w%253D&md5=8db9f397aa72ae776017ffe829aa30ce</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2018.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2018.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DBharatham%26aufirst%3DN.%26aulast%3DSlavish%26aufirst%3DP.%2BJ.%26aulast%3DShadrick%26aufirst%3DW.%2BR.%26aulast%3DYoung%26aufirst%3DB.%2BM.%26aulast%3DShelat%26aufirst%3DA.%2BA.%26atitle%3DThe%2520Role%2520of%2520ZA%2520Channel%2520Water-Mediated%2520Interactions%2520in%2520the%2520Design%2520of%2520Bromodomain-Selective%2520BET%2520Inhibitors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2018%26volume%3D81%26spage%3D197%26epage%3D210%26doi%3D10.1016%2Fj.jmgm.2018.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, K.</span></span> <span> </span><span class="NLM_article-title">The Polar Warhead of a TRIM24 Bromodomain Inhibitor Rearranges a Water-Mediated Interaction Network</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>284</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1082</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1111/febs.14041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1111%2Ffebs.14041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2017&pages=1082-1095&issue=7&author=J.+Liuauthor=F.+Liauthor=H.+Baoauthor=Y.+Jiangauthor=S.+Zhangauthor=R.+Maauthor=J.+Gaoauthor=J.+Wuauthor=K.+Ruan&title=The+Polar+Warhead+of+a+TRIM24+Bromodomain+Inhibitor+Rearranges+a+Water-Mediated+Interaction+Network&doi=10.1111%2Ffebs.14041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1111%2Ffebs.14041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.14041%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBao%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRuan%26aufirst%3DK.%26atitle%3DThe%2520Polar%2520Warhead%2520of%2520a%2520TRIM24%2520Bromodomain%2520Inhibitor%2520Rearranges%2520a%2520Water-Mediated%2520Interaction%2520Network%26jtitle%3DFEBS%2520J.%26date%3D2017%26volume%3D284%26issue%3D7%26spage%3D1082%26epage%3D1095%26doi%3D10.1111%2Ffebs.14041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canagarajah, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassisà, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobb, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Meguid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsmith, E. J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Inhibitor Selectivity in MAP Kinases</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1117</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(98)00113-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2FS0969-2126%2898%2900113-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=1117-1128&issue=9&author=Z.+Wangauthor=B.+J.+Canagarajahauthor=J.+C.+Boehmauthor=S.+Kassis%C3%A0author=M.+H.+Cobbauthor=P.+R.+Youngauthor=S.+Abdel-Meguidauthor=J.+L.+Adamsauthor=E.+J.+Goldsmith&title=Structural+Basis+of+Inhibitor+Selectivity+in+MAP+Kinases&doi=10.1016%2FS0969-2126%2898%2900113-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2898%2900113-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252898%252900113-0%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DCanagarajah%26aufirst%3DB.%2BJ.%26aulast%3DBoehm%26aufirst%3DJ.%2BC.%26aulast%3DKassis%25C3%25A0%26aufirst%3DS.%26aulast%3DCobb%26aufirst%3DM.%2BH.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DAbdel-Meguid%26aufirst%3DS.%26aulast%3DAdams%26aufirst%3DJ.%2BL.%26aulast%3DGoldsmith%26aufirst%3DE.%2BJ.%26atitle%3DStructural%2520Basis%2520of%2520Inhibitor%2520Selectivity%2520in%2520MAP%2520Kinases%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26issue%3D9%26spage%3D1117%26epage%3D1128%26doi%3D10.1016%2FS0969-2126%2898%2900113-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span> <span> </span><span class="NLM_article-title">The Cellular Thermal Shift Assay for Evaluating Drug Target Interactions in Cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&issue=9&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundb%C3%A4ckauthor=P.+Nordlundauthor=D.+M.+Molina&title=The+Cellular+Thermal+Shift+Assay+for+Evaluating+Drug+Target+Interactions+in+Cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0ljkbvVAmylSVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26atitle%3DThe%2520Cellular%2520Thermal%2520Shift%2520Assay%2520for%2520Evaluating%2520Drug%2520Target%2520Interactions%2520in%2520Cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAngelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Design and Characterization of Bivalent BET Inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1096</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnchembio.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=27775715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1089-1096&issue=12&author=M.+Tanakaauthor=J.+M.+Robertsauthor=H.-S.+Seoauthor=A.+Souzaauthor=J.+Paulkauthor=T.+G.+Scottauthor=S.+L.+DeAngeloauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Design+and+Characterization+of+Bivalent+BET+Inhibitors&doi=10.1038%2Fnchembio.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of bivalent BET inhibitors</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Seo, Hyuk-Soo; Souza, Amanda; Paulk, Joshiawa; Scott, Thomas G.; DeAngelo, Stephen L.; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1089-1096</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cellular signaling is often propagated by multivalent interactions.  Multivalency creates avidity, allowing stable biophys. recognition.  Multivalency is an attractive strategy for achieving potent binding to protein targets, as the affinity of bivalent ligands is often greater than the sum of monovalent affinities.  The bromodomain and extraterminal domain (BET) family of transcriptional coactivators features tandem bromodomains through which BET proteins bind acetylated histones and transcription factors.  All reported antagonists of the BET protein BRD4 bind in a monovalent fashion.  Here the authors describe, to the knowledge for the first time, a bivalent BET bromodomain inhibitor-MT1-which has unprecedented potency.  Biophys. and biochem. studies suggest MT1 is an intramol. bivalent BRD4 binder that is more than 100-fold more potent, in cellular assays, than the corresponding monovalent antagonist, JQ1.  MT1 significantly (P < 0.05) delayed leukemia progression in mice, as compared to JQ1.  These data qualify a powerful chem. probe for BET bromodomains and a rationale for further development of multidomain inhibitors of epigenetic reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQwc6X0VYYbVg90H21EOLACvtfcHk0ljkbvVAmylSVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurrM&md5=7319203dac409d5dee3c580b49fcf185</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2209%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDesign%2520and%2520Characterization%2520of%2520Bivalent%2520BET%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26issue%3D12%26spage%3D1089%26epage%3D1096%26doi%3D10.1038%2Fnchembio.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span> (<span class="NLM_issue">6141</span>),  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&issue=6141&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+Drug+Target+Engagement+in+Cells+and+Tissues+Using+the+Cellular+Thermal+Shift+Assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0lgcPn9BempDTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520Drug%2520Target%2520Engagement%2520in%2520Cells%2520and%2520Tissues%2520Using%2520the%2520Cellular%2520Thermal%2520Shift%2520Assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26issue%3D6141%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin, B. M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-Containing Protein 4 – Independent Transcriptional Activation by Autoimmune Regulator (AIRE) and NF-κB</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">4993</span>– <span class="NLM_lpage">5004</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA117.001518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1074%2Fjbc.RA117.001518" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=4993-5004&issue=14&author=F.+Huangauthor=W.+Shaoauthor=K.+Fujinagaauthor=B.+M.+Peterlin&title=Bromodomain-Containing+Protein+4+%E2%80%93+Independent+Transcriptional+Activation+by+Autoimmune+Regulator+%28AIRE%29+and+NF-%CE%BAB&doi=10.1074%2Fjbc.RA117.001518"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA117.001518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA117.001518%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DF.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DFujinaga%26aufirst%3DK.%26aulast%3DPeterlin%26aufirst%3DB.%2BM.%26atitle%3DBromodomain-Containing%2520Protein%25204%2520%25E2%2580%2593%2520Independent%2520Transcriptional%2520Activation%2520by%2520Autoimmune%2520Regulator%2520%2528AIRE%2529%2520and%2520NF-%25CE%25BAB%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26issue%3D14%26spage%3D4993%26epage%3D5004%26doi%3D10.1074%2Fjbc.RA117.001518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberspächer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmees, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span> <span> </span><span class="NLM_article-title">Affinity Map of Bromodomain Protein 4 (BRD4) Interactions with the Histone H4 Tail and the Small Molecule Inhibitor JQ1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">9304</span>– <span class="NLM_lpage">9319</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.523019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1074%2Fjbc.M113.523019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=24497639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVWjs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=9304-9319&issue=13&author=M.+Jungauthor=M.+Philpottauthor=S.+M%C3%BCllerauthor=J.+Schulzeauthor=V.+Badockauthor=U.+Ebersp%C3%A4cherauthor=D.+Moosmayerauthor=B.+Baderauthor=N.+Schmeesauthor=A.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=B.+Haendler&title=Affinity+Map+of+Bromodomain+Protein+4+%28BRD4%29+Interactions+with+the+Histone+H4+Tail+and+the+Small+Molecule+Inhibitor+JQ1&doi=10.1074%2Fjbc.M113.523019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Affinity Map of Bromodomain Protein 4 (BRD4) Interactions with the Histone H4 Tail and the Small Molecule Inhibitor JQ1</span></div><div class="casAuthors">Jung, Marie; Philpott, Martin; Mueller, Susanne; Schulze, Jessica; Badock, Volker; Eberspaecher, Uwe; Moosmayer, Dieter; Bader, Benjamin; Schmees, Norbert; Fernandez-Montalvan, Amaury; Haendler, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9304-9319</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Bromodomain protein 4 (BRD4) is a member of the bromodomain and extra-terminal domain (BET) protein family.  It binds to acetylated histone tails via its tandem bromodomains BD1 and BD2 and forms a complex with the pos. transcription elongation factor b, which controls phosphorylation of RNA polymerase II, ultimately leading to stimulation of transcription elongation.  An essential role of BRD4 in cell proliferation and cancer growth has been reported in several recent studies.  We analyzed the binding of BRD4 BD1 and BD2 to different partners and showed that the strongest interactions took place with di- and tetra-acetylated peptides derived from the histone 4 N-terminal tail.  We also found that several histone 4 residues neighboring the acetylated lysines significantly influenced binding.  We generated 10 different BRD4 BD1 mutants and analyzed their affinities to acetylated histone tails and to the BET inhibitor JQ1 using several complementary biochem. and biophys. methods.  The impact of these mutations was confirmed in a cellular environment.  Altogether, the results show that Trp-81, Tyr-97, Asn-140, and Met-149 play similarly important roles in the recognition of acetylated histones and JQ1.  Pro-82, Leu-94, Asp-145, and Ile-146 have a more differentiated role, suggesting that different kinds of interactions take place and that resistance mutations compatible with BRD4 function are possible.  Our study extends the knowledge on the contribution of individual BRD4 amino acids to histone and JQ1 binding and may help in the design of new BET antagonists with improved pharmacol. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8wAGYLccT1bVg90H21EOLACvtfcHk0lgcPn9BempDTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVWjs70%253D&md5=bf1d882e680e82660065b3e7b0a0f0f9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.523019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.523019%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchulze%26aufirst%3DJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DEbersp%25C3%25A4cher%26aufirst%3DU.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DBader%26aufirst%3DB.%26aulast%3DSchmees%26aufirst%3DN.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%26aulast%3DHaendler%26aufirst%3DB.%26atitle%3DAffinity%2520Map%2520of%2520Bromodomain%2520Protein%25204%2520%2528BRD4%2529%2520Interactions%2520with%2520the%2520Histone%2520H4%2520Tail%2520and%2520the%2520Small%2520Molecule%2520Inhibitor%2520JQ1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26issue%3D13%26spage%3D9304%26epage%3D9319%26doi%3D10.1074%2Fjbc.M113.523019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekido, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akahoshi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukaida, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, K.</span></span> <span> </span><span class="NLM_article-title">Essential Involvement of Interleukin-8 (IL-8) in Acute Inflammation</span>. <i>J. Leukocyte Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">564</span>, <span class="refDoi"> DOI: 10.1002/jlb.56.5.559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1002%2Fjlb.56.5.559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=7964163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADyaK2cXmvFymtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1994&pages=559-564&issue=5&author=A.+Haradaauthor=N.+Sekidoauthor=T.+Akahoshiauthor=T.+Wadaauthor=N.+Mukaidaauthor=K.+Matsushima&title=Essential+Involvement+of+Interleukin-8+%28IL-8%29+in+Acute+Inflammation&doi=10.1002%2Fjlb.56.5.559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Essential involvement of interleukin-8 (IL-8) in acute inflammation</span></div><div class="casAuthors">Harada, Akihisa; Sekido, Nobuaki; Akahoshi, Tohru; Wada, Takashi; Mukaida, Naofumi; Matsushima, Kouji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">559-64</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A review with 48 refs. summarizing the effects of anti-interleukin-8 antibody on rabbit exptl. models of acute inflammation, including lipopolysaccharide-induced dermatitis, lipopolysaccharide/interleukin-1-induced arthritis, ischemia-reperfusion injury in lung, and acute immune complex-type glomerulonephritis.  The authors also discuss the potential use of anti-interleukin-8 antibody for the treatment of inflammatory reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaVYZTnD1V9rVg90H21EOLACvtfcHk0lgcPn9BempDTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmvFymtbo%253D&md5=03b803d7d0d920fdbe6de0a8f7b42260</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjlb.56.5.559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjlb.56.5.559%26sid%3Dliteratum%253Aachs%26aulast%3DHarada%26aufirst%3DA.%26aulast%3DSekido%26aufirst%3DN.%26aulast%3DAkahoshi%26aufirst%3DT.%26aulast%3DWada%26aufirst%3DT.%26aulast%3DMukaida%26aufirst%3DN.%26aulast%3DMatsushima%26aufirst%3DK.%26atitle%3DEssential%2520Involvement%2520of%2520Interleukin-8%2520%2528IL-8%2529%2520in%2520Acute%2520Inflammation%26jtitle%3DJ.%2520Leukocyte%2520Biol.%26date%3D1994%26volume%3D56%26issue%3D5%26spage%3D559%26epage%3D564%26doi%3D10.1002%2Fjlb.56.5.559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hexum, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummond, K. M.</span></span> <span> </span><span class="NLM_article-title">A Redox Economical Synthesis of Bioactive 6,12-Guaianolides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2644</span>– <span class="NLM_lpage">2647</span>, <span class="refDoi"> DOI: 10.1021/ol400904y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol400904y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=2644-2647&issue=11&author=B.+Wenauthor=J.+K.+Hexumauthor=J.+C.+Widenauthor=D.+A.+Harkiauthor=K.+M.+Brummond&title=A+Redox+Economical+Synthesis+of+Bioactive+6%2C12-Guaianolides&doi=10.1021%2Fol400904y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fol400904y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol400904y%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DB.%26aulast%3DHexum%26aufirst%3DJ.%2BK.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DBrummond%26aufirst%3DK.%2BM.%26atitle%3DA%2520Redox%2520Economical%2520Synthesis%2520of%2520Bioactive%25206%252C12-Guaianolides%26jtitle%3DOrg.%2520Lett.%26date%3D2013%26volume%3D15%26issue%3D11%26spage%3D2644%26epage%3D2647%26doi%3D10.1021%2Fol400904y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of Inflammation by a Synthetic Histone Mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span> (<span class="NLM_issue">7327</span>),  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&issue=7327&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+Inflammation+by+a+Synthetic+Histone+Mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0ljPux5r1QJWaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520Inflammation%2520by%2520a%2520Synthetic%2520Histone%2520Mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26issue%3D7327%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolajczyk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>190</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">3670</span>– <span class="NLM_lpage">3678</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1202838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670-3678&issue=7&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+Protein+Function+Is+Required+for+Inflammation%3A+Brd2+Genetic+Disruption+and+BET+Inhibitor+JQ1+Impair+Mouse+Macrophage+Inflammatory+Responses&doi=10.4049%2Fjimmunol.1202838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-α while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0ljPux5r1QJWaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520Protein%2520Function%2520Is%2520Required%2520for%2520Inflammation%253A%2520Brd2%2520Genetic%2520Disruption%2520and%2520BET%2520Inhibitor%2520JQ1%2520Impair%2520Mouse%2520Macrophage%2520Inflammatory%2520Responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26issue%3D7%26spage%3D3670%26epage%3D3678%26doi%3D10.4049%2Fjimmunol.1202838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milletti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermann, J. C.</span></span> <span> </span><span class="NLM_article-title">Targeted Kinase Selectivity from Kinase Profiling Data</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1021/ml300012r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300012r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=383-386&issue=5&author=F.+Millettiauthor=J.+C.+Hermann&title=Targeted+Kinase+Selectivity+from+Kinase+Profiling+Data&doi=10.1021%2Fml300012r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fml300012r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300012r%26sid%3Dliteratum%253Aachs%26aulast%3DMilletti%26aufirst%3DF.%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26atitle%3DTargeted%2520Kinase%2520Selectivity%2520from%2520Kinase%2520Profiling%2520Data%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26issue%3D5%26spage%3D383%26epage%3D386%26doi%3D10.1021%2Fml300012r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&issue=11&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lidTzBKDTi63Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26issue%3D11%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibel, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laydon, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenthal, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fier-Thompson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abt, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soreson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smietana, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garigipati, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erhard, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krog, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheldrake, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. L.</span></span> <span> </span><span class="NLM_article-title">Regulation of Stress-Induced Cytokine Production by Pyridinylimidazoles; Inhibition of CSBP Kinase</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(96)00212-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2FS0968-0896%2896%2900212-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=49-64&issue=1&author=T.+F.+Gallagherauthor=G.+L.+Seibelauthor=S.+Kassisauthor=J.+T.+Laydonauthor=M.+J.+Blumenthalauthor=J.+C.+Leeauthor=D.+Leeauthor=J.+C.+Boehmauthor=S.+M.+Fier-Thompsonauthor=J.+W.+Abtauthor=M.+E.+Soresonauthor=J.+M.+Smietanaauthor=R.+F.+Hallauthor=R.+S.+Garigipatiauthor=P.+E.+Benderauthor=K.+F.+Erhardauthor=A.+J.+Krogauthor=G.+A.+Hofmannauthor=P.+L.+Sheldrakeauthor=P.+C.+McDonnellauthor=S.+Kumarauthor=P.+R.+Youngauthor=J.+L.+Adams&title=Regulation+of+Stress-Induced+Cytokine+Production+by+Pyridinylimidazoles%3B+Inhibition+of+CSBP+Kinase&doi=10.1016%2FS0968-0896%2896%2900212-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2896%2900212-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252896%252900212-X%26sid%3Dliteratum%253Aachs%26aulast%3DGallagher%26aufirst%3DT.%2BF.%26aulast%3DSeibel%26aufirst%3DG.%2BL.%26aulast%3DKassis%26aufirst%3DS.%26aulast%3DLaydon%26aufirst%3DJ.%2BT.%26aulast%3DBlumenthal%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DBoehm%26aufirst%3DJ.%2BC.%26aulast%3DFier-Thompson%26aufirst%3DS.%2BM.%26aulast%3DAbt%26aufirst%3DJ.%2BW.%26aulast%3DSoreson%26aufirst%3DM.%2BE.%26aulast%3DSmietana%26aufirst%3DJ.%2BM.%26aulast%3DHall%26aufirst%3DR.%2BF.%26aulast%3DGarigipati%26aufirst%3DR.%2BS.%26aulast%3DBender%26aufirst%3DP.%2BE.%26aulast%3DErhard%26aufirst%3DK.%2BF.%26aulast%3DKrog%26aufirst%3DA.%2BJ.%26aulast%3DHofmann%26aufirst%3DG.%2BA.%26aulast%3DSheldrake%26aufirst%3DP.%2BL.%26aulast%3DMcDonnell%26aufirst%3DP.%2BC.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DAdams%26aufirst%3DJ.%2BL.%26atitle%3DRegulation%2520of%2520Stress-Induced%2520Cytokine%2520Production%2520by%2520Pyridinylimidazoles%253B%2520Inhibition%2520of%2520CSBP%2520Kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1997%26volume%3D5%26issue%3D1%26spage%3D49%26epage%3D64%26doi%3D10.1016%2FS0968-0896%2896%2900212-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9305</span>– <span class="NLM_lpage">9320</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00438</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00438" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9305-9320&issue=20&author=L.+Carlinoauthor=G.+Rastelli&title=Dual+Kinase-Bromodomain+Inhibitors+in+Anticancer+Drug+Discovery%3A+A+Structural+and+Pharmacological+Perspective&doi=10.1021%2Facs.jmedchem.6b00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span></div><div class="casAuthors">Carlino, Luca; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9305-9320</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases play crucial roles in several cell transformation processes and are validated drug targets for many human diseases, including cancer.  Nevertheless, most tumors have eluded the effects of inhibition of a single kinase by activating resistance mechanisms and/or alternative pathways and escape mechanisms.  In recent years, multitarget approaches directed toward inhibition of kinases and targets of different families have received increasing attention.  In particular, co-targeting kinases and bromodomain epigenetic reader proteins has rapidly emerged as a promising approach to cancer drug development.  In this manuscript, we will review the recent discoveries that led to the identification and optimization of dual kinase/bromodomain inhibitors.  We will analyze and compare the structural features required for dual inhibition and comment on the potential of this approach in anticancer drug discovery.  Moreover, we will introduce computational approaches useful for the identification of dual kinase/bromodomain inhibitors and generate ad hoc pharmacophore and docking models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkHDSBWCGdbVg90H21EOLACvtfcHk0lidTzBKDTi63Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP&md5=1351a235bba09782160a86ae3f904833</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00438%26sid%3Dliteratum%253Aachs%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520in%2520Anticancer%2520Drug%2520Discovery%253A%2520A%2520Structural%2520and%2520Pharmacological%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D20%26spage%3D9305%26epage%3D9320%26doi%3D10.1021%2Facs.jmedchem.6b00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlich, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durden, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutateladze, T. G.</span></span> <span> </span><span class="NLM_article-title">Dual-Activity PI3K–BRD4 Inhibitor for the Orthogonal Inhibition of MYC to Block Tumor Growth and Metastasis</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">E1072</span>– <span class="NLM_lpage">E1080</span>, <span class="refDoi"> DOI: 10.1073/pnas.1613091114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1073%2Fpnas.1613091114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E1072-E1080&issue=7&author=F.+H.+Andrewsauthor=A.+R.+Singhauthor=S.+Joshiauthor=C.+A.+Smithauthor=G.+A.+Moralesauthor=J.+R.+Garlichauthor=D.+L.+Durdenauthor=T.+G.+Kutateladze&title=Dual-Activity+PI3K%E2%80%93BRD4+Inhibitor+for+the+Orthogonal+Inhibition+of+MYC+to+Block+Tumor+Growth+and+Metastasis&doi=10.1073%2Fpnas.1613091114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1613091114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1613091114%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DF.%2BH.%26aulast%3DSingh%26aufirst%3DA.%2BR.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DC.%2BA.%26aulast%3DMorales%26aufirst%3DG.%2BA.%26aulast%3DGarlich%26aufirst%3DJ.%2BR.%26aulast%3DDurden%26aufirst%3DD.%2BL.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DDual-Activity%2520PI3K%25E2%2580%2593BRD4%2520Inhibitor%2520for%2520the%2520Orthogonal%2520Inhibition%2520of%2520MYC%2520to%2520Block%2520Tumor%2520Growth%2520and%2520Metastasis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26issue%3D7%26spage%3DE1072%26epage%3DE1080%26doi%3D10.1073%2Fpnas.1613091114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. L.</span>; <span class="NLM_string-name">Sheldrake, P. W.</span>; <span class="NLM_string-name">Gallagher, T. F.</span>; <span class="NLM_string-name">Garigipati, R.</span></span> <span> </span><span class="NLM_article-title">Novel 1,4,5-Substituted Imidazole Compounds and Compositions for Use in Therapy as Cytokine Inhibitors</span>. U.S. Patent <span class="NLM_patent">US5593992A</span>, <span class="NLM_year">1995</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&author=J.+L.+Adams&author=P.+W.+Sheldrake&author=T.+F.+Gallagher&author=R.+Garigipati&title=Novel+1%2C4%2C5-Substituted+Imidazole+Compounds+and+Compositions+for+Use+in+Therapy+as+Cytokine+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%2BL.%26atitle%3DNovel%25201%252C4%252C5-Substituted%2520Imidazole%2520Compounds%2520and%2520Compositions%2520for%2520Use%2520in%2520Therapy%2520as%2520Cytokine%2520Inhibitors%26date%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryzhakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarret, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouklovsky, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, G.</span></span> <span> </span><span class="NLM_article-title">Radical-Mediated Dearomatization of Indoles with Sulfinate Reagents for the Synthesis of Fluorinated Spirocyclic Indolines</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">6336</span>– <span class="NLM_lpage">6339</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.7b03155</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.7b03155" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=6336-6339&issue=23&author=D.+Ryzhakovauthor=M.+Jarretauthor=R.+Guillotauthor=C.+Kouklovskyauthor=G.+Vincent&title=Radical-Mediated+Dearomatization+of+Indoles+with+Sulfinate+Reagents+for+the+Synthesis+of+Fluorinated+Spirocyclic+Indolines&doi=10.1021%2Facs.orglett.7b03155"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.7b03155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.7b03155%26sid%3Dliteratum%253Aachs%26aulast%3DRyzhakov%26aufirst%3DD.%26aulast%3DJarret%26aufirst%3DM.%26aulast%3DGuillot%26aufirst%3DR.%26aulast%3DKouklovsky%26aufirst%3DC.%26aulast%3DVincent%26aufirst%3DG.%26atitle%3DRadical-Mediated%2520Dearomatization%2520of%2520Indoles%2520with%2520Sulfinate%2520Reagents%2520for%2520the%2520Synthesis%2520of%2520Fluorinated%2520Spirocyclic%2520Indolines%26jtitle%3DOrg.%2520Lett.%26date%3D2017%26volume%3D19%26issue%3D23%26spage%3D6336%26epage%3D6339%26doi%3D10.1021%2Facs.orglett.7b03155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.</span>; <span class="NLM_string-name">Erdman, P. E.</span>; <span class="NLM_string-name">Fung, L.</span>; <span class="NLM_string-name">Mercurio, F.</span>; <span class="NLM_string-name">Sullivan, R.</span>; <span class="NLM_string-name">Torres, E.</span></span> <span> </span><span class="NLM_article-title">Compounds Targeting Proteins, Compositions, Methods, and Uses Thereof</span>. U.S. Patent <span class="NLM_patent">20180170948</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=K.+Chan&author=P.+E.+Erdman&author=L.+Fung&author=F.+Mercurio&author=R.+Sullivan&author=E.+Torres&title=Compounds+Targeting+Proteins%2C+Compositions%2C+Methods%2C+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.%26atitle%3DCompounds%2520Targeting%2520Proteins%252C%2520Compositions%252C%2520Methods%252C%2520and%2520Uses%2520Thereof%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erijman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernheim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shifman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peleg, Y.</span></span> <span> </span><span class="NLM_article-title">Transfer-PCR (TPCR): A Highway for DNA Cloning and Protein Engineering</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>175</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2011.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1016%2Fj.jsb.2011.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=21515384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslWnu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2011&pages=171-177&issue=2&author=A.+Erijmanauthor=A.+Dantesauthor=R.+Bernheimauthor=J.+M.+Shifmanauthor=Y.+Peleg&title=Transfer-PCR+%28TPCR%29%3A+A+Highway+for+DNA+Cloning+and+Protein+Engineering&doi=10.1016%2Fj.jsb.2011.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Transfer-PCR (TPCR): A highway for DNA cloning and protein engineering</span></div><div class="casAuthors">Erijman, Ariel; Dantes, Ada; Bernheim, Reut; Shifman, Julia M.; Peleg, Yoav</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">171-177</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNA cloning and protein engineering are basic methodologies employed for various applications in all life-science disciplines.  Manipulations of DNA however, could be a lengthy process that slows down subsequent expts.  To facilitate both DNA cloning and protein engineering, Transfer-PCR (TPCR) is presented, a novel approach that integrates in a single tube, PCR amplification of the target DNA from an origin vector and its subsequent integration into the destination vector.  TPCR can be applied for incorporation of DNA fragments into any desired position within a circular plasmid without the need for purifn. of the intermediate PCR product and without the use of any com. kit.  Several examples demonstrate the applicability of the TPCR platform for both DNA cloning and for multiple-site targeted mutagenesis.  In both cases, the TPCR reaction was most efficient within a narrow range of primer concns.  In mutagenesis, TPCR was primarily advantageous for generation of combinatorial libraries of targeted mutants but could be also applied to generation of variants with specific multiple mutations throughout the target gene.  Adaptation of the TPCR platform should facilitate, simplify and significantly reduce time and costs for diverse protein structure and functional studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojVuamZPNVCrVg90H21EOLACvtfcHk0lgnzu4cJfoBMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslWnu70%253D&md5=8235dc3a1b08a4c5eb2e5d6f1054aab1</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2011.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2011.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DErijman%26aufirst%3DA.%26aulast%3DDantes%26aufirst%3DA.%26aulast%3DBernheim%26aufirst%3DR.%26aulast%3DShifman%26aufirst%3DJ.%2BM.%26aulast%3DPeleg%26aufirst%3DY.%26atitle%3DTransfer-PCR%2520%2528TPCR%2529%253A%2520A%2520Highway%2520for%2520DNA%2520Cloning%2520and%2520Protein%2520Engineering%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D175%26issue%3D2%26spage%3D171%26epage%3D177%26doi%3D10.1016%2Fj.jsb.2011.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&issue=2&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgnzu4cJfoBMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&issue=2&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgnzu4cJfoBMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&issue=4&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgU2MQrujj0Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hexum, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tello-Aburto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struntz, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harned, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span> <span> </span><span class="NLM_article-title">Bicyclic Cyclohexenones as Inhibitors of NF-κB Signaling</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1021/ml300034a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300034a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=459-464&issue=6&author=J.+K.+Hexumauthor=R.+Tello-Aburtoauthor=N.+B.+Struntzauthor=A.+M.+Harnedauthor=D.+A.+Harki&title=Bicyclic+Cyclohexenones+as+Inhibitors+of+NF-%CE%BAB+Signaling&doi=10.1021%2Fml300034a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fml300034a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300034a%26sid%3Dliteratum%253Aachs%26aulast%3DHexum%26aufirst%3DJ.%2BK.%26aulast%3DTello-Aburto%26aufirst%3DR.%26aulast%3DStruntz%26aufirst%3DN.%2BB.%26aulast%3DHarned%26aufirst%3DA.%2BM.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26atitle%3DBicyclic%2520Cyclohexenones%2520as%2520Inhibitors%2520of%2520NF-%25CE%25BAB%2520Signaling%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26issue%3D6%26spage%3D459%26epage%3D464%26doi%3D10.1021%2Fml300034a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH7','PDB','6MH7'); return false;">PDB: 6MH7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH1','PDB','6MH1'); return false;">PDB: 6MH1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IOR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IOR','PDB','4IOR'); return false;">PDB: 4IOR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OUO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OUO','PDB','2OUO'); return false;">PDB: 2OUO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7F','PDB','4O7F'); return false;">PDB: 4O7F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7B','PDB','4O7B'); return false;">PDB: 4O7B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEM','PDB','2YEM'); return false;">PDB: 2YEM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEU','PDB','5UEU'); return false;">PDB: 5UEU</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i74"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01248">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_34714"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01248">10.1021/acs.jmedchem.8b01248</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular-string files for the final compounds synthesized (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_001.csv">CSV</a>)</p></li><li><p class="inline">Compound-purity analysis, fluorescence anisotropy, affinity determination, bromodomain ALPHAScreen, PrOF NMR titrations, binding interactions of IV and V, bromodomain-selectivity profiling, kinase-affinity determination, kinase-selectivity profiling, cellular toxicity, inhibition of p38-MAPK activity, reporter assay on p38-MAPK inhibitors, refinement statistics, NMR spectra of final compounds, and NMR spectra of novel intermediates (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_001.csv">jm8b01248_si_001.csv (1.7 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01248/suppl_file/jm8b01248_si_002.pdf">jm8b01248_si_002.pdf (9.66 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">BRD4(1) bound to compound <b>IV</b>, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH7">6MH7</a>; BRD4(1) bound to compound <b>V</b>, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MH1">6MH1</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-20%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01248%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01248" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b063ae853d4e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
