
<html lang="en"     class="pb-page"  data-request-id="642b00d4-1dd4-4a97-92f8-f5a7aa435827"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2021.64.issue-7;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c01810"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Rapid Optimization of the Metabolic Stability of a Human Immunodeficiency Virus Type-1 Capsid Inhibitor Using a Multistep Computational Workflow" /></meta><meta name="dc.Creator" content="Megan E.  Meuser" /></meta><meta name="dc.Creator" content="Poli Adi Narayana  Reddy" /></meta><meta name="dc.Creator" content="Alexej  Dick" /></meta><meta name="dc.Creator" content="Jean Marc  Maurancy" /></meta><meta name="dc.Creator" content="Joseph M.  Salvino" /></meta><meta name="dc.Creator" content="Simon  Cocklin" /></meta><meta name="dc.Description" content="Poor metabolic stability of the human immunodeficiency virus type-1 (HIV-1) capsid (CA) inhibitor PF-74 is a major concern in its development toward clinical use. To improve on the metabolic stabil..." /></meta><meta name="Description" content="Poor metabolic stability of the human immunodeficiency virus type-1 (HIV-1) capsid (CA) inhibitor PF-74 is a major concern in its development toward clinical use. To improve on the metabolic stabil..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 22, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01810" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01810" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01810" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01810" /></link>
        
    
    

<title>Rapid Optimization of the Metabolic Stability of a Human Immunodeficiency Virus Type-1 Capsid Inhibitor Using a Multistep Computational Workflow | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01810" /></meta><meta property="og:title" content="Rapid Optimization of the Metabolic Stability of a Human Immunodeficiency Virus Type-1 Capsid Inhibitor Using a Multistep Computational Workflow" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0051.jpeg" /></meta><meta property="og:description" content="Poor metabolic stability of the human immunodeficiency virus type-1 (HIV-1) capsid (CA) inhibitor PF-74 is a major concern in its development toward clinical use. To improve on the metabolic stability, we employed a novel multistep computationally driven workflow, which facilitated the rapid design of improved PF-74 analogs in an efficient manner. Using this workflow, we designed three compounds that interact specifically with the CA interprotomer pocket, inhibit HIV-1 infection, and demonstrate enantiomeric preference. Moreover, using this workflow, we were able to increase the metabolic stability 204-fold in comparison to PF-74 in only three analog steps. These results demonstrate our ability to rapidly design CA compounds using a novel computational workflow that has improved metabolic stability over the parental compound. This workflow can be further applied to the redesign of PF-74 and other promising inhibitors with a stability shortfall." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01810"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01810">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01810&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01810&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01810&amp;href=/doi/10.1021/acs.jmedchem.0c01810" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3747-3766</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01564" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01898" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Rapid Optimization of the Metabolic Stability of a Human Immunodeficiency Virus Type-1 Capsid Inhibitor Using a Multistep Computational Workflow</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Megan E. Meuser</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Megan E. Meuser</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Molecular Biology, Drexel University College of Medicine, Rooms 10307, 10309, and 10315, 245 North 15th Street, Philadelphia, Pennsylvania 19102, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Megan+E.++Meuser">Megan E. Meuser</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1544-606X" title="Orcid link">http://orcid.org/0000-0002-1544-606X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Poli Adi Narayana Reddy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Poli Adi Narayana Reddy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Wistar Cancer Center Molecular Screening, The Wistar Institute, 3601 Spruce Street, Philadelphia, Pennsylvania 19104, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Poli+Adi+Narayana++Reddy">Poli Adi Narayana Reddy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexej Dick</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexej Dick</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Molecular Biology, Drexel University College of Medicine, Rooms 10307, 10309, and 10315, 245 North 15th Street, Philadelphia, Pennsylvania 19102, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexej++Dick">Alexej Dick</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jean Marc Maurancy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jean Marc Maurancy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Molecular Biology, Drexel University College of Medicine, Rooms 10307, 10309, and 10315, 245 North 15th Street, Philadelphia, Pennsylvania 19102, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jean+Marc++Maurancy">Jean Marc Maurancy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph M. Salvino</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph M. Salvino</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Wistar Cancer Center Molecular Screening, The Wistar Institute, 3601 Spruce Street, Philadelphia, Pennsylvania 19104, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+M.++Salvino">Joseph M. Salvino</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Simon Cocklin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon Cocklin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Molecular Biology, Drexel University College of Medicine, Rooms 10307, 10309, and 10315, 245 North 15th Street, Philadelphia, Pennsylvania 19102, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#8bf8e8b8bfb2cbeff9eef3eee7a5eeeffe"><span class="__cf_email__" data-cfemail="b2c1d181868bf2d6c0d7cad7de9cd7d6c7">[email protected]</span></a>. Tel.: Office - 215-762-7234; Lab - 215-762-4979. Fax: 215-762-4452.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon++Cocklin">Simon Cocklin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9074-741X" title="Orcid link">http://orcid.org/0000-0002-9074-741X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01810&amp;href=/doi/10.1021%2Facs.jmedchem.0c01810" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3747–3766</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 22, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 October 2020</li><li><span class="item_label"><b>Published</b> online</span>22 March 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01810" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01810</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3747%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMegan%2BE.%2BMeuser%252C%2BPoli%2BAdi%2BNarayana%2BReddy%252C%2BAlexej%2BDick%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D7%26contentID%3Dacs.jmedchem.0c01810%26title%3DRapid%2BOptimization%2Bof%2Bthe%2BMetabolic%2BStability%2Bof%2Ba%2BHuman%2BImmunodeficiency%2BVirus%2BType-1%2BCapsid%2BInhibitor%2BUsing%2Ba%2BMultistep%2BComputational%2BWorkflow%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3766%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01810"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">731</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01810" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Rapid Optimization of the Metabolic Stability of a Human Immunodeficiency Virus Type-1 Capsid Inhibitor Using a Multistep Computational Workflow&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Megan&quot;,&quot;last_name&quot;:&quot;E. Meuser&quot;},{&quot;first_name&quot;:&quot;Poli&quot;,&quot;last_name&quot;:&quot;Adi Narayana Reddy&quot;},{&quot;first_name&quot;:&quot;Alexej&quot;,&quot;last_name&quot;:&quot;Dick&quot;},{&quot;first_name&quot;:&quot;Jean&quot;,&quot;last_name&quot;:&quot;Marc Maurancy&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;M. Salvino&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;Cocklin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;3747-3766&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01810&quot;},&quot;abstract&quot;:&quot;Poor metabolic stability of the human immunodeficiency virus type-1 (HIV-1) capsid (CA) inhibitor PF-74 is a major concern in its development toward clinical use. To improve on the metabolic stability, we employed a novel multistep computationally driven workflow, which facilitated the rapid design of improved PF-74 analogs in an efficient manner. Using this workflow, we designed three compounds that interact specifically with the CA interprotomer pocket, inhibit HIV-1 infection, and demonstrate enantiomeric preference. Moreover, using this workflow, we were able to increase the metabolic stability 204-fold in comparison to PF-74 in only three analog steps. These results demonstrate our ability to rapidly design CA compounds using a novel computational workflow that has improved metabolic stability over the parental compound. This workflow can be further applied to the redesign of PF-74 and other promising inhibitors with a stability shortfall.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01810&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01810" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01810&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01810" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01810&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01810" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01810&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01810&amp;href=/doi/10.1021/acs.jmedchem.0c01810" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01810" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01810" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01810%26sid%3Dliteratum%253Aachs%26pmid%3D33750123%26genre%3Darticle%26aulast%3DMeuser%26date%3D2021%26atitle%3DRapid%2BOptimization%2Bof%2Bthe%2BMetabolic%2BStability%2Bof%2Ba%2BHuman%2BImmunodeficiency%2BVirus%2BType-1%2BCapsid%2BInhibitor%2BUsing%2Ba%2BMultistep%2BComputational%2BWorkflow%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D7%26spage%3D3747%26epage%3D3766%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (6)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292119" title="Surface plasmon resonance">Surface plasmon resonance</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291675" title="Stability">Stability</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/jmcmar.2021.64.issue-7/20210408/jmcmar.2021.64.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0051.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0051.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Poor metabolic stability of the human immunodeficiency virus type-1 (HIV-1) capsid (CA) inhibitor <b>PF-74</b> is a major concern in its development toward clinical use. To improve on the metabolic stability, we employed a novel multistep computationally driven workflow, which facilitated the rapid design of improved <b>PF-74</b> analogs in an efficient manner. Using this workflow, we designed three compounds that interact specifically with the CA interprotomer pocket, inhibit HIV-1 infection, and demonstrate enantiomeric preference. Moreover, using this workflow, we were able to increase the metabolic stability 204-fold in comparison to <b>PF-74</b> in only three analog steps. These results demonstrate our ability to rapidly design CA compounds using a novel computational workflow that has improved metabolic stability over the parental compound. This workflow can be further applied to the redesign of <b>PF-74</b> and other promising inhibitors with a stability shortfall.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84142" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84142" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">With the emergence of combination antiretroviral therapy (cART), the mortality rate associated with human immunodeficiency virus type-1 (HIV-1) infection has decreased. However, due to the emergence of drug-resistant strains and toxicities associated with current therapies, the demand remains for new inhibitors of HIV-1 replication. Capsid (CA) protein, a broadly conserved protein that is essential for HIV-1 replication, playing roles in both the early and late stages of the viral life cycle, has emerged as a promising new therapeutic target.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> During the maturation process, the CA protomers rearrange to form a conical core surrounding the viral contents that are composed of approximately 250 CA hexamers and 12 CA pentamers.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Within the structure of assembled CA multimers exists a conserved interprotomer pocket that has been shown to interact with host-cell proteins such as cleavage and polyadenylation specificity factor-6 (CPSF6) and nucleoporin 153 (NUP153).<a onclick="showRef(event, 'ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9">(6−9)</a> Disrupting these interactions via mutagenesis can interfere with HIV-1 replication, indicating that this interprotomer pocket is a prime target for the design of inhibitors. In agreement with this postulation, this interprotomer pocket has been identified as the mature binding site of the CA-targeted small molecule inhibitor PF-3450074 (<b>PF-74</b>).<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> However, despite its attractive properties and mechanism of action, we were the first group to demonstrate that <b>PF-74</b> is extremely metabolically labile, which limits its clinical utility.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Since this demonstration, a number of other groups have taken it upon themselves to improve this deficit in metabolic stability of <b>PF-74</b>, often employing large analoging strategies and testing selected compounds for increases in metabolic stability.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18">(11−18)</a> Unfortunately, some of the most promising compounds derived from <b>PF-74</b> have complicated chemical syntheses and would be difficult to scale up for clinical production.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> As such, it would be advantageous if metabolic stability could be integrated into a design workflow for new synthetically tractable compounds, rather than a consequence of brute force analoging.</div><div class="NLM_p last">Therefore, in this brief article, we describe the use of a multistep computational workflow that can be used during the design of compounds to rapidly make analogs of a parental compound with improved metabolic stability. Using this design workflow, we created three enantioselective inhibitors (<b>CX15</b>, <b>CX16</b>, and <b>CX17</b>) that specifically bound to the CA hexamer and inhibited HIV-1 infection. Upon metabolic stability analysis, we demonstrated that the <i>S</i>-enantiomeric form of one of the compounds, in agreement with computational prediction, had a 204-fold increase in half-life (<i>t</i><sub>1/2</sub>) in comparison to <b>PF-74</b>. This finding, along with another study employing this approach for metabolic stability improvement,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> now solidifies the general utility of including steps to address metabolic stability during the computational design process.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66531" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66531" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Multistep Computational Design of <b>CX15</b>, <b>CX16</b>, and <b>CX17</b></h3><div class="NLM_p">Driven by the deficits in the current HIV-1 CA inhibitor class, our group has previously targeted the interprotomer pocket of the assembled hexameric HIV-1 CA to find novel inhibitors of HIV-1 replication. We have used three different approaches to do this: (1) high content pharmacophore-based screening,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> (2) chemistry-driven analoging around the <b>PF-74</b> scaffold,<a onclick="showRef(event, 'ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16">(13−16)</a> and (3) computationally guided bioisosteric replacement.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> These efforts have provided compounds that target the hexameric configuration of the HIV-1 CA but lacked either the requisite potency and/or metabolic stability to be viable leads.</div><div class="NLM_p">The incredible metabolic lability of <b>PF-74</b> is a significant issue with this inhibitor.<a onclick="showRef(event, 'ref10 ref12 ref14'); return false;" href="javascript:void(0);" class="ref ref10 ref12 ref14">(10,12,14)</a> However, we have recently described a novel computational workflow that allowed us to identify a unique HIV-1 entry inhibitor scaffold that has improved metabolic stability.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Therefore, we sought to see whether this workflow (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) could be applied to the rapid improvement of an HIV-1 CA inhibitor based upon <b>PF-74</b>. Briefly, we first performed bioisosteric replacement experiments on the bioactive conformation of <b>PF-74</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XFZ">4XFZ</a>) using Spark (Spark version 10.5.6, Cresset, Litlington, Cambridgeshire, UK; http://<a href="http://www.cresset-group.com/spark/" class="extLink">www.cresset-group.com/spark/</a>).<a onclick="showRef(event, 'ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23">(21−23)</a> The results of the bioisosteric replacement experiments were then imported into SeeSAR 10.0 (BioSolveIT GmbH, Germany) and assessed for predicted affinity using the HYdrogen bond and DEhydration (HYDE) energy scoring function.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Compounds with high predicted affinity were then imported into StarDrop 6.6 (Optibrium Ltd., UK) and assessed for absorption, distribution, metabolism, and excretion/pharmacokinetics (ADME/PK) properties, such as log <i>S</i> and log <i>P</i>, using the oral noncentral nervous system (oral non-CNS) drug scoring profile.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> We also used StarDrop to assess for metabolic vulnerability to the major cytochrome P450 (CYP) enzymes using the P450 module (Optibrium Ltd., UK).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> This analysis predicted that the major CYP isoform most likely to metabolize these compounds, and <b>PF-74</b>, was CYP isoform 3A4. One caveat with the metabolic liability prediction is that it is based upon the assumption that all compounds interact with the CYP isoform with the same affinity. Therefore, to further refine the metabolic prediction analysis, the compounds were then docked and assessed for their binding affinities to CYP isoform 3A4 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4D78">4D78</a>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Finally, compounds with better predicted drug-like metrics and inferred improved metabolic stability were then redocked into the HIV-1 CA hexamer using AutoDock 4.0,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> as implemented in DockingServer (<a href="http://www.dockingserver.com" class="extLink">www.dockingserver.com</a>)<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and were determined to have very similar binding poses to <b>PF-74</b>, indicating the robust choices of bioisosteric replacements (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,C). As a quality control, <b>PF-74</b> was docked to the HIV-1 CA protein in the interprotomer pocket to ensure that the derived prediction matched the crystal structure and to provide evidence that we could have confidence in our docking procedure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). From the final list of compounds that successfully passed through the above workflow, a compound in which a 1<i>H</i>-indazol-7-amine group replaced the methylindole headgroup of <b>PF-74</b> (designated <b>CX15</b>) and a compound in which a 1<i>H</i>-indazol group replaced the benzene tail group of <b>PF-74</b> (designated <b>CX16</b>) were chosen. Interestingly, a chimeric compound containing both of these replacements (designated <b>CX17</b>) was predicted to retain the potency of <b>PF-74</b> but with much improved metabolic stability. All three compounds were chosen for synthesis as racemates before the assessment of enantioselectivity (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> highlights the improvement of the drug-like properties of our compounds as compared to <b>PF-74</b>, as assessed in StarDrop (Optibrium Ltd., UK). Within this analysis, <b>CX15</b> and <b>CX16</b> showed a substantial improvement in the oral non-CNS scoring function as compared to <b>PF-74</b>, whereas <b>CX17</b> scored similarly with the drug-like function but displayed increased predicted aqueous solubility (log <i>S</i>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Computational workflow. Workflow chart highlighting the steps used in the computational design process of compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking of racemic <b>CX15</b>, <b>CX16</b>, and <b>CX17</b>. (A) Compounds chosen for analysis. Regions that were modified on <b>PF-74</b> during the bioisosteric replacement are highlighted in pink (<b>CX15</b>), yellow (<b>CX16</b>), and green (<b>CX17</b>). The chemical structures were drawn with ChemAxon software (Budapest, Hungary). (B) Crystal structure of HIV-1 CA with compounds, <b>PF-74</b> in blue (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XFZ">4XFZ</a>) and docked <b>CX15</b> in pink, <b>CX16</b> in yellow, and <b>CX17</b> in green in the interprotomer pocket (highlighted by a black box) of a CA hexamer (for simplicity, only an extracted dimer is shown). (C) Magnification of each docked compound, <b>CX15</b> in pink, <b>CX16</b> in yellow, <b>CX17</b> in green, and <b>PF-74</b> in gray in the predicted binding site aligned with the <b>PF-74</b>-bound crystal structure (blue).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Computational analysis of drug-like properties of <b>CX15</b>, <b>CX16</b>, and <b>CX17</b>. Oral non-CNS scoring profile of <b>CX15</b>, <b>CX16</b>, <b>CX17</b>, and <b>PF-74</b> calculated with StarDrop (Optibrium Ltd., UK).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Affinity and Potency Determination of Racemic <b>CX15</b>, <b>CX16</b>, and <b>CX17</b></h3><div class="NLM_p">After synthesis and prior to antiviral testing, we sought to establish that the new compounds retained the target specificity of the parental <b>PF-74</b> compound to the CA hexamer. We chose to demonstrate this via surface plasmon resonance (SPR). <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> shows the representative sensorgrams for each compound interacting with the disulfide-stabilized HIV-1<sub>NL4-3</sub> capsid hexamer,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> shows the kinetic parameters obtained after analysis. SPR analysis showed that each of the three novel compounds interacts with the immobilized disulfide-stabilized CA hexamer robustly and with a broadly similar kinetic signature to <b>PF-74</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). All four compounds have a very rapid on-rate but with slightly different off-rates, with <b>CX16</b> having the slowest off-rate (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These different off-rates lead to the different affinities observed for each compound (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Despite their different off-rates, all compounds had affinities in the nM range within 5-fold of <b>PF-74</b>, with <b>CX16</b> showing a marginally improved <i>K</i><sub>D</sub> over <b>PF-74</b> (43.9 and 95.4 nM, respectively). The SPR analysis results confirm that the newly designed compounds maintain the target specificity of the parental compound.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Representative sensorgrams depicting the interaction of compounds with the CA hexamer. SPR sensorgrams showing (A) <b>PF-74</b> (5 μM starting concentration with 1:4 serial dilutions), (B) <b>CX15</b> (5 μM starting concentration with 1:4 serial dilutions), (C) <b>CX16</b> (5 μM starting concentration with 1:4 serial dilutions), and (D) <b>CX17</b> (2 μM starting concentration with 1:4 serial dilutions) interacting with the immobilized disulfide-stabilized NL4-3 CA hexamer. Colored lines represent actual data collected from the dilution series, whereas black lines signify the fits to a 1:1 binding model. Interaction parameters derived from 5 sets of data are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Kinetics and Affinity of Racemic Compounds for the Disulfide-Stabilized HIV<sub>NL4-3</sub> CA Hexamer</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>a</sub> (M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>d</sub> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PF-74</b></td><td class="colsep0 rowsep0" align="char" char="±">3.07 ± 0.103 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char="±">2.93 ± 0.182 × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="left">95.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>CX15</b></td><td class="colsep0 rowsep0" align="char" char="±">5.62 ± 0.299 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char="±">5.62 ± 0.229 × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="left">189</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>CX16</b></td><td class="colsep0 rowsep0" align="char" char="±">2.75 ± 0.100 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char="±">1.21 ± 0.037 × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="left">43.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>CX17</b></td><td class="colsep0 rowsep0" align="char" char="±">2.20 ± 0.083 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char="±">1.06 ± 0 × 10<sup>–1</sup></td><td class="colsep0 rowsep0" align="left">494</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>D</sub> values were derived using the average <i>k</i><sub>a</sub> and <i>k</i><sub>d</sub> values.</p></div></div></div><div class="NLM_p">After confirmation that each of the new compounds retains their target specificity, we sought to determine their antiviral activities. Using a single-round infectious HIV-1 pseudoviral assay,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> the potencies of each compound for inhibition of the virus were tested in the early stages of the HIV-1 life cycle. This analysis revealed that two of the compounds inhibited at concentrations in agreement with both their predicted and experimentally determined affinities. However, <b>CX17</b> had an IC<sub>50</sub> approximately 5-fold higher than its predicted and experimentally determined affinity, perhaps hinting toward an enantiomer bias within the racemic mixture (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Simultaneous analysis of the toxicity of each compound showed half-maximal cytotoxic concentration values (CC<sub>50</sub>) all higher than that of the parental <b>PF-74</b> compound. Taken together, this demonstrates that our design strategy results in compounds that retain key properties of the parental compound, namely, target specificity and antiviral action but reduced toxicity.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Potency and Toxicity Values of Racemic <b>PF-74</b>, <b>CX15</b>, <b>CX16</b>, and <b>CX17</b> against HIV-1 Pseudoviruses</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char="±">CC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">therapeutic index (CC<sub>50</sub>/IC<sub>50</sub>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PF-74</b></td><td class="colsep0 rowsep0" align="char" char="±">55.5 ± 9.5</td><td class="colsep0 rowsep0" align="char" char="±">36.8 ± 16.4</td><td class="colsep0 rowsep0" align="left">664</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>CX15</b></td><td class="colsep0 rowsep0" align="char" char="±">97.2 ± 12.2</td><td class="colsep0 rowsep0" align="char" char="±">83.5 ± 19.5</td><td class="colsep0 rowsep0" align="left">859</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>CX16</b></td><td class="colsep0 rowsep0" align="char" char="±">38.3 ± 4.96</td><td class="colsep0 rowsep0" align="char" char="±">76.9 ± 9.5</td><td class="colsep0 rowsep0" align="left">2008</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>CX17</b></td><td class="colsep0 rowsep0" align="char" char="±">2420.0 ± 305.0</td><td class="colsep0 rowsep0" align="char" char="±">415.0 ± 207.0</td><td class="colsep0 rowsep0" align="left">171</td></tr></tbody></table></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Binding Site Mapping</h3><div class="NLM_p">As we have demonstrated antiviral activity and target specificity to the CA hexamer, we next wanted to verify that the new compounds shared the same interprotomer binding site as <b>PF-74</b> and to validate our docking hypotheses. With no complex structure in hand at this time, binding site verification and validation of contributing residues crucial for binding provide essential information for further optimization for our HIV-1 CA inhibitor class. To achieve this, we chose to perform mutagenesis of residues within the predicted binding pocket, guided by our docking models. When analyzing the 2D ligand-protein interaction network (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S1</a>) from our docking models, all compounds are predicted to make slightly different contacts. Therefore, we chose to prioritize the mutagenesis of residues that contribute the most to the overall predicted binding energies for each compound (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Table S1</a>). Specifically, we chose residues N53, L56, N57, M66, L69, K70, and I73, as the side chains of these residues have potential to form hydrogen bonds, cation-π, or hydrophobic interactions with our compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S1</a>). We chose to alter the selected residues to alanine as substitution with alanine eliminates side-chain interactions but with minimal effects upon main-chain conformation, hence preserving overall protein structure.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Percent maximal response of compounds <b>PF-74</b>, <b>CX15</b>, <b>CX16</b>, and <b>CX17</b> in the CA hexamer interprotomer pockets of the wild type as compared to mutants. Compounds were tested at a single concentration of 5 μM to ensure saturation of each surface. Responses were normalized to percent theoretical maximal binding response to allow for cross-comparison. The red dashed line represents the average of the wild-type (wt) binding responses. Error bars represent the standard deviation of at least three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The recombinant HIV-1 CA proteins with introduced alanine mutations at positions 53, 56, 57, 66, 69, 70, and 73 were overproduced and purified as previously described.<a onclick="showRef(event, 'ref10 ref20 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref10 ref20 ref30 ref31">(10,20,30,31)</a> However, the hexameric CA fractions were subsequently isolated using size-exclusion chromatography before SPR analysis.</div><div class="NLM_p">If our docking models are correct, then we expect that performing alanine mutagenesis would allow us to identify residues involved in the compound-protein interaction. As expected, we consistently saw a decrease in the percent binding upon mutation of our chosen residues to varying degrees. However, with mutants N57A, M66A, and K70A, we observed the most significant reduction in percent binding across all of the compounds as compared to the wild-type (wt) CA (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><div class="NLM_p">To understand the reasons for the pronounced effect of the N57A, M66A, and K70A CA mutants on the interactions with the compounds, we completed a computational assessment. The electrostatic complementarity (EC) of a ligand-receptor complex can provide important insights into the nature and strength of the complex.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Since electrostatic interactions are key contributors to the free energy of biological binding processes (enthalpic component Δ<i>G</i>), we chose to look at the EC map of each compound in the context of our models of the wt and mutant CA proteins. Analysis of the EC of these compounds within the binding sites from our docked models indicated that these mutants increased the electrostatic clash (or decreased the EC) for each of the compounds (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) as compared to the wt HIV-1 CA protein. This finding likely contributes to the reduction in binding observed with our compounds for these mutant CA proteins.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Electrostatic complementarity (EC) analysis of <b>PF-74</b>, <b>CX15</b>, <b>CX16</b>, and <b>CX17</b> within the wild type (wt) and HIV-1 capsid (NL4-3) mutants. The left panel shows the docking poses of <b>CX15</b>, <b>CX16</b>, <b>CX17</b> (colors scheme as in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), and the cocrystal structure of the HIV-1 NL4-3 capsid bound to <b>PF-74</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XFZ">4XFZ</a>). Green labeled residues are contributing hydrogen bonds (dashed green lines), and purple labeled residues contribute to cation-pi interactions to stabilize the complex (both if double-colored). The letter F next to the residue highlights that the adjacent HIV-1 capsid protomer contributes residues for complex stabilization. The letter O next to the residue highlights that a main-chain carbonyl is involved in hydrogen bonding. The right panel shows the EC surface map of each compound in the context of the wt or indicated in silico generated and energy minimized mutant. Each compound is shown from the front view (left) and in a 180° view (right) for the wt and mutant. Regions of the ligand surface with perfect electrostatic complementarity with the protein are colored green, while the areas with perfect electrostatic clash are colored red. Neutral EC areas are colored white.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As we have affinity, potency, and kinetic information for the interaction of these compounds with wt and mutant CA proteins, we sought to see any correlations that we could observe between these parameters and EC. All compounds displayed similar on-rate kinetics. However, the affinity (<i>K</i><sub>D</sub>) and the off-rates differed substantially between all four compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), with <b>CX16</b> displaying the slowest and <b>CX17</b> the fastest off-rate. In the context of the wt HIV-1 CA, we could observe a correlation trend between the EC scores and the affinities as well as the off-rates, showing higher EC scores for compounds with lower <i>K</i><sub>D</sub> and slower off-rates and lower EC scores for compounds with higher <i>K</i><sub>D</sub> or faster off-rates (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B with <b>CX16</b>, <b>PF-74</b>, <b>CX15</b>, and <b>CX17</b>, in this order). Interestingly, this EC/<i>K</i><sub>D</sub> and EC/off-rate correlation was disrupted by introducing the N57A and K70A mutations (data not shown), showing the importance of these residues in contributing electrostatic contacts for compound interaction. We have previously noted such correlations with our novel HIV-1 entry inhibitors.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The correlations between the EC, affinity (<i>K</i><sub>D</sub>), and off-rate of our compounds (including <b>PF-74</b>) to its target site can be utilized as powerful tools in the design and screening processes to help further our optimization efforts to improve upon the CX compounds.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Affinity and off-rates correlated with electrostatic complementarity. (A) Dissociation constant (<i>K</i><sub>D</sub>) of <b>CX16</b>, <b>PF-74</b>, <b>CX15</b>, and <b>CX17</b> (in this order) correlated with the EC. (B) Off-rates of <b>CX16</b>, <b>PF-74</b>, <b>CX15</b>, and <b>CX17</b> (in this order) correlated with the EC in the wt HIV-1 CA context. SCI, normalized surface integral of the complementarity score; Pearson R, Pearson correlation coefficient; Spearman rho rank, Spearman rank correlation coefficient.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Conservation Analysis of the CA Interprotomer Pocket</h3><div class="NLM_p">Since we observed that we could introduce mutations within the interprotomer pocket that reduce the binding and presumably the potency of the compounds, we next sought to gauge the likelihood of these mutations arising as compound resistance changes. First, we performed a conservation analysis of 5774 CA sequences from the Los Alamos Database (<a href="http://www.hiv.lanl.gov" class="extLink">www.hiv.lanl.gov</a>) to look at the probability of these residues to be naturally mutated. This analysis showed that the residues within the interprotomer pocket are highly conserved (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, mutated residues highlighted in blue). This extreme conservation indicates that these residues have critical structural and/or functional roles within the virus and that these contact points are not likely to mutate in isolation due to selective pressure during drug treatment. It is known that CA is a highly conserved and genetically fragile protein with about 70% random point mutations throughout the protein causing a replication-defective virus.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Specifically, previous extensive mutagenesis analysis of residues N57, M66, L69, and I73 revealed that mutation of these residues produced nonviable viruses.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Despite the genetic fragility of CA, a <b>PF-74</b>-resistant virus named 5Mut, which contains 5 mutations in the NTD (both in and outside of the <b>PF-74</b> binding site), has been isolated. This mutant was obtained by serial passaging the virus in the presence of increasing concentrations of a closely related analog of <b>PF-74</b>.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Notably, 5Mut contains a mutation at K70, which individually has been observed to decrease viral fitness; however, this fitness is restored upon the presence of the other compensatory mutations.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> This indicates that multiple mutations are required to both confer <b>PF-74</b> resistance and maintain viral fitness; therefore, we did not incorporate our single mutations into the viral backbone for infectivity assays as the resultant virus would likely be attenuated. However, this combined information again highlights the attractive nature of this interprotomer pocket for therapeutic targeting.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Conservation of residues within the CA interprotomer binding pocket. Probability of conservation of residues from 5774 sequences within the Los Alamos Database for the residues within the interprotomer pocket of (A) protomer F and (B) protomer G. Residue numbers correspond to their placement within Hxbc2. Blue letters indicate residues that were mutated during the binding site mapping. The size of the letter indicates the probability of conservation. Generated using the AnalyzeAlign tool on the Los Alamos Database (<a href="http://www.hiv.lanl.gov" class="extLink">www.hiv.lanl.gov</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Synthesis of Enantiomerically Pure Compounds</h3><div class="NLM_p last">Initially, each of these compounds was synthesized as racemic mixtures to ensure that the computational design workflow used led to the generation of active compounds that bound to the interprotomer pocket of the CA hexamer. When looking at the potencies derived for each compound, especially <b>CX17</b>, in comparison to their affinities, we noticed that the IC<sub>50</sub> values seemed to be slightly higher than expected. This, of course, could be due to several reasons; however, in the case of compounds with chirality, such a discrepancy may be attributed to enantioselectivity.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> When a chiral compound is biologically active as a racemate, one enantiomer can be more active than the other, with the more active enantiomer being called the eutomer and the less active or inactive compound being called the distomer.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> Enantioselectivity is an important but often underappreciated factor in the drug discovery process. Therefore, this led us to establish if our CX compounds (and <b>PF-74</b>) displayed enantioselectivity and if one enantiomeric form of our compounds was more active than the other. Therefore, we synthesized the enantiomerically pure forms of each compound and confirmed their structures by NMR and purity by LC–MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figures S2–S12</a>).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Potency and Affinity Determination of Enantiomerically Pure Compounds against HIV-1 Pseudoviruses</h3><div class="NLM_p">After synthesizing and isolating the pure enantiomers, we wanted to test their activities. We again began by testing their binding via SPR to the disulfide-stabilized CA hexamer to determine the affinity and specificity of each compound. To our surprise, there was little to no binding interaction observed with each of the <i>R</i>-enantiomers (<b><i>R</i>-PF-74</b>, <b><i>R</i>-CX15</b>, <b><i>R</i>-CX16</b>, and <b><i>R</i>-CX17</b>), while each of the <i>S</i>-enantiomers (<b><i>S</i>-PF-74</b>, <b><i>S</i>-CX15</b>, <b><i>S</i>-CX16</b>, and <b><i>S</i>-CX17</b>) showed robust binding with similar binding profiles to the racemic compounds (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). Moreover, by isolating the <i>S</i>-enantiomers, we saw an improvement in the affinity for each compound in comparison to the racemic mixtures (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Again, showing that we can improve upon the affinity of parental <b>PF-74</b>, our highest affinity compound <b><i>S</i>-CX16</b> showed over a 2-fold improvement in affinity over <b><i>S</i>-PF-74</b>.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Sensorgrams depicting the interaction of enantiomerically pure compounds with CA hexamers. SPR sensorgrams showing (A) <b><i>R</i>-PF-74</b> (5 μM starting concentration with 1:4 serial dilutions), (B) <b><i>S</i>-PF-74</b> (5 μM starting concentration with 1:4 serial dilutions), (C) <b><i>R</i>-CX15</b> (5 μM starting concentration with 1:4 serial dilutions), (D) <b><i>S</i>-CX15</b> (5 μM starting concentration with 1:4 serial dilutions), (E) <b><i>R</i>-CX16</b> (5 μM starting concentration with 1:4 serial dilutions), (F) <b><i>S</i>-CX16</b> (5 μM starting concentration with 1:4 serial dilutions), (G) <b><i>R</i>-CX17</b> (2 μM starting concentration with 1:4 serial dilutions), and (H) <b><i>S</i>-CX17</b> (2 μM starting concentration with 1:4 serial dilutions) interacting with the immobilized disulfide-stabilized NL4-3 CA hexamer. Colored lines represent actual data collected from the dilution series, whereas black lines signify the fits to a 1:1 binding model. Interaction parameters derived from 5 sets of data are given in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Kinetics and Affinity of Enantiomerically Pure Compounds for the Disulfide-Stabilized HIV<sub>NL4-3</sub> CA Hexamer</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>a</sub> (M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>d</sub> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub> (nM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-PF-74</b></td><td class="colsep0 rowsep0" align="char" char="±">4.75 ± 0.169 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char="±">2.92 ± 0.111 × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="left">61.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX15</b></td><td class="colsep0 rowsep0" align="char" char="±">4.76 ± 0.216 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char="±">6.60 ± 0.328 × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="left">139</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX16</b></td><td class="colsep0 rowsep0" align="char" char="±">4.37 ± 0.158 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char="±">1.09 ± 0.043 × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="left">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX17</b></td><td class="colsep0 rowsep0" align="char" char="±">2.86 ± 0.041 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="char" char="±">9.66 ± 0.145 × 10<sup>–2</sup></td><td class="colsep0 rowsep0" align="left">337</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>D</sub> values were derived using the average <i>k</i><sub>a</sub> and <i>k</i><sub>d</sub> values.</p></div></div></div><div class="NLM_p">Using the single-round infectious HIV-1 pseudotyped virus, we determined the potency for each of the enantiomerically pure compounds in the early stages of the HIV-1 life cycle. It was clear that each compound had an active form and an inactive one since the <i>R</i>-enantiomers, which showed no binding to the CA hexamer via SPR, also had no activity in the early-stage antiviral assay (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). To our delight, we also observed an improvement in the IC<sub>50</sub> values obtained from the active enantiomers in comparison to the racemic compounds, with <b><i>S</i>-CX16</b> showing improved potency in relation to the active <b><i>S</i>-PF-74</b> (20.6 nM vs 80 nM, respectively). This data shows that there is an enantiomeric preference for our compounds, with the <i>S</i>-enantiomers being the active eutomer and the <i>R</i>-enantiomers being the inactive distomer. By isolating the active eutomer of each compound, we were also able to improve upon the affinity and potency in relation to the starting racemic mixtures and in relation to the parental compound, <b>PF-74</b>. These values are also more in line with the predicted affinities from DockingServer.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> This resolves the inconsistencies observed between the IC<sub>50</sub> values and affinities of the racemic compounds, indicates that the docked models utilized the active enantiomer, and confirms the enantioselective properties seen for these compounds. Moreover, although slightly diminished, the antiviral potency of <b><i>S</i>-CX17</b> was changed on only the order of 3.5-fold, demonstrating that this computational workflow can be used to maintain the potency of the parental compound largely.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Potency and Toxicity of Enantiomerically Pure Compounds against HIV-1 Pseudoviruses<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">therapeutic index (CC<sub>50</sub>/IC<sub>50</sub>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>R</i>-PF-74</b></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-PF-74</b></td><td class="colsep0 rowsep0" align="left">80.0 ± 5.7</td><td class="colsep0 rowsep0" align="left">66.7 ± 6.8</td><td class="colsep0 rowsep0" align="left">834</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>R</i>-CX15</b></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX15</b></td><td class="colsep0 rowsep0" align="left">62.1 ± 8.2</td><td class="colsep0 rowsep0" align="left">101.0 ± 30.0</td><td class="colsep0 rowsep0" align="left">1626</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>R</i>-CX16</b></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX16</b></td><td class="colsep0 rowsep0" align="left">20.6 ± 1.8</td><td class="colsep0 rowsep0" align="left">46.9 ± 12.8</td><td class="colsep0 rowsep0" align="left">2281</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>R</i>-CX17</b></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX17</b></td><td class="colsep0 rowsep0" align="left">279.0 ± 52.6</td><td class="colsep0 rowsep0" align="left">102.0 ± 33.5</td><td class="colsep0 rowsep0" align="left">365</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">NA = not active.</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Metabolic Stability of Enantiomerically Pure CX Compounds versus <b>PF-74</b></h3><div class="NLM_p">Since a large pitfall of <b>PF-74</b> is its metabolic lability, one of our main goals while designing these compounds was to improve upon the metabolic half-life.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> After the characterization of the enantiomerically pure compounds, we sought to test their metabolic stability. Utilizing a human liver microsome assay, we determined the half-life of each compound.<a onclick="showRef(event, 'ref10 ref19'); return false;" href="javascript:void(0);" class="ref ref10 ref19">(10,19)</a> As we had predicted from the computational workflow, each of our novel compounds had improved metabolic stability over active <b><i>S</i>-PF-74</b>, with <b><i>S</i>-CX17</b> showing the most dramatic improvement with a 204-fold increase in its half-life (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Table S1</a>). Overall, this result shows that considering the metabolic vulnerability to the major CYP enzymes within the computational workflow can be a useful tool for multiple compound classes to consider the metabolic stability of newly designed compounds and improve upon this parameter rapidly in fewer analog steps.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Metabolic Stability of Active Enantiomerically Pure Compounds Derived from a Human Liver Microsome Assay</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">fold increase (over <b><i>S</i>-PF-74</b>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-PF-74</b></td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX15</b></td><td class="colsep0 rowsep0" align="char" char=".">15.4</td><td class="colsep0 rowsep0" align="left">30.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX16</b></td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">2.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b><i>S</i>-CX17</b></td><td class="colsep0 rowsep0" align="char" char=".">102.0</td><td class="colsep0 rowsep0" align="left">204</td></tr></tbody></table></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31246" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31246" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last"><b>PF-74</b> is a promising small molecule for biochemical studies that inhibits the proper function of the CA protein; however, <b>PF-74</b> has limitations, including its poor metabolic stability. Using our novel computational workflow, we were able to optimize this class of CA-targeting compounds and improve the potency 4-fold and the metabolic stability 204-fold in few analog steps and simple substitutions. Furthermore, we found new correlations between the EC, off-rate, and potency of these compounds. Implementing these correlations into our computational workflow will allow for further optimization of this class of compounds to bring them toward the realm of clinical utility. Moreover, implementing this workflow will also enable us to improve upon compounds from other inhibitor classes and compounds for different targets. In summary, this study shows that we can rapidly improve upon the affinity, potency, and metabolic stability of <b>PF-74</b> by designing synthetically tractable compounds without performing extensive analoging.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80545" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80545" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis of Compounds</h3><div class="NLM_p">The purity of all compounds is ≥95% as determined by LC–MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figures S2–S12</a>).</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Synthesis of <b>CX15</b></h4><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Preparation of <i>tert</i>-Butyl(1-(methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-yl)carbamate (Intermediate <b>1</b>)</h5><div class="NLM_p">At 0 °C, to a solution of (<i>tert</i>-butoxycarbonyl)phenylalanine (1.0 g, 3.8 mmol) in DMF (10 mL) were added <i>N</i>-methylaniline (444 mg, 4.2 mmol), HATU (1.7 g, 4.5 mmol), and DIPEA (974.0 mg, 7.5 mmol). After the reaction was stirred at RT for 1 h, the mixture was diluted with ethyl acetate (30 mL) and then washed with brine (10 mL × 3). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (PE/EA = 3:1, v/v) to give <i>tert</i>-butyl(1-(methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-yl)carbamate as a yellow oil (1.3 g, <i>Y</i> = 97.0%) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 355.19.</div><figure id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of <i>tert</i>-Butyl(1-(methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-yl)carbamate (Intermediate <b>1</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> Preparation of 2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide Hydrochloride (Intermediate <b>2</b>)</h5><div class="NLM_p">A mixture of <i>tert</i>-butyl(1-(methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-yl)carbamate (1.3 g, 3.7 mmol) in HCl/dioxane (4 M, 15 mL) was stirred at RT for 1 h. Then, the reaction solvents were removed under vacuum to give crude 2-amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropan-amide hydrochloride as a yellow oil (1.1 g) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), which was used in the next step without purification. LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 255.08.</div><figure id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of 2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide Hydrochloride (Intermediate <b>2</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Preparation of <i>N</i>,<i>N</i>-Dimethyl-1-(7-nitro-1<i>H</i>-indol-3-yl)methanamine (Intermediate <b>3</b>)</h5><div class="NLM_p">A mixture of <i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyldiaminomethane (1.6 g, 15.6 mol) dissolved in acetic acid (30 mL) was added dropwise to a solution of 7-nitro-1<i>H</i>-indole (2.3 g, 14.2 mol) in acetic acid (30 mL) over 60 min. After the reaction mixture was stirred at RT for 3.5 h, the reaction mixture was cooled to 0 °C and adjusted to pH = 11 with 20% aqueous sodium hydroxide. Then, the reaction mixture was extracted with CHCl<sub>3</sub> (3 × 300 mL). The combined organic extracts were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give crude <i>N</i>,<i>N</i>-dimethyl-1-(7-nitro-1<i>H</i>-indol-3-yl)methanamine as a yellow solid (3.1 g, <i>Y</i> = 99.7%) (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>), which was used in the next step without purification. LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 220.08; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.83 (br. s, 1H), 8.16 (d, <i>J</i> = 8.0 Hz, 1H), 8.10 (d, <i>J</i> = 8.0 Hz, 1H), 7.30 (d, <i>J</i> = 2.0 Hz, 1H), 7.21 (t, <i>J</i> = 8.0 Hz, 1H), 3.64 (s, 2H), 2.28 (s, 6H).</div><figure id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of <i>N</i>,<i>N</i>-Dimethyl-1-(7-nitro-1<i>H</i>-indol-3-yl)methanamine (Intermediate <b>3</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> Preparation of 2-(7-Nitro-1<i>H</i>-indol-3-yl)acetonitrile (Intermediate <b>4</b>)</h5><div class="NLM_p">A flask was charged with <i>N</i>,<i>N</i>-dimethyl-1-(7-nitro-1<i>H</i>-indol-3-yl)methanamine (100.0 mg, 0.46 mmol) and iodomethane (162.0 mg, 1.14 mmol) in DMF/H<sub>2</sub>O/THF (1:1:50, V/V; 5 mL). The reaction mixture was refluxed for 15 min until a white precipitate was formed, and then, potassium cyanide (149.0 mg, 2.28 mmol) was added to the reaction mixture. The resulted mixture was refluxed for another 2 h. After cooling to RT, the resulted reaction mixture was filtered, and the filtrate was concentrated under reduced pressure. The residue was triturated with MeOH to give 2-(7-nitro-1<i>H</i>-indol-3-yl)acetonitrile as a yellow solid (50.0 mg, <i>Y</i> = 54.5%) (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M-1)<sup>−</sup> = 199.93; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.93 (br. s, 1H), 8.13–8.18 (m, 2H), 7.55 (d, <i>J</i> = 2.0 Hz, 1H), 7.32 (t, <i>J</i> = 8.0 Hz, 1H), 4.17 (s, 2H).</div><figure id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Preparation of 2-(7-Nitro-1<i>H</i>-indol-3-yl)acetonitrile (Intermediate <b>4</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Preparation of 2-(7-Nitro-1<i>H</i>-indol-3-yl)acetic Acid (Intermediate <b>5</b>)</h5><div class="NLM_p">A mixture of 2-(7-nitro-1<i>H</i>-indol-3-yl)acetonitrile (50.0 mg, 0.25 mmol) in 3 N aqueous HCI (5 mL) was stirred at 95 °C for 72 h. After cooling to RT, the resulted mixture was adjusted to pH = 11 with 10 N aqueous NaOH solution and washed with EtOAc (10 mL). The aqueous layer was adjusted to pH = 2 with 1 N aqueous HCI solution and extracted with EtOAc (10 mL × 3). The combined organic extracts were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give crude 2-(7-nitro-1<i>H</i>-indol-3-yl)acetic acid as a yellow solid (55.0 mg) (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>), which was used in the next step without purification. LC–MS (ESI): <i>m</i>/<i>z</i> (M-1)<sup>−</sup> = 218.89; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.27 (br. s, 1H), 11.81 (s, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 1H), 8.10 (d, <i>J</i> = 8.0 Hz, 1H), 7.50 (d, <i>J</i> = 2.0 Hz, 1H), 7.30 (t, <i>J</i> = 8.0 Hz, 1H), 3.82 (s, 2H).</div><figure id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Preparation of 2-(7-Nitro-1<i>H</i>-indol-3-yl)acetic Acid (Intermediate <b>5</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Preparation of <i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (Intermediate <b>6</b>)</h5><div class="NLM_p">At 0 °C, to a mixture of 2-(7-nitro-1<i>H</i>-indol-3-yl)acetic acid (55.0 mg, 0.25 mmol) in DMF (5 mL) were added 2-amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (70.0 mg, 0.28 mmol), HATU (114.0 mg, 0.3 mmol), and DIPEA (65.0 mg, 0.5 mmol); then, the reaction mixture was stirred at RT for 1 h. Then, the reaction mixture was diluted with ethyl acetate (10 mL) and washed with brine (5 mL × 3). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (PE/EA = 3:1, v/v) to give <i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)-acetamido)-<i>N</i>,3-diphenylpropanamide as a yellow solid (45.0 mg, <i>Y</i> = 39.5%) (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 457.15; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.67 (s, 1H), 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.06 (d, <i>J</i> = 8.0 Hz, 1H), 7.82 (d, <i>J</i> = 8.0 Hz, 1H), 7.43–7.37 (m, 3H), 7.31 (d, <i>J</i> = 2.0 Hz, 1H), 7.24–7.22 (m, 2H), 7.15–7.11 (m, 4H), 6.78–6.77 (m, 2H), 4.47–4.40 (m, 1H), 3.59–3.51 (m, 2H), 3.15 (s, 3H), 2.87–2.83 (m, 1H), 2.69–2.63 (m, 1H).</div><figure id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Preparation of <i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (Intermediate <b>6</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Preparation of 2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>CX15</b>)</h5><div class="NLM_p">To a mixture of <i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (45.0 mg, 0.1 mmol) in EtOH/H<sub>2</sub>O (5 mL, 4:1, V/V) were added Fe powder (28.0 mg, 0.5 mmol) and NH<sub>4</sub>CI (27.0 mg, 0.5 mmol); then, the reaction mixture was stirred at 80 °C for 1 h. After cooling down to RT, the reaction mixture was filtered using Celite. The filtrate was concentrated under reduced pressure. The residue was purified by prep-HPLC (C18, 40–100% MeCN in H<sub>2</sub>O with 0.1% formic acid) to give 2-(2-(7-amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide as a white solid (11.0 mg, <i>Y</i> = 26.2%) (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.52 (d, <i>J</i> = 8.0 Hz, 1H), 7.25–7.26 (m, 3H), 6.97–7.08 (m, 3H), 6.93–6.88 (m, 2H), 6.79–6.71 (m, 2H), 6.65–6.63 (m, 2H), 6.45–6.43 (m, 1H), 4.52–4.57 (m, 1H), 3.47 (d, <i>J</i> = 5.2 Hz, 2H), 3.06 (s, 3H), 2.72–2.79 (m, 1H), 2.50–2.56 (m, 1H); LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 427.18 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S2</a>).</div><figure id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0017.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Preparation of 2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>CX15</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Synthesis of <b>CX16</b></h4><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Preparation of <i>tert</i>-Butyl (1-((1<i>H</i>-Indazol-5-yl)amino)-1-oxo-3-phenylpropan-2-yl) Carbamate</h5><div class="NLM_p">At room temperature, to a mixture of 1<i>H</i>-indazol-5-amine (1.5 g, 11.26 mmol) in DMF (20 mL) were added (<i>tert</i>-butoxycarbonyl)phenylalanine (3.29 g, 12.4 mmol), HATU (5.14 g, 13.5 mmol), and DIPEA (4.36 g, 33.75 mmol); then, the reaction mixture was stirred at RT for 3 h. Then, the reaction mixture was diluted with ethyl acetate and washed with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (PE/EA = 3:1, v/v) to give <i>tert</i>-butyl (1-((1<i>H</i>-indazol-5-yl)amino)-1-oxo-3-phenylpropan-2-yl)carbamate as a yellow solid (3.2 g, 75% yield) (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 381.25.</div><figure id="sch8" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0018.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Preparation of <i>tert</i>-Butyl (1-((1<i>H</i>-Indazol-5-yl)amino)-1-oxo-3-phenylpropan-2-yl) Carbamate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Preparation of <i>tert</i>-Butyl 5-(2-((<i>tert</i>-Butoxycarbonyl)amino)-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate</h5><div class="NLM_p">At room temperature, to a mixture of <i>tert</i>-butyl (1-((1<i>H</i>-indazol-5-yl)amino)-1-oxo-3-phenylpropan-2-yl)carbamate (1.7 g, 4.47 mmol) in THF(20 mL) were added Boc<sub>2</sub>O (2 g, 8.94 mmol) and DMAP (20 mg); then, the reaction mixture was stirred at RT for 2 h. Then, the reaction mixture was diluted with ethyl acetate and washed with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (PE/EA = 3:1, v/v) to give <i>tert</i>-butyl 5-(2-((<i>tert</i>-butoxycarbonyl)amino)-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate as a yellow solid (1.8 g, 84% yield) (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 479.38.</div><figure id="sch9" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0019.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Preparation of <i>tert</i>-Butyl 5-(2-((<i>tert</i>-Butoxycarbonyl)amino)-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Preparation of <i>tert</i>-Butyl 5-(2-((<i>tert</i>-Butoxycarbonyl)amino)-<i>N</i>-methyl-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate</h5><div class="NLM_p">At room temperature, to a mixture of <i>tert</i>-butyl 5-(2-((<i>tert</i>-butoxycarbonyl)amino)-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate (1.4 g, 2.92 mmol) in MeCN (20 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (2.84 g, 8.76 mmol) and MeI (0.62 g, 5.84 mmol); then, the reaction mixture was stirred at RT for 4 h. Then, the reaction mixture was diluted with ethyl acetate and washed with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (PE/EA = 3:1, v/v) to give <i>tert</i>-butyl 5-(2-((<i>tert</i>-butoxycarbonyl)amino)-<i>N</i>-methyl-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate as a yellow solid (1.15 g, 80% yield) (<a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 495.28.</div><figure id="sch10" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0020.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Preparation of <i>tert</i>-Butyl 5-(2-((<i>tert</i>-Butoxycarbonyl)amino)-<i>N</i>-methyl-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> Preparation of 2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide</h5><div class="NLM_p">To a mixture of <i>tert</i>-butyl 5-(2-((<i>tert</i>-butoxycarbonyl)amino)-<i>N</i>-methyl- 3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate (300 mg, 0.61 mmol) in DCM was added TFA; then, the reaction mixture was stirred at RT for 1 h. The reaction mixture was concentrated to be used in the next step without purification (<a class="ref internalNav" href="#sch11" aria-label="Scheme 11">Scheme 11</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 295.19.</div><figure id="sch11" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0021.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Preparation of 2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> Preparation of <i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b>CX16</b>)</h5><div class="NLM_p">At room temperature, to a mixture of 2-amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (83 mg, 0.28 mmol) in DMF (5 mL) were added 2-(2-methyl-1<i>H</i>-indol-3-yl)acetic acid (59 mg, 0.31 mmol), HATU (129.0 mg, 0.34 mmol), and DIPEA (110.0 mg, 0.85 mmol); then, the reaction mixture was stirred at RT for 2 h. Then, the reaction mixture was filtered using Celite. The filtrate was concentrated under reduced pressure. The residue was purified by prep-HPLC (C18, 40–100% MeCN in H<sub>2</sub>O with 0.1% formic acid) to give N-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide as a white solid (20 mg, 15% yield) (<a class="ref internalNav" href="#sch12" aria-label="Scheme 12">Scheme 12</a>). <sup>1</sup>H NMR (400 MHz, DMSO): δ 13.21 (s, 1H), 10.69 (s, 1H), 8.23 (d, <i>J</i> = 7.9 Hz, 1H), 8.01 (s, 1H), 7.55–7.39 (m, 2H), 7.28 (d, <i>J</i> = 7.8 Hz, 1H), 7.20–7.03 (m, 5H), 6.96–6.90 (m, 1H), 6.86–6.80 (m, 1H), 6.76 (d, <i>J</i> = 7.0 Hz, 2H), 4.40–4.30 (m, 1H), 3.45–3.38 (m, 2H), 3.16 (s, 3H), 2.88 (dd, <i>J</i> = 13.3, 4.7 Hz, 1H), 2.66 (dd, <i>J</i> = 13.3, 9.2 Hz, 1H), 2.22 (s, 3H); LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 466.29 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S3</a>).</div><figure id="sch12" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0022.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 12. Preparation of <i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b>CX16</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Synthesis of <b>CX17</b></h4><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> Preparation of <i>tert</i>-Butyl (1-((1<i>H</i>-Indazol-5-yl)amino)-1-oxo-3-phenylpropan-2-yl) Carbamate</h5><div class="NLM_p">At room temperature, a solution of 7-nitro-1<i>H</i>-indazole (3.0 g, 18.4 mmol), iodine (7 g, 27.6 mmol), and potassium hydroxide (2.0 g, 36.8 mmol) in DMF (30 mL) was stirred overnight. After quenching with 6 mol/L hydrochloric acid (7 mL), the mixture was diluted with ethyl acetate (30 mL) and then washed with brine (90 mL × 3). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give 3-iodo-7-nitro-1<i>H</i>-indazole as a yellow solid (5 g; yield, 94%) (<a class="ref internalNav" href="#sch13" aria-label="Scheme 13">Scheme 13</a>), which was used in the next step without purification. LC–MS (ESI): <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 289.86.</div><figure id="sch13" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0023.jpeg" id="gr22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 13. Preparation of <i>tert</i>-Butyl (1-((1<i>H</i>-Indazol-5-yl)amino)-1-oxo-3-phenylpropan-2-yl) Carbamate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> Preparation of 3-Iodo-1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazole</h5><div class="NLM_p">At room temperature, a solution of 3-iodo-7-nitro-1<i>H</i>-indazole (1.77 g, 6.12 mmol), potassium hydroxide (687 mg, 12.24 mmol), and PMBCl (1.44 g, 9.18 mmol) in acetone (25 mL) and water (5 mL) was stirred overnight under N<sub>2</sub>. After quenching with 6 mol/L hydrochloric acid to pH = 6–7, the mixture was diluted with ethyl acetate (30 mL) and then washed with brine (30 mL × 3). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (0–20% ethyl acetate in petroleum ether) to give 3-iodo-1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazole as a yellow solid (980 mg; yield, 39%) (<a class="ref internalNav" href="#sch14" aria-label="Scheme 14">Scheme 14</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 410.23.</div><figure id="sch14" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0024.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 14. Preparation of 3-Iodo-1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazole</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> Preparation of Dimethyl 2-(1-(4-Methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)malonate</h5><div class="NLM_p">At room temperature, a suspension of 3-iodo-1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazole (930 mg, 2.27 mmol), dimethyl malonate (450 mg, 3.41 mmol), 2-picolinic acid (56 mg, 0.45 mmol), copper(I) iodide (86 mg, 0.45 mmol), and cesium carbonate (1.48 g, 4.5 mmol) in dioxane (10 mL) was heated up to 100 °C and stirred for 5 h under N<sub>2</sub>. The suspension was concentrated under reduced pressure to give the crude product, which was purified by silica gel column chromatography (0–30% ethyl acetate in petroleum ether) to give 2-(1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)malonate as a yellow solid (688 mg; yield, 73%) (<a class="ref internalNav" href="#sch15" aria-label="Scheme 15">Scheme 15</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 414.21.</div><figure id="sch15" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0025.jpeg" id="gr24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 15. Preparation of Dimethyl 2-(1-(4-Methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)malonate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch15"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> Preparation of 2-(1-(4-Methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)acetic Acid</h5><div class="NLM_p">At room temperature, a suspension of 2-(1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)malonate (688 mg, 1.66 mmol) and potassium hydroxide (140 mg, 2.49 mmol) in ethanol (10 mL) was heated up to 80 °C and stirred overnight under N<sub>2</sub>. After cooling down to room temperature and being acidified by 1 mol/L hydrochloric acid to pH = 3–4, the solution was diluted with ethyl acetate (30 mL) and then washed with brine (30 mL × 3). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give product 2-(1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)acetic acid as a brown solid (530 mg; yield, 93%) (<a class="ref internalNav" href="#sch16" aria-label="Scheme 16">Scheme 16</a>), which was used in the next step without purification. LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 342.15.</div><figure id="sch16" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Scheme 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0026.jpeg" id="gr25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 16. Preparation of 2-(1-(4-Methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)acetic Acid</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch16"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> Preparation of 2-(7-Nitro-1<i>H</i>-indazol-3-yl)acetic Acid</h5><div class="NLM_p">At room temperature, a suspension of 2-(1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)acetic acid (500 mg, 1.46 mmol), trifluoroacetic acid (0.5 mL), and trifluoromethanesulfonic acid (0.5 mL) in 1,2-dichloroethane (5 mL) was heated up to reflux and stirred for 1 h under N<sub>2</sub>. After cooling down to room temperature, the solution was concentrated under reduced pressure to give product 2-(7-nitro-1<i>H</i>-indazol-3-yl)acetic acid as a black solid (400 mg) (<a class="ref internalNav" href="#sch17" aria-label="Scheme 17">Scheme 17</a>), which was used in the next step without purification. LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 222.09.</div><figure id="sch17" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0027.jpeg" id="gr26" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 17. Preparation of 2-(7-Nitro-1<i>H</i>-indazol-3-yl)acetic Acid</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch17"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> Preparation of 2-(7-Amino-1<i>H</i>-indazol-3-yl)acetic Acid</h5><div class="NLM_p">At room temperature, a suspension of 2-(7-nitro-1<i>H</i>-indazol-3-yl)acetic acid (400 mg, 2.27 mmol) and Pd(OH)<sub>2</sub>/C (100 mg) in methanol (5 mL) was stirred for 2 h under H<sub>2</sub>. After the reaction mixture was filtered using Celite, the filtrate was concentrated under reduced pressure to give the crude product, which was purified by prep-HPLC (C18, 40–100% acetonitrile in water with 0.1% formic acid) to afford 2-(7-amino-1<i>H</i>-indazol-3-yl)acetic acid as a white solid (97.0 mg; two-step yield, 35%) (<a class="ref internalNav" href="#sch18" aria-label="Scheme 18">Scheme 18</a>). LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 192.13. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.36 (s, 1H), 6.87 (d, <i>J</i> = 7.6 Hz, 1H), 6.80 (t, <i>J</i> = 7.6 Hz, 1H), 6.45 (d, <i>J</i> = 6.8 Hz, 1H), 5.32 (s, 2H), 3.80 (s, 2H).</div><figure id="sch18" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0028.jpeg" id="gr27" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 18. Preparation of 2-(7-Amino-1<i>H</i>-indazol-3-yl)acetic Acid</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch18"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> Preparation of 2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>CX17</b>)</h5><div class="NLM_p">At room temperature, a suspension of 2-(7-amino-1<i>H</i>-indazol-3-yl)acetic acid (50 mg, 0.26 mmol), 2-amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide, HATU, (198.8 mg, 0.52 mmol), and DIPEA (101 mg, 0.78 mmol) in DMF (5 mL) was stirred for 2 h. Then, the reaction mixture was diluted with ethyl acetate (20 mL) and washed with brine (10 mL × 3). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (0–30% ethyl acetate in petroleum ether) to give the crude product, which was purified by prep-HPLC (C18, 10–100% acetonitrile in water with 0.1% formic acid) to afford 2-(2-(7-amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide as a white solid (18 mg; yield, 15%) (<a class="ref internalNav" href="#sch19" aria-label="Scheme 19">Scheme 19</a>). <sup>1</sup>H NMR (400 MHz, DMSO): δ 13.19 (s, 1H), 12.27 (s, 1H), 8.49 (d, <i>J</i> = 7.9 Hz, 1H), 8.00 (s, 1H), 7.50 (d, <i>J</i> = 8.7 Hz, 1H), 7.41–7.27 (m, 1H), 7.19–7.03 (m, 4H), 6.80 (d, <i>J</i> = 6.8 Hz, 2H), 6.72–6.64 (m, 2H), 6.40 (dd, <i>J</i> = 5.9, 1.9 Hz, 1H), 5.21 (s, 2H), 4.44–4.36 (m, 1H), 3.66 (s, 2H), 3.16 (s, 3H), 2.92 (dd, 1H), 2.68 (dd, <i>J</i> = 13.3, 8.7 Hz, 1H); LC–MS (ESI): <i>m</i>/<i>z</i> (M + 1)<sup>+</sup> = 468.66 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S4</a>).</div><figure id="sch19" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0029.jpeg" id="gr28" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 19. Preparation of 2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>CX17</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch19"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Synthesis of <b><i>S</i>-PF-74</b></h4><div id="sec4_1_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> Preparation of (<i>S</i>)-<i>tert</i>-Butyl 1-(Methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-5</b>)</h5><div class="NLM_p">To a stirred solution of (<i>S</i>)-2-(<i>tert</i>-butoxycarbonylamino)-3-phenylpropanoic acid (1.0 g, 3.77 mmol) and <i>N</i>-methylaniline (0.40 g, 3.77 mmol) in 20 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (1.21 g, 9.42 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (3.12 g, 9.43 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford the title compound (1.13 g, 3.12 mmol, and 85%) (<a class="ref internalNav" href="#sch20" aria-label="Scheme 20">Scheme 20</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch20" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0030.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0030.jpeg" id="gr29" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 20. Preparation of (<i>S</i>)-<i>tert</i>-Butyl 1-(Methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-5</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.44–7.28 (m, 3H), 7.24–7.15 (m, 3H), 6.98–6.78 (m, 4H), 5.18 (d, <i>J</i> = 7.6 Hz, 1H), 4.53 (d, <i>J</i> = 7.1 Hz, 1H), 3.21 (s, 3H), 2.87 (dd, <i>J</i> = 13.1, 7.4 Hz, 1H), 2.68 (dd, <i>J</i> = 12.6, 6.7 Hz, 1H), 1.38 (s, 9H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>26</sub>N<sub>2</sub>NaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 377.2; found, 377.2.</div></div><div id="sec4_1_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> Preparation of (<i>S</i>)-2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>AP-5-9</b>)</h5><div class="NLM_p">To a stirred solution of (<i>S</i>)-<i>tert</i>-butyl 1-(methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (1.0 g, 2.82 mmol) in 15 mL of CH<sub>2</sub>Cl<sub>2</sub> at 0 °C was added 5 mL of TFA. The reaction mixture was then warmed to room temperature and stirred for 2 h. Completion of the reaction was confirmed by LC–MS. Volatiles were evaporated under reduced pressure to yield the crude product, which was codistilled with 20 mL of toluene to afford the title compound as a trifluoroacetic acid salt (1.02 g, 2.77 mmol, and 98%) as a viscous liquid (<a class="ref internalNav" href="#sch21" aria-label="Scheme 21">Scheme 21</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch21" data-index="30" class="article__inlineFigure"><h2 class="fig-label">Scheme 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0031.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0031.jpeg" id="gr30" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 21. Preparation of (<i>S</i>)-2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>AP-5-9</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0031.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch21"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.52–7.35 (m, 3H), 7.32–7.21 (m, 2H), 7.21–7.11 (m, 1H), 6.99 (dd, <i>J</i> = 46.4, 19.9 Hz, 2H), 6.86 (dd, <i>J</i> = 7.5, 1.7 Hz, 2H), 4.24 (t, <i>J</i> = 7.2 Hz, 1H), 3.24 (s, 2H), 3.04 (dd, <i>J</i> = 13.9, 6.3 Hz, 1H), 2.95–2.77 (m, 1H).</div><div class="NLM_p"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.45–7.36 (m, 3H), 7.31–7.21 (m, 3H), 7.20–7.13 (m, 1H), 6.97 (s, 2H), 6.86 (dd, <i>J</i> = 7.5, 1.7 Hz, 2H), 4.24 (t, <i>J</i> = 7.2 Hz, 3H), 3.24 (s, 8H), 3.04 (dd, <i>J</i> = 13.9, 6.3 Hz, 3H), 2.97–2.82 (m, 3H).</div><div class="NLM_p last">Mass <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>18</sub>N<sub>2</sub>NaO<sup>+</sup> [M + Na]<sup>+</sup>, 277.1; found, 277.1.</div></div><div id="sec4_1_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> Preparation of (<i>S</i>)-<i>N</i>-Methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b><i>S</i>-PF-74</b>)</h5><div class="NLM_p">To a stirred solution of 2-(2-methyl-1<i>H</i>-indol-3-yl)acetic acid (63 mg, 0.33 mmol) and (<i>S</i>)-2-amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide 2,2,2-trifluoroacetate (136 mg, 0.37 mmol) in 6 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (108 mg, 0.83 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (275 mg, 0.43 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 1 h at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford the title compound (113 mg, 0.26 mmol, and 80%) (<a class="ref internalNav" href="#sch22" aria-label="Scheme 22">Scheme 22</a>). The product was confirmed by <sup>1</sup>H NMR and MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S5</a>).</div><figure id="sch22" data-index="31" class="article__inlineFigure"><h2 class="fig-label">Scheme 22</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0032.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0032.jpeg" id="gr31" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 22. Preparation of (<i>S</i>)-<i>N</i>-Methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b><i>S</i>-PF-74</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0032.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch22"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.00 (s, 1H), 7.41–7.31 (m, 4H), 7.29 (d, <i>J</i> = 8.0 Hz, 1H), 7.18–7.11 (m, 2H), 7.11–7.03 (m, 3H), 6.93 (d, <i>J</i> = 5.5 Hz, 2H), 6.66 (d, <i>J</i> = 7.1 Hz, 2H), 6.18 (d, <i>J</i> = 8.1 Hz, 1H), 4.78 (dd, <i>J</i> = 15.2, 7.1 Hz, 1H), 3.65–3.51 (m, 2H), 3.16 (s, 3H), 2.72 (dd, <i>J</i> = 13.3, 6.9 Hz, 1H), 2.52 (dd, <i>J</i> = 13.3, 7.0 Hz, 1H), 2.31 (s, 3H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>27</sub>N<sub>3</sub>NaO<sub>2</sub><sup>+</sup> [M + Na]<sup>+</sup>, 448.2; found, 448.2.</div></div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Synthesis of <b><i>R</i>-PF-74</b></h4><div id="sec4_1_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> Preparation of (<i>R</i>)-<i>tert</i>-Butyl 1-(Methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-4</b>)</h5><div class="NLM_p">To a stirred solution of (<i>R</i>)-2-(<i>tert</i>-butoxycarbonylamino)-3-phenylpropanoic acid (1.0 g, 3.77 mmol) and <i>N</i>-methylaniline (0.4 g, 3.77 mmol) in 20 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (1.21 g, 9.42 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (3.1 g, 4.9 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford the title compound (1.20 g, 3.39 mmol, and 90%) (<a class="ref internalNav" href="#sch23" aria-label="Scheme 23">Scheme 23</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch23" data-index="32" class="article__inlineFigure"><h2 class="fig-label">Scheme 23</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0033.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0033.jpeg" id="gr32" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 23. Preparation of (<i>R</i>)-<i>tert</i>-Butyl 1-(Methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-4</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0033.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch23"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.44–7.27 (m, 3H), 7.24–7.15 (m, 3H), 6.97–6.76 (m, 4H), 5.18 (d, <i>J</i> = 7.4 Hz, 1H), 4.53 (d, <i>J</i> = 6.9 Hz, 1H), 3.21 (s, 3H), 2.87 (dd, <i>J</i> = 13.1, 7.3 Hz, 1H), 2.68 (dd, <i>J</i> = 12.5, 6.6 Hz, 1H), 1.37 (s, 9H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>26</sub>N<sub>2</sub>NaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 377.2; found, 377.2.</div></div><div id="sec4_1_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> Preparation of (<i>R</i>)-2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>AP-5-8</b>)</h5><div class="NLM_p">To a stirred solution of (<i>R</i>)-<i>tert</i>-butyl 1-(methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (1.1 g, 2.82 mmol) in 15 mL of CH<sub>2</sub>Cl<sub>2</sub> at 0 °C was added 5 mL of TFA. The reaction mixture was then warmed to room temperature and stirred for 2 h. Completion of the reaction was confirmed by LC–MS. Volatiles were evaporated under reduced pressure to yield the crude product, which was codistilled with 20 mL of toluene to afford the title compound as a trifluoroacetic acid salt (1.1 g, 2.80 mmol, and 97%) as a viscous liquid (<a class="ref internalNav" href="#sch24" aria-label="Scheme 24">Scheme 24</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch24" data-index="33" class="article__inlineFigure"><h2 class="fig-label">Scheme 24</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0034.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0034.jpeg" id="gr33" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 24. Preparation of (<i>R</i>)-2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>AP-5-8</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0034.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch24"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.95 (s, 2H), 7.44–7.35 (m, 3H), 7.29–7.22 (m, 3H), 7.19–7.13 (m, 1H), 6.98 (d, <i>J</i> = 12.4 Hz, 1H), 6.86 (dd, <i>J</i> = 7.6, 1.7 Hz, 2H), 4.25 (t, <i>J</i> = 7.2 Hz, 1H), 3.23 (s, 3H), 3.04 (dd, <i>J</i> = 13.9, 6.4 Hz, 1H), 2.98–2.84 (m, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>18</sub>N<sub>2</sub>NaO<sup>+</sup> [M + Na]<sup>+</sup>, 277.1; found, 277.1.</div></div><div id="sec4_1_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> Preparation of (<i>R</i>)-<i>N</i>-Methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b><i>R</i>-PF-74</b>)</h5><div class="NLM_p">To a stirred solution of 2-(2-methyl-1<i>H</i>-indol-3-yl)acetic acid (100 mg, 0.53 mmol) and (<i>R</i>)-2-amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide 2,2,2-trifluoroacetate (214 mg, 0.58 mmol) in 6 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (171 mg, 1.32 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (438 mg, 0.69 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford the title compound (184 mg, 0.43 mmol, and 82%) (<a class="ref internalNav" href="#sch25" aria-label="Scheme 25">Scheme 25</a>). The product was confirmed by <sup>1</sup>H NMR and MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S6</a>).</div><figure id="sch25" data-index="34" class="article__inlineFigure"><h2 class="fig-label">Scheme 25</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0035.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0035.jpeg" id="gr34" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 25. Preparation of (<i>R</i>)-<i>N</i>-Methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b><i>R</i>-PF-74</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0035.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch25"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.00 (s, 1H), 7.41–7.31 (m, 4H), 7.29 (d, <i>J</i> = 8.0 Hz, 1H), 7.18–7.11 (m, 2H), 7.11–7.03 (m, 3H), 6.93 (d, <i>J</i> = 5.7 Hz, 2H), 6.66 (d, <i>J</i> = 7.1 Hz, 2H), 6.18 (d, <i>J</i> = 8.3 Hz, 1H), 4.78 (dd, <i>J</i> = 15.2, 7.1 Hz, 1H), 3.67–3.46 (m, 2H), 3.16 (s, 3H), 2.72 (dd, <i>J</i> = 13.3, 6.9 Hz, 1H), 2.52 (dd, <i>J</i> = 13.3, 7.0 Hz, 1H), 2.31 (s, 3H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>27</sub>N<sub>3</sub>NaO<sub>2</sub><sup>+</sup> [M + Na]<sup>+</sup>, 448.2; found, 448.2.</div></div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Synthesis of <b><i>S</i>-CX15</b></h4><div id="sec4_1_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> Preparation of (<i>S</i>)-<i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b>AP-5-47</b>)</h5><div class="NLM_p">To a stirred solution of 2-(7-nitro-1<i>H</i>-indol-3-yl)acetic acid (100 mg, 0.5 mmol) and (<i>S</i>)-2-amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide 2,2,2-trifluoroacetate (184 mg, 0.55 mmol) in 20 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (193 mg, 1.5 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (413 mg, 0.65 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford AP-5-47 (171 mg, 0.37 mmol, and 75%) (<a class="ref internalNav" href="#sch26" aria-label="Scheme 26">Scheme 26</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch26" data-index="35" class="article__inlineFigure"><h2 class="fig-label">Scheme 26</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0036.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0036.jpeg" id="gr35" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 26. Preparation of (<i>S</i>)-<i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b>AP-5-47</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0036.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch26"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.85 (s, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 1H), 7.78 (d, <i>J</i> = 7.8 Hz, 1H), 7.42–7.31 (m, 3H), 7.21–7.12 (m, 2H), 7.08 (t, <i>J</i> = 7.2 Hz, 2H), 6.96 (t, <i>J</i> = 11.2 Hz, 2H), 6.72 (d, <i>J</i> = 7.0 Hz, 2H), 6.20 (d, <i>J</i> = 8.1 Hz, 1H), 4.83 (dd, <i>J</i> = 15.2, 7.3 Hz, 1H), 3.74–3.59 (m, 2H), 3.20 (s, 3H), 2.80 (dd, <i>J</i> = 13.5, 6.8 Hz, 1H), 2.80 (dd, <i>J</i> = 13.5, 6.8 Hz, 1H), 2.59 (dd, <i>J</i> = 13.5, 7.5 Hz, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub><sup>+</sup> [M + H]<sup>+</sup>, 457.2; found, 457.3.</div></div><div id="sec4_1_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> Preparation of (<i>S</i>)-2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b><i>S</i>-CX15</b>)</h5><div class="NLM_p">At room temperature, Pd/C (10%, 30 mg) was added to as stirred solution of (<i>S</i>)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (90 mg, 0.2 mmol) in EtOH (15 mL) and stirred under a Hydrogen atmosphere at room temperature for 16 h. Completion of the reaction was confirmed by LC–MS, the reaction was filtered over Celite, and the filter bed was washed with EtOH (20 mL). The combined organic portions were evaporated to afford the crude product, which was purified by flash column chromatography to afford the title compound (63 mg, 0.15 mmol, and 75%) (<a class="ref internalNav" href="#sch27" aria-label="Scheme 27">Scheme 27</a>). The product was confirmed by <sup>1</sup>H NMR and MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S7</a>).</div><figure id="sch27" data-index="36" class="article__inlineFigure"><h2 class="fig-label">Scheme 27</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0037.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0037.jpeg" id="gr36" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 27. Preparation of (<i>S</i>)-2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b><i>S</i>-CX15</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0037.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch27"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO): δ 10.35 (s, 1H), 8.19 (d, <i>J</i> = 7.7 Hz, 1H), 7.53–7.30 (m, 3H), 7.26–7.03 (m, 5H), 6.96 (s, 1H), 6.79 (d, <i>J</i> = 3.6 Hz, 2H), 6.68–6.54 (m, 2H), 6.38–6.06 (m, 1H), 4.93 (s, 2H), 4.44 (d, <i>J</i> = 4.4 Hz, 1H), 3.51–3.34 (m, 2H), 3.13 (s, 3H), 2.81 (dd, <i>J</i> = 13.3, 4.4 Hz, 1H), 2.71–2.55 (m, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>27</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>, 427.2; found, 427.3.</div></div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> Synthesis of <b><i>R</i>-CX15</b></h4><div id="sec4_1_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> Preparation of (<i>R</i>)-<i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b>AP-5-46</b>)</h5><div class="NLM_p">To a stirred solution of 2-(7-nitro-1<i>H</i>-indol-3-yl)acetic acid (100 mg, 0.45 mmol) and (<i>R</i>)-2-amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide 2,2,2-trifluoroacetate (184 mg, 0.5 mmol) in 20 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (175 mg, 1.36 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (376 mg, 1.36 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 1 h at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford 9 (144 mg, 0.32 mmol, and 70%) (<a class="ref internalNav" href="#sch28" aria-label="Scheme 28">Scheme 28</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch28" data-index="37" class="article__inlineFigure"><h2 class="fig-label">Scheme 28</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0038.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0038.jpeg" id="gr37" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 28. Preparation of (<i>R</i>)-<i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b>AP-5-46</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0038.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch28"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.85 (s, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 1H), 7.78 (d, <i>J</i> = 7.8 Hz, 1H), 7.43–7.28 (m, 3H), 7.21–7.12 (m, 2H), 7.08 (t, <i>J</i> = 7.2 Hz, 2H), 6.96 (t, <i>J</i> = 11.2 Hz, 2H), 6.72 (d, <i>J</i> = 7.0 Hz, 2H), 6.21 (d, <i>J</i> = 8.1 Hz, 1H), 4.83 (dd, <i>J</i> = 15.2, 7.3 Hz, 1H), 3.76–3.59 (m, 2H), 3.20 (s, 3H), 2.80 (dd, <i>J</i> = 13.4, 6.7 Hz, 1H), 2.59 (dd, <i>J</i> = 13.5, 7.4 Hz, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub><sup>+</sup> [M + H]<sup>+</sup>, 457.2; found, 457.3.</div></div><div id="sec4_1_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> Preparation of (<i>R</i>)-2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b><i>R</i>-CX15</b>)</h5><div class="NLM_p">To as stirred solution of (<i>R</i>)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (90 mg, 0.2 mmol) in EtOH (15 mL) at room temperature Pd/C (10%, 30 mg) was added and stirred under a hydrogen atmosphere at room temperature for 16 h. Completion of the reaction was confirmed by LC–MS, the reaction was filtered over Celite, and the filter bed was washed with EtOH (20 mL). The combined organic portions were evaporated to afford the crude product, which was purified by flash column chromatography to afford the title compound (60 mg, 0.14 mmol, and 70%) (<a class="ref internalNav" href="#sch29" aria-label="Scheme 29">Scheme 29</a>). The product was confirmed by <sup>1</sup>H NMR and MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S8</a>).</div><figure id="sch29" data-index="38" class="article__inlineFigure"><h2 class="fig-label">Scheme 29</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0039.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0039.jpeg" id="gr38" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 29. Preparation of (<i>R</i>)-2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b><i>R</i>-CX15</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0039.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch29"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO): δ 10.35 (s, 1H), 8.19 (d, <i>J</i> = 7.8 Hz, 1H), 7.52–7.26 (m, 3H), 7.17 (dd, <i>J</i> = 26.0, 5.1 Hz, 5H), 6.97 (s, 1H), 6.79 (d, <i>J</i> = 3.6 Hz, 2H), 6.65 (dd, <i>J</i> = 7.6, 5.4 Hz, 2H), 6.28 (dd, <i>J</i> = 8.2, 4.4 Hz, 1H), 4.92 (s, 2H), 4.55–4.31 (m, 1H), 3.49–3.34 (m, 2H), 3.13 (s, 3H), 2.81 (dd, <i>J</i> = 13.3, 4.5 Hz, 1H), 2.64 (dd, <i>J</i> = 13.3, 9.5 Hz, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>27</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>, 427.3; found, 427.3.</div></div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> Synthesis of <b><i>S</i>-CX16</b></h4><div id="sec4_1_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> Preparation of <i>N</i>-Methyl-1<i>H</i>-indazol-5-amine (<b>AP-5-69</b>)</h5><div class="NLM_p">To a stirred solution of <i>tert</i>-butyl 5-(2,2,2-trifluoro-<i>N</i>-methylacetamido)-1<i>H</i>-indazole-1-carboxylate (prepared as reported procedure <i>JMC</i>, 2016, 59, 3793<i>–</i>3807) (5.84 g, 17.01 mmol) in methanol (90 mL) and water (45 mL) at room temperature, potassium carbonate (11.75 g, 85.05 mmol) was added. The reaction was heated to 85 °C and stirred overnight with a condenser. The reaction was cooled down and neutralized with 1 N HCl. The aqueous layer was extracted with ethyl acetate (2 × 100 mL), washed with aq NaCl (2 × 100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The crude product was purified by flash column chromatography to afford the title compound as a white solid (1.5 g, 10.2 mmol, and 60%) (<a class="ref internalNav" href="#sch30" aria-label="Scheme 30">Scheme 30</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch30" data-index="39" class="article__inlineFigure"><h2 class="fig-label">Scheme 30</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0040.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0040.jpeg" id="gr39" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 30. Preparation of <i>N</i>-Methyl-1<i>H</i>-indazol-5-amine (<b>AP-5-69</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0040.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch30"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO): δ 12.63 (s, 1H), 7.74 (d, <i>J</i> = 15.2 Hz, 1H), 7.28 (t, <i>J</i> = 9.3 Hz, 1H), 6.82–6.74 (m, 1H), 6.63–6.50 (m, 1H), 5.38 (t, <i>J</i> = 9.8 Hz, 1H), 2.68 (d, <i>J</i> = 4.7 Hz, 3H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>8</sub>H<sub>10</sub>N<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>, 147.0; found, 147.0.</div></div><div id="sec4_1_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> Preparation of (<i>S</i>)-<i>tert</i>-Butyl 1-((1<i>H</i>-Indazol-5-yl)(methyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-93</b>)</h5><div class="NLM_p">To a stirred solution of <i>N</i>-methyl-1<i>H</i>-indazol-5-amine (600 mg, 4.08 mmol) and (<i>S</i>)-2-(<i>tert</i>-butoxycarbonylamino)-3-phenylpropanoic acid (1.08 g, 4.08 mmol) in 15 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (1.58 g, 12.24 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (3.11 g, 4.89 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 1 h at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford the title compound (1.05 g, 2.65 mmol, and 65%) (<a class="ref internalNav" href="#sch31" aria-label="Scheme 31">Scheme 31</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch31" data-index="40" class="article__inlineFigure"><h2 class="fig-label">Scheme 31</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0041.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0041.jpeg" id="gr40" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 31. Preparation of (<i>S</i>)-<i>tert</i>-Butyl 1-((1<i>H</i>-Indazol-5-yl)(methyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-93</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0041.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch31"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.41 (s, 1H), 8.13–7.73 (m, 1H), 7.51–7.33 (m, 1H), 7.31–7.09 (m, 5H), 7.01–6.71 (m, 3H), 5.15 (d, <i>J</i> = 57.7 Hz, 1H), 4.45 (d, <i>J</i> = 7.3 Hz, 1H), 3.22 (m, 3H), 2.91 (dt, <i>J</i> = 17.3, 9.4 Hz, 1H), 2.74 (dd, <i>J</i> = 12.7, 5.9 Hz, 1H), 1.39 (s, 9H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>27</sub>N<sub>4</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>, 395.2; found, 395.3.</div></div><div id="sec4_1_8_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> Preparation of (<i>S</i>)-2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>AP-5-99</b>)</h5><div class="NLM_p">To a stirred solution of (<i>S</i>)-<i>tert</i>-butyl 1-((1<i>H</i>-indazol-5-yl)(methyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (1.04 g, 2.64 mmol) in 16 mL of CH<sub>2</sub>Cl<sub>2</sub> at 0 °C was added 4 mL of TFA. The reaction mixture was then warmed to room temperature and stirred for 2 h. Completion of the reaction was confirmed by LC–MS. Volatiles were evaporated under reduced pressure to yield the crude product, which was codistilled with 20 mL of toluene to afford title compound as a TFA salt (1.07 mg, 2.64 mmol, and 100%) as a viscous liquid (<a class="ref internalNav" href="#sch32" aria-label="Scheme 32">Scheme 32</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch32" data-index="41" class="article__inlineFigure"><h2 class="fig-label">Scheme 32</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0042.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0042.jpeg" id="gr41" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 32. Preparation of (<i>S</i>)-2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>AP-5-99</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0042.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch32"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO): δ 13.31 (s, 1H), 8.30 (s, 2H), 8.17–7.93 (m, 1H), 7.70–7.45 (m, 1H), 7.47–7.05 (m, 4H), 6.94–6.74 (m, 2H), 3.18 (m, 3H), 3.06–2.91 (m, 1H), 2.88–2.73 (m, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>17</sub>H<sub>19</sub>N<sub>4</sub>O<sup>+</sup> [M + H]<sup>+</sup>, 295.2; found, 295.1.</div></div><div id="sec4_1_8_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> Preparation of (<i>S</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b><i>S</i>-CX16</b>)</h5><div class="NLM_p">To a stirred solution of 2-(2-methyl-1<i>H</i>-indol-3-yl)acetic acid (70 mg, 0.37 mmol) and (<i>S</i>)-2-amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide 2,2,2-trifluoroacetate (166 mg, 0.41 mmol) in 7 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (143 mg, 1.11 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (282 mg, 0.44 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by reverse phase HPLC to afford the title compound (69 mg, 0.15 mmol, and 40%) (<a class="ref internalNav" href="#sch33" aria-label="Scheme 33">Scheme 33</a>). The product was confirmed by <sup>1</sup>H NMR and MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S9</a>).</div><figure id="sch33" data-index="42" class="article__inlineFigure"><h2 class="fig-label">Scheme 33</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0043.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0043.jpeg" id="gr42" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 33. Preparation of (<i>S</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b><i>S</i>-CX16</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0043.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch33"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.87 (s, 1H), 8.33–8.09 (m, 1H), 8.05–7.76 (m, 1H), 7.43–7.30 (m, 1H), 7.31–7.23 (m, 3H), 7.20 (dd, <i>J</i> = 9.2, 5.6 Hz, 1H), 7.16–7.04 (m, 4H), 6.73 (t, <i>J</i> = 11.0 Hz, 2H), 6.51–6.18 (m, 1H), 4.78 (dt, <i>J</i> = 14.9, 7.4 Hz, 1H), 3.70–3.48 (m, 2H), 3.20 (s, 3H), 2.80 (dt, <i>J</i> = 23.6, 11.8 Hz, 1H), 2.62 (dd, <i>J</i> = 13.1, 6.5 Hz, 1H), 2.37–2.15 (m, 3H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>28</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>, 466.2; found, 466.3.</div></div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> Synthesis of <b><i>R</i>-CX16</b></h4><div id="sec4_1_9_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> Preparation of (<i>R</i>)-<i>tert</i>-Butyl 1-((1<i>H</i>-Indazol-5-yl)(methyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-72</b>)</h5><div class="NLM_p">To a stirred solution of (<i>R</i>)-2-(<i>tert</i>-butoxycarbonylamino)-3-phenylpropanoic acid (1.2 g, 4.52 mmol) and <i>N</i>-methyl-1<i>H</i>-indazol-5-amine (665 mg, 4.52 mmol) in 20 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (1.46 g, 11.3 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (3.74 g, 5.87 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford the title compound (0.74 g, 1.89 mmol, and 45%) (<a class="ref internalNav" href="#sch34" aria-label="Scheme 34">Scheme 34</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch34" data-index="43" class="article__inlineFigure"><h2 class="fig-label">Scheme 34</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0044.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0044.jpeg" id="gr43" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 34. Preparation of (<i>R</i>)-<i>tert</i>-Butyl 1-((1<i>H</i>-Indazol-5-yl)(methyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-72</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0044.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch34"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.59 (s, 1H), 8.04–7.87 (m, 1H), 7.43–7.29 (m, 2H), 7.29–7.16 (m, 4H), 6.94 (d, <i>J</i> = 6.6 Hz, 2H), 6.87–6.78 (m, 1H), 5.30 (t, <i>J</i> = 4.3 Hz, 1H), 4.47 (d, <i>J</i> = 7.1 Hz, 1H), 4.17–4.04 (m, 1H), 3.23 (s, 3H), 2.95 (dd, <i>J</i> = 12.9, 8.4 Hz, 1H), 2.89 (s, 1H), 2.76 (dd, <i>J</i> = 12.8, 6.0 Hz, 1H), 1.42 (d, <i>J</i> = 23.6 Hz, 8H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>26</sub>N<sub>4</sub>NaO<sub>3</sub><sup>+</sup> [M + Na]<sup>+</sup>, 417.2; found, 417.3.</div></div><div id="sec4_1_9_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> Preparation of (<i>R</i>)-2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>AP-5-73</b>)</h5><div class="NLM_p">To a stirred solution of (<i>R</i>)-<i>tert</i>-butyl 1-((1<i>H</i>-indazol-5-yl)(methyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (720 mg, 1.82 mmol) in 16 mL of CH<sub>2</sub>Cl<sub>2</sub> at 0 °C was added 4 mL of TFA. The reaction mixture was then warmed to room temperature and stirred for 2 h. Completion of the reaction was confirmed by LC–MS. Volatiles were evaporated under reduced pressure to yield the crude product, which was codistilled with 20 mL of toluene to afford the title compound as a TFA salt (745 mg, 1.79 mmol, and 98%) as a viscous liquid (<a class="ref internalNav" href="#sch35" aria-label="Scheme 35">Scheme 35</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch35" data-index="44" class="article__inlineFigure"><h2 class="fig-label">Scheme 35</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0045.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0045.jpeg" id="gr44" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 35. Preparation of (<i>R</i>)-2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>AP-5-73</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0045.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch35"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 8.18–7.79 (m, 1H), 7.74–7.08 (m, 5H), 6.90 (d, <i>J</i> = 7.2 Hz, 2H), 3.92 (s, 1H), 3.27 (s, 3H), 3.16–3.00 (m, 1H), 2.84 (dd, <i>J</i> = 13.2, 6.4 Hz, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>17</sub>H<sub>18</sub>N<sub>4</sub>NaO<sup>+</sup> [M + Na]<sup>+</sup>, 317.1; found, 317.1.</div></div><div id="sec4_1_9_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> Preparation of (<i>R</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b><i>R</i>-CX16</b>)</h5><div class="NLM_p">To a stirred solution of 2-(2-methyl-1<i>H</i>-indol-3-yl)acetic acid (70 mg, 0.37 mmol) and (<i>R</i>)-2-amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide 2,2,2-trifluoroacetate (151 mg, 0.37 mmol) in 7 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (143 mg, 1.11 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (282 mg, 0.44 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by reverse phase HPLC to afford the title compound (74 mg, 0.16 mmol, and 43%) (<a class="ref internalNav" href="#sch36" aria-label="Scheme 36">Scheme 36</a>). The product was confirmed by <sup>1</sup>H NMR and MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S10</a>).</div><figure id="sch36" data-index="45" class="article__inlineFigure"><h2 class="fig-label">Scheme 36</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0046.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0046.jpeg" id="gr45" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 36. Preparation of (<i>R</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b><i>R</i>-CX16</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0046.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch36"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.86 (s, 1H), 8.16 (s, 1H), 7.95 (s, 1H), 7.37 (t, <i>J</i> = 9.2 Hz, 1H), 7.31–7.23 (m, 3H), 7.20 (t, <i>J</i> = 7.4 Hz, 3H), 7.16–7.05 (m, 3H), 6.75 (d, <i>J</i> = 7.2 Hz, 2H), 6.36 (d, <i>J</i> = 8.4 Hz, 2H), 4.77 (dd, <i>J</i> = 14.8, 7.9 Hz, 2H), 3.74–3.52 (m, 2H), 3.20 (s, 3H), 2.81 (dd, <i>J</i> = 13.1, 7.9 Hz, 1H), 2.62 (dd, <i>J</i> = 13.1, 6.5 Hz, 1H), 2.29 (s, 3H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>28</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>, 466.2; found, 466.3.</div></div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> Synthesis of <b><i>S</i>-CX17</b></h4><div id="sec4_1_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> Preparation of (<i>S</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indazol-3-yl)acetamido)-3-phenylpropanamide (<b>102</b>)</h5><div class="NLM_p">To a stirred solution of 2-(7-nitro-1<i>H</i>-indazol-3-yl)acetic acid (100 mg, 0.45 mmol) and (<i>S</i>)-2-amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide 2,2,2-trifluoroacetate (203 mg, 0.49 mmol) in 6 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (175 mg, 1.36 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (374 mg, 0.58 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford the title compound (115 mg, 0.23 mmol, and 51%) (<a class="ref internalNav" href="#sch37" aria-label="Scheme 37">Scheme 37</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch37" data-index="46" class="article__inlineFigure"><h2 class="fig-label">Scheme 37</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0047.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0047.jpeg" id="gr46" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 37. Preparation of (<i>S</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indazol-3-yl)acetamido)-3-phenylpropanamide (<b>102</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0047.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch37"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO): δ 13.63 (s, 1H), 13.14 (d, <i>J</i> = 35.9 Hz, 1H), 8.72 (d, <i>J</i> = 7.9 Hz, 1H), 8.30 (dd, <i>J</i> = 7.8, 0.7 Hz, 1H), 8.03 (s, 1H), 7.97 (d, <i>J</i> = 7.7 Hz, 1H), 7.50 (t, <i>J</i> = 12.2 Hz, 1H), 7.28–7.18 (m, 1H), 7.18–7.01 (m, 4H), 6.87–6.66 (m, 2H), 4.53–4.34 (m, 1H), 3.85 (s, 2H), 3.23–3.11 (m, 3H), 3.02–2.85 (m, 1H), 2.70 (dd, <i>J</i> = 13.4, 9.2 Hz, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>24</sub>N7O<sub>4</sub><sup>+</sup> [M + H]<sup>+</sup>, 498.2; found, 498.2.</div></div><div id="sec4_1_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> Preparation of (<i>S</i>)-2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b><i>S</i>-CX17</b>)</h5><div class="NLM_p">To as stirred solution of (<i>S</i>)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indazol-3-yl)acetamido)-3-phenylpropanamide (110 mg, 0.22 mmol) in EtOH (10 mL) at room temperature Pd/C (10%, 30 mg) was added and stirred under a hydrogen atmosphere at room temperature for 16 h. Completion of the reaction was confirmed by LC–MS, the reaction was filtered over Celite, and the filter bed was washed with EtOH (20 mL). The combined organic portions were evaporated to afford the crude product, which was purified by flash column chromatography to afford the title compound (80 mg, 0.17 mmol, and 77.6%) (<a class="ref internalNav" href="#sch38" aria-label="Scheme 38">Scheme 38</a>). The product was confirmed by <sup>1</sup>H NMR and MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S11</a>).</div><figure id="sch38" data-index="47" class="article__inlineFigure"><h2 class="fig-label">Scheme 38</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0048.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0048.jpeg" id="gr47" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 38. Preparation of (<i>S</i>)-2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b><i>S</i>-CX17</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0048.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch38"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO): δ 13.14 (d, <i>J</i> = 36.9 Hz, 1H), 12.24 (s, 1H), 8.48 (d, <i>J</i> = 7.9 Hz, 1H), 8.03 (d, <i>J</i> = 24.3 Hz, 1H), 7.47 (dd, <i>J</i> = 21.3, 11.4 Hz, 1H), 7.25–6.96 (m, 4H), 6.78 (t, <i>J</i> = 13.5 Hz, 2H), 6.75–6.61 (m, 2H), 6.40 (dd, <i>J</i> = 5.8, 2.2 Hz, 1H), 5.20 (s, 2H), 4.51–4.32 (m, 1H), 3.66 (s, 2H), 3.15 (d, <i>J</i> = 5.2 Hz, 3H), 2.92 (dd, <i>J</i> = 13.4, 5.4 Hz, 1H), 2.78–2.62 (m, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>26</sub>N<sub>7</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>, 468.2; found, 468.2.</div></div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Synthesis of <b><i>R</i>-CX17</b></h4><div id="sec4_1_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> Preparation of (<i>R</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indazol-3-yl)acetamido)-3-phenylpropanamide (<b>AP-5-104</b>)</h5><div class="NLM_p">To a stirred solution of 2-(7-nitro-1<i>H</i>-indazol-3-yl)acetic acid (150 mg, 0.68 mmol) and (<i>R</i>)-2-amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide 2,2,2-trifluoroacetate (305 mg, 0.75 mmol) in 7 mL of dry CH<sub>2</sub>Cl<sub>2</sub> at 0 °C were added diisopropyl ethyl amine (263 mg, 2.04 mmol) and T3P 50% solution in CH<sub>2</sub>Cl<sub>2</sub> by weight (562 mg, 0.88 mmol) dropwise simultaneously. The reaction mixture was allowed to stir for 30 min at 0 °C. Completion of the reaction was confirmed by LC–MS. The reaction mixture was quenched with cold water, and the product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the crude product was purified by flash column chromatography to afford the title compound (237 mg, 0.47 mmol, and 70%) (<a class="ref internalNav" href="#sch39" aria-label="Scheme 39">Scheme 39</a>). The product was confirmed by <sup>1</sup>H NMR and MS.</div><figure id="sch39" data-index="48" class="article__inlineFigure"><h2 class="fig-label">Scheme 39</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0049.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0049.jpeg" id="gr48" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 39. Preparation of (<i>R</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indazol-3-yl)acetamido)-3-phenylpropanamide (<b>AP-5-104</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0049.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch39"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO): δ 13.63 (s, 1H), 13.19 (s, 1H), 8.72 (d, <i>J</i> = 8.0 Hz, 1H), 8.30 (d, <i>J</i> = 7.2 Hz, 1H), 8.03 (s, 1H), 7.97 (d, <i>J</i> = 7.5 Hz, 1H), 7.59–7.39 (m, 2H), 7.23 (t, <i>J</i> = 7.9 Hz, 1H), 7.19–7.04 (m, 3H), 6.80 (d, <i>J</i> = 6.6 Hz, 2H), 4.43 (d, <i>J</i> = 5.1 Hz, 1H), 3.84 (d, <i>J</i> = 12.0 Hz, 2H), 3.21–3.10 (m, 3H), 2.95 (dd, <i>J</i> = 13.3, 5.1 Hz, 1H), 2.70 (dd, <i>J</i> = 13.5, 9.3 Hz, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>24</sub>N<sub>7</sub>O<sub>4</sub><sup>+</sup> [M + H]<sup>+</sup>, 498.2; found, 498.3.</div></div><div id="sec4_1_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> Preparation of (<i>R</i>)-2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b><i>R</i>-CX17</b>)</h5><div class="NLM_p">To as stirred solution of (<i>R</i>)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indazol-3-yl)acetamido)-3-phenylpropanamide (160 mg, 0.32 mmol) in EtOH (10 mL) at room temperature Pd/C (10%, 40 mg) was added and stirred under a hydrogen atmosphere at room temperature for 16 h. Completion of the reaction was confirmed by LC–MS, the reaction was filtered over Celite, and the filter bed was washed with EtOH (20 mL). The combined organic portions were evaporated to afford the crude product, which was purified by flash column chromatography to afford the title compound (100 mg, 0.21 mmol, and 66.7%) (<a class="ref internalNav" href="#sch40" aria-label="Scheme 40">Scheme 40</a>). The product was confirmed by <sup>1</sup>H NMR and MS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Figure S12</a>).</div><figure id="sch40" data-index="49" class="article__inlineFigure"><h2 class="fig-label">Scheme 40</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0050.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0050.jpeg" id="gr49" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 40. Preparation of (<i>R</i>)-2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b><i>R</i>-CX17</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0050.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch40"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO): δ 13.18 (s, 1H), 12.24 (s, 1H), 8.48 (d, <i>J</i> = 7.9 Hz, 1H), 8.00 (s, 1H), 7.50 (d, <i>J</i> = 8.7 Hz, 1H), 7.35 (d, <i>J</i> = 39.0 Hz, 1H), 7.22–6.99 (m, 4H), 6.80 (d, <i>J</i> = 6.7 Hz, 2H), 6.74–6.63 (m, 2H), 6.46–6.36 (m, 1H), 5.20 (s, 2H), 4.49–4.32 (m, 1H), 3.66 (s, 2H), 3.22–3.09 (m, 4H), 2.92 (dd, <i>J</i> = 13.3, 5.4 Hz, 1H), 2.69 (dd, <i>J</i> = 13.3, 8.8 Hz, 1H). Mass <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>26</sub>N<sub>7</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>, 468.3; found, 468.2.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> Metabolic Stability Evaluation of <b>PF-74</b> in Human Liver Microsomes</h3><div class="NLM_p last">Metabolic stability analysis was performed in human liver microsomes as outlined in Lu et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and was performed by HD Biosciences (Shanghai, China).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> Cells</h3><div class="NLM_p last">Human embryonic kidney 293T cells (a gift from Dr. Irwin Chaiken, Drexel University, Philadelphia, PA) were cultured in Dulbecco’s modified Eagle’s medium (DMEM), 10% FBS, 100 U/mL penicillin, 100 μg/mL streptomycin, and 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine. Human astroglioma U87 cells stably expressing CD4/CCR5 or CD4/CXCR4 (obtained from Prof. Hongkui Deng, Peking University, and Prof. Dan Littman, New York University, USA, through the AIDS Research and Reference Reagent Program, Division of AIDS, NIAID, NIH)<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> were cultured in DMEM supplemented with 10% FBS, 100 U/mL penicillin, 100 μg/mL streptomycin, 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, 300 μg/mL G418 (Thermo Scientific, Waltham, MA), and 1 μg/mL puromycin (Thermo Scientific). Cells were incubated continuously, unless otherwise stated, at 37 °C in a humidified 5% CO<sub>2</sub>/95% air environment.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> Proteins</h3><div class="NLM_p last">IgG b12 anti HIV-1 gp120 was obtained through the NIH AIDS Reagent Program, Division of AIDS, NIAID, NIH (anti-HIV-1 gp120 monoclonal (IgG1 b12) from Dr. Dennis Burton and Carlos Barbas); p24 was produced in-house as previously described.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Briefly, a vector containing C-terminally His-tagged HIV-1<sub>NL4-3</sub> CA (a gift from Dr. Eric Barklis, Oregon Health and Science University, Portland, OR) was transformed into BL21-Codon Plus (DE3)-RIL Competent Cells (Agilent Technologies, Wilmington, DE) and grown in an autoinduction ZYP-5052 medium overnight with shaking (225 rpm) at 30 °C.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Bacterial cultures were spun down at 7000 rpm, and the supernatant was discarded. Cell pellets were resuspended in PBS and lysed via sonication. The resultant supernatant was clarified and immediately applied to a Talon cobalt resin affinity column (Clontech Laboratories, Mountain View, CA). Protein was eluted using 1X PBS with 250 mM imidazole. Purified CA-H6 monomers were dialyzed overnight into 20 mM Tris-HCl pH 8.0 at 4 °C, concentrated to 120 μM, flash-frozen in liquid nitrogen, aliquoted, and stored at −80 °C. The CA hexamer was generated by introducing mutations at the following sites: A14C, E45C, W184A, and M185A through site-directed mutagenesis (Stratagene). The CA hexamer construct was expressed and purified following the same protocol as described above. After purification, the CA-H6 hexamers were dialyzed into 200 mM ßME followed by sequential dialysis to slowly remove ßME to allow for hexamer assembly.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i113" class="anchor-spacer"></div><h3 class="article-section__title" id="_i113"> Production of Mutant CA Hexamers</h3><div class="NLM_p last">PCR was performed following the Platinum SuperFi DNA Polymerase protocol (Invitrogen) to introduce site-directed mutations. Primers for site-directed mutagenesis used can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">Table S3</a>. After PCR amplification, the samples were treated with 1 μL/μg DpnI for an hour and a half to ensure digestion. The digested amplicon and digested plasmid DNA were transformed, and mutations were confirmed via Sanger sequencing through Genewiz. Protein expression and purification were performed as described above. Briefly, mutant CA was transformed into Bl21 DE3 (RIL) (Agilent Technologies) and expressed in ZYP5052 overnight at 30 °C with shaking at 250 rpm. The cells were then sonicated, and the clarified lysate was purified via cobalt affinity chromatography. After sequential dialysis, the assembled CA hexamers underwent size-exclusion chromatography using a Superdex, 200 Increase 10/300 GL column to isolate pure hexamers.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> Production of Pseudotyped Viruses</h3><div class="NLM_p last">Single-round infectious envelope-pseudotyped luciferase-reporter viruses were produced by dual transfection of two vectors (3 μg of vector 1 and 4 μg of vector 2) in 6-well plated 293T cells (1 × 10<sup>6</sup> cells/well).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Vector 1 is an envelope-deficient HIV-1 pNL4-3-Luc+R-E plasmid that carries the luciferase-reporter gene.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Vector 2 is a plasmid expressing the HIV-1<sub>BG505</sub> gp160 Env.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Transfections of these vectors were carried out via calcium phosphate (ProFection Mammalian Transfection System, Promega, Madison, WI) for 5 h. Following the 5 h transfection incubation, the DNA-containing medium was removed, and cells were washed with DMEM and replenished with fresh culture media. Supernatants containing pseudoviruses were collected 72 h post-transfection, clarified, filtered, aliquoted, and stored at −80 °C.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i115" class="anchor-spacer"></div><h3 class="article-section__title" id="_i115"> ELISA-Based Quantification of p24 Content</h3><div class="NLM_p last">An ELISA plate was coated with 50 ng of mouse anti-p24 (Abcam, ab9071) overnight at 4 °C, blocked with 3% BSA for 2 h at room temperature, and washed with 0.5% Tween in PBS. Pseudoviral stocks were lysed with 0.1% Triton X-100 (Sigma-Aldrich, St. Louis, MO) at 37 °C for 1 h and added to the plate overnight at 4 °C. Simultaneously, the p24 protein (produced, purified, and validated as described above) was added for the generation of a standard curve. Following the overnight incubation, the plate was washed with PBST, and 1:5000 dilution of rabbit anti-p24 (Abcam, ab63913) was added for 2 h at room temperature. After washing the unbound rabbit anti-p24 off the plate with PBST, goat anti-rabbit-HRP at a 1:5000 dilution was added for 1 h at room temperature. The plate was then extensively washed with PBST. Subsequently, a solution of 0.4 mg/mL <i>o</i>-phenylenediamine in a phosphate–citrate buffer with sodium perborate (Sigma-Aldrich) was added and incubated in the dark for 30 min. Optical densities were then obtained at 450 nm in a Multiskan GO microplate spectrophotometer (Thermo Scientific).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> Single-Round Infection Assay</h3><div class="NLM_p last">The details of the single-round HIV-1 infection assay have been published previously.<a onclick="showRef(event, 'ref43 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref43 ref45 ref46">(43,45,46)</a> Briefly, U87.CD4.CCR5/CXCR4 (1.2 × 10<sup>4</sup> cells/well) target cells were seeded in 96-well luminometer-compatible tissue culture plates (Greiner Bio-One). After 24 h, compounds and DMSO (vehicle control for compounds, Sigma) were mixed with pseudotyped viruses (normalized to p24 content), and the mixture was added to the target cells and incubated for 48 h at 37 °C. Following this, the medium was removed from each well, and the cells were lysed by the addition of 50 μL/well luciferase lysis buffer (Promega) and one freeze–thaw cycle. A GloMax 96 microplate luminometer (Promega) was used to measure the luciferase activity of each well after the addition of 50 μL/well luciferase assay substrate (Promega).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i117" class="anchor-spacer"></div><h3 class="article-section__title" id="_i117"> Cellular Toxicity</h3><div class="NLM_p last">The viability of the U87.CD4.CCR5/CXCR4 cells was determined by the Cell Counting Kit-8 cell proliferation and cytotoxicity assay (Dojindo Molecular Technologies, Rockville, MD) as per the manufacturer’s instructions. U87.CD4.CCR5/CXCR4 cells were seeded at a density of 1.2 × 10<sup>4</sup> cells/well in 96-well tissue culture plates (Olympus plastics) 24 h before treatment. Cells were treated with compounds (1 mM–0.3 μM) or DMSO (Sigma) for 48 h at 37 °C. Subsequently, 10 μL of the CCK-8 solution was added to each well and allowed to incubate at 37 °C for 4 h before absorbance at 450 nm was measured in a Multiskan GO microplate spectrophotometer (Thermo Scientific). Untreated cells were used as a medium control, and 0.1% SDS-treated cells served as a positive control. The 50% cytotoxicity (CC<sub>50</sub>) value was defined as the concentration of the compounds that reduced the absorbance of treated cells by 50% as compared with control cells.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i118" class="anchor-spacer"></div><h3 class="article-section__title" id="_i118"> SPR Direct Interaction Analysis</h3><div class="NLM_p">All binding assays were performed on a ProteOn XPR36 SPR Protein Interaction Array System (Bio-Rad Laboratories, Hercules, CA). The instrument temperature was set at 25 °C for all kinetic analyses. ProteOn GLH sensor chips were preconditioned with two short pulses each (10 s) of 50 mM NaOH, 100 mM HCl, and 0.5% sodium dodecyl sulfide. Then, the system was equilibrated with PBST buffer (20 mM sodium phosphate, 150 mM NaCl, and 0.005% polysorbate 20, pH 7.4). The surface of a GLH sensor chip was activated with a 1:100 dilution of a 1:1 mixture of 1-ethyl-3-(3-dimethylaminopropyl) carbodiimide hydrochloride (0.2 M) and sulfo-<i>N</i>-hydroxysuccinimide (0.05 M). Immediately after chip activation, the HIV-1 NL4-3 capsid protein constructs, purified as described above, were prepared at a concentration of 100 μg/mL in 10 mM sodium acetate, pH 5.0, and injected across ligand flow channels for 5 min at a flow rate of 30 uL/min. Then, after unreacted protein had been washed out, excess active ester groups on the sensor surface were capped by a 5 min injection of 1 M ethanolamine HCl (pH 8.0) at a flow rate of 5 μL/min. A reference surface was similarly created by immobilizing a nonspecific protein (IgG b12 anti-HIV-1 gp120 was obtained through the NIH AIDS Reagent Program, Division of AIDS, NIAID, NIH; anti-HIV-1 gp120 monoclonal (IgG1 b12) from Dr. Dennis Burton and Carlos Barbas) and was used as a background to correct nonspecific binding. To prepare a compound for direct binding analysis, compound stock solutions, along with 100% DMSO and totaling 30 μL, were made to a final volume of 1 mL by addition of sample preparation buffer (PBS, pH 7.4). Preparation of analytes in this manner ensured that the concentration of DMSO was matched with that of running buffer with 3% DMSO. Serial dilutions were then prepared in the running buffer (PBS, 3% DMSO, and 0.005% polysorbate 20, pH 7.4) and injected at a flow rate of 100 μL/min, for a 1 min association phase followed by up to a 5 min dissociation phase using the “one-shot kinetics” capability of the Proteon instrument. Data were analyzed using ProteOn Manager Software version 3.0 (Bio-Rad). The responses from the reference flow cell were subtracted to account for the nonspecific binding and injection artifacts. Experimental data were fitted to a simple 1:1 binding model (where applied). The average kinetic parameters (association [<i>k</i><sub>a</sub>] and dissociation [<i>k</i><sub>d</sub>] rates) generated from 3–6 data sets were used to define the equilibrium dissociation constant (<i>K</i><sub>D</sub>).</div><div class="NLM_p last">Binding site analysis via SPR was conducted similarly to that in Kortagere et al.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Wild-type and mutant HIV-1 CA hexamer proteins were attached to the surface of a GLH sensor chip by standard amine chemistry as described above. Compounds were injected over these surfaces at a concentration of 50 μM at a flow rate of 100 μL/min, for a 1 min association phase, and the response at equilibrium was recorded. For comparison and to take into account minor differences in the ligand density of the mutant surfaces, responses were normalized to the theoretical maximum response (<i>R</i><sub>max</sub>) for a given surface, assuming a 1:1 interaction. The responses from at least 3 replicate injections over each CA surface were analyzed.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> Molecular Docking and Electrostatic Complementarity (EC) Analysis</h3><div class="NLM_p last">Electrostatic complementarity analysis was achieved using Flare 2.0.1 (Cresset, Litlington, Cambridgeshire, UK; <a href="http://www.cresset-group.com/forge/" class="extLink">http://www.cresset-group.com/forge/</a>) for the wt and in silico mutated HIV-1 NL4-3 capsid protein.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i120"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01810" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37048" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37048" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01810?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01810</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings, potency and toxicity values, and kinetic and affinity data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">PDBs of the docking models for each compound (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_002.pdb" class="ext-link">PDB</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_003.pdb" class="ext-link">PDB</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_004.pdb" class="ext-link">PDB</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Supplemental figures including the 2D interaction schemes and the LC–MS and <sup>1</sup>H NMR spectra of each synthesized compound as well as tables summarizing the predicted interaction energy for binding site residues, metabolic stability parameters, and primers used for mutagenesis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_001.csv">jm0c01810_si_001.csv (1.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_002.pdb">jm0c01810_si_002.pdb (265.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_003.pdb">jm0c01810_si_003.pdb (266.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_004.pdb">jm0c01810_si_004.pdb (266.27 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_005.pdb">jm0c01810_si_005.pdb (268.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf">jm0c01810_si_006.pdf (1.39 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01810" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38046" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38046" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon Cocklin</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry & Molecular Biology, Drexel University College of Medicine, Rooms 10307, 10309, and 10315, 245 North 15th Street, Philadelphia, Pennsylvania 19102, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9074-741X" title="Orcid link">http://orcid.org/0000-0002-9074-741X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b5c6d686818cf5d1c7d0cdd0d99bd0d1c0"><span class="__cf_email__" data-cfemail="6f1c0c5c5b562f0b1d0a170a03410a0b1a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Megan E. Meuser</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry & Molecular Biology, Drexel University College of Medicine, Rooms 10307, 10309, and 10315, 245 North 15th Street, Philadelphia, Pennsylvania 19102, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1544-606X" title="Orcid link">http://orcid.org/0000-0002-1544-606X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Poli Adi Narayana Reddy</span> - <span class="hlFld-Affiliation affiliation">The
Wistar Cancer Center Molecular Screening, The Wistar Institute, 3601 Spruce Street, Philadelphia, Pennsylvania 19104, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexej Dick</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry & Molecular Biology, Drexel University College of Medicine, Rooms 10307, 10309, and 10315, 245 North 15th Street, Philadelphia, Pennsylvania 19102, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jean Marc Maurancy</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry & Molecular Biology, Drexel University College of Medicine, Rooms 10307, 10309, and 10315, 245 North 15th Street, Philadelphia, Pennsylvania 19102, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph M. Salvino</span> - <span class="hlFld-Affiliation affiliation">The
Wistar Cancer Center Molecular Screening, The Wistar Institute, 3601 Spruce Street, Philadelphia, Pennsylvania 19104, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.E.M. and P.A.N.R. contributed equally. The manuscript was written through the contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i122">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41212" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41212" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by NIH/NIAID grant AI118415 (Cocklin, PI) and NIH/NIAID grant R01AI150491 (Cocklin, PI, Salvino, Co-I). M.E.M. is also partially supported by NIH training grant T32-MH079785. A.D. is partially funded by NS089435 (Cocklin, Co-I [Nonnemacher, PI]). We would like to thank Cresset (UK; <a href="http://www.cresset-group.com/" class="extLink">http://www.cresset-group.com/</a>) for access to their software via an academic license.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS</td></tr><tr><td class="NLM_term">HIV-1</td><td class="NLM_def"><p class="first last">human immunodeficiency virus type-1</p></td></tr><tr><td class="NLM_term">CA</td><td class="NLM_def"><p class="first last">capsid</p></td></tr><tr><td class="NLM_term">cART</td><td class="NLM_def"><p class="first last">combination antiretroviral therapy</p></td></tr><tr><td class="NLM_term">CPSF6</td><td class="NLM_def"><p class="first last">cleavage and polyadenylation specificity factor-6</p></td></tr><tr><td class="NLM_term">NUP153</td><td class="NLM_def"><p class="first last">nucleoporin 153</p></td></tr><tr><td class="NLM_term">PF-3450074</td><td class="NLM_def"><p class="first last"><b>PF-74</b></p></td></tr><tr><td class="NLM_term"><i>t</i><sub>1/2</sub></td><td class="NLM_def"><p class="first last">half-life</p></td></tr><tr><td class="NLM_term">HYDE</td><td class="NLM_def"><p class="first last">hydrogen bond and dehydration</p></td></tr><tr><td class="NLM_term">ADME/PK</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and excretion/pharmacokinetics</p></td></tr><tr><td class="NLM_term">oral non-CNS</td><td class="NLM_def"><p class="first last">oral noncentral nervous system</p></td></tr><tr><td class="NLM_term">CYP</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">CC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal cytotoxic concentration</p></td></tr><tr><td class="NLM_term">wt</td><td class="NLM_def"><p class="first last">wild-type</p></td></tr><tr><td class="NLM_term">EC</td><td class="NLM_def"><p class="first last">electrostatic complementarity</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i124">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44477" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44477" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hope, T. J.</span></span> <span> </span><span class="NLM_article-title">HIV-1 capsid: the multifaceted key player in HIV-1 infection</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1038/nrmicro3503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1038%2Fnrmicro3503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=26179359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyks7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=471-483&author=E.+M.+Campbellauthor=T.+J.+Hope&title=HIV-1+capsid%3A+the+multifaceted+key+player+in+HIV-1+infection&doi=10.1038%2Fnrmicro3503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 capsid: the multifaceted key player in HIV-1 infection</span></div><div class="casAuthors">Campbell, Edward M.; Hope, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">471-483</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In a mature, infectious HIV-1 virion, the viral genome is housed within a conical capsid core made from the viral capsid (CA) protein.  The CA protein and the structure into which it assembles facilitate virtually every step of infection through a series of interactions with multiple host cell factors.  This Review describes the understanding of the interactions between the viral capsid core and several cellular factors that enable efficient HIV-1 genome replication, timely core disassembly, nuclear import and the integration of the viral genome into the genome of the target cell.  The authors then discuss how elucidating these interactions can reveal new targets for therapeutic interactions against HIV-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr9ouJsQtk4LVg90H21EOLACvtfcHk0lhtDWPy6Cx8Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyks7%252FO&md5=fa2779bfe0322d9d1a9e709d18dc448f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro3503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro3503%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DE.%2BM.%26aulast%3DHope%26aufirst%3DT.%2BJ.%26atitle%3DHIV-1%2520capsid%253A%2520the%2520multifaceted%2520key%2520player%2520in%2520HIV-1%2520infection%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2015%26volume%3D13%26spage%3D471%26epage%3D483%26doi%3D10.1038%2Fnrmicro3503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundquist, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krausslich, H. G.</span></span> <span> </span><span class="NLM_article-title">HIV-1 assembly, budding, and maturation</span>. <i>Cold Spring Harb. Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">a006924</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a006924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1101%2Fcshperspect.a006924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=22762019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlemtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=a006924&author=W.+I.+Sundquistauthor=H.+G.+Krausslich&title=HIV-1+assembly%2C+budding%2C+and+maturation&doi=10.1101%2Fcshperspect.a006924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 assembly, budding, and maturation</span></div><div class="casAuthors">Sundquist, Wesley I.; Krausslich, Hans-Georg</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">a006924/1-a006924/24</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  A defining property of retroviruses is their ability to assemble into particles that can leave producer cells and spread infection to susceptible cells and hosts.  Virion morphogenesis can be divided into three stages: assembly, wherein the virion is created and essential components are packaged; budding, wherein the virion crosses the plasma membrane and obtains its lipid envelope; and maturation, wherein the virion changes structure and becomes infectious.  All of these stages are coordinated by the Gag polyprotein and its proteolytic maturation products, which function as the major structural proteins of the virus.  Here, we review our current understanding of the mechanisms of HIV-1 assembly, budding, and maturation, starting with a general overview and then providing detailed descriptions of each of the different stages of virion morphogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH1Quq8ytnBrVg90H21EOLACvtfcHk0lhtDWPy6Cx8Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlemtbs%253D&md5=91c7bf7383d85be14ef6f5200990f451</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a006924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a006924%26sid%3Dliteratum%253Aachs%26aulast%3DSundquist%26aufirst%3DW.%2BI.%26aulast%3DKrausslich%26aufirst%3DH.%2BG.%26atitle%3DHIV-1%2520assembly%252C%2520budding%252C%2520and%2520maturation%26jtitle%3DCold%2520Spring%2520Harb.%2520Perspect.%2520Med.%26date%3D2012%26volume%3D2%26spage%3Da006924%26doi%3D10.1101%2Fcshperspect.a006924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rihn, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loman, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakker, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gifford, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rixon, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bieniasz, P. D.</span></span> <span> </span><span class="NLM_article-title">Extreme genetic fragility of the HIV-1 capsid</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e1003461</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1003461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1003461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=23818857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFCntrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&author=S.+J.+Rihnauthor=S.+J.+Wilsonauthor=N.+J.+Lomanauthor=M.+Alimauthor=S.+E.+Bakkerauthor=D.+Bhellaauthor=R.+J.+Giffordauthor=F.+J.+Rixonauthor=P.+D.+Bieniasz&title=Extreme+genetic+fragility+of+the+HIV-1+capsid&doi=10.1371%2Fjournal.ppat.1003461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Extreme genetic fragility of the HIV-1 capsid</span></div><div class="casAuthors">Rihn, Suzannah J.; Wilson, Sam J.; Loman, Nick J.; Alim, Mudathir; Bakker, Saskia E.; Bhella, David; Gifford, Robert J.; Rixon, Frazer J.; Bieniasz, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1003461</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Genetic robustness, or fragility, is defined as the ability, or lack thereof, of a biol. entity to maintain function in the face of mutations.  Viruses that replicate via RNA intermediates exhibit high mutation rates, and robustness should be particularly advantageous to them.  The capsid (CA) domain of the HIV-1 Gag protein is under strong pressure to conserve functional roles in viral assembly, maturation, uncoating, and nuclear import.  However, CA is also under strong immunol. pressure to diversify.  Therefore, it would be particularly advantageous for CA to evolve genetic robustness.  To measure the genetic robustness of HIV-1 CA, the authors generated a library of single amino acid substitution mutants, encompassing almost half the residues in CA.  Strikingly, the authors found HIV-1 CA to be the most genetically fragile protein that has been analyzed using such an approach, with 70% of mutations yielding replication-defective viruses.  Although CA participates in several steps in HIV-1 replication, anal. of conditionally (temp. sensitive) and constitutively non-viable mutants revealed that the biol. basis for its genetic fragility was primarily the need to coordinate the accurate and efficient assembly of mature virions.  All mutations that exist in naturally occurring HIV-1 subtype B populations at a frequency >3%, and were also present in the mutant library, had fitness levels that were >40% of WT.  However, a substantial fraction of mutations with high fitness did not occur in natural populations, suggesting another form of selection pressure limiting variation in vivo.  Addnl., known protective CTL epitopes occurred preferentially in domains of the HIV-1 CA that were even more genetically fragile than HIV-1 CA as a whole.  The extreme genetic fragility of HIV-1 CA may be one reason why cell-mediated immune responses to Gag correlate with better prognosis in HIV-1 infection, and suggests that CA is a good target for therapy and vaccination strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMV98aKUfzWbVg90H21EOLACvtfcHk0lhheWsqooMO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCntrfL&md5=812c2768fb6972527b8dc7e6f5dbe94a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1003461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1003461%26sid%3Dliteratum%253Aachs%26aulast%3DRihn%26aufirst%3DS.%2BJ.%26aulast%3DWilson%26aufirst%3DS.%2BJ.%26aulast%3DLoman%26aufirst%3DN.%2BJ.%26aulast%3DAlim%26aufirst%3DM.%26aulast%3DBakker%26aufirst%3DS.%2BE.%26aulast%3DBhella%26aufirst%3DD.%26aulast%3DGifford%26aufirst%3DR.%2BJ.%26aulast%3DRixon%26aufirst%3DF.%2BJ.%26aulast%3DBieniasz%26aufirst%3DP.%2BD.%26atitle%3DExtreme%2520genetic%2520fragility%2520of%2520the%2520HIV-1%2520capsid%26jtitle%3DPLoS%2520Pathog.%26date%3D2013%26volume%3D9%26doi%3D10.1371%2Fjournal.ppat.1003461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunanan, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Structure, function, and interactions of the HIV-1 capsid protein</span>. <i>life</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">100</span>, <span class="refDoi"> DOI: 10.3390/life11020100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3390%2Flife11020100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=33572761" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=100&author=E.+Rossiauthor=M.+E.+Meuserauthor=C.+J.+Cunananauthor=S.+Cocklin&title=Structure%2C+function%2C+and+interactions+of+the+HIV-1+capsid+protein&doi=10.3390%2Flife11020100"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Flife11020100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Flife11020100%26sid%3Dliteratum%253Aachs%26aulast%3DRossi%26aufirst%3DE.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DCunanan%26aufirst%3DC.%2BJ.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DStructure%252C%2520function%252C%2520and%2520interactions%2520of%2520the%2520HIV-1%2520capsid%2520protein%26jtitle%3Dlife%26date%3D2021%26volume%3D11%26spage%3D100%26doi%3D10.3390%2Flife11020100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundquist, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finch, J. T.</span></span> <span> </span><span class="NLM_article-title">Image reconstructions of helical assemblies of the HIV-1 CA protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>407</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">413</span>, <span class="refDoi"> DOI: 10.1038/35030177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1038%2F35030177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=11014200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntF2ntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=407&publication_year=2000&pages=409-413&author=S.+Liauthor=C.+P.+Hillauthor=W.+I.+Sundquistauthor=J.+T.+Finch&title=Image+reconstructions+of+helical+assemblies+of+the+HIV-1+CA+protein&doi=10.1038%2F35030177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Image reconstructions of helical assemblies of the HIV-1 CA protein</span></div><div class="casAuthors">Li, Su; Hill, Christopher P.; Sundquist, Wesley I.; Finch, John T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">407</span>
        (<span class="NLM_cas:issue">6802</span>),
    <span class="NLM_cas:pages">409-413</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The type 1 human immunodeficiency virus (HIV-1) contains a conical capsid comprising ∼1500 CA proteins subunits, which organizes the viral RNA genome for uncoating and replication in a new host cell.  In vitro, CA spontaneously assemblies into helical tubes and cones that resemble authentic viral capsids.  Here we describe electron cryo-microscopy and image reconstructions of CA tubes from six different helical families.  In spite of their polymorphism, all tubes are composed of hexameric rings of CA arranged with approx. local p6 lattice symmetry.  Crystal structures of the two CA domains were 'docked' into the reconstructed d., which showed that the amino-terminal domains form the hexameric rings and the carboxy-terminal dimerization domains connect each ring to six neighbors.  We propose a mol. model for the HIV-1 capsid that follows the principles of a fullerene cone, in which the body of the cone is composed of curved hexagonal arrays of CA rings and the ends are closed by inclusion of 12 pentagonal 'defects'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT7d67nnEKwrVg90H21EOLACvtfcHk0lhheWsqooMO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntF2ntbk%253D&md5=2efab9ee5fd98a7517c6d2f1aa6450aa</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F35030177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35030177%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DHill%26aufirst%3DC.%2BP.%26aulast%3DSundquist%26aufirst%3DW.%2BI.%26aulast%3DFinch%26aufirst%3DJ.%2BT.%26atitle%3DImage%2520reconstructions%2520of%2520helical%2520assemblies%2520of%2520the%2520HIV-1%2520CA%2520protein%26jtitle%3DNature%26date%3D2000%26volume%3D407%26spage%3D409%26epage%3D413%26doi%3D10.1038%2F35030177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fricke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zadrozny, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedzicki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demeler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pornillos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganser-Pornillos, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Griffero, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeager, M.</span></span> <span> </span><span class="NLM_article-title">Structural basis of HIV-1 capsid recognition by PF74 and CPSF6</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">18625</span>– <span class="NLM_lpage">18630</span>, <span class="refDoi"> DOI: 10.1073/pnas.1419945112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1073%2Fpnas.1419945112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=25518861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=18625-18630&author=A.+Bhattacharyaauthor=S.+L.+Alamauthor=T.+Frickeauthor=K.+Zadroznyauthor=J.+Sedzickiauthor=A.+B.+Taylorauthor=B.+Demelerauthor=O.+Pornillosauthor=B.+K.+Ganser-Pornillosauthor=F.+Diaz-Grifferoauthor=D.+N.+Ivanovauthor=M.+Yeager&title=Structural+basis+of+HIV-1+capsid+recognition+by+PF74+and+CPSF6&doi=10.1073%2Fpnas.1419945112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of HIV-1 capsid recognition by PF74 and CPSF6</span></div><div class="casAuthors">Bhattacharya, Akash; Alam, Steven L.; Fricke, Thomas; Zadrozny, Kaneil; Sedzicki, Jaroslaw; Taylor, Alexander B.; Demeler, Borries; Pornillos, Owen; Ganser-Pornillos, Barbie K.; Diaz-Griffero, Felipe; Ivanov, Dmitri N.; Yeager, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">18625-18630</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Upon infection of susceptible cells by HIV-1, the conical capsid formed by ∼250 hexamers and 12 pentamers of the CA protein is delivered to the cytoplasm.  The capsid shields the RNA genome and proteins required for reverse transcription.  In addn., the surface of the capsid mediates numerous host-virus interactions, which either promote infection or enable viral restriction by innate immune responses.  In the intact capsid, there is an intermol. interface between the N-terminal domain (NTD) of one subunit and the C-terminal domain (CTD) of the adjacent subunit within the same hexameric ring.  The NTD-CTD interface is crit. for capsid assembly, both as an architectural element of the CA hexamer and pentamer and as a mechanistic element for generating lattice curvature.  Here we report biochem. expts. showing that PF-3450074 (PF74), a drug that inhibits HIV-1 infection, as well as host proteins cleavage and polyadenylation specific factor 6 (CPSF6) and nucleoporin 153 kDa (NUP153), bind to the CA hexamer with at least 10-fold higher affinities compared with nonassembled CA or isolated CA domains.  The crystal structure of PF74 in complex with the CA hexamer reveals that PF74 binds in a preformed pocket encompassing the NTD-CTD interface, suggesting that the principal inhibitory target of PF74 is the assembled capsid.  Likewise, CPSF6 binds in the same pocket.  Given that the NTD-CTD interface is a specific mol. signature of assembled hexamers in the capsid, binding of NUP153 at this site suggests that key features of capsid architecture remain intact upon delivery of the preintegration complex to the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpis5JJGK8jYLVg90H21EOLACvtfcHk0lhheWsqooMO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7%252FE&md5=9e7927ef280400df591c52ca51363b65</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1419945112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1419945112%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharya%26aufirst%3DA.%26aulast%3DAlam%26aufirst%3DS.%2BL.%26aulast%3DFricke%26aufirst%3DT.%26aulast%3DZadrozny%26aufirst%3DK.%26aulast%3DSedzicki%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DA.%2BB.%26aulast%3DDemeler%26aufirst%3DB.%26aulast%3DPornillos%26aufirst%3DO.%26aulast%3DGanser-Pornillos%26aufirst%3DB.%2BK.%26aulast%3DDiaz-Griffero%26aufirst%3DF.%26aulast%3DIvanov%26aufirst%3DD.%2BN.%26aulast%3DYeager%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520of%2520HIV-1%2520capsid%2520recognition%2520by%2520PF74%2520and%2520CPSF6%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D18625%26epage%3D18630%26doi%3D10.1073%2Fpnas.1419945112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwan, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halambage, U. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiken, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. C.</span></span> <span> </span><span class="NLM_article-title">Host cofactors and pharmacologic ligands share an essential interface in HIV-1 capsid that is lost upon disassembly</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1004459</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1004459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1004459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=25356722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyisbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=A.+J.+Priceauthor=D.+A.+Jacquesauthor=W.+A.+McEwanauthor=A.+J.+Fletcherauthor=S.+Essigauthor=J.+W.+Chinauthor=U.+D.+Halambageauthor=C.+Aikenauthor=L.+C.+James&title=Host+cofactors+and+pharmacologic+ligands+share+an+essential+interface+in+HIV-1+capsid+that+is+lost+upon+disassembly&doi=10.1371%2Fjournal.ppat.1004459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Host cofactors and pharmacologic ligands share an essential interface in HIV-1 capsid that is lost upon disassembly</span></div><div class="casAuthors">Price, Amanda J.; Jacques, David A.; McEwan, William A.; Fletcher, Adam J.; Essig, Sebastian; Chin, Jason W.; Halambage, Upul D.; Aiken, Christopher; James, Leo C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1004459/1-e1004459/17, 17 pp.</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The HIV-1 capsid is involved in all infectious steps from reverse transcription to integration site selection, and is the target of multiple host cell and pharmacol. ligands.  However, structural studies have been limited to capsid monomers (CA), and the mechanistic basis for how these ligands influence infection is not well understood.  Here we show that a multi-subunit interface formed exclusively within CA hexamers mediates binding to linear epitopes within cellular cofactors NUP153 and CPSF6, and is competed for by the antiretroviral compds. PF74 and BI-2.  Each ligand is anchored via a shared phenylalanine-glycine (FG) motif to a pocket within the N-terminal domain of one monomer, and all but BI-2 also make essential interactions across the N-terminal domain: C-terminal domain (NTD:CTD) interface to a second monomer.  Dissocn. of hexamer into CA monomers prevents high affinity interaction with CPSF6 and PF74, and abolishes binding to NUP153.  The second interface is conformationally dynamic, but binding of NUP153 or CPSF6 peptides is accommodated by only one conformation.  NUP153 and CPSF6 have overlapping binding sites, but each makes unique CA interactions that, when mutated selectively, perturb cofactor dependency.  These results reveal that multiple ligands share an overlapping interface in HIV-1 capsid that is lost upon viral disassembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqln0TIYBQnCrVg90H21EOLACvtfcHk0liuUM4w0oTksQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyisbbN&md5=9165cd26ee15021a47448579c32c45ca</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1004459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1004459%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DJacques%26aufirst%3DD.%2BA.%26aulast%3DMcEwan%26aufirst%3DW.%2BA.%26aulast%3DFletcher%26aufirst%3DA.%2BJ.%26aulast%3DEssig%26aufirst%3DS.%26aulast%3DChin%26aufirst%3DJ.%2BW.%26aulast%3DHalambage%26aufirst%3DU.%2BD.%26aulast%3DAiken%26aufirst%3DC.%26aulast%3DJames%26aufirst%3DL.%2BC.%26atitle%3DHost%2520cofactors%2520and%2520pharmacologic%2520ligands%2520share%2520an%2520essential%2520interface%2520in%2520HIV-1%2520capsid%2520that%2520is%2520lost%2520upon%2520disassembly%26jtitle%3DPLoS%2520Pathog.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.ppat.1004459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KewalRamani, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towers, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. C.</span></span> <span> </span><span class="NLM_article-title">CPSF6 defines a conserved capsid interface that modulates HIV-1 replication</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002896</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1002896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1002896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=22956906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlSjsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=A.+J.+Priceauthor=A.+J.+Fletcherauthor=T.+Schallerauthor=T.+Elliottauthor=K.+Leeauthor=V.+N.+KewalRamaniauthor=J.+W.+Chinauthor=G.+J.+Towersauthor=L.+C.+James&title=CPSF6+defines+a+conserved+capsid+interface+that+modulates+HIV-1+replication&doi=10.1371%2Fjournal.ppat.1002896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">CPSF6 defines a conserved capsid interface that modulates HIV-1 replication</span></div><div class="casAuthors">Price, Amanda J.; Fletcher, Adam J.; Schaller, Torsten; Elliott, Tom; Lee, Kyeong Eun; Kewal Ramani, Vineet N.; Chin, Jason W.; Towers, Greg J.; James, Leo C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e1002896</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The HIV-1 genome enters cells inside a shell comprised of capsid (CA) protein.  Variation in CA sequence alters HIV-1 infectivity and escape from host restriction factors.  However, apart from the Cyclophilin A-binding loop, CA has no known interfaces with which to interact with cellular cofactors.  Here we describe a novel protein-protein interface in the N-terminal domain of HIV-1 CA, detd. by X-ray crystallog., which mediates both viral restriction and host cofactor dependence.  The interface is highly conserved across lentiviruses and is accessible in the context of a hexameric lattice.  Mutation of the interface prevents binding to and restriction by CPSF6-358, a truncated cytosolic form of the RNA processing factor, cleavage and polyadenylation specific factor 6 (CPSF6).  Furthermore, mutations that prevent CPSF6 binding also relieve dependence on nuclear entry cofactors TNPO3 and RanBP2.  These results suggest that the HIV-1 capsid mediates direct host cofactor interactions to facilitate viral infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiWdPM8k1FVLVg90H21EOLACvtfcHk0liuUM4w0oTksQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlSjsrbE&md5=9d419609d09371e976ba85d5209255a3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1002896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1002896%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DFletcher%26aufirst%3DA.%2BJ.%26aulast%3DSchaller%26aufirst%3DT.%26aulast%3DElliott%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKewalRamani%26aufirst%3DV.%2BN.%26aulast%3DChin%26aufirst%3DJ.%2BW.%26aulast%3DTowers%26aufirst%3DG.%2BJ.%26aulast%3DJames%26aufirst%3DL.%2BC.%26atitle%3DCPSF6%2520defines%2520a%2520conserved%2520capsid%2520interface%2520that%2520modulates%2520HIV-1%2520replication%26jtitle%3DPLoS%2520Pathog.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.ppat.1002896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matreyek, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yücel, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelman, A.</span></span> <span> </span><span class="NLM_article-title">Nucleoporin NUP153 phenylalanine-glycine motifs engage a common binding pocket within the HIV-1 capsid protein to mediate lentiviral infectivity</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e1003693</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1003693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1003693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=24130490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFaltbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003693&author=K.+A.+Matreyekauthor=S.+S.+Y%C3%BCcelauthor=X.+Liauthor=A.+Engelman&title=Nucleoporin+NUP153+phenylalanine-glycine+motifs+engage+a+common+binding+pocket+within+the+HIV-1+capsid+protein+to+mediate+lentiviral+infectivity&doi=10.1371%2Fjournal.ppat.1003693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoporin NUP153 phenylalanine-glycine motifs engage a common binding pocket within the HIV-1 capsid protein to mediate lentiviral infectivity</span></div><div class="casAuthors">Matreyek, Kenneth A.; Yucel, Sara S.; Li, Xiang; Engelman, Alan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1003693/1-e1003693/21</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Lentiviruses can infect non-dividing cells, and various cellular transport proteins provide crucial functions for lentiviral nuclear entry and integration.  We previously showed that the viral capsid (CA) protein mediated the dependency on cellular nucleoporin (NUP) 153 during HIV-1 infection, and now demonstrate a direct interaction between the CA N-terminal domain and the phenylalanine-glycine (FG)-repeat enriched NUP153 C-terminal domain (NUP153C).  NUP153C fused to the effector domains of the rhesus Trim5α restriction factor (Trim-NUP153C) potently restricted HIV-1, providing an intracellular readout for the NUP153C-CA interaction during retroviral infection.  Primate lentiviruses and equine infectious anemia virus (EIAV) bound NUP153C under these conditions, results that correlated with direct binding between purified proteins in vitro.  These binding phenotypes moreover correlated with the requirement for endogenous NUP153 protein during virus infection.  Mutagenesis expts. concordantly identified NUP153C and CA residues important for binding and lentiviral infectivity.  Different FG motifs within NUP153C mediated binding to HIV-1 vs. EIAV capsids.  HIV-1 CA binding mapped to residues that line the common alpha helix 3/4 hydrophobic pocket that also mediates binding to the small mol. PF-3450074 (PF74) inhibitor and cleavage and polyadenylation specific factor 6 (CPSF6) protein, with Asn57 (Asp58 in EIAV) playing a particularly important role.  PF74 and CPSF6 accordingly each competed with NUP153C for binding to the HIV-1 CA pocket, and significantly higher concns. of PF74 were needed to inhibit HIV-1 infection in the face of Trim-NUP153C expression or NUP153 knockdown.  Correlation between CA mutant viral cell cycle and NUP153 dependencies moreover indicates that the NUP153C-CA interaction underlies the ability of HIV-1 to infect non-dividing cells.  Our results highlight similar mechanisms of binding for disparate host factors to the same region of HIV-1 CA during viral ingress.  We conclude that a subset of lentiviral CA proteins directly engage FG-motifs present on NUP153 to affect viral nuclear import.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0M6R0jl8aCbVg90H21EOLACvtfcHk0lgmSojGhoAGhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFaltbg%253D&md5=6691bf0c32a1a50eec96dcda2cf57b48</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1003693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1003693%26sid%3Dliteratum%253Aachs%26aulast%3DMatreyek%26aufirst%3DK.%2BA.%26aulast%3DY%25C3%25BCcel%26aufirst%3DS.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DEngelman%26aufirst%3DA.%26atitle%3DNucleoporin%2520NUP153%2520phenylalanine-glycine%2520motifs%2520engage%2520a%2520common%2520binding%2520pocket%2520within%2520the%2520HIV-1%2520capsid%2520protein%2520to%2520mediate%2520lentiviral%2520infectivity%26jtitle%3DPLoS%2520Pathog.%26date%3D2013%26volume%3D9%26spage%3De1003693%26doi%3D10.1371%2Fjournal.ppat.1003693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. P.</span>; <span class="NLM_string-name">Francis, A. C.</span>; <span class="NLM_string-name">Meuser, M. E.</span>; <span class="NLM_string-name">Mankowski, M.</span>; <span class="NLM_string-name">Ptak, R. G.</span>; <span class="NLM_string-name">Rashad, A. A.</span>; <span class="NLM_string-name">Melikyan, G. B.</span>; <span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Exploring Modifications of an HIV-1 Capsid Inhibitor: Design, Synthesis, and Mechanism of Action</span>.  <i>J. Drug Des. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">5</span> ().</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=30393786" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&author=J.+P.+Xu&author=A.+C.+Francis&author=M.+E.+Meuser&author=M.+Mankowski&author=R.+G.+Ptak&author=A.+A.+Rashad&author=G.+B.+Melikyan&author=S.+Cocklin&title=Exploring+Modifications+of+an+HIV-1+Capsid+Inhibitor%3A+Design%2C+Synthesis%2C+and+Mechanism+of+Action"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%2BP.%26atitle%3DExploring%2520Modifications%2520of%2520an%2520HIV-1%2520Capsid%2520Inhibitor%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Mechanism%2520of%2520Action%26jtitle%3DJ.%2520Drug%2520Des.%2520Res.%26date%3D2018%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casey, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernekar, S. K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahani, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hachiya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tedbury, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Chemical profiling of HIV-1 capsid-targeting antiviral PF74</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>200</i></span>,  <span class="NLM_fpage">112427</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.ejmech.2020.112427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=32438252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsl2mt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=200&publication_year=2020&pages=112427&author=L.+Wangauthor=M.+C.+Caseyauthor=S.+K.+V.+Vernekarauthor=H.+T.+Doauthor=R.+L.+Sahaniauthor=K.+A.+Kirbyauthor=H.+Duauthor=A.+Hachiyaauthor=H.+Zhangauthor=P.+R.+Tedburyauthor=J.+Xieauthor=S.+G.+Sarafianosauthor=Z.+Wang&title=Chemical+profiling+of+HIV-1+capsid-targeting+antiviral+PF74&doi=10.1016%2Fj.ejmech.2020.112427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical profiling of HIV-1 capsid-targeting antiviral PF74</span></div><div class="casAuthors">Wang, Lei; Casey, Mary C.; Vernekar, Sanjeev Kumar V.; Do, Ha T.; Sahani, Rajkumar Lalji; Kirby, Karen A.; Du, Haijuan; Hachiya, Atsuko; Zhang, Huanchun; Tedbury, Philip R.; Xie, Jiashu; Sarafianos, Stefan G.; Wang, Zhengqiang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">200</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112427</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The capsid protein (CA) of HIV-1 plays essential roles in multiple steps of the viral replication cycle by assembling into functional capsid core, controlling the kinetics of uncoating and nuclear entry, and interacting with various host factors.  Targeting CA represents an attractive yet underexplored antiviral approach.  Of all known CA-targeting small mol. chemotypes, the peptidomimetic PF74, I, is particularly interesting because it binds to the same pocket used by a few important host factors, resulting in highly desirable antiviral phenotypes.  However, further development of I entails understanding its pharmacophore and mitigating its poor metabolic stability.  We report herein the design, synthesis, and evaluation of a large no. of I analogs aiming to provide a comprehensive chem. profiling of I and advance the understanding on its detailed binding mechanism and pharmacophore.  The analogs, contg. structural variations mainly in the aniline domain and/or the indole domain, were assayed for their effect on stability of CA hexamers, antiviral activity, and cytotoxicity.  Selected analogs were also tested for metabolic stability in liver microsomes, alone or in the presence of a CYP3A inhibitor.  Collectively, our studies identified important pharmacophore elements and revealed addnl. binding features of I, which could aid in future design of improved ligands to better probe the mol. basis of CA-host factor interactions, design strategies to disrupt them, and ultimately identify viable CA-targeting antiviral leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6uw0dmkMDbbVg90H21EOLACvtfcHk0lgmSojGhoAGhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsl2mt74%253D&md5=7bedb9a56ce8d6dcc82e9ee352052141</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112427%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DCasey%26aufirst%3DM.%2BC.%26aulast%3DVernekar%26aufirst%3DS.%2BK.%2BV.%26aulast%3DDo%26aufirst%3DH.%2BT.%26aulast%3DSahani%26aufirst%3DR.%2BL.%26aulast%3DKirby%26aufirst%3DK.%2BA.%26aulast%3DDu%26aufirst%3DH.%26aulast%3DHachiya%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DTedbury%26aufirst%3DP.%2BR.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DChemical%2520profiling%2520of%2520HIV-1%2520capsid-targeting%2520antiviral%2520PF74%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D200%26spage%3D112427%26doi%3D10.1016%2Fj.ejmech.2020.112427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vernekar, S. K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahani, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casey, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kankanala, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tedbury, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Toward Structurally Novel and Metabolically Stable HIV-1 Capsid-Targeting Small Molecules</span>. <i>Viruses</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">452</span>, <span class="refDoi"> DOI: 10.3390/v12040452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3390%2Fv12040452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1SjtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=452&author=S.+K.+V.+Vernekarauthor=R.+L.+Sahaniauthor=M.+C.+Caseyauthor=J.+Kankanalaauthor=L.+Wangauthor=K.+A.+Kirbyauthor=H.+Duauthor=H.+Zhangauthor=P.+R.+Tedburyauthor=J.+Xieauthor=S.+G.+Sarafianosauthor=Z.+Wang&title=Toward+Structurally+Novel+and+Metabolically+Stable+HIV-1+Capsid-Targeting+Small+Molecules&doi=10.3390%2Fv12040452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Toward structurally novel and metabolically stable HIV-1 capsid-targeting small molecules</span></div><div class="casAuthors">Vernekar, Sanjeev Kumar V.; Sahani, Rajkumar Lalji; Casey, Mary C.; Kankanala, Jayakanth; Wang, Lei; Kirby, Karen A.; Du, Haijuan; Zhang, Huanchun; Tedbury, Philip R.; Xie, Jiashu; Sarafianos, Stefan G.; Wang, Zhengqiang</div><div class="citationInfo"><span class="NLM_cas:title">Viruses</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">452</span>CODEN:
                <span class="NLM_cas:coden">VIRUBR</span>;
        ISSN:<span class="NLM_cas:issn">1999-4915</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">HIV-1 capsid protein (CA) plays an important role in many steps of viral replication and represents an appealing antiviral target.  Several CA-targeting small mols. of various chemotypes have been studied, but the peptidomimetic PF74 has drawn particular interest due to its potent antiviral activity, well-characterized binding mode, and unique mechanism of action.  Importantly, PF74 competes against important host factors for binding, conferring highly desirable antiviral phenotypes.  However, further development of PF74 is hindered by its prohibitively poor metabolic stability, which necessitates the search for structurally novel and metabolically stable chemotypes.  We have conducted a pharmacophore-based shape similarity search for compds. mimicking PF74.  We report herein the analog synthesis and structure-activity relationship (SAR) of two hits from the search, and a third hit designed via mol. hybridization.  All analogs were characterized for their effect on CA hexamer stability, antiviral activity, and cytotoxicity.  These assays identified three active compds. that moderately stabilize CA hexamer and inhibit HIV-1.  The most potent analog (10) inhibited HIV-1 comparably to PF74 but demonstrated drastically improved metabolic stability in liver microsomes (31 min vs. 0.7 min t1/2).  Collectively, the current studies identified a structurally novel and metabolically stable PF74-like chemotype for targeting HIV-1 CA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNXpsA3fUhj7Vg90H21EOLACvtfcHk0lgt2s0-_RJU8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1SjtLbM&md5=5ab729e4264f03f493a34ec090f5f3e2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3390%2Fv12040452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fv12040452%26sid%3Dliteratum%253Aachs%26aulast%3DVernekar%26aufirst%3DS.%2BK.%2BV.%26aulast%3DSahani%26aufirst%3DR.%2BL.%26aulast%3DCasey%26aufirst%3DM.%2BC.%26aulast%3DKankanala%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DKirby%26aufirst%3DK.%2BA.%26aulast%3DDu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DTedbury%26aufirst%3DP.%2BR.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DToward%2520Structurally%2520Novel%2520and%2520Metabolically%2520Stable%2520HIV-1%2520Capsid-Targeting%2520Small%2520Molecules%26jtitle%3DViruses%26date%3D2020%26volume%3D12%26spage%3D452%26doi%3D10.3390%2Fv12040452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalloum, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and structure-activity relationships of 4-phenyl-1H-1,2,3-triazole phenylalanine derivatives as novel HIV-1 capsid inhibitors with promising antiviral activities</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>190</i></span>,  <span class="NLM_fpage">112085</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.ejmech.2020.112085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=32066010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFGqur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2020&pages=112085&author=L.+Sunauthor=T.+Huangauthor=A.+Dickauthor=M.+E.+Meuserauthor=W.+A.+Zalloumauthor=C.+H.+Chenauthor=X.+Dingauthor=P.+Gaoauthor=S.+Cocklinauthor=K.+H.+Leeauthor=P.+Zhanauthor=X.+Liu&title=Design%2C+synthesis+and+structure-activity+relationships+of+4-phenyl-1H-1%2C2%2C3-triazole+phenylalanine+derivatives+as+novel+HIV-1+capsid+inhibitors+with+promising+antiviral+activities&doi=10.1016%2Fj.ejmech.2020.112085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and structure-activity relationships of 4-phenyl-1H-1,2,3-triazole phenylalanine derivatives as novel HIV-1 capsid inhibitors with promising antiviral activities</span></div><div class="casAuthors">Sun, Lin; Huang, Tianguang; Dick, Alexej; Meuser, Megan E.; Zalloum, Waleed A.; Chen, Chin-Ho; Ding, Xiao; Gao, Ping; Cocklin, Simon; Lee, Kuo-Hsiung; Zhan, Peng; Liu, Xinyong</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112085</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">HIV-1 CA is involved in different stages of the viral replication cycle, performing essential roles in both early (uncoating, reverse transcription, nuclear import, integration) and late events (assembly).  Recent efforts have demonstrated HIV-1 CA protein as a prospective therapeutic target for the development of new antivirals.  The most extensively studied CA inhibitor, PF-3450074 (PF-74, discovered by Pfizer), that targets an inter-protomer pocket within the CA hexamer.  Herein we reported the design, synthesis, and biol. evaluation of a series of 4-phenyl-1H-1,2,3-triazole phenylalanine derivs. as HIV-1 CA inhibitors based on PF-74 scaffold.  Most of the analogs demonstrated potent antiviral activities, among them, the anti-HIV-1 activity of (I) (R1 = C:O-2-F-C6H4) (EC50 = 3.13μM) is particularly prominent.  The SPR binding assay of selected compds. I (R1 = C:O-2-F-C6H4, C:O-3-F-C6H4) and (II) suggested direct and effective interaction with recombinant CA proteins.  The mechanism of action studies also demonstrated that I (R1 = C:O-2-F-C6H4) displays the effects in both the early and late stages of HIV-1 replication.  To explore the potential binding mode of the here presented analogs, I (R1 = C:O-2-F-C6H4) was analyzed by MD simulation to predict its binding to the active site of HIV-1 CA monomer.  In conclusion, this novel series of antivirals can serve as a starting point for the development of a new generation of HIV-1 treatment regimen and highlights the potentiality of CA as a therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnj7yCN6FM7Vg90H21EOLACvtfcHk0lgt2s0-_RJU8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFGqur8%253D&md5=0eece1c4f60dd7d97f80015fb915ed6d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112085%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DDick%26aufirst%3DA.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DZalloum%26aufirst%3DW.%2BA.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DDesign%252C%2520synthesis%2520and%2520structure-activity%2520relationships%2520of%25204-phenyl-1H-1%252C2%252C3-triazole%2520phenylalanine%2520derivatives%2520as%2520novel%2520HIV-1%2520capsid%2520inhibitors%2520with%2520promising%2520antiviral%2520activities%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D190%26spage%3D112085%26doi%3D10.1016%2Fj.ejmech.2020.112085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalloum, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherukupalli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Clercq, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannecouque, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Mechanism Study of Benzenesulfonamide-Containing Phenylalanine Derivatives as Novel HIV-1 Capsid Inhibitors with Improved Antiviral Activities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">4790</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00015</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnt12ru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=4790&author=L.+Sunauthor=A.+Dickauthor=M.+E.+Meuserauthor=T.+Huangauthor=W.+A.+Zalloumauthor=C.+H.+Chenauthor=S.+Cherukupalliauthor=S.+Xuauthor=X.+Dingauthor=P.+Gaoauthor=D.+Kangauthor=E.+De+Clercqauthor=C.+Pannecouqueauthor=S.+Cocklinauthor=K.+H.+Leeauthor=X.+Liuauthor=P.+Zhan&title=Design%2C+Synthesis%2C+and+Mechanism+Study+of+Benzenesulfonamide-Containing+Phenylalanine+Derivatives+as+Novel+HIV-1+Capsid+Inhibitors+with+Improved+Antiviral+Activities&doi=10.1021%2Facs.jmedchem.0c00015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Mechanism Study of Benzenesulfonamide-Containing Phenylalanine Derivatives as Novel HIV-1 Capsid Inhibitors with Improved Antiviral Activities</span></div><div class="casAuthors">Sun, Lin; Dick, Alexej; Meuser, Megan E.; Huang, Tianguang; Zalloum, Waleed A.; Chen, Chin-Ho; Cherukupalli, Srinivasulu; Xu, Shujing; Ding, Xiao; Gao, Ping; Kang, Dongwei; De Clercq, Erik; Pannecouque, Christophe; Cocklin, Simon; Lee, Kuo-Hsiung; Liu, Xinyong; Zhan, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4790-4810</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 CA protein has gained remarkable attention as a promising therapeutic target for the development of new antivirals, due to its pivotal roles in HIV-1 replication (structural and regulatory).  Herein, we report the design and synthesis of three series of benzenesulfonamide-contg. phenylalanine derivs. obtained by further structural modifications of PF-74 to aid in the discovery of more potent and drug-like HIV-1 CA inhibitors.  Structure-activity relationship studies of these compds. led to the identification of new phenylalanine derivs. with a piperazinone moiety, represented by compd. (I), which exhibited anti-HIV-1NL4-3 activity 5.78-fold better than PF-74.  Interestingly, I also showed anti-HIV-2ROD activity (EC50 = 31 nΜ), with almost 120 times increased potency over PF-74.  However, due to the higher significance of HIV-1 as compared to HIV-2 for the human population, this manuscript focused on the mechanism of action of our compds. in the context of HIV-1.  SPR studies on representative compds. confirmed CA as the binding target.  The action stage detn. assay demonstrated that these inhibitors exhibited antiviral activities with a dual-stage inhibition profile.  The early-stage inhibitory activity of compd. I was 6.25 times more potent as compared to PF-74, but appears to work via accelerating capsid core assembly rather than stabilization.  However, the mechanism by which they exert their antiviral activity in the late-stage appears to be the same as PF-74 with less infectious HIV-1 virions are produced in their presence as judged p24 content studies.  MD simulations provided the key rationale for the promising antiviral potency of I.  Addnl., I exhibited modest increase in HLM and human plasma metabolic stabilities as compared to PF-74, as well as moderately improved pharmacokinetic profile, favorable oral bioavailability, and no acute toxicity.  These studies provide insights and serves as a starting point for subsequent medicinal chem. efforts in optimizing these promising HIV inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwKdnsc-Iy-LVg90H21EOLACvtfcHk0ljRVeu5HnNlbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnt12ru7s%253D&md5=77a89a34489fc04cbe4c29b5bdb4b396</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00015%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DDick%26aufirst%3DA.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DZalloum%26aufirst%3DW.%2BA.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DCherukupalli%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DDe%2BClercq%26aufirst%3DE.%26aulast%3DPannecouque%26aufirst%3DC.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DP.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Mechanism%2520Study%2520of%2520Benzenesulfonamide-Containing%2520Phenylalanine%2520Derivatives%2520as%2520Novel%2520HIV-1%2520Capsid%2520Inhibitors%2520with%2520Improved%2520Antiviral%2520Activities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26spage%3D4790%26doi%3D10.1021%2Facs.jmedchem.0c00015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalloum, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel 1,4-disubstituted 1,2,3-triazole phenylalanine derivatives as HIV-1 capsid inhibitors</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">28961</span>– <span class="NLM_lpage">28986</span>, <span class="refDoi"> DOI: 10.1039/C9RA05869A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1039%2FC9RA05869A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=32089839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslyitr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=28961-28986&author=X.+Jiangauthor=G.+Wuauthor=W.+A.+Zalloumauthor=M.+E.+Meuserauthor=A.+Dickauthor=L.+Sunauthor=C.+H.+Chenauthor=D.+Kangauthor=L.+Jingauthor=R.+Jiaauthor=S.+Cocklinauthor=K.+H.+Leeauthor=X.+Liuauthor=P.+Zhan&title=Discovery+of+novel+1%2C4-disubstituted+1%2C2%2C3-triazole+phenylalanine+derivatives+as+HIV-1+capsid+inhibitors&doi=10.1039%2FC9RA05869A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel 1,4-disubstituted 1,2,3-triazole phenylalanine derivatives as HIV-1 capsid inhibitors</span></div><div class="casAuthors">Jiang, Xiangyi; Wu, Gaochan; Zalloum, Waleed A.; Meuser, Megan E.; Dick, Alexej; Sun, Lin; Chen, Chin-Ho; Kang, Dongwei; Jing, Lanlan; Jia, Ruifang; Cocklin, Simon; Lee, Kuo-Hsiung; Liu, Xinyong; Zhan, Peng</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">28961-28986</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The HIV-1 capsid (CA) protein plays crucial roles in both early and late stages of the viral life cycle, which has intrigued researchers to develop anti-HIV drugs that target it.  Accordingly, in this research, we report the design, synthesis and biol. evaluation of a series of novel phenylalanine derivs. as HIV-1 CA protein inhibitors using the Cu(I)-catalyzed azide and alkyne 1,3-dipolar cycloaddn. (CuAAC) reaction.  Among this series of inhibitors, compd. I displayed a remarkable anti-HIV activity (EC50 = 2.13μM, CC50 > 35.49μM).  Furthermore, surface plasmon resonance (SPR) binding assays showed that compds. I and PF-74 (II, lead compd.) have similar affinities to the HIV-1 CA monomer.  Further investigation showed that the weak permeability and water soly. of representative compds. were probably the important factors that restricted their cell-based activity.  Preliminary structure-activity relationships (SARs) were inferred based on the activities of these compds. and their known structure.  The most promising new compd. was studied with mol. dynamics simulation (MD) to det. the preferred interactions with the drug target.  Finally, the activities of members of this series of inhibitors were deeply inspected to find the potential reasons for their anti-HIV-1 activity from various perspectives.  This highlights the important factors required to design compds. with improved potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraZBDm_k3IR7Vg90H21EOLACvtfcHk0ljRVeu5HnNlbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslyitr3K&md5=c3c80658dcfe4edeb06f7cd9ea348082</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1039%2FC9RA05869A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9RA05869A%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DZalloum%26aufirst%3DW.%2BA.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DDick%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DJing%26aufirst%3DL.%26aulast%3DJia%26aufirst%3DR.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520novel%25201%252C4-disubstituted%25201%252C2%252C3-triazole%2520phenylalanine%2520derivatives%2520as%2520HIV-1%2520capsid%2520inhibitors%26jtitle%3DRSC%2520Adv.%26date%3D2019%26volume%3D9%26spage%3D28961%26epage%3D28986%26doi%3D10.1039%2FC9RA05869A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalloum, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of phenylalanine derivatives as potent HIV-1 capsid inhibitors from click chemistry-based compound library</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">478</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.ejmech.2018.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=30243152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOltL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2018&pages=478-492&author=G.+Wuauthor=W.+A.+Zalloumauthor=M.+E.+Meuserauthor=L.+Jingauthor=D.+Kangauthor=C.+H.+Chenauthor=Y.+Tianauthor=F.+Zhangauthor=S.+Cocklinauthor=K.+H.+Leeauthor=X.+Liuauthor=P.+Zhan&title=Discovery+of+phenylalanine+derivatives+as+potent+HIV-1+capsid+inhibitors+from+click+chemistry-based+compound+library&doi=10.1016%2Fj.ejmech.2018.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of phenylalanine derivatives as potent HIV-1 capsid inhibitors from Click chemistry-based compound library</span></div><div class="casAuthors">Wu, Gaochan; Zalloum, Waleed A.; Meuser, Megan E.; Jing, Lanlan; Kang, Dongwei; Chen, Chin-Ho; Tian, Ye; Zhang, Fangfang; Cocklin, Simon; Lee, Kuo-Hsiung; Liu, Xinyong; Zhan, Peng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">478-492</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The HIV-1 capsid (CA) protein plays essential roles in both early and late stages of HIV-1 replication and is considered an important, clin. unexploited therapeutic target.  As such, small drug-like mols. that inhibit this crit. HIV-1 protein have become a priority for several groups.  Therefore, in this study we explore small mol. targeting of the CA protein, and in particular a very attractive inter-protomer pocket.  We report the design, parallel synthesis, and anti-HIV-1 activity evaluation of a series of novel phenylalanine derivs. as HIV-1 CA protein inhibitors synthesized via Cu(I)-catalyzed alkyne-azide 1,3-dipolar cycloaddn. (CuAAC) reaction.  We demonstrate robust inhibitory activity over a range of potencies against the HIV-1 NL4-3 ref. strain.  In particular, compd. (I) exhibited the greatest potency and lowest toxicity within this new series with an EC50 value of 4.33 μM and CC50 value of >57.74 μM (SI > 13.33).  These values are very similar to the lead compd. PF-74 (EC50 = 5.95 μM, CC50 > 70.50 μM, SI > 11.85) in our assay, despite significant structural difference.  Furthermore, we demonstrate via surface plasmon resonance (SPR) binding assays that I interacts robustly with recombinant HIV-1 CA and exhibits antiviral activity in both the early and late stages of HIV-1 replication.  Overall, the novel parallel synthesis and structure-activity relationships (SARs) identified within this study set the foundation for further rational optimization and discovery of CA-targeting compds. with improved potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog7rdhyTNYnbVg90H21EOLACvtfcHk0ljRVeu5HnNlbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOltL%252FP&md5=6a45e0472739f33bfcc7c816c3b1d102</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%26aulast%3DZalloum%26aufirst%3DW.%2BA.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DJing%26aufirst%3DL.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520phenylalanine%2520derivatives%2520as%2520potent%2520HIV-1%2520capsid%2520inhibitors%2520from%2520click%2520chemistry-based%2520compound%2520library%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D158%26spage%3D478%26epage%3D492%26doi%3D10.1016%2Fj.ejmech.2018.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Link, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somoza, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bam, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canales, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brizgys, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graupe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morganelli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halcomb, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazerwith, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novikov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villaseñor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mwangi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedziela-Majka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papalia, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavitt, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stepan, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagratis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clancy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmadyar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, T. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolckenhauer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callebaut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ram, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinclair, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruane, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crofoot, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brainard, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundquist, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakowicz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chester, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daar, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yant, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span> <span> </span><span class="NLM_article-title">Clinical targeting of HIV capsid protein with a long-acting small molecule</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">614</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2443-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1038%2Fs41586-020-2443-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=32612233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlShtL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=614&author=J.+O.+Linkauthor=M.+S.+Rheeauthor=W.+C.+Tseauthor=J.+Zhengauthor=J.+R.+Somozaauthor=W.+Roweauthor=R.+Begleyauthor=A.+Chiuauthor=A.+Mulatoauthor=D.+Hansenauthor=E.+Singerauthor=L.+K.+Tsaiauthor=R.+A.+Bamauthor=C.+H.+Chouauthor=E.+Canalesauthor=G.+Brizgysauthor=J.+R.+Zhangauthor=J.+Liauthor=M.+Graupeauthor=P.+Morganelliauthor=Q.+Liuauthor=Q.+Wuauthor=R.+L.+Halcombauthor=R.+D.+Saitoauthor=S.+D.+Schroederauthor=S.+E.+Lazerwithauthor=S.+Bondyauthor=D.+Jinauthor=M.+Hungauthor=N.+Novikovauthor=X.+Liuauthor=A.+G.+Villase%C3%B1orauthor=C.+E.+Cannizzaroauthor=E.+Y.+Huauthor=R.+L.+Andersonauthor=T.+C.+Applebyauthor=B.+Luauthor=J.+Mwangiauthor=A.+Liclicanauthor=A.+Niedziela-Majkaauthor=G.+A.+Papaliaauthor=M.+H.+Wongauthor=S.+A.+Leavittauthor=Y.+Xuauthor=D.+Koditekauthor=G.+J.+Stepanauthor=H.+Yuauthor=N.+Pagratisauthor=S.+Clancyauthor=S.+Ahmadyarauthor=T.+Z.+Caiauthor=S.+Sellersauthor=S.+A.+Wolckenhauerauthor=J.+Lingauthor=C.+Callebautauthor=N.+Margotauthor=R.+R.+Ramauthor=Y.+P.+Liuauthor=R.+Hylandauthor=G.+I.+Sinclairauthor=P.+J.+Ruaneauthor=G.+E.+Crofootauthor=C.+K.+McDonaldauthor=D.+M.+Brainardauthor=L.+Ladauthor=S.+Swaminathanauthor=W.+I.+Sundquistauthor=R.+Sakowiczauthor=A.+E.+Chesterauthor=W.+E.+Leeauthor=E.+S.+Daarauthor=S.+R.+Yantauthor=T.+Cihlar&title=Clinical+targeting+of+HIV+capsid+protein+with+a+long-acting+small+molecule&doi=10.1038%2Fs41586-020-2443-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical targeting of HIV capsid protein with a long-acting small molecule</span></div><div class="casAuthors">Link, John O.; Rhee, Martin S.; Tse, Winston C.; Zheng, Jim; Somoza, John R.; Rowe, William; Begley, Rebecca; Chiu, Anna; Mulato, Andrew; Hansen, Derek; Singer, Eric; Tsai, Luong K.; Bam, Rujuta A.; Chou, Chien-Hung; Canales, Eda; Brizgys, Gediminas; Zhang, Jennifer R.; Li, Jiayao; Graupe, Michael; Morganelli, Philip; Liu, Qi; Wu, Qiaoyin; Halcomb, Randall L.; Saito, Roland D.; Schroeder, Scott D.; Lazerwith, Scott E.; Bondy, Steven; Jin, Debi; Hung, Magdeleine; Novikov, Nikolai; Liu, Xiaohong; Villasenor, Armando G.; Cannizzaro, Carina E.; Hu, Eric Y.; Anderson, Robert L.; Appleby, Todd C.; Lu, Bing; Mwangi, Judy; Liclican, Albert; Niedziela-Majka, Anita; Papalia, Giuseppe A.; Wong, Melanie H.; Leavitt, Stephanie A.; Xu, Yili; Koditek, David; Stepan, George J.; Yu, Helen; Pagratis, Nikos; Clancy, Sheila; Ahmadyar, Shekeba; Cai, Terrence Z.; Sellers, Scott; Wolckenhauer, Scott A.; Ling, John; Callebaut, Christian; Margot, Nicolas; Ram, Renee R.; Liu, Ya-Pei; Hyland, Rob; Sinclair, Gary I.; Ruane, Peter J.; Crofoot, Gordon E.; McDonald, Cheryl K.; Brainard, Diana M.; Lad, Latesh; Swaminathan, Swami; Sundquist, Wesley I.; Sakowicz, Roman; Chester, Anne E.; Lee, William E.; Daar, Eric S.; Yant, Stephen R.; Cihlar, Tomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">7822</span>),
    <span class="NLM_cas:pages">614-618</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Oral antiretroviral agents provide life-saving treatments for millions of people living with HIV, and can prevent new infections via pre-exposure prophylaxis1-5.  However, some people living with HIV who are heavily treatment-experienced have limited or no treatment options, owing to multidrug resistance6.  In addn., suboptimal adherence to oral daily regimens can neg. affect the outcome of treatment-which contributes to virol. failure, resistance generation and viral transmission-as well as of pre-exposure prophylaxis, leading to new infections1,2,4,7-9.  Long-acting agents from new antiretroviral classes can provide much-needed treatment options for people living with HIV who are heavily treatment-experienced, and addnl. can improve adherence10.  Here we describe GS-6207, a small mol. that disrupts the functions of HIV capsid protein and is amenable to long-acting therapy owing to its high potency, low in vivo systemic clearance and slow release kinetics from the s.c. injection site.  Drawing on X-ray crystallog. information, we designed GS-6207 to bind tightly at a conserved interface between capsid protein monomers, where it interferes with capsid-protein-mediated interactions between proteins that are essential for multiple phases of the viral replication cycle.  GS-6207 exhibits antiviral activity at picomolar concns. against all subtypes of HIV-1 that we tested, and shows high synergy and no cross-resistance with approved antiretroviral drugs.  In phase-1 clin. studies, monotherapy with a single s.c. dose of GS-6207 (450 mg) resulted in a mean log10-transformed redn. of plasma viral load of 2.2 after 9 days, and showed sustained plasma exposure at antivirally active concns. for more than 6 mo.  These results provide clin. validation for therapies that target the functions of HIV capsid protein, and demonstrate the potential of GS-6207 as a long-acting agent to treat or prevent infection with HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDdDh1BVoW47Vg90H21EOLACvtfcHk0litsyXMEranKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlShtL7K&md5=24a655318952e26a71587a4bc44307e2</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2443-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2443-1%26sid%3Dliteratum%253Aachs%26aulast%3DLink%26aufirst%3DJ.%2BO.%26aulast%3DRhee%26aufirst%3DM.%2BS.%26aulast%3DTse%26aufirst%3DW.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DSomoza%26aufirst%3DJ.%2BR.%26aulast%3DRowe%26aufirst%3DW.%26aulast%3DBegley%26aufirst%3DR.%26aulast%3DChiu%26aufirst%3DA.%26aulast%3DMulato%26aufirst%3DA.%26aulast%3DHansen%26aufirst%3DD.%26aulast%3DSinger%26aufirst%3DE.%26aulast%3DTsai%26aufirst%3DL.%2BK.%26aulast%3DBam%26aufirst%3DR.%2BA.%26aulast%3DChou%26aufirst%3DC.%2BH.%26aulast%3DCanales%26aufirst%3DE.%26aulast%3DBrizgys%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DGraupe%26aufirst%3DM.%26aulast%3DMorganelli%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DHalcomb%26aufirst%3DR.%2BL.%26aulast%3DSaito%26aufirst%3DR.%2BD.%26aulast%3DSchroeder%26aufirst%3DS.%2BD.%26aulast%3DLazerwith%26aufirst%3DS.%2BE.%26aulast%3DBondy%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DD.%26aulast%3DHung%26aufirst%3DM.%26aulast%3DNovikov%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DVillase%25C3%25B1or%26aufirst%3DA.%2BG.%26aulast%3DCannizzaro%26aufirst%3DC.%2BE.%26aulast%3DHu%26aufirst%3DE.%2BY.%26aulast%3DAnderson%26aufirst%3DR.%2BL.%26aulast%3DAppleby%26aufirst%3DT.%2BC.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DMwangi%26aufirst%3DJ.%26aulast%3DLiclican%26aufirst%3DA.%26aulast%3DNiedziela-Majka%26aufirst%3DA.%26aulast%3DPapalia%26aufirst%3DG.%2BA.%26aulast%3DWong%26aufirst%3DM.%2BH.%26aulast%3DLeavitt%26aufirst%3DS.%2BA.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DStepan%26aufirst%3DG.%2BJ.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DPagratis%26aufirst%3DN.%26aulast%3DClancy%26aufirst%3DS.%26aulast%3DAhmadyar%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DT.%2BZ.%26aulast%3DSellers%26aufirst%3DS.%26aulast%3DWolckenhauer%26aufirst%3DS.%2BA.%26aulast%3DLing%26aufirst%3DJ.%26aulast%3DCallebaut%26aufirst%3DC.%26aulast%3DMargot%26aufirst%3DN.%26aulast%3DRam%26aufirst%3DR.%2BR.%26aulast%3DLiu%26aufirst%3DY.%2BP.%26aulast%3DHyland%26aufirst%3DR.%26aulast%3DSinclair%26aufirst%3DG.%2BI.%26aulast%3DRuane%26aufirst%3DP.%2BJ.%26aulast%3DCrofoot%26aufirst%3DG.%2BE.%26aulast%3DMcDonald%26aufirst%3DC.%2BK.%26aulast%3DBrainard%26aufirst%3DD.%2BM.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DSundquist%26aufirst%3DW.%2BI.%26aulast%3DSakowicz%26aufirst%3DR.%26aulast%3DChester%26aufirst%3DA.%2BE.%26aulast%3DLee%26aufirst%3DW.%2BE.%26aulast%3DDaar%26aufirst%3DE.%2BS.%26aulast%3DYant%26aufirst%3DS.%2BR.%26aulast%3DCihlar%26aufirst%3DT.%26atitle%3DClinical%2520targeting%2520of%2520HIV%2520capsid%2520protein%2520with%2520a%2520long-acting%2520small%2520molecule%26jtitle%3DNature%26date%3D2020%26spage%3D614%26doi%3D10.1038%2Fs41586-020-2443-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallazzi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neogi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sönnerborg, A.</span></span> <span> </span><span class="NLM_article-title">GS-CA Compounds: First-In-Class HIV-1 Capsid Inhibitors Covering Multiple Grounds</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1227</span>, <span class="refDoi"> DOI: 10.3389/fmicb.2019.01227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3389%2Ffmicb.2019.01227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=31312185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A280%3ADC%252BB3MzotVWnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1227&author=K.+Singhauthor=F.+Gallazziauthor=K.+J.+Hillauthor=D.+H.+Burkeauthor=M.+J.+Langeauthor=T.+P.+Quinnauthor=U.+Neogiauthor=A.+S%C3%B6nnerborg&title=GS-CA+Compounds%3A+First-In-Class+HIV-1+Capsid+Inhibitors+Covering+Multiple+Grounds&doi=10.3389%2Ffmicb.2019.01227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">GS-CA Compounds: First-In-Class HIV-1 Capsid Inhibitors Covering Multiple Grounds</span></div><div class="casAuthors">Singh Kamal; Hill Kyle J; Burke Donald H; Lange Margaret J; Singh Kamal; Gallazzi Fabio; Hill Kyle J; Burke Donald H; Lange Margaret J; Singh Kamal; Neogi Ujjwal; Sonnerborg Anders; Gallazzi Fabio; Quinn Thomas P; Sonnerborg Anders</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1227</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">Recently reported HIV-1 capsid (CA) inhibitors GS-CA1 and GS-6207 (an analog of GS-CA1) are first-in-class compounds with long-acting potential.  Reportedly, both compounds have greater potency than currently approved anti-HIV drugs.  Due to the limited access to experimental data and the compounds themselves, a detailed mechanism of their inhibition is yet to be delineated.  Using crystal structures of capsid-hexamers bound to well-studied capsid inhibitor PF74 and molecular modeling, we predict that GS-CA compounds bind in the pocket that is shared by previously reported CA inhibitors and host factors.  Additionally, comparative modeling suggests that GS-CA compounds have unique structural features contributing to interactions with capsid.  To test their proposed binding mode, we also report the design of a cyclic peptide combining structural units from GS-CA compounds, host factors, and previously reported capsid inhibitors.  This peptide (Pep-1) binds CA-hexamer with a docking score comparable to GS-CA compounds.  Affinity determination by MicroScale thermophoresis (MST) assays showed that CA binds Pep-1 with a ~7-fold better affinity than well-studied capsid inhibitor PF74, suggesting that it can be developed as a possible CA inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSM3uMyo0R5PhWfpMXQLRstfW6udTcc2eZIXCm2ZCYVSrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzotVWnug%253D%253D&md5=ef152804cda76b304c928ba1a0b7c654</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2019.01227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2019.01227%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DK.%26aulast%3DGallazzi%26aufirst%3DF.%26aulast%3DHill%26aufirst%3DK.%2BJ.%26aulast%3DBurke%26aufirst%3DD.%2BH.%26aulast%3DLange%26aufirst%3DM.%2BJ.%26aulast%3DQuinn%26aufirst%3DT.%2BP.%26aulast%3DNeogi%26aufirst%3DU.%26aulast%3DS%25C3%25B6nnerborg%26aufirst%3DA.%26atitle%3DGS-CA%2520Compounds%253A%2520First-In-Class%2520HIV-1%2520Capsid%2520Inhibitors%2520Covering%2520Multiple%2520Grounds%26jtitle%3DFront.%2520Microbiol.%26date%3D2019%26volume%3D10%26spage%3D1227%26doi%3D10.3389%2Ffmicb.2019.01227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karadsheh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Composition and Orientation of the Core Region of Novel HIV-1 Entry Inhibitors Influences Metabolic Stability</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1430</span>, <span class="refDoi"> DOI: 10.3390/molecules25061430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3390%2Fmolecules25061430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosFWqtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=1430&author=R.+Karadshehauthor=M.+E.+Meuserauthor=S.+Cocklin&title=Composition+and+Orientation+of+the+Core+Region+of+Novel+HIV-1+Entry+Inhibitors+Influences+Metabolic+Stability&doi=10.3390%2Fmolecules25061430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Composition and orientation of the core region of novel HIV-1 entry inhibitors influences metabolic stability</span></div><div class="casAuthors">Karadsheh, Rama; Meuser, Megan E.; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1430</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Fostemsavir/temsavir is an investigational HIV-1 entry inhibitor currently in late-stage clin. trials.  Although it holds promise to be a first-in-class Env-targeted entry inhibitor for the clinic, issues with bioavailability relegate its use to salvage therapies only.  As such, the development of a small mol. HIV-1 entry inhibitor that can be used in std. combination antiretroviral therapy (cART) remains a longstanding goal for the field.  We previously demonstrated the ability of extending the chemotypes available to this class of inhibitor as the first step towards this overarching goal.  In addn. to poor soly., metabolic stability is a crucial determinant of bioavailability.  Therefore, in this short communication, we assess the metabolic stabilities of five of our novel chemotype entry inhibitors.  We found that changing the piperazine core region of temsavir alters the stability of the compd. in human liver microsome assays.  Moreover, we identified an entry inhibitor with more than twice the metabolic stability of temsavir and demonstrated that the orientation of the core replacement is crit. for this increase.  This work further demonstrates the feasibility of our long-term goal-to design an entry inhibitor with improved drug-like qualities-and warrants expanded studies to achieve this.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocjASbOlm-v7Vg90H21EOLACvtfcHk0ljQq2qFwObafw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosFWqtr8%253D&md5=4298fd7cfd1cd7bf4b00c6b94c905b22</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25061430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25061430%26sid%3Dliteratum%253Aachs%26aulast%3DKaradsheh%26aufirst%3DR.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DComposition%2520and%2520Orientation%2520of%2520the%2520Core%2520Region%2520of%2520Novel%2520HIV-1%2520Entry%2520Inhibitors%2520Influences%2520Metabolic%2520Stability%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D1430%26doi%3D10.3390%2Fmolecules25061430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Identification of a small molecule HIV-1 inhibitor that targets the capsid hexamer</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.12.087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.bmcl.2015.12.087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=26747394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=824-828&author=J.+P.+Xuauthor=J.+D.+Bransonauthor=R.+Lawrenceauthor=S.+Cocklin&title=Identification+of+a+small+molecule+HIV-1+inhibitor+that+targets+the+capsid+hexamer&doi=10.1016%2Fj.bmcl.2015.12.087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a small molecule HIV-1 inhibitor that targets the capsid hexamer</span></div><div class="casAuthors">Xu, Jimmy P.; Branson, Jeffrey D.; Lawrence, Rae; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">824-828</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The HIV-1 CA protein is an attractive therapeutic target for the development of new antivirals.  An inter-protomer pocket within the hexamer configuration of the CA, which is a binding site for key host dependency factors, is an esp. appealing region for small mol. targeting.  Using a field-based pharmacophore derived from an inhibitor known to interact with this region, coupled to biochem. and biol. assessment, the authors have identified a new compd. that inhibits HIV-1 infection and that targets the assembled CA hexamer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMgMK74WiSBbVg90H21EOLACvtfcHk0ljQq2qFwObafw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOltA%253D%253D&md5=c0a7dbedce0d39a14deaca41859374c2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.12.087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.12.087%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%2BP.%26aulast%3DBranson%26aufirst%3DJ.%2BD.%26aulast%3DLawrence%26aufirst%3DR.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520a%2520small%2520molecule%2520HIV-1%2520inhibitor%2520that%2520targets%2520the%2520capsid%2520hexamer%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D824%26epage%3D828%26doi%3D10.1016%2Fj.bmcl.2015.12.087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseright, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinter, A.</span></span> <span> </span><span class="NLM_article-title">Molecular field extrema as descriptors of biological activity: definition and validation</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">665</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1021/ci050357s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050357s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD28XisFCgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=665-676&author=T.+Cheeserightauthor=M.+Mackeyauthor=S.+Roseauthor=A.+Vinter&title=Molecular+field+extrema+as+descriptors+of+biological+activity%3A+definition+and+validation&doi=10.1021%2Fci050357s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Field Extrema as Descriptors of Biological Activity: Definition and Validation</span></div><div class="casAuthors">Cheeseright, Tim; Mackey, Mark; Rose, Sally; Vinter, Andy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">665-676</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The paper describes the generation of four types of three-dimensional mol. field descriptors or 'field points' as extrema of electrostatic, steric, and hydrophobic fields.  These field points are used to define the properties necessary for a mol. to bind in a characteristic way into a specified active site.  The hypothesis is that compds. showing a similar field point pattern are likely to bind at the same target site regardless of structure.  The methodol. to test this idea is illustrated using HIV NNRTI and thrombin ligands and validated across seven other targets.  From the in silico comparisons of field point overlays, the exptl. obsd. binding poses of these ligands in their resp. sites can be reproduced from pairwise comparisons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM6IB4phN5j7Vg90H21EOLACvtfcHk0ljQq2qFwObafw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisFCgsQ%253D%253D&md5=7cdf681c467143d9f95a3a2cbc5ae669</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci050357s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050357s%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseright%26aufirst%3DT.%26aulast%3DMackey%26aufirst%3DM.%26aulast%3DRose%26aufirst%3DS.%26aulast%3DVinter%26aufirst%3DA.%26atitle%3DMolecular%2520field%2520extrema%2520as%2520descriptors%2520of%2520biological%2520activity%253A%2520definition%2520and%2520validation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D665%26epage%3D676%26doi%3D10.1021%2Fci050357s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseright, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scoffin, R. A.</span></span> <span> </span><span class="NLM_article-title">High content pharmacophores from molecular fields: a biologically relevant method for comparing and understanding ligands</span>. <i>Curr. Comput.-Aided Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">190</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.2174/157340911796504314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.2174%2F157340911796504314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=21726191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCks7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=190-205&author=T.+J.+Cheeserightauthor=M.+D.+Mackeyauthor=R.+A.+Scoffin&title=High+content+pharmacophores+from+molecular+fields%3A+a+biologically+relevant+method+for+comparing+and+understanding+ligands&doi=10.2174%2F157340911796504314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">High content pharmacophores from molecular fields: a biologically relevant method for comparing and understanding ligands</span></div><div class="casAuthors">Cheeseright, Timothy J.; Mackey, Mark D.; Scoffin, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">190-205</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The question of how and why a small mol. binds to a protein is central to ligand-based drug discovery.  The traditional way of approaching these questions is pharmacophore anal.  However, pharmacophores as usually applied lack quantitation and subtlety.  An improvement is to consider the electrostatic and steric fields of the ligand directly.  Mol. fields provide a rich view of the potential interactions that a mol. can make and can be validated through exptl. data on mol. interactions and through quantum mechanics calcns.  A technique is presented in this review for comparing mols. using mol. fields and assigning similarity scores.  This high information content method can be used to align mols. for SAR anal., to det. the bioactive conformation from ligand data, and to screen large libraries of compds. for structurally unrelated actives.  An extension to allow interactive exploration of chem. space via bioisostere anal. is also reviewed.  Examples from the literature showing the success of these methods are presented, and future directions discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT897qyXPtyrVg90H21EOLACvtfcHk0lhaeJKRFJ50Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCks7%252FO&md5=a80c6b8718a9ef6bfe580a3378afa13f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.2174%2F157340911796504314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340911796504314%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseright%26aufirst%3DT.%2BJ.%26aulast%3DMackey%26aufirst%3DM.%2BD.%26aulast%3DScoffin%26aufirst%3DR.%2BA.%26atitle%3DHigh%2520content%2520pharmacophores%2520from%2520molecular%2520fields%253A%2520a%2520biologically%2520relevant%2520method%2520for%2520comparing%2520and%2520understanding%2520ligands%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2011%26volume%3D7%26spage%3D190%26epage%3D205%26doi%3D10.2174%2F157340911796504314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gres, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KewalRamani, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pornillos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span> <span> </span><span class="NLM_article-title">STRUCTURAL VIROLOGY. X-ray crystal structures of native HIV-1 capsid protein reveal conformational variability</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>349</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1126/science.aaa5936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1126%2Fscience.aaa5936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=26044298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOktLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=2015&pages=99-103&author=A.+T.+Gresauthor=K.+A.+Kirbyauthor=V.+N.+KewalRamaniauthor=J.+J.+Tannerauthor=O.+Pornillosauthor=S.+G.+Sarafianos&title=STRUCTURAL+VIROLOGY.+X-ray+crystal+structures+of+native+HIV-1+capsid+protein+reveal+conformational+variability&doi=10.1126%2Fscience.aaa5936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray crystal structures of native HIV-1 capsid protein reveal conformational variability</span></div><div class="casAuthors">Gres, Anna T.; Kirby, Karen A.; KewalRamani, Vineet N.; Tanner, John J.; Pornillos, Owen; Sarafianos, Stefan G.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">349</span>
        (<span class="NLM_cas:issue">6243</span>),
    <span class="NLM_cas:pages">99-103</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The detailed mol. interactions between native HIV-1 capsid protein (CA) hexamers that shield the viral genome and proteins have been elusive.  We report crystal structures describing interactions between CA monomers related by sixfold symmetry within hexamers (intrahexamer) and threefold and twofold symmetry between neighboring hexamers (interhexamer).  The structures describe how CA builds hexagonal lattices, the foundation of mature capsids.  Lattice structure depends on an adaptable hydration layer modulating interactions among CA mols.  Disruption of this layer alters interhexamer interfaces, highlighting an inherent structural variability.  A CA-targeting antiviral affects capsid stability by binding across CA mols. and subtly altering interhexamer interfaces remote to the ligand-binding site.  Inherent structural plasticity, hydration layer rearrangement, and effector binding affect capsid stability and have functional implications for the retroviral life cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk_3_G_w_yabVg90H21EOLACvtfcHk0lhaeJKRFJ50Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOktLvF&md5=7de62606c7753476c56a61c9dcd92acc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaa5936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaa5936%26sid%3Dliteratum%253Aachs%26aulast%3DGres%26aufirst%3DA.%2BT.%26aulast%3DKirby%26aufirst%3DK.%2BA.%26aulast%3DKewalRamani%26aufirst%3DV.%2BN.%26aulast%3DTanner%26aufirst%3DJ.%2BJ.%26aulast%3DPornillos%26aufirst%3DO.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26atitle%3DSTRUCTURAL%2520VIROLOGY.%2520X-ray%2520crystal%2520structures%2520of%2520native%2520HIV-1%2520capsid%2520protein%2520reveal%2520conformational%2520variability%26jtitle%3DScience%26date%3D2015%26volume%3D349%26spage%3D99%26epage%3D103%26doi%3D10.1126%2Fscience.aaa5936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reulecke, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Towards an integrated description of hydrogen bonding and dehydration: decreasing false positives in virtual screening with the HYDE scoring function</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">885</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200700319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1002%2Fcmdc.200700319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=18384086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvVWmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=885-897&author=I.+Reuleckeauthor=G.+Langeauthor=J.+Albrechtauthor=R.+Kleinauthor=M.+Rarey&title=Towards+an+integrated+description+of+hydrogen+bonding+and+dehydration%3A+decreasing+false+positives+in+virtual+screening+with+the+HYDE+scoring+function&doi=10.1002%2Fcmdc.200700319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Towards an integrated description of hydrogen bonding and dehydration: decreasing false positives in virtual screening with the HYDE scoring function</span></div><div class="casAuthors">Reulecke, Ingo; Lange, Gudrun; Albrecht, Juergen; Klein, Robert; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">885-897</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We developed a new empirical scoring function, HYDE, for the evaluation of protein-ligand complexes.  HYDE ests. binding free energy based on two terms for dehydration and hydrogen bonding only.  The essential feature of this scoring function is the integrated use of log P-derived at. increments for the prediction of free dehydration energy and hydrogen bonding energy.  Taking the dehydration of atoms within the interface into account shows that some atoms contribute favorably to the overall score, while others contribute unfavorably.  For instance, hydrogen bond functions are penalized if they are dehydrated unless they can overcompensate this loss by forming a hydrogen bond with excellent geometry.  The main stabilizing contribution represents the removal of apolar groups from the water: the hydrophobic effect.  Initial studies using the DUD dataset show that with HYDE, there is a significant decrease in false positives, a reasonable categorization of compds. as either non-binders, weak, medium or strong binders, and in particular, there is a generally applicable and thermodynamically sensible cutoff score below which there is a high likelihood that the compd. is indeed a binder.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBdc8JePMq8rVg90H21EOLACvtfcHk0lhaeJKRFJ50Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvVWmt7g%253D&md5=4fac432f70e5fa2373ec46450418c868</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200700319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200700319%26sid%3Dliteratum%253Aachs%26aulast%3DReulecke%26aufirst%3DI.%26aulast%3DLange%26aufirst%3DG.%26aulast%3DAlbrecht%26aufirst%3DJ.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DTowards%2520an%2520integrated%2520description%2520of%2520hydrogen%2520bonding%2520and%2520dehydration%253A%2520decreasing%2520false%2520positives%2520in%2520virtual%2520screening%2520with%2520the%2520HYDE%2520scoring%2520function%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D885%26epage%3D897%26doi%3D10.1002%2Fcmdc.200700319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tuyishime, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danish, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Princiotto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankowski, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombart, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esikov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berniac, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of novel small-molecule HIV-1 entry inhibitors using field-based virtual screening and bioisosteric replacement</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">5439</span>– <span class="NLM_lpage">5445</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.bmcl.2014.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=25454268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOru7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=5439-5445&author=M.+Tuyishimeauthor=M.+Danishauthor=A.+Princiottoauthor=M.+K.+Mankowskiauthor=R.+Lawrenceauthor=H.+G.+Lombartauthor=K.+Esikovauthor=J.+Berniacauthor=K.+Liangauthor=J.+Jiauthor=R.+G.+Ptakauthor=N.+Madaniauthor=S.+Cocklin&title=Discovery+and+optimization+of+novel+small-molecule+HIV-1+entry+inhibitors+using+field-based+virtual+screening+and+bioisosteric+replacement&doi=10.1016%2Fj.bmcl.2014.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of novel small-molecule HIV-1 entry inhibitors using field-based virtual screening and bioisosteric replacement</span></div><div class="casAuthors">Tuyishime, Marina; Danish, Matt; Princiotto, Amy; Mankowski, Marie K.; Lawrence, Rae; Lombart, Henry-Georges; Esikov, Kirill; Berniac, Joel; Liang, Kuang; Ji, Jingjing; Ptak, Roger G.; Madani, Navid; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5439-5445</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">With the emergence of drug-resistant strains and the cumulative toxicities assocd. with current therapies, demand remains for new inhibitors of HIV-1 replication.  The inhibition of HIV-1 entry is an attractive, yet underexploited therapeutic approach with implications for salvage and preexposure prophylactic regimens, as well as topical microbicides.  Using the combination of a field-derived bioactive conformation template to perform virtual screening and iterative bioisosteric replacements, coupled with in silico predictions of absorption, distribution, metab., and excretion, we have identified new leads for HIV-1 entry inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9DnVfZzTHZrVg90H21EOLACvtfcHk0lisaJD7Tsx_FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOru7rI&md5=a2759a4d6f613ae366dee15b8f66c2ed</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DTuyishime%26aufirst%3DM.%26aulast%3DDanish%26aufirst%3DM.%26aulast%3DPrinciotto%26aufirst%3DA.%26aulast%3DMankowski%26aufirst%3DM.%2BK.%26aulast%3DLawrence%26aufirst%3DR.%26aulast%3DLombart%26aufirst%3DH.%2BG.%26aulast%3DEsikov%26aufirst%3DK.%26aulast%3DBerniac%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DK.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DPtak%26aufirst%3DR.%2BG.%26aulast%3DMadani%26aufirst%3DN.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520novel%2520small-molecule%2520HIV-1%2520entry%2520inhibitors%2520using%2520field-based%2520virtual%2520screening%2520and%2520bioisosteric%2520replacement%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D5439%26epage%3D5445%26doi%3D10.1016%2Fj.bmcl.2014.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyzack, J. D.</span></span> <span> </span><span class="NLM_article-title">WhichP450: a multi-class categorical model to predict the major metabolising CYP450 isoform for a compound</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">537</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1007/s10822-018-0107-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1007%2Fs10822-018-0107-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=29464466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjtFCntr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=537-546&author=P.+A.+Huntauthor=M.+D.+Segallauthor=J.+D.+Tyzack&title=WhichP450%3A+a+multi-class+categorical+model+to+predict+the+major+metabolising+CYP450+isoform+for+a+compound&doi=10.1007%2Fs10822-018-0107-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-class categorical model to predict the major metabolising CYP450 isoform for a compound</span></div><div class="casAuthors">Hunt, Peter A.; Segall, Matthew D.; Tyzack, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">537-546</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">In the development of novel pharmaceuticals, the knowledge of how many, and which, Cytochrome P 450 isoforms are involved in the phase I metab. of a compd. is important.  Potential problems can arise if a compd. is metabolised predominantly by a single isoform in terms of drug-drug interactions or genetic polymorphisms that would lead to variations in exposure in the general population.  Combined with models of regioselectivities of metab. by each isoform, such a model would also aid in the prediction of the metabolites likely to be formed by P 450-mediated metab.  We describe the generation of a multi-class random forest model to predict which, out of a list of the seven leading Cytochrome P 450 isoforms, would be the major metabolising isoforms for a novel compd.  The model has a 76% success rate with a top-1 criterion and an 88% success rate for a top-2 criterion and shows significant enrichment over randomised models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouAXWrvgcNp7Vg90H21EOLACvtfcHk0lisaJD7Tsx_FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjtFCntr4%253D&md5=a66eb70996e0984998cc88baae426055</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2Fs10822-018-0107-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-018-0107-0%26sid%3Dliteratum%253Aachs%26aulast%3DHunt%26aufirst%3DP.%2BA.%26aulast%3DSegall%26aufirst%3DM.%2BD.%26aulast%3DTyzack%26aufirst%3DJ.%2BD.%26atitle%3DWhichP450%253A%2520a%2520multi-class%2520categorical%2520model%2520to%2520predict%2520the%2520major%2520metabolising%2520CYP450%2520isoform%2520for%2520a%2520compound%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2018%26volume%3D32%26spage%3D537%26epage%3D546%26doi%3D10.1007%2Fs10822-018-0107-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulos, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevrioukova, I. F.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Inhibitor Design for Evaluation of a CYP3A4 Pharmacophore Model</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4210</span>– <span class="NLM_lpage">4220</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01146</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01146" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2mtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4210-4220&author=P.+Kaurauthor=A.+R.+Chamberlinauthor=T.+L.+Poulosauthor=I.+F.+Sevrioukova&title=Structure-Based+Inhibitor+Design+for+Evaluation+of+a+CYP3A4+Pharmacophore+Model&doi=10.1021%2Facs.jmedchem.5b01146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Inhibitor Design for Evaluation of a CYP3A4 Pharmacophore Model</span></div><div class="casAuthors">Kaur, Parminder; Chamberlin, A. Richard; Poulos, Thomas L.; Sevrioukova, Irina F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4210-4220</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human cytochrome P 450 3A4 (CYP3A4) is a key xenobiotic-metabolizing enzyme that oxidizes and clears the majority of drugs.  CYP3A4 inhibition may lead to drug-drug interactions, toxicity, and other adverse effects but, in some cases, could be beneficial and enhance therapeutic efficiency of coadministered pharmaceuticals that are metabolized by CYP3A4.  On the basis of the investigations of analogs of ritonavir, a potent CYP3A4 inactivator and pharmacoenhancer, the authors have built a pharmacophore model for a CYP3A4-specific inhibitor.  This study is the first attempt to test this model using a set of rationally designed compds.  The functional and structural data presented here agree well with the proposed pharmacophore.  In particular, the authors confirmed the importance of a flexible backbone, the H-bond donor/acceptor moiety, and aromaticity of the side group analogous to Phe-2 of ritonavir and demonstrated the leading role of hydrophobic interactions at the sites adjacent to the heme and phenylalanine cluster in the ligand binding process.  The x-ray structures of CYP3A4 bound to the rationally designed inhibitors provide deeper insights into the mechanism of the CYP3A4-ligand interaction.  Most importantly, two of the compds. I and II that are less complex than ritonavir have comparable submicromolar affinity and inhibitory potency for CYP3A4 and, thus, could serve as templates for synthesis of second generation inhibitors for further evaluation and optimization of the pharmacophore model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2kZVZ3OeVr7Vg90H21EOLACvtfcHk0lisaJD7Tsx_FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2mtLvM&md5=4c9cd27c3a091aae985fcb70d7ffd957</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01146%26sid%3Dliteratum%253Aachs%26aulast%3DKaur%26aufirst%3DP.%26aulast%3DChamberlin%26aufirst%3DA.%2BR.%26aulast%3DPoulos%26aufirst%3DT.%2BL.%26aulast%3DSevrioukova%26aufirst%3DI.%2BF.%26atitle%3DStructure-Based%2520Inhibitor%2520Design%2520for%2520Evaluation%2520of%2520a%2520CYP3A4%2520Pharmacophore%2520Model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4210%26epage%3D4220%26doi%3D10.1021%2Facs.jmedchem.5b01146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span> <span> </span><span class="NLM_article-title">A semiempirical free energy force field with charge-based desolvation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1145</span>– <span class="NLM_lpage">1152</span>, <span class="refDoi"> DOI: 10.1002/jcc.20634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1002%2Fjcc.20634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=17274016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVSms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=1145-1152&author=R.+Hueyauthor=G.+M.+Morrisauthor=A.+J.+Olsonauthor=D.+S.+Goodsell&title=A+semiempirical+free+energy+force+field+with+charge-based+desolvation&doi=10.1002%2Fjcc.20634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A semiempirical free energy force field with charge-based desolvation</span></div><div class="casAuthors">Huey, Ruth; Morris, Garrett M.; Olson, Arthur J.; Goodsell, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1145-1152</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development and testing of a semiempirical free energy force field for use in AutoDock4 and similar grid-based docking methods.  The force field is based on a comprehensive thermodn. model that allows incorporation of intramol. energies into the predicted free energy of binding.  It also incorporates a charge-based method for evaluation of desolvation designed to use a typical set of atom types.  The method has been calibrated on a set of 188 diverse protein-ligand complexes of known structure and binding energy, and tested on a set of 100 complexes of ligands with retroviral proteases.  The force field shows improvement in redocking simulations over the previous AutoDock3 force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonvP0et7yst7Vg90H21EOLACvtfcHk0lhL_nrrBEwFQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVSms78%253D&md5=6fe6ef9bc0bd8149203a2b7351a2f15c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20634%26sid%3Dliteratum%253Aachs%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26atitle%3DA%2520semiempirical%2520free%2520energy%2520force%2520field%2520with%2520charge-based%2520desolvation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D1145%26epage%3D1152%26doi%3D10.1002%2Fjcc.20634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bikadi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazai, E.</span></span> <span> </span><span class="NLM_article-title">Application of the PM6 semi-empirical method to modeling proteins enhances docking accuracy of AutoDock</span>. <i>Aust. J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">15</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-1-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1186%2F1758-2946-1-15" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=15&author=Z.+Bikadiauthor=E.+Hazai&title=Application+of+the+PM6+semi-empirical+method+to+modeling+proteins+enhances+docking+accuracy+of+AutoDock&doi=10.1186%2F1758-2946-1-15"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-1-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-1-15%26sid%3Dliteratum%253Aachs%26aulast%3DBikadi%26aufirst%3DZ.%26aulast%3DHazai%26aufirst%3DE.%26atitle%3DApplication%2520of%2520the%2520PM6%2520semi-empirical%2520method%2520to%2520modeling%2520proteins%2520enhances%2520docking%2520accuracy%2520of%2520AutoDock%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D15%26doi%3D10.1186%2F1758-2946-1-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pornillos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganser-Pornillos, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banumathi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeager, M.</span></span> <span> </span><span class="NLM_article-title">Disulfide bond stabilization of the hexameric capsomer of human immunodeficiency virus</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>401</i></span>,  <span class="NLM_fpage">985</span>– <span class="NLM_lpage">995</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.06.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.jmb.2010.06.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=20600115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVaqs7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=401&publication_year=2010&pages=985-995&author=O.+Pornillosauthor=B.+K.+Ganser-Pornillosauthor=S.+Banumathiauthor=Y.+Huaauthor=M.+Yeager&title=Disulfide+bond+stabilization+of+the+hexameric+capsomer+of+human+immunodeficiency+virus&doi=10.1016%2Fj.jmb.2010.06.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Disulfide Bond Stabilization of the Hexameric Capsomer of Human Immunodeficiency Virus</span></div><div class="casAuthors">Pornillos, Owen; Ganser-Pornillos, Barbie K.; Banumathi, Sankaran; Hua, Yuanzi; Yeager, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">401</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">985-995</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The human immunodeficiency virus type 1 (HIV-1) capsid is modeled as a fullerene cone that is composed of ∼ 250 hexamers and 12 pentamers of the viral CA protein.  Structures of CA hexamers have been difficult to obtain because the hexamer-stabilizing interactions are inherently weak, and CA tends to spontaneously assemble into capsid-like particles.  Here, we describe a two-step biochem. strategy to obtain sol. CA hexamers for crystn.  First, the hexamer was stabilized by engineering disulfide cross-links (either A14C/E45C or A42C/T54C) between the N-terminal domains of adjacent subunits.  Second, the cross-linked hexamers were prevented from polymg. further into hyperstable capsid-like structures by mutations (W184A and M185A) that interfered with dimeric assocn. between the C-terminal domains that link adjacent hexamers.  The structures of two different cross-linked CA hexamers were nearly identical, and we combined the non-mutated portions of the structures to generate an at. resoln. model for the native hexamer.  This hybrid approach for structure detn. should be applicable to other viral capsomers and protein-protein complexes in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQVYZvCFtxWbVg90H21EOLACvtfcHk0lhL_nrrBEwFQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVaqs7jL&md5=43ce1eaed1b5aaa2032a37e2b6e44d5a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.06.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.06.042%26sid%3Dliteratum%253Aachs%26aulast%3DPornillos%26aufirst%3DO.%26aulast%3DGanser-Pornillos%26aufirst%3DB.%2BK.%26aulast%3DBanumathi%26aufirst%3DS.%26aulast%3DHua%26aufirst%3DY.%26aulast%3DYeager%26aufirst%3DM.%26atitle%3DDisulfide%2520bond%2520stabilization%2520of%2520the%2520hexameric%2520capsomer%2520of%2520human%2520immunodeficiency%2520virus%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D401%26spage%3D985%26epage%3D995%26doi%3D10.1016%2Fj.jmb.2010.06.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kortagere, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankowski, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships of a novel capsid targeted inhibitor of HIV-1 replication</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3080</span>– <span class="NLM_lpage">3090</span>, <span class="refDoi"> DOI: 10.1021/ci500437r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500437r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslemtrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3080-3090&author=S.+Kortagereauthor=J.+P.+Xuauthor=M.+K.+Mankowskiauthor=R.+G.+Ptakauthor=S.+Cocklin&title=Structure-activity+relationships+of+a+novel+capsid+targeted+inhibitor+of+HIV-1+replication&doi=10.1021%2Fci500437r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships of a Novel Capsid Targeted Inhibitor of HIV-1 Replication</span></div><div class="casAuthors">Kortagere, Sandhya; Xu, Jimmy P.; Mankowski, Marie K.; Ptak, Roger G.; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3080-3090</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the considerable successes of highly active antiretroviral therapy (HAART) for the treatment of HIV/AIDS, cumulative drug toxicities and the development of multidrug-resistant virus necessitate the search for new classes of antiretroviral agents with novel modes of action.  The HIV-1 capsid (CA) protein has been structurally and functionally characterized as a druggable target.  We have recently designed a novel small mol. inhibitor I-XW-053 using the hybrid structure based method to block the interface between CA N-terminal domains (NTD-NTD interface) with micromolar affinity.  In an effort to optimize and improve the efficacy of I-XW-053, we have developed the structure activity relationship of I-XW-053 compd. series using ligand efficiency methods.  Fifty-six analogs of I-XW-053 were designed that could be subclassified into four different core domains based on their ligand efficiency values computed as the ratio of binding efficiency (BEI) and surface efficiency (SEI) indexes.  Compd. 34 belonging to subcore-3 showed an 11-fold improvement over I-XW-053 in blocking HIV-1 replication in primary human peripheral blood mononuclear cells (PBMCs).  Surface plasmon resonance expts. confirmed the binding of compd. 34 to purified HIV-1 CA protein.  Mol. docking studies on compd. 34 and I-XW-053 to HIV-1 CA protein suggested that they both bind to NTD-NTD interface region but with different binding modes, which was further validated using site-directed mutagenesis studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokJa0411XKtbVg90H21EOLACvtfcHk0lhL_nrrBEwFQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslemtrnJ&md5=14c475975eaf073a56d42978f5e1390f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci500437r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500437r%26sid%3Dliteratum%253Aachs%26aulast%3DKortagere%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%2BP.%26aulast%3DMankowski%26aufirst%3DM.%2BK.%26aulast%3DPtak%26aufirst%3DR.%2BG.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DStructure-activity%2520relationships%2520of%2520a%2520novel%2520capsid%2520targeted%2520inhibitor%2520of%2520HIV-1%2520replication%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3080%26epage%3D3090%26doi%3D10.1021%2Fci500437r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M. D.</span></span> <span> </span><span class="NLM_article-title">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein–Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3036</span>– <span class="NLM_lpage">3050</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3036-3050&author=M.+R.+Bauerauthor=M.+D.+Mackey&title=Electrostatic+Complementarity+as+a+Fast+and+Effective+Tool+to+Optimize+Binding+and+Selectivity+of+Protein%E2%80%93Ligand+Complexes&doi=10.1021%2Facs.jmedchem.8b01925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein-Ligand Complexes</span></div><div class="casAuthors">Bauer, Matthias R.; Mackey, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3036-3050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Electrostatic interactions between small mols. and their resp. receptors are essential for mol. recognition and are also key contributors to the binding free energy.  Assessing the electrostatic match of protein-ligand complexes therefore provides important insights into why ligands bind and what can be changed to improve binding.  Ideally, the ligand and protein electrostatic potentials at the protein-ligand interaction interface should maximize their complementarity while minimizing desolvation penalties.  In this work, we present a fast and efficient tool to calc. and visualize the electrostatic complementarity (EC) of protein-ligand complexes.  We compiled benchmark sets demonstrating electrostatically driven structure-activity relationships (SAR) from literature data, including kinase, protein-protein interaction, and GPCR targets, and used these to demonstrate that the EC method can visualize, rationalize, and predict electrostatically driven ligand affinity changes and help to predict compd. selectivity.  The methodol. presented here for the anal. of EC is a powerful and versatile tool for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3pdUzJ9T0trVg90H21EOLACvtfcHk0lid6jLZr3YS6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D&md5=a1abb5abe7fa8d73bee264bc94fd2da1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01925%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DMackey%26aufirst%3DM.%2BD.%26atitle%3DElectrostatic%2520Complementarity%2520as%2520a%2520Fast%2520and%2520Effective%2520Tool%2520to%2520Optimize%2520Binding%2520and%2520Selectivity%2520of%2520Protein%25E2%2580%2593Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3036%26epage%3D3050%26doi%3D10.1021%2Facs.jmedchem.8b01925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rashad, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Kinetic Characterization of Novel HIV-1 Entry Inhibitors: Discovery of a Relationship between Off-Rate and Potency</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1940</span>, <span class="refDoi"> DOI: 10.3390/molecules23081940</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3390%2Fmolecules23081940" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGgtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1940&author=M.+E.+Meuserauthor=M.+B.+Murphyauthor=A.+A.+Rashadauthor=S.+Cocklin&title=Kinetic+Characterization+of+Novel+HIV-1+Entry+Inhibitors%3A+Discovery+of+a+Relationship+between+Off-Rate+and+Potency&doi=10.3390%2Fmolecules23081940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of novel HIV-1 entry inhibitors: discovery of a relationship between off-rate and potency</span></div><div class="casAuthors">Meuser, Megan E.; Murphy, Michael B.; Rashad, Adel A.; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1940/1-1940/16</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The entry of HIV-1 into permissible cells remains an extremely attractive and underexploited therapeutic intervention point.  We have previously demonstrated the ability to extend the chemotypes available for optimization in the entry inhibitor class using computational means.  Here, we continue this effort, designing and testing three novel compds. with the ability to inhibit HIV-1 entry.  We demonstrate that alteration of the core moiety of these entry inhibitors directly influences the potency of the compds., despite common proximal and distal groups.  Moreover, by establishing for the first time a surface plasmon resonance (SPR)-based interaction assay with sol. recombinant SOSIP Env trimers, we demonstrate that the off-rate (kd) parameter shows the strongest correlation with potency in an antiviral assay.  Finally, we establish an underappreciated relationship between the potency of a ligand and its degree of electrostatic complementarity (EC) with its target, the Env complex.  These findings not only broaden the chem. space in this inhibitor class, but also establish a rapid and simple assay to evaluate future HIV-1 entry inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPNtIXs0I2fLVg90H21EOLACvtfcHk0lid6jLZr3YS6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGgtb%252FI&md5=17312f8c72b2fb244a606d00aa47ab15</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23081940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23081940%26sid%3Dliteratum%253Aachs%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DMurphy%26aufirst%3DM.%2BB.%26aulast%3DRashad%26aufirst%3DA.%2BA.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DKinetic%2520Characterization%2520of%2520Novel%2520HIV-1%2520Entry%2520Inhibitors%253A%2520Discovery%2520of%2520a%2520Relationship%2520between%2520Off-Rate%2520and%2520Potency%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1940%26doi%3D10.3390%2Fmolecules23081940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickford, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irving, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciaramella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaacson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kjerrstrom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, S. L.</span></span> <span> </span><span class="NLM_article-title">HIV capsid is a tractable target for small molecule therapeutic intervention</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e1001220</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1001220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1001220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=21170360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252FltlGnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&author=W.+S.+Blairauthor=C.+Pickfordauthor=S.+L.+Irvingauthor=D.+G.+Brownauthor=M.+Andersonauthor=R.+Bazinauthor=J.+Caoauthor=G.+Ciaramellaauthor=J.+Isaacsonauthor=L.+Jacksonauthor=R.+Huntauthor=A.+Kjerrstromauthor=J.+A.+Niemanauthor=A.+K.+Patickauthor=M.+Perrosauthor=A.+D.+Scottauthor=K.+Whitbyauthor=H.+Wuauthor=S.+L.+Butler&title=HIV+capsid+is+a+tractable+target+for+small+molecule+therapeutic+intervention&doi=10.1371%2Fjournal.ppat.1001220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">HIV capsid is a tractable target for small molecule therapeutic intervention</span></div><div class="casAuthors">Blair Wade S; Pickford Chris; Irving Stephen L; Brown David G; Anderson Marie; Bazin Richard; Cao Joan; Ciaramella Giuseppe; Isaacson Jason; Jackson Lynn; Hunt Rachael; Kjerrstrom Anne; Nieman James A; Patick Amy K; Perros Manos; Scott Andrew D; Whitby Kevin; Wu Hua; Butler Scott L</div><div class="citationInfo"><span class="NLM_cas:title">PLoS pathogens</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1001220</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Despite a high current standard of care in antiretroviral therapy for HIV, multidrug-resistant strains continue to emerge, underscoring the need for additional novel mechanism inhibitors that will offer expanded therapeutic options in the clinic.  We report a new class of small molecule antiretroviral compounds that directly target HIV-1 capsid (CA) via a novel mechanism of action.  The compounds exhibit potent antiviral activity against HIV-1 laboratory strains, clinical isolates, and HIV-2, and inhibit both early and late events in the viral replication cycle.  We present mechanistic studies indicating that these early and late activities result from the compound affecting viral uncoating and assembly, respectively.  We show that amino acid substitutions in the N-terminal domain of HIV-1 CA are sufficient to confer resistance to this class of compounds, identifying CA as the target in infected cells.  A high-resolution co-crystal structure of the compound bound to HIV-1 CA reveals a novel binding pocket in the N-terminal domain of the protein.  Our data demonstrate that broad-spectrum antiviral activity can be achieved by targeting this new binding site and reveal HIV CA as a tractable drug target for HIV therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRN3Dm4etvdX1MGsG_HTXYFfW6udTcc2eZt_OAT5B2VErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252FltlGnsA%253D%253D&md5=4b133434dd66f16f983e3856f3d6b07b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1001220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1001220%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DW.%2BS.%26aulast%3DPickford%26aufirst%3DC.%26aulast%3DIrving%26aufirst%3DS.%2BL.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DAnderson%26aufirst%3DM.%26aulast%3DBazin%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DCiaramella%26aufirst%3DG.%26aulast%3DIsaacson%26aufirst%3DJ.%26aulast%3DJackson%26aufirst%3DL.%26aulast%3DHunt%26aufirst%3DR.%26aulast%3DKjerrstrom%26aufirst%3DA.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DPerros%26aufirst%3DM.%26aulast%3DScott%26aufirst%3DA.%2BD.%26aulast%3DWhitby%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DButler%26aufirst%3DS.%2BL.%26atitle%3DHIV%2520capsid%2520is%2520a%2520tractable%2520target%2520for%2520small%2520molecule%2520therapeutic%2520intervention%26jtitle%3DPLoS%2520Pathog.%26date%3D2010%26volume%3D6%26doi%3D10.1371%2Fjournal.ppat.1001220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halambage, U. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burse, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiken, C.</span></span> <span> </span><span class="NLM_article-title">Compensatory substitutions in the HIV-1 capsid reduce the fitness cost associated with resistance to a capsid-targeting small-molecule inhibitor</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1128/JVI.01411-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.01411-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=25320302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2qur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=208-219&author=J.+Shiauthor=J.+Zhouauthor=U.+D.+Halambageauthor=V.+B.+Shahauthor=M.+J.+Burseauthor=H.+Wuauthor=W.+S.+Blairauthor=S.+L.+Butlerauthor=C.+Aiken&title=Compensatory+substitutions+in+the+HIV-1+capsid+reduce+the+fitness+cost+associated+with+resistance+to+a+capsid-targeting+small-molecule+inhibitor&doi=10.1128%2FJVI.01411-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Compensatory substitutions in the HIV-1 capsid reduce the fitness cost associated with resistance to a capsid-targeting small-molecule inhibitor</span></div><div class="casAuthors">Shi, Jiong; Zhou, Jing; Halambage, Upul D.; Shah, Vaibhav B.; Burse, Mallori J.; Wu, Hua; Blair, Wade S.; Butler, Scott L.; Aiken, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">208-219, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The HIV-1 capsid plays multiple roles in infection and is an emerging therapeutic target.  The small-mol. HIV-1 inhibitor PF-3450074 (PF74) blocks HIV-1 at an early postentry stage by binding the viral capsid and interfering with its function.  Selection for resistance resulted in accumulation of five amino acid changes in the viral CA protein, which collectively reduced binding of the compd. to HIV-1 particles.  In the present study, we dissected the individual and combinatorial contributions of each of the five substitutions Q67H, K70R, H87P, T107N, and L111I to PF74 resistance, PF74 binding, and HIV-1 infectivity.  Q67H, K70R, and T107N each conferred low-level resistance to PF74 and collectively conferred strong resistance.  The substitutions K70R and L111I impaired HIV-1 infectivity, which was partially restored by the other substitutions at positions 67 and 107.  PF74 binding to HIV-1 particles was reduced by the Q67H, K70R, and T107N substitutions, consistent with the location of these positions in the inhibitor-binding pocket.  Replication of the 5Mut virus was markedly impaired in cultured macrophages, reminiscent of the previously reported N74D CA mutant. 5Mut substitutions also reduced the binding of the host protein CPSF6 to assembled CA complexes in vitro and permitted infection of cells expressing the inhibitory protein CPSF6-358.  Our results demonstrate that strong resistance to PF74 requires accumulation of multiple substitutions in CA to inhibit PF74 binding and compensate for fitness impairments assocd. with some of the sequence changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6wr7ky5b0U7Vg90H21EOLACvtfcHk0liJu2gk08l3wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2qur0%253D&md5=ede0246cb047a706caa5584d6324d2e3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1128%2FJVI.01411-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01411-14%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHalambage%26aufirst%3DU.%2BD.%26aulast%3DShah%26aufirst%3DV.%2BB.%26aulast%3DBurse%26aufirst%3DM.%2BJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DBlair%26aufirst%3DW.%2BS.%26aulast%3DButler%26aufirst%3DS.%2BL.%26aulast%3DAiken%26aufirst%3DC.%26atitle%3DCompensatory%2520substitutions%2520in%2520the%2520HIV-1%2520capsid%2520reduce%2520the%2520fitness%2520cost%2520associated%2520with%2520resistance%2520to%2520a%2520capsid-targeting%2520small-molecule%2520inhibitor%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D208%26epage%3D219%26doi%3D10.1128%2FJVI.01411-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham-Huy, C.</span></span> <span> </span><span class="NLM_article-title">Chiral drugs: an overview</span>. <i>Int. J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">100</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=23674971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlSltrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=85-100&author=L.+A.+Nguyenauthor=H.+Heauthor=C.+Pham-Huy&title=Chiral+drugs%3A+an+overview"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral drugs. An overview</span></div><div class="casAuthors">Nguyen, Lien Ai; He, Hua; Chuong, Pham-Huy</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biomedical Science (Monterey Park, CA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-100</span>CODEN:
                <span class="NLM_cas:coden">IJBSDB</span>;
        ISSN:<span class="NLM_cas:issn">1550-9702</span>.
    
            (<span class="NLM_cas:orgname">Master Publishing Group</span>)
        </div><div class="casAbstract">A review.  About more than half of the drugs currently in use are chiral compds. and near 90% of the last ones are marketed as racemates consisting of an equimolar mixt. of two enantiomers.  Although they have the same chem. structure, most isomers of chiral drugs exhibit marked differences in biol. activities such as pharmacol., toxicol., pharmacokinetics, metab. etc.  Some mechanisms of these properties are also explained.  Therefore, it is important to promote the chiral sepn. and anal. of racemic drugs in pharmaceutical industry as well as in clinic in order to eliminate the unwanted isomer from the prepn. and to find an optimal treatment and a right therapeutic control for the patient.  In this article, we review the nomenclature, pharmacol., toxicol., pharmacokinetics, metab. etc of some usual chiral drugs as well as their mechanisms.  Different techniques used for the chiral sepn. in pharmaceutical industry as well as in clin. analyses are also examd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovoqsLM_3bxLVg90H21EOLACvtfcHk0liJu2gk08l3wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlSltrnJ&md5=254cbca060bb4cfd04eac365488f5b62</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DL.%2BA.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DPham-Huy%26aufirst%3DC.%26atitle%3DChiral%2520drugs%253A%2520an%2520overview%26jtitle%3DInt.%2520J.%2520Biomed.%2520Sci.%26date%3D2006%26volume%3D2%26spage%3D85%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.-k.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span> <span> </span><span class="NLM_article-title">Chiral kinase inhibitors</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">800</span>– <span class="NLM_lpage">809</span>, <span class="refDoi"> DOI: 10.2174/156802611795165052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.2174%2F156802611795165052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=21291394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3MXls1Ojsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=800-809&author=J.-k.+Jiangauthor=M.+Shenauthor=C.+J.+Thomasauthor=M.+B.+Boxer&title=Chiral+kinase+inhibitors&doi=10.2174%2F156802611795165052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral kinase inhibitors</span></div><div class="casAuthors">Jiang, Jian-kang; Shen, Min; Thomas, Craig J.; Boxer, Mathew B.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">800-809</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Small mol. kinase inhibitors are important tools for studying cellular signaling pathways, phenotypes and are, occasionally, useful clin. agents.  With stereochem. pervasive throughout the mols. of life it is no surprise that a single stereocenter can bestow a ligand with distinct binding affinities to various protein targets.  While the majority of small mol. kinase inhibitors reported to date are achiral, a no. of asym. compds. show great utility as tools for probing kinase-assocd. biomol. events as well as promising therapeutic leads.  The mechanism by which chirality is introduced varies but includes screening of chiral libraries, incorporation of chiral centers during optimization efforts and the rational installation of a chiral moiety as guided by structural and modeling efforts.  Here we discuss several advanced chiral small mol. kinase inhibitors where stereochem. plays an important role in terms of potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNwyVAiHC8JrVg90H21EOLACvtfcHk0liJu2gk08l3wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXls1Ojsrs%253D&md5=e838981d7aea95bcca374b10706fa2f3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F156802611795165052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611795165052%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DJ.-k.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26atitle%3DChiral%2520kinase%2520inhibitors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D800%26epage%3D809%26doi%3D10.2174%2F156802611795165052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallegos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttamsingh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miwa, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balani, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, L. S.</span></span> <span> </span><span class="NLM_article-title">Comparison of intrinsic clearance in liver microsomes and hepatocytes from rats and humans: evaluation of free fraction and uptake in hepatocytes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1600</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1124/dmd.106.010793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1124%2Fdmd.106.010793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=16790553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1SksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=1600-1605&author=C.+Luauthor=P.+Liauthor=R.+Gallegosauthor=V.+Uttamsinghauthor=C.+Q.+Xiaauthor=G.+T.+Miwaauthor=S.+K.+Balaniauthor=L.+S.+Gan&title=Comparison+of+intrinsic+clearance+in+liver+microsomes+and+hepatocytes+from+rats+and+humans%3A+evaluation+of+free+fraction+and+uptake+in+hepatocytes&doi=10.1124%2Fdmd.106.010793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of intrinsic clearance in liver microsomes and hepatocytes from rats and humans: evaluation of free fraction and uptake in hepatocytes</span></div><div class="casAuthors">Lu, Chuang; Li, Ping; Gallegos, Richard; Uttamsingh, Vinita; Xia, Cindy Q.; Miwa, Gerald T.; Balani, Suresh K.; Gan, Liang-Shang</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1600-1605</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Apparent intrinsic clearance (CLint,app) of 7-ethoxycoumarin, phenacetin, propranolol, and midazolam was measured using rat and human liver microsomes and freshly isolated and cryopreserved hepatocytes to det. factors responsible for differences in rates of metab. in these systems.  The cryopreserved and freshly isolated hepatocytes generally provided similar results, although there was greater variability using the latter system.  The CLint,app values in hepatocytes are obsd. to be lower than that in microsomes, and this difference becomes greater for compds. with high CLint,app.  This could partly be attributed to the differences in the free fraction (fu).  The fu in hepatocyte incubations (fu,hep-inc) was influenced not only by the free fraction of compds. in the incubation buffer (fu,buffer) but also by the rate consts. of uptake (kup) and metab. (kmet).  This report provides a new derivation for fu,hep-inc, which can be expressed as fu,hep-inc = [kup/(kmet + kup)]/[1 + (Chep/Cbuffer) × (Vhep/Vbuffer)], where the Chep, Cbuffer, Vhep, and Vbuffer represent the concns. of a compd. in hepatocytes and buffer and vols. of hepatocytes and buffer, resp.  For midazolam, the fu,hep-inc was calcd., and the max. metab. rate in hepatocytes was shown to be limited by the uptake rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkV9lSFxDFT7Vg90H21EOLACvtfcHk0lhSWTDmzoxSrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1SksLk%253D&md5=3d3ff22990742b26dc24c84a6aab720e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1124%2Fdmd.106.010793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.106.010793%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DGallegos%26aufirst%3DR.%26aulast%3DUttamsingh%26aufirst%3DV.%26aulast%3DXia%26aufirst%3DC.%2BQ.%26aulast%3DMiwa%26aufirst%3DG.%2BT.%26aulast%3DBalani%26aufirst%3DS.%2BK.%26aulast%3DGan%26aufirst%3DL.%2BS.%26atitle%3DComparison%2520of%2520intrinsic%2520clearance%2520in%2520liver%2520microsomes%2520and%2520hepatocytes%2520from%2520rats%2520and%2520humans%253A%2520evaluation%2520of%2520free%2520fraction%2520and%2520uptake%2520in%2520hepatocytes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2006%26volume%3D34%26spage%3D1600%26epage%3D1605%26doi%3D10.1124%2Fdmd.106.010793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Björndal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiore, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colognesi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarlatti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenyö, E. M.</span></span> <span> </span><span class="NLM_article-title">Coreceptor usage of primary human immunodeficiency virus type 1 isolates varies according to biological phenotype</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">7478</span>– <span class="NLM_lpage">7487</span>, <span class="refDoi"> DOI: 10.1128/JVI.71.10.7478-7487.1997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.71.10.7478-7487.1997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=9311827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADyaK2sXmt1Omsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=1997&pages=7478-7487&author=A.+Bj%C3%B6rndalauthor=H.+Dengauthor=M.+Janssonauthor=J.+R.+Fioreauthor=C.+Colognesiauthor=A.+Karlssonauthor=J.+Albertauthor=G.+Scarlattiauthor=D.+R.+Littmanauthor=E.+M.+Feny%C3%B6&title=Coreceptor+usage+of+primary+human+immunodeficiency+virus+type+1+isolates+varies+according+to+biological+phenotype&doi=10.1128%2FJVI.71.10.7478-7487.1997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Coreceptor usage of primary human immunodeficiency virus type 1 isolates varies according to biological phenotype</span></div><div class="casAuthors">Bjorndal, Asa; Deng, Hongkui; Jansson, Marianne; Fiore, Jose R.; Colognesi, Claudia; Karlsson, Anders; Albert, Jan; Scarlatti, Gabriella; Littman, Dan R.; Fenyo, Eva Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7478-7487</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The biol. phenotype of primary human immunodeficiency virus type 1 (HIV-1) isolates varies according to the severity of the HIV infection.  Here the authors show that the two previously described groups of rapid/high, syncytium-inducing (SI) and slow/low, non-syncytium-inducing (NSI) isolates are distinguished by their ability to utilize different chemokine receptors for entry into target cells.  Recent studies have identified the C-X-C chemokine receptor CXCR4 (also named fusin or Lestr) and the C-C chemokine receptor CCR5 as the principal entry cofactors for T-cell-line-tropic and non-T-cell-line-tropic HIV-1, resp.  Using U87.CD4 glioma cell lines, stably expressing the chemokine receptor CCR1, CCR2b, CCR3, CCR5, or CXCR4, the authors have tested chemokine receptor specificity for a panel of genetically diverse envelope glycoprotein genes cloned from primary HIV-1 isolates and have found that receptor usage was closely assocd. with the biol. phenotype of the virus isolate but not the genetic subtype.  They have also analyzed a panel of 36 well-characterized primary HIV-1 isolates for syncytium induction and replication in the same series of cell lines.  Infection by slow/low viruses was restricted to cells expressing CCR5, whereas rapid/high viruses could use a variety of chemokine receptors.  In addn. to the regular use of CXCR4, many rapid/high viruses used CCR5 and some also used CCR3 and CCR2b.  Progressive HIV-1 infection is characterized by the emergence of viruses resistant to inhibition by β-chemokines, which corresponded to changes in coreceptor usage.  The broadening of the host range may even enable the use of uncharacterized coreceptors, in that two isolates from immunodeficient patients infected the parental U87.CD4 cell line lacking any engineered coreceptor.  Two primary isolates with multiple coreceptor usage were shown to consist of mixed populations, one with a narrow host range using CCR5 only and the other with a broad host range using CCR3, CCR5, or CXCR4, similar to the original population.  The results show that all 36 primary HIV-1 isolates induce syncytia, provided that target cells carry the particular coreceptor required by the virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xCNHpx6JQrVg90H21EOLACvtfcHk0lhSWTDmzoxSrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt1Omsr4%253D&md5=9c509d74aaeb47df187b01a3d874ec48</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FJVI.71.10.7478-7487.1997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.71.10.7478-7487.1997%26sid%3Dliteratum%253Aachs%26aulast%3DBj%25C3%25B6rndal%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DJansson%26aufirst%3DM.%26aulast%3DFiore%26aufirst%3DJ.%2BR.%26aulast%3DColognesi%26aufirst%3DC.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DAlbert%26aufirst%3DJ.%26aulast%3DScarlatti%26aufirst%3DG.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26aulast%3DFeny%25C3%25B6%26aufirst%3DE.%2BM.%26atitle%3DCoreceptor%2520usage%2520of%2520primary%2520human%2520immunodeficiency%2520virus%2520type%25201%2520isolates%2520varies%2520according%2520to%2520biological%2520phenotype%26jtitle%3DJ.%2520Virol.%26date%3D1997%26volume%3D71%26spage%3D7478%26epage%3D7487%26doi%3D10.1128%2FJVI.71.10.7478-7487.1997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zentner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sierra, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciunas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankowski, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinnik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedichev, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Garcia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Identification of a small-molecule inhibitor of HIV-1 assembly that targets the phosphatidylinositol (4,5)-bisphosphate binding site of the HIV-1 matrix protein</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">426</span>– <span class="NLM_lpage">432</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201200577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1002%2Fcmdc.201200577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=23361947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGiurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=426-432&author=I.+Zentnerauthor=L.-J.+Sierraauthor=A.+K.+Fraserauthor=L.+Maciunasauthor=M.+K.+Mankowskiauthor=A.+Vinnikauthor=P.+Fedichevauthor=R.+G.+Ptakauthor=J.+Martin-Garciaauthor=S.+Cocklin&title=Identification+of+a+small-molecule+inhibitor+of+HIV-1+assembly+that+targets+the+phosphatidylinositol+%284%2C5%29-bisphosphate+binding+site+of+the+HIV-1+matrix+protein&doi=10.1002%2Fcmdc.201200577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Small-Molecule Inhibitor of HIV-1 Assembly that Targets the Phosphatidylinositol (4,5)-bisphosphate Binding Site of the HIV-1 Matrix Protein</span></div><div class="casAuthors">Zentner, Isaac; Sierra, Luz-Jeannette; Fraser, Ayesha K.; Maciunas, Lina; Mankowski, Marie K.; Vinnik, Andrei; Fedichev, Peter; Ptak, Roger G.; Martin-Garcia, Julio; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">426-432</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of drug resistance remains a crit. problem for current HIV-1 antiviral therapies, creating a need for new inhibitors of HIV-1 replication.  We previously reported on a novel anti-HIV-1 compd., N2-(phenoxyacetyl)-N-[4-(1-piperidinylcarbonyl)benzyl]glycinamide (14), that binds to the highly conserved phosphatidylinositol (4,5)-bisphosphate (PI(4,5)P2) binding pocket of the HIV-1 matrix (MA) protein.  In this study, we re-evaluate the hits from the virtual screen used to identify compd. 14 and test them directly in an HIV-1 replication assay using primary human peripheral blood mononuclear cells.  This study resulted in the identification of three new compds. with antiviral activity; 2-(4-{[3-(4-fluorophenyl)-1,2,4-oxadiazol-5-yl]methyl})-1-piperazinyl)-N-(4-methylphenyl)acetamide (7), 3-(2-ethoxyphenyl)-5-[[4-(4-nitrophenyl)piperazin-1-yl]methyl]-1,2,4-oxadiazole (17), and N-[4-ethoxy-3-(1-piperidinylsulfonyl)phenyl]-2-(imidazo[2,1-b][1,3]thiazol-6-yl)acetamide (18), with compd. 7 being the most potent of these hits.  Mechanistic studies on 7 demonstrated that it directly interacts with and functions through HIV-1 MA.  In accordance with our drug target, compd. 7 competes with PI(4,5)P2 for MA binding and, as a result, diminishes the prodn. of new virus.  Mutation of residues within the PI(4,5)P2 binding site of MA decreased the antiviral effect of compd. 7.  Addnl., compd. 7 displays a broadly neutralizing anti-HIV activity, with IC50 values of 7.5-15.6 μM for the group M isolates tested.  Taken together, these results point towards a novel chem. probe that can be used to more closely study the biol. role of MA and could, through further optimization, lead to a new class of anti-HIV-1 therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr11-gHfqPpe7Vg90H21EOLACvtfcHk0lhSWTDmzoxSrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGiurY%253D&md5=19c5f1741bbf8c4bd87716021fa20e19</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200577%26sid%3Dliteratum%253Aachs%26aulast%3DZentner%26aufirst%3DI.%26aulast%3DSierra%26aufirst%3DL.-J.%26aulast%3DFraser%26aufirst%3DA.%2BK.%26aulast%3DMaciunas%26aufirst%3DL.%26aulast%3DMankowski%26aufirst%3DM.%2BK.%26aulast%3DVinnik%26aufirst%3DA.%26aulast%3DFedichev%26aufirst%3DP.%26aulast%3DPtak%26aufirst%3DR.%2BG.%26aulast%3DMartin-Garcia%26aufirst%3DJ.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520a%2520small-molecule%2520inhibitor%2520of%2520HIV-1%2520assembly%2520that%2520targets%2520the%2520phosphatidylinositol%2520%25284%252C5%2529-bisphosphate%2520binding%2520site%2520of%2520the%2520HIV-1%2520matrix%2520protein%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D426%26epage%3D432%26doi%3D10.1002%2Fcmdc.201200577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kortagere, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankowski, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zentner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Princiotto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sierra, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passic, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stavale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krebs, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Garcia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodroski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. B.,  III</span></span> <span> </span><span class="NLM_article-title">Inhibiting early-stage events in HIV-1 replication by small-molecule targeting of the HIV-1 capsid</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">8472</span>– <span class="NLM_lpage">8481</span>, <span class="refDoi"> DOI: 10.1128/JVI.05006-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.05006-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=22647699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2jtLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=8472-8481&author=S.+Kortagereauthor=N.+Madaniauthor=M.+K.+Mankowskiauthor=A.+Schonauthor=I.+Zentnerauthor=G.+Swaminathanauthor=A.+Princiottoauthor=K.+Anthonyauthor=A.+Ozaauthor=L.+J.+Sierraauthor=S.+R.+Passicauthor=X.+Wangauthor=D.+M.+Jonesauthor=E.+Stavaleauthor=F.+C.+Krebsauthor=J.+Martin-Garciaauthor=E.+Freireauthor=R.+G.+Ptakauthor=J.+Sodroskiauthor=S.+Cocklinauthor=A.+B.+Smith&title=Inhibiting+early-stage+events+in+HIV-1+replication+by+small-molecule+targeting+of+the+HIV-1+capsid&doi=10.1128%2FJVI.05006-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting early-stage events in HIV-1 replication by small-molecule targeting of the HIV-1 capsid</span></div><div class="casAuthors">Kortagere, Sandhya; Madani, Navid; Mankowski, Marie K.; Schon, Arne; Zentner, Isaac; Swaminathan, Gokul; Princiotto, Amy; Anthony, Kevin; Oza, Apara; Sierra, Luz-Jeannette; Passic, Shendra R.; Wang, Xiaozhao; Jones, David M.; Stavale, Eric; Krebs, Fred C.; Martin-Garcia, Julio; Freire, Ernesto; Ptak, Roger G.; Sodroski, Joseph; Cocklin, Simon; Smith, Amos B., III</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8472-8481</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The HIV-1 capsid (CA) protein plays essential roles in both early and late stages of virl replication and has emerged as a novel drug target.  We report hybrid structure-based virtual screening to identify small mols. with the potential to interact with the N-terminal domain (NTD) of HIV-1 CA and disrupt early, preintegration steps of the HIV-1 replication cycle.  The small mol. 4,4'-[dibenzo[b,d]furan-2,8-diylbis(5-phenyl-1H-imidazole-4,2-diyl)]dibenzoic acid (CK026), which had anti-HIV-1 activity in single- and multiple-round infections but failed to inhibit viral replication in peripheral blood mononuclear cells (PBMCs), was identified.  Three analogs of CK026 with reduced size and better drug-like properties were synthesized and assessed.  Compd. I-XW-053 (4-(4,5-diphenyl-1H-imidazol-2-yl)benzoic acid) retained all of the antiviral activity of the parental compd. and inhibited the replication of a diverse panel of primary HIV-1 isolates in PBMCs, while displaying no appreciable cytotoxicity.  This antiviral activity was specific to HIV-1, as I-XW-053 displayed no effect on the replication of SIV or against a panel of nonretroviruses.  Direct interaction of I-XW-053 was quantified with wild-type and mutant CA protein using surface plasmon resonance and isothermal titrn. calorimetry.  Mutation of Ile37 and Arg173, which are required for interaction with compd. I-XW-053, crippled the virus at an early, preintegration step.  Using quant. PCR, we demonstrated that treatment with I-XW-053 inhibited HIV-1 reverse transcription in multiple cell types, indirectly pointing to dysfunction in the uncoating process.  In summary, we have identified a CA-specific compd. that targets and inhibits a novel region in the NTD-NTD interface, affects uncoating, and possesses broad-spectrum anti-HIV-1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Tdf1mUZ85bVg90H21EOLACvtfcHk0ljXUmp2dqjCoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2jtLzE&md5=324cf8cda7a0aa6e07b5b8e529c91aaf</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1128%2FJVI.05006-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.05006-11%26sid%3Dliteratum%253Aachs%26aulast%3DKortagere%26aufirst%3DS.%26aulast%3DMadani%26aufirst%3DN.%26aulast%3DMankowski%26aufirst%3DM.%2BK.%26aulast%3DSchon%26aufirst%3DA.%26aulast%3DZentner%26aufirst%3DI.%26aulast%3DSwaminathan%26aufirst%3DG.%26aulast%3DPrinciotto%26aufirst%3DA.%26aulast%3DAnthony%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DA.%26aulast%3DSierra%26aufirst%3DL.%2BJ.%26aulast%3DPassic%26aufirst%3DS.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DJones%26aufirst%3DD.%2BM.%26aulast%3DStavale%26aufirst%3DE.%26aulast%3DKrebs%26aufirst%3DF.%2BC.%26aulast%3DMartin-Garcia%26aufirst%3DJ.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DPtak%26aufirst%3DR.%2BG.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26atitle%3DInhibiting%2520early-stage%2520events%2520in%2520HIV-1%2520replication%2520by%2520small-molecule%2520targeting%2520of%2520the%2520HIV-1%2520capsid%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D8472%26epage%3D8481%26doi%3D10.1128%2FJVI.05006-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Studier, F. W.</span></span> <span> </span><span class="NLM_article-title">Protein production by auto-induction in high density shaking cultures</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1016/j.pep.2005.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.pep.2005.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=15915565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=207-234&author=F.+W.+Studier&title=Protein+production+by+auto-induction+in+high+density+shaking+cultures&doi=10.1016%2Fj.pep.2005.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Protein production by auto-induction in high-density shaking cultures</span></div><div class="casAuthors">Studier, F. William</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-234</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Inducible expression systems in which T7 RNA polymerase transcribes coding sequences cloned under control of a T7lac promoter efficiently produce a wide variety of proteins in Escherichia coli.  Investigation of factors that affect stability, growth, and induction of T7 expression strains in shaking vessels led to the recognition that sporadic, unintended induction of expression in complex media, previously reported by others, is almost certainly caused by small amts. of lactose.  Glucose prevents induction by lactose by well-studied mechanisms.  Amino acids also inhibit induction by lactose during log-phase growth, and high rates of aeration inhibit induction at low lactose concns.  These observations, and metabolic balancing of pH, allowed development of reliable non-inducing and auto-inducing media in which batch cultures grow to high densities.  Expression strains grown to satn. in non-inducing media retain plasmid and remain fully viable for weeks in the refrigerator, making it easy to prep. many freezer stocks in parallel and use working stocks for an extended period.  Auto-induction allows efficient screening of many clones in parallel for expression and soly., as cultures have only to be inoculated and grown to satn., and yields of target protein are typically several-fold higher than obtained by conventional IPTG induction.  Auto-inducing media have been developed for labeling proteins with selenomethionine, 15N or 13C, and for prodn. of target proteins by arabinose induction of T7 RNA polymerase from the pBAD promoter in BL21-AI.  Selenomethionine labeling was equally efficient in the commonly used methionine auxotroph B834(DE3)(found to be metE) or the prototroph BL21(DE3).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYNqgn15xP_bVg90H21EOLACvtfcHk0ljXUmp2dqjCoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D&md5=d31b76747c808ce6fbb27c7a1192b6b8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2005.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2005.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DStudier%26aufirst%3DF.%2BW.%26atitle%3DProtein%2520production%2520by%2520auto-induction%2520in%2520high%2520density%2520shaking%2520cultures%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2005%26volume%3D41%26spage%3D207%26epage%3D234%26doi%3D10.1016%2Fj.pep.2005.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connor, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, N. R.</span></span> <span> </span><span class="NLM_article-title">Vpr is required for efficient replication of human immunodeficiency virus type-1 in mononuclear phagocytes</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>206</i></span>,  <span class="NLM_fpage">935</span>– <span class="NLM_lpage">944</span>, <span class="refDoi"> DOI: 10.1006/viro.1995.1016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1006%2Fviro.1995.1016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=7531918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADyaK2MXjs1WkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=206&publication_year=1995&pages=935-944&author=R.+I.+Connorauthor=B.+K.+Chenauthor=S.+Choeauthor=N.+R.+Landau&title=Vpr+is+required+for+efficient+replication+of+human+immunodeficiency+virus+type-1+in+mononuclear+phagocytes&doi=10.1006%2Fviro.1995.1016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Vpr is required for efficient replication of human immunodeficiency virus type-1 in mononuclear phagocytes</span></div><div class="casAuthors">Connor, Ruth I.; Chen, Benjamin Kuan; Choe, Sunny; Landau, Nathaniel R.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">206</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">935-44</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">HIV-1 vpr encodes a 96-amino acid, nuclear protein whose function is not well understood.  Unlike the other lentivirus regulatory proteins, Vpr is present in virions at relatively high copy no.  In cells, Vpr is localized to the nucleus.  Possible functions for vpr consistent with these findings include the nuclear import of preintegration complexes, transactivation of cellular genes, or induction of cellular differentiation.  The authors show here, using both replication competent, macrophage-tropic virus and a sensitive, single-cycle luciferase HIV-1 reporter vector, that vpr is important for efficient viral replication in primary monocyte/macrophages, but appears to play no role in activated or resting T cell infection.  The block to infection in monocytes was localized by PCR anal. of newly synthesized viral DNA and with the luciferase reporter vector to a stage in the viral life cycle after entry and reverse transcription, yet prior to, or at the time of, proviral transcription.  In addn., infection of mononuclear phagocyte with virions that had been loaded with Vpr mols. in the producer cells by trans-complementation still showed a vpr-phenotype.  These data suggest a role for vpr mols. produced in newly infected cells, in addn. to its presumed function in the virion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgyuTZ6ZIJ-7Vg90H21EOLACvtfcHk0ljXUmp2dqjCoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjs1WkurY%253D&md5=09620dfc7aaf57ec9c46ffaef438630f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1006%2Fviro.1995.1016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fviro.1995.1016%26sid%3Dliteratum%253Aachs%26aulast%3DConnor%26aufirst%3DR.%2BI.%26aulast%3DChen%26aufirst%3DB.%2BK.%26aulast%3DChoe%26aufirst%3DS.%26aulast%3DLandau%26aufirst%3DN.%2BR.%26atitle%3DVpr%2520is%2520required%2520for%2520efficient%2520replication%2520of%2520human%2520immunodeficiency%2520virus%2520type-1%2520in%2520mononuclear%2520phagocytes%26jtitle%3DVirology%26date%3D1995%26volume%3D206%26spage%3D935%26epage%3D944%26doi%3D10.1006%2Fviro.1995.1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosakovsky Pond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeStefano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phogat, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poignard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fling, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaBranche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montefiori, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koff, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parks, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caulfield, M. J.</span></span> <span> </span><span class="NLM_article-title">Identification of an HIV-1 clade A envelope that exhibits broad antigenicity and neutralization sensitivity and elicits antibodies targeting three distinct epitopes</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">5372</span>– <span class="NLM_lpage">5383</span>, <span class="refDoi"> DOI: 10.1128/JVI.02827-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.02827-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=23468492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsV2lsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=5372-5383&author=S.+Hoffenbergauthor=R.+Powellauthor=A.+Carpovauthor=D.+Wagnerauthor=A.+Wilsonauthor=S.+Kosakovsky+Pondauthor=R.+Lindsayauthor=H.+Arendtauthor=J.+DeStefanoauthor=S.+Phogatauthor=P.+Poignardauthor=S.+P.+Flingauthor=M.+Simekauthor=C.+LaBrancheauthor=D.+Montefioriauthor=T.+Wrinauthor=P.+Phungauthor=D.+Burtonauthor=W.+Koffauthor=C.+R.+Kingauthor=C.+L.+Parksauthor=M.+J.+Caulfield&title=Identification+of+an+HIV-1+clade+A+envelope+that+exhibits+broad+antigenicity+and+neutralization+sensitivity+and+elicits+antibodies+targeting+three+distinct+epitopes&doi=10.1128%2FJVI.02827-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an HIV-1 clad A envelope that exhibits broad antigenicity and neutralization sensitivity and elicits antibodies targeting three distinct epitopes</span></div><div class="casAuthors">Hoffenberg, Simon; Powell, Rebecca; Carpov, Alexei; Wagner, Denise; Wilson, Aaron; Pond, Sergei Kosakovsky; Lindsay, Ross; Arendt, Heather; DeStefano, Joanne; Phogat, Sanjay; Polgnard, Pascal; Fling, Steven P.; Simek, Melissa; LaBranche, Celia; Montefiori, David; Wrin, Terri; Phung, Pham; Burton, Dennis; Koff, Wayne; King, C. Richter; Parks, Christopher L.; Caulfield, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5372-5383</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Broadly neutralizing antibodies (bNAbs) PG9 and PG16 were isolated from an International AIDS Vaccine Initiative (IAVI) Protocol G subject infected with human immunodeficiency virus type 1 (HIV-1) clade A.  Both antibodies are highly potent and neutralize greater than 70% of viruses tested.  We sought to begin immunogen design based on viral sequences from this patient; however, pseudoviruses prepd. with 19 envelope sequences from this subject were resistant to neutralization by PG9 and PG16.  Therefore, we used a bioinformatics approach to identify closely related viruses that were potentially sensitive to PG9 and PG16.  A most-recent common ancestor (MRCA) sequence for the viral envelope (Env) was detd. and aligned with 99 subtype A gp160 sequences from the Los Alamos HIV database.  Virus BG505.W6M.ENV.C2 (BG505) was found to have the highest degree of homol. (73%) to the MRCA sequence.  Pseudoviruses prepd. with this Env were sensitive to neutralization with a broad panel of bNAbs, including PG9 and PG16.  When expressed by 293T cells as sol. gp120, the BG505 monomer bound well to both PG9 and PG16.  We further showed that a point mutation (L111A) enabled more efficient prodn. of a stable gp120 monomer that preserves the major neutralization epitopes.  Finally, we showed that an adjuvanted formulation of this gp120 protein elicited neutralizing antibodies in rabbits (following a gp120 DNA vaccine prime) and that the antisera competed with bNAbs from 3 classes of nonoverlapping epitopes.  Thus, the BG505 Env protein warrants further investigation as an HIV vaccine candidate, as a stand-alone protein, or as a component of a vaccine vector.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGourszMpqvydrVg90H21EOLACvtfcHk0lhoJQUFbqZmyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsV2lsrk%253D&md5=b2d64d84e548e0045483638e7d800f66</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1128%2FJVI.02827-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02827-12%26sid%3Dliteratum%253Aachs%26aulast%3DHoffenberg%26aufirst%3DS.%26aulast%3DPowell%26aufirst%3DR.%26aulast%3DCarpov%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DKosakovsky%2BPond%26aufirst%3DS.%26aulast%3DLindsay%26aufirst%3DR.%26aulast%3DArendt%26aufirst%3DH.%26aulast%3DDeStefano%26aufirst%3DJ.%26aulast%3DPhogat%26aufirst%3DS.%26aulast%3DPoignard%26aufirst%3DP.%26aulast%3DFling%26aufirst%3DS.%2BP.%26aulast%3DSimek%26aufirst%3DM.%26aulast%3DLaBranche%26aufirst%3DC.%26aulast%3DMontefiori%26aufirst%3DD.%26aulast%3DWrin%26aufirst%3DT.%26aulast%3DPhung%26aufirst%3DP.%26aulast%3DBurton%26aufirst%3DD.%26aulast%3DKoff%26aufirst%3DW.%26aulast%3DKing%26aufirst%3DC.%2BR.%26aulast%3DParks%26aufirst%3DC.%2BL.%26aulast%3DCaulfield%26aufirst%3DM.%2BJ.%26atitle%3DIdentification%2520of%2520an%2520HIV-1%2520clade%2520A%2520envelope%2520that%2520exhibits%2520broad%2520antigenicity%2520and%2520neutralization%2520sensitivity%2520and%2520elicits%2520antibodies%2520targeting%2520three%2520distinct%2520epitopes%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D5372%26epage%3D5383%26doi%3D10.1128%2FJVI.02827-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Querido, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmagadda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriakose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicala, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-García, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaiken, I. M.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum anti-human immunodeficiency virus (HIV) potential of a peptide HIV type 1 entry inhibitor</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">3645</span>– <span class="NLM_lpage">3648</span>, <span class="refDoi"> DOI: 10.1128/JVI.01778-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.01778-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=17251295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjslWqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=3645-3648&author=S.+Cocklinauthor=H.+Gopiauthor=B.+Queridoauthor=M.+Nimmagaddaauthor=S.+Kuriakoseauthor=C.+Cicalaauthor=S.+Ajithauthor=S.+Baxterauthor=J.+Arthosauthor=J.+Mart%C3%ADn-Garc%C3%ADaauthor=I.+M.+Chaiken&title=Broad-spectrum+anti-human+immunodeficiency+virus+%28HIV%29+potential+of+a+peptide+HIV+type+1+entry+inhibitor&doi=10.1128%2FJVI.01778-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum anti-human immunodeficiency virus (HIV) potential of a peptide HIV type 1 entry inhibitor</span></div><div class="casAuthors">Cocklin, Simon; Gopi, Hosahudya; Querido, Bianca; Nimmagadda, Manideepthi; Kuriakose, Syna; Cicala, Claudia; Ajith, Sandya; Baxter, Sabine; Arthos, James; Martin-Garcia, Julio; Chaiken, Irwin M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3645-3648</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The AIDS epidemic continues to spread at an alarming rate worldwide, esp. in developing countries.  One approach to solving this problem is the generation of anti-human immunodeficiency virus (HIV) compds. with inhibition spectra broad enough to include globally prevailing forms of the virus.  We have examd. the HIV type 1 (HIV-1) envelope specificity of a recently identified entry inhibitor candidate, HNG-105, using surface plasmon resonance spectroscopy and pseudovirus inhibition assays.  The combined results suggest that the HNG-105 mol. may be effective across the HIV-1 subtypes, and they highlight its potential as a lead for developing therapeutic and microbicidal agents to help combat the spread of AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ZqavCczIgrVg90H21EOLACvtfcHk0lhoJQUFbqZmyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjslWqsbg%253D&md5=3c0000200d2703af02c9cf52e4595e3e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1128%2FJVI.01778-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01778-06%26sid%3Dliteratum%253Aachs%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DGopi%26aufirst%3DH.%26aulast%3DQuerido%26aufirst%3DB.%26aulast%3DNimmagadda%26aufirst%3DM.%26aulast%3DKuriakose%26aufirst%3DS.%26aulast%3DCicala%26aufirst%3DC.%26aulast%3DAjith%26aufirst%3DS.%26aulast%3DBaxter%26aufirst%3DS.%26aulast%3DArthos%26aufirst%3DJ.%26aulast%3DMart%25C3%25ADn-Garc%25C3%25ADa%26aufirst%3DJ.%26aulast%3DChaiken%26aufirst%3DI.%2BM.%26atitle%3DBroad-spectrum%2520anti-human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520potential%2520of%2520a%2520peptide%2520HIV%2520type%25201%2520entry%2520inhibitor%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D3645%26epage%3D3648%26doi%3D10.1128%2FJVI.01778-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Marzio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, N. R.</span></span> <span> </span><span class="NLM_article-title">Human immunodeficiency virus type 1 viral protein R (Vpr) arrests cells in the G2 phase of the cell cycle by inhibiting p34cdc2 activity</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">6705</span>– <span class="NLM_lpage">6711</span>, <span class="refDoi"> DOI: 10.1128/JVI.69.11.6705-6711.1995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.69.11.6705-6711.1995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=7474080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADyaK2MXoslWkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1995&pages=6705-6711&author=J.+Heauthor=S.+Choeauthor=R.+Walkerauthor=P.+Di+Marzioauthor=D.+O.+Morganauthor=N.+R.+Landau&title=Human+immunodeficiency+virus+type+1+viral+protein+R+%28Vpr%29+arrests+cells+in+the+G2+phase+of+the+cell+cycle+by+inhibiting+p34cdc2+activity&doi=10.1128%2FJVI.69.11.6705-6711.1995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Human immunodeficiency virus type 1 viral protein R (Vpr) arrests cells in the G2 phase of the cell cycle by inhibiting p34cdc2 activity</span></div><div class="casAuthors">He, Jianglin; Choe, Sunny; Walker, Robert; Di Marzio, Paola; Morgan, David O.; Landau, Nathaniel R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6705-11</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Vpr accessory gene product of human immunodeficiency virus types 1 and 2 and simian immunodeficiency virus is believed to play a role in permitting entry of the viral core into the nucleus of nondividing cells.  A second role for Vpr was recently suggested by (M. E. Rogel, et al., 1995) who showed that Vpr prevents the establishment in vitro of chronically infected HIV producer cell lines, apparently by causing infected cells to arrest in the G2/M phase of the cell cycle.  In cycling cells, progression from G2 to M phase is driven by activation of the p34cdc2/cyclin B complex, an event caused, in part, by dephosphorylation of two regulatory amino acids of p34cdc2 (Thr-14 and Tyr-15).  We show here that Vpr arrests the cell cycle in G2 by preventing the activation of the p34cdc2/cyclin B complex.  Vpr expression in cells caused p34cdc2 to remain in the phosphorylated, inactive state.  The p34cdc2/cyclin B complexes immunopptn. from cells expressing Vpr were almost completely inactive in a histone H1 kinase assay.  Coexpression of a constitutively active mutant p34cdc2 mol. with Vpr relieved the G2 arrest.  These findings strongly suggest that Vpr arrests cells in G2 by preventing the activation of the p34cdc2/cyclin B complex that is required for entry into M phase.  In vivo, Vpr might, by preventing p34cdc2 activation, delay or prevent apoptosis of infected cells.  This would increase the amt. of virus each infected cell produced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfPunHH8sjFbVg90H21EOLACvtfcHk0lhoJQUFbqZmyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXoslWkt74%253D&md5=cf13092726ac54f93a13fb9439e971c2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1128%2FJVI.69.11.6705-6711.1995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.69.11.6705-6711.1995%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DChoe%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%26aulast%3DDi%2BMarzio%26aufirst%3DP.%26aulast%3DMorgan%26aufirst%3DD.%2BO.%26aulast%3DLandau%26aufirst%3DN.%2BR.%26atitle%3DHuman%2520immunodeficiency%2520virus%2520type%25201%2520viral%2520protein%2520R%2520%2528Vpr%2529%2520arrests%2520cells%2520in%2520the%2520G2%2520phase%2520of%2520the%2520cell%2520cycle%2520by%2520inhibiting%2520p34cdc2%2520activity%26jtitle%3DJ.%2520Virol.%26date%3D1995%26volume%3D69%26spage%3D6705%26epage%3D6711%26doi%3D10.1128%2FJVI.69.11.6705-6711.1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stuart  Ruddell</span>, <span class="hlFld-ContribAuthor ">Elena  Sugrue</span>, <span class="hlFld-ContribAuthor ">Sarah  Memarzadeh</span>, <span class="hlFld-ContribAuthor ">Lorna Mae  Hellam</span>, <span class="hlFld-ContribAuthor ">Sam J.  Wilson</span>, <span class="hlFld-ContribAuthor ">David J.  France</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, Enantiomeric Resolution and Biological Evaluation of HIV Capsid Inhibition Activity for Racemic, (S)- and (R)-PF74. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (13)
                                     , 3919. <a href="https://doi.org/10.3390/molecules26133919" title="DOI URL">https://doi.org/10.3390/molecules26133919</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26133919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26133919%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSynthesis%25252C%252BEnantiomeric%252BResolution%252Band%252BBiological%252BEvaluation%252Bof%252BHIV%252BCapsid%252BInhibition%252BActivity%252Bfor%252BRacemic%25252C%252B%252528S%252529-%252Band%252B%252528R%252529-PF74%26aulast%3DRuddell%26aufirst%3DStuart%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D13%26spage%3D3919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0051.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0051.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Computational workflow. Workflow chart highlighting the steps used in the computational design process of compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking of racemic <b>CX15</b>, <b>CX16</b>, and <b>CX17</b>. (A) Compounds chosen for analysis. Regions that were modified on <b>PF-74</b> during the bioisosteric replacement are highlighted in pink (<b>CX15</b>), yellow (<b>CX16</b>), and green (<b>CX17</b>). The chemical structures were drawn with ChemAxon software (Budapest, Hungary). (B) Crystal structure of HIV-1 CA with compounds, <b>PF-74</b> in blue (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XFZ">4XFZ</a>) and docked <b>CX15</b> in pink, <b>CX16</b> in yellow, and <b>CX17</b> in green in the interprotomer pocket (highlighted by a black box) of a CA hexamer (for simplicity, only an extracted dimer is shown). (C) Magnification of each docked compound, <b>CX15</b> in pink, <b>CX16</b> in yellow, <b>CX17</b> in green, and <b>PF-74</b> in gray in the predicted binding site aligned with the <b>PF-74</b>-bound crystal structure (blue).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Computational analysis of drug-like properties of <b>CX15</b>, <b>CX16</b>, and <b>CX17</b>. Oral non-CNS scoring profile of <b>CX15</b>, <b>CX16</b>, <b>CX17</b>, and <b>PF-74</b> calculated with StarDrop (Optibrium Ltd., UK).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Representative sensorgrams depicting the interaction of compounds with the CA hexamer. SPR sensorgrams showing (A) <b>PF-74</b> (5 μM starting concentration with 1:4 serial dilutions), (B) <b>CX15</b> (5 μM starting concentration with 1:4 serial dilutions), (C) <b>CX16</b> (5 μM starting concentration with 1:4 serial dilutions), and (D) <b>CX17</b> (2 μM starting concentration with 1:4 serial dilutions) interacting with the immobilized disulfide-stabilized NL4-3 CA hexamer. Colored lines represent actual data collected from the dilution series, whereas black lines signify the fits to a 1:1 binding model. Interaction parameters derived from 5 sets of data are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Percent maximal response of compounds <b>PF-74</b>, <b>CX15</b>, <b>CX16</b>, and <b>CX17</b> in the CA hexamer interprotomer pockets of the wild type as compared to mutants. Compounds were tested at a single concentration of 5 μM to ensure saturation of each surface. Responses were normalized to percent theoretical maximal binding response to allow for cross-comparison. The red dashed line represents the average of the wild-type (wt) binding responses. Error bars represent the standard deviation of at least three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Electrostatic complementarity (EC) analysis of <b>PF-74</b>, <b>CX15</b>, <b>CX16</b>, and <b>CX17</b> within the wild type (wt) and HIV-1 capsid (NL4-3) mutants. The left panel shows the docking poses of <b>CX15</b>, <b>CX16</b>, <b>CX17</b> (colors scheme as in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), and the cocrystal structure of the HIV-1 NL4-3 capsid bound to <b>PF-74</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XFZ">4XFZ</a>). Green labeled residues are contributing hydrogen bonds (dashed green lines), and purple labeled residues contribute to cation-pi interactions to stabilize the complex (both if double-colored). The letter F next to the residue highlights that the adjacent HIV-1 capsid protomer contributes residues for complex stabilization. The letter O next to the residue highlights that a main-chain carbonyl is involved in hydrogen bonding. The right panel shows the EC surface map of each compound in the context of the wt or indicated in silico generated and energy minimized mutant. Each compound is shown from the front view (left) and in a 180° view (right) for the wt and mutant. Regions of the ligand surface with perfect electrostatic complementarity with the protein are colored green, while the areas with perfect electrostatic clash are colored red. Neutral EC areas are colored white.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Affinity and off-rates correlated with electrostatic complementarity. (A) Dissociation constant (<i>K</i><sub>D</sub>) of <b>CX16</b>, <b>PF-74</b>, <b>CX15</b>, and <b>CX17</b> (in this order) correlated with the EC. (B) Off-rates of <b>CX16</b>, <b>PF-74</b>, <b>CX15</b>, and <b>CX17</b> (in this order) correlated with the EC in the wt HIV-1 CA context. SCI, normalized surface integral of the complementarity score; Pearson R, Pearson correlation coefficient; Spearman rho rank, Spearman rank correlation coefficient.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Conservation of residues within the CA interprotomer binding pocket. Probability of conservation of residues from 5774 sequences within the Los Alamos Database for the residues within the interprotomer pocket of (A) protomer F and (B) protomer G. Residue numbers correspond to their placement within Hxbc2. Blue letters indicate residues that were mutated during the binding site mapping. The size of the letter indicates the probability of conservation. Generated using the AnalyzeAlign tool on the Los Alamos Database (<a href="http://www.hiv.lanl.gov" class="extLink">www.hiv.lanl.gov</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Sensorgrams depicting the interaction of enantiomerically pure compounds with CA hexamers. SPR sensorgrams showing (A) <b><i>R</i>-PF-74</b> (5 μM starting concentration with 1:4 serial dilutions), (B) <b><i>S</i>-PF-74</b> (5 μM starting concentration with 1:4 serial dilutions), (C) <b><i>R</i>-CX15</b> (5 μM starting concentration with 1:4 serial dilutions), (D) <b><i>S</i>-CX15</b> (5 μM starting concentration with 1:4 serial dilutions), (E) <b><i>R</i>-CX16</b> (5 μM starting concentration with 1:4 serial dilutions), (F) <b><i>S</i>-CX16</b> (5 μM starting concentration with 1:4 serial dilutions), (G) <b><i>R</i>-CX17</b> (2 μM starting concentration with 1:4 serial dilutions), and (H) <b><i>S</i>-CX17</b> (2 μM starting concentration with 1:4 serial dilutions) interacting with the immobilized disulfide-stabilized NL4-3 CA hexamer. Colored lines represent actual data collected from the dilution series, whereas black lines signify the fits to a 1:1 binding model. Interaction parameters derived from 5 sets of data are given in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of <i>tert</i>-Butyl(1-(methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-yl)carbamate (Intermediate <b>1</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of 2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide Hydrochloride (Intermediate <b>2</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of <i>N</i>,<i>N</i>-Dimethyl-1-(7-nitro-1<i>H</i>-indol-3-yl)methanamine (Intermediate <b>3</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Preparation of 2-(7-Nitro-1<i>H</i>-indol-3-yl)acetonitrile (Intermediate <b>4</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Preparation of 2-(7-Nitro-1<i>H</i>-indol-3-yl)acetic Acid (Intermediate <b>5</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Preparation of <i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (Intermediate <b>6</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0017.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Preparation of 2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>CX15</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch8" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0018.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Preparation of <i>tert</i>-Butyl (1-((1<i>H</i>-Indazol-5-yl)amino)-1-oxo-3-phenylpropan-2-yl) Carbamate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch9" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0019.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Preparation of <i>tert</i>-Butyl 5-(2-((<i>tert</i>-Butoxycarbonyl)amino)-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch10" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0020.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Preparation of <i>tert</i>-Butyl 5-(2-((<i>tert</i>-Butoxycarbonyl)amino)-<i>N</i>-methyl-3-phenylpropanamido)-1<i>H</i>-indazole-1-carboxylate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch11" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0021.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Preparation of 2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch12" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0022.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 12. Preparation of <i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b>CX16</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch13" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0023.jpeg" id="rightTab-gr22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 13. Preparation of <i>tert</i>-Butyl (1-((1<i>H</i>-Indazol-5-yl)amino)-1-oxo-3-phenylpropan-2-yl) Carbamate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch14" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0024.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 14. Preparation of 3-Iodo-1-(4-methoxybenzyl)-7-nitro-1<i>H</i>-indazole</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch15" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0025.jpeg" id="rightTab-gr24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 15. Preparation of Dimethyl 2-(1-(4-Methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)malonate</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch16" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Scheme 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0026.jpeg" id="rightTab-gr25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 16. Preparation of 2-(1-(4-Methoxybenzyl)-7-nitro-1<i>H</i>-indazol-3-yl)acetic Acid</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch17" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0027.jpeg" id="rightTab-gr26" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 17. Preparation of 2-(7-Nitro-1<i>H</i>-indazol-3-yl)acetic Acid</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch18" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0028.jpeg" id="rightTab-gr27" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 18. Preparation of 2-(7-Amino-1<i>H</i>-indazol-3-yl)acetic Acid</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch19" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0029.jpeg" id="rightTab-gr28" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 19. Preparation of 2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>CX17</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch20" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0030.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0030.jpeg" id="rightTab-gr29" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 20. Preparation of (<i>S</i>)-<i>tert</i>-Butyl 1-(Methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-5</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch21" data-index="30" class="article__inlineFigure"><h2 class="fig-label">Scheme 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0031.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0031.jpeg" id="rightTab-gr30" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 21. Preparation of (<i>S</i>)-2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>AP-5-9</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0031.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch21"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch22" data-index="31" class="article__inlineFigure"><h2 class="fig-label">Scheme 22</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0032.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0032.jpeg" id="rightTab-gr31" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 22. Preparation of (<i>S</i>)-<i>N</i>-Methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b><i>S</i>-PF-74</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0032.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch22"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch23" data-index="32" class="article__inlineFigure"><h2 class="fig-label">Scheme 23</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0033.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0033.jpeg" id="rightTab-gr32" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 23. Preparation of (<i>R</i>)-<i>tert</i>-Butyl 1-(Methyl(phenyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-4</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0033.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch23"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch24" data-index="33" class="article__inlineFigure"><h2 class="fig-label">Scheme 24</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0034.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0034.jpeg" id="rightTab-gr33" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 24. Preparation of (<i>R</i>)-2-Amino-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b>AP-5-8</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0034.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch24"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch25" data-index="34" class="article__inlineFigure"><h2 class="fig-label">Scheme 25</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0035.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0035.jpeg" id="rightTab-gr34" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 25. Preparation of (<i>R</i>)-<i>N</i>-Methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b><i>R</i>-PF-74</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0035.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch25"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch26" data-index="35" class="article__inlineFigure"><h2 class="fig-label">Scheme 26</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0036.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0036.jpeg" id="rightTab-gr35" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 26. Preparation of (<i>S</i>)-<i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b>AP-5-47</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0036.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch26"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch27" data-index="36" class="article__inlineFigure"><h2 class="fig-label">Scheme 27</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0037.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0037.jpeg" id="rightTab-gr36" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 27. Preparation of (<i>S</i>)-2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b><i>S</i>-CX15</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0037.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch27"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch28" data-index="37" class="article__inlineFigure"><h2 class="fig-label">Scheme 28</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0038.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0038.jpeg" id="rightTab-gr37" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 28. Preparation of (<i>R</i>)-<i>N</i>-Methyl-2-(2-(7-nitro-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>,3-diphenylpropanamide (<b>AP-5-46</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0038.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch28"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch29" data-index="38" class="article__inlineFigure"><h2 class="fig-label">Scheme 29</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0039.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0039.jpeg" id="rightTab-gr38" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 29. Preparation of (<i>R</i>)-2-(2-(7-Amino-1<i>H</i>-indol-3-yl)acetamido)-<i>N</i>-methyl-<i>N</i>,3-diphenylpropanamide (<b><i>R</i>-CX15</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0039.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch29"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch30" data-index="39" class="article__inlineFigure"><h2 class="fig-label">Scheme 30</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0040.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0040.jpeg" id="rightTab-gr39" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 30. Preparation of <i>N</i>-Methyl-1<i>H</i>-indazol-5-amine (<b>AP-5-69</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0040.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch30"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch31" data-index="40" class="article__inlineFigure"><h2 class="fig-label">Scheme 31</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0041.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0041.jpeg" id="rightTab-gr40" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 31. Preparation of (<i>S</i>)-<i>tert</i>-Butyl 1-((1<i>H</i>-Indazol-5-yl)(methyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-93</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0041.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch31"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch32" data-index="41" class="article__inlineFigure"><h2 class="fig-label">Scheme 32</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0042.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0042.jpeg" id="rightTab-gr41" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 32. Preparation of (<i>S</i>)-2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>AP-5-99</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0042.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch32"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch33" data-index="42" class="article__inlineFigure"><h2 class="fig-label">Scheme 33</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0043.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0043.jpeg" id="rightTab-gr42" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 33. Preparation of (<i>S</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b><i>S</i>-CX16</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0043.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch33"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch34" data-index="43" class="article__inlineFigure"><h2 class="fig-label">Scheme 34</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0044.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0044.jpeg" id="rightTab-gr43" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 34. Preparation of (<i>R</i>)-<i>tert</i>-Butyl 1-((1<i>H</i>-Indazol-5-yl)(methyl)amino)-1-oxo-3-phenylpropan-2-ylcarbamate (<b>AP-5-72</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0044.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch34"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch35" data-index="44" class="article__inlineFigure"><h2 class="fig-label">Scheme 35</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0045.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0045.jpeg" id="rightTab-gr44" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 35. Preparation of (<i>R</i>)-2-Amino-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b>AP-5-73</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0045.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch35"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch36" data-index="45" class="article__inlineFigure"><h2 class="fig-label">Scheme 36</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0046.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0046.jpeg" id="rightTab-gr45" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 36. Preparation of (<i>R</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(2-methyl-1<i>H</i>-indol-3-yl)acetamido)-3-phenylpropanamide (<b><i>R</i>-CX16</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0046.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch36"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch37" data-index="46" class="article__inlineFigure"><h2 class="fig-label">Scheme 37</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0047.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0047.jpeg" id="rightTab-gr46" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 37. Preparation of (<i>S</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indazol-3-yl)acetamido)-3-phenylpropanamide (<b>102</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0047.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch37"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch38" data-index="47" class="article__inlineFigure"><h2 class="fig-label">Scheme 38</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0048.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0048.jpeg" id="rightTab-gr47" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 38. Preparation of (<i>S</i>)-2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b><i>S</i>-CX17</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0048.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch38"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch39" data-index="48" class="article__inlineFigure"><h2 class="fig-label">Scheme 39</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0049.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0049.jpeg" id="rightTab-gr48" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 39. Preparation of (<i>R</i>)-<i>N</i>-(1<i>H</i>-Indazol-5-yl)-<i>N</i>-methyl-2-(2-(7-nitro-1<i>H</i>-indazol-3-yl)acetamido)-3-phenylpropanamide (<b>AP-5-104</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0049.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch39"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch40" data-index="49" class="article__inlineFigure"><h2 class="fig-label">Scheme 40</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/medium/jm0c01810_0050.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0050.jpeg" id="rightTab-gr49" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 40. Preparation of (<i>R</i>)-2-(2-(7-Amino-1<i>H</i>-indazol-3-yl)acetamido)-<i>N</i>-(1<i>H</i>-indazol-5-yl)-<i>N</i>-methyl-3-phenylpropanamide (<b><i>R</i>-CX17</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01810/20210331/images/large/jm0c01810_0050.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01810&amp;id=sch40"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i124">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hope, T. J.</span></span> <span> </span><span class="NLM_article-title">HIV-1 capsid: the multifaceted key player in HIV-1 infection</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1038/nrmicro3503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1038%2Fnrmicro3503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=26179359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyks7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=471-483&author=E.+M.+Campbellauthor=T.+J.+Hope&title=HIV-1+capsid%3A+the+multifaceted+key+player+in+HIV-1+infection&doi=10.1038%2Fnrmicro3503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 capsid: the multifaceted key player in HIV-1 infection</span></div><div class="casAuthors">Campbell, Edward M.; Hope, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">471-483</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In a mature, infectious HIV-1 virion, the viral genome is housed within a conical capsid core made from the viral capsid (CA) protein.  The CA protein and the structure into which it assembles facilitate virtually every step of infection through a series of interactions with multiple host cell factors.  This Review describes the understanding of the interactions between the viral capsid core and several cellular factors that enable efficient HIV-1 genome replication, timely core disassembly, nuclear import and the integration of the viral genome into the genome of the target cell.  The authors then discuss how elucidating these interactions can reveal new targets for therapeutic interactions against HIV-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr9ouJsQtk4LVg90H21EOLACvtfcHk0li4OTM257K1QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyks7%252FO&md5=fa2779bfe0322d9d1a9e709d18dc448f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro3503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro3503%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DE.%2BM.%26aulast%3DHope%26aufirst%3DT.%2BJ.%26atitle%3DHIV-1%2520capsid%253A%2520the%2520multifaceted%2520key%2520player%2520in%2520HIV-1%2520infection%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2015%26volume%3D13%26spage%3D471%26epage%3D483%26doi%3D10.1038%2Fnrmicro3503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundquist, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krausslich, H. G.</span></span> <span> </span><span class="NLM_article-title">HIV-1 assembly, budding, and maturation</span>. <i>Cold Spring Harb. Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">a006924</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a006924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1101%2Fcshperspect.a006924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=22762019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlemtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=a006924&author=W.+I.+Sundquistauthor=H.+G.+Krausslich&title=HIV-1+assembly%2C+budding%2C+and+maturation&doi=10.1101%2Fcshperspect.a006924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 assembly, budding, and maturation</span></div><div class="casAuthors">Sundquist, Wesley I.; Krausslich, Hans-Georg</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">a006924/1-a006924/24</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  A defining property of retroviruses is their ability to assemble into particles that can leave producer cells and spread infection to susceptible cells and hosts.  Virion morphogenesis can be divided into three stages: assembly, wherein the virion is created and essential components are packaged; budding, wherein the virion crosses the plasma membrane and obtains its lipid envelope; and maturation, wherein the virion changes structure and becomes infectious.  All of these stages are coordinated by the Gag polyprotein and its proteolytic maturation products, which function as the major structural proteins of the virus.  Here, we review our current understanding of the mechanisms of HIV-1 assembly, budding, and maturation, starting with a general overview and then providing detailed descriptions of each of the different stages of virion morphogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH1Quq8ytnBrVg90H21EOLACvtfcHk0li4OTM257K1QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlemtbs%253D&md5=91c7bf7383d85be14ef6f5200990f451</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a006924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a006924%26sid%3Dliteratum%253Aachs%26aulast%3DSundquist%26aufirst%3DW.%2BI.%26aulast%3DKrausslich%26aufirst%3DH.%2BG.%26atitle%3DHIV-1%2520assembly%252C%2520budding%252C%2520and%2520maturation%26jtitle%3DCold%2520Spring%2520Harb.%2520Perspect.%2520Med.%26date%3D2012%26volume%3D2%26spage%3Da006924%26doi%3D10.1101%2Fcshperspect.a006924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rihn, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loman, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakker, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gifford, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rixon, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bieniasz, P. D.</span></span> <span> </span><span class="NLM_article-title">Extreme genetic fragility of the HIV-1 capsid</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e1003461</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1003461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1003461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=23818857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFCntrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&author=S.+J.+Rihnauthor=S.+J.+Wilsonauthor=N.+J.+Lomanauthor=M.+Alimauthor=S.+E.+Bakkerauthor=D.+Bhellaauthor=R.+J.+Giffordauthor=F.+J.+Rixonauthor=P.+D.+Bieniasz&title=Extreme+genetic+fragility+of+the+HIV-1+capsid&doi=10.1371%2Fjournal.ppat.1003461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Extreme genetic fragility of the HIV-1 capsid</span></div><div class="casAuthors">Rihn, Suzannah J.; Wilson, Sam J.; Loman, Nick J.; Alim, Mudathir; Bakker, Saskia E.; Bhella, David; Gifford, Robert J.; Rixon, Frazer J.; Bieniasz, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1003461</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Genetic robustness, or fragility, is defined as the ability, or lack thereof, of a biol. entity to maintain function in the face of mutations.  Viruses that replicate via RNA intermediates exhibit high mutation rates, and robustness should be particularly advantageous to them.  The capsid (CA) domain of the HIV-1 Gag protein is under strong pressure to conserve functional roles in viral assembly, maturation, uncoating, and nuclear import.  However, CA is also under strong immunol. pressure to diversify.  Therefore, it would be particularly advantageous for CA to evolve genetic robustness.  To measure the genetic robustness of HIV-1 CA, the authors generated a library of single amino acid substitution mutants, encompassing almost half the residues in CA.  Strikingly, the authors found HIV-1 CA to be the most genetically fragile protein that has been analyzed using such an approach, with 70% of mutations yielding replication-defective viruses.  Although CA participates in several steps in HIV-1 replication, anal. of conditionally (temp. sensitive) and constitutively non-viable mutants revealed that the biol. basis for its genetic fragility was primarily the need to coordinate the accurate and efficient assembly of mature virions.  All mutations that exist in naturally occurring HIV-1 subtype B populations at a frequency >3%, and were also present in the mutant library, had fitness levels that were >40% of WT.  However, a substantial fraction of mutations with high fitness did not occur in natural populations, suggesting another form of selection pressure limiting variation in vivo.  Addnl., known protective CTL epitopes occurred preferentially in domains of the HIV-1 CA that were even more genetically fragile than HIV-1 CA as a whole.  The extreme genetic fragility of HIV-1 CA may be one reason why cell-mediated immune responses to Gag correlate with better prognosis in HIV-1 infection, and suggests that CA is a good target for therapy and vaccination strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMV98aKUfzWbVg90H21EOLACvtfcHk0ljvxUekZUHxJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCntrfL&md5=812c2768fb6972527b8dc7e6f5dbe94a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1003461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1003461%26sid%3Dliteratum%253Aachs%26aulast%3DRihn%26aufirst%3DS.%2BJ.%26aulast%3DWilson%26aufirst%3DS.%2BJ.%26aulast%3DLoman%26aufirst%3DN.%2BJ.%26aulast%3DAlim%26aufirst%3DM.%26aulast%3DBakker%26aufirst%3DS.%2BE.%26aulast%3DBhella%26aufirst%3DD.%26aulast%3DGifford%26aufirst%3DR.%2BJ.%26aulast%3DRixon%26aufirst%3DF.%2BJ.%26aulast%3DBieniasz%26aufirst%3DP.%2BD.%26atitle%3DExtreme%2520genetic%2520fragility%2520of%2520the%2520HIV-1%2520capsid%26jtitle%3DPLoS%2520Pathog.%26date%3D2013%26volume%3D9%26doi%3D10.1371%2Fjournal.ppat.1003461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunanan, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Structure, function, and interactions of the HIV-1 capsid protein</span>. <i>life</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">100</span>, <span class="refDoi"> DOI: 10.3390/life11020100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3390%2Flife11020100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=33572761" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=100&author=E.+Rossiauthor=M.+E.+Meuserauthor=C.+J.+Cunananauthor=S.+Cocklin&title=Structure%2C+function%2C+and+interactions+of+the+HIV-1+capsid+protein&doi=10.3390%2Flife11020100"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Flife11020100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Flife11020100%26sid%3Dliteratum%253Aachs%26aulast%3DRossi%26aufirst%3DE.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DCunanan%26aufirst%3DC.%2BJ.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DStructure%252C%2520function%252C%2520and%2520interactions%2520of%2520the%2520HIV-1%2520capsid%2520protein%26jtitle%3Dlife%26date%3D2021%26volume%3D11%26spage%3D100%26doi%3D10.3390%2Flife11020100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundquist, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finch, J. T.</span></span> <span> </span><span class="NLM_article-title">Image reconstructions of helical assemblies of the HIV-1 CA protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>407</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">413</span>, <span class="refDoi"> DOI: 10.1038/35030177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1038%2F35030177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=11014200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntF2ntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=407&publication_year=2000&pages=409-413&author=S.+Liauthor=C.+P.+Hillauthor=W.+I.+Sundquistauthor=J.+T.+Finch&title=Image+reconstructions+of+helical+assemblies+of+the+HIV-1+CA+protein&doi=10.1038%2F35030177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Image reconstructions of helical assemblies of the HIV-1 CA protein</span></div><div class="casAuthors">Li, Su; Hill, Christopher P.; Sundquist, Wesley I.; Finch, John T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">407</span>
        (<span class="NLM_cas:issue">6802</span>),
    <span class="NLM_cas:pages">409-413</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The type 1 human immunodeficiency virus (HIV-1) contains a conical capsid comprising ∼1500 CA proteins subunits, which organizes the viral RNA genome for uncoating and replication in a new host cell.  In vitro, CA spontaneously assemblies into helical tubes and cones that resemble authentic viral capsids.  Here we describe electron cryo-microscopy and image reconstructions of CA tubes from six different helical families.  In spite of their polymorphism, all tubes are composed of hexameric rings of CA arranged with approx. local p6 lattice symmetry.  Crystal structures of the two CA domains were 'docked' into the reconstructed d., which showed that the amino-terminal domains form the hexameric rings and the carboxy-terminal dimerization domains connect each ring to six neighbors.  We propose a mol. model for the HIV-1 capsid that follows the principles of a fullerene cone, in which the body of the cone is composed of curved hexagonal arrays of CA rings and the ends are closed by inclusion of 12 pentagonal 'defects'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT7d67nnEKwrVg90H21EOLACvtfcHk0ljvxUekZUHxJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntF2ntbk%253D&md5=2efab9ee5fd98a7517c6d2f1aa6450aa</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F35030177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35030177%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DHill%26aufirst%3DC.%2BP.%26aulast%3DSundquist%26aufirst%3DW.%2BI.%26aulast%3DFinch%26aufirst%3DJ.%2BT.%26atitle%3DImage%2520reconstructions%2520of%2520helical%2520assemblies%2520of%2520the%2520HIV-1%2520CA%2520protein%26jtitle%3DNature%26date%3D2000%26volume%3D407%26spage%3D409%26epage%3D413%26doi%3D10.1038%2F35030177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fricke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zadrozny, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedzicki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demeler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pornillos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganser-Pornillos, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Griffero, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeager, M.</span></span> <span> </span><span class="NLM_article-title">Structural basis of HIV-1 capsid recognition by PF74 and CPSF6</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">18625</span>– <span class="NLM_lpage">18630</span>, <span class="refDoi"> DOI: 10.1073/pnas.1419945112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1073%2Fpnas.1419945112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=25518861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=18625-18630&author=A.+Bhattacharyaauthor=S.+L.+Alamauthor=T.+Frickeauthor=K.+Zadroznyauthor=J.+Sedzickiauthor=A.+B.+Taylorauthor=B.+Demelerauthor=O.+Pornillosauthor=B.+K.+Ganser-Pornillosauthor=F.+Diaz-Grifferoauthor=D.+N.+Ivanovauthor=M.+Yeager&title=Structural+basis+of+HIV-1+capsid+recognition+by+PF74+and+CPSF6&doi=10.1073%2Fpnas.1419945112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of HIV-1 capsid recognition by PF74 and CPSF6</span></div><div class="casAuthors">Bhattacharya, Akash; Alam, Steven L.; Fricke, Thomas; Zadrozny, Kaneil; Sedzicki, Jaroslaw; Taylor, Alexander B.; Demeler, Borries; Pornillos, Owen; Ganser-Pornillos, Barbie K.; Diaz-Griffero, Felipe; Ivanov, Dmitri N.; Yeager, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">18625-18630</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Upon infection of susceptible cells by HIV-1, the conical capsid formed by ∼250 hexamers and 12 pentamers of the CA protein is delivered to the cytoplasm.  The capsid shields the RNA genome and proteins required for reverse transcription.  In addn., the surface of the capsid mediates numerous host-virus interactions, which either promote infection or enable viral restriction by innate immune responses.  In the intact capsid, there is an intermol. interface between the N-terminal domain (NTD) of one subunit and the C-terminal domain (CTD) of the adjacent subunit within the same hexameric ring.  The NTD-CTD interface is crit. for capsid assembly, both as an architectural element of the CA hexamer and pentamer and as a mechanistic element for generating lattice curvature.  Here we report biochem. expts. showing that PF-3450074 (PF74), a drug that inhibits HIV-1 infection, as well as host proteins cleavage and polyadenylation specific factor 6 (CPSF6) and nucleoporin 153 kDa (NUP153), bind to the CA hexamer with at least 10-fold higher affinities compared with nonassembled CA or isolated CA domains.  The crystal structure of PF74 in complex with the CA hexamer reveals that PF74 binds in a preformed pocket encompassing the NTD-CTD interface, suggesting that the principal inhibitory target of PF74 is the assembled capsid.  Likewise, CPSF6 binds in the same pocket.  Given that the NTD-CTD interface is a specific mol. signature of assembled hexamers in the capsid, binding of NUP153 at this site suggests that key features of capsid architecture remain intact upon delivery of the preintegration complex to the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpis5JJGK8jYLVg90H21EOLACvtfcHk0lg85dP_sIMpFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7%252FE&md5=9e7927ef280400df591c52ca51363b65</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1419945112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1419945112%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharya%26aufirst%3DA.%26aulast%3DAlam%26aufirst%3DS.%2BL.%26aulast%3DFricke%26aufirst%3DT.%26aulast%3DZadrozny%26aufirst%3DK.%26aulast%3DSedzicki%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DA.%2BB.%26aulast%3DDemeler%26aufirst%3DB.%26aulast%3DPornillos%26aufirst%3DO.%26aulast%3DGanser-Pornillos%26aufirst%3DB.%2BK.%26aulast%3DDiaz-Griffero%26aufirst%3DF.%26aulast%3DIvanov%26aufirst%3DD.%2BN.%26aulast%3DYeager%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520of%2520HIV-1%2520capsid%2520recognition%2520by%2520PF74%2520and%2520CPSF6%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D18625%26epage%3D18630%26doi%3D10.1073%2Fpnas.1419945112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwan, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halambage, U. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiken, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. C.</span></span> <span> </span><span class="NLM_article-title">Host cofactors and pharmacologic ligands share an essential interface in HIV-1 capsid that is lost upon disassembly</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1004459</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1004459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1004459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=25356722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyisbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=A.+J.+Priceauthor=D.+A.+Jacquesauthor=W.+A.+McEwanauthor=A.+J.+Fletcherauthor=S.+Essigauthor=J.+W.+Chinauthor=U.+D.+Halambageauthor=C.+Aikenauthor=L.+C.+James&title=Host+cofactors+and+pharmacologic+ligands+share+an+essential+interface+in+HIV-1+capsid+that+is+lost+upon+disassembly&doi=10.1371%2Fjournal.ppat.1004459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Host cofactors and pharmacologic ligands share an essential interface in HIV-1 capsid that is lost upon disassembly</span></div><div class="casAuthors">Price, Amanda J.; Jacques, David A.; McEwan, William A.; Fletcher, Adam J.; Essig, Sebastian; Chin, Jason W.; Halambage, Upul D.; Aiken, Christopher; James, Leo C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1004459/1-e1004459/17, 17 pp.</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The HIV-1 capsid is involved in all infectious steps from reverse transcription to integration site selection, and is the target of multiple host cell and pharmacol. ligands.  However, structural studies have been limited to capsid monomers (CA), and the mechanistic basis for how these ligands influence infection is not well understood.  Here we show that a multi-subunit interface formed exclusively within CA hexamers mediates binding to linear epitopes within cellular cofactors NUP153 and CPSF6, and is competed for by the antiretroviral compds. PF74 and BI-2.  Each ligand is anchored via a shared phenylalanine-glycine (FG) motif to a pocket within the N-terminal domain of one monomer, and all but BI-2 also make essential interactions across the N-terminal domain: C-terminal domain (NTD:CTD) interface to a second monomer.  Dissocn. of hexamer into CA monomers prevents high affinity interaction with CPSF6 and PF74, and abolishes binding to NUP153.  The second interface is conformationally dynamic, but binding of NUP153 or CPSF6 peptides is accommodated by only one conformation.  NUP153 and CPSF6 have overlapping binding sites, but each makes unique CA interactions that, when mutated selectively, perturb cofactor dependency.  These results reveal that multiple ligands share an overlapping interface in HIV-1 capsid that is lost upon viral disassembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqln0TIYBQnCrVg90H21EOLACvtfcHk0lg85dP_sIMpFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyisbbN&md5=9165cd26ee15021a47448579c32c45ca</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1004459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1004459%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DJacques%26aufirst%3DD.%2BA.%26aulast%3DMcEwan%26aufirst%3DW.%2BA.%26aulast%3DFletcher%26aufirst%3DA.%2BJ.%26aulast%3DEssig%26aufirst%3DS.%26aulast%3DChin%26aufirst%3DJ.%2BW.%26aulast%3DHalambage%26aufirst%3DU.%2BD.%26aulast%3DAiken%26aufirst%3DC.%26aulast%3DJames%26aufirst%3DL.%2BC.%26atitle%3DHost%2520cofactors%2520and%2520pharmacologic%2520ligands%2520share%2520an%2520essential%2520interface%2520in%2520HIV-1%2520capsid%2520that%2520is%2520lost%2520upon%2520disassembly%26jtitle%3DPLoS%2520Pathog.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.ppat.1004459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KewalRamani, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towers, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. C.</span></span> <span> </span><span class="NLM_article-title">CPSF6 defines a conserved capsid interface that modulates HIV-1 replication</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002896</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1002896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1002896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=22956906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlSjsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=A.+J.+Priceauthor=A.+J.+Fletcherauthor=T.+Schallerauthor=T.+Elliottauthor=K.+Leeauthor=V.+N.+KewalRamaniauthor=J.+W.+Chinauthor=G.+J.+Towersauthor=L.+C.+James&title=CPSF6+defines+a+conserved+capsid+interface+that+modulates+HIV-1+replication&doi=10.1371%2Fjournal.ppat.1002896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">CPSF6 defines a conserved capsid interface that modulates HIV-1 replication</span></div><div class="casAuthors">Price, Amanda J.; Fletcher, Adam J.; Schaller, Torsten; Elliott, Tom; Lee, Kyeong Eun; Kewal Ramani, Vineet N.; Chin, Jason W.; Towers, Greg J.; James, Leo C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e1002896</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The HIV-1 genome enters cells inside a shell comprised of capsid (CA) protein.  Variation in CA sequence alters HIV-1 infectivity and escape from host restriction factors.  However, apart from the Cyclophilin A-binding loop, CA has no known interfaces with which to interact with cellular cofactors.  Here we describe a novel protein-protein interface in the N-terminal domain of HIV-1 CA, detd. by X-ray crystallog., which mediates both viral restriction and host cofactor dependence.  The interface is highly conserved across lentiviruses and is accessible in the context of a hexameric lattice.  Mutation of the interface prevents binding to and restriction by CPSF6-358, a truncated cytosolic form of the RNA processing factor, cleavage and polyadenylation specific factor 6 (CPSF6).  Furthermore, mutations that prevent CPSF6 binding also relieve dependence on nuclear entry cofactors TNPO3 and RanBP2.  These results suggest that the HIV-1 capsid mediates direct host cofactor interactions to facilitate viral infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiWdPM8k1FVLVg90H21EOLACvtfcHk0lgGiu-l373Q4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlSjsrbE&md5=9d419609d09371e976ba85d5209255a3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1002896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1002896%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DFletcher%26aufirst%3DA.%2BJ.%26aulast%3DSchaller%26aufirst%3DT.%26aulast%3DElliott%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKewalRamani%26aufirst%3DV.%2BN.%26aulast%3DChin%26aufirst%3DJ.%2BW.%26aulast%3DTowers%26aufirst%3DG.%2BJ.%26aulast%3DJames%26aufirst%3DL.%2BC.%26atitle%3DCPSF6%2520defines%2520a%2520conserved%2520capsid%2520interface%2520that%2520modulates%2520HIV-1%2520replication%26jtitle%3DPLoS%2520Pathog.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.ppat.1002896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matreyek, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yücel, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelman, A.</span></span> <span> </span><span class="NLM_article-title">Nucleoporin NUP153 phenylalanine-glycine motifs engage a common binding pocket within the HIV-1 capsid protein to mediate lentiviral infectivity</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e1003693</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1003693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1003693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=24130490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFaltbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003693&author=K.+A.+Matreyekauthor=S.+S.+Y%C3%BCcelauthor=X.+Liauthor=A.+Engelman&title=Nucleoporin+NUP153+phenylalanine-glycine+motifs+engage+a+common+binding+pocket+within+the+HIV-1+capsid+protein+to+mediate+lentiviral+infectivity&doi=10.1371%2Fjournal.ppat.1003693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoporin NUP153 phenylalanine-glycine motifs engage a common binding pocket within the HIV-1 capsid protein to mediate lentiviral infectivity</span></div><div class="casAuthors">Matreyek, Kenneth A.; Yucel, Sara S.; Li, Xiang; Engelman, Alan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1003693/1-e1003693/21</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Lentiviruses can infect non-dividing cells, and various cellular transport proteins provide crucial functions for lentiviral nuclear entry and integration.  We previously showed that the viral capsid (CA) protein mediated the dependency on cellular nucleoporin (NUP) 153 during HIV-1 infection, and now demonstrate a direct interaction between the CA N-terminal domain and the phenylalanine-glycine (FG)-repeat enriched NUP153 C-terminal domain (NUP153C).  NUP153C fused to the effector domains of the rhesus Trim5α restriction factor (Trim-NUP153C) potently restricted HIV-1, providing an intracellular readout for the NUP153C-CA interaction during retroviral infection.  Primate lentiviruses and equine infectious anemia virus (EIAV) bound NUP153C under these conditions, results that correlated with direct binding between purified proteins in vitro.  These binding phenotypes moreover correlated with the requirement for endogenous NUP153 protein during virus infection.  Mutagenesis expts. concordantly identified NUP153C and CA residues important for binding and lentiviral infectivity.  Different FG motifs within NUP153C mediated binding to HIV-1 vs. EIAV capsids.  HIV-1 CA binding mapped to residues that line the common alpha helix 3/4 hydrophobic pocket that also mediates binding to the small mol. PF-3450074 (PF74) inhibitor and cleavage and polyadenylation specific factor 6 (CPSF6) protein, with Asn57 (Asp58 in EIAV) playing a particularly important role.  PF74 and CPSF6 accordingly each competed with NUP153C for binding to the HIV-1 CA pocket, and significantly higher concns. of PF74 were needed to inhibit HIV-1 infection in the face of Trim-NUP153C expression or NUP153 knockdown.  Correlation between CA mutant viral cell cycle and NUP153 dependencies moreover indicates that the NUP153C-CA interaction underlies the ability of HIV-1 to infect non-dividing cells.  Our results highlight similar mechanisms of binding for disparate host factors to the same region of HIV-1 CA during viral ingress.  We conclude that a subset of lentiviral CA proteins directly engage FG-motifs present on NUP153 to affect viral nuclear import.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0M6R0jl8aCbVg90H21EOLACvtfcHk0lgGiu-l373Q4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFaltbg%253D&md5=6691bf0c32a1a50eec96dcda2cf57b48</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1003693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1003693%26sid%3Dliteratum%253Aachs%26aulast%3DMatreyek%26aufirst%3DK.%2BA.%26aulast%3DY%25C3%25BCcel%26aufirst%3DS.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DEngelman%26aufirst%3DA.%26atitle%3DNucleoporin%2520NUP153%2520phenylalanine-glycine%2520motifs%2520engage%2520a%2520common%2520binding%2520pocket%2520within%2520the%2520HIV-1%2520capsid%2520protein%2520to%2520mediate%2520lentiviral%2520infectivity%26jtitle%3DPLoS%2520Pathog.%26date%3D2013%26volume%3D9%26spage%3De1003693%26doi%3D10.1371%2Fjournal.ppat.1003693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. P.</span>; <span class="NLM_string-name">Francis, A. C.</span>; <span class="NLM_string-name">Meuser, M. E.</span>; <span class="NLM_string-name">Mankowski, M.</span>; <span class="NLM_string-name">Ptak, R. G.</span>; <span class="NLM_string-name">Rashad, A. A.</span>; <span class="NLM_string-name">Melikyan, G. B.</span>; <span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Exploring Modifications of an HIV-1 Capsid Inhibitor: Design, Synthesis, and Mechanism of Action</span>.  <i>J. Drug Des. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">5</span> ().</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=30393786" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&author=J.+P.+Xu&author=A.+C.+Francis&author=M.+E.+Meuser&author=M.+Mankowski&author=R.+G.+Ptak&author=A.+A.+Rashad&author=G.+B.+Melikyan&author=S.+Cocklin&title=Exploring+Modifications+of+an+HIV-1+Capsid+Inhibitor%3A+Design%2C+Synthesis%2C+and+Mechanism+of+Action"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%2BP.%26atitle%3DExploring%2520Modifications%2520of%2520an%2520HIV-1%2520Capsid%2520Inhibitor%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Mechanism%2520of%2520Action%26jtitle%3DJ.%2520Drug%2520Des.%2520Res.%26date%3D2018%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casey, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernekar, S. K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahani, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hachiya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tedbury, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Chemical profiling of HIV-1 capsid-targeting antiviral PF74</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>200</i></span>,  <span class="NLM_fpage">112427</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.ejmech.2020.112427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=32438252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsl2mt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=200&publication_year=2020&pages=112427&author=L.+Wangauthor=M.+C.+Caseyauthor=S.+K.+V.+Vernekarauthor=H.+T.+Doauthor=R.+L.+Sahaniauthor=K.+A.+Kirbyauthor=H.+Duauthor=A.+Hachiyaauthor=H.+Zhangauthor=P.+R.+Tedburyauthor=J.+Xieauthor=S.+G.+Sarafianosauthor=Z.+Wang&title=Chemical+profiling+of+HIV-1+capsid-targeting+antiviral+PF74&doi=10.1016%2Fj.ejmech.2020.112427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical profiling of HIV-1 capsid-targeting antiviral PF74</span></div><div class="casAuthors">Wang, Lei; Casey, Mary C.; Vernekar, Sanjeev Kumar V.; Do, Ha T.; Sahani, Rajkumar Lalji; Kirby, Karen A.; Du, Haijuan; Hachiya, Atsuko; Zhang, Huanchun; Tedbury, Philip R.; Xie, Jiashu; Sarafianos, Stefan G.; Wang, Zhengqiang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">200</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112427</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The capsid protein (CA) of HIV-1 plays essential roles in multiple steps of the viral replication cycle by assembling into functional capsid core, controlling the kinetics of uncoating and nuclear entry, and interacting with various host factors.  Targeting CA represents an attractive yet underexplored antiviral approach.  Of all known CA-targeting small mol. chemotypes, the peptidomimetic PF74, I, is particularly interesting because it binds to the same pocket used by a few important host factors, resulting in highly desirable antiviral phenotypes.  However, further development of I entails understanding its pharmacophore and mitigating its poor metabolic stability.  We report herein the design, synthesis, and evaluation of a large no. of I analogs aiming to provide a comprehensive chem. profiling of I and advance the understanding on its detailed binding mechanism and pharmacophore.  The analogs, contg. structural variations mainly in the aniline domain and/or the indole domain, were assayed for their effect on stability of CA hexamers, antiviral activity, and cytotoxicity.  Selected analogs were also tested for metabolic stability in liver microsomes, alone or in the presence of a CYP3A inhibitor.  Collectively, our studies identified important pharmacophore elements and revealed addnl. binding features of I, which could aid in future design of improved ligands to better probe the mol. basis of CA-host factor interactions, design strategies to disrupt them, and ultimately identify viable CA-targeting antiviral leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6uw0dmkMDbbVg90H21EOLACvtfcHk0ljL1ZIZP3XNZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsl2mt74%253D&md5=7bedb9a56ce8d6dcc82e9ee352052141</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112427%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DCasey%26aufirst%3DM.%2BC.%26aulast%3DVernekar%26aufirst%3DS.%2BK.%2BV.%26aulast%3DDo%26aufirst%3DH.%2BT.%26aulast%3DSahani%26aufirst%3DR.%2BL.%26aulast%3DKirby%26aufirst%3DK.%2BA.%26aulast%3DDu%26aufirst%3DH.%26aulast%3DHachiya%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DTedbury%26aufirst%3DP.%2BR.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DChemical%2520profiling%2520of%2520HIV-1%2520capsid-targeting%2520antiviral%2520PF74%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D200%26spage%3D112427%26doi%3D10.1016%2Fj.ejmech.2020.112427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vernekar, S. K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahani, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casey, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kankanala, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tedbury, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Toward Structurally Novel and Metabolically Stable HIV-1 Capsid-Targeting Small Molecules</span>. <i>Viruses</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">452</span>, <span class="refDoi"> DOI: 10.3390/v12040452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3390%2Fv12040452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1SjtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=452&author=S.+K.+V.+Vernekarauthor=R.+L.+Sahaniauthor=M.+C.+Caseyauthor=J.+Kankanalaauthor=L.+Wangauthor=K.+A.+Kirbyauthor=H.+Duauthor=H.+Zhangauthor=P.+R.+Tedburyauthor=J.+Xieauthor=S.+G.+Sarafianosauthor=Z.+Wang&title=Toward+Structurally+Novel+and+Metabolically+Stable+HIV-1+Capsid-Targeting+Small+Molecules&doi=10.3390%2Fv12040452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Toward structurally novel and metabolically stable HIV-1 capsid-targeting small molecules</span></div><div class="casAuthors">Vernekar, Sanjeev Kumar V.; Sahani, Rajkumar Lalji; Casey, Mary C.; Kankanala, Jayakanth; Wang, Lei; Kirby, Karen A.; Du, Haijuan; Zhang, Huanchun; Tedbury, Philip R.; Xie, Jiashu; Sarafianos, Stefan G.; Wang, Zhengqiang</div><div class="citationInfo"><span class="NLM_cas:title">Viruses</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">452</span>CODEN:
                <span class="NLM_cas:coden">VIRUBR</span>;
        ISSN:<span class="NLM_cas:issn">1999-4915</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">HIV-1 capsid protein (CA) plays an important role in many steps of viral replication and represents an appealing antiviral target.  Several CA-targeting small mols. of various chemotypes have been studied, but the peptidomimetic PF74 has drawn particular interest due to its potent antiviral activity, well-characterized binding mode, and unique mechanism of action.  Importantly, PF74 competes against important host factors for binding, conferring highly desirable antiviral phenotypes.  However, further development of PF74 is hindered by its prohibitively poor metabolic stability, which necessitates the search for structurally novel and metabolically stable chemotypes.  We have conducted a pharmacophore-based shape similarity search for compds. mimicking PF74.  We report herein the analog synthesis and structure-activity relationship (SAR) of two hits from the search, and a third hit designed via mol. hybridization.  All analogs were characterized for their effect on CA hexamer stability, antiviral activity, and cytotoxicity.  These assays identified three active compds. that moderately stabilize CA hexamer and inhibit HIV-1.  The most potent analog (10) inhibited HIV-1 comparably to PF74 but demonstrated drastically improved metabolic stability in liver microsomes (31 min vs. 0.7 min t1/2).  Collectively, the current studies identified a structurally novel and metabolically stable PF74-like chemotype for targeting HIV-1 CA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNXpsA3fUhj7Vg90H21EOLACvtfcHk0ljL1ZIZP3XNZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1SjtLbM&md5=5ab729e4264f03f493a34ec090f5f3e2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3390%2Fv12040452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fv12040452%26sid%3Dliteratum%253Aachs%26aulast%3DVernekar%26aufirst%3DS.%2BK.%2BV.%26aulast%3DSahani%26aufirst%3DR.%2BL.%26aulast%3DCasey%26aufirst%3DM.%2BC.%26aulast%3DKankanala%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DKirby%26aufirst%3DK.%2BA.%26aulast%3DDu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DTedbury%26aufirst%3DP.%2BR.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DToward%2520Structurally%2520Novel%2520and%2520Metabolically%2520Stable%2520HIV-1%2520Capsid-Targeting%2520Small%2520Molecules%26jtitle%3DViruses%26date%3D2020%26volume%3D12%26spage%3D452%26doi%3D10.3390%2Fv12040452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalloum, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and structure-activity relationships of 4-phenyl-1H-1,2,3-triazole phenylalanine derivatives as novel HIV-1 capsid inhibitors with promising antiviral activities</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>190</i></span>,  <span class="NLM_fpage">112085</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.ejmech.2020.112085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=32066010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFGqur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2020&pages=112085&author=L.+Sunauthor=T.+Huangauthor=A.+Dickauthor=M.+E.+Meuserauthor=W.+A.+Zalloumauthor=C.+H.+Chenauthor=X.+Dingauthor=P.+Gaoauthor=S.+Cocklinauthor=K.+H.+Leeauthor=P.+Zhanauthor=X.+Liu&title=Design%2C+synthesis+and+structure-activity+relationships+of+4-phenyl-1H-1%2C2%2C3-triazole+phenylalanine+derivatives+as+novel+HIV-1+capsid+inhibitors+with+promising+antiviral+activities&doi=10.1016%2Fj.ejmech.2020.112085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and structure-activity relationships of 4-phenyl-1H-1,2,3-triazole phenylalanine derivatives as novel HIV-1 capsid inhibitors with promising antiviral activities</span></div><div class="casAuthors">Sun, Lin; Huang, Tianguang; Dick, Alexej; Meuser, Megan E.; Zalloum, Waleed A.; Chen, Chin-Ho; Ding, Xiao; Gao, Ping; Cocklin, Simon; Lee, Kuo-Hsiung; Zhan, Peng; Liu, Xinyong</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112085</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">HIV-1 CA is involved in different stages of the viral replication cycle, performing essential roles in both early (uncoating, reverse transcription, nuclear import, integration) and late events (assembly).  Recent efforts have demonstrated HIV-1 CA protein as a prospective therapeutic target for the development of new antivirals.  The most extensively studied CA inhibitor, PF-3450074 (PF-74, discovered by Pfizer), that targets an inter-protomer pocket within the CA hexamer.  Herein we reported the design, synthesis, and biol. evaluation of a series of 4-phenyl-1H-1,2,3-triazole phenylalanine derivs. as HIV-1 CA inhibitors based on PF-74 scaffold.  Most of the analogs demonstrated potent antiviral activities, among them, the anti-HIV-1 activity of (I) (R1 = C:O-2-F-C6H4) (EC50 = 3.13μM) is particularly prominent.  The SPR binding assay of selected compds. I (R1 = C:O-2-F-C6H4, C:O-3-F-C6H4) and (II) suggested direct and effective interaction with recombinant CA proteins.  The mechanism of action studies also demonstrated that I (R1 = C:O-2-F-C6H4) displays the effects in both the early and late stages of HIV-1 replication.  To explore the potential binding mode of the here presented analogs, I (R1 = C:O-2-F-C6H4) was analyzed by MD simulation to predict its binding to the active site of HIV-1 CA monomer.  In conclusion, this novel series of antivirals can serve as a starting point for the development of a new generation of HIV-1 treatment regimen and highlights the potentiality of CA as a therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnj7yCN6FM7Vg90H21EOLACvtfcHk0lgDvKMLbwdqYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFGqur8%253D&md5=0eece1c4f60dd7d97f80015fb915ed6d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112085%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DDick%26aufirst%3DA.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DZalloum%26aufirst%3DW.%2BA.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DDesign%252C%2520synthesis%2520and%2520structure-activity%2520relationships%2520of%25204-phenyl-1H-1%252C2%252C3-triazole%2520phenylalanine%2520derivatives%2520as%2520novel%2520HIV-1%2520capsid%2520inhibitors%2520with%2520promising%2520antiviral%2520activities%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D190%26spage%3D112085%26doi%3D10.1016%2Fj.ejmech.2020.112085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalloum, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherukupalli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Clercq, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannecouque, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Mechanism Study of Benzenesulfonamide-Containing Phenylalanine Derivatives as Novel HIV-1 Capsid Inhibitors with Improved Antiviral Activities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">4790</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00015</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnt12ru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=4790&author=L.+Sunauthor=A.+Dickauthor=M.+E.+Meuserauthor=T.+Huangauthor=W.+A.+Zalloumauthor=C.+H.+Chenauthor=S.+Cherukupalliauthor=S.+Xuauthor=X.+Dingauthor=P.+Gaoauthor=D.+Kangauthor=E.+De+Clercqauthor=C.+Pannecouqueauthor=S.+Cocklinauthor=K.+H.+Leeauthor=X.+Liuauthor=P.+Zhan&title=Design%2C+Synthesis%2C+and+Mechanism+Study+of+Benzenesulfonamide-Containing+Phenylalanine+Derivatives+as+Novel+HIV-1+Capsid+Inhibitors+with+Improved+Antiviral+Activities&doi=10.1021%2Facs.jmedchem.0c00015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Mechanism Study of Benzenesulfonamide-Containing Phenylalanine Derivatives as Novel HIV-1 Capsid Inhibitors with Improved Antiviral Activities</span></div><div class="casAuthors">Sun, Lin; Dick, Alexej; Meuser, Megan E.; Huang, Tianguang; Zalloum, Waleed A.; Chen, Chin-Ho; Cherukupalli, Srinivasulu; Xu, Shujing; Ding, Xiao; Gao, Ping; Kang, Dongwei; De Clercq, Erik; Pannecouque, Christophe; Cocklin, Simon; Lee, Kuo-Hsiung; Liu, Xinyong; Zhan, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4790-4810</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 CA protein has gained remarkable attention as a promising therapeutic target for the development of new antivirals, due to its pivotal roles in HIV-1 replication (structural and regulatory).  Herein, we report the design and synthesis of three series of benzenesulfonamide-contg. phenylalanine derivs. obtained by further structural modifications of PF-74 to aid in the discovery of more potent and drug-like HIV-1 CA inhibitors.  Structure-activity relationship studies of these compds. led to the identification of new phenylalanine derivs. with a piperazinone moiety, represented by compd. (I), which exhibited anti-HIV-1NL4-3 activity 5.78-fold better than PF-74.  Interestingly, I also showed anti-HIV-2ROD activity (EC50 = 31 nΜ), with almost 120 times increased potency over PF-74.  However, due to the higher significance of HIV-1 as compared to HIV-2 for the human population, this manuscript focused on the mechanism of action of our compds. in the context of HIV-1.  SPR studies on representative compds. confirmed CA as the binding target.  The action stage detn. assay demonstrated that these inhibitors exhibited antiviral activities with a dual-stage inhibition profile.  The early-stage inhibitory activity of compd. I was 6.25 times more potent as compared to PF-74, but appears to work via accelerating capsid core assembly rather than stabilization.  However, the mechanism by which they exert their antiviral activity in the late-stage appears to be the same as PF-74 with less infectious HIV-1 virions are produced in their presence as judged p24 content studies.  MD simulations provided the key rationale for the promising antiviral potency of I.  Addnl., I exhibited modest increase in HLM and human plasma metabolic stabilities as compared to PF-74, as well as moderately improved pharmacokinetic profile, favorable oral bioavailability, and no acute toxicity.  These studies provide insights and serves as a starting point for subsequent medicinal chem. efforts in optimizing these promising HIV inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwKdnsc-Iy-LVg90H21EOLACvtfcHk0lgDvKMLbwdqYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnt12ru7s%253D&md5=77a89a34489fc04cbe4c29b5bdb4b396</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00015%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DDick%26aufirst%3DA.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DZalloum%26aufirst%3DW.%2BA.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DCherukupalli%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DDe%2BClercq%26aufirst%3DE.%26aulast%3DPannecouque%26aufirst%3DC.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DP.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Mechanism%2520Study%2520of%2520Benzenesulfonamide-Containing%2520Phenylalanine%2520Derivatives%2520as%2520Novel%2520HIV-1%2520Capsid%2520Inhibitors%2520with%2520Improved%2520Antiviral%2520Activities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26spage%3D4790%26doi%3D10.1021%2Facs.jmedchem.0c00015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalloum, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel 1,4-disubstituted 1,2,3-triazole phenylalanine derivatives as HIV-1 capsid inhibitors</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">28961</span>– <span class="NLM_lpage">28986</span>, <span class="refDoi"> DOI: 10.1039/C9RA05869A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1039%2FC9RA05869A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=32089839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslyitr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=28961-28986&author=X.+Jiangauthor=G.+Wuauthor=W.+A.+Zalloumauthor=M.+E.+Meuserauthor=A.+Dickauthor=L.+Sunauthor=C.+H.+Chenauthor=D.+Kangauthor=L.+Jingauthor=R.+Jiaauthor=S.+Cocklinauthor=K.+H.+Leeauthor=X.+Liuauthor=P.+Zhan&title=Discovery+of+novel+1%2C4-disubstituted+1%2C2%2C3-triazole+phenylalanine+derivatives+as+HIV-1+capsid+inhibitors&doi=10.1039%2FC9RA05869A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel 1,4-disubstituted 1,2,3-triazole phenylalanine derivatives as HIV-1 capsid inhibitors</span></div><div class="casAuthors">Jiang, Xiangyi; Wu, Gaochan; Zalloum, Waleed A.; Meuser, Megan E.; Dick, Alexej; Sun, Lin; Chen, Chin-Ho; Kang, Dongwei; Jing, Lanlan; Jia, Ruifang; Cocklin, Simon; Lee, Kuo-Hsiung; Liu, Xinyong; Zhan, Peng</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">28961-28986</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The HIV-1 capsid (CA) protein plays crucial roles in both early and late stages of the viral life cycle, which has intrigued researchers to develop anti-HIV drugs that target it.  Accordingly, in this research, we report the design, synthesis and biol. evaluation of a series of novel phenylalanine derivs. as HIV-1 CA protein inhibitors using the Cu(I)-catalyzed azide and alkyne 1,3-dipolar cycloaddn. (CuAAC) reaction.  Among this series of inhibitors, compd. I displayed a remarkable anti-HIV activity (EC50 = 2.13μM, CC50 > 35.49μM).  Furthermore, surface plasmon resonance (SPR) binding assays showed that compds. I and PF-74 (II, lead compd.) have similar affinities to the HIV-1 CA monomer.  Further investigation showed that the weak permeability and water soly. of representative compds. were probably the important factors that restricted their cell-based activity.  Preliminary structure-activity relationships (SARs) were inferred based on the activities of these compds. and their known structure.  The most promising new compd. was studied with mol. dynamics simulation (MD) to det. the preferred interactions with the drug target.  Finally, the activities of members of this series of inhibitors were deeply inspected to find the potential reasons for their anti-HIV-1 activity from various perspectives.  This highlights the important factors required to design compds. with improved potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraZBDm_k3IR7Vg90H21EOLACvtfcHk0ljU4Pr2n4pavw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslyitr3K&md5=c3c80658dcfe4edeb06f7cd9ea348082</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1039%2FC9RA05869A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9RA05869A%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DZalloum%26aufirst%3DW.%2BA.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DDick%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DJing%26aufirst%3DL.%26aulast%3DJia%26aufirst%3DR.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520novel%25201%252C4-disubstituted%25201%252C2%252C3-triazole%2520phenylalanine%2520derivatives%2520as%2520HIV-1%2520capsid%2520inhibitors%26jtitle%3DRSC%2520Adv.%26date%3D2019%26volume%3D9%26spage%3D28961%26epage%3D28986%26doi%3D10.1039%2FC9RA05869A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalloum, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of phenylalanine derivatives as potent HIV-1 capsid inhibitors from click chemistry-based compound library</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">478</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.ejmech.2018.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=30243152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOltL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2018&pages=478-492&author=G.+Wuauthor=W.+A.+Zalloumauthor=M.+E.+Meuserauthor=L.+Jingauthor=D.+Kangauthor=C.+H.+Chenauthor=Y.+Tianauthor=F.+Zhangauthor=S.+Cocklinauthor=K.+H.+Leeauthor=X.+Liuauthor=P.+Zhan&title=Discovery+of+phenylalanine+derivatives+as+potent+HIV-1+capsid+inhibitors+from+click+chemistry-based+compound+library&doi=10.1016%2Fj.ejmech.2018.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of phenylalanine derivatives as potent HIV-1 capsid inhibitors from Click chemistry-based compound library</span></div><div class="casAuthors">Wu, Gaochan; Zalloum, Waleed A.; Meuser, Megan E.; Jing, Lanlan; Kang, Dongwei; Chen, Chin-Ho; Tian, Ye; Zhang, Fangfang; Cocklin, Simon; Lee, Kuo-Hsiung; Liu, Xinyong; Zhan, Peng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">478-492</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The HIV-1 capsid (CA) protein plays essential roles in both early and late stages of HIV-1 replication and is considered an important, clin. unexploited therapeutic target.  As such, small drug-like mols. that inhibit this crit. HIV-1 protein have become a priority for several groups.  Therefore, in this study we explore small mol. targeting of the CA protein, and in particular a very attractive inter-protomer pocket.  We report the design, parallel synthesis, and anti-HIV-1 activity evaluation of a series of novel phenylalanine derivs. as HIV-1 CA protein inhibitors synthesized via Cu(I)-catalyzed alkyne-azide 1,3-dipolar cycloaddn. (CuAAC) reaction.  We demonstrate robust inhibitory activity over a range of potencies against the HIV-1 NL4-3 ref. strain.  In particular, compd. (I) exhibited the greatest potency and lowest toxicity within this new series with an EC50 value of 4.33 μM and CC50 value of >57.74 μM (SI > 13.33).  These values are very similar to the lead compd. PF-74 (EC50 = 5.95 μM, CC50 > 70.50 μM, SI > 11.85) in our assay, despite significant structural difference.  Furthermore, we demonstrate via surface plasmon resonance (SPR) binding assays that I interacts robustly with recombinant HIV-1 CA and exhibits antiviral activity in both the early and late stages of HIV-1 replication.  Overall, the novel parallel synthesis and structure-activity relationships (SARs) identified within this study set the foundation for further rational optimization and discovery of CA-targeting compds. with improved potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog7rdhyTNYnbVg90H21EOLACvtfcHk0ljU4Pr2n4pavw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOltL%252FP&md5=6a45e0472739f33bfcc7c816c3b1d102</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%26aulast%3DZalloum%26aufirst%3DW.%2BA.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DJing%26aufirst%3DL.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520phenylalanine%2520derivatives%2520as%2520potent%2520HIV-1%2520capsid%2520inhibitors%2520from%2520click%2520chemistry-based%2520compound%2520library%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D158%26spage%3D478%26epage%3D492%26doi%3D10.1016%2Fj.ejmech.2018.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Link, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somoza, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bam, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canales, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brizgys, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graupe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morganelli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halcomb, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazerwith, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novikov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villaseñor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mwangi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedziela-Majka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papalia, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavitt, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stepan, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagratis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clancy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmadyar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, T. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolckenhauer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callebaut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ram, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinclair, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruane, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crofoot, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brainard, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundquist, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakowicz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chester, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daar, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yant, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span> <span> </span><span class="NLM_article-title">Clinical targeting of HIV capsid protein with a long-acting small molecule</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">614</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2443-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1038%2Fs41586-020-2443-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=32612233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlShtL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=614&author=J.+O.+Linkauthor=M.+S.+Rheeauthor=W.+C.+Tseauthor=J.+Zhengauthor=J.+R.+Somozaauthor=W.+Roweauthor=R.+Begleyauthor=A.+Chiuauthor=A.+Mulatoauthor=D.+Hansenauthor=E.+Singerauthor=L.+K.+Tsaiauthor=R.+A.+Bamauthor=C.+H.+Chouauthor=E.+Canalesauthor=G.+Brizgysauthor=J.+R.+Zhangauthor=J.+Liauthor=M.+Graupeauthor=P.+Morganelliauthor=Q.+Liuauthor=Q.+Wuauthor=R.+L.+Halcombauthor=R.+D.+Saitoauthor=S.+D.+Schroederauthor=S.+E.+Lazerwithauthor=S.+Bondyauthor=D.+Jinauthor=M.+Hungauthor=N.+Novikovauthor=X.+Liuauthor=A.+G.+Villase%C3%B1orauthor=C.+E.+Cannizzaroauthor=E.+Y.+Huauthor=R.+L.+Andersonauthor=T.+C.+Applebyauthor=B.+Luauthor=J.+Mwangiauthor=A.+Liclicanauthor=A.+Niedziela-Majkaauthor=G.+A.+Papaliaauthor=M.+H.+Wongauthor=S.+A.+Leavittauthor=Y.+Xuauthor=D.+Koditekauthor=G.+J.+Stepanauthor=H.+Yuauthor=N.+Pagratisauthor=S.+Clancyauthor=S.+Ahmadyarauthor=T.+Z.+Caiauthor=S.+Sellersauthor=S.+A.+Wolckenhauerauthor=J.+Lingauthor=C.+Callebautauthor=N.+Margotauthor=R.+R.+Ramauthor=Y.+P.+Liuauthor=R.+Hylandauthor=G.+I.+Sinclairauthor=P.+J.+Ruaneauthor=G.+E.+Crofootauthor=C.+K.+McDonaldauthor=D.+M.+Brainardauthor=L.+Ladauthor=S.+Swaminathanauthor=W.+I.+Sundquistauthor=R.+Sakowiczauthor=A.+E.+Chesterauthor=W.+E.+Leeauthor=E.+S.+Daarauthor=S.+R.+Yantauthor=T.+Cihlar&title=Clinical+targeting+of+HIV+capsid+protein+with+a+long-acting+small+molecule&doi=10.1038%2Fs41586-020-2443-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical targeting of HIV capsid protein with a long-acting small molecule</span></div><div class="casAuthors">Link, John O.; Rhee, Martin S.; Tse, Winston C.; Zheng, Jim; Somoza, John R.; Rowe, William; Begley, Rebecca; Chiu, Anna; Mulato, Andrew; Hansen, Derek; Singer, Eric; Tsai, Luong K.; Bam, Rujuta A.; Chou, Chien-Hung; Canales, Eda; Brizgys, Gediminas; Zhang, Jennifer R.; Li, Jiayao; Graupe, Michael; Morganelli, Philip; Liu, Qi; Wu, Qiaoyin; Halcomb, Randall L.; Saito, Roland D.; Schroeder, Scott D.; Lazerwith, Scott E.; Bondy, Steven; Jin, Debi; Hung, Magdeleine; Novikov, Nikolai; Liu, Xiaohong; Villasenor, Armando G.; Cannizzaro, Carina E.; Hu, Eric Y.; Anderson, Robert L.; Appleby, Todd C.; Lu, Bing; Mwangi, Judy; Liclican, Albert; Niedziela-Majka, Anita; Papalia, Giuseppe A.; Wong, Melanie H.; Leavitt, Stephanie A.; Xu, Yili; Koditek, David; Stepan, George J.; Yu, Helen; Pagratis, Nikos; Clancy, Sheila; Ahmadyar, Shekeba; Cai, Terrence Z.; Sellers, Scott; Wolckenhauer, Scott A.; Ling, John; Callebaut, Christian; Margot, Nicolas; Ram, Renee R.; Liu, Ya-Pei; Hyland, Rob; Sinclair, Gary I.; Ruane, Peter J.; Crofoot, Gordon E.; McDonald, Cheryl K.; Brainard, Diana M.; Lad, Latesh; Swaminathan, Swami; Sundquist, Wesley I.; Sakowicz, Roman; Chester, Anne E.; Lee, William E.; Daar, Eric S.; Yant, Stephen R.; Cihlar, Tomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">7822</span>),
    <span class="NLM_cas:pages">614-618</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Oral antiretroviral agents provide life-saving treatments for millions of people living with HIV, and can prevent new infections via pre-exposure prophylaxis1-5.  However, some people living with HIV who are heavily treatment-experienced have limited or no treatment options, owing to multidrug resistance6.  In addn., suboptimal adherence to oral daily regimens can neg. affect the outcome of treatment-which contributes to virol. failure, resistance generation and viral transmission-as well as of pre-exposure prophylaxis, leading to new infections1,2,4,7-9.  Long-acting agents from new antiretroviral classes can provide much-needed treatment options for people living with HIV who are heavily treatment-experienced, and addnl. can improve adherence10.  Here we describe GS-6207, a small mol. that disrupts the functions of HIV capsid protein and is amenable to long-acting therapy owing to its high potency, low in vivo systemic clearance and slow release kinetics from the s.c. injection site.  Drawing on X-ray crystallog. information, we designed GS-6207 to bind tightly at a conserved interface between capsid protein monomers, where it interferes with capsid-protein-mediated interactions between proteins that are essential for multiple phases of the viral replication cycle.  GS-6207 exhibits antiviral activity at picomolar concns. against all subtypes of HIV-1 that we tested, and shows high synergy and no cross-resistance with approved antiretroviral drugs.  In phase-1 clin. studies, monotherapy with a single s.c. dose of GS-6207 (450 mg) resulted in a mean log10-transformed redn. of plasma viral load of 2.2 after 9 days, and showed sustained plasma exposure at antivirally active concns. for more than 6 mo.  These results provide clin. validation for therapies that target the functions of HIV capsid protein, and demonstrate the potential of GS-6207 as a long-acting agent to treat or prevent infection with HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDdDh1BVoW47Vg90H21EOLACvtfcHk0liciwBSGQRN7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlShtL7K&md5=24a655318952e26a71587a4bc44307e2</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2443-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2443-1%26sid%3Dliteratum%253Aachs%26aulast%3DLink%26aufirst%3DJ.%2BO.%26aulast%3DRhee%26aufirst%3DM.%2BS.%26aulast%3DTse%26aufirst%3DW.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DSomoza%26aufirst%3DJ.%2BR.%26aulast%3DRowe%26aufirst%3DW.%26aulast%3DBegley%26aufirst%3DR.%26aulast%3DChiu%26aufirst%3DA.%26aulast%3DMulato%26aufirst%3DA.%26aulast%3DHansen%26aufirst%3DD.%26aulast%3DSinger%26aufirst%3DE.%26aulast%3DTsai%26aufirst%3DL.%2BK.%26aulast%3DBam%26aufirst%3DR.%2BA.%26aulast%3DChou%26aufirst%3DC.%2BH.%26aulast%3DCanales%26aufirst%3DE.%26aulast%3DBrizgys%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DGraupe%26aufirst%3DM.%26aulast%3DMorganelli%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DHalcomb%26aufirst%3DR.%2BL.%26aulast%3DSaito%26aufirst%3DR.%2BD.%26aulast%3DSchroeder%26aufirst%3DS.%2BD.%26aulast%3DLazerwith%26aufirst%3DS.%2BE.%26aulast%3DBondy%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DD.%26aulast%3DHung%26aufirst%3DM.%26aulast%3DNovikov%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DVillase%25C3%25B1or%26aufirst%3DA.%2BG.%26aulast%3DCannizzaro%26aufirst%3DC.%2BE.%26aulast%3DHu%26aufirst%3DE.%2BY.%26aulast%3DAnderson%26aufirst%3DR.%2BL.%26aulast%3DAppleby%26aufirst%3DT.%2BC.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DMwangi%26aufirst%3DJ.%26aulast%3DLiclican%26aufirst%3DA.%26aulast%3DNiedziela-Majka%26aufirst%3DA.%26aulast%3DPapalia%26aufirst%3DG.%2BA.%26aulast%3DWong%26aufirst%3DM.%2BH.%26aulast%3DLeavitt%26aufirst%3DS.%2BA.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DStepan%26aufirst%3DG.%2BJ.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DPagratis%26aufirst%3DN.%26aulast%3DClancy%26aufirst%3DS.%26aulast%3DAhmadyar%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DT.%2BZ.%26aulast%3DSellers%26aufirst%3DS.%26aulast%3DWolckenhauer%26aufirst%3DS.%2BA.%26aulast%3DLing%26aufirst%3DJ.%26aulast%3DCallebaut%26aufirst%3DC.%26aulast%3DMargot%26aufirst%3DN.%26aulast%3DRam%26aufirst%3DR.%2BR.%26aulast%3DLiu%26aufirst%3DY.%2BP.%26aulast%3DHyland%26aufirst%3DR.%26aulast%3DSinclair%26aufirst%3DG.%2BI.%26aulast%3DRuane%26aufirst%3DP.%2BJ.%26aulast%3DCrofoot%26aufirst%3DG.%2BE.%26aulast%3DMcDonald%26aufirst%3DC.%2BK.%26aulast%3DBrainard%26aufirst%3DD.%2BM.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DSundquist%26aufirst%3DW.%2BI.%26aulast%3DSakowicz%26aufirst%3DR.%26aulast%3DChester%26aufirst%3DA.%2BE.%26aulast%3DLee%26aufirst%3DW.%2BE.%26aulast%3DDaar%26aufirst%3DE.%2BS.%26aulast%3DYant%26aufirst%3DS.%2BR.%26aulast%3DCihlar%26aufirst%3DT.%26atitle%3DClinical%2520targeting%2520of%2520HIV%2520capsid%2520protein%2520with%2520a%2520long-acting%2520small%2520molecule%26jtitle%3DNature%26date%3D2020%26spage%3D614%26doi%3D10.1038%2Fs41586-020-2443-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallazzi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neogi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sönnerborg, A.</span></span> <span> </span><span class="NLM_article-title">GS-CA Compounds: First-In-Class HIV-1 Capsid Inhibitors Covering Multiple Grounds</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1227</span>, <span class="refDoi"> DOI: 10.3389/fmicb.2019.01227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3389%2Ffmicb.2019.01227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=31312185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A280%3ADC%252BB3MzotVWnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1227&author=K.+Singhauthor=F.+Gallazziauthor=K.+J.+Hillauthor=D.+H.+Burkeauthor=M.+J.+Langeauthor=T.+P.+Quinnauthor=U.+Neogiauthor=A.+S%C3%B6nnerborg&title=GS-CA+Compounds%3A+First-In-Class+HIV-1+Capsid+Inhibitors+Covering+Multiple+Grounds&doi=10.3389%2Ffmicb.2019.01227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">GS-CA Compounds: First-In-Class HIV-1 Capsid Inhibitors Covering Multiple Grounds</span></div><div class="casAuthors">Singh Kamal; Hill Kyle J; Burke Donald H; Lange Margaret J; Singh Kamal; Gallazzi Fabio; Hill Kyle J; Burke Donald H; Lange Margaret J; Singh Kamal; Neogi Ujjwal; Sonnerborg Anders; Gallazzi Fabio; Quinn Thomas P; Sonnerborg Anders</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1227</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">Recently reported HIV-1 capsid (CA) inhibitors GS-CA1 and GS-6207 (an analog of GS-CA1) are first-in-class compounds with long-acting potential.  Reportedly, both compounds have greater potency than currently approved anti-HIV drugs.  Due to the limited access to experimental data and the compounds themselves, a detailed mechanism of their inhibition is yet to be delineated.  Using crystal structures of capsid-hexamers bound to well-studied capsid inhibitor PF74 and molecular modeling, we predict that GS-CA compounds bind in the pocket that is shared by previously reported CA inhibitors and host factors.  Additionally, comparative modeling suggests that GS-CA compounds have unique structural features contributing to interactions with capsid.  To test their proposed binding mode, we also report the design of a cyclic peptide combining structural units from GS-CA compounds, host factors, and previously reported capsid inhibitors.  This peptide (Pep-1) binds CA-hexamer with a docking score comparable to GS-CA compounds.  Affinity determination by MicroScale thermophoresis (MST) assays showed that CA binds Pep-1 with a ~7-fold better affinity than well-studied capsid inhibitor PF74, suggesting that it can be developed as a possible CA inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSM3uMyo0R5PhWfpMXQLRstfW6udTcc2ea07wcWMGPivrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzotVWnug%253D%253D&md5=ef152804cda76b304c928ba1a0b7c654</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2019.01227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2019.01227%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DK.%26aulast%3DGallazzi%26aufirst%3DF.%26aulast%3DHill%26aufirst%3DK.%2BJ.%26aulast%3DBurke%26aufirst%3DD.%2BH.%26aulast%3DLange%26aufirst%3DM.%2BJ.%26aulast%3DQuinn%26aufirst%3DT.%2BP.%26aulast%3DNeogi%26aufirst%3DU.%26aulast%3DS%25C3%25B6nnerborg%26aufirst%3DA.%26atitle%3DGS-CA%2520Compounds%253A%2520First-In-Class%2520HIV-1%2520Capsid%2520Inhibitors%2520Covering%2520Multiple%2520Grounds%26jtitle%3DFront.%2520Microbiol.%26date%3D2019%26volume%3D10%26spage%3D1227%26doi%3D10.3389%2Ffmicb.2019.01227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karadsheh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Composition and Orientation of the Core Region of Novel HIV-1 Entry Inhibitors Influences Metabolic Stability</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1430</span>, <span class="refDoi"> DOI: 10.3390/molecules25061430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3390%2Fmolecules25061430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosFWqtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=1430&author=R.+Karadshehauthor=M.+E.+Meuserauthor=S.+Cocklin&title=Composition+and+Orientation+of+the+Core+Region+of+Novel+HIV-1+Entry+Inhibitors+Influences+Metabolic+Stability&doi=10.3390%2Fmolecules25061430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Composition and orientation of the core region of novel HIV-1 entry inhibitors influences metabolic stability</span></div><div class="casAuthors">Karadsheh, Rama; Meuser, Megan E.; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1430</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Fostemsavir/temsavir is an investigational HIV-1 entry inhibitor currently in late-stage clin. trials.  Although it holds promise to be a first-in-class Env-targeted entry inhibitor for the clinic, issues with bioavailability relegate its use to salvage therapies only.  As such, the development of a small mol. HIV-1 entry inhibitor that can be used in std. combination antiretroviral therapy (cART) remains a longstanding goal for the field.  We previously demonstrated the ability of extending the chemotypes available to this class of inhibitor as the first step towards this overarching goal.  In addn. to poor soly., metabolic stability is a crucial determinant of bioavailability.  Therefore, in this short communication, we assess the metabolic stabilities of five of our novel chemotype entry inhibitors.  We found that changing the piperazine core region of temsavir alters the stability of the compd. in human liver microsome assays.  Moreover, we identified an entry inhibitor with more than twice the metabolic stability of temsavir and demonstrated that the orientation of the core replacement is crit. for this increase.  This work further demonstrates the feasibility of our long-term goal-to design an entry inhibitor with improved drug-like qualities-and warrants expanded studies to achieve this.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocjASbOlm-v7Vg90H21EOLACvtfcHk0lgiUdc4Y3rcAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosFWqtr8%253D&md5=4298fd7cfd1cd7bf4b00c6b94c905b22</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3390%2Fmolecules25061430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules25061430%26sid%3Dliteratum%253Aachs%26aulast%3DKaradsheh%26aufirst%3DR.%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DComposition%2520and%2520Orientation%2520of%2520the%2520Core%2520Region%2520of%2520Novel%2520HIV-1%2520Entry%2520Inhibitors%2520Influences%2520Metabolic%2520Stability%26jtitle%3DMolecules%26date%3D2020%26volume%3D25%26spage%3D1430%26doi%3D10.3390%2Fmolecules25061430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Identification of a small molecule HIV-1 inhibitor that targets the capsid hexamer</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.12.087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.bmcl.2015.12.087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=26747394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=824-828&author=J.+P.+Xuauthor=J.+D.+Bransonauthor=R.+Lawrenceauthor=S.+Cocklin&title=Identification+of+a+small+molecule+HIV-1+inhibitor+that+targets+the+capsid+hexamer&doi=10.1016%2Fj.bmcl.2015.12.087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a small molecule HIV-1 inhibitor that targets the capsid hexamer</span></div><div class="casAuthors">Xu, Jimmy P.; Branson, Jeffrey D.; Lawrence, Rae; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">824-828</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The HIV-1 CA protein is an attractive therapeutic target for the development of new antivirals.  An inter-protomer pocket within the hexamer configuration of the CA, which is a binding site for key host dependency factors, is an esp. appealing region for small mol. targeting.  Using a field-based pharmacophore derived from an inhibitor known to interact with this region, coupled to biochem. and biol. assessment, the authors have identified a new compd. that inhibits HIV-1 infection and that targets the assembled CA hexamer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMgMK74WiSBbVg90H21EOLACvtfcHk0lgiUdc4Y3rcAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOltA%253D%253D&md5=c0a7dbedce0d39a14deaca41859374c2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.12.087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.12.087%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%2BP.%26aulast%3DBranson%26aufirst%3DJ.%2BD.%26aulast%3DLawrence%26aufirst%3DR.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520a%2520small%2520molecule%2520HIV-1%2520inhibitor%2520that%2520targets%2520the%2520capsid%2520hexamer%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D824%26epage%3D828%26doi%3D10.1016%2Fj.bmcl.2015.12.087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseright, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinter, A.</span></span> <span> </span><span class="NLM_article-title">Molecular field extrema as descriptors of biological activity: definition and validation</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">665</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1021/ci050357s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050357s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD28XisFCgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=665-676&author=T.+Cheeserightauthor=M.+Mackeyauthor=S.+Roseauthor=A.+Vinter&title=Molecular+field+extrema+as+descriptors+of+biological+activity%3A+definition+and+validation&doi=10.1021%2Fci050357s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Field Extrema as Descriptors of Biological Activity: Definition and Validation</span></div><div class="casAuthors">Cheeseright, Tim; Mackey, Mark; Rose, Sally; Vinter, Andy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">665-676</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The paper describes the generation of four types of three-dimensional mol. field descriptors or 'field points' as extrema of electrostatic, steric, and hydrophobic fields.  These field points are used to define the properties necessary for a mol. to bind in a characteristic way into a specified active site.  The hypothesis is that compds. showing a similar field point pattern are likely to bind at the same target site regardless of structure.  The methodol. to test this idea is illustrated using HIV NNRTI and thrombin ligands and validated across seven other targets.  From the in silico comparisons of field point overlays, the exptl. obsd. binding poses of these ligands in their resp. sites can be reproduced from pairwise comparisons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM6IB4phN5j7Vg90H21EOLACvtfcHk0lgiUdc4Y3rcAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisFCgsQ%253D%253D&md5=7cdf681c467143d9f95a3a2cbc5ae669</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci050357s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050357s%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseright%26aufirst%3DT.%26aulast%3DMackey%26aufirst%3DM.%26aulast%3DRose%26aufirst%3DS.%26aulast%3DVinter%26aufirst%3DA.%26atitle%3DMolecular%2520field%2520extrema%2520as%2520descriptors%2520of%2520biological%2520activity%253A%2520definition%2520and%2520validation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D665%26epage%3D676%26doi%3D10.1021%2Fci050357s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseright, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scoffin, R. A.</span></span> <span> </span><span class="NLM_article-title">High content pharmacophores from molecular fields: a biologically relevant method for comparing and understanding ligands</span>. <i>Curr. Comput.-Aided Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">190</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.2174/157340911796504314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.2174%2F157340911796504314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=21726191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCks7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=190-205&author=T.+J.+Cheeserightauthor=M.+D.+Mackeyauthor=R.+A.+Scoffin&title=High+content+pharmacophores+from+molecular+fields%3A+a+biologically+relevant+method+for+comparing+and+understanding+ligands&doi=10.2174%2F157340911796504314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">High content pharmacophores from molecular fields: a biologically relevant method for comparing and understanding ligands</span></div><div class="casAuthors">Cheeseright, Timothy J.; Mackey, Mark D.; Scoffin, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">190-205</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The question of how and why a small mol. binds to a protein is central to ligand-based drug discovery.  The traditional way of approaching these questions is pharmacophore anal.  However, pharmacophores as usually applied lack quantitation and subtlety.  An improvement is to consider the electrostatic and steric fields of the ligand directly.  Mol. fields provide a rich view of the potential interactions that a mol. can make and can be validated through exptl. data on mol. interactions and through quantum mechanics calcns.  A technique is presented in this review for comparing mols. using mol. fields and assigning similarity scores.  This high information content method can be used to align mols. for SAR anal., to det. the bioactive conformation from ligand data, and to screen large libraries of compds. for structurally unrelated actives.  An extension to allow interactive exploration of chem. space via bioisostere anal. is also reviewed.  Examples from the literature showing the success of these methods are presented, and future directions discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT897qyXPtyrVg90H21EOLACvtfcHk0ljNWkE6ZW74tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCks7%252FO&md5=a80c6b8718a9ef6bfe580a3378afa13f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.2174%2F157340911796504314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340911796504314%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseright%26aufirst%3DT.%2BJ.%26aulast%3DMackey%26aufirst%3DM.%2BD.%26aulast%3DScoffin%26aufirst%3DR.%2BA.%26atitle%3DHigh%2520content%2520pharmacophores%2520from%2520molecular%2520fields%253A%2520a%2520biologically%2520relevant%2520method%2520for%2520comparing%2520and%2520understanding%2520ligands%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2011%26volume%3D7%26spage%3D190%26epage%3D205%26doi%3D10.2174%2F157340911796504314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gres, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KewalRamani, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pornillos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span> <span> </span><span class="NLM_article-title">STRUCTURAL VIROLOGY. X-ray crystal structures of native HIV-1 capsid protein reveal conformational variability</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>349</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1126/science.aaa5936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1126%2Fscience.aaa5936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=26044298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOktLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=349&publication_year=2015&pages=99-103&author=A.+T.+Gresauthor=K.+A.+Kirbyauthor=V.+N.+KewalRamaniauthor=J.+J.+Tannerauthor=O.+Pornillosauthor=S.+G.+Sarafianos&title=STRUCTURAL+VIROLOGY.+X-ray+crystal+structures+of+native+HIV-1+capsid+protein+reveal+conformational+variability&doi=10.1126%2Fscience.aaa5936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray crystal structures of native HIV-1 capsid protein reveal conformational variability</span></div><div class="casAuthors">Gres, Anna T.; Kirby, Karen A.; KewalRamani, Vineet N.; Tanner, John J.; Pornillos, Owen; Sarafianos, Stefan G.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">349</span>
        (<span class="NLM_cas:issue">6243</span>),
    <span class="NLM_cas:pages">99-103</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The detailed mol. interactions between native HIV-1 capsid protein (CA) hexamers that shield the viral genome and proteins have been elusive.  We report crystal structures describing interactions between CA monomers related by sixfold symmetry within hexamers (intrahexamer) and threefold and twofold symmetry between neighboring hexamers (interhexamer).  The structures describe how CA builds hexagonal lattices, the foundation of mature capsids.  Lattice structure depends on an adaptable hydration layer modulating interactions among CA mols.  Disruption of this layer alters interhexamer interfaces, highlighting an inherent structural variability.  A CA-targeting antiviral affects capsid stability by binding across CA mols. and subtly altering interhexamer interfaces remote to the ligand-binding site.  Inherent structural plasticity, hydration layer rearrangement, and effector binding affect capsid stability and have functional implications for the retroviral life cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk_3_G_w_yabVg90H21EOLACvtfcHk0ljNWkE6ZW74tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOktLvF&md5=7de62606c7753476c56a61c9dcd92acc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaa5936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaa5936%26sid%3Dliteratum%253Aachs%26aulast%3DGres%26aufirst%3DA.%2BT.%26aulast%3DKirby%26aufirst%3DK.%2BA.%26aulast%3DKewalRamani%26aufirst%3DV.%2BN.%26aulast%3DTanner%26aufirst%3DJ.%2BJ.%26aulast%3DPornillos%26aufirst%3DO.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26atitle%3DSTRUCTURAL%2520VIROLOGY.%2520X-ray%2520crystal%2520structures%2520of%2520native%2520HIV-1%2520capsid%2520protein%2520reveal%2520conformational%2520variability%26jtitle%3DScience%26date%3D2015%26volume%3D349%26spage%3D99%26epage%3D103%26doi%3D10.1126%2Fscience.aaa5936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reulecke, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Towards an integrated description of hydrogen bonding and dehydration: decreasing false positives in virtual screening with the HYDE scoring function</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">885</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200700319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1002%2Fcmdc.200700319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=18384086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvVWmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=885-897&author=I.+Reuleckeauthor=G.+Langeauthor=J.+Albrechtauthor=R.+Kleinauthor=M.+Rarey&title=Towards+an+integrated+description+of+hydrogen+bonding+and+dehydration%3A+decreasing+false+positives+in+virtual+screening+with+the+HYDE+scoring+function&doi=10.1002%2Fcmdc.200700319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Towards an integrated description of hydrogen bonding and dehydration: decreasing false positives in virtual screening with the HYDE scoring function</span></div><div class="casAuthors">Reulecke, Ingo; Lange, Gudrun; Albrecht, Juergen; Klein, Robert; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">885-897</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We developed a new empirical scoring function, HYDE, for the evaluation of protein-ligand complexes.  HYDE ests. binding free energy based on two terms for dehydration and hydrogen bonding only.  The essential feature of this scoring function is the integrated use of log P-derived at. increments for the prediction of free dehydration energy and hydrogen bonding energy.  Taking the dehydration of atoms within the interface into account shows that some atoms contribute favorably to the overall score, while others contribute unfavorably.  For instance, hydrogen bond functions are penalized if they are dehydrated unless they can overcompensate this loss by forming a hydrogen bond with excellent geometry.  The main stabilizing contribution represents the removal of apolar groups from the water: the hydrophobic effect.  Initial studies using the DUD dataset show that with HYDE, there is a significant decrease in false positives, a reasonable categorization of compds. as either non-binders, weak, medium or strong binders, and in particular, there is a generally applicable and thermodynamically sensible cutoff score below which there is a high likelihood that the compd. is indeed a binder.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBdc8JePMq8rVg90H21EOLACvtfcHk0ljNWkE6ZW74tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvVWmt7g%253D&md5=4fac432f70e5fa2373ec46450418c868</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200700319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200700319%26sid%3Dliteratum%253Aachs%26aulast%3DReulecke%26aufirst%3DI.%26aulast%3DLange%26aufirst%3DG.%26aulast%3DAlbrecht%26aufirst%3DJ.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DTowards%2520an%2520integrated%2520description%2520of%2520hydrogen%2520bonding%2520and%2520dehydration%253A%2520decreasing%2520false%2520positives%2520in%2520virtual%2520screening%2520with%2520the%2520HYDE%2520scoring%2520function%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D885%26epage%3D897%26doi%3D10.1002%2Fcmdc.200700319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tuyishime, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danish, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Princiotto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankowski, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombart, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esikov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berniac, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of novel small-molecule HIV-1 entry inhibitors using field-based virtual screening and bioisosteric replacement</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">5439</span>– <span class="NLM_lpage">5445</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.bmcl.2014.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=25454268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOru7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=5439-5445&author=M.+Tuyishimeauthor=M.+Danishauthor=A.+Princiottoauthor=M.+K.+Mankowskiauthor=R.+Lawrenceauthor=H.+G.+Lombartauthor=K.+Esikovauthor=J.+Berniacauthor=K.+Liangauthor=J.+Jiauthor=R.+G.+Ptakauthor=N.+Madaniauthor=S.+Cocklin&title=Discovery+and+optimization+of+novel+small-molecule+HIV-1+entry+inhibitors+using+field-based+virtual+screening+and+bioisosteric+replacement&doi=10.1016%2Fj.bmcl.2014.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of novel small-molecule HIV-1 entry inhibitors using field-based virtual screening and bioisosteric replacement</span></div><div class="casAuthors">Tuyishime, Marina; Danish, Matt; Princiotto, Amy; Mankowski, Marie K.; Lawrence, Rae; Lombart, Henry-Georges; Esikov, Kirill; Berniac, Joel; Liang, Kuang; Ji, Jingjing; Ptak, Roger G.; Madani, Navid; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5439-5445</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">With the emergence of drug-resistant strains and the cumulative toxicities assocd. with current therapies, demand remains for new inhibitors of HIV-1 replication.  The inhibition of HIV-1 entry is an attractive, yet underexploited therapeutic approach with implications for salvage and preexposure prophylactic regimens, as well as topical microbicides.  Using the combination of a field-derived bioactive conformation template to perform virtual screening and iterative bioisosteric replacements, coupled with in silico predictions of absorption, distribution, metab., and excretion, we have identified new leads for HIV-1 entry inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9DnVfZzTHZrVg90H21EOLACvtfcHk0liLOd0LJDOBQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOru7rI&md5=a2759a4d6f613ae366dee15b8f66c2ed</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DTuyishime%26aufirst%3DM.%26aulast%3DDanish%26aufirst%3DM.%26aulast%3DPrinciotto%26aufirst%3DA.%26aulast%3DMankowski%26aufirst%3DM.%2BK.%26aulast%3DLawrence%26aufirst%3DR.%26aulast%3DLombart%26aufirst%3DH.%2BG.%26aulast%3DEsikov%26aufirst%3DK.%26aulast%3DBerniac%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DK.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DPtak%26aufirst%3DR.%2BG.%26aulast%3DMadani%26aufirst%3DN.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520novel%2520small-molecule%2520HIV-1%2520entry%2520inhibitors%2520using%2520field-based%2520virtual%2520screening%2520and%2520bioisosteric%2520replacement%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D5439%26epage%3D5445%26doi%3D10.1016%2Fj.bmcl.2014.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyzack, J. D.</span></span> <span> </span><span class="NLM_article-title">WhichP450: a multi-class categorical model to predict the major metabolising CYP450 isoform for a compound</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">537</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1007/s10822-018-0107-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1007%2Fs10822-018-0107-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=29464466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjtFCntr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=537-546&author=P.+A.+Huntauthor=M.+D.+Segallauthor=J.+D.+Tyzack&title=WhichP450%3A+a+multi-class+categorical+model+to+predict+the+major+metabolising+CYP450+isoform+for+a+compound&doi=10.1007%2Fs10822-018-0107-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-class categorical model to predict the major metabolising CYP450 isoform for a compound</span></div><div class="casAuthors">Hunt, Peter A.; Segall, Matthew D.; Tyzack, Jonathan D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">537-546</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">In the development of novel pharmaceuticals, the knowledge of how many, and which, Cytochrome P 450 isoforms are involved in the phase I metab. of a compd. is important.  Potential problems can arise if a compd. is metabolised predominantly by a single isoform in terms of drug-drug interactions or genetic polymorphisms that would lead to variations in exposure in the general population.  Combined with models of regioselectivities of metab. by each isoform, such a model would also aid in the prediction of the metabolites likely to be formed by P 450-mediated metab.  We describe the generation of a multi-class random forest model to predict which, out of a list of the seven leading Cytochrome P 450 isoforms, would be the major metabolising isoforms for a novel compd.  The model has a 76% success rate with a top-1 criterion and an 88% success rate for a top-2 criterion and shows significant enrichment over randomised models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouAXWrvgcNp7Vg90H21EOLACvtfcHk0liLOd0LJDOBQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjtFCntr4%253D&md5=a66eb70996e0984998cc88baae426055</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2Fs10822-018-0107-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-018-0107-0%26sid%3Dliteratum%253Aachs%26aulast%3DHunt%26aufirst%3DP.%2BA.%26aulast%3DSegall%26aufirst%3DM.%2BD.%26aulast%3DTyzack%26aufirst%3DJ.%2BD.%26atitle%3DWhichP450%253A%2520a%2520multi-class%2520categorical%2520model%2520to%2520predict%2520the%2520major%2520metabolising%2520CYP450%2520isoform%2520for%2520a%2520compound%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2018%26volume%3D32%26spage%3D537%26epage%3D546%26doi%3D10.1007%2Fs10822-018-0107-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulos, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevrioukova, I. F.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Inhibitor Design for Evaluation of a CYP3A4 Pharmacophore Model</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4210</span>– <span class="NLM_lpage">4220</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01146</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01146" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2mtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4210-4220&author=P.+Kaurauthor=A.+R.+Chamberlinauthor=T.+L.+Poulosauthor=I.+F.+Sevrioukova&title=Structure-Based+Inhibitor+Design+for+Evaluation+of+a+CYP3A4+Pharmacophore+Model&doi=10.1021%2Facs.jmedchem.5b01146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Inhibitor Design for Evaluation of a CYP3A4 Pharmacophore Model</span></div><div class="casAuthors">Kaur, Parminder; Chamberlin, A. Richard; Poulos, Thomas L.; Sevrioukova, Irina F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4210-4220</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human cytochrome P 450 3A4 (CYP3A4) is a key xenobiotic-metabolizing enzyme that oxidizes and clears the majority of drugs.  CYP3A4 inhibition may lead to drug-drug interactions, toxicity, and other adverse effects but, in some cases, could be beneficial and enhance therapeutic efficiency of coadministered pharmaceuticals that are metabolized by CYP3A4.  On the basis of the investigations of analogs of ritonavir, a potent CYP3A4 inactivator and pharmacoenhancer, the authors have built a pharmacophore model for a CYP3A4-specific inhibitor.  This study is the first attempt to test this model using a set of rationally designed compds.  The functional and structural data presented here agree well with the proposed pharmacophore.  In particular, the authors confirmed the importance of a flexible backbone, the H-bond donor/acceptor moiety, and aromaticity of the side group analogous to Phe-2 of ritonavir and demonstrated the leading role of hydrophobic interactions at the sites adjacent to the heme and phenylalanine cluster in the ligand binding process.  The x-ray structures of CYP3A4 bound to the rationally designed inhibitors provide deeper insights into the mechanism of the CYP3A4-ligand interaction.  Most importantly, two of the compds. I and II that are less complex than ritonavir have comparable submicromolar affinity and inhibitory potency for CYP3A4 and, thus, could serve as templates for synthesis of second generation inhibitors for further evaluation and optimization of the pharmacophore model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2kZVZ3OeVr7Vg90H21EOLACvtfcHk0liLOd0LJDOBQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2mtLvM&md5=4c9cd27c3a091aae985fcb70d7ffd957</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01146%26sid%3Dliteratum%253Aachs%26aulast%3DKaur%26aufirst%3DP.%26aulast%3DChamberlin%26aufirst%3DA.%2BR.%26aulast%3DPoulos%26aufirst%3DT.%2BL.%26aulast%3DSevrioukova%26aufirst%3DI.%2BF.%26atitle%3DStructure-Based%2520Inhibitor%2520Design%2520for%2520Evaluation%2520of%2520a%2520CYP3A4%2520Pharmacophore%2520Model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4210%26epage%3D4220%26doi%3D10.1021%2Facs.jmedchem.5b01146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span> <span> </span><span class="NLM_article-title">A semiempirical free energy force field with charge-based desolvation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1145</span>– <span class="NLM_lpage">1152</span>, <span class="refDoi"> DOI: 10.1002/jcc.20634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1002%2Fjcc.20634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=17274016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVSms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=1145-1152&author=R.+Hueyauthor=G.+M.+Morrisauthor=A.+J.+Olsonauthor=D.+S.+Goodsell&title=A+semiempirical+free+energy+force+field+with+charge-based+desolvation&doi=10.1002%2Fjcc.20634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A semiempirical free energy force field with charge-based desolvation</span></div><div class="casAuthors">Huey, Ruth; Morris, Garrett M.; Olson, Arthur J.; Goodsell, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1145-1152</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development and testing of a semiempirical free energy force field for use in AutoDock4 and similar grid-based docking methods.  The force field is based on a comprehensive thermodn. model that allows incorporation of intramol. energies into the predicted free energy of binding.  It also incorporates a charge-based method for evaluation of desolvation designed to use a typical set of atom types.  The method has been calibrated on a set of 188 diverse protein-ligand complexes of known structure and binding energy, and tested on a set of 100 complexes of ligands with retroviral proteases.  The force field shows improvement in redocking simulations over the previous AutoDock3 force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonvP0et7yst7Vg90H21EOLACvtfcHk0lhoWuQFXq4BTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVSms78%253D&md5=6fe6ef9bc0bd8149203a2b7351a2f15c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20634%26sid%3Dliteratum%253Aachs%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26atitle%3DA%2520semiempirical%2520free%2520energy%2520force%2520field%2520with%2520charge-based%2520desolvation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D1145%26epage%3D1152%26doi%3D10.1002%2Fjcc.20634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bikadi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazai, E.</span></span> <span> </span><span class="NLM_article-title">Application of the PM6 semi-empirical method to modeling proteins enhances docking accuracy of AutoDock</span>. <i>Aust. J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">15</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-1-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1186%2F1758-2946-1-15" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=15&author=Z.+Bikadiauthor=E.+Hazai&title=Application+of+the+PM6+semi-empirical+method+to+modeling+proteins+enhances+docking+accuracy+of+AutoDock&doi=10.1186%2F1758-2946-1-15"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-1-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-1-15%26sid%3Dliteratum%253Aachs%26aulast%3DBikadi%26aufirst%3DZ.%26aulast%3DHazai%26aufirst%3DE.%26atitle%3DApplication%2520of%2520the%2520PM6%2520semi-empirical%2520method%2520to%2520modeling%2520proteins%2520enhances%2520docking%2520accuracy%2520of%2520AutoDock%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D15%26doi%3D10.1186%2F1758-2946-1-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pornillos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganser-Pornillos, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banumathi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeager, M.</span></span> <span> </span><span class="NLM_article-title">Disulfide bond stabilization of the hexameric capsomer of human immunodeficiency virus</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>401</i></span>,  <span class="NLM_fpage">985</span>– <span class="NLM_lpage">995</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.06.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.jmb.2010.06.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=20600115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVaqs7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=401&publication_year=2010&pages=985-995&author=O.+Pornillosauthor=B.+K.+Ganser-Pornillosauthor=S.+Banumathiauthor=Y.+Huaauthor=M.+Yeager&title=Disulfide+bond+stabilization+of+the+hexameric+capsomer+of+human+immunodeficiency+virus&doi=10.1016%2Fj.jmb.2010.06.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Disulfide Bond Stabilization of the Hexameric Capsomer of Human Immunodeficiency Virus</span></div><div class="casAuthors">Pornillos, Owen; Ganser-Pornillos, Barbie K.; Banumathi, Sankaran; Hua, Yuanzi; Yeager, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">401</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">985-995</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The human immunodeficiency virus type 1 (HIV-1) capsid is modeled as a fullerene cone that is composed of ∼ 250 hexamers and 12 pentamers of the viral CA protein.  Structures of CA hexamers have been difficult to obtain because the hexamer-stabilizing interactions are inherently weak, and CA tends to spontaneously assemble into capsid-like particles.  Here, we describe a two-step biochem. strategy to obtain sol. CA hexamers for crystn.  First, the hexamer was stabilized by engineering disulfide cross-links (either A14C/E45C or A42C/T54C) between the N-terminal domains of adjacent subunits.  Second, the cross-linked hexamers were prevented from polymg. further into hyperstable capsid-like structures by mutations (W184A and M185A) that interfered with dimeric assocn. between the C-terminal domains that link adjacent hexamers.  The structures of two different cross-linked CA hexamers were nearly identical, and we combined the non-mutated portions of the structures to generate an at. resoln. model for the native hexamer.  This hybrid approach for structure detn. should be applicable to other viral capsomers and protein-protein complexes in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQVYZvCFtxWbVg90H21EOLACvtfcHk0lhoWuQFXq4BTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVaqs7jL&md5=43ce1eaed1b5aaa2032a37e2b6e44d5a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.06.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.06.042%26sid%3Dliteratum%253Aachs%26aulast%3DPornillos%26aufirst%3DO.%26aulast%3DGanser-Pornillos%26aufirst%3DB.%2BK.%26aulast%3DBanumathi%26aufirst%3DS.%26aulast%3DHua%26aufirst%3DY.%26aulast%3DYeager%26aufirst%3DM.%26atitle%3DDisulfide%2520bond%2520stabilization%2520of%2520the%2520hexameric%2520capsomer%2520of%2520human%2520immunodeficiency%2520virus%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D401%26spage%3D985%26epage%3D995%26doi%3D10.1016%2Fj.jmb.2010.06.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kortagere, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankowski, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships of a novel capsid targeted inhibitor of HIV-1 replication</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3080</span>– <span class="NLM_lpage">3090</span>, <span class="refDoi"> DOI: 10.1021/ci500437r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500437r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslemtrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3080-3090&author=S.+Kortagereauthor=J.+P.+Xuauthor=M.+K.+Mankowskiauthor=R.+G.+Ptakauthor=S.+Cocklin&title=Structure-activity+relationships+of+a+novel+capsid+targeted+inhibitor+of+HIV-1+replication&doi=10.1021%2Fci500437r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships of a Novel Capsid Targeted Inhibitor of HIV-1 Replication</span></div><div class="casAuthors">Kortagere, Sandhya; Xu, Jimmy P.; Mankowski, Marie K.; Ptak, Roger G.; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3080-3090</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the considerable successes of highly active antiretroviral therapy (HAART) for the treatment of HIV/AIDS, cumulative drug toxicities and the development of multidrug-resistant virus necessitate the search for new classes of antiretroviral agents with novel modes of action.  The HIV-1 capsid (CA) protein has been structurally and functionally characterized as a druggable target.  We have recently designed a novel small mol. inhibitor I-XW-053 using the hybrid structure based method to block the interface between CA N-terminal domains (NTD-NTD interface) with micromolar affinity.  In an effort to optimize and improve the efficacy of I-XW-053, we have developed the structure activity relationship of I-XW-053 compd. series using ligand efficiency methods.  Fifty-six analogs of I-XW-053 were designed that could be subclassified into four different core domains based on their ligand efficiency values computed as the ratio of binding efficiency (BEI) and surface efficiency (SEI) indexes.  Compd. 34 belonging to subcore-3 showed an 11-fold improvement over I-XW-053 in blocking HIV-1 replication in primary human peripheral blood mononuclear cells (PBMCs).  Surface plasmon resonance expts. confirmed the binding of compd. 34 to purified HIV-1 CA protein.  Mol. docking studies on compd. 34 and I-XW-053 to HIV-1 CA protein suggested that they both bind to NTD-NTD interface region but with different binding modes, which was further validated using site-directed mutagenesis studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokJa0411XKtbVg90H21EOLACvtfcHk0lhoWuQFXq4BTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslemtrnJ&md5=14c475975eaf073a56d42978f5e1390f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci500437r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500437r%26sid%3Dliteratum%253Aachs%26aulast%3DKortagere%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%2BP.%26aulast%3DMankowski%26aufirst%3DM.%2BK.%26aulast%3DPtak%26aufirst%3DR.%2BG.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DStructure-activity%2520relationships%2520of%2520a%2520novel%2520capsid%2520targeted%2520inhibitor%2520of%2520HIV-1%2520replication%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3080%26epage%3D3090%26doi%3D10.1021%2Fci500437r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M. D.</span></span> <span> </span><span class="NLM_article-title">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein–Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3036</span>– <span class="NLM_lpage">3050</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3036-3050&author=M.+R.+Bauerauthor=M.+D.+Mackey&title=Electrostatic+Complementarity+as+a+Fast+and+Effective+Tool+to+Optimize+Binding+and+Selectivity+of+Protein%E2%80%93Ligand+Complexes&doi=10.1021%2Facs.jmedchem.8b01925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein-Ligand Complexes</span></div><div class="casAuthors">Bauer, Matthias R.; Mackey, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3036-3050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Electrostatic interactions between small mols. and their resp. receptors are essential for mol. recognition and are also key contributors to the binding free energy.  Assessing the electrostatic match of protein-ligand complexes therefore provides important insights into why ligands bind and what can be changed to improve binding.  Ideally, the ligand and protein electrostatic potentials at the protein-ligand interaction interface should maximize their complementarity while minimizing desolvation penalties.  In this work, we present a fast and efficient tool to calc. and visualize the electrostatic complementarity (EC) of protein-ligand complexes.  We compiled benchmark sets demonstrating electrostatically driven structure-activity relationships (SAR) from literature data, including kinase, protein-protein interaction, and GPCR targets, and used these to demonstrate that the EC method can visualize, rationalize, and predict electrostatically driven ligand affinity changes and help to predict compd. selectivity.  The methodol. presented here for the anal. of EC is a powerful and versatile tool for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3pdUzJ9T0trVg90H21EOLACvtfcHk0ljCgSI3fZd-IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D&md5=a1abb5abe7fa8d73bee264bc94fd2da1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01925%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DMackey%26aufirst%3DM.%2BD.%26atitle%3DElectrostatic%2520Complementarity%2520as%2520a%2520Fast%2520and%2520Effective%2520Tool%2520to%2520Optimize%2520Binding%2520and%2520Selectivity%2520of%2520Protein%25E2%2580%2593Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3036%26epage%3D3050%26doi%3D10.1021%2Facs.jmedchem.8b01925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meuser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rashad, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Kinetic Characterization of Novel HIV-1 Entry Inhibitors: Discovery of a Relationship between Off-Rate and Potency</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1940</span>, <span class="refDoi"> DOI: 10.3390/molecules23081940</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.3390%2Fmolecules23081940" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGgtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1940&author=M.+E.+Meuserauthor=M.+B.+Murphyauthor=A.+A.+Rashadauthor=S.+Cocklin&title=Kinetic+Characterization+of+Novel+HIV-1+Entry+Inhibitors%3A+Discovery+of+a+Relationship+between+Off-Rate+and+Potency&doi=10.3390%2Fmolecules23081940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of novel HIV-1 entry inhibitors: discovery of a relationship between off-rate and potency</span></div><div class="casAuthors">Meuser, Megan E.; Murphy, Michael B.; Rashad, Adel A.; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1940/1-1940/16</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The entry of HIV-1 into permissible cells remains an extremely attractive and underexploited therapeutic intervention point.  We have previously demonstrated the ability to extend the chemotypes available for optimization in the entry inhibitor class using computational means.  Here, we continue this effort, designing and testing three novel compds. with the ability to inhibit HIV-1 entry.  We demonstrate that alteration of the core moiety of these entry inhibitors directly influences the potency of the compds., despite common proximal and distal groups.  Moreover, by establishing for the first time a surface plasmon resonance (SPR)-based interaction assay with sol. recombinant SOSIP Env trimers, we demonstrate that the off-rate (kd) parameter shows the strongest correlation with potency in an antiviral assay.  Finally, we establish an underappreciated relationship between the potency of a ligand and its degree of electrostatic complementarity (EC) with its target, the Env complex.  These findings not only broaden the chem. space in this inhibitor class, but also establish a rapid and simple assay to evaluate future HIV-1 entry inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPNtIXs0I2fLVg90H21EOLACvtfcHk0ljCgSI3fZd-IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGgtb%252FI&md5=17312f8c72b2fb244a606d00aa47ab15</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23081940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23081940%26sid%3Dliteratum%253Aachs%26aulast%3DMeuser%26aufirst%3DM.%2BE.%26aulast%3DMurphy%26aufirst%3DM.%2BB.%26aulast%3DRashad%26aufirst%3DA.%2BA.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DKinetic%2520Characterization%2520of%2520Novel%2520HIV-1%2520Entry%2520Inhibitors%253A%2520Discovery%2520of%2520a%2520Relationship%2520between%2520Off-Rate%2520and%2520Potency%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1940%26doi%3D10.3390%2Fmolecules23081940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickford, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irving, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciaramella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaacson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kjerrstrom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, S. L.</span></span> <span> </span><span class="NLM_article-title">HIV capsid is a tractable target for small molecule therapeutic intervention</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e1001220</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1001220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1371%2Fjournal.ppat.1001220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=21170360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252FltlGnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&author=W.+S.+Blairauthor=C.+Pickfordauthor=S.+L.+Irvingauthor=D.+G.+Brownauthor=M.+Andersonauthor=R.+Bazinauthor=J.+Caoauthor=G.+Ciaramellaauthor=J.+Isaacsonauthor=L.+Jacksonauthor=R.+Huntauthor=A.+Kjerrstromauthor=J.+A.+Niemanauthor=A.+K.+Patickauthor=M.+Perrosauthor=A.+D.+Scottauthor=K.+Whitbyauthor=H.+Wuauthor=S.+L.+Butler&title=HIV+capsid+is+a+tractable+target+for+small+molecule+therapeutic+intervention&doi=10.1371%2Fjournal.ppat.1001220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">HIV capsid is a tractable target for small molecule therapeutic intervention</span></div><div class="casAuthors">Blair Wade S; Pickford Chris; Irving Stephen L; Brown David G; Anderson Marie; Bazin Richard; Cao Joan; Ciaramella Giuseppe; Isaacson Jason; Jackson Lynn; Hunt Rachael; Kjerrstrom Anne; Nieman James A; Patick Amy K; Perros Manos; Scott Andrew D; Whitby Kevin; Wu Hua; Butler Scott L</div><div class="citationInfo"><span class="NLM_cas:title">PLoS pathogens</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1001220</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Despite a high current standard of care in antiretroviral therapy for HIV, multidrug-resistant strains continue to emerge, underscoring the need for additional novel mechanism inhibitors that will offer expanded therapeutic options in the clinic.  We report a new class of small molecule antiretroviral compounds that directly target HIV-1 capsid (CA) via a novel mechanism of action.  The compounds exhibit potent antiviral activity against HIV-1 laboratory strains, clinical isolates, and HIV-2, and inhibit both early and late events in the viral replication cycle.  We present mechanistic studies indicating that these early and late activities result from the compound affecting viral uncoating and assembly, respectively.  We show that amino acid substitutions in the N-terminal domain of HIV-1 CA are sufficient to confer resistance to this class of compounds, identifying CA as the target in infected cells.  A high-resolution co-crystal structure of the compound bound to HIV-1 CA reveals a novel binding pocket in the N-terminal domain of the protein.  Our data demonstrate that broad-spectrum antiviral activity can be achieved by targeting this new binding site and reveal HIV CA as a tractable drug target for HIV therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRN3Dm4etvdX1MGsG_HTXYFfW6udTcc2eafgpLbxQaj7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252FltlGnsA%253D%253D&md5=4b133434dd66f16f983e3856f3d6b07b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1001220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1001220%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DW.%2BS.%26aulast%3DPickford%26aufirst%3DC.%26aulast%3DIrving%26aufirst%3DS.%2BL.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DAnderson%26aufirst%3DM.%26aulast%3DBazin%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DCiaramella%26aufirst%3DG.%26aulast%3DIsaacson%26aufirst%3DJ.%26aulast%3DJackson%26aufirst%3DL.%26aulast%3DHunt%26aufirst%3DR.%26aulast%3DKjerrstrom%26aufirst%3DA.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DPerros%26aufirst%3DM.%26aulast%3DScott%26aufirst%3DA.%2BD.%26aulast%3DWhitby%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DButler%26aufirst%3DS.%2BL.%26atitle%3DHIV%2520capsid%2520is%2520a%2520tractable%2520target%2520for%2520small%2520molecule%2520therapeutic%2520intervention%26jtitle%3DPLoS%2520Pathog.%26date%3D2010%26volume%3D6%26doi%3D10.1371%2Fjournal.ppat.1001220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halambage, U. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burse, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiken, C.</span></span> <span> </span><span class="NLM_article-title">Compensatory substitutions in the HIV-1 capsid reduce the fitness cost associated with resistance to a capsid-targeting small-molecule inhibitor</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1128/JVI.01411-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.01411-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=25320302" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2qur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=208-219&author=J.+Shiauthor=J.+Zhouauthor=U.+D.+Halambageauthor=V.+B.+Shahauthor=M.+J.+Burseauthor=H.+Wuauthor=W.+S.+Blairauthor=S.+L.+Butlerauthor=C.+Aiken&title=Compensatory+substitutions+in+the+HIV-1+capsid+reduce+the+fitness+cost+associated+with+resistance+to+a+capsid-targeting+small-molecule+inhibitor&doi=10.1128%2FJVI.01411-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Compensatory substitutions in the HIV-1 capsid reduce the fitness cost associated with resistance to a capsid-targeting small-molecule inhibitor</span></div><div class="casAuthors">Shi, Jiong; Zhou, Jing; Halambage, Upul D.; Shah, Vaibhav B.; Burse, Mallori J.; Wu, Hua; Blair, Wade S.; Butler, Scott L.; Aiken, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">208-219, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The HIV-1 capsid plays multiple roles in infection and is an emerging therapeutic target.  The small-mol. HIV-1 inhibitor PF-3450074 (PF74) blocks HIV-1 at an early postentry stage by binding the viral capsid and interfering with its function.  Selection for resistance resulted in accumulation of five amino acid changes in the viral CA protein, which collectively reduced binding of the compd. to HIV-1 particles.  In the present study, we dissected the individual and combinatorial contributions of each of the five substitutions Q67H, K70R, H87P, T107N, and L111I to PF74 resistance, PF74 binding, and HIV-1 infectivity.  Q67H, K70R, and T107N each conferred low-level resistance to PF74 and collectively conferred strong resistance.  The substitutions K70R and L111I impaired HIV-1 infectivity, which was partially restored by the other substitutions at positions 67 and 107.  PF74 binding to HIV-1 particles was reduced by the Q67H, K70R, and T107N substitutions, consistent with the location of these positions in the inhibitor-binding pocket.  Replication of the 5Mut virus was markedly impaired in cultured macrophages, reminiscent of the previously reported N74D CA mutant. 5Mut substitutions also reduced the binding of the host protein CPSF6 to assembled CA complexes in vitro and permitted infection of cells expressing the inhibitory protein CPSF6-358.  Our results demonstrate that strong resistance to PF74 requires accumulation of multiple substitutions in CA to inhibit PF74 binding and compensate for fitness impairments assocd. with some of the sequence changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6wr7ky5b0U7Vg90H21EOLACvtfcHk0lguKAwMz8i4QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2qur0%253D&md5=ede0246cb047a706caa5584d6324d2e3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1128%2FJVI.01411-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01411-14%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHalambage%26aufirst%3DU.%2BD.%26aulast%3DShah%26aufirst%3DV.%2BB.%26aulast%3DBurse%26aufirst%3DM.%2BJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DBlair%26aufirst%3DW.%2BS.%26aulast%3DButler%26aufirst%3DS.%2BL.%26aulast%3DAiken%26aufirst%3DC.%26atitle%3DCompensatory%2520substitutions%2520in%2520the%2520HIV-1%2520capsid%2520reduce%2520the%2520fitness%2520cost%2520associated%2520with%2520resistance%2520to%2520a%2520capsid-targeting%2520small-molecule%2520inhibitor%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D208%26epage%3D219%26doi%3D10.1128%2FJVI.01411-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham-Huy, C.</span></span> <span> </span><span class="NLM_article-title">Chiral drugs: an overview</span>. <i>Int. J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">100</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=23674971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlSltrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=85-100&author=L.+A.+Nguyenauthor=H.+Heauthor=C.+Pham-Huy&title=Chiral+drugs%3A+an+overview"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral drugs. An overview</span></div><div class="casAuthors">Nguyen, Lien Ai; He, Hua; Chuong, Pham-Huy</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biomedical Science (Monterey Park, CA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-100</span>CODEN:
                <span class="NLM_cas:coden">IJBSDB</span>;
        ISSN:<span class="NLM_cas:issn">1550-9702</span>.
    
            (<span class="NLM_cas:orgname">Master Publishing Group</span>)
        </div><div class="casAbstract">A review.  About more than half of the drugs currently in use are chiral compds. and near 90% of the last ones are marketed as racemates consisting of an equimolar mixt. of two enantiomers.  Although they have the same chem. structure, most isomers of chiral drugs exhibit marked differences in biol. activities such as pharmacol., toxicol., pharmacokinetics, metab. etc.  Some mechanisms of these properties are also explained.  Therefore, it is important to promote the chiral sepn. and anal. of racemic drugs in pharmaceutical industry as well as in clinic in order to eliminate the unwanted isomer from the prepn. and to find an optimal treatment and a right therapeutic control for the patient.  In this article, we review the nomenclature, pharmacol., toxicol., pharmacokinetics, metab. etc of some usual chiral drugs as well as their mechanisms.  Different techniques used for the chiral sepn. in pharmaceutical industry as well as in clin. analyses are also examd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovoqsLM_3bxLVg90H21EOLACvtfcHk0lguKAwMz8i4QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlSltrnJ&md5=254cbca060bb4cfd04eac365488f5b62</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DL.%2BA.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DPham-Huy%26aufirst%3DC.%26atitle%3DChiral%2520drugs%253A%2520an%2520overview%26jtitle%3DInt.%2520J.%2520Biomed.%2520Sci.%26date%3D2006%26volume%3D2%26spage%3D85%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.-k.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span> <span> </span><span class="NLM_article-title">Chiral kinase inhibitors</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">800</span>– <span class="NLM_lpage">809</span>, <span class="refDoi"> DOI: 10.2174/156802611795165052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.2174%2F156802611795165052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=21291394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3MXls1Ojsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=800-809&author=J.-k.+Jiangauthor=M.+Shenauthor=C.+J.+Thomasauthor=M.+B.+Boxer&title=Chiral+kinase+inhibitors&doi=10.2174%2F156802611795165052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral kinase inhibitors</span></div><div class="casAuthors">Jiang, Jian-kang; Shen, Min; Thomas, Craig J.; Boxer, Mathew B.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">800-809</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Small mol. kinase inhibitors are important tools for studying cellular signaling pathways, phenotypes and are, occasionally, useful clin. agents.  With stereochem. pervasive throughout the mols. of life it is no surprise that a single stereocenter can bestow a ligand with distinct binding affinities to various protein targets.  While the majority of small mol. kinase inhibitors reported to date are achiral, a no. of asym. compds. show great utility as tools for probing kinase-assocd. biomol. events as well as promising therapeutic leads.  The mechanism by which chirality is introduced varies but includes screening of chiral libraries, incorporation of chiral centers during optimization efforts and the rational installation of a chiral moiety as guided by structural and modeling efforts.  Here we discuss several advanced chiral small mol. kinase inhibitors where stereochem. plays an important role in terms of potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNwyVAiHC8JrVg90H21EOLACvtfcHk0lguKAwMz8i4QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXls1Ojsrs%253D&md5=e838981d7aea95bcca374b10706fa2f3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F156802611795165052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611795165052%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DJ.-k.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26atitle%3DChiral%2520kinase%2520inhibitors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D800%26epage%3D809%26doi%3D10.2174%2F156802611795165052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallegos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uttamsingh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miwa, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balani, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, L. S.</span></span> <span> </span><span class="NLM_article-title">Comparison of intrinsic clearance in liver microsomes and hepatocytes from rats and humans: evaluation of free fraction and uptake in hepatocytes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1600</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1124/dmd.106.010793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1124%2Fdmd.106.010793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=16790553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1SksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=1600-1605&author=C.+Luauthor=P.+Liauthor=R.+Gallegosauthor=V.+Uttamsinghauthor=C.+Q.+Xiaauthor=G.+T.+Miwaauthor=S.+K.+Balaniauthor=L.+S.+Gan&title=Comparison+of+intrinsic+clearance+in+liver+microsomes+and+hepatocytes+from+rats+and+humans%3A+evaluation+of+free+fraction+and+uptake+in+hepatocytes&doi=10.1124%2Fdmd.106.010793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of intrinsic clearance in liver microsomes and hepatocytes from rats and humans: evaluation of free fraction and uptake in hepatocytes</span></div><div class="casAuthors">Lu, Chuang; Li, Ping; Gallegos, Richard; Uttamsingh, Vinita; Xia, Cindy Q.; Miwa, Gerald T.; Balani, Suresh K.; Gan, Liang-Shang</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1600-1605</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Apparent intrinsic clearance (CLint,app) of 7-ethoxycoumarin, phenacetin, propranolol, and midazolam was measured using rat and human liver microsomes and freshly isolated and cryopreserved hepatocytes to det. factors responsible for differences in rates of metab. in these systems.  The cryopreserved and freshly isolated hepatocytes generally provided similar results, although there was greater variability using the latter system.  The CLint,app values in hepatocytes are obsd. to be lower than that in microsomes, and this difference becomes greater for compds. with high CLint,app.  This could partly be attributed to the differences in the free fraction (fu).  The fu in hepatocyte incubations (fu,hep-inc) was influenced not only by the free fraction of compds. in the incubation buffer (fu,buffer) but also by the rate consts. of uptake (kup) and metab. (kmet).  This report provides a new derivation for fu,hep-inc, which can be expressed as fu,hep-inc = [kup/(kmet + kup)]/[1 + (Chep/Cbuffer) × (Vhep/Vbuffer)], where the Chep, Cbuffer, Vhep, and Vbuffer represent the concns. of a compd. in hepatocytes and buffer and vols. of hepatocytes and buffer, resp.  For midazolam, the fu,hep-inc was calcd., and the max. metab. rate in hepatocytes was shown to be limited by the uptake rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkV9lSFxDFT7Vg90H21EOLACvtfcHk0limbKkRk5uS4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1SksLk%253D&md5=3d3ff22990742b26dc24c84a6aab720e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1124%2Fdmd.106.010793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.106.010793%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DGallegos%26aufirst%3DR.%26aulast%3DUttamsingh%26aufirst%3DV.%26aulast%3DXia%26aufirst%3DC.%2BQ.%26aulast%3DMiwa%26aufirst%3DG.%2BT.%26aulast%3DBalani%26aufirst%3DS.%2BK.%26aulast%3DGan%26aufirst%3DL.%2BS.%26atitle%3DComparison%2520of%2520intrinsic%2520clearance%2520in%2520liver%2520microsomes%2520and%2520hepatocytes%2520from%2520rats%2520and%2520humans%253A%2520evaluation%2520of%2520free%2520fraction%2520and%2520uptake%2520in%2520hepatocytes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2006%26volume%3D34%26spage%3D1600%26epage%3D1605%26doi%3D10.1124%2Fdmd.106.010793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Björndal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiore, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colognesi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarlatti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenyö, E. M.</span></span> <span> </span><span class="NLM_article-title">Coreceptor usage of primary human immunodeficiency virus type 1 isolates varies according to biological phenotype</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">7478</span>– <span class="NLM_lpage">7487</span>, <span class="refDoi"> DOI: 10.1128/JVI.71.10.7478-7487.1997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.71.10.7478-7487.1997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=9311827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADyaK2sXmt1Omsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=1997&pages=7478-7487&author=A.+Bj%C3%B6rndalauthor=H.+Dengauthor=M.+Janssonauthor=J.+R.+Fioreauthor=C.+Colognesiauthor=A.+Karlssonauthor=J.+Albertauthor=G.+Scarlattiauthor=D.+R.+Littmanauthor=E.+M.+Feny%C3%B6&title=Coreceptor+usage+of+primary+human+immunodeficiency+virus+type+1+isolates+varies+according+to+biological+phenotype&doi=10.1128%2FJVI.71.10.7478-7487.1997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Coreceptor usage of primary human immunodeficiency virus type 1 isolates varies according to biological phenotype</span></div><div class="casAuthors">Bjorndal, Asa; Deng, Hongkui; Jansson, Marianne; Fiore, Jose R.; Colognesi, Claudia; Karlsson, Anders; Albert, Jan; Scarlatti, Gabriella; Littman, Dan R.; Fenyo, Eva Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7478-7487</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The biol. phenotype of primary human immunodeficiency virus type 1 (HIV-1) isolates varies according to the severity of the HIV infection.  Here the authors show that the two previously described groups of rapid/high, syncytium-inducing (SI) and slow/low, non-syncytium-inducing (NSI) isolates are distinguished by their ability to utilize different chemokine receptors for entry into target cells.  Recent studies have identified the C-X-C chemokine receptor CXCR4 (also named fusin or Lestr) and the C-C chemokine receptor CCR5 as the principal entry cofactors for T-cell-line-tropic and non-T-cell-line-tropic HIV-1, resp.  Using U87.CD4 glioma cell lines, stably expressing the chemokine receptor CCR1, CCR2b, CCR3, CCR5, or CXCR4, the authors have tested chemokine receptor specificity for a panel of genetically diverse envelope glycoprotein genes cloned from primary HIV-1 isolates and have found that receptor usage was closely assocd. with the biol. phenotype of the virus isolate but not the genetic subtype.  They have also analyzed a panel of 36 well-characterized primary HIV-1 isolates for syncytium induction and replication in the same series of cell lines.  Infection by slow/low viruses was restricted to cells expressing CCR5, whereas rapid/high viruses could use a variety of chemokine receptors.  In addn. to the regular use of CXCR4, many rapid/high viruses used CCR5 and some also used CCR3 and CCR2b.  Progressive HIV-1 infection is characterized by the emergence of viruses resistant to inhibition by β-chemokines, which corresponded to changes in coreceptor usage.  The broadening of the host range may even enable the use of uncharacterized coreceptors, in that two isolates from immunodeficient patients infected the parental U87.CD4 cell line lacking any engineered coreceptor.  Two primary isolates with multiple coreceptor usage were shown to consist of mixed populations, one with a narrow host range using CCR5 only and the other with a broad host range using CCR3, CCR5, or CXCR4, similar to the original population.  The results show that all 36 primary HIV-1 isolates induce syncytia, provided that target cells carry the particular coreceptor required by the virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xCNHpx6JQrVg90H21EOLACvtfcHk0limbKkRk5uS4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt1Omsr4%253D&md5=9c509d74aaeb47df187b01a3d874ec48</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FJVI.71.10.7478-7487.1997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.71.10.7478-7487.1997%26sid%3Dliteratum%253Aachs%26aulast%3DBj%25C3%25B6rndal%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DJansson%26aufirst%3DM.%26aulast%3DFiore%26aufirst%3DJ.%2BR.%26aulast%3DColognesi%26aufirst%3DC.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DAlbert%26aufirst%3DJ.%26aulast%3DScarlatti%26aufirst%3DG.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26aulast%3DFeny%25C3%25B6%26aufirst%3DE.%2BM.%26atitle%3DCoreceptor%2520usage%2520of%2520primary%2520human%2520immunodeficiency%2520virus%2520type%25201%2520isolates%2520varies%2520according%2520to%2520biological%2520phenotype%26jtitle%3DJ.%2520Virol.%26date%3D1997%26volume%3D71%26spage%3D7478%26epage%3D7487%26doi%3D10.1128%2FJVI.71.10.7478-7487.1997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zentner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sierra, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciunas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankowski, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinnik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedichev, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Garcia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span> <span> </span><span class="NLM_article-title">Identification of a small-molecule inhibitor of HIV-1 assembly that targets the phosphatidylinositol (4,5)-bisphosphate binding site of the HIV-1 matrix protein</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">426</span>– <span class="NLM_lpage">432</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201200577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1002%2Fcmdc.201200577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=23361947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGiurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=426-432&author=I.+Zentnerauthor=L.-J.+Sierraauthor=A.+K.+Fraserauthor=L.+Maciunasauthor=M.+K.+Mankowskiauthor=A.+Vinnikauthor=P.+Fedichevauthor=R.+G.+Ptakauthor=J.+Martin-Garciaauthor=S.+Cocklin&title=Identification+of+a+small-molecule+inhibitor+of+HIV-1+assembly+that+targets+the+phosphatidylinositol+%284%2C5%29-bisphosphate+binding+site+of+the+HIV-1+matrix+protein&doi=10.1002%2Fcmdc.201200577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Small-Molecule Inhibitor of HIV-1 Assembly that Targets the Phosphatidylinositol (4,5)-bisphosphate Binding Site of the HIV-1 Matrix Protein</span></div><div class="casAuthors">Zentner, Isaac; Sierra, Luz-Jeannette; Fraser, Ayesha K.; Maciunas, Lina; Mankowski, Marie K.; Vinnik, Andrei; Fedichev, Peter; Ptak, Roger G.; Martin-Garcia, Julio; Cocklin, Simon</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">426-432</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of drug resistance remains a crit. problem for current HIV-1 antiviral therapies, creating a need for new inhibitors of HIV-1 replication.  We previously reported on a novel anti-HIV-1 compd., N2-(phenoxyacetyl)-N-[4-(1-piperidinylcarbonyl)benzyl]glycinamide (14), that binds to the highly conserved phosphatidylinositol (4,5)-bisphosphate (PI(4,5)P2) binding pocket of the HIV-1 matrix (MA) protein.  In this study, we re-evaluate the hits from the virtual screen used to identify compd. 14 and test them directly in an HIV-1 replication assay using primary human peripheral blood mononuclear cells.  This study resulted in the identification of three new compds. with antiviral activity; 2-(4-{[3-(4-fluorophenyl)-1,2,4-oxadiazol-5-yl]methyl})-1-piperazinyl)-N-(4-methylphenyl)acetamide (7), 3-(2-ethoxyphenyl)-5-[[4-(4-nitrophenyl)piperazin-1-yl]methyl]-1,2,4-oxadiazole (17), and N-[4-ethoxy-3-(1-piperidinylsulfonyl)phenyl]-2-(imidazo[2,1-b][1,3]thiazol-6-yl)acetamide (18), with compd. 7 being the most potent of these hits.  Mechanistic studies on 7 demonstrated that it directly interacts with and functions through HIV-1 MA.  In accordance with our drug target, compd. 7 competes with PI(4,5)P2 for MA binding and, as a result, diminishes the prodn. of new virus.  Mutation of residues within the PI(4,5)P2 binding site of MA decreased the antiviral effect of compd. 7.  Addnl., compd. 7 displays a broadly neutralizing anti-HIV activity, with IC50 values of 7.5-15.6 μM for the group M isolates tested.  Taken together, these results point towards a novel chem. probe that can be used to more closely study the biol. role of MA and could, through further optimization, lead to a new class of anti-HIV-1 therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr11-gHfqPpe7Vg90H21EOLACvtfcHk0limbKkRk5uS4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGiurY%253D&md5=19c5f1741bbf8c4bd87716021fa20e19</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200577%26sid%3Dliteratum%253Aachs%26aulast%3DZentner%26aufirst%3DI.%26aulast%3DSierra%26aufirst%3DL.-J.%26aulast%3DFraser%26aufirst%3DA.%2BK.%26aulast%3DMaciunas%26aufirst%3DL.%26aulast%3DMankowski%26aufirst%3DM.%2BK.%26aulast%3DVinnik%26aufirst%3DA.%26aulast%3DFedichev%26aufirst%3DP.%26aulast%3DPtak%26aufirst%3DR.%2BG.%26aulast%3DMartin-Garcia%26aufirst%3DJ.%26aulast%3DCocklin%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520a%2520small-molecule%2520inhibitor%2520of%2520HIV-1%2520assembly%2520that%2520targets%2520the%2520phosphatidylinositol%2520%25284%252C5%2529-bisphosphate%2520binding%2520site%2520of%2520the%2520HIV-1%2520matrix%2520protein%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D426%26epage%3D432%26doi%3D10.1002%2Fcmdc.201200577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kortagere, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mankowski, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zentner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Princiotto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sierra, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passic, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stavale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krebs, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Garcia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodroski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. B.,  III</span></span> <span> </span><span class="NLM_article-title">Inhibiting early-stage events in HIV-1 replication by small-molecule targeting of the HIV-1 capsid</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">8472</span>– <span class="NLM_lpage">8481</span>, <span class="refDoi"> DOI: 10.1128/JVI.05006-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.05006-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=22647699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2jtLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=8472-8481&author=S.+Kortagereauthor=N.+Madaniauthor=M.+K.+Mankowskiauthor=A.+Schonauthor=I.+Zentnerauthor=G.+Swaminathanauthor=A.+Princiottoauthor=K.+Anthonyauthor=A.+Ozaauthor=L.+J.+Sierraauthor=S.+R.+Passicauthor=X.+Wangauthor=D.+M.+Jonesauthor=E.+Stavaleauthor=F.+C.+Krebsauthor=J.+Martin-Garciaauthor=E.+Freireauthor=R.+G.+Ptakauthor=J.+Sodroskiauthor=S.+Cocklinauthor=A.+B.+Smith&title=Inhibiting+early-stage+events+in+HIV-1+replication+by+small-molecule+targeting+of+the+HIV-1+capsid&doi=10.1128%2FJVI.05006-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting early-stage events in HIV-1 replication by small-molecule targeting of the HIV-1 capsid</span></div><div class="casAuthors">Kortagere, Sandhya; Madani, Navid; Mankowski, Marie K.; Schon, Arne; Zentner, Isaac; Swaminathan, Gokul; Princiotto, Amy; Anthony, Kevin; Oza, Apara; Sierra, Luz-Jeannette; Passic, Shendra R.; Wang, Xiaozhao; Jones, David M.; Stavale, Eric; Krebs, Fred C.; Martin-Garcia, Julio; Freire, Ernesto; Ptak, Roger G.; Sodroski, Joseph; Cocklin, Simon; Smith, Amos B., III</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8472-8481</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The HIV-1 capsid (CA) protein plays essential roles in both early and late stages of virl replication and has emerged as a novel drug target.  We report hybrid structure-based virtual screening to identify small mols. with the potential to interact with the N-terminal domain (NTD) of HIV-1 CA and disrupt early, preintegration steps of the HIV-1 replication cycle.  The small mol. 4,4'-[dibenzo[b,d]furan-2,8-diylbis(5-phenyl-1H-imidazole-4,2-diyl)]dibenzoic acid (CK026), which had anti-HIV-1 activity in single- and multiple-round infections but failed to inhibit viral replication in peripheral blood mononuclear cells (PBMCs), was identified.  Three analogs of CK026 with reduced size and better drug-like properties were synthesized and assessed.  Compd. I-XW-053 (4-(4,5-diphenyl-1H-imidazol-2-yl)benzoic acid) retained all of the antiviral activity of the parental compd. and inhibited the replication of a diverse panel of primary HIV-1 isolates in PBMCs, while displaying no appreciable cytotoxicity.  This antiviral activity was specific to HIV-1, as I-XW-053 displayed no effect on the replication of SIV or against a panel of nonretroviruses.  Direct interaction of I-XW-053 was quantified with wild-type and mutant CA protein using surface plasmon resonance and isothermal titrn. calorimetry.  Mutation of Ile37 and Arg173, which are required for interaction with compd. I-XW-053, crippled the virus at an early, preintegration step.  Using quant. PCR, we demonstrated that treatment with I-XW-053 inhibited HIV-1 reverse transcription in multiple cell types, indirectly pointing to dysfunction in the uncoating process.  In summary, we have identified a CA-specific compd. that targets and inhibits a novel region in the NTD-NTD interface, affects uncoating, and possesses broad-spectrum anti-HIV-1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Tdf1mUZ85bVg90H21EOLACvtfcHk0lhXOIe7ZFK_4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2jtLzE&md5=324cf8cda7a0aa6e07b5b8e529c91aaf</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1128%2FJVI.05006-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.05006-11%26sid%3Dliteratum%253Aachs%26aulast%3DKortagere%26aufirst%3DS.%26aulast%3DMadani%26aufirst%3DN.%26aulast%3DMankowski%26aufirst%3DM.%2BK.%26aulast%3DSchon%26aufirst%3DA.%26aulast%3DZentner%26aufirst%3DI.%26aulast%3DSwaminathan%26aufirst%3DG.%26aulast%3DPrinciotto%26aufirst%3DA.%26aulast%3DAnthony%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DA.%26aulast%3DSierra%26aufirst%3DL.%2BJ.%26aulast%3DPassic%26aufirst%3DS.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DJones%26aufirst%3DD.%2BM.%26aulast%3DStavale%26aufirst%3DE.%26aulast%3DKrebs%26aufirst%3DF.%2BC.%26aulast%3DMartin-Garcia%26aufirst%3DJ.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DPtak%26aufirst%3DR.%2BG.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26atitle%3DInhibiting%2520early-stage%2520events%2520in%2520HIV-1%2520replication%2520by%2520small-molecule%2520targeting%2520of%2520the%2520HIV-1%2520capsid%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D8472%26epage%3D8481%26doi%3D10.1128%2FJVI.05006-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Studier, F. W.</span></span> <span> </span><span class="NLM_article-title">Protein production by auto-induction in high density shaking cultures</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1016/j.pep.2005.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1016%2Fj.pep.2005.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=15915565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=207-234&author=F.+W.+Studier&title=Protein+production+by+auto-induction+in+high+density+shaking+cultures&doi=10.1016%2Fj.pep.2005.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Protein production by auto-induction in high-density shaking cultures</span></div><div class="casAuthors">Studier, F. William</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-234</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Inducible expression systems in which T7 RNA polymerase transcribes coding sequences cloned under control of a T7lac promoter efficiently produce a wide variety of proteins in Escherichia coli.  Investigation of factors that affect stability, growth, and induction of T7 expression strains in shaking vessels led to the recognition that sporadic, unintended induction of expression in complex media, previously reported by others, is almost certainly caused by small amts. of lactose.  Glucose prevents induction by lactose by well-studied mechanisms.  Amino acids also inhibit induction by lactose during log-phase growth, and high rates of aeration inhibit induction at low lactose concns.  These observations, and metabolic balancing of pH, allowed development of reliable non-inducing and auto-inducing media in which batch cultures grow to high densities.  Expression strains grown to satn. in non-inducing media retain plasmid and remain fully viable for weeks in the refrigerator, making it easy to prep. many freezer stocks in parallel and use working stocks for an extended period.  Auto-induction allows efficient screening of many clones in parallel for expression and soly., as cultures have only to be inoculated and grown to satn., and yields of target protein are typically several-fold higher than obtained by conventional IPTG induction.  Auto-inducing media have been developed for labeling proteins with selenomethionine, 15N or 13C, and for prodn. of target proteins by arabinose induction of T7 RNA polymerase from the pBAD promoter in BL21-AI.  Selenomethionine labeling was equally efficient in the commonly used methionine auxotroph B834(DE3)(found to be metE) or the prototroph BL21(DE3).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYNqgn15xP_bVg90H21EOLACvtfcHk0lhXOIe7ZFK_4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsFCktLw%253D&md5=d31b76747c808ce6fbb27c7a1192b6b8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2005.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2005.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DStudier%26aufirst%3DF.%2BW.%26atitle%3DProtein%2520production%2520by%2520auto-induction%2520in%2520high%2520density%2520shaking%2520cultures%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2005%26volume%3D41%26spage%3D207%26epage%3D234%26doi%3D10.1016%2Fj.pep.2005.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connor, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, N. R.</span></span> <span> </span><span class="NLM_article-title">Vpr is required for efficient replication of human immunodeficiency virus type-1 in mononuclear phagocytes</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>206</i></span>,  <span class="NLM_fpage">935</span>– <span class="NLM_lpage">944</span>, <span class="refDoi"> DOI: 10.1006/viro.1995.1016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1006%2Fviro.1995.1016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=7531918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADyaK2MXjs1WkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=206&publication_year=1995&pages=935-944&author=R.+I.+Connorauthor=B.+K.+Chenauthor=S.+Choeauthor=N.+R.+Landau&title=Vpr+is+required+for+efficient+replication+of+human+immunodeficiency+virus+type-1+in+mononuclear+phagocytes&doi=10.1006%2Fviro.1995.1016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Vpr is required for efficient replication of human immunodeficiency virus type-1 in mononuclear phagocytes</span></div><div class="casAuthors">Connor, Ruth I.; Chen, Benjamin Kuan; Choe, Sunny; Landau, Nathaniel R.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">206</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">935-44</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">HIV-1 vpr encodes a 96-amino acid, nuclear protein whose function is not well understood.  Unlike the other lentivirus regulatory proteins, Vpr is present in virions at relatively high copy no.  In cells, Vpr is localized to the nucleus.  Possible functions for vpr consistent with these findings include the nuclear import of preintegration complexes, transactivation of cellular genes, or induction of cellular differentiation.  The authors show here, using both replication competent, macrophage-tropic virus and a sensitive, single-cycle luciferase HIV-1 reporter vector, that vpr is important for efficient viral replication in primary monocyte/macrophages, but appears to play no role in activated or resting T cell infection.  The block to infection in monocytes was localized by PCR anal. of newly synthesized viral DNA and with the luciferase reporter vector to a stage in the viral life cycle after entry and reverse transcription, yet prior to, or at the time of, proviral transcription.  In addn., infection of mononuclear phagocyte with virions that had been loaded with Vpr mols. in the producer cells by trans-complementation still showed a vpr-phenotype.  These data suggest a role for vpr mols. produced in newly infected cells, in addn. to its presumed function in the virion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgyuTZ6ZIJ-7Vg90H21EOLACvtfcHk0lhXOIe7ZFK_4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjs1WkurY%253D&md5=09620dfc7aaf57ec9c46ffaef438630f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1006%2Fviro.1995.1016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fviro.1995.1016%26sid%3Dliteratum%253Aachs%26aulast%3DConnor%26aufirst%3DR.%2BI.%26aulast%3DChen%26aufirst%3DB.%2BK.%26aulast%3DChoe%26aufirst%3DS.%26aulast%3DLandau%26aufirst%3DN.%2BR.%26atitle%3DVpr%2520is%2520required%2520for%2520efficient%2520replication%2520of%2520human%2520immunodeficiency%2520virus%2520type-1%2520in%2520mononuclear%2520phagocytes%26jtitle%3DVirology%26date%3D1995%26volume%3D206%26spage%3D935%26epage%3D944%26doi%3D10.1006%2Fviro.1995.1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosakovsky Pond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeStefano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phogat, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poignard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fling, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaBranche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montefiori, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koff, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parks, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caulfield, M. J.</span></span> <span> </span><span class="NLM_article-title">Identification of an HIV-1 clade A envelope that exhibits broad antigenicity and neutralization sensitivity and elicits antibodies targeting three distinct epitopes</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">5372</span>– <span class="NLM_lpage">5383</span>, <span class="refDoi"> DOI: 10.1128/JVI.02827-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.02827-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=23468492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsV2lsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=5372-5383&author=S.+Hoffenbergauthor=R.+Powellauthor=A.+Carpovauthor=D.+Wagnerauthor=A.+Wilsonauthor=S.+Kosakovsky+Pondauthor=R.+Lindsayauthor=H.+Arendtauthor=J.+DeStefanoauthor=S.+Phogatauthor=P.+Poignardauthor=S.+P.+Flingauthor=M.+Simekauthor=C.+LaBrancheauthor=D.+Montefioriauthor=T.+Wrinauthor=P.+Phungauthor=D.+Burtonauthor=W.+Koffauthor=C.+R.+Kingauthor=C.+L.+Parksauthor=M.+J.+Caulfield&title=Identification+of+an+HIV-1+clade+A+envelope+that+exhibits+broad+antigenicity+and+neutralization+sensitivity+and+elicits+antibodies+targeting+three+distinct+epitopes&doi=10.1128%2FJVI.02827-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an HIV-1 clad A envelope that exhibits broad antigenicity and neutralization sensitivity and elicits antibodies targeting three distinct epitopes</span></div><div class="casAuthors">Hoffenberg, Simon; Powell, Rebecca; Carpov, Alexei; Wagner, Denise; Wilson, Aaron; Pond, Sergei Kosakovsky; Lindsay, Ross; Arendt, Heather; DeStefano, Joanne; Phogat, Sanjay; Polgnard, Pascal; Fling, Steven P.; Simek, Melissa; LaBranche, Celia; Montefiori, David; Wrin, Terri; Phung, Pham; Burton, Dennis; Koff, Wayne; King, C. Richter; Parks, Christopher L.; Caulfield, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5372-5383</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Broadly neutralizing antibodies (bNAbs) PG9 and PG16 were isolated from an International AIDS Vaccine Initiative (IAVI) Protocol G subject infected with human immunodeficiency virus type 1 (HIV-1) clade A.  Both antibodies are highly potent and neutralize greater than 70% of viruses tested.  We sought to begin immunogen design based on viral sequences from this patient; however, pseudoviruses prepd. with 19 envelope sequences from this subject were resistant to neutralization by PG9 and PG16.  Therefore, we used a bioinformatics approach to identify closely related viruses that were potentially sensitive to PG9 and PG16.  A most-recent common ancestor (MRCA) sequence for the viral envelope (Env) was detd. and aligned with 99 subtype A gp160 sequences from the Los Alamos HIV database.  Virus BG505.W6M.ENV.C2 (BG505) was found to have the highest degree of homol. (73%) to the MRCA sequence.  Pseudoviruses prepd. with this Env were sensitive to neutralization with a broad panel of bNAbs, including PG9 and PG16.  When expressed by 293T cells as sol. gp120, the BG505 monomer bound well to both PG9 and PG16.  We further showed that a point mutation (L111A) enabled more efficient prodn. of a stable gp120 monomer that preserves the major neutralization epitopes.  Finally, we showed that an adjuvanted formulation of this gp120 protein elicited neutralizing antibodies in rabbits (following a gp120 DNA vaccine prime) and that the antisera competed with bNAbs from 3 classes of nonoverlapping epitopes.  Thus, the BG505 Env protein warrants further investigation as an HIV vaccine candidate, as a stand-alone protein, or as a component of a vaccine vector.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGourszMpqvydrVg90H21EOLACvtfcHk0ljA7rVnWsvLog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsV2lsrk%253D&md5=b2d64d84e548e0045483638e7d800f66</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1128%2FJVI.02827-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02827-12%26sid%3Dliteratum%253Aachs%26aulast%3DHoffenberg%26aufirst%3DS.%26aulast%3DPowell%26aufirst%3DR.%26aulast%3DCarpov%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DKosakovsky%2BPond%26aufirst%3DS.%26aulast%3DLindsay%26aufirst%3DR.%26aulast%3DArendt%26aufirst%3DH.%26aulast%3DDeStefano%26aufirst%3DJ.%26aulast%3DPhogat%26aufirst%3DS.%26aulast%3DPoignard%26aufirst%3DP.%26aulast%3DFling%26aufirst%3DS.%2BP.%26aulast%3DSimek%26aufirst%3DM.%26aulast%3DLaBranche%26aufirst%3DC.%26aulast%3DMontefiori%26aufirst%3DD.%26aulast%3DWrin%26aufirst%3DT.%26aulast%3DPhung%26aufirst%3DP.%26aulast%3DBurton%26aufirst%3DD.%26aulast%3DKoff%26aufirst%3DW.%26aulast%3DKing%26aufirst%3DC.%2BR.%26aulast%3DParks%26aufirst%3DC.%2BL.%26aulast%3DCaulfield%26aufirst%3DM.%2BJ.%26atitle%3DIdentification%2520of%2520an%2520HIV-1%2520clade%2520A%2520envelope%2520that%2520exhibits%2520broad%2520antigenicity%2520and%2520neutralization%2520sensitivity%2520and%2520elicits%2520antibodies%2520targeting%2520three%2520distinct%2520epitopes%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D5372%26epage%3D5383%26doi%3D10.1128%2FJVI.02827-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cocklin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Querido, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmagadda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriakose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicala, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-García, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaiken, I. M.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum anti-human immunodeficiency virus (HIV) potential of a peptide HIV type 1 entry inhibitor</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">3645</span>– <span class="NLM_lpage">3648</span>, <span class="refDoi"> DOI: 10.1128/JVI.01778-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.01778-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=17251295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjslWqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=3645-3648&author=S.+Cocklinauthor=H.+Gopiauthor=B.+Queridoauthor=M.+Nimmagaddaauthor=S.+Kuriakoseauthor=C.+Cicalaauthor=S.+Ajithauthor=S.+Baxterauthor=J.+Arthosauthor=J.+Mart%C3%ADn-Garc%C3%ADaauthor=I.+M.+Chaiken&title=Broad-spectrum+anti-human+immunodeficiency+virus+%28HIV%29+potential+of+a+peptide+HIV+type+1+entry+inhibitor&doi=10.1128%2FJVI.01778-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum anti-human immunodeficiency virus (HIV) potential of a peptide HIV type 1 entry inhibitor</span></div><div class="casAuthors">Cocklin, Simon; Gopi, Hosahudya; Querido, Bianca; Nimmagadda, Manideepthi; Kuriakose, Syna; Cicala, Claudia; Ajith, Sandya; Baxter, Sabine; Arthos, James; Martin-Garcia, Julio; Chaiken, Irwin M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3645-3648</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The AIDS epidemic continues to spread at an alarming rate worldwide, esp. in developing countries.  One approach to solving this problem is the generation of anti-human immunodeficiency virus (HIV) compds. with inhibition spectra broad enough to include globally prevailing forms of the virus.  We have examd. the HIV type 1 (HIV-1) envelope specificity of a recently identified entry inhibitor candidate, HNG-105, using surface plasmon resonance spectroscopy and pseudovirus inhibition assays.  The combined results suggest that the HNG-105 mol. may be effective across the HIV-1 subtypes, and they highlight its potential as a lead for developing therapeutic and microbicidal agents to help combat the spread of AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ZqavCczIgrVg90H21EOLACvtfcHk0ljA7rVnWsvLog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjslWqsbg%253D&md5=3c0000200d2703af02c9cf52e4595e3e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1128%2FJVI.01778-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01778-06%26sid%3Dliteratum%253Aachs%26aulast%3DCocklin%26aufirst%3DS.%26aulast%3DGopi%26aufirst%3DH.%26aulast%3DQuerido%26aufirst%3DB.%26aulast%3DNimmagadda%26aufirst%3DM.%26aulast%3DKuriakose%26aufirst%3DS.%26aulast%3DCicala%26aufirst%3DC.%26aulast%3DAjith%26aufirst%3DS.%26aulast%3DBaxter%26aufirst%3DS.%26aulast%3DArthos%26aufirst%3DJ.%26aulast%3DMart%25C3%25ADn-Garc%25C3%25ADa%26aufirst%3DJ.%26aulast%3DChaiken%26aufirst%3DI.%2BM.%26atitle%3DBroad-spectrum%2520anti-human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520potential%2520of%2520a%2520peptide%2520HIV%2520type%25201%2520entry%2520inhibitor%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D3645%26epage%3D3648%26doi%3D10.1128%2FJVI.01778-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Marzio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, N. R.</span></span> <span> </span><span class="NLM_article-title">Human immunodeficiency virus type 1 viral protein R (Vpr) arrests cells in the G2 phase of the cell cycle by inhibiting p34cdc2 activity</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">6705</span>– <span class="NLM_lpage">6711</span>, <span class="refDoi"> DOI: 10.1128/JVI.69.11.6705-6711.1995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=10.1128%2FJVI.69.11.6705-6711.1995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=7474080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;key=1%3ACAS%3A528%3ADyaK2MXoslWkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1995&pages=6705-6711&author=J.+Heauthor=S.+Choeauthor=R.+Walkerauthor=P.+Di+Marzioauthor=D.+O.+Morganauthor=N.+R.+Landau&title=Human+immunodeficiency+virus+type+1+viral+protein+R+%28Vpr%29+arrests+cells+in+the+G2+phase+of+the+cell+cycle+by+inhibiting+p34cdc2+activity&doi=10.1128%2FJVI.69.11.6705-6711.1995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Human immunodeficiency virus type 1 viral protein R (Vpr) arrests cells in the G2 phase of the cell cycle by inhibiting p34cdc2 activity</span></div><div class="casAuthors">He, Jianglin; Choe, Sunny; Walker, Robert; Di Marzio, Paola; Morgan, David O.; Landau, Nathaniel R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6705-11</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Vpr accessory gene product of human immunodeficiency virus types 1 and 2 and simian immunodeficiency virus is believed to play a role in permitting entry of the viral core into the nucleus of nondividing cells.  A second role for Vpr was recently suggested by (M. E. Rogel, et al., 1995) who showed that Vpr prevents the establishment in vitro of chronically infected HIV producer cell lines, apparently by causing infected cells to arrest in the G2/M phase of the cell cycle.  In cycling cells, progression from G2 to M phase is driven by activation of the p34cdc2/cyclin B complex, an event caused, in part, by dephosphorylation of two regulatory amino acids of p34cdc2 (Thr-14 and Tyr-15).  We show here that Vpr arrests the cell cycle in G2 by preventing the activation of the p34cdc2/cyclin B complex.  Vpr expression in cells caused p34cdc2 to remain in the phosphorylated, inactive state.  The p34cdc2/cyclin B complexes immunopptn. from cells expressing Vpr were almost completely inactive in a histone H1 kinase assay.  Coexpression of a constitutively active mutant p34cdc2 mol. with Vpr relieved the G2 arrest.  These findings strongly suggest that Vpr arrests cells in G2 by preventing the activation of the p34cdc2/cyclin B complex that is required for entry into M phase.  In vivo, Vpr might, by preventing p34cdc2 activation, delay or prevent apoptosis of infected cells.  This would increase the amt. of virus each infected cell produced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfPunHH8sjFbVg90H21EOLACvtfcHk0ljA7rVnWsvLog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXoslWkt74%253D&md5=cf13092726ac54f93a13fb9439e971c2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1128%2FJVI.69.11.6705-6711.1995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.69.11.6705-6711.1995%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DChoe%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%26aulast%3DDi%2BMarzio%26aufirst%3DP.%26aulast%3DMorgan%26aufirst%3DD.%2BO.%26aulast%3DLandau%26aufirst%3DN.%2BR.%26atitle%3DHuman%2520immunodeficiency%2520virus%2520type%25201%2520viral%2520protein%2520R%2520%2528Vpr%2529%2520arrests%2520cells%2520in%2520the%2520G2%2520phase%2520of%2520the%2520cell%2520cycle%2520by%2520inhibiting%2520p34cdc2%2520activity%26jtitle%3DJ.%2520Virol.%26date%3D1995%26volume%3D69%26spage%3D6705%26epage%3D6711%26doi%3D10.1128%2FJVI.69.11.6705-6711.1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XFZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XFZ','PDB','4XFZ'); return false;">PDB: 4XFZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4D78" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4D78','PDB','4D78'); return false;">PDB: 4D78</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i120"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01810">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_52823"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01810?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01810</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings, potency and toxicity values, and kinetic and affinity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_001.csv">CSV</a>)</p></li><li><p class="inline">PDBs of the docking models for each compound (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_002.pdb">PDB</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_003.pdb">PDB</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_004.pdb">PDB</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Supplemental figures including the 2D interaction schemes and the LC–MS and <sup>1</sup>H NMR spectra of each synthesized compound as well as tables summarizing the predicted interaction energy for binding site residues, metabolic stability parameters, and primers used for mutagenesis (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_001.csv">jm0c01810_si_001.csv (1.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_002.pdb">jm0c01810_si_002.pdb (265.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_003.pdb">jm0c01810_si_003.pdb (266.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_004.pdb">jm0c01810_si_004.pdb (266.27 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_005.pdb">jm0c01810_si_005.pdb (268.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01810/suppl_file/jm0c01810_si_006.pdf">jm0c01810_si_006.pdf (1.39 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01810&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-7%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01810" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01810" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992eaf0d36d18f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
