
<html lang="en"     class="pb-page"  data-request-id="7b86b696-9103-4a90-97af-c125d4257628"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b00713;issue:issue:10.1021/jmcmar.2018.61.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Mathematical and Structural Characterization of Strong Nonadditive Structure–Activity Relationship Caused by Protein Conformational Changes" /></meta><meta name="dc.Creator" content="Laurent  Gomez" /></meta><meta name="dc.Creator" content="Rui  Xu" /></meta><meta name="dc.Creator" content="William  Sinko" /></meta><meta name="dc.Creator" content="Brandon  Selfridge" /></meta><meta name="dc.Creator" content="William  Vernier" /></meta><meta name="dc.Creator" content="Kiev  Ly" /></meta><meta name="dc.Creator" content="Richard  Truong" /></meta><meta name="dc.Creator" content="Markus  Metz" /></meta><meta name="dc.Creator" content="Tami  Marrone" /></meta><meta name="dc.Creator" content="Kristen  Sebring" /></meta><meta name="dc.Creator" content="Yingzhou  Yan" /></meta><meta name="dc.Creator" content="Brent  Appleton" /></meta><meta name="dc.Creator" content="Kathleen  Aertgeerts" /></meta><meta name="dc.Creator" content="Mark Eben  Massari" /></meta><meta name="dc.Creator" content="J. Guy  Breitenbucher" /></meta><meta name="dc.Description" content="In medicinal chemistry, accurate prediction of additivity-based structure–activity relationship (SAR) analysis rests on three assumptions: (1) a consistent binding pose of the central scaffold, (2)..." /></meta><meta name="Description" content="In medicinal chemistry, accurate prediction of additivity-based structure–activity relationship (SAR) analysis rests on three assumptions: (1) a consistent binding pose of the central scaffold, (2)..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 2, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00713" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00713" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00713" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00713" /></link>
        
    
    

<title>Mathematical and Structural Characterization of Strong Nonadditive Structure–Activity Relationship Caused by Protein Conformational Changes | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00713" /></meta><meta property="og:title" content="Mathematical and Structural Characterization of Strong Nonadditive Structure–Activity Relationship Caused by Protein Conformational Changes" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0010.jpeg" /></meta><meta property="og:description" content="In medicinal chemistry, accurate prediction of additivity-based structure–activity relationship (SAR) analysis rests on three assumptions: (1) a consistent binding pose of the central scaffold, (2) no interaction between the R group substituents, and (3) a relatively rigid binding pocket in which the R group substituents act independently. Previously, examples of nonadditive SAR have been documented in systems that deviate from the first two assumptions. Local protein structural change upon ligand binding, through induced fit or conformational selection, although a well-known phenomenon that invalidates the third assumption, has not been linked to nonadditive SAR conclusively. Here, for the first time, we present clear structural evidence that the formation of a hydrophobic pocket upon ligand binding in PDE2 catalytic site reduces the size of another distinct subpocket and contributes to strong nonadditive SAR between two otherwise distant R groups." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00713"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00713">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00713&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00713&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00713&amp;href=/doi/10.1021/acs.jmedchem.8b00713" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 7754-7766</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00700" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00743" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Mathematical and Structural Characterization of Strong Nonadditive Structure–Activity Relationship Caused by Protein Conformational Changes</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Laurent Gomez</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laurent Gomez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4a2d25272f30262b3f382f243e7a0a2d272b232664292527"><span class="__cf_email__" data-cfemail="4d2a22202837212c383f2823397d0d2a202c2421632e2220">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laurent++Gomez">Laurent Gomez</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9172-7840" title="Orcid link">http://orcid.org/0000-0002-9172-7840</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rui Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rui Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rui++Xu">Rui Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William Sinko</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William Sinko</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William++Sinko">William Sinko</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Brandon Selfridge</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brandon Selfridge</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brandon++Selfridge">Brandon Selfridge</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William Vernier</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William Vernier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William++Vernier">William Vernier</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kiev Ly</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kiev Ly</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kiev++Ly">Kiev Ly</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard Truong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard Truong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Truong">Richard Truong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Markus Metz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Markus Metz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Markus++Metz">Markus Metz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tami Marrone</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tami Marrone</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tami++Marrone">Tami Marrone</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kristen Sebring</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kristen Sebring</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kristen++Sebring">Kristen Sebring</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yingzhou Yan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yingzhou Yan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yingzhou++Yan">Yingzhou Yan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Brent Appleton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brent Appleton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brent++Appleton">Brent Appleton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kathleen Aertgeerts</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kathleen Aertgeerts</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kathleen++Aertgeerts">Kathleen Aertgeerts</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark Eben Massari</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark Eben Massari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark+Eben++Massari">Mark Eben Massari</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">J. Guy Breitenbucher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">J. Guy Breitenbucher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J.+Guy++Breitenbucher">J. Guy Breitenbucher</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00713&amp;href=/doi/10.1021%2Facs.jmedchem.8b00713" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 7754–7766</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 2, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>3 May 2018</li><li><span class="item_label"><b>Published</b> online</span>2 August 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 September 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00713" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00713</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7754%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLaurent%2BGomez%252C%2BRui%2BXu%252C%2BWilliam%2BSinko%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D17%26contentID%3Dacs.jmedchem.8b00713%26title%3DMathematical%2Band%2BStructural%2BCharacterization%2Bof%2BStrong%2BNonadditive%2BStructure%25E2%2580%2593Activity%2BRelationship%2BCaused%2Bby%2BProtein%2BConformational%2BChanges%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7766%26publicationDate%3DSeptember%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00713"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1251</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">7</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00713" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Mathematical and Structural Characterization of Strong Nonadditive Structure–Activity Relationship Caused by Protein Conformational Changes&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Laurent&quot;,&quot;last_name&quot;:&quot;Gomez&quot;},{&quot;first_name&quot;:&quot;Rui&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;Sinko&quot;},{&quot;first_name&quot;:&quot;Brandon&quot;,&quot;last_name&quot;:&quot;Selfridge&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;Vernier&quot;},{&quot;first_name&quot;:&quot;Kiev&quot;,&quot;last_name&quot;:&quot;Ly&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Truong&quot;},{&quot;first_name&quot;:&quot;Markus&quot;,&quot;last_name&quot;:&quot;Metz&quot;},{&quot;first_name&quot;:&quot;Tami&quot;,&quot;last_name&quot;:&quot;Marrone&quot;},{&quot;first_name&quot;:&quot;Kristen&quot;,&quot;last_name&quot;:&quot;Sebring&quot;},{&quot;first_name&quot;:&quot;Yingzhou&quot;,&quot;last_name&quot;:&quot;Yan&quot;},{&quot;first_name&quot;:&quot;Brent&quot;,&quot;last_name&quot;:&quot;Appleton&quot;},{&quot;first_name&quot;:&quot;Kathleen&quot;,&quot;last_name&quot;:&quot;Aertgeerts&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;Eben Massari&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Guy Breitenbucher&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;7754-7766&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00713&quot;},&quot;abstract&quot;:&quot;In medicinal chemistry, accurate prediction of additivity-based structure–activity relationship (SAR) analysis rests on three assumptions: (1) a consistent binding pose of the central scaffold, (2) no interaction between the R group substituents, and (3) a relatively rigid binding pocket in which the R group substituents act independently. Previously, examples of nonadditive SAR have been documented in systems that deviate from the first two assumptions. Local protein structural change upon ligand binding, through induced fit or conformational selection, although a well-known phenomenon that invalidates the third assumption, has not been linked to nonadditive SAR conclusively. Here, for the first time, we present clear structural evidence that the formation of a hydrophobic pocket upon ligand binding in PDE2 catalytic site reduces the size of another distinct subpocket and contributes to strong nonadditive SAR between two otherwise distant R groups.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00713&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00713" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00713&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00713" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00713&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00713" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00713&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00713&amp;href=/doi/10.1021/acs.jmedchem.8b00713" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00713" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00713" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00713%26sid%3Dliteratum%253Aachs%26pmid%3D30070482%26genre%3Darticle%26aulast%3DGomez%26date%3D2018%26atitle%3DMathematical%2Band%2BStructural%2BCharacterization%2Bof%2BStrong%2BNonadditive%2BStructure%25E2%2580%2593Activity%2BRelationship%2BCaused%2Bby%2BProtein%2BConformational%2BChanges%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D17%26spage%3D7754%26epage%3D7766%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291012" title="Organic acids">Organic acids</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=290723" title="Solutions">Solutions</a>,</li><li><a href="/action/doSearch?ConceptID=292317" title="Ethyl groups">Ethyl groups</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/jmcmar.2018.61.issue-17/20180913/jmcmar.2018.61.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">In medicinal chemistry, accurate prediction of additivity-based structure–activity relationship (SAR) analysis rests on three assumptions: (1) a consistent binding pose of the central scaffold, (2) no interaction between the R group substituents, and (3) a relatively rigid binding pocket in which the R group substituents act independently. Previously, examples of nonadditive SAR have been documented in systems that deviate from the first two assumptions. Local protein structural change upon ligand binding, through induced fit or conformational selection, although a well-known phenomenon that invalidates the third assumption, has not been linked to nonadditive SAR conclusively. Here, for the first time, we present clear structural evidence that the formation of a hydrophobic pocket upon ligand binding in PDE2 catalytic site reduces the size of another distinct subpocket and contributes to strong nonadditive SAR between two otherwise distant R groups.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10654" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10654" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Rational small molecule drug discovery is based on the underlying assumption that the biological properties of a molecule can be predictably altered by incorporating specific structural modifications to that molecule (structure–activity relationships, SARs). The process of lead optimization relies on a level of intuitive pattern recognition by the chemist in the form of a SAR. Given the significance of identifying predictable SAR patterns, it was inevitable that chemists would attempt to improve their intuition by mathematically quantifying the relationship between structure and biological effect (QSAR). One of the first proposed quantitative models to exploit the additivity of SAR relationships was published in 1964 by Free and Wilson, a model now referred to as Free–Wilson additivity.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Although the Free–Wilson model of additivity was quite a simple mathematical concept, the idea has had far reaching implications in drug discovery. Today the medicinal chemistry literature is dominated by papers that optimize one R group, followed by a second R group sequentially (Linear Analoging) rather than creating a matrix of compounds altering multiple R groups simultaneously (Combinatorial Analoging). When chemists engage in linear analoging they make the implicit assumption of additivity articulated in Free and Wilson’s original work (i.e., the best R<sup>1</sup> is the best regardless of the nature of R<sup>2</sup>). Additionally, this assumption underlies many chemical and physical descriptors that are calculated to correlate a compound’s calculated physical properties and biological effect (MW is innately additive; most methods of calculating tPSA and cLogP also rely on the assumption of additivity).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> In fact, the value of nearly all QSAR models that seek to mathematically correlate a molecule’s calculated physical properties with a biological effect would be greatly compromised if the system being studied was found to be inconsistent with classical Free–Wilson additivity.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">Furthermore, most if not all computational scoring functions,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> including Free Energy Perturbation Methods such as FEP<sup>+</sup>,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> work best under the conditions of additivity. However, challenges such as binding mode changes of the ligand and binding site changes are currently being actively investigated.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a></div><div class="NLM_p">In 2006, we described a method in which the Free–Wilson additivity of a combinatorial matrix could be used to evaluate SARs.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> The method we developed allowed one to determine the overall Free–Wilson additivity within a combinatorial matrix of compounds. In cases where nonadditivity was verified, we reasonably hypothesized that this indicated a change in binding mode between the ligands and the target protein, or suggested an interaction between the varied substituents within the set of ligands. Although this hypothesis was intuitive, we had no structural confirmation that nonadditive behavior actually resulted in observable binding mode changes. In fact, although a fundamental change in binding mode is often cited as the cause of nonadditive behavior in medicinal chemistry papers, there are actually very few examples where this is rigorously confirmed with structural data.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">In 2008, researchers from Janssen published an article whereby they used a similar method to quantify the additivity intrinsic to small molecule interactions with a variety of biological targets. The authors came to the conclusion that some targets were prone to display nonadditive SAR, while others were more generally additive in their interactions.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, this work did not contain any structural data that could have provided useful rational and understanding for specific observed nonadditive behavior.</div><div class="NLM_p">With increased knowledge such as available ligand SAR and structural data about the target in question nonadditive behavior can be studied, and in several cases, it was explained by alternative intramolecular substituent interactions or switched binding modes.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a></div><div class="NLM_p last">Recently, Kramer and co-workers published a very clever procedure for determining nonadditive SAR. They applied this method to publicly available SAR data.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In this work, Kramer was able to explain nonadditive data in terms of changes in ligand binding modes within the ligand–protein crystal structures in a few cases. In none of the 15 cases analyzed did the nonadditivity described involve a significant induced conformational change in the protein’s structure. Given the large number of examples of nonadditive behavior described in the medicinal chemistry literature, we suggest that a medicinal chemistry project should never be attempted without first quantifying the degree of nonadditivity intrinsic to the particular system being studied. Here, we employ Kramer’s method as means of quantitating the degree of nonadditivity for any set of compounds using PDE2 as a specific example system. We also demonstrate how obtaining crystal structure of ligands bound to the PDE2 enzyme lead to the identification of putative protein conformational changes responsible for the nonadditive effects seen.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83078" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83078" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this current work, we demonstrate convincingly through the use of quantitative additivity analysis that nonadditivity was significant within the matrix studied. Computational tools and X-ray crystallography were then used to demonstrate protein conformational differences among ligand cocomplexes, which unambiguously explained the source of the nonadditive behavior. Taking together, these findings constitute a case of nonadditive SAR caused by an imposed structural requirement on the protein.</div><div class="NLM_p">The system investigated here uses a triazoloquinozaline scaffold developed by researchers at Janssen R&D as PDE2 inhibitors.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Each compound was synthesized according to the synthetic route shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Condensation of alanine with 4-fluoro-3-nitrobenzoic acid <b>1</b> under basic conditions afforded the desired intermediate <b>2</b> in 60% yield. A subsequent reduction of the nitro group followed by a spontaneous intramolecular cyclization yielded tetrahydroquinoxaline <b>3</b>. The intermediate was then oxidized to the desired dihydroquinoxaline <b>4</b> using hydrogen peroxide under basic conditions. Esterification of the acid moiety was accomplished by treating compound <b>4</b> with ethanol and sulfuric acid. The corresponding dihydroquinoxaline <b>5</b> was then chlorinated under standard conditions, phosphorus oxychloride, and <i>N</i>,<i>N</i>-dimethylaniline, to afford the key intermediate <b>6</b> in moderate yields (17%) after column chromatography. The first set of substitutions was introduced after condensation of five differently substituted 3-chlorobenzhydrazides with intermediate <b>6</b> to provide all six triazoloquinoxaline intermediates, <b>7</b>–<b>12</b>, in moderate to good yields. Saponification of the ester functionality under basic conditions followed by amide bond formation afforded the desired matrix of compounds, <b>19</b>–<b>48</b>, which is described in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. The design was intended to test the steric tolerance of the PDE2 active site at the variable R<sup>1</sup> and R<sup>2</sup> positions in the matrix. It should be noted that a 3D energy minimized model of the compounds suggests no possible direct interaction between R<sup>1</sup> and R<sup>2</sup> due to the rigid nature of the triazoloquinozaline scaffold (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). A crystal structure of compound <b>1</b> (R<sup>1</sup> = Me, R<sup>2</sup> = H) confirmed this observation and suggests that R<sup>1</sup> and R<sup>2</sup> fit into two distinct, nonoverlapping protein pockets present in the catalytic site of PDE2 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The triazoloquinozaline core is sandwiched in the π-clamp formed by Phe862, Phe830, and Ile826. The amide functionality is situated above the π-clamp, positioning the R<sup>1</sup> group in a solvent-exposed region surrounded by lipophilic residues such as Met847. The chlorophenyl group is buried deep inside the binding pocket, near the bound metals, and orients R<sup>2</sup> substitutions toward a water-filled channel between bound metals and Leu770.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: a. alanine, H<sub>2</sub>O, NaHCO<sub>3</sub>, 95 °C; b. Raney Ni, NaHCO<sub>3</sub>, H<sub>2</sub>O, 60 psi; c. (i) 2 N NaOH, 30% H<sub>2</sub>O<sub>2</sub>, 100 °C; (ii) HCl; d. EtOH, H<sub>2</sub>SO<sub>4</sub>, 80 °C; e. POCl<sub>3</sub>, <i>N</i>,<i>N</i>-dimethyl aniline, 120 °C; f. ArCONHNH<sub>2</sub>, <i>n</i>-BuOH, 160 °C; g. LiOH, THF/H<sub>2</sub>O; h. DIEA, HATU, R<sup>1</sup>NH<sub>2</sub>, DMF.</p></p></figure><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Matrix of compounds and lowest energy conformation for compound <b>19</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of compound <b>19</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7E">6C7E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">All compounds were tested for their inhibition of the PDE2 enzyme, reported as pIC<sub>50</sub> , and run in quadruplicate with a mean standard error of all experiments of 0.10 (range of errors = 0.16–0.01; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). We initially plotted our data in a 3D bar graph format by varying the R<sup>1</sup> substituent on the <i>y</i> axis and the R<sup>2</sup> substituent on the <i>x</i> axis (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Upon analysis, this matrix represented a good data set for determining additivity due to measurable enzyme inhibition for all compounds in the matrix and an acceptable range of activities (pIC<sub>50</sub> = 7.6–9.8; range = 2.2 log units) well above the mean standard error of the assay.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. PDE2 pIC<sub>50</sub> values of the compounds in the matrix.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. pIC<sub>50</sub> Data for All Compounds Described in the Combinatorial Matrix</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" rowspan="2" colspan="2" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center">R<sup>1</sup></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Me</th><th class="colsep0 rowsep0" align="center">iPr</th><th class="colsep0 rowsep0" align="center">iBu</th><th class="colsep0 rowsep0" align="center"><i>t</i>Bu</th><th class="colsep0 rowsep0" align="center">adamantyl</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="12" align="left">R<sup>2</sup></td><td class="colsep0 rowsep0" rowspan="2" align="center">H</td><td class="colsep0 rowsep0" align="center"><b>19</b></td><td class="colsep0 rowsep0" align="center"><b>20</b></td><td class="colsep0 rowsep0" align="center"><b>21</b></td><td class="colsep0 rowsep0" align="center"><b>22</b></td><td class="colsep0 rowsep0" align="center"><b>23</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">8.55</td><td class="colsep0 rowsep0" align="center">9.57</td><td class="colsep0 rowsep0" align="center">9.52</td><td class="colsep0 rowsep0" align="center">9.47</td><td class="colsep0 rowsep0" align="center">9.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="center">F</td><td class="colsep0 rowsep0" align="center"><b>24</b></td><td class="colsep0 rowsep0" align="center"><b>25</b></td><td class="colsep0 rowsep0" align="center"><b>26</b></td><td class="colsep0 rowsep0" align="center"><b>27</b></td><td class="colsep0 rowsep0" align="center"><b>28</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">8.69</td><td class="colsep0 rowsep0" align="center">9.17</td><td class="colsep0 rowsep0" align="center">9.37</td><td class="colsep0 rowsep0" align="center">9.30</td><td class="colsep0 rowsep0" align="center">9.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="center">OMe</td><td class="colsep0 rowsep0" align="center"><b>29</b></td><td class="colsep0 rowsep0" align="center"><b>30</b></td><td class="colsep0 rowsep0" align="center"><b>31</b></td><td class="colsep0 rowsep0" align="center"><b>32</b></td><td class="colsep0 rowsep0" align="center"><b>33</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">8.74</td><td class="colsep0 rowsep0" align="center">8.96</td><td class="colsep0 rowsep0" align="center">9.42</td><td class="colsep0 rowsep0" align="center">9.00</td><td class="colsep0 rowsep0" align="center">9.49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="center">OiPr</td><td class="colsep0 rowsep0" align="center"><b>34</b></td><td class="colsep0 rowsep0" align="center"><b>35</b></td><td class="colsep0 rowsep0" align="center"><b>36</b></td><td class="colsep0 rowsep0" align="center"><b>37</b></td><td class="colsep0 rowsep0" align="center"><b>38</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">9.16</td><td class="colsep0 rowsep0" align="center">8.99</td><td class="colsep0 rowsep0" align="center">8.93</td><td class="colsep0 rowsep0" align="center">8.30</td><td class="colsep0 rowsep0" align="center">7.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="center">OiBu</td><td class="colsep0 rowsep0" align="center"><b>39</b></td><td class="colsep0 rowsep0" align="center"><b>40</b></td><td class="colsep0 rowsep0" align="center"><b>41</b></td><td class="colsep0 rowsep0" align="center"><b>42</b></td><td class="colsep0 rowsep0" align="center"><b>43</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">9.61</td><td class="colsep0 rowsep0" align="center">9.46</td><td class="colsep0 rowsep0" align="center">9.29</td><td class="colsep0 rowsep0" align="center">8.75</td><td class="colsep0 rowsep0" align="center">7.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="center">O<i>t</i>Bu</td><td class="colsep0 rowsep0" align="center"><b>44</b></td><td class="colsep0 rowsep0" align="center"><b>45</b></td><td class="colsep0 rowsep0" align="center"><b>46</b></td><td class="colsep0 rowsep0" align="center"><b>47</b></td><td class="colsep0 rowsep0" align="center"><b>48</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">9.16</td><td class="colsep0 rowsep0" align="center">9.10</td><td class="colsep0 rowsep0" align="center">8.88</td><td class="colsep0 rowsep0" align="center">8.35</td><td class="colsep0 rowsep0" align="center">7.58</td></tr></tbody></table></div></div><div class="NLM_p">Simple visual inspection of the data depicted in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> suggests that there may be some nonadditive behavior in the matrix. To quantify the degree of nonadditivity, we utilized the double-transformation procedure described by Kramer.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Using this method, nonadditivity can be quantified for any set of four compounds that undergo two substituent changes in two dimensions. This is illustrated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. PDE2 pIC<sub>50</sub> values of selected compounds in the matrix demonstrating a lack of additivity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Starting with the root compound <b>19</b> in the back left (R<sup>1</sup> = Me, R<sup>2</sup> = H; pIC<sub>50</sub> = 8.6), we can transform R<sup>1</sup> to get compound <b>21</b> (R<sup>1</sup> = iBu, R<sup>2</sup> = H; pIC<sub>50</sub> = 9.5), which results in a pIC<sub>50</sub> increase of 0.9. Additionally, when we transform R<sup>2</sup> providing compound <b>34</b> (R<sup>1</sup> = Me, R<sup>2</sup> = OiPr; pIC<sub>50</sub> = 9.2), this results in a pIC<sub>50</sub> increase of 0.6. However, when we perform both transformations to get compound <b>36</b> (R<sup>1</sup> = iBu, R<sup>2</sup> = OiPr; pIC<sub>50</sub> = 8.9), we get a compound that is significantly less potent than we would expect if there was perfect additivity (additive pIC<sub>50</sub> = 8.6 + 0.9 + 0.6 = 10.1). Thus, we quantitatively characterize this transformation as nonadditive by 1.2 log units (10.1–8.9 = 1.2). Using the equation below allows for the quantification of nonadditivity (NAd) in any double-transformation of this type, as discussed by Kramer.<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_m001.gif" alt="" id="_i6" /></img></span>In order to quantify the nonadditive behavior of the entire combinatorial matrix, one can use the equation above on all double-transformations. Each compound is used as a starting point for 20 double-transformations considering R<sup>1</sup> of 5 and R<sup>2</sup> of 6 varied substituents. After removal of duplicate cycles, 300 NAd values are produced, and their frequencies as a function of nonadditivity are shown as a histogram in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. As well, the experimentally observed and theoretical nonadditivity distribution with a mean standard error of 0.1 are shown. All nonadditivity values larger than ±0.52 cannot be explained with experimentally uncertainty. Thus, we can determine with 99% confidence that 54% of the double transformation data contained in the current matrix is truly nonadditive and not the result of experimental uncertainty. However, we cannot judge the NAd value <0.52 as being innately additive, there is simply too much experimental variance to make a judgment on these transformations.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Theoretical nonadditivity distribution expected from mean standard error of the assay of 0.1 log units (red line) and real nonadditivity distribution (blue line). A histogram for experimental data with <i>y</i> axis on the right side is also shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Once the nonadditivity was quantitatively confirmed, we sought to understand the physical nature of the observed nonadditivity within the set of compounds. Upon inspection of the double transformation data, the largest nonadditive value obtained in the matrix arose from the transformations; R<sup>1</sup> = H to OiBu, and R<sup>2</sup> = Me to adamantyl, resulting in an NAd = 3.1. Interestingly, this represented the largest change in molecular volume within the matrix resulting in a highly significant change in expected activity. In addition, the second largest nonadditive transformations also suggested that the size of the R-groups was playing a significant role in the nonadditive behavior; R<sup>1</sup> = H to O<i>t</i>Bu, and R<sup>2</sup> = Me to Ad. Because of this, we rearranged the matrix in order of the molecular volume of the substituents and plotted the NAd calculated for each compound relative to the smallest compound in the matrix (<b>19</b>; R<sup>1</sup> = Me, R<sup>2</sup> = H) using Kramer’s double transformation method (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Graphical representation of the nonadditivity values for each compound relative to compound <b>19</b>; the dotted line at NAd = 0.5 represents the value that provides 95% confidence that the nonadditivity observed cannot be explained by the experimental uncertainty.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> clearly illustrates that there is a consistent nonadditive relationship within the matrix and that the nonadditivity is related to the size of the variable groups in the matrix. Seemingly, one can increase the size of each of the R-groups independently and increase the activity of the compounds, but if the size of both groups are increased together, a drop in activity occurs.</div><div class="NLM_p">Since a direct interaction between R<sup>1</sup> and R<sup>2</sup> substituents is unlikely to be the source of the nonadditive behavior as we discussed previously, we decided to rely on crystallographic methods to investigate the cause of this observed phenomenon. Therefore, we selected six candidates with varying sizes of R<sup>1</sup> and R<sup>2</sup>, including all four compounds presented in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> for structural determination in the cocomplex of PDE2 enzyme (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv" class="ext-link">Supporting Information</a>). The results indicate that all six compounds adopt a similar binding mode (RMSD values ≤ 1 Å; see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv" class="ext-link">Supporting Information</a>) and rule out the possibility of ligand rearrangement in the binding site, which has traditionally been a common explanation for nonadditivity. Analyses of our crystal structures indicate that both R<sup>1</sup> and R<sup>2</sup> pockets can independently accommodate large hydrophobic functional groups. For example, the adamantyl group, which is the largest R<sup>1</sup> in our tested matrix, fits in the R<sup>1</sup> pocket and is surrounded by Leu770, Met847, and other hydrophobic residues at the entrance of the catalytic site (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv" class="ext-link">Supporting Information</a>). These favorable interactions contribute to about 16-fold improvement for PDE2 when compared to analog <b>19</b> and <b>23</b> (R<sup>1</sup> = Me and Ad, respectively, with R<sup>2</sup> = H, NAd = 1.2). More importantly and unlike the R<sup>1</sup> pocket, significant conformational changes are necessary to accommodate large R<sup>2</sup> groups.</div><div class="NLM_p">In ligand cocomplexes where a large R<sup>2</sup> group is presented to this part of the PDE2 enzyme, protein rearrangement is observed, which primarily involves the shift of Leu770 residue and open a hydrophobic pocket near the bound metals. For example, the compound with the largest R<sup>2</sup> group of the matrix (<b>39</b>; R<sup>1</sup> = Me and R<sup>2</sup> = OiBu) inserts into the newly formed pocket (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv" class="ext-link">Supporting Information</a>) and improves PDE2 inhibition by 7.9-fold when comparing analogs <b>19</b> and <b>39</b> (R<sup>2</sup> = H and OiBu, respectively, with R<sup>1</sup> = Me, NAd = 0.9). Similar structural changes have previously been reported for PDE2 in complex with inhibitor BAY60-7550<a onclick="showRef(event, 'cit15a'); return false;" href="javascript:void(0);" class="ref cit15a">(15a)</a> as well as with similar PDE2 analogs described herein.<a onclick="showRef(event, 'cit15b'); return false;" href="javascript:void(0);" class="ref cit15b">(15b)</a></div><div class="NLM_p">This “leucine flip” is mostly localized in the side chain of Leu770. Similar side chain flexibility as well as larger scale conformational changes concomitant with ligand binding have been well documented<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> in the literature. Protein structural changes upon ligand binding can originate from either conformational selection of one of several pre-existing conformations or an induced fit of the protein by the ligand.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p">Comparing the structures of compounds <b>23</b> and <b>39</b> highlights the structural basis for the observed nonadditivity. Favorable binding of large R<sup>1</sup> and R<sup>2</sup> substituents requires close interaction with residue Leu770, which can adapt different side chain conformations (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). In this case, Leu770 works as a molecular divider that distributes ligand binding volume between R<sup>1</sup> and R<sup>2</sup> pockets. A large R<sup>1</sup> group inevitably results in a small R<sup>2</sup> pocket, and vice versa. For this series of compounds, size increase of both R<sup>1</sup> and R<sup>2</sup> groups leads to steric clashes around Leu770 and deviation from prediction by perfect additivity. A three-dimensional representation of the nonadditive SAR as a function of R-group total surface area is shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. The <i>x</i> and <i>y</i> axes represent the total surface area (tSA) for R<sup>1</sup> and R<sup>2</sup>, respectively, plotted against the deviation from additivity (z axis; NAd). As can be seen, the largest analog <b>43</b> (R<sup>1</sup> = Ad, tSA = 312.9 Å; R<sup>2</sup> = OiBu, tSA = 244.9) leads to the largest NAd of 3.1. The structural determination of compound <b>43</b> reveals an isobutyl group buried in the R<sup>2</sup> hydrophobic pocket and a highly disordered adamantyl group in the R<sup>1</sup> solvent-exposed pocket that displays ambiguous electron density and therefore cannot be modeled confidently. We reasoned that this is due to the steric clash between the adamantyl group and Leu770 when the R<sup>2</sup> hydrophobic pocket is formed. Compound <b>43</b> shows the largest difference between the actual PDE2 pIC<sub>50</sub> and that predicted by perfect additivity. Thus, the nonadditivity we observed can be explained by an indirect, target-mediated interaction between the size R<sup>1</sup> and R<sup>2</sup> groups and the two pockets present in the enzyme.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlay of compounds <b>23</b> (in cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7D">6C7D</a>) and <b>39</b> (in orange, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7F">6C7F</a>) bound to PDE2 enzyme.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Correlation between R<sup>1</sup> and R<sup>2</sup> surface areas and deviation from additive SAR (NAd).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Given the importance and complexity of ligand–protein interactions in drug discovery, medicinal chemists continuously rely on structure–activity relationships to rationally design molecules and improve their binding properties. It is therefore crucial that SAR analyses are carefully conducted in order to properly and accurately determine the optimization strategy. In this study, we have demonstrated that a process that includes a mathematical analysis of nonadditivity coupled with protein–ligand cocomplex structural information reveals an unprecedented protein conformational change as the cause of the nonadditive SAR. Our study shows that optimal inhibition of PDE2 can be achieved by balancing the size of either R<sup>1</sup> or R<sup>2</sup> substituents. Our results indicate, however, that the optimization of these two vectors will diverge due to nonadditive SAR, and therefore, no assumptions should be made. This also represents a challenge for computational scientists to predict the binding energies as conformational changes in the binding site that introduce an additional source of error. In general, free energy calculations seem to work better if only the ligand is modified in the binding site.</div><div class="NLM_p last">We expect that the approach and the findings discussed herein should be generally applicable to other biological targets. In effect, these results give medicinal chemists the ability to analyze and rationalize nonadditive SARs, which unlock the potential of more efficiently improving ligand–protein interactions.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> General Methods</h3><div class="NLM_p last">Solvents and reagents were purchased and used without further purification unless otherwise noted. Flash column chromatography (FCC) was performed on Teledyne Isco CombiFlash instruments equipped with silica gel columns. Preparative HPLC purification was performed on a Waters Preparative LCMS system equipped with a Waters XBridge Prep C18 5 μm OBD 19 × 50 mm column. Elution was achieved with increasing concentration of acetonitrile in water and 0.05% TFA added as modifier. Analytical HPLC was performed on an Acquity UPLC system equipped with a BEH C18 1.7 μM 2.1 × 50 mm column. Elution was achieved with increasing concentration of acetonitrile in water and 0.1% formic acid added as modifier. NMR was performed on a Bruker Ultrashield 400 Plus spectrometer. Chemical shifts (δ) are reported in ppm relative to the residual solvent signal. Data for <sup>1</sup>H/<sup>13</sup>C NMR spectra are reported as follows: chemical shift (multiplicity, coupling constants, number of hydrogens). Abbreviations are as follows: s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), br (broad). The purity of all compounds tested was determined by LCMS and <sup>1</sup>H NMR to be >95%. High-resolution mass spectral data were obtained from the University of California, San Diego.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> 4-((1-Carboxyethyl)amino)-3-nitrobenzoic Acid (<b>2</b>)</h3><div class="NLM_p last">To a stirring solution of 4-fluoro-3-nitrobenzoic acid (<b>1</b>; 50.0 g, 270 mmol, 1.0 equiv) in H<sub>2</sub>O (500 mL) was added <span class="smallcaps smallerCapital">l</span>-alanine (48 g, 540 mmol, 2.0 equiv) followed by NaHCO<sub>3</sub> (68 g, 810 mmol, 3.0 equiv). The reaction mixture was heated to 100 °C. After heating the reaction mixture for 36 h at 100 °C, the solution was cooled to room temperature and acidified with 6 N HCl. The resultant precipitate was filtered and dried under vacuum to afford the title compound (<b>2</b>; 41 g, 60% yield) as a light red solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.16 (brs, 2H), 8.70 (d, <i>J</i> = 6.9 Hz, 1H), 8.64 (d, <i>J</i> = 1.8 Hz, 1H), 7.99 (dd, <i>J</i> = 9.0, 2.1 Hz, 1H), 7.09 (d, <i>J</i> = 9.3 Hz, 1H), 4.62–4.53 (m, 1H), 1.50 (d, <i>J</i> = 6.9 Hz, 3H). MS(ESI): <i>m</i>/<i>z</i> 255.16 [M + H]<sup>+</sup>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> 2-Methyl-3-oxo-3,4-dihydroquinoxaline-6-carboxylic Acid (<b>4</b>)</h3><div class="NLM_p">To a stirring solution of 4-((1-carboxyethyl)amino)-3-nitrobenzoic acid (<b>2</b>; 41.0 g, 161 mmol, 1 equiv) in H<sub>2</sub>O (400 mL) in a bomb apparatus was added Raney Ni (30 g wet) followed by NaHCO<sub>3</sub> (40.7 g, 484 mmol, 3.0 equiv). The apparatus was evacuated and refilled with N<sub>2</sub> (2×) before charging with H<sub>2</sub> at 60 psi. After stirring under a H<sub>2</sub> atmosphere at 60 psi for 60 h, the reaction mixture was filtered through a pad of Celite and concentrated in vacuo to afford 2-methyl-3-oxo-3,4-dihydroquinoxaline-6-carboxylic acid (<b>3</b>; 40.0 g) crude, which was carried on without further purification. MS(ESI): <i>m</i>/<i>z</i> 207.04 [M + H]<sup>+</sup>.</div><div class="NLM_p last">The above crude (<b>3</b>; 40.0 g, 194 mmol theoretical, 1.0 equiv) was added to a stirring solution of sodium hydroxide (38.8 g, 5.0 equiv) in H<sub>2</sub>O (400 mL) and 30% H<sub>2</sub>O<sub>2</sub> solution (40 mL) at 0 °C. The reaction mixture was heated to 100 °C. After stirring at 100 °C for 16 h, the reaction mixture was cooled to 0 °C and acidified with 6 N HCl. The precipitated solid was filtered, dried, and triturated with Et<sub>2</sub>O (40 mL) to afford the title compound (<b>4</b>; 20.0 g, 61% yield overall yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.46 (brs, 1H), 7.89 (s, 1H), 7.77 (brs, 2H), 3.86 (brs, 1H), 2.43 (s, 3H). MS(ESI): <i>m</i>/<i>z</i> 205.09 [M + H]<sup>+</sup>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Ethyl 2-Methyl-3-oxo-3,4-dihydroquinoxaline-6-carboxylate (<b>5</b>)</h3><div class="NLM_p last">To a stirring solution of 2-methyl-3-oxo-3,4-dihydroquinoxaline-6-carboxylic acid (<b>4</b>; 20.0 g, 86.1 mmol, 1.0 equiv) in EtOH (400 mL) at 0 °C was added H<sub>2</sub>SO<sub>4</sub> (10 mL). The reaction mixture was warmed to 80 °C. After stirring at 80 °C for 16 h, the reaction mixture was cooled to room temperature and concentrated in vacuo. The crude material was basified with NaHCO<sub>3</sub>(aq.) and extracted with EtOAc (3 × 250 mL). The combined organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford the title compound (<b>5</b>; 13.5 g, 68% yield) as a pink solid, which was carried on without any further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.42 (brs, 1H), 7.89 (s, 1H), 7.78 (brs, 2H), 4.35 (q, <i>J</i> = 7.6 Hz, 2H), 2.43 (s, 3H), 1.34 (t, <i>J</i> = 7.2 Hz, 3H). MS(ESI): <i>m</i>/<i>z</i> 233.15 [M + H]<sup>+</sup>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Ethyl 3-Chloro-2-methylquinoxaline-6-carboxylate (<b>6</b>)</h3><div class="NLM_p last">To a solution of ethyl 2-methyl-3-oxo-3,4-dihydroquinoxaline-6-carboxylate (<b>5</b>; 13.5 g, 58.1 mmol, 1.0 equiv) in N,N-dimethyl aniline (74 mL) was added POCl<sub>3</sub> (54.0 mL, 581 mmol, 10 equiv) dropwise. The resulting mixture was stirred at room temperature for 1 h then heated to 120 °C. After heating the reaction mixture for 1h at 120 °C, the solution was cooled to 0 °C, diluted with NaHSO<sub>4</sub>(aq.), and extracted with EtOAc (3 × 250 mL). The combined organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to obtain a crude solid. The crude solid was purified by neutral alumina gel chromatography (2% DCM in petroleum ether) to afford the title compound (<b>6</b>; 2.5 g, 17%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.70 (d, <i>J</i> = 2.0 Hz, 1H), 8.35 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.06 (d, <i>J</i> = 8.8 Hz, 1H), 4.46 (q, <i>J</i> = 6.8 Hz, 2H), 2.87 (s, 3H), 1.45 (t, <i>J</i> = 6.8 Hz, 3H). MS(ESI): <i>m</i>/<i>z</i> 251.04 [M + H]<sup>+</sup>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Ethyl 1-(2-Chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>7</b>)</h3><div class="NLM_p last">To a stirring mixture of ethyl 3-chloro-2-methylquinoxaline-6-carboxylate (<b>6</b>; 3.0 g, 12 mmol, 1.0 equiv) in <i>n</i>-BuOH (45 mL) in a sealed tube was added 3-chlorobenzhydrazide (3.04 g, 17.9 mmol, 1.5 equiv). The tube was sealed and heated to 160 °C. After heating at 160 °C for 3 h, the reaction mixture was cooled to room temperature and diluted with EtOAc (50 mL) and H<sub>2</sub>O (50 mL). The organic layer was separated and concentrated in vacuo to afford a crude solid. This crude mixture was purified by reversed phase HPLC (MeCN and 0.1% aqueous NH<sub>4</sub>OAc as eluent) to afford <b>7</b> (0.800 g, 18% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.21 (d, <i>J</i> = 8.4 Hz, 1H), 8.09 (d, <i>J</i> = 8.4 Hz, 1H), 7.96 (s, 1H), 7.70 (m, 3H), 7.60 (m, 1H), 4.27 (q, <i>J</i> = 6.9 Hz, 2H), 3.12 (s, 3H), 1.28 (s, <i>J</i> = 6.9 Hz, 3H). MS(ESI): <i>m</i>/<i>z</i> 367.23 [M + H]<sup>+</sup>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Ethyl 1-(2-Chloro-5-fluorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>8</b>)</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Preparation of 2-Chloro-5-fluorobenzohydrazide (<b>48</b>)</h4><div class="NLM_p">To a stirring solution of methyl 2-chloro-5-fluorobenzoate (1.0 g, 5.3 mmol, 1.0 equiv) in EtOH (10 mL) was added hydrazine hydrate (2.8 g, 53 mmol, 10.0 equiv). The reaction mixture was heated to 50 °C for 1 h. The reaction mixture was cooled to 0 °C and triturated with Et<sub>2</sub>O (20 mL). The resultant precipitate was filtered to afford 2-chloro-5-fluorobenzohydrazide (<b>48</b>; 0.800 g, 80% yield) as white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.5 (brs, 1H), 7.38 (dd, <i>J</i> = 8.7, 4.8 Hz, 1H), 7.36–7.26 (m, 2H). MS(ESI): <i>m</i>/<i>z</i> 189.06 [M + H] <sup>+</sup>.</div><div class="NLM_p last">To a stirring solution of ethyl 3-chloro-2-methylquinoxaline-6-carboxylate (<b>6</b>; 3.0 g, 12.7 mmol, 1.0 equiv) in <i>n</i>-BuOH (45 mL) in a microwave vial was added hydrazide <b>48</b> (3.1 g, 16.5 mmol, 1.3 equiv). The vial was capped and heated at 160 °C in the microwave for 1 h. After heating at 160 °C in the microwave for 1 h, the reaction mixture was cooled to room temperature and concentrated in vacuo to afford crude compound. The crude compound was purified by reverse phase column chromatography (MeCN and 0.1% aqueous NH<sub>4</sub>OAc as eluent) to afford <b>6</b> (1.0 g, 21% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.21 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 8.11 (d, <i>J</i> = 8.8 Hz, 1H), 7.98 (d, <i>J</i> = 1.6 Hz, 1H), 7.67 (dd, <i>J</i> = 4.8, 8.8 Hz, 1H), 7.47 (m, 2H), 4.29 (q, <i>J</i> = 7.6 Hz, 2H), 3.12 (s, 3H), 1.3 (t, <i>J</i> = 7.6 Hz, 3H). MS(ESI): <i>m</i>/<i>z</i> 370.92 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Ethyl 1-(2-Chloro-5-methoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>9</b>)</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Preparation of 2-Chloro-5-methoxybenzohydrazide (<b>49</b>)</h4><div class="NLM_p">To a stirring solution of 2-chloro-5-hydroxybenzoic acid (5.0 g, 29 mmol, 1.0 equiv) in DMF (50 mL) at 0 °C was added K<sub>2</sub>CO<sub>3</sub> (20.0 g, 145 mmol, 5.0 equiv) followed by methyl iodide (21.7 mL, 348 mmol, 12.0 equiv). The reaction mixture was heated to 40 °C. After heating at 40 °C for 12 h, the reaction mixture was cooled to room temperature and diluted with precooled H<sub>2</sub>O (50 mL). The aqueous layer was extracted with ethyl acetate (3 × 50 mL). The combined organics were washed with H<sub>2</sub>O (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford methyl 2-chloro-5-methoxybenzoate (5.2 g, 89% yield) as a liquid that was used without further purification. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.38–7.32 (m, 2H), 6.96 (dd, <i>J</i> = 9.0, 3.0 Hz, 1H), 3.93 (s, 3H), 3.82 (s, 3H). MS(ESI): <i>m</i>/<i>z</i> 201.19 [M + H]<sup>+</sup>. To a solution of 2-chloro-5-methoxybenzoate (5.0 g, 24.9 mmol, 1.0 equiv) in EtOH (50 mL) was added hydrazine hydrate (12.5 mL, 249 mmol, 10.0 equiv). The reaction mixture was heated to 50 °C. After heating for 3 h at 50 °C, the reaction mixture was cooled to 0 °C and triturated with Et<sub>2</sub>O (100 mL). The formed precipitate was filtered to afford the title compound (<b>49</b>; 5.2 g, 84% purity by LCMS) as a white solid that was used without further purification. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (brs, 1H), 7.38 (d, <i>J</i> = 8.7 Hz, 1H), 7.01 (dd, <i>J</i> = 9.0, 3.3 Hz, 1H), 6.93 (d, <i>J</i> = 3.0 Hz, 1H), 3.77 (s, 3H). MS(ESI): <i>m</i>/<i>z</i> 201.06 [M + H]<sup>+</sup>.</div><div class="NLM_p last">To a stirring solution of ethyl 3-chloro-2-methylquinoxaline-6-carboxylate (<b>6</b>; 0.75 g, 3.0 mmol, 1.0 equiv) in <i>n</i>-BuOH (11.3 mL) in a microwave vial was added 2-chloro-5-methoxybenzohydrazide (<b>49</b>; 0.78 g, 3.9 mmol 1.3 equiv). The microwave vial was capped and heated to 160 °C in the microwave. After heating at 160 °C in the microwave for 0.5 h, the reaction mixture was cooled to room temperature and concentrated in vacuo to afford crude solid. The crude compound was purified by reverse phase column chromatography (MeCN and 0.1% aqueous NH4OAc as eluent) to afford ethyl 1-(2-chloro-5-methoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>9</b>; 0.35 g, 29% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.22 (dd, <i>J</i> = 8.4, 1.6 Hz, 1H), 8.10 (d, <i>J</i> = 12.8 Hz, 1H), 8.05 (d, <i>J</i> = 1.6 Hz, 1H), 7.56 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H), 7.21 (m, 2H), 4.29 (q, <i>J</i> = 7.6 Hz, 2H), 3.87 (s, 3H), 3.11 (s, 3H), 1.3 (t, <i>J</i> = 7.6 Hz, 3H). MS(ESI): <i>m</i>/<i>z</i> 397.2 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Ethyl 1-(2-Chloro-5-isopropoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>10</b>)</h3><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Preparation of 2-Chloro-5-isopropoxybenzohydrazide (<b>50</b>)</h4><div class="NLM_p">To a stirring solution of 2-chloro-5-hydroxybenzoic acid (7.0 g, 41 mmol, 1.0 equiv) in DMF (70 mL) at 0 °C was added K<sub>2</sub>CO<sub>3</sub> (28.0 g, 203 mmol, 5.0 equiv) and isopropyl iodide (20.3 mL, 203 mmol, 5.0 equiv). The reaction mixture was heated to 80 °C. After stirring at 80 °C for 12 h, the reaction mixture was cooled to room temperature, diluted with precooled H<sub>2</sub>O, and extracted with EtOAc (3 × 70 mL). The combined organics were washed with H<sub>2</sub>O (70 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford isopropyl 2-chloro-5-isopropoxybenzoate (8.1 g, 77% yield) as a liquid that was used without further purification. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.32–7.26 (m, 2H), 6.91 (dd, <i>J</i> = 9.3, 3.3 Hz, 1H), 5.28–5.24 (m, 1H), 4.58–4.49 (m, 1H), 1.38 (d, <i>J</i> = 6.0 Hz, 6H), 1.33 (d, <i>J</i> = 6.3 Hz, 6H). MS(ESI): <i>m</i>/<i>z</i> 256.12 [M + H]<sup>+</sup>. To a stirring solution of isopropyl 2-chloro-5-isopropoxybenzoate (8.0 g, 31 mmol, 1.0 equiv) in EtOH (80 mL) was added hydrazine hydrate (15.6 mL, 310 mmol, 10.0 equiv). The reaction mixture was heated to 80 °C. After heating at 80 °C for 12 h, the reaction mixture was cooled to room temperature and concentrated in vacuo to obtain a crude solid. The crude solid was purified by silica gel chromatography (3% MeOH in DCM) to afford 2-chloro-5-isopropoxybenzohydrazide (<b>50</b>; 7.1 g, 100%) as an off-white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.52 (brs, 1H), 7.35 (d, <i>J</i> = 9.0 Hz, 1H), 6.98 (dd, <i>J</i> = 9.0, 3.0 Hz, 1H), 6.88 (d, <i>J</i> = 3.0 Hz, 1H), 4.66–4.58 (m, 1H), 4.45 (brs, 2H), 1.26 (d, <i>J</i> = 6.0 Hz, 6H). MS(ESI): <i>m</i>/<i>z</i> 229.03 [M + H]+.</div><div class="NLM_p last">To a stirring solution of ethyl 3-chloro-2-methylquinoxaline-6-carboxylate (<b>6</b>; 0.75 g, 3.0 mmol, 1.0 equiv) in <i>n</i>-BuOH (11.3 mL) in a microwave vial was added 2-chloro-5-isopropoxybenzohydrazide (50; 0.89 g, 3.9 mmol, 1.3 equiv). The vial was capped, and the reaction mixture was heated to 160 °C in the microwave. After heating for 30 min at 160 °C in the microwave, the reaction mixture was cooled to room temperature and concentrated in vacuo to obtain a crude solid. The crude solid was purified by reverse phase HPLC (Reveleris C-18, 80% MeCN in 0.1% NH4OAc) to afford the title compound (<b>10</b>; 0.50 g, 39% yield) as an off-white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.15 (m, 2H), 7.85 (brs, 1H), 7.73 (d, <i>J</i> = 8.4 Hz, 1H), 7.41–7.35 (m, 2H), 4.72–4.64 (m, 1H), 4.21 (q, <i>J</i> = 6.6 Hz, 2H), 2.98 (s, 3H), 1.31 (d, <i>J</i> = 6.3 Hz, 6H), 1.24 (t, <i>J</i> = 6.3 Hz, 3H). MS(ESI): <i>m</i>/<i>z</i> 425.16 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Ethyl 1-(2-Chloro-5-isobutoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>11</b>)</h3><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Preparation of 2-Chloro-5-isobutoxybenzohydrazide (<b>51</b>)</h4><div class="NLM_p">To a stirring solution of 2-chloro-5-hydroxybenzoic acid (7.0 g, 41 mmol, 1.0 equiv) in DMF (70 mL) at 0 °C was added K<sub>2</sub>CO<sub>3</sub> (28.0 g, 203 mmol, 5.0 equiv) and 1-bromo-2-methylpropane (21.7 mL, 203 mmol, 5.0 equiv). The reaction mixture was heated to 80 °C. After heating at 80 °C for 12 h, the reaction mixture was cooled to room temperature, diluted with precooled H<sub>2</sub>O, and extracted with EtOAc (3 × 70 mL). The combined organics were washed with H<sub>2</sub>O (70 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford isobutyl 2-chloro-5-isobutoxybenzoate (10.0 g, 35.1 mmol) as a liquid that was used without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33 (s, 1H), 7.31 (d, <i>J</i> = 6.0 Hz, 1H), 6.94 (dd, <i>J</i> = 8.8, 3.2 Hz, 1H), 4.12 (d, <i>J</i> = 6.4 Hz, 2H), 3.72 (d, <i>J</i> = 6.8 Hz, 2H), 2.11–2.06 (m, 2H), 1.03 (d, <i>J</i> = 2.4 Hz, 6H), 1.02 (d, <i>J</i> = 2.4 Hz, 6H). To a stirring solution of isobutyl 2-chloro-5-isobutoxybenzoate (10.0 g, 35.1 mmol, 1.0 equiv) in EtOH (100 mL) was added hydrazine hydrate (17.6 mL, 350 mmol, 10.0 equiv). The reaction mixture was heated to 80 °C. After heating at 80 °C for 12 h, the reaction mixture was cooled to room temperature and concentrated in vacuo to obtain a crude solid. The crude solid was purified by silica gel chromatography (3% MeOH in DCM) to afford 2-chloro-5-isobutoxybenzohydrazide (<b>51</b>; 8.1 g, 95% yield) as an off-white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.52 (brs, 1H), 7.36 (d, <i>J</i> = 9.0 Hz, 1H), 7.00 (dd, <i>J</i> = 9.0, 3.0 Hz, 1H), 6.91 (d, <i>J</i> = 3.0 Hz, 1H), 4.45 (brs, 2H), 3.75 (d, <i>J</i> = 6.6 Hz, 2H), 2.05–1.96 (m, 1H), 0.96 (d, <i>J</i> = 6.6 Hz, 6H). MS(ESI): <i>m</i>/<i>z</i> 243.08 [M + H]<sup>+</sup>.</div><div class="NLM_p last">To a stirring solution of ethyl 3-chloro-2-methylquinoxaline-6-carboxylate (<b>6</b>; 0.75 g, 3.0 mmol, 1.0 equiv) in <i>n</i>-BuOH (11.3 mL) was added 2-chloro-5-isobutoxybenzohydrazide (<b>51</b>; 0.94 g, 3.87 mmol, 1.3 equiv) in a microwave vial. The vial was capped and heated to 160 °C in the microwave. After heating at 160 °C for 30 min in the microwave, the reaction mixture was cooled to room temperature and concentrated in vacuo to obtain a crude solid. The crude solid was purified by reversed phase chromatography (Reveleris C-18, 80% MeCN in 0.1% NH<sub>4</sub>OAc(aq.)) to afford ethyl 1-(2-chloro-5-isobutoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>11</b>; 0.55 g, 42% yield) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.15–8.14 (m, 2H), 7.85 (brs, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 1H), 7.43–7.39 (m, 2H), 4.21 (q, <i>J</i> = 7.2 Hz, 2H), 3.82 (t, <i>J</i> = 6.4 Hz, 2H), 2.98 (s, 3H), 2.05–2.02 (m, 1H), 1.22 (t, <i>J</i> = 6.8 Hz, 3H), 0.97 (dd, <i>J</i> = 6.4, 2.8 Hz, 6H). MS(ESI): <i>m</i>/<i>z</i> 438.8 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Ethyl 1-(5-(<i>tert</i>-Butoxy)-2-chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>12</b>)</h3><div id="sec4_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Preparation of 5-(<i>tert</i>-Butoxy)-2-chlorobenzohydrazide (<b>52</b>)</h4><div class="NLM_p">To a stirred solution of methyl 2-chloro-5-hydroxybenzoate (5.0 g, 26.8 mmol, 1.0 equiv) in toluene (50 mL) was added <i>N</i>,<i>N</i>-dimethylformamide di<i>tert</i>-butyl acetal (12.8 mL, 53.6 mmol, 2.0 equiv). The reaction mixture was heated to 110 °C. After heating at 110 °C for 16 h, the reaction mixture was cooled to room temperature, and additional <i>N</i>,<i>N</i>-dimethylformamide di<i>tert</i>-butyl acetal (6.4 mL, 26.8 mmol, 1.0 equiv) was added. The reaction mixture was heated back to 110 °C. After heating an additional 16 h at 110 °C, the reaction was cooled to room temperature and concentrated in vacuo to obtain a crude solid. The crude solid was purified by silica gel chromatography (10% EtOAc in hexanes) to afford methyl 5-(tert-butoxy)-2-chlorobenzoate (2.2 g, 34% yield) as a colorless liquid and recovered starting material (2.5 g, 50%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.47 (d, <i>J</i> = 8.7 Hz, 1H), 7.35 (d, <i>J</i> = 2.7 Hz, 1H), 7.20 (dd, <i>J</i> = 9.0, 3.0 Hz, 1H), 3.85 (s, 3H), 1.31 (s, 9H). MS(ESI): <i>m</i>/<i>z</i> 243.10 [M + H]<sup>+</sup>. To a stirring solution of methyl 5-(<i>tert</i>-butoxy)-2-chlorobenzoate (3.30 g, 13.6 mmol, 1.0 equiv) in EtOH (66 mL) was added hydrazine hydrate (11.3 mL, 136 mmol, 10.0 equiv). The reaction mixture was heated to 80 °C. After heating at 80 °C for 16 h, the reaction mixture was cooled to room temperature and concentrated in vacuo to obtain crude solid. The crude solid was purified by silica gel chromatography (3% EtOH in EtOAc) to afford the title compound (<b>52</b>; 2.4 g, 73% yield) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.54 (brs, 1H), 7.37 (d, <i>J</i> = 8.4 Hz, 1H), 7.04 (dd, <i>J</i> = 8.8, 3.2 Hz, 1H), 6.93 (d, <i>J</i> = 2.8 Hz, 1H), 4.47 (brs, 2H), 1.30 (s, 9H). MS(ESI): <i>m</i>/<i>z</i> 243.12 [M + H]<sup>+</sup>.</div><div class="NLM_p last">To a stirring solution of ethyl 3-chloro-2-methylquinoxaline-6-carboxylate (<b>6</b>; 0.12 g, 0.48 mmol, 1.0 equiv) in 2,6-lutidine (0.6 mL) was added 5-(<i>tert</i>-butoxy)-2-chlorobenzohydrazide (<b>52</b>; 0.12 g, 0.48 mmol, 1.0 equiv) The reaction mixture was concentrated in vacuo to afford a crude solid. The crude solid was purified by silica gel chromatography (30% EtOAc in heptanes) to afford the title compound (<b>12</b>; 0.15 g, 71% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.28 (dd, <i>J</i> = 1.6, 8.4 Hz, 1H), 8.17 (d, <i>J</i> = 8.4 Hz, 1H), 8.05–7.99 (m, 1H), 7.63–7.56 (m, 1H), 7.38–7.31 (m, 2H), 3.16 (s, 3H), 1.43 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 168.8, 155.6, 155.1, 147.4, 144.7, 139.5, 130.8, 130.1, 129.1, 128.8, 128.7, 127.6,127.4, 125.4, 117.5, 80.3, 29.7, 28.8, 21.5. MS(ESI): <i>m</i>/<i>z</i> 410.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 1-(2-Chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylic Acid (<b>13</b>)</h4><div class="NLM_p last">To a stirring solution of ethyl 1-(2-chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (0.600 g, 1.77 mmol) in THF/H<sub>2</sub>O (2:1, 16.5 mL) at 0 °C was added LiOH (0.064 g, 2.7 mmol, 1.5 equiv). The reaction mixture was warmed to room temperature. After stirring 2 h, the organics were concentrated in vacuo, and the remaining aqueous layer was acidified with KHSO<sub>4</sub>. The aqueous layer was extracted with 5% MeOH in DCM (3 × 20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude solid. The crude solid was triturated with Et<sub>2</sub>O (6 mL) to afford the title compound (<b>13</b>; 0.500 g, 84% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.20 (brs, 1H), 8.12 (s, 2H), 7.83 (m, 3H), 7.78 (s, 1H), 7.71 (dt, <i>J</i> = 11.6, 2.0 Hz, 1H), 2.98 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 155.0, 147.4, 144.8, 138.9, 134.1, 133.9, 133.3, 130.8, 130.5, 130.2, 128.8, 128.4,127.8, 125.6, 116.8, 21.4. EI-HR calcd for C<sub>17</sub>H<sub>12</sub>ClN<sub>4</sub>O<sub>2</sub> (M<sup>+</sup>) 339.0643, found 339.0642.</div></div><div id="sec4_11_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 1-(2-Chloro-5-fluorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylic Acid (<b>14</b>)</h4><div class="NLM_p last">To a stirring solution of ethyl 1-(2-chloro-5-fluorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (1.8 g, 4.9 mmol) in THF/H<sub>2</sub>O (2:1, 49 mL) was added LiOH (0.30 g, 7.4 mmol, 1.5 equiv). After 3 h, the organics were concentrated in vacuo. The resultant aqueous layer was acidified with NaHSO<sub>4</sub>, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford crude compound. The crude compound was triturated with Et<sub>2</sub>O (16 mL) to afford the title compound (<b>14</b>; 1.6 g, 92% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.20 (brs, 1H), 8.15 (s, 2H), 7.94 (dd, <i>J</i> = 9.2, 4.8 Hz, 1H), 7.84–7.83 (m, 3H), 2.99 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 162.81–159.73 (m, 1C), 155.0, 146.3 (d, <i>J</i> = 1.5 Hz, 1C), 144.8, 138.8, 132.6 (d, <i>J</i> = 8.8 Hz, 1C),130.9, 130.3, 129.9 (d, <i>J</i> = 3.7 Hz, 1C), 129.4 (d, <i>J</i> = 9.5 Hz, 1C), 128.5, 125.4, 121.28–120.76 (m, 1C), 120.67–120.22 (m, 1C), 116.7, 21.4. EI-HR calcd for C<sub>17</sub>H<sub>11</sub>ClFN<sub>4</sub>O<sub>2</sub> (M<sup>+</sup>) 357.0549, found 357.0548.</div></div><div id="sec4_11_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 1-(2-Chloro-5-methoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylic Acid (<b>15</b>)</h4><div class="NLM_p last">To a stirring solution of ethyl 1-(2-chloro-5-methoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>9</b>; 1.3 g, 3.3 mmol, 1.0 equiv) in THF/H<sub>2</sub>O (2:1, 36 mL) was added LiOH (0.21 g, 4.9 mmol, 1.5 equiv). After stirring at room temperature for 3 h, the organics were concentrated in vacuo. The aqueous layer was acidified with NaHSO<sub>4</sub>, extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford the crude solid. This crude mixture was triturated with Et<sub>2</sub>O (5 mL) to afford the title compound (<b>15</b>; 0.71 g, 58% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.2 (brs, 1H), 8.13 (d, <i>J</i> = 0.8 Hz, 2H), 7.86 (s, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 1H), 7.42 (m, 2H), 3.83 (s, 3H), 2.97 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 159.1, 155.0, 147.3, 144.7, 138.8, 131.5, 130.8, 130.1, 128.4, 128.4, 125.5, 125.2, 119.6, 118.3, 117.0, 56.5, 21.4. EI-HR calcd for C<sub>18</sub>H<sub>14</sub>ClN<sub>4</sub>O<sub>3</sub> (M<sup>+</sup>) 369.0749, found 369.0749.</div></div><div id="sec4_11_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 1-(2-Chloro-5-isopropoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylic Acid (<b>16</b>)</h4><div class="NLM_p last">To a stirring solution of ethyl 1-(2-chloro-5-isopropoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>10</b>; 3.0 g, 7.1 mmol, 1.0 equiv) in 1:1 mixture of THF/H<sub>2</sub>O (120 mL) was added lithium hydroxide (0.42 g, 18 mmol, 2.5 equiv). After stirring for 4 h, the reaction mixture was concentrated in vacuo to obtain a crude solid. The crude solid was dissolved in H<sub>2</sub>O (40 mL) and washed with EtOAc (3 × 40 mL). The aqueous layer was acidified with NaHSO<sub>4</sub> (aq.) and extracted with EtOAc (3 × 40 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford the title compound (<b>16</b>; 2.15 g, 76% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.21 (brs, 1H), 8.12 (d, <i>J</i> = 0.8 Hz, 2H), 7.85 (brs, 1H), 7.71 (d, <i>J</i> = 8.8 Hz, 1H), 7.40 (d, <i>J</i> = 3.2 Hz, 1H), 7.36 (dd, <i>J</i> = 8.8, 3.2 Hz, 1H), 4.68 (m, 1H), 2.98 (s, 3H), 1.28 (dd, <i>J</i> = 14.8, 6.0 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 157.3, 154.9, 147.2, 144.7, 138.8, 131.6, 130.9, 130.1, 128.5, 128.4, 125.6, 124.8, 121.2, 119.9, 117.0, 70.9, 22.0, 21.9, 21.4. EI-HR calcd for C<sub>20</sub>H<sub>18</sub>ClN<sub>4</sub>O<sub>3</sub> (M<sup>+</sup>) 397.1062, found 397.1061.</div></div><div id="sec4_11_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 1-(2-Chloro-5-isobutoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylic Acid (<b>17</b>)</h4><div class="NLM_p last">To a stirring solution of ethyl 1-(2-chloro-5-isobutoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>11</b>; 2.35 g, 5.35 mmol, 1.0 equiv) in 1:1 mixture of THF/H<sub>2</sub>O (94 mL) was added lithium hydroxide (0.32 g, 13 mmol, 2.5 equiv). After stirring for 3 h, the reaction mixture was concentrated in vacuo to obtain a crude solid. The crude solid was dissolved in H<sub>2</sub>O (35 mL) and washed with EtOAc (3 × 35 mL). The aqueous layer was acidified with NaHSO<sub>4</sub> (aq.) and extracted with EtOAc (3 × 40 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford the title compound (<b>17</b>; 1.73 g, 79% yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.21 (brs, 1H), 8.13 (d, <i>J</i> = 0.9 Hz, 2H), 7.87 (brs, 1H), 7.72 (d, <i>J</i> = 9.0 Hz, 1H), 7.42–7.36 (m, 2H), 3.85–3.80 (m, 2H), 2.98 (s, 3H), 2.06–1.99 (m, 1H), 0.96 (dd, <i>J</i> = 6.9, 3.3 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 158.6, 155.0, 147.3, 144.7, 138.9, 131.5, 130.8, 130.1, 128.4, 128.4, 125.6,125.0, 120.2, 118.8, 117.0, 75.0, 28.0, 21.4, 19.4. EI-HR calcd for C<sub>21</sub>H<sub>20</sub>ClN<sub>4</sub>O<sub>3</sub> (M<sup>+</sup>) 411.1218, found 411.1215.</div></div><div id="sec4_11_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-(5-(<i>tert</i>-Butoxy)-2-chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylic Acid (<b>18</b>)</h4><div class="NLM_p last">To a stirring solution of ethyl 1-(5-(tert-butoxy)-2-chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxylate (<b>12</b>; 0.80 g, 1.8 mmol, 1.0 equiv) in THF (8 mL) was added 1 M LiOH(aq.) (0.32 g, 8.0 mmol, 4.4 equiv). The reaction mixture was heated to 50 °C. After heating at 50 °C for 2 h, the reaction mixture was cooled to room temperature and diluted with H<sub>2</sub>O (25 mL). The pH was adjusted to 5 with 1 N HCl and extracted with DCM (2 × 25 mL). The combined organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford the title compound (<b>18</b>; 0.60 g, 80% yield) as a light-yellow solid that was used without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.10–9.25 (m, 1 H), 8.34–8.24 (m, 1 H), 8.19 (m, 1 H), 8.06–7.99 (m, 1 H), 7.63–7.55 (m, 1 H), 7.35–7.30 (m, 2 H), 3.25–3.07 (m, 3 H), 1.41 (s, 9 H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 168.8, 155.6, 155.1, 147.4, 144.7, 139.5, 130.8, 130.1, 129.1, 128.8, 128.7, 127.6, 127.4, 125.4, 117.5, 80.3, 29.7, 28.8, 21.5. EI-HR calcd for C<sub>21</sub>H<sub>20</sub>ClN<sub>4</sub>O<sub>3</sub> (M<sup>+</sup>) 411.1218, found 411.1218.</div></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> General Procedure for Amide Coupling Reaction</h3><div class="NLM_p">To a stirring solution of carboxylic acid (0.20 mmol, 1.0 equiv) in DMF (1.0 mL) was added HATU (0.24 mmol, 1.2 equiv), diisopropylethylamine (0.41 mmol, 2.1 equiv), and respective amine (0.26 mmol, 1.3 equiv). After 2 h, the reaction mixture was filtered and purified by reversed phase HPLC. The resultant aqueous fractions were basified with NaHCO<sub>3</sub>(aq.), extracted with DCM (3 × 20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford the respective desired amide product.</div><div id="sec4_12_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 1-(2-Chlorophenyl)-<i>N</i>,4-dimethyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>19</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling reaction with carboxylic acid <b>13</b> (70.0 mg, 0.21 mmol) afforded the title compound (<b>19</b>; 67 mg, 92% yield), isolated by filtration without purification by reversed phase HPLC. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.71–8.56 (m, 1H), 8.17–7.97 (m, 2H), 7.89–7.65 (m, 5H), 2.97 (s, 3H), 2.73 (d, <i>J</i> = 4.5 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.0, 154.2, 147.4, 144.8, 137.8, 134.5, 134.2, 133.8, 133.2, 130.6, 129.7, 128.8, 127.8, 125.8, 125.6, 115.7, 40.7, 40.3, 39.8, 39.4, 26.9, 21.4. EI-HR calcd for C<sub>18</sub>H<sub>15</sub>ClN<sub>5</sub>O (M<sup>+</sup>) 352.0960, found 352.0960.</div></div><div id="sec4_12_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 1-(2-Chlorophenyl)-<i>N</i>-isopropyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>20</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling reaction with carboxylic acid <b>13</b> (80.0 mg, 0.24 mmol) and isopropylamine (40 μL, 0.47 mmol, 2.0 equiv) afforded the title compound (<b>20</b>; 43 mg, 48% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.31 (d, <i>J</i> = 7.7 Hz, 1H), 8.10 (s, 2H), 7.87–7.79 (m, 3H), 7.74–7.68 (m, 2H), 4.00 (dd, <i>J</i> = 6.7, 14.1 Hz, 1H), 2.97 (s, 3H), 1.11 (dd, <i>J</i> = 6.5, 10.1 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.8, 154.1, 147.4, 144.8, 137.7, 134.9, 134.2, 133.8, 133.3, 130.5, 129.6, 128.7, 127.8, 126.0, 125.6, 115.7, 41.7, 22.7, 22.7, 21.4. EI-HR calcd for C<sub>20</sub>H<sub>19</sub>ClN<sub>5</sub>O (M<sup>+</sup>) 380.1270, found 380.1273.</div></div><div id="sec4_12_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 1-(2-Chlorophenyl)-<i>N</i>-isobutyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>21</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid <b>13</b> (70.0 mg, 0.21 mmol) and isobutylamine (27 μL, 0.27 mmol, 1.3 equiv) afforded the title compound (<b>21</b>; 83 mg, 100% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (d, <i>J</i> = 8.4 Hz, 1H), 7.90 (dd, <i>J</i> = 1.5, 8.4 Hz, 1H), 7.73–7.60 (m, 4H), 7.61–7.49 (m, 1H), 6.31 (br. s., 1H), 3.18 (td, <i>J</i> = 6.5, 10.9 Hz, 2H), 3.07 (s, 3H), 1.91–1.68 (m, 1H), 0.90 (d, <i>J</i> = 6.7 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.4, 154.4, 147.5, 144.8, 138.0, 135.0, 134.6, 133.1, 132.6, 130.2, 130.0, 127.9, 127.5, 125. 7, 125.5, 114.6, 77.4, 77.1, 76.8, 47.5, 28.4, 21.4, 20.2. EI-HR calcd for C<sub>21</sub>H<sub>21</sub>ClN<sub>5</sub>O (M<sup>+</sup>) 394.1429, found 394.1427.</div></div><div id="sec4_12_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-(<i>tert</i>-Butyl)-1-(2-chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>22</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid <b>13</b> (55 mg, 0.16 mmol) and <i>tert</i>-butylamine (22 μL, 0.21 mmol, 1.3 equiv) afforded the title compound (<b>22</b>; 32 mg, 50% yield). <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.08 (d, <i>J</i> = 8.6 Hz, 1 H), 7.95 (dd, <i>J</i> = 8.6, 1.7 Hz, 1 H), 7.71–7.82 (m, 3 H), 7.54–7.71 (m, 2 H), 3.07 (s, 3 H), 1.37 (s, 9 H). <sup>13</sup>C NMR (101 MHz, MeOD-<i>d</i><sub>4</sub>) δ 166.3, 153.9, 147.7, 144.6, 137.5, 136.4,134.9, 133.3, 132.5, 130.2, 129.6, 128.0, 126.9, 126.4, 125.1, 114.7, 51.7, 28.0, 20.5. EI-HR calcd for C<sub>21</sub>H<sub>21</sub>ClN<sub>5</sub>O (M<sup>+</sup>) 394.1429, found 394.1427.</div></div><div id="sec4_12_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>-((3s,5s,7s)-Adamantan-1-yl)-1-(2-chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>23</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid <b>13</b> (80 mg, 0.24 mmol) and adamantylamine (46 mg, 0.31 mmol, 1.3 equiv) afforded the title compound (<b>23</b>; 74 mg, 66% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.09–8.05 (m, 1H), 8.04–8.00 (m, 1H), 7.88–7.81 (m, 3H), 7.75–7.67 (m, 1H), 7.62 (s, 1H), 7.58 (d, <i>J</i> = 1.6 Hz, 1H), 2.97 (s, 3H), 2.04 (br. s., 3H), 1.96 (d, <i>J</i> = 2.4 Hz, 6H), 1.64 (br. s., 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.7, 154.0, 147.3, 144.8, 137.6, 136.2, 134.2, 133.8, 133.4, 130.5, 129.5, 128.7, 127.7, 126.6, 125.4, 115.4, 52.1, 41.2, 36.5, 29.3, 21.3. EI-HR calcd for C<sub>27</sub>H<sub>27</sub>ClN<sub>5</sub>O (M<sup>+</sup>) 472.1899, found 472.1899.</div></div><div id="sec4_12_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 1-(2-Chloro-5-fluorophenyl)-<i>N</i>,4-dimethyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>24</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>14</b>; 70 mg, 0.20 mmol) and methylamine (0.13 mL, 0.26 mmol, 1.3 equiv) afforded title compound (<b>24</b>; 72 mg, 99% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.66 (d, <i>J</i> = 4.0 Hz, 1H), 8.32 (s, 1H), 8.10 (d, <i>J</i> = 7.5 Hz, 2H), 7.97–7.87 (m, 1H), 7.78 (s, 3H), 2.97 (s, 3H), 2.76 (d, <i>J</i> = 4.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.9, 162.5, 160.0, 154.1, 146.3, 144.8, 137.7, 134.5, 132.6, 132.5, 129.9, 129.9, 129.8, 129.5, 129.4, 125.7, 125.6, 121.1, 120.9, 120.5, 120.3, 115.7, 79.7, 40.7, 40.2, 39.8, 39.4, 26.9, 21.4. EI-HR calcd for C<sub>18</sub>H<sub>14</sub>ClFN<sub>5</sub>O (M<sup>+</sup>) 370.0865, found 370.0862.</div></div><div id="sec4_12_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 1-(2-Chloro-5-fluorophenyl)-<i>N</i>-isopropyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>25</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>14</b>; 70 mg, 0.20 mmol) and isopropylamine (22 μL, 0.26 mmol, 1.3 equiv) afforded title compound (<b>25</b>; 68 mg, 87% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.39 (d, <i>J</i> = 7.1 Hz, 1H), 8.12 (d, <i>J</i> = 4.4 Hz, 2H), 7.73 (br. s., 4H), 4.23–3.86 (m, 1H), 2.97 (br. s., 3H), 1.13 (t, <i>J</i> = 6.3 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.7, 163.6, 162.5, 160.0, 154.1, 146.2, 144.8, 137.7, 134.9, 134.9, 132.6, 132.5, 129.9, 129.7, 129.4, 126.1, 125.4, 121.1, 120.9, 120.6, 120.4, 115.7, 41.7, 41.6, 40.7, 40.2, 39.8, 39.4, 22.7, 21.4. EI-HR calcd for C<sub>20</sub>H<sub>18</sub>ClFN<sub>5</sub>O (M<sup>+</sup>) 398.1178, found 398.1177.</div></div><div id="sec4_12_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 1-(2-Chloro-5-fluorophenyl)-<i>N</i>-isobutyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>26</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>14</b>; 70 mg, 0.20 mmol) and isobutylamine (25 μL, 0.26 mmol, 1.3 equiv) afforded title compound (<b>26</b>; 74 mg, 92% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.12 (br. s., 2H), 7.99–7.55 (m, 4H), 3.48–2.75 (m, 5H), 1.78 (br. s., 1H), 0.86 (br. s., 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.9, 164.8, 162.5, 160.0, 154.1, 146.3, 144.8, 137.7, 135.0, 135.0, 132.6, 132.5, 129.8, 129.8, 129.4, 129.4, 126.1, 125.5, 121.1, 120.9, 120.6, 120.3, 115.5, 47.3, 47.1, 40.6, 40.2, 39.8, 39.4, 28.5, 21.4, 20.6. EI-HR calcd for C<sub>21</sub>H<sub>20</sub>ClFN<sub>5</sub>O (M<sup>+</sup>) 412.1335, found 412.1334.</div></div><div id="sec4_12_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(<i>tert</i>-Butyl)-1-(2-chloro-5-fluorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>27</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>14</b>; 70 mg, 0.20 mmol) and <i>tert</i>-butylamine (27 μL, 0.26 mmol, 1.3 equiv) afforded the title compound (<b>27</b>; 73 mg, 91% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05 (d, <i>J</i> = 7.9 Hz, 1H), 7.88 (d, <i>J</i> = 7.8 Hz, 1H), 7.73–7.57 (m, 2H), 7.53–7.33 (m, 2H), 5.77 (br. s., 1H), 3.08 (br. s., 3H), 1.40 (br. s., 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.6, 162.4, 159.9, 154.2, 146.2, 144.8, 137.7, 135.9, 131.6, 131.5, 130.21, 130.17, 130.06, 129.2, 129.1, 125.9, 125.1, 120.3, 120.2, 120.0, 119.9, 114.0, 77.3, 77.2, 77.0, 51.9, 28.7, 21.3. EI-HR calcd for C<sub>21</sub>H<sub>20</sub>ClFN<sub>5</sub>O (M<sup>+</sup>) 412.1335, found 412.1332.</div></div><div id="sec4_12_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-((3s,5s,7s)-Adamantan-1-yl)-1-(2-chloro-5-fluorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>28</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>14</b>; 70 mg, 0.20 mmol) and adamantylamine (39 mg, 0.26 mmol, 1.3 equiv) afforded title compound (<b>28</b>; 74 mg, 77% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.32 (s, 1H), 8.13–8.02 (m, 2H), 7.93 (dd, <i>J</i> = 5.0, 8.9 Hz, 1H), 7.85–7.69 (m, 3H), 7.60 (s, 1H), 2.98 (s, 3H), 2.12–2.02 (m, 3H), 1.98 (br. s., 6H), 1.65 (br. s., 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.7, 162.5, 160.0, 153.9, 146.2, 144.8, 137.6, 136.2, 132.6, 132.5, 130.0, 129.9, 129.6, 129.5, 129.4, 126.8, 125.2, 121.1, 120.9, 120.7, 120.5, 115.5, 79.7, 52.2, 41.2, 40.7, 40.3, 39.8, 39.4, 36.5, 29.3, 21.4. EI-HR calcd for C<sub>27</sub>H<sub>26</sub>ClFN<sub>5</sub>O (M<sup>+</sup>) 490.1804, found 490.1802.</div></div><div id="sec4_12_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 1-(2-Chloro-5-methoxyphenyl)-<i>N</i>,4-dimethyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>29</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling reaction with carboxylic acid (<b>15</b>; 100 mg, 0.27 mmol) and methylamine (0.27 mL, 0.54 mmol, 2.0 equiv) afforded methyl amide (<b>29</b>; 33 mg, 31% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.63 (d, <i>J</i> = 4.5 Hz, 1H), 8.14–8.10 (m, 1H), 8.09–8.04 (m, 1H), 7.85 (d, <i>J</i> = 1.5 Hz, 1H), 7.76–7.69 (m, 1H), 7.43–7.36 (m, 2H), 3.85 (s, 3H), 2.97 (s, 3H), 2.75 (d, <i>J</i> = 4.5 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.0, 159.0, 154.1, 147.3, 144.7, 137.7, 134.5, 131.5, 129.7, 128.5, 125.7, 125.6, 125.2, 119.5, 118.3, 115.9, 56.4, 26.9, 21.3. EI-HR calcd for C<sub>19</sub>H<sub>17</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 382.1065, found 382.1065.</div></div><div id="sec4_12_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 1-(2-Chloro-5-methoxyphenyl)-<i>N</i>-isopropyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>30</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>15</b>; 80 mg, 0.22 mmol) and isopropylamine (17 mg, 0.28 mmol, 1.3 equiv) afforded title compound (16 mg, 18% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.33 (d, <i>J</i> = 7.7 Hz, 1H), 8.14–8.08 (m, 2H), 7.79 (s, 1H), 7.72 (d, <i>J</i> = 8.8 Hz, 1H), 7.43–7.41 (m, 1H), 7.40–7.36 (m, 1H), 4.09–3.96 (m, 1H), 3.85 (s, 3H), 2.97 (s, 3H), 1.12 (dd, <i>J</i> = 6.7, 8.2 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.9, 159.0, 154.0, 147.3, 144.7, 137.7, 135.0, 131.4, 129.6, 128.5, 126.0, 125.6, 125.2, 119.5, 118.4, 115.9, 56.4, 41.7, 22.7, 22.7, 21.3. EI-HR calcd for C<sub>21</sub>H<sub>21</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 410.1378, found 410.1377.</div></div><div id="sec4_12_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 1-(2-Chloro-5-methoxyphenyl)-<i>N</i>-isobutyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>31</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>15</b>; 80 mg, 0.22 mmol) and isobutylamine (21 mg, 0.28 mmol, 1.3 equiv) afforded the title compound (<b>31</b>; 35 mg, 38% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.58 (t, <i>J</i> = 5.7 Hz, 1H), 8.17–8.04 (m, 2H), 7.82–7.77 (m, 1H), 7.75–7.65 (m, 1H), 7.41 (d, <i>J</i> = 3.1 Hz, 1H), 7.40–7.35 (m, 1H), 3.84 (s, 3H), 3.11–2.94 (m, 5H), 1.85–1.72 (m, 1H), 0.85 (dd, <i>J</i> = 0.7, 6.7 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.0, 159.0, 154.1, 147.3, 144.8, 137.7, 135.1, 131.4, 129.7, 128.5, 126.0, 125.6, 125.2, 119.5, 118.4, 115.7, 56.4, 47.2, 28.5, 21.3. EI-HR calcd for C<sub>22</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 424.1535, found 424.1534.</div></div><div id="sec4_12_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>-(<i>tert</i>-Butyl)-1-(2-chloro-5-methoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>32</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>15</b>; 80 mg, 0.22 mmol) and <i>tert</i>-butylamine (21 mg, 0.28 mmol, 1.3 equiv) afforded the title compound (<b>32</b>; 30 mg, 33% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.10–8.06 (m, 1H), 8.05–8.00 (m, 1H), 7.78 (s, 1H), 7.75–7.70 (m, 1H), 7.64 (d, <i>J</i> = 1.6 Hz, 1H), 7.44–7.42 (m, 1H), 7.41–7.37 (m, 1H), 3.88–3.83 (m, 3H), 2.97 (s, 3H), 1.30 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.2, 159.0, 153.9, 147.2, 144.7, 137.5, 136.2, 131.4, 129.5, 128.4, 126.7, 125.31, 125.26, 119.6, 118.4, 115.6, 56.4, 51.4, 28.8, 21.3. EI-HR calcd for C<sub>22</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 424.1535, found 424.1534.</div></div><div id="sec4_12_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-((3s,5s,7s)-Adamantan-1-yl)-1-(2-chloro-5-methoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>33</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>15</b>; 80 mg, 0.22 mmol) and adamantylamine (43 mg, 0.28 mmol, 1.3 equiv) afforded the title compound (<b>23</b>; 35 mg, 32% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.09–8.05 (m, 1H), 8.04–7.99 (m, 1H), 7.73 (d, <i>J</i> = 8.8 Hz, 1H), 7.63 (d, <i>J</i> = 1.5 Hz, 2H), 7.45–7.37 (m, 2H), 3.85 (s, 3H), 2.97 (s, 3H), 2.04 (br. s., 3H), 1.99–1.94 (m, 6H), 1.65 (br. s., 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.9, 159.0, 153.9, 147.2, 144.7, 137.5, 136.3, 131.4, 129.5, 128.4, 126.7, 125.32, 125.26, 119.7, 118.4, 115.6, 56.4, 52.1, 41.2, 36.5, 21.3. EI-HR calcd for C<sub>28</sub>H<sub>29</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 502.2004, found 502.2003.</div></div><div id="sec4_12_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 1-(2-Chloro-5-isopropoxyphenyl)-<i>N</i>,4-dimethyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>34</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>16</b>; 80 mg, 0.20 mmol) and 2 M methylamine (131 μL, 0.26 mmol, 1.3 equiv) afforded the title compound (<b>34</b>; 61 mg, 74% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.62 (q, <i>J</i> = 4.5 Hz, 1H) 8.04–8.13 (m, 2 H) 7.85 (d, <i>J</i> = 1.5 Hz, 1 H) 7.69 (d, <i>J</i> = 8.6 Hz, 1 H) 7.34–7.40 (m, 2H) 4.70 (quin, <i>J</i> = 6.0 Hz, 1H) 2.96 (s, 3 H) 2.75 (d, <i>J</i> = 4.4 Hz, 3H) 1.29 (dd, <i>J</i> = 16.6, 6.0 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.0, 157.2, 154.1, 147.3, 144.7, 137.7, 134.6, 131.5, 129.7, 128.5, 125.8, 125.6, 124.8, 121.1, 119. 9, 115.9, 70.8, 26.9, 22.1, 22.0, 21.3. EI-HR calcd for C<sub>21</sub>H<sub>21</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 410.1378, found 410.1375.</div></div><div id="sec4_12_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1-(2-Chloro-5-isopropoxyphenyl)-<i>N</i>-isopropyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>35</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>16</b>; 80 mg, 0.20 mmol) and isopropylamine (23 μL, 0.26 mmol, 1.3 equiv) afforded the title compound (<b>35</b>; 30 mg, 34% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.12 (d, <i>J</i> = 8.4 Hz, 1H) 7.97 (dd, <i>J</i> = 8.4, 1.8 Hz, 1H) 7.74 (d, <i>J</i> = 1.7 Hz, 1H) 7.58 (d, <i>J</i> = 8.7 Hz, 1H) 7.22–7.24 (m, 1H) 7.18–7.22 (m, 1H) 5.52–5.64 (m, 1H) 4.53–4.67 (m, 1H) 4.17–4.29 (m, 1H) 3.13 (s, 3H) 1.41 (d, <i>J</i> = 6.1 Hz, 3H) 1.36 (d, <i>J</i> = 6.1 Hz, 3H) 1.23 (t, <i>J</i> = 6.3 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.6, 157.4, 154.5, 147.5, 144.8, 138.0, 134.7, 130.9, 130.1, 128.3, 126.0, 125.6, 125.4, 120.6, 119.4, 114.3, 71.0, 42.2, 22.8, 21.9, 21.4. EI-HR calcd for C<sub>23</sub>H<sub>25</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 438.1691, found 438.1689.</div></div><div id="sec4_12_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 1-(2-Chloro-5-isopropoxyphenyl)-<i>N</i>-isobutyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>36</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>16</b>; 80 mg, 0.20 mmol) and isobutylamine (26 μL, 0.26 mmol, 1.3 equiv) afforded the title compound (<b>36</b>; 76 mg, 83% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.59 (t, <i>J</i> = 5.7 Hz, 1H) 8.04–8.16 (m, 2 H) 7.80 (d, <i>J</i> = 1.3 Hz, 1H) 7.68 (d, <i>J</i> = 8.8 Hz, 1H) 7.31–7.40 (m, 2H) 4.69 (spt, <i>J</i> = 6.0 Hz, 1H) 2.93–3.11 (m, 5H) 1.71–1.84 (m, 1H) 1.28 (dd, <i>J</i> = 17.6, 6.0 Hz, 6H) 0.85 (dd, <i>J</i> = 6.7, 1.4 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ, 165.0, 157.2, 154.0, 147.3, 144.7, 137.7, 135.1, 131.5, 129.7, 128.5, 126.0, 125.7, 124.8, 121.1, 119.8, 115.8, 70.8, 47.3, 28.5, 22.1, 22.0, 21.4, 20.6. EI-HR calcd for C<sub>24</sub>H<sub>27</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 452.1848, found 452.1847.</div></div><div id="sec4_12_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-(<i>tert</i>-Butyl)-1-(2-chloro-5-isopropoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>37</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>16</b>; 45 mg, 0.11 mmol) and <i>tert</i>-butylamine (15 μL, 0.15 mmol, 1.3 equiv) afforded the title compound (<b>37</b>; 20 mg, 39% yield). <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.10 (d, <i>J</i> = 8.3 Hz, 1H), 7.95 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 7.74–7.58 (m, 2H), 7.37 (s, 2H), 4.72 (dt, <i>J</i> = 12.0, 6.1 Hz, 1H), 3.04 (s, 3H), 1.41–1.34 (m, 15H). <sup>13</sup>C NMR (101 MHz, MeOD-<i>d</i><sub>4</sub>) δ 166.9, 157.6, 153.7, 147.6, 144.6, 137.6, 136.5, 131.0, 129.3, 127.6, 126.1, 125.2, 120.7, 119.0, 115.0, 70.7, 51.4, 27.5, 20.79, 20.76, 19.8. EI-HR calcd for C<sub>24</sub>H<sub>27</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 452.1848, found 452.1847.</div></div><div id="sec4_12_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-((3s,5s,7s)-Adamantan-1-yl)-1-(2-chloro-5-isopropoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>38</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>16</b>; 80 mg, 0.20 mmol) and adamantylamine (40 mg, 0.26 mmol, 1.3 equiv) afforded the title compound (<b>38</b>; 24 mg, 23% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (d, <i>J</i> = 8.4 Hz, 1H) 7.98 (dd, <i>J</i> = 8.4, 1.8 Hz, 1H) 7.61 (d, <i>J</i> = 1.6 Hz, 1H) 7.57 (d, <i>J</i> = 8.8 Hz, 1H) 7.21–7.23 (m, 1H) 7.17–7.21 (m, 1H) 5.42 (s, 1H) 4.54–4.66 (m, 1H) 3.11 (s, 3H) 2.15 (br. s., 3H) 1.99–2.05 (m, 6H) 1.71–1.77 (m, 6H) 1.41 (d, <i>J</i> = 6.0 Hz, 3H) 1.37 (d, <i>J</i> = 6.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.8, 157.4, 154.3, 147.3, 144.9, 137.8, 136.0, 131.0, 130.2, 128.5, 126.6, 125.6, 125.2, 120.6, 119.5, 113.5, 71.1, 52.6, 41.6, 36.3, 29.5, 21.9, 21.4. EI-HR calcd for C<sub>30</sub>H<sub>33</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 530.2317, found 530.2316.</div></div><div id="sec4_12_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 1-(2-Chloro-5-isobutoxyphenyl)-<i>N</i>,4-dimethyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>39</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>17</b>; 80 mg, 0.19 mmol) and 2 M methylamine (127 μL, 0.25 mmol, 1.3 equiv) afforded the title compound (<b>39</b>; 29 mg, 35% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05 (d, <i>J</i> = 8.4 Hz, 1H) 7.93 (d, <i>J</i> = 1.6 Hz, 1H) 7.89 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H) 7.56 (d, <i>J</i> = 8.9 Hz, 1H) 7.15–7.23 (m, 2H) 6.36 (d, <i>J</i> = 4.5 Hz, 1H) 3.70–3.83 (m, 2H) 3.09 (s, 3H) 2.97 (d, <i>J</i> = 4.8 Hz, 3H) 2.01–2.19 (m, 1H) 1.02 (dd, <i>J</i> = 6.7, 0.6 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.0, 158.6, 154.5, 147.8, 144.8, 138.0, 134.2, 130.9, 129.9, 127.9, 125.6, 125.6, 125.2, 119.6, 118.1, 115.4, 75.1, 28.2, 27.1, 21.4, 19.2. EI-HR calcd for C<sub>22</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 424.1535, found 424.1532.</div></div><div id="sec4_12_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 1-(2-Chloro-5-isobutoxyphenyl)-<i>N</i>-isopropyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>40</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>17</b>; 80 mg, 0.19 mmol) and isopropylamine (22 μL, 0.25 mmol, 1.3 equiv) afforded the title compound (<b>40</b>; 30 mg, 34% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.06 (d, <i>J</i> = 8.4 Hz, 1H) 7.94 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H) 7.75 (d, <i>J</i> = 1.6 Hz, 1H) 7.55 (d, <i>J</i> = 8.7 Hz, 1H) 7.22–7.24 (m, 1H) 7.16–7.21 (m, 1H) 5.78 (d, <i>J</i> = 7.7 Hz, 1H) 4.13–4.25 (m, 1H) 3.71–3.82 (m, 2H) 3.09 (s, 3H) 2.04–2.16 (m, 1H) 1.18–1.23 (m, 6H) 1.02 (d, <i>J</i> = 6.7 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.6, 158.6, 154.4, 147.5, 144.8, 137.9, 134.7, 130.8, 130.0, 128.2, 125.9, 125.6, 125.4, 119.6, 118.2, 114.5, 75.1, 42.2, 28.2, 22. 8, 21.4, 19.1. EI-HR calcd for C<sub>24</sub>H<sub>27</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 452.1848, found 452.1847.</div></div><div id="sec4_12_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 1-(2-Chloro-5-isobutoxyphenyl)-<i>N</i>-isobutyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>41</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>17</b>; 80 mg, 0.19 mmol) and isobutylamine (25 μL, 0.25 mmol, 1.3 equiv) afforded the title compound (<b>41</b>; 29 mg, 32% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.09 (d, <i>J</i> = 8.4 Hz, 1H) 7.96 (dd, <i>J</i> = 8.4, 1.8 Hz, 1H) 7.81 (d, <i>J</i> = 1.6 Hz, 1H) 7.54 (d, <i>J</i> = 8.8 Hz, 1H) 7.23 (d, <i>J</i> = 2.9 Hz, 1H) 7.19 (dd, <i>J</i> = 8.9, 3.0 Hz, 1H) 6.03 (t, <i>J</i> = 5.3 Hz, 1H) 3.70–3.83 (m, 2H) 3.17–3.29 (m, 2H) 3.10 (s, 3H) 2.04–2.18 (m, 1H) 1.79–1.87 (m, 1H) 1.03 (d, <i>J</i> = 6.6 Hz, 6H) 0.95 (dd, <i>J</i> = 6.7, 1.8 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.6, 158.6, 154.5, 147.6, 144.8, 138.0, 134.6, 130.8, 130.1, 128.1, 125.9, 125.6, 125.5, 119.6, 118.2, 114.4, 75.1, 47.5, 28.5, 28.2, 21.4, 20.2, 19.2. EI-HR calcd for C<sub>25</sub>H<sub>29</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 466.2004, found 466.2000.</div></div><div id="sec4_12_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-(<i>tert</i>-Butyl)-1-(2-chloro-5-isobutoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>42</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>17</b>; 80 mg, 0.19 mmol) and <i>tert</i>-butylamine (27 μL, 0.25 mmol, 1.3 equiv) afforded the title compound (42; 51 mg, 57% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (d, <i>J</i> = 8.4 Hz, 1H) 7.98 (dd, <i>J</i> = 8.4, 1.8 Hz, 1H) 7.62 (d, <i>J</i> = 1.6 Hz, 1H) 7.56 (d, <i>J</i> = 8.8 Hz, 1H) 7.24–7.26 (m, 1H) 7.19–7.23 (m, 1H) 5.59 (s, 1H) 3.71–3.84 (m, 2H) 3.11 (s, 3H) 2.07–2.19 (m, 1H) 1.41 (s, 9H) 1.04 (d, <i>J</i> = 6.7 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.0, 158.7, 154.3, 147.4, 144.8, 137.8, 135.9, 130.8, 130.2, 128.4, 126.5, 125.7, 125.2, 119.6, 118.2, 113.6, 75.2, 51.9, 28.8, 28.2, 21.4, 19.1. EI-HR calcd for C<sub>25</sub>H<sub>29</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 466.2004, found 466.2004.</div></div><div id="sec4_12_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-((3s,5s,7s)-Adamantan-1-yl)-1-(2-chloro-5-isobutoxyphenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>43</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling reaction with carboxylic acid (<b>17</b>; 80 mg, 0.19 mmol) and adamantylamine (38 mg, 0.25 mmol, 1.3 equiv) afforded the title compound (<b>43</b>; 43 mg, 41% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (d, <i>J</i> = 8.4 Hz, 1H) 7.97 (dd, <i>J</i> = 8.4, 1.8 Hz, 1H) 7.61 (d, <i>J</i> = 1.6 Hz, 1H) 7.57 (d, <i>J</i> = 8.8 Hz, 1H) 7.26–7.23 (m, 1H) 7.23–7.18 (m, 1H) 5.43 (s, 1H) 3.72–3.84 (m, 2H) 3.10 (s, 3H) 2.17–2.12 (m, 3H) 2.12–2.07 (m, 1H) 2.02 (d, <i>J</i> = 2.4 Hz, 6H) 1.76–1.70 (m, 6H) 1.04 (d, <i>J</i> = 6.7 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.8, 158.6, 154.3, 147.3, 144.8, 137.8, 136.0, 130.9, 130.2, 128.4, 126.5, 125.7, 125.2, 119.6, 118.2, 113.6, 75.2, 52.6, 41.6, 36.3, 29.4, 28.2, 21.4, 19.1. EI-HR calcd for C<sub>31</sub>H<sub>35</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 544.2474, found 544.2470.</div></div><div id="sec4_12_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 1-(5-(<i>tert</i>-Butoxy)-2-chlorophenyl)-<i>N</i>,4-dimethyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>44</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>18</b>; 45 mg, 0.11 mmol) and 2.0 M methylamine (70 μL, 0.14 mmol, 1.3 equiv) afforded the title compound (<b>44</b>; 24 mg, 52% yield). <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.10 (d, <i>J</i> = 8.6 Hz, 1H), 7.99 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 7.90 (d, <i>J</i> = 1.7 Hz, 1H), 7.67 (d, <i>J</i> = 8.6 Hz, 1H), 7.45–7.37 (m, 2H), 3.05 (s, 3H), 2.88 (s, 3H), 1.42 (s, 9H). <sup>13</sup>C NMR (101 MHz, MeOD-<i>d</i><sub>4</sub>) δ 166.7, 155.3, 154.0, 147.5, 144.6, 137.8, 134.63, 130.8, 129.4, 128.6, 128.5, 127.2, 127.0, 125.5, 125.3, 115.6, 80.0, 27.9, 25.9, 20.0. EI-HR calcd for C<sub>22</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 424.1535, found 424.1535.</div></div><div id="sec4_12_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 1-(5-(<i>tert</i>-Butoxy)-2-chlorophenyl)-<i>N</i>-isopropyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>45</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>18</b>; 45 mg, 0.11 mmol) and isopropylamine (8.4 mg, 0.14 mmol, 1.3 equiv) afforded the title compound (<b>18</b>; 25 mg, 51% yield). <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.15–8.06 (m, 1H), 8.05–7.97 (m, 1H), 7.82 (d, <i>J</i> = 1.7 Hz, 1H), 7.76–7.63 (m, 1H), 7.50–7.37 (m, 2H), 4.11 (dt, <i>J</i> = 13.2, 6.6 Hz, 1H), 3.04 (s, 3H), 1.42 (s, 9H),1.22 (dd, <i>J</i> = 9.8, 6.6 Hz, 6H). <sup>13</sup>C NMR (101 MHz, MeOD-<i>d</i><sub>4</sub>) δ 165.7, 155.4, 153.9, 147.4, 144.6, 137.8, 135.1, 130.7, 129.3, 128.5, 128.4, 127.3, 127.1, 125.6, 125.4,115.5, 79.9, 42.0, 27.7, 21.2, 21.1,19.8. EI-HR calcd for C<sub>24</sub>H<sub>27</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 452.1848, found 452.1850.</div></div><div id="sec4_12_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 1-(5-(<i>tert</i>-Butoxy)-2-chlorophenyl)-<i>N</i>-isobutyl-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>46</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>18</b>; 10 mg, 0.024 mmol) and isobutylamine (2.3 mg, 0.032 mmol, 1.3 equiv) afforded the title compound (<b>46</b>; 7 mg, 57% yield). <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.13 (d, <i>J</i> = 8.3 Hz, 1H), 8.02 (dd, <i>J</i> = 8.4, 1.8 Hz, 1H), 7.86 (d, <i>J</i> = 1.7 Hz, 1H), 7.71–7.65 (m, 1H), 7.45–7.39 (m, 2H), 3.23–3.06 (m, 2H), 3.05 (s, 3H), 1.87 (dt, <i>J</i> = 13.6, 6.8 Hz, 1H), 1.42 (s, 9H), 0.95 (dd, <i>J</i> = 6.7, 0.9 Hz, 6H). <sup>13</sup>C NMR (101 MHz, MeOD-<i>d</i><sub>4</sub>) δ 166.6, 155.38, 154.0,147.4, 144.6,137.8, 135.1, 130.7, 129.4, 128.5, 128.4, 127.2, 127.1, 125.5, 115.4, 79.9, 28.4, 27.7, 19.8, 19.22, 19.19. EI-HR calcd for C<sub>25</sub>H<sub>29</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 466.2004, found 466.2002.</div></div><div id="sec4_12_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 1-(5-(<i>tert</i>-Butoxy)-2-chlorophenyl)-<i>N</i>-(<i>tert</i>-butyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>47</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>18</b>; 30 mg, 0.073 mmol) and <i>tert</i>-butylamine (10.0 μL, 0.095 mmol, 1.3 equiv) afforded the title compound (<b>47</b>; 16 mg, 47% yield). <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.09 (d, <i>J</i> = 8.3 Hz, 1H), 8.0 (dd, <i>J</i> = 8.3,1.7 Hz, 1H), 7.74–7.63 (m, 3H), 7.48–7.42 (m, 2H), 3.04 (s, 3H), 1.43 (s, 9H) 1.40 (s, 9H). <sup>13</sup>C NMR (101 MHz, MeOD-<i>d</i><sub>4</sub>) δ 166.6, 155.4, 153.8, 147.3, 144.6, 137.6, 136.5, 130.7, 129.3, 128.5, 127.3, 127.1, 126.0, 125.2, 115.0, 79.9, 51.4, 27.7, 27.5, 19.8. EI-HR calcd for C<sub>25</sub>H<sub>29</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 466.2004, found 466.2003.</div></div><div id="sec4_12_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-((3s,5s,7s)-Adamantan-1-yl)-1-(5-(<i>tert</i>-butoxy)-2-chlorophenyl)-4-methyl-[1,2,4]triazolo[4,3-<i>a</i>]quinoxaline-8-carboxamide (<b>48</b>)</h4><div class="NLM_p last">Using the general procedure for amide coupling with carboxylic acid (<b>18</b>; 42 mg, 0.10 mmol) and adamantylamine (20 mg, 0.13 mmol, 1.3 equiv) afforded the title compound (<b>48</b>; 15 mg, 27% yield). <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>) δ ppm 1.43 (s, 9 H) 1.72–1.82 (m, 6 H) 2.03–2.16 (m, 9 H) 3.05 (s, 3 H) 7.38 (s, 0.5 H) 7.39–7.48 (m, 2 H) 7.67 (d, <i>J</i> = 1.7 Hz, 1 H) 7.70 (d, <i>J</i> = 8.8 Hz, 1 H) 7.84–7.87 (m, 0.5 H) 7.95 (dd, <i>J</i> = 8.6, 1.7 Hz, 1 H) 8.10 (d, <i>J</i> = 8.3 Hz, 1 H) <sup>13</sup>C NMR (101 MHz, MeOD-<i>d</i><sub>4</sub>) δ 166.3, 155.3, 153.7, 147.3, 144.6, 137.6, 136.6, 130.8, 129.4, 128.6, 128.4, 127.3, 127.1, 126.1, 125.1, 114.9, 80.0, 52.4, 40.9, 36.1, 29.5, 29.4, 27.9, 20.0. EI-HR calcd for C<sub>31</sub>H<sub>35</sub>ClN<sub>5</sub>O<sub>2</sub> (M<sup>+</sup>) 544.2474, found 544.2470.</div></div></div><div id="sec4_12_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> PDE2 Crystallography</h3><div class="NLM_p last">Crystal structures of PDE2 ligand cocomplexes were determined following previously described methods.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The catalytic domain of PDE2 (residues 579–918) was overexpressed in baculovirus insect cells and purified through affinity and size-exclusion chromatography. Crystals were obtained in the presence of 2 mM 3-isobutyl-1-methylxanthine (IBMX), which was added to the well solution containing 17–19% PEG3350, 0.2 M MgCl<sub>2</sub>, and 0.1 M Tris, pH 8.4. Crystals were then soaked overnight in reservoir buffer containing 1 mM of target compound and flash frozen in liquid nitrogen in the presence of 25% glycerol used as a cryoprotectant. The diffraction data sets were collected at the Advanced Light Source beamline 5.0.1 and processed with DIALS<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> or HKL2000.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The structures were solved by molecular replacement using Phaser,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> model building using COOT,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and refinement using PHENIX.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The structure of compound <b>43</b> shows a highly disordered adamantyl group in the R<sup>1</sup> solvent-exposed pocket, and therefore, the occupancy of the adamantyl group was set to zero during refinement. Data collection and refinement statistics are summarized in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv" class="ext-link">Supporting Information</a>. Figures representing the crystal structures were generated with PyMOL (The PyMOL Molecular Graphics System, Version 1.7.6.4 Schrödinger, LLC).</div></div><div id="sec4_12_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> PDE Inhibition</h3><div class="NLM_p last">PDE2 inhibition IC<sub>50</sub> values were determined by an IMAP assay measuring inhibition of FAM-cAMP hydrolysis by full length PDE2 enzymes. Specifically, in 1536-well white plates, 250 pg per well GST tagged PDE2 was dispensed in 2.5 μL of IMAP assay buffer consisting of 10 mM Tris, pH 7.2, 10 mM MgCl<sub>2</sub>, 1 mM DTT, 0.1% fatty acid free BSA, 10 U/mL calmodulin, and 2.5 mM CaCl<sub>2</sub>. Thirty nanoliters of compound was then added from 1 mM stock in DMSO using the Kalypsys 1536 pintool. Plates were incubated for 5 min at rt before dispensing 1.5 μL of 533 nM FAM-cAMP for a final concentration of 200 nM. The plates were incubated 30 min at rt after a brief centrifugation. The assay was terminated by adding 5 μL of IMAP binding reagent Tb complex to each well, prepared according to manufacturer’s recommendations. Plates were incubated 1 h at rt and read on a Viewlux multimode plate reader (PerkinElmer).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00713" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00713" class="ext-link">10.1021/acs.jmedchem.8b00713</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Activity data for all compounds in the matrix, computational details (lowest energy conformation, tSA calculation, RMSD, and nonadditivity values), and protein crystallographic data statistics tables (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv">jm8b00713_si_001.csv (2.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_002.pdf">jm8b00713_si_002.pdf (432.08 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Crystal structures of PDE2 in complex with <b>19</b>, <b>23</b>, <b>31</b>, <b>39</b>, <b>43</b>, and <b>46</b> have been deposited in the RCSB Protein Data Bank under the accession codes of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7E">6C7E</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7D">6C7D</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7I">6C7I</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7F">6C7F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7G">6C7G</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7J">6C7J</a>, respectively.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b00713" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78235" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78235" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurent Gomez</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9172-7840" title="Orcid link">http://orcid.org/0000-0002-9172-7840</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f4939b99918e98958186919a80c4b49399959d98da979b99"><span class="__cf_email__" data-cfemail="cbaca4a6aeb1a7aabeb9aea5bffb8baca6aaa2a7e5a8a4a6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rui Xu</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William Sinko</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brandon Selfridge</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William Vernier</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kiev Ly</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Truong</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Markus Metz</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tami Marrone</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kristen Sebring</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yingzhou Yan</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brent Appleton</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kathleen Aertgeerts</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark Eben Massari</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Guy Breitenbucher</span> - <span class="hlFld-Affiliation affiliation">Dart Neuroscience LLC, 12278 Scripps Summit Drive, San Diego, California 92131, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4711-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21838" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21838" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Advanced Light Source is supported by the Director, Office of Science, Office of Basic Energy Sciences, of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS USED</td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">QSAR</td><td class="NLM_def"><p class="first last">quantitative structure–activity relationship</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term">tPSA</td><td class="NLM_def"><p class="first last">topological polar surface area</p></td></tr><tr><td class="NLM_term">FEP</td><td class="NLM_def"><p class="first last">free energy perturbation</p></td></tr><tr><td class="NLM_term">PDE2</td><td class="NLM_def"><p class="first last">phosphodiesterase 2</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">cLogP</td><td class="NLM_def"><p class="first last">calculated partition coefficient</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10247" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10247" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 22 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Free, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. W.</span></span> <span> </span><span class="NLM_article-title">A mathematical contribution to structure-activity studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">399</span>, <span class="refDoi"> DOI: 10.1021/jm00334a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00334a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADyaF2cXkt1OltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1964&pages=395-399&author=S.+M.+Freeauthor=J.+W.+Wilson&title=A+mathematical+contribution+to+structure-activity+studies&doi=10.1021%2Fjm00334a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A mathematical contribution to structure-activity studies</span></div><div class="casAuthors">Free, Spencer M., Jr.; Wilson, James W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">395-9</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A math. technique is suggested as a means of describing structure-activity relation of a series of chem. analogs.  The data requirements included specific side chain arrangements and performance characteristics of all analogs tested.  Two examples illustrate the use of the additive math. model where the performance characteristics are measures of biol. activity.  The results rank the structural changes per position by estg. the amt. of biol. response attributed to each change.  The estimates are both pos. and neg.  Several uses for the math. soln. are suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRyTj3xI-aH7Vg90H21EOLACvtfcHk0liQz3__qPFzoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXkt1OltbY%253D&md5=88fabb64d7a5635506a11418a2744bd6</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm00334a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00334a001%26sid%3Dliteratum%253Aachs%26aulast%3DFree%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DJ.%2BW.%26atitle%3DA%2520mathematical%2520contribution%2520to%2520structure-activity%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1964%26volume%3D7%26spage%3D395%26epage%3D399%26doi%3D10.1021%2Fjm00334a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cammarata, A.</span></span> <span> </span><span class="NLM_article-title">Interrelationship of the regression models used for structure-activity analyses</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1021/jm00276a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00276a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1972&pages=573-577&author=A.+Cammarata&title=Interrelationship+of+the+regression+models+used+for+structure-activity+analyses&doi=10.1021%2Fjm00276a001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm00276a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00276a001%26sid%3Dliteratum%253Aachs%26aulast%3DCammarata%26aufirst%3DA.%26atitle%3DInterrelationship%2520of%2520the%2520regression%2520models%2520used%2520for%2520structure-activity%2520analyses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1972%26volume%3D15%26spage%3D573%26epage%3D577%26doi%3D10.1021%2Fjm00276a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jencks, W. P.</span></span> <span> </span><span class="NLM_article-title">On the attribution and additivity of binding energies</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">4046</span>– <span class="NLM_lpage">4050</span>, <span class="refDoi"> DOI: 10.1073/pnas.78.7.4046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1073%2Fpnas.78.7.4046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=16593049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1981&pages=4046-4050&author=W.+P.+Jencks&title=On+the+attribution+and+additivity+of+binding+energies&doi=10.1073%2Fpnas.78.7.4046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">On the attribution and additivity of binding energies</span></div><div class="casAuthors">Jencks, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4046-50</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">It can be useful to describe the Gibbs free energy changes for the binding to a protein of a mol., A-B, and of its component parts, A and B, in terms of the intrinsic binding energies of A and B, ΔGiA and ΔGiB, and a connection Gibbs energy, ΔGs, that is derived largely from changes in translational and rotational entropy.  The relation between ΔGsand ΔGiA and ΔGiB is discussed.  This empirical approach avoids the difficult of interpreting obsd. ΔH and TΔS values for aq. solns.  The ΔGi and ΔGsterms can be large for binding to enzymes and other proteins.  The use and interpretation of these binding energies are discussed, esp. with respect to ATP interaction with myosin ATPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2X3Thp5VESLVg90H21EOLACvtfcHk0lhY8TL136GuTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D&md5=1cdbbb8b6e3f9d3ff0ba2c662c772375</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.78.7.4046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.78.7.4046%26sid%3Dliteratum%253Aachs%26aulast%3DJencks%26aufirst%3DW.%2BP.%26atitle%3DOn%2520the%2520attribution%2520and%2520additivity%2520of%2520binding%2520energies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1981%26volume%3D78%26spage%3D4046%26epage%3D4050%26doi%3D10.1073%2Fpnas.78.7.4046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillet, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span> <span> </span><span class="NLM_article-title">Assessment of additive/nonadditive effects in structure-activity relationships: implications for iterative drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">7552</span>– <span class="NLM_lpage">7562</span>, <span class="refDoi"> DOI: 10.1021/jm801070q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801070q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2jsb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7552-7562&author=Y.+Patelauthor=V.+J.+Gilletauthor=T.+Howeauthor=J.+Pastorauthor=J.+Oyarzabalauthor=P.+Willett&title=Assessment+of+additive%2Fnonadditive+effects+in+structure-activity+relationships%3A+implications+for+iterative+drug+design&doi=10.1021%2Fjm801070q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Additive/Nonadditive Effects in Structure-Activity Relationships: Implications for Iterative Drug Design</span></div><div class="casAuthors">Patel, Yogendra; Gillet, Valerie J.; Howe, Trevor; Pastor, Joaquin; Oyarzabal, Julen; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7552-7562</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free-Wilson (FW) anal. is common practice in medicinal chem. and is based on the assumption that the contributions to activity made by substituents at different substitution positions are additive.  We analyze eight near complete combinatorial libraries assayed on several different biol. response(s) (GPCR, ion channel, kinase and P 450 targets) and show that only half-exhibit clear additive behavior, which leads us to question the concept of additivity that is widely taken for granted in drug discovery.  Next, we report a series of retrospective expts. in which subsets are extd. from the libraries for FW anal. to det. the min. attributes (size, distribution of substituents, and activity range) necessary to reach the same conclusion about additive/nonadditive effects.  These attributes can provide guidelines on when it is appropriate to apply FW anal. as well as for library design, and they also have important implications for further steps in iterative drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoenaUR82iX37Vg90H21EOLACvtfcHk0lgYsg-C-R3Zww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtl2jsb7L&md5=9c7d2af2b19b0cb52fb2a7edaefb5a67</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm801070q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801070q%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DY.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DPastor%26aufirst%3DJ.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DAssessment%2520of%2520additive%252Fnonadditive%2520effects%2520in%2520structure-activity%2520relationships%253A%2520implications%2520for%2520iterative%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7552%26epage%3D7562%26doi%3D10.1021%2Fjm801070q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuning, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wesler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span> <span> </span><span class="NLM_article-title">Accurate and reliable prediction of relative ligand binding potency in prospective drug discovery by way of a modern free-energy calculation protocol and force field</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">2695</span>– <span class="NLM_lpage">2703</span>, <span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beuningauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Weslerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+reliable+prediction+of+relative+ligand+binding+potency+in+prospective+drug+discovery+by+way+of+a+modern+free-energy+calculation+protocol+and+force+field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0lgYsg-C-R3Zww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuning%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWesler%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520reliable%2520prediction%2520of%2520relative%2520ligand%2520binding%2520potency%2520in%2520prospective%2520drug%2520discovery%2520by%2520way%2520of%2520a%2520modern%2520free-energy%2520calculation%2520protocol%2520and%2520force%2520field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span>Drug Design
Data Resource. <a href="https://drugdesigndata.org/" class="extLink">https://drugdesigndata.org/</a> (accessed on June 8th <span class="NLM_year">2017</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Drug+Design%0AData+Resource.+https%3A%2F%2Fdrugdesigndata.org%2F+%28accessed+on+June+8th+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">Predicting binding free energies: frontiers and benchmarks</span>. <i>Annu. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1146/annurev-biophys-070816-033654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1146%2Fannurev-biophys-070816-033654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=28399632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvFClsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2017&pages=531-58&author=D.+Mobleyauthor=M.+K.+Gilson&title=Predicting+binding+free+energies%3A+frontiers+and+benchmarks&doi=10.1146%2Fannurev-biophys-070816-033654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Binding Free Energies: Frontiers and Benchmarks</span></div><div class="casAuthors">Mobley, David L.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">531-558</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Binding free energy calcns. based on mol. simulations provide predicted affinities for biomol. complexes.  These calcns. begin with a detailed description of a system, including its chem. compn. and the interactions among its components.  Simulations of the system are then used to compute thermodn. information, such as binding affinities.  Because of their promise for guiding mol. design, these calcns. have recently begun to see widespread applications in early-stage drug discovery.  However, many hurdles remain in making them a robust and reliable tool.  In this review, we highlight key challenges of these calcns., discuss some examples of these challenges, and call for the designation of std. community benchmark test systems that will help the research community generate and evaluate progress.  In our view, progress will require careful assessment and evaluation of new methods, force fields, and modeling innovations on well-characterized benchmark systems, and we lay out our vision for how this can be achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJJQBeP6sFOrVg90H21EOLACvtfcHk0lgL8gSQZJfYjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvFClsr0%253D&md5=38af2949ba205daaf9899b87c4e1a2bd</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-070816-033654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-070816-033654%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DPredicting%2520binding%2520free%2520energies%253A%2520frontiers%2520and%2520benchmarks%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2017%26volume%3D46%26spage%3D531%26epage%3D58%26doi%3D10.1146%2Fannurev-biophys-070816-033654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit7b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Advances in activity cliff research</span>. <i>Mol. Inf.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1002/minf.201600023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1002%2Fminf.201600023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=27492084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC28XltVOiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=181-191&author=D.+Dimovaauthor=J.+Bajorath&title=Advances+in+activity+cliff+research&doi=10.1002%2Fminf.201600023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in Activity Cliff Research</span></div><div class="casAuthors">Dimova, Dilyana; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">181-191</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Activity cliffs, i.e. similar compds. with large potency differences, are of interest from a chem. and informatics viewpoint; as a source of structure-activity relationship information, for compd. optimization, and activity prediction.  Herein, recent highlights of activity cliff research are discussed including studies that have further extended our understanding of activity cliffs, yielded unprecedented insights, or paved the way for practical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprlJLFAKwUELVg90H21EOLACvtfcHk0lgL8gSQZJfYjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltVOiu78%253D&md5=09066b5cbe326e9061964db60502ef5f</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1002%2Fminf.201600023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201600023%26sid%3Dliteratum%253Aachs%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DAdvances%2520in%2520activity%2520cliff%2520research%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D181%26epage%3D191%26doi%3D10.1002%2Fminf.201600023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McClure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankley, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">Pyrazole CCK(1) receptor antagonists. Part 1: solution-phase library synthesis and determination of Free-Wilson additivity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2005.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.bmcl.2005.09.048" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=72-76&author=K.+McClureauthor=M.+Hackauthor=L.+Huangauthor=C.+Sehonauthor=M.+Mortonauthor=T.+Barrettauthor=N.+Shankleyauthor=J.+G.+Breitenbucher&title=Pyrazole+CCK%281%29+receptor+antagonists.+Part+1%3A+solution-phase+library+synthesis+and+determination+of+Free-Wilson+additivity&doi=10.1016%2Fj.bmcl.2005.09.048"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.09.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DMcClure%26aufirst%3DK.%26aulast%3DHack%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DSehon%26aufirst%3DC.%26aulast%3DMorton%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DT.%26aulast%3DShankley%26aufirst%3DN.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DPyrazole%2520CCK%25281%2529%2520receptor%2520antagonists.%2520Part%25201%253A%2520solution-phase%2520library%2520synthesis%2520and%2520determination%2520of%2520Free-Wilson%2520additivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D72%26epage%3D76%26doi%3D10.1016%2Fj.bmcl.2005.09.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankley, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">Pyrazole CCK(1) receptor antagonists. Part 2: SAR studies by solid-phase library synthesis and determination of Free-Wilson additivity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2005.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.bmcl.2005.09.041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=77-80&author=C.+Sehonauthor=K.+McClureauthor=M.+Hackauthor=M.+Mortonauthor=L.+Gomezauthor=L.+Liauthor=T.+D.+Barrettauthor=N.+Shankleyauthor=J.+G.+Breitenbucher&title=Pyrazole+CCK%281%29+receptor+antagonists.+Part+2%3A+SAR+studies+by+solid-phase+library+synthesis+and+determination+of+Free-Wilson+additivity&doi=10.1016%2Fj.bmcl.2005.09.041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DSehon%26aufirst%3DC.%26aulast%3DMcClure%26aufirst%3DK.%26aulast%3DHack%26aufirst%3DM.%26aulast%3DMorton%26aufirst%3DM.%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DBarrett%26aufirst%3DT.%2BD.%26aulast%3DShankley%26aufirst%3DN.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DPyrazole%2520CCK%25281%2529%2520receptor%2520antagonists.%2520Part%25202%253A%2520SAR%2520studies%2520by%2520solid-phase%2520library%2520synthesis%2520and%2520determination%2520of%2520Free-Wilson%2520additivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D77%26epage%3D80%26doi%3D10.1016%2Fj.bmcl.2005.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Non-additivity of functional group contributions in protein-ligand binding: a comprehensive study by crystallography and isothermal titration calorimetry</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>397</i></span>,  <span class="NLM_fpage">1042</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.jmb.2010.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=20156458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=397&publication_year=2010&pages=1042-1054&author=B.+Baumauthor=L.+Muleyauthor=M.+Smolinskiauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&title=Non-additivity+of+functional+group+contributions+in+protein-ligand+binding%3A+a+comprehensive+study+by+crystallography+and+isothermal+titration+calorimetry&doi=10.1016%2Fj.jmb.2010.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Non-additivity of Functional Group Contributions in Protein-Ligand Binding: A Comprehensive Study by Crystallography and Isothermal Titration Calorimetry</span></div><div class="casAuthors">Baum, Bernhard; Muley, Laveena; Smolinski, Michael; Heine, Andreas; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">397</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1042-1054</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Additivity of functional group contributions to protein-ligand binding is a very popular concept in medicinal chem. as the basis of rational design and optimized lead structures.  Most of the currently applied scoring functions for docking build on such additivity models.  Even though the limitation of this concept is well known, case studies examg. in detail why additivity fails at the mol. level are still very scarce.  The present study shows, by use of crystal structure anal. and isothermal titrn. calorimetry for a congeneric series of thrombin inhibitors, that extensive cooperative effects between hydrophobic contacts and hydrogen bond formation are intimately coupled via dynamic properties of the formed complexes.  The formation of optimal lipophilic contacts with the surface of the thrombin S3 pocket and the full desolvation of this pocket can conflict with the formation of an optimal hydrogen bond between ligand and protein.  The mutual contributions of the competing interactions depend on the size of the ligand hydrophobic substituent and influence the residual mobility of ligand portions at the binding site.  Anal. of the individual crystal structures and factorizing the free energy into enthalpy and entropy demonstrates that binding affinity of the ligands results from a mixt. of enthalpic contributions from hydrogen bonding and hydrophobic contacts, and entropic considerations involving an increasing loss of residual mobility of the bound ligands.  This complex picture of mutually competing and partially compensating enthalpic and entropic effects dets. the non-additivity of free energy contributions to ligand binding at the mol. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGousWs7tLpU3LVg90H21EOLACvtfcHk0liXbVFvY2GF_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKmurg%253D&md5=2f61dde175a6ee27551179644eee3f00</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DMuley%26aufirst%3DL.%26aulast%3DSmolinski%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DNon-additivity%2520of%2520functional%2520group%2520contributions%2520in%2520protein-ligand%2520binding%253A%2520a%2520comprehensive%2520study%2520by%2520crystallography%2520and%2520isothermal%2520titration%2520calorimetry%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D397%26spage%3D1042%26epage%3D1054%26doi%3D10.1016%2Fj.jmb.2010.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schauperl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldner, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podewitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Binding pose flip explained via enthalpic and entropic contributions</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">345</span>– <span class="NLM_lpage">354</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosF2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=345-354&author=M.+Schauperlauthor=P.+Czodrowskiauthor=E.+J.+Fuchsauthor=R.+G.+Huberauthor=B.+J.+Waldnerauthor=M.+Podewitzauthor=C.+Kramerauthor=K.+R.+Liedl&title=Binding+pose+flip+explained+via+enthalpic+and+entropic+contributions&doi=10.1021%2Facs.jcim.6b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Pose Flip Explained via Enthalpic and Entropic Contributions</span></div><div class="casAuthors">Schauperl, Michael; Czodrowski, Paul; Fuchs, Julian E.; Huber, Roland G.; Waldner, Birgit J.; Podewitz, Maren; Kramer, Christian; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">345-354</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The anomalous binding modes of 5 highly similar fragments of angiopoietin receptor TIE2 inhibitors, showing 3 distinct binding poses were investigated.  The authors report a quant. rationalization for the changes in binding pose based on mol. dynamics (MD) simulations.  The authors investigated 5 fragments in complex with the transforming growth factor-β receptor type 1 kinase domain.  Analyses of these MD simulations using Grid Inhomogeneous Solvation Theory (GIST), pKa calcns., and a tool to investigate enthalpic differences upon binding unraveled the various thermodn. contributions to the different binding modes.  While one binding mode flip could be rationalized by steric repulsion, the 2nd binding pose flip revealed a different protonation state for one of the ligands, leading to different enthalpic and entropic contributions to the binding free energy.  While one binding pose was stabilized by the displacement of entropically unfavored water mols. (binding pose detd. by solvation entropy), ligands in the other binding pose were stabilized by strong enthalpic interactions, overcompensating the unfavorable water entropy in this pose (binding pose detd. by enthalpic interactions).  This anal. elucidated unprecedented details detg. the flipping of the binding modes, which can elegantly explain the exptl. findings for this system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyksMbMzYXP7Vg90H21EOLACvtfcHk0lgltViCdR48Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosF2huw%253D%253D&md5=9fa0737411901c4d27ae9954dca37ecf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00483%26sid%3Dliteratum%253Aachs%26aulast%3DSchauperl%26aufirst%3DM.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DFuchs%26aufirst%3DE.%2BJ.%26aulast%3DHuber%26aufirst%3DR.%2BG.%26aulast%3DWaldner%26aufirst%3DB.%2BJ.%26aulast%3DPodewitz%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DBinding%2520pose%2520flip%2520explained%2520via%2520enthalpic%2520and%2520entropic%2520contributions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D345%26epage%3D354%26doi%3D10.1021%2Facs.jcim.6b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Charges shift protonation: neutron diffraction reveals that aniline and 2-aminopyridine become protonated upon binding to trypsin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4887</span>– <span class="NLM_lpage">4890</span>, <span class="refDoi"> DOI: 10.1002/anie.201701038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1002%2Fanie.201701038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvF2ltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=4887-4890&author=J.+Schiebelauthor=R.+Gaspariauthor=A.+Sandnerauthor=K.+Ngoauthor=H.+J.+Gerberauthor=A.+Cavalliauthor=A.+Ostermannauthor=A.+Heineauthor=G.+Klebe&title=Charges+shift+protonation%3A+neutron+diffraction+reveals+that+aniline+and+2-aminopyridine+become+protonated+upon+binding+to+trypsin&doi=10.1002%2Fanie.201701038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Charges shift protonation: Neutron diffraction reveals that aniline and 2-aminopyridine become protonated upon binding to trypsin</span></div><div class="casAuthors">Schiebel, Johannes; Gaspari, Roberto; Sandner, Anna; Ngo, Khang; Gerber, Hans-Dieter; Cavalli, Andrea; Ostermann, Andreas; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4887-4890</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">H atoms play a key role in protein-ligand recognition.  They det. the quality of established H-bonding networks and define the protonation of bound ligands.  Structural visualization of H atoms by x-ray crystallog. is rarely possible.  Here, we used neutron diffraction to det. the positions of the H atoms in the ligands, aniline and 2-aminopyridine, bound to the archetypical serine protease, trypsin.  The resulting structures showed the best resoln. so far achieved for proteins larger than 100 residues and allowed an accurate description of the protonation states and interactions with nearby water mols.  Despite its low pKa of 4.6 and a large distance of 3.6 Å to the charged Asp-189 residue at the bottom of the trypsin S1 pocket, the amino group of aniline became protonated, whereas in 2-aminopyridine, the pyridine N atom picked up the proton although its NH2 group was 1.6 Å closer to Asp-189.  Therefore, apart from charge-charge distances, tautomer stability was decisive for the resulting binding poses, an aspect that is pivotal for predicting correct binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg449p0i3o-7Vg90H21EOLACvtfcHk0lgltViCdR48Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvF2ltr8%253D&md5=a48df0f5959e03a0fdfc528de58bbb6f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201701038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201701038%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DSandner%26aufirst%3DA.%26aulast%3DNgo%26aufirst%3DK.%26aulast%3DGerber%26aufirst%3DH.%2BJ.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DOstermann%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DCharges%2520shift%2520protonation%253A%2520neutron%2520diffraction%2520reveals%2520that%2520aniline%2520and%25202-aminopyridine%2520become%2520protonated%2520upon%2520binding%2520to%2520trypsin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D4887%26epage%3D4890%26doi%3D10.1002%2Fanie.201701038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Strong nonadditivity as a key structure-activity relationship feature: distinguishing structural changes from assay artifacts</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">483</span>– <span class="NLM_lpage">494</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFamtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=483-494&author=C.+Kramerauthor=J.+E.+Fuchsauthor=K.+R.+Liedl&title=Strong+nonadditivity+as+a+key+structure-activity+relationship+feature%3A+distinguishing+structural+changes+from+assay+artifacts&doi=10.1021%2Facs.jcim.5b00018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Strong Nonadditivity as a Key Structure-Activity Relationship Feature: Distinguishing Structural Changes from Assay Artifacts</span></div><div class="casAuthors">Kramer, Christian; Fuchs, Julian E.; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">483-494</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nonadditivity in protein-ligand affinity data represents highly instructive structure-activity relation (SAR) features that indicate structural changes and have the potential to guide rational drug design.  At the same time, nonadditivity is a challenge for both basic SAR anal. as well as many ligand-based data anal. techniques such as Free-Wilson Anal. and Matched Mol. Pair anal., since linear substituent contribution models inherently assume additivity and thus do not work in such cases.  While structural causes for nonadditivity have been analyzed anecdotally, no systematic approaches to interpret and use nonadditivity prospectively have been developed yet.  In this contribution, the authors lay the statistical framework for systematic anal. of nonadditivity in a SAR series.  First, the authors develop a general metric to quantify nonadditivity.  Then, the authors demonstrate the non-negligible impact of exptl. uncertainty that creates apparent nonadditivity, and the authors introduce techniques to handle exptl. uncertainty.  Finally, the authors analyze public SAR data sets for strong nonadditivity and use recourse to the original publications and available x-ray structures to find structural explanations for the nonadditivity obsd.  The authors find that all cases of strong nonadditivity (ΔΔpKi and ΔΔpIC50 > 2.0 log units) with sufficient structural information to generate reasonable hypothesis involve changes in binding mode.  With the appropriate statistical basis, nonadditivity anal. offers a variety of new attempts for various areas in computer-aided drug design, including the validation of scoring functions and free energy perturbation approaches, binding pocket classification, and novel features in SAR anal. tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAdEaoXXhBu7Vg90H21EOLACvtfcHk0lhbqrlNa9r55Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFamtrw%253D&md5=4f17f57045ef0d7bed85a511b104bc6a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00018%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DStrong%2520nonadditivity%2520as%2520a%2520key%2520structure-activity%2520relationship%2520feature%253A%2520distinguishing%2520structural%2520changes%2520from%2520assay%2520artifacts%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D483%26epage%3D494%26doi%3D10.1021%2Facs.jcim.5b00018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andres-Gil, J. I.</span>; <span class="NLM_string-name">Rombouts, F. J. R.</span>; <span class="NLM_string-name">Trabanco-Suarez, A. A.</span>; <span class="NLM_string-name">Vanhoof, G. C. P.</span>; <span class="NLM_string-name">Buijnsters, A.</span>; <span class="NLM_string-name">Guillemont, J. E. G.</span>; <span class="NLM_string-name">Bormans, G. M. R.</span>; <span class="NLM_string-name">Celen, S. J. L.</span>; <span class="NLM_string-name">Vliegen, M.</span></span> <span> </span><span class="NLM_article-title">Preparation of (Un)labeled 1-aryl-4-methyl-[1,2,4]triazolo[4,3-a]quinoxaline Derivatives as PDE2 and PDE10 Inhibitors for Therapy and Imaging</span>. <span class="NLM_patent">WO2013000924</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+I.+Andres-Gil&author=F.+J.+R.+Rombouts&author=A.+A.+Trabanco-Suarez&author=G.+C.+P.+Vanhoof&author=A.+Buijnsters&author=J.+E.+G.+Guillemont&author=G.+M.+R.+Bormans&author=S.+J.+L.+Celen&author=M.+Vliegen&title=Preparation+of+%28Un%29labeled+1-aryl-4-methyl-%5B1%2C2%2C4%5Dtriazolo%5B4%2C3-a%5Dquinoxaline+Derivatives+as+PDE2+and+PDE10+Inhibitors+for+Therapy+and+Imaging"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAndres-Gil%26aufirst%3DJ.%2BI.%26atitle%3DPreparation%2520of%2520%2528Un%2529labeled%25201-aryl-4-methyl-%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-a%255Dquinoxaline%2520Derivatives%2520as%2520PDE2%2520and%2520PDE10%2520Inhibitors%2520for%2520Therapy%2520and%2520Imaging%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span> <span> </span><span class="NLM_article-title">X-ray crystal structure of phosphodiesterase 2 in complex with a highly selective, nanomolar inhibitor reveals a binding-induced pocket important for selectivity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">11708</span>– <span class="NLM_lpage">11711</span>, <span class="refDoi"> DOI: 10.1021/ja404449g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja404449g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=11708-11711&author=J.+Zhuauthor=Q.+Yangauthor=D.+Daiauthor=Q.+Huang&title=X-ray+crystal+structure+of+phosphodiesterase+2+in+complex+with+a+highly+selective%2C+nanomolar+inhibitor+reveals+a+binding-induced+pocket+important+for+selectivity&doi=10.1021%2Fja404449g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1021%2Fja404449g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja404449g%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DDai%26aufirst%3DD.%26aulast%3DHuang%26aufirst%3DQ.%26atitle%3DX-ray%2520crystal%2520structure%2520of%2520phosphodiesterase%25202%2520in%2520complex%2520with%2520a%2520highly%2520selective%252C%2520nanomolar%2520inhibitor%2520reveals%2520a%2520binding-induced%2520pocket%2520important%2520for%2520selectivity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D11708%26epage%3D11711%26doi%3D10.1021%2Fja404449g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keränen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Predicting binding free energies of PDE2 inhibitors. The difficulties of protein conformation</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4883</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-23039-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1038%2Fs41598-018-23039-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=29559702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A280%3ADC%252BC1MnjtFKlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=4883&author=L.+P%C3%A9rez-Benitoauthor=H.+Ker%C3%A4nenauthor=H.+van+Vlijmenauthor=G.+Tresadern&title=Predicting+binding+free+energies+of+PDE2+inhibitors.+The+difficulties+of+protein+conformation&doi=10.1038%2Fs41598-018-23039-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Binding Free Energies of PDE2 Inhibitors. The Difficulties of Protein Conformation</span></div><div class="casAuthors">Perez-Benito Laura; Keranen Henrik; van Vlijmen Herman; Tresadern Gary; Keranen Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4883</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A congeneric series of 21 phosphodiesterase 2 (PDE2) inhibitors are reported.  Crystal structures show how the molecules can occupy a 'top-pocket' of the active site.  Molecules with small substituents do not enter the pocket, a critical leucine (Leu770) is closed and water molecules are present.  Large substituents enter the pocket, opening the Leu770 conformation and displacing the waters.  We also report an X-ray structure revealing a new conformation of the PDE2 active site domain.  The relative binding affinities of these compounds were studied with free energy perturbation (FEP) methods and it represents an attractive real-world test case.  In general, the calculations could predict the energy of small-to-small, or large-to-large molecule perturbations.  However, accurately capturing the transition from small-to-large proved challenging.  Only when using alternative protein conformations did results improve.  The new X-ray structure, along with a modelled dimer, conferred stability to the catalytic domain during the FEP molecular dynamics (MD) simulations, increasing the convergence and thereby improving the prediction of ΔΔG of binding for some small-to-large transitions.  In summary, we found the most significant improvement in results when using different protein structures, and this data set is useful for future free energy validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMdLif4rzX6u_Q9URG6Dx8fW6udTcc2eacVlx23-w5Srntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnjtFKlsA%253D%253D&md5=44c26c5bd2fe3e35bba344abb8262eeb</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-23039-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-23039-5%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DKer%25C3%25A4nen%26aufirst%3DH.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DPredicting%2520binding%2520free%2520energies%2520of%2520PDE2%2520inhibitors.%2520The%2520difficulties%2520of%2520protein%2520conformation%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D4883%26doi%3D10.1038%2Fs41598-018-23039-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Najmanovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuttner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, M.</span></span> <span> </span><span class="NLM_article-title">Side-chain flexibility in proteins upon ligand binding</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(20000515)39:3<261::AID-PROT90>3.0.CO;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1002%2F%28SICI%291097-0134%2820000515%2939%3A3%3C261%3A%3AAID-PROT90%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10737948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BD3cXislGqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=261-268&author=R.+Najmanovichauthor=J.+Kuttnerauthor=V.+Sobolevauthor=M.+Edelman&title=Side-chain+flexibility+in+proteins+upon+ligand+binding&doi=10.1002%2F%28SICI%291097-0134%2820000515%2939%3A3%3C261%3A%3AAID-PROT90%3E3.0.CO%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16aR"><div class="casContent"><span class="casTitleNuber">16a</span><div class="casTitle"><span class="NLM_cas:atitle">Side-chain flexibility in proteins upon ligand binding</span></div><div class="casAuthors">Najmanovich, Rafael; Kuttner, Josef; Sobolev, Vladimir; Edelman, Marvin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-268</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Ligand binding may involve a wide range of structural changes in the receptor protein, from hinge movement of entire domains to small side-chain rearrangements in the binding pocket residues.  The anal. of side chain flexibility gives insights valuable to improve docking algorithms and can provide an index of amino-acid side-chain flexibility potentially useful in mol. biol. and protein engineering studies.  In this study we analyzed side-chain rearrangements upon ligand binding.  We constructed two non-redundant databases (980 and 353 entries) of "paired" protein structures in complexed (holo-protein) and uncomplexed (apo-protein) forms from the PDB macromol. structural database.  The no. and identity of binding pocket residues that undergo side-chain conformational changes were detd.  We show that, in general, only a small no. of residues in the pocket undergo such changes (e.g., ∼85% of cases show changes in three residues or less).  The flexibility scale has the following order: Lys > Arg, Gln, Met > Glu, Ile, Leu > Asn, Thr, Val, Tyr, Ser, His, Asp > Cys, Trp, Phe; thus, Lys side chains in binding pockets flex 25 times more often then do the Phe side chains.  Normalizing for the no. of flexible dihedral bonds in each amino acid attenuates the scale somewhat, however, the clear trend of large, polar amino acids being more flexible in the pocket than arom. ones remains.  We found no correlation between backbone movement of a residue upon ligand binding and the flexibility of its side chain.  These results are relevant to 1: redn. of search space in docking algorithms by inclusion of side-chain flexibility for a limited no. of binding pocket residues; and 2: utilization of the amino acid flexibility scale in protein engineering studies to alter the flexibility of binding pockets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQBLfK6Ht7TLVg90H21EOLACvtfcHk0lhhk03v8oARaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXislGqtb8%253D&md5=d78e02ffb901136c1438429365ed5ba4</span></div><a href="/servlet/linkout?suffix=cit16a&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2820000515%2939%3A3%3C261%3A%3AAID-PROT90%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252820000515%252939%253A3%253C261%253A%253AAID-PROT90%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DNajmanovich%26aufirst%3DR.%26aulast%3DKuttner%26aufirst%3DJ.%26aulast%3DSobolev%26aufirst%3DV.%26aulast%3DEdelman%26aufirst%3DM.%26atitle%3DSide-chain%2520flexibility%2520in%2520proteins%2520upon%2520ligand%2520binding%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2000%26volume%3D39%26spage%3D261%26epage%3D268%26doi%3D10.1002%2F%28SICI%291097-0134%2820000515%2939%3A3%3C261%3A%3AAID-PROT90%3E3.0.CO%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit16b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Teague, S. J.</span></span> <span> </span><span class="NLM_article-title">Implications of protein flexibility for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nrd1129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1038%2Fnrd1129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=12838268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVaktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=527-541&author=S.+J.+Teague&title=Implications+of+protein+flexibility+for+drug+discovery&doi=10.1038%2Fnrd1129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16bR"><div class="casContent"><span class="casTitleNuber">16b</span><div class="casTitle"><span class="NLM_cas:atitle">Implications of protein flexibility for drug discovery</span></div><div class="casAuthors">Teague, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">527-541</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins are in const. motion between different conformational states with similar energies.  This has often been ignored in drug design.  However, protein flexibility is fundamental to understanding the ways in which drugs exert biol. effects, their binding-site location, binding orientation, binding kinetics, metab. and transport.  Protein flexibility allows increased affinity to be achieved between a drug and its target.  This is crucial, because the lipophilicity and no. of polar interactions allowed for an oral drug is limited by absorption, distribution, metab. and toxicol. considerations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8-I4F-CgSbVg90H21EOLACvtfcHk0liwl4tozbvbTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVaktrs%253D&md5=850b8838a78c1e00493da3089a0c62e9</span></div><a href="/servlet/linkout?suffix=cit16b&amp;dbid=16384&amp;doi=10.1038%2Fnrd1129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1129%26sid%3Dliteratum%253Aachs%26aulast%3DTeague%26aufirst%3DS.%2BJ.%26atitle%3DImplications%2520of%2520protein%2520flexibility%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D527%26epage%3D541%26doi%3D10.1038%2Fnrd1129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dogan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, P.</span></span> <span> </span><span class="NLM_article-title">Distinguishing induced fit from conformational selection</span>. <i>Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>189</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.bpc.2014.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.bpc.2014.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=24747333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvVWltb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2014&pages=33-39&author=S.+Gianniauthor=J.+Doganauthor=P.+Jemth&title=Distinguishing+induced+fit+from+conformational+selection&doi=10.1016%2Fj.bpc.2014.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Distinguishing induced fit from conformational selection</span></div><div class="casAuthors">Gianni, Stefano; Dogan, Jakob; Jemth, Per</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33-39</span>CODEN:
                <span class="NLM_cas:coden">BICIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0301-4622</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The interactions between proteins and ligands often involve a conformational change in the protein.  This conformational change can occur before (conformational selection) or after (induced fit) the assocn. with ligand.  It is often very difficult to distinguish induced fit from conformational selection when hyperbolic binding kinetics are obsd.  In light of a recent paper in this journal (Vogt et al., Biophys. Chem., 186, 2014, 13-21) and the current interest in binding mechanisms emerging from obsd. sampling of distinct conformations in protein domains, as well as from the field of intrinsically disordered proteins, we here describe a kinetic method that, at least in some cases, unequivocally distinguishes induced fit from conformational selection.  The method relies on measuring the obsd. rate const. λ for binding and varying both the protein and the ligand in sep. expts.  Whereas induced fit always yields a hyperbolic dependence of increasing λ values, the conformational selection mechanism gives rise to distinct kinetics when the ligand and protein (displaying the conformational change) concn. is varied in sep. expts.  We provide examples from the literature and discuss the limitations of the approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBGtc2RoRf-7Vg90H21EOLACvtfcHk0liwl4tozbvbTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmvVWltb0%253D&md5=ecbe065454d8d9315638b5960b800c38</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1016%2Fj.bpc.2014.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpc.2014.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DGianni%26aufirst%3DS.%26aulast%3DDogan%26aufirst%3DJ.%26aulast%3DJemth%26aufirst%3DP.%26atitle%3DDistinguishing%2520induced%2520fit%2520from%2520conformational%2520selection%26jtitle%3DBiophys.%2520Chem.%26date%3D2014%26volume%3D189%26spage%3D33%26epage%3D39%26doi%3D10.1016%2Fj.bpc.2014.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit17b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Cera, E.</span></span> <span> </span><span class="NLM_article-title">Essential role of conformational selection in ligand binding</span>. <i>Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>186</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1016/j.bpc.2013.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.bpc.2013.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=24113284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2gtbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=186&publication_year=2014&pages=13-21&author=A.+D.+Vogtauthor=N.+Pozziauthor=Z.+Chenauthor=E.+Di+Cera&title=Essential+role+of+conformational+selection+in+ligand+binding&doi=10.1016%2Fj.bpc.2013.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17bR"><div class="casContent"><span class="casTitleNuber">17b</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of conformational selection in ligand binding</span></div><div class="casAuthors">Vogt, Austin D.; Pozzi, Nicola; Chen, Zhiwei; Di Cera, Enrico</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">186</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13-21</span>CODEN:
                <span class="NLM_cas:coden">BICIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0301-4622</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Two competing and mutually exclusive mechanisms of ligand recognition, conformational selection and induced fit, have dominated the interpretation of ligand binding by biopolymers for almost 6 decades.  Conformational selection posits the pre-existence of multiple conformations of the macromol. from which the ligand selects the optimal one.  Induced fit, on the other hand, postulates the existence of conformational rearrangements of the original conformation into an optimal one that are induced by the binding of the ligand.  In the former case, conformational transitions precede the binding event; in the latter, conformational changes follow the binding step.  Kineticists have used a facile criterion to distinguish between the 2 mechanisms based on the dependence of the rate of relaxation to equil. (kobs) on the ligand concn., [L].  A value of kobs decreasing hyperbolically with [L] has been seen as diagnostic of conformational selection, while a value of kobs increasing hyperbolically with [L] has been considered diagnostic of induced fit.  However, this simple conclusion is only valid under the rather unrealistic assumption of conformational transitions being much slower than binding and dissocn. events.  In general, induced fit only produces values of kobs that increase with [L], but conformational selection is more versatile and is assocd. with values of kobs that increase with, decrease with, or are independent of [L].  The richer repertoire of kinetic properties of conformational selection applies to kinetic mechanisms with single or multiple saturable relaxations and explains the behavior of nearly all exptl. systems reported in the literature thus far.  Conformational selection is always sufficient and often necessary to account for the relaxation kinetics of ligand binding to a biol. macromol. and is therefore an essential component of any binding mechanism.  On the other hand, induced fit is never necessary and only sufficient in a few cases.  Therefore, the long assumed importance and preponderance of induced fit as a mechanism of ligand binding should be reconsidered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrglcks_Y6CTLVg90H21EOLACvtfcHk0liwl4tozbvbTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2gtbbI&md5=b63184a2f3f7ae6b7c291cb8277aaf3e</span></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.1016%2Fj.bpc.2013.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpc.2013.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DA.%2BD.%26aulast%3DPozzi%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DEssential%2520role%2520of%2520conformational%2520selection%2520in%2520ligand%2520binding%26jtitle%3DBiophys.%2520Chem.%26date%3D2014%26volume%3D186%26spage%3D13%26epage%3D21%26doi%3D10.1016%2Fj.bpc.2013.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watermana, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winterb, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurstb, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Monterob, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattnec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewsterc, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauterc, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">The DIALS framework for integration software</span>. <i>CCP4 Newsl. Protein Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">19</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=16-19&author=D.+G.+Watermanaauthor=G.+Winterbauthor=J.+M.+Parkhurstbauthor=L.+Fuentes-Monterobauthor=J.+Hattnecauthor=A.+Brewstercauthor=N.+K.+Sautercauthor=G.+Evans&title=The+DIALS+framework+for+integration+software"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWatermana%26aufirst%3DD.%2BG.%26aulast%3DWinterb%26aufirst%3DG.%26aulast%3DParkhurstb%26aufirst%3DJ.%2BM.%26aulast%3DFuentes-Monterob%26aufirst%3DL.%26aulast%3DHattnec%26aufirst%3DJ.%26aulast%3DBrewsterc%26aufirst%3DA.%26aulast%3DSauterc%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DThe%2520DIALS%2520framework%2520for%2520integration%2520software%26jtitle%3DCCP4%2520Newsl.%2520Protein%2520Crystallogr.%26date%3D2013%26volume%3D49%26spage%3D16%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. In  <i>Methods in Enzymology: Macromolecular Crystallography, part A</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C.W.,  Jr.</span>, <span class="NLM_string-name">Sweet, R. M.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=C.W.+Carter&author=R.+M.+Sweet&title=Methods+in+Enzymology%3A+Macromolecular+Crystallography%2C+part+A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMethods%2520in%2520Enzymology%253A%2520Macromolecular%2520Crystallography%252C%2520part%2520A%26aulast%3DCarter%26aufirst%3DC.W.%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhtKuxgyJXS5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liPYKX40yWJVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0liPYKX40yWJVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7a','cit7b'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15a','cit15b'],'ref16':['cit16a','cit16b'],'ref17':['cit17a','cit17b'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 7 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Hans Matter, Christian Buning, Dan Dragos Stefanescu, Sven Ruf, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Hessler</span>. </span><span class="cited-content_cbyCitation_article-title">Using Graph Databases to Investigate Trends in Structure–Activity Relationship Networks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6120-6134. <a href="https://doi.org/10.1021/acs.jcim.0c00947" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00947</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00947%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DUsing%252BGraph%252BDatabases%252Bto%252BInvestigate%252BTrends%252Bin%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BNetworks%26aulast%3DMatter%26aufirst%3DHans%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14082020%26date%3D27112020%26volume%3D60%26issue%3D12%26spage%3D6120%26epage%3D6134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Christian Kramer</span>. </span><span class="cited-content_cbyCitation_article-title">Nonadditivity Analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (9)
                                     , 4034-4042. <a href="https://doi.org/10.1021/acs.jcim.9b00631" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00631</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00631%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNonadditivity%252BAnalysis%26aulast%3DKramer%26aufirst%3DChristian%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D30072019%26date%3D11092019%26volume%3D59%26issue%3D9%26spage%3D4034%26epage%3D4042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Laurent Gomez</span>. </span><span class="cited-content_cbyCitation_article-title">Decision Making in Medicinal Chemistry: The Power of Our Intuition. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (10)
                                     , 956-958. <a href="https://doi.org/10.1021/acsmedchemlett.8b00359" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDecision%252BMaking%252Bin%252BMedicinal%252BChemistry%25253A%252BThe%252BPower%252Bof%252BOur%252BIntuition%26aulast%3DGomez%26aufirst%3DLaurent%26date%3D2018%26date%3D2018%26date%3D11092018%26volume%3D9%26issue%3D10%26spage%3D956%26epage%3D958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D.  Gogishvili</span>, <span class="hlFld-ContribAuthor ">E.  Nittinger</span>, <span class="hlFld-ContribAuthor ">C.  Margreitter</span>, <span class="hlFld-ContribAuthor ">C.  Tyrchan</span>. </span><span class="cited-content_cbyCitation_article-title">Nonadditivity in public and inhouse data: implications for drug design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2021,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-021-00525-z" title="DOI URL">https://doi.org/10.1186/s13321-021-00525-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-021-00525-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-021-00525-z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DNonadditivity%252Bin%252Bpublic%252Band%252Binhouse%252Bdata%25253A%252Bimplications%252Bfor%252Bdrug%252Bdesign%26aulast%3DGogishvili%26aufirst%3DD.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Long  Tang</span>, <span class="hlFld-ContribAuthor ">Jianchun  Jiang</span>, <span class="hlFld-ContribAuthor ">Guoqiang  Song</span>, <span class="hlFld-ContribAuthor ">Yajing  Wang</span>, <span class="hlFld-ContribAuthor ">Ziheng  Zhuang</span>, <span class="hlFld-ContribAuthor ">Ying  Tan</span>, <span class="hlFld-ContribAuthor ">Yan  Xia</span>, <span class="hlFld-ContribAuthor ">Xianfeng  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoqing  Feng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Biological Evaluation of Novel 6H-Benzo[c]chromen-6-one Derivatives as Potential Phosphodiesterase II Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (11)
                                     , 5680. <a href="https://doi.org/10.3390/ijms22115680" title="DOI URL">https://doi.org/10.3390/ijms22115680</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22115680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22115680%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BEvaluation%252Bof%252BNovel%252B6H-Benzo%25255Bc%25255Dchromen-6-one%252BDerivatives%252Bas%252BPotential%252BPhosphodiesterase%252BII%252BInhibitors%26aulast%3DTang%26aufirst%3DLong%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D11%26spage%3D5680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ritawidya</span>, <span class="hlFld-ContribAuthor ">Wenzel</span>, <span class="hlFld-ContribAuthor ">Teodoro</span>, <span class="hlFld-ContribAuthor ">Toussaint</span>, <span class="hlFld-ContribAuthor ">Kranz</span>, <span class="hlFld-ContribAuthor ">Deuther-Conrad</span>, <span class="hlFld-ContribAuthor ">Dukic-Stefanovic</span>, <span class="hlFld-ContribAuthor ">Ludwig</span>, <span class="hlFld-ContribAuthor ">Scheunemann</span>, <span class="hlFld-ContribAuthor ">Brust</span>. </span><span class="cited-content_cbyCitation_article-title">Radiosynthesis and Biological Investigation of a Novel Fluorine-18 Labeled Benzoimidazotriazine- Based Radioligand for the Imaging of Phosphodiesterase 2A with Positron Emission Tomography. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (22)
                                     , 4149. <a href="https://doi.org/10.3390/molecules24224149" title="DOI URL">https://doi.org/10.3390/molecules24224149</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24224149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24224149%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRadiosynthesis%252Band%252BBiological%252BInvestigation%252Bof%252Ba%252BNovel%252BFluorine-18%252BLabeled%252BBenzoimidazotriazine-%252BBased%252BRadioligand%252Bfor%252Bthe%252BImaging%252Bof%252BPhosphodiesterase%252B2A%252Bwith%252BPositron%252BEmission%252BTomography%26aulast%3DRitawidya%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D22%26spage%3D4149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ritawidya</span>, <span class="hlFld-ContribAuthor ">Ludwig</span>, <span class="hlFld-ContribAuthor ">Briel</span>, <span class="hlFld-ContribAuthor ">Brust</span>, <span class="hlFld-ContribAuthor ">Scheunemann</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and In Vitro Evaluation of 8-Pyridinyl-Substituted Benzo[e]imidazo[2,1-c][1,2,4]triazines as Phosphodiesterase 2A Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (15)
                                     , 2791. <a href="https://doi.org/10.3390/molecules24152791" title="DOI URL">https://doi.org/10.3390/molecules24152791</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24152791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24152791%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSynthesis%252Band%252BIn%252BVitro%252BEvaluation%252Bof%252B8-Pyridinyl-Substituted%252BBenzo%25255Be%25255Dimidazo%25255B2%25252C1-c%25255D%25255B1%25252C2%25252C4%25255Dtriazines%252Bas%252BPhosphodiesterase%252B2A%252BInhibitors%26aulast%3DRitawidya%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D15%26spage%3D2791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: a. alanine, H<sub>2</sub>O, NaHCO<sub>3</sub>, 95 °C; b. Raney Ni, NaHCO<sub>3</sub>, H<sub>2</sub>O, 60 psi; c. (i) 2 N NaOH, 30% H<sub>2</sub>O<sub>2</sub>, 100 °C; (ii) HCl; d. EtOH, H<sub>2</sub>SO<sub>4</sub>, 80 °C; e. POCl<sub>3</sub>, <i>N</i>,<i>N</i>-dimethyl aniline, 120 °C; f. ArCONHNH<sub>2</sub>, <i>n</i>-BuOH, 160 °C; g. LiOH, THF/H<sub>2</sub>O; h. DIEA, HATU, R<sup>1</sup>NH<sub>2</sub>, DMF.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Matrix of compounds and lowest energy conformation for compound <b>19</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of compound <b>19</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7E">6C7E</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. PDE2 pIC<sub>50</sub> values of the compounds in the matrix.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. PDE2 pIC<sub>50</sub> values of selected compounds in the matrix demonstrating a lack of additivity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Theoretical nonadditivity distribution expected from mean standard error of the assay of 0.1 log units (red line) and real nonadditivity distribution (blue line). A histogram for experimental data with <i>y</i> axis on the right side is also shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Graphical representation of the nonadditivity values for each compound relative to compound <b>19</b>; the dotted line at NAd = 0.5 represents the value that provides 95% confidence that the nonadditivity observed cannot be explained by the experimental uncertainty.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlay of compounds <b>23</b> (in cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7D">6C7D</a>) and <b>39</b> (in orange, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7F">6C7F</a>) bound to PDE2 enzyme.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/medium/jm-2018-00713m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Correlation between R<sup>1</sup> and R<sup>2</sup> surface areas and deviation from additive SAR (NAd).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-17/acs.jmedchem.8b00713/20180907/images/large/jm-2018-00713m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00713&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 22 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Free, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. W.</span></span> <span> </span><span class="NLM_article-title">A mathematical contribution to structure-activity studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">399</span>, <span class="refDoi"> DOI: 10.1021/jm00334a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00334a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADyaF2cXkt1OltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1964&pages=395-399&author=S.+M.+Freeauthor=J.+W.+Wilson&title=A+mathematical+contribution+to+structure-activity+studies&doi=10.1021%2Fjm00334a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A mathematical contribution to structure-activity studies</span></div><div class="casAuthors">Free, Spencer M., Jr.; Wilson, James W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">395-9</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A math. technique is suggested as a means of describing structure-activity relation of a series of chem. analogs.  The data requirements included specific side chain arrangements and performance characteristics of all analogs tested.  Two examples illustrate the use of the additive math. model where the performance characteristics are measures of biol. activity.  The results rank the structural changes per position by estg. the amt. of biol. response attributed to each change.  The estimates are both pos. and neg.  Several uses for the math. soln. are suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRyTj3xI-aH7Vg90H21EOLACvtfcHk0likOqaoCmzVhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXkt1OltbY%253D&md5=88fabb64d7a5635506a11418a2744bd6</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm00334a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00334a001%26sid%3Dliteratum%253Aachs%26aulast%3DFree%26aufirst%3DS.%2BM.%26aulast%3DWilson%26aufirst%3DJ.%2BW.%26atitle%3DA%2520mathematical%2520contribution%2520to%2520structure-activity%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1964%26volume%3D7%26spage%3D395%26epage%3D399%26doi%3D10.1021%2Fjm00334a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cammarata, A.</span></span> <span> </span><span class="NLM_article-title">Interrelationship of the regression models used for structure-activity analyses</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1021/jm00276a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00276a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1972&pages=573-577&author=A.+Cammarata&title=Interrelationship+of+the+regression+models+used+for+structure-activity+analyses&doi=10.1021%2Fjm00276a001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm00276a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00276a001%26sid%3Dliteratum%253Aachs%26aulast%3DCammarata%26aufirst%3DA.%26atitle%3DInterrelationship%2520of%2520the%2520regression%2520models%2520used%2520for%2520structure-activity%2520analyses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1972%26volume%3D15%26spage%3D573%26epage%3D577%26doi%3D10.1021%2Fjm00276a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jencks, W. P.</span></span> <span> </span><span class="NLM_article-title">On the attribution and additivity of binding energies</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">4046</span>– <span class="NLM_lpage">4050</span>, <span class="refDoi"> DOI: 10.1073/pnas.78.7.4046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1073%2Fpnas.78.7.4046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=16593049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1981&pages=4046-4050&author=W.+P.+Jencks&title=On+the+attribution+and+additivity+of+binding+energies&doi=10.1073%2Fpnas.78.7.4046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">On the attribution and additivity of binding energies</span></div><div class="casAuthors">Jencks, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4046-50</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">It can be useful to describe the Gibbs free energy changes for the binding to a protein of a mol., A-B, and of its component parts, A and B, in terms of the intrinsic binding energies of A and B, ΔGiA and ΔGiB, and a connection Gibbs energy, ΔGs, that is derived largely from changes in translational and rotational entropy.  The relation between ΔGsand ΔGiA and ΔGiB is discussed.  This empirical approach avoids the difficult of interpreting obsd. ΔH and TΔS values for aq. solns.  The ΔGi and ΔGsterms can be large for binding to enzymes and other proteins.  The use and interpretation of these binding energies are discussed, esp. with respect to ATP interaction with myosin ATPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2X3Thp5VESLVg90H21EOLACvtfcHk0littRk3dqa_Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D&md5=1cdbbb8b6e3f9d3ff0ba2c662c772375</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.78.7.4046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.78.7.4046%26sid%3Dliteratum%253Aachs%26aulast%3DJencks%26aufirst%3DW.%2BP.%26atitle%3DOn%2520the%2520attribution%2520and%2520additivity%2520of%2520binding%2520energies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1981%26volume%3D78%26spage%3D4046%26epage%3D4050%26doi%3D10.1073%2Fpnas.78.7.4046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillet, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span> <span> </span><span class="NLM_article-title">Assessment of additive/nonadditive effects in structure-activity relationships: implications for iterative drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">7552</span>– <span class="NLM_lpage">7562</span>, <span class="refDoi"> DOI: 10.1021/jm801070q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801070q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2jsb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7552-7562&author=Y.+Patelauthor=V.+J.+Gilletauthor=T.+Howeauthor=J.+Pastorauthor=J.+Oyarzabalauthor=P.+Willett&title=Assessment+of+additive%2Fnonadditive+effects+in+structure-activity+relationships%3A+implications+for+iterative+drug+design&doi=10.1021%2Fjm801070q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Additive/Nonadditive Effects in Structure-Activity Relationships: Implications for Iterative Drug Design</span></div><div class="casAuthors">Patel, Yogendra; Gillet, Valerie J.; Howe, Trevor; Pastor, Joaquin; Oyarzabal, Julen; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7552-7562</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Free-Wilson (FW) anal. is common practice in medicinal chem. and is based on the assumption that the contributions to activity made by substituents at different substitution positions are additive.  We analyze eight near complete combinatorial libraries assayed on several different biol. response(s) (GPCR, ion channel, kinase and P 450 targets) and show that only half-exhibit clear additive behavior, which leads us to question the concept of additivity that is widely taken for granted in drug discovery.  Next, we report a series of retrospective expts. in which subsets are extd. from the libraries for FW anal. to det. the min. attributes (size, distribution of substituents, and activity range) necessary to reach the same conclusion about additive/nonadditive effects.  These attributes can provide guidelines on when it is appropriate to apply FW anal. as well as for library design, and they also have important implications for further steps in iterative drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoenaUR82iX37Vg90H21EOLACvtfcHk0littRk3dqa_Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtl2jsb7L&md5=9c7d2af2b19b0cb52fb2a7edaefb5a67</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm801070q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801070q%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DY.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DPastor%26aufirst%3DJ.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DAssessment%2520of%2520additive%252Fnonadditive%2520effects%2520in%2520structure-activity%2520relationships%253A%2520implications%2520for%2520iterative%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7552%26epage%3D7562%26doi%3D10.1021%2Fjm801070q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuning, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wesler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span> <span> </span><span class="NLM_article-title">Accurate and reliable prediction of relative ligand binding potency in prospective drug discovery by way of a modern free-energy calculation protocol and force field</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">2695</span>– <span class="NLM_lpage">2703</span>, <span class="refDoi"> DOI: 10.1021/ja512751q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja512751q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2695-2703&author=L.+Wangauthor=Y.+Wuauthor=Y.+Dengauthor=B.+Kimauthor=L.+Pierceauthor=G.+Krilovauthor=D.+Lupyanauthor=S.+Robinsonauthor=M.+Dahlgrenauthor=J.+Greenwoodauthor=D.+L.+Romeroauthor=C.+Masseauthor=J.+L.+Knightauthor=T.+Steinbrecherauthor=T.+Beuningauthor=W.+Dammauthor=E.+Harderauthor=W.+Shermanauthor=M.+Brewerauthor=R.+Weslerauthor=M.+Murckoauthor=L.+Fryeauthor=R.+Faridauthor=T.+Linauthor=D.+L.+Mobleyauthor=W.+L.+Jorgensenauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=R.+Abel&title=Accurate+and+reliable+prediction+of+relative+ligand+binding+potency+in+prospective+drug+discovery+by+way+of+a+modern+free-energy+calculation+protocol+and+force+field&doi=10.1021%2Fja512751q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</span></div><div class="casAuthors">Wang, Lingle; Wu, Yujie; Deng, Yuqing; Kim, Byungchan; Pierce, Levi; Krilov, Goran; Lupyan, Dmitry; Robinson, Shaughnessy; Dahlgren, Markus K.; Greenwood, Jeremy; Romero, Donna L.; Masse, Craig; Knight, Jennifer L.; Steinbrecher, Thomas; Beuming, Thijs; Damm, Wolfgang; Harder, Ed; Sherman, Woody; Brewer, Mark; Wester, Ron; Murcko, Mark; Frye, Leah; Farid, Ramy; Lin, Teng; Mobley, David L.; Jorgensen, William L.; Berne, Bruce J.; Friesner, Richard A.; Abel, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2695-2703</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Designing tight-binding ligands is a primary objective of small-mol. drug discovery.  Over the past few decades, free-energy calcns. have benefited from improved force fields and sampling algorithms, as well as the advent of low-cost parallel computing.  However, it has proven to be challenging to reliably achieve the level of accuracy that would be needed to guide lead optimization (∼5× in binding affinity) for a wide range of ligands and protein targets.  Not surprisingly, widespread com. application of free-energy simulations has been limited due to the lack of large-scale validation coupled with the tech. challenges traditionally assocd. with running these types of calcns.  Here, we report an approach that achieves an unprecedented level of accuracy across a broad range of target classes and ligands, with retrospective results encompassing 200 ligands and a wide variety of chem. perturbations, many of which involve significant changes in ligand chem. structures.  In addn., we have applied the method in prospective drug discovery projects and found a significant improvement in the quality of the compds. synthesized that have been predicted to be potent.  Compds. predicted to be potent by this approach have a substantial redn. in false positives relative to compds. synthesized on the basis of other computational or medicinal chem. approaches.  Furthermore, the results are consistent with those obtained from our retrospective studies, demonstrating the robustness and broad range of applicability of this approach, which can be used to drive decisions in lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor9UtXEyLPpLVg90H21EOLACvtfcHk0ljjCFLRkL_imA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2iuro%253D&md5=37a4f4a6c085f47ed531342643b6c33b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fja512751q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja512751q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DPierce%26aufirst%3DL.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DRobinson%26aufirst%3DS.%26aulast%3DDahlgren%26aufirst%3DM.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DRomero%26aufirst%3DD.%2BL.%26aulast%3DMasse%26aufirst%3DC.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DBeuning%26aufirst%3DT.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DBrewer%26aufirst%3DM.%26aulast%3DWesler%26aufirst%3DR.%26aulast%3DMurcko%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMobley%26aufirst%3DD.%2BL.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DAbel%26aufirst%3DR.%26atitle%3DAccurate%2520and%2520reliable%2520prediction%2520of%2520relative%2520ligand%2520binding%2520potency%2520in%2520prospective%2520drug%2520discovery%2520by%2520way%2520of%2520a%2520modern%2520free-energy%2520calculation%2520protocol%2520and%2520force%2520field%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D2695%26epage%3D2703%26doi%3D10.1021%2Fja512751q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span>Drug Design
Data Resource. <a href="https://drugdesigndata.org/" class="extLink">https://drugdesigndata.org/</a> (accessed on June 8th <span class="NLM_year">2017</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Drug+Design%0AData+Resource.+https%3A%2F%2Fdrugdesigndata.org%2F+%28accessed+on+June+8th+2017%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">Predicting binding free energies: frontiers and benchmarks</span>. <i>Annu. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1146/annurev-biophys-070816-033654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1146%2Fannurev-biophys-070816-033654" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=28399632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvFClsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2017&pages=531-58&author=D.+Mobleyauthor=M.+K.+Gilson&title=Predicting+binding+free+energies%3A+frontiers+and+benchmarks&doi=10.1146%2Fannurev-biophys-070816-033654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Binding Free Energies: Frontiers and Benchmarks</span></div><div class="casAuthors">Mobley, David L.; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">531-558</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Binding free energy calcns. based on mol. simulations provide predicted affinities for biomol. complexes.  These calcns. begin with a detailed description of a system, including its chem. compn. and the interactions among its components.  Simulations of the system are then used to compute thermodn. information, such as binding affinities.  Because of their promise for guiding mol. design, these calcns. have recently begun to see widespread applications in early-stage drug discovery.  However, many hurdles remain in making them a robust and reliable tool.  In this review, we highlight key challenges of these calcns., discuss some examples of these challenges, and call for the designation of std. community benchmark test systems that will help the research community generate and evaluate progress.  In our view, progress will require careful assessment and evaluation of new methods, force fields, and modeling innovations on well-characterized benchmark systems, and we lay out our vision for how this can be achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJJQBeP6sFOrVg90H21EOLACvtfcHk0ljjCFLRkL_imA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvFClsr0%253D&md5=38af2949ba205daaf9899b87c4e1a2bd</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-070816-033654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-070816-033654%26sid%3Dliteratum%253Aachs%26aulast%3DMobley%26aufirst%3DD.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DPredicting%2520binding%2520free%2520energies%253A%2520frontiers%2520and%2520benchmarks%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2017%26volume%3D46%26spage%3D531%26epage%3D58%26doi%3D10.1146%2Fannurev-biophys-070816-033654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit7b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dimova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Advances in activity cliff research</span>. <i>Mol. Inf.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1002/minf.201600023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1002%2Fminf.201600023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=27492084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC28XltVOiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=181-191&author=D.+Dimovaauthor=J.+Bajorath&title=Advances+in+activity+cliff+research&doi=10.1002%2Fminf.201600023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in Activity Cliff Research</span></div><div class="casAuthors">Dimova, Dilyana; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">181-191</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Activity cliffs, i.e. similar compds. with large potency differences, are of interest from a chem. and informatics viewpoint; as a source of structure-activity relationship information, for compd. optimization, and activity prediction.  Herein, recent highlights of activity cliff research are discussed including studies that have further extended our understanding of activity cliffs, yielded unprecedented insights, or paved the way for practical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprlJLFAKwUELVg90H21EOLACvtfcHk0liCL_QNsqVTtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltVOiu78%253D&md5=09066b5cbe326e9061964db60502ef5f</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1002%2Fminf.201600023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201600023%26sid%3Dliteratum%253Aachs%26aulast%3DDimova%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DAdvances%2520in%2520activity%2520cliff%2520research%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D181%26epage%3D191%26doi%3D10.1002%2Fminf.201600023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McClure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankley, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">Pyrazole CCK(1) receptor antagonists. Part 1: solution-phase library synthesis and determination of Free-Wilson additivity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2005.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.bmcl.2005.09.048" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=72-76&author=K.+McClureauthor=M.+Hackauthor=L.+Huangauthor=C.+Sehonauthor=M.+Mortonauthor=T.+Barrettauthor=N.+Shankleyauthor=J.+G.+Breitenbucher&title=Pyrazole+CCK%281%29+receptor+antagonists.+Part+1%3A+solution-phase+library+synthesis+and+determination+of+Free-Wilson+additivity&doi=10.1016%2Fj.bmcl.2005.09.048"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.09.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DMcClure%26aufirst%3DK.%26aulast%3DHack%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DSehon%26aufirst%3DC.%26aulast%3DMorton%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DT.%26aulast%3DShankley%26aufirst%3DN.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DPyrazole%2520CCK%25281%2529%2520receptor%2520antagonists.%2520Part%25201%253A%2520solution-phase%2520library%2520synthesis%2520and%2520determination%2520of%2520Free-Wilson%2520additivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D72%26epage%3D76%26doi%3D10.1016%2Fj.bmcl.2005.09.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankley, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span> <span> </span><span class="NLM_article-title">Pyrazole CCK(1) receptor antagonists. Part 2: SAR studies by solid-phase library synthesis and determination of Free-Wilson additivity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2005.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.bmcl.2005.09.041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=77-80&author=C.+Sehonauthor=K.+McClureauthor=M.+Hackauthor=M.+Mortonauthor=L.+Gomezauthor=L.+Liauthor=T.+D.+Barrettauthor=N.+Shankleyauthor=J.+G.+Breitenbucher&title=Pyrazole+CCK%281%29+receptor+antagonists.+Part+2%3A+SAR+studies+by+solid-phase+library+synthesis+and+determination+of+Free-Wilson+additivity&doi=10.1016%2Fj.bmcl.2005.09.041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DSehon%26aufirst%3DC.%26aulast%3DMcClure%26aufirst%3DK.%26aulast%3DHack%26aufirst%3DM.%26aulast%3DMorton%26aufirst%3DM.%26aulast%3DGomez%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DBarrett%26aufirst%3DT.%2BD.%26aulast%3DShankley%26aufirst%3DN.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26atitle%3DPyrazole%2520CCK%25281%2529%2520receptor%2520antagonists.%2520Part%25202%253A%2520SAR%2520studies%2520by%2520solid-phase%2520library%2520synthesis%2520and%2520determination%2520of%2520Free-Wilson%2520additivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D77%26epage%3D80%26doi%3D10.1016%2Fj.bmcl.2005.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Non-additivity of functional group contributions in protein-ligand binding: a comprehensive study by crystallography and isothermal titration calorimetry</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>397</i></span>,  <span class="NLM_fpage">1042</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.jmb.2010.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=20156458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=397&publication_year=2010&pages=1042-1054&author=B.+Baumauthor=L.+Muleyauthor=M.+Smolinskiauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&title=Non-additivity+of+functional+group+contributions+in+protein-ligand+binding%3A+a+comprehensive+study+by+crystallography+and+isothermal+titration+calorimetry&doi=10.1016%2Fj.jmb.2010.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Non-additivity of Functional Group Contributions in Protein-Ligand Binding: A Comprehensive Study by Crystallography and Isothermal Titration Calorimetry</span></div><div class="casAuthors">Baum, Bernhard; Muley, Laveena; Smolinski, Michael; Heine, Andreas; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">397</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1042-1054</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Additivity of functional group contributions to protein-ligand binding is a very popular concept in medicinal chem. as the basis of rational design and optimized lead structures.  Most of the currently applied scoring functions for docking build on such additivity models.  Even though the limitation of this concept is well known, case studies examg. in detail why additivity fails at the mol. level are still very scarce.  The present study shows, by use of crystal structure anal. and isothermal titrn. calorimetry for a congeneric series of thrombin inhibitors, that extensive cooperative effects between hydrophobic contacts and hydrogen bond formation are intimately coupled via dynamic properties of the formed complexes.  The formation of optimal lipophilic contacts with the surface of the thrombin S3 pocket and the full desolvation of this pocket can conflict with the formation of an optimal hydrogen bond between ligand and protein.  The mutual contributions of the competing interactions depend on the size of the ligand hydrophobic substituent and influence the residual mobility of ligand portions at the binding site.  Anal. of the individual crystal structures and factorizing the free energy into enthalpy and entropy demonstrates that binding affinity of the ligands results from a mixt. of enthalpic contributions from hydrogen bonding and hydrophobic contacts, and entropic considerations involving an increasing loss of residual mobility of the bound ligands.  This complex picture of mutually competing and partially compensating enthalpic and entropic effects dets. the non-additivity of free energy contributions to ligand binding at the mol. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGousWs7tLpU3LVg90H21EOLACvtfcHk0ljjMVX-soXU4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKmurg%253D&md5=2f61dde175a6ee27551179644eee3f00</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DMuley%26aufirst%3DL.%26aulast%3DSmolinski%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DNon-additivity%2520of%2520functional%2520group%2520contributions%2520in%2520protein-ligand%2520binding%253A%2520a%2520comprehensive%2520study%2520by%2520crystallography%2520and%2520isothermal%2520titration%2520calorimetry%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D397%26spage%3D1042%26epage%3D1054%26doi%3D10.1016%2Fj.jmb.2010.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schauperl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldner, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podewitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Binding pose flip explained via enthalpic and entropic contributions</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">345</span>– <span class="NLM_lpage">354</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosF2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=345-354&author=M.+Schauperlauthor=P.+Czodrowskiauthor=E.+J.+Fuchsauthor=R.+G.+Huberauthor=B.+J.+Waldnerauthor=M.+Podewitzauthor=C.+Kramerauthor=K.+R.+Liedl&title=Binding+pose+flip+explained+via+enthalpic+and+entropic+contributions&doi=10.1021%2Facs.jcim.6b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Pose Flip Explained via Enthalpic and Entropic Contributions</span></div><div class="casAuthors">Schauperl, Michael; Czodrowski, Paul; Fuchs, Julian E.; Huber, Roland G.; Waldner, Birgit J.; Podewitz, Maren; Kramer, Christian; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">345-354</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The anomalous binding modes of 5 highly similar fragments of angiopoietin receptor TIE2 inhibitors, showing 3 distinct binding poses were investigated.  The authors report a quant. rationalization for the changes in binding pose based on mol. dynamics (MD) simulations.  The authors investigated 5 fragments in complex with the transforming growth factor-β receptor type 1 kinase domain.  Analyses of these MD simulations using Grid Inhomogeneous Solvation Theory (GIST), pKa calcns., and a tool to investigate enthalpic differences upon binding unraveled the various thermodn. contributions to the different binding modes.  While one binding mode flip could be rationalized by steric repulsion, the 2nd binding pose flip revealed a different protonation state for one of the ligands, leading to different enthalpic and entropic contributions to the binding free energy.  While one binding pose was stabilized by the displacement of entropically unfavored water mols. (binding pose detd. by solvation entropy), ligands in the other binding pose were stabilized by strong enthalpic interactions, overcompensating the unfavorable water entropy in this pose (binding pose detd. by enthalpic interactions).  This anal. elucidated unprecedented details detg. the flipping of the binding modes, which can elegantly explain the exptl. findings for this system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyksMbMzYXP7Vg90H21EOLACvtfcHk0ljjMVX-soXU4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosF2huw%253D%253D&md5=9fa0737411901c4d27ae9954dca37ecf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00483%26sid%3Dliteratum%253Aachs%26aulast%3DSchauperl%26aufirst%3DM.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DFuchs%26aufirst%3DE.%2BJ.%26aulast%3DHuber%26aufirst%3DR.%2BG.%26aulast%3DWaldner%26aufirst%3DB.%2BJ.%26aulast%3DPodewitz%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DBinding%2520pose%2520flip%2520explained%2520via%2520enthalpic%2520and%2520entropic%2520contributions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D345%26epage%3D354%26doi%3D10.1021%2Facs.jcim.6b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Charges shift protonation: neutron diffraction reveals that aniline and 2-aminopyridine become protonated upon binding to trypsin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4887</span>– <span class="NLM_lpage">4890</span>, <span class="refDoi"> DOI: 10.1002/anie.201701038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1002%2Fanie.201701038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvF2ltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=4887-4890&author=J.+Schiebelauthor=R.+Gaspariauthor=A.+Sandnerauthor=K.+Ngoauthor=H.+J.+Gerberauthor=A.+Cavalliauthor=A.+Ostermannauthor=A.+Heineauthor=G.+Klebe&title=Charges+shift+protonation%3A+neutron+diffraction+reveals+that+aniline+and+2-aminopyridine+become+protonated+upon+binding+to+trypsin&doi=10.1002%2Fanie.201701038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Charges shift protonation: Neutron diffraction reveals that aniline and 2-aminopyridine become protonated upon binding to trypsin</span></div><div class="casAuthors">Schiebel, Johannes; Gaspari, Roberto; Sandner, Anna; Ngo, Khang; Gerber, Hans-Dieter; Cavalli, Andrea; Ostermann, Andreas; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4887-4890</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">H atoms play a key role in protein-ligand recognition.  They det. the quality of established H-bonding networks and define the protonation of bound ligands.  Structural visualization of H atoms by x-ray crystallog. is rarely possible.  Here, we used neutron diffraction to det. the positions of the H atoms in the ligands, aniline and 2-aminopyridine, bound to the archetypical serine protease, trypsin.  The resulting structures showed the best resoln. so far achieved for proteins larger than 100 residues and allowed an accurate description of the protonation states and interactions with nearby water mols.  Despite its low pKa of 4.6 and a large distance of 3.6 Å to the charged Asp-189 residue at the bottom of the trypsin S1 pocket, the amino group of aniline became protonated, whereas in 2-aminopyridine, the pyridine N atom picked up the proton although its NH2 group was 1.6 Å closer to Asp-189.  Therefore, apart from charge-charge distances, tautomer stability was decisive for the resulting binding poses, an aspect that is pivotal for predicting correct binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg449p0i3o-7Vg90H21EOLACvtfcHk0lgQf_sEzwATyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvF2ltr8%253D&md5=a48df0f5959e03a0fdfc528de58bbb6f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201701038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201701038%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DSandner%26aufirst%3DA.%26aulast%3DNgo%26aufirst%3DK.%26aulast%3DGerber%26aufirst%3DH.%2BJ.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DOstermann%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DCharges%2520shift%2520protonation%253A%2520neutron%2520diffraction%2520reveals%2520that%2520aniline%2520and%25202-aminopyridine%2520become%2520protonated%2520upon%2520binding%2520to%2520trypsin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D4887%26epage%3D4890%26doi%3D10.1002%2Fanie.201701038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Strong nonadditivity as a key structure-activity relationship feature: distinguishing structural changes from assay artifacts</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">483</span>– <span class="NLM_lpage">494</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFamtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=483-494&author=C.+Kramerauthor=J.+E.+Fuchsauthor=K.+R.+Liedl&title=Strong+nonadditivity+as+a+key+structure-activity+relationship+feature%3A+distinguishing+structural+changes+from+assay+artifacts&doi=10.1021%2Facs.jcim.5b00018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Strong Nonadditivity as a Key Structure-Activity Relationship Feature: Distinguishing Structural Changes from Assay Artifacts</span></div><div class="casAuthors">Kramer, Christian; Fuchs, Julian E.; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">483-494</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nonadditivity in protein-ligand affinity data represents highly instructive structure-activity relation (SAR) features that indicate structural changes and have the potential to guide rational drug design.  At the same time, nonadditivity is a challenge for both basic SAR anal. as well as many ligand-based data anal. techniques such as Free-Wilson Anal. and Matched Mol. Pair anal., since linear substituent contribution models inherently assume additivity and thus do not work in such cases.  While structural causes for nonadditivity have been analyzed anecdotally, no systematic approaches to interpret and use nonadditivity prospectively have been developed yet.  In this contribution, the authors lay the statistical framework for systematic anal. of nonadditivity in a SAR series.  First, the authors develop a general metric to quantify nonadditivity.  Then, the authors demonstrate the non-negligible impact of exptl. uncertainty that creates apparent nonadditivity, and the authors introduce techniques to handle exptl. uncertainty.  Finally, the authors analyze public SAR data sets for strong nonadditivity and use recourse to the original publications and available x-ray structures to find structural explanations for the nonadditivity obsd.  The authors find that all cases of strong nonadditivity (ΔΔpKi and ΔΔpIC50 > 2.0 log units) with sufficient structural information to generate reasonable hypothesis involve changes in binding mode.  With the appropriate statistical basis, nonadditivity anal. offers a variety of new attempts for various areas in computer-aided drug design, including the validation of scoring functions and free energy perturbation approaches, binding pocket classification, and novel features in SAR anal. tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAdEaoXXhBu7Vg90H21EOLACvtfcHk0lgQf_sEzwATyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFamtrw%253D&md5=4f17f57045ef0d7bed85a511b104bc6a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00018%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DStrong%2520nonadditivity%2520as%2520a%2520key%2520structure-activity%2520relationship%2520feature%253A%2520distinguishing%2520structural%2520changes%2520from%2520assay%2520artifacts%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D483%26epage%3D494%26doi%3D10.1021%2Facs.jcim.5b00018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andres-Gil, J. I.</span>; <span class="NLM_string-name">Rombouts, F. J. R.</span>; <span class="NLM_string-name">Trabanco-Suarez, A. A.</span>; <span class="NLM_string-name">Vanhoof, G. C. P.</span>; <span class="NLM_string-name">Buijnsters, A.</span>; <span class="NLM_string-name">Guillemont, J. E. G.</span>; <span class="NLM_string-name">Bormans, G. M. R.</span>; <span class="NLM_string-name">Celen, S. J. L.</span>; <span class="NLM_string-name">Vliegen, M.</span></span> <span> </span><span class="NLM_article-title">Preparation of (Un)labeled 1-aryl-4-methyl-[1,2,4]triazolo[4,3-a]quinoxaline Derivatives as PDE2 and PDE10 Inhibitors for Therapy and Imaging</span>. <span class="NLM_patent">WO2013000924</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+I.+Andres-Gil&author=F.+J.+R.+Rombouts&author=A.+A.+Trabanco-Suarez&author=G.+C.+P.+Vanhoof&author=A.+Buijnsters&author=J.+E.+G.+Guillemont&author=G.+M.+R.+Bormans&author=S.+J.+L.+Celen&author=M.+Vliegen&title=Preparation+of+%28Un%29labeled+1-aryl-4-methyl-%5B1%2C2%2C4%5Dtriazolo%5B4%2C3-a%5Dquinoxaline+Derivatives+as+PDE2+and+PDE10+Inhibitors+for+Therapy+and+Imaging"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAndres-Gil%26aufirst%3DJ.%2BI.%26atitle%3DPreparation%2520of%2520%2528Un%2529labeled%25201-aryl-4-methyl-%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-a%255Dquinoxaline%2520Derivatives%2520as%2520PDE2%2520and%2520PDE10%2520Inhibitors%2520for%2520Therapy%2520and%2520Imaging%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span> <span> </span><span class="NLM_article-title">X-ray crystal structure of phosphodiesterase 2 in complex with a highly selective, nanomolar inhibitor reveals a binding-induced pocket important for selectivity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">11708</span>– <span class="NLM_lpage">11711</span>, <span class="refDoi"> DOI: 10.1021/ja404449g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja404449g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=11708-11711&author=J.+Zhuauthor=Q.+Yangauthor=D.+Daiauthor=Q.+Huang&title=X-ray+crystal+structure+of+phosphodiesterase+2+in+complex+with+a+highly+selective%2C+nanomolar+inhibitor+reveals+a+binding-induced+pocket+important+for+selectivity&doi=10.1021%2Fja404449g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1021%2Fja404449g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja404449g%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DDai%26aufirst%3DD.%26aulast%3DHuang%26aufirst%3DQ.%26atitle%3DX-ray%2520crystal%2520structure%2520of%2520phosphodiesterase%25202%2520in%2520complex%2520with%2520a%2520highly%2520selective%252C%2520nanomolar%2520inhibitor%2520reveals%2520a%2520binding-induced%2520pocket%2520important%2520for%2520selectivity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D11708%26epage%3D11711%26doi%3D10.1021%2Fja404449g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Benito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keränen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vlijmen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tresadern, G.</span></span> <span> </span><span class="NLM_article-title">Predicting binding free energies of PDE2 inhibitors. The difficulties of protein conformation</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4883</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-23039-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1038%2Fs41598-018-23039-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=29559702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A280%3ADC%252BC1MnjtFKlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=4883&author=L.+P%C3%A9rez-Benitoauthor=H.+Ker%C3%A4nenauthor=H.+van+Vlijmenauthor=G.+Tresadern&title=Predicting+binding+free+energies+of+PDE2+inhibitors.+The+difficulties+of+protein+conformation&doi=10.1038%2Fs41598-018-23039-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Binding Free Energies of PDE2 Inhibitors. The Difficulties of Protein Conformation</span></div><div class="casAuthors">Perez-Benito Laura; Keranen Henrik; van Vlijmen Herman; Tresadern Gary; Keranen Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4883</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A congeneric series of 21 phosphodiesterase 2 (PDE2) inhibitors are reported.  Crystal structures show how the molecules can occupy a 'top-pocket' of the active site.  Molecules with small substituents do not enter the pocket, a critical leucine (Leu770) is closed and water molecules are present.  Large substituents enter the pocket, opening the Leu770 conformation and displacing the waters.  We also report an X-ray structure revealing a new conformation of the PDE2 active site domain.  The relative binding affinities of these compounds were studied with free energy perturbation (FEP) methods and it represents an attractive real-world test case.  In general, the calculations could predict the energy of small-to-small, or large-to-large molecule perturbations.  However, accurately capturing the transition from small-to-large proved challenging.  Only when using alternative protein conformations did results improve.  The new X-ray structure, along with a modelled dimer, conferred stability to the catalytic domain during the FEP molecular dynamics (MD) simulations, increasing the convergence and thereby improving the prediction of ΔΔG of binding for some small-to-large transitions.  In summary, we found the most significant improvement in results when using different protein structures, and this data set is useful for future free energy validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMdLif4rzX6u_Q9URG6Dx8fW6udTcc2eY3x30ezSrXCrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnjtFKlsA%253D%253D&md5=44c26c5bd2fe3e35bba344abb8262eeb</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-23039-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-23039-5%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Benito%26aufirst%3DL.%26aulast%3DKer%25C3%25A4nen%26aufirst%3DH.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%26aulast%3DTresadern%26aufirst%3DG.%26atitle%3DPredicting%2520binding%2520free%2520energies%2520of%2520PDE2%2520inhibitors.%2520The%2520difficulties%2520of%2520protein%2520conformation%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D4883%26doi%3D10.1038%2Fs41598-018-23039-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Najmanovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuttner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, M.</span></span> <span> </span><span class="NLM_article-title">Side-chain flexibility in proteins upon ligand binding</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(20000515)39:3<261::AID-PROT90>3.0.CO;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1002%2F%28SICI%291097-0134%2820000515%2939%3A3%3C261%3A%3AAID-PROT90%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10737948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BD3cXislGqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=261-268&author=R.+Najmanovichauthor=J.+Kuttnerauthor=V.+Sobolevauthor=M.+Edelman&title=Side-chain+flexibility+in+proteins+upon+ligand+binding&doi=10.1002%2F%28SICI%291097-0134%2820000515%2939%3A3%3C261%3A%3AAID-PROT90%3E3.0.CO%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16aR"><div class="casContent"><span class="casTitleNuber">16a</span><div class="casTitle"><span class="NLM_cas:atitle">Side-chain flexibility in proteins upon ligand binding</span></div><div class="casAuthors">Najmanovich, Rafael; Kuttner, Josef; Sobolev, Vladimir; Edelman, Marvin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-268</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Ligand binding may involve a wide range of structural changes in the receptor protein, from hinge movement of entire domains to small side-chain rearrangements in the binding pocket residues.  The anal. of side chain flexibility gives insights valuable to improve docking algorithms and can provide an index of amino-acid side-chain flexibility potentially useful in mol. biol. and protein engineering studies.  In this study we analyzed side-chain rearrangements upon ligand binding.  We constructed two non-redundant databases (980 and 353 entries) of "paired" protein structures in complexed (holo-protein) and uncomplexed (apo-protein) forms from the PDB macromol. structural database.  The no. and identity of binding pocket residues that undergo side-chain conformational changes were detd.  We show that, in general, only a small no. of residues in the pocket undergo such changes (e.g., ∼85% of cases show changes in three residues or less).  The flexibility scale has the following order: Lys > Arg, Gln, Met > Glu, Ile, Leu > Asn, Thr, Val, Tyr, Ser, His, Asp > Cys, Trp, Phe; thus, Lys side chains in binding pockets flex 25 times more often then do the Phe side chains.  Normalizing for the no. of flexible dihedral bonds in each amino acid attenuates the scale somewhat, however, the clear trend of large, polar amino acids being more flexible in the pocket than arom. ones remains.  We found no correlation between backbone movement of a residue upon ligand binding and the flexibility of its side chain.  These results are relevant to 1: redn. of search space in docking algorithms by inclusion of side-chain flexibility for a limited no. of binding pocket residues; and 2: utilization of the amino acid flexibility scale in protein engineering studies to alter the flexibility of binding pockets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQBLfK6Ht7TLVg90H21EOLACvtfcHk0lj4u0GT4yflrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXislGqtb8%253D&md5=d78e02ffb901136c1438429365ed5ba4</span></div><a href="/servlet/linkout?suffix=cit16a&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2820000515%2939%3A3%3C261%3A%3AAID-PROT90%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252820000515%252939%253A3%253C261%253A%253AAID-PROT90%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DNajmanovich%26aufirst%3DR.%26aulast%3DKuttner%26aufirst%3DJ.%26aulast%3DSobolev%26aufirst%3DV.%26aulast%3DEdelman%26aufirst%3DM.%26atitle%3DSide-chain%2520flexibility%2520in%2520proteins%2520upon%2520ligand%2520binding%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2000%26volume%3D39%26spage%3D261%26epage%3D268%26doi%3D10.1002%2F%28SICI%291097-0134%2820000515%2939%3A3%3C261%3A%3AAID-PROT90%3E3.0.CO%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit16b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Teague, S. J.</span></span> <span> </span><span class="NLM_article-title">Implications of protein flexibility for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nrd1129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1038%2Fnrd1129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=12838268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVaktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=527-541&author=S.+J.+Teague&title=Implications+of+protein+flexibility+for+drug+discovery&doi=10.1038%2Fnrd1129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16bR"><div class="casContent"><span class="casTitleNuber">16b</span><div class="casTitle"><span class="NLM_cas:atitle">Implications of protein flexibility for drug discovery</span></div><div class="casAuthors">Teague, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">527-541</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins are in const. motion between different conformational states with similar energies.  This has often been ignored in drug design.  However, protein flexibility is fundamental to understanding the ways in which drugs exert biol. effects, their binding-site location, binding orientation, binding kinetics, metab. and transport.  Protein flexibility allows increased affinity to be achieved between a drug and its target.  This is crucial, because the lipophilicity and no. of polar interactions allowed for an oral drug is limited by absorption, distribution, metab. and toxicol. considerations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8-I4F-CgSbVg90H21EOLACvtfcHk0lg55ebtCDTR7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVaktrs%253D&md5=850b8838a78c1e00493da3089a0c62e9</span></div><a href="/servlet/linkout?suffix=cit16b&amp;dbid=16384&amp;doi=10.1038%2Fnrd1129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1129%26sid%3Dliteratum%253Aachs%26aulast%3DTeague%26aufirst%3DS.%2BJ.%26atitle%3DImplications%2520of%2520protein%2520flexibility%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D527%26epage%3D541%26doi%3D10.1038%2Fnrd1129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dogan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, P.</span></span> <span> </span><span class="NLM_article-title">Distinguishing induced fit from conformational selection</span>. <i>Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>189</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.bpc.2014.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.bpc.2014.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=24747333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvVWltb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2014&pages=33-39&author=S.+Gianniauthor=J.+Doganauthor=P.+Jemth&title=Distinguishing+induced+fit+from+conformational+selection&doi=10.1016%2Fj.bpc.2014.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Distinguishing induced fit from conformational selection</span></div><div class="casAuthors">Gianni, Stefano; Dogan, Jakob; Jemth, Per</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33-39</span>CODEN:
                <span class="NLM_cas:coden">BICIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0301-4622</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The interactions between proteins and ligands often involve a conformational change in the protein.  This conformational change can occur before (conformational selection) or after (induced fit) the assocn. with ligand.  It is often very difficult to distinguish induced fit from conformational selection when hyperbolic binding kinetics are obsd.  In light of a recent paper in this journal (Vogt et al., Biophys. Chem., 186, 2014, 13-21) and the current interest in binding mechanisms emerging from obsd. sampling of distinct conformations in protein domains, as well as from the field of intrinsically disordered proteins, we here describe a kinetic method that, at least in some cases, unequivocally distinguishes induced fit from conformational selection.  The method relies on measuring the obsd. rate const. λ for binding and varying both the protein and the ligand in sep. expts.  Whereas induced fit always yields a hyperbolic dependence of increasing λ values, the conformational selection mechanism gives rise to distinct kinetics when the ligand and protein (displaying the conformational change) concn. is varied in sep. expts.  We provide examples from the literature and discuss the limitations of the approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBGtc2RoRf-7Vg90H21EOLACvtfcHk0lg55ebtCDTR7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmvVWltb0%253D&md5=ecbe065454d8d9315638b5960b800c38</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1016%2Fj.bpc.2014.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpc.2014.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DGianni%26aufirst%3DS.%26aulast%3DDogan%26aufirst%3DJ.%26aulast%3DJemth%26aufirst%3DP.%26atitle%3DDistinguishing%2520induced%2520fit%2520from%2520conformational%2520selection%26jtitle%3DBiophys.%2520Chem.%26date%3D2014%26volume%3D189%26spage%3D33%26epage%3D39%26doi%3D10.1016%2Fj.bpc.2014.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit17b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Cera, E.</span></span> <span> </span><span class="NLM_article-title">Essential role of conformational selection in ligand binding</span>. <i>Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>186</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1016/j.bpc.2013.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2Fj.bpc.2013.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=24113284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2gtbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=186&publication_year=2014&pages=13-21&author=A.+D.+Vogtauthor=N.+Pozziauthor=Z.+Chenauthor=E.+Di+Cera&title=Essential+role+of+conformational+selection+in+ligand+binding&doi=10.1016%2Fj.bpc.2013.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17bR"><div class="casContent"><span class="casTitleNuber">17b</span><div class="casTitle"><span class="NLM_cas:atitle">Essential role of conformational selection in ligand binding</span></div><div class="casAuthors">Vogt, Austin D.; Pozzi, Nicola; Chen, Zhiwei; Di Cera, Enrico</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">186</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13-21</span>CODEN:
                <span class="NLM_cas:coden">BICIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0301-4622</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Two competing and mutually exclusive mechanisms of ligand recognition, conformational selection and induced fit, have dominated the interpretation of ligand binding by biopolymers for almost 6 decades.  Conformational selection posits the pre-existence of multiple conformations of the macromol. from which the ligand selects the optimal one.  Induced fit, on the other hand, postulates the existence of conformational rearrangements of the original conformation into an optimal one that are induced by the binding of the ligand.  In the former case, conformational transitions precede the binding event; in the latter, conformational changes follow the binding step.  Kineticists have used a facile criterion to distinguish between the 2 mechanisms based on the dependence of the rate of relaxation to equil. (kobs) on the ligand concn., [L].  A value of kobs decreasing hyperbolically with [L] has been seen as diagnostic of conformational selection, while a value of kobs increasing hyperbolically with [L] has been considered diagnostic of induced fit.  However, this simple conclusion is only valid under the rather unrealistic assumption of conformational transitions being much slower than binding and dissocn. events.  In general, induced fit only produces values of kobs that increase with [L], but conformational selection is more versatile and is assocd. with values of kobs that increase with, decrease with, or are independent of [L].  The richer repertoire of kinetic properties of conformational selection applies to kinetic mechanisms with single or multiple saturable relaxations and explains the behavior of nearly all exptl. systems reported in the literature thus far.  Conformational selection is always sufficient and often necessary to account for the relaxation kinetics of ligand binding to a biol. macromol. and is therefore an essential component of any binding mechanism.  On the other hand, induced fit is never necessary and only sufficient in a few cases.  Therefore, the long assumed importance and preponderance of induced fit as a mechanism of ligand binding should be reconsidered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrglcks_Y6CTLVg90H21EOLACvtfcHk0lh4NB5sKoQOyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2gtbbI&md5=b63184a2f3f7ae6b7c291cb8277aaf3e</span></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.1016%2Fj.bpc.2013.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpc.2013.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DA.%2BD.%26aulast%3DPozzi%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DEssential%2520role%2520of%2520conformational%2520selection%2520in%2520ligand%2520binding%26jtitle%3DBiophys.%2520Chem.%26date%3D2014%26volume%3D186%26spage%3D13%26epage%3D21%26doi%3D10.1016%2Fj.bpc.2013.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watermana, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winterb, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurstb, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Monterob, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattnec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewsterc, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauterc, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">The DIALS framework for integration software</span>. <i>CCP4 Newsl. Protein Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">19</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=16-19&author=D.+G.+Watermanaauthor=G.+Winterbauthor=J.+M.+Parkhurstbauthor=L.+Fuentes-Monterobauthor=J.+Hattnecauthor=A.+Brewstercauthor=N.+K.+Sautercauthor=G.+Evans&title=The+DIALS+framework+for+integration+software"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWatermana%26aufirst%3DD.%2BG.%26aulast%3DWinterb%26aufirst%3DG.%26aulast%3DParkhurstb%26aufirst%3DJ.%2BM.%26aulast%3DFuentes-Monterob%26aufirst%3DL.%26aulast%3DHattnec%26aufirst%3DJ.%26aulast%3DBrewsterc%26aufirst%3DA.%26aulast%3DSauterc%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DThe%2520DIALS%2520framework%2520for%2520integration%2520software%26jtitle%3DCCP4%2520Newsl.%2520Protein%2520Crystallogr.%26date%3D2013%26volume%3D49%26spage%3D16%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. In  <i>Methods in Enzymology: Macromolecular Crystallography, part A</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C.W.,  Jr.</span>, <span class="NLM_string-name">Sweet, R. M.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=C.W.+Carter&author=R.+M.+Sweet&title=Methods+in+Enzymology%3A+Macromolecular+Crystallography%2C+part+A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMethods%2520in%2520Enzymology%253A%2520Macromolecular%2520Crystallography%252C%2520part%2520A%26aulast%3DCarter%26aufirst%3DC.W.%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lh4NB5sKoQOyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgOhW3fh5HriQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lgOhW3fh5HriQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7E" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7E','PDB','6C7E'); return false;">PDB: 6C7E</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7D','PDB','6C7D'); return false;">PDB: 6C7D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7F','PDB','6C7F'); return false;">PDB: 6C7F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7I','PDB','6C7I'); return false;">PDB: 6C7I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7G','PDB','6C7G'); return false;">PDB: 6C7G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7J','PDB','6C7J'); return false;">PDB: 6C7J</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i64"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00713">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_15155"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00713">10.1021/acs.jmedchem.8b00713</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv">CSV</a>)</p></li><li><p class="inline">Activity data for all compounds in the matrix, computational details (lowest energy conformation, tSA calculation, RMSD, and nonadditivity values), and protein crystallographic data statistics tables (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_001.csv">jm8b00713_si_001.csv (2.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00713/suppl_file/jm8b00713_si_002.pdf">jm8b00713_si_002.pdf (432.08 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Crystal structures of PDE2 in complex with <b>19</b>, <b>23</b>, <b>31</b>, <b>39</b>, <b>43</b>, and <b>46</b> have been deposited in the RCSB Protein Data Bank under the accession codes of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7E">6C7E</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7D">6C7D</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7I">6C7I</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7F">6C7F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7G">6C7G</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C7J">6C7J</a>, respectively.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00713&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00713%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00713" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a4629b1d22f0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
