
<html lang="en"     class="pb-page"  data-request-id="8db84959-2994-431a-a93a-0da033d595fd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b01887;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-5;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Cofactor Analogues as Active Site Probes in Lysine Acetyltransferases" /></meta><meta name="dc.Creator" content="Roman&#xA;P.  Simon" /></meta><meta name="dc.Creator" content="Tobias  Rumpf" /></meta><meta name="dc.Creator" content="Vaida  Linkuviene" /></meta><meta name="dc.Creator" content="Daumantas  Matulis" /></meta><meta name="dc.Creator" content="Asifa  Akhtar" /></meta><meta name="dc.Creator" content="Manfred  Jung" /></meta><meta name="dc.Description" content="Lysine acetyltransferases (KATs, also termed histone acetyltransferases, HATs) catalyze the acetylation of substrate lysine residues by employing the cofactor acetyl-coenzyme A (AcCoA), thereby pro..." /></meta><meta name="Description" content="Lysine acetyltransferases (KATs, also termed histone acetyltransferases, HATs) catalyze the acetylation of substrate lysine residues by employing the cofactor acetyl-coenzyme A (AcCoA), thereby pro..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 20, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01887" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01887" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01887" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01887" /></link>
        
    
    

<title>Cofactor Analogues as Active Site Probes in Lysine Acetyltransferases | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01887" /></meta><meta property="og:title" content="Cofactor Analogues as Active Site Probes in Lysine Acetyltransferases" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0010.jpeg" /></meta><meta property="og:description" content="Lysine acetyltransferases (KATs, also termed histone acetyltransferases, HATs) catalyze the acetylation of substrate lysine residues by employing the cofactor acetyl-coenzyme A (AcCoA), thereby providing a dynamic control mechanism of protein function. Because of their major involvement in cell development and homeostasis, small-molecule modulators of KAT activity are urgently needed to assess their therapeutic potential and for probing their underlying biology. Recent advances in the field suggest that targeting the cofactor binding site represents a promising strategy for identifying potent and selective ligands. Here, we present the synthesis of two functional cofactor-based chemical probes and their usage as mechanistic tools in a broadly applicable assay platform. A fluorescence polarization (FP)-based binding assay was combined with biolayer interferometry competition analysis and a FP competition activity immunoassay to enable easy, reliable, and profound evaluation of ligands that target the KAT cofactor binding site." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01887"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01887">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01887&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01887&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01887&amp;href=/doi/10.1021/acs.jmedchem.8b01887" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2582-2597</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01878" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01927" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Cofactor Analogues as Active Site Probes in Lysine Acetyltransferases</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Roman P. Simon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roman P. Simon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Sciences, University of Freiburg, Albertstraße 25, 79104 Freiburg im Breisgau, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roman%0AP.++Simon">Roman P. Simon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tobias Rumpf</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tobias Rumpf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chromatin Regulation, Max-Planck-Institute of Immunobiology and Epigenetics, Stuebeweg 51, 79108 Freiburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tobias++Rumpf">Tobias Rumpf</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vaida Linkuviene</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vaida Linkuviene</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biothermodynamics and Drug Design, Institute of Biotechnology, Life Sciences Center, Vilnius University, Saulėtekio 7, 10257 Vilnius, Lithuania</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vaida++Linkuviene">Vaida Linkuviene</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daumantas Matulis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daumantas Matulis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biothermodynamics and Drug Design, Institute of Biotechnology, Life Sciences Center, Vilnius University, Saulėtekio 7, 10257 Vilnius, Lithuania</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daumantas++Matulis">Daumantas Matulis</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6178-6276" title="Orcid link">http://orcid.org/0000-0002-6178-6276</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Asifa Akhtar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Asifa Akhtar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chromatin Regulation, Max-Planck-Institute of Immunobiology and Epigenetics, Stuebeweg 51, 79108 Freiburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Asifa++Akhtar">Asifa Akhtar</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Manfred Jung</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Manfred Jung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Sciences, University of Freiburg, Albertstraße 25, 79104 Freiburg im Breisgau, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#f29f939c94809796dc98879c95b2829a93809f93889b97dc879c9bdf9480979b90878095dc9697"><span class="__cf_email__" data-cfemail="016c606f677364652f6b746f6641716960736c607b68642f746f682c67736468637473662f6564">[email protected]</span></a>. Phone: +497612034896. Fax: +497612036321.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Manfred++Jung">Manfred Jung</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6361-7716" title="Orcid link">http://orcid.org/0000-0002-6361-7716</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01887&amp;href=/doi/10.1021%2Facs.jmedchem.8b01887" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2582–2597</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 20, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 November 2018</li><li><span class="item_label"><b>Published</b> online</span>20 February 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 March 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01887" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01887</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2582%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRoman%250AP.%2BSimon%252C%2BTobias%2BRumpf%252C%2BVaida%2BLinkuviene%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D5%26contentID%3Dacs.jmedchem.8b01887%26title%3DCofactor%2BAnalogues%2Bas%2BActive%2BSite%2BProbes%2Bin%2BLysine%2BAcetyltransferases%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2597%26publicationDate%3DMarch%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01887"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1049</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01887" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Cofactor Analogues as Active Site Probes in Lysine Acetyltransferases&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Roman\nP.&quot;,&quot;last_name&quot;:&quot;Simon&quot;},{&quot;first_name&quot;:&quot;Tobias&quot;,&quot;last_name&quot;:&quot;Rumpf&quot;},{&quot;first_name&quot;:&quot;Vaida&quot;,&quot;last_name&quot;:&quot;Linkuviene&quot;},{&quot;first_name&quot;:&quot;Daumantas&quot;,&quot;last_name&quot;:&quot;Matulis&quot;},{&quot;first_name&quot;:&quot;Asifa&quot;,&quot;last_name&quot;:&quot;Akhtar&quot;},{&quot;first_name&quot;:&quot;Manfred&quot;,&quot;last_name&quot;:&quot;Jung&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;2582-2597&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01887&quot;},&quot;abstract&quot;:&quot;Lysine acetyltransferases (KATs, also termed histone acetyltransferases, HATs) catalyze the acetylation of substrate lysine residues by employing the cofactor acetyl-coenzyme A (AcCoA), thereby providing a dynamic control mechanism of protein function. Because of their major involvement in cell development and homeostasis, small-molecule modulators of KAT activity are urgently needed to assess their therapeutic potential and for probing their underlying biology. Recent advances in the field suggest that targeting the cofactor binding site represents a promising strategy for identifying potent and selective ligands. Here, we present the synthesis of two functional cofactor-based chemical probes and their usage as mechanistic tools in a broadly applicable assay platform. A fluorescence polarization (FP)-based binding assay was combined with biolayer interferometry competition analysis and a FP competition activity immunoassay to enable easy, reliable, and profound evaluation of ligands that target the KAT cofac&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01887&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01887" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01887&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01887" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01887&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01887" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01887&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01887&amp;href=/doi/10.1021/acs.jmedchem.8b01887" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01887" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01887" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01887&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01887%26sid%3Dliteratum%253Aachs%26pmid%3D30785747%26genre%3Darticle%26aulast%3DSimon%26date%3D2019%26atitle%3DCofactor%2BAnalogues%2Bas%2BActive%2BSite%2BProbes%2Bin%2BLysine%2BAcetyltransferases%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D5%26spage%3D2582%26epage%3D2597%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291912" title="Fluorescence">Fluorescence</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292601" title="Probes">Probes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/jmcmar.2019.62.issue-5/20190314/jmcmar.2019.62.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Lysine acetyltransferases (KATs, also termed histone acetyltransferases, HATs) catalyze the acetylation of substrate lysine residues by employing the cofactor acetyl-coenzyme A (AcCoA), thereby providing a dynamic control mechanism of protein function. Because of their major involvement in cell development and homeostasis, small-molecule modulators of KAT activity are urgently needed to assess their therapeutic potential and for probing their underlying biology. Recent advances in the field suggest that targeting the cofactor binding site represents a promising strategy for identifying potent and selective ligands. Here, we present the synthesis of two functional cofactor-based chemical probes and their usage as mechanistic tools in a broadly applicable assay platform. A fluorescence polarization (FP)-based binding assay was combined with biolayer interferometry competition analysis and a FP competition activity immunoassay to enable easy, reliable, and profound evaluation of ligands that target the KAT cofactor binding site.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78430" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78430" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The reversible acetylation of lysine residues has been identified as a means of fundamental regulation of protein and genome function, which is conserved from prokaryotes to mammals.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The dynamic acetylation equilibrium within a cell is balanced by lysine acetyltransferases (KATs) and lysine deacetylases (KDACs).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Although originally identified as modifiers of histone proteins (HATs and HDACs), the enzymatic activity of these enzymes has also been demonstrated for a variety of nonhistone substrates, underlining their important role in cell physiology.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> All KATs employ the cofactor acetyl-coenzyme A (AcCoA (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>)) to transfer an acetyl group to the ε-N moiety of lysine residues of substrate enzymes, releasing the formed acetamide and CoA. Although AcCoA is the general KAT cosubstrate, acylation activity employing other short-chain acyl-CoAs has been demonstrated for some KAT enzymes.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a> On the basis of sequence similarity and catalytic mechanism, the major KAT activity containing proteins identified in humans are grouped into three distinct families, named the GNAT (Gcn5 related <i>N</i>-acetyltransferase) family (including Gcn5,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> pCAF,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> HAT1,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> ATAT-1<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a>), the metazoan specific p300/CBP family (including p300<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and CBP<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a>), and the MYST family (including MOF,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> MOZ, MORF,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Tip60<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and HBO1<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a>). Multiple other enzymes have been reported to catalyze protein lysine acetylation among which are yeast homologues of mammalian KATs, nonstructurally related proteins, and poorly characterized KAT candidates.<a onclick="showRef(event, 'ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21">(18−21)</a> The catalytic domains of KAT enzymes share high sequence homology within a distinct family and little to no sequence similarity to other families. These core domains are generally flanked by multiple other protein modules, which directly or indirectly contribute to substrate specificity and efficacy of the acetylation reaction. The functional consequences of enzymatic lysine acetylation as a post-translational protein modification are attributable to multiple mechanisms including recruitment of acetyl lysine binding proteins, protein stabilization, autoacetylation, and chromatin relaxation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Dysregulation of these processes as a consequence of KAT aberration have been causally correlated with a number of malignancy phenotypes such as neurodegenerative disease,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> viral and parasitic infections,<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> inflammation,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> cancer,<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27−29)</a> and metabolic disorders.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Mutations in gene sequence, changes in expression levels, and fusion proteins as a consequence of chromosomal translocation represent possible causes for the dysregulated KAT activity in such diseases. Because many KATs associate with effector proteins under physiological conditions, it has been challenging to dissect whether a certain biological effect is attributable to their enzymatic activity or to their scaffolding function in a cellular context.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> An appropriate way to encounter this problem is the use of potent and specific small-molecule inhibitors to assess the biological consequences and therapeutic potential of KAT inhibition. Despite major efforts toward the identification and development of such compounds, only two drug-like inhibitors for p300/CBP<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> (KAT3a/b) and MOZ/MORF<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (KAT6a/b), respectively, were made available until to date.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> Early approaches to this topic focused on bisubstrate mimics via covalent connection of the physiological cofactor to substrate peptides (acetonyl-CoA, Lys-CoA, and H3-CoA-20; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> While these compounds suffer from poor cell permeability and a complex structure, they exhibit potent and selective KAT inhibition. The concept was adopted for the generation of functionalized chemical probes mainly for the profiling of KAT substrates and for chemoproteomic applications.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Selected examples are depicted in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>: the reactive compound desthiobiotin-sulfone-CoA was used for active site cysteine labeling of several acetyltransferases,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> the cofactor surrogates <b>1a</b> and <b>1b</b> led to functional labeling of KAT substrates,<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> and the bisubstrate inhibitor-based probe <b>1c</b> equipped with an biotin affinity handle was used for selective enrichment of KATs from crude cell lysates.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> In the present article, we describe our adaptation of the bisubstrate strategy to generate and characterize functionalized cofactor-based probes of reduced structural complexity to assemble a versatile and broadly applicable assay platform. The catalytic domain of pCAF (KAT2B) was used to develop a fluorescence polarization (FP)-based displacement assay supplemented with a biolayer interferometry (BLI) assay that allows for the measurement of real time binding data. Both assays were then transferred to p300 (KAT3B) and MOF (KAT8) to show the versatility of the platform. These test systems were combined with a FP competition activity immunoassay to enable easy, reliable, and profound screening, characterization, and profiling of candidate compounds that target the KAT catalytic domain.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. General structures of selected CoA-derived metabolites, KAT inhibitors, and chemical probes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04582" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04582" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Probe Design and Synthesis</h3><div class="NLM_p">Although there is a lot of data published on the inhibitory properties of cofactor analogues and bisubstrate inhibitors, comparably little is known about the thermodynamic binding affinities of respective compounds. <i>K</i><sub>D</sub> values of 0.64 ± 0.12 and 8.7 ± 1.2 μM have been published for AcCoA binding to pCAF and MOF, respectively.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> For p300, an affinity constant of 0.74 ± 0.06 μM for the interaction with the nonhydrolyzable reference inhibitor acetonyl-CoA has been reported.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> While it has been shown that extensive modification of the peptide part of bisubstrate inhibitors can result in better inhibitory potency and KAT specificity, we sought for a way to generate CoA analogues with a broader scope of KAT interactions to expand the applicability of these probes. We therefore conceptually reduced the peptide part to a single lysine-mimicking aminohexanoic acid residue that bridges the CoA part of the probe with a functional label (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Immobilized acetonyl-CoA has previously been used for affinity enrichment of KATs originating from different enzyme families.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> We hypothesized that such functionally labeled CoA analogues would exhibit sufficiently high binding affinities to be used as chemical tool compounds in screening and profiling of cofactor competitive ligands. The synthesis of our probes was started with the assembly of a bifunctionalized linker, which was subsequently coupled to CoA and the functional label (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Therefore, Boc-protected 6-aminohexanoic acid (<b>2</b>) was condensed with propargylamine to the alkynylated intermediate <b>3</b> using standard amide bond formation. Deprotection followed by chloroacylation in DMF yielded the bifunctionalized linker <b>4</b>. We next assembled the fluorescence tag by submitting amine <b>5</b> to a 1<i>H</i>-imidazole-1-sulfonyl azide-promoted diazotransfer to generate the azide <b>6</b>, which was treated with oxalyl chloride in dichloromethane in the presence of catalytic amounts of DMF. The resulting acyl chloride was coupled to stoichiometric amounts of 6-aminofluorescein to yield the fluorescence tag <b>7</b>. Finally, alkyl linker <b>4</b> was converted to the respective probes. For the fluorescence-labeled derivative, <b>4</b> and <b>7</b> were combined in a Cu-catalyzed Huisgen cycloaddition to furnish compound <b>8</b>, which was further reacted with CoA in a S<sub>N</sub>2 reaction to generate the probe <b>11</b>. In the case of the affinity-labeled probe, nucleophilic substitution of the alkyl halide was carried out first (<b>9</b>), followed by the cycloaddition reaction to yield the desired compound <b>12</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). In the next step, we selected pCAF as a prototypical KAT for probe validation and assay development, because of its high cofactor binding affinity and well-defined binding mode. The catalytic domain of pCAF fused to a <i>N</i>-terminal His-tag (pCAF(493-658) was expressed and purified from <i>Escherichia coli</i> cells to provide sufficient amounts of protein. To test whether our compounds are capable to bind to the investigated KAT proteins, a pull-down experiment was carried out. Streptavidin-coated magnetic beads were used in combination with <b>12</b> to individually capture recombinant proteins from solutions in the presence of BSA (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S1). Control experiments containing either 100 μM acetonyl-CoA or 50 μM biotin were included for every KAT. The SDS-gel analysis of the isolated fractions showed successful pull-down of all three KAT proteins, which is indicative for effective binding of <b>12</b> to the tested enzymes. Next, we aimed at quantifying the binding affinities of the probes. Isothermal titration calorimetry (ITC) measurements of pCAF(493-658) interacting with the physiological cofactor AcCoA, reference inhibitor acetonyl-CoA, fluorescence labeled probe <b>11</b>, or affinity-labeled probe <b>12</b> yielded <i>K</i><sub>D</sub> values of 0.77 ± 0.17, 0.71 ± 0.31, 0.75 ± 0.36, and 1.39 ± 0.23 μM, respectively (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S2). These results are in good agreement with the previously published binding affinity for AcCoA (0.64 ± 0.12 μM) and demonstrate the possibility to functionally label at the selected derivatization site whilst maintaining binding affinities comparable to the physiological ligand. With the probes <b>11</b> and <b>12</b> validated as efficient binders of three different KAT proteins, we then turned our attention to the applications of the tool compounds.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of new functionally labeled cofactor analogues <b>11</b> and <b>12</b> from this study. Labels are colored in orange and lysine mimicking linkers are depicted in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Cofactor-Based Probes<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions (a) propargylamine, HBTU, DIPEA, DCM, rt, overnight; (b) TFA/DCM 1:1, 1 h then chloroacetyl chloride, DMF, 0 °C to rt, 1 h; (c) 1<i>H</i>-imidazole-1-sulfonyl azide × HCl, CuSO<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, MeOH, rt, overnight; (d) 1. (COCl)<sub>2</sub>, cat. DMF, DCM, rt, 5 h; 2. 6-aminofluorescein, NaHCO<sub>3</sub>, THF, rt, overnight; (e) coenzyme A, 20% ethanol in 0.1 M TEAB buffer pH 8, 0 °C to rt, 2 h; (f): alkyl azide, TBTA, CuSO<sub>4</sub>, sodium ascorbate, <i>tert</i>-BuOH/H<sub>2</sub>O/DMF (1:1:1), rt, 12 h. DIPEA = <i>N</i>,<i>N</i>-diisopropylethylamine, rt = room temperature, TEAB = triethylammonium bicarbonate, TBTA = Tris[(1-benzyl-1<i>H</i>-1,2,3-triazol-4-yl)methyl]amine.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> FP Competition Assay</h3><div class="NLM_p">The labeled CoA analogue <b>11</b> was used as a fluorescent tracer to set up a competition binding the FP assay. As the degree of polarization is inversely proportional to the molecular mobility and therefore correlates with the size of the fluorescent species, the polarization signal increases with the fraction of <b>11</b> bound to the receptor (here KAT protein). In a setup with fixed concentrations of <b>11</b> and KAT protein, competing test ligands reduce the fraction of tracer that is bound to the receptor, which results in a decrease of measured mP values (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). To assess the optimal probe concentration, we prepared a serial dilution of <b>11</b> in assay buffer covering a concentration range of 3.8–30 nM in 20 μL volume and measured the fluorescence intensities (FIs) in the <i>p</i>- and <i>s</i>-plane. The resulting values were plotted against probe concentrations to show a linear increase of FI for both planes (<i>R</i><sup>2</sup> = 0.999 and 0.999) and a stable ratio of FI values over the entire concentration range (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S3a). A probe concentration of 8 nM was selected for further experiments, as this amount of <b>11</b> gave FI values that exceeded the values measured for buffer controls five times, which resulted in a stable and robust FP signal. We next performed a saturation binding experiment by titrating a dilution series of pCAF(493-658) protein to <b>11</b> in the presence or without a saturating concentration of a reference inhibitor (200 μM acetonyl-CoA). To control for intrinsic protein fluorescence, the protein dilution series was also incubated without a fluorescent tracer. The FIs for <i>p</i>-plane and <i>s</i>-plane were then measured from all wells and values from the protein fluorescence controls were subtracted from tracer-containing wells for every pCAF(493-658) protein concentration. From the resulting FIs, the mP values were calculated. From reference inhibitor containing wells, the nonspecific portion of measured FP signals was determined and values were corrected accordingly. With the resulting signals for specific binding of <b>11</b> to pCAF(493-658) we were able to calculate the affinity constant of the interaction, which was determined to be <i>K</i><sub>D</sub> = 824 ± 40 nM by nonlinear regression using a one-site binding fit (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). This value is in excellent agreement with our results from ITC measurements and sufficiently low to use <b>11</b> as a fluorescent tracer in FP applications. Further, by calculating total FIs, we were able to show that there is no substantial fluorescence-quenching event evident upon binding of tracer to the protein (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S3b). To test the competitive displacement of <b>11</b> from binding to pCAF(493-658), we then titrated acetonyl-CoA to mixtures containing fixed concentrations of protein and tracer (400 and 8 nM, respectively) and measured the FP signal at different time points. The relative competition compared with control wells was calculated for every ligand concentration, and resulting values were plotted against acetonyl-CoA concentrations. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b shows the results from this experiment after nonlinear curve fitting, demonstrating a stable dose–response at all measured time-points. From obtained IC<sub>50</sub> values, an affinity constant for the interaction of acetonyl-CoA to pCAF(493-658) of <i>K</i><sub>D</sub> = 592 ± 66 nM was calculated, which again is in good agreement with the value determined by ITC. With optimized assay parameters of 400 nM pCAF(493-658), 8 nM <b>11</b>, and 30 min incubation time, we investigated the influence of the increasing DMSO content on our system. Therefore, we measured positive controls, containing all assay components and increasing amounts of DMSO (v/v), and compared them to negative controls containing 50 μM acetonyl-CoA additionally (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S3c). The signal window of the assay did not decrease by more than 10% for concentrations up to 8% (v/v) of the organic solvent. We selected a standard DMSO concentration of 5% (v/v) for the screening of ligands targeting the CoA binding site and evaluated the reproducibility and robustness of the assay by preparing 32 positive and 32 negative (containing 50 μM acetonyl-CoA) controls on a single 384-well plate. From the FP signals of these measurements, the <i>Z</i>′-factor for the assay was calculated as described by Zhang et al. as a surrogate for the assay quality (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S3d).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> With a <i>Z</i>′-factor of 0.81, our assay meets the requirements for a stable and robust assay (<i>Z</i>′ > 0.5) and facilitates the screening for competitive ligands targeting the cofactor binding site.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Schematic depictions of the FP and BLI assay and response plots of pCAF(493-658) within these assay systems. (a) Setup of the FP assay. The FP signal is dependent on the rotational freedom of the fluorescent tracer, which is influenced by the presence of competing ligands. (b) Saturation binding curve of pCAF(493-658) binding to 8 nM of <b>11</b> and the dose–response curve of reference inhibitor acetonyl-CoA against 400 nM pCAF and 8 nM <b>11</b> after different time points. Data represent the mean and standard deviation of triplicate measurements. (c) Setup of the BLI assay. Compound <b>12</b> was immobilized on streptavidin coated sensors and the association and dissociation of enzyme in the presence of possible competitors is monitored by measuring the optical interference pattern of two light beams within the sensor. (d) Sensorgrams and saturation binding curve of a concentration series of pCAF(493-658) in the BLI assay. (e) Sensorgrams and dose–response curve of the reference inhibitor titration against 250 nM enzyme. Data represent the mean and standard deviation of triplicate measurements. Results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> BLI Binding Assay</h3><div class="NLM_p last">As a confirmatory biophysical assay, we chose BLI as a method to revalidate candidate ligands identified as competitors by the FP competition assay. BLI measures the interference pattern of white light that is reflected on a biocompatible surface immobilized on a sensor tip compared with a reference beam. Upon binding of biomolecules to the functionalized surface, the optical thickness of this layer increases, which results in a shift of the interference pattern that is expressed in nanometers. The increase of the BLI signal is proportional to the number of biomolecules bound to the functionalized surface and can be measured in real time, proving a means for the determination of kinetic data of the molecular interaction. In our approach, we used streptavidin-coated sensor tips to immobilize the biotinylated CoA analogue <b>12</b> onto the biosurface and investigate the binding of acetyltransferase protein to the immobilized probe (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). First, we determined the optimal loading density of <b>12</b> on the biosensors. Oversaturation of biosensors can cause steric hindrance and aggregation, whereas too little ligand immobilized can result in insufficient signal intensities. To assess a suitable loading concentration, a dilution series of <b>12</b> in assay buffer was prepared and the BLI signal increase during the loading step and subsequently for the binding phase, during which the loaded sensors were exposed to a 1 μM solution of pCAF(493-658), were monitored (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S4a,b). A probe concentration of 125 nM was found to give a linear signal increase in the loading phase without reaching saturation while the signal increase during the binding phase remained sufficiently high. Next, we decided to measure a <i>K</i><sub>D</sub> value for the interaction of pCAF(493-658) to immobilized <b>12</b>. Because global fitting of sensorgrams to a one site binding model for different protein concentrations was unprecise in the dissociation phase, we decided to use a saturation binding approach for the determination of the affinity constant (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S4c). Therefore, we incubated preloaded biosensors with a dilution series of protein concentrations spanning from 0.04 to 30 μM, which was supposed to cover a concentration range of approximately 0.05–20 times <i>K</i><sub>D</sub>. Exposure of a biotin-blocked biosensor to 30 μM pCAF(493-658) in assay buffer showed no significant increase in binding signal, suggesting the absence of unspecific interactions of the protein to the biosensor. BLI signals for the association and dissociation phases were recorded as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). Report points at the end of the association phase were then used to extract binding signals from the recorded sensorgrams for every protein concentration used. Plotting of these values against protein concentration followed by nonlinear curve fitting to a one site binding model gave a <i>K</i><sub>D</sub> value of 1.00 ± 0.28 μM, which is in good agreement with values determined by ITC and FP (1.39 ± 0.23 and 0.71 ± 0.08 μM, respectively). To test the feasibility of the system to evaluate compounds as competitive ligands, we tested the influence of increasing concentrations of reference inhibitor in protein solutions on the sensorgrams of the association step. A fixed concentration of 250 nM pCAF(493-658) was incubated with different amounts of acetonyl-CoA and the association curves were recorded on the BLI instrument (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e). We observed a concentration-dependent decrease in binding signal with a maximum that almost reached blank (loaded sensors in assay buffer) level. Comparing the BLI signals at report points set at the end of the association phase allowed us to quantify the degree of competition for every acetonyl-CoA concentration and plotting of respective nm values against ligand concentrations gave an IC<sub>50</sub> value of 0.78 ± 0.10 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e). This value is very close to the <i>K</i><sub>D</sub> values for the interaction of pCAF(493-658) with acetonyl-CoA determined by other methods and shows the utility of the assay for the validation of candidate ligands.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Applicability of the Assays to Other Family KATs</h3><div class="NLM_p">To test the applicability of the FP assay and the BLI assay to other KAT family members, we chose two representative KATs, p300(p300(1284-1673)) of the p300/CBP family and MOF(MOF(174-449)) of the MYST family and subjected them to the same cycles of assay optimization as described for pCAF (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). For p300(1284-1673), slow but tight binding to the fluorescent probe was observed and in comparison to pCAF, longer incubation times were needed to reach equilibrium. The FP saturation binding experiment yielded a <i>K</i><sub>D</sub> value of 396 ± 31 nM for binding to <b>11</b>, which allowed calculating an affinity constant of <i>K</i><sub>D</sub> = 653 ± 56 nM for the reference inhibitor from the dose–response curve. This value is close to the published value of 780 ± 60 nM. The slow binding of p300(1284-1673) was also observable in the BLI assay, and the protocol had to be adjusted accordingly. Instead of loading the biosensor with <b>12</b> first, we incubated the protein with a 125 nM probe alone or in the presence of competing ligands for 30 min and measured the association to the streptavidin-coated sensor from these solutions. While this does not allow for deducing any kinetic data from the association curves, the new protocol proved to be usable to evaluate ligands competitive to <b>12</b>. Titrating acetonyl-CoA in this assay resulted in an IC<sub>50</sub> of 779 ± 30 nM. Notably, the tight binding of p300(1284-1673) to <b>12</b> was reflected in only very little dissociation of the protein from the sensor tip during the dissociation step (<i>k</i><sub>off</sub> = 2.1 × 10<sup>–4</sup>•s<sup>–1</sup>). For MOF(174-449), the saturation binding experiment yielded a <i>K</i><sub>D</sub> value of 1.39 ± 0.11 μM for binding to <b>11</b>. The increase in the FP signal in this experiment was less pronounced compared with the other tested KAT proteins. The affinity constant for MOF(174-449) binding to acetonyl-CoA was determined from this experiment and resulted in <i>K</i><sub>D</sub> = 1.58 ± 0.14 μM, which is distinctly lower than the published value for the protein cofactor interaction (8.7 ± 1.2 μM). Because we were not able to measure BLI signals at sufficiently high protein concentrations to reach saturation binding, we determined a <i>K</i><sub>D</sub> = 4.13 ± 0.4 μM for MOF(174-449) binding to <b>12</b> by applying a global one site binding fit to the sensorgrams of a protein dilution series (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S4d). Titrating the reference inhibitor against 4 μM protein in this assay resulted in an IC<sub>50</sub> of 1.35 ± 0.18 μM for acetonyl-CoA. Together, these experiments demonstrate the broad applicability of <b>11</b> and <b>12</b> as chemical tool compounds for KAT enzymes and the versatility of our assays platform (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Results from the method transfer to other KATs. (a) Titration curves of p300(1284-1673) and MOF(174-449) against 8 nM <b>11</b> and dose–response curves of reference inhibitor against both enzymes at optimized assay concentrations of 120 and 800 nM, respectively. (b) Sensorgrams and corresponding dose–response curves from BLI experiments measuring the concentration dependent competition of acetonyl-CoA with p300(1284-1673) and MOF(174-449) for binding to immobilized <b>12</b>. Data represent the mean and standard deviation of duplicate measurements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. pCAF-Binding Constants Determined for the Reference Inhibitor Acetonyl-CoA and the Probes</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="±"><i>K</i><sub>D</sub> (IC<sub>50</sub>) ± SD [μM]</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">method</th><th class="colsep0 rowsep0" align="center" char="±">acetonyl-CoA</th><th class="colsep0 rowsep0" align="center" char="±">Fl-hex-CoA (<b>11</b>)</th><th class="colsep0 rowsep0" align="center" char="±">biotin-PEGhex-CoA (<b>12</b>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ITC</td><td class="colsep0 rowsep0" align="char" char="±">0.71 ± 0.31</td><td class="colsep0 rowsep0" align="char" char="±">0.75 ± 0.36</td><td class="colsep0 rowsep0" align="char" char="±">1.39 ± 0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BLI<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">(0.78 ± 0.10)</td><td class="colsep0 rowsep0" align="char" char="±">(0.98 ± 0.10)</td><td class="colsep0 rowsep0" align="char" char="±">1.00 ± 0.28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FP</td><td class="colsep0 rowsep0" align="char" char="±">0.59 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">0.82 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.71 ± 0.08</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Values determined by BLI in the competition binding experiment were not recalculated to <i>K</i><sub>D</sub> values but reported as the measured IC<sub>50</sub> in parenthesis.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> FP Competition Activity Immunoassay</h3><div class="NLM_p">To complement the biophysical test systems, we sought for a nonradioactive assay to measure the activity of the investigated acetyltransferases under in vitro conditions. We also aimed at an approach that does not require the use of labeled substrates or reactive detection reagents, which can lead to data artefacts such as high background signals or false positive screening hits. Because of our good experience with FP, we hypothesized that it would be possible to design an assay that measures the enzyme activity-dependent competition of the reaction product to a fluorescence-labeled tracer for binding to a specific antibody, as it has been reported for kinases (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> This would allow for free scaling of substrate and cofactor concentrations and the use of unlabeled and more physiological peptide or protein substrates. Although the preferred H3 acetylation site for pCAF has been reported to be H3K14, we had obtained good signal to background ratios using a polyclonal anti-acH3K9 antibody (#39137, Active Motif) in our biochemical assays (data not shown). As a fluorescent tracer, we used a peptide that contains the first 14 amino acids of the histone H3 N-terminal sequence with a rhodamine label attached to the side chain of K 14 and an acetylated K9 moiety (acH3K9-Rh). For MOF(174-449) and p300(1284-1673), we expected a broader spectrum of acetylation products. Therefore, a tetra-acetylated H4(1-23) peptide with a C-terminal fluorescein label was used as a tracer (acH4-Fl). The exact sequence of peptides can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>. The fluorescent properties of acH3K9-Rh and acH4-Fl were investigated as described for <b>11</b> and tracer concentrations of 8 and 3 nM were selected, respectively (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S5a). In the next step, different antibodies were titrated to solutions containing the indicated concentration of acH3K9-Rh or acH4-Fl and 10 μM of H3(1-21) peptide or H4(1-23) peptide, respectively, in 25 μL volume on 384-well microtiter plates and the changes in FP signals were recorded. Satisfactorily, we identified two antibodies that were capable to bind their epitope in solution, thereby inducing a concentration-dependent FP signal increase (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). Our selected anti-acH3K9 antibody caused an almost linear increase in mP values up to the highest concentration tested (5 μg/mL), which corresponds to a 1:200 dilution in a final assay volume of 25 μL. For acH4-Fl, the only antibody with a good response in this experiment was a monoclonal anti-acH4 antibody (#177790, abcam) directed against tetra-acetylated H4 (K5, 8, 12, 16). Addition of anti-acH4 to the tracer peptide resulted in a steep increase of the FP signal that leveled around 160 mP for antibody concentrations exceeding 0.5 μg/mL (1:700 in 25 μL volume). The feasibility of the setup to measure the KAT activity was then investigated for established tracer/receptor pares (8 nM acH3K9-Rh/5 μg/mL anti-acH3K9 and 3 nM acH4-Fl/0.4 μg/mL anti-acH4, respectively). For successful activity measurements, the product of the enzymatic reaction has to be competitive to the tracer peptide for binding to the antibody. If the system fulfills this requirement, the degree of competition can be calculated from recorded FP values and is proportional to enzyme activity. Concentration series of enzymes were incubated with a substrate peptide or protein (50 μM H3(1-21) for pCAF(493-658), 5 μM full length H4-His for MOF(174-449) and 5 μM H4(1-23) for p300(1284-1673)) and 20 μM AcCoA for 60 min. Enzymatic reactions were then stopped by adding 2.5 μL of tracer solution containing 500 μM acetonyl-CoA followed by a short incubation step. Finally, 2.5 μL of antibody solution in assay buffer was dispensed into every well, and after an additional 30 min incubation step, FP signals were recorded and competition values calculated. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b shows a clear increase in competition with increasing enzyme concentrations, which strongly suggests a concentration-dependent displacement of the tracer from the receptor by the product of enzymatic turnover. Further, the use of full-length H4 histone protein as a substrate demonstrates the capability of the assay to use physiological protein substrates. Enzyme concentrations from the ends of the linear portions of the enzyme progression curves were selected for further experiments (30 nM pCAF(493-658), 50 nM MOF(174-449), and 15 nM p300(1284-1673)). We then determined the assay-specific <i>K</i><sub>m</sub> values for AcCoA and selected screening concentrations that, again, were at the upper end of the linear portion of the Michaelis–Menten plots to preserve a sufficient assay window (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S5b). With the defined assay parameters, dose–response curves for the reference inhibitor were recorded (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S5c) and <i>K</i><sub>i</sub> values were calculated according to the method of Yung-Chi and Prusoff.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> We also determined the assay-specific <i>Z</i>′-factors (all >0.5) as a measure of assay robustness (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S5d). The exact results from our activity assay development and reference inhibitor validation are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Results from antibody and enzyme titrations. (a) Titration curves of anti-acH4-Ab and anti-acH3K9-Ab against 3 nM acH4-Fl and 8 nM acH3K9-Rh tracer peptide, respectively, after 30 min incubation. (b) Concentration–response curves for acetyltransferases after 60 min incubation with substrate (50 μM H3(1-21) for pCAF(493-658), 5 μM full length H4-His for MOF(174-449), 5 μM H4(1-23) for p300(1284-1673)), and 20 μM AcCoA. The degree of competition was measured after stopping the reaction by addition of the corresponding tracer/antibody pair. Data represent the mean and standard deviation of triplicate measurements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Schematic Depiction of the FP Competition Activity Immunoassay<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Substrate peptide is processed by the acetyltransferase in the course of the enzymatic reaction. After incubation, the reaction is stopped by addition of reference inhibitor and the tracer/receptor pair is added. Acetylated reaction product competes with a fluorescence labeled peptide (tracer) for binding to the modification specific antibody (receptor). The decrease in fluorescence polarization of the tracer peptide is proportional to its displacement from the antibody and therefore a measure for enzymatic turnover.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Results from FP Competition Activity Immonoassay Optimization</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">enzyme conc. [nM]</th><th class="colsep0 rowsep0" align="center">substrate (conc. [μM])</th><th class="colsep0 rowsep0" align="center">AcCoA conc. (<i>K</i><sub>m</sub>) [μM]</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>i</sub> ref. inhib. ± SD [μM]</th><th class="colsep0 rowsep0" align="center" char="."><i>Z</i>′-factor</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">His-pCAF<sub>(493-658)</sub></td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">H3(1-21) (50)</td><td class="colsep0 rowsep0" align="left">6 (2.6)</td><td class="colsep0 rowsep0" align="char" char="±">1.32 ± 0.42</td><td class="colsep0 rowsep0" align="char" char=".">0.56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOF<sub>(174-449)</sub></td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">H4-His (5)</td><td class="colsep0 rowsep0" align="left">10 (5.6)</td><td class="colsep0 rowsep0" align="char" char="±">2.83 ± 0.61</td><td class="colsep0 rowsep0" align="char" char=".">0.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">p300<sub>(1284-1673)</sub></td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">H4(1-23) (5)</td><td class="colsep0 rowsep0" align="left">4 (1.3)</td><td class="colsep0 rowsep0" align="char" char="±">0.88 ± 0.14</td><td class="colsep0 rowsep0" align="char" char=".">0.79</td></tr></tbody></table></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Versatility of the Assay Platform</h3><div class="NLM_p">With the evaluated tool compounds and optimized assay parameters, the utility of the assay platform for different applications was investigated.</div><div id="sec2_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Profiling of Acyl-CoA Variants</h4><div class="NLM_p">First, we followed observations by Montgomery et al.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and Ringel and Wolberger<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> who have reported inhibitory properties of metabolic acyl-CoAs (e.g., butyryl-CoA and several long-chain acyl-CoAs (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>)) on the GNAT family KAT hGcn5. To validate the role of the chain length and quantify the degree of interaction, a small set of acyl-CoA cofactor analogues were profiled for their binding properties to the closely related Gcn5 homologue pCAF. We included the enzymatic reaction product CoA (C-0), the physiological cofactor AcCoA (C-2), butyryl-CoA (C-4), lauroyl-CoA (C-12), aromatic benzoyl-CoA, and the more hydrophilic acyl-CoA variant hydroxybutyryl-CoA. A comparison of the competition profiles from FP and BLI assays showed a strong increase of interaction for the physiological cofactor compared with the enzymatic reaction product CoA (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a). Elongation of the carbon chain to a butyryl residue then resulted in a drop of competition to the CoA level. The same effect was observed for structurally more demanding acyl-CoA variants benzoyl-CoA and hydroxybutyryl-CoA. Further elongation to the C-12 derivative lauroyl-CoA then led to a strong increase of potency that exceeded the level of AcCoA. Recording the dose–response curves of acyl-CoA variants with increasing chain length in the FP assay allowed us to calculate the affinity constants of respective interactions from IC<sub>50</sub> values (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c). We then revalidated our findings by comparing the results from FP and BLI assays to the stabilizing effects (Δ<i>T</i><sub>m</sub>) of tested CoA analogues in a fluorescent thermal stability assay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b). <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> shows the results from these experiments and assigns lauroyl-CoA as a potent binder (<i>K</i><sub>D</sub> = 0.18 ± 0.01 μM) of pCAF(493-658) with a five times higher affinity than the physiological cofactor (<i>K</i><sub>D</sub> = 1.03 ± 0.09 μM). To test whether the high affinity also translates to inhibitory potency, lauroyl-CoA was evaluated in the FP competition activity immunoassay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>d). The compound inhibits pCAF(493-658) more potently (<i>K</i><sub>i</sub> = 0.78 ± 0.18 μM) than the reference inhibitor acetonyl-CoA (<i>K</i><sub>i</sub> = 1.32 ± 0.24 μM). Molecular docking to hGcn5 and crystallographic data of p300 have shown that the alkyl chain of CoA variants is positioned in the lysine binding site of the enzymes.<a onclick="showRef(event, 'ref42 ref45'); return false;" href="javascript:void(0);" class="ref ref42 ref45">(42,45)</a> Structural analysis of p300 further revealed the presence of a hydrophobic binding pocket located at the catalytic center of the protein, a structural feature absent in pCAF and MOF. This pocket was proposed to accommodate the alkyl portion of structurally more demanding acyl-CoAs after the substrate induced ligand rearrangement during the catalysis of the acyl transfer by p300.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> To compare the effects of acyl-CoAs on pCAF(493-658) with effects on p300(1284-1673), the ability of both enzymes to use acyl-CoA variants as cofactor for enzymatic turnover of a peptide substrate was tested in a thiol scavenging assay that detects the acyl-CoA hydrolysis product CoA (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S6a). As expected, p300, but not pCAF, is capable to use butyryl-CoA and hydroxybutyryl-CoA for enzymatic turnover, although the efficiency of turnover is decreased compared with AcCoA. In the presence of lauroyl-CoA, no activity could be detected within this assay. The profile of acyl-CoAs in the FP assay was then recorded for p300(1284-1673). Interestingly, in contrast to pCAF, the degree of competition for binding to p300 did not show an initial decrease with increasing chain length but increased continuously from C-2 to C-12 (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S6b). Dose–response analysis revealed that, despite being less effective cofactors, p300(1284-1673) binds butyryl-CoA and lauroyl-CoA with significantly higher affinities than the predominant cofactor AcCoA (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c). This observation is in agreement with inhibition data from the FP competition activity immunoassay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>d). We then tested whether the increased affinity of lauroyl-CoA could be attributable to additional interactions of the ligand with the peptide-binding site of the enzymes. Therefore, a peptide-binding assay (Alpha) was performed, which showed considerable competition of lauroyl-CoA but not of the reference inhibitor or probe <b>11</b>, for binding of His-tagged pCAF to the biotinylated H3(1-21) peptide (the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">Supporting Information</a>, Figure S6c). Together, our results propose that the comparatively high affinity of lauroyl-CoA for pCAF(493-658) and p300(1284-1673) is the result of additional hydrophobic interactions of the ligand with the substrate-binding site of the respective protein. In the case of pCAF, this site seems to be located distally to the lysine binding groove where the N- and C-terminal regions have been shown to form strong van der Waals contacts with the physiological substrates.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Results from acyl-CoA profiling and binding site mapping. (a) Comparison of FP and BLI assay results from acyl-CoA profiling against pCAF(493-658). (b) Thermal shift profiles of acyl-CoAs interacting with 4 μM pCAF(493-658). The melting temperature shifts compared with apo-pCAF are depicted in the insertion. (c) Dose–response curves for acyl-CoA variants in the FP assay (top: pCAF(493-658); bottom: p300(1284-1673)). The curves for butyryl-CoA are depicted in red. (d) Dose–response curves of lauroyl-CoA (solid lines) and reference inhibitor acetonyl-CoA (dashed lines) in the FP competition activity immunoassay for p300 (orange) and pCAF (green). (e) Competition profiles of cofactor fragments against all three KAT enzymes in the FP assay. All data represent the mean and standard deviation of triplicate (pCAF) or duplicate (p300 and MOF) measurements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. FP Assay Results from Acyl-CoA Profiling</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±"><i>K</i><sub>D</sub> (IC<sub>50</sub>) ± SD [μM]</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±">pCAF(493-658)</th><th class="colsep0 rowsep0" align="center" char="±">p300(1284-1673)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CoA-SH</td><td class="colsep0 rowsep0" align="char" char="±">9.01 ± 2.86</td><td class="colsep0 rowsep0" align="char" char="±">5.45 ± 0.57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AcCoA</td><td class="colsep0 rowsep0" align="char" char="±">1.03 ± 0.16</td><td class="colsep0 rowsep0" align="char" char="±">0.64 ± 0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">butyryl-CoA</td><td class="colsep0 rowsep0" align="char" char="±">7.05 ± 0.66</td><td class="colsep0 rowsep0" align="char" char="±">0.38 ± 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lauroyl-CoA</td><td class="colsep0 rowsep0" align="char" char="±">0.18 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">(0.08 ± 0.02)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">The IC<sub>50</sub> of lauroyl-CoA for p300 in the FP assay is too low to reliably calculate the affinity constant. Therefore, the IC<sub>50</sub> is reported here.</p></div></div></div></div><div id="sec2_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Probing the Cofactor Binding Site</h4><div class="NLM_p last">We were interested in the possibility of using <b>11</b> to identify structural features of the cofactor that are crucial for binding to the respective KAT enzyme. A comparison of such features for all our tested KATs could possibly reveal differences in protein cofactor interactions, which could prove useful in the design of more specific ligands targeting the CoA binding site. To do so, a set of chemical entities containing different structural motifs of the cofactor was assembled and each compound was tested at a fixed concentration of 1 mM against every KAT in the FP assay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>e). By this, we were also able to validate the assay for its applicability to fragment-based approaches. The results from this experiment are depicted in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. The strong interaction of CoA’s phosphate backbone with the cofactor binding site has previously been shown for pCAF,<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> p300,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and MOF<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> by crystallographic means and our findings accordingly show a pronounced decrease in competition for fragments lacking one or more phosphate moieties. For all tested KATs, the segmentation of the cofactor into two individual fragment portions, the pantetheine part and adenosine phosphate part, led to an almost complete loss of interaction. Only the adenosine 3′,5′-diphosphate fragment exhibited modest competition with the probe for binding to KAT proteins. For MOF, competition exceeding 20% was also observed for adenosine 5′-diphosphate (ADP), which was completely inactive against pCAF and p300. Unphosphorylated pantethein did not exhibit any significant competitive displacement of the fluorescent tracer, as it was proposed based on the findings of previous studies on the interaction of pantethein analogues with acetyltransferases.<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> A comparison of competition values for the three KAT enzymes shows an overall better amenability of MOF to CoA-fragment competition, which likely is due to the comparatively lower affinity of MOF(174-449) to <b>11</b>. In the case of p300(1284-1673), a strong effect on binding was observable for the 3′-phosphate residue, as the cofactor fragment lacking only this moiety (dephospho-CoA) was not able to promote full competition to <b>11</b>, even at a concentration as high as 1 mM (CoA FP-assay IC<sub>50</sub> = 3.0 ± 0.2 μM). Together, our comparison of fragment competition enables an interaction mapping of the cofactor binding site that is in good accordance with findings from structural biology approaches. These results suggest that the tested KAT enzymes share a strong dependence of binding potency on the presence and precise orientation of phosphate residues within the cofactor structure. Still, differences in the overall competition profile of cofactor fragments between the different KAT enzymes could be observed in our FP approach, demonstrating the applicability and versatility of <b>11</b> for this method.</div></div><div id="sec2_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Fragment-Based Ligand Screening</h4><div class="NLM_p">To show the feasibility of the assay platform to screen larger quantities of candidate ligands for binding to KAT protein, we performed a fragment-based approach. One thousand small molecule fragments from a commercially available library (rule of 3 compliant) were evaluated in the FP assay against pCAF(493-658). Compounds were first screened at 1 mM in duplicates, and a statistic cutoff of two times the mean standard deviation over all measurement was applied (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). In a second round, 62 initial hits were tested at three different concentrations (3, 1, and 0.3 mM). Fragments that gave at least 30% inhibition at 3 mM and a twofold increase in inhibition over the concentration range were considered as hits. Autofluorescent compounds were discarded. The remaining 27 compounds were then subjected to a fluorescent thermal stability assay, and destabilizing fragments (Δ<i>T</i><sub>m</sub> < −1 °C) were sorted out (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). Eventually, 14 fragments that showed a good response in the FP assay and no protein destabilizing effects were evaluated for their competitive effects at 1 mM in the BLI assay (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>c). Three fragments prevented binding of pCAF in this assay by more than 20%, and their structures are depicted in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>d. These fragments share the structural motif of a five- or six-membered aromatic heterocycle with a carboxylic acid residue in position 2, which indicates that we successfully enriched a certain chemotype that showed effective competition to our probes for binding to the cofactor binding site of pCAF(493-658) in two orthogonal assay systems.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Results from ligand screening. (a) Bar graph and cumulative distribution of the primary screening. The threshold was set to 2 × SD (σ) over all measurements. (b) Thermal shifts of screening hits at 1 mM compound concentrations. Destabilizing compounds were sorted out. (c) BLI sensorgrams of hits from the confirmatory assay. (d) Structures of the three fragment screening hits.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61442" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61442" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In the present study, we report the development of two cofactor-based chemical tool compounds, fluorescently labeled compound <b>11</b> and biotin labeled compound <b>12</b>, and their utilization in development of a biophysical assay platform for the evaluation of candidate ligands targeting the KAT cofactor binding site. The structures of these probes were deduced from the bisubstrate concept of covalently linking the products of enzymatic turnover, CoA, and an acetylated lysine mimic, via a nonhydrolyzable thioether linkage. By reducing the substrate-mimicking moiety to a simple aminohexanoic acid residue and coupling of the functional label to this site, the resulting compounds exhibit a broad applicability across KAT enzymes originating from different enzyme families, while maintaining sufficiently high binding affinities to be used in binding assays. While such tool compounds lack the cell permeability and selectivity to be used in biological systems, they provide a valuable means for in vitro evaluations. Recent reports of the first potent and selective small molecule inhibitors of p300/CBP<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and MOZ/MORF<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> activity, both targeting the cofactor binging site of the respective enzyme, underline the potential of our cofactor competition binding approach. The target compounds were synthesized via a convergent route with an intermediate clickable handle that enables easy functionalization with a label of choice. We used the fluorescence-labeled probe <b>11</b> to set up a FP-based competition assay for the evaluation of active site ligands. The relative increase in molecular volume upon binding and the conformational constraint of the fluorophore are the main drivers for the generation of a sufficient dynamic assay window. While larger peptide-based bisubstrate ligands could provide higher affinity and subtype selectivity, they are limited by their structural flexibility at possible label attachment sites and high molecular weights. The comparatively smaller size of <b>11</b> and the labeling in a position that places the fluorophore close to the lysine binding site facilitated its use as a tracer in FP applications with sufficient signal intensities. After successful implementation of the FP-assay, we optimized assay parameters for representative KAT enzymes from each of the three major families. The same was achieved with the affinity-labeled cofactor derivative <b>12</b> in a BLI-based setup. Utilizing the reference inhibitor acetonyl-CoA, these assays were then used to record thermodynamic and kinetic binding data, which is in excellent agreement with previously published values and data from ITC measurements, thus validating our probes and assays as sensitive and versatile tools for the identification and characterization of cofactor competitive ligands. The combination of binding assays with a system that measures enzyme activity provides an orthogonal means to assess the functional consequences of cofactor competition by candidate ligands. For this approach, we reported a nonradioactive FP competition activity immunoassay that does not rely on substrate labels, reactive detection reagents, or functional surfaces, but allows free scaling of substrate and cofactor concentrations and the use of whole protein substrates, which resemble a more physiological background. Although such assay formats have been reported for kinases, to our knowledge, this is the first report of a FP competition immunoassay measuring the KAT activity to date. The application of the entire assay platform was further validated by three different approaches. First, profiling of acyl-CoA variants revealed distinct binding affinity patterns dependent on the alkyl chain length of the acyl moiety for pCAF and p300. The potent binding of lauroyl-CoA to these enzymes was also shown to translate into inhibitory potency, and for pCAF, we were able to rationalize our findings by demonstrating competition of the long-chain acyl-CoA to the peptide substrate, suggesting additional interactions with the protein at the substrate binding site. Further, we used cofactor fragments to identify critical interaction sites and systematically tested structural motifs of the physiological ligand to reveal subtype selective characteristics. <b>11</b> and <b>12</b> were finally used to screen a commercially available fragment library in consecutive screening steps, and we identified three structurally analogous compounds that show profound and dose-dependent competition to the probes for binding to pCAF, whilst not showing any disruption of protein integrity or assay interference. Such screening hits provide excellent candidates for future cocrystallization and crystal soaking experiments to develop the initial hits into new lead-like candidates by structure-guided medicinal chemistry approaches.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Because the here reported tool compounds are likely promiscuous to other CoA utilizing enzymes, our biophysical approach is presumably transferable to other in vitro setups. In summary, we propose that our probes and assays are versatile and valuable tools for the in vitro evaluation of KAT characteristics, which will prove useful for the identification and profiling of new drug candidates targeting the cofactor binding site.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General</h3><div class="NLM_p last">Starting materials (chemicals) were purchased from commercial suppliers and used without any further purification. Solvents were used in p.a. quality and dried according to common procedures, if necessary. Thin-layer chromatography for reaction monitoring was performed with alumina plates coated with Merck silica gel 60 F254 (layer thickness: 0.2 mm) or Merck silica gel 60 RP-18 F254 (layer thickness: 0.2 mm) and analyzed under UV-light (254 nm). Flash column chromatography was performed with TELOS Flash-LL Silica Columns 60M (0.040–0.063 mm, 230–400 mesh) as a stationary phase on a Biotage Isolera One automated flash purification system with UV–vis detector. Size exclusion chromatography of compounds was conducted using Sephadex-G10 media and dest. H<sub>2</sub>O as the eluent. All yields were not optimized. NMR spectra were recorded using a Bruker AVANCE III HD Spectrometer (<sup>1</sup>H: 400 MHz, <sup>13</sup>C: 100 MHz) instrument. The spectra are referenced against the NMR solvent and are reported as follows: chemical shift δ (ppm), integration, multiplicity (s = singlet, d = doublet, dd = doublet of doublets, t = triplet, q = quartet, p = quintet, m = multiplet), coupling constant (<i>J</i> in Hz), and assignment. Mass spectra were recorded on a LCQ Advantage or Exactive device (Thermo Fisher Scientific, Waltham, MA) or an Expression CMS device (Advion, Ithaca, NY). Purity was determined by high-performance liquid chromatography (HPLC) analysis using an Agilent 1260 Infinity series system with a Synergi Kinetex XB-C18 100 Å, 250 × 4.6 mm as the analytical column and UV detection (λ = 210 nm (HPLC-method A) or 260 nm (HPLC-method B). A linear gradient of eluent A (water + 0.05% (v/v) TFA) and eluent B (ACN + 0.05% (v/v) TFA) of 0–4 min, <i>A</i> = 90%, <i>B</i> = 10%; 4–29 min, linear increase to <i>B</i> = 100%; 29–31 min, <i>B</i> = 100%, 31–40 min <i>A</i> = 90%, <i>B</i> = 10% was applied at a constant flow rate of 1 mL·min<sup>–1</sup>. All synthesized compounds were found to possess purities ≥95% by the HPLC analysis. Peptides were purchased from PSL GmbH, Heidelberg, Germany. p300(1284-1673) enzyme was purchased from Enzo Life Science, Farmingdale, USA.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> PAINS Analysis</h3><div class="NLM_p last">Active compounds were tested for known classes of assay interference compounds with the publicly available online tool “False Positive Remover” (<a href="http://www.cbligand.org" class="extLink">www.cbligand.org</a>). None of the tested compounds were flagged as PAINS. In addition, all compounds reported in this study were characterized in at least two individual assays with different readout techniques to provide orthogonal prove of their specific activities.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Chemistry</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Method A</h4><div class="NLM_p last">General procedure for the Cu-catalyzed Huisgen cycloaddition. Alkyne (1 equiv) and azide (1.1 equiv) were dissolved in a water/<i>tert</i>-butanol mixture (2 mL, 1:1). TBTA (0.1 equiv) was dissolved in 1 mL DMF and added to the reaction mixture. An aqueous CuSO<sub>4</sub> solution (0.1 M, 0.1 equiv) and aqueous solution of sodium ascorbate (0.1 M, 0.2 equiv) were added in that order. The resulting reaction mixture was stirred for 12 h at room temperature under a nitrogen atmosphere. After completion, the reaction was quenched by the addition of 5 mL of H<sub>2</sub>O, and the aqueous phase was extracted with ethyl acetate (3 × 15 mL). The product containing the layer was collected, and solvents were evaporated under reduced pressure. The crude product was purified using automated column chromatography or semi preparative HPLC.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Method B</h4><div class="NLM_p last">The general procedure for the nucleophilic halide substitution is as follows: To a solution of CoA trilithium salt (1 equiv) in 0.1 M TEAB buffer pH 8.5/EtOH (4 mL, 1:1) at 0 °C was added dropwise chloroacetone (1.2 equiv) in 2 mL EtOH. The solution was stirred under a nitrogen atmosphere at 0 °C for 2 h and then allowed to warm to room temperature and stirred overnight. The reaction was quenched by the addition of 10 mL H<sub>2</sub>O, and the aqueous phase was washed with ethyl acetate (3 × 20 mL). The aqueous layer was separated, and solvents were evaporated under reduced pressure. The crude product was then purified using semi-preparative HPLC.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Acetonyl-CoA</h4><div class="NLM_p last">Acetonyl-CoA was prepared by Method B using CoA trilithium salt (300 mg, 0.37 mmol) and chloroacetone (35 μL, 0.44 mmol). 255 mg (0.4 equiv TEA), 80% yield; <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 8.65 (1H, s, H-17), 8.43 (1H, s, H-20), 6.21 (1H, d, <i>J</i> = 5.4 Hz, H-16), 4.89 (2H, d, <i>J</i> = 5.3 Hz, H-14 and H-15), 4.60 (1H, s, H-13), 4.26 (2H, s, H-12), 4.01 (1H, s, H-7), 3.86 (1H, dd, <i>J</i> = 9.5, 4.9 Hz, H-11), 3.61 (1H, dd, <i>J</i> = 10.0, 4.9 Hz, H-11), 3.52 (2H, s, CO-C<i>H</i><sub>2</sub>-S), 3.46 (2H, t, <i>J</i> = 6.6 Hz, H-5), 3.31 (2H, t, <i>J</i> = 6.6 Hz, H-2), 2.60 (2H, t, <i>J</i> = 6.6 Hz, H-1), 2.46 (2H, t, <i>J</i> = 6.5 Hz, H-4), 2.28 (3H, s, C<i>H</i><sub>3</sub>-CO), 0.94 (3H, d, <i>J</i> = 7.7 Hz, H-9 or 10), 0.81 (3H, s, H-9 or 10). HPLC purity (HPLC-method A): 97.5% (<i>t</i><sub>R</sub> = 4.72 min); HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>40</sub>N<sub>7</sub>O<sub>17</sub>P<sub>3</sub>S [M – H]<sup>−</sup>, 822.1341; [M + Li – 2H]<sup>−</sup>, 828.0678; found [M – H]<sup>−</sup>, 822.1332; [M + Li – 2H]<sup>−</sup>, 828.1414.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>tert</i>-Butyl(6-oxo-6-(prop-2-yn-1-ylamino)hexyl)carbamate (<b>3</b>)</h4><div class="NLM_p last">6-<i>tert</i>-Butylaminocaproic acid (<b>2</b>) (2.169 g; 9.28 mmol) and HBTU (3.520 g, 9.28 mmol) were mixed in 50 mL dichloromethane, and DIPEA (4.85 mL, 27.8 mmol) was added. After stirring for 5 min, propargylamine (646 μL, 9.38 mmol) was added dropwise to the mixture. The reaction was stirred overnight and the organic phase was washed with saturated NaHCO<sub>3</sub> solution, 1 M HCl, and brine (3 × 50 mL, each). The organic layer was then dried over Na<sub>2</sub>SO<sub>4</sub> and the solvent was removed by evaporation. Purification of the crude material was achieved by automated column chromatography (cyclohexane/ethyl acetate, 20–100%) to obtain compound <b>5</b> as a white solid. 1.668 g, 67% yield; <sup>1</sup>H NMR (400 MHz, methanol-d<sub>4</sub>): δ 3.94 (2H, d, <i>J</i> = 2.6 Hz, NH-C<i>H</i><sub>2</sub>-C≡CH), 3.02 (2H, t, <i>J</i> = 7.0 Hz, H-6′), 2.57 (1H, t, <i>J</i> = 2.6 Hz, C≡C<i>H</i>), 2.19 (2H, t, <i>J</i> = 7.5 Hz, H-2′), 1.62 (2H, p, <i>J</i> = 7.5 Hz, H-5′), 1.52–1.45 (2H, m, H-3′), 1.43 (9H, s, (C(C<i>H</i><sub>3</sub>)<sub>3</sub>), 1.37–1.28 (2H, m, H-4′). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: calcd for, C<sub>14</sub>H<sub>24</sub>N<sub>2</sub>O<sub>3</sub> [M + Na]<sup>+</sup>, 291.2; found [M + Na]<sup>+</sup>, 291.4.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 6-(2-Chloroacetamido)-<i>N</i>-(prop-2-yn-1-yl)hexanamide (<b>4</b>)</h4><div class="NLM_p last">The Boc-group of propargylation product <b>3</b> (1.208 g, 4.50 mmol) was cleaved by addition of 15 mL TFA/dichloromethane and stirring for 2 h. Solvents were then evaporated and remaining TFA was coevaporated with methanol (5×). The residue was then dissolved in 11 mL DMF, and DIPEA (3.91 mL, 22.5 mmol) was added to the solution. After stirring for 10 min, the mixture was cooled to 0 °C, and chloroacetyl chloride (717 μL, 9.01 mmol) in 2 mL DMF was added dropwise over a period of 30 min. The reaction was quenched after 2 h by adding 50 mL of saturated NaHCO<sub>3</sub> solution. Crude product <b>4</b> was extracted from the aqueous phase with DCM (3 × 50 mL), and the organic layers were combined and dried over Na<sub>2</sub>SO<sub>4</sub> to give a black solid after evaporation of the solvent. The residue was purified by means of column chromatography using a gradient of methanol in dichloromethane (3–12%). Product <b>4</b> was isolated as an off-white solid. 0.848 g, 77% yield; <sup>1</sup>H NMR (400 MHz, methanol-d<sub>4</sub>): δ 4.02 (2H, s, NHCO-C<i>H</i><sub>2</sub>-Cl), 3.94 (2H, d, <i>J</i> = 2.6 Hz, NH-C<i>H</i><sub>2</sub>-C≡CH), 3.22 (2H, t, <i>J</i> = 7.1 Hz, H-6′), 2.57 (1H, t, <i>J</i> = 2.6 Hz, C≡C<i>H</i>), 2.20 (2H, t, <i>J</i> = 7.5 Hz, H-2′), 1.63 (2H, p, <i>J</i> = 15.2, 7.5 Hz, H-5′), 1.54 (2H, p, <i>J</i> = 7.5 Hz, H-3′), 1.40–1.30 (2H, m, H-4′). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: calcd for C<sub>11</sub>H<sub>17</sub>ClN<sub>2</sub>O<sub>2</sub> [M + Na]<sup>+</sup>, 267.1; found [M + Na]<sup>+</sup>, 267.0.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 4-Azidobutanoic Acid (<b>6</b>)</h4><div class="NLM_p last">γ-Aminobutanoic acid (<b>5</b>) (1.050 g, 10.18 mmol), imidazole-1-sulfonylazide × HCl (2.565 g, 12.26 mmol), potassium carbonate (2.084 g, 15.08 mmol), and copper sulfate × 5 H<sub>2</sub>O (29.3 mg, 0.12 mmol) were dissolved in methanol and stirred overnight under nitrogen atmosphere. The solvent was then removed by evaporation, and the residue was taken up in demin. H<sub>2</sub>O and acidified to pH 2 using HCl (5 M). The resulting suspension was extracted with ethyl acetate (3 × 100 mL), and the organic layers were combined and dried over Na<sub>2</sub>SO<sub>4</sub>. After removing the organic solvent by evaporation, the resulting crude material was purified by automated column chromatography using a gradient of 1–5% methanol in dichloromethane to yield product <b>6</b> as colorless oil. 1.135 g, 86% yield; <sup>1</sup>H NMR (400 MHz, methanol-d<sub>4</sub>): δ 3.36 (2H, t, <i>J</i> = 6.8 Hz, H-2″), 2.39 (2H, t, <i>J</i> = 7.3 Hz, H-4″), 1.86 (2H, p, <i>J</i> = 6.9 Hz, H-3″). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: calcd for C<sub>4</sub>H<sub>7</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 130.1; found [M + H]<sup>+</sup>, 130.1.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-Azido-<i>N</i>-(3′,6′-dihydroxy-3-oxo-3<i>H</i>-spiro[isobenzofuran-1,9′-xanthen]-6-yl)butanamide (<b>7</b>)</h4><div class="NLM_p last">4-Azidobutanoic acid (<b>6</b>) (1.33 g, 8.0 mmol) was dissolved in 20 mL of dichloromethane, and 2.07 mL of oxalyl chloride (24.0 mmol) was added to this solution. After addition of one drop of DMF, the mixture was stirred for 5 h. Evaporation of the solvents yielded the acyl chloride, which was directly used in the next step without further purification. 4-azidobutanoyl chloride (42 mg, crude) was dissolved in 10 mL of THF and dropwise added to the same amount of cooled solvent containing 0.9 equiv of fluoresceinamine (6-isomer, 103 mg, 0.3 mmol) and 10 equiv of NaHCO<sub>3</sub> (252 mg, 3 mmol). After complete addition of the acyl chloride, the ice bath was removed. The mixture was stirred overnight, and the reaction was subsequently quenched by adding 15 mL of 5% NaHCO<sub>3</sub> aqueous solution. The organic solvent was evaporated, and the remaining solution was acidified to pH 2 by adding 5 M HCl. Ethyl acetate (3 × 25 mL) was added to the suspension to extract the product from the aqueous layer. After evaporation of the organic solvent, the product containing residue was purified by column chromatography using a gradient of 80–100% ethyl acetate in cyclohexane. The desired product <b>7</b> was isolated as an orange solid. 113 mg, 82% yield; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.92 (1H, d, <i>J</i> = 8.5 Hz, H-8″), 7.76 (1H, dd, <i>J</i> = 8.4, 1.8 Hz, H-9″), 7.58 (1H, d, <i>J</i> = 1.4 Hz, H-11″), 6.67 (2H, d, <i>J</i> = 2.4 Hz, H-17″and H-20″), 6.64–6.60 (2H, m, H-14″ and H-23″), 6.54 (2H, dd, <i>J</i> = 8.7, 2.4 Hz, H-15″ and H-22″), 3.36–3.32 (2H, m, H-2″), 2.45 (2H, t, <i>J</i> = 7.3 Hz, H-4″), 1.87 (2H, p, <i>J</i> = 6.9 Hz, H-3″). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>18</sub>N<sub>4</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 459.1; found [M + H]<sup>+</sup>, 459.2.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 6-(2-Chloroacetamido)-<i>N</i>-((1-(4-((3′,6′-dihydroxy-3-oxo-3<i>H</i>-spiro[isobenzofuran-1,9′-xanthen]-6-yl)amino)-4-oxobutyl)-1<i>H</i>-1,2,3-triazol-4-yl)methyl)hexanamide (<b>8</b>)</h4><div class="NLM_p last"><b>8</b> was prepared by Method A using <b>4</b> (40 mg, 0.16 mmol) and <b>7</b> (80 mg, 0.18 mmol). 75 mg, 66% yield; <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.90 (1H, d, <i>J</i> = 8.4 Hz, H-8″), 7.81 (1H, s, H-9′), 7.70 (1H, dd, <i>J</i> = 8.4, 1.7 Hz, H-9″), 7.58 (1H, d, <i>J</i> = 1.5 Hz, H-11″), 6.67 (2H, d, <i>J</i> = 2.4 Hz, H-17′ and H-20″), 6.62 (2H, d, <i>J</i> = 8.7 Hz, H-14″ and H-23″), 6.54 (2H, dd, <i>J</i> = 8.7, 2.4 Hz, H-15″ and H-22″), 4.42 (2H, t, <i>J</i> = 6.8 Hz, H-4″), 4.35 (2H, s, H-7′), 4.00 (2H, s, NHCO-C<i>H</i><sub>2</sub>-Cl), 3.18 (2H, t, <i>J</i> = 7.1 Hz, H-6′), 2.38 (2H, t, <i>J</i> = 7.1 Hz, H-2″), 2.25–2.12 (4H, m, H-3″ and H2′), 1.68–1.43 (4H, m, H-3′ and H-5′), 1.31 (2H, m, H-4′). MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>35</sub>ClN<sub>6</sub>O<sub>8</sub> [M + Na]<sup>+</sup>, 725.2105; found [M + Na]<sup>+</sup>, 725.2097.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Fl-hex-CoA (<b>11</b>)</h4><div class="NLM_p last"><b>11</b> was prepared by Method B using CoA trilithium salt (40 mg, 0.05 mmol) and <b>8</b> (42 mg, 0.06 mmol). 34 mg, 48% yield; <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 8.46 (1H, s, H-17), 8.13 (1H, s, H-20), 7.79–7.73 (2H, m, H-9′ and H-8″), 7.48 (1H, dd, H-9″), 7.22 (1H, d, <i>J</i> = 2.1 Hz, H-11″), 7.07 (2H, d, <i>J</i> = 9.1 Hz, H14″ and H-23″), 6.56–6.49 (4H, m, H-15″, H-17″, H-20″ and H-22″), 6.08 (1H, d, <i>J</i> = 6.9 Hz, H-16), 4.97–4.67 (2H, m, H-14 and H-15), 4.53–4.47 (1H, m, H-13), 4.39 (2H, t, <i>J</i> = 6.5 Hz, H-4″), 4.22–4.11 (4H, m, H-12 and H-7′), 3.91 (1H, s, H7), 3.76–3.70 (1H, m, H-11), 3.48–3.42 (1H, m, H-11), 3.39–3.30 (2H, m, H-2), 3.18 (2H, t, <i>J</i> = 6.6 Hz, H-5), 3.06 (2H, s, H-11′), 2.98 (2H, t, <i>J</i> = 6.9 Hz, H-6′), 2.50 (2H, t, <i>J</i> = 6.7 Hz, H-1), 2.38–2.29 (4H, m, H-4 and H-2″), 2.20–2.12 (2H, m, H3″), 2.06 (2H, t, <i>J</i> = 7.3 Hz, H-2′), 1.40 (2H,, dt, <i>J</i> = 14.4, 7.2 Hz, H-5′), 1.30 (2H, dt, <i>J</i> = 15.0, 7.1 Hz, H-3′), 1.12–1.02 (2H, m, H4′), 0.77 (3H, s, H-9 or 10), 0.62 (3H, s, H-9 or 10). HPLC purity (HPLC-method-B): 95.0% (<i>t</i><sub>R</sub> = 12.30 min); HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i>: calcd for C<sub>56</sub>H<sub>70</sub>N<sub>13</sub>O<sub>24</sub>P<sub>3</sub>S [M – 3H]<sup>3–</sup>, 476.7791, [M + Na – 3H]<sup>2–</sup>, 726.6632; found [M – 3H]<sup>3–</sup>, 476.7790; [M + Na – 3H]<sup>2–</sup>, 726.6630.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Biotin-PEGhex-CoA (<b>12</b>)</h4><div class="NLM_p last"><b>12</b> was prepared by Method B using CoA trilithium salt (41 mg, 0.05 mmol) and <b>4</b> (15 mg, 0.06 mmol). followed by method A using the reaction product and <b>10</b> (45 mg, 0.06 mmol). 16 mg, 18% yield over 2 steps; <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 8.62 (1H, s, H-17), 8.39 (1H, s, H-20), 8.07 (1H, s, H-9′), 6.17 (1H, d, <i>J</i> = 4.7 Hz, H-16), 4.97–4.84 (2H, m, H-14 and H-15), 4.68–4.49 (4H, m, H-13, H-7″, and H-11″), 4.49–4.31 (3H, m, H-7′ and H-9″), 4.24 (2H, s, H-12), 4.01–3.77 (4H, m, H-7, H-11, and H-12″), 3.67–3.50 (43H, m, H-11 and triazole–CH<sub>2</sub>–CH<sub>2</sub>–[O–C<i>H</i><sub>2</sub>–C<i>H</i><sub>2</sub>]<sub>10</sub>–O–CH<sub>2</sub>–CH<sub>2</sub>–NHCO), 3.42 (2H, t, H-5), 3.36–3.21 (5H, m, H-2, H-6″, and H-14″), 3.19 (2H, s, H-11′), 3.11 (2H, t, <i>J</i> = 6.5 Hz, H-6′), 2.92 (1H, dd, <i>J</i> = 4.9, 13.1 Hz, H-10″), 2.70 (1H, d, <i>J</i> = 13.2 Hz, H-10″), 2.62 (2H, t, H-1), 2.42 (2H, t, H-4), 2.31–2.07 (4H, m, H-2′ and H-2″), 1.71–1.16 (12H, m, H-3′, H-4′, H-5′, H-3″, H-4″, and H-5″), 0.89 (3H, s, H-9 or 10), 0.78 (3H, s, H-9 or 10). HPLC purity (HPLC-method B): 95.3% (<i>t</i><sub>R</sub> = 11.40 min); HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i>: calcd for C<sub>66</sub>H<sub>116</sub>N<sub>15</sub>O<sub>31</sub>P<sub>3</sub>S<sub>2</sub> [M + K – 2H]<sup>−</sup>, 1808.6102; found [M + K – 2H]<sup>−</sup>, 1808.5994.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Protein Expression and Purification</h3><div class="NLM_p last"><i>E. coli</i> BL21 (DE3) cells were transformed with a pQE-80L plasmid containing a 0.5 kb insert coding for the catalytic domain of human pCAF protein (bp 1923-2420 with an N-terminal His-tag). Bacteria were cultured in an LB medium complemented with 100 mg/L ampicillin at 37 °C until an OD<sub>600</sub> of 0.6–0.8. The recombinant expression of the His-tagged pCAF catalytic domain (aa 493-658) was induced with 1 mM isopropyl-α-<span class="smallcaps smallerCapital">d</span>-thiogalactopyranoside (IPTG) and incubated overnight at 18 °C. The cell suspension was then centrifuged, and the cell pellet was resuspended in 40 mL of lysis buffer (50 mM Tris (pH 8.0), 150 mM NaCl, 1 mM β-mercaptoethanol, and 10% glycerol (v/v) with protease inhibitors (cOmplete, Mini Tablets, EDTA-free; Roche, Basel, CHE)). Cell lysis was carried out with the aid of sonification (5 × 15 s with 1 min pauses in between (Branson Sonifier II W-250, Heinemann, Schwäbisch Gmünd, GER)), and the resulting suspension was centrifuged (9000 rcf, 1 h, 4 °C) to separate the cell debris from the soluble protein fraction. Purification of the His-tagged protein was carried out on an automated chromatography system (NGC Chromatography System, Bio-Rad Laboratories, Hercules, USA). The clear supernatant after centrifugation was filtered and loaded on a 5 mL nickel-charged IMAC column (HisTrap HP; GE Healthcare, Chalfont St Giles, UK). After washing off nonspecifically bound proteins with a washing buffer (50 mM Tris (pH 8.0), 150 mM NaCl, 1 mM β-mercaptoethanol, 20 mM imidazole), a second washing step at high salt concentration (washing buffer containing 2 M NaCl and no imidazole) was included to remove residual proteins and nucleic acid contaminations. The desired protein was then eluted using 300 mM imidazole in a washing buffer, and the protein-containing fractions were collected, pooled, and concentrated by ultrafiltration (Vivaspin Turbo 4 concentrators, Sartorius, Göttingen, GER) to a final volume of 2 mL. Further purification of the protein solution was achieved by gel filtration on a HiLoad 26/60 Superdex 200 prep grade column (Pharmacia, New Jersey, USA) equilibrated with gel-filtration buffer (22 mM Na-citrate (pH 6.5), 330 mM NaCl). Collected fractions containing the monodisperse enzyme were again pooled and concentrated by ultrafiltration, and glycerol (10% (v/v)) was added. The purity of the preparation was checked by SDS-page gel analysis, and the protein concentration was determined by measuring the UV absorbance at 280 nm (Nanodrop 1000 Spectrometer, Thermo Fisher Scientific, Waltham, USA). The enzyme solution was then aliquoted, flash frozen in liquid nitrogen, and stored at −80 °C until used. <i>E. coli</i> Rosetta 2 (DE3) pLysS cells were transformed with a modified pET15b vector coding for MOF(174-449) with an N-terminal His<sub>10</sub>-tag followed by a TEV protease cleavage site. Overexpression was induced in LB media at an OD of ∼0.5 by the addition of IPTG (0.1 mM) and carried out at 18 °C overnight. After the harvest, the bacterial pellets were resuspended in a lysis buffer (50 mM HEPES, 500 mM NaCl, 20 mM imidazole, 5 mM β-mercaptoethanol, 0.01% (w/v) taurodeoxycholate, pH 7.5) supplemented with cOmplete Mini EDTA-free protease inhibitor cocktail tablets (Roche) and lysed via sonication. After removal of cell debris, the supernatant was applied to a HisTrap HP column (1 mL, GE Healthcare, Freiburg), washed extensively with lysis buffer, and then eluted with an imidazole gradient (20–300 mM). Fractions containing MOF were pooled, concentrated, and purified using a HiLoad 16/60 Superdex 75 gel-filtration column (GE Healthcare, Freiburg; 25 mM HEPES, 500 mM NaCl, pH 7.5). MOF-containing fractions were pooled, concentrated, and flash-frozen in liquid nitrogen. All chromatography steps were performed on Äkta prime chromatography systems. Identity of MOF was verified by mass spectrometry and protein concentration was determined using the Bradford assay.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> FP Assay</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Saturation Binding Experiment and <i>K</i><sub>D</sub> Determination</h4><div class="NLM_p">Recombinant acetyltransferase protein was serially diluted in assay buffer (50 mM HEPES (pH 7.5), 150 mM NaCl, 0.1 mM EDTA, 0.01% BSA (w/v), and 0.01% Tween-20 (v/v) (plus 1 mM DTT in the case of p300)) to a 2× intermediate concentration. From these dilutions, 10 μL was mixed with an equal volume of a 16 nM Fl-hex-CoA solution in assay buffer on a 384-well assay plate (cat. no. 11866091, Greiner Bio-One, Kremsmünster, AUT) (20 μL of the final volume containing 8 nM Fl-hex-CoA and appropriate concentrations of corresponding acetyltransferase protein (0–20 μM for His-pCAF(493-658), 0–3 μM for MOF(174-449) and 0–0.75 μM for p300(1284-1673))). The plate was spun down, incubated, and measured using a 2102 EnVision Multilabel Reader (PerkinElmer, Waltham, USA) operating in FP-mode (ex. 480 nm, em. <i>p</i>-plane 535 nm, em. <i>s</i>-plane 535 nm). All tested protein concentrations were also measured after incubation with a 8 nM fluorescent probe Fl-hex-CoA in the presence of competing ligand acetonyl-CoA (200 μM final concentration), to simulate a situation, where the probe is completely displaced from the protein. In addition, the FIs for <i>p</i>-plane and <i>s</i>-plane were measured for protein dilutions in assay buffer without Fl-hex-CoA to detect any intrinsic fluorescence not attributable to the fluorescent probe. The measured <i>I</i><sub>∥</sub> and <i>I</i><sub>∠</sub> values were corrected for intrinsic protein fluorescence, and the FP (<i>P</i>) was calculated for every protein concentration incubated with 8 nM Fl-hex-CoA in the presence (<i>P</i><sub>I</sub>) or without (<i>P</i><sub>M</sub>) competing ligand (100 μM acetonyl-CoA) using the following equation<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_m001.gif" alt="" id="_i36" /></img></span></div><div class="NLM_p">In this equation, <i>G</i> is an instrument specific constant. The FP values for samples containing no protein (<i>P</i><sub>L</sub>) or the highest protein concentration (<i>P</i><sub>L×R</sub>), respectively, resemble situations where the fluorescent probe is either free in solution or fully bound to the receptor. From these values, the fraction of bound Fl-hex-CoA (<i>F</i><sub>B</sub>) can be calculated for every protein concentration with<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_m002.gif" alt="" id="_i37" /></img></span></div><div class="NLM_p">The contribution of nonspecific binding (<i>P</i><sub>NS</sub>) was then determined for every protein concentration using the following equation<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_m003.gif" alt="" id="_i38" /></img></span></div><div class="NLM_p last">The specific FP signals (<i>P</i><sub>S</sub>) were calculated by subtracting the nonspecific portions from <i>P</i><sub>M</sub> values. The saturation binding curve was then generated by plotting <i>P</i><sub>S</sub> values against protein concentrations and the <i>K</i><sub>D</sub> value for the interaction of Fl-hex-CoA with acetyltransferase protein was calculated by fitting the corresponding results to a Hill equation using OriginPro 9 software.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Ligand Evaluation</h4><div class="NLM_p">Test compounds were appropriately diluted in DMSO or H<sub>2</sub>O to a 20× intermediate concentration. Resulting solutions were then mixed with an equal volume of assay buffer and 2 μL of the resulting dilutions were spotted into the wells of a black 384-well nonbinding microplate. For positive controls (<i>P</i><sub>P</sub>), 2 μL of the solvent/assay buffer mixture without the test compound were used. Next, 8 μL of a 10 nM Fl-hex-CoA solution were dispensed into each well, followed by 10 μL of acetyltransferase protein in assay buffer (<i>P</i><sub>C</sub>). Negative controls (<i>P</i><sub>N</sub>) included 10 μL of assay buffer instead of protein solution, and acetonyl-CoA (50 μM final concentration) was used as a reference inhibitor. Plates were then sealed with adhesive foil (TopSeal-A, PerkinElmer, Waltham, USA), spun down for 1 min at 300 rcf, and incubated at room temperature. Upon completion, the FI signals for <i>p</i>-plane and <i>s</i>-plane (<i>I</i><sub>∥</sub> and <i>I</i><sub>∠</sub>) were read using a 2102 EnVision Multilabel Reader (PerkinElmer, Waltham, USA) operating in FP-mode (ex. 480 nm, em. <i>p</i>-plane 535 nm, em. <i>s</i>-plane 535 nm). Results were blank corrected and the FP for every well was calculated as described above. Competition values (<i>C</i>) were calculated from the averages of controls (<i>P</i><sub>P</sub> and <i>P</i><sub>N</sub>) and determined <i>P</i><sub>C</sub> values with the use of the following equation<span class="NLM_disp-formula" id="ueq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_m004.gif" alt="" id="_i40" /></img></span></div><div class="NLM_p last">Calculation of IC<sub>50</sub> values and optical visualization were done with OriginPro 9 software by plotting competition values against logarithmic compound concentrations and applying a logistic 4 parameter sigmoidal fit with variable slope.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> BLI Assay</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>K</i><sub>D</sub> Value Determination for the Interaction of His-pCAF(493-658) and Biotin-PEGhex-CoA</h4><div class="NLM_p last">Streptavidin-coated biosensors (cat. no. 18-5019, Pall FortéBio, Menlo Park, USA) were hydrated in assay buffer (50 mM HEPES (pH 7.5), 300 mM NaCl, 0.1 mM EDTA, 0.1% BSA (w/v), 0.01% Tween-20 (v/v) (plus 1 mM DTT in the case of p300)) and loaded with biotinylated probe from a 125 nM Biotin-PEGhex-CoA solution placed in the drop holder. The loaded sensor tips were then dipped into solutions of serially diluted His-pCAF(493-658) (0.04–30 μM), and the association of protein to the sensor tip was recorded for 120 s from the drop holder position on a BLItz system (Pall FortéBio, Menlo Park, USA). Subsequently, the sensor tips were exposed to assay buffer, and the dissociation of protein from the sensor tip was recorded for 120 s from the tube position. A control run without the enzyme was recorded for association and dissociation, and the measured sample curves were referenced against this control. Global fitting of these data (for His-pCAF(493-658) concentrations spanning 0.04–3.33 μM) to a 1:1 binding model using the BLItz Pro 1.2 software gave the rate constants of association (<i>k</i><sub>a</sub>) and dissociation (<i>k</i><sub>d</sub>), as well as the affinity constant (<i>K</i><sub>D</sub>) for the interaction of His-pCAF(493-658) and Biotin-PEGhex-CoA. In a different attempt, report points of the association signal after 140 s total measurement time were plotted against His-pCAF(493-658) concentration, and the <i>K</i><sub>D</sub> value was calculated by fitting the data to a Hill equation using OriginPro 9 software.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Ligand Evaluation</h4><div class="NLM_p">To evaluate candidate ligands for competition to Biotin-PEGhex-CoA for binding to His-pCAF(493-658), compounds were appropriately diluted in DMSO or H<sub>2</sub>O and 1 μL of each compound dilution was mixed with 19 μL acetyltransferase protein in assay buffer (final concentrations: 250 nM for His-pCAF(493-658), 5 μM for MOF(174-449)). Positive controls received 1 μL of the pure solvent (DMSO or H<sub>2</sub>O). The mixtures were incubated for 30 min at room temperature. At the same time, streptavidin-coated biosensors were hydrated in assay buffer and subsequently loaded with biotinylated capture ligands by exposing the sensor tips to a 125 nM solution of Biotin-PEGhex-CoA for 120 s in the drop holder position. After loading, a 30 s baseline was generated in assay buffer from the tube position. Preincubated protein solutions were then transferred to the drop holder in 4 μL volume, and the association of protein to the immobilized capture ligand was measured for 120 s from these solutions, followed by a 120 s dissociation step in a tube filled with assay buffer. Recorded binding curves were referenced against a control run in assay buffer in the absence of protein. The binding signals at report points after 110 s of association (<i>B</i><sub>M</sub>) were used to calculate the degree of competition (<i>C</i>) for every ligand concentration compared with positive controls (<i>B</i><sub>P</sub>) by using the following equation<span class="NLM_disp-formula" id="ueq5"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_m005.gif" alt="" id="_i44" /></img></span></div><div class="NLM_p">The calculated competition values were plotted against logarithmic compound concentrations and the data were fit to a four parameter logistic fit with variable slope using OriginPro 9 software.</div><div class="NLM_p last">For experiments on p300(1284-1673), 500 nM enzyme in assay buffer were incubated with 125 nM Biotin-PEGhex-CoA and appropriately diluted candidate ligands for 60 min at room temperature in 20 μL volume. Fresh biosensor tips were hydrated and a 30 s baseline was recorded from assay buffer. Biosensors were then dipped into 4 μL of these mixtures, and the binding signal was measured for 120 s from the drop holder position. Competition values were determined as described above.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> FP Competition Activity Immunoassay</h3><div class="NLM_p">Experiments were performed in assay buffer (50 mM HEPES (pH 7.5), 150 mM NaCl, 0.1 mM EDTA, 0.01% BSA (w/v), 0.01% Tween-20 (v/v) (+1 mM DTT for experiments with p300) in 25 μL final volume on black 384-well nonbinding microplates (cat. no. 11866091, Greiner Bio-One, Kremsmünster, AUT). After dispensing of assay components, plates were spun down for 1 min at 300 rcf, sealed with adhesive foil (TopSeal-A, PerkinElmer, Waltham, USA) and incubated at room temperature. Blank controls containing 25 μL of assay buffer were included in all measurements. Combinations of specific antibodies and acetylated fluorescent tracer peptides were utilized as detection reagents, and the FI signals of polarized light in parallel plane (<i>I</i><sub>∥</sub>) and in perpendicular plane (<i>I</i><sub>∠</sub>) with respect to the excitation light were quantified using a 2102 EnVision Multilabel Reader (PerkinElmer, Waltham, USA) operating in FP-mode (for acH3K9-Rh: ex. 531 nm, em. <i>p</i>-plane 595 nm, em. <i>s</i>-plane 595 nm; for acH4-Fl: 480 nm, em. <i>p</i>-plane 535 nm, em. <i>s</i>-plane 535 nm).</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Antibody Validation</h4><div class="NLM_p last">Candidate antibodies were diluted in assay buffer by the factor of 1:50. Fluorescent tracer peptides were diluted in assay buffer to 16 nM for acH3K9-Rh or 6 nM for acH4-Fl. Then, 12.5 μL of each intermediate dilution was mixed in wells of an assay plate and incubated for 30 min at room temperature. Negative controls only received the tracer peptide and 12.5 μL of assay buffer. Blanks containing 25 μL of assay buffer were also included for background correction. After incubation, the plate was spun down and FP signals were measured from all wells. The two antibodies that gave the best increase in FP signal compared with negative controls anti-acH3K9 (cat. no. 39137, Active Motif, Carlsbad, USA) and anti-acH4 (cat. no. 177790, abcam, Cambridge, UK) were then titrated against fixed concentrations of their respective tracer peptide in 25 μL final assay volume. After 30 min incubation, the FP values were determined and plotted against antibody concentrations.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Enzyme Titration</h4><div class="NLM_p last">From a serial dilution series of enzyme in assay buffer (final concentrations spanning 0–100 nM), 5 μL was mixed with 10 μL of the substrate (final concentrations: 50 μM H3(1-21) for pCAF(493-658), 5 μM full length H4-His for MOF(174-449) and 5 μM H4(1-23) for p300(1284-1673)) solution and 5 μL 80 μM AcCoA in assay buffer to a final volume of 20 μL in assay plates. The enzymatic reaction was allowed to proceed for 60 min and then stopped by adding 2.5 μL of solutions containing 500 μM acetonyl-CoA and either 80 nM acH3K9-Rh or 30 nM acH4-Fl peptide. After spinning down the plate at 300 rcf and a short incubation step, 2.5 μL of antibody solution (50 μg/mL anti-acH3K9 or 4 μg/mL anti-acH4, respectively) was pipetted to each well, and the plate was incubated for 30 min at room temperature. Positive controls (<i>P</i><sub>P</sub>) received 5 μL of assay buffer instead of enzyme dilution and 2.5 μL of assay buffer instead of antibody dilution. Finally, plates were spun down at 300 rcf, and the FP signals were measured from all wells. FP values measured at 0 nM enzyme concentration (<i>P</i><sub>N</sub>) were used as a negative control. The degree of competition (<i>C</i>) was then calculated for every enzyme concentration by calculating the mean FP value of duplicate measurements (<i>P</i><sub>M</sub>) and inserting the values into the following equation<span class="NLM_disp-formula" id="ueq6"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_m006.gif" alt="" id="_i48" /></img></span></div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Ligand Evaluation</h4><div class="NLM_p">Serial dilution series of test compounds were prepared from 10 mM stock solutions and assay buffer. Recombinant enzyme (5 μL) was mixed with 10 μL of substrate solution and 1 μL of test compound (<i>A</i><sub>I</sub>) or 1 μL of assay buffer (positive control, <i>A</i><sub>P</sub>) or 1 μL of 1 mM acetonyl-CoA solution (negative control, <i>A</i><sub>N</sub>) and incubated for 5 min at room temperature. The enzymatic reaction was initiated by adding 4 μL of AcCoA diluted in assay buffer to the reaction mixtures. Reactions were incubated at room temperature for 60 min with gentle shaking. Tracer peptide was diluted to a 10× intermediate concentration in assay buffer containing 1 mM acetonyl-CoA, and 2.5 μL of this mixture was added to every reaction and control wells followed by 2.5 μL of a 10× concentrated antibody dilution, after a short incubation step. The plates were spun down, incubated for 30 min at room temperature, and FP signals were measured as described above. Inhibition values (<i>I</i>) were calculated from the averages of controls (<i>A</i><sub>P</sub> and <i>A</i><sub>N</sub>) and measured <i>A</i><sub>I</sub> values with the use of the following equation<span class="NLM_disp-formula" id="ueq7"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_m007.gif" alt="" id="_i50" /></img></span></div><div class="NLM_p last">Calculation of IC<sub>50</sub> values were done with OriginPro 9 software by plotting inhibition values against logarithmic compound concentrations and applying a logistic 4 parameter sigmoidal fit with a variable slope.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01887" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22060" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22060" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01887" class="ext-link">10.1021/acs.jmedchem.8b01887</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">HPLC chromatograms and NMR spectra of reference inhibitor acetonyl-CoA and compounds <b>11</b>, and <b>12</b>; information on atom numbering and used peptides; supplemental methods and additional figures illustrating probe validation, ITC measurements, FP, BLI, and activity assay optimization and compound profiling; and supplemental references (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf">jm8b01887_si_001.pdf (1.23 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_002.csv">jm8b01887_si_002.csv (2.15 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01887" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10166" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10166" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manfred Jung</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Sciences, University of
Freiburg, Albertstraße
25, 79104 Freiburg im Breisgau, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6361-7716" title="Orcid link">http://orcid.org/0000-0002-6361-7716</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dbb6bab5bda9bebff5b1aeb5bc9babb3baa9b6baa1b2bef5aeb5b2f6bda9beb2b9aea9bcf5bfbe"><span class="__cf_email__" data-cfemail="d7bab6b9b1a5b2b3f9bda2b9b097a7bfb6a5bab6adbeb2f9a2b9befab1a5b2beb5a2a5b0f9b3b2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roman
P. Simon</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Sciences, University of
Freiburg, Albertstraße
25, 79104 Freiburg im Breisgau, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobias Rumpf</span> - <span class="hlFld-Affiliation affiliation">Department
of Chromatin Regulation, Max-Planck-Institute
of Immunobiology and Epigenetics, Stuebeweg 51, 79108 Freiburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vaida Linkuviene</span> - <span class="hlFld-Affiliation affiliation">Department
of Biothermodynamics and Drug Design, Institute of Biotechnology,
Life Sciences Center, Vilnius University, Saulėtekio 7, 10257 Vilnius, Lithuania</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daumantas Matulis</span> - <span class="hlFld-Affiliation affiliation">Department
of Biothermodynamics and Drug Design, Institute of Biotechnology,
Life Sciences Center, Vilnius University, Saulėtekio 7, 10257 Vilnius, Lithuania</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6178-6276" title="Orcid link">http://orcid.org/0000-0002-6178-6276</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Asifa Akhtar</span> - <span class="hlFld-Affiliation affiliation">Department
of Chromatin Regulation, Max-Planck-Institute
of Immunobiology and Epigenetics, Stuebeweg 51, 79108 Freiburg, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43721" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43721" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Marie Markones and Prof. Heiko Heerklotz, University of Freiburg, Germany, for their help with ITC data collection and Marcel Wilde, Jan-Patrick Steitz and Prof. Michael Müller, University of Freiburg, Germany, for their help with pCAF expression and purification. We further acknowledge Sascha Ferlaino and the MagRes unit for NMR measurements. This work was funded by the Deutsche Forschungsgemeinschaft (DFG)—SPP 1463, 235777276/GRK1976, and SFB992.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Ab</td><td class="NLM_def"><p class="first last">antibody</p></td></tr><tr><td class="NLM_term">alpha</td><td class="NLM_def"><p class="first last">amplified luminescent proximity homogeneous assay</p></td></tr><tr><td class="NLM_term">BLI</td><td class="NLM_def"><p class="first last">biolayer interferometry</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">Fl</td><td class="NLM_def"><p class="first last">fluorescein</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">HBTU</td><td class="NLM_def"><p class="first last">3-[bis(dimethylamino)methyliumyl]-3<i>H</i>-benzotriazol-1-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>D</sub></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">MOF</td><td class="NLM_def"><p class="first last">male absent on the first</p></td></tr><tr><td class="NLM_term">pCAF</td><td class="NLM_def"><p class="first last">p300/CBP associated factor</p></td></tr><tr><td class="NLM_term">rcf</td><td class="NLM_def"><p class="first last">relative centrifugal force</p></td></tr><tr><td class="NLM_term">Rh</td><td class="NLM_def"><p class="first last">rhodanin</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">TBTA</td><td class="NLM_def"><p class="first last">Tris[(1-benzyl-1H-1,2,3-triazol-4-yl)methyl]amine</p></td></tr><tr><td class="NLM_term">TEAB</td><td class="NLM_def"><p class="first last">triethylammonium bicarbonate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i54">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33276" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33276" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnad, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walther, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span> <span> </span><span class="NLM_article-title">Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>325</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1126/science.1175371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1126%2Fscience.1175371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=19608861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=834-840&author=C.+Choudharyauthor=C.+Kumarauthor=F.+Gnadauthor=M.+L.+Nielsenauthor=M.+Rehmanauthor=T.+C.+Waltherauthor=J.+V.+Olsenauthor=M.+Mann&title=Lysine+Acetylation+Targets+Protein+Complexes+and+Co-Regulates+Major+Cellular+Functions&doi=10.1126%2Fscience.1175371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions</span></div><div class="casAuthors">Choudhary, Chunaram; Kumar, Chanchal; Gnad, Florian; Nielsen, Michael L.; Rehman, Michael; Walther, Tobias C.; Olsen, Jesper V.; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5942</span>),
    <span class="NLM_cas:pages">834-840</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lysine acetylation is a reversible posttranslational modification of proteins and plays a key role in regulating gene expression.  Technol. limitations have so far prevented a global anal. of lysine acetylation's cellular roles.  We used high-resoln. mass spectrometry to identify 3600 lysine acetylation sites on 1750 proteins and quantified acetylation changes in response to the deacetylase inhibitors suberoylanilide hydroxamic acid and MS-275.  Lysine acetylation preferentially targets large macromol. complexes involved in diverse cellular processes, such as chromatin remodeling, cell cycle, splicing, nuclear transport, and actin nucleation.  Acetylation impaired phosphorylation-dependent interactions of 14-3-3 and regulated the yeast cyclin-dependent kinase Cdc28.  Our data demonstrate that the regulatory scope of lysine acetylation is broad and comparable with that of other major posttranslational modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5eICW58ngkrVg90H21EOLACvtfcHk0ljQT-xBr920yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D&md5=8f643db5ba91c50f0b61763ff3095d2c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1175371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1175371%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DKumar%26aufirst%3DC.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DNielsen%26aufirst%3DM.%2BL.%26aulast%3DRehman%26aufirst%3DM.%26aulast%3DWalther%26aufirst%3DT.%2BC.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DLysine%2520Acetylation%2520Targets%2520Protein%2520Complexes%2520and%2520Co-Regulates%2520Major%2520Cellular%2520Functions%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D834%26epage%3D840%26doi%3D10.1126%2Fscience.1175371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K.-L.</span></span> <span> </span><span class="NLM_article-title">Regulation of Cellular Metabolism by Protein Lysine Acetylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1004</span>, <span class="refDoi"> DOI: 10.1126/science.1179689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1126%2Fscience.1179689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=20167786" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1000-1004&author=S.+Zhaoauthor=W.+Xuauthor=W.+Jiangauthor=W.+Yuauthor=Y.+Linauthor=T.+Zhangauthor=J.+Yaoauthor=L.+Zhouauthor=Y.+Zengauthor=H.+Liauthor=Y.+Liauthor=J.+Shiauthor=W.+Anauthor=S.+M.+Hancockauthor=F.+Heauthor=L.+Qinauthor=J.+Chinauthor=P.+Yangauthor=X.+Chenauthor=Q.+Leiauthor=Y.+Xiongauthor=K.-L.+Guan&title=Regulation+of+Cellular+Metabolism+by+Protein+Lysine+Acetylation&doi=10.1126%2Fscience.1179689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of cellular metabolism by protein lysine acetylation</span></div><div class="casAuthors">Zhao, Shimin; Xu, Wei; Jiang, Wenqing; Yu, Wei; Lin, Yan; Zhang, Tengfei; Yao, Jun; Zhou, Li; Zeng, Yaxue; Li, Hong; Li, Yixue; Shi, Jiong; An, Wenlin; Hancock, Susan M.; He, Fuchu; Qin, Lunxiu; Chin, Jason; Yang, Pengyuan; Chen, Xian; Lei, Qunying; Xiong, Yue; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5968</span>),
    <span class="NLM_cas:pages">1000-1004</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Protein Lys acetylation has emerged as a key post-translational modification in cellular regulation, in particular through the modification of histones and nuclear transcription regulators.  Here, the authors show that acetylation of Lys residues is a prevalent modification in enzymes that catalyze intermediate metab.  Virtually every enzyme in glycolysis, gluconeogenesis, the tricarboxylic acid (TCA) cycle, the urea cycle, fatty acid metab., and glycogen metab. was found to be acetylated in human liver tissue.  The concn. of metabolic fuels, such as glucose, amino acids, and fatty acids, influenced the acetylation status of metabolic enzymes.  Acetylation activated enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase in fatty acid oxidn. and malate dehydrogenase in the TCA cycle, inhibited argininosuccinate lyase in the urea cycle, and destabilized phosphoenolpyruvate carboxykinase in gluconeogenesis.  Thus, this study revealed that acetylation plays a major role in metabolic regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbdnDsuudXV7Vg90H21EOLACvtfcHk0ljzUNkOqFlIVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbo%253D&md5=d2c3c897077b9285842a7c55e8f8737e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179689%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DAn%26aufirst%3DW.%26aulast%3DHancock%26aufirst%3DS.%2BM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DChin%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLei%26aufirst%3DQ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DK.-L.%26atitle%3DRegulation%2520of%2520Cellular%2520Metabolism%2520by%2520Protein%2520Lysine%2520Acetylation%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1000%26epage%3D1004%26doi%3D10.1126%2Fscience.1179689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span> <span> </span><span class="NLM_article-title">Lysine Acetylation Goes Global: From Epigenetics to Metabolism and Therapeutics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1252</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00181</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00181" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1216-1252&author=I.+Aliauthor=R.+J.+Conradauthor=E.+Verdinauthor=M.+Ott&title=Lysine+Acetylation+Goes+Global%3A+From+Epigenetics+to+Metabolism+and+Therapeutics&doi=10.1021%2Facs.chemrev.7b00181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine acetylation goes global: From epigenetics to metabolism and therapeutics</span></div><div class="casAuthors">Ali, Ibraheem; Conrad, Ryan J.; Verdin, Eric; Ott, Melanie</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1216-1252</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The post-translational acetylation of protein Lys residues by protein lysine acetyltransferases has emerged as a key regulatory mechanism in all eukaryotic organisms.  Originally discovered in 1963 as a unique modification of histones, acetylation marks are now found on thousands of non-histone proteins located in virtually every cellular compartment.  Here, we summarize key findings in the field of protein acetylation over the past 20 yr with a focus on recent discoveries in nuclear, cytoplasmic, and mitochondrial compartments.  Collectively, these findings have elevated protein acetylation as a major post-translational modification, underscoring its physiol. relevance in gene regulation, cell signaling, metab., and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWVJmagBGDLLVg90H21EOLACvtfcHk0ljzUNkOqFlIVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKnsrw%253D&md5=ecb4497c9a416e806182b2e3ef16df55</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00181%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DI.%26aulast%3DConrad%26aufirst%3DR.%2BJ.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DLysine%2520Acetylation%2520Goes%2520Global%253A%2520From%2520Epigenetics%2520to%2520Metabolism%2520and%2520Therapeutics%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D1216%26epage%3D1252%26doi%3D10.1021%2Facs.chemrev.7b00181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kho, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grishin, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Substrate and Functional Diversity of Lysine Acetylation Revealed by a Proteomics Survey</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">607</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.06.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.molcel.2006.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=16916647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1Snurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=607-618&author=S.+C.+Kimauthor=R.+Sprungauthor=Y.+Chenauthor=Y.+Xuauthor=H.+Ballauthor=J.+Peiauthor=T.+Chengauthor=Y.+Khoauthor=H.+Xiaoauthor=L.+Xiaoauthor=N.+V.+Grishinauthor=M.+Whiteauthor=X.-J.+Yangauthor=Y.+Zhao&title=Substrate+and+Functional+Diversity+of+Lysine+Acetylation+Revealed+by+a+Proteomics+Survey&doi=10.1016%2Fj.molcel.2006.06.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate and functional diversity of lysine acetylation revealed by a proteomics survey</span></div><div class="casAuthors">Kim, Sung Chan; Sprung, Robert; Chen, Yue; Xu, Yingda; Ball, Haydn; Pei, Jimin; Cheng, Tzuling; Kho, Yoonjung; Xiao, Hao; Xiao, Lin; Grishin, Nick V.; White, Michael; Yang, Xiang-Jiao; Zhao, Yingming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">607-618</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Acetylation of proteins on lysine residues is a dynamic posttranslational modification that is known to play a key role in regulating transcription and other DNA-dependent nuclear processes.  However, the extent of this modification in diverse cellular proteins remains largely unknown, presenting a major bottleneck for lysine-acetylation biol.  Here we report the first proteomic survey of this modification, identifying 388 acetylation sites in 195 proteins among proteins derived from HeLa cells and mouse liver mitochondria.  In addn. to regulators of chromatin-based cellular processes, nonnuclear localized proteins with diverse functions were identified.  Most strikingly, acetyllysine was found in more than 20% of mitochondrial proteins, including many longevity regulators and metab. enzymes.  Our study reveals previously unappreciated roles for lysine acetylation in the regulation of diverse cellular pathways outside of the nucleus.  The combined data sets offer a rich source for further characterization of the contribution of this modification to cellular physiol. and human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo05vjtLKl917Vg90H21EOLACvtfcHk0ljzUNkOqFlIVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1Snurk%253D&md5=b8f45b2a9461d0824fc49aefcc038eba</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BC.%26aulast%3DSprung%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DBall%26aufirst%3DH.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DKho%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DGrishin%26aufirst%3DN.%2BV.%26aulast%3DWhite%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DX.-J.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DSubstrate%2520and%2520Functional%2520Diversity%2520of%2520Lysine%2520Acetylation%2520Revealed%2520by%2520a%2520Proteomics%2520Survey%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D607%26epage%3D618%26doi%3D10.1016%2Fj.molcel.2006.06.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leemhuis, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packman, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nightingale, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollfelder, F.</span></span> <span> </span><span class="NLM_article-title">The Human Histone Acetyltransferase P/Caf Is a Promiscuous Histone Propionyltransferase</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">499</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1002/cbic.200700556</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1002%2Fcbic.200700556" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=18247445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1Wht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=499-503&author=H.+Leemhuisauthor=L.+C.+Packmanauthor=K.+P.+Nightingaleauthor=F.+Hollfelder&title=The+Human+Histone+Acetyltransferase+P%2FCaf+Is+a+Promiscuous+Histone+Propionyltransferase&doi=10.1002%2Fcbic.200700556"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The human histone acetyltransferase P/CAF is a promiscuous histone propionyltransferase</span></div><div class="casAuthors">Leemhuis, Hans; Packman, Len C.; Nightingale, Karl P.; Hollfelder, Florian</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">499-503</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Revisiting previous mass spectrometric fragmentation patterns of nuclear proteins from human HeLa S3 cells revealed the existence of two new histone marks, namely propionylated and butyrylated lysines on H4 histones.  This observation raises the question how these modifications were brought about.  In vitro anal. indicates that several HATs can both acetylate and propionylate H4 peptides, although the lysine specificity of these modifications was not explored.  The authors now report that the human histone acetyltransferase P/CAF has the ability to transfer propionyl and acetyl groups with similar efficiency and specificity.  Taken together, the observation of propionyl marks in biol. samples, the substantial and residue-specific promiscuous histone propionyltransferase activity of a human enzyme identified here, and the existence of depropionylating activities make it conceivable that histone propionylation activity has a regulatory biol. role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl31vgEvRUm7Vg90H21EOLACvtfcHk0liCTEF5aPq3hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1Wht7o%253D&md5=bddcacc6988392dd5a17c3e173cf6b70</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200700556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200700556%26sid%3Dliteratum%253Aachs%26aulast%3DLeemhuis%26aufirst%3DH.%26aulast%3DPackman%26aufirst%3DL.%2BC.%26aulast%3DNightingale%26aufirst%3DK.%2BP.%26aulast%3DHollfelder%26aufirst%3DF.%26atitle%3DThe%2520Human%2520Histone%2520Acetyltransferase%2520P%252FCaf%2520Is%2520a%2520Promiscuous%2520Histone%2520Propionyltransferase%26jtitle%3DChemBioChem%26date%3D2008%26volume%3D9%26spage%3D499%26epage%3D503%26doi%3D10.1002%2Fcbic.200700556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-j.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Revealing the Protein Propionylation Activity of the Histone Acetyltransferase Mof (Males Absent on the First)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">3410</span>– <span class="NLM_lpage">3420</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra117.000529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1074%2Fjbc.ra117.000529" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=3410-3420&author=Z.+Hanauthor=H.+Wuauthor=S.+Kimauthor=X.+Yangauthor=Q.+Liauthor=H.+Huangauthor=H.+Caiauthor=M.+G.+Bartlettauthor=A.+Dongauthor=H.+Zengauthor=P.+J.+Brownauthor=X.-j.+Yangauthor=C.+H.+Arrowsmithauthor=Y.+Zhaoauthor=Y.+G.+Zheng&title=Revealing+the+Protein+Propionylation+Activity+of+the+Histone+Acetyltransferase+Mof+%28Males+Absent+on+the+First%29&doi=10.1074%2Fjbc.ra117.000529"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.ra117.000529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.ra117.000529%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DBartlett%26aufirst%3DM.%2BG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DYang%26aufirst%3DX.-j.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DRevealing%2520the%2520Protein%2520Propionylation%2520Activity%2520of%2520the%2520Histone%2520Acetyltransferase%2520Mof%2520%2528Males%2520Absent%2520on%2520the%2520First%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D3410%26epage%3D3420%26doi%3D10.1074%2Fjbc.ra117.000529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falck, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Lysine Propionylation and Butyrylation Are Novel Post-Translational Modifications in Histones</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">812</span>– <span class="NLM_lpage">819</span>, <span class="refDoi"> DOI: 10.1074/mcp.m700021-mcp200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1074%2Fmcp.m700021-mcp200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=17267393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsV2ktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=812-819&author=Y.+Chenauthor=R.+Sprungauthor=Y.+Tangauthor=H.+Ballauthor=B.+Sangrasauthor=S.+C.+Kimauthor=J.+R.+Falckauthor=J.+Pengauthor=W.+Guauthor=Y.+Zhao&title=Lysine+Propionylation+and+Butyrylation+Are+Novel+Post-Translational+Modifications+in+Histones&doi=10.1074%2Fmcp.m700021-mcp200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine propionylation and butyrylation are novel post-translational modifications in histones</span></div><div class="casAuthors">Chen, Yue; Sprung, Robert; Tang, Yi; Ball, Haydn; Sangras, Bhavani; Kim, Sung Chan; Falck, John R.; Peng, Junmin; Gu, Wei; Zhao, Yingming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">812-819</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9476</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The pos. charged lysine residue plays an important role in protein folding and functions.  Neutralization of the charge often has a profound impact on the substrate proteins.  Accordingly all the known post-translational modifications at lysine have pivotal roles in cell physiol. and pathol.  Here we report the discovery of two novel, in vivo lysine modifications in histones, lysine propionylation and butyrylation.  We confirmed, by in vitro labeling and peptide mapping by mass spectrometry, that two previously known acetyltransferases, p300 and CREB-binding protein, could catalyze lysine propionylation and lysine butyrylation in histones.  Finally p300 and CREB-binding protein could carry out autopropionylation and autobutyrylation in vitro.  Taken together, our results conclusively establish that lysine propionylation and lysine butyrylation are novel post-translational modifications.  Given the unique roles of propionyl-CoA and butyryl-CoA in energy metab. and the significant structural changes induced by the modifications, the two modifications are likely to have important but distinct functions in the regulation of biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFDkCYOTsOjbVg90H21EOLACvtfcHk0liCTEF5aPq3hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsV2ktLs%253D&md5=dd151e4055229bdbd0c26670454d03b2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fmcp.m700021-mcp200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.m700021-mcp200%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSprung%26aufirst%3DR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DBall%26aufirst%3DH.%26aulast%3DSangras%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DS.%2BC.%26aulast%3DFalck%26aufirst%3DJ.%2BR.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DGu%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DLysine%2520Propionylation%2520and%2520Butyrylation%2520Are%2520Novel%2520Post-Translational%2520Modifications%2520in%2520Histones%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2007%26volume%3D6%26spage%3D812%26epage%3D819%26doi%3D10.1074%2Fmcp.m700021-mcp200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">An Activity Gel Assay Detects a Single, Catalytically Active Histone Acetyltransferase Subunit in Tetrahymena Macronuclei</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">6364</span>– <span class="NLM_lpage">6368</span>, <span class="refDoi"> DOI: 10.1073/pnas.92.14.6364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1073%2Fpnas.92.14.6364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=7603997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK2MXmslKqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=6364-6368&author=J.+E.+Brownellauthor=C.+D.+Allis&title=An+Activity+Gel+Assay+Detects+a+Single%2C+Catalytically+Active+Histone+Acetyltransferase+Subunit+in+Tetrahymena+Macronuclei&doi=10.1073%2Fpnas.92.14.6364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">An activity gel assay detects a single, catalytically active histone acetyltransferase subunit in Tetrahymena macronuclei</span></div><div class="casAuthors">Brownell, James E.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6364-8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Macronuclei of the ciliated protozoan Tetrahymena thermophila possess a histone acetyltransferase activity closely assocd. with transcription-related histone acetylation.  Nothing definitive is known concerning the polypeptide compn. of this activity in Tetrahymena or any comparable activity from any cellular source.  An acetyltransferase activity gel assay was developed which identifies a catalytically active subunit of this enzyme in Tetrahymena.  This activity gel assay detects a single polypeptide of 55 kDa (p55) in crude macronuclear exts., as well as in column-purified fractions, which incorporates [3H]acetate from [3H]acetyl-CoA into core histone substrates polymd. directly into SDS polyacrylamide gels.  P55 copurifies precisely with acetyltransferase activity through all chromatog. steps examd., including reverse-phase HPLC.  Gel-filtration chromatog. of this activity indicates a mol. mass of 220 kDa, suggesting that the native enzyme may consist of four identical subunits of 55 kDa.  Furthermore, p55 is tightly assocd. with di- and greater polynucleosomes and therefore may be defined as a component of histone acetyltransferase type A - i.e., chromatin assocd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvronhG60mcLVg90H21EOLACvtfcHk0ljLRHG0WJc6xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmslKqtb8%253D&md5=f4e5e9f280f14e847fca2471e1f7633d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.14.6364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.14.6364%26sid%3Dliteratum%253Aachs%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DAn%2520Activity%2520Gel%2520Assay%2520Detects%2520a%2520Single%252C%2520Catalytically%2520Active%2520Histone%2520Acetyltransferase%2520Subunit%2520in%2520Tetrahymena%2520Macronuclei%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1995%26volume%3D92%26spage%3D6364%26epage%3D6368%26doi%3D10.1073%2Fpnas.92.14.6364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogryzko, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikawa, J.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span> <span> </span><span class="NLM_article-title">A P300/Cbp-Associated Factor That Competes with the Adenoviral Oncoprotein E1a</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>382</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1038/382319a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2F382319a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=8684459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK28Xks1Wgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=1996&pages=319-324&author=X.-J.+Yangauthor=V.+V.+Ogryzkoauthor=J.-i.+Nishikawaauthor=B.+H.+Howardauthor=Y.+Nakatani&title=A+P300%2FCbp-Associated+Factor+That+Competes+with+the+Adenoviral+Oncoprotein+E1a&doi=10.1038%2F382319a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A p300/CBP-associated factor that competes with the adenoviral oncoprotein E1A</span></div><div class="casAuthors">Yang, Xiang-Jiao; Ogryzko, Vasily V.; Nishikawa, Jun-ichi; Howard, Bruce H.; Nakatani, Yoshihiro</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">6589</span>),
    <span class="NLM_cas:pages">319-324</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The adenoviral oncoprotein E1A induces progression through the cell cycle by binding to the products of the p300/CBP and retinoblastoma gene families.  A new cellular p300/CBP-assocd. factor (P/CAF) having intrinsic histone acetylase activity has been identified that competes with E1A.  Exogenous expression of P/CAF in HeLa cells inhibits cell-cycle progression and counteracts the mitogenic activity of E1A.  E1A disturbs the normal cellular interaction between p300/CBP and its assocd. histone acetylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAEhSgaMP-wLVg90H21EOLACvtfcHk0ljLRHG0WJc6xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xks1Wgurc%253D&md5=b18c2c0f8ff02df9ff7586dfe670bfc0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2F382319a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F382319a0%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.-J.%26aulast%3DOgryzko%26aufirst%3DV.%2BV.%26aulast%3DNishikawa%26aufirst%3DJ.-i.%26aulast%3DHoward%26aufirst%3DB.%2BH.%26aulast%3DNakatani%26aufirst%3DY.%26atitle%3DA%2520P300%252FCbp-Associated%2520Factor%2520That%2520Competes%2520with%2520the%2520Adenoviral%2520Oncoprotein%2520E1a%26jtitle%3DNature%26date%3D1996%26volume%3D382%26spage%3D319%26epage%3D324%26doi%3D10.1038%2F382319a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parthun, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottschling, D. E.</span></span> <span> </span><span class="NLM_article-title">The Major Cytoplasmic Histone Acetyltransferase in Yeast: Links to Chromatin Replication and Histone Metabolism</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(00)81325-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fs0092-8674%2800%2981325-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=8858151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK28XmtFeitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=1996&pages=85-94&author=M.+R.+Parthunauthor=J.+Widomauthor=D.+E.+Gottschling&title=The+Major+Cytoplasmic+Histone+Acetyltransferase+in+Yeast%3A+Links+to+Chromatin+Replication+and+Histone+Metabolism&doi=10.1016%2Fs0092-8674%2800%2981325-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The major cytoplasmic histone acetyltransferase in yeast: links to chromatin replication and histone metabolism</span></div><div class="casAuthors">Parthun, Mark R.; Widom, Jonathan; Gottschling, Daniel E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-94</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We have isolated the predominant cytoplasmic histone acetyltransferase activity from Saccharomyces cerevisiae.  This enzyme acetylates the lysine at residue 12 of free histone H4 but does not modify histone H4 when packaged in chromatin.  The activity contains two proteins, Hat1p and Hat2p.  Hat1p is the catalytic subunit of the histone acetyltransferase and has an intrinsic substrate specificity that modifies lysine in the recognition sequence GXGKXG.  The specificity of the enzyme in the yeast cytoplasm is restricted relative to recombinant Hat1p suggesting that it is neg. regulated in vivo.  Hat2p, which is required for high affinity binding to the acetyltransferase to histone H4, is highly related to Rbap48, which is a subunit of the chromatin assembly factor, CAF-1, and copurifies with the human histone deacetylase HD1.  We propose that the Hat2p/HRbap48 family serve as escorts of histone metab. enzymes to facilitate their interaction with histone H4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohFSyni3vvwbVg90H21EOLACvtfcHk0ljLRHG0WJc6xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtFeitbc%253D&md5=d3e0b69fbe893306aebe308d56df8804</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fs0092-8674%2800%2981325-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0092-8674%252800%252981325-2%26sid%3Dliteratum%253Aachs%26aulast%3DParthun%26aufirst%3DM.%2BR.%26aulast%3DWidom%26aufirst%3DJ.%26aulast%3DGottschling%26aufirst%3DD.%2BE.%26atitle%3DThe%2520Major%2520Cytoplasmic%2520Histone%2520Acetyltransferase%2520in%2520Yeast%253A%2520Links%2520to%2520Chromatin%2520Replication%2520and%2520Histone%2520Metabolism%26jtitle%3DCell%26date%3D1996%26volume%3D87%26spage%3D85%26epage%3D94%26doi%3D10.1016%2Fs0092-8674%2800%2981325-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedmann, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachury, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structure of the -tubulin acetyltransferase, TAT1, and implications for tubulin-specific acetylation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">19655</span>– <span class="NLM_lpage">19660</span>, <span class="refDoi"> DOI: 10.1073/pnas.1209357109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1073%2Fpnas.1209357109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=19655-19660&author=D.+R.+Friedmannauthor=A.+Aguilarauthor=J.+Fanauthor=M.+V.+Nachuryauthor=R.+Marmorstein&title=Structure+of+the+-tubulin+acetyltransferase%2C+TAT1%2C+and+implications+for+tubulin-specific+acetylation&doi=10.1073%2Fpnas.1209357109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1209357109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1209357109%26sid%3Dliteratum%253Aachs%26aulast%3DFriedmann%26aufirst%3DD.%2BR.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DNachury%26aufirst%3DM.%2BV.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructure%2520of%2520the%2520-tubulin%2520acetyltransferase%252C%2520TAT1%252C%2520and%2520implications%2520for%2520tubulin-specific%2520acetylation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D19655%26epage%3D19660%26doi%3D10.1073%2Fpnas.1209357109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlow, E.</span></span> <span> </span><span class="NLM_article-title">Cellular Targets for Transformation by the Adenovirus E1a Proteins</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(89)90984-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2F0092-8674%2889%2990984-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=2521301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaL1MXhsVyrt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1989&pages=67-75&author=P.+Whyteauthor=N.+M.+Williamsonauthor=E.+Harlow&title=Cellular+Targets+for+Transformation+by+the+Adenovirus+E1a+Proteins&doi=10.1016%2F0092-8674%2889%2990984-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular targets for transformation by the adenovirus E1A proteins</span></div><div class="casAuthors">Whyte, Peter; Williamson, Nicola M.; Harlow, Ed</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-75</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Three cellular proteins, including species of 300,000 daltons and 107,000 daltons as well as p105-RB, the product of the retinoblastoma susceptibility gene, stably interact with the adenovirus E1A proteins.  To help det. the functional basis of these interactions, the E1A regions that participate in these interactions were mapped using a series of deletion mutants.  The 300,000-dalton and the 107,000-dalton proteins interacted with sequences within amino acids 1 to 76 and 121 to 127, resp.  Interaction with the third cellular protein, p105-RB, required the presence of sequences from 2 noncontiguous regions of the E1A polypeptide chain, amino acids 30 to 60 and 121 to 127.  The regions of E1A that are required for these interactions coincided precisely with the regions of E1A that are required for its transforming function.  These results suggest that the interactions with these cellular proteins are fundamental to the transforming activity of E1A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxvLkwEOAXp7Vg90H21EOLACvtfcHk0lgcTDT_4vr99A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhsVyrt70%253D&md5=9980cd29a2ab10a08491a9a1d59ba694</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2889%2990984-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252889%252990984-7%26sid%3Dliteratum%253Aachs%26aulast%3DWhyte%26aufirst%3DP.%26aulast%3DWilliamson%26aufirst%3DN.%2BM.%26aulast%3DHarlow%26aufirst%3DE.%26atitle%3DCellular%2520Targets%2520for%2520Transformation%2520by%2520the%2520Adenovirus%2520E1a%2520Proteins%26jtitle%3DCell%26date%3D1989%26volume%3D56%26spage%3D67%26epage%3D75%26doi%3D10.1016%2F0092-8674%2889%2990984-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chrivia, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, R. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montminy, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, R. H.</span></span> <span> </span><span class="NLM_article-title">Phosphorylated Creb Binds Specifically to the Nuclear Protein Cbp</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>365</i></span>,  <span class="NLM_fpage">855</span>– <span class="NLM_lpage">859</span>, <span class="refDoi"> DOI: 10.1038/365855a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2F365855a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=8413673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK2cXisVyiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=1993&pages=855-859&author=J.+C.+Chriviaauthor=R.+P.+S.+Kwokauthor=N.+Lambauthor=M.+Hagiwaraauthor=M.+R.+Montminyauthor=R.+H.+Goodman&title=Phosphorylated+Creb+Binds+Specifically+to+the+Nuclear+Protein+Cbp&doi=10.1038%2F365855a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylated CREB binds specifically to the nuclear proterin CBP</span></div><div class="casAuthors">Chrivia, John C.; Kwok, Roland P. S.; Lamb, Ned; Hagiwara, Masatoshi; Montminy, Marc R.; Goodman, Richard H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">6449</span>),
    <span class="NLM_cas:pages">855-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Regulation of gene expression by cAMP frequently involves a DNA element known as the cAMP-regulated enhancer (CRE).  Many transcription factors bind to this element, including the protein CREB, which is activated as a result of phosphorylation by protein kinase A.  This modification stimulates interaction with one or more of the general transcription factors or, alternatively, allows recruitment of a co-activator.  Here the authors report that CREB phosphorylated by protein kinase A binds specifically to a nuclear protein of Mr 265K termed CBP (for CREB-binding protein).  Fusion of a heterologous DNA-binding domain to the amino terminus of CBP enables the chimeric protein to function as a protein kinase A-regulated transcriptional activator.  It is proposed that CBP may participate in cAMP-regulated gene expression by interacting with the activated phosphorylated form of CREB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG6PKqMwiJE7Vg90H21EOLACvtfcHk0lgcTDT_4vr99A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXisVyiuw%253D%253D&md5=24f264edb41ca11d06d0e85f95d5308e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2F365855a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F365855a0%26sid%3Dliteratum%253Aachs%26aulast%3DChrivia%26aufirst%3DJ.%2BC.%26aulast%3DKwok%26aufirst%3DR.%2BP.%2BS.%26aulast%3DLamb%26aufirst%3DN.%26aulast%3DHagiwara%26aufirst%3DM.%26aulast%3DMontminy%26aufirst%3DM.%2BR.%26aulast%3DGoodman%26aufirst%3DR.%2BH.%26atitle%3DPhosphorylated%2520Creb%2520Binds%2520Specifically%2520to%2520the%2520Nuclear%2520Protein%2520Cbp%26jtitle%3DNature%26date%3D1993%26volume%3D365%26spage%3D855%26epage%3D859%26doi%3D10.1038%2F365855a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannuti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurnagel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucchesi, J. C.</span></span> <span> </span><span class="NLM_article-title">The Drosophila Msl Complex Acetylates Histone H4 at Lysine 16, a Chromatin Modification Linked to Dosage Compensation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">312</span>– <span class="NLM_lpage">318</span>, <span class="refDoi"> DOI: 10.1128/mcb.20.1.312-318.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1128%2Fmcb.20.1.312-318.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10594033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt1Wr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=312-318&author=E.+R.+Smithauthor=A.+Pannutiauthor=W.+Guauthor=A.+Steurnagelauthor=R.+G.+Cookauthor=C.+D.+Allisauthor=J.+C.+Lucchesi&title=The+Drosophila+Msl+Complex+Acetylates+Histone+H4+at+Lysine+16%2C+a+Chromatin+Modification+Linked+to+Dosage+Compensation&doi=10.1128%2Fmcb.20.1.312-318.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Drosophila MSL complex acetylates histone H4 at lysine 16, a chromatin modification linked to dosage compensation</span></div><div class="casAuthors">Smith, Edwin R.; Pannuti, Antonio; Gu, Weigang; Steurnagel, Arnd; Cook, Richard G.; Allis, C. David; Lucchesi, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">312-318</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In Drosophila, dosage compensation - the equalization of most X-linked gene products in males and females - is achieved by a 2-fold enhancement of the level of transcription of the X chromosome in males relative to each X chromosome in females.  A complex consisting of ≥5 gene products preferentially binds the X chromosome at numerous sites in males and results in a significant increase in the presence of a specific histone isoform, histone 4 acetylated at lysine 16.  Recently, RNA transcripts (roX1 and roX2) encoded by 2 different genes have also been found assocd. with the X chromosome in males.  A complex contg. MSL1, -2, and -3, MOF, MLE, and roX2 RNA was partially purified and demonstrated to exclusively acetylate H4 at lysine 16 on nucleosomal substrates.  These results demonstrate that the MSL complex is responsible for the specific chromatin modification characteristic of the X chromosome in Drosophila males.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBlzN1norgSrVg90H21EOLACvtfcHk0lgcTDT_4vr99A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlt1Wr&md5=90c1a8e9526fd6691d49b455cc62f012</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2Fmcb.20.1.312-318.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.20.1.312-318.2000%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DE.%2BR.%26aulast%3DPannuti%26aufirst%3DA.%26aulast%3DGu%26aufirst%3DW.%26aulast%3DSteurnagel%26aufirst%3DA.%26aulast%3DCook%26aufirst%3DR.%2BG.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DLucchesi%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520Drosophila%2520Msl%2520Complex%2520Acetylates%2520Histone%2520H4%2520at%2520Lysine%252016%252C%2520a%2520Chromatin%2520Modification%2520Linked%2520to%2520Dosage%2520Compensation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2000%26volume%3D20%26spage%3D312%26epage%3D318%26doi%3D10.1128%2Fmcb.20.1.312-318.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dent, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shilatifard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">New Nomenclature for Chromatin-Modifying Enzymes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.10.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.cell.2007.10.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=18022353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVSgur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2007&pages=633-636&author=C.+D.+Allisauthor=S.+L.+Bergerauthor=J.+Coteauthor=S.+Dentauthor=T.+Jenuwienauthor=T.+Kouzaridesauthor=L.+Pillusauthor=D.+Reinbergauthor=Y.+Shiauthor=R.+Shiekhattarauthor=A.+Shilatifardauthor=J.+Workmanauthor=Y.+Zhang&title=New+Nomenclature+for+Chromatin-Modifying+Enzymes&doi=10.1016%2Fj.cell.2007.10.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">New nomenclature for chromatin-modifying enzymes</span></div><div class="casAuthors">Allis, C. David; Berger, Shelley L.; Cote, Jacques; Dent, Sharon; Jenuwien, Thomas; Kouzarides, Tony; Pillus, Lorraine; Reinberg, Danny; Shi, Yang; Shiekhattar, Ramin; Shilatifard, Ali; Workman, Jerry; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">633-636</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A new nomenclature is proposed for the characterized members of the families of lysine demethylases, acetyltransferases, and lysine methyltransferases.  Many of these enzymes were initially found to target histones as substrates but were subsequently shown to also target non-histone proteins.  Therefore, these enzymes have been given a more generic name that reflects: (1) the type of enzymic activity that they perform and (2) the type of residue that they modify.  Thus lysine demethylases will become K-demethylases (KDMs), acetyltransferases will become K-acetyltransferases (KATs), and lysine methyltransferases will become K-methyltransferases (KMTs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyTFCLpvkjcbVg90H21EOLACvtfcHk0liq1UBlWamSfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVSgur%252FI&md5=6e12983346b153ab0999e5fdc3928ba2</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.10.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.10.039%26sid%3Dliteratum%253Aachs%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26aulast%3DCote%26aufirst%3DJ.%26aulast%3DDent%26aufirst%3DS.%26aulast%3DJenuwien%26aufirst%3DT.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPillus%26aufirst%3DL.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DShiekhattar%26aufirst%3DR.%26aulast%3DShilatifard%26aufirst%3DA.%26aulast%3DWorkman%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DNew%2520Nomenclature%2520for%2520Chromatin-Modifying%2520Enzymes%26jtitle%3DCell%26date%3D2007%26volume%3D131%26spage%3D633%26epage%3D636%26doi%3D10.1016%2Fj.cell.2007.10.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sapountzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, C. N.</span></span> <span> </span><span class="NLM_article-title">Cellular Functions of Tip60</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1496</span>– <span class="NLM_lpage">1509</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2006.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.biocel.2006.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=16698308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvVGjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2006&pages=1496-1509&author=V.+Sapountziauthor=I.+R.+Loganauthor=C.+N.+Robson&title=Cellular+Functions+of+Tip60&doi=10.1016%2Fj.biocel.2006.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular functions of TIP60</span></div><div class="casAuthors">Sapountzi, Vasileia; Logan, Ian R.; Robson, Craig N.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1496-1509</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  TIP60 was originally identified as a cellular acetyltransferase protein that interacts with HIV-1 Tat.  As a consequence, the role of TIP60 in transcriptional regulation has been investigated intensively.  Recent data suggest that TIP60 has more divergent functions than originally thought and roles for TIP60 in many processes, such as cellular signaling, DNA damage repair, cell cycle and checkpoint control and apoptosis are emerging.  TIP60 is a tightly regulated transcriptional coregulator, acting in a large multiprotein complex for a range of transcription factors including androgen receptor, Myc, STAT3, NF-κB, E2F1 and p53.  This usually involves recruitment of TIP60 acetyltransferase activities to chromatin.  Addnl., in response to DNA double strand breaks, TIP60 is recruited to DNA lesions where it participates both in the initial as well as the final stages of repair.  Here, the authors describe how TIP60 is a multifunctional enzyme involved in multiple nuclear transactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7jUBUBslgrbVg90H21EOLACvtfcHk0liq1UBlWamSfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvVGjtrw%253D&md5=c170361cb50984a86da9835ef8e7feb5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2006.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2006.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DSapountzi%26aufirst%3DV.%26aulast%3DLogan%26aufirst%3DI.%2BR.%26aulast%3DRobson%26aufirst%3DC.%2BN.%26atitle%3DCellular%2520Functions%2520of%2520Tip60%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2006%26volume%3D38%26spage%3D1496%26epage%3D1509%26doi%3D10.1016%2Fj.biocel.2006.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iizuka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stillman, B.</span></span> <span> </span><span class="NLM_article-title">Histone Acetyltransferase Hbo1 Interacts with the Orc1 Subunit of the Human Initiator Protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">23027</span>– <span class="NLM_lpage">23034</span>, <span class="refDoi"> DOI: 10.1074/jbc.274.33.23027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1074%2Fjbc.274.33.23027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=23027-23034&author=M.+Iizukaauthor=B.+Stillman&title=Histone+Acetyltransferase+Hbo1+Interacts+with+the+Orc1+Subunit+of+the+Human+Initiator+Protein&doi=10.1074%2Fjbc.274.33.23027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.33.23027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.33.23027%26sid%3Dliteratum%253Aachs%26aulast%3DIizuka%26aufirst%3DM.%26aulast%3DStillman%26aufirst%3DB.%26atitle%3DHistone%2520Acetyltransferase%2520Hbo1%2520Interacts%2520with%2520the%2520Orc1%2520Subunit%2520of%2520the%2520Human%2520Initiator%2520Protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D23027%26epage%3D23034%26doi%3D10.1074%2Fjbc.274.33.23027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karmodiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anamika, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muley, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradhan, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhide, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galande, S.</span></span> <span> </span><span class="NLM_article-title">Camello, a Novel Family of Histone Acetyltransferases That Acetylate Histone H4 and Is Essential for Zebrafish Development</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">6076</span>, <span class="refDoi"> DOI: 10.1038/srep06076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fsrep06076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=25123547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVWjsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=6076&author=K.+Karmodiyaauthor=K.+Anamikaauthor=V.+Muleyauthor=S.+J.+Pradhanauthor=Y.+Bhideauthor=S.+Galande&title=Camello%2C+a+Novel+Family+of+Histone+Acetyltransferases+That+Acetylate+Histone+H4+and+Is+Essential+for+Zebrafish+Development&doi=10.1038%2Fsrep06076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Camello, a novel family of Histone Acetyltransferases that acetylate histone H4 and is essential for zebrafish development</span></div><div class="casAuthors">Karmodiya, Krishanpal; Anamika, Krishanpal; Muley, Vijaykumar; Pradhan, Saurabh J.; Bhide, Yoshita; Galande, Sanjeev</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6076</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In this study, we have investigated genome-wide occurrence of Histone Acetyltransferases (HATs) in genomes of Mus musculus and Danio rerio on the basis of presence of HAT domain.  Our study identified a group of proteins that lacks characteristic features of known HAT families, relatively smaller in size and has no other assocd. domains.  Most of the proteins in this unclassified group are Camello proteins, which are not yet known and classified as functional HATs.  Our in vitro and in vivo anal. revealed that Camello family proteins are active HATs and exhibit specificity towards histone H4.  Interestingly, Camello proteins are among the first identified HATs showing perinuclear localization.  Moreover, Camello proteins are evolutionarily conserved in all chordates and are obsd. for the first time in cnidarians in phylogeny.  Furthermore, knockdown of Camello protein (CMLO3) in zebrafish embryos exhibited defects in axis elongation and head formation.  Thus, our study identified a novel family of active HATs that is specific for histone H4 acetylation, exhibits perinuclear localization and is essential for zebrafish development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo925EY-3OwiLVg90H21EOLACvtfcHk0liq1UBlWamSfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVWjsb0%253D&md5=121a536d0c2f61814b9f3c4facdf79bb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsrep06076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep06076%26sid%3Dliteratum%253Aachs%26aulast%3DKarmodiya%26aufirst%3DK.%26aulast%3DAnamika%26aufirst%3DK.%26aulast%3DMuley%26aufirst%3DV.%26aulast%3DPradhan%26aufirst%3DS.%2BJ.%26aulast%3DBhide%26aufirst%3DY.%26aulast%3DGalande%26aufirst%3DS.%26atitle%3DCamello%252C%2520a%2520Novel%2520Family%2520of%2520Histone%2520Acetyltransferases%2520That%2520Acetylate%2520Histone%2520H4%2520and%2520Is%2520Essential%2520for%2520Zebrafish%2520Development%26jtitle%3DSci.%2520Rep.%26date%3D2014%26volume%3D4%26spage%3D6076%26doi%3D10.1038%2Fsrep06076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirayama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassone-Corsi, P.</span></span> <span> </span><span class="NLM_article-title">Circadian Regulator Clock Is a Histone Acetyltransferase</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.03.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.cell.2006.03.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=16678094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD28XkslCjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=497-508&author=M.+Doiauthor=J.+Hirayamaauthor=P.+Sassone-Corsi&title=Circadian+Regulator+Clock+Is+a+Histone+Acetyltransferase&doi=10.1016%2Fj.cell.2006.03.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Circadian regulator CLOCK is a histone acetyltransferase</span></div><div class="casAuthors">Doi, Masao; Hirayama, Jun; Sassone-Corsi, Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">497-508</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mol. machinery that governs circadian rhythmicity comprises proteins whose interplay generates time-specific transcription of clock genes.  The role of chromatin remodeling in a physiol. setting such as the circadian clock is yet unclear.  Here, the authors show that transcription factor CLOCK, a central component of the circadian pacemaker, has histone acetyltransferase (HAT) activity.  CLOCK was found to share homol. with acetyl-CoA binding motifs within the MYST family of HATs.  CLOCK displayed high sequence similarity to ACTR, a member of SRC family of HATs, with which it shares also enzymic specificity for histones H3 and H4.  BMAL1, the heterodimerization partner of CLOCK, enhanced HAT function.  The HAT activity of CLOCK is essential to rescue circadian rhythmicity and activation of clock genes in Clock mutant cells.  Identification of CLOCK as a novel type of DNA binding HAT reveals that chromatin remodeling is crucial for the core clock mechanism and identifies unforeseen links between histone acetylation and cellular physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaemR1QPQRpLVg90H21EOLACvtfcHk0ljPvk8Y1yvaBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkslCjtLg%253D&md5=048bfdb95b7a14ff46d03aec22c44a47</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.03.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.03.033%26sid%3Dliteratum%253Aachs%26aulast%3DDoi%26aufirst%3DM.%26aulast%3DHirayama%26aufirst%3DJ.%26aulast%3DSassone-Corsi%26aufirst%3DP.%26atitle%3DCircadian%2520Regulator%2520Clock%2520Is%2520a%2520Histone%2520Acetyltransferase%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D497%26epage%3D508%26doi%3D10.1016%2Fj.cell.2006.03.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiltz, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privalsky, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, R. M.</span></span> <span> </span><span class="NLM_article-title">Nuclear Receptor Coactivator Actr Is a Novel Histone Acetyltransferase and Forms a Multimeric Activation Complex with P/Caf and Cbp/P300</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(00)80516-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fs0092-8674%2800%2980516-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=9267036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK2sXlsVKrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1997&pages=569-580&author=H.+Chenauthor=R.+J.+Linauthor=R.+L.+Schiltzauthor=D.+Chakravartiauthor=A.+Nashauthor=L.+Nagyauthor=M.+L.+Privalskyauthor=Y.+Nakataniauthor=R.+M.+Evans&title=Nuclear+Receptor+Coactivator+Actr+Is+a+Novel+Histone+Acetyltransferase+and+Forms+a+Multimeric+Activation+Complex+with+P%2FCaf+and+Cbp%2FP300&doi=10.1016%2Fs0092-8674%2800%2980516-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear receptor coactivator ACTR is a novel histone acetyltransferase and forms a multimeric activation complex with P/CAF and CBP/p300</span></div><div class="casAuthors">Chen, Hongwu; Lin, Richard J.; Schiltz, R. Louis; Chakravarti, Debabrata; Nash, Alyssa; Nagy, Laszlo; Privalsky, Martin L.; Nakatani, Yoshihiro; Evans, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-580</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The authors report the identification of a novel cofactor, ACTR, that directly binds nuclear receptors and stimulates their transcriptional activities in a hormone-dependent fashion.  ACTR also recruits two other nuclear factors, CBP and P/CAF, and thus plays a central role in creating a multisubunit coactivator complex.  In addn., and unexpectedly, the authors show that purified ACTR is a potent histone acetyltransferase and appears to define a distinct evolutionary branch to this recently described family.  Thus, hormonal activation by nuclear receptors involves the mutual recruitment of at least three classes of histone acetyltransferases that may act cooperatively as an enzymic unit to reverse the effects of histone deacetylase shown to be part of the nuclear receptor corepressor complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyMwDt8gKDc7Vg90H21EOLACvtfcHk0ljPvk8Y1yvaBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlsVKrsLw%253D&md5=6ca020e64b62362e309b36309af9fc37</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fs0092-8674%2800%2980516-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0092-8674%252800%252980516-4%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DR.%2BJ.%26aulast%3DSchiltz%26aufirst%3DR.%2BL.%26aulast%3DChakravarti%26aufirst%3DD.%26aulast%3DNash%26aufirst%3DA.%26aulast%3DNagy%26aufirst%3DL.%26aulast%3DPrivalsky%26aufirst%3DM.%2BL.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26atitle%3DNuclear%2520Receptor%2520Coactivator%2520Actr%2520Is%2520a%2520Novel%2520Histone%2520Acetyltransferase%2520and%2520Forms%2520a%2520Multimeric%2520Activation%2520Complex%2520with%2520P%252FCaf%2520and%2520Cbp%252FP300%26jtitle%3DCell%26date%3D1997%26volume%3D90%26spage%3D569%26epage%3D580%26doi%3D10.1016%2Fs0092-8674%2800%2980516-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span> <span> </span><span class="NLM_article-title">Two Human Orthologues of Eco1/Ctf7 Acetyltransferases Are Both Required for Proper Sister-Chromatid Cohesion</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3908</span>– <span class="NLM_lpage">3918</span>, <span class="refDoi"> DOI: 10.1091/mbc.e04-12-1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1091%2Fmbc.e04-12-1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=15958495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD2MXosVCnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2005&pages=3908-3918&author=F.+Houauthor=H.+Zou&title=Two+Human+Orthologues+of+Eco1%2FCtf7+Acetyltransferases+Are+Both+Required+for+Proper+Sister-Chromatid+Cohesion&doi=10.1091%2Fmbc.e04-12-1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Two human orthologues of Eco1/Ctf7 acetyltransferases are both required for proper sister-chromatid cohesion</span></div><div class="casAuthors">Hou, Fajian; Zou, Hui</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3908-3918</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1059-1524</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Genetic studies in yeast and Drosophila have uncovered a conserved acetyltransferase involved in sister-chromatid cohesion.  Here, we describe the two human orthologs, previously named EFO1/ESCO1 and EFO2/ESCO2.  Similar to their yeast (Eco1/Ctf7 and Eso1) and fly (deco) counterparts, both proteins feature a conserved C-terminal domain consisting of a H2C2 zinc finger motif and an acetyltransferase domain that is able to catalyze auto-acetylation reaction in vitro.  However, no similarity can be detected outside of the conserved domain.  RNA interference depletion expt. revealed that EFO1/ESCO1 and EFO2/ESCO2 were not redundant and that both were required for proper sister-chromatid cohesion.  The difference between EFO1 and EFO2 also is reflected in their cell cycle regulation.  In mitosis, EFO1 is phosphorylated, whereas EFO2 is degraded.  Furthermore, both proteins assoc. with chromosomes, and the chromosome binding depends on the diverse N-terminal domains.  We propose that EFO1 and EFO2 are targeted to different chromosome structures to help establish or maintain sister-chromatid cohesion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUy_cAA2M0vbVg90H21EOLACvtfcHk0ljPvk8Y1yvaBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXosVCnsL0%253D&md5=10b34fe2c094f397efee70fd1c430d2f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e04-12-1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e04-12-1063%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DF.%26aulast%3DZou%26aufirst%3DH.%26atitle%3DTwo%2520Human%2520Orthologues%2520of%2520Eco1%252FCtf7%2520Acetyltransferases%2520Are%2520Both%2520Required%2520for%2520Proper%2520Sister-Chromatid%2520Cohesion%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2005%26volume%3D16%26spage%3D3908%26epage%3D3918%26doi%3D10.1091%2Fmbc.e04-12-1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span> <span> </span><span class="NLM_article-title">Protein Lysine Acylation and Cysteine Succination by Intermediates of Energy Metabolism</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">947</span>– <span class="NLM_lpage">960</span>, <span class="refDoi"> DOI: 10.1021/cb3001793</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3001793" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC38XmslGqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=947-960&author=H.+Linauthor=X.+Suauthor=B.+He&title=Protein+Lysine+Acylation+and+Cysteine+Succination+by+Intermediates+of+Energy+Metabolism&doi=10.1021%2Fcb3001793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Protein lysine acylation and cysteine succination by intermediates of energy metabolism</span></div><div class="casAuthors">Lin, Hening; Su, Xiaoyang; He, Bin</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">947-960</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In the past few years, several new protein post-translational modifications that use intermediates in metab. have been discovered.  These include various acyl-Lys modifications (formylation, propionylation, butyrylation, crotonylation, malonylation, succinylation, myristoylation) and Cys succination.  Here, the authors review the discovery and the current understanding of these modifications.  Several of these modifications are regulated by the deacylases, sirtuins, which use NAD, an important metabolic small mol.  Interestingly, several of these modifications in turn regulate the activity of metabolic enzymes.  These new modifications reveal interesting connections between metab. and protein post-translational modifications and raise many questions for future investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzcRGttJMoRbVg90H21EOLACvtfcHk0ljPvk8Y1yvaBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmslGqsb0%253D&md5=e96b0583359af83bbbbce5e7e36236f9</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fcb3001793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3001793%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DB.%26atitle%3DProtein%2520Lysine%2520Acylation%2520and%2520Cysteine%2520Succination%2520by%2520Intermediates%2520of%2520Energy%2520Metabolism%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D947%26epage%3D960%26doi%3D10.1021%2Fcb3001793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousiges, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvi, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutillier, A.-L.</span></span> <span> </span><span class="NLM_article-title">Acetyltransferases (Hats) as Targets for Neurological Therapeutics</span>. <i>Neurotherapeutics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">588</span>, <span class="refDoi"> DOI: 10.1007/s13311-013-0204-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1007%2Fs13311-013-0204-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=24006237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ylurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=568-588&author=A.+Schneiderauthor=S.+Chatterjeeauthor=O.+Bousigesauthor=B.+R.+Selviauthor=A.+Swaminathanauthor=R.+Casselauthor=F.+Blancauthor=T.+K.+Kunduauthor=A.-L.+Boutillier&title=Acetyltransferases+%28Hats%29+as+Targets+for+Neurological+Therapeutics&doi=10.1007%2Fs13311-013-0204-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyltransferases (HATs) as Targets for Neurological Therapeutics</span></div><div class="casAuthors">Schneider, Anne; Chatterjee, Snehajyoti; Bousiges, Olivier; Selvi, B. Ruthrotha; Swaminathan, Amrutha; Cassel, Raphaelle; Blanc, Frederic; Kundu, Tapas K.; Boutillier, Anne-Laurence</div><div class="citationInfo"><span class="NLM_cas:title">Neurotherapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">568-588</span>CODEN:
                <span class="NLM_cas:coden">NEURNV</span>;
        ISSN:<span class="NLM_cas:issn">1878-7479</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The acetylation of histone and non-histone proteins controls a great deal of cellular functions, thereby affecting the entire organism, including the brain.  Acetylation modifications are mediated through histone acetyltransferases (HAT) and deacetylases (HDAC), and the balance of these enzymes regulates neuronal homeostasis, maintaining the pre-existing acetyl marks responsible for the global chromatin structure, as well as regulating specific dynamic acetyl marks that respond to changes and facilitate neurons to encode and strengthen long-term events in the brain circuitry (e.g., memory formation).  Unfortunately, the dysfunction of these finely-tuned regulations might lead to pathol. conditions, and the deregulation of the HAT/HDAC balance has been implicated in neurol. disorders.  During the last decade, research has focused on HDAC inhibitors that induce a histone hyperacetylated state to compensate acetylation deficits.  The use of these inhibitors as a therapeutic option was efficient in several animal models of neurol. disorders.  The elaboration of new cell-permeant HAT activators opens a new era of research on acetylation regulation.  Although pathol. animal models have not been tested yet, HAT activator mols. have already proven to be beneficial in ameliorating brain functions assocd. with learning and memory, and adult neurogenesis in wild-type animals.  Thus, HAT activator mols. contribute to an exciting area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPdlLQ5zuSkrVg90H21EOLACvtfcHk0lhb4wx0Co60Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ylurnN&md5=de7c6c89fa41c1311f892178d12b057d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs13311-013-0204-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13311-013-0204-7%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DA.%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DBousiges%26aufirst%3DO.%26aulast%3DSelvi%26aufirst%3DB.%2BR.%26aulast%3DSwaminathan%26aufirst%3DA.%26aulast%3DCassel%26aufirst%3DR.%26aulast%3DBlanc%26aufirst%3DF.%26aulast%3DKundu%26aufirst%3DT.%2BK.%26aulast%3DBoutillier%26aufirst%3DA.-L.%26atitle%3DAcetyltransferases%2520%2528Hats%2529%2520as%2520Targets%2520for%2520Neurological%2520Therapeutics%26jtitle%3DNeurotherapeutics%26date%3D2013%26volume%3D10%26spage%3D568%26epage%3D588%26doi%3D10.1007%2Fs13311-013-0204-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cereseto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manganaro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terreni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fittipaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lusic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacca, M.</span></span> <span> </span><span class="NLM_article-title">Acetylation of Hiv-1 Integrase by P300 Regulates Viral Integration</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3070</span>– <span class="NLM_lpage">3081</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fsj.emboj.7600770" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=3070-3081&author=A.+Ceresetoauthor=L.+Manganaroauthor=M.+I.+Gutierrezauthor=M.+Terreniauthor=A.+Fittipaldiauthor=M.+Lusicauthor=A.+Marcelloauthor=M.+Giacca&title=Acetylation+of+Hiv-1+Integrase+by+P300+Regulates+Viral+Integration&doi=10.1038%2Fsj.emboj.7600770"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600770%26sid%3Dliteratum%253Aachs%26aulast%3DCereseto%26aufirst%3DA.%26aulast%3DManganaro%26aufirst%3DL.%26aulast%3DGutierrez%26aufirst%3DM.%2BI.%26aulast%3DTerreni%26aufirst%3DM.%26aulast%3DFittipaldi%26aufirst%3DA.%26aulast%3DLusic%26aufirst%3DM.%26aulast%3DMarcello%26aufirst%3DA.%26aulast%3DGiacca%26aufirst%3DM.%26atitle%3DAcetylation%2520of%2520Hiv-1%2520Integrase%2520by%2520P300%2520Regulates%2520Viral%2520Integration%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26spage%3D3070%26epage%3D3081%26doi%3D10.1038%2Fsj.emboj.7600770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carneiro, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Abreu da Silva, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, E. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caby, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancelot, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanderstraete, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furdas, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantappié, M. R.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Changes Modulate Schistosome Egg Formation and Are a Novel Target for Reducing Transmission of Schistosomiasis</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e1004116</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1004116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1371%2Fjournal.ppat.1004116" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1004116&author=V.+C.+Carneiroauthor=I.+C.+de+Abreu+da+Silvaauthor=E.+J.+L.+Torresauthor=S.+Cabyauthor=J.+Lancelotauthor=M.+Vanderstraeteauthor=S.+D.+Furdasauthor=M.+Jungauthor=R.+J.+Pierceauthor=M.+R.+Fantappi%C3%A9&title=Epigenetic+Changes+Modulate+Schistosome+Egg+Formation+and+Are+a+Novel+Target+for+Reducing+Transmission+of+Schistosomiasis&doi=10.1371%2Fjournal.ppat.1004116"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1004116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1004116%26sid%3Dliteratum%253Aachs%26aulast%3DCarneiro%26aufirst%3DV.%2BC.%26aulast%3Dde%2BAbreu%2Bda%2BSilva%26aufirst%3DI.%2BC.%26aulast%3DTorres%26aufirst%3DE.%2BJ.%2BL.%26aulast%3DCaby%26aufirst%3DS.%26aulast%3DLancelot%26aufirst%3DJ.%26aulast%3DVanderstraete%26aufirst%3DM.%26aulast%3DFurdas%26aufirst%3DS.%2BD.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DFantappi%25C3%25A9%26aufirst%3DM.%2BR.%26atitle%3DEpigenetic%2520Changes%2520Modulate%2520Schistosome%2520Egg%2520Formation%2520and%2520Are%2520a%2520Novel%2520Target%2520for%2520Reducing%2520Transmission%2520of%2520Schistosomiasis%26jtitle%3DPLoS%2520Pathog.%26date%3D2014%26volume%3D10%26spage%3De1004116%26doi%3D10.1371%2Fjournal.ppat.1004116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Bosch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of Histone Acetyltransferases and Deacetylases Are Potential Drugs for Inflammatory Diseases</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">654</span>– <span class="NLM_lpage">660</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.drudis.2013.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=24269836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVWjsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=654-660&author=F.+J.+Dekkerauthor=T.+van+den+Boschauthor=N.+I.+Martin&title=Small+Molecule+Inhibitors+of+Histone+Acetyltransferases+and+Deacetylases+Are+Potential+Drugs+for+Inflammatory+Diseases&doi=10.1016%2Fj.drudis.2013.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of histone acetyltransferases and deacetylases are potential drugs for inflammatory diseases</span></div><div class="casAuthors">Dekker, Frank J.; van den Bosch, Thea; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">654-660</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a reversible post-translational modification (PTM) of cellular proteins and represents an important regulatory switch in signal transduction.  Lysine acetylation, in combination with other PTMs, directs the outcomes as well as the activation levels of important signal transduction pathways such as the nuclear factor (NF)-κB pathway.  Small mol. modulators of the 'writers' (HATs) and 'erasers' (HDACs) can regulate the NF-κB pathway in a specific manner.  This review focuses on the effects of frequently used HAT and HDAC inhibitors on the NF-κB signal transduction pathway and inflammatory responses, and their potential as novel therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmVBCrhkoMsLVg90H21EOLACvtfcHk0lgew1c6T4PEbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVWjsrrE&md5=d7d5eb5f350daa449cefd865433d4263</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DDekker%26aufirst%3DF.%2BJ.%26aulast%3Dvan%2Bden%2BBosch%26aufirst%3DT.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520Histone%2520Acetyltransferases%2520and%2520Deacetylases%2520Are%2520Potential%2520Drugs%2520for%2520Inflammatory%2520Diseases%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D654%26epage%3D660%26doi%3D10.1016%2Fj.drudis.2013.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okan, NA</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bales, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nascimento, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, PA</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medrano, EE</span></span> <span> </span><span class="NLM_article-title">Down-regulation of p300/CBP histone acetyltransferase activates a senescence checkpoint in human melanocytes</span>. <i>Cancer Res</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6231</span>– <span class="NLM_lpage">9</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=6231-9&author=D.+Bandyopadhyayauthor=NA+Okanauthor=E.+Balesauthor=L.+Nascimentoauthor=PA+Coleauthor=EE+Medrano&title=Down-regulation+of+p300%2FCBP+histone+acetyltransferase+activates+a+senescence+checkpoint+in+human+melanocytes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DOkan%26aufirst%3DNA%26aulast%3DBales%26aufirst%3DE.%26aulast%3DNascimento%26aufirst%3DL.%26aulast%3DCole%26aufirst%3DPA%26aulast%3DMedrano%26aufirst%3DEE%26atitle%3DDown-regulation%2520of%2520p300%252FCBP%2520histone%2520acetyltransferase%2520activates%2520a%2520senescence%2520checkpoint%2520in%2520human%2520melanocytes%26jtitle%3DCancer%2520Res%26date%3D2002%26volume%3D62%26spage%3D6231%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heery, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Targeting of Lysine Acetyltransferases in Human Disease: A Progress Report</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2006.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.drudis.2006.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=17198977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252Fgt1ynsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=88-99&author=D.+M.+Heeryauthor=P.+M.+Fischer&title=Pharmacological+Targeting+of+Lysine+Acetyltransferases+in+Human+Disease%3A+A+Progress+Report&doi=10.1016%2Fj.drudis.2006.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of lysine acetyltransferases in human disease: a progress report</span></div><div class="casAuthors">Heery David M; Fischer Peter M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">88-99</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Lysine acetyltransferases (LATs) are a structurally disparate group of enzymes involved in regulating transcription by participating as cofactors in transcriptional regulatory complexes, and by acetylation of lysine residues in histones and other proteins.  Aberrant LAT function probably plays an important part in the pathogenesis of certain cancers, especially leukaemias and endocrine tumours.  However, LAT activity might also be an important drug target in a range of other indications, including inflammatory lung diseases, viral infections and metabolic disorders.  At present, comparatively few LAT inhibitors are known, but progress regarding the understanding of their structural and functional biology is now beginning to reveal LATs as promising new epigenetic drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTI14vYK2Q1LGxpATR2EQURfW6udTcc2eaEyr2Vz2xCPbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252Fgt1ynsQ%253D%253D&md5=38036849f8d8e653407401712d9e70ee</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DHeery%26aufirst%3DD.%2BM.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26atitle%3DPharmacological%2520Targeting%2520of%2520Lysine%2520Acetyltransferases%2520in%2520Human%2520Disease%253A%2520A%2520Progress%2520Report%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D88%26epage%3D99%26doi%3D10.1016%2Fj.drudis.2006.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kost, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soong, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakadai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischle, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimer, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">A Stable Transcription Factor Complex Nucleated by Oligomeric Aml1-Eto Controls Leukaemogenesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>500</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1038/nature12287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnature12287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=23812588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKhtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=500&publication_year=2013&pages=93-97&author=X.-J.+Sunauthor=Z.+Wangauthor=L.+Wangauthor=Y.+Jiangauthor=N.+Kostauthor=T.+D.+Soongauthor=W.-Y.+Chenauthor=Z.+Tangauthor=T.+Nakadaiauthor=O.+Elementoauthor=W.+Fischleauthor=A.+Melnickauthor=D.+J.+Patelauthor=S.+D.+Nimerauthor=R.+G.+Roeder&title=A+Stable+Transcription+Factor+Complex+Nucleated+by+Oligomeric+Aml1-Eto+Controls+Leukaemogenesis&doi=10.1038%2Fnature12287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A stable transcription factor complex nucleated by oligomeric AML1-ETO controls leukaemogenesis</span></div><div class="casAuthors">Sun, Xiao-Jian; Wang, Zhanxin; Wang, Lan; Jiang, Yanwen; Kost, Nils; Soong, T. David; Chen, Wei-Yi; Tang, Zhanyun; Nakadai, Tomoyoshi; Elemento, Olivier; Fischle, Wolfgang; Melnick, Ari; Patel, Dinshaw J.; Nimer, Stephen D.; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">500</span>
        (<span class="NLM_cas:issue">7460</span>),
    <span class="NLM_cas:pages">93-97</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Transcription factors are frequently altered in leukemia through chromosomal translocation, mutation or aberrant expression.  AML1-ETO, a fusion protein generated by the t(8;21) translocation in acute myeloid leukemia, is a transcription factor implicated in both gene repression and activation.  AML1-ETO oligomerization, mediated by the NHR2 domain, is crit. for leukemogenesis, making it important to identify co-regulatory factors that 'read' the NHR2 oligomerization and contribute to leukemogenesis.  Here we show that, in human leukemic cells, AML1-ETO resides in and functions through a stable AML1-ETO-contg. transcription factor complex (AETFC) that contains several hematopoietic transcription (co)factors.  These AETFC components stabilize the complex through multivalent interactions, provide multiple DNA-binding domains for diverse target genes, co-localize genome wide, cooperatively regulate gene expression, and contribute to leukemogenesis.  Within the AETFC complex, AML1-ETO oligomerization is required for a specific interaction between the oligomerized NHR2 domain and a novel NHR2-binding (N2B) motif in E proteins.  Crystallog. anal. of the NHR2-N2B complex reveals a unique interaction pattern in which an N2B peptide makes direct contact with side chains of two NHR2 domains as a dimer, providing a novel model of how dimeric/oligomeric transcription factors create a new protein-binding interface through dimerization/oligomerization.  Intriguingly, disruption of this interaction by point mutations abrogates AML1-ETO-induced hematopoietic stem/progenitor cell self-renewal and leukemogenesis.  These results reveal new mechanisms of action of AML1-ETO, and provide a potential therapeutic target in t(8;21)-pos. acute myeloid leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv_tFeYiKM_bVg90H21EOLACvtfcHk0lgew1c6T4PEbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKhtbrI&md5=757dd656d6e371ec1503cca2a95103d9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature12287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12287%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.-J.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKost%26aufirst%3DN.%26aulast%3DSoong%26aufirst%3DT.%2BD.%26aulast%3DChen%26aufirst%3DW.-Y.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DNakadai%26aufirst%3DT.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DFischle%26aufirst%3DW.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DA%2520Stable%2520Transcription%2520Factor%2520Complex%2520Nucleated%2520by%2520Oligomeric%2520Aml1-Eto%2520Controls%2520Leukaemogenesis%26jtitle%3DNature%26date%3D2013%26volume%3D500%26spage%3D93%26epage%3D97%26doi%3D10.1038%2Fnature12287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, S.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Coactivator P300 Regulates Glucose-Induced Gene Expression in Endothelial Cells</span>. <i>Am. J. Physiol. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">E127</span>– <span class="NLM_lpage">E137</span>, <span class="refDoi"> DOI: 10.1152/ajpendo.00432.2009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1152%2Fajpendo.00432.2009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=19903865" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2010&pages=E127-E137&author=S.+Chenauthor=B.+Fengauthor=B.+Georgeauthor=R.+Chakrabartiauthor=M.+Chenauthor=S.+Chakrabarti&title=Transcriptional+Coactivator+P300+Regulates+Glucose-Induced+Gene+Expression+in+Endothelial+Cells&doi=10.1152%2Fajpendo.00432.2009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1152%2Fajpendo.00432.2009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpendo.00432.2009%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DGeorge%26aufirst%3DB.%26aulast%3DChakrabarti%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DChakrabarti%26aufirst%3DS.%26atitle%3DTranscriptional%2520Coactivator%2520P300%2520Regulates%2520Glucose-Induced%2520Gene%2520Expression%2520in%2520Endothelial%2520Cells%26jtitle%3DAm.%2520J.%2520Physiol.%2520Endocrinol.%2520Metab.%26date%3D2010%26volume%3D298%26spage%3DE127%26epage%3DE137%26doi%3D10.1152%2Fajpendo.00432.2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, J. L.</span></span> <span> </span><span class="NLM_article-title">Histone Acetyltransferase Complexes: One Size Doesn’t Fit All</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1038/nrm2145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnrm2145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=17380162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtlygsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=284-295&author=K.+K.+Leeauthor=J.+L.+Workman&title=Histone+Acetyltransferase+Complexes%3A+One+Size+Doesn%E2%80%99t+Fit+All&doi=10.1038%2Fnrm2145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Histone acetyltransferase complexes: one size doesn't fit all</span></div><div class="casAuthors">Lee, Kenneth K.; Workman, Jerry L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">284-295</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past 10 years, the study of histone acetyltransferases (HATs) has advanced significantly, and a no. of HATs have been isolated from various organisms.  It emerged that HATs are highly diverse and generally contain multiple subunits.  The functions of the catalytic subunit depend largely on the context of the other subunits in the complex.  We are just beginning to understand the specialized roles of HAT complexes in chromosome decondensation, DNA-damage repair and the modification of non-histone substrates, as well as their role in the broader epigenetic landscape, including the role of protein domains within HAT complexes and the dynamic interplay between HAT complexes and existing histone modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQQxOFwAIvkLVg90H21EOLACvtfcHk0lgJVkXsvkRNuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtlygsrw%253D&md5=598e09c551d6d6de93c3bf10e348d0c9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrm2145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2145%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DK.%2BK.%26aulast%3DWorkman%26aufirst%3DJ.%2BL.%26atitle%3DHistone%2520Acetyltransferase%2520Complexes%253A%2520One%2520Size%2520Doesn%25E2%2580%2599t%2520Fit%2520All%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D284%26epage%3D295%26doi%3D10.1038%2Fnrm2145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lasko, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Digiammarino, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaves, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algire, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karukurichi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Drie, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElligott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesicki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, K. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Selective Catalytic P300/Cbp Inhibitor That Targets Lineage-Specific Tumours</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>550</i></span>,  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nature24028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnature24028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=28953875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=128-132&author=L.+M.+Laskoauthor=C.+G.+Jakobauthor=R.+P.+Edaljiauthor=W.+Qiuauthor=D.+Montgomeryauthor=E.+L.+Digiammarinoauthor=T.+M.+Hansenauthor=R.+M.+Risiauthor=R.+Freyauthor=V.+Manavesauthor=B.+Shawauthor=M.+Algireauthor=P.+Hesslerauthor=L.+T.+Lamauthor=T.+Uzielauthor=E.+Faivreauthor=D.+Fergusonauthor=F.+G.+Buchananauthor=R.+L.+Martinauthor=M.+Torrentauthor=G.+G.+Chiangauthor=K.+Karukurichiauthor=J.+W.+Langstonauthor=B.+T.+Weinertauthor=C.+Choudharyauthor=P.+de+Vriesauthor=J.+H.+Van+Drieauthor=D.+McElligottauthor=E.+Kesickiauthor=R.+Marmorsteinauthor=C.+Sunauthor=P.+A.+Coleauthor=S.+H.+Rosenbergauthor=M.+R.+Michaelidesauthor=A.+Laiauthor=K.+D.+Bromberg&title=Discovery+of+a+Selective+Catalytic+P300%2FCbp+Inhibitor+That+Targets+Lineage-Specific+Tumours&doi=10.1038%2Fnature24028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective catalytic p300/CBP inhibitor that targets lineage-specific tumours</span></div><div class="casAuthors">Lasko, Loren M.; Jakob, Clarissa G.; Edalji, Rohinton P.; Qiu, Wei; Montgomery, Debra; Digiammarino, Enrico L.; Hansen, T. Matt; Risi, Roberto M.; Frey, Robin; Manaves, Vlasios; Shaw, Bailin; Algire, Mikkel; Hessler, Paul; Lam, Lloyd T.; Uziel, Tamar; Faivre, Emily; Ferguson, Debra; Buchanan, Fritz G.; Martin, Ruth L.; Torrent, Maricel; Chiang, Gary G.; Karukurichi, Kannan; Langston, J. William; Weinert, Brian T.; Choudhary, Chunaram; de Vries, Peter; Van Drie, John H.; McElligott, David; Kesicki, Ed; Marmorstein, Ronen; Sun, Chaohong; Cole, Philip A.; Rosenberg, Saul H.; Michaelides, Michael R.; Lai, Albert; Bromberg, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">550</span>
        (<span class="NLM_cas:issue">7674</span>),
    <span class="NLM_cas:pages">128-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The dynamic and reversible acetylation of proteins, catalyzed by histone acetyltransferases (HATs) and histone deacetylases (HDACs), is a major epigenetic regulatory mechanism of gene transcription and is assocd. with multiple diseases.  Histone deacetylase inhibitors are currently approved to treat certain cancers, but progress on the development of drug-like histone actyltransferase inhibitors has lagged behind.  The histone acetyltransferase paralogues p300 and CREB-binding protein (CBP) are key transcriptional co-activators that are essential for a multitude of cellular processes, and have also been implicated in human pathol. conditions (including cancer).  Current inhibitors of the p300 and CBP histone acetyltransferase domains, including natural products, bi-substrate analogs and the widely used small mol. C646, lack potency or selectivity.  Here, we describe A-485, a potent, selective and drug-like catalytic inhibitor of p300 and CBP.  We present a high resoln. (1.95 Å) co-crystal structure of a small mol. bound to the catalytic active site of p300 and demonstrate that A-485 competes with acetyl CoA (acetyl-CoA).  A-485 selectively inhibited proliferation in lineage-specific tumor types, including several haematol. malignancies and androgen receptor-pos. prostate cancer.  A-485 inhibited the androgen receptor transcriptional program in both androgen-sensitive and castration-resistant prostate cancer and inhibited tumor growth in a castration-resistant xenograft model.  These results demonstrate the feasibility of using small mol. inhibitors to selectively target the catalytic activity of histone acetyltransferases, which may provide effective treatments for transcriptional activator-driven malignancies and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqfy_rMa47C7Vg90H21EOLACvtfcHk0lgJVkXsvkRNuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsLzM&md5=050103c08a6abe2e0035573085fff6be</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnature24028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24028%26sid%3Dliteratum%253Aachs%26aulast%3DLasko%26aufirst%3DL.%2BM.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DMontgomery%26aufirst%3DD.%26aulast%3DDigiammarino%26aufirst%3DE.%2BL.%26aulast%3DHansen%26aufirst%3DT.%2BM.%26aulast%3DRisi%26aufirst%3DR.%2BM.%26aulast%3DFrey%26aufirst%3DR.%26aulast%3DManaves%26aufirst%3DV.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DAlgire%26aufirst%3DM.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DMartin%26aufirst%3DR.%2BL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DKarukurichi%26aufirst%3DK.%26aulast%3DLangston%26aufirst%3DJ.%2BW.%26aulast%3DWeinert%26aufirst%3DB.%2BT.%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3Dde%2BVries%26aufirst%3DP.%26aulast%3DVan%2BDrie%26aufirst%3DJ.%2BH.%26aulast%3DMcElligott%26aufirst%3DD.%26aulast%3DKesicki%26aufirst%3DE.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBromberg%26aufirst%3DK.%2BD.%26atitle%3DDiscovery%2520of%2520a%2520Selective%2520Catalytic%2520P300%252FCbp%2520Inhibitor%2520That%2520Targets%2520Lineage-Specific%2520Tumours%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D128%26epage%3D132%26doi%3D10.1038%2Fnature24028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leaver, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downer, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wichmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRae, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagiakos, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mieruszynski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanyai, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamasco, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sealey, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamudio, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnham, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheikh, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubrey, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doggett, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Silva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilling, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattarki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolezal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rautela, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newbold, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntington, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peat, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Street, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monahan, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, T.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Histone Acetyltransferases Kat6a/B Induce Senescence and Arrest Tumour Growth</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>560</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0387-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fs41586-018-0387-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=30069049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVensrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=560&publication_year=2018&pages=253-257&author=J.+B.+Baellauthor=D.+J.+Leaverauthor=S.+J.+Hermansauthor=G.+L.+Kellyauthor=M.+S.+Brennanauthor=N.+L.+Downerauthor=N.+Nguyenauthor=J.+Wichmannauthor=H.+M.+McRaeauthor=Y.+Yangauthor=B.+Clearyauthor=H.+R.+Lagiakosauthor=S.+Mieruszynskiauthor=G.+Paciniauthor=H.+K.+Vanyaiauthor=M.+I.+Bergamascoauthor=R.+E.+Mayauthor=B.+K.+Daveyauthor=K.+J.+Morganauthor=A.+J.+Sealeyauthor=B.+Wangauthor=N.+Zamudioauthor=S.+Wilcoxauthor=A.+L.+Garnhamauthor=B.+N.+Sheikhauthor=B.+J.+Aubreyauthor=K.+Doggettauthor=M.+C.+Chungauthor=M.+de+Silvaauthor=J.+Bentleyauthor=P.+Pillingauthor=M.+Hattarkiauthor=O.+Dolezalauthor=M.+L.+Dennisauthor=H.+Falkauthor=B.+Renauthor=S.+A.+Charmanauthor=K.+L.+Whiteauthor=J.+Rautelaauthor=A.+Newboldauthor=E.+D.+Hawkinsauthor=R.+W.+Johnstoneauthor=N.+D.+Huntingtonauthor=T.+S.+Peatauthor=J.+K.+Heathauthor=A.+Strasserauthor=M.+W.+Parkerauthor=G.+K.+Smythauthor=I.+P.+Streetauthor=B.+J.+Monahanauthor=A.+K.+Vossauthor=T.+Thomas&title=Inhibitors+of+Histone+Acetyltransferases+Kat6a%2FB+Induce+Senescence+and+Arrest+Tumour+Growth&doi=10.1038%2Fs41586-018-0387-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of histone acetyltransferases KAT6A/B induce senescence and arrest tumor growth</span></div><div class="casAuthors">Baell, Jonathan B.; Leaver, David J.; Hermans, Stefan J.; Kelly, Gemma L.; Brennan, Margs S.; Downer, Natalie L.; Nguyen, Nghi; Wichmann, Johannes; McRae, Helen M.; Yang, Yuqing; Cleary, Ben; Lagiakos, H. Rachel; Mieruszynski, Stephen; Pacini, Guido; Vanyai, Hannah K.; Bergamasco, Maria I.; May, Rose E.; Davey, Bethany K.; Morgan, Kimberly J.; Sealey, Andrew J.; Wang, Beinan; Zamudio, Natasha; Wilcox, Stephen; Garnham, Alexandra L.; Sheikh, Bilal N.; Aubrey, Brandon J.; Doggett, Karen; Chung, Matthew C.; de Silva, Melanie; Bentley, John; Pilling, Pat; Hattarki, Meghan; Dolezal, Olan; Dennis, Matthew L.; Falk, Hendrik; Ren, Bin; Charman, Susan A.; White, Karen L.; Rautela, Jai; Newbold, Andrea; Hawkins, Edwin D.; Johnstone, Ricky W.; Huntington, Nicholas D.; Peat, Thomas S.; Heath, Joan K.; Strasser, Andreas; Parker, Michael W.; Smyth, Gordon K.; Street, Ian P.; Monahan, Brendon J.; Voss, Anne K.; Thomas, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">560</span>
        (<span class="NLM_cas:issue">7717</span>),
    <span class="NLM_cas:pages">253-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Acetylation of histones by lysine acetyltransferases (KATs) is essential for chromatin organization and function1.  Among the genes coding for the MYST family of KATs (KAT5-KAT8) are the oncogenes KAT6A (also known as MOZ) and KAT6B (also known as MORF and QKF)2,3.  KAT6A has essential roles in normal hematopoietic stem cells4-6 and is the target of recurrent chromosomal translocations, causing acute myeloid leukemia7,8.  Similarly, chromosomal translocations in KAT6B have been identified in diverse cancers8.  KAT6A suppresses cellular senescence through the regulation of suppressors of the CDKN2A locus9,10, a function that requires its KAT activity10.  Loss of one allele of KAT6A extends the median survival of mice with MYC-induced lymphoma from 105 to 413 days11.  These findings suggest that inhibition of KAT6A and KAT6B may provide a therapeutic benefit in cancer.  Here we present highly potent, selective inhibitors of KAT6A and KAT6B, denoted WM-8014 and WM-1119.  Biochem. and structural studies demonstrate that these compds. are reversible competitors of acetyl CoA and inhibit MYST-catalyzed histone acetylation.  WM-8014 and WM-1119 induce cell cycle exit and cellular senescence without causing DNA damage.  Senescence is INK4A/ARF-dependent and is accompanied by changes in gene expression that are typical of loss of KAT6A function.  WM-8014 potentiates oncogene-induced senescence in vitro and in a zebrafish model of hepatocellular carcinoma.  WM-1119, which has increased bioavailability, arrests the progression of lymphoma in mice.  We anticipate that this class of inhibitors will help to accelerate the development of therapeutics that target gene transcription regulated by histone acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0njCpvnQW_7Vg90H21EOLACvtfcHk0lhPttZrunKRmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVensrjL&md5=d7ac24156dfa199e2257b58f85131b30</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0387-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0387-5%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DLeaver%26aufirst%3DD.%2BJ.%26aulast%3DHermans%26aufirst%3DS.%2BJ.%26aulast%3DKelly%26aufirst%3DG.%2BL.%26aulast%3DBrennan%26aufirst%3DM.%2BS.%26aulast%3DDowner%26aufirst%3DN.%2BL.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DWichmann%26aufirst%3DJ.%26aulast%3DMcRae%26aufirst%3DH.%2BM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DCleary%26aufirst%3DB.%26aulast%3DLagiakos%26aufirst%3DH.%2BR.%26aulast%3DMieruszynski%26aufirst%3DS.%26aulast%3DPacini%26aufirst%3DG.%26aulast%3DVanyai%26aufirst%3DH.%2BK.%26aulast%3DBergamasco%26aufirst%3DM.%2BI.%26aulast%3DMay%26aufirst%3DR.%2BE.%26aulast%3DDavey%26aufirst%3DB.%2BK.%26aulast%3DMorgan%26aufirst%3DK.%2BJ.%26aulast%3DSealey%26aufirst%3DA.%2BJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DZamudio%26aufirst%3DN.%26aulast%3DWilcox%26aufirst%3DS.%26aulast%3DGarnham%26aufirst%3DA.%2BL.%26aulast%3DSheikh%26aufirst%3DB.%2BN.%26aulast%3DAubrey%26aufirst%3DB.%2BJ.%26aulast%3DDoggett%26aufirst%3DK.%26aulast%3DChung%26aufirst%3DM.%2BC.%26aulast%3Dde%2BSilva%26aufirst%3DM.%26aulast%3DBentley%26aufirst%3DJ.%26aulast%3DPilling%26aufirst%3DP.%26aulast%3DHattarki%26aufirst%3DM.%26aulast%3DDolezal%26aufirst%3DO.%26aulast%3DDennis%26aufirst%3DM.%2BL.%26aulast%3DFalk%26aufirst%3DH.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DCharman%26aufirst%3DS.%2BA.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DRautela%26aufirst%3DJ.%26aulast%3DNewbold%26aufirst%3DA.%26aulast%3DHawkins%26aufirst%3DE.%2BD.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DHuntington%26aufirst%3DN.%2BD.%26aulast%3DPeat%26aufirst%3DT.%2BS.%26aulast%3DHeath%26aufirst%3DJ.%2BK.%26aulast%3DStrasser%26aufirst%3DA.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26aulast%3DStreet%26aufirst%3DI.%2BP.%26aulast%3DMonahan%26aufirst%3DB.%2BJ.%26aulast%3DVoss%26aufirst%3DA.%2BK.%26aulast%3DThomas%26aufirst%3DT.%26atitle%3DInhibitors%2520of%2520Histone%2520Acetyltransferases%2520Kat6a%252FB%2520Induce%2520Senescence%2520and%2520Arrest%2520Tumour%2520Growth%26jtitle%3DNature%26date%3D2018%26volume%3D560%26spage%3D253%26epage%3D257%26doi%3D10.1038%2Fs41586-018-0387-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhalabi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Katching-up on Small Molecule Modulators of Lysine Acetyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1249</span>– <span class="NLM_lpage">1270</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01502</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01502" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygt7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1249-1270&author=R.+P.+Simonauthor=D.+Robaaauthor=Z.+Alhalabiauthor=W.+Sipplauthor=M.+Jung&title=Katching-up+on+Small+Molecule+Modulators+of+Lysine+Acetyltransferases&doi=10.1021%2Facs.jmedchem.5b01502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">KATching-Up on Small Molecule Modulators of Lysine Acetyltransferases</span></div><div class="casAuthors">Simon, Roman P.; Robaa, Dina; Alhalabi, Zayan; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1249-1270</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The reversible acetylation of lysines is one of the best characterized epigenetic modifications.  Its involvement in many key physiol. and pathol. processes has been documented in numerous studies.  Lysine deacetylases (KDACs) and acetyltransferases (KATs) maintain the acetylation equil. at histones but also many other proteins.  Besides acetylation, also other acyl groups are reversibly installed at the side chain of lysines in proteins.  Because of their involvement in disease, KDACs and KATs were proposed to be promising drug targets, and for KDACs, indeed, five inhibitors are now approved for human use.  While there is a similar level of evidence for the potential of KATs as drug targets, no inhibitor is in clin. trials.  Here, we review the evidence for the diverse roles of KATs in disease pathol., provide an overview of structural features and the available modulators, including those targeting the bromodomains of KATs, and present an outlook.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFfZhEDBtCVLVg90H21EOLACvtfcHk0lhPttZrunKRmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVygt7rK&md5=afcf613db1cfa04320b93aed8b25cec1</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01502%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DR.%2BP.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DAlhalabi%26aufirst%3DZ.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DKatching-up%2520on%2520Small%2520Molecule%2520Modulators%2520of%2520Lysine%2520Acetyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1249%26epage%3D1270%26doi%3D10.1021%2Facs.jmedchem.5b01502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Organ, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuellar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrimp, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span> <span> </span><span class="NLM_article-title">Assay Interference and Off-Target Liabilities of Reported Histone Acetyltransferase Inhibitors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1527</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01657-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fs41467-017-01657-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=29142305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A280%3ADC%252BC1M3hs1Oisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1527&author=J.+L.+Dahlinauthor=K.+M.+Nelsonauthor=J.+M.+Strasserauthor=D.+Barsyte-Lovejoyauthor=M.+M.+Szewczykauthor=S.+Organauthor=M.+Cuellarauthor=G.+Singhauthor=J.+H.+Shrimpauthor=N.+Nguyenauthor=J.+L.+Meierauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=J.+B.+Baellauthor=M.+A.+Walters&title=Assay+Interference+and+Off-Target+Liabilities+of+Reported+Histone+Acetyltransferase+Inhibitors&doi=10.1038%2Fs41467-017-01657-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Assay interference and off-target liabilities of reported histone acetyltransferase inhibitors</span></div><div class="casAuthors">Dahlin Jayme L; Nelson Kathryn M; Strasser Jessica M; Cuellar Matthew; Singh Gurpreet; Walters Michael A; Barsyte-Lovejoy Dalia; Szewczyk Magdalena M; Organ Shawna; Arrowsmith Cheryl H; Brown Peter J; Shrimp Jonathan H; Meier Jordan L; Nguyen Nghi; Baell Jonathan B; Baell Jonathan B</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1527</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Many compounds with potentially reactive chemical motifs and poor physicochemical properties are published as selective modulators of biomolecules without sufficient validation and then propagated in the scientific literature as useful chemical probes.  Several histone acetyltransferase (HAT) inhibitors with these liabilities are now routinely used to probe epigenetic pathways.  We profile the most commonly used HAT inhibitors and confirm that the majority of them are nonselective interference compounds.  Most (15 out of 23, 65%) of the inhibitors are flagged by ALARM NMR, an industry-developed counter-screen for promiscuous compounds.  Biochemical counter-screens confirm that most of these compounds are either thiol-reactive or aggregators.  Selectivity panels show many of these compounds modulate unrelated targets in vitro, while several also demonstrate nonspecific effects in cell assays.  These data demonstrate the usefulness of performing counter-screens for bioassay promiscuity and assay interference, and raise caution about the utility of many widely used, but insufficiently validated, compounds employed in chemical biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ25EPALk8nTBKDYG3bNImrfW6udTcc2eYPRxNQRKdP2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3hs1Oisw%253D%253D&md5=07b4dbf2b38cebeacdc9db856a92f339</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01657-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01657-3%26sid%3Dliteratum%253Aachs%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DNelson%26aufirst%3DK.%2BM.%26aulast%3DStrasser%26aufirst%3DJ.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DOrgan%26aufirst%3DS.%26aulast%3DCuellar%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DShrimp%26aufirst%3DJ.%2BH.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DMeier%26aufirst%3DJ.%2BL.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DAssay%2520Interference%2520and%2520Off-Target%2520Liabilities%2520of%2520Reported%2520Histone%2520Acetyltransferase%2520Inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1527%26doi%3D10.1038%2Fs41467-017-01657-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soccio, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ait-Si-Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolffe, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">HATs off</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(00)80452-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fs1097-2765%2800%2980452-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=589-595&author=O.+D.+Lauauthor=T.+K.+Kunduauthor=R.+E.+Soccioauthor=S.+Ait-Si-Aliauthor=E.+M.+Khalilauthor=A.+Vassilevauthor=A.+P.+Wolffeauthor=Y.+Nakataniauthor=R.+G.+Roederauthor=P.+A.+Cole&title=HATs+off&doi=10.1016%2Fs1097-2765%2800%2980452-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2800%2980452-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252800%252980452-9%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DO.%2BD.%26aulast%3DKundu%26aufirst%3DT.%2BK.%26aulast%3DSoccio%26aufirst%3DR.%2BE.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26aulast%3DKhalil%26aufirst%3DE.%2BM.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DWolffe%26aufirst%3DA.%2BP.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DHATs%2520off%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D5%26spage%3D589%26epage%3D595%26doi%3D10.1016%2Fs1097-2765%2800%2980452-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rubenstein, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dryer, R.</span></span> <span> </span><span class="NLM_article-title">S-Acetonyl-Coa. A Nonreactive Analog of Acetyl-Coa</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>255</i></span>,  <span class="NLM_fpage">7858</span>– <span class="NLM_lpage">7862</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=255&publication_year=1980&pages=7858-7862&author=P.+Rubensteinauthor=R.+Dryer&title=S-Acetonyl-Coa.+A+Nonreactive+Analog+of+Acetyl-Coa"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRubenstein%26aufirst%3DP.%26aulast%3DDryer%26aufirst%3DR.%26atitle%3DS-Acetonyl-Coa.%2520A%2520Nonreactive%2520Analog%2520of%2520Acetyl-Coa%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1980%26volume%3D255%26spage%3D7858%26epage%3D7862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology Approaches for Investigating the Functions of Lysine Acetyltransferases</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1162</span>– <span class="NLM_lpage">1184</span>, <span class="refDoi"> DOI: 10.1002/anie.201704745</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1002%2Fanie.201704745" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1162-1184&author=M.+Heauthor=Z.+Hanauthor=L.+Liuauthor=Y.+G.+Zheng&title=Chemical+Biology+Approaches+for+Investigating+the+Functions+of+Lysine+Acetyltransferases&doi=10.1002%2Fanie.201704745"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fanie.201704745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201704745%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DChemical%2520Biology%2520Approaches%2520for%2520Investigating%2520the%2520Functions%2520of%2520Lysine%2520Acetyltransferases%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1162%26epage%3D1184%26doi%3D10.1002%2Fanie.201704745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelleher, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">A Selective Chemical Probe for Coenzyme a-Requiring Enzymes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">7621</span>– <span class="NLM_lpage">7624</span>, <span class="refDoi"> DOI: 10.1002/anie.200702485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1002%2Fanie.200702485" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=7621-7624&author=Y.+Hwangauthor=P.+R.+Thompsonauthor=L.+Wangauthor=L.+Jiangauthor=N.+L.+Kelleherauthor=P.+A.+Cole&title=A+Selective+Chemical+Probe+for+Coenzyme+a-Requiring+Enzymes&doi=10.1002%2Fanie.200702485"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.200702485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200702485%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DY.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DKelleher%26aufirst%3DN.%2BL.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520Selective%2520Chemical%2520Probe%2520for%2520Coenzyme%2520a-Requiring%2520Enzymes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2007%26volume%3D46%26spage%3D7621%26epage%3D7624%26doi%3D10.1002%2Fanie.200702485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Profiling Cellular Substrates of Lysine Acetyltransferases Gcn5 and P300 with Orthogonal Labeling and Click Chemistry</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1547</span>– <span class="NLM_lpage">1555</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00114</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00114" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt12rtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1547-1555&author=Z.+Hanauthor=C.-w.+Chouauthor=X.+Yangauthor=M.+G.+Bartlettauthor=Y.+G.+Zheng&title=Profiling+Cellular+Substrates+of+Lysine+Acetyltransferases+Gcn5+and+P300+with+Orthogonal+Labeling+and+Click+Chemistry&doi=10.1021%2Facschembio.7b00114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling Cellular Substrates of Lysine Acetyltransferases GCN5 and p300 with Orthogonal Labeling and Click Chemistry</span></div><div class="casAuthors">Han, Zhen; Chou, Chau-wen; Yang, Xiangkun; Bartlett, Michael G.; Zheng, Y. George</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1547-1555</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300 and GCN5 are two representative lysine acetyltransferases (KATs) in mammalian cells.  It was recently reported that they possess multiple acyltransferase activities including acetylation, propionylation, and butyrylation of the ε-amino group of lysine residues of histones and non-histone protein substrates.  Although thousands of acetylated substrates and acetylation sites have been identified by mass spectrometry-based proteomic screening, our knowledge about the causative connections between individual KAT members and their corresponding sub-acylomes remain very limited.  Herein, we applied 3-azidopropionyl CoA (3AZ-CoA) as a bioorthogonal surrogate of acetyl-, propionyl- and butyryl-CoA for KAT substrate identification.  We successfully attached the azide as a chem. warhead to cellular substrates of wild-type p300 and engineered GCN5.  The substrates were subsequently labeled with biotin tag through the copper-catalyzed azide-alkyne cycloaddn. (CuAAC).  Following protein enrichment on streptavidin-coated resin, we conducted LC-MS/MS studies from which more than four hundred proteins were identified as GCN5 or p300 substrate candidates.  These proteins are either p300- or GCN5-unique or shared by the two KATs and are extensively involved in various biol. events including gene expression, cell cycle, and cellular metab.  We also exptl. validated two novel substrates of GCN5, i.e., IQGAP1 and SMC1.  These results demonstrate extensive engagement of GCN5 and p300 in cellular pathways and provide new insights into understanding their functions in specific biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOrvzPGraY7rVg90H21EOLACvtfcHk0lhVeYM-g_OXUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmt12rtro%253D&md5=1a41940467a3d05736228e311e51189a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00114%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DChou%26aufirst%3DC.-w.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBartlett%26aufirst%3DM.%2BG.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DProfiling%2520Cellular%2520Substrates%2520of%2520Lysine%2520Acetyltransferases%2520Gcn5%2520and%2520P300%2520with%2520Orthogonal%2520Labeling%2520and%2520Click%2520Chemistry%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D1547%26epage%3D1555%26doi%3D10.1021%2Facschembio.7b00114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Labeling Lysine Acetyltransferase Substrates with Engineered Enzymes and Functionalized Cofactor Surrogates</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">7791</span>– <span class="NLM_lpage">7794</span>, <span class="refDoi"> DOI: 10.1021/ja311636b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311636b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVWisbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=7791-7794&author=C.+Yangauthor=J.+Miauthor=Y.+Fengauthor=L.+Ngoauthor=T.+Gaoauthor=L.+Yanauthor=Y.+G.+Zheng&title=Labeling+Lysine+Acetyltransferase+Substrates+with+Engineered+Enzymes+and+Functionalized+Cofactor+Surrogates&doi=10.1021%2Fja311636b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Labeling Lysine Acetyltransferase Substrates with Engineered Enzymes and Functionalized Cofactor Surrogates</span></div><div class="casAuthors">Yang, Chao; Mi, Jiaqi; Feng, You; Ngo, Liza; Gao, Tielong; Yan, Leilei; Zheng, Yujun George</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7791-7794</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Elucidating biol. and pathol. functions of protein lysine acetyltransferases (KATs) greatly depends on the knowledge of the dynamic and spatial localization of their enzymic targets in the cellular proteome.  We report the design and application of chem. probes for facile labeling and detection of substrates of the three major human KAT enzymes.  In this approach, we create engineered KATs in conjunction with synthetic Ac-CoA surrogates to effectively label KAT substrates even in the presence of competitive nascent cofactor acetyl-CoA.  The functionalized and transferable acyl moiety of the Ac-CoA analogs further allowed the labeled substrates to be probed with alkynyl or azido-tagged fluorescent reporters by the copper-catalyzed azide-alkyne cycloaddn.  The synthetic cofactors, in combination with either native or rationally engineered KAT enzymes, provide a versatile chem. biol. strategy to label and profile cellular targets of KATs at the proteomic level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxR6hjOF2JsrVg90H21EOLACvtfcHk0lhVeYM-g_OXUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVWisbw%253D&md5=1a970f3a3802a2b77441d00fabbf0589</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja311636b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311636b%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DMi%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DNgo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DLabeling%2520Lysine%2520Acetyltransferase%2520Substrates%2520with%2520Engineered%2520Enzymes%2520and%2520Functionalized%2520Cofactor%2520Surrogates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D7791%26epage%3D7794%26doi%3D10.1021%2Fja311636b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorum, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guasch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicklaus, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, J. L.</span></span> <span> </span><span class="NLM_article-title">Metabolic Regulation of Histone Acetyltransferases by Endogenous Acyl-Coa Cofactors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1030</span>– <span class="NLM_lpage">1039</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.chembiol.2015.06.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=26190825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2ksL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=1030-1039&author=D.+C.+Montgomeryauthor=A.+W.+Sorumauthor=L.+Guaschauthor=M.+C.+Nicklausauthor=J.+L.+Meier&title=Metabolic+Regulation+of+Histone+Acetyltransferases+by+Endogenous+Acyl-Coa+Cofactors&doi=10.1016%2Fj.chembiol.2015.06.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Regulation of Histone Acetyltransferases by Endogenous Acyl-CoA Cofactors</span></div><div class="casAuthors">Montgomery, David C.; Sorum, Alexander W.; Guasch, Laura; Nicklaus, Marc C.; Meier, Jordan L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1030-1039</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The finding that chromatin modifications are sensitive to changes in cellular cofactor levels potentially links altered tumor cell metab. and gene expression.  However, the specific enzymes and metabolites that connect these two processes remain obscure.  Characterizing these metabolic-epigenetic axes is crit. to understanding how metab. supports signaling in cancer, and developing therapeutic strategies to disrupt this process.  Here, the authors describe a chem. approach to define the metabolic regulation of lysine acetyltransferase (KAT) enzymes.  Using a novel chemoproteomic probe, the authors identify a previously unreported interaction between palmitoyl CoA (palmitoyl-CoA) and KAT enzymes.  Further anal. reveals that palmitoyl-CoA is a potent inhibitor of KAT activity and that fatty acyl-CoA precursors reduce cellular histone acetylation levels.  These studies implicate fatty acyl-CoAs as endogenous regulators of histone acetylation, and suggest novel strategies for the study and metabolic modulation of epigenetic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ2ykKcB6OU7Vg90H21EOLACvtfcHk0lhVeYM-g_OXUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2ksL3P&md5=dd2a665db3e0b430bae58a9e35c25d85</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DD.%2BC.%26aulast%3DSorum%26aufirst%3DA.%2BW.%26aulast%3DGuasch%26aufirst%3DL.%26aulast%3DNicklaus%26aufirst%3DM.%2BC.%26aulast%3DMeier%26aufirst%3DJ.%2BL.%26atitle%3DMetabolic%2520Regulation%2520of%2520Histone%2520Acetyltransferases%2520by%2520Endogenous%2520Acyl-Coa%2520Cofactors%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D1030%26epage%3D1039%26doi%3D10.1016%2Fj.chembiol.2015.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Human Histone Acetyltransferase P/Caf</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">15652</span>, <span class="refDoi"> DOI: 10.1021/bi005121q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi005121q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotVGgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=15652&author=K.+G.+Tannerauthor=M.+R.+Langerauthor=J.+M.+Denu&title=Kinetic+Mechanism+of+Human+Histone+Acetyltransferase+P%2FCaf&doi=10.1021%2Fbi005121q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic mechanism of human histone acetyltransferase P/CAF. [Erratum to document cited in CA133:360391]</span></div><div class="casAuthors">Tanner, Kirk G.; Langer, Michael R.; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">15652</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">-Assocd. factor histone acetyltransferase by Lau, O. D., Courtney, A. D., Vassilev, A., Marzilli, L. A., Cotter, R. J., Nakatani, Y., and Cole, P. A. (2000) J. Biol. Chem. 275, 12953-21959, should have appeared in the final version of the Discussion section.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYL9bSHcbmzLVg90H21EOLACvtfcHk0ljngeyLHm19CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotVGgu78%253D&md5=62bf2f1230a44c80c460f22371141b60</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi005121q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi005121q%26sid%3Dliteratum%253Aachs%26aulast%3DTanner%26aufirst%3DK.%2BG.%26aulast%3DLanger%26aufirst%3DM.%2BR.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DKinetic%2520Mechanism%2520of%2520Human%2520Histone%2520Acetyltransferase%2520P%252FCaf%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D15652%26doi%3D10.1021%2Fbi005121q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wapenaar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Wouden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groves, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, F. J.</span></span> <span> </span><span class="NLM_article-title">Enzyme Kinetics and Inhibition of Histone Acetyltransferase Kat8</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.ejmech.2015.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=26505788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslensr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2015&pages=289-296&author=H.+Wapenaarauthor=P.+E.+van+der+Woudenauthor=M.+R.+Grovesauthor=D.+Rotiliauthor=A.+Maiauthor=F.+J.+Dekker&title=Enzyme+Kinetics+and+Inhibition+of+Histone+Acetyltransferase+Kat8&doi=10.1016%2Fj.ejmech.2015.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme kinetics and inhibition of histone acetyltransferase KAT8</span></div><div class="casAuthors">Wapenaar, Hannah; van der Wouden, Petra E.; Groves, Matthew R.; Rotili, Dante; Mai, Antonello; Dekker, Frank J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">289-296</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Lysine acetyltransferase 8 (KAT8) is a histone acetyltransferase (HAT) responsible for acetylating lysine 16 on histone H4 (H4K16) and plays a role in cell cycle progression as well as acetylation of the tumor suppressor protein p53.  Further studies on its biol. function and drug discovery initiatives will benefit from the development of small mol. inhibitors for this enzyme.  As a first step towards this aim we investigated the enzyme kinetics of this bi-substrate enzyme.  The kinetic expts. indicate a ping-pong mechanism in which the enzyme binds Ac-CoA first, followed by binding of the histone substrate.  This mechanism is supported by affinity measurements of both substrates using isothermal titrn. calorimetry (ITC).  Using this information, the KAT8 inhibition of a focused compd. collection around the non-selective HAT inhibitor anacardic acid has been investigated.  Kinetic studies with anacardic acid were performed, based on which a model for the catalytic activity of KAT8 and the inhibitory action of anacardic acid (AA) was proposed.  This enabled the calcn. of the inhibition const. Ki of anacardic acid derivs. using an adaptation of the Cheng-Prusoff equation.  The results described in this study give insight into the catalytic mechanism of KAT8 and present the first well-characterized small-mol. inhibitors for this HAT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPdUOXyrmJwLVg90H21EOLACvtfcHk0ljngeyLHm19CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslensr7L&md5=333561a2d8d12536c62cbd40e3ba1a00</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DWapenaar%26aufirst%3DH.%26aulast%3Dvan%2Bder%2BWouden%26aufirst%3DP.%2BE.%26aulast%3DGroves%26aufirst%3DM.%2BR.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DDekker%26aufirst%3DF.%2BJ.%26atitle%3DEnzyme%2520Kinetics%2520and%2520Inhibition%2520of%2520Histone%2520Acetyltransferase%2520Kat8%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D105%26spage%3D289%26epage%3D296%26doi%3D10.1016%2Fj.ejmech.2015.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaczmarska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goudarzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Márquez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panne, D.</span></span> <span> </span><span class="NLM_article-title">Structure of P300 in Complex with Acyl-Coa Variants</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnchembio.2217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=27820805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=21-29&author=Z.+Kaczmarskaauthor=E.+Ortegaauthor=A.+Goudarziauthor=H.+Huangauthor=S.+Kimauthor=J.+A.+M%C3%A1rquezauthor=Y.+Zhaoauthor=S.+Khochbinauthor=D.+Panne&title=Structure+of+P300+in+Complex+with+Acyl-Coa+Variants&doi=10.1038%2Fnchembio.2217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of p300 in complex with acyl-CoA variants</span></div><div class="casAuthors">Kaczmarska, Zuzanna; Ortega, Esther; Goudarzi, Afsaneh; Huang, He; Kim, Sunjoo; Marquez, Jose A.; Zhao, Yingming; Khochbin, Saadi; Panne, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-29</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone acetylation plays an important role in transcriptional activation.  Histones are also modified by chem. diverse acylations that are frequently deposited by p300, a transcriptional coactivator that uses a no. of different acyl-CoA cofactors.  Here, the authors report that while p300 is a robust acetylase, its activity gets weaker with increasing acyl-CoA chain length.  Crystal structures of p300 in complex with propionyl-, crotonyl-, or butyryl-CoA showed that the aliph. portions of these cofactors were bound in the lysine substrate-binding tunnel in a conformation that was incompatible with substrate transfer.  Lysine substrate binding was predicted to remodel the acyl-CoA ligands into a conformation compatible with acyl-chain transfer.  This remodeling required that the aliph. portion of acyl-CoA be accommodated in a hydrophobic pocket in the enzyme active site.  The size of the pocket and its aliph. nature exclude long-chain and charged acyl-CoA variants, presumably explaining the cofactor preference for p300.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGKLa3WbifTrVg90H21EOLACvtfcHk0ljngeyLHm19CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurzK&md5=4db48a8cd73d48bcb8d94bb980e064a0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2217%26sid%3Dliteratum%253Aachs%26aulast%3DKaczmarska%26aufirst%3DZ.%26aulast%3DOrtega%26aufirst%3DE.%26aulast%3DGoudarzi%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DM%25C3%25A1rquez%26aufirst%3DJ.%2BA.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DKhochbin%26aufirst%3DS.%26aulast%3DPanne%26aufirst%3DD.%26atitle%3DStructure%2520of%2520P300%2520in%2520Complex%2520with%2520Acyl-Coa%2520Variants%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D21%26epage%3D29%26doi%3D10.1038%2Fnchembio.2217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlick, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, J. L.</span></span> <span> </span><span class="NLM_article-title">Global Profiling of Acetyltransferase Feedback Regulation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">6388</span>– <span class="NLM_lpage">6391</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b03036</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b03036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlCmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=6388-6391&author=D.+C.+Montgomeryauthor=J.+M.+Garlickauthor=R.+A.+Kulkarniauthor=S.+Kennedyauthor=A.+Allali-Hassaniauthor=Y.-M.+Kuoauthor=A.+J.+Andrewsauthor=H.+Wuauthor=M.+Vedadiauthor=J.+L.+Meier&title=Global+Profiling+of+Acetyltransferase+Feedback+Regulation&doi=10.1021%2Fjacs.6b03036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Global Profiling of Acetyltransferase Feedback Regulation</span></div><div class="casAuthors">Montgomery, David C.; Garlick, Julie M.; Kulkarni, Rhushikesh A.; Kennedy, Steven; Allali-Hassani, Abdellah; Kuo, Yin-Ming; Andrews, Andrew J.; Wu, Hong; Vedadi, Masoud; Meier, Jordan L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6388-6391</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine acetyltransferases (KATs) are key mediators of cell signaling.  Methods capable of providing new insights into their regulation thus constitute an important goal.  Here we report an optimized platform for profiling KAT-ligand interactions in complex proteomes using inhibitor-functionalized capture resins.  This approach greatly expands the scope of KATs, KAT complexes, and CoA-dependent enzymes accessible to chemoproteomic methods.  This enhanced profiling platform is then applied in the most comprehensive anal. to date of KAT inhibition by the feedback metabolite CoA.  Our studies reveal that members of the KAT superfamily possess a spectrum of sensitivity to CoA and highlight NAT10 as a novel KAT that may be susceptible to metabolic feedback inhibition.  This platform provides a powerful tool to define the potency and selectivity of reversible stimuli, such as small mols. and metabolites, that regulate KAT-dependent signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcly-8PByigLVg90H21EOLACvtfcHk0ljngeyLHm19CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlCmur0%253D&md5=426663d4fb51b9da29bfc2b778b61bdc</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b03036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b03036%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DD.%2BC.%26aulast%3DGarlick%26aufirst%3DJ.%2BM.%26aulast%3DKulkarni%26aufirst%3DR.%2BA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKuo%26aufirst%3DY.-M.%26aulast%3DAndrews%26aufirst%3DA.%2BJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMeier%26aufirst%3DJ.%2BL.%26atitle%3DGlobal%2520Profiling%2520of%2520Acetyltransferase%2520Feedback%2520Regulation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D6388%26epage%3D6391%26doi%3D10.1021%2Fjacs.6b03036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldenburg, K. R.</span></span> <span> </span><span class="NLM_article-title">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span>. <i>J. Biomol. Screen</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1177/108705719900400206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1177%2F108705719900400206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10838414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=67-73&author=J.-H.+Zhangauthor=T.+D.+Chungauthor=K.+R.+Oldenburg&title=A+Simple+Statistical+Parameter+for+Use+in+Evaluation+and+Validation+of+High+Throughput+Screening+Assays&doi=10.1177%2F108705719900400206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span></div><div class="casAuthors">Zhang; Chung; Oldenburg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ability to identify active compounds (3hits2) from large chemical libraries accurately and rapidly has been the ultimate goal in developing high-throughput screening (HTS) assays.  The ability to identify hits from a particular HTS assay depends largely on the suitability or quality of the assay used in the screening.  The criteria or parameters for evaluating the 3suitability2 of an HTS assay for hit identification are not well defined and hence it still remains difficult to compare the quality of assays directly.  In this report, a screening window coefficient, called 3Z-factor,2 is defined.  This coefficient is reflective of both the assay signal dynamic range and the data variation associated with the signal measurements, and therefore is suitable for assay quality assessment.  The Z-factor is a dimensionless, simple statistical characteristic for each HTS assay.  The Z-factor provides a useful tool for comparison and evaluation of the quality of assays, and can be utilized in assay optimization and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkwW5JDdXnbsNIRw0GKUoEfW6udTcc2ebFd-GuJ1qflLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D&md5=0ec4ed2ccdd3b4a16521567a0975d31f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1177%2F108705719900400206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705719900400206%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.-H.%26aulast%3DChung%26aufirst%3DT.%2BD.%26aulast%3DOldenburg%26aufirst%3DK.%2BR.%26atitle%3DA%2520Simple%2520Statistical%2520Parameter%2520for%2520Use%2520in%2520Evaluation%2520and%2520Validation%2520of%2520High%2520Throughput%2520Screening%2520Assays%26jtitle%3DJ.%2520Biomol.%2520Screen%26date%3D1999%26volume%3D4%26spage%3D67%26epage%3D73%26doi%3D10.1177%2F108705719900400206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seethala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzel, R.</span></span> <span> </span><span class="NLM_article-title">A Fluorescence Polarization Competition Immunoassay for Tyrosine Kinases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>255</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1006/abio.1997.2455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1006%2Fabio.1997.2455" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=255&publication_year=1998&pages=257-262&author=R.+Seethalaauthor=R.+Menzel&title=A+Fluorescence+Polarization+Competition+Immunoassay+for+Tyrosine+Kinases&doi=10.1006%2Fabio.1997.2455"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1006%2Fabio.1997.2455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1997.2455%26sid%3Dliteratum%253Aachs%26aulast%3DSeethala%26aufirst%3DR.%26aulast%3DMenzel%26aufirst%3DR.%26atitle%3DA%2520Fluorescence%2520Polarization%2520Competition%2520Immunoassay%2520for%2520Tyrosine%2520Kinases%26jtitle%3DAnal.%2520Biochem.%26date%3D1998%26volume%3D255%26spage%3D257%26epage%3D262%26doi%3D10.1006%2Fabio.1997.2455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yung-Chi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the inhibition constant (K1) and the concentration of inhibitor which causes 50 per cent inhibition (I50) of an enzymatic reaction</span>. <i>Biochem Pharmacol</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28K1%29+and+the+concentration+of+inhibitor+which+causes+50+per+cent+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528K1%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2520per%2520cent%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem%2520Pharmacol%26date%3D1973%26volume%3D22%26spage%3D3099%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ringel, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolberger, C.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Acyl-Group Discrimination by Human Gcn5l2</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">848</span>, <span class="refDoi"> DOI: 10.1107/s2059798316007907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1107%2Fs2059798316007907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSntr7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=841-848&author=A.+E.+Ringelauthor=C.+Wolberger&title=Structural+Basis+for+Acyl-Group+Discrimination+by+Human+Gcn5l2&doi=10.1107%2Fs2059798316007907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for acyl-group discrimination by human Gcn5L2</span></div><div class="casAuthors">Ringel, Alison E.; Wolberger, Cynthia</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">841-848</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Gcn5 is a conserved acetyltransferase that regulates transcription by acetylating the N-terminal tails of histones.  Motivated by recent studies identifying a chem. diverse array of lysine acyl modifications in vivo, the acyl-chain specificity of the acetyltransferase human Gcn5 (Gcn5L2) was examd.  Whereas Gcn5L2 robustly catalyzes lysine acetylation, the acyltransferase activity of Gcn5L2 becomes progressively weaker with increasing acyl-chain length.  To understand how Gcn5 discriminates between different acyl-CoA mols., structures of the catalytic domain of human Gcn5L2 bound to propionyl-CoA and butyryl-CoA were detd.  Although the active site of Gcn5L2 can accommodate propionyl-CoA and butyryl-CoA without major structural rearrangements, butyryl-CoA adopts a conformation incompatible with catalysis that obstructs the path of the incoming lysine residue and acts as a competitive inhibitor of Gcn5L2 vs. acetyl-CoA.  These structures demonstrate how Gcn5L2 discriminates between acyl-chain donors and explain why Gcn5L2 has weak activity for acyl moieties that are larger than an acetyl group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa0SleEtFeYbVg90H21EOLACvtfcHk0ljfNrDSYLFCXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSntr7M&md5=df96aaf49e264a661821a765e8774844</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2Fs2059798316007907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs2059798316007907%26sid%3Dliteratum%253Aachs%26aulast%3DRingel%26aufirst%3DA.%2BE.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DStructural%2520Basis%2520for%2520Acyl-Group%2520Discrimination%2520by%2520Human%2520Gcn5l2%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2016%26volume%3D72%26spage%3D841%26epage%3D848%26doi%3D10.1107%2Fs2059798316007907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clements, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Histone Acetyltransferase Domain of the Human Pcaf Transcriptional Regulator Bound to Coenzyme A</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3521</span>– <span class="NLM_lpage">3532</span>, <span class="refDoi"> DOI: 10.1093/emboj/18.13.3521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1093%2Femboj%2F18.13.3521" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=3521-3532&author=A.+Clementsauthor=J.+R.+Rojasauthor=R.+C.+Trievelauthor=L.+Wangauthor=S.+L.+Bergerauthor=R.+Marmorstein&title=Crystal+Structure+of+the+Histone+Acetyltransferase+Domain+of+the+Human+Pcaf+Transcriptional+Regulator+Bound+to+Coenzyme+A&doi=10.1093%2Femboj%2F18.13.3521"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F18.13.3521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F18.13.3521%26sid%3Dliteratum%253Aachs%26aulast%3DClements%26aufirst%3DA.%26aulast%3DRojas%26aufirst%3DJ.%2BR.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Histone%2520Acetyltransferase%2520Domain%2520of%2520the%2520Human%2520Pcaf%2520Transcriptional%2520Regulator%2520Bound%2520to%2520Coenzyme%2520A%26jtitle%3DEMBO%2520J.%26date%3D1999%26volume%3D18%26spage%3D3521%26epage%3D3532%26doi%3D10.1093%2Femboj%2F18.13.3521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">The Structural Basis of Protein Acetylation by the P300/Cbp Transcriptional Coactivator</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>451</i></span>,  <span class="NLM_fpage">846</span>– <span class="NLM_lpage">850</span>, <span class="refDoi"> DOI: 10.1038/nature06546</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnature06546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=18273021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvVyms74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2008&pages=846-850&author=X.+Liuauthor=L.+Wangauthor=K.+Zhaoauthor=P.+R.+Thompsonauthor=Y.+Hwangauthor=R.+Marmorsteinauthor=P.+A.+Cole&title=The+Structural+Basis+of+Protein+Acetylation+by+the+P300%2FCbp+Transcriptional+Coactivator&doi=10.1038%2Fnature06546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis of protein acetylation by the p300/CBP transcriptional coactivator</span></div><div class="casAuthors">Liu, Xin; Wang, Ling; Zhao, Kehao; Thompson, Paul R.; Hwang, Yousang; Marmorstein, Ronen; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">7180</span>),
    <span class="NLM_cas:pages">846-850</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The transcriptional coactivator p300/CBP (CREBBP) is a histone acetyltransferase (HAT) that regulates gene expression by acetylating histones and other transcription factors.  Dysregulation of p300/CBP HAT activity contributes to various diseases including cancer.  Sequence alignments, enzymol. expts. and inhibitor studies on p300/CBP have led to contradictory results about its catalytic mechanism and its structural relation to the Gcn5/PCAF and MYST HATs.  Here we describe a high-resoln. x-ray crystal structure of a semi-synthetic heterodimeric p300 HAT domain in complex with a bi-substrate inhibitor, Lys-CoA.  This structure shows that p300/CBP is a distant cousin of other structurally characterized HATs, but reveals several novel features that explain the broad substrate specificity and preference for nearby basic residues.  Based on this structure and accompanying biochem. data, we propose that p300/CBP uses an unusual 'hit-and-run' (Theorell-Chance) catalytic mechanism that is distinct from other characterized HATs.  Several disease-assocd. mutations can also be readily accounted for by the p300 HAT structure.  These studies pave the way for new epigenetic therapies involving modulation of p300/CBP HAT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeYOui0tFND7Vg90H21EOLACvtfcHk0lj1Q-2BeeP8qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvVyms74%253D&md5=c393f167458f8532f2d3b06e96f7eb00</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnature06546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06546%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DHwang%26aufirst%3DY.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DThe%2520Structural%2520Basis%2520of%2520Protein%2520Acetylation%2520by%2520the%2520P300%252FCbp%2520Transcriptional%2520Coactivator%26jtitle%3DNature%26date%3D2008%26volume%3D451%26spage%3D846%26epage%3D850%26doi%3D10.1038%2Fnature06546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadlec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallacli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Weatherby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, A.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Mof and Msl3 Recruitment into the Dosage Compensation Complex by Msl1</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">142</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1038/nsmb.1960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnsmb.1960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=21217699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=142-149&author=J.+Kadlecauthor=E.+Hallacliauthor=M.+Lippauthor=H.+Holzauthor=J.+Sanchez-Weatherbyauthor=S.+Cusackauthor=A.+Akhtar&title=Structural+Basis+for+Mof+and+Msl3+Recruitment+into+the+Dosage+Compensation+Complex+by+Msl1&doi=10.1038%2Fnsmb.1960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for MOF and MSL3 recruitment into the dosage compensation complex by MSL1</span></div><div class="casAuthors">Kadlec, Jan; Hallacli, Erinc; Lipp, Michael; Holz, Herbert; Sanchez-Weatherby, Juan; Cusack, Stephen; Akhtar, Asifa</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">142-149</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The male-specific lethal (MSL) complex is required for dosage compensation in Drosophila melanogaster, and analogous complexes exist in mammals.  We report structures of binary complexes of mammalian MSL3 and the histone acetyltransferase (HAT) MOF with consecutive segments of MSL1.  MSL1 interacts with MSL3 as an extended chain forming an extensive hydrophobic interface, whereas the MSL1-MOF interface involves electrostatic interactions between the HAT domain and a long helix of MSL1.  This structure provides insights into the catalytic mechanism of MOF and enables us to show analogous interactions of MOF with NSL1.  In Drosophila, selective disruption of Msl1 interactions with Msl3 or Mof severely affects Msl1 targeting to the body of dosage-compensated genes and several high-affinity sites, without affecting promoter binding.  We propose that Msl1 acts as a scaffold for MSL complex assembly to achieve specific targeting to the X chromosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLrkomJizA6LVg90H21EOLACvtfcHk0lj1Q-2BeeP8qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtVKrtA%253D%253D&md5=92dd57cdb95159c724113fd5aaba9557</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1960%26sid%3Dliteratum%253Aachs%26aulast%3DKadlec%26aufirst%3DJ.%26aulast%3DHallacli%26aufirst%3DE.%26aulast%3DLipp%26aufirst%3DM.%26aulast%3DHolz%26aufirst%3DH.%26aulast%3DSanchez-Weatherby%26aufirst%3DJ.%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DAkhtar%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520for%2520Mof%2520and%2520Msl3%2520Recruitment%2520into%2520the%2520Dosage%2520Compensation%2520Complex%2520by%2520Msl1%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D142%26epage%3D149%26doi%3D10.1038%2Fnsmb.1960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cebrat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugherty, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Analysis of Potential Prodrugs of Coenzyme a Analogues for the Inhibition of the Histone Acetyltransferase P300</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3307</span>– <span class="NLM_lpage">3313</span>, <span class="refDoi"> DOI: 10.1016/s0968-0896(03)00265-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fs0968-0896%2803%2900265-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=3307-3313&author=M.+Cebratauthor=C.+M.+Kimauthor=P.+R.+Thompsonauthor=M.+Daughertyauthor=P.+A.+Cole&title=Synthesis+and+Analysis+of+Potential+Prodrugs+of+Coenzyme+a+Analogues+for+the+Inhibition+of+the+Histone+Acetyltransferase+P300&doi=10.1016%2Fs0968-0896%2803%2900265-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fs0968-0896%2803%2900265-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0968-0896%252803%252900265-7%26sid%3Dliteratum%253Aachs%26aulast%3DCebrat%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DC.%2BM.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DDaugherty%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DSynthesis%2520and%2520Analysis%2520of%2520Potential%2520Prodrugs%2520of%2520Coenzyme%2520a%2520Analogues%2520for%2520the%2520Inhibition%2520of%2520the%2520Histone%2520Acetyltransferase%2520P300%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26spage%3D3307%26epage%3D3313%26doi%3D10.1016%2Fs0968-0896%2803%2900265-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Enzymatic and Cellular Study of a Serotonin N-Acetyltransferase Phosphopantetheine-Based Prodrug</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2147</span>– <span class="NLM_lpage">2155</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2006.12.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.bmc.2006.12.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=2147-2155&author=Y.+Hwangauthor=S.+Gangulyauthor=A.+K.+Hoauthor=D.+C.+Kleinauthor=P.+A.+Cole&title=Enzymatic+and+Cellular+Study+of+a+Serotonin+N-Acetyltransferase+Phosphopantetheine-Based+Prodrug&doi=10.1016%2Fj.bmc.2006.12.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.12.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.12.016%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DY.%26aulast%3DGanguly%26aufirst%3DS.%26aulast%3DHo%26aufirst%3DA.%2BK.%26aulast%3DKlein%26aufirst%3DD.%2BC.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DEnzymatic%2520and%2520Cellular%2520Study%2520of%2520a%2520Serotonin%2520N-Acetyltransferase%2520Phosphopantetheine-Based%2520Prodrug%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D2147%26epage%3D2155%26doi%3D10.1016%2Fj.bmc.2006.12.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B. D.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery and Its Application to Challenging Drug Targets</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1042/ebc20170029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1042%2Febc20170029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=29118094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=475-484&author=A.+J.+Priceauthor=S.+Howardauthor=B.+D.+Cons&title=Fragment-Based+Drug+Discovery+and+Its+Application+to+Challenging+Drug+Targets&doi=10.1042%2Febc20170029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery and its application to challenging drug targets</span></div><div class="casAuthors">Price Amanda J; Howard Steven; Cons Benjamin D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">475-484</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a technique for identifying low molecular weight chemical starting points for drug discovery.  Since its inception 20 years ago, FBDD has grown in popularity to the point where it is now an established technique in industry and academia.  The approach involves the biophysical screening of proteins against collections of low molecular weight compounds (fragments).  Although fragments bind to proteins with relatively low affinity, they form efficient, high quality binding interactions with the protein architecture as they have to overcome a significant entropy barrier to bind.  Of the biophysical methods available for fragment screening, X-ray protein crystallography is one of the most sensitive and least prone to false positives.  It also provides detailed structural information of the protein-fragment complex at the atomic level.  Fragment-based screening using X-ray crystallography is therefore an efficient method for identifying binding hotspots on proteins, which can then be exploited by chemists and biologists for the discovery of new drugs.  The use of FBDD is illustrated here with a recently published case study of a drug discovery programme targeting the challenging protein-protein interaction Kelch-like ECH-associated protein 1:nuclear factor erythroid 2-related factor 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0VHFYivNCdPRfW6udTcc2ebC7bWzuFnQTrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D&md5=f8572516bfb1ecc4a0ef57b0c5ba1bc0</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1042%2Febc20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Febc20170029%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DCons%26aufirst%3DB.%2BD.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520and%2520Its%2520Application%2520to%2520Challenging%2520Drug%2520Targets%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D475%26epage%3D484%26doi%3D10.1042%2Febc20170029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yihang Jing, Jose L. Montano, Michaella Levy, Jeffrey E. Lopez, Pei-Pei Kung, Paul Richardson, Krzysztof Krajewski, Laurence Florens, Michael P. Washburn, <span class="NLM_string-name hlFld-ContribAuthor">Jordan L. Meier</span>. </span><span class="cited-content_cbyCitation_article-title">Harnessing Ionic Selectivity in Acetyltransferase Chemoproteomic Probes. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (1)
                                     , 27-34. <a href="https://doi.org/10.1021/acschembio.0c00766" title="DOI URL">https://doi.org/10.1021/acschembio.0c00766</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00766%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DHarnessing%252BIonic%252BSelectivity%252Bin%252BAcetyltransferase%252BChemoproteomic%252BProbes%26aulast%3DJing%26aufirst%3DYihang%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29092020%26date%3D16122020%26date%3D29122020%26volume%3D16%26issue%3D1%26spage%3D27%26epage%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giordano  Proietti</span>, <span class="hlFld-ContribAuthor ">Yali  Wang</span>, <span class="hlFld-ContribAuthor ">Giorgio  Rainone</span>, <span class="hlFld-ContribAuthor ">Jasmin  Mecinović</span>. </span><span class="cited-content_cbyCitation_article-title">Effect of lysine side chain length on histone lysine acetyltransferase catalysis. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-69510-0" title="DOI URL">https://doi.org/10.1038/s41598-020-69510-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-69510-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-69510-0%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DEffect%252Bof%252Blysine%252Bside%252Bchain%252Blength%252Bon%252Bhistone%252Blysine%252Bacetyltransferase%252Bcatalysis%26aulast%3DProietti%26aufirst%3DGiordano%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krzysztof  Stolarczyk</span>, <span class="hlFld-ContribAuthor ">Jerzy  Rogalski</span>, <span class="hlFld-ContribAuthor ">Renata  Bilewicz</span>. </span><span class="cited-content_cbyCitation_article-title">NAD(P)-dependent glucose dehydrogenase: Applications for biosensors, bioelectrodes, and biofuel cells. </span><span class="cited-content_cbyCitation_journal-name">Bioelectrochemistry</span><span> <strong>2020,</strong> <em>135 </em>, 107574. <a href="https://doi.org/10.1016/j.bioelechem.2020.107574" title="DOI URL">https://doi.org/10.1016/j.bioelechem.2020.107574</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioelechem.2020.107574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioelechem.2020.107574%26sid%3Dliteratum%253Aachs%26jtitle%3DBioelectrochemistry%26atitle%3DNAD%252528P%252529-dependent%252Bglucose%252Bdehydrogenase%25253A%252BApplications%252Bfor%252Bbiosensors%25252C%252Bbioelectrodes%25252C%252Band%252Bbiofuel%252Bcells%26aulast%3DStolarczyk%26aufirst%3DKrzysztof%26date%3D2020%26volume%3D135%26spage%3D107574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiabao  Song</span>, <span class="hlFld-ContribAuthor ">Y. George  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Bioorthogonal Reporters for Detecting and Profiling Protein Acetylation and Acylation. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2020,</strong> <em>25 </em>
                                    (2)
                                     , 148-162. <a href="https://doi.org/10.1177/2472555219887144" title="DOI URL">https://doi.org/10.1177/2472555219887144</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555219887144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555219887144%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DBioorthogonal%252BReporters%252Bfor%252BDetecting%252Band%252BProfiling%252BProtein%252BAcetylation%252Band%252BAcylation%26aulast%3DSong%26aufirst%3DJiabao%26date%3D2020%26date%3D2019%26volume%3D25%26issue%3D2%26spage%3D148%26epage%3D162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. General structures of selected CoA-derived metabolites, KAT inhibitors, and chemical probes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of new functionally labeled cofactor analogues <b>11</b> and <b>12</b> from this study. Labels are colored in orange and lysine mimicking linkers are depicted in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Cofactor-Based Probes<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions (a) propargylamine, HBTU, DIPEA, DCM, rt, overnight; (b) TFA/DCM 1:1, 1 h then chloroacetyl chloride, DMF, 0 °C to rt, 1 h; (c) 1<i>H</i>-imidazole-1-sulfonyl azide × HCl, CuSO<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, MeOH, rt, overnight; (d) 1. (COCl)<sub>2</sub>, cat. DMF, DCM, rt, 5 h; 2. 6-aminofluorescein, NaHCO<sub>3</sub>, THF, rt, overnight; (e) coenzyme A, 20% ethanol in 0.1 M TEAB buffer pH 8, 0 °C to rt, 2 h; (f): alkyl azide, TBTA, CuSO<sub>4</sub>, sodium ascorbate, <i>tert</i>-BuOH/H<sub>2</sub>O/DMF (1:1:1), rt, 12 h. DIPEA = <i>N</i>,<i>N</i>-diisopropylethylamine, rt = room temperature, TEAB = triethylammonium bicarbonate, TBTA = Tris[(1-benzyl-1<i>H</i>-1,2,3-triazol-4-yl)methyl]amine.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Schematic depictions of the FP and BLI assay and response plots of pCAF(493-658) within these assay systems. (a) Setup of the FP assay. The FP signal is dependent on the rotational freedom of the fluorescent tracer, which is influenced by the presence of competing ligands. (b) Saturation binding curve of pCAF(493-658) binding to 8 nM of <b>11</b> and the dose–response curve of reference inhibitor acetonyl-CoA against 400 nM pCAF and 8 nM <b>11</b> after different time points. Data represent the mean and standard deviation of triplicate measurements. (c) Setup of the BLI assay. Compound <b>12</b> was immobilized on streptavidin coated sensors and the association and dissociation of enzyme in the presence of possible competitors is monitored by measuring the optical interference pattern of two light beams within the sensor. (d) Sensorgrams and saturation binding curve of a concentration series of pCAF(493-658) in the BLI assay. (e) Sensorgrams and dose–response curve of the reference inhibitor titration against 250 nM enzyme. Data represent the mean and standard deviation of triplicate measurements. Results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Results from the method transfer to other KATs. (a) Titration curves of p300(1284-1673) and MOF(174-449) against 8 nM <b>11</b> and dose–response curves of reference inhibitor against both enzymes at optimized assay concentrations of 120 and 800 nM, respectively. (b) Sensorgrams and corresponding dose–response curves from BLI experiments measuring the concentration dependent competition of acetonyl-CoA with p300(1284-1673) and MOF(174-449) for binding to immobilized <b>12</b>. Data represent the mean and standard deviation of duplicate measurements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Results from antibody and enzyme titrations. (a) Titration curves of anti-acH4-Ab and anti-acH3K9-Ab against 3 nM acH4-Fl and 8 nM acH3K9-Rh tracer peptide, respectively, after 30 min incubation. (b) Concentration–response curves for acetyltransferases after 60 min incubation with substrate (50 μM H3(1-21) for pCAF(493-658), 5 μM full length H4-His for MOF(174-449), 5 μM H4(1-23) for p300(1284-1673)), and 20 μM AcCoA. The degree of competition was measured after stopping the reaction by addition of the corresponding tracer/antibody pair. Data represent the mean and standard deviation of triplicate measurements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Schematic Depiction of the FP Competition Activity Immunoassay<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Substrate peptide is processed by the acetyltransferase in the course of the enzymatic reaction. After incubation, the reaction is stopped by addition of reference inhibitor and the tracer/receptor pair is added. Acetylated reaction product competes with a fluorescence labeled peptide (tracer) for binding to the modification specific antibody (receptor). The decrease in fluorescence polarization of the tracer peptide is proportional to its displacement from the antibody and therefore a measure for enzymatic turnover.</p></p></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Results from acyl-CoA profiling and binding site mapping. (a) Comparison of FP and BLI assay results from acyl-CoA profiling against pCAF(493-658). (b) Thermal shift profiles of acyl-CoAs interacting with 4 μM pCAF(493-658). The melting temperature shifts compared with apo-pCAF are depicted in the insertion. (c) Dose–response curves for acyl-CoA variants in the FP assay (top: pCAF(493-658); bottom: p300(1284-1673)). The curves for butyryl-CoA are depicted in red. (d) Dose–response curves of lauroyl-CoA (solid lines) and reference inhibitor acetonyl-CoA (dashed lines) in the FP competition activity immunoassay for p300 (orange) and pCAF (green). (e) Competition profiles of cofactor fragments against all three KAT enzymes in the FP assay. All data represent the mean and standard deviation of triplicate (pCAF) or duplicate (p300 and MOF) measurements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/medium/jm-2018-01887c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Results from ligand screening. (a) Bar graph and cumulative distribution of the primary screening. The threshold was set to 2 × SD (σ) over all measurements. (b) Thermal shifts of screening hits at 1 mM compound concentrations. Destabilizing compounds were sorted out. (c) BLI sensorgrams of hits from the confirmatory assay. (d) Structures of the three fragment screening hits.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01887/20190308/images/large/jm-2018-01887c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01887&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i54">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26347" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26347" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnad, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walther, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span> <span> </span><span class="NLM_article-title">Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>325</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1126/science.1175371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1126%2Fscience.1175371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=19608861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=834-840&author=C.+Choudharyauthor=C.+Kumarauthor=F.+Gnadauthor=M.+L.+Nielsenauthor=M.+Rehmanauthor=T.+C.+Waltherauthor=J.+V.+Olsenauthor=M.+Mann&title=Lysine+Acetylation+Targets+Protein+Complexes+and+Co-Regulates+Major+Cellular+Functions&doi=10.1126%2Fscience.1175371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions</span></div><div class="casAuthors">Choudhary, Chunaram; Kumar, Chanchal; Gnad, Florian; Nielsen, Michael L.; Rehman, Michael; Walther, Tobias C.; Olsen, Jesper V.; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5942</span>),
    <span class="NLM_cas:pages">834-840</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lysine acetylation is a reversible posttranslational modification of proteins and plays a key role in regulating gene expression.  Technol. limitations have so far prevented a global anal. of lysine acetylation's cellular roles.  We used high-resoln. mass spectrometry to identify 3600 lysine acetylation sites on 1750 proteins and quantified acetylation changes in response to the deacetylase inhibitors suberoylanilide hydroxamic acid and MS-275.  Lysine acetylation preferentially targets large macromol. complexes involved in diverse cellular processes, such as chromatin remodeling, cell cycle, splicing, nuclear transport, and actin nucleation.  Acetylation impaired phosphorylation-dependent interactions of 14-3-3 and regulated the yeast cyclin-dependent kinase Cdc28.  Our data demonstrate that the regulatory scope of lysine acetylation is broad and comparable with that of other major posttranslational modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5eICW58ngkrVg90H21EOLACvtfcHk0lgfZ7FolpqBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D&md5=8f643db5ba91c50f0b61763ff3095d2c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1175371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1175371%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DKumar%26aufirst%3DC.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DNielsen%26aufirst%3DM.%2BL.%26aulast%3DRehman%26aufirst%3DM.%26aulast%3DWalther%26aufirst%3DT.%2BC.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DLysine%2520Acetylation%2520Targets%2520Protein%2520Complexes%2520and%2520Co-Regulates%2520Major%2520Cellular%2520Functions%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D834%26epage%3D840%26doi%3D10.1126%2Fscience.1175371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K.-L.</span></span> <span> </span><span class="NLM_article-title">Regulation of Cellular Metabolism by Protein Lysine Acetylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1004</span>, <span class="refDoi"> DOI: 10.1126/science.1179689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1126%2Fscience.1179689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=20167786" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1000-1004&author=S.+Zhaoauthor=W.+Xuauthor=W.+Jiangauthor=W.+Yuauthor=Y.+Linauthor=T.+Zhangauthor=J.+Yaoauthor=L.+Zhouauthor=Y.+Zengauthor=H.+Liauthor=Y.+Liauthor=J.+Shiauthor=W.+Anauthor=S.+M.+Hancockauthor=F.+Heauthor=L.+Qinauthor=J.+Chinauthor=P.+Yangauthor=X.+Chenauthor=Q.+Leiauthor=Y.+Xiongauthor=K.-L.+Guan&title=Regulation+of+Cellular+Metabolism+by+Protein+Lysine+Acetylation&doi=10.1126%2Fscience.1179689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of cellular metabolism by protein lysine acetylation</span></div><div class="casAuthors">Zhao, Shimin; Xu, Wei; Jiang, Wenqing; Yu, Wei; Lin, Yan; Zhang, Tengfei; Yao, Jun; Zhou, Li; Zeng, Yaxue; Li, Hong; Li, Yixue; Shi, Jiong; An, Wenlin; Hancock, Susan M.; He, Fuchu; Qin, Lunxiu; Chin, Jason; Yang, Pengyuan; Chen, Xian; Lei, Qunying; Xiong, Yue; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5968</span>),
    <span class="NLM_cas:pages">1000-1004</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Protein Lys acetylation has emerged as a key post-translational modification in cellular regulation, in particular through the modification of histones and nuclear transcription regulators.  Here, the authors show that acetylation of Lys residues is a prevalent modification in enzymes that catalyze intermediate metab.  Virtually every enzyme in glycolysis, gluconeogenesis, the tricarboxylic acid (TCA) cycle, the urea cycle, fatty acid metab., and glycogen metab. was found to be acetylated in human liver tissue.  The concn. of metabolic fuels, such as glucose, amino acids, and fatty acids, influenced the acetylation status of metabolic enzymes.  Acetylation activated enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase in fatty acid oxidn. and malate dehydrogenase in the TCA cycle, inhibited argininosuccinate lyase in the urea cycle, and destabilized phosphoenolpyruvate carboxykinase in gluconeogenesis.  Thus, this study revealed that acetylation plays a major role in metabolic regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbdnDsuudXV7Vg90H21EOLACvtfcHk0lgfZ7FolpqBuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbo%253D&md5=d2c3c897077b9285842a7c55e8f8737e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179689%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DAn%26aufirst%3DW.%26aulast%3DHancock%26aufirst%3DS.%2BM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DChin%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLei%26aufirst%3DQ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DK.-L.%26atitle%3DRegulation%2520of%2520Cellular%2520Metabolism%2520by%2520Protein%2520Lysine%2520Acetylation%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1000%26epage%3D1004%26doi%3D10.1126%2Fscience.1179689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span> <span> </span><span class="NLM_article-title">Lysine Acetylation Goes Global: From Epigenetics to Metabolism and Therapeutics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1252</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00181</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00181" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1216-1252&author=I.+Aliauthor=R.+J.+Conradauthor=E.+Verdinauthor=M.+Ott&title=Lysine+Acetylation+Goes+Global%3A+From+Epigenetics+to+Metabolism+and+Therapeutics&doi=10.1021%2Facs.chemrev.7b00181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine acetylation goes global: From epigenetics to metabolism and therapeutics</span></div><div class="casAuthors">Ali, Ibraheem; Conrad, Ryan J.; Verdin, Eric; Ott, Melanie</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1216-1252</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The post-translational acetylation of protein Lys residues by protein lysine acetyltransferases has emerged as a key regulatory mechanism in all eukaryotic organisms.  Originally discovered in 1963 as a unique modification of histones, acetylation marks are now found on thousands of non-histone proteins located in virtually every cellular compartment.  Here, we summarize key findings in the field of protein acetylation over the past 20 yr with a focus on recent discoveries in nuclear, cytoplasmic, and mitochondrial compartments.  Collectively, these findings have elevated protein acetylation as a major post-translational modification, underscoring its physiol. relevance in gene regulation, cell signaling, metab., and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWVJmagBGDLLVg90H21EOLACvtfcHk0liqphxrNHkonQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKnsrw%253D&md5=ecb4497c9a416e806182b2e3ef16df55</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00181%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DI.%26aulast%3DConrad%26aufirst%3DR.%2BJ.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DLysine%2520Acetylation%2520Goes%2520Global%253A%2520From%2520Epigenetics%2520to%2520Metabolism%2520and%2520Therapeutics%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D1216%26epage%3D1252%26doi%3D10.1021%2Facs.chemrev.7b00181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kho, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grishin, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Substrate and Functional Diversity of Lysine Acetylation Revealed by a Proteomics Survey</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">607</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.06.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.molcel.2006.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=16916647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1Snurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=607-618&author=S.+C.+Kimauthor=R.+Sprungauthor=Y.+Chenauthor=Y.+Xuauthor=H.+Ballauthor=J.+Peiauthor=T.+Chengauthor=Y.+Khoauthor=H.+Xiaoauthor=L.+Xiaoauthor=N.+V.+Grishinauthor=M.+Whiteauthor=X.-J.+Yangauthor=Y.+Zhao&title=Substrate+and+Functional+Diversity+of+Lysine+Acetylation+Revealed+by+a+Proteomics+Survey&doi=10.1016%2Fj.molcel.2006.06.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate and functional diversity of lysine acetylation revealed by a proteomics survey</span></div><div class="casAuthors">Kim, Sung Chan; Sprung, Robert; Chen, Yue; Xu, Yingda; Ball, Haydn; Pei, Jimin; Cheng, Tzuling; Kho, Yoonjung; Xiao, Hao; Xiao, Lin; Grishin, Nick V.; White, Michael; Yang, Xiang-Jiao; Zhao, Yingming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">607-618</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Acetylation of proteins on lysine residues is a dynamic posttranslational modification that is known to play a key role in regulating transcription and other DNA-dependent nuclear processes.  However, the extent of this modification in diverse cellular proteins remains largely unknown, presenting a major bottleneck for lysine-acetylation biol.  Here we report the first proteomic survey of this modification, identifying 388 acetylation sites in 195 proteins among proteins derived from HeLa cells and mouse liver mitochondria.  In addn. to regulators of chromatin-based cellular processes, nonnuclear localized proteins with diverse functions were identified.  Most strikingly, acetyllysine was found in more than 20% of mitochondrial proteins, including many longevity regulators and metab. enzymes.  Our study reveals previously unappreciated roles for lysine acetylation in the regulation of diverse cellular pathways outside of the nucleus.  The combined data sets offer a rich source for further characterization of the contribution of this modification to cellular physiol. and human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo05vjtLKl917Vg90H21EOLACvtfcHk0liqphxrNHkonQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1Snurk%253D&md5=b8f45b2a9461d0824fc49aefcc038eba</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BC.%26aulast%3DSprung%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DBall%26aufirst%3DH.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DKho%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DGrishin%26aufirst%3DN.%2BV.%26aulast%3DWhite%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DX.-J.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DSubstrate%2520and%2520Functional%2520Diversity%2520of%2520Lysine%2520Acetylation%2520Revealed%2520by%2520a%2520Proteomics%2520Survey%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D607%26epage%3D618%26doi%3D10.1016%2Fj.molcel.2006.06.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leemhuis, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packman, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nightingale, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollfelder, F.</span></span> <span> </span><span class="NLM_article-title">The Human Histone Acetyltransferase P/Caf Is a Promiscuous Histone Propionyltransferase</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">499</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1002/cbic.200700556</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1002%2Fcbic.200700556" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=18247445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1Wht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=499-503&author=H.+Leemhuisauthor=L.+C.+Packmanauthor=K.+P.+Nightingaleauthor=F.+Hollfelder&title=The+Human+Histone+Acetyltransferase+P%2FCaf+Is+a+Promiscuous+Histone+Propionyltransferase&doi=10.1002%2Fcbic.200700556"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The human histone acetyltransferase P/CAF is a promiscuous histone propionyltransferase</span></div><div class="casAuthors">Leemhuis, Hans; Packman, Len C.; Nightingale, Karl P.; Hollfelder, Florian</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">499-503</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Revisiting previous mass spectrometric fragmentation patterns of nuclear proteins from human HeLa S3 cells revealed the existence of two new histone marks, namely propionylated and butyrylated lysines on H4 histones.  This observation raises the question how these modifications were brought about.  In vitro anal. indicates that several HATs can both acetylate and propionylate H4 peptides, although the lysine specificity of these modifications was not explored.  The authors now report that the human histone acetyltransferase P/CAF has the ability to transfer propionyl and acetyl groups with similar efficiency and specificity.  Taken together, the observation of propionyl marks in biol. samples, the substantial and residue-specific promiscuous histone propionyltransferase activity of a human enzyme identified here, and the existence of depropionylating activities make it conceivable that histone propionylation activity has a regulatory biol. role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl31vgEvRUm7Vg90H21EOLACvtfcHk0liqphxrNHkonQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1Wht7o%253D&md5=bddcacc6988392dd5a17c3e173cf6b70</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200700556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200700556%26sid%3Dliteratum%253Aachs%26aulast%3DLeemhuis%26aufirst%3DH.%26aulast%3DPackman%26aufirst%3DL.%2BC.%26aulast%3DNightingale%26aufirst%3DK.%2BP.%26aulast%3DHollfelder%26aufirst%3DF.%26atitle%3DThe%2520Human%2520Histone%2520Acetyltransferase%2520P%252FCaf%2520Is%2520a%2520Promiscuous%2520Histone%2520Propionyltransferase%26jtitle%3DChemBioChem%26date%3D2008%26volume%3D9%26spage%3D499%26epage%3D503%26doi%3D10.1002%2Fcbic.200700556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-j.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Revealing the Protein Propionylation Activity of the Histone Acetyltransferase Mof (Males Absent on the First)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">3410</span>– <span class="NLM_lpage">3420</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra117.000529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1074%2Fjbc.ra117.000529" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=3410-3420&author=Z.+Hanauthor=H.+Wuauthor=S.+Kimauthor=X.+Yangauthor=Q.+Liauthor=H.+Huangauthor=H.+Caiauthor=M.+G.+Bartlettauthor=A.+Dongauthor=H.+Zengauthor=P.+J.+Brownauthor=X.-j.+Yangauthor=C.+H.+Arrowsmithauthor=Y.+Zhaoauthor=Y.+G.+Zheng&title=Revealing+the+Protein+Propionylation+Activity+of+the+Histone+Acetyltransferase+Mof+%28Males+Absent+on+the+First%29&doi=10.1074%2Fjbc.ra117.000529"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.ra117.000529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.ra117.000529%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DBartlett%26aufirst%3DM.%2BG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DYang%26aufirst%3DX.-j.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DRevealing%2520the%2520Protein%2520Propionylation%2520Activity%2520of%2520the%2520Histone%2520Acetyltransferase%2520Mof%2520%2528Males%2520Absent%2520on%2520the%2520First%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D3410%26epage%3D3420%26doi%3D10.1074%2Fjbc.ra117.000529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangras, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falck, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Lysine Propionylation and Butyrylation Are Novel Post-Translational Modifications in Histones</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">812</span>– <span class="NLM_lpage">819</span>, <span class="refDoi"> DOI: 10.1074/mcp.m700021-mcp200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1074%2Fmcp.m700021-mcp200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=17267393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsV2ktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=812-819&author=Y.+Chenauthor=R.+Sprungauthor=Y.+Tangauthor=H.+Ballauthor=B.+Sangrasauthor=S.+C.+Kimauthor=J.+R.+Falckauthor=J.+Pengauthor=W.+Guauthor=Y.+Zhao&title=Lysine+Propionylation+and+Butyrylation+Are+Novel+Post-Translational+Modifications+in+Histones&doi=10.1074%2Fmcp.m700021-mcp200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine propionylation and butyrylation are novel post-translational modifications in histones</span></div><div class="casAuthors">Chen, Yue; Sprung, Robert; Tang, Yi; Ball, Haydn; Sangras, Bhavani; Kim, Sung Chan; Falck, John R.; Peng, Junmin; Gu, Wei; Zhao, Yingming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">812-819</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9476</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The pos. charged lysine residue plays an important role in protein folding and functions.  Neutralization of the charge often has a profound impact on the substrate proteins.  Accordingly all the known post-translational modifications at lysine have pivotal roles in cell physiol. and pathol.  Here we report the discovery of two novel, in vivo lysine modifications in histones, lysine propionylation and butyrylation.  We confirmed, by in vitro labeling and peptide mapping by mass spectrometry, that two previously known acetyltransferases, p300 and CREB-binding protein, could catalyze lysine propionylation and lysine butyrylation in histones.  Finally p300 and CREB-binding protein could carry out autopropionylation and autobutyrylation in vitro.  Taken together, our results conclusively establish that lysine propionylation and lysine butyrylation are novel post-translational modifications.  Given the unique roles of propionyl-CoA and butyryl-CoA in energy metab. and the significant structural changes induced by the modifications, the two modifications are likely to have important but distinct functions in the regulation of biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFDkCYOTsOjbVg90H21EOLACvtfcHk0ljgVNQhrkSqlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsV2ktLs%253D&md5=dd151e4055229bdbd0c26670454d03b2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fmcp.m700021-mcp200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.m700021-mcp200%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSprung%26aufirst%3DR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DBall%26aufirst%3DH.%26aulast%3DSangras%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DS.%2BC.%26aulast%3DFalck%26aufirst%3DJ.%2BR.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DGu%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DLysine%2520Propionylation%2520and%2520Butyrylation%2520Are%2520Novel%2520Post-Translational%2520Modifications%2520in%2520Histones%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2007%26volume%3D6%26spage%3D812%26epage%3D819%26doi%3D10.1074%2Fmcp.m700021-mcp200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">An Activity Gel Assay Detects a Single, Catalytically Active Histone Acetyltransferase Subunit in Tetrahymena Macronuclei</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">6364</span>– <span class="NLM_lpage">6368</span>, <span class="refDoi"> DOI: 10.1073/pnas.92.14.6364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1073%2Fpnas.92.14.6364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=7603997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK2MXmslKqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=6364-6368&author=J.+E.+Brownellauthor=C.+D.+Allis&title=An+Activity+Gel+Assay+Detects+a+Single%2C+Catalytically+Active+Histone+Acetyltransferase+Subunit+in+Tetrahymena+Macronuclei&doi=10.1073%2Fpnas.92.14.6364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">An activity gel assay detects a single, catalytically active histone acetyltransferase subunit in Tetrahymena macronuclei</span></div><div class="casAuthors">Brownell, James E.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6364-8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Macronuclei of the ciliated protozoan Tetrahymena thermophila possess a histone acetyltransferase activity closely assocd. with transcription-related histone acetylation.  Nothing definitive is known concerning the polypeptide compn. of this activity in Tetrahymena or any comparable activity from any cellular source.  An acetyltransferase activity gel assay was developed which identifies a catalytically active subunit of this enzyme in Tetrahymena.  This activity gel assay detects a single polypeptide of 55 kDa (p55) in crude macronuclear exts., as well as in column-purified fractions, which incorporates [3H]acetate from [3H]acetyl-CoA into core histone substrates polymd. directly into SDS polyacrylamide gels.  P55 copurifies precisely with acetyltransferase activity through all chromatog. steps examd., including reverse-phase HPLC.  Gel-filtration chromatog. of this activity indicates a mol. mass of 220 kDa, suggesting that the native enzyme may consist of four identical subunits of 55 kDa.  Furthermore, p55 is tightly assocd. with di- and greater polynucleosomes and therefore may be defined as a component of histone acetyltransferase type A - i.e., chromatin assocd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvronhG60mcLVg90H21EOLACvtfcHk0ljgVNQhrkSqlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmslKqtb8%253D&md5=f4e5e9f280f14e847fca2471e1f7633d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.14.6364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.14.6364%26sid%3Dliteratum%253Aachs%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DAn%2520Activity%2520Gel%2520Assay%2520Detects%2520a%2520Single%252C%2520Catalytically%2520Active%2520Histone%2520Acetyltransferase%2520Subunit%2520in%2520Tetrahymena%2520Macronuclei%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1995%26volume%3D92%26spage%3D6364%26epage%3D6368%26doi%3D10.1073%2Fpnas.92.14.6364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogryzko, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikawa, J.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span> <span> </span><span class="NLM_article-title">A P300/Cbp-Associated Factor That Competes with the Adenoviral Oncoprotein E1a</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>382</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1038/382319a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2F382319a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=8684459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK28Xks1Wgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=1996&pages=319-324&author=X.-J.+Yangauthor=V.+V.+Ogryzkoauthor=J.-i.+Nishikawaauthor=B.+H.+Howardauthor=Y.+Nakatani&title=A+P300%2FCbp-Associated+Factor+That+Competes+with+the+Adenoviral+Oncoprotein+E1a&doi=10.1038%2F382319a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A p300/CBP-associated factor that competes with the adenoviral oncoprotein E1A</span></div><div class="casAuthors">Yang, Xiang-Jiao; Ogryzko, Vasily V.; Nishikawa, Jun-ichi; Howard, Bruce H.; Nakatani, Yoshihiro</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">6589</span>),
    <span class="NLM_cas:pages">319-324</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The adenoviral oncoprotein E1A induces progression through the cell cycle by binding to the products of the p300/CBP and retinoblastoma gene families.  A new cellular p300/CBP-assocd. factor (P/CAF) having intrinsic histone acetylase activity has been identified that competes with E1A.  Exogenous expression of P/CAF in HeLa cells inhibits cell-cycle progression and counteracts the mitogenic activity of E1A.  E1A disturbs the normal cellular interaction between p300/CBP and its assocd. histone acetylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAEhSgaMP-wLVg90H21EOLACvtfcHk0ljc8humcj60NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xks1Wgurc%253D&md5=b18c2c0f8ff02df9ff7586dfe670bfc0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2F382319a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F382319a0%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.-J.%26aulast%3DOgryzko%26aufirst%3DV.%2BV.%26aulast%3DNishikawa%26aufirst%3DJ.-i.%26aulast%3DHoward%26aufirst%3DB.%2BH.%26aulast%3DNakatani%26aufirst%3DY.%26atitle%3DA%2520P300%252FCbp-Associated%2520Factor%2520That%2520Competes%2520with%2520the%2520Adenoviral%2520Oncoprotein%2520E1a%26jtitle%3DNature%26date%3D1996%26volume%3D382%26spage%3D319%26epage%3D324%26doi%3D10.1038%2F382319a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parthun, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottschling, D. E.</span></span> <span> </span><span class="NLM_article-title">The Major Cytoplasmic Histone Acetyltransferase in Yeast: Links to Chromatin Replication and Histone Metabolism</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(00)81325-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fs0092-8674%2800%2981325-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=8858151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK28XmtFeitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=1996&pages=85-94&author=M.+R.+Parthunauthor=J.+Widomauthor=D.+E.+Gottschling&title=The+Major+Cytoplasmic+Histone+Acetyltransferase+in+Yeast%3A+Links+to+Chromatin+Replication+and+Histone+Metabolism&doi=10.1016%2Fs0092-8674%2800%2981325-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The major cytoplasmic histone acetyltransferase in yeast: links to chromatin replication and histone metabolism</span></div><div class="casAuthors">Parthun, Mark R.; Widom, Jonathan; Gottschling, Daniel E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-94</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We have isolated the predominant cytoplasmic histone acetyltransferase activity from Saccharomyces cerevisiae.  This enzyme acetylates the lysine at residue 12 of free histone H4 but does not modify histone H4 when packaged in chromatin.  The activity contains two proteins, Hat1p and Hat2p.  Hat1p is the catalytic subunit of the histone acetyltransferase and has an intrinsic substrate specificity that modifies lysine in the recognition sequence GXGKXG.  The specificity of the enzyme in the yeast cytoplasm is restricted relative to recombinant Hat1p suggesting that it is neg. regulated in vivo.  Hat2p, which is required for high affinity binding to the acetyltransferase to histone H4, is highly related to Rbap48, which is a subunit of the chromatin assembly factor, CAF-1, and copurifies with the human histone deacetylase HD1.  We propose that the Hat2p/HRbap48 family serve as escorts of histone metab. enzymes to facilitate their interaction with histone H4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohFSyni3vvwbVg90H21EOLACvtfcHk0ljc8humcj60NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtFeitbc%253D&md5=d3e0b69fbe893306aebe308d56df8804</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fs0092-8674%2800%2981325-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0092-8674%252800%252981325-2%26sid%3Dliteratum%253Aachs%26aulast%3DParthun%26aufirst%3DM.%2BR.%26aulast%3DWidom%26aufirst%3DJ.%26aulast%3DGottschling%26aufirst%3DD.%2BE.%26atitle%3DThe%2520Major%2520Cytoplasmic%2520Histone%2520Acetyltransferase%2520in%2520Yeast%253A%2520Links%2520to%2520Chromatin%2520Replication%2520and%2520Histone%2520Metabolism%26jtitle%3DCell%26date%3D1996%26volume%3D87%26spage%3D85%26epage%3D94%26doi%3D10.1016%2Fs0092-8674%2800%2981325-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedmann, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachury, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structure of the -tubulin acetyltransferase, TAT1, and implications for tubulin-specific acetylation</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">19655</span>– <span class="NLM_lpage">19660</span>, <span class="refDoi"> DOI: 10.1073/pnas.1209357109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1073%2Fpnas.1209357109" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=19655-19660&author=D.+R.+Friedmannauthor=A.+Aguilarauthor=J.+Fanauthor=M.+V.+Nachuryauthor=R.+Marmorstein&title=Structure+of+the+-tubulin+acetyltransferase%2C+TAT1%2C+and+implications+for+tubulin-specific+acetylation&doi=10.1073%2Fpnas.1209357109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1209357109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1209357109%26sid%3Dliteratum%253Aachs%26aulast%3DFriedmann%26aufirst%3DD.%2BR.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DNachury%26aufirst%3DM.%2BV.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructure%2520of%2520the%2520-tubulin%2520acetyltransferase%252C%2520TAT1%252C%2520and%2520implications%2520for%2520tubulin-specific%2520acetylation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D19655%26epage%3D19660%26doi%3D10.1073%2Fpnas.1209357109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlow, E.</span></span> <span> </span><span class="NLM_article-title">Cellular Targets for Transformation by the Adenovirus E1a Proteins</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(89)90984-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2F0092-8674%2889%2990984-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=2521301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaL1MXhsVyrt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1989&pages=67-75&author=P.+Whyteauthor=N.+M.+Williamsonauthor=E.+Harlow&title=Cellular+Targets+for+Transformation+by+the+Adenovirus+E1a+Proteins&doi=10.1016%2F0092-8674%2889%2990984-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular targets for transformation by the adenovirus E1A proteins</span></div><div class="casAuthors">Whyte, Peter; Williamson, Nicola M.; Harlow, Ed</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-75</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Three cellular proteins, including species of 300,000 daltons and 107,000 daltons as well as p105-RB, the product of the retinoblastoma susceptibility gene, stably interact with the adenovirus E1A proteins.  To help det. the functional basis of these interactions, the E1A regions that participate in these interactions were mapped using a series of deletion mutants.  The 300,000-dalton and the 107,000-dalton proteins interacted with sequences within amino acids 1 to 76 and 121 to 127, resp.  Interaction with the third cellular protein, p105-RB, required the presence of sequences from 2 noncontiguous regions of the E1A polypeptide chain, amino acids 30 to 60 and 121 to 127.  The regions of E1A that are required for these interactions coincided precisely with the regions of E1A that are required for its transforming function.  These results suggest that the interactions with these cellular proteins are fundamental to the transforming activity of E1A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxvLkwEOAXp7Vg90H21EOLACvtfcHk0ljc8humcj60NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhsVyrt70%253D&md5=9980cd29a2ab10a08491a9a1d59ba694</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2889%2990984-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252889%252990984-7%26sid%3Dliteratum%253Aachs%26aulast%3DWhyte%26aufirst%3DP.%26aulast%3DWilliamson%26aufirst%3DN.%2BM.%26aulast%3DHarlow%26aufirst%3DE.%26atitle%3DCellular%2520Targets%2520for%2520Transformation%2520by%2520the%2520Adenovirus%2520E1a%2520Proteins%26jtitle%3DCell%26date%3D1989%26volume%3D56%26spage%3D67%26epage%3D75%26doi%3D10.1016%2F0092-8674%2889%2990984-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chrivia, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, R. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montminy, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, R. H.</span></span> <span> </span><span class="NLM_article-title">Phosphorylated Creb Binds Specifically to the Nuclear Protein Cbp</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>365</i></span>,  <span class="NLM_fpage">855</span>– <span class="NLM_lpage">859</span>, <span class="refDoi"> DOI: 10.1038/365855a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2F365855a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=8413673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK2cXisVyiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=1993&pages=855-859&author=J.+C.+Chriviaauthor=R.+P.+S.+Kwokauthor=N.+Lambauthor=M.+Hagiwaraauthor=M.+R.+Montminyauthor=R.+H.+Goodman&title=Phosphorylated+Creb+Binds+Specifically+to+the+Nuclear+Protein+Cbp&doi=10.1038%2F365855a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylated CREB binds specifically to the nuclear proterin CBP</span></div><div class="casAuthors">Chrivia, John C.; Kwok, Roland P. S.; Lamb, Ned; Hagiwara, Masatoshi; Montminy, Marc R.; Goodman, Richard H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">6449</span>),
    <span class="NLM_cas:pages">855-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Regulation of gene expression by cAMP frequently involves a DNA element known as the cAMP-regulated enhancer (CRE).  Many transcription factors bind to this element, including the protein CREB, which is activated as a result of phosphorylation by protein kinase A.  This modification stimulates interaction with one or more of the general transcription factors or, alternatively, allows recruitment of a co-activator.  Here the authors report that CREB phosphorylated by protein kinase A binds specifically to a nuclear protein of Mr 265K termed CBP (for CREB-binding protein).  Fusion of a heterologous DNA-binding domain to the amino terminus of CBP enables the chimeric protein to function as a protein kinase A-regulated transcriptional activator.  It is proposed that CBP may participate in cAMP-regulated gene expression by interacting with the activated phosphorylated form of CREB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG6PKqMwiJE7Vg90H21EOLACvtfcHk0lj7ECXzAbggGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXisVyiuw%253D%253D&md5=24f264edb41ca11d06d0e85f95d5308e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2F365855a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F365855a0%26sid%3Dliteratum%253Aachs%26aulast%3DChrivia%26aufirst%3DJ.%2BC.%26aulast%3DKwok%26aufirst%3DR.%2BP.%2BS.%26aulast%3DLamb%26aufirst%3DN.%26aulast%3DHagiwara%26aufirst%3DM.%26aulast%3DMontminy%26aufirst%3DM.%2BR.%26aulast%3DGoodman%26aufirst%3DR.%2BH.%26atitle%3DPhosphorylated%2520Creb%2520Binds%2520Specifically%2520to%2520the%2520Nuclear%2520Protein%2520Cbp%26jtitle%3DNature%26date%3D1993%26volume%3D365%26spage%3D855%26epage%3D859%26doi%3D10.1038%2F365855a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannuti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurnagel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucchesi, J. C.</span></span> <span> </span><span class="NLM_article-title">The Drosophila Msl Complex Acetylates Histone H4 at Lysine 16, a Chromatin Modification Linked to Dosage Compensation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">312</span>– <span class="NLM_lpage">318</span>, <span class="refDoi"> DOI: 10.1128/mcb.20.1.312-318.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1128%2Fmcb.20.1.312-318.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10594033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt1Wr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=312-318&author=E.+R.+Smithauthor=A.+Pannutiauthor=W.+Guauthor=A.+Steurnagelauthor=R.+G.+Cookauthor=C.+D.+Allisauthor=J.+C.+Lucchesi&title=The+Drosophila+Msl+Complex+Acetylates+Histone+H4+at+Lysine+16%2C+a+Chromatin+Modification+Linked+to+Dosage+Compensation&doi=10.1128%2Fmcb.20.1.312-318.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Drosophila MSL complex acetylates histone H4 at lysine 16, a chromatin modification linked to dosage compensation</span></div><div class="casAuthors">Smith, Edwin R.; Pannuti, Antonio; Gu, Weigang; Steurnagel, Arnd; Cook, Richard G.; Allis, C. David; Lucchesi, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">312-318</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In Drosophila, dosage compensation - the equalization of most X-linked gene products in males and females - is achieved by a 2-fold enhancement of the level of transcription of the X chromosome in males relative to each X chromosome in females.  A complex consisting of ≥5 gene products preferentially binds the X chromosome at numerous sites in males and results in a significant increase in the presence of a specific histone isoform, histone 4 acetylated at lysine 16.  Recently, RNA transcripts (roX1 and roX2) encoded by 2 different genes have also been found assocd. with the X chromosome in males.  A complex contg. MSL1, -2, and -3, MOF, MLE, and roX2 RNA was partially purified and demonstrated to exclusively acetylate H4 at lysine 16 on nucleosomal substrates.  These results demonstrate that the MSL complex is responsible for the specific chromatin modification characteristic of the X chromosome in Drosophila males.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBlzN1norgSrVg90H21EOLACvtfcHk0lj7ECXzAbggGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlt1Wr&md5=90c1a8e9526fd6691d49b455cc62f012</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2Fmcb.20.1.312-318.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.20.1.312-318.2000%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DE.%2BR.%26aulast%3DPannuti%26aufirst%3DA.%26aulast%3DGu%26aufirst%3DW.%26aulast%3DSteurnagel%26aufirst%3DA.%26aulast%3DCook%26aufirst%3DR.%2BG.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DLucchesi%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520Drosophila%2520Msl%2520Complex%2520Acetylates%2520Histone%2520H4%2520at%2520Lysine%252016%252C%2520a%2520Chromatin%2520Modification%2520Linked%2520to%2520Dosage%2520Compensation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2000%26volume%3D20%26spage%3D312%26epage%3D318%26doi%3D10.1128%2Fmcb.20.1.312-318.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dent, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shilatifard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">New Nomenclature for Chromatin-Modifying Enzymes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.10.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.cell.2007.10.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=18022353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVSgur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2007&pages=633-636&author=C.+D.+Allisauthor=S.+L.+Bergerauthor=J.+Coteauthor=S.+Dentauthor=T.+Jenuwienauthor=T.+Kouzaridesauthor=L.+Pillusauthor=D.+Reinbergauthor=Y.+Shiauthor=R.+Shiekhattarauthor=A.+Shilatifardauthor=J.+Workmanauthor=Y.+Zhang&title=New+Nomenclature+for+Chromatin-Modifying+Enzymes&doi=10.1016%2Fj.cell.2007.10.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">New nomenclature for chromatin-modifying enzymes</span></div><div class="casAuthors">Allis, C. David; Berger, Shelley L.; Cote, Jacques; Dent, Sharon; Jenuwien, Thomas; Kouzarides, Tony; Pillus, Lorraine; Reinberg, Danny; Shi, Yang; Shiekhattar, Ramin; Shilatifard, Ali; Workman, Jerry; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">633-636</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A new nomenclature is proposed for the characterized members of the families of lysine demethylases, acetyltransferases, and lysine methyltransferases.  Many of these enzymes were initially found to target histones as substrates but were subsequently shown to also target non-histone proteins.  Therefore, these enzymes have been given a more generic name that reflects: (1) the type of enzymic activity that they perform and (2) the type of residue that they modify.  Thus lysine demethylases will become K-demethylases (KDMs), acetyltransferases will become K-acetyltransferases (KATs), and lysine methyltransferases will become K-methyltransferases (KMTs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyTFCLpvkjcbVg90H21EOLACvtfcHk0lj7ECXzAbggGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVSgur%252FI&md5=6e12983346b153ab0999e5fdc3928ba2</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.10.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.10.039%26sid%3Dliteratum%253Aachs%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26aulast%3DCote%26aufirst%3DJ.%26aulast%3DDent%26aufirst%3DS.%26aulast%3DJenuwien%26aufirst%3DT.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPillus%26aufirst%3DL.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DShiekhattar%26aufirst%3DR.%26aulast%3DShilatifard%26aufirst%3DA.%26aulast%3DWorkman%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DNew%2520Nomenclature%2520for%2520Chromatin-Modifying%2520Enzymes%26jtitle%3DCell%26date%3D2007%26volume%3D131%26spage%3D633%26epage%3D636%26doi%3D10.1016%2Fj.cell.2007.10.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sapountzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, C. N.</span></span> <span> </span><span class="NLM_article-title">Cellular Functions of Tip60</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1496</span>– <span class="NLM_lpage">1509</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2006.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.biocel.2006.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=16698308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvVGjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2006&pages=1496-1509&author=V.+Sapountziauthor=I.+R.+Loganauthor=C.+N.+Robson&title=Cellular+Functions+of+Tip60&doi=10.1016%2Fj.biocel.2006.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular functions of TIP60</span></div><div class="casAuthors">Sapountzi, Vasileia; Logan, Ian R.; Robson, Craig N.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1496-1509</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  TIP60 was originally identified as a cellular acetyltransferase protein that interacts with HIV-1 Tat.  As a consequence, the role of TIP60 in transcriptional regulation has been investigated intensively.  Recent data suggest that TIP60 has more divergent functions than originally thought and roles for TIP60 in many processes, such as cellular signaling, DNA damage repair, cell cycle and checkpoint control and apoptosis are emerging.  TIP60 is a tightly regulated transcriptional coregulator, acting in a large multiprotein complex for a range of transcription factors including androgen receptor, Myc, STAT3, NF-κB, E2F1 and p53.  This usually involves recruitment of TIP60 acetyltransferase activities to chromatin.  Addnl., in response to DNA double strand breaks, TIP60 is recruited to DNA lesions where it participates both in the initial as well as the final stages of repair.  Here, the authors describe how TIP60 is a multifunctional enzyme involved in multiple nuclear transactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7jUBUBslgrbVg90H21EOLACvtfcHk0ljpXoSu5-QsDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvVGjtrw%253D&md5=c170361cb50984a86da9835ef8e7feb5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2006.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2006.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DSapountzi%26aufirst%3DV.%26aulast%3DLogan%26aufirst%3DI.%2BR.%26aulast%3DRobson%26aufirst%3DC.%2BN.%26atitle%3DCellular%2520Functions%2520of%2520Tip60%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2006%26volume%3D38%26spage%3D1496%26epage%3D1509%26doi%3D10.1016%2Fj.biocel.2006.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iizuka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stillman, B.</span></span> <span> </span><span class="NLM_article-title">Histone Acetyltransferase Hbo1 Interacts with the Orc1 Subunit of the Human Initiator Protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">23027</span>– <span class="NLM_lpage">23034</span>, <span class="refDoi"> DOI: 10.1074/jbc.274.33.23027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1074%2Fjbc.274.33.23027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=23027-23034&author=M.+Iizukaauthor=B.+Stillman&title=Histone+Acetyltransferase+Hbo1+Interacts+with+the+Orc1+Subunit+of+the+Human+Initiator+Protein&doi=10.1074%2Fjbc.274.33.23027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.33.23027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.33.23027%26sid%3Dliteratum%253Aachs%26aulast%3DIizuka%26aufirst%3DM.%26aulast%3DStillman%26aufirst%3DB.%26atitle%3DHistone%2520Acetyltransferase%2520Hbo1%2520Interacts%2520with%2520the%2520Orc1%2520Subunit%2520of%2520the%2520Human%2520Initiator%2520Protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D23027%26epage%3D23034%26doi%3D10.1074%2Fjbc.274.33.23027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karmodiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anamika, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muley, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradhan, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhide, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galande, S.</span></span> <span> </span><span class="NLM_article-title">Camello, a Novel Family of Histone Acetyltransferases That Acetylate Histone H4 and Is Essential for Zebrafish Development</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">6076</span>, <span class="refDoi"> DOI: 10.1038/srep06076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fsrep06076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=25123547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVWjsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=6076&author=K.+Karmodiyaauthor=K.+Anamikaauthor=V.+Muleyauthor=S.+J.+Pradhanauthor=Y.+Bhideauthor=S.+Galande&title=Camello%2C+a+Novel+Family+of+Histone+Acetyltransferases+That+Acetylate+Histone+H4+and+Is+Essential+for+Zebrafish+Development&doi=10.1038%2Fsrep06076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Camello, a novel family of Histone Acetyltransferases that acetylate histone H4 and is essential for zebrafish development</span></div><div class="casAuthors">Karmodiya, Krishanpal; Anamika, Krishanpal; Muley, Vijaykumar; Pradhan, Saurabh J.; Bhide, Yoshita; Galande, Sanjeev</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6076</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In this study, we have investigated genome-wide occurrence of Histone Acetyltransferases (HATs) in genomes of Mus musculus and Danio rerio on the basis of presence of HAT domain.  Our study identified a group of proteins that lacks characteristic features of known HAT families, relatively smaller in size and has no other assocd. domains.  Most of the proteins in this unclassified group are Camello proteins, which are not yet known and classified as functional HATs.  Our in vitro and in vivo anal. revealed that Camello family proteins are active HATs and exhibit specificity towards histone H4.  Interestingly, Camello proteins are among the first identified HATs showing perinuclear localization.  Moreover, Camello proteins are evolutionarily conserved in all chordates and are obsd. for the first time in cnidarians in phylogeny.  Furthermore, knockdown of Camello protein (CMLO3) in zebrafish embryos exhibited defects in axis elongation and head formation.  Thus, our study identified a novel family of active HATs that is specific for histone H4 acetylation, exhibits perinuclear localization and is essential for zebrafish development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo925EY-3OwiLVg90H21EOLACvtfcHk0ljpXoSu5-QsDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVWjsb0%253D&md5=121a536d0c2f61814b9f3c4facdf79bb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsrep06076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep06076%26sid%3Dliteratum%253Aachs%26aulast%3DKarmodiya%26aufirst%3DK.%26aulast%3DAnamika%26aufirst%3DK.%26aulast%3DMuley%26aufirst%3DV.%26aulast%3DPradhan%26aufirst%3DS.%2BJ.%26aulast%3DBhide%26aufirst%3DY.%26aulast%3DGalande%26aufirst%3DS.%26atitle%3DCamello%252C%2520a%2520Novel%2520Family%2520of%2520Histone%2520Acetyltransferases%2520That%2520Acetylate%2520Histone%2520H4%2520and%2520Is%2520Essential%2520for%2520Zebrafish%2520Development%26jtitle%3DSci.%2520Rep.%26date%3D2014%26volume%3D4%26spage%3D6076%26doi%3D10.1038%2Fsrep06076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirayama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassone-Corsi, P.</span></span> <span> </span><span class="NLM_article-title">Circadian Regulator Clock Is a Histone Acetyltransferase</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.03.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.cell.2006.03.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=16678094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD28XkslCjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=497-508&author=M.+Doiauthor=J.+Hirayamaauthor=P.+Sassone-Corsi&title=Circadian+Regulator+Clock+Is+a+Histone+Acetyltransferase&doi=10.1016%2Fj.cell.2006.03.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Circadian regulator CLOCK is a histone acetyltransferase</span></div><div class="casAuthors">Doi, Masao; Hirayama, Jun; Sassone-Corsi, Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">497-508</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mol. machinery that governs circadian rhythmicity comprises proteins whose interplay generates time-specific transcription of clock genes.  The role of chromatin remodeling in a physiol. setting such as the circadian clock is yet unclear.  Here, the authors show that transcription factor CLOCK, a central component of the circadian pacemaker, has histone acetyltransferase (HAT) activity.  CLOCK was found to share homol. with acetyl-CoA binding motifs within the MYST family of HATs.  CLOCK displayed high sequence similarity to ACTR, a member of SRC family of HATs, with which it shares also enzymic specificity for histones H3 and H4.  BMAL1, the heterodimerization partner of CLOCK, enhanced HAT function.  The HAT activity of CLOCK is essential to rescue circadian rhythmicity and activation of clock genes in Clock mutant cells.  Identification of CLOCK as a novel type of DNA binding HAT reveals that chromatin remodeling is crucial for the core clock mechanism and identifies unforeseen links between histone acetylation and cellular physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaemR1QPQRpLVg90H21EOLACvtfcHk0liGgIenRsv_Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkslCjtLg%253D&md5=048bfdb95b7a14ff46d03aec22c44a47</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.03.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.03.033%26sid%3Dliteratum%253Aachs%26aulast%3DDoi%26aufirst%3DM.%26aulast%3DHirayama%26aufirst%3DJ.%26aulast%3DSassone-Corsi%26aufirst%3DP.%26atitle%3DCircadian%2520Regulator%2520Clock%2520Is%2520a%2520Histone%2520Acetyltransferase%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D497%26epage%3D508%26doi%3D10.1016%2Fj.cell.2006.03.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiltz, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privalsky, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, R. M.</span></span> <span> </span><span class="NLM_article-title">Nuclear Receptor Coactivator Actr Is a Novel Histone Acetyltransferase and Forms a Multimeric Activation Complex with P/Caf and Cbp/P300</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">569</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(00)80516-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fs0092-8674%2800%2980516-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=9267036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADyaK2sXlsVKrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1997&pages=569-580&author=H.+Chenauthor=R.+J.+Linauthor=R.+L.+Schiltzauthor=D.+Chakravartiauthor=A.+Nashauthor=L.+Nagyauthor=M.+L.+Privalskyauthor=Y.+Nakataniauthor=R.+M.+Evans&title=Nuclear+Receptor+Coactivator+Actr+Is+a+Novel+Histone+Acetyltransferase+and+Forms+a+Multimeric+Activation+Complex+with+P%2FCaf+and+Cbp%2FP300&doi=10.1016%2Fs0092-8674%2800%2980516-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear receptor coactivator ACTR is a novel histone acetyltransferase and forms a multimeric activation complex with P/CAF and CBP/p300</span></div><div class="casAuthors">Chen, Hongwu; Lin, Richard J.; Schiltz, R. Louis; Chakravarti, Debabrata; Nash, Alyssa; Nagy, Laszlo; Privalsky, Martin L.; Nakatani, Yoshihiro; Evans, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-580</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The authors report the identification of a novel cofactor, ACTR, that directly binds nuclear receptors and stimulates their transcriptional activities in a hormone-dependent fashion.  ACTR also recruits two other nuclear factors, CBP and P/CAF, and thus plays a central role in creating a multisubunit coactivator complex.  In addn., and unexpectedly, the authors show that purified ACTR is a potent histone acetyltransferase and appears to define a distinct evolutionary branch to this recently described family.  Thus, hormonal activation by nuclear receptors involves the mutual recruitment of at least three classes of histone acetyltransferases that may act cooperatively as an enzymic unit to reverse the effects of histone deacetylase shown to be part of the nuclear receptor corepressor complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyMwDt8gKDc7Vg90H21EOLACvtfcHk0lgwjeWZdCEuMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlsVKrsLw%253D&md5=6ca020e64b62362e309b36309af9fc37</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fs0092-8674%2800%2980516-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0092-8674%252800%252980516-4%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DR.%2BJ.%26aulast%3DSchiltz%26aufirst%3DR.%2BL.%26aulast%3DChakravarti%26aufirst%3DD.%26aulast%3DNash%26aufirst%3DA.%26aulast%3DNagy%26aufirst%3DL.%26aulast%3DPrivalsky%26aufirst%3DM.%2BL.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26atitle%3DNuclear%2520Receptor%2520Coactivator%2520Actr%2520Is%2520a%2520Novel%2520Histone%2520Acetyltransferase%2520and%2520Forms%2520a%2520Multimeric%2520Activation%2520Complex%2520with%2520P%252FCaf%2520and%2520Cbp%252FP300%26jtitle%3DCell%26date%3D1997%26volume%3D90%26spage%3D569%26epage%3D580%26doi%3D10.1016%2Fs0092-8674%2800%2980516-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span> <span> </span><span class="NLM_article-title">Two Human Orthologues of Eco1/Ctf7 Acetyltransferases Are Both Required for Proper Sister-Chromatid Cohesion</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3908</span>– <span class="NLM_lpage">3918</span>, <span class="refDoi"> DOI: 10.1091/mbc.e04-12-1063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1091%2Fmbc.e04-12-1063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=15958495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD2MXosVCnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2005&pages=3908-3918&author=F.+Houauthor=H.+Zou&title=Two+Human+Orthologues+of+Eco1%2FCtf7+Acetyltransferases+Are+Both+Required+for+Proper+Sister-Chromatid+Cohesion&doi=10.1091%2Fmbc.e04-12-1063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Two human orthologues of Eco1/Ctf7 acetyltransferases are both required for proper sister-chromatid cohesion</span></div><div class="casAuthors">Hou, Fajian; Zou, Hui</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3908-3918</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1059-1524</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Genetic studies in yeast and Drosophila have uncovered a conserved acetyltransferase involved in sister-chromatid cohesion.  Here, we describe the two human orthologs, previously named EFO1/ESCO1 and EFO2/ESCO2.  Similar to their yeast (Eco1/Ctf7 and Eso1) and fly (deco) counterparts, both proteins feature a conserved C-terminal domain consisting of a H2C2 zinc finger motif and an acetyltransferase domain that is able to catalyze auto-acetylation reaction in vitro.  However, no similarity can be detected outside of the conserved domain.  RNA interference depletion expt. revealed that EFO1/ESCO1 and EFO2/ESCO2 were not redundant and that both were required for proper sister-chromatid cohesion.  The difference between EFO1 and EFO2 also is reflected in their cell cycle regulation.  In mitosis, EFO1 is phosphorylated, whereas EFO2 is degraded.  Furthermore, both proteins assoc. with chromosomes, and the chromosome binding depends on the diverse N-terminal domains.  We propose that EFO1 and EFO2 are targeted to different chromosome structures to help establish or maintain sister-chromatid cohesion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUy_cAA2M0vbVg90H21EOLACvtfcHk0lgwjeWZdCEuMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXosVCnsL0%253D&md5=10b34fe2c094f397efee70fd1c430d2f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e04-12-1063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e04-12-1063%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DF.%26aulast%3DZou%26aufirst%3DH.%26atitle%3DTwo%2520Human%2520Orthologues%2520of%2520Eco1%252FCtf7%2520Acetyltransferases%2520Are%2520Both%2520Required%2520for%2520Proper%2520Sister-Chromatid%2520Cohesion%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2005%26volume%3D16%26spage%3D3908%26epage%3D3918%26doi%3D10.1091%2Fmbc.e04-12-1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span> <span> </span><span class="NLM_article-title">Protein Lysine Acylation and Cysteine Succination by Intermediates of Energy Metabolism</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">947</span>– <span class="NLM_lpage">960</span>, <span class="refDoi"> DOI: 10.1021/cb3001793</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3001793" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC38XmslGqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=947-960&author=H.+Linauthor=X.+Suauthor=B.+He&title=Protein+Lysine+Acylation+and+Cysteine+Succination+by+Intermediates+of+Energy+Metabolism&doi=10.1021%2Fcb3001793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Protein lysine acylation and cysteine succination by intermediates of energy metabolism</span></div><div class="casAuthors">Lin, Hening; Su, Xiaoyang; He, Bin</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">947-960</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In the past few years, several new protein post-translational modifications that use intermediates in metab. have been discovered.  These include various acyl-Lys modifications (formylation, propionylation, butyrylation, crotonylation, malonylation, succinylation, myristoylation) and Cys succination.  Here, the authors review the discovery and the current understanding of these modifications.  Several of these modifications are regulated by the deacylases, sirtuins, which use NAD, an important metabolic small mol.  Interestingly, several of these modifications in turn regulate the activity of metabolic enzymes.  These new modifications reveal interesting connections between metab. and protein post-translational modifications and raise many questions for future investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzcRGttJMoRbVg90H21EOLACvtfcHk0lgwjeWZdCEuMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmslGqsb0%253D&md5=e96b0583359af83bbbbce5e7e36236f9</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fcb3001793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3001793%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DB.%26atitle%3DProtein%2520Lysine%2520Acylation%2520and%2520Cysteine%2520Succination%2520by%2520Intermediates%2520of%2520Energy%2520Metabolism%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D947%26epage%3D960%26doi%3D10.1021%2Fcb3001793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousiges, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvi, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutillier, A.-L.</span></span> <span> </span><span class="NLM_article-title">Acetyltransferases (Hats) as Targets for Neurological Therapeutics</span>. <i>Neurotherapeutics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">568</span>– <span class="NLM_lpage">588</span>, <span class="refDoi"> DOI: 10.1007/s13311-013-0204-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1007%2Fs13311-013-0204-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=24006237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ylurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=568-588&author=A.+Schneiderauthor=S.+Chatterjeeauthor=O.+Bousigesauthor=B.+R.+Selviauthor=A.+Swaminathanauthor=R.+Casselauthor=F.+Blancauthor=T.+K.+Kunduauthor=A.-L.+Boutillier&title=Acetyltransferases+%28Hats%29+as+Targets+for+Neurological+Therapeutics&doi=10.1007%2Fs13311-013-0204-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyltransferases (HATs) as Targets for Neurological Therapeutics</span></div><div class="casAuthors">Schneider, Anne; Chatterjee, Snehajyoti; Bousiges, Olivier; Selvi, B. Ruthrotha; Swaminathan, Amrutha; Cassel, Raphaelle; Blanc, Frederic; Kundu, Tapas K.; Boutillier, Anne-Laurence</div><div class="citationInfo"><span class="NLM_cas:title">Neurotherapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">568-588</span>CODEN:
                <span class="NLM_cas:coden">NEURNV</span>;
        ISSN:<span class="NLM_cas:issn">1878-7479</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The acetylation of histone and non-histone proteins controls a great deal of cellular functions, thereby affecting the entire organism, including the brain.  Acetylation modifications are mediated through histone acetyltransferases (HAT) and deacetylases (HDAC), and the balance of these enzymes regulates neuronal homeostasis, maintaining the pre-existing acetyl marks responsible for the global chromatin structure, as well as regulating specific dynamic acetyl marks that respond to changes and facilitate neurons to encode and strengthen long-term events in the brain circuitry (e.g., memory formation).  Unfortunately, the dysfunction of these finely-tuned regulations might lead to pathol. conditions, and the deregulation of the HAT/HDAC balance has been implicated in neurol. disorders.  During the last decade, research has focused on HDAC inhibitors that induce a histone hyperacetylated state to compensate acetylation deficits.  The use of these inhibitors as a therapeutic option was efficient in several animal models of neurol. disorders.  The elaboration of new cell-permeant HAT activators opens a new era of research on acetylation regulation.  Although pathol. animal models have not been tested yet, HAT activator mols. have already proven to be beneficial in ameliorating brain functions assocd. with learning and memory, and adult neurogenesis in wild-type animals.  Thus, HAT activator mols. contribute to an exciting area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPdlLQ5zuSkrVg90H21EOLACvtfcHk0ljOLpRYNM8okw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ylurnN&md5=de7c6c89fa41c1311f892178d12b057d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs13311-013-0204-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13311-013-0204-7%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DA.%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DBousiges%26aufirst%3DO.%26aulast%3DSelvi%26aufirst%3DB.%2BR.%26aulast%3DSwaminathan%26aufirst%3DA.%26aulast%3DCassel%26aufirst%3DR.%26aulast%3DBlanc%26aufirst%3DF.%26aulast%3DKundu%26aufirst%3DT.%2BK.%26aulast%3DBoutillier%26aufirst%3DA.-L.%26atitle%3DAcetyltransferases%2520%2528Hats%2529%2520as%2520Targets%2520for%2520Neurological%2520Therapeutics%26jtitle%3DNeurotherapeutics%26date%3D2013%26volume%3D10%26spage%3D568%26epage%3D588%26doi%3D10.1007%2Fs13311-013-0204-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cereseto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manganaro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terreni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fittipaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lusic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacca, M.</span></span> <span> </span><span class="NLM_article-title">Acetylation of Hiv-1 Integrase by P300 Regulates Viral Integration</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3070</span>– <span class="NLM_lpage">3081</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fsj.emboj.7600770" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=3070-3081&author=A.+Ceresetoauthor=L.+Manganaroauthor=M.+I.+Gutierrezauthor=M.+Terreniauthor=A.+Fittipaldiauthor=M.+Lusicauthor=A.+Marcelloauthor=M.+Giacca&title=Acetylation+of+Hiv-1+Integrase+by+P300+Regulates+Viral+Integration&doi=10.1038%2Fsj.emboj.7600770"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600770%26sid%3Dliteratum%253Aachs%26aulast%3DCereseto%26aufirst%3DA.%26aulast%3DManganaro%26aufirst%3DL.%26aulast%3DGutierrez%26aufirst%3DM.%2BI.%26aulast%3DTerreni%26aufirst%3DM.%26aulast%3DFittipaldi%26aufirst%3DA.%26aulast%3DLusic%26aufirst%3DM.%26aulast%3DMarcello%26aufirst%3DA.%26aulast%3DGiacca%26aufirst%3DM.%26atitle%3DAcetylation%2520of%2520Hiv-1%2520Integrase%2520by%2520P300%2520Regulates%2520Viral%2520Integration%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26spage%3D3070%26epage%3D3081%26doi%3D10.1038%2Fsj.emboj.7600770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carneiro, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Abreu da Silva, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, E. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caby, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancelot, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanderstraete, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furdas, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantappié, M. R.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Changes Modulate Schistosome Egg Formation and Are a Novel Target for Reducing Transmission of Schistosomiasis</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e1004116</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1004116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1371%2Fjournal.ppat.1004116" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1004116&author=V.+C.+Carneiroauthor=I.+C.+de+Abreu+da+Silvaauthor=E.+J.+L.+Torresauthor=S.+Cabyauthor=J.+Lancelotauthor=M.+Vanderstraeteauthor=S.+D.+Furdasauthor=M.+Jungauthor=R.+J.+Pierceauthor=M.+R.+Fantappi%C3%A9&title=Epigenetic+Changes+Modulate+Schistosome+Egg+Formation+and+Are+a+Novel+Target+for+Reducing+Transmission+of+Schistosomiasis&doi=10.1371%2Fjournal.ppat.1004116"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1004116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1004116%26sid%3Dliteratum%253Aachs%26aulast%3DCarneiro%26aufirst%3DV.%2BC.%26aulast%3Dde%2BAbreu%2Bda%2BSilva%26aufirst%3DI.%2BC.%26aulast%3DTorres%26aufirst%3DE.%2BJ.%2BL.%26aulast%3DCaby%26aufirst%3DS.%26aulast%3DLancelot%26aufirst%3DJ.%26aulast%3DVanderstraete%26aufirst%3DM.%26aulast%3DFurdas%26aufirst%3DS.%2BD.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DFantappi%25C3%25A9%26aufirst%3DM.%2BR.%26atitle%3DEpigenetic%2520Changes%2520Modulate%2520Schistosome%2520Egg%2520Formation%2520and%2520Are%2520a%2520Novel%2520Target%2520for%2520Reducing%2520Transmission%2520of%2520Schistosomiasis%26jtitle%3DPLoS%2520Pathog.%26date%3D2014%26volume%3D10%26spage%3De1004116%26doi%3D10.1371%2Fjournal.ppat.1004116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Bosch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of Histone Acetyltransferases and Deacetylases Are Potential Drugs for Inflammatory Diseases</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">654</span>– <span class="NLM_lpage">660</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.drudis.2013.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=24269836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVWjsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=654-660&author=F.+J.+Dekkerauthor=T.+van+den+Boschauthor=N.+I.+Martin&title=Small+Molecule+Inhibitors+of+Histone+Acetyltransferases+and+Deacetylases+Are+Potential+Drugs+for+Inflammatory+Diseases&doi=10.1016%2Fj.drudis.2013.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of histone acetyltransferases and deacetylases are potential drugs for inflammatory diseases</span></div><div class="casAuthors">Dekker, Frank J.; van den Bosch, Thea; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">654-660</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a reversible post-translational modification (PTM) of cellular proteins and represents an important regulatory switch in signal transduction.  Lysine acetylation, in combination with other PTMs, directs the outcomes as well as the activation levels of important signal transduction pathways such as the nuclear factor (NF)-κB pathway.  Small mol. modulators of the 'writers' (HATs) and 'erasers' (HDACs) can regulate the NF-κB pathway in a specific manner.  This review focuses on the effects of frequently used HAT and HDAC inhibitors on the NF-κB signal transduction pathway and inflammatory responses, and their potential as novel therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmVBCrhkoMsLVg90H21EOLACvtfcHk0ljOLpRYNM8okw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVWjsrrE&md5=d7d5eb5f350daa449cefd865433d4263</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DDekker%26aufirst%3DF.%2BJ.%26aulast%3Dvan%2Bden%2BBosch%26aufirst%3DT.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520Histone%2520Acetyltransferases%2520and%2520Deacetylases%2520Are%2520Potential%2520Drugs%2520for%2520Inflammatory%2520Diseases%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D654%26epage%3D660%26doi%3D10.1016%2Fj.drudis.2013.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okan, NA</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bales, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nascimento, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, PA</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medrano, EE</span></span> <span> </span><span class="NLM_article-title">Down-regulation of p300/CBP histone acetyltransferase activates a senescence checkpoint in human melanocytes</span>. <i>Cancer Res</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6231</span>– <span class="NLM_lpage">9</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=6231-9&author=D.+Bandyopadhyayauthor=NA+Okanauthor=E.+Balesauthor=L.+Nascimentoauthor=PA+Coleauthor=EE+Medrano&title=Down-regulation+of+p300%2FCBP+histone+acetyltransferase+activates+a+senescence+checkpoint+in+human+melanocytes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DOkan%26aufirst%3DNA%26aulast%3DBales%26aufirst%3DE.%26aulast%3DNascimento%26aufirst%3DL.%26aulast%3DCole%26aufirst%3DPA%26aulast%3DMedrano%26aufirst%3DEE%26atitle%3DDown-regulation%2520of%2520p300%252FCBP%2520histone%2520acetyltransferase%2520activates%2520a%2520senescence%2520checkpoint%2520in%2520human%2520melanocytes%26jtitle%3DCancer%2520Res%26date%3D2002%26volume%3D62%26spage%3D6231%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heery, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Targeting of Lysine Acetyltransferases in Human Disease: A Progress Report</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2006.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.drudis.2006.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=17198977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252Fgt1ynsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=88-99&author=D.+M.+Heeryauthor=P.+M.+Fischer&title=Pharmacological+Targeting+of+Lysine+Acetyltransferases+in+Human+Disease%3A+A+Progress+Report&doi=10.1016%2Fj.drudis.2006.11.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of lysine acetyltransferases in human disease: a progress report</span></div><div class="casAuthors">Heery David M; Fischer Peter M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">88-99</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Lysine acetyltransferases (LATs) are a structurally disparate group of enzymes involved in regulating transcription by participating as cofactors in transcriptional regulatory complexes, and by acetylation of lysine residues in histones and other proteins.  Aberrant LAT function probably plays an important part in the pathogenesis of certain cancers, especially leukaemias and endocrine tumours.  However, LAT activity might also be an important drug target in a range of other indications, including inflammatory lung diseases, viral infections and metabolic disorders.  At present, comparatively few LAT inhibitors are known, but progress regarding the understanding of their structural and functional biology is now beginning to reveal LATs as promising new epigenetic drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTI14vYK2Q1LGxpATR2EQURfW6udTcc2ebT_1s4VmDxr7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252Fgt1ynsQ%253D%253D&md5=38036849f8d8e653407401712d9e70ee</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DHeery%26aufirst%3DD.%2BM.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26atitle%3DPharmacological%2520Targeting%2520of%2520Lysine%2520Acetyltransferases%2520in%2520Human%2520Disease%253A%2520A%2520Progress%2520Report%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D88%26epage%3D99%26doi%3D10.1016%2Fj.drudis.2006.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kost, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soong, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakadai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischle, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimer, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">A Stable Transcription Factor Complex Nucleated by Oligomeric Aml1-Eto Controls Leukaemogenesis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>500</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1038/nature12287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnature12287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=23812588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKhtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=500&publication_year=2013&pages=93-97&author=X.-J.+Sunauthor=Z.+Wangauthor=L.+Wangauthor=Y.+Jiangauthor=N.+Kostauthor=T.+D.+Soongauthor=W.-Y.+Chenauthor=Z.+Tangauthor=T.+Nakadaiauthor=O.+Elementoauthor=W.+Fischleauthor=A.+Melnickauthor=D.+J.+Patelauthor=S.+D.+Nimerauthor=R.+G.+Roeder&title=A+Stable+Transcription+Factor+Complex+Nucleated+by+Oligomeric+Aml1-Eto+Controls+Leukaemogenesis&doi=10.1038%2Fnature12287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A stable transcription factor complex nucleated by oligomeric AML1-ETO controls leukaemogenesis</span></div><div class="casAuthors">Sun, Xiao-Jian; Wang, Zhanxin; Wang, Lan; Jiang, Yanwen; Kost, Nils; Soong, T. David; Chen, Wei-Yi; Tang, Zhanyun; Nakadai, Tomoyoshi; Elemento, Olivier; Fischle, Wolfgang; Melnick, Ari; Patel, Dinshaw J.; Nimer, Stephen D.; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">500</span>
        (<span class="NLM_cas:issue">7460</span>),
    <span class="NLM_cas:pages">93-97</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Transcription factors are frequently altered in leukemia through chromosomal translocation, mutation or aberrant expression.  AML1-ETO, a fusion protein generated by the t(8;21) translocation in acute myeloid leukemia, is a transcription factor implicated in both gene repression and activation.  AML1-ETO oligomerization, mediated by the NHR2 domain, is crit. for leukemogenesis, making it important to identify co-regulatory factors that 'read' the NHR2 oligomerization and contribute to leukemogenesis.  Here we show that, in human leukemic cells, AML1-ETO resides in and functions through a stable AML1-ETO-contg. transcription factor complex (AETFC) that contains several hematopoietic transcription (co)factors.  These AETFC components stabilize the complex through multivalent interactions, provide multiple DNA-binding domains for diverse target genes, co-localize genome wide, cooperatively regulate gene expression, and contribute to leukemogenesis.  Within the AETFC complex, AML1-ETO oligomerization is required for a specific interaction between the oligomerized NHR2 domain and a novel NHR2-binding (N2B) motif in E proteins.  Crystallog. anal. of the NHR2-N2B complex reveals a unique interaction pattern in which an N2B peptide makes direct contact with side chains of two NHR2 domains as a dimer, providing a novel model of how dimeric/oligomeric transcription factors create a new protein-binding interface through dimerization/oligomerization.  Intriguingly, disruption of this interaction by point mutations abrogates AML1-ETO-induced hematopoietic stem/progenitor cell self-renewal and leukemogenesis.  These results reveal new mechanisms of action of AML1-ETO, and provide a potential therapeutic target in t(8;21)-pos. acute myeloid leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv_tFeYiKM_bVg90H21EOLACvtfcHk0ljPTz9d3Qi1uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKhtbrI&md5=757dd656d6e371ec1503cca2a95103d9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature12287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12287%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.-J.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKost%26aufirst%3DN.%26aulast%3DSoong%26aufirst%3DT.%2BD.%26aulast%3DChen%26aufirst%3DW.-Y.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DNakadai%26aufirst%3DT.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DFischle%26aufirst%3DW.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DA%2520Stable%2520Transcription%2520Factor%2520Complex%2520Nucleated%2520by%2520Oligomeric%2520Aml1-Eto%2520Controls%2520Leukaemogenesis%26jtitle%3DNature%26date%3D2013%26volume%3D500%26spage%3D93%26epage%3D97%26doi%3D10.1038%2Fnature12287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, S.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Coactivator P300 Regulates Glucose-Induced Gene Expression in Endothelial Cells</span>. <i>Am. J. Physiol. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">E127</span>– <span class="NLM_lpage">E137</span>, <span class="refDoi"> DOI: 10.1152/ajpendo.00432.2009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1152%2Fajpendo.00432.2009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=19903865" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2010&pages=E127-E137&author=S.+Chenauthor=B.+Fengauthor=B.+Georgeauthor=R.+Chakrabartiauthor=M.+Chenauthor=S.+Chakrabarti&title=Transcriptional+Coactivator+P300+Regulates+Glucose-Induced+Gene+Expression+in+Endothelial+Cells&doi=10.1152%2Fajpendo.00432.2009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1152%2Fajpendo.00432.2009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpendo.00432.2009%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DGeorge%26aufirst%3DB.%26aulast%3DChakrabarti%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DChakrabarti%26aufirst%3DS.%26atitle%3DTranscriptional%2520Coactivator%2520P300%2520Regulates%2520Glucose-Induced%2520Gene%2520Expression%2520in%2520Endothelial%2520Cells%26jtitle%3DAm.%2520J.%2520Physiol.%2520Endocrinol.%2520Metab.%26date%3D2010%26volume%3D298%26spage%3DE127%26epage%3DE137%26doi%3D10.1152%2Fajpendo.00432.2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, J. L.</span></span> <span> </span><span class="NLM_article-title">Histone Acetyltransferase Complexes: One Size Doesn’t Fit All</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1038/nrm2145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnrm2145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=17380162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtlygsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=284-295&author=K.+K.+Leeauthor=J.+L.+Workman&title=Histone+Acetyltransferase+Complexes%3A+One+Size+Doesn%E2%80%99t+Fit+All&doi=10.1038%2Fnrm2145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Histone acetyltransferase complexes: one size doesn't fit all</span></div><div class="casAuthors">Lee, Kenneth K.; Workman, Jerry L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">284-295</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past 10 years, the study of histone acetyltransferases (HATs) has advanced significantly, and a no. of HATs have been isolated from various organisms.  It emerged that HATs are highly diverse and generally contain multiple subunits.  The functions of the catalytic subunit depend largely on the context of the other subunits in the complex.  We are just beginning to understand the specialized roles of HAT complexes in chromosome decondensation, DNA-damage repair and the modification of non-histone substrates, as well as their role in the broader epigenetic landscape, including the role of protein domains within HAT complexes and the dynamic interplay between HAT complexes and existing histone modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQQxOFwAIvkLVg90H21EOLACvtfcHk0lixEJo-5vbm7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtlygsrw%253D&md5=598e09c551d6d6de93c3bf10e348d0c9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrm2145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2145%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DK.%2BK.%26aulast%3DWorkman%26aufirst%3DJ.%2BL.%26atitle%3DHistone%2520Acetyltransferase%2520Complexes%253A%2520One%2520Size%2520Doesn%25E2%2580%2599t%2520Fit%2520All%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D284%26epage%3D295%26doi%3D10.1038%2Fnrm2145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lasko, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Digiammarino, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaves, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algire, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karukurichi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Drie, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElligott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesicki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, K. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Selective Catalytic P300/Cbp Inhibitor That Targets Lineage-Specific Tumours</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>550</i></span>,  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nature24028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnature24028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=28953875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=128-132&author=L.+M.+Laskoauthor=C.+G.+Jakobauthor=R.+P.+Edaljiauthor=W.+Qiuauthor=D.+Montgomeryauthor=E.+L.+Digiammarinoauthor=T.+M.+Hansenauthor=R.+M.+Risiauthor=R.+Freyauthor=V.+Manavesauthor=B.+Shawauthor=M.+Algireauthor=P.+Hesslerauthor=L.+T.+Lamauthor=T.+Uzielauthor=E.+Faivreauthor=D.+Fergusonauthor=F.+G.+Buchananauthor=R.+L.+Martinauthor=M.+Torrentauthor=G.+G.+Chiangauthor=K.+Karukurichiauthor=J.+W.+Langstonauthor=B.+T.+Weinertauthor=C.+Choudharyauthor=P.+de+Vriesauthor=J.+H.+Van+Drieauthor=D.+McElligottauthor=E.+Kesickiauthor=R.+Marmorsteinauthor=C.+Sunauthor=P.+A.+Coleauthor=S.+H.+Rosenbergauthor=M.+R.+Michaelidesauthor=A.+Laiauthor=K.+D.+Bromberg&title=Discovery+of+a+Selective+Catalytic+P300%2FCbp+Inhibitor+That+Targets+Lineage-Specific+Tumours&doi=10.1038%2Fnature24028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective catalytic p300/CBP inhibitor that targets lineage-specific tumours</span></div><div class="casAuthors">Lasko, Loren M.; Jakob, Clarissa G.; Edalji, Rohinton P.; Qiu, Wei; Montgomery, Debra; Digiammarino, Enrico L.; Hansen, T. Matt; Risi, Roberto M.; Frey, Robin; Manaves, Vlasios; Shaw, Bailin; Algire, Mikkel; Hessler, Paul; Lam, Lloyd T.; Uziel, Tamar; Faivre, Emily; Ferguson, Debra; Buchanan, Fritz G.; Martin, Ruth L.; Torrent, Maricel; Chiang, Gary G.; Karukurichi, Kannan; Langston, J. William; Weinert, Brian T.; Choudhary, Chunaram; de Vries, Peter; Van Drie, John H.; McElligott, David; Kesicki, Ed; Marmorstein, Ronen; Sun, Chaohong; Cole, Philip A.; Rosenberg, Saul H.; Michaelides, Michael R.; Lai, Albert; Bromberg, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">550</span>
        (<span class="NLM_cas:issue">7674</span>),
    <span class="NLM_cas:pages">128-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The dynamic and reversible acetylation of proteins, catalyzed by histone acetyltransferases (HATs) and histone deacetylases (HDACs), is a major epigenetic regulatory mechanism of gene transcription and is assocd. with multiple diseases.  Histone deacetylase inhibitors are currently approved to treat certain cancers, but progress on the development of drug-like histone actyltransferase inhibitors has lagged behind.  The histone acetyltransferase paralogues p300 and CREB-binding protein (CBP) are key transcriptional co-activators that are essential for a multitude of cellular processes, and have also been implicated in human pathol. conditions (including cancer).  Current inhibitors of the p300 and CBP histone acetyltransferase domains, including natural products, bi-substrate analogs and the widely used small mol. C646, lack potency or selectivity.  Here, we describe A-485, a potent, selective and drug-like catalytic inhibitor of p300 and CBP.  We present a high resoln. (1.95 Å) co-crystal structure of a small mol. bound to the catalytic active site of p300 and demonstrate that A-485 competes with acetyl CoA (acetyl-CoA).  A-485 selectively inhibited proliferation in lineage-specific tumor types, including several haematol. malignancies and androgen receptor-pos. prostate cancer.  A-485 inhibited the androgen receptor transcriptional program in both androgen-sensitive and castration-resistant prostate cancer and inhibited tumor growth in a castration-resistant xenograft model.  These results demonstrate the feasibility of using small mol. inhibitors to selectively target the catalytic activity of histone acetyltransferases, which may provide effective treatments for transcriptional activator-driven malignancies and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqfy_rMa47C7Vg90H21EOLACvtfcHk0lixEJo-5vbm7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsLzM&md5=050103c08a6abe2e0035573085fff6be</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnature24028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24028%26sid%3Dliteratum%253Aachs%26aulast%3DLasko%26aufirst%3DL.%2BM.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DMontgomery%26aufirst%3DD.%26aulast%3DDigiammarino%26aufirst%3DE.%2BL.%26aulast%3DHansen%26aufirst%3DT.%2BM.%26aulast%3DRisi%26aufirst%3DR.%2BM.%26aulast%3DFrey%26aufirst%3DR.%26aulast%3DManaves%26aufirst%3DV.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DAlgire%26aufirst%3DM.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DMartin%26aufirst%3DR.%2BL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DKarukurichi%26aufirst%3DK.%26aulast%3DLangston%26aufirst%3DJ.%2BW.%26aulast%3DWeinert%26aufirst%3DB.%2BT.%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3Dde%2BVries%26aufirst%3DP.%26aulast%3DVan%2BDrie%26aufirst%3DJ.%2BH.%26aulast%3DMcElligott%26aufirst%3DD.%26aulast%3DKesicki%26aufirst%3DE.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBromberg%26aufirst%3DK.%2BD.%26atitle%3DDiscovery%2520of%2520a%2520Selective%2520Catalytic%2520P300%252FCbp%2520Inhibitor%2520That%2520Targets%2520Lineage-Specific%2520Tumours%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D128%26epage%3D132%26doi%3D10.1038%2Fnature24028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leaver, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downer, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wichmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRae, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagiakos, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mieruszynski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanyai, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamasco, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sealey, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamudio, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnham, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheikh, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubrey, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doggett, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Silva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilling, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattarki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolezal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rautela, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newbold, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntington, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peat, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Street, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monahan, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, T.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Histone Acetyltransferases Kat6a/B Induce Senescence and Arrest Tumour Growth</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>560</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0387-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fs41586-018-0387-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=30069049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVensrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=560&publication_year=2018&pages=253-257&author=J.+B.+Baellauthor=D.+J.+Leaverauthor=S.+J.+Hermansauthor=G.+L.+Kellyauthor=M.+S.+Brennanauthor=N.+L.+Downerauthor=N.+Nguyenauthor=J.+Wichmannauthor=H.+M.+McRaeauthor=Y.+Yangauthor=B.+Clearyauthor=H.+R.+Lagiakosauthor=S.+Mieruszynskiauthor=G.+Paciniauthor=H.+K.+Vanyaiauthor=M.+I.+Bergamascoauthor=R.+E.+Mayauthor=B.+K.+Daveyauthor=K.+J.+Morganauthor=A.+J.+Sealeyauthor=B.+Wangauthor=N.+Zamudioauthor=S.+Wilcoxauthor=A.+L.+Garnhamauthor=B.+N.+Sheikhauthor=B.+J.+Aubreyauthor=K.+Doggettauthor=M.+C.+Chungauthor=M.+de+Silvaauthor=J.+Bentleyauthor=P.+Pillingauthor=M.+Hattarkiauthor=O.+Dolezalauthor=M.+L.+Dennisauthor=H.+Falkauthor=B.+Renauthor=S.+A.+Charmanauthor=K.+L.+Whiteauthor=J.+Rautelaauthor=A.+Newboldauthor=E.+D.+Hawkinsauthor=R.+W.+Johnstoneauthor=N.+D.+Huntingtonauthor=T.+S.+Peatauthor=J.+K.+Heathauthor=A.+Strasserauthor=M.+W.+Parkerauthor=G.+K.+Smythauthor=I.+P.+Streetauthor=B.+J.+Monahanauthor=A.+K.+Vossauthor=T.+Thomas&title=Inhibitors+of+Histone+Acetyltransferases+Kat6a%2FB+Induce+Senescence+and+Arrest+Tumour+Growth&doi=10.1038%2Fs41586-018-0387-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of histone acetyltransferases KAT6A/B induce senescence and arrest tumor growth</span></div><div class="casAuthors">Baell, Jonathan B.; Leaver, David J.; Hermans, Stefan J.; Kelly, Gemma L.; Brennan, Margs S.; Downer, Natalie L.; Nguyen, Nghi; Wichmann, Johannes; McRae, Helen M.; Yang, Yuqing; Cleary, Ben; Lagiakos, H. Rachel; Mieruszynski, Stephen; Pacini, Guido; Vanyai, Hannah K.; Bergamasco, Maria I.; May, Rose E.; Davey, Bethany K.; Morgan, Kimberly J.; Sealey, Andrew J.; Wang, Beinan; Zamudio, Natasha; Wilcox, Stephen; Garnham, Alexandra L.; Sheikh, Bilal N.; Aubrey, Brandon J.; Doggett, Karen; Chung, Matthew C.; de Silva, Melanie; Bentley, John; Pilling, Pat; Hattarki, Meghan; Dolezal, Olan; Dennis, Matthew L.; Falk, Hendrik; Ren, Bin; Charman, Susan A.; White, Karen L.; Rautela, Jai; Newbold, Andrea; Hawkins, Edwin D.; Johnstone, Ricky W.; Huntington, Nicholas D.; Peat, Thomas S.; Heath, Joan K.; Strasser, Andreas; Parker, Michael W.; Smyth, Gordon K.; Street, Ian P.; Monahan, Brendon J.; Voss, Anne K.; Thomas, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">560</span>
        (<span class="NLM_cas:issue">7717</span>),
    <span class="NLM_cas:pages">253-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Acetylation of histones by lysine acetyltransferases (KATs) is essential for chromatin organization and function1.  Among the genes coding for the MYST family of KATs (KAT5-KAT8) are the oncogenes KAT6A (also known as MOZ) and KAT6B (also known as MORF and QKF)2,3.  KAT6A has essential roles in normal hematopoietic stem cells4-6 and is the target of recurrent chromosomal translocations, causing acute myeloid leukemia7,8.  Similarly, chromosomal translocations in KAT6B have been identified in diverse cancers8.  KAT6A suppresses cellular senescence through the regulation of suppressors of the CDKN2A locus9,10, a function that requires its KAT activity10.  Loss of one allele of KAT6A extends the median survival of mice with MYC-induced lymphoma from 105 to 413 days11.  These findings suggest that inhibition of KAT6A and KAT6B may provide a therapeutic benefit in cancer.  Here we present highly potent, selective inhibitors of KAT6A and KAT6B, denoted WM-8014 and WM-1119.  Biochem. and structural studies demonstrate that these compds. are reversible competitors of acetyl CoA and inhibit MYST-catalyzed histone acetylation.  WM-8014 and WM-1119 induce cell cycle exit and cellular senescence without causing DNA damage.  Senescence is INK4A/ARF-dependent and is accompanied by changes in gene expression that are typical of loss of KAT6A function.  WM-8014 potentiates oncogene-induced senescence in vitro and in a zebrafish model of hepatocellular carcinoma.  WM-1119, which has increased bioavailability, arrests the progression of lymphoma in mice.  We anticipate that this class of inhibitors will help to accelerate the development of therapeutics that target gene transcription regulated by histone acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0njCpvnQW_7Vg90H21EOLACvtfcHk0lj8IGCbMVpODg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVensrjL&md5=d7ac24156dfa199e2257b58f85131b30</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0387-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0387-5%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DLeaver%26aufirst%3DD.%2BJ.%26aulast%3DHermans%26aufirst%3DS.%2BJ.%26aulast%3DKelly%26aufirst%3DG.%2BL.%26aulast%3DBrennan%26aufirst%3DM.%2BS.%26aulast%3DDowner%26aufirst%3DN.%2BL.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DWichmann%26aufirst%3DJ.%26aulast%3DMcRae%26aufirst%3DH.%2BM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DCleary%26aufirst%3DB.%26aulast%3DLagiakos%26aufirst%3DH.%2BR.%26aulast%3DMieruszynski%26aufirst%3DS.%26aulast%3DPacini%26aufirst%3DG.%26aulast%3DVanyai%26aufirst%3DH.%2BK.%26aulast%3DBergamasco%26aufirst%3DM.%2BI.%26aulast%3DMay%26aufirst%3DR.%2BE.%26aulast%3DDavey%26aufirst%3DB.%2BK.%26aulast%3DMorgan%26aufirst%3DK.%2BJ.%26aulast%3DSealey%26aufirst%3DA.%2BJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DZamudio%26aufirst%3DN.%26aulast%3DWilcox%26aufirst%3DS.%26aulast%3DGarnham%26aufirst%3DA.%2BL.%26aulast%3DSheikh%26aufirst%3DB.%2BN.%26aulast%3DAubrey%26aufirst%3DB.%2BJ.%26aulast%3DDoggett%26aufirst%3DK.%26aulast%3DChung%26aufirst%3DM.%2BC.%26aulast%3Dde%2BSilva%26aufirst%3DM.%26aulast%3DBentley%26aufirst%3DJ.%26aulast%3DPilling%26aufirst%3DP.%26aulast%3DHattarki%26aufirst%3DM.%26aulast%3DDolezal%26aufirst%3DO.%26aulast%3DDennis%26aufirst%3DM.%2BL.%26aulast%3DFalk%26aufirst%3DH.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DCharman%26aufirst%3DS.%2BA.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DRautela%26aufirst%3DJ.%26aulast%3DNewbold%26aufirst%3DA.%26aulast%3DHawkins%26aufirst%3DE.%2BD.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DHuntington%26aufirst%3DN.%2BD.%26aulast%3DPeat%26aufirst%3DT.%2BS.%26aulast%3DHeath%26aufirst%3DJ.%2BK.%26aulast%3DStrasser%26aufirst%3DA.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26aulast%3DStreet%26aufirst%3DI.%2BP.%26aulast%3DMonahan%26aufirst%3DB.%2BJ.%26aulast%3DVoss%26aufirst%3DA.%2BK.%26aulast%3DThomas%26aufirst%3DT.%26atitle%3DInhibitors%2520of%2520Histone%2520Acetyltransferases%2520Kat6a%252FB%2520Induce%2520Senescence%2520and%2520Arrest%2520Tumour%2520Growth%26jtitle%3DNature%26date%3D2018%26volume%3D560%26spage%3D253%26epage%3D257%26doi%3D10.1038%2Fs41586-018-0387-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhalabi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Katching-up on Small Molecule Modulators of Lysine Acetyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1249</span>– <span class="NLM_lpage">1270</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01502</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01502" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygt7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1249-1270&author=R.+P.+Simonauthor=D.+Robaaauthor=Z.+Alhalabiauthor=W.+Sipplauthor=M.+Jung&title=Katching-up+on+Small+Molecule+Modulators+of+Lysine+Acetyltransferases&doi=10.1021%2Facs.jmedchem.5b01502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">KATching-Up on Small Molecule Modulators of Lysine Acetyltransferases</span></div><div class="casAuthors">Simon, Roman P.; Robaa, Dina; Alhalabi, Zayan; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1249-1270</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The reversible acetylation of lysines is one of the best characterized epigenetic modifications.  Its involvement in many key physiol. and pathol. processes has been documented in numerous studies.  Lysine deacetylases (KDACs) and acetyltransferases (KATs) maintain the acetylation equil. at histones but also many other proteins.  Besides acetylation, also other acyl groups are reversibly installed at the side chain of lysines in proteins.  Because of their involvement in disease, KDACs and KATs were proposed to be promising drug targets, and for KDACs, indeed, five inhibitors are now approved for human use.  While there is a similar level of evidence for the potential of KATs as drug targets, no inhibitor is in clin. trials.  Here, we review the evidence for the diverse roles of KATs in disease pathol., provide an overview of structural features and the available modulators, including those targeting the bromodomains of KATs, and present an outlook.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFfZhEDBtCVLVg90H21EOLACvtfcHk0lj8IGCbMVpODg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVygt7rK&md5=afcf613db1cfa04320b93aed8b25cec1</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01502%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DR.%2BP.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DAlhalabi%26aufirst%3DZ.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DKatching-up%2520on%2520Small%2520Molecule%2520Modulators%2520of%2520Lysine%2520Acetyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1249%26epage%3D1270%26doi%3D10.1021%2Facs.jmedchem.5b01502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Organ, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuellar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrimp, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span> <span> </span><span class="NLM_article-title">Assay Interference and Off-Target Liabilities of Reported Histone Acetyltransferase Inhibitors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1527</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01657-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fs41467-017-01657-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=29142305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A280%3ADC%252BC1M3hs1Oisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1527&author=J.+L.+Dahlinauthor=K.+M.+Nelsonauthor=J.+M.+Strasserauthor=D.+Barsyte-Lovejoyauthor=M.+M.+Szewczykauthor=S.+Organauthor=M.+Cuellarauthor=G.+Singhauthor=J.+H.+Shrimpauthor=N.+Nguyenauthor=J.+L.+Meierauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=J.+B.+Baellauthor=M.+A.+Walters&title=Assay+Interference+and+Off-Target+Liabilities+of+Reported+Histone+Acetyltransferase+Inhibitors&doi=10.1038%2Fs41467-017-01657-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Assay interference and off-target liabilities of reported histone acetyltransferase inhibitors</span></div><div class="casAuthors">Dahlin Jayme L; Nelson Kathryn M; Strasser Jessica M; Cuellar Matthew; Singh Gurpreet; Walters Michael A; Barsyte-Lovejoy Dalia; Szewczyk Magdalena M; Organ Shawna; Arrowsmith Cheryl H; Brown Peter J; Shrimp Jonathan H; Meier Jordan L; Nguyen Nghi; Baell Jonathan B; Baell Jonathan B</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1527</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Many compounds with potentially reactive chemical motifs and poor physicochemical properties are published as selective modulators of biomolecules without sufficient validation and then propagated in the scientific literature as useful chemical probes.  Several histone acetyltransferase (HAT) inhibitors with these liabilities are now routinely used to probe epigenetic pathways.  We profile the most commonly used HAT inhibitors and confirm that the majority of them are nonselective interference compounds.  Most (15 out of 23, 65%) of the inhibitors are flagged by ALARM NMR, an industry-developed counter-screen for promiscuous compounds.  Biochemical counter-screens confirm that most of these compounds are either thiol-reactive or aggregators.  Selectivity panels show many of these compounds modulate unrelated targets in vitro, while several also demonstrate nonspecific effects in cell assays.  These data demonstrate the usefulness of performing counter-screens for bioassay promiscuity and assay interference, and raise caution about the utility of many widely used, but insufficiently validated, compounds employed in chemical biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ25EPALk8nTBKDYG3bNImrfW6udTcc2eYOOrZpQi_xjbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3hs1Oisw%253D%253D&md5=07b4dbf2b38cebeacdc9db856a92f339</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01657-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01657-3%26sid%3Dliteratum%253Aachs%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DNelson%26aufirst%3DK.%2BM.%26aulast%3DStrasser%26aufirst%3DJ.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DOrgan%26aufirst%3DS.%26aulast%3DCuellar%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DShrimp%26aufirst%3DJ.%2BH.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DMeier%26aufirst%3DJ.%2BL.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DAssay%2520Interference%2520and%2520Off-Target%2520Liabilities%2520of%2520Reported%2520Histone%2520Acetyltransferase%2520Inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1527%26doi%3D10.1038%2Fs41467-017-01657-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soccio, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ait-Si-Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolffe, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">HATs off</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(00)80452-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fs1097-2765%2800%2980452-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=589-595&author=O.+D.+Lauauthor=T.+K.+Kunduauthor=R.+E.+Soccioauthor=S.+Ait-Si-Aliauthor=E.+M.+Khalilauthor=A.+Vassilevauthor=A.+P.+Wolffeauthor=Y.+Nakataniauthor=R.+G.+Roederauthor=P.+A.+Cole&title=HATs+off&doi=10.1016%2Fs1097-2765%2800%2980452-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2800%2980452-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252800%252980452-9%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DO.%2BD.%26aulast%3DKundu%26aufirst%3DT.%2BK.%26aulast%3DSoccio%26aufirst%3DR.%2BE.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26aulast%3DKhalil%26aufirst%3DE.%2BM.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DWolffe%26aufirst%3DA.%2BP.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DHATs%2520off%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D5%26spage%3D589%26epage%3D595%26doi%3D10.1016%2Fs1097-2765%2800%2980452-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rubenstein, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dryer, R.</span></span> <span> </span><span class="NLM_article-title">S-Acetonyl-Coa. A Nonreactive Analog of Acetyl-Coa</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>255</i></span>,  <span class="NLM_fpage">7858</span>– <span class="NLM_lpage">7862</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=255&publication_year=1980&pages=7858-7862&author=P.+Rubensteinauthor=R.+Dryer&title=S-Acetonyl-Coa.+A+Nonreactive+Analog+of+Acetyl-Coa"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRubenstein%26aufirst%3DP.%26aulast%3DDryer%26aufirst%3DR.%26atitle%3DS-Acetonyl-Coa.%2520A%2520Nonreactive%2520Analog%2520of%2520Acetyl-Coa%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1980%26volume%3D255%26spage%3D7858%26epage%3D7862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology Approaches for Investigating the Functions of Lysine Acetyltransferases</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1162</span>– <span class="NLM_lpage">1184</span>, <span class="refDoi"> DOI: 10.1002/anie.201704745</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1002%2Fanie.201704745" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1162-1184&author=M.+Heauthor=Z.+Hanauthor=L.+Liuauthor=Y.+G.+Zheng&title=Chemical+Biology+Approaches+for+Investigating+the+Functions+of+Lysine+Acetyltransferases&doi=10.1002%2Fanie.201704745"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fanie.201704745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201704745%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DChemical%2520Biology%2520Approaches%2520for%2520Investigating%2520the%2520Functions%2520of%2520Lysine%2520Acetyltransferases%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1162%26epage%3D1184%26doi%3D10.1002%2Fanie.201704745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelleher, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">A Selective Chemical Probe for Coenzyme a-Requiring Enzymes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">7621</span>– <span class="NLM_lpage">7624</span>, <span class="refDoi"> DOI: 10.1002/anie.200702485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1002%2Fanie.200702485" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=7621-7624&author=Y.+Hwangauthor=P.+R.+Thompsonauthor=L.+Wangauthor=L.+Jiangauthor=N.+L.+Kelleherauthor=P.+A.+Cole&title=A+Selective+Chemical+Probe+for+Coenzyme+a-Requiring+Enzymes&doi=10.1002%2Fanie.200702485"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.200702485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200702485%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DY.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DKelleher%26aufirst%3DN.%2BL.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520Selective%2520Chemical%2520Probe%2520for%2520Coenzyme%2520a-Requiring%2520Enzymes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2007%26volume%3D46%26spage%3D7621%26epage%3D7624%26doi%3D10.1002%2Fanie.200702485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Profiling Cellular Substrates of Lysine Acetyltransferases Gcn5 and P300 with Orthogonal Labeling and Click Chemistry</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1547</span>– <span class="NLM_lpage">1555</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00114</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00114" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt12rtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1547-1555&author=Z.+Hanauthor=C.-w.+Chouauthor=X.+Yangauthor=M.+G.+Bartlettauthor=Y.+G.+Zheng&title=Profiling+Cellular+Substrates+of+Lysine+Acetyltransferases+Gcn5+and+P300+with+Orthogonal+Labeling+and+Click+Chemistry&doi=10.1021%2Facschembio.7b00114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling Cellular Substrates of Lysine Acetyltransferases GCN5 and p300 with Orthogonal Labeling and Click Chemistry</span></div><div class="casAuthors">Han, Zhen; Chou, Chau-wen; Yang, Xiangkun; Bartlett, Michael G.; Zheng, Y. George</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1547-1555</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P300 and GCN5 are two representative lysine acetyltransferases (KATs) in mammalian cells.  It was recently reported that they possess multiple acyltransferase activities including acetylation, propionylation, and butyrylation of the ε-amino group of lysine residues of histones and non-histone protein substrates.  Although thousands of acetylated substrates and acetylation sites have been identified by mass spectrometry-based proteomic screening, our knowledge about the causative connections between individual KAT members and their corresponding sub-acylomes remain very limited.  Herein, we applied 3-azidopropionyl CoA (3AZ-CoA) as a bioorthogonal surrogate of acetyl-, propionyl- and butyryl-CoA for KAT substrate identification.  We successfully attached the azide as a chem. warhead to cellular substrates of wild-type p300 and engineered GCN5.  The substrates were subsequently labeled with biotin tag through the copper-catalyzed azide-alkyne cycloaddn. (CuAAC).  Following protein enrichment on streptavidin-coated resin, we conducted LC-MS/MS studies from which more than four hundred proteins were identified as GCN5 or p300 substrate candidates.  These proteins are either p300- or GCN5-unique or shared by the two KATs and are extensively involved in various biol. events including gene expression, cell cycle, and cellular metab.  We also exptl. validated two novel substrates of GCN5, i.e., IQGAP1 and SMC1.  These results demonstrate extensive engagement of GCN5 and p300 in cellular pathways and provide new insights into understanding their functions in specific biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOrvzPGraY7rVg90H21EOLACvtfcHk0lh67tiI0KNBmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmt12rtro%253D&md5=1a41940467a3d05736228e311e51189a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00114%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DChou%26aufirst%3DC.-w.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBartlett%26aufirst%3DM.%2BG.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DProfiling%2520Cellular%2520Substrates%2520of%2520Lysine%2520Acetyltransferases%2520Gcn5%2520and%2520P300%2520with%2520Orthogonal%2520Labeling%2520and%2520Click%2520Chemistry%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D1547%26epage%3D1555%26doi%3D10.1021%2Facschembio.7b00114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">Labeling Lysine Acetyltransferase Substrates with Engineered Enzymes and Functionalized Cofactor Surrogates</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">7791</span>– <span class="NLM_lpage">7794</span>, <span class="refDoi"> DOI: 10.1021/ja311636b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311636b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVWisbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=7791-7794&author=C.+Yangauthor=J.+Miauthor=Y.+Fengauthor=L.+Ngoauthor=T.+Gaoauthor=L.+Yanauthor=Y.+G.+Zheng&title=Labeling+Lysine+Acetyltransferase+Substrates+with+Engineered+Enzymes+and+Functionalized+Cofactor+Surrogates&doi=10.1021%2Fja311636b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Labeling Lysine Acetyltransferase Substrates with Engineered Enzymes and Functionalized Cofactor Surrogates</span></div><div class="casAuthors">Yang, Chao; Mi, Jiaqi; Feng, You; Ngo, Liza; Gao, Tielong; Yan, Leilei; Zheng, Yujun George</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7791-7794</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Elucidating biol. and pathol. functions of protein lysine acetyltransferases (KATs) greatly depends on the knowledge of the dynamic and spatial localization of their enzymic targets in the cellular proteome.  We report the design and application of chem. probes for facile labeling and detection of substrates of the three major human KAT enzymes.  In this approach, we create engineered KATs in conjunction with synthetic Ac-CoA surrogates to effectively label KAT substrates even in the presence of competitive nascent cofactor acetyl-CoA.  The functionalized and transferable acyl moiety of the Ac-CoA analogs further allowed the labeled substrates to be probed with alkynyl or azido-tagged fluorescent reporters by the copper-catalyzed azide-alkyne cycloaddn.  The synthetic cofactors, in combination with either native or rationally engineered KAT enzymes, provide a versatile chem. biol. strategy to label and profile cellular targets of KATs at the proteomic level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxR6hjOF2JsrVg90H21EOLACvtfcHk0lj4PDv3JaXHIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVWisbw%253D&md5=1a970f3a3802a2b77441d00fabbf0589</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja311636b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311636b%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DMi%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DNgo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DLabeling%2520Lysine%2520Acetyltransferase%2520Substrates%2520with%2520Engineered%2520Enzymes%2520and%2520Functionalized%2520Cofactor%2520Surrogates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D7791%26epage%3D7794%26doi%3D10.1021%2Fja311636b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorum, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guasch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicklaus, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, J. L.</span></span> <span> </span><span class="NLM_article-title">Metabolic Regulation of Histone Acetyltransferases by Endogenous Acyl-Coa Cofactors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1030</span>– <span class="NLM_lpage">1039</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.chembiol.2015.06.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=26190825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2ksL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=1030-1039&author=D.+C.+Montgomeryauthor=A.+W.+Sorumauthor=L.+Guaschauthor=M.+C.+Nicklausauthor=J.+L.+Meier&title=Metabolic+Regulation+of+Histone+Acetyltransferases+by+Endogenous+Acyl-Coa+Cofactors&doi=10.1016%2Fj.chembiol.2015.06.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Regulation of Histone Acetyltransferases by Endogenous Acyl-CoA Cofactors</span></div><div class="casAuthors">Montgomery, David C.; Sorum, Alexander W.; Guasch, Laura; Nicklaus, Marc C.; Meier, Jordan L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1030-1039</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The finding that chromatin modifications are sensitive to changes in cellular cofactor levels potentially links altered tumor cell metab. and gene expression.  However, the specific enzymes and metabolites that connect these two processes remain obscure.  Characterizing these metabolic-epigenetic axes is crit. to understanding how metab. supports signaling in cancer, and developing therapeutic strategies to disrupt this process.  Here, the authors describe a chem. approach to define the metabolic regulation of lysine acetyltransferase (KAT) enzymes.  Using a novel chemoproteomic probe, the authors identify a previously unreported interaction between palmitoyl CoA (palmitoyl-CoA) and KAT enzymes.  Further anal. reveals that palmitoyl-CoA is a potent inhibitor of KAT activity and that fatty acyl-CoA precursors reduce cellular histone acetylation levels.  These studies implicate fatty acyl-CoAs as endogenous regulators of histone acetylation, and suggest novel strategies for the study and metabolic modulation of epigenetic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ2ykKcB6OU7Vg90H21EOLACvtfcHk0lj4PDv3JaXHIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2ksL3P&md5=dd2a665db3e0b430bae58a9e35c25d85</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DD.%2BC.%26aulast%3DSorum%26aufirst%3DA.%2BW.%26aulast%3DGuasch%26aufirst%3DL.%26aulast%3DNicklaus%26aufirst%3DM.%2BC.%26aulast%3DMeier%26aufirst%3DJ.%2BL.%26atitle%3DMetabolic%2520Regulation%2520of%2520Histone%2520Acetyltransferases%2520by%2520Endogenous%2520Acyl-Coa%2520Cofactors%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D1030%26epage%3D1039%26doi%3D10.1016%2Fj.chembiol.2015.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Human Histone Acetyltransferase P/Caf</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">15652</span>, <span class="refDoi"> DOI: 10.1021/bi005121q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi005121q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotVGgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=15652&author=K.+G.+Tannerauthor=M.+R.+Langerauthor=J.+M.+Denu&title=Kinetic+Mechanism+of+Human+Histone+Acetyltransferase+P%2FCaf&doi=10.1021%2Fbi005121q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic mechanism of human histone acetyltransferase P/CAF. [Erratum to document cited in CA133:360391]</span></div><div class="casAuthors">Tanner, Kirk G.; Langer, Michael R.; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">15652</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">-Assocd. factor histone acetyltransferase by Lau, O. D., Courtney, A. D., Vassilev, A., Marzilli, L. A., Cotter, R. J., Nakatani, Y., and Cole, P. A. (2000) J. Biol. Chem. 275, 12953-21959, should have appeared in the final version of the Discussion section.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYL9bSHcbmzLVg90H21EOLACvtfcHk0lj4PDv3JaXHIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotVGgu78%253D&md5=62bf2f1230a44c80c460f22371141b60</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi005121q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi005121q%26sid%3Dliteratum%253Aachs%26aulast%3DTanner%26aufirst%3DK.%2BG.%26aulast%3DLanger%26aufirst%3DM.%2BR.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DKinetic%2520Mechanism%2520of%2520Human%2520Histone%2520Acetyltransferase%2520P%252FCaf%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D15652%26doi%3D10.1021%2Fbi005121q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wapenaar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Wouden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groves, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, F. J.</span></span> <span> </span><span class="NLM_article-title">Enzyme Kinetics and Inhibition of Histone Acetyltransferase Kat8</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.ejmech.2015.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=26505788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslensr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2015&pages=289-296&author=H.+Wapenaarauthor=P.+E.+van+der+Woudenauthor=M.+R.+Grovesauthor=D.+Rotiliauthor=A.+Maiauthor=F.+J.+Dekker&title=Enzyme+Kinetics+and+Inhibition+of+Histone+Acetyltransferase+Kat8&doi=10.1016%2Fj.ejmech.2015.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme kinetics and inhibition of histone acetyltransferase KAT8</span></div><div class="casAuthors">Wapenaar, Hannah; van der Wouden, Petra E.; Groves, Matthew R.; Rotili, Dante; Mai, Antonello; Dekker, Frank J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">289-296</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Lysine acetyltransferase 8 (KAT8) is a histone acetyltransferase (HAT) responsible for acetylating lysine 16 on histone H4 (H4K16) and plays a role in cell cycle progression as well as acetylation of the tumor suppressor protein p53.  Further studies on its biol. function and drug discovery initiatives will benefit from the development of small mol. inhibitors for this enzyme.  As a first step towards this aim we investigated the enzyme kinetics of this bi-substrate enzyme.  The kinetic expts. indicate a ping-pong mechanism in which the enzyme binds Ac-CoA first, followed by binding of the histone substrate.  This mechanism is supported by affinity measurements of both substrates using isothermal titrn. calorimetry (ITC).  Using this information, the KAT8 inhibition of a focused compd. collection around the non-selective HAT inhibitor anacardic acid has been investigated.  Kinetic studies with anacardic acid were performed, based on which a model for the catalytic activity of KAT8 and the inhibitory action of anacardic acid (AA) was proposed.  This enabled the calcn. of the inhibition const. Ki of anacardic acid derivs. using an adaptation of the Cheng-Prusoff equation.  The results described in this study give insight into the catalytic mechanism of KAT8 and present the first well-characterized small-mol. inhibitors for this HAT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPdUOXyrmJwLVg90H21EOLACvtfcHk0lj6d9v7vq04WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslensr7L&md5=333561a2d8d12536c62cbd40e3ba1a00</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DWapenaar%26aufirst%3DH.%26aulast%3Dvan%2Bder%2BWouden%26aufirst%3DP.%2BE.%26aulast%3DGroves%26aufirst%3DM.%2BR.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DDekker%26aufirst%3DF.%2BJ.%26atitle%3DEnzyme%2520Kinetics%2520and%2520Inhibition%2520of%2520Histone%2520Acetyltransferase%2520Kat8%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D105%26spage%3D289%26epage%3D296%26doi%3D10.1016%2Fj.ejmech.2015.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaczmarska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goudarzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Márquez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panne, D.</span></span> <span> </span><span class="NLM_article-title">Structure of P300 in Complex with Acyl-Coa Variants</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnchembio.2217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=27820805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=21-29&author=Z.+Kaczmarskaauthor=E.+Ortegaauthor=A.+Goudarziauthor=H.+Huangauthor=S.+Kimauthor=J.+A.+M%C3%A1rquezauthor=Y.+Zhaoauthor=S.+Khochbinauthor=D.+Panne&title=Structure+of+P300+in+Complex+with+Acyl-Coa+Variants&doi=10.1038%2Fnchembio.2217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of p300 in complex with acyl-CoA variants</span></div><div class="casAuthors">Kaczmarska, Zuzanna; Ortega, Esther; Goudarzi, Afsaneh; Huang, He; Kim, Sunjoo; Marquez, Jose A.; Zhao, Yingming; Khochbin, Saadi; Panne, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-29</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone acetylation plays an important role in transcriptional activation.  Histones are also modified by chem. diverse acylations that are frequently deposited by p300, a transcriptional coactivator that uses a no. of different acyl-CoA cofactors.  Here, the authors report that while p300 is a robust acetylase, its activity gets weaker with increasing acyl-CoA chain length.  Crystal structures of p300 in complex with propionyl-, crotonyl-, or butyryl-CoA showed that the aliph. portions of these cofactors were bound in the lysine substrate-binding tunnel in a conformation that was incompatible with substrate transfer.  Lysine substrate binding was predicted to remodel the acyl-CoA ligands into a conformation compatible with acyl-chain transfer.  This remodeling required that the aliph. portion of acyl-CoA be accommodated in a hydrophobic pocket in the enzyme active site.  The size of the pocket and its aliph. nature exclude long-chain and charged acyl-CoA variants, presumably explaining the cofactor preference for p300.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGKLa3WbifTrVg90H21EOLACvtfcHk0lj6d9v7vq04WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurzK&md5=4db48a8cd73d48bcb8d94bb980e064a0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2217%26sid%3Dliteratum%253Aachs%26aulast%3DKaczmarska%26aufirst%3DZ.%26aulast%3DOrtega%26aufirst%3DE.%26aulast%3DGoudarzi%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DM%25C3%25A1rquez%26aufirst%3DJ.%2BA.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DKhochbin%26aufirst%3DS.%26aulast%3DPanne%26aufirst%3DD.%26atitle%3DStructure%2520of%2520P300%2520in%2520Complex%2520with%2520Acyl-Coa%2520Variants%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D21%26epage%3D29%26doi%3D10.1038%2Fnchembio.2217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlick, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, J. L.</span></span> <span> </span><span class="NLM_article-title">Global Profiling of Acetyltransferase Feedback Regulation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">6388</span>– <span class="NLM_lpage">6391</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b03036</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b03036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlCmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=6388-6391&author=D.+C.+Montgomeryauthor=J.+M.+Garlickauthor=R.+A.+Kulkarniauthor=S.+Kennedyauthor=A.+Allali-Hassaniauthor=Y.-M.+Kuoauthor=A.+J.+Andrewsauthor=H.+Wuauthor=M.+Vedadiauthor=J.+L.+Meier&title=Global+Profiling+of+Acetyltransferase+Feedback+Regulation&doi=10.1021%2Fjacs.6b03036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Global Profiling of Acetyltransferase Feedback Regulation</span></div><div class="casAuthors">Montgomery, David C.; Garlick, Julie M.; Kulkarni, Rhushikesh A.; Kennedy, Steven; Allali-Hassani, Abdellah; Kuo, Yin-Ming; Andrews, Andrew J.; Wu, Hong; Vedadi, Masoud; Meier, Jordan L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6388-6391</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine acetyltransferases (KATs) are key mediators of cell signaling.  Methods capable of providing new insights into their regulation thus constitute an important goal.  Here we report an optimized platform for profiling KAT-ligand interactions in complex proteomes using inhibitor-functionalized capture resins.  This approach greatly expands the scope of KATs, KAT complexes, and CoA-dependent enzymes accessible to chemoproteomic methods.  This enhanced profiling platform is then applied in the most comprehensive anal. to date of KAT inhibition by the feedback metabolite CoA.  Our studies reveal that members of the KAT superfamily possess a spectrum of sensitivity to CoA and highlight NAT10 as a novel KAT that may be susceptible to metabolic feedback inhibition.  This platform provides a powerful tool to define the potency and selectivity of reversible stimuli, such as small mols. and metabolites, that regulate KAT-dependent signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcly-8PByigLVg90H21EOLACvtfcHk0lj6d9v7vq04WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlCmur0%253D&md5=426663d4fb51b9da29bfc2b778b61bdc</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b03036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b03036%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DD.%2BC.%26aulast%3DGarlick%26aufirst%3DJ.%2BM.%26aulast%3DKulkarni%26aufirst%3DR.%2BA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKuo%26aufirst%3DY.-M.%26aulast%3DAndrews%26aufirst%3DA.%2BJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMeier%26aufirst%3DJ.%2BL.%26atitle%3DGlobal%2520Profiling%2520of%2520Acetyltransferase%2520Feedback%2520Regulation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D6388%26epage%3D6391%26doi%3D10.1021%2Fjacs.6b03036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldenburg, K. R.</span></span> <span> </span><span class="NLM_article-title">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span>. <i>J. Biomol. Screen</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1177/108705719900400206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1177%2F108705719900400206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10838414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=67-73&author=J.-H.+Zhangauthor=T.+D.+Chungauthor=K.+R.+Oldenburg&title=A+Simple+Statistical+Parameter+for+Use+in+Evaluation+and+Validation+of+High+Throughput+Screening+Assays&doi=10.1177%2F108705719900400206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span></div><div class="casAuthors">Zhang; Chung; Oldenburg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ability to identify active compounds (3hits2) from large chemical libraries accurately and rapidly has been the ultimate goal in developing high-throughput screening (HTS) assays.  The ability to identify hits from a particular HTS assay depends largely on the suitability or quality of the assay used in the screening.  The criteria or parameters for evaluating the 3suitability2 of an HTS assay for hit identification are not well defined and hence it still remains difficult to compare the quality of assays directly.  In this report, a screening window coefficient, called 3Z-factor,2 is defined.  This coefficient is reflective of both the assay signal dynamic range and the data variation associated with the signal measurements, and therefore is suitable for assay quality assessment.  The Z-factor is a dimensionless, simple statistical characteristic for each HTS assay.  The Z-factor provides a useful tool for comparison and evaluation of the quality of assays, and can be utilized in assay optimization and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkwW5JDdXnbsNIRw0GKUoEfW6udTcc2eaQkFs0ZXtiBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D&md5=0ec4ed2ccdd3b4a16521567a0975d31f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1177%2F108705719900400206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705719900400206%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.-H.%26aulast%3DChung%26aufirst%3DT.%2BD.%26aulast%3DOldenburg%26aufirst%3DK.%2BR.%26atitle%3DA%2520Simple%2520Statistical%2520Parameter%2520for%2520Use%2520in%2520Evaluation%2520and%2520Validation%2520of%2520High%2520Throughput%2520Screening%2520Assays%26jtitle%3DJ.%2520Biomol.%2520Screen%26date%3D1999%26volume%3D4%26spage%3D67%26epage%3D73%26doi%3D10.1177%2F108705719900400206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seethala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzel, R.</span></span> <span> </span><span class="NLM_article-title">A Fluorescence Polarization Competition Immunoassay for Tyrosine Kinases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>255</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1006/abio.1997.2455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1006%2Fabio.1997.2455" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=255&publication_year=1998&pages=257-262&author=R.+Seethalaauthor=R.+Menzel&title=A+Fluorescence+Polarization+Competition+Immunoassay+for+Tyrosine+Kinases&doi=10.1006%2Fabio.1997.2455"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1006%2Fabio.1997.2455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1997.2455%26sid%3Dliteratum%253Aachs%26aulast%3DSeethala%26aufirst%3DR.%26aulast%3DMenzel%26aufirst%3DR.%26atitle%3DA%2520Fluorescence%2520Polarization%2520Competition%2520Immunoassay%2520for%2520Tyrosine%2520Kinases%26jtitle%3DAnal.%2520Biochem.%26date%3D1998%26volume%3D255%26spage%3D257%26epage%3D262%26doi%3D10.1006%2Fabio.1997.2455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yung-Chi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the inhibition constant (K1) and the concentration of inhibitor which causes 50 per cent inhibition (I50) of an enzymatic reaction</span>. <i>Biochem Pharmacol</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28K1%29+and+the+concentration+of+inhibitor+which+causes+50+per+cent+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528K1%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2520per%2520cent%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem%2520Pharmacol%26date%3D1973%26volume%3D22%26spage%3D3099%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ringel, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolberger, C.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Acyl-Group Discrimination by Human Gcn5l2</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">848</span>, <span class="refDoi"> DOI: 10.1107/s2059798316007907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1107%2Fs2059798316007907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSntr7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=841-848&author=A.+E.+Ringelauthor=C.+Wolberger&title=Structural+Basis+for+Acyl-Group+Discrimination+by+Human+Gcn5l2&doi=10.1107%2Fs2059798316007907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for acyl-group discrimination by human Gcn5L2</span></div><div class="casAuthors">Ringel, Alison E.; Wolberger, Cynthia</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">841-848</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Gcn5 is a conserved acetyltransferase that regulates transcription by acetylating the N-terminal tails of histones.  Motivated by recent studies identifying a chem. diverse array of lysine acyl modifications in vivo, the acyl-chain specificity of the acetyltransferase human Gcn5 (Gcn5L2) was examd.  Whereas Gcn5L2 robustly catalyzes lysine acetylation, the acyltransferase activity of Gcn5L2 becomes progressively weaker with increasing acyl-chain length.  To understand how Gcn5 discriminates between different acyl-CoA mols., structures of the catalytic domain of human Gcn5L2 bound to propionyl-CoA and butyryl-CoA were detd.  Although the active site of Gcn5L2 can accommodate propionyl-CoA and butyryl-CoA without major structural rearrangements, butyryl-CoA adopts a conformation incompatible with catalysis that obstructs the path of the incoming lysine residue and acts as a competitive inhibitor of Gcn5L2 vs. acetyl-CoA.  These structures demonstrate how Gcn5L2 discriminates between acyl-chain donors and explain why Gcn5L2 has weak activity for acyl moieties that are larger than an acetyl group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa0SleEtFeYbVg90H21EOLACvtfcHk0ljT6wBspwy0OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSntr7M&md5=df96aaf49e264a661821a765e8774844</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2Fs2059798316007907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs2059798316007907%26sid%3Dliteratum%253Aachs%26aulast%3DRingel%26aufirst%3DA.%2BE.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DStructural%2520Basis%2520for%2520Acyl-Group%2520Discrimination%2520by%2520Human%2520Gcn5l2%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2016%26volume%3D72%26spage%3D841%26epage%3D848%26doi%3D10.1107%2Fs2059798316007907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clements, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Histone Acetyltransferase Domain of the Human Pcaf Transcriptional Regulator Bound to Coenzyme A</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3521</span>– <span class="NLM_lpage">3532</span>, <span class="refDoi"> DOI: 10.1093/emboj/18.13.3521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1093%2Femboj%2F18.13.3521" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=3521-3532&author=A.+Clementsauthor=J.+R.+Rojasauthor=R.+C.+Trievelauthor=L.+Wangauthor=S.+L.+Bergerauthor=R.+Marmorstein&title=Crystal+Structure+of+the+Histone+Acetyltransferase+Domain+of+the+Human+Pcaf+Transcriptional+Regulator+Bound+to+Coenzyme+A&doi=10.1093%2Femboj%2F18.13.3521"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F18.13.3521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F18.13.3521%26sid%3Dliteratum%253Aachs%26aulast%3DClements%26aufirst%3DA.%26aulast%3DRojas%26aufirst%3DJ.%2BR.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Histone%2520Acetyltransferase%2520Domain%2520of%2520the%2520Human%2520Pcaf%2520Transcriptional%2520Regulator%2520Bound%2520to%2520Coenzyme%2520A%26jtitle%3DEMBO%2520J.%26date%3D1999%26volume%3D18%26spage%3D3521%26epage%3D3532%26doi%3D10.1093%2Femboj%2F18.13.3521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">The Structural Basis of Protein Acetylation by the P300/Cbp Transcriptional Coactivator</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>451</i></span>,  <span class="NLM_fpage">846</span>– <span class="NLM_lpage">850</span>, <span class="refDoi"> DOI: 10.1038/nature06546</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnature06546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=18273021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvVyms74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2008&pages=846-850&author=X.+Liuauthor=L.+Wangauthor=K.+Zhaoauthor=P.+R.+Thompsonauthor=Y.+Hwangauthor=R.+Marmorsteinauthor=P.+A.+Cole&title=The+Structural+Basis+of+Protein+Acetylation+by+the+P300%2FCbp+Transcriptional+Coactivator&doi=10.1038%2Fnature06546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis of protein acetylation by the p300/CBP transcriptional coactivator</span></div><div class="casAuthors">Liu, Xin; Wang, Ling; Zhao, Kehao; Thompson, Paul R.; Hwang, Yousang; Marmorstein, Ronen; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">7180</span>),
    <span class="NLM_cas:pages">846-850</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The transcriptional coactivator p300/CBP (CREBBP) is a histone acetyltransferase (HAT) that regulates gene expression by acetylating histones and other transcription factors.  Dysregulation of p300/CBP HAT activity contributes to various diseases including cancer.  Sequence alignments, enzymol. expts. and inhibitor studies on p300/CBP have led to contradictory results about its catalytic mechanism and its structural relation to the Gcn5/PCAF and MYST HATs.  Here we describe a high-resoln. x-ray crystal structure of a semi-synthetic heterodimeric p300 HAT domain in complex with a bi-substrate inhibitor, Lys-CoA.  This structure shows that p300/CBP is a distant cousin of other structurally characterized HATs, but reveals several novel features that explain the broad substrate specificity and preference for nearby basic residues.  Based on this structure and accompanying biochem. data, we propose that p300/CBP uses an unusual 'hit-and-run' (Theorell-Chance) catalytic mechanism that is distinct from other characterized HATs.  Several disease-assocd. mutations can also be readily accounted for by the p300 HAT structure.  These studies pave the way for new epigenetic therapies involving modulation of p300/CBP HAT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeYOui0tFND7Vg90H21EOLACvtfcHk0ljCjRhdyVBSpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvVyms74%253D&md5=c393f167458f8532f2d3b06e96f7eb00</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnature06546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06546%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DHwang%26aufirst%3DY.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DThe%2520Structural%2520Basis%2520of%2520Protein%2520Acetylation%2520by%2520the%2520P300%252FCbp%2520Transcriptional%2520Coactivator%26jtitle%3DNature%26date%3D2008%26volume%3D451%26spage%3D846%26epage%3D850%26doi%3D10.1038%2Fnature06546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadlec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallacli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Weatherby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, A.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Mof and Msl3 Recruitment into the Dosage Compensation Complex by Msl1</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">142</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1038/nsmb.1960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1038%2Fnsmb.1960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=21217699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=142-149&author=J.+Kadlecauthor=E.+Hallacliauthor=M.+Lippauthor=H.+Holzauthor=J.+Sanchez-Weatherbyauthor=S.+Cusackauthor=A.+Akhtar&title=Structural+Basis+for+Mof+and+Msl3+Recruitment+into+the+Dosage+Compensation+Complex+by+Msl1&doi=10.1038%2Fnsmb.1960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for MOF and MSL3 recruitment into the dosage compensation complex by MSL1</span></div><div class="casAuthors">Kadlec, Jan; Hallacli, Erinc; Lipp, Michael; Holz, Herbert; Sanchez-Weatherby, Juan; Cusack, Stephen; Akhtar, Asifa</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">142-149</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The male-specific lethal (MSL) complex is required for dosage compensation in Drosophila melanogaster, and analogous complexes exist in mammals.  We report structures of binary complexes of mammalian MSL3 and the histone acetyltransferase (HAT) MOF with consecutive segments of MSL1.  MSL1 interacts with MSL3 as an extended chain forming an extensive hydrophobic interface, whereas the MSL1-MOF interface involves electrostatic interactions between the HAT domain and a long helix of MSL1.  This structure provides insights into the catalytic mechanism of MOF and enables us to show analogous interactions of MOF with NSL1.  In Drosophila, selective disruption of Msl1 interactions with Msl3 or Mof severely affects Msl1 targeting to the body of dosage-compensated genes and several high-affinity sites, without affecting promoter binding.  We propose that Msl1 acts as a scaffold for MSL complex assembly to achieve specific targeting to the X chromosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLrkomJizA6LVg90H21EOLACvtfcHk0ljCjRhdyVBSpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtVKrtA%253D%253D&md5=92dd57cdb95159c724113fd5aaba9557</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1960%26sid%3Dliteratum%253Aachs%26aulast%3DKadlec%26aufirst%3DJ.%26aulast%3DHallacli%26aufirst%3DE.%26aulast%3DLipp%26aufirst%3DM.%26aulast%3DHolz%26aufirst%3DH.%26aulast%3DSanchez-Weatherby%26aufirst%3DJ.%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DAkhtar%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520for%2520Mof%2520and%2520Msl3%2520Recruitment%2520into%2520the%2520Dosage%2520Compensation%2520Complex%2520by%2520Msl1%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D142%26epage%3D149%26doi%3D10.1038%2Fnsmb.1960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cebrat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugherty, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Analysis of Potential Prodrugs of Coenzyme a Analogues for the Inhibition of the Histone Acetyltransferase P300</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3307</span>– <span class="NLM_lpage">3313</span>, <span class="refDoi"> DOI: 10.1016/s0968-0896(03)00265-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fs0968-0896%2803%2900265-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=3307-3313&author=M.+Cebratauthor=C.+M.+Kimauthor=P.+R.+Thompsonauthor=M.+Daughertyauthor=P.+A.+Cole&title=Synthesis+and+Analysis+of+Potential+Prodrugs+of+Coenzyme+a+Analogues+for+the+Inhibition+of+the+Histone+Acetyltransferase+P300&doi=10.1016%2Fs0968-0896%2803%2900265-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fs0968-0896%2803%2900265-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0968-0896%252803%252900265-7%26sid%3Dliteratum%253Aachs%26aulast%3DCebrat%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DC.%2BM.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DDaugherty%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DSynthesis%2520and%2520Analysis%2520of%2520Potential%2520Prodrugs%2520of%2520Coenzyme%2520a%2520Analogues%2520for%2520the%2520Inhibition%2520of%2520the%2520Histone%2520Acetyltransferase%2520P300%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26spage%3D3307%26epage%3D3313%26doi%3D10.1016%2Fs0968-0896%2803%2900265-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Enzymatic and Cellular Study of a Serotonin N-Acetyltransferase Phosphopantetheine-Based Prodrug</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2147</span>– <span class="NLM_lpage">2155</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2006.12.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1016%2Fj.bmc.2006.12.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=2147-2155&author=Y.+Hwangauthor=S.+Gangulyauthor=A.+K.+Hoauthor=D.+C.+Kleinauthor=P.+A.+Cole&title=Enzymatic+and+Cellular+Study+of+a+Serotonin+N-Acetyltransferase+Phosphopantetheine-Based+Prodrug&doi=10.1016%2Fj.bmc.2006.12.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.12.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.12.016%26sid%3Dliteratum%253Aachs%26aulast%3DHwang%26aufirst%3DY.%26aulast%3DGanguly%26aufirst%3DS.%26aulast%3DHo%26aufirst%3DA.%2BK.%26aulast%3DKlein%26aufirst%3DD.%2BC.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DEnzymatic%2520and%2520Cellular%2520Study%2520of%2520a%2520Serotonin%2520N-Acetyltransferase%2520Phosphopantetheine-Based%2520Prodrug%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D2147%26epage%3D2155%26doi%3D10.1016%2Fj.bmc.2006.12.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B. D.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery and Its Application to Challenging Drug Targets</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1042/ebc20170029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=10.1042%2Febc20170029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=29118094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=475-484&author=A.+J.+Priceauthor=S.+Howardauthor=B.+D.+Cons&title=Fragment-Based+Drug+Discovery+and+Its+Application+to+Challenging+Drug+Targets&doi=10.1042%2Febc20170029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery and its application to challenging drug targets</span></div><div class="casAuthors">Price Amanda J; Howard Steven; Cons Benjamin D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">475-484</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a technique for identifying low molecular weight chemical starting points for drug discovery.  Since its inception 20 years ago, FBDD has grown in popularity to the point where it is now an established technique in industry and academia.  The approach involves the biophysical screening of proteins against collections of low molecular weight compounds (fragments).  Although fragments bind to proteins with relatively low affinity, they form efficient, high quality binding interactions with the protein architecture as they have to overcome a significant entropy barrier to bind.  Of the biophysical methods available for fragment screening, X-ray protein crystallography is one of the most sensitive and least prone to false positives.  It also provides detailed structural information of the protein-fragment complex at the atomic level.  Fragment-based screening using X-ray crystallography is therefore an efficient method for identifying binding hotspots on proteins, which can then be exploited by chemists and biologists for the discovery of new drugs.  The use of FBDD is illustrated here with a recently published case study of a drug discovery programme targeting the challenging protein-protein interaction Kelch-like ECH-associated protein 1:nuclear factor erythroid 2-related factor 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0VHFYivNCdPRfW6udTcc2ea6g_IrtFLO2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D&md5=f8572516bfb1ecc4a0ef57b0c5ba1bc0</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1042%2Febc20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Febc20170029%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DCons%26aufirst%3DB.%2BD.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520and%2520Its%2520Application%2520to%2520Challenging%2520Drug%2520Targets%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D475%26epage%3D484%26doi%3D10.1042%2Febc20170029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i51"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01887">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_27833"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01887">10.1021/acs.jmedchem.8b01887</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">HPLC chromatograms and NMR spectra of reference inhibitor acetonyl-CoA and compounds <b>11</b>, and <b>12</b>; information on atom numbering and used peptides; supplemental methods and additional figures illustrating probe validation, ITC measurements, FP, BLI, and activity assay optimization and compound profiling; and supplemental references (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_001.pdf">jm8b01887_si_001.pdf (1.23 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01887/suppl_file/jm8b01887_si_002.csv">jm8b01887_si_002.csv (2.15 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01887&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01887%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-5%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01887" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a4719c6c3d82","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
