
<html lang="en"     class="pb-page"  data-request-id="6605a57b-2b67-4eb3-a1e6-6fceb6c20277"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-24;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01609;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Potent and Selective Mitogen-Activated Protein Kinase Kinase 1/2 (MEK1/2) Heterobifunctional Small-molecule Degraders" /></meta><meta name="dc.Creator" content="Jianping  Hu" /></meta><meta name="dc.Creator" content="Jieli  Wei" /></meta><meta name="dc.Creator" content="Hyerin  Yim" /></meta><meta name="dc.Creator" content="Li  Wang" /></meta><meta name="dc.Creator" content="Ling  Xie" /></meta><meta name="dc.Creator" content="Margaret S.  Jin" /></meta><meta name="dc.Creator" content="Md  Kabir" /></meta><meta name="dc.Creator" content="Lihuai  Qin" /></meta><meta name="dc.Creator" content="Xian  Chen" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Previously, we reported a first-in-class von Hippel–Lindau (VHL)-recruiting mitogen-activated protein kinase kinases 1 and 2 (MEK1/2) degrader, MS432. To date, only two MEK1/2 degrader papers have ..." /></meta><meta name="Description" content="Previously, we reported a first-in-class von Hippel–Lindau (VHL)-recruiting mitogen-activated protein kinase kinases 1 and 2 (MEK1/2) degrader, MS432. To date, only two MEK1/2 degrader papers have ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 7, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01609" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01609" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01609" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01609" /></link>
        
    
    

<title>Potent and Selective Mitogen-Activated Protein Kinase Kinase 1/2 (MEK1/2) Heterobifunctional Small-molecule Degraders | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01609" /></meta><meta property="og:title" content="Potent and Selective Mitogen-Activated Protein Kinase Kinase 1/2 (MEK1/2) Heterobifunctional Small-molecule Degraders" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0025.jpeg" /></meta><meta property="og:description" content="Previously, we reported a first-in-class von Hippel–Lindau (VHL)-recruiting mitogen-activated protein kinase kinases 1 and 2 (MEK1/2) degrader, MS432. To date, only two MEK1/2 degrader papers have been published and very limited structure–activity relationships (SAR) have been reported. Here, we describe our extensive SAR studies exploring both von Hippel–Lindau (VHL) and cereblon (CRBN) E3 ligase ligands and a variety of linkers, which resulted in two novel, improved VHL-recruiting MEK1/2 degraders, 24 (MS928) and 27 (MS934), and the first CRBN-recruiting MEK1/2 degrader 50 (MS910). These compounds potently and selectively degraded MEK1/2 by hijacking the ubiquitin-proteasome system, inhibited downstream signaling, and suppressed cancer cell proliferation. Furthermore, concurrent inhibition of BRAF or PI3K significantly potentiated the antitumor activity of degrader 27, suggesting that the combination of MEK1/2 degradation with BRAF or PI3K inhibition may provide potential therapeutic benefits. Finally, besides being more potent, degrader 27 displayed improved plasma exposure levels in mice, representing the best MEK1/2 degrader to date for in vivo studies." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01609"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01609">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01609&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01609&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01609&amp;href=/doi/10.1021/acs.jmedchem.0c01609" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 15883-15905</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01588" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01647" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Potent and Selective Mitogen-Activated Protein Kinase Kinase 1/2 (MEK1/2) Heterobifunctional Small-molecule Degraders</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jianping Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianping Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianping++Hu">Jianping Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jieli Wei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jieli Wei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jieli++Wei">Jieli Wei</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hyerin Yim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hyerin Yim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hyerin++Yim">Hyerin Yim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Li Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Li Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Li++Wang">Li Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ling Xie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ling Xie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ling++Xie">Ling Xie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Margaret S. Jin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Margaret S. Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Margaret+S.++Jin">Margaret S. Jin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Md Kabir</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Md Kabir</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Md++Kabir">Md Kabir</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lihuai Qin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lihuai Qin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lihuai++Qin">Lihuai Qin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xian Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xian Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xian++Chen">Xian Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#1a7073747d3476736f5a77696977347f7e6f"><span class="__cf_email__" data-cfemail="f3999a9d94dd9f9a86b39e80809edd969786">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2e44474f40004447406e435d5d43004b4a5b"><span class="__cf_email__" data-cfemail="f59f9c949bdb9f9c9bb598868698db909180">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01609&amp;href=/doi/10.1021%2Facs.jmedchem.0c01609" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 15883–15905</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 7, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 September 2020</li><li><span class="item_label"><b>Published</b> online</span>7 December 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01609" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01609</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D15883%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJianping%2BHu%252C%2BJieli%2BWei%252C%2BHyerin%2BYim%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D24%26contentID%3Dacs.jmedchem.0c01609%26title%3DPotent%2Band%2BSelective%2BMitogen-Activated%2BProtein%2BKinase%2BKinase%2B1%252F2%2B%2528MEK1%252F2%2529%2BHeterobifunctional%2BSmall-molecule%2BDegraders%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D15905%26publicationDate%3DDecember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01609"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1460</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01609" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Potent and Selective Mitogen-Activated Protein Kinase Kinase 1/2 (MEK1/2) Heterobifunctional Small-molecule Degraders&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jianping&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Jieli&quot;,&quot;last_name&quot;:&quot;Wei&quot;},{&quot;first_name&quot;:&quot;Hyerin&quot;,&quot;last_name&quot;:&quot;Yim&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Ling&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Margaret&quot;,&quot;last_name&quot;:&quot;S. Jin&quot;},{&quot;first_name&quot;:&quot;Md&quot;,&quot;last_name&quot;:&quot;Kabir&quot;},{&quot;first_name&quot;:&quot;Lihuai&quot;,&quot;last_name&quot;:&quot;Qin&quot;},{&quot;first_name&quot;:&quot;Xian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;15883-15905&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01609&quot;},&quot;abstract&quot;:&quot;Previously, we reported a first-in-class von Hippel–Lindau (VHL)-recruiting mitogen-activated protein kinase kinases 1 and 2 (MEK1/2) degrader, MS432. To date, only two MEK1/2 degrader papers have been published and very limited structure–activity relationships (SAR) have been reported. Here, we describe our extensive SAR studies exploring both von Hippel–Lindau (VHL) and cereblon (CRBN) E3 ligase ligands and a variety of linkers, which resulted in two novel, improved VHL-recruiting MEK1/2 degraders, 24 (MS928) and 27 (MS934), and the first CRBN-recruiting MEK1/2 degrader 50 (MS910). These compounds potently and selectively degraded MEK1/2 by hijacking the ubiquitin-proteasome system, inhibited downstream signaling, and suppressed cancer cell proliferation. Furthermore, concurrent inhibition of BRAF or PI3K significantly potentiated the antitumor activity of degrader 27, suggesting that the combination of MEK1/2 degradation with BRAF or PI3K inhibition may provide potential therapeutic benefits. Finally, besi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01609&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01609" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01609&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01609" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01609&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01609" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01609&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01609&amp;href=/doi/10.1021/acs.jmedchem.0c01609" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01609" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01609" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01609%26sid%3Dliteratum%253Aachs%26pmid%3D33284613%26genre%3Darticle%26aulast%3DHu%26date%3D2020%26atitle%3DPotent%2Band%2BSelective%2BMitogen-Activated%2BProtein%2BKinase%2BKinase%2B1%252F2%2B%2528MEK1%252F2%2529%2BHeterobifunctional%2BSmall-molecule%2BDegraders%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D24%26spage%3D15883%26epage%3D15905%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291297" title="Cell signaling">Cell signaling</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/jmcmar.2020.63.issue-24/20201224/jmcmar.2020.63.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Previously, we reported a first-in-class von Hippel–Lindau (VHL)-recruiting mitogen-activated protein kinase kinases 1 and 2 (MEK1/2) degrader, MS432. To date, only two MEK1/2 degrader papers have been published and very limited structure–activity relationships (SAR) have been reported. Here, we describe our extensive SAR studies exploring both von Hippel–Lindau (VHL) and cereblon (CRBN) E3 ligase ligands and a variety of linkers, which resulted in two novel, improved VHL-recruiting MEK1/2 degraders, <b>24</b> (MS928) and <b>27</b> (MS934), and the first CRBN-recruiting MEK1/2 degrader <b>50</b> (MS910). These compounds potently and selectively degraded MEK1/2 by hijacking the ubiquitin-proteasome system, inhibited downstream signaling, and suppressed cancer cell proliferation. Furthermore, concurrent inhibition of BRAF or PI3K significantly potentiated the antitumor activity of degrader <b>27</b>, suggesting that the combination of MEK1/2 degradation with BRAF or PI3K inhibition may provide potential therapeutic benefits. Finally, besides being more potent, degrader <b>27</b> displayed improved plasma exposure levels in mice, representing the best MEK1/2 degrader to date for in vivo studies.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37632" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37632" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Mitogen-activated protein kinase kinases 1 and 2 (MEK1/2) are critical components of the mitogen-activated protein kinase (MAPK)/extracellular signal-regulated kinase (ERK) signaling transduction pathway, which transmits an extracellular signal into the nucleus through a cascade activation of multiple proteins, including Ras, Raf, MEK, and ERK.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> As crucial proteins in the MAPK/ERK signaling, MEK1/2 are phosphorylated and activated by its upstream RAF kinases.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Activation of MEK1/2 consequently leads to the phosphorylation of ERK at the threonine and tyrosine residues and activation of the ERK signaling.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The MAPK/ERK signaling pathway is associated with a broad array of cellular processes, including cell proliferation, differentiation, cell survival, and apoptosis.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Aberrant regulation of this pathway through hyperactivation and mutation has been implicated in a variety of human cancers, such as melanoma, nonsmall cell lung cancer (NSCLC), colorectal cancer, primary brain tumors, and hepatocellular carcinoma.<a onclick="showRef(event, 'ref1 ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref1 ref8 ref9 ref10 ref11 ref12">(1,8−12)</a></div><div class="NLM_p">Pharmacological inhibition of the MAPK/ERK signaling pathway by targeting the catalytic function of Ras, Raf, MEK, and ERK has resulted in multiple FDA approved drugs and many inhibitors in clinic development.<a onclick="showRef(event, 'ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20">(13−20)</a> PD184352 was the first MEK1/2 inhibitor that entered clinical trials.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, the development of PD184352 was terminated due to the lack of clinical efficacy.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Optimization of PD184352 led to another clinical MEK1/2 inhibitor PD0325901 (<b>1</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) with improved potency, solubility, and metabolic stability.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> PD0325901 is a typical non-ATP competitive MEK1/2 inhibitor, which occupies an allosteric binding pocket adjacent to the ATP binding site.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Numerous MEK inhibitors have been reported subsequently.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Although MEK inhibitors can be used as a single agent to treat diseases, the combination of MEK and BRAF inhibitors has shown delayed drug resistance and prolonged progression-free survival.<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27−29)</a> As a result, combination therapies, such as MEK inhibitor trametinib (<b>2</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) with BRAF inhibitor dabrafenib, MEK inhibitor cobimetinib with BRAF inhibitor vemurafenib, and MEK inhibitor binimetinib with BRAF inhibitor encorafenib, have been approved by the FDA for treating BRAF-mutated melanoma.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33">(30−33)</a> These combination therapies have shown good efficacies for treating melanoma patients. However, the acquired drug resistance through reactivation and mutation has been reported.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> Therefore, new therapeutic strategies to delay or overcome drug resistance are desired.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of MEK inhibitors <b>1</b> and <b>2</b>, and MEK degraders <b>3</b> and <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Recently, proteolysis targeting chimeras (PROTACs) have emerged as a novel therapeutic strategy by reducing the protein level of oncological targets.<a onclick="showRef(event, 'ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38 ref39">(36−39)</a> PROTACs are heterobifunctional small molecules, composing of a ligand that binds the target protein, a ligand of an E3 ligase, and a linker that connects these two ligands. PROTACs bring the target protein and E3 ligase to close proximity, resulting in polyubiquitination of the target protein and its subsequent degradation by hijacking the ubiquitin-proteasome system (UPS).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The PROTAC technology has been applied to the degradation of the protein targets in the MAPK/ERK signaling pathway, including Ras,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Raf,<a onclick="showRef(event, 'ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43">(41−43)</a> MEK,<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> and ERK.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> In addition to the important functions of MEK1/2 in the canonical ERK signaling cascade, MEK1/2 possess other biological roles by phosphorylation of MyoD, HSF1, and β-arrestin 2.<a onclick="showRef(event, 'ref47 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49">(47−49)</a> Furthermore, MEK1/2 have noncatalytic functions, which have been associated with nuclear export of ERK and PPAR, repression of MyoD transactivation, and regulation of FOXO1 localization.<a onclick="showRef(event, 'ref50 ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref50 ref51 ref52 ref53">(50−53)</a> Therefore, reduction of MEK1/2 protein levels using PROTACs is expected to diminish both catalytic and noncatalytic functions of the proteins and could have more profound pharmacological effects than inhibition of the kinase activity alone.</div><div class="NLM_p">Recently, we reported a first-in-class PD0325901-derived MEK degrader <b>3</b> (MS432) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which recruits the E3 ligase von Hippel–Lindau (VHL).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Compound <b>3</b> potently degraded MEK1/2 proteins and effectively inhibited the proliferation of BRAF-mutated colorectal cancer cell line HT-29 and melanoma cell line SK-MEL-28. Shortly after this first publication, Vollmer et al. reported another VHL-recruiting MEK1/2 degrader <b>4</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which effectively inhibited the proliferation of melanoma cell line A375.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> To date, these two papers are the only MEK1/2 degrader papers. Furthermore, very limited structure–activity relationships (SAR) were reported in these two studies.</div><div class="NLM_p last">Here, we report extensive SAR studies of the PD0325901-derived MEK1/2 degraders bearing a variety of alkyl or polyethylene glycol (PEG) linkers by recruiting either VHL or cereblon (CRBN) E3 ligase. These SAR studies have led to two improved VHL-recruiting MEK1/2 degraders, <b>24</b> (MS928) and <b>27</b> (MS934), and the first CRBN-recruiting MEK1/2 degrader <b>50</b> (MS910). We also developed a negative control for each of these three MEK1/2 degraders.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Structure–Activity Relationship Studies</h3><div class="NLM_p">Previously, we evaluated the effect of a very limited number of PD0325901-derived putative MEK1/2 degraders (<b>3</b>, <b>10</b>, <b>14</b>, <b>19</b>, <b>28</b>, and <b>35</b>) on degrading MEK1/2 in HT-29 cells by Western blots.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> To directly compare the potency of these previously reported compounds with new compounds we synthesized, we used HT-29 cells as the primary cell line for screening all of the putative degraders in this extensive SAR study.</div><div class="NLM_p">First, we assessed the antiproliferation potency of VHL-recruiting MEK1/2 compounds bearing alkylene (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) or PEG (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) linkers by treating HT-29 cells for 3 days. As illustrated in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compounds with shorter alkylene linkers, such as methylene (<b>5</b>), ethylene (<b>6</b>), propylene (<b>7</b>), butylene (<b>8</b>), pentylene (<b>9</b>), and hexylene (<b>10</b>),<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> were less potent (GI<sub>50</sub> = 1.4–5.0 μM) than the ones with longer linkers (GI<sub>50</sub> = 0.4–0.6 μM), such as heptylene (<b>11</b>), octylene (<b>12</b>), nonylene (<b>13</b>), and decylene (<b>14</b>).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Compounds with one to four PEG units in the linker moiety (<b>15</b>–<b>21</b>) did not display high potency at inhibiting HT-29 cell growth (GI<sub>50</sub> = 1.3–4.8 μM, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Antiproliferation Potency of Compounds <b>5</b>–<b>14</b> in HT-29 Cells<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0017.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd.</th><th class="colsep0 rowsep0" align="center"><i>n</i></th><th class="colsep0 rowsep0" align="center" char="±">HT-29 GI<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char="±">1.8 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char="±">5.0 ± 1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char="±">3.1 ± 0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char="±">3.0 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="char" char="±">2.1 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="char" char="±">1.4 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="char" char="±">0.5 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="char" char="±">0.5 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">GI<sub>50</sub> values, represented by mean value ± standard deviation (SD), were obtained from at least two independent experiments. In GI<sub>50</sub> curves, each concentration point was performed in duplicate or triplicate.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">The compounds were previously reported.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Antiproliferation Potency of Compounds <b>15</b>–<b>21</b> in HT-29 Cells<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0018.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0019.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">GI<sub>50</sub> values, represented by mean value ± SD, were obtained from at least two independent experiments. In GI<sub>50</sub> curves, each concentration point was performed in duplicate or triplicate.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">The compound was previously reported.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></p></div></div><div></div></div><div class="NLM_p">In our previous study, we showed that <b>14</b> (GI<sub>50</sub> = 0.4 ± 0.2 μM) was able to significantly degrade MEK1/2 protein levels in HT-29 cells at 0.3, 1, and 3 μM concentrations.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> To confirm the MEK1/2 degradation capability of compounds with similar antiproliferation potency, we treated HT-29 cells with <b>11</b> (GI<sub>50</sub> = 0.5 ± 0.2 μM) or <b>13</b> (GI<sub>50</sub> = 0.5 ± 0.03 μM) at 0.3, 1, and 3 μM concentrations for 24 h. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, similar to <b>14</b>, both <b>11</b> and <b>13</b> effectively degraded MEK1 and MEK2 proteins.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Effect of compounds <b>11</b>, <b>13</b>, and <b>14</b> on MEK1/2 protein levels in HT-29 cells. The cells were treated with dimethyl sulfoxide (DMSO) or serial dilution of indicated compounds for 24 h. MEK1/2 protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We previously showed that the addition of a benzylic methyl group to the VHL binding moiety significantly improved the MEK1/2 degradation potency of the degraders.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Consistent with this result, compound <b>3</b> (GI<sub>50</sub> = 130 ± 38 nM, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) showed 3-fold antiproliferation potency improvement over compound <b>14</b> (GI<sub>50</sub> = 0.4 ± 0.2 μM, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Shorten the linker length from decylene (<b>3</b>) to nonylene (<b>22</b>) slightly decreased antiproliferation potency (GI<sub>50</sub> = 240 ± 55 nM). A longer undecylene linker (<b>23</b>) resulted in slightly better antiproliferation potency (GI<sub>50</sub> = 99 ± 8 nM). Interestingly, the introduction of a <i>N</i>-methyl group to the aminopropoxy moiety (<b>24</b>) improved antiproliferation potency (GI<sub>50</sub> = 32 ± 8 nM) by about 4-fold compared with <b>3</b>. In addition, extending the aminopropoxy moiety to an aminobutoxy moiety also led to improved antiproliferation potency. Compared to the aminopropoxy-containing compounds <b>22</b>, <b>3</b>, and <b>23</b>, the corresponding aminobutoxy-bearing analogues <b>25</b>, <b>26</b>, and <b>27</b> showed approximately 4-, 3-, and 4-fold increased antiproliferation potency in HT-29 cells (GI<sub>50</sub> = 55 ± 5 nM for <b>25</b>; 43 ± 1 nM for <b>26</b>; and 23 ± 5 nM for <b>27</b>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Antiproliferation Potency of Compounds <b>22</b>–<b>27</b> in HT-29 Cells<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0020.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd.</th><th class="colsep0 rowsep0" align="center"><i>m</i></th><th class="colsep0 rowsep0" align="center"><i>n</i></th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">HT-29 GI<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="±">240 ± 55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="±">130 ± 38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="±">99 ± 8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char="±">32 ± 8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="±">55 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="±">43 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="±">23 ± 5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">GI<sub>50</sub> values, represented by mean value ± SD, were obtained from at least two independent experiments. In GI<sub>50</sub> curves, each concentration point was performed in duplicate or triplicate.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">The compound was previously reported.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></p></div></div></div><div class="NLM_p">Next, we determined MEK1/2 degradation potencies of the two most potent VHL-recruiting compounds <b>24</b> and <b>27</b>. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">S1A, S1C</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, both <b>24</b> and <b>27</b> potently reduced MEK1/2 protein levels (<b>24</b>: DC<sub>50</sub> = 18 ± 3 nM for MEK1 and 8 ± 1 nM for MEK2; <b>27</b>: DC<sub>50</sub> = 18 ± 1 nM for MEK1 and 9 ± 3 nM for MEK2) and inhibited phosphorylation of MEK and ERK in a concentration-dependent manner in HT-29 cells. In SK-MEL-28 cells, <b>24</b> and <b>27</b> also effectively reduced MEK1/2 protein levels (<b>24</b>: DC<sub>50</sub> = 16 ± 3 nM for MEK1 and 6 ± 1 nM for MEK2; <b>27</b>: DC<sub>50</sub> = 10 ± 1 nM for MEK1 and 4 ± 1 nM for MEK2) and inhibited phosphorylated MEK and ERK levels (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">S1B, S1D</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Taken together, these results indicate that compounds <b>24</b> and <b>27</b> are improved MEK1/2 degraders, which are more potent than the previously reported MEK1/2 degrader, compound <b>3</b>, at reducing MEK1/2 protein levels, inhibiting MEK and ERK phosphorylation, and suppressing cancer cell growth.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compounds <b>24</b> and <b>27</b> degrade MEK1 and MEK2 and inhibit pMEK and pERK in a concentration-dependent manner. HT-29 (A) and SK-MEL-28 (B) cells were treated with DMSO or serial dilution of indicated compounds for 24 h. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. MEK1/2 Degradation Potencies of Compounds <b>24</b> and <b>27</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center">target protein</th><th class="colsep0 rowsep0" align="center" char="±">DC<sub>50</sub> (compound <b>24</b>, nM)</th><th class="colsep0 rowsep0" align="center" char="±">DC<sub>50</sub> (compound <b>27</b>, nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">HT-29</td><td class="colsep0 rowsep0" align="left">MEK1</td><td class="colsep0 rowsep0" align="char" char="±">18 ± 3</td><td class="colsep0 rowsep0" align="char" char="±">18 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MEK2</td><td class="colsep0 rowsep0" align="char" char="±">8 ± 1</td><td class="colsep0 rowsep0" align="char" char="±">9 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">SK-MEL-28</td><td class="colsep0 rowsep0" align="left">MEK1</td><td class="colsep0 rowsep0" align="char" char="±">16 ± 3</td><td class="colsep0 rowsep0" align="char" char="±">10 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MEK2</td><td class="colsep0 rowsep0" align="char" char="±">6 ± 1</td><td class="colsep0 rowsep0" align="char" char="±">4 ± 1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">DC<sub>50</sub> values are presented as the mean ± SD from three independent experiments.</p></div></div></div><div class="NLM_p">In addition to the above VHL-recruiting MEK1/2 degraders, we also investigated SAR of CRBN-recruiting compounds. First, we designed putative MEK1/2 degraders by attaching a variety of linkers to the 4-amino group of pomalidomide and evaluated their antiproliferative activity in HT-29 cells (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Compared to the previously reported compound <b>28</b>,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> which possesses the shortest ethylene linker, CRBN-recruiting analogues bearing a longer alkyl linker, such as propylene (<b>29</b>), butylene (<b>30</b>), pentylene (<b>31</b>), hexylene (<b>32</b>), heptylene (<b>33</b>), or octylene (<b>34</b>), did not improve the antiproliferation potency in HT-29 cells. For compounds with PEG linkers, longer linkers with four (<b>38</b>: GI<sub>50</sub> = 0.6 ± 0.09 μM) and five (<b>39</b>: GI<sub>50</sub> = 0.6 ± 0.1 μM) PEG units resulted in slightly more potent compounds than shorter PEG linkers (<b>35</b>–<b>37</b>). Next, we explored another set of MEK1/2 putative degraders by attaching a variety of alkyl (<b>40</b>–<b>45</b>) and PEG (<b>46</b>–<b>50</b>) linkers to the 5-amino group of pomalidomide (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). In general, derivatization at the 5-position of pomalidomide provided more potent compounds compared to the ones derived from the 4-position. The compound with the longest alkyl (<b>45</b>: GI<sub>50</sub> = 0.3 ± 0.1 μM) or PEG (<b>50</b>: GI<sub>50</sub> = 0.3 ± 0.06 μM) linker showed the best potency against the growth of HT-29 cells.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Antiproliferation Potency of Compounds <b>28</b>–<b>39</b> in HT-29 Cells<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0021.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0022.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">GI<sub>50</sub> values, represented by mean value ± SD, were obtained from at least two independent experiments. In GI<sub>50</sub> curves, each concentration point was performed in duplicate or triplicate.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">The compound was previously reported.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Antiproliferation Potency of Compounds <b>40</b>–<b>50</b> in HT-29 Cells<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0023.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0024.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">GI<sub>50</sub> values, represented by mean value ± SD, were obtained from at least two independent experiments. In GI<sub>50</sub> curves, each concentration point was performed in duplicate or triplicate.</p></div></div><div></div></div><div class="NLM_p">Based on the antiproliferation potency of the CRBN-recruiting compounds, we chose seven most potent (GI<sub>50</sub> = 0.3–0.5 μM) compounds, <b>28</b>, <b>40</b>, <b>42</b>, <b>44</b>, <b>45</b>, <b>49</b>, and <b>50</b>, to evaluate their effects on MEK1/2 degradation in HT-29 cells. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, compounds <b>28</b> and <b>40</b> did not show obvious MEK1/2 degradation at 0.3, 1, and 3 μM after 24 h treatment. Interestingly, compound <b>42</b> demonstrated significant selectivity at degrading MEK2 over MEK1, which is the first example that significant selectivity for degradation of MEK2 over MEK1 can be achieved. This compound could serve as a good starting point for the development of more selective MEK2 degraders. Compounds <b>44</b>, <b>45</b>, <b>49</b>, and <b>50</b> significantly degraded both MEK1 and MEK2 proteins at 0.3, 1, and 3 μM concentrations. These results, however, revealed the significant disconnection between the antiproliferation potency and MEK1/2 degradation activity of these CRBN-recruiting compounds, suggesting that in addition to their MEK1/2 kinase inhibition activity, compounds <b>28</b>, <b>40</b>, and <b>42</b> may also hit some off-targets, which may include CRBN neo-substrates. A number of studies have reported that some CRBN-recruiting PROTACs indeed degrade CRBN neo-substrates induced by immunomodulatory drugs (IMiDs).<a onclick="showRef(event, 'ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56">(54−56)</a> Our results further suggest that antiproliferation potency alone cannot predict the degradation capability of CRBN-recruiting degraders. Because compound <b>50</b> was the most effective at both degrading MEK1/2 and inhibiting cell growth among these CRBN-recruiting compounds, we further evaluated the degradation potency of this compound in both HT-29 and SK-MEL-28 cells at a variety of compound concentrations. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">S2</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>, compound <b>50</b> potently degraded MEK1 (DC<sub>50</sub> = 118 ± 23 nM in HT-29 cells; and 94 ± 3 nM in SK-MEL-28 cells) and MEK2 (DC<sub>50</sub> = 55 ± 19 nM in HT-29 cells; and 38 ± 15 nM in SK-MEL-28 cells) and inhibited MEK and ERK phosphorylation in a concentration-dependent manner. To the best of our knowledge, compound <b>50</b> is the first potent CRBN-recruiting MEK1/2 degrader.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Effects of compounds <b>28</b>, <b>40</b>, <b>42</b>, <b>44</b>, <b>45</b>, <b>49</b>, and <b>50</b> on MEK1/2 protein levels in HT-29 cells. The cells were treated with DMSO or serial dilution of indicated compounds for 24 h. MEK1/2 protein levels were determined by Western blots and normalized with α-tubulin. (B) Compound <b>50</b> reduces MEK1/2 protein levels and inhibits the downstream ERK signaling in a concentration-dependent manner in HT-29 and SK-MEL-28 cells. HT-29 and SK-MEL-28 cells were treated with DMSO or serial dilution of compound <b>50</b> for 24 h. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. MEK1/2 Degradation Potencies of Compound <b>50</b><a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center">target protein</th><th class="colsep0 rowsep0" align="center" char="±">DC<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">HT-29</td><td class="colsep0 rowsep0" align="left">MEK1</td><td class="colsep0 rowsep0" align="char" char="±">118 ± 23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MEK2</td><td class="colsep0 rowsep0" align="char" char="±">55 ± 19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">SK-MEL-28</td><td class="colsep0 rowsep0" align="left">MEK1</td><td class="colsep0 rowsep0" align="char" char="±">94 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MEK2</td><td class="colsep0 rowsep0" align="char" char="±">38 ± 15</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">DC<sub>50</sub> values are presented as the mean ± SD from three independent experiments.</p></div></div></div><div class="NLM_p">Based on considerations of structural diversity, MEK1/2 degradation potency, and antiproliferation potency, we chose compounds <b>24</b> (MS928), <b>27</b> (MS934), and <b>50</b> (MS910) for further evaluation. In addition, we designed the corresponding control compounds, <b>51</b> (MS928N), <b>52</b> (MS934N), and <b>53</b> (MS910N), for these MEK1/2 degraders. Compounds <b>51</b> and <b>52</b> are diastereomers of compounds <b>24</b> and <b>27</b> at the hydroxyproline moiety (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). It is known that such a diastereoisomer diminishes the binding to the VHL E3 ligase.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Compound <b>53</b> bears a <i>N</i>-methylated glutarimide moiety (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), which disrupts the binding to the CRBN E3 ligase.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Chemical structures of MEK degraders and their corresponding controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Binding Affinities of Compounds <b>24</b>, <b>27</b>, and <b>50</b> and Their Control Compounds to MEK1/2</h3><div class="NLM_p">The binding affinities of PD0325901, <b>24</b>, <b>27</b>, <b>50</b>, and the control compounds (<b>51</b>–<b>53</b>) to MEK1/2 were determined using the KINOMEscan assay from DiscoverX (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">Figure S3</a>). Compared with MEK1/2 inhibitor PD0325901 (<i>K</i><sub>d</sub> = 0.044 ± 0.004 μM for MEK1; and 0.10 ± 0.03 μM for MEK2), the degraders <b>24</b>, <b>27</b>, and <b>50</b> showed approximately 9- to 15-fold decreased binding affinities to MEK1 (<i>K</i><sub>d</sub> = 0.40–0.66 μM), and 14- to 24-fold decreased binding affinities to MEK2 (<i>K</i><sub>d</sub> = 1.4–2.4 μM). Considering the significant chemical structure changes at the nonsolvent exposed glycerol moiety of PD0325901, the decreased binding affinities of these degraders were not unexpected. On the other hand, modifications at the linker and the E3 ligand moieties were very well tolerated. In addition, the control compounds (<b>51</b>–<b>53</b>) displayed similar binding affinities to MEK1 and MEK2 as degraders <b>24</b>, <b>27</b>, and <b>50</b> and all of the three degraders and their control compounds showed comparable binding affinities to MEK1 and MEK2. Interestingly, although the binding affinities of degraders <b>24</b>, <b>27</b>, and <b>50</b> are stronger for MEK1 over MEK2, these degraders showed slightly better degradation potency for MEK2 over MEK1 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B; <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Tables <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>), suggesting that MEK1/2 binding affinity is not the most critical factor for determining degradation potency here.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. MEK1/2 Binding Affinities of Compounds <b>24</b>, <b>27</b>, and <b>50</b>–<b>53</b> in KINOMEscan Assay<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd.</th><th class="colsep0 rowsep0" align="center" char="±">MEK1 <i>K</i><sub>d</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">MEK2 <i>K</i><sub>d</sub> (μM)</th><th class="colsep0 rowsep0" align="center">cpd.</th><th class="colsep0 rowsep0" align="center" char="±">MEK1 <i>K</i><sub>d</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">MEK2 <i>K</i><sub>d</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PD0325901</td><td class="colsep0 rowsep0" align="char" char="±">0.044 ± 0.004</td><td class="colsep0 rowsep0" align="char" char="±">0.10 ± 0.03</td><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="char" char="±">0.38 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">1.7 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="char" char="±">0.40 ± 0.001</td><td class="colsep0 rowsep0" align="char" char="±">1.4 ± 0.54</td><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="char" char="±">0.82 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">2.2 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="char" char="±">0.40 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">2.0 ± 0.23</td><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">3.8 ± 0.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="char" char="±">0.66 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">2.4 ± 0.13</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">The experiments were performed in duplicate and the <i>K</i><sub>d</sub> values are presented as the mean ± SD.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Kinetics of MEK1/2 Degradation Induced by Compounds <b>24</b>, <b>27</b>, and <b>50</b></h3><div class="NLM_p">We next conducted time-course experiments to understand the kinetics of MEK1/2 degradation and downstream signaling inhibition induced by compounds <b>24</b>, <b>27</b>, and <b>50</b> in HT-29 cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, the VHL-recruiting degraders <b>24</b> and <b>27</b> (at 0.1 μM) induced significant MEK1/2 degradation and ERK phosphorylation inhibition within 2 h, and the maximum degradation (<i>D</i><sub>max</sub>) and ERK phosphorylation inhibition were reached within 8 h. Compared with the VHL-recruiting degraders, the CRBN-recruiting degrader <b>50</b> (at 0.3 μM) showed a slower kinetic effect. Significant MEK1/2 degradation occurred at 4 h, and the maximum degradation (<i>D</i><sub>max</sub>) was reached within 8–10 h. Consistently, compound <b>50</b> inhibited ERK phosphorylation at 8–10 h, albeit less effective compared to compounds <b>24</b> and <b>27</b>.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compounds <b>24, 27</b>, and <b>50</b> degrade MEK1 and MEK2 and inhibit pERK signaling in a time-dependent manner. HT-29 cells treated with compound <b>24</b> or <b>27</b> (0.1 μM) or <b>50</b> (0.3 μM) were harvested at indicated time points. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Mechanism of Action (MOA) of MEK1/2 Degradation Induced by Compounds <b>24</b>, <b>27</b>, and <b>50</b></h3><div class="NLM_p">To demonstrate that the observed MEK1/2 degradation effect of compounds <b>24</b>, <b>27</b>, and <b>50</b> is VHL- or CRBN-dependent, we first confirmed that control compounds <b>51</b>, <b>52</b>, and <b>53</b>, which possess modified moieties with diminished binding to the VHL or CRBN ligase, did not reduce MEK1/2 protein levels in either HT-29 or SK-MEL-28 cells (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), indicating that binding to either VHL or CRBN ligase is required for MEK1/2 degradation. In addition, these control compounds were much less potent at inhibiting MEK and ERK phosphorylation compared with degraders <b>24</b>, <b>27</b>, and <b>50</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B, and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). These results suggest that compounds <b>24</b>, <b>27</b>, and <b>50</b> inhibited the downstream ERK signaling mainly through the reduction of MEK1/2 protein levels instead of inhibition of MEK1/2 kinase activities.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Control compounds <b>51</b>–<b>53</b> do not induce MEK1/2 degradation and are less potent at inhibiting the downstream ERK signaling in HT-29 and SK-MEL-28 cells after 24 h treatment. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next performed a series of rescue experiments to demonstrate that the MEK1/2 degradation induced by compounds <b>24</b>, <b>27</b>, and <b>50</b> is through hijacking the ubiquitin-proteasome system (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Pretreatment of HT-29 cells with MEK1/2 inhibitor PD0325901 (1 μM) effectively blocked the degradation of MEK1/2 induced by <b>24</b>, <b>27</b>, or <b>50</b>, confirming that MEK1/2 binding is required for MEK1/2 degradation. Pretreatment of HT-29 cells with the proteasome inhibitor MG-132 (3 μM) also diminished the MEK1/2 degradation, indicating that the degrader-induced MEK1/2 degradation is mediated by the proteasome. In addition, pretreatment with the neddylation inhibitor MLN4924 (3 μM) rescued MEK1/2 protein levels in HT-29 cells, implying that the degradation requires an active E3 complex. Moreover, the VHL ligand VH 032 (10 μM) was able to diminish the effect of VHL-recruiting degraders <b>24</b> and <b>27</b>, and the CRBN ligand pomalidomide (POMA, 5 μM) also significantly prevented MEK1/2 degradation induced by the CRBN-recruiting degrader <b>50</b>, further confirming that MEK1/2 degradation by <b>24</b> or <b>27</b> is VHL-dependent and MEK1/2 degradation by <b>50</b> is CRBN-dependent. Interestingly, pretreatment with PD0325901, VH 032, POMA, MG-132, and MLN4924 not only prevented MEK1/2 proteins from degradation but also restored the pMEK level in HT-29 cells (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Except for PD0325901, all of these compounds also effectively restored the pERK protein level (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Taken together, these studies have demonstrated that MEK1/2 degradation induced by compounds <b>24</b>, <b>27</b>, and <b>50</b> is through hijacking the ubiquitin-proteasome system.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Degradation of MEK1/2 induced by compound <b>24</b>, <b>27</b>, or <b>50</b> is mediated by the ubiquitin-proteasome system. HT-29 cells were pretreated with DMSO, PD0325901 (1 μM), MG-132 (3 μM), MLN4924 (3 μM), VH 032 (10 μM), or pomalidomide (POMA, 5 μM) for 2 h, before <b>24</b> or <b>27</b> (0.1 μM), or <b>50</b> (0.3 μM) was added. The cells were incubated for another 8 h. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Selectivity of MEK1/2 Degraders <b>27</b> and <b>50</b></h3><div class="NLM_p">To evaluate the selectivity of the VHL-recruiting degrader <b>27</b> and CRBN-recruiting degrader <b>50</b>, we performed unbiased proteomics studies using protein samples from HT-29 cells treated with <b>27</b> (0.1 μM), <b>50</b> (0.3 μM), <b>52</b> (0.1 μM), <b>53</b> (0.3 μM), or DMSO for 8 h (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">S4</a>). Compared with DMSO control, the global proteome analysis showed that control compounds <b>52</b> and <b>53</b> did not significantly change the levels of any quantified 5,165 proteins (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A,B). On the other hand, degraders <b>27</b> and <b>50</b> only significantly decreased MEK1 and MEK2 protein levels out of the quantified 5165 proteins (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C,D). It has been reported that CRBN-recruiting PROTAC degraders have the potential to maintain the activities of IMiDs (e.g., degrading neo-substrates of CRBN). Since the global proteomic study was performed using one concentration (0.3 μM) of degrader <b>50</b> at a single time point (8 h), it is possible that we could miss some off-targets at elevated degrader concentrations and/or with extended treatment time. Furthermore, the proteomic study did not identify unique peptides for IKZF1/3 proteins. Therefore, we evaluated the protein levels of a few CRBN neo-substrates, including GSPT1, IKZF1/3, and ZFP91, in HT-29 cells treated with degrader <b>50</b> at various concentrations for 24 h using Western blotting analysis. As illustrated in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">Figure S5</a>, after 24 h treatment, degrader <b>50</b> did not significantly reduce protein levels of GSPT1, IKZF1, and ZFP91 at concentrations up to 10 μM. However, the IKZF3 protein level was significantly decreased by compound <b>50</b> at concentrations above 1 μM. Taken together, these results indicate that: (1) the VHL-recruiting degrader <b>27</b> has excellent selectivity for MEK1/2; and (2) the CRBN-recruiting degrader <b>50</b> is also selective for MEK1/2 in general. While we did not identify any off-targets in the global proteomic study, Western blotting analysis revealed that compound <b>50</b> was able to degrade the CRBN neo-substrate IKZF3, but not other neo-substrates such as GSPT1, IKZF1, and ZFP91. Nevertheless, compound <b>50</b> was more potent at degrading MEK1/2 than IKZF3.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0010.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compounds <b>27</b> and <b>50</b> selectively degrade MEK1 and MEK2 in HT-29 cells. HT-29 cells were treated with 0.1% DMSO, <b>27</b> (0.1 μM), <b>50</b> (0.3 μM), <b>52</b> (0.1 μM), or <b>53</b> (0.3 μM) for 8 h before they were harvested for mass spectrometry analysis. Volcano plots of the −log 10 (<i>p-</i>value) vs the log 2 fold change are displayed. Proteins outside the significance lines are labeled with pink or purple color (FDR = 0.05, S0 = 1). <i>P</i> values were calculated from the data of two technical replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Antiproliferative Activities of Compounds <b>24</b>, <b>27</b>, and <b>50</b></h3><div class="NLM_p">To demonstrate the advantages of MEK1/2 protein degradation over kinase activity inhibition, we evaluated the antiproliferative activities of <b>24</b>, <b>27</b>, and <b>50</b> and their corresponding control compounds <b>51</b>, <b>52</b>, and <b>53</b> in HT-29 (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A–C) and SK-MEL-28 (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>D–F) cells. Compared with the control compounds, all three MEK1/2 degraders showed better potency at inhibiting the growth of HT-29 cells. For example, <b>24</b> (GI<sub>50</sub> = 32 ± 8 nM) was 40-fold more potent than <b>51</b> (GI<sub>50</sub> = 1300 ± 160 nM); <b>27</b> (GI<sub>50</sub> = 23 ± 5 nM) was 35-fold more potent than <b>52</b> (GI<sub>50</sub> = 830 ± 85 nM); and <b>50</b> (GI<sub>50</sub> = 303 ± 57 nM) was 8-fold more potent than <b>53</b> (GI<sub>50</sub> = 2600 ± 79 nM). Similar significant differentiation between degraders and control compounds was observed in SK-MEL-28 cells. For example, <b>24</b> (GI<sub>50</sub> = 56 ± 4 nM) showed 21-fold improved potency over <b>51</b> (GI<sub>50</sub> = 1200 ± 250 nM); <b>27</b> (GI<sub>50</sub> = 40 ± 10 nM) showed 32-fold improved potency over <b>52</b> (GI<sub>50</sub> = 1300 ± 190 nM); and <b>50</b> (GI<sub>50</sub> = 780 ± 100 nM) showed 3-fold improved potency over <b>53</b> (GI<sub>50</sub> = 2500 ± 150 nM). Compared with the previously reported compound <b>3</b> (GI<sub>50</sub> = 130 ± 38 nM for HT-29 cells and 83 ± 15 nM for SK-MEL-28 cells), the new VHL-recruiting degraders <b>24</b> and <b>27</b> showed improved antiproliferation potency in both HT-29 and SK-MEL-28 cells.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0011.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compounds <b>24</b>, <b>27</b>, and <b>50</b> significantly suppress the growth of HT-29 and SK-MEL-28 cells. HT-29 (A–C) and SK-MEL-28 (D–F) cells were treated with serial dilutions of indicated compounds for 3 days, followed by WST-8 assay to get a live cell signal. For each concentration point, in quadruplicate, each mean value ± SD from three independent experiments is shown in the curves. (G–I) Clonogenic assay was conducted in HT-29 cells. HT-29 cells were treated with indicated compounds for 10 days. The Petri dish images are representative of two independent experiments. The cells were fixed and stained with crystal violet.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also assessed the potency of <b>24</b>, <b>27</b>, and <b>50</b> and their corresponding controls <b>51</b>, <b>52</b>, and <b>53</b> at inhibiting colony formation of HT-29 cells (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>G–I). Consistent with the cell antiproliferation assay results, degraders <b>24</b>, <b>27</b>, and <b>50</b> inhibited colony formation of HT-29 cells much more effectively than the corresponding controls <b>51</b>, <b>52</b>, and <b>53</b>, highlighting potential benefits of MEK1/2 protein degradation over inhibition.</div><div class="NLM_p">Taken together, these results indicated that MEK1/2 degraders <b>24</b>, <b>27</b>, and <b>50</b> were more potent at inhibiting cancer cell growth and colony formation than the control compounds that had similar binding affinities to MEK1/2 as the corresponding degraders but cannot degrade MEK1/2. Compared with the MEK1/2 inhibitor PD0325901, <b>24</b> and <b>27</b> were less potent at inhibiting the cell growth and colony formation, probably due to their decreased binding affinities to both MEK1 and MEK2 and their suboptimal physicochemical properties (e.g., much higher molecular weight) to penetrate the cell membrane.</div><div class="NLM_p">In addition, compared with compound <b>4</b>,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> a recently reported VHL-recruiting MEK1/2 degrader, compounds <b>24</b> and <b>27</b> displayed better potency at inhibiting the growth of HT-29, SK-MEL-28, H3122, and SUDHL1 cells (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). In H3122 and SUDHL1 cells, western blotting results clearly indicated that compounds <b>24</b> and <b>27</b>, but not the controls <b>51</b> and <b>52</b>, induced MEK1/2 degradation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">Figure S6</a>). Compounds <b>24</b> and <b>27</b> were also more potent than controls <b>51</b> and <b>52</b> at inhibition of MEK and ERK phosphorylation in both cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">Figure S6</a>). Notably, in SUDHL1 cells, compounds <b>24</b> (GI<sub>50</sub> = 360 ± 50 nM) and <b>27</b> (GI<sub>50</sub> = 330 ± 100 nM) exhibited similar antiproliferation potency as PD0325901 (GI<sub>50</sub> = 350 ± 70 nM) and was much more potent than compound <b>4</b> (GI<sub>50</sub> = 2400 ± 300 nM) (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>D).</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0012.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Antiproliferation effects of compounds <b>24</b> and <b>27</b> versus compound <b>4</b> in four different cancer cell lines. HT-29 (A), SK-MEL-28 (B), H3122 (C), and SUDHL1 (D) cells were treated with serial dilutions of indicated compounds for 3 days, followed by WST-8 assay to get a live cell signal. GraphPad Prism 8 was used to plot column graphs (A–C) or concentration curves (D) and calculate GI<sub>50</sub> values (D). Each concentration point was in duplicate (A, B) or triplicate (C, D). The GI<sub>50</sub> values (D) are presented as the mean ± SD from three independent experiments. Statistical significance was analyzed by Student’s <i>t</i>-tests for each concentration in the columns comparing to DMSO (A–C), <i>P</i> ≥ 0.05, ns; 0.01 ≤ <i>P</i> < 0.05, *; 0.001 ≤ <i>P</i> < 0.01, **; and <i>P</i> < 0.001, ***.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Combination Treatment</h3><div class="NLM_p">Concurrent inhibition of MEK and BRAF kinases has shown significant advantages over inhibition of BRAF alone by overcoming the paradoxical MAPK pathway activation.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Multiple combination therapies with MEK and BRAF inhibitors have been approved by the FDA to treat BRAF-mutated melanoma.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33">(30−33)</a> In addition, due to the broad crosstalk between the RAF/MEK/ERK and PI3K/AKT pathways,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> hyperactivation of the PI3K/AKT signaling has been recognized as a resistant mechanism for inhibitors that target the RAF/MEK/ERK pathway.<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62,63)</a> Therefore, concurrent inhibition of both RAF/MEK/ERK and PI3K/AKT pathways could provide another therapeutic strategy.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div><div class="NLM_p">To further explore the therapeutic potentials of MEK degraders, we evaluated the effect of combining our MEK1/2 degrader <b>27</b> with either BRAF inhibitor PLX4032 or PI3K inhibitor ZSTK474 on inhibiting the growth of HT-29 and SK-MEL-28 cells. First, we determined that the GI<sub>50</sub> values of PLX4032 and ZSTK474 were around 100 and 300 nM, respectively, for HT-29 and SK-MEL-28 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">Figure S7</a>). We next treated these two types of cells with degrader <b>27</b> at a series of concentrations combined with a fixed concentration of either PLX4032 (100 nM) or ZSTK474 (400 nM), which is around their respective GI<sub>50</sub> values. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>, both PLX4032 and ZSTK474 significantly potentiated the antiproliferation effect of <b>27</b> in HT-29 cells and moderately enhanced the antiproliferation effect of <b>27</b> in SK-MEL-28 cells (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A,B). Therefore, the combination of a MEK degrader with an inhibitor targeting another component of the same RAF/MEK/ERK signaling pathway or a different PI3K/AKT pathway could potentially provide a better therapeutic outcome than the MEK degrader as a single agent.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0013.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Concurrent inhibition of BRAF or PI3K potentiates antiproliferation potency of compound <b>27</b>. HT-29 (A) and SK-MEL-28 (B) cells were treated with indicated compounds for 3 days, followed by WST-8 assay to get a live cell signal. Each concentration point was in triplicate (A, B). Each mean value ± SD from three independent experiments is shown in the graphs. Statistical significance was analyzed by Student’s <i>t</i>-tests for each concentration in the columns comparing to DMSO. <i>P</i> ≥ 0.05, ns; 0.01 ≤ <i>P</i> < 0.05, *; 0.001 ≤ <i>P</i> < 0.01, **; and <i>P</i> < 0.001, ***.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> In Vivo Mouse Pharmacokinetic (PK) Study</h3><div class="NLM_p">We evaluated the in vivo PK properties of the best two MEK1/2 degraders, <b>24</b> and <b>27</b>, in mice. Following a single intraperitoneal (IP) injection of <b>24</b> or <b>27</b> at 50 mg/kg dose, compound concentrations in plasma were monitored at 0.5, 2, and 8 h postinjection. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>, the plasma concentrations of both compounds were well above their DC<sub>50</sub> and GI<sub>50</sub> values in HT-29 or SK-MEL-28 cells. Compound <b>27</b> showed better plasma exposure than compound <b>24</b> over the 8 h period. In addition, compound <b>27</b> was cleared more slowly than compound <b>24</b>, which could be due to the fact that compound <b>24</b> possesses the metabolically labile <i>N</i>-methyl group located in the middle of the linker. Moreover, the plasma exposure level of compound <b>27</b> was also higher than that of the previously published MEK degrader <b>3</b>.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> It is also worth noting that both <b>24</b> and <b>27</b> were very well tolerated by the studied mice and no clinical signs or adverse effects were observed. In summary, we identified an improved MEK1/2 degrader, <b>27</b>, which is suitable for in vivo efficacy studies. Compound <b>27</b> is not only more potent but also has higher in vivo exposure than the previously reported MEK1/2 degrader <b>3</b>.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0014.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Plasma concentrations of compounds <b>24</b> and <b>27</b> following a single intraperitoneal administration at 50 mg/kg for 8 h in male Swiss Albino mice. Plasma concentrations at each time point are presented as the mean ± SD from three mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Synthesis of Novel Compounds</h3><div class="NLM_p">The syntheses of compounds <b>5</b>–<b>23</b>, <b>25</b>–<b>50</b>, <b>52</b>, and <b>53</b> are depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, using synthetic routes similar to those reported previously.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Briefly, nucleophilic substitutions between commercially available 1,3-dioxolane <b>54</b> or <b>55</b> with <i>N</i>-hydroxyphthalimide afforded intermediate <b>56</b> or <b>57</b>, which was converted to alkoxyamine <b>58</b> or <b>59</b> by hydrazinolysis. Amide condensation of alkoxyamine <b>58</b> or <b>59</b> with 3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzoic acid provided alkoxyamide <b>60</b> or <b>61</b>. Removal of the dioxolane protecting group under acidic conditions unmasked the aldehyde <b>62</b> or <b>63</b>, which was transformed into the putative MEK1/2 degraders through reductive amination reaction with different VHL ligand-based linkers or CRBN ligand-based linkers, which were synthesized according to published procedures.<a onclick="showRef(event, 'ref44 ref64'); return false;" href="javascript:void(0);" class="ref ref44 ref64">(44,64)</a></div><figure id="sch1" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0015.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesizing Compounds <b>5</b>–<b>23</b>, <b>25</b>–<b>50</b>, <b>52</b>, and <b>53</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DBU, <i>N</i>,<i>N</i>-dimethylformamide (DMF), 50 °C, 4 h; (b) N<sub>2</sub>H<sub>4</sub>.H<sub>2</sub>O, MeOH, dichloromethane (DCM), rt, 2 h; (c) 3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzoic acid, HOAt, EDCI, NMM, DMSO, rt, overnight; (d) 3 M HCl, tetrahydrofuran (THF), rt, 6 h; (e) different linkers, NaBH<sub>3</sub>CN, DCM, MeOH, rt, overnight.</p></p></figure><div class="NLM_p">The syntheses of compounds <b>24</b> and <b>51</b> are outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. <i>N</i>-Alkylation of known compound <b>64</b><a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> provided the methylated intermediate <b>65</b>, which was converted to compound <b>66</b> or <b>67</b> through amide coupling and followed by deprotection. Reductive amination reaction with <b>62</b> converted <b>66</b> or <b>67</b> to compound <b>24</b> or <b>51</b>, respectively.</div><figure id="sch2" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0016.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesizing Compounds <b>24</b> and <b>51</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CH<sub>3</sub>I, NaH, DMF, rt, 18 h; (b) (1) HOAt, EDCI, NMM, DMSO, rt, overnight; (2) trifluoroacetic acid (TFA), DCM, rt, 2 h; (c) <b>62</b>, NaBH<sub>3</sub>CN, DCM, MeOH, rt, overnight.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38031" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38031" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We conducted extensive SAR studies of the PD0325901-derived MEK1/2 degraders by exploring a large set of linkers and several E3 ligase ligands. From these studies, we identified two novel, improved VHL-recruiting MEK1/2 degraders <b>24</b> and <b>27</b>, both of which potently induced MEK1/2 protein degradation in colorectal cancer cell line HT-29 and melanoma cell line SK-MEL-28 in concentration and time-dependent manners. Our MOA studies confirmed that the MEK1/2 degradation induced by these compounds was through hijacking the ubiquitin-proteasome system. Degrader <b>27</b> displayed excellent protein degradation selectivity in a global proteomic study and reduced the protein levels of only MEK1/2 out of over 5000 quantified proteins. Compared with previously reported VHL-recruiting degraders <b>3</b> and <b>4</b>, the new degraders <b>24</b> and <b>27</b> were more potent at inhibiting the growth of HT-29, SK-MEL-28, H3122, and SUDHL1 cells. In addition, compounds <b>24</b> and <b>27</b> were more potent than compound <b>3</b> at degrading MEK1/2 in HT-29 and SK-MEL-28 cells. Furthermore, degraders <b>24</b> and <b>27</b> were much more potent at inhibiting cancer cell growth than their corresponding controls that have similar binding affinities to MEK1/2 as degraders <b>24</b> and <b>27</b>, but cannot degrade MEK1/2, indicating potential advantages of pharmacological degradation of MEK1/2 over inhibition of the MEK1/2 kinase activities. Moreover, concurrent pharmacological inhibition of either BRAF or PI3K potentiated the antiproliferation potency of <b>27</b> in HT-29 or SK-MEL-28 cells, suggesting that the combination of a MEK1/2 degrader with an inhibitor of BRAF or PI3K may provide potential therapeutic benefits over the MEK1/2 degrader as a single agent. Furthermore, degrader <b>27</b> displayed good plasma exposure in mice, which is higher than that of the previously reported degrader <b>3</b>. Thus, it is suitable for in vivo studies.</div><div class="NLM_p">In addition to the two improved VHL-recruiting MEK1/2 degraders, we also identified the first CRBN-recruiting MEK1/2 degrader <b>50</b>, which effectively degraded MEK1/2 and inhibited the growth of HT-29 and SK-MEL-28 cells. Rescue experiments revealed that MEK1/2 degradation induced by <b>50</b> was dependent on the ubiquitin-proteasome system and binding to MEK1/2. Global proteomic analysis showed that degrader <b>50</b> down-regulated only MEK1/2 proteins. Subsequent Western blotting analysis identified IKZF3, but not other CBRN neo-substrates, including GSPT1, as an off-target of <b>50</b>. Further optimization is necessary to improve the potency and selectivity of this CRBN-recruiting MEK1/2 degrader.</div><div class="NLM_p last">In summary, our extensive SAR studies led to a novel, potent, and selective VHL-recruiting degrader, <b>27</b>, which is more potent and has higher plasma exposure in mice than that of previously reported MEK1/2 degraders. Compound <b>27</b> is the best MEK1/2 degrader to date for in vivo studies. In addition, the SAR studies resulted in the first CRBN-recruiting MEK1/2 degrader <b>50</b>, which potently and selectively induced MEK1/2 degradation. Our studies paved the way for further development and optimization of MEK1/2 heterobifunctional small-molecule degraders.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20071" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20071" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Chemistry General Procedures</h3><div class="NLM_p">All chemical reagents were purchased from commercial vendors and used in syntheses without further purification. A Teledyne ISCO Combi<i>Flash</i> Rf<sup>+</sup> instrument equipped with a variable wavelength UV detector and a fraction collector was used to conduct flash column chromatography. HP C18 Redi<i>Sep</i> Rf reverse-phase silica columns were also used for the purification of certain polar products. An Agilent 1200 series system with a DAD detector and a 2.1 mm × 150 mm Zorbax 300SB-C18 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min for chromatography were used to obtain high-performance liquid chromatography (HPLC) spectra for all final compounds. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). A Waters Acquity I-Class ultra-performance liquid chromatography (UPLC) system with a PDA detector was used to generate UPLC spectra for all compounds. Chromatography was performed using a 2.1 mm × 30 mm ACQUITY UPLC BEH C18 1.7 μm column with water containing 3% acetonitrile and 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.8 mL/min. The gradient program was as follows: 1–99% B (1–1.5 min) and 99–1% B (1.5–2.5 min). High-resolution mass spectra (HRMS) data were obtained in positive ion mode using an Agilent G1969A API-TOF with an electrospray ionization (ESI) source. Nuclear Magnetic Resonance (NMR) spectra were obtained on a Bruker DRX-600 spectrometer with 600 MHz for proton (<sup>1</sup>H NMR) or a Bruker DXI 800 MHz spectrometer with 800 MHz for proton (<sup>1</sup>H NMR) or 200 MHz for carbon (<sup>13</sup>C NMR); chemical shifts are reported in ppm (δ). Preparative HPLC was performed using Agilent Prep 1200 series with a UV detector set to 220 nm. Samples were injected into a Phenomenex Luna 75 mm × 30 mm, 5 μm, C18 column at room temperature. The flow rate was 40 mL/min. A linear gradient was used with 10% (or 50%) of MeOH (A) in H<sub>2</sub>O (with 0.1% TFA) (B) to 100% of MeOH (A). HPLC and UPLC were used to establish the purity of target compounds. All final compounds had >95% purity using the HPLC and UPLC methods described above.</div><div class="NLM_p">Compounds <b>5</b>–<b>53</b> were prepared according to the procedures for the synthesis of compound <b>3</b> in our previous work.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-11-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,9-dioxo-3-oxa-2,7,10-triazadodecan-12-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>5</b>)</h4><div class="NLM_p last">White solid, 40% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.30 (d, <i>J</i> = 4.2 Hz, 1H), 7.53–7.47 (m, 3H), 7.47–7.44 (m, 2H), 7.40–7.36 (m, 2H), 7.01 (td, <i>J</i> = 9.1, 6.9 Hz, 1H), 6.68 (td, <i>J</i> = 8.7, 3.8 Hz, 1H), 4.58–4.55 (m, 2H), 4.53 (d, <i>J</i> = 8.4 Hz, 2H), 4.51–4.48 (dt, <i>J</i> = 4.5, 2.5 Hz, 1H), 4.37 (d, <i>J</i> = 15.6 Hz, 1H), 4.08 (dt, <i>J</i> = 9.8, 5.0 Hz, 1H), 4.03 (dq, <i>J</i> = 9.9, 5.3 Hz, 1H), 3.98 (d, <i>J</i> = 15.8 Hz, 1H), 3.87 (d, <i>J</i> = 15.8 Hz, 1H), 3.81 (d, <i>J</i> = 11.0 Hz, 1H), 3.76 (dd, <i>J</i> = 10.9, 3.8 Hz, 1H), 2.52 (s, 3H), 2.25–2.20 (m, 1H), 2.11–2.03 (m, 4H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.14 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>40</sub>H<sub>46</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 936.2222; found: 936.2047.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-12-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,10-dioxo-3-oxa-2,7,11-triazatridecan-13-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>6</b>)</h4><div class="NLM_p last">White solid, 42% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.96 (s, 1H), 7.51–7.45 (m, 3H), 7.44–7.40 (m, 3H), 7.38 (dt, <i>J</i> = 8.3, 1.4 Hz, 1H), 7.05 (td, <i>J</i> = 9.1, 6.9 Hz, 1H), 6.66 (td, <i>J</i> = 8.7, 3.9 Hz, 1H), 4.59–4.53 (m, 3H), 4.50–4.47 (m, 1H), 4.34 (d, <i>J</i> = 15.5 Hz, 1H), 4.03 (t, <i>J</i> = 5.2 Hz, 2H), 3.90 (d, <i>J</i> = 10.9 Hz, 1H), 3.75 (dd, <i>J</i> = 10.9, 3.9 Hz, 1H), 3.35–3.32 (m, 2H), 3.27–3.22 (m, 2H), 2.78 (t, <i>J</i> = 6.6 Hz, 2H), 2.47 (s, 3H), 2.25–2.20 (m, 1H), 2.08 (ddd, <i>J</i> = 13.4, 9.3, 4.3 Hz, 1H), 2.05–1.99 (m, 2H), 1.02 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.18 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>41</sub>H<sub>48</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 950.2378; found: 950.2339.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-13-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,11-dioxo-3-oxa-2,7,12-triazatetradecan-14-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>7</b>)</h4><div class="NLM_p last">White solid, 38% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.98 (s, 1H), 7.50–7.45 (m, 3H), 7.44–7.38 (m, 3H), 7.37 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.06 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.64 (td, <i>J</i> = 8.8, 3.9 Hz, 1H), 4.58 (d, <i>J</i> = 6.8 Hz, 1H), 4.56 (dd, <i>J</i> = 3.0, 1.4 Hz, 1H), 4.53 (d, <i>J</i> = 9.7 Hz, 1H), 4.51–4.49 (m, 1H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.03 (dp, <i>J</i> = 11.8, 4.2 Hz, 2H), 3.91 (d, <i>J</i> = 10.9 Hz, 1H), 3.79 (dd, <i>J</i> = 10.9, 3.9 Hz, 1H), 3.22 (t, <i>J</i> = 5.9 Hz, 2H), 3.08 (td, <i>J</i> = 7.5, 1.8 Hz, 2H), 2.48 (s, 3H), 2.48–2.44 (m, 2H), 2.22 (ddt, <i>J</i> = 13.2, 7.6, 1.9 Hz, 1H), 2.08 (ddd, <i>J</i> = 13.3, 9.3, 4.4 Hz, 1H), 2.04–1.97 (m, 4H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.11 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>42</sub>H<sub>50</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 964.2535; found: 964.2422.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-14-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,12-dioxo-3-oxa-2,7,13-triazapentadecan-15-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>8</b>)</h4><div class="NLM_p last">White solid, 36% yield<sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.88 (s, 1H), 7.49 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.46 (d, <i>J</i> = 8.2 Hz, 2H), 7.43–7.40 (m, 2H), 7.37 (ddd, <i>J</i> = 8.5, 2.0, 1.0 Hz, 2H), 7.07 (td, <i>J</i> = 9.2, 7.1 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 3.9 Hz, 1H), 4.61 (s, 1H), 4.59–4.53 (m, 2H), 4.50 (dd, <i>J</i> = 4.2, 2.2 Hz, 1H), 4.35 (d, <i>J</i> = 15.5 Hz, 1H), 4.06–4.00 (m, 2H), 3.89 (dt, <i>J</i> = 11.1, 1.7 Hz, 1H), 3.80 (dd, <i>J</i> = 10.9, 3.9 Hz, 1H), 3.22 (t, <i>J</i> = 5.9 Hz, 2H), 3.05 (t, <i>J</i> = 7.3 Hz, 2H), 2.47 (s, 3H), 2.34 (q, <i>J</i> = 7.5 Hz, 2H), 2.22 (ddt, <i>J</i> = 13.1, 7.5, 2.0 Hz, 1H), 2.09 (ddd, <i>J</i> = 13.3, 9.2, 4.5 Hz, 1H), 2.04–1.97 (m, 2H), 1.81–1.68 (m, 4H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.13 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>43</sub>H<sub>52</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 978.2691; found: 978.2665.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-15-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,13-dioxo-3-oxa-2,7,14-triazahexadecan-16-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>9</b>)</h4><div class="NLM_p last">White solid, 28% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.93 (s, 1H), 7.51–7.45 (m, 3H), 7.42 (d, <i>J</i> = 7.8 Hz, 3H), 7.37 (d, <i>J</i> = 9.1 Hz, 1H), 7.07 (q, <i>J</i> = 8.6 Hz, 1H), 6.63 (td, <i>J</i> = 8.7, 4.0 Hz, 1H), 4.64 (s, 1H), 4.61–4.52 (m, 2H), 4.50 (s, 1H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.04 (t, <i>J</i> = 5.2 Hz, 2H), 3.91 (d, <i>J</i> = 11.0 Hz, 1H), 3.81 (dd, <i>J</i> = 11.0, 3.9 Hz, 1H), 3.23 (t, <i>J</i> = 5.9 Hz, 2H), 3.04 (t, <i>J</i> = 7.9 Hz, 2H), 2.48 (s, 3H), 2.35–2.26 (m, 2H), 2.22 (dd, <i>J</i> = 13.2, 7.8 Hz, 1H), 2.09 (ddd, <i>J</i> = 13.3, 9.1, 4.4 Hz, 1H), 2.02 (dd, <i>J</i> = 11.0, 5.3 Hz, 2H), 1.74 (p, <i>J</i> = 7.8 Hz, 2H), 1.65 (p, <i>J</i> = 7.2 Hz, 2H), 1.42 (p, <i>J</i> = 7.8 Hz, 2H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.19 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>44</sub>H<sub>54</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 992.2848; found: 992.2816.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-16-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,14-dioxo-3-oxa-2,7,15-triazaheptadecan-17-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>10</b>)</h4><div class="NLM_p last">White solid, 27% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.94 (d, <i>J</i> = 6.0 Hz, 1H), 7.51–7.45 (m, 3H), 7.44–7.39 (m, 3H), 7.38 (d, <i>J</i> = 8.8 Hz, 1H), 7.07 (q, <i>J</i> = 8.9 Hz, 1H), 6.63 (td, <i>J</i> = 8.6, 4.0 Hz, 1H), 4.67–4.63 (m, 1H), 4.58–4.55 (m, 1H), 4.53 (d, <i>J</i> = 10.4 Hz, 1H), 4.51–4.49 (m, 1H), 4.36 (d, <i>J</i> = 15.4 Hz, 1H), 4.05 (t, <i>J</i> = 5.1 Hz, 2H), 3.90 (d, <i>J</i> = 11.0 Hz, 1H), 3.81 (dd, <i>J</i> = 10.9, 4.0 Hz, 1H), 3.23 (t, <i>J</i> = 5.9 Hz, 2H), 3.10–2.99 (m, 2H), 2.48 (s, 3H), 2.31 (dt, <i>J</i> = 14.9, 7.6 Hz, 1H), 2.28–2.25 (m, 1H), 2.25–2.19 (m, 1H), 2.08 (ddd, <i>J</i> = 13.3, 9.1, 4.4 Hz, 1H), 2.05–1.99 (m, 2H), 1.73 (p, <i>J</i> = 7.8 Hz, 2H), 1.61 (dt, <i>J</i> = 13.7, 6.9 Hz, 2H), 1.42 (p, <i>J</i> = 7.1 Hz, 2H), 1.37 (q, <i>J</i> = 7.2 Hz, 2H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.19 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>45</sub>H<sub>56</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1006.3004; found: 1006.2966.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-17-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,15-dioxo-3-oxa-2,7,16-triazaoctadecan-18-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>11</b>)</h4><div class="NLM_p last">White solid, 30% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.92 (s, 1H), 7.52–7.45 (m, 3H), 7.44–7.39 (m, 3H), 7.38 (dt, <i>J</i> = 8.6, 1.3 Hz, 1H), 7.07 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.63 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 4.67–4.62 (m, 1H), 4.60–4.55 (m, 1H), 4.54 (d, <i>J</i> = 12.9 Hz, 1H), 4.51–4.48 (m, 1H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.06 (t, <i>J</i> = 5.1 Hz, 2H), 3.90 (d, <i>J</i> = 11.0 Hz, 1H), 3.81 (dd, <i>J</i> = 11.0, 3.9 Hz, 1H), 3.23 (t, <i>J</i> = 5.9 Hz, 2H), 3.09–2.97 (m, 2H), 2.48 (s, 3H), 2.32–2.27 (m, 1H), 2.26–2.20 (m, 2H), 2.08 (ddd, <i>J</i> = 13.3, 9.2, 4.5 Hz, 1H), 2.02 (dt, <i>J</i> = 10.1, 5.0 Hz, 2H), 1.72 (p, <i>J</i> = 7.7 Hz, 2H), 1.61 (dp, <i>J</i> = 14.5, 7.1 Hz, 2H), 1.43–1.38 (m, 2H), 1.38–1.28 (m, 4H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.17 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>46</sub>H<sub>58</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1020.3161; found: 1020.3086.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-18-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,16-dioxo-3-oxa-2,7,17-triazanonadecan-19-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>12</b>)</h4><div class="NLM_p last">White solid, 31% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.97 (s, 1H), 7.51–7.45 (m, 3H), 7.45–7.39 (m, 3H), 7.37 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.07 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 4.64 (s, 1H), 4.60–4.55 (m, 1H), 4.54–4.52 (m, 1H), 4.50 (dq, <i>J</i> = 4.1, 2.1 Hz, 1H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.07 (dd, <i>J</i> = 5.8, 4.5 Hz, 2H), 3.90 (dt, <i>J</i> = 11.2, 1.7 Hz, 1H), 3.81 (dd, <i>J</i> = 10.9, 3.9 Hz, 1H), 3.23 (t, <i>J</i> = 5.8 Hz, 2H), 3.10–2.90 (m, 2H), 2.48 (s, 3H), 2.33–2.26 (m, 1H), 2.25–2.18 (m, 2H), 2.08 (ddd, <i>J</i> = 13.3, 9.1, 4.5 Hz, 1H), 2.02 (dt, <i>J</i> = 10.2, 5.1 Hz, 2H), 1.72 (p, <i>J</i> = 7.8 Hz, 2H), 1.59 (q, <i>J</i> = 7.2 Hz, 2H), 1.42–1.36 (d, <i>J</i> = 7.9 Hz, 2H), 1.35–1.29 (m, 6H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.12 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>47</sub>H<sub>60</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1034.3317; found: 1034.3217.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-19-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,17-dioxo-3-oxa-2,7,18-triazaicosan-20-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>13</b>)</h4><div class="NLM_p last">White solid, 31% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.96 (s, 1H), 7.50–7.44 (m, 3H), 7.44–7.39 (m, 3H), 7.38 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.07 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 4.63 (s, 1H), 4.59–4.55 (m, 1H), 4.54 (d, <i>J</i> = 12.1 Hz, 1H), 4.50 (dd, <i>J</i> = 4.3, 2.2 Hz, 1H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.11–4.02 (m, 2H), 3.90 (d, <i>J</i> = 11.0 Hz, 1H), 3.80 (dd, <i>J</i> = 11.0, 3.9 Hz, 1H), 3.24 (t, <i>J</i> = 5.9 Hz, 2H), 3.08–2.99 (m, 2H), 2.48 (s, 3H), 2.29 (ddd, <i>J</i> = 15.2, 8.4, 7.0 Hz, 1H), 2.26–2.19 (m, 2H), 2.08 (ddd, <i>J</i> = 13.3, 9.1, 4.5 Hz, 1H), 2.02 (dt, <i>J</i> = 10.3, 5.1 Hz, 2H), 1.71 (p, <i>J</i> = 7.7 Hz, 2H), 1.65–1.53 (m, 2H), 1.42–1.36 (m, 2H), 1.34–1.25 (m, 8H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.22 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>48</sub>H<sub>62</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1048.3474; found: 1078.3372.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,18-dioxo-3-oxa-2,7,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>14</b>)</h4><div class="NLM_p last">White solid, 24% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.94 (s, 1H), 7.50–7.44 (m, 3H), 7.44–7.39 (m, 3H), 7.38 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.07 (td, <i>J</i> = 9.2, 6.9 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 4.64 (s, 1H), 4.59–4.55 (m, 1H), 4.54 (d, <i>J</i> = 12.6 Hz, 1H), 4.51–4.49 (m, 1H), 4.36 (d, <i>J</i> = 15.5 Hz, 1H), 4.13–4.03 (m, 2H), 3.90 (d, <i>J</i> = 11.0 Hz, 1H), 3.81 (dd, <i>J</i> = 11.0, 3.9 Hz, 1H), 3.24 (t, <i>J</i> = 5.8 Hz, 2H), 3.10–2.97 (m, 2H), 2.48 (s, 3H), 2.32–2.26 (m, 1H), 2.26–2.19 (m, 2H), 2.08 (ddd, <i>J</i> = 13.4, 9.2, 4.5 Hz, 1H), 2.05–1.99 (m, 2H), 1.71 (p, <i>J</i> = 7.7 Hz, 2H), 1.59 (tq, <i>J</i> = 14.3, 6.9 Hz, 2H), 1.44–1.35 (m, 2H), 1.32–1.29 (m, 6H), 1.29–1.26 (m, 4H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.26 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>49</sub>H<sub>64</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1062.3630; found: 1062.3556.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-14-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,12-dioxo-3,10-dioxa-2,7,13-triazapentadecan-15-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>15</b>)</h4><div class="NLM_p last">White solid, 33% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.98 (s, 1H), 7.50–7.44 (m, 3H), 7.42 (d, <i>J</i> = 8.3 Hz, 2H), 7.40–7.35 (m, 2H), 7.06 (q, <i>J</i> = 8.8 Hz, 1H), 6.62 (td, <i>J</i> = 8.8, 4.0 Hz, 1H), 4.67 (s, 1H), 4.59–4.55 (m, 1H), 4.54 (d, <i>J</i> = 3.3 Hz, 1H), 4.51 (d, <i>J</i> = 10.2 Hz, 1H), 4.35 (d, <i>J</i> = 15.4 Hz, 1H), 4.12 (s, 1H), 4.09 (s, 1H), 4.07–4.03 (m, 3H), 3.91–3.84 (m, 3H), 3.80 (dd, <i>J</i> = 11.0, 3.7 Hz, 1H), 3.36–3.34 (m, 1H), 3.28 (p, <i>J</i> = 1.7 Hz, 1H), 2.48 (s, 3H), 2.24 (dd, <i>J</i> = 13.1, 7.7 Hz, 1H), 2.12–2.02 (m, 4H), 1.01 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.13 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>42</sub>H<sub>50</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>7</sub>S<sup>+</sup>: 980.2484; found: 980.2361.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-15-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,13-dioxo-3,10-dioxa-2,7,14-triazahexadecan-16-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>16</b>)</h4><div class="NLM_p last">White solid, 27% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.73 (s, 1H), 7.50 (t, <i>J</i> = 6.7 Hz, 2H), 7.46 (d, <i>J</i> = 8.2 Hz, 2H), 7.43 (dd, <i>J</i> = 10.6, 2.0 Hz, 1H), 7.39 (t, <i>J</i> = 7.2 Hz, 1H), 7.34 (d, <i>J</i> = 8.1 Hz, 1H), 7.01 (d, <i>J</i> = 7.7 Hz, 1H), 6.68–6.54 (m, 1H), 4.63–4.55 (m, 2H), 4.50 (d, <i>J</i> = 14.8 Hz, 2H), 4.43 (d, <i>J</i> = 15.6 Hz, 1H), 4.04 (t, <i>J</i> = 5.4 Hz, 2H), 3.98–3.89 (m, 2H), 3.85–3.68 (m, 6H), 2.57 (s, 6H), 2.26 (t, <i>J</i> = 10.6 Hz, 1H), 2.07 (q, <i>J</i> = 6.3, 5.3 Hz, 4H), 1.01 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.15 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>43</sub>H<sub>52</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>7</sub>S<sup>+</sup>: 994.2640; found: 994.2478.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-17-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,15-dioxo-3,10,13-trioxa-2,7,16-triazaoctadecan-18-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>17</b>)</h4><div class="NLM_p last">White solid, 26% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.95 (s, 1H), 7.50–7.45 (m, 2H), 7.45–7.41 (m, 3H), 7.40–7.35 (m, 2H), 7.16–6.99 (m, 1H), 6.63 (td, <i>J</i> = 8.8, 4.0 Hz, 1H), 4.77–4.70 (m, 1H), 4.59–4.53 (m, 1H), 4.50 (d, <i>J</i> = 6.0 Hz, 1H), 4.48–4.38 (m, 2H), 4.08–4.00 (m, 3H), 3.96 (d, <i>J</i> = 15.8 Hz, 1H), 3.88 (d, <i>J</i> = 11.2 Hz, 1H), 3.81 (dt, <i>J</i> = 9.8, 3.8 Hz, 3H), 3.67–3.61 (m, 4H), 3.30–3.23 (m, 3H), 2.47 (s, 3H), 2.33–2.21 (m, 1H), 2.14–2.06 (m, 2H), 2.04 (d, <i>J</i> = 14.1 Hz, 2H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.19 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>44</sub>H<sub>54</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>8</sub>S<sup>+</sup>: 1024.2746; found: 1024.2692.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-18-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,16-dioxo-3,10,13-trioxa-2,7,17-triazanonadecan-19-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl) benzyl)pyrrolidine-2-carboxamide (<b>18</b>)</h4><div class="NLM_p last">White solid, 24% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.28 (s, 1H), 7.51–7.47 (m, 3H), 7.45 (d, <i>J</i> = 8.1 Hz, 2H), 7.41 (t, <i>J</i> = 6.9 Hz, 1H), 7.38–7.35 (m, 1H), 7.05 (q, <i>J</i> = 8.6 Hz, 1H), 6.63 (td, <i>J</i> = 8.6, 4.1 Hz, 1H), 4.64 (s, 1H), 4.58–4.53 (m, 1H), 4.52–4.49 (m, 2H), 4.40 (d, <i>J</i> = 15.6 Hz, 1H), 4.07 (t, <i>J</i> = 5.2 Hz, 2H), 3.96–3.88 (m, 1H), 3.83–3.76 (m, 3H), 3.74–3.63 (m, 3H), 3.58 (t, <i>J</i> = 4.5 Hz, 2H), 3.56–3.52 (m, 2H), 3.30–3.23 (m, 3H), 2.57–2.52 (m, 1H), 2.52 (s, 3H), 2.50–2.43 (m, 1H), 2.24 (dd, <i>J</i> = 13.2, 7.6 Hz, 1H), 2.14–2.00 (m, 3H), 1.02 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.14 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>45</sub>H<sub>56</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>8</sub>S<sup>+</sup>: 1038.2902; found: 1038.2758.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,18-dioxo-3,10,13,16-tetraoxa-2,7,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>19</b>)</h4><div class="NLM_p last">White solid, 36% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.99 (s, 1H), 7.49–7.45 (m, 3H), 7.42 (d, <i>J</i> = 8.2 Hz, 2H), 7.41–7.38 (m, 1H), 7.37 (dt, <i>J</i> = 8.4, 1.3 Hz, 1H), 7.05 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.64 (td, <i>J</i> = 8.8, 4.2 Hz, 1H), 4.68 (s, 1H), 4.56 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 4.53 (d, <i>J</i> = 11.4 Hz, 1H), 4.51–4.48 (m, 1H), 4.39 (d, <i>J</i> = 15.5 Hz, 1H), 4.14–4.00 (m, 4H), 3.88 (d, <i>J</i> = 11.1 Hz, 1H), 3.81 (dd, <i>J</i> = 10.9, 3.8 Hz, 1H), 3.77 (dd, <i>J</i> = 5.7, 4.3 Hz, 2H), 3.71–3.65 (m, 2H), 3.63–3.58 (m, 4H), 3.58–3.55 (m, 2H), 3.30–3.23 (m, 4H), 2.48 (s, 3H), 2.24 (ddt, <i>J</i> = 13.2, 7.5, 1.8 Hz, 1H), 2.08 (ddd, <i>J</i> = 13.4, 9.5, 4.3 Hz, 1H), 2.05–2.00 (m, 2H), 1.04 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.23 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>46</sub>H<sub>58</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>9</sub>S<sup>+</sup>: 1068.3008; found: 1068.2944.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-21-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,19-dioxo-3,10,13,16-tetraoxa-2,7,20-triazadocosan-22-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>20</b>)</h4><div class="NLM_p last">White solid, 18% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.02 (s, 1H), 7.51–7.44 (m, 3H), 7.44–7.39 (m, 3H), 7.38 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.18–6.99 (m, 1H), 6.64 (td, <i>J</i> = 8.7, 4.2 Hz, 1H), 4.64 (s, 1H), 4.59–4.55 (m, 1H), 4.53 (d, <i>J</i> = 15.4 Hz, 1H), 4.49 (d, <i>J</i> = 3.3 Hz, 1H), 4.37 (d, <i>J</i> = 15.5 Hz, 1H), 4.15–4.02 (m, 2H), 3.89 (d, <i>J</i> = 11.0 Hz, 1H), 3.80 (dd, <i>J</i> = 11.0, 3.9 Hz, 1H), 3.78–3.75 (m, 2H), 3.74–3.66 (m, 2H), 3.60–3.57 (dd, <i>J</i> = 5.8, 2.2 Hz, 3H), 3.56–3.53 (m, 5H), 3.30–3.25 (m, 4H), 2.61–2.52 (m, 1H), 2.51–2.39 (m, 4H), 2.27–2.19 (m, 1H), 2.09 (dt, <i>J</i> = 8.3, 4.3 Hz, 1H), 2.05 (td, <i>J</i> = 6.7, 4.5 Hz, 2H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.10 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>47</sub>H<sub>60</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>9</sub>S<sup>+</sup>: 1082.3165; found: 1082.3084.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-24-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,22-dioxo-3,10,13,16,19-pentaoxa-2,7,23-triazapentacosan-25-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>21</b>)</h4><div class="NLM_p last">White solid, 28% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.93 (s, 1H), 7.50–7.44 (m, 3H), 7.43–7.39 (m, 3H), 7.38 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.06 (td, <i>J</i> = 9.1, 6.8 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 4.0 Hz, 1H), 4.68–4.63 (m, 1H), 4.58–4.54 (m, 1H), 4.54–4.51 (m, 1H), 4.51–4.48 (m, 1H), 4.39–4.33 (m, 1H), 4.11–4.03 (m, 2H), 3.93–3.86 (m, 1H), 3.83–3.76 (m, 3H), 3.75–3.67 (m, 2H), 3.63–3.58 (m, 6H), 3.57 (dq, <i>J</i> = 5.7, 3.2, 2.6 Hz, 6H), 3.30–3.25 (m, 4H), 2.57 (ddd, <i>J</i> = 15.2, 7.2, 5.2 Hz, 1H), 2.50–2.41 (m, 4H), 2.26–2.19 (m, 1H), 2.09 (dt, <i>J</i> = 8.4, 4.2 Hz, 1H), 2.07–2.02 (m, 2H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.16 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>49</sub>H<sub>64</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>10</sub>S<sup>+</sup>: 1126.3427; found: 1126.3422.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-19-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,17-dioxo-3-oxa-2,7,18-triazaicosan-20-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>22</b>)</h4><div class="NLM_p last">White solid, 19% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.03 (s, 1H), 7.48–7.44 (m, 3H), 7.43 (d, <i>J</i> = 8.2 Hz, 2H), 7.40–7.36 (m, 2H), 7.07 (td, <i>J</i> = 9.2, 6.9 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 5.03–4.98 (m, 1H), 4.62 (s, 1H), 4.59–4.53 (m, 1H), 4.43 (td, <i>J</i> = 4.2, 2.0 Hz, 1H), 4.14–4.04 (m, 2H), 3.87 (d, <i>J</i> = 11.0 Hz, 1H), 3.75 (dd, <i>J</i> = 11.0, 4.0 Hz, 1H), 3.24 (t, <i>J</i> = 5.9 Hz, 2H), 3.12–2.98 (m, 2H), 2.49 (s, 3H), 2.33–2.27 (m, 1H), 2.24 (dd, <i>J</i> = 8.2, 6.3 Hz, 1H), 2.22–2.17 (m, 1H), 2.03 (dt, <i>J</i> = 10.3, 5.0 Hz, 2H), 1.95 (ddd, <i>J</i> = 13.3, 9.0, 4.5 Hz, 1H), 1.72 (p, <i>J</i> = 7.8 Hz, 2H), 1.65–1.55 (m, 2H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H), 1.42–1.37 (m, 2H), 1.35–1.26 (m, 8H), 1.03 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.59 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>49</sub>H<sub>64</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1062.3630; found: 1062.3769.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-21-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,19-dioxo-3-oxa-2,7,20-triazadocosan-22-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>23</b>)</h4><div class="NLM_p last">White solid, 17% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.12 (d, <i>J</i> = 1.7 Hz, 1H), 7.49–7.42 (m, 5H), 7.39 (dd, <i>J</i> = 16.6, 8.0 Hz, 2H), 7.07 (q, <i>J</i> = 8.5, 8.0 Hz, 1H), 6.70–6.57 (m, 1H), 4.64–4.60 (m, 1H), 4.59–4.54 (m, 1H), 4.45–4.41 (m, 1H), 4.15–4.03 (m, 2H), 3.88 (d, <i>J</i> = 10.9 Hz, 1H), 3.79–3.72 (m, 1H), 3.24 (t, <i>J</i> = 5.8 Hz, 2H), 3.04 (t, <i>J</i> = 7.8 Hz, 2H), 2.55–2.46 (m, 3H), 2.30 (dt, <i>J</i> = 15.6, 8.0 Hz, 1H), 2.26–2.17 (m, 2H), 2.06–2.00 (m, 3H), 1.98–1.90 (m, 1H), 1.75–1.68 (m, 2H), 1.65–1.54 (m, 2H), 1.51 (dd, <i>J</i> = 7.1, 1.7 Hz, 3H), 1.42–1.36 (m, 2H), 1.35–1.24 (m, 12H), 1.04 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.12 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>51</sub>H<sub>68</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1090.3943; found: 1090.3857.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-7-methyl-1,18-dioxo-3-oxa-2,7,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>24</b>)</h4><div class="NLM_p last">White solid, 16% yield. <sup>1</sup>H NMR (800 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.89 (s, 1H), 7.51 (d, <i>J</i> = 7.9 Hz, 2H), 7.45 (t, <i>J</i> = 10.7 Hz, 3H), 7.39 (s, 1H), 7.36 (d, <i>J</i> = 8.7 Hz, 1H), 7.06 (q, <i>J</i> = 8.3 Hz, 1H), 6.62 (td, <i>J</i> = 8.7, 3.8 Hz, 1H), 5.02 (q, <i>J</i> = 7.2 Hz, 1H), 4.55 (t, <i>J</i> = 7.4 Hz, 1H), 4.49 (s, 1H), 4.47–4.43 (m, 1H), 4.11 (t, <i>J</i> = 7.6 Hz, 1H), 4.06–4.01 (m, 1H), 3.93 (dd, <i>J</i> = 10.8, 5.0 Hz, 1H), 3.68 (dd, <i>J</i> = 11.1, 3.6 Hz, 1H), 3.45 (q, <i>J</i> = 7.7, 6.6 Hz, 1H), 3.04 (td, <i>J</i> = 11.9, 5.0 Hz, 1H), 2.89 (s, 3H), 2.49 (s, 3H), 2.27 (dt, <i>J</i> = 14.7, 7.9 Hz, 1H), 2.20 (ddd, <i>J</i> = 13.7, 9.0, 4.8 Hz, 2H), 2.10 (dt, <i>J</i> = 12.8, 5.8 Hz, 3H), 1.80–1.67 (m, 2H), 1.59–1.52 (m, 2H), 1.45 (d, <i>J</i> = 7.1 Hz, 3H), 1.36–1.18 (m, 14H), 1.05 (s, 9H). <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i><sub>4</sub>) δ 175.8, 173.0, 172.1, 168.2, 155.3, 153.6, 153.0, 148.4, 145.6, 144.2, 134.5, 133.6, 132.4, 131.1, 130.3 (2 × C), 127.5 (2 × C), 127.3, 125.6, 125.3, 125.1, 120.8, 120.2, 111.6, 82.3, 77.3, 70.8, 68.8, 60.4, 58.8, 57.8, 57.6, 57.1, 49.9, 40.3, 38.6, 36.5, 36.3, 30.3, 30.2, 30.1, 29.9, 27.3, 26.9 (2 × C), 26.8, 25.1, 23.9, 22.2, 15.4. HPLC > 99%, <i>t</i><sub>R</sub> = 4.62 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>51</sub>H<sub>68</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1090.3943; found: 1090.3970.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,18-dioxo-3-oxa-2,8,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>25</b>)</h4><div class="NLM_p last">White solid, 22% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.03 (s, 1H), 7.52–7.42 (m, 5H), 7.41–7.33 (m, 2H), 7.06 (q, <i>J</i> = 8.7 Hz, 1H), 6.61 (td, <i>J</i> = 8.8, 4.3 Hz, 1H), 5.00 (q, <i>J</i> = 6.8 Hz, 1H), 4.63 (s, 1H), 4.57 (t, <i>J</i> = 8.3 Hz, 1H), 4.43 (s, 1H), 3.93 (t, <i>J</i> = 5.5 Hz, 2H), 3.87 (d, <i>J</i> = 11.0 Hz, 1H), 3.75 (dd, <i>J</i> = 11.0, 3.9 Hz, 1H), 3.10 (t, <i>J</i> = 7.4 Hz, 2H), 3.05–2.95 (m, 2H), 2.50 (s, 3H), 2.30 (dt, <i>J</i> = 15.0, 7.7 Hz, 1H), 2.25 (dd, <i>J</i> = 8.1, 6.3 Hz, 1H), 2.22–2.17 (m, 1H), 1.95 (ddd, <i>J</i> = 13.3, 9.0, 4.6 Hz, 1H), 1.88 (p, <i>J</i> = 7.0 Hz, 2H), 1.76 (q, <i>J</i> = 6.1 Hz, 2H), 1.68 (p, <i>J</i> = 7.6 Hz, 2H), 1.63–1.55 (m, 2H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H), 1.42–1.28 (m, 10H), 1.04 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.56 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>50</sub>H<sub>66</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1076.3787; found: 1076.3827.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-21-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,19-dioxo-3-oxa-2,8,20-triazadocosan-22-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>26</b>)</h4><div class="NLM_p last">White solid, 23% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.93 (s, 1H), 7.50–7.41 (m, 5H), 7.37 (dd, <i>J</i> = 8.5, 6.6 Hz, 2H), 7.06 (q, <i>J</i> = 8.8 Hz, 1H), 6.62 (td, <i>J</i> = 8.7, 4.2 Hz, 1H), 5.02–4.98 (m, 1H), 4.65–4.61 (m, 1H), 4.58–4.55 (m, 1H), 4.45–4.42 (m, 1H), 3.93 (t, <i>J</i> = 5.6 Hz, 2H), 3.87 (d, <i>J</i> = 11.1 Hz, 1H), 3.75 (dd, <i>J</i> = 11.0, 4.0 Hz, 1H), 3.10 (t, <i>J</i> = 7.3 Hz, 2H), 3.04–2.95 (m, 2H), 2.48 (s, 3H), 2.30 (dt, <i>J</i> = 15.0, 7.6 Hz, 1H), 2.26–2.23 (m, 1H), 2.22–2.17 (m, 1H), 1.95 (ddd, <i>J</i> = 13.4, 9.1, 4.6 Hz, 1H), 1.88 (t, <i>J</i> = 7.1 Hz, 2H), 1.76 (q, <i>J</i> = 6.1 Hz, 2H), 1.68 (p, <i>J</i> = 7.6 Hz, 2H), 1.63–1.56 (m, 2H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H), 1.39–1.35 (m, 2H), 1.34–1.27 (m, 10H), 1.04 (s, 9H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.63 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>51</sub>H<sub>68</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>:1090.3943; found, 1090.3991.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-22-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,20-dioxo-3-oxa-2,8,21-triazatricosan-23-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>27</b>)</h4><div class="NLM_p last">White solid, 25% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.00 (s, 1H), 7.47–7.41 (m, 5H), 7.39 (td, <i>J</i> = 6.4, 3.0 Hz, 1H), 7.37–7.34 (m, 1H), 7.05 (td, <i>J</i> = 9.1, 6.9 Hz, 1H), 6.61 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 5.00 (q, <i>J</i> = 7.0 Hz, 1H), 4.62 (s, 1H), 4.57 (dd, <i>J</i> = 9.1, 7.5 Hz, 1H), 4.44 (dp, <i>J</i> = 4.3, 2.0 Hz, 1H), 3.93 (t, <i>J</i> = 5.6 Hz, 2H), 3.88 (dt, <i>J</i> = 11.3, 1.8 Hz, 1H), 3.75 (dd, <i>J</i> = 11.0, 4.0 Hz, 1H), 3.10 (t, <i>J</i> = 7.3 Hz, 2H), 3.02–2.95 (m, 2H), 2.49 (s, 3H), 2.33–2.27 (m, 1H), 2.26–2.23 (m, 1H), 2.23–2.17 (m, 1H), 1.96 (ddd, <i>J</i> = 13.3, 9.0, 4.5 Hz, 1H), 1.88 (p, <i>J</i> = 7.1 Hz, 2H), 1.75 (h, <i>J</i> = 5.5, 5.0 Hz, 2H), 1.68 (p, <i>J</i> = 7.9 Hz, 2H), 1.63–1.56 (m, 2H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H), 1.41–1.25 (m, 14H), 1.04 (s, 9H); <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i><sub>4</sub>) δ 176.2, 173.3, 172.0, 167.2, 155.0, 153.4, 152.7, 148.8, 145.2, 134.4, 133.2, 132.4, 131.1, 130.5, 130.2 (2 × C), 128.1, 127.6 (2 × C), 125.4, 125.1, 125.0, 121.4, 120.6, 111.5, 82.1, 77.0, 70.3, 68.9, 60.5, 59.3, 56.7, 49.7, 41.1, 38.8, 36.4, 35.5, 30.4, 30.3 (2 × C), 30.2, 30.1, 30.0, 27.4, 27.0, 26.8 (2 × C), 26.7, 26.1, 24.4, 22.4, 15.7. HPLC > 99%, <i>t</i><sub>R</sub> = 4.61 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>52</sub>H<sub>70</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1104.4100; found: 1104.3876.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-(3-((2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>28</b>)</h4><div class="NLM_p last">White solid, 20% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.54 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.37–7.34 (m, 1H), 7.32 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.29 (dt, <i>J</i> = 8.6, 1.2 Hz, 1H), 7.09 (d, <i>J</i> = 8.5 Hz, 1H), 7.06–7.00 (m, 2H), 6.58 (td, <i>J</i> = 8.7, 4.6 Hz, 1H), 4.95–4.91 (m, 1H), 4.19–4.02 (m, 2H), 3.77 (td, <i>J</i> = 5.7, 2.3 Hz, 2H), 3.37 (t, <i>J</i> = 5.8 Hz, 2H), 3.33 (t, <i>J</i> = 5.9 Hz, 2H), 2.79 (ddd, <i>J</i> = 17.5, 13.9, 5.4 Hz, 1H), 2.67 (ddd, <i>J</i> = 17.5, 4.6, 2.6 Hz, 1H), 2.52 (qd, <i>J</i> = 13.1, 4.5 Hz, 1H), 2.11–2.00 (m, 2H), 1.94 (dtd, <i>J</i> = 13.1, 5.3, 2.5 Hz, 1H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.17 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>29</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 765.1140; found: 765.1210.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-(3-((3-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)propyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>29</b>)</h4><div class="NLM_p last">White solid, 21% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.50 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.40–7.37 (m, 1H), 7.35 (dd, <i>J</i> = 10.8, 1.9 Hz, 1H), 7.27 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.06 (dd, <i>J</i> = 9.3, 7.0 Hz, 1H), 7.03 (d, <i>J</i> = 2.4 Hz, 1H), 7.02 (d, <i>J</i> = 3.9 Hz, 1H), 6.56 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 5.01 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.08 (dd, <i>J</i> = 5.7, 4.4 Hz, 2H), 3.49 (t, <i>J</i> = 6.5 Hz, 2H), 3.27 (t, <i>J</i> = 5.7 Hz, 2H), 3.21 (t, <i>J</i> = 7.2 Hz, 2H), 2.84 (ddd, <i>J</i> = 17.5, 13.9, 5.3 Hz, 1H), 2.73 (ddd, <i>J</i> = 17.4, 4.4, 2.6 Hz, 1H), 2.66 (qd, <i>J</i> = 13.1, 4.5 Hz, 1H), 2.13–2.07 (m, 2H), 2.06–1.99 (m, 3H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.16 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>31</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 779.1296; found: 779.1603.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-(3-((4-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)butyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>30</b>)</h4><div class="NLM_p last">White solid, 24% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.52 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.40 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.38–7.35 (m, 1H), 7.27 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.09–7.03 (m, 1H), 7.02 (d, <i>J</i> = 7.1 Hz, 1H), 6.99 (d, <i>J</i> = 8.5 Hz, 1H), 6.60 (td, <i>J</i> = 8.7, 4.6 Hz, 1H), 5.01–4.93 (m, 1H), 4.17–4.01 (m, 2H), 3.35 (td, <i>J</i> = 6.6, 1.6 Hz, 2H), 3.25 (t, <i>J</i> = 5.8 Hz, 2H), 3.13 (t, <i>J</i> = 7.4 Hz, 2H), 2.82 (ddd, <i>J</i> = 17.5, 13.9, 5.3 Hz, 1H), 2.72 (ddd, <i>J</i> = 17.5, 4.5, 2.6 Hz, 1H), 2.63 (qd, <i>J</i> = 13.1, 4.5 Hz, 1H), 2.08–1.97 (m, 3H), 1.89 (p, <i>J</i> = 7.3 Hz, 2H), 1.83–1.75 (m, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.20 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>33</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 793.1453; found: 793.1498.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-(3-((5-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)pentyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>31</b>)</h4><div class="NLM_p last">White solid, 23% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.51 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.42 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.40–7.37 (m, 1H), 7.31 (dt, <i>J</i> = 8.4, 1.3 Hz, 1H), 7.07–7.00 (m, 2H), 6.98 (d, <i>J</i> = 8.6 Hz, 1H), 6.60 (td, <i>J</i> = 8.8, 4.6 Hz, 1H), 5.00 (dd, <i>J</i> = 12.7, 5.5 Hz, 1H), 4.08 (t, <i>J</i> = 5.3 Hz, 2H), 3.32–3.28 (m, 2H), 3.25 (t, <i>J</i> = 5.7 Hz, 2H), 3.09 (t, <i>J</i> = 7.6 Hz, 2H), 2.84 (ddd, <i>J</i> = 17.4, 13.8, 5.3 Hz, 1H), 2.78–2.71 (m, 1H), 2.70–2.62 (m, 1H), 2.09–1.95 (m, 3H), 1.82 (p, <i>J</i> = 7.8 Hz, 2H), 1.71 (p, <i>J</i> = 7.0 Hz, 2H), 1.56 (tt, <i>J</i> = 10.0, 6.3 Hz, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.19 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>35</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 807.1609; found: 807.1524.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-(3-((6-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>32</b>)</h4><div class="NLM_p last">White solid, 27% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.52 (dd, <i>J</i> = 8.6, 7.0 Hz, 1H), 7.43 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.39 (ddd, <i>J</i> = 8.9, 5.3, 1.7 Hz, 1H), 7.34 (dt, <i>J</i> = 8.4, 1.3 Hz, 1H), 7.05 (dd, <i>J</i> = 9.3, 7.1 Hz, 1H), 7.03 (d, <i>J</i> = 7.0 Hz, 1H), 7.00 (d, <i>J</i> = 8.5 Hz, 1H), 6.61 (td, <i>J</i> = 8.7, 4.4 Hz, 1H), 5.03 (dd, <i>J</i> = 12.7, 5.4 Hz, 1H), 4.08 (t, <i>J</i> = 5.1 Hz, 2H), 3.30–3.28 (m, 2H), 3.24 (t, <i>J</i> = 5.8 Hz, 2H), 3.10–3.02 (m, 2H), 2.88–2.80 (m, 1H), 2.73 (ddd, <i>J</i> = 16.2, 4.0, 2.1 Hz, 1H), 2.67 (td, <i>J</i> = 13.3, 4.4 Hz, 1H), 2.10–2.05 (m, 1H), 2.05–2.00 (m, 2H), 1.76 (q, <i>J</i> = 7.4 Hz, 2H), 1.64 (q, <i>J</i> = 6.9 Hz, 2H), 1.47 (t, <i>J</i> = 3.7 Hz, 4H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.22 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>37</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 821.1766; found: 821.1712.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-(3-((7-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)heptyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>33</b>)</h4><div class="NLM_p last">White solid, 33% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.52 (dd, <i>J</i> = 8.6, 7.0 Hz, 1H), 7.44 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.39 (ddd, <i>J</i> = 8.7, 5.3, 1.7 Hz, 1H), 7.35 (dt, <i>J</i> = 8.6, 1.4 Hz, 1H), 7.07–7.01 (m, 2H), 7.00 (d, <i>J</i> = 8.5 Hz, 1H), 6.62 (td, <i>J</i> = 8.7, 4.4 Hz, 1H), 5.03 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.20–3.95 (m, 2H), 3.29 (t, <i>J</i> = 7.1 Hz, 2H), 3.23 (t, <i>J</i> = 5.8 Hz, 2H), 3.07–3.02 (m, 2H), 2.85 (ddd, <i>J</i> = 17.5, 14.0, 5.4 Hz, 1H), 2.76–2.71 (m, 1H), 2.71–2.66 (m, 1H), 2.12–2.06 (m, 1H), 2.03 (td, <i>J</i> = 6.0, 4.4 Hz, 2H), 1.74 (p, <i>J</i> = 7.6 Hz, 2H), 1.65 (p, <i>J</i> = 6.9 Hz, 2H), 1.51–1.35 (m, 6H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.27 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>39</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 835.1922; found: 835.1902.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-(3-((8-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)octyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>34</b>)</h4><div class="NLM_p last">White solid, 34% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.53 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.45 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.40 (dd, <i>J</i> = 8.6, 5.5 Hz, 1H), 7.36 (dt, <i>J</i> = 8.4, 1.3 Hz, 1H), 7.05 (dd, <i>J</i> = 9.2, 7.1 Hz, 1H), 7.03 (d, <i>J</i> = 7.1 Hz, 1H), 7.01 (d, <i>J</i> = 8.5 Hz, 1H), 6.63 (td, <i>J</i> = 8.7, 4.4 Hz, 1H), 5.04 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.08 (dd, <i>J</i> = 5.8, 4.5 Hz, 2H), 3.30–3.28 (m, 2H), 3.23 (t, <i>J</i> = 5.8 Hz, 2H), 3.08–3.00 (m, 2H), 2.85 (ddd, <i>J</i> = 17.5, 13.9, 5.3 Hz, 1H), 2.78–2.60 (m, 2H), 2.09 (dtd, <i>J</i> = 10.2, 5.7, 2.5 Hz, 1H), 2.05–1.99 (m, 2H), 1.72 (p, <i>J</i> = 7.6 Hz, 2H), 1.64 (p, <i>J</i> = 7.0 Hz, 2H), 1.41 (d, <i>J</i> = 5.4 Hz, 4H), 1.38–1.33 (m, 4H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.31 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>41</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 849.2079; found: 849.2108.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-(3-((2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)amino) propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>35</b>)</h4><div class="NLM_p last">White solid, 28% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.46 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.38 (dd, <i>J</i> = 10.7, 2.0 Hz, 1H), 7.35 (dd, <i>J</i> = 7.7, 2.3 Hz, 1H), 7.26 (dt, <i>J</i> = 8.3, 1.3 Hz, 1H), 6.96 (d, <i>J</i> = 7.0 Hz, 1H), 6.95–6.88 (m, 2H), 6.57 (td, <i>J</i> = 8.8, 4.7 Hz, 1H), 4.98–4.93 (m, 1H), 4.15–4.07 (m, 2H), 3.93–3.83 (m, 2H), 3.77 (ddd, <i>J</i> = 6.0, 4.5, 1.9 Hz, 2H), 3.45 (dd, <i>J</i> = 5.7, 4.2 Hz, 2H), 3.34 (t, <i>J</i> = 5.2 Hz, 4H), 2.82 (ddd, <i>J</i> = 17.5, 13.8, 5.3 Hz, 1H), 2.71 (ddd, <i>J</i> = 17.5, 4.5, 2.7 Hz, 1H), 2.60 (qd, <i>J</i> = 13.1, 4.5 Hz, 1H), 2.09–2.03 (m, 2H), 2.00 (dtd, <i>J</i> = 13.1, 5.3, 3.0 Hz, 1H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.19 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>33</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>7</sub><sup>+</sup>: 809.1402; found: 809.1599.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-(3-((2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethoxy) ethyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>36</b>)</h4><div class="NLM_p last">White solid, 27% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.54 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.44 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.39–7.31 (m, 2H), 7.09–6.97 (m, 3H), 6.63 (td, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.03 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.04 (t, <i>J</i> = 5.0 Hz, 2H), 3.81 (t, <i>J</i> = 5.0 Hz, 2H), 3.67 (t, <i>J</i> = 5.1 Hz, 2H), 3.66–3.58 (m, 4H), 3.45 (dd, <i>J</i> = 5.8, 4.5 Hz, 2H), 3.27 (t, <i>J</i> = 6.0 Hz, 2H), 3.24 (t, <i>J</i> = 5.1 Hz, 2H), 2.83 (ddd, <i>J</i> = 17.5, 13.9, 5.3 Hz, 1H), 2.76–2.56 (m, 2H), 2.12–2.04 (m, 1H), 2.01 (dt, <i>J</i> = 10.5, 5.2 Hz, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.18 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>37</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>8</sub><sup>+</sup>: 853.1664; found: 853.1614.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-((1-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)-3,6,9-trioxa-12-azapenta decan-15-yl)oxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>37</b>)</h4><div class="NLM_p last">White solid, 28% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.54 (dd, <i>J</i> = 8.5, 7.0 Hz, 1H), 7.46 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.40–7.31 (m, 2H), 7.04 (dt, <i>J</i> = 15.9, 8.7 Hz, 3H), 6.63 (td, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.04 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.03 (t, <i>J</i> = 5.1 Hz, 2H), 3.78–3.74 (m, 2H), 3.70 (t, <i>J</i> = 5.2 Hz, 2H), 3.65–3.60 (m, 4H), 3.60 (s, 4H), 3.48 (t, <i>J</i> = 5.1 Hz, 2H), 3.26 (t, <i>J</i> = 6.0 Hz, 2H), 3.24 (t, <i>J</i> = 5.0 Hz, 2H), 2.84 (ddd, <i>J</i> = 17.6, 14.0, 5.3 Hz, 1H), 2.73 (ddd, <i>J</i> = 17.5, 4.3, 2.6 Hz, 1H), 2.71–2.65 (m, 1H), 2.17–2.05 (m, 1H), 2.04–1.97 (m, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.18 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>41</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>9</sub><sup>+</sup>: 897.1926; found: 897.2312.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-((1-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)-3,6,9,12-tetraoxa-15-azaoctadecan-18-yl)oxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>38</b>)</h4><div class="NLM_p last">White solid, 27% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.54 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.47 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.43–7.31 (m, 2H), 7.10–6.98 (m, 3H), 6.64 (td, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.04 (dd, <i>J</i> = 12.7, 5.6 Hz, 1H), 4.04 (t, <i>J</i> = 5.2 Hz, 2H), 3.80–3.74 (m, 2H), 3.71 (t, <i>J</i> = 5.2 Hz, 2H), 3.67–3.62 (m, 4H), 3.60 (dt, <i>J</i> = 3.8, 2.6 Hz, 2H), 3.59–3.54 (m, 6H), 3.49 (t, <i>J</i> = 5.2 Hz, 2H), 3.28 (t, <i>J</i> = 6.0 Hz, 2H), 3.25 (t, <i>J</i> = 5.0 Hz, 2H), 2.85 (ddd, <i>J</i> = 17.6, 14.0, 5.3 Hz, 1H), 2.76–2.72 (m, 1H), 2.69 (td, <i>J</i> = 13.3, 4.5 Hz, 1H), 2.09 (ddt, <i>J</i> = 13.1, 5.5, 2.8 Hz, 1H), 2.02 (dt, <i>J</i> = 10.5, 5.2 Hz, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.21 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>39</sub>H<sub>45</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>10</sub><sup>+</sup>: 941.2188; found: 941.2169.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-((1-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)-3,6,9,12,15-pentaoxa-18-azahenicosan-21-yl)oxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>39</b>)</h4><div class="NLM_p last">White solid, 38% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.54 (dd, <i>J</i> = 8.5, 7.1 Hz, 1H), 7.47 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.40 (ddd, <i>J</i> = 9.0, 5.3, 1.7 Hz, 1H), 7.37 (ddd, <i>J</i> = 8.5, 2.0, 1.0 Hz, 1H), 7.11–6.99 (m, 3H), 6.64 (td, <i>J</i> = 8.7, 4.2 Hz, 1H), 5.04 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.05 (t, <i>J</i> = 5.2 Hz, 2H), 3.79–3.75 (m, 2H), 3.71 (t, <i>J</i> = 5.2 Hz, 2H), 3.65 (s, 4H), 3.64–3.61 (m, 2H), 3.60–3.58 (m, 2H), 3.58–3.56 (m, 4H), 3.54 (dd, <i>J</i> = 6.0, 2.7 Hz, 3H), 3.48 (t, <i>J</i> = 5.2 Hz, 2H), 3.29 (t, <i>J</i> = 6.1 Hz, 2H), 3.26 (t, <i>J</i> = 5.1 Hz, 2H), 2.85 (ddd, <i>J</i> = 17.6, 14.0, 5.3 Hz, 1H), 2.75 (dd, <i>J</i> = 4.4, 2.6 Hz, 1H), 2.74–2.70 (m, 1H), 2.70–2.66 (m, 1H), 2.15–2.07 (m, 1H), 2.06–1.99 (m, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.21 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>41</sub>H<sub>49</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>11</sub><sup>+</sup>: 985.2451; found: 985.2562.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-(3-((3-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)propyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>40</b>)</h4><div class="NLM_p last">White solid, 28% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.50 (d, <i>J</i> = 8.3 Hz, 1H), 7.36 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.31 (d, <i>J</i> = 6.6 Hz, 1H), 7.30 (dt, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.04 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.94 (d, <i>J</i> = 2.2 Hz, 1H), 6.81 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.58 (td, <i>J</i> = 8.7, 4.2 Hz, 1H), 5.02 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.13–4.02 (m, 2H), 3.36 (t, <i>J</i> = 6.9 Hz, 2H), 3.27 (t, <i>J</i> = 5.7 Hz, 2H), 3.22 (t, <i>J</i> = 7.3 Hz, 2H), 2.85 (ddd, <i>J</i> = 17.4, 13.9, 5.4 Hz, 1H), 2.73 (ddd, <i>J</i> = 17.4, 4.5, 2.6 Hz, 1H), 2.66 (qd, <i>J</i> = 13.1, 4.5 Hz, 1H), 2.13–2.07 (m, 2H), 2.07–2.02 (m, 3H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.13 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>31</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 779.1296; found: 779.1390.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>N</i>-(3-((4-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)butyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>41</b>)</h4><div class="NLM_p last">White solid, 20% yield. <sup>1</sup>H NMR (600 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.88 (s, 1H), 9.34 (s, 1H), 8.80 (s, 1H), 7.65–7.57 (m, 1H), 7.54 (d, <i>J</i> = 8.4 Hz, 1H), 7.47 (dd, <i>J</i> = 10.7, 2.0 Hz, 1H), 7.40 (dd, <i>J</i> = 8.5, 1.9 Hz, 1H), 7.08 (td, <i>J</i> = 9.3, 7.1 Hz, 1H), 6.98 (d, <i>J</i> = 2.1 Hz, 1H), 6.90 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.76 (td, <i>J</i> = 8.8, 5.2 Hz, 1H), 5.04 (dd, <i>J</i> = 12.6, 5.4 Hz, 1H), 4.19 (t, <i>J</i> = 5.2 Hz, 2H), 3.40 (s, 3H), 3.32 (t, <i>J</i> = 6.8 Hz, 3H), 3.00–2.90 (m, 1H), 2.85–2.67 (m, 2H), 2.20–2.11 (m, 3H), 2.04–1.98 (m, 2H), 1.84 (p, <i>J</i> = 7.0 Hz, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.21 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>33</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 793.1453; found: 793.1475.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>N</i>-(3-((5-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)pentyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>42</b>)</h4><div class="NLM_p last">White solid, 17% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.41 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.39–7.36 (m, 1H), 7.35–7.31 (m, 1H), 7.05 (q, <i>J</i> = 8.7 Hz, 1H), 6.93 (d, <i>J</i> = 2.1 Hz, 1H), 6.81 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.61 (td, <i>J</i> = 8.7, 4.2 Hz, 1H), 5.03 (dd, <i>J</i> = 12.6, 5.4 Hz, 1H), 4.07 (t, <i>J</i> = 5.1 Hz, 2H), 3.24 (t, <i>J</i> = 5.8 Hz, 2H), 3.20 (t, <i>J</i> = 7.0 Hz, 2H), 3.10–3.03 (m, 2H), 2.91–2.81 (m, 1H), 2.78–2.71 (m, 1H), 2.68 (td, <i>J</i> = 13.3, 4.4 Hz, 1H), 2.12–2.06 (m, 1H), 2.05–1.99 (m, 2H), 1.81 (p, <i>J</i> = 7.9 Hz, 2H), 1.70 (p, <i>J</i> = 7.1 Hz, 2H), 1.54 (p, <i>J</i> = 7.7, 7.3 Hz, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.29 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>35</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 807.1609; found: 807.1607.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-(3-((6-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)hexyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>43</b>)</h4><div class="NLM_p last">White solid, 12% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.43 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.39 (t, <i>J</i> = 7.1 Hz, 1H), 7.35 (dd, <i>J</i> = 8.5, 1.5 Hz, 1H), 7.05 (q, <i>J</i> = 8.7 Hz, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.81 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.62 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 5.08–4.99 (m, 1H), 4.07 (t, <i>J</i> = 5.1 Hz, 2H), 3.24 (t, <i>J</i> = 5.8 Hz, 2H), 3.19 (t, <i>J</i> = 7.1 Hz, 2H), 3.09–3.02 (m, 2H), 2.86 (ddd, <i>J</i> = 22.7, 12.4, 5.4 Hz, 1H), 2.78–2.71 (m, 1H), 2.68 (td, <i>J</i> = 13.3, 4.5 Hz, 1H), 2.13–2.05 (m, 1H), 2.06–1.99 (m, 2H), 1.80–1.73 (m, 2H), 1.65 (t, <i>J</i> = 7.0 Hz, 2H), 1.55–1.42 (m, 4H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.42 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>37</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 821.1766; found: 821.1807.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-(3-((7-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)heptyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>44</b>)</h4><div class="NLM_p last">White solid, 15% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.53 (d, <i>J</i> = 8.3 Hz, 1H), 7.43 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.41–7.38 (m, 1H), 7.35 (dt, <i>J</i> = 8.4, 1.3 Hz, 1H), 7.05 (td, <i>J</i> = 9.2, 7.0 Hz, 1H), 6.93 (d, <i>J</i> = 2.2 Hz, 1H), 6.80 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 6.62 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 5.03 (dd, <i>J</i> = 12.8, 5.5 Hz, 1H), 4.07 (t, <i>J</i> = 5.1 Hz, 2H), 3.23 (t, <i>J</i> = 5.8 Hz, 2H), 3.18 (t, <i>J</i> = 7.1 Hz, 2H), 3.08–3.02 (m, 2H), 2.85 (ddd, <i>J</i> = 17.7, 14.0, 5.3 Hz, 1H), 2.77–2.71 (m, 1H), 2.68 (td, <i>J</i> = 13.3, 4.4 Hz, 1H), 2.08 (dtd, <i>J</i> = 14.9, 5.2, 4.3, 2.5 Hz, 1H), 2.03 (dt, <i>J</i> = 6.5, 3.1 Hz, 2H), 1.74 (q, <i>J</i> = 7.7 Hz, 2H), 1.64 (q, <i>J</i> = 7.1 Hz, 2H), 1.47–1.35 (m, 6H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.61 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>39</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 835.1922; found: 835.1927.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>N</i>-(3-((8-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)octyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>45</b>)</h4><div class="NLM_p last">White solid, 18% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.54 (d, <i>J</i> = 8.4 Hz, 1H), 7.45 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.40 (d, <i>J</i> = 7.4 Hz, 1H), 7.36 (d, <i>J</i> = 8.7 Hz, 1H), 7.09–7.02 (m, 1H), 6.94 (d, <i>J</i> = 2.2 Hz, 1H), 6.81 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.63 (q, <i>J</i> = 4.5 Hz, 1H), 5.03 (dd, <i>J</i> = 12.7, 5.6 Hz, 1H), 4.07 (t, <i>J</i> = 5.1 Hz, 2H), 3.23 (t, <i>J</i> = 5.8 Hz, 2H), 3.19 (t, <i>J</i> = 7.1 Hz, 2H), 3.04 (t, <i>J</i> = 7.9 Hz, 2H), 2.90–2.81 (m, 1H), 2.78–2.71 (m, 1H), 2.72–2.65 (m, 1H), 2.12–2.06 (m, 1H), 2.06–2.00 (m, 2H), 1.73 (d, <i>J</i> = 7.9 Hz, 2H), 1.68–1.61 (m, 2H), 1.46–1.40 (m, 4H), 1.39–1.33 (m, 4H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.70 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>41</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>6</sub><sup>+</sup>: 849.2079; found: 849.2111.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-(3-((2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)ethoxy)ethyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>46</b>)</h4><div class="NLM_p last">White solid, 38% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.50–7.40 (m, 2H), 7.33 (dd, <i>J</i> = 8.5, 1.6 Hz, 1H), 7.29 (t, <i>J</i> = 7.2 Hz, 1H), 7.00 (q, <i>J</i> = 8.6 Hz, 1H), 6.86 (d, <i>J</i> = 2.2 Hz, 1H), 6.74 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.59 (td, <i>J</i> = 8.7, 4.4 Hz, 1H), 5.03 (dd, <i>J</i> = 12.7, 5.5 Hz, 1H), 4.06 (t, <i>J</i> = 5.0 Hz, 2H), 3.87–3.77 (m, 2H), 3.70 (t, <i>J</i> = 5.3 Hz, 2H), 3.33 (t, <i>J</i> = 5.4 Hz, 2H), 3.32–3.30 (m, 4H), 2.85 (ddd, <i>J</i> = 17.2, 13.8, 5.3 Hz, 1H), 2.77–2.72 (m, 1H), 2.68 (qd, <i>J</i> = 13.1, 4.4 Hz, 1H), 2.11–2.07 (m, 1H), 2.05 (t, <i>J</i> = 5.8 Hz, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.15 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>33</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>7</sub><sup>+</sup>: 809.1402; found: 809.1418.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i>-(3-((2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)ethoxy)ethoxy)ethyl)amino)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>47</b>)</h4><div class="NLM_p last">White solid, 27% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.54 (d, <i>J</i> = 8.4 Hz, 1H), 7.46 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.37 (t, <i>J</i> = 8.8 Hz, 2H), 7.09–7.02 (m, 1H), 7.01 (d, <i>J</i> = 2.2 Hz, 1H), 6.84 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.63 (td, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.09–5.00 (m, 1H), 4.04 (t, <i>J</i> = 5.2 Hz, 2H), 3.77 (t, <i>J</i> = 5.0 Hz, 2H), 3.69–3.56 (m, 6H), 3.38 (t, <i>J</i> = 5.4 Hz, 2H), 3.27 (t, <i>J</i> = 6.0 Hz, 2H), 3.23 (t, <i>J</i> = 5.0 Hz, 2H), 2.85 (ddd, <i>J</i> = 18.6, 14.0, 5.4 Hz, 1H), 2.77–2.71 (m, 1H), 2.69 (dd, <i>J</i> = 13.4, 4.3 Hz, 1H), 2.12–2.07 (m, 1H), 2.02 (dd, <i>J</i> = 11.5, 5.9 Hz, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.21 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>37</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>8</sub><sup>+</sup>: 853.1664; found: 853.1651.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-((1-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)-3,6,9-trioxa-12-azapenta decan-15-yl)oxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>48</b>)</h4><div class="NLM_p last">White solid, 16% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.55 (d, <i>J</i> = 8.4 Hz, 1H), 7.46 (dd, <i>J</i> = 10.7, 2.0 Hz, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.35 (dd, <i>J</i> = 8.5, 1.6 Hz, 1H), 7.07–7.02 (m, 1H), 7.01 (d, <i>J</i> = 2.1 Hz, 1H), 6.86 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 6.63 (td, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.03 (dd, <i>J</i> = 12.7, 5.5 Hz, 1H), 4.04 (t, <i>J</i> = 5.2 Hz, 2H), 3.81–3.73 (m, 2H), 3.68 (t, <i>J</i> = 5.3 Hz, 2H), 3.62 (qd, <i>J</i> = 3.4, 1.6 Hz, 4H), 3.59 (s, 4H), 3.40 (t, <i>J</i> = 5.3 Hz, 2H), 3.25 (dt, <i>J</i> = 10.1, 5.5 Hz, 4H), 2.84 (ddd, <i>J</i> = 17.1, 13.8, 5.3 Hz, 1H), 2.78–2.71 (m, 1H), 2.67 (td, <i>J</i> = 13.2, 4.3 Hz, 1H), 2.12–2.04 (m, 1H), 2.04–1.99 (m, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.27 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>41</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>9</sub><sup>+</sup>: 897.1926; found: 897.1900.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-((1-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)-3,6,9,12-tetraoxa-15-azaoctadecan-18-yl)oxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>49</b>)</h4><div class="NLM_p last">White solid, 19% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.55 (d, <i>J</i> = 8.3 Hz, 1H), 7.46 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.39 (d, <i>J</i> = 8.2 Hz, 1H), 7.36 (dt, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.09–7.02 (m, 1H), 7.02 (d, <i>J</i> = 2.1 Hz, 1H), 6.86 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.03 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.04 (t, <i>J</i> = 5.2 Hz, 2H), 3.81–3.73 (m, 2H), 3.69 (t, <i>J</i> = 5.3 Hz, 2H), 3.66–3.62 (m, 4H), 3.61–3.59 (m, 2H), 3.58–3.55 (m, 6H), 3.40 (t, <i>J</i> = 5.3 Hz, 2H), 3.26 (t, <i>J</i> = 6.1 Hz, 2H), 3.24 (t, <i>J</i> = 5.1 Hz, 2H), 2.84 (ddd, <i>J</i> = 17.7, 14.0, 5.4 Hz, 1H), 2.76–2.70 (m, 1H), 2.70–2.64 (m, 1H), 2.11–2.05 (m, 1H), 2.02 (dt, <i>J</i> = 10.5, 5.2 Hz, 2H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.35 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>39</sub>H<sub>45</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>10</sub><sup>+</sup>: 941.2188; found: 941.2445.</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>N</i>-((1-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)-3,6,9,12,15-pentaoxa-18-azahenicosan-21-yl)oxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>50</b>)</h4><div class="NLM_p last">White solid, 17% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.44 (d, <i>J</i> = 8.4 Hz, 1H), 7.35 (dd, <i>J</i> = 10.6, 2.0 Hz, 1H), 7.30 (ddd, <i>J</i> = 9.0, 5.3, 1.7 Hz, 1H), 7.27–7.24 (m, 1H), 6.94 (td, <i>J</i> = 9.2, 6.9 Hz, 1H), 6.90 (d, <i>J</i> = 2.2 Hz, 1H), 6.75 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.53 (td, <i>J</i> = 8.7, 4.2 Hz, 1H), 4.92 (dd, <i>J</i> = 12.7, 5.5 Hz, 1H), 3.95 (t, <i>J</i> = 5.2 Hz, 2H), 3.72–3.62 (m, 2H), 3.58 (t, <i>J</i> = 5.3 Hz, 2H), 3.53 (s, 4H), 3.52–3.50 (m, 2H), 3.50–3.48 (m, 2H), 3.47 (td, <i>J</i> = 4.7, 4.0, 1.6 Hz, 3H), 3.44 (dt, <i>J</i> = 4.9, 2.1 Hz, 4H), 3.28 (t, <i>J</i> = 5.3 Hz, 2H), 3.21 (p, <i>J</i> = 1.7 Hz, 1H), 3.18 (t, <i>J</i> = 6.1 Hz, 2H), 3.15 (t, <i>J</i> = 5.0 Hz, 2H), 2.74 (ddd, <i>J</i> = 17.1, 13.7, 5.2 Hz, 1H), 2.67–2.61 (m, 1H), 2.57 (td, <i>J</i> = 13.3, 4.4 Hz, 1H), 1.98 (dtd, <i>J</i> = 10.5, 5.6, 4.8, 2.9 Hz, 1H), 1.93 (dt, <i>J</i> = 10.7, 5.2 Hz, 2H). <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i><sub>4</sub>) δ 174.5, 171.5, 169.4, 169.0, 168.0, 155.9, 155.2, 153.5, 145.7, 144.0, 135.6, 134.4, 133.6, 132.3, 126.0, 125.6, 125.1, 121.0, 120.1, 118.2, 116.7, 111.4, 106.8, 82.4, 77.2, 71.4, 71.3, 71.2 (2 × C), 71.1 (3 × C), 71.0, 70.2, 66.7, 50.1, 48.0, 43.8, 32.0, 25.4, 23.6. HPLC > 99%, <i>t</i><sub>R</sub> = 4.37 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>41</sub>H<sub>49</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>11</sub><sup>+</sup>: 985.2451; found: 985.3103.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (2<i>R</i>,4<i>S</i>)-1-((<i>S</i>)-20-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-7-methyl-1,18-dioxo-3-oxa-2,7,19-triazahenicosan-21-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl) phenyl)ethyl)pyrrolidine-2-carboxamide (<b>51</b>)</h4><div class="NLM_p last">White solid, 20% yield. <sup>1</sup>H NMR (800 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.89 (s, 1H), 7.48 (d, <i>J</i> = 10.6 Hz, 1H), 7.44 (d, <i>J</i> = 8.0 Hz, 2H), 7.42 (d, <i>J</i> = 7.9 Hz, 2H), 7.38 (t, <i>J</i> = 7.3 Hz, 2H), 7.07 (q, <i>J</i> = 8.5 Hz, 1H), 6.64 (td, <i>J</i> = 8.7, 3.8 Hz, 1H), 5.00 (q, <i>J</i> = 7.2 Hz, 1H), 4.63 (s, 1H), 4.57 (t, <i>J</i> = 8.3 Hz, 1H), 4.44 (s, 1H), 4.10 (s, 1H), 4.04 (s, 1H), 3.87 (d, <i>J</i> = 11.0 Hz, 1H), 3.75 (dd, <i>J</i> = 11.2, 4.1 Hz, 1H), 3.45 (d, <i>J</i> = 6.6 Hz, 1H), 3.10–3.00 (m, 1H), 2.90 (s, 3H), 2.48 (s, 3H), 2.29 (dt, <i>J</i> = 15.0, 7.7 Hz, 1H), 2.23 (t, <i>J</i> = 7.2 Hz, 1H), 2.21–2.17 (m, 1H), 2.13–2.08 (m, 2H), 1.96 (ddd, <i>J</i> = 13.3, 8.9, 4.6 Hz, 1H), 1.77 (s, 1H), 1.73 (s, 1H), 1.62–1.55 (m, 2H), 1.51 (d, <i>J</i> = 7.1 Hz, 3H), 1.39–1.24 (m, 14H), 1.04 (s, 9H); <sup>13</sup>C NMR (201 MHz, methanol-<i>d</i><sub>4</sub>) δ 175.8, 173.0, 172.1, 168.2, 155.1, 154.9, 153.9, 153.7, 152.7, 148.8, 145.4, 134.5, 133.4, 132.4, 131.3, 130.3 (2 × C), 127.4 (2 × C), 127.3, 125.6, 125.2, 120.8, 120.2, 111.6, 82.3, 77.3, 70.8, 60.4, 58.8, 57.8, 57.6, 57.1, 49.9, 40.3, 38.6, 36.5, 36.3, 30.3 (2 × C), 30.1, 29.9, 27.3, 26.9 (3 × C), 26.8, 25.1, 23.9, 22.2, 15.6. HPLC > 99%, <i>t</i><sub>R</sub> = 4.75 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>51</sub>H<sub>68</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1090.3943; found: 1090.4114.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> (2<i>R</i>,4<i>S</i>)-1-((<i>S</i>)-22-(<i>tert</i>-Butyl)-1-(3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)phenyl)-1,20-dioxo-3-oxa-2,8,21-triazatricosan-23-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl) ethyl)pyrrolidine-2-carboxamide (<b>52</b>)</h4><div class="NLM_p last">White solid, 22% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.01 (s, 1H), 7.52 (d, <i>J</i> = 8.0 Hz, 2H), 7.47–7.42 (m, 3H), 7.40–7.37 (m, 1H), 7.35 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 7.13–6.98 (m, 1H), 6.61 (td, <i>J</i> = 8.7, 4.3 Hz, 1H), 5.02 (q, <i>J</i> = 7.0 Hz, 1H), 4.55 (dd, <i>J</i> = 8.2, 6.5 Hz, 1H), 4.49 (s, 1H), 4.45 (q, <i>J</i> = 4.6 Hz, 1H), 3.93 (q, <i>J</i> = 5.2 Hz, 3H), 3.69 (dd, <i>J</i> = 10.8, 3.5 Hz, 1H), 3.10 (t, <i>J</i> = 7.3 Hz, 2H), 3.02–2.93 (m, 2H), 2.50 (s, 3H), 2.33–2.25 (m, 1H), 2.19 (dt, <i>J</i> = 14.4, 7.9 Hz, 2H), 2.10 (dt, <i>J</i> = 12.6, 5.8 Hz, 1H), 1.88 (p, <i>J</i> = 7.2 Hz, 2H), 1.75 (p, <i>J</i> = 6.5 Hz, 2H), 1.66 (p, <i>J</i> = 7.7 Hz, 2H), 1.55 (dt, <i>J</i> = 22.0, 7.1 Hz, 2H), 1.45 (d, <i>J</i> = 7.1 Hz, 3H), 1.38–1.31 (m, 2H), 1.31–1.21 (m, 12H), 1.06 (s, 9H); <sup>13</sup>C NMR (201 MHz, methanol-<i>d</i><sub>4</sub>) δ 175.8, 173.0, 172.1, 167.2, 162.4, 154.8, 153.6, 152.7, 148.8, 145.5, 134.4, 133.2, 132.4, 131.3, 130.3 (2 × C), 127.4 (2 × C), 127.3, 125.4, 125.1, 125.0, 121.4, 120.6, 111.5, 82.1, 77.0, 70.8, 60.4, 58.8, 57.8, 49.9, 38.6, 36.5, 36.3, 30.4 (2 × C), 30.3, 30.2, 30.1, 30.0, 27.4, 27.0, 26.9 (4 × C), 26.8, 26.1, 24.4, 22.2, 15.6. HPLC > 99%, <i>t</i><sub>R</sub> = 4.84 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>52</sub>H<sub>70</sub>F<sub>3</sub>IN<sub>7</sub>O<sub>6</sub>S<sup>+</sup>: 1104.4100; found: 1104.4016.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 3,4-Difluoro-2-((2-fluoro-4-iodophenyl)amino)-<i>N</i>-((1-((2-(1-methyl-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)-3,6,9,12,15-pentaoxa-18-azahenicosan-21-yl)oxy)benzamide (<b>53</b>)</h4><div class="NLM_p last">White solid, 27% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.45 (d, <i>J</i> = 8.4 Hz, 1H), 7.36 (dd, <i>J</i> = 10.6, 1.9 Hz, 1H), 7.30 (ddd, <i>J</i> = 9.0, 5.3, 1.8 Hz, 1H), 7.26 (dt, <i>J</i> = 8.5, 1.4 Hz, 1H), 6.94 (td, <i>J</i> = 9.3, 7.0 Hz, 1H), 6.91 (d, <i>J</i> = 2.2 Hz, 1H), 6.76 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.53 (td, <i>J</i> = 8.7, 4.2 Hz, 1H), 4.95 (dd, <i>J</i> = 12.9, 5.5 Hz, 1H), 3.95 (t, <i>J</i> = 5.2 Hz, 2H), 3.65 (dd, <i>J</i> = 5.7, 4.3 Hz, 2H), 3.59 (t, <i>J</i> = 5.3 Hz, 2H), 3.54 (s, 4H), 3.52–3.51 (m, 1H), 3.49 (dd, <i>J</i> = 5.9, 2.5 Hz, 2H), 3.48–3.46 (m, 3H), 3.46–3.43 (m, 4H), 3.29 (t, <i>J</i> = 5.3 Hz, 2H), 3.21 (p, <i>J</i> = 1.6 Hz, 2H), 3.18 (t, <i>J</i> = 6.1 Hz, 2H), 3.15 (t, <i>J</i> = 5.0 Hz, 2H), 3.03 (s, 3H), 2.79–2.76 (m, 1H), 2.76 (d, <i>J</i> = 4.0 Hz, 1H), 2.62–2.48 (m, 1H), 1.97 (ddd, <i>J</i> = 9.9, 5.0, 2.9 Hz, 1H), 1.95–1.91 (m, 2H). <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i><sub>4</sub>) δ 173.5, 171.3, 169.4, 169.1, 168.0, 155.9, 155.2, 153.6, 145.7, 144.0, 135.7, 134.5, 133.6, 132.3, 126.0, 125.6, 125.2, 121.0, 120.1, 118.2, 116.7, 111.4, 106.8, 82.4, 77.2, 71.4, 71.4, 71.3, 71.2 (2 × C), 71.1 (2 × C), 71.1, 70.2, 66.8, 50.8, 48.1, 43.9, 32.3, 27.2, 25.4, 22.9. HPLC > 99%, <i>t</i><sub>R</sub> = 4.42 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>42</sub>H<sub>51</sub>F<sub>3</sub>IN<sub>6</sub>O<sub>11</sub><sup>+</sup>: 999.2607; found: 999.4250.</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>N</i>-(3-(1,3-Dioxolan-2-yl)propoxy)-3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzamide (<b>61</b>)</h4><div class="NLM_p last">Compound <b>61</b> was prepared according to the previously published procedures for the synthesis of compound <b>60</b>.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> White solid, 60% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.45 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 7.35 (ddd, <i>J</i> = 8.5, 2.0, 1.1 Hz, 1H), 7.31 (ddd, <i>J</i> = 8.5, 2.0, 1.0 Hz, 1H), 7.03 (td, <i>J</i> = 9.3, 7.1 Hz, 1H), 6.60 (td, <i>J</i> = 8.7, 4.1 Hz, 1H), 5.74 (s, 1H), 3.95–3.89 (m, 2H), 3.87–3.83 (m, 2H), 3.83–3.80 (m, 2H), 1.77–1.66 (m, 4H). HPLC > 99%, <i>t</i><sub>R</sub> = 4.48 min. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>F<sub>3</sub>IN<sub>2</sub>O<sub>4</sub><sup>+</sup>: 523.0336; found: 523.0324.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 11-((<i>tert</i>-Butoxycarbonyl)(methyl)amino)undecanoic Acid (<b>65</b>)</h4><div class="NLM_p last">To a solution of compound <b>64</b> (1.0 g, 2.6 mmol) in DMF (4 mL) was added NaH (0.3 g, 7.8 mmol) at 0 °C. The reaction mixture was stirred at rt for 0.5 h before CH<sub>3</sub>I (0.3 mL, 5.2 mmol) was added at 0 °C. After the reaction mixture was stirred for 18 h, the mixture was poured into ice water and filtered. It was then washed with methanol, and the filtrate was evaporated to obtain a white solid (0.4 g, 46% yield) without purification. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 3.65 (s, 1H), 3.21 (d, <i>J</i> = 7.5 Hz, 1H), 3.01 (t, <i>J</i> = 7.1 Hz, 2H), 2.83 (s, 1H), 2.31 (t, <i>J</i> = 7.4 Hz, 1H), 2.27 (t, <i>J</i> = 7.4 Hz, 2H), 1.60 (t, <i>J</i> = 7.1 Hz, 2H), 1.46–1.42 (m, 10H), 1.36–1.26 (m, 12H).</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-3,3-Dimethyl-2-(11-(methylamino)undecanamido)butanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>66</b>)</h4><div class="NLM_p">To a solution of compound <b>65</b> (0.4 g, 1.2 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (0.5 g, 1.2 mmol) in DMSO (5 mL) were added HOAt (0.24 g, 1.8 mmol), EDCI (0.35 g, 1.8 mmol), and <i>N</i>-methylmorpholine (0.4 mL, 3.6 mmol). The reaction mixture was stirred at rt overnight. The reaction mixture was monitored by UPLC. Upon completion, it was purified by reverse-phase column chromatography to provide the title compound <b>66</b> (0.5 g, 67% yield) as a white solid. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.98 (s, 1H), 7.47–7.41 (m, 4H), 5.00 (q, <i>J</i> = 6.9 Hz, 1H), 4.63 (d, <i>J</i> = 2.6 Hz, 1H), 4.59–4.53 (m, 1H), 4.44 (dp, <i>J</i> = 4.3, 1.9 Hz, 1H), 3.88 (dt, <i>J</i> = 11.3, 1.7 Hz, 1H), 3.75 (dd, <i>J</i> = 11.0, 4.0 Hz, 1H), 3.04–2.86 (m, 2H), 2.69 (s, 3H), 2.49 (s, 3H), 2.30 (ddd, <i>J</i> = 14.1, 8.3, 7.1 Hz, 1H), 2.27–2.22 (m, 1H), 2.20 (ddt, <i>J</i> = 13.1, 7.7, 2.0 Hz, 1H), 1.96 (ddd, <i>J</i> = 13.3, 9.0, 4.5 Hz, 1H), 1.72–1.63 (m, 2H), 1.59 (dd, <i>J</i> = 15.8, 7.2 Hz, 2H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H), 1.34 (d, <i>J</i> = 13.7 Hz, 12H), 1.04 (s, 9H).</div><div class="NLM_p last">Compound <b>67</b> was made by following similar procedures as compound <b>66</b>.</div></div><div id="sec4_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (2<i>R</i>,4<i>S</i>)-1-((<i>S</i>)-3,3-Dimethyl-2-(11-(methylamino)undecanamido)butanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>67</b>)</h4><div class="NLM_p last">White solid, 60% yield. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>35</sub>H<sub>56</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup>: 642.4048; found: 642.4124.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Cell Culture</h3><div class="NLM_p last">Cells were cultivated in Dulbecco’s modified Eagle’s medium (DMEM), RPMI-1640, or Eagle’s minimum essential medium (EMEM) supplemented with 10% FBS, 100 units/mL of penicillin, and 100 g/mL of streptomycin.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Western Blot</h3><div class="NLM_p last">Cells were lysed on ice for 30 min with the lysis buffer (50 mM Tris pH 7.4, 1% IGEPAL CA-630, 150 mM NaCl, 1 mM ethylenediaminetetraacetic acid (EDTA), and 1 mM AESBF), supplemented with protease and phosphatase inhibitor cocktail (A32961, Thermo Fisher Scientific). The samples were centrifuged at 12 000<i>g</i> for 10 min at 4 °C to obtain the supernatant as a cell lysate. Protein concentrations were quantified using the Pierce rapid gold bicinchoninic acid (BCA) protein assay kit. The primary antibodies used were MEK1 (2352, CST), MEK2 (9147, CST), pMEK (9121, CST), ERK (4696, CST), pERK (4370, CST), α-tubulin (T6074, Sigma-Aldrich), IKZF1 (9034, CST), IKZF3 (15103, CST), ZFP91 (PA5-41199, Thermo Fisher Scientific), and GSPT1 (ab126090, Abcam). Fluorescence-labeled IRDye 800CW goat anti-rabbit IgG (926-32211, LI-COR) and IRDye 680CW donkey anti-mouse IgG (926-68072, LI-COR) and the OdysseyCLx imaging system (LI-COR) were used to obtain protein signals, which were then analyzed by Image Studio Lite software (LI-COR). DC<sub>50</sub> values were obtained with GraphPad Prism 8 from the data of three independent experiments.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Binding Affinity Assays</h3><div class="NLM_p last">For most assays, kinase-tagged T7 phage strains were prepared in an <i>Escherichia coli</i> host derived from the BL21 strain. <i>E. coli</i> were grown to log-phase and infected with T7 phage and incubated with shaking at 32 °C until lysis. The lysates were centrifuged and filtered to remove cell debris. The remaining kinases were produced in HEK-293 cells and subsequently tagged with DNA for qPCR detection. Streptavidin-coated magnetic beads were treated with biotinylated small-molecule ligands for 30 min at rt to generate affinity resins for kinase assays. The liganded beads were blocked with excess biotin and washed with blocking buffer (SeaBlock (Pierce), 1% bovine serum albumin (BSA), 0.05% Tween 20, 1 mM dithiothreitol (DTT)) to remove unbound ligand and to reduce nonspecific binding. Binding reactions were assembled by combining kinases, liganded affinity beads, and test compounds in 1× binding buffer (20% SeaBlock, 0.17 × phosphate-buffered saline (PBS), 0.05% Tween 20, 6 mM DTT). Test compounds were prepared as 111X stocks in 100% DMSO. Kds were determined using an 11-point 3-fold compound dilution series with three DMSO control points. All compounds for <i>K</i><sub>d</sub> measurements are distributed by acoustic transfer (noncontact dispensing) in 100% DMSO. The compounds were then diluted directly into the assays such that the final concentration of DMSO was 0.9%. All reactions were performed in a polypropylene 384-well plate. Each was a final volume of 0.02 mL. The assay plates were incubated at room temperature with shaking for 1 h, and the affinity beads were washed with wash buffer (1 PBS, 0.05% Tween 20). The beads were then resuspended in elution buffer (1 PBS, 0.05% Tween 20, 0.5 μM nonbiotinylated affinity ligand) and incubated at room temperature with shaking for 30 min. The kinase concentration in the eluates was measured by the quantitative polymerase chain reaction (qPCR).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Proteomics Studies</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> Protein Extraction and Digestion</h4><div class="NLM_p last">Cells with indicated treatments were harvested and lysed in lysis buffer (8 M urea, 100 mM tris-HCl pH 8.0). Sonication (5 s on 5 s off, 2 × 30 s) was performed to shear genome DNA. The lysate was centrifugated for 30 min at 3500<i>g</i> at 4 °C and the supernatant was transferred to a clean tube. The protein concentration was determined (BCA assay) and the protein was reduced with 5 mM DTT (dithiothreitol), alkylated with 15 mM IAA (iodoacetamide) in the dark conditions, and then diluted with 3 volume of buffer 25 mM Tris (pH 8.0) and 1 mM CaCl<sub>2</sub>. The final urea concentration is 2 M. Trypsin was added into protein solution with 1:100 ratio (trypsin:protein) and digested 12–16 h or overnight at rt.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> Mass Spectrometry Analysis</h4><div class="NLM_p">Peptides were cleaned up by C18 stage tips and the concentration was determined (Peptide assay, 23275, Thermo Fisher Scientific). The clean peptides were dissolved in 0.1% formic acid and analyzed on a Q Exactive HF-X coupled with an Easy nanoLC 1200 (Thermo Fisher Scientific, San Jose, CA). Then, 0.5 g of peptides were loaded onto an Acclaim PepMap RSLC C18 Column (150 mm 75 m ID, C18, 2 m, Thermo Fisher). Analytical separation of all peptides was achieved with a 130 min gradient. A linear gradient of 5–30% buffer B over 110 min was executed at a 300 nL/min flow rate followed by a ramp to 100% B in 5 min, and 15 min wash with 100% B, where buffer A was aqueous 0.1% formic acid, and buffer B was 80% acetonitrile and 0.1% formic acid.</div><div class="NLM_p last">Liquid chromatography–mass spectrometry (LC–MS) experiments were performed in a data-dependent mode with full MS (externally calibrated to a mass accuracy of <5 ppm and a resolution of 60 000 at <i>m</i>/<i>z</i> 200) followed by high-energy collision-activated dissociation-MS/MS of the top 20 most intense ions with a resolution of 15 000 at <i>m</i>/<i>z</i> 200. High-energy collision-activated dissociation-MS/MS was used to dissociate peptides at a normalized collision energy of 27 eV in the presence of nitrogen bath gas atoms. Dynamic exclusion was 30 s. There were two technical replicates for one treatment, and each sample was subjected to three technical LC–MS replicates.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> MS Data Analysis</h4><div class="NLM_p last">Mass spectra processing and peptide identification were performed on the Andromeda search engine in MaxQuant software (Version 1.6.10.43) against a human UniProt database (<a href="https://UP000005640" class="ext-link">UP000005640</a>). All searches were conducted with a defined modification of cysteine carbamidomethylation, with methionine oxidation and protein amino-terminal acetylation as dynamic modifications. Peptides were confidently identified using a target-decoy approach with a peptide false discovery rate (FDR) of 1% and a protein FDR of 1%. A minimum peptide length of 7 amino acids was required, maximally two missed cleavages were allowed, the initial mass deviation for precursor ion was up to 7 ppm, and the maximum allowed mass deviation for fragment ions was 0.5 Da. Data processing and statistical analysis were performed on Perseus (Version 1.6.10.50). Protein quantitation was performed on technical replicate runs, and a two-sample <i>t</i>-test statistics were used to report statistically significant expression fold-changes.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Cell Viability Assay and Clonogenic Assay</h3><div class="NLM_p last">The assays were conducted according to the protocols reported previously.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Mouse PK Studies</h3><div class="NLM_p last">The in vivo PK studies were conducted for compounds <b>24</b> and <b>27</b> using three male Swiss Albino mice per compound. The mice were administered intraperitoneally with solution formulation (5% <i>N</i>-methyl-2-pyrrolidinone (NMP), 5% solutol HS-15, and 90% normal saline) of compound <b>24</b> or <b>27</b> at a single dose of 50 mg/kg. Sixty microliters of blood samples were collected from each mouse at 0.5, 2, and 8 h. Plasma was harvested by centrifugation of blood and stored at −70 ± 10 °C until analysis. Plasma samples were quantified by the fit-for-purpose LC–MS/MS method (LLOQ: 5.02 ng/mL for plasma). These mouse PK experiments were performed according to the Institutional Animal Care and Use Committee-approved protocol.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Statistic Methods</h3><div class="NLM_p last">For all data, the number of biologically independent experiments and technical replicates, error bars, and <i>P</i>- values are described in figure legends, respectively. At least two independent experiments were conducted for all biological studies. The proteomics study was conducted in duplicate. Two-tailed Student’s <i>t</i>-tests were used for the indicated analyses, respectively, <i>P</i> ≥ 0.05, ns; 0.01 < <i>P</i> < 0.05, *; 0.001 < <i>P</i> < 0.01, **; and <i>P</i> < 0.001, ***.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i89"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01609" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12774" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12774" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01609?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01609</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">MEK1/2 degradation induced by compounds <b>24</b> and <b>27</b>; MEK1/2 degradation induced by compound <b>50</b>; binding affinity curves of compounds <b>24</b>, <b>27</b>, and <b>50</b>–<b>53</b>; Western blot validation of proteomics studies of compound <b>50</b>; Neo-substrates of CRBN degradation by compound <b>50</b>; compounds <b>24</b> and <b>27</b> degraded MEK1/2 proteins and inhibited downstream ERK signaling in H3122 and SUDHL1 cells; and antiproliferation potencies of PLX4032 and ZSTK474 in HT-29 and SK-MEL-28 cell lines; and <sup>1</sup>H, <sup>13</sup>C, HPLC, and HRMS spectra of compounds <b>24</b>, <b>27</b>, and <b>50</b>–<b>53</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf">jm0c01609_si_001.pdf (3.18 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_002.csv">jm0c01609_si_002.csv (7.04 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01609" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72023" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72023" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#513b383f367f3d3824113c22223c7f343524"><span class="__cf_email__" data-cfemail="4c2625222b622025390c213f3f2162292839">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#43292a222d6d292a2d032e30302e6d262736"><span class="__cf_email__" data-cfemail="49232028276723202709243a3a24672c2d3c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianping Hu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jieli Wei</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hyerin Yim</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling Xie</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Margaret S. Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Md Kabir</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lihuai Qin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xian Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, University
of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.H. and J.W. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): J.H., J.W., H.Y., M.K., J.L., and J.J. are inventors of a patent application filed by the Icahn School of Medicine at Mount Sinai.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e8450-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i91">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00440" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00440" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported in part by the P30CA196521 grant (to J.J.) from the US National Institutes of Health and an endowed professorship from the Icahn School of Medicine at Mount Sinai (to J.J.). This work utilized the AVANCE NEO 600 MHz NMR Spectrometer System that was upgraded with funding from a National Institutes of Health SIG grant 1S10OD025132-01A1. We thank Drs. Poulikos Poulikakos, Ernesto Guccione, E. Premkumar Reddy, and James A. Fagin for providing cell lines.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">ERK</td><td class="NLM_def"><p class="first last">extracellular signal-regulated kinase</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">MEK1 and MEK2</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase kinases 1 and 2</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">nonsmall cell lung cancer</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">UPS</td><td class="NLM_def"><p class="first last">ubiquitin-proteasome system</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i93">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07242" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07242" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 65 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Targeting the RAF-MEK-ERK mitogen-activated protein kinase cascade for the treatment of cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3291</span>– <span class="NLM_lpage">3310</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fsj.onc.1210422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=17496923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1Wktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3291-3310&author=P.+J.+Robertsauthor=C.+J.+Der&title=Targeting+the+RAF-MEK-ERK+mitogen-activated+protein+kinase+cascade+for+the+treatment+of+cancer&doi=10.1038%2Fsj.onc.1210422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Raf-MEK-ERK mitogen-activated protein kinase cascade for the treatment of cancer</span></div><div class="casAuthors">Roberts, P. J.; Der, C. J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3291-3310</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mitogen-activated protein kinase (MAPK) cascades are key signaling pathways involved in the regulation of normal cell proliferation, survival and differentiation.  Aberrant regulation of MAPK cascades contribute to cancer and other human diseases.  In particular, the extracellular signal-regulated kinase (ERK) MAPK pathway has been the subject of intense research scrutiny leading to the development of pharmacol. inhibitors for the treatment of cancer.  ERK is a downstream component of an evolutionarily conserved signaling module that is activated by the Raf serine/threonine kinases.  Raf activates the MAPK/ERK kinase (MEK)1/2 dual-specificity protein kinases, which then activate ERK1/2.  The mutational activation of Raf in human cancers supports the important role of this pathway in human oncogenesis.  Addnl., the Raf-MEK-ERK pathway is a key downstream effector of the Ras small GTPase, the most frequently mutated oncogene in human cancers.  Finally, Ras is a key downstream effector of the epidermal growth factor receptor (EGFR), which is mutationally activated and/or overexpressed in a wide variety of human cancers.  ERK activation also promotes upregulated expression of EGFR ligands, promoting an autocrine growth loop crit. for tumor growth.  Thus, the EGFR-Ras-Raf-MEK-ERK signaling network has been the subject of intense research and pharmaceutical scrutiny to identify novel target-based approaches for cancer treatment.  In this review, we summarize the current status of the different approaches and targets that are under evaluation and development for the therapeutic intervention of this key signaling pathway in human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7hdVDVyRBHrVg90H21EOLACvtfcHk0liicULoKDlg1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1Wktb4%253D&md5=bf56109e3b4defc12004471da30178c0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210422%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DP.%2BJ.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DTargeting%2520the%2520RAF-MEK-ERK%2520mitogen-activated%2520protein%2520kinase%2520cascade%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D3291%26epage%3D3310%26doi%3D10.1038%2Fsj.onc.1210422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eblen, S. T.</span></span> <span> </span><span class="NLM_article-title">Extracellular-regulated kinases: signaling from RAS to ERK substrates to control biological outcomes</span>. <i>Adv. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1016/bs.acr.2018.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fbs.acr.2018.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29551131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFGrsr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2018&pages=99-142&author=S.+T.+Eblen&title=Extracellular-regulated+kinases%3A+signaling+from+RAS+to+ERK+substrates+to+control+biological+outcomes&doi=10.1016%2Fbs.acr.2018.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular-regulated kinases: signaling from Ras to ERK substrates to control biological outcomes</span></div><div class="casAuthors">Eblen, Scott T.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-142</span>CODEN:
                <span class="NLM_cas:coden">ACRSAJ</span>;
        ISSN:<span class="NLM_cas:issn">0065-230X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The extracellular-regulated kinases ERK1 and ERK2 are evolutionarily conserved, ubiquitous serine-threonine kinases that are involved in regulating cellular signaling in both normal and pathol. conditions.  Their expression is crit. for development and their hyperactivation is a major factor in cancer development and progression.  Since their discovery as one of the major signaling mediators activated by mitogens and Ras mutation, we have learned much about their regulation, including their activation, binding partners and substrates.  In this review I will discuss some of what has been discovered about the members of the Ras to ERK pathway, including regulation of their activation by growth factors and cell adhesion pathways.  Looking downstream of ERK activation I will also highlight some of the many ERK substrates that have been discovered, including those involved in feedback regulation, cell migration and cell cycle progression through the control of transcription, pre-mRNA splicing and protein synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2KWjVn2VpKrVg90H21EOLACvtfcHk0lhIcI8BgFL8wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFGrsr3J&md5=54d0bf34a1a52c1a938536341b945054</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fbs.acr.2018.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.acr.2018.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DEblen%26aufirst%3DS.%2BT.%26atitle%3DExtracellular-regulated%2520kinases%253A%2520signaling%2520from%2520RAS%2520to%2520ERK%2520substrates%2520to%2520control%2520biological%2520outcomes%26jtitle%3DAdv.%2520Cancer%2520Res.%26date%3D2018%26volume%3D138%26spage%3D99%26epage%3D142%26doi%3D10.1016%2Fbs.acr.2018.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frémin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meloche, S.</span></span> <span> </span><span class="NLM_article-title">From basic research to clinical development of MEK1/2 inhibitors for cancer therapy</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">8</span> <span class="refDoi"> DOI: 10.1186/1756-8722-3-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1186%2F1756-8722-3-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=20149254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BC3c7ltFGnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&author=C.+Fr%C3%A9minauthor=S.+Meloche&title=From+basic+research+to+clinical+development+of+MEK1%2F2+inhibitors+for+cancer+therapy&doi=10.1186%2F1756-8722-3-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">From basic research to clinical development of MEK1/2 inhibitors for cancer therapy</span></div><div class="casAuthors">Fremin Christophe; Meloche Sylvain</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Ras-dependent Raf/MEK/ERK1/2 mitogen-activated protein (MAP) kinase signaling pathway is a major regulator of cell proliferation and survival.  Not surprisingly, hyperactivation of this pathway is frequently observed in human malignancies as a result of aberrant activation of receptor tyrosine kinases or gain-of-function mutations in RAS or RAF genes.  Components of the ERK1/2 pathway are therefore viewed as attractive candidates for the development of targeted therapies of cancer.  In this article, we briefly review the basic research that has laid the groundwork for the clinical development of small molecules inhibitors of the ERK1/2 pathway.  We then present the current state of clinical evaluation of MEK1/2 inhibitors in cancer and discuss challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRXiqh48Jb4JresJRIHUP1ofW6udTcc2eYgcj9dAT8eTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7ltFGnsg%253D%253D&md5=b6b8c91b259b37885fd951001b4bd817</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2F1756-8722-3-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8722-3-8%26sid%3Dliteratum%253Aachs%26aulast%3DFr%25C3%25A9min%26aufirst%3DC.%26aulast%3DMeloche%26aufirst%3DS.%26atitle%3DFrom%2520basic%2520research%2520to%2520clinical%2520development%2520of%2520MEK1%252F2%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2010%26volume%3D3%26doi%3D10.1186%2F1756-8722-3-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">MEK1/2 dual-specificity protein kinases: structure and regulation</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>417</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2011.11.145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.bbrc.2011.11.145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=22177953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1Oqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2012&pages=5-10&author=R.+Roskoski&title=MEK1%2F2+dual-specificity+protein+kinases%3A+structure+and+regulation&doi=10.1016%2Fj.bbrc.2011.11.145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1/2 dual-specificity protein kinases: Structure and regulation</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">417</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  MEK1 and MEK2 are related protein kinases that participate in the RAS-RAF-MEK-ERK signal transduction cascade.  This cascade participates in the regulation of a large variety of processes including apoptosis, cell cycle progression, cell migration, differentiation, metab., and proliferation.  Moreover, oncogenic mutations in RAS or B-RAF are responsible for a large proportion of human cancers.  MEK1 is activated by phosphorylation of Ser-218 and Ser-222 in its activation segment as catalyzed by RAF kinases in an intricate process that involves a KSR scaffold.  In addn. to functioning as a scaffold, the kinase activity of KSR is also required for MEK activation.  MEK1 regulation is unusual in that Ser-212 phosphorylation in its activation segment is inhibitory.  Moreover, active ERK kinase catalyzes a feedback inhibitory phosphorylation of MEK1 Thr-292 that serves to down-regulate the pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRX9pGR2iVKbVg90H21EOLACvtfcHk0lhIcI8BgFL8wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1Oqtg%253D%253D&md5=4bdfbf6ebaa90055a314922fa1056929</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2011.11.145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2011.11.145%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DMEK1%252F2%2520dual-specificity%2520protein%2520kinases%253A%2520structure%2520and%2520regulation%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2012%26volume%3D417%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.bbrc.2011.11.145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span> <span> </span><span class="NLM_article-title">Properties of MEKs, the kinases that phosphorylate and activate the extracellular signal-regulated kinases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">23933</span>– <span class="NLM_lpage">23939</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=8226933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADyaK3sXlvFygtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=23933-23939&author=C.+F.+Zhengauthor=K.+L.+Guan&title=Properties+of+MEKs%2C+the+kinases+that+phosphorylate+and+activate+the+extracellular+signal-regulated+kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Properties of MEKs, the kinases that phosphorylate and activate the extracellular signal-regulated kinases</span></div><div class="casAuthors">Zheng, Chao Feng; Guan, Kun Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">23933-9</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Activation of extracellular signal-regulated kinase (ERK) or mitogen-activated protein kinase by MEK (mitogen-activated protein kinase or extracellular signal-regulated kinase) is an essential event in the mitogenic growth factor signal transduction.  The authors now demonstrate that three recombinant MEKs (MEK1, MEK2, MEK3) show remarkably different activity toward recombinant ERK1 and ERK2.  MEK2 is the most active ERK activator.  The recombinant MEK1 has an activity approx. seven times lower than that of MEK2.  MEK3, which is identical to MEK1 except for missing an internal 26-amino acid residue and probably represents an alternative splicing product of MEK1, shows neither autophosphorylation nor ERK-activating activity.  Recombinant MEK1 and MEK2 can be activated by epidermal growth factor-stimulated SWISS3T3 cell lysate.  MEK1 and MEK2 can also be activated by autophosphorylation.  Autophosphorylation of MEKs correlates with their ability to phosphorylate and activate ERKs.  Phosphorylation of MEK is also stimulated by ERK.  Phosphoamino acid anal. showed that ERK1 preferentially phosphorylated threonine residue of MEKs.  MEKs complex with ERKs in vitro.  Interestingly, MEK3 also forms a complex with ERK1, although it is totally inactive as an ERK activator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLXezUZaU78rVg90H21EOLACvtfcHk0lgIwDUivyTSYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlvFygtrs%253D&md5=f43bb5309432d722c03efc7958ef050f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DC.%2BF.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26atitle%3DProperties%2520of%2520MEKs%252C%2520the%2520kinases%2520that%2520phosphorylate%2520and%2520activate%2520the%2520extracellular%2520signal-regulated%2520kinases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D23933%26epage%3D23939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karin, M.</span></span> <span> </span><span class="NLM_article-title">Mammalian MAP kinase signalling cascades</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>410</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1038/35065000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2F35065000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=11242034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhvVSgs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2001&pages=37-40&author=L.+Changauthor=M.+Karin&title=Mammalian+MAP+kinase+signalling+cascades&doi=10.1038%2F35065000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian MAP kinase signalling cascades</span></div><div class="casAuthors">Chang, Lufen; Karin, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">410</span>
        (<span class="NLM_cas:issue">6824</span>),
    <span class="NLM_cas:pages">37-40</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review, with 52 refs.  Mitogen-activated protein kinases (MAPKs) are important signal transducing enzymes, unique to eukaryotes, that are involved in many facets of cellular regulation.  Initial research concd. on defining the components and organization of MAPK signaling cascades, but recent studies have begun to shed light on the physiol. functions of these cascades in the control of gene expression, cell proliferation, and programmed cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHwmx2Bk-5dbVg90H21EOLACvtfcHk0lgIwDUivyTSYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhvVSgs7k%253D&md5=486ff61090fc371dff36b307b4e75c88</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F35065000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35065000%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DL.%26aulast%3DKarin%26aufirst%3DM.%26atitle%3DMammalian%2520MAP%2520kinase%2520signalling%2520cascades%26jtitle%3DNature%26date%3D2001%26volume%3D410%26spage%3D37%26epage%3D40%26doi%3D10.1038%2F35065000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebmann, C.</span></span> <span> </span><span class="NLM_article-title">Regulation of MAP kinase activity by peptide receptor signalling pathway: Paradigms of multiplicity</span>. <i>Cell Signalling</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">785</span>, <span class="refDoi"> DOI: 10.1016/S0898-6568(01)00192-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS0898-6568%2801%2900192-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=11583913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnt1Shu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2001&pages=777-785&author=C.+Liebmann&title=Regulation+of+MAP+kinase+activity+by+peptide+receptor+signalling+pathway%3A+Paradigms+of+multiplicity&doi=10.1016%2FS0898-6568%2801%2900192-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MAP kinase activity by peptide receptor signalling pathway: Paradigms of multiplicity</span></div><div class="casAuthors">Liebmann, C.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">777-785</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) can stimulate the mitogen-activated protein kinase (MAPK) cascade and thereby induce cellular proliferation like receptor tyrosine kinases (RTKs).  Work over the past 5 yr has established several models which reduce the links of Gi-, Gq-, and Gs-coupled receptors to MAPK on few principle pathways.  They include (1) Ras-dependent activation of MAPK via transactivation of RTKs such as the EGF receptor (EGFR), (2) Ras-independent MAPK activation via protein kinase C (PKC) that converges with the RTK signaling at the level of Raf, and (3) activation as well as inactivation of MAPK via the cAMP/protein kinase A (PKA) pathway in dependency on the type of Raf.  Most of these generalizing hypotheses are founded on exptl. data obtained from expression studies and using a limited set of individual receptors.  This review will compare these models with pathways to MAPK found for a great variety of peptide hormone and neuropeptide receptor subtypes in various cells.  It becomes evident that under endogenous conditions, the transactivation pathway is less dominant as postulated, whereas pathways involving isoforms of PKC and, esp., phosphoinositide 3-kinase (PI-3K) appear to play a more important role as assumed so far.  Highly cell-specific and unusual connections of signaling proteins towards MAPK, in particular tumor cells, might provide points of attacks for new therapeutic concepts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq788Kr6Dawu7Vg90H21EOLACvtfcHk0lgIwDUivyTSYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnt1Shu7g%253D&md5=fd2819d92926824fdae4f08087fcf7eb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0898-6568%2801%2900192-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0898-6568%252801%252900192-9%26sid%3Dliteratum%253Aachs%26aulast%3DLiebmann%26aufirst%3DC.%26atitle%3DRegulation%2520of%2520MAP%2520kinase%2520activity%2520by%2520peptide%2520receptor%2520signalling%2520pathway%253A%2520Paradigms%2520of%2520multiplicity%26jtitle%3DCell%2520Signalling%26date%3D2001%26volume%3D13%26spage%3D777%26epage%3D785%26doi%3D10.1016%2FS0898-6568%2801%2900192-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsakiridis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsiani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirte, H.</span></span> <span> </span><span class="NLM_article-title">Mitogen-activated protein kinase (MAPK) expression and activation in non-small cell lung cancer (NSCLC): investigating their prognostic significance and role in radiation responses</span>. <i>Radiother. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">S40</span>– <span class="NLM_lpage">S41</span>, <span class="refDoi"> DOI: 10.1016/S0167-8140(04)80634-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS0167-8140%2804%2980634-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2004&pages=S40-S41&author=T.+Tsakiridisauthor=J.+Wrightauthor=E.+Tsianiauthor=M.+Kalinaauthor=G.+Singhauthor=L.+Vincicauthor=H.+Hirte&title=Mitogen-activated+protein+kinase+%28MAPK%29+expression+and+activation+in+non-small+cell+lung+cancer+%28NSCLC%29%3A+investigating+their+prognostic+significance+and+role+in+radiation+responses&doi=10.1016%2FS0167-8140%2804%2980634-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS0167-8140%2804%2980634-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0167-8140%252804%252980634-0%26sid%3Dliteratum%253Aachs%26aulast%3DTsakiridis%26aufirst%3DT.%26aulast%3DWright%26aufirst%3DJ.%26aulast%3DTsiani%26aufirst%3DE.%26aulast%3DKalina%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DVincic%26aufirst%3DL.%26aulast%3DHirte%26aufirst%3DH.%26atitle%3DMitogen-activated%2520protein%2520kinase%2520%2528MAPK%2529%2520expression%2520and%2520activation%2520in%2520non-small%2520cell%2520lung%2520cancer%2520%2528NSCLC%2529%253A%2520investigating%2520their%2520prognostic%2520significance%2520and%2520role%2520in%2520radiation%2520responses%26jtitle%3DRadiother.%2520Oncol.%26date%3D2004%26volume%3D72%26spage%3DS40%26epage%3DS41%26doi%3D10.1016%2FS0167-8140%2804%2980634-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pratilas, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span> <span> </span><span class="NLM_article-title">Therapeutic strategies for targeting BRAF in human cancer</span>. <i>Rev. Recent Clin. Trials</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.2174/157488707780599393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.2174%2F157488707780599393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=18473997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2sXms1Oisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=121-134&author=C.+A.+Pratilasauthor=D.+B.+Solit&title=Therapeutic+strategies+for+targeting+BRAF+in+human+cancer&doi=10.2174%2F157488707780599393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic strategies for targeting BRAF in human cancer</span></div><div class="casAuthors">Pratilas, Christine A.; Solit, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Reviews on Recent Clinical Trials</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-134</span>CODEN:
                <span class="NLM_cas:coden">RRCTB2</span>;
        ISSN:<span class="NLM_cas:issn">1574-8871</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Constitutive ERK activation is a common finding in human cancer and is often the result of activating mutations of BRAF and RAS.  BRAF missense mutations occur in approx. 8% of human tumors, most frequently in melanoma, papillary thyroid cancer and colon cancer.  Mutations in BRAF have been found predominantly in tumors in which RAS is commonly mutated but concurrent mutations of both BRAF and RAS are extremely rare.  Though over 40 different kinase domain mutations in BRAF have been identified, a single base-pair substitution in exon 15 at codon 600 (V600E) is found in over 80% of cases.  These mutations cluster in the glycine-rich loop and activation segments of the kinase and are predicted to induce kinase activation by disrupting the inhibitory glycine-rich loop/activation segment interaction which characterizes the inactive conformation.  The majority of mutations identified cause constitutive kinase activation with the V600E mutation demonstrating approx. 500-fold greater kinase activity than wild-type BRAF.  Supporting its classification as an oncogene, V600E BRAF stimulates ERK signaling, induces proliferation and is capable in model systems of promoting transformation.  However, BRAF mutations are common in nevi and colon polyps suggesting that BRAF mutation alone is insufficient for tumorigenesis and addnl. mutations are required for cancer development.  Though such data suggest that BRAF mutation is likely an early initiating event in tumors such as melanoma and colon cancer, preclin. studies suggest that tumors with V600E BRAF mutation remain dependent upon BRAF for proliferation and survival.  Given its frequent occurrence in human cancer and the continued requirement for BRAF activity in tumors with BRAF mutation, efforts are underway to develop targeted inhibitors of BRAF and its downstream effectors.  The first generation of RAF inhibitors, including sorafenib, were notable for their lack of specificity and potency for RAF and these agents have shown limited efficacy in tumors with a high incidence of BRAF mutation such as melanoma.  Novel inhibitors of the pathway with greater selectivity for BRAF and MEK are now in Phase 1 and 2 clin. trials with promising early results.  To maximize the likelihood of success with these agents, clin. trials enriched with patients whose tumors possess BRAF and RAS mutations have been proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGhODO3iHKHLVg90H21EOLACvtfcHk0lg6uLcxPangzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXms1Oisrs%253D&md5=78477e12284560df03576ef0d9b7fd8a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F157488707780599393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157488707780599393%26sid%3Dliteratum%253Aachs%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26atitle%3DTherapeutic%2520strategies%2520for%2520targeting%2520BRAF%2520in%2520human%2520cancer%26jtitle%3DRev.%2520Recent%2520Clin.%2520Trials%26date%3D2007%26volume%3D2%26spage%3D121%26epage%3D134%26doi%3D10.2174%2F157488707780599393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridlyand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kageshita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busam, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutzner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brocker, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBoit, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastian, B. C.</span></span> <span> </span><span class="NLM_article-title">Distinct sets of genetic alterations in melanoma</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">2135</span>– <span class="NLM_lpage">2147</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa050092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1056%2FNEJMoa050092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=16291983" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ajtLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2005&pages=2135-2147&author=J.+A.+Curtinauthor=J.+Fridlyandauthor=T.+Kageshitaauthor=H.+N.+Patelauthor=K.+J.+Busamauthor=H.+Kutznerauthor=K.+H.+Choauthor=S.+Aibaauthor=E.+B.+Brockerauthor=P.+E.+LeBoitauthor=D.+Pinkelauthor=B.+C.+Bastian&title=Distinct+sets+of+genetic+alterations+in+melanoma&doi=10.1056%2FNEJMoa050092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct sets of genetic alterations in melanoma</span></div><div class="casAuthors">Curtin, John A.; Fridlyand, Jane; Kageshita, Toshiro; Patel, Hetal N.; Busam, Klaus J.; Kutzner, Heinz; Cho, Kwang-Hyun; Aiba, Setsuya; Brocker, Eva-Bettina; LeBoit, Philip E.; Pinkel, Dan; Bastian, Boris C.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2135-2147</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND Exposure to UV light is a major causative factor in melanoma, although the relationship between risk and exposure is complex.  We hypothesized that the clin. heterogeneity is explained by genetically distinct types of melanoma with different susceptibility to UV light.  METHODS We compared genome-wide alterations in the no. of copies of DNA and mutational status of BRAF and N-RAS in 126 melanomas from four groups in which the degree of exposure to UV light differs: 30 melanomas from skin with chronic sun-induced damage and 40 melanomas from skin without such damage; 36 melanomas from palms, soles, and subungual (acral) sites; and 20 mucosal melanomas.  RESULTS We found significant differences in the frequencies of regional changes in the no. of copies of DNA and mutation frequencies in BRAF among the four groups of melanomas.  Samples could be correctly classified into the four groups with 70 percent accuracy on the basis of the changes in the no. of copies of genomic DNA.  In two-way com- parisons, melanomas arising on skin with signs of chronic sun-induced damage and skin without such signs could be correctly classified with 84 percent accuracy.  Acral melanoma could be distinguished from mucosal melanoma with 89 percent accuracy.  Eighty-one percent of melanomas on skin without chronic sun-induced damage had mutations in BRAF or N-RAS; the majority of melanomas in the other groups had mutations in neither gene.  Melanomas with wild-type BRAF or N-RAS frequently had increases in the no. of copies of the genes for cyclin-dependent kinase 4 (CDK4) and cyclin D1 (CCND1), downstream components of the RAS-BRAF pathway.  CONCLUSIONS The genetic alterations identified in melanomas at different sites and with different levels of sun exposure indicate that there are distinct genetic pathways in the development of melanoma and implicate CDK4 and CCND1 as independent oncogenes in melanomas without mutations in BRAF or N-RAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLgiGsuGhDMLVg90H21EOLACvtfcHk0lg6uLcxPangzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ajtLjP&md5=73fd3ec239a651dcc755fccd5336eab7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa050092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa050092%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DJ.%2BA.%26aulast%3DFridlyand%26aufirst%3DJ.%26aulast%3DKageshita%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DH.%2BN.%26aulast%3DBusam%26aufirst%3DK.%2BJ.%26aulast%3DKutzner%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DK.%2BH.%26aulast%3DAiba%26aufirst%3DS.%26aulast%3DBrocker%26aufirst%3DE.%2BB.%26aulast%3DLeBoit%26aufirst%3DP.%2BE.%26aulast%3DPinkel%26aufirst%3DD.%26aulast%3DBastian%26aufirst%3DB.%2BC.%26atitle%3DDistinct%2520sets%2520of%2520genetic%2520alterations%2520in%2520melanoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D353%26spage%3D2135%26epage%3D2147%26doi%3D10.1056%2FNEJMoa050092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goel, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warneke, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redston, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haluska, F. G.</span></span> <span> </span><span class="NLM_article-title">Examination of mutations in BRAF, NRAS, and PTEN in primary cutaneous melanoma</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1038/sj.jid.5700026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fsj.jid.5700026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=16417231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD28Xlt1Wju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2006&pages=154-160&author=V.+K.+Goelauthor=A.+J.+Lazarauthor=C.+L.+Warnekeauthor=M.+S.+Redstonauthor=F.+G.+Haluska&title=Examination+of+mutations+in+BRAF%2C+NRAS%2C+and+PTEN+in+primary+cutaneous+melanoma&doi=10.1038%2Fsj.jid.5700026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Examination of mutations in BRAF, NRAS, and PTEN in primary cutaneous melanoma</span></div><div class="casAuthors">Goel, Vikas K.; Lazar, Alexander J. F.; Warneke, Carla L.; Redston, Mark S.; Haluska, Frank G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">154-160</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Frequent somatic mutation of v-raf murine sarcoma viral oncogene homolog B (BRAF), a downstream effector of the rat sarcoma oncogene (RAS) signaling pathway, is described in melanoma and other tumors.  Our anal. of melanoma cell lines suggests that activating mutations in BRAF can occur simultaneously with inactivation of phosphatase and tensin homolog (PTEN), but neuroblastoma RAS (NRAS) mutations are not coincident.  We detd. the concurrent prevalence of mutations in BRAF and NRAS, and alteration of PTEN expression in 69 primary cutaneous melanomas.  BRAF mutations were seen in 57% of cases.  NRAS was mutated in 17% of samples, exclusively in exon 2.  Two cases showed concurrent BRAF and NRAS mutations.  Using immunohistochem., PTEN protein expression was lost or greatly reduced in 19% of tumors.  Seven tumors with reduced PTEN yielded DNA amenable to sequencing, and three also showed mutation in BRAF but none in NRAS.  In all, 11 (85%) of 13 tumors showing reduced PTEN expression were greater than 3.5 mm thick, and the assocn. of increasing Breslow thickness and loss or redn. of PTEN expression was statistically significant (P<0.0001).  Mutations in NRAS were not coincident with reduced PTEN expression, and the concurrent mutation of NRAS and BRAF was rare.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnKUIL416mYbVg90H21EOLACvtfcHk0lg6uLcxPangzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xlt1Wju7g%253D&md5=05dc8622b7b5ccfd17ae285f0b7bc2e0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsj.jid.5700026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.jid.5700026%26sid%3Dliteratum%253Aachs%26aulast%3DGoel%26aufirst%3DV.%2BK.%26aulast%3DLazar%26aufirst%3DA.%2BJ.%26aulast%3DWarneke%26aufirst%3DC.%2BL.%26aulast%3DRedston%26aufirst%3DM.%2BS.%26aulast%3DHaluska%26aufirst%3DF.%2BG.%26atitle%3DExamination%2520of%2520mutations%2520in%2520BRAF%252C%2520NRAS%252C%2520and%2520PTEN%2520in%2520primary%2520cutaneous%2520melanoma%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2006%26volume%3D126%26spage%3D154%26epage%3D160%26doi%3D10.1038%2Fsj.jid.5700026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adjei, A. A.</span></span> <span> </span><span class="NLM_article-title">The Ras/Raf/MAPK pathway</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1016/S1556-0864(15)31506-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1556-0864%2815%2931506-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=17409820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BD2s3gs1yrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=7-9&author=J.+R.+Molinaauthor=A.+A.+Adjei&title=The+Ras%2FRaf%2FMAPK+pathway&doi=10.1016%2FS1556-0864%2815%2931506-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The Ras/Raf/MAPK pathway</span></div><div class="casAuthors">Molina Julian R; Adjei Alex A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyYQXfFy_LN0ewzGPswiWOfW6udTcc2eb_XF3HwSWQB7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s3gs1yrug%253D%253D&md5=f36f1f3f53464529a76dbcbbf2efdb8a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS1556-0864%2815%2931506-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1556-0864%252815%252931506-9%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DJ.%2BR.%26aulast%3DAdjei%26aufirst%3DA.%2BA.%26atitle%3DThe%2520Ras%252FRaf%252FMAPK%2520pathway%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2006%26volume%3D1%26spage%3D7%26epage%3D9%26doi%3D10.1016%2FS1556-0864%2815%2931506-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilmartin, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleam, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minthorn, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rominger, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erskine, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zappacosta, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laquerre, S. G.</span></span> <span> </span><span class="NLM_article-title">GSK1120212 (JTP-74057) is an inhibitor of MEK activity and activation with favorable pharmacokinetic properties for sustained in vivo pathway inhibition</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1000</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-10-2200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F1078-0432.CCR-10-2200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=21245089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1eit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=989-1000&author=A.+G.+Gilmartinauthor=M.+R.+Bleamauthor=A.+Groyauthor=K.+G.+Mossauthor=E.+A.+Minthornauthor=S.+G.+Kulkarniauthor=C.+M.+Romingerauthor=S.+Erskineauthor=K.+E.+Fisherauthor=J.+Yangauthor=F.+Zappacostaauthor=R.+Annanauthor=D.+Suttonauthor=S.+G.+Laquerre&title=GSK1120212+%28JTP-74057%29+is+an+inhibitor+of+MEK+activity+and+activation+with+favorable+pharmacokinetic+properties+for+sustained+in+vivo+pathway+inhibition&doi=10.1158%2F1078-0432.CCR-10-2200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">GSK1120212 (JTP-74057) Is an Inhibitor of MEK Activity and Activation with Favorable Pharmacokinetic Properties for Sustained In Vivo Pathway Inhibition</span></div><div class="casAuthors">Gilmartin, Aidan G.; Bleam, Maureen R.; Groy, Arthur; Moss, Katherine G.; Minthorn, Elisabeth A.; Kulkarni, Swarupa G.; Rominger, Cynthia M.; Erskine, Symon; Fisher, Kelly E.; Yang, Jing-Song; Zappacosta, Francesca; Annan, Roland; Sutton, David; Laquerre, Sylvie G.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">989-1000</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Despite their preclin. promise, previous MEK inhibitors have shown little benefit for patients.  This likely reflects the narrow therapeutic window for MEK inhibitors due to the essential role of the P42/44 MAPK pathway in many nontumor tissues.  GSK1120212 is a potent and selective allosteric inhibitor of the MEK1 and MEK2 (MEK1/2) enzymes with promising antitumor activity in a phase I clin. trial (ASCO 2010).  Our studies characterize GSK1120212' enzymic, cellular, and in vivo activities, describing its unusually long circulating half-life.  Exptl. Design: Enzymic studies were conducted to det. GSK1120212 inhibition of recombinant MEK, following or preceding RAF kinase activation.  Cellular studies examd. GSK1120212 inhibition of ERK1 and 2 phosphorylation (p-ERK1/2) as well as MEK1/2 phosphorylation and activation.  Further studies explored the sensitivity of cancer cell lines, and drug pharmacokinetics and efficacy in multiple tumor xenograft models.  Results: In enzymic and cellular studies, GSK1120212 inhibits MEK1/2 kinase activity and prevents Raf-dependent MEK phosphorylation (S217 for MEK1), producing prolonged p-ERK1/2 inhibition.  Potent cell growth inhibition was evident in most tumor lines with mutant BRAF or Ras.  In xenografted tumor models, GSK1120212 orally dosed once daily had a long circulating half-life and sustained suppression of p-ERK1/2 for more than 24 h; GSK1120212 also reduced tumor Ki67, increased p27Kip1/CDKN1B, and caused tumor growth inhibition in multiple tumor models.  The largest antitumor effect was among tumors harboring mutant BRAF or Ras.  Conclusions: GSK1120212 combines high potency, selectivity, and long circulating half-life, offering promise for successfully targeting the narrow therapeutic window anticipated for clin. MEK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMOQvgszKIfbVg90H21EOLACvtfcHk0lgDSk0cd7h94g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXis1eit7k%253D&md5=6154be5fa0afb034c6a5995508d7ea9e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-2200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-2200%26sid%3Dliteratum%253Aachs%26aulast%3DGilmartin%26aufirst%3DA.%2BG.%26aulast%3DBleam%26aufirst%3DM.%2BR.%26aulast%3DGroy%26aufirst%3DA.%26aulast%3DMoss%26aufirst%3DK.%2BG.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DKulkarni%26aufirst%3DS.%2BG.%26aulast%3DRominger%26aufirst%3DC.%2BM.%26aulast%3DErskine%26aufirst%3DS.%26aulast%3DFisher%26aufirst%3DK.%2BE.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZappacosta%26aufirst%3DF.%26aulast%3DAnnan%26aufirst%3DR.%26aulast%3DSutton%26aufirst%3DD.%26aulast%3DLaquerre%26aufirst%3DS.%2BG.%26atitle%3DGSK1120212%2520%2528JTP-74057%2529%2520is%2520an%2520inhibitor%2520of%2520MEK%2520activity%2520and%2520activation%2520with%2520favorable%2520pharmacokinetic%2520properties%2520for%2520sustained%2520in%2520vivo%2520pathway%2520inhibition%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2011%26volume%3D17%26spage%3D989%26epage%3D1000%26doi%3D10.1158%2F1078-0432.CCR-10-2200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Den Otter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeppen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engst, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvin, M.</span></span> <span> </span><span class="NLM_article-title">Intermittent administration of MEK inhibitor GDC-0973 plus PI3K inhibitor GDC-0941 triggers robust apoptosis and tumor growth inhibition</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-1515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F0008-5472.CAN-11-1515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=22084396" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=210-219&author=K.+P.+Hoeflichauthor=M.+Merchantauthor=C.+Orrauthor=J.+Chanauthor=D.+Den+Otterauthor=L.+Berryauthor=I.+Kasmanauthor=H.+Koeppenauthor=K.+Riceauthor=N.+Y.+Yangauthor=S.+Engstauthor=S.+Johnstonauthor=L.+S.+Friedmanauthor=M.+Belvin&title=Intermittent+administration+of+MEK+inhibitor+GDC-0973+plus+PI3K+inhibitor+GDC-0941+triggers+robust+apoptosis+and+tumor+growth+inhibition&doi=10.1158%2F0008-5472.CAN-11-1515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Intermittent Administration of MEK Inhibitor GDC-0973 plus PI3K Inhibitor GDC-0941 Triggers Robust Apoptosis and Tumor Growth Inhibition</span></div><div class="casAuthors">Hoeflich, Klaus P.; Merchant, Mark; Orr, Christine; Chan, Jocelyn; Den Otter, Doug; Berry, Leanne; Kasman, Ian; Koeppen, Hartmut; Rice, Ken; Yang, Nai-Ying; Engst, Stefan; Johnston, Stuart; Friedman, Lori S.; Belvin, Marcia</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-219</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Combinations of MAP/ERK kinase (MEK) and phosphoinositide 3-kinase (PI3K) inhibitors have shown promise in preclin. cancer models, leading to the initiation of clin. trials cotargeting these two key cancer signaling pathways.  GDC-0973, a novel selective MEK inhibitor, and GDC-0941, a class I PI3K inhibitor, are in early stage clin. trials as both single agents and in combination.  The discovery of these selective inhibitors has allowed investigation into the precise effects of combining inhibitors of two major signaling branches downstream of RAS.  Here, we investigated multiple biomarkers in the mitogen-activated protein kinase (MAPK) and PI3K pathway to search for points of convergence that explain the increased apoptosis seen in combination.  Using washout studies in vitro and alternate dosing schedules in mice, we showed that intermittent inhibition of the PI3K and MAPK pathway is sufficient for efficacy in BRAF and KRAS mutant cancer cells.  The combination of GDC-0973 with the PI3K inhibitor GDC-0941 resulted in combination efficacy in vitro and in vivo via induction of biomarkers assocd. with apoptosis, including Bcl-2 family proapoptotic regulators.  Therefore, these data suggest that continuous exposure of MEK and PI3K inhibitors in combination is not required for efficacy in preclin. cancer models and that sustained effects on downstream apoptosis biomarkers can be obsd. in response to intermittent dosing.  Cancer Res; 72(1); 210-9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokHbA7g6hLGrVg90H21EOLACvtfcHk0ljovZwlsb5Ggg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCltA%253D%253D&md5=40032bf35d5e6335856a645d5c331df8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-1515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-1515%26sid%3Dliteratum%253Aachs%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DOrr%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DDen%2BOtter%26aufirst%3DD.%26aulast%3DBerry%26aufirst%3DL.%26aulast%3DKasman%26aufirst%3DI.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DRice%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DN.%2BY.%26aulast%3DEngst%26aufirst%3DS.%26aulast%3DJohnston%26aufirst%3DS.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DBelvin%26aufirst%3DM.%26atitle%3DIntermittent%2520administration%2520of%2520MEK%2520inhibitor%2520GDC-0973%2520plus%2520PI3K%2520inhibitor%2520GDC-0941%2520triggers%2520robust%2520apoptosis%2520and%2520tumor%2520growth%2520inhibition%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D210%26epage%3D219%26doi%3D10.1158%2F0008-5472.CAN-11-1515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woessner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssikatos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poch, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, K.</span></span> <span> </span><span class="NLM_article-title">Abstract 2515: preclinical development of ARRY-162, a potent and selective MEK 1/2 inhibitor</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">2515</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM10-2515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F1538-7445.AM10-2515" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=2515&author=P.+A.+Leeauthor=E.+Wallaceauthor=A.+Marlowauthor=T.+Yehauthor=V.+Marshauthor=D.+Andersonauthor=R.+Woessnerauthor=B.+Hurleyauthor=J.+Lyssikatosauthor=G.+Pochauthor=S.+Grossauthor=S.+Ranaauthor=S.+Winskiauthor=K.+Koch&title=Abstract+2515%3A+preclinical+development+of+ARRY-162%2C+a+potent+and+selective+MEK+1%2F2+inhibitor&doi=10.1158%2F1538-7445.AM10-2515"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM10-2515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM10-2515%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DP.%2BA.%26aulast%3DWallace%26aufirst%3DE.%26aulast%3DMarlow%26aufirst%3DA.%26aulast%3DYeh%26aufirst%3DT.%26aulast%3DMarsh%26aufirst%3DV.%26aulast%3DAnderson%26aufirst%3DD.%26aulast%3DWoessner%26aufirst%3DR.%26aulast%3DHurley%26aufirst%3DB.%26aulast%3DLyssikatos%26aufirst%3DJ.%26aulast%3DPoch%26aufirst%3DG.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DRana%26aufirst%3DS.%26aulast%3DWinski%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DK.%26atitle%3DAbstract%25202515%253A%2520preclinical%2520development%2520of%2520ARRY-162%252C%2520a%2520potent%2520and%2520selective%2520MEK%25201%252F2%2520inhibitor%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D2515%26doi%3D10.1158%2F1538-7445.AM10-2515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, U. R.</span></span> <span> </span><span class="NLM_article-title">Use of mitogenic cascade blockers for treatment of C-Raf induced lung adenoma in vivo: CI-1040 strongly reduces growth and improves lung structure</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">24</span> <span class="refDoi"> DOI: 10.1186/1471-2407-4-24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1186%2F1471-2407-4-24" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=15171791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BD2czhsF2guw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&author=B.+W.+Kramerauthor=R.+Gotzauthor=U.+R.+Rapp&title=Use+of+mitogenic+cascade+blockers+for+treatment+of+C-Raf+induced+lung+adenoma+in+vivo%3A+CI-1040+strongly+reduces+growth+and+improves+lung+structure&doi=10.1186%2F1471-2407-4-24"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Use of mitogenic cascade blockers for treatment of C-Raf induced lung adenoma in vivo: CI-1040 strongly reduces growth and improves lung structure</span></div><div class="casAuthors">Kramer Boris W; Gotz Rudolf; Rapp Ulf R</div><div class="citationInfo"><span class="NLM_cas:title">BMC cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Signaling networks promoting cell growth and proliferation are frequently deregulated in cancer.  Tumors often are highly dependent on such signaling pathways and may become hypersensitive to downregulation of key components within these signaling cascades.  The classical mitogenic cascade transmits stimuli from growth factor receptors via Ras, Raf, MEK and ERK to the cell nucleus and provides attractive molecular targets for cancer treatment.  For example, Ras and Raf kinase inhibitors are already in a number of ongoing phase II and phase III clinical trials.  In this study the effect of the Raf kinase inhibitor BAY 43-9006 and of the MEK inhibitor CI-1040 (PD184352) on a Raf dependent lung tumor mouse model was analyzed in detail.  METHODS:  We have generated a lung cancer mouse model by targeting constitutively active C-Raf kinase to the lung.  These mice develop adenomas within 4 months of life.  At this time-point they received daily intraperitoneal injections of either 100 mg/kg BAY 43-9006 or CI-1040 for additional 21 days.  Thereafter, lungs were isolated and the following parameters were analyzed using histology and immunohistochemistry: overall lung structure, frequency of adenoma foci, proliferation rate, ERK activity, caspase-3 activation, and lung differentiation.  RESULTS:  Both inhibitors were equally effective in vitro using a sensitive Raf/MEK/ERK ELISA.  In vivo, the systemic administration of the MEK inhibitor CI-1040 reduced adenoma formation to a third and significantly restored lung structure.  The proliferation rate of lung cells of mice treated with CL-1040 was decreased without any obvious effects on differentiation of pneumocytes.  In contrast, the Raf inhibitor BAY 43-9006 did not influence adenoma formation in vivo.  CONCLUSION:  The MEK inhibitor CI-1040 may be used for the treatment of Ras and/or Raf-dependent human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSBd-ofJhdpmm52AMpI4GK2fW6udTcc2earuiSEIfRMJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czhsF2guw%253D%253D&md5=34f11f9b5bb568c02259d04f54b5664b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-4-24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-4-24%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DB.%2BW.%26aulast%3DGotz%26aufirst%3DR.%26aulast%3DRapp%26aufirst%3DU.%2BR.%26atitle%3DUse%2520of%2520mitogenic%2520cascade%2520blockers%2520for%2520treatment%2520of%2520C-Raf%2520induced%2520lung%2520adenoma%2520in%2520vivo%253A%2520CI-1040%2520strongly%2520reduces%2520growth%2520and%2520improves%2520lung%2520structure%26jtitle%3DBMC%2520Cancer%26date%3D2004%26volume%3D4%26doi%3D10.1186%2F1471-2407-4-24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeannin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vansteenkiste, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franke, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinsted, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zazulina, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crino, L.</span></span> <span> </span><span class="NLM_article-title">Selumetinib plus docetaxel for KRAS-mutant advanced non-small-cell lung cancer: a randomised, multicentre, placebo-controlled, phase 2 study</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(12)70489-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1470-2045%2812%2970489-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=23200175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BC3s7otVCnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=38-47&author=P.+A.+J%C3%A4nneauthor=A.+T.+Shawauthor=J.+R.+Pereiraauthor=G.+Jeanninauthor=J.+Vansteenkisteauthor=C.+Barriosauthor=F.+A.+Frankeauthor=L.+Grinstedauthor=V.+Zazulinaauthor=P.+Smithauthor=I.+Smithauthor=L.+Crino&title=Selumetinib+plus+docetaxel+for+KRAS-mutant+advanced+non-small-cell+lung+cancer%3A+a+randomised%2C+multicentre%2C+placebo-controlled%2C+phase+2+study&doi=10.1016%2FS1470-2045%2812%2970489-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Selumetinib plus docetaxel for KRAS-mutant advanced non-small-cell lung cancer: a randomised, multicentre, placebo-controlled, phase 2 study</span></div><div class="casAuthors">Janne Pasi A; Shaw Alice T; Pereira Jose Rodrigues; Jeannin Gaelle; Vansteenkiste Johan; Barrios Carlos; Franke Fabio Andre; Grinsted Lynda; Zazulina Victoria; Smith Paul; Smith Ian; Crino Lucio</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-47</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  No targeted therapies are available for KRAS-mutant non-small-cell lung cancer (NSCLC).  Selumetinib is an inhibitor of MEK1/MEK2, downstream of KRAS, with preclinical evidence of synergistic activity with docetaxel in KRAS-mutant cancers.  We did a prospective, randomised, phase 2 trial to assess selumetinib plus docetaxel in previously treated patients with advanced KRAS-mutant NSCLC.  METHODS:  Eligible patients were older than 18 years of age; had histologically or cytologically confirmed stage IIIB-IV KRAS-mutant NSCLC; had failed first-line therapy for advanced NSCLC; had WHO performance status of 0-1; had not received previous therapy with either a MEK inhibitor or docetaxel; and had adequate bone marrow, renal, and liver function.  Patients were randomly assigned (in a 1:1 ratio) to either oral selumetinib (75 mg twice daily in a 21 day cycle) or placebo; all patients received intravenous docetaxel (75 mg/m(2) on day 1 of a 21 day cycle).  Randomisation was done with an interactive voice response system and investigators, patients, data analysts, and the trial sponsor were masked to treatment assignment.  The primary endpoint was overall survival, analysed for all patients with confirmed KRAS mutations.  This study is registered with ClinicalTrials.gov, number NCT00890825.  FINDINGS:  Between April 20, 2009, and June 30, 2010, we randomly assigned 44 patients to receive selumetinib and docetaxel (selumetinib group) and 43 to receive placebo and docetaxel (placebo group).  Of these, one patient in the selumetinib group and three in the placebo group were excluded from efficacy analyses because their tumours were not confirmed to be KRAS-mutation positive.  Median overall survival was 9·4 months (6·8-13·6) in the selumetinib group and 5·2 months (95% CI 3·8-non-calculable) in the placebo group (hazard ratio [HR] for death 0·80, 80% CI 0·56-1·14; one-sided p=0·21).  Median progression-free survival was 5·3 months (4·6-6·4) in the selumetinib group and 2·1 months (95% CI 1·4-3·7) in the placebo group (HR for progression 0·58, 80% CI 0·42-0·79; one-sided p=0·014). 16 (37%) patients in the selumetinib group and none in the placebo group had an objective response (p<0·0001).  Adverse events of grade 3 or higher occurred in 36 (82%) patients in the selumetinib group and 28 (67%) patients in the placebo group.  The most common grade 3-4 adverse events were neutropenia (29 [67%] of 43 patients in the selumetinib group vs 23 [55%] of 42 patients in the placebo group), febrile neutropenia (eight [18%] of 44 patients in the selumetinib group vs none in the placebo group), dyspnoea (one [2%] of 44 patients in the selumetinib group vs five [12%] of 42 in the placebo group), and asthenia (four [9%] of 44 patients in the selumetinib group vs none in the placebo group).  INTERPRETATION:  Selumetinib plus docetaxel has promising efficacy, albeit with a higher number of adverse events than with docetaxel alone, in previously treated advanced KRAS-mutant NSCLC.  These findings warrant further clinical investigation of selumetinib plus docetaxel in KRAS-mutant NSCLC.  FUNDING:  AstraZeneca.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR_DnSQjq4ctlm8bhe6roZYfW6udTcc2earuiSEIfRMJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7otVCnug%253D%253D&md5=3987f8d7cb411810eb05c507a0bc9a73</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2812%2970489-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252812%252970489-8%26sid%3Dliteratum%253Aachs%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DShaw%26aufirst%3DA.%2BT.%26aulast%3DPereira%26aufirst%3DJ.%2BR.%26aulast%3DJeannin%26aufirst%3DG.%26aulast%3DVansteenkiste%26aufirst%3DJ.%26aulast%3DBarrios%26aufirst%3DC.%26aulast%3DFranke%26aufirst%3DF.%2BA.%26aulast%3DGrinsted%26aufirst%3DL.%26aulast%3DZazulina%26aufirst%3DV.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DI.%26aulast%3DCrino%26aufirst%3DL.%26atitle%3DSelumetinib%2520plus%2520docetaxel%2520for%2520KRAS-mutant%2520advanced%2520non-small-cell%2520lung%2520cancer%253A%2520a%2520randomised%252C%2520multicentre%252C%2520placebo-controlled%252C%2520phase%25202%2520study%26jtitle%3DLancet%2520Oncol.%26date%3D2013%26volume%3D14%26spage%3D38%26epage%3D47%26doi%3D10.1016%2FS1470-2045%2812%2970489-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span> </span><span class="NLM_article-title">MEK Inhibitor PD-325901 To Treat Advanced
Breast Cancer, Colon Cancer,
And Melanoma</span>. <a href="https://clinicaltrials.gov/ct2/show/NCT00147550" class="extLink">https://clinicaltrials.gov/ct2/show/NCT00147550</a> (accessed Sep 7, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MEK+Inhibitor+PD-325901+To+Treat+Advanced%0ABreast+Cancer%2C+Colon+Cancer%2C%0AAnd+Melanoma.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT00147550+%28accessed+Sep+7%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DMEK%2520Inhibitor%2520PD-325901%2520To%2520Treat%2520Advanced%250ABreast%2520Cancer%252C%2520Colon%2520Cancer%252C%250AAnd%2520Melanoma%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, A. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uddin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunkel, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petriccione, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haque, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenblum, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisapia, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratilas, C. A.</span></span> <span> </span><span class="NLM_article-title">A secondary mutation in BRAF confers resistance to RAF inhibition in a BRAF(V600E)-mutant brain tumor</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1130</span>– <span class="NLM_lpage">1141</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-17-1263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F2159-8290.CD-17-1263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29880583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFyju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1130-1141&author=J.+Wangauthor=Z.+Yaoauthor=P.+Jonssonauthor=A.+N.+Allenauthor=A.+C.+R.+Qinauthor=S.+Uddinauthor=I.+J.+Dunkelauthor=M.+Petriccioneauthor=K.+Manovaauthor=S.+Haqueauthor=M.+K.+Rosenblumauthor=D.+J.+Pisapiaauthor=N.+Rosenauthor=B.+S.+Taylorauthor=C.+A.+Pratilas&title=A+secondary+mutation+in+BRAF+confers+resistance+to+RAF+inhibition+in+a+BRAF%28V600E%29-mutant+brain+tumor&doi=10.1158%2F2159-8290.CD-17-1263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A seconcitry mutation in BRAF confers resistanie to RAF inhibition in a BRAFV600E-mutant brain tumor</span></div><div class="casAuthors">Wang, Jiawan; Yao, Zhan; Jonsson, Philip; Allen, Amy N.; Qin, Alice Can Ran; Uddin, Sharnieen; Dunke, Ira J.; Petriccione, Mary; Manova, Katia; Hague, Sofia; Rosenblum, Marc K.; Pisapia, David J.; Rosen, Neal; Taylor, Barry S.; Pratilas, Christine A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1130-1141</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">BRAFV600E hyperactivates ERK and signals as a RAF inhibitor-sensitive monomer.  Although RAF inhibitors can produce impressive clin. responses in patients with mutant BRAF tumors, the mechanisms of resistance to these drugs are incompletely characterized.  Here, we report a complete response followed by clin. progression in a patient with a BRAFV600E-mutant brain tumor treated with dabrafenib.  Whole-exome sequencing revealed a secondary BRAFL514V mutation at progression that was not present in the pretreatment tumor.  Expressing BRAFV600E/L514V induces ERK signaling, promotes RAF dimer formation, and is sufficient to confer resistance to dabrafenib.  Newer RAF dimer inhibitors and an ERK inhibitor are effective against BRAFL514V-mediated resistance.  Collectively, our results validate a novel biochem. mechanism of RAF inhibitor resistance mediated by a secondary mutation, emphasizing that, like driver mutations in cancer, the spectrum of mutations that drive resist- ance to targeted therapy are heterogeneous and perhaps emerge with a lineage-specific prevalence.  SIGNIFICANCE: In contrast to receptor tyrosine kinases, in which secondary mutations are often responsible for acquired resistance, second-site mutations in BRAF have not been validated in clini- cally acquired resistance to RAF inhibitors.  We demonstrate a secondary mutation in BRAF (V600E/ L514V) following progression on dabrafenib and confirm functionally that this mutation is responsible for resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV9HqL6aWJ5bVg90H21EOLACvtfcHk0lh7gQiLi4NbFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFyju74%253D&md5=98359fdb20ec6b0f4e94f53eaf30d5eb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-1263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-1263%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DJonsson%26aufirst%3DP.%26aulast%3DAllen%26aufirst%3DA.%2BN.%26aulast%3DQin%26aufirst%3DA.%2BC.%2BR.%26aulast%3DUddin%26aufirst%3DS.%26aulast%3DDunkel%26aufirst%3DI.%2BJ.%26aulast%3DPetriccione%26aufirst%3DM.%26aulast%3DManova%26aufirst%3DK.%26aulast%3DHaque%26aufirst%3DS.%26aulast%3DRosenblum%26aufirst%3DM.%2BK.%26aulast%3DPisapia%26aufirst%3DD.%2BJ.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DTaylor%26aufirst%3DB.%2BS.%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26atitle%3DA%2520secondary%2520mutation%2520in%2520BRAF%2520confers%2520resistance%2520to%2520RAF%2520inhibition%2520in%2520a%2520BRAF%2528V600E%2529-mutant%2520brain%2520tumor%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D1130%26epage%3D1141%26doi%3D10.1158%2F2159-8290.CD-17-1263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koya, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriceau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hugo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chodon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kefford, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, R. S.</span></span> <span> </span><span class="NLM_article-title">A novel AKT1 mutant amplifies an adaptive melanoma response to BRAF inhibition</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-13-0279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F2159-8290.CD-13-0279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=24265152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVCmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=69-79&author=H.+Shiauthor=A.+Hongauthor=X.+Kongauthor=R.+C.+Koyaauthor=C.+Songauthor=G.+Moriceauauthor=W.+Hugoauthor=C.+C.+Yuauthor=C.+Ngauthor=T.+Chodonauthor=R.+A.+Scolyerauthor=R.+F.+Keffordauthor=A.+Ribasauthor=G.+V.+Longauthor=R.+S.+Lo&title=A+novel+AKT1+mutant+amplifies+an+adaptive+melanoma+response+to+BRAF+inhibition&doi=10.1158%2F2159-8290.CD-13-0279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel AKT1 Mutant Amplifies an Adaptive Melanoma Response to BRAF Inhibition</span></div><div class="casAuthors">Shi, Hubing; Hong, Aayoung; Kong, Xiangju; Koya, Richard C.; Song, Chunying; Moriceau, Gatien; Hugo, Willy; Yu, Clarissa C.; Ng, Charles; Chodon, Thinle; Scolyer, Richard A.; Kefford, Richard F.; Ribas, Antoni; Long, Georgina V.; Lo, Roger S.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-79</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">BRAF inhibitor (BRAFi) therapy leads to remarkable anti melanoma responses, but the initial tumor shrinkage is commonly incomplete, providing a nidus for subsequent disease progression.  Adaptive signaling may underlie early BRAFi resistance and influence the selection pattern for genetic variants, causing late, acquired resistance.  We show here that BRAFi (or BRAFi + MEKi) therapy in patients frequently led to rebound phosphorylated AKT (p-AKT) levels in their melanomas early on-treatment.  In cell lines, BRAFi treatment led to rebound levels of receptor tyrosine kinases (RTK; including PDGFβ), phosphatidyl (3,4,5)-triphosphate (PIP3), pleckstrin homol. domain recruitment, and p-AKT.  PTEN expression limited this BRAFi-elicited PI3K-AKT signaling, which could be rescued by the introduction of a mutant AKT1 (Q79K) known to confer acquired BRAFi resistance.  Functionally, AKTIQ79k conferred BRAFi resistance via amplification of BRAFi-elicited P13K-AKT signaling.  In addn., mitogen- activated protein kinase pathway inhibition enhanced clonogenic growth dependency on PI3K or AKT.  Thus, adaptive or genetic upregulation of AKT critically participates in melanoma survival during BRAFi therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWO5YkeSGMcrVg90H21EOLACvtfcHk0lh7gQiLi4NbFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVCmsw%253D%253D&md5=aaf2a9e96591b5f5e2e8180400d0b641</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-13-0279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-13-0279%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DA.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DKoya%26aufirst%3DR.%2BC.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DMoriceau%26aufirst%3DG.%26aulast%3DHugo%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DC.%2BC.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DChodon%26aufirst%3DT.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DKefford%26aufirst%3DR.%2BF.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DLo%26aufirst%3DR.%2BS.%26atitle%3DA%2520novel%2520AKT1%2520mutant%2520amplifies%2520an%2520adaptive%2520melanoma%2520response%2520to%2520BRAF%2520inhibition%26jtitle%3DCancer%2520Discovery%26date%3D2014%26volume%3D4%26spage%3D69%26epage%3D79%26doi%3D10.1158%2F2159-8290.CD-13-0279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lorusso, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adjei, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varterasian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadgeel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLuca, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruzek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asbury, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Becelaere, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. B.</span></span> <span> </span><span class="NLM_article-title">Phase I and pharmacodynamic study of the oral MEK inhibitor CI-1040 in patients with advanced malignancies</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5281</span>– <span class="NLM_lpage">5293</span>, <span class="refDoi"> DOI: 10.1200/JCO.2005.14.415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1200%2FJCO.2005.14.415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=16009947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslOrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=5281-5293&author=P.+M.+Lorussoauthor=A.+A.+Adjeiauthor=M.+Varterasianauthor=S.+Gadgeelauthor=J.+Reidauthor=D.+Y.+Mitchellauthor=L.+Hansonauthor=P.+DeLucaauthor=L.+Bruzekauthor=J.+Piensauthor=P.+Asburyauthor=K.+Van+Becelaereauthor=R.+Herreraauthor=J.+Sebolt-Leopoldauthor=M.+B.+Meyer&title=Phase+I+and+pharmacodynamic+study+of+the+oral+MEK+inhibitor+CI-1040+in+patients+with+advanced+malignancies&doi=10.1200%2FJCO.2005.14.415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I and pharmacodynamic study of the oral MEK inhibitor CI-1040 in patients with advanced malignancies</span></div><div class="casAuthors">LoRusso, Patricia M.; Adjei, Alex A.; Varterasian, Mary; Gadgeel, Shirish; Reid, Joel; Mitchell, David Y.; Hanson, Lorelei; DeLuca, Pamela; Bruzek, Laura; Piens, Jill; Asbury, Peggy; Van Becelaere, Keri; Herrera, Roman; Sebolt-Leopold, Judith; Meyer, Mark B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5281-5293</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose: This phase I study was undertaken to define the toxicity, pharmacokinetics, pharmacodynamics, max. tolerated dose (MTD), and clin. activity of CI-1040, a small-mol. inhibitor of the dual-specificity kinases MEK(mitogen-activated protein kinase kinase) -1 and MEK2, in patients with advanced malignancy.  Patients and Methods: CI-1040 was tested in multiple daily dosing frequencies administered for 21 days repeated every 28 days leading ultimately to continuous administration, and effect of food on absorption was tested.  Single dose and steady-state pharmacokinetics were assessed during cycle 1 and phosphorylated extracellular receptor kinase (pERK) levels were assessed in WBCs and also in tumor tissue from selected patients.  Results: Seventy-seven patients received CI-1040 at dose levels ranging from 100 mg QD to 800 mg tid.  Grade 3 asthenia was dose limiting at the highest dose level tested, 800 mg tid administered with food.  Ninety-eight percent of all drug-related adverse events were grade 1 or 2 in severity; most common toxicities included diarrhea, asthenia, rash, nausea, and vomiting.  Plasma concns. of CI-1040 and its active metabolite, PD 0184264, increased in a less than dose proportional manner from 100 to 800 mg QD.  Administration with a high-fat meal resulted in an increase in drug exposure.  The MTD and recommended phase II dose was 800 mg BID administered with food.  Sixty-six patients were assessable for response.  One partial response was achieved in a patient with pancreatic cancer and 19 patients (28%) achieved stable disease lasting a median of 5.5 mo (range, 4 to 17 mo).  Inhibition of tumor pERK (median, 73%; range, 46% to 100%) was demonstrated in 10 patients.  Conclusion: CI-1040 was well tolerated at 800 mg BID administered with food.  Both target suppression and antitumor activity were demonstrated in this phase I study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn9_7twRmM4LVg90H21EOLACvtfcHk0lgeZYaomLOAwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslOrtL8%253D&md5=6af34b8a2037ede667d91e6a3e884d87</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1200%2FJCO.2005.14.415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2005.14.415%26sid%3Dliteratum%253Aachs%26aulast%3DLorusso%26aufirst%3DP.%2BM.%26aulast%3DAdjei%26aufirst%3DA.%2BA.%26aulast%3DVarterasian%26aufirst%3DM.%26aulast%3DGadgeel%26aufirst%3DS.%26aulast%3DReid%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DD.%2BY.%26aulast%3DHanson%26aufirst%3DL.%26aulast%3DDeLuca%26aufirst%3DP.%26aulast%3DBruzek%26aufirst%3DL.%26aulast%3DPiens%26aufirst%3DJ.%26aulast%3DAsbury%26aufirst%3DP.%26aulast%3DVan%2BBecelaere%26aufirst%3DK.%26aulast%3DHerrera%26aufirst%3DR.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DMeyer%26aufirst%3DM.%2BB.%26atitle%3DPhase%2520I%2520and%2520pharmacodynamic%2520study%2520of%2520the%2520oral%2520MEK%2520inhibitor%2520CI-1040%2520in%2520patients%2520with%2520advanced%2520malignancies%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2005%26volume%3D23%26spage%3D5281%26epage%3D5293%26doi%3D10.1200%2FJCO.2005.14.415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wabnitz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wabnitz, D. A.</span></span> <span> </span><span class="NLM_article-title">In vitro and in vivo metabolism of the anti-cancer agent CI-1040, a MEK inhibitor, in rat, monkey, and human</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1670</span>– <span class="NLM_lpage">1679</span>, <span class="refDoi"> DOI: 10.1023/B:PHAM.0000041464.27579.d0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1023%2FB%3APHAM.0000041464.27579.d0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=15497695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFOqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2004&pages=1670-1679&author=P.+A.+Wabnitzauthor=D.+Mitchellauthor=D.+A.+Wabnitz&title=In+vitro+and+in+vivo+metabolism+of+the+anti-cancer+agent+CI-1040%2C+a+MEK+inhibitor%2C+in+rat%2C+monkey%2C+and+human&doi=10.1023%2FB%3APHAM.0000041464.27579.d0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro and in Vivo Metabolism of the Anti-Cancer Agent CI-1040, a MEK Inhibitor, in Rat, Monkey, and Human</span></div><div class="casAuthors">Wabnitz, Paul A.; Mitchell, David; Wabnitz, David A. M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1670-1679</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer Science+Business Media, Inc.</span>)
        </div><div class="casAbstract">Purpose. The use of in vitro and in vivo models using both rodent and nonrodent species plays an important role with regard to metab. during the drug development process.  In this study, the authors compared the metab. of a MEK inhibitor (CI-1040) using in vitro and in vivo models with that obsd. in a cancer patient.  Methods. Radiolabeled CI-1040 was assessed for metab. using rat and monkey liver microsomes and hepatocytes, as well as in Wistar rats and cynomolgus monkeys via oral administration.  Human bile and plasma samples were obtained immediately after administration of CI-1040 to a patient with advanced colon cancer.  A combination of HPLC-radiochromatog. (HPLC-RAM), LC/MS and LC/MS/MS expts. were used to analyze all resulting metabolites.  Unlabeled CI-1040 was administered (100 mg/day, QD) for 15 days to a patient suffering from colon cancer.  Bile was collected by the insertion of a T-tube directly into the bile duct over a 14-h period.  Metabolites were also monitored in the patient's plasma.  Results. Anal. of the metabolites in all species using in vitro and animal models demonstrated that CI-1040 undergoes extensive oxidative metab. (14 metabolites identified) with subsequent glucuronidation of the hydroxylated metabolites.  Metabolites were predominantly excreted through the bile in the animal models.  Conclusions. Overall, the in vitro and animal models in combination provided comprehensive coverage for all metabolites obsd. in human bile and plasma.  In conclusion, the results obtained in this study demonstrate the utility of conducting investigations across species in order to gain complete coverage for successfully predicting human metabolites of new compds. in development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrprxSVj_DzBrVg90H21EOLACvtfcHk0lgeZYaomLOAwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFOqu7w%253D&md5=f1303d36674c1190ecf2213996783edb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1023%2FB%3APHAM.0000041464.27579.d0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253APHAM.0000041464.27579.d0%26sid%3Dliteratum%253Aachs%26aulast%3DWabnitz%26aufirst%3DP.%2BA.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DWabnitz%26aufirst%3DD.%2BA.%26atitle%3DIn%2520vitro%2520and%2520in%2520vivo%2520metabolism%2520of%2520the%2520anti-cancer%2520agent%2520CI-1040%252C%2520a%2520MEK%2520inhibitor%252C%2520in%2520rat%252C%2520monkey%252C%2520and%2520human%26jtitle%3DPharm.%2520Res.%26date%3D2004%26volume%3D21%26spage%3D1670%26epage%3D1679%26doi%3D10.1023%2FB%3APHAM.0000041464.27579.d0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boerner, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">Clinical experience of MEK inhibitors in cancer therapy</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>1773</i></span>,  <span class="NLM_fpage">1248</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2006.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.bbamcr.2006.11.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=17194493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2sXosFyksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1773&publication_year=2007&pages=1248-1255&author=D.+Wangauthor=S.+A.+Boernerauthor=J.+D.+Winklerauthor=P.+M.+LoRusso&title=Clinical+experience+of+MEK+inhibitors+in+cancer+therapy&doi=10.1016%2Fj.bbamcr.2006.11.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical experience of MEK inhibitors in cancer therapy</span></div><div class="casAuthors">Wang, Ding; Boerner, Scott A.; Winkler, James D.; LoRusso, Patricia M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">1773</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1248-1255</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding new therapies to assist in the treatment of cancer is a major challenge of clin. research.  Small mols. that inhibit different mol. targets at the different levels of the MAPK pathway have been developed.  Several MEK inhibitors have been examd. in early-phase clin. trials and the current state of clin. results using these therapies is presented here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8_rSmIaOyh7Vg90H21EOLACvtfcHk0lgeZYaomLOAwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXosFyksbs%253D&md5=93c566b1c3956e11a3a682e6cc9fc000</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2006.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2006.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DD.%26aulast%3DBoerner%26aufirst%3DS.%2BA.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DClinical%2520experience%2520of%2520MEK%2520inhibitors%2520in%2520cancer%2520therapy%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2007%26volume%3D1773%26spage%3D1248%26epage%3D1255%26doi%3D10.1016%2Fj.bbamcr.2006.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudley, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saltiel, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fergus, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamme, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LePage, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leopold, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Becelaere, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marston, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, W. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span> <span> </span><span class="NLM_article-title">The discovery of the benzhydroxamate MEK inhibitors CI-1040 and PD 0325901</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6501</span>– <span class="NLM_lpage">6504</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.10.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.bmcl.2008.10.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=18952427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWjsL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=6501-6504&author=S.+D.+Barrettauthor=A.+J.+Bridgesauthor=D.+T.+Dudleyauthor=A.+R.+Saltielauthor=J.+H.+Fergusauthor=C.+M.+Flammeauthor=A.+M.+Delaneyauthor=M.+Kaufmanauthor=S.+LePageauthor=W.+R.+Leopoldauthor=S.+A.+Przybranowskiauthor=J.+Sebolt-Leopoldauthor=K.+Van+Becelaereauthor=A.+M.+Dohertyauthor=R.+M.+Kennedyauthor=D.+Marstonauthor=W.+A.+Howardauthor=Y.+Smithauthor=J.+S.+Warmusauthor=H.+Tecle&title=The+discovery+of+the+benzhydroxamate+MEK+inhibitors+CI-1040+and+PD+0325901&doi=10.1016%2Fj.bmcl.2008.10.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of the benzhydroxamate MEK inhibitors CI-1040 and PD 0325901</span></div><div class="casAuthors">Barrett, Stephen D.; Bridges, Alexander J.; Dudley, David T.; Saltiel, Alan R.; Fergus, James H.; Flamme, Cathlin M.; Delaney, Amy M.; Kaufman, Michael; LePage, Sophie; Leopold, Wilbur R.; Przybranowski, Sally A.; Sebolt-Leopold, Judith; Van Becelaere, Keri; Doherty, Annette M.; Kennedy, Robert M.; Marston, Dan; Howard, W. Allen, Jr.; Smith, Yvonne; Warmus, Joseph S.; Tecle, Haile</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6501-6504</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of benzhydroxamate esters derived from their precursor anthranilic acids have been prepd. and have been identified as potent MEK inhibitors.  CI-1040 (I) was the first MEK inhibitor to demonstrate in vivo activity in preclin. animal models and subsequently became the first MEK inhibitor to enter clin. trial.  CI-1040 suffered however from poor exposure due to its poor soly. and rapid clearance, and as a result, development of the compd. was terminated.  Optimization of the diphenylamine core and modification of the hydroxamate side chain for cell potency, soly., and exposure with oral delivery resulted in the discovery of the clin. candidate PD 0325901 (II).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpam9NxZxX93rVg90H21EOLACvtfcHk0lhcJyt9FD_-3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWjsL%252FI&md5=e64f7ae6a446c504528621e7d14d67ce</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.10.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.10.054%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DS.%2BD.%26aulast%3DBridges%26aufirst%3DA.%2BJ.%26aulast%3DDudley%26aufirst%3DD.%2BT.%26aulast%3DSaltiel%26aufirst%3DA.%2BR.%26aulast%3DFergus%26aufirst%3DJ.%2BH.%26aulast%3DFlamme%26aufirst%3DC.%2BM.%26aulast%3DDelaney%26aufirst%3DA.%2BM.%26aulast%3DKaufman%26aufirst%3DM.%26aulast%3DLePage%26aufirst%3DS.%26aulast%3DLeopold%26aufirst%3DW.%2BR.%26aulast%3DPrzybranowski%26aufirst%3DS.%2BA.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DVan%2BBecelaere%26aufirst%3DK.%26aulast%3DDoherty%26aufirst%3DA.%2BM.%26aulast%3DKennedy%26aufirst%3DR.%2BM.%26aulast%3DMarston%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DW.%2BA.%26aulast%3DSmith%26aufirst%3DY.%26aulast%3DWarmus%26aufirst%3DJ.%2BS.%26aulast%3DTecle%26aufirst%3DH.%26atitle%3DThe%2520discovery%2520of%2520the%2520benzhydroxamate%2520MEK%2520inhibitors%2520CI-1040%2520and%2520PD%25200325901%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D6501%26epage%3D6504%26doi%3D10.1016%2Fj.bmcl.2008.10.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, R.</span></span> <span> </span><span class="NLM_article-title">Targeting the mitogen-activated protein kinase cascade to treat cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">947</span>, <span class="refDoi"> DOI: 10.1038/nrc1503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fnrc1503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=15573115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=937-947&author=J.+S.+Sebolt-Leopoldauthor=R.+Herrera&title=Targeting+the+mitogen-activated+protein+kinase+cascade+to+treat+cancer&doi=10.1038%2Fnrc1503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the mitogen-activated protein kinase cascade to treat cancer</span></div><div class="casAuthors">Sebolt-Leopold, Judith S.; Herrera, Roman</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">937-947</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The RAS-mitogen activated protein kinase (MAPK) signaling pathway has long been viewed as an attractive pathway for anticancer therapies, based on its central role in regulating the growth and survival of cells from a broad spectrum of human tumors.  Small-mol. inhibitors designed to target various steps of this pathway have entered clin. trials.  The safety and effectiveness of these inhibitors as well as the role of MAPK pathway in the therapeutic intervention are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6UoubyiRaoLVg90H21EOLACvtfcHk0lhcJyt9FD_-3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjO&md5=6b6ead5165e040552dfca68d7a6c0477</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrc1503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1503%26sid%3Dliteratum%253Aachs%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%2BS.%26aulast%3DHerrera%26aufirst%3DR.%26atitle%3DTargeting%2520the%2520mitogen-activated%2520protein%2520kinase%2520cascade%2520to%2520treat%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D937%26epage%3D947%26doi%3D10.1038%2Fnrc1503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span> <span> </span><span class="NLM_article-title">Current development status of MEK inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1551</span>, <span class="refDoi"> DOI: 10.3390/molecules22101551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.3390%2Fmolecules22101551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Wjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1551&author=Y.+Chengauthor=H.+Tian&title=Current+development+status+of+MEK+inhibitors&doi=10.3390%2Fmolecules22101551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Current development status of MEK inhibitors</span></div><div class="casAuthors">Cheng, Ying; Tian, Hongqi</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1551/1-1551/20</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The current development status of mitogen-activated protein kinase kinase (MEK) inhibitors, including the preclin. data and clin. study progress, has been summarized in this review.  Different MEK inhibitors, possessing specific physicochem. properties and bioactivity characteristics, may provide different options for patients seeking treatment for cancer.  Moreover, the combination of the MEK inhibitors with other therapies-such as chemotherapy, targeted therapy, and immunotherapy-may be a promising approach for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosd6s1Lx6dYbVg90H21EOLACvtfcHk0lhcJyt9FD_-3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Wjsrk%253D&md5=0210d125f8a46468d67adb4f9aaf6cf3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fmolecules22101551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules22101551%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DH.%26atitle%3DCurrent%2520development%2520status%2520of%2520MEK%2520inhibitors%26jtitle%3DMolecules%26date%3D2017%26volume%3D22%26spage%3D1551%26doi%3D10.3390%2Fmolecules22101551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreck, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guajardo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhart, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossman, S. A.</span></span> <span> </span><span class="NLM_article-title">Concurrent BRAF/MEK inhibitors in BRAF V600-mutant high-grade primary brain tumors</span>. <i>J. Natl. Compr. Cancer Network</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">347</span>, <span class="refDoi"> DOI: 10.6004/jnccn.2017.7052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.6004%2Fjnccn.2017.7052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29632053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BC1MjgtlOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=343-347&author=K.+C.+Schreckauthor=A.+Guajardoauthor=D.+D.+M.+Linauthor=C.+G.+Eberhartauthor=S.+A.+Grossman&title=Concurrent+BRAF%2FMEK+inhibitors+in+BRAF+V600-mutant+high-grade+primary+brain+tumors&doi=10.6004%2Fjnccn.2017.7052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Concurrent BRAF/MEK Inhibitors in BRAF V600-Mutant High-Grade Primary Brain Tumors</span></div><div class="casAuthors">Schreck Karisa C; Guajardo Andrew; Lin Doris D M; Eberhart Charles G; Grossman Stuart A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Comprehensive Cancer Network : JNCCN</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">343-347</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BRAF V600 mutations are being identified in patients with primary brain tumors more often as molecular testing becomes widely available.  Targeted treatment with BRAF inhibitors has been attempted in individual cases with some responses, whereas others showed no response or developed resistance.  Preclinical work suggests that gliomas could be more responsive to the concurrent use of BRAF and MEK inhibition for MAP kinase pathway suppression.  This report presents 2 cases of malignant brain tumors with BRAF V600E mutations that were resistant to radiation and temozolomide, and reports on their response to targeted treatment with the BRAF and MEK inhibitors dabrafenib and trametinib.  One patient with an anaplastic pleomorphic xanthoastrocytoma experienced a partial response for 14 months, demonstrated by progressive tumor shrinkage and clinical improvement; however, this was followed by clinical and radiographic progression.  The patient with glioblastoma continued to have stable disease after 16 months of treatment.  These cases are encouraging in a disease that urgently needs new treatments.  Further work is necessary to understand response rates, duration, and survival in primary brain tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTLBwzYQSlXeuLt9HypzAgQfW6udTcc2eZiQlwLEbLYYLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjgtlOruw%253D%253D&md5=ba18710a1e0d3a34f47ce7d25a743291</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.6004%2Fjnccn.2017.7052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6004%252Fjnccn.2017.7052%26sid%3Dliteratum%253Aachs%26aulast%3DSchreck%26aufirst%3DK.%2BC.%26aulast%3DGuajardo%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DD.%2BD.%2BM.%26aulast%3DEberhart%26aufirst%3DC.%2BG.%26aulast%3DGrossman%26aufirst%3DS.%2BA.%26atitle%3DConcurrent%2520BRAF%252FMEK%2520inhibitors%2520in%2520BRAF%2520V600-mutant%2520high-grade%2520primary%2520brain%2520tumors%26jtitle%3DJ.%2520Natl.%2520Compr.%2520Cancer%2520Network%26date%3D2018%26volume%3D16%26spage%3D343%26epage%3D347%26doi%3D10.6004%2Fjnccn.2017.7052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimaldi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Festino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanella, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strudel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascierto, P. A.</span></span> <span> </span><span class="NLM_article-title">MEK inhibitors in the treatment of metastatic melanoma and solid tumors</span>. <i>Am. J. Clin. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1007/s40257-017-0292-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1007%2Fs40257-017-0292-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=28537004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BC1cngsFCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=745-754&author=A.+M.+Grimaldiauthor=E.+Simeoneauthor=L.+Festinoauthor=V.+Vanellaauthor=M.+Strudelauthor=P.+A.+Ascierto&title=MEK+inhibitors+in+the+treatment+of+metastatic+melanoma+and+solid+tumors&doi=10.1007%2Fs40257-017-0292-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">MEK Inhibitors in the Treatment of Metastatic Melanoma and Solid Tumors</span></div><div class="casAuthors">Grimaldi Antonio M; Simeone Ester; Festino Lucia; Vanella Vito; Strudel Martina; Ascierto Paolo A</div><div class="citationInfo"><span class="NLM_cas:title">American journal of clinical dermatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">745-754</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The mitogen-activated protein kinase (MAPK) cascade is an intracellular signaling pathway involved in the regulation of cellular proliferation and the survival of tumor cells.  Several different mutations, involving BRAF or NRAS, exert an oncogenic effect by activating the MAPK pathway, resulting in an increase in cellular proliferation.  These mutations have become targets for new therapeutic strategies in melanoma and other cancers.  Selective MEK inhibitors have the ability to inhibit growth and induce cell death in BRAF- and NRAS-mutant melanoma cell lines.  MEK inhibitor therapy in combination with a BRAF inhibitor is more effective and less toxic than treatment with a BRAF inhibitor alone, and has become the standard of care for patients with BRAF-mutated melanoma.  Trametinib was the first MEK inhibitor approved for the treatment of BRAF-mutated metastatic melanoma not previously treated with BRAF inhibitors, and is also approved in combination with the BRAF inhibitor dabrafenib.  Furthermore, cobimetinib is another MEK inhibitor approved for the treatment of BRAF-mutated metastatic melanoma in combination with a BRAF inhibitor, vemurafenib.  The MEK inhibitor binimetinib in combination with the BRAF inhibitor encorafenib is in clinical development.  The addition of an anti-PD-1/PD-L1 agent, such as pembrolizumab, durvalumab or atezolizumab, to combined BRAF and MEK inhibition has shown considerable promise, with several trials ongoing in metastatic melanoma.  Binimetinib has also shown efficacy in NRAS-mutated melanoma patients.  Future possibilities for MEK inhibitors in advanced melanoma, as well as other solid tumors, include their use in combination with other targeted therapies (e.g. anti-CDK4/6 inhibitors) and/or various immune-modulating antibodies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTRMQVeZbuZD8RAV4D_Ins_fW6udTcc2eY3JkDSa-8RAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cngsFCgtA%253D%253D&md5=23b16eb79f7203a8acd54e8c9249357d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs40257-017-0292-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40257-017-0292-y%26sid%3Dliteratum%253Aachs%26aulast%3DGrimaldi%26aufirst%3DA.%2BM.%26aulast%3DSimeone%26aufirst%3DE.%26aulast%3DFestino%26aufirst%3DL.%26aulast%3DVanella%26aufirst%3DV.%26aulast%3DStrudel%26aufirst%3DM.%26aulast%3DAscierto%26aufirst%3DP.%2BA.%26atitle%3DMEK%2520inhibitors%2520in%2520the%2520treatment%2520of%2520metastatic%2520melanoma%2520and%2520solid%2520tumors%26jtitle%3DAm.%2520J.%2520Clin.%2520Dermatol.%26date%3D2017%26volume%3D18%26spage%3D745%26epage%3D754%26doi%3D10.1007%2Fs40257-017-0292-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heppt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tietze, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berking, C.</span></span> <span> </span><span class="NLM_article-title">Combination therapy of melanoma using kinase inhibitors</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1097/CCO.0000000000000160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1097%2FCCO.0000000000000160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25602684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitl2murw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=134-140&author=M.+V.+Hepptauthor=J.+K.+Tietzeauthor=S.+A.+Grafauthor=C.+Berking&title=Combination+therapy+of+melanoma+using+kinase+inhibitors&doi=10.1097%2FCCO.0000000000000160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Combination therapy of melanoma using kinase inhibitors</span></div><div class="casAuthors">Heppt, Markus V.; Tietze, Julia K.; Graf, Saskia A.; Berking, Carola</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-140</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: Treatment options for metastatic melanoma depend on the clin. course of the disease and the mol. profile such as mutations of the BRAF gene.  In this article, we review the current state of targeted therapy with kinase inhibitors.  Recent findings: Despite major advancements in targeted therapy of metastatic melanoma, most patients relapse and show progressive disease after 5-7 mo with single inhibition of BRAF or MEK.  Acquired resistance is virtually universal and mediated by diverse mitogen-activated protein kinase-dependent or independent mechanisms.  Recent evidence favors concurrent targeting of BRAF and MEK in patients with BRAFV600-mutated melanoma instead of BRAF inhibitor monotherapy.  The combination delays the onset of acquired resistance, resulting in increased progression-free and overall survival.  A growing no. of early trials evaluate the efficacy of inhibitors targeting addnl. pathways such as phospho-inositide 3-kinase/AKT in conjunction with BRAF or MEK.  Even though consistent and mature phase III study results are not yet available for these combinations, the repertoire of targeted therapy in metastatic melanoma is wide and promising.  Summary: The short era of single BRAF inhibition in BRAF-mutated melanoma is soon taken over by dual concurrent inhibition of MEK and BRAF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1rD_f2NR8yLVg90H21EOLACvtfcHk0liWZVV6Z2Ba4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitl2murw%253D&md5=b5f810e41c451a194daa09bdcbdb2151</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1097%2FCCO.0000000000000160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0000000000000160%26sid%3Dliteratum%253Aachs%26aulast%3DHeppt%26aufirst%3DM.%2BV.%26aulast%3DTietze%26aufirst%3DJ.%2BK.%26aulast%3DGraf%26aufirst%3DS.%2BA.%26aulast%3DBerking%26aufirst%3DC.%26atitle%3DCombination%2520therapy%2520of%2520melanoma%2520using%2520kinase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2015%26volume%3D27%26spage%3D134%26epage%3D140%26doi%3D10.1097%2FCCO.0000000000000160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kefford, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Infante, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fecher, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millward, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McArthur, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMarini, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paterl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K. D.</span></span> <span> </span><span class="NLM_article-title">Phase II study of the MEK1/MEK2 inhibitor trametinib in patients with metastatic BRAF-mutant cutaneous melanoma previously treated with or without a BRAF inhibitor</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">482</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1200/JCO.2012.43.5966</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1200%2FJCO.2012.43.5966" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=23248257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Sktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=482-489&author=K.+B.+Kimauthor=R.+Keffordauthor=A.+C.+Pavlickauthor=J.+R.+Infanteauthor=A.+Ribasauthor=J.+A.+Sosmanauthor=L.+A.+Fecherauthor=A.+Millwardauthor=G.+A.+McArthurauthor=P.+Hwuauthor=R.+Gonzalezauthor=P.+A.+Ottauthor=G.+V.+Longauthor=O.+S.+Gardnerauthor=D.+Ouelletauthor=Y.+Xuauthor=D.+J.+DeMariniauthor=N.+T.+Leauthor=K.+Paterlauthor=K.+D.+Lewis&title=Phase+II+study+of+the+MEK1%2FMEK2+inhibitor+trametinib+in+patients+with+metastatic+BRAF-mutant+cutaneous+melanoma+previously+treated+with+or+without+a+BRAF+inhibitor&doi=10.1200%2FJCO.2012.43.5966"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Phase II study of the MEK1/MEK2 inhibitor trametinib in patients with metastatic BRAF-mutant cutaneous melanoma previously treated with or without a BRAF inhibitor</span></div><div class="casAuthors">Kim, Kevin B.; Kefford, Richard; Pavlick, Anna C.; Infante, Jeffrey R.; Ribas, Antoni; Sosman, Jeffrey A.; Fecher, Leslie A.; Millward, Michael; McArthur, Grant A.; Hwu, Patrick; Gonzalez, Rene; Ott, Patrick A.; Long, Georgina V.; Gardner, Olivia S.; Ouellet, Daniele; Xu, Yanmei; DeMarini, Douglas J.; Le, Ngocdiep T.; Patel, Kiran; Lewis, Karl D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">482-489</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose: BRAF mutations promote melanoma cell proliferation and survival primarily through activation of MEK.  The purpose of this study was to det. the response rate (RR) for the selective, allosteric MEK1/MEK2 inhibitor trametinib (GSK1120212), in patients with metastatic BRAF-mutant melanoma.  Patients and Methods: This was an open-label, two-stage, phase II study with two cohorts.  Patients with metastatic BRAF-mutant melanoma previously treated with a BRAF inhibitor (cohort A) or treated with chemotherapy and/or immunotherapy (BRAF-inhibitor naive; cohort B) were enrolled.  Patients: received 2 mg of trametinib orally once daily.  Results: In cohort A (n = 40), there were no confirmed objective responses and 11 patients (28%) with stable disease (SD); the median progression-free survival (PFS) was 1.8 mo.  In cohort B (n = 57), there was one (2%) complete response, 13 (23%) partial responses (PRs), and 29 patients (51%) with SD (confirmed RR, 25%); the median PFS was 4.0 mo.  One patient each with BRAF K601E and BRAF V600R had prolonged PR.  The most frequent treatment-related adverse events for all patients were skin-related toxicity, nausea, peripheral edema, diarrhea, pruritis, and fatigue.  No cutaneous squamous cell carcinoma was obsd.  Conclusion Trametinib was well tolerated.  Significant clin. activity was obsd. in BRAF-inhibitor-naive patients previously treated with chemotherapy and/or immunotherapy.  Minimal clin. activity was obsd. as sequential therapy in patients previously treated with a BRAF inhibitor.  Together, these data suggest that BRAF-inhibitor resistance mechanisms likely confer resistance to MEK-inhibitor monotherapy.  These data support further evaluation of trametinib in BRAF-inhibitor-naive BRAF-mutant melanoma, including rarer forms of BRAF-mutant melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeb_z2i1Ifq7Vg90H21EOLACvtfcHk0liWZVV6Z2Ba4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Sktbc%253D&md5=dfa4cd44e8a9fc3b66e9c260af829648</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1200%2FJCO.2012.43.5966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2012.43.5966%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKefford%26aufirst%3DR.%26aulast%3DPavlick%26aufirst%3DA.%2BC.%26aulast%3DInfante%26aufirst%3DJ.%2BR.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DSosman%26aufirst%3DJ.%2BA.%26aulast%3DFecher%26aufirst%3DL.%2BA.%26aulast%3DMillward%26aufirst%3DA.%26aulast%3DMcArthur%26aufirst%3DG.%2BA.%26aulast%3DHwu%26aufirst%3DP.%26aulast%3DGonzalez%26aufirst%3DR.%26aulast%3DOtt%26aufirst%3DP.%2BA.%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DGardner%26aufirst%3DO.%2BS.%26aulast%3DOuellet%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeMarini%26aufirst%3DD.%2BJ.%26aulast%3DLe%26aufirst%3DN.%2BT.%26aulast%3DPaterl%26aufirst%3DK.%26aulast%3DLewis%26aufirst%3DK.%2BD.%26atitle%3DPhase%2520II%2520study%2520of%2520the%2520MEK1%252FMEK2%2520inhibitor%2520trametinib%2520in%2520patients%2520with%2520metastatic%2520BRAF-mutant%2520cutaneous%2520melanoma%2520previously%2520treated%2520with%2520or%2520without%2520a%2520BRAF%2520inhibitor%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2013%26volume%3D31%26spage%3D482%26epage%3D489%26doi%3D10.1200%2FJCO.2012.43.5966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dummer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascierto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arance, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liszkay, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadendorf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajsova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutzmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sileni, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutriaux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groot, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loquai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parseval, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickard, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandor, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, K. T.</span></span> <span> </span><span class="NLM_article-title">Overall survival in patients with BRAF-mutant melanoma receiving encorafenib plus binimetinib versus vemurafenib or encorafenib (columbus): a multicentre, open-label, randomised, Phase 3 Trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1315</span>– <span class="NLM_lpage">1327</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(18)30497-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1470-2045%2818%2930497-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=30219628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWgsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1315-1327&author=R.+Dummerauthor=P.+Asciertoauthor=H.+Gogasauthor=A.+Aranceauthor=M.+Mandalaauthor=G.+Liszkayauthor=C.+Garbeauthor=D.+Schadendorfauthor=I.+Krajsovaauthor=R.+Gutzmerauthor=V.+C.+Sileniauthor=C.+Dutriauxauthor=J.+W.+Grootauthor=N.+Yamazakiauthor=C.+Loquaiauthor=L.+Parsevalauthor=M.+D.+Pickardauthor=V.+Sandorauthor=C.+Robertauthor=K.+T.+Flaherty&title=Overall+survival+in+patients+with+BRAF-mutant+melanoma+receiving+encorafenib+plus+binimetinib+versus+vemurafenib+or+encorafenib+%28columbus%29%3A+a+multicentre%2C+open-label%2C+randomised%2C+Phase+3+Trial&doi=10.1016%2FS1470-2045%2818%2930497-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Overall survival in patients with BRAF-mutant melanoma receiving encorafenib plus binimetinib versus vemurafenib or encorafenib (COLUMBUS): a multicentre, open-label, randomised, phase 3 trial</span></div><div class="casAuthors">Dummer, Reinhard; Ascierto, Paolo A.; Gogas, Helen J.; Arance, Ana; Mandala, Mario; Liszkay, Gabriella; Garbe, Claus; Schadendorf, Dirk; Krajsova, Ivana; Gutzmer, Ralf; Chiarion Sileni, Vanna; Dutriaux, Caroline; de Groot, Jan Willem B.; Yamazaki, Naoya; Loquai, Carmen; Moutouh-de Parseval, Laure A.; Pickard, Michael D.; Sandor, Victor; Robert, Caroline; Flaherty, Keith T.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1315-1327</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Encorafenib plus binimetinib and encorafenib alone improved progression-free survival compared with vemurafenib in patients with BRAFV600-mutant melanoma in the COLUMBUS trial.  Here, we report the results of the secondary endpoint of overall survival.  COLUMBUS was a two-part, randomised, open-label, phase 3 study done at 162 hospitals in 28 countries.  Eligible patients were aged at least 18 years with histol. confirmed, locally advanced, unresectable, or metastatic cutaneous melanoma, or unknown primary melanoma, BRAFV600E or BRAFV600K mutation, an Eastern Cooperative Oncol. Group (ECOG) performance status of 0 or 1, and were treatment naive or had progressed on or after first-line immunotherapy.  In part 1 of the study, patients were randomly assigned (1:1:1) by use of interactive response technol. to receive oral encorafenib 450 mg once daily plus oral binimetinib 45 mg twice daily (encorafenib plus binimetinib group), oral encorafenib 300 mg once daily (encorafenib group), or oral vemurafenib 960 mg twice daily (vemurafenib group).  Randomisation was stratified by the American Joint Committee on Cancer stage, ECOG performance status, and BRAF mutation status.  The primary outcome of the trial, progression-free survival with encorafenib plus binimetinib vs. vemurafenib, was reported previously.  Here we present the prespecified interim overall survival anal.  Efficacy analyses were by intent to treat.  Safety was analyzed in patients who received at least one dose of study drug.  Part 2 of the study was initiated at the request of the US Food and Drug Administration to better understand the contribution of binimetinib to the combination therapy by comparing encorafenib 300 mg once daily plus binimetinib 45 mg twice daily with encorafenib 300 mg once daily alone.  Results of part 2 will be published sep.  This trial is ongoing and is registered with ClinicalTrials.gov, no. NCT01909453, and EudraCT, no. 2013-001176-38.  Between Dec 30, 2013, and Apr. 10, 2015, 577 of 1345 screened patients were randomly assigned to receive encorafenib plus binimetinib (n=192), encorafenib (n=194), or vemurafenib (n=191).  Median follow-up for overall survival was 36·8 mo (95% CI 35·9-37·5).  Median overall survival was 33·6 mo (95% CI 24·4-39·2) with encorafenib plus binimetinib and 16·9 mo (14·0-24·5) with vemurafenib (hazard ratio 0·61 [95% CI 0·47-0·79]; two-sided p<0·0001).  The most common grade 3 or 4 adverse events did not change substantially from the first report; those seen in more than 5% of patients treated with encorafenib plus binimetinib were increased γ-glutamyltransferase (18 [9%] of 192 patients), increased blood creatine phosphokinase (14 [7%]), and hypertension (12 [6%]); those seen with encorafenib alone were palmar-plantar erythrodysaesthesia syndrome (26 [14%] of 192 patients), myalgia (19 [10%]), and arthralgia (18 [9%]); and with vemurafenib the most common grade 3 or 4 adverse event was arthralgia (11 [6%] of 186 patients).  One death in the combination treatment group was considered by the investigator to be possibly related to treatment.  The combination of encorafenib plus binimetinib provided clin. meaningful efficacy with good tolerability as shown by improvements in both progression-free survival and overall survival compared with vemurafenib.  These data suggest that the combination of encorafenib plus binimetinib is likely to become an important therapeutic option in patients with BRAFV600-mutant melanoma.Array BioPharma, Novartis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofULG8QpgPeLVg90H21EOLACvtfcHk0lix2kx9iLxhkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWgsLjJ&md5=e376456c2363459c0a96b32d954260b0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2818%2930497-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252818%252930497-2%26sid%3Dliteratum%253Aachs%26aulast%3DDummer%26aufirst%3DR.%26aulast%3DAscierto%26aufirst%3DP.%26aulast%3DGogas%26aufirst%3DH.%26aulast%3DArance%26aufirst%3DA.%26aulast%3DMandala%26aufirst%3DM.%26aulast%3DLiszkay%26aufirst%3DG.%26aulast%3DGarbe%26aufirst%3DC.%26aulast%3DSchadendorf%26aufirst%3DD.%26aulast%3DKrajsova%26aufirst%3DI.%26aulast%3DGutzmer%26aufirst%3DR.%26aulast%3DSileni%26aufirst%3DV.%2BC.%26aulast%3DDutriaux%26aufirst%3DC.%26aulast%3DGroot%26aufirst%3DJ.%2BW.%26aulast%3DYamazaki%26aufirst%3DN.%26aulast%3DLoquai%26aufirst%3DC.%26aulast%3DParseval%26aufirst%3DL.%26aulast%3DPickard%26aufirst%3DM.%2BD.%26aulast%3DSandor%26aufirst%3DV.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DFlaherty%26aufirst%3DK.%2BT.%26atitle%3DOverall%2520survival%2520in%2520patients%2520with%2520BRAF-mutant%2520melanoma%2520receiving%2520encorafenib%2520plus%2520binimetinib%2520versus%2520vemurafenib%2520or%2520encorafenib%2520%2528columbus%2529%253A%2520a%2520multicentre%252C%2520open-label%252C%2520randomised%252C%2520Phase%25203%2520Trial%26jtitle%3DLancet%2520Oncol.%26date%3D2018%26volume%3D19%26spage%3D1315%26epage%3D1327%26doi%3D10.1016%2FS1470-2045%2818%2930497-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span> </span><span class="NLM_article-title">FDA Approves Encorafenib and Binimetinib
in Combination for Unresectable
or Metastatic Melanoma with BRAF Mutations</span>. <a href="https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf" class="extLink">https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf</a> (accessed Sep 7, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FDA+Approves+Encorafenib+and+Binimetinib%0Ain+Combination+for+Unresectable%0Aor+Metastatic+Melanoma+with+BRAF+Mutations.+https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fresources-information-approved-drugs%2Ffda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf+%28accessed+Sep+7%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DFDA%2520Approves%2520Encorafenib%2520and%2520Binimetinib%250Ain%2520Combination%2520for%2520Unresectable%250Aor%2520Metastatic%2520Melanoma%2520with%2520BRAF%2520Mutations%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Planchard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besse, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groen, H. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souquet, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quoix, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baik, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlesi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazieres, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novello, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upalawanna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Amelio, A. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mookerjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span> <span> </span><span class="NLM_article-title">Dabrafenib plus trametinib in patients with previously treated BRAF(V600E)-mutant metastatic non-small cell lung cancer: an open-label, multicentre phase 2 trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">984</span>– <span class="NLM_lpage">993</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(16)30146-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1470-2045%2816%2930146-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=27283860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslyjsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=984-993&author=D.+Planchardauthor=B.+Besseauthor=H.+J.+M.+Groenauthor=P.+J.+Souquetauthor=E.+Quoixauthor=C.+S.+Baikauthor=F.+Barlesiauthor=T.+M.+Kimauthor=J.+Mazieresauthor=S.+Novelloauthor=J.+R.+Rigasauthor=A.+Upalawannaauthor=A.+M.+D%E2%80%99Amelioauthor=P.+Zhangauthor=B.+Mookerjeeauthor=B.+E.+Johnson&title=Dabrafenib+plus+trametinib+in+patients+with+previously+treated+BRAF%28V600E%29-mutant+metastatic+non-small+cell+lung+cancer%3A+an+open-label%2C+multicentre+phase+2+trial&doi=10.1016%2FS1470-2045%2816%2930146-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dabrafenib plus trametinib in patients with previously treated BRAFV600E-mutant metastatic non-small cell lung cancer: an open-label, multicentre phase 2 trial</span></div><div class="casAuthors">Planchard, David; Besse, Benjamin; Groen, Harry J. M.; Souquet, Pierre-Jean; Quoix, Elisabeth; Baik, Christina S.; Barlesi, Fabrice; Kim, Tae Min; Mazieres, Julien; Novello, Silvia; Rigas, James R.; Upalawanna, Allison; D'Amelio, Anthony M. Jr; Zhang, Pingkuan; Mookerjee, Bijoyesh; Johnson, Bruce E.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">984-993</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRAF mutations act as an oncogenic driver via the mitogen-activated protein kinase (MAPK) pathway in non-small cell lung cancer (NSCLC).  BRAF inhibition has shown antitumor activity in patients with BRAFV600E-mutant NSCLC.  Dual MAPK pathway inhibition with BRAF and MEK inhibitors in BRAFV600E-mutant NSCLC might improve efficacy over BRAF inhibitor monotherapy based on observations in BRAFV600-mutant melanoma.  We aimed to assess the antitumor activity and safety of dabrafenib plus trametinib in patients with BRAFV600E-mutant NSCLC.  In this phase 2, multicentre, non-randomized, open-label study, we enrolled adult patients (aged ≥18 years) with pretreated metastatic stage IV BRAFV600E-mutant NSCLC who had documented tumor progression after at least one previous platinum-based chemotherapy and had had no more than three previous systemic anticancer therapies.  Patients with previous BRAF or MEK inhibitor treatment were ineligible.  Patients with brain metastases were allowed to enroll only if the lesions were asymptomatic, untreated (or stable more than 3 wk after local therapy if treated), and measured less than 1 cm.  Enrolled patients received oral dabrafenib (150 mg twice daily) plus oral trametinib (2 mg once daily) in continuous 21-day cycles until disease progression, unacceptable adverse events, withdrawal of consent, or death.  The primary endpoint was investigator-assessed overall response, which was assessed by intention to treat in the protocol-defined population (patients who received second-line or later treatment); safety was also assessed in this population and was assessed at least once every 3 wk, with adverse events, lab. values, and vital signs graded according to the Common Terminol. Criteria for Adverse Events version 4.0.  The study is ongoing but no longer recruiting patients.  This trial is registered with ClinicalTrials.gov, no. NCT01336634.  Between Dec 20, 2013, and Jan 14, 2015, 59 patients from 30 centers in nine countries across North America, Europe, and Asia met eligibility criteria.  Two patients who had previously been untreated due to protocol deviation were excluded; thus, 57 eligible patients were enrolled. 36 patients (63·2% [95% CI 49·3-75·6]) achieved an investigator-assessed overall response.  Serious adverse events were reported in 32 (56%) of 57 patients and included pyrexia in nine (16%), anemia in three (5%), confusional state in two (4%), decreased appetite in two (4%), hemoptysis in two (4%), hypercalcemia in two (4%), nausea in two (4%), and cutaneous squamous cell carcinoma in two (4%).  The most common grade 3-4 adverse events were neutropenia in five patients (9%), hyponatremia in four (7%), and anemia in three (5%).  Four patients died during the study from fatal adverse events judged to be unrelated to treatment (one retroperitoneal hemorrhage, one subarachnoid hemorrhage, one respiratory distress, and one from disease progression that was more severe than typical progression, as assessed by the investigator).  Dabrafenib plus trametinib could represent a new targeted therapy with robust antitumor activity and a manageable safety profile in patients with BRAFV600E-mutant NSCLC.GlaxoSmithKline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-qPV5AvwdnrVg90H21EOLACvtfcHk0lix2kx9iLxhkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslyjsL8%253D&md5=322443578a05f7f30ed21427413b38ca</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2816%2930146-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252816%252930146-2%26sid%3Dliteratum%253Aachs%26aulast%3DPlanchard%26aufirst%3DD.%26aulast%3DBesse%26aufirst%3DB.%26aulast%3DGroen%26aufirst%3DH.%2BJ.%2BM.%26aulast%3DSouquet%26aufirst%3DP.%2BJ.%26aulast%3DQuoix%26aufirst%3DE.%26aulast%3DBaik%26aufirst%3DC.%2BS.%26aulast%3DBarlesi%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DT.%2BM.%26aulast%3DMazieres%26aufirst%3DJ.%26aulast%3DNovello%26aufirst%3DS.%26aulast%3DRigas%26aufirst%3DJ.%2BR.%26aulast%3DUpalawanna%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Amelio%26aufirst%3DA.%2BM.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DMookerjee%26aufirst%3DB.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26atitle%3DDabrafenib%2520plus%2520trametinib%2520in%2520patients%2520with%2520previously%2520treated%2520BRAF%2528V600E%2529-mutant%2520metastatic%2520non-small%2520cell%2520lung%2520cancer%253A%2520an%2520open-label%252C%2520multicentre%2520phase%25202%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2016%26volume%3D17%26spage%3D984%26epage%3D993%26doi%3D10.1016%2FS1470-2045%2816%2930146-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pupo, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahheydari, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tembe, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saw, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kefford, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizos, H.</span></span> <span> </span><span class="NLM_article-title">Increased MAPK reactivation in early resistance to dabrafenib/trametinib combination therapy of BRAF-mutant metastatic melanoma</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">5694</span> <span class="refDoi"> DOI: 10.1038/ncomms6694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fncomms6694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25452114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVentb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&author=G.+V.+Longauthor=C.+Fungauthor=A.+M.+Menziesauthor=G.+M.+Pupoauthor=M.+S.+Carlinoauthor=J.+Hymanauthor=H.+Shahheydariauthor=V.+Tembeauthor=J.+F.+Thompsonauthor=R.+P.+Sawauthor=J.+Howleauthor=N.+K.+Haywardauthor=P.+Johanssonauthor=R.+A.+Scolyerauthor=R.+F.+Keffordauthor=H.+Rizos&title=Increased+MAPK+reactivation+in+early+resistance+to+dabrafenib%2Ftrametinib+combination+therapy+of+BRAF-mutant+metastatic+melanoma&doi=10.1038%2Fncomms6694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Increased MAPK reactivation in early resistance to dabrafenib/trametinib combination therapy of BRAF-mutant metastatic melanoma</span></div><div class="casAuthors">Long, Georgina V.; Fung, Carina; Menzies, Alexander M.; Pupo, Gulietta M.; Carlino, Matteo S.; Hyman, Jessica; Shahheydari, Hamideh; Tembe, Varsha; Thompson, John F.; Saw, Robyn P.; Howle, Julie; Hayward, Nicholas K.; Johansson, Peter; Scolyer, Richard A.; Kefford, Richard F.; Rizos, Helen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5694</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">One-third of BRAF-mutant metastatic melanoma patients treated with combined BRAF and MEK inhibition progress within 6 mo.  Treatment options for these patients remain limited.  Here we analyze 20 BRAFV600-mutant melanoma metastases derived from 10 patients treated with the combination of dabrafenib and trametinib for resistance mechanisms and genetic correlates of response.  Resistance mechanisms are identified in 9/11 progressing tumors and MAPK reactivation occurred in 9/10 tumors, commonly via BRAF amplification and mutations activating NRAS and MEK2.  Our data confirming that MEK2C125S, but not the synonymous MEK1C121S protein, confers resistance to combination therapy highlight the functional differences between these kinases and the preponderance of MEK2 mutations in combination therapy-resistant melanomas.  Exome sequencing did not identify addnl. progression-specific resistance candidates.  Nevertheless, most melanomas carried addnl. oncogenic mutations at baseline (for example, RAC1 and AKT3) that activate the MAPK and PI3K pathways and are thus predicted to diminish response to MAPK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhfIk5riiY_LVg90H21EOLACvtfcHk0lgATG6ZSGQtgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVentb4%253D&md5=64e244a00335113afeabaf6ab4db2974</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fncomms6694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6694%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DFung%26aufirst%3DC.%26aulast%3DMenzies%26aufirst%3DA.%2BM.%26aulast%3DPupo%26aufirst%3DG.%2BM.%26aulast%3DCarlino%26aufirst%3DM.%2BS.%26aulast%3DHyman%26aufirst%3DJ.%26aulast%3DShahheydari%26aufirst%3DH.%26aulast%3DTembe%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DJ.%2BF.%26aulast%3DSaw%26aufirst%3DR.%2BP.%26aulast%3DHowle%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DN.%2BK.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DKefford%26aufirst%3DR.%2BF.%26aulast%3DRizos%26aufirst%3DH.%26atitle%3DIncreased%2520MAPK%2520reactivation%2520in%2520early%2520resistance%2520to%2520dabrafenib%252Ftrametinib%2520combination%2520therapy%2520of%2520BRAF-mutant%2520metastatic%2520melanoma%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26doi%3D10.1038%2Fncomms6694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriceau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hugo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koya, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samatar, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanlou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchalski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seifert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlman, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, R. S.</span></span> <span> </span><span class="NLM_article-title">Tunable-combinatorial mechanisms of acquired resistance limit the efficacy of BRAF/MEK cotargeting but result in melanoma drug addiction</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2014.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.ccell.2014.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25600339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=240-256&author=G.+Moriceauauthor=W.+Hugoauthor=A.+Hongauthor=H.+Shiauthor=X.+Kongauthor=C.+C.+Yuauthor=R.+C.+Koyaauthor=A.+A.+Samatarauthor=N.+Khanlouauthor=J.+Braunauthor=K.+Ruchalskiauthor=H.+Seifertauthor=J.+Larkinauthor=K.+B.+Dahlmanauthor=D.+B.+Johnsonauthor=A.+Algaziauthor=J.+A.+Sosmanauthor=A.+Ribasauthor=R.+S.+Lo&title=Tunable-combinatorial+mechanisms+of+acquired+resistance+limit+the+efficacy+of+BRAF%2FMEK+cotargeting+but+result+in+melanoma+drug+addiction&doi=10.1016%2Fj.ccell.2014.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Tunable-Combinatorial Mechanisms of Acquired Resistance Limit the Efficacy of BRAF/MEK Cotargeting but Result in Melanoma Drug Addiction</span></div><div class="casAuthors">Moriceau, Gatien; Hugo, Willy; Hong, Aayoung; Shi, Hubing; Kong, Xiangju; Yu, Clarissa C.; Koya, Richard C.; Samatar, Ahmed A.; Khanlou, Negar; Braun, Jonathan; Ruchalski, Kathleen; Seifert, Heike; Larkin, James; Dahlman, Kimberly B.; Johnson, Douglas B.; Algazi, Alain; Sosman, Jeffrey A.; Ribas, Antoni; Lo, Roger S.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">240-256</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Combined BRAF- and MEK-targeted therapy improves upon BRAF inhibitor (BRAFi) therapy but is still beset by acquired resistance.  We show that melanomas acquire resistance to combined BRAF and MEK inhibition by augmenting or combining mechanisms of single-agent BRAFi resistance.  These double-drug resistance-assocd. genetic configurations significantly altered mol. interactions underlying MAPK pathway reactivation. V600EBRAF, expressed at supraphysiol. levels because of V600EBRAF ultra-amplification, dimerized with and activated CRAF.  In addn., MEK mutants enhanced interaction with overexpressed V600EBRAF via a regulatory interface at R662 of V600EBRAF.  Importantly, melanoma cell lines selected for resistance to BRAFi+MEKi, but not those to BRAFi alone, displayed robust drug addiction, providing a potentially exploitable therapeutic opportunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWdJyJ2wlmALVg90H21EOLACvtfcHk0lgATG6ZSGQtgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKgtrg%253D&md5=42788605b6c22cf70ff49304bf65ca19</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2014.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2014.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DMoriceau%26aufirst%3DG.%26aulast%3DHugo%26aufirst%3DW.%26aulast%3DHong%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DC.%2BC.%26aulast%3DKoya%26aufirst%3DR.%2BC.%26aulast%3DSamatar%26aufirst%3DA.%2BA.%26aulast%3DKhanlou%26aufirst%3DN.%26aulast%3DBraun%26aufirst%3DJ.%26aulast%3DRuchalski%26aufirst%3DK.%26aulast%3DSeifert%26aufirst%3DH.%26aulast%3DLarkin%26aufirst%3DJ.%26aulast%3DDahlman%26aufirst%3DK.%2BB.%26aulast%3DJohnson%26aufirst%3DD.%2BB.%26aulast%3DAlgazi%26aufirst%3DA.%26aulast%3DSosman%26aufirst%3DJ.%2BA.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DLo%26aufirst%3DR.%2BS.%26atitle%3DTunable-combinatorial%2520mechanisms%2520of%2520acquired%2520resistance%2520limit%2520the%2520efficacy%2520of%2520BRAF%252FMEK%2520cotargeting%2520but%2520result%2520in%2520melanoma%2520drug%2520addiction%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D240%26epage%3D256%26doi%3D10.1016%2Fj.ccell.2014.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: elements of PROTAC design</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.02.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.cbpa.2019.02.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31004963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXns1yrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2019&pages=111-119&author=S.+L.+Paivaauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+elements+of+PROTAC+design&doi=10.1016%2Fj.cbpa.2019.02.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: elements of PROTAC design</span></div><div class="casAuthors">Paiva, Stacey-Lynn; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn. using Proteolysis Targeting Chimeras (PROTACs) has emerged as a novel therapeutic modality in drug discovery.  PROTACs mediate the degrdn. of select proteins of interest (POIs) by hijacking the activity of E3 ubiquitin ligases for POI ubiquitination and subsequent degrdn. by the 26S proteasome.  This hijacking mechanism has been used to degrade various types of disease-relevant POIs.  In this review, we aim to highlight the recent advances in targeted protein degrdn. and describe the challenges that need to be addressed in order to efficiently develop potent PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol6Nk0VkUdzLVg90H21EOLACvtfcHk0livEna3y6_spA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXns1yrt7s%253D&md5=5783311db47da378180c14d67526a398</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.02.022%26sid%3Dliteratum%253Aachs%26aulast%3DPaiva%26aufirst%3DS.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520elements%2520of%2520PROTAC%2520design%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D50%26spage%3D111%26epage%3D119%26doi%3D10.1016%2Fj.cbpa.2019.02.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">64</span> <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2eYQ2zRNwhSknbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span> <span> </span><span class="NLM_article-title">Development of targeted protein degradation therapeutics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">944</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0362-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41589-019-0362-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31527835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=937-944&author=P.+P.+Chamberlainauthor=L.+G.+Hamann&title=Development+of+targeted+protein+degradation+therapeutics&doi=10.1038%2Fs41589-019-0362-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Development of targeted protein degradation therapeutics</span></div><div class="casAuthors">Chamberlain, Philip P.; Hamann, Lawrence G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">937-944</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted protein degrdn. as a therapeutic modality has seen dramatic progress and massive investment in recent years because of the convergence of two key scientific breakthroughs: optimization of first-generation peptidic proteolysis-targeted chimeras (PROTACs) into more drug-like mols. able to support in vivo proof of concept and the discovery that clin. mols. function as degraders by binding and repurposing the proteins cereblon and DCAF15.  This provided clin. validation for the general approach through the cereblon modulator class of drugs and provided highly drug-like and ligand-efficient E3 ligase binders upon which to tether target-binding moieties.  Increasingly rational and systematic approaches including biophys. and structural studies on ternary complexes are being leveraged as the field advances.  In this Perspective we summarize the discoveries that have laid the foundation for future degrdn. therapeutics, focusing on those classes of small mols. that redirect E3 ubiquitin ligases to non-native substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfcIEwVR-Xq7Vg90H21EOLACvtfcHk0livEna3y6_spA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF&md5=468e3fd87ba2b21607dac8b495e5ecf6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0362-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0362-y%26sid%3Dliteratum%253Aachs%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26atitle%3DDevelopment%2520of%2520targeted%2520protein%2520degradation%2520therapeutics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D937%26epage%3D944%26doi%3D10.1038%2Fs41589-019-0362-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0livEna3y6_spA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bond, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalawansha, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of oncogenic KRAS(G12C) by VHL-recruiting PROTACs</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1367</span>– <span class="NLM_lpage">1375</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.0c00411</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c00411" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlans7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=1367-1375&author=M.+J.+Bondauthor=L.+Chuauthor=D.+A.+Nalawanshaauthor=K.+Liauthor=C.+M.+Crews&title=Targeted+degradation+of+oncogenic+KRAS%28G12C%29+by+VHL-recruiting+PROTACs&doi=10.1021%2Facscentsci.0c00411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of Oncogenic KRASG12C by VHL-Recruiting PROTACs</span></div><div class="casAuthors">Bond, Michael J.; Chu, Ling; Nalawansha, Dhanusha A.; Li, Ke; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1367-1375</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KRAS is mutated in ~ 20% of human cancers and is one of the most sought-after targets for pharmacol. modulation, despite having historically been considered "undruggable." The discovery of potent covalent inhibitors of the KRASG12C mutant in recent years has sparked a new wave of interest in small mols. targeting KRAS.  While these inhibitors have shown promise in the clinic, we wanted to explore PROTAC-mediated degrdn. as a complementary strategy to modulate mutant KRAS.  Herein, we report the development of LC-2(I), the first PROTAC capable of degrading endogenous KRASG12C.  LC-2 covalently binds KRASG12C with a MRTX849 warhead and recruits the E3 ligase VHL, inducing rapid and sustained KRASG12C degrdn. leading to suppression of MAPK signaling in both homozygous and heterozygous KRASG12C cell lines.  LC-2 demonstrates that PROTAC-mediated degrdn. is a viable option for attenuating oncogenic KRAS levels and downstream signaling in cancer cells.  KRAS is one of the most sought-after targets for cancer therapy.  Herein, we report the development of LC-2, the first PROTAC capable of inducing endogenous KRASG12C degrdn. by recruiting VHL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra8EvL_8O3_rVg90H21EOLACvtfcHk0ljM8Bwo5_sUqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlans7zM&md5=09dc3d55acd5ecd3e8e0e18740830263</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c00411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c00411%26sid%3Dliteratum%253Aachs%26aulast%3DBond%26aufirst%3DM.%2BJ.%26aulast%3DChu%26aufirst%3DL.%26aulast%3DNalawansha%26aufirst%3DD.%2BA.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520degradation%2520of%2520oncogenic%2520KRAS%2528G12C%2529%2520by%2520VHL-recruiting%2520PROTACs%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2020%26volume%3D6%26spage%3D1367%26epage%3D1375%26doi%3D10.1021%2Facscentsci.0c00411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plewe, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective small molecule degraders of BRAF-V600E</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4069</span>– <span class="NLM_lpage">4080</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvFCjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4069-4080&author=X.+R.+Hanauthor=L.+Chenauthor=Y.+Weiauthor=W.+Yuauthor=Y.+Chenauthor=C.+Zhangauthor=B.+Jiaoauthor=T.+Shiauthor=L.+Sunauthor=C.+Zhangauthor=Y.+Xuauthor=M.+R.+Leeauthor=Y.+Luoauthor=M.+B.+Pleweauthor=J.+Wang&title=Discovery+of+selective+small+molecule+degraders+of+BRAF-V600E&doi=10.1021%2Facs.jmedchem.9b02083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Selective Small Molecule Degraders of BRAF-V600E</span></div><div class="casAuthors">Han, Xiao-Ran; Chen, Liqun; Wei, Yuanqi; Yu, Weihua; Chen, Yanke; Zhang, Chunyan; Jiao, Bingyang; Shi, Tingting; Sun, Lei; Zhang, Chao; Xu, Yang; Lee, Matthew R.; Luo, Ying; Plewe, Michael B.; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4069-4080</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRAF is among the most frequently mutated oncogenes in human cancers.  Multiple small mol. BRAF kinase inhibitors have been approved for treating melanoma carrying BRAF-V600 mutations.  However, the benefits of BRAF kinase inhibitors are generally short-lived.  Small mol.-mediated targeted protein degrdn. has recently emerged as a novel pharmaceutical strategy to remove disease proteins through hijacking the cellular ubiquitin proteasome system (UPS).  In this study, we developed thalidomide-based heterobifunctional compds. that induced selective degrdn. of BRAF-V600E, but not the wild-type BRAF.  Downregulation of BRAF-V600E suppressed the MEK/ERK kinase cascade in melanoma cells and impaired cell growth in culture.  Abolishing the interaction between degraders and cereblon or blocking the UPS significantly impaired the activities of these degraders, validating a mechanistic role of UPS in mediating targeted degrdn. of BRAF-V600E.  These findings highlight a new approach to modulate the functions of oncogenic BRAF mutants and provide a framework to treat BRAF-dependent human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCRmOwotGfh7Vg90H21EOLACvtfcHk0ljM8Bwo5_sUqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvFCjtbg%253D&md5=db67b3ec42b8a3eb9641c278c8cd17b6</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02083%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DJiao%26aufirst%3DB.%26aulast%3DShi%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DPlewe%26aufirst%3DM.%2BB.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520selective%2520small%2520molecule%2520degraders%2520of%2520BRAF-V600E%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4069%26epage%3D4080%26doi%3D10.1021%2Facs.jmedchem.9b02083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Posternak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maisonneuve, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlicky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goullet
de Rugy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prakesch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaygorodsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colwill, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccarelli, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batey, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taipale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurinov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uehling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrana, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span> <span> </span><span class="NLM_article-title">Functional characterization of a PROTAC directed against BRAF mutant V600E</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1170</span>– <span class="NLM_lpage">1178</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0609-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41589-020-0609-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=32778845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1OisLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1170-1178&author=G.+Posternakauthor=X.+Tangauthor=P.+Maisonneuveauthor=T.+Jinauthor=H.+Lavoieauthor=S.+Daouauthor=S.+Orlickyauthor=T.+Goullet%0Ade+Rugyauthor=L.+Caldwellauthor=K.+Chanauthor=A.+Amanauthor=M.+Prakeschauthor=G.+Podaauthor=P.+Maderauthor=C.+Wongauthor=S.+Maierauthor=J.+Kitaygorodskyauthor=B.+Larsenauthor=K.+Colwillauthor=Z.+Yinauthor=D.+F.+Ceccarelliauthor=R.+A.+Bateyauthor=M.+Taipaleauthor=I.+Kurinovauthor=D.+Uehlingauthor=J.+Wranaauthor=D.+Durocherauthor=A.+C.+Gingrasauthor=R.+Al-Awarauthor=M.+Therrienauthor=F.+Sicheri&title=Functional+characterization+of+a+PROTAC+directed+against+BRAF+mutant+V600E&doi=10.1038%2Fs41589-020-0609-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of a PROTAC directed against BRAF mutant V600E</span></div><div class="casAuthors">Posternak, Ganna; Tang, Xiaojing; Maisonneuve, Pierre; Jin, Ting; Lavoie, Hugo; Daou, Salima; Orlicky, Stephen; Goullet de Rugy, Theo; Caldwell, Lauren; Chan, Kin; Aman, Ahmed; Prakesch, Michael; Poda, Gennady; Mader, Pavel; Wong, Cassandra; Maier, Stefan; Kitaygorodsky, Julia; Larsen, Brett; Colwill, Karen; Yin, Zhe; Ceccarelli, Derek F.; Batey, Robert A.; Taipale, Mikko; Kurinov, Igor; Uehling, David; Wrana, Jeff; Durocher, Daniel; Gingras, Anne-Claude; Al-Awar, Rima; Therrien, Marc; Sicheri, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1170-1178</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The RAF family kinases function in the RAS-ERK pathway to transmit signals from activated RAS to the downstream kinases MEK and ERK.  This pathway regulates cell proliferation, differentiation and survival, enabling mutations in RAS and RAF to act as potent drivers of human cancers.  Drugs targeting the prevalent oncogenic mutant BRAF(V600E) have shown great efficacy in the clinic, but long-term effectiveness is limited by resistance mechanisms that often exploit the dimerization-dependent process by which RAF kinases are activated.  Here, we investigated a proteolysis-targeting chimera (PROTAC) approach to BRAF inhibition.  The most effective PROTAC, termed P4B, displayed superior specificity and inhibitory properties relative to non-PROTAC controls in BRAF(V600E) cell lines.  In addn., P4B displayed utility in cell lines harboring alternative BRAF mutations that impart resistance to conventional BRAF inhibitors.  This work provides a proof of concept for a substitute to conventional chem. inhibition to therapeutically constrain oncogenic BRAF. [graphic not available: see fulltext].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzRU_Yla3GkLVg90H21EOLACvtfcHk0lgngdFxsDA6VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1OisLnI&md5=18cb24da6d1fd343efe2d7f2f6f67d99</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0609-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0609-7%26sid%3Dliteratum%253Aachs%26aulast%3DPosternak%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DMaisonneuve%26aufirst%3DP.%26aulast%3DJin%26aufirst%3DT.%26aulast%3DLavoie%26aufirst%3DH.%26aulast%3DDaou%26aufirst%3DS.%26aulast%3DOrlicky%26aufirst%3DS.%26aulast%3DGoullet%2Bde%2BRugy%26aufirst%3DT.%26aulast%3DCaldwell%26aufirst%3DL.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DPrakesch%26aufirst%3DM.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DMader%26aufirst%3DP.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DMaier%26aufirst%3DS.%26aulast%3DKitaygorodsky%26aufirst%3DJ.%26aulast%3DLarsen%26aufirst%3DB.%26aulast%3DColwill%26aufirst%3DK.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DCeccarelli%26aufirst%3DD.%2BF.%26aulast%3DBatey%26aufirst%3DR.%2BA.%26aulast%3DTaipale%26aufirst%3DM.%26aulast%3DKurinov%26aufirst%3DI.%26aulast%3DUehling%26aufirst%3DD.%26aulast%3DWrana%26aufirst%3DJ.%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DTherrien%26aufirst%3DM.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DFunctional%2520characterization%2520of%2520a%2520PROTAC%2520directed%2520against%2520BRAF%2520mutant%2520V600E%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D1170%26epage%3D1178%26doi%3D10.1038%2Fs41589-020-0609-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alabi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Mutant-selective degradation by BRAF-targeting PROTACs</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1101/2020.08.10.245159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1101%2F2020.08.10.245159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=33173865" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=S.+Alabiauthor=S.+Jaime-Figueroaauthor=Z.+Yaoauthor=Y.+Gaoauthor=J.+Hinesauthor=K.+T.+Samarasingheauthor=L.+Vogtauthor=N.+Rosenauthor=C.+M.+Crews&title=Mutant-selective+degradation+by+BRAF-targeting+PROTACs&doi=10.1101%2F2020.08.10.245159"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1101%2F2020.08.10.245159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F2020.08.10.245159%26sid%3Dliteratum%253Aachs%26aulast%3DAlabi%26aufirst%3DS.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%26aulast%3DVogt%26aufirst%3DL.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DMutant-selective%2520degradation%2520by%2520BRAF-targeting%2520PROTACs%26jtitle%3DbioRxiv%26date%3D2020%26doi%3D10.1101%2F2020.08.10.245159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class mitogen-activated protein kinase kinase 1/2 degrader</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10911</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01528</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01528" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10897-10911&author=J.+Weiauthor=J.+Huauthor=L.+Wangauthor=L.+Xieauthor=M.+S.+Jinauthor=X.+Chenauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+a+first-in-class+mitogen-activated+protein+kinase+kinase+1%2F2+degrader&doi=10.1021%2Facs.jmedchem.9b01528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></div><div class="casAuthors">Wei, Jieli; Hu, Jianping; Wang, Li; Xie, Ling; Jin, Margaret S.; Chen, Xian; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10897-10911</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the "gatekeepers" of the ERK signaling output with redundant roles in controlling ERK activity.  Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs.  However, acquired resistance to MEK1/2 inhibitors has been obsd. in patients and new therapeutic strategies are needed to overcome the resistance.  Here, the authors report a first-in-class degrader of MEK1/2, MS432 (I), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells.  It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its neg. control MS432N and its effect was phenocopied by MEK1/2 knockdown.  Compd. I was highly selective for MEK1/2 in global proteomic profiling studies.  It was also bioavailable in mice and can be used for in vivo efficacy studies.  The authors provide two well-characterized chem. tools to the biomedical community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e5sg33oDYrVg90H21EOLACvtfcHk0lgngdFxsDA6VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK&md5=2c293b6b87990d6b814ec1dc88034f19</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520mitogen-activated%2520protein%2520kinase%2520kinase%25201%252F2%2520degrader%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10897%26epage%3D10911%26doi%3D10.1021%2Facs.jmedchem.9b01528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vollmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunoosamy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and biological evaluation of MEK PROTACs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00810</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00810" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlWkt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=157-162&author=S.+Vollmerauthor=D.+Cunoosamyauthor=H.+Lvauthor=H.+Fengauthor=X.+Liauthor=Z.+Nanauthor=W.+Yangauthor=M.+W.+D.+Perry&title=Design%2C+synthesis%2C+and+biological+evaluation+of+MEK+PROTACs&doi=10.1021%2Facs.jmedchem.9b00810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of MEK PROTACs</span></div><div class="casAuthors">Vollmer, Stefan; Cunoosamy, Danen; Lv, Huafei; Feng, Huanxi; Li, Xia; Nan, Ziyang; Yang, Wenzhen; Perry, Matthew W. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">157-162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PROteolysis TArgeting Chimeras (PROTACs) targeting the degrdn. of MEK have been designed based on allosteric MEK inhibitors.  Inhibition of the phosphorylation of ERK1/2 was less effective with the PROTACs than a small-mol. inhibitor; the best PROTACs, however, were more effective in inhibiting proliferation of A375 cells than an inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof7TjD-lFjAbVg90H21EOLACvtfcHk0lgngdFxsDA6VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlWkt77E&md5=c83d63e23d436e0ef1a3c98e994f1073</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00810%26sid%3Dliteratum%253Aachs%26aulast%3DVollmer%26aufirst%3DS.%26aulast%3DCunoosamy%26aufirst%3DD.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DNan%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520MEK%2520PROTACs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D157%26epage%3D162%26doi%3D10.1021%2Facs.jmedchem.9b00810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebraud, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Protein degradation by in-cell self-assembly of proteolysis targeting chimeras</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">934</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.6b00280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.6b00280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=927-934&author=H.+Lebraudauthor=D.+J.+Wrightauthor=C.+N.+Johnsonauthor=T.+D.+Heightman&title=Protein+degradation+by+in-cell+self-assembly+of+proteolysis+targeting+chimeras&doi=10.1021%2Facscentsci.6b00280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Degradation by In-Cell Self-Assembly of Proteolysis Targeting Chimeras</span></div><div class="casAuthors">Lebraud, Honorine; Wright, David J.; Johnson, Christopher N.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">927-934</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective degrdn. of proteins by proteolysis targeting chimeras (PROTACs) offers a promising potential alternative to protein inhibition for therapeutic intervention.  Current PROTAC mols. incorporate a ligand for the target protein, a linker, and an E3 ubiquitin ligase recruiting group, which bring together target protein and ubiquitinating machinery.  Such hetero-bifunctional mols. require significant linker optimization and possess high mol. wt., which can limit cellular permeation, soly., and other drug-like properties.  We show here that the hetero-bifunctional mol. can be formed intracellularly by bio-orthogonal click combination of two smaller precursors.  We designed a tetrazine tagged thalidomide deriv. which reacts rapidly with a trans-cyclo-octene tagged ligand of the target protein in cells to form a cereblon E3 ligase recruiting PROTAC mol.  The in-cell click-formed proteolysis targeting chimeras (CLIPTACs) were successfully used to degrade two key oncol. targets, BRD4 and ERK1/2.  ERK1/2 degrdn. was achieved using a CLIPTAC based on a covalent inhibitor.  We expect this approach to be readily extendable to other inhibitor-protein systems because the tagged E3 ligase recruiter is capable of undergoing the click reaction with a suitably tagged ligand of any protein of interest to elicit its degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRnQIoT7hbt7Vg90H21EOLACvtfcHk0lh0wxFN95IG5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ&md5=4cc9b4aaba6ee03cd2d4ea97a72728a5</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.6b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.6b00280%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DWright%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DProtein%2520degradation%2520by%2520in-cell%2520self-assembly%2520of%2520proteolysis%2520targeting%2520chimeras%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2016%26volume%3D2%26spage%3D927%26epage%3D934%26doi%3D10.1021%2Facscentsci.6b00280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jo, S. A.</span></span> <span> </span><span class="NLM_article-title">Mitogen-activated protein kinase kinase 1 (MEK1) stabilizes MyoD through direct phosphorylation at tyrosine 156 during myogenic differentiation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">18903</span>– <span class="NLM_lpage">18913</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.225128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1074%2Fjbc.M111.225128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=21454680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlGltbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=18903-18913&author=C.+Joauthor=S.+J.+Choauthor=S.+A.+Jo&title=Mitogen-activated+protein+kinase+kinase+1+%28MEK1%29+stabilizes+MyoD+through+direct+phosphorylation+at+tyrosine+156+during+myogenic+differentiation&doi=10.1074%2Fjbc.M111.225128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Mitogen-activated protein kinase kinase 1 (MEK1) stabilizes MyoD through direct phosphorylation at tyrosine 156 during myogenic differentiation</span></div><div class="casAuthors">Jo, Chulman; Cho, Sun-Jung; Jo, Sangmee Ahn</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">18903-18913</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Previously, the authors reported that MEK1 kinase activated in the mid-stage of skeletal muscle differentiation promotes myogenic differentiation.  To elucidate the mol. mechanism, the authors investigated the activity of MEK1 kinase for MyoD.  Activated MEK1 assocd. with MyoD in the nucleus of differentiating myoblasts.  An in vitro kinase assay using active MEK1, a 32P-labeled protein band corresponding to GST-MyoD was obsd. but not to mutant GST-MyoD-Y156F.  Tyr-156 phosphorylation in endogenous MyoD was detected with a specific anti-pMyoD-Y156 antibody; however, this response was blocked by PD 184352, a MEK-specific inhibitor.  These results indicated that activated MEK1 phosphorylates the MyoD Tyr-156 residue directly.  Interestingly, the protein level of mutant MyoD-Y156F decreased compared with that of wild-type protein, but was recovered in the presence of lactacystin, a proteasome inhibitor.  The protein level of MyoD-Y156E, which mimics phosphorylation at Tyr-156, was above that of wild-type protein, indicating that the phosphorylation protected MyoD from ubiquitin-proteasome-mediated degrdn.  In addn., the low protein level of MyoD-Y156F was recovered over that of the wild-type protein by an addnl. mutation at Leu-164, a crit. binding residue of MAFbx/AT-1, a Skp, Cullin, F-box (SCF) E3-ubiquitin ligase.  The amt. of MyoD co-pptd. with MAFbx/AT-1 also was reduced in the presence of active MEK1.  Thus, these results suggested that the phosphorylation probably interrupted the binding of MAFbx/AT-1 to MyoD and thereby increased its stability.  Collectively, these results suggest that MEK1 activated in differentiating myoblasts stimulates muscle differentiation by phosphorylating MyoD Tyr-156, which results in MyoD stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC9-gmuSStOrVg90H21EOLACvtfcHk0lh0wxFN95IG5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlGltbk%253D&md5=ce2a130978a3f32816846795071bb234</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.225128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.225128%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DC.%26aulast%3DCho%26aufirst%3DS.%2BJ.%26aulast%3DJo%26aufirst%3DS.%2BA.%26atitle%3DMitogen-activated%2520protein%2520kinase%2520kinase%25201%2520%2528MEK1%2529%2520stabilizes%2520MyoD%2520through%2520direct%2520phosphorylation%2520at%2520tyrosine%2520156%2520during%2520myogenic%2520differentiation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D18903%26epage%3D18913%26doi%3D10.1074%2Fjbc.M111.225128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampson, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, C.</span></span> <span> </span><span class="NLM_article-title">MEK guards proteome stability and inhibits tumor-suppressive amyloidogenesis via HSF1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.01.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.cell.2015.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25679764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVSguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2015&pages=729-744&author=Z.+Tangauthor=S.+Daiauthor=Y.+Heauthor=R.+A.+Dotyauthor=L.+D.+Shultzauthor=S.+B.+Sampsonauthor=C.+Dai&title=MEK+guards+proteome+stability+and+inhibits+tumor-suppressive+amyloidogenesis+via+HSF1&doi=10.1016%2Fj.cell.2015.01.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">MEK Guards Proteome Stability and Inhibits Tumor-Suppressive Amyloidogenesis via HSF1</span></div><div class="casAuthors">Tang, Zijian; Dai, Siyuan; He, Yishu; Doty, Rosalinda A.; Shultz, Leonard D.; Sampson, Stephen Byers; Dai, Chengkai</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">729-744</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Signaling through RAS/MAP kinase pathway is central to biol.  ERK has long been perceived as the only substrate for MEK.  Here, we report that HSF1, the master regulator of the proteotoxic stress response, is a new MEK substrate.  Beyond mediating cell-environment interactions, the MEK-HSF1 regulation impacts malignancy.  In tumor cells, MEK blockade inactivates HSF1 and thereby provokes proteomic chaos, presented as protein destabilization, aggregation, and, strikingly, amyloidogenesis.  Unlike their non-transformed counterparts, tumor cells are particularly susceptible to proteomic perturbation and amyloid induction.  Amyloidogenesis is tumor suppressive, reducing in vivo melanoma growth and contributing to the potent anti-neoplastic effects of proteotoxic stressors.  Our findings unveil a key biol. function of the oncogenic RAS-MEK signaling in guarding proteostasis and suppressing amyloidogenesis.  Thus, proteomic instability is an intrinsic feature of malignant state, and disrupting the fragile tumor proteostasis to promote amyloidogenesis may be a feasible therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUInK2yxXl4rVg90H21EOLACvtfcHk0lh0wxFN95IG5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVSguro%253D&md5=77038af3c279cec71a2163066c753e55</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DDoty%26aufirst%3DR.%2BA.%26aulast%3DShultz%26aufirst%3DL.%2BD.%26aulast%3DSampson%26aufirst%3DS.%2BB.%26aulast%3DDai%26aufirst%3DC.%26atitle%3DMEK%2520guards%2520proteome%2520stability%2520and%2520inhibits%2520tumor-suppressive%2520amyloidogenesis%2520via%2520HSF1%26jtitle%3DCell%26date%3D2015%26volume%3D160%26spage%3D729%26epage%3D744%26doi%3D10.1016%2Fj.cell.2015.01.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallay, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourquard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claeysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crepieux, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poupon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermoere, F.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of beta-arrestin 2 at Thr383 by MEK underlies beta-arrestin-dependent activation of Erk1/2 by GPCRs</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e23777</span> <span class="refDoi"> DOI: 10.7554/eLife.23777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.7554%2FeLife.23777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=28169830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWktbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=E.+Cassierauthor=N.+Gallayauthor=T.+Bourquardauthor=S.+Claeysenauthor=J.+Bockaertauthor=P.+Crepieuxauthor=A.+Pouponauthor=E.+Reiterauthor=P.+Marinauthor=F.+Vandermoere&title=Phosphorylation+of+beta-arrestin+2+at+Thr383+by+MEK+underlies+beta-arrestin-dependent+activation+of+Erk1%2F2+by+GPCRs&doi=10.7554%2FeLife.23777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of β-arrestin2 at Thr383 by MEK underlies β-arrestin-dependent activation of Erk1/2 by GPCRs</span></div><div class="casAuthors">Cassier, Elisabeth; Gallay, Nathalie; Bourquard, Thomas; Claeysen, Sylvie; Bockaert, Joel; Crepieux, Pascale; Poupon, Anne; Reiter, Eric; Marin, Philippe; Vandermoere, Franck</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e23777/1-e23777/21</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">In addn. to their role in desensitization and internalization of G protein-coupled receptors (GPCRs), β-arrestins are essential scaffolds linking GPCRs to Erk1/2 signaling.  However, their role in GPCR-operated Erk1/2 activation differs between GPCRs and the underlying mechanism remains poorly characterized.  Here, we show that activation of serotonin 5-HT2C receptors, which engage Erk1/2 pathway via a β-arrestin-dependent mechanism, promotes MEK-dependent β-arrestin2 phosphorylation at Thr383, a necessary step for Erk recruitment to the receptor/β-arrestin complex and Erk activation.  Likewise, Thr383 phosphorylation is involved in β arrestin-dependent Erk1/2 stimulation elicited by other GPCRs such as β2-adrenergic, FSH and CXCR4 receptors, but does not affect the β-arrestin-independent Erk1/2 activation by 5-HT4 receptor.  Collectively, these data show that β-arrestin2 phosphorylation at Thr383 underlies β-arrestin-dependent Erk1/2 activation by GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoh4tSQrz0-7Vg90H21EOLACvtfcHk0lgr5N8TbXnuXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWktbvM&md5=61522d61e8ac7e61f73835f89e0af4b9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.7554%2FeLife.23777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.23777%26sid%3Dliteratum%253Aachs%26aulast%3DCassier%26aufirst%3DE.%26aulast%3DGallay%26aufirst%3DN.%26aulast%3DBourquard%26aufirst%3DT.%26aulast%3DClaeysen%26aufirst%3DS.%26aulast%3DBockaert%26aufirst%3DJ.%26aulast%3DCrepieux%26aufirst%3DP.%26aulast%3DPoupon%26aufirst%3DA.%26aulast%3DReiter%26aufirst%3DE.%26aulast%3DMarin%26aufirst%3DP.%26aulast%3DVandermoere%26aufirst%3DF.%26atitle%3DPhosphorylation%2520of%2520beta-arrestin%25202%2520at%2520Thr383%2520by%2520MEK%2520underlies%2520beta-arrestin-dependent%2520activation%2520of%2520Erk1%252F2%2520by%2520GPCRs%26jtitle%3DeLife%26date%3D2017%26volume%3D6%26doi%3D10.7554%2FeLife.23777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, E.</span></span> <span> </span><span class="NLM_article-title">Nuclear export of MAP kinase (ERK) involves a MAP kinase kinase (MEK)-dependent active transport mechanism</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">849</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1083/jcb.148.5.849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1083%2Fjcb.148.5.849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10704436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhslansrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2000&pages=849-856&author=M.+Adachiauthor=M.+Fukudaauthor=E.+Nishida&title=Nuclear+export+of+MAP+kinase+%28ERK%29+involves+a+MAP+kinase+kinase+%28MEK%29-dependent+active+transport+mechanism&doi=10.1083%2Fjcb.148.5.849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear export of MAP kinase (ERK) involves a MAP kinase kinase (MEK)-dependent active transport mechanism</span></div><div class="casAuthors">Adachi, Makoto; Fukuda, Makoto; Nishida, Eisuke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">849-856</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">In response to extracellular stimuli, mitogen-activated protein kinase (MAPK, also known as ERK), which localizes to the cytoplasm in quiescent cells, translocates to the nucleus and then relocalizes to the cytoplasm again.  The relocalization of nuclear MAPK to the cytoplasm was not inhibited by cycloheximide, confirming that the relocalization is achieved by nuclear export, but not synthesis, of MAPK.  The nuclear export of MAPK was inhibited by leptomycin B (LMB), a specific inhibitor of the nuclear export signal (NES)-dependent transport.  We have then shown that MAP kinase kinase (MAPKK, also known as MEK), which mostly localizes to the cytoplasm because of its having NES, is able to shuttle between the cytoplasm and the nucleus constantly.  MAPK, when injected into the nucleus, was rapidly exported from the nucleus by coinjected wild-type MAPKK, but not by the NES-disrupted MAPKK.  In addn., injection of the fragment corresponding to the MAPK-binding site of MAPKK into the nucleus, which would disrupt the binding of MAPK to MAPKK in the nucleus, significantly inhibited the nuclear export of endogenous MAPK.  Taken together, these results suggest that the relocalization of nuclear MAPK to the cytoplasm involves a MAPKK-dependent, active transport mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXku6zczusYbVg90H21EOLACvtfcHk0lgr5N8TbXnuXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhslansrw%253D&md5=79e974b197237554b11b4c85726fc649</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1083%2Fjcb.148.5.849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.148.5.849%26sid%3Dliteratum%253Aachs%26aulast%3DAdachi%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DNishida%26aufirst%3DE.%26atitle%3DNuclear%2520export%2520of%2520MAP%2520kinase%2520%2528ERK%2529%2520involves%2520a%2520MAP%2520kinase%2520kinase%2520%2528MEK%2529-dependent%2520active%2520transport%2520mechanism%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2000%26volume%3D148%26spage%3D849%26epage%3D856%26doi%3D10.1083%2Fjcb.148.5.849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgermeister, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuderland, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanoch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liscovitch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seger, R.</span></span> <span> </span><span class="NLM_article-title">Interaction with MEK causes nuclear export and downregulation of peroxisome proliferator-activated receptor gamma</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">803</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1128/MCB.00601-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1128%2FMCB.00601-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=17101779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=803-817&author=E.+Burgermeisterauthor=D.+Chuderlandauthor=T.+Hanochauthor=M.+Meyerauthor=M.+Liscovitchauthor=R.+Seger&title=Interaction+with+MEK+causes+nuclear+export+and+downregulation+of+peroxisome+proliferator-activated+receptor+gamma&doi=10.1128%2FMCB.00601-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction with MEK causes nuclear export and downregulation of peroxisome proliferator-activated receptor</span></div><div class="casAuthors">Burgermeister, Elke; Chuderland, Dana; Hanoch, Tamar; Meyer, Markus; Liscovitch, Mordechai; Seger, Rony</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">803-817</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mitogen-activated protein kinase (MAPK)/extracellular signal-regulated kinase (ERK) cascade plays a central role in intracellular signaling by many extracellular stimuli.  One target of the ERK cascade is peroxisome proliferator-activated receptor γ (PPARγ), a nuclear receptor that promotes differentiation and apoptosis.  It was previously demonstrated that PPARγ activity is attenuated upon mitogenic stimulation due to phosphorylation of its Ser84 by ERKs.  Here we show that stimulation by tetradecanoyl phorbol acetate (TPA) attenuates PPARγ's activity in a MEK-dependent manner, even when Ser84 is mutated to Ala.  To elucidate the mechanism of attenuation, we found that PPARγ directly interacts with MEKs, which are the activators of ERKs, but not with ERKs themselves, both in vivo and in vitro.  This interaction is facilitated by MEKs' phosphorylation and is mediated by the basic D domain of MEK1 and the AF2 domain of PPARγ.  Immunofluorescence microscopy and subcellular fractionation revealed that MEK1 exports PPARγ from the nucleus, and this finding was supported by small interfering RNA knockdown of MEK1 and use of a cell-permeable interaction-blocking peptide, which prevented TPA-induced export of PPARγ from the nucleus.  Thus, we show here a novel mode of downregulation of PPARγ by its MEK-dependent redistribution from the nucleus to the cytosol.  This unanticipated role for the stimulation-induced nuclear shuttling of MEKs shows that MEKs can regulate addnl. signaling components besides the ERK cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyBt7HomvWmLVg90H21EOLACvtfcHk0lgr5N8TbXnuXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGht7s%253D&md5=60b9cd18e1c5ac33122ea01c109fcd02</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1128%2FMCB.00601-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00601-06%26sid%3Dliteratum%253Aachs%26aulast%3DBurgermeister%26aufirst%3DE.%26aulast%3DChuderland%26aufirst%3DD.%26aulast%3DHanoch%26aufirst%3DT.%26aulast%3DMeyer%26aufirst%3DM.%26aulast%3DLiscovitch%26aufirst%3DM.%26aulast%3DSeger%26aufirst%3DR.%26atitle%3DInteraction%2520with%2520MEK%2520causes%2520nuclear%2520export%2520and%2520downregulation%2520of%2520peroxisome%2520proliferator-activated%2520receptor%2520gamma%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D803%26epage%3D817%26doi%3D10.1128%2FMCB.00601-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perry, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudnicki, M. A.</span></span> <span> </span><span class="NLM_article-title">Activated MEK1 binds the nuclear MyoD transcriptional complex to repress transactivation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(01)00302-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1097-2765%2801%2900302-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=11545732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWgsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=291-301&author=R.+L.+Perryauthor=M.+H.+Parkerauthor=M.+A.+Rudnicki&title=Activated+MEK1+binds+the+nuclear+MyoD+transcriptional+complex+to+repress+transactivation&doi=10.1016%2FS1097-2765%2801%2900302-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Activated MEK1 binds the nuclear MyoD transcriptional complex to repress transactivation</span></div><div class="casAuthors">Perry, Robert L. S.; Parker, Maura H.; Rudnicki, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">291-301</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">To elucidate the mechanism through which MAPK signaling regulates the MyoD family of transcription factors, we investigated the role of the signaling intermediate MEK1 in myogenesis.  Transfection of activated MEK1 strongly repressed gene activation and myogenic conversion by the MyoD family.  This repression was not mediated by direct phosphorylation of MyoD or by changes in MyoD stability or subcellular distribution.  Deletion mapping revealed that MEK1-mediated repression required the MyoD amino-terminal transactivation domain.  Moreover, activated MEK1 was nuclearly localized and bound a complex contg. MyoD in a manner that is dependent on the presence of the MyoD amino terminus.  Together, these data demonstrate that MEK1 signaling has a strong neg. effect on MyoD activity via a novel mechanism involving binding of MEK1 to the nuclear MyoD transcriptional complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDVpuTEhArrVg90H21EOLACvtfcHk0lhAzvZKotBl-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWgsbg%253D&md5=323a010043b41d04cb60d5aa907aa3ed</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2900302-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252900302-1%26sid%3Dliteratum%253Aachs%26aulast%3DPerry%26aufirst%3DR.%2BL.%26aulast%3DParker%26aufirst%3DM.%2BH.%26aulast%3DRudnicki%26aufirst%3DM.%2BA.%26atitle%3DActivated%2520MEK1%2520binds%2520the%2520nuclear%2520MyoD%2520transcriptional%2520complex%2520to%2520repress%2520transactivation%26jtitle%3DMol.%2520Cell%26date%3D2001%26volume%3D8%26spage%3D291%26epage%3D301%26doi%3D10.1016%2FS1097-2765%2801%2900302-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Procaccia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ordan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendetz-Nezer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seger, R.</span></span> <span> </span><span class="NLM_article-title">Direct binding of MEK1 and MEK2 to AKT induces Foxo1 phosphorylation, cellular migration and metastasis</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">43078</span> <span class="refDoi"> DOI: 10.1038/srep43078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fsrep43078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=28225038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVCku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&author=S.+Procacciaauthor=M.+Ordanauthor=I.+Cohenauthor=S.+Bendetz-Nezerauthor=R.+Seger&title=Direct+binding+of+MEK1+and+MEK2+to+AKT+induces+Foxo1+phosphorylation%2C+cellular+migration+and+metastasis&doi=10.1038%2Fsrep43078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Direct binding of MEK1 and MEK2 to AKT induces Foxo1 phosphorylation, cellular migration and metastasis</span></div><div class="casAuthors">Procaccia, Shiri; Ordan, Merav; Cohen, Izel; Bendetz-Nezer, Sarit; Seger, Rony</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43078</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Crosstalk between the ERK cascade and other signaling pathways is one of the means by which it acquires its signaling specificity.  Here we identified a direct interaction of both MEK1 and MEK2 with AKT.  The interaction is mediated by the proline rich domain of MEK1/2 and regulated by phosphorylation of Ser298 in MEK1, or Ser306 in MEK2, which we identified here as a novel regulatory site.  We further developed a blocking peptide, which inhibits the interaction between MEK and AKT, and when applied to cells, affects migration and adhesion, but not proliferation.  The specific mechanism of action of the MEK-AKT complex involves phosphorylation of the migration-related transcription factor FoxO1.  Importantly, prevention of the interaction results in a decreased metastasis formation in a breast cancer mouse model.  Thus, the identified interaction both sheds light on how signaling specificity is detd., and represents a possible new therapeutic target for metastatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon2bDXG4r_P7Vg90H21EOLACvtfcHk0lhAzvZKotBl-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsVCku7Y%253D&md5=e399d969e2cebc3305c4aadf94b25196</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fsrep43078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep43078%26sid%3Dliteratum%253Aachs%26aulast%3DProcaccia%26aufirst%3DS.%26aulast%3DOrdan%26aufirst%3DM.%26aulast%3DCohen%26aufirst%3DI.%26aulast%3DBendetz-Nezer%26aufirst%3DS.%26aulast%3DSeger%26aufirst%3DR.%26atitle%3DDirect%2520binding%2520of%2520MEK1%2520and%2520MEK2%2520to%2520AKT%2520induces%2520Foxo1%2520phosphorylation%252C%2520cellular%2520migration%2520and%2520metastasis%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26doi%3D10.1038%2Fsrep43078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novakova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span> <span> </span><span class="NLM_article-title">Development of multifunctional histone deacetylase 6 degraders with potent antimyeloma activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7042</span>– <span class="NLM_lpage">7057</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00516</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00516" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlams7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7042-7057&author=H.+Wuauthor=K.+Yangauthor=Z.+Zhangauthor=E.+D.+Leistenauthor=Z.+Liauthor=H.+Xieauthor=J.+Liuauthor=K.+A.+Smithauthor=Z.+Novakovaauthor=C.+Barinkaauthor=W.+Tang&title=Development+of+multifunctional+histone+deacetylase+6+degraders+with+potent+antimyeloma+activity&doi=10.1021%2Facs.jmedchem.9b00516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Multifunctional Histone Deacetylase 6 Degraders with Potent Antimyeloma Activity</span></div><div class="casAuthors">Wu, Hao; Yang, Ka; Zhang, Zhongrui; Leisten, Eric D.; Li, Ziyuan; Xie, Haibo; Liu, Jin; Smith, Kerry A.; Novakova, Zora; Barinka, Cyril; Tang, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7042-7057</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) primarily catalyzes the removal of acetyl group from the side chain of acetylated lysine residues in cytoplasmic proteins such as α-tubulin and HSP90.  HDAC6 is involved in multiple disease-relevant pathways.  Based on the proteolysis targeting chimera strategy, we previously developed the first HDAC6 degrader by tethering a pan-HDAC inhibitor with cereblon (CRBN) E3 ubiquitin ligase ligand.  We herein report our new generation of multifunctional HDAC6 degraders by tethering selective HDAC6 inhibitor Nexturastat A with CRBN ligand that can synergize with HDAC6 degrdn. for the antiproliferation of multiple myeloma (MM).  This new class of degraders exhibited improved potency and selectivity for the degrdn. of HDAC6.  After the optimization of the linker length and linking positions, we discovered potent HDAC6 degraders with nanomolar DC50 and promising antiproliferation activity in multiple myeloma (MM) cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb58zf04V2GLVg90H21EOLACvtfcHk0lhAzvZKotBl-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlams7nO&md5=7aab402404a5fce732b4b68fd76395d4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00516%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLeisten%26aufirst%3DE.%2BD.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DK.%2BA.%26aulast%3DNovakova%26aufirst%3DZ.%26aulast%3DBarinka%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520multifunctional%2520histone%2520deacetylase%25206%2520degraders%2520with%2520potent%2520antimyeloma%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7042%26epage%3D7057%26doi%3D10.1021%2Facs.jmedchem.9b00516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">The dTAG system for immediate and target-specific protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0021-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41589-018-0021-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29581585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=431-441&author=B.+Nabetauthor=J.+M.+Robertsauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=S.+Dastjerdiauthor=A.+Yangauthor=A.+L.+Leggettauthor=M.+A.+Erbauthor=M.+A.+Lawlorauthor=A.+Souzaauthor=T.+G.+Scottauthor=S.+Vittoriauthor=J.+A.+Perryauthor=J.+Qiauthor=G.+E.+Winterauthor=K.+K.+Wongauthor=N.+S.+Grayauthor=J.+E.+Bradner&title=The+dTAG+system+for+immediate+and+target-specific+protein+degradation&doi=10.1038%2Fs41589-018-0021-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">The dTAG system for immediate and target-specific protein degradation</span></div><div class="casAuthors">Nabet, Behnam; Roberts, Justin M.; Buckley, Dennis L.; Paulk, Joshiawa; Dastjerdi, Shiva; Yang, Annan; Leggett, Alan L.; Erb, Michael A.; Lawlor, Matthew A.; Souza, Amanda; Scott, Thomas G.; Vittori, Sarah; Perry, Jennifer A.; Qi, Jun; Winter, Georg E.; Wong, Kwok-Kin; Gray, Nathanael S.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">431-441</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dissection of complex biol. systems requires target-specific control of the function or abundance of proteins.  Genetic perturbations are limited by off-target effects, multicomponent complexity, and irreversibility.  Most limiting is the requisite delay between modulation to exptl. measurement.  To enable the immediate and selective control of single protein abundance, we created a chem. biol. system that leverages the potency of cell-permeable heterobifunctional degraders.  The dTAG system pairs a novel degrader of FKBP12F36V with expression of FKBP12F36V in-frame with a protein of interest.  By transgene expression or CRISPR-mediated locus-specific knock-in, we exemplify a generalizable strategy to study the immediate consequence of protein loss.  Using dTAG, we observe an unexpected superior antiproliferative effect of pan-BET bromodomain degrdn. over selective BRD4 degrdn., characterize immediate effects of KRASG12V loss on proteomic signaling, and demonstrate rapid degrdn. in vivo.  This technol. platform will confer kinetic resoln. to biol. investigation and provide target validation in the context of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqwQvCDe-2I7Vg90H21EOLACvtfcHk0lgGjZQBX-DdsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D&md5=d7d9c111c68c84bc3aa9c2aad3f591a6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0021-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0021-8%26sid%3Dliteratum%253Aachs%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DLeggett%26aufirst%3DA.%2BL.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLawlor%26aufirst%3DM.%2BA.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520dTAG%2520system%2520for%2520immediate%2520and%2520target-specific%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D431%26epage%3D441%26doi%3D10.1038%2Fs41589-018-0021-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Irrizary, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czabaniuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niosi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span> <span> </span><span class="NLM_article-title">Delineating the role of cooperativity in the design of potent PROTACs for BTK</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E7285</span>– <span class="NLM_lpage">E7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.1803662115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1073%2Fpnas.1803662115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=30012605" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E7285-E7292&author=A.+Zorbaauthor=C.+Nguyenauthor=Y.+Xuauthor=J.+Starrauthor=K.+Borzilleriauthor=J.+Smithauthor=H.+Zhuauthor=K.+A.+Farleyauthor=W.+Dingauthor=J.+Schiemerauthor=X.+Fengauthor=J.+S.+Changauthor=D.+P.+Uccelloauthor=J.+A.+Youngauthor=C.+N.+Garcia-Irrizaryauthor=L.+Czabaniukauthor=B.+Schuffauthor=R.+Oliverauthor=J.+Montgomeryauthor=M.+M.+Haywardauthor=J.+Coeauthor=J.+Chenauthor=M.+Niosiauthor=S.+Luthraauthor=J.+C.+Shahauthor=A.+El-Kattanauthor=X.+Qiuauthor=G.+M.+Westauthor=M.+C.+Noeauthor=V.+Shanmugasundaramauthor=A.+M.+Gilbertauthor=M.+F.+Brownauthor=M.+F.+Calabrese&title=Delineating+the+role+of+cooperativity+in+the+design+of+potent+PROTACs+for+BTK&doi=10.1073%2Fpnas.1803662115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1803662115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1803662115%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DSchiemer%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DGarcia-Irrizary%26aufirst%3DC.%2BN.%26aulast%3DCzabaniuk%26aufirst%3DL.%26aulast%3DSchuff%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNiosi%26aufirst%3DM.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DJ.%2BC.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26atitle%3DDelineating%2520the%2520role%2520of%2520cooperativity%2520in%2520the%2520design%2520of%2520potent%2520PROTACs%2520for%2520BTK%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2018%26volume%3D115%26spage%3DE7285%26epage%3DE7292%26doi%3D10.1073%2Fpnas.1803662115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+M.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Q.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lgGjZQBX-DdsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%2BM.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%2BQ.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lhut-b1YNPsCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+Bohmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thoma&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0ljUoPp4M9IHOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBohm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThoma%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eroglu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span> <span> </span><span class="NLM_article-title">Combination therapy with BRAF and MEK inhibitors for melanoma: latest evidence and place in therapy</span>. <i>Ther. Adv. Med. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">48</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1177/1758834015616934</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1177%2F1758834015616934" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=26753005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC28XotValsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=48-56&author=Z.+Erogluauthor=A.+Ribas&title=Combination+therapy+with+BRAF+and+MEK+inhibitors+for+melanoma%3A+latest+evidence+and+place+in+therapy&doi=10.1177%2F1758834015616934"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Combination therapy with BRAF and MEK inhibitors for melanoma: latest evidence and place in therapy</span></div><div class="casAuthors">Eroglu, Zeynep; Ribas, Antoni</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Medical Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-56</span>CODEN:
                <span class="NLM_cas:coden">TAMOB8</span>;
        ISSN:<span class="NLM_cas:issn">1758-8340</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">Treatment with BRAF inhibitors such as vemurafenib or dabrafenib in patients with advanced BRAFV600 mutated melanoma has shown objective tumor responses in approx. half of the patients.  However, the duration of responses is limited in a majority of these patients, with progression-free survival rates around 6 mo due to tumor progression from development of acquired resistance.  Preclin. studies have suggested that concurrent inhibition of the BRAF kinases and MEK of the mitogen-activated protein kinase (MAPK) pathway could decrease MAPK-driven acquired resistance, resulting in longer duration of responses, higher rate of tumor responses, and a decrease in the cutaneous toxicities obsd. from paradoxical MAPK pathway activation with BRAF inhibitor monotherapy.  This review provides an overview of the currently available clin. trial data on BRAF and MEK inhibitors together and in combinations with other therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPeAe_WaeHr7Vg90H21EOLACvtfcHk0ljUoPp4M9IHOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotValsbc%253D&md5=9709de3d9c9dd47f7fb13ccc11e2b2a6</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1177%2F1758834015616934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1758834015616934%26sid%3Dliteratum%253Aachs%26aulast%3DEroglu%26aufirst%3DZ.%26aulast%3DRibas%26aufirst%3DA.%26atitle%3DCombination%2520therapy%2520with%2520BRAF%2520and%2520MEK%2520inhibitors%2520for%2520melanoma%253A%2520latest%2520evidence%2520and%2520place%2520in%2520therapy%26jtitle%3DTher.%2520Adv.%2520Med.%2520Oncol.%26date%3D2016%26volume%3D8%26spage%3D48%26epage%3D56%26doi%3D10.1177%2F1758834015616934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span> <span> </span><span class="NLM_article-title">AKT and ERK dual inhibitors: The way forward?</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>459</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2019.05.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.canlet.2019.05.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31128213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFensLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2019&pages=30-40&author=Z.+Caoauthor=Q.+Liaoauthor=M.+Suauthor=K.+Huangauthor=J.+Jinauthor=D.+Cao&title=AKT+and+ERK+dual+inhibitors%3A+The+way+forward%3F&doi=10.1016%2Fj.canlet.2019.05.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">AKT and ERK dual inhibitors: The way forward?</span></div><div class="casAuthors">Cao, Zhe; Liao, Qianjin; Su, Min; Huang, Kai; Jin, Junfei; Cao, Deliang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">459</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30-40</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Phosphatidylinositol 3-kinase (PI3K)/AKT pathway regulates cell growth, proliferation, survival, mobility and invasion.  Mitogen-activated protein kinase (MAPK)/extracellular signal-regulated kinase (ERK) pathway is also an important mitogenic signaling pathway involved in various cellular progresses.  AKT, also named protein kinase B (PKB), is a primary mediator of the PI3K signaling pathway; and ERK at the end of MAPK signaling is the unique substrate and downstream effector of mitogen-activated protein/extracellular signal-regulated kinase (MEK).  The AKT and ERK signaling are both aberrantly activated in a wide range of human cancers and have long been targeted for cancer therapy, but the clin. benefits of these targeted therapies have been limited due to complex cross-talk.  Novel strategies, such as AKT/ERK dual inhibitors, may be needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2mQop_AQFH7Vg90H21EOLACvtfcHk0ljUoPp4M9IHOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFensLnE&md5=bdb7fd9232b6f75f46b0800a2627a4e1</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2019.05.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2019.05.025%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLiao%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DD.%26atitle%3DAKT%2520and%2520ERK%2520dual%2520inhibitors%253A%2520The%2520way%2520forward%253F%26jtitle%3DCancer%2520Lett.%26date%3D2019%26volume%3D459%26spage%3D30%26epage%3D40%26doi%3D10.1016%2Fj.canlet.2019.05.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardasi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadei-Gardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallanzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrikou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascinu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelsomino, F.</span></span> <span> </span><span class="NLM_article-title">BRAF-mutated colorectal cancer: clinical and molecular insights</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5369</span>, <span class="refDoi"> DOI: 10.3390/ijms20215369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.3390%2Fijms20215369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptFShtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=5369&author=F.+Caputoauthor=C.+Santiniauthor=C.+Bardasiauthor=K.+Cermaauthor=A.+Casadei-Gardiniauthor=A.+Spallanzaniauthor=K.+Andrikouauthor=S.+Cascinuauthor=F.+Gelsomino&title=BRAF-mutated+colorectal+cancer%3A+clinical+and+molecular+insights&doi=10.3390%2Fijms20215369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">BRAF-mutated colorectal cancer: clinical and molecular insights</span></div><div class="casAuthors">Caputo, Francesco; Santini, Chiara; Bardasi, Camilla; Cerma, Krisida; Casadei-Gardini, Andrea; Spallanzani, Andrea; Andrikou, Kalliopi; Cascinu, Stefano; Gelsomino, Fabio</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5369</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Colorectal cancer (CRC) is one of the leading causes of mortality and morbidity in the world.  It is a heterogeneous disease, which can be classified into different subtypes, characterized by specific mol. and morphol. alterations.  In this context, BRAF mutations are found in about 10% of CRC patients and define a particular subtype, characterized by a dismal prognosis, with a median survival of less than 12 mo.  Chemotherapy plus bevacizumab is the current std. therapy in first-line treatment of BRAF-mutated metastatic CRC (mCRC), with triplet (FOLFOXIRI) plus bevacizumab as a valid option in patients with a good performance status.  BRAF inhibitors are not so effective as compared to melanoma, because of various resistance mechanisms.  However, the recently published results of the BEACON trial will establish a new std. of care in this setting.  This review provides insights into the mol. underpinnings underlying the resistance to std. treatment of BRAF-mutated CRCs, with a focus on their mol. heterogeneity and on the research perspectives both from a translational and a clin. point of view.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ0Jox-5s91bVg90H21EOLACvtfcHk0liXI6ZqbbbtFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptFShtbc%253D&md5=8da8f55e3b8146185952edc18cf3f2e0</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.3390%2Fijms20215369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20215369%26sid%3Dliteratum%253Aachs%26aulast%3DCaputo%26aufirst%3DF.%26aulast%3DSantini%26aufirst%3DC.%26aulast%3DBardasi%26aufirst%3DC.%26aulast%3DCerma%26aufirst%3DK.%26aulast%3DCasadei-Gardini%26aufirst%3DA.%26aulast%3DSpallanzani%26aufirst%3DA.%26aulast%3DAndrikou%26aufirst%3DK.%26aulast%3DCascinu%26aufirst%3DS.%26aulast%3DGelsomino%26aufirst%3DF.%26atitle%3DBRAF-mutated%2520colorectal%2520cancer%253A%2520clinical%2520and%2520molecular%2520insights%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D5369%26doi%3D10.3390%2Fijms20215369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aasen, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjerkvig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knappskog, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsen, F.</span></span> <span> </span><span class="NLM_article-title">Effective treatment of metastatic melanoma by combining MAPK and PI3K signaling pathway inhibitors</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4235</span>, <span class="refDoi"> DOI: 10.3390/ijms20174235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.3390%2Fijms20174235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVGrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=4235&author=S.+N.+Aasenauthor=H.+Parajuliauthor=T.+Hoangauthor=Z.+Fengauthor=K.+Stokkeauthor=J.+Wangauthor=K.+Royauthor=R.+Bjerkvigauthor=S.+Knappskogauthor=F.+Thorsen&title=Effective+treatment+of+metastatic+melanoma+by+combining+MAPK+and+PI3K+signaling+pathway+inhibitors&doi=10.3390%2Fijms20174235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Effective treatment of metastatic melanoma by combining MAPK and PI3K signaling pathway inhibitors</span></div><div class="casAuthors">Aasen, Synnoeve Nymark; Parajuli, Himalaya; Hoang, Tuyen; Feng, Zichao; Stokke, Krister; Wang, Jiwei; Roy, Kislay; Bjerkvig, Rolf; Knappskog, Stian; Thorsen, Frits</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4235</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Malignant melanoma is the most aggressive type of skin cancer and is closely assocd. with the development of brain metastases.  Despite aggressive treatment, the prognosis has traditionally been poor, necessitating improved therapies.  In melanoma, the mitogen activated protein kinase and the phosphoinositide 3-kinase signaling pathways are commonly altered, and therapeutically inhibiting one of the pathways often upregulates the other, leading to resistance.  Thus, combined treatment targeting both pathways is a promising strategy to overcome this.  Here, we studied the in vitro and in vivo effects of the PI3K inhibitor buparlisib and the MEK1/2 inhibitor trametinib, used either as targeted monotherapies or in combination, on patient-derived melanoma brain metastasis cell lines.  Scratch wound and trans-well assays were carried out to assess the migratory capacity of the cells upon drug treatment, whereas flow cytometry, apoptosis array and Western blots were used to study apoptosis.  Finally, an in vivo treatment expt. was carried out on NOD/SCID mice.  We show that combined therapy was more effective than monotherapy.  Combined treatment also more effectively increased apoptosis, and inhibited tumor growth in vivo.  This suggests a clin. potential of combined treatment to overcome ceased treatment activity which is often seen after monotherapies, and strongly encourages the evaluation of the treatment strategy on melanoma patients with brain metastases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobWDmITf6bWbVg90H21EOLACvtfcHk0liXI6ZqbbbtFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVGrsL0%253D&md5=9ccfa4cc09fa6b6ab5e48247f44c1c08</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.3390%2Fijms20174235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20174235%26sid%3Dliteratum%253Aachs%26aulast%3DAasen%26aufirst%3DS.%2BN.%26aulast%3DParajuli%26aufirst%3DH.%26aulast%3DHoang%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DStokke%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRoy%26aufirst%3DK.%26aulast%3DBjerkvig%26aufirst%3DR.%26aulast%3DKnappskog%26aufirst%3DS.%26aulast%3DThorsen%26aufirst%3DF.%26atitle%3DEffective%2520treatment%2520of%2520metastatic%2520melanoma%2520by%2520combining%2520MAPK%2520and%2520PI3K%2520signaling%2520pathway%2520inhibitors%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D4235%26doi%3D10.3390%2Fijms20174235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0liXI6ZqbbbtFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thierbach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartwig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taupitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruhl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, C.</span></span> <span> </span><span class="NLM_article-title">Highly monodisperse water-dispersable iron oxide nanoparticles for biomedical applications</span>. <i>J. Mater. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">7842</span>– <span class="NLM_lpage">7853</span>, <span class="refDoi"> DOI: 10.1039/c0jm01169j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1039%2Fc0jm01169j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVylsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=7842-7853&author=A.+Hofmannauthor=S.+Thierbachauthor=A.+Semischauthor=A.+Hartwigauthor=M.+Taupitzauthor=E.+Ruhlauthor=C.+Graf&title=Highly+monodisperse+water-dispersable+iron+oxide+nanoparticles+for+biomedical+applications&doi=10.1039%2Fc0jm01169j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Highly monodisperse water-dispersable iron oxide nanoparticles for biomedical applications</span></div><div class="casAuthors">Hofmann, Andreas; Thierbach, Steffen; Semisch, Annetta; Hartwig, Andrea; Taupitz, Matthias; Ruehl, Eckart; Graf, Christina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Materials Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">7842-7853</span>CODEN:
                <span class="NLM_cas:coden">JMACEP</span>;
        ISSN:<span class="NLM_cas:issn">0959-9428</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We demonstrate a unique approach for prepg. high quality iron oxide (Fe3O4) nanoparticles functionalized by newly developed multifunctional dendron ligands for biomedical applications.  These particles are suitable for magnetic resonance imaging (MRI), highly stable in aq. solns. as well as physiol. media and not cytotoxic.  In particular, oleic acid capped Fe3O4 particles (d = 12 ± 0.8 nm) were modified in a ligand exchange process by investigating several dendron ligands of variable size and an adjustable no. of polar end groups.  The dendron based ligands lead only to a slight increase in hydrodynamic diam. of the nanoparticles after the ligand exchange process (∼6 nm).  They also allow an adjustment of the particle polarity as well as a gradually variable surface functionalization.  Light scattering, transmission electron microscopy, and visible spectroscopy studies show consistently that the dendron-capped iron oxide nanoparticles exhibit excellent stability in various physiol. media as well as aq. solns. in a broad pH range.  It is also demonstrated by magnetic resonance studies that the magnetic relaxivity is almost not affected by the ligand exchange.  Therefore, such small particles might be of specific interest for cardiovascular MRI and MRI of extravascular targets.  In addn., the present approach opens new possibilities for the specific linking of biomols. to the particle surface, which can be beneficial for various biol. sensing and therapeutic applications.  The cytotoxicity of the Fe3O4 nanoparticles was evaluated using the WST-8 assay.  In the examd. concn. range up to 100 μg Fe/mL no significant decrease in cell viability was detected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq51NfZ9tw6urVg90H21EOLACvtfcHk0ljoFTFxlZk0EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVylsLjF&md5=626197097b6bed2ee328c2b0b0b33117</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1039%2Fc0jm01169j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0jm01169j%26sid%3Dliteratum%253Aachs%26aulast%3DHofmann%26aufirst%3DA.%26aulast%3DThierbach%26aufirst%3DS.%26aulast%3DSemisch%26aufirst%3DA.%26aulast%3DHartwig%26aufirst%3DA.%26aulast%3DTaupitz%26aufirst%3DM.%26aulast%3DRuhl%26aufirst%3DE.%26aulast%3DGraf%26aufirst%3DC.%26atitle%3DHighly%2520monodisperse%2520water-dispersable%2520iron%2520oxide%2520nanoparticles%2520for%2520biomedical%2520applications%26jtitle%3DJ.%2520Mater.%2520Chem.%26date%3D2010%26volume%3D20%26spage%3D7842%26epage%3D7853%26doi%3D10.1039%2Fc0jm01169j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kusal T.G.  Samarasinghe</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted protein degradation: A promise for undruggable proteins. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 934-951. <a href="https://doi.org/10.1016/j.chembiol.2021.04.011" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.04.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.04.011%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DTargeted%252Bprotein%252Bdegradation%25253A%252BA%252Bpromise%252Bfor%252Bundruggable%252Bproteins%26aulast%3DSamarasinghe%26aufirst%3DKusal%2BT.G.%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D934%26epage%3D951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandon  Dale</span>, <span class="hlFld-ContribAuthor ">Meng  Cheng</span>, <span class="hlFld-ContribAuthor ">Kwang-Su  Park</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Yue  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Advancing targeted protein degradation for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2021,</strong> <em>170 </em><a href="https://doi.org/10.1038/s41568-021-00365-x" title="DOI URL">https://doi.org/10.1038/s41568-021-00365-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-021-00365-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-021-00365-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DAdvancing%252Btargeted%252Bprotein%252Bdegradation%252Bfor%252Bcancer%252Btherapy%26aulast%3DDale%26aufirst%3DBrandon%26date%3D2021%26date%3D2021%26volume%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Han  Wang</span>, <span class="hlFld-ContribAuthor ">Cangxin  Zheng</span>, <span class="hlFld-ContribAuthor ">Zhenming  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaozuo  Gao</span>, <span class="hlFld-ContribAuthor ">Fengrong  Xu</span>, <span class="hlFld-ContribAuthor ">Yan  Niu</span>, <span class="hlFld-ContribAuthor ">Liangren  Zhang</span>, <span class="hlFld-ContribAuthor ">Ping  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress of MEK1/2 inhibitors and degraders in the treatment of cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>218 </em>, 113386. <a href="https://doi.org/10.1016/j.ejmech.2021.113386" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113386</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113386%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DResearch%252Bprogress%252Bof%252BMEK1%25252F2%252Binhibitors%252Band%252Bdegraders%252Bin%252Bthe%252Btreatment%252Bof%252Bcancer%26aulast%3DWang%26aufirst%3DChao%26date%3D2021%26volume%3D218%26spage%3D113386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of MEK inhibitors <b>1</b> and <b>2</b>, and MEK degraders <b>3</b> and <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Effect of compounds <b>11</b>, <b>13</b>, and <b>14</b> on MEK1/2 protein levels in HT-29 cells. The cells were treated with dimethyl sulfoxide (DMSO) or serial dilution of indicated compounds for 24 h. MEK1/2 protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compounds <b>24</b> and <b>27</b> degrade MEK1 and MEK2 and inhibit pMEK and pERK in a concentration-dependent manner. HT-29 (A) and SK-MEL-28 (B) cells were treated with DMSO or serial dilution of indicated compounds for 24 h. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Effects of compounds <b>28</b>, <b>40</b>, <b>42</b>, <b>44</b>, <b>45</b>, <b>49</b>, and <b>50</b> on MEK1/2 protein levels in HT-29 cells. The cells were treated with DMSO or serial dilution of indicated compounds for 24 h. MEK1/2 protein levels were determined by Western blots and normalized with α-tubulin. (B) Compound <b>50</b> reduces MEK1/2 protein levels and inhibits the downstream ERK signaling in a concentration-dependent manner in HT-29 and SK-MEL-28 cells. HT-29 and SK-MEL-28 cells were treated with DMSO or serial dilution of compound <b>50</b> for 24 h. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Chemical structures of MEK degraders and their corresponding controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compounds <b>24, 27</b>, and <b>50</b> degrade MEK1 and MEK2 and inhibit pERK signaling in a time-dependent manner. HT-29 cells treated with compound <b>24</b> or <b>27</b> (0.1 μM) or <b>50</b> (0.3 μM) were harvested at indicated time points. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Control compounds <b>51</b>–<b>53</b> do not induce MEK1/2 degradation and are less potent at inhibiting the downstream ERK signaling in HT-29 and SK-MEL-28 cells after 24 h treatment. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Degradation of MEK1/2 induced by compound <b>24</b>, <b>27</b>, or <b>50</b> is mediated by the ubiquitin-proteasome system. HT-29 cells were pretreated with DMSO, PD0325901 (1 μM), MG-132 (3 μM), MLN4924 (3 μM), VH 032 (10 μM), or pomalidomide (POMA, 5 μM) for 2 h, before <b>24</b> or <b>27</b> (0.1 μM), or <b>50</b> (0.3 μM) was added. The cells were incubated for another 8 h. The indicated protein levels were determined by Western blots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0010.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compounds <b>27</b> and <b>50</b> selectively degrade MEK1 and MEK2 in HT-29 cells. HT-29 cells were treated with 0.1% DMSO, <b>27</b> (0.1 μM), <b>50</b> (0.3 μM), <b>52</b> (0.1 μM), or <b>53</b> (0.3 μM) for 8 h before they were harvested for mass spectrometry analysis. Volcano plots of the −log 10 (<i>p-</i>value) vs the log 2 fold change are displayed. Proteins outside the significance lines are labeled with pink or purple color (FDR = 0.05, S0 = 1). <i>P</i> values were calculated from the data of two technical replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0011.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compounds <b>24</b>, <b>27</b>, and <b>50</b> significantly suppress the growth of HT-29 and SK-MEL-28 cells. HT-29 (A–C) and SK-MEL-28 (D–F) cells were treated with serial dilutions of indicated compounds for 3 days, followed by WST-8 assay to get a live cell signal. For each concentration point, in quadruplicate, each mean value ± SD from three independent experiments is shown in the curves. (G–I) Clonogenic assay was conducted in HT-29 cells. HT-29 cells were treated with indicated compounds for 10 days. The Petri dish images are representative of two independent experiments. The cells were fixed and stained with crystal violet.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0012.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Antiproliferation effects of compounds <b>24</b> and <b>27</b> versus compound <b>4</b> in four different cancer cell lines. HT-29 (A), SK-MEL-28 (B), H3122 (C), and SUDHL1 (D) cells were treated with serial dilutions of indicated compounds for 3 days, followed by WST-8 assay to get a live cell signal. GraphPad Prism 8 was used to plot column graphs (A–C) or concentration curves (D) and calculate GI<sub>50</sub> values (D). Each concentration point was in duplicate (A, B) or triplicate (C, D). The GI<sub>50</sub> values (D) are presented as the mean ± SD from three independent experiments. Statistical significance was analyzed by Student’s <i>t</i>-tests for each concentration in the columns comparing to DMSO (A–C), <i>P</i> ≥ 0.05, ns; 0.01 ≤ <i>P</i> < 0.05, *; 0.001 ≤ <i>P</i> < 0.01, **; and <i>P</i> < 0.001, ***.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0013.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Concurrent inhibition of BRAF or PI3K potentiates antiproliferation potency of compound <b>27</b>. HT-29 (A) and SK-MEL-28 (B) cells were treated with indicated compounds for 3 days, followed by WST-8 assay to get a live cell signal. Each concentration point was in triplicate (A, B). Each mean value ± SD from three independent experiments is shown in the graphs. Statistical significance was analyzed by Student’s <i>t</i>-tests for each concentration in the columns comparing to DMSO. <i>P</i> ≥ 0.05, ns; 0.01 ≤ <i>P</i> < 0.05, *; 0.001 ≤ <i>P</i> < 0.01, **; and <i>P</i> < 0.001, ***.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0014.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Plasma concentrations of compounds <b>24</b> and <b>27</b> following a single intraperitoneal administration at 50 mg/kg for 8 h in male Swiss Albino mice. Plasma concentrations at each time point are presented as the mean ± SD from three mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0015.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesizing Compounds <b>5</b>–<b>23</b>, <b>25</b>–<b>50</b>, <b>52</b>, and <b>53</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DBU, <i>N</i>,<i>N</i>-dimethylformamide (DMF), 50 °C, 4 h; (b) N<sub>2</sub>H<sub>4</sub>.H<sub>2</sub>O, MeOH, dichloromethane (DCM), rt, 2 h; (c) 3,4-difluoro-2-((2-fluoro-4-iodophenyl)amino)benzoic acid, HOAt, EDCI, NMM, DMSO, rt, overnight; (d) 3 M HCl, tetrahydrofuran (THF), rt, 6 h; (e) different linkers, NaBH<sub>3</sub>CN, DCM, MeOH, rt, overnight.</p></p></figure><figure data-id="sch2" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/medium/jm0c01609_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0016.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesizing Compounds <b>24</b> and <b>51</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01609/20201218/images/large/jm0c01609_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01609&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CH<sub>3</sub>I, NaH, DMF, rt, 18 h; (b) (1) HOAt, EDCI, NMM, DMSO, rt, overnight; (2) trifluoroacetic acid (TFA), DCM, rt, 2 h; (c) <b>62</b>, NaBH<sub>3</sub>CN, DCM, MeOH, rt, overnight.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i93">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32345" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32345" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 65 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Targeting the RAF-MEK-ERK mitogen-activated protein kinase cascade for the treatment of cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3291</span>– <span class="NLM_lpage">3310</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fsj.onc.1210422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=17496923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1Wktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3291-3310&author=P.+J.+Robertsauthor=C.+J.+Der&title=Targeting+the+RAF-MEK-ERK+mitogen-activated+protein+kinase+cascade+for+the+treatment+of+cancer&doi=10.1038%2Fsj.onc.1210422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Raf-MEK-ERK mitogen-activated protein kinase cascade for the treatment of cancer</span></div><div class="casAuthors">Roberts, P. J.; Der, C. J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3291-3310</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mitogen-activated protein kinase (MAPK) cascades are key signaling pathways involved in the regulation of normal cell proliferation, survival and differentiation.  Aberrant regulation of MAPK cascades contribute to cancer and other human diseases.  In particular, the extracellular signal-regulated kinase (ERK) MAPK pathway has been the subject of intense research scrutiny leading to the development of pharmacol. inhibitors for the treatment of cancer.  ERK is a downstream component of an evolutionarily conserved signaling module that is activated by the Raf serine/threonine kinases.  Raf activates the MAPK/ERK kinase (MEK)1/2 dual-specificity protein kinases, which then activate ERK1/2.  The mutational activation of Raf in human cancers supports the important role of this pathway in human oncogenesis.  Addnl., the Raf-MEK-ERK pathway is a key downstream effector of the Ras small GTPase, the most frequently mutated oncogene in human cancers.  Finally, Ras is a key downstream effector of the epidermal growth factor receptor (EGFR), which is mutationally activated and/or overexpressed in a wide variety of human cancers.  ERK activation also promotes upregulated expression of EGFR ligands, promoting an autocrine growth loop crit. for tumor growth.  Thus, the EGFR-Ras-Raf-MEK-ERK signaling network has been the subject of intense research and pharmaceutical scrutiny to identify novel target-based approaches for cancer treatment.  In this review, we summarize the current status of the different approaches and targets that are under evaluation and development for the therapeutic intervention of this key signaling pathway in human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7hdVDVyRBHrVg90H21EOLACvtfcHk0lgE1PTQ_I2_IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1Wktb4%253D&md5=bf56109e3b4defc12004471da30178c0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210422%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DP.%2BJ.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DTargeting%2520the%2520RAF-MEK-ERK%2520mitogen-activated%2520protein%2520kinase%2520cascade%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D3291%26epage%3D3310%26doi%3D10.1038%2Fsj.onc.1210422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eblen, S. T.</span></span> <span> </span><span class="NLM_article-title">Extracellular-regulated kinases: signaling from RAS to ERK substrates to control biological outcomes</span>. <i>Adv. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1016/bs.acr.2018.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fbs.acr.2018.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29551131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFGrsr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2018&pages=99-142&author=S.+T.+Eblen&title=Extracellular-regulated+kinases%3A+signaling+from+RAS+to+ERK+substrates+to+control+biological+outcomes&doi=10.1016%2Fbs.acr.2018.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular-regulated kinases: signaling from Ras to ERK substrates to control biological outcomes</span></div><div class="casAuthors">Eblen, Scott T.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-142</span>CODEN:
                <span class="NLM_cas:coden">ACRSAJ</span>;
        ISSN:<span class="NLM_cas:issn">0065-230X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The extracellular-regulated kinases ERK1 and ERK2 are evolutionarily conserved, ubiquitous serine-threonine kinases that are involved in regulating cellular signaling in both normal and pathol. conditions.  Their expression is crit. for development and their hyperactivation is a major factor in cancer development and progression.  Since their discovery as one of the major signaling mediators activated by mitogens and Ras mutation, we have learned much about their regulation, including their activation, binding partners and substrates.  In this review I will discuss some of what has been discovered about the members of the Ras to ERK pathway, including regulation of their activation by growth factors and cell adhesion pathways.  Looking downstream of ERK activation I will also highlight some of the many ERK substrates that have been discovered, including those involved in feedback regulation, cell migration and cell cycle progression through the control of transcription, pre-mRNA splicing and protein synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2KWjVn2VpKrVg90H21EOLACvtfcHk0lin-8fifEGqhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFGrsr3J&md5=54d0bf34a1a52c1a938536341b945054</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fbs.acr.2018.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.acr.2018.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DEblen%26aufirst%3DS.%2BT.%26atitle%3DExtracellular-regulated%2520kinases%253A%2520signaling%2520from%2520RAS%2520to%2520ERK%2520substrates%2520to%2520control%2520biological%2520outcomes%26jtitle%3DAdv.%2520Cancer%2520Res.%26date%3D2018%26volume%3D138%26spage%3D99%26epage%3D142%26doi%3D10.1016%2Fbs.acr.2018.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frémin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meloche, S.</span></span> <span> </span><span class="NLM_article-title">From basic research to clinical development of MEK1/2 inhibitors for cancer therapy</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">8</span> <span class="refDoi"> DOI: 10.1186/1756-8722-3-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1186%2F1756-8722-3-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=20149254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BC3c7ltFGnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&author=C.+Fr%C3%A9minauthor=S.+Meloche&title=From+basic+research+to+clinical+development+of+MEK1%2F2+inhibitors+for+cancer+therapy&doi=10.1186%2F1756-8722-3-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">From basic research to clinical development of MEK1/2 inhibitors for cancer therapy</span></div><div class="casAuthors">Fremin Christophe; Meloche Sylvain</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The Ras-dependent Raf/MEK/ERK1/2 mitogen-activated protein (MAP) kinase signaling pathway is a major regulator of cell proliferation and survival.  Not surprisingly, hyperactivation of this pathway is frequently observed in human malignancies as a result of aberrant activation of receptor tyrosine kinases or gain-of-function mutations in RAS or RAF genes.  Components of the ERK1/2 pathway are therefore viewed as attractive candidates for the development of targeted therapies of cancer.  In this article, we briefly review the basic research that has laid the groundwork for the clinical development of small molecules inhibitors of the ERK1/2 pathway.  We then present the current state of clinical evaluation of MEK1/2 inhibitors in cancer and discuss challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRXiqh48Jb4JresJRIHUP1ofW6udTcc2ebTAq6uzcp2Krntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7ltFGnsg%253D%253D&md5=b6b8c91b259b37885fd951001b4bd817</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2F1756-8722-3-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8722-3-8%26sid%3Dliteratum%253Aachs%26aulast%3DFr%25C3%25A9min%26aufirst%3DC.%26aulast%3DMeloche%26aufirst%3DS.%26atitle%3DFrom%2520basic%2520research%2520to%2520clinical%2520development%2520of%2520MEK1%252F2%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2010%26volume%3D3%26doi%3D10.1186%2F1756-8722-3-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">MEK1/2 dual-specificity protein kinases: structure and regulation</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>417</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2011.11.145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.bbrc.2011.11.145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=22177953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1Oqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2012&pages=5-10&author=R.+Roskoski&title=MEK1%2F2+dual-specificity+protein+kinases%3A+structure+and+regulation&doi=10.1016%2Fj.bbrc.2011.11.145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1/2 dual-specificity protein kinases: Structure and regulation</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">417</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  MEK1 and MEK2 are related protein kinases that participate in the RAS-RAF-MEK-ERK signal transduction cascade.  This cascade participates in the regulation of a large variety of processes including apoptosis, cell cycle progression, cell migration, differentiation, metab., and proliferation.  Moreover, oncogenic mutations in RAS or B-RAF are responsible for a large proportion of human cancers.  MEK1 is activated by phosphorylation of Ser-218 and Ser-222 in its activation segment as catalyzed by RAF kinases in an intricate process that involves a KSR scaffold.  In addn. to functioning as a scaffold, the kinase activity of KSR is also required for MEK activation.  MEK1 regulation is unusual in that Ser-212 phosphorylation in its activation segment is inhibitory.  Moreover, active ERK kinase catalyzes a feedback inhibitory phosphorylation of MEK1 Thr-292 that serves to down-regulate the pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRX9pGR2iVKbVg90H21EOLACvtfcHk0lin-8fifEGqhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xos1Oqtg%253D%253D&md5=4bdfbf6ebaa90055a314922fa1056929</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2011.11.145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2011.11.145%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DMEK1%252F2%2520dual-specificity%2520protein%2520kinases%253A%2520structure%2520and%2520regulation%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2012%26volume%3D417%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.bbrc.2011.11.145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span> <span> </span><span class="NLM_article-title">Properties of MEKs, the kinases that phosphorylate and activate the extracellular signal-regulated kinases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">23933</span>– <span class="NLM_lpage">23939</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=8226933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADyaK3sXlvFygtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=23933-23939&author=C.+F.+Zhengauthor=K.+L.+Guan&title=Properties+of+MEKs%2C+the+kinases+that+phosphorylate+and+activate+the+extracellular+signal-regulated+kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Properties of MEKs, the kinases that phosphorylate and activate the extracellular signal-regulated kinases</span></div><div class="casAuthors">Zheng, Chao Feng; Guan, Kun Liang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">23933-9</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Activation of extracellular signal-regulated kinase (ERK) or mitogen-activated protein kinase by MEK (mitogen-activated protein kinase or extracellular signal-regulated kinase) is an essential event in the mitogenic growth factor signal transduction.  The authors now demonstrate that three recombinant MEKs (MEK1, MEK2, MEK3) show remarkably different activity toward recombinant ERK1 and ERK2.  MEK2 is the most active ERK activator.  The recombinant MEK1 has an activity approx. seven times lower than that of MEK2.  MEK3, which is identical to MEK1 except for missing an internal 26-amino acid residue and probably represents an alternative splicing product of MEK1, shows neither autophosphorylation nor ERK-activating activity.  Recombinant MEK1 and MEK2 can be activated by epidermal growth factor-stimulated SWISS3T3 cell lysate.  MEK1 and MEK2 can also be activated by autophosphorylation.  Autophosphorylation of MEKs correlates with their ability to phosphorylate and activate ERKs.  Phosphorylation of MEK is also stimulated by ERK.  Phosphoamino acid anal. showed that ERK1 preferentially phosphorylated threonine residue of MEKs.  MEKs complex with ERKs in vitro.  Interestingly, MEK3 also forms a complex with ERK1, although it is totally inactive as an ERK activator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLXezUZaU78rVg90H21EOLACvtfcHk0lg78XXbTWnmVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlvFygtrs%253D&md5=f43bb5309432d722c03efc7958ef050f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DC.%2BF.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26atitle%3DProperties%2520of%2520MEKs%252C%2520the%2520kinases%2520that%2520phosphorylate%2520and%2520activate%2520the%2520extracellular%2520signal-regulated%2520kinases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D23933%26epage%3D23939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karin, M.</span></span> <span> </span><span class="NLM_article-title">Mammalian MAP kinase signalling cascades</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>410</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1038/35065000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2F35065000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=11242034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhvVSgs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2001&pages=37-40&author=L.+Changauthor=M.+Karin&title=Mammalian+MAP+kinase+signalling+cascades&doi=10.1038%2F35065000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian MAP kinase signalling cascades</span></div><div class="casAuthors">Chang, Lufen; Karin, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">410</span>
        (<span class="NLM_cas:issue">6824</span>),
    <span class="NLM_cas:pages">37-40</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review, with 52 refs.  Mitogen-activated protein kinases (MAPKs) are important signal transducing enzymes, unique to eukaryotes, that are involved in many facets of cellular regulation.  Initial research concd. on defining the components and organization of MAPK signaling cascades, but recent studies have begun to shed light on the physiol. functions of these cascades in the control of gene expression, cell proliferation, and programmed cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHwmx2Bk-5dbVg90H21EOLACvtfcHk0lg78XXbTWnmVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhvVSgs7k%253D&md5=486ff61090fc371dff36b307b4e75c88</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F35065000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35065000%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DL.%26aulast%3DKarin%26aufirst%3DM.%26atitle%3DMammalian%2520MAP%2520kinase%2520signalling%2520cascades%26jtitle%3DNature%26date%3D2001%26volume%3D410%26spage%3D37%26epage%3D40%26doi%3D10.1038%2F35065000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebmann, C.</span></span> <span> </span><span class="NLM_article-title">Regulation of MAP kinase activity by peptide receptor signalling pathway: Paradigms of multiplicity</span>. <i>Cell Signalling</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">785</span>, <span class="refDoi"> DOI: 10.1016/S0898-6568(01)00192-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS0898-6568%2801%2900192-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=11583913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnt1Shu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2001&pages=777-785&author=C.+Liebmann&title=Regulation+of+MAP+kinase+activity+by+peptide+receptor+signalling+pathway%3A+Paradigms+of+multiplicity&doi=10.1016%2FS0898-6568%2801%2900192-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MAP kinase activity by peptide receptor signalling pathway: Paradigms of multiplicity</span></div><div class="casAuthors">Liebmann, C.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">777-785</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) can stimulate the mitogen-activated protein kinase (MAPK) cascade and thereby induce cellular proliferation like receptor tyrosine kinases (RTKs).  Work over the past 5 yr has established several models which reduce the links of Gi-, Gq-, and Gs-coupled receptors to MAPK on few principle pathways.  They include (1) Ras-dependent activation of MAPK via transactivation of RTKs such as the EGF receptor (EGFR), (2) Ras-independent MAPK activation via protein kinase C (PKC) that converges with the RTK signaling at the level of Raf, and (3) activation as well as inactivation of MAPK via the cAMP/protein kinase A (PKA) pathway in dependency on the type of Raf.  Most of these generalizing hypotheses are founded on exptl. data obtained from expression studies and using a limited set of individual receptors.  This review will compare these models with pathways to MAPK found for a great variety of peptide hormone and neuropeptide receptor subtypes in various cells.  It becomes evident that under endogenous conditions, the transactivation pathway is less dominant as postulated, whereas pathways involving isoforms of PKC and, esp., phosphoinositide 3-kinase (PI-3K) appear to play a more important role as assumed so far.  Highly cell-specific and unusual connections of signaling proteins towards MAPK, in particular tumor cells, might provide points of attacks for new therapeutic concepts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq788Kr6Dawu7Vg90H21EOLACvtfcHk0lg78XXbTWnmVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnt1Shu7g%253D&md5=fd2819d92926824fdae4f08087fcf7eb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0898-6568%2801%2900192-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0898-6568%252801%252900192-9%26sid%3Dliteratum%253Aachs%26aulast%3DLiebmann%26aufirst%3DC.%26atitle%3DRegulation%2520of%2520MAP%2520kinase%2520activity%2520by%2520peptide%2520receptor%2520signalling%2520pathway%253A%2520Paradigms%2520of%2520multiplicity%26jtitle%3DCell%2520Signalling%26date%3D2001%26volume%3D13%26spage%3D777%26epage%3D785%26doi%3D10.1016%2FS0898-6568%2801%2900192-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsakiridis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsiani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirte, H.</span></span> <span> </span><span class="NLM_article-title">Mitogen-activated protein kinase (MAPK) expression and activation in non-small cell lung cancer (NSCLC): investigating their prognostic significance and role in radiation responses</span>. <i>Radiother. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">S40</span>– <span class="NLM_lpage">S41</span>, <span class="refDoi"> DOI: 10.1016/S0167-8140(04)80634-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS0167-8140%2804%2980634-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2004&pages=S40-S41&author=T.+Tsakiridisauthor=J.+Wrightauthor=E.+Tsianiauthor=M.+Kalinaauthor=G.+Singhauthor=L.+Vincicauthor=H.+Hirte&title=Mitogen-activated+protein+kinase+%28MAPK%29+expression+and+activation+in+non-small+cell+lung+cancer+%28NSCLC%29%3A+investigating+their+prognostic+significance+and+role+in+radiation+responses&doi=10.1016%2FS0167-8140%2804%2980634-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS0167-8140%2804%2980634-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0167-8140%252804%252980634-0%26sid%3Dliteratum%253Aachs%26aulast%3DTsakiridis%26aufirst%3DT.%26aulast%3DWright%26aufirst%3DJ.%26aulast%3DTsiani%26aufirst%3DE.%26aulast%3DKalina%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DVincic%26aufirst%3DL.%26aulast%3DHirte%26aufirst%3DH.%26atitle%3DMitogen-activated%2520protein%2520kinase%2520%2528MAPK%2529%2520expression%2520and%2520activation%2520in%2520non-small%2520cell%2520lung%2520cancer%2520%2528NSCLC%2529%253A%2520investigating%2520their%2520prognostic%2520significance%2520and%2520role%2520in%2520radiation%2520responses%26jtitle%3DRadiother.%2520Oncol.%26date%3D2004%26volume%3D72%26spage%3DS40%26epage%3DS41%26doi%3D10.1016%2FS0167-8140%2804%2980634-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pratilas, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span> <span> </span><span class="NLM_article-title">Therapeutic strategies for targeting BRAF in human cancer</span>. <i>Rev. Recent Clin. Trials</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.2174/157488707780599393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.2174%2F157488707780599393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=18473997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2sXms1Oisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=121-134&author=C.+A.+Pratilasauthor=D.+B.+Solit&title=Therapeutic+strategies+for+targeting+BRAF+in+human+cancer&doi=10.2174%2F157488707780599393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic strategies for targeting BRAF in human cancer</span></div><div class="casAuthors">Pratilas, Christine A.; Solit, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Reviews on Recent Clinical Trials</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-134</span>CODEN:
                <span class="NLM_cas:coden">RRCTB2</span>;
        ISSN:<span class="NLM_cas:issn">1574-8871</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Constitutive ERK activation is a common finding in human cancer and is often the result of activating mutations of BRAF and RAS.  BRAF missense mutations occur in approx. 8% of human tumors, most frequently in melanoma, papillary thyroid cancer and colon cancer.  Mutations in BRAF have been found predominantly in tumors in which RAS is commonly mutated but concurrent mutations of both BRAF and RAS are extremely rare.  Though over 40 different kinase domain mutations in BRAF have been identified, a single base-pair substitution in exon 15 at codon 600 (V600E) is found in over 80% of cases.  These mutations cluster in the glycine-rich loop and activation segments of the kinase and are predicted to induce kinase activation by disrupting the inhibitory glycine-rich loop/activation segment interaction which characterizes the inactive conformation.  The majority of mutations identified cause constitutive kinase activation with the V600E mutation demonstrating approx. 500-fold greater kinase activity than wild-type BRAF.  Supporting its classification as an oncogene, V600E BRAF stimulates ERK signaling, induces proliferation and is capable in model systems of promoting transformation.  However, BRAF mutations are common in nevi and colon polyps suggesting that BRAF mutation alone is insufficient for tumorigenesis and addnl. mutations are required for cancer development.  Though such data suggest that BRAF mutation is likely an early initiating event in tumors such as melanoma and colon cancer, preclin. studies suggest that tumors with V600E BRAF mutation remain dependent upon BRAF for proliferation and survival.  Given its frequent occurrence in human cancer and the continued requirement for BRAF activity in tumors with BRAF mutation, efforts are underway to develop targeted inhibitors of BRAF and its downstream effectors.  The first generation of RAF inhibitors, including sorafenib, were notable for their lack of specificity and potency for RAF and these agents have shown limited efficacy in tumors with a high incidence of BRAF mutation such as melanoma.  Novel inhibitors of the pathway with greater selectivity for BRAF and MEK are now in Phase 1 and 2 clin. trials with promising early results.  To maximize the likelihood of success with these agents, clin. trials enriched with patients whose tumors possess BRAF and RAS mutations have been proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGhODO3iHKHLVg90H21EOLACvtfcHk0ljQyF1ldqXanQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXms1Oisrs%253D&md5=78477e12284560df03576ef0d9b7fd8a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F157488707780599393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157488707780599393%26sid%3Dliteratum%253Aachs%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26atitle%3DTherapeutic%2520strategies%2520for%2520targeting%2520BRAF%2520in%2520human%2520cancer%26jtitle%3DRev.%2520Recent%2520Clin.%2520Trials%26date%3D2007%26volume%3D2%26spage%3D121%26epage%3D134%26doi%3D10.2174%2F157488707780599393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridlyand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kageshita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busam, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutzner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brocker, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBoit, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastian, B. C.</span></span> <span> </span><span class="NLM_article-title">Distinct sets of genetic alterations in melanoma</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">2135</span>– <span class="NLM_lpage">2147</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa050092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1056%2FNEJMoa050092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=16291983" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ajtLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2005&pages=2135-2147&author=J.+A.+Curtinauthor=J.+Fridlyandauthor=T.+Kageshitaauthor=H.+N.+Patelauthor=K.+J.+Busamauthor=H.+Kutznerauthor=K.+H.+Choauthor=S.+Aibaauthor=E.+B.+Brockerauthor=P.+E.+LeBoitauthor=D.+Pinkelauthor=B.+C.+Bastian&title=Distinct+sets+of+genetic+alterations+in+melanoma&doi=10.1056%2FNEJMoa050092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct sets of genetic alterations in melanoma</span></div><div class="casAuthors">Curtin, John A.; Fridlyand, Jane; Kageshita, Toshiro; Patel, Hetal N.; Busam, Klaus J.; Kutzner, Heinz; Cho, Kwang-Hyun; Aiba, Setsuya; Brocker, Eva-Bettina; LeBoit, Philip E.; Pinkel, Dan; Bastian, Boris C.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2135-2147</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND Exposure to UV light is a major causative factor in melanoma, although the relationship between risk and exposure is complex.  We hypothesized that the clin. heterogeneity is explained by genetically distinct types of melanoma with different susceptibility to UV light.  METHODS We compared genome-wide alterations in the no. of copies of DNA and mutational status of BRAF and N-RAS in 126 melanomas from four groups in which the degree of exposure to UV light differs: 30 melanomas from skin with chronic sun-induced damage and 40 melanomas from skin without such damage; 36 melanomas from palms, soles, and subungual (acral) sites; and 20 mucosal melanomas.  RESULTS We found significant differences in the frequencies of regional changes in the no. of copies of DNA and mutation frequencies in BRAF among the four groups of melanomas.  Samples could be correctly classified into the four groups with 70 percent accuracy on the basis of the changes in the no. of copies of genomic DNA.  In two-way com- parisons, melanomas arising on skin with signs of chronic sun-induced damage and skin without such signs could be correctly classified with 84 percent accuracy.  Acral melanoma could be distinguished from mucosal melanoma with 89 percent accuracy.  Eighty-one percent of melanomas on skin without chronic sun-induced damage had mutations in BRAF or N-RAS; the majority of melanomas in the other groups had mutations in neither gene.  Melanomas with wild-type BRAF or N-RAS frequently had increases in the no. of copies of the genes for cyclin-dependent kinase 4 (CDK4) and cyclin D1 (CCND1), downstream components of the RAS-BRAF pathway.  CONCLUSIONS The genetic alterations identified in melanomas at different sites and with different levels of sun exposure indicate that there are distinct genetic pathways in the development of melanoma and implicate CDK4 and CCND1 as independent oncogenes in melanomas without mutations in BRAF or N-RAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLgiGsuGhDMLVg90H21EOLACvtfcHk0ljQyF1ldqXanQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ajtLjP&md5=73fd3ec239a651dcc755fccd5336eab7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa050092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa050092%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DJ.%2BA.%26aulast%3DFridlyand%26aufirst%3DJ.%26aulast%3DKageshita%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DH.%2BN.%26aulast%3DBusam%26aufirst%3DK.%2BJ.%26aulast%3DKutzner%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DK.%2BH.%26aulast%3DAiba%26aufirst%3DS.%26aulast%3DBrocker%26aufirst%3DE.%2BB.%26aulast%3DLeBoit%26aufirst%3DP.%2BE.%26aulast%3DPinkel%26aufirst%3DD.%26aulast%3DBastian%26aufirst%3DB.%2BC.%26atitle%3DDistinct%2520sets%2520of%2520genetic%2520alterations%2520in%2520melanoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D353%26spage%3D2135%26epage%3D2147%26doi%3D10.1056%2FNEJMoa050092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goel, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warneke, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redston, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haluska, F. G.</span></span> <span> </span><span class="NLM_article-title">Examination of mutations in BRAF, NRAS, and PTEN in primary cutaneous melanoma</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1038/sj.jid.5700026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fsj.jid.5700026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=16417231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD28Xlt1Wju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2006&pages=154-160&author=V.+K.+Goelauthor=A.+J.+Lazarauthor=C.+L.+Warnekeauthor=M.+S.+Redstonauthor=F.+G.+Haluska&title=Examination+of+mutations+in+BRAF%2C+NRAS%2C+and+PTEN+in+primary+cutaneous+melanoma&doi=10.1038%2Fsj.jid.5700026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Examination of mutations in BRAF, NRAS, and PTEN in primary cutaneous melanoma</span></div><div class="casAuthors">Goel, Vikas K.; Lazar, Alexander J. F.; Warneke, Carla L.; Redston, Mark S.; Haluska, Frank G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">154-160</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Frequent somatic mutation of v-raf murine sarcoma viral oncogene homolog B (BRAF), a downstream effector of the rat sarcoma oncogene (RAS) signaling pathway, is described in melanoma and other tumors.  Our anal. of melanoma cell lines suggests that activating mutations in BRAF can occur simultaneously with inactivation of phosphatase and tensin homolog (PTEN), but neuroblastoma RAS (NRAS) mutations are not coincident.  We detd. the concurrent prevalence of mutations in BRAF and NRAS, and alteration of PTEN expression in 69 primary cutaneous melanomas.  BRAF mutations were seen in 57% of cases.  NRAS was mutated in 17% of samples, exclusively in exon 2.  Two cases showed concurrent BRAF and NRAS mutations.  Using immunohistochem., PTEN protein expression was lost or greatly reduced in 19% of tumors.  Seven tumors with reduced PTEN yielded DNA amenable to sequencing, and three also showed mutation in BRAF but none in NRAS.  In all, 11 (85%) of 13 tumors showing reduced PTEN expression were greater than 3.5 mm thick, and the assocn. of increasing Breslow thickness and loss or redn. of PTEN expression was statistically significant (P<0.0001).  Mutations in NRAS were not coincident with reduced PTEN expression, and the concurrent mutation of NRAS and BRAF was rare.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnKUIL416mYbVg90H21EOLACvtfcHk0ljQyF1ldqXanQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xlt1Wju7g%253D&md5=05dc8622b7b5ccfd17ae285f0b7bc2e0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsj.jid.5700026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.jid.5700026%26sid%3Dliteratum%253Aachs%26aulast%3DGoel%26aufirst%3DV.%2BK.%26aulast%3DLazar%26aufirst%3DA.%2BJ.%26aulast%3DWarneke%26aufirst%3DC.%2BL.%26aulast%3DRedston%26aufirst%3DM.%2BS.%26aulast%3DHaluska%26aufirst%3DF.%2BG.%26atitle%3DExamination%2520of%2520mutations%2520in%2520BRAF%252C%2520NRAS%252C%2520and%2520PTEN%2520in%2520primary%2520cutaneous%2520melanoma%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2006%26volume%3D126%26spage%3D154%26epage%3D160%26doi%3D10.1038%2Fsj.jid.5700026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adjei, A. A.</span></span> <span> </span><span class="NLM_article-title">The Ras/Raf/MAPK pathway</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1016/S1556-0864(15)31506-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1556-0864%2815%2931506-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=17409820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BD2s3gs1yrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=7-9&author=J.+R.+Molinaauthor=A.+A.+Adjei&title=The+Ras%2FRaf%2FMAPK+pathway&doi=10.1016%2FS1556-0864%2815%2931506-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The Ras/Raf/MAPK pathway</span></div><div class="casAuthors">Molina Julian R; Adjei Alex A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyYQXfFy_LN0ewzGPswiWOfW6udTcc2eYrU8FUlvp2pLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s3gs1yrug%253D%253D&md5=f36f1f3f53464529a76dbcbbf2efdb8a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS1556-0864%2815%2931506-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1556-0864%252815%252931506-9%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DJ.%2BR.%26aulast%3DAdjei%26aufirst%3DA.%2BA.%26atitle%3DThe%2520Ras%252FRaf%252FMAPK%2520pathway%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2006%26volume%3D1%26spage%3D7%26epage%3D9%26doi%3D10.1016%2FS1556-0864%2815%2931506-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilmartin, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleam, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minthorn, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rominger, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erskine, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zappacosta, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laquerre, S. G.</span></span> <span> </span><span class="NLM_article-title">GSK1120212 (JTP-74057) is an inhibitor of MEK activity and activation with favorable pharmacokinetic properties for sustained in vivo pathway inhibition</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1000</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-10-2200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F1078-0432.CCR-10-2200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=21245089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1eit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=989-1000&author=A.+G.+Gilmartinauthor=M.+R.+Bleamauthor=A.+Groyauthor=K.+G.+Mossauthor=E.+A.+Minthornauthor=S.+G.+Kulkarniauthor=C.+M.+Romingerauthor=S.+Erskineauthor=K.+E.+Fisherauthor=J.+Yangauthor=F.+Zappacostaauthor=R.+Annanauthor=D.+Suttonauthor=S.+G.+Laquerre&title=GSK1120212+%28JTP-74057%29+is+an+inhibitor+of+MEK+activity+and+activation+with+favorable+pharmacokinetic+properties+for+sustained+in+vivo+pathway+inhibition&doi=10.1158%2F1078-0432.CCR-10-2200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">GSK1120212 (JTP-74057) Is an Inhibitor of MEK Activity and Activation with Favorable Pharmacokinetic Properties for Sustained In Vivo Pathway Inhibition</span></div><div class="casAuthors">Gilmartin, Aidan G.; Bleam, Maureen R.; Groy, Arthur; Moss, Katherine G.; Minthorn, Elisabeth A.; Kulkarni, Swarupa G.; Rominger, Cynthia M.; Erskine, Symon; Fisher, Kelly E.; Yang, Jing-Song; Zappacosta, Francesca; Annan, Roland; Sutton, David; Laquerre, Sylvie G.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">989-1000</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Despite their preclin. promise, previous MEK inhibitors have shown little benefit for patients.  This likely reflects the narrow therapeutic window for MEK inhibitors due to the essential role of the P42/44 MAPK pathway in many nontumor tissues.  GSK1120212 is a potent and selective allosteric inhibitor of the MEK1 and MEK2 (MEK1/2) enzymes with promising antitumor activity in a phase I clin. trial (ASCO 2010).  Our studies characterize GSK1120212' enzymic, cellular, and in vivo activities, describing its unusually long circulating half-life.  Exptl. Design: Enzymic studies were conducted to det. GSK1120212 inhibition of recombinant MEK, following or preceding RAF kinase activation.  Cellular studies examd. GSK1120212 inhibition of ERK1 and 2 phosphorylation (p-ERK1/2) as well as MEK1/2 phosphorylation and activation.  Further studies explored the sensitivity of cancer cell lines, and drug pharmacokinetics and efficacy in multiple tumor xenograft models.  Results: In enzymic and cellular studies, GSK1120212 inhibits MEK1/2 kinase activity and prevents Raf-dependent MEK phosphorylation (S217 for MEK1), producing prolonged p-ERK1/2 inhibition.  Potent cell growth inhibition was evident in most tumor lines with mutant BRAF or Ras.  In xenografted tumor models, GSK1120212 orally dosed once daily had a long circulating half-life and sustained suppression of p-ERK1/2 for more than 24 h; GSK1120212 also reduced tumor Ki67, increased p27Kip1/CDKN1B, and caused tumor growth inhibition in multiple tumor models.  The largest antitumor effect was among tumors harboring mutant BRAF or Ras.  Conclusions: GSK1120212 combines high potency, selectivity, and long circulating half-life, offering promise for successfully targeting the narrow therapeutic window anticipated for clin. MEK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMOQvgszKIfbVg90H21EOLACvtfcHk0liljSVAvCs_ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXis1eit7k%253D&md5=6154be5fa0afb034c6a5995508d7ea9e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-2200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-2200%26sid%3Dliteratum%253Aachs%26aulast%3DGilmartin%26aufirst%3DA.%2BG.%26aulast%3DBleam%26aufirst%3DM.%2BR.%26aulast%3DGroy%26aufirst%3DA.%26aulast%3DMoss%26aufirst%3DK.%2BG.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DKulkarni%26aufirst%3DS.%2BG.%26aulast%3DRominger%26aufirst%3DC.%2BM.%26aulast%3DErskine%26aufirst%3DS.%26aulast%3DFisher%26aufirst%3DK.%2BE.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZappacosta%26aufirst%3DF.%26aulast%3DAnnan%26aufirst%3DR.%26aulast%3DSutton%26aufirst%3DD.%26aulast%3DLaquerre%26aufirst%3DS.%2BG.%26atitle%3DGSK1120212%2520%2528JTP-74057%2529%2520is%2520an%2520inhibitor%2520of%2520MEK%2520activity%2520and%2520activation%2520with%2520favorable%2520pharmacokinetic%2520properties%2520for%2520sustained%2520in%2520vivo%2520pathway%2520inhibition%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2011%26volume%3D17%26spage%3D989%26epage%3D1000%26doi%3D10.1158%2F1078-0432.CCR-10-2200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Den Otter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeppen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engst, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvin, M.</span></span> <span> </span><span class="NLM_article-title">Intermittent administration of MEK inhibitor GDC-0973 plus PI3K inhibitor GDC-0941 triggers robust apoptosis and tumor growth inhibition</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-1515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F0008-5472.CAN-11-1515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=22084396" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=210-219&author=K.+P.+Hoeflichauthor=M.+Merchantauthor=C.+Orrauthor=J.+Chanauthor=D.+Den+Otterauthor=L.+Berryauthor=I.+Kasmanauthor=H.+Koeppenauthor=K.+Riceauthor=N.+Y.+Yangauthor=S.+Engstauthor=S.+Johnstonauthor=L.+S.+Friedmanauthor=M.+Belvin&title=Intermittent+administration+of+MEK+inhibitor+GDC-0973+plus+PI3K+inhibitor+GDC-0941+triggers+robust+apoptosis+and+tumor+growth+inhibition&doi=10.1158%2F0008-5472.CAN-11-1515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Intermittent Administration of MEK Inhibitor GDC-0973 plus PI3K Inhibitor GDC-0941 Triggers Robust Apoptosis and Tumor Growth Inhibition</span></div><div class="casAuthors">Hoeflich, Klaus P.; Merchant, Mark; Orr, Christine; Chan, Jocelyn; Den Otter, Doug; Berry, Leanne; Kasman, Ian; Koeppen, Hartmut; Rice, Ken; Yang, Nai-Ying; Engst, Stefan; Johnston, Stuart; Friedman, Lori S.; Belvin, Marcia</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-219</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Combinations of MAP/ERK kinase (MEK) and phosphoinositide 3-kinase (PI3K) inhibitors have shown promise in preclin. cancer models, leading to the initiation of clin. trials cotargeting these two key cancer signaling pathways.  GDC-0973, a novel selective MEK inhibitor, and GDC-0941, a class I PI3K inhibitor, are in early stage clin. trials as both single agents and in combination.  The discovery of these selective inhibitors has allowed investigation into the precise effects of combining inhibitors of two major signaling branches downstream of RAS.  Here, we investigated multiple biomarkers in the mitogen-activated protein kinase (MAPK) and PI3K pathway to search for points of convergence that explain the increased apoptosis seen in combination.  Using washout studies in vitro and alternate dosing schedules in mice, we showed that intermittent inhibition of the PI3K and MAPK pathway is sufficient for efficacy in BRAF and KRAS mutant cancer cells.  The combination of GDC-0973 with the PI3K inhibitor GDC-0941 resulted in combination efficacy in vitro and in vivo via induction of biomarkers assocd. with apoptosis, including Bcl-2 family proapoptotic regulators.  Therefore, these data suggest that continuous exposure of MEK and PI3K inhibitors in combination is not required for efficacy in preclin. cancer models and that sustained effects on downstream apoptosis biomarkers can be obsd. in response to intermittent dosing.  Cancer Res; 72(1); 210-9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokHbA7g6hLGrVg90H21EOLACvtfcHk0liljSVAvCs_ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCltA%253D%253D&md5=40032bf35d5e6335856a645d5c331df8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-1515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-1515%26sid%3Dliteratum%253Aachs%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DOrr%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DDen%2BOtter%26aufirst%3DD.%26aulast%3DBerry%26aufirst%3DL.%26aulast%3DKasman%26aufirst%3DI.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DRice%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DN.%2BY.%26aulast%3DEngst%26aufirst%3DS.%26aulast%3DJohnston%26aufirst%3DS.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DBelvin%26aufirst%3DM.%26atitle%3DIntermittent%2520administration%2520of%2520MEK%2520inhibitor%2520GDC-0973%2520plus%2520PI3K%2520inhibitor%2520GDC-0941%2520triggers%2520robust%2520apoptosis%2520and%2520tumor%2520growth%2520inhibition%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D210%26epage%3D219%26doi%3D10.1158%2F0008-5472.CAN-11-1515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woessner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssikatos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poch, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, K.</span></span> <span> </span><span class="NLM_article-title">Abstract 2515: preclinical development of ARRY-162, a potent and selective MEK 1/2 inhibitor</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">2515</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM10-2515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F1538-7445.AM10-2515" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=2515&author=P.+A.+Leeauthor=E.+Wallaceauthor=A.+Marlowauthor=T.+Yehauthor=V.+Marshauthor=D.+Andersonauthor=R.+Woessnerauthor=B.+Hurleyauthor=J.+Lyssikatosauthor=G.+Pochauthor=S.+Grossauthor=S.+Ranaauthor=S.+Winskiauthor=K.+Koch&title=Abstract+2515%3A+preclinical+development+of+ARRY-162%2C+a+potent+and+selective+MEK+1%2F2+inhibitor&doi=10.1158%2F1538-7445.AM10-2515"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM10-2515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM10-2515%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DP.%2BA.%26aulast%3DWallace%26aufirst%3DE.%26aulast%3DMarlow%26aufirst%3DA.%26aulast%3DYeh%26aufirst%3DT.%26aulast%3DMarsh%26aufirst%3DV.%26aulast%3DAnderson%26aufirst%3DD.%26aulast%3DWoessner%26aufirst%3DR.%26aulast%3DHurley%26aufirst%3DB.%26aulast%3DLyssikatos%26aufirst%3DJ.%26aulast%3DPoch%26aufirst%3DG.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DRana%26aufirst%3DS.%26aulast%3DWinski%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DK.%26atitle%3DAbstract%25202515%253A%2520preclinical%2520development%2520of%2520ARRY-162%252C%2520a%2520potent%2520and%2520selective%2520MEK%25201%252F2%2520inhibitor%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D2515%26doi%3D10.1158%2F1538-7445.AM10-2515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, U. R.</span></span> <span> </span><span class="NLM_article-title">Use of mitogenic cascade blockers for treatment of C-Raf induced lung adenoma in vivo: CI-1040 strongly reduces growth and improves lung structure</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">24</span> <span class="refDoi"> DOI: 10.1186/1471-2407-4-24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1186%2F1471-2407-4-24" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=15171791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BD2czhsF2guw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&author=B.+W.+Kramerauthor=R.+Gotzauthor=U.+R.+Rapp&title=Use+of+mitogenic+cascade+blockers+for+treatment+of+C-Raf+induced+lung+adenoma+in+vivo%3A+CI-1040+strongly+reduces+growth+and+improves+lung+structure&doi=10.1186%2F1471-2407-4-24"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Use of mitogenic cascade blockers for treatment of C-Raf induced lung adenoma in vivo: CI-1040 strongly reduces growth and improves lung structure</span></div><div class="casAuthors">Kramer Boris W; Gotz Rudolf; Rapp Ulf R</div><div class="citationInfo"><span class="NLM_cas:title">BMC cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Signaling networks promoting cell growth and proliferation are frequently deregulated in cancer.  Tumors often are highly dependent on such signaling pathways and may become hypersensitive to downregulation of key components within these signaling cascades.  The classical mitogenic cascade transmits stimuli from growth factor receptors via Ras, Raf, MEK and ERK to the cell nucleus and provides attractive molecular targets for cancer treatment.  For example, Ras and Raf kinase inhibitors are already in a number of ongoing phase II and phase III clinical trials.  In this study the effect of the Raf kinase inhibitor BAY 43-9006 and of the MEK inhibitor CI-1040 (PD184352) on a Raf dependent lung tumor mouse model was analyzed in detail.  METHODS:  We have generated a lung cancer mouse model by targeting constitutively active C-Raf kinase to the lung.  These mice develop adenomas within 4 months of life.  At this time-point they received daily intraperitoneal injections of either 100 mg/kg BAY 43-9006 or CI-1040 for additional 21 days.  Thereafter, lungs were isolated and the following parameters were analyzed using histology and immunohistochemistry: overall lung structure, frequency of adenoma foci, proliferation rate, ERK activity, caspase-3 activation, and lung differentiation.  RESULTS:  Both inhibitors were equally effective in vitro using a sensitive Raf/MEK/ERK ELISA.  In vivo, the systemic administration of the MEK inhibitor CI-1040 reduced adenoma formation to a third and significantly restored lung structure.  The proliferation rate of lung cells of mice treated with CL-1040 was decreased without any obvious effects on differentiation of pneumocytes.  In contrast, the Raf inhibitor BAY 43-9006 did not influence adenoma formation in vivo.  CONCLUSION:  The MEK inhibitor CI-1040 may be used for the treatment of Ras and/or Raf-dependent human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSBd-ofJhdpmm52AMpI4GK2fW6udTcc2eYz4dqciGir2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czhsF2guw%253D%253D&md5=34f11f9b5bb568c02259d04f54b5664b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-4-24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-4-24%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DB.%2BW.%26aulast%3DGotz%26aufirst%3DR.%26aulast%3DRapp%26aufirst%3DU.%2BR.%26atitle%3DUse%2520of%2520mitogenic%2520cascade%2520blockers%2520for%2520treatment%2520of%2520C-Raf%2520induced%2520lung%2520adenoma%2520in%2520vivo%253A%2520CI-1040%2520strongly%2520reduces%2520growth%2520and%2520improves%2520lung%2520structure%26jtitle%3DBMC%2520Cancer%26date%3D2004%26volume%3D4%26doi%3D10.1186%2F1471-2407-4-24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeannin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vansteenkiste, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franke, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinsted, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zazulina, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crino, L.</span></span> <span> </span><span class="NLM_article-title">Selumetinib plus docetaxel for KRAS-mutant advanced non-small-cell lung cancer: a randomised, multicentre, placebo-controlled, phase 2 study</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(12)70489-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1470-2045%2812%2970489-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=23200175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BC3s7otVCnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=38-47&author=P.+A.+J%C3%A4nneauthor=A.+T.+Shawauthor=J.+R.+Pereiraauthor=G.+Jeanninauthor=J.+Vansteenkisteauthor=C.+Barriosauthor=F.+A.+Frankeauthor=L.+Grinstedauthor=V.+Zazulinaauthor=P.+Smithauthor=I.+Smithauthor=L.+Crino&title=Selumetinib+plus+docetaxel+for+KRAS-mutant+advanced+non-small-cell+lung+cancer%3A+a+randomised%2C+multicentre%2C+placebo-controlled%2C+phase+2+study&doi=10.1016%2FS1470-2045%2812%2970489-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Selumetinib plus docetaxel for KRAS-mutant advanced non-small-cell lung cancer: a randomised, multicentre, placebo-controlled, phase 2 study</span></div><div class="casAuthors">Janne Pasi A; Shaw Alice T; Pereira Jose Rodrigues; Jeannin Gaelle; Vansteenkiste Johan; Barrios Carlos; Franke Fabio Andre; Grinsted Lynda; Zazulina Victoria; Smith Paul; Smith Ian; Crino Lucio</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-47</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  No targeted therapies are available for KRAS-mutant non-small-cell lung cancer (NSCLC).  Selumetinib is an inhibitor of MEK1/MEK2, downstream of KRAS, with preclinical evidence of synergistic activity with docetaxel in KRAS-mutant cancers.  We did a prospective, randomised, phase 2 trial to assess selumetinib plus docetaxel in previously treated patients with advanced KRAS-mutant NSCLC.  METHODS:  Eligible patients were older than 18 years of age; had histologically or cytologically confirmed stage IIIB-IV KRAS-mutant NSCLC; had failed first-line therapy for advanced NSCLC; had WHO performance status of 0-1; had not received previous therapy with either a MEK inhibitor or docetaxel; and had adequate bone marrow, renal, and liver function.  Patients were randomly assigned (in a 1:1 ratio) to either oral selumetinib (75 mg twice daily in a 21 day cycle) or placebo; all patients received intravenous docetaxel (75 mg/m(2) on day 1 of a 21 day cycle).  Randomisation was done with an interactive voice response system and investigators, patients, data analysts, and the trial sponsor were masked to treatment assignment.  The primary endpoint was overall survival, analysed for all patients with confirmed KRAS mutations.  This study is registered with ClinicalTrials.gov, number NCT00890825.  FINDINGS:  Between April 20, 2009, and June 30, 2010, we randomly assigned 44 patients to receive selumetinib and docetaxel (selumetinib group) and 43 to receive placebo and docetaxel (placebo group).  Of these, one patient in the selumetinib group and three in the placebo group were excluded from efficacy analyses because their tumours were not confirmed to be KRAS-mutation positive.  Median overall survival was 9·4 months (6·8-13·6) in the selumetinib group and 5·2 months (95% CI 3·8-non-calculable) in the placebo group (hazard ratio [HR] for death 0·80, 80% CI 0·56-1·14; one-sided p=0·21).  Median progression-free survival was 5·3 months (4·6-6·4) in the selumetinib group and 2·1 months (95% CI 1·4-3·7) in the placebo group (HR for progression 0·58, 80% CI 0·42-0·79; one-sided p=0·014). 16 (37%) patients in the selumetinib group and none in the placebo group had an objective response (p<0·0001).  Adverse events of grade 3 or higher occurred in 36 (82%) patients in the selumetinib group and 28 (67%) patients in the placebo group.  The most common grade 3-4 adverse events were neutropenia (29 [67%] of 43 patients in the selumetinib group vs 23 [55%] of 42 patients in the placebo group), febrile neutropenia (eight [18%] of 44 patients in the selumetinib group vs none in the placebo group), dyspnoea (one [2%] of 44 patients in the selumetinib group vs five [12%] of 42 in the placebo group), and asthenia (four [9%] of 44 patients in the selumetinib group vs none in the placebo group).  INTERPRETATION:  Selumetinib plus docetaxel has promising efficacy, albeit with a higher number of adverse events than with docetaxel alone, in previously treated advanced KRAS-mutant NSCLC.  These findings warrant further clinical investigation of selumetinib plus docetaxel in KRAS-mutant NSCLC.  FUNDING:  AstraZeneca.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR_DnSQjq4ctlm8bhe6roZYfW6udTcc2eYz4dqciGir2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7otVCnug%253D%253D&md5=3987f8d7cb411810eb05c507a0bc9a73</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2812%2970489-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252812%252970489-8%26sid%3Dliteratum%253Aachs%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DShaw%26aufirst%3DA.%2BT.%26aulast%3DPereira%26aufirst%3DJ.%2BR.%26aulast%3DJeannin%26aufirst%3DG.%26aulast%3DVansteenkiste%26aufirst%3DJ.%26aulast%3DBarrios%26aufirst%3DC.%26aulast%3DFranke%26aufirst%3DF.%2BA.%26aulast%3DGrinsted%26aufirst%3DL.%26aulast%3DZazulina%26aufirst%3DV.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DI.%26aulast%3DCrino%26aufirst%3DL.%26atitle%3DSelumetinib%2520plus%2520docetaxel%2520for%2520KRAS-mutant%2520advanced%2520non-small-cell%2520lung%2520cancer%253A%2520a%2520randomised%252C%2520multicentre%252C%2520placebo-controlled%252C%2520phase%25202%2520study%26jtitle%3DLancet%2520Oncol.%26date%3D2013%26volume%3D14%26spage%3D38%26epage%3D47%26doi%3D10.1016%2FS1470-2045%2812%2970489-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span> </span><span class="NLM_article-title">MEK Inhibitor PD-325901 To Treat Advanced
Breast Cancer, Colon Cancer,
And Melanoma</span>. <a href="https://clinicaltrials.gov/ct2/show/NCT00147550" class="extLink">https://clinicaltrials.gov/ct2/show/NCT00147550</a> (accessed Sep 7, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MEK+Inhibitor+PD-325901+To+Treat+Advanced%0ABreast+Cancer%2C+Colon+Cancer%2C%0AAnd+Melanoma.+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT00147550+%28accessed+Sep+7%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DMEK%2520Inhibitor%2520PD-325901%2520To%2520Treat%2520Advanced%250ABreast%2520Cancer%252C%2520Colon%2520Cancer%252C%250AAnd%2520Melanoma%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, A. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uddin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunkel, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petriccione, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haque, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenblum, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisapia, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratilas, C. A.</span></span> <span> </span><span class="NLM_article-title">A secondary mutation in BRAF confers resistance to RAF inhibition in a BRAF(V600E)-mutant brain tumor</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1130</span>– <span class="NLM_lpage">1141</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-17-1263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F2159-8290.CD-17-1263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29880583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFyju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1130-1141&author=J.+Wangauthor=Z.+Yaoauthor=P.+Jonssonauthor=A.+N.+Allenauthor=A.+C.+R.+Qinauthor=S.+Uddinauthor=I.+J.+Dunkelauthor=M.+Petriccioneauthor=K.+Manovaauthor=S.+Haqueauthor=M.+K.+Rosenblumauthor=D.+J.+Pisapiaauthor=N.+Rosenauthor=B.+S.+Taylorauthor=C.+A.+Pratilas&title=A+secondary+mutation+in+BRAF+confers+resistance+to+RAF+inhibition+in+a+BRAF%28V600E%29-mutant+brain+tumor&doi=10.1158%2F2159-8290.CD-17-1263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A seconcitry mutation in BRAF confers resistanie to RAF inhibition in a BRAFV600E-mutant brain tumor</span></div><div class="casAuthors">Wang, Jiawan; Yao, Zhan; Jonsson, Philip; Allen, Amy N.; Qin, Alice Can Ran; Uddin, Sharnieen; Dunke, Ira J.; Petriccione, Mary; Manova, Katia; Hague, Sofia; Rosenblum, Marc K.; Pisapia, David J.; Rosen, Neal; Taylor, Barry S.; Pratilas, Christine A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1130-1141</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">BRAFV600E hyperactivates ERK and signals as a RAF inhibitor-sensitive monomer.  Although RAF inhibitors can produce impressive clin. responses in patients with mutant BRAF tumors, the mechanisms of resistance to these drugs are incompletely characterized.  Here, we report a complete response followed by clin. progression in a patient with a BRAFV600E-mutant brain tumor treated with dabrafenib.  Whole-exome sequencing revealed a secondary BRAFL514V mutation at progression that was not present in the pretreatment tumor.  Expressing BRAFV600E/L514V induces ERK signaling, promotes RAF dimer formation, and is sufficient to confer resistance to dabrafenib.  Newer RAF dimer inhibitors and an ERK inhibitor are effective against BRAFL514V-mediated resistance.  Collectively, our results validate a novel biochem. mechanism of RAF inhibitor resistance mediated by a secondary mutation, emphasizing that, like driver mutations in cancer, the spectrum of mutations that drive resist- ance to targeted therapy are heterogeneous and perhaps emerge with a lineage-specific prevalence.  SIGNIFICANCE: In contrast to receptor tyrosine kinases, in which secondary mutations are often responsible for acquired resistance, second-site mutations in BRAF have not been validated in clini- cally acquired resistance to RAF inhibitors.  We demonstrate a secondary mutation in BRAF (V600E/ L514V) following progression on dabrafenib and confirm functionally that this mutation is responsible for resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV9HqL6aWJ5bVg90H21EOLACvtfcHk0liOc7pQUgztlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFyju74%253D&md5=98359fdb20ec6b0f4e94f53eaf30d5eb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-1263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-1263%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DJonsson%26aufirst%3DP.%26aulast%3DAllen%26aufirst%3DA.%2BN.%26aulast%3DQin%26aufirst%3DA.%2BC.%2BR.%26aulast%3DUddin%26aufirst%3DS.%26aulast%3DDunkel%26aufirst%3DI.%2BJ.%26aulast%3DPetriccione%26aufirst%3DM.%26aulast%3DManova%26aufirst%3DK.%26aulast%3DHaque%26aufirst%3DS.%26aulast%3DRosenblum%26aufirst%3DM.%2BK.%26aulast%3DPisapia%26aufirst%3DD.%2BJ.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DTaylor%26aufirst%3DB.%2BS.%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26atitle%3DA%2520secondary%2520mutation%2520in%2520BRAF%2520confers%2520resistance%2520to%2520RAF%2520inhibition%2520in%2520a%2520BRAF%2528V600E%2529-mutant%2520brain%2520tumor%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D1130%26epage%3D1141%26doi%3D10.1158%2F2159-8290.CD-17-1263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koya, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriceau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hugo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chodon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kefford, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, R. S.</span></span> <span> </span><span class="NLM_article-title">A novel AKT1 mutant amplifies an adaptive melanoma response to BRAF inhibition</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-13-0279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1158%2F2159-8290.CD-13-0279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=24265152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVCmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=69-79&author=H.+Shiauthor=A.+Hongauthor=X.+Kongauthor=R.+C.+Koyaauthor=C.+Songauthor=G.+Moriceauauthor=W.+Hugoauthor=C.+C.+Yuauthor=C.+Ngauthor=T.+Chodonauthor=R.+A.+Scolyerauthor=R.+F.+Keffordauthor=A.+Ribasauthor=G.+V.+Longauthor=R.+S.+Lo&title=A+novel+AKT1+mutant+amplifies+an+adaptive+melanoma+response+to+BRAF+inhibition&doi=10.1158%2F2159-8290.CD-13-0279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel AKT1 Mutant Amplifies an Adaptive Melanoma Response to BRAF Inhibition</span></div><div class="casAuthors">Shi, Hubing; Hong, Aayoung; Kong, Xiangju; Koya, Richard C.; Song, Chunying; Moriceau, Gatien; Hugo, Willy; Yu, Clarissa C.; Ng, Charles; Chodon, Thinle; Scolyer, Richard A.; Kefford, Richard F.; Ribas, Antoni; Long, Georgina V.; Lo, Roger S.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-79</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">BRAF inhibitor (BRAFi) therapy leads to remarkable anti melanoma responses, but the initial tumor shrinkage is commonly incomplete, providing a nidus for subsequent disease progression.  Adaptive signaling may underlie early BRAFi resistance and influence the selection pattern for genetic variants, causing late, acquired resistance.  We show here that BRAFi (or BRAFi + MEKi) therapy in patients frequently led to rebound phosphorylated AKT (p-AKT) levels in their melanomas early on-treatment.  In cell lines, BRAFi treatment led to rebound levels of receptor tyrosine kinases (RTK; including PDGFβ), phosphatidyl (3,4,5)-triphosphate (PIP3), pleckstrin homol. domain recruitment, and p-AKT.  PTEN expression limited this BRAFi-elicited PI3K-AKT signaling, which could be rescued by the introduction of a mutant AKT1 (Q79K) known to confer acquired BRAFi resistance.  Functionally, AKTIQ79k conferred BRAFi resistance via amplification of BRAFi-elicited P13K-AKT signaling.  In addn., mitogen- activated protein kinase pathway inhibition enhanced clonogenic growth dependency on PI3K or AKT.  Thus, adaptive or genetic upregulation of AKT critically participates in melanoma survival during BRAFi therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWO5YkeSGMcrVg90H21EOLACvtfcHk0liOc7pQUgztlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVCmsw%253D%253D&md5=aaf2a9e96591b5f5e2e8180400d0b641</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-13-0279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-13-0279%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DA.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DKoya%26aufirst%3DR.%2BC.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DMoriceau%26aufirst%3DG.%26aulast%3DHugo%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DC.%2BC.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DChodon%26aufirst%3DT.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DKefford%26aufirst%3DR.%2BF.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DLo%26aufirst%3DR.%2BS.%26atitle%3DA%2520novel%2520AKT1%2520mutant%2520amplifies%2520an%2520adaptive%2520melanoma%2520response%2520to%2520BRAF%2520inhibition%26jtitle%3DCancer%2520Discovery%26date%3D2014%26volume%3D4%26spage%3D69%26epage%3D79%26doi%3D10.1158%2F2159-8290.CD-13-0279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lorusso, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adjei, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varterasian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadgeel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLuca, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruzek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asbury, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Becelaere, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. B.</span></span> <span> </span><span class="NLM_article-title">Phase I and pharmacodynamic study of the oral MEK inhibitor CI-1040 in patients with advanced malignancies</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5281</span>– <span class="NLM_lpage">5293</span>, <span class="refDoi"> DOI: 10.1200/JCO.2005.14.415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1200%2FJCO.2005.14.415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=16009947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslOrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=5281-5293&author=P.+M.+Lorussoauthor=A.+A.+Adjeiauthor=M.+Varterasianauthor=S.+Gadgeelauthor=J.+Reidauthor=D.+Y.+Mitchellauthor=L.+Hansonauthor=P.+DeLucaauthor=L.+Bruzekauthor=J.+Piensauthor=P.+Asburyauthor=K.+Van+Becelaereauthor=R.+Herreraauthor=J.+Sebolt-Leopoldauthor=M.+B.+Meyer&title=Phase+I+and+pharmacodynamic+study+of+the+oral+MEK+inhibitor+CI-1040+in+patients+with+advanced+malignancies&doi=10.1200%2FJCO.2005.14.415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I and pharmacodynamic study of the oral MEK inhibitor CI-1040 in patients with advanced malignancies</span></div><div class="casAuthors">LoRusso, Patricia M.; Adjei, Alex A.; Varterasian, Mary; Gadgeel, Shirish; Reid, Joel; Mitchell, David Y.; Hanson, Lorelei; DeLuca, Pamela; Bruzek, Laura; Piens, Jill; Asbury, Peggy; Van Becelaere, Keri; Herrera, Roman; Sebolt-Leopold, Judith; Meyer, Mark B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5281-5293</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose: This phase I study was undertaken to define the toxicity, pharmacokinetics, pharmacodynamics, max. tolerated dose (MTD), and clin. activity of CI-1040, a small-mol. inhibitor of the dual-specificity kinases MEK(mitogen-activated protein kinase kinase) -1 and MEK2, in patients with advanced malignancy.  Patients and Methods: CI-1040 was tested in multiple daily dosing frequencies administered for 21 days repeated every 28 days leading ultimately to continuous administration, and effect of food on absorption was tested.  Single dose and steady-state pharmacokinetics were assessed during cycle 1 and phosphorylated extracellular receptor kinase (pERK) levels were assessed in WBCs and also in tumor tissue from selected patients.  Results: Seventy-seven patients received CI-1040 at dose levels ranging from 100 mg QD to 800 mg tid.  Grade 3 asthenia was dose limiting at the highest dose level tested, 800 mg tid administered with food.  Ninety-eight percent of all drug-related adverse events were grade 1 or 2 in severity; most common toxicities included diarrhea, asthenia, rash, nausea, and vomiting.  Plasma concns. of CI-1040 and its active metabolite, PD 0184264, increased in a less than dose proportional manner from 100 to 800 mg QD.  Administration with a high-fat meal resulted in an increase in drug exposure.  The MTD and recommended phase II dose was 800 mg BID administered with food.  Sixty-six patients were assessable for response.  One partial response was achieved in a patient with pancreatic cancer and 19 patients (28%) achieved stable disease lasting a median of 5.5 mo (range, 4 to 17 mo).  Inhibition of tumor pERK (median, 73%; range, 46% to 100%) was demonstrated in 10 patients.  Conclusion: CI-1040 was well tolerated at 800 mg BID administered with food.  Both target suppression and antitumor activity were demonstrated in this phase I study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn9_7twRmM4LVg90H21EOLACvtfcHk0lh3RBjm0AfOlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslOrtL8%253D&md5=6af34b8a2037ede667d91e6a3e884d87</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1200%2FJCO.2005.14.415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2005.14.415%26sid%3Dliteratum%253Aachs%26aulast%3DLorusso%26aufirst%3DP.%2BM.%26aulast%3DAdjei%26aufirst%3DA.%2BA.%26aulast%3DVarterasian%26aufirst%3DM.%26aulast%3DGadgeel%26aufirst%3DS.%26aulast%3DReid%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DD.%2BY.%26aulast%3DHanson%26aufirst%3DL.%26aulast%3DDeLuca%26aufirst%3DP.%26aulast%3DBruzek%26aufirst%3DL.%26aulast%3DPiens%26aufirst%3DJ.%26aulast%3DAsbury%26aufirst%3DP.%26aulast%3DVan%2BBecelaere%26aufirst%3DK.%26aulast%3DHerrera%26aufirst%3DR.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DMeyer%26aufirst%3DM.%2BB.%26atitle%3DPhase%2520I%2520and%2520pharmacodynamic%2520study%2520of%2520the%2520oral%2520MEK%2520inhibitor%2520CI-1040%2520in%2520patients%2520with%2520advanced%2520malignancies%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2005%26volume%3D23%26spage%3D5281%26epage%3D5293%26doi%3D10.1200%2FJCO.2005.14.415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wabnitz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wabnitz, D. A.</span></span> <span> </span><span class="NLM_article-title">In vitro and in vivo metabolism of the anti-cancer agent CI-1040, a MEK inhibitor, in rat, monkey, and human</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1670</span>– <span class="NLM_lpage">1679</span>, <span class="refDoi"> DOI: 10.1023/B:PHAM.0000041464.27579.d0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1023%2FB%3APHAM.0000041464.27579.d0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=15497695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFOqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2004&pages=1670-1679&author=P.+A.+Wabnitzauthor=D.+Mitchellauthor=D.+A.+Wabnitz&title=In+vitro+and+in+vivo+metabolism+of+the+anti-cancer+agent+CI-1040%2C+a+MEK+inhibitor%2C+in+rat%2C+monkey%2C+and+human&doi=10.1023%2FB%3APHAM.0000041464.27579.d0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro and in Vivo Metabolism of the Anti-Cancer Agent CI-1040, a MEK Inhibitor, in Rat, Monkey, and Human</span></div><div class="casAuthors">Wabnitz, Paul A.; Mitchell, David; Wabnitz, David A. M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1670-1679</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer Science+Business Media, Inc.</span>)
        </div><div class="casAbstract">Purpose. The use of in vitro and in vivo models using both rodent and nonrodent species plays an important role with regard to metab. during the drug development process.  In this study, the authors compared the metab. of a MEK inhibitor (CI-1040) using in vitro and in vivo models with that obsd. in a cancer patient.  Methods. Radiolabeled CI-1040 was assessed for metab. using rat and monkey liver microsomes and hepatocytes, as well as in Wistar rats and cynomolgus monkeys via oral administration.  Human bile and plasma samples were obtained immediately after administration of CI-1040 to a patient with advanced colon cancer.  A combination of HPLC-radiochromatog. (HPLC-RAM), LC/MS and LC/MS/MS expts. were used to analyze all resulting metabolites.  Unlabeled CI-1040 was administered (100 mg/day, QD) for 15 days to a patient suffering from colon cancer.  Bile was collected by the insertion of a T-tube directly into the bile duct over a 14-h period.  Metabolites were also monitored in the patient's plasma.  Results. Anal. of the metabolites in all species using in vitro and animal models demonstrated that CI-1040 undergoes extensive oxidative metab. (14 metabolites identified) with subsequent glucuronidation of the hydroxylated metabolites.  Metabolites were predominantly excreted through the bile in the animal models.  Conclusions. Overall, the in vitro and animal models in combination provided comprehensive coverage for all metabolites obsd. in human bile and plasma.  In conclusion, the results obtained in this study demonstrate the utility of conducting investigations across species in order to gain complete coverage for successfully predicting human metabolites of new compds. in development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrprxSVj_DzBrVg90H21EOLACvtfcHk0lh3RBjm0AfOlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFOqu7w%253D&md5=f1303d36674c1190ecf2213996783edb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1023%2FB%3APHAM.0000041464.27579.d0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253APHAM.0000041464.27579.d0%26sid%3Dliteratum%253Aachs%26aulast%3DWabnitz%26aufirst%3DP.%2BA.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DWabnitz%26aufirst%3DD.%2BA.%26atitle%3DIn%2520vitro%2520and%2520in%2520vivo%2520metabolism%2520of%2520the%2520anti-cancer%2520agent%2520CI-1040%252C%2520a%2520MEK%2520inhibitor%252C%2520in%2520rat%252C%2520monkey%252C%2520and%2520human%26jtitle%3DPharm.%2520Res.%26date%3D2004%26volume%3D21%26spage%3D1670%26epage%3D1679%26doi%3D10.1023%2FB%3APHAM.0000041464.27579.d0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boerner, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">Clinical experience of MEK inhibitors in cancer therapy</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>1773</i></span>,  <span class="NLM_fpage">1248</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2006.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.bbamcr.2006.11.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=17194493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2sXosFyksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1773&publication_year=2007&pages=1248-1255&author=D.+Wangauthor=S.+A.+Boernerauthor=J.+D.+Winklerauthor=P.+M.+LoRusso&title=Clinical+experience+of+MEK+inhibitors+in+cancer+therapy&doi=10.1016%2Fj.bbamcr.2006.11.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical experience of MEK inhibitors in cancer therapy</span></div><div class="casAuthors">Wang, Ding; Boerner, Scott A.; Winkler, James D.; LoRusso, Patricia M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">1773</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1248-1255</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding new therapies to assist in the treatment of cancer is a major challenge of clin. research.  Small mols. that inhibit different mol. targets at the different levels of the MAPK pathway have been developed.  Several MEK inhibitors have been examd. in early-phase clin. trials and the current state of clin. results using these therapies is presented here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8_rSmIaOyh7Vg90H21EOLACvtfcHk0lh3RBjm0AfOlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXosFyksbs%253D&md5=93c566b1c3956e11a3a682e6cc9fc000</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2006.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2006.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DD.%26aulast%3DBoerner%26aufirst%3DS.%2BA.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DClinical%2520experience%2520of%2520MEK%2520inhibitors%2520in%2520cancer%2520therapy%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2007%26volume%3D1773%26spage%3D1248%26epage%3D1255%26doi%3D10.1016%2Fj.bbamcr.2006.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudley, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saltiel, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fergus, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamme, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LePage, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leopold, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Becelaere, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marston, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, W. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span> <span> </span><span class="NLM_article-title">The discovery of the benzhydroxamate MEK inhibitors CI-1040 and PD 0325901</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6501</span>– <span class="NLM_lpage">6504</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.10.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.bmcl.2008.10.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=18952427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWjsL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=6501-6504&author=S.+D.+Barrettauthor=A.+J.+Bridgesauthor=D.+T.+Dudleyauthor=A.+R.+Saltielauthor=J.+H.+Fergusauthor=C.+M.+Flammeauthor=A.+M.+Delaneyauthor=M.+Kaufmanauthor=S.+LePageauthor=W.+R.+Leopoldauthor=S.+A.+Przybranowskiauthor=J.+Sebolt-Leopoldauthor=K.+Van+Becelaereauthor=A.+M.+Dohertyauthor=R.+M.+Kennedyauthor=D.+Marstonauthor=W.+A.+Howardauthor=Y.+Smithauthor=J.+S.+Warmusauthor=H.+Tecle&title=The+discovery+of+the+benzhydroxamate+MEK+inhibitors+CI-1040+and+PD+0325901&doi=10.1016%2Fj.bmcl.2008.10.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of the benzhydroxamate MEK inhibitors CI-1040 and PD 0325901</span></div><div class="casAuthors">Barrett, Stephen D.; Bridges, Alexander J.; Dudley, David T.; Saltiel, Alan R.; Fergus, James H.; Flamme, Cathlin M.; Delaney, Amy M.; Kaufman, Michael; LePage, Sophie; Leopold, Wilbur R.; Przybranowski, Sally A.; Sebolt-Leopold, Judith; Van Becelaere, Keri; Doherty, Annette M.; Kennedy, Robert M.; Marston, Dan; Howard, W. Allen, Jr.; Smith, Yvonne; Warmus, Joseph S.; Tecle, Haile</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6501-6504</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of benzhydroxamate esters derived from their precursor anthranilic acids have been prepd. and have been identified as potent MEK inhibitors.  CI-1040 (I) was the first MEK inhibitor to demonstrate in vivo activity in preclin. animal models and subsequently became the first MEK inhibitor to enter clin. trial.  CI-1040 suffered however from poor exposure due to its poor soly. and rapid clearance, and as a result, development of the compd. was terminated.  Optimization of the diphenylamine core and modification of the hydroxamate side chain for cell potency, soly., and exposure with oral delivery resulted in the discovery of the clin. candidate PD 0325901 (II).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpam9NxZxX93rVg90H21EOLACvtfcHk0ljsfZXN8o7dqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWjsL%252FI&md5=e64f7ae6a446c504528621e7d14d67ce</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.10.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.10.054%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DS.%2BD.%26aulast%3DBridges%26aufirst%3DA.%2BJ.%26aulast%3DDudley%26aufirst%3DD.%2BT.%26aulast%3DSaltiel%26aufirst%3DA.%2BR.%26aulast%3DFergus%26aufirst%3DJ.%2BH.%26aulast%3DFlamme%26aufirst%3DC.%2BM.%26aulast%3DDelaney%26aufirst%3DA.%2BM.%26aulast%3DKaufman%26aufirst%3DM.%26aulast%3DLePage%26aufirst%3DS.%26aulast%3DLeopold%26aufirst%3DW.%2BR.%26aulast%3DPrzybranowski%26aufirst%3DS.%2BA.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DVan%2BBecelaere%26aufirst%3DK.%26aulast%3DDoherty%26aufirst%3DA.%2BM.%26aulast%3DKennedy%26aufirst%3DR.%2BM.%26aulast%3DMarston%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DW.%2BA.%26aulast%3DSmith%26aufirst%3DY.%26aulast%3DWarmus%26aufirst%3DJ.%2BS.%26aulast%3DTecle%26aufirst%3DH.%26atitle%3DThe%2520discovery%2520of%2520the%2520benzhydroxamate%2520MEK%2520inhibitors%2520CI-1040%2520and%2520PD%25200325901%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D6501%26epage%3D6504%26doi%3D10.1016%2Fj.bmcl.2008.10.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, R.</span></span> <span> </span><span class="NLM_article-title">Targeting the mitogen-activated protein kinase cascade to treat cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">947</span>, <span class="refDoi"> DOI: 10.1038/nrc1503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fnrc1503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=15573115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=937-947&author=J.+S.+Sebolt-Leopoldauthor=R.+Herrera&title=Targeting+the+mitogen-activated+protein+kinase+cascade+to+treat+cancer&doi=10.1038%2Fnrc1503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the mitogen-activated protein kinase cascade to treat cancer</span></div><div class="casAuthors">Sebolt-Leopold, Judith S.; Herrera, Roman</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">937-947</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The RAS-mitogen activated protein kinase (MAPK) signaling pathway has long been viewed as an attractive pathway for anticancer therapies, based on its central role in regulating the growth and survival of cells from a broad spectrum of human tumors.  Small-mol. inhibitors designed to target various steps of this pathway have entered clin. trials.  The safety and effectiveness of these inhibitors as well as the role of MAPK pathway in the therapeutic intervention are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6UoubyiRaoLVg90H21EOLACvtfcHk0ljsfZXN8o7dqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjO&md5=6b6ead5165e040552dfca68d7a6c0477</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrc1503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1503%26sid%3Dliteratum%253Aachs%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%2BS.%26aulast%3DHerrera%26aufirst%3DR.%26atitle%3DTargeting%2520the%2520mitogen-activated%2520protein%2520kinase%2520cascade%2520to%2520treat%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D937%26epage%3D947%26doi%3D10.1038%2Fnrc1503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span> <span> </span><span class="NLM_article-title">Current development status of MEK inhibitors</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1551</span>, <span class="refDoi"> DOI: 10.3390/molecules22101551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.3390%2Fmolecules22101551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Wjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1551&author=Y.+Chengauthor=H.+Tian&title=Current+development+status+of+MEK+inhibitors&doi=10.3390%2Fmolecules22101551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Current development status of MEK inhibitors</span></div><div class="casAuthors">Cheng, Ying; Tian, Hongqi</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1551/1-1551/20</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The current development status of mitogen-activated protein kinase kinase (MEK) inhibitors, including the preclin. data and clin. study progress, has been summarized in this review.  Different MEK inhibitors, possessing specific physicochem. properties and bioactivity characteristics, may provide different options for patients seeking treatment for cancer.  Moreover, the combination of the MEK inhibitors with other therapies-such as chemotherapy, targeted therapy, and immunotherapy-may be a promising approach for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosd6s1Lx6dYbVg90H21EOLACvtfcHk0ljsfZXN8o7dqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Wjsrk%253D&md5=0210d125f8a46468d67adb4f9aaf6cf3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fmolecules22101551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules22101551%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DH.%26atitle%3DCurrent%2520development%2520status%2520of%2520MEK%2520inhibitors%26jtitle%3DMolecules%26date%3D2017%26volume%3D22%26spage%3D1551%26doi%3D10.3390%2Fmolecules22101551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreck, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guajardo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhart, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossman, S. A.</span></span> <span> </span><span class="NLM_article-title">Concurrent BRAF/MEK inhibitors in BRAF V600-mutant high-grade primary brain tumors</span>. <i>J. Natl. Compr. Cancer Network</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">347</span>, <span class="refDoi"> DOI: 10.6004/jnccn.2017.7052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.6004%2Fjnccn.2017.7052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29632053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BC1MjgtlOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=343-347&author=K.+C.+Schreckauthor=A.+Guajardoauthor=D.+D.+M.+Linauthor=C.+G.+Eberhartauthor=S.+A.+Grossman&title=Concurrent+BRAF%2FMEK+inhibitors+in+BRAF+V600-mutant+high-grade+primary+brain+tumors&doi=10.6004%2Fjnccn.2017.7052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Concurrent BRAF/MEK Inhibitors in BRAF V600-Mutant High-Grade Primary Brain Tumors</span></div><div class="casAuthors">Schreck Karisa C; Guajardo Andrew; Lin Doris D M; Eberhart Charles G; Grossman Stuart A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Comprehensive Cancer Network : JNCCN</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">343-347</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BRAF V600 mutations are being identified in patients with primary brain tumors more often as molecular testing becomes widely available.  Targeted treatment with BRAF inhibitors has been attempted in individual cases with some responses, whereas others showed no response or developed resistance.  Preclinical work suggests that gliomas could be more responsive to the concurrent use of BRAF and MEK inhibition for MAP kinase pathway suppression.  This report presents 2 cases of malignant brain tumors with BRAF V600E mutations that were resistant to radiation and temozolomide, and reports on their response to targeted treatment with the BRAF and MEK inhibitors dabrafenib and trametinib.  One patient with an anaplastic pleomorphic xanthoastrocytoma experienced a partial response for 14 months, demonstrated by progressive tumor shrinkage and clinical improvement; however, this was followed by clinical and radiographic progression.  The patient with glioblastoma continued to have stable disease after 16 months of treatment.  These cases are encouraging in a disease that urgently needs new treatments.  Further work is necessary to understand response rates, duration, and survival in primary brain tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTLBwzYQSlXeuLt9HypzAgQfW6udTcc2ea-jCEoJ8Kjbrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjgtlOruw%253D%253D&md5=ba18710a1e0d3a34f47ce7d25a743291</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.6004%2Fjnccn.2017.7052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.6004%252Fjnccn.2017.7052%26sid%3Dliteratum%253Aachs%26aulast%3DSchreck%26aufirst%3DK.%2BC.%26aulast%3DGuajardo%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DD.%2BD.%2BM.%26aulast%3DEberhart%26aufirst%3DC.%2BG.%26aulast%3DGrossman%26aufirst%3DS.%2BA.%26atitle%3DConcurrent%2520BRAF%252FMEK%2520inhibitors%2520in%2520BRAF%2520V600-mutant%2520high-grade%2520primary%2520brain%2520tumors%26jtitle%3DJ.%2520Natl.%2520Compr.%2520Cancer%2520Network%26date%3D2018%26volume%3D16%26spage%3D343%26epage%3D347%26doi%3D10.6004%2Fjnccn.2017.7052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimaldi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Festino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanella, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strudel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascierto, P. A.</span></span> <span> </span><span class="NLM_article-title">MEK inhibitors in the treatment of metastatic melanoma and solid tumors</span>. <i>Am. J. Clin. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1007/s40257-017-0292-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1007%2Fs40257-017-0292-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=28537004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BC1cngsFCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=745-754&author=A.+M.+Grimaldiauthor=E.+Simeoneauthor=L.+Festinoauthor=V.+Vanellaauthor=M.+Strudelauthor=P.+A.+Ascierto&title=MEK+inhibitors+in+the+treatment+of+metastatic+melanoma+and+solid+tumors&doi=10.1007%2Fs40257-017-0292-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">MEK Inhibitors in the Treatment of Metastatic Melanoma and Solid Tumors</span></div><div class="casAuthors">Grimaldi Antonio M; Simeone Ester; Festino Lucia; Vanella Vito; Strudel Martina; Ascierto Paolo A</div><div class="citationInfo"><span class="NLM_cas:title">American journal of clinical dermatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">745-754</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The mitogen-activated protein kinase (MAPK) cascade is an intracellular signaling pathway involved in the regulation of cellular proliferation and the survival of tumor cells.  Several different mutations, involving BRAF or NRAS, exert an oncogenic effect by activating the MAPK pathway, resulting in an increase in cellular proliferation.  These mutations have become targets for new therapeutic strategies in melanoma and other cancers.  Selective MEK inhibitors have the ability to inhibit growth and induce cell death in BRAF- and NRAS-mutant melanoma cell lines.  MEK inhibitor therapy in combination with a BRAF inhibitor is more effective and less toxic than treatment with a BRAF inhibitor alone, and has become the standard of care for patients with BRAF-mutated melanoma.  Trametinib was the first MEK inhibitor approved for the treatment of BRAF-mutated metastatic melanoma not previously treated with BRAF inhibitors, and is also approved in combination with the BRAF inhibitor dabrafenib.  Furthermore, cobimetinib is another MEK inhibitor approved for the treatment of BRAF-mutated metastatic melanoma in combination with a BRAF inhibitor, vemurafenib.  The MEK inhibitor binimetinib in combination with the BRAF inhibitor encorafenib is in clinical development.  The addition of an anti-PD-1/PD-L1 agent, such as pembrolizumab, durvalumab or atezolizumab, to combined BRAF and MEK inhibition has shown considerable promise, with several trials ongoing in metastatic melanoma.  Binimetinib has also shown efficacy in NRAS-mutated melanoma patients.  Future possibilities for MEK inhibitors in advanced melanoma, as well as other solid tumors, include their use in combination with other targeted therapies (e.g. anti-CDK4/6 inhibitors) and/or various immune-modulating antibodies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTRMQVeZbuZD8RAV4D_Ins_fW6udTcc2ebsrp-pw2cG-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cngsFCgtA%253D%253D&md5=23b16eb79f7203a8acd54e8c9249357d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs40257-017-0292-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40257-017-0292-y%26sid%3Dliteratum%253Aachs%26aulast%3DGrimaldi%26aufirst%3DA.%2BM.%26aulast%3DSimeone%26aufirst%3DE.%26aulast%3DFestino%26aufirst%3DL.%26aulast%3DVanella%26aufirst%3DV.%26aulast%3DStrudel%26aufirst%3DM.%26aulast%3DAscierto%26aufirst%3DP.%2BA.%26atitle%3DMEK%2520inhibitors%2520in%2520the%2520treatment%2520of%2520metastatic%2520melanoma%2520and%2520solid%2520tumors%26jtitle%3DAm.%2520J.%2520Clin.%2520Dermatol.%26date%3D2017%26volume%3D18%26spage%3D745%26epage%3D754%26doi%3D10.1007%2Fs40257-017-0292-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heppt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tietze, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berking, C.</span></span> <span> </span><span class="NLM_article-title">Combination therapy of melanoma using kinase inhibitors</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1097/CCO.0000000000000160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1097%2FCCO.0000000000000160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25602684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitl2murw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=134-140&author=M.+V.+Hepptauthor=J.+K.+Tietzeauthor=S.+A.+Grafauthor=C.+Berking&title=Combination+therapy+of+melanoma+using+kinase+inhibitors&doi=10.1097%2FCCO.0000000000000160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Combination therapy of melanoma using kinase inhibitors</span></div><div class="casAuthors">Heppt, Markus V.; Tietze, Julia K.; Graf, Saskia A.; Berking, Carola</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-140</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: Treatment options for metastatic melanoma depend on the clin. course of the disease and the mol. profile such as mutations of the BRAF gene.  In this article, we review the current state of targeted therapy with kinase inhibitors.  Recent findings: Despite major advancements in targeted therapy of metastatic melanoma, most patients relapse and show progressive disease after 5-7 mo with single inhibition of BRAF or MEK.  Acquired resistance is virtually universal and mediated by diverse mitogen-activated protein kinase-dependent or independent mechanisms.  Recent evidence favors concurrent targeting of BRAF and MEK in patients with BRAFV600-mutated melanoma instead of BRAF inhibitor monotherapy.  The combination delays the onset of acquired resistance, resulting in increased progression-free and overall survival.  A growing no. of early trials evaluate the efficacy of inhibitors targeting addnl. pathways such as phospho-inositide 3-kinase/AKT in conjunction with BRAF or MEK.  Even though consistent and mature phase III study results are not yet available for these combinations, the repertoire of targeted therapy in metastatic melanoma is wide and promising.  Summary: The short era of single BRAF inhibition in BRAF-mutated melanoma is soon taken over by dual concurrent inhibition of MEK and BRAF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1rD_f2NR8yLVg90H21EOLACvtfcHk0ljxntD9AeDhqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitl2murw%253D&md5=b5f810e41c451a194daa09bdcbdb2151</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1097%2FCCO.0000000000000160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0000000000000160%26sid%3Dliteratum%253Aachs%26aulast%3DHeppt%26aufirst%3DM.%2BV.%26aulast%3DTietze%26aufirst%3DJ.%2BK.%26aulast%3DGraf%26aufirst%3DS.%2BA.%26aulast%3DBerking%26aufirst%3DC.%26atitle%3DCombination%2520therapy%2520of%2520melanoma%2520using%2520kinase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2015%26volume%3D27%26spage%3D134%26epage%3D140%26doi%3D10.1097%2FCCO.0000000000000160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kefford, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Infante, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fecher, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millward, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McArthur, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMarini, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paterl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K. D.</span></span> <span> </span><span class="NLM_article-title">Phase II study of the MEK1/MEK2 inhibitor trametinib in patients with metastatic BRAF-mutant cutaneous melanoma previously treated with or without a BRAF inhibitor</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">482</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1200/JCO.2012.43.5966</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1200%2FJCO.2012.43.5966" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=23248257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Sktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=482-489&author=K.+B.+Kimauthor=R.+Keffordauthor=A.+C.+Pavlickauthor=J.+R.+Infanteauthor=A.+Ribasauthor=J.+A.+Sosmanauthor=L.+A.+Fecherauthor=A.+Millwardauthor=G.+A.+McArthurauthor=P.+Hwuauthor=R.+Gonzalezauthor=P.+A.+Ottauthor=G.+V.+Longauthor=O.+S.+Gardnerauthor=D.+Ouelletauthor=Y.+Xuauthor=D.+J.+DeMariniauthor=N.+T.+Leauthor=K.+Paterlauthor=K.+D.+Lewis&title=Phase+II+study+of+the+MEK1%2FMEK2+inhibitor+trametinib+in+patients+with+metastatic+BRAF-mutant+cutaneous+melanoma+previously+treated+with+or+without+a+BRAF+inhibitor&doi=10.1200%2FJCO.2012.43.5966"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Phase II study of the MEK1/MEK2 inhibitor trametinib in patients with metastatic BRAF-mutant cutaneous melanoma previously treated with or without a BRAF inhibitor</span></div><div class="casAuthors">Kim, Kevin B.; Kefford, Richard; Pavlick, Anna C.; Infante, Jeffrey R.; Ribas, Antoni; Sosman, Jeffrey A.; Fecher, Leslie A.; Millward, Michael; McArthur, Grant A.; Hwu, Patrick; Gonzalez, Rene; Ott, Patrick A.; Long, Georgina V.; Gardner, Olivia S.; Ouellet, Daniele; Xu, Yanmei; DeMarini, Douglas J.; Le, Ngocdiep T.; Patel, Kiran; Lewis, Karl D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">482-489</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">0732-183X</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose: BRAF mutations promote melanoma cell proliferation and survival primarily through activation of MEK.  The purpose of this study was to det. the response rate (RR) for the selective, allosteric MEK1/MEK2 inhibitor trametinib (GSK1120212), in patients with metastatic BRAF-mutant melanoma.  Patients and Methods: This was an open-label, two-stage, phase II study with two cohorts.  Patients with metastatic BRAF-mutant melanoma previously treated with a BRAF inhibitor (cohort A) or treated with chemotherapy and/or immunotherapy (BRAF-inhibitor naive; cohort B) were enrolled.  Patients: received 2 mg of trametinib orally once daily.  Results: In cohort A (n = 40), there were no confirmed objective responses and 11 patients (28%) with stable disease (SD); the median progression-free survival (PFS) was 1.8 mo.  In cohort B (n = 57), there was one (2%) complete response, 13 (23%) partial responses (PRs), and 29 patients (51%) with SD (confirmed RR, 25%); the median PFS was 4.0 mo.  One patient each with BRAF K601E and BRAF V600R had prolonged PR.  The most frequent treatment-related adverse events for all patients were skin-related toxicity, nausea, peripheral edema, diarrhea, pruritis, and fatigue.  No cutaneous squamous cell carcinoma was obsd.  Conclusion Trametinib was well tolerated.  Significant clin. activity was obsd. in BRAF-inhibitor-naive patients previously treated with chemotherapy and/or immunotherapy.  Minimal clin. activity was obsd. as sequential therapy in patients previously treated with a BRAF inhibitor.  Together, these data suggest that BRAF-inhibitor resistance mechanisms likely confer resistance to MEK-inhibitor monotherapy.  These data support further evaluation of trametinib in BRAF-inhibitor-naive BRAF-mutant melanoma, including rarer forms of BRAF-mutant melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeb_z2i1Ifq7Vg90H21EOLACvtfcHk0ljxntD9AeDhqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Sktbc%253D&md5=dfa4cd44e8a9fc3b66e9c260af829648</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1200%2FJCO.2012.43.5966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2012.43.5966%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKefford%26aufirst%3DR.%26aulast%3DPavlick%26aufirst%3DA.%2BC.%26aulast%3DInfante%26aufirst%3DJ.%2BR.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DSosman%26aufirst%3DJ.%2BA.%26aulast%3DFecher%26aufirst%3DL.%2BA.%26aulast%3DMillward%26aufirst%3DA.%26aulast%3DMcArthur%26aufirst%3DG.%2BA.%26aulast%3DHwu%26aufirst%3DP.%26aulast%3DGonzalez%26aufirst%3DR.%26aulast%3DOtt%26aufirst%3DP.%2BA.%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DGardner%26aufirst%3DO.%2BS.%26aulast%3DOuellet%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeMarini%26aufirst%3DD.%2BJ.%26aulast%3DLe%26aufirst%3DN.%2BT.%26aulast%3DPaterl%26aufirst%3DK.%26aulast%3DLewis%26aufirst%3DK.%2BD.%26atitle%3DPhase%2520II%2520study%2520of%2520the%2520MEK1%252FMEK2%2520inhibitor%2520trametinib%2520in%2520patients%2520with%2520metastatic%2520BRAF-mutant%2520cutaneous%2520melanoma%2520previously%2520treated%2520with%2520or%2520without%2520a%2520BRAF%2520inhibitor%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2013%26volume%3D31%26spage%3D482%26epage%3D489%26doi%3D10.1200%2FJCO.2012.43.5966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dummer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascierto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arance, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liszkay, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadendorf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajsova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutzmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sileni, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutriaux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groot, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loquai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parseval, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickard, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandor, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, K. T.</span></span> <span> </span><span class="NLM_article-title">Overall survival in patients with BRAF-mutant melanoma receiving encorafenib plus binimetinib versus vemurafenib or encorafenib (columbus): a multicentre, open-label, randomised, Phase 3 Trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1315</span>– <span class="NLM_lpage">1327</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(18)30497-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1470-2045%2818%2930497-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=30219628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWgsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1315-1327&author=R.+Dummerauthor=P.+Asciertoauthor=H.+Gogasauthor=A.+Aranceauthor=M.+Mandalaauthor=G.+Liszkayauthor=C.+Garbeauthor=D.+Schadendorfauthor=I.+Krajsovaauthor=R.+Gutzmerauthor=V.+C.+Sileniauthor=C.+Dutriauxauthor=J.+W.+Grootauthor=N.+Yamazakiauthor=C.+Loquaiauthor=L.+Parsevalauthor=M.+D.+Pickardauthor=V.+Sandorauthor=C.+Robertauthor=K.+T.+Flaherty&title=Overall+survival+in+patients+with+BRAF-mutant+melanoma+receiving+encorafenib+plus+binimetinib+versus+vemurafenib+or+encorafenib+%28columbus%29%3A+a+multicentre%2C+open-label%2C+randomised%2C+Phase+3+Trial&doi=10.1016%2FS1470-2045%2818%2930497-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Overall survival in patients with BRAF-mutant melanoma receiving encorafenib plus binimetinib versus vemurafenib or encorafenib (COLUMBUS): a multicentre, open-label, randomised, phase 3 trial</span></div><div class="casAuthors">Dummer, Reinhard; Ascierto, Paolo A.; Gogas, Helen J.; Arance, Ana; Mandala, Mario; Liszkay, Gabriella; Garbe, Claus; Schadendorf, Dirk; Krajsova, Ivana; Gutzmer, Ralf; Chiarion Sileni, Vanna; Dutriaux, Caroline; de Groot, Jan Willem B.; Yamazaki, Naoya; Loquai, Carmen; Moutouh-de Parseval, Laure A.; Pickard, Michael D.; Sandor, Victor; Robert, Caroline; Flaherty, Keith T.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1315-1327</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Encorafenib plus binimetinib and encorafenib alone improved progression-free survival compared with vemurafenib in patients with BRAFV600-mutant melanoma in the COLUMBUS trial.  Here, we report the results of the secondary endpoint of overall survival.  COLUMBUS was a two-part, randomised, open-label, phase 3 study done at 162 hospitals in 28 countries.  Eligible patients were aged at least 18 years with histol. confirmed, locally advanced, unresectable, or metastatic cutaneous melanoma, or unknown primary melanoma, BRAFV600E or BRAFV600K mutation, an Eastern Cooperative Oncol. Group (ECOG) performance status of 0 or 1, and were treatment naive or had progressed on or after first-line immunotherapy.  In part 1 of the study, patients were randomly assigned (1:1:1) by use of interactive response technol. to receive oral encorafenib 450 mg once daily plus oral binimetinib 45 mg twice daily (encorafenib plus binimetinib group), oral encorafenib 300 mg once daily (encorafenib group), or oral vemurafenib 960 mg twice daily (vemurafenib group).  Randomisation was stratified by the American Joint Committee on Cancer stage, ECOG performance status, and BRAF mutation status.  The primary outcome of the trial, progression-free survival with encorafenib plus binimetinib vs. vemurafenib, was reported previously.  Here we present the prespecified interim overall survival anal.  Efficacy analyses were by intent to treat.  Safety was analyzed in patients who received at least one dose of study drug.  Part 2 of the study was initiated at the request of the US Food and Drug Administration to better understand the contribution of binimetinib to the combination therapy by comparing encorafenib 300 mg once daily plus binimetinib 45 mg twice daily with encorafenib 300 mg once daily alone.  Results of part 2 will be published sep.  This trial is ongoing and is registered with ClinicalTrials.gov, no. NCT01909453, and EudraCT, no. 2013-001176-38.  Between Dec 30, 2013, and Apr. 10, 2015, 577 of 1345 screened patients were randomly assigned to receive encorafenib plus binimetinib (n=192), encorafenib (n=194), or vemurafenib (n=191).  Median follow-up for overall survival was 36·8 mo (95% CI 35·9-37·5).  Median overall survival was 33·6 mo (95% CI 24·4-39·2) with encorafenib plus binimetinib and 16·9 mo (14·0-24·5) with vemurafenib (hazard ratio 0·61 [95% CI 0·47-0·79]; two-sided p<0·0001).  The most common grade 3 or 4 adverse events did not change substantially from the first report; those seen in more than 5% of patients treated with encorafenib plus binimetinib were increased γ-glutamyltransferase (18 [9%] of 192 patients), increased blood creatine phosphokinase (14 [7%]), and hypertension (12 [6%]); those seen with encorafenib alone were palmar-plantar erythrodysaesthesia syndrome (26 [14%] of 192 patients), myalgia (19 [10%]), and arthralgia (18 [9%]); and with vemurafenib the most common grade 3 or 4 adverse event was arthralgia (11 [6%] of 186 patients).  One death in the combination treatment group was considered by the investigator to be possibly related to treatment.  The combination of encorafenib plus binimetinib provided clin. meaningful efficacy with good tolerability as shown by improvements in both progression-free survival and overall survival compared with vemurafenib.  These data suggest that the combination of encorafenib plus binimetinib is likely to become an important therapeutic option in patients with BRAFV600-mutant melanoma.Array BioPharma, Novartis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofULG8QpgPeLVg90H21EOLACvtfcHk0ljOjE8cCGOTag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWgsLjJ&md5=e376456c2363459c0a96b32d954260b0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2818%2930497-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252818%252930497-2%26sid%3Dliteratum%253Aachs%26aulast%3DDummer%26aufirst%3DR.%26aulast%3DAscierto%26aufirst%3DP.%26aulast%3DGogas%26aufirst%3DH.%26aulast%3DArance%26aufirst%3DA.%26aulast%3DMandala%26aufirst%3DM.%26aulast%3DLiszkay%26aufirst%3DG.%26aulast%3DGarbe%26aufirst%3DC.%26aulast%3DSchadendorf%26aufirst%3DD.%26aulast%3DKrajsova%26aufirst%3DI.%26aulast%3DGutzmer%26aufirst%3DR.%26aulast%3DSileni%26aufirst%3DV.%2BC.%26aulast%3DDutriaux%26aufirst%3DC.%26aulast%3DGroot%26aufirst%3DJ.%2BW.%26aulast%3DYamazaki%26aufirst%3DN.%26aulast%3DLoquai%26aufirst%3DC.%26aulast%3DParseval%26aufirst%3DL.%26aulast%3DPickard%26aufirst%3DM.%2BD.%26aulast%3DSandor%26aufirst%3DV.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DFlaherty%26aufirst%3DK.%2BT.%26atitle%3DOverall%2520survival%2520in%2520patients%2520with%2520BRAF-mutant%2520melanoma%2520receiving%2520encorafenib%2520plus%2520binimetinib%2520versus%2520vemurafenib%2520or%2520encorafenib%2520%2528columbus%2529%253A%2520a%2520multicentre%252C%2520open-label%252C%2520randomised%252C%2520Phase%25203%2520Trial%26jtitle%3DLancet%2520Oncol.%26date%3D2018%26volume%3D19%26spage%3D1315%26epage%3D1327%26doi%3D10.1016%2FS1470-2045%2818%2930497-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span> </span><span class="NLM_article-title">FDA Approves Encorafenib and Binimetinib
in Combination for Unresectable
or Metastatic Melanoma with BRAF Mutations</span>. <a href="https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf" class="extLink">https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf</a> (accessed Sep 7, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FDA+Approves+Encorafenib+and+Binimetinib%0Ain+Combination+for+Unresectable%0Aor+Metastatic+Melanoma+with+BRAF+Mutations.+https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fresources-information-approved-drugs%2Ffda-approves-encorafenib-and-binimetinib-combination-unresectable-or-metastatic-melanoma-braf+%28accessed+Sep+7%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DFDA%2520Approves%2520Encorafenib%2520and%2520Binimetinib%250Ain%2520Combination%2520for%2520Unresectable%250Aor%2520Metastatic%2520Melanoma%2520with%2520BRAF%2520Mutations%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Planchard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besse, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groen, H. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souquet, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quoix, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baik, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlesi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazieres, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novello, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upalawanna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Amelio, A. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mookerjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span> <span> </span><span class="NLM_article-title">Dabrafenib plus trametinib in patients with previously treated BRAF(V600E)-mutant metastatic non-small cell lung cancer: an open-label, multicentre phase 2 trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">984</span>– <span class="NLM_lpage">993</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(16)30146-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1470-2045%2816%2930146-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=27283860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslyjsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=984-993&author=D.+Planchardauthor=B.+Besseauthor=H.+J.+M.+Groenauthor=P.+J.+Souquetauthor=E.+Quoixauthor=C.+S.+Baikauthor=F.+Barlesiauthor=T.+M.+Kimauthor=J.+Mazieresauthor=S.+Novelloauthor=J.+R.+Rigasauthor=A.+Upalawannaauthor=A.+M.+D%E2%80%99Amelioauthor=P.+Zhangauthor=B.+Mookerjeeauthor=B.+E.+Johnson&title=Dabrafenib+plus+trametinib+in+patients+with+previously+treated+BRAF%28V600E%29-mutant+metastatic+non-small+cell+lung+cancer%3A+an+open-label%2C+multicentre+phase+2+trial&doi=10.1016%2FS1470-2045%2816%2930146-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dabrafenib plus trametinib in patients with previously treated BRAFV600E-mutant metastatic non-small cell lung cancer: an open-label, multicentre phase 2 trial</span></div><div class="casAuthors">Planchard, David; Besse, Benjamin; Groen, Harry J. M.; Souquet, Pierre-Jean; Quoix, Elisabeth; Baik, Christina S.; Barlesi, Fabrice; Kim, Tae Min; Mazieres, Julien; Novello, Silvia; Rigas, James R.; Upalawanna, Allison; D'Amelio, Anthony M. Jr; Zhang, Pingkuan; Mookerjee, Bijoyesh; Johnson, Bruce E.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">984-993</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRAF mutations act as an oncogenic driver via the mitogen-activated protein kinase (MAPK) pathway in non-small cell lung cancer (NSCLC).  BRAF inhibition has shown antitumor activity in patients with BRAFV600E-mutant NSCLC.  Dual MAPK pathway inhibition with BRAF and MEK inhibitors in BRAFV600E-mutant NSCLC might improve efficacy over BRAF inhibitor monotherapy based on observations in BRAFV600-mutant melanoma.  We aimed to assess the antitumor activity and safety of dabrafenib plus trametinib in patients with BRAFV600E-mutant NSCLC.  In this phase 2, multicentre, non-randomized, open-label study, we enrolled adult patients (aged ≥18 years) with pretreated metastatic stage IV BRAFV600E-mutant NSCLC who had documented tumor progression after at least one previous platinum-based chemotherapy and had had no more than three previous systemic anticancer therapies.  Patients with previous BRAF or MEK inhibitor treatment were ineligible.  Patients with brain metastases were allowed to enroll only if the lesions were asymptomatic, untreated (or stable more than 3 wk after local therapy if treated), and measured less than 1 cm.  Enrolled patients received oral dabrafenib (150 mg twice daily) plus oral trametinib (2 mg once daily) in continuous 21-day cycles until disease progression, unacceptable adverse events, withdrawal of consent, or death.  The primary endpoint was investigator-assessed overall response, which was assessed by intention to treat in the protocol-defined population (patients who received second-line or later treatment); safety was also assessed in this population and was assessed at least once every 3 wk, with adverse events, lab. values, and vital signs graded according to the Common Terminol. Criteria for Adverse Events version 4.0.  The study is ongoing but no longer recruiting patients.  This trial is registered with ClinicalTrials.gov, no. NCT01336634.  Between Dec 20, 2013, and Jan 14, 2015, 59 patients from 30 centers in nine countries across North America, Europe, and Asia met eligibility criteria.  Two patients who had previously been untreated due to protocol deviation were excluded; thus, 57 eligible patients were enrolled. 36 patients (63·2% [95% CI 49·3-75·6]) achieved an investigator-assessed overall response.  Serious adverse events were reported in 32 (56%) of 57 patients and included pyrexia in nine (16%), anemia in three (5%), confusional state in two (4%), decreased appetite in two (4%), hemoptysis in two (4%), hypercalcemia in two (4%), nausea in two (4%), and cutaneous squamous cell carcinoma in two (4%).  The most common grade 3-4 adverse events were neutropenia in five patients (9%), hyponatremia in four (7%), and anemia in three (5%).  Four patients died during the study from fatal adverse events judged to be unrelated to treatment (one retroperitoneal hemorrhage, one subarachnoid hemorrhage, one respiratory distress, and one from disease progression that was more severe than typical progression, as assessed by the investigator).  Dabrafenib plus trametinib could represent a new targeted therapy with robust antitumor activity and a manageable safety profile in patients with BRAFV600E-mutant NSCLC.GlaxoSmithKline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-qPV5AvwdnrVg90H21EOLACvtfcHk0ljOjE8cCGOTag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslyjsL8%253D&md5=322443578a05f7f30ed21427413b38ca</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2816%2930146-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252816%252930146-2%26sid%3Dliteratum%253Aachs%26aulast%3DPlanchard%26aufirst%3DD.%26aulast%3DBesse%26aufirst%3DB.%26aulast%3DGroen%26aufirst%3DH.%2BJ.%2BM.%26aulast%3DSouquet%26aufirst%3DP.%2BJ.%26aulast%3DQuoix%26aufirst%3DE.%26aulast%3DBaik%26aufirst%3DC.%2BS.%26aulast%3DBarlesi%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DT.%2BM.%26aulast%3DMazieres%26aufirst%3DJ.%26aulast%3DNovello%26aufirst%3DS.%26aulast%3DRigas%26aufirst%3DJ.%2BR.%26aulast%3DUpalawanna%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Amelio%26aufirst%3DA.%2BM.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DMookerjee%26aufirst%3DB.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26atitle%3DDabrafenib%2520plus%2520trametinib%2520in%2520patients%2520with%2520previously%2520treated%2520BRAF%2528V600E%2529-mutant%2520metastatic%2520non-small%2520cell%2520lung%2520cancer%253A%2520an%2520open-label%252C%2520multicentre%2520phase%25202%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2016%26volume%3D17%26spage%3D984%26epage%3D993%26doi%3D10.1016%2FS1470-2045%2816%2930146-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pupo, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahheydari, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tembe, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saw, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scolyer, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kefford, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizos, H.</span></span> <span> </span><span class="NLM_article-title">Increased MAPK reactivation in early resistance to dabrafenib/trametinib combination therapy of BRAF-mutant metastatic melanoma</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">5694</span> <span class="refDoi"> DOI: 10.1038/ncomms6694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fncomms6694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25452114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVentb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&author=G.+V.+Longauthor=C.+Fungauthor=A.+M.+Menziesauthor=G.+M.+Pupoauthor=M.+S.+Carlinoauthor=J.+Hymanauthor=H.+Shahheydariauthor=V.+Tembeauthor=J.+F.+Thompsonauthor=R.+P.+Sawauthor=J.+Howleauthor=N.+K.+Haywardauthor=P.+Johanssonauthor=R.+A.+Scolyerauthor=R.+F.+Keffordauthor=H.+Rizos&title=Increased+MAPK+reactivation+in+early+resistance+to+dabrafenib%2Ftrametinib+combination+therapy+of+BRAF-mutant+metastatic+melanoma&doi=10.1038%2Fncomms6694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Increased MAPK reactivation in early resistance to dabrafenib/trametinib combination therapy of BRAF-mutant metastatic melanoma</span></div><div class="casAuthors">Long, Georgina V.; Fung, Carina; Menzies, Alexander M.; Pupo, Gulietta M.; Carlino, Matteo S.; Hyman, Jessica; Shahheydari, Hamideh; Tembe, Varsha; Thompson, John F.; Saw, Robyn P.; Howle, Julie; Hayward, Nicholas K.; Johansson, Peter; Scolyer, Richard A.; Kefford, Richard F.; Rizos, Helen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5694</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">One-third of BRAF-mutant metastatic melanoma patients treated with combined BRAF and MEK inhibition progress within 6 mo.  Treatment options for these patients remain limited.  Here we analyze 20 BRAFV600-mutant melanoma metastases derived from 10 patients treated with the combination of dabrafenib and trametinib for resistance mechanisms and genetic correlates of response.  Resistance mechanisms are identified in 9/11 progressing tumors and MAPK reactivation occurred in 9/10 tumors, commonly via BRAF amplification and mutations activating NRAS and MEK2.  Our data confirming that MEK2C125S, but not the synonymous MEK1C121S protein, confers resistance to combination therapy highlight the functional differences between these kinases and the preponderance of MEK2 mutations in combination therapy-resistant melanomas.  Exome sequencing did not identify addnl. progression-specific resistance candidates.  Nevertheless, most melanomas carried addnl. oncogenic mutations at baseline (for example, RAC1 and AKT3) that activate the MAPK and PI3K pathways and are thus predicted to diminish response to MAPK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhfIk5riiY_LVg90H21EOLACvtfcHk0ljfsnIHVu1Ylg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVentb4%253D&md5=64e244a00335113afeabaf6ab4db2974</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fncomms6694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms6694%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DG.%2BV.%26aulast%3DFung%26aufirst%3DC.%26aulast%3DMenzies%26aufirst%3DA.%2BM.%26aulast%3DPupo%26aufirst%3DG.%2BM.%26aulast%3DCarlino%26aufirst%3DM.%2BS.%26aulast%3DHyman%26aufirst%3DJ.%26aulast%3DShahheydari%26aufirst%3DH.%26aulast%3DTembe%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DJ.%2BF.%26aulast%3DSaw%26aufirst%3DR.%2BP.%26aulast%3DHowle%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DN.%2BK.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DScolyer%26aufirst%3DR.%2BA.%26aulast%3DKefford%26aufirst%3DR.%2BF.%26aulast%3DRizos%26aufirst%3DH.%26atitle%3DIncreased%2520MAPK%2520reactivation%2520in%2520early%2520resistance%2520to%2520dabrafenib%252Ftrametinib%2520combination%2520therapy%2520of%2520BRAF-mutant%2520metastatic%2520melanoma%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26doi%3D10.1038%2Fncomms6694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriceau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hugo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koya, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samatar, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanlou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchalski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seifert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlman, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, R. S.</span></span> <span> </span><span class="NLM_article-title">Tunable-combinatorial mechanisms of acquired resistance limit the efficacy of BRAF/MEK cotargeting but result in melanoma drug addiction</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2014.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.ccell.2014.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25600339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=240-256&author=G.+Moriceauauthor=W.+Hugoauthor=A.+Hongauthor=H.+Shiauthor=X.+Kongauthor=C.+C.+Yuauthor=R.+C.+Koyaauthor=A.+A.+Samatarauthor=N.+Khanlouauthor=J.+Braunauthor=K.+Ruchalskiauthor=H.+Seifertauthor=J.+Larkinauthor=K.+B.+Dahlmanauthor=D.+B.+Johnsonauthor=A.+Algaziauthor=J.+A.+Sosmanauthor=A.+Ribasauthor=R.+S.+Lo&title=Tunable-combinatorial+mechanisms+of+acquired+resistance+limit+the+efficacy+of+BRAF%2FMEK+cotargeting+but+result+in+melanoma+drug+addiction&doi=10.1016%2Fj.ccell.2014.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Tunable-Combinatorial Mechanisms of Acquired Resistance Limit the Efficacy of BRAF/MEK Cotargeting but Result in Melanoma Drug Addiction</span></div><div class="casAuthors">Moriceau, Gatien; Hugo, Willy; Hong, Aayoung; Shi, Hubing; Kong, Xiangju; Yu, Clarissa C.; Koya, Richard C.; Samatar, Ahmed A.; Khanlou, Negar; Braun, Jonathan; Ruchalski, Kathleen; Seifert, Heike; Larkin, James; Dahlman, Kimberly B.; Johnson, Douglas B.; Algazi, Alain; Sosman, Jeffrey A.; Ribas, Antoni; Lo, Roger S.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">240-256</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Combined BRAF- and MEK-targeted therapy improves upon BRAF inhibitor (BRAFi) therapy but is still beset by acquired resistance.  We show that melanomas acquire resistance to combined BRAF and MEK inhibition by augmenting or combining mechanisms of single-agent BRAFi resistance.  These double-drug resistance-assocd. genetic configurations significantly altered mol. interactions underlying MAPK pathway reactivation. V600EBRAF, expressed at supraphysiol. levels because of V600EBRAF ultra-amplification, dimerized with and activated CRAF.  In addn., MEK mutants enhanced interaction with overexpressed V600EBRAF via a regulatory interface at R662 of V600EBRAF.  Importantly, melanoma cell lines selected for resistance to BRAFi+MEKi, but not those to BRAFi alone, displayed robust drug addiction, providing a potentially exploitable therapeutic opportunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWdJyJ2wlmALVg90H21EOLACvtfcHk0ljfsnIHVu1Ylg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKgtrg%253D&md5=42788605b6c22cf70ff49304bf65ca19</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2014.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2014.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DMoriceau%26aufirst%3DG.%26aulast%3DHugo%26aufirst%3DW.%26aulast%3DHong%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DC.%2BC.%26aulast%3DKoya%26aufirst%3DR.%2BC.%26aulast%3DSamatar%26aufirst%3DA.%2BA.%26aulast%3DKhanlou%26aufirst%3DN.%26aulast%3DBraun%26aufirst%3DJ.%26aulast%3DRuchalski%26aufirst%3DK.%26aulast%3DSeifert%26aufirst%3DH.%26aulast%3DLarkin%26aufirst%3DJ.%26aulast%3DDahlman%26aufirst%3DK.%2BB.%26aulast%3DJohnson%26aufirst%3DD.%2BB.%26aulast%3DAlgazi%26aufirst%3DA.%26aulast%3DSosman%26aufirst%3DJ.%2BA.%26aulast%3DRibas%26aufirst%3DA.%26aulast%3DLo%26aufirst%3DR.%2BS.%26atitle%3DTunable-combinatorial%2520mechanisms%2520of%2520acquired%2520resistance%2520limit%2520the%2520efficacy%2520of%2520BRAF%252FMEK%2520cotargeting%2520but%2520result%2520in%2520melanoma%2520drug%2520addiction%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26spage%3D240%26epage%3D256%26doi%3D10.1016%2Fj.ccell.2014.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: elements of PROTAC design</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.02.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.cbpa.2019.02.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31004963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXns1yrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2019&pages=111-119&author=S.+L.+Paivaauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+elements+of+PROTAC+design&doi=10.1016%2Fj.cbpa.2019.02.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: elements of PROTAC design</span></div><div class="casAuthors">Paiva, Stacey-Lynn; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn. using Proteolysis Targeting Chimeras (PROTACs) has emerged as a novel therapeutic modality in drug discovery.  PROTACs mediate the degrdn. of select proteins of interest (POIs) by hijacking the activity of E3 ubiquitin ligases for POI ubiquitination and subsequent degrdn. by the 26S proteasome.  This hijacking mechanism has been used to degrade various types of disease-relevant POIs.  In this review, we aim to highlight the recent advances in targeted protein degrdn. and describe the challenges that need to be addressed in order to efficiently develop potent PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol6Nk0VkUdzLVg90H21EOLACvtfcHk0ljfsnIHVu1Ylg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXns1yrt7s%253D&md5=5783311db47da378180c14d67526a398</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.02.022%26sid%3Dliteratum%253Aachs%26aulast%3DPaiva%26aufirst%3DS.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520elements%2520of%2520PROTAC%2520design%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D50%26spage%3D111%26epage%3D119%26doi%3D10.1016%2Fj.cbpa.2019.02.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">64</span> <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2eYvZ3r74sQhNLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span> <span> </span><span class="NLM_article-title">Development of targeted protein degradation therapeutics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">944</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0362-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41589-019-0362-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31527835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=937-944&author=P.+P.+Chamberlainauthor=L.+G.+Hamann&title=Development+of+targeted+protein+degradation+therapeutics&doi=10.1038%2Fs41589-019-0362-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Development of targeted protein degradation therapeutics</span></div><div class="casAuthors">Chamberlain, Philip P.; Hamann, Lawrence G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">937-944</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted protein degrdn. as a therapeutic modality has seen dramatic progress and massive investment in recent years because of the convergence of two key scientific breakthroughs: optimization of first-generation peptidic proteolysis-targeted chimeras (PROTACs) into more drug-like mols. able to support in vivo proof of concept and the discovery that clin. mols. function as degraders by binding and repurposing the proteins cereblon and DCAF15.  This provided clin. validation for the general approach through the cereblon modulator class of drugs and provided highly drug-like and ligand-efficient E3 ligase binders upon which to tether target-binding moieties.  Increasingly rational and systematic approaches including biophys. and structural studies on ternary complexes are being leveraged as the field advances.  In this Perspective we summarize the discoveries that have laid the foundation for future degrdn. therapeutics, focusing on those classes of small mols. that redirect E3 ubiquitin ligases to non-native substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfcIEwVR-Xq7Vg90H21EOLACvtfcHk0ljNkqm-VhQLww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygur%252FF&md5=468e3fd87ba2b21607dac8b495e5ecf6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0362-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0362-y%26sid%3Dliteratum%253Aachs%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26atitle%3DDevelopment%2520of%2520targeted%2520protein%2520degradation%2520therapeutics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D937%26epage%3D944%26doi%3D10.1038%2Fs41589-019-0362-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0ljNkqm-VhQLww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bond, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalawansha, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of oncogenic KRAS(G12C) by VHL-recruiting PROTACs</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1367</span>– <span class="NLM_lpage">1375</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.0c00411</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c00411" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlans7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=1367-1375&author=M.+J.+Bondauthor=L.+Chuauthor=D.+A.+Nalawanshaauthor=K.+Liauthor=C.+M.+Crews&title=Targeted+degradation+of+oncogenic+KRAS%28G12C%29+by+VHL-recruiting+PROTACs&doi=10.1021%2Facscentsci.0c00411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of Oncogenic KRASG12C by VHL-Recruiting PROTACs</span></div><div class="casAuthors">Bond, Michael J.; Chu, Ling; Nalawansha, Dhanusha A.; Li, Ke; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1367-1375</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KRAS is mutated in ~ 20% of human cancers and is one of the most sought-after targets for pharmacol. modulation, despite having historically been considered "undruggable." The discovery of potent covalent inhibitors of the KRASG12C mutant in recent years has sparked a new wave of interest in small mols. targeting KRAS.  While these inhibitors have shown promise in the clinic, we wanted to explore PROTAC-mediated degrdn. as a complementary strategy to modulate mutant KRAS.  Herein, we report the development of LC-2(I), the first PROTAC capable of degrading endogenous KRASG12C.  LC-2 covalently binds KRASG12C with a MRTX849 warhead and recruits the E3 ligase VHL, inducing rapid and sustained KRASG12C degrdn. leading to suppression of MAPK signaling in both homozygous and heterozygous KRASG12C cell lines.  LC-2 demonstrates that PROTAC-mediated degrdn. is a viable option for attenuating oncogenic KRAS levels and downstream signaling in cancer cells.  KRAS is one of the most sought-after targets for cancer therapy.  Herein, we report the development of LC-2, the first PROTAC capable of inducing endogenous KRASG12C degrdn. by recruiting VHL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra8EvL_8O3_rVg90H21EOLACvtfcHk0lgYoVkicNr5zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlans7zM&md5=09dc3d55acd5ecd3e8e0e18740830263</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c00411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c00411%26sid%3Dliteratum%253Aachs%26aulast%3DBond%26aufirst%3DM.%2BJ.%26aulast%3DChu%26aufirst%3DL.%26aulast%3DNalawansha%26aufirst%3DD.%2BA.%26aulast%3DLi%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520degradation%2520of%2520oncogenic%2520KRAS%2528G12C%2529%2520by%2520VHL-recruiting%2520PROTACs%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2020%26volume%3D6%26spage%3D1367%26epage%3D1375%26doi%3D10.1021%2Facscentsci.0c00411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plewe, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective small molecule degraders of BRAF-V600E</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4069</span>– <span class="NLM_lpage">4080</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvFCjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4069-4080&author=X.+R.+Hanauthor=L.+Chenauthor=Y.+Weiauthor=W.+Yuauthor=Y.+Chenauthor=C.+Zhangauthor=B.+Jiaoauthor=T.+Shiauthor=L.+Sunauthor=C.+Zhangauthor=Y.+Xuauthor=M.+R.+Leeauthor=Y.+Luoauthor=M.+B.+Pleweauthor=J.+Wang&title=Discovery+of+selective+small+molecule+degraders+of+BRAF-V600E&doi=10.1021%2Facs.jmedchem.9b02083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Selective Small Molecule Degraders of BRAF-V600E</span></div><div class="casAuthors">Han, Xiao-Ran; Chen, Liqun; Wei, Yuanqi; Yu, Weihua; Chen, Yanke; Zhang, Chunyan; Jiao, Bingyang; Shi, Tingting; Sun, Lei; Zhang, Chao; Xu, Yang; Lee, Matthew R.; Luo, Ying; Plewe, Michael B.; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4069-4080</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRAF is among the most frequently mutated oncogenes in human cancers.  Multiple small mol. BRAF kinase inhibitors have been approved for treating melanoma carrying BRAF-V600 mutations.  However, the benefits of BRAF kinase inhibitors are generally short-lived.  Small mol.-mediated targeted protein degrdn. has recently emerged as a novel pharmaceutical strategy to remove disease proteins through hijacking the cellular ubiquitin proteasome system (UPS).  In this study, we developed thalidomide-based heterobifunctional compds. that induced selective degrdn. of BRAF-V600E, but not the wild-type BRAF.  Downregulation of BRAF-V600E suppressed the MEK/ERK kinase cascade in melanoma cells and impaired cell growth in culture.  Abolishing the interaction between degraders and cereblon or blocking the UPS significantly impaired the activities of these degraders, validating a mechanistic role of UPS in mediating targeted degrdn. of BRAF-V600E.  These findings highlight a new approach to modulate the functions of oncogenic BRAF mutants and provide a framework to treat BRAF-dependent human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCRmOwotGfh7Vg90H21EOLACvtfcHk0lgYoVkicNr5zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvFCjtbg%253D&md5=db67b3ec42b8a3eb9641c278c8cd17b6</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02083%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DJiao%26aufirst%3DB.%26aulast%3DShi%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DPlewe%26aufirst%3DM.%2BB.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520selective%2520small%2520molecule%2520degraders%2520of%2520BRAF-V600E%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4069%26epage%3D4080%26doi%3D10.1021%2Facs.jmedchem.9b02083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Posternak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maisonneuve, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlicky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goullet
de Rugy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prakesch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaygorodsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colwill, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccarelli, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batey, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taipale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurinov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uehling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrana, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span> <span> </span><span class="NLM_article-title">Functional characterization of a PROTAC directed against BRAF mutant V600E</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1170</span>– <span class="NLM_lpage">1178</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0609-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41589-020-0609-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=32778845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1OisLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1170-1178&author=G.+Posternakauthor=X.+Tangauthor=P.+Maisonneuveauthor=T.+Jinauthor=H.+Lavoieauthor=S.+Daouauthor=S.+Orlickyauthor=T.+Goullet%0Ade+Rugyauthor=L.+Caldwellauthor=K.+Chanauthor=A.+Amanauthor=M.+Prakeschauthor=G.+Podaauthor=P.+Maderauthor=C.+Wongauthor=S.+Maierauthor=J.+Kitaygorodskyauthor=B.+Larsenauthor=K.+Colwillauthor=Z.+Yinauthor=D.+F.+Ceccarelliauthor=R.+A.+Bateyauthor=M.+Taipaleauthor=I.+Kurinovauthor=D.+Uehlingauthor=J.+Wranaauthor=D.+Durocherauthor=A.+C.+Gingrasauthor=R.+Al-Awarauthor=M.+Therrienauthor=F.+Sicheri&title=Functional+characterization+of+a+PROTAC+directed+against+BRAF+mutant+V600E&doi=10.1038%2Fs41589-020-0609-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of a PROTAC directed against BRAF mutant V600E</span></div><div class="casAuthors">Posternak, Ganna; Tang, Xiaojing; Maisonneuve, Pierre; Jin, Ting; Lavoie, Hugo; Daou, Salima; Orlicky, Stephen; Goullet de Rugy, Theo; Caldwell, Lauren; Chan, Kin; Aman, Ahmed; Prakesch, Michael; Poda, Gennady; Mader, Pavel; Wong, Cassandra; Maier, Stefan; Kitaygorodsky, Julia; Larsen, Brett; Colwill, Karen; Yin, Zhe; Ceccarelli, Derek F.; Batey, Robert A.; Taipale, Mikko; Kurinov, Igor; Uehling, David; Wrana, Jeff; Durocher, Daniel; Gingras, Anne-Claude; Al-Awar, Rima; Therrien, Marc; Sicheri, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1170-1178</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The RAF family kinases function in the RAS-ERK pathway to transmit signals from activated RAS to the downstream kinases MEK and ERK.  This pathway regulates cell proliferation, differentiation and survival, enabling mutations in RAS and RAF to act as potent drivers of human cancers.  Drugs targeting the prevalent oncogenic mutant BRAF(V600E) have shown great efficacy in the clinic, but long-term effectiveness is limited by resistance mechanisms that often exploit the dimerization-dependent process by which RAF kinases are activated.  Here, we investigated a proteolysis-targeting chimera (PROTAC) approach to BRAF inhibition.  The most effective PROTAC, termed P4B, displayed superior specificity and inhibitory properties relative to non-PROTAC controls in BRAF(V600E) cell lines.  In addn., P4B displayed utility in cell lines harboring alternative BRAF mutations that impart resistance to conventional BRAF inhibitors.  This work provides a proof of concept for a substitute to conventional chem. inhibition to therapeutically constrain oncogenic BRAF. [graphic not available: see fulltext].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzRU_Yla3GkLVg90H21EOLACvtfcHk0lgYoVkicNr5zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1OisLnI&md5=18cb24da6d1fd343efe2d7f2f6f67d99</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0609-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0609-7%26sid%3Dliteratum%253Aachs%26aulast%3DPosternak%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DMaisonneuve%26aufirst%3DP.%26aulast%3DJin%26aufirst%3DT.%26aulast%3DLavoie%26aufirst%3DH.%26aulast%3DDaou%26aufirst%3DS.%26aulast%3DOrlicky%26aufirst%3DS.%26aulast%3DGoullet%2Bde%2BRugy%26aufirst%3DT.%26aulast%3DCaldwell%26aufirst%3DL.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DPrakesch%26aufirst%3DM.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DMader%26aufirst%3DP.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DMaier%26aufirst%3DS.%26aulast%3DKitaygorodsky%26aufirst%3DJ.%26aulast%3DLarsen%26aufirst%3DB.%26aulast%3DColwill%26aufirst%3DK.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DCeccarelli%26aufirst%3DD.%2BF.%26aulast%3DBatey%26aufirst%3DR.%2BA.%26aulast%3DTaipale%26aufirst%3DM.%26aulast%3DKurinov%26aufirst%3DI.%26aulast%3DUehling%26aufirst%3DD.%26aulast%3DWrana%26aufirst%3DJ.%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DTherrien%26aufirst%3DM.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DFunctional%2520characterization%2520of%2520a%2520PROTAC%2520directed%2520against%2520BRAF%2520mutant%2520V600E%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D1170%26epage%3D1178%26doi%3D10.1038%2Fs41589-020-0609-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alabi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Mutant-selective degradation by BRAF-targeting PROTACs</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1101/2020.08.10.245159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1101%2F2020.08.10.245159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=33173865" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=S.+Alabiauthor=S.+Jaime-Figueroaauthor=Z.+Yaoauthor=Y.+Gaoauthor=J.+Hinesauthor=K.+T.+Samarasingheauthor=L.+Vogtauthor=N.+Rosenauthor=C.+M.+Crews&title=Mutant-selective+degradation+by+BRAF-targeting+PROTACs&doi=10.1101%2F2020.08.10.245159"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1101%2F2020.08.10.245159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F2020.08.10.245159%26sid%3Dliteratum%253Aachs%26aulast%3DAlabi%26aufirst%3DS.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%26aulast%3DVogt%26aufirst%3DL.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DMutant-selective%2520degradation%2520by%2520BRAF-targeting%2520PROTACs%26jtitle%3DbioRxiv%26date%3D2020%26doi%3D10.1101%2F2020.08.10.245159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class mitogen-activated protein kinase kinase 1/2 degrader</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10911</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01528</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01528" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10897-10911&author=J.+Weiauthor=J.+Huauthor=L.+Wangauthor=L.+Xieauthor=M.+S.+Jinauthor=X.+Chenauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+a+first-in-class+mitogen-activated+protein+kinase+kinase+1%2F2+degrader&doi=10.1021%2Facs.jmedchem.9b01528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></div><div class="casAuthors">Wei, Jieli; Hu, Jianping; Wang, Li; Xie, Ling; Jin, Margaret S.; Chen, Xian; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10897-10911</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the "gatekeepers" of the ERK signaling output with redundant roles in controlling ERK activity.  Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs.  However, acquired resistance to MEK1/2 inhibitors has been obsd. in patients and new therapeutic strategies are needed to overcome the resistance.  Here, the authors report a first-in-class degrader of MEK1/2, MS432 (I), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells.  It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its neg. control MS432N and its effect was phenocopied by MEK1/2 knockdown.  Compd. I was highly selective for MEK1/2 in global proteomic profiling studies.  It was also bioavailable in mice and can be used for in vivo efficacy studies.  The authors provide two well-characterized chem. tools to the biomedical community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e5sg33oDYrVg90H21EOLACvtfcHk0lhKS-YR3nDADA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK&md5=2c293b6b87990d6b814ec1dc88034f19</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520mitogen-activated%2520protein%2520kinase%2520kinase%25201%252F2%2520degrader%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10897%26epage%3D10911%26doi%3D10.1021%2Facs.jmedchem.9b01528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vollmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunoosamy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and biological evaluation of MEK PROTACs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00810</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00810" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlWkt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=157-162&author=S.+Vollmerauthor=D.+Cunoosamyauthor=H.+Lvauthor=H.+Fengauthor=X.+Liauthor=Z.+Nanauthor=W.+Yangauthor=M.+W.+D.+Perry&title=Design%2C+synthesis%2C+and+biological+evaluation+of+MEK+PROTACs&doi=10.1021%2Facs.jmedchem.9b00810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of MEK PROTACs</span></div><div class="casAuthors">Vollmer, Stefan; Cunoosamy, Danen; Lv, Huafei; Feng, Huanxi; Li, Xia; Nan, Ziyang; Yang, Wenzhen; Perry, Matthew W. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">157-162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PROteolysis TArgeting Chimeras (PROTACs) targeting the degrdn. of MEK have been designed based on allosteric MEK inhibitors.  Inhibition of the phosphorylation of ERK1/2 was less effective with the PROTACs than a small-mol. inhibitor; the best PROTACs, however, were more effective in inhibiting proliferation of A375 cells than an inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof7TjD-lFjAbVg90H21EOLACvtfcHk0lhKS-YR3nDADA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlWkt77E&md5=c83d63e23d436e0ef1a3c98e994f1073</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00810%26sid%3Dliteratum%253Aachs%26aulast%3DVollmer%26aufirst%3DS.%26aulast%3DCunoosamy%26aufirst%3DD.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DNan%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520MEK%2520PROTACs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D157%26epage%3D162%26doi%3D10.1021%2Facs.jmedchem.9b00810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebraud, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Protein degradation by in-cell self-assembly of proteolysis targeting chimeras</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">934</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.6b00280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.6b00280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=927-934&author=H.+Lebraudauthor=D.+J.+Wrightauthor=C.+N.+Johnsonauthor=T.+D.+Heightman&title=Protein+degradation+by+in-cell+self-assembly+of+proteolysis+targeting+chimeras&doi=10.1021%2Facscentsci.6b00280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Degradation by In-Cell Self-Assembly of Proteolysis Targeting Chimeras</span></div><div class="casAuthors">Lebraud, Honorine; Wright, David J.; Johnson, Christopher N.; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">927-934</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective degrdn. of proteins by proteolysis targeting chimeras (PROTACs) offers a promising potential alternative to protein inhibition for therapeutic intervention.  Current PROTAC mols. incorporate a ligand for the target protein, a linker, and an E3 ubiquitin ligase recruiting group, which bring together target protein and ubiquitinating machinery.  Such hetero-bifunctional mols. require significant linker optimization and possess high mol. wt., which can limit cellular permeation, soly., and other drug-like properties.  We show here that the hetero-bifunctional mol. can be formed intracellularly by bio-orthogonal click combination of two smaller precursors.  We designed a tetrazine tagged thalidomide deriv. which reacts rapidly with a trans-cyclo-octene tagged ligand of the target protein in cells to form a cereblon E3 ligase recruiting PROTAC mol.  The in-cell click-formed proteolysis targeting chimeras (CLIPTACs) were successfully used to degrade two key oncol. targets, BRD4 and ERK1/2.  ERK1/2 degrdn. was achieved using a CLIPTAC based on a covalent inhibitor.  We expect this approach to be readily extendable to other inhibitor-protein systems because the tagged E3 ligase recruiter is capable of undergoing the click reaction with a suitably tagged ligand of any protein of interest to elicit its degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRnQIoT7hbt7Vg90H21EOLACvtfcHk0ljeUwqGqhLfbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOltbbJ&md5=4cc9b4aaba6ee03cd2d4ea97a72728a5</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.6b00280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.6b00280%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DWright%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DProtein%2520degradation%2520by%2520in-cell%2520self-assembly%2520of%2520proteolysis%2520targeting%2520chimeras%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2016%26volume%3D2%26spage%3D927%26epage%3D934%26doi%3D10.1021%2Facscentsci.6b00280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jo, S. A.</span></span> <span> </span><span class="NLM_article-title">Mitogen-activated protein kinase kinase 1 (MEK1) stabilizes MyoD through direct phosphorylation at tyrosine 156 during myogenic differentiation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">18903</span>– <span class="NLM_lpage">18913</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.225128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1074%2Fjbc.M111.225128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=21454680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlGltbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=18903-18913&author=C.+Joauthor=S.+J.+Choauthor=S.+A.+Jo&title=Mitogen-activated+protein+kinase+kinase+1+%28MEK1%29+stabilizes+MyoD+through+direct+phosphorylation+at+tyrosine+156+during+myogenic+differentiation&doi=10.1074%2Fjbc.M111.225128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Mitogen-activated protein kinase kinase 1 (MEK1) stabilizes MyoD through direct phosphorylation at tyrosine 156 during myogenic differentiation</span></div><div class="casAuthors">Jo, Chulman; Cho, Sun-Jung; Jo, Sangmee Ahn</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">18903-18913</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Previously, the authors reported that MEK1 kinase activated in the mid-stage of skeletal muscle differentiation promotes myogenic differentiation.  To elucidate the mol. mechanism, the authors investigated the activity of MEK1 kinase for MyoD.  Activated MEK1 assocd. with MyoD in the nucleus of differentiating myoblasts.  An in vitro kinase assay using active MEK1, a 32P-labeled protein band corresponding to GST-MyoD was obsd. but not to mutant GST-MyoD-Y156F.  Tyr-156 phosphorylation in endogenous MyoD was detected with a specific anti-pMyoD-Y156 antibody; however, this response was blocked by PD 184352, a MEK-specific inhibitor.  These results indicated that activated MEK1 phosphorylates the MyoD Tyr-156 residue directly.  Interestingly, the protein level of mutant MyoD-Y156F decreased compared with that of wild-type protein, but was recovered in the presence of lactacystin, a proteasome inhibitor.  The protein level of MyoD-Y156E, which mimics phosphorylation at Tyr-156, was above that of wild-type protein, indicating that the phosphorylation protected MyoD from ubiquitin-proteasome-mediated degrdn.  In addn., the low protein level of MyoD-Y156F was recovered over that of the wild-type protein by an addnl. mutation at Leu-164, a crit. binding residue of MAFbx/AT-1, a Skp, Cullin, F-box (SCF) E3-ubiquitin ligase.  The amt. of MyoD co-pptd. with MAFbx/AT-1 also was reduced in the presence of active MEK1.  Thus, these results suggested that the phosphorylation probably interrupted the binding of MAFbx/AT-1 to MyoD and thereby increased its stability.  Collectively, these results suggest that MEK1 activated in differentiating myoblasts stimulates muscle differentiation by phosphorylating MyoD Tyr-156, which results in MyoD stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC9-gmuSStOrVg90H21EOLACvtfcHk0ljeUwqGqhLfbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlGltbk%253D&md5=ce2a130978a3f32816846795071bb234</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.225128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.225128%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DC.%26aulast%3DCho%26aufirst%3DS.%2BJ.%26aulast%3DJo%26aufirst%3DS.%2BA.%26atitle%3DMitogen-activated%2520protein%2520kinase%2520kinase%25201%2520%2528MEK1%2529%2520stabilizes%2520MyoD%2520through%2520direct%2520phosphorylation%2520at%2520tyrosine%2520156%2520during%2520myogenic%2520differentiation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D18903%26epage%3D18913%26doi%3D10.1074%2Fjbc.M111.225128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampson, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, C.</span></span> <span> </span><span class="NLM_article-title">MEK guards proteome stability and inhibits tumor-suppressive amyloidogenesis via HSF1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.01.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.cell.2015.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25679764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVSguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2015&pages=729-744&author=Z.+Tangauthor=S.+Daiauthor=Y.+Heauthor=R.+A.+Dotyauthor=L.+D.+Shultzauthor=S.+B.+Sampsonauthor=C.+Dai&title=MEK+guards+proteome+stability+and+inhibits+tumor-suppressive+amyloidogenesis+via+HSF1&doi=10.1016%2Fj.cell.2015.01.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">MEK Guards Proteome Stability and Inhibits Tumor-Suppressive Amyloidogenesis via HSF1</span></div><div class="casAuthors">Tang, Zijian; Dai, Siyuan; He, Yishu; Doty, Rosalinda A.; Shultz, Leonard D.; Sampson, Stephen Byers; Dai, Chengkai</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">729-744</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Signaling through RAS/MAP kinase pathway is central to biol.  ERK has long been perceived as the only substrate for MEK.  Here, we report that HSF1, the master regulator of the proteotoxic stress response, is a new MEK substrate.  Beyond mediating cell-environment interactions, the MEK-HSF1 regulation impacts malignancy.  In tumor cells, MEK blockade inactivates HSF1 and thereby provokes proteomic chaos, presented as protein destabilization, aggregation, and, strikingly, amyloidogenesis.  Unlike their non-transformed counterparts, tumor cells are particularly susceptible to proteomic perturbation and amyloid induction.  Amyloidogenesis is tumor suppressive, reducing in vivo melanoma growth and contributing to the potent anti-neoplastic effects of proteotoxic stressors.  Our findings unveil a key biol. function of the oncogenic RAS-MEK signaling in guarding proteostasis and suppressing amyloidogenesis.  Thus, proteomic instability is an intrinsic feature of malignant state, and disrupting the fragile tumor proteostasis to promote amyloidogenesis may be a feasible therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUInK2yxXl4rVg90H21EOLACvtfcHk0ljeUwqGqhLfbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVSguro%253D&md5=77038af3c279cec71a2163066c753e55</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DDoty%26aufirst%3DR.%2BA.%26aulast%3DShultz%26aufirst%3DL.%2BD.%26aulast%3DSampson%26aufirst%3DS.%2BB.%26aulast%3DDai%26aufirst%3DC.%26atitle%3DMEK%2520guards%2520proteome%2520stability%2520and%2520inhibits%2520tumor-suppressive%2520amyloidogenesis%2520via%2520HSF1%26jtitle%3DCell%26date%3D2015%26volume%3D160%26spage%3D729%26epage%3D744%26doi%3D10.1016%2Fj.cell.2015.01.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallay, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourquard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claeysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crepieux, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poupon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermoere, F.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of beta-arrestin 2 at Thr383 by MEK underlies beta-arrestin-dependent activation of Erk1/2 by GPCRs</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e23777</span> <span class="refDoi"> DOI: 10.7554/eLife.23777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.7554%2FeLife.23777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=28169830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWktbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=E.+Cassierauthor=N.+Gallayauthor=T.+Bourquardauthor=S.+Claeysenauthor=J.+Bockaertauthor=P.+Crepieuxauthor=A.+Pouponauthor=E.+Reiterauthor=P.+Marinauthor=F.+Vandermoere&title=Phosphorylation+of+beta-arrestin+2+at+Thr383+by+MEK+underlies+beta-arrestin-dependent+activation+of+Erk1%2F2+by+GPCRs&doi=10.7554%2FeLife.23777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of β-arrestin2 at Thr383 by MEK underlies β-arrestin-dependent activation of Erk1/2 by GPCRs</span></div><div class="casAuthors">Cassier, Elisabeth; Gallay, Nathalie; Bourquard, Thomas; Claeysen, Sylvie; Bockaert, Joel; Crepieux, Pascale; Poupon, Anne; Reiter, Eric; Marin, Philippe; Vandermoere, Franck</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e23777/1-e23777/21</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">In addn. to their role in desensitization and internalization of G protein-coupled receptors (GPCRs), β-arrestins are essential scaffolds linking GPCRs to Erk1/2 signaling.  However, their role in GPCR-operated Erk1/2 activation differs between GPCRs and the underlying mechanism remains poorly characterized.  Here, we show that activation of serotonin 5-HT2C receptors, which engage Erk1/2 pathway via a β-arrestin-dependent mechanism, promotes MEK-dependent β-arrestin2 phosphorylation at Thr383, a necessary step for Erk recruitment to the receptor/β-arrestin complex and Erk activation.  Likewise, Thr383 phosphorylation is involved in β arrestin-dependent Erk1/2 stimulation elicited by other GPCRs such as β2-adrenergic, FSH and CXCR4 receptors, but does not affect the β-arrestin-independent Erk1/2 activation by 5-HT4 receptor.  Collectively, these data show that β-arrestin2 phosphorylation at Thr383 underlies β-arrestin-dependent Erk1/2 activation by GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoh4tSQrz0-7Vg90H21EOLACvtfcHk0ljeUwqGqhLfbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWktbvM&md5=61522d61e8ac7e61f73835f89e0af4b9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.7554%2FeLife.23777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.23777%26sid%3Dliteratum%253Aachs%26aulast%3DCassier%26aufirst%3DE.%26aulast%3DGallay%26aufirst%3DN.%26aulast%3DBourquard%26aufirst%3DT.%26aulast%3DClaeysen%26aufirst%3DS.%26aulast%3DBockaert%26aufirst%3DJ.%26aulast%3DCrepieux%26aufirst%3DP.%26aulast%3DPoupon%26aufirst%3DA.%26aulast%3DReiter%26aufirst%3DE.%26aulast%3DMarin%26aufirst%3DP.%26aulast%3DVandermoere%26aufirst%3DF.%26atitle%3DPhosphorylation%2520of%2520beta-arrestin%25202%2520at%2520Thr383%2520by%2520MEK%2520underlies%2520beta-arrestin-dependent%2520activation%2520of%2520Erk1%252F2%2520by%2520GPCRs%26jtitle%3DeLife%26date%3D2017%26volume%3D6%26doi%3D10.7554%2FeLife.23777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, E.</span></span> <span> </span><span class="NLM_article-title">Nuclear export of MAP kinase (ERK) involves a MAP kinase kinase (MEK)-dependent active transport mechanism</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">849</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1083/jcb.148.5.849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1083%2Fjcb.148.5.849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10704436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhslansrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2000&pages=849-856&author=M.+Adachiauthor=M.+Fukudaauthor=E.+Nishida&title=Nuclear+export+of+MAP+kinase+%28ERK%29+involves+a+MAP+kinase+kinase+%28MEK%29-dependent+active+transport+mechanism&doi=10.1083%2Fjcb.148.5.849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear export of MAP kinase (ERK) involves a MAP kinase kinase (MEK)-dependent active transport mechanism</span></div><div class="casAuthors">Adachi, Makoto; Fukuda, Makoto; Nishida, Eisuke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">849-856</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">In response to extracellular stimuli, mitogen-activated protein kinase (MAPK, also known as ERK), which localizes to the cytoplasm in quiescent cells, translocates to the nucleus and then relocalizes to the cytoplasm again.  The relocalization of nuclear MAPK to the cytoplasm was not inhibited by cycloheximide, confirming that the relocalization is achieved by nuclear export, but not synthesis, of MAPK.  The nuclear export of MAPK was inhibited by leptomycin B (LMB), a specific inhibitor of the nuclear export signal (NES)-dependent transport.  We have then shown that MAP kinase kinase (MAPKK, also known as MEK), which mostly localizes to the cytoplasm because of its having NES, is able to shuttle between the cytoplasm and the nucleus constantly.  MAPK, when injected into the nucleus, was rapidly exported from the nucleus by coinjected wild-type MAPKK, but not by the NES-disrupted MAPKK.  In addn., injection of the fragment corresponding to the MAPK-binding site of MAPKK into the nucleus, which would disrupt the binding of MAPK to MAPKK in the nucleus, significantly inhibited the nuclear export of endogenous MAPK.  Taken together, these results suggest that the relocalization of nuclear MAPK to the cytoplasm involves a MAPKK-dependent, active transport mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXku6zczusYbVg90H21EOLACvtfcHk0ljhsBme0XtmWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhslansrw%253D&md5=79e974b197237554b11b4c85726fc649</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1083%2Fjcb.148.5.849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.148.5.849%26sid%3Dliteratum%253Aachs%26aulast%3DAdachi%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DNishida%26aufirst%3DE.%26atitle%3DNuclear%2520export%2520of%2520MAP%2520kinase%2520%2528ERK%2529%2520involves%2520a%2520MAP%2520kinase%2520kinase%2520%2528MEK%2529-dependent%2520active%2520transport%2520mechanism%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2000%26volume%3D148%26spage%3D849%26epage%3D856%26doi%3D10.1083%2Fjcb.148.5.849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgermeister, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuderland, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanoch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liscovitch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seger, R.</span></span> <span> </span><span class="NLM_article-title">Interaction with MEK causes nuclear export and downregulation of peroxisome proliferator-activated receptor gamma</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">803</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1128/MCB.00601-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1128%2FMCB.00601-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=17101779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=803-817&author=E.+Burgermeisterauthor=D.+Chuderlandauthor=T.+Hanochauthor=M.+Meyerauthor=M.+Liscovitchauthor=R.+Seger&title=Interaction+with+MEK+causes+nuclear+export+and+downregulation+of+peroxisome+proliferator-activated+receptor+gamma&doi=10.1128%2FMCB.00601-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction with MEK causes nuclear export and downregulation of peroxisome proliferator-activated receptor</span></div><div class="casAuthors">Burgermeister, Elke; Chuderland, Dana; Hanoch, Tamar; Meyer, Markus; Liscovitch, Mordechai; Seger, Rony</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">803-817</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mitogen-activated protein kinase (MAPK)/extracellular signal-regulated kinase (ERK) cascade plays a central role in intracellular signaling by many extracellular stimuli.  One target of the ERK cascade is peroxisome proliferator-activated receptor γ (PPARγ), a nuclear receptor that promotes differentiation and apoptosis.  It was previously demonstrated that PPARγ activity is attenuated upon mitogenic stimulation due to phosphorylation of its Ser84 by ERKs.  Here we show that stimulation by tetradecanoyl phorbol acetate (TPA) attenuates PPARγ's activity in a MEK-dependent manner, even when Ser84 is mutated to Ala.  To elucidate the mechanism of attenuation, we found that PPARγ directly interacts with MEKs, which are the activators of ERKs, but not with ERKs themselves, both in vivo and in vitro.  This interaction is facilitated by MEKs' phosphorylation and is mediated by the basic D domain of MEK1 and the AF2 domain of PPARγ.  Immunofluorescence microscopy and subcellular fractionation revealed that MEK1 exports PPARγ from the nucleus, and this finding was supported by small interfering RNA knockdown of MEK1 and use of a cell-permeable interaction-blocking peptide, which prevented TPA-induced export of PPARγ from the nucleus.  Thus, we show here a novel mode of downregulation of PPARγ by its MEK-dependent redistribution from the nucleus to the cytosol.  This unanticipated role for the stimulation-induced nuclear shuttling of MEKs shows that MEKs can regulate addnl. signaling components besides the ERK cascade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyBt7HomvWmLVg90H21EOLACvtfcHk0ljhsBme0XtmWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGht7s%253D&md5=60b9cd18e1c5ac33122ea01c109fcd02</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1128%2FMCB.00601-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00601-06%26sid%3Dliteratum%253Aachs%26aulast%3DBurgermeister%26aufirst%3DE.%26aulast%3DChuderland%26aufirst%3DD.%26aulast%3DHanoch%26aufirst%3DT.%26aulast%3DMeyer%26aufirst%3DM.%26aulast%3DLiscovitch%26aufirst%3DM.%26aulast%3DSeger%26aufirst%3DR.%26atitle%3DInteraction%2520with%2520MEK%2520causes%2520nuclear%2520export%2520and%2520downregulation%2520of%2520peroxisome%2520proliferator-activated%2520receptor%2520gamma%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D803%26epage%3D817%26doi%3D10.1128%2FMCB.00601-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perry, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudnicki, M. A.</span></span> <span> </span><span class="NLM_article-title">Activated MEK1 binds the nuclear MyoD transcriptional complex to repress transactivation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(01)00302-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2FS1097-2765%2801%2900302-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=11545732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWgsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=291-301&author=R.+L.+Perryauthor=M.+H.+Parkerauthor=M.+A.+Rudnicki&title=Activated+MEK1+binds+the+nuclear+MyoD+transcriptional+complex+to+repress+transactivation&doi=10.1016%2FS1097-2765%2801%2900302-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Activated MEK1 binds the nuclear MyoD transcriptional complex to repress transactivation</span></div><div class="casAuthors">Perry, Robert L. S.; Parker, Maura H.; Rudnicki, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">291-301</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">To elucidate the mechanism through which MAPK signaling regulates the MyoD family of transcription factors, we investigated the role of the signaling intermediate MEK1 in myogenesis.  Transfection of activated MEK1 strongly repressed gene activation and myogenic conversion by the MyoD family.  This repression was not mediated by direct phosphorylation of MyoD or by changes in MyoD stability or subcellular distribution.  Deletion mapping revealed that MEK1-mediated repression required the MyoD amino-terminal transactivation domain.  Moreover, activated MEK1 was nuclearly localized and bound a complex contg. MyoD in a manner that is dependent on the presence of the MyoD amino terminus.  Together, these data demonstrate that MEK1 signaling has a strong neg. effect on MyoD activity via a novel mechanism involving binding of MEK1 to the nuclear MyoD transcriptional complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDVpuTEhArrVg90H21EOLACvtfcHk0ljhsBme0XtmWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWgsbg%253D&md5=323a010043b41d04cb60d5aa907aa3ed</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2900302-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252900302-1%26sid%3Dliteratum%253Aachs%26aulast%3DPerry%26aufirst%3DR.%2BL.%26aulast%3DParker%26aufirst%3DM.%2BH.%26aulast%3DRudnicki%26aufirst%3DM.%2BA.%26atitle%3DActivated%2520MEK1%2520binds%2520the%2520nuclear%2520MyoD%2520transcriptional%2520complex%2520to%2520repress%2520transactivation%26jtitle%3DMol.%2520Cell%26date%3D2001%26volume%3D8%26spage%3D291%26epage%3D301%26doi%3D10.1016%2FS1097-2765%2801%2900302-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Procaccia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ordan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendetz-Nezer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seger, R.</span></span> <span> </span><span class="NLM_article-title">Direct binding of MEK1 and MEK2 to AKT induces Foxo1 phosphorylation, cellular migration and metastasis</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">43078</span> <span class="refDoi"> DOI: 10.1038/srep43078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fsrep43078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=28225038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVCku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&author=S.+Procacciaauthor=M.+Ordanauthor=I.+Cohenauthor=S.+Bendetz-Nezerauthor=R.+Seger&title=Direct+binding+of+MEK1+and+MEK2+to+AKT+induces+Foxo1+phosphorylation%2C+cellular+migration+and+metastasis&doi=10.1038%2Fsrep43078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Direct binding of MEK1 and MEK2 to AKT induces Foxo1 phosphorylation, cellular migration and metastasis</span></div><div class="casAuthors">Procaccia, Shiri; Ordan, Merav; Cohen, Izel; Bendetz-Nezer, Sarit; Seger, Rony</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43078</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Crosstalk between the ERK cascade and other signaling pathways is one of the means by which it acquires its signaling specificity.  Here we identified a direct interaction of both MEK1 and MEK2 with AKT.  The interaction is mediated by the proline rich domain of MEK1/2 and regulated by phosphorylation of Ser298 in MEK1, or Ser306 in MEK2, which we identified here as a novel regulatory site.  We further developed a blocking peptide, which inhibits the interaction between MEK and AKT, and when applied to cells, affects migration and adhesion, but not proliferation.  The specific mechanism of action of the MEK-AKT complex involves phosphorylation of the migration-related transcription factor FoxO1.  Importantly, prevention of the interaction results in a decreased metastasis formation in a breast cancer mouse model.  Thus, the identified interaction both sheds light on how signaling specificity is detd., and represents a possible new therapeutic target for metastatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon2bDXG4r_P7Vg90H21EOLACvtfcHk0lh1oYQosbmjkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsVCku7Y%253D&md5=e399d969e2cebc3305c4aadf94b25196</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fsrep43078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep43078%26sid%3Dliteratum%253Aachs%26aulast%3DProcaccia%26aufirst%3DS.%26aulast%3DOrdan%26aufirst%3DM.%26aulast%3DCohen%26aufirst%3DI.%26aulast%3DBendetz-Nezer%26aufirst%3DS.%26aulast%3DSeger%26aufirst%3DR.%26atitle%3DDirect%2520binding%2520of%2520MEK1%2520and%2520MEK2%2520to%2520AKT%2520induces%2520Foxo1%2520phosphorylation%252C%2520cellular%2520migration%2520and%2520metastasis%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26doi%3D10.1038%2Fsrep43078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novakova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span> <span> </span><span class="NLM_article-title">Development of multifunctional histone deacetylase 6 degraders with potent antimyeloma activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7042</span>– <span class="NLM_lpage">7057</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00516</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00516" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlams7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7042-7057&author=H.+Wuauthor=K.+Yangauthor=Z.+Zhangauthor=E.+D.+Leistenauthor=Z.+Liauthor=H.+Xieauthor=J.+Liuauthor=K.+A.+Smithauthor=Z.+Novakovaauthor=C.+Barinkaauthor=W.+Tang&title=Development+of+multifunctional+histone+deacetylase+6+degraders+with+potent+antimyeloma+activity&doi=10.1021%2Facs.jmedchem.9b00516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Multifunctional Histone Deacetylase 6 Degraders with Potent Antimyeloma Activity</span></div><div class="casAuthors">Wu, Hao; Yang, Ka; Zhang, Zhongrui; Leisten, Eric D.; Li, Ziyuan; Xie, Haibo; Liu, Jin; Smith, Kerry A.; Novakova, Zora; Barinka, Cyril; Tang, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7042-7057</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) primarily catalyzes the removal of acetyl group from the side chain of acetylated lysine residues in cytoplasmic proteins such as α-tubulin and HSP90.  HDAC6 is involved in multiple disease-relevant pathways.  Based on the proteolysis targeting chimera strategy, we previously developed the first HDAC6 degrader by tethering a pan-HDAC inhibitor with cereblon (CRBN) E3 ubiquitin ligase ligand.  We herein report our new generation of multifunctional HDAC6 degraders by tethering selective HDAC6 inhibitor Nexturastat A with CRBN ligand that can synergize with HDAC6 degrdn. for the antiproliferation of multiple myeloma (MM).  This new class of degraders exhibited improved potency and selectivity for the degrdn. of HDAC6.  After the optimization of the linker length and linking positions, we discovered potent HDAC6 degraders with nanomolar DC50 and promising antiproliferation activity in multiple myeloma (MM) cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb58zf04V2GLVg90H21EOLACvtfcHk0lh1oYQosbmjkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlams7nO&md5=7aab402404a5fce732b4b68fd76395d4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00516%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLeisten%26aufirst%3DE.%2BD.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DK.%2BA.%26aulast%3DNovakova%26aufirst%3DZ.%26aulast%3DBarinka%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520multifunctional%2520histone%2520deacetylase%25206%2520degraders%2520with%2520potent%2520antimyeloma%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7042%26epage%3D7057%26doi%3D10.1021%2Facs.jmedchem.9b00516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nabet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">The dTAG system for immediate and target-specific protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0021-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fs41589-018-0021-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29581585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=431-441&author=B.+Nabetauthor=J.+M.+Robertsauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=S.+Dastjerdiauthor=A.+Yangauthor=A.+L.+Leggettauthor=M.+A.+Erbauthor=M.+A.+Lawlorauthor=A.+Souzaauthor=T.+G.+Scottauthor=S.+Vittoriauthor=J.+A.+Perryauthor=J.+Qiauthor=G.+E.+Winterauthor=K.+K.+Wongauthor=N.+S.+Grayauthor=J.+E.+Bradner&title=The+dTAG+system+for+immediate+and+target-specific+protein+degradation&doi=10.1038%2Fs41589-018-0021-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">The dTAG system for immediate and target-specific protein degradation</span></div><div class="casAuthors">Nabet, Behnam; Roberts, Justin M.; Buckley, Dennis L.; Paulk, Joshiawa; Dastjerdi, Shiva; Yang, Annan; Leggett, Alan L.; Erb, Michael A.; Lawlor, Matthew A.; Souza, Amanda; Scott, Thomas G.; Vittori, Sarah; Perry, Jennifer A.; Qi, Jun; Winter, Georg E.; Wong, Kwok-Kin; Gray, Nathanael S.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">431-441</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dissection of complex biol. systems requires target-specific control of the function or abundance of proteins.  Genetic perturbations are limited by off-target effects, multicomponent complexity, and irreversibility.  Most limiting is the requisite delay between modulation to exptl. measurement.  To enable the immediate and selective control of single protein abundance, we created a chem. biol. system that leverages the potency of cell-permeable heterobifunctional degraders.  The dTAG system pairs a novel degrader of FKBP12F36V with expression of FKBP12F36V in-frame with a protein of interest.  By transgene expression or CRISPR-mediated locus-specific knock-in, we exemplify a generalizable strategy to study the immediate consequence of protein loss.  Using dTAG, we observe an unexpected superior antiproliferative effect of pan-BET bromodomain degrdn. over selective BRD4 degrdn., characterize immediate effects of KRASG12V loss on proteomic signaling, and demonstrate rapid degrdn. in vivo.  This technol. platform will confer kinetic resoln. to biol. investigation and provide target validation in the context of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqwQvCDe-2I7Vg90H21EOLACvtfcHk0lh1oYQosbmjkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFGmsbs%253D&md5=d7d9c111c68c84bc3aa9c2aad3f591a6</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0021-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0021-8%26sid%3Dliteratum%253Aachs%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DLeggett%26aufirst%3DA.%2BL.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLawlor%26aufirst%3DM.%2BA.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520dTAG%2520system%2520for%2520immediate%2520and%2520target-specific%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D431%26epage%3D441%26doi%3D10.1038%2Fs41589-018-0021-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Irrizary, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czabaniuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niosi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span> <span> </span><span class="NLM_article-title">Delineating the role of cooperativity in the design of potent PROTACs for BTK</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E7285</span>– <span class="NLM_lpage">E7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.1803662115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1073%2Fpnas.1803662115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=30012605" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E7285-E7292&author=A.+Zorbaauthor=C.+Nguyenauthor=Y.+Xuauthor=J.+Starrauthor=K.+Borzilleriauthor=J.+Smithauthor=H.+Zhuauthor=K.+A.+Farleyauthor=W.+Dingauthor=J.+Schiemerauthor=X.+Fengauthor=J.+S.+Changauthor=D.+P.+Uccelloauthor=J.+A.+Youngauthor=C.+N.+Garcia-Irrizaryauthor=L.+Czabaniukauthor=B.+Schuffauthor=R.+Oliverauthor=J.+Montgomeryauthor=M.+M.+Haywardauthor=J.+Coeauthor=J.+Chenauthor=M.+Niosiauthor=S.+Luthraauthor=J.+C.+Shahauthor=A.+El-Kattanauthor=X.+Qiuauthor=G.+M.+Westauthor=M.+C.+Noeauthor=V.+Shanmugasundaramauthor=A.+M.+Gilbertauthor=M.+F.+Brownauthor=M.+F.+Calabrese&title=Delineating+the+role+of+cooperativity+in+the+design+of+potent+PROTACs+for+BTK&doi=10.1073%2Fpnas.1803662115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1803662115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1803662115%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DSchiemer%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DGarcia-Irrizary%26aufirst%3DC.%2BN.%26aulast%3DCzabaniuk%26aufirst%3DL.%26aulast%3DSchuff%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNiosi%26aufirst%3DM.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DJ.%2BC.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26atitle%3DDelineating%2520the%2520role%2520of%2520cooperativity%2520in%2520the%2520design%2520of%2520potent%2520PROTACs%2520for%2520BTK%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2018%26volume%3D115%26spage%3DE7285%26epage%3DE7292%26doi%3D10.1073%2Fpnas.1803662115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+M.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Q.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lhjzDi8AzVojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%2BM.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%2BQ.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lhjzDi8AzVojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+Bohmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thoma&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0lixnkXBPkrNJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBohm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThoma%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eroglu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, A.</span></span> <span> </span><span class="NLM_article-title">Combination therapy with BRAF and MEK inhibitors for melanoma: latest evidence and place in therapy</span>. <i>Ther. Adv. Med. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">48</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1177/1758834015616934</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1177%2F1758834015616934" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=26753005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC28XotValsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=48-56&author=Z.+Erogluauthor=A.+Ribas&title=Combination+therapy+with+BRAF+and+MEK+inhibitors+for+melanoma%3A+latest+evidence+and+place+in+therapy&doi=10.1177%2F1758834015616934"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Combination therapy with BRAF and MEK inhibitors for melanoma: latest evidence and place in therapy</span></div><div class="casAuthors">Eroglu, Zeynep; Ribas, Antoni</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Medical Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-56</span>CODEN:
                <span class="NLM_cas:coden">TAMOB8</span>;
        ISSN:<span class="NLM_cas:issn">1758-8340</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">Treatment with BRAF inhibitors such as vemurafenib or dabrafenib in patients with advanced BRAFV600 mutated melanoma has shown objective tumor responses in approx. half of the patients.  However, the duration of responses is limited in a majority of these patients, with progression-free survival rates around 6 mo due to tumor progression from development of acquired resistance.  Preclin. studies have suggested that concurrent inhibition of the BRAF kinases and MEK of the mitogen-activated protein kinase (MAPK) pathway could decrease MAPK-driven acquired resistance, resulting in longer duration of responses, higher rate of tumor responses, and a decrease in the cutaneous toxicities obsd. from paradoxical MAPK pathway activation with BRAF inhibitor monotherapy.  This review provides an overview of the currently available clin. trial data on BRAF and MEK inhibitors together and in combinations with other therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPeAe_WaeHr7Vg90H21EOLACvtfcHk0lixnkXBPkrNJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotValsbc%253D&md5=9709de3d9c9dd47f7fb13ccc11e2b2a6</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1177%2F1758834015616934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1758834015616934%26sid%3Dliteratum%253Aachs%26aulast%3DEroglu%26aufirst%3DZ.%26aulast%3DRibas%26aufirst%3DA.%26atitle%3DCombination%2520therapy%2520with%2520BRAF%2520and%2520MEK%2520inhibitors%2520for%2520melanoma%253A%2520latest%2520evidence%2520and%2520place%2520in%2520therapy%26jtitle%3DTher.%2520Adv.%2520Med.%2520Oncol.%26date%3D2016%26volume%3D8%26spage%3D48%26epage%3D56%26doi%3D10.1177%2F1758834015616934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span> <span> </span><span class="NLM_article-title">AKT and ERK dual inhibitors: The way forward?</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>459</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2019.05.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.canlet.2019.05.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=31128213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFensLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2019&pages=30-40&author=Z.+Caoauthor=Q.+Liaoauthor=M.+Suauthor=K.+Huangauthor=J.+Jinauthor=D.+Cao&title=AKT+and+ERK+dual+inhibitors%3A+The+way+forward%3F&doi=10.1016%2Fj.canlet.2019.05.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">AKT and ERK dual inhibitors: The way forward?</span></div><div class="casAuthors">Cao, Zhe; Liao, Qianjin; Su, Min; Huang, Kai; Jin, Junfei; Cao, Deliang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">459</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30-40</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Phosphatidylinositol 3-kinase (PI3K)/AKT pathway regulates cell growth, proliferation, survival, mobility and invasion.  Mitogen-activated protein kinase (MAPK)/extracellular signal-regulated kinase (ERK) pathway is also an important mitogenic signaling pathway involved in various cellular progresses.  AKT, also named protein kinase B (PKB), is a primary mediator of the PI3K signaling pathway; and ERK at the end of MAPK signaling is the unique substrate and downstream effector of mitogen-activated protein/extracellular signal-regulated kinase (MEK).  The AKT and ERK signaling are both aberrantly activated in a wide range of human cancers and have long been targeted for cancer therapy, but the clin. benefits of these targeted therapies have been limited due to complex cross-talk.  Novel strategies, such as AKT/ERK dual inhibitors, may be needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2mQop_AQFH7Vg90H21EOLACvtfcHk0lixnkXBPkrNJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFensLnE&md5=bdb7fd9232b6f75f46b0800a2627a4e1</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2019.05.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2019.05.025%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DLiao%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DD.%26atitle%3DAKT%2520and%2520ERK%2520dual%2520inhibitors%253A%2520The%2520way%2520forward%253F%26jtitle%3DCancer%2520Lett.%26date%3D2019%26volume%3D459%26spage%3D30%26epage%3D40%26doi%3D10.1016%2Fj.canlet.2019.05.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardasi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadei-Gardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallanzani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrikou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascinu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelsomino, F.</span></span> <span> </span><span class="NLM_article-title">BRAF-mutated colorectal cancer: clinical and molecular insights</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5369</span>, <span class="refDoi"> DOI: 10.3390/ijms20215369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.3390%2Fijms20215369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptFShtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=5369&author=F.+Caputoauthor=C.+Santiniauthor=C.+Bardasiauthor=K.+Cermaauthor=A.+Casadei-Gardiniauthor=A.+Spallanzaniauthor=K.+Andrikouauthor=S.+Cascinuauthor=F.+Gelsomino&title=BRAF-mutated+colorectal+cancer%3A+clinical+and+molecular+insights&doi=10.3390%2Fijms20215369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">BRAF-mutated colorectal cancer: clinical and molecular insights</span></div><div class="casAuthors">Caputo, Francesco; Santini, Chiara; Bardasi, Camilla; Cerma, Krisida; Casadei-Gardini, Andrea; Spallanzani, Andrea; Andrikou, Kalliopi; Cascinu, Stefano; Gelsomino, Fabio</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5369</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Colorectal cancer (CRC) is one of the leading causes of mortality and morbidity in the world.  It is a heterogeneous disease, which can be classified into different subtypes, characterized by specific mol. and morphol. alterations.  In this context, BRAF mutations are found in about 10% of CRC patients and define a particular subtype, characterized by a dismal prognosis, with a median survival of less than 12 mo.  Chemotherapy plus bevacizumab is the current std. therapy in first-line treatment of BRAF-mutated metastatic CRC (mCRC), with triplet (FOLFOXIRI) plus bevacizumab as a valid option in patients with a good performance status.  BRAF inhibitors are not so effective as compared to melanoma, because of various resistance mechanisms.  However, the recently published results of the BEACON trial will establish a new std. of care in this setting.  This review provides insights into the mol. underpinnings underlying the resistance to std. treatment of BRAF-mutated CRCs, with a focus on their mol. heterogeneity and on the research perspectives both from a translational and a clin. point of view.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ0Jox-5s91bVg90H21EOLACvtfcHk0lj7R4XnXQnOFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptFShtbc%253D&md5=8da8f55e3b8146185952edc18cf3f2e0</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.3390%2Fijms20215369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20215369%26sid%3Dliteratum%253Aachs%26aulast%3DCaputo%26aufirst%3DF.%26aulast%3DSantini%26aufirst%3DC.%26aulast%3DBardasi%26aufirst%3DC.%26aulast%3DCerma%26aufirst%3DK.%26aulast%3DCasadei-Gardini%26aufirst%3DA.%26aulast%3DSpallanzani%26aufirst%3DA.%26aulast%3DAndrikou%26aufirst%3DK.%26aulast%3DCascinu%26aufirst%3DS.%26aulast%3DGelsomino%26aufirst%3DF.%26atitle%3DBRAF-mutated%2520colorectal%2520cancer%253A%2520clinical%2520and%2520molecular%2520insights%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D5369%26doi%3D10.3390%2Fijms20215369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aasen, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parajuli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjerkvig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knappskog, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsen, F.</span></span> <span> </span><span class="NLM_article-title">Effective treatment of metastatic melanoma by combining MAPK and PI3K signaling pathway inhibitors</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4235</span>, <span class="refDoi"> DOI: 10.3390/ijms20174235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.3390%2Fijms20174235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVGrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=4235&author=S.+N.+Aasenauthor=H.+Parajuliauthor=T.+Hoangauthor=Z.+Fengauthor=K.+Stokkeauthor=J.+Wangauthor=K.+Royauthor=R.+Bjerkvigauthor=S.+Knappskogauthor=F.+Thorsen&title=Effective+treatment+of+metastatic+melanoma+by+combining+MAPK+and+PI3K+signaling+pathway+inhibitors&doi=10.3390%2Fijms20174235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Effective treatment of metastatic melanoma by combining MAPK and PI3K signaling pathway inhibitors</span></div><div class="casAuthors">Aasen, Synnoeve Nymark; Parajuli, Himalaya; Hoang, Tuyen; Feng, Zichao; Stokke, Krister; Wang, Jiwei; Roy, Kislay; Bjerkvig, Rolf; Knappskog, Stian; Thorsen, Frits</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4235</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Malignant melanoma is the most aggressive type of skin cancer and is closely assocd. with the development of brain metastases.  Despite aggressive treatment, the prognosis has traditionally been poor, necessitating improved therapies.  In melanoma, the mitogen activated protein kinase and the phosphoinositide 3-kinase signaling pathways are commonly altered, and therapeutically inhibiting one of the pathways often upregulates the other, leading to resistance.  Thus, combined treatment targeting both pathways is a promising strategy to overcome this.  Here, we studied the in vitro and in vivo effects of the PI3K inhibitor buparlisib and the MEK1/2 inhibitor trametinib, used either as targeted monotherapies or in combination, on patient-derived melanoma brain metastasis cell lines.  Scratch wound and trans-well assays were carried out to assess the migratory capacity of the cells upon drug treatment, whereas flow cytometry, apoptosis array and Western blots were used to study apoptosis.  Finally, an in vivo treatment expt. was carried out on NOD/SCID mice.  We show that combined therapy was more effective than monotherapy.  Combined treatment also more effectively increased apoptosis, and inhibited tumor growth in vivo.  This suggests a clin. potential of combined treatment to overcome ceased treatment activity which is often seen after monotherapies, and strongly encourages the evaluation of the treatment strategy on melanoma patients with brain metastases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobWDmITf6bWbVg90H21EOLACvtfcHk0lj7R4XnXQnOFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVGrsL0%253D&md5=9ccfa4cc09fa6b6ab5e48247f44c1c08</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.3390%2Fijms20174235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20174235%26sid%3Dliteratum%253Aachs%26aulast%3DAasen%26aufirst%3DS.%2BN.%26aulast%3DParajuli%26aufirst%3DH.%26aulast%3DHoang%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DStokke%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRoy%26aufirst%3DK.%26aulast%3DBjerkvig%26aufirst%3DR.%26aulast%3DKnappskog%26aufirst%3DS.%26aulast%3DThorsen%26aufirst%3DF.%26atitle%3DEffective%2520treatment%2520of%2520metastatic%2520melanoma%2520by%2520combining%2520MAPK%2520and%2520PI3K%2520signaling%2520pathway%2520inhibitors%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D4235%26doi%3D10.3390%2Fijms20174235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lj7R4XnXQnOFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thierbach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartwig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taupitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruhl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, C.</span></span> <span> </span><span class="NLM_article-title">Highly monodisperse water-dispersable iron oxide nanoparticles for biomedical applications</span>. <i>J. Mater. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">7842</span>– <span class="NLM_lpage">7853</span>, <span class="refDoi"> DOI: 10.1039/c0jm01169j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=10.1039%2Fc0jm01169j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVylsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=7842-7853&author=A.+Hofmannauthor=S.+Thierbachauthor=A.+Semischauthor=A.+Hartwigauthor=M.+Taupitzauthor=E.+Ruhlauthor=C.+Graf&title=Highly+monodisperse+water-dispersable+iron+oxide+nanoparticles+for+biomedical+applications&doi=10.1039%2Fc0jm01169j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Highly monodisperse water-dispersable iron oxide nanoparticles for biomedical applications</span></div><div class="casAuthors">Hofmann, Andreas; Thierbach, Steffen; Semisch, Annetta; Hartwig, Andrea; Taupitz, Matthias; Ruehl, Eckart; Graf, Christina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Materials Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">7842-7853</span>CODEN:
                <span class="NLM_cas:coden">JMACEP</span>;
        ISSN:<span class="NLM_cas:issn">0959-9428</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We demonstrate a unique approach for prepg. high quality iron oxide (Fe3O4) nanoparticles functionalized by newly developed multifunctional dendron ligands for biomedical applications.  These particles are suitable for magnetic resonance imaging (MRI), highly stable in aq. solns. as well as physiol. media and not cytotoxic.  In particular, oleic acid capped Fe3O4 particles (d = 12 ± 0.8 nm) were modified in a ligand exchange process by investigating several dendron ligands of variable size and an adjustable no. of polar end groups.  The dendron based ligands lead only to a slight increase in hydrodynamic diam. of the nanoparticles after the ligand exchange process (∼6 nm).  They also allow an adjustment of the particle polarity as well as a gradually variable surface functionalization.  Light scattering, transmission electron microscopy, and visible spectroscopy studies show consistently that the dendron-capped iron oxide nanoparticles exhibit excellent stability in various physiol. media as well as aq. solns. in a broad pH range.  It is also demonstrated by magnetic resonance studies that the magnetic relaxivity is almost not affected by the ligand exchange.  Therefore, such small particles might be of specific interest for cardiovascular MRI and MRI of extravascular targets.  In addn., the present approach opens new possibilities for the specific linking of biomols. to the particle surface, which can be beneficial for various biol. sensing and therapeutic applications.  The cytotoxicity of the Fe3O4 nanoparticles was evaluated using the WST-8 assay.  In the examd. concn. range up to 100 μg Fe/mL no significant decrease in cell viability was detected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq51NfZ9tw6urVg90H21EOLACvtfcHk0liYKyPORAdrGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVylsLjF&md5=626197097b6bed2ee328c2b0b0b33117</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1039%2Fc0jm01169j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0jm01169j%26sid%3Dliteratum%253Aachs%26aulast%3DHofmann%26aufirst%3DA.%26aulast%3DThierbach%26aufirst%3DS.%26aulast%3DSemisch%26aufirst%3DA.%26aulast%3DHartwig%26aufirst%3DA.%26aulast%3DTaupitz%26aufirst%3DM.%26aulast%3DRuhl%26aufirst%3DE.%26aulast%3DGraf%26aufirst%3DC.%26atitle%3DHighly%2520monodisperse%2520water-dispersable%2520iron%2520oxide%2520nanoparticles%2520for%2520biomedical%2520applications%26jtitle%3DJ.%2520Mater.%2520Chem.%26date%3D2010%26volume%3D20%26spage%3D7842%26epage%3D7853%26doi%3D10.1039%2Fc0jm01169j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i89"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01609">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_03716"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01609?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01609</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">MEK1/2 degradation induced by compounds <b>24</b> and <b>27</b>; MEK1/2 degradation induced by compound <b>50</b>; binding affinity curves of compounds <b>24</b>, <b>27</b>, and <b>50</b>–<b>53</b>; Western blot validation of proteomics studies of compound <b>50</b>; Neo-substrates of CRBN degradation by compound <b>50</b>; compounds <b>24</b> and <b>27</b> degraded MEK1/2 proteins and inhibited downstream ERK signaling in H3122 and SUDHL1 cells; and antiproliferation potencies of PLX4032 and ZSTK474 in HT-29 and SK-MEL-28 cell lines; and <sup>1</sup>H, <sup>13</sup>C, HPLC, and HRMS spectra of compounds <b>24</b>, <b>27</b>, and <b>50</b>–<b>53</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_001.pdf">jm0c01609_si_001.pdf (3.18 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01609/suppl_file/jm0c01609_si_002.csv">jm0c01609_si_002.csv (7.04 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01609&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-24%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01609%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01609" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bc9a5bdc3dac","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
