
<html lang="en"     class="pb-page"  data-request-id="bcd7caa4-8198-4ad4-95b3-b997bfb73088"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-13;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.9b01942;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Fragment-based Differential Targeting of PPI Stabilizer Interfaces" /></meta><meta name="dc.Creator" content="Xavier  Guillory" /></meta><meta name="dc.Creator" content="Madita  Wolter" /></meta><meta name="dc.Creator" content="Seppe  Leysen" /></meta><meta name="dc.Creator" content="João Filipe  Neves" /></meta><meta name="dc.Creator" content="Ave  Kuusk" /></meta><meta name="dc.Creator" content="Sylvia  Genet" /></meta><meta name="dc.Creator" content="Bente  Somsen" /></meta><meta name="dc.Creator" content="John Kenneth  Morrow" /></meta><meta name="dc.Creator" content="Emma  Rivers" /></meta><meta name="dc.Creator" content="Lotte  van Beek" /></meta><meta name="dc.Creator" content="Joe  Patel" /></meta><meta name="dc.Creator" content="Robert  Goodnow" /></meta><meta name="dc.Creator" content="Heike  Schoenherr" /></meta><meta name="dc.Creator" content="Nathan  Fuller" /></meta><meta name="dc.Creator" content="Qing  Cao" /></meta><meta name="dc.Creator" content="Richard G.  Doveston" /></meta><meta name="dc.Creator" content="Luc  Brunsveld" /></meta><meta name="dc.Creator" content="Michelle R.  Arkin" /></meta><meta name="dc.Creator" content="Paola  Castaldi" /></meta><meta name="dc.Creator" content="Helen  Boyd" /></meta><meta name="dc.Creator" content="Isabelle  Landrieu" /></meta><meta name="dc.Creator" content="Hongming  Chen" /></meta><meta name="dc.Creator" content="Christian  Ottmann" /></meta><meta name="dc.Description" content="Stabilization of protein–protein interactions (PPIs) holds great potential for therapeutic agents, as illustrated by the successful drugs rapamycin and lenalidomide. However, how such interface-bin..." /></meta><meta name="Description" content="Stabilization of protein–protein interactions (PPIs) holds great potential for therapeutic agents, as illustrated by the successful drugs rapamycin and lenalidomide. However, how such interface-bin..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 5, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01942" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01942" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01942" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01942" /></link>
        
    
    

<title>Fragment-based Differential Targeting of PPI Stabilizer Interfaces | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01942" /></meta><meta property="og:title" content="Fragment-based Differential Targeting of PPI Stabilizer Interfaces" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0006.jpeg" /></meta><meta property="og:description" content="Stabilization of protein–protein interactions (PPIs) holds great potential for therapeutic agents, as illustrated by the successful drugs rapamycin and lenalidomide. However, how such interface-binding molecules can be created in a rational, bottom-up manner is a largely unanswered question. We report here how a fragment-based approach can be used to identify chemical starting points for the development of small-molecule stabilizers that differentiate between two different PPI interfaces of the adapter protein 14-3-3. The fragments discriminately bind to the interface of 14-3-3 with the recognition motif of either the tumor suppressor protein p53 or the oncogenic transcription factor TAZ. This X-ray crystallography driven study shows that the rim of the interface of individual 14-3-3 complexes can be targeted in a differential manner with fragments that represent promising starting points for the development of specific 14-3-3 PPI stabilizers." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01942"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01942">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01942&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01942&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01942&amp;href=/doi/10.1021/acs.jmedchem.9b01942" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 6694-6707</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02052" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01929" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment-based Differential Targeting of PPI Stabilizer Interfaces</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xavier Guillory</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xavier Guillory</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xavier++Guillory">Xavier Guillory</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Madita Wolter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Madita Wolter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Madita++Wolter">Madita Wolter</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Seppe Leysen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Seppe Leysen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Seppe++Leysen">Seppe Leysen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">João Filipe Neves</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">João Filipe Neves</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CNRS ERL9002 Integrative Structural Biology F-59000 Lille, France</div><div class="loa-info-affiliations-info">Univ. Lille, Inserm, CHU Lille, Institut Pasteur de Lille, U1167-RID-AGE, Risk Factors and Molecular Determinants of Aging-Related Diseases, F-59000 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jo%C3%A3o+Filipe++Neves">João Filipe Neves</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ave Kuusk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ave Kuusk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div><div class="loa-info-affiliations-info">Hit Discovery, Discovery Sciences, Biopharmaceutical R&D, AstraZeneca, Gothenburg, 431 50 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ave++Kuusk">Ave Kuusk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sylvia Genet</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sylvia Genet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sylvia++Genet">Sylvia Genet</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bente Somsen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bente Somsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bente++Somsen">Bente Somsen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John Kenneth Morrow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John Kenneth Morrow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry and Small Molecule Discovery Center (SMDC), University of California, San Francisco, California 94143, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+Kenneth++Morrow">John Kenneth Morrow</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Emma Rivers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emma Rivers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hit Discovery, Discovery Sciences, Biopharmaceutical R&D, AstraZeneca, Gothenburg, 431 50 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emma++Rivers">Emma Rivers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lotte van Beek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lotte van Beek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lotte++van+Beek">Lotte van Beek</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joe Patel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joe Patel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joe++Patel">Joe Patel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert Goodnow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert Goodnow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert++Goodnow">Robert Goodnow</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heike Schoenherr</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heike Schoenherr</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heike++Schoenherr">Heike Schoenherr</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nathan Fuller</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nathan Fuller</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nathan++Fuller">Nathan Fuller</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qing Cao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qing Cao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qing++Cao">Qing Cao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard G. Doveston</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard G. Doveston</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard+G.++Doveston">Richard G. Doveston</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Luc Brunsveld</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luc Brunsveld</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luc++Brunsveld">Luc Brunsveld</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5675-511X" title="Orcid link">http://orcid.org/0000-0001-5675-511X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michelle R. Arkin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michelle R. Arkin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry and Small Molecule Discovery Center (SMDC), University of California, San Francisco, California 94143, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michelle+R.++Arkin">Michelle R. Arkin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9366-6770" title="Orcid link">http://orcid.org/0000-0002-9366-6770</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paola Castaldi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Castaldi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Castaldi">Paola Castaldi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Helen Boyd</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helen Boyd</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hit Discovery, Discovery Sciences, Biopharmaceutical R&D, AstraZeneca, Gothenburg, 431 50 Mölndal, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helen++Boyd">Helen Boyd</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Isabelle Landrieu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Isabelle Landrieu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CNRS ERL9002 Integrative Structural Biology F-59000 Lille, France</div><div class="loa-info-affiliations-info">Univ. Lille, Inserm, CHU Lille, Institut Pasteur de Lille, U1167-RID-AGE, Risk Factors and Molecular Determinants of Aging-Related Diseases, F-59000 Lille, France</div></div><span class="conrtib-corresp"><strong>*</strong>For I.L.: <a href="/cdn-cgi/l/email-protection#cba2b8aaa9aea7a7aee5a7aaa5afb9a2aebe8bbea5a2bde6a7a2a7a7aee5adb9"><span class="__cf_email__" data-cfemail="4f263c2e2d2a23232a61232e212b3d262a3a0f3a21263962232623232a61293d">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Isabelle++Landrieu">Isabelle Landrieu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4883-2637" title="Orcid link">http://orcid.org/0000-0002-4883-2637</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongming Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongming Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hit Discovery, Discovery Sciences, Biopharmaceutical R&D, AstraZeneca, Gothenburg, 431 50 Mölndal, Sweden</div></div><span class="conrtib-corresp"><strong>*</strong>ForH.C.: <a href="/cdn-cgi/l/email-protection#5e3631303933373039703d363b30696f1e36312a333f3732703d3133"><span class="__cf_email__" data-cfemail="8ae2e5e4ede7e3e4eda4e9e2efe4bdbbcae2e5fee7ebe3e6a4e9e5e7">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongming++Chen">Hongming Chen</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Christian Ottmann</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Ottmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Chemical Biology, Department of Biomedical Engineering and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</div><div class="loa-info-affiliations-info">Department of Organic Chemistry, University of Duisburg-Essen, 47057 Duisburg, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>For C.O.: phone, +31 40-247-2835; email, <a href="/cdn-cgi/l/email-protection#bcdf92d3c8c8d1ddd2d2fcc8c9d992d2d0"><span class="__cf_email__" data-cfemail="7a1954150e0e171b14143a0e0f1f541416">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Ottmann">Christian Ottmann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7315-0315" title="Orcid link">http://orcid.org/0000-0001-7315-0315</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01942&amp;href=/doi/10.1021%2Facs.jmedchem.9b01942" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 6694–6707</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 5, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 November 2019</li><li><span class="item_label"><b>Published</b> online</span>5 June 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 July 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01942" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01942</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc text">with CC-BY-NC-ND<a href="/page/policy/authorchoice_ccbyncnd_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01942"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4371</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01942" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment-based Differential Targeting of PPI Stabilizer Interfaces&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xavier&quot;,&quot;last_name&quot;:&quot;Guillory&quot;},{&quot;first_name&quot;:&quot;Madita&quot;,&quot;last_name&quot;:&quot;Wolter&quot;},{&quot;first_name&quot;:&quot;Seppe&quot;,&quot;last_name&quot;:&quot;Leysen&quot;},{&quot;first_name&quot;:&quot;João&quot;,&quot;last_name&quot;:&quot;Filipe Neves&quot;},{&quot;first_name&quot;:&quot;Ave&quot;,&quot;last_name&quot;:&quot;Kuusk&quot;},{&quot;first_name&quot;:&quot;Sylvia&quot;,&quot;last_name&quot;:&quot;Genet&quot;},{&quot;first_name&quot;:&quot;Bente&quot;,&quot;last_name&quot;:&quot;Somsen&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Kenneth Morrow&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;Rivers&quot;},{&quot;first_name&quot;:&quot;Lotte&quot;,&quot;last_name&quot;:&quot;van Beek&quot;},{&quot;first_name&quot;:&quot;Joe&quot;,&quot;last_name&quot;:&quot;Patel&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;Goodnow&quot;},{&quot;first_name&quot;:&quot;Heike&quot;,&quot;last_name&quot;:&quot;Schoenherr&quot;},{&quot;first_name&quot;:&quot;Nathan&quot;,&quot;last_name&quot;:&quot;Fuller&quot;},{&quot;first_name&quot;:&quot;Qing&quot;,&quot;last_name&quot;:&quot;Cao&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;G. Doveston&quot;},{&quot;first_name&quot;:&quot;Luc&quot;,&quot;last_name&quot;:&quot;Brunsveld&quot;},{&quot;first_name&quot;:&quot;Michelle&quot;,&quot;last_name&quot;:&quot;R. Arkin&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Castaldi&quot;},{&quot;first_name&quot;:&quot;Helen&quot;,&quot;last_name&quot;:&quot;Boyd&quot;},{&quot;first_name&quot;:&quot;Isabelle&quot;,&quot;last_name&quot;:&quot;Landrieu&quot;},{&quot;first_name&quot;:&quot;Hongming&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Ottmann&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;6694-6707&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01942&quot;},&quot;abstract&quot;:&quot;Stabilization of protein–protein interactions (PPIs) holds great potential for therapeutic agents, as illustrated by the successful drugs rapamycin and lenalidomide. However, how such interface-binding molecules can be created in a rational, bottom-up manner is a largely unanswered question. We report here how a fragment-based approach can be used to identify chemical starting points for the development of small-molecule stabilizers that differentiate between two different PPI interfaces of the adapter protein 14-3-3. The fragments discriminately bind to the interface of 14-3-3 with the recognition motif of either the tumor suppressor protein p53 or the oncogenic transcription factor TAZ. This X-ray crystallography driven study shows that the rim of the interface of individual 14-3-3 complexes can be targeted in a differential manner with fragments that represent promising starting points for the development of specific 14-3-3 PPI stabilizers.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01942&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01942" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01942&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01942" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01942&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01942" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01942&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01942&amp;href=/doi/10.1021/acs.jmedchem.9b01942" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01942" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01942" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01942&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01942%26sid%3Dliteratum%253Aachs%26pmid%3D32501690%26genre%3Darticle%26aulast%3DGuillory%26date%3D2020%26atitle%3DFragment-based%2BDifferential%2BTargeting%2Bof%2BPPI%2BStabilizer%2BInterfaces%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D13%26spage%3D6694%26epage%3D6707%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290817" title="Interfaces">Interfaces</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=290865" title="Nitrogen">Nitrogen</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/jmcmar.2020.63.issue-13/20200709/jmcmar.2020.63.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Stabilization of protein–protein interactions (PPIs) holds great potential for therapeutic agents, as illustrated by the successful drugs rapamycin and lenalidomide. However, how such interface-binding molecules can be created in a rational, bottom-up manner is a largely unanswered question. We report here how a fragment-based approach can be used to identify chemical starting points for the development of small-molecule stabilizers that differentiate between two different PPI interfaces of the adapter protein 14-3-3. The fragments discriminately bind to the interface of 14-3-3 with the recognition motif of either the tumor suppressor protein p53 or the oncogenic transcription factor TAZ. This X-ray crystallography driven study shows that the rim of the interface of individual 14-3-3 complexes can be targeted in a differential manner with fragments that represent promising starting points for the development of specific 14-3-3 PPI stabilizers.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16334" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16334" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">PPI modulation is one of the most promising and active fields in current drug discovery and chemical biology.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> However, PPI modulation is in many instances understood synonymously with PPI inhibition, disregarding the complementary strategy of stabilizing PPIs. This is despite the fact that a multitude of natural products<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and a growing number of synthetic compounds<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> validate the feasibility and value of small-molecule PPI stabilization. Stabilizers of PPIs display a number of advantages like their uncompetitive nature and their potential for higher specificity. They bind to composite pockets at the interface of their target protein complexes that naturally can show a much higher structural variability compared for example to active sites of enzymes or ligand binding pockets of receptors. Consequently, the potential to develop selective PPI-stabilizing compounds might be higher than with ligands that bind to conventional drug targets. An especially interesting target for PPI modulation is the adapter protein 14-3-3. This “hub” protein, consisting of a family of seven isoforms, interacts with several hundred partner proteins<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> and regulates the activity of many disease-related proteins like C-Raf<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a> (cancer), Tau<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> (Alzheimer’s), and CFTR<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> (cystic fibrosis).</div><div class="NLM_p">Here, we focused on the interaction of 14-3-3 with the transcriptional coactivator TAZ and the tumor suppressor protein p53. TAZ interacts with 14-3-3 via a motif surrounding S89, which, as part of the Hippo signaling pathway, is phosphorylated by LATS1/2, resulting in cytoplasmic retention of TAZ and thus its functional deactivation.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Loss of this regulation leads to abrogation of contact inhibition, which is one of the prerequisites of cancer formation and growth of solid tumors. Binding of 14-3-3 to p53, following phosphorylation of T387 of p53 in response to DNA damage, is protective against MDM2-mediated degradation.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In addition, 14-3-3 binding to p53 facilitates p53 dimer–dimer interaction<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and stabilization of the functional p53 tetramer.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> 14-3-3 thus plays an important role in maintaining the tumor suppressor role of p53 by stimulating p53’s transcriptional activity. Enhancing the binding of 14-3-3 to these two cancer-related proteins with a small-molecule 14-3-3 PPI stabilizer could convey a positive therapeutic effect in many cancer types.</div><div class="NLM_p">Importantly, the potential “druggability” of 14-3-3 proteins has been validated by nature in the form of the natural products class of the fusicoccanes. These diterpene glycosides have been shown to stabilize a number of 14-3-3 PPIs and display biological activities that might be of therapeutic benefit in cancer,<a onclick="showRef(event, 'ref5 ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref5 ref16 ref17 ref18 ref19 ref20">(5,16−20)</a> cystic fibrosis,<a onclick="showRef(event, 'ref10 ref11 ref21'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref21">(10,11,21)</a> and axon regeneration.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The eponymous molecule, the fungal toxin fusicoccin A (FC-A), was shown to stabilize the interaction of 14-3-3 proteins with the regulatory C-terminus of the plant plasma ATPase, thereby activating this important proton pump.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Later it was shown that the related compound cotylenin A displays a remarkable anticancer activity while showing a very low general toxicity.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> One possible target for this activity was suggested by us to be the negative regulatory complex between the N-terminus of the oncogenic kinase C-Raf and 14-3-3 proteins.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The crystal structure of cotylenin A in complex with 14-3-3ζ and the diphospho peptide C-RafpS233pS259 showed that the natural product binds to a gap in the interface pocket simultaneously, establishing contacts to both 14-3-3 and the C-Raf-derived peptide. Recently, an optimized semisynthetic fusicoccane was shown to display enhanced antiproliferative activity against cancer cells.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Furthermore, we reported that the natural product FC-A displayed a beneficial effect for a potential therapy of breast cancer and cystic fibrosis by stabilizing the inhibitory interaction of 14-3-3 with estrogen receptor α<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> or the chloride channel CFTR.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Although the fusicoccanes are very useful proof-of-principle molecules, their optimization is not without synthetic challenges. Therefore, additional approaches to identify starting points for the development of PPI stabilizers are needed. Because traditional high throughput-screening (HTS) were so far rewarded with only limited success,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> alternative methods like fragment-based drug discovery (FBDD) holds the most promise to achieve this goal.</div><div class="NLM_p last">In this study, we report the identification of fragments that bind to the distinct interfaces of 14-3-3 with the partner proteins p53 and TAZ, hinting at the possibility to introduce specificity at an early stage of fragment-based optimization of 14-3-3 PPI stabilizing ligands.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03734" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03734" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Screening for Fragments That Bind to the Interface of 14-3-3 with TAZ and p53</h3><div class="NLM_p">For the identification of interface-binding fragments that might be developed into PPI-stabilizing compounds, we employed a crystallography-based screen of fragment cocktails.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Here, crystals of the binary 14-3-3 complexes were soaked in solutions of fragment mixtures (100 cocktails of five each, part of the X-ray crystallography subset of the AstraZeneca fragments collection) at a final concentration of 10 mM per fragment. This screen identified two scaffolds that bound to a shallow pocket that is located in the central binding channel of 14-3-3. Both fragment scaffolds contain an amidine group (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,D) that engages in a salt-bridge with the side chain carboxyl group of glutamic acid 14 (E14) of 14-3-3 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,F). The planar aromatic ring systems sit in a shallow pocket that is comprised of 14-3-3 residues E39, N42, and L43 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). The first fragment (AZ-001) can adopt two orientations with the chloride substituent either pointing toward or away from the peptide-binding channel of 14-3-3. The second fragment (AZ-002) shows a more defined orientation, with the benzyl substituent of the thiophene ring pointing away from the peptide, resting on an uncharged cushion comprised by the hydrocarbon part of C38 and E39 of helix 3 of 14-3-3. Structural analogues of AZ-002, namely AZ-003 and AZ-004, were additionally selected from the screen and observed to bind to 14-3-3 in the presence of the p53pT387 but not contacting the peptide (Supporting Information (SI), <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figures S1–S2</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Identification of fragments binding to the interface of 14-3-3σ with p53. (A) Fragment AZ-001. (B) Crystal structure of AZ-001 (yellow sticks) in complex with 14-3-3σ (white surface) and p53pT387 (orange sticks), PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S40">6S40</a>. The final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown as blue mesh. (C) Detailed view of the binding pocket of AZ-001. The most prominent interaction is a salt-bridge (dotted black line) between the amidine of AZ-001 and carboxyl group of E14 of 14-3-3σ. (D) Fragment AZ-002. (E) Crystal structure of AZ-002 (yellow sticks) in complex with 14-3-3σ (white surface) and p53pT387 (orange sticks), PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWI">6RWI</a>. The final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown as blue mesh. (F) Detailed view of the binding pocket of AZ-002. Also with AZ-002, the most prominent interaction is a salt-bridge (dotted black line) between the amidine of AZ-002 and carboxyl group of E14 of 14-3-3σ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> AZ-003 Interacts with Both 14-3-3 and TAZ and Induces a Conformational Change in the TAZpS89 Peptide</h3><div class="NLM_p">In contrast to the 14-3-3/p53pT387 complex, the binding site of AZ-003 lies directly at the interface of 14-3-3 with the TAZpS89 peptide (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B). When this fragment was soaked into crystals of the binary complex, we observed a significant change in the conformation of the TAZ peptide (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D). Most obvious is the additional electron density at the C-terminus of the peptide, which allowed us to build three additional amino acids (L96-T98). Second, a peptide flip is observed between L94 and Q95 which allows the main chain carbonyl oxygen of Q95 to establish a polar contact with the amine group of AZ-003 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). Finally, the carbonyl oxygen of L96 is interacting in the same manner with this amine substituent. It is likely that these interactions are the reason for the observed changes in the conformation of the TAZ peptide because the similar structural analogue of AZ-004 does not invoke comparable structural differences (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S3</a>). Complementary protein-based NMR experiments allowed us to confirm the binding site of AZ-003 to 14-3-3σ and to the 14-3-3σ/TAZpS89 complex in solution (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figures S4, S5</a>). Further characterization of the effect of AZ-003 on the binary complex using fluorescence polarization (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S6</a>) revealed an increase in anisotropy in the control experiment (no protein), a sign of interferences which make this assay unsuitable for this particular fragment. Even though a stabilizing activity could not be confirmed by FP for AZ-003, based on the observed conformational change induced by the newly formed interactions with two residues of TAZpS89, we believe that fragment AZ-003 represents a promising chemical starting point that could be developed into a more potent orthosteric stabilizer, for example, by modulating the phenyl ring attached to the thiophene C4 position.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. AZ-003 bound to 14-3-3σ/TAZpS89. (A) Structure of fragment AZ-003. (B) Crystal structure of AZ-003 (yellow sticks) bound to the complex of 14-3-3σ (white surface) and TAZpS89 (green sticks), PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RLH">6RLH</a>. The final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown as blue mesh. (C) Contacts between AZ-003, 14-3-3σ, and TAZpS89. The structure of TAZpS89 bound to 14-3-3σ in the absence of AZ-003 is superimposed as magenta sticks. (D) Details of the changes binding of AZ-003 induces in the conformation of the TAZpS89 peptide. The peptide backbone flip is indicated by an orange arrow, and the distance between the positions of the main-chain carbonyl oxygen is shown. Polar contacts between the amino group of AZ-003 and main-chain carbonyls of TAZpS89 are indicated as dotted, black lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Preliminary Fragment Extension Confers Selectivity toward 14-3-3σ in Complex with p53pT387 over TAZpS89</h3><div class="NLM_p">On the basis of the structural data presented, fragments were designed and custom synthesized (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Scheme S1–S4</a>), leading to the identification of fragments (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) that were compatible with binding to the 14-3-3σ/p53pT387 complex, but not to the 14-3-3σ/TAZpS89 complex. Several structural features of the 14-3-3σ/p53pT387 complex have to be taken into consideration for the prospect of achieving specificity in stabilization. First, the sharp U-turn that the peptide is showing C-terminal of E388 is enabled by the consecutive presence of both a glycine (G389) and a proline (P390). This architecture is further supported by the unique feature that the entire protein ends three amino acids further, with the sequence DSD where both the side chain of the ultimate aspartate and the free C-terminus engage the guanidinium group of R60 in a salt bridge. Together with the rather unusual glutamate at position +1 (E388) after the phosphorylated threonine (pT387), this makes quite a unique interface (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B,E), which might allow for the development of a specific orthosteric 14-3-3σ/p53pT387 PPI stabilizer. With these requirements in mind, synthesized fragments AZ-005, AZ-006, and AZ-007 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) represent useful chemical starting points for structural optimizations toward p53pT387-specific 14-3-3 PPI stabilizers. The crystal structures of these fragments soaked into binary crystals of 14-3-3σ with the p53 peptide (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) showed that the nitrogen-bearing extensions of the core scaffold point toward the p53 peptide and reach into the water network between 14-3-3 and p53 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Because the TAZ peptide occupies the entire length of the 14-3-3 binding channel, these binding modes are expected to be incompatible with the presence of the TAZ peptide (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S7</a>). Consequently, we could not identify additional electron density after soaking these fragments into 14-3-3σ/TAZpS89 crystals, with the exception of AZ-006, which can form a disulfide bond with C38 and is therefore able to compete with TAZpS89 for binding to 14-3-3, resulting in a clear loss of electron density of the TAZ peptide, but of parts of the fragment, too (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S8</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. AZ fragments that reach into the water network between 14-3-3σ and p53pT387. (A) Fragments AZ-005, AZ-006, and AZ-007. (B) Crystal structures of the fragments bound to the complex of 14-3-3σ (white surface) and p53pT387 (orange sticks), PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RX2">6RX2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5L">6R5L</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWH">6RWH</a>, respectively. The final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown as blue mesh. (C) Contacts between the fragments and the water network between 14-3-3σ and p53pT387.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> AZ-008 Binds to and Shows Weak Stabilization of the 14-3-3/p53pT387 Complex</h3><div class="NLM_p">For the goal of developing an orthosteric PPI stabilizer of the 14-3-3/p53pT387 complex, it seems necessary that an extended fragment based on AZ-005–AZ-007 has to reach the p53 peptide and preferentially engage the side chain of E388. These considerations led to the design of AZ-008 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), which in a docking calculation is predicted to contact the carboxyl group of E388 with its terminal amino group (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). We tried to soak this compound into preformed 14-3-3/p53pT387 crystals and also undertook extensive cocrystallization trials, but unfortunately we were not able to detect any additional electron density by X-ray crystallography for compound AZ-008. We thus used WaterLOGSY NMR experiments to detect the potential interaction of AZ-008 to 14-3-3σ. Positive <sup>1</sup>H signals of the ligand in the presence of the protein are indicative of binding. Positive NMR signals in the WaterLOGSY spectra of AZ-008 mixed to 14-3-3σ, both in the presence or absence of the p53pT387 peptide, confirmed the interactions (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S9</a>). The two-dimensional spectrum (<sup>1</sup>H–<sup>15</sup>N TROSY-HSQC) of 14-3-3σ (<sup>15</sup>N<sup>2</sup>H labeled) was next used as reporter to further characterize the binding. Because the resonances in this spectrum are sensitive to the chemical environment of the corresponding residues, a modification in the presence of a ligand of a resonance chemical shift value, and/or intensity, reports an interaction.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> In addition, chemical shift assignments of 14-3-3σ have been performed, allowing to link a resonance in the spectrum to a specific amino acid residue in the 14-3-3σ sequence. The addition of AZ-008 to 14-3-3σ or 14-3-3σ in the presence of the p53pT387 peptide resulted in detectable broadening (intensity decrease) and chemical shift value perturbation (CSP) of a few resonances in the <sup>1</sup>H–<sup>15</sup>N 2D spectrum of the protein (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figures S10–S11</a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). On the basis of the signal assignments, we concluded that addition of AZ-008 affected mainly the intensity of resonances of residues R41, N42, E115, and F119 of 14-3-3σ (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D, SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S11</a>A,B), with no variation in the AZ-008 binding pocket mapping in the presence of the p53pT387 peptide (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A, and SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S10</a>). These experiments indicated that binding of AZ-008 occurs in the same pocket in both cases, with or without the p53pT387 peptide. Visualizing the corresponding amino acid perturbation induced by AZ-008 on the 14-3-3σ structure clearly highlighted a binding site (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C), which can be matched to data from the cocrystal structure of 14-3-3σ and p53pT387 in the presence of its analogues AZ-001/002 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The binding pose resulting from docking AZ-008 to the 14-3-3σ/p53pT387 complex (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) is in accordance with both the NMR data and the bound-conformation adopted by other AZ fragments based on this scaffold.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. AZ-008 binds to the 14-3-3σ/p53pT387 complex. (A) Structure of AZ-008. (B) Docking of AZ-008 (yellow sticks) bound to the 14-3-3σ (white sticks) in complex with p53pT387 (orange sticks). (C) Mapping on the crystal structure of 14-3-3σ (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YZ5">1YZ5</a>) of the amino acid residues corresponding to the most affected resonances by the presence of AZ-008. Residues whose corresponding correlation peak intensities were the most affected are colored in red, and an additional 5–8 are colored in brown. (D) Plot of the intensity ratios <i>I</i>/<i>I</i><sub>0</sub> of corresponding <sup>1</sup>H–<sup>15</sup>N correlation peaks in the 2D 14-3-3σ spectra corresponding to the bound 14-3-3σ (100 μM 14-3-3σ in the presence of 2000 μM AZ-008, <i>I</i>) and free 14-3-3σ (100 μM 14-3-3σ <i>I</i><sub>0</sub>) protein (<i>y</i> axis) versus 14-3-3σ amino acid sequence (<i>x</i> axis). A total of 133 correlation peak intensities are shown. The <i>x</i> axis is not proportional to the sequence length. The helices of 14-3-3σ are identified below the <i>x</i> axis as blue cartoons, while disordered regions are represented by green lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. AZ-008 stabilizes the 14-3-3σ/p53pT387 complex. (A) Plot of the intensity ratios <i>I</i>/<i>I</i><sub>0</sub> (<i>y</i>-axis) of corresponding <sup>1</sup>H–<sup>15</sup>N correlation peaks in the 2D 14-3-3σΔC17 spectra of, in red: the peptide-bound 14-3-3σΔC17 (100 μM 14-3-3σ in the presence of 500 μM p53pT387, <i>I</i>) and free 14-3-3σ (100 μM 14-3-3σ <i>I</i><sub>0</sub>); in blue: corresponding to the peptide and fragment-bound 14-3-3σ (100 μM 14-3-3σ with 500 μM p53pT387 and 2000 μM AZ-008, <i>I</i>) and free 14-3-3σ (100 μM 14-3-3σ, <i>I</i><sub>0</sub>) protein. Note that the boxed regions correspond to the same areas affected in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> D and therefore show that the binding site of the fragment is the same when the peptide is present or not. (B) <sup>1</sup>H NMR signals of the spectrum of 100 μM 14-3-3σ (in black), 100 μM 14-3-3σ in the presence of 500 μM p53pT387 (in red), and 100 μM 14-3-3σ in the simultaneous presence of 500 μM p53pT387 and 2000 μM AZ-008 (in blue). The NMR signals correspond to the Hδ protons of L383 of the p53pT387 peptide. CSP in the peptide signals can be observed when AZ-008 is added to the solution containing the 14-3-3σ/p53pT387 complex. (C,D) Plot of the ratios of the bound (<i>I</i>)/free (<i>I</i><sub>0</sub>) <sup>1</sup>H–<sup>15</sup>N TROSY-HSQC correlation peak intensities of individual resonances corresponding to residues of 100 μM 14-3-3σ, remote from the interaction site (C) or close to the p53pT387 interaction site of (D), in the presence of 500 μM p53pT387 (in red) and 100 μM 14-3-3σ in the simultaneous presence of 500 μM p53pT387 and 2000 μM AZ-008 (in blue). Error bars of <i>I</i>/<i>I</i><sub>0</sub> ratios were calculated based on the maximum noise estimate of each spectrum. Note that the resonances corresponding to regions affected by the p53pT387 peptide binding were considerably more affected in the presence of AZ-008. (E) Titration of 14-3-3σ to a fluorescently labeled p53pT387 peptide in the absence and presence of 1 mM of AZ-008. Shown are average and standard deviation of three independent experiments. (F) SPR measurement of 14-3-3σ binding to immobilized p53pT387 peptide in the absence and presence of 1 mM AZ-008. Shown are average and standard deviation of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Interestingly, addition of AZ-008 also induced chemical shift perturbations of the <sup>1</sup>H NMR signals of the bound peptide (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S12</a>). In particular, the <sup>1</sup>H signals (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) of the Hδ methyl protons of L383 of the p53pT387 peptide are affected by AZ-008 binding despite the fact that they are not close to the fragment binding site. Perturbation of these signals thus did not correspond to a direct effect due to the local change of environment. This data rather suggests that the binding of the fragment impacted the global binding of the peptide to 14-3-3σ. Moreover, the extent of decrease of the intensity of specific resonances in the 2D spectrum of 14-3-3σ in the presence of AZ-008 reports on the amount of complex present in solution. The intensity of resonances corresponding to amino acid residues located in, or close to, the p53pT387 binding site specifically decreased with the addition of AZ-008 to a solution containing the 14-3-3σ/p53pT387 complex (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D) and, reversely, the extent of CSP increased (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S11C</a>). Combined with the results of other methods, this observation indicates that the amount of 14-3-3σ/p53pT387 complex present in solution was increased when both ligands were present, suggesting that the complex was therefore stabilized in the presence of AZ-008. Further experiments using fluorescence polarization (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E, SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S13</a>) and surface plasmon resonance (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F, SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S14</a>) also suggested that AZ-008 is able to stabilize the interaction between 14-3-3σ and a p53pT387 peptide (32mer) containing the complete C-terminal domain. The enthalpically favorable contact between the amino substituent of AZ-008 and E388 of p53pT387 observed in the docking complex described above helps to explain the observed effect. Although this effect is objectively small, it appears to be tangible as it was confirmed quantitatively by two separate methods, but also qualitatively by NMR. Moreover, the 2-fold effect observed here is in accordance with the change in activity that small, noncovalent molecular fragments are expected to induce. Most importantly, these results hint at the possibility of transforming inactive fragments initially identified in X-ray crystallography and/or NMR studies into active molecules through rational design of structural extensions.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Targeted stabilization of PPIs is an underexplored concept in drug discovery despite numerous examples from natural products and synthetic molecules advocating for this principal strategy.<a onclick="showRef(event, 'ref2 ref3 ref30'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref30">(2,3,30)</a> Successful drugs that operate by stabilizing PPIs have almost exclusively only been recognized as PPI stabilizers after their therapeutic relevance was shown and in many cases even long after FDA approval. One famous example of such drugs is thalidomide and its derivatives, for example, lenalidomide,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> which are collectively called IMiDs (immunomodulatory drugs). Lenalidomide acts as a “molecular glue” that hijacks a ubiquitin ligase for the targeted degradation of a neosubstrate protein.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Another example is the history of the immunosuppressants rapamycin and FK506 which puzzled the community for a long time, how these natural products actually convey their therapeutic effect. Eventually it was shown that although they initially bind to the same primary receptor protein, FKBP12, their actual functional targets are the phosphatase calcineurin (FK506) and the protein kinase mTOR (rapamycin) that were “glued” to FKBP12 by the natural products and thus inactivated.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a></div><div class="NLM_p">A third, recent success story of the PPI induction and stabilization concept is proteolysis targeting chimeras (PROTACs), a technology employing heterobifunctional small molecules that facilitate the interaction of a target protein with a ubiquitin E3 ligase.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> This approach has not only attracted a lot of attention in the chemical biology field but is currently explored by many pharma companies and at least half a dozen start-up companies. Together with the aforementioned, rather serendipity-driven success examples of IMiDs and “molecular glue” natural products, PROTACs may define a new conceptional era of pharmacologic intervention where an enhancement or induction of a native regulatory process (targeted protein degradation) instead of a direct protein inhibition is the underlying principle.</div><div class="NLM_p">Our approach to enhance the regulatory function of 14-3-3 PPIs with different partners like p53 or TAZ is an innovative extension of this concept with the added advantage that a physiological, already implemented regulatory network is addressed with small molecules and also that function-enhancing PPIs, as in the case of 14-3-3/p53, can be promoted. We and others have shown in the past that especially the natural product class of the fusicoccanes can be employed as orthosteric stabilizers of 14-3-3 PPIs.<a onclick="showRef(event, 'ref7 ref11 ref21 ref22 ref26 ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref7 ref11 ref21 ref22 ref26 ref36 ref37 ref38 ref39">(7,11,21,22,26,36−39)</a> However, with only supramolecular ligands<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> that are hard to trace by medicinal chemistry, and two screening hits of 14-3-3 PPIs,<a onclick="showRef(event, 'ref27 ref42'); return false;" href="javascript:void(0);" class="ref ref27 ref42">(27,42)</a> there is a clear need to expand the chemical diversity of 14-3-3 PPI stabilizers. A very useful approach to achieve this goal is fragment-based drug discovery (FBDD).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> We have shown that fragments binding to 14-3-3 protein complexes can be identified<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> and in particular covalent fragment discovery is a fruitful approach,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> as can be seen in the context of the current work, too (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figure S15</a>). Especially in the new field of targeting rim-of-the interface pockets with orthosteric stabilizers, FBDD is extremely valuable because it circumvents the bias of traditional compound collections used for high throughput-screening and allows to efficiently probe the chemical space to identify starting points for the development of PPI stabilizers. However, one potential drawback of FBDD in regards to the discovery of PPI stabilizers might be that most identified fragments are silent in classical biophysical assays used for high-throughput screening, as perfectly exemplified within this study. X-ray crystallography and NMR studies therefore appear as the methods of choice, to first find fragment scaffolds capable of binding to the target protein(s), and then develop their activities, a goal that has not yet been successfully accomplished and that represents a considerable challenge.</div><div class="NLM_p last">As our example has shown, specificity for two different, but closely related PPI interfaces can be aimed for even at the very early stage of extending an initial fragment hit. Through a relatively limited series of protein X-ray crystallography identified fragments (see also SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Figures S16–S34</a>) we were able to identify promising chemical starting points for the future development of active stabilizers that discriminate between the 14-3-3 complexes with p53pT387 and TAZpS89. This is important for the prospect to make the widespread 14-3-3 interactome accessible for drug discovery but also bears implications for other PPIs. An analysis by the Skolnick group has shown that in the crystal structures of 1,611 protein–protein complexes, the majority of “ligandable” pockets are located with a 6 Å distance from the protein interfaces,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> which is well exemplified by the fragment-binding site of our study. This area, possibly privileged for small-molecule accommodation, is the region where orthosteric PPI stabilizers are supposed to bind. Together with the overall high structural diversity of PPI interfaces this observation nourishes the prospect that for many regulatory protein complexes a specific, rim-of-the interface targeting, orthosteric stabilizer might be developed.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54373" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54373" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Peptide Synthesis</h3><div class="NLM_p last">FITC labeled and acetylated 13mer TAZpS89 were synthesized via Fmoc solid-phase peptide synthesis (SPPS) using an automated Intavis MultiPepRSi peptide synthesizer and a rink amide AM resin (Novabiochem; 0.59 mmol/g loading). The peptides used for NMR measurements and for crystallization studies were acetylated at the N terminus (1:1:3 Ac2O/pyridine/NMP) before resin cleavage. The peptides used for FP assays were manually labeled with FITC (Sigma-Aldrich) attached to a β-alanine linker introduced via Fmoc-β-Ala-OH (Iris Biotech) under standard SPPS conditions. All peptides were purified by preparative reversed-phase HPLC with MS detection. TAMRA labeled 32-mer p53 peptide and acetylated 12-mer p53 were commercially synthesized by Anaspec and Caslo, respectively.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> 14-3-3 Expression</h3><div class="NLM_p last">His<sub>6</sub>-tagged 14-3-3σ full-length and ΔC (truncated after T231; for crystallography) were expressed in BL21(DE3) competent cells with a pPROEX HTb plasmid and purified with a nickel column. The His<sub>6</sub>-tag was cleaved off using TEV protease, and a second purification was performed by size-exclusion chromatography (Superdex75). The proteins were dialyzed against FP or crystallization buffers before use.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Crystallography</h3><div class="NLM_p">14-3-3σΔC/TAZpS89 crystals: 14-3-3σΔC and acetylated TAZpS89 peptide, both in crystallization buffer (25 mM HEPES pH 7.5, 100 mM NaCl, 10 mM MgCl<sub>2</sub> and 2 mM β-mercaptoethanol), were combined in a 1:2 14-3-3σΔc/TAZpS89 molar ratio with a final protein concentration of 12 mg/mL. After overnight complexation, hanging-drop crystallization wells were setup using a 1:1 ratio of precipitation buffer (95 mM HEPES pH 7.1, 0.19 M CaCl<sub>2</sub>, 5% glycerol, and 27% PEG400) and complexation mixture for a total drop volume of 2 μL. Crystals were grown within 10 days at 4 °C.</div><div class="NLM_p">14-3-3σΔC/p53pT387 crystals: 14-3-3σΔc and acetylated p53pT387 peptide, both in crystallization buffer (25 mM HEPES pH 7.5, 100 mM NaCl, 10 mM MgCl<sub>2</sub>, and 2 mM β-mercaptoethanol), were combined in a 1:2 14-3-3σΔc/TAZ molar ratio with a final protein concentration of 12 mg/mL. After overnight complexation, hanging-drop crystallization wells were setup using a 1:1 ratio of precipitation buffer (0.1 M HEPES pH 7.5, 0.2 M CaCl<sub>2</sub>, 5% glycerol, 2 mM β-mercaptoethanol, and 28% PEG400) and complexation mixture for a total drop volume of 2 μL. Crystals were grown within 10 days at 4 °C.</div><div class="NLM_p">The compounds screened were part of the X-ray crystallography subset of the AstraZeneca fragments collection. Fragment soaking was performed on crystals of 10 days and older by adding 0.2 μL of a 100 mM stock solution in dimethyl sulfoxide to the 2 μL drop for a final concentration of 20 mM. Crystals were fished after a minimum of 2 days and flash-cooled in liquid nitrogen.</div><div class="NLM_p last">Data sets were collected either on an in-house Rigaku Micromax-003 (Rigaku Europe, Kemsing Sevenoaks, UK) sealed tube X-ray source and a Dectris Pilatus 200 K detector (DECTRIS Ltd., Baden-Daettwil, Switzerland) or at the DESY PETRA III synchrotron beamline P11 (DESY, Hamburg, Germany). All data sets were indexed, integrated, and scaled using either iMosflm or DIALS followed by AIMLESS. Phasing was performed using molecular replacement in Molrep using PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N75">5N75</a> (14-3-3σ/TAZ) or <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MHC">5MHC</a> (14-3-3σ/p53) as search model. Phaser, Coot, and phenix.refine were thereafter used in alternating cycles of model building and refinement.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Production of <sup>15</sup>N<sup>2</sup>H Labeled 14-3-3σΔC for NMR Spectroscopy</h3><div class="NLM_p last">The <sup>15</sup>N<sup>2</sup>H labeled 14-3-3σ (ΔC, cleaved after T231) for NMR studies was expressed in <i>Echerichia coli</i> BL21 (DE3) cells transformed with a pProExHtb vector carrying the cDNA to expres,s an N-terminally His<sub>6</sub>-tagged human 14-3-3σ. Briefly, the cells were grown in 1 L of deuterated M9 minimal medium supplemented with 2 g/L <sup>12</sup>C<sub>6</sub><sup>2</sup>H<sub>7</sub> glucose, 1 g/L <sup>15</sup>N ammonium chloride, 0.4 g/L Isogro <sup>15</sup>N<sup>12</sup>C<sup>2</sup>H Powder Growth Medium (Sigma-Aldrich), and 100 μg/mL ampicillin and the recombinant protein was then purified by affinity chromatography using a Ni-NTA column (GE Healthcare). The His<sub>6</sub>-tag was further cleaved by the TEV protease and the protein was further dialyzed overnight at 4 °C against NMR buffer (100 mM sodium phosphate, pH 6.8, 50 mM NaCl), concentrated, aliquoted, flash frozen, and stored at −80 °C. A detailed protocol can be found at.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> <sup>15</sup>N–<sup>1</sup>H TROSY-HSQC NMR Spectroscopy</h3><div class="NLM_p last"><sup>1</sup>H–<sup>15</sup>N transverse relaxation optimized spectroscopy-heteronuclear single quantum coherence spectroscopy (TROSY-HSQC) spectra were acquired in 3 mm tubes (sample volume 200 μL) using a 900 MHz Bruker Avance Neo spectrometer, equipped with a cryoprobe. The spectra were recorded at 32 °C, in a buffer containing 100 mM sodium phosphate, 50 mM NaCl, pH 6.8, 4% (v/v) DMSO-<i>d</i><sub>6</sub>, 1 mM DTT, EDTA-free protease inhibitor cocktail (Roche, Basel, Switzerland), and 10% (v/v) D<sub>2</sub>O. The experiments were recorded with 3072 complex data points in the direct dimension and 128 complex data points in the indirect dimension, with 184 scans per increment. For the evaluation of the binding of AZ-008 to the 14-3-3σ/p53 complex, spectra of <sup>15</sup>N<sup>2</sup>H labeled 14-3-3σΔC 100 μM were recorded in the presence and absence of 2000 μM AZ-008, 500 μM p53pT387 peptide, and simultaneously 500 μM p53pT387 peptide and 2000 μM AZ-008. For the evaluation of the binding of AZ-003 to the 14-3-3σ/TAZ complex, spectra of <sup>15</sup>N<sup>2</sup>H labeled 14-3-3σ 100 μM were recorded in the presence and absence of 2000 μM AZ-003, 125 μM TAZpS89 peptide, and simultaneously 125 μM TAZpS89 peptide and 2000 μM AZ-003. Assignments of the backbone resonances of <sup>15</sup>N<sup>2</sup>H labeled 14-3-3σΔC were previously reported.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> A <sup>1</sup>H spectrum with water-suppression was additionally recorded for each sample for monitoring the peptide signals. The reference for the <sup>1</sup>H chemical shift was relative to 4,4-dimethyl-4-silapentane-1-sulfonic acid (DSS) while <sup>15</sup>N chemical shift values were referenced indirectly. Spectra were collected and processed with Topspin 4.0 (Bruker Biospin, Karlsruhe, Germany) and analyzed with Sparky 3.12 (T. D. Goddard and D. G. Kneller, SPARKY 3, University of California, San Francisco). CSPs verified as chemical shift positions (in ppm) on the <sup>1</sup>H–<sup>15</sup>N TROSY-HSQC were calculated using the following equation:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_m001.gif" alt="" id="_i15" /></img></span></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> WaterLOGSY NMR Experiments</h3><div class="NLM_p last">WaterLOGSY spectra were acquired in 5 mm tubes (sample volume 530 μL) using a 600 MHz Bruker Avance I spectrometer equipped with a CPQCI cryogenic probe. The spectra were recorded with 32 768 complex data points, with 1024 scans per increment and with a mixing time of 1.7 s (acquisition time of 94 min). The spectra were acquired at 16 °C in a buffer containing 100 mM sodium phosphate, 50 mM NaCl, pH 6.8, and 10% (v/v) D<sub>2</sub>O. The final concentration of DMSO-<i>d</i><sub>6</sub> was 2% (v/v) and was kept constant for all the experiments. For the evaluation of the binding of AZ-008 to the 14-3-3σ/p53 complex, WaterLOGSY spectra were recorded on solutions containing 14-3-3σ 25 μM in the presence and absence of 500 μM AZ-008, 50 μM p53pT387 peptide, and simultaneously 50 μM p53pT387 peptide and 500 μM AZ-008. An additional control experiment consisted of the acquisition of a waterLOGSY spectrum in the presence of 500 μM AZ-008 and in the absence of protein and peptide. A <sup>1</sup>H spectrum with water-suppression was additionally recorded for each sample. Spectra were collected, processed and analyzed with Topspin 4.0 (Bruker Biospin, Karlsruhe, Germany).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> In Silico Docking of AZ-008 in 14-3-3/p53 Complex</h3><div class="NLM_p last">Docking experiments of fragments libraries were performed using Schrödinger Drug Discovery suite (Schrödinger Release 2019-1: Schrödinger, LLC, New York, NY, 2019). The Schrödinger protein preparation wizard was used to prepare the crystal structure. Hydrogen atoms were added and possible metal binding states generated. The protonation and tautomeric states of Asp, Glu, Arg, Lys, and His were adjusted to match a pH of 7.5 and possible orientations of Asn and Gln residues were generated. Hydrogen bond sampling with adjustment of active site water molecule orientations was performed using PROPKA. Water molecules with fewer than two hydrogen bonds to nonwaters were deleted. Finally, the protein–ligand complexes were subjected to geometry refinements using the OPLS3 force field in restrained minimizations. The ligands were prepared using LigPrep in the Schrödinger suite. The OPLS3 force field was used in all ligand preparation steps. Possible protonation and ionization states were assigned to each ligand using EpiK at pH 7.5. Possible stereoisomers, tautomeric states, and metal binding sites were generated.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Surface Plasmon Resonance (SPR)</h3><div class="NLM_p last">SPR analysis was performed using a Biacore 3000 instrument. CTD Thr387 phosphorylated 32-mer peptide of p53 (AnaSpec) was immobilized onto a CMD200 M sensor chip (Xantec Bioanalytics) by using standard NHS/EDC amine-coupling chemistry. Peptide immobilization was carried out at a flow rate of 20 μL/min and protein titration experiments at 10 μL/min. All measurements were conducted at 25 °C in HBSP+ buffer (0.01 M HEPES pH 7.4, 0.15 M NaCl, 0.05% Surfactant P20, 1% DMSO). 14-3-3σ was injected at indicated concentrations in the presence of 1 mM AZ-008 or 1% DMSO control. KD values were obtained by first fitting the non-normalized data using the “log(agonist) vs response (three parameters)” equation of GraphPad Prism, then taking the fitted upper plateau for data normalization, subsequently using the “Sigmoidal dose-response (variable slope)” equation with a constrain to 100% activity.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Fluorescence Polarization (FP) Assay</h3><div class="NLM_p last">Fluorescence polarization measurements were conducted in FP-buffer (10 mM HEPES pH 7.4, 150 mM NaCl, 0.01% TWEEN-20, 1.0 mg/mL BSA). <i>Protein titration</i>s: dilution series of 14-3-3σ were made in triplet in low binding Corning black round bottom 384-well plates in FP-buffer containing a fixed concentration of labeled 32mer peptide (10 nM TAMRA-p53) and 1% DMSO or 1 mM AZ-008. Plate was incubated at room temperature protected from light for 1 h and polarization was measured using PHERAstar (BMG Labtech) microplate reader (λ<sub>ex</sub>, 535 nm; λ<sub>em</sub>, 590 nm). KD values were obtained by first fitting the non-normalized data using the “log(agonist) vs response (three parameters)” equation of GraphPad Prism, then taking the fitted upper plateau for data normalization, subsequently using the “Sigmoidal dose-response (variable slope)” equation with a constrain to 100% activity. <i>Compound titrations</i>: dilution series of the fragments dissolved in DMSO were made with fixed concentration of labeled peptides (FITC-TAZpS89:10 nM; TAMRA-p53pT387:10 nM) and protein (40 nM and 10 μM 14-3-3σ for TAZ and p53, respectively) (Control: DMSO concentration 1% v/v). The Corning 384-well plates (black, low volume, low binding, round-bottom) were incubated at 4C protected from light overnight and polarization was measured using Tecan Infinite F500 plate reader (Thermo Fisher) using appropriate excitation and emission wavelength for FITC (λ<sub>ex</sub>, 485 nm; λ<sub>em</sub>, 535 nm) and TAMRA (λ<sub>ex</sub>, 535 nm; λ<sub>em</sub>, 590 nm)</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Synthesis and Characterization of Fragments</h3><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> General Chemistry Statement</h4><div class="NLM_p last">All solvents and chemicals used were of reagent grade. Purity and characterization of compounds were established by a combination of LC–MS and NMR techniques. LC-MS spectra were obtained on a Shimadzu UFLC fitted with a Shimadzu LCMS-2020 mass spectrometer and a Waters BEH C18 (50 mm × 2.1 mm, 1.7 μm) or Shim-pack XR-ODS (50 mm × 3.0 mm, 2.2 μm) or Phenomenex Gemini-NX 3μ C18 110A (50 mm × 3.0 mm, 3 μm) column at a flow rate of 1.2 mL/min 95% A to 95% B over 2.0 min with a 0.6 min hold, where A = 0.1% formic acid or 0.05% TFA in water and B = acetonitrile. Purity was determined by UV absorption at a wavelength of 254 nm, and the mass ion was determined by electrospray ionization (ESI, Micromass Instrument). All final compounds were found to be of 95% purity or greater, as assessed by LC–MS and <sup>1</sup>H NMR. <sup>1</sup>H NMR spectra were recorded on a 300 or 400 MHz using Bruker spectrometers and using DMSO-<i>d</i><sub>6</sub>, CDCl<sub>3</sub>, or MeOD-<i>d</i><sub>4</sub> as solvent depending on solubility. Data for <sup>1</sup>H NMR were reported as chemical shifts (in parts per million from internal standard tetramethylsilane), multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, h = heptet, m = multiplet, br = broad) and integration. Preparative high-performance liquid chromatography (HPLC) was performed on a Waters or Phenomenex column using decreasingly polar mixtures of water (containing 1% formic acid or 1% aqueous NH4OH) and acetonitrile. Purification by flash column chromatography (FCC) was typically performed using silica gel (Merck 7734 grade), and solvent mixtures and gradients are recorded herein. All reactions were performed under nitrogen at room temperature (rt) unless otherwise stated.</div></div><div id="sec4_10_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Synthesis Pathway to 7-[(2-Amino-1-methyl-ethyl)amino]benzothiophene-2-carboxamidine (AZ-005) (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Scheme S1</a>)</h4><div id="sec4_10_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Step a: Methyl 7-[[2-(<i>tert</i>-Butoxycarbonylamino)-1-methyl-ethyl]amino]benzothiophene-2-carboxylate (<b>1</b>)</h5><div class="NLM_p last">Xantphos (341 mg, 0.59 mmol) and Pd<sub>2</sub>(dba)<sub>3</sub> (270 mg, 0.30 mmol) were added to Cs<sub>2</sub>CO<sub>3</sub> (2884 mg, 8.85 mmol), methyl 7-bromobenzo[<i>b</i>]thiophene-2-carboxylate (800 mg, 2.95 mmol), and <i>tert</i>-butyl (2-aminopropyl)carbamate (617 mg, 3.54 mmol) in toluene (20 mL) at 20 °C under nitrogen. The resulting solution was stirred at 130 °C for 4 h. The crude product was purified by flash silica chromatography, elution gradient 0–10% EtOAc in petroleum ether. Pure fractions were evaporated to dryness to afford the title compound (1000 mg, 93%) as a gum. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.08–1.23 (d, <i>J</i> = 6.3 Hz, 3H), 1.40 (s, 9H), 2.92–3.08 (m, 1H), 3.16–3.31 (m, 1H), 3.62–3.79 (m, 1H), 3.89 (s, 3H), 5.38–5.50 (m, 1H), 6.73–6.84 (m, 1H), 7.03–7.13 (m, 1H), 7.27–7.31 (m, 2H), 8.10 (s, 1H). MS (ESI) [M + H]<sup>+</sup> = 365.</div></div><div id="sec4_10_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Step b: Methyl 7-[(2-Amino-1-methyl-ethyl)amino]benzothiophene-2-carboxylate (<b>2</b>)</h5><div class="NLM_p last">TFA (0.106 mL, 1.37 mmol) was added to methyl 7-((1-((tert-butoxycarbonyl)amino)propan-2-yl)amino)benzo[<i>b</i>]thiophene-2-carboxylate (500 mg, 1.37 mmol) in DCM (20 mL) at 20 °C. The resulting solution was stirred at rt for 1 h. The crude product was purified by flash silica chromatography, elution gradient 0–10% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (360 mg, 99%) as a gum. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.20–1.26 (d, <i>J</i> = 6.4 Hz, 3H), 2.94–3.11 (m, 2H), 3.90 (s, 3H), 3.92–4.01 (m, 1H), 5.35–5.78 (br s, 1H), 6.75–6.81 (d, <i>J</i> = 6.5 Hz, 1H), 7.28–7.40 (m, 2H), 7.87 (s, 2H), 8.13 (s, 1H). MS (ESI) [M + H]<sup>+</sup> = 265.</div></div><div id="sec4_10_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> Step c: 7-[(2-Amino-1-methyl-ethyl)amino]benzothiophene-2-carboxamidine (<b>AZ-005</b>)</h5><div class="NLM_p last">Methyl 7-((1-aminopropan-2-yl)amino)benzo[<i>b</i>]thiophene-2-carboxylate (300 mg, 1.13 mmol) was added to ammonium chloride (243 mg, 4.54 mmol) and trimethylaluminum (327 mg, 4.54 mmol) in toluene (50 mL) at 20 °C under nitrogen. The resulting solution was stirred at 110 °C for 4 h. The crude product was purified by preparative column: XBridge Prep OBD C18 column 19 mm × 250 mm, 5 μm; mobile phase A, water (0.1%FA); mobile phase B, MeCN; flow rate, 20 mL/min; gradient, 2% B to 30% B. Fractions containing the desired compound were evaporated to dryness to afford the title compound (30.0 mg, 7.72%) as a solid. 1H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 1.29–1.46 (d, <i>J</i> = 6.4 Hz, 3H), 3.10–3.24 (m, 2H), 4.03–4.17 (m, 1H), 6.89–7.00 (dd, <i>J</i> = 6.7, 1.9 Hz, 1H), 7.39–7.49 (m, 2H), 8.20 (s, 1H), 8.57 (s, 2H). MS (ESI) [M + H]<sup>+</sup> = 249.</div></div></div><div id="sec4_10_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Synthesis Pathway to 4-[(2-Amino-1-methyl-ethyl)amino]-6-(sulfanylmethyl)benzothiophene-2-carboxamidine (<b>AZ-006</b>) (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Scheme S2</a>)</h4><div id="sec4_10_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Step a: Methyl 4-Bromo-6-methyl-benzothiophene-2-carboxylate</h5><div class="NLM_p last">Methyl 2-mercaptoacetate (2.69 g, 25.34 mmol) was added to 2-bromo-6-fluoro-4-methylbenzaldehyde (5 g, 23.04 mmol) and K<sub>2</sub>CO<sub>3</sub> (6.37 g, 46.08 mmol) in DMF (20 mL) at 25 °C. The resulting suspension was stirred at 100 °C for 2 h. The reaction mixture was diluted with EtOAc (250 mL) and washed sequentially with saturated brine (200 mL × 3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford the title compound (5.0 g, 76%) as a solid. Taken through to the next step without further purification.</div></div><div id="sec4_10_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Step b: 4-Bromo-6-methyl-benzothiophene-2-carboxamide</h5><div class="NLM_p last">A solution of methyl 4-bromo-6-methylbenzo[<i>b</i>]thiophene-2-carboxylate (6.5 g, 22.79 mmol) in NH<sub>3</sub> in MeOH (7 M) (75 mL) was stirred at 110 °C overnight. The mixture was concentrated to obtain the crude product, which was used in next step directly.</div></div><div id="sec4_10_1_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Step c: 4-Bromo-6-methyl-benzothiophene-2-carbonitrile (<b>3</b>)</h5><div class="NLM_p last">TFA (10.98 mL, 77.74 mmol) was added to a solution of 4-bromo-6-methylbenzo[<i>b</i>]thiophene-2-carboxamide (7 g, 25.91 mmol) and pyridine (10.48 mL, 129.56 mmol) in DCM (30 mL). The mixture was stirred at 25 °C for 0.5 h. The reaction mixture was diluted with DCM (200 mL) and washed sequentially with 0.1 M HCl (100 mL × 2). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford the title compound (6 g) as a solid. Taken through to the next step without further purification.</div></div><div id="sec4_10_1_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Step d: 4-Bromo-6-(bromomethyl)benzothiophene-2-carbonitrile</h5><div class="NLM_p last">NBS (1.853 g, 10.41 mmol) was added in one portion to 4-bromo-6-methylbenzo[<i>b</i>]thiophene-2-carbonitrile (2.5 g, 9.92 mmol) and BPO (0.240 g, 0.99 mmol) in CCl<sub>4</sub> (150 mL) at 90 °C. The resulting solution was stirred at 90 °C for 2 h. The reaction mixture was diluted with EtOAc (200 mL) and washed sequentially with saturated brine (150 mL × 2). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0–2% EtOAc in petroleum ether. Pure fractions were evaporated to dryness to afford the title compound (2.70 g, 82%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 4.89 (s, 2H), 7.91 (s, 1H), 8.31 (s, 1H), 8.39 (s, 1H).</div></div><div id="sec4_10_1_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Step e: (4-Bromo-2-cyano-benzothiophen-6-yl)methyl Acetate (<b>4</b>)</h5><div class="NLM_p last">Silver acetate (1.639 g, 9.82 mmol) was added to 4-bromo-6-(bromomethyl)benzo[<i>b</i>]thiophene-2-carbonitrile (2.5 g, 7.55 mmol) in DMF (12 mL) at 20 °C under nitrogen. The resulting mixture was stirred at 60 °C for 1 h. The reaction mixture was filtered through Celite. The crude product was purified by flash C18-chromatography, elution gradient 40–50% MeCN in water. Pure fractions were evaporated to dryness to afford the title compound (2.100 g, 90%) as a solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.13 (s, 3H), 5.23 (s, 2H), 7.82 (s, 1H), 8.22 (s, 1H), 8.38 (s, 1H).</div></div><div id="sec4_10_1_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Step f: [4-[[2-(<i>tert</i>-Butoxycarbonylamino)-1-methyl-ethyl]amino]-2-cyano-benzothiophen-6-yl]methyl Acetate (<b>5</b>)</h5><div class="NLM_p last">Pd<sub>2</sub>(dba)<sub>3</sub> (0.768 g, 0.84 mmol) was added to (4-bromo-2-cyanobenzo[<i>b</i>]thiophen-6-yl)methyl acetate (2.6 g, 8.38 mmol), <i>tert</i>-butyl (2-aminopropyl)carbamate (1.753 g, 10.06 mmol), Cs<sub>2</sub>CO<sub>3</sub> (3.43g,10.52 mmol), and Xantphos (0.970 g, 1.68 mmol) in toluene (25 mL) at 25 °C under nitrogen. The resulting solution was stirred at 130 °C for 3 h. The reaction mixture was filtered using a Buchner funnel, and the solvent was removed under reduced pressure. The crude product was purified by flash silica chromatography, elution gradient 1–10% EtOAc in petroleum ether. Pure fractions were evaporated to dryness to afford the title compound (1.700 g, 50.3%) as a solid. MS (ESI) [M + H]<sup>+</sup> = 404.</div></div><div id="sec4_10_1_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Step g: <i>tert</i>-Butyl <i>N</i>-[2-[[2-(<i>N</i>-Hydroxycarbamimidoyl)-6-(hydroxymethyl)benzothiophene-4-yl]amino]propyl]carbamate</h5><div class="NLM_p last">Hydroxylamine hydrochloride (0.861 g, 12.39 mmol) was added to (4-((1-((<i>tert</i>-butoxycarbonyl)amino)propan-2-yl)amino)-2-cyanobenzo[<i>b</i>]thiophen-6-yl)methyl acetate (1 g, 2.48 mmol), K<sub>2</sub>CO<sub>3</sub> (2.055 g, 14.87 mmol) in MeOH (20 mL) at 25 °C under nitrogen. The resulting solution was stirred at 50 °C for 1 h. The reaction mixture was filtered through Celite. The crude product was purified by flash silica chromatography, elution gradient 0–20% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (0.600 g, 61.4%) as a solid. MS (ESI) [M + H]<sup>+</sup> = 395.</div></div><div id="sec4_10_1_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Step h: <i>tert</i>-Butyl <i>N</i>-[2-[[2-Carbamimidoyl-6-(hydroxymethyl)benzothiophen-4-yl]amino]propyl]carbamate</h5><div class="NLM_p last">Nickel (0.2 g, 3.41 mmol) was added to <i>tert</i>-butyl (2-((2-(<i>N</i>-hydroxycarbamimidoyl)-6-(hydroxymethyl)benzo[<i>b</i>]thiophen-4-yl)amino)propyl)carbamate (0.6 g, 1.52 mmol) and acetic acid (0.5 mL, 8.73 mmol) in MeOH (15 mL) at 25 °C under nitrogen. The resulting solution was stirred at 40 °C for 1 h under an atmosphere of hydrogen. The mixture was filtered through a Celite pad, and the solvent was removed under reduced pressure. The reaction mixture was adjusted to pH 8. The crude product was purified by flash silica chromatography, elution gradient 0–20% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (0.450 g, 78%) as a solid. MS (ESI) [M + H]<sup>+</sup> = 379.</div></div><div id="sec4_10_1_2_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Step i: <i>tert</i>-Butyl <i>N</i>-[4-[[2-(<i>tert</i>-Butoxycarbonylamino)-1-methyl-ethyl]amino]-6-(hydroxymethyl)benzothiophene-2-carboximidoyl]carbamate (<b>6</b>)</h5><div class="NLM_p last">Boc-anhydride (0.920 mL, 3.96 mmol) was added slowly to <i>tert</i>-butyl (2-((2-carbamimidoyl-6-(hydroxymethyl)benzo[<i>b</i>]thiophen-4-yl)amino)propyl)carbamate (0.4 g, 1.06 mmol), DMAP (0.129 g, 1.06 mmol) and triethylamine (1.841 mL, 13.21 mmol) in DMF (8 mL) at 0 °C under nitrogen. The resulting solution was stirred at 20 °C for 4 h. The reaction mixture was diluted with EtOAc (100 mL) and washed sequentially with saturated brine (20 mL × 3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0–10% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (0.450 g, 35.6%) as a solid. MS (ESI) [M + H]<sup>+</sup> = 479.</div></div><div id="sec4_10_1_2_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Step j: [4-[[2-(<i>tert</i>-Butoxycarbonylamino)-1-methyl-ethyl]amino]-2-(<i>N</i>-<i>tert</i>-butoxycarbonylcarbamimidoyl)benzothiophen-6-yl]methylmethanesulfonate</h5><div class="NLM_p last">Ms-Cl (0.078 mL, 1.00 mmol) was added dropwise to <i>tert</i>-butyl (2-((2-(<i>N</i>-(<i>tert</i>-butoxycarbonyl)carbamimidoyl)-6-(hydroxymethyl)benzo[<i>b</i>]thiophen-4-yl)amino)propyl)carbamate (0.4 g, 0.84 mmol) and triethylamine (0.291 mL, 2.09 mmol) in DCM (25 mL) at 0 °C under nitrogen. The resulting solution was stirred at 20 °C for 30 min. The reaction mixture was diluted with DCM (100 mL) and washed sequentially with saturated brine (25 mL × 3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. This was used in the next step directly without further purification.</div></div><div id="sec4_10_1_2_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Step k: S-[[4-[[2-(<i>tert</i>-Butoxycarbonylamino)-1-methyl-ethyl]amino]-2-(<i>N</i>-<i>tert</i>-butoxycarbonylcarbamimidoyl)benzothiophen-6-yl]methyl]ethanethioate</h5><div class="NLM_p last">Potassium thioacetate (0.164 g, 1.44 mmol) was added to (4-((1-((<i>tert</i>-butoxycarbonyl)amino)propan-2-yl)amino)-2-(<i>N</i>-(<i>tert</i>-butoxycarbonyl)carbamimidoyl)benzo[<i>b</i>]thiophen-6-yl)methylmethanesulfonate (0.4 g, 0.72 mmol) in DMF (10 mL) at 20 °C under nitrogen. The resulting solution was stirred at 20 °C for 1 h. The reaction mixture was diluted with EtOAc (100 mL) and washed sequentially with saturated brine (20 mL × 3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0–10% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (0.180 g, 46.7%) as a gum. <sup>1</sup>H NMR (300 MHz, methanol-<i>d</i><sub>4</sub>) δ 0.85–1.05 (2 H, m), 1.24–1.35 (4 H, m), 1.42 (s, 9 H), 1.56 (9 H, s), 2.34–2.37 (m, 3 H), 4.19 (2 H, s), 6.54 (1 H, s), 7.08 (1 H, s), 8.17 (s, 1H). MS (ESI) [M + H]<sup>+</sup> = 537.5.</div></div><div id="sec4_10_1_2_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Step l: <i>S</i>-[[4-[(2-Amino-1-methyl-ethyl)amino]-2-carbamimidoyl-benzothiophen-6-yl]methyl] ethanethioate (<b>7</b>)</h5><div class="NLM_p last">TFA (2 mL, 25.96 mmol) was added dropwise to <i>S</i>-((4-((1-((<i>tert</i>-butoxycarbonyl)amino)propan-2-yl)amino)-2-(<i>N</i>-(<i>tert</i>-butoxycarbonyl)carbamimidoyl)benzo[<i>b</i>]thiophen-6-yl)methyl) ethanethioate (0.15 g, 0.28 mmol) in DCM (6 mL) at 20 °C under nitrogen. The resulting solution was stirred at 20 °C for 2 h. The mixture was concentrated in vacuo, and the crude product was used in the next step directly without further purification.</div></div><div id="sec4_10_1_2_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Step m: 4-[(2-Amino-1-methyl-ethyl)amino]-6-[[[4-[(2-amino-1-methyl-ethyl)amino]-2-carbamimidoyl-benzothiophen-6-yl]methyldisulfanyl]methyl]benzothiophene-2-carboxamidine</h5><div class="NLM_p last">K<sub>2</sub>CO<sub>3</sub> (0.411 g, 2.97 mmol) was added to <i>S</i>-((4-((1-aminopropan-2-yl)amino)-2-carbamimidoylbenzo[<i>b</i>]thiophen-6-yl)methyl) ethanethioate (0.2 g, 0.59 mmol) in MeOH (5 mL) at 20 °C under nitrogen. The resulting solution was stirred at 20 °C for 1 h. The crude product was purified by preparative HPLC (Column: Xselect CSH OBD column 30 mm × 150 mm, 5 μm; mobile phase A, water (0.05%TFA); mobile phase B, MeCN; flow rate, 60 mL/min. Fractions containing the desired compound were evaporated to dryness to afford the title compound (0.025 g, 12.50%) as a solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.22–1.31 (d, <i>J</i> = 6.3, 6H), 2.99–3.08 (d, <i>J</i> = 6.0, 4H), 3.90–4.00 (m, 6H), 6.58 (s, 2H), 7.20 (s, 2H), 8.58 (s, 2H). MS (ESI) [M + H]<sup>+</sup> = 587.5.</div></div><div id="sec4_10_1_2_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> Step n: 4-[(2-Amino-1-methyl-ethyl)amino]-6-(sulfanylmethyl)benzothiophene-2-carboxamidine (<b>AZ-006</b>)</h5><div class="NLM_p last">Trimethylphosphane in THF (1M, 0.1 mL, 0.10 mmol) was added in one portion to 6,6′-(disulfanediylbis(methylene))bis(4-((1-aminopropan-2-yl)amino)benzo[<i>b</i>]thiophene-2-carboximidamide) (20 mg, 0.03 mmol) in THF (2 mL) and water (0.2 mL) at 20 °C under nitrogen. The resulting solution was stirred at 20 °C for 30 min. The crude product was purified by preparative HPLC (Column: XBridge Prep OBD C18 column 19 mm × 250 mm, 5 μm; mobile phase A, water (0.05%TFA); mobile phase B, MeCN; flow rate, 20 mL/min. Fractions containing the desired compound were evaporated to dryness to afford the title compound (4.00 mg, 26.2%) as a solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.23–1.32 (d, <i>J</i> = 6.4, 3H), 2.97–3.05 (d, <i>J</i> = 6.4, 2H), 3.78 (s, 2H), 3.90–3.99 (m, 1H), 6.62 (s, 1H), 7.28 (s, 1H), 8.50 (s, 1H). MS (ESI) [M + H]<sup>+</sup> = 295.</div></div></div><div id="sec4_10_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Synthesis Pathway to 5-(1<i>H</i>-Imidazol-4-yl)-4-phenyl-thiophene-2-carboxamidine (<b>AZ-007</b>) (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Scheme S3</a>)</h4><div id="sec4_10_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> Step a: Methyl 5-Bromo-4-phenyl-thiophene-2-carboxylate (<b>8</b>)</h5><div class="NLM_p last">NBS (1.305 g, 7.33 mmol) was added to methyl 4-phenylthiophene-2-carboxylate (1.600 g, 7.33 mmol) in DMF (5 mL) at 25 °C. The resulting solution was stirred at 50 °C for 4 h. The crude product was purified by flash C18-flash chromatography, elution gradient 0–50% MeCN in water. Pure fractions were evaporated to dryness to afford the title compound (1.600 g, 73.5%) as a solid. MS (ESI) [M + H]<sup>+</sup> = 297/299.</div></div><div id="sec4_10_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> Step b: Methyl 4-Phenyl-5-[1-(2-trimethylsilylethoxymethyl)imidazol-4-yl]thiophene-2-carboxylate (<b>9</b>)</h5><div class="NLM_p last">A solution of Na<sub>2</sub>CO<sub>3</sub> (490 mg, 4.63 mmol) in water (1.000 mL) was added to a stirred suspension of methyl 5-bromo-4-phenylthiophene-2-carboxylate (500 mg, 1.68 mmol), 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-imidazole (<b>16</b>) (500 mg, 1.54 mmol) and Pd(Ph<sub>3</sub>P)<sub>4</sub> (178 mg, 0.15 mmol) in 1,4-dioxane (5 mL) at 25 °C under nitrogen. The resulting suspension was stirred at 90 °C overnight. The reaction mixture was diluted with EtOAc (200 mL), and washed sequentially with saturated brine (100 mL × 3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash C18-flash chromatography, elution gradient 0–60% MeCN in water. Pure fractions were evaporated to dryness to afford the title compound (300 mg, 46.9%) as an oil. MS (ESI) [M + H]<sup>+</sup> = 415.</div></div><div id="sec4_10_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> Step c: Methyl 5-(1<i>H</i>-Imidazol-4-yl)-4-phenyl-thiophene-2-carboxylate (<b>10</b>)</h5><div class="NLM_p last">HCl (1 mL, 32.91 mmol) was added to methyl 4-phenyl-5-(1-((2-(trimethylsilyl)ethoxy)methyl)-1,4,3,2-imidazol-4-yl)thiophene-2-carboxylate (200 mg, 0.48 mmol) in MeOH (5 mL) at 25 °C under nitrogen. The resulting solution was stirred at rt overnight. The solvent was removed under reduced pressure.The crude product was purified by flash C18-flash chromatography, elution gradient 0–50% MeCN in water. Pure fractions were evaporated to dryness to afford the title compound (80 mg, 58.3%) as an oil. MS (ESI) [M + H]<sup>+</sup> = 285.</div></div><div id="sec4_10_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> Step d: 5-(1<i>H</i>-Imidazol-4-yl)-4-phenyl-thiophene-2-carboxamidine (<b>AZ-007</b>)</h5><div class="NLM_p last">trimethylaluminum (81 mg, 1.13 mmol) was added to ammonium chloride (60.2 mg, 1.13 mmol) in toluene (3 mL) at 0 °C. The resulting suspension was stirred at rt for 1 h. A solution of methyl 5-(1<i>H</i>-imidazol-5-yl)-4-phenylthiophene-2-carboxylate (80 mg, 0.28 mmol) in toluene (3.00 mL) was added dropwise to the reaction at 25 °C under nitrogen. The resulting solution was stirred at 110 °C for 2 h. The reaction mixture was filtered through silica and washed with DCM:MeOH (5:1).The solvent was removed under reduced pressure.The crude product was purified by preparative HPLC (XBridge Prep C18 OBD column, 5 μ silica, 19 mm diameter, 150 mm length), using decreasingly polar mixtures of water (containing 0.1% formic acid) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford the title compound (15.00 mg, 15.76%) as a solid. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 6.65–6.70 (1 H, m), 7.40–7.55 (5 H, m), 7.75 (1 H, s), 7.88 (1 H, s), 8.49 (1 H, br s). MS (ESI) [M + H]<sup>+</sup> = 269.</div></div><div id="sec4_10_1_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> Trimethyl-[2-[[4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)imidazol-1-yl]methoxy]ethyl]silane (<b>16</b>)</h5><div class="NLM_p"><i>Step a</i>. NaH (150 mg, 3.74 mmol) was added slowly to 5-bromo-1<i>H</i>-imidazole (500 mg, 3.40 mmol) in THF (10 mL) at 0 °C under nitrogen. The resulting suspension was stirred at rt for 1 h. (2-(Chloromethoxy)ethyl)trimethylsilane (681 mg, 4.08 mmol) was added dropwise to the reaction at 0 °C over a period of 5 min under nitrogen. The resulting suspension was stirred at rt for 1 h. The reaction mixture was concentrated and diluted with EtOAc (250 mL) and washed sequentially with saturated brine (150 mL × 3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash C18-flash chromatography, elution gradient 0–50% MeCN in water. Pure fractions were evaporated to dryness to afford 5-bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-imidazole (900 mg, 95%) as a colorless oil. MS (ESI) [M + H]<sup>+</sup> = 277. <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 277; acid, HPLC <i>t</i><sub>R</sub> = 1.114 min.</div><div class="NLM_p last"><i>Step b</i>. PdCl<sub>2</sub>(dppf)–CH<sub>2</sub>Cl<sub>2</sub> adduct (14.73 mg, 0.02 mmol) was added to 4-bromo-1-((2-(trimethylsilyl)ethoxy)methyl)-1<i>H</i>-imidazole (500 mg, 1.80 mmol), 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane) (916 mg, 3.61 mmol) and potassium acetate (354 mg, 3.61 mmol) in 1,4-dioxane (5 mL) at 25 °C under nitrogen. The resulting suspension was stirred at 90 °C overnight. The reaction mixture was filtered through silica and afforded the title compound. The product was used in the next step without further purification.</div></div></div><div id="sec4_10_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Synthesis Pathway to <i>N</i>-(2-Aminoethyl)-2-carbamimidoyl-7-isopropoxy-benzothiophene-4-carboxamide (<b>AZ-008</b>) (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">Scheme S4</a>)</h4><div id="sec4_10_1_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> Step a: 6-Bromo-3-[(dimethyl-λ<sup>3</sup>-oxidanyl)methyl]-2-fluoro-benzaldehyde (<b>11</b>)</h5><div class="NLM_p last">BuLi (2.5M, 32.4 mL, 81.09 mmol) was added dropwise to diisopropylamine (7.81 g, 77.23 mmol) in THF (200 mL) at −78 °C under nitrogen. The resulting solution was stirred at −20 °C for 40 min. The reaction was incomplete and further 4-bromo-2-fluoro-1-isopropoxybenzene (18 g, 77.23 mmol) was added at −78 °C and the solution was stirred at −78 °C for a further 40 min. The reaction was incomplete and DMF (6 mL, 77.49 mmol) was added and the mixture was stirred at −78 °C for a further 1 h. The reaction mixture was quenched with saturated NH<sub>4</sub>Cl (20 mL), extracted with EtOAc (250 mL × 3), and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford colorless oil. The crude product was purified by flash silica chromatography, elution gradient 0–5% EtOAc in petroleum ether. Pure fractions were evaporated to dryness to afford the title compound (18.90 g, 94%) as a solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.17–1.43 (d, <i>J</i> 6.0, 6H), 4.60–4.79 (m, 1H), 7.38–7.58 (m, 2H), 10.17 (s, 1H).</div></div><div id="sec4_10_1_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> Step b: Methyl 4-Bromo-7-isopropoxy-benzothiophene-2-carboxylate</h5><div class="NLM_p last">Methyl 2-mercaptoacetate (8.07 g, 76.01 mmol) was added to 6-bromo-2-fluoro-3-isopropoxybenzaldehyde (18.9 g, 72.39 mmol) and K<sub>2</sub>CO<sub>3</sub> (15.01 g, 108.58 mmol) in DMF (60 mL) at 20 °C under nitrogen. The resulting mixture was stirred at 100 °C for 1.5 h. The reaction mixture was diluted with EtOAc (500 mL) and washed sequentially with saturated brine (75 mL × 4). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0–10% EtOAc in petroleum ether. Pure fractions were evaporated to dryness to afford the title compound (22.00 g, 92%) as a solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.20–1.40 (d, <i>J</i> 6.0, 6H), 3.90 (s, 3H), 4.77–4.92 (m, 1H), 7.05–7.14 (d, <i>J</i> 8.5, 1H), 7.59–7.68 (d, <i>J</i> 8.4, 1H), 7.92–8.00 (m, 1H).</div></div><div id="sec4_10_1_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> Step c: 4-Bromo-7-isopropoxy-benzothiophene-2-carboxamide</h5><div class="NLM_p last">Methyl 4-bromo-7-isopropoxybenzo[<i>b</i>]thiophene-2-carboxylate (3 g, 9.11 mmol) was added to 7 M NH<sub>3</sub> in MeOH (100 mL, 700.00 mmol) at 20 °C under nitrogen. The resulting solution was stirred at 90 °C for 14 h. The solvent was removed under reduced pressure. The product was used in the next step directly without further purification.</div></div><div id="sec4_10_1_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> Step d: 4-Bromo-7-isopropoxy-benzothiophene-2-carbonitrile (<b>12</b>)</h5><div class="NLM_p last">TFA (3.10 mL, 21.96 mmol) was added slowly to 4-bromo-7-isopropoxybenzo[<i>b</i>]thiophene-2-carboxamide (2.3 g, 7.32 mmol) and pyridine (2.96 mL, 36.60 mmol) in DCM (150 mL) at 0 °C under nitrogen. The resulting solution was stirred at rt for 1 h. The reaction mixture was diluted with DCM (100) and washed sequentially with 0.1 M HCl (75 mL × 3) and water (50 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to the title compound (2.010 g, 93%) as a solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.32–1.41 (d, <i>J</i> = 6.0, 6H), 4.82–4.97 (m, 1H), 7.15–7.25 (d, <i>J</i> = 8.6, 1H), 7.68–7.77 (d, <i>J</i> = 8.4, 1H), 8.32–8.39 (s, 1H).</div></div><div id="sec4_10_1_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> Step e: 2-Cyano-7-isopropoxy-benzothiophene-4-carboxylic Acid (<b>13</b>)</h5><div class="NLM_p last">BuLi in THF (2M, 15.61 mL, 31.22 mmol) was added dropwise to 4-bromo-7-isopropoxybenzo[<i>b</i>]thiophene-2-carbonitrile (0.8 g, 2.70 mmol) in THF (50 mL) and dry ice (2.97 mmol, 1.1 equiv) at −78 °C under nitrogen. The resulting solution was stirred at −78 °C for 10 min. The reaction was incomplete, and further dry ice (2.97 mmol) was added in one portion and the solution was stirred at −78 °C for a further 30 min. The reaction mixture was quenched with saturated NH<sub>4</sub>Cl (25 mL), extracted with EtOAc (3 × 100 mL), and the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford colorless oil. The crude product was purified by flash silica chromatography, elution gradient 0–10% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (0.550 g, 78%) as a gum. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.33–1.45 (m, 6H), 4.95–5.09 (m, 1H), 7.21–7.38 (m, 1H), 8.11–8.24 (m, 1H), 8.85 (s, 1H). MS (ESI) [M – H]<sup>−</sup> = 260.</div></div><div id="sec4_10_1_4_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> Step f: <i>tert</i>-Butyl <i>N</i>-[2-[(2-Cyano-7-isopropoxy-benzothiophene-4-carbonyl)amino]ethyl]carbamate (<b>14</b>)</h5><div class="NLM_p last">HATU (1.040 g, 2.74 mmol) was added to 2-cyano-7-isopropoxybenzo[<i>b</i>]thiophene-4-carboxylic acid (0.55 g, 2.10 mmol), <i>tert</i>-butyl (2-aminoethyl)carbamate (0.405 g, 2.53 mmol), and triethylamine (1.467 mL, 10.52 mmol) in DMF (10 mL) at 20 °C under nitrogen. The resulting solution was stirred at 20 °C for 1 h. The reaction mixture was diluted with EtOAc (100 mL) and washed sequentially with saturated brine (25 mL × 3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0–10% EtOAc in petroleum ether. Pure fractions were evaporated to dryness to afford the title compound (0.350 g, 41.2%) as a solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.33–1.43 (m, 15H), 1.72–1.83 (m, 2H), 3.11–3.22 (m, 2H), 3.56–3.67 (m, 2H), 6.96 (s, 1H), 7.26–7.35 (d, <i>J</i> = 8.4, 1H), 7.83–7.93 (d, <i>J</i> = 8.3, 1H), 8.71 (s, 1H). MS (ESI) [M + H]<sup>+</sup> = 404.</div></div><div id="sec4_10_1_4_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> Step g: <i>N</i>-(2-Aminoethyl)-2-cyano-7-isopropoxy-benzothiophene-4-carboxamide</h5><div class="NLM_p last">TFA (3 mL, 38.94 mmol) was added to <i>tert</i>-butyl (2-(2-cyano-7-isopropoxybenzo[<i>b</i>]thiophene-4-carboxamido)ethyl)carbamate (0.35 g, 0.87 mmol) in DCM (12 mL) at 20 °C under nitrogen. The resulting solution was stirred at rt for 1 h. The solvent was removed under reduced pressure. The reaction mixture was adjusted to pH 7. The crude product was purified by flash silica chromatography, elution gradient 0–20% MeOH in DCM. Pure fractions were evaporated to dryness to afford the title compound (0.250 g, 95%) as a solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.33–1.44 (m, 6H), 3.00–3.10 (m, 2H), 3.67–3.98 (m, 2H), 4.88–5.10 (m, 1H), 7.27–7.38 (m, 1H), 7.99–8.11 (m, 2H), 8.78 (s, 1 H). MS (ESI) [M + H]<sup>+</sup> = 304.</div></div><div id="sec4_10_1_4_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> Step h: <i>N</i>-(2-Aminoethyl)-2-(<i>N</i>-hydroxycarbamimidoyl)-7-isopropoxy-benzothiophene-4-carboxamide (<b>15</b>)</h5><div class="NLM_p last">Hydroxylammonium chloride (0.369 g, 5.31 mmol) was added to <i>N</i>-(2-aminoethyl)-2-cyano-7-isopropoxybenzo[<i>b</i>]thiophene-4-carboxamide (0.23 g, 0.76 mmol) and K<sub>2</sub>CO<sub>3</sub> (0.733 g, 5.31 mmol) in MeOH (8 mL) at 25 °C under nitrogen. The resulting solution was stirred at 50 °C for 1 h. The reaction mixture was filtered through Celite. The solvent was removed under reduced pressure to afford the title compound as a solid. The product was used in the next step directly without further purification.</div></div><div id="sec4_10_1_4_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> Step i: <i>N</i>-(2-Aminoethyl)-2-carbamimidoyl-7-isopropoxy-benzothiophene-4-carboxamide (<b>AZ-008</b>)</h5><div class="NLM_p last">Nickel (0.2 g, 3.41 mmol) was added to <i>N</i>-(2-aminoethyl)-2-(<i>N</i>-hydroxycarbamimidoyl)-7-isopropoxybenzo[<i>b</i>]thiophene-4-carboxamide (0.15 g, 0.45 mmol) and acetic acid (0.5 mL, 8.73 mmol) in MeOH (5 mL) at 25 °C under nitrogen. The resulting solution was stirred at 40 °C for 1 h under an atmosphere of hydrogen. The mixture was filtered through a Celite pad. The solvent was removed under reduced pressure. The crude product was purified by preparative HPLC column: Xselect CSH OBD column 30 mm × 150 mm, 5 μm; mobile phase A, water (0.05%TFA), mobile phase B, MeCN; flow rate, 60 mL/min. Fractions containing the desired compound were evaporated to dryness to afford the title compound (0.025 g, 8.67%) as a white solid. <sup>1</sup>H NMR (300 MHz, methanol-<i>d</i><sub>4</sub>) δ 1.43–1.52 (d, <i>J</i> = 6.0, 6H), 3.18–3.29 (t, <i>J</i> = 5.9, 2H), 3.65–3.79 (t, <i>J</i> = 6.0, 2H), 4.94–5.04 (m, 1H), 7.14–7.24 (d, <i>J</i> = 8.5, 1H), 7.90–7.99 (d, <i>J</i> = 8.3, 1H), 8.87 (s, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.25, 159.88, 158.88, 158.60, 154.39, 138.36, 132.01, 131.91, 130.72, 128.51, 124.14, 108.77, 71.88, 39.05, 22.16. MS (ESI) [M + H]<sup>+</sup> = 321. HRMS (ES<sup>+</sup>) for C<sub>15</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub>S [M + H]<sup>+</sup>: calcd 321.1385, found 321.1392.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i57"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01942" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33466" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33466" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01942?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01942</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional results and X-ray crystallography tables (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">PDB docking AZ-008 to p53 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf">jm9b01942_si_001.pdf (6.15 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_002.csv">jm9b01942_si_002.csv (1.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_003.pdb">jm9b01942_si_003.pdb (322.99 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors have been deposited in the Protein Data Bank under the following accession codes. 14-3-3σ/TAZpS89/fragment complexes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RHC">6RHC</a> (AZ-003); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLW">6SLW</a> (AZ-004); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJQ">6RJQ</a> (AZ-006); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLX">6SLX</a> (AZ-010); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJL">6RJL</a> (AZ-018); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RP6">6RP6</a> (AZ-019). 14-3-3σ/p53pT387/Fragment complexes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S40">6S40</a> (AZ-001); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWI">6RWI</a> (AZ-002); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL3">6RL3</a> (AZ-003); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S9Q">6S9Q</a> (AZ-004); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RX2">6RX2</a> (AZ-005); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5L">6R5L</a> (AZ-006); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWH">6RWH</a> (AZ-007); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWS">6RWS</a> (AZ-009); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWU">6RWU</a> (AZ-010); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIP">6SIP</a> (AZ-011); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIQ">6SIQ</a> (AZ-012); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLV">6SLV</a> (AZ-013); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RK8">6RK8</a> (AZ-014); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJZ">6RJZ</a> (AZ-015); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RM5">6RM5</a> (AZ-016); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIO">6SIO</a> (AZ-017); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S39">6S39</a> (AZ-018); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S3C">6S3C</a> (AZ-019); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIN">6SIN</a> (AZ-020); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKK">6RKK</a> (AZ-21); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKM">6RKM</a> (AZ-022); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKI">6RKI</a> (AZ-023); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL6">6RL6</a> (AZ-024); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL4">6RL4</a> (AZ-025); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RM7">6RM7</a> (AZ-026). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01942" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15723" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15723" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Isabelle Landrieu</span> - <span class="hlFld-Affiliation affiliation">CNRS
ERL9002 Integrative Structural Biology F-59000 Lille, France</span>; 
    <span class="hlFld-Affiliation affiliation">Univ.
Lille, Inserm, CHU Lille, Institut Pasteur de Lille, U1167-RID-AGE,
Risk Factors and Molecular Determinants of Aging-Related Diseases, F-59000 Lille, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4883-2637" title="Orcid link">http://orcid.org/0000-0002-4883-2637</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b6dfc5d7d4d3dadad398dad7d8d2c4dfd3c3f6c3d8dfc09bdadfdadad398d0c4"><span class="__cf_email__" data-cfemail="a9c0dac8cbccc5c5cc87c5c8c7cddbc0ccdce9dcc7c0df84c5c0c5c5cc87cfdb">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongming Chen</span> - <span class="hlFld-Affiliation affiliation">Hit
Discovery, Discovery Sciences, Biopharmaceutical R&D, AstraZeneca,
Gothenburg, 431 50 Mölndal, Sweden</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4e2621202923272029602d262b20797f0e26213a232f2722602d2123"><span class="__cf_email__" data-cfemail="50383f3e373d393e377e3338353e676110383f243d31393c7e333f3d">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Ottmann</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Organic Chemistry, University of Duisburg-Essen, 47057 Duisburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7315-0315" title="Orcid link">http://orcid.org/0000-0001-7315-0315</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a9ca87c6ddddc4c8c7c7e9dddccc87c7c5"><span class="__cf_email__" data-cfemail="e586cb8a919188848b8ba5919080cb8b89">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xavier Guillory</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span>; 
    <span>Present Address:
                        For X.G.: Univ Rennes, CNRS, ISCR (Institut des Sciences Chimiques de Rennes), UMR 6226, F-35000 Rennes, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Madita Wolter</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Seppe Leysen</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span>; 
    <span>Present Address:
                        For S.L.: UCB Pharma, 208 Bath Road, Slough, Berkshire SL1 3WE, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">João Filipe Neves</span> - <span class="hlFld-Affiliation affiliation">CNRS
ERL9002 Integrative Structural Biology F-59000 Lille, France</span>; 
    <span class="hlFld-Affiliation affiliation">Univ.
Lille, Inserm, CHU Lille, Institut Pasteur de Lille, U1167-RID-AGE,
Risk Factors and Molecular Determinants of Aging-Related Diseases, F-59000 Lille, France</span>; 
    <span>Present Address:
                        For J.F.N.: Covance, Zandvoortstraat 2, 2800 Mechelen, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ave Kuusk</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span>; 
    <span class="hlFld-Affiliation affiliation">Hit
Discovery, Discovery Sciences, Biopharmaceutical R&D, AstraZeneca,
Gothenburg, 431 50 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sylvia Genet</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bente Somsen</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Kenneth Morrow</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry and Small Molecule Discovery Center (SMDC), University of California, San Francisco, California 94143, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma Rivers</span> - <span class="hlFld-Affiliation affiliation">Hit
Discovery, Discovery Sciences, Biopharmaceutical R&D, AstraZeneca,
Gothenburg, 431 50 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lotte van Beek</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joe Patel</span> - <span class="hlFld-Affiliation affiliation">Oncology
and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse
Park, Waltham, Massachusetts 02451, United States</span>; 
    <span>Present Address:
                        For J.P.: C4Therapeutics, 490 Arsenal Way no. 200, Watertown, Massachusetts 02472, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert Goodnow</span> - <span class="hlFld-Affiliation affiliation">Oncology
and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse
Park, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heike Schoenherr</span> - <span class="hlFld-Affiliation affiliation">Oncology
and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse
Park, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nathan Fuller</span> - <span class="hlFld-Affiliation affiliation">Oncology
and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse
Park, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qing Cao</span> - <span class="hlFld-Affiliation affiliation">Oncology
and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse
Park, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard G. Doveston</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span>; 
    <span>Present Address:
                        For R.G.D.: Leicester Institute of Structural and Chemical Biology and School of Chemistry, University of Leicester, University Road, Leicester LE1 7RH, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luc Brunsveld</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Chemical Biology, Department of Biomedical Engineering and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600MB Eindhoven, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5675-511X" title="Orcid link">http://orcid.org/0000-0001-5675-511X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michelle R. Arkin</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry and Small Molecule Discovery Center (SMDC), University of California, San Francisco, California 94143, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9366-6770" title="Orcid link">http://orcid.org/0000-0002-9366-6770</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Castaldi</span> - <span class="hlFld-Affiliation affiliation">Oncology
and Discovery Sciences, IMED Biotech Unit, AstraZeneca, Gatehouse
Park, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helen Boyd</span> - <span class="hlFld-Affiliation affiliation">Hit
Discovery, Discovery Sciences, Biopharmaceutical R&D, AstraZeneca,
Gothenburg, 431 50 Mölndal, Sweden</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>X.G., M.W., and S.L. contributed equally. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2450-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61488" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61488" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge François-Xavier Cantrelle for assistance on NMR data acquisition. The NMR facilities were funded by the Nord Region Council, CNRS, Institut Pasteur de Lille, the European Community (ERDF), the French Ministry of Research, and the University of Lille and by the CTRL CPER, cofunded by the European Union with the European Regional Development Fund (ERDF), by the Hauts de France Regional Council (contract no. 17003781), Métropole Européenne de Lille (contract no. 2016_ESR_05), and French State (contract no. 2017-R3-CTRL phase 1).We acknowledge support for the NMR facilities from TGE RMN THC (CNRS, FR-3050) and FRABio (Univ. Lille, CNRS, FR-3688). The research is supported by funding from the European Union through the AEGIS (H2020-MSCA-ITN-2015, grant no. 67555), and the TASPPI project (H2020-MSCA-ITN-2015, grant no. 675179). J.F.N and I.L acknowledge LabEx (Laboratory of Excellence) for financial support on the scope of the DISTALZ consortium (ANR, ANR-11-LABX-009).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">fragment-based drug discovery</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">PROTACs</td><td class="NLM_def"><p class="first last">proteolysis targeting chimeras</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">FA</td><td class="NLM_def"><p class="first last">formic acid</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-dimethylaminopyridine</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">NBS</td><td class="NLM_def"><p class="first last"><i>N</i>-bromosuccinimide</p></td></tr><tr><td class="NLM_term">Ms-Cl</td><td class="NLM_def"><p class="first last">methanesulfonyl chloride</p></td></tr><tr><td class="NLM_term">CSP</td><td class="NLM_def"><p class="first last">chemical shift perturbation</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05552" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05552" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1102</span>– <span class="NLM_lpage">1114</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.chembiol.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=25237857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1102-1114&issue=9&author=M.+R.+Arkinauthor=Y.+Tangauthor=J.+A.+Wells&title=Small-Molecule+Inhibitors+of+Protein-Protein+Interactions%3A+Progressing+toward+the+Reality&doi=10.1016%2Fj.chembiol.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span></div><div class="casAuthors">Arkin, Michelle R.; Tang, Yinyan; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1114</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The past 20 years have seen many advances in our understanding of protein-protein interactions (PPIs) and how to target them with small-mol. therapeutics.  In 2004, we reviewed some early successes; since then, potent inhibitors have been developed for diverse protein complexes, and compds. are now in clin. trials for six targets.  Surprisingly, many of these PPI clin. candidates have efficiency metrics typical of "lead-like" or "drug-like" mols. and are orally available.  Successful discovery efforts have integrated multiple disciplines and make use of all the modern tools of target-based discovery-structure, computation, screening, and biomarkers.  PPIs become progressively more challenging as the interfaces become more complex, i.e., as binding epitopes are displayed on primary, secondary, or tertiary structures.  Here, we review the last 10 years of progress, focusing on the properties of PPI inhibitors that have advanced to clin. trials and prospects for the future of PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP01wt-4lfE7Vg90H21EOLACvtfcHk0lgVUPUk4l2Fsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J&md5=aec3897dc9bc66656723bc56c8d69155</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-Molecule%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%253A%2520Progressing%2520toward%2520the%2520Reality%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26issue%3D9%26spage%3D1102%26epage%3D1114%26doi%3D10.1016%2Fj.chembiol.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briels, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Stabilization of Protein-Protein Interactions in Chemical Biology and Drug Discovery</span>. <i>Prog. Biophys. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2015.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.pbiomolbio.2015.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=26093250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKmsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2015&pages=10-19&issue=1&author=D.+Bierauthor=P.+Thielauthor=J.+Brielsauthor=C.+Ottmann&title=Stabilization+of+Protein-Protein+Interactions+in+Chemical+Biology+and+Drug+Discovery&doi=10.1016%2Fj.pbiomolbio.2015.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of Protein-Protein Interactions in chemical biology and drug discovery</span></div><div class="casAuthors">Bier, David; Thiel, Philipp; Briels, Jeroen; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10-19</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  More than 300,000 Protein-Protein Interactions (PPIs) can be found in human cells.  This no. is significantly larger than the no. of single proteins, which are the classical targets for pharmacol. intervention.  Hence, specific and potent modulation of PPIs by small, drug-like mols. would tremendously enlarge the "druggable genome" enabling novel ways of drug discovery for essentially every human disease.  This strategy is esp. promising in diseases with difficult targets like intrinsically disordered proteins or transcription factors, for example neurodegeneration or metabolic diseases.  Whereas the potential of PPI modulation has been recognized in terms of the development of inhibitors that disrupt or prevent a binary protein complex, the opposite (or complementary) strategy to stabilize PPIs has not yet been realized in a systematic manner.  This fact is rather surprising given the no. of impressive natural product examples that confer their activity by stabilizing specific PPIs.  In addn., in recent years more and more examples of synthetic mols. are being published that work as PPI stabilizers, despite the fact that in the majority they initially have not been designed as such.  Here, we describe examples from both the natural products as well as the synthetic mols. advocating for a stronger consideration of the PPI stabilization approach in chem. biol. and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAhN329QU_arVg90H21EOLACvtfcHk0lgVUPUk4l2Fsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKmsb3F&md5=8a51dfc8088641b0bfa658250cd6117a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2015.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DBier%26aufirst%3DD.%26aulast%3DThiel%26aufirst%3DP.%26aulast%3DBriels%26aufirst%3DJ.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DStabilization%2520of%2520Protein-Protein%2520Interactions%2520in%2520Chemical%2520Biology%2520and%2520Drug%2520Discovery%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D119%26issue%3D1%26spage%3D10%26epage%3D19%26doi%3D10.1016%2Fj.pbiomolbio.2015.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrei, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karawajczyk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickhoff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doveston, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Stabilization of Protein-Protein Interactions in Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">925</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1346608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1080%2F17460441.2017.1346608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=28695752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOrsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=925-940&issue=9&author=S.+A.+Andreiauthor=E.+Sijbesmaauthor=M.+Hannauthor=J.+Davisauthor=G.+O%E2%80%99Mahonyauthor=M.+W.+D.+Perryauthor=A.+Karawajczykauthor=J.+Eickhoffauthor=L.+Brunsveldauthor=R.+G.+Dovestonauthor=L.-G.+Milroyauthor=C.+Ottmann&title=Stabilization+of+Protein-Protein+Interactions+in+Drug+Discovery&doi=10.1080%2F17460441.2017.1346608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of protein-protein interactions in drug discovery</span></div><div class="casAuthors">Andrei, Sebastian A.; Sijbesma, Eline; Hann, Michael; Davis, Jeremy; O'Mahony, Gavin; Perry, Matthew W. D.; Karawajczyk, Anna; Eickhoff, Jan; Brunsveld, Luc; Doveston, Richard G.; Milroy, Lech-Gustav; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">925-940</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">: PPIs are involved in every disease and specific modulation of these PPIs with small mols. would significantly improve our prospects of developing therapeutic agents.  Both industry and academia have engaged in the identification and use of PPI inhibitors.  However in comparison, the opposite strategy of employing small-mol. stabilizers of PPIs is underrepresented in drug discovery.: PPI stabilization has not been exploited in a systematic manner.  Rather, this concept validated by a no. of therapeutically used natural products like rapamycin and paclitaxel has been shown retrospectively to be the basis of the activity of synthetic mols. originating from drug discovery projects among them lenalidomide and tafamidis.  Here, the authors cover the growing no. of synthetic small-mol. PPI stabilizers to advocate for a stronger consideration of this as a drug discovery approach.: Both the natural products and the growing no. of synthetic mols. show that PPI stabilization is a viable strategy for drug discovery.  There is certainly a significant challenge to adapt compd. libraries, screening techniques and downstream methodologies to identify, characterize and optimize PPI stabilizers, but the examples of mols. reviewed here in our opinion justify these efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojOzbEj0fPcLVg90H21EOLACvtfcHk0lgXn8ar-Nnb0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOrsbnK&md5=fd6aa91fbca0e69af4a4bdb2f9ce850c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1346608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1346608%26sid%3Dliteratum%253Aachs%26aulast%3DAndrei%26aufirst%3DS.%2BA.%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DHann%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DG.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26aulast%3DKarawajczyk%26aufirst%3DA.%26aulast%3DEickhoff%26aufirst%3DJ.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DDoveston%26aufirst%3DR.%2BG.%26aulast%3DMilroy%26aufirst%3DL.-G.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DStabilization%2520of%2520Protein-Protein%2520Interactions%2520in%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2017%26volume%3D12%26issue%3D9%26spage%3D925%26epage%3D940%26doi%3D10.1080%2F17460441.2017.1346608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stafford, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKintosh, C.</span></span> <span> </span><span class="NLM_article-title">Bioinformatic and Experimental Survey of 14–3-3-Binding Sites</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>427</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1042/BJ20091834</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FBJ20091834" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=20141511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyntrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=427&publication_year=2010&pages=69-78&issue=1&author=C.+Johnsonauthor=S.+Crowtherauthor=M.+J.+Staffordauthor=D.+G.+Campbellauthor=R.+Tothauthor=C.+MacKintosh&title=Bioinformatic+and+Experimental+Survey+of+14%E2%80%933-3-Binding+Sites&doi=10.1042%2FBJ20091834"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bioinformatic and experimental survey of 14-3-3-binding sites</span></div><div class="casAuthors">Johnson, Catherine; Crowther, Sandra; Stafford, Margaret J.; Campbell, David G.; Toth, Rachel; MacKintosh, Carol</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">427</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-78</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">More than 200 phosphorylated 14-3-3-binding sites in the literature were analyzed to define 14-3-3 specificities, identify relevant protein kinases, and give insights into how cellular 14-3-3/phosphoprotein networks work.  Mode I RXX(pS/pT)XP motifs dominate, although the +2 proline residue occurs in less than half, and LX(R/K)SX(pS/pT)XP is prominent in plant 14-3-3-binding sites.  Proline at +1 is rarely reported, and such motifs did not stand up to exptl. reanal. of human Ndel1.  Instead, we discovered that 14-3-3 interacts with two residues that are phosphorylated by basophilic kinases and located in the DISC1 (disrupted-in-schizophrenia 1)-interacting region of Ndel1 that is implicated in cognitive disorders.  These data conform with the general findings that there are different subtypes of 14-3-3-binding sites that overlap with the specificities of different basophilic AGC (protein kinase A/protein kinase G/protein kinase C family) and CaMK (Ca2+/calmodulin-dependent protein kinase) protein kinases, and a 14-3-3 dimer often engages with two tandem phosphorylated sites, which is a configuration with special signalling, mech. and evolutionary properties.  Thus 14-3-3 dimers can be digital logic gates that integrate more than one input to generate an action, and coincidence detectors when the two binding sites are phosphorylated by different protein kinases.  Paired sites are generally located within disordered regions and/or straddle either side of functional domains, indicating how 14-3-3 dimers modulate the conformations and/or interactions of their targets.  Finally, 14-3-3 proteins bind to members of several multi-protein families.  Two 14-3-3-binding sites are conserved across the class IIa histone deacetylases, whereas other protein families display differential regulation by 14-3-3s.  We speculate that 14-3-3 dimers may have contributed to the evolution of such families, tailoring regulatory inputs to different physiol. demands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfY5xjqSQDjLVg90H21EOLACvtfcHk0lgXn8ar-Nnb0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyntrg%253D&md5=bfe2c524bc5efe8707004f1adb76bf55</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1042%2FBJ20091834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20091834%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DCrowther%26aufirst%3DS.%26aulast%3DStafford%26aufirst%3DM.%2BJ.%26aulast%3DCampbell%26aufirst%3DD.%2BG.%26aulast%3DToth%26aufirst%3DR.%26aulast%3DMacKintosh%26aufirst%3DC.%26atitle%3DBioinformatic%2520and%2520Experimental%2520Survey%2520of%252014%25E2%2580%25933-3-Binding%2520Sites%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D427%26issue%3D1%26spage%3D69%26epage%3D78%26doi%3D10.1042%2FBJ20091834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baljuls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinders, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nekhoroshkova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibilski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajalingam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hekman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, U. R.</span></span> <span> </span><span class="NLM_article-title">Regulation of RAF Activity by 14–3-3 Proteins: RAF Kinases Associate Functionally with Both Homo- and Heterodimeric Forms of 14–3-3 Proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">3183</span>– <span class="NLM_lpage">3194</span>, <span class="refDoi"> DOI: 10.1074/jbc.M804795200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1074%2Fjbc.M804795200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=19049963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsV2mug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=3183-3194&issue=5&author=A.+Fischerauthor=A.+Baljulsauthor=J.+Reindersauthor=E.+Nekhoroshkovaauthor=C.+Sibilskiauthor=R.+Metzauthor=S.+Albertauthor=K.+Rajalingamauthor=M.+Hekmanauthor=U.+R.+Rapp&title=Regulation+of+RAF+Activity+by+14%E2%80%933-3+Proteins%3A+RAF+Kinases+Associate+Functionally+with+Both+Homo-+and+Heterodimeric+Forms+of+14%E2%80%933-3+Proteins&doi=10.1074%2Fjbc.M804795200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of RAF Activity by 14-3-3 Proteins: RAF Kinases Associate Functionally with both Homo and Heterodimeric forms of 14-3-3 Proteins</span></div><div class="casAuthors">Fischer, Andreas; Baljuls, Angela; Reinders, Joerg; Nekhoroshkova, Elena; Sibilski, Claudia; Metz, Renate; Albert, Stefan; Rajalingam, Krishnaraj; Hekman, Mirko; Rapp, Ulf R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3183-3194</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mammalian 14-3-3 proteins play a crucial role in the activation process of RAF kinases.  However, little is known about the selectivity of the mammalian 14-3-3 isoforms with respect to RAF assocn. and activation.  Using mass spectrometry, we analyzed the compn. of the 14-3-3 isoforms attached to RAF kinases and found that B-RAF assocs. in vivo with 14-3-3 at much higher diversity than A- and C-RAF.  We also examd. in vitro binding of purified mammalian 14-3-3 proteins to RAF kinases using surface plasmon resonance techniques.  While B- and C-RAF exhibited binding to all seven 14-3-3 isoforms, A-RAF bound with considerably lower affinities to ε, τ, and σ 14-3-3.  These findings indicate that 14-3-3 proteins assoc. with RAF isoforms in a pronounced isoform-specific manner.  Because 14-3-3 proteins appear in dimeric forms, we addressed the question of whether both homo- and heterodimeric forms of 14-3-3 proteins participate in RAF signaling.  For that purpose, the budding yeast Saccharomyces cerevisiae, possessing only two 14-3-3 isoforms (BMH1 and BMH2), served as testing system.  By deletion of the single BMH2 gene, we found that both homo- and heterodimeric forms of 14-3-3 can participate in RAF activation.  Furthermore, we show that A-, B-, and C-RAF activity is differentially regulated by its C-terminal and internal 14-3-3 binding domain.  Finally, prohibitin, a scaffold protein that affects C-RAF activation in a stimulatory manner, proved to interfere with the internal 14-3-3 binding site in C-RAF.  Together, our results shed more light on the complex mechanism of RAF activation, particularly with respect to activation steps that are mediated by 14-3-3 proteins and prohibitin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo1kD9RoVWHrVg90H21EOLACvtfcHk0lgXn8ar-Nnb0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpsV2mug%253D%253D&md5=8ef2f7b54787f7bdcd4594f97a4327a8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M804795200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M804795200%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DBaljuls%26aufirst%3DA.%26aulast%3DReinders%26aufirst%3DJ.%26aulast%3DNekhoroshkova%26aufirst%3DE.%26aulast%3DSibilski%26aufirst%3DC.%26aulast%3DMetz%26aufirst%3DR.%26aulast%3DAlbert%26aufirst%3DS.%26aulast%3DRajalingam%26aufirst%3DK.%26aulast%3DHekman%26aufirst%3DM.%26aulast%3DRapp%26aufirst%3DU.%2BR.%26atitle%3DRegulation%2520of%2520RAF%2520Activity%2520by%252014%25E2%2580%25933-3%2520Proteins%253A%2520RAF%2520Kinases%2520Associate%2520Functionally%2520with%2520Both%2520Homo-%2520and%2520Heterodimeric%2520Forms%2520of%252014%25E2%2580%25933-3%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26issue%3D5%26spage%3D3183%26epage%3D3194%26doi%3D10.1074%2Fjbc.M804795200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molzan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Synergistic Binding of the Phosphorylated S233- and S259-Binding Sites of C-RAF to One 14–3-3ζ Dimer</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>423</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">495</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.jmb.2012.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22922483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGqt7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=423&publication_year=2012&pages=486-495&issue=4&author=M.+Molzanauthor=C.+Ottmann&title=Synergistic+Binding+of+the+Phosphorylated+S233-+and+S259-Binding+Sites+of+C-RAF+to+One+14%E2%80%933-3%CE%B6+Dimer&doi=10.1016%2Fj.jmb.2012.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic Binding of the Phosphorylated S233- and S259-Binding Sites of C-RAF to One 14-3-3ζ Dimer</span></div><div class="casAuthors">Molzan, Manuela; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">423</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-495</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">C-RAF kinase is a central component of the Ras-RAF-MEK (mitogen-activated protein kinase/extracellular signal-regulated kinase)-ERK (extracellular signal-regulated kinase) pathway, which has been shown to be activated in 30% of human tumors.  14-3-3 Proteins inactivate C-RAF by binding to the two N-terminal phosphorylation-dependent binding sites surrounding S233 and S259.  14-3-3 Proteins can bind two target sequences located on one polypeptide chain simultaneously, thereby increasing binding affinity compared to single-site binding and possibly allowing regulated 14-3-3 binding through gatekeeper phosphorylation.  To date, it was unclear whether 14-3-3 proteins can bind the two N-terminal phosphorylation-dependent binding sites of C-RAF simultaneously.  Fluorescence polarization (FP) using phosphorylated peptides demonstrated that S233 is the low-affinity and S259 is the high-affinity binding site, while simultaneous engagement of both sites by 14-3-3ζ enhances affinity compared to single-site binding.  Detn. of a 1:1 stoichiometry for the di-phosphorylated peptide binding to one 14-3-3ζ dimer with isothermal titrn. calorimetry was supported by the crystal structure of the 14-3-3ζ/C-RAFpS233,pS259 complex.  Cellular localization studies validate the significance of these sites for cytoplasmic retention of C-RAF, suggesting an extended mechanism of RAF regulation by 14-3-3 proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWsGiYB2r9z7Vg90H21EOLACvtfcHk0ljI3nlzIOk1kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGqt7%252FN&md5=5b7d6cddd74ffb26a7b1d597da4a09fe</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DMolzan%26aufirst%3DM.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSynergistic%2520Binding%2520of%2520the%2520Phosphorylated%2520S233-%2520and%2520S259-Binding%2520Sites%2520of%2520C-RAF%2520to%2520One%252014%25E2%2580%25933-3%25CE%25B6%2520Dimer%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D423%26issue%3D4%26spage%3D486%26epage%3D495%26doi%3D10.1016%2Fj.jmb.2012.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molzan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röglin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skwarczynska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbuecher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkanda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Stabilization of Physical RAF/14–3-3 Interaction by Cotylenin A as Treatment Strategy for RAS Mutant Cancers</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1869</span>– <span class="NLM_lpage">1875</span>, <span class="refDoi"> DOI: 10.1021/cb4003464</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4003464" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVeisLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1869-1875&issue=9&author=M.+Molzanauthor=S.+Kasperauthor=L.+R%C3%B6glinauthor=M.+Skwarczynskaauthor=T.+Sassaauthor=T.+Inoueauthor=F.+Breitenbuecherauthor=J.+Ohkandaauthor=N.+Katoauthor=M.+Schulerauthor=C.+Ottmann&title=Stabilization+of+Physical+RAF%2F14%E2%80%933-3+Interaction+by+Cotylenin+A+as+Treatment+Strategy+for+RAS+Mutant+Cancers&doi=10.1021%2Fcb4003464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of Physical RAF/14-3-3 Interaction by Cotylenin A as Treatment Strategy for RAS Mutant Cancers</span></div><div class="casAuthors">Molzan, Manuela; Kasper, Stefan; Roeglin, Lars; Skwarczynska, Malgorzata; Sassa, Takeshi; Inoue, Takatsugu; Breitenbuecher, Frank; Ohkanda, Junko; Kato, Nobuo; Schuler, Martin; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1869-1875</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One-third of all human cancers harbor somatic RAS mutations.  This leads to aberrant activation of downstream signaling pathways involving the RAF kinases.  Current ATP-competitive RAF inhibitors are active in cancers with somatic RAF mutations, such as BRAFV600 mutant melanomas.  However, they paradoxically promote the growth of RAS mutant tumors, partly due to the complex interplay between different homo- and heterodimers of A-RAF, B-RAF, and C-RAF.  Based on pathway anal. and structure-guided compd. identification, we describe the natural product cotylenin-A (CN-A) as stabilizer of the phys. interaction of C-RAF with 14-3-3 proteins.  CN-A binds to inhibitory 14-3-3 interaction sites of C-RAF, pSer233, and pSer259, but not to the activating interaction site, pSer621.  While CN-A alone is inactive in RAS mutant cancer models, combined treatment with CN-A and an anti-EGFR antibody synergistically suppresses tumor growth in vitro and in vivo.  This defines a novel pharmacol. strategy for treatment of RAS mutant cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYNk-se5ruGrVg90H21EOLACvtfcHk0ljI3nlzIOk1kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVeisLrE&md5=f1dfbdbc95a185c7b893ebc45ac72e92</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb4003464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4003464%26sid%3Dliteratum%253Aachs%26aulast%3DMolzan%26aufirst%3DM.%26aulast%3DKasper%26aufirst%3DS.%26aulast%3DR%25C3%25B6glin%26aufirst%3DL.%26aulast%3DSkwarczynska%26aufirst%3DM.%26aulast%3DSassa%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DBreitenbuecher%26aufirst%3DF.%26aulast%3DOhkanda%26aufirst%3DJ.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DSchuler%26aufirst%3DM.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DStabilization%2520of%2520Physical%2520RAF%252F14%25E2%2580%25933-3%2520Interaction%2520by%2520Cotylenin%2520A%2520as%2520Treatment%2520Strategy%2520for%2520RAS%2520Mutant%2520Cancers%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26issue%3D9%26spage%3D1869%26epage%3D1875%26doi%3D10.1021%2Fcb4003464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leclerc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paudel, H. K.</span></span> <span> </span><span class="NLM_article-title">Interaction of 14–3-3ζ with Microtubule-Associated Protein Tau within Alzheimer’s Disease Neurofibrillary Tangles</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">37</span>),  <span class="NLM_fpage">6445</span>– <span class="NLM_lpage">6455</span>, <span class="refDoi"> DOI: 10.1021/bi400442d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi400442d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht12hurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=6445-6455&issue=37&author=H.+Y.+Qureshiauthor=T.+Liauthor=R.+MacDonaldauthor=C.+M.+Choauthor=N.+Leclercauthor=H.+K.+Paudel&title=Interaction+of+14%E2%80%933-3%CE%B6+with+Microtubule-Associated+Protein+Tau+within+Alzheimer%E2%80%99s+Disease+Neurofibrillary+Tangles&doi=10.1021%2Fbi400442d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of 14-3-3ζ with Microtubule-Associated Protein Tau within Alzheimer's Disease Neurofibrillary Tangles</span></div><div class="casAuthors">Qureshi, Hamid Y.; Li, Tong; MacDonald, Ryen; Cho, Chul Min; Leclerc, Nicole; Paudel, Hemant K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">6445-6455</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is characterized by the presence of abnormal, straight filaments and paired helical filaments (PHFs) that are coated with amorphous aggregates.  When PHFs are treated with alkali, they untwist and form filaments with a ribbonlike morphol.  Tau protein is the major component of all of these ultrastructures.  14-3-3ζ Is present in NFTs and is significantly upregulated in AD brain.  The mol. basis of the assocn. of 14-3-3ζ within NFTs and the pathol. significance of its assocn. are not known.  In this study, we have found that 14-3-3ζ is copurified and co-immunoppts. with tau from NFTs of AD brain ext.  In vitro, tau binds to both phosphorylated and nonphosphorylated tau.  When incubated with 14-3-3ζ, tau forms amorphous aggregates, single-stranded, straight filaments, ribbonlike filaments, and PHF-like filaments, all of which resemble the corresponding ultrastructures found in AD brain.  Immuno-electron microscopy detd. that both tau and 14-3-3ζ are present in these ultrastructures and that they are formed in an incubation time-dependent manner.  Amorphous aggregates are formed first.  As the incubation time increases, the size of amorphous aggregates increases and they are incorporated into single-stranded filaments.  Single-stranded filaments laterally assoc. to form double-stranded, ribbonlike, and PHF-like filaments.  Both tau and phosphorylated tau aggregate in a similar manner when they are incubated with 14-3-3ζ.  Our data suggest that 14-3-3ζ has a role in the fibrillization of tau in AD brain, and that tau phosphorylation does not affect 14-3-3ζ-induced tau aggregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5oervCC-UabVg90H21EOLACvtfcHk0ljI3nlzIOk1kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht12hurfP&md5=95c065c72827033dc4318cb09041603a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi400442d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi400442d%26sid%3Dliteratum%253Aachs%26aulast%3DQureshi%26aufirst%3DH.%2BY.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMacDonald%26aufirst%3DR.%26aulast%3DCho%26aufirst%3DC.%2BM.%26aulast%3DLeclerc%26aufirst%3DN.%26aulast%3DPaudel%26aufirst%3DH.%2BK.%26atitle%3DInteraction%2520of%252014%25E2%2580%25933-3%25CE%25B6%2520with%2520Microtubule-Associated%2520Protein%2520Tau%2520within%2520Alzheimer%25E2%2580%2599s%2520Disease%2520Neurofibrillary%2520Tangles%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26issue%3D37%26spage%3D6445%26epage%3D6455%26doi%3D10.1021%2Fbi400442d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smet-Nocca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, Y.-H.</span></span> <span> </span><span class="NLM_article-title">Involvement of 14–3-3 in Tubulin Instability and Impaired Axon Development Is Mediated by Tau</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4133</span>– <span class="NLM_lpage">4144</span>, <span class="refDoi"> DOI: 10.1096/fj.14-265009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1096%2Ffj.14-265009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=26103986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1SlsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=4133-4144&issue=10&author=Y.+Jooauthor=B.+Schumacherauthor=I.+Landrieuauthor=M.+Bartelauthor=C.+Smet-Noccaauthor=A.+Jangauthor=H.+S.+Choiauthor=N.+L.+Jeonauthor=K.-A.+Changauthor=H.-S.+Kimauthor=C.+Ottmannauthor=Y.-H.+Suh&title=Involvement+of+14%E2%80%933-3+in+Tubulin+Instability+and+Impaired+Axon+Development+Is+Mediated+by+Tau&doi=10.1096%2Ffj.14-265009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Involvement of 14-3-3 in tubulin instability and impaired axon development is mediated by Tau</span></div><div class="casAuthors">Joo, Yuyoung; Schumacher, Benjamin; Landrieu, Isabelle; Bartel, Maria; Smet-Nocca, Caroline; Jang, Ahram; Choi, Hee Soon; Jeon, Noo Li; Chang, Keun-A.; Kim, Hye-Sun; Ottmann, Christian; Suh, Yoo-Hun</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4133-4144</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">14-3-3 Proteins act as adapters that exert their function by interacting with their various protein partners.  14-3-3 Proteins have been implicated in a variety of human diseases including neurodegenerative diseases.  14-3-3 Proteins have recently been reported to be abundant in the neurofibrillary tangles (NFTs) obsd. inside the neurons of brains affected by Alzheimer's disease (AD).  These NFTs are mainly constituted of phosphorylated Tau protein, a microtubule-assocd. protein known to bind 14-3-3.  Despite this indication of 14-3-3 protein involvement in the AD pathogenesis, the role of 14-3-3 in the Tauopathy remains to be clarified.  In the present study, we shed light on the role of 14-3-3 proteins in the mol. pathways leading to Tauopathies.  Overexpression of the 14-3-3s isoform resulted in a disruption of the tubulin cytoskeleton and prevented neuritic outgrowth in neurons.  NMR studies validated the phosphorylated residues pSer214 and pSer324 in Tau as the 2 primary sites for 14-3-3 binding, with the crystal structure of 14-3-3s in complex with Tau-pSer214 and Tau-pSer324 revealing the mol. details of the interaction.  These data suggest a rationale for a possible pharmacol. intervention of the Tau/14-3-3 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhkxjaBskBDrVg90H21EOLACvtfcHk0lgbAaAMFiAmew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1SlsL0%253D&md5=d0346a716b7362c6e69b253d18fea48d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1096%2Ffj.14-265009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.14-265009%26sid%3Dliteratum%253Aachs%26aulast%3DJoo%26aufirst%3DY.%26aulast%3DSchumacher%26aufirst%3DB.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DSmet-Nocca%26aufirst%3DC.%26aulast%3DJang%26aufirst%3DA.%26aulast%3DChoi%26aufirst%3DH.%2BS.%26aulast%3DJeon%26aufirst%3DN.%2BL.%26aulast%3DChang%26aufirst%3DK.-A.%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DSuh%26aufirst%3DY.-H.%26atitle%3DInvolvement%2520of%252014%25E2%2580%25933-3%2520in%2520Tubulin%2520Instability%2520and%2520Impaired%2520Axon%2520Development%2520Is%2520Mediated%2520by%2520Tau%26jtitle%3DFASEB%2520J.%26date%3D2015%26volume%3D29%26issue%3D10%26spage%3D4133%26epage%3D4144%26doi%3D10.1096%2Ffj.14-265009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bozoky, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krzeminski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhandiram, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birtley, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Zahrani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frizzell, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman-Kay, J. D.</span></span> <span> </span><span class="NLM_article-title">Regulatory R Region of the CFTR Chloride Channel Is a Dynamic Integrator of Phospho-Dependent Intra- and Intermolecular Interactions</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">47</span>),  <span class="NLM_fpage">E4427</span>– <span class="NLM_lpage">4436</span>, <span class="refDoi"> DOI: 10.1073/pnas.1315104110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1073%2Fpnas.1315104110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=24191035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCisLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=E4427-4436&issue=47&author=Z.+Bozokyauthor=M.+Krzeminskiauthor=R.+Muhandiramauthor=J.+R.+Birtleyauthor=A.+Al-Zahraniauthor=P.+J.+Thomasauthor=R.+A.+Frizzellauthor=R.+C.+Fordauthor=J.+D.+Forman-Kay&title=Regulatory+R+Region+of+the+CFTR+Chloride+Channel+Is+a+Dynamic+Integrator+of+Phospho-Dependent+Intra-+and+Intermolecular+Interactions&doi=10.1073%2Fpnas.1315104110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory R region of the CFTR chloride channel is a dynamic integrator of phospho-dependent intra- and intermolecular interactions</span></div><div class="casAuthors">Bozoky, Zoltan; Krzeminski, Mickael; Muhandiram, Ranjith; Birtley, James R.; Al-Zahrani, Ateeq; Thomas, Philip J.; Frizzell, Raymond A.; Ford, Robert C.; Forman-Kay, Julie D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">E4427-E4436,SE4427/1-SE4427/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Intrinsically disordered proteins (IDPs) play crucial roles in regulatory processes and often function as protein interaction hubs.  Here, we present a detailed characterization of a full-length disordered hub protein region involved in multiple dynamic complexes.  We performed NMR, CD, and fluorescence binding studies on the nonphosphorylated and highly PKA-phosphorylated human cystic fibrosis transmembrane conductance regulator (CFTR) regulatory region, a ∼200-residue disordered segment involved in phosphorylation-dependent regulation of channel trafficking and gating.  Our data provide evidence for dynamic, phosphorylation-dependent, multisite interactions of various segments of the regulatory region for its intra- and intermol. partners, including the CFTR nucleotide binding domains (NBDs) 1 and 2, a 42-residue peptide from the C terminus of CFTR, the SLC26A3 sulfate transporter and antisigma factor antagonist (STAS) domain, and 14-3-3β.  Because of its large no. of binding partners, multivalent binding of individually weak sites facilitates rapid exchange between free and bound states to allow the regulatory region to engage with different partners and generate a graded or rheostat-like response to phosphorylation.  Our results enrich the understanding of how disordered binding segments interact with multiple targets.  We present structural models consistent with our data that illustrate this dynamic aspect of phospho-regulation of CFTR by the disordered regulatory region.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyg0jY6--uubVg90H21EOLACvtfcHk0lgbAaAMFiAmew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCisLzM&md5=1f8155a6eb49fbaa8fdc5677cf41fb0a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1315104110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1315104110%26sid%3Dliteratum%253Aachs%26aulast%3DBozoky%26aufirst%3DZ.%26aulast%3DKrzeminski%26aufirst%3DM.%26aulast%3DMuhandiram%26aufirst%3DR.%26aulast%3DBirtley%26aufirst%3DJ.%2BR.%26aulast%3DAl-Zahrani%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DP.%2BJ.%26aulast%3DFrizzell%26aufirst%3DR.%2BA.%26aulast%3DFord%26aufirst%3DR.%2BC.%26aulast%3DForman-Kay%26aufirst%3DJ.%2BD.%26atitle%3DRegulatory%2520R%2520Region%2520of%2520the%2520CFTR%2520Chloride%2520Channel%2520Is%2520a%2520Dynamic%2520Integrator%2520of%2520Phospho-Dependent%2520Intra-%2520and%2520Intermolecular%2520Interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26issue%3D47%26spage%3DE4427%26epage%3D4436%26doi%3D10.1073%2Fpnas.1315104110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stevers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S. F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Scheppingen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, R. M. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlile, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Characterization and Small-Molecule Stabilization of the Multisite Tandem Binding between 14–3-3 and the R Domain of CFTR</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">E1152</span>– <span class="NLM_lpage">1161</span>, <span class="refDoi"> DOI: 10.1073/pnas.1516631113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1073%2Fpnas.1516631113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=26888287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC28XislaqsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=E1152-1161&issue=9&author=L.+M.+Steversauthor=C.+V.+Lamauthor=S.+F.+R.+Leysenauthor=F.+A.+Meijerauthor=D.+S.+van+Scheppingenauthor=R.+M.+J.+M.+de+Vriesauthor=G.+W.+Carlileauthor=L.+G.+Milroyauthor=D.+Y.+Thomasauthor=L.+Brunsveldauthor=C.+Ottmann&title=Characterization+and+Small-Molecule+Stabilization+of+the+Multisite+Tandem+Binding+between+14%E2%80%933-3+and+the+R+Domain+of+CFTR&doi=10.1073%2Fpnas.1516631113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization and small-molecule stabilization of the multisite tandem binding between 14-3-3 and the R domain of CFTR</span></div><div class="casAuthors">Stevers, Loes M.; Lam, Chan V.; Leysen, Seppe F. R.; Meijer, Femke A.; van Scheppingen, Daphne S.; de Vries, Rens M. J. M.; Carlile, Graeme W.; Milroy, Lech G.; Thomas, David Y.; Brunsveld, Luc; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">E1152-E1161</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cystic fibrosis is a fatal genetic disease, most frequently caused by the retention of the CFTR (cystic fibrosis transmembrane conductance regulator) mutant protein in the endoplasmic reticulum (ER).  The binding of the 14-3-3 protein to the CFTR regulatory (R) domain has been found to enhance CFTR trafficking to the plasma membrane.  To define the mechanism of action of this protein-protein interaction, we have examd. the interaction in vitro.  The disordered multiphosphorylated R domain contains nine different 14-3-3 binding motifs.  Furthermore, the 14-3-3 protein forms a dimer contg. two amphipathic grooves that can potentially bind these phosphorylated motifs.  This results in a no. of possible binding mechanisms between these two proteins.  Using multiple biochem. assays and crystal structures, we show that the interaction between them is governed by two binding sites: the key binding site of CFTR (pS768) occupies one groove of the 14-3-3 dimer, and a weaker, secondary binding site occupies the other binding groove.  We show that fusicoccin-A, a natural-product tool compd. used in studies of 14-3-3 biol., can stabilize the interaction between 14-3-3 and CFTR by selectively interacting with a secondary binding motif of CFTR (pS753).  The stabilization of this interaction stimulates the trafficking of mutant CFTR to the plasma membrane.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXJUVLOg21FbVg90H21EOLACvtfcHk0lgbAaAMFiAmew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislaqsLo%253D&md5=148d49c057fc8635f9436b3735f86e68</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1516631113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1516631113%26sid%3Dliteratum%253Aachs%26aulast%3DStevers%26aufirst%3DL.%2BM.%26aulast%3DLam%26aufirst%3DC.%2BV.%26aulast%3DLeysen%26aufirst%3DS.%2BF.%2BR.%26aulast%3DMeijer%26aufirst%3DF.%2BA.%26aulast%3Dvan%2BScheppingen%26aufirst%3DD.%2BS.%26aulast%3Dde%2BVries%26aufirst%3DR.%2BM.%2BJ.%2BM.%26aulast%3DCarlile%26aufirst%3DG.%2BW.%26aulast%3DMilroy%26aufirst%3DL.%2BG.%26aulast%3DThomas%26aufirst%3DD.%2BY.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DCharacterization%2520and%2520Small-Molecule%2520Stabilization%2520of%2520the%2520Multisite%2520Tandem%2520Binding%2520between%252014%25E2%2580%25933-3%2520and%2520the%2520R%2520Domain%2520of%2520CFTR%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26issue%3D9%26spage%3DE1152%26epage%3D1161%26doi%3D10.1073%2Fpnas.1516631113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunziker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span> <span> </span><span class="NLM_article-title">A Role for TAZ in Migration, Invasion, and Tumorigenesis of Breast Cancer Cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2592</span>– <span class="NLM_lpage">2598</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-2696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1158%2F0008-5472.CAN-07-2696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=18413727" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1GitL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=2592-2598&issue=8&author=S.+W.+Chanauthor=C.+J.+Limauthor=K.+Guoauthor=C.+P.+Ngauthor=I.+Leeauthor=W.+Hunzikerauthor=Q.+Zengauthor=W.+Hong&title=A+Role+for+TAZ+in+Migration%2C+Invasion%2C+and+Tumorigenesis+of+Breast+Cancer+Cells&doi=10.1158%2F0008-5472.CAN-07-2696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A Role for TAZ in Migration, Invasion, and Tumorigenesis of Breast Cancer Cells</span></div><div class="casAuthors">Chan, Siew Wee; Lim, Chun Jye; Guo, Ke; Ng, Chee Peng; Lee, Ian; Hunziker, Walter; Zeng, Qi; Hong, Wanjin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2592-2598</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">TAZ (WWTR1), identified as a 14-3-3 binding protein with a PDZ binding motif, modulates mesenchymal stem cell differentiation.  We now show that TAZ plays a crit. role in the migration, invasion, and tumorigenesis of breast cancer cells.  TAZ is conspicuously expressed in human breast cancer cell lines in which its expression levels generally correlate with the invasiveness of cancer cells.  Overexpression of TAZ in low-expressing MCF10A cells causes morphol. changes characteristic of cell transformation and promotes cell migration and invasion.  Conversely, RNA interference-mediated knockdown of TAZ expression in MCF7 and Hs578T cells reduces cell migration and invasion.  TAZ knockdown in MCF7 cells also retards anchorage-independent growth in soft agar and tumorigenesis in nude mice.  Significantly, TAZ is overexpressed in ∼20% of breast cancer samples.  These results indicate that TAZ plays a role in the migration, invasion, and tumorigenesis of breast cancer cells and thus presents a novel target for the detection and treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMyv6KhRaRN7Vg90H21EOLACvtfcHk0liiLm1fxUCuoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1GitL8%253D&md5=4b58785b0ab40c38ba0ff8c168d4a609</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-2696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-2696%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DS.%2BW.%26aulast%3DLim%26aufirst%3DC.%2BJ.%26aulast%3DGuo%26aufirst%3DK.%26aulast%3DNg%26aufirst%3DC.%2BP.%26aulast%3DLee%26aufirst%3DI.%26aulast%3DHunziker%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DHong%26aufirst%3DW.%26atitle%3DA%2520Role%2520for%2520TAZ%2520in%2520Migration%252C%2520Invasion%252C%2520and%2520Tumorigenesis%2520of%2520Breast%2520Cancer%2520Cells%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26issue%3D8%26spage%3D2592%26epage%3D2598%26doi%3D10.1158%2F0008-5472.CAN-07-2696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.-H.</span></span> <span> </span><span class="NLM_article-title">14–3-3 Sigma Positively Regulates P53 and Suppresses Tumor Growth</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">7096</span>– <span class="NLM_lpage">7107</span>, <span class="refDoi"> DOI: 10.1128/MCB.23.20.7096-7107.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1128%2FMCB.23.20.7096-7107.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=14517281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotVSrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=7096-7107&issue=20&author=H.-Y.+Yangauthor=Y.-Y.+Wenauthor=C.-H.+Chenauthor=G.+Lozanoauthor=M.-H.+Lee&title=14%E2%80%933-3+Sigma+Positively+Regulates+P53+and+Suppresses+Tumor+Growth&doi=10.1128%2FMCB.23.20.7096-7107.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">14-3-3σ positively regulates p53 and suppresses tumor growth</span></div><div class="casAuthors">Yang, Heng-Yin; Wen, Yu-Ye; Chen, Chih-Hsin; Lozano, Guillermina; Lee, Mong-Hong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7096-7107</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The 14-3-3σ (sigma) protein, a neg. regulator of the cell cycle, is a human mammary epithelium-specific marker that is downregulated in transformed mammary carcinoma cells.  It has also been identified as a p53-inducible gene product involved in cell cycle checkpoint control after DNA damage.  Although 14-3-3σ is linked to p53-regulated cell cycle checkpoint control, detailed mechanisms of how cell cycle regulation occurs remain unclear.  Decreased expression of 14-3-3σ was recently reported in several types of carcinomas, further suggesting that the neg. regulatory role of 14-3-3σ in the cell cycle is compromised during tumorigenesis.  However, this possible tumor-suppressive role of 14-3-3σ has not yet been characterized.  Here, the authors studied the link between 14-3-3σ activities and p53 regulation.  The authors found that 14-3-3σ interacted with p53 in response to the DNA-damaging agent adriamycin.  Importantly, 14-3-3σ expression led to stabilized expression of p53.  In studying the mol. mechanism of this increased stabilization of p53, the authors found that 14-3-3σ antagonized the biol. functions of Mdm2 by blocking Mdm2-mediated p53 ubiquitination and nuclear export.  In addn., the authors found that 14-3-3σ facilitated the oligomerization of p53 and enhanced p53's transcriptional activity.  As a target gene of p53, 14-3-3σ appears to have a pos. feedback effect on p53 activity.  Significantly, the authors also showed that overexpression of 14-3-3σ inhibited oncogene-activated tumorigenicity in a tetracycline-regulated 14-3-3σ system.  These results defined an important p53 regulatory loop and suggested that 14-3-3σ expression can be considered for therapeutic intervention in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtjTAMn0T1XbVg90H21EOLACvtfcHk0liiLm1fxUCuoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotVSrtbo%253D&md5=3b89488426cb6f8d9b60c9c402a24999</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.20.7096-7107.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.20.7096-7107.2003%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DH.-Y.%26aulast%3DWen%26aufirst%3DY.-Y.%26aulast%3DChen%26aufirst%3DC.-H.%26aulast%3DLozano%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DM.-H.%26atitle%3D14%25E2%2580%25933-3%2520Sigma%2520Positively%2520Regulates%2520P53%2520and%2520Suppresses%2520Tumor%2520Growth%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2003%26volume%3D23%26issue%3D20%26spage%3D7096%26epage%3D7107%26doi%3D10.1128%2FMCB.23.20.7096-7107.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopalan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sade, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsley, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span> <span> </span><span class="NLM_article-title">Mechanistic Differences in the Transcriptional Activation of P53 by 14–3-3 Isoforms</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">893</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp1041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1093%2Fnar%2Fgkp1041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=19933256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSlurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=893-906&issue=3&author=S.+Rajagopalanauthor=R.+S.+Sadeauthor=F.+M.+Townsleyauthor=A.+R.+Fersht&title=Mechanistic+Differences+in+the+Transcriptional+Activation+of+P53+by+14%E2%80%933-3+Isoforms&doi=10.1093%2Fnar%2Fgkp1041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic differences in the transcriptional activation of p53 by 14-3-3 isoforms</span></div><div class="casAuthors">Rajagopalan, Sridharan; Sade, Robert S.; Townsley, Fiona M.; Fersht, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">893-906</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">P53 maintains genome integrity by initiating the transcription of genes involved in cell-cycle arrest, senescence, apoptosis and DNA repair.  The activity of p53 is regulated by both post-translational modifications and protein-protein interactions. p53 that has been phosphorylated at S366, S378 and T387 binds 14-3-3 proteins in vitro.  Here, we show that these sites are potential 14-3-3 binding sites in vivo.  Epsilon (ε) and gamma (γ) isoforms required phosphorylation at either of these sites for efficient interaction with p53, while for sigma (σ) and tau (τ) these sites are dispensable.  Further, σ and τ bound more weakly to p53 C-terminal phosphopeptides than did ε and γ.  However, the four isoforms bound tightly to di-phosphorylated p53 C-terminal peptides than did the mono-phosphorylated counterparts.  Interestingly, all the isoforms studied transcriptionally activated wild-type p53. σ and τ stabilized p53 levels in cells, while ε and γ stimulated p53-DNA binding activity in vitro.  Overall, the results suggest that structurally and functionally similar 14-3-3 isoforms may exert their regulatory potential on p53 through different mechanisms.  We discuss the isoform-specific roles of 14-3-3 in p53 stabilization and activation of specific-DNA binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRVozP2__gyLVg90H21EOLACvtfcHk0liiLm1fxUCuoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSlurc%253D&md5=11f92e04e5017515a4bbfc1bd65bfd1d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp1041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp1041%26sid%3Dliteratum%253Aachs%26aulast%3DRajagopalan%26aufirst%3DS.%26aulast%3DSade%26aufirst%3DR.%2BS.%26aulast%3DTownsley%26aufirst%3DF.%2BM.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3DMechanistic%2520Differences%2520in%2520the%2520Transcriptional%2520Activation%2520of%2520P53%2520by%252014%25E2%2580%25933-3%2520Isoforms%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26issue%3D3%26spage%3D893%26epage%3D906%26doi%3D10.1093%2Fnar%2Fgkp1041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopalan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaulent, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veprintsev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span> <span> </span><span class="NLM_article-title">14–3-3 Activation of DNA Binding of P53 by Enhancing Its Association into Tetramers</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">5983</span>– <span class="NLM_lpage">5991</span>, <span class="refDoi"> DOI: 10.1093/nar/gkn598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1093%2Fnar%2Fgkn598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=18812399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Kgu7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=5983-5991&issue=18&author=S.+Rajagopalanauthor=A.+M.+Jaulentauthor=M.+Wellsauthor=D.+B.+Veprintsevauthor=A.+R.+Fersht&title=14%E2%80%933-3+Activation+of+DNA+Binding+of+P53+by+Enhancing+Its+Association+into+Tetramers&doi=10.1093%2Fnar%2Fgkn598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">14-3-3 activation of DNA binding of p53 by enhancing its association into tetramers</span></div><div class="casAuthors">Rajagopalan, Sridharan; Jaulent, Agnes M.; Wells, Mark; Veprintsev, Dmitry B.; Fersht, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5983-5991</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Activation of the tumor suppressor p53 on DNA damage involves post-translational modification by phosphorylation and acetylation.  Phosphorylation of certain residues is crit. for p53 stabilization and plays an important role in DNA-binding activity.  The 14-3-3 family of proteins activates the DNA-binding affinity of p53 upon stress by binding to a site in its intrinsically disordered C-terminal domain contg. a phosphorylated serine at 378.  We have screened various p53 C-terminal phosphorylated peptides for binding to two different isoforms of 14-3-3, ε and γ.  We found that phosphorylation at either S366 or T387 caused even tighter binding to 14-3-3.  We made by semi-synthesis a tetrameric construct comprised of the tetramerization plus C-terminal domains of p53 that was phosphorylated on S366, S378 and T387.  It bound 10 times tighter than did the monomeric counterpart to dimeric 14-3-3.  We showed indirectly from binding curves and directly from fluorescence-detection anal. ultracentrifugation that 14-3-3 enhanced the binding of sequence-specific DNA to p53 by causing p53 dimers to form tetramers at lower concns.  If the in vitro data extrapolate to in vivo, then it is an attractive hypothesis that p53 activity may be subject to control by accessory proteins lowering its tetramer-dimer dissocn. const. from its normal value of 120-150 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8YjWdbhc0eLVg90H21EOLACvtfcHk0lgAUi7o5MSSOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Kgu7fJ&md5=277d9a9c81ff6b592c444be0cf2f4194</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn598%26sid%3Dliteratum%253Aachs%26aulast%3DRajagopalan%26aufirst%3DS.%26aulast%3DJaulent%26aufirst%3DA.%2BM.%26aulast%3DWells%26aufirst%3DM.%26aulast%3DVeprintsev%26aufirst%3DD.%2BB.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3D14%25E2%2580%25933-3%2520Activation%2520of%2520DNA%2520Binding%2520of%2520P53%2520by%2520Enhancing%2520Its%2520Association%2520into%2520Tetramers%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26issue%3D18%26spage%3D5983%26epage%3D5991%26doi%3D10.1093%2Fnar%2Fgkn598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carriere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheerathodi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rush, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballif, B. A.</span></span> <span> </span><span class="NLM_article-title">RSK Phosphorylates SOS1 Creating 14–3-3-Docking Sites and Negatively Regulating MAPK Activation</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>447</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1042/BJ20120938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FBJ20120938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22827337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlCjsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=447&publication_year=2012&pages=159-166&issue=1&author=M.+Sahaauthor=A.+Carriereauthor=M.+Cheerathodiauthor=X.+Zhangauthor=G.+Lavoieauthor=J.+Rushauthor=P.+P.+Rouxauthor=B.+A.+Ballif&title=RSK+Phosphorylates+SOS1+Creating+14%E2%80%933-3-Docking+Sites+and+Negatively+Regulating+MAPK+Activation&doi=10.1042%2FBJ20120938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">RSK phosphorylates SOS1 creating 14-3-3-docking sites and negatively regulating MAPK activation</span></div><div class="casAuthors">Saha, Madhurima; Carriere, Audrey; Cheerathodi, Mujeeburahiman; Zhang, Xiaocui; Lavoie, Genevieve; Rush, John; Roux, Philippe P.; Ballif, Bryan A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">447</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">159-166</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The extent and duration of MAPK (mitogen-activated protein kinase) signalling govern a diversity of normal and aberrant cellular outcomes.  Genetic and pharmacol. disruption of the MAPK-activated kinase RSK (ribosomal S6 kinase) leads to elevated MAPK activity indicative of a RSK-dependent neg. feedback loop.  Using biochem., pharmacol. and quant. MS approaches we show that RSK phosphorylates the Ras activator SOS1 (Son of Sevenless homolog 1) in cultured cells on two C-terminal residues, Ser1134 and Ser1161.  Furthermore, we find that RSK-dependent SOS1 phosphorylation creates 14-3-3-binding sites.  We show that mutating Ser1134 and Ser1161 disrupts 14-3-3 binding and modestly increases and extends MAPK activation.  Together these data suggest that one mechanism whereby RSK neg. regulates MAPK activation is via site-specific SOS1 phosphorylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4pH9etGvwsbVg90H21EOLACvtfcHk0lgAUi7o5MSSOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlCjsLrF&md5=c39f673b533b03e22e96b67e84dc2360</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1042%2FBJ20120938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20120938%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DM.%26aulast%3DCarriere%26aufirst%3DA.%26aulast%3DCheerathodi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLavoie%26aufirst%3DG.%26aulast%3DRush%26aufirst%3DJ.%26aulast%3DRoux%26aufirst%3DP.%2BP.%26aulast%3DBallif%26aufirst%3DB.%2BA.%26atitle%3DRSK%2520Phosphorylates%2520SOS1%2520Creating%252014%25E2%2580%25933-3-Docking%2520Sites%2520and%2520Negatively%2520Regulating%2520MAPK%2520Activation%26jtitle%3DBiochem.%2520J.%26date%3D2012%26volume%3D447%26issue%3D1%26spage%3D159%26epage%3D166%26doi%3D10.1042%2FBJ20120938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centorrino, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Structural Characterization of 14–3-3ζ in Complex with the Human Son of Sevenless Homolog 1 (SOS1)</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>202</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">215</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2018.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.jsb.2018.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=29408703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Ogu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=202&publication_year=2018&pages=210-215&issue=3&author=A.+Balloneauthor=F.+Centorrinoauthor=M.+Wolterauthor=C.+Ottmann&title=Structural+Characterization+of+14%E2%80%933-3%CE%B6+in+Complex+with+the+Human+Son+of+Sevenless+Homolog+1+%28SOS1%29&doi=10.1016%2Fj.jsb.2018.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structural characterization of 14-3-3ζ in complex with the human Son of sevenless homolog 1 (SOS1)</span></div><div class="casAuthors">Ballone, Alice; Centorrino, Federica; Wolter, Madita; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">202</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">210-215</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The deviant Ras activation machinery is found in approx. 30% of all human cancers.  SOS1 is an important protagonist of this pathway that plays a key-role in aberrant cell proliferation and differentiation.  Interaction of SOS1 with 14-3-3 proteins modulates SOS1 activity in Ras-MAPK signaling.  In the present study, we analyze the 14-3-3/SOS1 protein-protein interaction (PPI) by different biochem. assays and report the high resoln. crystal structure of a 13-mer motif of SOS1 bound to 14-3-3 ζ.  These structural and functional insights are important for the evaluation of this PPI interface for small-mol. stabilization as a new starting point for modulating the Ras-Raf-MAPK pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs8EIra65oLLVg90H21EOLACvtfcHk0lgAUi7o5MSSOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Ogu7w%253D&md5=451fa5e9f1d5c04e928b0edc99916913</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2018.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2018.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DBallone%26aufirst%3DA.%26aulast%3DCentorrino%26aufirst%3DF.%26aulast%3DWolter%26aufirst%3DM.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DStructural%2520Characterization%2520of%252014%25E2%2580%25933-3%25CE%25B6%2520in%2520Complex%2520with%2520the%2520Human%2520Son%2520of%2520Sevenless%2520Homolog%25201%2520%2528SOS1%2529%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D202%26issue%3D3%26spage%3D210%26epage%3D215%26doi%3D10.1016%2Fj.jsb.2018.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ory, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrads, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenstra, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, D. K.</span></span> <span> </span><span class="NLM_article-title">Protein Phosphatase 2A Positively Regulates Ras Signaling by Dephosphorylating KSR1 and Raf-1 on Critical 14–3-3 Binding Sites</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">1356</span>– <span class="NLM_lpage">1364</span>, <span class="refDoi"> DOI: 10.1016/S0960-9822(03)00535-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2FS0960-9822%2803%2900535-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=12932319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmslentLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=1356-1364&issue=16&author=S.+Oryauthor=M.+Zhouauthor=T.+P.+Conradsauthor=T.+D.+Veenstraauthor=D.+K.+Morrison&title=Protein+Phosphatase+2A+Positively+Regulates+Ras+Signaling+by+Dephosphorylating+KSR1+and+Raf-1+on+Critical+14%E2%80%933-3+Binding+Sites&doi=10.1016%2FS0960-9822%2803%2900535-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Phosphatase 2A Positively Regulates Ras Signaling by Dephosphorylating KSR1 and Raf-1 on Critical 14-3-3 Binding Sites</span></div><div class="casAuthors">Ory, Stephane; Zhou, Ming; Conrads, Thomas P.; Veenstra, Timothy D.; Morrison, Deborah K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1356-1364</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: Kinase Suppressor of Ras (KSR) is a conserved component of the Ras pathway that acts as a mol. scaffold to facilitate signal transmission through the MAPK cascade.  Although recruitment of KSR1 from the cytosol to the plasma membrane is required for its scaffolding function, the precise mechanism(s) regulating the translocation of KSR1 have not been fully elucidated.  Results: Using mass spectrometry to analyze the KSR1-scaffolding complex, we identify the serine/threonine protein phosphatase PP2A as a KSR1-assocd. protein and show that PP2A is a crit. regulator of KSR1 activity.  We find that the enzymic core subunits of PP2A (PR65A and catalytic C) constitutively assoc. with the N-terminal domain of KSR1, whereas binding of the regulatory PR55B subunit is induced by growth factor treatment.  Specific inhibition of PP2A activity prevents the growth factor-induced dephosphorylation event involved in the membrane recruitment of KSR1 and blocks the activation of KSR1-assocd. MEK and ERK.  Moreover, we find that PP2A activity is required for activation of the Raf-1 kinase and that both Raf and KSR1 must be dephosphorylated by PP2A on crit. regulatory 14-3-3 binding sites for KSR1 to promote MAPK pathway activation.  Conclusions: These findings identify KSR1 as novel substrate of PP2A and demonstrate the inducible dephosphorylation of KSR1 in response to Ras pathway activation.  Further, these results elucidate a common regulatory mechanism for KSR1 and Raf-1 whereby their localization and activity are modulated by the PP2A-mediated dephosphorylation of crit. 14-3-3 binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgdlQTl7RWhbVg90H21EOLACvtfcHk0lhJLnpjtvMAAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmslentLk%253D&md5=b49148bfad0b58ed82204dfdf32a2abd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2803%2900535-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252803%252900535-9%26sid%3Dliteratum%253Aachs%26aulast%3DOry%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DConrads%26aufirst%3DT.%2BP.%26aulast%3DVeenstra%26aufirst%3DT.%2BD.%26aulast%3DMorrison%26aufirst%3DD.%2BK.%26atitle%3DProtein%2520Phosphatase%25202A%2520Positively%2520Regulates%2520Ras%2520Signaling%2520by%2520Dephosphorylating%2520KSR1%2520and%2520Raf-1%2520on%2520Critical%252014%25E2%2580%25933-3%2520Binding%2520Sites%26jtitle%3DCurr.%2520Biol.%26date%3D2003%26volume%3D13%26issue%3D16%26spage%3D1356%26epage%3D1364%26doi%3D10.1016%2FS0960-9822%2803%2900535-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Lorenzo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of BRAF by AMPK Impairs BRAF-KSR1 Association and Cell Proliferation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.08.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.molcel.2013.08.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=24095280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOls73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=161-172&issue=2&author=C.-H.+Shenauthor=P.+Yuanauthor=R.+Perez-Lorenzoauthor=Y.+Zhangauthor=S.+X.+Leeauthor=Y.+Ouauthor=J.+M.+Asaraauthor=L.+C.+Cantleyauthor=B.+Zheng&title=Phosphorylation+of+BRAF+by+AMPK+Impairs+BRAF-KSR1+Association+and+Cell+Proliferation&doi=10.1016%2Fj.molcel.2013.08.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of BRAF by AMPK Impairs BRAF-KSR1 Association and Cell Proliferation</span></div><div class="casAuthors">Shen, Che-Hung; Yuan, Ping; Perez-Lorenzo, Rolando; Zhang, Yaqing; Lee, Sze Xian; Ou, Yang; Asara, John M.; Cantley, Lewis C.; Zheng, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">161-172</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">BRAF is an oncogenic protein kinase that drives cell growth and proliferation through the MEK-ERK signaling pathway.  BRAF inhibitors have demonstrated antitumor efficacy in melanoma therapy but have also been found to be assocd. with the development of cutaneous squamous cell carcinomas (cSCCs) in certain patients.  Here, we report that BRAF is phosphorylated at Ser729 by AMP-activated protein kinase (AMPK), a crit. energy sensor.  This phosphorylation promotes the assocn. of BRAF with 14-3-3 proteins and disrupts its interaction with the KSR1 scaffolding protein, leading to attenuation of the MEK-ERK signaling.  We also show that phosphorylation of BRAF by AMPK impairs keratinocyte cell proliferation and cell-cycle progression.  Furthermore, AMPK activation attenuates BRAF inhibitor-induced ERK hyperactivation in keratinocytes and epidermal hyperplasia in mouse skin.  Our findings reveal a mechanism for regulating BRAF signaling in response to energy stress and suggest a strategy for preventing the development of cSCCs assocd. with BRAF-targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEDKxlkrxbWLVg90H21EOLACvtfcHk0lhJLnpjtvMAAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOls73P&md5=f0af44261dee02ae14626ad29f4fa9b0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.08.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.08.044%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DC.-H.%26aulast%3DYuan%26aufirst%3DP.%26aulast%3DPerez-Lorenzo%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DS.%2BX.%26aulast%3DOu%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DZheng%26aufirst%3DB.%26atitle%3DPhosphorylation%2520of%2520BRAF%2520by%2520AMPK%2520Impairs%2520BRAF-KSR1%2520Association%2520and%2520Cell%2520Proliferation%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D52%26issue%3D2%26spage%3D161%26epage%3D172%26doi%3D10.1016%2Fj.molcel.2013.08.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dumaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marais, R.</span></span> <span> </span><span class="NLM_article-title">Protein Kinase A Blocks Raf-1 Activity by Stimulating 14–3-3 Binding and Blocking Raf-1 Interaction with Ras</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span> (<span class="NLM_issue">32</span>),  <span class="NLM_fpage">29819</span>– <span class="NLM_lpage">29823</span>, <span class="refDoi"> DOI: 10.1074/jbc.C300182200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1074%2Fjbc.C300182200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=12801936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvFOnsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=29819-29823&issue=32&author=N.+Dumazauthor=R.+Marais&title=Protein+Kinase+A+Blocks+Raf-1+Activity+by+Stimulating+14%E2%80%933-3+Binding+and+Blocking+Raf-1+Interaction+with+Ras&doi=10.1074%2Fjbc.C300182200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase A Blocks Raf-1 Activity by Stimulating 14-3-3 Binding and Blocking Raf-1 Interaction with Ras</span></div><div class="casAuthors">Dumaz, Nicolas; Marais, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">29819-29823</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">CAMP blocks Raf-1 activation by stimulating its phosphorylation on serine 43 (Ser43), serine 233 (Ser233), and serine 259 (Ser259).  We show here that phosphorylation of all three sites blocks Raf-1 binding to Ras·GTP in vivo and that cAMP stimulates binding of 14-3-3 proteins to Ser233 and Ser259.  We also show that Raf-1 and protein kinase A (PKA) form a complex in vivo that is disrupted by cAMP and that ablation of PKA by use of small interfering RNA blocks phosphorylation by cAMP.  The ability of PKA to block Raf-1 activation is ablated by the PKA inhibitor H89.  These studies suggest that Raf-1 and cAMP form a signaling complex in cells.  Upon activation of PKA, Raf-1 is phosphorylated and 14-3-3 binds, blocking Raf-1 recruitment to the plasma membrane and preventing its activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMN_lumn0pG7Vg90H21EOLACvtfcHk0lhJLnpjtvMAAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvFOnsLY%253D&md5=a6c108371f72eb18e311366f6b7ca311</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C300182200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C300182200%26sid%3Dliteratum%253Aachs%26aulast%3DDumaz%26aufirst%3DN.%26aulast%3DMarais%26aufirst%3DR.%26atitle%3DProtein%2520Kinase%2520A%2520Blocks%2520Raf-1%2520Activity%2520by%2520Stimulating%252014%25E2%2580%25933-3%2520Binding%2520and%2520Blocking%2520Raf-1%2520Interaction%2520with%2520Ras%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26issue%3D32%26spage%3D29819%26epage%3D29823%26doi%3D10.1074%2Fjbc.C300182200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Paula, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozóky, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman-Kay, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frizzell, R. A.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation-Dependent 14–3-3 Protein Interactions Regulate CFTR Biogenesis</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">996</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1091/mbc.e11-08-0662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1091%2Fmbc.e11-08-0662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22278744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVOgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=996-1009&issue=6&author=X.+Liangauthor=A.+C.+Da+Paulaauthor=Z.+Boz%C3%B3kyauthor=H.+Zhangauthor=C.+A.+Bertrandauthor=K.+W.+Petersauthor=J.+D.+Forman-Kayauthor=R.+A.+Frizzell&title=Phosphorylation-Dependent+14%E2%80%933-3+Protein+Interactions+Regulate+CFTR+Biogenesis&doi=10.1091%2Fmbc.e11-08-0662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation-dependent 14-3-3 protein interactions regulate CFTR biogenesis</span></div><div class="casAuthors">Liang, Xiubin; Da Paula, Ana Carina; Bozoky, Zoltan; Zhang, Hui; Bertrand, Carol A.; Peters, Kathryn W.; Forman-Kay, Julie D.; Frizzell, Raymond A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">996-1009</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Cystic fibrosis transmembrane conductance regulator (CFTR) is a cAMP/protein kinase A (PKA)-regulated chloride channel whose phosphorylation controls anion secretion across epithelial cell apical membranes.  We examd. the hypothesis that cAMP/PKA stimulation regulates CFTR biogenesis posttranslationally, based on predicted 14-3-3 binding motifs within CFTR and forskolin-induced CFTR expression.  The 14-3-3β, γ, and ε isoforms were expressed in airway cells and interacted with CFTR in coimmunopptn. assays.  Forskolin stimulation (15 min) increased 14-3-3β and ε binding to immature and mature CFTR (bands B and C), and 14-3-3 overexpression increased CFTR bands B and C and cell surface band C.  In pulse-chase expts., 14-3-3β increased the synthesis of immature CFTR, reduced its degrdn. rate, and increased conversion of immature to mature CFTR.  Conversely, 14-3-3β knockdown decreased CFTR B and C bands (70 and 55%) and elicited parallel redns. in cell surface CFTR and forskolin-stimulated anion efflux.  In vitro, 14-3-3β interacted with the CFTR regulatory region, and by NMR anal., this interaction occurred at known PKA phosphorylated sites.  In coimmunopptn. assays, forskolin stimulated the CFTR/14-3-3β interaction while reducing CFTR's interaction with coat protein complex 1 (COP1).  Thus 14-3-3 binding to phosphorylated CFTR augments its biogenesis by reducing retrograde retrieval of CFTR to the endoplasmic reticulum.  This mechanism permits cAMP/PKA stimulation to make more CFTR available for anion secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBTsCTTTWAQ7Vg90H21EOLACvtfcHk0lhJLnpjtvMAAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVOgtrw%253D&md5=1faad75eaf1ebf2e232e23293b170e82</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e11-08-0662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e11-08-0662%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DDa%2BPaula%26aufirst%3DA.%2BC.%26aulast%3DBoz%25C3%25B3ky%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DBertrand%26aufirst%3DC.%2BA.%26aulast%3DPeters%26aufirst%3DK.%2BW.%26aulast%3DForman-Kay%26aufirst%3DJ.%2BD.%26aulast%3DFrizzell%26aufirst%3DR.%2BA.%26atitle%3DPhosphorylation-Dependent%252014%25E2%2580%25933-3%2520Protein%2520Interactions%2520Regulate%2520CFTR%2520Biogenesis%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2012%26volume%3D23%26issue%3D6%26spage%3D996%26epage%3D1009%26doi%3D10.1091%2Fmbc.e11-08-0662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morquette, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madwar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, A. E.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Stabilization of 14–3-3 Protein-Protein Interactions Stimulates Axon Regeneration</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1082</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2017.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.neuron.2017.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=28279353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktVylu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=1082&issue=5&author=A.+Kaplanauthor=B.+Morquetteauthor=A.+Kronerauthor=S.+Leongauthor=C.+Madwarauthor=R.+Sanzauthor=S.+L.+Banerjeeauthor=J.+Antelauthor=N.+Bissonauthor=S.+Davidauthor=A.+E.+Fournier&title=Small-Molecule+Stabilization+of+14%E2%80%933-3+Protein-Protein+Interactions+Stimulates+Axon+Regeneration&doi=10.1016%2Fj.neuron.2017.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Stabilization of 14-3-3 Protein-Protein Interactions Stimulates Axon Regeneration</span></div><div class="casAuthors">Kaplan, Andrew; Morquette, Barbara; Kroner, Antje; Leong, Soo Yuen; Madwar, Carolin; Sanz, Ricardo; Banerjee, Sara L.; Antel, Jack; Bisson, Nicolas; David, Samuel; Fournier, Alyson E.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1082-1093.e5</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Damaged central nervous system (CNS) neurons have a poor ability to spontaneously regenerate, causing persistent functional deficits after injury.  Therapies that stimulate axon growth are needed to repair CNS damage.  14-3-3 Adaptors are hub proteins that are attractive targets to manipulate cell signaling.  We identify a pos. role for 14-3-3s in axon growth and uncover a developmental regulation of the phosphorylation and function of 14-3-3s.  We show that fusicoccin-A (FC-A), a small-mol. stabilizer of 14-3-3 protein-protein interactions, stimulates axon growth in vitro and regeneration in vivo.  We show that FC-A stabilizes a complex between 14-3-3 and the stress response regulator GCN1, inducing GCN1 turnover and neurite outgrowth.  These findings show that 14-3-3 adaptor protein complexes are druggable targets and identify a new class of small mols. that may be further optimized for the repair of CNS damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGtMq7zLhfkrVg90H21EOLACvtfcHk0lhWqhUOB995mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktVylu7o%253D&md5=b06351a1f1093cbc2a0cb260b53d9d26</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2017.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2017.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DKaplan%26aufirst%3DA.%26aulast%3DMorquette%26aufirst%3DB.%26aulast%3DKroner%26aufirst%3DA.%26aulast%3DLeong%26aufirst%3DS.%26aulast%3DMadwar%26aufirst%3DC.%26aulast%3DSanz%26aufirst%3DR.%26aulast%3DBanerjee%26aufirst%3DS.%2BL.%26aulast%3DAntel%26aufirst%3DJ.%26aulast%3DBisson%26aufirst%3DN.%26aulast%3DDavid%26aufirst%3DS.%26aulast%3DFournier%26aufirst%3DA.%2BE.%26atitle%3DSmall-Molecule%2520Stabilization%2520of%252014%25E2%2580%25933-3%2520Protein-Protein%2520Interactions%2520Stimulates%2520Axon%2520Regeneration%26jtitle%3DNeuron%26date%3D2017%26volume%3D93%26issue%3D5%26spage%3D1082%26doi%3D10.1016%2Fj.neuron.2017.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaspert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schauer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weyand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermeeren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duby, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigaud, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oecking, C.</span></span> <span> </span><span class="NLM_article-title">Structure of a 14–3-3 Coordinated Hexamer of the Plant Plasma Membrane H+ -ATPase by Combining X-Ray Crystallography and Electron Cryomicroscopy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.molcel.2006.12.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=17289589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=427-440&issue=3&author=C.+Ottmannauthor=S.+Marcoauthor=N.+Jaspertauthor=C.+Marconauthor=N.+Schauerauthor=M.+Weyandauthor=C.+Vandermeerenauthor=G.+Dubyauthor=M.+Boutryauthor=A.+Wittinghoferauthor=J.-L.+Rigaudauthor=C.+Oecking&title=Structure+of+a+14%E2%80%933-3+Coordinated+Hexamer+of+the+Plant+Plasma+Membrane+H%2B+-ATPase+by+Combining+X-Ray+Crystallography+and+Electron+Cryomicroscopy&doi=10.1016%2Fj.molcel.2006.12.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a 14-3-3 coordinated hexamer of the plant plasma membrane H+-ATPase by combining X-ray crystallography and electron cryomicroscopy</span></div><div class="casAuthors">Ottmann, Christian; Marco, Sergio; Jaspert, Nina; Marcon, Caroline; Schauer, Nicolas; Weyand, Michael; Vandermeeren, Caroline; Duby, Geoffrey; Boutry, Marc; Wittinghofer, Alfred; Rigaud, Jean-Louis; Oecking, Claudia</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">427-440</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Regulatory 14-3-3 proteins activate the plant plasma membrane H+-ATPase by binding to its C-terminal autoinhibitory domain.  This interaction requires phosphorylation of a C-terminal, mode III, recognition motif as well as an adjacent span of approx. 50 amino acids.  Here we report the x-ray crystal structure of 14-3-3 in complex with the entire binding motif, revealing a previously unidentified mode of interaction.  A 14-3-3 dimer simultaneously binds two H+-ATPase peptides, each of which forms a loop within the typical 14-3-3 binding groove and therefore exits from the center of the dimer.  Several H+-ATPase mutants support this structure detn.  Accordingly, 14-3-3 binding could result in H+-ATPase oligomerization.  Indeed, by using single-particle electron cryomicroscopy, the 3D reconstruction of the purified H+-ATPase/14-3-3 complex demonstrates a hexameric arrangement.  Fitting of 14-3-3 and H+-ATPase at. structures into the 3D reconstruction map suggests the spatial arrangement of the holocomplex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2piE2NL1L1bVg90H21EOLACvtfcHk0lhWqhUOB995mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jtbY%253D&md5=6d2a855f5f90a0c2265c1c3381232d8c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DMarco%26aufirst%3DS.%26aulast%3DJaspert%26aufirst%3DN.%26aulast%3DMarcon%26aufirst%3DC.%26aulast%3DSchauer%26aufirst%3DN.%26aulast%3DWeyand%26aufirst%3DM.%26aulast%3DVandermeeren%26aufirst%3DC.%26aulast%3DDuby%26aufirst%3DG.%26aulast%3DBoutry%26aufirst%3DM.%26aulast%3DWittinghofer%26aufirst%3DA.%26aulast%3DRigaud%26aufirst%3DJ.-L.%26aulast%3DOecking%26aufirst%3DC.%26atitle%3DStructure%2520of%2520a%252014%25E2%2580%25933-3%2520Coordinated%2520Hexamer%2520of%2520the%2520Plant%2520Plasma%2520Membrane%2520H%252B%2520-ATPase%2520by%2520Combining%2520X-Ray%2520Crystallography%2520and%2520Electron%2520Cryomicroscopy%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26issue%3D3%26spage%3D427%26epage%3D440%26doi%3D10.1016%2Fj.molcel.2006.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asahi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazeki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubohara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, N.</span></span> <span> </span><span class="NLM_article-title">Cotylenin A, a Plant-Growth Regulator, Induces the Differentiation in Murine and Human Myeloid Leukemia Cells</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>238</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">758</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1997.7385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1006%2Fbbrc.1997.7385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=9325163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADyaK2sXmsFCnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=238&publication_year=1997&pages=758-763&issue=3&author=K.+Asahiauthor=Y.+Honmaauthor=K.+Hazekiauthor=T.+Sassaauthor=Y.+Kuboharaauthor=A.+Sakuraiauthor=N.+Takahashi&title=Cotylenin+A%2C+a+Plant-Growth+Regulator%2C+Induces+the+Differentiation+in+Murine+and+Human+Myeloid+Leukemia+Cells&doi=10.1006%2Fbbrc.1997.7385"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Cotylenin A, a plant-growth regulator, induces the differentiation in murine and human myeloid leukemia cells</span></div><div class="casAuthors">Asahi, Ken-Ichi; Honma, Yoshio; Hazeki, Kaoru; Sassa, Takeshi; Kubohara, Yuzuru; Sakurai, Akira; Takahashi, Nobutaka</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">238</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">758-763</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Protein factors playing a significant part in differentiation and development have been recently elucidated.  However, low-mol. factors which also seem to be essential remain still unknown, although only retinoic acid has become such a candidate.  Cotylenins had been isolated as the plant-growth regulators, and have been found to affect a no. of physiol. processes of higher plants.  Here, the authors report that at the concns. above 12.5μg/mL (20μM) cotylenin A induced the functional and morphol. differentiation in murine (M1) and human myeloid leukemia (HL-60) cells.  Although cotylenin A has some similarity to PMA both in carbotricyclic diterpene structure and in biol. activity, i.e., differentiation-induction of HL-60 cells into macrophages, the activation of PKC and the elevation of Ca2+-levels by cotylenin A were not obsd.  Quite recently it has been reported that fusicoccin (closely related to cotylenin A)-targets are 14-3-3 proteins, which are at the crosspoint of a huge array of signaling and regulatory pathways.  Thus, cotylenin A might become a useful tool for the elucidation of mol. mechanisms of differentiation and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWC4R3xieQx7Vg90H21EOLACvtfcHk0lidb-UV3MiKPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsFCnsrw%253D&md5=80deb88912f3c705eea9021e568752b7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1997.7385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1997.7385%26sid%3Dliteratum%253Aachs%26aulast%3DAsahi%26aufirst%3DK.%26aulast%3DHonma%26aufirst%3DY.%26aulast%3DHazeki%26aufirst%3DK.%26aulast%3DSassa%26aufirst%3DT.%26aulast%3DKubohara%26aufirst%3DY.%26aulast%3DSakurai%26aufirst%3DA.%26aulast%3DTakahashi%26aufirst%3DN.%26atitle%3DCotylenin%2520A%252C%2520a%2520Plant-Growth%2520Regulator%252C%2520Induces%2520the%2520Differentiation%2520in%2520Murine%2520and%2520Human%2520Myeloid%2520Leukemia%2520Cells%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D238%26issue%3D3%26spage%3D758%26epage%3D763%26doi%3D10.1006%2Fbbrc.1997.7385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrei, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vink, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span> <span> </span><span class="NLM_article-title">Rationally Designed Semisynthetic Natural Product Analogues for Stabilization of 14–3-3 Protein-Protein Interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">13470</span>– <span class="NLM_lpage">13474</span>, <span class="refDoi"> DOI: 10.1002/anie.201806584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fanie.201806584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslahsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=13470-13474&issue=41&author=S.+A.+Andreiauthor=P.+de+Vinkauthor=E.+Sijbesmaauthor=L.+Hanauthor=L.+Brunsveldauthor=N.+Katoauthor=C.+Ottmannauthor=Y.+Higuchi&title=Rationally+Designed+Semisynthetic+Natural+Product+Analogues+for+Stabilization+of+14%E2%80%933-3+Protein-Protein+Interactions&doi=10.1002%2Fanie.201806584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rationally Designed Semisynthetic Natural Product Analogues for Stabilization of 14-3-3 Protein-Protein Interactions</span></div><div class="casAuthors">Andrei, Sebastian A.; de Vink, Pim; Sijbesma, Eline; Han, Ling; Brunsveld, Luc; Kato, Nobuo; Ottmann, Christian; Higuchi, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">13470-13474</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The natural product family of fusicoccanes are stabilizers of 14-3-3 mediated protein-protein interactions (PPIs), some of which possess antitumor activity.  In this study, the first use of mol. dynamics (MD) to rationally design PPI stabilizers with increased potency is presented.  Synthesis of a focused library, with subsequent characterization by fluorescence polarization, mutational studies, and X-ray crystallog. confirmed the power of the MD-based design approach, revealing the potential for an addnl. hydrogen bond with the 14-3-3 protein to lead to significantly increased potency.  Addnl., these compds. exert their action in a cellular environment with increased potency.  The newly found polar interaction could provide an anchoring point for new small-mol. PPI stabilizers.  These results facilitate the development of fusicoccanes towards drugs or tool compds., as well as allowing the study of the fundamental principles behind PPI stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmn44vv_9gLLVg90H21EOLACvtfcHk0lidb-UV3MiKPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslahsL%252FO&md5=c168fae0a4306a739bd4681cb712b701</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201806584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201806584%26sid%3Dliteratum%253Aachs%26aulast%3DAndrei%26aufirst%3DS.%2BA.%26aulast%3Dde%2BVink%26aufirst%3DP.%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DHiguchi%26aufirst%3DY.%26atitle%3DRationally%2520Designed%2520Semisynthetic%2520Natural%2520Product%2520Analogues%2520for%2520Stabilization%2520of%252014%25E2%2580%25933-3%2520Protein-Protein%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26issue%3D41%26spage%3D13470%26epage%3D13474%26doi%3D10.1002%2Fanie.201806584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Vries-van Leeuwen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da Costa Pereira, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flach, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piersma, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haase, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yalcin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalides, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feenstra, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Greef, T. F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Boer, A. H.</span></span> <span> </span><span class="NLM_article-title">Interaction of 14–3-3 Proteins with the Estrogen Receptor Alpha F Domain Provides a Drug Target Interface</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">8894</span>– <span class="NLM_lpage">8899</span>, <span class="refDoi"> DOI: 10.1073/pnas.1220809110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1073%2Fpnas.1220809110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=23676274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFait7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8894-8899&issue=22&author=I.+J.+De%0AVries-van+Leeuwenauthor=D.+da+Costa+Pereiraauthor=K.+D.+Flachauthor=S.+R.+Piersmaauthor=C.+Haaseauthor=D.+Bierauthor=Z.+Yalcinauthor=R.+Michalidesauthor=K.+A.+Feenstraauthor=C.+R.+Jim%C3%A9nezauthor=T.+F.+A.+de+Greefauthor=L.+Brunsveldauthor=C.+Ottmannauthor=W.+Zwartauthor=A.+H.+de+Boer&title=Interaction+of+14%E2%80%933-3+Proteins+with+the+Estrogen+Receptor+Alpha+F+Domain+Provides+a+Drug+Target+Interface&doi=10.1073%2Fpnas.1220809110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of 14-3-3 proteins with the Estrogen Receptor Alpha F domain provides a drug target interface</span></div><div class="casAuthors">De Vries-van Leeuwen, Ingrid J.; Pereira, Daniel da Costa; Flach, Koen D.; Piersma, Sander R.; Haase, Christian; Bier, David; Yalcin, Zeliha; Michalides, Rob; Feenstra, K. Anton; Jimenez, Connie R.; de Greef, Tom F. A.; Brunsveld, Luc; Ottmann, Christian; Zwart, Wilbert; de Boer, Albertus H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8894-8899, S8894/1-S8894/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Estrogen receptor alpha (ERa) is involved in numerous physiol. and pathol. processes, including breast cancer.  Breast cancer therapy is therefore currently directed at inhibiting the transcriptional potency of ERa, either by blocking estrogen prodn. through aromatase inhibitors or antiestrogens that compete for hormone binding.  Due to resistance, new treatment modalities are needed and as ERα dimerization is essential for its activity, interference with receptor dimerization offers a new opportunity to exploit in drug design.  Here we describe a unique mechanism of how ERα dimerization is neg. controlled by interaction with 14-3-3 proteins at the extreme C terminus of the receptor.  Moreover, the small-mol. fusicoccin (FC) stabilizes this ERα/14-3-3 interaction.  Cocrystn. of the trimeric ERα/14-3-3/FC complex provides the structural basis for this stabilization and shows the importance of phosphorylation of the penultimate Threonine (ERα-T594) for high-affinity interaction.  We confirm that T594 is a distinct ERα phosphorylation site in the breast cancer cell line MCF-7 using a phospho-T594-specific antibody and by mass spectrometry.  In line with its ERα/14-3-3 interaction stabilizing effect, fusicoccin reduces the estradiol-stimulated ERα dimerization, inhibits ERα/chromatin interactions and downstream gene expression, resulting in decreased cell proliferation.  Herewith, a unique functional phosphosite and an alternative regulation mechanism of ERα are provided, together with a small mol. that selectively targets this ERα/14-3-3 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0rga7Qdqyn7Vg90H21EOLACvtfcHk0lidb-UV3MiKPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFait7%252FP&md5=d17c1a1122f4b525b35651d3d9f629f0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1220809110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1220809110%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVries-van%2BLeeuwen%26aufirst%3DI.%2BJ.%26aulast%3Dda%2BCosta%2BPereira%26aufirst%3DD.%26aulast%3DFlach%26aufirst%3DK.%2BD.%26aulast%3DPiersma%26aufirst%3DS.%2BR.%26aulast%3DHaase%26aufirst%3DC.%26aulast%3DBier%26aufirst%3DD.%26aulast%3DYalcin%26aufirst%3DZ.%26aulast%3DMichalides%26aufirst%3DR.%26aulast%3DFeenstra%26aufirst%3DK.%2BA.%26aulast%3DJim%25C3%25A9nez%26aufirst%3DC.%2BR.%26aulast%3Dde%2BGreef%26aufirst%3DT.%2BF.%2BA.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DZwart%26aufirst%3DW.%26aulast%3Dde%2BBoer%26aufirst%3DA.%2BH.%26atitle%3DInteraction%2520of%252014%25E2%2580%25933-3%2520Proteins%2520with%2520the%2520Estrogen%2520Receptor%2520Alpha%2520F%2520Domain%2520Provides%2520a%2520Drug%2520Target%2520Interface%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26issue%3D22%26spage%3D8894%26epage%3D8899%26doi%3D10.1073%2Fpnas.1220809110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bovens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Identification and Structure of Small-Molecule Stabilizers of 14–3-3 Protein-Protein Interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">4129</span>– <span class="NLM_lpage">4132</span>, <span class="refDoi"> DOI: 10.1002/anie.200907203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fanie.200907203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvFSksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=4129-4132&issue=24&author=R.+Roseauthor=S.+Erdmannauthor=S.+Bovensauthor=A.+Wolfauthor=M.+Roseauthor=S.+Hennigauthor=H.+Waldmannauthor=C.+Ottmann&title=Identification+and+Structure+of+Small-Molecule+Stabilizers+of+14%E2%80%933-3+Protein-Protein+Interactions&doi=10.1002%2Fanie.200907203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Structure of Small-Molecule Stabilizers of 14-3-3 Protein-Protein Interactions</span></div><div class="casAuthors">Rose, Rolf; Erdmann, Silke; Bovens, Stefanie; Wolf, Alexander; Rose, Micheline; Hennig, Sven; Waldmann, Herbert; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4129-4132, S4129/1-S4129/13</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein, the identification of two small mols. that selectively stabilize the 14-3-3/PMA2 protein-protein interaction by binding to two adjacent sites in the interface, and which are active in vivo, is reported.  To identify potential stabilizing mols., a 37,000-member compd. library was screened in a surface-based format monitoring the binding of green fluorescent protein (GFP) fused with 14-3-3 to surface-immobilized glutathione S-transferase (GST) fused with PMA2-CT52 (the C-terminal 52 amino acids of PMA2).  The initial screen successfully identified two structurally unrelated compds. with the desired activity.  For these compds., the kinetics of the stabilizing activity were detd. by means of surface plasmon resonance (SPR).  To this end, the 14-3-3 interaction domain of PMA2, that is the 52 C-terminal amino acids (CT52) was immobilized on the dextran matrix of a BiaCore chip, and binding of the 14-3-3 protein was measured in the presence of the protein-protein interaction-stabilizing small mol.  The tri-substituted pyrrolinone hit compd., pyrrolidone1, showed assocn. kinetics similar to fusicoccin but with a more rapid dissocn., resulting in a KD of 80 μM.  The dipeptide epibestatin displayed a different kinetic behavior in stabilizing the 14-3-3/PMA2 complex, with a slower assocn. than fusicoccin and pyrrolidone1.  However, the dissocn. kinetics of epibestatin resembled that of fusicoccin; that is the protein complex was very stable once it was constituted and dissocn. was very slow.  As a result, the calcd. KD of the 14-3-3/PMA2 complex in the presence of epibestatin was 1.8 μM.  In the absence of any stabilizing compd. no binding of 14-3-3 to immobilized PMA2 could be measured (DMSO control).  To unravel the mechanistic basis of the obviously different binding modes of pyrrolidone1 and epibestatin, the crystal structures the ternary complexes with 14-3-3 and the C-terminal 30 amino acids of PMA2 (PMA2-CT30) were solved.  Co-crystals were obtained by mixing 14-3-3 protein and PMA2-CT30 (1:1.5 ratio) in the presence of either 2 mM pyrrolidone1 or 2 mM epibestatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoOBHBStJvjrVg90H21EOLACvtfcHk0liNkN242rP0Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvFSksr4%253D&md5=c45dbceb27fe5e53ea367319cc735451</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.200907203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200907203%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DR.%26aulast%3DErdmann%26aufirst%3DS.%26aulast%3DBovens%26aufirst%3DS.%26aulast%3DWolf%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DM.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DIdentification%2520and%2520Structure%2520of%2520Small-Molecule%2520Stabilizers%2520of%252014%25E2%2580%25933-3%2520Protein-Protein%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26issue%3D24%26spage%3D4129%26epage%3D4132%26doi%3D10.1002%2Fanie.200907203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B. D.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery and Its Application to Challenging Drug Targets</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1042/EBC20170029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FEBC20170029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=29118094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=475-484&issue=5&author=A.+J.+Priceauthor=S.+Howardauthor=B.+D.+Cons&title=Fragment-Based+Drug+Discovery+and+Its+Application+to+Challenging+Drug+Targets&doi=10.1042%2FEBC20170029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery and its application to challenging drug targets</span></div><div class="casAuthors">Price Amanda J; Howard Steven; Cons Benjamin D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">475-484</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a technique for identifying low molecular weight chemical starting points for drug discovery.  Since its inception 20 years ago, FBDD has grown in popularity to the point where it is now an established technique in industry and academia.  The approach involves the biophysical screening of proteins against collections of low molecular weight compounds (fragments).  Although fragments bind to proteins with relatively low affinity, they form efficient, high quality binding interactions with the protein architecture as they have to overcome a significant entropy barrier to bind.  Of the biophysical methods available for fragment screening, X-ray protein crystallography is one of the most sensitive and least prone to false positives.  It also provides detailed structural information of the protein-fragment complex at the atomic level.  Fragment-based screening using X-ray crystallography is therefore an efficient method for identifying binding hotspots on proteins, which can then be exploited by chemists and biologists for the discovery of new drugs.  The use of FBDD is illustrated here with a recently published case study of a drug discovery programme targeting the challenging protein-protein interaction Kelch-like ECH-associated protein 1:nuclear factor erythroid 2-related factor 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0VHFYivNCdPRfW6udTcc2eYEW1KjNMRTobntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D&md5=f8572516bfb1ecc4a0ef57b0c5ba1bc0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FEBC20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170029%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DCons%26aufirst%3DB.%2BD.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520and%2520Its%2520Application%2520to%2520Challenging%2520Drug%2520Targets%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D475%26epage%3D484%26doi%3D10.1042%2FEBC20170029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neves, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merzougui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boll, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanoulle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrelle, F.-X.</span></span> <span> </span><span class="NLM_article-title">Backbone Chemical Shift Assignments of Human 14–3-3σ</span>. <i>Biomol. NMR Assignments</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1007/s12104-018-9860-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1007%2Fs12104-018-9860-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=30377945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVCksrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2019&pages=103-107&issue=1&author=J.+F.+Nevesauthor=I.+Landrieuauthor=H.+Merzouguiauthor=E.+Bollauthor=X.+Hanoulleauthor=F.-X.+Cantrelle&title=Backbone+Chemical+Shift+Assignments+of+Human+14%E2%80%933-3%CF%83&doi=10.1007%2Fs12104-018-9860-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Backbone chemical shift assignments of human 14-3-3s</span></div><div class="casAuthors">Neves, Joao Filipe; Landrieu, Isabelle; Merzougui, Hamida; Boll, Emmanuelle; Hanoulle, Xavier; Cantrelle, Francois-Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecular NMR Assignments</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">BNAIBR</span>;
        ISSN:<span class="NLM_cas:issn">1874-270X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">14-3-3 proteins are a group of seven dimeric adapter proteins that exert their biol. function by interacting with hundreds of phosphorylated proteins, thus influencing their sub-cellular localization, activity or stability in the cell.  Due to this remarkable interaction network, 14-3-3 proteins have been assocd. with several pathologies and the protein-protein interactions (PPIs) established with a no. of partners are now considered promising drug targets.  The activity of 14-3-3 proteins is often isoform specific and to our knowledge only one out of seven isoforms, 14-3-3ζ, has been assigned.  Despite the availability of the crystal structures of all seven isoforms of 14-3-3, the addnl. NMR assignments of 14-3-3 proteins are important for both biol. mechanism studies and chem. biol. approaches.  Herein, we present a robust backbone assignment of 14-3-3s, which will allow advances in the discovery of potential therapeutic compds.  This assignment is now being applied to the discovery of both inhibitors and stabilizers of 14-3-3 PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT5a0Tg2yhtrVg90H21EOLACvtfcHk0liNkN242rP0Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVCksrbP&md5=aa9c44a9f9c8c55b718457b144a12521</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs12104-018-9860-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12104-018-9860-1%26sid%3Dliteratum%253Aachs%26aulast%3DNeves%26aufirst%3DJ.%2BF.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DMerzougui%26aufirst%3DH.%26aulast%3DBoll%26aufirst%3DE.%26aulast%3DHanoulle%26aufirst%3DX.%26aulast%3DCantrelle%26aufirst%3DF.-X.%26atitle%3DBackbone%2520Chemical%2520Shift%2520Assignments%2520of%2520Human%252014%25E2%2580%25933-3%25CF%2583%26jtitle%3DBiomol.%2520NMR%2520Assignments%26date%3D2019%26volume%3D13%26issue%3D1%26spage%3D103%26epage%3D107%26doi%3D10.1007%2Fs12104-018-9860-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Stabilization of Protein-Protein Interactions: An Underestimated Concept in Drug Discovery?</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2012</span>– <span class="NLM_lpage">2018</span>, <span class="refDoi"> DOI: 10.1002/anie.201107616</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fanie.201107616" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=2012-2018&issue=9&author=P.+Thielauthor=M.+Kaiserauthor=C.+Ottmann&title=Small-Molecule+Stabilization+of+Protein-Protein+Interactions%3A+An+Underestimated+Concept+in+Drug+Discovery%3F&doi=10.1002%2Fanie.201107616"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Stabilization of Protein-Protein Interactions: An Underestimated Concept in Drug Discovery?</span></div><div class="casAuthors">Thiel, Philipp; Kaiser, Markus; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2012-2018</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The modulation of protein-protein interactions (PPIs) has been recognized as one of the most challenging tasks in drug discovery.  While their systematic development has long been considered as intractable, this view has changed over the last years, with the first drug candidates undergoing clin. studies.  To date, the vast majority of PPI modulators are interaction inhibitors.  However, in many biol. contexts a prolonged lifespan of a PPI might be desirable, calling for the complementary approach of PPI stabilization.  In fact, nature offers impressive examples of this concept and some PPI-stabilizing natural products have already found application as important drugs.  Moreover, directed small-mol. stabilization has recently been demonstrated.  Therefore, it is time to take a closer look at the constructive side of modulating PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSOQe9isrVGrVg90H21EOLACvtfcHk0liNkN242rP0Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCrsbk%253D&md5=7b08cfbf9af0b49cdf36e739c907c88a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107616&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107616%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DP.%26aulast%3DKaiser%26aufirst%3DM.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSmall-Molecule%2520Stabilization%2520of%2520Protein-Protein%2520Interactions%253A%2520An%2520Underestimated%2520Concept%2520in%2520Drug%2520Discovery%253F%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26issue%3D9%26spage%3D2012%26epage%3D2018%26doi%3D10.1002%2Fanie.201107616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arfons, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarus, H. M.</span></span> <span> </span><span class="NLM_article-title">The Rise, Fall and Subsequent Triumph of Thalidomide: Lessons Learned in Drug Development</span>. <i>Ther. Adv. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1177/2040620711413165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1177%2F2040620711413165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=23556097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=291-308&issue=5&author=W.+Rehmanauthor=L.+M.+Arfonsauthor=H.+M.+Lazarus&title=The+Rise%2C+Fall+and+Subsequent+Triumph+of+Thalidomide%3A+Lessons+Learned+in+Drug+Development&doi=10.1177%2F2040620711413165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The rise, fall and subsequent triumph of thalidomide: lessons learned in drug development</span></div><div class="casAuthors">Rehman, Waqas; Arfons, Lisa M.; Lazarus, Hillard M.</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Hematology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">291-308</span>CODEN:
                <span class="NLM_cas:coden">TAHHAT</span>;
        ISSN:<span class="NLM_cas:issn">2040-6207</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Perhaps no other drug in modern medicine rivals the dramatic revitalization of thalidomide.  Originally marketed as a sedative, thalidomide gained immense popularity worldwide among pregnant women because of its effective anti-emetic properties in morning sickness.  Mounting evidence of human teratogenicity marked a dramatic fall from grace and led to widespread social, legal and economic ramifications.  Despite its tragic past thalidomide emerged several decades later as a novel and highly effective agent in the treatment of various inflammatory and malignant diseases.  In 2006 thalidomide completed its remarkable renaissance becoming the first new agent in over a decade to gain approval for the treatment of plasma cell myeloma.  The catastrophic collapse yet subsequent revival of thalidomide provides important lessons in drug development.  Never entirely abandoned by the medical community, thalidomide resurfaced as an important drug once the mechanisms of action were further studied and better understood.  Ongoing research and development of related drugs such as lenalidomide now represent a class of irreplaceable drugs in hematol. malignancies.  Further, the tragedies assocd. with this agent stimulated the legislation which revamped the FDA regulatory process, expanded patient informed consent procedures and mandated more transparency from drug manufacturers.  Finally, we review recent clin. trials summarizing selected medical indications for thalidomide with an emphasis on hematol. malignancies.  Herein, we provide a historic perspective regarding the up-and-down development of thalidomide.  Using PubMed databases we conducted searches using thalidomide and assocd. keywords highlighting pharmacol., mechanisms of action, and clin. uses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUo5hN4zSsD7Vg90H21EOLACvtfcHk0liXrHvdk_OcZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVGgsA%253D%253D&md5=a0f10d1e152e9c0db704fef9da69ae9b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1177%2F2040620711413165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2040620711413165%26sid%3Dliteratum%253Aachs%26aulast%3DRehman%26aufirst%3DW.%26aulast%3DArfons%26aufirst%3DL.%2BM.%26aulast%3DLazarus%26aufirst%3DH.%2BM.%26atitle%3DThe%2520Rise%252C%2520Fall%2520and%2520Subsequent%2520Triumph%2520of%2520Thalidomide%253A%2520Lessons%2520Learned%2520in%2520Drug%2520Development%26jtitle%3DTher.%2520Adv.%2520Hematol.%26date%3D2011%26volume%3D2%26issue%3D5%26spage%3D291%26epage%3D308%26doi%3D10.1177%2F2040620711413165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Recognition of Ternary Complexes: A New Dimension in the Structure-Guided Design of Chemical Degraders</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1042/EBC20170041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FEBC20170041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=29118097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=505-516&issue=5&author=S.+J.+Hughesauthor=A.+Ciulli&title=Molecular+Recognition+of+Ternary+Complexes%3A+A+New+Dimension+in+the+Structure-Guided+Design+of+Chemical+Degraders&doi=10.1042%2FEBC20170041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span></div><div class="casAuthors">Hughes Scott J; Ciulli Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">505-516</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular glues and bivalent inducers of protein degradation (also known as PROTACs) represent a fascinating new modality in pharmacotherapeutics: the potential to knockdown previously thought 'undruggable' targets at sub-stoichiometric concentrations in ways not possible using conventional inhibitors.  Mounting evidence suggests these chemical agents, in concert with their target proteins, can be modelled as three-body binding equilibria that can exhibit significant cooperativity as a result of specific ligand-induced molecular recognition.  Despite this, many existing drug design and optimization regimens still fixate on binary target engagement, in part due to limited structural data on ternary complexes.  Recent crystal structures of protein complexes mediated by degrader molecules, including the first PROTAC ternary complex, underscore the importance of protein-protein interactions and intramolecular contacts to the mode of action of this class of compounds.  These discoveries have opened the door to a new paradigm for structure-guided drug design: borrowing surface area and molecular recognition from nature to elicit cellular signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0b5XHngOXiV_fW6udTcc2eZxWeuGRvieVrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D&md5=7470e559372f05d34c5924524a6d04bc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1042%2FEBC20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170041%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520Recognition%2520of%2520Ternary%2520Complexes%253A%2520A%2520New%2520Dimension%2520in%2520the%2520Structure-Guided%2520Design%2520of%2520Chemical%2520Degraders%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D505%26epage%3D516%26doi%3D10.1042%2FEBC20170041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Calcineurin Is a Common Target of Cyclophilin-Cyclosporin A and FKBP-FK506 Complexes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90124-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2F0092-8674%2891%2990124-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1715244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADyaK3MXlsV2qu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=1991&pages=807-815&issue=4&author=J.+Liuauthor=J.+D.+Farmerauthor=W.+S.+Laneauthor=J.+Friedmanauthor=I.+Weissmanauthor=S.+L.+Schreiber&title=Calcineurin+Is+a+Common+Target+of+Cyclophilin-Cyclosporin+A+and+FKBP-FK506+Complexes&doi=10.1016%2F0092-8674%2891%2990124-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Calcineurin is a common target of cyclophilin-cyclosporin A and FKBP-FK506 complexes</span></div><div class="casAuthors">Liu, Jun; Farmer, Jesse D., Jr.; Lane, Willam S.; Friedman, Jeff; Weissman, Irving; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">807-15</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Although the immediate receptors (immunophilins) of the immunosuppressants cyclosporin A (CsA) and FK506 are distinct, their similar mechanisms of inhibition of cell signaling suggest that their assocd. immunophilin complexes interact with a common target.  The authors report that the complexes cyclophilin-CsA and FKBP-FK506 (but not cyclophilin, FKBP, FKBP-rapamycin, or FKBP-506BD) competitively bind to and inhibit the Ca2+- and calmodulin-dependent phosphatase calcineurin, although the binding and inhibition of calcineurin do not require calmodulin.  These results suggest that calcineurin is involved in a common step assocd. with T cell receptor and IgE receptor signaling pathways and that cyclophilin and FKBP mediate the actions of CsA and FK506, resp., by forming drug-dependent complexes with and altering the activity of calcineurin-calmodulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxfN8mg-hNhLVg90H21EOLACvtfcHk0liXrHvdk_OcZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlsV2qu7w%253D&md5=00c4883ea86e6b2c554f8f72ed717ac9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990124-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990124-H%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFarmer%26aufirst%3DJ.%2BD.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DFriedman%26aufirst%3DJ.%26aulast%3DWeissman%26aufirst%3DI.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DCalcineurin%2520Is%2520a%2520Common%2520Target%2520of%2520Cyclophilin-Cyclosporin%2520A%2520and%2520FKBP-FK506%2520Complexes%26jtitle%3DCell%26date%3D1991%26volume%3D66%26issue%3D4%26spage%3D807%26epage%3D815%26doi%3D10.1016%2F0092-8674%2891%2990124-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albers, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">A Mammalian Protein Targeted by G1-Arresting Rapamycin-Receptor Complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>369</i></span> (<span class="NLM_issue">6483</span>),  <span class="NLM_fpage">756</span>– <span class="NLM_lpage">758</span>, <span class="refDoi"> DOI: 10.1038/369756a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1038%2F369756a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=8008069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADyaK2cXlsFSiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=1994&pages=756-758&issue=6483&author=E.+J.+Brownauthor=M.+W.+Albersauthor=T.+B.+Shinauthor=K.+Ichikawaauthor=C.+T.+Keithauthor=W.+S.+Laneauthor=S.+L.+Schreiber&title=A+Mammalian+Protein+Targeted+by+G1-Arresting+Rapamycin-Receptor+Complex&doi=10.1038%2F369756a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A mammalian protein targeted by G1-arresting rapamycin-receptor complex</span></div><div class="casAuthors">Brown, Eric J.; Albers, Mark W.; Shin, Tae Bum; Ichikawa, Kazuo; Keith, Curtis T.; Lane, William S.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">6483</span>),
    <span class="NLM_cas:pages">756-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The structurally related natural products rapamycin and FK506 bind to the same intracellular receptor, FKBP12, yet the resulting complexes interfere with distinct signaling pathway.  FKBP12-rapamycin inhibits progression through the G1 phase of the cell cycle in osteosarcoma, liver and T cells as well as in yeast, and interferes with mitogenic signaling pathways that are involved in G1 progression, namely with activation of the protein p70S6k and cyclin-dependent kinases.  Here the authors isolate a mammalian FKBP-rapamycin-assocd. protein (FRAP) whose binding to structural variants of rapamycin complexed to FKBP12 correlates with the ability of these ligands to inhibit cell-cycle progression.  Peptide sequences from purified bovine FRAP were used to isolate a human cDNA clone that is highly related to the DRR1/TOR1 and DRR2/TOR2 gene products form Saccharomyces cerevisiae.  Although it has not been previously demonstrated that either of the DRR/TOR gene products can bind the FKBP-rapamycin complex directly, these yeast genes have been genetically linked to a rapamycin-sensitive pathway and are through to encode lipid kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdC-Z1-1ylr7Vg90H21EOLACvtfcHk0ljlVycso9XDAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlsFSiu74%253D&md5=b5c34fc2333e81011d6730cc196eb21a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F369756a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F369756a0%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DE.%2BJ.%26aulast%3DAlbers%26aufirst%3DM.%2BW.%26aulast%3DShin%26aufirst%3DT.%2BB.%26aulast%3DIchikawa%26aufirst%3DK.%26aulast%3DKeith%26aufirst%3DC.%2BT.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520Mammalian%2520Protein%2520Targeted%2520by%2520G1-Arresting%2520Rapamycin-Receptor%2520Complex%26jtitle%3DNature%26date%3D1994%26volume%3D369%26issue%3D6483%26spage%3D756%26epage%3D758%26doi%3D10.1038%2F369756a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation by PROTACs</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">138</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.pharmthera.2017.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=28223226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=138-144&author=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=C.+M.+Crews&title=Targeted+Protein+Degradation+by+PROTACs&doi=10.1016%2Fj.pharmthera.2017.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation by PROTACs</span></div><div class="casAuthors">Neklesa, Taavi K.; Winkler, James D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">138-144</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using the PROTAC technol. is emerging as a novel therapeutic method to address diseases driven by the aberrant expression of a disease-causing protein.  PROTAC mols. are bifunctional small mols. that simultaneously bind a target protein and an E3-ubiquitin ligase, thus causing ubiquitination and degrdn. of the target protein by the proteasome.  Like small mols., PROTAC mols. possess good tissue distribution and the ability to target intracellular proteins.  Herein, we highlight the advantages of protein degrdn. using PROTACs, and provide specific examples where degrdn. offers therapeutic benefit over classical enzyme inhibition.  Foremost, PROTACs can degrade proteins regardless of their function.  This includes the currently "undruggable" proteome, which comprises approx. 85% of all human proteins.  Other beneficial aspects of protein degrdn. include the ability to target overexpressed and mutated proteins, as well as the potential to demonstrate prolonged pharmacodynamics effect beyond drug exposure.  Lastly, due to their catalytic nature and the pre-requisite ubiquitination step, an exquisitely potent mols. with a high degree of degrdn. selectivity can be designed.  Impressive preclin. in vitro and in vivo PROTAC data have been published, and these data have propelled the development of clin. viable PROTACs.  With the mol. wt. falling in the 700-1000 Da range, the delivery and bioavailability of PROTACs remain the largest hurdles on the way to the clinic.  Solving these issues and demonstrating proof of concept clin. data will be the focus of many labs over the next few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4RSLWYqqBMrVg90H21EOLACvtfcHk0ljlVycso9XDAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D&md5=08a74402a274a13c8c484cd68167a5a1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%2520by%2520PROTACs%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D174%26spage%3D138%26epage%3D144%26doi%3D10.1016%2Fj.pharmthera.2017.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anders, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschinsky, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitta, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preisig-Müller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlichthörl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renigunta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkanda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daut, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">A Semisynthetic Fusicoccane Stabilizes a Protein-Protein Interaction and Enhances the Expression of K+ Channels at the Cell Surface</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.03.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.chembiol.2013.03.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=23601647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1Cmt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=583-593&issue=4&author=C.+Andersauthor=Y.+Higuchiauthor=K.+Koschinskyauthor=M.+Bartelauthor=B.+Schumacherauthor=P.+Thielauthor=H.+Nittaauthor=R.+Preisig-M%C3%BCllerauthor=G.+Schlichth%C3%B6rlauthor=V.+Reniguntaauthor=J.+Ohkandaauthor=J.+Dautauthor=N.+Katoauthor=C.+Ottmann&title=A+Semisynthetic+Fusicoccane+Stabilizes+a+Protein-Protein+Interaction+and+Enhances+the+Expression+of+K%2B+Channels+at+the+Cell+Surface&doi=10.1016%2Fj.chembiol.2013.03.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">A Semisynthetic Fusicoccane Stabilizes a Protein-Protein Interaction and Enhances the Expression of K+ Channels at the Cell Surface</span></div><div class="casAuthors">Anders, Carolin; Higuchi, Yusuke; Koschinsky, Kristin; Bartel, Maria; Schumacher, Benjamin; Thiel, Philipp; Nitta, Hajime; Preisig-Mueller, Regina; Schlichthoerl, Guenter; Renigunta, Vijay; Ohkanda, Junko; Daut, Juergen; Kato, Nobuo; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">583-593</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. stabilization of protein-protein interactions is an emerging field in chem. biol.  We show how fusicoccanes, originally identified as fungal toxins acting on plants, promote the interaction of 14-3-3 proteins with the human potassium channel TASK-3 and present a semisynthetic fusicoccane deriv. (FC-THF) that targets the 14-3-3 recognition motif (mode 3) in TASK-3.  In the presence of FC-THF, the binding of 14-3-3 proteins to TASK-3 was increased 19-fold and protein crystallog. provided the at. details of the effects of FC-THF on this interaction.  We also tested the functional effects of FC-THF on TASK channels heterologously expressed in Xenopus oocytes.  Incubation with 10 μM FC-THF was found to promote the transport of TASK channels to the cell membrane, leading to a significantly higher d. of channels at the surface membrane and increased potassium current.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV8Cuo6SYkbLVg90H21EOLACvtfcHk0ljlVycso9XDAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1Cmt7Y%253D&md5=bb8b65604416a21f972ed491948d17d5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.03.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.03.015%26sid%3Dliteratum%253Aachs%26aulast%3DAnders%26aufirst%3DC.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DKoschinsky%26aufirst%3DK.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DSchumacher%26aufirst%3DB.%26aulast%3DThiel%26aufirst%3DP.%26aulast%3DNitta%26aufirst%3DH.%26aulast%3DPreisig-M%25C3%25BCller%26aufirst%3DR.%26aulast%3DSchlichth%25C3%25B6rl%26aufirst%3DG.%26aulast%3DRenigunta%26aufirst%3DV.%26aulast%3DOhkanda%26aufirst%3DJ.%26aulast%3DDaut%26aufirst%3DJ.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DA%2520Semisynthetic%2520Fusicoccane%2520Stabilizes%2520a%2520Protein-Protein%2520Interaction%2520and%2520Enhances%2520the%2520Expression%2520of%2520K%252B%2520Channels%2520at%2520the%2520Cell%2520Surface%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26issue%3D4%26spage%3D583%26epage%3D593%26doi%3D10.1016%2Fj.chembiol.2013.03.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sies, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halbach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haranosono, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Stabilization of the 14–3-3/Gab2 Protein-Protein Interaction (PPI) Interface</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">911</span>– <span class="NLM_lpage">918</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fcmdc.201500484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=26644359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKgs73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=911-918&issue=8&author=D.+Bierauthor=M.+Bartelauthor=K.+Siesauthor=S.+Halbachauthor=Y.+Higuchiauthor=Y.+Haranosonoauthor=T.+Brummerauthor=N.+Katoauthor=C.+Ottmann&title=Small-Molecule+Stabilization+of+the+14%E2%80%933-3%2FGab2+Protein-Protein+Interaction+%28PPI%29+Interface&doi=10.1002%2Fcmdc.201500484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule stabilization of the 14-3-3/Gab2 protein-protein interaction (PPI) interface</span></div><div class="casAuthors">Bier, David; Bartel, Maria; Sies, Katharina; Halbach, Sebastian; Higuchi, Yusuke; Haranosono, Yu; Brummer, Tilman; Kato, Nobuo; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">911-918</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Small-mol. modulation of protein-protein interactions (PPIs) is one of the most promising new areas in drug discovery.  In the vast majority of cases, only inhibition or disruption of PPIs is realized, whereas the complementary strategy of targeted stabilization of PPIs is clearly under-represented.  Here, the authors report the example of a semi-synthetic natural product deriv., ISIR-005, that stabilizes the cancer-relevant interaction of adaptor protein 14-3-3ζ and Gab2.  The crystal structure of ISIR-005 in complex with 14-3-3ζ and the binding motif of Gab2 comprising 2 phosphorylation sites (Gab2pS210pT391) showed how the stabilizing mol. binds to the rim-of-the-interface of the protein complex.  Only in the direct vicinity of 14-3-3/Gab2pT391 site was a pre-formed pocket occupied by ISIR-005; binding of the Gab2pS210 motif to 14-3-3ζ did not create an interface pocket suitable for the mol.  Accordingly, ISIR-005 only stabilizes the binding of Gab2pT391, but not the Gab2pS210 site.  This study represents structural and biochem. proof of the druggability of the 14-3-3/Gab2 PPI interface with important implications for the development of PPI stabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6MZMr-enjWrVg90H21EOLACvtfcHk0lj9yQfQ-n7KJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKgs73J&md5=94d97339c17d6c5398909ac79f7d9b75</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500484%26sid%3Dliteratum%253Aachs%26aulast%3DBier%26aufirst%3DD.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DSies%26aufirst%3DK.%26aulast%3DHalbach%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DHaranosono%26aufirst%3DY.%26aulast%3DBrummer%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSmall-Molecule%2520Stabilization%2520of%2520the%252014%25E2%2580%25933-3%252FGab2%2520Protein-Protein%2520Interaction%2520%2528PPI%2529%2520Interface%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26issue%3D8%26spage%3D911%26epage%3D918%26doi%3D10.1002%2Fcmdc.201500484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lucente, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visconti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aducci, P.</span></span> <span> </span><span class="NLM_article-title">The Phytotoxin Fusicoccin Promotes Platelet Aggregation via 14–3-3-Glycoprotein Ib-IX-V Interaction</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>436</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">436</span>, <span class="refDoi"> DOI: 10.1042/BJ20102037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FBJ20102037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=21395556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFGit70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2011&pages=429-436&issue=2&author=L.+Camoniauthor=C.+Di+Lucenteauthor=S.+Viscontiauthor=P.+Aducci&title=The+Phytotoxin+Fusicoccin+Promotes+Platelet+Aggregation+via+14%E2%80%933-3-Glycoprotein+Ib-IX-V+Interaction&doi=10.1042%2FBJ20102037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The phytotoxin fusicoccin promotes platelet aggregation via 14-3-3-glycoprotein Ib-IX-V interaction</span></div><div class="casAuthors">Camoni, Lorenzo; Di Lucente, Cristina; Visconti, Sabina; Aducci, Patrizia</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">429-436</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The fungal toxin fusicoccin induces plant wilting by affecting ion transport across the plasma membrane of plant cell.  The activity of this toxin is so far unknown in humans.  In the present study fusicoccin is able to affect the platelet aggregation process.  The toxin assocs. with platelet intracellular binding sites and induces aggregation in platelet-rich plasma in a dose-dependent manner.  The authors identified the adhesion receptor glycoprotein Ib-IX-V as fusicoccin target.  The toxin promotes the binding of the regulatory 14-3-3 proteins to glycoprotein Ibα and hampers that to glycoprotein Ibβ subunit.  As a result, platelet adhesion to von Willebrand factor is stimulated, leading to platelet spreading and integrin αIIbβ3 activation.  The authors anticipate the present study to be a starting point for future therapeutic use of fusicoccin in genetic bleeding diseases characterized by qual. or quant. abnormalities of the platelet membrane-adhesion receptors.  Furthermore, the present study also sets the stage for future work to det. the potential pharmacol. application of fusicoccin as a drug directed to other 14-3-3-target complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHxEaLMHhJrrVg90H21EOLACvtfcHk0lj9yQfQ-n7KJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFGit70%253D&md5=52e02275949e81f260417fcc75e7b7fa</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1042%2FBJ20102037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20102037%26sid%3Dliteratum%253Aachs%26aulast%3DCamoni%26aufirst%3DL.%26aulast%3DDi%2BLucente%26aufirst%3DC.%26aulast%3DVisconti%26aufirst%3DS.%26aulast%3DAducci%26aufirst%3DP.%26atitle%3DThe%2520Phytotoxin%2520Fusicoccin%2520Promotes%2520Platelet%2520Aggregation%2520via%252014%25E2%2580%25933-3-Glycoprotein%2520Ib-IX-V%2520Interaction%26jtitle%3DBiochem.%2520J.%26date%3D2011%26volume%3D436%26issue%3D2%26spage%3D429%26epage%3D436%26doi%3D10.1042%2FBJ20102037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paiardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aducci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cervoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutruzzolà, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lucente, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascarella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visconti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camoni, L.</span></span> <span> </span><span class="NLM_article-title">The Phytotoxin Fusicoccin Differently Regulates 14–3-3 Proteins Association to Mode III Targets</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1002/iub.1239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fiub.1239" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=24408864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1ensg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2014&pages=52-62&issue=1&author=A.+Paiardiniauthor=P.+Aducciauthor=L.+Cervoniauthor=F.+Cutruzzol%C3%A0author=C.+Di+Lucenteauthor=G.+Jansonauthor=S.+Pascarellaauthor=S.+Rinaldoauthor=S.+Viscontiauthor=L.+Camoni&title=The+Phytotoxin+Fusicoccin+Differently+Regulates+14%E2%80%933-3+Proteins+Association+to+Mode+III+Targets&doi=10.1002%2Fiub.1239"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The phytotoxin fusicoccin differently regulates 14-3-3 proteins association to mode III targets</span></div><div class="casAuthors">Paiardini, Alessandro; Aducci, Patrizia; Cervoni, Laura; Cutruzzola, Francesca; Di Lucente, Cristina; Janson, Giacomo; Pascarella, Stefano; Rinaldo, Serena; Visconti, Sabina; Camoni, Lorenzo</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-62</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">Modulation of the interaction of regulatory 14-3-3 proteins to their physiol. partners through small cell-permeant mols. is a promising strategy to control cellular processes where 14-3-3s are engaged.  Here, we show that the fungal phytotoxin fusicoccin (FC), known to stabilize 14-3-3 assocn. to the plant plasma membrane H+-ATPase, is able to stabilize 14-3-3 interaction to several client proteins with a mode III binding motif.  Isothermal titrn. calorimetry anal. of the interaction between 14-3-3s and different peptides reproducing a mode III binding site demonstrated the FC ability to stimulate 14-3-3 the assocn.  Moreover, mol. docking studies provided the structural rationale for the differential FC effect, which exclusively depends on the biochem. properties of the residue in peptide C-terminal position.  Our study proposes FC as a promising tool to control cellular processes regulated by 14-3-3 proteins, opening new perspectives on its potential pharmacol. applications. © 2014 IUBMB Life, 2014.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwaPxiwp-vtrVg90H21EOLACvtfcHk0lj9yQfQ-n7KJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1ensg%253D%253D&md5=52771e2713a2fff7c139145b6048679e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fiub.1239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.1239%26sid%3Dliteratum%253Aachs%26aulast%3DPaiardini%26aufirst%3DA.%26aulast%3DAducci%26aufirst%3DP.%26aulast%3DCervoni%26aufirst%3DL.%26aulast%3DCutruzzol%25C3%25A0%26aufirst%3DF.%26aulast%3DDi%2BLucente%26aufirst%3DC.%26aulast%3DJanson%26aufirst%3DG.%26aulast%3DPascarella%26aufirst%3DS.%26aulast%3DRinaldo%26aufirst%3DS.%26aulast%3DVisconti%26aufirst%3DS.%26aulast%3DCamoni%26aufirst%3DL.%26atitle%3DThe%2520Phytotoxin%2520Fusicoccin%2520Differently%2520Regulates%252014%25E2%2580%25933-3%2520Proteins%2520Association%2520to%2520Mode%2520III%2520Targets%26jtitle%3DIUBMB%2520Life%26date%3D2014%26volume%3D66%26issue%3D1%26spage%3D52%26epage%3D62%26doi%3D10.1002%2Fiub.1239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo-Rodriguez, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Anguita, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klärner, F.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Garcia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Molecular Tweezers Modulate 14–3-3 Protein-Protein Interactions</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1038/nchem.1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1038%2Fnchem.1570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=23422566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislSqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=234-239&issue=3&author=D.+Bierauthor=R.+Roseauthor=K.+Bravo-Rodriguezauthor=M.+Bartelauthor=J.+M.+Ramirez-Anguitaauthor=S.+Duttauthor=C.+Wilchauthor=F.-G.+Kl%C3%A4rnerauthor=E.+Sanchez-Garciaauthor=T.+Schraderauthor=C.+Ottmann&title=Molecular+Tweezers+Modulate+14%E2%80%933-3+Protein-Protein+Interactions&doi=10.1038%2Fnchem.1570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular tweezers modulate 14-3-3 protein-protein interactions</span></div><div class="casAuthors">Bier, David; Rose, Rolf; Bravo-Rodriguez, Kenny; Bartel, Maria; Ramirez-Anguita, Juan Manuel; Dutt, Som; Wilch, Constanze; Klaerner, Frank-Gerrit; Sanchez-Garcia, Elsa; Schrader, Thomas; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-239</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Supramol. chem. has recently emerged as a promising way to modulate protein functions, but devising mols. that will interact with a protein in the desired manner is difficult as many competing interactions exist in a biol. environment (with solvents, salts or different sites for the target biomol.).  We now show that lysine-specific mol. tweezers bind to a 14-3-3 adapter protein and modulate its interaction with partner proteins.  The tweezers inhibit binding between the 14-3-3 protein and two partner proteins - a phosphorylated (c-Raf) protein and an unphosphorylated one (ExoS) - in a concn.-dependent manner.  Protein crystallog. shows that this effect arises from the binding of the tweezers to a single surface-exposed lysine (Lys214) of the 14-3-3 protein in the proximity of its central channel, which normally binds the partner proteins.  A combination of structural anal. and computer simulations provides rules for the tweezers' binding preferences, thus allowing us to predict their influence on this type of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVofbsBIufTLVg90H21EOLACvtfcHk0lj9yQfQ-n7KJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislSqur0%253D&md5=7aa2fdf28d3f5643530fb6705803e112</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1570%26sid%3Dliteratum%253Aachs%26aulast%3DBier%26aufirst%3DD.%26aulast%3DRose%26aufirst%3DR.%26aulast%3DBravo-Rodriguez%26aufirst%3DK.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DRamirez-Anguita%26aufirst%3DJ.%2BM.%26aulast%3DDutt%26aufirst%3DS.%26aulast%3DWilch%26aufirst%3DC.%26aulast%3DKl%25C3%25A4rner%26aufirst%3DF.-G.%26aulast%3DSanchez-Garcia%26aufirst%3DE.%26aulast%3DSchrader%26aufirst%3DT.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DMolecular%2520Tweezers%2520Modulate%252014%25E2%2580%25933-3%2520Protein-Protein%2520Interactions%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26issue%3D3%26spage%3D234%26epage%3D239%26doi%3D10.1038%2Fnchem.1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo-Rodriguez, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sowislok, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillory, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briels, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heid, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wettig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Garcia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">The Molecular Tweezer CLR01 Stabilizes a Disordered Protein-Protein Interface</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">45</span>),  <span class="NLM_fpage">16256</span>– <span class="NLM_lpage">16263</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b07939</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b07939" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KnsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=16256-16263&issue=45&author=D.+Bierauthor=S.+Mittalauthor=K.+Bravo-Rodriguezauthor=A.+Sowislokauthor=X.+Guilloryauthor=J.+Brielsauthor=C.+Heidauthor=M.+Bartelauthor=B.+Wettigauthor=L.+Brunsveldauthor=E.+Sanchez-Garciaauthor=T.+Schraderauthor=C.+Ottmann&title=The+Molecular+Tweezer+CLR01+Stabilizes+a+Disordered+Protein-Protein+Interface&doi=10.1021%2Fjacs.7b07939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The Molecular Tweezer CLR01 Stabilizes a Disordered Protein-Protein Interface</span></div><div class="casAuthors">Bier, David; Mittal, Sumit; Bravo-Rodriguez, Kenny; Sowislok, Andrea; Guillory, Xavier; Briels, Jeroen; Heid, Christian; Bartel, Maria; Wettig, Burkhard; Brunsveld, Luc; Sanchez-Garcia, Elsa; Schrader, Thomas; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">16256-16263</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein regions that are involved in protein-protein interactions (PPIs) very often display a high degree of intrinsic disorder, which is reduced during the recognition process.  A prime example is binding of the rigid 14-3-3 adapter proteins to their numerous partner proteins, whose recognition motifs undergo an extensive disorder-to-order transition.  In this context, it is highly desirable to control this entropy-costly process using tailored stabilizing agents.  This study reveals how the mol. tweezer CLR01 tunes the 14-3-3/Cdc25C phosphoSer-216 protein-protein interaction.  Protein crystallog., biophys. affinity detn., and biomol. simulations unanimously delivered a remarkable finding: a supramol. "Janus" ligand can bind simultaneously to a flexible peptidic PPI recognition motif and to a well-structured adapter protein.  This binding fills a gap in the protein-protein interface, "freezes" one of the conformational states of the intrinsically disordered Cdc25C protein partner, and enhances the apparent affinity of the interaction.  This is the 1st structural and functional proof of a supramol. ligand targeting a PPI interface and stabilizing the binding of an intrinsically disordered recognition motif to a rigid partner protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJI6JdwJSqELVg90H21EOLACvtfcHk0lgpoEs93P4ZHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KnsbzE&md5=b68488d6d9534f1c640811b58e851ac3</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b07939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b07939%26sid%3Dliteratum%253Aachs%26aulast%3DBier%26aufirst%3DD.%26aulast%3DMittal%26aufirst%3DS.%26aulast%3DBravo-Rodriguez%26aufirst%3DK.%26aulast%3DSowislok%26aufirst%3DA.%26aulast%3DGuillory%26aufirst%3DX.%26aulast%3DBriels%26aufirst%3DJ.%26aulast%3DHeid%26aufirst%3DC.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DWettig%26aufirst%3DB.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DSanchez-Garcia%26aufirst%3DE.%26aulast%3DSchrader%26aufirst%3DT.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DThe%2520Molecular%2520Tweezer%2520CLR01%2520Stabilizes%2520a%2520Disordered%2520Protein-Protein%2520Interface%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D45%26spage%3D16256%26epage%3D16263%26doi%3D10.1021%2Fjacs.7b07939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">An Optimised Small-Molecule Stabiliser of the 14–3-3-PMA2 Protein-Protein Interaction</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">6520</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1002/chem.201103761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fchem.201103761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22467351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=6520-6527&issue=21&author=A.+Richterauthor=R.+Roseauthor=C.+Hedbergauthor=H.+Waldmannauthor=C.+Ottmann&title=An+Optimised+Small-Molecule+Stabiliser+of+the+14%E2%80%933-3-PMA2+Protein-Protein+Interaction&doi=10.1002%2Fchem.201103761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">An Optimised Small-Molecule Stabiliser of the 14-3-3-PMA2 Protein-Protein Interaction</span></div><div class="casAuthors">Richter, Anja; Rose, Rolf; Hedberg, Christian; Waldmann, Herbert; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6520-6527, S6520/1-S6520/11</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Modulation of protein-protein interactions (PPIs) is a highly demanding, but also a very promising approach in chem. biol. and targeted drug discovery.  In contrast to inhibiting PPIs with small, chem. tractable mols., stabilization of these interactions can only be achieved with complex natural products, like rapamycin, FK506, taxol, forskolin, brefeldin and fusicoccin.  Fusicoccin stabilizes the activatory complex of the plant H+-ATPase PMA2 and 14-3-3 proteins.  Recently, the stabilizing effect of fusicoccin could be mimicked by a trisubstituted pyrrolinone (pyrrolidone1, 1).  Here, the authors report the synthesis, functional activity and crystal structure of derivs. of 1 that stabilize the 14-3-3-PMA2 complex.  With a limited compd. collection three modifications that are important for activity enhancement could be detd.: (1) conversion of the pyrrolinone scaffold into a pyrazole, (2) introduction of a tetrazole moiety to the Ph ring that contacts PMA2, and (3) addn. of a bromine to the Ph ring that exclusively contacts the 14-3-3 protein.  The crystal structure of a pyrazole deriv. of 1 in complex with 14-3-3 and PMA2 revealed that the more rigid core of this mol. positions the stabilizer deeper into the rim of the interface, enlarging esp. the contact surface to PMA2.  Combination of the aforementioned features gave rise to a mol. (I) that displays a threefold increase in stabilizing the 14-3-3-PMA2 complex over 1.  Compd. I and the other active derivs. show no effect on two other important 14-3-3 protein-protein interactions, i.e., with CRaf and p53.  This is the first study that describes the successful optimization of a PPI stabilizer identified by screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF-pisQc62uLVg90H21EOLACvtfcHk0lgpoEs93P4ZHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGmsbg%253D&md5=59d07edb06986bf0f8bca795b2cd07b9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fchem.201103761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201103761%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DR.%26aulast%3DHedberg%26aufirst%3DC.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DAn%2520Optimised%2520Small-Molecule%2520Stabiliser%2520of%2520the%252014%25E2%2580%25933-3-PMA2%2520Protein-Protein%2520Interaction%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2012%26volume%3D18%26issue%3D21%26spage%3D6520%26epage%3D6527%26doi%3D10.1002%2Fchem.201103761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skora, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Identification of Two Secondary Ligand Binding Sites in 14–3-3 Proteins Using Fragment Screening</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">3972</span>– <span class="NLM_lpage">3982</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.7b00153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.7b00153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A280%3ADC%252BC1cjkvFKnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=3972-3982&issue=30&author=E.+Sijbesmaauthor=L.+Skoraauthor=S.+Leysenauthor=L.+Brunsveldauthor=U.+Kochauthor=P.+Nussbaumerauthor=W.+Jahnkeauthor=C.+Ottmann&title=Identification+of+Two+Secondary+Ligand+Binding+Sites+in+14%E2%80%933-3+Proteins+Using+Fragment+Screening&doi=10.1021%2Facs.biochem.7b00153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Two Secondary Ligand Binding Sites in 14-3-3 Proteins Using Fragment Screening</span></div><div class="casAuthors">Sijbesma Eline; Leysen Seppe; Brunsveld Luc; Ottmann Christian; Skora Lukasz; Jahnke Wolfgang; Koch Uwe; Nussbaumer Peter; Ottmann Christian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">3972-3982</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteins typically interact with multiple binding partners, and often different parts of their surfaces are employed to establish these protein-protein interactions (PPIs).  Members of the class of 14-3-3 adapter proteins bind to several hundred other proteins in the cell.  Multiple small molecules for the modulation of 14-3-3 PPIs have been disclosed; however, they all target the conserved phosphopeptide binding channel, so that selectivity is difficult to achieve.  Here we report on the discovery of two individual secondary binding sites that have been identified by combining nuclear magnetic resonance-based fragment screening and X-ray crystallography.  The two pockets that these fragments occupy are part of at least three physiologically relevant and structurally characterized 14-3-3 PPI interfaces, including those with serotonin N-acetyltransferase and plant transcription factor FT.  In addition, the high degree of conservation of the two sites implies their relevance for 14-3-3 PPIs.  This first identification of secondary sites on 14-3-3 proteins bound by small molecule ligands might facilitate the development of new chemical tool compounds for more selective PPI modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDbS6TEx8PgqNc3HjJRoFAfW6udTcc2eZXn8tR7LpaiLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjkvFKnsg%253D%253D&md5=0f05f0492b5dee35edb079ce8d945a98</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.7b00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.7b00153%26sid%3Dliteratum%253Aachs%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DSkora%26aufirst%3DL.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DKoch%26aufirst%3DU.%26aulast%3DNussbaumer%26aufirst%3DP.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520Two%2520Secondary%2520Ligand%2520Binding%2520Sites%2520in%252014%25E2%2580%25933-3%2520Proteins%2520Using%2520Fragment%2520Screening%26jtitle%3DBiochemistry%26date%3D2017%26volume%3D56%26issue%3D30%26spage%3D3972%26epage%3D3982%26doi%3D10.1021%2Facs.biochem.7b00153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valenti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrelle, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hristeva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini
Santo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obšil, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanoulle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzalis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Set-up and Screening of a Fragment Library Targeting the 14–3-3 Protein Interface</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1796</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1039/C9MD00215D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1039%2FC9MD00215D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=31814953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVWjsbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1796-1802&issue=10&author=D.+Valentiauthor=J.+F.+Nevesauthor=F.-X.+Cantrelleauthor=S.+Hristevaauthor=D.+Lentini%0ASantoauthor=T.+Ob%C5%A1ilauthor=X.+Hanoulleauthor=L.+M.+Levyauthor=D.+Tzalisauthor=I.+Landrieuauthor=C.+Ottmann&title=Set-up+and+Screening+of+a+Fragment+Library+Targeting+the+14%E2%80%933-3+Protein+Interface&doi=10.1039%2FC9MD00215D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Set-up and screening of a fragment library targeting the 14-3-3 protein interface</span></div><div class="casAuthors">Valenti, Dario; Neves, Joao Filipe; Cantrelle, Francois-Xavier; Hristeva, Stanimira; Lentini Santo, Domenico; Obsil, Tomas; Hanoulle, Xavier; Levy, Laura M.; Tzalis, Dimitrios; Landrieu, Isabelle; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1796-1802</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are at the core of regulation mechanisms in biol. systems and consequently became an attractive target for therapeutic intervention.  PPIs involving the adapter protein 14-3-3 are representative examples given the broad range of partner proteins forming a complex with one of its seven human isoforms.  Given the challenges represented by the nature of these interactions, fragment-based approaches offer a valid alternative for the development of PPI modulators.  After having assembled a fragment set tailored on PPIs' modulation, we started a screening campaign on the sigma isoform of 14-3-3 adapter proteins.  Through the use of both mono- and bi-dimensional NMR spectroscopy measurements, coupled with differential scanning fluorimetry, three fragment hits were identified.  These mols. bind the protein at two different regions distant from the usual binding groove highlighting new possibilities for selective modulation of 14-3-3 complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwc2MbT8kL7rVg90H21EOLACvtfcHk0ljKMcm9-SdyDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVWjsbfM&md5=07f022f6f9b5f8d225dd5b7562679934</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1039%2FC9MD00215D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00215D%26sid%3Dliteratum%253Aachs%26aulast%3DValenti%26aufirst%3DD.%26aulast%3DNeves%26aufirst%3DJ.%2BF.%26aulast%3DCantrelle%26aufirst%3DF.-X.%26aulast%3DHristeva%26aufirst%3DS.%26aulast%3DLentini%2BSanto%26aufirst%3DD.%26aulast%3DOb%25C5%25A1il%26aufirst%3DT.%26aulast%3DHanoulle%26aufirst%3DX.%26aulast%3DLevy%26aufirst%3DL.%2BM.%26aulast%3DTzalis%26aufirst%3DD.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSet-up%2520and%2520Screening%2520of%2520a%2520Fragment%2520Library%2520Targeting%2520the%252014%25E2%2580%25933-3%2520Protein%2520Interface%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26issue%3D10%26spage%3D1796%26epage%3D1802%26doi%3D10.1039%2FC9MD00215D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallenbeck, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vink, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skóra, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Site-Directed Fragment-Based Screening for the Discovery of Protein-Protein Interaction Stabilizers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">3524</span>– <span class="NLM_lpage">3531</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b11658</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b11658" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVWjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=3524-3531&issue=8&author=E.+Sijbesmaauthor=K.+K.+Hallenbeckauthor=S.+Leysenauthor=P.+J.+de+Vinkauthor=L.+Sk%C3%B3raauthor=W.+Jahnkeauthor=L.+Brunsveldauthor=M.+R.+Arkinauthor=C.+Ottmann&title=Site-Directed+Fragment-Based+Screening+for+the+Discovery+of+Protein-Protein+Interaction+Stabilizers&doi=10.1021%2Fjacs.8b11658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Site-Directed Fragment-Based Screening for the Discovery of Protein-Protein Interaction Stabilizers</span></div><div class="casAuthors">Sijbesma, Eline; Hallenbeck, Kenneth K.; Leysen, Seppe; de Vink, Pim J.; Skora, Lukasz; Jahnke, Wolfgang; Brunsveld, Luc; Arkin, Michelle R.; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3524-3531</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modulation of protein-protein interactions (PPIs) by small mols. has emerged as a valuable approach in drug discovery.  Compared to direct inhibition, PPI stabilization is vastly underexplored but has strong advantages, including the ability to gain selectivity by targeting an interface formed only upon assocn. of proteins.  Here, we present the application of a site-directed screening technique based on disulfide trapping (tethering) to select for fragments that enhance the affinity between protein partners.  We target the phosphorylation-dependent interaction between the hub protein 14-3-3σ and a peptide derived from Estrogen Receptor α (ERα), an important breast cancer target that is neg. regulated by 14-3-3σ.  We identify orthosteric stabilizers that increase 14-3-3/ERα affinity up to 40-fold and propose the mechanism of stabilization based on X-ray crystal structures.  These fragments already display partial selectivity toward ERα-like motifs over other representative 14-3-3 clients.  This first of its kind study illustrates the potential of the tethering approach to overcome the hurdles in systematic PPI stabilizer discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1_kzNxRmRPrVg90H21EOLACvtfcHk0ljKMcm9-SdyDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVWjtrY%253D&md5=5e68f397675437fffdca4cc47ed8e078</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b11658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b11658%26sid%3Dliteratum%253Aachs%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DHallenbeck%26aufirst%3DK.%2BK.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3Dde%2BVink%26aufirst%3DP.%2BJ.%26aulast%3DSk%25C3%25B3ra%26aufirst%3DL.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSite-Directed%2520Fragment-Based%2520Screening%2520for%2520the%2520Discovery%2520of%2520Protein-Protein%2520Interaction%2520Stabilizers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26issue%3D8%26spage%3D3524%26epage%3D3531%26doi%3D10.1021%2Fjacs.8b11658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skolnick, J.</span></span> <span> </span><span class="NLM_article-title">The Distribution of Ligand-Binding Pockets around Protein-Protein Interfaces Suggests a General Mechanism for Pocket Formation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">3784</span>– <span class="NLM_lpage">3789</span>, <span class="refDoi"> DOI: 10.1073/pnas.1117768109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1073%2Fpnas.1117768109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22355140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvFymsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=3784-3789&issue=10&author=M.+Gaoauthor=J.+Skolnick&title=The+Distribution+of+Ligand-Binding+Pockets+around+Protein-Protein+Interfaces+Suggests+a+General+Mechanism+for+Pocket+Formation&doi=10.1073%2Fpnas.1117768109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The distribution of ligand-binding pockets around protein-protein interfaces suggests a general mechanism for pocket formation</span></div><div class="casAuthors">Gao, Mu; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3784-3789</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein-protein and protein-ligand interactions are ubiquitous in a biol. cell.  Here, we report a comprehensive study of the distribution of protein-ligand interaction sites, namely ligand-binding pockets, around protein-protein interfaces where protein-protein interactions occur.  We inspected a representative set of 1,611 representative protein-protein complexes and identified pockets with a potential for binding small mol. ligands.  The majority of these pockets are within a 6 Å distance from protein interfaces.  Accordingly, in about half of ligand-bound protein-protein complexes, amino acids from both sides of a protein interface are involved in direct contacts with at least one ligand.  Statistically, ligands are closer to a protein-protein interface than a random surface patch of the same solvent accessible surface area.  Similar results are obtained in an anal. of the ligand distribution around domain-domain interfaces of 1,416 nonredundant, two-domain protein structures.  Furthermore, comparable sized pockets as obsd. in exptl. structures are present in artificially generated protein complexes, suggesting that the prominent appearance of pockets around protein interfaces is mainly a structural consequence of protein packing and thus, is an intrinsic geometric feature of protein structure.  Nature may take advantage of such a structural feature by selecting and further optimizing for biol. function.  We propose that packing nearby protein-protein or domain-domain interfaces is a major route to the formation of ligand-binding pockets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMMAJh2-zxkrVg90H21EOLACvtfcHk0ljKMcm9-SdyDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvFymsbc%253D&md5=696417061d6c6575c0b4608bffdc3dd3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1117768109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1117768109%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DM.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DThe%2520Distribution%2520of%2520Ligand-Binding%2520Pockets%2520around%2520Protein-Protein%2520Interfaces%2520Suggests%2520a%2520General%2520Mechanism%2520for%2520Pocket%2520Formation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D10%26spage%3D3784%26epage%3D3789%26doi%3D10.1073%2Fpnas.1117768109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jeffrey A. Dewey, Saara-Anne Azizi, Vivian Lu, <span class="NLM_string-name hlFld-ContribAuthor">Bryan C. Dickinson</span>. </span><span class="cited-content_cbyCitation_article-title">A System for the Evolution of Protein–Protein Interaction Inducers. </span><span class="cited-content_cbyCitation_journal-name">ACS Synthetic Biology</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acssynbio.1c00276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facssynbio.1c00276%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Synthetic%2520Biology%26atitle%3DA%252BSystem%252Bfor%252Bthe%252BEvolution%252Bof%252BProtein%2525E2%252580%252593Protein%252BInteraction%252BInducers%26aulast%3DDewey%26aufirst%3DJeffrey%2BA.%26date%3D2021%26date%3D2021%26date%3D16062021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alice Ballone, Francesca Picarazzi, Christine Prosser, Jeremy Davis, Christian Ottmann, <span class="NLM_string-name hlFld-ContribAuthor">Mattia Mori</span>. </span><span class="cited-content_cbyCitation_article-title">Experimental and Computational Druggability Exploration of the 14-3-3ζ/SOS1pS1161 PPI Interface. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6555-6565. <a href="https://doi.org/10.1021/acs.jcim.0c00722" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00722</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00722%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExperimental%252Band%252BComputational%252BDruggability%252BExploration%252Bof%252Bthe%252B14-3-3%2525CE%2525B6%25252FSOS1pS1161%252BPPI%252BInterface%26aulast%3DBallone%26aufirst%3DAlice%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D25062020%26date%3D02112020%26volume%3D60%26issue%3D12%26spage%3D6555%26epage%3D6565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiaqi  Liu</span>, <span class="hlFld-ContribAuthor ">Shengliang  Cao</span>, <span class="hlFld-ContribAuthor ">Guofei  Ding</span>, <span class="hlFld-ContribAuthor ">Bin  Wang</span>, <span class="hlFld-ContribAuthor ">Yingchao  Li</span>, <span class="hlFld-ContribAuthor ">Yuzhong  Zhao</span>, <span class="hlFld-ContribAuthor ">Qingyuan  Shao</span>, <span class="hlFld-ContribAuthor ">Jian  Feng</span>, <span class="hlFld-ContribAuthor ">Sidang  Liu</span>, <span class="hlFld-ContribAuthor ">Liting  Qin</span>, <span class="hlFld-ContribAuthor ">Yihong  Xiao</span>. </span><span class="cited-content_cbyCitation_article-title">The role of 14‐3‐3 proteins in cell signalling pathways and virus infection. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular and Molecular Medicine</span><span> <strong>2021,</strong> <em>25 </em>
                                    (9)
                                     , 4173-4182. <a href="https://doi.org/10.1111/jcmm.16490" title="DOI URL">https://doi.org/10.1111/jcmm.16490</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/jcmm.16490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fjcmm.16490%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520and%2520Molecular%2520Medicine%26atitle%3DThe%252Brole%252Bof%252B14%2525E2%252580%2525903%2525E2%252580%2525903%252Bproteins%252Bin%252Bcell%252Bsignalling%252Bpathways%252Band%252Bvirus%252Binfection%26aulast%3DLiu%26aufirst%3DJiaqi%26date%3D2021%26date%3D2021%26volume%3D25%26issue%3D9%26spage%3D4173%26epage%3D4182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tatjana  Ruks</span>, <span class="hlFld-ContribAuthor ">Kateryna  Loza</span>, <span class="hlFld-ContribAuthor ">Marc  Heggen</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>, <span class="hlFld-ContribAuthor ">Peter  Bayer</span>, <span class="hlFld-ContribAuthor ">Christine  Beuck</span>, <span class="hlFld-ContribAuthor ">Matthias  Epple</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the Surface of the Protein 14‐3‐3 by Ultrasmall (1.5 nm) Gold Nanoparticles Carrying the Specific Peptide CRaf. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (8)
                                     , 1456-1463. <a href="https://doi.org/10.1002/cbic.202000761" title="DOI URL">https://doi.org/10.1002/cbic.202000761</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000761%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DTargeting%252Bthe%252BSurface%252Bof%252Bthe%252BProtein%252B14%2525E2%252580%2525903%2525E2%252580%2525903%252Bby%252BUltrasmall%252B%2525281.5%2525E2%252580%252585nm%252529%252BGold%252BNanoparticles%252BCarrying%252Bthe%252BSpecific%252BPeptide%252BCRaf%26aulast%3DRuks%26aufirst%3DTatjana%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D8%26spage%3D1456%26epage%3D1463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lorenzo  Soini</span>, <span class="hlFld-ContribAuthor ">Seppe  Leysen</span>, <span class="hlFld-ContribAuthor ">Jeremy  Davis</span>, <span class="hlFld-ContribAuthor ">Marta  Westwood</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>. </span><span class="cited-content_cbyCitation_article-title">The 14‐3‐3/SLP76 protein–protein interaction in T‐cell receptor signalling: a structural and biophysical characterization. </span><span class="cited-content_cbyCitation_journal-name">FEBS Letters</span><span> <strong>2021,</strong> <em>595 </em>
                                    (3)
                                     , 404-414. <a href="https://doi.org/10.1002/1873-3468.13993" title="DOI URL">https://doi.org/10.1002/1873-3468.13993</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/1873-3468.13993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F1873-3468.13993%26sid%3Dliteratum%253Aachs%26jtitle%3DFEBS%2520Letters%26atitle%3DThe%252B14%2525E2%252580%2525903%2525E2%252580%2525903%25252FSLP76%252Bprotein%2525E2%252580%252593protein%252Binteraction%252Bin%252BT%2525E2%252580%252590cell%252Breceptor%252Bsignalling%25253A%252Ba%252Bstructural%252Band%252Bbiophysical%252Bcharacterization%26aulast%3DSoini%26aufirst%3DLorenzo%26date%3D2021%26date%3D2020%26volume%3D595%26issue%3D3%26spage%3D404%26epage%3D414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lorenzo  Soini</span>, <span class="hlFld-ContribAuthor ">Martin  Redhead</span>, <span class="hlFld-ContribAuthor ">Marta  Westwood</span>, <span class="hlFld-ContribAuthor ">Seppe  Leysen</span>, <span class="hlFld-ContribAuthor ">Jeremy  Davis</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of molecular glues of the SLP76/14-3-3 protein–protein interaction. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>119 </em><a href="https://doi.org/10.1039/D1MD00172H" title="DOI URL">https://doi.org/10.1039/D1MD00172H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00172H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00172H%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bmolecular%252Bglues%252Bof%252Bthe%252BSLP76%25252F14-3-3%252Bprotein%2525E2%252580%252593protein%252Binteraction%26aulast%3DSoini%26aufirst%3DLorenzo%26date%3D2021%26date%3D2021%26volume%3D119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ghazi  Aljabal</span>, <span class="hlFld-ContribAuthor ">Beow Keat  Yap</span>. </span><span class="cited-content_cbyCitation_article-title">14-3-3σ and Its Modulators in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2020,</strong> <em>13 </em>
                                    (12)
                                     , 441. <a href="https://doi.org/10.3390/ph13120441" title="DOI URL">https://doi.org/10.3390/ph13120441</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph13120441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph13120441%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3D14-3-3%2525CF%252583%252Band%252BIts%252BModulators%252Bin%252BCancer%26aulast%3DAljabal%26aufirst%3DGhazi%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D12%26spage%3D441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alice  Ballone</span>, <span class="hlFld-ContribAuthor ">Roxanne A.  Lau</span>, <span class="hlFld-ContribAuthor ">Fabian P. A.  Zweipfenning</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>. </span><span class="cited-content_cbyCitation_article-title">A new soaking procedure for X-ray crystallographic structural determination of protein–peptide complexes. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section F Structural Biology Communications</span><span> <strong>2020,</strong> <em>76 </em>
                                    (10)
                                     , 501-507. <a href="https://doi.org/10.1107/S2053230X2001122X" title="DOI URL">https://doi.org/10.1107/S2053230X2001122X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2053230X2001122X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2053230X2001122X%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520F%2520Structural%2520Biology%2520Communications%26atitle%3DA%252Bnew%252Bsoaking%252Bprocedure%252Bfor%252BX-ray%252Bcrystallographic%252Bstructural%252Bdetermination%252Bof%252Bprotein%2525E2%252580%252593peptide%252Bcomplexes%26aulast%3DBallone%26aufirst%3DAlice%26date%3D2020%26date%3D2020%26volume%3D76%26issue%3D10%26spage%3D501%26epage%3D507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Danlin  Liu</span>, <span class="hlFld-ContribAuthor ">Gavin  Richardson</span>, <span class="hlFld-ContribAuthor ">Fehmi M.  Benli</span>, <span class="hlFld-ContribAuthor ">Catherine  Park</span>, <span class="hlFld-ContribAuthor ">João V.  de Souza</span>, <span class="hlFld-ContribAuthor ">Agnieszka K.  Bronowska</span>, <span class="hlFld-ContribAuthor ">Ioakim  Spyridopoulos</span>. </span><span class="cited-content_cbyCitation_article-title">Inflammageing in the cardiovascular system: mechanisms, emerging targets, and novel therapeutic strategies. </span><span class="cited-content_cbyCitation_journal-name">Clinical Science</span><span> <strong>2020,</strong> <em>134 </em>
                                    (17)
                                     , 2243-2262. <a href="https://doi.org/10.1042/CS20191213" title="DOI URL">https://doi.org/10.1042/CS20191213</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/CS20191213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FCS20191213%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Science%26atitle%3DInflammageing%252Bin%252Bthe%252Bcardiovascular%252Bsystem%25253A%252Bmechanisms%25252C%252Bemerging%252Btargets%25252C%252Band%252Bnovel%252Btherapeutic%252Bstrategies%26aulast%3DLiu%26aufirst%3DDanlin%26date%3D2020%26date%3D2020%26volume%3D134%26issue%3D17%26spage%3D2243%26epage%3D2262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter  Bayer</span>, <span class="hlFld-ContribAuthor ">Anja  Matena</span>, <span class="hlFld-ContribAuthor ">Christine  Beuck</span>. </span><span class="cited-content_cbyCitation_article-title">NMR Spectroscopy of supramolecular chemistry on protein surfaces. </span><span class="cited-content_cbyCitation_journal-name">Beilstein Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>16 </em>, 2505-2522. <a href="https://doi.org/10.3762/bjoc.16.203" title="DOI URL">https://doi.org/10.3762/bjoc.16.203</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3762/bjoc.16.203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3762%2Fbjoc.16.203%26sid%3Dliteratum%253Aachs%26jtitle%3DBeilstein%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DNMR%252BSpectroscopy%252Bof%252Bsupramolecular%252Bchemistry%252Bon%252Bprotein%252Bsurfaces%26aulast%3DBayer%26aufirst%3DPeter%26date%3D2020%26date%3D2020%26volume%3D16%26spage%3D2505%26epage%3D2522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Identification of fragments binding to the interface of 14-3-3σ with p53. (A) Fragment AZ-001. (B) Crystal structure of AZ-001 (yellow sticks) in complex with 14-3-3σ (white surface) and p53pT387 (orange sticks), PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S40">6S40</a>. The final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown as blue mesh. (C) Detailed view of the binding pocket of AZ-001. The most prominent interaction is a salt-bridge (dotted black line) between the amidine of AZ-001 and carboxyl group of E14 of 14-3-3σ. (D) Fragment AZ-002. (E) Crystal structure of AZ-002 (yellow sticks) in complex with 14-3-3σ (white surface) and p53pT387 (orange sticks), PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWI">6RWI</a>. The final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown as blue mesh. (F) Detailed view of the binding pocket of AZ-002. Also with AZ-002, the most prominent interaction is a salt-bridge (dotted black line) between the amidine of AZ-002 and carboxyl group of E14 of 14-3-3σ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. AZ-003 bound to 14-3-3σ/TAZpS89. (A) Structure of fragment AZ-003. (B) Crystal structure of AZ-003 (yellow sticks) bound to the complex of 14-3-3σ (white surface) and TAZpS89 (green sticks), PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RLH">6RLH</a>. The final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown as blue mesh. (C) Contacts between AZ-003, 14-3-3σ, and TAZpS89. The structure of TAZpS89 bound to 14-3-3σ in the absence of AZ-003 is superimposed as magenta sticks. (D) Details of the changes binding of AZ-003 induces in the conformation of the TAZpS89 peptide. The peptide backbone flip is indicated by an orange arrow, and the distance between the positions of the main-chain carbonyl oxygen is shown. Polar contacts between the amino group of AZ-003 and main-chain carbonyls of TAZpS89 are indicated as dotted, black lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. AZ fragments that reach into the water network between 14-3-3σ and p53pT387. (A) Fragments AZ-005, AZ-006, and AZ-007. (B) Crystal structures of the fragments bound to the complex of 14-3-3σ (white surface) and p53pT387 (orange sticks), PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RX2">6RX2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5L">6R5L</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWH">6RWH</a>, respectively. The final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown as blue mesh. (C) Contacts between the fragments and the water network between 14-3-3σ and p53pT387.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. AZ-008 binds to the 14-3-3σ/p53pT387 complex. (A) Structure of AZ-008. (B) Docking of AZ-008 (yellow sticks) bound to the 14-3-3σ (white sticks) in complex with p53pT387 (orange sticks). (C) Mapping on the crystal structure of 14-3-3σ (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YZ5">1YZ5</a>) of the amino acid residues corresponding to the most affected resonances by the presence of AZ-008. Residues whose corresponding correlation peak intensities were the most affected are colored in red, and an additional 5–8 are colored in brown. (D) Plot of the intensity ratios <i>I</i>/<i>I</i><sub>0</sub> of corresponding <sup>1</sup>H–<sup>15</sup>N correlation peaks in the 2D 14-3-3σ spectra corresponding to the bound 14-3-3σ (100 μM 14-3-3σ in the presence of 2000 μM AZ-008, <i>I</i>) and free 14-3-3σ (100 μM 14-3-3σ <i>I</i><sub>0</sub>) protein (<i>y</i> axis) versus 14-3-3σ amino acid sequence (<i>x</i> axis). A total of 133 correlation peak intensities are shown. The <i>x</i> axis is not proportional to the sequence length. The helices of 14-3-3σ are identified below the <i>x</i> axis as blue cartoons, while disordered regions are represented by green lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/medium/jm9b01942_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. AZ-008 stabilizes the 14-3-3σ/p53pT387 complex. (A) Plot of the intensity ratios <i>I</i>/<i>I</i><sub>0</sub> (<i>y</i>-axis) of corresponding <sup>1</sup>H–<sup>15</sup>N correlation peaks in the 2D 14-3-3σΔC17 spectra of, in red: the peptide-bound 14-3-3σΔC17 (100 μM 14-3-3σ in the presence of 500 μM p53pT387, <i>I</i>) and free 14-3-3σ (100 μM 14-3-3σ <i>I</i><sub>0</sub>); in blue: corresponding to the peptide and fragment-bound 14-3-3σ (100 μM 14-3-3σ with 500 μM p53pT387 and 2000 μM AZ-008, <i>I</i>) and free 14-3-3σ (100 μM 14-3-3σ, <i>I</i><sub>0</sub>) protein. Note that the boxed regions correspond to the same areas affected in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> D and therefore show that the binding site of the fragment is the same when the peptide is present or not. (B) <sup>1</sup>H NMR signals of the spectrum of 100 μM 14-3-3σ (in black), 100 μM 14-3-3σ in the presence of 500 μM p53pT387 (in red), and 100 μM 14-3-3σ in the simultaneous presence of 500 μM p53pT387 and 2000 μM AZ-008 (in blue). The NMR signals correspond to the Hδ protons of L383 of the p53pT387 peptide. CSP in the peptide signals can be observed when AZ-008 is added to the solution containing the 14-3-3σ/p53pT387 complex. (C,D) Plot of the ratios of the bound (<i>I</i>)/free (<i>I</i><sub>0</sub>) <sup>1</sup>H–<sup>15</sup>N TROSY-HSQC correlation peak intensities of individual resonances corresponding to residues of 100 μM 14-3-3σ, remote from the interaction site (C) or close to the p53pT387 interaction site of (D), in the presence of 500 μM p53pT387 (in red) and 100 μM 14-3-3σ in the simultaneous presence of 500 μM p53pT387 and 2000 μM AZ-008 (in blue). Error bars of <i>I</i>/<i>I</i><sub>0</sub> ratios were calculated based on the maximum noise estimate of each spectrum. Note that the resonances corresponding to regions affected by the p53pT387 peptide binding were considerably more affected in the presence of AZ-008. (E) Titration of 14-3-3σ to a fluorescently labeled p53pT387 peptide in the absence and presence of 1 mM of AZ-008. Shown are average and standard deviation of three independent experiments. (F) SPR measurement of 14-3-3σ binding to immobilized p53pT387 peptide in the absence and presence of 1 mM AZ-008. Shown are average and standard deviation of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01942/20200701/images/large/jm9b01942_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01942&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73574" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73574" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1102</span>– <span class="NLM_lpage">1114</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.chembiol.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=25237857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1102-1114&issue=9&author=M.+R.+Arkinauthor=Y.+Tangauthor=J.+A.+Wells&title=Small-Molecule+Inhibitors+of+Protein-Protein+Interactions%3A+Progressing+toward+the+Reality&doi=10.1016%2Fj.chembiol.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span></div><div class="casAuthors">Arkin, Michelle R.; Tang, Yinyan; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1114</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The past 20 years have seen many advances in our understanding of protein-protein interactions (PPIs) and how to target them with small-mol. therapeutics.  In 2004, we reviewed some early successes; since then, potent inhibitors have been developed for diverse protein complexes, and compds. are now in clin. trials for six targets.  Surprisingly, many of these PPI clin. candidates have efficiency metrics typical of "lead-like" or "drug-like" mols. and are orally available.  Successful discovery efforts have integrated multiple disciplines and make use of all the modern tools of target-based discovery-structure, computation, screening, and biomarkers.  PPIs become progressively more challenging as the interfaces become more complex, i.e., as binding epitopes are displayed on primary, secondary, or tertiary structures.  Here, we review the last 10 years of progress, focusing on the properties of PPI inhibitors that have advanced to clin. trials and prospects for the future of PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP01wt-4lfE7Vg90H21EOLACvtfcHk0liF5oJCtjvOTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J&md5=aec3897dc9bc66656723bc56c8d69155</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-Molecule%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%253A%2520Progressing%2520toward%2520the%2520Reality%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26issue%3D9%26spage%3D1102%26epage%3D1114%26doi%3D10.1016%2Fj.chembiol.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briels, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Stabilization of Protein-Protein Interactions in Chemical Biology and Drug Discovery</span>. <i>Prog. Biophys. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1016/j.pbiomolbio.2015.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.pbiomolbio.2015.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=26093250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKmsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2015&pages=10-19&issue=1&author=D.+Bierauthor=P.+Thielauthor=J.+Brielsauthor=C.+Ottmann&title=Stabilization+of+Protein-Protein+Interactions+in+Chemical+Biology+and+Drug+Discovery&doi=10.1016%2Fj.pbiomolbio.2015.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of Protein-Protein Interactions in chemical biology and drug discovery</span></div><div class="casAuthors">Bier, David; Thiel, Philipp; Briels, Jeroen; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Biophysics & Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10-19</span>CODEN:
                <span class="NLM_cas:coden">PBIMAC</span>;
        ISSN:<span class="NLM_cas:issn">0079-6107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  More than 300,000 Protein-Protein Interactions (PPIs) can be found in human cells.  This no. is significantly larger than the no. of single proteins, which are the classical targets for pharmacol. intervention.  Hence, specific and potent modulation of PPIs by small, drug-like mols. would tremendously enlarge the "druggable genome" enabling novel ways of drug discovery for essentially every human disease.  This strategy is esp. promising in diseases with difficult targets like intrinsically disordered proteins or transcription factors, for example neurodegeneration or metabolic diseases.  Whereas the potential of PPI modulation has been recognized in terms of the development of inhibitors that disrupt or prevent a binary protein complex, the opposite (or complementary) strategy to stabilize PPIs has not yet been realized in a systematic manner.  This fact is rather surprising given the no. of impressive natural product examples that confer their activity by stabilizing specific PPIs.  In addn., in recent years more and more examples of synthetic mols. are being published that work as PPI stabilizers, despite the fact that in the majority they initially have not been designed as such.  Here, we describe examples from both the natural products as well as the synthetic mols. advocating for a stronger consideration of the PPI stabilization approach in chem. biol. and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAhN329QU_arVg90H21EOLACvtfcHk0liF5oJCtjvOTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKmsb3F&md5=8a51dfc8088641b0bfa658250cd6117a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.pbiomolbio.2015.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbiomolbio.2015.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DBier%26aufirst%3DD.%26aulast%3DThiel%26aufirst%3DP.%26aulast%3DBriels%26aufirst%3DJ.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DStabilization%2520of%2520Protein-Protein%2520Interactions%2520in%2520Chemical%2520Biology%2520and%2520Drug%2520Discovery%26jtitle%3DProg.%2520Biophys.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D119%26issue%3D1%26spage%3D10%26epage%3D19%26doi%3D10.1016%2Fj.pbiomolbio.2015.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrei, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karawajczyk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickhoff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doveston, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Stabilization of Protein-Protein Interactions in Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">925</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1346608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1080%2F17460441.2017.1346608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=28695752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOrsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=925-940&issue=9&author=S.+A.+Andreiauthor=E.+Sijbesmaauthor=M.+Hannauthor=J.+Davisauthor=G.+O%E2%80%99Mahonyauthor=M.+W.+D.+Perryauthor=A.+Karawajczykauthor=J.+Eickhoffauthor=L.+Brunsveldauthor=R.+G.+Dovestonauthor=L.-G.+Milroyauthor=C.+Ottmann&title=Stabilization+of+Protein-Protein+Interactions+in+Drug+Discovery&doi=10.1080%2F17460441.2017.1346608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of protein-protein interactions in drug discovery</span></div><div class="casAuthors">Andrei, Sebastian A.; Sijbesma, Eline; Hann, Michael; Davis, Jeremy; O'Mahony, Gavin; Perry, Matthew W. D.; Karawajczyk, Anna; Eickhoff, Jan; Brunsveld, Luc; Doveston, Richard G.; Milroy, Lech-Gustav; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">925-940</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">: PPIs are involved in every disease and specific modulation of these PPIs with small mols. would significantly improve our prospects of developing therapeutic agents.  Both industry and academia have engaged in the identification and use of PPI inhibitors.  However in comparison, the opposite strategy of employing small-mol. stabilizers of PPIs is underrepresented in drug discovery.: PPI stabilization has not been exploited in a systematic manner.  Rather, this concept validated by a no. of therapeutically used natural products like rapamycin and paclitaxel has been shown retrospectively to be the basis of the activity of synthetic mols. originating from drug discovery projects among them lenalidomide and tafamidis.  Here, the authors cover the growing no. of synthetic small-mol. PPI stabilizers to advocate for a stronger consideration of this as a drug discovery approach.: Both the natural products and the growing no. of synthetic mols. show that PPI stabilization is a viable strategy for drug discovery.  There is certainly a significant challenge to adapt compd. libraries, screening techniques and downstream methodologies to identify, characterize and optimize PPI stabilizers, but the examples of mols. reviewed here in our opinion justify these efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojOzbEj0fPcLVg90H21EOLACvtfcHk0lgeULDiack7bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOrsbnK&md5=fd6aa91fbca0e69af4a4bdb2f9ce850c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1346608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1346608%26sid%3Dliteratum%253Aachs%26aulast%3DAndrei%26aufirst%3DS.%2BA.%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DHann%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DG.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26aulast%3DKarawajczyk%26aufirst%3DA.%26aulast%3DEickhoff%26aufirst%3DJ.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DDoveston%26aufirst%3DR.%2BG.%26aulast%3DMilroy%26aufirst%3DL.-G.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DStabilization%2520of%2520Protein-Protein%2520Interactions%2520in%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2017%26volume%3D12%26issue%3D9%26spage%3D925%26epage%3D940%26doi%3D10.1080%2F17460441.2017.1346608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stafford, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKintosh, C.</span></span> <span> </span><span class="NLM_article-title">Bioinformatic and Experimental Survey of 14–3-3-Binding Sites</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>427</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1042/BJ20091834</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FBJ20091834" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=20141511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyntrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=427&publication_year=2010&pages=69-78&issue=1&author=C.+Johnsonauthor=S.+Crowtherauthor=M.+J.+Staffordauthor=D.+G.+Campbellauthor=R.+Tothauthor=C.+MacKintosh&title=Bioinformatic+and+Experimental+Survey+of+14%E2%80%933-3-Binding+Sites&doi=10.1042%2FBJ20091834"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bioinformatic and experimental survey of 14-3-3-binding sites</span></div><div class="casAuthors">Johnson, Catherine; Crowther, Sandra; Stafford, Margaret J.; Campbell, David G.; Toth, Rachel; MacKintosh, Carol</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">427</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-78</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">More than 200 phosphorylated 14-3-3-binding sites in the literature were analyzed to define 14-3-3 specificities, identify relevant protein kinases, and give insights into how cellular 14-3-3/phosphoprotein networks work.  Mode I RXX(pS/pT)XP motifs dominate, although the +2 proline residue occurs in less than half, and LX(R/K)SX(pS/pT)XP is prominent in plant 14-3-3-binding sites.  Proline at +1 is rarely reported, and such motifs did not stand up to exptl. reanal. of human Ndel1.  Instead, we discovered that 14-3-3 interacts with two residues that are phosphorylated by basophilic kinases and located in the DISC1 (disrupted-in-schizophrenia 1)-interacting region of Ndel1 that is implicated in cognitive disorders.  These data conform with the general findings that there are different subtypes of 14-3-3-binding sites that overlap with the specificities of different basophilic AGC (protein kinase A/protein kinase G/protein kinase C family) and CaMK (Ca2+/calmodulin-dependent protein kinase) protein kinases, and a 14-3-3 dimer often engages with two tandem phosphorylated sites, which is a configuration with special signalling, mech. and evolutionary properties.  Thus 14-3-3 dimers can be digital logic gates that integrate more than one input to generate an action, and coincidence detectors when the two binding sites are phosphorylated by different protein kinases.  Paired sites are generally located within disordered regions and/or straddle either side of functional domains, indicating how 14-3-3 dimers modulate the conformations and/or interactions of their targets.  Finally, 14-3-3 proteins bind to members of several multi-protein families.  Two 14-3-3-binding sites are conserved across the class IIa histone deacetylases, whereas other protein families display differential regulation by 14-3-3s.  We speculate that 14-3-3 dimers may have contributed to the evolution of such families, tailoring regulatory inputs to different physiol. demands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfY5xjqSQDjLVg90H21EOLACvtfcHk0lgeULDiack7bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyntrg%253D&md5=bfe2c524bc5efe8707004f1adb76bf55</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1042%2FBJ20091834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20091834%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DCrowther%26aufirst%3DS.%26aulast%3DStafford%26aufirst%3DM.%2BJ.%26aulast%3DCampbell%26aufirst%3DD.%2BG.%26aulast%3DToth%26aufirst%3DR.%26aulast%3DMacKintosh%26aufirst%3DC.%26atitle%3DBioinformatic%2520and%2520Experimental%2520Survey%2520of%252014%25E2%2580%25933-3-Binding%2520Sites%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D427%26issue%3D1%26spage%3D69%26epage%3D78%26doi%3D10.1042%2FBJ20091834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baljuls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinders, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nekhoroshkova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibilski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajalingam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hekman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, U. R.</span></span> <span> </span><span class="NLM_article-title">Regulation of RAF Activity by 14–3-3 Proteins: RAF Kinases Associate Functionally with Both Homo- and Heterodimeric Forms of 14–3-3 Proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">3183</span>– <span class="NLM_lpage">3194</span>, <span class="refDoi"> DOI: 10.1074/jbc.M804795200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1074%2Fjbc.M804795200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=19049963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpsV2mug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=3183-3194&issue=5&author=A.+Fischerauthor=A.+Baljulsauthor=J.+Reindersauthor=E.+Nekhoroshkovaauthor=C.+Sibilskiauthor=R.+Metzauthor=S.+Albertauthor=K.+Rajalingamauthor=M.+Hekmanauthor=U.+R.+Rapp&title=Regulation+of+RAF+Activity+by+14%E2%80%933-3+Proteins%3A+RAF+Kinases+Associate+Functionally+with+Both+Homo-+and+Heterodimeric+Forms+of+14%E2%80%933-3+Proteins&doi=10.1074%2Fjbc.M804795200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of RAF Activity by 14-3-3 Proteins: RAF Kinases Associate Functionally with both Homo and Heterodimeric forms of 14-3-3 Proteins</span></div><div class="casAuthors">Fischer, Andreas; Baljuls, Angela; Reinders, Joerg; Nekhoroshkova, Elena; Sibilski, Claudia; Metz, Renate; Albert, Stefan; Rajalingam, Krishnaraj; Hekman, Mirko; Rapp, Ulf R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3183-3194</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mammalian 14-3-3 proteins play a crucial role in the activation process of RAF kinases.  However, little is known about the selectivity of the mammalian 14-3-3 isoforms with respect to RAF assocn. and activation.  Using mass spectrometry, we analyzed the compn. of the 14-3-3 isoforms attached to RAF kinases and found that B-RAF assocs. in vivo with 14-3-3 at much higher diversity than A- and C-RAF.  We also examd. in vitro binding of purified mammalian 14-3-3 proteins to RAF kinases using surface plasmon resonance techniques.  While B- and C-RAF exhibited binding to all seven 14-3-3 isoforms, A-RAF bound with considerably lower affinities to ε, τ, and σ 14-3-3.  These findings indicate that 14-3-3 proteins assoc. with RAF isoforms in a pronounced isoform-specific manner.  Because 14-3-3 proteins appear in dimeric forms, we addressed the question of whether both homo- and heterodimeric forms of 14-3-3 proteins participate in RAF signaling.  For that purpose, the budding yeast Saccharomyces cerevisiae, possessing only two 14-3-3 isoforms (BMH1 and BMH2), served as testing system.  By deletion of the single BMH2 gene, we found that both homo- and heterodimeric forms of 14-3-3 can participate in RAF activation.  Furthermore, we show that A-, B-, and C-RAF activity is differentially regulated by its C-terminal and internal 14-3-3 binding domain.  Finally, prohibitin, a scaffold protein that affects C-RAF activation in a stimulatory manner, proved to interfere with the internal 14-3-3 binding site in C-RAF.  Together, our results shed more light on the complex mechanism of RAF activation, particularly with respect to activation steps that are mediated by 14-3-3 proteins and prohibitin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo1kD9RoVWHrVg90H21EOLACvtfcHk0lgeULDiack7bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpsV2mug%253D%253D&md5=8ef2f7b54787f7bdcd4594f97a4327a8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M804795200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M804795200%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DBaljuls%26aufirst%3DA.%26aulast%3DReinders%26aufirst%3DJ.%26aulast%3DNekhoroshkova%26aufirst%3DE.%26aulast%3DSibilski%26aufirst%3DC.%26aulast%3DMetz%26aufirst%3DR.%26aulast%3DAlbert%26aufirst%3DS.%26aulast%3DRajalingam%26aufirst%3DK.%26aulast%3DHekman%26aufirst%3DM.%26aulast%3DRapp%26aufirst%3DU.%2BR.%26atitle%3DRegulation%2520of%2520RAF%2520Activity%2520by%252014%25E2%2580%25933-3%2520Proteins%253A%2520RAF%2520Kinases%2520Associate%2520Functionally%2520with%2520Both%2520Homo-%2520and%2520Heterodimeric%2520Forms%2520of%252014%25E2%2580%25933-3%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26issue%3D5%26spage%3D3183%26epage%3D3194%26doi%3D10.1074%2Fjbc.M804795200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molzan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Synergistic Binding of the Phosphorylated S233- and S259-Binding Sites of C-RAF to One 14–3-3ζ Dimer</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>423</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">495</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.jmb.2012.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22922483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGqt7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=423&publication_year=2012&pages=486-495&issue=4&author=M.+Molzanauthor=C.+Ottmann&title=Synergistic+Binding+of+the+Phosphorylated+S233-+and+S259-Binding+Sites+of+C-RAF+to+One+14%E2%80%933-3%CE%B6+Dimer&doi=10.1016%2Fj.jmb.2012.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic Binding of the Phosphorylated S233- and S259-Binding Sites of C-RAF to One 14-3-3ζ Dimer</span></div><div class="casAuthors">Molzan, Manuela; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">423</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-495</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">C-RAF kinase is a central component of the Ras-RAF-MEK (mitogen-activated protein kinase/extracellular signal-regulated kinase)-ERK (extracellular signal-regulated kinase) pathway, which has been shown to be activated in 30% of human tumors.  14-3-3 Proteins inactivate C-RAF by binding to the two N-terminal phosphorylation-dependent binding sites surrounding S233 and S259.  14-3-3 Proteins can bind two target sequences located on one polypeptide chain simultaneously, thereby increasing binding affinity compared to single-site binding and possibly allowing regulated 14-3-3 binding through gatekeeper phosphorylation.  To date, it was unclear whether 14-3-3 proteins can bind the two N-terminal phosphorylation-dependent binding sites of C-RAF simultaneously.  Fluorescence polarization (FP) using phosphorylated peptides demonstrated that S233 is the low-affinity and S259 is the high-affinity binding site, while simultaneous engagement of both sites by 14-3-3ζ enhances affinity compared to single-site binding.  Detn. of a 1:1 stoichiometry for the di-phosphorylated peptide binding to one 14-3-3ζ dimer with isothermal titrn. calorimetry was supported by the crystal structure of the 14-3-3ζ/C-RAFpS233,pS259 complex.  Cellular localization studies validate the significance of these sites for cytoplasmic retention of C-RAF, suggesting an extended mechanism of RAF regulation by 14-3-3 proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWsGiYB2r9z7Vg90H21EOLACvtfcHk0lg0D1LTNKbZdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGqt7%252FN&md5=5b7d6cddd74ffb26a7b1d597da4a09fe</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DMolzan%26aufirst%3DM.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSynergistic%2520Binding%2520of%2520the%2520Phosphorylated%2520S233-%2520and%2520S259-Binding%2520Sites%2520of%2520C-RAF%2520to%2520One%252014%25E2%2580%25933-3%25CE%25B6%2520Dimer%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D423%26issue%3D4%26spage%3D486%26epage%3D495%26doi%3D10.1016%2Fj.jmb.2012.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molzan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röglin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skwarczynska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbuecher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkanda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Stabilization of Physical RAF/14–3-3 Interaction by Cotylenin A as Treatment Strategy for RAS Mutant Cancers</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1869</span>– <span class="NLM_lpage">1875</span>, <span class="refDoi"> DOI: 10.1021/cb4003464</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4003464" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVeisLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1869-1875&issue=9&author=M.+Molzanauthor=S.+Kasperauthor=L.+R%C3%B6glinauthor=M.+Skwarczynskaauthor=T.+Sassaauthor=T.+Inoueauthor=F.+Breitenbuecherauthor=J.+Ohkandaauthor=N.+Katoauthor=M.+Schulerauthor=C.+Ottmann&title=Stabilization+of+Physical+RAF%2F14%E2%80%933-3+Interaction+by+Cotylenin+A+as+Treatment+Strategy+for+RAS+Mutant+Cancers&doi=10.1021%2Fcb4003464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of Physical RAF/14-3-3 Interaction by Cotylenin A as Treatment Strategy for RAS Mutant Cancers</span></div><div class="casAuthors">Molzan, Manuela; Kasper, Stefan; Roeglin, Lars; Skwarczynska, Malgorzata; Sassa, Takeshi; Inoue, Takatsugu; Breitenbuecher, Frank; Ohkanda, Junko; Kato, Nobuo; Schuler, Martin; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1869-1875</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One-third of all human cancers harbor somatic RAS mutations.  This leads to aberrant activation of downstream signaling pathways involving the RAF kinases.  Current ATP-competitive RAF inhibitors are active in cancers with somatic RAF mutations, such as BRAFV600 mutant melanomas.  However, they paradoxically promote the growth of RAS mutant tumors, partly due to the complex interplay between different homo- and heterodimers of A-RAF, B-RAF, and C-RAF.  Based on pathway anal. and structure-guided compd. identification, we describe the natural product cotylenin-A (CN-A) as stabilizer of the phys. interaction of C-RAF with 14-3-3 proteins.  CN-A binds to inhibitory 14-3-3 interaction sites of C-RAF, pSer233, and pSer259, but not to the activating interaction site, pSer621.  While CN-A alone is inactive in RAS mutant cancer models, combined treatment with CN-A and an anti-EGFR antibody synergistically suppresses tumor growth in vitro and in vivo.  This defines a novel pharmacol. strategy for treatment of RAS mutant cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYNk-se5ruGrVg90H21EOLACvtfcHk0lg0D1LTNKbZdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVeisLrE&md5=f1dfbdbc95a185c7b893ebc45ac72e92</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb4003464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4003464%26sid%3Dliteratum%253Aachs%26aulast%3DMolzan%26aufirst%3DM.%26aulast%3DKasper%26aufirst%3DS.%26aulast%3DR%25C3%25B6glin%26aufirst%3DL.%26aulast%3DSkwarczynska%26aufirst%3DM.%26aulast%3DSassa%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DBreitenbuecher%26aufirst%3DF.%26aulast%3DOhkanda%26aufirst%3DJ.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DSchuler%26aufirst%3DM.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DStabilization%2520of%2520Physical%2520RAF%252F14%25E2%2580%25933-3%2520Interaction%2520by%2520Cotylenin%2520A%2520as%2520Treatment%2520Strategy%2520for%2520RAS%2520Mutant%2520Cancers%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26issue%3D9%26spage%3D1869%26epage%3D1875%26doi%3D10.1021%2Fcb4003464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leclerc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paudel, H. K.</span></span> <span> </span><span class="NLM_article-title">Interaction of 14–3-3ζ with Microtubule-Associated Protein Tau within Alzheimer’s Disease Neurofibrillary Tangles</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">37</span>),  <span class="NLM_fpage">6445</span>– <span class="NLM_lpage">6455</span>, <span class="refDoi"> DOI: 10.1021/bi400442d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi400442d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht12hurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=6445-6455&issue=37&author=H.+Y.+Qureshiauthor=T.+Liauthor=R.+MacDonaldauthor=C.+M.+Choauthor=N.+Leclercauthor=H.+K.+Paudel&title=Interaction+of+14%E2%80%933-3%CE%B6+with+Microtubule-Associated+Protein+Tau+within+Alzheimer%E2%80%99s+Disease+Neurofibrillary+Tangles&doi=10.1021%2Fbi400442d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of 14-3-3ζ with Microtubule-Associated Protein Tau within Alzheimer's Disease Neurofibrillary Tangles</span></div><div class="casAuthors">Qureshi, Hamid Y.; Li, Tong; MacDonald, Ryen; Cho, Chul Min; Leclerc, Nicole; Paudel, Hemant K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">6445-6455</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is characterized by the presence of abnormal, straight filaments and paired helical filaments (PHFs) that are coated with amorphous aggregates.  When PHFs are treated with alkali, they untwist and form filaments with a ribbonlike morphol.  Tau protein is the major component of all of these ultrastructures.  14-3-3ζ Is present in NFTs and is significantly upregulated in AD brain.  The mol. basis of the assocn. of 14-3-3ζ within NFTs and the pathol. significance of its assocn. are not known.  In this study, we have found that 14-3-3ζ is copurified and co-immunoppts. with tau from NFTs of AD brain ext.  In vitro, tau binds to both phosphorylated and nonphosphorylated tau.  When incubated with 14-3-3ζ, tau forms amorphous aggregates, single-stranded, straight filaments, ribbonlike filaments, and PHF-like filaments, all of which resemble the corresponding ultrastructures found in AD brain.  Immuno-electron microscopy detd. that both tau and 14-3-3ζ are present in these ultrastructures and that they are formed in an incubation time-dependent manner.  Amorphous aggregates are formed first.  As the incubation time increases, the size of amorphous aggregates increases and they are incorporated into single-stranded filaments.  Single-stranded filaments laterally assoc. to form double-stranded, ribbonlike, and PHF-like filaments.  Both tau and phosphorylated tau aggregate in a similar manner when they are incubated with 14-3-3ζ.  Our data suggest that 14-3-3ζ has a role in the fibrillization of tau in AD brain, and that tau phosphorylation does not affect 14-3-3ζ-induced tau aggregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5oervCC-UabVg90H21EOLACvtfcHk0lg0D1LTNKbZdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht12hurfP&md5=95c065c72827033dc4318cb09041603a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi400442d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi400442d%26sid%3Dliteratum%253Aachs%26aulast%3DQureshi%26aufirst%3DH.%2BY.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMacDonald%26aufirst%3DR.%26aulast%3DCho%26aufirst%3DC.%2BM.%26aulast%3DLeclerc%26aufirst%3DN.%26aulast%3DPaudel%26aufirst%3DH.%2BK.%26atitle%3DInteraction%2520of%252014%25E2%2580%25933-3%25CE%25B6%2520with%2520Microtubule-Associated%2520Protein%2520Tau%2520within%2520Alzheimer%25E2%2580%2599s%2520Disease%2520Neurofibrillary%2520Tangles%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26issue%3D37%26spage%3D6445%26epage%3D6455%26doi%3D10.1021%2Fbi400442d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smet-Nocca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, Y.-H.</span></span> <span> </span><span class="NLM_article-title">Involvement of 14–3-3 in Tubulin Instability and Impaired Axon Development Is Mediated by Tau</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4133</span>– <span class="NLM_lpage">4144</span>, <span class="refDoi"> DOI: 10.1096/fj.14-265009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1096%2Ffj.14-265009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=26103986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1SlsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=4133-4144&issue=10&author=Y.+Jooauthor=B.+Schumacherauthor=I.+Landrieuauthor=M.+Bartelauthor=C.+Smet-Noccaauthor=A.+Jangauthor=H.+S.+Choiauthor=N.+L.+Jeonauthor=K.-A.+Changauthor=H.-S.+Kimauthor=C.+Ottmannauthor=Y.-H.+Suh&title=Involvement+of+14%E2%80%933-3+in+Tubulin+Instability+and+Impaired+Axon+Development+Is+Mediated+by+Tau&doi=10.1096%2Ffj.14-265009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Involvement of 14-3-3 in tubulin instability and impaired axon development is mediated by Tau</span></div><div class="casAuthors">Joo, Yuyoung; Schumacher, Benjamin; Landrieu, Isabelle; Bartel, Maria; Smet-Nocca, Caroline; Jang, Ahram; Choi, Hee Soon; Jeon, Noo Li; Chang, Keun-A.; Kim, Hye-Sun; Ottmann, Christian; Suh, Yoo-Hun</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4133-4144</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">14-3-3 Proteins act as adapters that exert their function by interacting with their various protein partners.  14-3-3 Proteins have been implicated in a variety of human diseases including neurodegenerative diseases.  14-3-3 Proteins have recently been reported to be abundant in the neurofibrillary tangles (NFTs) obsd. inside the neurons of brains affected by Alzheimer's disease (AD).  These NFTs are mainly constituted of phosphorylated Tau protein, a microtubule-assocd. protein known to bind 14-3-3.  Despite this indication of 14-3-3 protein involvement in the AD pathogenesis, the role of 14-3-3 in the Tauopathy remains to be clarified.  In the present study, we shed light on the role of 14-3-3 proteins in the mol. pathways leading to Tauopathies.  Overexpression of the 14-3-3s isoform resulted in a disruption of the tubulin cytoskeleton and prevented neuritic outgrowth in neurons.  NMR studies validated the phosphorylated residues pSer214 and pSer324 in Tau as the 2 primary sites for 14-3-3 binding, with the crystal structure of 14-3-3s in complex with Tau-pSer214 and Tau-pSer324 revealing the mol. details of the interaction.  These data suggest a rationale for a possible pharmacol. intervention of the Tau/14-3-3 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhkxjaBskBDrVg90H21EOLACvtfcHk0li-BCXwqzM27w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1SlsL0%253D&md5=d0346a716b7362c6e69b253d18fea48d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1096%2Ffj.14-265009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.14-265009%26sid%3Dliteratum%253Aachs%26aulast%3DJoo%26aufirst%3DY.%26aulast%3DSchumacher%26aufirst%3DB.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DSmet-Nocca%26aufirst%3DC.%26aulast%3DJang%26aufirst%3DA.%26aulast%3DChoi%26aufirst%3DH.%2BS.%26aulast%3DJeon%26aufirst%3DN.%2BL.%26aulast%3DChang%26aufirst%3DK.-A.%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DSuh%26aufirst%3DY.-H.%26atitle%3DInvolvement%2520of%252014%25E2%2580%25933-3%2520in%2520Tubulin%2520Instability%2520and%2520Impaired%2520Axon%2520Development%2520Is%2520Mediated%2520by%2520Tau%26jtitle%3DFASEB%2520J.%26date%3D2015%26volume%3D29%26issue%3D10%26spage%3D4133%26epage%3D4144%26doi%3D10.1096%2Ffj.14-265009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bozoky, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krzeminski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhandiram, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birtley, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Zahrani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frizzell, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman-Kay, J. D.</span></span> <span> </span><span class="NLM_article-title">Regulatory R Region of the CFTR Chloride Channel Is a Dynamic Integrator of Phospho-Dependent Intra- and Intermolecular Interactions</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">47</span>),  <span class="NLM_fpage">E4427</span>– <span class="NLM_lpage">4436</span>, <span class="refDoi"> DOI: 10.1073/pnas.1315104110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1073%2Fpnas.1315104110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=24191035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCisLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=E4427-4436&issue=47&author=Z.+Bozokyauthor=M.+Krzeminskiauthor=R.+Muhandiramauthor=J.+R.+Birtleyauthor=A.+Al-Zahraniauthor=P.+J.+Thomasauthor=R.+A.+Frizzellauthor=R.+C.+Fordauthor=J.+D.+Forman-Kay&title=Regulatory+R+Region+of+the+CFTR+Chloride+Channel+Is+a+Dynamic+Integrator+of+Phospho-Dependent+Intra-+and+Intermolecular+Interactions&doi=10.1073%2Fpnas.1315104110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory R region of the CFTR chloride channel is a dynamic integrator of phospho-dependent intra- and intermolecular interactions</span></div><div class="casAuthors">Bozoky, Zoltan; Krzeminski, Mickael; Muhandiram, Ranjith; Birtley, James R.; Al-Zahrani, Ateeq; Thomas, Philip J.; Frizzell, Raymond A.; Ford, Robert C.; Forman-Kay, Julie D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">E4427-E4436,SE4427/1-SE4427/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Intrinsically disordered proteins (IDPs) play crucial roles in regulatory processes and often function as protein interaction hubs.  Here, we present a detailed characterization of a full-length disordered hub protein region involved in multiple dynamic complexes.  We performed NMR, CD, and fluorescence binding studies on the nonphosphorylated and highly PKA-phosphorylated human cystic fibrosis transmembrane conductance regulator (CFTR) regulatory region, a ∼200-residue disordered segment involved in phosphorylation-dependent regulation of channel trafficking and gating.  Our data provide evidence for dynamic, phosphorylation-dependent, multisite interactions of various segments of the regulatory region for its intra- and intermol. partners, including the CFTR nucleotide binding domains (NBDs) 1 and 2, a 42-residue peptide from the C terminus of CFTR, the SLC26A3 sulfate transporter and antisigma factor antagonist (STAS) domain, and 14-3-3β.  Because of its large no. of binding partners, multivalent binding of individually weak sites facilitates rapid exchange between free and bound states to allow the regulatory region to engage with different partners and generate a graded or rheostat-like response to phosphorylation.  Our results enrich the understanding of how disordered binding segments interact with multiple targets.  We present structural models consistent with our data that illustrate this dynamic aspect of phospho-regulation of CFTR by the disordered regulatory region.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyg0jY6--uubVg90H21EOLACvtfcHk0li-BCXwqzM27w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCisLzM&md5=1f8155a6eb49fbaa8fdc5677cf41fb0a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1315104110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1315104110%26sid%3Dliteratum%253Aachs%26aulast%3DBozoky%26aufirst%3DZ.%26aulast%3DKrzeminski%26aufirst%3DM.%26aulast%3DMuhandiram%26aufirst%3DR.%26aulast%3DBirtley%26aufirst%3DJ.%2BR.%26aulast%3DAl-Zahrani%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DP.%2BJ.%26aulast%3DFrizzell%26aufirst%3DR.%2BA.%26aulast%3DFord%26aufirst%3DR.%2BC.%26aulast%3DForman-Kay%26aufirst%3DJ.%2BD.%26atitle%3DRegulatory%2520R%2520Region%2520of%2520the%2520CFTR%2520Chloride%2520Channel%2520Is%2520a%2520Dynamic%2520Integrator%2520of%2520Phospho-Dependent%2520Intra-%2520and%2520Intermolecular%2520Interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26issue%3D47%26spage%3DE4427%26epage%3D4436%26doi%3D10.1073%2Fpnas.1315104110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stevers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S. F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Scheppingen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, R. M. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlile, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Characterization and Small-Molecule Stabilization of the Multisite Tandem Binding between 14–3-3 and the R Domain of CFTR</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">E1152</span>– <span class="NLM_lpage">1161</span>, <span class="refDoi"> DOI: 10.1073/pnas.1516631113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1073%2Fpnas.1516631113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=26888287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC28XislaqsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=E1152-1161&issue=9&author=L.+M.+Steversauthor=C.+V.+Lamauthor=S.+F.+R.+Leysenauthor=F.+A.+Meijerauthor=D.+S.+van+Scheppingenauthor=R.+M.+J.+M.+de+Vriesauthor=G.+W.+Carlileauthor=L.+G.+Milroyauthor=D.+Y.+Thomasauthor=L.+Brunsveldauthor=C.+Ottmann&title=Characterization+and+Small-Molecule+Stabilization+of+the+Multisite+Tandem+Binding+between+14%E2%80%933-3+and+the+R+Domain+of+CFTR&doi=10.1073%2Fpnas.1516631113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization and small-molecule stabilization of the multisite tandem binding between 14-3-3 and the R domain of CFTR</span></div><div class="casAuthors">Stevers, Loes M.; Lam, Chan V.; Leysen, Seppe F. R.; Meijer, Femke A.; van Scheppingen, Daphne S.; de Vries, Rens M. J. M.; Carlile, Graeme W.; Milroy, Lech G.; Thomas, David Y.; Brunsveld, Luc; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">E1152-E1161</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cystic fibrosis is a fatal genetic disease, most frequently caused by the retention of the CFTR (cystic fibrosis transmembrane conductance regulator) mutant protein in the endoplasmic reticulum (ER).  The binding of the 14-3-3 protein to the CFTR regulatory (R) domain has been found to enhance CFTR trafficking to the plasma membrane.  To define the mechanism of action of this protein-protein interaction, we have examd. the interaction in vitro.  The disordered multiphosphorylated R domain contains nine different 14-3-3 binding motifs.  Furthermore, the 14-3-3 protein forms a dimer contg. two amphipathic grooves that can potentially bind these phosphorylated motifs.  This results in a no. of possible binding mechanisms between these two proteins.  Using multiple biochem. assays and crystal structures, we show that the interaction between them is governed by two binding sites: the key binding site of CFTR (pS768) occupies one groove of the 14-3-3 dimer, and a weaker, secondary binding site occupies the other binding groove.  We show that fusicoccin-A, a natural-product tool compd. used in studies of 14-3-3 biol., can stabilize the interaction between 14-3-3 and CFTR by selectively interacting with a secondary binding motif of CFTR (pS753).  The stabilization of this interaction stimulates the trafficking of mutant CFTR to the plasma membrane.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXJUVLOg21FbVg90H21EOLACvtfcHk0li-BCXwqzM27w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislaqsLo%253D&md5=148d49c057fc8635f9436b3735f86e68</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1516631113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1516631113%26sid%3Dliteratum%253Aachs%26aulast%3DStevers%26aufirst%3DL.%2BM.%26aulast%3DLam%26aufirst%3DC.%2BV.%26aulast%3DLeysen%26aufirst%3DS.%2BF.%2BR.%26aulast%3DMeijer%26aufirst%3DF.%2BA.%26aulast%3Dvan%2BScheppingen%26aufirst%3DD.%2BS.%26aulast%3Dde%2BVries%26aufirst%3DR.%2BM.%2BJ.%2BM.%26aulast%3DCarlile%26aufirst%3DG.%2BW.%26aulast%3DMilroy%26aufirst%3DL.%2BG.%26aulast%3DThomas%26aufirst%3DD.%2BY.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DCharacterization%2520and%2520Small-Molecule%2520Stabilization%2520of%2520the%2520Multisite%2520Tandem%2520Binding%2520between%252014%25E2%2580%25933-3%2520and%2520the%2520R%2520Domain%2520of%2520CFTR%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26issue%3D9%26spage%3DE1152%26epage%3D1161%26doi%3D10.1073%2Fpnas.1516631113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunziker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span> <span> </span><span class="NLM_article-title">A Role for TAZ in Migration, Invasion, and Tumorigenesis of Breast Cancer Cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2592</span>– <span class="NLM_lpage">2598</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-2696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1158%2F0008-5472.CAN-07-2696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=18413727" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1GitL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=2592-2598&issue=8&author=S.+W.+Chanauthor=C.+J.+Limauthor=K.+Guoauthor=C.+P.+Ngauthor=I.+Leeauthor=W.+Hunzikerauthor=Q.+Zengauthor=W.+Hong&title=A+Role+for+TAZ+in+Migration%2C+Invasion%2C+and+Tumorigenesis+of+Breast+Cancer+Cells&doi=10.1158%2F0008-5472.CAN-07-2696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A Role for TAZ in Migration, Invasion, and Tumorigenesis of Breast Cancer Cells</span></div><div class="casAuthors">Chan, Siew Wee; Lim, Chun Jye; Guo, Ke; Ng, Chee Peng; Lee, Ian; Hunziker, Walter; Zeng, Qi; Hong, Wanjin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2592-2598</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">TAZ (WWTR1), identified as a 14-3-3 binding protein with a PDZ binding motif, modulates mesenchymal stem cell differentiation.  We now show that TAZ plays a crit. role in the migration, invasion, and tumorigenesis of breast cancer cells.  TAZ is conspicuously expressed in human breast cancer cell lines in which its expression levels generally correlate with the invasiveness of cancer cells.  Overexpression of TAZ in low-expressing MCF10A cells causes morphol. changes characteristic of cell transformation and promotes cell migration and invasion.  Conversely, RNA interference-mediated knockdown of TAZ expression in MCF7 and Hs578T cells reduces cell migration and invasion.  TAZ knockdown in MCF7 cells also retards anchorage-independent growth in soft agar and tumorigenesis in nude mice.  Significantly, TAZ is overexpressed in ∼20% of breast cancer samples.  These results indicate that TAZ plays a role in the migration, invasion, and tumorigenesis of breast cancer cells and thus presents a novel target for the detection and treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMyv6KhRaRN7Vg90H21EOLACvtfcHk0liEj2QdhR83LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1GitL8%253D&md5=4b58785b0ab40c38ba0ff8c168d4a609</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-2696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-2696%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DS.%2BW.%26aulast%3DLim%26aufirst%3DC.%2BJ.%26aulast%3DGuo%26aufirst%3DK.%26aulast%3DNg%26aufirst%3DC.%2BP.%26aulast%3DLee%26aufirst%3DI.%26aulast%3DHunziker%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DHong%26aufirst%3DW.%26atitle%3DA%2520Role%2520for%2520TAZ%2520in%2520Migration%252C%2520Invasion%252C%2520and%2520Tumorigenesis%2520of%2520Breast%2520Cancer%2520Cells%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26issue%3D8%26spage%3D2592%26epage%3D2598%26doi%3D10.1158%2F0008-5472.CAN-07-2696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.-H.</span></span> <span> </span><span class="NLM_article-title">14–3-3 Sigma Positively Regulates P53 and Suppresses Tumor Growth</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">7096</span>– <span class="NLM_lpage">7107</span>, <span class="refDoi"> DOI: 10.1128/MCB.23.20.7096-7107.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1128%2FMCB.23.20.7096-7107.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=14517281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotVSrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=7096-7107&issue=20&author=H.-Y.+Yangauthor=Y.-Y.+Wenauthor=C.-H.+Chenauthor=G.+Lozanoauthor=M.-H.+Lee&title=14%E2%80%933-3+Sigma+Positively+Regulates+P53+and+Suppresses+Tumor+Growth&doi=10.1128%2FMCB.23.20.7096-7107.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">14-3-3σ positively regulates p53 and suppresses tumor growth</span></div><div class="casAuthors">Yang, Heng-Yin; Wen, Yu-Ye; Chen, Chih-Hsin; Lozano, Guillermina; Lee, Mong-Hong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7096-7107</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The 14-3-3σ (sigma) protein, a neg. regulator of the cell cycle, is a human mammary epithelium-specific marker that is downregulated in transformed mammary carcinoma cells.  It has also been identified as a p53-inducible gene product involved in cell cycle checkpoint control after DNA damage.  Although 14-3-3σ is linked to p53-regulated cell cycle checkpoint control, detailed mechanisms of how cell cycle regulation occurs remain unclear.  Decreased expression of 14-3-3σ was recently reported in several types of carcinomas, further suggesting that the neg. regulatory role of 14-3-3σ in the cell cycle is compromised during tumorigenesis.  However, this possible tumor-suppressive role of 14-3-3σ has not yet been characterized.  Here, the authors studied the link between 14-3-3σ activities and p53 regulation.  The authors found that 14-3-3σ interacted with p53 in response to the DNA-damaging agent adriamycin.  Importantly, 14-3-3σ expression led to stabilized expression of p53.  In studying the mol. mechanism of this increased stabilization of p53, the authors found that 14-3-3σ antagonized the biol. functions of Mdm2 by blocking Mdm2-mediated p53 ubiquitination and nuclear export.  In addn., the authors found that 14-3-3σ facilitated the oligomerization of p53 and enhanced p53's transcriptional activity.  As a target gene of p53, 14-3-3σ appears to have a pos. feedback effect on p53 activity.  Significantly, the authors also showed that overexpression of 14-3-3σ inhibited oncogene-activated tumorigenicity in a tetracycline-regulated 14-3-3σ system.  These results defined an important p53 regulatory loop and suggested that 14-3-3σ expression can be considered for therapeutic intervention in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtjTAMn0T1XbVg90H21EOLACvtfcHk0liEj2QdhR83LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotVSrtbo%253D&md5=3b89488426cb6f8d9b60c9c402a24999</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.20.7096-7107.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.20.7096-7107.2003%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DH.-Y.%26aulast%3DWen%26aufirst%3DY.-Y.%26aulast%3DChen%26aufirst%3DC.-H.%26aulast%3DLozano%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DM.-H.%26atitle%3D14%25E2%2580%25933-3%2520Sigma%2520Positively%2520Regulates%2520P53%2520and%2520Suppresses%2520Tumor%2520Growth%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2003%26volume%3D23%26issue%3D20%26spage%3D7096%26epage%3D7107%26doi%3D10.1128%2FMCB.23.20.7096-7107.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopalan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sade, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsley, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span> <span> </span><span class="NLM_article-title">Mechanistic Differences in the Transcriptional Activation of P53 by 14–3-3 Isoforms</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">893</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp1041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1093%2Fnar%2Fgkp1041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=19933256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSlurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=893-906&issue=3&author=S.+Rajagopalanauthor=R.+S.+Sadeauthor=F.+M.+Townsleyauthor=A.+R.+Fersht&title=Mechanistic+Differences+in+the+Transcriptional+Activation+of+P53+by+14%E2%80%933-3+Isoforms&doi=10.1093%2Fnar%2Fgkp1041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic differences in the transcriptional activation of p53 by 14-3-3 isoforms</span></div><div class="casAuthors">Rajagopalan, Sridharan; Sade, Robert S.; Townsley, Fiona M.; Fersht, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">893-906</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">P53 maintains genome integrity by initiating the transcription of genes involved in cell-cycle arrest, senescence, apoptosis and DNA repair.  The activity of p53 is regulated by both post-translational modifications and protein-protein interactions. p53 that has been phosphorylated at S366, S378 and T387 binds 14-3-3 proteins in vitro.  Here, we show that these sites are potential 14-3-3 binding sites in vivo.  Epsilon (ε) and gamma (γ) isoforms required phosphorylation at either of these sites for efficient interaction with p53, while for sigma (σ) and tau (τ) these sites are dispensable.  Further, σ and τ bound more weakly to p53 C-terminal phosphopeptides than did ε and γ.  However, the four isoforms bound tightly to di-phosphorylated p53 C-terminal peptides than did the mono-phosphorylated counterparts.  Interestingly, all the isoforms studied transcriptionally activated wild-type p53. σ and τ stabilized p53 levels in cells, while ε and γ stimulated p53-DNA binding activity in vitro.  Overall, the results suggest that structurally and functionally similar 14-3-3 isoforms may exert their regulatory potential on p53 through different mechanisms.  We discuss the isoform-specific roles of 14-3-3 in p53 stabilization and activation of specific-DNA binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRVozP2__gyLVg90H21EOLACvtfcHk0liEj2QdhR83LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSlurc%253D&md5=11f92e04e5017515a4bbfc1bd65bfd1d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp1041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp1041%26sid%3Dliteratum%253Aachs%26aulast%3DRajagopalan%26aufirst%3DS.%26aulast%3DSade%26aufirst%3DR.%2BS.%26aulast%3DTownsley%26aufirst%3DF.%2BM.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3DMechanistic%2520Differences%2520in%2520the%2520Transcriptional%2520Activation%2520of%2520P53%2520by%252014%25E2%2580%25933-3%2520Isoforms%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26issue%3D3%26spage%3D893%26epage%3D906%26doi%3D10.1093%2Fnar%2Fgkp1041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopalan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaulent, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veprintsev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span> <span> </span><span class="NLM_article-title">14–3-3 Activation of DNA Binding of P53 by Enhancing Its Association into Tetramers</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">5983</span>– <span class="NLM_lpage">5991</span>, <span class="refDoi"> DOI: 10.1093/nar/gkn598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1093%2Fnar%2Fgkn598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=18812399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Kgu7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=5983-5991&issue=18&author=S.+Rajagopalanauthor=A.+M.+Jaulentauthor=M.+Wellsauthor=D.+B.+Veprintsevauthor=A.+R.+Fersht&title=14%E2%80%933-3+Activation+of+DNA+Binding+of+P53+by+Enhancing+Its+Association+into+Tetramers&doi=10.1093%2Fnar%2Fgkn598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">14-3-3 activation of DNA binding of p53 by enhancing its association into tetramers</span></div><div class="casAuthors">Rajagopalan, Sridharan; Jaulent, Agnes M.; Wells, Mark; Veprintsev, Dmitry B.; Fersht, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5983-5991</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Activation of the tumor suppressor p53 on DNA damage involves post-translational modification by phosphorylation and acetylation.  Phosphorylation of certain residues is crit. for p53 stabilization and plays an important role in DNA-binding activity.  The 14-3-3 family of proteins activates the DNA-binding affinity of p53 upon stress by binding to a site in its intrinsically disordered C-terminal domain contg. a phosphorylated serine at 378.  We have screened various p53 C-terminal phosphorylated peptides for binding to two different isoforms of 14-3-3, ε and γ.  We found that phosphorylation at either S366 or T387 caused even tighter binding to 14-3-3.  We made by semi-synthesis a tetrameric construct comprised of the tetramerization plus C-terminal domains of p53 that was phosphorylated on S366, S378 and T387.  It bound 10 times tighter than did the monomeric counterpart to dimeric 14-3-3.  We showed indirectly from binding curves and directly from fluorescence-detection anal. ultracentrifugation that 14-3-3 enhanced the binding of sequence-specific DNA to p53 by causing p53 dimers to form tetramers at lower concns.  If the in vitro data extrapolate to in vivo, then it is an attractive hypothesis that p53 activity may be subject to control by accessory proteins lowering its tetramer-dimer dissocn. const. from its normal value of 120-150 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8YjWdbhc0eLVg90H21EOLACvtfcHk0lgx0uyZomCgzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Kgu7fJ&md5=277d9a9c81ff6b592c444be0cf2f4194</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn598%26sid%3Dliteratum%253Aachs%26aulast%3DRajagopalan%26aufirst%3DS.%26aulast%3DJaulent%26aufirst%3DA.%2BM.%26aulast%3DWells%26aufirst%3DM.%26aulast%3DVeprintsev%26aufirst%3DD.%2BB.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3D14%25E2%2580%25933-3%2520Activation%2520of%2520DNA%2520Binding%2520of%2520P53%2520by%2520Enhancing%2520Its%2520Association%2520into%2520Tetramers%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26issue%3D18%26spage%3D5983%26epage%3D5991%26doi%3D10.1093%2Fnar%2Fgkn598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carriere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheerathodi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rush, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballif, B. A.</span></span> <span> </span><span class="NLM_article-title">RSK Phosphorylates SOS1 Creating 14–3-3-Docking Sites and Negatively Regulating MAPK Activation</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>447</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1042/BJ20120938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FBJ20120938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22827337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlCjsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=447&publication_year=2012&pages=159-166&issue=1&author=M.+Sahaauthor=A.+Carriereauthor=M.+Cheerathodiauthor=X.+Zhangauthor=G.+Lavoieauthor=J.+Rushauthor=P.+P.+Rouxauthor=B.+A.+Ballif&title=RSK+Phosphorylates+SOS1+Creating+14%E2%80%933-3-Docking+Sites+and+Negatively+Regulating+MAPK+Activation&doi=10.1042%2FBJ20120938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">RSK phosphorylates SOS1 creating 14-3-3-docking sites and negatively regulating MAPK activation</span></div><div class="casAuthors">Saha, Madhurima; Carriere, Audrey; Cheerathodi, Mujeeburahiman; Zhang, Xiaocui; Lavoie, Genevieve; Rush, John; Roux, Philippe P.; Ballif, Bryan A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">447</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">159-166</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The extent and duration of MAPK (mitogen-activated protein kinase) signalling govern a diversity of normal and aberrant cellular outcomes.  Genetic and pharmacol. disruption of the MAPK-activated kinase RSK (ribosomal S6 kinase) leads to elevated MAPK activity indicative of a RSK-dependent neg. feedback loop.  Using biochem., pharmacol. and quant. MS approaches we show that RSK phosphorylates the Ras activator SOS1 (Son of Sevenless homolog 1) in cultured cells on two C-terminal residues, Ser1134 and Ser1161.  Furthermore, we find that RSK-dependent SOS1 phosphorylation creates 14-3-3-binding sites.  We show that mutating Ser1134 and Ser1161 disrupts 14-3-3 binding and modestly increases and extends MAPK activation.  Together these data suggest that one mechanism whereby RSK neg. regulates MAPK activation is via site-specific SOS1 phosphorylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4pH9etGvwsbVg90H21EOLACvtfcHk0lgx0uyZomCgzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlCjsLrF&md5=c39f673b533b03e22e96b67e84dc2360</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1042%2FBJ20120938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20120938%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DM.%26aulast%3DCarriere%26aufirst%3DA.%26aulast%3DCheerathodi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLavoie%26aufirst%3DG.%26aulast%3DRush%26aufirst%3DJ.%26aulast%3DRoux%26aufirst%3DP.%2BP.%26aulast%3DBallif%26aufirst%3DB.%2BA.%26atitle%3DRSK%2520Phosphorylates%2520SOS1%2520Creating%252014%25E2%2580%25933-3-Docking%2520Sites%2520and%2520Negatively%2520Regulating%2520MAPK%2520Activation%26jtitle%3DBiochem.%2520J.%26date%3D2012%26volume%3D447%26issue%3D1%26spage%3D159%26epage%3D166%26doi%3D10.1042%2FBJ20120938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centorrino, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Structural Characterization of 14–3-3ζ in Complex with the Human Son of Sevenless Homolog 1 (SOS1)</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>202</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">215</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2018.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.jsb.2018.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=29408703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Ogu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=202&publication_year=2018&pages=210-215&issue=3&author=A.+Balloneauthor=F.+Centorrinoauthor=M.+Wolterauthor=C.+Ottmann&title=Structural+Characterization+of+14%E2%80%933-3%CE%B6+in+Complex+with+the+Human+Son+of+Sevenless+Homolog+1+%28SOS1%29&doi=10.1016%2Fj.jsb.2018.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structural characterization of 14-3-3ζ in complex with the human Son of sevenless homolog 1 (SOS1)</span></div><div class="casAuthors">Ballone, Alice; Centorrino, Federica; Wolter, Madita; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">202</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">210-215</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The deviant Ras activation machinery is found in approx. 30% of all human cancers.  SOS1 is an important protagonist of this pathway that plays a key-role in aberrant cell proliferation and differentiation.  Interaction of SOS1 with 14-3-3 proteins modulates SOS1 activity in Ras-MAPK signaling.  In the present study, we analyze the 14-3-3/SOS1 protein-protein interaction (PPI) by different biochem. assays and report the high resoln. crystal structure of a 13-mer motif of SOS1 bound to 14-3-3 ζ.  These structural and functional insights are important for the evaluation of this PPI interface for small-mol. stabilization as a new starting point for modulating the Ras-Raf-MAPK pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs8EIra65oLLVg90H21EOLACvtfcHk0lgx0uyZomCgzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Ogu7w%253D&md5=451fa5e9f1d5c04e928b0edc99916913</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2018.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2018.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DBallone%26aufirst%3DA.%26aulast%3DCentorrino%26aufirst%3DF.%26aulast%3DWolter%26aufirst%3DM.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DStructural%2520Characterization%2520of%252014%25E2%2580%25933-3%25CE%25B6%2520in%2520Complex%2520with%2520the%2520Human%2520Son%2520of%2520Sevenless%2520Homolog%25201%2520%2528SOS1%2529%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D202%26issue%3D3%26spage%3D210%26epage%3D215%26doi%3D10.1016%2Fj.jsb.2018.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ory, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrads, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenstra, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, D. K.</span></span> <span> </span><span class="NLM_article-title">Protein Phosphatase 2A Positively Regulates Ras Signaling by Dephosphorylating KSR1 and Raf-1 on Critical 14–3-3 Binding Sites</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">1356</span>– <span class="NLM_lpage">1364</span>, <span class="refDoi"> DOI: 10.1016/S0960-9822(03)00535-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2FS0960-9822%2803%2900535-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=12932319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmslentLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=1356-1364&issue=16&author=S.+Oryauthor=M.+Zhouauthor=T.+P.+Conradsauthor=T.+D.+Veenstraauthor=D.+K.+Morrison&title=Protein+Phosphatase+2A+Positively+Regulates+Ras+Signaling+by+Dephosphorylating+KSR1+and+Raf-1+on+Critical+14%E2%80%933-3+Binding+Sites&doi=10.1016%2FS0960-9822%2803%2900535-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Phosphatase 2A Positively Regulates Ras Signaling by Dephosphorylating KSR1 and Raf-1 on Critical 14-3-3 Binding Sites</span></div><div class="casAuthors">Ory, Stephane; Zhou, Ming; Conrads, Thomas P.; Veenstra, Timothy D.; Morrison, Deborah K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1356-1364</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: Kinase Suppressor of Ras (KSR) is a conserved component of the Ras pathway that acts as a mol. scaffold to facilitate signal transmission through the MAPK cascade.  Although recruitment of KSR1 from the cytosol to the plasma membrane is required for its scaffolding function, the precise mechanism(s) regulating the translocation of KSR1 have not been fully elucidated.  Results: Using mass spectrometry to analyze the KSR1-scaffolding complex, we identify the serine/threonine protein phosphatase PP2A as a KSR1-assocd. protein and show that PP2A is a crit. regulator of KSR1 activity.  We find that the enzymic core subunits of PP2A (PR65A and catalytic C) constitutively assoc. with the N-terminal domain of KSR1, whereas binding of the regulatory PR55B subunit is induced by growth factor treatment.  Specific inhibition of PP2A activity prevents the growth factor-induced dephosphorylation event involved in the membrane recruitment of KSR1 and blocks the activation of KSR1-assocd. MEK and ERK.  Moreover, we find that PP2A activity is required for activation of the Raf-1 kinase and that both Raf and KSR1 must be dephosphorylated by PP2A on crit. regulatory 14-3-3 binding sites for KSR1 to promote MAPK pathway activation.  Conclusions: These findings identify KSR1 as novel substrate of PP2A and demonstrate the inducible dephosphorylation of KSR1 in response to Ras pathway activation.  Further, these results elucidate a common regulatory mechanism for KSR1 and Raf-1 whereby their localization and activity are modulated by the PP2A-mediated dephosphorylation of crit. 14-3-3 binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgdlQTl7RWhbVg90H21EOLACvtfcHk0ljlTAQXRSGIHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmslentLk%253D&md5=b49148bfad0b58ed82204dfdf32a2abd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2803%2900535-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252803%252900535-9%26sid%3Dliteratum%253Aachs%26aulast%3DOry%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DConrads%26aufirst%3DT.%2BP.%26aulast%3DVeenstra%26aufirst%3DT.%2BD.%26aulast%3DMorrison%26aufirst%3DD.%2BK.%26atitle%3DProtein%2520Phosphatase%25202A%2520Positively%2520Regulates%2520Ras%2520Signaling%2520by%2520Dephosphorylating%2520KSR1%2520and%2520Raf-1%2520on%2520Critical%252014%25E2%2580%25933-3%2520Binding%2520Sites%26jtitle%3DCurr.%2520Biol.%26date%3D2003%26volume%3D13%26issue%3D16%26spage%3D1356%26epage%3D1364%26doi%3D10.1016%2FS0960-9822%2803%2900535-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Lorenzo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of BRAF by AMPK Impairs BRAF-KSR1 Association and Cell Proliferation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.08.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.molcel.2013.08.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=24095280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOls73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=161-172&issue=2&author=C.-H.+Shenauthor=P.+Yuanauthor=R.+Perez-Lorenzoauthor=Y.+Zhangauthor=S.+X.+Leeauthor=Y.+Ouauthor=J.+M.+Asaraauthor=L.+C.+Cantleyauthor=B.+Zheng&title=Phosphorylation+of+BRAF+by+AMPK+Impairs+BRAF-KSR1+Association+and+Cell+Proliferation&doi=10.1016%2Fj.molcel.2013.08.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of BRAF by AMPK Impairs BRAF-KSR1 Association and Cell Proliferation</span></div><div class="casAuthors">Shen, Che-Hung; Yuan, Ping; Perez-Lorenzo, Rolando; Zhang, Yaqing; Lee, Sze Xian; Ou, Yang; Asara, John M.; Cantley, Lewis C.; Zheng, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">161-172</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">BRAF is an oncogenic protein kinase that drives cell growth and proliferation through the MEK-ERK signaling pathway.  BRAF inhibitors have demonstrated antitumor efficacy in melanoma therapy but have also been found to be assocd. with the development of cutaneous squamous cell carcinomas (cSCCs) in certain patients.  Here, we report that BRAF is phosphorylated at Ser729 by AMP-activated protein kinase (AMPK), a crit. energy sensor.  This phosphorylation promotes the assocn. of BRAF with 14-3-3 proteins and disrupts its interaction with the KSR1 scaffolding protein, leading to attenuation of the MEK-ERK signaling.  We also show that phosphorylation of BRAF by AMPK impairs keratinocyte cell proliferation and cell-cycle progression.  Furthermore, AMPK activation attenuates BRAF inhibitor-induced ERK hyperactivation in keratinocytes and epidermal hyperplasia in mouse skin.  Our findings reveal a mechanism for regulating BRAF signaling in response to energy stress and suggest a strategy for preventing the development of cSCCs assocd. with BRAF-targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEDKxlkrxbWLVg90H21EOLACvtfcHk0ljlTAQXRSGIHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOls73P&md5=f0af44261dee02ae14626ad29f4fa9b0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.08.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.08.044%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DC.-H.%26aulast%3DYuan%26aufirst%3DP.%26aulast%3DPerez-Lorenzo%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DS.%2BX.%26aulast%3DOu%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DZheng%26aufirst%3DB.%26atitle%3DPhosphorylation%2520of%2520BRAF%2520by%2520AMPK%2520Impairs%2520BRAF-KSR1%2520Association%2520and%2520Cell%2520Proliferation%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D52%26issue%3D2%26spage%3D161%26epage%3D172%26doi%3D10.1016%2Fj.molcel.2013.08.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dumaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marais, R.</span></span> <span> </span><span class="NLM_article-title">Protein Kinase A Blocks Raf-1 Activity by Stimulating 14–3-3 Binding and Blocking Raf-1 Interaction with Ras</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span> (<span class="NLM_issue">32</span>),  <span class="NLM_fpage">29819</span>– <span class="NLM_lpage">29823</span>, <span class="refDoi"> DOI: 10.1074/jbc.C300182200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1074%2Fjbc.C300182200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=12801936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlvFOnsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=29819-29823&issue=32&author=N.+Dumazauthor=R.+Marais&title=Protein+Kinase+A+Blocks+Raf-1+Activity+by+Stimulating+14%E2%80%933-3+Binding+and+Blocking+Raf-1+Interaction+with+Ras&doi=10.1074%2Fjbc.C300182200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase A Blocks Raf-1 Activity by Stimulating 14-3-3 Binding and Blocking Raf-1 Interaction with Ras</span></div><div class="casAuthors">Dumaz, Nicolas; Marais, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">29819-29823</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">CAMP blocks Raf-1 activation by stimulating its phosphorylation on serine 43 (Ser43), serine 233 (Ser233), and serine 259 (Ser259).  We show here that phosphorylation of all three sites blocks Raf-1 binding to Ras·GTP in vivo and that cAMP stimulates binding of 14-3-3 proteins to Ser233 and Ser259.  We also show that Raf-1 and protein kinase A (PKA) form a complex in vivo that is disrupted by cAMP and that ablation of PKA by use of small interfering RNA blocks phosphorylation by cAMP.  The ability of PKA to block Raf-1 activation is ablated by the PKA inhibitor H89.  These studies suggest that Raf-1 and cAMP form a signaling complex in cells.  Upon activation of PKA, Raf-1 is phosphorylated and 14-3-3 binds, blocking Raf-1 recruitment to the plasma membrane and preventing its activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMN_lumn0pG7Vg90H21EOLACvtfcHk0ljlTAQXRSGIHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlvFOnsLY%253D&md5=a6c108371f72eb18e311366f6b7ca311</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C300182200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C300182200%26sid%3Dliteratum%253Aachs%26aulast%3DDumaz%26aufirst%3DN.%26aulast%3DMarais%26aufirst%3DR.%26atitle%3DProtein%2520Kinase%2520A%2520Blocks%2520Raf-1%2520Activity%2520by%2520Stimulating%252014%25E2%2580%25933-3%2520Binding%2520and%2520Blocking%2520Raf-1%2520Interaction%2520with%2520Ras%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26issue%3D32%26spage%3D29819%26epage%3D29823%26doi%3D10.1074%2Fjbc.C300182200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Paula, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozóky, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman-Kay, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frizzell, R. A.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation-Dependent 14–3-3 Protein Interactions Regulate CFTR Biogenesis</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">996</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1091/mbc.e11-08-0662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1091%2Fmbc.e11-08-0662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22278744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVOgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2012&pages=996-1009&issue=6&author=X.+Liangauthor=A.+C.+Da+Paulaauthor=Z.+Boz%C3%B3kyauthor=H.+Zhangauthor=C.+A.+Bertrandauthor=K.+W.+Petersauthor=J.+D.+Forman-Kayauthor=R.+A.+Frizzell&title=Phosphorylation-Dependent+14%E2%80%933-3+Protein+Interactions+Regulate+CFTR+Biogenesis&doi=10.1091%2Fmbc.e11-08-0662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation-dependent 14-3-3 protein interactions regulate CFTR biogenesis</span></div><div class="casAuthors">Liang, Xiubin; Da Paula, Ana Carina; Bozoky, Zoltan; Zhang, Hui; Bertrand, Carol A.; Peters, Kathryn W.; Forman-Kay, Julie D.; Frizzell, Raymond A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">996-1009</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Cystic fibrosis transmembrane conductance regulator (CFTR) is a cAMP/protein kinase A (PKA)-regulated chloride channel whose phosphorylation controls anion secretion across epithelial cell apical membranes.  We examd. the hypothesis that cAMP/PKA stimulation regulates CFTR biogenesis posttranslationally, based on predicted 14-3-3 binding motifs within CFTR and forskolin-induced CFTR expression.  The 14-3-3β, γ, and ε isoforms were expressed in airway cells and interacted with CFTR in coimmunopptn. assays.  Forskolin stimulation (15 min) increased 14-3-3β and ε binding to immature and mature CFTR (bands B and C), and 14-3-3 overexpression increased CFTR bands B and C and cell surface band C.  In pulse-chase expts., 14-3-3β increased the synthesis of immature CFTR, reduced its degrdn. rate, and increased conversion of immature to mature CFTR.  Conversely, 14-3-3β knockdown decreased CFTR B and C bands (70 and 55%) and elicited parallel redns. in cell surface CFTR and forskolin-stimulated anion efflux.  In vitro, 14-3-3β interacted with the CFTR regulatory region, and by NMR anal., this interaction occurred at known PKA phosphorylated sites.  In coimmunopptn. assays, forskolin stimulated the CFTR/14-3-3β interaction while reducing CFTR's interaction with coat protein complex 1 (COP1).  Thus 14-3-3 binding to phosphorylated CFTR augments its biogenesis by reducing retrograde retrieval of CFTR to the endoplasmic reticulum.  This mechanism permits cAMP/PKA stimulation to make more CFTR available for anion secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBTsCTTTWAQ7Vg90H21EOLACvtfcHk0lj9_rOjzX8RDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVOgtrw%253D&md5=1faad75eaf1ebf2e232e23293b170e82</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e11-08-0662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e11-08-0662%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DDa%2BPaula%26aufirst%3DA.%2BC.%26aulast%3DBoz%25C3%25B3ky%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DBertrand%26aufirst%3DC.%2BA.%26aulast%3DPeters%26aufirst%3DK.%2BW.%26aulast%3DForman-Kay%26aufirst%3DJ.%2BD.%26aulast%3DFrizzell%26aufirst%3DR.%2BA.%26atitle%3DPhosphorylation-Dependent%252014%25E2%2580%25933-3%2520Protein%2520Interactions%2520Regulate%2520CFTR%2520Biogenesis%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2012%26volume%3D23%26issue%3D6%26spage%3D996%26epage%3D1009%26doi%3D10.1091%2Fmbc.e11-08-0662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morquette, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madwar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, A. E.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Stabilization of 14–3-3 Protein-Protein Interactions Stimulates Axon Regeneration</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1082</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2017.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.neuron.2017.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=28279353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktVylu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=1082&issue=5&author=A.+Kaplanauthor=B.+Morquetteauthor=A.+Kronerauthor=S.+Leongauthor=C.+Madwarauthor=R.+Sanzauthor=S.+L.+Banerjeeauthor=J.+Antelauthor=N.+Bissonauthor=S.+Davidauthor=A.+E.+Fournier&title=Small-Molecule+Stabilization+of+14%E2%80%933-3+Protein-Protein+Interactions+Stimulates+Axon+Regeneration&doi=10.1016%2Fj.neuron.2017.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Stabilization of 14-3-3 Protein-Protein Interactions Stimulates Axon Regeneration</span></div><div class="casAuthors">Kaplan, Andrew; Morquette, Barbara; Kroner, Antje; Leong, Soo Yuen; Madwar, Carolin; Sanz, Ricardo; Banerjee, Sara L.; Antel, Jack; Bisson, Nicolas; David, Samuel; Fournier, Alyson E.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1082-1093.e5</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Damaged central nervous system (CNS) neurons have a poor ability to spontaneously regenerate, causing persistent functional deficits after injury.  Therapies that stimulate axon growth are needed to repair CNS damage.  14-3-3 Adaptors are hub proteins that are attractive targets to manipulate cell signaling.  We identify a pos. role for 14-3-3s in axon growth and uncover a developmental regulation of the phosphorylation and function of 14-3-3s.  We show that fusicoccin-A (FC-A), a small-mol. stabilizer of 14-3-3 protein-protein interactions, stimulates axon growth in vitro and regeneration in vivo.  We show that FC-A stabilizes a complex between 14-3-3 and the stress response regulator GCN1, inducing GCN1 turnover and neurite outgrowth.  These findings show that 14-3-3 adaptor protein complexes are druggable targets and identify a new class of small mols. that may be further optimized for the repair of CNS damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGtMq7zLhfkrVg90H21EOLACvtfcHk0lj9_rOjzX8RDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktVylu7o%253D&md5=b06351a1f1093cbc2a0cb260b53d9d26</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2017.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2017.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DKaplan%26aufirst%3DA.%26aulast%3DMorquette%26aufirst%3DB.%26aulast%3DKroner%26aufirst%3DA.%26aulast%3DLeong%26aufirst%3DS.%26aulast%3DMadwar%26aufirst%3DC.%26aulast%3DSanz%26aufirst%3DR.%26aulast%3DBanerjee%26aufirst%3DS.%2BL.%26aulast%3DAntel%26aufirst%3DJ.%26aulast%3DBisson%26aufirst%3DN.%26aulast%3DDavid%26aufirst%3DS.%26aulast%3DFournier%26aufirst%3DA.%2BE.%26atitle%3DSmall-Molecule%2520Stabilization%2520of%252014%25E2%2580%25933-3%2520Protein-Protein%2520Interactions%2520Stimulates%2520Axon%2520Regeneration%26jtitle%3DNeuron%26date%3D2017%26volume%3D93%26issue%3D5%26spage%3D1082%26doi%3D10.1016%2Fj.neuron.2017.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaspert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schauer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weyand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermeeren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duby, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittinghofer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigaud, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oecking, C.</span></span> <span> </span><span class="NLM_article-title">Structure of a 14–3-3 Coordinated Hexamer of the Plant Plasma Membrane H+ -ATPase by Combining X-Ray Crystallography and Electron Cryomicroscopy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.molcel.2006.12.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=17289589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=427-440&issue=3&author=C.+Ottmannauthor=S.+Marcoauthor=N.+Jaspertauthor=C.+Marconauthor=N.+Schauerauthor=M.+Weyandauthor=C.+Vandermeerenauthor=G.+Dubyauthor=M.+Boutryauthor=A.+Wittinghoferauthor=J.-L.+Rigaudauthor=C.+Oecking&title=Structure+of+a+14%E2%80%933-3+Coordinated+Hexamer+of+the+Plant+Plasma+Membrane+H%2B+-ATPase+by+Combining+X-Ray+Crystallography+and+Electron+Cryomicroscopy&doi=10.1016%2Fj.molcel.2006.12.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a 14-3-3 coordinated hexamer of the plant plasma membrane H+-ATPase by combining X-ray crystallography and electron cryomicroscopy</span></div><div class="casAuthors">Ottmann, Christian; Marco, Sergio; Jaspert, Nina; Marcon, Caroline; Schauer, Nicolas; Weyand, Michael; Vandermeeren, Caroline; Duby, Geoffrey; Boutry, Marc; Wittinghofer, Alfred; Rigaud, Jean-Louis; Oecking, Claudia</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">427-440</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Regulatory 14-3-3 proteins activate the plant plasma membrane H+-ATPase by binding to its C-terminal autoinhibitory domain.  This interaction requires phosphorylation of a C-terminal, mode III, recognition motif as well as an adjacent span of approx. 50 amino acids.  Here we report the x-ray crystal structure of 14-3-3 in complex with the entire binding motif, revealing a previously unidentified mode of interaction.  A 14-3-3 dimer simultaneously binds two H+-ATPase peptides, each of which forms a loop within the typical 14-3-3 binding groove and therefore exits from the center of the dimer.  Several H+-ATPase mutants support this structure detn.  Accordingly, 14-3-3 binding could result in H+-ATPase oligomerization.  Indeed, by using single-particle electron cryomicroscopy, the 3D reconstruction of the purified H+-ATPase/14-3-3 complex demonstrates a hexameric arrangement.  Fitting of 14-3-3 and H+-ATPase at. structures into the 3D reconstruction map suggests the spatial arrangement of the holocomplex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2piE2NL1L1bVg90H21EOLACvtfcHk0ljEZ9tykIA_NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jtbY%253D&md5=6d2a855f5f90a0c2265c1c3381232d8c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DMarco%26aufirst%3DS.%26aulast%3DJaspert%26aufirst%3DN.%26aulast%3DMarcon%26aufirst%3DC.%26aulast%3DSchauer%26aufirst%3DN.%26aulast%3DWeyand%26aufirst%3DM.%26aulast%3DVandermeeren%26aufirst%3DC.%26aulast%3DDuby%26aufirst%3DG.%26aulast%3DBoutry%26aufirst%3DM.%26aulast%3DWittinghofer%26aufirst%3DA.%26aulast%3DRigaud%26aufirst%3DJ.-L.%26aulast%3DOecking%26aufirst%3DC.%26atitle%3DStructure%2520of%2520a%252014%25E2%2580%25933-3%2520Coordinated%2520Hexamer%2520of%2520the%2520Plant%2520Plasma%2520Membrane%2520H%252B%2520-ATPase%2520by%2520Combining%2520X-Ray%2520Crystallography%2520and%2520Electron%2520Cryomicroscopy%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26issue%3D3%26spage%3D427%26epage%3D440%26doi%3D10.1016%2Fj.molcel.2006.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asahi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazeki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubohara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, N.</span></span> <span> </span><span class="NLM_article-title">Cotylenin A, a Plant-Growth Regulator, Induces the Differentiation in Murine and Human Myeloid Leukemia Cells</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>238</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">758</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1997.7385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1006%2Fbbrc.1997.7385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=9325163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADyaK2sXmsFCnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=238&publication_year=1997&pages=758-763&issue=3&author=K.+Asahiauthor=Y.+Honmaauthor=K.+Hazekiauthor=T.+Sassaauthor=Y.+Kuboharaauthor=A.+Sakuraiauthor=N.+Takahashi&title=Cotylenin+A%2C+a+Plant-Growth+Regulator%2C+Induces+the+Differentiation+in+Murine+and+Human+Myeloid+Leukemia+Cells&doi=10.1006%2Fbbrc.1997.7385"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Cotylenin A, a plant-growth regulator, induces the differentiation in murine and human myeloid leukemia cells</span></div><div class="casAuthors">Asahi, Ken-Ichi; Honma, Yoshio; Hazeki, Kaoru; Sassa, Takeshi; Kubohara, Yuzuru; Sakurai, Akira; Takahashi, Nobutaka</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">238</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">758-763</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Protein factors playing a significant part in differentiation and development have been recently elucidated.  However, low-mol. factors which also seem to be essential remain still unknown, although only retinoic acid has become such a candidate.  Cotylenins had been isolated as the plant-growth regulators, and have been found to affect a no. of physiol. processes of higher plants.  Here, the authors report that at the concns. above 12.5μg/mL (20μM) cotylenin A induced the functional and morphol. differentiation in murine (M1) and human myeloid leukemia (HL-60) cells.  Although cotylenin A has some similarity to PMA both in carbotricyclic diterpene structure and in biol. activity, i.e., differentiation-induction of HL-60 cells into macrophages, the activation of PKC and the elevation of Ca2+-levels by cotylenin A were not obsd.  Quite recently it has been reported that fusicoccin (closely related to cotylenin A)-targets are 14-3-3 proteins, which are at the crosspoint of a huge array of signaling and regulatory pathways.  Thus, cotylenin A might become a useful tool for the elucidation of mol. mechanisms of differentiation and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWC4R3xieQx7Vg90H21EOLACvtfcHk0ljEZ9tykIA_NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmsFCnsrw%253D&md5=80deb88912f3c705eea9021e568752b7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1997.7385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1997.7385%26sid%3Dliteratum%253Aachs%26aulast%3DAsahi%26aufirst%3DK.%26aulast%3DHonma%26aufirst%3DY.%26aulast%3DHazeki%26aufirst%3DK.%26aulast%3DSassa%26aufirst%3DT.%26aulast%3DKubohara%26aufirst%3DY.%26aulast%3DSakurai%26aufirst%3DA.%26aulast%3DTakahashi%26aufirst%3DN.%26atitle%3DCotylenin%2520A%252C%2520a%2520Plant-Growth%2520Regulator%252C%2520Induces%2520the%2520Differentiation%2520in%2520Murine%2520and%2520Human%2520Myeloid%2520Leukemia%2520Cells%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1997%26volume%3D238%26issue%3D3%26spage%3D758%26epage%3D763%26doi%3D10.1006%2Fbbrc.1997.7385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrei, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vink, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span> <span> </span><span class="NLM_article-title">Rationally Designed Semisynthetic Natural Product Analogues for Stabilization of 14–3-3 Protein-Protein Interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">13470</span>– <span class="NLM_lpage">13474</span>, <span class="refDoi"> DOI: 10.1002/anie.201806584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fanie.201806584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslahsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=13470-13474&issue=41&author=S.+A.+Andreiauthor=P.+de+Vinkauthor=E.+Sijbesmaauthor=L.+Hanauthor=L.+Brunsveldauthor=N.+Katoauthor=C.+Ottmannauthor=Y.+Higuchi&title=Rationally+Designed+Semisynthetic+Natural+Product+Analogues+for+Stabilization+of+14%E2%80%933-3+Protein-Protein+Interactions&doi=10.1002%2Fanie.201806584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Rationally Designed Semisynthetic Natural Product Analogues for Stabilization of 14-3-3 Protein-Protein Interactions</span></div><div class="casAuthors">Andrei, Sebastian A.; de Vink, Pim; Sijbesma, Eline; Han, Ling; Brunsveld, Luc; Kato, Nobuo; Ottmann, Christian; Higuchi, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">13470-13474</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The natural product family of fusicoccanes are stabilizers of 14-3-3 mediated protein-protein interactions (PPIs), some of which possess antitumor activity.  In this study, the first use of mol. dynamics (MD) to rationally design PPI stabilizers with increased potency is presented.  Synthesis of a focused library, with subsequent characterization by fluorescence polarization, mutational studies, and X-ray crystallog. confirmed the power of the MD-based design approach, revealing the potential for an addnl. hydrogen bond with the 14-3-3 protein to lead to significantly increased potency.  Addnl., these compds. exert their action in a cellular environment with increased potency.  The newly found polar interaction could provide an anchoring point for new small-mol. PPI stabilizers.  These results facilitate the development of fusicoccanes towards drugs or tool compds., as well as allowing the study of the fundamental principles behind PPI stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmn44vv_9gLLVg90H21EOLACvtfcHk0ljEZ9tykIA_NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslahsL%252FO&md5=c168fae0a4306a739bd4681cb712b701</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201806584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201806584%26sid%3Dliteratum%253Aachs%26aulast%3DAndrei%26aufirst%3DS.%2BA.%26aulast%3Dde%2BVink%26aufirst%3DP.%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DHiguchi%26aufirst%3DY.%26atitle%3DRationally%2520Designed%2520Semisynthetic%2520Natural%2520Product%2520Analogues%2520for%2520Stabilization%2520of%252014%25E2%2580%25933-3%2520Protein-Protein%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26issue%3D41%26spage%3D13470%26epage%3D13474%26doi%3D10.1002%2Fanie.201806584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Vries-van Leeuwen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da Costa Pereira, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flach, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piersma, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haase, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yalcin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalides, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feenstra, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Greef, T. F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Boer, A. H.</span></span> <span> </span><span class="NLM_article-title">Interaction of 14–3-3 Proteins with the Estrogen Receptor Alpha F Domain Provides a Drug Target Interface</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">8894</span>– <span class="NLM_lpage">8899</span>, <span class="refDoi"> DOI: 10.1073/pnas.1220809110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1073%2Fpnas.1220809110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=23676274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFait7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8894-8899&issue=22&author=I.+J.+De%0AVries-van+Leeuwenauthor=D.+da+Costa+Pereiraauthor=K.+D.+Flachauthor=S.+R.+Piersmaauthor=C.+Haaseauthor=D.+Bierauthor=Z.+Yalcinauthor=R.+Michalidesauthor=K.+A.+Feenstraauthor=C.+R.+Jim%C3%A9nezauthor=T.+F.+A.+de+Greefauthor=L.+Brunsveldauthor=C.+Ottmannauthor=W.+Zwartauthor=A.+H.+de+Boer&title=Interaction+of+14%E2%80%933-3+Proteins+with+the+Estrogen+Receptor+Alpha+F+Domain+Provides+a+Drug+Target+Interface&doi=10.1073%2Fpnas.1220809110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of 14-3-3 proteins with the Estrogen Receptor Alpha F domain provides a drug target interface</span></div><div class="casAuthors">De Vries-van Leeuwen, Ingrid J.; Pereira, Daniel da Costa; Flach, Koen D.; Piersma, Sander R.; Haase, Christian; Bier, David; Yalcin, Zeliha; Michalides, Rob; Feenstra, K. Anton; Jimenez, Connie R.; de Greef, Tom F. A.; Brunsveld, Luc; Ottmann, Christian; Zwart, Wilbert; de Boer, Albertus H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8894-8899, S8894/1-S8894/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Estrogen receptor alpha (ERa) is involved in numerous physiol. and pathol. processes, including breast cancer.  Breast cancer therapy is therefore currently directed at inhibiting the transcriptional potency of ERa, either by blocking estrogen prodn. through aromatase inhibitors or antiestrogens that compete for hormone binding.  Due to resistance, new treatment modalities are needed and as ERα dimerization is essential for its activity, interference with receptor dimerization offers a new opportunity to exploit in drug design.  Here we describe a unique mechanism of how ERα dimerization is neg. controlled by interaction with 14-3-3 proteins at the extreme C terminus of the receptor.  Moreover, the small-mol. fusicoccin (FC) stabilizes this ERα/14-3-3 interaction.  Cocrystn. of the trimeric ERα/14-3-3/FC complex provides the structural basis for this stabilization and shows the importance of phosphorylation of the penultimate Threonine (ERα-T594) for high-affinity interaction.  We confirm that T594 is a distinct ERα phosphorylation site in the breast cancer cell line MCF-7 using a phospho-T594-specific antibody and by mass spectrometry.  In line with its ERα/14-3-3 interaction stabilizing effect, fusicoccin reduces the estradiol-stimulated ERα dimerization, inhibits ERα/chromatin interactions and downstream gene expression, resulting in decreased cell proliferation.  Herewith, a unique functional phosphosite and an alternative regulation mechanism of ERα are provided, together with a small mol. that selectively targets this ERα/14-3-3 interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0rga7Qdqyn7Vg90H21EOLACvtfcHk0ljEZ9tykIA_NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFait7%252FP&md5=d17c1a1122f4b525b35651d3d9f629f0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1220809110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1220809110%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVries-van%2BLeeuwen%26aufirst%3DI.%2BJ.%26aulast%3Dda%2BCosta%2BPereira%26aufirst%3DD.%26aulast%3DFlach%26aufirst%3DK.%2BD.%26aulast%3DPiersma%26aufirst%3DS.%2BR.%26aulast%3DHaase%26aufirst%3DC.%26aulast%3DBier%26aufirst%3DD.%26aulast%3DYalcin%26aufirst%3DZ.%26aulast%3DMichalides%26aufirst%3DR.%26aulast%3DFeenstra%26aufirst%3DK.%2BA.%26aulast%3DJim%25C3%25A9nez%26aufirst%3DC.%2BR.%26aulast%3Dde%2BGreef%26aufirst%3DT.%2BF.%2BA.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DZwart%26aufirst%3DW.%26aulast%3Dde%2BBoer%26aufirst%3DA.%2BH.%26atitle%3DInteraction%2520of%252014%25E2%2580%25933-3%2520Proteins%2520with%2520the%2520Estrogen%2520Receptor%2520Alpha%2520F%2520Domain%2520Provides%2520a%2520Drug%2520Target%2520Interface%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26issue%3D22%26spage%3D8894%26epage%3D8899%26doi%3D10.1073%2Fpnas.1220809110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bovens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Identification and Structure of Small-Molecule Stabilizers of 14–3-3 Protein-Protein Interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">4129</span>– <span class="NLM_lpage">4132</span>, <span class="refDoi"> DOI: 10.1002/anie.200907203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fanie.200907203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvFSksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=4129-4132&issue=24&author=R.+Roseauthor=S.+Erdmannauthor=S.+Bovensauthor=A.+Wolfauthor=M.+Roseauthor=S.+Hennigauthor=H.+Waldmannauthor=C.+Ottmann&title=Identification+and+Structure+of+Small-Molecule+Stabilizers+of+14%E2%80%933-3+Protein-Protein+Interactions&doi=10.1002%2Fanie.200907203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Structure of Small-Molecule Stabilizers of 14-3-3 Protein-Protein Interactions</span></div><div class="casAuthors">Rose, Rolf; Erdmann, Silke; Bovens, Stefanie; Wolf, Alexander; Rose, Micheline; Hennig, Sven; Waldmann, Herbert; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4129-4132, S4129/1-S4129/13</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein, the identification of two small mols. that selectively stabilize the 14-3-3/PMA2 protein-protein interaction by binding to two adjacent sites in the interface, and which are active in vivo, is reported.  To identify potential stabilizing mols., a 37,000-member compd. library was screened in a surface-based format monitoring the binding of green fluorescent protein (GFP) fused with 14-3-3 to surface-immobilized glutathione S-transferase (GST) fused with PMA2-CT52 (the C-terminal 52 amino acids of PMA2).  The initial screen successfully identified two structurally unrelated compds. with the desired activity.  For these compds., the kinetics of the stabilizing activity were detd. by means of surface plasmon resonance (SPR).  To this end, the 14-3-3 interaction domain of PMA2, that is the 52 C-terminal amino acids (CT52) was immobilized on the dextran matrix of a BiaCore chip, and binding of the 14-3-3 protein was measured in the presence of the protein-protein interaction-stabilizing small mol.  The tri-substituted pyrrolinone hit compd., pyrrolidone1, showed assocn. kinetics similar to fusicoccin but with a more rapid dissocn., resulting in a KD of 80 μM.  The dipeptide epibestatin displayed a different kinetic behavior in stabilizing the 14-3-3/PMA2 complex, with a slower assocn. than fusicoccin and pyrrolidone1.  However, the dissocn. kinetics of epibestatin resembled that of fusicoccin; that is the protein complex was very stable once it was constituted and dissocn. was very slow.  As a result, the calcd. KD of the 14-3-3/PMA2 complex in the presence of epibestatin was 1.8 μM.  In the absence of any stabilizing compd. no binding of 14-3-3 to immobilized PMA2 could be measured (DMSO control).  To unravel the mechanistic basis of the obviously different binding modes of pyrrolidone1 and epibestatin, the crystal structures the ternary complexes with 14-3-3 and the C-terminal 30 amino acids of PMA2 (PMA2-CT30) were solved.  Co-crystals were obtained by mixing 14-3-3 protein and PMA2-CT30 (1:1.5 ratio) in the presence of either 2 mM pyrrolidone1 or 2 mM epibestatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoOBHBStJvjrVg90H21EOLACvtfcHk0lh0QL8Q5ckCVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvFSksr4%253D&md5=c45dbceb27fe5e53ea367319cc735451</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.200907203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200907203%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DR.%26aulast%3DErdmann%26aufirst%3DS.%26aulast%3DBovens%26aufirst%3DS.%26aulast%3DWolf%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DM.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DIdentification%2520and%2520Structure%2520of%2520Small-Molecule%2520Stabilizers%2520of%252014%25E2%2580%25933-3%2520Protein-Protein%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26issue%3D24%26spage%3D4129%26epage%3D4132%26doi%3D10.1002%2Fanie.200907203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B. D.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery and Its Application to Challenging Drug Targets</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1042/EBC20170029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FEBC20170029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=29118094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=475-484&issue=5&author=A.+J.+Priceauthor=S.+Howardauthor=B.+D.+Cons&title=Fragment-Based+Drug+Discovery+and+Its+Application+to+Challenging+Drug+Targets&doi=10.1042%2FEBC20170029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery and its application to challenging drug targets</span></div><div class="casAuthors">Price Amanda J; Howard Steven; Cons Benjamin D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">475-484</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a technique for identifying low molecular weight chemical starting points for drug discovery.  Since its inception 20 years ago, FBDD has grown in popularity to the point where it is now an established technique in industry and academia.  The approach involves the biophysical screening of proteins against collections of low molecular weight compounds (fragments).  Although fragments bind to proteins with relatively low affinity, they form efficient, high quality binding interactions with the protein architecture as they have to overcome a significant entropy barrier to bind.  Of the biophysical methods available for fragment screening, X-ray protein crystallography is one of the most sensitive and least prone to false positives.  It also provides detailed structural information of the protein-fragment complex at the atomic level.  Fragment-based screening using X-ray crystallography is therefore an efficient method for identifying binding hotspots on proteins, which can then be exploited by chemists and biologists for the discovery of new drugs.  The use of FBDD is illustrated here with a recently published case study of a drug discovery programme targeting the challenging protein-protein interaction Kelch-like ECH-associated protein 1:nuclear factor erythroid 2-related factor 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0VHFYivNCdPRfW6udTcc2eZ2RGUUKkLZ2bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D&md5=f8572516bfb1ecc4a0ef57b0c5ba1bc0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FEBC20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170029%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DCons%26aufirst%3DB.%2BD.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520and%2520Its%2520Application%2520to%2520Challenging%2520Drug%2520Targets%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D475%26epage%3D484%26doi%3D10.1042%2FEBC20170029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neves, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merzougui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boll, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanoulle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrelle, F.-X.</span></span> <span> </span><span class="NLM_article-title">Backbone Chemical Shift Assignments of Human 14–3-3σ</span>. <i>Biomol. NMR Assignments</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1007/s12104-018-9860-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1007%2Fs12104-018-9860-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=30377945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVCksrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2019&pages=103-107&issue=1&author=J.+F.+Nevesauthor=I.+Landrieuauthor=H.+Merzouguiauthor=E.+Bollauthor=X.+Hanoulleauthor=F.-X.+Cantrelle&title=Backbone+Chemical+Shift+Assignments+of+Human+14%E2%80%933-3%CF%83&doi=10.1007%2Fs12104-018-9860-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Backbone chemical shift assignments of human 14-3-3s</span></div><div class="casAuthors">Neves, Joao Filipe; Landrieu, Isabelle; Merzougui, Hamida; Boll, Emmanuelle; Hanoulle, Xavier; Cantrelle, Francois-Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecular NMR Assignments</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">BNAIBR</span>;
        ISSN:<span class="NLM_cas:issn">1874-270X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">14-3-3 proteins are a group of seven dimeric adapter proteins that exert their biol. function by interacting with hundreds of phosphorylated proteins, thus influencing their sub-cellular localization, activity or stability in the cell.  Due to this remarkable interaction network, 14-3-3 proteins have been assocd. with several pathologies and the protein-protein interactions (PPIs) established with a no. of partners are now considered promising drug targets.  The activity of 14-3-3 proteins is often isoform specific and to our knowledge only one out of seven isoforms, 14-3-3ζ, has been assigned.  Despite the availability of the crystal structures of all seven isoforms of 14-3-3, the addnl. NMR assignments of 14-3-3 proteins are important for both biol. mechanism studies and chem. biol. approaches.  Herein, we present a robust backbone assignment of 14-3-3s, which will allow advances in the discovery of potential therapeutic compds.  This assignment is now being applied to the discovery of both inhibitors and stabilizers of 14-3-3 PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT5a0Tg2yhtrVg90H21EOLACvtfcHk0lh0QL8Q5ckCVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVCksrbP&md5=aa9c44a9f9c8c55b718457b144a12521</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs12104-018-9860-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12104-018-9860-1%26sid%3Dliteratum%253Aachs%26aulast%3DNeves%26aufirst%3DJ.%2BF.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DMerzougui%26aufirst%3DH.%26aulast%3DBoll%26aufirst%3DE.%26aulast%3DHanoulle%26aufirst%3DX.%26aulast%3DCantrelle%26aufirst%3DF.-X.%26atitle%3DBackbone%2520Chemical%2520Shift%2520Assignments%2520of%2520Human%252014%25E2%2580%25933-3%25CF%2583%26jtitle%3DBiomol.%2520NMR%2520Assignments%26date%3D2019%26volume%3D13%26issue%3D1%26spage%3D103%26epage%3D107%26doi%3D10.1007%2Fs12104-018-9860-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Stabilization of Protein-Protein Interactions: An Underestimated Concept in Drug Discovery?</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2012</span>– <span class="NLM_lpage">2018</span>, <span class="refDoi"> DOI: 10.1002/anie.201107616</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fanie.201107616" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=2012-2018&issue=9&author=P.+Thielauthor=M.+Kaiserauthor=C.+Ottmann&title=Small-Molecule+Stabilization+of+Protein-Protein+Interactions%3A+An+Underestimated+Concept+in+Drug+Discovery%3F&doi=10.1002%2Fanie.201107616"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Stabilization of Protein-Protein Interactions: An Underestimated Concept in Drug Discovery?</span></div><div class="casAuthors">Thiel, Philipp; Kaiser, Markus; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2012-2018</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The modulation of protein-protein interactions (PPIs) has been recognized as one of the most challenging tasks in drug discovery.  While their systematic development has long been considered as intractable, this view has changed over the last years, with the first drug candidates undergoing clin. studies.  To date, the vast majority of PPI modulators are interaction inhibitors.  However, in many biol. contexts a prolonged lifespan of a PPI might be desirable, calling for the complementary approach of PPI stabilization.  In fact, nature offers impressive examples of this concept and some PPI-stabilizing natural products have already found application as important drugs.  Moreover, directed small-mol. stabilization has recently been demonstrated.  Therefore, it is time to take a closer look at the constructive side of modulating PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSOQe9isrVGrVg90H21EOLACvtfcHk0li2QbcgDsYvXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCrsbk%253D&md5=7b08cfbf9af0b49cdf36e739c907c88a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107616&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107616%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DP.%26aulast%3DKaiser%26aufirst%3DM.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSmall-Molecule%2520Stabilization%2520of%2520Protein-Protein%2520Interactions%253A%2520An%2520Underestimated%2520Concept%2520in%2520Drug%2520Discovery%253F%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26issue%3D9%26spage%3D2012%26epage%3D2018%26doi%3D10.1002%2Fanie.201107616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arfons, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarus, H. M.</span></span> <span> </span><span class="NLM_article-title">The Rise, Fall and Subsequent Triumph of Thalidomide: Lessons Learned in Drug Development</span>. <i>Ther. Adv. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1177/2040620711413165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1177%2F2040620711413165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=23556097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=291-308&issue=5&author=W.+Rehmanauthor=L.+M.+Arfonsauthor=H.+M.+Lazarus&title=The+Rise%2C+Fall+and+Subsequent+Triumph+of+Thalidomide%3A+Lessons+Learned+in+Drug+Development&doi=10.1177%2F2040620711413165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The rise, fall and subsequent triumph of thalidomide: lessons learned in drug development</span></div><div class="casAuthors">Rehman, Waqas; Arfons, Lisa M.; Lazarus, Hillard M.</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Hematology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">291-308</span>CODEN:
                <span class="NLM_cas:coden">TAHHAT</span>;
        ISSN:<span class="NLM_cas:issn">2040-6207</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Perhaps no other drug in modern medicine rivals the dramatic revitalization of thalidomide.  Originally marketed as a sedative, thalidomide gained immense popularity worldwide among pregnant women because of its effective anti-emetic properties in morning sickness.  Mounting evidence of human teratogenicity marked a dramatic fall from grace and led to widespread social, legal and economic ramifications.  Despite its tragic past thalidomide emerged several decades later as a novel and highly effective agent in the treatment of various inflammatory and malignant diseases.  In 2006 thalidomide completed its remarkable renaissance becoming the first new agent in over a decade to gain approval for the treatment of plasma cell myeloma.  The catastrophic collapse yet subsequent revival of thalidomide provides important lessons in drug development.  Never entirely abandoned by the medical community, thalidomide resurfaced as an important drug once the mechanisms of action were further studied and better understood.  Ongoing research and development of related drugs such as lenalidomide now represent a class of irreplaceable drugs in hematol. malignancies.  Further, the tragedies assocd. with this agent stimulated the legislation which revamped the FDA regulatory process, expanded patient informed consent procedures and mandated more transparency from drug manufacturers.  Finally, we review recent clin. trials summarizing selected medical indications for thalidomide with an emphasis on hematol. malignancies.  Herein, we provide a historic perspective regarding the up-and-down development of thalidomide.  Using PubMed databases we conducted searches using thalidomide and assocd. keywords highlighting pharmacol., mechanisms of action, and clin. uses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUo5hN4zSsD7Vg90H21EOLACvtfcHk0li2QbcgDsYvXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVGgsA%253D%253D&md5=a0f10d1e152e9c0db704fef9da69ae9b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1177%2F2040620711413165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2040620711413165%26sid%3Dliteratum%253Aachs%26aulast%3DRehman%26aufirst%3DW.%26aulast%3DArfons%26aufirst%3DL.%2BM.%26aulast%3DLazarus%26aufirst%3DH.%2BM.%26atitle%3DThe%2520Rise%252C%2520Fall%2520and%2520Subsequent%2520Triumph%2520of%2520Thalidomide%253A%2520Lessons%2520Learned%2520in%2520Drug%2520Development%26jtitle%3DTher.%2520Adv.%2520Hematol.%26date%3D2011%26volume%3D2%26issue%3D5%26spage%3D291%26epage%3D308%26doi%3D10.1177%2F2040620711413165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Recognition of Ternary Complexes: A New Dimension in the Structure-Guided Design of Chemical Degraders</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1042/EBC20170041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FEBC20170041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=29118097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=505-516&issue=5&author=S.+J.+Hughesauthor=A.+Ciulli&title=Molecular+Recognition+of+Ternary+Complexes%3A+A+New+Dimension+in+the+Structure-Guided+Design+of+Chemical+Degraders&doi=10.1042%2FEBC20170041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span></div><div class="casAuthors">Hughes Scott J; Ciulli Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">505-516</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular glues and bivalent inducers of protein degradation (also known as PROTACs) represent a fascinating new modality in pharmacotherapeutics: the potential to knockdown previously thought 'undruggable' targets at sub-stoichiometric concentrations in ways not possible using conventional inhibitors.  Mounting evidence suggests these chemical agents, in concert with their target proteins, can be modelled as three-body binding equilibria that can exhibit significant cooperativity as a result of specific ligand-induced molecular recognition.  Despite this, many existing drug design and optimization regimens still fixate on binary target engagement, in part due to limited structural data on ternary complexes.  Recent crystal structures of protein complexes mediated by degrader molecules, including the first PROTAC ternary complex, underscore the importance of protein-protein interactions and intramolecular contacts to the mode of action of this class of compounds.  These discoveries have opened the door to a new paradigm for structure-guided drug design: borrowing surface area and molecular recognition from nature to elicit cellular signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0b5XHngOXiV_fW6udTcc2ebH56Xg9DliT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D&md5=7470e559372f05d34c5924524a6d04bc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1042%2FEBC20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170041%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520Recognition%2520of%2520Ternary%2520Complexes%253A%2520A%2520New%2520Dimension%2520in%2520the%2520Structure-Guided%2520Design%2520of%2520Chemical%2520Degraders%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D505%26epage%3D516%26doi%3D10.1042%2FEBC20170041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Calcineurin Is a Common Target of Cyclophilin-Cyclosporin A and FKBP-FK506 Complexes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90124-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2F0092-8674%2891%2990124-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1715244" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADyaK3MXlsV2qu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=1991&pages=807-815&issue=4&author=J.+Liuauthor=J.+D.+Farmerauthor=W.+S.+Laneauthor=J.+Friedmanauthor=I.+Weissmanauthor=S.+L.+Schreiber&title=Calcineurin+Is+a+Common+Target+of+Cyclophilin-Cyclosporin+A+and+FKBP-FK506+Complexes&doi=10.1016%2F0092-8674%2891%2990124-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Calcineurin is a common target of cyclophilin-cyclosporin A and FKBP-FK506 complexes</span></div><div class="casAuthors">Liu, Jun; Farmer, Jesse D., Jr.; Lane, Willam S.; Friedman, Jeff; Weissman, Irving; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">807-15</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Although the immediate receptors (immunophilins) of the immunosuppressants cyclosporin A (CsA) and FK506 are distinct, their similar mechanisms of inhibition of cell signaling suggest that their assocd. immunophilin complexes interact with a common target.  The authors report that the complexes cyclophilin-CsA and FKBP-FK506 (but not cyclophilin, FKBP, FKBP-rapamycin, or FKBP-506BD) competitively bind to and inhibit the Ca2+- and calmodulin-dependent phosphatase calcineurin, although the binding and inhibition of calcineurin do not require calmodulin.  These results suggest that calcineurin is involved in a common step assocd. with T cell receptor and IgE receptor signaling pathways and that cyclophilin and FKBP mediate the actions of CsA and FK506, resp., by forming drug-dependent complexes with and altering the activity of calcineurin-calmodulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxfN8mg-hNhLVg90H21EOLACvtfcHk0liB5J2KKPfpig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlsV2qu7w%253D&md5=00c4883ea86e6b2c554f8f72ed717ac9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990124-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990124-H%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DFarmer%26aufirst%3DJ.%2BD.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DFriedman%26aufirst%3DJ.%26aulast%3DWeissman%26aufirst%3DI.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DCalcineurin%2520Is%2520a%2520Common%2520Target%2520of%2520Cyclophilin-Cyclosporin%2520A%2520and%2520FKBP-FK506%2520Complexes%26jtitle%3DCell%26date%3D1991%26volume%3D66%26issue%3D4%26spage%3D807%26epage%3D815%26doi%3D10.1016%2F0092-8674%2891%2990124-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albers, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">A Mammalian Protein Targeted by G1-Arresting Rapamycin-Receptor Complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>369</i></span> (<span class="NLM_issue">6483</span>),  <span class="NLM_fpage">756</span>– <span class="NLM_lpage">758</span>, <span class="refDoi"> DOI: 10.1038/369756a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1038%2F369756a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=8008069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADyaK2cXlsFSiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=1994&pages=756-758&issue=6483&author=E.+J.+Brownauthor=M.+W.+Albersauthor=T.+B.+Shinauthor=K.+Ichikawaauthor=C.+T.+Keithauthor=W.+S.+Laneauthor=S.+L.+Schreiber&title=A+Mammalian+Protein+Targeted+by+G1-Arresting+Rapamycin-Receptor+Complex&doi=10.1038%2F369756a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A mammalian protein targeted by G1-arresting rapamycin-receptor complex</span></div><div class="casAuthors">Brown, Eric J.; Albers, Mark W.; Shin, Tae Bum; Ichikawa, Kazuo; Keith, Curtis T.; Lane, William S.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">6483</span>),
    <span class="NLM_cas:pages">756-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The structurally related natural products rapamycin and FK506 bind to the same intracellular receptor, FKBP12, yet the resulting complexes interfere with distinct signaling pathway.  FKBP12-rapamycin inhibits progression through the G1 phase of the cell cycle in osteosarcoma, liver and T cells as well as in yeast, and interferes with mitogenic signaling pathways that are involved in G1 progression, namely with activation of the protein p70S6k and cyclin-dependent kinases.  Here the authors isolate a mammalian FKBP-rapamycin-assocd. protein (FRAP) whose binding to structural variants of rapamycin complexed to FKBP12 correlates with the ability of these ligands to inhibit cell-cycle progression.  Peptide sequences from purified bovine FRAP were used to isolate a human cDNA clone that is highly related to the DRR1/TOR1 and DRR2/TOR2 gene products form Saccharomyces cerevisiae.  Although it has not been previously demonstrated that either of the DRR/TOR gene products can bind the FKBP-rapamycin complex directly, these yeast genes have been genetically linked to a rapamycin-sensitive pathway and are through to encode lipid kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdC-Z1-1ylr7Vg90H21EOLACvtfcHk0liB5J2KKPfpig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlsFSiu74%253D&md5=b5c34fc2333e81011d6730cc196eb21a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F369756a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F369756a0%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DE.%2BJ.%26aulast%3DAlbers%26aufirst%3DM.%2BW.%26aulast%3DShin%26aufirst%3DT.%2BB.%26aulast%3DIchikawa%26aufirst%3DK.%26aulast%3DKeith%26aufirst%3DC.%2BT.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520Mammalian%2520Protein%2520Targeted%2520by%2520G1-Arresting%2520Rapamycin-Receptor%2520Complex%26jtitle%3DNature%26date%3D1994%26volume%3D369%26issue%3D6483%26spage%3D756%26epage%3D758%26doi%3D10.1038%2F369756a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation by PROTACs</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">138</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.pharmthera.2017.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=28223226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=138-144&author=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=C.+M.+Crews&title=Targeted+Protein+Degradation+by+PROTACs&doi=10.1016%2Fj.pharmthera.2017.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation by PROTACs</span></div><div class="casAuthors">Neklesa, Taavi K.; Winkler, James D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">138-144</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using the PROTAC technol. is emerging as a novel therapeutic method to address diseases driven by the aberrant expression of a disease-causing protein.  PROTAC mols. are bifunctional small mols. that simultaneously bind a target protein and an E3-ubiquitin ligase, thus causing ubiquitination and degrdn. of the target protein by the proteasome.  Like small mols., PROTAC mols. possess good tissue distribution and the ability to target intracellular proteins.  Herein, we highlight the advantages of protein degrdn. using PROTACs, and provide specific examples where degrdn. offers therapeutic benefit over classical enzyme inhibition.  Foremost, PROTACs can degrade proteins regardless of their function.  This includes the currently "undruggable" proteome, which comprises approx. 85% of all human proteins.  Other beneficial aspects of protein degrdn. include the ability to target overexpressed and mutated proteins, as well as the potential to demonstrate prolonged pharmacodynamics effect beyond drug exposure.  Lastly, due to their catalytic nature and the pre-requisite ubiquitination step, an exquisitely potent mols. with a high degree of degrdn. selectivity can be designed.  Impressive preclin. in vitro and in vivo PROTAC data have been published, and these data have propelled the development of clin. viable PROTACs.  With the mol. wt. falling in the 700-1000 Da range, the delivery and bioavailability of PROTACs remain the largest hurdles on the way to the clinic.  Solving these issues and demonstrating proof of concept clin. data will be the focus of many labs over the next few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4RSLWYqqBMrVg90H21EOLACvtfcHk0liB5J2KKPfpig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D&md5=08a74402a274a13c8c484cd68167a5a1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%2520by%2520PROTACs%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D174%26spage%3D138%26epage%3D144%26doi%3D10.1016%2Fj.pharmthera.2017.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anders, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschinsky, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitta, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preisig-Müller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlichthörl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renigunta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkanda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daut, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">A Semisynthetic Fusicoccane Stabilizes a Protein-Protein Interaction and Enhances the Expression of K+ Channels at the Cell Surface</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.03.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1016%2Fj.chembiol.2013.03.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=23601647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1Cmt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=583-593&issue=4&author=C.+Andersauthor=Y.+Higuchiauthor=K.+Koschinskyauthor=M.+Bartelauthor=B.+Schumacherauthor=P.+Thielauthor=H.+Nittaauthor=R.+Preisig-M%C3%BCllerauthor=G.+Schlichth%C3%B6rlauthor=V.+Reniguntaauthor=J.+Ohkandaauthor=J.+Dautauthor=N.+Katoauthor=C.+Ottmann&title=A+Semisynthetic+Fusicoccane+Stabilizes+a+Protein-Protein+Interaction+and+Enhances+the+Expression+of+K%2B+Channels+at+the+Cell+Surface&doi=10.1016%2Fj.chembiol.2013.03.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">A Semisynthetic Fusicoccane Stabilizes a Protein-Protein Interaction and Enhances the Expression of K+ Channels at the Cell Surface</span></div><div class="casAuthors">Anders, Carolin; Higuchi, Yusuke; Koschinsky, Kristin; Bartel, Maria; Schumacher, Benjamin; Thiel, Philipp; Nitta, Hajime; Preisig-Mueller, Regina; Schlichthoerl, Guenter; Renigunta, Vijay; Ohkanda, Junko; Daut, Juergen; Kato, Nobuo; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">583-593</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. stabilization of protein-protein interactions is an emerging field in chem. biol.  We show how fusicoccanes, originally identified as fungal toxins acting on plants, promote the interaction of 14-3-3 proteins with the human potassium channel TASK-3 and present a semisynthetic fusicoccane deriv. (FC-THF) that targets the 14-3-3 recognition motif (mode 3) in TASK-3.  In the presence of FC-THF, the binding of 14-3-3 proteins to TASK-3 was increased 19-fold and protein crystallog. provided the at. details of the effects of FC-THF on this interaction.  We also tested the functional effects of FC-THF on TASK channels heterologously expressed in Xenopus oocytes.  Incubation with 10 μM FC-THF was found to promote the transport of TASK channels to the cell membrane, leading to a significantly higher d. of channels at the surface membrane and increased potassium current.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV8Cuo6SYkbLVg90H21EOLACvtfcHk0liB5J2KKPfpig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1Cmt7Y%253D&md5=bb8b65604416a21f972ed491948d17d5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.03.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.03.015%26sid%3Dliteratum%253Aachs%26aulast%3DAnders%26aufirst%3DC.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DKoschinsky%26aufirst%3DK.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DSchumacher%26aufirst%3DB.%26aulast%3DThiel%26aufirst%3DP.%26aulast%3DNitta%26aufirst%3DH.%26aulast%3DPreisig-M%25C3%25BCller%26aufirst%3DR.%26aulast%3DSchlichth%25C3%25B6rl%26aufirst%3DG.%26aulast%3DRenigunta%26aufirst%3DV.%26aulast%3DOhkanda%26aufirst%3DJ.%26aulast%3DDaut%26aufirst%3DJ.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DA%2520Semisynthetic%2520Fusicoccane%2520Stabilizes%2520a%2520Protein-Protein%2520Interaction%2520and%2520Enhances%2520the%2520Expression%2520of%2520K%252B%2520Channels%2520at%2520the%2520Cell%2520Surface%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26issue%3D4%26spage%3D583%26epage%3D593%26doi%3D10.1016%2Fj.chembiol.2013.03.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sies, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halbach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haranosono, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Stabilization of the 14–3-3/Gab2 Protein-Protein Interaction (PPI) Interface</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">911</span>– <span class="NLM_lpage">918</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fcmdc.201500484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=26644359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKgs73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=911-918&issue=8&author=D.+Bierauthor=M.+Bartelauthor=K.+Siesauthor=S.+Halbachauthor=Y.+Higuchiauthor=Y.+Haranosonoauthor=T.+Brummerauthor=N.+Katoauthor=C.+Ottmann&title=Small-Molecule+Stabilization+of+the+14%E2%80%933-3%2FGab2+Protein-Protein+Interaction+%28PPI%29+Interface&doi=10.1002%2Fcmdc.201500484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule stabilization of the 14-3-3/Gab2 protein-protein interaction (PPI) interface</span></div><div class="casAuthors">Bier, David; Bartel, Maria; Sies, Katharina; Halbach, Sebastian; Higuchi, Yusuke; Haranosono, Yu; Brummer, Tilman; Kato, Nobuo; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">911-918</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Small-mol. modulation of protein-protein interactions (PPIs) is one of the most promising new areas in drug discovery.  In the vast majority of cases, only inhibition or disruption of PPIs is realized, whereas the complementary strategy of targeted stabilization of PPIs is clearly under-represented.  Here, the authors report the example of a semi-synthetic natural product deriv., ISIR-005, that stabilizes the cancer-relevant interaction of adaptor protein 14-3-3ζ and Gab2.  The crystal structure of ISIR-005 in complex with 14-3-3ζ and the binding motif of Gab2 comprising 2 phosphorylation sites (Gab2pS210pT391) showed how the stabilizing mol. binds to the rim-of-the-interface of the protein complex.  Only in the direct vicinity of 14-3-3/Gab2pT391 site was a pre-formed pocket occupied by ISIR-005; binding of the Gab2pS210 motif to 14-3-3ζ did not create an interface pocket suitable for the mol.  Accordingly, ISIR-005 only stabilizes the binding of Gab2pT391, but not the Gab2pS210 site.  This study represents structural and biochem. proof of the druggability of the 14-3-3/Gab2 PPI interface with important implications for the development of PPI stabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6MZMr-enjWrVg90H21EOLACvtfcHk0lh2Ym6buJt5kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKgs73J&md5=94d97339c17d6c5398909ac79f7d9b75</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500484%26sid%3Dliteratum%253Aachs%26aulast%3DBier%26aufirst%3DD.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DSies%26aufirst%3DK.%26aulast%3DHalbach%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DHaranosono%26aufirst%3DY.%26aulast%3DBrummer%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSmall-Molecule%2520Stabilization%2520of%2520the%252014%25E2%2580%25933-3%252FGab2%2520Protein-Protein%2520Interaction%2520%2528PPI%2529%2520Interface%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26issue%3D8%26spage%3D911%26epage%3D918%26doi%3D10.1002%2Fcmdc.201500484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lucente, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visconti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aducci, P.</span></span> <span> </span><span class="NLM_article-title">The Phytotoxin Fusicoccin Promotes Platelet Aggregation via 14–3-3-Glycoprotein Ib-IX-V Interaction</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>436</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">436</span>, <span class="refDoi"> DOI: 10.1042/BJ20102037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1042%2FBJ20102037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=21395556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFGit70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2011&pages=429-436&issue=2&author=L.+Camoniauthor=C.+Di+Lucenteauthor=S.+Viscontiauthor=P.+Aducci&title=The+Phytotoxin+Fusicoccin+Promotes+Platelet+Aggregation+via+14%E2%80%933-3-Glycoprotein+Ib-IX-V+Interaction&doi=10.1042%2FBJ20102037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The phytotoxin fusicoccin promotes platelet aggregation via 14-3-3-glycoprotein Ib-IX-V interaction</span></div><div class="casAuthors">Camoni, Lorenzo; Di Lucente, Cristina; Visconti, Sabina; Aducci, Patrizia</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">429-436</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The fungal toxin fusicoccin induces plant wilting by affecting ion transport across the plasma membrane of plant cell.  The activity of this toxin is so far unknown in humans.  In the present study fusicoccin is able to affect the platelet aggregation process.  The toxin assocs. with platelet intracellular binding sites and induces aggregation in platelet-rich plasma in a dose-dependent manner.  The authors identified the adhesion receptor glycoprotein Ib-IX-V as fusicoccin target.  The toxin promotes the binding of the regulatory 14-3-3 proteins to glycoprotein Ibα and hampers that to glycoprotein Ibβ subunit.  As a result, platelet adhesion to von Willebrand factor is stimulated, leading to platelet spreading and integrin αIIbβ3 activation.  The authors anticipate the present study to be a starting point for future therapeutic use of fusicoccin in genetic bleeding diseases characterized by qual. or quant. abnormalities of the platelet membrane-adhesion receptors.  Furthermore, the present study also sets the stage for future work to det. the potential pharmacol. application of fusicoccin as a drug directed to other 14-3-3-target complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHxEaLMHhJrrVg90H21EOLACvtfcHk0lh2Ym6buJt5kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFGit70%253D&md5=52e02275949e81f260417fcc75e7b7fa</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1042%2FBJ20102037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20102037%26sid%3Dliteratum%253Aachs%26aulast%3DCamoni%26aufirst%3DL.%26aulast%3DDi%2BLucente%26aufirst%3DC.%26aulast%3DVisconti%26aufirst%3DS.%26aulast%3DAducci%26aufirst%3DP.%26atitle%3DThe%2520Phytotoxin%2520Fusicoccin%2520Promotes%2520Platelet%2520Aggregation%2520via%252014%25E2%2580%25933-3-Glycoprotein%2520Ib-IX-V%2520Interaction%26jtitle%3DBiochem.%2520J.%26date%3D2011%26volume%3D436%26issue%3D2%26spage%3D429%26epage%3D436%26doi%3D10.1042%2FBJ20102037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paiardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aducci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cervoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutruzzolà, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lucente, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascarella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visconti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camoni, L.</span></span> <span> </span><span class="NLM_article-title">The Phytotoxin Fusicoccin Differently Regulates 14–3-3 Proteins Association to Mode III Targets</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1002/iub.1239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fiub.1239" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=24408864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1ensg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2014&pages=52-62&issue=1&author=A.+Paiardiniauthor=P.+Aducciauthor=L.+Cervoniauthor=F.+Cutruzzol%C3%A0author=C.+Di+Lucenteauthor=G.+Jansonauthor=S.+Pascarellaauthor=S.+Rinaldoauthor=S.+Viscontiauthor=L.+Camoni&title=The+Phytotoxin+Fusicoccin+Differently+Regulates+14%E2%80%933-3+Proteins+Association+to+Mode+III+Targets&doi=10.1002%2Fiub.1239"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The phytotoxin fusicoccin differently regulates 14-3-3 proteins association to mode III targets</span></div><div class="casAuthors">Paiardini, Alessandro; Aducci, Patrizia; Cervoni, Laura; Cutruzzola, Francesca; Di Lucente, Cristina; Janson, Giacomo; Pascarella, Stefano; Rinaldo, Serena; Visconti, Sabina; Camoni, Lorenzo</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-62</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">Modulation of the interaction of regulatory 14-3-3 proteins to their physiol. partners through small cell-permeant mols. is a promising strategy to control cellular processes where 14-3-3s are engaged.  Here, we show that the fungal phytotoxin fusicoccin (FC), known to stabilize 14-3-3 assocn. to the plant plasma membrane H+-ATPase, is able to stabilize 14-3-3 interaction to several client proteins with a mode III binding motif.  Isothermal titrn. calorimetry anal. of the interaction between 14-3-3s and different peptides reproducing a mode III binding site demonstrated the FC ability to stimulate 14-3-3 the assocn.  Moreover, mol. docking studies provided the structural rationale for the differential FC effect, which exclusively depends on the biochem. properties of the residue in peptide C-terminal position.  Our study proposes FC as a promising tool to control cellular processes regulated by 14-3-3 proteins, opening new perspectives on its potential pharmacol. applications. © 2014 IUBMB Life, 2014.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwaPxiwp-vtrVg90H21EOLACvtfcHk0lh2Ym6buJt5kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1ensg%253D%253D&md5=52771e2713a2fff7c139145b6048679e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fiub.1239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.1239%26sid%3Dliteratum%253Aachs%26aulast%3DPaiardini%26aufirst%3DA.%26aulast%3DAducci%26aufirst%3DP.%26aulast%3DCervoni%26aufirst%3DL.%26aulast%3DCutruzzol%25C3%25A0%26aufirst%3DF.%26aulast%3DDi%2BLucente%26aufirst%3DC.%26aulast%3DJanson%26aufirst%3DG.%26aulast%3DPascarella%26aufirst%3DS.%26aulast%3DRinaldo%26aufirst%3DS.%26aulast%3DVisconti%26aufirst%3DS.%26aulast%3DCamoni%26aufirst%3DL.%26atitle%3DThe%2520Phytotoxin%2520Fusicoccin%2520Differently%2520Regulates%252014%25E2%2580%25933-3%2520Proteins%2520Association%2520to%2520Mode%2520III%2520Targets%26jtitle%3DIUBMB%2520Life%26date%3D2014%26volume%3D66%26issue%3D1%26spage%3D52%26epage%3D62%26doi%3D10.1002%2Fiub.1239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo-Rodriguez, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Anguita, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klärner, F.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Garcia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Molecular Tweezers Modulate 14–3-3 Protein-Protein Interactions</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1038/nchem.1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1038%2Fnchem.1570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=23422566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislSqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=234-239&issue=3&author=D.+Bierauthor=R.+Roseauthor=K.+Bravo-Rodriguezauthor=M.+Bartelauthor=J.+M.+Ramirez-Anguitaauthor=S.+Duttauthor=C.+Wilchauthor=F.-G.+Kl%C3%A4rnerauthor=E.+Sanchez-Garciaauthor=T.+Schraderauthor=C.+Ottmann&title=Molecular+Tweezers+Modulate+14%E2%80%933-3+Protein-Protein+Interactions&doi=10.1038%2Fnchem.1570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular tweezers modulate 14-3-3 protein-protein interactions</span></div><div class="casAuthors">Bier, David; Rose, Rolf; Bravo-Rodriguez, Kenny; Bartel, Maria; Ramirez-Anguita, Juan Manuel; Dutt, Som; Wilch, Constanze; Klaerner, Frank-Gerrit; Sanchez-Garcia, Elsa; Schrader, Thomas; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-239</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Supramol. chem. has recently emerged as a promising way to modulate protein functions, but devising mols. that will interact with a protein in the desired manner is difficult as many competing interactions exist in a biol. environment (with solvents, salts or different sites for the target biomol.).  We now show that lysine-specific mol. tweezers bind to a 14-3-3 adapter protein and modulate its interaction with partner proteins.  The tweezers inhibit binding between the 14-3-3 protein and two partner proteins - a phosphorylated (c-Raf) protein and an unphosphorylated one (ExoS) - in a concn.-dependent manner.  Protein crystallog. shows that this effect arises from the binding of the tweezers to a single surface-exposed lysine (Lys214) of the 14-3-3 protein in the proximity of its central channel, which normally binds the partner proteins.  A combination of structural anal. and computer simulations provides rules for the tweezers' binding preferences, thus allowing us to predict their influence on this type of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVofbsBIufTLVg90H21EOLACvtfcHk0lgte4UKEhXCpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislSqur0%253D&md5=7aa2fdf28d3f5643530fb6705803e112</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1570%26sid%3Dliteratum%253Aachs%26aulast%3DBier%26aufirst%3DD.%26aulast%3DRose%26aufirst%3DR.%26aulast%3DBravo-Rodriguez%26aufirst%3DK.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DRamirez-Anguita%26aufirst%3DJ.%2BM.%26aulast%3DDutt%26aufirst%3DS.%26aulast%3DWilch%26aufirst%3DC.%26aulast%3DKl%25C3%25A4rner%26aufirst%3DF.-G.%26aulast%3DSanchez-Garcia%26aufirst%3DE.%26aulast%3DSchrader%26aufirst%3DT.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DMolecular%2520Tweezers%2520Modulate%252014%25E2%2580%25933-3%2520Protein-Protein%2520Interactions%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26issue%3D3%26spage%3D234%26epage%3D239%26doi%3D10.1038%2Fnchem.1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo-Rodriguez, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sowislok, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillory, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briels, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heid, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wettig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Garcia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">The Molecular Tweezer CLR01 Stabilizes a Disordered Protein-Protein Interface</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">45</span>),  <span class="NLM_fpage">16256</span>– <span class="NLM_lpage">16263</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b07939</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b07939" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KnsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=16256-16263&issue=45&author=D.+Bierauthor=S.+Mittalauthor=K.+Bravo-Rodriguezauthor=A.+Sowislokauthor=X.+Guilloryauthor=J.+Brielsauthor=C.+Heidauthor=M.+Bartelauthor=B.+Wettigauthor=L.+Brunsveldauthor=E.+Sanchez-Garciaauthor=T.+Schraderauthor=C.+Ottmann&title=The+Molecular+Tweezer+CLR01+Stabilizes+a+Disordered+Protein-Protein+Interface&doi=10.1021%2Fjacs.7b07939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The Molecular Tweezer CLR01 Stabilizes a Disordered Protein-Protein Interface</span></div><div class="casAuthors">Bier, David; Mittal, Sumit; Bravo-Rodriguez, Kenny; Sowislok, Andrea; Guillory, Xavier; Briels, Jeroen; Heid, Christian; Bartel, Maria; Wettig, Burkhard; Brunsveld, Luc; Sanchez-Garcia, Elsa; Schrader, Thomas; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">16256-16263</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein regions that are involved in protein-protein interactions (PPIs) very often display a high degree of intrinsic disorder, which is reduced during the recognition process.  A prime example is binding of the rigid 14-3-3 adapter proteins to their numerous partner proteins, whose recognition motifs undergo an extensive disorder-to-order transition.  In this context, it is highly desirable to control this entropy-costly process using tailored stabilizing agents.  This study reveals how the mol. tweezer CLR01 tunes the 14-3-3/Cdc25C phosphoSer-216 protein-protein interaction.  Protein crystallog., biophys. affinity detn., and biomol. simulations unanimously delivered a remarkable finding: a supramol. "Janus" ligand can bind simultaneously to a flexible peptidic PPI recognition motif and to a well-structured adapter protein.  This binding fills a gap in the protein-protein interface, "freezes" one of the conformational states of the intrinsically disordered Cdc25C protein partner, and enhances the apparent affinity of the interaction.  This is the 1st structural and functional proof of a supramol. ligand targeting a PPI interface and stabilizing the binding of an intrinsically disordered recognition motif to a rigid partner protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJI6JdwJSqELVg90H21EOLACvtfcHk0lgte4UKEhXCpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KnsbzE&md5=b68488d6d9534f1c640811b58e851ac3</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b07939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b07939%26sid%3Dliteratum%253Aachs%26aulast%3DBier%26aufirst%3DD.%26aulast%3DMittal%26aufirst%3DS.%26aulast%3DBravo-Rodriguez%26aufirst%3DK.%26aulast%3DSowislok%26aufirst%3DA.%26aulast%3DGuillory%26aufirst%3DX.%26aulast%3DBriels%26aufirst%3DJ.%26aulast%3DHeid%26aufirst%3DC.%26aulast%3DBartel%26aufirst%3DM.%26aulast%3DWettig%26aufirst%3DB.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DSanchez-Garcia%26aufirst%3DE.%26aulast%3DSchrader%26aufirst%3DT.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DThe%2520Molecular%2520Tweezer%2520CLR01%2520Stabilizes%2520a%2520Disordered%2520Protein-Protein%2520Interface%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D45%26spage%3D16256%26epage%3D16263%26doi%3D10.1021%2Fjacs.7b07939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">An Optimised Small-Molecule Stabiliser of the 14–3-3-PMA2 Protein-Protein Interaction</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">6520</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1002/chem.201103761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1002%2Fchem.201103761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22467351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=6520-6527&issue=21&author=A.+Richterauthor=R.+Roseauthor=C.+Hedbergauthor=H.+Waldmannauthor=C.+Ottmann&title=An+Optimised+Small-Molecule+Stabiliser+of+the+14%E2%80%933-3-PMA2+Protein-Protein+Interaction&doi=10.1002%2Fchem.201103761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">An Optimised Small-Molecule Stabiliser of the 14-3-3-PMA2 Protein-Protein Interaction</span></div><div class="casAuthors">Richter, Anja; Rose, Rolf; Hedberg, Christian; Waldmann, Herbert; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6520-6527, S6520/1-S6520/11</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Modulation of protein-protein interactions (PPIs) is a highly demanding, but also a very promising approach in chem. biol. and targeted drug discovery.  In contrast to inhibiting PPIs with small, chem. tractable mols., stabilization of these interactions can only be achieved with complex natural products, like rapamycin, FK506, taxol, forskolin, brefeldin and fusicoccin.  Fusicoccin stabilizes the activatory complex of the plant H+-ATPase PMA2 and 14-3-3 proteins.  Recently, the stabilizing effect of fusicoccin could be mimicked by a trisubstituted pyrrolinone (pyrrolidone1, 1).  Here, the authors report the synthesis, functional activity and crystal structure of derivs. of 1 that stabilize the 14-3-3-PMA2 complex.  With a limited compd. collection three modifications that are important for activity enhancement could be detd.: (1) conversion of the pyrrolinone scaffold into a pyrazole, (2) introduction of a tetrazole moiety to the Ph ring that contacts PMA2, and (3) addn. of a bromine to the Ph ring that exclusively contacts the 14-3-3 protein.  The crystal structure of a pyrazole deriv. of 1 in complex with 14-3-3 and PMA2 revealed that the more rigid core of this mol. positions the stabilizer deeper into the rim of the interface, enlarging esp. the contact surface to PMA2.  Combination of the aforementioned features gave rise to a mol. (I) that displays a threefold increase in stabilizing the 14-3-3-PMA2 complex over 1.  Compd. I and the other active derivs. show no effect on two other important 14-3-3 protein-protein interactions, i.e., with CRaf and p53.  This is the first study that describes the successful optimization of a PPI stabilizer identified by screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF-pisQc62uLVg90H21EOLACvtfcHk0lgte4UKEhXCpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGmsbg%253D&md5=59d07edb06986bf0f8bca795b2cd07b9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fchem.201103761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201103761%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DR.%26aulast%3DHedberg%26aufirst%3DC.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DAn%2520Optimised%2520Small-Molecule%2520Stabiliser%2520of%2520the%252014%25E2%2580%25933-3-PMA2%2520Protein-Protein%2520Interaction%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2012%26volume%3D18%26issue%3D21%26spage%3D6520%26epage%3D6527%26doi%3D10.1002%2Fchem.201103761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skora, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Identification of Two Secondary Ligand Binding Sites in 14–3-3 Proteins Using Fragment Screening</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">3972</span>– <span class="NLM_lpage">3982</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.7b00153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.7b00153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A280%3ADC%252BC1cjkvFKnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=3972-3982&issue=30&author=E.+Sijbesmaauthor=L.+Skoraauthor=S.+Leysenauthor=L.+Brunsveldauthor=U.+Kochauthor=P.+Nussbaumerauthor=W.+Jahnkeauthor=C.+Ottmann&title=Identification+of+Two+Secondary+Ligand+Binding+Sites+in+14%E2%80%933-3+Proteins+Using+Fragment+Screening&doi=10.1021%2Facs.biochem.7b00153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Two Secondary Ligand Binding Sites in 14-3-3 Proteins Using Fragment Screening</span></div><div class="casAuthors">Sijbesma Eline; Leysen Seppe; Brunsveld Luc; Ottmann Christian; Skora Lukasz; Jahnke Wolfgang; Koch Uwe; Nussbaumer Peter; Ottmann Christian</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">3972-3982</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteins typically interact with multiple binding partners, and often different parts of their surfaces are employed to establish these protein-protein interactions (PPIs).  Members of the class of 14-3-3 adapter proteins bind to several hundred other proteins in the cell.  Multiple small molecules for the modulation of 14-3-3 PPIs have been disclosed; however, they all target the conserved phosphopeptide binding channel, so that selectivity is difficult to achieve.  Here we report on the discovery of two individual secondary binding sites that have been identified by combining nuclear magnetic resonance-based fragment screening and X-ray crystallography.  The two pockets that these fragments occupy are part of at least three physiologically relevant and structurally characterized 14-3-3 PPI interfaces, including those with serotonin N-acetyltransferase and plant transcription factor FT.  In addition, the high degree of conservation of the two sites implies their relevance for 14-3-3 PPIs.  This first identification of secondary sites on 14-3-3 proteins bound by small molecule ligands might facilitate the development of new chemical tool compounds for more selective PPI modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDbS6TEx8PgqNc3HjJRoFAfW6udTcc2eafGfR3EDPhaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjkvFKnsg%253D%253D&md5=0f05f0492b5dee35edb079ce8d945a98</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.7b00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.7b00153%26sid%3Dliteratum%253Aachs%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DSkora%26aufirst%3DL.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DKoch%26aufirst%3DU.%26aulast%3DNussbaumer%26aufirst%3DP.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520Two%2520Secondary%2520Ligand%2520Binding%2520Sites%2520in%252014%25E2%2580%25933-3%2520Proteins%2520Using%2520Fragment%2520Screening%26jtitle%3DBiochemistry%26date%3D2017%26volume%3D56%26issue%3D30%26spage%3D3972%26epage%3D3982%26doi%3D10.1021%2Facs.biochem.7b00153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valenti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrelle, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hristeva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lentini
Santo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obšil, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanoulle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzalis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Set-up and Screening of a Fragment Library Targeting the 14–3-3 Protein Interface</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1796</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1039/C9MD00215D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1039%2FC9MD00215D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=31814953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVWjsbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1796-1802&issue=10&author=D.+Valentiauthor=J.+F.+Nevesauthor=F.-X.+Cantrelleauthor=S.+Hristevaauthor=D.+Lentini%0ASantoauthor=T.+Ob%C5%A1ilauthor=X.+Hanoulleauthor=L.+M.+Levyauthor=D.+Tzalisauthor=I.+Landrieuauthor=C.+Ottmann&title=Set-up+and+Screening+of+a+Fragment+Library+Targeting+the+14%E2%80%933-3+Protein+Interface&doi=10.1039%2FC9MD00215D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Set-up and screening of a fragment library targeting the 14-3-3 protein interface</span></div><div class="casAuthors">Valenti, Dario; Neves, Joao Filipe; Cantrelle, Francois-Xavier; Hristeva, Stanimira; Lentini Santo, Domenico; Obsil, Tomas; Hanoulle, Xavier; Levy, Laura M.; Tzalis, Dimitrios; Landrieu, Isabelle; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1796-1802</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are at the core of regulation mechanisms in biol. systems and consequently became an attractive target for therapeutic intervention.  PPIs involving the adapter protein 14-3-3 are representative examples given the broad range of partner proteins forming a complex with one of its seven human isoforms.  Given the challenges represented by the nature of these interactions, fragment-based approaches offer a valid alternative for the development of PPI modulators.  After having assembled a fragment set tailored on PPIs' modulation, we started a screening campaign on the sigma isoform of 14-3-3 adapter proteins.  Through the use of both mono- and bi-dimensional NMR spectroscopy measurements, coupled with differential scanning fluorimetry, three fragment hits were identified.  These mols. bind the protein at two different regions distant from the usual binding groove highlighting new possibilities for selective modulation of 14-3-3 complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwc2MbT8kL7rVg90H21EOLACvtfcHk0lhyoQZ8s6snOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVWjsbfM&md5=07f022f6f9b5f8d225dd5b7562679934</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1039%2FC9MD00215D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00215D%26sid%3Dliteratum%253Aachs%26aulast%3DValenti%26aufirst%3DD.%26aulast%3DNeves%26aufirst%3DJ.%2BF.%26aulast%3DCantrelle%26aufirst%3DF.-X.%26aulast%3DHristeva%26aufirst%3DS.%26aulast%3DLentini%2BSanto%26aufirst%3DD.%26aulast%3DOb%25C5%25A1il%26aufirst%3DT.%26aulast%3DHanoulle%26aufirst%3DX.%26aulast%3DLevy%26aufirst%3DL.%2BM.%26aulast%3DTzalis%26aufirst%3DD.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSet-up%2520and%2520Screening%2520of%2520a%2520Fragment%2520Library%2520Targeting%2520the%252014%25E2%2580%25933-3%2520Protein%2520Interface%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26issue%3D10%26spage%3D1796%26epage%3D1802%26doi%3D10.1039%2FC9MD00215D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallenbeck, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vink, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skóra, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Site-Directed Fragment-Based Screening for the Discovery of Protein-Protein Interaction Stabilizers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">3524</span>– <span class="NLM_lpage">3531</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b11658</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b11658" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVWjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=3524-3531&issue=8&author=E.+Sijbesmaauthor=K.+K.+Hallenbeckauthor=S.+Leysenauthor=P.+J.+de+Vinkauthor=L.+Sk%C3%B3raauthor=W.+Jahnkeauthor=L.+Brunsveldauthor=M.+R.+Arkinauthor=C.+Ottmann&title=Site-Directed+Fragment-Based+Screening+for+the+Discovery+of+Protein-Protein+Interaction+Stabilizers&doi=10.1021%2Fjacs.8b11658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Site-Directed Fragment-Based Screening for the Discovery of Protein-Protein Interaction Stabilizers</span></div><div class="casAuthors">Sijbesma, Eline; Hallenbeck, Kenneth K.; Leysen, Seppe; de Vink, Pim J.; Skora, Lukasz; Jahnke, Wolfgang; Brunsveld, Luc; Arkin, Michelle R.; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3524-3531</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modulation of protein-protein interactions (PPIs) by small mols. has emerged as a valuable approach in drug discovery.  Compared to direct inhibition, PPI stabilization is vastly underexplored but has strong advantages, including the ability to gain selectivity by targeting an interface formed only upon assocn. of proteins.  Here, we present the application of a site-directed screening technique based on disulfide trapping (tethering) to select for fragments that enhance the affinity between protein partners.  We target the phosphorylation-dependent interaction between the hub protein 14-3-3σ and a peptide derived from Estrogen Receptor α (ERα), an important breast cancer target that is neg. regulated by 14-3-3σ.  We identify orthosteric stabilizers that increase 14-3-3/ERα affinity up to 40-fold and propose the mechanism of stabilization based on X-ray crystal structures.  These fragments already display partial selectivity toward ERα-like motifs over other representative 14-3-3 clients.  This first of its kind study illustrates the potential of the tethering approach to overcome the hurdles in systematic PPI stabilizer discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1_kzNxRmRPrVg90H21EOLACvtfcHk0lhyoQZ8s6snOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVWjtrY%253D&md5=5e68f397675437fffdca4cc47ed8e078</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b11658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b11658%26sid%3Dliteratum%253Aachs%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DHallenbeck%26aufirst%3DK.%2BK.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3Dde%2BVink%26aufirst%3DP.%2BJ.%26aulast%3DSk%25C3%25B3ra%26aufirst%3DL.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSite-Directed%2520Fragment-Based%2520Screening%2520for%2520the%2520Discovery%2520of%2520Protein-Protein%2520Interaction%2520Stabilizers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26issue%3D8%26spage%3D3524%26epage%3D3531%26doi%3D10.1021%2Fjacs.8b11658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skolnick, J.</span></span> <span> </span><span class="NLM_article-title">The Distribution of Ligand-Binding Pockets around Protein-Protein Interfaces Suggests a General Mechanism for Pocket Formation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">3784</span>– <span class="NLM_lpage">3789</span>, <span class="refDoi"> DOI: 10.1073/pnas.1117768109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=10.1073%2Fpnas.1117768109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=22355140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvFymsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=3784-3789&issue=10&author=M.+Gaoauthor=J.+Skolnick&title=The+Distribution+of+Ligand-Binding+Pockets+around+Protein-Protein+Interfaces+Suggests+a+General+Mechanism+for+Pocket+Formation&doi=10.1073%2Fpnas.1117768109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The distribution of ligand-binding pockets around protein-protein interfaces suggests a general mechanism for pocket formation</span></div><div class="casAuthors">Gao, Mu; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3784-3789</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein-protein and protein-ligand interactions are ubiquitous in a biol. cell.  Here, we report a comprehensive study of the distribution of protein-ligand interaction sites, namely ligand-binding pockets, around protein-protein interfaces where protein-protein interactions occur.  We inspected a representative set of 1,611 representative protein-protein complexes and identified pockets with a potential for binding small mol. ligands.  The majority of these pockets are within a 6 Å distance from protein interfaces.  Accordingly, in about half of ligand-bound protein-protein complexes, amino acids from both sides of a protein interface are involved in direct contacts with at least one ligand.  Statistically, ligands are closer to a protein-protein interface than a random surface patch of the same solvent accessible surface area.  Similar results are obtained in an anal. of the ligand distribution around domain-domain interfaces of 1,416 nonredundant, two-domain protein structures.  Furthermore, comparable sized pockets as obsd. in exptl. structures are present in artificially generated protein complexes, suggesting that the prominent appearance of pockets around protein interfaces is mainly a structural consequence of protein packing and thus, is an intrinsic geometric feature of protein structure.  Nature may take advantage of such a structural feature by selecting and further optimizing for biol. function.  We propose that packing nearby protein-protein or domain-domain interfaces is a major route to the formation of ligand-binding pockets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMMAJh2-zxkrVg90H21EOLACvtfcHk0liVdK30AhWztw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvFymsbc%253D&md5=696417061d6c6575c0b4608bffdc3dd3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1117768109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1117768109%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DM.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DThe%2520Distribution%2520of%2520Ligand-Binding%2520Pockets%2520around%2520Protein-Protein%2520Interfaces%2520Suggests%2520a%2520General%2520Mechanism%2520for%2520Pocket%2520Formation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D10%26spage%3D3784%26epage%3D3789%26doi%3D10.1073%2Fpnas.1117768109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S40" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S40','PDB','6S40'); return false;">PDB: 6S40</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWI','PDB','6RWI'); return false;">PDB: 6RWI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RLH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RLH','PDB','6RLH'); return false;">PDB: 6RLH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RX2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RX2','PDB','6RX2'); return false;">PDB: 6RX2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5L','PDB','6R5L'); return false;">PDB: 6R5L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWH','PDB','6RWH'); return false;">PDB: 6RWH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YZ5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YZ5','PDB','1YZ5'); return false;">PDB: 1YZ5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N75" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N75','PDB','5N75'); return false;">PDB: 5N75</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MHC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MHC','PDB','5MHC'); return false;">PDB: 5MHC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RHC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RHC','PDB','6RHC'); return false;">PDB: 6RHC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLW','PDB','6SLW'); return false;">PDB: 6SLW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJQ','PDB','6RJQ'); return false;">PDB: 6RJQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLX','PDB','6SLX'); return false;">PDB: 6SLX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJL','PDB','6RJL'); return false;">PDB: 6RJL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RP6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RP6','PDB','6RP6'); return false;">PDB: 6RP6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL3','PDB','6RL3'); return false;">PDB: 6RL3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S9Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S9Q','PDB','6S9Q'); return false;">PDB: 6S9Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWS','PDB','6RWS'); return false;">PDB: 6RWS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWU','PDB','6RWU'); return false;">PDB: 6RWU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIP','PDB','6SIP'); return false;">PDB: 6SIP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIQ','PDB','6SIQ'); return false;">PDB: 6SIQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLV','PDB','6SLV'); return false;">PDB: 6SLV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RK8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RK8','PDB','6RK8'); return false;">PDB: 6RK8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJZ','PDB','6RJZ'); return false;">PDB: 6RJZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RM5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RM5','PDB','6RM5'); return false;">PDB: 6RM5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIO','PDB','6SIO'); return false;">PDB: 6SIO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S39" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S39','PDB','6S39'); return false;">PDB: 6S39</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S3C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S3C','PDB','6S3C'); return false;">PDB: 6S3C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIN','PDB','6SIN'); return false;">PDB: 6SIN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKK','PDB','6RKK'); return false;">PDB: 6RKK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKM','PDB','6RKM'); return false;">PDB: 6RKM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKI','PDB','6RKI'); return false;">PDB: 6RKI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL6','PDB','6RL6'); return false;">PDB: 6RL6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL4','PDB','6RL4'); return false;">PDB: 6RL4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RM7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RM7','PDB','6RM7'); return false;">PDB: 6RM7</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i57"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01942">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10585"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01942?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01942</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional results and X-ray crystallography tables (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_002.csv">CSV</a>)</p></li><li><p class="inline">PDB docking AZ-008 to p53 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_001.pdf">jm9b01942_si_001.pdf (6.15 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_002.csv">jm9b01942_si_002.csv (1.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01942/suppl_file/jm9b01942_si_003.pdb">jm9b01942_si_003.pdb (322.99 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors have been deposited in the Protein Data Bank under the following accession codes. 14-3-3σ/TAZpS89/fragment complexes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RHC">6RHC</a> (AZ-003); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLW">6SLW</a> (AZ-004); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJQ">6RJQ</a> (AZ-006); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLX">6SLX</a> (AZ-010); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJL">6RJL</a> (AZ-018); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RP6">6RP6</a> (AZ-019). 14-3-3σ/p53pT387/Fragment complexes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S40">6S40</a> (AZ-001); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWI">6RWI</a> (AZ-002); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL3">6RL3</a> (AZ-003); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S9Q">6S9Q</a> (AZ-004); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RX2">6RX2</a> (AZ-005); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5L">6R5L</a> (AZ-006); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWH">6RWH</a> (AZ-007); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWS">6RWS</a> (AZ-009); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RWU">6RWU</a> (AZ-010); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIP">6SIP</a> (AZ-011); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIQ">6SIQ</a> (AZ-012); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SLV">6SLV</a> (AZ-013); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RK8">6RK8</a> (AZ-014); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJZ">6RJZ</a> (AZ-015); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RM5">6RM5</a> (AZ-016); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIO">6SIO</a> (AZ-017); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S39">6S39</a> (AZ-018); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S3C">6S3C</a> (AZ-019); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIN">6SIN</a> (AZ-020); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKK">6RKK</a> (AZ-21); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKM">6RKM</a> (AZ-022); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RKI">6RKI</a> (AZ-023); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL6">6RL6</a> (AZ-024); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RL4">6RL4</a> (AZ-025); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RM7">6RM7</a> (AZ-026). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01942&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-13%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01942%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01942" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993195ba0a227f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
