
<html lang="en"     class="pb-page"  data-request-id="767efd58-d412-44d1-b9da-0395e2220682"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c01441;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-1;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure–Activity Relationship Studies of Pyrimidine-4-Carboxamides as Inhibitors of N-Acylphosphatidylethanolamine Phospholipase D" /></meta><meta name="dc.Creator" content="Elliot D.  Mock" /></meta><meta name="dc.Creator" content="Ioli  Kotsogianni" /></meta><meta name="dc.Creator" content="Wouter P. F.  Driever" /></meta><meta name="dc.Creator" content="Carmen S.  Fonseca" /></meta><meta name="dc.Creator" content="Jelle M.  Vooijs" /></meta><meta name="dc.Creator" content="Hans  den Dulk" /></meta><meta name="dc.Creator" content="Constant A. A.  van Boeckel" /></meta><meta name="dc.Creator" content="Mario  van der Stelt" /></meta><meta name="dc.Description" content="N-Acylphosphatidylethanolamine phospholipase D (NAPE-PLD) is regarded as the main enzyme responsible for the biosynthesis of N-acylethanolamines (NAEs), a family of bioactive lipid mediators. Previ..." /></meta><meta name="Description" content="N-Acylphosphatidylethanolamine phospholipase D (NAPE-PLD) is regarded as the main enzyme responsible for the biosynthesis of N-acylethanolamines (NAEs), a family of bioactive lipid mediators. Previ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 31, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01441" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 The Authors. Published by American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01441" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01441" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01441" /></link>
        
    
    

<title>Structure–Activity Relationship Studies of Pyrimidine-4-Carboxamides as Inhibitors of N-Acylphosphatidylethanolamine Phospholipase D | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01441" /></meta><meta property="og:title" content="Structure–Activity Relationship Studies of Pyrimidine-4-Carboxamides as Inhibitors of N-Acylphosphatidylethanolamine Phospholipase D" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0020.jpeg" /></meta><meta property="og:description" content="N-Acylphosphatidylethanolamine phospholipase D (NAPE-PLD) is regarded as the main enzyme responsible for the biosynthesis of N-acylethanolamines (NAEs), a family of bioactive lipid mediators. Previously, we reported N-(cyclopropylmethyl)-6-((S)-3-hydroxypyrrolidin-1-yl)-2-((S)-3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (1, LEI-401) as the first potent and selective NAPE-PLD inhibitor that decreased NAEs in the brains of freely moving mice and modulated emotional behavior [Mock  Nat Chem. Biol., 2020, 16, 667−675]. Here, we describe the structure–activity relationship (SAR) of a library of pyrimidine-4-carboxamides as inhibitors of NAPE-PLD that led to the identification of LEI-401. A high-throughput screening hit was modified at three different substituents to optimize its potency and lipophilicity. Conformational restriction of an N-methylphenethylamine group by replacement with an (S)-3-phenylpiperidine increased the inhibitory potency 3-fold. Exchange of a morpholine substituent for an (S)-3-hydroxypyrrolidine reduced the lipophilicity and further increased activity by 10-fold, affording LEI-401 as a nanomolar potent inhibitor with drug-like properties. LEI-401 is a suitable pharmacological tool compound to investigate NAPE-PLD function in vitro and in vivo." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01441"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01441">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01441&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01441&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01441&amp;href=/doi/10.1021/acs.jmedchem.0c01441" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 481-515</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01431" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01420" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure–Activity Relationship Studies of Pyrimidine-4-Carboxamides as Inhibitors of <i>N</i>-Acylphosphatidylethanolamine Phospholipase D</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Elliot D. Mock</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elliot D. Mock</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Physiology, Leiden Institute of Chemistry, Leiden University and Oncode Institute, 2300 RA Leiden, Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elliot+D.++Mock">Elliot D. Mock</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1500-2856" title="Orcid link">http://orcid.org/0000-0002-1500-2856</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ioli Kotsogianni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ioli Kotsogianni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Physiology, Leiden Institute of Chemistry, Leiden University and Oncode Institute, 2300 RA Leiden, Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ioli++Kotsogianni">Ioli Kotsogianni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wouter P. F. Driever</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wouter P. F. Driever</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Physiology, Leiden Institute of Chemistry, Leiden University and Oncode Institute, 2300 RA Leiden, Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wouter+P.+F.++Driever">Wouter P. F. Driever</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carmen S. Fonseca</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carmen S. Fonseca</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Physiology, Leiden Institute of Chemistry, Leiden University and Oncode Institute, 2300 RA Leiden, Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carmen+S.++Fonseca">Carmen S. Fonseca</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jelle M. Vooijs</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jelle M. Vooijs</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Physiology, Leiden Institute of Chemistry, Leiden University and Oncode Institute, 2300 RA Leiden, Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jelle+M.++Vooijs">Jelle M. Vooijs</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hans den Dulk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hans den Dulk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Physiology, Leiden Institute of Chemistry, Leiden University and Oncode Institute, 2300 RA Leiden, Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hans++den+Dulk">Hans den Dulk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Constant A. A. van Boeckel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Constant A. A. van Boeckel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Physiology, Leiden Institute of Chemistry, Leiden University and Oncode Institute, 2300 RA Leiden, Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Constant+A.+A.++van+Boeckel">Constant A. A. van Boeckel</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Mario van der Stelt</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mario van der Stelt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Physiology, Leiden Institute of Chemistry, Leiden University and Oncode Institute, 2300 RA Leiden, Netherlands</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3954174f5857175d5c4b174a4d5c554d795a515c5417555c505d5c574c57504f175755"><span class="__cf_email__" data-cfemail="c3aeedb5a2adeda7a6b1edb0b7a6afb783a0aba6aeedafa6aaa7a6adb6adaab5edadaf">[email protected]</span></a>. Phone: +31 (0)71 527 4768.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mario++van+der+Stelt">Mario van der Stelt</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01441&amp;href=/doi/10.1021%2Facs.jmedchem.0c01441" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 481–515</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 31, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 August 2020</li><li><span class="item_label"><b>Published</b> online</span>31 December 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 January 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01441" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01441</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc text">with CC-BY-NC-ND<a href="/page/policy/authorchoice_ccbyncnd_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01441"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1569</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01441" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure–Activity Relationship Studies of Pyrimidine-4-Carboxamides as Inhibitors of N-Acylphosphatidylethanolamine Phospholipase D&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Elliot&quot;,&quot;last_name&quot;:&quot;D. Mock&quot;},{&quot;first_name&quot;:&quot;Ioli&quot;,&quot;last_name&quot;:&quot;Kotsogianni&quot;},{&quot;first_name&quot;:&quot;Wouter&quot;,&quot;last_name&quot;:&quot;P. F. Driever&quot;},{&quot;first_name&quot;:&quot;Carmen&quot;,&quot;last_name&quot;:&quot;S. Fonseca&quot;},{&quot;first_name&quot;:&quot;Jelle&quot;,&quot;last_name&quot;:&quot;M. Vooijs&quot;},{&quot;first_name&quot;:&quot;Hans&quot;,&quot;last_name&quot;:&quot;den Dulk&quot;},{&quot;first_name&quot;:&quot;Constant&quot;,&quot;last_name&quot;:&quot;A. A. van Boeckel&quot;},{&quot;first_name&quot;:&quot;Mario&quot;,&quot;last_name&quot;:&quot;van der Stelt&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;481-515&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01441&quot;},&quot;abstract&quot;:&quot;N-Acylphosphatidylethanolamine phospholipase D (NAPE-PLD) is regarded as the main enzyme responsible for the biosynthesis of N-acylethanolamines (NAEs), a family of bioactive lipid mediators. Previously, we reported N-(cyclopropylmethyl)-6-((S)-3-hydroxypyrrolidin-1-yl)-2-((S)-3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (1, LEI-401) as the first potent and selective NAPE-PLD inhibitor that decreased NAEs in the brains of freely moving mice and modulated emotional behavior [ Mock Nat Chem. Biol. , 2020 , 16 , 667 − 675 ]. Here, we describe the structure–activity relationship (SAR) of a library of pyrimidine-4-carboxamides as inhibitors of NAPE-PLD that led to the identification of LEI-401. A high-throughput screening hit was modified at three different substituents to optimize its potency and lipophilicity. Conformational restriction of an N-methylphenethylamine group by replacement with an (S)-3-phenylpiperidine increased the inhibitory potency 3-fold. Exchange of a morpholine substituent for an (S)-3-hyd&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01441&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01441" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01441&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01441" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01441&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01441" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01441&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01441&amp;href=/doi/10.1021/acs.jmedchem.0c01441" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01441" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01441" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01441%26sid%3Dliteratum%253Aachs%26pmid%3D33382264%26genre%3Darticle%26aulast%3DMock%26date%3D2021%26atitle%3DStructure%25E2%2580%2593Activity%2BRelationship%2BStudies%2Bof%2BPyrimidine-4-Carboxamides%2Bas%2BInhibitors%2Bof%2BN-Acylphosphatidylethanolamine%2BPhospholipase%2BD%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D1%26spage%3D481%26epage%3D515%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=291181" title="Amines">Amines</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/jmcmar.2021.64.issue-1/20210114/jmcmar.2021.64.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText"><i>N</i>-Acylphosphatidylethanolamine phospholipase D (NAPE-PLD) is regarded as the main enzyme responsible for the biosynthesis of <i>N</i>-acylethanolamines (NAEs), a family of bioactive lipid mediators. Previously, we reported <i>N</i>-(cyclopropylmethyl)-6-((<i>S</i>)-3-hydroxypyrrolidin-1-yl)-2-((<i>S</i>)-3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>1</b>, <b>LEI-401</b>) as the first potent and selective NAPE-PLD inhibitor that decreased NAEs in the brains of freely moving mice and modulated emotional behavior [<contrib-group person-group-type="allauthors"><span class="NLM_string-name">Mock</span></contrib-group>   <cite><i>Nat Chem. Biol.</i></cite>, <span class="NLM_year">2020</span>, <em>16</em>, 667−675]. Here, we describe the structure–activity relationship (SAR) of a library of pyrimidine-4-carboxamides as inhibitors of NAPE-PLD that led to the identification of <b>LEI-401</b>. A high-throughput screening hit was modified at three different substituents to optimize its potency and lipophilicity. Conformational restriction of an <i>N</i>-methylphenethylamine group by replacement with an (<i>S</i>)-3-phenylpiperidine increased the inhibitory potency 3-fold. Exchange of a morpholine substituent for an (<i>S</i>)-3-hydroxypyrrolidine reduced the lipophilicity and further increased activity by 10-fold, affording <b>LEI-401</b> as a nanomolar potent inhibitor with drug-like properties. <b>LEI-401</b> is a suitable pharmacological tool compound to investigate NAPE-PLD function <i>in vitro</i> and <i>in vivo</i>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81143" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81143" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><i>N</i>-Acylphosphatidylethanolamine phospholipase D (NAPE-PLD) is considered to be the principal enzyme that produces <i>N-</i>acylethanolamines (NAEs), a family of signaling lipids.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> NAPE-PLD catalyzes the hydrolysis of <i>N</i>-acylphosphatidylethanolamines (NAPEs) to NAEs, which includes the endocannabinoid anandamide (<i>N</i>-arachidonoylethanolamine, AEA).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The NAE lipids exert their biological activity through the activation of various G-protein-coupled receptors (cannabinoid receptors CB<sub>1</sub> and CB<sub>2</sub>, GPR55, GPR110, and GPR119), ion channels (transient receptor potential vanilloid 1, TRPV1), and nuclear receptors (peroxisome proliferator-activated receptor α, PPAR-α).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Accordingly, NAEs are involved in numerous physiological processes such as appetite, satiety, pain, inflammation, fertility, stress, and anxiety.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Furthermore, aberrant NAE levels are associated with metabolic syndrome and non-alcoholic steatohepatitis (NASH).<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> Thus, there is a need for pharmacological tools that can inhibit NAPE-PLD to study its role in cellular and animal models to further our understanding of NAE biology.</div><div class="NLM_p">To date, NAPE-PLD has been characterized in several biochemical studies.<a onclick="showRef(event, 'ref3 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref3 ref9 ref10">(3,9,10)</a> It was found to have a wide distribution among murine organs with higher abundance in brain, kidney, and testis tissues.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> A crystal structure of NAPE-PLD confirmed that the enzyme has a metallo-β-lactamase fold with two Zn<sup>2+</sup> ions in its active site.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> NAPE-PLD was shown to be membrane-associated and activated by phosphatidylethanolamine (PE), suggesting that the enzyme is constitutively active.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Interestingly, these structural studies also revealed that, <i>in vitro,</i> NAPE-PLD can form a homodimer, which binds to specific bile acids in an allosteric site that promote dimerization and increase enzymatic activity.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Also, polyamines have been reported to enhance NAPE-PLD activity <i>in vitro</i>.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Whether bile acids and polyamines play an active role in regulating NAPE-PLD dimerization and activity <i>in vivo</i> has yet to be established, in particular, in the CNS.</div><div class="NLM_p">Several inhibitors for NAPE-PLD have been reported (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Out of a small library of NAPE substrate mimics, phosphoramidate AHP-71B was described as an inhibitor with micromolar potency (half-maximum inhibitory concentration IC<sub>50</sub> ≈ 10 μM).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Other reported active compounds were the β-lactamase substrate nitrocefin,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> desketoraloxifene analogue <b>17b</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> that also targeted phospholipase D1 (PLD1), endogenous bile acid lithocholic acid,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and sulfonamide ARN19874.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> All compounds showed poor to moderate potency for NAPE-PLD <i>in vitro</i>. Of note, ARN19874 was able to increase NAPE levels in HEK293 cells but did not affect most NAE levels.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Recently, the disinfectant hexachlorophene was reported as a NAPE-PLD inhibitor with low micromolar activity.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> However, this compound has neurotoxic effects and is therefore not suited for <i>in vivo</i> use.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Thus, to study the biological functions of NAEs in cellular and animal models, new and more potent chemotypes are needed that can inhibit the enzymatic activity of NAPE-PLD.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of reported NAPE-PLD inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Previously, we performed a high-throughput screening of a library of ∼350,000 compounds, which identified pyrimidine-4-carboxamide <b>2</b> as an inhibitor of NAPE-PLD with sub-micromolar potency (pIC<sub>50</sub> = 6.09 ± 0.04, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Generation of a small library of close analogues of <b>2</b> afforded the optimized NAPE-PLD inhibitor <b>1</b> (<b>LEI-401</b>), which exhibited nanomolar potency (pIC<sub>50</sub> = 7.14 ± 0.04 μM, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). <b>LEI-401</b> reduced NAE levels including anandamide in Neuro-2a cells as well as in the brains of freely moving mice. In addition, <b>LEI-401</b> elicited a marked effect on emotional behavior in mice by activating the hypothalamus-pituitary–adrenal (HPA) axis and reducing fear extinction of an aversive memory. Here, we describe the structure–activity relationship (SAR) of a library of NAPE-PLD inhibitors that afforded <b>LEI-401</b>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of <i>in vivo</i> active NAPE-PLD inhibitor <b>1</b> (<b>LEI-401</b>), HTS-hit <b>2</b>, and the core pyrimidine-4-carboxamide scaffold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58471" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58471" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">To study the SAR of hit <b>2</b>, different synthetic routes were employed that allowed systematic variation of the pyrimidine scaffold, the R<sub>1</sub> amide, or R<sub>2</sub> and R<sub>3</sub> substituents (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). This led to the synthesis of compounds <b>1</b> and <b>3</b>–<b>107</b> with modified core scaffolds (compounds <b>3</b>–<b>6</b>) or modifications at R<sub>1</sub> (<b>7</b>–<b>30</b>), R<sub>2</sub> (<b>31</b>–<b>70</b>), and R<sub>3</sub> (<b>71</b>–<b>100</b>) or combinations thereof (<b>1</b> and <b>101</b>–<b>107</b>). First, the influence of the nitrogen atoms in the pyrimidyl ring was investigated with the synthesis of pyridyl analogues <b>3</b> and <b>4</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). For compound <b>3</b>, this commenced with the regioselective nucleophilic aromatic substitution (S<sub>N</sub>Ar) of dichloride <b>108</b> with <i>N</i>-methylphenethylamine generating <b>109</b>. Subsequent ester hydrolysis and amide coupling afforded <b>111</b>, which was converted to <b>3</b> with morpholine using Buchwald–Hartwig amination conditions.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Isomer <b>4</b> was synthesized in four steps from symmetric dichloride <b>112</b>: S<sub>N</sub>Ar with morpholine, ester hydrolysis, and amide coupling giving <b>115</b> followed by similar Pd-catalyzed amination with <i>N-</i>methylphenethylamine.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyridyl Analogues <b>3</b> and <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>N</i>-methylphenethylamine, DiPEA, MeOH, rt, 41%; (b) NaOH, THF, H<sub>2</sub>O, rt; for <b>110</b>: 89% and for <b>114</b>: 99%; (c) cyclopropylmethanamine, EDC·HCl, HOBt, DCM, rt; for <b>111</b>: 24% and for <b>115</b>: 80%; (d) morpholine, RuPhos-Pd-G3, NaOtBu, THF, toluene, 110 °C, 37%; (e) morpholine, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, reflux, 66%; (f) <i>N</i>-methylphenethylamine, RuPhos-Pd-G3, NaOtBu, THF, toluene, 110 °C, 41%.</p></p></figure><div class="NLM_p">Next, a systematic synthesis of analogues of <b>2</b> with varying R<sub>1</sub>, R<sub>2</sub>, and R<sub>3</sub> substituents was performed. R<sub>1</sub> amide derivatives were generated via two general four-step sequences, which either introduced the amide in the second (compounds <b>10</b>, <b>11</b>, <b>13</b>–<b>17</b>, <b>27</b>, and <b>28</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A) or final step (<b>8</b>, <b>9</b>, <b>12</b>, <b>18</b>, <b>23</b>, <b>25</b>, and <b>26</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>B). This shortened the synthetic sequence from three to only one reaction for each R<sub>1</sub> amide derivative, respectively. The route depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A was also used to synthesize R<sub>2</sub> amine analogues (<b>33</b>–<b>67</b> and <b>69</b>) and R<sub>3</sub> amine (<b>71</b>–<b>76</b> and <b>80</b>–<b>96</b>), heteroaromatic rings (<b>97</b> and <b>98</b>), or phenol derivatives (<b>99</b> and <b>100</b>). The synthesis started with orotic acid (<b>116</b>), which was converted to acyl chloride <b>117</b> using phosphorous oxychloride. Cold addition (−78 to 0 °C) of various primary amines gave amides <b>118a</b>–<b>k</b>. The more electrophilic 4-chloro substituent of the dichloropyrimidine was regioselectively substituted with various amine, heteroaromatic, or phenolic nucleophiles to afford <b>119a</b>–<b>af</b>. Finally, high temperature and/or microwave irradiation was used to couple different R<sub>3</sub> amines to the 2-chloropyrimidine scaffold, which provided the desired products. Non-commercially available <i>N</i>-methylphenethylamines that were used as R<sub>2</sub> amines were synthesized from benzylic halides <b>120a</b> and <b>120b</b>, which were converted to their corresponding nitriles (<b>121a</b> and <b>121b</b>) followed by hydrogenation, affording their primary amines (<b>122a</b> and <b>122b</b>). Mono-<i>N-</i>methylation was achieved by carbamoylation and subsequent LiAlH<sub>4</sub> reduction, giving the <i>N</i>-methylphenethylamines <b>123a</b>–<b>o</b>. Alternatively, phenethylamine was converted to the <i>N</i>-phenyl analogue <b>124</b> via Chan–Lam coupling<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> or to <i>N</i>-alkyl derivatives <b>125a</b>–<b>e</b> by reductive amination with aldehydes or ketones. The secondary route for introduction of the R<sub>1</sub> amide in the final step consisted of regioselective substitution of dichloropyrimidine <b>126</b> to give <b>127</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>B). Then, ester hydrolysis followed by coupling with <i>N</i>-methylphenethylamine gave carboxylic acid <b>19</b>, which was condensed with various amines. Molecules not listed in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> or <a class="ref internalNav" href="#sch2" aria-label="2">2</a> (compounds <b>5–7</b>, <b>20</b>–<b>22</b>, <b>24</b>, <b>29</b>, <b>30</b>, <b>32</b>, <b>68</b>, <b>69</b>, <b>77</b>–<b>79</b>, and <b>92</b>) were synthesized according to the routes described in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_001.pdf" class="ext-link">Schemes S1–S10</a>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. (A) General Synthetic Route for Analogues of Compound <b>2</b>; (B) Alternative Synthetic Route for Amide Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) POCl<sub>3</sub>, DMF, reflux, 60%; (b) R<sub>1</sub>NH<sub>2</sub>, Et<sub>3</sub>N, DCM, −78 °C to 0 °C, 78–99%; (c) (cyclo)alkylNH, DiPEA, MeOH, 0 °C, 32–99% or (hetero)arylOH or heteroarylNH, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 51–76%; (d) <b>123a</b>–<b>o</b> or <b>124</b> or <b>125a</b>–<b>e</b>, DiPEA, <i>n</i>-BuOH, μW, 160 °C or oil bath, 120 °C, 21% – 97%; (e) KCN, EtOH, dioxane, H<sub>2</sub>O, reflux, 84–99%; (f) H<sub>2</sub>, Pd/C, HCl, EtOH, rt, 98–99%; (g) methyl chloroformate, DiPEA, DCM, 0 °C to rt; (h) LiAlH<sub>4</sub>, THF, 0 °C to reflux, 40–94% over 2 steps; (i) phenylboronic acid, Cu(OAc)<sub>2</sub>·H<sub>2</sub>O, 4 Å MS, O<sub>2</sub>, DCE, rt, 32%; (j) aldehyde or ketone, NaB(OAc)<sub>3</sub>H, AcOH, DCM, rt, 18–63%. (k) NaOH, THF, MeOH, H<sub>2</sub>O, rt, 99%; (l) <i>N</i>-methylphenethylamine, DiPEA, <i>n</i>-BuOH, 120 °C, 51%; (m) R<sub>1</sub>NH<sub>2</sub>, PyBOP, DiPEA, DMF, 0 °C to rt, 43–55%.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Biology</h3><div class="NLM_p">A biochemical NAPE-PLD activity assay<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> was performed to measure the inhibitory potency of compounds <b>1</b>–<b>107</b> using membrane lysates of HEK293T cells overexpressing human NAPE-PLD. The assay uses the fluorescence-quenched substrate PED6 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_001.pdf" class="ext-link">Figure S1</a>), which can report on various phospholipase activities including PLA<sub>2</sub> and PLD. In the case of NAPE-PLD, hydrolysis of the PED6 phosphodiester results in the release of the quencher from the BODIPY fluorophore, providing a direct read-out of enzyme activity. The data are reported in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<named-content content-type="anchor" rid="tbl2" type="simple"></named-content><named-content content-type="anchor" rid="tbl3" type="simple"></named-content><named-content content-type="anchor" rid="tbl4" type="simple"></named-content><named-content content-type="anchor" rid="tbl5" type="simple"></named-content><named-content content-type="anchor" rid="tbl6" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a> as pIC<sub>50</sub> ± SEM (<i>N</i> = 2, <i>n</i> = 2; the mean of two independent experiments with two biological replicates). First, to identify the essential nitrogen atom contributions of the scaffold, pyridyl analogues <b>3</b> and <b>4</b>, pyrimidyl regioisomer <b>5</b>, and triazine <b>6</b> were evaluated (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The removal of the X<sub>2</sub>-nitrogen (compound <b>3</b>) but not X<sub>1</sub> (compound <b>4</b>) resulted in a 10-fold drop in potency. This suggested that the X<sub>2</sub>-nitrogen may form an important <i>H</i>-bond interaction with the protein, while the electron-withdrawing effect seems less important. A significant decrease in potency was also observed for regioisomer <b>5</b>, while triazine <b>6</b> was completely inactive. This indicated that the pyrimidine scaffold of the hit <b>2</b> was optimal.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Activity Data for Hit <b>2</b> and Scaffold Analogues <b>3</b>–<b>6</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0007.gif" alt="" id="gr6" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">X<sub>1</sub></th><th class="colsep0 rowsep0" align="center">X<sub>2</sub></th><th class="colsep0 rowsep0" align="center">X<sub>3</sub></th><th class="colsep0 rowsep0" align="center" char=".">pIC<sub>50</sub> ± SEM</th><th class="colsep0 rowsep0" align="center" char="."><i>c</i>Log<i>P</i><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="char" char=".">6.09 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">3.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="char" char=".">4.98 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">4.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="char" char=".">5.84 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">3.90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char=".">5.39 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">3.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="."><4.3</td><td class="colsep0 rowsep0" align="char" char=".">3.09</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><i>c</i>Log<i>P</i> was calculated using Chemdraw 15.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure–Activity Relationship Analysis of R<sub>1</sub> Amide Analogues <b>7</b>–<b>30</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0008.gif" alt="" id="gr7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0009.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last"><i>c</i>Log<i>P</i> was calculated using Chemdraw 15.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structure–Activity Relationship (SAR) Analysis of R<sub>2</sub> Analogues <b>31</b>–<b>53</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0010.gif" alt="" id="gr9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0011.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last"><i>c</i>Log<i>P</i> was calculated using Chemdraw 15.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Structure–Activity Relationship (SAR) Analysis of R<sub>2</sub> Analogues <b>54</b>–<b>59</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0012.gif" alt="" id="gr11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0013.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last"><i>c</i>Log<i>P</i> was calculated using Chemdraw 15.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Structure–Activity Relationship (SAR) Analysis of R<sub>2</sub> Analogues <b>60</b>–<b>70</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0014.gif" alt="" id="gr13" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0015.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last"><i>c</i>Log<i>P</i> was calculated using Chemdraw 15.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Structure–Activity Relationship (SAR) Analysis of R<sub>3</sub> Analogues <b>71</b>–<b>100</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0016.gif" alt="" id="gr15" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0017.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last"><i>c</i>Log<i>P</i> was calculated using Chemdraw 15.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Structure–Activity Relationship (SAR)-Analysis of Optimized Analogues <b>101</b>–<b>107</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0018.gif" alt="" id="gr17" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0019.gif" alt="" id="gr18" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last"><i>c</i>Log<i>P</i> was calculated using Chemdraw 15.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">Lipophilic efficiency (LipE) = pIC<sub>50</sub> – <i>c</i>Log<i>P</i>.</p></div></div><div></div></div><div class="NLM_p">Next, the influence of the amide R<sub>1</sub> substituent was investigated. Methylation of the amide in compound <b>2</b> resulted in complete loss of potency (compound <b>7</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), suggesting that the amide may form another hydrogen bond or, alternatively, that the methyl group has a steric clash with the enzyme. Removal of the methylene group (<b>8</b>) reduced the activity<b>,</b> whereas linear alkylamides <b>9</b>–<b>14</b> showed optimal inhibition with a propyl chain. Branching of the alkyl substituent and introduction of heteroatoms or larger aromatic groups were less favorable (compounds <b>15</b>, <b>16</b>, <b>18</b>, and <b>26</b>–<b>28</b>). The 10-fold drop in potency for isobutylamide <b>15</b> may be attributed to the increased size of the isobutyl group or lack of π character compared to the cyclopropyl moiety.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Of note, propargylamide <b>17</b> was equally active compared to the hit. Substituting the lipophilic amide for more polar analogues did not result in increased activities (compounds <b>19</b>–<b>25</b>), although glycine methyl ester <b>21</b> showed to be equipotent to <b>2</b>. Lastly, the amide bioisostere imidazole <b>29</b> displayed a substantial decrease in potency, while the amide isomer <b>30</b> was 10-fold less active than <b>2</b>. In conclusion, the cyclopropylmethylamide of <b>2</b> is the optimal R<sub>1</sub> substituent of the tested series, which suggests that it occupies a small lipophilic pocket.</div><div class="NLM_p">To assess the influence of the R<sub>2</sub> substituent on the inhibitory activity, a large number of structural analogues (<b>31</b>–<b>70</b>) were evaluated (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). Analogues <b>31</b>–<b>53</b> demonstrated that the <i>N</i>-methylphenethylamine is important for inhibitory activity as its complete removal resulted in inactive compounds (<b>31</b> and <b>32</b>) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). <i>N</i>-Methyl was found to be preferred over the hydrogen of <b>33</b>. A similar trend was apparent for benzylic amines <b>34</b> and <b>35</b>. Reducing (<b>34</b>) or increasing the alkyl chain length (<b>36</b> and <b>37</b>) decreased the potency, indicating that an ethylene linker is optimal. Various large substituents (<i>e</i>.<i>g</i>., phenyl) on the phenyl group were tolerated, but only at the <i>ortho</i> position (compounds <b>38</b>–<b>49</b>), suggesting that there is space in the binding pocket. Both electron-donating (methyl (<b>41</b>) and methoxy (<b>43</b>)) and withdrawing (chloro (<b>38</b>) and CF<sub>3</sub> (<b>45</b>)) substituents at the <i>para</i> position reduced the activity. Replacing the phenyl for a pyridyl ring was not favorable (<b>50</b>–<b>52</b>), while the thiophene isostere <b>53</b> displayed similar potency compared to <b>2</b>. <i>N</i>-Alkyl analogues <b>54</b>–<b>59</b> demonstrated that larger groups than methyl are allowed (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). In particular, isopropyl derivative <b>55</b> displayed a 2-fold increase in activity, albeit with a significant lipophilicity penalty. Next, several cyclic phenethylamine derivatives were evaluated (compounds <b>60</b>–<b>70</b>) to study the effect of conformational restriction by reducing the number of rotatable bonds (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). A 2-fold activity improvement was observed for both 3-phenylpiperidine <b>62</b> and 2-benzylpyrrolidine <b>63</b>. Introduction of heteroatoms in the piperidine ring was not favored as witnessed by morpholine <b>67</b> and piperazine <b>68</b>, but the activity could be recovered by introducing a <i>N</i>-benzyl group in the piperazine analogue <b>69</b>.</div><div class="NLM_p">To study the SAR of the R<sub>3</sub> substituent, inhibitors <b>71</b>–<b>100</b> were evaluated (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Substitution of the morpholine for a more hydrophobic piperidine (<b>71</b>) was allowed, while the 3,3-difluoropiperidine <b>72</b> increased the potency 2-fold. The 4-position of the morpholine ring was less favorable for substitution (compounds <b>73</b>–<b>80</b>). Replacing the morpholine with a dimethylamine <b>81</b> increased the activity 2-fold, suggesting that the morpholine <b>1</b> is too polar or may experience steric hindrance in the pocket. Several other small alkylamines were tested (<b>82</b>–<b>87</b>). Pyrrolidine <b>87</b> was the most effective with almost a 4-fold increase in potency. Substitutions on the pyrrolidine ring were investigated (compounds <b>88</b>–<b>94</b>), revealing that hydroxylation on the 3-position (<b>89</b>) resulted in similar potency to pyrrolidine <b>87</b> while decreasing the <i>c</i>Log<i>P</i> with more than one log unit. Both enantiomers of the 3-hydroxypyrrolidine (<b>90</b> and <b>91</b>) were equally active. Of note, introduction of aromatic substituents was allowed (<b>94</b>–<b>100</b>) but did not improve the potency of the inhibitors.</div><div class="NLM_p">Combination of the optimal R<sub>1</sub> (cyclopropylmethylamide), R<sub>2</sub> ((<i>R</i>/<i>S</i>)-3-phenylpiperidine), and various R<sub>3</sub> substituents (dimethylamine, morpholine, or (<i>R</i>/<i>S</i>)-3-hydroxypyrrolidine) resulted in compounds <b>1</b> and <b>101</b>–<b>107</b> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). It was found that the combination of (<i>S</i>)-3-phenylpiperidine with (<i>S</i>)-3-hydroxypyrrolidine afforded the most potent compound (<b>1</b>, pIC<sub>50</sub> = 7.14 ± 0.04, IC<sub>50</sub> = 72 nM; 95% confidence interval: 0.061–0.086 nM), having a 10-fold increase in activity compared to <b>2</b>. Importantly, <b>1</b> completely blocked the turnover of PED6 at a dose of 10 μM, indicating full efficacy (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_001.pdf" class="ext-link">Figure S2</a>). Interestingly, the (<i>R</i>,<i>R</i>) enantiomer of <b>1</b> (compound <b>107</b>) showed 3-fold-reduced activity. The substantial <i>c</i>Log<i>P</i> reduction for <b>1</b> resulted in the highest lipophilic efficiency of this series (LipE = 3.68). In view of the inhibitory activity and optimal LipE, compound <b>1</b> (termed <b>LEI-401</b>) was selected as the lead compound for further biological profiling.</div><div class="NLM_p">Our attempts to dock <b>LEI-401</b> in the active site of the reported NAPE-PLD crystal structure (PDB ID: 4QN9<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a>), did not provide binding poses that confidently recapitulated the SAR as described in this work. This may be attributed to the large hydrophobic binding cavity of the endogenous NAPE substrate, which facilitates a large number of possible poses for <b>LEI-401</b>. Alternatively, <b>LEI-401</b> may bind in an unidentified allosteric pocket. Future co-crystallization studies are needed to identify the binding pocket of <b>LEI-401</b> in NAPE-PLD.</div><div class="NLM_p">Because the biological profiling of NAPE-PLD inhibitors is mostly performed in mouse models, it was assessed whether <b>LEI-401</b> showed any species difference using recombinant mouse NAPE-PLD expressed in HEK293T cells. Despite high homology between human and mouse NAPE-PLD (89%), it was found that <b>LEI-401</b> showed somewhat lower potency (pIC<sub>50</sub> = 6.35 ± 0.04) for mouse NAPE-PLD, although optimal activity compared to other inhibitors was retained (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).</div><div class="NLM_p last">Lastly, the NAPE-PLD PED6 activity assay was used to compare the potency of <b>LEI-401</b> to three reported inhibitors: lithocholic acid, ARN19874, and hexachlorophene (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_001.pdf" class="ext-link">Figure S2, Table S1</a>). ARN19874 and hexachlorophene were active (IC<sub>50</sub> of 54 μM and 11 μM, respectively) in a similar order of magnitude as previously reported,<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> whereas lithocholic acid was not active.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86012" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86012" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have described the optimization of a library of pyrimidine-4-carboxamide derivatives as inhibitors of the NAE-producing enzyme NAPE-PLD. Our primary focus was to increase the potency of hit compound <b>2</b> and to improve its physicochemical properties to allow <i>in vivo</i> use. The main findings of the SAR of <b>2</b> are depicted in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. No improvement in inhibitory activity could be achieved by changing the substituent at R<sub>1</sub>, which suggests that it may bind in a shallow lipophilic pocket. Conformational restriction of the <i>N</i>-methylphenethylamine substituent at R<sub>2</sub> by introduction of an (<i>S</i>)-3-phenylpiperidine afforded a 3-fold potency increase. Exchange of the morpholine group at R<sub>3</sub> for the smaller and more polar (<i>S</i>)-3-hydroxypyrrolidine gave a 10-fold increase in activity when combined with the optimal R<sub>2</sub> substituent. This provided the most potent NAPE-PLD inhibitor so far, termed <b>LEI-401</b> (pIC<sub>50</sub> ± SEM = 7.14 ± 0.04; <i>K</i><sub>i</sub> = 27 nM), with favorable drug-like properties. Previously, we have shown target engagement of <b>LEI-401</b> with NAPE-PLD in live cell photoaffinity labeling experiments.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Furthermore, <b>LEI-401</b> decreased anandamide levels in neuronal cells and in mouse brain at a dose of 30 mg/kg (intraperitoneal injection). At this dose, <b>LEI-401</b> displayed profound effects on mouse emotional behavior. We anticipate that <b>LEI-401</b>, by blocking the biosynthesis of NAEs, will provide new opportunities to study the biological role of NAPE-PLD and its enzymatic products in health and disease.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure–activity map for the pyrimidine-4-carboxamide NAPE-PLD inhibitor library.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10870" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10870" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Biological Procedures</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Cloning of Plasmid DNA</h4><div class="NLM_p last">Full length human cDNA of human or mouse NAPE-PLD (obtained from Natsuo Ueda) was cloned into mammalian expression vector pcDNA3.1, containing a C-terminal FLAG-tag and genes for ampicillin and neomycin resistance. All plasmids were grown in XL-10 Z-competent cells and prepped (Maxi Prep, Qiagen). Constructs were verified by Sanger sequencing (Macrogen).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Cell Culture</h4><div class="NLM_p last">HEK293T cells (ATCC) were cultured at 37 °C and 7% CO<sub>2</sub> in DMEM (Sigma Aldrich, D6546) with GlutaMax (2 mM), penicillin (100 μg/mL, Duchefa), streptomycin (100 μg/mL, Duchefa), and 10% (v/v) newborn calf serum (Seradigm). Cells were passaged twice a week to appropriate confluence by thorough pipetting.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Transient Transfection</h4><div class="NLM_p last">One day before transfection 10<sup>7</sup> cells were seeded on a 15 cm dish. Two hours before transfection, the medium was refreshed with 13 mL of the medium. Transfection was performed with polyethyleneimine (PEI, 60 μg/dish) in a ratio of 3:1 with plasmid DNA (20 μg per dish). The PEI and plasmid DNA were incubated in serum-free medium (2 mL/dish) at rt for 15 min followed by dropwise addition to the cells. Transfection with the empty pcDNA3.1 vector was used to generate control (mock) samples. The medium was refreshed after 24 h and cells were harvested after 48 or 72 h in cold PBS. Cells were centrifuged (10 min, 200<i>g</i>, 4 °C), and the supernatant was removed. The cell pellets were flash-frozen in liquid N<sub>2</sub> and stored at −80 °C.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Cell Lysate Preparation</h4><div class="NLM_p last">Cell pellets were resuspended in cold lysis buffer (20 mM HEPES, 2 mM DTT, 0.25 M sucrose, 1 mM MgCl<sub>2</sub>, and 2.5 U/mL benzonase) and incubated 30 min in ice. The cytosolic fraction (supernatant) was separated from the membranes by ultracentrifugation (30 min, 100,000<i>g</i>, 4 °C). The pellet was resuspended in storage buffer: 20 mM HEPES and 2 mM DTT (membrane fraction). All samples were stored at −80 °C. Protein concentrations were determined using a Bradford assay (Bio-Rad).</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> NAPE-PLD Surrogate Substrate Activity Assay</h4><div class="NLM_p last">The NAPE-PLD activity assay was performed as previously described.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The membrane fraction from transient overexpression of human or mouse NAPE-PLD in HEK293T cells was diluted to 0.4 μg/μL in assay buffer (50 mM Tris–HCl pH 7.5, 150 mM NaCl, and 0.02% v/v Triton X-100). The substrate PED6 (Invitrogen, D23739) 1 mM stock in DMSO was consecutively diluted in DMSO (10×) and in assay buffer (10×) to make a 10 μM working solution. Relevant concentrations of compounds (100× working solution) were prepared in DMSO. The assay was performed in a black Greiner 96-well plate (flat bottom), final volume of 100 μL. The compound or DMSO was incubated with membrane protein lysate (final concentration of 0.04 μg/μL) for 30 min at 37 °C. Then, substrate PED6 was added (final concentration of 1 μM) and the measurement was started immediately on a TECAN infinite M1000 pro at 37 °C (excitation 485 nm, emission 535 nm, gain = 100), scanning every 2 min for 1 h. Mock membrane lysate was used for background subtraction. The slope of <i>t</i> = 4 min to <i>t</i> = 14 min was used as the enzymatic rate (RFU/min), which was normalized to DMSO as 100%. IC<sub>50</sub> curves were generated using Graphpad Prism v6 (log(inhibitor) vs normalized response with variable slope). <i>K<sub>i</sub></i> values were calculated from the Cheng–Prusoff equation <i>K<sub>i</sub></i> = IC<sub>50</sub>/(1 + ([<i>S</i>]/<i>K</i><sub>M</sub>)) where <i>K</i><sub>M</sub> = 0.59 μM for both mouse and human NAPE-PLD. All measurements were performed in <i>N</i> = 2 and <i>n</i> = 2 or <i>N</i> = 2 and <i>n</i> = 4 for controls and with <i>Z</i>′ ≥ 0.6.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Synthetic Procedures</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> General</h4><div class="NLM_p last">All chemicals (Sigma-Aldrich, Fluka, Acros, Merck, Combi-Blocks, Fluorochem, TCI) were used as received. All solvents used for reactions were of analytical grade. THF, Et<sub>2</sub>O, DMF, CH<sub>3</sub>CN, and DCM were dried over activated 4 Å molecular sieves; MeOH over 3 Å molecular sieves. Flash chromatography was performed on silica gel (Screening Devices BV, 40–63 μm, 60 Å). The eluent EtOAc was of technical grade and distilled before use. Reactions were monitored by thin-layer chromatography (TLC) analysis using Merck aluminum sheets (silica gel 60, F<sub>254</sub>). Compounds were visualized by UV absorption (254 nm) and spraying for general compounds: KMnO<sub>4</sub> (20 g/L) and K<sub>2</sub>CO<sub>3</sub> (10 g/L) in water or, for amines, ninhydrin (0.75 g/L) and acetic acid (12.5 mL/L) in ethanol followed by charring at ∼150 °C. <sup>1</sup>H and <sup>13</sup>C NMR experiments were recorded on a Bruker AV-300 (300/75 MHz), Bruker AV-400 (400/101 MHz), Bruker DMX-400 (400/101 MHz), Bruker AV-500 (500/126 MHz), and Bruker AV-600 (600/151 MHz). Chemical shifts are given in parts per million (δ) relative to tetramethylsilane or CDCl<sub>3</sub> as internal standards. Multiplicity: s = singlet, br s = broad singlet, d = doublet, dd = doublet of doublet, t = triplet, q = quartet, p = pentet, m = multiplet. Coupling constants (<i>J</i>) are given in hertz. LC–MS measurements were performed on a Thermo Finnigan LCQ Advantage MAX ion-trap mass spectrometer (ESI<sup>+</sup>) coupled to a Surveyor HPLC system (Thermo Finnigan) equipped with a standard C18 (Gemini, 4.6 mm D × 50 mm L, 5 μm particle size, Phenomenex) analytical column and buffers A: H<sub>2</sub>O, B: CH<sub>3</sub>CN, and C: 0.1% aq. TFA. High-resolution mass spectra were recorded on an LTQ Orbitrap (Thermo Finnigan) mass spectrometer or a Synapt G2-Si high-definition mass spectrometer (Waters) equipped with an electrospray ion source in the positive mode (source voltage of 3.5 kV, sheath gas flow of 10 mL/min, and capillary temperature of 250 °C) with resolution R = 60,000 at <i>m</i>/<i>z</i> 400 (mass range <i>m</i>/<i>z</i> = 150–2000) and dioctylphthalate (m/z = 391.28428) as a lock mass. Preparative HPLC was performed on a Waters Acquity Ultra Performance LC with a C18 column (Gemini, 150 x 21.2 mm, Phenomenex). All final compounds were determined to be >95% pure by integrating UV intensity recorded via HPLC.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General Procedure A</h4><div class="NLM_p last">A microwave tube with a magnetic stir bar was charged with the appropriate 2-chloropyrimidine (1 equiv), <i>n</i>-BuOH (0.2 M), the appropriate amine (1.5 equiv), and DiPEA (3–4 equiv). The tube was capped, flushed with N<sub>2</sub>, and heated to 160 °C in a microwave reactor (75 W) for 4–36 h or heated to 120 °C in an oil bath for 1–6 days. When the reaction was completed as judged by LC–MS, it was transferred to a round-bottom flask, concentrated under reduced pressure, and coevaporated with toluene (2×). The residue was purified by silica gel column chromatography, affording the product, or alternatively by HPLC–MS purification, yielding the TFA salt. The free base was generated by dissolving the TFA salt in EtOAc followed by washing with sat. aq. NaHCO<sub>3</sub> (2×). The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure, affording the pure product.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> General Procedure B</h4><div class="NLM_p last">A round-bottom flask was charged with carboxylic acid (1 equiv) and dissolved in dry DMF (0.2 M). PyBOP (1.2–1.5 equiv), DiPEA (3–5 equiv), and the appropriate amine (1.2–5 equiv) were added, and the mixture was stirred overnight at rt. Work-up involved dilution with EtOAc, washing with H<sub>2</sub>O (1×) and brine (2×), drying (Na<sub>2</sub>SO<sub>4</sub>), filtering, and concentrating under reduced pressure. The residue was purified by silica gel column chromatography, affording the pure product.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Procedure C</h4><div class="NLM_p last">A microwave vial was charged with dichloropyrimidine (1 equiv) and dry MeOH (0.1 M) and cooled to 0 °C. DiPEA (1.5–2.5 equiv) and the appropriate amine (1.05 equiv) were added, and the mixture was stirred for 1–2 h at 0 °C. The solvents were evaporated under reduced pressure. The vial was charged with <i>n</i>-BuOH (0.2 M), <i>N</i>-methylphenethylamine (1.5 equiv), and DiPEA (3–4 equiv). The tube was capped, flushed with N<sub>2</sub>, and heated to 160 °C in a microwave reactor (75 W) for 4 h. When the reaction was completed as judged by LC–MS, it was transferred to a round-bottom flask, concentrated under reduced pressure, and co-evaporated with toluene (2×). The residue was purified by silica gel column chromatography, affording the product, or alternatively by HPLC–MS purification, yielding the TFA salt. The free base was generated by dissolving the TFA salt in EtOAc followed by washing with sat. aq. NaHCO<sub>3</sub> (2x). The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure, affording the pure product.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> General Procedure D</h4><div class="NLM_p last">A round-bottom flask with dry DCM (0.1 M) was charged via syringe with 2,6-dichloropyrimidine-4-carbonyl chloride (1 equiv) and cooled to −78 °C. Et<sub>3</sub>N (1.3–2.3 equiv) and the appropriate amine (1.025 equiv) were added, and the mixture was stirred, while letting the acetone bath warm up to 0 °C (3–4 h). The mixture was transferred to a separatory funnel, and the organic layer was washed with H<sub>2</sub>O (2×) and brine (1×), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. Silica gel column chromatography afforded the pure amide.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> General Procedure E</h4><div class="NLM_p last">A round-bottom flask was charged with the dichloropyrimidine (1 equiv) and dry MeOH (0.1 M) and cooled to 0 °C. DiPEA (1.5–2.5 equiv) and the appropriate amine (1.05 equiv) were added, and the mixture was stirred for 1–2 h at 0 °C. The solvents were evaporated under reduced pressure, and the crude material was purified by silica gel column chromatography, affording the pure product.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> General Procedure F</h4><div class="NLM_p last">A round-bottom flask was charged with the dichloropyrimidine (1 equiv) and dry DMF (0.1 M). K<sub>2</sub>CO<sub>3</sub> (1.5 equiv) and the appropriate phenol or heteroaryl (1.05 equiv) were added, and the mixture was stirred overnight at rt. H<sub>2</sub>O was added, and the mixture was extracted with EtOAc (3×). The organic layers were combined and washed with brine (2×), dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure. The residue was purified by silica gel column chromatography, affording the pure product.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> General Procedure G</h4><div id="sec4_2_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Carbamoylation</h5><div class="NLM_p last">a round-bottom flask was charged with the primary amine (1 equiv) and dry DCM (0.2 M). The solution was cooled to 0 °C and DiPEA (2 equiv) and methylchloroformate (1.5 equiv) were added. The reaction was stirred and allowed to warm up to room temperature over 1–2 h. Then, the mixture was diluted with DCM and washed with sat. aq. NaHCO<sub>3</sub> (2×), brine (1×), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The resulting crude material was purified by silica gel column chromatography, affording the methyl carbamate.</div></div><div id="sec4_2_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Carbamate Reduction</h5><div class="NLM_p last">A round-bottom flask was charged with the methyl carbamate (1 equiv) and dry THF (0.15 M). The solution was cooled to 0 °C, and LiAlH<sub>4</sub> (2 M in THF solution, 1.6 equiv) was added dropwise. The reaction was then stirred at reflux for 1–2 h. The Fieser workup involved dilution of the reaction mixture with Et<sub>2</sub>O (3×) and cooling to 0 °C followed by the sequential addition of water (1 μL for every 1 mg of LiAlH<sub>4</sub>), NaOH (aq) 15% (1 μL for every 1 mg of LiAlH<sub>4</sub>), and water (3 μL for every 1 mg of LiAlH<sub>4</sub>). The mixture was allowed to warm to room temperature and stirred for 15 min. Then, it was dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure to afford the product as a clear oil, which was used without further purification or purified by silica gel chromatography.</div></div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-(Cyclopropylmethyl)-6-((<i>S</i>)-3-hydroxypyrrolidin-1-yl)-2-((<i>S</i>)-3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>1</b>, <b>LEI-401</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119aa</b> (37 mg, 0.12 mmol, 1 equiv), DiPEA (65 μL, 0.37 mmol, 3 equiv), and (<i>S</i>)-3-phenylpiperidine (26 mg, 0.16 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (70% → 100% EtOAc/pentane) afforded the product (26 mg, 62 μmol, 51%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (br s, 1H), 7.40–7.20 (m, 5H), 6.53 (s, 1H), 4.84 (t, <i>J</i> = 14.3 Hz, 2H), 4.57 (s, 1H), 3.91–3.37 (m, 4H), 3.36–3.18 (m, 2H), 2.96–2.81 (m, 2H), 2.81–2.70 (m, 1H), 2.17–1.94 (m, 3H), 1.93–1.50 (m, 4H), 1.11–0.97 (m, 1H), 0.59–0.43 (m, 2H), 0.26 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.97, 162.17, 160.90, 155.63, 144.41, 128.62, 127.31, 126.58, 91.75, 71.02, 70.38, 54.95, 51.36, 44.68, 44.39, 44.05, 42.53, 32.19, 25.59, 10.91, 3.50. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 422.2551 calculated, 422.2555 found.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N-</i>(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-morpholino-pyrimidine-4-carboxamide (<b>2</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (59 mg, 0.20 mmol, 1 equiv), <i>N-</i>methylphenethylamine HBr salt (66 mg, 0.30 mmol, 1.5 equiv), and DiPEA (140 μL, 0.80 mmol, 4 equiv). Total heating time: 8 h at 160 °C with μW irradiation. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (40 mg, 0.10 mmol, 52%). TLC: <i>R</i><sub>f</sub> = 0.3 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (br s, 1H), 7.34–7.25 (m, 2H), 7.25–7.12 (m, 3H), 6.72 (s, 1H), 3.88–3.72 (m, 6H), 3.72–3.55 (m, 4H), 3.30 (t, <i>J</i> = 6.5 Hz, 2H), 3.13 (s, 3H), 2.90 (t, <i>J</i> = 7.7 Hz, 2H), 1.14–0.99 (m, 1H), 0.64–0.44 (m, 2H), 0.38–0.19 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.66, 163.97, 160.86, 156.78, 139.92, 128.95, 128.58, 126.29, 90.08, 66.74, 51.68, 44.50, 44.11, 35.70, 33.93, 10.88, 3.48. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 396.2394 calculated, 396.2387 found.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>-(Cyclopropylmethyl)-6-(methyl(phenethyl)amino)-4-morpholino-picolinamide (<b>3</b>)</h4><div class="NLM_p last">A microwave vial with a magnetic stir bar under N<sub>2</sub> was charged with 4-chloropyridine <b>111</b> (30 mg, 87 μmol, 1 equiv), morpholine (9 μL, 0.1 mmol, 1.2 equiv), and dry toluene (87 μL). The vial was capped, and the solution was purged with N<sub>2</sub>. This was followed by the addition of RuPhosPd G3 (0.01 M THF solution, 237 μL, 2.37 μmol, 0.027 equiv) and NaOtBu (2 M THF solution, 97 μL, 0.19 mmol, 2.2 equiv), and the mixture was purged again with N<sub>2</sub> and stirred in a preheated oil bath at 110 °C for 44 h. The mixture was filtered through a plug of Celite, and the filtrate was concentrated under reduced pressure. The crude material was purified by silica gel column chromatography (30% → 60% EtOAc/pentane), affording the product (5 mg, 13 μmol, 15%). TLC: <i>R</i><sub>f</sub> = 0.2 (30% EtOAc/pentane) and recovered starting material (11 mg, 32 μmol, 37%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.03 (t, <i>J</i> = 5.7 Hz, 1H), 7.29 (t, <i>J</i> = 7.3 Hz, 2H), 7.24–7.13 (m, 4H), 5.77 (d, <i>J</i> = 2.0 Hz, 1H), 3.91–3.81 (m, 4H), 3.63 (t, <i>J</i> = 7.4 Hz, 2H), 3.51–3.40 (m, 4H), 3.34–3.29 (m, 2H), 2.92–2.83 (m, 5H), 1.11–1.05 (m, 1H), 0.58–0.50 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 165.54, 160.20, 156.01, 148.69, 139.16, 129.00, 128.80, 126.61, 99.04, 90.28, 66.98, 54.09, 46.37, 44.16, 38.55, 33.55, 11.08, 3.57. HRMS [C<sub>23</sub>H<sub>30</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 395.2442 calculated, 395.2438 found.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-morpholino-isonicotinamide (<b>4</b>)</h4><div class="NLM_p last">A microwave vial with a magnetic stir bar under N<sub>2</sub> was charged with 2-chloropyridine <b>115</b> (31 mg, 0.1 mmol, 1 equiv), <i>N</i>-methylphenethylamine HBr salt (28 mg, 0.13 mmol, 1.3 equiv), and dry toluene (0.1 mL). The vial was capped, and the solution purged with N<sub>2</sub>. This was followed by the addition of RuPhosPd G3 (0.01 M THF solution, 100 μL, 1 μmol, 0.01 equiv) and NaOtBu (2 M THF solution, 120 μL, 0.24 mmol, 2.4 equiv), and the mixture was purged again with N<sub>2</sub> and stirred in a preheated oil bath at 110 °C. After 24 h, the reaction was complete as judged by LC–MS. The mixture was filtered through a plug of Celite, and the filtrate was concentrated under reduced pressure to provide the crude material. Purification by HPLC (C18 reverse phase, 45% → 55% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT, 12.3 min) afforded the product (16 mg, 40 μmol, 41%). TLC: <i>R</i><sub>f</sub> = 0.5 (60% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.35–7.24 (m, 2H), 7.25–7.14 (m, 3H), 6.24–6.13 (m, 2H), 6.10 (s, 1H), 3.87–3.78 (m, 4H), 3.78–3.67 (m, 2H), 3.59–3.47 (m, 4H), 3.28 (dd, <i>J</i> = 7.2, 5.4 Hz, 2H), 2.98 (s, 3H), 2.93–2.82 (m, 2H), 1.15–0.95 (m, 1H), 0.64–0.48 (m, 2H), 0.35–0.19 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 168.07, 159.06, 157.76, 145.86, 140.05, 128.98, 128.60, 126.26, 92.90, 91.48, 66.94, 52.51, 45.80, 44.97, 36.87, 33.92, 10.82, 3.68. HRMS [C<sub>23</sub>H<sub>30</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 395.2442 calculated, 395.2434 found.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i>-(Cyclopropylmethyl)-6-(methyl(phenethyl)amino)-2-morpholinopyrimidine-4-carboxamide (<b>5</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 4-chloropyrimidine <b>129</b> (21 mg, 70 μmol, 1.0 equiv), <i>N</i>-methylphenethylamine HBr salt (16 mg, 70 μmol, 1 equiv), and DiPEA (36.6 μL, 0.21 mmol, 3 equiv) in MeOH. Total heating time: 6 h at 70 °C. Column chromatography (30% → 60% EtOAc/pentane) afforded the product (20 mg, 50 μmol, 71%). TLC: <i>R</i><sub>f</sub> = 0.4 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.44 (br s, 1H), 7.33–7.27 (m, 2H), 7.25–7.13 (m, 3H), 6.84 (s, 1H), 3.79 (br s, 10H), 3.38–3.22 (m, 2H), 3.02 (s, 3H), 2.90 (t, <i>J</i> = 7.4 Hz, 2H), 1.14–1.00 (m, 1H), 0.62–0.46 (m, 2H), 0.36–0.21 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 162.97, 159.05, 128.91, 128.82, 126.75, 93.22, 66.81, 44.92, 44.69, 10.71, 3.65. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 396.2394 calculated, 396.2385 found.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-(Cyclopropylmethyl)-4-(methyl(phenethyl)amino)-6-morpholino-1,3,5-triazine-2-carboxamide (<b>6</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with carboxylic acid <b>134</b> (17 mg, 50 μmol, 1 equiv), PyBOP (39 mg, 75 μmol, 1.5 equiv), DiPEA (35 μL, 0.2 mmol, 4.0 equiv), cyclopropylmethanamine (5.2 μL, 60 μmol, 1.2 equiv), and DMF (0.25 mL). The solution was stirred for 70 h, diluted with water (20 mL), and extracted with EtOAc (20 mL). The organic layer was washed with brine (2× 20 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography (40%→ 80% EtOAc/pentane), affording the product (13 mg, 34 μmol, 67%). <sup>1</sup>Η NMR analysis showed two rotamers in a ratio of 1:1 (CDCl<sub>3</sub>, 295 K), which was confirmed by high temperature <sup>1</sup>H NMR experiments. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, <i>T</i> = 295 K) δ 7.93–7.80 (m, 1H), 7.29 (t, <i>J</i> = 7.5 Hz, 2H), 7.25–7.15 (m, 3H), 3.99–3.77 (m, 4H), 3.89–3.76 (m, 2H), 3.77–3.69 (m, 4H), 3.31–3.26 (m, 2H), 3.13 (d, <i>J</i> = 38.5 Hz, 3H), 2.90 (q, <i>J</i> = 8.1 Hz, 2H), 1.12–1.00 (m, 1H), 0.59–0.50 (m, 2H), 0.32–0.25 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, T = 295 K) δ 165.20 (d, <i>J</i> = 34.3 Hz), 164.96 (d, <i>J</i> = 6.8 Hz), 164.32 (d, <i>J</i> = 20.0 Hz), 163.21 (d, <i>J</i> = 2.9 Hz), 139.27 (d, <i>J</i> = 5.3 Hz), 128.88, 128.67, 126.53 (d, <i>J</i> = 4.6 Hz), 66.89, 51.01 (d, <i>J</i> = 10.9 Hz), 44.47, 43.87, 35.22 (d, <i>J</i> = 30.2 Hz), 33.85 (d, <i>J</i> = 90.2 Hz), 10.84 (d, <i>J</i> = 2.3 Hz), 3.54. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, <i>T</i> = 332 K) δ 7.78 (br s, 1H), 7.29–7.24 (m, 2H), 7.23–7.13 (m, 3H), 3.85 (br s, 6H), 3.74–3.69 (m, 4H), 3.31–3.26 (m, 2H), 3.12 (d, <i>J</i> = 31.4 Hz, 3H), 2.91 (t, <i>J</i> = 7.5 Hz, 2H), 1.10–1.00 (m, 1H), 0.57–0.49 (m, 2H), 0.30–0.24 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, <i>T</i> = 332 K) δ 165.39, 165.39, 164.71, 163.32, 139.44, 128.93, 128.72, 126.57, 66.95, 51.06, 44.49, 44.12, 35.19, 33.73, 10.93, 3.50. HRMS [C<sub>21</sub>H<sub>28</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 397.2347 calculated, 397.2343 found.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(Cyclopropylmethyl)-<i>N</i>-methyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>7</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with amide <b>2</b> (36 mg, 90 μmol, 1 equiv), dry DMF (1.5 mL), and cooled to 0 °C. NaH (60% in mineral oil, 4 mg, 0.10 mmol, 1.1 equiv) was added, and the mixture was stirred for 30 min followed by addition of methyl iodide (11 μL, 0.18 mmol, 2 equiv). The reaction was allowed to warm to rt while stirring overnight. The reaction was quenched with H<sub>2</sub>O (20 mL) followed by extraction with EtOAc (3× 20 mL). The combined organic layers were washed with brine (1× 50 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (50 → 80% EtOAc/pentane) affording the product (18 mg, 40 μmol, 48%). TLC: <i>R</i><sub>f</sub> = 0.3 (60% EtOAc/pentane). <sup>1</sup>Η NMR analysis showed two rotamers in a ratio of 6:4 (CDCl<sub>3</sub>, 298 K), which was confirmed by high-temperature <sup>1</sup>H NMR experiments. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.32–7.24 (m, 2H), 7.23–7.16 (m, 3H), 6.07–6.02 (m, 1H), 3.83–3.71 (m, 6H), 3.62–3.55 (m, 4H), 3.43–3.23 (m, 2H), 3.16–3.10 (m, 3H), 3.10–3.02 (m, 3H), 2.92–2.83 (m, 2H), 1.14–1.01 (m, 1H), 0.60–0.43 (m, 2H), 0.36–0.12 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.34, 162.35, 160.82, 156.31, 140.04, 128.91, 128.52, 126.20, 90.43, 66.76, 55.26, 51.72, 51.57, 51.44, 44.33, 36.75, 35.80, 35.69, 33.94, 33.10, 10.33, 9.26, 3.66. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 410.2551 calculated, 410.2545 found.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-Cyclopropyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>8</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_3" aria-label="General Procedure B">General Procedure B</a> using carboxylic acid <b>19</b> (34 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), PyBOP (78 mg, 0.12 mmol, 1.2 equiv), and cyclopropylamine (8.3 μL, 0.12 mmol, 1.2 equiv). Column chromatography (50% → 80% EtOAc/pentane) afforded the product (8 mg, 44 μmol, 66%). TLC: R<sub>f</sub> = 0.3 (60% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.92 (br s, 1H), 7.34–7.27 (m, 2H), 7.25–7.15 (m, 3H), 6.70 (s, 1H), 3.84–3.72 (m, 6H), 3.66 (br s, 4H), 3.09 (s, 3H), 2.94–2.82 (m, 3H), 1.36–1.21 (m, 1H), 0.92–0.82 (m, 2H), 0.67–0.58 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.25, 163.97, 160.84, 156.51, 139.98, 128.84, 128.64, 126.34, 89.86, 66.78, 51.67, 44.52, 35.72, 33.98, 22.53, 6.78. HRMS [C<sub>21</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 382.2238 calculated, 382.2241 found.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 2-(Methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>9</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_3" aria-label="General Procedure B">General Procedure B</a> using carboxylic acid <b>19</b> (27 mg, 79 μmol, 1 equiv), DiPEA (56 μL, 0.32 mmol, 4 equiv), PyBOP (62 mg, 0.12 mmol, 1.5 equiv), HOBt (16 mg, 0.12 mmol, 1.5 equiv), and ammonium chloride (15 mg, 0.32 mmol, 3.5 equiv). Column chromatography (80% → 100% EtOAc/pentane) afforded the product (20 mg, 59 μmol, 74%). TLC: <i>R</i><sub>f</sub> = 0.5 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.74 (br s, 1H), 7.34–7.25 (m, 2H), 7.25–7.16 (m, 3H), 6.71 (br s, 1H), 5.83 (s, 1H), 3.86–3.72 (m, 6H), 3.66 (br s, 4H), 3.09 (s, 3H), 2.90 (t, <i>J</i> = 7.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 167.46, 163.96, 161.04, 156.27, 139.93, 128.86, 128.62, 126.31, 90.17, 66.74, 51.60, 44.50, 35.77, 33.97. HRMS [C<sub>18</sub>H<sub>23</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 342.1925 calculated, 342.1934 found.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>N</i>-Methyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide TFA Salt (<b>10</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119a</b> (8.5:1 mixture of regioisomers) (51 mg, 0.20 mmol, 1 equiv), DiPEA (139 μL, 0.80 mmol, 4 equiv), and <i>N-</i>methylphenethylamine HBr salt (65 mg, 0.30 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Purification by preparative HPLC (C18 reverse phase, 25% to 35% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT = 8.77) afforded the product as the TFA salt (83 mg, 0.18 mmol, 88%). TLC: <i>R</i><sub>f</sub> = 0.3 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.30–7.12 (m, 5H), 6.90 (s, 1H), 3.92 (t, <i>J</i> = 7.0 Hz, 2H), 3.85–3.69 (m, 8H), 3.18 (s, 3H), 3.00–2.88 (m, 5H). <sup>13</sup>C NMR (101 MHz, MeOD) δ 162.45, 162.16 (q, <i>J</i> = 35.8 Hz), 154.95, 147.48, 139.88, 129.99, 129.67, 127.62, 117.79 (q, <i>J</i> = 291.3 Hz), 94.22, 67.37, 53.01, 46.62, 36.22, 34.27, 26.90. HRMS [C<sub>19</sub>H<sub>25</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 356.2081 calculated, 356.2079 found.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i>-Ethyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>11</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119b</b> (54 mg, 0.20 mmol, 1 equiv), DiPEA (139 μL, 0.80 mmol, 4 equiv), and <i>N</i>-methylphenethylamine HBr salt (65 mg, 0.30 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (50% → 70% EtOAc/pentane) afforded the product (64 mg, 0.17 mmol, 86%). TLC: <i>R</i><sub>f</sub> = 0.3 (50% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.88 (s, 1H), 7.32–7.26 (m, 2H), 7.24–7.16 (m, 3H), 6.71 (s, 1H), 3.84–3.77 (m, 2H), 3.77–3.73 (m, 4H), 3.69–3.62 (m, 4H), 3.50–3.42 (m, 2H), 3.11 (s, 3H), 2.94–2.84 (m, 2H), 1.25 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 164.65, 163.98, 160.88, 156.82, 139.99, 128.83, 128.59, 126.29, 90.02, 66.74, 51.63, 44.51, 35.71, 34.26, 33.98, 14.94. HRMS [C<sub>20</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 370.2238 calculated, 370.2236 found.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>-Propyl-2-(methyl(phenethyl)amino)-6-morpholino-pyrimidine-4-carboxamide (<b>12</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_3" aria-label="General Procedure B">General Procedure B</a> using carboxylic acid <b>19</b> (23 mg, 67 μmol, 1 equiv), DiPEA (60 μL, 0.34 mmol, 3 equiv), PyBOP (52 mg, 0.10 mmol, 1.5 equiv), and propylamine HCl salt (8 mg, 80 μmol, 1.2 equiv). Column chromatography (40% → 60% EtOAc/pentane) afforded the product (17 mg, 44 μmol, 66%). TLC: <i>R</i><sub>f</sub> = 0.3 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.97 (br s, 1H), 7.34–7.25 (m, 2H), 7.25–7.16 (m, 3H), 6.72 (s, 1H), 3.87–3.72 (m, 6H), 3.66 (br s, 4H), 3.39 (q, <i>J</i> = 6.7 Hz, 2H), 3.10 (s, 3H), 2.89 (t, <i>J</i> = 7.5 Hz, 2H), 1.68–1.60 (m, 2H), 0.99 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.80, 164.01, 160.86, 156.82, 139.98, 128.85, 128.61, 126.31, 90.06, 66.77, 51.65, 44.52, 41.08, 35.73, 33.98, 23.01, 11.61. HRMS [C<sub>21</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 384.2394 calculated, 384.2394 found.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-Butyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>13</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119c</b> (30 mg, 0.10 mmol, 1 equiv), DiPEA (70 μL, 0.40 mmol, 4 equiv), and <i>N</i>-methylphenethylamine HBr salt (32 mg, 0.15 mmol, 1.5 equiv). Total heating time: 45 h at 120 °C. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (29 mg, 73 μmol, 73%). TLC: <i>R</i><sub>f</sub> = 0.6 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (br s, 1H), 7.33–7.25 (m, 2H), 7.24–7.16 (m, 3H), 6.72 (s, 1H), 3.84–3.72 (m, 6H), 3.66 (br s, 4H), 3.42 (q, <i>J</i> = 6.6 Hz, 2H), 3.11 (s, 3H), 2.95–2.84 (m, 2H), 1.60 (p, <i>J</i> = 7.1 Hz, 2H), 1.42 (h, <i>J</i> = 7.3 Hz, 2H), 0.96 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.74, 164.00, 160.86, 156.83, 139.97, 128.84, 128.60, 126.31, 90.05, 66.76, 51.64, 44.52, 39.12, 35.72, 33.97, 31.81, 20.30, 13.94. HRMS [C<sub>22</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 398.2551 calculated, 398.2560 found.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N-</i>Hexyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>14</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119d</b> (33 mg, 0.10 mmol, 1 equiv), DiPEA (70 μL, 0.40 mmol, 4 equiv), and <i>N-</i>methylphenethylamine HBr salt (32 mg, 0.15 mmol, 1.5 equiv). Total heating time: 3 days at 120 °C. Column chromatography (40% → 50% EtOAc/ pentane) afforded the product (36 mg, 85 μmol, 85%). TLC: <i>R</i><sub>f</sub> = 0.6 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (s, 1H), 7.33–7.25 (m, 2H), 7.25–7.17 (m, 3H), 6.72 (s, 1H), 3.83–3.73 (m, 6H), 3.71–3.62 (m, 4H), 3.41 (q, <i>J</i> = 6.9 Hz, 2H), 3.10 (s, 3H), 2.95–2.85 (m, 2H), 1.61 (p, <i>J</i> = 7.6, 7.2 Hz, 2H), 1.44–1.27 (m, 6H), 0.94–0.84 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.71, 164.01, 160.86, 156.85, 139.97, 128.84, 128.60, 126.30, 90.05, 66.76, 51.65, 44.53, 39.43, 35.73, 33.98, 31.63, 29.68, 26.78, 22.67, 14.14. HRMS [C<sub>24</sub>H<sub>35</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 426.2864 calculated, 426.2857 found.</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-Isobutyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>15</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119e</b> (30 mg, 0.10 mmol, 1 equiv), DiPEA (70 μL, 0.40 mmol, 4 equiv), and <i>N</i>-methylphenethylamine HBr salt (32 mg, 0.15 mmol, 1.5 equiv). Total heating time: 3 days at 120 °C. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (29 mg, 73 μmol, 73%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (br s, 1H), 7.33–7.25 (m, 2H), 7.25–7.16 (m, 3H), 6.72 (s, 1H), 3.88–3.73 (m, 6H), 3.73–3.62 (m, 4H), 3.26 (t, <i>J</i> = 6.5 Hz, 2H), 3.10 (s, 3H), 2.96–2.85 (m, 2H), 1.97–1.82 (m, 1H), 0.98 (d, <i>J</i> = 6.7 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.77, 164.01, 160.84, 156.84, 139.94, 128.84, 128.58, 126.29, 90.07, 66.75, 51.63, 46.65, 44.52, 35.72, 33.98, 28.80, 20.28. HRMS [C<sub>22</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 398.2551 calculated, 398.2552 found.</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-Neopentyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>16</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119f</b> (31 mg, 0.10 mmol, 1 equiv), DiPEA (70 μL, 0.40 mmol, 4 equiv), and <i>N-</i>methylphenethylamine HBr salt (32 mg, 0.15 mmol, 1.5 equiv). Total heating time: 3 days at 120 °C. Column chromatography (20% → 50% EtOAc/pentane) afforded the product (30 mg, 73 μmol, 73%). TLC: <i>R</i><sub>f</sub> = 0.6 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.14 (br s, 1H), 7.32–7.25 (m, 2H), 7.24–7.17 (m, 3H), 6.73 (s, 1H), 3.85–3.78 (m, 2H), 3.78–3.73 (m, 4H), 3.71–3.61 (m, 4H), 3.23 (d, <i>J</i> = 6.6 Hz, 2H), 3.10 (s, 3H), 2.95–2.86 (m, 2H), 0.97 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.82, 164.03, 160.82, 156.82, 139.89, 128.85, 128.59, 126.30, 90.15, 66.76, 51.62, 50.57, 44.53, 35.72, 33.99, 32.28, 27.38. HRMS [C<sub>23</sub>H<sub>33</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 412.2707 calculated, 412.2710 found.</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-(Methyl(phenethyl)amino)-6-morpholino-<i>N</i>-(prop-2-yn-1-yl)pyrimidine-4-carboxamide (<b>17</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119g</b> (42 mg, 0.15 mmol, 1 equiv), DiPEA (105 μL, 0.60 mmol, 4 equiv), and <i>N</i>-methylphenethylamine HBr salt (49 mg, 0.225 mmol, 1.5 equiv). Total heating time: 45 h at 120 °C. Column chromatography (30% → 50% EtOAc/pentane) afforded the product (43 mg, 0.11 mmol, 76%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (br s, 1H), 7.33–7.25 (m, 2H), 7.25–7.17 (m, 3H), 6.69 (s, 1H), 4.22 (dd, <i>J</i> = 5.6, 2.5 Hz, 2H), 3.86–3.72 (m, 6H), 3.72–3.60 (m, 4H), 3.11 (s, 3H), 2.96–2.85 (m, 2H), 2.29 (t, <i>J</i> = 2.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.60, 163.89, 160.88, 156.03, 139.91, 128.90, 128.60, 126.30, 90.15, 79.57, 71.67, 66.72, 51.70, 44.50, 35.75, 33.98, 29.21. HRMS [C<sub>21</sub>H<sub>25</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 380.2081 calculated, 380.2089 found.</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-(2,2,2-Trifluoroethyl)-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>18</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_3" aria-label="General Procedure B">General Procedure B</a> using carboxylic acid <b>19</b> (25 mg, 73 μmol, 1 equiv), DiPEA (51 μL, 0.29 mmol, 4 equiv), PyBOP (57 mg, 0.11 mmol, 1.5 equiv), and 2,2,2-trifluoroethylamine HCl salt (12 mg, 88 μmol, 1.2 equiv). Column chromatography (30% → 40% EtOAc/pentane) afforded the product (17 mg, 40 μmol, 55%). TLC: <i>R</i><sub>f</sub> = 0.8 (50% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.20 (br s, 1H), 7.32–7.26 (m, 2H), 7.23–7.17 (m, 3H), 6.70 (s, 1H), 4.12–4.02 (m, 2H), 3.83–3.78 (m, 2H), 3.78–3.74 (m, 4H), 3.71–3.59 (m, 4H), 3.11 (s, 3H), 2.92–2.86 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 165.31, 163.88, 160.92, 155.42, 139.85, 128.86, 128.65, 126.37, 124.31 (q, <i>J</i> = 278.4 Hz), 90.45, 66.75, 51.70, 44.58, 40.89 (q, <i>J</i> = 34.8 Hz), 35.77, 34.04. HRMS [C<sub>20</sub>H<sub>24</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 424.1955 calculated, 424.1958 found.</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 2-(Methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxylic acid (<b>19</b>)</h4><div id="sec4_2_27_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> Ester Hydrolysis</h5><div class="NLM_p last">A round-bottom flask was charged with methyl ester <b>127</b> (680 mg, 2.64 mmol, 1 equiv) in 12.5 mL of THF/MeOH (4:1). A 1.5 M aqueous NaOH solution (1.76 mL, 2.64 mmol, 1 equiv) was added together with 0.75 mL of H<sub>2</sub>O. The reaction was stirred overnight at rt after which the solvents were evaporated yielding the product as the Na<sup>+</sup> salt (<b>128</b>), which was used without further purification (779 mg, 2.64 mmol, quant.).</div></div><div id="sec4_2_27_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> S<sub>N</sub>Ar Reaction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>128</b> (244 mg, 1.0 mmol, 1 equiv), DiPEA (0.52 mL, 3.0 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (189 μL, 1.3 mmol, 1.3 equiv). Total heating time: 6 days at 120 °C. Column chromatography (2.5% → 15% MeOH/DCM) afforded the product (175 mg, 0.51 mmol, 51%). TLC: <i>R</i><sub>f</sub> = 0.5 (100% EtOAc with 3 drops of AcOH). <sup>1</sup>H NMR (400 MHz, MeOD + CDCl<sub>3</sub>) δ 7.35–7.26 (m, 2H), 7.26–7.15 (m, 3H), 6.87 (s, 1H), 3.91 (t, <i>J</i> = 7.0 Hz, 2H), 3.81 (br s, 8H), 3.17 (s, 3H), 2.97 (t, <i>J</i> = 7.0 Hz, 2H). <sup>13</sup>C NMR (101 MHz, MeOD + CDCl<sub>3</sub>) δ 161.70, 152.17, 148.40, 137.57, 133.96, 128.50, 128.38, 126.57, 93.89, 66.02, 51.86, 45.10, 33.13. HRMS [C<sub>18</sub>H<sub>22</sub>N<sub>4</sub>O<sub>3</sub> + H]<sup>+</sup>: 343.1765 calculated, 343.1772 found.</div></div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (2-(Methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carbonyl)glycine (<b>20</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>135</b> (150 mg, 0.50 mmol, 1 equiv), DiPEA (0.43 mL, 2.5 mmol, 5 equiv), and <i>N</i>-methylphenethylamine HBr salt (162 mg, 0.76 mmol, 1.5 equiv). Total heating time: 6 h at 160 °C with μW irradiation. Purification by HPLC (C18 reverse phase, 10% to 70% CH<sub>3</sub>CN/H<sub>2</sub>O + 50 mM NH<sub>4</sub>HCO<sub>3</sub>) afforded the product (40 mg, 0.10 mmol, 20%). TLC: <i>R</i><sub>f</sub> = 0.3 (5% MeOH/DCM). <sup>1</sup>H NMR (500 MHz, MeOD + CDCl<sub>3</sub>) δ 7.32–7.14 (m, 5H), 6.65 (s, 1H), 4.01 (s, 2H), 3.83 (t, <i>J</i> = 7.3 Hz, 2H), 3.80–3.75 (m, 4H), 3.73–3.60 (m, 4H), 3.11 (s, 3H), 2.91 (t, <i>J</i> = 7.4 Hz, 2H). <sup>13</sup>C NMR (126 MHz, MeOD + CDCl<sub>3</sub>) δ 173.63, 164.96, 163.55, 160.59, 155.89, 139.54, 128.50, 128.00, 125.67, 89.33, 66.26, 51.10, 49.06, 43.96, 42.64, 39.58, 35.05, 33.47, 20.71. HRMS [C<sub>20</sub>H<sub>25</sub>N<sub>5</sub>O<sub>4</sub> + H]<sup>+</sup>: 400.1979 calculated, 400.1984 found.</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Methyl (2-(Methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carbonyl) Glycinate (<b>21</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with carboxylic acid <b>20</b> (28 mg, 70 μmol, 1 equiv) in dry DCM (1.5 mL). This was followed by addition of HOBt (15 mg, 0.11 mmol, 1.5 equiv) and EDC·HCl (20 mg, 0.11 mmol, 1.5 equiv). The reaction was stirred for 1 h at rt, and after which, MeOH (11 μL, 0.28 mmol, 4 equiv) was added and then stirred overnight at rt. The reaction was diluted with EtOAc (25 mL), washed with sat. aq. NaHCO<sub>3</sub> (2× 25 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified using silica gel column chromatography (60% → 80% EtOAc/pentane) affording the product (18 mg, 44 μmol, 62%). TLC: <i>R</i><sub>f</sub> = 0.3 (70% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.40 (br s, 1H), 7.39–7.13 (m, 5H), 6.69 (s, 1H), 4.22 (d, <i>J</i> = 5.5 Hz, 2H), 3.86–3.71 (m, 9H), 3.71–3.59 (m, 4H), 3.11 (s, 3H), 2.96–2.85 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.25, 165.14, 163.91, 160.92, 155.99, 139.96, 128.98, 128.58, 126.27, 90.13, 66.75, 52.52, 51.68, 44.51, 41.37, 35.79, 34.01. HRMS [C<sub>21</sub>H<sub>27</sub>N<sub>5</sub>O<sub>4</sub> + H]<sup>+</sup>: 414.2136 calculated, 414.2144 found.</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-(2-(Methylamino)-2-oxoethyl)-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>22</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_3" aria-label="General Procedure B">General Procedure B</a> using carboxylic acid <b>20</b> (12 mg, 30 μmol, 1 equiv), DiPEA (21 μL, 120 μmol, 4 equiv), PyBOP (19 mg, 45 μmol, 1.5 equiv), and methylamine HCl salt (3 mg, 36 μmol, 1.2 equiv). Column chromatography (2.5% → 10% MeOH/DCM) afforded the product (6 mg, 15 μmol, 48%). TLC: <i>R</i><sub>f</sub> = 0.4 (5% MeOH/DCM). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.43 (br s, 1H), 7.32–7.26 (m, 2H), 7.23–7.16 (m, 3H), 6.69 (br s, 1H), 6.22 (br s, 1H), 4.08 (d, <i>J</i> = 6.1 Hz, 2H), 3.86–3.79 (m, 2H), 3.79–3.73 (m, 4H), 3.67 (br s, 4H), 3.10 (s, 3H), 2.93–2.86 (m, 2H), 2.84 (d, <i>J</i> = 4.9 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 169.58, 165.74, 163.84, 160.90, 155.66, 139.91, 128.93, 128.65, 126.36, 90.21, 66.75, 51.65, 44.60, 43.84, 35.86, 34.06, 26.41. HRMS [C<sub>21</sub>H<sub>28</sub>N<sub>6</sub>O<sub>3</sub> + H]<sup>+</sup>: 413.2296 calculated, 413.2294 found.</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-(2-Hydroxyethyl)-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>23</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_3" aria-label="General Procedure B">General Procedure B</a> using carboxylic acid <b>19</b> (39 mg, 0.11 mmol, 1 equiv), DiPEA (60 μL, 0.34 mmol, 3 equiv), PyBOP (89 mg, 0.17 mmol, 1.5 equiv), and ethanolamine (34 μL, 0.57 mmol, 5 equiv). Column chromatography (70% → 100% EtOAc/pentane to 5% MeOH/EtOAc) afforded the product (25 mg, 65 μmol, 59%). TLC: <i>R</i><sub>f</sub> = 0.3 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.29 (br s, 1H), 7.33–7.27 (m, 2H), 7.24–7.16 (m, 3H), 6.70 (s, 1H), 3.87–3.78 (m, 4H), 3.78–3.74 (m, 4H), 3.70–3.62 (m, 4H), 3.62–3.55 (m, 2H), 3.10 (s, 3H), 2.98–2.80 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.19, 163.93, 160.87, 156.29, 139.96, 128.89, 128.61, 126.32, 90.14, 66.74, 62.71, 51.61, 44.51, 42.71, 35.77, 33.98. HRMS [C<sub>20</sub>H<sub>27</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 386.2187 calculated, 386.2191 found.</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-(2-Methoxyethyl)-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>24</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with alcohol <b>23</b> (17 mg, 44 μmol, 1 equiv) in dry DMF (1 mL) and cooled to 0 °C. NaOtBu (2 M in THF, 33 μL, 66 μmol, 1.5 equiv) and methyl iodide (3.1 μL, 49 μmol, 1.1 equiv) were added. The reaction was allowed to warm to rt while stirring overnight. EtOAc (25 mL) was added followed by washing with H<sub>2</sub>O (1× 25 mL) and brine (2× 25 mL), drying (Na<sub>2</sub>SO<sub>4</sub>), filtering, and concentrating under reduced pressure. The residue was purified by silica gel column chromatography (70 → 80% EtOAc/pentane), affording the product (5 mg, 13 μmol, 28%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.26 (br s, 1H), 7.33–7.27 (m, 2H), 7.25–7.16 (m, 3H), 6.71 (s, 1H), 3.85–3.71 (m, 6H), 3.72–3.59 (m, 6H), 3.59–3.51 (m, 2H), 3.38 (s, 3H), 3.11 (s, 3H), 2.95–2.83 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.96, 163.98, 160.89, 156.67, 139.95, 128.93, 128.62, 126.29, 90.08, 71.38, 66.78, 59.02, 51.72, 44.53, 39.25, 35.77, 33.96. HRMS [C<sub>21</sub>H<sub>29</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 400.2343 calculated, 400.2345 found.</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-(Cyanomethyl)-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>25</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_3" aria-label="General Procedure B">General Procedure B</a> using carboxylic acid <b>19</b> (21 mg, 61 μmol, 1 equiv), DiPEA (53 μL, 0.31 mmol, 5 equiv), PyBOP (48 mg, 92 μmol, 1.5 equiv), and aminoacetonitrile bisulfate (11 mg, 73 μmol, 1.2 equiv). Column chromatography (50% - > 70% EtOAc/pentane) afforded the product (10 mg, 26 μmol, 43%). TLC: <i>R</i><sub>f</sub> = 0.6 (60% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.12 (br s, 1H), 7.35–7.27 (m, 2H), 7.25–7.14 (m, 3H), 6.67 (s, 1H), 4.34 (d, <i>J</i> = 6.1 Hz, 2H), 3.86–3.79 (m, 2H), 3.79–3.74 (m, 4H), 3.66 (br s, 4H), 3.10 (s, 3H), 2.95–2.84 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.12, 163.72, 160.92, 154.96, 139.94, 128.88, 128.68, 126.41, 115.95, 90.37, 66.71, 51.58, 44.50, 35.86, 34.05, 27.52. HRMS [C<sub>20</sub>H<sub>24</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 381.2034 calculated, 381.2042 found.</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-(Thiazol-2-ylmethyl)-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>26</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_3" aria-label="General Procedure B">General Procedure B</a> using carboxylic acid <b>19</b> (27 mg, 79 μmol, 1 equiv), DiPEA (82 μL, 0.47 mmol, 6 equiv), PyBOP (62 mg, 0.12 mmol, 1.5 equiv), and 2-aminomethylthiazole double HCl salt (19 mg, 0.10 mmol, 1.3 equiv). Purification by preparative HPLC (C18 reverse phase, 34% to 37% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA) afforded the product (11 mg, 25 μmol, 32%). TLC: <i>R</i><sub>f</sub> = 0.3 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.62 (br s, 1H), 7.74 (d, <i>J</i> = 3.3 Hz, 1H), 7.30 (d, <i>J</i> = 3.3 Hz, 1H), 7.30–7.21 (m, 2H), 7.23–7.15 (m, 3H), 6.75 (s, 1H), 4.95 (d, <i>J</i> = 6.2 Hz, 2H), 3.86–3.72 (m, 6H), 3.72–3.60 (m, 4H), 3.10 (s, 3H), 2.94–2.82 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 167.49, 165.05, 163.87, 160.86, 155.95, 142.63, 139.91, 128.91, 128.59, 126.29, 119.73, 90.33, 66.75, 51.65, 44.53, 41.06, 35.81, 33.99. HRMS [C<sub>22</sub>H<sub>26</sub>N<sub>6</sub>O<sub>2</sub>S + H]<sup>+</sup>: 439.1911 calculated, 439.1913 found.</div></div><div id="sec4_2_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>N-</i>Benzyl-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>27</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119i</b> (67 mg, 0.20 mmol, 1 equiv), DiPEA (139 μL, 0.80 mmol, 4 equiv), and <i>N-</i>methylphenethylamine HBr salt (65 mg, 0.30 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (75 mg, 0.17 mmol, 87%). TLC: <i>R</i><sub>f</sub> = 0.8 (60% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.25 (br s, 1H), 7.39–7.33 (m, 4H), 7.33–7.26 (m, 1H), 7.24–7.14 (m, 3H), 7.14–7.04 (m, 2H), 6.76 (s, 1H), 4.63 (d, <i>J</i> = 6.1 Hz, 2H), 3.80–3.71 (m, 6H), 3.71–3.62 (m, 4H), 3.08 (s, 3H), 2.89–2.80 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.83, 163.90, 160.82, 156.52, 139.79, 138.35, 128.80, 128.77, 128.54, 127.69, 127.51, 126.20, 90.17, 66.70, 51.63, 44.46, 43.37, 35.68, 33.88. HRMS [C<sub>25</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 432.2394 calculated, 432.2390 found.</div></div><div id="sec4_2_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>N</i>-([1,1′-Biphenyl]-4-ylmethyl)-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>28</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119j</b> (41 mg, 0.10 mmol, 1 equiv), DiPEA (70 μL, 0.40 mmol, 4 equiv), and <i>N-</i>methylphenethylamine HBr salt (32 mg, 0.15 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (40%→ 60% EtOAc/pentane) afforded the product (40 mg, 79 μmol, 79%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.29 (br s, 1H), 7.57 (d, <i>J</i> = 7.9 Hz, 4H), 7.47–7.39 (m, 4H), 7.38–7.30 (m, 1H), 7.23–7.07 (m, 5H), 6.77 (s, 1H), 4.68 (d, <i>J</i> = 6.1 Hz, 2H), 3.81–3.72 (m, 6H), 3.71–3.61 (m, 4H), 3.08 (s, 3H), 2.90–2.81 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.92, 163.94, 160.86, 156.54, 140.84, 140.51, 139.82, 137.43, 128.88, 128.79, 128.56, 128.17, 127.56, 127.42, 127.17, 126.24, 90.22, 66.73, 51.65, 44.50, 43.12, 35.73, 33.91. HRMS [C<sub>31</sub>H<sub>33</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 508.2707 calculated, 508.2704 found.</div></div><div id="sec4_2_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 4-(5-Cyclopropyl-1<i>H</i>-imidazol-2-yl)-<i>N</i>-methyl-6-morpholino-<i>N</i>-phenethylpyrimidin-2-amine (<b>29</b>)</h4><div id="sec4_2_37_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> Acyloxymethylketone Synthesis</h5><div class="NLM_p last">A round-bottom flask was charged with carboxylic acid <b>19</b> (53 mg, 0.15 mmol, 1 equiv) in dry DMF (1 mL). Cs<sub>2</sub>CO<sub>3</sub> (91 mg, 0.28 mmol, 1.8 equiv) and 2-bromocyclopropylethanone (16 μL, 0.16 mmol, 1.05 equiv) were added, and the mixture was stirred for 1.5 h. The reaction was diluted with EtOAc (25 mL), and the mixture was washed with H<sub>2</sub>O (1× 25 mL) and brine (2× 25 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (30% → 50% EtOAc/pentane), affording the acyloxymethylketone <b>136</b> (34 mg, 80 μmol, 53%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.36–7.12 (m, 5H), 6.63 (s, 1H), 5.04 (s, 2H), 3.87–3.78 (m, 2H), 3.78–3.72 (m, 4H), 3.71–3.53 (m, 4H), 3.14 (s, 3H), 2.97–2.80 (m, 2H), 2.16–2.00 (m, 1H), 1.22–1.08 (m, 2H), 1.04–0.88 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 203.64, 165.50, 163.61, 161.85, 154.67, 139.97, 128.93, 128.50, 126.17, 93.17, 69.31, 66.69, 51.54, 44.41, 35.64, 33.77, 17.38, 11.70.</div></div><div id="sec4_2_37_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> Imidazole Synthesis</h5><div class="NLM_p last">A microwave vial was charged with acyloxymethylketone <b>136</b> (34 mg, 80 μmol, 1 equiv) and NH<sub>4</sub>OAc (31 mg, 0.40 mmol, 5 equiv) in xylene (0.7 mL). The vial was capped and stirred at 140 °C for 2 h. Purification by preparative HPLC (C18 reverse phase, 35% to 40% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA) afforded the product (2 mg, 5 μmol, 6%). TLC: <i>R</i><sub>f</sub> = 0.7 (40% EtOAc/pentane). <sup>1</sup>H NMR (850 MHz, CDCl<sub>3</sub>) δ 9.89 (br s, 1H), 7.33–7.27 (m, 2H), 7.24–7.16 (m, 3H), 6.98–6.49 (m, 2H), 3.84 (s, 2H), 3.80–3.73 (m, 4H), 3.68 (br s, 4H), 3.14 (s, 3H), 2.97–2.83 (m, 2H), 1.91 (br s, 1H), 0.92 (br s, 2H), 0.74 (br s, 2H). <sup>13</sup>C NMR (214 MHz, CDCl<sub>3</sub>) δ 163.48, 161.12, 154.40, 145.32, 140.18, 128.91, 128.66, 126.35, 111.74, 87.19, 66.84, 51.62, 44.66, 35.84, 34.08, 9.44, 7.18, 6.01. HRMS [C<sub>23</sub>H<sub>28</sub>N<sub>5</sub>O + H]<sup>+</sup>: 405.2397 calculated, 405.2403 found.</div></div></div><div id="sec4_2_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 2-Cyclopropyl-<i>N</i>-(2-(methyl(phenethyl)amino)-6-morpholinopyrimidin-4-yl)acetamide (<b>30</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with 2-cyclopropylacetic acid (10.5 μL, 0.12 mmol, 1.2 equiv) and dry DCM-<i>d</i><sub>2</sub> (0.5 mL). Oxalyl chloride (21 μL, 0.24 mmol, 2.4 equiv) was added, and the solution was stirred for 3 h. When the reaction was completed as judged by NMR, it was concentrated under reduced pressure followed by the addition of aminopyrimidine <b>139</b> (31 mg, 0.1 mmol, 1 equiv) in dry DCM (0.5 mL). The reaction was stirred for 3 h and then concentrated under reduced pressure. The residue was purified by flash column chromatography (0% → 3% MeOH/DCM) affording the product (14 mg, 34 μmol, 34%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.85 (br s, 1H), 7.33–7.27 (m, 2H), 7.24–7.17 (m, 3H), 6.86 (s, 1H), 3.79–3.70 (m, 6H), 3.64–3.56 (m, 4H), 3.04 (s, 3H), 2.91–2.84 (m, 2H), 2.29 (d, <i>J</i> = 7.1 Hz, 2H), 1.15–1.03 (m, 1H), 0.72–0.65 (m, 2H), 0.30–0.25 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.84, 164.73, 160.88, 157.66, 140.19, 128.92, 128.54, 126.20, 80.09, 66.85, 51.30, 44.70, 42.92, 35.44, 33.92, 7.16, 4.75. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 396.2394 calculated, 396.2390 found.</div></div><div id="sec4_2_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 2-Chloro-<i>N-</i>(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>31</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (1.7 g, 7.07 mmol, 1 equiv), DiPEA (1.85 mL, 10.6 mmol, 1.5 equiv), and morpholine (0.64 mL, 7.42 mmol, 1.05 equiv). Column chromatography (30% → 60% EtOAc/pentane) afforded the product (1.7 g, 6.3 mmol, 89%). TLC: <i>R</i><sub>f</sub> = 0.5 in 40% EtOAc/pentane. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (t, <i>J</i> = 5.9 Hz, 1H), 7.28 (s, 1H), 3.87–3.63 (m, 8H), 3.33–3.25 (m, 2H), 1.12–1.00 (m, 1H), 0.61–0.52 (m, 2H), 0.32–0.24 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.83, 162.21, 159.84, 157.90, 99.36, 66.35, 44.45, 10.65, 3.64. Regioselectivity was confirmed by <sup>1</sup>H-NOESY NMR analysis. HRMS [C<sub>13</sub>H<sub>17</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 297.1113 calculated, 297.1116 found. Regioisomer 6-chloro-<i>N-</i>(cyclopropylmethyl)-2-morpholinopyrimidine-4-carboxamide (<b>129</b>) was also obtained (99 mg, 0.33 mmol, 5%). <i>R</i><sub>f</sub> = 0.6 in 40% EtOAc/pentane. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.80–7.65 (m, 1H), 7.32 (s, 1H), 3.90–3.72 (m, 8H), 3.30 (t, <i>J</i> = 6.5 Hz, 2H), 1.14–0.98 (m, 1H), 0.69–0.46 (m, 2H), 0.39–0.17 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.48, 162.31, 160.68, 158.86, 107.51, 66.64, 44.42, 44.30, 10.80, 3.55.</div></div><div id="sec4_2_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-(Cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>32</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with 2-chloropyrimidine <b>31</b> (24 mg, 80 μmol, 1 equiv), NaHCO<sub>3</sub> (8 mg, 0.10 mmol, 1.2 equiv), and MeOH (0.5 mL). The solution was purged with N<sub>2</sub> followed by addition of Pd/C (10% w/w, 40 mg, 50 μmol, 5 mol %), purged again with N<sub>2</sub> and then stirred overnight under a H<sub>2</sub> atmosphere (balloon). The mixture was filtered through a plug of Celite, which was washed with MeOH and the filtrate was concentrated under reduced pressure. The residue was purified by silica gel column chromatography (40% → 70% EtOAc/pentane) to afford the product (20 mg, 76 μmol, 95%). TLC: <i>R</i><sub>f</sub> = 0.2 (40% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.57 (s, 1H), 8.09 (t, <i>J</i> = 5.7 Hz, 1H), 7.36 (s, 1H), 3.80–3.77 (m, 4H), 3.76–3.67 (m, 4H), 3.33–3.28 (m, 2H), 1.10–1.02 (m, 1H), 0.59–0.54 (m, 2H), 0.32–0.26 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 163.56, 163.02, 157.44, 155.78, 100.60, 66.59, 44.44, 44.37, 10.79, 3.65. HRMS [C<sub>13</sub>H<sub>18</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 263.1503 calculated, 263.1502 found.</div></div><div id="sec4_2_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-(Cyclopropylmethyl)-6-morpholino-2-(phenethylamino)pyrimidine-4-carboxamide (<b>33</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (59 mg, 0.20 mmol, 1 equiv), 2-phenethylamine (30 μL, 0.24 mmol, 1.2 equiv) and DiPEA (70 μL, 0.40 mmol, 2 equiv). Total heating time: 8 h at 160 °C with μW irradiation. Column chromatography (2% → 5% MeOH/DCM) afforded the product (40 mg, 0.10 mmol, 50%). TLC: <i>R</i><sub>f</sub> = 0.4 (4% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (br s, 1H), 7.39–7.28 (m, 2H), 7.28–7.15 (m, 3H), 6.78 (s, 1H), 4.96 (br s, 1H), 3.84–3.71 (m, 4H), 3.71–3.49 (m, 6H), 3.27 (t, <i>J</i> = 6.4 Hz, 2H), 2.92 (t, <i>J</i> = 7.2 Hz, 2H), 1.13–0.95 (m, 1H), 0.64–0.41 (m, 2H), 0.37–0.18 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.46, 163.98, 161.46, 156.84, 139.40, 128.89, 128.71, 126.53, 91.72, 66.68, 44.50, 43.00, 36.13, 10.79, 3.63. HRMS [C<sub>21</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 382.2238 calculated, 382.2241 found.</div></div><div id="sec4_2_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 2-(Benzyl(methyl)amino)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>34</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (59 mg, 0.20 mmol, 1 equiv), <i>N</i>-methylbenzylamine (38 μL, 0.30 mmol, 1.5 equiv), and DiPEA (140 μL, 0.80 mmol, 4 equiv). Total heating time: 8 h at 160 °C with μW irradiation. Column chromatography (2% → 5% MeOH/DCM) afforded the product (40 mg, 0.10 mmol, 50%). TLC: <i>R</i><sub>f</sub> = 0.5 (4% MeOH/DCM). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.96 (br s, 1H), 7.34–7.28 (m, 2H), 7.28–7.21 (m, 3H), 6.75 (s, 1H), 4.85 (s, 2H), 3.82–3.69 (m, 4H), 3.69–3.54 (m, 4H), 3.26 (t, <i>J</i> = 6.5 Hz, 2H), 3.15 (s, 3H), 1.10–0.95 (m, 1H), 0.56–0.44 (m, 2H), 0.29–0.17 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.58, 163.97, 161.35, 156.85, 139.11, 128.56, 127.34, 127.03, 90.41, 66.68, 52.76, 44.47, 44.00, 35.18, 10.85, 3.43. HRMS [C<sub>21</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 382.2238 calculated, 382.2241 found.</div></div><div id="sec4_2_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>N</i>-(Cyclopropylmethyl)-2-(benzylamino)-6-morpholinopyrimidine-4-carboxamide (<b>35</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (59 mg, 0.20 mmol, 1 equiv), DiPEA (0.14 mL, 0.80 mmol, 4 equiv), and benzylamine (33 μL, 0.30 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (2% → 4% MeOH/DCM) afforded the product (19 mg, 50 μmol, 25%). TLC: <i>R</i><sub>f</sub> = 0.5 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (br s, 1H), 7.41–7.19 (m, 5H), 6.78 (s, 1H), 5.24 (br s, 1H), 4.60 (d, <i>J</i> = 5.9 Hz, 2H), 3.76–3.69 (m, 4H), 3.69–3.60 (m, 4H), 3.25 (t, <i>J</i> = 7.1, 5.8 Hz, 2H), 1.09–0.95 (m, 1H), 0.57–0.48 (m, 2H), 0.29–0.22 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 164.20, 164.03, 161.41, 156.91, 139.73, 128.65, 127.53, 127.27, 91.86, 66.67, 45.79, 44.48, 44.25, 10.78, 3.59. HRMS [C<sub>20</sub>H<sub>25</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 368.2081 calculated, 368.2081 found.</div></div><div id="sec4_2_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>N-</i>(Cyclopropylmethyl)-2-(methyl(3-phenylpropyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>36</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (39 mg, 0.13 mmol, 1 equiv), amine <b>123b</b> (30 mg, 0.20 mmol, 1.5 equiv), and DiPEA (87 μL, 0.53 mmol, 4 equiv). Total heating time: 48 h at 120 °C. Column chromatography (30% → 60% EtOAc/pentane) afforded the product (38 mg, 93 μmol, 71%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98 (br s, 1H), 7.32–7.22 (m, 2H), 7.22–7.14 (m, 3H), 6.69 (s, 1H), 3.76–3.68 (m, 4H), 3.65–3.59 (m, 2H), 3.59–3.51 (m, 4H), 3.28 (t, <i>J</i> = 7.0, 5.9 Hz, 2H), 3.14 (s, 3H), 2.66 (t, <i>J</i> = 7.5 Hz, 2H), 1.95 (p, <i>J</i> = 9.0, 7.5 Hz, 2H), 1.10–0.98 (m, 1H), 0.57–0.49 (m, 2H), 0.30–0.24 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.69, 163.93, 161.04, 156.75, 142.00, 128.49, 128.43, 125.93, 89.96, 66.73, 48.99, 44.42, 43.97, 35.31, 33.52, 28.92, 10.93, 3.43. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 410.2551 calculated, 410.2548 found.</div></div><div id="sec4_2_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(4-phenylbutyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>37</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>123c</b> (23 mg, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 24 h at 120 °C. Column chromatography (30% → 60% EtOAc/pentane) afforded the product (40 mg, 94 μmol, 94%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (t, <i>J</i> = 5.8 Hz, 1H), 7.31–7.22 (m, 2H), 7.21–7.12 (m, 3H), 6.70 (s, 1H), 3.77–3.70 (m, 4H), 3.67–3.58 (m, 6H), 3.31–3.24 (m, 2H), 3.11 (s, 3H), 2.73–2.59 (m, 2H), 1.76–1.56 (m, <i>J</i> = 3.5, 2.9 Hz, 4H), 1.11–0.95 (m, 1H), 0.56–0.44 (m, 2H), 0.28–0.18 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.70, 163.96, 161.10, 156.74, 142.44, 128.42, 128.40, 125.87, 89.89, 66.72, 49.12, 44.47, 44.00, 35.88, 35.32, 29.03, 27.20, 10.86, 3.39. HRMS [C<sub>24</sub>H<sub>33</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 424.2707 calculated, 424.2706 found.</div></div><div id="sec4_2_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 2-((4-Chlorophenethyl)(methyl)amino)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>38</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>123d</b> (34 mg, 0.20 mmol, 2 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 25 h at 120 °C. Column chromatography (30% → 70% EtOAc/pentane) afforded the product (10 mg, 30 μmol, 30%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98 (br s, 1H), 7.28–7.23 (m, 2H), 7.17–7.09 (m, 2H), 6.73 (s, 1H), 3.82–3.72 (m, 6H), 3.65 (t, <i>J</i> = 4.8 Hz, 4H), 3.35–3.24 (m, 2H), 3.09 (s, 3H), 2.94–2.82 (m, 2H), 1.12–1.01 (m, 1H), 0.60–0.49 (m, 2H), 0.36–0.23 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.64, 163.98, 156.81, 138.45, 132.12, 130.22, 128.72, 90.28, 66.76, 51.50, 44.53, 44.15, 35.86, 33.39, 10.91, 3.51. HRMS [C<sub>22</sub>H<sub>28</sub>ClN<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 430.2004 calculated, 430.2004 found.</div></div><div id="sec4_2_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 2-((3-Chlorophenethyl)(methyl)amino)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>39</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (20 mg, 70 μmol, 1 equiv), amine <b>123e</b> (20 mg, 0.12 mmol, 1.5 equiv) and DiPEA (49 μL, 0.28 mmol, 4 equiv). Total heating time: 5 days at 120 °C. Column chromatography (30% → 50% EtOAc/pentane) afforded the product (26 mg, 62 μmol, 89%). TLC: <i>R</i><sub>f</sub> = 0.6 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.00 (t, <i>J</i> = 5.9 Hz, 1H), 7.25–7.14 (m, 3H), 7.14–7.02 (m, 1H), 6.73 (s, 1H), 3.86–3.72 (m, 6H), 3.66 (t, <i>J</i> = 4.9 Hz, 4H), 3.30 (dd, <i>J</i> = 7.1, 5.8 Hz, 2H), 3.12 (s, 3H), 2.93–2.83 (m, 2H), 1.12–1.01 (m, 1H), 0.60–0.50 (m, 2H), 0.33–0.24 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.62, 163.98, 160.86, 156.78, 141.98, 134.32, 129.82, 128.99, 127.06, 126.50, 90.27, 66.74, 51.34, 44.51, 44.12, 35.72, 33.68, 10.90, 3.48. HRMS [C<sub>22</sub>H<sub>28</sub>ClN<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 430.2004 calculated, 430.2004 found.</div></div><div id="sec4_2_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 2-((2-Chlorophenethyl)(methyl)amino)-<i>N-</i>(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>40</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>123f</b> (28 mg, 0.17 mmol, 1.7 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 3 days at 120 °C. Purification by HPLC (C18 reverse phase, 35% → 45% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT 10.8 min) afforded the product (31 mg, 70 μmol, 70%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.06 (br s, 1H), 7.37–7.32 (m, 1H), 7.21–7.12 (m, 3H), 6.70 (s, 1H), 3.89–3.83 (m, 2H), 3.78–3.74 (m, 4H), 3.69–3.62 (m, 4H), 3.29 (dd, <i>J</i> = 7.0, 5.9 Hz, 2H), 3.14 (s, 3H), 3.06–3.02 (m, 2H), 1.13–1.01 (m, 1H), 0.59–0.51 (m, 2H), 0.31–0.27 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.70, 163.95, 160.97, 156.79, 137.46, 134.14, 131.17, 129.60, 127.93, 127.01, 90.11, 66.78, 49.36, 44.51, 44.14, 35.52, 31.93, 10.97, 3.59. HRMS [C<sub>22</sub>H<sub>28</sub>ClN<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 430.2004 calculated, 430.2003 found.</div></div><div id="sec4_2_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(4-methylphenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>41</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>123g</b> (23 mg, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 3 days at 120 °C. Column chromatography (30% → 50% EtOAc/pentane) afforded the product (16 mg, 40 μmol, 40%). TLC: <i>R</i><sub>f</sub> = 0.8 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (t, <i>J</i> = 5.8 Hz, 1H), 7.11 (s, 4H), 6.72 (s, 1H), 3.82–3.72 (m, 6H), 3.66 (t, <i>J</i> = 4.8 Hz, 4H), 3.30 (dd, <i>J</i> = 7.1, 5.8 Hz, 2H), 3.13 (s, 3H), 2.90–2.82 (m, 2H), 2.33 (s, 3H), 1.14–0.99 (m, 1H), 0.61–0.49 (m, 2H), 0.33–0.25 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.72, 164.01, 160.89, 156.79, 136.81, 135.80, 129.28, 128.72, 90.06, 66.76, 51.81, 44.52, 44.12, 35.69, 33.46, 21.16, 10.90, 3.49. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 410.2551 calculated, 410.2549 found.</div></div><div id="sec4_2_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(2-methylphenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>42</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>123h</b> (23 mg, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 4 days at 120 °C. Column chromatography (30% → 60% EtOAc/pentane) afforded the product (39 mg, 95 μmol, 95%). TLC: <i>R</i><sub>f</sub> = 0.4 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (t, <i>J</i> = 5.9 Hz, 1H), 7.23–7.06 (m, 4H), 6.73 (s, 1H), 3.85–3.71 (m, 6H), 3.66 (t, <i>J</i> = 4.8 Hz, 4H), 3.29 (dd, <i>J</i> = 7.1, 5.9 Hz, 2H), 3.15 (s, 3H), 2.97–2.83 (m, 2H), 2.39 (s, 3H), 1.15–0.98 (m, 1H), 0.62–0.47 (m, 2H), 0.34–0.22 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.69, 163.99, 160.91, 156.84, 137.97, 136.13, 130.38, 129.54, 126.51, 126.22, 90.12, 66.74, 50.23, 44.50, 44.12, 35.58, 31.31, 19.46, 10.91, 3.52. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 410.2551 calculated, 410.2546 found.</div></div><div id="sec4_2_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>N</i>-(Cyclopropylmethyl)-2-((4-methoxyphenethyl)(methyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>43</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>123i</b> (36 mg, 0.20 mmol, 2 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 7 days at 120 °C. Column chromatography (30% → 70% EtOAc/pentane) afforded the product (38 mg, 90 μmol, 90%). TLC: <i>R</i><sub>f</sub> = 0.4 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (t, <i>J</i> = 5.6 Hz, 1H), 7.18–7.07 (m, 2H), 6.87–6.80 (m, 2H), 6.72 (s, 1H), 3.79 (s, 3H), 3.79–3.72 (m, 6H), 3.66 (t, <i>J</i> = 4.8 Hz, 4H), 3.30 (dd, <i>J</i> = 7.1, 5.8 Hz, 2H), 3.11 (s, 3H), 2.88–2.81 (m, 2H), 1.11–1.02 (m, 1H), 0.58–0.52 (m, 2H), 0.32–0.27 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.71, 164.00, 160.88, 158.16, 156.79, 131.97, 129.76, 114.01, 90.06, 66.76, 55.39, 51.86, 44.51, 44.11, 35.75, 33.01, 10.90, 3.49. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 426.2500 calculated, 426.2497 found.</div></div><div id="sec4_2_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> <i>N</i>-(Cyclopropylmethyl)-2-((2-methoxyphenethyl)(methyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>43</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>123j</b> (24 mg, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 8 h at 160 °C with μW irradiation. Column chromatography (30% → 70% EtOAc/pentane) afforded the product (42 mg, 0.10 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.3 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (t, <i>J</i> = 6.0 Hz, 1H), 7.20 (td, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.13 (dd, <i>J</i> = 7.3, 1.7 Hz, 1H), 6.94–6.82 (m, 2H), 6.71 (s, 1H), 3.83 (s, 3H), 3.81–3.72 (m, 6H), 3.66 (t, <i>J</i> = 4.8 Hz, 4H), 3.30 (dd, <i>J</i> = 7.0, 5.9 Hz, 2H), 3.12 (s, 3H), 3.03–2.84 (m, 2H), 1.13–0.97 (m, 1H), 0.65–0.44 (m, 2H), 0.35–0.20 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.81, 163.99, 160.95, 157.81, 156.77, 130.55, 128.37, 127.64, 120.64, 110.48, 89.88, 66.79, 55.47, 49.93, 44.49, 44.04, 35.65, 28.70, 10.94, 3.47. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 426.2500 calculated, 426.2496 found.</div></div><div id="sec4_2_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(4-(trifluoromethyl)phenethyl)amino)-6-morpholino-pyrimidine-4-carboxamide (<b>45</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (22 mg, 73 μmol, 1 equiv), amine <b>123k</b> (29 mg, 0.10 mmol, 1.5 equiv), and DiPEA (51 μL, 0.29 mmol, 4 equiv). Total heating time: 25 h at 120 °C. Purification by HPLC (C18 reverse phase, 47% → 55% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT 12 min) afforded the product (9 mg, 20 μmol, 28%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98 (br s, 1H), 7.54 (d, <i>J</i> = 8.0 Hz, 2H), 7.32 (d, <i>J</i> = 7.9 Hz, 2H), 6.73 (s, 1H), 3.87–3.79 (m, 2H), 3.79–3.71 (m, 4H), 3.65 (t, <i>J</i> = 4.8 Hz, 4H), 3.35–3.24 (m, 2H), 3.10 (s, 3H), 3.02–2.93 (m, 2H), 1.12–0.99 (m, 1H), 0.65–0.48 (m, 2H), 0.39–0.22 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.61, 163.98, 160.88, 156.81, 144.18 (q, <i>J</i> = 1.4 Hz), 129.21, 128.89, 128.57, 125.73, 125.51 (q, <i>J</i> = 3.7 Hz), 123.03, 90.37, 66.75, 51.30, 44.51, 44.13, 35.87, 33.91, 10.92, 3.50. HRMS [C<sub>23</sub>H<sub>28</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 464.2268 calculated, 464.2267 found.</div></div><div id="sec4_2_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(4-phenoxyphenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>46</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>123l</b> (34 mg, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 48 h at 120 °C. Column chromatography (30% → 70% EtOAc/pentane) afforded the product (26 mg, 50 μmol, 50%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 8.00 (br s, 1H), 7.35–7.28 (m, 2H), 7.19–7.13 (m, 2H), 7.10–7.06 (m, 1H), 6.99–6.89 (m, 4H), 6.72 (s, 1H), 3.84–3.78 (m, 2H), 3.78–3.73 (m, 4H), 3.69–3.63 (m, 4H), 3.31–3.26 (m, 2H), 3.13 (s, 3H), 2.93–2.86 (m, 2H), 1.11–0.98 (m, 1H), 0.56–0.51 (m, 2H), 0.29–0.25 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) δ 164.67, 164.00, 160.92, 157.63, 156.80, 155.60, 134.93, 130.09, 129.83, 123.17, 119.27, 118.63, 90.12, 66.75, 51.67, 44.54, 44.12, 35.73, 33.25, 10.90, 3.49. HRMS [C<sub>28</sub>H<sub>33</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 488.2656 calculated, 488.2656 found.</div></div><div id="sec4_2_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(3-phenoxyphenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>47</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (25 mg, 84 μmol, 1 equiv), amine <b>123m</b> (28 mg, 0.12 mmol, 1.5 equiv), and DiPEA (60 μL, 0.34 mmol, 4 equiv). Total heating time: 5 days at 120 °C. Purification by HPLC (C18 reverse phase, 40% → 50% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT 11.2 min) afforded the product (14 mg, 29 μmol, 34%). TLC: <i>R</i><sub>f</sub> = 0.5 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.00 (br s, 1H), 7.35–7.28 (m, 2H), 7.29–7.20 (m, 1H), 7.13–7.06 (m, 1H), 6.99–6.94 (m, 3H), 6.92–6.87 (m, 1H), 6.86–6.83 (m, 1H), 6.71 (s, 1H), 3.84–3.77 (m, 2H), 3.76–3.70 (m, 4H), 3.67–3.61 (m, 4H), 3.30–3.24 (m, 2H), 3.12 (s, 3H), 2.93–2.83 (m, 2H), 1.11–0.98 (m, 1H), 0.57–0.49 (m, 2H), 0.30–0.24 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.67, 163.98, 160.89, 157.41, 157.32, 156.81, 142.03, 129.86, 123.91, 123.26, 119.57, 118.70, 116.91, 90.19, 66.75, 51.48, 44.52, 44.14, 35.69, 33.87, 10.90, 3.51. HRMS [C<sub>28</sub>H<sub>33</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 488.2656 calculated, 488.2653 found.</div></div><div id="sec4_2_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(2-phenoxyphenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>48</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (34 mg, 0.10 mmol, 1 equiv), amine <b>123n</b> (37 mg, 0.16 mmol, 1.6 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 3 days at 120 °C. Column chromatography (30% → 50% EtOAc/pentane) afforded the product (45 mg, 90 μmol, 90%). TLC: <i>R</i><sub>f</sub> = 0.5 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.06 (t, <i>J</i> = 5.4 Hz, 1H), 7.34–7.28 (m, 2H), 7.26 (dd, <i>J</i> = 7.4, 1.5 Hz, 1H), 7.17 (td, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.12–7.00 (m, 2H), 6.93 (dd, <i>J</i> = 8.6, 0.9 Hz, 2H), 6.86 (d, <i>J</i> = 8.0 Hz, 1H), 6.69 (s, 1H), 3.90–3.79 (m, 2H), 3.76–3.54 (m, 8H), 3.27 (t, <i>J</i> = 6.4 Hz, 2H), 3.07 (s, 3H), 2.99–2.88 (m, 2H), 1.11–0.98 (m, 1H), 0.60–0.44 (m, 2H), 0.27 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.74, 163.94, 160.90, 157.74, 156.72, 155.09, 131.31, 129.89, 127.84, 123.92, 122.99, 119.36, 118.07, 89.98, 66.70, 50.17, 44.44, 44.03, 35.57, 28.69, 10.92, 3.49. HRMS [C<sub>28</sub>H<sub>33</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 488.2656 calculated, 488.2653 found.</div></div><div id="sec4_2_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 2-((2-([1,1′-Biphenyl]-2-yl)ethyl)(methyl)amino)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>48</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (29 mg, 96 μmol, 1 equiv), amine <b>123o</b> (31 mg, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 24 h at 160 °C with μW irradiation. Column chromatography (30% → 60% EtOAc/pentane) afforded the product (43 mg, 91 μmol, 95%). TLC: <i>R</i><sub>f</sub> = 0.4 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.97 (t, <i>J</i> = 5.9 Hz, 1H), 7.41–7.19 (m, 9H), 6.68 (s, 1H), 3.77–3.70 (m, 4H), 3.66–3.57 (m, 6H), 3.28 (dd, <i>J</i> = 7.0, 5.8 Hz, 2H), 2.94–2.89 (m, 2H), 2.77 (s, 3H), 1.10–1.00 (m, 1H), 0.59–0.49 (m, 2H), 0.31–0.25 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.69, 163.87, 160.70, 156.65, 142.44, 141.63, 137.33, 130.26, 129.83, 129.28, 128.15, 127.56, 126.94, 126.29, 89.99, 66.72, 51.00, 44.44, 44.04, 35.31, 30.93, 10.90, 3.47. HRMS [C<sub>28</sub>H<sub>33</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 472.2707 calculated, 472.2703 found.</div></div><div id="sec4_2_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(2-(pyridin-4-yl)ethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>50</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), <i>N</i>-methyl-2-(pyridin-4-yl)ethan-1-amine (21 μL, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 17 h at 120 °C. Column chromatography (2% - > 6% MeOH/DCM) afforded the product (8 mg, 21 μmol, 21%). TLC: <i>R</i><sub>f</sub> = 0.2 (4% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.51 (d, <i>J</i> = 5.7 Hz, 2H), 7.96 (br s, 1H), 7.15 (d, <i>J</i> = 5.7 Hz, 2H), 6.74 (s, 1H), 3.90–3.80 (m, 2H), 3.80–3.72 (m, 4H), 3.72–3.53 (m, 4H), 3.36–3.24 (m, 2H), 3.11 (s, 3H), 2.98–2.85 (m, 2H), 1.12–0.99 (m, 1H), 0.64–0.47 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.54, 163.96, 160.83, 156.80, 149.96, 148.99, 124.32, 90.46, 66.74, 50.59, 44.50, 44.13, 35.87, 33.44, 10.91, 3.51. HRMS [C<sub>21</sub>H<sub>28</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 397.2347 calculated, 397.2345 found.</div></div><div id="sec4_2_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(2-(pyridin-3-yl)ethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>51</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), <i>N</i>-methyl-2-(pyridin-3-yl)ethan-1-amine (21 μL, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 17 h at 120 °C. Column chromatography (2% → 6% MeOH/DCM) afforded the product (11 mg, 29 μmol, 29%). TLC: <i>R</i><sub>f</sub> = 0.15 (4% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.65–8.28 (m, 2H), 7.95 (br s, 1H), 7.51 (d, <i>J</i> = 7.9 Hz, 1H), 7.21 (dd, <i>J</i> = 7.9, 4.8 Hz, 1H), 6.72 (s, 1H), 3.89–3.69 (m, 6H), 3.69–3.53 (m, 4H), 3.29 (t, <i>J</i> = 6.4 Hz, 2H), 3.11 (s, 3H), 2.92 (t, <i>J</i> = 7.4 Hz, 2H), 1.12–0.99 (m, 1H), 0.66–0.45 (m, 2H), 0.36–0.19 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.57, 163.95, 160.86, 156.77, 150.27, 147.86, 136.33, 135.30, 123.50, 90.36, 66.74, 51.19, 44.49, 44.15, 35.84, 31.23, 10.91, 3.51. HRMS [C<sub>21</sub>H<sub>28</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 397.2347 calculated, 397.2345 found.</div></div><div id="sec4_2_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(2-(pyridin-2-yl)ethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>52</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), <i>N</i>-methyl-2-(pyridin-2-yl)ethan-1-amine (21 μL, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 8 h at 160 °C with μW irradiation. Column chromatography (3% → 4% MeOH/DCM) afforded the product (26 mg, 66 μmol, 66%). TLC: <i>R</i><sub>f</sub> = 0.3 (3% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.66–8.44 (m, 1H), 8.18 (br s, 1H), 7.58 (td, <i>J</i> = 7.6, 1.7 Hz, 1H), 7.21–7.03 (m, 2H), 6.72 (s, 1H), 3.99 (t, <i>J</i> = 7.2 Hz, 2H), 3.88–3.71 (m, 4H), 3.71–3.54 (m, 4H), 3.31 (t, <i>J</i> = 6.4 Hz, 2H), 3.21–3.03 (m, 5H), 1.16–1.00 (m, 1H), 0.62–0.43 (m, 2H), 0.29 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.62, 163.91, 160.79, 159.86, 156.60, 149.21, 136.68, 123.60, 121.48, 90.12, 66.72, 49.98, 44.48, 44.01, 36.15, 35.62, 10.94, 3.45. HRMS [C<sub>21</sub>H<sub>28</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 397.2347 calculated, 397.2345 found.</div></div><div id="sec4_2_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(2-(thiophen-2-yl)ethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>53</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and <i>N</i>-methyl-2-thiopheneethylamine (18 μL, 0.13 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (16 mg, 40 μmol, 40%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (t, <i>J</i> = 5.3 Hz, 1H), 7.15 (dd, <i>J</i> = 5.1, 1.1 Hz, 1H), 6.94 (dd, <i>J</i> = 5.1, 3.4 Hz, 1H), 6.83 (d, <i>J</i> = 2.9 Hz, 1H), 6.73 (s, 1H), 3.88–3.80 (m, 2H), 3.80–3.72 (m, 4H), 3.71–3.62 (m, 4H), 3.33–3.26 (m, 2H), 3.20–3.08 (m, 5H), 1.13–0.99 (m, 1H), 0.60–0.48 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.62, 163.97, 160.84, 156.81, 142.18, 127.06, 125.03, 123.68, 90.26, 66.75, 51.90, 44.51, 44.13, 35.75, 27.94, 10.91, 3.55. HRMS [C<sub>20</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub>S + H]<sup>+</sup>: 402.1958 calculated, 402.1956 found.</div></div><div id="sec4_2_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> <i>N</i>-(Cyclopropylmethyl)-2-(ethyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>54</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (28 mg, 93 μmol, 1 equiv), amine <b>125a</b> (21 mg, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 12 h at 160 °C with μW irradiation. Column chromatography (20% → 50% EtOAc/pentane) afforded the product (15 mg, 40 μmol, 43%). TLC: <i>R</i><sub>f</sub> = 0.3 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (t, <i>J</i> = 5.9 Hz, 1H), 7.35–7.27 (m, 2H), 7.25–7.20 (m, 3H), 6.72 (s, 1H), 3.80–3.73 (m, 6H), 3.70–3.63 (m, 4H), 3.58 (q, <i>J</i> = 7.0 Hz, 2H), 3.30 (dd, <i>J</i> = 7.1, 5.8 Hz, 2H), 2.97–2.89 (m, 2H), 1.18 (t, <i>J</i> = 7.0 Hz, 3H), 1.11–1.01 (m, 1H), 0.59–0.51 (m, 2H), 0.31–0.25 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.74, 164.12, 160.29, 156.86, 140.06, 128.83, 128.62, 126.32, 90.06, 66.78, 49.83, 44.53, 44.09, 43.04, 34.76, 13.24, 10.90, 3.44. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 410.2551 calculated, 410.2549 found.</div></div><div id="sec4_2_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> <i>N</i>-(Cyclopropylmethyl)-2-(isopropyl(phenethyl)amino)-6-morpholinopyrimidine-4-carboxamide (<b>55</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (59 mg, 0.20 mmol, 1 equiv), amine <b>125b</b> (50 mg, 0.30 mmol, 1.5 equiv), and DiPEA (140 μL, 0.80 mmol, 4 equiv). Total heating time: 12 h at 160 °C with μW irradiation. Column chromatography (50% → 60% EtOAc/pentane) afforded the product (19 mg, 40 μmol, 22%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05 (br s, 1H), 7.37–7.29 (m, 2H), 7.29–7.20 (m, 3H), 6.75 (s, 1H), 4.96 (hept, <i>J</i> = 6.8 Hz, 1H), 3.82–3.74 (m, 4H), 3.72–3.64 (m, 4H), 3.64–3.55 (m, 2H), 3.35–3.26 (m, 2H), 3.01–2.88 (m, 2H), 1.24 (d, <i>J</i> = 6.8 Hz, 6H), 1.11–1.00 (m, 1H), 0.60–0.49 (m, 2H), 0.33–0.23 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.79, 164.06, 160.40, 156.81, 140.36, 128.68, 128.65, 126.35, 90.26, 66.77, 46.38, 44.57, 44.48, 44.17, 36.29, 20.65, 10.90, 3.51. HRMS [C<sub>24</sub>H<sub>33</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 424.2707 calculated, 424.2705 found.</div></div><div id="sec4_2_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 2-(Cyclopropyl(phenethyl)amino)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>56</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>125c</b> (25 mg, 0.16 mmol, 1.6 equiv), and DiPEA (115 μL, 0.60 mmol, 6 equiv). Total heating time: 36 h at 160 °C with μW irradiation. Purification by HPLC (C18 reverse phase, 37% → 47% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA) afforded the product (7 mg, 20 μmol, 20%). TLC: <i>R</i><sub>f</sub> = 0.4 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.17 (s, 1H), 7.34–7.24 (m, 2H), 7.25–7.16 (m, 3H), 6.79 (s, 1H), 3.87–3.73 (m, 6H), 3.69 (t, <i>J</i> = 4.7 Hz, 4H), 3.30 (dd, <i>J</i> = 7.1, 5.7 Hz, 2H), 2.98–2.85 (m, 2H), 2.73–2.64 (m, 1H), 1.14–0.98 (m, 1H), 0.90–0.77 (m, 2H), 0.68–0.59 (m, 2H), 0.59–0.49 (m, 2H), 0.35–0.22 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.62, 163.99, 162.12, 156.64, 140.18, 128.88, 128.59, 126.31, 91.08, 66.78, 50.52, 44.57, 44.11, 34.58, 30.41, 10.86, 8.67, 3.42. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 422.2551 calculated, 422.2549 found.</div></div><div id="sec4_2_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> <i>N</i>-(Cyclopropylmethyl)-6-morpholino-2-(phenethyl(phenyl)amino)pyrimidine-4-carboxamide (<b>57</b>)</h4><div class="NLM_p last">A microwave vial with a magnetic stir bar under N<sub>2</sub> was charged with 2-chloropyrimidine <b>31</b> (29 mg, 98 μmol, 1 equiv), amine <b>124</b> (24 mg, 0.12 mmol, 1.2 equiv), and dry toluene (0.1 mL). The vial was capped, and the solution purged with N<sub>2</sub>. This was followed by the addition of RuPhosPd G3 (0.01 M THF solution, 100 μL, 1 μmol, 0.01 equiv) and NaOtBu (2 M THF solution, 60 μL, 0.12 mmol, 1.2 equiv), and the mixture was purged again with N<sub>2</sub> and stirred in a preheated oil bath at 110 °C. After 24 h, the reaction was complete as judged by LC–MS. The mixture was filtered through a plug of Celite, and the filtrate was concentrated under reduced pressure to provide the crude material. Purification by HPLC (C18 reverse phase, 5% → 50% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT 12.0 min) afforded the product (12 mg, 26 μmol, 27%). TLC: <i>R</i><sub>f</sub> = 0.4 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.79 (t, <i>J</i> = 4.9 Hz, 1H), 7.43–7.33 (m, 2H), 7.32–7.17 (m, 8H), 6.79 (s, 1H), 4.28–4.11 (m, 2H), 3.79–3.67 (m, 4H), 3.60 (t, <i>J</i> = 4.8 Hz, 4H), 3.20 (dd, <i>J</i> = 7.1, 5.7 Hz, 2H), 3.09–2.96 (m, 2H), 1.02–0.89 (m, 1H), 0.55–0.41 (m, 2H), 0.27–0.12 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.24, 163.97, 160.50, 156.64, 144.67, 139.63, 128.84, 128.62, 127.72, 126.40, 125.73, 91.42, 66.71, 52.33, 44.51, 43.96, 34.64, 10.70, 3.34. HRMS [C<sub>27</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 458.2551 calculated, 458.2547 found.</div></div><div id="sec4_2_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 2-(Benzyl(phenethyl)amino)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>58</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (26 mg, 88 μmol, 1 equiv), amine <b>125d</b> (28 mg, 0.13 mmol, 1.5 equiv), and DiPEA (44 μL, 0.25 mmol, 3 equiv). Total heating time: 7 days at 120 °C. Column chromatography (30% → 50% EtOAc/pentane) afforded the product (20 mg, 40 μmol, 48%). TLC: <i>R</i><sub>f</sub> = 0.6 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, MeOD + CDCl<sub>3</sub>) δ 7.40–7.08 (m, 10H), 6.71 (s, 1H), 4.78 (s, 2H), 3.95–3.54 (m, 10H), 3.23 (br s, 2H), 2.98–2.83 (m, 2H), 1.02 (br s, 1H), 0.54 (br s, 2H), 0.25 (br s, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.60, 164.09, 160.81, 156.93, 139.90, 139.50, 128.82, 128.62, 128.57, 127.31, 127.04, 126.34, 90.66, 66.72, 51.39, 49.88, 44.56, 44.13, 34.22, 10.84, 3.47. HRMS [C<sub>28</sub>H<sub>33</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 472.2707 calculated, 472.2704 found.</div></div><div id="sec4_2_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> <i>N</i>-(Cyclopropylmethyl)-2-(diphenethylamino)-6-morpholinopyrimidine-4-carboxamide (<b>59</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>125e</b> (37 mg, 0.16 mmol, 1.6 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 12 h at 160 °C with μW irradiation. Column chromatography (20% → 50% EtOAc/pentane) afforded the product (24 mg, 50 μmol, 50%). TLC: <i>R</i><sub>f</sub> = 0.5 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (t, <i>J</i> = 5.8 Hz, 1H), 7.34–7.26 (m, 4H), 7.24–7.15 (m, 6H), 6.75 (s, 1H), 3.83–3.60 (m, 12H), 3.30 (dd, <i>J</i> = 7.1, 5.8 Hz, 2H), 2.97–2.82 (m, 4H), 1.12–1.00 (m, 1H), 0.63–0.49 (m, 2H), 0.35–0.23 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.66, 164.08, 160.34, 156.84, 139.91, 128.82, 128.62, 126.35, 90.34, 66.75, 50.69, 44.57, 44.23, 34.58, 10.86, 3.52. HRMS [C<sub>29</sub>H<sub>35</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 486.2864 calculated, 486.2861 found.</div></div><div id="sec4_2_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> <i>N</i>-(Cyclopropylmethyl)-2-(3,4-dihydroisoquinolin-2(1<i>H</i>)-yl)-6-morpholinopyrimidine-4-carboxamide (<b>60</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (29 mg, 97 μmol, 1 equiv), 1,2,3,4-tetrahydroisoquinoline (19 μL, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 18 h at 120 °C. Column chromatography (20% → 60% EtOAc/pentane) afforded the product (34 mg, 90 μmol, 93%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (t, <i>J</i> = 6.0 Hz, 1H), 7.25–7.13 (m, 4H), 6.76 (s, 1H), 4.90 (s, 2H), 4.04 (t, <i>J</i> = 5.9 Hz, 2H), 3.81–3.62 (m, 8H), 3.36–3.28 (m, 2H), 2.93 (t, <i>J</i> = 5.8 Hz, 2H), 1.16–1.01 (m, 1H), 0.61–0.51 (m, 2H), 0.31 (dt, <i>J</i> = 6.1, 4.6 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.61, 163.99, 160.78, 156.88, 135.40, 134.49, 128.77, 126.58, 126.40, 126.23, 90.80, 66.76, 46.46, 44.54, 44.10, 41.63, 29.14, 10.99, 3.54. HRMS [C<sub>22</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 394.2238 calculated, 394.2231 found.</div></div><div id="sec4_2_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> (±)-<i>N</i>-(Cyclopropylmethyl)-6-morpholino-2-(3-phenylpyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>61</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), (±)-3-phenylpyrrolidine (22 μL, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (30% → 70% EtOAc/pentane) afforded the product (39 mg, 95 μmol, 95%). TLC: <i>R</i><sub>f</sub> = 0.4 (40% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.09 (br s, 1H), 7.40–7.29 (m, 4H), 7.29–7.24 (m, 1H), 6.75 (s, 1H), 4.18–4.03 (m, 1H), 3.88 (t, <i>J</i> = 9.5 Hz, 1H), 3.79–3.71 (m, 4H), 3.71–3.59 (m, 5H), 3.58–3.51 (m, 1H), 3.51–3.43 (m, 1H), 3.35–3.20 (m, 2H), 2.43–2.33 (m, 1H), 2.18–2.06 (m, 1H), 1.13–0.97 (m, 1H), 0.61–0.44 (m, 2H), 0.34–0.20 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.65, 163.89, 159.58, 156.77, 142.22, 128.70, 127.29, 126.84, 90.15, 66.73, 53.27, 46.49, 44.42, 44.19, 44.02, 33.28, 10.90, 3.49. HRMS [C<sub>23</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 408.2394 calculated, 408.2391 found.</div></div><div id="sec4_2_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> (±)-<i>N-</i>(Cyclopropylmethyl)-6-morpholino-2-(3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>62</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), (±)-3-phenylpiperidine (24 μL, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (40% → 70% EtOAc/pentane) afforded the product (37 mg, 88 μmol, 88%). TLC: <i>R</i><sub>f</sub> = 0.3 (40% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.97 (t, <i>J</i> = 5.9 Hz, 1H), 7.35 (t, <i>J</i> = 7.5 Hz, 2H), 7.32–7.22 (m, 3H), 6.73 (s, 1H), 4.89–4.70 (m, 2H), 3.74 (t, <i>J</i> = 4.8 Hz, 4H), 3.69–3.56 (m, 4H), 3.37–3.16 (m, 2H), 2.97–2.83 (m, 2H), 2.76 (tt, <i>J</i> = 11.5, 3.7 Hz, 1H), 2.12–2.02 (m, 1H), 1.91–1.83 (m, 1H), 1.82–1.72 (m, 1H), 1.72–1.57 (m, 1H), 1.11–0.99 (m, 1H), 0.61–0.43 (m, 2H), 0.27 (q, <i>J</i> = 5.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.60, 164.07, 160.94, 156.93, 144.28, 128.65, 127.30, 126.66, 90.57, 66.71, 51.24, 44.67, 44.48, 44.04, 42.63, 32.27, 25.56, 10.94, 3.50, 3.48. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 422.2551 calculated, 422.2548 found.</div></div><div id="sec4_2_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> (±)-2-(2-Benzylpyrrolidin-1-yl)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>63</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), (±)-2-benzylpyrrolidine (24 μL, 0.15 mmol, 1.5 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 3 days at 120 °C. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (42 mg, 0.10 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.11 (t, <i>J</i> = 5.8 Hz, 1H), 7.35–7.25 (m, 2H), 7.26–7.18 (m, 3H), 6.76 (s, 1H), 4.48–4.34 (m, 1H), 3.80–3.73 (m, 4H), 3.73–3.65 (m, 4H), 3.65–3.59 (m, 1H), 3.59–3.50 (m, 1H), 3.37–3.24 (m, 3H), 2.59 (dd, <i>J</i> = 13.1, 9.7 Hz, 1H), 1.91–1.82 (m, 4H), 1.14–0.98 (m, 1H), 0.65–0.46 (m, 2H), 0.37–0.21 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.71, 164.02, 159.56, 156.78, 139.83, 129.39, 128.49, 126.24, 90.22, 66.76, 59.06, 47.38, 44.54, 44.18, 39.30, 29.42, 23.16, 10.87, 3.51. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 422.2551 calculated, 422.2549 found.</div></div><div id="sec4_2_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> (±)-2-(2-Benzylpiperidin-1-yl)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>64</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), (±)-2-benzylpiperidine hydrochloride (52 mg, 0.24 mmol, 2.4 equiv), and DiPEA (104 μL, 0.60 mmol, 6 equiv). Total heating time: 6 days at 120 °C. Column chromatography (30% → 60% EtOAc/pentane) afforded the product (12 mg, 27 μmol, 27%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.98 (t, <i>J</i> = 5.2 Hz, 1H), 7.26 (t, <i>J</i> = 7.4 Hz, 2H), 7.24–7.16 (m, 3H), 6.70 (s, 1H), 5.13–4.97 (m, 1H), 4.77–4.60 (m, 1H), 3.82–3.73 (m, 4H), 3.66 (br s, 4H), 3.40–3.21 (m, 2H), 3.07–2.98 (m, 1H), 2.95 (dd, <i>J</i> = 13.1, 10.0 Hz, 1H), 2.81 (dd, <i>J</i> = 13.1, 5.1 Hz, 1H), 1.85–1.73 (m, 2H), 1.73–1.67 (m, 2H), 1.57–1.46 (m, 2H), 1.13–1.04 (m, 1H), 0.63–0.53 (m, 2H), 0.31 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) δ 164.75, 163.97, 160.84, 156.81, 140.16, 129.26, 128.47, 126.16, 90.34, 66.78, 52.27, 44.53, 44.24, 39.47, 35.32, 26.29, 25.82, 19.30, 10.95, 3.65, 3.61. HRMS [C<sub>25</sub>H<sub>33</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 436.2707 calculated, 436.2706 found.</div></div><div id="sec4_2_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> (±)-<i>N</i>-(Cyclopropylmethyl)-2-(2-(cyclohexylmethyl)piperidin-1-yl)-6-morpholinopyrimidine-4-carboxamide (<b>65</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (24 mg, 80 μmol, 1 equiv), DiPEA (70 μL, 0.4 mmol, 5 equiv), and (±)-2-(cyclohexylmethyl)piperidine (22 mg, 0.12 mmol, 1.5 equiv). Total heating time: 24 h at 160 °C with μW irradiation. Column chromatography (10% → 60% EtOAc/pentane) afforded the product (6 mg, 14 μmol, 17%). TLC: <i>R</i><sub>f</sub> = 0.6 (50% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.98 (s, 1H), 6.67 (s, 1H), 5.07–4.94 (m, 1H), 4.61 (dd, <i>J</i> = 13.8, 4.4 Hz, 1H), 3.81–3.70 (m, 4H), 3.70–3.56 (m, 4H), 3.37–3.17 (m, 2H), 2.90 (td, <i>J</i> = 13.1, 2.5 Hz, 1H), 1.80 (dd, <i>J</i> = 25.9, 12.9 Hz, 2H), 1.73–1.59 (m, 8H), 1.56–1.38 (m, 3H), 1.24–1.10 (m, 4H), 1.10–1.00 (m, 1H), 0.99–0.86 (m, 2H), 0.62–0.47 (m, 2H), 0.27 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 164.86, 164.07, 160.88, 156.90, 89.87, 66.82, 47.48, 44.55, 44.22, 38.83, 36.74, 34.55, 34.22, 33.47, 29.85, 28.23, 26.76, 26.48, 26.42, 25.94, 19.46, 10.90, 3.57, 3.54. HRMS [C<sub>25</sub>H<sub>39</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 442.3177 calculated, 442.3174 found.</div></div><div id="sec4_2_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> (±)-<i>N</i>-(Cyclopropylmethyl)-2-(2-(4-methoxybenzyl)piperidin-1-yl)-6-morpholinopyrimidine-4-carboxamide (<b>66</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), (±)-2-(4-methoxybenzyl)piperidine (46 mg, 0.22 mmol, 2.2 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 28 h at 160 °C with μW irradiation. Purification by HPLC (C18 reverse phase, 5% → 90% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT 9.3 min) afforded the product (13 mg, 29 μmol, 29%). TLC: <i>R</i><sub>f</sub> = 0.3 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98 (t, <i>J</i> = 5.8 Hz, 1H), 7.16–7.10 (m, 2H), 6.85–6.75 (m, 2H), 6.69 (s, 1H), 5.00 (dt, <i>J</i> = 10.4, 4.9 Hz, 1H), 4.67 (dd, <i>J</i> = 13.5, 3.7 Hz, 1H), 3.84–3.72 (m, 7H), 3.65 (t, <i>J</i> = 4.8 Hz, 4H), 3.40–3.21 (m, 2H), 3.00 (td, <i>J</i> = 13.2, 2.8 Hz, 1H), 2.90 (dd, <i>J</i> = 13.2, 10.0 Hz, 1H), 2.74 (dd, <i>J</i> = 13.2, 5.1 Hz, 1H), 1.85–1.71 (m, 2H), 1.65 (s, 2H), 1.59–1.42 (m, 2H), 1.15–1.01 (m, 1H), 0.64–0.50 (m, 2H), 0.36–0.25 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.78, 164.00, 160.88, 158.03, 156.84, 132.18, 130.15, 113.89, 90.31, 66.79, 55.40, 52.38, 44.54, 44.23, 39.47, 34.37, 26.18, 25.83, 19.29, 10.97, 3.65, 3.61. HRMS [C<sub>26</sub>H<sub>35</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 466.2813 calculated, 466.2809 found.</div></div><div id="sec4_2_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> (±)-<i>N</i>-(Cyclopropylmethyl)-6-morpholino-2-(2-phenylmorpholino)pyrimidine-4-carboxamide (<b>67</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and (±)-2-phenylmorpholine (21 μL, 0.13 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (37 mg, 87 μmol, 87%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.92 (br s, 1H), 7.50–7.31 (m, 5H), 6.80 (s, 1H), 4.66 (d, <i>J</i> = 13.3 Hz, 1H), 4.59–4.48 (m, 2H), 4.20–4.13 (m, 1H), 3.80 (td, <i>J</i> = 11.8, 2.8 Hz, 1H), 3.77–3.70 (m, 4H), 3.70–3.59 (m, 4H), 3.36–3.22 (m, 2H), 3.22–3.12 (m, 1H), 2.97 (dd, <i>J</i> = 13.3, 10.6 Hz, 1H), 1.12–0.99 (m, 1H), 0.60–0.46 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.34, 163.92, 160.97, 156.85, 139.91, 128.62, 128.23, 126.51, 91.50, 78.26, 67.02, 66.67, 50.63, 44.47, 44.11, 43.99, 10.94, 3.53. HRMS [C<sub>23</sub>H<sub>29</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 424.2343 calculated, 424.2340 found.</div></div><div id="sec4_2_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> (±)-<i>N</i>-(Cyclopropylmethyl)-6-morpholino-2-(3-phenylpiperazin-1-yl)pyrimidine-4-carboxamide (<b>68</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with Cbz-protected amine <b>70</b> (56 mg, 0.10 mmol, 1 equiv) and MeOH (0.5 mL). The solution was purged with N<sub>2</sub> and Pd/C (10% w/w, 50 mg, 50 μmol, 0.5 equiv) was added. The mixture was purged with N<sub>2</sub> and then with H<sub>2</sub> and stirred for 2 h under a H<sub>2</sub> atmosphere (balloon). The mixture was filtered through a plug of Celite and washed with MeOH, and the filtrate was concentrated under reduced pressure. The residue was purified by silica gel column chromatography (1% → 4% MeOH/DCM) to afford the product (38 mg, 90 μmol, 90%). TLC: R<sub>f</sub> = 0.3 (2% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (t, <i>J</i> = 5.9 Hz, 1H), 7.51–7.44 (m, 2H), 7.43–7.30 (m, 3H), 6.76 (s, 1H), 4.69 (d, <i>J</i> = 12.5 Hz, 2H), 3.88–3.78 (m, 1H), 3.78–3.71 (m, 4H), 3.68–3.61 (m, 4H), 3.32–3.23 (m, 2H), 3.21 (d, <i>J</i> = 10.6 Hz, 1H), 3.13–2.94 (m, 2H), 2.90 (t, <i>J</i> = 11.7 Hz, 1H), 2.11 (br s, 1H), 1.09–1.00 (m, 1H), 0.57–0.47 (m, 2H), 0.30–0.24 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.51, 163.99, 161.01, 156.89, 142.01, 128.73, 127.97, 127.32, 91.05, 66.72, 60.57, 51.48, 46.27, 44.49, 44.26, 44.10, 10.97, 3.55. HRMS [C<sub>23</sub>H<sub>30</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 423.2503 calculated, 423.2501 found.</div></div><div id="sec4_2_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> (±)-2-(4-Benzyl-3-phenylpiperazin-1-yl)-<i>N</i>-(cyclopropylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>69</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with amine <b>68</b> (19 mg, 45 μmol, 1 equiv) in dry CH<sub>3</sub>CN (0.5 mL). This was followed by DiPEA (16 μL, 90 μmol, 2 equiv) and benzyl bromide (6.4 μL, 54 μmol, 1.2 equiv). The reaction was stirred for 4 h at rt after which the solvents were concentrated under reduced pressure. The residue was purified by silica gel column chromatography (1 → 5% MeOH/DCM) affording the product (17 mg, 33 μmol, 74%). TLC: R<sub>f</sub> = 0.5 (2% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.90 (t, <i>J</i> = 5.7 Hz, 1H), 7.57 (d, <i>J</i> = 7.2 Hz, 2H), 7.41 (t, <i>J</i> = 7.4 Hz, 2H), 7.36–7.27 (m, 5H), 7.25–7.18 (m, 1H), 6.74 (s, 1H), 4.70–4.54 (m, 2H), 3.83 (d, <i>J</i> = 13.4 Hz, 1H), 3.77–3.67 (m, 4H), 3.66–3.54 (m, 4H), 3.35 (dd, <i>J</i> = 10.6, 3.1 Hz, 1H), 3.32–3.18 (m, 2H), 3.08 (td, <i>J</i> = 12.7, 2.7 Hz, 1H), 3.03–2.92 (m, 2H), 2.87 (d, <i>J</i> = 13.4 Hz, 1H), 2.17 (td, <i>J</i> = 11.8, 3.0 Hz, 1H), 1.09–0.95 (m, 1H), 0.57–0.44 (m, 2H), 0.25 (q, <i>J</i> = 4.6 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.48, 164.02, 160.74, 156.91, 141.70, 138.93, 128.92, 128.87, 128.29, 128.20, 127.87, 127.00, 90.97, 67.34, 66.71, 59.23, 51.90, 51.73, 44.47, 44.36, 44.08, 10.92, 3.53, 3.51. HRMS [C<sub>30</sub>H<sub>36</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 513.2973 calculated, 513.2973 found.</div></div><div id="sec4_2_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> (±)-Benzyl 4-(4-((Cyclopropylmethyl)carbamoyl)-6-morpholinopyrimidin-2-yl)-2-phenylpiperazine-1-carboxylate (<b>70</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), amine <b>141</b> (45 mg, 0.12 mmol, 1.2 equiv), and DiPEA (70 μL, 0.40 mmol, 4 equiv). Total heating time: 41 h at 120 °C. Column chromatography (40% → 70% EtOAc/pentane) afforded the product (56 mg, 0.1 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.3 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.90 (t, <i>J</i> = 5.9 Hz, 1H), 7.45–7.19 (m, 10H), 6.77 (s, 1H), 5.47 (br s, 1H), 5.31–5.12 (m, 2H), 5.04 (d, <i>J</i> = 13.8 Hz, 1H), 4.52–4.31 (m, 1H), 4.17 (d, <i>J</i> = 9.4 Hz, 1H), 3.75 (t, <i>J</i> = 4.8 Hz, 4H), 3.71–3.60 (m, 4H), 3.53 (d, <i>J</i> = 13.4 Hz, 1H), 3.29 (t, <i>J</i> = 6.5 Hz, 2H), 3.17 (d, <i>J</i> = 9.2 Hz, 2H), 1.13–0.97 (m, 1H), 0.61–0.46 (m, 2H), 0.34–0.21 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.36, 163.88, 160.76, 156.82, 155.79, 139.31, 136.59, 128.68, 128.65, 128.23, 128.03, 127.44, 127.04, 91.44, 67.65, 66.70, 45.36, 44.54, 44.13, 43.82, 39.89, 10.93, 3.53. HRMS [C<sub>31</sub>H<sub>36</sub>N<sub>6</sub>O<sub>4</sub> + H]<sup>+</sup>: 557.2871 calculated, 557.2869 found.</div></div><div id="sec4_2_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(piperidin-1-yl)pyrimidine-4-carboxamide (<b>71</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119k</b> (44 mg, 0.15 mmol, 1 equiv), DiPEA (105 μL, 0.60 mmol, 4 equiv), and <i>N</i>-methylphenethylamine HBr salt (49 mg, 0.23 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (20% → 50% EtOAc/pentane) afforded the product (37 mg, 94 μmol, 63%). TLC: <i>R</i><sub>f</sub> = 0.4 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.07 (t, <i>J</i> = 4.8 Hz, 1H), 7.35–7.16 (m, 5H), 6.76 (s, 1H), 3.84–3.75 (m, 2H), 3.67 (br s, 4H), 3.34–3.25 (m, 2H), 3.13 (s, 3H), 2.95–2.87 (m, 2H), 1.74–1.65 (m, 2H), 1.65–1.55 (m, 4H), 1.13–1.00 (m, 1H), 0.59–0.50 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.02, 163.41, 161.01, 156.35, 140.10, 128.88, 128.56, 126.22, 90.36, 51.80, 45.37, 44.07, 35.66, 33.94, 25.77, 24.95, 10.91, 3.48. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O + H]<sup>+</sup>: 394.2601 calculated, 394.2592 found.</div></div><div id="sec4_2_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> <i>N</i>-(Cyclopropylmethyl)-6-(3,3-difluoropiperidin-1-yl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>72</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119l</b> (50 mg, 0.15 mmol, 1 equiv), DiPEA (78 μL, 0.45 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (33 μL, 0.27 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (10% → 30% EtOAc/pentane) afforded the product (10 mg, 23 μmol, 15%). TLC: <i>R</i><sub>f</sub> = 0.5 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (br s, 1H), 7.34–7.27 (m, 2H), 7.26–7.18 (m, 3H), 6.78 (s, 1H), 4.01 (t, <i>J</i> = 11.7 Hz, 2H), 3.86–3.73 (m, 2H), 3.71–3.58 (m, 2H), 3.35–3.25 (m, 2H), 3.13 (s, 3H), 2.95–2.86 (m, 2H), 2.17–2.03 (m, 2H), 1.88–1.78 (m, 2H), 1.12–1.01 (m, 1H), 0.61–0.50 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.61, 163.70, 160.80, 157.10, 139.92, 128.91, 128.63, 126.32, 119.79 (t, <i>J</i> = 244.2 Hz), 90.16, 51.89, 49.40 (t, <i>J</i> = 32.8 Hz), 44.14, 43.73, 35.80, 33.92, 33.03 (t, <i>J</i> = 23.5 Hz), 22.07 (t, <i>J</i> = 4.4 Hz), 10.91, 3.50. HRMS [C<sub>23</sub>H<sub>29</sub>F<sub>2</sub>N<sub>5</sub>O + H]<sup>+</sup>: 430.2413 calculated, 430.2419 found.</div></div><div id="sec4_2_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> <i>N</i>-(Cyclopropylmethyl)-6-(4,4-difluoropiperidin-1-yl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>72</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119m</b> (45 mg, 0.14 mmol, 1 equiv), DiPEA (73 μL, 0.42 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (31 μL, 0.21 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (5% → 30% EtOAc/pentane) afforded the product (27 mg, 63 μmol, 45%). TLC: <i>R</i><sub>f</sub> = 0.5 (15% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (t, <i>J</i> = 5.2 Hz, 1H), 7.33–7.27 (m, 2H), 7.25–7.18 (m, 3H), 6.79 (s, 1H), 3.88–3.74 (m, 6H), 3.36–3.25 (m, 2H), 3.13 (s, 3H), 2.95–2.85 (m, 2H), 2.06–1.92 (m, 4H), 1.13–0.99 (m, 1H), 0.63–0.50 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.58, 163.23, 160.99, 157.16, 139.85, 128.83, 128.62, 126.35, 122.10 (t, <i>J</i> = 242.0 Hz), 90.06, 51.73, 44.13, 41.27, 35.74, 33.95, 33.84 (t, <i>J</i> = 23.0 Hz), 10.89, 3.49. HRMS [C<sub>23</sub>H<sub>29</sub>F<sub>2</sub>N<sub>5</sub>O + H]<sup>+</sup>: 430.2413 calculated, 430.2422 found.</div></div><div id="sec4_2_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-thiomorpholinopyrimidine-4-carboxamide (<b>74</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119n</b> (41 mg, 0.13 mmol, 1 equiv), DiPEA (91 μL, 0.52 mmol, 4 equiv), and <i>N</i>-methylphenethylamine HBr salt (43 mg, 0.20 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (10% → 40% EtOAc/pentane) afforded the product (51 mg, 0.12 mmol, 95%). TLC: <i>R</i><sub>f</sub> = 0.4 (25% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05 (br s, 1H), 7.33–7.27 (m, 2H), 7.25–7.18 (m, 3H), 6.73 (s, 1H), 4.03 (br s, 4H), 3.84–3.74 (m, 2H), 3.35–3.26 (m, 2H), 3.13 (s, 3H), 2.96–2.85 (m, 2H), 2.70–2.58 (m, 4H), 1.13–0.99 (m, 1H), 0.55 (q, <i>J</i> = 5.7 Hz, 2H), 0.29 (q, <i>J</i> = 4.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.67, 163.05, 156.85, 139.87, 128.80, 128.58, 126.29, 90.34, 51.74, 47.24, 44.08, 35.71, 33.90, 26.68, 10.88, 3.47. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>OS + H]<sup>+</sup>: 412.2166 calculated, 412.2159 found.</div></div><div id="sec4_2_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> <i>N</i>-(Cyclopropylmethyl)-6-(1,1-dioxidothiomorpholino)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>75</b>)</h4><div class="NLM_p last">The title compound was prepared according to G<a class="ref internalNav" href="#sec4_2_2" aria-label="eneral Procedure A">eneral Procedure A</a> using 2-chloropyrimidine <b>119o</b> (4:1 mixture of regioisomers) (35 mg, 0.10 mmol, 1 equiv), DiPEA (70 μL, 0.40 mmol, 4 equiv), and <i>N-</i>methylphenethylamine HBr salt (32 mg, 0.15 mmol, 1.5 equiv). Total heating time: 2 days at 120 °C. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (35 mg, 79 μmol, 79%). TLC: <i>R</i><sub>f</sub> = 0.6 (60% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.99 (br s, 1H), 7.36–7.26 (m, 2H), 7.26–7.16 (m, 3H), 6.81 (s, 1H), 4.20 (br s, 4H), 3.83 (t, <i>J</i> = 7.5 Hz, 2H), 3.30 (t, <i>J</i> = 6.4 Hz, 2H), 3.13 (s, 3H), 3.04 (br s, 4H), 2.91 (t, <i>J</i> = 7.4 Hz, 2H), 1.14–1.00 (m, 1H), 0.66–0.49 (m, 2H), 0.37–0.24 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.04, 162.66, 160.94, 157.93, 139.56, 128.76, 128.68, 126.49, 89.88, 51.67, 51.54, 44.19, 43.04, 35.82, 33.95, 10.86, 3.50. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O<sub>3</sub>S + H]<sup>+</sup>: 444.2064 calculated, 444.2074 found.</div></div><div id="sec4_2_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(4-methylpiperazin-1-yl)pyrimidine-4-carboxamide (<b>76</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119p</b> (42 mg, 0.14 mmol, 1 equiv), DiPEA (98 μL, 0.56 mmol, 4 equiv), and <i>N</i>-methylphenethylamine HBr salt (44 mg, 0.20 mmol, 1.5 equiv). Total heating time: 45 h at 120 °C. Purification by preparative HPLC (C18 reverse phase, 25% to 35% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT 8.98 min) afforded the product (21 mg, 51 μmol, 37%). TLC: <i>R</i><sub>f</sub> = 0.3 (5% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (t, <i>J</i> = 5.4 Hz, 1H), 7.33–7.26 (m, 2H), 7.25–7.18 (m, 3H), 6.74 (s, 1H), 3.87–3.73 (m, 6H), 3.34–3.25 (m, 2H), 3.13 (s, 3H), 2.94–2.86 (m, 2H), 2.64–2.51 (m, 4H), 2.42 (s, 3H), 1.12–1.00 (m, 1H), 0.60–0.50 (m, 2H), 0.29 (q, <i>J</i> = 4.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.70, 163.60, 160.91, 156.82, 139.95, 128.85, 128.60, 126.30, 90.22, 54.58, 51.72, 45.88, 44.12, 43.58, 35.71, 33.94, 10.89, 3.49. HRMS [C<sub>23</sub>H<sub>32</sub>N<sub>6</sub>O + H]<sup>+</sup>: 409.2710 calculated, 409.2708 found.</div></div><div id="sec4_2_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(piperazin-1-yl)pyrimidine-4-carboxamide (<b>77</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with Cbz-protected amine <b>80</b> (175 mg, 0.33 mmol, 1 equiv), dry MeOH (3 mL), and AcOH (0.3 mL). The flask was purged with N<sub>2</sub> followed by addition of Pd/C (10% w/w, 18 mg, 0.02 mmol, 5 mol %) and then purging with H<sub>2</sub> (balloon). The reaction was stirred for 2 days then filtered over a cellulose filter (Whatman), which was washed with MeOH. The filtrate was concentrated under reduced pressure, and the residue was purified using silica gel column chromatography (2.5% → 5% MeOH/DCM with 5% Et<sub>3</sub>N) affording the product (82 mg, 0.21 mmol, 63%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05 (t, <i>J</i> = 5.5 Hz, 1H), 7.35–7.26 (m, 2H), 7.26–7.17 (m, 3H), 6.74 (s, 1H), 3.84–3.76 (m, 2H), 3.75–3.58 (m, 4H), 3.30 (t, <i>J</i> = 6.5 Hz, 2H), 3.13 (s, 3H), 2.96–2.85 (m, 6H), 1.96 (s, 1H), 1.13–1.00 (m, 1H), 0.61–0.47 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.78, 163.74, 160.85, 156.51, 139.95, 128.80, 128.52, 126.20, 90.16, 51.66, 45.94, 45.26, 44.02, 35.62, 33.88, 10.84, 3.42. HRMS [C<sub>22</sub>H<sub>30</sub>N<sub>6</sub>O + H]<sup>+</sup>: 395.2554 calculated, 395.2558 found.</div></div><div id="sec4_2_86" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> 6-(4-Acetylpiperazin-1-yl)-<i>N</i>-(cyclopropylmethyl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>78</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with amine <b>77</b> (22 mg, 56 μmol, 1 equiv) in dry DCM (1.5 mL). This was followed by addition of DiPEA (49 μL, 0.28 mmol, 5 equiv) and Ac<sub>2</sub>O (10.5 μL, 0.11 mmol, 2 equiv). The reaction was stirred for 3 h at rt, and after which, it was diluted with EtOAc (25 mL). The organic layer was washed with sat. aq. NaHCO<sub>3</sub> (1× 25 mL) and brine (1× 25 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (2.5% → 10% MeOH/DCM) affording the product (19 mg, 44 μmol, 78%). TLC: <i>R</i><sub>f</sub> = 0.8 (10% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (t, <i>J</i> = 5.3 Hz, 1H), 7.34–7.26 (m, 2H), 7.26–7.17 (m, 3H), 6.74 (s, 1H), 3.87–3.77 (m, 2H), 3.77–3.65 (m, 6H), 3.59–3.50 (m, 2H), 3.34–3.26 (m, 2H), 3.13 (s, 3H), 2.95–2.86 (m, 2H), 2.15 (s, 3H), 1.14–0.99 (m, 1H), 0.61–0.50 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 169.32, 164.57, 163.63, 160.85, 156.97, 139.84, 128.82, 128.61, 126.33, 90.13, 51.70, 45.94, 44.12, 43.82, 41.12, 35.73, 33.92, 21.57, 10.87, 3.48. HRMS [C<sub>24</sub>H<sub>32</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 437.2660 calculated, 437.2661 found.</div></div><div id="sec4_2_87" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> 6-(4-Benzoylpiperazin-1-yl)-<i>N</i>-(cyclopropylmethyl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>79</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with amine <b>77</b> (22 mg, 56 μmol, 1 equiv) in dry DCM (1.5 mL). This was followed by Et<sub>3</sub>N (16 μL, 0.11 mmol, 2 equiv) and benzoyl chloride (8 μL, 67 μmol, 1.2 equiv). The reaction was stirred for 3 h at rt, and after which, it was diluted with EtOAc (25 mL). The organic layer was washed with sat. aq. NaHCO<sub>3</sub> (1× 25 mL) and brine (1× 25 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (60% → 80% EtOAc/pentane) affording the product (20 mg, 40 μmol, 72%). TLC: <i>R</i><sub>f</sub> = 0.3 (60% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (br s, 1H), 7.49–7.40 (m, 5H), 7.32–7.25 (m, 2H), 7.25–7.17 (m, 3H), 6.75 (s, 1H), 3.95–3.74 (m, 6H), 3.66 (br s, 2H), 3.52 (br s, 2H), 3.36–3.25 (m, 2H), 3.12 (s, 3H), 2.94–2.82 (m, 2H), 1.14–0.99 (m, 1H), 0.63–0.48 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.77, 164.61, 163.71, 160.87, 156.99, 139.85, 135.48, 130.14, 128.83, 128.76, 128.62, 128.49, 127.23, 126.34, 90.21, 51.70, 47.55, 44.16, 42.05, 35.75, 33.94, 10.88, 3.50. HRMS [C<sub>29</sub>H<sub>34</sub>N<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>: 499.2816 calculated, 499.2825 found.</div></div><div id="sec4_2_88" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Benzyl 4-(6-((Cyclopropylmethyl)carbamoyl)-2-(methyl(phenethyl)amino)pyrimidin-4-yl)piperazine-1-carboxylate (<b>80</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119q</b> (202 mg, 0.47 mmol, 1 equiv), DiPEA (0.40 mL, 2.28 mmol, 5 equiv), and <i>N</i>-methylphenethylamine HBr salt (161 mg, 0.74 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (30% → 60% EtOAc/pentane) afforded the product (199 mg, 0.38 mmol, 80%). TLC: <i>R</i><sub>f</sub> = 0.6 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (t, <i>J</i> = 5.4 Hz, 1H), 7.42–7.25 (m, 7H), 7.24–7.17 (m, 3H), 6.73 (s, 1H), 5.17 (s, 2H), 3.86–3.77 (m, 2H), 3.69 (br s, 4H), 3.62–3.53 (m, 4H), 3.34–3.26 (m, 2H), 3.12 (s, 3H), 2.94–2.85 (m, 2H), 1.12–1.00 (m, 1H), 0.61–0.50 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.58, 163.64, 160.85, 156.90, 155.32, 139.84, 136.56, 128.79, 128.62, 128.56, 128.23, 128.08, 126.27, 90.16, 67.46, 51.66, 44.07, 43.86, 43.53, 35.67, 33.91, 10.85, 3.44. HRMS [C<sub>30</sub>H<sub>36</sub>N<sub>6</sub>O<sub>3</sub> + H]<sup>+</sup>: 529.2922 calculated, 529.2933 found.</div></div><div id="sec4_2_89" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> <i>N</i>-(Cyclopropylmethyl)-6-(dimethylamino)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>81</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119r</b> (27 mg, 0.11 mmol, 1 equiv), <i>N</i>-methylphenethylamine HBr salt (35 mg, 0.16 mmol, 1.6 equiv), and DiPEA (92 μL, 0.53 mmol, 5 equiv). Total heating time: 8 h at 160 °C with μW irradiation. Column chromatography (20% → 50% EtOAc/pentane) afforded the product (32 mg, 90 μmol, 90%). TLC: <i>R</i><sub>f</sub> = 0.6 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (t, <i>J</i> = 5.8 Hz, 1H), 7.34–7.27 (m, 2H), 7.26–7.17 (m, 3H), 6.69 (s, 1H), 3.92–3.70 (m, 2H), 3.30 (dd, <i>J</i> = 7.1, 5.8 Hz, 2H), 3.23–3.01 (m, 9H), 3.00–2.80 (m, 2H), 1.17–0.98 (m, 1H), 0.64–0.46 (m, 2H), 0.36–0.22 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.89, 164.01, 160.68, 155.88, 140.01, 128.79, 128.47, 126.12, 90.10, 51.66, 43.98, 37.16, 35.52, 33.84, 10.82, 3.39. HRMS [C<sub>20</sub>H<sub>27</sub>N<sub>5</sub>O + H]<sup>+</sup>: 354.2288 calculated, 354.2290 found.</div></div><div id="sec4_2_90" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(methylamino)pyrimidine-4-carboxamide (<b>82</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119s</b> (25 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (22 μL, 0.15 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (50% → 70% EtOAc/pentane) afforded the product (26 mg, 77 μmol, 77%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (br s, 1H), 7.33–7.27 (m, 2H), 7.26–7.18 (m, 3H), 6.54 (s, 1H), 5.13–4.72 (m, 1H), 3.93–3.66 (m, 2H), 3.34–3.24 (m, 2H), 3.13 (s, 3H), 2.98 (d, <i>J</i> = 4.9 Hz, 3H), 2.94–2.87 (m, 2H), 1.12–1.00 (m, 1H), 0.61–0.50 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.97, 164.71, 161.06, 156.00, 140.05, 128.89, 128.55, 126.24, 92.54, 51.66, 44.06, 35.62, 33.93, 28.23, 10.89, 3.48. HRMS [C<sub>19</sub>H<sub>25</sub>N<sub>5</sub>O + H]<sup>+</sup>: 340.2132 calculated, 340.2138 found.</div></div><div id="sec4_2_91" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> <i>N</i>-(Cyclopropylmethyl)-6-((2-hydroxyethyl)(methyl)amino)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>83</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_4" aria-label="General Procedure C">General Procedure C</a> using dichloropyrimidine <b>118a</b> (39 mg, 0.16 mmol, 1 equiv), DiPEA (43 μL, 0.24 mmol, 1.5 equiv), and <i>N</i>-methylethanolamine (13 μL, 0.16 mmol, 1.0 equiv) in MeOH (1.6 mL) followed by concentration and addition of DiPEA (84 μL, 0.48 mmol, 3 equiv), <i>N</i>-methylphenethylamine (35 μL, 0.24 mmol, 1.5 equiv), and <i>n</i>-BuOH (0.75 mL). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (20% → 40% EtOAc/pentane) afforded the product (15 mg, 33 μmol, 21%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (br s, 1H), 7.33–7.27 (m, 2H), 7.26–7.18 (m, 3H), 6.68 (s, 1H), 4.12–3.52 (m, 7H), 3.33–3.26 (m, 2H), 3.14 (s, 3H), 3.11 (s, 3H), 2.93–2.86 (m, 2H), 1.12–1.00 (m, 1H), 0.60–0.51 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.77, 164.63, 160.52, 156.34, 139.80, 128.86, 128.60, 126.32, 90.28, 62.14, 52.62, 51.83, 44.12, 37.11, 35.91, 33.97, 10.86, 3.48. HRMS [C<sub>21</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 384.2394 calculated, 384.2399 found.</div></div><div id="sec4_2_92" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> <i>N</i>-(Cyclopropylmethyl)-6-(diethylamino)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>84</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119t</b> (37 mg, 0.13 mmol, 1 equiv), DiPEA (68 μL, 0.39 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (28 μL, 0.20 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (20% → 40% EtOAc/pentane) afforded the product (30 mg, 79 μmol, 60%). TLC: <i>R</i><sub>f</sub> = 0.8 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (br s, 1H), 7.40–7.14 (m, 5H), 6.65 (s, 1H), 3.92–3.71 (m, 2H), 3.54 (br s, 4H), 3.30 (t, <i>J</i> = 6.3 Hz, 2H), 3.14 (s, 3H), 3.02–2.85 (m, 2H), 1.20 (t, <i>J</i> = 6.9 Hz, 6H), 1.13–0.99 (m, 1H), 0.64–0.46 (m, 2H), 0.37–0.21 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.05, 162.59, 160.96, 155.94, 140.12, 128.87, 128.55, 126.21, 90.19, 51.79, 44.03, 42.44, 35.60, 34.04, 13.26, 10.92, 3.46. HRMS [C<sub>22</sub>H<sub>31</sub>N<sub>5</sub>O + H]<sup>+</sup>: 382.2601 calculated, 382.2599 found.</div></div><div id="sec4_2_93" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(azetidin-1-yl)pyrimidine-4-carboxamide (<b>85</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119u</b> (27 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (22 μL, 0.15 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (25 mg, 68 μmol, 68%). TLC: <i>R</i><sub>f</sub> = 0.6 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (br s, 1H), 7.33–7.27 (m, 2H), 7.26–7.16 (m, 3H), 6.38 (s, 1H), 4.11 (t, <i>J</i> = 7.5 Hz, 4H), 3.85–3.71 (m, 2H), 3.35–3.26 (m, 2H), 3.13 (s, 3H), 2.94–2.84 (m, 2H), 2.40 (p, <i>J</i> = 7.5 Hz, 2H), 1.14–1.00 (m, 1H), 0.60–0.48 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.78, 164.75, 160.96, 155.60, 140.10, 128.90, 128.55, 126.22, 89.42, 51.73, 49.83, 44.07, 35.54, 33.88, 16.75, 10.90, 3.49. HRMS [C<sub>21</sub>H<sub>27</sub>N<sub>5</sub>O + H]<sup>+</sup>: 366.2288 calculated, 366.2296 found.</div></div><div id="sec4_2_94" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(2-oxa-6-azaspiro[3.3]heptan-6-yl)pyrimidine-4-carboxamide (<b>86</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119v</b> (31 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (22 μL, 0.15 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (70% → 90% EtOAc/pentane) afforded the product (7 mg, 17 μmol, 17%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.00 (br s, 1H), 7.35–7.27 (m, 2H), 7.26–7.19 (m, 3H), 6.39 (s, 1H), 4.85 (s, 4H), 4.24 (s, 4H), 3.84–3.71 (m, 2H), 3.36–3.22 (m, 2H), 3.12 (s, 3H), 2.94–2.82 (m, 2H), 1.13–0.98 (m, 1H), 0.60–0.48 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.68, 164.52, 160.91, 156.07, 139.97, 128.89, 128.62, 126.32, 89.61, 81.23, 59.40, 51.71, 44.12, 39.24, 35.61, 33.89, 10.90, 3.51. HRMS [C<sub>23</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 408.2394 calculated, 408.2396 found.</div></div><div id="sec4_2_95" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(pyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>87</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119w</b> (28 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (22 μL, 0.15 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (30% → 50% EtOAc/pentane) afforded the product (26 mg, 69 μmol, 69%). TLC: <i>R</i><sub>f</sub> = 0.7 (40% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.09 (t, <i>J</i> = 5.3 Hz, 1H), 7.33–7.27 (m, 2H), 7.27–7.18 (m, 3H), 6.54 (s, 1H), 3.89–3.71 (m, 2H), 3.62 (br s, 2H), 3.53–3.34 (m, 2H), 3.30 (dd, <i>J</i> = 6.9, 5.9 Hz, 2H), 3.14 (s, 3H), 2.97–2.85 (m, 2H), 2.15–1.85 (m, 4H), 1.12–1.01 (m, 1H), 0.63–0.47 (m, 2H), 0.35–0.22 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 165.03, 161.92, 160.90, 155.49, 140.19, 128.89, 128.54, 126.18, 91.37, 51.73, 46.43, 44.05, 35.55, 33.95, 25.78, 25.01, 10.91, 3.47. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O + H]<sup>+</sup>: 380.2445 calculated, 380.2452 found.</div></div><div id="sec4_2_96" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> <i>N</i>-(Cyclopropylmethyl)-6-(3,3-difluoropyrrolidin-1-yl)-2-(methyl(phenethyl) amino)pyrimidine-4-carboxamide (<b>88</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119x</b> (32 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (22 μL, 0.15 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (20% → 40% EtOAc/pentane) afforded the product (34 mg, 82 μmol, 82%). TLC: <i>R</i><sub>f</sub> = 0.8 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (br s, 1H), 7.34–7.27 (m, 2H), 7.25–7.18 (m, 3H), 6.52 (br s, 1H), 3.89 (br s, 2H), 3.84–3.78 (m, 2H), 3.74 (br s, 2H), 3.37–3.22 (m, 2H), 3.13 (s, 3H), 2.97–2.83 (m, 2H), 2.56–2.37 (m, 2H), 1.14–1.00 (m, 1H), 0.64–0.47 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.50, 162.05, 160.76, 156.36, 139.91, 128.86, 128.61, 127.48 (t, <i>J</i> = 247.3 Hz), 126.31, 90.39, 53.35 (t, <i>J</i> = 32.2 Hz), 51.68, 44.12, 43.95, 35.66, 33.94 (t, <i>J</i> = 22.8 Hz), 10.88, 3.50. HRMS [C<sub>22</sub>H<sub>27</sub>F<sub>2</sub>N<sub>5</sub>O + H]<sup>+</sup>: 416.2256 calculated, 416.2260 found.</div></div><div id="sec4_2_97" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> (±)-<i>N</i>-(Cyclopropylmethyl)-6-(3-hydroxypyrrolidin-1-yl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>89</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119y</b> (65 mg, 0.22 mmol, 1 equiv), DiPEA (115 μL, 0.66 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (48 μL, 0.33 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (70% → 100% EtOAc/pentane) afforded the product (60 mg, 0.15 mmol, 69%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (t, <i>J</i> = 5.2 Hz, 1H), 7.32–7.26 (m, 2H), 7.25–7.17 (m, 3H), 6.50 (s, 1H), 4.58 (br s, 1H), 3.98–3.33 (m, 6H), 3.31–3.23 (m, 2H), 3.11 (s, 3H), 3.02–2.62 (m, 3H), 2.08 (br s, 2H), 1.10–0.98 (m, 1H), 0.63–0.45 (m, 2H), 0.27 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.08, 162.07, 160.78, 155.35, 140.10, 128.87, 128.54, 126.19, 91.26, 54.89, 51.73, 44.36, 44.10, 35.57, 33.91, 10.83, 3.48. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 396.2394 calculated, 396.2400 found.</div></div><div id="sec4_2_98" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> (<i>R</i>)-<i>N</i>-(Cyclopropylmethyl)-6-(3-hydroxypyrrolidin-1-yl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>90</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119z</b> (38 mg, 0.13 mmol, 1 equiv), DiPEA (67 μL, 0.38 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (28 μL, 0.19 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (80% → 100% EtOAc/pentane) afforded the product (37 mg, 94 μmol, 78%). <i>ee</i>: >99% (as determined by chiral HPLC using 75:25 hexane/ethanol, Chiralcell OD). TLC: <i>R</i><sub>f</sub> = 0.3 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.09 (br s, 1H), 7.33–7.27 (m, 2H), 7.26–7.17 (m, 3H), 6.52 (s, 1H), 4.59 (br s, 1H), 3.79 (dd, <i>J</i> = 8.7, 5.8 Hz, 2H), 3.74–3.37 (m, 4H), 3.33–3.22 (m, 2H), 3.12 (s, 3H), 2.96–2.82 (m, 2H), 2.70–2.21 (m, 1H), 2.09 (br s, 2H), 1.12–0.99 (m, 1H), 0.61–0.49 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.04, 162.12, 160.82, 155.49, 140.13, 128.90, 128.58, 126.22, 91.28, 70.52, 54.92, 51.75, 44.34, 44.12, 35.60, 33.93, 10.87, 3.50. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 396.2394 calculated, 396.2394 found.</div></div><div id="sec4_2_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> (<i>S</i>)-<i>N</i>-(Cyclopropylmethyl)-6-(3-hydroxypyrrolidin-1-yl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>91</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119aa</b> (36 mg, 0.12 mmol, 1 equiv), DiPEA (63 μL, 0.36 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (27 μL, 0.18 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (80% → 100% EtOAc/pentane) afforded the product (36 mg, 91 μmol, 70%). <i>ee</i>: 97% (as determined by chiral HPLC using 75:25 hexane/ethanol, Chiralcell OD). TLC: <i>R</i><sub>f</sub> = 0.3 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (t, <i>J</i> = 5.1 Hz, 1H), 7.35–7.27 (m, 2H), 7.25–7.15 (m, 3H), 6.51 (s, 1H), 4.58 (br s, 1H), 3.82–3.74 (m, 2H), 3.74–3.35 (m, 4H), 3.32–3.22 (m, 2H), 3.12 (s, 3H), 2.94–2.84 (m, 2H), 2.80–2.48 (m, 1H), 2.08 (br s, 2H), 1.12–0.98 (m, 1H), 0.60–0.48 (m, 2H), 0.27 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.07, 162.07, 160.77, 155.36, 140.10, 128.88, 128.55, 126.20, 91.26, 70.88, 70.32, 54.89, 51.74, 44.35, 44.11, 35.58, 33.91, 10.83, 3.48. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 396.2394 calculated, 396.2398 found.</div></div><div id="sec4_2_100" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> (±)-<i>N</i>-(Cyclopropylmethyl)-6-(3-methoxypyrrolidin-1-yl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>92</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with alcohol <b>89</b> (35 mg, 88 μmol, 1 equiv) in dry DMF (0.5 mL) and cooled to 0 °C. NaH (60% in mineral oil, 4 mg, 106 μmol, 1.2 equiv) was added, and the mixture was stirred for 15 min followed by addition of methyl iodide (6.0 μL, 97 μmol, 1.1 equiv). The reaction was allowed to warm to rt while stirring overnight. The reaction was quenched with H<sub>2</sub>O (20 mL) followed by extraction with DCM (3× 20 mL), drying (Na<sub>2</sub>SO<sub>4</sub>), filtering, and concentrating under reduced pressure. The residue was purified by silica gel column chromatography (60 → 70% EtOAc/pentane), affording the product (16 mg, 39 μmol, 44%). TLC: <i>R</i><sub>f</sub> = 0.4 (60% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.07 (br s, 1H), 7.33–7.27 (m, 2H), 7.26–7.17 (m, 3H), 6.54 (s, 1H), 4.07 (br s, 1H), 3.88–3.72 (m, 3H), 3.72–3.45 (m, 3H), 3.37 (s, 3H), 3.33–3.23 (m, 2H), 3.13 (s, 3H), 2.98–2.84 (m, 2H), 2.29–1.96 (m, 2H), 1.14–0.99 (m, 1H), 0.62–0.48 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.96, 162.10, 160.86, 155.62, 140.16, 128.91, 128.57, 126.22, 91.25, 79.95, 79.15, 56.76, 51.75, 51.42, 44.42, 44.09, 35.62, 33.94, 30.45, 10.91, 3.50. HRMS [C<sub>23</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 410.2551 calculated, 410.2549 found.</div></div><div id="sec4_2_101" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> (±)-<i>N</i>-(Cyclopropylmethyl)-6-(3-(dimethylamino)pyrrolidin-1-yl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>93</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119ab</b> (50 mg, 0.15 mmol, 1 equiv), DiPEA (78 μL, 0.60 mmol, 4 equiv), and <i>N</i>-methylphenethylamine (33 μL, 0.23 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (2.5% → 10% MeOH/DCM) afforded the product (3 mg, 7 μmol, 5%). TLC: <i>R</i><sub>f</sub> = 0.5 (5% MeOH/DCM). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 8.10 (br s, 1H), 7.31–7.27 (m, 2H), 7.24–7.18 (m, 3H), 6.69 (s, 1H), 3.98 (dd, <i>J</i> = 10.6, 7.2 Hz, 1H), 3.92–3.64 (m, 3H), 3.58–3.50 (m, 1H), 3.42 (br s, 1H), 3.35–3.25 (m, 2H), 2.99 (br s, 3H), 2.89 (t, <i>J</i> = 7.7 Hz, 2H), 2.41 (br s, 6H), 2.24 (br s, 1H), 2.06–1.89 (m, 1H), 1.80–1.51 (m, 1H), 1.12–1.02 (m, 1H), 0.59–0.49 (m, 2H), 0.34–0.24 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) δ 164.89, 163.37, 159.54, 156.17, 128.99, 128.68, 126.44, 90.62, 65.56, 51.98, 45.61, 44.02, 33.80, 28.71, 11.00, 3.52. HRMS [C<sub>24</sub>H<sub>34</sub>N<sub>6</sub>O + H]<sup>+</sup>: 423.2867 calculated, 423.2868 found.</div></div><div id="sec4_2_102" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> (±)-<i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(3-phenylpyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>94</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_4" aria-label="General Procedure C">General Procedure C</a> using dichloropyrimidine <b>118a</b> (40 mg, 0.16 mmol, 1 equiv), DiPEA (43 μL, 0.24 mmol, 1.5 equiv), and (±)-3-phenylpyrrolidine (24 μL, 0.16 mmol, 1.0 equiv) in MeOH (1.6 mL) followed by concentration and addition of DiPEA (84 μL, 0.48 mmol, 3 equiv), <i>N</i>-methylphenethylamine (35 μL, 0.24 mmol, 1.5 equiv), and <i>n</i>-BuOH (0.75 mL). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (70% → 90% EtOAc/pentane) afforded the product (35 mg, 91 μmol, 57%). TLC: <i>R</i><sub>f</sub> = 0.4 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (br s, 1H), 7.39–7.32 (m, 2H), 7.32–7.17 (m, 8H), 6.57 (s, 1H), 4.19 (br s, 1H), 3.94 (br s, 1H), 3.81 (br s, 2H), 3.74–3.38 (m, 3H), 3.30 (t, <i>J</i> = 6.4 Hz, 2H), 3.14 (s, 3H), 2.92 (br s, 2H), 2.42 (br s, 1H), 2.16 (br s, 1H), 1.14–1.00 (m, 1H), 0.62–0.46 (m, 2H), 0.29 (q, <i>J</i> = 4.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.96, 161.90, 160.92, 155.72, 141.79, 140.15, 128.92, 128.80, 128.57, 127.19, 126.97, 126.23, 91.13, 52.89, 51.76, 46.36, 44.10, 43.39, 35.62, 33.98, 33.31, 10.93, 3.51. HRMS [C<sub>28</sub>H<sub>33</sub>N<sub>5</sub>O + H]<sup>+</sup>: 456.2758 calculated, 456.2757 found.</div></div><div id="sec4_2_103" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(benzyl(methyl)amino)pyrimidine-4-carboxamide (<b>95</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_4" aria-label="General Procedure C">General Procedure C</a> using dichloropyrimidine <b>118a</b> (37 mg, 0.15 mmol, 1 equiv), DiPEA (39 μL, 0.23 mmol, 1.5 equiv), and <i>N</i>-methylbenzylamine (19 μL, 0.15 mmol, 1.0 equiv) in MeOH (1.5 mL) followed by concentration, and addition of DiPEA (78 μL, 0.45 mmol, 3 equiv), <i>N</i>-methylphenethylamine (33 μL, 0.23 mmol, 1.5 equiv), and <i>n</i>-BuOH (0.75 mL). Total heating time: 4 h at 160 °C with μW irradiation. Purification by HPLC (C18 reverse phase, 43% to 49% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA) afforded the product (47 mg, 0.11 mmol, 73%). TLC: <i>R</i><sub>f</sub> = 0.6 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.08 (br s, 1H), 7.36–7.28 (m, 2H), 7.28–7.03 (m, 8H), 6.75 (br s, 1H), 4.86 (br s, 2H), 3.77 (br s, 2H), 3.39–3.22 (m, 2H), 3.13 (s, 3H), 3.08 (br s, 2H), 2.86 (br s, 2H), 1.14–0.98 (m, 1H), 0.62–0.48 (m, 2H), 0.29 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.88, 164.13, 160.83, 156.51, 139.96, 128.89, 128.74, 128.57, 127.24, 126.24, 90.07, 52.48, 51.85, 44.14, 35.75, 33.95, 10.94, 3.52. HRMS [C<sub>26</sub>H<sub>31</sub>N<sub>5</sub>O + H]<sup>+</sup>: 430.2601 calculated, 430.2604 found.</div></div><div id="sec4_2_104" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> <i>N</i>-(Cyclopropylmethyl)-2,6-bis(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>96</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using dichloropyrimidine <b>118a</b> (28 mg, 0.11 mmol, 1 equiv), DiPEA (79 μL, 0.46 mmol, 4 equiv), and <i>N</i>-methylphenethylamine (41 μL, 0.28 mmol, 2.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (20% → 40% EtOAc/pentane) afforded the product (30 mg, 68 μmol, 62%). TLC: <i>R</i><sub>f</sub> = 0.6 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.09 (t, <i>J</i> = 5.2 Hz, 1H), 7.32–7.24 (m, 4H), 7.24–7.12 (m, 6H), 6.68 (br s, 1H), 4.02–3.56 (m, 4H), 3.34–3.26 (m, 2H), 3.15 (s, 3H), 2.99 (br s, 3H), 2.96–2.87 (m, 4H), 1.15–1.01 (m, 1H), 0.60–0.48 (m, 2H), 0.29 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.96, 163.42, 160.86, 156.14, 140.03, 128.97, 128.90, 128.68, 128.60, 126.43, 126.25, 90.24, 51.72, 44.09, 35.71, 34.01, 33.85, 10.93, 3.50. HRMS [C<sub>27</sub>H<sub>33</sub>N<sub>5</sub>O + H]<sup>+</sup>: 444.2758 calculated, 444.2765 found.</div></div><div id="sec4_2_105" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(1<i>H</i>-pyrazol-1-yl)pyrimidine-4-carboxamide (<b>97</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119ac</b> (28 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (22 μL, 0.15 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (20% → 40% EtOAc/pentane) afforded the product (30 mg, 80 μmol, 80%). TLC: <i>R</i><sub>f</sub> = 0.5 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.51 (s, 1H), 7.98–7.81 (m, 2H), 7.81–7.73 (m, 1H), 7.33–7.27 (m, 2H), 7.26–7.18 (m, 3H), 6.56–6.39 (m, 1H), 3.97–3.83 (m, 2H), 3.33 (dd, <i>J</i> = 6.9, 6.0 Hz, 2H), 3.21 (s, 3H), 3.01–2.91 (m, 2H), 1.18–1.00 (m, 1H), 0.57 (q, <i>J</i> = 5.4 Hz, 2H), 0.30 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.17, 160.57, 159.73, 159.48, 143.40, 139.40, 128.84, 128.69, 127.30, 126.52, 108.54, 94.69, 51.83, 44.24, 36.01, 33.79, 10.87, 3.52. HRMS [C<sub>21</sub>H<sub>24</sub>N<sub>6</sub>O + H]<sup>+</sup>: 377.2084 calculated, 377.2088 found.</div></div><div id="sec4_2_106" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> <i>N</i>-(Cyclopropylmethyl)-6-(1<i>H</i>-imidazol-1-yl)-2-(methyl(phenethyl)amino)pyrimidine-4-carboxamide (<b>98</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119ad</b> (42 mg, 0.15 mmol, 1 equiv), DiPEA (78 μL, 0.45 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (33 μL, 0.23 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Purification by preparative HPLC (C18 reverse phase, 35% to 45% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT 8.87 min) afforded the product (27 mg, 72 μmol, 48%). TLC: <i>R</i><sub>f</sub> = 0.5 (5% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.46 (s, 1H), 7.99–7.72 (m, 1H), 7.68 (s, 1H), 7.33–7.27 (m, 3H), 7.25–7.17 (m, 4H), 4.00–3.81 (m, 2H), 3.38–3.29 (m, 2H), 3.22 (s, 3H), 3.02–2.90 (m, 2H), 1.16–1.03 (m, 1H), 0.69–0.51 (m, 2H), 0.32 (q, <i>J</i> = 4.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 162.88, 160.89, 160.19, 157.03, 139.14, 135.37, 131.10, 128.82, 128.73, 126.61, 116.00, 93.80, 51.89, 44.36, 36.06, 33.78, 10.84, 3.56. HRMS [C<sub>21</sub>H<sub>24</sub>N<sub>6</sub>O + H]<sup>+</sup>: 377.2084 calculated, 377.2087 found.</div></div><div id="sec4_2_107" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-phenoxypyrimidine-4-carboxamide (<b>99</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119ae</b> (49 mg, 0.16 mmol, 1 equiv), DiPEA (84 μL, 0.48 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (35 μL, 0.24 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (5% → 25% EtOAc/pentane) afforded the product (40 mg, 99 μmol, 62%). TLC: <i>R</i><sub>f</sub> = 0.5 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91 (br s, 1H), 7.46–7.36 (m, 2H), 7.35–7.05 (m, 7H), 7.04–6.68 (m, 2H), 4.00–3.39 (m, 2H), 3.30 (t, <i>J</i> = 6.4 Hz, 2H), 3.10 (br s, 3H), 2.97–2.53 (m, 2H), 1.13–1.01 (m, 1H), 0.56 (q, <i>J</i> = 5.6 Hz, 2H), 0.35–0.23 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.62, 163.46, 160.96, 159.59, 152.85, 139.32, 129.64, 128.89, 128.50, 126.33, 125.47, 122.04, 51.85, 44.18, 35.78, 33.63, 10.88, 3.51. HRMS [C<sub>24</sub>H<sub>26</sub>N<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 403.2129 calculated, 403.2137 found.</div></div><div id="sec4_2_108" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> <i>N</i>-(Cyclopropylmethyl)-2-(methyl(phenethyl)amino)-6-(pyridin-3-yloxy) Pyrimidine-4-carboxamide (<b>100</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119af</b> (38 mg, 0.12 mmol, 1 equiv), DiPEA (62 μL, 0.36 mmol, 3 equiv), and <i>N</i>-methylphenethylamine (26 μL, 0.18 mmol, 1.5 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (60% → 100% EtOAc/pentane) afforded the product (26 mg, 64 μmol, 54%). TLC: <i>R</i><sub>f</sub> = 0.5 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.66–8.42 (m, 2H), 7.90 (br s, 1H), 7.65–7.44 (m, 1H), 7.41–7.32 (m, 1H), 7.31–7.12 (m, 4H), 7.07–6.71 (m, 2H), 3.82 (br s, 1H), 3.51 (br s, 1H), 3.38–3.26 (m, 2H), 3.13 (br s, 2H), 2.93 (br s, 2H), 2.65 (br s, 1H), 1.15–1.02 (m, 1H), 0.57 (q, <i>J</i> = 5.1 Hz, 2H), 0.30 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.88, 163.22, 160.69, 160.01, 149.45, 146.41, 144.26, 129.68, 128.80, 128.56, 126.44, 123.96, 51.74, 44.23, 35.78, 33.59, 10.88, 3.53. HRMS [C<sub>23</sub>H<sub>25</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 404.2081 calculated, 404.2088 found.</div></div><div id="sec4_2_109" class="NLM_sec NLM_sec_level_3"><div id="ac_i138" class="anchor-spacer"></div><h4 class="article-section__title" id="_i138"> (<i>S</i>)<i>-N</i>-(Cyclopropylmethyl)-6-(dimethylamino)-2-(3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>101</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119r</b> (25 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and (<i>S</i>)-3-phenylpiperidine (21 mg, 0.13 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (30% → 50% EtOAc/pentane) afforded the product (28 mg, 74 μmol, 74%). <i>ee</i>: >99% (as determined by chiral HPLC using 70:30 hexane/isopropanol, Chiralcell OD). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (br s, 1H), 7.43–7.16 (m, 5H), 6.70 (s, 1H), 4.84 (t, <i>J</i> = 14.1 Hz, 2H), 3.39–3.19 (m, 2H), 3.10 (s, 6H), 2.89 (t, <i>J</i> = 12.1 Hz, 2H), 2.77 (t, <i>J</i> = 10.2 Hz, 1H), 2.07 (d, <i>J</i> = 13.1 Hz, 1H), 1.88–1.60 (m, 3H), 1.14–0.98 (m, 1H), 0.63–0.44 (m, 2H), 0.36–0.20 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.91, 164.18, 160.89, 156.13, 144.43, 128.61, 127.31, 126.58, 90.71, 51.38, 44.70, 44.02, 42.56, 37.30, 32.21, 25.58, 10.96, 3.51, 3.49. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O + H]<sup>+</sup>: 380.2445 calculated, 380.2452 found.</div></div><div id="sec4_2_110" class="NLM_sec NLM_sec_level_3"><div id="ac_i139" class="anchor-spacer"></div><h4 class="article-section__title" id="_i139"> (<i>R</i>)<i>-N</i>-(Cyclopropylmethyl)-6-(dimethylamino)-2-(3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>102</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119r</b> (25 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv) and (<i>R</i>)-3-phenylpiperidine (21 mg, 0.13 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (30% → 50% EtOAc/pentane) afforded the product (32 mg, 84 μmol, 84%). <i>ee</i>: >97% (as determined by chiral HPLC using 70:30 hexane/isopropanol, Chiralcell OD). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.01 (t, <i>J</i> = 5.4 Hz, 1H), 7.39–7.28 (m, 4H), 7.28–7.21 (m, 1H), 6.70 (s, 1H), 4.95–4.74 (m, 2H), 3.40–3.18 (m, 2H), 3.10 (s, 6H), 2.96–2.83 (m, 2H), 2.83–2.70 (m, 1H), 2.07 (d, <i>J</i> = 13.9 Hz, 1H), 1.90–1.81 (m, 1H), 1.81–1.62 (m, 2H), 1.11–0.99 (m, 1H), 0.59–0.46 (m, 2H), 0.27 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.91, 164.18, 160.90, 156.12, 144.43, 128.61, 127.31, 126.58, 90.71, 51.38, 44.70, 44.02, 42.56, 37.30, 32.21, 25.58, 10.96, 3.51, 3.49. HRMS [C<sub>22</sub>H<sub>29</sub>N<sub>5</sub>O + H]<sup>+</sup>: 380.2445 calculated, 380.2452 found.</div></div><div id="sec4_2_111" class="NLM_sec NLM_sec_level_3"><div id="ac_i140" class="anchor-spacer"></div><h4 class="article-section__title" id="_i140"> (<i>S</i>)-<i>N</i>-(Cyclopropylmethyl)-6-morpholino-2-(3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>103</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>31</b> (30 mg, 0.10 mmol, 1 equiv), DiPEA (52 μL, 0.30 mmol, 3 equiv), and (<i>S</i>)-3-phenylpiperidine (21 mg, 0.13 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (40% → 60% EtOAc/pentane) afforded the product (41 mg, 97 μmol, 97%). TLC: <i>R</i><sub>f</sub> = 0.6 (50% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.97 (br s, 1H), 7.40–7.32 (m, 2H), 7.32–7.21 (m, 3H), 6.73 (s, 1H), 4.90–4.70 (m, 2H), 3.83–3.70 (m, 4H), 3.69–3.56 (m, 4H), 3.36–3.19 (m, 2H), 2.97–2.83 (m, 2H), 2.76 (tt, <i>J</i> = 11.5, 3.6 Hz, 1H), 2.13–2.02 (m, 1H), 1.90–1.72 (m, 2H), 1.72–1.59 (m, 1H), 1.12–0.98 (m, 1H), 0.59–0.45 (m, 2H), 0.27 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 164.63, 164.11, 160.97, 156.96, 144.30, 128.66, 127.30, 126.67, 90.60, 66.73, 51.27, 44.71, 44.52, 44.06, 42.64, 32.28, 25.57, 10.94, 3.50, 3.48. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 422.2551 calculated, 422.2549 found.</div></div><div id="sec4_2_112" class="NLM_sec NLM_sec_level_3"><div id="ac_i141" class="anchor-spacer"></div><h4 class="article-section__title" id="_i141"> (<i>S</i>)-<i>N</i>-(Cyclopropylmethyl)-6-(1,1-dioxidothiomorpholino)-2-(3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>104</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119o</b> (4:1 mixture of regioisomers) (35 mg, 0.10 mmol, 1 equiv), DiPEA (53 μL, 0.30 mmol, 3 equiv), and (<i>S</i>)-3-phenylpiperidine (21 mg, 0.13 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Purification by preparative HPLC (C18 reverse phase, 45% to 55% CH<sub>3</sub>CN/H<sub>2</sub>O + 0.2% TFA, RT 12.52 min) afforded the product (27 mg, 57 μmol, 56%). TLC: <i>R</i><sub>f</sub> = 0.5 (60% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.94 (br s, 1H), 7.43–7.32 (m, 2H), 7.32–7.27 (m, 3H), 6.83 (s, 1H), 4.78 (t, <i>J</i> = 11.0 Hz, 2H), 4.19 (br s, 4H), 3.38–3.19 (m, 2H), 3.12–2.99 (m, 4H), 2.94 (t, <i>J</i> = 12.2 Hz, 2H), 2.82–2.68 (m, 1H), 2.10 (dd, <i>J</i> = 17.2, 4.5 Hz, 1H), 1.88 (d, <i>J</i> = 13.1 Hz, 1H), 1.85–1.72 (m, 1H), 1.72–1.57 (m, 1H), 1.11–0.96 (m, 1H), 0.54 (q, <i>J</i> = 5.3 Hz, 2H), 0.28 (q, <i>J</i> = 4.9 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 164.07, 162.85, 160.95, 158.16, 143.95, 128.77, 127.24, 126.85, 90.31, 51.54, 51.28, 44.77, 44.14, 43.08, 42.71, 32.18, 25.50, 10.92, 3.52, 3.50. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>3</sub>S + H]<sup>+</sup>: 470.2220 calculated, 470.2223 found.</div></div><div id="sec4_2_113" class="NLM_sec NLM_sec_level_3"><div id="ac_i142" class="anchor-spacer"></div><h4 class="article-section__title" id="_i142"> <i>N</i>-(Cyclopropylmethyl)-6-((<i>R</i>)-3-hydroxypyrrolidin-1-yl)-2-((<i>S</i>)-3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>105</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119z</b> (24 mg, 81 μmol, 1 equiv), DiPEA (42 μL, 0.24 mmol, 3 equiv), and (<i>S</i>)-3-phenylpiperidine (17 mg, 0.11 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (70% → 90% EtOAc/pentane) afforded the product (23 mg, 55 μmol, 67%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (br s, 1H), 7.39–7.22 (m, 5H), 6.54 (s, 1H), 4.84 (t, <i>J</i> = 14.0 Hz, 2H), 4.57 (s, 1H), 3.86–3.37 (m, 4H), 3.37–3.17 (m, 2H), 2.87 (t, <i>J</i> = 12.0 Hz, 2H), 2.81–2.70 (m, 1H), 2.16–1.93 (m, 3H), 1.90–1.54 (m, 4H), 1.11–0.97 (m, 1H), 0.59–0.45 (m, 2H), 0.27 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.93, 162.20, 160.91, 155.71, 144.42, 128.64, 127.33, 126.60, 91.76, 54.96, 51.30, 44.67, 44.38, 44.07, 42.59, 32.24, 25.59, 10.93, 3.52. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 422.2551 calculated, 422.2551 found.</div></div><div id="sec4_2_114" class="NLM_sec NLM_sec_level_3"><div id="ac_i143" class="anchor-spacer"></div><h4 class="article-section__title" id="_i143"> <i>N</i>-(Cyclopropylmethyl)-6-((<i>S</i>)-3-hydroxypyrrolidin-1-yl)-2-((<i>R</i>)-3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>106</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119aa</b> (32 mg, 0.11 mmol, 1 equiv), DiPEA (56 μL, 0.32 mmol, 3 equiv), and (<i>R</i>)-3-phenylpiperidine (23 mg, 0.14 mmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (70% → 100% EtOAc/pentane) afforded the product (26 mg, 62 μmol, 56%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (t, <i>J</i> = 5.7 Hz, 1H), 7.40–7.19 (m, 5H), 6.53 (s, 1H), 4.84 (t, <i>J</i> = 14.3 Hz, 2H), 4.57 (s, 1H), 3.90–3.37 (m, 4H), 3.37–3.17 (m, 2H), 2.94–2.81 (m, 2H), 2.81–2.69 (m, 1H), 2.16–1.96 (m, 3H), 1.89–1.57 (m, 4H), 1.10–0.98 (m, 1H), 0.60–0.42 (m, 2H), 0.26 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.84, 162.10, 160.81, 155.58, 144.32, 128.52, 127.22, 126.48, 91.64, 70.84, 70.16, 54.84, 51.19, 44.55, 44.27, 43.95, 42.47, 32.13, 25.48, 10.82, 3.40. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 422.2551 calculated, 422.2552 found.</div></div><div id="sec4_2_115" class="NLM_sec NLM_sec_level_3"><div id="ac_i144" class="anchor-spacer"></div><h4 class="article-section__title" id="_i144"> <i>N</i>-(Cyclopropylmethyl)-6-((<i>R</i>)-3-hydroxypyrrolidin-1-yl)-2-((<i>R</i>)-3-phenylpiperidin-1-yl)pyrimidine-4-carboxamide (<b>107</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_2" aria-label="General Procedure A">General Procedure A</a> using 2-chloropyrimidine <b>119z</b> (22 mg, 74 μmol, 1 equiv), DiPEA (39 μL, 0.22 mmol, 3 equiv), and (<i>R</i>)-3-phenylpiperidine (16 mg, 96 μmol, 1.3 equiv). Total heating time: 4 h at 160 °C with μW irradiation. Column chromatography (70% → 90% EtOAc/pentane) afforded the product (23 mg, 55 μmol, 74%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.03 (t, <i>J</i> = 5.7 Hz, 1H), 7.40–7.19 (m, 5H), 6.53 (s, 1H), 4.84 (t, <i>J</i> = 14.3 Hz, 2H), 4.57 (s, 1H), 3.90–3.37 (m, 4H), 3.37–3.17 (m, 2H), 2.94–2.81 (m, 2H), 2.81–2.69 (m, 1H), 2.56 (s, 1H), 2.07 (d, <i>J</i> = 12.7 Hz, 3H), 1.89–1.57 (m, 4H), 1.10–0.98 (m, 1H), 0.60–0.42 (m, 2H), 0.26 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.96, 162.19, 160.91, 155.67, 144.43, 128.62, 127.32, 126.59, 91.75, 70.36, 54.95, 51.36, 44.68, 44.38, 44.06, 42.54, 32.20, 25.60, 10.92, 3.51. HRMS [C<sub>24</sub>H<sub>31</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>: 422.2551 calculated, 422.2552 found.</div></div><div id="sec4_2_116" class="NLM_sec NLM_sec_level_3"><div id="ac_i145" class="anchor-spacer"></div><h4 class="article-section__title" id="_i145"> Methyl 4-Chloro-6-(methyl(phenethyl)amino)picolinate (<b>109</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with methyl 4,6-dichloropicolinate (<b>108</b>) (206 mg, 0.99 mmol, 1 equiv), <i>N</i>-methylphenethylamine HBr salt (218 mg, 1.01 mmol, 1.02 equiv), DiPEA (436 μL, 2.5 mmol, 2.5 equiv), and dry MeOH (2 mL). The solution was stirred at rt for 3 days and then refluxed for 24 h. The reaction mixture was concentrated under reduced pressure, and the residue was purified by silica gel column chromatography (5% → 35% EtOAc/pentane), affording the product (122 mg, 0.40 mmol, 41%). TLC: <i>R</i><sub>f</sub> = 0.2 (5% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.35–7.26 (m, 3H), 7.23 (t, <i>J</i> = 7.3 Hz, 1H), 7.16 (d, <i>J</i> = 7.0 Hz, 2H), 6.56 (d, <i>J</i> = 2.3 Hz, 1H), 3.96 (s, 3H), 3.63 (t, <i>J</i> = 7.3 Hz, 2H), 2.93–2.81 (m, 5H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.68, 155.45, 152.41, 147.85, 138.19, 128.84, 128.80, 126.84, 107.83, 107.79, 53.84, 53.05, 38.50, 33.14. Regioselectivity was confirmed by <sup>1</sup>H and <sup>13</sup>C-HMBC and <sup>1</sup>H-NOESY 2D NMR. HRMS [C<sub>16</sub>H<sub>17</sub>ClN<sub>2</sub>O<sub>2</sub> + H]<sup>+</sup>: 305.1051 calculated, 305.1054 found.</div></div><div id="sec4_2_117" class="NLM_sec NLM_sec_level_3"><div id="ac_i146" class="anchor-spacer"></div><h4 class="article-section__title" id="_i146"> 4-Chloro-6-(methyl(phenethyl)amino)picolinic Acid (<b>110</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with methyl ester <b>109</b> (122 mg, 0.4 mmol, 1 equiv) and THF (2 mL). An aqueous 1.5 M NaOH solution (0.53 mL, 0.8 mmol, 2 equiv) was added dropwise, and the reaction was stirred for 1.5 h at rt. The mixture was cooled to 0 °C and acidified to pH 1 by dropwise addition of 37% w/w aq. HCl. The mixture was then extracted with DCM (3× 5 mL), and the combined organic layers were washed with brine (1× 15 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure to afford the product (104 mg, 0.36 mmol, 89%). TLC <i>R</i><sub>f</sub> = 0.1 (5% MeOH/DCM with 3 drops of AcOH). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.92 (br s, 1H), 7.37–7.20 (m, 4H), 7.20–7.12 (m, 2H), 6.57 (d, <i>J</i> = 2.4 Hz, 1H), 3.66 (t, <i>J</i> = 7.2 Hz, 2H), 2.96–2.81 (m, 5H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.25, 156.33, 150.90, 146.25, 138.01, 128.96, 128.86, 127.01, 108.27, 105.87, 54.04, 38.78, 33.21.</div></div><div id="sec4_2_118" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> 4-Chloro-<i>N-</i>(cyclopropylmethyl)-6-(methyl(phenethyl)amino)picolinamide (<b>111</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with carboxylic acid <b>110</b> (104 mg, 0.36 mmol, 1 equiv), HOBt (73 mg, 0.47 mmol, 1.3 equiv), EDC hydrochloride (102 mg, 0.53 mmol, 1.5 equiv), and dry DCM (1.8 mL). The suspension was stirred for 1 h at rt followed by the addition of cyclopropylmethanamine (37 μL, 0.43 mmol, 1.2 equiv). After stirring for 20 h, the solvent was removed under reduced pressure and the residue was dissolved in EtOAc (10 mL) and washed with 1 M aq. HCl (1× 10 mL), sat. aq. NaHCO<sub>3</sub> (1× 10 mL), and brine (1× 10 mL). The organic layer was dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The crude material was purified by silica gel column chromatography (isocratic, 30% EtOAc/pentane), affording the product (30 mg, 87 μmol, 9%), <i>R</i><sub>f</sub> = 0.35 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.99 (t, <i>J</i> = 5.9 Hz, 1H), 7.43 (d, <i>J</i> = 2.4 Hz, 1H), 7.35–7.26 (m, 2H), 7.28–7.18 (m, 1H), 7.20–7.15 (m, 2H), 6.50 (d, <i>J</i> = 2.4 Hz, 1H), 3.64 (t, <i>J</i> = 7.3 Hz, 2H), 3.30 (dd, <i>J</i> = 7.1, 5.9 Hz, 2H), 2.91–2.84 (m, 5H), 1.13–1.01 (m, 1H), 0.58–0.52 (m, 2H), 0.31–0.26 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.90, 155.94, 151.08, 150.38, 138.41, 128.93, 128.90, 126.86, 106.84, 104.87, 54.02, 44.51, 38.71, 33.25, 10.92, 3.76.</div></div><div id="sec4_2_119" class="NLM_sec NLM_sec_level_3"><div id="ac_i148" class="anchor-spacer"></div><h4 class="article-section__title" id="_i148"> Methyl 2-Chloro-6-morpholinoisonicotinate (<b>113</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with methyl-2,6-dichloroisonicotinate (<b>112</b>) (410 mg, 1.99 mmol, 1 equiv), K<sub>2</sub>CO<sub>3</sub> (549 mg, 3.97 mmol, 2 equiv), morpholine (259 μL, 3.00 mmol, 1.5 equiv), and dry CH<sub>3</sub>CN (10 mL). The mixture was heated to reflux. After 45 h, the reaction was complete as judged by TLC and cooled to room temperature. The mixture was filtered, and the filtrate was concentrated under reduced pressure. The crude material was purified by silica gel column chromatography (10% → 30% EtOAc/pentane), affording the product (323 mg, 1.26 mmol, 66%). TLC: <i>R</i><sub>f</sub> = 0.6 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.13 (d, <i>J</i> = 0.9 Hz, 1H), 7.07 (d, <i>J</i> = 1.0 Hz, 1H), 3.92 (s, 3H), 3.86–3.76 (m, 4H), 3.64–3.52 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.16, 159.40, 150.36, 141.33, 111.77, 104.69, 66.55, 52.83, 45.26. HRMS [C<sub>11</sub>H<sub>13</sub>ClN<sub>2</sub>O<sub>3</sub> + H]<sup>+</sup>: 257.0688 calculated, 257.0690 found.</div></div><div id="sec4_2_120" class="NLM_sec NLM_sec_level_3"><div id="ac_i149" class="anchor-spacer"></div><h4 class="article-section__title" id="_i149"> 2-Chloro-6-morpholinoisonicotinic Acid (<b>114</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with methyl ester <b>113</b> (323 mg, 1.26 mmol, 1 equiv) and THF (5 mL). A 1 M aqueous solution of NaOH (2.52 mL, 2.52 mmol, 2 equiv) was added dropwise. After stirring for 20 min at room temperature, the reaction mixture was acidified carefully with 37% w/w HCl to pH 1, and THF was removed under reduced pressure. The mixture was extracted with DCM (3× 10 mL), the combined organic layers were washed with brine (1× 15 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure affording the product (325 mg, 1.26 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.2 (30% EtOAc/pentane with 3 drops of AcOH). <sup>1</sup>H NMR (400 MHz, MeOD + CDCl<sub>3</sub>) δ 7.12 (s, 1H), 7.09 (s, 1H), 3.78 (t, <i>J</i> = 4.9 Hz, 4H), 3.54 (t, <i>J</i> = 4.9 Hz, 4H). <sup>13</sup>C NMR (101 MHz, MeOD + CDCl<sub>3</sub>) δ 167.10, 160.29, 150.79, 143.21, 112.56, 105.74, 67.18, 45.94. HRMS [C<sub>10</sub>H<sub>11</sub>ClN<sub>2</sub>O<sub>3</sub> + H]<sup>+</sup>: 243.0531 calculated, 243.0533 found.</div></div><div id="sec4_2_121" class="NLM_sec NLM_sec_level_3"><div id="ac_i150" class="anchor-spacer"></div><h4 class="article-section__title" id="_i150"> 2-Chloro-<i>N-</i>(cyclopropylmethyl)-6-morpholinoisonicotinamide (<b>115</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with carboxylic acid <b>114</b> (325 mg, 1.26 mmol, 1 equiv), EDC hydrochloride (302 mg, 1.95 mmol, 1.5 equiv), and HOBt (298 mg, 1.95 mmol, 1.5 equiv) and dry DCM (7 mL). The suspension was stirred for 1 h at room temperature followed by addition of cyclopropylmethanamine (139 μL, 1.6 mmol, 1.2 equiv). After 20 h, DCM was removed under reduced pressure and the residue was dissolved in EtOAc (15 mL) and sequentially washed with 1 M HCl (aq) (2× 15 mL), sat. aq. NaHCO<sub>3</sub> (2× 15 mL), and brine (1× 20 mL). The organic layer was dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The resulting crude material was purified by silica gel column chromatography (30% → 60% EtOAc/pentane) affording the product (296 mg, 1 mmol, 80%). TLC: <i>R</i><sub>f</sub> = 0.3 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.91 (s, 1H), 6.81 (s, 1H), 6.45–6.31 (m, 1H), 3.84–3.73 (m, 4H), 3.66–3.51 (m, 4H), 3.28 (dd, <i>J</i> = 7.2, 5.4 Hz, 2H), 1.12–0.98 (m, 1H), 0.64–0.52 (m, 2H), 0.38–0.21 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.28, 159.52, 150.25, 146.27, 109.10, 103.25, 66.61, 45.27, 45.25, 10.68, 3.73. HRMS [C<sub>14</sub>H<sub>18</sub>ClN<sub>3</sub>O<sub>2</sub> + H]<sup>+</sup>: 296.1160 calculated, 296.1158 found.</div></div><div id="sec4_2_122" class="NLM_sec NLM_sec_level_3"><div id="ac_i151" class="anchor-spacer"></div><h4 class="article-section__title" id="_i151"> 2,6-Dichloropyrimidine-4-carbonyl chloride (<b>117</b>)</h4><div class="NLM_p last">In a 500 mL round-bottom flask, orotic acid (<b>116</b>) (15.6 g, 100 mmol, 1 equiv) was dissolved in phosphorous oxychloride (46 mL, 500 mmol, 5 equiv.) and 10 drops of DMF were added. The mixture was heated to reflux and stirred for 19 h. <i>n</i>-Hexane (250 mL) was added, and the mixture was stirred vigorously for 10 min and then transferred to a separatory funnel containing 100 mL of H<sub>2</sub>O. The flask was washed with 50 mL of hexane. After shaking, the aqueous layer was removed, and the organic layer was washed with brine (1× 100 mL), dried (MgSO<sub>4</sub>), and concentrated under reduced pressure to yield the product (12.8 g, 60.4 mmol, 60%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.00 (s, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 167.17, 165.27, 161.56, 158.19, 119.70.</div></div><div id="sec4_2_123" class="NLM_sec NLM_sec_level_3"><div id="ac_i152" class="anchor-spacer"></div><h4 class="article-section__title" id="_i152"> 2,6-Dichloro-<i>N</i>-(cyclopropylmethyl)pyrimidine-4-carboxamide (<b>118a</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.63 mL, 5 mmol, 1 equiv), Et<sub>3</sub>N (0.91 mL, 6.5 mmol, 1.3 equiv), and cyclopropylmethanamine (444 μL, 5.13 mmol, 1.025 equiv). Column chromatography (5% → 20% EtOAc/pentane) afforded the product (0.99 g, 4.0 mmol, 81%). TLC: <i>R</i><sub>f</sub> = 0.8 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.11 (s, 1H), 7.96 (br s, 1H), 3.44–3.27 (m, 2H), 1.20–1.03 (m, 1H), 0.68–0.52 (m, 2H), 0.32 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.74, 160.44, 160.04, 159.77, 118.22, 44.72, 10.57, 3.67. HRMS [C<sub>9</sub>H<sub>9</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 246.0195 calculated, 246.0196 found.</div></div><div id="sec4_2_124" class="NLM_sec NLM_sec_level_3"><div id="ac_i153" class="anchor-spacer"></div><h4 class="article-section__title" id="_i153"> 2,6-Dichloro-<i>N</i>-methylpyrimidine-4-carboxamide (<b>118b</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.63 mL, 5.0 mmol, 1 equiv), Et<sub>3</sub>N (1.6 mL, 11.5 mmol, 2.3 equiv), and methylamine HCl salt (0.35 g, 5.13 mmol, 1.025 equiv). Column chromatography (10% → 30% EtOAc/pentane) afforded the product (0.97 g, 4.7 mmol, 94%). TLC: <i>R</i><sub>f</sub> = 0.3 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (s, 1H), 7.90 (br s, 1H), 3.08 (d, <i>J</i> = 5.1 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.84, 160.85, 160.25, 159.82, 118.11, 26.51. HRMS [C<sub>6</sub>H<sub>5</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 205.9882 calculated, 205.9884 found.</div></div><div id="sec4_2_125" class="NLM_sec NLM_sec_level_3"><div id="ac_i154" class="anchor-spacer"></div><h4 class="article-section__title" id="_i154"> 2,6-Dichloro-<i>N</i>-ethylpyrimidine-4-carboxamide (<b>118c</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.63 mL, 5 mmol, 1 equiv), Et<sub>3</sub>N (1.6 mL, 11.5 mmol, 2.3 equiv), and ethylamine HCl salt (0.42 g, 5.13 mmol, 1.025 equiv). Column chromatography (5% → 20% EtOAc/pentane) afforded the product (0.88 g, 4.0 mmol, 80%). TLC: <i>R</i><sub>f</sub> = 0.6 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (s, 1H), 7.83 (br s, 1H), 3.65–3.38 (m, 2H), 1.30 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.84, 160.47, 160.06, 159.83, 118.18, 34.86, 14.63. HRMS [C<sub>7</sub>H<sub>7</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 220.0039 calculated, 220.0040 found.</div></div><div id="sec4_2_126" class="NLM_sec NLM_sec_level_3"><div id="ac_i155" class="anchor-spacer"></div><h4 class="article-section__title" id="_i155"> 2,6-Dichloro-<i>N</i>-butyl-pyrimidine-4-carboxamide (<b>118d</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.25 mL, 2.0 mmol, 1 equiv), Et<sub>3</sub>N (0.36 mL, 2.60 mmol, 1.3 equiv), and <i>n</i>-butylamine (0.20 mL, 2.05 mmol, 1.025 equiv). Column chromatography (5% → 20% EtOAc/pentane) afforded the product (0.50 g, 2.0 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.7 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (s, 1H), 7.85 (br s, 1H), 3.49 (q, <i>J</i> = 7.0 Hz, 2H), 1.74–1.56 (m, 2H), 1.53–1.35 (m, 2H), 0.97 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.78, 160.46, 160.13, 159.78, 118.18, 39.64, 31.40, 20.09, 13.72. HRMS [C<sub>9</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 248.0352 calculated, 248.0354 found.</div></div><div id="sec4_2_127" class="NLM_sec NLM_sec_level_3"><div id="ac_i156" class="anchor-spacer"></div><h4 class="article-section__title" id="_i156"> 2,6-Dichloro-<i>N</i>-hexyl-pyrimidine-4-carboxamide (<b>118e</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.25 mL, 2.0 mmol, 1 equiv), Et<sub>3</sub>N (0.36 mL, 2.6 mmol, 1.3 equiv), and <i>n</i>-hexylamine (0.27 mL, 2.05 mmol, 1.025 equiv). Column chromatography (5% → 20% EtOAc/pentane) afforded the product (0.58 g, 2.0 mmol, 99%). TLC: R<sub>f</sub> = 0.6 (10% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (s, 1H), 7.89 (br s, 1H), 3.48 (q, <i>J</i> = 7.0 Hz, 2H), 1.66 (p, <i>J</i> = 7.8, 7.4 Hz, 2H), 1.46–1.23 (m, 6H), 0.98–0.83 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.71, 160.46, 160.06, 159.73, 118.14, 39.91, 31.38, 29.30, 26.55, 22.48, 13.96. HRMS [C<sub>11</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 276.0665 calculated, 276.0668 found.</div></div><div id="sec4_2_128" class="NLM_sec NLM_sec_level_3"><div id="ac_i157" class="anchor-spacer"></div><h4 class="article-section__title" id="_i157"> 2,6-Dichloro-<i>N</i>-isobutylpyrimidine-4-carboxamide (<b>118f</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.25 mL, 2.0 mmol, 1 equiv), Et<sub>3</sub>N (0.36 mL, 2.60 mmol, 1.3 equiv), and isobutylamine (0.20 mL, 2.05 mmol, 1.025 equiv). Column chromatography (5% → 20% EtOAc/pentane) afforded the product (0.50 g, 2.0 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.8 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.11 (s, 1H), 7.94 (br s, 1H), 3.33 (t, <i>J</i> = 6.6 Hz, 2H), 2.07–1.87 (m, 1H), 1.00 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.68, 160.41, 160.15, 159.70, 118.17, 47.10, 28.52, 20.07. HRMS [C<sub>9</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 248.0352 calculated, 248.0354 found.</div></div><div id="sec4_2_129" class="NLM_sec NLM_sec_level_3"><div id="ac_i158" class="anchor-spacer"></div><h4 class="article-section__title" id="_i158"> 2,6-Dichloro-<i>N</i>-neopentylpyrimidine-4-carboxamide (<b>118g</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.25 mL, 2.0 mmol, 1 equiv), Et<sub>3</sub>N (0.36 mL, 2.6 mmol, 1.3 equiv), and neopentylamine (0.24 mL, 2.05 mmol, 1.025 equiv). Column chromatography (5% → 20% EtOAc/pentane) afforded the product (0.58 g, 2.0 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.9 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.11 (s, 1H), 7.91 (br s, 1H), 3.30 (d, <i>J</i> = 6.7 Hz, 2H), 1.01 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.69, 160.38, 160.22, 159.71, 118.24, 50.89, 32.35, 27.20. HRMS [C<sub>10</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 262.0508 calculated, 262.0510 found.</div></div><div id="sec4_2_130" class="NLM_sec NLM_sec_level_3"><div id="ac_i159" class="anchor-spacer"></div><h4 class="article-section__title" id="_i159"> 2,6-Dichloro-<i>N</i>-(prop-2-yn-1-yl)pyrimidine-4-carboxamide (<b>118h</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.25 mL, 2.0 mmol, 1 equiv), Et<sub>3</sub>N (0.36 mL, 2.60 mmol, 1.3 equiv), and propargylamine (0.13 mL, 2.05 mmol, 1.025 equiv). Column chromatography (5% → 20% EtOAc/pentane) afforded the product (0.44 g, 1.91 mmol, 96%). TLC: <i>R</i><sub>f</sub> = 0.6 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.24–8.00 (m, 2H), 4.30 (dd, <i>J</i> = 5.7, 2.6 Hz, 2H), 2.36 (t, <i>J</i> = 2.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.92, 160.08, 159.94, 159.63, 118.35, 78.30, 72.42, 29.52. HRMS [C<sub>8</sub>H<sub>5</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 229.9882 calculated, 229.9884 found.</div></div><div id="sec4_2_131" class="NLM_sec NLM_sec_level_3"><div id="ac_i160" class="anchor-spacer"></div><h4 class="article-section__title" id="_i160"> Methyl (2,6-Dichloropyrimidine-4-carbonyl)glycinate (<b>118i</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.32 mL, 2.5 mmol, 1 equiv), Et<sub>3</sub>N (0.80 mL, 5.75 mmol, 2.3 equiv), and glycine methylester HCl salt (0.32 g, 2.56 mmol, 1.025 equiv). Column chromatography (5% → 20% EtOAc/pentane) afforded the product (0.51 g, 1.95 mmol, 78%). TLC: <i>R</i><sub>f</sub> = 0.5 (30% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.31 (br s, 1H), 8.09 (s, 1H), 4.28 (d, <i>J</i> = 5.8 Hz, 2H), 3.81 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 169.30, 164.89, 160.65, 160.02, 159.55, 118.31, 52.66, 41.35. HRMS [C<sub>8</sub>H<sub>7</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub> + H]<sup>+</sup>: 263.9937 calculated, 263.9939 found.</div></div><div id="sec4_2_132" class="NLM_sec NLM_sec_level_3"><div id="ac_i161" class="anchor-spacer"></div><h4 class="article-section__title" id="_i161"> 2,6-Dichloropyrimidine-<i>N-</i>benzyl-4-carboxamide (<b>118j</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.63 mL, 5.0 mmol, 1 equiv), Et<sub>3</sub>N (0.91 mL, 6.5 mmol, 1.3 equiv), and benzylamine (0.56 mL, 5.13 mmol, 1.025 equiv). Column chromatography (10% → 25% EtOAc/pentane) afforded the product (1.38 g, 4.9 mmol, 98%). TLC: <i>R</i><sub>f</sub> = 0.7 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.18 (br s, 1H), 8.08 (s, 1H), 7.39–7.23 (m, 5H), 4.63 (d, <i>J</i> = 6.2 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.81, 160.20, 160.17, 159.81, 137.00, 128.81, 127.90, 127.86, 118.35, 43.82. HRMS [C<sub>12</sub>H<sub>9</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 282.0195 calculated, 282.0197 found.</div></div><div id="sec4_2_133" class="NLM_sec NLM_sec_level_3"><div id="ac_i162" class="anchor-spacer"></div><h4 class="article-section__title" id="_i162"> 2,6-Dichloropyrimidine-<i>N</i>-([1,1′-biphenyl]-4-ylmethyl)-4-carboxamide (<b>118k</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_5" aria-label="General Procedure D">General Procedure D</a> using 2,6-dichloropyrimidine-4-carbonyl chloride <b>117</b> (0.63 mL, 5.0 mmol, 1 equiv), Et<sub>3</sub>N (0.91 mL, 6.5 mmol, 1.3 equiv), and 4-phenylbenzylamine (0.94 g, 5.13 mmol, 1.025 equiv). Column chromatography (10% → 30% EtOAc/pentane) afforded the product (1.46 g, 4.1 mmol, 81%). TLC: <i>R</i><sub>f</sub> = 0.8 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.16 (t, <i>J</i> = 5.7 Hz, 1H), 8.07 (s, 1H), 7.54 (d, <i>J</i> = 7.9 Hz, 4H), 7.46–7.29 (m, 5H), 4.65 (d, <i>J</i> = 6.2 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.86, 160.26, 160.14, 159.86, 140.82, 140.40, 136.01, 128.84, 128.43, 127.52, 127.49, 127.03, 118.37, 43.57. HRMS [C<sub>18</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>3</sub>O + H]<sup>+</sup>: 358.0508 calculated, 358.0508 found.</div></div><div id="sec4_2_134" class="NLM_sec NLM_sec_level_3"><div id="ac_i163" class="anchor-spacer"></div><h4 class="article-section__title" id="_i163"> 2-Chloro-<i>N</i>-methyl-6-morpholinopyrimidine-4-carboxamide (<b>119a</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118b</b> (0.31 g, 1.5 mmol, 1 equiv), DiPEA (0.39 mL, 2.25 mmol, 1.5 equiv), and morpholine (0.14 mL, 1.58 mmol, 1.05 equiv). Column chromatography (40% → 80% EtOAc/pentane) afforded the product as an 8.5:1 mixture of regioisomers (0.39 g, 1.5 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.4 (100% EtOAc/pentane). Major regioisomer: <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.93–7.77 (m, 1H), 7.26 (s, 1H), 3.83–3.61 (m, 8H), 3.00 (d, <i>J</i> = 5.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.82, 162.92, 159.79, 157.70, 99.16, 66.34, 44.60, 44.38, 26.19. HRMS [C<sub>10</sub>H<sub>13</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 257.0800 calculated, 257.0800 found.</div></div><div id="sec4_2_135" class="NLM_sec NLM_sec_level_3"><div id="ac_i164" class="anchor-spacer"></div><h4 class="article-section__title" id="_i164"> 2-Chloro-<i>N</i>-ethyl-6-morpholinopyrimidine-4-carboxamide (<b>119b</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118c</b> (0.42 g, 1.89 mmol, 1 equiv), DiPEA (0.49 mL, 2.84 mmol, 1.5 equiv), and morpholine (0.17 mL, 1.98 mmol, 1.05 equiv). Column chromatography (50% → 70% EtOAc/pentane) afforded the product (0.48 g, 1.77 mmol, 94%). TLC: <i>R</i><sub>f</sub> = 0.2 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.82 (br s, 1H), 7.27 (s, 1H), 3.99–3.55 (m, 8H), 3.54–3.37 (m, 2H), 1.25 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.86, 162.17, 159.85, 157.93, 99.27, 66.39, 44.63, 34.53, 14.71. HRMS [C<sub>11</sub>H<sub>15</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 271.0956 calculated, 271.0957 found.</div></div><div id="sec4_2_136" class="NLM_sec NLM_sec_level_3"><div id="ac_i165" class="anchor-spacer"></div><h4 class="article-section__title" id="_i165"> 2-Chloro-<i>N</i>-butyl-6-morpholinopyrimidine-4-carboxamide (<b>119c</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118d</b> (193 mg, 0.78 mmol, 1 equiv), DiPEA (203 μL, 1.17 mmol, 1.5 equiv), and morpholine (71 μL, 0.82 mmol, 1.05 equiv). Column chromatography (40% → 60% EtOAc/pentane) afforded the product (212 mg, 0.71 mmol, 91%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.84 (br s, 1H), 7.27 (s, 1H), 3.91–3.61 (m, 8H), 3.43 (q, <i>J</i> = 7.1 Hz, 2H), 1.70–1.51 (m, 2H), 1.48–1.34 (m, 2H), 0.95 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.79, 162.19, 159.75, 157.88, 99.21, 66.31, 44.57, 39.29, 31.48, 20.07, 13.72. HRMS [C<sub>13</sub>H<sub>19</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 299.1269 calculated, 299.1269 found.</div></div><div id="sec4_2_137" class="NLM_sec NLM_sec_level_3"><div id="ac_i166" class="anchor-spacer"></div><h4 class="article-section__title" id="_i166"> 2-Chloro-<i>N</i>-hexyl-6-morpholinopyrimidine-4-carboxamide (<b>119d</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118e</b> (0.28 g, 1.02 mmol, 1 equiv), DiPEA (0.27 mL, 1.54 mmol, 1.5 equiv), and morpholine (94 μL, 1.08 mmol, 1.05 equiv). Column chromatography (40% → 70% EtOAc/pentane) afforded the product (0.31 g, 0.95 mmol, 93%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.87 (t, <i>J</i> = 5.8 Hz, 1H), 7.28 (s, 1H), 4.01–3.53 (m, 8H), 3.42 (q, <i>J</i> = 6.9 Hz, 2H), 1.61 (p, <i>J</i> = 7.7, 7.3 Hz, 2H), 1.46–1.22 (m, 6H), 0.99–0.80 (m, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.71, 162.11, 159.68, 157.81, 99.17, 66.25, 44.55, 39.56, 31.37, 29.34, 26.53, 22.45, 13.97. HRMS [C<sub>15</sub>H<sub>23</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 327.1582 calculated, 327.1582 found.</div></div><div id="sec4_2_138" class="NLM_sec NLM_sec_level_3"><div id="ac_i167" class="anchor-spacer"></div><h4 class="article-section__title" id="_i167"> 2-Chloro-<i>N</i>-isobutyl-6-morpholinopyrimidine-4-carboxamide (<b>119e</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118f</b> (275 mg, 1.11 mmol, 1 equiv), DiPEA (290 μL, 1.66 mmol, 1.5 equiv), and morpholine (101 μL, 1.16 mmol, 1.05 equiv). Column chromatography (40% → 60% EtOAc/pentane) afforded the product (330 mg, 1.11 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91 (t, <i>J</i> = 5.8 Hz, 1H), 7.28 (s, 1H), 4.01–3.52 (m, 8H), 3.26 (t, <i>J</i> = 6.7 Hz, 2H), 2.00–1.81 (m, 1H), 0.97 (d, <i>J</i> = 6.7 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.75, 162.25, 159.71, 157.83, 99.23, 66.26, 46.82, 44.56, 28.58, 20.11. HRMS [C<sub>13</sub>H<sub>19</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 299.1269 calculated, 299.1270 found.</div></div><div id="sec4_2_139" class="NLM_sec NLM_sec_level_3"><div id="ac_i168" class="anchor-spacer"></div><h4 class="article-section__title" id="_i168"> 2-Chloro-6-morpholino-<i>N</i>-neopentylpyrimidine-4-carboxamide (<b>119f</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118g</b> (0.25 g, 0.96 mmol, 1 equiv), DiPEA (0.25 mL, 1.44 mmol, 1.5 equiv), and morpholine (88 μL, 1.01 mmol, 1.05 equiv). Column chromatography (30% → 60% EtOAc/pentane) afforded the product (0.30 g, 0.96 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.5 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.92 (br s, 1H), 7.28 (s, 1H), 3.96–3.50 (m, 8H), 3.24 (d, <i>J</i> = 6.7 Hz, 2H), 0.98 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.77, 162.36, 159.74, 157.82, 99.30, 66.27, 50.64, 44.60, 32.34, 27.23. HRMS [C<sub>14</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 313.1426 calculated, 313.1424 found.</div></div><div id="sec4_2_140" class="NLM_sec NLM_sec_level_3"><div id="ac_i169" class="anchor-spacer"></div><h4 class="article-section__title" id="_i169"> 2-Chloro-6-morpholino-<i>N</i>-(prop-2-yn-1-yl)pyrimidine-4-carboxamide (<b>119g</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118h</b> (221 mg, 0.96 mmol, 1 equiv), DiPEA (251 μL, 1.44 mmol, 1.5 equiv), and morpholine (88 μL, 1.01 mmol, 1.05 equiv). Column chromatography (40% → 60% EtOAc/pentane) afforded the product (249 mg, 0.89 mmol, 92%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.00 (br s, 1H), 7.26 (s, 1H), 4.22 (dd, <i>J</i> = 5.7, 2.6 Hz, 2H), 3.88–3.59 (m, 8H), 2.29 (t, <i>J</i> = 2.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.84, 162.23, 160.04, 157.20, 99.53, 78.84, 72.05, 66.41, 29.32. HRMS [C<sub>12</sub>H<sub>13</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 281.0800 calculated, 281.0800 found.</div></div><div id="sec4_2_141" class="NLM_sec NLM_sec_level_3"><div id="ac_i170" class="anchor-spacer"></div><h4 class="article-section__title" id="_i170"> Methyl (2-Chloro-6-morpholinopyrimidine-4-carbonyl)glycinate (<b>119h</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118i</b> (396 mg, 1.50 mmol, 1 equiv), DiPEA (392 μL, 2.25 mmol, 1.5 equiv), and morpholine (137 μL, 1.58 mmol, 1.05 equiv). Column chromatography (60% → 80% EtOAc/pentane) afforded the product (298 mg, 0.95 mmol, 63%). TLC: <i>R</i><sub>f</sub> = 0.5 (70% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.25 (t, <i>J</i> = 5.4 Hz, 1H), 7.25 (s, 1H), 4.22 (d, <i>J</i> = 5.9 Hz, 2H), 3.90–3.61 (m, 11H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 169.59, 163.83, 162.85, 160.06, 157.09, 99.50, 66.39, 52.51, 44.69, 41.29. HRMS [C<sub>12</sub>H<sub>15</sub>ClN<sub>4</sub>O<sub>4</sub> + H]<sup>+</sup>: 315.0855 calculated, 315.0851 found.</div></div><div id="sec4_2_142" class="NLM_sec NLM_sec_level_3"><div id="ac_i171" class="anchor-spacer"></div><h4 class="article-section__title" id="_i171"> 2-Chloro-<i>N</i>-benzyl-6-morpholinopyrimidine-4-carboxamide (<b>119i</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118j</b> (0.71 g, 2.50 mmol, 1 equiv), DiPEA (0.65 mL, 3.75 mmol, 1.5 equiv), and morpholine (0.23 mL, 2.63 mmol, 1.05 equiv). Column chromatography (30% → 60% EtOAc/pentane) afforded the product (0.68 g, 2.03 mmol, 81%). TLC: <i>R</i><sub>f</sub> = 0.6 (60% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.18 (br s, 1H), 7.39–7.24 (m, 6H), 4.61 (d, <i>J</i> = 6.2 Hz, 2H), 3.90–3.54 (m, 8H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.79, 162.37, 159.88, 157.63, 137.59, 128.77, 127.92, 127.67, 99.52, 66.37, 44.47, 43.60. HRMS [C<sub>16</sub>H<sub>17</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 333.1113 calculated, 333.1112 found.</div></div><div id="sec4_2_143" class="NLM_sec NLM_sec_level_3"><div id="ac_i172" class="anchor-spacer"></div><h4 class="article-section__title" id="_i172"> 2-Chloro-<i>N</i>-([1,1’-biphenyl]-4-ylmethyl)-6-morpholinopyrimidine-4-carboxamide (<b>119j</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118k</b> (0.31 g, 1.0 mmol, 1 equiv), DiPEA (0.26 mL, 1.5 mmol, 1.5 equiv), and morpholine (91 μL, 1.05 mmol, 1.05 equiv). Column chromatography (20% → 50% EtOAc/pentane) afforded the product (0.38 g, 0.94 mmol, 94%). TLC: <i>R</i><sub>f</sub> = 0.4 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.24 (t, <i>J</i> = 6.0 Hz, 1H), 7.61–7.52 (m, 4H), 7.47–7.31 (m, 5H), 7.28 (s, 1H), 4.64 (d, <i>J</i> = 6.2 Hz, 2H), 3.91–3.45 (m, 8H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.79, 162.44, 159.91, 157.63, 140.62, 136.62, 128.86, 128.43, 127.48, 127.11, 99.53, 66.37, 44.65, 43.36. HRMS [C<sub>22</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 409.1426 calculated, 409.1421 found.</div></div><div id="sec4_2_144" class="NLM_sec NLM_sec_level_3"><div id="ac_i173" class="anchor-spacer"></div><h4 class="article-section__title" id="_i173"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(piperidin-1-yl)pyrimidine-4-carboxamide (<b>119k</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (258 mg, 1.05 mmol, 1 equiv), DiPEA (274 μL, 1.58 mmol, 1.5 equiv), and piperidine (109 μL, 1.10 mmol, 1.05 equiv). Column chromatography (10% → 40% EtOAc/pentane) afforded the product (293 mg, 0.99 mmol, 95%). TLC: <i>R</i><sub>f</sub> = 0.2 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.95 (br s, 1H), 7.27 (s, 1H), 3.71 (br s, 4H), 3.29 (t, <i>J</i> = 6.5 Hz, 2H), 1.85–1.52 (m, 6H), 1.15–0.98 (m, 1H), 0.55 (q, <i>J</i> = 5.4 Hz, 2H), 0.28 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.12, 162.45, 159.77, 157.35, 99.29, 44.31, 25.55, 24.31, 10.61, 3.58. HRMS [C<sub>14</sub>H<sub>19</sub>ClN<sub>4</sub>O + H]<sup>+</sup>: 295.1320 calculated, 295.1321 found.</div></div><div id="sec4_2_145" class="NLM_sec NLM_sec_level_3"><div id="ac_i174" class="anchor-spacer"></div><h4 class="article-section__title" id="_i174"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(3,3-difluoropiperidin-1-yl)pyrimidine-4-carboxamide (<b>119l</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (0.10 g, 0.41 mmol, 1 equiv), DiPEA (100 μL, 0.57 mmol, 2.4 equiv), and 3,3-difluoropiperidine HCl salt (68 mg, 0.43 mmol, 1.05 equiv). Column chromatography (10% → 40% EtOAc/pentane) afforded the product (88 mg, 0.26 mmol, 63%). TLC: <i>R</i><sub>f</sub> = 0.4 (20% EtOAc/pentane). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.91 (br s, 1H), 7.35 (s, 1H), 3.99 (br s, 2H), 3.74 (br s, 2H), 3.29 (t, <i>J</i> = 6.5 Hz, 2H), 2.13 (tt, <i>J</i> = 13.3, 6.3 Hz, 2H), 2.02–1.74 (m, 2H), 1.17–0.95 (m, 1H), 0.67–0.44 (m, 2H), 0.28 (q, <i>J</i> = 5.1 Hz, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 164.08, 162.15, 159.87, 158.33, 119.17 (t, <i>J</i> = 244.8 Hz), 99.62, 44.54, 44.05, 32.63 (t, <i>J</i> = 23.5 Hz), 29.76, 21.70 (t, <i>J</i> = 4.5 Hz), 10.72, 3.71. HRMS [C<sub>14</sub>H<sub>17</sub>ClF<sub>2</sub>N<sub>4</sub>O + H]<sup>+</sup>: 331.1132 calculated, 331.1127 found.</div></div><div id="sec4_2_146" class="NLM_sec NLM_sec_level_3"><div id="ac_i175" class="anchor-spacer"></div><h4 class="article-section__title" id="_i175"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(4,4-difluoropiperidin-1-yl)pyrimidine-4-carboxamide (<b>119m</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (0.12 g, 0.49 mmol, 1 equiv), DiPEA (0.21 mL, 1.23 mmol, 2.5 equiv), and 4,4-difluoropiperidine HCl salt (86 mg, 0.52 mmol, 1.05 equiv). Column chromatography (5% → 25% EtOAc/pentane) afforded the product (90 mg, 0.27 mmol, 55%). TLC: <i>R</i><sub>f</sub> = 0.4 (15% EtOAc/pentane). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.91 (br s, 1H), 7.35 (s, 1H), 3.89 (br s, 4H), 3.38–3.20 (m, 2H), 2.06 (tt, <i>J</i> = 13.2, 5.9 Hz, 4H), 1.13–0.98 (m, 1H), 0.64–0.48 (m, 2H), 0.29 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 163.47, 162.18, 160.08, 158.40, 121.29 (t, <i>J</i> = 242.4 Hz), 99.44, 44.55, 41.49, 33.82 (t, <i>J</i> = 23.7 Hz), 10.72, 3.72. HRMS [C<sub>14</sub>H<sub>17</sub>ClF<sub>2</sub>N<sub>4</sub>O + H]<sup>+</sup>: 331.1132 calculated, 331.1127 found.</div></div><div id="sec4_2_147" class="NLM_sec NLM_sec_level_3"><div id="ac_i176" class="anchor-spacer"></div><h4 class="article-section__title" id="_i176"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-thiomorpholinopyrimidine-4-carboxamide (<b>119n</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (117 mg, 0.48 mmol, 1 equiv), DiPEA (124 μL, 0.71 mmol, 1.5 equiv), and thiomorpholine (51 μL, 0.50 mmol, 1.05 equiv). Column chromatography (20% → 50% EtOAc/pentane) afforded the product (48 mg, 0.15 mmol, 32%). TLC: <i>R</i><sub>f</sub> = 0.2 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91 (br s, 1H), 7.28 (s, 1H), 4.07 (br s, 4H), 3.36–3.23 (m, 2H), 2.74–2.63 (m, 4H), 1.12–0.99 (m, 1H), 0.65–0.48 (m, 2H), 0.29 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.33, 162.30, 160.04, 158.07, 99.66, 44.55, 27.05, 10.73, 3.74. HRMS [C<sub>13</sub>H<sub>17</sub>ClN<sub>4</sub>OS + H]<sup>+</sup>: 313.0884 calculated, 313.0884 found.</div></div><div id="sec4_2_148" class="NLM_sec NLM_sec_level_3"><div id="ac_i177" class="anchor-spacer"></div><h4 class="article-section__title" id="_i177"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(1,1-dioxidothiomorpholino)pyrimidine-4-carboxamide (<b>119o</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (92 mg, 0.37 mmol, 1 equiv), DiPEA (98 μL, 0.56 mmol, 1.5 equiv), and thiomorpholine-1,1-dioxide (53 mg, 0.39 mmol, 1.05 equiv). Column chromatography (60% → 70% EtOAc/pentane) afforded the product as a 4:1 mixture of regioisomers (69 mg, 0.20 mmol, 54%). TLC: <i>R</i><sub>f</sub> = 0.5 (60% EtOAc/pentane). Major regioisomer: <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub> + MeOD) δ 7.41 (s, 1H), 4.26 (br s, 4H), 3.24 (d, <i>J</i> = 7.1 Hz, 2H), 3.18–3.12 (m, 4H), 1.10–0.98 (m, 1H), 0.59–0.49 (m, 2H), 0.32–0.21 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub> + MeOD) δ 163.16, 162.11, 159.92, 158.47, 99.62, 51.14, 44.23, 42.86, 42.43, 10.18, 3.19. HRMS [C<sub>13</sub>H<sub>17</sub>ClN<sub>4</sub>O<sub>3</sub>S + H]<sup>+</sup>: 345.0783 calculated, 345.0782 found.</div></div><div id="sec4_2_149" class="NLM_sec NLM_sec_level_3"><div id="ac_i178" class="anchor-spacer"></div><h4 class="article-section__title" id="_i178"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(4-methylpiperazin-1-yl)pyrimidine-4-carboxamide (<b>119p</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (96 mg, 0.39 mmol, 1 equiv), DiPEA (102 μL, 0.59 mmol, 1.5 equiv), and 1-methylpiperazine (45 μL, 0.41 mmol, 1.05 equiv). Column chromatography (5% → 10% MeOH/DCM) afforded the product (77 mg, 0.25 mmol, 64%). TLC: <i>R</i><sub>f</sub> = 0.4 (5% MeOH/DCM). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.99–7.84 (m, 1H), 7.28 (s, 1H), 3.96–3.58 (m, 4H), 3.35–3.24 (m, 2H), 2.54–2.44 (m, 4H), 2.34 (s, 3H), 1.12–0.99 (m, 1H), 0.64–0.50 (m, 2H), 0.28 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 163.61, 162.39, 159.88, 157.74, 99.52, 54.50, 46.06, 44.49, 43.44, 10.70, 3.71. HRMS [C<sub>14</sub>H<sub>20</sub>ClN<sub>5</sub>O + H]<sup>+</sup>: 310.1429 calculated, 310.1429 found.</div></div><div id="sec4_2_150" class="NLM_sec NLM_sec_level_3"><div id="ac_i179" class="anchor-spacer"></div><h4 class="article-section__title" id="_i179"> Benzyl-4-(2-chloro-6-((cyclopropylmethyl)carbamoyl)pyrimidin-4-yl) piperazine-1-carboxylate (<b>119q</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (181 mg, 0.74 mmol, 1 equiv), DiPEA (193 μL, 1.11 mmol, 1.5 equiv), and 1-Cbz-piperazine (149 μL, 0.77 mmol, 1.05 equiv). Column chromatography (30% → 60% EtOAc/pentane) afforded the product (290 mg, 0.67 mmol, 91%). TLC: R<sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.92 (t, <i>J</i> = 5.7 Hz, 1H), 7.40–7.30 (m, 5H), 7.28 (s, 1H), 5.17 (s, 2H), 3.74 (br s, 4H), 3.65–3.55 (m, 4H), 3.34–3.22 (m, 2H), 1.12–0.98 (m, 1H), 0.63–0.46 (m, 2H), 0.36–0.19 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.64, 162.10, 159.81, 158.01, 155.01, 136.26, 128.54, 128.20, 128.00, 99.44, 67.50, 44.40, 43.17, 10.63, 3.61.</div></div><div id="sec4_2_151" class="NLM_sec NLM_sec_level_3"><div id="ac_i180" class="anchor-spacer"></div><h4 class="article-section__title" id="_i180"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(dimethylamino)pyrimidine-4-carboxamide (<b>119r</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (246 mg, 1.0 mmol, 1 equiv), DiPEA (261 μL, 1.5 mmol, 1.5 equiv), and dimethylamine (2 M in THF, 0.53 mL, 1.05 mmol, 1.05 equiv). Column chromatography (40% → 60% EtOAc/pentane) afforded the product (225 mg, 0.88 mmol, 88%). TLC: <i>R</i><sub>f</sub> = 0.5 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.97 (br s, 1H), 7.19 (s, 1H), 3.33–3.27 (m, 2H), 3.27–3.06 (m, 6H), 1.13–0.99 (m, 1H), 0.62–0.49 (m, 2H), 0.29 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.03, 162.29, 159.36, 156.93, 99.19, 44.26, 37.76, 37.29, 10.56, 3.53. HRMS [C<sub>11</sub>H<sub>15</sub>ClN<sub>4</sub>O + H]<sup>+</sup>: 255.1007 calculated, 255.1005 found.</div></div><div id="sec4_2_152" class="NLM_sec NLM_sec_level_3"><div id="ac_i181" class="anchor-spacer"></div><h4 class="article-section__title" id="_i181"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(methylamino)pyrimidine-4-carboxamide (<b>119s</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (123 mg, 0.50 mmol, 1 equiv), DiPEA (218 μL, 1.25 mmol, 2.5 equiv), and methylamine HCl salt (35 mg, 0.53 mmol, 1.05 equiv). Column chromatography (50% → 70% EtOAc/pentane) afforded the product (77 mg, 0.32 mmol, 64%). TLC: <i>R</i><sub>f</sub> = 0.4 (60% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.93 (br s, 1H), 7.40–7.04 (m, 1H), 6.62–6.20 (m, 1H), 3.29 (br s, 2H), 3.04 (br s, 3H), 1.13–1.00 (m, 1H), 0.65–0.50 (m, 2H), 0.29 (q, <i>J</i> = 4.9 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 175.67, 166.23, 165.44, 162.63, 162.22, 160.34, 158.43, 155.80, 103.55, 98.02, 44.55, 29.81, 29.06, 28.22, 3.76. HRMS [C<sub>10</sub>H<sub>13</sub>ClN<sub>4</sub>O + H]<sup>+</sup>: 241.0851 calculated, 241.0849 found.</div></div><div id="sec4_2_153" class="NLM_sec NLM_sec_level_3"><div id="ac_i182" class="anchor-spacer"></div><h4 class="article-section__title" id="_i182"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(diethylamino)pyrimidine-4-carboxamide (<b>119t</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (123 mg, 0.50 mmol, 1 equiv), DiPEA (131 μL, 0.75 mmol, 1.5 equiv), and diethylamine (55 μL, 0.53 mmol, 1.05 equiv). Column chromatography (20% → 50% EtOAc/pentane) afforded the product (94 mg, 0.33 mmol, 66%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (br s, 1H), 7.16 (s, 1H), 3.77–3.37 (m, 4H), 3.34–3.24 (m, 2H), 1.22 (t, <i>J</i> = 7.0 Hz, 6H), 1.12–1.00 (m, 1H), 0.61–0.51 (m, 2H), 0.29 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 162.96, 162.57, 159.79, 157.11, 99.29, 44.41, 42.96, 42.73, 12.64, 10.69, 3.67. HRMS [C<sub>13</sub>H<sub>19</sub>ClN<sub>4</sub>O + H]<sup>+</sup>: 283.1320 calculated, 283.1319 found.</div></div><div id="sec4_2_154" class="NLM_sec NLM_sec_level_3"><div id="ac_i183" class="anchor-spacer"></div><h4 class="article-section__title" id="_i183"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(azetidin-1-yl)pyrimidine-4-carboxamide (<b>119u</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (98 mg, 0.40 mmol, 1 equiv), DiPEA (174 μL, 1.0 mmol, 2.5 equiv), and azetidine HCl salt (39 mg, 0.42 mmol, 1.05 equiv). Column chromatography (30% → 50% EtOAc/pentane) afforded the product (79 mg, 0.30 mmol, 74%). TLC: <i>R</i><sub>f</sub> = 0.3 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.90 (br s, 1H), 6.88 (s, 1H), 4.44–4.07 (m, 4H), 3.28 (dd, <i>J</i> = 7.0, 6.0 Hz, 2H), 2.49 (p, <i>J</i> = 7.6 Hz, 2H), 1.14–0.99 (m, 1H), 0.62–0.51 (m, 2H), 0.28 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.91, 162.30, 159.95, 156.54, 98.54, 50.20, 44.44, 16.51, 10.69, 3.69. HRMS [C<sub>12</sub>H<sub>15</sub>ClN<sub>4</sub>O + H]<sup>+</sup>: 267.1007 calculated, 267.1005 found.</div></div><div id="sec4_2_155" class="NLM_sec NLM_sec_level_3"><div id="ac_i184" class="anchor-spacer"></div><h4 class="article-section__title" id="_i184"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(2-oxa-6-azaspiro[3.3]heptan-6-yl)pyrimidine-4-carboxamide (<b>119v</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (98 mg, 0.40 mmol, 1 equiv), DiPEA (174 μL, 1.0 mmol, 2.5 equiv), and 2-oxa-6-azaspiro[3.3]heptane hemioxalate salt (61 mg, 0.42 mmol, 1.05 equiv). Column chromatography (40% → 100% EtOAc/pentane) afforded the product (96 mg, 0.31 mmol, 78%). TLC: <i>R</i><sub>f</sub> = 0.2 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.88 (br s, 1H), 6.92 (s, 1H), 4.86 (br s, 4H), 4.35 (s, 4H), 3.28 (dd, <i>J</i> = 7.0, 6.0 Hz, 2H), 1.11–0.98 (m, 1H), 0.63–0.50 (m, 2H), 0.34–0.23 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.01, 162.11, 160.06, 157.10, 98.96, 80.71, 59.73, 44.56, 39.16, 10.72, 3.76. HRMS [C<sub>14</sub>H<sub>17</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 309.1113 calculated, 309.1110 found.</div></div><div id="sec4_2_156" class="NLM_sec NLM_sec_level_3"><div id="ac_i185" class="anchor-spacer"></div><h4 class="article-section__title" id="_i185"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(pyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>119w</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (123 mg, 0.50 mmol, 1 equiv), DiPEA (131 μL, 0.75 mmol, 1.5 equiv), and pyrrolidine (43 μL, 0.53 mmol, 1.05 equiv). Column chromatography (30% → 60% EtOAc/pentane) afforded the product (150 mg, 0.40 mmol, 79%). TLC: <i>R</i><sub>f</sub> = 0.5 (40% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.93 (br s, 1H), 7.05 (s, 1H), 3.65 (t, <i>J</i> = 6.6 Hz, 2H), 3.46 (t, <i>J</i> = 6.7 Hz, 2H), 3.29 (t, <i>J</i> = 6.5 Hz, 2H), 2.09 (p, <i>J</i> = 6.3 Hz, 2H), 2.00 (p, <i>J</i> = 6.3 Hz, 2H), 1.13–0.99 (m, 1H), 0.56 (q, <i>J</i> = 5.2 Hz, 2H), 0.29 (q, <i>J</i> = 4.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 162.52, 161.97, 159.61, 156.52, 100.53, 47.23, 46.98, 44.43, 25.52, 24.82, 10.69, 3.69. HRMS [C<sub>13</sub>H<sub>17</sub>ClN<sub>4</sub>O + H]<sup>+</sup>: 281.1164 calculated, 281.1160 found.</div></div><div id="sec4_2_157" class="NLM_sec NLM_sec_level_3"><div id="ac_i186" class="anchor-spacer"></div><h4 class="article-section__title" id="_i186"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(3,3-difluoropyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>119x</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (98 mg, 0.40 mmol, 1 equiv), DiPEA (174 μL, 1.0 mmol, 2.5 equiv), and 3,3-difluoropyrrolidine HCl salt (60 mg, 0.42 mmol, 1.05 equiv). Column chromatography (20% → 40% EtOAc/pentane) afforded the product (108 mg, 0.34 mmol, 85%). TLC: <i>R</i><sub>f</sub> = 0.6 (40% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.90 (br s, 1H), 7.22–6.87 (m, 1H), 4.19–3.59 (m, 4H), 3.36–3.22 (m, 2H), 2.55 (br s, 2H), 1.14–0.97 (m, 1H), 0.63–0.50 (m, 2H), 0.37–0.18 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 162.52, 162.11, 159.90, 157.70, 100.24, 99.83, 53.66 (t, <i>J</i> = 32.9 Hz), 44.62, 33.91, 3.78. HRMS [C<sub>13</sub>H<sub>15</sub>ClF<sub>2</sub>N<sub>4</sub>O + H]<sup>+</sup>: 317.0975 calculated, 317.0974 found.</div></div><div id="sec4_2_158" class="NLM_sec NLM_sec_level_3"><div id="ac_i187" class="anchor-spacer"></div><h4 class="article-section__title" id="_i187"> (±)-2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(3-hydroxypyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>119y</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (98 mg, 0.40 mmol, 1 equiv), DiPEA (174 μL, 1.0 mmol, 2.5 equiv), and (±)-3-hydroxypyrrolidine HCl salt (52 mg, 0.42 mmol, 1.05 equiv). Column chromatography (70% → 100% EtOAc/pentane) afforded the product (89 mg, 0.30 mmol, 75%). TLC: <i>R</i><sub>f</sub> = 0.2 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98 (t, <i>J</i> = 5.3 Hz, 1H), 7.08–6.89 (m, 1H), 4.73–4.55 (m, 1H), 3.87–3.45 (m, 4H), 3.33–3.20 (m, 2H), 2.25–2.07 (m, 2H), 1.12–0.98 (m, 1H), 0.63–0.50 (m, 2H), 0.29 (q, <i>J</i> = 4.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 162.63, 162.32, 162.16, 159.59, 156.38, 100.55, 100.49, 70.46, 69.78, 55.58, 55.35, 45.26, 45.02, 44.58, 33.87, 33.29, 10.62, 3.75. HRMS [C<sub>13</sub>H<sub>17</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 297.1113 calculated, 297.1110 found.</div></div><div id="sec4_2_159" class="NLM_sec NLM_sec_level_3"><div id="ac_i188" class="anchor-spacer"></div><h4 class="article-section__title" id="_i188"> (<i>R</i>)-2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(3-hydroxypyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>119z</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (98 mg, 0.40 mmol, 1 equiv), DiPEA (174 μL, 1.0 mmol, 2.5 equiv), and (<i>R</i>)-3-hydroxypyrrolidine HCl salt (52 mg, 0.42 mmol, 1.05 equiv). Column chromatography (70% → 100% EtOAc/pentane) afforded the product (105 mg, 0.35 mmol, 88%). TLC: <i>R</i><sub>f</sub> = 0.2 (80% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98 (t, <i>J</i> = 5.4 Hz, 1H), 7.08–6.91 (m, 1H), 4.74–4.54 (m, 1H), 3.90–3.39 (m, 5H), 3.34–3.20 (m, 2H), 2.24–2.07 (m, 2H), 1.13–0.98 (m, 1H), 0.63–0.48 (m, 2H), 0.36–0.17 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 162.64, 162.34, 162.18, 159.73, 159.60, 156.40, 100.57, 100.52, 70.50, 69.81, 55.59, 55.36, 45.27, 45.03, 44.59, 33.87, 33.32, 10.63, 3.76. HRMS [C<sub>13</sub>H<sub>17</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 297.1113 calculated, 297.1110 found.</div></div><div id="sec4_2_160" class="NLM_sec NLM_sec_level_3"><div id="ac_i189" class="anchor-spacer"></div><h4 class="article-section__title" id="_i189"> (<i>S</i>)-2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(3-hydroxypyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>119aa</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (98 mg, 0.40 mmol, 1 equiv), DiPEA (174 μL, 1.0 mmol, 2.5 equiv), and (<i>S</i>)-3-hydroxypyrrolidine HCl salt (52 mg, 0.42 mmol, 1.05 equiv). Column chromatography (70% → 100% EtOAc/pentane) afforded the product (97 mg, 0.33 mmol, 82%). TLC: <i>R</i><sub>f</sub> = 0.2 (80% EtOAc/pentane). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.98 (t, <i>J</i> = 5.2 Hz, 1H), 7.11–6.86 (m, 1H), 4.81–4.47 (m, 1H), 3.93–3.35 (m, 5H), 3.33–3.21 (m, 2H), 2.25–2.06 (m, 2H), 1.16–0.96 (m, 1H), 0.57 (q, <i>J</i> = 5.5 Hz, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 162.65, 162.34, 162.19, 159.73, 159.61, 156.41, 100.55, 100.51, 70.48, 69.79, 55.58, 55.35, 45.27, 45.03, 44.58, 33.87, 33.31, 10.62, 3.74. HRMS [C<sub>13</sub>H<sub>17</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 297.1113 calculated, 297.1110 found.</div></div><div id="sec4_2_161" class="NLM_sec NLM_sec_level_3"><div id="ac_i190" class="anchor-spacer"></div><h4 class="article-section__title" id="_i190"> (±)-2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(3-(dimethylamino)pyrrolidin-1-yl)pyrimidine-4-carboxamide (<b>119ab</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using dichloropyrimidine <b>118a</b> (98 mg, 0.40 mmol, 1 equiv), DiPEA (244 μL, 1.4 mmol, 3.5 equiv), and (±)-3-(dimethylamino)pyrrolidine double HCl salt (79 mg, 0.42 mmol, 1.05 equiv). Column chromatography (2.5% → 10% MeOH/DCM) afforded the product (50 mg, 0.15 mmol, 39%). TLC: <i>R</i><sub>f</sub> = 0.4 (5% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10–7.77 (m, 1H), 7.06 (s, 1H), 4.06–3.91 (m, 2H), 3.90–3.60 (m, 1H), 3.59–3.32 (m, 2H), 3.32–3.25 (m, 2H), 2.95–2.70 (m, 1H), 2.40–2.30 (m, 6H), 2.30–2.17 (m, 1H), 2.08–1.78 (m, 1H), 1.13–0.99 (m, 1H), 0.66–0.45 (m, 2H), 0.35–0.21 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 171.69, 163.73, 162.56, 162.47, 162.15, 161.99, 159.76, 159.69, 159.50, 158.70, 158.27, 156.79, 106.57, 100.46, 100.22, 93.93, 65.41, 64.76, 53.71, 51.47, 51.33, 50.76, 46.42, 46.22, 45.79, 44.54, 44.51, 44.42, 44.28, 44.22, 44.02, 30.41, 29.81, 10.90, 10.85, 10.74, 3.76, 3.51, 3.47. HRMS [C<sub>15</sub>H<sub>22</sub>ClN<sub>5</sub>O + H]<sup>+</sup>: 324.1586 calculated, 324.1583 found.</div></div><div id="sec4_2_162" class="NLM_sec NLM_sec_level_3"><div id="ac_i191" class="anchor-spacer"></div><h4 class="article-section__title" id="_i191"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(1<i>H</i>-pyrazol-1-yl)pyrimidine-4-carboxamide (<b>119ac</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_7" aria-label="General Procedure F">General Procedure F</a> using dichloropyrimidine <b>118a</b> (123 mg, 0.50 mmol, 1 equiv), K<sub>2</sub>CO<sub>3</sub> (104 mg, 0.75 mmol, 1.5 equiv), and pyrazole (36 mg, 0.53 mmol, 1.05 equiv). Column chromatography (10% → 30% EtOAc/pentane) afforded the product (76 mg, 0.27 mmol, 55%). TLC: <i>R</i><sub>f</sub> = 0.8 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.61 (s, 1H), 8.56 (d, <i>J</i> = 2.7 Hz, 1H), 7.92 (br s, 1H), 7.85 (d, <i>J</i> = 1.3 Hz, 1H), 6.55 (dd, <i>J</i> = 2.8, 1.6 Hz, 1H), 3.36 (dd, <i>J</i> = 7.1, 5.9 Hz, 2H), 1.19–1.00 (m, 1H), 0.71–0.49 (m, 2H), 0.32 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 161.28, 160.88, 160.48, 159.64, 145.18, 128.34, 110.22, 105.51, 44.73, 3.76. HRMS [C<sub>12</sub>H<sub>12</sub>ClN<sub>5</sub>O + H]<sup>+</sup>: 278.0803 calculated, 278.0802 found.</div></div><div id="sec4_2_163" class="NLM_sec NLM_sec_level_3"><div id="ac_i192" class="anchor-spacer"></div><h4 class="article-section__title" id="_i192"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(1<i>H</i>-imidazol-1-yl)pyrimidine-4-carboxamide (<b>119ad</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_7" aria-label="General Procedure F">General Procedure F</a> using dichloropyrimidine <b>118a</b> (123 mg, 0.50 mmol, 1 equiv), K<sub>2</sub>CO<sub>3</sub> (104 mg, 0.75 mmol, 1.5 equiv), and imidazole (36 mg, 0.53 mmol, 1.05 equiv). Column chromatography (60% → 80% EtOAc/pentane) afforded the product as an inseparable mixture of regioisomers (2.5:1, 6-imidazolyl: 2-imidazolyl), which was used for the following step without further purification (76 mg, 0.27 mmol, 55%). TLC: <i>R</i><sub>f</sub> = 0.4 (5% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.56 (s, 1H), 8.10 (s, 1H), 7.98 (br s, 1H), 7.77 (t, <i>J</i> = 1.4 Hz, 1H), 7.26 (s, 1H), 3.47–3.26 (m, 2H), 1.18–1.03 (m, 1H), 0.68–0.53 (m, 2H), 0.41–0.27 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 161.93, 160.55, 159.94, 158.14, 135.73, 132.38, 116.02, 104.68, 44.84, 10.66, 3.76. HRMS [C<sub>12</sub>H<sub>12</sub>ClN<sub>5</sub>O + H]<sup>+</sup>: 278.0803 calculated, 278.0800 found.</div></div><div id="sec4_2_164" class="NLM_sec NLM_sec_level_3"><div id="ac_i193" class="anchor-spacer"></div><h4 class="article-section__title" id="_i193"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-phenoxypyrimidine-4-carboxamide (<b>119ae</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_7" aria-label="General Procedure F">General Procedure F</a> using dichloropyrimidine <b>118a</b> (100 mg, 0.41 mmol, 1 equiv), K<sub>2</sub>CO<sub>3</sub> (86 mg, 0.62 mmol, 1.5 equiv), and phenol (44 mg, 0.43 mmol, 1.05 equiv). Column chromatography (5% → 30% EtOAc/pentane) afforded the product (100 mg, 0.33 mmol, 80%). TLC: <i>R</i><sub>f</sub> = 0.4 (20% EtOAc/pentane). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.90 (br s, 1H), 7.54 (s, 1H), 7.50–7.39 (m, 2H), 7.36–7.28 (m, 1H), 7.20–7.08 (m, 2H), 3.41–3.17 (m, 2H), 1.15–0.96 (m, 1H), 0.66–0.47 (m, 2H), 0.29 (q, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 172.25, 161.17, 161.02, 159.73, 151.84, 130.19, 126.62, 121.27, 104.58, 44.68, 10.70, 3.75. HRMS [C<sub>15</sub>H<sub>14</sub>ClN<sub>3</sub>O<sub>2</sub> + H]<sup>+</sup>: 304.0847 calculated, 304.0845 found.</div></div><div id="sec4_2_165" class="NLM_sec NLM_sec_level_3"><div id="ac_i194" class="anchor-spacer"></div><h4 class="article-section__title" id="_i194"> 2-Chloro-<i>N</i>-(cyclopropylmethyl)-6-(pyridin-3-yloxy)pyrimidine-4-carboxamide (<b>119af</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_7" aria-label="General Procedure F">General Procedure F</a> using dichloropyrimidine <b>118a</b> (0.11 g, 0.45 mmol, 1 equiv), K<sub>2</sub>CO<sub>3</sub> (93 mg, 0.68 mmol, 1.5 equiv), and pyridine-3-ol (45 mg, 0.47 mmol, 1.05 equiv). Column chromatography (60% → 100% EtOAc/pentane) afforded the product (70 mg, 0.23 mmol, 51%). TLC: <i>R</i><sub>f</sub> = 0.4 (80% EtOAc/pentane). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.56 (dd, <i>J</i> = 9.3, 3.2 Hz, 2H), 7.94 (br s, 1H), 7.69 (s, 1H), 7.63–7.50 (m, 1H), 7.43 (dd, <i>J</i> = 8.3, 4.7 Hz, 1H), 3.46–3.19 (m, 2H), 1.17–0.99 (m, 1H), 0.66–0.51 (m, 2H), 0.31 (q, <i>J</i> = 4.8 Hz, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ 171.32, 161.44, 160.74, 159.43, 148.48, 147.52, 143.41, 129.09, 124.31, 105.26, 44.66, 10.66, 3.71. HRMS [C<sub>14</sub>H<sub>13</sub>ClN<sub>4</sub>O<sub>2</sub> + H]<sup>+</sup>: 305.0800 calculated, 305.0796 found.</div></div><div id="sec4_2_166" class="NLM_sec NLM_sec_level_3"><div id="ac_i195" class="anchor-spacer"></div><h4 class="article-section__title" id="_i195"> 2-(3-Phenoxyphenyl)acetonitrile (<b>121a</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with 1-(chloromethyl)-3-phenoxybenzene (<b>120a</b>) (0.37 mL, 2 mmol, 1 equiv) in dioxane/EtOH/H<sub>2</sub>O (2:2:1, 6 mL) and KCN (260 mg, 4 mmol, 2 equiv). The reaction mixture was stirred overnight at reflux. The mixture was diluted with water and extracted with EtOAc (3×). The organic layer was washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The crude material was purified by silica gel column chromatography (5% → 7% EtOAc/pentane) to provide the product (352 mg, 1.68 mmol, 84%). TLC: <i>R</i><sub>f</sub> = 0.6 (10% EtOAc/pentane). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.37–7.19 (m, 3H), 7.15–7.03 (m, 1H), 7.04–6.94 (m, 3H), 6.95–6.84 (m, 2H), 3.56 (s, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 157.73, 156.32, 131.81, 130.23, 129.71, 123.58, 122.34, 118.97, 117.96, 117.78, 117.51, 23.00.</div></div><div id="sec4_2_167" class="NLM_sec NLM_sec_level_3"><div id="ac_i196" class="anchor-spacer"></div><h4 class="article-section__title" id="_i196"> 2-([1,1′-Biphenyl]-2-yl)acetonitrile (<b>121b</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with 2-(bromomethyl)-1,1′-biphenyl (<b>120b</b>) (0.37 mL, 2 mmol, 1 equiv), solvent mixture dioxane/EtOH/H<sub>2</sub>O (2:2:1, 6 mL), and KCN (260 mg, 4 mmol, 2 equiv). The reaction mixture was stirred overnight at reflux. The mixture was diluted with water and extracted with EtOAc (3×). The organic layer was washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The crude material was purified by silica gel column chromatography (0.5% → 5% EtOAc/pentane) to provide the product (400 mg, 2.0 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.6 (3% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.59–7.12 (m, 9H), 3.57 (s, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 141.72, 139.78, 130.35, 128.86, 128.84, 128.59, 128.12, 127.68, 118.23, 21.92.</div></div><div id="sec4_2_168" class="NLM_sec NLM_sec_level_3"><div id="ac_i197" class="anchor-spacer"></div><h4 class="article-section__title" id="_i197"> <i>N</i>-Methylphenethylamine HBr Salt (<b>123a</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with methylamine (33 wt % in ethanol, 8.83 mL, 73.0 mmol, 10 equiv) and cooled to 0 <sup>ο</sup>C. (2-Bromoethyl)benzene (1.00 mL, 7.3 mmol, 1 equiv) was added, and the reaction was stirred and allowed to warm up to room temperature. After 40 h, the reaction showed complete conversion on TLC and the solvents were concentrated under reduced pressure. The product was obtained as a mixture with the di-substituted by product <i>N</i>-methyl-<i>N</i>-phenethyl-2-phenylethan-1-amine (9:1) and used without further purification (1.4 g, 6.6 mmol, 90%). TLC: <i>R</i><sub>f</sub> = 0.35 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.52–6.99 (m, 5H), 3.42–3.10 (m, 2H), 3.10–2.85 (m, 2H), 2.67 (s, 3H). <sup>13</sup>C NMR (101 MHz, MeOD) δ 137.75, 129.86, 129.78, 128.10, 51.50, 33.92, 33.31.</div></div><div id="sec4_2_169" class="NLM_sec NLM_sec_level_3"><div id="ac_i198" class="anchor-spacer"></div><h4 class="article-section__title" id="_i198"> <i>N</i>-Methyl-3-phenylpropan-1-amine (<b>123b</b>)</h4><div id="sec4_2_169_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i199" class="anchor-spacer"></div><h5 class="article-section__title" id="_i199"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 3-phenylpropan-1-amine (71 μL, 0.50 mmol, 1 equiv), methylchloroformate (58 μL, 0.75 mmol, 1.5 equiv), and DiPEA (174 μL, 1.0 mmol, 2 equiv). Column chromatography (20% - > 50% EtOAc/pentane) afforded the product (91 mg, 0.47 mmol, 94%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33–7.22 (m, 2H), 7.22–7.13 (m, 3H), 4.82 (br s, 1H), 3.65 (s, 3H), 3.26–3.07 (m, 2H), 2.69–2.57 (m, 2H), 1.82 (p, <i>J</i> = 7.3 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.18, 141.47, 128.49, 128.40, 126.02, 52.08, 40.68, 33.06, 31.68. HRMS [C<sub>11</sub>H<sub>15</sub>NO<sub>2</sub> + H]<sup>+</sup>: 194.1176 calculated, 194.1175 found.</div></div><div id="sec4_2_169_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i200" class="anchor-spacer"></div><h5 class="article-section__title" id="_i200"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (91 mg, 0.47 mmol, 1 equiv), LiAlH<sub>4</sub> (2 M THF solution, 0.75 mL, 1.54 mmol, 3.3 equiv), and was used without further purification (39 mg, 0.13 mmol, 27%). TLC: <i>R</i><sub>f</sub> = 0.1 (5% MeOH/DCM). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.31–7.13 (m, 5H), 2.68–2.63 (m, 2H), 2.63–2.58 (m, 2H), 2.42 (s, 3H), 1.90–1.76 (m, 2H), 1.71–1.58 (m, 1H). HRMS [C<sub>10</sub>H<sub>15</sub>N + H]<sup>+</sup>: 150.1277 calculated, 150.1278 found.</div></div></div><div id="sec4_2_170" class="NLM_sec NLM_sec_level_3"><div id="ac_i201" class="anchor-spacer"></div><h4 class="article-section__title" id="_i201"> <i>N</i>-Methyl-4-phenylbutan-1-amine (<b>123c</b>)</h4><div id="sec4_2_170_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i202" class="anchor-spacer"></div><h5 class="article-section__title" id="_i202"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 4-phenylbutan-1-amine (79 μL, 0.50 mmol, 1 equiv), methylchloroformate (58 μL, 0.75 mmol, 1.5 equiv), and DiPEA (174 μL, 1.0 mmol, 2 equiv). Column chromatography (10% → 40% EtOAc/pentane) afforded the product (97 mg, 0.47 mmol, 94%). TLC: <i>R</i><sub>f</sub> = 0.6 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.26 (t, <i>J</i> = 7.5 Hz, 2H), 7.17 (t, <i>J</i> = 7.9 Hz, 3H), 4.80 (br s, 1H), 3.64 (s, 3H), 3.29–3.00 (m, 2H), 2.61 (t, <i>J</i> = 7.6 Hz, 2H), 1.69–1.57 (m, 2H), 1.51 (p, <i>J</i> = 6.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.15, 142.15, 128.42, 128.37, 125.84, 52.01, 40.93, 35.54, 29.65, 28.55. HRMS [C<sub>12</sub>H<sub>17</sub>NO<sub>2</sub> + H]<sup>+</sup>: 208.1332 calculated, 208.1333 found.</div></div><div id="sec4_2_170_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i203" class="anchor-spacer"></div><h5 class="article-section__title" id="_i203"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (97 mg, 0.47 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.38 mL, 0.75 mmol, 1.6 equiv) and was used without further purification (64 mg, 0.39 mmol, 83%). TLC: <i>R</i><sub>f</sub> = 0.1 (5% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.31–7.21 (m, 2H), 7.20–7.11 (m, 3H), 2.62 (t, <i>J</i> = 7.6 Hz, 2H), 2.59–2.54 (m, 2H), 2.40 (s, 3H), 1.77–1.57 (m, 3H), 1.57–1.46 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 142.51, 128.45, 128.31, 125.73, 52.05, 36.57, 35.90, 29.61, 29.24. HRMS [C<sub>11</sub>H<sub>17</sub>N + H]<sup>+</sup>: 164.1434 calculated, 164.1433 found.</div></div></div><div id="sec4_2_171" class="NLM_sec NLM_sec_level_3"><div id="ac_i204" class="anchor-spacer"></div><h4 class="article-section__title" id="_i204"> 2-(4-Chlorophenyl)-N-methylethan-1-amine (<b>123d</b>)</h4><div id="sec4_2_171_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i205" class="anchor-spacer"></div><h5 class="article-section__title" id="_i205"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(4-chlorophenyl)ethan-1-amine (70 μL, 0.50 mmol, 1 equiv), methylchloroformate (58 μL, 0.75 mmol, 1.5 equiv), and DiPEA (174 μL, 1.0 mmol, 2 equiv). Column chromatography (10% → 50% EtOAc/pentane) afforded the product (104 mg, 0.50 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.35–7.21 (m, 2H), 7.16–7.05 (m, 2H), 5.01–4.39 (m, 1H), 3.65 (s, 3H), 3.41 (q, <i>J</i> = 6.7 Hz, 2H), 2.78 (t, <i>J</i> = 7.0 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.04, 137.32, 132.39, 130.22, 128.80, 52.20, 42.17, 35.62. HRMS [C<sub>10</sub>H<sub>12</sub>ClNO<sub>2</sub> + H]<sup>+</sup>: 214.0629 calculated, 214.0631 found.</div></div><div id="sec4_2_171_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i206" class="anchor-spacer"></div><h5 class="article-section__title" id="_i206"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (104 mg, 0.48 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.39 mL, 0.77 mmol, 1.6 equiv). Column chromatography (isocratic, 5% MeOH/DCM + 0.5% Et<sub>3</sub>N) afforded the product (36 mg, 0.21 mmol, 44%). TLC: <i>R</i><sub>f</sub> = 0.1 (2% MeOH/DCM with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.28–7.20 (m, 2H), 7.19–7.10 (m, 2H), 2.92–2.74 (m, 4H), 2.66–2.52 (m, 1H), 2.45 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 138.30, 132.03, 130.14, 128.66, 52.87, 36.12, 35.26. HRMS [C<sub>9</sub>H<sub>12</sub>ClN + H]<sup>+</sup>: 170.0731 calculated, 170.0732 found.</div></div></div><div id="sec4_2_172" class="NLM_sec NLM_sec_level_3"><div id="ac_i207" class="anchor-spacer"></div><h4 class="article-section__title" id="_i207"> 2-(3-Chlorophenyl)-<i>N</i>-methylethan-1-amine (<b>123e</b>)</h4><div id="sec4_2_172_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i208" class="anchor-spacer"></div><h5 class="article-section__title" id="_i208"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(3-chlorophenyl)ethan-1-amine (70 μL, 0.40 mmol, 1 equiv), methylchloroformate (58 μL, 0.75 mmol, 1.5 equiv), and DiPEA (174 μL, 2.0 mmol, 2 equiv). Column chromatography (20% → 40% EtOAc/pentane) afforded the product (67 mg, 0.31 mmol, 77%). TLC: <i>R</i><sub>f</sub> = 0.7 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.25–7.15 (m, 3H), 7.11–7.03 (m, 1H), 4.89 (br s, 1H), 3.65 (s, 3H), 3.41 (q, <i>J</i> = 6.8 Hz, 2H), 2.78 (t, <i>J</i> = 7.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.05, 140.94, 134.37, 129.90, 128.93, 127.04, 126.75, 52.15, 42.04, 35.90. HRMS [C<sub>10</sub>H<sub>12</sub>ClNO<sub>2</sub> + H]<sup>+</sup>: 214.0629 calculated, 214.0630 found.</div></div><div id="sec4_2_172_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i209" class="anchor-spacer"></div><h5 class="article-section__title" id="_i209"> Carbamate Reduction:</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (67 mg, 0.31 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.250 mL, 0.50 mmol, 1.6 equiv) and was used without further purification (33 mg, 0.14 mmol, 63%). <i>R</i><sub>f</sub> = 0.15 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.26–7.16 (m, 3H), 7.13–7.04 (m, 1H), 2.92–2.76 (m, 4H), 2.47 (s, 3H), 2.45 (br s, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 141.75, 134.40, 129.90, 128.92, 127.06, 126.63, 52.69, 36.06, 35.51. HRMS [C<sub>9</sub>H<sub>12</sub>ClN + H]<sup>+</sup>: 170.0731 calculated, 170.0730 found.</div></div></div><div id="sec4_2_173" class="NLM_sec NLM_sec_level_3"><div id="ac_i210" class="anchor-spacer"></div><h4 class="article-section__title" id="_i210"> 2-(2-Chlorophenyl)-N-methylethan-1-amine (<b>123f</b>)</h4><div id="sec4_2_173_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i211" class="anchor-spacer"></div><h5 class="article-section__title" id="_i211"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(2-chlorophenyl)ethan-1-amine (141 μL, 1.0 mmol, 1 equiv), methylchloroformate (116 μL, 1.5 mmol, 1.5 equiv), and DiPEA (348 μL, 2.0 mmol, 2 equiv). Column chromatography (30% → 50% EtOAc/pentane) afforded the product (183 mg, 0.85 mmol, 86%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.39–7.30 (m, 1H), 7.25–7.12 (m, 3H), 4.94 (br s, 1H), 3.65 (s, 3H), 3.44 (q, <i>J</i> = 6.7 Hz, 2H), 2.95 (t, <i>J</i> = 7.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.10, 136.50, 134.17, 131.06, 129.65, 128.06, 126.97, 52.10, 40.67, 33.93. HRMS [C<sub>10</sub>H<sub>12</sub>ClNO<sub>2</sub> + H]<sup>+</sup>: 214.0629 calculated, 214.0630 found.</div></div><div id="sec4_2_173_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i212" class="anchor-spacer"></div><h5 class="article-section__title" id="_i212"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (183 mg, 0.85 mmol, 1 equiv), LiAlH<sub>4</sub> (2 M THF solution, 0.68 mL, 1.36 mmol, 1.6 equiv) and was used without further purification (130 mg, 0.77 mmol, 90%). TLC: R<sub>f</sub> = 0.1 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.38–7.26 (m, 1H), 7.26–7.08 (m, 3H), 3.02–2.73 (m, 4H), 2.50–2.26 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 137.55, 134.08, 130.82, 129.58, 127.67, 126.81, 51.32, 36.18, 33.82. HRMS [C<sub>9</sub>H<sub>12</sub>ClN + H]<sup>+</sup>: 170.0731 calculated, 170.0732 found.</div></div></div><div id="sec4_2_174" class="NLM_sec NLM_sec_level_3"><div id="ac_i213" class="anchor-spacer"></div><h4 class="article-section__title" id="_i213"> <i>N</i>-Methyl-2-(<i>p</i>-tolyl)ethan-1-amine (<b>123g</b>)</h4><div id="sec4_2_174_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i214" class="anchor-spacer"></div><h5 class="article-section__title" id="_i214"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(<i>p</i>-tolyl)ethan-1-amine (145 μL, 1.0 mmol, 1 equiv), methylchloroformate (116 μL, 1.5 mmol, 1.5 equiv), and DiPEA (348 μL, 2.0 mmol, 2 equiv). Column chromatography (30% → 50% EtOAc/pentane) afforded the product (198 mg, 1.0 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.18–6.91 (m, 4H), 5.02 (br s, 1H), 3.61 (s, 3H), 3.38 (q, <i>J</i> = 6.8 Hz, 2H), 2.74 (t, <i>J</i> = 7.2 Hz, 2H), 2.30 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.00, 135.82, 135.66, 129.18, 128.55, 51.85, 42.26, 35.60, 20.91. HRMS [C<sub>11</sub>H<sub>15</sub>NO<sub>2</sub> + H]<sup>+</sup>: 194.1176 calculated, 194.1176 found.</div></div><div id="sec4_2_174_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i215" class="anchor-spacer"></div><h5 class="article-section__title" id="_i215"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (198 mg, 1.0 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.80 mL, 1.0 mmol, 1.6 equiv) and was used without further purification (130 mg, 0.87 mmol, 87%). TLC: <i>R</i><sub>f</sub> = 0.2 (5% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.11–7.07 (m, 4H), 2.83–2.72 (m, 4H), 2.41 (s, 3H), 2.33–2.28 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 136.87, 135.59, 129.17, 128.60, 80.66, 53.26, 36.28, 35.65, 21.02. HRMS [C<sub>10</sub>H<sub>15</sub>N + H]<sup>+</sup>: 150.1277 calculated, 150.1277 found.</div></div></div><div id="sec4_2_175" class="NLM_sec NLM_sec_level_3"><div id="ac_i216" class="anchor-spacer"></div><h4 class="article-section__title" id="_i216"> <i>N</i>-Methyl-2-(<i>o</i>-tolyl)ethan-1-amine (<b>123h</b>)</h4><div id="sec4_2_175_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i217" class="anchor-spacer"></div><h5 class="article-section__title" id="_i217"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(<i>o</i>-tolyl)ethan-1-amine (70 μL, 0.50 mmol, 1 equiv), methylchloroformate (58 μL, 0.75 mmol, 1.5 equiv), and DiPEA (174 μL, 1.0 mmol, 2 equiv). Column chromatography (30% → 50% EtOAc/pentane) afforded the product (102 mg, 0.50 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.19–7.07 (m, 4H), 5.00–4.79 (m, 1H), 3.65 (s, 3H), 3.37 (q, <i>J</i> = 6.9 Hz, 2H), 2.81 (t, <i>J</i> = 7.4 Hz, 2H), 2.32 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.12, 136.93, 136.37, 130.44, 129.34, 126.62, 126.11, 52.03, 41.16, 33.59, 19.29. HRMS [C<sub>11</sub>H<sub>15</sub>NO<sub>2</sub> + H]<sup>+</sup>: 194.1176 calculated, 194.1176 found.</div></div><div id="sec4_2_175_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i218" class="anchor-spacer"></div><h5 class="article-section__title" id="_i218"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (102 mg, 0.5 mmol, 1 equiv) and LiAlH<sub>4</sub> (1 M THF solution, 0.85 mL, 0.85 mmol, 1.6 equiv) and was used without further purification (74 mg, 0.50 mmol, 94%). TLC: <i>R</i><sub>f</sub> = 0.1 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.21–7.02 (m, 4H), 2.80 (s, 4H), 2.45 (s, 3H), 2.32 (s, 3H), 1.71 (br s, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 138.15, 136.21, 130.34, 129.25, 126.29, 126.01, 52.11, 36.44, 33.56, 19.42. HRMS [C<sub>10</sub>H<sub>15</sub>N + H]<sup>+</sup>: 150.1277 calculated, 150.1277 found.</div></div></div><div id="sec4_2_176" class="NLM_sec NLM_sec_level_3"><div id="ac_i219" class="anchor-spacer"></div><h4 class="article-section__title" id="_i219"> 2-(4-Methoxyphenyl)-<i>N</i>-methylethan-1-amine (<b>123i</b>)</h4><div id="sec4_2_176_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i220" class="anchor-spacer"></div><h5 class="article-section__title" id="_i220"> Carbamoylation</h5><div class="NLM_p last">the methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(4-methoxyphenyl)ethan-1-amine (73 μL, 0.50 mmol, 1 equiv), methylchloroformate (58 μL, 0.75 mmol, 1.5 equiv), and DiPEA (174 μL, 1.0 mmol, 2 equiv). Column chromatography (10% → 40% EtOAc/pentane) afforded the product (108 mg, 0.50 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.6 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.09 (d, <i>J</i> = 8.5 Hz, 2H), 6.83 (d, 2H), 4.93 (br s, 1H), 3.77 (s, 3H), 3.63 (s, 3H), 3.38 (q, <i>J</i> = 6.8 Hz, 2H), 2.73 (t, <i>J</i> = 7.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 158.21, 157.04, 130.80, 129.69, 113.99, 55.21, 51.96, 42.42, 35.21. HRMS [C<sub>11</sub>H<sub>15</sub>NO<sub>3</sub> + H]<sup>+</sup>: 210.1125 calculated, 210.1125 found.</div></div><div id="sec4_2_176_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i221" class="anchor-spacer"></div><h5 class="article-section__title" id="_i221"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (105 mg, 0.50 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.40 mL, 0.80 mmol, 1.6 equiv) and was used without further purification (78 mg, 0.47 mmol, 93%). TLC: <i>R</i><sub>f</sub> = 0.1 (5% MeOH/DCM with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.17–7.05 (m, 2H), 6.87–6.77 (m, 2H), 3.77 (s, 3H), 2.87–2.65 (m, 4H), 2.49–2.19 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 158.08, 132.00, 129.68, 113.96, 55.30, 53.37, 36.29, 35.18. HRMS [C<sub>10</sub>H<sub>15</sub>NO + H]<sup>+</sup>: 166.1226 calculated, 166.1226 found.</div></div></div><div id="sec4_2_177" class="NLM_sec NLM_sec_level_3"><div id="ac_i222" class="anchor-spacer"></div><h4 class="article-section__title" id="_i222"> 2-(2-Methoxyphenyl)-<i>N</i>-methylethan-1-amine (<b>123j</b>)</h4><div id="sec4_2_177_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i223" class="anchor-spacer"></div><h5 class="article-section__title" id="_i223"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(2-methoxyphenyl)ethan-1-amine (145 μL, 1.0 mmol, 1 equiv), methylchloroformate (116 μL, 1.5 mmol, 1.5 equiv), and DiPEA (348 μL, 2.0 mmol, 2 equiv). Column chromatography (20% → 40% EtOAc/pentane) afforded the product (230 mg, 1.0 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.5 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.19 (td, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.11 (d, <i>J</i> = 6.9 Hz, 1H), 6.94–6.78 (m, 2H), 5.19–4.71 (m, 1H), 3.79 (s, 3H), 3.61 (s, 3H), 3.39 (q, <i>J</i> = 6.6 Hz, 2H), 2.81 (t, <i>J</i> = 6.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.48, 157.04, 130.51, 127.75, 127.20, 120.51, 110.27, 55.14, 51.81, 41.08, 30.65. HRMS [C<sub>11</sub>H<sub>15</sub>NO<sub>3</sub> + H]<sup>+</sup>: 210.1125 calculated, 210.1125 found.</div></div><div id="sec4_2_177_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i224" class="anchor-spacer"></div><h5 class="article-section__title" id="_i224"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (230 mg, 1.0 mmol, 1 equiv) and LiAlH<sub>4</sub> (1 M THF solution, 1.6 mL, 1.6 mmol, 1.6 equiv) and was used without further purification (160 mg, 0.97 mmol, 88%). TLC: <i>R</i><sub>f</sub> = 0.3 (2% MeOH/DCM with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.24–7.07 (m, 2H), 6.95–6.74 (m, 2H), 3.80 (s, 3H), 2.90–2.70 (m, 4H), 2.43 (s, 3H), 1.26 (br s, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.59, 130.33, 128.44, 127.36, 120.39, 110.30, 55.20, 51.85, 36.37, 30.70. HRMS [C<sub>10</sub>H<sub>15</sub>NO + H]<sup>+</sup>: 166.1226 calculated, 166.1225 found.</div></div></div><div id="sec4_2_178" class="NLM_sec NLM_sec_level_3"><div id="ac_i225" class="anchor-spacer"></div><h4 class="article-section__title" id="_i225"> <i>N</i>-Methyl-2-(4-(trifluoromethyl)phenyl)ethan-1-amine (<b>123k</b>)</h4><div id="sec4_2_178_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i226" class="anchor-spacer"></div><h5 class="article-section__title" id="_i226"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(4-(trifluoromethyl)phenyl)ethan-1-amine (80 μL, 0.50 mmol, 1 equiv), methylchloroformate (58 μL, 0.75 mmol, 1.5 equiv), and DiPEA (174 μL, 1.0 mmol, 2 equiv). Column chromatography (5% → 40% EtOAc/pentane) afforded the product (113 mg, 0.45 mmol, 91%). TLC: <i>R</i><sub>f</sub> = 0.7 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.56 (d, <i>J</i> = 8.0 Hz, 2H), 7.31 (d, <i>J</i> = 8.0 Hz, 2H), 4.89–4.64 (m, 1H), 3.66 (s, 3H), 3.45 (q, <i>J</i> = 6.7 Hz, 2H), 2.88 (t, <i>J</i> = 7.0 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.06, 143.06, 129.25, 128.82, 128.37, 125.62 (q, <i>J</i> = 3.8 Hz), 122.97, 52.25, 42.03, 36.14. HRMS [C<sub>11</sub>H<sub>12</sub>F<sub>3</sub>NO<sub>2</sub> + H]<sup>+</sup>: 248.0893 calculated, 248.0896 found.</div></div><div id="sec4_2_178_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i227" class="anchor-spacer"></div><h5 class="article-section__title" id="_i227"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (113 mg, 0.45 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.36 mL, 0.72 mmol, 1.6 equiv). Column chromatography (isocratic, 5% MeOH/DCM + 0.5% Et<sub>3</sub>N) afforded the product (40 mg, 0.20 mmol, 44%). TLC: <i>R</i><sub>f</sub> = 0.3 (2% MeOH/DCM with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.55 (d, <i>J</i> = 8.0 Hz, 2H), 7.34 (d, <i>J</i> = 7.9 Hz, 2H), 3.80 (br s, 1H), 2.99–2.91 (m, 4H), 2.51 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 143.51, 129.13, 125.57 (q, <i>J</i> = 3.7 Hz), 125.41, 123.25, 52.36, 35.73, 35.29. HRMS [C<sub>10</sub>H<sub>12</sub>F<sub>3</sub>N + H]<sup>+</sup>: 204.0995 calculated, 204.0996 found.</div></div></div><div id="sec4_2_179" class="NLM_sec NLM_sec_level_3"><div id="ac_i228" class="anchor-spacer"></div><h4 class="article-section__title" id="_i228"> <i>N</i>-Methyl-2-(4-phenoxyphenyl)ethan-1-amine (<b>123l</b>)</h4><div id="sec4_2_179_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i229" class="anchor-spacer"></div><h5 class="article-section__title" id="_i229"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(4-phenoxyphenyl)ethan-1-amine TFA salt (211 mg, 0.64 mmol, 1 equiv), methylchloroformate (74 μL, 0.96 mmol, 1.5 equiv), and DiPEA (245 μL, 1.41 mmol, 2.2 equiv). Column chromatography (10% → 40% EtOAc/pentane) afforded the product (145 mg, 0.53 mmol, 83%). TLC: <i>R</i><sub>f</sub> = 0.8 (100% EtOAc). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.40–7.27 (m, 2H), 7.22–7.04 (m, 3H), 7.03–6.89 (m, 4H), 4.96–4.65 (m, 1H), 3.65 (s, 3H), 3.51–3.29 (m, 2H), 2.77 (t, <i>J</i> = 7.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.36, 157.05, 155.85, 133.69, 130.06, 129.77, 123.20, 119.12, 118.76, 52.08, 42.36, 35.48. HRMS [C<sub>16</sub>H<sub>17</sub>NO<sub>3</sub> + H]<sup>+</sup>: 272.1281 calculated, 272.1281 found.</div></div><div id="sec4_2_179_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i230" class="anchor-spacer"></div><h5 class="article-section__title" id="_i230"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (144 mg, 0.53 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.42 mL, 0.85 mmol, 1.6 equiv). Column chromatography (isocratic, 5% MeOH/DCM + 0.5% Et<sub>3</sub>N) afforded the product (65 mg, 0.29 mmol, 55%). TLC: <i>R</i><sub>f</sub> = 0.1 (5% MeOH/DCM with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.35–7.27 (m, 2H), 7.20–7.14 (m, 2H), 7.12–7.03 (m, 1H), 7.02–6.96 (m, 2H), 6.96–6.91 (m, 2H), 3.08 (br s, 1H), 2.92–2.80 (m, 4H), 2.48 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 157.46, 155.65, 134.48, 130.01, 129.77, 123.15, 119.13, 118.73, 53.02, 36.00, 35.00. HRMS [C<sub>15</sub>H<sub>17</sub>NO + H]<sup>+</sup>: 228.1383 calculated, 228.1385 found.</div></div></div><div id="sec4_2_180" class="NLM_sec NLM_sec_level_3"><div id="ac_i231" class="anchor-spacer"></div><h4 class="article-section__title" id="_i231"> <i>N-</i>Methyl-2-(3-phenoxyphenyl)ethan-1-amine (<b>123m</b>)</h4><div id="sec4_2_180_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i232" class="anchor-spacer"></div><h5 class="article-section__title" id="_i232"> Nitrile Reduction</h5><div class="NLM_p last">A round-bottom flask was charged with nitrile <b>121a</b> (211 mg, 1 mmol, 1 equiv), EtOH (5 mL), and 37% w/w aqueous HCl (0.16 mL, 2 mmol, 2 equiv). N<sub>2</sub> was bubbled through the solution for 5 min, and Pd/C (10% w/w, 52 mg, 50 μmol, 5 mol %) was added. The mixture was purged with N<sub>2</sub> and then with H<sub>2</sub> and was kept under a H<sub>2</sub> atmosphere (balloon). After 50 h, the reaction was complete as judged by TLC. The mixture was filtered over a Whatman filter, which was washed with EtOH, and the filtrate was concentrated under reduced pressure. This afforded the primary amine (<b>122a</b>) as the HCl salt, which was used without further purification (246 mg, 0.98 mmol, 98%). TLC: <i>R</i><sub>f</sub> = 0.05 (6% MeOH/DCM).</div></div><div id="sec4_2_180_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i233" class="anchor-spacer"></div><h5 class="article-section__title" id="_i233"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the primary amine <b>122a</b> (246 mg, 0.98 mmol, 1 equiv), methylchloroformate (115 μL, 1.48 mmol, 1.5 equiv), and DiPEA (517 μL, 3.0 mmol, 3 equiv). Column chromatography (5% → 40% EtOAc/pentane) afforded the product (176 mg, 0.65 mmol, 66%). TLC: <i>R</i><sub>f</sub> = 0.3 (10% EtOAc/pentane).</div></div><div id="sec4_2_180_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i234" class="anchor-spacer"></div><h5 class="article-section__title" id="_i234"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (170 mg, 0.62 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.52 mL, 1.04 mmol, 1.6 equiv) and was used without further purification (125 mg, 0.55 mmol, 88%). TLC: <i>R</i><sub>f</sub> = 0.1 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.35–7.28 (m, 2H), 7.25–7.20 (m, 1H), 7.12–7.05 (m, 1H), 7.03–6.97 (m, 2H), 6.95–6.91 (m, 1H), 6.89–6.81 (m, 2H), 2.91–2.67 (m, 5H), 2.41 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.45, 157.19, 142.10, 129.77, 129.74, 123.65, 123.26, 119.12, 118.94, 116.55, 52.94, 36.27, 35.99. HRMS [C<sub>15</sub>H<sub>17</sub>NO + H]<sup>+</sup>: 228.1383 calculated, 228.1384 found.</div></div></div><div id="sec4_2_181" class="NLM_sec NLM_sec_level_3"><div id="ac_i235" class="anchor-spacer"></div><h4 class="article-section__title" id="_i235"> <i>N</i>-Methyl-2-(2-phenoxyphenyl)ethan-1-amine (<b>123n</b>)</h4><div id="sec4_2_181_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i236" class="anchor-spacer"></div><h5 class="article-section__title" id="_i236"> Carbamoylation</h5><div class="NLM_p last">The methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using 2-(2-phenoxyphenyl)ethan-1-amine (99 μL, 0.50 mmol, 1 equiv), methylchloroformate (58 μL, 0.75 mmol, 1.5 equiv), and DiPEA (174 μL, 1.0 mmol, 2 equiv). Column chromatography (20% → 40% EtOAc/pentane) afforded the product (115 mg, 0.42 mmol, 84%). TLC: <i>R</i><sub>f</sub> = 0.7 (30% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.36–7.22 (m, 3H), 7.21–7.15 (m, 1H), 7.10–7.04 (m, 2H), 6.96–6.91 (m, 2H), 6.89–6.83 (m, 1H), 4.89 (br s, 1H), 3.62 (s, 3H), 3.44 (q, <i>J</i> = 6.6 Hz, 2H), 2.85 (t, <i>J</i> = 6.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.49, 157.08, 154.97, 131.27, 130.31, 129.85, 128.11, 123.97, 123.02, 119.26, 118.07, 52.05, 41.35, 30.73. HRMS [C<sub>16</sub>H<sub>17</sub>NO<sub>3</sub> + H]<sup>+</sup>: 272.1281 calculated, 272.1281 found.</div></div><div id="sec4_2_181_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i237" class="anchor-spacer"></div><h5 class="article-section__title" id="_i237"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (115 mg, 0.42 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.34 mL, 0.67 mmol, 1.6 equiv). Column chromatography (4% → 7% MeOH/DCM + 0.5% Et<sub>3</sub>N) afforded the product (72 mg, 0.32 mmol, 76%). TLC: <i>R</i><sub>f</sub> = 0.3 (4% MeOH/DCM with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.34–7.23 (m, 3H), 7.18 (td, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.11–7.01 (m, 2H), 6.97–6.83 (m, 3H), 3.04 (br s, 1H), 2.87 (s, 4H), 2.42 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.79, 154.81, 131.17, 131.12, 129.80, 127.88, 123.99, 122.82, 119.58, 117.88, 51.55, 35.80, 30.23. HRMS [C<sub>15</sub>H<sub>17</sub>NO + H]<sup>+</sup>: 228.1383 calculated, 228.1383 found.</div></div></div><div id="sec4_2_182" class="NLM_sec NLM_sec_level_3"><div id="ac_i238" class="anchor-spacer"></div><h4 class="article-section__title" id="_i238"> 2-([1,1′-Biphenyl]-2-yl)-<i>N</i>-methylethan-1-amine (<b>123o</b>)</h4><div id="sec4_2_182_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i239" class="anchor-spacer"></div><h5 class="article-section__title" id="_i239"> Nitrile Reduction</h5><div class="NLM_p last">A round-bottom flask was charged with nitrile <b>121b</b> (195 mg, 1 mmol, 1 equiv), EtOH (5 mL), and 37% w/w aqueous HCl (0.16 mL, 2 mmol, 2 equiv). N<sub>2</sub> was bubbled through the solution for 5 min and Pd/C (10% w/w, 52 mg, 50 μmol, 5 mol %) was added. The mixture was purged with N<sub>2</sub> and then with H<sub>2</sub> and was kept under a H<sub>2</sub> atmosphere (balloon). After 3.5 days, the reaction was complete as judged by TLC. The mixture was filtered over a Whatman filter, and the filtrate was concentrated under reduced pressure to afford the primary amine (<b>122b</b>) as the HCl salt, which was used without further purification (235 mg, 1.0 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.15 (6% MeOH/DCM).</div></div><div id="sec4_2_182_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i240" class="anchor-spacer"></div><h5 class="article-section__title" id="_i240"> Carbamoylation</h5><div class="NLM_p last">the methyl carbamate was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the amine HCl salt <b>122b</b> (235 mg, 1.0 mmol, 1 equiv), methylchloroformate (116 μL, 1.5 mmol, 1.5 equiv), and DiPEA (522 μL, 3.0 mmol, 3 equiv). Column chromatography (10% → 40% EtOAc/pentane) afforded the product (210 mg, 0.82 mmol, 82%). TLC: <i>R</i><sub>f</sub> = 0.5 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.48–7.16 (m, 9H), 4.86–4.47 (m, 1H), 3.56 (s, 3H), 3.28–3.08 (m, 2H), 2.79 (t, <i>J</i> = 7.2 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 156.89, 142.40, 141.50, 136.16, 130.32, 129.68, 129.19, 128.26, 127.64, 127.05, 126.46, 51.98, 41.83, 33.26. HRMS [C<sub>16</sub>H<sub>17</sub>NO<sub>2</sub> + H]<sup>+</sup>: 256.1332 calculated, 256.1333 found.</div></div><div id="sec4_2_182_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i241" class="anchor-spacer"></div><h5 class="article-section__title" id="_i241"> Carbamate Reduction</h5><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_8" aria-label="General Procedure G">General Procedure G</a> using the methyl carbamate (210 mg, 0.82 mmol, 1 equiv) and LiAlH<sub>4</sub> (2 M THF solution, 0.66 mL, 1.32 mmol, 1.6 equiv) and was used without further purification (157 mg, 0.74 mmol, 90%). TLC: <i>R</i><sub>f</sub> = 0.3 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.42–7.36 (m, 2H), 7.35–7.19 (m, 7H), 2.83–2.73 (m, 2H), 2.70–2.58 (m, 2H), 2.26 (s, 3H), 2.11 (br s, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 142.24, 141.70, 137.29, 130.25, 129.56, 129.22, 128.16, 127.50, 126.93, 126.13, 52.77, 36.06, 33.16. HRMS [C<sub>15</sub>H<sub>17</sub>N + H]<sup>+</sup>: 212.1434 calculated, 212.1434 found.</div></div></div><div id="sec4_2_183" class="NLM_sec NLM_sec_level_3"><div id="ac_i242" class="anchor-spacer"></div><h4 class="article-section__title" id="_i242"> <i>N</i>-Phenethylaniline (<b>124</b>)</h4><div class="NLM_p last">A round-bottom flask under a N<sub>2</sub> atmosphere was charged with phenylboronic acid (488 mg, 4 mmol, 2 equiv), Cu(OAc)<sub>2</sub>·H<sub>2</sub>O (40 mg, 0.2 mmol, 0.1 equiv), powdered 4 Å molecular sieves (1.5 g), and dry DCE (16 mL). The suspension was stirred for 5 min at room temperature followed by addition of 2-phenethylamine (252 μL, 2 mmol, 1 equiv). The blue mixture was purged using a balloon of O<sub>2</sub> causing a color shift to purple and stirred under an Ο<sub>2</sub> atmosphere for 26 h. The mixture was then filtered through a plug of Celite and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (2% → 8% EtOAc/pentane) affording the product (127 mg, 0.64 mmol, 32%). TLC: <i>R</i><sub>f</sub> = 0.6 (5% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.34–7.25 (m, 2H), 7.24–7.12 (m, 5H), 6.74–6.66 (m, 1H), 6.61–6.54 (m, 2H), 3.63 (br s, 1H), 3.36 (t, <i>J</i> = 7.1 Hz, 2H), 2.87 (t, <i>J</i> = 7.0 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 148.06, 139.38, 129.35, 128.86, 128.67, 126.48, 117.50, 113.04, 45.08, 35.55. HRMS [C<sub>14</sub>H<sub>15</sub>N + H]<sup>+</sup>: 198.1277 calculated, 198.1275 found.</div></div><div id="sec4_2_184" class="NLM_sec NLM_sec_level_3"><div id="ac_i243" class="anchor-spacer"></div><h4 class="article-section__title" id="_i243"> <i>N</i>-Ethyl-2-phenylethan-1-amine (<b>125a</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with acetaldehyde (56 μL, 1 mmol, 1 equiv), 2-phenethylamine (189 μL, 1.5 mmol, 1.5 equiv), and dry DCM (10 mL). The solution was stirred at room temperature for 10 min, and then NaHB(OAc)<sub>3</sub> (424 mg, 2 mmol, 2 equiv) was added. After 16 h, the reaction was complete as judged by TLC. The mixture was diluted with DCM (20 mL), washed with sat. aq. NaHCO<sub>3</sub> (1× 30 mL), brine (1× 30 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (5% → 7% MeOH/DCM with 0.5% Et<sub>3</sub>N) affording the product (27 mg, 0.18 mmol, 18%). TLC: <i>R</i><sub>f</sub> = 0.3 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.34–7.24 (m, 2H), 7.24–7.16 (m, 3H), 2.96–2.77 (m, 4H), 2.68 (q, <i>J</i> = 7.1 Hz, 2H), 1.10 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 140.14, 128.83, 128.59, 126.27, 51.11, 46.98, 44.09, 15.24. HRMS [C<sub>10</sub>H<sub>15</sub>N + H]<sup>+</sup>: 150.1277 calculated, 150.1277 found.</div></div><div id="sec4_2_185" class="NLM_sec NLM_sec_level_3"><div id="ac_i244" class="anchor-spacer"></div><h4 class="article-section__title" id="_i244"> <i>N</i>-Phenethylpropan-2<b>-</b>amine (<b>125b</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with NaBH<sub>4</sub> (218 mg, 5.76 mmol, 1.9 equiv) and dry DCM (5 mL). The solution was stirred in an ice bath for 3 min, and glacial acetic acid (1 mL, 14.4 mmol, 5.8 equiv) was added. The mixture was stirred for 1 h at 0 °C and 30 min at room temperature. Separately, a solution of 2-phenethylamine (377 μL, 3 mmol, 1 equiv) was prepared in dry DCM (2.5 mL) and acetone (198 μL, 2.7 mmol, 0.9 equiv). This solution was added dropwise to the NaBH(OAc)<sub>3</sub> mixture and was stirred at rt overnight. After 20 h, the reaction mixture was acidified to pH 2 with 0.1 M HCl (aq.) and washed with DCM (2× 30 mL) to remove by-products. Then, the solution was basified with 1 M NaOH (aq.) until pH 10 and extracted with DCM (3× 50 mL). The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure, and the residue was purified by silica gel column chromatography (isocratic, 0.5% Et<sub>3</sub>N/EtOAc) to provide the product (277 mg, 1.7 mmol, 63%). TLC: <i>R</i><sub>f</sub> = 0.3 (100% EtOAc with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.30–7.23 (m, 2H), 7.22–7.14 (m, 3H), 2.88–2.82 (m, 2H), 2.81–2.74 (m, 3H), 1.08 (br s, 1H), 1.03 (d, <i>J</i> = 6.4 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 139.98, 128.52, 128.28, 125.95, 48.70, 48.38, 36.48, 22.80. HRMS [C<sub>11</sub>H<sub>17</sub>N + H]<sup>+</sup>: 164.1434 calculated, 164.1434 found.</div></div><div id="sec4_2_186" class="NLM_sec NLM_sec_level_3"><div id="ac_i245" class="anchor-spacer"></div><h4 class="article-section__title" id="_i245"> <i>N</i>-Phenethylcyclopropanamine (<b>125c</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with phenylacetaldehyde (125 μL, 1 mmol, 1 equiv), cyclopropanamine (139 μL, 2 mmol, 2 equiv), and dry DCM (5 mL). The solution was stirred at room temperature for 10 min, and then NaHB(OAc)<sub>3</sub> (425 mg, 2 mmol, 2 equiv) and glacial AcOH (114 μL, 2 mmol, 2 equiv) were added. After 21 h, the reaction mixture was diluted with DCM (10 mL), washed with sat. aq. NaHCO<sub>3</sub> (1× 15 mL), brine (1× 15 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (30% → 50% EtOAc/pentane with 0.5% Et<sub>3</sub>N) affording the product (85 mg, 0.53 mmol, 53%), <i>R</i><sub>f</sub> = 0.1 (6% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.42–7.07 (m, 5H), 3.01–2.93 (m, 2H), 2.80 (t, <i>J</i> = 7.2 Hz, 2H), 2.26–2.02 (m, 2H), 0.47–0.40 (m, 2H), 0.37–0.31 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 140.23, 128.78, 128.54, 126.19, 50.84, 36.37, 30.24, 6.33. HRMS [C<sub>11</sub>H<sub>15</sub>N + H]<sup>+</sup>: 162.1277 calculated, 162.1277 found.</div></div><div id="sec4_2_187" class="NLM_sec NLM_sec_level_3"><div id="ac_i246" class="anchor-spacer"></div><h4 class="article-section__title" id="_i246"> <i>N</i>-Benzyl-2-phenylethan-1-amine (<b>125d</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with phenylacetaldehyde (125 μL, 1 mmol, 1 equiv), benzylamine (218 μL, 2 mmol, 2 equiv), and DCM (10 mL). The solution was stirred at room temperature for 3 min, and then NaHB(OAc)<sub>3</sub> (424 mg, 2 mmol, 2 equiv) and glacial AcOH (114 μL, 2 mmol, 2 equiv) were added. After 18 h, the reaction mixture was diluted with DCM (20 mL), washed with sat. aq. NaHCO<sub>3</sub> (1× 30 mL) and brine (1× 30 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography with (4% → 5% MeOH/DCM with 0.5% Et<sub>3</sub>N) affording the product (125 mg, 0.59 mmol, 59%). TLC: <i>R</i><sub>f</sub> = 0.4 (5% MeOH/DCM with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.31–7.21 (m, 7H), 7.21–7.17 (m, 3H), 3.77 (s, 2H), 2.91–2.85 (m, 2H), 2.84–2.78 (m, 2H), 1.97 (br s, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 140.21, 140.04, 128.77, 128.51, 128.44, 128.15, 126.98, 126.20, 53.88, 50.57, 36.36.</div></div><div id="sec4_2_188" class="NLM_sec NLM_sec_level_3"><div id="ac_i247" class="anchor-spacer"></div><h4 class="article-section__title" id="_i247"> Diphenethylamine (<b>125e</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with phenylacetaldehyde (125 μL, 1 mmol, 1 equiv), 2-phenethylamine (251 μL, 2 mmol, 2 equiv), and dry DCM (5 mL). The solution was stirred for 10 min, and then NaHB(OAc)<sub>3</sub> (425 mg, 2 mmol, 2 equiv) and glacial AcOH (114 μL, 2 mmol, 2 equiv) were added. After 19 h, the reaction mixture was diluted with DCM (5 mL), washed with sat. aq. NaHCO<sub>3</sub> (1× 10 mL), brine (1× 10 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (isocratic, 5% MeOH/DCM with 0.5% Et<sub>3</sub>N), affording the product (128 mg, 0.57 mmol, 57%). TLC: <i>R</i><sub>f</sub> = 0.3 (6% MeOH/DCM with 3 drops of Et<sub>3</sub>N). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.73–6.67 (m, 10H), 3.05–2.61 (m, 8H), 1.96 (br s, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 139.91, 128.67, 128.45, 126.12, 51.00, 36.25. HRMS [C<sub>16</sub>H<sub>19</sub>N + H]<sup>+</sup>: 226.1590 calculated, 226.1591 found.</div></div><div id="sec4_2_189" class="NLM_sec NLM_sec_level_3"><div id="ac_i248" class="anchor-spacer"></div><h4 class="article-section__title" id="_i248"> Methyl 2-Chloro-6-morpholinopyrimidine-4-carboxylate (<b>127</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_2_6" aria-label="General Procedure E">General Procedure E</a> using methyl 2,6-dichloropyrimidine-4-carboxylate (<b>126</b>) (0.62 g, 3.0 mmol, 1 equiv), DiPEA (0.78 mL, 4.5 mmol, 1.5 equiv), and morpholine (0.27 mL, 3.15 mmol, 1.05 equiv). Column chromatography (40% → 60% EtOAc/pentane) afforded the product (684 mg, 2.65 mmol, 88%). TLC: <i>R</i><sub>f</sub> = 0.3 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.19 (s, 1H), 3.96 (s, 3H), 3.91–3.55 (m, 8H). <sup>13</sup>C NMR (101 MHz,) δ 163.80, 163.09, 160.65, 155.10, 101.89, 65.83, 52.84, 44.18. HRMS [C<sub>10</sub>H<sub>12</sub>ClN<sub>3</sub>O<sub>3</sub> + H]<sup>+</sup>: 258.0640 calculated, 258.0638 found.</div></div><div id="sec4_2_190" class="NLM_sec NLM_sec_level_3"><div id="ac_i249" class="anchor-spacer"></div><h4 class="article-section__title" id="_i249"> 2-4-Dichloro-6-morpholino-1,3,5-triazine (<b>131</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with cyanuric chloride (0.92 g, 5.0 mmol, 1 equiv) and DCM (16 mL) and cooled to 0 °C. A mixture of morpholine (0.44 mL, 5.0 mmol, 1 equiv) and DiPEA (0.96 mL, 5.5 mmol, 1.1 equiv) in DCM (4 mL) was added dropwise. The solution was stirred for 2 h followed by the addition of 1 M HCl (25 mL). The organic layer was separated, and the water layer was extracted with DCM (25 mL). The combined organic layers were washed with water (25 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (5% → 60% Et<sub>2</sub>O/pentane) affording the product (1.18 g, 3.6 mmol, 72%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.92–3.87 (m, 4H), 3.78–3.73 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.56, 164.21, 66.51, 44.58. HRMS [C<sub>7</sub>H<sub>8</sub>Cl<sub>2</sub>N<sub>4</sub>O + H]<sup>+</sup>: 235.0148 calculated, 235.0145 found.</div></div><div id="sec4_2_191" class="NLM_sec NLM_sec_level_3"><div id="ac_i250" class="anchor-spacer"></div><h4 class="article-section__title" id="_i250"> 2-Chloro-4-(methyl(phenethyl)amino)-6-morpholino-1,3,5-triazine (<b>132</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with 2–4-dichloro-1,3,5-triazine <b>131</b> (0.59 g, 2.5 mmol, 1 equiv), K<sub>2</sub>CO<sub>3</sub> (1.38 g, 10.0 mmol, 4 equiv), and acetone (10 mL). <i>N</i>-Methylphenethylamine (0.36 mL, 2.5 mmol, 1 equiv) in acetone (2.5 mL) was added dropwise. The solution was stirred for 3 h followed by the addition of 1 M HCl (25 mL). The organic layer was separated, and the water layer was extracted with DCM (25 mL). The combined organic layers were washed with water (25 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (0% - > 20% Et<sub>2</sub>O/pentane) affording the product (0.79 g, 2.4 mmol, 95%). <sup>1</sup>Η NMR analysis showed two rotamers in a ratio of 1:1 (CDCl<sub>3</sub>, 295 K). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33–7.26 (m, 2H), 7.25–7.14 (m, 3H), 3.85–3.64 (m, 10H), 3.03 (d, <i>J</i> = 35.3 Hz, 3H), 2.93–2.84 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 169.44 (d, <i>J</i> = 3.7 Hz), 164.84 (d, <i>J</i> = 7.5 Hz), 164.50 (d, <i>J</i> = 16.3 Hz), 139.11, 128.92 (d, <i>J</i> = 17.3 Hz), 128.63 (d, <i>J</i> = 9.6 Hz), 126.49 (d, <i>J</i> = 8.9 Hz), 66.77, 50.97 (d, <i>J</i> = 27.9 Hz), 43.89, 35.36 (d, <i>J</i> = 2.7 Hz), 33.78 (d, <i>J</i> = 60.1 Hz). HRMS [C<sub>16</sub>H<sub>20</sub>ClN<sub>5</sub>O + H]<sup>+</sup>: 334.1429 calculated, 334.1425 found.</div></div><div id="sec4_2_192" class="NLM_sec NLM_sec_level_3"><div id="ac_i251" class="anchor-spacer"></div><h4 class="article-section__title" id="_i251"> 2-Cyano-4-(methyl(phenethyl)amino)-6-morpholino-1,3,5-triazine (<b>133</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with 2-chloro-1,3,5-triazine <b>132</b> (0.33 g, 1.0 mmol, 1 equiv), DABCO (22 mg, 0.2 mmol, 0.2 equiv), KCN (72 mg, 1.1 mmol, 1.1 equiv), and DMF (5 mL). The solution was stirred for 68 h followed by the addition of KCN (72 mg, 1.1 mmol, 1.1 equiv). After 3 h, the solution was diluted with sat. aq. NaHCO<sub>3</sub> (25 mL) and extracted with DCM (2× 25 mL). The combined organic layers were washed with brine (25 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography (0% → 25% Et<sub>2</sub>O/pentane), affording the product (0.17 g, 0.53 mmol, 53%). <sup>1</sup>Η NMR analysis showed two rotamers in a ratio of 1:1 (CDCl<sub>3</sub>, 295 K). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.32–7.27 (m, 2H), 7.25–7.20 (m, 2H), 7.17–7.14 (m, 1H), 3.83–3.68 (m, 10H), 3.04 (d, <i>J</i> = 37.5 Hz, 3H), 2.87 (td, <i>J</i> = 7.7, 1.9 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 163.98 (d, <i>J</i> = 4.8 Hz), 163.57 (d, <i>J</i> = 17.7 Hz), 151.91 (d, <i>J</i> = 14.0 Hz), 138.90 (d, <i>J</i> = 15.8 Hz), 128.93 (d, <i>J</i> = 24.0 Hz), 128.70 (d, <i>J</i> = 9.3 Hz), 126.63 (d, <i>J</i> = 7.4 Hz), 115.77 (d, <i>J</i> = 12.1 Hz), 66.74 (d, <i>J</i> = 30.0 Hz), 50.93 (d, <i>J</i> = 25.3 Hz), 43.77 (d, <i>J</i> = 61.7 Hz), 35.28 (d, <i>J</i> = 26.0 Hz), 33.73 (d, <i>J</i> = 94.9 Hz). HRMS [C<sub>17</sub>H<sub>20</sub>N<sub>6</sub>O + H]<sup>+</sup>: 325.1771 calculated, 325.1768 found.</div></div><div id="sec4_2_193" class="NLM_sec NLM_sec_level_3"><div id="ac_i252" class="anchor-spacer"></div><h4 class="article-section__title" id="_i252"> 4-(Methyl(phenethyl)amino)-6-morpholino-1,3,5-triazine-2-carboxylic Acid (<b>134</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with 2-cyano-1,3,5-triazine <b>133</b> (97 mg, 0.3 mmol, 1 equiv) and 1 M NaOH (1.5 mL). The solution was stirred at 60 °C for 2.5 h followed by the addition of THF (1.5 mL). The solution was stirred at 60 °C for 5 days, quenched with 1 M HCl (20 mL), and extracted with DCM (3× 20 mL). The combined organic layers were washed with brine (30 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography (0% → 10% MeOH/DCM), affording the product (88 mg, 0.26 mmol, 85%). <sup>1</sup>Η NMR analysis showed two rotamers in a ratio of 1:1 (CDCl<sub>3</sub>, 295 K). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.32–7.14 (m, 5H), 3.95–3.78 (m, 6H), 3.74 (t, <i>J</i> = 4.6 Hz, 4H), 3.10 (d, <i>J</i> = 43.6 Hz, 3H), 2.90 (t, <i>J</i> = 7.3 Hz, 2H). <sup>13</sup>C NMR (101 MHz, MeOD) δ 167.17, 164.97, 164.94, 164.78, 139.35, 129.11 (d, <i>J</i> = 12.6 Hz), 128.78 (d, <i>J</i> = 6.3 Hz), 126.65 (d, <i>J</i> = 4.1 Hz), 67.00, 51.11 (d, <i>J</i> = 33.1 Hz), 49.86, 44.10, 35.36 (d, <i>J</i> = 14.3 Hz), 33.92 (d, <i>J</i> = 77.5 Hz). HRMS [C<sub>17</sub>H<sub>21</sub>N<sub>5</sub>O<sub>3</sub> + H]<sup>+</sup>: 344.1717 calculated, 344.1714 found.</div></div><div id="sec4_2_194" class="NLM_sec NLM_sec_level_3"><div id="ac_i253" class="anchor-spacer"></div><h4 class="article-section__title" id="_i253"> Sodium (2-Chloro-6-morpholinopyrimidine-4-carbonyl) Glycinate (<b>135</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with methyl ester <b>119h</b> (267 mg, 0.85 mmol, 1 equiv) in 5 mL of THF/MeOH (4:1, v/v). A 1.5 M aqueous NaOH solution (0.57 mL, 0.85 mmol, 1 equiv) was added together with 0.43 mL of H<sub>2</sub>O. The reaction was stirred overnight at rt, and after which, the solvents were evaporated, yielding the product, which was used without further purification (270 mg, 0.85 mmol, quant.). TLC: <i>R</i><sub>f</sub> = 0.2 (5% MeOH/DCM). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.30 (s, 1H), 3.95–3.86 (m, 2H), 3.81–3.65 (m, 8H). <sup>13</sup>C NMR (101 MHz, MeOD) δ 180.70, 159.60, 159.40, 158.60, 152.43, 99.06, 65.99, 43.24. HRMS [C<sub>11</sub>H<sub>13</sub>ClN<sub>4</sub>O<sub>4</sub> + H]<sup>+</sup>: 301.0698 calculated, 301.0696 found.</div></div><div id="sec4_2_195" class="NLM_sec NLM_sec_level_3"><div id="ac_i254" class="anchor-spacer"></div><h4 class="article-section__title" id="_i254"> 4-Amino-6-chloro-2-(methyl(phenethyl)amino)pyrimidine (<b>138</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with 4-amino-2,6-dichloropyrimidine (1.0 g, 6.1 mmol, 1 equiv), <i>N</i>-methylphenethylamine (0.66 mL, 6.7 mmol, 1.1 equiv), DiPEA (3.2 mL, 18.3 mmol, 3 equiv), and dry 2-PrOH (6.1 mL). The solution was refluxed for 16 h followed by the addition of water (30 mL), and the resulting mixture was extracted with EtOAc (30 mL). The organic layer was washed with water (2× 30 mL). The combined water layers were back-extracted with ethyl acetate (30 mL), and the combined organic layers were washed with brine (30 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (10% → 40% EtOAc/pentane), affording the product (1.0 g, 3.9 mmol, 64%). <sup>1</sup>H NMR (300 MHz, MeOD) δ 7.30–7.08 (m, 5H), 5.77 (s, 1H), 4.60 (d, <i>J</i> = 4.8 Hz, 2H), 3.72 (dd, <i>J</i> = 8.3, 6.8 Hz, 2H), 2.95 (s, 3H), 2.91–2.77 (m, 2H). <sup>13</sup>C NMR (75 MHz, MeOD) δ 164.52, 161.00, 159.03, 139.25, 128.44, 127.91, 125.67, 91.72, 50.97, 35.18, 33.27. HRMS [C<sub>13</sub>H<sub>15</sub>ClN<sub>4</sub> + H]<sup>+</sup>: 263.1058 calculated, 263.1056 found.</div></div><div id="sec4_2_196" class="NLM_sec NLM_sec_level_3"><div id="ac_i255" class="anchor-spacer"></div><h4 class="article-section__title" id="_i255"> 4-Amino-2-(methyl(phenethyl)amino)-6-morpholinopyrimidine (<b>139</b>)</h4><div class="NLM_p last">A microwave vial was charged with 6-chloropyrimidine <b>138</b> (0.13 g, 0.5 mmol, 1 equiv), morpholine (65 μL, 0.75 mmol, 1.5 equiv), DiPEA (0.26 mL, 1.5 mmol, 3 equiv), and <i>n</i>-BuOH (2.5 mL). The vial was capped and heated to 180 °C in a microwave reactor (75 W) for 10 h. The solution was concentrated under reduced pressure and coevaporated with toluene (2×). The residue was purified by flash column chromatography (0% → 4% MeOH/DCM) affording the product (0.14 g, 0.44 mmol, 89%). Regioselectivity was confirmed by <sup>1</sup>H-NOESY NMR analysis and is in accordance with previous reports.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.31–7.26 (m, 2H), 7.24–7.17 (m, 3H), 5.05 (s, 1H), 4.45 (s, 2H), 3.78–3.72 (m, 6H), 3.53–3.48 (m, 4H), 3.05 (s, 3H), 2.91–2.85 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 164.51, 140.37, 128.98, 128.52, 126.12, 73.73, 66.87, 51.47, 44.76, 35.57, 34.01. HRMS [C<sub>17</sub>H<sub>23</sub>N<sub>5</sub>O + H]<sup>+</sup>: 314.1975 calculated, 314.1973 found.</div></div><div id="sec4_2_197" class="NLM_sec NLM_sec_level_3"><div id="ac_i256" class="anchor-spacer"></div><h4 class="article-section__title" id="_i256"> (±)-Benzyl 2-Phenylpiperazine-1-carboxylate Hydrochloride (<b>141</b>)</h4><div id="sec4_2_197_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i257" class="anchor-spacer"></div><h5 class="article-section__title" id="_i257"> Cbz Protection</h5><div class="NLM_p last">A round-bottom flask was charged with <i>tert</i>-butyl 3-phenylpiperazine-1-carboxylate (0.29 g, 1.1 mmol, 1 equiv), NaHCO<sub>3</sub> (0.46 g, 5.5 mmol, 5 equiv) in THF/H<sub>2</sub>O (8 mL, 1:1). The mixture was cooled to 0 °C and CbzCl (0.19 mL, 1.3 mmol, 1.2 equiv) was added dropwise. The reaction mixture was allowed to warm to room temperature and stirred for 1.5 h. The mixture was extracted with EtOAc (3× 20 mL), and the combined organic layers were washed with brine (50 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by silica gel column chromatography (5% → 40% EtOAc/pentane), affording the product (0.40 g, 1.0 mmol, 90%). TLC: <i>R</i><sub>f</sub> = 0.4 (20% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.55–7.04 (m, 10H), 5.36 (br s, 1H), 5.27–5.12 (m, 2H), 4.50 (d, <i>J</i> = 13.9 Hz, 1H), 4.13–3.68 (m, 2H), 3.33 (dd, <i>J</i> = 14.0, 4.3 Hz, 1H), 3.19–2.79 (m, 2H), 1.42 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 155.77, 154.43, 138.40, 136.48, 128.66, 128.61, 128.21, 128.00, 127.39, 126.88, 80.32, 67.64, 54.15, 46.04, 42.81, 39.80, 28.42. HRMS [C<sub>23</sub>H<sub>28</sub>N<sub>2</sub>O<sub>4</sub> + Na]<sup>+</sup>: 419.1941 calculated, 419.1934 found.</div></div><div id="sec4_2_197_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i258" class="anchor-spacer"></div><h5 class="article-section__title" id="_i258"> Boc Deprotection</h5><div class="NLM_p last">A round-bottom flask was charged with the double-protected piperazine (103 mg, 0.26 mmol, 1 equiv) and HCl (4 M solution in 1,4-dioxane, 1 mL, 4 mmol, 15.5 equiv) and stirred at rt. After 1 h, the reaction was concentrated under reduced pressure to afford the product as the hydrochloride salt (87 mg, 0.26 mmol, 99%). TLC: <i>R</i><sub>f</sub> = 0.3 (50% EtOAc/pentane). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.24 (br s, 1H), 9.33–8.80 (m, 1H), 7.51–7.02 (m, 10H), 5.55 (br s, 1H), 5.29–4.95 (m, 2H), 4.22 (d, <i>J</i> = 14.1 Hz, 1H), 3.99–3.77 (m, 1H), 3.49–3.22 (m, 3H), 3.06 (br s, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 155.26, 135.78, 135.54, 129.31, 128.64, 128.42, 128.28, 128.11, 126.34, 68.19, 51.77, 44.38, 42.59, 36.98.2</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i259"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01441" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45214" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45214" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01441?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01441</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Schemes S1–S10 describing the synthesis of compounds <b>5</b>–<b>7</b>, <b>20</b>–<b>22</b>, <b>24</b>, <b>29</b>, <b>30</b>, <b>32</b>, <b>68</b>, <b>69</b>, <b>77</b>–<b>79</b>, and <b>92</b>, <sup>1</sup>H and <sup>13</sup>C NMR data of all inhibitors, and HPLC traces of compounds <b>1</b> and <b>2</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_001.pdf">jm0c01441_si_001.pdf (20.43 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_002.csv">jm0c01441_si_002.csv (10.03 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01441" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mario van der Stelt</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Physiology,
Leiden Institute of Chemistry, Leiden University
and Oncode Institute, 2300 RA Leiden, Netherlands</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0865267e6966266c6d7a267b7c6d647c486b606d6526646d616c6d667d66617e266664"><span class="__cf_email__" data-cfemail="5f3271293e31713b3a2d712c2b3a332b1f3c373a3271333a363b3a312a313629713133">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elliot D. Mock</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Physiology,
Leiden Institute of Chemistry, Leiden University
and Oncode Institute, 2300 RA Leiden, Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1500-2856" title="Orcid link">http://orcid.org/0000-0002-1500-2856</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ioli Kotsogianni</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Physiology,
Leiden Institute of Chemistry, Leiden University
and Oncode Institute, 2300 RA Leiden, Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wouter P. F. Driever</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Physiology,
Leiden Institute of Chemistry, Leiden University
and Oncode Institute, 2300 RA Leiden, Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carmen S. Fonseca</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Physiology,
Leiden Institute of Chemistry, Leiden University
and Oncode Institute, 2300 RA Leiden, Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jelle M. Vooijs</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Physiology,
Leiden Institute of Chemistry, Leiden University
and Oncode Institute, 2300 RA Leiden, Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hans den Dulk</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Physiology,
Leiden Institute of Chemistry, Leiden University
and Oncode Institute, 2300 RA Leiden, Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Constant A. A. van Boeckel</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Physiology,
Leiden Institute of Chemistry, Leiden University
and Oncode Institute, 2300 RA Leiden, Netherlands</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): E.D.M., I.K., C.A.A.v.B. and M.v.d.S. are listed as inventors on patent application WO 2019/229250A1 filed by Leiden University in which inhibitors of NAPE-PLD are disclosed.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i260">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26604" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26604" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">M.v.d.S. was supported by a VICI grant from the Netherlands Organization for Scientific Research (NWO) and funding from Oncode Institute. Leiden University Faculty of Science “Profiling Programme: Endocannabinoids” is also acknowledged for financial support to E.D.M. and M.v.d.S. Hans van den Elst and Sebastian Grimm are kindly acknowledged for HPLC purifications, Rian van den Nieuwendijk for optical rotation measurements, and Karthick Sai Sankar Gupta and Fons Lefeber for helping with NMR experiments.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">NAPE-PLD</td><td class="NLM_def"><p class="first last"><i>N</i>-acylphosphatidylethanolamine phospholipase D</p></td></tr><tr><td class="NLM_term">NAPE</td><td class="NLM_def"><p class="first last"><i>N</i>-acylphosphatidylethanolamine</p></td></tr><tr><td class="NLM_term">NAE</td><td class="NLM_def"><p class="first last"><i>N</i>-acylethanolamine</p></td></tr><tr><td class="NLM_term">PE</td><td class="NLM_def"><p class="first last">phosphatidylethanolamine</p></td></tr><tr><td class="NLM_term">PA</td><td class="NLM_def"><p class="first last">phosphatidic acid</p></td></tr><tr><td class="NLM_term">ECS</td><td class="NLM_def"><p class="first last">endocannabinoid system</p></td></tr><tr><td class="NLM_term">AEA</td><td class="NLM_def"><p class="first last"><i>N</i>-arachidonoylethanolamine</p></td></tr><tr><td class="NLM_term">CB<sub>1/2</sub></td><td class="NLM_def"><p class="first last">cannabinoid receptor 1/2</p></td></tr><tr><td class="NLM_term">TRPV1</td><td class="NLM_def"><p class="first last">transient receptor potential vanilloid 1</p></td></tr><tr><td class="NLM_term">PPAR-α</td><td class="NLM_def"><p class="first last">peroxisome proliferator-activated receptor α</p></td></tr><tr><td class="NLM_term">PLD1</td><td class="NLM_def"><p class="first last">phospholipase D1</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">NASH</td><td class="NLM_def"><p class="first last">non-alcoholic steatohepatitis</p></td></tr><tr><td class="NLM_term">HPA</td><td class="NLM_def"><p class="first last">hypothalamus-pituitary–adrenal</p></td></tr><tr><td class="NLM_term">LipE</td><td class="NLM_def"><p class="first last">Lipophilic efficiency</p></td></tr><tr><td class="NLM_term">DiPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-(dimethylamino)propyl)-<i>N</i>′-ethylcarbodiimide</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">PyBOP</td><td class="NLM_def"><p class="first last">(benzotriazol-1-yloxy)tripyrrolidinophosphonium hexafluorophosphate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i262">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84443" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84443" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 22 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mock, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustafa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunduz-Cinar, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cinar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrie, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantae, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varga, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paloczi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miliano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donvito, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Gracht, A. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotsogianni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Wel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soethoudt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendel, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakker, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillo, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florea, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Hurk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grether, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Boeckel, C. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hankemeier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buczynski, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacher, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichtman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a NAPE-PLD inhibitor that modulates emotional behavior in mice</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0528-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1038%2Fs41589-020-0528-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=32393901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptVKhs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=667-675&author=E.+D.+Mockauthor=M.+Mustafaauthor=O.+Gunduz-Cinarauthor=R.+Cinarauthor=G.+N.+Petrieauthor=V.+Kantaeauthor=X.+Diauthor=D.+Ogasawaraauthor=Z.+V.+Vargaauthor=J.+Palocziauthor=C.+Milianoauthor=G.+Donvitoauthor=A.+C.+M.+van+Esbroeckauthor=A.+M.+F.+van+der+Grachtauthor=I.+Kotsogianniauthor=J.+K.+Parkauthor=A.+Martellaauthor=T.+van+der+Welauthor=M.+Soethoudtauthor=M.+Jiangauthor=T.+J.+Wendelauthor=A.+P.+A.+Janssenauthor=A.+T.+Bakkerauthor=C.+M.+Donovanauthor=L.+I.+Castilloauthor=B.+I.+Floreaauthor=J.+Watauthor=H.+van+den+Hurkauthor=M.+Wittwerauthor=U.+Gretherauthor=A.+Holmesauthor=C.+A.+A.+van+Boeckelauthor=T.+Hankemeierauthor=B.+F.+Cravattauthor=M.+W.+Buczynskiauthor=M.+N.+Hillauthor=P.+Pacherauthor=A.+H.+Lichtmanauthor=M.+van+der%0AStelt&title=Discovery+of+a+NAPE-PLD+inhibitor+that+modulates+emotional+behavior+in+mice&doi=10.1038%2Fs41589-020-0528-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a NAPE-PLD inhibitor that modulates emotional behavior in mice</span></div><div class="casAuthors">Mock, Elliot D.; Mustafa, Mohammed; Gunduz-Cinar, Ozge; Cinar, Resat; Petrie, Gavin N.; Kantae, Vasudev; Di, Xinyu; Ogasawara, Daisuke; Varga, Zoltan V.; Paloczi, Janos; Miliano, Cristina; Donvito, Giulia; van Esbroeck, Annelot C. M.; van der Gracht, Anouk M. F.; Kotsogianni, Ioli; Park, Joshua K.; Martella, Andrea; van der Wel, Tom; Soethoudt, Marjolein; Jiang, Ming; Wendel, Tiemen J.; Janssen, Antonius P. A.; Bakker, Alexander T.; Donovan, Colleen M.; Castillo, Laura I.; Florea, Bogdan I.; Wat, Jesse; van den Hurk, Helma; Wittwer, Matthias; Grether, Uwe; Holmes, Andrew; van Boeckel, Constant A. A.; Hankemeier, Thomas; Cravatt, Benjamin F.; Buczynski, Matthew W.; Hill, Matthew N.; Pacher, Pal; Lichtman, Aron H.; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">667-675</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: N-acylethanolamines (NAEs), which include the endocannabinoid anandamide, represent an important family of signaling lipids in the brain.  The lack of chem. probes that modulate NAE biosynthesis in living systems hamper the understanding of the biol. role of these lipids.  Using a high-throughput screen, chem. proteomics and targeted lipidomics, we report here the discovery and characterization of LEI-401 as a CNS-active N-acylphosphatidylethanolamine phospholipase D (NAPE-PLD) inhibitor.  LEI-401 reduced NAE levels in neuroblastoma cells and in the brain of freely moving mice, but not in NAPE-PLD KO cells and mice, resp.  LEI-401 activated the hypothalamus-pituitary-adrenal axis and impaired fear extinction, thereby emulating the effect of a cannabinoid CB1 receptor antagonist, which could be reversed by a fatty acid amide hydrolase inhibitor.  Our findings highlight the distinctive role of NAPE-PLD in NAE biosynthesis in the brain and suggest the presence of an endogenous NAE tone controlling emotional behavior. [graphic not available: see fulltext].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzC9MSWVmbH7Vg90H21EOLACvtfcHk0lj0N3aZAc0_VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptVKhs7g%253D&md5=093b78b3c98c31031915eea3bb596a18</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0528-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0528-7%26sid%3Dliteratum%253Aachs%26aulast%3DMock%26aufirst%3DE.%2BD.%26aulast%3DMustafa%26aufirst%3DM.%26aulast%3DGunduz-Cinar%26aufirst%3DO.%26aulast%3DCinar%26aufirst%3DR.%26aulast%3DPetrie%26aufirst%3DG.%2BN.%26aulast%3DKantae%26aufirst%3DV.%26aulast%3DDi%26aufirst%3DX.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DVarga%26aufirst%3DZ.%2BV.%26aulast%3DPaloczi%26aufirst%3DJ.%26aulast%3DMiliano%26aufirst%3DC.%26aulast%3DDonvito%26aufirst%3DG.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3Dvan%2Bder%2BGracht%26aufirst%3DA.%2BM.%2BF.%26aulast%3DKotsogianni%26aufirst%3DI.%26aulast%3DPark%26aufirst%3DJ.%2BK.%26aulast%3DMartella%26aufirst%3DA.%26aulast%3Dvan%2Bder%2BWel%26aufirst%3DT.%26aulast%3DSoethoudt%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DM.%26aulast%3DWendel%26aufirst%3DT.%2BJ.%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3DBakker%26aufirst%3DA.%2BT.%26aulast%3DDonovan%26aufirst%3DC.%2BM.%26aulast%3DCastillo%26aufirst%3DL.%2BI.%26aulast%3DFlorea%26aufirst%3DB.%2BI.%26aulast%3DWat%26aufirst%3DJ.%26aulast%3Dvan%2Bden%2BHurk%26aufirst%3DH.%26aulast%3DWittwer%26aufirst%3DM.%26aulast%3DGrether%26aufirst%3DU.%26aulast%3DHolmes%26aufirst%3DA.%26aulast%3Dvan%2BBoeckel%26aufirst%3DC.%2BA.%2BA.%26aulast%3DHankemeier%26aufirst%3DT.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DBuczynski%26aufirst%3DM.%2BW.%26aulast%3DHill%26aufirst%3DM.%2BN.%26aulast%3DPacher%26aufirst%3DP.%26aulast%3DLichtman%26aufirst%3DA.%2BH.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520NAPE-PLD%2520inhibitor%2520that%2520modulates%2520emotional%2520behavior%2520in%2520mice%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D667%26epage%3D675%26doi%3D10.1038%2Fs41589-020-0528-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Mammalian enzymes responsible for the biosynthesis of <i>N</i>-acylethanolamines</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1862</span>,  <span class="NLM_volume"><i>2017</i></span>,  <span class="NLM_fpage">1546</span>– <span class="NLM_lpage">1561</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=1862&pages=1546-1561&author=Z.+Hussainauthor=T.+Uyamaauthor=K.+Tsuboiauthor=N.+Ueda&title=Mammalian+enzymes+responsible+for+the+biosynthesis+of+N-acylethanolamines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DZ.%26aulast%3DUyama%26aufirst%3DT.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DMammalian%2520enzymes%2520responsible%2520for%2520the%2520biosynthesis%2520of%2520N-acylethanolamines%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1862%26volume%3D2017%26spage%3D1546%26epage%3D1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morishita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Molecular characterization of a phospholipase D generating anandamide and its congeners</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">5298</span>– <span class="NLM_lpage">5305</span>, <span class="refDoi"> DOI: 10.1074/jbc.M306642200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1074%2Fjbc.M306642200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=14634025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFeqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=5298-5305&author=Y.+Okamotoauthor=J.+Morishitaauthor=K.+Tsuboiauthor=T.+Tonaiauthor=N.+Ueda&title=Molecular+characterization+of+a+phospholipase+D+generating+anandamide+and+its+congeners&doi=10.1074%2Fjbc.M306642200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Characterization of a Phospholipase D Generating Anandamide and Its Congeners</span></div><div class="casAuthors">Okamoto, Yasuo; Morishita, Jun; Tsuboi, Kazuhito; Tonai, Takeharu; Ueda, Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5298-5305</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Anandamide (N-arachidonoylethanolamine) is known to be an endogenous ligand of cannabinoid and vanilloid receptors.  Its congeners (collectively referred to as N-acylethanolamines) also show a variety of biol. activities.  These compds. are principally formed from their corresponding N-acylphosphatidylethanolamines by a phosphodiesterase of the phospholipase D-type in animal tissues.  The authors purified the enzyme from rat heart, and by the use of the sequences of its internal peptides cloned its cDNAs from mouse, rat, and human.  The deduced amino acid sequences were composed of 393-396 residues, and showed that the enzyme has no homol. with the known phospholipase D enzymes but is classified as a member of the zinc metallohydrolase family of the β-lactamase fold.  As was overexpressed in COS-7 cells, the recombinant enzyme generated anandamide and other N-acylethanolamines from their corresponding N-acylphosphatidylethanolamines at comparable rates.  In contrast, the enzyme was inactive with phosphatidylcholine and phosphatidylethanolamine.  Assays of the enzyme activity and the mRNA and protein levels revealed its wide distribution in murine organs with higher contents in the brain, kidney, and testis.  These results confirm that a specific phospholipase D is responsible for the generation of N-acylethanolamines including anandamide, strongly suggesting the physiol. importance of lipid mols. of this class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK_-VVe-jk7rVg90H21EOLACvtfcHk0lhHCBQBoZuYPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFeqtrY%253D&md5=f1c00a4d0e21163c03b5a70d2f46c721</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M306642200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M306642200%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DY.%26aulast%3DMorishita%26aufirst%3DJ.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DTonai%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DMolecular%2520characterization%2520of%2520a%2520phospholipase%2520D%2520generating%2520anandamide%2520and%2520its%2520congeners%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D5298%26epage%3D5305%26doi%3D10.1074%2Fjbc.M306642200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Endocannabinoids and related <i>N</i>-acylethanolamines: biological activities and metabolism</span>. <i>Inflammation Regener.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">28</span>, <span class="refDoi"> DOI: 10.1186/s41232-018-0086-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1186%2Fs41232-018-0086-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=30288203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1eqt7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=28&author=K.+Tsuboiauthor=T.+Uyamaauthor=Y.+Okamotoauthor=N.+Ueda&title=Endocannabinoids+and+related+N-acylethanolamines%3A+biological+activities+and+metabolism&doi=10.1186%2Fs41232-018-0086-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Endocannabinoids and related N-acylethanolamines: biological activities and metabolism</span></div><div class="casAuthors">Tsuboi, Kazuhito; Uyama, Toru; Okamoto, Yasuo; Ueda, Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Inflammation and Regeneration</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">28</span>CODEN:
                <span class="NLM_cas:coden">IRNECG</span>;
        ISSN:<span class="NLM_cas:issn">1880-9693</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The plant Cannabis sativa contains cannabinoids represented by Δ9-tetrahydrocannabinol, which exert psychoactivity and immunomodulation through cannabinoid CB1 and CB2 receptors, resp., in animal tissues.  Arachidonoylethanolamide (also referred to as anandamide) and 2-arachidonoylglycerol (2-AG) are well known as two major endogenous agonists of these receptors (termed "endocannabinoids") and show various cannabimimetic bioactivities.  However, only 2-AG is a full agonist for CB1 and CB2 and mediates retrograde signals at the synapse, strongly suggesting that 2-AG is physiol. more important than anandamide.  The metabolic pathways of these two endocannabinoids are completely different. 2-AG is mostly produced from inositol phospholipids via diacylglycerol by phospholipase C and diacylglycerol lipase and then degraded by monoacylglycerol lipase.  On the other hand, anandamide is concomitantly produced with larger amts. of other N-acylethanolamines via N-acyl-phosphatidylethanolamines (NAPEs).  Although this pathway consists of calcium-dependent N-acyltransferase and NAPE-hydrolyzing phospholipase D, recent studies revealed the involvement of several new enzymes.  Quant. major N-acylethanolamines include palmitoylethanolamide and oleoylethanolamide, which do not bind to cannabinoid receptors but exert anti-inflammatory, analgesic, and anorexic effects through receptors such as peroxisome proliferator-activated receptor α.  The biosynthesis of these non-endocannabinoid N-acylethanolamines rather than anandamide may be the primary significance of this pathway.  Here, we provide an overview of the biol. activities and metabs. of endocannabinoids (2-AG and anandamide) and non-endocannabinoid N-acylethanolamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUWzSp4LibbLVg90H21EOLACvtfcHk0lhHCBQBoZuYPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1eqt7zN&md5=0ac9f596d18fd9d769a1f67f18bf8223</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1186%2Fs41232-018-0086-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs41232-018-0086-5%26sid%3Dliteratum%253Aachs%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DUyama%26aufirst%3DT.%26aulast%3DOkamoto%26aufirst%3DY.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DEndocannabinoids%2520and%2520related%2520N-acylethanolamines%253A%2520biological%2520activities%2520and%2520metabolism%26jtitle%3DInflammation%2520Regener.%26date%3D2018%26volume%3D38%26spage%3D28%26doi%3D10.1186%2Fs41232-018-0086-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maccarrone, M.</span></span> <span> </span><span class="NLM_article-title">Metabolism of the endocannabinoid anandamide: open questions after 25 years</span>. <i>Front. Mol. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">166</span>, <span class="refDoi"> DOI: 10.3389/fnmol.2017.00166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.3389%2Ffnmol.2017.00166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=28611591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A280%3ADC%252BC1cnotVWitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=166&author=M.+Maccarrone&title=Metabolism+of+the+endocannabinoid+anandamide%3A+open+questions+after+25+years&doi=10.3389%2Ffnmol.2017.00166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of the Endocannabinoid Anandamide: Open Questions after 25 Years</span></div><div class="casAuthors">Maccarrone Mauro; Maccarrone Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in molecular neuroscience</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">166</span>
        ISSN:<span class="NLM_cas:issn">1662-5099</span>.
    </div><div class="casAbstract">Cannabis extracts have been used for centuries, but its main active principle α(9)-tetrahydrocannabinol (THC) was identified about 50 years ago.  Yet, it is only 25 years ago that the first endogenous ligand of the same receptors engaged by the cannabis agents was discovered.  This "endocannabinoid (eCB)" was identified as N-arachidonoylethanolamine (or anandamide (AEA)), and was shown to have several receptors, metabolic enzymes and transporters that altogether drive its biological activity.  Here I report on the latest advances about AEA metabolism, with the aim of focusing open questions still awaiting an answer for a deeper understanding of AEA activity, and for translating AEA-based drugs into novel therapeutics for human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT31pdfVsY5rhWfpMXQLRstfW6udTcc2eayS5lAPMXOsrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnotVWitA%253D%253D&md5=5534e6a37182be0810bd932e9c5eb5a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3389%2Ffnmol.2017.00166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnmol.2017.00166%26sid%3Dliteratum%253Aachs%26aulast%3DMaccarrone%26aufirst%3DM.%26atitle%3DMetabolism%2520of%2520the%2520endocannabinoid%2520anandamide%253A%2520open%2520questions%2520after%252025%2520years%26jtitle%3DFront.%2520Mol.%2520Neurosci.%26date%3D2017%26volume%3D10%26spage%3D166%26doi%3D10.3389%2Ffnmol.2017.00166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mazier, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saucisse, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatta-Cherifi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cota, D.</span></span> <span> </span><span class="NLM_article-title">The endocannabinoid system: pivotal orchestrator of obesity and metabolic disease</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2015.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.tem.2015.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=26412154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKnsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=524-537&author=W.+Mazierauthor=N.+Saucisseauthor=B.+Gatta-Cherifiauthor=D.+Cota&title=The+endocannabinoid+system%3A+pivotal+orchestrator+of+obesity+and+metabolic+disease&doi=10.1016%2Fj.tem.2015.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The Endocannabinoid System: Pivotal Orchestrator of Obesity and Metabolic Disease</span></div><div class="casAuthors">Mazier, Wilfrid; Saucisse, Nicolas; Gatta-Cherifi, Blandine; Cota, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">524-537</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The endocannabinoid system (ECS) functions to adjust behavior and metab. according to environmental changes in food availability.  Its actions range from the regulation of sensory responses to the development of preference for the consumption of calorically-rich food and control of its metabolic handling.  ECS activity is beneficial when access to food is scarce or unpredictable.  However, when food is plentiful, the ECS favors obesity and metabolic disease.  We review recent advances in understanding the roles of the ECS in energy balance, and discuss newly identified mechanisms of action that, after the withdrawal of first generation cannabinoid type 1 (CB1) receptor antagonists for the treatment of obesity, have made the ECS once again an attractive target for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_3BI1bJVo57Vg90H21EOLACvtfcHk0lidhIkgwEhpJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKnsL%252FO&md5=3721054e445191f88ac812b079cd83ac</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2015.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2015.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DMazier%26aufirst%3DW.%26aulast%3DSaucisse%26aufirst%3DN.%26aulast%3DGatta-Cherifi%26aufirst%3DB.%26aulast%3DCota%26aufirst%3DD.%26atitle%3DThe%2520endocannabinoid%2520system%253A%2520pivotal%2520orchestrator%2520of%2520obesity%2520and%2520metabolic%2520disease%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2015%26volume%3D26%26spage%3D524%26epage%3D537%26doi%3D10.1016%2Fj.tem.2015.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimberly, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nath, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keyes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larson, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corey, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerszten, R. E.</span></span> <span> </span><span class="NLM_article-title">Metabolite profiling identifies anandamide as a biomarker of nonalcoholic steatohepatitis</span>. <i>JCI Insight</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e92989</span> <span class="refDoi"> DOI: 10.1172/jci.insight.92989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1172%2Fjci.insight.92989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=28469090" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&author=W.+T.+Kimberlyauthor=J.+F.+O%E2%80%99Sullivanauthor=A.+K.+Nathauthor=M.+Keyesauthor=X.+Shiauthor=M.+G.+Larsonauthor=Q.+Yangauthor=M.+T.+Longauthor=R.+Vasanauthor=R.+T.+Petersonauthor=T.+J.+Wangauthor=K.+E.+Coreyauthor=R.+E.+Gerszten&title=Metabolite+profiling+identifies+anandamide+as+a+biomarker+of+nonalcoholic+steatohepatitis&doi=10.1172%2Fjci.insight.92989"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1172%2Fjci.insight.92989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci.insight.92989%26sid%3Dliteratum%253Aachs%26aulast%3DKimberly%26aufirst%3DW.%2BT.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DJ.%2BF.%26aulast%3DNath%26aufirst%3DA.%2BK.%26aulast%3DKeyes%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DLarson%26aufirst%3DM.%2BG.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DLong%26aufirst%3DM.%2BT.%26aulast%3DVasan%26aufirst%3DR.%26aulast%3DPeterson%26aufirst%3DR.%2BT.%26aulast%3DWang%26aufirst%3DT.%2BJ.%26aulast%3DCorey%26aufirst%3DK.%2BE.%26aulast%3DGerszten%26aufirst%3DR.%2BE.%26atitle%3DMetabolite%2520profiling%2520identifies%2520anandamide%2520as%2520a%2520biomarker%2520of%2520nonalcoholic%2520steatohepatitis%26jtitle%3DJCI%2520Insight%26date%3D2017%26volume%3D2%26doi%3D10.1172%2Fjci.insight.92989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fanelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mezzullo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repaci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belluomo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibarra Gasparini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Dalmazi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastroroberto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vicennati, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gambineri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morselli-Labate, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasquali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagotto, U.</span></span> <span> </span><span class="NLM_article-title">Profiling plasma <i>N</i>-acylethanolamine levels and their ratios as a biomarker of obesity and dysmetabolism</span>. <i>Mol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1016/j.molmet.2018.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.molmet.2018.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=29935920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=82-94&author=F.+Fanelliauthor=M.+Mezzulloauthor=A.+Repaciauthor=I.+Belluomoauthor=D.+Ibarra+Gaspariniauthor=G.+Di+Dalmaziauthor=M.+Mastrorobertoauthor=V.+Vicennatiauthor=A.+Gambineriauthor=A.+M.+Morselli-Labateauthor=R.+Pasqualiauthor=U.+Pagotto&title=Profiling+plasma+N-acylethanolamine+levels+and+their+ratios+as+a+biomarker+of+obesity+and+dysmetabolism&doi=10.1016%2Fj.molmet.2018.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling plasma N-Acylethanolamine levels and their ratios as a biomarker of obesity and dysmetabolism</span></div><div class="casAuthors">Fanelli, Flaminia; Mezzullo, Marco; Repaci, Andrea; Belluomo, Ilaria; Ibarra Gasparini, Daniela; Di Dalmazi, Guido; Mastroroberto, Marianna; Vicennati, Valentina; Gambineri, Alessandra; Morselli-Labate, Antonio Maria; Pasquali, Renato; Pagotto, Uberto</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Metabolism</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">82-94</span>CODEN:
                <span class="NLM_cas:coden">MMOEAS</span>;
        ISSN:<span class="NLM_cas:issn">2212-8778</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">N-acylethanolamines play different roles in energy balance; anandamide (AEA) stimulates energy intake and storage, N-palmitoylethanolamide (PEA) counters inflammation, and N-oleoylethanolamide (OEA) mediates anorectic signals and lipid oxidn.  Inconsistencies in the assocn. of plasma N-acylethanolamines with human obesity and cardiometabolic risk have emerged among previous studies, possibly caused by heterogeneous cohorts and designs, and by unstandardized N-acylethanolamine measurements.  We aimed to characterize changes in the plasma profile, including N-acylethanolamine levels and ratios assocd. with obesity, menopause in women, and aging in men, and to define the significance of such a profile as a biomarker for metabolic imbalance.  Adult, drug-free women (n=103 premenopausal and n=81 menopausal) and men (n=144) were stratified according to the body mass index (BMI) into normal wt. (NW; BMI: 18.5-24.9 kg/m2), overweight (OW; BMI: 25.0-29.9 kg/m2), and obese (OB; BMI:>30.0 kg/m2).  Anthropometric and metabolic parameters were detd.  Validated blood processing and anal. procedures for N-acylethanolamine measurements were used.  We investigated the effect of BMI and menopause in women, and BMI and age in men, as well as the BMI-independent influence of metabolic parameters on the N-acylethanolamine profile.  BMI and waist circumference directly assocd. with AEA in women and men, and with PEA in permenopausal women and in men, while BMI directly assocd. with OEA in premenopausal women and in men.  BMI, in both genders, and waist circumference, in women only, inversely assocd. with PEA/AEA and OEA/AEA.  Menopause increased N-acylethanolamine levels, whereas ageing resulted in increasing OEA relative abundance in men.  AEA and OEA abundances in premenopausal, and PEA and OEA abundances in lean menopausal women, were directly assocd. with hypertension.  Conversely, PEA and OEA abundances lowered with hypertension in elderly men.  Insulin resistance was assocd. with changes in N-acylethanolamine ratios specific for premenopausal (reduced PEA/AEA and OEA/AEA), menopausal (reduced OEA/AEA) women and men (reduced OEA/AEA and OEA/PEA).  PEA and OEA levels increased with total cholesterol, and OEA abundance specifically increased with HDL-cholesterol.  Elevated triglyceride levels were assocd. with increased N-acylethanolamine levels only in menopausal women.  Obesity-related N-acylethanolamine hypertone is characterized by imbalanced N-acylethanoamine ratios.  The profile given by a combination of N-acylethanolamine abs. levels and ratios enables imbalances to be identified in relationship with different metabolic parameters, with specific relevance according to gender, menopause and age, representing a useful means for monitoring metabolic health.  Finally, N-acylethanolamine system appears a promising target for intervention strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoohmBCJAciyrVg90H21EOLACvtfcHk0lidhIkgwEhpJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsb7F&md5=6170dd31651a203204ab6125c0b77f35</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.molmet.2018.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmet.2018.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DFanelli%26aufirst%3DF.%26aulast%3DMezzullo%26aufirst%3DM.%26aulast%3DRepaci%26aufirst%3DA.%26aulast%3DBelluomo%26aufirst%3DI.%26aulast%3DIbarra%2BGasparini%26aufirst%3DD.%26aulast%3DDi%2BDalmazi%26aufirst%3DG.%26aulast%3DMastroroberto%26aufirst%3DM.%26aulast%3DVicennati%26aufirst%3DV.%26aulast%3DGambineri%26aufirst%3DA.%26aulast%3DMorselli-Labate%26aufirst%3DA.%2BM.%26aulast%3DPasquali%26aufirst%3DR.%26aulast%3DPagotto%26aufirst%3DU.%26atitle%3DProfiling%2520plasma%2520N-acylethanolamine%2520levels%2520and%2520their%2520ratios%2520as%2520a%2520biomarker%2520of%2520obesity%2520and%2520dysmetabolism%26jtitle%3DMol.%2520Metab.%26date%3D2018%26volume%3D14%26spage%3D82%26epage%3D94%26doi%3D10.1016%2Fj.molmet.2018.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morishita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyatake, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Functional analysis of the purified anandamide-generating phospholipase D as a member of the metallo-β-lactamase family</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">12325</span>– <span class="NLM_lpage">12335</span>, <span class="refDoi"> DOI: 10.1074/jbc.M512359200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1074%2Fjbc.M512359200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=16527816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFKksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=12325-12335&author=J.+Wangauthor=Y.+Okamotoauthor=J.+Morishitaauthor=K.+Tsuboiauthor=A.+Miyatakeauthor=N.+Ueda&title=Functional+analysis+of+the+purified+anandamide-generating+phospholipase+D+as+a+member+of+the+metallo-%CE%B2-lactamase+family&doi=10.1074%2Fjbc.M512359200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Analysis of the Purified Anandamide-generating Phospholipase D as a Member of the Metallo-β-lactamase Family</span></div><div class="casAuthors">Wang, Jun; Okamoto, Yasuo; Morishita, Jun; Tsuboi, Kazuhito; Miyatake, Akira; Ueda, Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">12325-12335</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In animal tissues, bioactive N-acylethanolamines including the endocannabinoid anandamide are formed from their corresponding N-acylphosphatidylethanolamines (NAPEs) by the catalysis of a specific phospholipase D (NAPE-PLD) that belongs to the metallo-β-lactamase family.  Despite its potential physiol. importance, NAPE-PLD has not yet been characterized with a purified enzyme prepn.  In the present study we expressed a recombinant NAPE-PLD in Escherichia coli and highly purified it.  The purified enzyme was remarkably activated in a dose-dependent manner by millimolar concns. of Mg2+ as well as Ca2+ and, hence, appeared to be constitutively active.  The enzyme showed extremely high specificity for NAPEs among various glycerophospholipids but did not reveal obvious selectivity for different long chain or medium chain N-acyl species of NAPEs.  These results suggested the ability of NAPE-PLD to degrade different NAPEs without damaging other membrane phospholipids.  Metal anal. revealed the presence of catalytically important zinc in NAPE-PLD.  In addn., site-directed mutagenesis studies were addressed to several histidine and aspartic acid residues of NAPE-PLD that are highly conserved within the metallo-β-lactamase family.  Single mutations of Asp-147, His-185, His-187, Asp-189, His-190, His-253, Asp-284, and His-321 caused abolishment or remarkable redn. of the catalytic activity.  Moreover, when six cysteine residues were individually mutated to serine, only C224S showed a considerably reduced activity.  The activities of L207F and H380R found as single nucleotide polymorphisms were also low.  Thus, NAPE-PLD appeared to function through a mechanism similar to those of the well characterized members of this family but play a unique role in the lipid metab. of animal tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHPr_ieu3LxrVg90H21EOLACvtfcHk0lgYLgCUtehT8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFKksrY%253D&md5=aedf6e484205f3af45d2743f551de502</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M512359200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M512359200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DOkamoto%26aufirst%3DY.%26aulast%3DMorishita%26aufirst%3DJ.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DMiyatake%26aufirst%3DA.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DFunctional%2520analysis%2520of%2520the%2520purified%2520anandamide-generating%2520phospholipase%2520D%2520as%2520a%2520member%2520of%2520the%2520metallo-%25CE%25B2-lactamase%2520family%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D12325%26epage%3D12335%26doi%3D10.1074%2Fjbc.M512359200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">The stimulatory effect of phosphatidylethanolamine on <i>N</i>-acylphosphatidylethanolamine-hydrolyzing phospholipase D (NAPE-PLD)</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2007.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.neuropharm.2007.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=17655883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A280%3ADC%252BD1c%252FgtVWnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2008&pages=8-15&author=J.+Wangauthor=Y.+Okamotoauthor=K.+Tsuboiauthor=N.+Ueda&title=The+stimulatory+effect+of+phosphatidylethanolamine+on+N-acylphosphatidylethanolamine-hydrolyzing+phospholipase+D+%28NAPE-PLD%29&doi=10.1016%2Fj.neuropharm.2007.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The stimulatory effect of phosphatidylethanolamine on N-acylphosphatidylethanolamine-hydrolyzing phospholipase D (NAPE-PLD)</span></div><div class="casAuthors">Wang Jun; Okamoto Yasuo; Tsuboi Kazuhito; Ueda Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-15</span>
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    </div><div class="casAbstract">N-Acylphosphatidylethanolamine (NAPE)-hydrolyzing phospholipase D (NAPE-PLD) is a membrane-bound enzyme which releases the endocannabinoid anandamide and other bioactive N-acylethanolamines from their corresponding NAPEs in animal tissues.  Our previous studies showed that NAPE-PLD solubilized from the membrane is remarkably stimulated by millimolar concentrations of Ca(2+) while the membrane-bound form is much less sensitive to Ca(2+).  This finding suggested that certain membrane constituents diminished the stimulatory effect of Ca(2+).  In the present studies, we examined the effects of membrane fractions from COS-7 cells and brain tissue on the purified recombinant rat NAPE-PLD, and found that heat-stable membrane component(s) dose-dependently activated NAPE-PLD up to 4.8-5.0 fold.  In the presence of the membrane fractions, however, the stimulatory effect of Ca(2+) on the purified NAPE-PLD was considerably reduced.  When it was examined if the membrane fractions can be replaced with various pure phospholipids, phosphatidylethanolamine activated NAPE-PLD up to 3.3 fold, which was followed by decrease in the stimulatory effects of Ca(2+) and several other divalent cations.  These results suggest that membrane components including phosphatidylethanolamine keep the membrane-associated form of NAPE-PLD constitutively active.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTD0xdaMAOZYIJKJbLTgsvCfW6udTcc2eYIs3o0QlnWJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c%252FgtVWnsw%253D%253D&md5=68fdf5888fb468a4de4d50855e336772</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2007.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2007.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DOkamoto%26aufirst%3DY.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DThe%2520stimulatory%2520effect%2520of%2520phosphatidylethanolamine%2520on%2520N-acylphosphatidylethanolamine-hydrolyzing%2520phospholipase%2520D%2520%2528NAPE-PLD%2529%26jtitle%3DNeuropharmacology%26date%3D2008%26volume%3D54%26spage%3D8%26epage%3D15%26doi%3D10.1016%2Fj.neuropharm.2007.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magotti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babagoli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marotta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piomelli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garau, G.</span></span> <span> </span><span class="NLM_article-title">Structure of human <i>N</i>-acylphosphatidylethanolamine-hydrolyzing phospholipase D: regulation of fatty acid ethanolamide biosynthesis by bile acids</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1016/j.str.2014.12.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.str.2014.12.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=25684574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVSjt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=598-604&author=P.+Magottiauthor=I.+Bauerauthor=M.+Igarashiauthor=M.+Babagoliauthor=R.+Marottaauthor=D.+Piomelliauthor=G.+Garau&title=Structure+of+human+N-acylphosphatidylethanolamine-hydrolyzing+phospholipase+D%3A+regulation+of+fatty+acid+ethanolamide+biosynthesis+by+bile+acids&doi=10.1016%2Fj.str.2014.12.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of human N-acylphosphatidylethanolamine-hydrolyzing phospholipase D: Regulation of fatty acid ethanolamide biosynthesis by bile acids</span></div><div class="casAuthors">Magotti, Paola; Bauer, Inga; Igarashi, Miki; Babagoli, Masih; Marotta, Roberto; Piomelli, Daniele; Garau, Gianpiero</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-604</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The fatty acid ethanolamides (FAEs) are lipid mediators present in all organisms and involved in highly conserved biol. functions, such as innate immunity, energy balance, and stress control.  They are produced from membrane N-acylphosphatidylethanolamines (NAPEs) and include agonists for G protein-coupled receptors (e.g., cannabinoid receptors) and nuclear receptors (e.g., PPAR-α).  Here, the authors report the crystal structure of human NAPE-hydrolyzing phospholipase D (NAPE-PLD) at 2.65 Å resoln., a membrane enzyme that catalyzes FAE formation in mammals.  NAPE-PLD forms homodimers partly sepd. by an internal ∼9-Å-wide channel and uniquely adapted to assoc. with phospholipids.  A hydrophobic cavity provides an entryway for NAPE into the active site, where a binuclear Zn2+ center orchestrates its hydrolysis.  Bile acids bind with high affinity to selective pockets in this cavity, enhancing dimer assembly and enabling catalysis.  These elements offer multiple targets for the design of small-mol. NAPE-PLD modulators with potential applications in inflammation and metabolic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAkur49s4h2bVg90H21EOLACvtfcHk0lgYLgCUtehT8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVSjt7s%253D&md5=c80aae05403e5fc59a3bb6db1ec673ec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.12.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.12.018%26sid%3Dliteratum%253Aachs%26aulast%3DMagotti%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DIgarashi%26aufirst%3DM.%26aulast%3DBabagoli%26aufirst%3DM.%26aulast%3DMarotta%26aufirst%3DR.%26aulast%3DPiomelli%26aufirst%3DD.%26aulast%3DGarau%26aufirst%3DG.%26atitle%3DStructure%2520of%2520human%2520N-acylphosphatidylethanolamine-hydrolyzing%2520phospholipase%2520D%253A%2520regulation%2520of%2520fatty%2520acid%2520ethanolamide%2520biosynthesis%2520by%2520bile%2520acids%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D598%26epage%3D604%26doi%3D10.1016%2Fj.str.2014.12.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Margheritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magotti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peruzzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romeo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natali, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostarda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gioiello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piomelli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garau, G.</span></span> <span> </span><span class="NLM_article-title">Bile acid recognition by NAPE-PLD</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2908</span>– <span class="NLM_lpage">2914</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00624</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00624" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWltr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2908-2914&author=E.+Margheritisauthor=B.+Castellaniauthor=P.+Magottiauthor=S.+Peruzziauthor=E.+Romeoauthor=F.+Nataliauthor=S.+Mostardaauthor=A.+Gioielloauthor=D.+Piomelliauthor=G.+Garau&title=Bile+acid+recognition+by+NAPE-PLD&doi=10.1021%2Facschembio.6b00624"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Bile Acid Recognition by NAPE-PLD</span></div><div class="casAuthors">Margheritis, Eleonora; Castellani, Beatrice; Magotti, Paola; Peruzzi, Sara; Romeo, Elisa; Natali, Francesca; Mostarda, Serena; Gioiello, Antimo; Piomelli, Daniele; Garau, Gianpiero</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2908-2914</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The membrane-assocd. enzyme, N-acylphosphatidylethanolamine specific-phospholipase D (NAPE-PLD), generates the endogenous cannabinoid, arachidonylethanolamide, and other lipid signaling amides, including oleoylethanolamide and palmitoylethanolamide.  These bioactive mols. play important roles in several physiol. pathways including stress and pain response, appetite, and lifespan.  Recently, the authors reported the crystal structure of human NAPE-PLD and discovered specific binding sites for the bile acid, deoxycholic acid.  Here, the authors demonstrate that in the presence of this secondary bile acid, the stiffness of the protein measured by elastic neutron scattering increased, and NAPE-PLD resulted in ∼7-fold faster catalysis of the hydrolysis of the more unsatd. substrate, N-arachidonoylphosphatidylethanolamine, compared with N-palmitoylphosphatidylethanolamine.  Chenodeoxycholic acid and glyco- or tauro-dihydroxy conjugates could also bind to NAPE-PLD and drive its activation.  The only natural monohydroxy bile acid, lithocholic acid, showed an affinity of ∼20 μM and acted instead as a reversible inhibitor (IC50 = 68 μM).  Overall, these findings provided important insights into the allosteric regulation of the enzyme mediated by bile acid cofactors, and revealed that NAPE-PLD responds primarily to the no. and position of their OH groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXv-uNnhoOMrVg90H21EOLACvtfcHk0lh83Ofu2ehs4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWltr%252FL&md5=18527f3a1cc40d1305378c552b9685a7</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00624%26sid%3Dliteratum%253Aachs%26aulast%3DMargheritis%26aufirst%3DE.%26aulast%3DCastellani%26aufirst%3DB.%26aulast%3DMagotti%26aufirst%3DP.%26aulast%3DPeruzzi%26aufirst%3DS.%26aulast%3DRomeo%26aufirst%3DE.%26aulast%3DNatali%26aufirst%3DF.%26aulast%3DMostarda%26aufirst%3DS.%26aulast%3DGioiello%26aufirst%3DA.%26aulast%3DPiomelli%26aufirst%3DD.%26aulast%3DGarau%26aufirst%3DG.%26atitle%3DBile%2520acid%2520recognition%2520by%2520NAPE-PLD%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2908%26epage%3D2914%26doi%3D10.1021%2Facschembio.6b00624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Activation of <i>N</i>-acylethanolamine-releasing phospholipase D by polyamines</span>. <i>Chem. Phys. Lipids</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1016/s0009-3084(02)00015-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2FS0009-3084%2802%2900015-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=12047899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD38XktFOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2002&pages=77-84&author=Q.+Liuauthor=T.+Tonaiauthor=N.+Ueda&title=Activation+of+N-acylethanolamine-releasing+phospholipase+D+by+polyamines&doi=10.1016%2Fs0009-3084%2802%2900015-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of N-acylethanolamine-releasing phospholipase D by polyamines</span></div><div class="casAuthors">Liu, Qian; Tonai, Takeharu; Ueda, Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry and Physics of Lipids</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">77-84</span>CODEN:
                <span class="NLM_cas:coden">CPLIA4</span>;
        ISSN:<span class="NLM_cas:issn">0009-3084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">N-acylethanolamines including anandamide (an endogenous ligand of cannabinoid receptors) are biosynthesized from N-acyl-phosphatidylethanolamine (PE) by a phosphodiesterase of the phospholipase D type.  The enzyme partially purified from the particulate fraction of rat heart hydrolyzed N-palmitoyl-PE to N-palmitoylethanolamine with a specific activity of 50 nmol/min per mg protein at 37° in the presence of 10 mM CaCl2.  We found that the enzyme was highly activated in dose-dependent manner by polyamines like spermine, spermidine, and putrescine.  Spermine was the most potent with an EC50 value around 0.1 mM, and increased the specific enzyme activity 27 fold up to 53 nmol/min per mg protein.  However, a synergistic effect of spermine and the known activator (Ca2+ or Triton X-100) was not obsd.  The spermine-stimulated enzyme was also active with N-arachidonoyl-PE (a precursor of anandamide).  Thus, polyamines may function as endogenous activators to control the biosynthesis of anandamide and other N-acylethanolamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHZlLwSGWMBLVg90H21EOLACvtfcHk0lh83Ofu2ehs4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktFOhu70%253D&md5=4f5a830031ca427391b36a0806fee026</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0009-3084%2802%2900015-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-3084%252802%252900015-4%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DTonai%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DActivation%2520of%2520N-acylethanolamine-releasing%2520phospholipase%2520D%2520by%2520polyamines%26jtitle%3DChem.%2520Phys.%2520Lipids%26date%3D2002%26volume%3D115%26spage%3D77%26epage%3D84%26doi%3D10.1016%2Fs0009-3084%2802%2900015-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickering, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begtrup, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. S.</span></span> <span> </span><span class="NLM_article-title">Effect of synthetic and natural phospholipids on <i>N</i>-acylphosphatidylethanolamine-hydrolyzing phospholipase D activity</span>. <i>Chem. Phys. Lipids</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.chemphyslip.2009.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.chemphyslip.2009.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=19715685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Gnt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2009&pages=53-61&author=G.+Petersenauthor=A.+H.+Pedersenauthor=D.+S.+Pickeringauthor=M.+Begtrupauthor=H.+S.+Hansen&title=Effect+of+synthetic+and+natural+phospholipids+on+N-acylphosphatidylethanolamine-hydrolyzing+phospholipase+D+activity&doi=10.1016%2Fj.chemphyslip.2009.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of synthetic and natural phospholipids on N-acylphosphatidylethanolamine-hydrolyzing phospholipase D activity</span></div><div class="casAuthors">Petersen, Gitte; Pedersen, Anders H.; Pickering, Darryl S.; Begtrup, Mikael; Hansen, Harald S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry and Physics of Lipids</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">53-61</span>CODEN:
                <span class="NLM_cas:coden">CPLIA4</span>;
        ISSN:<span class="NLM_cas:issn">0009-3084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">N-Acylethanolamines (NAEs) constitute a family of endogenous bioactive lipids that includes arachidonoylethanolamide (anandamide), palmitoylethanolamide (PEA) and oleoylethanolamide (OEA).  These lipids are formed from their resp. N-acylated ethanolamine phospholipid (NAPE) precursor by the action of a phospholipase D enzyme (NAPE-PLD).  Anandamide, OEA, and PEA are all bioactive lipids that may influence, amongst others: neuroinflammation, food intake, and oocyte implantation.  Here we have synthesized a no. of NAPE analogs with variation in the phosphoester structure.  The NAPE analogs as well as selected phospholipids and β-lactamase substrates were tested as potential modifiers of cloned human NAPE-PLD in an enzyme assay involving a 14C-labeled diether-NAPE substrate.  One hit was identified, namely 1,2-dihexanoyl-glycero-N-(3-(tetradecanoylamino)propyl)phosphoramidate (AHP-71B) which showed inhibitory activity and may serve as template for further structure-activity developments.  Furthermore, NAPE-PLD was activated by phosphatidylethanolamine and inhibited by the β-lactamase substrate nitrocefin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh6CP241fNVbVg90H21EOLACvtfcHk0lh83Ofu2ehs4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Gnt7zP&md5=6280d2b0dac4471e3d8b0b6b4c294e29</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chemphyslip.2009.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chemphyslip.2009.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DG.%26aulast%3DPedersen%26aufirst%3DA.%2BH.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DBegtrup%26aufirst%3DM.%26aulast%3DHansen%26aufirst%3DH.%2BS.%26atitle%3DEffect%2520of%2520synthetic%2520and%2520natural%2520phospholipids%2520on%2520N-acylphosphatidylethanolamine-hydrolyzing%2520phospholipase%2520D%2520activity%26jtitle%3DChem.%2520Phys.%2520Lipids%26date%3D2009%26volume%3D162%26spage%3D53%26epage%3D61%26doi%3D10.1016%2Fj.chemphyslip.2009.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Reilly, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavieri, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, D.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larock, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of desketoraloxifene analogues as inhibitors of mammalian, Pseudomonas aeruginosa, and NAPE phospholipase D enzymes</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">432</span>, <span class="refDoi"> DOI: 10.1021/cb500828m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500828m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVyru77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=421-432&author=S.+A.+Scottauthor=C.+T.+Spencerauthor=M.+C.+O%E2%80%99Reillyauthor=K.+A.+Brownauthor=R.+R.+Lavieriauthor=C.-H.+Choauthor=D.-I.+Jungauthor=R.+C.+Larockauthor=H.+A.+Brownauthor=C.+W.+Lindsley&title=Discovery+of+desketoraloxifene+analogues+as+inhibitors+of+mammalian%2C+Pseudomonas+aeruginosa%2C+and+NAPE+phospholipase+D+enzymes&doi=10.1021%2Fcb500828m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Desketoraloxifene Analogues as Inhibitors of Mammalian, Pseudomonas aeruginosa, and NAPE Phospholipase D Enzymes</span></div><div class="casAuthors">Scott, Sarah A.; Spencer, Cierra T.; O'Reilly, Matthew C.; Brown, Kyle A.; Lavieri, Robert R.; Cho, Chul-Hee; Jung, Dai-Il; Larock, Richard C.; Brown, H. Alex; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">421-432</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phospholipase D (PLD) hydrolyzes cellular lipids to produce the important lipid second messenger phosphatidic acid.  A PLD enzyme expressed by Pseudomonas aeruginosa (PldA) has been shown to be important in bacterial infection, and NAPE-PLD has emerged as being key in the synthesis of endocannabinoids.  In order to better understand the biol. and therapeutic potential of these less explored PLD enzymes, small mol. tools are required.  Selective estrogen receptor modulators (SERMs) have been previously shown to inhibit mammalian PLD (PLD1 and PLD2).  By targeted screening of a library of SERM analogs, addnl. parallel synthesis, and evaluation in multiple PLD assays, the authors discovered a novel desketoraloxifene-based scaffold that inhibited not only the two mammalian PLDs but also structurally divergent PldA and NAPE-PLD.  This finding represents an important first step toward the development of small mols. possessing universal inhibition of divergent PLD enzymes to advance the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3qbMYFrILrLVg90H21EOLACvtfcHk0lizGJj8S0vozQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVyru77M&md5=864be7f1f24d7dfc4a46801162bdab55</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcb500828m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500828m%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DS.%2BA.%26aulast%3DSpencer%26aufirst%3DC.%2BT.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DM.%2BC.%26aulast%3DBrown%26aufirst%3DK.%2BA.%26aulast%3DLavieri%26aufirst%3DR.%2BR.%26aulast%3DCho%26aufirst%3DC.-H.%26aulast%3DJung%26aufirst%3DD.-I.%26aulast%3DLarock%26aufirst%3DR.%2BC.%26aulast%3DBrown%26aufirst%3DH.%2BA.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DDiscovery%2520of%2520desketoraloxifene%2520analogues%2520as%2520inhibitors%2520of%2520mammalian%252C%2520Pseudomonas%2520aeruginosa%252C%2520and%2520NAPE%2520phospholipase%2520D%2520enzymes%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D421%26epage%3D432%26doi%3D10.1021%2Fcb500828m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Castellani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamanti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pizzirani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maccesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Realini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magotti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakkum, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piomelli, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and characterization of the first inhibitor of <i>N</i>-acylphosphatidylethanolamine phospholipase D (NAPE-PLD)</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">12814</span>– <span class="NLM_lpage">12817</span>, <span class="refDoi"> DOI: 10.1039/C7CC07582K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1039%2FC7CC07582K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=29143042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWksLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=12814-12817&author=B.+Castellaniauthor=E.+Diamantiauthor=D.+Pizziraniauthor=P.+Tardiaauthor=M.+Maccesiauthor=N.+Realiniauthor=P.+Magottiauthor=G.+Garauauthor=T.+Bakkumauthor=S.+Rivaraauthor=M.+Morauthor=D.+Piomelli&title=Synthesis+and+characterization+of+the+first+inhibitor+of+N-acylphosphatidylethanolamine+phospholipase+D+%28NAPE-PLD%29&doi=10.1039%2FC7CC07582K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and characterization of the first inhibitor of N-acylphosphatidylethanolamine phospholipase D (NAPE-PLD)</span></div><div class="casAuthors">Castellani, Beatrice; Diamanti, Eleonora; Pizzirani, Daniela; Tardia, Piero; Maccesi, Martina; Realini, Natalia; Magotti, Paola; Garau, Gianpiero; Bakkum, Thomas; Rivara, Silvia; Mor, Marco; Piomelli, Daniele</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">95</span>),
    <span class="NLM_cas:pages">12814-12817</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">N-Acylphosphatidylethanolamine phospholipase D (NAPE-PLD) is a membrane-assocd. zinc enzyme that catalyzes the hydrolysis of N-acylphosphatidylethanolamines (NAPEs) into fatty acid ethanolamides (FAEs).  Here, we describe the identification of the first small-mol. NAPE-PLD inhibitor, the quinazoline sulfonamide deriv. 2,4-dioxo-N-[4-(4-pyridyl)phenyl]-1H-quinazoline-6-sulfonamide, ARN19874.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNZXUDG2Sz3bVg90H21EOLACvtfcHk0lizGJj8S0vozQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWksLbM&md5=ac246067aa18f87e5a7ccc00bc2395e1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1039%2FC7CC07582K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC07582K%26sid%3Dliteratum%253Aachs%26aulast%3DCastellani%26aufirst%3DB.%26aulast%3DDiamanti%26aufirst%3DE.%26aulast%3DPizzirani%26aufirst%3DD.%26aulast%3DTardia%26aufirst%3DP.%26aulast%3DMaccesi%26aufirst%3DM.%26aulast%3DRealini%26aufirst%3DN.%26aulast%3DMagotti%26aufirst%3DP.%26aulast%3DGarau%26aufirst%3DG.%26aulast%3DBakkum%26aufirst%3DT.%26aulast%3DRivara%26aufirst%3DS.%26aulast%3DMor%26aufirst%3DM.%26aulast%3DPiomelli%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520characterization%2520of%2520the%2520first%2520inhibitor%2520of%2520N-acylphosphatidylethanolamine%2520phospholipase%2520D%2520%2528NAPE-PLD%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D12814%26epage%3D12817%26doi%3D10.1039%2FC7CC07582K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrow, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashhadi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S. S.</span></span> <span> </span><span class="NLM_article-title">Symmetrically substituted dichlorophenes inhibit <i>N</i>-acyl-phosphatidylethanolamine phospholipase D</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">7289</span>– <span class="NLM_lpage">7300</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA120.013362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1074%2Fjbc.RA120.013362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=32284327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Wqt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2020&pages=7289-7300&author=G.+Aggarwalauthor=J.+E.+Zarrowauthor=Z.+Mashhadiauthor=C.+R.+Flynnauthor=P.+Vinsonauthor=C.+D.+Weaverauthor=S.+S.+Davies&title=Symmetrically+substituted+dichlorophenes+inhibit+N-acyl-phosphatidylethanolamine+phospholipase+D&doi=10.1074%2Fjbc.RA120.013362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Symmetrically substituted dichlorophenes inhibit N-acyl-phosphatidylethanolamine phospholipase D</span></div><div class="casAuthors">Aggarwal, Geetika; Zarrow, Jonah E.; Mashhadi, Zahra; Flynn, C. Robb; Vinson, Paige; Weaver, C. David; Davies, Sean S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7289-7301</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">1083-351X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">N-Acyl-phosphatidylethanolamine phospholipase D (NAPE-PLD) (EC 3.1.4.4) catalyzes the final step in the biosynthesis of N-acyl-ethanolamides.  Reduced NAPE-PLD expression and activity may contribute to obesity and inflammation, but a lack of effective NAPE-PLD inhibitors has been a major obstacle to elucidating the role of NAPE-PLD and N-acyl-ethanolamide biosynthesis in these processes.  The endogenous bile acid lithocholic acid (LCA) inhibits NAPE-PLD activity (with an IC50 of 68 μm), but LCA is also a highly potent ligand for TGR5 (EC50 0.52 μm).  Recently, the first selective small-mol. inhibitor of NAPE-PLD, ARN19874, has been reported (having an IC50 of 34 μm).  To identify more potent inhibitors of NAPE-PLD, here we used a quenched fluorescent NAPE analog, PED-A1, as a substrate for recombinant mouse Nape-pld to screen a panel of bile acids and a library of exptl. compds. (the Spectrum Collection).  Muricholic acids and several other bile acids inhibited Nape-pld with potency similar to that of LCA.  We identified 14 potent Nape-pld inhibitors in the Spectrum Collection, with the two most potent (IC50 = ~ 2 μm) being sym. substituted dichlorophenes, i.e. hexachlorophene and bithionol.  Structure-activity relationship assays using addnl. substituted dichlorophenes identified key moieties needed for Nape-pld inhibition.  Both hexachlorophene and bithionol exhibited significant selectivity for Nape-pld compared with nontarget lipase activities such as Streptomyces chromofuscus PLD or serum lipase.  Both also effectively inhibited NAPE-PLD activity in cultured HEK293 cells.  We conclude that sym. substituted dichlorophenes potently inhibit NAPE-PLD in cultured cells and have significant selectivity for NAPE-PLD vs. other tissue-assocd. lipases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE49oiAvett7Vg90H21EOLACvtfcHk0lizGJj8S0vozQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Wqt7fM&md5=66f0e9624585923c106958bd3fe3e221</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA120.013362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA120.013362%26sid%3Dliteratum%253Aachs%26aulast%3DAggarwal%26aufirst%3DG.%26aulast%3DZarrow%26aufirst%3DJ.%2BE.%26aulast%3DMashhadi%26aufirst%3DZ.%26aulast%3DFlynn%26aufirst%3DC.%2BR.%26aulast%3DVinson%26aufirst%3DP.%26aulast%3DWeaver%26aufirst%3DC.%2BD.%26aulast%3DDavies%26aufirst%3DS.%2BS.%26atitle%3DSymmetrically%2520substituted%2520dichlorophenes%2520inhibit%2520N-acyl-phosphatidylethanolamine%2520phospholipase%2520D%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2020%26volume%3D295%26spage%3D7289%26epage%3D7300%26doi%3D10.1074%2Fjbc.RA120.013362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shuman, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leech, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvord, E. C.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Neurotoxicity of hexachlorophene in humans: II. a clinicopathological study of 46 premature infants</span>. <i>Arch. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1001/archneur.1975.00490470064009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1001%2Farchneur.1975.00490470064009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1137506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A280%3ADyaE2M7ot12rtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1975&pages=320-325&author=R.+M.+Shumanauthor=R.+W.+Leechauthor=E.+C.+Alvord&title=Neurotoxicity+of+hexachlorophene+in+humans%3A+II.+a+clinicopathological+study+of+46+premature+infants&doi=10.1001%2Farchneur.1975.00490470064009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Neurotoxicity of hexachlorophene in humans. II. A clinicopathological study of 46 premature infants</span></div><div class="casAuthors">Shuman R M; Leech R W; Alvord E C Jr</div><div class="citationInfo"><span class="NLM_cas:title">Archives of neurology</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">320-5</span>
        ISSN:<span class="NLM_cas:issn">0003-9942</span>.
    </div><div class="casAbstract">To assess neurotoxic effects of hexachlorophene in the human population previously shown to be most at risk, a blind clinicopathological analysis was made of all premature infants under 1,400 gm birth weight who survived at least four days and were examined by autopsy over a 7.5-year period.  Repeated whole-body bathing of premature newborn infants in 3% hexachlorophene-bearing soap (undiluted pHisoHex) shows a significant statistical association with a vacuolar encephalopathy of the brain stem reticular formation.  The prevalence of the vacuolar encephalopathy in premature infants on whom we have adequate brain stem histological information appears to be related to the number of exposures to hexachlorophene, the concentration of hexachlorophene, the thoroughness of rinsing, and other factors (including exposure to ultraviolet light).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQS2wRdp4ZhvrP3FVUi_gf8fW6udTcc2eZ3aWxKzcgFBLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE2M7ot12rtg%253D%253D&md5=aa23a37d65d73e459d35cca655b10f8d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1001%2Farchneur.1975.00490470064009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Farchneur.1975.00490470064009%26sid%3Dliteratum%253Aachs%26aulast%3DShuman%26aufirst%3DR.%2BM.%26aulast%3DLeech%26aufirst%3DR.%2BW.%26aulast%3DAlvord%26aufirst%3DE.%2BC.%26atitle%3DNeurotoxicity%2520of%2520hexachlorophene%2520in%2520humans%253A%2520II.%2520a%2520clinicopathological%2520study%2520of%252046%2520premature%2520infants%26jtitle%3DArch.%2520Neurol.%26date%3D1975%26volume%3D32%26spage%3D320%26epage%3D325%26doi%3D10.1001%2Farchneur.1975.00490470064009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudge, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Design and preparation of new palladium precatalysts for C–C and C–N cross-coupling reactions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">920</span>, <span class="refDoi"> DOI: 10.1039/C2SC20903A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1039%2FC2SC20903A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=23667737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=916-920&author=N.+C.+Brunoauthor=M.+T.+Tudgeauthor=S.+L.+Buchwald&title=Design+and+preparation+of+new+palladium+precatalysts+for+C%E2%80%93C+and+C%E2%80%93N+cross-coupling+reactions&doi=10.1039%2FC2SC20903A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Design and preparation of new palladium precatalysts for C-C and C-N cross-coupling reactions</span></div><div class="casAuthors">Bruno, Nicholas C.; Tudge, Matthew T.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">916-920</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A series of easily prepd., phosphine-ligated palladium precatalysts based on the 2-aminobiphenyl scaffold have been prepd.  The role of the precatalyst-assocd. labile halide (or pseudohalide) in the formation and stability of the palladacycle has been examd.  It was found that replacing the chloride in the previous version of the precatalyst with a mesylate leads to a new class of precatalysts with improved soln. stability and that are readily prepd. from a wider range of phosphine ligands.  The differences between the previous version of precatalyst and that reported here are explored.  In addn., the reactivity of the latter is examd. in a range of C-C and C-N bond forming reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfHp6oyg4k_bVg90H21EOLACvtfcHk0lhurLe9gJHlOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKjtr0%253D&md5=5b50ac18cb67251e5439199c6746dba8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC2SC20903A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2SC20903A%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DN.%2BC.%26aulast%3DTudge%26aufirst%3DM.%2BT.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DDesign%2520and%2520preparation%2520of%2520new%2520palladium%2520precatalysts%2520for%2520C%25E2%2580%2593C%2520and%2520C%25E2%2580%2593N%2520cross-coupling%2520reactions%26jtitle%3DChem.%2520Sci.%26date%3D2013%26volume%3D4%26spage%3D916%26epage%3D920%26doi%3D10.1039%2FC2SC20903A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quach, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batey, R. A.</span></span> <span> </span><span class="NLM_article-title">Ligand- and base-free copper(II)-catalyzed C–N bond formation: cross-coupling reactions of organoboron compounds with aliphatic amines and anilines</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">4397</span>– <span class="NLM_lpage">4400</span>, <span class="refDoi"> DOI: 10.1021/ol035681s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol035681s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotlartLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2003&pages=4397-4400&author=T.+D.+Quachauthor=R.+A.+Batey&title=Ligand-+and+base-free+copper%28II%29-catalyzed+C%E2%80%93N+bond+formation%3A+cross-coupling+reactions+of+organoboron+compounds+with+aliphatic+amines+and+anilines&doi=10.1021%2Fol035681s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand- and Base-Free Copper(II)-Catalyzed C-N Bond Formation: Cross-Coupling Reactions of Organoboron Compounds with Aliphatic Amines and Anilines</span></div><div class="casAuthors">Quach, Tan D.; Batey, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4397-4400</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A ligandless and base-free Cu-catalyzed protocol for the cross-coupling of arylboronic acids and potassium aryltrifluoroborate salts with primary and secondary aliph. amines and anilines to yield a wide variety of secondary and tertiary amines is described.  The process utilizes catalytic copper(II) acetate monohydrate and 4 Å mol. sieves in dichloromethane at slightly elevated temps. under an atm. of oxygen.  A broad range of functional groups is tolerated on both of the cross-coupling partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraNMwqBwBxFrVg90H21EOLACvtfcHk0lhurLe9gJHlOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotlartLY%253D&md5=af170572d82b811ba5d1d574bde81372</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fol035681s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol035681s%26sid%3Dliteratum%253Aachs%26aulast%3DQuach%26aufirst%3DT.%2BD.%26aulast%3DBatey%26aufirst%3DR.%2BA.%26atitle%3DLigand-%2520and%2520base-free%2520copper%2528II%2529-catalyzed%2520C%25E2%2580%2593N%2520bond%2520formation%253A%2520cross-coupling%2520reactions%2520of%2520organoboron%2520compounds%2520with%2520aliphatic%2520amines%2520and%2520anilines%26jtitle%3DOrg.%2520Lett.%26date%3D2003%26volume%3D5%26spage%3D4397%26epage%3D4400%26doi%3D10.1021%2Fol035681s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talele, T. T.</span></span> <span> </span><span class="NLM_article-title">The “cyclopropyl fragment” is a versatile player that frequently appears in preclinical/clinical drug molecules</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8712</span>– <span class="NLM_lpage">8756</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00472</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00472" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8712-8756&author=T.+T.+Talele&title=The+%E2%80%9Ccyclopropyl+fragment%E2%80%9D+is+a+versatile+player+that+frequently+appears+in+preclinical%2Fclinical+drug+molecules&doi=10.1021%2Facs.jmedchem.6b00472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The "Cyclopropyl Fragment" is a Versatile Player that Frequently Appears in Preclinical/Clinical Drug Molecules</span></div><div class="casAuthors">Talele, Tanaji T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8712-8756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing use of the cyclopropyl ring in drug development to transition drug candidates from the preclin. to clin. stage.  Important features of the cyclopropane ring are, the (1) coplanarity of the three carbon atoms, (2) relatively shorter (1.51 Å) C-C bonds, (3) enhanced π-character of C-C bonds, and (4) C-H bonds are shorter and stronger than those in alkanes.  The present review will focus on the contributions that a cyclopropyl ring makes to the properties of drugs contg. it.  Consequently, the cyclopropyl ring addresses multiple roadblocks that can occur during drug discovery such as (a) enhancing potency, (b) reducing off-target effects, (c) increasing metabolic stability, (d) increasing brain permeability, (e) decreasing plasma clearance, (f) contributing to an entropically more favorable binding to the receptor, (g) conformational restriction of peptides/peptidomimetics to prevent proteolytic hydrolysis, and (h) altering drug pKa to reduce its P-glycoprotein efflux ratio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVIJAG1XVRG7Vg90H21EOLACvtfcHk0liC_TFUWl2Emg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D&md5=b5586458ec4ab5e2d7586f5739663d1f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00472%26sid%3Dliteratum%253Aachs%26aulast%3DTalele%26aufirst%3DT.%2BT.%26atitle%3DThe%2520%25E2%2580%259Ccyclopropyl%2520fragment%25E2%2580%259D%2520is%2520a%2520versatile%2520player%2520that%2520frequently%2520appears%2520in%2520preclinical%252Fclinical%2520drug%2520molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8712%26epage%3D8756%26doi%3D10.1021%2Facs.jmedchem.6b00472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenau, A.</span></span> <span> </span><span class="NLM_article-title">A practical strategy for the synthesis of 2-dialkylamino-4-arylamino-6-aminopyrimidines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5888</span>– <span class="NLM_lpage">5893</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2009.07.154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.tetlet.2009.07.154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFarsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2009&pages=5888-5893&author=C.+Liauthor=A.+Rosenau&title=A+practical+strategy+for+the+synthesis+of+2-dialkylamino-4-arylamino-6-aminopyrimidines&doi=10.1016%2Fj.tetlet.2009.07.154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A practical strategy for the synthesis of 2-dialkylamino-4-arylamino-6-aminopyrimidines</span></div><div class="casAuthors">Li, Chaomin; Rosenau, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">5888-5893</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Starting from com. available 4-amino-2,6-dichloropyrimidine, a practical 4-step synthesis of 2-dialkylamino-4-arylamino-6-aminopyrimidines was developed.  This strategy could introduce a diverse set of secondary amines and arylamines to displace the 2- and 4-Cl groups.  The products of this route are otherwise difficult to access.  In addn., 6-amino arylation was carried out to demonstrate the reactivity and utility of 2-dialkylamino-4-arylamino-6-aminopyrimidines as building blocks for assembling interesting pyrimidinamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtl2shPWSpcrVg90H21EOLACvtfcHk0liC_TFUWl2Emg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFarsrrJ&md5=dfa22f29b058d81cfbb9ca427184457d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2009.07.154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2009.07.154%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DRosenau%26aufirst%3DA.%26atitle%3DA%2520practical%2520strategy%2520for%2520the%2520synthesis%2520of%25202-dialkylamino-4-arylamino-6-aminopyrimidines%26jtitle%3DTetrahedron%2520Lett.%26date%3D2009%26volume%3D50%26spage%3D5888%26epage%3D5893%26doi%3D10.1016%2Fj.tetlet.2009.07.154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of reported NAPE-PLD inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of <i>in vivo</i> active NAPE-PLD inhibitor <b>1</b> (<b>LEI-401</b>), HTS-hit <b>2</b>, and the core pyrimidine-4-carboxamide scaffold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyridyl Analogues <b>3</b> and <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>N</i>-methylphenethylamine, DiPEA, MeOH, rt, 41%; (b) NaOH, THF, H<sub>2</sub>O, rt; for <b>110</b>: 89% and for <b>114</b>: 99%; (c) cyclopropylmethanamine, EDC·HCl, HOBt, DCM, rt; for <b>111</b>: 24% and for <b>115</b>: 80%; (d) morpholine, RuPhos-Pd-G3, NaOtBu, THF, toluene, 110 °C, 37%; (e) morpholine, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, reflux, 66%; (f) <i>N</i>-methylphenethylamine, RuPhos-Pd-G3, NaOtBu, THF, toluene, 110 °C, 41%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. (A) General Synthetic Route for Analogues of Compound <b>2</b>; (B) Alternative Synthetic Route for Amide Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) POCl<sub>3</sub>, DMF, reflux, 60%; (b) R<sub>1</sub>NH<sub>2</sub>, Et<sub>3</sub>N, DCM, −78 °C to 0 °C, 78–99%; (c) (cyclo)alkylNH, DiPEA, MeOH, 0 °C, 32–99% or (hetero)arylOH or heteroarylNH, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 51–76%; (d) <b>123a</b>–<b>o</b> or <b>124</b> or <b>125a</b>–<b>e</b>, DiPEA, <i>n</i>-BuOH, μW, 160 °C or oil bath, 120 °C, 21% – 97%; (e) KCN, EtOH, dioxane, H<sub>2</sub>O, reflux, 84–99%; (f) H<sub>2</sub>, Pd/C, HCl, EtOH, rt, 98–99%; (g) methyl chloroformate, DiPEA, DCM, 0 °C to rt; (h) LiAlH<sub>4</sub>, THF, 0 °C to reflux, 40–94% over 2 steps; (i) phenylboronic acid, Cu(OAc)<sub>2</sub>·H<sub>2</sub>O, 4 Å MS, O<sub>2</sub>, DCE, rt, 32%; (j) aldehyde or ketone, NaB(OAc)<sub>3</sub>H, AcOH, DCM, rt, 18–63%. (k) NaOH, THF, MeOH, H<sub>2</sub>O, rt, 99%; (l) <i>N</i>-methylphenethylamine, DiPEA, <i>n</i>-BuOH, 120 °C, 51%; (m) R<sub>1</sub>NH<sub>2</sub>, PyBOP, DiPEA, DMF, 0 °C to rt, 43–55%.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/medium/jm0c01441_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure–activity map for the pyrimidine-4-carboxamide NAPE-PLD inhibitor library.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01441/20210107/images/large/jm0c01441_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01441&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i262">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60187" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60187" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 22 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mock, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustafa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunduz-Cinar, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cinar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrie, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantae, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varga, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paloczi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miliano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donvito, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Gracht, A. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotsogianni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martella, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Wel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soethoudt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendel, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakker, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillo, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florea, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Hurk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grether, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Boeckel, C. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hankemeier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buczynski, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacher, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichtman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a NAPE-PLD inhibitor that modulates emotional behavior in mice</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0528-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1038%2Fs41589-020-0528-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=32393901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptVKhs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=667-675&author=E.+D.+Mockauthor=M.+Mustafaauthor=O.+Gunduz-Cinarauthor=R.+Cinarauthor=G.+N.+Petrieauthor=V.+Kantaeauthor=X.+Diauthor=D.+Ogasawaraauthor=Z.+V.+Vargaauthor=J.+Palocziauthor=C.+Milianoauthor=G.+Donvitoauthor=A.+C.+M.+van+Esbroeckauthor=A.+M.+F.+van+der+Grachtauthor=I.+Kotsogianniauthor=J.+K.+Parkauthor=A.+Martellaauthor=T.+van+der+Welauthor=M.+Soethoudtauthor=M.+Jiangauthor=T.+J.+Wendelauthor=A.+P.+A.+Janssenauthor=A.+T.+Bakkerauthor=C.+M.+Donovanauthor=L.+I.+Castilloauthor=B.+I.+Floreaauthor=J.+Watauthor=H.+van+den+Hurkauthor=M.+Wittwerauthor=U.+Gretherauthor=A.+Holmesauthor=C.+A.+A.+van+Boeckelauthor=T.+Hankemeierauthor=B.+F.+Cravattauthor=M.+W.+Buczynskiauthor=M.+N.+Hillauthor=P.+Pacherauthor=A.+H.+Lichtmanauthor=M.+van+der%0AStelt&title=Discovery+of+a+NAPE-PLD+inhibitor+that+modulates+emotional+behavior+in+mice&doi=10.1038%2Fs41589-020-0528-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a NAPE-PLD inhibitor that modulates emotional behavior in mice</span></div><div class="casAuthors">Mock, Elliot D.; Mustafa, Mohammed; Gunduz-Cinar, Ozge; Cinar, Resat; Petrie, Gavin N.; Kantae, Vasudev; Di, Xinyu; Ogasawara, Daisuke; Varga, Zoltan V.; Paloczi, Janos; Miliano, Cristina; Donvito, Giulia; van Esbroeck, Annelot C. M.; van der Gracht, Anouk M. F.; Kotsogianni, Ioli; Park, Joshua K.; Martella, Andrea; van der Wel, Tom; Soethoudt, Marjolein; Jiang, Ming; Wendel, Tiemen J.; Janssen, Antonius P. A.; Bakker, Alexander T.; Donovan, Colleen M.; Castillo, Laura I.; Florea, Bogdan I.; Wat, Jesse; van den Hurk, Helma; Wittwer, Matthias; Grether, Uwe; Holmes, Andrew; van Boeckel, Constant A. A.; Hankemeier, Thomas; Cravatt, Benjamin F.; Buczynski, Matthew W.; Hill, Matthew N.; Pacher, Pal; Lichtman, Aron H.; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">667-675</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: N-acylethanolamines (NAEs), which include the endocannabinoid anandamide, represent an important family of signaling lipids in the brain.  The lack of chem. probes that modulate NAE biosynthesis in living systems hamper the understanding of the biol. role of these lipids.  Using a high-throughput screen, chem. proteomics and targeted lipidomics, we report here the discovery and characterization of LEI-401 as a CNS-active N-acylphosphatidylethanolamine phospholipase D (NAPE-PLD) inhibitor.  LEI-401 reduced NAE levels in neuroblastoma cells and in the brain of freely moving mice, but not in NAPE-PLD KO cells and mice, resp.  LEI-401 activated the hypothalamus-pituitary-adrenal axis and impaired fear extinction, thereby emulating the effect of a cannabinoid CB1 receptor antagonist, which could be reversed by a fatty acid amide hydrolase inhibitor.  Our findings highlight the distinctive role of NAPE-PLD in NAE biosynthesis in the brain and suggest the presence of an endogenous NAE tone controlling emotional behavior. [graphic not available: see fulltext].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzC9MSWVmbH7Vg90H21EOLACvtfcHk0lhp8Y5abWgjSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptVKhs7g%253D&md5=093b78b3c98c31031915eea3bb596a18</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0528-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0528-7%26sid%3Dliteratum%253Aachs%26aulast%3DMock%26aufirst%3DE.%2BD.%26aulast%3DMustafa%26aufirst%3DM.%26aulast%3DGunduz-Cinar%26aufirst%3DO.%26aulast%3DCinar%26aufirst%3DR.%26aulast%3DPetrie%26aufirst%3DG.%2BN.%26aulast%3DKantae%26aufirst%3DV.%26aulast%3DDi%26aufirst%3DX.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DVarga%26aufirst%3DZ.%2BV.%26aulast%3DPaloczi%26aufirst%3DJ.%26aulast%3DMiliano%26aufirst%3DC.%26aulast%3DDonvito%26aufirst%3DG.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3Dvan%2Bder%2BGracht%26aufirst%3DA.%2BM.%2BF.%26aulast%3DKotsogianni%26aufirst%3DI.%26aulast%3DPark%26aufirst%3DJ.%2BK.%26aulast%3DMartella%26aufirst%3DA.%26aulast%3Dvan%2Bder%2BWel%26aufirst%3DT.%26aulast%3DSoethoudt%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DM.%26aulast%3DWendel%26aufirst%3DT.%2BJ.%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3DBakker%26aufirst%3DA.%2BT.%26aulast%3DDonovan%26aufirst%3DC.%2BM.%26aulast%3DCastillo%26aufirst%3DL.%2BI.%26aulast%3DFlorea%26aufirst%3DB.%2BI.%26aulast%3DWat%26aufirst%3DJ.%26aulast%3Dvan%2Bden%2BHurk%26aufirst%3DH.%26aulast%3DWittwer%26aufirst%3DM.%26aulast%3DGrether%26aufirst%3DU.%26aulast%3DHolmes%26aufirst%3DA.%26aulast%3Dvan%2BBoeckel%26aufirst%3DC.%2BA.%2BA.%26aulast%3DHankemeier%26aufirst%3DT.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DBuczynski%26aufirst%3DM.%2BW.%26aulast%3DHill%26aufirst%3DM.%2BN.%26aulast%3DPacher%26aufirst%3DP.%26aulast%3DLichtman%26aufirst%3DA.%2BH.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520NAPE-PLD%2520inhibitor%2520that%2520modulates%2520emotional%2520behavior%2520in%2520mice%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D667%26epage%3D675%26doi%3D10.1038%2Fs41589-020-0528-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Mammalian enzymes responsible for the biosynthesis of <i>N</i>-acylethanolamines</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1862</span>,  <span class="NLM_volume"><i>2017</i></span>,  <span class="NLM_fpage">1546</span>– <span class="NLM_lpage">1561</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=1862&pages=1546-1561&author=Z.+Hussainauthor=T.+Uyamaauthor=K.+Tsuboiauthor=N.+Ueda&title=Mammalian+enzymes+responsible+for+the+biosynthesis+of+N-acylethanolamines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DZ.%26aulast%3DUyama%26aufirst%3DT.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DMammalian%2520enzymes%2520responsible%2520for%2520the%2520biosynthesis%2520of%2520N-acylethanolamines%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1862%26volume%3D2017%26spage%3D1546%26epage%3D1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morishita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Molecular characterization of a phospholipase D generating anandamide and its congeners</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">5298</span>– <span class="NLM_lpage">5305</span>, <span class="refDoi"> DOI: 10.1074/jbc.M306642200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1074%2Fjbc.M306642200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=14634025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFeqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=5298-5305&author=Y.+Okamotoauthor=J.+Morishitaauthor=K.+Tsuboiauthor=T.+Tonaiauthor=N.+Ueda&title=Molecular+characterization+of+a+phospholipase+D+generating+anandamide+and+its+congeners&doi=10.1074%2Fjbc.M306642200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Characterization of a Phospholipase D Generating Anandamide and Its Congeners</span></div><div class="casAuthors">Okamoto, Yasuo; Morishita, Jun; Tsuboi, Kazuhito; Tonai, Takeharu; Ueda, Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5298-5305</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Anandamide (N-arachidonoylethanolamine) is known to be an endogenous ligand of cannabinoid and vanilloid receptors.  Its congeners (collectively referred to as N-acylethanolamines) also show a variety of biol. activities.  These compds. are principally formed from their corresponding N-acylphosphatidylethanolamines by a phosphodiesterase of the phospholipase D-type in animal tissues.  The authors purified the enzyme from rat heart, and by the use of the sequences of its internal peptides cloned its cDNAs from mouse, rat, and human.  The deduced amino acid sequences were composed of 393-396 residues, and showed that the enzyme has no homol. with the known phospholipase D enzymes but is classified as a member of the zinc metallohydrolase family of the β-lactamase fold.  As was overexpressed in COS-7 cells, the recombinant enzyme generated anandamide and other N-acylethanolamines from their corresponding N-acylphosphatidylethanolamines at comparable rates.  In contrast, the enzyme was inactive with phosphatidylcholine and phosphatidylethanolamine.  Assays of the enzyme activity and the mRNA and protein levels revealed its wide distribution in murine organs with higher contents in the brain, kidney, and testis.  These results confirm that a specific phospholipase D is responsible for the generation of N-acylethanolamines including anandamide, strongly suggesting the physiol. importance of lipid mols. of this class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK_-VVe-jk7rVg90H21EOLACvtfcHk0lgvsDyApAhlvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFeqtrY%253D&md5=f1c00a4d0e21163c03b5a70d2f46c721</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M306642200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M306642200%26sid%3Dliteratum%253Aachs%26aulast%3DOkamoto%26aufirst%3DY.%26aulast%3DMorishita%26aufirst%3DJ.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DTonai%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DMolecular%2520characterization%2520of%2520a%2520phospholipase%2520D%2520generating%2520anandamide%2520and%2520its%2520congeners%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D5298%26epage%3D5305%26doi%3D10.1074%2Fjbc.M306642200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Endocannabinoids and related <i>N</i>-acylethanolamines: biological activities and metabolism</span>. <i>Inflammation Regener.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">28</span>, <span class="refDoi"> DOI: 10.1186/s41232-018-0086-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1186%2Fs41232-018-0086-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=30288203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1eqt7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=28&author=K.+Tsuboiauthor=T.+Uyamaauthor=Y.+Okamotoauthor=N.+Ueda&title=Endocannabinoids+and+related+N-acylethanolamines%3A+biological+activities+and+metabolism&doi=10.1186%2Fs41232-018-0086-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Endocannabinoids and related N-acylethanolamines: biological activities and metabolism</span></div><div class="casAuthors">Tsuboi, Kazuhito; Uyama, Toru; Okamoto, Yasuo; Ueda, Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Inflammation and Regeneration</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">28</span>CODEN:
                <span class="NLM_cas:coden">IRNECG</span>;
        ISSN:<span class="NLM_cas:issn">1880-9693</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The plant Cannabis sativa contains cannabinoids represented by Δ9-tetrahydrocannabinol, which exert psychoactivity and immunomodulation through cannabinoid CB1 and CB2 receptors, resp., in animal tissues.  Arachidonoylethanolamide (also referred to as anandamide) and 2-arachidonoylglycerol (2-AG) are well known as two major endogenous agonists of these receptors (termed "endocannabinoids") and show various cannabimimetic bioactivities.  However, only 2-AG is a full agonist for CB1 and CB2 and mediates retrograde signals at the synapse, strongly suggesting that 2-AG is physiol. more important than anandamide.  The metabolic pathways of these two endocannabinoids are completely different. 2-AG is mostly produced from inositol phospholipids via diacylglycerol by phospholipase C and diacylglycerol lipase and then degraded by monoacylglycerol lipase.  On the other hand, anandamide is concomitantly produced with larger amts. of other N-acylethanolamines via N-acyl-phosphatidylethanolamines (NAPEs).  Although this pathway consists of calcium-dependent N-acyltransferase and NAPE-hydrolyzing phospholipase D, recent studies revealed the involvement of several new enzymes.  Quant. major N-acylethanolamines include palmitoylethanolamide and oleoylethanolamide, which do not bind to cannabinoid receptors but exert anti-inflammatory, analgesic, and anorexic effects through receptors such as peroxisome proliferator-activated receptor α.  The biosynthesis of these non-endocannabinoid N-acylethanolamines rather than anandamide may be the primary significance of this pathway.  Here, we provide an overview of the biol. activities and metabs. of endocannabinoids (2-AG and anandamide) and non-endocannabinoid N-acylethanolamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUWzSp4LibbLVg90H21EOLACvtfcHk0lgvsDyApAhlvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1eqt7zN&md5=0ac9f596d18fd9d769a1f67f18bf8223</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1186%2Fs41232-018-0086-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs41232-018-0086-5%26sid%3Dliteratum%253Aachs%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DUyama%26aufirst%3DT.%26aulast%3DOkamoto%26aufirst%3DY.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DEndocannabinoids%2520and%2520related%2520N-acylethanolamines%253A%2520biological%2520activities%2520and%2520metabolism%26jtitle%3DInflammation%2520Regener.%26date%3D2018%26volume%3D38%26spage%3D28%26doi%3D10.1186%2Fs41232-018-0086-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maccarrone, M.</span></span> <span> </span><span class="NLM_article-title">Metabolism of the endocannabinoid anandamide: open questions after 25 years</span>. <i>Front. Mol. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">166</span>, <span class="refDoi"> DOI: 10.3389/fnmol.2017.00166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.3389%2Ffnmol.2017.00166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=28611591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A280%3ADC%252BC1cnotVWitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=166&author=M.+Maccarrone&title=Metabolism+of+the+endocannabinoid+anandamide%3A+open+questions+after+25+years&doi=10.3389%2Ffnmol.2017.00166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of the Endocannabinoid Anandamide: Open Questions after 25 Years</span></div><div class="casAuthors">Maccarrone Mauro; Maccarrone Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in molecular neuroscience</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">166</span>
        ISSN:<span class="NLM_cas:issn">1662-5099</span>.
    </div><div class="casAbstract">Cannabis extracts have been used for centuries, but its main active principle α(9)-tetrahydrocannabinol (THC) was identified about 50 years ago.  Yet, it is only 25 years ago that the first endogenous ligand of the same receptors engaged by the cannabis agents was discovered.  This "endocannabinoid (eCB)" was identified as N-arachidonoylethanolamine (or anandamide (AEA)), and was shown to have several receptors, metabolic enzymes and transporters that altogether drive its biological activity.  Here I report on the latest advances about AEA metabolism, with the aim of focusing open questions still awaiting an answer for a deeper understanding of AEA activity, and for translating AEA-based drugs into novel therapeutics for human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT31pdfVsY5rhWfpMXQLRstfW6udTcc2eYMLFPFR6KkT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnotVWitA%253D%253D&md5=5534e6a37182be0810bd932e9c5eb5a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3389%2Ffnmol.2017.00166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnmol.2017.00166%26sid%3Dliteratum%253Aachs%26aulast%3DMaccarrone%26aufirst%3DM.%26atitle%3DMetabolism%2520of%2520the%2520endocannabinoid%2520anandamide%253A%2520open%2520questions%2520after%252025%2520years%26jtitle%3DFront.%2520Mol.%2520Neurosci.%26date%3D2017%26volume%3D10%26spage%3D166%26doi%3D10.3389%2Ffnmol.2017.00166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mazier, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saucisse, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatta-Cherifi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cota, D.</span></span> <span> </span><span class="NLM_article-title">The endocannabinoid system: pivotal orchestrator of obesity and metabolic disease</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2015.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.tem.2015.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=26412154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKnsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=524-537&author=W.+Mazierauthor=N.+Saucisseauthor=B.+Gatta-Cherifiauthor=D.+Cota&title=The+endocannabinoid+system%3A+pivotal+orchestrator+of+obesity+and+metabolic+disease&doi=10.1016%2Fj.tem.2015.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The Endocannabinoid System: Pivotal Orchestrator of Obesity and Metabolic Disease</span></div><div class="casAuthors">Mazier, Wilfrid; Saucisse, Nicolas; Gatta-Cherifi, Blandine; Cota, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">524-537</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The endocannabinoid system (ECS) functions to adjust behavior and metab. according to environmental changes in food availability.  Its actions range from the regulation of sensory responses to the development of preference for the consumption of calorically-rich food and control of its metabolic handling.  ECS activity is beneficial when access to food is scarce or unpredictable.  However, when food is plentiful, the ECS favors obesity and metabolic disease.  We review recent advances in understanding the roles of the ECS in energy balance, and discuss newly identified mechanisms of action that, after the withdrawal of first generation cannabinoid type 1 (CB1) receptor antagonists for the treatment of obesity, have made the ECS once again an attractive target for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_3BI1bJVo57Vg90H21EOLACvtfcHk0lgvsDyApAhlvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKnsL%252FO&md5=3721054e445191f88ac812b079cd83ac</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2015.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2015.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DMazier%26aufirst%3DW.%26aulast%3DSaucisse%26aufirst%3DN.%26aulast%3DGatta-Cherifi%26aufirst%3DB.%26aulast%3DCota%26aufirst%3DD.%26atitle%3DThe%2520endocannabinoid%2520system%253A%2520pivotal%2520orchestrator%2520of%2520obesity%2520and%2520metabolic%2520disease%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2015%26volume%3D26%26spage%3D524%26epage%3D537%26doi%3D10.1016%2Fj.tem.2015.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimberly, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nath, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keyes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larson, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corey, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerszten, R. E.</span></span> <span> </span><span class="NLM_article-title">Metabolite profiling identifies anandamide as a biomarker of nonalcoholic steatohepatitis</span>. <i>JCI Insight</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e92989</span> <span class="refDoi"> DOI: 10.1172/jci.insight.92989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1172%2Fjci.insight.92989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=28469090" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&author=W.+T.+Kimberlyauthor=J.+F.+O%E2%80%99Sullivanauthor=A.+K.+Nathauthor=M.+Keyesauthor=X.+Shiauthor=M.+G.+Larsonauthor=Q.+Yangauthor=M.+T.+Longauthor=R.+Vasanauthor=R.+T.+Petersonauthor=T.+J.+Wangauthor=K.+E.+Coreyauthor=R.+E.+Gerszten&title=Metabolite+profiling+identifies+anandamide+as+a+biomarker+of+nonalcoholic+steatohepatitis&doi=10.1172%2Fjci.insight.92989"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1172%2Fjci.insight.92989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci.insight.92989%26sid%3Dliteratum%253Aachs%26aulast%3DKimberly%26aufirst%3DW.%2BT.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DJ.%2BF.%26aulast%3DNath%26aufirst%3DA.%2BK.%26aulast%3DKeyes%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DLarson%26aufirst%3DM.%2BG.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DLong%26aufirst%3DM.%2BT.%26aulast%3DVasan%26aufirst%3DR.%26aulast%3DPeterson%26aufirst%3DR.%2BT.%26aulast%3DWang%26aufirst%3DT.%2BJ.%26aulast%3DCorey%26aufirst%3DK.%2BE.%26aulast%3DGerszten%26aufirst%3DR.%2BE.%26atitle%3DMetabolite%2520profiling%2520identifies%2520anandamide%2520as%2520a%2520biomarker%2520of%2520nonalcoholic%2520steatohepatitis%26jtitle%3DJCI%2520Insight%26date%3D2017%26volume%3D2%26doi%3D10.1172%2Fjci.insight.92989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fanelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mezzullo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repaci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belluomo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibarra Gasparini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Dalmazi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastroroberto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vicennati, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gambineri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morselli-Labate, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasquali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagotto, U.</span></span> <span> </span><span class="NLM_article-title">Profiling plasma <i>N</i>-acylethanolamine levels and their ratios as a biomarker of obesity and dysmetabolism</span>. <i>Mol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1016/j.molmet.2018.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.molmet.2018.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=29935920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=82-94&author=F.+Fanelliauthor=M.+Mezzulloauthor=A.+Repaciauthor=I.+Belluomoauthor=D.+Ibarra+Gaspariniauthor=G.+Di+Dalmaziauthor=M.+Mastrorobertoauthor=V.+Vicennatiauthor=A.+Gambineriauthor=A.+M.+Morselli-Labateauthor=R.+Pasqualiauthor=U.+Pagotto&title=Profiling+plasma+N-acylethanolamine+levels+and+their+ratios+as+a+biomarker+of+obesity+and+dysmetabolism&doi=10.1016%2Fj.molmet.2018.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling plasma N-Acylethanolamine levels and their ratios as a biomarker of obesity and dysmetabolism</span></div><div class="casAuthors">Fanelli, Flaminia; Mezzullo, Marco; Repaci, Andrea; Belluomo, Ilaria; Ibarra Gasparini, Daniela; Di Dalmazi, Guido; Mastroroberto, Marianna; Vicennati, Valentina; Gambineri, Alessandra; Morselli-Labate, Antonio Maria; Pasquali, Renato; Pagotto, Uberto</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Metabolism</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">82-94</span>CODEN:
                <span class="NLM_cas:coden">MMOEAS</span>;
        ISSN:<span class="NLM_cas:issn">2212-8778</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">N-acylethanolamines play different roles in energy balance; anandamide (AEA) stimulates energy intake and storage, N-palmitoylethanolamide (PEA) counters inflammation, and N-oleoylethanolamide (OEA) mediates anorectic signals and lipid oxidn.  Inconsistencies in the assocn. of plasma N-acylethanolamines with human obesity and cardiometabolic risk have emerged among previous studies, possibly caused by heterogeneous cohorts and designs, and by unstandardized N-acylethanolamine measurements.  We aimed to characterize changes in the plasma profile, including N-acylethanolamine levels and ratios assocd. with obesity, menopause in women, and aging in men, and to define the significance of such a profile as a biomarker for metabolic imbalance.  Adult, drug-free women (n=103 premenopausal and n=81 menopausal) and men (n=144) were stratified according to the body mass index (BMI) into normal wt. (NW; BMI: 18.5-24.9 kg/m2), overweight (OW; BMI: 25.0-29.9 kg/m2), and obese (OB; BMI:>30.0 kg/m2).  Anthropometric and metabolic parameters were detd.  Validated blood processing and anal. procedures for N-acylethanolamine measurements were used.  We investigated the effect of BMI and menopause in women, and BMI and age in men, as well as the BMI-independent influence of metabolic parameters on the N-acylethanolamine profile.  BMI and waist circumference directly assocd. with AEA in women and men, and with PEA in permenopausal women and in men, while BMI directly assocd. with OEA in premenopausal women and in men.  BMI, in both genders, and waist circumference, in women only, inversely assocd. with PEA/AEA and OEA/AEA.  Menopause increased N-acylethanolamine levels, whereas ageing resulted in increasing OEA relative abundance in men.  AEA and OEA abundances in premenopausal, and PEA and OEA abundances in lean menopausal women, were directly assocd. with hypertension.  Conversely, PEA and OEA abundances lowered with hypertension in elderly men.  Insulin resistance was assocd. with changes in N-acylethanolamine ratios specific for premenopausal (reduced PEA/AEA and OEA/AEA), menopausal (reduced OEA/AEA) women and men (reduced OEA/AEA and OEA/PEA).  PEA and OEA levels increased with total cholesterol, and OEA abundance specifically increased with HDL-cholesterol.  Elevated triglyceride levels were assocd. with increased N-acylethanolamine levels only in menopausal women.  Obesity-related N-acylethanolamine hypertone is characterized by imbalanced N-acylethanoamine ratios.  The profile given by a combination of N-acylethanolamine abs. levels and ratios enables imbalances to be identified in relationship with different metabolic parameters, with specific relevance according to gender, menopause and age, representing a useful means for monitoring metabolic health.  Finally, N-acylethanolamine system appears a promising target for intervention strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoohmBCJAciyrVg90H21EOLACvtfcHk0liSt9Mq1cz-Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsb7F&md5=6170dd31651a203204ab6125c0b77f35</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.molmet.2018.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmet.2018.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DFanelli%26aufirst%3DF.%26aulast%3DMezzullo%26aufirst%3DM.%26aulast%3DRepaci%26aufirst%3DA.%26aulast%3DBelluomo%26aufirst%3DI.%26aulast%3DIbarra%2BGasparini%26aufirst%3DD.%26aulast%3DDi%2BDalmazi%26aufirst%3DG.%26aulast%3DMastroroberto%26aufirst%3DM.%26aulast%3DVicennati%26aufirst%3DV.%26aulast%3DGambineri%26aufirst%3DA.%26aulast%3DMorselli-Labate%26aufirst%3DA.%2BM.%26aulast%3DPasquali%26aufirst%3DR.%26aulast%3DPagotto%26aufirst%3DU.%26atitle%3DProfiling%2520plasma%2520N-acylethanolamine%2520levels%2520and%2520their%2520ratios%2520as%2520a%2520biomarker%2520of%2520obesity%2520and%2520dysmetabolism%26jtitle%3DMol.%2520Metab.%26date%3D2018%26volume%3D14%26spage%3D82%26epage%3D94%26doi%3D10.1016%2Fj.molmet.2018.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morishita, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyatake, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Functional analysis of the purified anandamide-generating phospholipase D as a member of the metallo-β-lactamase family</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">12325</span>– <span class="NLM_lpage">12335</span>, <span class="refDoi"> DOI: 10.1074/jbc.M512359200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1074%2Fjbc.M512359200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=16527816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFKksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=12325-12335&author=J.+Wangauthor=Y.+Okamotoauthor=J.+Morishitaauthor=K.+Tsuboiauthor=A.+Miyatakeauthor=N.+Ueda&title=Functional+analysis+of+the+purified+anandamide-generating+phospholipase+D+as+a+member+of+the+metallo-%CE%B2-lactamase+family&doi=10.1074%2Fjbc.M512359200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Analysis of the Purified Anandamide-generating Phospholipase D as a Member of the Metallo-β-lactamase Family</span></div><div class="casAuthors">Wang, Jun; Okamoto, Yasuo; Morishita, Jun; Tsuboi, Kazuhito; Miyatake, Akira; Ueda, Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">12325-12335</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In animal tissues, bioactive N-acylethanolamines including the endocannabinoid anandamide are formed from their corresponding N-acylphosphatidylethanolamines (NAPEs) by the catalysis of a specific phospholipase D (NAPE-PLD) that belongs to the metallo-β-lactamase family.  Despite its potential physiol. importance, NAPE-PLD has not yet been characterized with a purified enzyme prepn.  In the present study we expressed a recombinant NAPE-PLD in Escherichia coli and highly purified it.  The purified enzyme was remarkably activated in a dose-dependent manner by millimolar concns. of Mg2+ as well as Ca2+ and, hence, appeared to be constitutively active.  The enzyme showed extremely high specificity for NAPEs among various glycerophospholipids but did not reveal obvious selectivity for different long chain or medium chain N-acyl species of NAPEs.  These results suggested the ability of NAPE-PLD to degrade different NAPEs without damaging other membrane phospholipids.  Metal anal. revealed the presence of catalytically important zinc in NAPE-PLD.  In addn., site-directed mutagenesis studies were addressed to several histidine and aspartic acid residues of NAPE-PLD that are highly conserved within the metallo-β-lactamase family.  Single mutations of Asp-147, His-185, His-187, Asp-189, His-190, His-253, Asp-284, and His-321 caused abolishment or remarkable redn. of the catalytic activity.  Moreover, when six cysteine residues were individually mutated to serine, only C224S showed a considerably reduced activity.  The activities of L207F and H380R found as single nucleotide polymorphisms were also low.  Thus, NAPE-PLD appeared to function through a mechanism similar to those of the well characterized members of this family but play a unique role in the lipid metab. of animal tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHPr_ieu3LxrVg90H21EOLACvtfcHk0lgzNk9c7I5FFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFKksrY%253D&md5=aedf6e484205f3af45d2743f551de502</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M512359200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M512359200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DOkamoto%26aufirst%3DY.%26aulast%3DMorishita%26aufirst%3DJ.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DMiyatake%26aufirst%3DA.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DFunctional%2520analysis%2520of%2520the%2520purified%2520anandamide-generating%2520phospholipase%2520D%2520as%2520a%2520member%2520of%2520the%2520metallo-%25CE%25B2-lactamase%2520family%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D12325%26epage%3D12335%26doi%3D10.1074%2Fjbc.M512359200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">The stimulatory effect of phosphatidylethanolamine on <i>N</i>-acylphosphatidylethanolamine-hydrolyzing phospholipase D (NAPE-PLD)</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2007.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.neuropharm.2007.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=17655883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A280%3ADC%252BD1c%252FgtVWnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2008&pages=8-15&author=J.+Wangauthor=Y.+Okamotoauthor=K.+Tsuboiauthor=N.+Ueda&title=The+stimulatory+effect+of+phosphatidylethanolamine+on+N-acylphosphatidylethanolamine-hydrolyzing+phospholipase+D+%28NAPE-PLD%29&doi=10.1016%2Fj.neuropharm.2007.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The stimulatory effect of phosphatidylethanolamine on N-acylphosphatidylethanolamine-hydrolyzing phospholipase D (NAPE-PLD)</span></div><div class="casAuthors">Wang Jun; Okamoto Yasuo; Tsuboi Kazuhito; Ueda Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-15</span>
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    </div><div class="casAbstract">N-Acylphosphatidylethanolamine (NAPE)-hydrolyzing phospholipase D (NAPE-PLD) is a membrane-bound enzyme which releases the endocannabinoid anandamide and other bioactive N-acylethanolamines from their corresponding NAPEs in animal tissues.  Our previous studies showed that NAPE-PLD solubilized from the membrane is remarkably stimulated by millimolar concentrations of Ca(2+) while the membrane-bound form is much less sensitive to Ca(2+).  This finding suggested that certain membrane constituents diminished the stimulatory effect of Ca(2+).  In the present studies, we examined the effects of membrane fractions from COS-7 cells and brain tissue on the purified recombinant rat NAPE-PLD, and found that heat-stable membrane component(s) dose-dependently activated NAPE-PLD up to 4.8-5.0 fold.  In the presence of the membrane fractions, however, the stimulatory effect of Ca(2+) on the purified NAPE-PLD was considerably reduced.  When it was examined if the membrane fractions can be replaced with various pure phospholipids, phosphatidylethanolamine activated NAPE-PLD up to 3.3 fold, which was followed by decrease in the stimulatory effects of Ca(2+) and several other divalent cations.  These results suggest that membrane components including phosphatidylethanolamine keep the membrane-associated form of NAPE-PLD constitutively active.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTD0xdaMAOZYIJKJbLTgsvCfW6udTcc2ebMxEgkETCMgrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c%252FgtVWnsw%253D%253D&md5=68fdf5888fb468a4de4d50855e336772</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2007.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2007.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DOkamoto%26aufirst%3DY.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DThe%2520stimulatory%2520effect%2520of%2520phosphatidylethanolamine%2520on%2520N-acylphosphatidylethanolamine-hydrolyzing%2520phospholipase%2520D%2520%2528NAPE-PLD%2529%26jtitle%3DNeuropharmacology%26date%3D2008%26volume%3D54%26spage%3D8%26epage%3D15%26doi%3D10.1016%2Fj.neuropharm.2007.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magotti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igarashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babagoli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marotta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piomelli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garau, G.</span></span> <span> </span><span class="NLM_article-title">Structure of human <i>N</i>-acylphosphatidylethanolamine-hydrolyzing phospholipase D: regulation of fatty acid ethanolamide biosynthesis by bile acids</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1016/j.str.2014.12.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.str.2014.12.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=25684574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVSjt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=598-604&author=P.+Magottiauthor=I.+Bauerauthor=M.+Igarashiauthor=M.+Babagoliauthor=R.+Marottaauthor=D.+Piomelliauthor=G.+Garau&title=Structure+of+human+N-acylphosphatidylethanolamine-hydrolyzing+phospholipase+D%3A+regulation+of+fatty+acid+ethanolamide+biosynthesis+by+bile+acids&doi=10.1016%2Fj.str.2014.12.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of human N-acylphosphatidylethanolamine-hydrolyzing phospholipase D: Regulation of fatty acid ethanolamide biosynthesis by bile acids</span></div><div class="casAuthors">Magotti, Paola; Bauer, Inga; Igarashi, Miki; Babagoli, Masih; Marotta, Roberto; Piomelli, Daniele; Garau, Gianpiero</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-604</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The fatty acid ethanolamides (FAEs) are lipid mediators present in all organisms and involved in highly conserved biol. functions, such as innate immunity, energy balance, and stress control.  They are produced from membrane N-acylphosphatidylethanolamines (NAPEs) and include agonists for G protein-coupled receptors (e.g., cannabinoid receptors) and nuclear receptors (e.g., PPAR-α).  Here, the authors report the crystal structure of human NAPE-hydrolyzing phospholipase D (NAPE-PLD) at 2.65 Å resoln., a membrane enzyme that catalyzes FAE formation in mammals.  NAPE-PLD forms homodimers partly sepd. by an internal ∼9-Å-wide channel and uniquely adapted to assoc. with phospholipids.  A hydrophobic cavity provides an entryway for NAPE into the active site, where a binuclear Zn2+ center orchestrates its hydrolysis.  Bile acids bind with high affinity to selective pockets in this cavity, enhancing dimer assembly and enabling catalysis.  These elements offer multiple targets for the design of small-mol. NAPE-PLD modulators with potential applications in inflammation and metabolic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAkur49s4h2bVg90H21EOLACvtfcHk0lgzNk9c7I5FFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVSjt7s%253D&md5=c80aae05403e5fc59a3bb6db1ec673ec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.12.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.12.018%26sid%3Dliteratum%253Aachs%26aulast%3DMagotti%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DIgarashi%26aufirst%3DM.%26aulast%3DBabagoli%26aufirst%3DM.%26aulast%3DMarotta%26aufirst%3DR.%26aulast%3DPiomelli%26aufirst%3DD.%26aulast%3DGarau%26aufirst%3DG.%26atitle%3DStructure%2520of%2520human%2520N-acylphosphatidylethanolamine-hydrolyzing%2520phospholipase%2520D%253A%2520regulation%2520of%2520fatty%2520acid%2520ethanolamide%2520biosynthesis%2520by%2520bile%2520acids%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D598%26epage%3D604%26doi%3D10.1016%2Fj.str.2014.12.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Margheritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magotti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peruzzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romeo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natali, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostarda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gioiello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piomelli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garau, G.</span></span> <span> </span><span class="NLM_article-title">Bile acid recognition by NAPE-PLD</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2908</span>– <span class="NLM_lpage">2914</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00624</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00624" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWltr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2908-2914&author=E.+Margheritisauthor=B.+Castellaniauthor=P.+Magottiauthor=S.+Peruzziauthor=E.+Romeoauthor=F.+Nataliauthor=S.+Mostardaauthor=A.+Gioielloauthor=D.+Piomelliauthor=G.+Garau&title=Bile+acid+recognition+by+NAPE-PLD&doi=10.1021%2Facschembio.6b00624"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Bile Acid Recognition by NAPE-PLD</span></div><div class="casAuthors">Margheritis, Eleonora; Castellani, Beatrice; Magotti, Paola; Peruzzi, Sara; Romeo, Elisa; Natali, Francesca; Mostarda, Serena; Gioiello, Antimo; Piomelli, Daniele; Garau, Gianpiero</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2908-2914</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The membrane-assocd. enzyme, N-acylphosphatidylethanolamine specific-phospholipase D (NAPE-PLD), generates the endogenous cannabinoid, arachidonylethanolamide, and other lipid signaling amides, including oleoylethanolamide and palmitoylethanolamide.  These bioactive mols. play important roles in several physiol. pathways including stress and pain response, appetite, and lifespan.  Recently, the authors reported the crystal structure of human NAPE-PLD and discovered specific binding sites for the bile acid, deoxycholic acid.  Here, the authors demonstrate that in the presence of this secondary bile acid, the stiffness of the protein measured by elastic neutron scattering increased, and NAPE-PLD resulted in ∼7-fold faster catalysis of the hydrolysis of the more unsatd. substrate, N-arachidonoylphosphatidylethanolamine, compared with N-palmitoylphosphatidylethanolamine.  Chenodeoxycholic acid and glyco- or tauro-dihydroxy conjugates could also bind to NAPE-PLD and drive its activation.  The only natural monohydroxy bile acid, lithocholic acid, showed an affinity of ∼20 μM and acted instead as a reversible inhibitor (IC50 = 68 μM).  Overall, these findings provided important insights into the allosteric regulation of the enzyme mediated by bile acid cofactors, and revealed that NAPE-PLD responds primarily to the no. and position of their OH groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXv-uNnhoOMrVg90H21EOLACvtfcHk0lgKcYZUMtHsEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWltr%252FL&md5=18527f3a1cc40d1305378c552b9685a7</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00624%26sid%3Dliteratum%253Aachs%26aulast%3DMargheritis%26aufirst%3DE.%26aulast%3DCastellani%26aufirst%3DB.%26aulast%3DMagotti%26aufirst%3DP.%26aulast%3DPeruzzi%26aufirst%3DS.%26aulast%3DRomeo%26aufirst%3DE.%26aulast%3DNatali%26aufirst%3DF.%26aulast%3DMostarda%26aufirst%3DS.%26aulast%3DGioiello%26aufirst%3DA.%26aulast%3DPiomelli%26aufirst%3DD.%26aulast%3DGarau%26aufirst%3DG.%26atitle%3DBile%2520acid%2520recognition%2520by%2520NAPE-PLD%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2908%26epage%3D2914%26doi%3D10.1021%2Facschembio.6b00624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, N.</span></span> <span> </span><span class="NLM_article-title">Activation of <i>N</i>-acylethanolamine-releasing phospholipase D by polyamines</span>. <i>Chem. Phys. Lipids</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1016/s0009-3084(02)00015-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2FS0009-3084%2802%2900015-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=12047899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD38XktFOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2002&pages=77-84&author=Q.+Liuauthor=T.+Tonaiauthor=N.+Ueda&title=Activation+of+N-acylethanolamine-releasing+phospholipase+D+by+polyamines&doi=10.1016%2Fs0009-3084%2802%2900015-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of N-acylethanolamine-releasing phospholipase D by polyamines</span></div><div class="casAuthors">Liu, Qian; Tonai, Takeharu; Ueda, Natsuo</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry and Physics of Lipids</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">77-84</span>CODEN:
                <span class="NLM_cas:coden">CPLIA4</span>;
        ISSN:<span class="NLM_cas:issn">0009-3084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">N-acylethanolamines including anandamide (an endogenous ligand of cannabinoid receptors) are biosynthesized from N-acyl-phosphatidylethanolamine (PE) by a phosphodiesterase of the phospholipase D type.  The enzyme partially purified from the particulate fraction of rat heart hydrolyzed N-palmitoyl-PE to N-palmitoylethanolamine with a specific activity of 50 nmol/min per mg protein at 37° in the presence of 10 mM CaCl2.  We found that the enzyme was highly activated in dose-dependent manner by polyamines like spermine, spermidine, and putrescine.  Spermine was the most potent with an EC50 value around 0.1 mM, and increased the specific enzyme activity 27 fold up to 53 nmol/min per mg protein.  However, a synergistic effect of spermine and the known activator (Ca2+ or Triton X-100) was not obsd.  The spermine-stimulated enzyme was also active with N-arachidonoyl-PE (a precursor of anandamide).  Thus, polyamines may function as endogenous activators to control the biosynthesis of anandamide and other N-acylethanolamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHZlLwSGWMBLVg90H21EOLACvtfcHk0lgKcYZUMtHsEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktFOhu70%253D&md5=4f5a830031ca427391b36a0806fee026</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0009-3084%2802%2900015-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0009-3084%252802%252900015-4%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DTonai%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DN.%26atitle%3DActivation%2520of%2520N-acylethanolamine-releasing%2520phospholipase%2520D%2520by%2520polyamines%26jtitle%3DChem.%2520Phys.%2520Lipids%26date%3D2002%26volume%3D115%26spage%3D77%26epage%3D84%26doi%3D10.1016%2Fs0009-3084%2802%2900015-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickering, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begtrup, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. S.</span></span> <span> </span><span class="NLM_article-title">Effect of synthetic and natural phospholipids on <i>N</i>-acylphosphatidylethanolamine-hydrolyzing phospholipase D activity</span>. <i>Chem. Phys. Lipids</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.chemphyslip.2009.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.chemphyslip.2009.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=19715685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Gnt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2009&pages=53-61&author=G.+Petersenauthor=A.+H.+Pedersenauthor=D.+S.+Pickeringauthor=M.+Begtrupauthor=H.+S.+Hansen&title=Effect+of+synthetic+and+natural+phospholipids+on+N-acylphosphatidylethanolamine-hydrolyzing+phospholipase+D+activity&doi=10.1016%2Fj.chemphyslip.2009.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of synthetic and natural phospholipids on N-acylphosphatidylethanolamine-hydrolyzing phospholipase D activity</span></div><div class="casAuthors">Petersen, Gitte; Pedersen, Anders H.; Pickering, Darryl S.; Begtrup, Mikael; Hansen, Harald S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry and Physics of Lipids</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">53-61</span>CODEN:
                <span class="NLM_cas:coden">CPLIA4</span>;
        ISSN:<span class="NLM_cas:issn">0009-3084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">N-Acylethanolamines (NAEs) constitute a family of endogenous bioactive lipids that includes arachidonoylethanolamide (anandamide), palmitoylethanolamide (PEA) and oleoylethanolamide (OEA).  These lipids are formed from their resp. N-acylated ethanolamine phospholipid (NAPE) precursor by the action of a phospholipase D enzyme (NAPE-PLD).  Anandamide, OEA, and PEA are all bioactive lipids that may influence, amongst others: neuroinflammation, food intake, and oocyte implantation.  Here we have synthesized a no. of NAPE analogs with variation in the phosphoester structure.  The NAPE analogs as well as selected phospholipids and β-lactamase substrates were tested as potential modifiers of cloned human NAPE-PLD in an enzyme assay involving a 14C-labeled diether-NAPE substrate.  One hit was identified, namely 1,2-dihexanoyl-glycero-N-(3-(tetradecanoylamino)propyl)phosphoramidate (AHP-71B) which showed inhibitory activity and may serve as template for further structure-activity developments.  Furthermore, NAPE-PLD was activated by phosphatidylethanolamine and inhibited by the β-lactamase substrate nitrocefin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh6CP241fNVbVg90H21EOLACvtfcHk0lgAWHGoeMEmfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Gnt7zP&md5=6280d2b0dac4471e3d8b0b6b4c294e29</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chemphyslip.2009.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chemphyslip.2009.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DG.%26aulast%3DPedersen%26aufirst%3DA.%2BH.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DBegtrup%26aufirst%3DM.%26aulast%3DHansen%26aufirst%3DH.%2BS.%26atitle%3DEffect%2520of%2520synthetic%2520and%2520natural%2520phospholipids%2520on%2520N-acylphosphatidylethanolamine-hydrolyzing%2520phospholipase%2520D%2520activity%26jtitle%3DChem.%2520Phys.%2520Lipids%26date%3D2009%26volume%3D162%26spage%3D53%26epage%3D61%26doi%3D10.1016%2Fj.chemphyslip.2009.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Reilly, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavieri, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, D.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larock, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of desketoraloxifene analogues as inhibitors of mammalian, Pseudomonas aeruginosa, and NAPE phospholipase D enzymes</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">432</span>, <span class="refDoi"> DOI: 10.1021/cb500828m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500828m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVyru77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=421-432&author=S.+A.+Scottauthor=C.+T.+Spencerauthor=M.+C.+O%E2%80%99Reillyauthor=K.+A.+Brownauthor=R.+R.+Lavieriauthor=C.-H.+Choauthor=D.-I.+Jungauthor=R.+C.+Larockauthor=H.+A.+Brownauthor=C.+W.+Lindsley&title=Discovery+of+desketoraloxifene+analogues+as+inhibitors+of+mammalian%2C+Pseudomonas+aeruginosa%2C+and+NAPE+phospholipase+D+enzymes&doi=10.1021%2Fcb500828m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Desketoraloxifene Analogues as Inhibitors of Mammalian, Pseudomonas aeruginosa, and NAPE Phospholipase D Enzymes</span></div><div class="casAuthors">Scott, Sarah A.; Spencer, Cierra T.; O'Reilly, Matthew C.; Brown, Kyle A.; Lavieri, Robert R.; Cho, Chul-Hee; Jung, Dai-Il; Larock, Richard C.; Brown, H. Alex; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">421-432</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phospholipase D (PLD) hydrolyzes cellular lipids to produce the important lipid second messenger phosphatidic acid.  A PLD enzyme expressed by Pseudomonas aeruginosa (PldA) has been shown to be important in bacterial infection, and NAPE-PLD has emerged as being key in the synthesis of endocannabinoids.  In order to better understand the biol. and therapeutic potential of these less explored PLD enzymes, small mol. tools are required.  Selective estrogen receptor modulators (SERMs) have been previously shown to inhibit mammalian PLD (PLD1 and PLD2).  By targeted screening of a library of SERM analogs, addnl. parallel synthesis, and evaluation in multiple PLD assays, the authors discovered a novel desketoraloxifene-based scaffold that inhibited not only the two mammalian PLDs but also structurally divergent PldA and NAPE-PLD.  This finding represents an important first step toward the development of small mols. possessing universal inhibition of divergent PLD enzymes to advance the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3qbMYFrILrLVg90H21EOLACvtfcHk0lgAWHGoeMEmfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVyru77M&md5=864be7f1f24d7dfc4a46801162bdab55</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcb500828m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500828m%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DS.%2BA.%26aulast%3DSpencer%26aufirst%3DC.%2BT.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DM.%2BC.%26aulast%3DBrown%26aufirst%3DK.%2BA.%26aulast%3DLavieri%26aufirst%3DR.%2BR.%26aulast%3DCho%26aufirst%3DC.-H.%26aulast%3DJung%26aufirst%3DD.-I.%26aulast%3DLarock%26aufirst%3DR.%2BC.%26aulast%3DBrown%26aufirst%3DH.%2BA.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DDiscovery%2520of%2520desketoraloxifene%2520analogues%2520as%2520inhibitors%2520of%2520mammalian%252C%2520Pseudomonas%2520aeruginosa%252C%2520and%2520NAPE%2520phospholipase%2520D%2520enzymes%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D421%26epage%3D432%26doi%3D10.1021%2Fcb500828m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Castellani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamanti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pizzirani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maccesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Realini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magotti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakkum, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piomelli, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and characterization of the first inhibitor of <i>N</i>-acylphosphatidylethanolamine phospholipase D (NAPE-PLD)</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">12814</span>– <span class="NLM_lpage">12817</span>, <span class="refDoi"> DOI: 10.1039/C7CC07582K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1039%2FC7CC07582K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=29143042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWksLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=12814-12817&author=B.+Castellaniauthor=E.+Diamantiauthor=D.+Pizziraniauthor=P.+Tardiaauthor=M.+Maccesiauthor=N.+Realiniauthor=P.+Magottiauthor=G.+Garauauthor=T.+Bakkumauthor=S.+Rivaraauthor=M.+Morauthor=D.+Piomelli&title=Synthesis+and+characterization+of+the+first+inhibitor+of+N-acylphosphatidylethanolamine+phospholipase+D+%28NAPE-PLD%29&doi=10.1039%2FC7CC07582K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and characterization of the first inhibitor of N-acylphosphatidylethanolamine phospholipase D (NAPE-PLD)</span></div><div class="casAuthors">Castellani, Beatrice; Diamanti, Eleonora; Pizzirani, Daniela; Tardia, Piero; Maccesi, Martina; Realini, Natalia; Magotti, Paola; Garau, Gianpiero; Bakkum, Thomas; Rivara, Silvia; Mor, Marco; Piomelli, Daniele</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">95</span>),
    <span class="NLM_cas:pages">12814-12817</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">N-Acylphosphatidylethanolamine phospholipase D (NAPE-PLD) is a membrane-assocd. zinc enzyme that catalyzes the hydrolysis of N-acylphosphatidylethanolamines (NAPEs) into fatty acid ethanolamides (FAEs).  Here, we describe the identification of the first small-mol. NAPE-PLD inhibitor, the quinazoline sulfonamide deriv. 2,4-dioxo-N-[4-(4-pyridyl)phenyl]-1H-quinazoline-6-sulfonamide, ARN19874.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNZXUDG2Sz3bVg90H21EOLACvtfcHk0lgAWHGoeMEmfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWksLbM&md5=ac246067aa18f87e5a7ccc00bc2395e1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1039%2FC7CC07582K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC07582K%26sid%3Dliteratum%253Aachs%26aulast%3DCastellani%26aufirst%3DB.%26aulast%3DDiamanti%26aufirst%3DE.%26aulast%3DPizzirani%26aufirst%3DD.%26aulast%3DTardia%26aufirst%3DP.%26aulast%3DMaccesi%26aufirst%3DM.%26aulast%3DRealini%26aufirst%3DN.%26aulast%3DMagotti%26aufirst%3DP.%26aulast%3DGarau%26aufirst%3DG.%26aulast%3DBakkum%26aufirst%3DT.%26aulast%3DRivara%26aufirst%3DS.%26aulast%3DMor%26aufirst%3DM.%26aulast%3DPiomelli%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520characterization%2520of%2520the%2520first%2520inhibitor%2520of%2520N-acylphosphatidylethanolamine%2520phospholipase%2520D%2520%2528NAPE-PLD%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D12814%26epage%3D12817%26doi%3D10.1039%2FC7CC07582K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrow, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashhadi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S. S.</span></span> <span> </span><span class="NLM_article-title">Symmetrically substituted dichlorophenes inhibit <i>N</i>-acyl-phosphatidylethanolamine phospholipase D</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">7289</span>– <span class="NLM_lpage">7300</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA120.013362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1074%2Fjbc.RA120.013362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=32284327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Wqt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2020&pages=7289-7300&author=G.+Aggarwalauthor=J.+E.+Zarrowauthor=Z.+Mashhadiauthor=C.+R.+Flynnauthor=P.+Vinsonauthor=C.+D.+Weaverauthor=S.+S.+Davies&title=Symmetrically+substituted+dichlorophenes+inhibit+N-acyl-phosphatidylethanolamine+phospholipase+D&doi=10.1074%2Fjbc.RA120.013362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Symmetrically substituted dichlorophenes inhibit N-acyl-phosphatidylethanolamine phospholipase D</span></div><div class="casAuthors">Aggarwal, Geetika; Zarrow, Jonah E.; Mashhadi, Zahra; Flynn, C. Robb; Vinson, Paige; Weaver, C. David; Davies, Sean S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7289-7301</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">1083-351X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">N-Acyl-phosphatidylethanolamine phospholipase D (NAPE-PLD) (EC 3.1.4.4) catalyzes the final step in the biosynthesis of N-acyl-ethanolamides.  Reduced NAPE-PLD expression and activity may contribute to obesity and inflammation, but a lack of effective NAPE-PLD inhibitors has been a major obstacle to elucidating the role of NAPE-PLD and N-acyl-ethanolamide biosynthesis in these processes.  The endogenous bile acid lithocholic acid (LCA) inhibits NAPE-PLD activity (with an IC50 of 68 μm), but LCA is also a highly potent ligand for TGR5 (EC50 0.52 μm).  Recently, the first selective small-mol. inhibitor of NAPE-PLD, ARN19874, has been reported (having an IC50 of 34 μm).  To identify more potent inhibitors of NAPE-PLD, here we used a quenched fluorescent NAPE analog, PED-A1, as a substrate for recombinant mouse Nape-pld to screen a panel of bile acids and a library of exptl. compds. (the Spectrum Collection).  Muricholic acids and several other bile acids inhibited Nape-pld with potency similar to that of LCA.  We identified 14 potent Nape-pld inhibitors in the Spectrum Collection, with the two most potent (IC50 = ~ 2 μm) being sym. substituted dichlorophenes, i.e. hexachlorophene and bithionol.  Structure-activity relationship assays using addnl. substituted dichlorophenes identified key moieties needed for Nape-pld inhibition.  Both hexachlorophene and bithionol exhibited significant selectivity for Nape-pld compared with nontarget lipase activities such as Streptomyces chromofuscus PLD or serum lipase.  Both also effectively inhibited NAPE-PLD activity in cultured HEK293 cells.  We conclude that sym. substituted dichlorophenes potently inhibit NAPE-PLD in cultured cells and have significant selectivity for NAPE-PLD vs. other tissue-assocd. lipases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE49oiAvett7Vg90H21EOLACvtfcHk0lgl6bCN1x75Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Wqt7fM&md5=66f0e9624585923c106958bd3fe3e221</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA120.013362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA120.013362%26sid%3Dliteratum%253Aachs%26aulast%3DAggarwal%26aufirst%3DG.%26aulast%3DZarrow%26aufirst%3DJ.%2BE.%26aulast%3DMashhadi%26aufirst%3DZ.%26aulast%3DFlynn%26aufirst%3DC.%2BR.%26aulast%3DVinson%26aufirst%3DP.%26aulast%3DWeaver%26aufirst%3DC.%2BD.%26aulast%3DDavies%26aufirst%3DS.%2BS.%26atitle%3DSymmetrically%2520substituted%2520dichlorophenes%2520inhibit%2520N-acyl-phosphatidylethanolamine%2520phospholipase%2520D%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2020%26volume%3D295%26spage%3D7289%26epage%3D7300%26doi%3D10.1074%2Fjbc.RA120.013362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shuman, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leech, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvord, E. C.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Neurotoxicity of hexachlorophene in humans: II. a clinicopathological study of 46 premature infants</span>. <i>Arch. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1001/archneur.1975.00490470064009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1001%2Farchneur.1975.00490470064009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1137506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A280%3ADyaE2M7ot12rtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1975&pages=320-325&author=R.+M.+Shumanauthor=R.+W.+Leechauthor=E.+C.+Alvord&title=Neurotoxicity+of+hexachlorophene+in+humans%3A+II.+a+clinicopathological+study+of+46+premature+infants&doi=10.1001%2Farchneur.1975.00490470064009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Neurotoxicity of hexachlorophene in humans. II. A clinicopathological study of 46 premature infants</span></div><div class="casAuthors">Shuman R M; Leech R W; Alvord E C Jr</div><div class="citationInfo"><span class="NLM_cas:title">Archives of neurology</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">320-5</span>
        ISSN:<span class="NLM_cas:issn">0003-9942</span>.
    </div><div class="casAbstract">To assess neurotoxic effects of hexachlorophene in the human population previously shown to be most at risk, a blind clinicopathological analysis was made of all premature infants under 1,400 gm birth weight who survived at least four days and were examined by autopsy over a 7.5-year period.  Repeated whole-body bathing of premature newborn infants in 3% hexachlorophene-bearing soap (undiluted pHisoHex) shows a significant statistical association with a vacuolar encephalopathy of the brain stem reticular formation.  The prevalence of the vacuolar encephalopathy in premature infants on whom we have adequate brain stem histological information appears to be related to the number of exposures to hexachlorophene, the concentration of hexachlorophene, the thoroughness of rinsing, and other factors (including exposure to ultraviolet light).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQS2wRdp4ZhvrP3FVUi_gf8fW6udTcc2eYdpp9_7F1yq7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE2M7ot12rtg%253D%253D&md5=aa23a37d65d73e459d35cca655b10f8d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1001%2Farchneur.1975.00490470064009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Farchneur.1975.00490470064009%26sid%3Dliteratum%253Aachs%26aulast%3DShuman%26aufirst%3DR.%2BM.%26aulast%3DLeech%26aufirst%3DR.%2BW.%26aulast%3DAlvord%26aufirst%3DE.%2BC.%26atitle%3DNeurotoxicity%2520of%2520hexachlorophene%2520in%2520humans%253A%2520II.%2520a%2520clinicopathological%2520study%2520of%252046%2520premature%2520infants%26jtitle%3DArch.%2520Neurol.%26date%3D1975%26volume%3D32%26spage%3D320%26epage%3D325%26doi%3D10.1001%2Farchneur.1975.00490470064009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudge, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Design and preparation of new palladium precatalysts for C–C and C–N cross-coupling reactions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">920</span>, <span class="refDoi"> DOI: 10.1039/C2SC20903A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1039%2FC2SC20903A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=23667737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=916-920&author=N.+C.+Brunoauthor=M.+T.+Tudgeauthor=S.+L.+Buchwald&title=Design+and+preparation+of+new+palladium+precatalysts+for+C%E2%80%93C+and+C%E2%80%93N+cross-coupling+reactions&doi=10.1039%2FC2SC20903A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Design and preparation of new palladium precatalysts for C-C and C-N cross-coupling reactions</span></div><div class="casAuthors">Bruno, Nicholas C.; Tudge, Matthew T.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">916-920</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A series of easily prepd., phosphine-ligated palladium precatalysts based on the 2-aminobiphenyl scaffold have been prepd.  The role of the precatalyst-assocd. labile halide (or pseudohalide) in the formation and stability of the palladacycle has been examd.  It was found that replacing the chloride in the previous version of the precatalyst with a mesylate leads to a new class of precatalysts with improved soln. stability and that are readily prepd. from a wider range of phosphine ligands.  The differences between the previous version of precatalyst and that reported here are explored.  In addn., the reactivity of the latter is examd. in a range of C-C and C-N bond forming reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfHp6oyg4k_bVg90H21EOLACvtfcHk0lgl6bCN1x75Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKjtr0%253D&md5=5b50ac18cb67251e5439199c6746dba8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC2SC20903A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2SC20903A%26sid%3Dliteratum%253Aachs%26aulast%3DBruno%26aufirst%3DN.%2BC.%26aulast%3DTudge%26aufirst%3DM.%2BT.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DDesign%2520and%2520preparation%2520of%2520new%2520palladium%2520precatalysts%2520for%2520C%25E2%2580%2593C%2520and%2520C%25E2%2580%2593N%2520cross-coupling%2520reactions%26jtitle%3DChem.%2520Sci.%26date%3D2013%26volume%3D4%26spage%3D916%26epage%3D920%26doi%3D10.1039%2FC2SC20903A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quach, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batey, R. A.</span></span> <span> </span><span class="NLM_article-title">Ligand- and base-free copper(II)-catalyzed C–N bond formation: cross-coupling reactions of organoboron compounds with aliphatic amines and anilines</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">4397</span>– <span class="NLM_lpage">4400</span>, <span class="refDoi"> DOI: 10.1021/ol035681s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol035681s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotlartLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2003&pages=4397-4400&author=T.+D.+Quachauthor=R.+A.+Batey&title=Ligand-+and+base-free+copper%28II%29-catalyzed+C%E2%80%93N+bond+formation%3A+cross-coupling+reactions+of+organoboron+compounds+with+aliphatic+amines+and+anilines&doi=10.1021%2Fol035681s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand- and Base-Free Copper(II)-Catalyzed C-N Bond Formation: Cross-Coupling Reactions of Organoboron Compounds with Aliphatic Amines and Anilines</span></div><div class="casAuthors">Quach, Tan D.; Batey, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4397-4400</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A ligandless and base-free Cu-catalyzed protocol for the cross-coupling of arylboronic acids and potassium aryltrifluoroborate salts with primary and secondary aliph. amines and anilines to yield a wide variety of secondary and tertiary amines is described.  The process utilizes catalytic copper(II) acetate monohydrate and 4 Å mol. sieves in dichloromethane at slightly elevated temps. under an atm. of oxygen.  A broad range of functional groups is tolerated on both of the cross-coupling partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraNMwqBwBxFrVg90H21EOLACvtfcHk0lhhfSv6fLRb4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotlartLY%253D&md5=af170572d82b811ba5d1d574bde81372</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fol035681s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol035681s%26sid%3Dliteratum%253Aachs%26aulast%3DQuach%26aufirst%3DT.%2BD.%26aulast%3DBatey%26aufirst%3DR.%2BA.%26atitle%3DLigand-%2520and%2520base-free%2520copper%2528II%2529-catalyzed%2520C%25E2%2580%2593N%2520bond%2520formation%253A%2520cross-coupling%2520reactions%2520of%2520organoboron%2520compounds%2520with%2520aliphatic%2520amines%2520and%2520anilines%26jtitle%3DOrg.%2520Lett.%26date%3D2003%26volume%3D5%26spage%3D4397%26epage%3D4400%26doi%3D10.1021%2Fol035681s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talele, T. T.</span></span> <span> </span><span class="NLM_article-title">The “cyclopropyl fragment” is a versatile player that frequently appears in preclinical/clinical drug molecules</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8712</span>– <span class="NLM_lpage">8756</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00472</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00472" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8712-8756&author=T.+T.+Talele&title=The+%E2%80%9Ccyclopropyl+fragment%E2%80%9D+is+a+versatile+player+that+frequently+appears+in+preclinical%2Fclinical+drug+molecules&doi=10.1021%2Facs.jmedchem.6b00472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The "Cyclopropyl Fragment" is a Versatile Player that Frequently Appears in Preclinical/Clinical Drug Molecules</span></div><div class="casAuthors">Talele, Tanaji T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8712-8756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing use of the cyclopropyl ring in drug development to transition drug candidates from the preclin. to clin. stage.  Important features of the cyclopropane ring are, the (1) coplanarity of the three carbon atoms, (2) relatively shorter (1.51 Å) C-C bonds, (3) enhanced π-character of C-C bonds, and (4) C-H bonds are shorter and stronger than those in alkanes.  The present review will focus on the contributions that a cyclopropyl ring makes to the properties of drugs contg. it.  Consequently, the cyclopropyl ring addresses multiple roadblocks that can occur during drug discovery such as (a) enhancing potency, (b) reducing off-target effects, (c) increasing metabolic stability, (d) increasing brain permeability, (e) decreasing plasma clearance, (f) contributing to an entropically more favorable binding to the receptor, (g) conformational restriction of peptides/peptidomimetics to prevent proteolytic hydrolysis, and (h) altering drug pKa to reduce its P-glycoprotein efflux ratio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVIJAG1XVRG7Vg90H21EOLACvtfcHk0lhhfSv6fLRb4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D&md5=b5586458ec4ab5e2d7586f5739663d1f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00472%26sid%3Dliteratum%253Aachs%26aulast%3DTalele%26aufirst%3DT.%2BT.%26atitle%3DThe%2520%25E2%2580%259Ccyclopropyl%2520fragment%25E2%2580%259D%2520is%2520a%2520versatile%2520player%2520that%2520frequently%2520appears%2520in%2520preclinical%252Fclinical%2520drug%2520molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8712%26epage%3D8756%26doi%3D10.1021%2Facs.jmedchem.6b00472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenau, A.</span></span> <span> </span><span class="NLM_article-title">A practical strategy for the synthesis of 2-dialkylamino-4-arylamino-6-aminopyrimidines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5888</span>– <span class="NLM_lpage">5893</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2009.07.154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=10.1016%2Fj.tetlet.2009.07.154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFarsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2009&pages=5888-5893&author=C.+Liauthor=A.+Rosenau&title=A+practical+strategy+for+the+synthesis+of+2-dialkylamino-4-arylamino-6-aminopyrimidines&doi=10.1016%2Fj.tetlet.2009.07.154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A practical strategy for the synthesis of 2-dialkylamino-4-arylamino-6-aminopyrimidines</span></div><div class="casAuthors">Li, Chaomin; Rosenau, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">5888-5893</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Starting from com. available 4-amino-2,6-dichloropyrimidine, a practical 4-step synthesis of 2-dialkylamino-4-arylamino-6-aminopyrimidines was developed.  This strategy could introduce a diverse set of secondary amines and arylamines to displace the 2- and 4-Cl groups.  The products of this route are otherwise difficult to access.  In addn., 6-amino arylation was carried out to demonstrate the reactivity and utility of 2-dialkylamino-4-arylamino-6-aminopyrimidines as building blocks for assembling interesting pyrimidinamines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtl2shPWSpcrVg90H21EOLACvtfcHk0lhhfSv6fLRb4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFarsrrJ&md5=dfa22f29b058d81cfbb9ca427184457d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2009.07.154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2009.07.154%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DRosenau%26aufirst%3DA.%26atitle%3DA%2520practical%2520strategy%2520for%2520the%2520synthesis%2520of%25202-dialkylamino-4-arylamino-6-aminopyrimidines%26jtitle%3DTetrahedron%2520Lett.%26date%3D2009%26volume%3D50%26spage%3D5888%26epage%3D5893%26doi%3D10.1016%2Fj.tetlet.2009.07.154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i259"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01441">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_02144"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01441?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01441</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Schemes S1–S10 describing the synthesis of compounds <b>5</b>–<b>7</b>, <b>20</b>–<b>22</b>, <b>24</b>, <b>29</b>, <b>30</b>, <b>32</b>, <b>68</b>, <b>69</b>, <b>77</b>–<b>79</b>, and <b>92</b>, <sup>1</sup>H and <sup>13</sup>C NMR data of all inhibitors, and HPLC traces of compounds <b>1</b> and <b>2</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_001.pdf">jm0c01441_si_001.pdf (20.43 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01441/suppl_file/jm0c01441_si_002.csv">jm0c01441_si_002.csv (10.03 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01441&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01441%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-1%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01441" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bc7cff083c16","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
