
<html lang="en"     class="pb-page"  data-request-id="b72ccd93-1229-4672-9f8e-a966345efa7d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-1;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.0c01522;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Generation of Highly Selective, Potent, and Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors" /></meta><meta name="dc.Creator" content="Rachel A.  Rowlands" /></meta><meta name="dc.Creator" content="Qiuyan  Chen" /></meta><meta name="dc.Creator" content="Renee A.  Bouley" /></meta><meta name="dc.Creator" content="Larisa V.  Avramova" /></meta><meta name="dc.Creator" content="John J. G.  Tesmer" /></meta><meta name="dc.Creator" content="Andrew D.  White" /></meta><meta name="dc.Description" content="The ability of G protein-coupled receptor (GPCR) kinases (GRKs) to regulate the desensitization of GPCRs has made GRK2 and GRK5 attractive targets for treating diseases such as heart failure and ca..." /></meta><meta name="Description" content="The ability of G protein-coupled receptor (GPCR) kinases (GRKs) to regulate the desensitization of GPCRs has made GRK2 and GRK5 attractive targets for treating diseases such as heart failure and ca..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 4, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01522" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01522" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01522" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01522" /></link>
        
    
    

<title>Generation of Highly Selective, Potent, and Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01522" /></meta><meta property="og:title" content="Generation of Highly Selective, Potent, and Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0015.jpeg" /></meta><meta property="og:description" content="The ability of G protein-coupled receptor (GPCR) kinases (GRKs) to regulate the desensitization of GPCRs has made GRK2 and GRK5 attractive targets for treating diseases such as heart failure and cancer. Previously, our work showed that Cys474, a GRK5 subfamily-specific residue located on a flexible loop adjacent to the active site, can be used as a covalent handle to achieve selective inhibition of GRK5 over GRK2 subfamily members. However, the potency of the most selective inhibitors remained modest. Herein, we describe a successful campaign to adapt an indolinone scaffold with covalent warheads, resulting in a series of 2-haloacetyl-containing compounds that react quickly and exhibit three orders of magnitude selectivity for GRK5 over GRK2 and low nanomolar potency. They however retain a similar selectivity profile across the kinome as the core scaffold, which was based on Sunitinib." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01522"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01522">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01522&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01522&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01522&amp;href=/doi/10.1021/acs.jmedchem.0c01522" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 566-585</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01484" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01536" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Generation of Highly Selective, Potent, and Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Rachel A. Rowlands</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rachel A. Rowlands</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vahlteich Medicinal Chemistry Core, College of Pharmacy, University of Michigan, 428 Church St, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rachel+A.++Rowlands">Rachel A. Rowlands</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qiuyan Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qiuyan Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Biological Sciences and of Medicinal Chemistry and Molecular Pharmacology, Purdue University, 915 W State St, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qiuyan++Chen">Qiuyan Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Renee A. Bouley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Renee A. Bouley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, Departments of Pharmacology and Biological Chemistry, University of Michigan, 210 Washtenaw Ave, Ann Arbor Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Renee+A.++Bouley">Renee A. Bouley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Larisa V. Avramova</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Larisa V. Avramova</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Biological Sciences and of Medicinal Chemistry and Molecular Pharmacology, Purdue University, 915 W State St, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Larisa+V.++Avramova">Larisa V. Avramova</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John J. G. Tesmer</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John J. G. Tesmer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Biological Sciences and of Medicinal Chemistry and Molecular Pharmacology, Purdue University, 915 W State St, West Lafayette, Indiana 47907, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#cea4baabbda3abbc8ebebbbcaabbabe0abaabb"><span class="__cf_email__" data-cfemail="acc6d8c9dfc1c9deecdcd9dec8d9c982c9c8d9">[email protected]</span></a>. Phone: 765-494-1807.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+J.+G.++Tesmer">John J. G. Tesmer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1125-3727" title="Orcid link">http://orcid.org/0000-0003-1125-3727</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Andrew D. White</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew D. White</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vahlteich Medicinal Chemistry Core, College of Pharmacy, University of Michigan, 428 Church St, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#67100f0e130609030327120a0e040f49020312"><span class="__cf_email__" data-cfemail="88ffe0e1fce9e6ececc8fde5e1ebe0a6edecfd">[email protected]</span></a>. Phone: 734-647-7374.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+D.++White">Andrew D. White</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01522&amp;href=/doi/10.1021%2Facs.jmedchem.0c01522" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 566–585</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 4, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 August 2020</li><li><span class="item_label"><b>Published</b> online</span>4 January 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 January 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01522" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01522</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D566%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRachel%2BA.%2BRowlands%252C%2BQiuyan%2BChen%252C%2BRenee%2BA.%2BBouley%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D1%26contentID%3Dacs.jmedchem.0c01522%26title%3DGeneration%2Bof%2BHighly%2BSelective%252C%2BPotent%252C%2Band%2BCovalent%2BG%2BProtein-Coupled%2BReceptor%2BKinase%2B5%2BInhibitors%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D585%26publicationDate%3DJanuary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01522"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1231</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01522" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Generation of Highly Selective, Potent, and Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Rachel&quot;,&quot;last_name&quot;:&quot;A. Rowlands&quot;},{&quot;first_name&quot;:&quot;Qiuyan&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Renee&quot;,&quot;last_name&quot;:&quot;A. Bouley&quot;},{&quot;first_name&quot;:&quot;Larisa&quot;,&quot;last_name&quot;:&quot;V. Avramova&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;J. G. Tesmer&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;D. White&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;566-585&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01522&quot;},&quot;abstract&quot;:&quot;The ability of G protein-coupled receptor (GPCR) kinases (GRKs) to regulate the desensitization of GPCRs has made GRK2 and GRK5 attractive targets for treating diseases such as heart failure and cancer. Previously, our work showed that Cys474, a GRK5 subfamily-specific residue located on a flexible loop adjacent to the active site, can be used as a covalent handle to achieve selective inhibition of GRK5 over GRK2 subfamily members. However, the potency of the most selective inhibitors remained modest. Herein, we describe a successful campaign to adapt an indolinone scaffold with covalent warheads, resulting in a series of 2-haloacetyl-containing compounds that react quickly and exhibit three orders of magnitude selectivity for GRK5 over GRK2 and low nanomolar potency. They however retain a similar selectivity profile across the kinome as the core scaffold, which was based on Sunitinib.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01522&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01522" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01522&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01522" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01522&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01522" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01522&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01522&amp;href=/doi/10.1021/acs.jmedchem.0c01522" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01522" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01522" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01522%26sid%3Dliteratum%253Aachs%26pmid%3D33393767%26genre%3Darticle%26aulast%3DRowlands%26date%3D2021%26atitle%3DGeneration%2Bof%2BHighly%2BSelective%252C%2BPotent%252C%2Band%2BCovalent%2BG%2BProtein-Coupled%2BReceptor%2BKinase%2B5%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D1%26spage%3D566%26epage%3D585%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (29)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/jmcmar.2021.64.issue-1/20210114/jmcmar.2021.64.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The ability of G protein-coupled receptor (GPCR) kinases (GRKs) to regulate the desensitization of GPCRs has made GRK2 and GRK5 attractive targets for treating diseases such as heart failure and cancer. Previously, our work showed that Cys474, a GRK5 subfamily-specific residue located on a flexible loop adjacent to the active site, can be used as a covalent handle to achieve selective inhibition of GRK5 over GRK2 subfamily members. However, the potency of the most selective inhibitors remained modest. Herein, we describe a successful campaign to adapt an indolinone scaffold with covalent warheads, resulting in a series of 2-haloacetyl-containing compounds that react quickly and exhibit three orders of magnitude selectivity for GRK5 over GRK2 and low nanomolar potency. They however retain a similar selectivity profile across the kinome as the core scaffold, which was based on Sunitinib.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05423" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05423" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">G protein-coupled receptor (GPCR) kinases (GRKs) selectively recognize and phosphorylate activated GPCRs, leading to their desensitization and internalization, a process critical for maintaining cellular homeostasis. The seven human GRKs (GRK1–GRK7) are classified via structural and sequence similarity into three subfamilies: GRK1 (GRK1 and GRK7), GRK2 (GRK2 and GRK3), and GRK4 (GRK4, GRK5, and GRK6).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> GRK1 and GRK7 are found primarily in the retina and GRK4 in the testes, whereas GRK2, GRK3, GRK5, and GRK6 are more ubiquitously expressed. Of these kinases, GRK2 and GRK5 are the two isoforms with the highest concentration in cardiovascular tissue. Because they are overexpressed in the diseased heart and their inhibition or ablation has been shown to prevent heart failure and hypertrophic cardiomyopathy, they have become important therapeutic targets.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> GRK2 and GRK5 are also potential targets for treatment of cancer<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> and other pathophysiological conditions.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> GRK5 is further unique among GRKs because it undergoes Ca<sup>2+</sup>/calmodulin-dependent nuclear localization, where it phosphorylates histone deacetylase 5 (HDAC5), inducing an increase in transcription of associated genes.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In studies where GRK5 was knocked down, cardiomyocytes were protected from hypertrophic cardiomyopathy.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The influence of GRK5 in progressive heart failure and hypertrophic cardiomyopathy however remains unclear, in part because GRK2 can also mediate hypertrophic responses<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and there are few compounds known to have clear GRK5 selectivity that could be used to test mechanistic hypotheses in physiologically relevant cells or animals.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18718" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18718" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Previously, we developed a set of GRK5/6-selective pyrrolopyrimidine inhibitors using a covalent strategy.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> This effort established that Cys474, located on a loop known as the active site tether (AST) that packs over the ATP binding site in AGC kinases, can serve as a covalent handle to achieve GRK5 selectivity. However, the affinities of the best compounds were low micromolar at best. Because the intrinsic affinity of the compound plays an important role in dictating the concentration of the protein-inhibitor covalent complex, which must persist long enough for a covalent reaction to occur,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> the discovery of a more intrinsically potent scaffold for GRK5 inhibition was prioritized. To this end, we considered a set of known GRK5 modulators derived from an indolinone scaffold present in the FDA-approved receptor tyrosine kinase (RTK) inhibitor Sunitinib (<b>1</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), a compound that targets both GRK5 and multiple RTKs including the platelet-derived growth factor receptor and vascular endothelial growth factor receptor.<a onclick="showRef(event, 'ref4 ref11'); return false;" href="javascript:void(0);" class="ref ref4 ref11">(4,11)</a> We focused on Ullrich-57 (<b>5a</b>), which was reported to have low nanomolar activity against GRK5 (GRK5 IC<sub>50</sub> = <0.1 μM), although its selectivity was not reported.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> We independently synthesized <b>5a</b> and also tested its parent compound, Sunitinib, and showed that <b>5a</b> has orders of magnitude more potency against GRK5 than our previous pyrrolopyrimidine scaffold and that both exhibit one to two orders of magnitude selectivity for GRK5 over GRK2 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Modeling the <b>5a</b> complex with GRK5 was performed by docking in the program MOE. The highest scoring pose predicted that <b>5a</b> would bind in the active site of GRK5 in a fashion that would allow the formation of three hydrogen bonds with the hinge, which would consequently project its diethylamine moiety toward the AST loop (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). We hypothesized that replacing the diethylamine arm of the scaffold with thiol reactive warheads could allow for covalent attachment to Cys474 and generation of even more selective and potent covalent GRK5 inhibitors. Thus, the goal for our first series of compounds was to identify a covalent warhead that affords the highest incorporation and the most selectivity for GRK5 over GRK2. As a control, potency against the canonical AGC kinase protein kinase A (PKA) was also evaluated; however, all but one of our compounds had negligible effects against this enzyme.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Lead compounds <b>1</b> and <b>5a</b> with design strategy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking models of lead compound <b>5a</b> and the most potent derivative <b>9j</b> and their comparison with the GRK5·CCG215022 complex. (A) <b>5a</b>, with pink carbons, red oxygens, and blue nitrogens docked using the program MOE into the structure of GRK5 (gray, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WNK">4WNK</a>), with the addition of the AST loop from GRK6 (purple, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NYN">3NYN</a>) because this element was disordered in the 4WNK structure and contains the target Cys474 residue. The diamine moiety extends toward the AST loop where Cys474 is located. Three hydrogen bonds, shown as dashed lines, are formed with the hinge of the kinase domain. None of the compounds in the <b>5a–5f</b> series were able to form adducts despite some that demonstrated high potency. (B) <b>9j</b> (yellow carbons) wherein the amide functionality in the covalent warhead was flipped relative to <b>5a</b> and therefore forms one less hydrogen bond with the hinge. Being less constrained, we hypothesize that this modification allows the warhead to leave the hinge region along a different vector that leads to less steric collisions with the AST and to closer proximity with Cys474. Consequently, compounds in the <b>9a</b> series show adduct formation. Bromine is colored green. (C) Comparison of the fluorophenyl groups of <b>9j</b> and CCG215022 (from 4WNK). The fluorine atom is colored cyan. Both ligands are modeled to form a hydrogen bond with Asp329, an invariant catalytic residue in protein kinases. However, it is unclear whether this region of the scaffold is docked correctly because the conformation of the GRK5 kinase domain, and in particular its P loop and AST, is not known. The SAR in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> is best explained with the fluorophenyl packing under the P loop as it does in the CCG215022 complex.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Values (μM ± SD) and Reactivity of Indolinone Compounds<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0011.gif" alt="" id="gr10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0012.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">All data were fit to a log([inhibitor]) versus response model with variable slope and automatic outlier rejection in GraphPad Prism. Curves that had <i>R</i><sup>2</sup> values less than 0.8 after fitting were omitted. ND, not determined. NA, not applicable. Values in parentheses indicate the number of independent experimental curves. CCG215022 and paroxetine are positive controls for GRK5 and GRK2, respectively.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Incubation time needed to observe adduct formation with GRK5 by intact mass MS.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last"><b>5a</b> and <b>5b</b> are the (<i>R</i>) and (<i>S</i>) enantiomers of Ullrich-57, respectively.</p></div></div><div></div></div><div class="NLM_p">Synthesis for this series began with an amide coupling to give common intermediate <b>3</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In a convergent line of synthesis, a secondary amide coupling with the starting material <b>6</b> gave intermediates <b>4a</b>–<b>4e</b>. Combining the two lines of synthesis, Knoevenagel condensation yielded compounds <b>5a</b>–<b>5f</b>, wherein <b>5a</b> and <b>5b</b> represent the (<i>R</i>) and (<i>S</i>) enantiomers of Ullrich-57. We found that the (<i>S</i>)-enantiomer <b>5b</b> was over 1000-fold less potent (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), indicating that the (<i>R</i>)-enantiomer (<b>5a</b>) places the methyl and benzyl pendants in a more ideal position. The structural rationale for this based on the dock remains elusive because of unknowns about the overall conformation of the GRK5 kinase domain (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). All other compounds were therefore synthesized with the same stereoconfiguration as <b>5a</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Convergent Route to <b>5a–5f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span><b>5a</b> and <b>5b</b> are enantiomers derived from <b>3</b> and <b>3a</b>, respectively.</p></p></figure><div class="NLM_p">Initial diversification tested a series of alkenyl or alkynyl amines as covalent modifiers. The propargyl analogue, <b>5c</b>, demonstrated similar potency to <b>5a</b> (GRK5 IC<sub>50</sub> = 21 nM) but exhibited over a magnitude higher selectivity over GRK2 (2100-fold). We speculate that this high level of selectivity is due to a potential clash of the propargyl warhead with the GRK2 AST and/or large lobe, given the predicted vector for the alkynyl and alkenyl groups (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). For all other amide linked compounds with an alkenyl or alkynyl warhead (<b>5d</b>–<b>5f</b>), the selectivity for GRK5 over GRK2 remained between 330 and 1400 fold, but in each case, the IC<sub>50</sub> for GRK5 was higher than that of <b>5c</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Linker length also contributed to GRK5 affinity, as demonstrated by a comparison of <b>5c</b> with <b>5e</b>, and <b>5d</b> with <b>5f</b>, wherein the latter compounds have homologated alkenyl and alkynyl warheads and somewhat less potency. Thus, a single methylene linker to the reactive center seems optimal for maintaining GRK5 activity among these trial compounds.</div><div class="NLM_p">By reversing the warhead amide, the covalent warheads used in our previous work<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> became synthetically accessible. To accomplish this, Knoevenagel condensation with 3,5-dimethyl-4-nitro-1-pyrrolocarbaldehyde yielded a common intermediate, <b>7a</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). <b>7a</b> was itself tested for inhibitory activity because it possesses the ability to release nitric oxide, a known vasodilator, rendering it a possible dual mechanism compound.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The nitro group was however poorly tolerated (GRK5 IC<sub>50</sub> = 730 nM) and only 10-fold selective for GRK5. A zinc-catalyzed reduction of the nitro group of <b>7a</b> to the free amine <b>8a</b> allowed for rapid derivatization through amide coupling to yield final compounds <b>9a</b>–<b>9h</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). <b>9a</b>, which features a 2-butynyl acid warhead that reacted with Cys474 in our prior study,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> exhibited low micromolar potency (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The commonly used acrylamide and vinyl sulfonamide variants, <b>9b</b> and <b>9c</b>, respectively, showed high nanomolar activity against GRK5 and retained >100-fold selectivity over GRK2. The 2-chloroacetylamido-containing compound <b>9e</b> showed moderate potency (GRK5 IC<sub>50</sub> = 220 nM) but 1500-fold selectivity over GRK2. Because <b>5a</b> had low nanomolar potency, we introduced a similar reactive appendage, dimethylaminobutenoic acid, in <b>9f</b>. This compound showed an increased IC<sub>50</sub> (360 nM, thus 24-fold higher than <b>5a</b>), but only 50-fold selectivity over GRK2. The 2-bromoacetylamido compound <b>9g</b> was found to have a greater potency against GRK5 (IC<sub>50</sub> = 8.6 nM) than its chloro analog <b>9e</b>, while retaining a high level of selectivity against GRK2 (1400-fold). Thus, we concluded that the bromo group of <b>9g</b> must be better at filling a lipophilic pocket than the chloro group, but it was not possible to model this because the structure of the AST loop is uncertain in our GRK5/6 overlay model (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). It was also observed that <b>9g</b> took longer to fully engage Cys474 within the 30 min incubation time frame (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). <b>9h</b> was also potent against GRK5 (IC<sub>50</sub> = 80 nM), whereas compounds <b>10a</b> and <b>10b</b> exhibited only moderate potency.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Intact protein MS for compounds <b>9e</b>, <b>9g</b>, and <b>9j</b>. (A) GRK5 (blue) and GRK5+<b>9e</b> (black) incubated for 30 min. <b>9e</b> demonstrated full labeling of GRK5. (B) GRK5 and GRK5+<b>9g</b> (pink) incubated for 30 min. <b>9g</b> only labeled 50% of GRK5. (C) GRK5 and GRK5+<b>9j</b> (red) incubated for 30 min. <b>9j</b> fully labeled GRK5 in this timeframe. (D) GRK5-C474S mutant (purple) and GRK5-C474S+<b>9e</b> (teal) incubated for 30 min. <b>9e</b> did not label GRK5-C474S, indicating that <b>9e</b> and related compounds are engaging Cys474.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. General Synthetic Route to <b>9a–9h</b> and <b>10a</b> and <b>10b</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We evaluated adduct formation in this series at 30 min and 3 h by intact mass spectrometry (MS) (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf" class="ext-link">Figure S1</a>). Only a few compounds exhibited a signal for GRK5 at the 3 h timepoint. <b>5c</b>, which exhibited similar potency to the parent compound <b>5a</b> and over 2000-fold selectivity against GRK2, was surprisingly not covalent under our conditions. Neither were compounds <b>5d</b>–<b>5f</b>. <b>9a</b> and <b>9b</b> were likewise unreactive at the 30 min timepoint. The vinyl sulfonamide <b>9c</b> was able to react after a 3 h incubation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf" class="ext-link">Figure S1</a>) despite its moderate affinity (740 nM). The chloroacetyl <b>9e</b> however had nearly complete covalent engagement by 30 min (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). <b>9e</b> was unreactive against GRK5-C474S, consistent with Cys474 being the covalent handle (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). Its bromoacetyl analog <b>9g</b> also rapidly reacted with GRK5 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), but <b>9h</b>, <b>10a</b>, and <b>10b</b> were unreactive at 30 min. Based on the sum of the data, we rationalize that compounds in the <b>5c</b>–<b>5f</b> series were unreactive because of constraints placed on the warhead by increased hydrogen bond interactions with the GRK5 hinge relative to <b>9c</b>, <b>9e</b>, and <b>9g</b> where the attaching amide was flipped, affording a different vector for the attached warhead and less conformational constraints (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B).</div><div class="NLM_p">At this point, <b>9e</b> and <b>9g</b> provided the most reactivity combined with the most selectivity for GRK5 over GRK2 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Therefore, we initialized structure–activity relationships (SAR) around indolinone scaffolds bearing 2-haloacetylamido warheads (<a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Benzyl and pyridyl pendants in the R<sub>1</sub> position (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) were explored first. Overall, in terms of potency, GRK5 seemed to have a strong preference for smaller electron-withdrawing substituents in the <i>para</i>-position of the benzyl groups: F (<b>9j</b>) > Cl (<b>9o</b>) > H (<b>9e</b>) > CH<sub>3</sub> (<b>9n</b>). The MOE docked models do not explain this preference, but such behavior would be expected if the benzyl pendant packs instead under the P loop of the active site. In fact, most known GRK inhibitors, including CCG215022, tend to have benzyl groups that pack in this site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The enhancement in potency of 3-Me (<b>9i</b>) is large compared to <b>9e</b>, suggesting that a 3-Me, 4-F analog would be even more potent. However, a lack of appropriate chemical precursors prevented us from studying such combinations. We note that we took advantage of the <i>meta</i>-position of the fluorobenzyl group of paroxetine to make a highly successful series of GRK2 selective inhibitors.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The position of the nitrogen in pyridyl pendants also appeared to make a small difference. In <b>9k</b>, the potency was 3.5-fold higher than that of <b>9l</b> with an <i>ortho</i>-nitrogen, suggesting once again that the <i>para</i>-nitrogen in <b>9k</b> fulfills an electronic deficiency. <b>9j</b>, with a 4-fluoro substituent, had 55-fold more potency (IC<sub>50</sub> = 4 nM) relative to the parent compound <b>9e</b>. It was also able to rapidly form a covalent interaction with Cys474 within 30 min (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C), and <b>9j</b> was thus the most intriguing lead of the second series (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). When the 4-fluoro substituent was maintained and a 2-bromoacetylamido warhead was used, the resulting compound, <b>9p</b>, showed slightly lower potency (IC<sub>50</sub> = 15 nM) than <b>9j</b> or the des-fluoro compound <b>9g</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). This is different than expected from comparison of <b>9g</b> and <b>9e</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), where the bromo substitution rendered much higher potency. The structural explanation for this is unclear.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. General Synthetic Route to <b>9i–9p</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. General Synthetic Route to <b>9q</b>–<b>9t</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. IC<sub>50</sub> Values (μM ± SD) and Reactivity of Indolinone Variants with a Haloacetylamido Warhead<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0013.gif" alt="" id="gr12" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0014.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Data were fit to a log([inhibitor]) versus response model with variable slope and automatic outlier rejection in GraphPad Prism. Curves with <i>R</i><sup>2</sup> values < 0.8 after fitting were omitted. ND, not determined. Numbers in parentheses indicate the number of independent experimental curves.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Incubation time needed to observe adduct formation with GRK5 by intact mass MS.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Same as <b>9j</b> but with a 2-bromoacteylamido warhead.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">IC<sub>50</sub> estimates for this compound for GRK5 and GRK2 are likely high due to its poor solubility in our assay system.</p></div></div><div></div></div><div class="NLM_p">Given the profound effects of chirality on activity exhibited by compounds <b>5a</b> and <b>5b</b>, we also expanded SAR around the benzylic position of the scaffold (R<sub>2</sub> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). When the stereocenter was removed (<b>9q</b>), there was a small increase in potency (IC<sub>50</sub> = 130 nM) relative to the parent compound <b>9e</b> (less than 2-fold). However, there were also similar increases in potency for <b>9s</b> (GRK5 IC<sub>50</sub> = 87 nM) and <b>9t</b> (GRK5 IC<sub>50</sub> = 95 nM). The geminal dimethyl of <b>9r</b> however had greatly reduced potency (IC<sub>50</sub> = 12 μM), but the poor solubility of this compound under our assay conditions may have artifactually increased IC<sub>50</sub> measurements. Therefore, we concluded that the benzylic position (R<sub>2</sub> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) is fairly insensitive to modification, at least when the groups concerned are the size of isopropyl or smaller. Accordingly, when <b>9q</b>, <b>9r</b>, and <b>9s</b> were incubated for 8 h, covalent interaction with GRK5 was observed as in <b>9e</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_001.xls" class="ext-link">Figure S2</a>). It is not clear why <b>9t</b> did not also react.</div><div class="NLM_p">Finally, although the indolinone scaffold offers high potency and selectivity for GRK5 over GRK2, there were a few anticipated drawbacks in terms of its pharmacokinetic properties. First, <b>9a</b>–<b>9t</b> are less soluble due to the number of aromatic rings and their rigid, linear conformation. The pyridyl pendants of <b>9k</b> and <b>9l</b> were a first attempt to address this issue. <b>9l</b> was found to improve solubility 3.5-fold over <b>9e</b> to 125 μg/mL (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_001.xls" class="ext-link">Table S1</a>), which is still ∼3-fold less than reported for Sunitinib (350 μg/mL). Metabolic liability was also a concern because of the high lipophilicity of our compounds and thus their potential metabolism by CYP3A4. Pyridyl pendants are also known to limit metabolic liability relative to benzyl rings, as are fluorine-containing pendants, as in <b>9j</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_001.xls" class="ext-link">Figure S4</a>). We indeed found that <b>9j</b> indeed had a longer half-life (HLM <i>t</i><sub>1/2</sub> = 18.9 min) compared to the des-fluoro compound <b>9e</b> (HLM <i>t</i><sub>1/2</sub> = 13.3 min). However, there was an opposite trend in MLMs. Limited commercial availability of fluorine-substituted benzylic amines from the chiral pool however limited our ability to explore the additional <i>ortho</i>- and <i>meta</i>-substitution patterns.</div><div class="NLM_p">Ullrich-57 (<b>5a</b>) is already a potent (15 nM) and moderately selective GRK5 inhibitor relative to GRK2 (74-fold). Our overarching goal here was to ascertain whether potency and selectivity could be improved by covalent modification of a residue unique to the GRK5 subfamily of kinases. Interestingly, although <b>9e</b>, <b>9g</b>, and <b>9i</b>–<b>9t</b> all had 2-haloacetylamido warheads, only a few were able to label Cys474 within a 30 min time period. The absence of reactivity could be explained by the higher IC<sub>50</sub> exhibited by many of these analogs. Those compounds with lower affinity for GRK5, for example, <b>9c</b>, will bind in the active site of GRK5 more transiently than <b>9e</b>, resulting in a longer incubation period being needed to detect covalent engagement (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf" class="ext-link">Figure S1</a>). <b>9e</b>, <b>9g</b>, and <b>9j</b> were not only the most efficacious at modifying GRK5 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), but also they spanned a range of IC<sub>50</sub> values (220 to 4 nM) and exhibited selectivity for GRK5 over GRK2 by over three orders of magnitude, 20-fold more than that exhibited by the noncovalent compound <b>5a</b>. We therefore performed a detailed kinetic analysis on these compounds to gauge the impact of their reactivity on measured potency and selectivity. We first showed that all three compounds exhibited 3–5-fold less potency against GRK5-C474S (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), whereas our control compound CCG215022 was essentially unchanged. We then determined <i>K</i><sub>I</sub> and <i>k</i><sub>inact</sub> for these compounds (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), which showed that <b>9e</b>, <b>9g</b>, and <b>9j</b> fully reacted with GRK5 before the first time point (hence <i>k</i><sub>inact</sub> > 1 min<sup>–1</sup>) and that <i>K</i><sub>I</sub> scaled with the observed IC<sub>50</sub> values reported for these compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). This shared reactivity is consistent with the fact that they all contain haloacetyl warheads. The results further suggest that a direct comparison of the SAR among compounds in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> along with <b>9e</b> and <b>9g</b> from <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> is reasonable. It also implies that in this situation where the reactive cysteine is on a flexible loop near the ligand binding site, <i>K</i><sub>I</sub> dominates the IC<sub>50</sub> measurements. More reactive warheads like 2-haloacetyls may also be required to take advantage of a more transiently associated cysteine side chain given the anticipated conformational variability of the AST loop.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinetic analysis for (A, B) <b>9e</b>, (C, D) <b>9g</b>, and (E, F) <b>9j</b>. Extracted values for <i>K</i><sub>I</sub> and <i>k</i><sub>inact</sub> from fitting the plots on the right are given in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Data points represent three measurements with error bars indicating standard deviation. Because the time courses (left panels) were not linear, only the 1 min time point was used to estimate <i>k</i><sub>obs</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Impact and Kinetics of Covalent Modification at GRK5-Cys474<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="2" align="center">compound</th><th class="rowsep1 colsep0" align="center">GRK5 IC<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" align="center">GRK5-C474S IC<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" align="center">GRK5-C474S/GRK5</th><th class="rowsep1 colsep0" align="center"><i>K</i><sub>I</sub> (μM)</th><th class="rowsep1 colsep0" align="center"><i>k</i><sub>inact</sub> (min<sup>–1</sup>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CCG273220</td><td class="colsep0 rowsep0" align="left"><b>9e</b></td><td class="colsep0 rowsep0" align="left">0.22 ± 0.04 (3)</td><td class="colsep0 rowsep0" align="left">0.60 ± 0.2 (5)</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">1.0 ± 0.3 (3)</td><td class="colsep0 rowsep0" align="left">>1.0 ± 0.05 (3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CCG273463</td><td class="colsep0 rowsep0" align="left"><b>9g</b></td><td class="colsep0 rowsep0" align="left">0.0086 ± 0.003 (7)</td><td class="colsep0 rowsep0" align="left">0.044 ± 0.02 (5)</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">0.11 ± 0.03 (3)</td><td class="colsep0 rowsep0" align="left">>1.0 ± 0.03 (3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CCG273441</td><td class="colsep0 rowsep0" align="left"><b>9j</b></td><td class="colsep0 rowsep0" align="left">0.0038 ± 0.001 (7)</td><td class="colsep0 rowsep0" align="left">0.019 ± 0.007 (5)</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">0.019 ± 0.01 (3)</td><td class="colsep0 rowsep0" align="left">>0.9 ± 0.02 (3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CCG215022</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.28 ± 0.01 (6)</td><td class="colsep0 rowsep0" align="left">0.22 ± 0.05 (5)</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">n/a</td><td class="colsep0 rowsep0" align="left">n/a</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Errors correspond to standard deviation, with the number of replicates given in parentheses. n/a, not applicable.</p></div></div></div><div class="NLM_p">Having confirmed that <b>5c</b> and <b>9g</b> were potent and highly GRK5 selective relative to GRK2, we explored their kinome-wide selectivity. When tested at 1 μM (∼1000-fold higher than their IC<sub>50</sub> values), <b>5c</b> inhibits GRK5 at 92% and GRK6 at 94%, whereas <b>9g</b> inhibits GRK5 at 93% and GRK6 at 97%. GRK3, a  close homolog of GRK2, was inhibited at only 21%. Both compounds had many off-target effects across the kinome (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf" class="ext-link">Figure S3</a>), which is not surprising because the indolinone series is derived from Sunitinib, which can similarly inhibit the activity of many RTKs. We confirmed that selectivity improved at lower concentrations by testing <b>9g</b> at 100 and 10 nM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). <b>9g</b> however continued to inhibit a small number of tyrosine kinases and Ca<sup>2+</sup>/calmodulin-dependent protein kinases at the lowest dose, at which GRK5 was 50% inhibited, consistent with the IC<sub>50</sub> value measured in our assays (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Kinome-wide selectivity panel for <b>9g</b> at concentrations of (a) 1 μM, (b) 0.1 μM, and (c) 0.01 μM. Locations of GRK5 and GRK6 are denoted by arrows. A clear dose response is evident, with fewer kinases inhibited at the lowest concentration of <b>9g</b>. Illustration reproduced courtesy of Cell Signaling Technology, Inc. (<a href="http://www.cellsignal.com" class="extLink">www.cellsignal.com</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57005" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57005" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have developed a potent series of covalent inhibitors based on the indolinone scaffold that show higher potency and three orders of magnitude selectivity for GRK5 over the GRK2 subfamily, improving on the potency and selectivity of the parent compound Ullrich-57 (4- and 74-fold, respectively). Our MS and kinetic results further suggested that their ability to covalently engage a cysteine unique to the GRK5 subfamily, which contributes 3–5-fold to their IC<sub>50</sub> values under our assay conditions, is responsible for these improved characteristics. The compounds thus set the stage for cell-based assays that would allow one to tease apart the roles of GRK5 versus GRK2 in cellular processes linked to cardiovascular disease and cancer. They also set the stage for a future generation of GRK5 subfamily selective therapeutics that would address potential toxicity issues associated with the use of strongly reactive 2-haloacetyls and further improve selectivity versus other subfamilies of protein kinases as well as their moderate solubility and metabolic stability. Such would facilitate their transition into <i>in vivo</i> trials. While this paper was under review, a new GRK5 inhibitor, KR-39038, with 20 nM IC<sub>50</sub> was reported, although experiments addressing its selectivity versus GRK2 and other kinases and its toxicology were not provided.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This compound showed mild positive effects in a rodent model of hypertrophy but exhibited low bioavailability and a short half-life (<1 h). These results reinforce the potential utility of covalent GRK5 inhibitors that could have long-lasting effects even if the circulating compound is rapidly cleared, which would in turn mitigate off-target toxicity.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27033" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27033" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> General Chemistry</h3><div class="NLM_p last">All reagents from commercial sources were used without further purification unless otherwise noted. <sup>1</sup>H NMR spectra were taken in DMSO-<i>d</i><sub>6</sub>, MeOD, or CDCl<sub>3</sub> at room temperature on Varian MR 400 MHz, Varian Vnmrs 500 MHz, and Varian Vnmrs 700 MHz instruments. The reported chemical shifts for the <sup>1</sup>H NMR spectra were recorded in parts per million (ppm) on the δ scale from an internal tetramethylsilane standard (0.0 ppm). Small-molecule mass spectrometry data was measured using a Waters Corporation Micromass LCT or Agilent6230 Q-TOF instrument. HPLC was used to determine purity of compounds on an Agilent 1100 series with an Agilent Zorbax Eclipse Plus-C18 column. A gradient of 10–90% acetonitrile/water over 6 min followed by 90% acetonitrile/water for 7 min was used with detection at 254 nm. Purity of all compounds was >95% as determined by HPLC. The Sunitinib scaffold is well known to photoisomerize in solution,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and thus our compounds represent a 70:30 mix of active versus inactive isomers. All figures in the paper depict the active isomer.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Intact Protein MS and Tandem MS/MS</h3><div class="NLM_p last">Intact protein MS was acquired with a Phenomenex C4 column paired with an Agilent 6545 Q-TOF LC/MS. For intact MS and tandem MS, all samples were prepared with 20 μM GRK in assay buffer (see below) and 1 mM compound and incubated at 4 °C for 3 h before being quenched with 1.0 μL of formic acid. In tandem MS/MS, we chose Glu-C as the restricting enzyme to avoid small fragments with mass-to-charge ratios below the limit of detection. All samples were digested with Glu-C sequencing enzyme, procured from Sigma Aldrich (Roche Life Sciences subsidiary), and used without further purification. MS/MS experiments were run on a nano-LC (Dionex RSLC-nano) with an Orbitrap Fusion Tribrid ETD mass spectrometer. This work was conducted by the Proteomics Resource Facility at the University of Michigan.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Structural Models and Docking</h3><div class="NLM_p last">GRK5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WNK">4WNK</a>)<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and GRK6 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NYN">3NYN</a>)<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> were loaded into Molecular Operating Environment 2018.01 (Molecular Operating Environment (MOE), 2018.01; Chemical Computing Group ULC, 1010 Sherbrooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7, 2018), and the proteins were prepared using the QuickPrep function. The sequences of both proteins were aligned and used to create a superposition of the two proteins and a hybrid structure with the kinase domain of GRK5 and the AST region of GRK6 was created for docking analysis (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The highest scoring docked pose for each compound in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Inhibition Assays</h3><div class="NLM_p">For compounds <b>5a</b>–<b>9t</b>, IC<sub>50</sub> values for human GRK5 and bovine GRK2 were determined using a radiometric assay, described as follows. GRK (50 nM) was incubated for 3–5 min with 500 nM porcine brain tubulin (PurSolutions) and 0.01–50 μM inhibitor in 20 mM HEPES (pH 7.0), 2 mM MgCl<sub>2</sub>, 0.025% dodecylmaltoside (DDM), and 1% DMSO prior to initiation with the addition of 5 μM ATP supplemented with radioactive [γ-<sup>32</sup>P]-ATP (PerkinElmer Life Sciences). Reactions were quenched at 8 min by addition of 5 μL of 4X SDS gel loading dye to the 10 μL reactions. Samples (12 μL) were separated on a 4–15% Criterion TGX precast gel (Bio-Rad). For potent inhibitors with low nanomolar IC<sub>50</sub>, the inhibitor concentration was adjusted to approximately 0–50× [IC<sub>50</sub>], which was estimated from the first run to get more accurate measurements. Gels were dried, exposed to a storage phosphor screen overnight, and scanned using a Personal Molecular Imager (Bio-Rad). Bands corresponding to phosphorylated tubulin were quantified using ImageQuant, plotted as a function of log[inhibitor], and fit to the four-parameter log(inhibitor) vs response model in GraphPad Prism 7.03 to determine the IC<sub>50</sub>, mean, and standard deviation values. Outliers were eliminated automatically at a 1% Q value. Experiments were performed at least three times.</div><div class="NLM_p">PKA inhibition assays were performed with the ADP-Glo system (Promega Corporation) according to the manufacturer’s instructions. PKA (500 nM) was incubated with 1 μg of CREBtide (KRREILSRRPSYR) (Genscript Corporation) substrate, 50 μM ATP, and  inhibitor for 30 min in 20 mM HEPES (pH 7.0), 2 mM MgCl<sub>2</sub>, 0.025% dodecylmaltoside (DDM), and 4% DMSO. The concentration range of each inhibitor varies depending on its solubility at 4% DMSO with the highest concentration from 100 to 500 μM. After the initial reaction,  ADP-Glo reagent was added to the reaction and allowed to incubate for an additional 40 min. Last, the kinase detection reagent was added and allowed to incubate for 30 min, and luminescence was measured with a FlexStation 3 Multi-mode Microplate Reader (Molecular Devices). All data was analyzed in the same way as in the GRK inhibition assay. Experiments were performed three times in duplicate.</div><div class="NLM_p last">Standard control compounds are run during each assay to assess consistency across time, experimenters, and subtle changes in assay conditions that are sometimes required to keep compounds soluble and dispersed (<i>e.g.</i>, through addition of DDM or 3% DMSO). Paroxetine was used as a control for GRK2<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and PKA and CCG215022 for GRK5.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Covalent Inhibition Kinetic Analysis</h3><div class="NLM_p last">For the covalent inhibitors <b>9e</b>, <b>9g</b>, and <b>9i</b>, <i>K</i><sub>I</sub> and <i>k</i><sub>inact</sub> were determined using a radiometric assay as follows. The reactions contained 50 nM GRK5 and 5 μM porcine brain tubulin (PurSolutions) in 20 mM HEPES (pH 7.0), 2 mM MgCl<sub>2</sub>, 0.025% DDM, and 1% DMSO. Inhibitors at different concentrations (0.02–1 μM for <b>9j</b> and <b>9g</b>, and 0.3–20 μM for <b>9e</b>) were incubated with the reaction mix for 30 s prior to initiation with the addition of 5 μM ATP supplemented with radioactive [γ-<sup>32</sup>P]-ATP (PerkinElmer Life Sciences). Reactions were quenched at 1, 2, and 5 min by addition of 10 μL of 4X SDS gel loading dye to the 10 μL reactions. Samples (12 μL) were separated on a 4–15% Criterion TGX precast gel (Bio-Rad). Gels were dried, exposed to a storage phosphor screen overnight, and scanned using a Personal Molecular Imager (Bio-Rad). Bands corresponding to phosphorylated tubulin were quantified using ImageQuant, background-corrected, normalized to the intensity level of phosphorylation tubulin in the absence of any inhibitor, and plotted as a function of time. Because the inactivation process became nonlinear after the first time point, the observed inactivation rate (<i>k</i><sub>obs</sub>) was estimated using the 1 min time point. The <i>k</i><sub>obs</sub> values were replotted against inhibitor concentration and fitted to the equation, <i>k</i><sub>obs</sub> = <i>k</i><sub>inact</sub>[<i>I</i>]/(<i>K</i><sub>I</sub> + [<i>I</i>]), to obtain <i>k</i><sub>inact</sub> and <i>K</i><sub>I</sub> in GraphPad Prism 7.03. Experiments were performed three times.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Thermodynamic Solubility and Microsomal Stability</h3><div class="NLM_p last">Thermodynamic solubility for compounds was determined by Analiza Inc. (Cleveland OH, <a href="http://analiza.com" class="extLink">analiza.com</a>) using a miniaturized shake-flask solubility assay. Microsomal stability was determined by the Pharmacokinetics and Mass Spectrometry Core at the University of Michigan. Compounds <b>9c</b>, <b>9e</b>, and <b>9j</b> were dissolved in DMSO (1 mM) and then further diluted to 100 μM with 0.1 M phosphate buffer (with 3.3 mM MgCl<sub>2</sub>). Microsomes (20 mg/mL) in 0.1 M phosphate buffer (with 3.3 mM MgCl) were dosed with 20 μL of NADPH (4 mg in 240 μL of 0.1 M phosphate buffer) and incubated at 37 °C for 3 min. Microsomes were then dosed with 4 μL of 100 μM of <b>9c</b>, <b>9e</b>, and <b>9j</b>. At the following time points (0, 5, 10, 15, 30, 45, and 60 min), the reaction solutions were  stopped with cold acetonitrile containing 25 nM CE302 as an internal standard. The incubation solution was centrifuged at 3500 rpm for 10 min to precipitate protein. The supernatant was used for LC/MS/MS analysis.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Chemical Synthesis and Validation</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (<i>R</i>)-2-Oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>3</b>)</h4><div class="NLM_p last">To a round-bottom flask was added 297.7 mg (1.69 mmol) of 2-oxoindoline-5-carboxylic acid, dissolved in 7 mL of dry DMF. To this dark red solution were added 0.244 mL (1.88 mmol) of (<i>S</i>)-1-phenylethan-1-amine, 0.300 mL (1.69 mmol) of DIPEA, and 746.1 mg (1.95 mmol) of HATU. The resultant dark red solution was allowed to stir at room temperature for 12 h, then added 200 mL of sat. Na<sub>2</sub>CO<sub>3</sub>, and extracted with EtOAc (3 × 100 mL). The combined organic layer was washed with brine (2 × 50 mL) and then dried over MgSO<sub>4.</sub> Purified by column chromatography (0–15% DCM/MeOH) to give the desired product as a strawberry pink solid. Yield: 401.9 mg, 84%. Molecular formula: C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 280.12 ESI-MS found: 281.1283 [M + 1] HPLC: 5.198. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (s, 1H), 8.63 (d, <i>J</i> = 8.0 Hz, 1H), 7.78 (d, <i>J</i> = 7.0 Hz, 2H), 7.39–7.35 (m, 2H), 7.31 (dd, <i>J</i> = 8.4, 6.8 Hz, 2H), 7.23–7.18 (m, 1H), 6.88–6.83 (m, 1H), 5.15 (p, <i>J</i> = 7.2 Hz, 1H), 3.53 (s, 2H), 1.46 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO) δ 177.17, 165.76, 146.85, 145.65, 129.29, 128.67, 126.52, 126.06, 108.90, 48.85, 36.09, 22.84.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (<i>S</i>)-2-Oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>3a</b>)</h4><div class="NLM_p last">To a round-bottom flask was added 201.8 mg (1.13 mmol) of 2-oxoindoline-5-carboxylic acid dissolved in 7 mL of dry DMF. To this dark red solution were added 0.160 mL (1.24 mmol) of (<i>S</i>)-1-phenylethan-1-amine, 0.200 mL (1.13 mmol) of DIPEA, and 492.4 mg (1.30 mmol) of HATU. The resultant dark red solution was allowed to stir at room temperature for 12 h, then added 200 mL of sat. Na<sub>2</sub>CO<sub>3</sub>, and extracted with EtOAc (3 × 100 mL). The combined organic layer was washed with brine (2 × 50 mL) and then dried over MgSO<sub>4.</sub> Purified by column chromatography (0–15% DCM/MeOH) to give the desired product as a strawberry pink solid. Yield: 280 mg, 84%. Molecular formula: C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 280.12 ESI-MS found: 281.0903 [M + 1] HPLC: 5.249. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.67 (s, 1H), 8.66 (d, <i>J</i> = 8.0 Hz, 1H), 7.78 (d, <i>J</i> = 7.2 Hz, 2H), 7.38 (d, <i>J</i> = 8.4 Hz, 2H), 7.31 (t, <i>J</i> = 7.6 Hz, 2H), 7.20 (t, <i>J</i> = 7.3 Hz, 1H), 6.86 (d, <i>J</i> = 8.2 Hz, 1H), 5.15 (p, <i>J</i> = 7.2 Hz, 1H), 3.53 (s, 2H), 1.46 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.83, 165.50, 146.49, 145.22, 128.27, 127.77, 127.54, 126.60, 126.14, 125.64, 123.69, 108.62, 48.52, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 35.70, 22.40.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (<i>R</i>)-2-Oxo-<i>N</i>-(1-(<i>m</i>-tolyl)ethyl)indoline-5-carboxamide (<b>3b</b>)</h4><div class="NLM_p last">To a round-bottom flask was added 236.1 mg (1.33 mmol) of 2-oxoindoline-5-carboxylic acid, dissolved in 7 mL of dry DMF. To this dark red solution were added 0.200 mL (1.47 mmol) of (<i>R</i>)-1-(<i>m</i>-tolyl)ethan-1-amine, 0.300 mL (1.73 mmol) of DIPEA, and 510.2 mg (1.33 mmol) of HATU. The resultant dark red solution was allowed to stir at room temperature for 12 h, then added 200 mL of sat. Na<sub>2</sub>CO<sub>3</sub>, and extracted with EtOAc (3 × 100 mL). The combined organic layer was washed with brine (2 × 50 mL) and then dried over MgSO<sub>4.</sub> Purified by column chromatography (0–15% DCM/MeOH) to give the desired product as a strawberry pink solid. Yield: 125.6 mg, 31%. Molecular formula: C<sub>18</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 294.14 ESI-MS found: 295.0167 [M + 1] HPLC: 5.395. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.61 (s, 1H), 8.58 (d, <i>J</i> = 8.1 Hz, 1H), 7.77 (d, <i>J</i> = 8.1 Hz, 2H), 7.18 (p, <i>J</i> = 7.6 Hz, 4H), 7.02 (d, <i>J</i> = 7.3 Hz, 1H), 6.85 (d, <i>J</i> = 7.9 Hz, 1H), 5.11 (p, <i>J</i> = 7.3 Hz, 1H), 2.28 (s, 3H), 1.44 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.62, 165.16, 146.32, 145.07, 137.11, 128.06, 127.65, 127.48, 127.10, 126.66, 125.53, 123.51, 123.09, 108.36, 48.29, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 38.22, 35.57, 22.34, 21.11.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (<i>R</i>)-<i>N</i>-(1-(4-Fluorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>3c</b>)</h4><div class="NLM_p last">To a round-bottom flask was added 246.6 mg (1.41 mmol) of 2-oxoindoline-5-carboxylic acid, dissolved in 7 mL of dry DMF. To this dark red solution were added 0.200 mL (1.55 mmol) of (<i>R</i>)-1-(4-fluorophenyl)ethan-1-amine, 0.25 mL (1.41 mmol) of DIPEA, and 667.1 mg (1.61 mmol) of HATU. The resultant dark red solution was allowed to stir at room temperature for 12 h, then added 200 mL of sat. Na<sub>2</sub>CO<sub>3</sub>, and extracted with EtOAc (3 × 100 mL). The combined organic layer was washed with brine (2 × 50 mL) and then dried over MgSO<sub>4.</sub> Purified by column chromatography (0–15% DCM/MeOH) to give the desired product as a strawberry pink solid. Yield: 315 mg, 71%. Molecular formula: C<sub>17</sub>H<sub>15</sub>FN<sub>2</sub>O<sub>2</sub> ESI-MS calc: 298.11 ESI-MS found: 299.1216 [M + 1] HPLC: 5.414. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.61 (s, 1H), 8.62 (d, <i>J</i> = 8.0 Hz, 1H), 7.76 (d, <i>J</i> = 7.5 Hz, 2H), 7.43–7.38 (m, 2H), 7.16–7.08 (m, 2H), 6.87–6.82 (m, 1H), 5.14 (p, <i>J</i> = 7.2 Hz, 1H), 3.53 (s, 2H), 1.45 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.84, 165.47, 160.34, 146.46, 141.34, 128.06, 128.01, 127.77, 127.49, 125.69, 123.63, 114.99, 114.87, 108.55, 47.90, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 38.33, 35.67, 22.35.</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (<i>R</i>)-2-Oxo-<i>N</i>-(1-(pyridin-4-yl)ethyl)indoline-5-carboxamide (<b>3d</b>)</h4><div class="NLM_p last">To a round-bottom flask was added 248.1 mg (1.41 mmol) of 2-oxoindoline-5-carboxylic acid dissolved in 7 mL of dry DMF. To this dark red solution were added 0.200 mL (1.55 mmol) of (<i>R</i>)-1-(pyridin-4-yl)ethan-1-amine, 0.25 mL (1.41 mmol) of DIPEA, and 652.4 mg (1.61 mmol) of HATU. The resultant dark red solution was allowed to stir at room temperature for 12 h, then added 200 mL of sat. Na<sub>2</sub>CO<sub>3</sub>, and extracted with EtOAc (3 × 100 mL). The combined organic layer was washed with brine (2 × 50 mL) and then dried over MgSO<sub>4.</sub> Purified by column chromatography (0–15% DCM/MeOH) to give the desired product as a strawberry pink solid. Yield: 200 mg, 48%. Molecular formula: C<sub>16</sub>H<sub>15</sub>N<sub>3</sub>O<sub>2</sub> ESI-MS calc: 281.12 ESI-MS found: 282.1236 [M + 1] HPLC: 2.378. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.64 (s, 1H), 8.74 (dd, <i>J</i> = 7.6, 2.1 Hz, 1H), 8.55–8.52 (m, 2H), 7.79 (dd, <i>J</i> = 7.5, 2.2 Hz, 2H), 7.45–7.42 (m, 2H), 7.08 (d, <i>J</i> = 2.3 Hz, 1H), 7.01 (s, 1H), 6.89–6.85 (m, 1H), 5.14 (td, <i>J</i> = 7.3, 2.2 Hz, 1H), 3.55 (s, 2H), 3.46–3.41 (m, 3H), 1.47 (dd, <i>J</i> = 7.2, 2.3 Hz, 3H), 1.07–1.03 (m, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.63, 165.65, 155.16, 148.63, 146.56, 127.76, 127.03, 125.62, 123.57, 121.58, 108.43, 56.00, 47.92, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 35.56, 21.45, 18.54.</div></div><div id="sec4_7_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (<i>R</i>)-2-Oxo-<i>N</i>-(1-(pyridin-2-yl)ethyl)indoline-5-carboxamide (<b>3e</b>)</h4><div class="NLM_p last">To a round-bottom flask was added 251.1 mg (1.41 mmol) of 2-oxoindoline-5-carboxylic acid dissolved in 7 mL of dry DMF. To this dark red solution were added 0.200 mL (1.55 mmol) of (<i>R</i>)-1-(pyridin-2-yl)ethan-1-amine, 0.25 mL (1.41 mmol) of DIPEA, and 625.1 mg (1.62 mmol) of HATU. The resultant dark red solution was allowed to stir at room temperature for 12 h, then added 200 mL of sat. Na<sub>2</sub>CO<sub>3</sub>, and extracted with EtOAc (3 × 100 mL). The combined organic layer was washed with brine (2 × 50 mL) and then dried over MgSO<sub>4.</sub> Purified by column chromatography (0–15% DCM/MeOH) to give the desired product as a strawberry pink solid. Yield: 165 mg, 39%. Molecular formula: C<sub>16</sub>H<sub>15</sub>N<sub>3</sub>O<sub>2</sub> ESI-MS calc: 281.12 ESI-MS found: 282.1236 [M + 1] HPLC: 2.111. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (d, <i>J</i> = 5.2 Hz, 1H), 8.63 (t, <i>J</i> = 6.6 Hz, 1H), 8.51 (t, <i>J</i> = 5.3 Hz, 1H), 7.80 (d, <i>J</i> = 5.6 Hz, 2H), 7.74 (q, <i>J</i> = 7.0 Hz, 1H), 7.38 (t, <i>J</i> = 6.8 Hz, 1H), 7.24 (q, <i>J</i> = 6.0 Hz, 1H), 6.86 (t, <i>J</i> = 6.7 Hz, 1H), 5.17 (p, <i>J</i> = 7.2 Hz, 1H), 3.54 (d, <i>J</i> = 5.2 Hz, 3H), 1.49 (t, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 177.27, 166.16, 163.50, 149.12, 137.26, 128.18, 127.79, 126.11, 124.08, 122.46, 120.59, 108.98, 50.76, 40.22, 40.10, 39.98, 39.86, 39.74, 39.62, 39.50, 36.07, 21.45.</div></div><div id="sec4_7_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (<i>R</i>)-<i>N</i>-(1-(3-Chlorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>3f</b>)</h4><div class="NLM_p last">To a round-bottom flask was added 251.1 mg (1.41 mmol) of 2-oxoindoline-5-carboxylic acid dissolved in 7 mL of dry DMF. To this dark red solution were added 0.200 mL (1.55 mmol) of (<i>R</i>)-1-(3-chlorophenyl)ethan-1-amine, 0.25 mL (1.41 mmol) of DIPEA, and 717.7 mg (1.91 mmol) of HATU. The resultant dark red solution was allowed to stir at room temperature for 12 h, then added 200 mL of sat. Na<sub>2</sub>CO<sub>3</sub>, and extracted with EtOAc (3 × 100 mL). The combined organic layer was washed with brine (2 × 50 mL) and then dried over MgSO<sub>4.</sub> Purified by column chromatography (0–15% DCM/MeOH) to give the desired product as a strawberry pink solid. Yield: 423.3 mg, 91%. Molecular formula: C<sub>17</sub>H<sub>15</sub>ClN<sub>2</sub>O<sub>2</sub> ESI-MS calc: 314.08 ESI-MS found: 315.0438 [M + 1] HPLC: 5.730. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.64–10.57 (m, 1H), 8.65 (dd, <i>J</i> = 7.5, 2.2 Hz, 1H), 7.78–7.74 (m, 2H), 7.41 (t, <i>J</i> = 2.1 Hz, 1H), 7.33 (dt, <i>J</i> = 6.7, 1.8 Hz, 2H), 7.26 (dp, <i>J</i> = 6.4, 2.2 Hz, 1H), 6.85 (dd, <i>J</i> = 8.1, 2.5 Hz, 1H), 5.15–5.06 (m, 1H), 3.53 (s, 2H), 1.45–1.42 (m, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.91, 165.61, 148.09, 146.73, 133.15, 130.39, 127.97, 126.74, 126.14, 125.88, 125.13, 123.78, 108.70, 56.27, 48.40, 40.12, 40.00, 39.88, 39.76, 39.64, 39.52, 39.40, 35.83, 22.43, 18.80.</div></div><div id="sec4_7_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (<i>R</i>)-2-Oxo-<i>N</i>-(1-(<i>p</i>-tolyl)ethyl)indoline-5-carboxamide (<b>3g</b>)</h4><div class="NLM_p last">To a round-bottom flask was added 244.2 mg (1.41 mmol) of 2-oxoindoline-5-carboxylic acid dissolved in 7 mL of dry DMF. To this dark red solution were added 0.220 mL (1.55 mmol) of (<i>R</i>)-1-(3-chlorophenyl)ethan-1-amine, 0.25 mL (1.41 mmol) of DIPEA, and 622.1 mg (1.62 mmol) of HATU. The resultant dark red solution was allowed to stir at room temperature for 12 h, then added 200 mL of sat. Na<sub>2</sub>CO<sub>3</sub>, and extracted with EtOAc (3 × 100 mL). The combined organic layer was washed with brine (2 × 50 mL) and then dried over MgSO<sub>4.</sub> Purified by column chromatography (0–15% DCM/MeOH) to give the desired product as a strawberry pink solid. Yield: 300.8 mg, 69%. Molecular formula: C<sub>18</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 294.14 ESI-MS found: 317 [M + Na] HPLC: 5.556. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.61 (s, 1H), 8.57 (d, <i>J</i> = 8.1 Hz, 1H), 7.77 (d, <i>J</i> = 8.1 Hz, 2H), 7.26 (d, <i>J</i> = 7.8 Hz, 2H), 7.11 (d, <i>J</i> = 7.8 Hz, 2H), 6.85 (d, <i>J</i> = 8.0 Hz, 1H), 5.12 (p, <i>J</i> = 7.3 Hz, 1H), 3.53 (s, 2H), 2.26 (s, 3H), 1.44 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.63, 165.17, 146.30, 142.10, 135.44, 128.66, 127.64, 127.54, 125.93, 125.51, 123.51, 108.36, 48.03, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 35.57, 22.29, 20.59.</div></div><div id="sec4_7_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>R</i>)-<i>N</i>-(1-(4-Chlorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>3h</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>3</b>. Yields a strawberry pink solid, 471.0 mg, quantitative yield. Molecular formula: C<sub>17</sub>H<sub>15</sub>ClN<sub>2</sub>O<sub>2</sub> ESI-MS calc: 314.08 ESI-MS found: 337.0717 [M + Na] HPLC: 5.682. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.61 (s, 1H), 8.64 (d, <i>J</i> = 7.9 Hz, 1H), 7.76 (d, <i>J</i> = 7.9 Hz, 2H), 7.42–7.34 (m, 4H), 6.85 (d, <i>J</i> = 8.1 Hz, 1H), 5.13 (p, <i>J</i> = 7.2 Hz, 1H), 3.53 (s, 2H), 1.45 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 176.61, 165.29, 146.40, 144.17, 130.98, 128.09, 127.92, 127.67, 127.28, 125.55, 123.49, 108.37, 47.87, 40.00, 39.83, 39.67, 39.50, 39.33, 39.17, 39.00, 35.55, 22.09.</div></div><div id="sec4_7_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-Benzyl-2-oxoindoline-5-carboxamide (<b>3i</b>)</h4><div class="NLM_p last">Prepared with protocol described for <b>3</b>. Yields a strawberry pink solid, 174.5 mg, 56%. Molecular formula: C<sub>16</sub>H<sub>14</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 266.11 ESI-MS found: 267.2180 [M + 1] HPLC: 4.822. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.61 (s, 1H), 8.87 (t, <i>J</i> = 6.1 Hz, 1H), 7.79–7.75 (m, 2H), 7.34–7.28 (m, 5H), 7.23 (dt, <i>J</i> = 7.2, 4.3 Hz, 2H), 6.86 (d, <i>J</i> = 8.0 Hz, 1H), 4.46 (d, <i>J</i> = 5.9 Hz, 2H), 3.53 (s, 2H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.60, 165.95, 146.42, 139.88, 128.65, 128.19, 128.12, 127.70, 127.59, 127.47, 127.29, 127.11, 126.61, 125.66, 123.45, 108.44, 47.80, 45.00, 42.52, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 38.22, 35.57.</div></div><div id="sec4_7_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2-Oxo-<i>N</i>-(2-phenylpropan-2-yl)indoline-5-carboxamide (<b>3j</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>3</b>. Yields a strawberry pink solid, 210.8 mg, 48%. Molecular formula: C<sub>18</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 294.14 ESI-MS found: 295.1456 [M + 1], 317.1275 [M + Na] HPLC: 5.199. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.60 (s, 1H), 8.23 (s, 1H), 7.74 (s, 1H), 7.74–7.71 (m, 1H), 7.35 (d, <i>J</i> = 7.8 Hz, 2H), 7.26 (t, <i>J</i> = 7.7 Hz, 2H), 7.15 (t, <i>J</i> = 7.2 Hz, 1H), 6.84 (d, <i>J</i> = 8.1 Hz, 1H), 3.53 (s, 2H), 1.65 (s, 6H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.64, 165.50, 148.19, 146.15, 128.51, 128.35, 127.82, 1s27.67, 125.59, 125.37, 124.81, 124.61, 123.65, 108.26, 55.22, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 35.59, 29.66.</div></div><div id="sec4_7_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>R</i>)-<i>N</i>-(2-Methyl-1-phenylpropyl)-2-oxoindoline-5-carboxamide (<b>3k</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>3</b>. Yields a strawberry pink solid, 389.9 mg, quantitative yield. Molecular formula: C<sub>19</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 308.15 ESI-MS found: 309.2172 HPLC: 5.793. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.44 (s, 1H), 8.37 (d, <i>J</i> = 8.9 Hz, 1H), 7.58 (dd, <i>J</i> = 10.7, 2.9 Hz, 2H), 7.22 (d, <i>J</i> = 7.6 Hz, 2H), 7.12 (t, <i>J</i> = 7.5 Hz, 2H), 7.03 (t, <i>J</i> = 7.3 Hz, 1H), 6.68 (d, <i>J</i> = 8.0 Hz, 1H), 4.48 (t, <i>J</i> = 9.2 Hz, 1H), 3.36 (s, 2H), 1.06 (d, <i>J</i> = 6.4 Hz, 2H), 0.83 (d, <i>J</i> = 6.5 Hz, 3H), 0.53 (d, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.69, 165.65, 164.62, 162.32, 146.32, 143.33, 128.02, 127.75, 127.70, 127.41, 126.61, 125.54, 123.53, 108.43, 59.92, 56.08, 41.67, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 38.23, 35.78, 35.62, 32.55, 30.77, 20.09, 19.90, 18.56.</div></div><div id="sec4_7_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 2-Oxo-<i>N</i>-(3-phenyloxetan-3-yl)indoline-5-carboxamide (<b>3l</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>3</b>. Yields a strawberry pink solid, 62.1 mg, 31%. Molecular formula: C<sub>18</sub>H<sub>16</sub>N<sub>2</sub>O<sub>3</sub> ESI-MS calc: 308.12 ESI-MS found: 309.1235 HPLC: 4.601. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.49 (d, <i>J</i> = 4.6 Hz, 1H), 9.16 (d, <i>J</i> = 4.6 Hz, 1H), 7.65–7.60 (m, 2H), 7.38–7.34 (m, 2H), 7.22–7.18 (m, 2H), 7.10 (tdd, <i>J</i> = 7.3, 4.9, 2.2 Hz, 1H), 6.74–6.70 (m, 1H), 4.82 (dd, <i>J</i> = 6.7, 4.8 Hz, 2H), 4.59 (dd, <i>J</i> = 6.7, 4.8 Hz, 2H), 3.38 (d, <i>J</i> = 4.7 Hz, 2H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 176.66, 165.18, 146.78, 143.10, 129.97, 128.28, 128.17, 127.72, 126.91, 126.72, 126.06, 125.80, 125.41, 124.81, 123.53, 120.11, 108.72, 108.56, 81.89, 58.27, 53.49, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 38.23, 35.58, 35.52, 30.66, 14.07.</div></div><div id="sec4_7_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-(2-(Diethylamino)ethyl)-5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxamide (<b>4a</b>)</h4><div class="NLM_p last">To a round-bottom flask were added 200.9 mg (1.20 mmol) of 5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxylic acid, 365.8 mg (1.79 mmol) of EDCI, 277.7 mg (1.79 mmol) of HOBt, 7 mL of dry DMF, 0.200 mL (1.44 mmol) of <i>N</i>1,<i>N</i>1-diethylethane-1,2-diamine, and 0.34 mL (2.39 mmol) of TEA. The dark red solution was allowed to stir at room temperature for 12 h before being quenched with water and extracted with DCM (3 × 30 mL). The combined organic layer was washed with brine (2 × 20 mL) and then with 10% citric acid (3 × 50 mL), drawing the desired product into the water layer<i>.</i> The aqueous layer was basified with Na<sub>2</sub>CO<sub>3</sub>, bringing the pH up to 10, and then extracted with DCM (4 × 50 mL). The combined organic layers were dried over MgSO<sub>4</sub> and then evaporated onto a silica gel and purified by column chromatography (5–15% MeOH/DCM). The solvent was removed under pressure to give a yellow solid. Result: light yellow solid, 90 mg, 27%. Molecular formula: C<sub>14</sub>H<sub>23</sub>N<sub>3</sub>O<sub>2</sub>, ESI-MS calc: 265.18 ESI-MS found: 266.1749 HPLC: 2.681. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.85 (s, 1H), 9.54 (s, 1H), 7.95 (s, 2H), 2.34 (d, <i>J</i> = 20.8 Hz, 7H), 0.99 (s, 6H).</div></div><div id="sec4_7_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 5-Formyl-2,4-dimethyl-<i>N</i>-(prop-2-yn-1-yl)-1<i>H</i>-pyrrole-3-carboxamide (<b>4b</b>)</h4><div class="NLM_p last">To a round-bottom flask were added 199.4 mg (1.20 mmol) of 5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxylic acid, 688.0 mg (1.79 mmol) of HATU, 6 mL of dry DMF, 0.100 mL (1.44 mmol) of propargylamine, and 0.34 mL (2.39 mmol) of TEA. The dark red solution was allowed to stir at room temperature for 12 h before being quenched with water and extracted with DCM (3 × 30 mL). The combined organic layers were taken and washed with brine (2 × 20 mL) and then with 10% citric acid (3 × 50 mL), taking the desired product into the aqueous layer. The aqueous layer was basified with Na<sub>2</sub>CO<sub>3</sub>, bringing the pH up to 10, and then extracted with DCM (4 × 50 mL). The combined organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed to give the final product as a light orange oil. Result: light orange oil, 61 mg, 25%. Molecular formula: C<sub>11</sub>H<sub>12</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 204.09 ESI-MS found: 205.0873 [M + 1], 242.1167 [M + K] HPLC: 3.625. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.87 (s, 1H), 9.55 (s, 1H), 7.95 (s, 16H), 3.97 (ddd, <i>J</i> = 12.7, 5.7, 2.5 Hz, 3H), 3.08 (t, <i>J</i> = 2.4 Hz, 1H), 2.36 (s, 3H), 2.31 (s, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 177.82, 164.77, 139.11, 138.51, 128.26, 119.20, 89.92, 82.16, 72.95, 55.38, 40.24, 38.71, 28.42, 12.90, 10.00.</div></div><div id="sec4_7_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-Allyl-5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxamide (<b>4c</b>)</h4><div class="NLM_p last">To a round-bottom flask were added 204.4 mg (1.20 mmol) of 5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxylic acid, 350.3 mg of EDCI (1.79 mmol), 275.3 mg (1.79 mmol) of HOBt, 6 mL of dry DMF, 0.110 mL (1.44 mmol) of allylamine, and 0.34 mL (2.39 mmol) of TEA. The dark red solution was allowed to stir at room temperature for 12 h before being quenched with water and extracted with DCM (3 × 30 mL)<i>.</i> Combined organic layers were taken and washed with LiCl (3 × 20 mL) and then with 10% citric acid (3 × 50 mL), drawing the desired product into the aqueous layer. The aqueous layer was basified with Na<sub>2</sub>CO<sub>3</sub>, bringing the pH up to 8, and then extracted with DCM (4 × 50 mL). The combined organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed to give a yellow solid. Result: light yellow solid, 64 mg, 25%. Molecular formula: C<sub>11</sub>H<sub>14</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 206.11 ESI-MS: 246.1621 [M+ MeCN] HPLC: 3.885. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.38 (s, 1H), 9.54 (s, 1H),8.17 (s, 1H), 6.01 (tt, <i>J</i> = 10.8, 5.4 Hz, 1H), 5.87 (ddt, <i>J</i> = 16.4, 10.6, 5.3 Hz, 2H), 5.18 (t, <i>J</i> = 15.7 Hz, 3H), 5.08 (dd, <i>J</i> = 16.5, 10.2 Hz, 3H), 4.14 (d, <i>J</i> = 5.5 Hz, 2H), 3.82 (t, <i>J</i> = 5.7 Hz, 3H), 2.31 (s, 5H), 2.23 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 161.78, 150.29, 135.45, 135.21, 115.01, 114.35, 114.20, 40.54, 40.50, 39.50, 39.33, 39.17, 39.00, 38.83, 38.67, 38.50, 35.26, 30.25, 11.95, 9.27.</div></div><div id="sec4_7_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>-(But-3-yn-1-yl)-5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxamide (<b>4d</b>)</h4><div class="NLM_p last">To a round-bottom flask were added 199.6 mg (1.20 mmol) of 5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxylic acid, 684.3 mg (1.79 mmol) of HATU, 6 mL of dry DMF, 0.100 mL (1.44 mmol) of 1-amino-3-butyne, and 0.30 mL (2.39 mmol) of DIPEA. The dark red solution was allowed to stir at room temperature for 12 h before being quenched with water and extracted with DCM (3 × 30 mL). The combined organic layers were washed with LiCl (2 × 20 mL) and then with 10% citric acid (3 × 50 mL), drawing the desired product into the water layer. The aqueous layer was basified with Na<sub>2</sub>CO<sub>3</sub>, bringing the pH up to 10, and then extracted with DCM (4 × 50 mL). The combined organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed under pressure to give the final product as a yellow solid. Result: light yellow solid, 99.6 mg, 38.1%. Molecular formula: C<sub>12</sub>H<sub>14</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 218.11 ESI-MS found: 219.1129 HPLC: 3.997. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.83 (s, 1H), 2.83 (s, 1H), 2.69 (d, <i>J</i> = 1.0 Hz, 1H), 2.60 (d, <i>J</i> = 2.4 Hz, 3H), 2.37 (s, 4H), 2.32 (s, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 178.94, 164.54, 160.23, 152.56, 147.04, 146.58, 126.55, 115.31, 88.48, 82.43, 72.02, 55.77, 37.85, 18.85, 13.97, 10.55.</div></div><div id="sec4_7_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-(But-3-en-1-yl)-5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxamide (<b>4e</b>)</h4><div class="NLM_p last">To a round-bottom flask were added 197.8 mg (1.20 mmol) of 5-formyl-2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxylic acid, 685.8 mg (1.79 mmol) of HATU, 6 mL of dry DMF, 107.1 mg (1.44 mmol) of but-3-en-1-amine, and 0.45 mL (2.40 mmol) of DIPEA. The dark red solution was allowed to stir at room temperature for 12 h before being quenched with water and extracted with DCM (3 × 30 mL). The combined organic layers were washed with brine (2 × 20 mL) and then with 10% citric acid (3 × 50 mL), drawing the desired material in to the aqueous layer. The aqueous layer was basified with Na<sub>2</sub>CO<sub>3</sub>, bringing the pH up to 10, and then extracted with DCM (4 × 50 mL). The combined organic layer was dried over MgSO<sub>4</sub>. Result: light orange oil, 106.1 mg, 37.2%. Molecular formula: C<sub>12</sub>H<sub>16</sub>N<sub>2</sub>O<sub>2</sub> ESI-MS calc: 220.12 MS: 221.1304 HPLC: 5.554. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 8.84 (d, <i>J</i> = 4.5 Hz, 1H), 8.73 (d, <i>J</i> = 8.4 Hz, 1H), 7.95 (s, 4H), 7.66 (dd, <i>J</i> = 8.7, 4.6 Hz, 1H), 5.12–4.99 (m, 1H), 2.59 (s, 7H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 179.37, 165.02, 162.73, 152.99, 146.27, 140.66, 134.88, 130.23, 122.09, 36.21, 31.20, 14.39, 10.98.</div></div><div id="sec4_7_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>R</i>,<i>Z</i>)-3-((4-((2-(Diethylamino)ethyl)carbamoyl)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>5a</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 97.9 mg (0.371 mmol) of (<b>3</b>) and 107.0 mg (0.357 mmol) of (<b>4a</b>), all of which were dissolved in abs. EtOH (3.5 mL). To this solution were added two drops of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then the product was filtered off as an orange solid. Yield: orange solid, 38.3 mg, 22%. Molecular formula: C<sub>31</sub>H<sub>37</sub>N<sub>5</sub>O<sub>3</sub> ESI-MS calc: 527.29 ESI-MS found: 528.2955 HPLC: 5.621. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.59 (s, 1H), 11.13 (s, 1H), 8.59 (d, <i>J</i> = 8.1 Hz, 1H), 8.25 (s, 1H), 7.71 (d, <i>J</i> = 6.8 Hz, 2H), 7.44 (d, <i>J</i> = 6.0 Hz, 1H), 7.41 (d, <i>J</i> = 7.9 Hz, 2H), 7.33 (t, <i>J</i> = 7.7 Hz, 2H), 7.22 (t, <i>J</i> = 7.7 Hz, 1H), 6.93 (d, <i>J</i> = 8.3 Hz, 1H), 5.19 (t, <i>J</i> = 7.5 Hz, 1H), 3.28 (d, <i>J</i> = 7.0 Hz, 2H), 2.44 (d, <i>J</i> = 6.7 Hz, 6H), 1.51 (d, <i>J</i> = 7.1 Hz, 3H), 0.97 (t, <i>J</i> = 7.1 Hz, 7H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.90, 165.95, 164.71, 145.11, 140.64, 136.48, 129.93, 128.30, 127.78, 126.64, 126.37, 126.18, 125.86, 125.28, 124.09, 120.76, 117.69, 114.37, 108.94, 51.72, 48.55, 46.59, 45.44, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 37.06, 22.35, 13.40, 11.92, 10.79.</div></div><div id="sec4_7_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>S</i>,<i>Z</i>)-3-((4-((2-(Diethylamino)ethyl)carbamoyl)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>5b</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 91.2 mg (0.325 mmol) of (<b>3a</b>) and 79.9 mg (0.299 mmol) of (<b>4a</b>), all of which were dissolved in abs. EtOH (2.5 mL). To this solution were added two drops of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then the product was filtered off as an orange solid. Yield: orange solid, 38.3 mg, 22%. Molecular formula: C<sub>31</sub>H<sub>37</sub>N<sub>5</sub>O<sub>3</sub> ESI-MS calc: 527.29 ESI MS found: 528.2043 HPLC: 5.753. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.59 (s, 1H), 11.16 (s, 1H), 8.67 (s, 1H), 8.32 (d, <i>J</i> = 5.5 Hz, 1H), 7.75–7.68 (m, 2H), 7.48 (s, 1H), 7.42 (d, <i>J</i> = 7.7 Hz, 3H), 7.32 (t, <i>J</i> = 7.6 Hz, 3H), 7.22 (t, <i>J</i> = 7.3 Hz, 1H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 5.19 (p, <i>J</i> = 7.2 Hz, 1H), 2.45 (d, <i>J</i> = 3.4 Hz, 8H), 2.37 (s, 1H), 2.32 (s, 1H), 2.08 (s, 1H), 1.51 (d, <i>J</i> = 7.1 Hz, 4H), 0.99 (t, <i>J</i> = 7.5 Hz, 8H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 170.28, 147.82, 141.01, 130.32, 128.66, 127.23, 126.98, 126.61, 126.26, 125.66, 124.60, 118.30, 106.84, 102.36, 48.92, 47.02, 22.80, 17.46, 13.83, 11.24, 9.18.</div></div><div id="sec4_7_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-(prop-2-yn-1-ylcarbamoyl)-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>5c</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 75.1 mg (0.27 mmol) of (<b>3</b>) and 50 mg (0.28 mmol) of (<b>4b</b>), all of which were dissolved in abs. EtOH (1.8 mL). To this solution were added 0.05 mL of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and an orange solid was filtered off. The solid was rinsed with cold EtOH to give the final product. Yield: bright orange solid, 58.6 mg, 46%. Molecular formula: C<sub>28</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 466.20 ESI-MS found: 467.2037 HPLC: 6.360. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.61 (s, 1H), 11.15 (s, 1H), 8.61 (d, <i>J</i> = 8.0 Hz, 1H), 8.26 (s, 1H), 8.06 (t, <i>J</i> = 5.5 Hz, 1H), 7.71 (d, <i>J</i> = 9.0 Hz, 2H), 7.41 (d, <i>J</i> = 7.8 Hz, 2H), 7.33 (q, <i>J</i> = 5.8, 3.9 Hz, 2H), 7.23 (t, <i>J</i> = 7.2 Hz, 1H), 6.94 (d, <i>J</i> = 8.2 Hz, 1H), 5.20 (t, <i>J</i> = 7.7 Hz, 1H), 4.02 (d, <i>J</i> = 5.5 Hz, 2H), 3.14–3.08 (m, 1H), 2.44 (d, <i>J</i> = 9.2 Hz, 7H), 1.51 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.30, 166.26, 145.52, 141.06, 136.92, 130.46, 128.67, 128.29, 127.00, 126.82, 126.58, 126.28, 124.49, 120.45, 118.23, 115.03, 109.28, 82.17, 73.04, 48.87, 28.49, 22.77, 13.76, 11.13.</div></div><div id="sec4_7_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>R</i>,<i>Z</i>)-3-((4-(Allylcarbamoyl)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>5d</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 53.6 mg (0.27 mmol) of (<b>3</b>) and 50 mg (0.28 mmol) of (<b>4c</b>), all of which were dissolved in abs. EtOH (3 mL). To this solution were added two drops (0.05 mL) of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then an orange solid was filtered off. The solid was washed with cold EtOH to give the final product. Result: orange solid, 79.5 mg, 63%. Molecular formula: C<sub>28</sub>H<sub>28</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 468.22 ESI-MS found: 469.2224 HPLC: 6.450. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.60 (s, 1H), 11.14 (s, 1H), 8.60 (d, <i>J</i> = 8.0 Hz, 1H), 8.25 (d, <i>J</i> = 1.6 Hz, 1H), 7.83 (t, <i>J</i> = 5.8 Hz, 1H), 7.73–7.68 (m, 2H), 7.41 (d, <i>J</i> = 7.6 Hz, 2H), 7.33 (t, <i>J</i> = 7.6 Hz, 2H), 7.22 (t, <i>J</i> = 7.3 Hz, 1H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 5.90 (ddt, <i>J</i> = 15.7, 10.3, 5.2 Hz, 1H), 5.22–5.18 (m, 2H), 5.10 (dd, <i>J</i> = 10.3, 1.8 Hz, 1H), 3.87 (t, <i>J</i> = 5.6 Hz, 2H), 2.44 (d, <i>J</i> = 7.8 Hz, 6H), 1.51 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.50, 145.29, 135.97, 130.84, 129.65, 128.44, 128.02, 126.34, 115.20, 108.42, 103.80, 91.16, 59.93, 22.53, 13.55, 10.93.</div></div><div id="sec4_7_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>R</i>,<i>Z</i>)-3-((4-(But-3-yn-1-ylcarbamoyl)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>5e</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 99.9 mg (0.357 mmol) of (<b>3</b>) and 82 mg of (<b>4d</b>), all of which were dissolved in abs. EtOH (3.5 mL). To this solution were added two drops (0.05 mL) of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the solution was cooled to room temperature and then an orange solid was filtered off. The solid was washed with cold EtOH to give the final product. Yield: orange solid, 19 mg, 10%. Molecular formula: C<sub>29</sub>H<sub>28</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 480.22 ESI-MS found: 481.2226 HPLC: 6.444. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.60 (s, 1H), 11.15 (s, 1H), 8.62 (d, <i>J</i> = 8.0 Hz, 1H), 8.25 (s, 1H), 7.79 (t, <i>J</i> = 5.8 Hz, 1H), 7.71 (d, <i>J</i> = 6.3 Hz, 2H), 7.41 (d, <i>J</i> = 7.6 Hz, 2H), 7.33 (t, <i>J</i> = 7.6 Hz, 2H), 7.23 (t, <i>J</i> = 7.3 Hz, 1H), 6.94 (d, <i>J</i> = 8.2 Hz, 1H), 5.19 (p, <i>J</i> = 7.3 Hz, 1H), 2.86 (t, <i>J</i> = 2.5 Hz, 1H), 2.45 (d, <i>J</i> = 12.0 Hz, 6H), 2.42 (td, <i>J</i> = 7.1, 2.4 Hz, 2H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.83, 165.81, 164.71, 145.08, 140.56, 136.34, 130.00, 128.23, 127.80, 126.55, 126.12, 125.79, 125.18, 124.08, 120.61, 117.76, 82.50, 72.12, 48.42, 37.95, 22.33, 13.33, 10.70.</div></div><div id="sec4_7_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>R</i>,<i>Z</i>)-3-((4-(But-3-en-1-ylcarbamoyl)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>5f</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 94.8 mg (0.27 mmol) of (<b>3</b>) and 87.2 mg (0.28 mmol) of (<b>4e</b>), all of which were dissolved in abs. EtOH (2.2 mL). To this solution were added two drops (0.07 mL) of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the solution was cooled to room temperature and then purified by column chromatography. Fractions were collected, and the solvent was removed under pressure to give the desired product as an orange solid. Yield: orange solid, 103 mg, 59%. Molecular formula: C<sub>29</sub>H<sub>30</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 482.23 ESI-MS found: 483.2382 [M + 1] HPLC: 6.720. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.54 (s, 1H), 11.13 (s, 1H), 8.61 (d, <i>J</i> = 8.2 Hz, 2H), 8.27–8.18 (m, 2H), 7.73–7.71 (m, 1H), 7.68 (s, 1H), 7.42 (d, <i>J</i> = 8.3 Hz, 3H), 7.35–7.31 (m, 4H), 7.25–7.21 (m, 2H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 5.21 (p, <i>J</i> = 7.3 Hz, 2H), 2.43 (d, <i>J</i> = 9.0 Hz, 2H), 2.29 (d, <i>J</i> = 11.4 Hz, 7H), 1.63–1.59 (m, 3H), 1.51 (d, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.79, 165.81, 165.04, 145.07, 140.57, 133.90, 128.74, 128.19, 127.78, 126.52, 126.11, 125.22, 123.96, 120.71, 117.70, 116.21, 114.09, 108.74, 48.41, 35.77, 33.74, 22.26, 12.40, 10.15.</div></div><div id="sec4_7_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>7a</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 91.2 mg (0.892 mmol) of (<b>3</b>) and 79.9 mg (1.39 mmol) of 3,5-dimethyl-4-nitro-1<i>H</i>-pyrrole-2-carbaldehyde, all of which were dissolved in abs. EtOH (2.5 mL). To this solution were added two drops (0.05 mL) of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and an orange solid was filtered off. The solid was washed with cold EtOH to give the desired product. Result: orange solid, 269.3 mg, 69%. Molecular formula: C<sub>24</sub>H<sub>22</sub>N<sub>4</sub>O<sub>4</sub> ESI-MS calc: 430.16 ESI-MS found: 431.1715 HPLC: 7.400. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.63 (d, <i>J</i> = 8.0 Hz, 1H), 8.36 (d, <i>J</i> = 1.6 Hz, 1H), 7.83 (s, 1H), 7.76 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.41 (d, <i>J</i> = 7.5 Hz, 2H), 7.33 (t, <i>J</i> = 7.6 Hz, 2H), 7.22 (t, <i>J</i> = 7.3 Hz, 1H), 6.97 (d, <i>J</i> = 8.1 Hz, 1H), 5.19 (p, <i>J</i> = 7.2 Hz, 1H), 2.08 (d, <i>J</i> = 1.0 Hz, 1H), 1.50 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.90, 165.69, 144.98, 142.47, 136.80, 132.12, 130.48, 129.11, 128.20, 127.74, 126.10, 125.56, 123.33, 121.76, 115.38, 53.66, 22.28, 18.23, 10.50.</div></div><div id="sec4_7_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(<i>m</i>-tolyl)ethyl)indoline-5-carboxamide (<b>7b</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 200 mg (0.679 mmol) of (<b>3b</b>) and 139.6 mg (0.815 mmol) of 3,5-dimethyl-4-nitro-1<i>H</i>-pyrrole-2-carbaldehyde, all of which were dissolved in abs. EtOH (6.0 mL). To this solution were added 0.05 mL of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then the product was filtered off as a lemon yellow solid. Yield: lemon yellow solid, 100 mg, 29.8%. Molecular formula: C<sub>25</sub>H<sub>24</sub>N<sub>4</sub>O<sub>4</sub> ESI-MS calc: 444.18 ESI-MS found: 445.1862 [M + 1] HPLC: 7.732. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.34 (s, 1H), 8.58 (d, <i>J</i> = 8.0 Hz, 1H), 8.33 (d, <i>J</i> = 1.6 Hz, 1H), 7.78 (s, 1H), 7.76 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 7.23–7.18 (m, 3H), 7.03 (d, <i>J</i> = 6.1 Hz, 1H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 5.16 (p, <i>J</i> = 7.2 Hz, 1H), 2.62 (s, 3H), 2.57 (s, 3H), 2.30 (s, 3H), 1.49 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.83, 165.49, 144.90, 141.26, 137.14, 136.75, 133.66, 128.34, 128.09, 127.51, 127.15, 126.74, 125.07, 124.71, 124.39, 123.59, 123.17, 119.39, 118.96, 109.09, 48.36, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 22.26, 21.11, 14.76, 11.21.</div></div><div id="sec4_7_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(1-(4-fluorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>7c</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 128 mg (0.429 mmol) of (<b>3c</b>) and 101.3 mg (0.60 mmol) of 3,5-dimethyl-4-nitro-1<i>H</i>-pyrrole-2-carbaldehyde, all of which were dissolved in abs. EtOH (3.0 mL). To this solution were added 0.05 mL of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then the product was filtered off as an orange solid. Yield: orange solid, 96.3 mg, 50%. Molecular formula: C<sub>24</sub>H<sub>21</sub>FN<sub>4</sub>O<sub>4</sub> ESI-MS calc: 448.15 ESI-MS found: 449.1619 [M + 1] HPLC: 7.422. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.37 (s, 1H), 8.64 (d, <i>J</i> = 7.8 Hz, 1H), 8.35 (d, <i>J</i> = 1.7 Hz, 1H), 7.83 (s, 1H), 7.76 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.47–7.44 (m, 2H), 7.18–7.14 (m, 2H), 6.97 (d, <i>J</i> = 8.1 Hz, 1H), 5.19 (p, <i>J</i> = 7.2 Hz, 1H), 2.65 (s, 3H), 2.60 (s, 3H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.85, 165.62, 136.79, 133.69, 128.31, 128.02, 127.97, 124.71, 123.71, 119.03, 114.88, 114.76, 50.22, 47.86, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 22.26, 14.78, 11.22.</div></div><div id="sec4_7_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(pyridin-4-yl)ethyl)indoline-5-carboxamide (<b>7d</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 100 mg (0.455 mmol) of (<b>3d</b>) and 77.1 mg (0.459 mmol) of 3,5-dimethyl-4-nitro-1<i>H</i>-pyrrole-2-carbaldehyde, all of which were dissolved in abs. EtOH (3.0 mL). To this solution were added 0.05 mL of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then the product was filtered off as a yellow solid. Yield: yellow solid, 48.0 mg, 31.3%. Molecular formula: C<sub>23</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub> ESI-MS calc: 431.16 ESI-MS found: 432.1656 [M + 1] HPLC: 5.496. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.36 (s, 1H), 8.73 (d, <i>J</i> = 7.6 Hz, 1H), 8.52–8.50 (m, 2H), 8.36 (d, <i>J</i> = 1.6 Hz, 1H), 7.82 (s, 1H), 7.77 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 7.40 (d, <i>J</i> = 5.2 Hz, 2H), 6.97 (d, <i>J</i> = 8.0 Hz, 1H), 5.15 (p, <i>J</i> = 7.3 Hz, 1H), 2.64 (s, 3H), 2.59 (s, 3H), 1.51 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.11, 166.24, 153.97, 149.80, 137.07, 135.63, 128.31, 127.80, 125.45, 124.98, 124.02, 121.55, 119.37, 109.42, 48.18, 40.12, 40.00, 39.88, 39.76, 39.64, 39.52, 39.40, 21.78, 15.04, 11.48.</div></div><div id="sec4_7_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(pyridin-2-yl)ethyl)indoline-5-carboxamide (<b>7e</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 165 mg (0.587 mmol) of (<b>3e</b>) and 122.8 mg (0.762 mmol) of 3,5-dimethyl-4-nitro-1<i>H</i>-pyrrole-2-carbaldehyde, all of which were dissolved in abs. EtOH (3.0 mL). To this solution were added 0.05 mL of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then the product was filtered off as a yellow solid. Yield: yellow solid, 116 mg, 34%. Molecular formula: C<sub>23</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub> ESI-MS calc: 431.16 ESI-MS found: 432.0558 [M + 1] HPLC: 5.49. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.36 (d, <i>J</i> = 7.2 Hz, 1H), 8.65–8.58 (m, 1H), 8.55–8.47 (m, 1H), 8.39 (t, <i>J</i> = 7.7 Hz, 1H), 7.78 (ddd, <i>J</i> = 33.0, 15.8, 7.2 Hz, 3H), 7.42 (q, <i>J</i> = 10.5, 9.0 Hz, 1H), 7.25 (dd, <i>J</i> = 12.7, 6.8 Hz, 1H), 6.95 (d, <i>J</i> = 7.0 Hz, 1H), 5.22 (q, <i>J</i> = 8.8, 8.0 Hz, 1H), 2.66–2.60 (m, 3H), 2.59 (s, 3H), 1.56–1.49 (m, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.21, 165.90, 148.79, 136.83, 128.23, 124.86, 123.81, 120.44, 119.09, 109.32, 58.91, 50.44, 40.00, 39.88, 39.76, 39.74, 39.64, 39.62, 39.52, 39.40, 39.28, 21.09, 11.38.</div></div><div id="sec4_7_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (<i>R</i>,<i>Z</i>)-<i>N</i>-(1-(3-Chlorophenyl)ethyl)-3-((3,5-dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxoindoline-5-carboxamide (<b>7f</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 242.2 mg (0.769 mmol) of (<b>3f</b>) and 150.1 mg (0.892 mmol) of 3,5-dimethyl-4-nitro-1<i>H</i>-pyrrole-2-carbaldehyde, all of which were dissolved in abs. EtOH (6.0 mL). To this solution were added 0.05 mL of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then the product was filtered off as an orange solid. Yield: orange solid, 46 mg, 16%. Molecular formula: C<sub>24</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>4</sub> ESI-MS calc: 464.13 ESI-MS found: 465.1318 HPLC: 7.699. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.38 (s, 1H), 8.68 (d, <i>J</i> = 7.8 Hz, 1H), 8.36 (d, <i>J</i> = 1.6 Hz, 1H), 7.84 (s, 1H), 7.76 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.46 (d, <i>J</i> = 2.1 Hz, 1H), 7.37 (d, <i>J</i> = 6.6 Hz, 2H), 7.29 (dt, <i>J</i> = 6.7, 2.3 Hz, 1H), 6.97 (d, <i>J</i> = 8.1 Hz, 1H), 5.17 (p, <i>J</i> = 7.2 Hz, 1H), 2.65 (s, 3H), 2.60 (s, 3H), 1.50 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 165.88, 147.85, 144.31, 138.20, 130.27, 126.64, 126.10, 125.34, 125.04, 123.98, 119.28, 109.31, 102.54, 48.41, 40.00, 39.88, 39.76, 39.64, 39.52, 39.40, 39.28, 22.27, 14.93.</div></div><div id="sec4_7_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(<i>p</i>-tolyl)ethyl)indoline-5-carboxamide (<b>7g</b>)</h4><div class="NLM_p last">To a dried sealed tube were added 253.9 mg (0.862 mmol) of (<b>3g</b>) and 140.9 mg (0.837 mmol) of 3,5-dimethyl-4-nitro-1<i>H</i>-pyrrole-2-carbaldehyde, all of which were dissolved in abs. EtOH (3.0 mL). To this solution was added 0.05 mL of piperidine and heated to reflux (95 °C) for 4 h. Once complete, the reaction was cooled to room temperature and then the product was filtered off as an orange solid. Yield: orange solid, 76.1 mg, 25%. Molecular formula: C<sub>25</sub>H<sub>24</sub>N<sub>4</sub>O<sub>4</sub> ESI-MS calc: 444.18 ESI-MS found: 445.1864 [M + 1] HPLC: 7.662. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.73 (s, 1H), 8.57 (d, <i>J</i> = 8.2 Hz, 2H), 8.35 (d, <i>J</i> = 1.7 Hz, 1H), 7.83 (s, 1H), 7.76 (dd, <i>J</i> = 8.1, 1.7 Hz, 2H), 7.30 (d, <i>J</i> = 7.8 Hz, 3H), 7.14 (d, <i>J</i> = 7.9 Hz, 3H), 6.96 (d, <i>J</i> = 8.0 Hz, 2H), 5.17 (p, <i>J</i> = 7.2 Hz, 1H), 2.55 (d, <i>J</i> = 8.7 Hz, 6H), 2.28 (s, 5H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.11, 165.77, 155.70, 142.17, 141.51, 137.04, 135.76, 135.76, 128.94, 126.27, 125.36, 119.23, 109.39, 52.91, 40.12, 40.00, 39.88, 39.76, 39.64, 39.52, 39.40, 22.49, 20.86, 15.04, 11.49, 10.19.</div></div><div id="sec4_7_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (<i>Z</i>)-<i>N</i>-Benzyl-3-((3,5-dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxoindoline-5-carboxamide (<b>7i</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>7a</b>. Yields an orange solid, 150 mg, 55%. Molecular formula: C<sub>23</sub>H<sub>20</sub>N<sub>4</sub>O<sub>4</sub> ESI-MS calc: 416.15 ESI-MS found: 417.2934, HPLC: 7.158. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.44–11.06 (m, 1H), 8.84 (t, <i>J</i> = 6.1 Hz, 1H), 8.34 (s, 1H), 7.76 (d, <i>J</i> = 8.3 Hz, 1H), 7.69 (s, 1H), 7.34 (d, <i>J</i> = 5.6 Hz, 4H), 7.25 (d, <i>J</i> = 6.6 Hz, 1H), 6.92 (d, <i>J</i> = 8.2 Hz, 1H), 4.52 (d, <i>J</i> = 5.7 Hz, 2H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.75, 166.10, 141.33, 139.82, 136.69, 133.57, 128.23, 127.89, 127.37, 127.18, 126.67, 124.95, 124.69, 124.44, 123.25, 119.29, 118.57, 109.16, 42.62, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 14.73, 11.11.</div></div><div id="sec4_7_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(2-phenylpropan-2-yl)indoline-5-carboxamide (<b>7j</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>7a</b>. Yields an orange solid, 76.1 mg, 32.1%. Molecular formula: C<sub>25</sub>H<sub>24</sub>N<sub>4</sub>O<sub>4</sub> ESI-MS calc: 444.18 ESI-MS found: 445.2680 [M + 1] HPLC: 7.713. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.32 (s, 1H), 8.31 (d, <i>J</i> = 1.7 Hz, 1H), 8.25 (s, 1H), 7.80 (s, 1H), 7.72 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.42–7.39 (m, 2H), 7.28 (t, <i>J</i> = 7.8 Hz, 2H), 7.17 (t, <i>J</i> = 7.3 Hz, 1H), 6.92 (d, <i>J</i> = 8.1 Hz, 1H), 2.61 (s, 3H), 2.57 (s, 3H), 1.70 (s, 6H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.84, 165.70, 148.16, 141.11, 136.71, 133.63, 129.19, 127.83, 127.58, 125.62, 125.05, 124.73, 124.69, 124.29, 123.61, 119.44, 119.02, 108.96, 55.31, 39.86, 39.74, 39.62, 39.50, 39.38, 39.26, 39.14, 29.65, 14.76, 11.21.</div></div><div id="sec4_7_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(2-methyl-1-phenylpropyl)-2-oxoindoline-5-carboxamide (<b>7k</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>7a</b>. Yields an orange solid, 144.6 mg, 64.8%. Molecular formula: C<sub>26</sub>H<sub>26</sub>N<sub>4</sub>O<sub>4</sub> ESI-MS calc: 458.20 ESI-MS found: 459.2109 HPLC: 7.896. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.32 (s, 1H), 8.56 (p, <i>J</i> = 9.3, 8.3 Hz, 1H), 8.29 (dq, <i>J</i> = 13.4, 7.5 Hz, 1H), 7.88–7.63 (m, 2H), 7.48–7.28 (m, 4H), 7.22 (qd, <i>J</i> = 14.5, 9.4, 7.5 Hz, 1H), 6.94 (tt, <i>J</i> = 14.2, 7.5 Hz, 1H), 4.79–4.64 (m, 1H), 2.64–2.58 (m, 3H), 2.58 (s, 3H), 1.05 (tt, <i>J</i> = 13.4, 8.0 Hz, 3H), 0.75 (dp, <i>J</i> = 25.8, 6.7 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.69, 165.86, 143.12, 141.05, 127.89, 127.29, 127.17, 126.47, 125.00, 124.61, 124.32, 123.62, 119.24, 119.05, 108.85, 59.80, 39.74, 39.62, 39.50, 39.38, 39.26, 39.15, 39.03, 32.40, 19.97, 19.82, 14.64, 11.05.</div></div><div id="sec4_7_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (<i>Z</i>)-3-((3,5-Dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(3-phenyloxetan-3-yl)indoline-5-carboxamide (<b>7l</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>7a</b>. Yields an orange solid, 47 mg, 42%. Molecular formula: C<sub>25</sub>H<sub>22</sub>N<sub>4</sub>O<sub>5</sub> ESI-MS calc: 458.16 ESI-MS found: 459.1656 HPLC: 6.923. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.39 (s, 1H), 9.36 (s, 1H), 8.40 (d, <i>J</i> = 2.0 Hz, 1H), 7.83 (d, <i>J</i> = 2.2 Hz, 1H), 7.81–7.77 (m, 1H), 7.59–7.55 (m, 2H), 7.40 (t, <i>J</i> = 7.8 Hz, 2H), 7.29 (t, <i>J</i> = 7.4 Hz, 1H), 6.99 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 5.04 (d, <i>J</i> = 6.7 Hz, 2H), 4.81 (d, <i>J</i> = 6.7 Hz, 2H), 2.63 (d, <i>J</i> = 2.0 Hz, 3H), 2.58 (d, <i>J</i> = 2.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.85, 165.50, 142.99, 141.58, 136.84, 133.69, 127.65, 127.44, 126.92, 125.26, 124.89, 124.72, 124.58, 123.78, 119.24, 109.26, 81.88, 58.33, 14.77, 11.15.</div></div><div id="sec4_7_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>R</i>,<i>Z</i>)-<i>N</i>-(1-(4-Chlorophenyl)ethyl)-3-((3,5-dimethyl-4-nitro-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxoindoline-5-carboxamide (<b>7k</b>)</h4><div class="NLM_p last">Prepared using the protocol described for <b>7a</b>. Yields an orange solid, 213.3 mg, 96%. Molecular formula: C<sub>24</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>4</sub> ESI-MS calc: 464.13 ESI-MS found: 465.1318 [M + 1] HPLC: 7.795. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.34 (s, 1H), 8.65 (d, <i>J</i> = 7.8 Hz, 1H), 8.35–8.30 (m, 1H), 7.79 (s, 1H), 7.75 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.43 (d, <i>J</i> = 8.3 Hz, 2H), 7.39 (d, <i>J</i> = 8.5 Hz, 3H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 5.17 (p, <i>J</i> = 7.2 Hz, 1H), 2.62 (s, 3H), 2.57 (s, 3H), 1.49 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 169.83, 165.67, 144.05, 141.32, 136.76, 133.66, 131.02, 128.18, 128.10, 128.00, 127.49, 125.11, 124.70, 124.41, 123.63, 119.35, 118.98, 109.10, 48.00, 40.00, 39.83, 39.67, 39.50, 39.34, 39.17, 39.00, 22.07, 14.76, 11.21.</div></div><div id="sec4_7_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>8a</b>)</h4><div class="NLM_p last">To a flask were added 90.7 mg of (<b>7a</b>) and 5 mL of 2:1 EtOH/EtOAc. To this slurry were added 226.8 mg (14 equiv) of Zn powder and 2 mL (150 equiv) of AcOH. The turbid orange solution was allowed to stir at 50 °C for 2 h. Once complete, the reaction was cooled to room temperature and then EtOAc was added before basifying with sat. Na<sub>2</sub>CO<sub>3</sub>. The basified aqueous layer was extracted with EtOAc (3 × 30 mL) and then washed with water and brine (1 × 30 mL). The organic layer was dried over MgSO<sub>4</sub>, and then solvent was removed under pressure to give the desired product as a red solid. Because the free amine is very reactive, it was moved forward without further characterization. Result: orange solid, 66.7 mg, 79.0%. Molecular formula: C<sub>24</sub>H<sub>24</sub>N<sub>4</sub>O<sub>2</sub> ESI-MS calc: 400.19 ESI-MS found: 401.1955 HPLC: 5.177. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.45 (s, 1H), 10.88 (s, 1H), 8.56 (d, <i>J</i> = 8.0 Hz, 1H), 8.12 (d, <i>J</i> = 1.6 Hz, 1H), 7.62 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 7.46 (s, 1H), 7.41 (d, <i>J</i> = 7.6 Hz, 3H), 7.33 (t, <i>J</i> = 7.6 Hz, 3H), 7.22 (t, <i>J</i> = 7.3 Hz, 1H), 6.89 (d, <i>J</i> = 8.1 Hz, 1H), 5.20 (q, <i>J</i> = 7.4 Hz, 1H), 4.02 (d, <i>J</i> = 14.5 Hz, 2H), 2.26 (s, 3H), 2.17 (s, 3H), 1.51 (d, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_7_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(<i>m</i>-tolyl)ethyl)indoline-5-carboxamide (<b>8b</b>)</h4><div class="NLM_p last">To a flask were added 86.4 mg of (<b>7b</b>) and 5 mL of 2:1 EtOH/EtOAc. To this slurry were added 210.9 mg (14 equiv) of Zn powder and 2 mL (150 equiv) of AcOH. The turbid orange solution was allowed to stir at 50 °C for 2 h. Once complete, the reaction was cooled to room temperature and then EtOAc was added before basifying with sat. Na<sub>2</sub>CO<sub>3</sub>. The basified aqueous layer was extracted with EtOAc (3 × 30 mL) and then washed with water and brine (1 × 30 mL). The organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed under pressure to give the desired product as a red solid. Note that the free amine is very reactive, so it was moved forward without further characterization. Result: red solid, 86.6 mg, 94%. Molecular formula: C<sub>25</sub>H<sub>26</sub>N<sub>4</sub>O<sub>2</sub> ESI-MS calc: 414.21 ESI-MS found: 415.2120 [M + 1] HPLC: 5.627.</div></div><div id="sec4_7_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(1-(4-fluorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>8c</b>)</h4><div class="NLM_p last">To a flask were added 96.3 mg of (<b>7c</b>) and 5 mL of 2:1 EtOH/EtOAc. To this slurry were added 295 mg (14 equiv) of Zn powder and 2 mL (150 equiv) of AcOH. The turbid orange solution was allowed to stir at 50 °C for 2 h. Once complete, the reaction was cooled to room temperature and then EtOAc was added before basifying with sat. Na<sub>2</sub>CO<sub>3</sub>. The basified aqueous layer was extracted with EtOAc (3 × 30 mL) and then washed with water and brine (1 x 30 mL). The organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed under pressure to give the desired product as a red solid. The free amine is very reactive, so it was moved forward without further characterization. Result: red solid, 96 mg, 100%. Molecular formula: C<sub>24</sub>H<sub>23</sub>FN<sub>4</sub>O<sub>2</sub> ESI-MS calc: 418.18 ESI-MS found: 419.1938 [M + 1] HPLC: 5.713.</div></div><div id="sec4_7_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(pyridin-4-yl)ethyl)indoline-5-carboxamide (<b>8d</b>)</h4><div class="NLM_p last">To a flask were added 48.0 mg of (<b>7d</b>) and 5 mL of 2:1 EtOH/EtOAc. To this slurry were added 208 mg (14 equiv) of Zn powder and 2 mL (150 equiv) of AcOH. The turbid orange solution was allowed to stir at 50 °C for 2 h. Once complete, the reaction was cooled to room temperature and then EtOAc was added before basifying with sat. Na<sub>2</sub>CO<sub>3</sub>. The basified aqueous layer was extracted with EtOAc (3 × 30 mL) and then washed with water and brine (1 ×30 mL). The organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed under pressure to give the desired product as a red solid. The free amine is very reactive, so it was moved forward without further characterization. Result: red solid, 41.5 mg, 89%. Molecular formula: C<sub>23</sub>H<sub>23</sub>N<sub>5</sub>O<sub>2</sub> ESI-MS calc: 401.19 ESI-MS found: 402.1920 [M + 1] HPLC: 3.928.</div></div><div id="sec4_7_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(pyridin-2-yl)ethyl)indoline-5-carboxamide (<b>8e</b>)</h4><div class="NLM_p last">To a flask were added 73.9 mg of (<b>7e</b>) and 5 mL of 2:1 EtOH/EtOAc. To this slurry were added 289 mg (14 equiv) of Zn powder and 2 mL (150 equiv) of AcOH. The turbid orange solution was allowed to stir at 50 °C for 2 h. Once complete, the reaction was cooled to room temperature and then EtOAc was added before basifying with sat. Na<sub>2</sub>CO<sub>3</sub>. The basified aqueous layer was extracted with EtOAc (3 × 30 mL) and then washed with water and brine (1 × 30 mL). The organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed under pressure to give the desired product as a red solid. The free amine is very reactive, so it was moved forward without further characterization. Result: red solid, 73.8 mg, 100%. Molecular formula: C<sub>23</sub>H<sub>23</sub>N<sub>5</sub>O<sub>2</sub> ESI-MS calc: 401.19 ESI-MS found: 402.1450 [M + 1] HPLC: 3.920.</div></div><div id="sec4_7_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(1-(3-chlorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>8f</b>)</h4><div class="NLM_p last">To a flask were added 46.0 mg of (<b>7f</b>) and 5 mL of 2:1 EtOH/EtOAc. To this slurry were added 170.1 mg (14 equiv) of Zn powder and 1.4 mL (150 equiv) of AcOH. The turbid orange solution was allowed to stir at 50 °C for 2 h. Once complete, the reaction was cooled to room temperature and then EtOAc was added before basifying with sat. Na<sub>2</sub>CO<sub>3</sub>. The basified aqueous layer was extracted with EtOAc (3 × 30 mL) and then washed with water and brine (1 × 30 mL). The organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed under pressure to give the desired product as a red solid. The free amine is very reactive, so it was moved forward without further characterization. Result: red solid, 40 mg, 95%. Molecular formula: C<sub>24</sub>H<sub>23</sub>ClN<sub>4</sub>O<sub>2</sub> ESI-MS calc: 434.15 ESI-MS found: 435.1569 [M + 1] HPLC: 5.556.</div></div><div id="sec4_7_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(<i>p</i>-tolyl)ethyl)indoline-5-carboxamide (<b>8g</b>)</h4><div class="NLM_p last">To a flask were added 76.1 mg of (<b>7g</b>) and 5 mL of 2:1 EtOH/EtOAc. To this slurry were added 314.4 mg (14 equiv) of Zn powder and 2.0 mL (150 equiv) of AcOH. The turbid orange solution was allowed to stir at 50 °C for 2 h. Once complete, the reaction was cooled to room temperature and then EtOAc was added before basifying with sat. Na<sub>2</sub>CO<sub>3</sub>. The basified aqueous layer was extracted with EtOAc (3 × 30 mL) and then washed with water and brine (1 × 30 mL). The organic layer was dried over MgSO<sub>4</sub>, and then the solvent was removed under pressure to give the desired product as a red solid. The free amine is very reactive, so it was moved forward without further characterization. Result: red solid, 220 mg, 100%. Molecular formula: C<sub>25</sub>H<sub>26</sub>N<sub>4</sub>O<sub>2</sub> ESI-MS calc: 414.21 ESI-MS found: 415.2119 [M + 1] HPLC: 5.573.</div></div><div id="sec4_7_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(1-(4-chlorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>8h</b>)</h4><div class="NLM_p last">Synthesized using protocol described for <b>8a</b>. Yields a red solid, 36.1 mg, 48%. Molecular formula: C<sub>24</sub>H<sub>23</sub>ClN<sub>4</sub>O<sub>2</sub> ESI-MS calc: 434.15 ESI-MS found: 435.15612 HPLC: 5.398.</div></div><div id="sec4_7_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> (<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-benzyl-2-oxoindoline-5-carboxamide (<b>8i</b>)</h4><div class="NLM_p last">Synthesized using protocol described for <b>8a</b>. Yields a red solid, 210 mg, quantitative yield. Molecular formula: C<sub>23</sub>H<sub>22</sub>N<sub>4</sub>O<sub>2</sub> ESI-MS calc: 386.17 ESI-MS found: 387.1810 HPLC: 5.129.</div></div><div id="sec4_7_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(2-phenylpropan-2-yl)indoline-5-carboxamide (<b>8j</b>)</h4><div class="NLM_p last">Synthesized using protocol described for <b>8a</b>. Yields a red solid, 307 mg, quantitative yield. Molecular formula: C<sub>25</sub>H<sub>26</sub>N<sub>4</sub>O<sub>2</sub> ESI-MS calc: 414.21 ESI-MS found: 415.2119 [M + 1] HPLC: 5.245.</div></div><div id="sec4_7_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (<i>R</i>,<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(2-methyl-1-phenylpropyl)-2-oxoindoline-5-carboxamide (<b>8k</b>)</h4><div class="NLM_p last">Synthesized using protocol described for <b>8a</b>. Yields a red solid, 220 mg, quantitative yield. Molecular formula: C<sub>26</sub>H<sub>28</sub>N<sub>4</sub>O<sub>2</sub> ESI-MS calc: 428.22 ESI-MS found: 429.2201 HPLC: 5.497.</div></div><div id="sec4_7_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (<i>Z</i>)-3-((4-Amino-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(3-phenyloxetan-3-yl)indoline-5-carboxamide (<b>8l</b>)</h4><div class="NLM_p last">Synthesized using protocol described for <b>8a</b>. Yields a red solid, 20.6 mg, 47%. Molecular formula: C<sub>25</sub>H<sub>24</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 428.18 ESI-MS found: 429.1494 HPLC: 4.710.</div></div><div id="sec4_7_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (<i>R</i>,<i>Z</i>)-3-((4-(But-2-ynamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>9a</b>)</h4><div class="NLM_p last">To a flask were added 22.3 mg (0.37 mmol) of butynoic acid, 157.5 mg (0.37 mmol) of HATU, cat. DMAP, and 50 mg (0.19 mmol) of <b>8a</b> dissolved in 1 mL of DMF. To this bright red solution was added 0.25 mL (1.3 mmol) of DIPEA, and the solution was allowed to stir at room temperature for 2 h. Once complete, the reaction mixture was diluted with EtOAc, washed with sat. LiCl, and then dried over MgSO<sub>4</sub>. The crude material was purified by preparatory TLC with 50% acetone/hexanes. The desired band was collected, the material was rinsed off the silica gel with acetone, and then the solvent was removed under pressure to give the desired product. Result: yellow solid, 17.5 mg, 20%. Molecular formula: C<sub>28</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 466.20 ESI-MS found: 467.2071 HPLC: 6.336. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.53 (s, 1H), 11.07 (s, 1H), 9.78 (s, 1H), 8.86 (d, <i>J</i> = 4.5 Hz, 2H), 8.68 (d, <i>J</i> = 8.8 Hz, 2H), 8.60 (d, <i>J</i> = 7.7 Hz, 2H), 8.21 (s, 1H), 7.69 (d, <i>J</i> = 8.9 Hz, 2H), 7.65–7.62 (m, 3H), 7.41 (d, <i>J</i> = 7.9 Hz, 5H), 7.33 (t, <i>J</i> = 7.3 Hz, 7H), 7.23 (d, <i>J</i> = 7.0 Hz, 3H), 6.93 (d, <i>J</i> = 8.1 Hz, 1H), 6.59 (s, 3H), 5.21–5.17 (m, 1H), 2.19 (s, 4H), 2.17 (s, 4H), 2.03 (s, 3H), 1.52 (s, 2H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.28, 167.22, 165.79, 149.50, 148.74, 145.19, 139.73, 135.08, 130.24, 128.98, 128.19, 127.41, 126.55, 125.97, 125.02, 122.58, 120.62, 117.41, 112.97, 55.89, 22.67, 12.09, 9.59, 3.60.</div></div><div id="sec4_7_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (<i>R</i>,<i>Z</i>)-3-((4-Acrylamido-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>9b</b>)</h4><div class="NLM_p last">To a round-bottom flask that contained <b>8a</b> (50 mg, 0.12 mmol) were added 0.01 mL (0.12 mmol) of acrylic acid, 58.8 mg (0.12 mmol) of HATU, cat. DMAP, and 0.50 mL (0.87 mmol) of TEA. All materials were dissolved in 2 mL of DMF and sonicated to give a homogeneous solution. The solution was allowed to stir at 56 °C for 2 h. Once complete, the reaction was quenched with sat. LiCl and then extracted with EtOAc (3 × 30 mL). The combined organic layers were then dried over MgSO<sub>4</sub>, and the material was purified by preparatory TLC (50% acetone/hexanes) collecting the baseline product, which was washed off the silica gel with acetone. The solvent was removed to give the final product as an orange solid (13.9 mg, 24%). Molecular formula: C<sub>27</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 454.20 ESI-MS found: 455.1632 [M + 1] HPLC: 6.147. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.53 (s, 1H), 11.07 (s, 1H), 9.41 (s, 1H), 8.60 (d, <i>J</i> = 8.1 Hz, 2H), 8.21 (d, <i>J</i> = 9.5 Hz, 1H), 7.69 (d, <i>J</i> = 8.3 Hz, 2H), 7.66 (s, 1H), 7.41 (d, <i>J</i> = 7.8 Hz, 5H), 7.33 (t, <i>J</i> = 7.5 Hz, 5H), 7.22 (t, <i>J</i> = 7.4 Hz, 3H), 6.93 (dd, <i>J</i> = 8.1, 3.6 Hz, 1H), 6.45 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.23–6.19 (m, 1H), 5.72 (dd, <i>J</i> = 10.7, 1.9 Hz, 1H), 5.19 (t, <i>J</i> = 7.3 Hz, 2H), 2.21 (s, 3H), 2.20 (s, 3H), 1.50 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 180.81, 169.69, 169.00, 167.87, 165.86, 153.99, 145.07, 141.22, 140.27, 136.23, 131.78, 129.29, 128.18, 127.56, 126.86, 126.49, 126.09, 124.16, 120.41, 109.40, 108.64, 53.84, 22.28.</div></div><div id="sec4_7_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (<i>R</i>,<i>Z</i>)-3-((3,5-Dimethyl-4-(vinylsulfonamido)-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>9c</b>)</h4><div class="NLM_p last">Step 1: To a dried flask were added 0.03 mL (0.12 mmol) of vinyl sulfonic acid, two drops of DMF, and 2 mL of DCM. The solution was cooled to 0 °C, and then 0.02 mL (0.14 mmol) of oxalyl chloride was added in one portion. The solution was warmed to room temperature and allowed to stir until complete (1 h). Once complete, then the solvent was removed, and the resultant clear oil was rinsed with DCM (3 × 15 mL) and dried under high pressure until ready for step 2. Step 2: To the flask holding the acid chloride was added <b>8a</b> (50 mg, 0.12 mmol) dissolved in 3 mL of THF. To this murky orange solution was added 0.50 mL (7 equiv) of TEA, and the solution was allowed to stir at room temperature for 5 h. Once complete, the solvent was removed under pressure and a yellow residue was brought back up in EtOAc. The organic layer was washed with sat. Na<sub>2</sub>CO<sub>3</sub> and then brine (1 × 30 mL). It was purified using a preparatory TLC plate (40% acetone/hexanes), collecting secondary spot (Rf = 0.2–0.3). The material was rinsed off silica with acetone, and the solvent was removed to give a dark orange solid as the desired product. Result: orange solid, 16 mg, 27%. Molecular formula: C<sub>26</sub>H<sub>26</sub>N<sub>4</sub>O<sub>4</sub>S ESI-MS calc: 490.17 ESI-MS found: 491.1249 HPLC: 6.43. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.50 (s, 1H), 11.38 (s, 1H), 11.09 (s, 1H), 8.98 (s, 1H), 8.64–8.57 (m, 2H), 8.36 (s, 1H), 8.20 (s, 1H), 7.84 (s, 1H), 7.77 (d, <i>J</i> = 8.3 Hz, 1H), 7.70 (d, <i>J</i> = 8.2 Hz, 1H), 7.63 (s, 1H), 7.41 (d, <i>J</i> = 7.8 Hz, 5H), 7.33 (t, <i>J</i> = 7.6 Hz, 5H), 7.21 (d, <i>J</i> = 7.3 Hz, 2H), 6.98 (s, 1H), 6.93 (d, <i>J</i> = 8.1 Hz, 1H), 6.84 (dd, <i>J</i> = 16.5, 9.8 Hz, 1H), 5.93 (d, <i>J</i> = 8.0 Hz, 1H), 5.19 (t, <i>J</i> = 7.4 Hz, 2H), 2.29 (s, 4H), 2.26 (s, 3H), 1.50 (d, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.90, 165.56, 145.07, 141.33, 136.84, 133.74, 128.32, 128.19, 127.65, 126.53, 126.15, 125.20, 124.79, 124.46, 123.83, 119.49, 119.12, 109.21, 48.52, 45.44, 39.88, 39.76, 39.64, 39.52, 39.40, 39.28, 39.16, 22.33, 14.83, 11.29, 8.63.</div></div><div id="sec4_7_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Cyanoacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>9d</b>)</h4><div class="NLM_p last">To a round-bottom flask were added 20.2 mg of <b>8a</b> (0.05 mmol), 6.7 mg (0.075 mmol) of cyanoacetic acid, and 24.6 mg (0.1 mmol) of DMTMM. All materials were brought up in 2 mL of DMF, and the resulting solution was allowed to stir at room temperature for 12 h. Once complete, the reaction was quenched with sat. NaCl and extracted with EtOAc (3 × 50 mL). The organic layer was washed with sat. Na<sub>2</sub>CO<sub>3</sub> and brine (1 × 50 mL) and then dried over MgSO<sub>4</sub>. The solvent was removed under pressure, evaporating the material onto a silica gel. It was purified by column chromatography (5–100% acetone/hexanes). The column was flushed with 7 N NH<sub>3</sub> in MeOH to yield the final product as a bright orange solid. Result: orange solid, 10 mg, 43%. Molecular formula: C<sub>27</sub>H<sub>25</sub>N<sub>5</sub>O<sub>3</sub> ESI-MS calc: 467.20 ESI-MS found: 468.1297 HPLC: 6.100, <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.51 (s, 1H), 11.08 (s, 1H), 9.52 (s, 1H), 8.60 (d, <i>J</i> = 8.2 Hz, 1H), 8.21 (d, <i>J</i> = 1.7 Hz, 1H), 7.69 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.65 (s, 1H), 7.41 (d, <i>J</i> = 7.7 Hz, 2H), 7.33 (t, <i>J</i> = 7.6 Hz, 3H), 7.22 (t, <i>J</i> = 7.3 Hz, 1H), 6.93 (d, <i>J</i> = 8.1 Hz, 1H), 5.19 (p, <i>J</i> = 7.2 Hz, 1H), 3.88 (s, 2H), 2.20 (dd, <i>J</i> = 12.9, 10.8 Hz, 6H), 1.50 (d, <i>J</i> = 7.1 Hz, H).</div></div><div id="sec4_7_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>9e</b>)</h4><div class="NLM_p last">To a dried round-bottom flask was added <b>8a</b> (60 mg, 0.15 mmol) dissolved in 3 mL of THF. The yellow solution was cooled to 0 °C, and 0.01 mL (0.18 mmol) of chloroacetylchloride was added dropwise. The solution was allowed to stir at 0 °C for 30 min. Once complete by TLC, it was quenched with water and extracted with EtOAc. The solvent was removed, and the crude material was purified using a prep plate (50% acetone/hexanes) collecting the major product. The desired product was rinsed off of a silica gel, and then the solvent was removed to give a bright yellow solid. The yellow solid was washed with DCM and sonicated to give a red solid. The red solid was then pulped in water/acetone (30:1) to give the final product. Result: red solid, 25.1 mg, 35%. Molecular formula: C<sub>26</sub>H<sub>25</sub>ClN<sub>4</sub>O<sub>3</sub> ESI-MS calc: 476.16 ESI-MS found: 477.0999 HPLC: 6.388. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.08 (s, 1H), 9.53 (s, 1H), 8.60 (d, <i>J</i> = 8.0 Hz, 1H), 8.22 (s, 1H), 7.69 (d, <i>J</i> = 8.2 Hz, 1H), 7.65 (s, 1H), 7.41 (d, <i>J</i> = 7.8 Hz, 2H), 7.33 (t, <i>J</i> = 7.5 Hz, 2H), 7.22 (t, <i>J</i> = 7.4 Hz, 1H), 6.93 (d, <i>J</i> = 8.0 Hz, 1H), 5.19 (t, <i>J</i> = 7.5 Hz, −1H), 2.20 (d, <i>J</i> = 12.1 Hz, 7H), 2.08 (s, 3H), 1.50 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.67, 168.51, 156.78, 145.12, 140.30, 131.64, 129.98, 128.13, 127.50, 126.89, 126.44, 126.11, 124.45, 122.62, 121.01, 117.42, 116.93, 112.94, 48.39, 42.71, 22.29, 11.67, 9.24.</div></div><div id="sec4_7_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> (<i>Z</i>)-3-((4-((<i>E</i>)-4-(Dimethylamino)but-2-enamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-((<i>R</i>)-1-phenylethyl)indoline-5-carboxamide (<b>9f</b>)</h4><div class="NLM_p last">To a round-bottom flask were added <b>8a</b> (50 mg, 0.12 mmol), 20.2 mg (0.12 mmol) of <i>N</i>,<i>N</i>-dimethylaminobutenoic acid, 36 mg (0.12 mmol) of DMTMM, and 0.20 mL (0.24 mmol) of TEA. All materials were brought up in 2 mL of DMF, and the resulting solution was allowed to stir at room temperature for 12 h. Once complete, the reaction was quenched with sat. NaCl and extracted with EtOAc (3 × 50 mL). The organic layer was washed with sat. Na<sub>2</sub>CO<sub>3</sub> and brine (1 × 50 mL) and then dried over MgSO<sub>4</sub>. The solvent was removed under pressure, evaporating the material onto a silica gel. It was purified by column chromatography (5–100% acetone/hexanes). The column was flushed with 7 N NH<sub>3</sub> in MeOH to yield the final product as a bright yellow oil. Result: yellow oil, 13 mg, 20%. Molecular formula: C<sub>30</sub>H<sub>33</sub>N<sub>5</sub>O<sub>3</sub> ESI-MS calc: 511.26 ESI-MS found: 512.2110 [M + 1], 534.1905 [M + Na] HPLC: 5.424. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.56 (s, 1H), 11.06 (s, 2H), 9.29 (s, 2H), 8.59 (d, <i>J</i> = 7.9 Hz, 2H), 8.21 (d, <i>J</i> = 1.5 Hz, 2H), 7.68 (d, <i>J</i> = 8.1 Hz, 2H), 7.65 (s, 2H), 7.41 (d, <i>J</i> = 7.6 Hz, 4H), 7.33 (t, <i>J</i> = 7.6 Hz, 4H), 7.22 (t, <i>J</i> = 7.5 Hz, 2H), 6.93 (d, <i>J</i> = 8.1 Hz, 2H), 6.71–6.64 (m, 1H), 6.27 (d, <i>J</i> = 15.5 Hz, 1H), 5.21–5.17 (m, 1H), 3.05 (d, <i>J</i> = 5.6 Hz, 2H), 2.21 (s, 6H), 2.19 (s, 3H), 2.18 (s, 3H), 1.50 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.09, 166.24, 163.98, 145.47, 141.00, 140.66, 132.25, 128.68, 128.55, 127.91, 127.30, 126.87, 126.48, 126.08, 125.81, 125.74, 125.61, 124.86, 124.52, 122.19, 117.61, 112.99, 109.03, 62.41, 48.96, 25.82, 22.64, 14.44, 12.32, 9.77.</div></div><div id="sec4_7_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Bromoacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-phenylethyl)indoline-5-carboxamide (<b>9g</b>)</h4><div class="NLM_p last">To a round-bottom flask were added 83.7 mg of <b>8a</b> (0.21 mmol), 60.1 mg (0.41 mmol) of bromoacetic acid, and 100 mg (0.25 mmol) of DMTMM. All materials were brought up in 2 mL of DMF, and the resulting solution was allowed to stir at room temperature for 1.5 h. Once complete, the reaction was quenched with brine and then extracted with EtOAc (3 × 20 mL). The combined organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, and then the solvent was removed under pressure to give a dark orange solid. The solid was then dissolved in DCM, and the resultant solution was sonicated to give a dark red precipitate. The solid was filtered off and rinsed with excess cold DCM to give the final product. Result: red solid, 42.9 mg, 39%. Molecular formula: C<sub>26</sub>H<sub>25</sub>BrN<sub>4</sub>O<sub>3</sub> ESI-MS calc: 520.11 ESI-MS found: 523.1164 HPLC: 6.397. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.08 (s, 1H), 9.60 (s, 1H), 8.60 (d, <i>J</i> = 7.9 Hz, 1H), 8.21 (d, <i>J</i> = 1.6 Hz, 1H), 7.69 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.65 (s, 1H), 7.41 (d, <i>J</i> = 7.7 Hz, 2H), 7.33 (t, <i>J</i> = 7.6 Hz, 2H), 7.22 (t, <i>J</i> = 7.3 Hz, 1H), 6.93 (d, <i>J</i> = 8.1 Hz, 1H), 5.20 (p, <i>J</i> = 7.2 Hz, 1H), 4.03 (s, 2H), 2.20 (d, <i>J</i> = 10.4 Hz, 6H), 1.51 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.68, 165.82, 165.54, 145.05, 140.30, 131.64, 128.15, 127.59, 126.78, 126.47, 126.07, 125.76, 125.33, 124.44, 124.13, 121.01, 117.35, 112.95, 108.62, 48.34, 29.33, 22.26, 11.61, 9.14.</div></div><div id="sec4_7_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> (<i>Z</i>)-3-((4-(3-Chloro-2-hydroxypropanamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-((<i>R</i>)-1-phenylethyl)indoline-5-carboxamide (<b>9h</b>)</h4><div class="NLM_p last">To a round-bottom flask were added 80 mg of <b>8a</b> (0.20 mmol), 31.1 mg (0.26 mmol) of 3-chloro-2-hydroxypropionic acid, and 81.8 mg (0.32 mmol) of DMTMM. All materials were brought up in 2 mL of DMF, and the resulting solution was allowed to stir at room temperature for 12 h. Once complete, the reaction was quenched with sat. NaCl and extracted with EtOAc (3 × 50 mL). The organic layer was washed with sat. Na<sub>2</sub>CO<sub>3</sub> and brine (1 × 50 mL) and then dried over MgSO<sub>4</sub>. The solvent was removed under pressure, evaporating the material onto a silica gel. It was purified by column chromatography (5–100% acetone/hexanes). Result: orange solid, 72.8 mg, 58%. Molecular formula: C<sub>27</sub>H<sub>27</sub>ClN<sub>4</sub>O<sub>4</sub> ESI-MS calc: 506.17 ESI-MS found: 507.1809 [M + 1], 540.2377 [M + H + MeOH] HPLC: 6.043. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.58–13.42 (m, 1H), 11.07 (d, <i>J</i> = 4.9 Hz, 1H), 8.60 (d, <i>J</i> = 8.1 Hz, 1H), 8.23–8.19 (m, 1H), 7.95 (s, 1H), 7.69–7.67 (m, 1H), 7.64 (d, <i>J</i> = 5.9 Hz, 1H), 7.41 (d, <i>J</i> = 7.2 Hz, 2H), 7.33 (t, <i>J</i> = 7.1 Hz, 3H), 7.22 (t, <i>J</i> = 7.2 Hz, 1H), 6.93 (dt, <i>J</i> = 8.2, 2.1 Hz, 1H), 5.19 (p, <i>J</i> = 7.3 Hz, 1H), 3.96 (d, <i>J</i> = 7.9 Hz, 1H), 3.74 (s, 1H), 2.21–2.18 (m, 4H), 2.16 (s, 2H), 1.50 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 172.44, 166.31, 162.74, 145.53, 128.63, 127.95, 126.94, 126.54, 125.88, 124.64, 121.80, 117.71, 115.92, 109.07, 107.36, 55.41, 48.82, 22.73, 21.51, 9.74.</div></div><div id="sec4_7_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(<i>m</i>-tolyl)ethyl)indoline-5-carboxamide (<b>9i</b>)</h4><div class="NLM_p last">To a dried round-bottom flask was added 86.6 mg of <b>8b</b> (0.209 mmol) dissolved in 3 mL of THF. The yellow solution was cooled to 0 °C and 0.04 mL (0.30 mmol) of chloroacetylchloride was added dropwise. The solution was allowed to stir at 0 °C for 30 min. Once complete by TLC, the reaction was quenched with water and extracted with EtOAc. The solvent was removed, and the crude material was purified using a prep plate (50% acetone/hexanes) collecting the major product. The desired product was rinsed off of a silica gel, and then the solvent was removed to give a bright yellow solid. The yellow solid was washed with DCM and sonicated to give a red solid. The red solid was then pulped in water/acetone (30:1) to give the final product. Result: red solid, 57.2 mg, 55%. Molecular formula: C<sub>27</sub>H<sub>27</sub>ClN<sub>4</sub>O<sub>3</sub> ESI-MS calc: 490.18 ESI-MS found: 491.1133 HPLC: 6.68. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.07 (s, 1H), 9.53 (s, 1H), 8.56 (d, <i>J</i> = 8.1 Hz, 1H), 8.21 (s, 1H), 7.74–7.62 (m, 2H), 7.24–7.15 (m, 4H), 7.03 (s, 1H), 6.93 (d, <i>J</i> = 8.0 Hz, 1H), 5.15 (s, 1H), 4.26 (s, 2H), 2.30 (s, 3H), 2.20 (d, <i>J</i> = 8.9 Hz, 6H), 1.49 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 170.18, 169.06, 166.24, 165.84, 145.51, 140.80, 137.63, 132.18, 128.59, 128.10, 127.62, 127.24, 126.26, 125.83, 124.93, 123.68, 121.47, 117.93, 113.45, 109.13, 48.80, 43.22, 22.79, 21.62, 12.15, 9.70.</div></div><div id="sec4_7_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(1-(4-fluorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>9j</b>)</h4><div class="NLM_p last">To a dried round-bottom flask was added 96.0 mg of <b>8c</b> (0.23 mmol) dissolved in 4 mL of THF. The yellow solution was cooled to 0 °C, and 0.02 mL (0.25 mmol) of chloroacetylchloride was added dropwise. The solution was allowed to stir at 0 °C for 30 min. Once complete by TLC, the reaction was quenched with water and extracted with EtOAc. The solvent was removed, and the crude material was purified using a prep plate (50% acetone/hexanes) collecting the major product. The desired product was rinsed off of a silica gel, and then the solvent was removed to give a bright yellow solid. The yellow solid was washed with DCM and sonicated to give a red solid. The red solid was then pulped in water/acetone (30:1) to give the final product. Result: red solid, 46.5 mg, 41%. Molecular formula: C<sub>26</sub>H<sub>24</sub>FClN<sub>4</sub>O<sub>3</sub> ESI-MS calc: 494.15 ESI-MS found: 495.0480 [M + 1] HPLC: 6.439. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.08 (s, 1H), 9.54 (s, 1H), 8.63 (d, <i>J</i> = 7.9 Hz, 1H), 8.23 (d, <i>J</i> = 1.7 Hz, 1H), 7.68 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 7.66 (s, 1H), 7.46–7.43 (m, 2H), 7.15 (tt, <i>J</i> = 9.9, 3.2 Hz, 3H), 6.93 (d, <i>J</i> = 8.2 Hz, 1H), 5.19 (p, <i>J</i> = 7.2 Hz, 1H), 4.28–4.26 (m, 4H), 2.20 (d, <i>J</i> = 10.1 Hz, 6H), 1.50 (d, <i>J</i> = 6.9 Hz, 4H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.20, 169.07, 166.36, 165.86, 162.12, 160.75, 141.78, 141.76, 140.85, 132.22, 128.54, 128.49, 128.41, 128.02, 127.42, 126.27, 125.87, 124.96, 124.70, 121.50, 117.90, 115.38, 115.26, 113.44, 109.15, 48.33, 43.23, 41.97, 40.37, 40.25, 40.13, 40.01, 39.89, 39.77, 39.65, 36.07, 29.51, 22.78, 12.16, 9.72.</div></div><div id="sec4_7_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(pyridin-4-yl)ethyl)indoline-5-carboxamide (<b>9k</b>)</h4><div class="NLM_p last">To a dried round-bottom flask was added 41.5 mg of <b>8d</b> (0.103 mmol) dissolved in 2 mL of THF. The yellow solution was cooled to 0 °C, and 0.01 mL (0.103 mmol) of chloroacetylchloride was added dropwise. The solution was allowed to stir at 0 °C for 30 min. Once complete by TLC, the reaction was quenched with water and extracted with EtOAc. The solvent was removed, and the crude material was purified using a prep plate (50% acetone/hexanes) collecting the major product as the desired product. Result: red solid, 6.3 mg, 13%. Molecular formula: C<sub>25</sub>H<sub>24</sub>ClN<sub>5</sub>O<sub>3</sub> ESI-MS calc: 477.16 ESI-MS found: 478.1734 [M + H] HPLC: 4.363. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.51 (s, 1H), 11.15 (s, 1H), 10.56 (s, 1H), 9.86 (s, 1H), 8.73 (s, 1H), 8.48 (s, 2H), 7.75 (s, 1H), 7.72 (d, <i>J</i> = 7.9 Hz, 1H), 7.34 (d, <i>J</i> = 7.6 Hz, 2H), 7.27 (s, 1H), 6.96 (d, <i>J</i> = 7.9 Hz, 1H), 5.33 (t, <i>J</i> = 7.0 Hz, 1H), 4.27 (s, 2H), 2.21 (s, 6H), 1.50 (d, <i>J</i> = 7.8 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 168.53, 166.49, 165.34, 154.01, 141.60, 140.65, 130.38, 129.30, 129.05, 127.97, 124.37, 123.04, 122.68, 121.09, 119.53, 118.36, 108.73, 54.90, 42.72, 21.66, 10.38, 8.77.</div></div><div id="sec4_7_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(pyridin-2-yl)ethyl)indoline-5-carboxamide (<b>9l</b>)</h4><div class="NLM_p last">To a dried round-bottom flask was added 73.8 mg of <b>8e</b> (0.18 mmol) dissolved in 4 mL of THF. The yellow solution was cooled to 0 °C, and 0.04 mL (0.50 mmol) of chloroacetylchloride was added dropwise. The solution was allowed to stir at 0 °C for 30 min. Once complete by TLC, the solvent was removed under pressure and then DCM was added to the dark red residue. The DCM solution was sonicated to give a dark red precipitate. The solid was filtered off and rinsed with THF and DCM to give the desired product as a dark red solid. Result: red solid, 37.1 mg, 42%. Molecular formula: C<sub>25</sub>H<sub>24</sub>ClN<sub>5</sub>O<sub>3</sub> ESI-MS calc: 477.16 ESI-MS found: 478.1547 [M + 1] HPLC: 4.743. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.28 (t, <i>J</i> = 13.6 Hz, 1H), 11.02 (q, <i>J</i> = 7.5, 5.9 Hz, 1H), 9.86 (s, 1H), 8.98 (d, <i>J</i> = 52.4 Hz, 1H), 8.57 (d, <i>J</i> = 9.4 Hz, 1H), 8.42 (d, <i>J</i> = 9.4 Hz, 1H), 8.26 (q, <i>J</i> = 9.9, 9.4 Hz, 2H), 7.86 (d, <i>J</i> = 11.1 Hz, 1H), 7.64 (s, 2H), 7.58 (d, <i>J</i> = 9.2 Hz, 1H), 7.50 (dt, <i>J</i> = 27.4, 8.6 Hz, 1H), 6.72 (p, <i>J</i> = 8.1 Hz, 1H), 5.23–5.16 (m, 1H), 4.03 (t, <i>J</i> = 8.6 Hz, 1H), 2.92 (d, <i>J</i> = 16.9 Hz, 14H), 2.26 (d, <i>J</i> = 10.2 Hz, 7H), 2.17 (dt, <i>J</i> = 18.1, 8.8 Hz, 4H), 1.97 (q, <i>J</i> = 8.9 Hz, 2H), 1.48–1.40 (m, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 169.75, 166.56, 165.36, 159.41, 145.67, 142.08, 141.13, 140.72, 131.91, 128.62, 127.17, 126.78, 126.44, 125.11, 124.85, 124.26, 123.99, 121.10, 118.54, 52.68, 42.73, 20.48, 11.23, 8.81.</div></div><div id="sec4_7_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(1-(3-chlorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>9m</b>)</h4><div class="NLM_p last">To a dried round-bottom flask was added 40 mg of <b>8f</b> (0.16 mmol) dissolved in 4 mL of THF. The yellow solution was cooled to 0 °C, and 0.01 mL (0.18 mmol) of chloroacetylchloride was added dropwise. The solution was allowed to stir at 0 °C for 30 min. Once complete by TLC, the solvent was removed under pressure and then DCM was added to the dark red residue. The DCM solution was sonicated to give a dark red precipitate. The solid was filtered off and rinsed with THF and DCM to give the desired product as a dark red solid. Result: red solid, 23.4 mg, 28%. Molecular formula: C<sub>26</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 510.12 ESI-MS found: 511.1579 [M + 1] HPLC: 6.742. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.09 (s, 1H), 9.53 (s, 1H), 8.66 (d, <i>J</i> = 7.9 Hz, 1H), 8.21 (d, <i>J</i> = 1.7 Hz, 1H), 7.68 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.66 (s, 1H), 7.47–7.45 (m, 1H), 7.38–7.36 (m, 2H), 7.29 (dq, <i>J</i> = 6.2, 2.2 Hz, 1H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 5.17 (p, <i>J</i> = 7.2 Hz, 1H), 4.26 (s, 2H), 2.20 (d, <i>J</i> = 8.6 Hz, 6H), 1.50 (d, <i>J</i> = 7.1 Hz, 4H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 169.51, 165.17, 147.64, 140.23, 132.71, 131.57, 129.96, 127.20, 126.77, 126.30, 125.79, 125.55, 125.23, 124.73, 124.28, 120.82, 117.27, 112.71, 48.03, 42.55, 39.83, 39.67, 39.50, 39.33, 39.17, 39.00, 38.83, 21.96, 11.48, 9.02.</div></div><div id="sec4_7_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(1-(<i>p</i>-tolyl)ethyl)indoline-5-carboxamide (<b>9n</b>)</h4><div class="NLM_p last">To a dried round-bottom flask was added 93 mg of <b>8g</b> (0.19 mmol) dissolved in 5 mL of THF. The yellow solution was cooled to 0 °C, and 0.1 mL (1.3 mmol) of chloroacetylchloride was added dropwise. The solution was allowed to stir at 0 °C for 30 min. Once complete by TLC, the solvent was removed under pressure and then DCM was added to the dark red residue. The DCM solution was sonicated to give a dark red precipitate. The solid was filtered off and rinsed with THF and DCM to give the desired product as a dark red solid. Result: red solid, 73.6 mg, 77%. Molecular formula: C<sub>27</sub>H<sub>27</sub>ClN<sub>4</sub>O<sub>3</sub> ESI-MS calc: 490.18 ESI-MS found: 473.05 [M-Cl] HPLC: 6.625. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.51 (s, 1H), 11.06 (s, 1H), 9.52 (s, 1H), 8.53 (d, <i>J</i> = 8.0 Hz, 1H), 8.20 (s, 1H), 7.71–7.65 (m, 1H), 7.64 (s, 1H), 7.27 (t, <i>J</i> = 10.7 Hz, 4H), 7.12 (d, <i>J</i> = 7.8 Hz, 6H), 6.92 (d, <i>J</i> = 8.1 Hz, 1H), 5.18–5.12 (m, 2H), 2.26 (d, <i>J</i> = 3.9 Hz, 8H), 1.48 (d, <i>J</i> = 7.1 Hz, 5H).</div></div><div id="sec4_7_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(1-(4-chlorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>9o</b>)</h4><div class="NLM_p last">Synthesized using the protocol described in <b>8h</b>. Yields a brown solid, 43.5 mg, 100%. Molecular formula: C<sub>26</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>3</sub> ESI-MS calc: 510.12 ESI-MS found: 511.12917 [M + 1] HPLC: 6.723. <sup>1</sup>H NMR (499 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.08 (d, <i>J</i> = 7.8 Hz, 1H), 9.53 (s, 1H), 8.64 (dd, <i>J</i> = 7.9, 5.3 Hz, 2H), 8.20 (d, <i>J</i> = 1.5 Hz, 1H), 7.67 (dt, <i>J</i> = 8.0, 2.1 Hz, 2H), 7.65 (s, 1H), 7.44–7.40 (m, 4H), 7.40–7.37 (m, 5H), 6.93 (d, <i>J</i> = 8.1 Hz, 1H), 5.16 (dq, <i>J</i> = 13.7, 7.0 Hz, 1H), 4.26 (d, <i>J</i> = 2.8 Hz, 2H), 2.20 (d, <i>J</i> = 8.4 Hz, 6H), 1.49 (d, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 170.31, 169.68, 165.92, 165.35, 144.15, 140.36, 131.73, 130.99, 128.11, 128.01, 127.44, 126.92, 126.09, 125.76, 125.35, 124.44, 123.36, 122.09, 121.43, 120.99, 120.43, 117.37, 108.64, 55.81, 42.72, 22.10, 11.65, 9.21.</div></div><div id="sec4_7_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Bromoacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(1-(4-fluorophenyl)ethyl)-2-oxoindoline-5-carboxamide (<b>9p</b>)</h4><div class="NLM_p last">This material was prepared using the protocol described for <b>8c</b>. Yields a bright orange solid, 13.7 mg, 23%. HRMS: 541.1106 [M + 1, Br<sup>81</sup>] HPLC: 6.449. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.08 (s, 1H), 9.60 (s, 1H), 8.61 (d, <i>J</i> = 8.1 Hz, 1H), 8.20 (s, 1H), 7.68 (d, <i>J</i> = 8.4 Hz, 1H), 7.65 (s, 1H), 7.44 (t, <i>J</i> = 4.8 Hz, 2H), 7.17–7.13 (m, 2H), 6.93 (dd, <i>J</i> = 8.2, 2.5 Hz, 1H), 5.19 (q, <i>J</i> = 7.4 Hz, 1H), 4.26 (d, <i>J</i> = 2.4 Hz, 1H), 4.03 (d, <i>J</i> = 2.4 Hz, 1H), 2.22–2.18 (m, 6H), 1.51–1.48 (m, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.07, 167.64, 166.24, 162.00, 155.00, 144.70, 140.72, 132.05, 129.71, 128.41, 127.92, 124.56, 122.76, 121.42, 117.77, 115.96, 115.26, 109.01, 48.19, 29.73, 22.66, 12.01, 9.54.</div></div><div id="sec4_7_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> (<i>Z</i>)-<i>N</i>-Benzyl-3-((4-(2-chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxoindoline-5-carboxamide (<b>9q</b>)</h4><div class="NLM_p last">Synthesized using protocol described for <b>9i</b>. Yields an orange solid, 18.2 mg, 21%. Molecular formula: C<sub>25</sub>H<sub>23</sub>ClN<sub>4</sub>O<sub>3</sub> ESI-MS calc: 462.15 ESI-MS found: 463.1529 [M + 1] HPLC: 6.177. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.49 (s, 1H), 11.08 (s, 1H), 9.52 (s, 1H), 8.85 (dt, <i>J</i> = 12.4, 6.1 Hz, 1H), 8.26 (d, <i>J</i> = 1.7 Hz, 1H), 7.78 (d, <i>J</i> = 9.4 Hz, 1H), 7.71 (dd, <i>J</i> = 8.2, 1.7 Hz, 1H), 7.64 (s, 1H), 7.35–7.32 (m, 4H), 7.24 (tt, <i>J</i> = 6.2, 3.0 Hz, 2H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 4.51 (d, <i>J</i> = 6.0 Hz, 2H), 4.26 (s, 2H), 2.21 (s, 3H), 2.18 (s, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.18, 166.91, 165.85, 140.89, 140.42, 132.22, 128.74, 128.72, 128.70, 127.98, 127.78, 127.68, 127.67, 127.62, 127.38, 127.15, 126.21, 125.93, 124.95, 124.54, 123.95, 121.48, 117.63, 113.41, 109.28, 68.98, 56.32, 43.22, 43.09, 43.03, 40.37, 40.25, 40.13, 40.01, 39.89, 39.77, 39.65, 36.08, 32.59, 30.08, 12.14, 11.67, 9.92, 9.65.</div></div><div id="sec4_7_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> (<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(2-phenylpropan-2-yl)indoline-5-carboxamide (<b>9r</b>)</h4><div class="NLM_p last">Synthesized using the protocol described for <b>9i</b>. Yields a brown solid, 14.2 mg, 15%. Molecular formula: C<sub>27</sub>H<sub>27</sub>ClN<sub>4</sub>O<sub>3</sub> ESI-MS calc: 490.18 ESI-MS found: 491.1832 [M + 1], 513.1656 [M + Na] HPLC: 6.63. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.06 (s, 1H), 9.53 (s, 1H), 8.24 (s, 1H), 8.19 (d, <i>J</i> = 1.8 Hz, 1H), 7.68 (s, 1H), 7.64 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.40 (dd, <i>J</i> = 7.9, 1.8 Hz, 3H), 7.28 (t, <i>J</i> = 7.8 Hz, 2H), 7.16 (dd, <i>J</i> = 8.3, 6.3 Hz, 1H), 6.91 (d, <i>J</i> = 8.1 Hz, 1H), 4.26 (s, 2H), 2.20 (d, <i>J</i> = 8.2 Hz, 6H), 1.69 (s, 6H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 170.12, 166.37, 165.75, 148.67, 143.56, 140.59, 132.03, 128.86, 128.24, 127.26, 126.00, 125.65, 125.11, 124.86, 124.66, 121.37, 117.90, 108.94, 55.69, 43.14, 30.13, 12.07, 9.61.</div></div><div id="sec4_7_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> (<i>R</i>,<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-<i>N</i>-(2-methyl-1-phenylpropyl)-2-oxoindoline-5-carboxamide (<b>9s</b>)</h4><div class="NLM_p last">Synthesized using the protocol described for <b>9i</b>. Yields an orange solid, 5.5 mg, 5.8%. Molecular formula: C<sub>28</sub>H<sub>29</sub>ClN<sub>4</sub>O<sub>3</sub> ESI-MS calc: 504.19 ESI-MS found: 505.1997 HPLC: 6.803. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.06 (s, 1H), 9.52 (s, 1H), 8.53 (d, <i>J</i> = 8.9 Hz, 1H), 8.14 (d, <i>J</i> = 1.6 Hz, 1H), 7.67–7.62 (m, 2H), 7.43–7.40 (m, 2H), 7.32 (t, <i>J</i> = 7.6 Hz, 2H), 7.22 (t, <i>J</i> = 7.3 Hz, 1H), 6.92 (d, <i>J</i> = 8.1 Hz, 1H), 4.69 (t, <i>J</i> = 9.2 Hz, 1H), 4.26 (s, 2H), 2.20 (d, <i>J</i> = 10.3 Hz, 6H), 1.04 (d, <i>J</i> = 6.6 Hz, 4H), 0.73 (d, <i>J</i> = 6.7 Hz, 4H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 170.14, 166.69, 165.80, 151.43, 143.78, 140.68, 132.14, 128.47, 128.45, 127.89, 127.78, 127.38, 127.05, 125.84, 124.90, 124.74, 121.43, 118.01, 108.97, 60.35, 43.19, 32.97, 20.45, 12.12, 9.62.</div></div><div id="sec4_7_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> (<i>Z</i>)-3-((4-(2-Chloroacetamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-(3-phenyloxetan-3-yl)indoline-5-carboxamide (<b>9t</b>)</h4><div class="NLM_p last">Synthesized using the protocol described for <b>9i</b>. Yields a red solid, 5.0 mg, 21%. Molecular formula: C<sub>27</sub>H<sub>25</sub>ClN<sub>4</sub>O<sub>4</sub> ESI-MS calc: 504.16 ESI-MS found: 505.1619 HPLC: 5.332. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.52 (s, 1H), 11.28 (s, 1H), 9.54 (d, <i>J</i> = 13.2 Hz, 1H), 8.43 (s, 1H), 7.84 (s, 2H), 7.77 (d, <i>J</i> = 13.4 Hz, 2H), 7.52 (t, <i>J</i> = 9.1 Hz, 4H), 7.39 (dt, <i>J</i> = 14.7, 7.8 Hz, 5H), 7.31 (dd, <i>J</i> = 16.7, 7.7 Hz, 3H), 7.19 (s, 1H), 7.11 (s, 1H), 7.04 (s, 2H), 4.85 (d, <i>J</i> = 9.0 Hz, 2H), 4.45 (t, <i>J</i> = 9.9 Hz, 2H), 4.27 (d, <i>J</i> = 3.1 Hz, 2H), 2.22 (d, <i>J</i> = 4.8 Hz, 6H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 170.16, 167.57, 165.90, 154.45, 143.80, 130.21, 129.17, 129.09, 129.04, 128.64, 126.55, 126.07, 125.78, 125.47, 125.31, 122.08, 121.39, 120.35, 115.60, 110.34, 78.59, 72.40, 43.21, 11.78, 9.23.</div></div><div id="sec4_7_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> (<i>Z</i>)-3-((4-((<i>E</i>)-2-Cyano-4,4-dimethylpent-2-enamido)-3,5-dimethyl-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-((<i>R</i>)-1-phenylethyl)indoline-5-carboxamide (<b>10a</b>)</h4><div class="NLM_p last">To a sealed tube were added 30 mg of <b>9g</b> (0.064 mmol), 0.05 mL of pivaldehyde (0.45 mmol), and 0.10 mL of piperidine (1.0 mmol). All materials were then dissolved in 3 mL of abs. EtOH. The solution was then heated to 75 °C for 12 h. Once complete, the reaction was cooled to room temperature and then the solvent was removed under pressure, evaporating the crude material onto a silica gel. The material was purified by column chromatography (50% acetone/hexanes) to give the desired product. Yields a yellow solid, 14 mg, 41%. MS: 536.1375 [M + 1], 558 [M + Na] HPLC: 7.69. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.54 (s, 1H), 11.09 (s, 1H), 9.57 (d, <i>J</i> = 11.5 Hz, 1H), 8.60 (s, 2H), 8.22 (s, 1H), 7.95 (s, 1H), 7.69 (d, <i>J</i> = 8.4 Hz, 1H), 7.66 (d, <i>J</i> = 9.8 Hz, 1H), 7.41 (d, <i>J</i> = 7.7 Hz, 3H), 7.33 (t, <i>J</i> = 7.5 Hz, 3H), 7.23 (d, <i>J</i> = 7.9 Hz, 2H), 6.93 (d, <i>J</i> = 8.4 Hz, 1H), 5.19 (s, 1H), 2.22–2.17 (m, 6H), 1.50 (d, <i>J</i> = 7.2 Hz, 4H), 1.15 (d, <i>J</i> = 9.5 Hz, 9H), 1.04 (d, <i>J</i> = 6.1 Hz, 3H). <sup>13</sup>C NMR (176 MHz, DMSO) δ 170.08, 167.90, 166.20, 160.95, 153.74, 145.51, 141.02, 140.65, 135.87, 132.53, 128.55, 128.02, 126.86, 126.46, 124.51, 123.07, 120.43, 117.75, 115.67, 98.56, 56.19, 28.98, 25.24, 22.64, 12.00, 9.70.</div></div><div id="sec4_7_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> (<i>Z</i>)-3-((3,5-Dimethyl-4-(oxirane-2-carboxamido)-1<i>H</i>-pyrrol-2-yl)methylene)-2-oxo-<i>N</i>-((<i>R</i>)-1-phenylethyl)indoline-5-carboxamide (<b>10b</b>)</h4><div class="NLM_p last">To a flask were added 36.4 mg of <b>9h</b> and 47.2 mg of K<sub>2</sub>CO<sub>3</sub>. All materials were brought up in 3.0 mL of acetone and 1 mL of MeCN, and then the reaction was refluxed (60 °C) for 30 min. After 30 min, the reaction was heated to 70 °C for 1 h, with cat. KI to push the reaction to completion. Once complete, the reaction was quenched with water (1 × 30 mL) and then extracted with EtOAc (2 × 25 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and then purified by column chromatography (25–100% acetone/hexanes). The desired fractions were collected, and the solvent was removed <i>in vacuo</i>, giving the product as a yellow solid. Result: yellow solid, 3.0 mg, 6%. Molecular formula: C<sub>27</sub>H<sub>26</sub>ClN<sub>4</sub>O<sub>4</sub> ESI-MS calc: 470.20 ESI-MS found: 471.2158 [M + 1] HPLC: 6.591. <sup>1</sup>H NMR (700 MHz, acetone-<i>d</i><sub>6</sub>) δ 13.62 (s, 1H), 10.01 (s, 1H), 8.24–8.22 (m, 2H), 7.83 (d, <i>J</i> = 8.3 Hz, 1H), 7.75 (dt, <i>J</i> = 8.1, 2.2 Hz, 1H), 7.69 (s, 1H), 7.46 (d, <i>J</i> = 8.0 Hz, 3H), 7.33 (t, <i>J</i> = 7.7 Hz, 3H), 7.23 (t, <i>J</i> = 7.3 Hz, 1H), 6.99 (dd, <i>J</i> = 8.3, 5.5 Hz, 1H), 5.33 (q, <i>J</i> = 7.3 Hz, 2H), 3.91 (s, 2H), 3.79 (s, 2H), 2.30 (s, 2H), 2.25 (d, <i>J</i> = 2.7 Hz, 3H), 2.19 (d, <i>J</i> = 5.5 Hz, 2H), 2.14 (d, <i>J</i> = 5.1 Hz, 3H), 1.56 (dd, <i>J</i> = 7.1, 1.3 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO) δ 171.84, 169.70, 167.77, 152.56, 145.07, 140.26, 132.32, 128.18, 127.58, 127.45, 126.50, 126.09, 124.43, 121.76, 120.08, 118.18, 108.61, 54.27, 54.08, 48.37, 22.28, 11.87, 9.44.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i90"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01522" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81225" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81225" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01522?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01522</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Kinase panel collated data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_001.xls" class="ext-link">XLS</a>)</p></li><li><p class="inline"><b>9c</b> forming a covalent adduct with GRK5, benzylic analogs forming a covalent adduct after 8 h incubation, kinome panel for <b>5c</b>, and thermodynamic solubility of indolinone series analogs (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular strings file (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_003.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 4WNK (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_004.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 3NYN (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_005.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 271421 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_006.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 271423 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_007.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 271441 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_008.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 271442 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_009.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273180 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_010.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273181 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_011.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273182 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_012.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273183 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_013.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273220 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_014.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273221 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_015.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273261 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_016.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273441 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_017.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273442 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_018.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273443 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_019.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273444 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_020.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273445 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_021.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273462 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_022.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273463 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_023.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273464 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_024.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273561 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_025.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273562 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_026.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273563 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_027.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273564 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_028.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273583 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_029.cif" class="ext-link">CIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_001.xls">jm0c01522_si_001.xls (419.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf">jm0c01522_si_002.pdf (1.42 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_003.csv">jm0c01522_si_003.csv (3.67 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_004.cif">jm0c01522_si_004.cif (610.44 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_005.cif">jm0c01522_si_005.cif (680.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_006.cif">jm0c01522_si_006.cif (6.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_007.cif">jm0c01522_si_007.cif (5.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_008.cif">jm0c01522_si_008.cif (5.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_009.cif">jm0c01522_si_009.cif (5.51 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_010.cif">jm0c01522_si_010.cif (5.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_011.cif">jm0c01522_si_011.cif (5.27 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_012.cif">jm0c01522_si_012.cif (5.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_013.cif">jm0c01522_si_013.cif (4.8 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_014.cif">jm0c01522_si_014.cif (5.2 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_015.cif">jm0c01522_si_015.cif (6.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_016.cif">jm0c01522_si_016.cif (5.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_017.cif">jm0c01522_si_017.cif (5.2 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_018.cif">jm0c01522_si_018.cif (5.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_019.cif">jm0c01522_si_019.cif (5.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_020.cif">jm0c01522_si_020.cif (5.2 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_021.cif">jm0c01522_si_021.cif (5.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_022.cif">jm0c01522_si_022.cif (5.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_023.cif">jm0c01522_si_023.cif (5.2 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_024.cif">jm0c01522_si_024.cif (5.51 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_025.cif">jm0c01522_si_025.cif (4.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_026.cif">jm0c01522_si_026.cif (5.67 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_027.cif">jm0c01522_si_027.cif (5.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_028.cif">jm0c01522_si_028.cif (5.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_029.cif">jm0c01522_si_029.cif (5.2 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01522" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04881" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04881" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John J. G. Tesmer</span> - <span class="hlFld-Affiliation affiliation">Departments
of Biological Sciences and of Medicinal Chemistry and Molecular Pharmacology, Purdue University, 915 W State St, West Lafayette, Indiana 47907, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1125-3727" title="Orcid link">http://orcid.org/0000-0003-1125-3727</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8be1ffeef8e6eef9cbfbfef9effeeea5eeeffe"><span class="__cf_email__" data-cfemail="d6bca2b3a5bbb3a496a6a3a4b2a3b3f8b3b2a3">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew D. White</span> - <span class="hlFld-Affiliation affiliation">Vahlteich
Medicinal Chemistry Core, College of Pharmacy, University of Michigan, 428 Church St, Ann Arbor, Michigan 48109, United
States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#13647b7a67727d777753667e7a707b3d767766"><span class="__cf_email__" data-cfemail="76011e1f021718121236031b1f151e58131203">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachel A. Rowlands</span> - <span class="hlFld-Affiliation affiliation">Vahlteich
Medicinal Chemistry Core, College of Pharmacy, University of Michigan, 428 Church St, Ann Arbor, Michigan 48109, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiuyan Chen</span> - <span class="hlFld-Affiliation affiliation">Departments
of Biological Sciences and of Medicinal Chemistry and Molecular Pharmacology, Purdue University, 915 W State St, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renee A. Bouley</span> - <span class="hlFld-Affiliation affiliation">Life
Sciences Institute, Departments of Pharmacology and Biological Chemistry, University of Michigan, 210 Washtenaw Ave, Ann Arbor Michigan 48109, United States</span>; 
    <span>Present Address:
                        Department of Chemistry and Biochemistry, Ohio State University, 1459 Mt. Vernon Ave, Marion, OH 43302, USA (R.A.B.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Larisa V. Avramova</span> - <span class="hlFld-Affiliation affiliation">Departments
of Biological Sciences and of Medicinal Chemistry and Molecular Pharmacology, Purdue University, 915 W State St, West Lafayette, Indiana 47907, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.A.R. and Q.C. made equal contributions.</p><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written primarily by R.A.R., Q.C., and J.J.G.T. All authors have given approval to the final version of the manuscript. R.A.R. synthesized all compounds and performed MS experiments. Q.C. performed covalent kinetic analysis, and Q.C., L.A., and R.A.B. determined IC50 values for GRK2, GRK5, GRK5-C474S, and PKA. R.A.B. expressed and purified GRK5 and GRK5-C474S. J.J.G.T., Q.C., and L.A. analyzed IC50 data.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by NIH grants HL071818, HL122416, and CA254402 and the Walther Cancer Foundation (to J.J.G.T.), the American Heart Association grants 19POST34450193 (to Q.C.) and 18POST33960047 (to R.A.B.), and a pilot grant from the University of Michigan Center for Discovery of New Medicines (to A.D.W.).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i94">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Venky Bashar and Kevin Conlon, of the University of Michigan Proteomic Resource Facility for assistance with tandem mass spectrometry efforts.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS</td></tr><tr><td class="NLM_term">AST</td><td class="NLM_def"><p class="first last">active site tether</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">GRK</td><td class="NLM_def"><p class="first last">G protein-coupled receptor kinase</p></td></tr><tr><td class="NLM_term">P-loop</td><td class="NLM_def"><p class="first last">phosphate-binding loop</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectrometry</p></td></tr><tr><td class="NLM_term">RTK</td><td class="NLM_def"><p class="first last">receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i96">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45002" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45002" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 20 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushegian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, V. V.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor kinases: more than just kinases and not only for GPCRs</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1016%2Fj.pharmthera.2011.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=21903131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Ciur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2012&pages=40-69&author=E.+V.+Gurevichauthor=J.+J.+G.+Tesmerauthor=A.+Mushegianauthor=V.+V.+Gurevich&title=G+protein-coupled+receptor+kinases%3A+more+than+just+kinases+and+not+only+for+GPCRs&doi=10.1016%2Fj.pharmthera.2011.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor kinases: More than just kinases and not only for GPCRs</span></div><div class="casAuthors">Gurevich, Eugenia V.; Tesmer, John J. G.; Mushegian, Arcady; Gurevich, Vsevolod V.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-69</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptor (GPCR) kinases (GRKs) are best known for their role in homologous desensitization of GPCRs.  GRKs phosphorylate activated receptors and promote high affinity binding of arrestins, which precludes G protein coupling.  GRKs have a multidomain structure, with the kinase domain inserted into a loop of a regulator of G protein signaling homol. domain.  Unlike many other kinases, GRKs do not need to be phosphorylated in their activation loop to achieve an activated state.  Instead, they are directly activated by docking with active GPCRs.  In this manner they are able to selectively phosphorylate Ser/Thr residues on only the activated form of the receptor, unlike related kinases such as protein kinase A.  GRKs also phosphorylate a variety of non-GPCR substrates and regulate several signaling pathways via direct interactions with other proteins in a phosphorylation-independent manner.  Multiple GRK subtypes are present in virtually every animal cell, with the highest expression levels found in neurons, with their extensive and complex signal regulation.  Insufficient or excessive GRK activity was implicated in a variety of human disorders, ranging from heart failure to depression to Parkinson's disease.  As key regulators of GPCR-dependent and -independent signaling pathways, GRKs are emerging drug targets and promising mol. tools for therapy.  Targeted modulation of expression and/or of activity of several GRK isoforms for therapeutic purposes was recently validated in cardiac disorders and Parkinson's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2us2UKmea7Vg90H21EOLACvtfcHk0lhcPRpMYlQ7-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Ciur%252FE&md5=3f132767c5dfe390c8e2a3b643d68ca3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DGurevich%26aufirst%3DE.%2BV.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DMushegian%26aufirst%3DA.%26aulast%3DGurevich%26aufirst%3DV.%2BV.%26atitle%3DG%2520protein-coupled%2520receptor%2520kinases%253A%2520more%2520than%2520just%2520kinases%2520and%2520not%2520only%2520for%2520GPCRs%26jtitle%3DPharmacol.%2520Ther.%26date%3D2012%26volume%3D133%26spage%3D40%26epage%3D69%26doi%3D10.1016%2Fj.pharmthera.2011.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lieu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span> <span> </span><span class="NLM_article-title">GRK2 and GRK5 as therapeutic targets and their role in maladaptive and pathological cardiac hypertrophy</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1080/14728222.2019.1575363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1080%2F14728222.2019.1575363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=30701991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsFeisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2019&pages=201-214&author=M.+Lieuauthor=W.+J.+Koch&title=GRK2+and+GRK5+as+therapeutic+targets+and+their+role+in+maladaptive+and+pathological+cardiac+hypertrophy&doi=10.1080%2F14728222.2019.1575363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">GRK2 and GRK5 as therapeutic targets and their role in maladaptive and pathological cardiac hypertrophy</span></div><div class="casAuthors">Lieu, Melissa; Koch, Walter J.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">201-214</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  One in every four deaths in the United States is attributed to cardiovascular disease, hence the development and employment of novel and effective therapeutics are necessary to improve the quality of life and survival of affected patient.  Pathol. hypertrophy is a maladaptive response by the heart to relieve wall stress that could result from cardiovascular disease.  Maladaptive hypertrophy can lead to further disease progression and complications such as heart failure; hence, efforts to target hypertrophy to prevent and treat further morbidity and mortality are necessary.: This review summarizes the compelling literature that describes the mechanistic role of GRK2 and GRK5 in maladaptive cardiac hypertrophy; it examines the approaches to inhibit these kinases in hypertrophic animal models and furthermore, it assesses the potential of GRK2 and GRK5 as therapeutic targets for hypertrophy.: GRK2 and GRK5 are novel therapeutic targets for pathol. hypertrophy and may have added benefits of ameliorating morbidity and mortality.  Despite the lesser researched role of GRK2 in cardiac hypertrophy, it may be the advantageous strategy for treating cardiac hypertrophy because of its role in other maladaptive pathways.  Anti-GRK2 therapy optimization and the discovery and development of specific GRK2 and GRK5 small-mol. inhibitors is necessary for the eventual application of successful, effective therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdt3-BjwlyurVg90H21EOLACvtfcHk0lhcPRpMYlQ7-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsFeisbc%253D&md5=28b46e0e3109e05071011c51929de619</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1080%2F14728222.2019.1575363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2019.1575363%26sid%3Dliteratum%253Aachs%26aulast%3DLieu%26aufirst%3DM.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26atitle%3DGRK2%2520and%2520GRK5%2520as%2520therapeutic%2520targets%2520and%2520their%2520role%2520in%2520maladaptive%2520and%2520pathological%2520cardiac%2520hypertrophy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2019%26volume%3D23%26spage%3D201%26epage%3D214%26doi%3D10.1080%2F14728222.2019.1575363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher-Bass, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynham, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled Receptor Kinase 2 as a Therapeutic Target for Heart Failure</span>. <i>Drug Discovery Today: Ther Strategies</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e155</span>– <span class="NLM_lpage">e162</span>, <span class="refDoi"> DOI: 10.1016/j.ddstr.2014.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1016%2Fj.ddstr.2014.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=24839449" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=e155-e162&author=S.+M.+Schumacher-Bassauthor=C.+J.+Traynhamauthor=W.+J.+Koch&title=G+protein-coupled+Receptor+Kinase+2+as+a+Therapeutic+Target+for+Heart+Failure&doi=10.1016%2Fj.ddstr.2014.01.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ddstr.2014.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddstr.2014.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DSchumacher-Bass%26aufirst%3DS.%2BM.%26aulast%3DTraynham%26aufirst%3DC.%2BJ.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26atitle%3DG%2520protein-coupled%2520Receptor%2520Kinase%25202%2520as%2520a%2520Therapeutic%2520Target%2520for%2520Heart%2520Failure%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Ther%2520Strategies%26date%3D2012%26volume%3D9%26spage%3De155%26epage%3De162%26doi%3D10.1016%2Fj.ddstr.2014.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sommer, A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcenberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljepoja, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhlich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roidl, A.</span></span> <span> </span><span class="NLM_article-title">Downregulation of GRK5 hampers the migration of breast cancer cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">15548</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-51923-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1038%2Fs41598-019-51923-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=31664083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A280%3ADC%252BB3MjivV2qsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=15548&author=A.-K.+Sommerauthor=M.+Falcenbergauthor=B.+Ljepojaauthor=T.+Fr%C3%B6hlichauthor=G.+J.+Arnoldauthor=E.+Wagnerauthor=A.+Roidl&title=Downregulation+of+GRK5+hampers+the+migration+of+breast+cancer+cells&doi=10.1038%2Fs41598-019-51923-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of GRK5 hampers the migration of breast cancer cells</span></div><div class="casAuthors">Sommer Ann-Katrin; Ljepoja Bojan; Wagner Ernst; Roidl Andreas; Falcenberg Mathias; Frohlich Thomas; Arnold Georg J</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15548</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Sunitinib is a multispecific kinase inhibitor and one of its targets is the kinase GRK5, which is regulating a multitude of G protein-coupled receptors (GPCRs).  In this study we demonstrate that a decreased GRK5 expression induced by knock-down experiments or sunitinib treatment hampers the migration of cancer cell lines.  A proteomic analysis revealed many pathways related to cell migration which were down regulated upon the GRK5 knock-down.  Furthermore, we found in MDA-MB-231 breast cancer cells that the inhibition of migration is mediated by the GPCR gastrin releasing peptide receptor (GRPR) leading to a reduced expression of migration regulating downstream targets like CDC42 and ROCK1.  An in silico Kaplan Meier analysis revealed that GRK5 and GRPR overexpression reduces the distant metastasis free survival in triple-negative breast cancer (TNBC) patients.  Thus, we suggest a novel anti-migratory effect of impaired GRK5 expression which induces a negative feedback loop on GRPR signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxCRTDbfA4a3thwrmtH4OBfW6udTcc2eYw4TDw1iNQGbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MjivV2qsw%253D%253D&md5=e25615748557349de1f81a313fc96c77</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-51923-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-51923-1%26sid%3Dliteratum%253Aachs%26aulast%3DSommer%26aufirst%3DA.-K.%26aulast%3DFalcenberg%26aufirst%3DM.%26aulast%3DLjepoja%26aufirst%3DB.%26aulast%3DFr%25C3%25B6hlich%26aufirst%3DT.%26aulast%3DArnold%26aufirst%3DG.%2BJ.%26aulast%3DWagner%26aufirst%3DE.%26aulast%3DRoidl%26aufirst%3DA.%26atitle%3DDownregulation%2520of%2520GRK5%2520hampers%2520the%2520migration%2520of%2520breast%2520cancer%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D15548%26doi%3D10.1038%2Fs41598-019-51923-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nogués, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacios-García, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reglero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayor, F.,  Jr.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor kinases (GRKs) in tumorigenesis and cancer progression: GPCR regulators and signaling hubs</span>. <i>Semin. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2017.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1016%2Fj.semcancer.2017.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=28473253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvVygtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2018&pages=78-90&author=L.+Nogu%C3%A9sauthor=J.+Palacios-Garc%C3%ADaauthor=C.+Regleroauthor=V.+Rivasauthor=M.+Nevesauthor=C.+Ribasauthor=P.+Penelaauthor=F.+Mayor&title=G+protein-coupled+receptor+kinases+%28GRKs%29+in+tumorigenesis+and+cancer+progression%3A+GPCR+regulators+and+signaling+hubs&doi=10.1016%2Fj.semcancer.2017.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor kinases (GRKs) in tumorigenesis and cancer progression: GPCR regulators and signaling hubs</span></div><div class="casAuthors">Nogues, Laura; Palacios-Garcia, Julia; Reglero, Clara; Rivas, Veronica; Neves, Maria; Ribas, Catalina; Penela, Petronila; Mayor, Federico, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">78-90</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Increasing evidences point to G protein-coupled receptor kinases (GRKs), a subfamily of protein kinase A/G/C-like kinases, as relevant players in cancer progression, in a cell-type and tumor-specific way.  Alterations in the expression and/or activity of particular GRKs have been identified in several types of tumors, and demonstrated to modulate the proliferation, survival or invasive properties of tumor cells by acting as integrating signaling nodes.  GRKs are able to regulate the functionality of both G protein-coupled receptors (GPCR) and growth factor receptors and to directly control cytosolic, cytoskeletal or nuclear signaling components of pathways relevant for these processes.  Furthermore, many chemokines as well as angiogenic and inflammatory factors present in the tumor microenvironment act through GPCR and other GRK-modulated signaling modules.  Changes in the dosage of certain GRKs in the tumor stroma can alter tumor angiogenesis and the homing of immune cells, thus putting forward these kinases as potentially relevant modulators of the carcinoma-fibroblast-endothelial-immune cell network fostering tumor development and dissemination.  A better understanding of the alterations in different GRK isoforms taking place during cancer development and metastasis in specific tumors and cell types and of its impact in signaling pathways would help to design novel therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS8VB_Dzy3y7Vg90H21EOLACvtfcHk0lgLKaherUTNnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvVygtrw%253D&md5=14c0885cf05ba7d3b236fc77374e37a9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2017.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2017.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DNogu%25C3%25A9s%26aufirst%3DL.%26aulast%3DPalacios-Garc%25C3%25ADa%26aufirst%3DJ.%26aulast%3DReglero%26aufirst%3DC.%26aulast%3DRivas%26aufirst%3DV.%26aulast%3DNeves%26aufirst%3DM.%26aulast%3DRibas%26aufirst%3DC.%26aulast%3DPenela%26aufirst%3DP.%26aulast%3DMayor%26aufirst%3DF.%26atitle%3DG%2520protein-coupled%2520receptor%2520kinases%2520%2528GRKs%2529%2520in%2520tumorigenesis%2520and%2520cancer%2520progression%253A%2520GPCR%2520regulators%2520and%2520signaling%2520hubs%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2018%26volume%3D48%26spage%3D78%26epage%3D90%26doi%3D10.1016%2Fj.semcancer.2017.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martini, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raake, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinge, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGeorge, B. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuprun, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckhart, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitcher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span> <span> </span><span class="NLM_article-title">Uncovering G protein-coupled receptor kinase-5 as a histone deacetylase kinase in the nucleus of cardiomyocytes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">12457</span>– <span class="NLM_lpage">12462</span>, <span class="refDoi"> DOI: 10.1073/pnas.0803153105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1073%2Fpnas.0803153105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=18711143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVKqsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=12457-12462&author=J.+S.+Martiniauthor=P.+Raakeauthor=L.+E.+Vingeauthor=B.+R.+DeGeorgeauthor=J.+K.+Chuprunauthor=D.+M.+Harrisauthor=E.+Gaoauthor=A.+D.+Eckhartauthor=J.+A.+Pitcherauthor=W.+J.+Koch&title=Uncovering+G+protein-coupled+receptor+kinase-5+as+a+histone+deacetylase+kinase+in+the+nucleus+of+cardiomyocytes&doi=10.1073%2Fpnas.0803153105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Uncovering G protein-coupled receptor kinase-5 as a histone deacetylase kinase in the nucleus of cardiomyocytes</span></div><div class="casAuthors">Martini, Jeffrey S.; Raake, Philip; Vinge, Leif E.; DeGeorge, Brent, Jr.; Chuprun, J. Kurt; Harris, David M.; Gao, Erhe; Eckhart, Andrea D.; Pitcher, Julie A.; Koch, Walter J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12457-12462</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">G protein-coupled receptor (GPCR) kinases (GRKs) are crit. regulators of cellular signaling and function.  In cardiomyocytes, GRK2 and GRK5 are two GRKs important for myocardial regulation, and both have been shown to be up-regulated in the dysfunctional heart.  We report that increased levels and activity of GRK5 in failing myocardium may have unique significance due to its nuclear localization, a property not shared by GRK2.  We find that transgenic mice with elevated cardiac GRK5 levels have exaggerated hypertrophy and early heart failure compared with control mice after pressure overload.  This pathol. is not present in cardiac GRK2-overexpressing mice or in mice with overexpression of a mutant GRK5 that is excluded from the nucleus.  Nuclear accumulation of GRK5 is enhanced in myocytes after aortic banding in vivo and in vitro in myocytes after increased Gαq activity, the trigger for pressure-overload hypertrophy.  GRK5 enhances activation of myocyte enhancer factor-2 (MEF2) in concert with Gq signals, demonstrating that nuclear localized GRK5 regulates gene transcription via a pathway critically linked to myocardial hypertrophy.  Mechanistically, we show that this is due to GRK5 acting, in a non-GPCR manner, as a class II histone deacetylase (HDAC) kinase because it can assoc. with and phosphorylate the MEF2 repressor, HDAC5.  Moreover, significant HDAC activity can be found with GRK5 in the heart.  Our data show that GRK5 is a nuclear HDAC kinase that plays a key role in maladaptive cardiac hypertrophy apparently independent of any action directly on GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZMzxx-9k9krVg90H21EOLACvtfcHk0lgLKaherUTNnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVKqsLjE&md5=46d4a997038c83fc06118cf7b808c097</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0803153105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0803153105%26sid%3Dliteratum%253Aachs%26aulast%3DMartini%26aufirst%3DJ.%2BS.%26aulast%3DRaake%26aufirst%3DP.%26aulast%3DVinge%26aufirst%3DL.%2BE.%26aulast%3DDeGeorge%26aufirst%3DB.%2BR.%26aulast%3DChuprun%26aufirst%3DJ.%2BK.%26aulast%3DHarris%26aufirst%3DD.%2BM.%26aulast%3DGao%26aufirst%3DE.%26aulast%3DEckhart%26aufirst%3DA.%2BD.%26aulast%3DPitcher%26aufirst%3DJ.%2BA.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26atitle%3DUncovering%2520G%2520protein-coupled%2520receptor%2520kinase-5%2520as%2520a%2520histone%2520deacetylase%2520kinase%2520in%2520the%2520nucleus%2520of%2520cardiomyocytes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D12457%26epage%3D12462%26doi%3D10.1073%2Fpnas.0803153105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gold, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Premont, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span> <span> </span><span class="NLM_article-title">Determining the absolute requirement of G protein-coupled receptor kinase 5 for pathological cardiac hypertrophy: short communication</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">1048</span>– <span class="NLM_lpage">1053</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.112.273367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1161%2FCIRCRESAHA.112.273367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=22859683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVWmt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2012&pages=1048-1053&author=J.+I.+Goldauthor=E.+Gaoauthor=X.+Shangauthor=R.+T.+Premontauthor=W.+J.+Koch&title=Determining+the+absolute+requirement+of+G+protein-coupled+receptor+kinase+5+for+pathological+cardiac+hypertrophy%3A+short+communication&doi=10.1161%2FCIRCRESAHA.112.273367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Determining the Absolute Requirement of G Protein-Coupled Receptor Kinase 5 for Pathological Cardiac Hypertrophy: Short Communication</span></div><div class="casAuthors">Gold, Jessica I.; Gao, Erhe; Shang, Xiying; Premont, Richard T.; Koch, Walter J.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1048-1053</span>CODEN:
                <span class="NLM_cas:coden">CIRUAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-7330</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Rationale: Heart failure (HF) is often the end phase of maladaptive cardiac hypertrophy.  A contributing factor is activation of a hypertrophic gene expression program controlled by decreased class II histone deacetylase (HDAC) transcriptional repression via HDAC phosphorylation.  Cardiac-specific overexpression of G protein-coupled receptor kinase-5 (GRK5) has previously been shown to possess nuclear activity as a HDAC5 kinase, promoting an intolerance to in vivo ventricular pressure overload; however, its endogenous requirement in adaptive and maladaptive hypertrophy remains unknown.  Objective: We used mouse models with global or cardiomyocyte-specific GRK5 gene deletion to det. the abs. requirement of endogenous GRK5 for cardiac hypertrophy and HF development after chronic hypertrophic stimuli.  Methods and results: Mice with global deletion of GRK5 were subjected to transverse aortic constriction.  At 12 wk, these mice showed attenuated hypertrophy, remodeling, and hypertrophic gene transcription along with preserved cardiac function.  Global GRK5 deletion also diminished hypertrophy and related gene expression due to chronic phenylephrine infusion.  We then generated mice with conditional, cardiac-specific deletion of GRK5 that also demonstrated similar protection from pathol. cardiac hypertrophy and HF after transverse aortic constriction.  Conclusions: These results define myocyte GRK5 as a crit. regulator of pathol. cardiac growth after ventricular pressure overload, supporting its role as an endogenous (patho)-physiol. HDAC kinase.  Further, these results define GRK5 as a potential therapeutic target to limit HF development after hypertrophic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0fXlXmqIa7Vg90H21EOLACvtfcHk0lgLKaherUTNnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVWmt77L&md5=027e21673aa8ef9b5f13c699a8b0b7d0</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.112.273367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.112.273367%26sid%3Dliteratum%253Aachs%26aulast%3DGold%26aufirst%3DJ.%2BI.%26aulast%3DGao%26aufirst%3DE.%26aulast%3DShang%26aufirst%3DX.%26aulast%3DPremont%26aufirst%3DR.%2BT.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26atitle%3DDetermining%2520the%2520absolute%2520requirement%2520of%2520G%2520protein-coupled%2520receptor%2520kinase%25205%2520for%2520pathological%2520cardiac%2520hypertrophy%253A%2520short%2520communication%26jtitle%3DCirc.%2520Res.%26date%3D2012%26volume%3D111%26spage%3D1048%26epage%3D1053%26doi%3D10.1161%2FCIRCRESAHA.112.273367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlegel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinkober, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinhardt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tscheschner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Most, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katus, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raake, P. W.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor kinase 2 promotes cardiac hypertrophy</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0182110</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0182110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1371%2Fjournal.pone.0182110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=28759639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvVGn" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=P.+Schlegelauthor=J.+Reinkoberauthor=E.+Meinhardtauthor=H.+Tscheschnerauthor=E.+Gaoauthor=S.+M.+Schumacherauthor=A.+Yuanauthor=J.+Backsauthor=P.+Mostauthor=T.+Wielandauthor=W.+J.+Kochauthor=H.+A.+Katusauthor=P.+W.+Raake&title=G+protein-coupled+receptor+kinase+2+promotes+cardiac+hypertrophy&doi=10.1371%2Fjournal.pone.0182110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor kinase 2 promotes cardiac hypertrophy</span></div><div class="casAuthors">Schlegel, Philipp; Reinkober, Julia; Meinhardt, Eric; Tscheschner, Henrike; Gao, Erhe; Schumacher, Sarah M.; Yuan, Ancai; Backs, Johannes; Most, Patrick; Wieland, Thomas; Koch, Walter J.; Katus, Hugo A.; Raake, Philip W.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0182110/1-e0182110/19</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The increase in protein activity and upregulation of G-protein coupled receptor kinase 2 (GRK2) is a hallmark of cardiac stress and heart failure.  Inhibition of GRK2 improved cardiac function and survival and diminished cardiac remodeling in various animal heart failure models.  The aim of the present study was to investigate the effects of GRK2 on cardiac hypertrophy and dissect potential mol. mechanisms.  In mice we obsd. increased GRK2 mRNA and protein levels following transverse aortic constriction (TAC).  Conditional GRK2 knockout mice showed attenuated hypertrophic response with preserved ventricular geometry 6 wk after TAC operation compared to wild-type animals.  In isolated neonatal rat ventricular cardiac myocytes stimulation with angiotensin II and phenylephrine enhanced GRK2 expression leading to enhanced signaling via protein kinase B (PKB or Akt), consecutively inhibiting glycogen synthase kinase 3 beta (GSK3β), such promoting nuclear accumulation and activation of nuclear factor of activated T-cells (NFAT).  Cardiac myocyte hypertrophy induced by in vitro GRK2 overexpression increased the cytosolic interaction of GRK2 and phosphoinositide 3-kinase γ (PI3Kγ).  Moreover, inhibition of PI3Kγ as well as GRK2 knock down prevented Akt activation resulting in halted NFAT activity and reduced cardiac myocyte hypertrophy.  Our data show that enhanced GRK2 expression triggers cardiac hypertrophy by GRK2-PI3Kγ mediated Akt phosphorylation and subsequent inactivation of GSK3β, resulting in enhanced NFAT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgmD4eo5m2prVg90H21EOLACvtfcHk0lhrTiWcOpjgdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvVGn&md5=0dd7e8dfead9d04772c97bad97e20520</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0182110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0182110%26sid%3Dliteratum%253Aachs%26aulast%3DSchlegel%26aufirst%3DP.%26aulast%3DReinkober%26aufirst%3DJ.%26aulast%3DMeinhardt%26aufirst%3DE.%26aulast%3DTscheschner%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DE.%26aulast%3DSchumacher%26aufirst%3DS.%2BM.%26aulast%3DYuan%26aufirst%3DA.%26aulast%3DBacks%26aufirst%3DJ.%26aulast%3DMost%26aufirst%3DP.%26aulast%3DWieland%26aufirst%3DT.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26aulast%3DKatus%26aufirst%3DH.%2BA.%26aulast%3DRaake%26aufirst%3DP.%2BW.%26atitle%3DG%2520protein-coupled%2520receptor%2520kinase%25202%2520promotes%2520cardiac%2520hypertrophy%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0182110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cato, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldschmidt, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avramova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A. D.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Selective, Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1628</span>– <span class="NLM_lpage">1634</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFGjtLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1628-1634&author=R.+A.+Rowlandsauthor=M.+C.+Catoauthor=H.+V.+Waldschmidtauthor=R.+A.+Bouleyauthor=Q.+Chenauthor=L.+Avramovaauthor=S.+D.+Larsenauthor=J.+J.+G.+Tesmerauthor=A.+D.+White&title=Structure-Based+Design+of+Selective%2C+Covalent+G+Protein-Coupled+Receptor+Kinase+5+Inhibitors&doi=10.1021%2Facsmedchemlett.9b00365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Selective, Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors</span></div><div class="casAuthors">Rowlands, Rachel A.; Cato, M. Claire; Waldschmidt, Helen V.; Bouley, Renee A.; Chen, Qiuyan; Avramova, Larisa; Larsen, Scott D.; Tesmer, John J. G.; White, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1628-1634</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability of G protein-coupled receptor (GPCR) kinases (GRKs) to regulate desensitization of GPCRs has made GRK2 and GRK5 attractive targets for treating heart failure and other diseases such as cancer.  Although advances have been made toward developing inhibitors that are selective for GRK2, there have been far fewer reports of GRK5 selective compds.  Herein, we describe the development of GRK5 subfamily selective inhibitors, 5 and 16d that covalently interact with a nonconserved cysteine (Cys474) unique to this subfamily.  Compds. 5 and 16d feature a highly amenable pyrrolopyrimidine scaffold that affords high nanomolar to low micromolar activity that can be easily modified with Michael acceptors with various reactivities and geometries.  Our work thereby establishes a new pathway toward further development of subfamily selective GRK inhibitors and establishes Cys474 as a new and useful covalent handle in GRK5 drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotuk0qvJybxbVg90H21EOLACvtfcHk0lhrTiWcOpjgdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFGjtLrJ&md5=6ff91bc5d9ddb70f3d83c6d40d7adc54</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00365%26sid%3Dliteratum%253Aachs%26aulast%3DRowlands%26aufirst%3DR.%2BA.%26aulast%3DCato%26aufirst%3DM.%2BC.%26aulast%3DWaldschmidt%26aufirst%3DH.%2BV.%26aulast%3DBouley%26aufirst%3DR.%2BA.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DAvramova%26aufirst%3DL.%26aulast%3DLarsen%26aufirst%3DS.%2BD.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DWhite%26aufirst%3DA.%2BD.%26atitle%3DStructure-Based%2520Design%2520of%2520Selective%252C%2520Covalent%2520G%2520Protein-Coupled%2520Receptor%2520Kinase%25205%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1628%26epage%3D1634%26doi%3D10.1021%2Facsmedchemlett.9b00365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petter, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0lhVI9Jxsezhxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Farrell, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngai, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louie, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, K. W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Town, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolich, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manning, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherrington, J. M.</span></span> <span> </span><span class="NLM_article-title">SU11248 is a novel FLT3 tyrosine kinase inhibitor with potent activity in vitro and in vivo</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">3597</span>– <span class="NLM_lpage">3605</span>, <span class="refDoi"> DOI: 10.1182/blood-2002-07-2307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1182%2Fblood-2002-07-2307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=12531805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjs1aqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2003&pages=3597-3605&author=A.+M.+O%E2%80%99Farrellauthor=T.+J.+Abramsauthor=H.+A.+Yuenauthor=T.+J.+Ngaiauthor=S.+G.+Louieauthor=K.+W.+H.+Yeeauthor=L.+M.+Wongauthor=W.+Hongauthor=L.+B.+Leeauthor=A.+Townauthor=B.+D.+Smolichauthor=W.+C.+Manningauthor=L.+J.+Murrayauthor=M.+C.+Heinrichauthor=J.+M.+Cherrington&title=SU11248+is+a+novel+FLT3+tyrosine+kinase+inhibitor+with+potent+activity+in+vitro+and+in+vivo&doi=10.1182%2Fblood-2002-07-2307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">SU11248 is a novel FLT3 tyrosine kinase inhibitor with potent activity in vitro and in vivo</span></div><div class="casAuthors">O'Farrell, Anne-Marie; Abrams, Tinya J.; Yuen, Helene A.; Ngai, Theresa J.; Louie, Sharianne G.; Yee, Kevin W. H.; Wong, Lily M.; Hong, Weiru; Lee, Leslie B.; Town, Ajia; Smolich, Beverly D.; Manning, William C.; Murray, Lesley J.; Heinrich, Michael C.; Cherrington, Julie M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3597-3605</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">FLT3 (fms-related tyrosine kinase/Flk2/Stk-2) is a receptor tyrosine kinase (RTK) primarily expressed on hematopoietic cells.  In blasts from acute myelogenous leukemia (AML) patients, 2 classes of FLT3 activating mutations have been identified: internal tandem duplication (ITD) mutations in the juxtamembrane domain (25%-30% of patients) and point mutations in the kinase domain activation loop (7%-8% of patients).  FLT3-ITD mutations are the most common mol. defect identified in AML and have been shown to be an independent prognostic factor for decreased survival.  FLT3-ITD is therefore an attractive mol. target for therapy.  SU11248 is a recently described selective inhibitor with selectivity for split kinase domain RTKs, including platelet-derived growth factor receptors, vascular endothelial growth factor receptors, and KIT.  We show that SU11248 also has potent activity against wild-type FLT3 (FLT3-WT), FLT3-ITD, and FLT3 activation loop (FLT3-Asp835) mutants in phosphorylation assays.  SU11248 inhibits FLT3-driven phosphorylation and induces apoptosis in vitro.  In addn., SU11248 inhibits FLT3-induced VEGF prodn.  The in vivo efficacy of SU11248 was investigated in 2 FLT3-ITD models: a s.c. tumor xenograft model and a bone marrow engraftment model.  We show that SU11248 (20 mg/kg/d) dramatically regresses FLT3-ITD tumors in the s.c. tumor xenograft model and prolongs survival in the bone marrow engraftment model.  Pharmacokinetic and pharmacodynamic anal. in s.c. tumors showed that a single administration of an efficacious drug dose potently inhibits FLT3-ITD phosphorylation for up to 16 h following a single dose.  These results suggest that further exploration of SU11248 activity in AML patients is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmUcmQRlrUD7Vg90H21EOLACvtfcHk0lhVI9Jxsezhxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjs1aqtrs%253D&md5=f996fe320ccf9449951baf97e372c694</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1182%2Fblood-2002-07-2307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2002-07-2307%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Farrell%26aufirst%3DA.%2BM.%26aulast%3DAbrams%26aufirst%3DT.%2BJ.%26aulast%3DYuen%26aufirst%3DH.%2BA.%26aulast%3DNgai%26aufirst%3DT.%2BJ.%26aulast%3DLouie%26aufirst%3DS.%2BG.%26aulast%3DYee%26aufirst%3DK.%2BW.%2BH.%26aulast%3DWong%26aufirst%3DL.%2BM.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DL.%2BB.%26aulast%3DTown%26aufirst%3DA.%26aulast%3DSmolich%26aufirst%3DB.%2BD.%26aulast%3DManning%26aufirst%3DW.%2BC.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26aulast%3DCherrington%26aufirst%3DJ.%2BM.%26atitle%3DSU11248%2520is%2520a%2520novel%2520FLT3%2520tyrosine%2520kinase%2520inhibitor%2520with%2520potent%2520activity%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBlood%26date%3D2003%26volume%3D101%26spage%3D3597%26epage%3D3605%26doi%3D10.1182%2Fblood-2002-07-2307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ullrich, A.</span>; <span class="NLM_string-name">Falcenberg, M.</span>; <span class="NLM_string-name">Örfi, Z.</span>; <span class="NLM_string-name">Kéri, G.</span>; <span class="NLM_string-name">ÖRFI, L.</span>; <span class="NLM_string-name">Horváth, Z.</span>; <span class="NLM_string-name">Szokol, B.</span>; <span class="NLM_string-name">Dobos, J.</span>; <span class="NLM_string-name">Nemes, Z.</span></span> <span> </span><span class="NLM_article-title">Indolinone Derivatives as GRK5 Modulators</span>. <span class="NLM_patent">WO2015022437 A1</span>, February 19, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=A.+Ullrich&author=M.+Falcenberg&author=Z.+%C3%96rfi&author=G.+K%C3%A9ri&author=L.+%C3%96RFI&author=Z.+Horv%C3%A1th&author=B.+Szokol&author=J.+Dobos&author=Z.+Nemes&title=Indolinone+Derivatives+as+GRK5+Modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DUllrich%26aufirst%3DA.%26atitle%3DIndolinone%2520Derivatives%2520as%2520GRK5%2520Modulators%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittman, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popel, A. S.</span></span> <span> </span><span class="NLM_article-title">Nitric oxide in the vasculature: where does it come from and where does it go? A quantitative perspective</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1198</span>, <span class="refDoi"> DOI: 10.1089/ars.2007.1959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1089%2Fars.2007.1959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=18331202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1ahsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2008&pages=1185-1198&author=K.+Chenauthor=R.+N.+Pittmanauthor=A.+S.+Popel&title=Nitric+oxide+in+the+vasculature%3A+where+does+it+come+from+and+where+does+it+go%3F+A+quantitative+perspective&doi=10.1089%2Fars.2007.1959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Nitric Oxide in the Vasculature: Where Does It Come From and Where Does It Go? A Quantitative Perspective</span></div><div class="casAuthors">Chen, Kejing; Pittman, Roland N.; Popel, Aleksander S.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1185-1198</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Nitric oxide (NO) affects two key aspects of O2 supply and demand: It regulates vascular tone and blood flow by activating sol. guanylate cyclase (sGC) in the vascular smooth muscle, and it controls mitochondrial O2 consumption by inhibiting cytochrome c oxidase.  However, significant gaps exist in our quant. understanding of the regulation of NO prodn. in the vascular region.  Large apparent discrepancies exist among the published reports that have analyzed the various pathways in terms of the perivascular NO concn., the efficacy of NO in causing vasodilation (EC50), its efficacy in tissue respiration (IC50), and the paracrine and endocrine NO release.  In this study, we review the NO literature, analyzing NO levels on various scales, identifying and analyzing the discrepancies in the reported data, and proposing hypotheses that can potentially reconcile these discrepancies.  Resolving these issues is highly relevant to improving our understanding of vascular biol. and to developing pharmaceutical agents that target NO pathways, such as vasodilating drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHZz9E1uczArVg90H21EOLACvtfcHk0lhVI9Jxsezhxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1ahsrw%253D&md5=70aa4bb25f9addb4191db75ac602d267</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1089%2Fars.2007.1959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2007.1959%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DK.%26aulast%3DPittman%26aufirst%3DR.%2BN.%26aulast%3DPopel%26aufirst%3DA.%2BS.%26atitle%3DNitric%2520oxide%2520in%2520the%2520vasculature%253A%2520where%2520does%2520it%2520come%2520from%2520and%2520where%2520does%2520it%2520go%253F%2520A%2520quantitative%2520perspective%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2008%26volume%3D10%26spage%3D1185%26epage%3D1198%26doi%3D10.1089%2Fars.2007.1959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Homan, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for small molecule inhibition of g protein-coupled receptor kinases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1021/cb5003976</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5003976" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVOjtL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=246-256&author=K.+T.+Homanauthor=J.+J.+G.+Tesmer&title=Molecular+basis+for+small+molecule+inhibition+of+g+protein-coupled+receptor+kinases&doi=10.1021%2Fcb5003976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for small molecule inhibition of G protein-coupled receptor kinases</span></div><div class="casAuthors">Homan, Kristoff T.; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">246-256</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Small mols. that inhibit the protein kinase A, G, and C (AGC) family of serine/threonine kinases can exert profound effects on cell homeostasis and thereby regulate fundamental processes such as heart rate, blood pressure, and metab., but there is not yet a clin. approved drug in the United States selective for a member of this family.  One subfamily of AGC kinases, the G protein-coupled receptor (GPCR) kinases (GRKs), initiates the desensitization of active GPCRs.  Of these, GRK2 has been directly implicated in the progression of heart failure.  Thus, there is great interest in the identification of GRK2-specific chem. probes that can be further developed into therapeutics.  Here, the authors compare crystal structures of small mol. inhibitors in complex with GRK2 to those of highly selective compds. in complex with Rho-assocd. coiled-coil contg. kinase 1 (ROCK1), a closely related AGC kinase.  This anal. suggests that reduced H-bond formation with the hinge of the kinase domain, occupation of the hydrophobic subsite, and, consequently, higher buried surface area are key drivers of potency and selectivity among GRK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw1BX1Q5uXt7Vg90H21EOLACvtfcHk0lim8lfU4HVJlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVOjtL3E&md5=c80248ae417a2d94f3d7bd788273b696</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fcb5003976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5003976%26sid%3Dliteratum%253Aachs%26aulast%3DHoman%26aufirst%3DK.%2BT.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26atitle%3DMolecular%2520basis%2520for%2520small%2520molecule%2520inhibition%2520of%2520g%2520protein-coupled%2520receptor%2520kinases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D246%26epage%3D256%26doi%3D10.1021%2Fcb5003976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waldschmidt, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cato, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz-Rodríguez, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannavo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S. D.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Highly Selective and Potent G Protein-Coupled Receptor Kinase 2 Inhibitors Based on Paroxetine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">3052</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00112</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00112" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslKlt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=3052&author=H.+V.+Waldschmidtauthor=K.+T.+Homanauthor=M.+C.+Catoauthor=O.+Cruz-Rodr%C3%ADguezauthor=A.+Cannavoauthor=M.+W.+Wilsonauthor=J.+Songauthor=J.+Y.+Cheungauthor=W.+J.+Kochauthor=J.+J.+G.+Tesmerauthor=S.+D.+Larsen&title=Structure-Based+Design+of+Highly+Selective+and+Potent+G+Protein-Coupled+Receptor+Kinase+2+Inhibitors+Based+on+Paroxetine&doi=10.1021%2Facs.jmedchem.7b00112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Highly Selective and Potent G Protein-Coupled Receptor Kinase 2 Inhibitors Based on Paroxetine</span></div><div class="casAuthors">Waldschmidt, Helen V.; Homan, Kristoff T.; Cato, Marilyn C.; Cruz-Rodriguez, Osvaldo; Cannavo, Alessandro; Wilson, Michael W.; Song, Jianliang; Cheung, Joseph Y.; Koch, Walter J.; Tesmer, John J. G.; Larsen, Scott D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3052-3069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In heart failure, the β-adrenergic receptors (βARs) become desensitized and uncoupled from heterotrimeric G proteins.  This process is initiated by G protein-coupled receptor kinases (GRKs), some of which are upregulated in the failing heart, making them desirable therapeutic targets.  The selective serotonin reuptake inhibitor, paroxetine, was previously identified as a GRK2 inhibitor.  Utilizing a structure-based drug design approach, we modified paroxetine to generate a small compd. library.  Included in this series is a highly potent and selective GRK2 inhibitor, 14as, with an IC50 of 30 nM against GRK2 and greater than 230-fold selectivity over other GRKs and kinases.  Furthermore, 14as showed a 100-fold improvement in cardiomyocyte contractility assays over paroxetine and a plasma concn. higher than its IC50 for over 7 h.  Three of these inhibitors, including 14as, were addnl. crystd. in complex with GRK2 to give insights into the structural determinants of potency and selectivity of these inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplLzAEiTydaLVg90H21EOLACvtfcHk0lim8lfU4HVJlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslKlt7Y%253D&md5=154677505ef1923a8bad5eda97803f7e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00112%26sid%3Dliteratum%253Aachs%26aulast%3DWaldschmidt%26aufirst%3DH.%2BV.%26aulast%3DHoman%26aufirst%3DK.%2BT.%26aulast%3DCato%26aufirst%3DM.%2BC.%26aulast%3DCruz-Rodr%25C3%25ADguez%26aufirst%3DO.%26aulast%3DCannavo%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DM.%2BW.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DJ.%2BY.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DLarsen%26aufirst%3DS.%2BD.%26atitle%3DStructure-Based%2520Design%2520of%2520Highly%2520Selective%2520and%2520Potent%2520G%2520Protein-Coupled%2520Receptor%2520Kinase%25202%2520Inhibitors%2520Based%2520on%2520Paroxetine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26spage%3D3052%26doi%3D10.1021%2Facs.jmedchem.7b00112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B. H.</span></span> <span> </span><span class="NLM_article-title">KR-39038, a Novel GRK5 Inhibitor, Attenuates Cardiac Hypertrophy and Improves Cardiac Function in Heart Failure</span>. <i>Biomol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">482</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.4062/biomolther.2020.129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.4062%2Fbiomolther.2020.129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A280%3ADC%252BB38bhsFGnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&pages=482-489&author=J.+H.+Leeauthor=H.+W.+Seoauthor=J.+Y.+Ryuauthor=C.+J.+Limauthor=K.+Y.+Yiauthor=K.+S.+Ohauthor=B.+H.+Lee&title=KR-39038%2C+a+Novel+GRK5+Inhibitor%2C+Attenuates+Cardiac+Hypertrophy+and+Improves+Cardiac+Function+in+Heart+Failure&doi=10.4062%2Fbiomolther.2020.129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">KR-39038, a Novel GRK5 Inhibitor, Attenuates Cardiac Hypertrophy and Improves Cardiac Function in Heart Failure</span></div><div class="casAuthors">Lee Jeong Hyun; Seo Ho Won; Ryu Jae Yong; Lim Chae Jo; Yi Kyu Yang; Oh Kwang-Seok; Lee Byung Ho; Lim Chae Jo; Yi Kyu Yang; Oh Kwang-Seok; Lee Byung Ho</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules & therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">482-489</span>
        ISSN:<span class="NLM_cas:issn">1976-9148</span>.
    </div><div class="casAbstract">G protein-coupled receptor kinase 5 (GRK5) has been considered as a potential target for the treatment of heart failure as it has been reported to be an important regulator of pathological cardiac hypertrophy.  To discover novel scaffolds that selectively inhibit GRK5, we have identified a novel small molecule inhibitor of GRK5, KR-39038 [7-((3-((4-((3-aminopropyl)amino)butyl)amino)propyl) amino)-2-(2-chlorophenyl)-6-fluoroquinazolin-4(3H)-one].  KR-39038 exhibited potent inhibitory activity (IC50 value=0.02 μM) against GRK5 and significantly inhibited angiotensin II-induced cellular hypertrophy and HDAC5 phosphorylation in neonatal cardiomyocytes.  In the pressure overload-induced cardiac hypertrophy mouse model, the daily oral administration of KR-39038 (30 mg/kg) for 14 days showed a 43% reduction in the left ventricular weight.  Besides, KR-39038 treatment (10 and 30 mg/kg/ day, p.o.) showed significant preservation of cardiac function and attenuation of myocardial remodeling in a rat model of chronic heart failure following coronary artery ligation.  These results suggest that potent GRK5 inhibitor could effectively attenuate both cardiac hypertrophy and dysfunction in experimental heart failure, and KR-39038 may be useful as an effective GRK5 inhibitor for pharmaceutical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTHu6dVJs3v-RS7coPn-RtfW6udTcc2ebmRdeot6yBwrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38bhsFGnsA%253D%253D&md5=737911b1b4a76d07e2e37612f841586c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4062%2Fbiomolther.2020.129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4062%252Fbiomolther.2020.129%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DSeo%26aufirst%3DH.%2BW.%26aulast%3DRyu%26aufirst%3DJ.%2BY.%26aulast%3DLim%26aufirst%3DC.%2BJ.%26aulast%3DYi%26aufirst%3DK.%2BY.%26aulast%3DOh%26aufirst%3DK.%2BS.%26aulast%3DLee%26aufirst%3DB.%2BH.%26atitle%3DKR-39038%252C%2520a%2520Novel%2520GRK5%2520Inhibitor%252C%2520Attenuates%2520Cardiac%2520Hypertrophy%2520and%2520Improves%2520Cardiac%2520Function%2520in%2520Heart%2520Failure%26jtitle%3DBiomol.%2520Ther.%26date%3D2020%26volume%3D28%26spage%3D482%26epage%3D489%26doi%3D10.4062%2Fbiomolther.2020.129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ngai, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, C. L. L.</span></span> <span> </span><span class="NLM_article-title">Photoinduced Isomerization and Hepatoxicities of Semaxanib, Sunitinib and Related 3-Substituted Indolin-2-ones</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1002%2Fcmdc.201500475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=26593399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyhtLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=72-80&author=M.+H.+Ngaiauthor=C.+L.+Soauthor=M.+B.+Sullivanauthor=H.+K.+Hoauthor=C.+L.+L.+Chai&title=Photoinduced+Isomerization+and+Hepatoxicities+of+Semaxanib%2C+Sunitinib+and+Related+3-Substituted+Indolin-2-ones&doi=10.1002%2Fcmdc.201500475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Photoinduced Isomerization and Hepatoxicities of Semaxanib, Sunitinib and Related 3-Substituted Indolin-2-ones</span></div><div class="casAuthors">Ngai, Mun Hong; So, Choon Leng; Sullivan, Michael B.; Ho, Han Kiat; Chai, Christina L. L.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-80</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">3-Substituted indolin-2-ones are an important class of compds. that display a wide range of biol. activities.  Sunitinib is an orally available multiple tyrosine kinase inhibitor that has been approved by the US Food and Drug Administration (FDA) for the treatment of renal cell cancer.  Sunitinib and a related compd., semaxanib, exist as thermodynamically stable Z isomers, which photoisomerize to E isomers in soln.  In this study, 17 3-substituted indolin-2-ones were synthesized, and the kinetics of their photoisomerization were studied by 1H NMR spectroscopy.  The rate consts. for photoisomerization ranged from 0.009 to 0.048 h-1.  Selected compds. were tested for cytotoxicity in the TAMH liver cell line.  E/Z mixts. of four compds. were also assessed for toxicity in the TAMH and HepG2 cell lines.  In some cases, the stereochem. pure drug was more toxic than the E/Z mixts., but a general statement cannot be made.  Our studies show that each stereoisomer could contribute differently to toxicity, suggesting that stereochem. purity issues that could arise from isomerization cannot be ignored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojaX9DuwbHPrVg90H21EOLACvtfcHk0lhvQ5OAKQHv4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyhtLnL&md5=fd2d515bdb036ad1b42fb3e749e990f0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500475%26sid%3Dliteratum%253Aachs%26aulast%3DNgai%26aufirst%3DM.%2BH.%26aulast%3DSo%26aufirst%3DC.%2BL.%26aulast%3DSullivan%26aufirst%3DM.%2BB.%26aulast%3DHo%26aufirst%3DH.%2BK.%26aulast%3DChai%26aufirst%3DC.%2BL.%2BL.%26atitle%3DPhotoinduced%2520Isomerization%2520and%2520Hepatoxicities%2520of%2520Semaxanib%252C%2520Sunitinib%2520and%2520Related%25203-Substituted%2520Indolin-2-ones%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D72%26epage%3D80%26doi%3D10.1002%2Fcmdc.201500475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Homan, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldschmidt, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glukhova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannavo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of G Protein-coupled Receptor Kinase 5 in Complex with a Rationally Designed Inhibitor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">20649</span>– <span class="NLM_lpage">20659</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.647370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1074%2Fjbc.M115.647370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=26032411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOrtrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=20649-20659&author=K.+T.+Homanauthor=H.+V.+Waldschmidtauthor=A.+Glukhovaauthor=A.+Cannavoauthor=J.+Songauthor=J.+Y.+Cheungauthor=W.+J.+Kochauthor=S.+D.+Larsenauthor=J.+J.+Tesmer&title=Crystal+Structure+of+G+Protein-coupled+Receptor+Kinase+5+in+Complex+with+a+Rationally+Designed+Inhibitor&doi=10.1074%2Fjbc.M115.647370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of G Protein-coupled Receptor Kinase 5 in Complex with a Rationally Designed Inhibitor</span></div><div class="casAuthors">Homan, Kristoff T.; Waldschmidt, Helen V.; Glukhova, Alisa; Cannavo, Alessandro; Song, Jianliang; Cheung, Joseph Y.; Koch, Walter J.; Larsen, Scott D.; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">20649-20659</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">G protein-coupled receptor kinases (GRKs) regulate cell signaling by initiating the desensitization of active G protein-coupled receptors.  The two most widely expressed GRKs (GRK2 and GRK5) play a role in cardiovascular disease and thus represent important targets for the development of novel therapeutic drugs.  In the course of a GRK2 structure-based drug design campaign, one inhibitor (CCG215022) exhibited nanomolar IC50 values against both GRK2 and GRK5 and good selectivity against other closely related kinases such as GRK1 and PKA.  Treatment of murine cardiomyocytes with CCG215022 resulted in significantly increased contractility at 20-fold lower concns. than paroxetine, an inhibitor with more modest selectivity for GRK2.  A 2.4 Å crystal structure of the GRK5·CCG215022 complex was detd. and revealed that the inhibitor binds in the active site similarly to its parent compd. GSK180736A.  As designed, its 2-pyridylmethyl amide side chain occupies the hydrophobic subsite of the active site where it forms three addnl. hydrogen bonds, including one with the catalytic lysine.  The overall conformation of the GRK5 kinase domain is similar to that of a previously detd. structure of GRK6 in what is proposed to be its active state, but the C-terminal region of the enzyme adopts a distinct conformation.  The kinetic properties of site-directed mutants in this region are consistent with the hypothesis that this novel C-terminal structure is representative of the membrane-bound conformation of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCa90PSxbJr7Vg90H21EOLACvtfcHk0lhvQ5OAKQHv4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOrtrfP&md5=1cd56381cfc8d5ec1265cc6672b94946</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.647370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.647370%26sid%3Dliteratum%253Aachs%26aulast%3DHoman%26aufirst%3DK.%2BT.%26aulast%3DWaldschmidt%26aufirst%3DH.%2BV.%26aulast%3DGlukhova%26aufirst%3DA.%26aulast%3DCannavo%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DJ.%2BY.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26aulast%3DLarsen%26aufirst%3DS.%2BD.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%26atitle%3DCrystal%2520Structure%2520of%2520G%2520Protein-coupled%2520Receptor%2520Kinase%25205%2520in%2520Complex%2520with%2520a%2520Rationally%2520Designed%2520Inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D20649%26epage%3D20659%26doi%3D10.1074%2Fjbc.M115.647370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boguth, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-c.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for activation of G protein-coupled receptor kinases</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">3249</span>– <span class="NLM_lpage">3259</span>, <span class="refDoi"> DOI: 10.1038/emboj.2010.206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1038%2Femboj.2010.206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=20729810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGgtb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=3249-3259&author=C.+A.+Boguthauthor=P.+Singhauthor=C.-c.+Huangauthor=J.+J.+G.+Tesmer&title=Molecular+basis+for+activation+of+G+protein-coupled+receptor+kinases&doi=10.1038%2Femboj.2010.206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for activation of G protein-coupled receptor kinases</span></div><div class="casAuthors">Boguth, Cassandra A.; Singh, Puja; Huang, Chih-chin; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3249-3259</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptor (GPCR) kinases (GRKs) selectively recognize and are allosterically regulated by activated GPCRs, but the mol. basis for this interaction is not understood.  Herein, we report crystal structures of GRK6 in which regions known to be crit. for receptor phosphorylation have coalesced to stabilize the kinase domain in a closed state and to form a likely receptor docking site.  The crux of this docking site is an extended N-terminal helix that bridges the large and small lobes of the kinase domain and lies adjacent to a basic surface of the protein proposed to bind anionic phospholipids.  Mutation of exposed, hydrophobic residues in the N-terminal helix selectively inhibits receptor, but not peptide phosphorylation, suggesting that these residues interact directly with GPCRs.  Our structural and biochem. results thus provide an explanation for how receptor recognition, phospholipid binding, and kinase activation are intimately coupled in GRKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxCitsTX_fhbVg90H21EOLACvtfcHk0lgif9cV-39Y9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGgtb%252FE&md5=15d6df48fad121f18286e4ac1db6edef</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.206%26sid%3Dliteratum%253Aachs%26aulast%3DBoguth%26aufirst%3DC.%2BA.%26aulast%3DSingh%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DC.-c.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26atitle%3DMolecular%2520basis%2520for%2520activation%2520of%2520G%2520protein-coupled%2520receptor%2520kinases%26jtitle%3DEMBO%2520J.%26date%3D2010%26volume%3D29%26spage%3D3249%26epage%3D3259%26doi%3D10.1038%2Femboj.2010.206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thal, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinkle, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuprun, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklar, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span> <span> </span><span class="NLM_article-title">Paroxetine is a direct inhibitor of g protein-coupled receptor kinase 2 and increases myocardial contractility</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1830</span>– <span class="NLM_lpage">1839</span>, <span class="refDoi"> DOI: 10.1021/cb3003013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3003013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOntLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1830-1839&author=D.+M.+Thalauthor=K.+T.+Homanauthor=J.+Chenauthor=E.+K.+Wuauthor=P.+M.+Hinkleauthor=Z.+M.+Huangauthor=J.+K.+Chuprunauthor=J.+Songauthor=E.+Gaoauthor=J.+Y.+Cheungauthor=L.+A.+Sklarauthor=W.+J.+Kochauthor=J.+J.+G.+Tesmer&title=Paroxetine+is+a+direct+inhibitor+of+g+protein-coupled+receptor+kinase+2+and+increases+myocardial+contractility&doi=10.1021%2Fcb3003013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Paroxetine Is a Direct Inhibitor of G Protein-Coupled Receptor Kinase 2 and Increases Myocardial Contractility</span></div><div class="casAuthors">Thal, David M.; Homan, Kristoff T.; Chen, Jun; Wu, Emily K.; Hinkle, Patricia M.; Huang, Z. Maggie; Chuprun, J. Kurt; Song, Jianliang; Gao, Erhe; Cheung, Joseph Y.; Sklar, Larry A.; Koch, Walter J.; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1830-1839</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G protein-coupled receptor kinase 2 (GRK2) is a well-established therapeutic target for the treatment of heart failure.  Herein we identify the selective serotonin reuptake inhibitor (SSRI) paroxetine as a selective inhibitor of GRK2 activity both in vitro and in living cells.  In the crystal structure of the GRK2·paroxetine-Gβγ complex, paroxetine binds in the active site of GRK2 and stabilizes the kinase domain in a novel conformation in which a unique regulatory loop forms part of the ligand binding site.  Isolated cardiomyocytes show increased isoproterenol-induced shortening and contraction amplitude in the presence of paroxetine, and pretreatment of mice with paroxetine before isoproterenol significantly increases left ventricular inotropic reserve in vivo with no significant effect on heart rate.  Neither is obsd. in the presence of the SSRI fluoxetine.  Our structural and functional results validate a widely available drug as a selective chem. probe for GRK2 and represent a starting point for the rational design of more potent and specific GRK2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaoH2b0cTO5rVg90H21EOLACvtfcHk0lgif9cV-39Y9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOntLfI&md5=08df20fa41a8b3241640db5de557a001</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcb3003013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3003013%26sid%3Dliteratum%253Aachs%26aulast%3DThal%26aufirst%3DD.%2BM.%26aulast%3DHoman%26aufirst%3DK.%2BT.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DE.%2BK.%26aulast%3DHinkle%26aufirst%3DP.%2BM.%26aulast%3DHuang%26aufirst%3DZ.%2BM.%26aulast%3DChuprun%26aufirst%3DJ.%2BK.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DE.%26aulast%3DCheung%26aufirst%3DJ.%2BY.%26aulast%3DSklar%26aufirst%3DL.%2BA.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26atitle%3DParoxetine%2520is%2520a%2520direct%2520inhibitor%2520of%2520g%2520protein-coupled%2520receptor%2520kinase%25202%2520and%2520increases%2520myocardial%2520contractility%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1830%26epage%3D1839%26doi%3D10.1021%2Fcb3003013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">David E. Uehling, Babu Joseph, Kim Chan Chung, Andrew X. Zhang, Spencer Ler, Michael A. Prakesch, Gennady Poda, Julie Grouleff, Ahmed Aman, Taira Kiyota, Chungyee Leung-Hagesteijn, John David Konda, Richard Marcellus, Carly Griffin, Ratheesh Subramaniam, Ayome Abibi, Craig A. Strathdee, Methvin B. Isaac, Rima Al-awar, <span class="NLM_string-name hlFld-ContribAuthor">Rodger E. Tiedemann</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Characterization of 4-Aminoquinazolines as Potent Inhibitors of the G Protein-Coupled Receptor Kinase 6 (GRK6) for the Treatment of Multiple Myeloma. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00506%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BCharacterization%252Bof%252B4-Aminoquinazolines%252Bas%252BPotent%252BInhibitors%252Bof%252Bthe%252BG%252BProtein-Coupled%252BReceptor%252BKinase%252B6%252B%252528GRK6%252529%252Bfor%252Bthe%252BTreatment%252Bof%252BMultiple%252BMyeloma%26aulast%3DUehling%26aufirst%3DDavid%2BE.%26date%3D2021%26date%3D2021%26date%3D19032021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Lead compounds <b>1</b> and <b>5a</b> with design strategy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking models of lead compound <b>5a</b> and the most potent derivative <b>9j</b> and their comparison with the GRK5·CCG215022 complex. (A) <b>5a</b>, with pink carbons, red oxygens, and blue nitrogens docked using the program MOE into the structure of GRK5 (gray, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WNK">4WNK</a>), with the addition of the AST loop from GRK6 (purple, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NYN">3NYN</a>) because this element was disordered in the 4WNK structure and contains the target Cys474 residue. The diamine moiety extends toward the AST loop where Cys474 is located. Three hydrogen bonds, shown as dashed lines, are formed with the hinge of the kinase domain. None of the compounds in the <b>5a–5f</b> series were able to form adducts despite some that demonstrated high potency. (B) <b>9j</b> (yellow carbons) wherein the amide functionality in the covalent warhead was flipped relative to <b>5a</b> and therefore forms one less hydrogen bond with the hinge. Being less constrained, we hypothesize that this modification allows the warhead to leave the hinge region along a different vector that leads to less steric collisions with the AST and to closer proximity with Cys474. Consequently, compounds in the <b>9a</b> series show adduct formation. Bromine is colored green. (C) Comparison of the fluorophenyl groups of <b>9j</b> and CCG215022 (from 4WNK). The fluorine atom is colored cyan. Both ligands are modeled to form a hydrogen bond with Asp329, an invariant catalytic residue in protein kinases. However, it is unclear whether this region of the scaffold is docked correctly because the conformation of the GRK5 kinase domain, and in particular its P loop and AST, is not known. The SAR in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> is best explained with the fluorophenyl packing under the P loop as it does in the CCG215022 complex.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Convergent Route to <b>5a–5f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span><b>5a</b> and <b>5b</b> are enantiomers derived from <b>3</b> and <b>3a</b>, respectively.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Intact protein MS for compounds <b>9e</b>, <b>9g</b>, and <b>9j</b>. (A) GRK5 (blue) and GRK5+<b>9e</b> (black) incubated for 30 min. <b>9e</b> demonstrated full labeling of GRK5. (B) GRK5 and GRK5+<b>9g</b> (pink) incubated for 30 min. <b>9g</b> only labeled 50% of GRK5. (C) GRK5 and GRK5+<b>9j</b> (red) incubated for 30 min. <b>9j</b> fully labeled GRK5 in this timeframe. (D) GRK5-C474S mutant (purple) and GRK5-C474S+<b>9e</b> (teal) incubated for 30 min. <b>9e</b> did not label GRK5-C474S, indicating that <b>9e</b> and related compounds are engaging Cys474.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. General Synthetic Route to <b>9a–9h</b> and <b>10a</b> and <b>10b</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. General Synthetic Route to <b>9i–9p</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. General Synthetic Route to <b>9q</b>–<b>9t</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinetic analysis for (A, B) <b>9e</b>, (C, D) <b>9g</b>, and (E, F) <b>9j</b>. Extracted values for <i>K</i><sub>I</sub> and <i>k</i><sub>inact</sub> from fitting the plots on the right are given in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Data points represent three measurements with error bars indicating standard deviation. Because the time courses (left panels) were not linear, only the 1 min time point was used to estimate <i>k</i><sub>obs</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/medium/jm0c01522_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Kinome-wide selectivity panel for <b>9g</b> at concentrations of (a) 1 μM, (b) 0.1 μM, and (c) 0.01 μM. Locations of GRK5 and GRK6 are denoted by arrows. A clear dose response is evident, with fewer kinases inhibited at the lowest concentration of <b>9g</b>. Illustration reproduced courtesy of Cell Signaling Technology, Inc. (<a href="http://www.cellsignal.com" class="extLink">www.cellsignal.com</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01522/20210107/images/large/jm0c01522_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01522&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i96">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78761" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78761" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 20 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushegian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, V. V.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor kinases: more than just kinases and not only for GPCRs</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1016%2Fj.pharmthera.2011.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=21903131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Ciur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2012&pages=40-69&author=E.+V.+Gurevichauthor=J.+J.+G.+Tesmerauthor=A.+Mushegianauthor=V.+V.+Gurevich&title=G+protein-coupled+receptor+kinases%3A+more+than+just+kinases+and+not+only+for+GPCRs&doi=10.1016%2Fj.pharmthera.2011.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor kinases: More than just kinases and not only for GPCRs</span></div><div class="casAuthors">Gurevich, Eugenia V.; Tesmer, John J. G.; Mushegian, Arcady; Gurevich, Vsevolod V.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-69</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptor (GPCR) kinases (GRKs) are best known for their role in homologous desensitization of GPCRs.  GRKs phosphorylate activated receptors and promote high affinity binding of arrestins, which precludes G protein coupling.  GRKs have a multidomain structure, with the kinase domain inserted into a loop of a regulator of G protein signaling homol. domain.  Unlike many other kinases, GRKs do not need to be phosphorylated in their activation loop to achieve an activated state.  Instead, they are directly activated by docking with active GPCRs.  In this manner they are able to selectively phosphorylate Ser/Thr residues on only the activated form of the receptor, unlike related kinases such as protein kinase A.  GRKs also phosphorylate a variety of non-GPCR substrates and regulate several signaling pathways via direct interactions with other proteins in a phosphorylation-independent manner.  Multiple GRK subtypes are present in virtually every animal cell, with the highest expression levels found in neurons, with their extensive and complex signal regulation.  Insufficient or excessive GRK activity was implicated in a variety of human disorders, ranging from heart failure to depression to Parkinson's disease.  As key regulators of GPCR-dependent and -independent signaling pathways, GRKs are emerging drug targets and promising mol. tools for therapy.  Targeted modulation of expression and/or of activity of several GRK isoforms for therapeutic purposes was recently validated in cardiac disorders and Parkinson's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2us2UKmea7Vg90H21EOLACvtfcHk0liN4mRpzOjjUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Ciur%252FE&md5=3f132767c5dfe390c8e2a3b643d68ca3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DGurevich%26aufirst%3DE.%2BV.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DMushegian%26aufirst%3DA.%26aulast%3DGurevich%26aufirst%3DV.%2BV.%26atitle%3DG%2520protein-coupled%2520receptor%2520kinases%253A%2520more%2520than%2520just%2520kinases%2520and%2520not%2520only%2520for%2520GPCRs%26jtitle%3DPharmacol.%2520Ther.%26date%3D2012%26volume%3D133%26spage%3D40%26epage%3D69%26doi%3D10.1016%2Fj.pharmthera.2011.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lieu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span> <span> </span><span class="NLM_article-title">GRK2 and GRK5 as therapeutic targets and their role in maladaptive and pathological cardiac hypertrophy</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1080/14728222.2019.1575363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1080%2F14728222.2019.1575363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=30701991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsFeisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2019&pages=201-214&author=M.+Lieuauthor=W.+J.+Koch&title=GRK2+and+GRK5+as+therapeutic+targets+and+their+role+in+maladaptive+and+pathological+cardiac+hypertrophy&doi=10.1080%2F14728222.2019.1575363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">GRK2 and GRK5 as therapeutic targets and their role in maladaptive and pathological cardiac hypertrophy</span></div><div class="casAuthors">Lieu, Melissa; Koch, Walter J.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">201-214</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  One in every four deaths in the United States is attributed to cardiovascular disease, hence the development and employment of novel and effective therapeutics are necessary to improve the quality of life and survival of affected patient.  Pathol. hypertrophy is a maladaptive response by the heart to relieve wall stress that could result from cardiovascular disease.  Maladaptive hypertrophy can lead to further disease progression and complications such as heart failure; hence, efforts to target hypertrophy to prevent and treat further morbidity and mortality are necessary.: This review summarizes the compelling literature that describes the mechanistic role of GRK2 and GRK5 in maladaptive cardiac hypertrophy; it examines the approaches to inhibit these kinases in hypertrophic animal models and furthermore, it assesses the potential of GRK2 and GRK5 as therapeutic targets for hypertrophy.: GRK2 and GRK5 are novel therapeutic targets for pathol. hypertrophy and may have added benefits of ameliorating morbidity and mortality.  Despite the lesser researched role of GRK2 in cardiac hypertrophy, it may be the advantageous strategy for treating cardiac hypertrophy because of its role in other maladaptive pathways.  Anti-GRK2 therapy optimization and the discovery and development of specific GRK2 and GRK5 small-mol. inhibitors is necessary for the eventual application of successful, effective therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdt3-BjwlyurVg90H21EOLACvtfcHk0liDe0G77N1qIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsFeisbc%253D&md5=28b46e0e3109e05071011c51929de619</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1080%2F14728222.2019.1575363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2019.1575363%26sid%3Dliteratum%253Aachs%26aulast%3DLieu%26aufirst%3DM.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26atitle%3DGRK2%2520and%2520GRK5%2520as%2520therapeutic%2520targets%2520and%2520their%2520role%2520in%2520maladaptive%2520and%2520pathological%2520cardiac%2520hypertrophy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2019%26volume%3D23%26spage%3D201%26epage%3D214%26doi%3D10.1080%2F14728222.2019.1575363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher-Bass, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynham, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled Receptor Kinase 2 as a Therapeutic Target for Heart Failure</span>. <i>Drug Discovery Today: Ther Strategies</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e155</span>– <span class="NLM_lpage">e162</span>, <span class="refDoi"> DOI: 10.1016/j.ddstr.2014.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1016%2Fj.ddstr.2014.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=24839449" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=e155-e162&author=S.+M.+Schumacher-Bassauthor=C.+J.+Traynhamauthor=W.+J.+Koch&title=G+protein-coupled+Receptor+Kinase+2+as+a+Therapeutic+Target+for+Heart+Failure&doi=10.1016%2Fj.ddstr.2014.01.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ddstr.2014.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddstr.2014.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DSchumacher-Bass%26aufirst%3DS.%2BM.%26aulast%3DTraynham%26aufirst%3DC.%2BJ.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26atitle%3DG%2520protein-coupled%2520Receptor%2520Kinase%25202%2520as%2520a%2520Therapeutic%2520Target%2520for%2520Heart%2520Failure%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Ther%2520Strategies%26date%3D2012%26volume%3D9%26spage%3De155%26epage%3De162%26doi%3D10.1016%2Fj.ddstr.2014.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sommer, A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcenberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljepoja, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhlich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roidl, A.</span></span> <span> </span><span class="NLM_article-title">Downregulation of GRK5 hampers the migration of breast cancer cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">15548</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-51923-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1038%2Fs41598-019-51923-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=31664083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A280%3ADC%252BB3MjivV2qsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=15548&author=A.-K.+Sommerauthor=M.+Falcenbergauthor=B.+Ljepojaauthor=T.+Fr%C3%B6hlichauthor=G.+J.+Arnoldauthor=E.+Wagnerauthor=A.+Roidl&title=Downregulation+of+GRK5+hampers+the+migration+of+breast+cancer+cells&doi=10.1038%2Fs41598-019-51923-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of GRK5 hampers the migration of breast cancer cells</span></div><div class="casAuthors">Sommer Ann-Katrin; Ljepoja Bojan; Wagner Ernst; Roidl Andreas; Falcenberg Mathias; Frohlich Thomas; Arnold Georg J</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15548</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Sunitinib is a multispecific kinase inhibitor and one of its targets is the kinase GRK5, which is regulating a multitude of G protein-coupled receptors (GPCRs).  In this study we demonstrate that a decreased GRK5 expression induced by knock-down experiments or sunitinib treatment hampers the migration of cancer cell lines.  A proteomic analysis revealed many pathways related to cell migration which were down regulated upon the GRK5 knock-down.  Furthermore, we found in MDA-MB-231 breast cancer cells that the inhibition of migration is mediated by the GPCR gastrin releasing peptide receptor (GRPR) leading to a reduced expression of migration regulating downstream targets like CDC42 and ROCK1.  An in silico Kaplan Meier analysis revealed that GRK5 and GRPR overexpression reduces the distant metastasis free survival in triple-negative breast cancer (TNBC) patients.  Thus, we suggest a novel anti-migratory effect of impaired GRK5 expression which induces a negative feedback loop on GRPR signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxCRTDbfA4a3thwrmtH4OBfW6udTcc2ebIhpMM2hn9JLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MjivV2qsw%253D%253D&md5=e25615748557349de1f81a313fc96c77</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-51923-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-51923-1%26sid%3Dliteratum%253Aachs%26aulast%3DSommer%26aufirst%3DA.-K.%26aulast%3DFalcenberg%26aufirst%3DM.%26aulast%3DLjepoja%26aufirst%3DB.%26aulast%3DFr%25C3%25B6hlich%26aufirst%3DT.%26aulast%3DArnold%26aufirst%3DG.%2BJ.%26aulast%3DWagner%26aufirst%3DE.%26aulast%3DRoidl%26aufirst%3DA.%26atitle%3DDownregulation%2520of%2520GRK5%2520hampers%2520the%2520migration%2520of%2520breast%2520cancer%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D15548%26doi%3D10.1038%2Fs41598-019-51923-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nogués, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacios-García, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reglero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayor, F.,  Jr.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor kinases (GRKs) in tumorigenesis and cancer progression: GPCR regulators and signaling hubs</span>. <i>Semin. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2017.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1016%2Fj.semcancer.2017.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=28473253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvVygtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2018&pages=78-90&author=L.+Nogu%C3%A9sauthor=J.+Palacios-Garc%C3%ADaauthor=C.+Regleroauthor=V.+Rivasauthor=M.+Nevesauthor=C.+Ribasauthor=P.+Penelaauthor=F.+Mayor&title=G+protein-coupled+receptor+kinases+%28GRKs%29+in+tumorigenesis+and+cancer+progression%3A+GPCR+regulators+and+signaling+hubs&doi=10.1016%2Fj.semcancer.2017.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor kinases (GRKs) in tumorigenesis and cancer progression: GPCR regulators and signaling hubs</span></div><div class="casAuthors">Nogues, Laura; Palacios-Garcia, Julia; Reglero, Clara; Rivas, Veronica; Neves, Maria; Ribas, Catalina; Penela, Petronila; Mayor, Federico, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">78-90</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Increasing evidences point to G protein-coupled receptor kinases (GRKs), a subfamily of protein kinase A/G/C-like kinases, as relevant players in cancer progression, in a cell-type and tumor-specific way.  Alterations in the expression and/or activity of particular GRKs have been identified in several types of tumors, and demonstrated to modulate the proliferation, survival or invasive properties of tumor cells by acting as integrating signaling nodes.  GRKs are able to regulate the functionality of both G protein-coupled receptors (GPCR) and growth factor receptors and to directly control cytosolic, cytoskeletal or nuclear signaling components of pathways relevant for these processes.  Furthermore, many chemokines as well as angiogenic and inflammatory factors present in the tumor microenvironment act through GPCR and other GRK-modulated signaling modules.  Changes in the dosage of certain GRKs in the tumor stroma can alter tumor angiogenesis and the homing of immune cells, thus putting forward these kinases as potentially relevant modulators of the carcinoma-fibroblast-endothelial-immune cell network fostering tumor development and dissemination.  A better understanding of the alterations in different GRK isoforms taking place during cancer development and metastasis in specific tumors and cell types and of its impact in signaling pathways would help to design novel therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS8VB_Dzy3y7Vg90H21EOLACvtfcHk0lgOKD5Iy02_CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvVygtrw%253D&md5=14c0885cf05ba7d3b236fc77374e37a9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2017.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2017.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DNogu%25C3%25A9s%26aufirst%3DL.%26aulast%3DPalacios-Garc%25C3%25ADa%26aufirst%3DJ.%26aulast%3DReglero%26aufirst%3DC.%26aulast%3DRivas%26aufirst%3DV.%26aulast%3DNeves%26aufirst%3DM.%26aulast%3DRibas%26aufirst%3DC.%26aulast%3DPenela%26aufirst%3DP.%26aulast%3DMayor%26aufirst%3DF.%26atitle%3DG%2520protein-coupled%2520receptor%2520kinases%2520%2528GRKs%2529%2520in%2520tumorigenesis%2520and%2520cancer%2520progression%253A%2520GPCR%2520regulators%2520and%2520signaling%2520hubs%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2018%26volume%3D48%26spage%3D78%26epage%3D90%26doi%3D10.1016%2Fj.semcancer.2017.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martini, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raake, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinge, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGeorge, B. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuprun, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckhart, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitcher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span> <span> </span><span class="NLM_article-title">Uncovering G protein-coupled receptor kinase-5 as a histone deacetylase kinase in the nucleus of cardiomyocytes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">12457</span>– <span class="NLM_lpage">12462</span>, <span class="refDoi"> DOI: 10.1073/pnas.0803153105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1073%2Fpnas.0803153105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=18711143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVKqsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=12457-12462&author=J.+S.+Martiniauthor=P.+Raakeauthor=L.+E.+Vingeauthor=B.+R.+DeGeorgeauthor=J.+K.+Chuprunauthor=D.+M.+Harrisauthor=E.+Gaoauthor=A.+D.+Eckhartauthor=J.+A.+Pitcherauthor=W.+J.+Koch&title=Uncovering+G+protein-coupled+receptor+kinase-5+as+a+histone+deacetylase+kinase+in+the+nucleus+of+cardiomyocytes&doi=10.1073%2Fpnas.0803153105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Uncovering G protein-coupled receptor kinase-5 as a histone deacetylase kinase in the nucleus of cardiomyocytes</span></div><div class="casAuthors">Martini, Jeffrey S.; Raake, Philip; Vinge, Leif E.; DeGeorge, Brent, Jr.; Chuprun, J. Kurt; Harris, David M.; Gao, Erhe; Eckhart, Andrea D.; Pitcher, Julie A.; Koch, Walter J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12457-12462</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">G protein-coupled receptor (GPCR) kinases (GRKs) are crit. regulators of cellular signaling and function.  In cardiomyocytes, GRK2 and GRK5 are two GRKs important for myocardial regulation, and both have been shown to be up-regulated in the dysfunctional heart.  We report that increased levels and activity of GRK5 in failing myocardium may have unique significance due to its nuclear localization, a property not shared by GRK2.  We find that transgenic mice with elevated cardiac GRK5 levels have exaggerated hypertrophy and early heart failure compared with control mice after pressure overload.  This pathol. is not present in cardiac GRK2-overexpressing mice or in mice with overexpression of a mutant GRK5 that is excluded from the nucleus.  Nuclear accumulation of GRK5 is enhanced in myocytes after aortic banding in vivo and in vitro in myocytes after increased Gαq activity, the trigger for pressure-overload hypertrophy.  GRK5 enhances activation of myocyte enhancer factor-2 (MEF2) in concert with Gq signals, demonstrating that nuclear localized GRK5 regulates gene transcription via a pathway critically linked to myocardial hypertrophy.  Mechanistically, we show that this is due to GRK5 acting, in a non-GPCR manner, as a class II histone deacetylase (HDAC) kinase because it can assoc. with and phosphorylate the MEF2 repressor, HDAC5.  Moreover, significant HDAC activity can be found with GRK5 in the heart.  Our data show that GRK5 is a nuclear HDAC kinase that plays a key role in maladaptive cardiac hypertrophy apparently independent of any action directly on GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZMzxx-9k9krVg90H21EOLACvtfcHk0lgOKD5Iy02_CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVKqsLjE&md5=46d4a997038c83fc06118cf7b808c097</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0803153105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0803153105%26sid%3Dliteratum%253Aachs%26aulast%3DMartini%26aufirst%3DJ.%2BS.%26aulast%3DRaake%26aufirst%3DP.%26aulast%3DVinge%26aufirst%3DL.%2BE.%26aulast%3DDeGeorge%26aufirst%3DB.%2BR.%26aulast%3DChuprun%26aufirst%3DJ.%2BK.%26aulast%3DHarris%26aufirst%3DD.%2BM.%26aulast%3DGao%26aufirst%3DE.%26aulast%3DEckhart%26aufirst%3DA.%2BD.%26aulast%3DPitcher%26aufirst%3DJ.%2BA.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26atitle%3DUncovering%2520G%2520protein-coupled%2520receptor%2520kinase-5%2520as%2520a%2520histone%2520deacetylase%2520kinase%2520in%2520the%2520nucleus%2520of%2520cardiomyocytes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D12457%26epage%3D12462%26doi%3D10.1073%2Fpnas.0803153105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gold, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Premont, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span> <span> </span><span class="NLM_article-title">Determining the absolute requirement of G protein-coupled receptor kinase 5 for pathological cardiac hypertrophy: short communication</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">1048</span>– <span class="NLM_lpage">1053</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.112.273367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1161%2FCIRCRESAHA.112.273367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=22859683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVWmt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2012&pages=1048-1053&author=J.+I.+Goldauthor=E.+Gaoauthor=X.+Shangauthor=R.+T.+Premontauthor=W.+J.+Koch&title=Determining+the+absolute+requirement+of+G+protein-coupled+receptor+kinase+5+for+pathological+cardiac+hypertrophy%3A+short+communication&doi=10.1161%2FCIRCRESAHA.112.273367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Determining the Absolute Requirement of G Protein-Coupled Receptor Kinase 5 for Pathological Cardiac Hypertrophy: Short Communication</span></div><div class="casAuthors">Gold, Jessica I.; Gao, Erhe; Shang, Xiying; Premont, Richard T.; Koch, Walter J.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1048-1053</span>CODEN:
                <span class="NLM_cas:coden">CIRUAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-7330</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Rationale: Heart failure (HF) is often the end phase of maladaptive cardiac hypertrophy.  A contributing factor is activation of a hypertrophic gene expression program controlled by decreased class II histone deacetylase (HDAC) transcriptional repression via HDAC phosphorylation.  Cardiac-specific overexpression of G protein-coupled receptor kinase-5 (GRK5) has previously been shown to possess nuclear activity as a HDAC5 kinase, promoting an intolerance to in vivo ventricular pressure overload; however, its endogenous requirement in adaptive and maladaptive hypertrophy remains unknown.  Objective: We used mouse models with global or cardiomyocyte-specific GRK5 gene deletion to det. the abs. requirement of endogenous GRK5 for cardiac hypertrophy and HF development after chronic hypertrophic stimuli.  Methods and results: Mice with global deletion of GRK5 were subjected to transverse aortic constriction.  At 12 wk, these mice showed attenuated hypertrophy, remodeling, and hypertrophic gene transcription along with preserved cardiac function.  Global GRK5 deletion also diminished hypertrophy and related gene expression due to chronic phenylephrine infusion.  We then generated mice with conditional, cardiac-specific deletion of GRK5 that also demonstrated similar protection from pathol. cardiac hypertrophy and HF after transverse aortic constriction.  Conclusions: These results define myocyte GRK5 as a crit. regulator of pathol. cardiac growth after ventricular pressure overload, supporting its role as an endogenous (patho)-physiol. HDAC kinase.  Further, these results define GRK5 as a potential therapeutic target to limit HF development after hypertrophic stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0fXlXmqIa7Vg90H21EOLACvtfcHk0lgOKD5Iy02_CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVWmt77L&md5=027e21673aa8ef9b5f13c699a8b0b7d0</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.112.273367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.112.273367%26sid%3Dliteratum%253Aachs%26aulast%3DGold%26aufirst%3DJ.%2BI.%26aulast%3DGao%26aufirst%3DE.%26aulast%3DShang%26aufirst%3DX.%26aulast%3DPremont%26aufirst%3DR.%2BT.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26atitle%3DDetermining%2520the%2520absolute%2520requirement%2520of%2520G%2520protein-coupled%2520receptor%2520kinase%25205%2520for%2520pathological%2520cardiac%2520hypertrophy%253A%2520short%2520communication%26jtitle%3DCirc.%2520Res.%26date%3D2012%26volume%3D111%26spage%3D1048%26epage%3D1053%26doi%3D10.1161%2FCIRCRESAHA.112.273367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlegel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinkober, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinhardt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tscheschner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Most, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katus, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raake, P. W.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor kinase 2 promotes cardiac hypertrophy</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0182110</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0182110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1371%2Fjournal.pone.0182110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=28759639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvVGn" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=P.+Schlegelauthor=J.+Reinkoberauthor=E.+Meinhardtauthor=H.+Tscheschnerauthor=E.+Gaoauthor=S.+M.+Schumacherauthor=A.+Yuanauthor=J.+Backsauthor=P.+Mostauthor=T.+Wielandauthor=W.+J.+Kochauthor=H.+A.+Katusauthor=P.+W.+Raake&title=G+protein-coupled+receptor+kinase+2+promotes+cardiac+hypertrophy&doi=10.1371%2Fjournal.pone.0182110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor kinase 2 promotes cardiac hypertrophy</span></div><div class="casAuthors">Schlegel, Philipp; Reinkober, Julia; Meinhardt, Eric; Tscheschner, Henrike; Gao, Erhe; Schumacher, Sarah M.; Yuan, Ancai; Backs, Johannes; Most, Patrick; Wieland, Thomas; Koch, Walter J.; Katus, Hugo A.; Raake, Philip W.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0182110/1-e0182110/19</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The increase in protein activity and upregulation of G-protein coupled receptor kinase 2 (GRK2) is a hallmark of cardiac stress and heart failure.  Inhibition of GRK2 improved cardiac function and survival and diminished cardiac remodeling in various animal heart failure models.  The aim of the present study was to investigate the effects of GRK2 on cardiac hypertrophy and dissect potential mol. mechanisms.  In mice we obsd. increased GRK2 mRNA and protein levels following transverse aortic constriction (TAC).  Conditional GRK2 knockout mice showed attenuated hypertrophic response with preserved ventricular geometry 6 wk after TAC operation compared to wild-type animals.  In isolated neonatal rat ventricular cardiac myocytes stimulation with angiotensin II and phenylephrine enhanced GRK2 expression leading to enhanced signaling via protein kinase B (PKB or Akt), consecutively inhibiting glycogen synthase kinase 3 beta (GSK3β), such promoting nuclear accumulation and activation of nuclear factor of activated T-cells (NFAT).  Cardiac myocyte hypertrophy induced by in vitro GRK2 overexpression increased the cytosolic interaction of GRK2 and phosphoinositide 3-kinase γ (PI3Kγ).  Moreover, inhibition of PI3Kγ as well as GRK2 knock down prevented Akt activation resulting in halted NFAT activity and reduced cardiac myocyte hypertrophy.  Our data show that enhanced GRK2 expression triggers cardiac hypertrophy by GRK2-PI3Kγ mediated Akt phosphorylation and subsequent inactivation of GSK3β, resulting in enhanced NFAT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgmD4eo5m2prVg90H21EOLACvtfcHk0lij4jfRONBrpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvVGn&md5=0dd7e8dfead9d04772c97bad97e20520</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0182110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0182110%26sid%3Dliteratum%253Aachs%26aulast%3DSchlegel%26aufirst%3DP.%26aulast%3DReinkober%26aufirst%3DJ.%26aulast%3DMeinhardt%26aufirst%3DE.%26aulast%3DTscheschner%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DE.%26aulast%3DSchumacher%26aufirst%3DS.%2BM.%26aulast%3DYuan%26aufirst%3DA.%26aulast%3DBacks%26aufirst%3DJ.%26aulast%3DMost%26aufirst%3DP.%26aulast%3DWieland%26aufirst%3DT.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26aulast%3DKatus%26aufirst%3DH.%2BA.%26aulast%3DRaake%26aufirst%3DP.%2BW.%26atitle%3DG%2520protein-coupled%2520receptor%2520kinase%25202%2520promotes%2520cardiac%2520hypertrophy%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0182110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cato, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldschmidt, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avramova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A. D.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Selective, Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1628</span>– <span class="NLM_lpage">1634</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFGjtLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1628-1634&author=R.+A.+Rowlandsauthor=M.+C.+Catoauthor=H.+V.+Waldschmidtauthor=R.+A.+Bouleyauthor=Q.+Chenauthor=L.+Avramovaauthor=S.+D.+Larsenauthor=J.+J.+G.+Tesmerauthor=A.+D.+White&title=Structure-Based+Design+of+Selective%2C+Covalent+G+Protein-Coupled+Receptor+Kinase+5+Inhibitors&doi=10.1021%2Facsmedchemlett.9b00365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Selective, Covalent G Protein-Coupled Receptor Kinase 5 Inhibitors</span></div><div class="casAuthors">Rowlands, Rachel A.; Cato, M. Claire; Waldschmidt, Helen V.; Bouley, Renee A.; Chen, Qiuyan; Avramova, Larisa; Larsen, Scott D.; Tesmer, John J. G.; White, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1628-1634</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ability of G protein-coupled receptor (GPCR) kinases (GRKs) to regulate desensitization of GPCRs has made GRK2 and GRK5 attractive targets for treating heart failure and other diseases such as cancer.  Although advances have been made toward developing inhibitors that are selective for GRK2, there have been far fewer reports of GRK5 selective compds.  Herein, we describe the development of GRK5 subfamily selective inhibitors, 5 and 16d that covalently interact with a nonconserved cysteine (Cys474) unique to this subfamily.  Compds. 5 and 16d feature a highly amenable pyrrolopyrimidine scaffold that affords high nanomolar to low micromolar activity that can be easily modified with Michael acceptors with various reactivities and geometries.  Our work thereby establishes a new pathway toward further development of subfamily selective GRK inhibitors and establishes Cys474 as a new and useful covalent handle in GRK5 drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotuk0qvJybxbVg90H21EOLACvtfcHk0lij4jfRONBrpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFGjtLrJ&md5=6ff91bc5d9ddb70f3d83c6d40d7adc54</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00365%26sid%3Dliteratum%253Aachs%26aulast%3DRowlands%26aufirst%3DR.%2BA.%26aulast%3DCato%26aufirst%3DM.%2BC.%26aulast%3DWaldschmidt%26aufirst%3DH.%2BV.%26aulast%3DBouley%26aufirst%3DR.%2BA.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DAvramova%26aufirst%3DL.%26aulast%3DLarsen%26aufirst%3DS.%2BD.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DWhite%26aufirst%3DA.%2BD.%26atitle%3DStructure-Based%2520Design%2520of%2520Selective%252C%2520Covalent%2520G%2520Protein-Coupled%2520Receptor%2520Kinase%25205%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1628%26epage%3D1634%26doi%3D10.1021%2Facsmedchemlett.9b00365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petter, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0liABX8G4Xe6cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Farrell, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngai, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louie, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, K. W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Town, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolich, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manning, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherrington, J. M.</span></span> <span> </span><span class="NLM_article-title">SU11248 is a novel FLT3 tyrosine kinase inhibitor with potent activity in vitro and in vivo</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">3597</span>– <span class="NLM_lpage">3605</span>, <span class="refDoi"> DOI: 10.1182/blood-2002-07-2307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1182%2Fblood-2002-07-2307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=12531805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjs1aqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2003&pages=3597-3605&author=A.+M.+O%E2%80%99Farrellauthor=T.+J.+Abramsauthor=H.+A.+Yuenauthor=T.+J.+Ngaiauthor=S.+G.+Louieauthor=K.+W.+H.+Yeeauthor=L.+M.+Wongauthor=W.+Hongauthor=L.+B.+Leeauthor=A.+Townauthor=B.+D.+Smolichauthor=W.+C.+Manningauthor=L.+J.+Murrayauthor=M.+C.+Heinrichauthor=J.+M.+Cherrington&title=SU11248+is+a+novel+FLT3+tyrosine+kinase+inhibitor+with+potent+activity+in+vitro+and+in+vivo&doi=10.1182%2Fblood-2002-07-2307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">SU11248 is a novel FLT3 tyrosine kinase inhibitor with potent activity in vitro and in vivo</span></div><div class="casAuthors">O'Farrell, Anne-Marie; Abrams, Tinya J.; Yuen, Helene A.; Ngai, Theresa J.; Louie, Sharianne G.; Yee, Kevin W. H.; Wong, Lily M.; Hong, Weiru; Lee, Leslie B.; Town, Ajia; Smolich, Beverly D.; Manning, William C.; Murray, Lesley J.; Heinrich, Michael C.; Cherrington, Julie M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3597-3605</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">FLT3 (fms-related tyrosine kinase/Flk2/Stk-2) is a receptor tyrosine kinase (RTK) primarily expressed on hematopoietic cells.  In blasts from acute myelogenous leukemia (AML) patients, 2 classes of FLT3 activating mutations have been identified: internal tandem duplication (ITD) mutations in the juxtamembrane domain (25%-30% of patients) and point mutations in the kinase domain activation loop (7%-8% of patients).  FLT3-ITD mutations are the most common mol. defect identified in AML and have been shown to be an independent prognostic factor for decreased survival.  FLT3-ITD is therefore an attractive mol. target for therapy.  SU11248 is a recently described selective inhibitor with selectivity for split kinase domain RTKs, including platelet-derived growth factor receptors, vascular endothelial growth factor receptors, and KIT.  We show that SU11248 also has potent activity against wild-type FLT3 (FLT3-WT), FLT3-ITD, and FLT3 activation loop (FLT3-Asp835) mutants in phosphorylation assays.  SU11248 inhibits FLT3-driven phosphorylation and induces apoptosis in vitro.  In addn., SU11248 inhibits FLT3-induced VEGF prodn.  The in vivo efficacy of SU11248 was investigated in 2 FLT3-ITD models: a s.c. tumor xenograft model and a bone marrow engraftment model.  We show that SU11248 (20 mg/kg/d) dramatically regresses FLT3-ITD tumors in the s.c. tumor xenograft model and prolongs survival in the bone marrow engraftment model.  Pharmacokinetic and pharmacodynamic anal. in s.c. tumors showed that a single administration of an efficacious drug dose potently inhibits FLT3-ITD phosphorylation for up to 16 h following a single dose.  These results suggest that further exploration of SU11248 activity in AML patients is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmUcmQRlrUD7Vg90H21EOLACvtfcHk0liABX8G4Xe6cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjs1aqtrs%253D&md5=f996fe320ccf9449951baf97e372c694</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1182%2Fblood-2002-07-2307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2002-07-2307%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Farrell%26aufirst%3DA.%2BM.%26aulast%3DAbrams%26aufirst%3DT.%2BJ.%26aulast%3DYuen%26aufirst%3DH.%2BA.%26aulast%3DNgai%26aufirst%3DT.%2BJ.%26aulast%3DLouie%26aufirst%3DS.%2BG.%26aulast%3DYee%26aufirst%3DK.%2BW.%2BH.%26aulast%3DWong%26aufirst%3DL.%2BM.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DL.%2BB.%26aulast%3DTown%26aufirst%3DA.%26aulast%3DSmolich%26aufirst%3DB.%2BD.%26aulast%3DManning%26aufirst%3DW.%2BC.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DHeinrich%26aufirst%3DM.%2BC.%26aulast%3DCherrington%26aufirst%3DJ.%2BM.%26atitle%3DSU11248%2520is%2520a%2520novel%2520FLT3%2520tyrosine%2520kinase%2520inhibitor%2520with%2520potent%2520activity%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBlood%26date%3D2003%26volume%3D101%26spage%3D3597%26epage%3D3605%26doi%3D10.1182%2Fblood-2002-07-2307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ullrich, A.</span>; <span class="NLM_string-name">Falcenberg, M.</span>; <span class="NLM_string-name">Örfi, Z.</span>; <span class="NLM_string-name">Kéri, G.</span>; <span class="NLM_string-name">ÖRFI, L.</span>; <span class="NLM_string-name">Horváth, Z.</span>; <span class="NLM_string-name">Szokol, B.</span>; <span class="NLM_string-name">Dobos, J.</span>; <span class="NLM_string-name">Nemes, Z.</span></span> <span> </span><span class="NLM_article-title">Indolinone Derivatives as GRK5 Modulators</span>. <span class="NLM_patent">WO2015022437 A1</span>, February 19, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=A.+Ullrich&author=M.+Falcenberg&author=Z.+%C3%96rfi&author=G.+K%C3%A9ri&author=L.+%C3%96RFI&author=Z.+Horv%C3%A1th&author=B.+Szokol&author=J.+Dobos&author=Z.+Nemes&title=Indolinone+Derivatives+as+GRK5+Modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DUllrich%26aufirst%3DA.%26atitle%3DIndolinone%2520Derivatives%2520as%2520GRK5%2520Modulators%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittman, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popel, A. S.</span></span> <span> </span><span class="NLM_article-title">Nitric oxide in the vasculature: where does it come from and where does it go? A quantitative perspective</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1198</span>, <span class="refDoi"> DOI: 10.1089/ars.2007.1959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1089%2Fars.2007.1959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=18331202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1ahsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2008&pages=1185-1198&author=K.+Chenauthor=R.+N.+Pittmanauthor=A.+S.+Popel&title=Nitric+oxide+in+the+vasculature%3A+where+does+it+come+from+and+where+does+it+go%3F+A+quantitative+perspective&doi=10.1089%2Fars.2007.1959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Nitric Oxide in the Vasculature: Where Does It Come From and Where Does It Go? A Quantitative Perspective</span></div><div class="casAuthors">Chen, Kejing; Pittman, Roland N.; Popel, Aleksander S.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1185-1198</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Nitric oxide (NO) affects two key aspects of O2 supply and demand: It regulates vascular tone and blood flow by activating sol. guanylate cyclase (sGC) in the vascular smooth muscle, and it controls mitochondrial O2 consumption by inhibiting cytochrome c oxidase.  However, significant gaps exist in our quant. understanding of the regulation of NO prodn. in the vascular region.  Large apparent discrepancies exist among the published reports that have analyzed the various pathways in terms of the perivascular NO concn., the efficacy of NO in causing vasodilation (EC50), its efficacy in tissue respiration (IC50), and the paracrine and endocrine NO release.  In this study, we review the NO literature, analyzing NO levels on various scales, identifying and analyzing the discrepancies in the reported data, and proposing hypotheses that can potentially reconcile these discrepancies.  Resolving these issues is highly relevant to improving our understanding of vascular biol. and to developing pharmaceutical agents that target NO pathways, such as vasodilating drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHZz9E1uczArVg90H21EOLACvtfcHk0lgDfRUAeODlyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1ahsrw%253D&md5=70aa4bb25f9addb4191db75ac602d267</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1089%2Fars.2007.1959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2007.1959%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DK.%26aulast%3DPittman%26aufirst%3DR.%2BN.%26aulast%3DPopel%26aufirst%3DA.%2BS.%26atitle%3DNitric%2520oxide%2520in%2520the%2520vasculature%253A%2520where%2520does%2520it%2520come%2520from%2520and%2520where%2520does%2520it%2520go%253F%2520A%2520quantitative%2520perspective%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2008%26volume%3D10%26spage%3D1185%26epage%3D1198%26doi%3D10.1089%2Fars.2007.1959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Homan, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for small molecule inhibition of g protein-coupled receptor kinases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1021/cb5003976</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5003976" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVOjtL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=246-256&author=K.+T.+Homanauthor=J.+J.+G.+Tesmer&title=Molecular+basis+for+small+molecule+inhibition+of+g+protein-coupled+receptor+kinases&doi=10.1021%2Fcb5003976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for small molecule inhibition of G protein-coupled receptor kinases</span></div><div class="casAuthors">Homan, Kristoff T.; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">246-256</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Small mols. that inhibit the protein kinase A, G, and C (AGC) family of serine/threonine kinases can exert profound effects on cell homeostasis and thereby regulate fundamental processes such as heart rate, blood pressure, and metab., but there is not yet a clin. approved drug in the United States selective for a member of this family.  One subfamily of AGC kinases, the G protein-coupled receptor (GPCR) kinases (GRKs), initiates the desensitization of active GPCRs.  Of these, GRK2 has been directly implicated in the progression of heart failure.  Thus, there is great interest in the identification of GRK2-specific chem. probes that can be further developed into therapeutics.  Here, the authors compare crystal structures of small mol. inhibitors in complex with GRK2 to those of highly selective compds. in complex with Rho-assocd. coiled-coil contg. kinase 1 (ROCK1), a closely related AGC kinase.  This anal. suggests that reduced H-bond formation with the hinge of the kinase domain, occupation of the hydrophobic subsite, and, consequently, higher buried surface area are key drivers of potency and selectivity among GRK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw1BX1Q5uXt7Vg90H21EOLACvtfcHk0lgDfRUAeODlyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVOjtL3E&md5=c80248ae417a2d94f3d7bd788273b696</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fcb5003976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5003976%26sid%3Dliteratum%253Aachs%26aulast%3DHoman%26aufirst%3DK.%2BT.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26atitle%3DMolecular%2520basis%2520for%2520small%2520molecule%2520inhibition%2520of%2520g%2520protein-coupled%2520receptor%2520kinases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D246%26epage%3D256%26doi%3D10.1021%2Fcb5003976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waldschmidt, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cato, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz-Rodríguez, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannavo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S. D.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Highly Selective and Potent G Protein-Coupled Receptor Kinase 2 Inhibitors Based on Paroxetine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">3052</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00112</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00112" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslKlt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=3052&author=H.+V.+Waldschmidtauthor=K.+T.+Homanauthor=M.+C.+Catoauthor=O.+Cruz-Rodr%C3%ADguezauthor=A.+Cannavoauthor=M.+W.+Wilsonauthor=J.+Songauthor=J.+Y.+Cheungauthor=W.+J.+Kochauthor=J.+J.+G.+Tesmerauthor=S.+D.+Larsen&title=Structure-Based+Design+of+Highly+Selective+and+Potent+G+Protein-Coupled+Receptor+Kinase+2+Inhibitors+Based+on+Paroxetine&doi=10.1021%2Facs.jmedchem.7b00112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Highly Selective and Potent G Protein-Coupled Receptor Kinase 2 Inhibitors Based on Paroxetine</span></div><div class="casAuthors">Waldschmidt, Helen V.; Homan, Kristoff T.; Cato, Marilyn C.; Cruz-Rodriguez, Osvaldo; Cannavo, Alessandro; Wilson, Michael W.; Song, Jianliang; Cheung, Joseph Y.; Koch, Walter J.; Tesmer, John J. G.; Larsen, Scott D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3052-3069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In heart failure, the β-adrenergic receptors (βARs) become desensitized and uncoupled from heterotrimeric G proteins.  This process is initiated by G protein-coupled receptor kinases (GRKs), some of which are upregulated in the failing heart, making them desirable therapeutic targets.  The selective serotonin reuptake inhibitor, paroxetine, was previously identified as a GRK2 inhibitor.  Utilizing a structure-based drug design approach, we modified paroxetine to generate a small compd. library.  Included in this series is a highly potent and selective GRK2 inhibitor, 14as, with an IC50 of 30 nM against GRK2 and greater than 230-fold selectivity over other GRKs and kinases.  Furthermore, 14as showed a 100-fold improvement in cardiomyocyte contractility assays over paroxetine and a plasma concn. higher than its IC50 for over 7 h.  Three of these inhibitors, including 14as, were addnl. crystd. in complex with GRK2 to give insights into the structural determinants of potency and selectivity of these inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplLzAEiTydaLVg90H21EOLACvtfcHk0lgDfRUAeODlyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslKlt7Y%253D&md5=154677505ef1923a8bad5eda97803f7e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00112%26sid%3Dliteratum%253Aachs%26aulast%3DWaldschmidt%26aufirst%3DH.%2BV.%26aulast%3DHoman%26aufirst%3DK.%2BT.%26aulast%3DCato%26aufirst%3DM.%2BC.%26aulast%3DCruz-Rodr%25C3%25ADguez%26aufirst%3DO.%26aulast%3DCannavo%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DM.%2BW.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DJ.%2BY.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DLarsen%26aufirst%3DS.%2BD.%26atitle%3DStructure-Based%2520Design%2520of%2520Highly%2520Selective%2520and%2520Potent%2520G%2520Protein-Coupled%2520Receptor%2520Kinase%25202%2520Inhibitors%2520Based%2520on%2520Paroxetine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26spage%3D3052%26doi%3D10.1021%2Facs.jmedchem.7b00112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B. H.</span></span> <span> </span><span class="NLM_article-title">KR-39038, a Novel GRK5 Inhibitor, Attenuates Cardiac Hypertrophy and Improves Cardiac Function in Heart Failure</span>. <i>Biomol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">482</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.4062/biomolther.2020.129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.4062%2Fbiomolther.2020.129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A280%3ADC%252BB38bhsFGnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&pages=482-489&author=J.+H.+Leeauthor=H.+W.+Seoauthor=J.+Y.+Ryuauthor=C.+J.+Limauthor=K.+Y.+Yiauthor=K.+S.+Ohauthor=B.+H.+Lee&title=KR-39038%2C+a+Novel+GRK5+Inhibitor%2C+Attenuates+Cardiac+Hypertrophy+and+Improves+Cardiac+Function+in+Heart+Failure&doi=10.4062%2Fbiomolther.2020.129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">KR-39038, a Novel GRK5 Inhibitor, Attenuates Cardiac Hypertrophy and Improves Cardiac Function in Heart Failure</span></div><div class="casAuthors">Lee Jeong Hyun; Seo Ho Won; Ryu Jae Yong; Lim Chae Jo; Yi Kyu Yang; Oh Kwang-Seok; Lee Byung Ho; Lim Chae Jo; Yi Kyu Yang; Oh Kwang-Seok; Lee Byung Ho</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules & therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">482-489</span>
        ISSN:<span class="NLM_cas:issn">1976-9148</span>.
    </div><div class="casAbstract">G protein-coupled receptor kinase 5 (GRK5) has been considered as a potential target for the treatment of heart failure as it has been reported to be an important regulator of pathological cardiac hypertrophy.  To discover novel scaffolds that selectively inhibit GRK5, we have identified a novel small molecule inhibitor of GRK5, KR-39038 [7-((3-((4-((3-aminopropyl)amino)butyl)amino)propyl) amino)-2-(2-chlorophenyl)-6-fluoroquinazolin-4(3H)-one].  KR-39038 exhibited potent inhibitory activity (IC50 value=0.02 μM) against GRK5 and significantly inhibited angiotensin II-induced cellular hypertrophy and HDAC5 phosphorylation in neonatal cardiomyocytes.  In the pressure overload-induced cardiac hypertrophy mouse model, the daily oral administration of KR-39038 (30 mg/kg) for 14 days showed a 43% reduction in the left ventricular weight.  Besides, KR-39038 treatment (10 and 30 mg/kg/ day, p.o.) showed significant preservation of cardiac function and attenuation of myocardial remodeling in a rat model of chronic heart failure following coronary artery ligation.  These results suggest that potent GRK5 inhibitor could effectively attenuate both cardiac hypertrophy and dysfunction in experimental heart failure, and KR-39038 may be useful as an effective GRK5 inhibitor for pharmaceutical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTHu6dVJs3v-RS7coPn-RtfW6udTcc2ebvbLllEKlOXbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38bhsFGnsA%253D%253D&md5=737911b1b4a76d07e2e37612f841586c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4062%2Fbiomolther.2020.129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4062%252Fbiomolther.2020.129%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DSeo%26aufirst%3DH.%2BW.%26aulast%3DRyu%26aufirst%3DJ.%2BY.%26aulast%3DLim%26aufirst%3DC.%2BJ.%26aulast%3DYi%26aufirst%3DK.%2BY.%26aulast%3DOh%26aufirst%3DK.%2BS.%26aulast%3DLee%26aufirst%3DB.%2BH.%26atitle%3DKR-39038%252C%2520a%2520Novel%2520GRK5%2520Inhibitor%252C%2520Attenuates%2520Cardiac%2520Hypertrophy%2520and%2520Improves%2520Cardiac%2520Function%2520in%2520Heart%2520Failure%26jtitle%3DBiomol.%2520Ther.%26date%3D2020%26volume%3D28%26spage%3D482%26epage%3D489%26doi%3D10.4062%2Fbiomolther.2020.129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ngai, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, C. L. L.</span></span> <span> </span><span class="NLM_article-title">Photoinduced Isomerization and Hepatoxicities of Semaxanib, Sunitinib and Related 3-Substituted Indolin-2-ones</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1002%2Fcmdc.201500475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=26593399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyhtLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=72-80&author=M.+H.+Ngaiauthor=C.+L.+Soauthor=M.+B.+Sullivanauthor=H.+K.+Hoauthor=C.+L.+L.+Chai&title=Photoinduced+Isomerization+and+Hepatoxicities+of+Semaxanib%2C+Sunitinib+and+Related+3-Substituted+Indolin-2-ones&doi=10.1002%2Fcmdc.201500475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Photoinduced Isomerization and Hepatoxicities of Semaxanib, Sunitinib and Related 3-Substituted Indolin-2-ones</span></div><div class="casAuthors">Ngai, Mun Hong; So, Choon Leng; Sullivan, Michael B.; Ho, Han Kiat; Chai, Christina L. L.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-80</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">3-Substituted indolin-2-ones are an important class of compds. that display a wide range of biol. activities.  Sunitinib is an orally available multiple tyrosine kinase inhibitor that has been approved by the US Food and Drug Administration (FDA) for the treatment of renal cell cancer.  Sunitinib and a related compd., semaxanib, exist as thermodynamically stable Z isomers, which photoisomerize to E isomers in soln.  In this study, 17 3-substituted indolin-2-ones were synthesized, and the kinetics of their photoisomerization were studied by 1H NMR spectroscopy.  The rate consts. for photoisomerization ranged from 0.009 to 0.048 h-1.  Selected compds. were tested for cytotoxicity in the TAMH liver cell line.  E/Z mixts. of four compds. were also assessed for toxicity in the TAMH and HepG2 cell lines.  In some cases, the stereochem. pure drug was more toxic than the E/Z mixts., but a general statement cannot be made.  Our studies show that each stereoisomer could contribute differently to toxicity, suggesting that stereochem. purity issues that could arise from isomerization cannot be ignored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojaX9DuwbHPrVg90H21EOLACvtfcHk0lh6jsKqvRkV0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyhtLnL&md5=fd2d515bdb036ad1b42fb3e749e990f0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500475%26sid%3Dliteratum%253Aachs%26aulast%3DNgai%26aufirst%3DM.%2BH.%26aulast%3DSo%26aufirst%3DC.%2BL.%26aulast%3DSullivan%26aufirst%3DM.%2BB.%26aulast%3DHo%26aufirst%3DH.%2BK.%26aulast%3DChai%26aufirst%3DC.%2BL.%2BL.%26atitle%3DPhotoinduced%2520Isomerization%2520and%2520Hepatoxicities%2520of%2520Semaxanib%252C%2520Sunitinib%2520and%2520Related%25203-Substituted%2520Indolin-2-ones%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D72%26epage%3D80%26doi%3D10.1002%2Fcmdc.201500475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Homan, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldschmidt, H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glukhova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannavo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of G Protein-coupled Receptor Kinase 5 in Complex with a Rationally Designed Inhibitor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">20649</span>– <span class="NLM_lpage">20659</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.647370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1074%2Fjbc.M115.647370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=26032411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOrtrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=20649-20659&author=K.+T.+Homanauthor=H.+V.+Waldschmidtauthor=A.+Glukhovaauthor=A.+Cannavoauthor=J.+Songauthor=J.+Y.+Cheungauthor=W.+J.+Kochauthor=S.+D.+Larsenauthor=J.+J.+Tesmer&title=Crystal+Structure+of+G+Protein-coupled+Receptor+Kinase+5+in+Complex+with+a+Rationally+Designed+Inhibitor&doi=10.1074%2Fjbc.M115.647370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of G Protein-coupled Receptor Kinase 5 in Complex with a Rationally Designed Inhibitor</span></div><div class="casAuthors">Homan, Kristoff T.; Waldschmidt, Helen V.; Glukhova, Alisa; Cannavo, Alessandro; Song, Jianliang; Cheung, Joseph Y.; Koch, Walter J.; Larsen, Scott D.; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">20649-20659</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">G protein-coupled receptor kinases (GRKs) regulate cell signaling by initiating the desensitization of active G protein-coupled receptors.  The two most widely expressed GRKs (GRK2 and GRK5) play a role in cardiovascular disease and thus represent important targets for the development of novel therapeutic drugs.  In the course of a GRK2 structure-based drug design campaign, one inhibitor (CCG215022) exhibited nanomolar IC50 values against both GRK2 and GRK5 and good selectivity against other closely related kinases such as GRK1 and PKA.  Treatment of murine cardiomyocytes with CCG215022 resulted in significantly increased contractility at 20-fold lower concns. than paroxetine, an inhibitor with more modest selectivity for GRK2.  A 2.4 Å crystal structure of the GRK5·CCG215022 complex was detd. and revealed that the inhibitor binds in the active site similarly to its parent compd. GSK180736A.  As designed, its 2-pyridylmethyl amide side chain occupies the hydrophobic subsite of the active site where it forms three addnl. hydrogen bonds, including one with the catalytic lysine.  The overall conformation of the GRK5 kinase domain is similar to that of a previously detd. structure of GRK6 in what is proposed to be its active state, but the C-terminal region of the enzyme adopts a distinct conformation.  The kinetic properties of site-directed mutants in this region are consistent with the hypothesis that this novel C-terminal structure is representative of the membrane-bound conformation of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCa90PSxbJr7Vg90H21EOLACvtfcHk0lh6jsKqvRkV0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOrtrfP&md5=1cd56381cfc8d5ec1265cc6672b94946</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.647370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.647370%26sid%3Dliteratum%253Aachs%26aulast%3DHoman%26aufirst%3DK.%2BT.%26aulast%3DWaldschmidt%26aufirst%3DH.%2BV.%26aulast%3DGlukhova%26aufirst%3DA.%26aulast%3DCannavo%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DJ.%2BY.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26aulast%3DLarsen%26aufirst%3DS.%2BD.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%26atitle%3DCrystal%2520Structure%2520of%2520G%2520Protein-coupled%2520Receptor%2520Kinase%25205%2520in%2520Complex%2520with%2520a%2520Rationally%2520Designed%2520Inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D20649%26epage%3D20659%26doi%3D10.1074%2Fjbc.M115.647370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boguth, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-c.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for activation of G protein-coupled receptor kinases</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">3249</span>– <span class="NLM_lpage">3259</span>, <span class="refDoi"> DOI: 10.1038/emboj.2010.206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=10.1038%2Femboj.2010.206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=20729810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGgtb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=3249-3259&author=C.+A.+Boguthauthor=P.+Singhauthor=C.-c.+Huangauthor=J.+J.+G.+Tesmer&title=Molecular+basis+for+activation+of+G+protein-coupled+receptor+kinases&doi=10.1038%2Femboj.2010.206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for activation of G protein-coupled receptor kinases</span></div><div class="casAuthors">Boguth, Cassandra A.; Singh, Puja; Huang, Chih-chin; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3249-3259</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptor (GPCR) kinases (GRKs) selectively recognize and are allosterically regulated by activated GPCRs, but the mol. basis for this interaction is not understood.  Herein, we report crystal structures of GRK6 in which regions known to be crit. for receptor phosphorylation have coalesced to stabilize the kinase domain in a closed state and to form a likely receptor docking site.  The crux of this docking site is an extended N-terminal helix that bridges the large and small lobes of the kinase domain and lies adjacent to a basic surface of the protein proposed to bind anionic phospholipids.  Mutation of exposed, hydrophobic residues in the N-terminal helix selectively inhibits receptor, but not peptide phosphorylation, suggesting that these residues interact directly with GPCRs.  Our structural and biochem. results thus provide an explanation for how receptor recognition, phospholipid binding, and kinase activation are intimately coupled in GRKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxCitsTX_fhbVg90H21EOLACvtfcHk0ljd7BsuG_wo3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGgtb%252FE&md5=15d6df48fad121f18286e4ac1db6edef</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.206%26sid%3Dliteratum%253Aachs%26aulast%3DBoguth%26aufirst%3DC.%2BA.%26aulast%3DSingh%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DC.-c.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26atitle%3DMolecular%2520basis%2520for%2520activation%2520of%2520G%2520protein-coupled%2520receptor%2520kinases%26jtitle%3DEMBO%2520J.%26date%3D2010%26volume%3D29%26spage%3D3249%26epage%3D3259%26doi%3D10.1038%2Femboj.2010.206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thal, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homan, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinkle, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuprun, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklar, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J. G.</span></span> <span> </span><span class="NLM_article-title">Paroxetine is a direct inhibitor of g protein-coupled receptor kinase 2 and increases myocardial contractility</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1830</span>– <span class="NLM_lpage">1839</span>, <span class="refDoi"> DOI: 10.1021/cb3003013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3003013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOntLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1830-1839&author=D.+M.+Thalauthor=K.+T.+Homanauthor=J.+Chenauthor=E.+K.+Wuauthor=P.+M.+Hinkleauthor=Z.+M.+Huangauthor=J.+K.+Chuprunauthor=J.+Songauthor=E.+Gaoauthor=J.+Y.+Cheungauthor=L.+A.+Sklarauthor=W.+J.+Kochauthor=J.+J.+G.+Tesmer&title=Paroxetine+is+a+direct+inhibitor+of+g+protein-coupled+receptor+kinase+2+and+increases+myocardial+contractility&doi=10.1021%2Fcb3003013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Paroxetine Is a Direct Inhibitor of G Protein-Coupled Receptor Kinase 2 and Increases Myocardial Contractility</span></div><div class="casAuthors">Thal, David M.; Homan, Kristoff T.; Chen, Jun; Wu, Emily K.; Hinkle, Patricia M.; Huang, Z. Maggie; Chuprun, J. Kurt; Song, Jianliang; Gao, Erhe; Cheung, Joseph Y.; Sklar, Larry A.; Koch, Walter J.; Tesmer, John J. G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1830-1839</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G protein-coupled receptor kinase 2 (GRK2) is a well-established therapeutic target for the treatment of heart failure.  Herein we identify the selective serotonin reuptake inhibitor (SSRI) paroxetine as a selective inhibitor of GRK2 activity both in vitro and in living cells.  In the crystal structure of the GRK2·paroxetine-Gβγ complex, paroxetine binds in the active site of GRK2 and stabilizes the kinase domain in a novel conformation in which a unique regulatory loop forms part of the ligand binding site.  Isolated cardiomyocytes show increased isoproterenol-induced shortening and contraction amplitude in the presence of paroxetine, and pretreatment of mice with paroxetine before isoproterenol significantly increases left ventricular inotropic reserve in vivo with no significant effect on heart rate.  Neither is obsd. in the presence of the SSRI fluoxetine.  Our structural and functional results validate a widely available drug as a selective chem. probe for GRK2 and represent a starting point for the rational design of more potent and specific GRK2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaoH2b0cTO5rVg90H21EOLACvtfcHk0ljd7BsuG_wo3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOntLfI&md5=08df20fa41a8b3241640db5de557a001</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcb3003013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3003013%26sid%3Dliteratum%253Aachs%26aulast%3DThal%26aufirst%3DD.%2BM.%26aulast%3DHoman%26aufirst%3DK.%2BT.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DE.%2BK.%26aulast%3DHinkle%26aufirst%3DP.%2BM.%26aulast%3DHuang%26aufirst%3DZ.%2BM.%26aulast%3DChuprun%26aufirst%3DJ.%2BK.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DE.%26aulast%3DCheung%26aufirst%3DJ.%2BY.%26aulast%3DSklar%26aufirst%3DL.%2BA.%26aulast%3DKoch%26aufirst%3DW.%2BJ.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%2BG.%26atitle%3DParoxetine%2520is%2520a%2520direct%2520inhibitor%2520of%2520g%2520protein-coupled%2520receptor%2520kinase%25202%2520and%2520increases%2520myocardial%2520contractility%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1830%26epage%3D1839%26doi%3D10.1021%2Fcb3003013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WNK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WNK','PDB','4WNK'); return false;">PDB: 4WNK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NYN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NYN','PDB','3NYN'); return false;">PDB: 3NYN</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i90"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_03632"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01522?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01522</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Kinase panel collated data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_001.xls">XLS</a>)</p></li><li><p class="inline"><b>9c</b> forming a covalent adduct with GRK5, benzylic analogs forming a covalent adduct after 8 h incubation, kinome panel for <b>5c</b>, and thermodynamic solubility of indolinone series analogs (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular strings file (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_003.csv">CSV</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 4WNK (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_004.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 3NYN (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_005.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 271421 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_006.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 271423 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_007.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 271441 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_008.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 271442 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_009.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273180 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_010.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273181 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_011.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273182 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_012.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273183 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_013.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273220 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_014.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273221 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_015.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273261 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_016.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273441 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_017.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273442 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_018.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273443 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_019.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273444 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_020.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273445 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_021.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273462 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_022.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273463 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_023.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273464 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_024.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273561 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_025.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273562 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_026.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273563 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_027.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273564 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_028.cif">CIF</a>)</p></li><li><p class="inline">Coordinates for docked ligand complex 273583 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_029.cif">CIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_001.xls">jm0c01522_si_001.xls (419.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_002.pdf">jm0c01522_si_002.pdf (1.42 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_003.csv">jm0c01522_si_003.csv (3.67 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_004.cif">jm0c01522_si_004.cif (610.44 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_005.cif">jm0c01522_si_005.cif (680.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_006.cif">jm0c01522_si_006.cif (6.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_007.cif">jm0c01522_si_007.cif (5.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_008.cif">jm0c01522_si_008.cif (5.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_009.cif">jm0c01522_si_009.cif (5.51 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_010.cif">jm0c01522_si_010.cif (5.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_011.cif">jm0c01522_si_011.cif (5.27 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_012.cif">jm0c01522_si_012.cif (5.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_013.cif">jm0c01522_si_013.cif (4.8 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_014.cif">jm0c01522_si_014.cif (5.2 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_015.cif">jm0c01522_si_015.cif (6.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_016.cif">jm0c01522_si_016.cif (5.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_017.cif">jm0c01522_si_017.cif (5.2 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_018.cif">jm0c01522_si_018.cif (5.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_019.cif">jm0c01522_si_019.cif (5.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_020.cif">jm0c01522_si_020.cif (5.2 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_021.cif">jm0c01522_si_021.cif (5.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_022.cif">jm0c01522_si_022.cif (5.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_023.cif">jm0c01522_si_023.cif (5.2 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_024.cif">jm0c01522_si_024.cif (5.51 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_025.cif">jm0c01522_si_025.cif (4.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_026.cif">jm0c01522_si_026.cif (5.67 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_027.cif">jm0c01522_si_027.cif (5.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_028.cif">jm0c01522_si_028.cif (5.43 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01522/suppl_file/jm0c01522_si_029.cif">jm0c01522_si_029.cif (5.2 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01522&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-1%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01522%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01522" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993617dc06d18f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
