
<html lang="en"     class="pb-page"  data-request-id="070eb5ea-8231-4652-be33-1e17e219ea45"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm501900s;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2015.58.issue-4"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors" /></meta><meta name="dc.Creator" content="Cynthia A.  Morgan" /></meta><meta name="dc.Creator" content="Thomas D.  Hurley" /></meta><meta name="dc.Description" content="Aldehyde dehydrogenases (ALDH) catalyze the irreversible oxidation of aldehydes to their corresponding carboxylic acid. Alterations in ALDH1A1 activity are associated with such diverse diseases as ..." /></meta><meta name="Description" content="Aldehyde dehydrogenases (ALDH) catalyze the irreversible oxidation of aldehydes to their corresponding carboxylic acid. Alterations in ALDH1A1 activity are associated with such diverse diseases as ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 10, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm501900s" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm501900s" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm501900s" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm501900s" /></link>
        
    
    

<title>Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm501900s" /></meta><meta property="og:title" content="Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0012.jpeg" /></meta><meta property="og:description" content="Aldehyde dehydrogenases (ALDH) catalyze the irreversible oxidation of aldehydes to their corresponding carboxylic acid. Alterations in ALDH1A1 activity are associated with such diverse diseases as cancer, Parkinson’s disease, obesity, and cataracts. Inhibitors of ALDH1A1 could aid in illuminating the role of this enzyme in disease processes. However, there are no commercially available selective inhibitors for ALDH1A1. Here we characterize two distinct chemical classes of inhibitors that are selective for human ALDH1A1 compared to eight other ALDH isoenzymes. The prototypical members of each structural class, CM026 and CM037, exhibit submicromolar inhibition constants but have different mechanisms of inhibition. The crystal structures of these compounds bound to ALDH1A1 demonstrate that they bind within the aldehyde binding pocket of ALDH1A1 and exploit the presence of a unique glycine residue to achieve their selectivity. These two novel and selective ALDH1A1 inhibitors may serve as chemical tools to better understand the contributions of ALDH1A1 to normal biology and to disease states." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm501900s"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm501900s">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm501900s&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm501900s&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm501900s&amp;href=/doi/10.1021/jm501900s" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1964-1975</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm5018863" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm501921k" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cynthia+A.++Morgan">Cynthia A. Morgan</a></span></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+D.++Hurley">Thomas D. Hurley</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Department of Biochemistry and Molecular Biology Indiana University School of Medicine 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 317-278-2008. E-mail: <a href="/cdn-cgi/l/email-protection#f7839f82859b928eb79e82d9929382"><span class="__cf_email__" data-cfemail="d6a2bea3a4bab3af96bfa3f8b3b2a3">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm501900s&amp;href=/doi/10.1021%2Fjm501900s" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1964–1975</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 29, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 December 2014</li><li><span class="item_label"><b>Published</b> online</span>10 February 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 February 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm501900s" title="DOI URL">https://doi.org/10.1021/jm501900s</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1964%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCynthia%2BA.%2BMorgan%252C%2BThomas%2BD.%2BHurley%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D58%26issueNum%3D4%26contentID%3Djm501900s%26title%3DCharacterization%2Bof%2BTwo%2BDistinct%2BStructural%2BClasses%2Bof%2BSelective%2BAldehyde%2BDehydrogenase%2B1A1%2BInhibitors%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1975%26publicationDate%3DFebruary%2B2015">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm501900s"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1838</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">40</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm501900s" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Characterization of Two Distinct Structural Classes of Selective Aldehyde Dehydrogenase 1A1 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Cynthia&quot;,&quot;last_name&quot;:&quot;A. Morgan&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;D. Hurley&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;1964-1975&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm501900s&quot;},&quot;abstract&quot;:&quot;Aldehyde dehydrogenases (ALDH) catalyze the irreversible oxidation of aldehydes to their corresponding carboxylic acid. Alterations in ALDH1A1 activity are associated with such diverse diseases as cancer, Parkinson’s disease, obesity, and cataracts. Inhibitors of ALDH1A1 could aid in illuminating the role of this enzyme in disease processes. However, there are no commercially available selective inhibitors for ALDH1A1. Here we characterize two distinct chemical classes of inhibitors that are selective for human ALDH1A1 compared to eight other ALDH isoenzymes. The prototypical members of each structural class, CM026 and CM037, exhibit submicromolar inhibition constants but have different mechanisms of inhibition. The crystal structures of these compounds bound to ALDH1A1 demonstrate that they bind within the aldehyde binding pocket of ALDH1A1 and exploit the presence of a unique glycine residue to achieve their selectivity. These two novel and selective ALDH1A1 inhibitors may serve as chemical tools to better &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501900s&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501900s" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501900s&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501900s" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm501900s&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501900s" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm501900s&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm501900s&amp;href=/doi/10.1021/jm501900s" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm501900s" /></input><a href="/doi/pdf/10.1021/jm501900s" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm501900s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm501900s%26sid%3Dliteratum%253Aachs%26pmid%3D25634381%26genre%3Darticle%26aulast%3DMorgan%26date%3D2015%26atitle%3DCharacterization%2Bof%2BTwo%2BDistinct%2BStructural%2BClasses%2Bof%2BSelective%2BAldehyde%2BDehydrogenase%2B1A1%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D4%26spage%3D1964%26epage%3D1975%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292236" title="Aldehydes">Aldehydes</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jmcmar.2015.58.issue-4/20150226/jmcmar.2015.58.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Aldehyde dehydrogenases (ALDH) catalyze the irreversible oxidation of aldehydes to their corresponding carboxylic acid. Alterations in ALDH1A1 activity are associated with such diverse diseases as cancer, Parkinson’s disease, obesity, and cataracts. Inhibitors of ALDH1A1 could aid in illuminating the role of this enzyme in disease processes. However, there are no commercially available selective inhibitors for ALDH1A1. Here we characterize two distinct chemical classes of inhibitors that are selective for human ALDH1A1 compared to eight other ALDH isoenzymes. The prototypical members of each structural class, CM026 and CM037, exhibit submicromolar inhibition constants but have different mechanisms of inhibition. The crystal structures of these compounds bound to ALDH1A1 demonstrate that they bind within the aldehyde binding pocket of ALDH1A1 and exploit the presence of a unique glycine residue to achieve their selectivity. These two novel and selective ALDH1A1 inhibitors may serve as chemical tools to better understand the contributions of ALDH1A1 to normal biology and to disease states.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Aldehydes are highly reactive compounds that can lead to cellular toxicity through their ability to form adducts with a variety of cellular nucleophiles found in proteins, nucleic acids, as well as small molecule metabolites. In humans, aldehyde detoxication occurs via three main enzyme systems: aldehyde oxidases, aldo-keto reductases, and aldehyde dehydrogenases. The human genome contains at least 19 functional genes for aldehyde dehydrogenases (ALDH) that catalyze the NAD(P)<sup>+</sup>-dependent oxidation of endogenous and exogenous aldehydes to their corresponding carboxylic acids or CoA esters. ALDHs differ in their tissue distribution, subcellular location, structure, as well as preferred substrates and are critical enzymes that contribute to numerous biological functions as well as to the cellular defense against aldehyde toxicity.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> They are involved in the synthesis of critical carboxylic acids including retinoic acid, a key regulator of cell growth and development,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and the neurotransmitter, γ-aminobutyric acid.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> A major role of the ALDH superfamily is protection from aldehyde-induced cytotoxicity.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Oxidative stress often results in lipid peroxidation, generating over 200 aldehydes, including 4-hydroxyhexenal, 4-hydroxynonenal, and malondialdehyde.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> These endogenously generated compounds can carbonylate proteins and have been associated with neurodegenerative disorders<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and aging.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> A variety of drugs, including ethanol and the anticancer drug cyclophosphamide, are metabolized via ALDH-dependent pathways.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In the environment, although some aldehydes have nonanthropogenic sources, motor vehicle exhaust, industrial applications, cigarette smoke, and other human activities are the primary sources for exogenous aldehydes, including formaldehyde, acetaldehyde, and acrolein.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">As a consequence of their critical contributions to aldehyde metabolism, loss of function mutations in ALDH genes are linked to a number of diseases. ALDH2 is the primary enzyme involved in the oxidation of acetaldehyde during ethanol metabolism,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and a single nucleotide polymorphism (SNP) results in an enzymatically crippled protein (ALDH2*2), in which acetaldehyde, derived from ethanol oxidation, accumulates and induces alcohol toxicity.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> Alterations in ALDH1A1 and ALDH2 expression or activity may play a role in Parkinson’s disease through the metabolism of the neurotransmitter dopamine, leading to increased levels of neurotoxic aldehydes, including 3,4-dihydroxyphenylacetaldehyde.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Modulation of ALDH2’s role in dopamine metabolism has been shown to affect cocaine seeking behavior.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Mutations in ALDH3A2 lead to Sjögren–Larsson syndrome, which is characterized by mental retardation, icthyosis, and spastic tetraplegia due to impaired metabolic clearance of sphingosine and plasmalogen metabolites.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> Mutations in other ALDH genes have been linked to pyridoxine-dependent epilepsy (ALDH7A1),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> type II hyperprolinemia resulting in mental retardation and seizures (ALDH4A1 and ALDH18A1),<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> and may possibly contribute to paranoid schizophrenia (ALDH3B1).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> A number of ALDHs have been associated with cancer and/or cancer stem cells, including ALDH1A1, ALDH1A2, ALDH1A3, ALDH1L1, ALDH2, ALDH3A1, ALDH4A1, and ALDH7A1.<a onclick="showRef(event, 'ref7 ref20'); return false;" href="javascript:void(0);" class="ref ref7 ref20">(7, 20)</a> Both ALDH1A1 and ALDH3A1 detoxify some oxazaphosphorine anticancer drugs and decrease the drug’s effectiveness.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> ALDH4A1 is p53-inducible and may minimize cellular damage due to oxidative stress.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> ALDH1A2 is a possible tumor suppressor gene in prostate cancer, likely via the enzyme’s role in retinoid metabolism.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The ALDH2*2 mutation has also been associated with a variety of cancers, possibly due to increased aldehyde-induced DNA damage.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25, 26)</a> Up-regulation of ALDH activity is also common in both normal and cancer stem cells.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Therefore, ALDH is considered a stem cell biomarker, and the ALDEFLUOR assay (Stemcell Technologies, Vancouver, Canada) uses this ALDH activity as a means to identify cancer stem cells.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p">ALDH1A1 (retinaldehyde dehydrogenase 1, RALDH1) is a highly conserved, cytosolic homotetramer (∼55 kDa monomers) that is widely expressed and found in numerous tissues, including brain, liver, kidneys, adipose, eye lens, and retina. A key role of ALDH1A1 is the oxidation of retinaldehyde to retinoic acid (RA), forming transcriptional regulators critical for normal cell growth and differentiation.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Both the substrate (retinaldehyde) and product (RA) are important for normal biological processes, including vision, cellular differentiation, and immune function.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> ALDH1A1 shares greater than 70% sequence identity to both ALDH1A2 and ALDH1A3 (RALDH2 and RALDH3, respectfully) and both also convert retinaldehyde to RA, but their roles may be more confined to embryogenesis and stem cell development.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> ALDH1A1 knockout mice are viable, but ALDH1A2 and ALDH1A3 knockout mice are lethal early in development or shortly after birth.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In addition to its similarity with other retinaldehyde dehydrogenases, ALDH1A1 also shares nearly 70% sequence identity to the mitochondrial ALDH2 and ALDH1B1 enzymes but shares less than 50% sequence identity with members of other ALDH families, including ALDH3A1, ALDH4A1, and ALDH5A1.</div><div class="NLM_p">The structure of mammalian ALDHs are similar, functioning as homomultimers, and each monomer contains three structural domains: a catalytic domain, a cofactor binding domain and an oligomerization domain.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The global structural similarities support identical active site residues whose contributions to aldehyde oxidation are well established. In addition to aldehyde oxidation activity, some ALDHs, including ALDH1A1, also possess an NAD<sup>+</sup>-independent esterase activity,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and this esterase activity uses the same active site residues as does the dehydrogenase activity.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Despite similarities in structure and function, as well as some overlap in substrate preferences, the isoenzymes of the ALDH family of proteins have evolved different aldehyde binding sites due to substitutions of the residues lining their respective substrate binding tunnels.<a onclick="showRef(event, 'ref31 ref34'); return false;" href="javascript:void(0);" class="ref ref31 ref34">(31, 34)</a> These differences have allowed for the development of activators and inhibitors of various isoenzymes as therapeutics. For example, disulfiram (commercial: Antabuse) is an inhibitor of both ALDH1A1 and ALDH2, and this drug can be used to treat alcoholism<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and potentially for cocaine addiction.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Alda-1 is a selective activator of ALDH2 and may offer cardiac protection following ischemic events by decreasing cytotoxic aldehyde levels.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Because multiple cancers have increased ALDH1A1 and/or ALDH3A1 activity leading to increased resistance to chemotherapeutic agents, selective inhibitors of these two isoenzymes are also of interest.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36, 37)</a></div><div class="NLM_p last">Available inhibitors of ALDHs have been recently reviewed.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> At this time, there are no commercially available inhibitors capable of distinguishing between ALDH1A1 and the other highly similar ALDH isoenzymes, ALDH1A2, ALDH1A3, ALDH1B1, and ALDH2. For instance, the control inhibitor used in the ALDEFLUOR assay, diethylaminobenzaldehyde (DEAB), is an effective inhibitor of at least three of these isoenzymes.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38, 39)</a> From a high-throughput screen (HTS),<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> we have identified two classes of ALDH1A1 inhibitors and have determined the structural and kinetic basis for their selectivity toward ALDH1A1.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76705" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76705" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Because the esterase and dehydrogenase activity of ALDH1A1 utilize similar active site residues, modulators of esterase function as convenient surrogates for the dehydrogenase activity, as has been shown previously with the discovery of ALDH3A1 inhibitors.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The esterase assay was selected for the screening assay instead of aldehyde oxidation to (1) minimize spectral overlap with the absorbance characteristic of the compounds in the library, and (2) eliminate compounds that compete with structurally conserved elements of the cofactor binding site. CM026 and CM037 emerged from the HTS as esterase modulators of ALDH1A1.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> CM026 (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A) has an atomic mass of 442.5 Da, and CM037 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A) has an atomic mass of 431.6 Da. These compounds have no structural similarity to any known, commercially available aldehyde dehydrogenase modulators.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of CM026, a selective inhibitor of ALDH1A1. (A) Structure of CM026. (B) Selectivity of 20 μM of CM026 with respect to nine ALDH isoenzymes. (C) IC<sub>50</sub> of CM026 with ALDH1A1.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (D) Lineweaver–Burk representation of noncompetitive inhibition for CM026 (0–4 μM) versus varied acetaldehyde (100–800 μM) at fixed concentration of NAD<sup>+</sup> (800 μM). (E) Lineweaver–Burk representation of uncompetitive inhibition for CM026 (0–3 μM) versus varied NAD<sup>+</sup> (25–250 μM) at fixed concentration of propionaldehyde (200 μM). The IC<sub>50</sub> curves and Lineweaver–Burk plots represent one of three experiments performed for each condition, with each point the mean/SEM of three data points at each concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characterization of CM037, a selective inhibitor of ALDH1A1. (A) The structure of CM037 with an atomic mass of 431.6 Da. (B) Selectivity of 20 μM of CM037 with respect to nine ALDH isoenzymes. (C) IC<sub>50</sub> of CM037 with ALDH1A1.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The IC<sub>50</sub> curve represents one of three experiments performed, with each point the mean/SEM of three data points at each concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Kinetic Characterization of CM026 and CM037</h3><div class="NLM_p">CM026 is a selective inhibitor for ALDH1A1 versus eight other ALDH isoenzymes examined. At a concentration of 20 μM, CM026 had no effect on seven other human ALDH isoenzymes (ALDH1A2, ALDH1A3, ALDH1B1, ALDH2, ALDH3A1, ALDH4A1, and ALDH5A1) as well as the carboxyl-terminal ALDH domain of rat ALDH1L1 (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B). At a concentration of 100 μM, CM026 modestly increased aldehyde oxidation catalyzed by ALDH1A2, ALDH1A3, and ALDH1B1. CM026 has good potency toward ALDH1A1 (IC<sub>50</sub> of 0.80 ± 0.06 μM,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C) for an initial hit compound. Complete inhibition of ALDH1A1 was not observed. CM026 has a noncompetitive partial mode of inhibition with respect to varied acetaldehyde, with a <i>K</i><sub>i</sub> of 0.80 ± 0.16 μM and β = 0.15 ± 0.03, indicating maximum inhibition at 0.15(<i>V</i><sub>max</sub>) (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>D). CM026 had an uncompetitive partial mode of inhibition with respect to varied NAD<sup>+</sup> and exhibited a <i>K</i><sub>i</sub> of 0.72 ± 0.03 μM and β = 0.10 ± 0.03 (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>E).</div><div class="NLM_p">It is possible that the size of the R-groups attached to the xanthine ring influences whether it is possible to slowly bind and release small aldehydes at the catalytic nucleophile, and such substrate length dependency was also observed with the activator Alda-1 in ALDH2.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p last">CM037 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A) was also selective for ALDH1A1 at 20 μM versus eight other ALDH isoenzymes tested (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B). While 20 μM CM037 had little effect on most ALDH isoenzymes tested, ALDH1A3 was inhibited approximately 20% at this concentration. Higher concentrations were not tested due to solubility limits of CM037 under these assay conditions. CM037 exhibits an IC<sub>50</sub> = 4.6 ± 0.8 μM toward ALDH1A1 versus the substrate propionaldehyde (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C) and a competitive mode of inhibition with respect to varied substrate acetaldehyde and an average <i>K</i><sub>i</sub> of 0.23 ± 0.06 μM from three independent inhibition experiments.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Structure–Activity Relationship for Analogues of CM026</h3><div class="NLM_p">There were 77 compounds in the initial screening hit list that were structurally similar to CM026.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> We have tested 17 members of this compound class, and they exhibit good selectivity for ALDH1A1 compared to ALDH1A2, ALDH1A3, ALDH1B1, ALDH2, and ALDH3A1 (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). At high concentrations (100 μM), some compounds activated the aldehyde oxidation activity of other ALDH1 enzymes tested (e.g., CM026), but in most cases these effects were observed as the solubility limit of the compounds in the assay system were approached, precluding a full dose–response study. At 20 μM, none of the compounds demonstrated activation exceeding 10% of control activity, suggesting any activation is of modest potency and not likely to impact these enzyme activities in vivo. These compounds share a common xanthine core structure with theophylline and caffeine, but neither of these stimulants affected ALDH1A1 activity at concentrations up to 250 μM, indicating that substituents at the R1 and R2 positions are necessary for ALDH1A1 inhibition (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Halogens on either R-group were not tolerated. CM053 was the most potent analogue examined, with an IC<sub>50</sub> = 210 ± 40 nM and a <i>K</i><sub>i</sub> = 96 ± 14 nM with noncompetitive tight mode of inhibition compared to varied substrate acetaldehyde (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). CM028 shares the same R2 group as CM053, but the isopentyl group at R1 has been replaced with a phenylpropyl group. CM028 is less potent, with an IC<sub>50</sub> = 2.0 ± 0.1 μM, and exhibits a <i>K</i><sub>i</sub> = 240 ± 40 nM with competitive tight mode of inhibition. The lower potency suggests that the phenylpropyl group might present steric conflicts not found with the smaller phenyl and isopentyl groups (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Unlike CM026, CM028 and CM053 demonstrated complete inhibition, which confirms that the nature of the R1 group alone does not influence the final extent of inhibition because both CM026 and CM053 share the same R1 group.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of CM053. (A) IC<sub>50</sub> of CM053 with ALDH1A1. (B) Lineweaver–Burk representation of noncompetitive tight inhibition for CM053 (0–300 nM) versus varied acetaldehyde (100–800 μM) at fixed concentration of NAD<sup>+</sup> (1000 μM). Both curves represent one of three experiments performed, with each point the mean/SEM of three data points at each concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR for CM026 Analogues<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0011.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">At 100 μM compound, NI stands for no inhibition and NI(A) indicates no inhibition but activation (* indicates <70% maximum inhibition).</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Crystal Structure of ALDH1A1 Complexed with CM026, CM053, and CM037</h3><div class="NLM_p">To determine the mechanism that underlies the ability of these compounds to selectively inhibit ALDH1A1, we used X-ray crystallography to determine the structure of the enzyme–compound complexes. We solved the crystal structures of human ALDH1A1 in complexes with CM026, CM053, and CM037 to resolutions between 1.80 and 1.95 Å (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). For CM026 and CM037, the naturally occurring N121S polymorphic variant of ALDH1A1 was used,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> while wtALDH1A1 was used for the structure with bound CM053. A comparison of the respective α carbons in the structure of the N121S·CM026 to those in the WT·CM053 generated an RMSD of 0.12 Å, indicating a high degree of similarity between WT and the N121S mutant as expected because the two have very similar kinetic behavior.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> CM026 binds near the solvent exposed exit of the substrate-binding site (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). The xanthine rings for both CM026 and CM053 are parallel to and approximately 3.6 Å from Tyr297, with which it interacts via hydrophobic π-stacking interactions. Four residues form hydrogen bonds with CM026; the xanthine ring interacts with His293, Cys302, and Gly458, while Trp178 interacts with the ketone group on R2. The isopentyl group of R1 projects toward Cys303 and fills much of the hydrophobic space bounded by Phe171 and Phe466. CM053 differs from CM026 only in its R2 group, which can form hydrogen bonds with two residues, Trp178 and Val460 (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). CM037 binds at a similar location to the CM026 compounds, but its long axis is oriented almost orthogonal to that of CM026 and CM053 (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A). Most of the tricyclic ring of CM037 is in a hydrophobic pocket formed by Phe171, Val460, and Phe466, with a potential hydrogen bond between the ring system’s carbonyl oxygen atom and the side chain of Cys302 (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B). The biggest structural adaptation to CM037 binding is the movement of Trp178 away from the substrate-binding site to accommodate the benzyl ring of CM037 (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C). This conformational movement appears to be dynamic and impacts the observed electron density for both the benzyl group of CM037 and of Trp178. Trp178 is well ordered in all other structures determined of human ALDH1A, including our CM026 and CM053 structures, but has weak density for the benzyl moiety of the indole ring in this complex (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>D). We would suggest that optimization of CM037 could be achieved by altering the thiophene and benzyl ring systems to alleviate these steric conflicts. To better understand the selectivity of these compounds for ALDH1A1, we compared these structures against human ALDH2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CW3">1CW3</a><a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a>), ALDH3A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SZA">3SZA</a><a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a>), and ALDH4A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V9G">3V9G</a><a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a>) and identified a critical glycine (Gly458) that is present near the xanthine ring binding site in ALDH1A1. This glycine is replaced by larger amino acid side chains in the other three human structures examined as well as in sheep ALDH1A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BXS">1BXS</a><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a>). In rat ALDH1A2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BI9">1BI9</a><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>), which shares 97% sequence identity to human ALDH1A2, this location is part of a small disordered loop not observed in the crystal structure.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Using sequence alignments of the human genes, Gly458 in ALDH1A1 is replaced by an asparagine in ALDH1A2, ALDH1A3, and ALDH1B1, an aspartate in ALDH2, and an isoleucine in ALDH3A1 (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>A). As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B, these side chains would interfere with the position of the xanthine ring, effectively eliminating the ability of these analogues to bind to any isoenzyme but ALDH1A1.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Data Collection and Refinement Statistics<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">ALDH1A1-CM026 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a></th><th class="colsep0 rowsep0" align="center">ALDH1A1-CM053 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WPN">4WPN</a></th><th class="colsep0 rowsep0" align="center">ALDH1A1-CM037 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L">4X4L</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left">Data Collection</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>422</td><td class="colsep0 rowsep0" align="left"><i>P</i>422</td><td class="colsep0 rowsep0" align="left"><i>P</i>422</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cell dimensions</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">109, 109, 83</td><td class="colsep0 rowsep0" align="left">109, 109, 83</td><td class="colsep0 rowsep0" align="left">109, 109, 83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (deg)</td><td class="colsep0 rowsep0" align="left">90, 90, 90</td><td class="colsep0 rowsep0" align="left">90, 90, 90</td><td class="colsep0 rowsep0" align="left">90, 90, 90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">50–1.80</td><td class="colsep0 rowsep0" align="left">50–1.95</td><td class="colsep0 rowsep0" align="left">50–1.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub></td><td class="colsep0 rowsep0" align="left">0.082 (0.59)</td><td class="colsep0 rowsep0" align="left">0.11 (0.66)</td><td class="colsep0 rowsep0" align="left">0.058 (0.70)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ<sub><i>i</i></sub></td><td class="colsep0 rowsep0" align="left">22.7 (4.7)</td><td class="colsep0 rowsep0" align="left">18.3 (3.7)</td><td class="colsep0 rowsep0" align="left">27.4 (3.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99 (100)</td><td class="colsep0 rowsep0" align="left">99 (100)</td><td class="colsep0 rowsep0" align="left">99 (100)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">11.7 (11.7)</td><td class="colsep0 rowsep0" align="left">9.3 (6.9)</td><td class="colsep0 rowsep0" align="left">8.5 (8.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of reflns</td><td class="colsep0 rowsep0" align="left">44544</td><td class="colsep0 rowsep0" align="left">35048</td><td class="colsep0 rowsep0" align="left">40517</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub>/<i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">0.19/0.22</td><td class="colsep0 rowsep0" align="left">0.19/0.24</td><td class="colsep0 rowsep0" align="left">0.19/0.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of atoms</td><td class="colsep0 rowsep0" align="left">4109</td><td class="colsep0 rowsep0" align="left">4080</td><td class="colsep0 rowsep0" align="left">4066</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">3833</td><td class="colsep0 rowsep0" align="left">3858</td><td class="colsep0 rowsep0" align="left">3806</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand/ion</td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">241</td><td class="colsep0 rowsep0" align="left">189</td><td class="colsep0 rowsep0" align="left">180</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.008</td><td class="colsep0 rowsep0" align="left">0.009</td><td class="colsep0 rowsep0" align="left">0.010</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (deg)</td><td class="colsep0 rowsep0" align="left">1.28</td><td class="colsep0 rowsep0" align="left">1.30</td><td class="colsep0 rowsep0" align="left">0.135</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Numbers in parentheses represent values of highest resolution shell.</p></div></div></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure of ALDH1A1 N121S with CM026 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>). (A) CM026 binds in the active site near Cys 303. Key residues are colored based on element, and Gly458 is shown in orange. (B) Two-dimensional representation of the key hydrogen bonds, illustrated with red dashed lines, and hydrophobic interactions, illustrated with black arcs, between ALDH1A1 and CM026. (C) Electron density maps of CM026, with the original <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in green contoured at 2.5 standard deviations and the final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in gray contoured at 1.0 standard deviations. A and C are generated in Pymol, while B is generated in ChemDraw.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure of human ALDH1A1 with CM053 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WPN">4WPN</a>). (A) Two dimensional representation of the key hydrogen bonds, illustrated with red dashed lines, and hydrophobic interactions, illustrated with black arcs, between ALDH1A1 and CM053. (B) Electron density maps of CM053 with the original <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in green contoured at 2.5 standard deviations and the final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in gray contoured at 1.0 standard deviations. A is generated in ChemDraw, and B is generated in Pymol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structure of ALDH1A1 N121S with CM037 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L">4X4L</a>). (A) CM037 binds in the active site near Cys 303. Key residues are colored based on element, and Gly458 is shown in orange. (B) Two-dimensional representation of the hydrophobic interactions, illustrated with black arcs, between ALDH1A1 and CM037. (C) Binding of CM037 induces structural changes in ALDH1A1 (in blue) compared to apo-ALDH1A1 (in gray), particularly at W178. NADH binding (in cyan) induces conformational changes at the cofactor binding site, as seen here with E269. (D) Electron density maps of CM037, with the original <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in green contoured at 2 standard deviations and the final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in gray contoured at 1.0 standard deviations. A, C, and D are generated in Pymol, while B is generated in ChemDraw.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structural basis of selectivity of CM026 for ALDH1A1. (A) Multiple sequence alignment in the region of ALDH1A1 Gly458 to the mature form of ALDH2. (B) Structure of ALDH1A1 with bound CM026 (blue) compared to ALDH2 (cyan) and ALDH3A1 (gray), indicating that a bulky amino acid such as the Asp of ALDH2 and Ile of ALDH3A1 would clash with CM026 and prevent the compound from inhibiting the enzyme. Sequence alignment was performed using NCBI delta-BLAST, while structural alignment was performed using least-square fit (LSQ) in Coot. B is generated in Pymol. PDB codes: ALDH1A1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>), ALDH2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VLE">2VLE</a>), and ALDH3A1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SZA">3SZA</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Characterization of ALDH1A1 G458N Mutant</h3><div class="NLM_p">To confirm whether Gly458 in ALDH1A1 directly impacts the selectivity of the CM026 analogues and CM037 for ALDH1A1, we mutated the glycine at this position to asparagine, as found in ALDH1A2 and ALDH1A3. We determined the kinetic parameters for acetaldehyde oxidation for both the wild-type and G458N enzymes (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). This mutation did not dramatically affect the enzyme’s catalytic efficiency for aldehyde oxidation. However, when Gly458 is mutated to asparagine, CM026 no longer inhibits the enzyme at concentrations up to 100 μM and none of the CM026 analogues inhibited the mutant more than 25% (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). Similarly, 20 μM CM037 no longer inhibited the G458N enzyme. However, the nonselective inhibitors DEAB and CM302<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> both inhibit G458N, with IC<sub>50</sub> values of 0.52 ± 0.10 and 3.1 ± 0.3 μM, respectively<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). These data support the hypothesis that the substrate-binding site, and in particular Gly458, determines the selectivity of both the CM026 and CM037 classes of compounds for ALDH1A1 and that bulkier side chains at this position in ALDH1A2, ALDH1A3, ALDH1B1, ALDH2, and ALDH3A1 occludes their binding to these ALDH isoenzymes.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Kinetic Parameters of ALDH1A1 WT and Mutant G458N</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>M</sub><sup>acetaldehyde</sup> (μM)</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>cat</sub>/<i>K</i><sub>M</sub> (min<sup>–1</sup> μM<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>i</sub><sup>CM026</sup> (μM)</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>i</sub><sup>CM037</sup> (μM)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub><sup>CM026</sup> (μM)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub><sup>CM037</sup> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub><sup>CM302</sup> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub><sup>DEAB</sup> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WT</td><td class="colsep0 rowsep0" align="char" char="±">177 ± 19</td><td class="colsep0 rowsep0" align="char" char="±">0.18 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.16</td><td class="colsep0 rowsep0" align="char" char="±">0.23 ± 0.06<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></td><td class="colsep0 rowsep0" align="center">0.80 ± 0.06<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></td><td class="colsep0 rowsep0" align="center">4.6 ± 0.8<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></td><td class="colsep0 rowsep0" align="char" char="±">1.0 ± 0.1<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></td><td class="colsep0 rowsep0" align="char" char="±">0.057 ± 0.005<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G458N</td><td class="colsep0 rowsep0" align="char" char="±">85.8 ± 1.6</td><td class="colsep0 rowsep0" align="char" char="±">0.21 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="center">NI</td><td class="colsep0 rowsep0" align="center">NI</td><td class="colsep0 rowsep0" align="char" char="±">3.1 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">0.52 ± 0.10</td></tr></tbody></table></div></div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Selectivity of compounds for WT vs G458N mutant. For CM026 and its analogues, 100 μM of compound was used. For CM037 and CM302, 20 μM of compound was used. Each value is mean/SEM (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26383" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26383" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Aldehyde dehydrogenases contribute to a variety of biological processes and disease states. ALDH1A1, in particular, has been linked to such diverse diseases as cancer, Parkinson’s disease, obesity, and cataracts. Therefore, selective inhibitors of ALDH1A1 would be of tremendous value to understanding the roles of this enzyme in both normal and disease processes. However, great structural and functional similarity exists, especially within the ALDH1 family, with five members sharing approximately 70% protein sequence identity or higher plus significant overlap in substrate utilization. To date, there are no ALDH1A1-selective inhibitors commercially available. Although comparisons of available ALDH structures indicate a high degree of overlap, there exist distinct surface topographies that may enable development of selective inhibitors (Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>). ALDH1A1 possesses a wider opening leading to the active site, whereas ALDH2 has a much more constricted, cylindrical shaped site. ALDH3A1 possesses a wider inner vestibule near the catalytic nucleophile, with a much narrower and curved entryway. The narrower entries in ALDH2 and ALDH3A1 eliminate the binding site for the CM026 and CM037 classes of inhibitors, in large part due to the side chain present at the position equivalent to Gly458. ALDH4A1 possess a serine at the position equivalent to Gly458. However, the loop structure in which it resides is different enough from ALDH1A1 to prevent compound binding. Similar to daidzin, both the CM026 and CM037 classes of compounds are planar, multiringed structures that adopt binding modes that take advantage of the topological characteristics unique to the ALDH isoenzyme toward which they demonstrate selectivity and neither of these new compounds can be accommodated in the restricted substrate binding sites of ALDH3A1 or ALDH2. Daidzin is a strong inhibitor of ALDH2, but it also inhibits ALDH1A1.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Daidzin binds in a similar location to CM026, but comparison of the structure of ALDH1A1 and ALDH2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VLE">2VLE</a><a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a>) bound to these respective compounds shows that the two compounds bind nearly perpendicular to each other in their respective binding modes. In particular, aligning the ALDH2–daidzin structure to ALDH1A1 demonstrates that daidzin binding need not engage ALDH1A1 near its unique G458 site (Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a>). As discussed by Lowe et al., the tighter binding pocket of ALDH2 compared to ALDH1A1 favors more intimate interactions between the ALDH2 and daidzin, increasing specificity.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> CM026 exploits a binding pocket that is not accessible in other ALDH isoenzymes due to their larger amino acid side chains at position 458.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Comparison of the active site topography of human ALDH1A1, ALDH2, and ALDH3A1. The three isoenzymes were aligned using LSQ in Coot and the surface figures generated via Pymol. The active site cysteine is shown in red for all three isoenzymes. G458 in ALDH1A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WJ9">4WJ9</a>) and its equivalent residues D457 in ALDH2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=ICW3">ICW3</a>) and I391 in ALDH3A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SZA">3SZA</a>) are shown in yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Comparison of the binding of CM026 to ALDH1A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>) with the binding of daidzin to ALDH2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VLE">2VLE</a>). ALDH1A1 is shown in light blue with CM026 in dark blue. ALDH2 is shown in light green with daidzin in dark green. The active site cysteine is shown in red for both isoenzymes. Although both compounds bind in the same location, they bind in a different orientation enabling CM026 to inhibit ALDH1A1 but not ALDH2 due to steric hindrance of D474 (D457 in the mature sequence, in orange), while daidzin inhibits both isoenzymes. Figure generated in Pymol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast to CM026, the directionality of CM037 binding within the ALDH1A1 active site resembles that of daidzin in ALDH2. However, the 4-oxo group and the branched structure of CM037 near the exit of the substrate binding site exploits the same G458 region for selectivity. Our structural studies identified Gly458 as a major contributor to the selectivity of the CM026 and CM037. Sequence alignments of the other 18 human ALDH isoenzymes indicate that the only other family member with a glycine in this position is ALDH16A1. However, the function of ALDH16A1 is unknown and has a three residue deletion in the active site loop which eliminates the conserved Cys nucleophile, suggesting this protein may have functions that are independent of aldehyde dehydrogenase activity.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The presence of a nonglycine residue at this position does not adversely affect catalytic activity toward small substrates or inhibition by the nonselective compounds, DEAB and CM302. Consequently, these compounds identified via an esterase-based high throughput screen successfully exploited a unique structural feature found primarily in primate ALDH1A1 enzymes, which further validates the use of the esterase activity as a screening tool for ALDH isoenzymes.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div class="NLM_p last">Selective inhibitors of ALDH1A1 are needed to understand the role of this enzyme in both normal and disease processes. As recently reviewed by Ma and Allan, a number of ALDH family members have been associated with both normal stem cells and cancer stem cells.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The viability of the ALDH1A1–/– mice suggests that ALDH1A1 is nonessential or can be compensated for by other family members during growth and development.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> On the other hand, ALDH1A1 is considered a biomarker for lung,<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> ovarian,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> prostate,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and a number of other cancers. Ovarian cancer cells form spheroids, cellular aggregates that aid metastasis.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Recently ALDH1A1 was shown to be upregulated in ovarian cancer spheroids.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> This study utilized our ALDH1A1-selective compound CM037 (published as A37) to disrupt spheroid formation and reduce cell viability, supporting the hypothesis that ALDH1A1 could serve as a target to improve cancer outcomes.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> ALDH1A1 and ALDH3A1 are both involved in the metabolism of the cancer drug cyclophosphamide, metabolizing the active compound to a less active form and contributing to drug resistance. ALDH3A1 inhibitors have been shown to increase sensitivity to the cyclophosphamide analogue mafosphamide when combined in cell lines with high ALDH3A1 expression.<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55, 56)</a> ALDH1A1 could serve as a similar target to minimize cyclophosphamide resistance in cancers with high ALDH1A1 levels. The functional role that ALDH1A1 contributes to stem cells and cancer metastasis is not understood. However, the discovery of these two novel classes of selective ALDH1A1 inhibitors may serve as chemical tools to uncover those roles in both normal and diseased cells.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81487" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81487" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Materials</h3><div class="NLM_p last">Reagents, including acetaldehyde, propionaldehyde, <i>para</i>-nitrophenylacetate, NAD<sup>+</sup>, and buffers were all purchased from Sigma-Aldrich unless otherwise noted. Compounds were purchased from ChemDiv Corp. (San Diego, CA) and were >95% pure based on the NMR spectra provided by the vendor. Following receipt from the vendor, their chemical identities were further verified by LC/MS in the Department of Chemistry, Indiana University–Purdue University, Indianapolis, and used without further purification.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> ALDH Expression and Purification</h3><div class="NLM_p last">ALDH1A1, ALDH1A2, ALDH1A3, ALDH1B1, ALDH2, and ALDH3A1 were produced and purified as previously described.<a onclick="showRef(event, 'ref36 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref36 ref56 ref57">(36, 56, 57)</a> The ALDH1A1 protein used for the ALDH1A1–CM026 crystal contained an Asn-to-Ser SNP at residue 121.<a onclick="showRef(event, 'ref40 ref58'); return false;" href="javascript:void(0);" class="ref ref40 ref58">(40, 58)</a> Unless where noted otherwise, ALDH1A1 WT protein was used for all aldehyde oxidation assays and for the ALDH1A1–CM053 structure. The full length cDNA for human ALDH4A1 and ALDH5A1 were generously provided by Daria Mochly-Rosen. ALDH4A1 was subcloned into the pET-28a expression plasmid and ALDH5A1 was in pTrcHis-Topo. The carboxyl terminal ALDH domain of rat ALDH1L1 was generously provided by Sergey Krupenko in the pRSET expression plasmid. ALDH1L1, ALDH4A1, and ALDH5A1 were expressed and purified as previously described for ALDH3A1<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> with the following modifications: (1) for ALDH1L1 and ALDH5A1, the medium contained 100 μg/mL ampicillin, (2) cells were lysed via three passages through a microfluidizer (DivTech Equipment), and (3) a single passage on a nickel-NTA column was used for purification, without the second Q-sepharose column used to purify ALDH3A1. Purified enzymes used for kinetics were stored at −80 °C. ALDH1A1 used for crystallization was stored at −20 °C in a 50% (v/v) solution with glycerol and dialyzed against 10 mM Na<sup>+</sup>-ACES pH 6.6 and 1 mM dithiothreitol at 4 °C.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Generation of Wild-Type ALDH1A1 and the G458N Mutant</h3><div class="NLM_p last">The QuikChange site-directed mutagenesis protocol was used to make point mutations. First, wtALDH1A1 was generated from the Weiner N121S polymorphism as previously described.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> A point mutation of wtALDH1A1 was performed to produce the G458N mutant. The ALDH1A1 G458N mutant was constructed using the forward primer 5′-GTG GGT GAA TTG CTA TAA CGT GGT AAG TGC CCAG-3′ and its complement. This G458N mutant was purified in the same way as other ALDH1A1 proteins (WT, N121S). G458N was stored at 2 mg/mL and 8 mg/mL at −80 °C. Kinetic experiments for G458N were performed in the same manner as WT protein.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Aldehyde Dehydrogenase Activity Assay</h3><div class="NLM_p last">Dehydrogenase activity of purified recombinant ALDH1A1, ALDH1A2, ALDH1A3, ALDH1B1, ALDH2, and ALDH3A1 was assayed spectrophotometrically by monitoring the formation of NADH at 340 nm (molar extinction coefficient of 6220 M<sup>–1</sup> cm<sup>–1</sup>) on a Beckman DU-640 or Cary 300 Bio UV–vis spectrophotometer. Characterization of ALDH1A1 WT and ALDH1A1 G458N were performed by covarying acetaldehyde and NAD<sup>+</sup> concentrations in reactions containing 150–300 nM enzyme, 50–500 or 20–200 μM acetaldehyde, and 50–500 μM NAD<sup>+</sup> in 50 mM sodium BES, pH 7.5, at 25 °C. Reactions were initiated by adding enzyme. Selectivity of compounds for dehydrogenase activity of ALDH1A1, ALDH1A2, ALDH1A3, ALDH1B1, and ALDH2 were measured in a solution containing 100–200 nM enzyme, 200 μM NAD<sup>+</sup>, 1% DMSO, and 100 μM propionaldehyde (nonsaturating for all enzymes tested, except ALDH2) in 50 mM sodium BES. ALDH3A1 activity was measured under the following conditions: 25 nM enzyme, 200 μM NAD<sup>+</sup>, 1% DMSO, and 300 μM benzaldehyde (∼<i>K</i><sub>M</sub>) in either 100 mM sodium phosphate buffer, pH 7.5, or 50 mM sodium BES, pH 7.5. All assays were performed at 25 °C and were initiated by the addition of the aldehyde substrate following a 2 min preincubation with compound and cofactor. Selectivity was initially tested using 20 μM compound, with ALDH1A1-selective modulators further tested using 100 μM compound. IC<sub>50</sub> values for propionaldehyde oxidation were calculated by varying the concentration of the compounds from 0 to 200 μM. After a 2 min preincubation with compound and NAD<sup>+</sup>, all reactions were initiated by the addition of propionaldehyde. Data were fit to the four parameter EC<sub>50</sub> equation using SigmaPlot (StatSys v12.3). The values represent the average of three independent experiments (each <i>n</i> = 3). The mode of inhibition was determined via steady-state kinetics by covarying inhibitor and substrate concentrations at fixed concentration of the second substrate. All reactions contained 100–150 nM ALDH1A1 and 1% DMSO in 50 mM sodium BES, pH 7.5 at 25 °C. When cofactor NAD<sup>+</sup> was varied, the reactions contained 200 μM propionaldehyde and 20–200 μM or 25–250 μM NAD<sup>+</sup> (<i>K</i><sub>m</sub> = 50 μM). When acetaldehyde was varied, the reactions contained 800 or 1000 μM NAD<sup>+</sup> and 100–800 μM acetaldehyde (<i>K</i><sub>m</sub> = 180 μM). All data were fit to competitive, noncompetitive, uncompetitive, and mixed inhibition models using both single substrate–single inhibitor or tight binding inhibition programs in SigmaPlot (StatSys v12.3). The appropriate model was selected through analysis of goodness-of-fit and the residuals of those fits. The values represent the average of three independent experiments (each <i>n</i> = 3) using at least two protein preps. In Sigmaplot, the data were best fit to the following equations:<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/jm-2014-01900s_m001.gif" alt="" /></img></span><span class="NLM_disp-formula" id="ueq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/jm-2014-01900s_m002.gif" alt="" /></img></span><span class="NLM_disp-formula" id="ueq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/jm-2014-01900s_m003.gif" alt="" /></img></span><span class="NLM_disp-formula" id="ueq4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/jm-2014-01900s_m004.gif" alt="" /></img></span></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Crystallization and Structure Determination of ALDH1A1 and Its Complexes with Inhibitors</h3><div class="NLM_p last">Crystals of ALDH1A1 at 3–5 mg/mL concentration were equilibrated against a crystallization solution of 100 mM Na<sup>+</sup> BisTris, pH 6.2–6.8, 8–12% PEG3350 (Hampton Research), 200 mM NaCl, and 5–10 mM YbCl<sub>3</sub> at 25 °C. For the enzyme·CM026 and enzyme·CM037 complex, ALDH1A1·N121S was used, while wtALDH1A1 was used for the ALDH1A1·CM053 complex. CM026 and CM053 complexes were prepared by soaking crystals overnight with crystallization solution containing 500 μM inhibitor with 2% (v/v) DMSO. The CM037 complex was prepared by soaking crystals with crystallization solution containing 500 μM inhibitor, 1% (v/v) DMSO, and 1 mM NAD<sup>+</sup> for 5 h. Cryoprotection of the crystals for flash-freezing utilized 20% (v/v) ethylene glycol in the ligand soaking solution. Diffraction data were collected at Beamline 19-ID for the ALDH1A1–CM026 complex and Beamline 19-BM for the ALDH1A1–CM053 complex, both operated by the Structural Biology Consortium at the Advanced Photon Source (APS), Argonne National Laboratory. For the ALDH1A1–CM037 complex, data were collected at Beamline 23-ID-D (GM/CA), sponsored by National Institute of General Medical Sciences and National Cancer Institute of the National Institutes of Health at APS. All diffraction data were indexed, integrated, and scaled using either the HKL2000 or HKL3000 program suites.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The CCP4 program suite<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> was used for molecular replacement and refinement, using human apoALDH1A1 structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WJ9">4WJ9</a>) as a model for both structures. The molecular graphics application Coot<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> was used for model building and Translation/Libration/Screw Motion Determination  (TLSMD) server was used to determine the appropriate TLS tensors for refinement of the protein.<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62, 63)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/jm501900s" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41685" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41685" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas D. Hurley</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry
and Molecular Biology Indiana University
School of Medicine 635
Barnhill Drive, Indianapolis, Indiana 46202, United
States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#85f1edf0f7e9e0fcc5ecf0abe0e1f0"><span class="__cf_email__" data-cfemail="582c302d2a343d2118312d763d3c2d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cynthia A. Morgan</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry
and Molecular Biology Indiana University
School of Medicine 635
Barnhill Drive, Indianapolis, Indiana 46202, United
States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Thomas D. Hurley holds significant financial equity in SAJE Pharma, LLC. However, none of the work described in this study is related to, based on or supported by the company.<br /></br></div></li></ul></div><div class="ack" id="ACK-d2748e1692-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27615" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27615" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Lan Chen, Ph.D. and the IU Chemical Genomics Core Facility for assistance with HTS, Dr. Bibek Parajuli, Lanmin Zhai, and Cameron Buchman for assistance producing and purifying ALDH enzymes, and Dr. Karl Dria for assistance with LC/MS. Special thanks to Dr. Sergey Krupenko at the University of North Carolina and Dr. Daria Mochly-Rosen at Stanford for providing ALDH expression contructs. Results shown in this report are derived from work performed at Argonne National Laboratory, Structural Biology Center at the Advanced Photon Source. Argonne is operated by UChicago Argonne, LLC, for the U.S. Department of Energy, Office of Biological and Environmental Research under contract DE-AC02-06CH11357. GM/<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="b9faf8f9f8e9ea">[email protected]</a> has been funded in whole or in part with Federal funds from the National Cancer Institute (ACB-12002) and the National Institute of General Medical Sciences (AGM-12006). This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357. This work was supported by the U.S. National Institutes of Health (grants R01AA018123 to T.D.H.).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i18" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i18"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i19" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i19"> Abbreviations Used</h2><tr><td class="NLM_term">ACES</td><td class="NLM_def"><p class="first last"><i>N</i>-(2-acetamido)-2-aminoethanesulfonic acid</p></td></tr><tr><td class="NLM_term">ALDH</td><td class="NLM_def"><p class="first last">aldehyde dehydrogenase</p></td></tr><tr><td class="NLM_term">BES</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-bis(2-hydroxyethyl)-2-aminoethanesulfonic acid</p></td></tr><tr><td class="NLM_term">BisTris</td><td class="NLM_def"><p class="first last">2-bis(2-hydroxyethyl)amino-2-(hydroxymethyl)-1,3-propanediol</p></td></tr><tr><td class="NLM_term">RA</td><td class="NLM_def"><p class="first last">retinoic acid</p></td></tr><tr><td class="NLM_term">RALDH</td><td class="NLM_def"><p class="first last">retinaldehyde dehydrogenase</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84520" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 63 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Vasiliou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estey, T.</span><span> </span><span class="NLM_article-title">Role of human aldehyde dehydrogenases in endobiotic and xenobiotic metabolism</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">279</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1081%2FDMR-120034001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=15237855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVSmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2004&pages=279-299&author=V.+Vasiliouauthor=A.+Pappaauthor=T.+Estey&title=Role+of+human+aldehyde+dehydrogenases+in+endobiotic+and+xenobiotic+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Human Aldehyde Dehydrogenases in Endobiotic and Xenobiotic Metabolism</span></div><div class="casAuthors">Vasiliou, Vasilis; Pappa, Aglaia; Estey, Tia</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-299</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">A review.  The human genome contains at least 17 genes that are members of the aldehyde dehydrogenase (ALDH) superfamily.  These genes encode NAD(P)+-dependent enzymes that oxidize a wide range of aldehydes to their corresponding carboxylic acids.  Aldehydes are highly reactive mols. that are intermediates or products involved in a broad spectrum of physiol., biol., and pharmacol. processes.  Aldehydes are generated during retinoic acid biosynthesis and the metab. of amino acids, lipids, carbohydrates, and drugs.  Mutations in several ALDH genes are the mol. basis of inborn errors of metab. and contribute to environmentally induced diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtaDV8EfwFcbVg90H21EOLACvtfcHk0lj9O7FkoLN7_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVSmurg%253D&md5=5e453a7e29a0e350cd7d88b8ef31744e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1081%2FDMR-120034001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1081%252FDMR-120034001%26sid%3Dliteratum%253Aachs%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DPappa%26aufirst%3DA.%26aulast%3DEstey%26aufirst%3DT.%26atitle%3DRole%2520of%2520human%2520aldehyde%2520dehydrogenases%2520in%2520endobiotic%2520and%2520xenobiotic%2520metabolism%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2004%26volume%3D36%26spage%3D279%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Yoshida, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dave, V.</span><span> </span><span class="NLM_article-title">Retinal oxidation activity and biological role of human cytosolic aldehyde dehydrogenase</span> <span class="citation_source-journal">Enzyme</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=1292933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK3sXitVyhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1992&pages=239-244&author=A.+Yoshidaauthor=L.+C.+Hsuauthor=V.+Dave&title=Retinal+oxidation+activity+and+biological+role+of+human+cytosolic+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Retinal oxidation activity and biological role of human cytosolic aldehyde dehydrogenase</span></div><div class="casAuthors">Yoshida, Akira; Hsu, Lily C.; Dave, Vibha</div><div class="citationInfo"><span class="NLM_cas:title">Enzyme</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4-5</span>),
    <span class="NLM_cas:pages">239-44</span>CODEN:
                <span class="NLM_cas:coden">ENZYBT</span>;
        ISSN:<span class="NLM_cas:issn">0013-9432</span>.
    </div><div class="casAbstract">The major human liver cytosolic aldehyde dehydrogenase isoenzyme 1 (ALDH1) is shown to exhibit strong activity for the oxidn. of retinal to retinoic acid, whereas the major mitochondrial isoenzyme 2 (ALDH2) and stomach cytosolic isoenzyme 3 (ALDH3) had no such activity.  The Km of ALDH1 for retinal was ∼0.06 μM at pH 7.5, and the catalytic efficiency (Vmax/Km) for retinal was ∼600-fold higher than that for acetaldehyde.  Thus, ALDH1 can efficiently produce retinoic acid from retinal in tissues with low retinal concns. (<0.1 μM).  The gene for ALDH1 was shown to have hormone response elements.  These findings suggest that the major physiol. substrate of human ALDH1 is retinal, and that its primary biol. role is the generation of retinoic acid, which results in the modulation of cell differentiation, including hormone-mediated development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfdQhgkxbQMbVg90H21EOLACvtfcHk0lj9O7FkoLN7_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXitVyhsbw%253D&md5=9ad5dbcc971d0aa52489f3551b6f13b5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DA.%26aulast%3DHsu%26aufirst%3DL.%2BC.%26aulast%3DDave%26aufirst%3DV.%26atitle%3DRetinal%2520oxidation%2520activity%2520and%2520biological%2520role%2520of%2520human%2520cytosolic%2520aldehyde%2520dehydrogenase%26jtitle%3DEnzyme%26date%3D1992%26volume%3D46%26spage%3D239%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kurys, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambroziak, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietruszko, R.</span><span> </span><span class="NLM_article-title">Human aldehyde dehydrogenase. Purification and characterization of a third isozyme with low <i>K</i><sub>m</sub> for gamma-aminobutyraldehyde</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">264</span><span class="NLM_x">, </span> <span class="NLM_fpage">4715</span><span class="NLM_x">–</span> <span class="NLM_lpage">4721</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=4715-4721&author=G.+Kurysauthor=W.+Ambroziakauthor=R.+Pietruszko&title=Human+aldehyde+dehydrogenase.+Purification+and+characterization+of+a+third+isozyme+with+low+Km+for+gamma-aminobutyraldehyde"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKurys%26aufirst%3DG.%26aulast%3DAmbroziak%26aufirst%3DW.%26aulast%3DPietruszko%26aufirst%3DR.%26atitle%3DHuman%2520aldehyde%2520dehydrogenase.%2520Purification%2520and%2520characterization%2520of%2520a%2520third%2520isozyme%2520with%2520low%2520Km%2520for%2520gamma-aminobutyraldehyde%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D4715%26epage%3D4721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Esterbauer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaur, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zollner, H.</span><span> </span><span class="NLM_article-title">Chemistry and biochemistry of 4-hydroxynonenal, malonaldehyde and related aldehydes</span> <span class="citation_source-journal">Free Radical Biol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2F0891-5849%2891%2990192-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=1937131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK3MXmsFSnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=81-128&author=H.+Esterbauerauthor=R.+J.+Schaurauthor=H.+Zollner&title=Chemistry+and+biochemistry+of+4-hydroxynonenal%2C+malonaldehyde+and+related+aldehydes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry and biochemistry of 4-hydroxynonenal, malonaldehyde and related aldehydes</span></div><div class="casAuthors">Esterbauer, Hermann; Schaur, Rudolf Joerg; Zollner, Helmward</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-128</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    </div><div class="casAbstract">A review, with 400 refs.  The purpose of this review is to provide a comprehensive summary on the chem. properties of 4-hydroxynonenal, 4-hydroxyhexenal, and malonaldehyde, the mechanisms of their formation and their occurrence in biol. systems and methods for their detn.  The authors will also review the reactions of 4-hydroxyalkenals and malonaldehyde with bimol. (amino acids, proteins, nucleic acid bases), their metab. in isolated cells and excretion in whole animals, as well as many types of biol. activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2QNkF-AU-EbVg90H21EOLACvtfcHk0lj9O7FkoLN7_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXmsFSnsbo%253D&md5=fce66ade3615bb95318c987bc7d63639</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2F0891-5849%2891%2990192-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0891-5849%252891%252990192-6%26sid%3Dliteratum%253Aachs%26aulast%3DEsterbauer%26aufirst%3DH.%26aulast%3DSchaur%26aufirst%3DR.%2BJ.%26aulast%3DZollner%26aufirst%3DH.%26atitle%3DChemistry%2520and%2520biochemistry%2520of%25204-hydroxynonenal%252C%2520malonaldehyde%2520and%2520related%2520aldehydes%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D1991%26volume%3D11%26spage%3D81%26epage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Sayre, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, G.</span><span> </span><span class="NLM_article-title">Chemistry and biochemistry of oxidative stress in neurodegenerative disease</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">738</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.2174%2F0929867013372922" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=11375746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjsVChtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=721-738&author=L.+M.+Sayreauthor=M.+A.+Smithauthor=G.+Perry&title=Chemistry+and+biochemistry+of+oxidative+stress+in+neurodegenerative+disease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry and biochemistry of oxidative stress in neurodegenerative disease</span></div><div class="casAuthors">Sayre, Lawrence M.; Smith, Mark A.; Perry, George</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">721-738</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review with 234 refs.  The age-related neurodegenerative diseases exemplified by Alzheimer's disease (AD), Lewy body diseases such as Parkinson's disease (PD), amyotrophic lateral sclerosis (ALS), and Huntington's disease are characterized by the deposition of abnormal forms of specific proteins in the brain.  Although several factors appear to underlie the pathol. depositions, the cause of neuronal death in each disease appears to be multifactorial.  In this regard, evidence in each case for a role of oxidative stress is provided by the finding that the pathol. deposits are immunoreactive to antibodies recognizing protein side-chains modified either directly by reactive oxygen or nitrogen species, or by products of lipid peroxidn. or glycoxidn.  Although the source(s) of increased oxidative damage are not entirely clear, the findings of increased localization of redox-active transition metals in the brain regions most affected is consistent with their contribution to oxidative stress.  It is tempting to speculate that free radical oxygen chem. plays a pathogenetic role in all these neurodegenerative conditions, though it is as yet undetd. what types of oxidative damage occur early in pathogenesis, and what types are secondary manifestations of dying neurons.  Delineation of the profile of oxidative damage in each disease will provide clues to how the specific neuronal populations are differentially affected by the individual disease conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9DAa0j4fYwbVg90H21EOLACvtfcHk0lgdQvJlExdm4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjsVChtbs%253D&md5=ff9d8a618f96b8fc7b6994e88e07191c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F0929867013372922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867013372922%26sid%3Dliteratum%253Aachs%26aulast%3DSayre%26aufirst%3DL.%2BM.%26aulast%3DSmith%26aufirst%3DM.%2BA.%26aulast%3DPerry%26aufirst%3DG.%26atitle%3DChemistry%2520and%2520biochemistry%2520of%2520oxidative%2520stress%2520in%2520neurodegenerative%2520disease%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2001%26volume%3D8%26spage%3D721%26epage%3D738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Chevion, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berenshtein, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stadtman, E. R.</span><span> </span><span class="NLM_article-title">Human studies related to protein oxidation: protein carbonyl content as a marker of damage</span> <span class="citation_source-journal">Free Radical Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl</span><span class="NLM_x">) </span> <span class="NLM_fpage">S99</span><span class="NLM_x">–</span> <span class="NLM_lpage">S108</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=S99-S108&issue=Suppl&author=M.+Chevionauthor=E.+Berenshteinauthor=E.+R.+Stadtman&title=Human+studies+related+to+protein+oxidation%3A+protein+carbonyl+content+as+a+marker+of+damage"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChevion%26aufirst%3DM.%26aulast%3DBerenshtein%26aufirst%3DE.%26aulast%3DStadtman%26aufirst%3DE.%2BR.%26atitle%3DHuman%2520studies%2520related%2520to%2520protein%2520oxidation%253A%2520protein%2520carbonyl%2520content%2520as%2520a%2520marker%2520of%2520damage%26jtitle%3DFree%2520Radical%2520Res.%26date%3D2000%26volume%3D33%26issue%3DSuppl%26spage%3DS99%26epage%3DS108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Marchitti, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brocker, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stagos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span> </span><span class="NLM_article-title">Non-P450 aldehyde oxidizing enzymes: the aldehyde dehydrogenase superfamily</span> <span class="citation_source-journal">Expert Opin. Drug Metab. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">697</span><span class="NLM_x">–</span> <span class="NLM_lpage">720</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1517%2F17425255.4.6.697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=18611112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1CgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=697-720&author=S.+A.+Marchittiauthor=C.+Brockerauthor=D.+Stagosauthor=V.+Vasiliou&title=Non-P450+aldehyde+oxidizing+enzymes%3A+the+aldehyde+dehydrogenase+superfamily"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Non-P450 aldehyde oxidizing enzymes: the aldehyde dehydrogenase superfamily</span></div><div class="casAuthors">Marchitti, Satori A.; Brocker, Chad; Stagos, Dimitrios; Vasiliou, Vasilis</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">697-720</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Aldehydes are highly reactive mols.  While several non-P 450 enzyme systems participate in their metab., one of the most important is the aldehyde dehydrogenase (ALDH) superfamily, composed of NAD(P)+-dependent enzymes that catalyze aldehyde oxidn.  This article presents a review of what is currently known about each member of the human ALDH superfamily including the pathophysiol. significance of these enzymes.  Relevant literature involving all members of the human ALDH family was extensively reviewed, with the primary focus on recent and novel findings.  To date, 19 ALDH genes have been identified in the human genome and mutations in these genes and subsequent inborn errors in aldehyde metab. are the mol. basis of several diseases, including Sjoegren-Larsson syndrome, type II hyperprolinemia, γ-hydroxybutyric aciduria and pyridoxine-dependent seizures.  ALDH enzymes also play important roles in embryogenesis and development, neurotransmission, oxidative stress and cancer.  Finally, ALDH enzymes display multiple catalytic and non-catalytic functions including ester hydrolysis, antioxidant properties, xenobiotic bioactivation and UV light absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6bSfChe5Ke7Vg90H21EOLACvtfcHk0lgdQvJlExdm4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1CgsLc%253D&md5=d377b38fcde9fd18bd30c7dc6b48811a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1517%2F17425255.4.6.697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.4.6.697%26sid%3Dliteratum%253Aachs%26aulast%3DMarchitti%26aufirst%3DS.%2BA.%26aulast%3DBrocker%26aufirst%3DC.%26aulast%3DStagos%26aufirst%3DD.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DNon-P450%2520aldehyde%2520oxidizing%2520enzymes%253A%2520the%2520aldehyde%2520dehydrogenase%2520superfamily%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2008%26volume%3D4%26spage%3D697%26epage%3D720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">O’Brien, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siraki, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shangari, N.</span><span> </span><span class="NLM_article-title">Aldehyde sources, metabolism, molecular toxicity mechanisms, and possible effects on human health</span> <span class="citation_source-journal">Crit. Rev. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">662</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1080%2F10408440591002183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=16417045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1agtr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2005&pages=609-662&author=P.+J.+O%E2%80%99Brienauthor=A.+G.+Sirakiauthor=N.+Shangari&title=Aldehyde+sources%2C+metabolism%2C+molecular+toxicity+mechanisms%2C+and+possible+effects+on+human+health"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde sources, metabolism, molecular toxicity mechanisms, and possible effects on human health</span></div><div class="casAuthors">O'Brien, Peter; Siraki, Arno; Shangari, Nandita</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">609-662</span>CODEN:
                <span class="NLM_cas:coden">CRTXB2</span>;
        ISSN:<span class="NLM_cas:issn">1040-8444</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">A review.  Aldehydes are org. compds. that are widespread in nature.  They can be formed endogenously by lipid peroxidn. (LPO), carbohydrate or metab. ascorbate autoxidn., amine oxidases, cytochrome P-450s, or myeloperoxidase-catalyzed metabolic activation.  This review compares the reactivity of many aldehydes towards biomols. particularly macromols.  Furthermore, it includes not only aldehydes of environmental or occupational concerns but also dietary aldehydes and aldehydes formed endogenously by intermediary metab.  Drugs that are aldehydes or form reactive aldehyde metabolites that cause side-effect toxicity are also included.  The effects of these aldehydes on biol. function, their contribution to human diseases, and the role of nucleic acid and protein carbonylation/oxidn. in mutagenicity and cytotoxicity mechanisms, resp., as well as carbonyl signal transduction and gene expression, are reviewed.  Aldehyde metabolic activation and detoxication by metabolizing enzymes are also reviewed, as well as the toxicol. and anticancer therapeutic effects of metabolizing enzyme inhibitors.  The human health risks from clin. and animal research studies are reviewed, including aldehydes as haptens in allergenic hypersensitivity diseases, respiratory allergies, and idiosyncratic drug toxicity; the potential carcinogenic risks of the carbonyl body burden; and the toxic effects of aldehydes in liver disease, embryo toxicity/teratogenicity, diabetes/hypertension, sclerosing peritonitis, cerebral ischemia/neurodegenerative diseases, and other aging-assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw561xTk_YUrVg90H21EOLACvtfcHk0lgdQvJlExdm4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1agtr3L&md5=b5be34c7f0253a8e988dd0e93427a399</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1080%2F10408440591002183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408440591002183%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DP.%2BJ.%26aulast%3DSiraki%26aufirst%3DA.%2BG.%26aulast%3DShangari%26aufirst%3DN.%26atitle%3DAldehyde%2520sources%252C%2520metabolism%252C%2520molecular%2520toxicity%2520mechanisms%252C%2520and%2520possible%2520effects%2520on%2520human%2520health%26jtitle%3DCrit.%2520Rev.%2520Toxicol.%26date%3D2005%26volume%3D35%26spage%3D609%26epage%3D662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Klyosov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rashkovetsky, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keung, W. M.</span><span> </span><span class="NLM_article-title">Possible role of liver cytosolic and mitochondrial aldehyde dehydrogenases in acetaldehyde metabolism</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">4445</span><span class="NLM_x">–</span> <span class="NLM_lpage">4456</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi9521093" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1996&pages=4445-4456&author=A.+A.+Klyosovauthor=L.+G.+Rashkovetskyauthor=M.+K.+Tahirauthor=W.+M.+Keung&title=Possible+role+of+liver+cytosolic+and+mitochondrial+aldehyde+dehydrogenases+in+acetaldehyde+metabolism"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fbi9521093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9521093%26sid%3Dliteratum%253Aachs%26aulast%3DKlyosov%26aufirst%3DA.%2BA.%26aulast%3DRashkovetsky%26aufirst%3DL.%2BG.%26aulast%3DTahir%26aufirst%3DM.%2BK.%26aulast%3DKeung%26aufirst%3DW.%2BM.%26atitle%3DPossible%2520role%2520of%2520liver%2520cytosolic%2520and%2520mitochondrial%2520aldehyde%2520dehydrogenases%2520in%2520acetaldehyde%2520metabolism%26jtitle%3DBiochemistry%26date%3D1996%26volume%3D35%26spage%3D4445%26epage%3D4456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Yoshida, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, I. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikawa, M.</span><span> </span><span class="NLM_article-title">Molecular abnormality of an inactive aldehyde dehydrogenase variant commonly found in Orientals</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">258</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=258-261&author=A.+Yoshidaauthor=I.+Y.+Huangauthor=M.+Ikawa&title=Molecular+abnormality+of+an+inactive+aldehyde+dehydrogenase+variant+commonly+found+in+Orientals"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DI.%2BY.%26aulast%3DIkawa%26aufirst%3DM.%26atitle%3DMolecular%2520abnormality%2520of%2520an%2520inactive%2520aldehyde%2520dehydrogenase%2520variant%2520commonly%2520found%2520in%2520Orientals%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1984%26volume%3D81%26spage%3D258%26epage%3D261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Peng, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsao, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, S. J.</span><span> </span><span class="NLM_article-title">Pharmacokinetic and pharmacodynamic basis for partial protection against alcoholism in Asians, heterozygous for the variant ALDH2*2 gene allele</span> <span class="citation_source-journal">Pharmacogenet. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">845</span><span class="NLM_x">–</span> <span class="NLM_lpage">855</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1097%2FFPC.0b013e3282609e67" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=17885622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVKhsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=845-855&author=G.+S.+Pengauthor=Y.+C.+Chenauthor=T.+P.+Tsaoauthor=M.+F.+Wangauthor=S.+J.+Yin&title=Pharmacokinetic+and+pharmacodynamic+basis+for+partial+protection+against+alcoholism+in+Asians%2C+heterozygous+for+the+variant+ALDH2*2+gene+allele"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetic and pharmacodynamic basis for partial protection against alcoholism in Asians, heterozygous for the variant ALDH2*2 gene allele</span></div><div class="casAuthors">Peng, Giia-Sheun; Chen, Yi-Chyan; Tsao, Tien-Ping; Wang, Ming-Fang; Yin, Shih-Jiun</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacogenetics and Genomics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">845-855</span>CODEN:
                <span class="NLM_cas:coden">PGHEAI</span>;
        ISSN:<span class="NLM_cas:issn">1744-6872</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Objectives: It has been well documented that although homozygosity of the variant aldehyde dehydrogenase-2 (ALDH2) gene allele, ALDH2*2, in Asians almost fully protects against alcoholism, the heterozygosity only affords a partial protection to varying degrees.  The full protection against alcoholism has been ascribed to the low-amt. of alc. hypersensitivity accompanied by a prolonged and large accumulation of acetaldehyde in blood (Peng et al. Pharmacogenetics 1999; 9:463-476).  The physiol. basis for the partial protection, however, remains unclear.  Methods: To address this question, we recruited a total of 32 adult Han Chinese men, matched by age, body-mass index, and nutritional state from a population base of 404 men.  The subjects were divided into 3 combinatorial genotypic groups of alc. dehydrogenase (ADH) and ALDH, i.e., ALDH2*1/*1-ADH1B*1/*1-ADH1C*1/*1 (n = 8), ALDH2*1/*1-ADH1B*2/*2-ADH1C*1/*1 (n = 8), and ALDH2*1/*2-ADH1B*2/*2-ADH1C*1/*1 (n = 16).  Following a moderate dose of ethanol (0.5 g/kg body wt.), blood ethanol/acetaldehyde/acetate concns., cardiac and extracranial/intracranial arterial hemodynamic parameters, as well as self-rated subjective sensations, were measured for 130 min.  Results: Heterozygotic ALDH2*1/*2 subjects were found to be strikingly responsive to the moderate amt. of alc., as evidenced by the prominent cardiovascular effects as well as subjective perceptions of general discomfort for as long as 2 h following ingestion.  The ADH1B polymorphisms did not appear to correlate with the pharmacokinetic and pharmacodynamic effects.  Conclusions: These results indicate that acetaldehyde, rather than ethanol or acetate, is primarily responsible for the obsd. alc. sensitivity reactions and suggest that substantially lower accumulation of blood acetaldehyde in the heterozygotes significantly reduces the aversion reaction to low amts. of alc. that permits other biol. as well as environmental factors to facilitate drinking and the according risk for the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowSjexF008DrVg90H21EOLACvtfcHk0lgx8eC8I_V9dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVKhsLfF&md5=534c77731b1f1b51c8890ad174ea3c8b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1097%2FFPC.0b013e3282609e67&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FFPC.0b013e3282609e67%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DG.%2BS.%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DTsao%26aufirst%3DT.%2BP.%26aulast%3DWang%26aufirst%3DM.%2BF.%26aulast%3DYin%26aufirst%3DS.%2BJ.%26atitle%3DPharmacokinetic%2520and%2520pharmacodynamic%2520basis%2520for%2520partial%2520protection%2520against%2520alcoholism%2520in%2520Asians%252C%2520heterozygous%2520for%2520the%2520variant%2520ALDH2%252A2%2520gene%2520allele%26jtitle%3DPharmacogenet.%2520Genomics%26date%3D2007%26volume%3D17%26spage%3D845%26epage%3D855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Marchitti, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deitrich, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span> </span><span class="NLM_article-title">Neurotoxicity and metabolism of the catecholamine-derived 3,4-dihydroxyphenylacetaldehyde and 3,4-dihydroxyphenylglycolaldehyde: the role of aldehyde dehydrogenase</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">150</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2007&pages=125-150&author=S.+A.+Marchittiauthor=R.+A.+Deitrichauthor=V.+Vasiliou&title=Neurotoxicity+and+metabolism+of+the+catecholamine-derived+3%2C4-dihydroxyphenylacetaldehyde+and+3%2C4-dihydroxyphenylglycolaldehyde%3A+the+role+of+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarchitti%26aufirst%3DS.%2BA.%26aulast%3DDeitrich%26aufirst%3DR.%2BA.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DNeurotoxicity%2520and%2520metabolism%2520of%2520the%2520catecholamine-derived%25203%252C4-dihydroxyphenylacetaldehyde%2520and%25203%252C4-dihydroxyphenylglycolaldehyde%253A%2520the%2520role%2520of%2520aldehyde%2520dehydrogenase%26jtitle%3DPharmacol.%2520Rev.%26date%3D2007%26volume%3D59%26spage%3D125%26epage%3D150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Yao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arolfo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olive, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zablocki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shryock, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackburn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diamond, I.</span><span> </span><span class="NLM_article-title">Inhibition of aldehyde dehydrogenase-2 suppresses cocaine seeking by generating THP, a cocaine use-dependent inhibitor of dopamine synthesis</span> <span class="citation_source-journal">Nature Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1024</span><span class="NLM_x">–</span> <span class="NLM_lpage">1028</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=1024-1028&author=L.+Yaoauthor=P.+Fanauthor=M.+Arolfoauthor=Z.+Jiangauthor=M.+F.+Oliveauthor=J.+Zablockiauthor=H.+L.+Sunauthor=N.+Chuauthor=J.+Leeauthor=H.+Y.+Kimauthor=K.+Leungauthor=J.+Shryockauthor=B.+Blackburnauthor=I.+Diamond&title=Inhibition+of+aldehyde+dehydrogenase-2+suppresses+cocaine+seeking+by+generating+THP%2C+a+cocaine+use-dependent+inhibitor+of+dopamine+synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DL.%26aulast%3DFan%26aufirst%3DP.%26aulast%3DArolfo%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DOlive%26aufirst%3DM.%2BF.%26aulast%3DZablocki%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DH.%2BL.%26aulast%3DChu%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%2BY.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DShryock%26aufirst%3DJ.%26aulast%3DBlackburn%26aufirst%3DB.%26aulast%3DDiamond%26aufirst%3DI.%26atitle%3DInhibition%2520of%2520aldehyde%2520dehydrogenase-2%2520suppresses%2520cocaine%2520seeking%2520by%2520generating%2520THP%252C%2520a%2520cocaine%2520use-dependent%2520inhibitor%2520of%2520dopamine%2520synthesis%26jtitle%3DNature%2520Med.%26date%3D2010%26volume%3D16%26spage%3D1024%26epage%3D1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Rizzo, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dammann, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craft, D. A.</span><span> </span><span class="NLM_article-title">Sjogren–Larsson syndrome. Impaired fatty alcohol oxidation in cultured fibroblasts due to deficient fatty alcohol:nicotinamide adenine dinucleotide oxidoreductase activity</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">738</span><span class="NLM_x">–</span> <span class="NLM_lpage">744</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1988&pages=738-744&author=W.+B.+Rizzoauthor=A.+L.+Dammannauthor=D.+A.+Craft&title=Sjogren%E2%80%93Larsson+syndrome.+Impaired+fatty+alcohol+oxidation+in+cultured+fibroblasts+due+to+deficient+fatty+alcohol%3Anicotinamide+adenine+dinucleotide+oxidoreductase+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRizzo%26aufirst%3DW.%2BB.%26aulast%3DDammann%26aufirst%3DA.%2BL.%26aulast%3DCraft%26aufirst%3DD.%2BA.%26atitle%3DSjogren%25E2%2580%2593Larsson%2520syndrome.%2520Impaired%2520fatty%2520alcohol%2520oxidation%2520in%2520cultured%2520fibroblasts%2520due%2520to%2520deficient%2520fatty%2520alcohol%253Anicotinamide%2520adenine%2520dinucleotide%2520oxidoreductase%2520activity%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1988%26volume%3D81%26spage%3D738%26epage%3D744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Rizzo, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, G.</span><span> </span><span class="NLM_article-title">Sjogren–Larsson syndrome: diversity of mutations and polymorphisms in the fatty aldehyde dehydrogenase gene (ALDH3A2)</span> <span class="citation_source-journal">Hum. Mutat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1002%2Fhumu.20181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=15931689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslSksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1-10&author=W.+B.+Rizzoauthor=G.+Carney&title=Sjogren%E2%80%93Larsson+syndrome%3A+diversity+of+mutations+and+polymorphisms+in+the+fatty+aldehyde+dehydrogenase+gene+%28ALDH3A2%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Sjogren-Larsson syndrome: diversity of mutations and polymorphisms in the fatty aldehyde dehydrogenase gene (ALDH3A2)</span></div><div class="casAuthors">Rizzo, William B.; Carney, Gael</div><div class="citationInfo"><span class="NLM_cas:title">Human Mutation</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-10</span>CODEN:
                <span class="NLM_cas:coden">HUMUE3</span>;
        ISSN:<span class="NLM_cas:issn">1059-7794</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  Sjogren-Larsson syndrome (SLS) is an autosomal recessive disorder characterized by ichthyosis, mental retardation, and spastic diplegia or tetraplegia.  The disease is caused by mutations in the ALDH3A2 gene (also known as FALDH and ALDH10) on chromosome 17p11.2 that encodes fatty aldehyde dehydrogenase (FALDH), an enzyme that catalyzes the oxidn. of long-chain aldehydes derived from lipid metab.  In SLS patients, 72 mutations have been identified, with a distribution that is scattered throughout the ALDH3A2 gene.  Most mutations are private but several common mutations have been detected, which probably reflect founder effects or recurrent mutational events.  Missense mutations comprise the most abundant class (38%) and expression studies indicate that most of these result in a profound redn. in enzyme activity.  Deletions account for about 25% of the mutations and range from single nucleotides to entire exons.  Twelve splice-site mutations have been demonstrated to cause aberrant splicing in cultured fibroblasts.  To date, more than a dozen intragenic ALDH3A2 polymorphisms consisting of SNPs and one microsatellite marker have been characterized, although none of them alter the FALDH protein sequence.  The striking mutational diversity in SLS offers a challenge for DNA-based diagnosis, but promises to provide a wealth of information about enzyme structure-function correlations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS0A16LhWAU7Vg90H21EOLACvtfcHk0lhcgpMxW7jzAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslSksL0%253D&md5=c5969126d454ddd0c65579a0215367d5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fhumu.20181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhumu.20181%26sid%3Dliteratum%253Aachs%26aulast%3DRizzo%26aufirst%3DW.%2BB.%26aulast%3DCarney%26aufirst%3DG.%26atitle%3DSjogren%25E2%2580%2593Larsson%2520syndrome%253A%2520diversity%2520of%2520mutations%2520and%2520polymorphisms%2520in%2520the%2520fatty%2520aldehyde%2520dehydrogenase%2520gene%2520%2528ALDH3A2%2529%26jtitle%3DHum.%2520Mutat.%26date%3D2005%26volume%3D26%26spage%3D1%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Mills, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Struys, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakobs, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plecko, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumgartner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willemsen, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omran, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tacke, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uhlenberg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weschke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clayton, P. T.</span><span> </span><span class="NLM_article-title">Mutations in antiquitin in individuals with pyridoxine-dependent seizures</span> <span class="citation_source-journal">Nature Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">309</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=307-309&author=P.+B.+Millsauthor=E.+Struysauthor=C.+Jakobsauthor=B.+Pleckoauthor=P.+Baxterauthor=M.+Baumgartnerauthor=M.+A.+Willemsenauthor=H.+Omranauthor=U.+Tackeauthor=B.+Uhlenbergauthor=B.+Weschkeauthor=P.+T.+Clayton&title=Mutations+in+antiquitin+in+individuals+with+pyridoxine-dependent+seizures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMills%26aufirst%3DP.%2BB.%26aulast%3DStruys%26aufirst%3DE.%26aulast%3DJakobs%26aufirst%3DC.%26aulast%3DPlecko%26aufirst%3DB.%26aulast%3DBaxter%26aufirst%3DP.%26aulast%3DBaumgartner%26aufirst%3DM.%26aulast%3DWillemsen%26aufirst%3DM.%2BA.%26aulast%3DOmran%26aufirst%3DH.%26aulast%3DTacke%26aufirst%3DU.%26aulast%3DUhlenberg%26aufirst%3DB.%26aulast%3DWeschke%26aufirst%3DB.%26aulast%3DClayton%26aufirst%3DP.%2BT.%26atitle%3DMutations%2520in%2520antiquitin%2520in%2520individuals%2520with%2520pyridoxine-dependent%2520seizures%26jtitle%3DNature%2520Med.%26date%3D2006%26volume%3D12%26spage%3D307%26epage%3D309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Geraghty, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaughn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jimenez-Sanchez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, C. A.</span><span> </span><span class="NLM_article-title">Mutations in the Delta1-pyrroline 5-carboxylate dehydrogenase gene cause type II hyperprolinemia</span> <span class="citation_source-journal">Hum. Mol. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1411</span><span class="NLM_x">–</span> <span class="NLM_lpage">1415</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1093%2Fhmg%2F7.9.1411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=9700195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK1cXmtVSlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1998&pages=1411-1415&author=M.+T.+Geraghtyauthor=D.+Vaughnauthor=A.+J.+Nicholsonauthor=W.+W.+Linauthor=G.+Jimenez-Sanchezauthor=C.+Obieauthor=M.+P.+Flynnauthor=D.+Valleauthor=C.+A.+Hu&title=Mutations+in+the+Delta1-pyrroline+5-carboxylate+dehydrogenase+gene+cause+type+II+hyperprolinemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the Δ1-pyrroline 5-carboxylate dehydrogenase gene cause type II hyperprolinemia</span></div><div class="casAuthors">Geraghty, Michael T.; Vaughn, D.; Nicholson, A. J.; Lin, Wei-Wen; Jimenez-Sanchez, Gerardo; Obie, Cassandra; Flynn, M. P.; Valle, David; Hu, Chien-an A.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1411-1415</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We surveyed Δ1-pyrroline 5-carboxylate dehydrogenase genes from four patients with hyperprolinemia type II using RT-PCR amplification, genomic PCR amplification and direct sequencing.  We found four mutant alleles, two with frameshift mutations [A7fs(-1) and G521fs(+1)] and two with missense mutations (S352L and P16L).  To test the functional consequences of three of these, we expressed them in a P5CDh-deficient strain of Saccharomyces cerevisiae.  In contrast to wild-type human P5CDh, yeast expressing S352L and G521fs(+1) failed to grow on proline and had no detectable P5CDh activity.  The P16L allele, however, produced fully functional P5CDh and subsequent anal. suggests that it is polymorphic in the relevant (Spanish) population.  Interestingly, the G521fs(+1) allele segregates in the large Irish Traveler pedigree used to define the HPII phenotype.  To our knowledge, this is the first description of the mol. basis for this inborn error.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo6wZtdarVurVg90H21EOLACvtfcHk0lhcgpMxW7jzAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtVSlsLg%253D&md5=5e068b38cf959de7c04dcc1bcea392d0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F7.9.1411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F7.9.1411%26sid%3Dliteratum%253Aachs%26aulast%3DGeraghty%26aufirst%3DM.%2BT.%26aulast%3DVaughn%26aufirst%3DD.%26aulast%3DNicholson%26aufirst%3DA.%2BJ.%26aulast%3DLin%26aufirst%3DW.%2BW.%26aulast%3DJimenez-Sanchez%26aufirst%3DG.%26aulast%3DObie%26aufirst%3DC.%26aulast%3DFlynn%26aufirst%3DM.%2BP.%26aulast%3DValle%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DC.%2BA.%26atitle%3DMutations%2520in%2520the%2520Delta1-pyrroline%25205-carboxylate%2520dehydrogenase%2520gene%2520cause%2520type%2520II%2520hyperprolinemia%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D1998%26volume%3D7%26spage%3D1411%26epage%3D1415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Farrant, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellor, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langley, G. J.</span><span> </span><span class="NLM_article-title">Pyridoxal phosphate deactivation by pyrroline-5-carboxylic acid. Increased risk of vitamin B6 deficiency and seizures in hyperprolinemia type II</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">15107</span><span class="NLM_x">–</span> <span class="NLM_lpage">15116</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=15107-15116&author=R.+D.+Farrantauthor=V.+Walkerauthor=G.+A.+Millsauthor=J.+M.+Mellorauthor=G.+J.+Langley&title=Pyridoxal+phosphate+deactivation+by+pyrroline-5-carboxylic+acid.+Increased+risk+of+vitamin+B6+deficiency+and+seizures+in+hyperprolinemia+type+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFarrant%26aufirst%3DR.%2BD.%26aulast%3DWalker%26aufirst%3DV.%26aulast%3DMills%26aufirst%3DG.%2BA.%26aulast%3DMellor%26aufirst%3DJ.%2BM.%26aulast%3DLangley%26aufirst%3DG.%2BJ.%26atitle%3DPyridoxal%2520phosphate%2520deactivation%2520by%2520pyrroline-5-carboxylic%2520acid.%2520Increased%2520risk%2520of%2520vitamin%2520B6%2520deficiency%2520and%2520seizures%2520in%2520hyperprolinemia%2520type%2520II%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D15107%26epage%3D15116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Marchitti, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlicky, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span> </span><span class="NLM_article-title">Expression and initial characterization of human ALDH3B1</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">356</span><span class="NLM_x">, </span> <span class="NLM_fpage">792</span><span class="NLM_x">–</span> <span class="NLM_lpage">798</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2007&pages=792-798&author=S.+A.+Marchittiauthor=D.+J.+Orlickyauthor=V.+Vasiliou&title=Expression+and+initial+characterization+of+human+ALDH3B1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarchitti%26aufirst%3DS.%2BA.%26aulast%3DOrlicky%26aufirst%3DD.%2BJ.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DExpression%2520and%2520initial%2520characterization%2520of%2520human%2520ALDH3B1%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D356%26spage%3D792%26epage%3D798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Ma, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allan, A. L.</span><span> </span><span class="NLM_article-title">The role of human aldehyde dehydrogenase in normal and cancer stem cells</span> <span class="citation_source-journal">Stem Cell Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">292</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1007%2Fs12015-010-9208-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=21103958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFWjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=292-306&author=I.+Maauthor=A.+L.+Allan&title=The+role+of+human+aldehyde+dehydrogenase+in+normal+and+cancer+stem+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The role of human aldehyde dehydrogenase in normal and cancer stem cells</span></div><div class="casAuthors">Ma, Irene; Allan, Alison L.</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reviews and Reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">292-306</span>CODEN:
                <span class="NLM_cas:coden">SCRREE</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Normal stem cells and cancer stem cells (CSCs) share similar properties, in that both have the capacity to self-renew and differentiate into multiple cell types.  In both the normal stem cell and cancer stem cell fields, there has been a great need for a universal marker that can effectively identify and isolate these rare populations of cells in order to characterize them and use this information for research and therapeutic purposes.  Currently, it would appear that certain isoenzymes of the aldehyde dehydrogenase (ALDH) superfamily may be able to fulfill this role as a marker for both normal and cancer stem cells.  ALDH has been identified as an important enzyme in the protection of normal hematopoietic stem cells, and is now also widely used as a marker to identify and isolate various types of normal stem cells and CSCs.  In addn., emerging evidence suggests that ALDH1 is not only a marker for stem cells, but may also play important functional roles related to self-protection, differentiation, and expansion.  This comprehensive review discusses the role that ALDH plays in normal stem cells and CSCs, with focus on the ALDH1 and ALDH3A1 isoenzymes.  Discrepancies in the functional themes between cell types and future perspectives for therapeutic applications are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog42zeNePwFLVg90H21EOLACvtfcHk0lgwZwG80upk0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFWjtbk%253D&md5=0fbede2e32f7ffcce91c8b6dba24711a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs12015-010-9208-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12015-010-9208-4%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DI.%26aulast%3DAllan%26aufirst%3DA.%2BL.%26atitle%3DThe%2520role%2520of%2520human%2520aldehyde%2520dehydrogenase%2520in%2520normal%2520and%2520cancer%2520stem%2520cells%26jtitle%3DStem%2520Cell%2520Rev.%26date%3D2011%26volume%3D7%26spage%3D292%26epage%3D306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Sladek, N. E.</span><span> </span><span class="NLM_article-title">Aldehyde dehydrogenase-mediated cellular relative insensitivity to the oxazaphosphorines</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">625</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=607-625&author=N.+E.+Sladek&title=Aldehyde+dehydrogenase-mediated+cellular+relative+insensitivity+to+the+oxazaphosphorines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSladek%26aufirst%3DN.%2BE.%26atitle%3DAldehyde%2520dehydrogenase-mediated%2520cellular%2520relative%2520insensitivity%2520to%2520the%2520oxazaphosphorines%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D1999%26volume%3D5%26spage%3D607%26epage%3D625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Agarwal, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Eitzen, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier-Tackmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goedde, H. W.</span><span> </span><span class="NLM_article-title">Metabolism of cyclophosphamide by aldehyde dehydrogenases</span> <span class="citation_source-journal">Adv. Exp. Med. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">372</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=1995&pages=115-122&author=D.+P.+Agarwalauthor=U.+von+Eitzenauthor=D.+Meier-Tackmannauthor=H.+W.+Goedde&title=Metabolism+of+cyclophosphamide+by+aldehyde+dehydrogenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwal%26aufirst%3DD.%2BP.%26aulast%3Dvon%2BEitzen%26aufirst%3DU.%26aulast%3DMeier-Tackmann%26aufirst%3DD.%26aulast%3DGoedde%26aufirst%3DH.%2BW.%26atitle%3DMetabolism%2520of%2520cyclophosphamide%2520by%2520aldehyde%2520dehydrogenases%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D1995%26volume%3D372%26spage%3D115%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Yoon, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arakawa, H.</span><span> </span><span class="NLM_article-title">Identification of ALDH4 as a p53-inducible gene and its protective role in cellular stresses</span> <span class="citation_source-journal">J. Hum. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">134</span><span class="NLM_x">–</span> <span class="NLM_lpage">140</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2004&pages=134-140&author=K.+A.+Yoonauthor=Y.+Nakamuraauthor=H.+Arakawa&title=Identification+of+ALDH4+as+a+p53-inducible+gene+and+its+protective+role+in+cellular+stresses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoon%26aufirst%3DK.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DArakawa%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520ALDH4%2520as%2520a%2520p53-inducible%2520gene%2520and%2520its%2520protective%2520role%2520in%2520cellular%2520stresses%26jtitle%3DJ.%2520Hum.%2520Genet.%26date%3D2004%26volume%3D49%26spage%3D134%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapointe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaygusuz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ong, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Rijn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollack, J. R.</span><span> </span><span class="NLM_article-title">The retinoic acid synthesis gene ALDH1a2 is a candidate tumor suppressor in prostate cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">8118</span><span class="NLM_x">–</span> <span class="NLM_lpage">8124</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=8118-8124&author=H.+Kimauthor=J.+Lapointeauthor=G.+Kaygusuzauthor=D.+E.+Ongauthor=C.+Liauthor=M.+van+de+Rijnauthor=J.+D.+Brooksauthor=J.+R.+Pollack&title=The+retinoic+acid+synthesis+gene+ALDH1a2+is+a+candidate+tumor+suppressor+in+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.%26aulast%3DLapointe%26aufirst%3DJ.%26aulast%3DKaygusuz%26aufirst%3DG.%26aulast%3DOng%26aufirst%3DD.%2BE.%26aulast%3DLi%26aufirst%3DC.%26aulast%3Dvan%2Bde%2BRijn%26aufirst%3DM.%26aulast%3DBrooks%26aufirst%3DJ.%2BD.%26aulast%3DPollack%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520retinoic%2520acid%2520synthesis%2520gene%2520ALDH1a2%2520is%2520a%2520candidate%2520tumor%2520suppressor%2520in%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D8118%26epage%3D8124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Matsuda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabushita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanaly, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibutani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, A.</span><span> </span><span class="NLM_article-title">Increased DNA damage in ALDH2-deficient alcoholics</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1374</span><span class="NLM_x">–</span> <span class="NLM_lpage">1378</span></span><div class="citationLinks">[<a href="/doi/10.1021/tx060113h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVynsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2006&pages=1374-1378&author=T.+Matsudaauthor=H.+Yabushitaauthor=R.+A.+Kanalyauthor=S.+Shibutaniauthor=A.+Yokoyama&title=Increased+DNA+damage+in+ALDH2-deficient+alcoholics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Increased DNA Damage in ALDH2-Deficient Alcoholics</span></div><div class="casAuthors">Matsuda, Tomonari; Yabushita, Hisatoshi; Kanaly, Robert A.; Shibutani, Shinya; Yokoyama, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1374-1378</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drinking alc. is a risk factor for cancers of the oral cavity, pharynx, larynx, and esophagus.  Although many studies suggest that acetaldehyde, a major metabolite of orally ingested alc., plays a crucial role in cancer initiation, the link between the aldehyde dehydrogenase-2 (ALDH2) genotype and acetaldehyde-derived DNA damage has not yet been explored.  We have developed a sensitive and quant. method for detecting the acetaldehyde-derived DNA adducts, N2-ethyl-2'-deoxyguanosine (N2-Et-dG), α-S- and α-R-methyl-γ-hydroxy-1,N2-propano-2'-deoxyguanosine (α-S-Me-γ-OH-PdG and α-R-Me-γ-OH-PdG), and N2-(2,6-dimethyl-1,3-dioxan-4-yl)-deoxyguanosine (N2-Dio-dG), by using liq. chromatog. electrospray tandem mass spectrometry (LC/ESI-MS/MS) and stable-isotope internal stds.  We detd. the DNA adducts in 44 blood DNA samples from Japanese alc. patients.  The levels of three acetaldehyde-derived DNA adducts, N2-Et-dG, α-S-Me-γ-OH-PdG, and α-R-Me-γ-OH-PdG, were significantly higher in alcoholics with the ALDH2*1/2*2 genotype compared to those with the ALDH2*1/2*1 genotype.  N2-Dio-dG was not detected in any of the DNA samples analyzed.  These results provide mol. evidence that the ALDH2 genotype affects the genotoxic damage caused by acetaldehyde.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQSJneK9U1XrVg90H21EOLACvtfcHk0lgGbhHydVb3Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVynsLk%253D&md5=b7fe8c0a95ed1ee6564ff73a75409998</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Ftx060113h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx060113h%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuda%26aufirst%3DT.%26aulast%3DYabushita%26aufirst%3DH.%26aulast%3DKanaly%26aufirst%3DR.%2BA.%26aulast%3DShibutani%26aufirst%3DS.%26aulast%3DYokoyama%26aufirst%3DA.%26atitle%3DIncreased%2520DNA%2520damage%2520in%2520ALDH2-deficient%2520alcoholics%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2006%26volume%3D19%26spage%3D1374%26epage%3D1378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Yokoyama, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsushita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maruyama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishii, H.</span><span> </span><span class="NLM_article-title">Alcohol and aldehyde dehydrogenase gene polymorphisms and oropharyngolaryngeal, esophageal and stomach cancers in Japanese alcoholics</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">433</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=433-439&author=A.+Yokoyamaauthor=T.+Muramatsuauthor=T.+Omoriauthor=T.+Yokoyamaauthor=S.+Matsushitaauthor=S.+Higuchiauthor=K.+Maruyamaauthor=H.+Ishii&title=Alcohol+and+aldehyde+dehydrogenase+gene+polymorphisms+and+oropharyngolaryngeal%2C+esophageal+and+stomach+cancers+in+Japanese+alcoholics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DOmori%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DT.%26aulast%3DMatsushita%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DS.%26aulast%3DMaruyama%26aufirst%3DK.%26aulast%3DIshii%26aufirst%3DH.%26atitle%3DAlcohol%2520and%2520aldehyde%2520dehydrogenase%2520gene%2520polymorphisms%2520and%2520oropharyngolaryngeal%252C%2520esophageal%2520and%2520stomach%2520cancers%2520in%2520Japanese%2520alcoholics%26jtitle%3DCarcinogenesis%26date%3D2001%26volume%3D22%26spage%3D433%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Jones, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barber, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vala, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collector, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufmann, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludeman, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colvin, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilton, J.</span><span> </span><span class="NLM_article-title">Assessment of aldehyde dehydrogenase in viable cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">2742</span><span class="NLM_x">–</span> <span class="NLM_lpage">2746</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=1995&pages=2742-2746&author=R.+J.+Jonesauthor=J.+P.+Barberauthor=M.+S.+Valaauthor=M.+I.+Collectorauthor=S.+H.+Kaufmannauthor=S.+M.+Ludemanauthor=O.+M.+Colvinauthor=J.+Hilton&title=Assessment+of+aldehyde+dehydrogenase+in+viable+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DR.%2BJ.%26aulast%3DBarber%26aufirst%3DJ.%2BP.%26aulast%3DVala%26aufirst%3DM.%2BS.%26aulast%3DCollector%26aufirst%3DM.%2BI.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%26aulast%3DLudeman%26aufirst%3DS.%2BM.%26aulast%3DColvin%26aufirst%3DO.%2BM.%26aulast%3DHilton%26aufirst%3DJ.%26atitle%3DAssessment%2520of%2520aldehyde%2520dehydrogenase%2520in%2520viable%2520cells%26jtitle%3DBlood%26date%3D1995%26volume%3D85%26spage%3D2742%26epage%3D2746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Chambon, P.</span><span> </span><span class="NLM_article-title">A decade of molecular biology of retinoic acid receptors</span> <span class="citation_source-journal">FASEB J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">940</span><span class="NLM_x">–</span> <span class="NLM_lpage">954</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1096%2Ffasebj.10.9.8801176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=8801176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK28Xks1Wnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1996&pages=940-954&author=P.+Chambon&title=A+decade+of+molecular+biology+of+retinoic+acid+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of molecular biology of retinoic acid receptors</span></div><div class="casAuthors">Chambon, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">940-954</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A review, with 120 refs., on the signaling pathway, regulation, and structure-activity relations of retinoic acid receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKohrdFWv7e7Vg90H21EOLACvtfcHk0lgGbhHydVb3Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xks1Wnsb0%253D&md5=d20140a9229ec056a20a7d79b16418b8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1096%2Ffasebj.10.9.8801176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffasebj.10.9.8801176%26sid%3Dliteratum%253Aachs%26aulast%3DChambon%26aufirst%3DP.%26atitle%3DA%2520decade%2520of%2520molecular%2520biology%2520of%2520retinoic%2520acid%2520receptors%26jtitle%3DFASEB%2520J.%26date%3D1996%26volume%3D10%26spage%3D940%26epage%3D954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandell, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trainor, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koentgen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duester, G.</span><span> </span><span class="NLM_article-title">Alcohol and aldehyde dehydrogenases: retinoid metabolic effects in mouse knockout models</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">1821</span><span class="NLM_x">, </span> <span class="NLM_fpage">198</span><span class="NLM_x">–</span> <span class="NLM_lpage">205</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1821&publication_year=2012&pages=198-205&author=S.+Kumarauthor=L.+L.+Sandellauthor=P.+A.+Trainorauthor=F.+Koentgenauthor=G.+Duester&title=Alcohol+and+aldehyde+dehydrogenases%3A+retinoid+metabolic+effects+in+mouse+knockout+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DSandell%26aufirst%3DL.%2BL.%26aulast%3DTrainor%26aufirst%3DP.%2BA.%26aulast%3DKoentgen%26aufirst%3DF.%26aulast%3DDuester%26aufirst%3DG.%26atitle%3DAlcohol%2520and%2520aldehyde%2520dehydrogenases%253A%2520retinoid%2520metabolic%2520effects%2520in%2520mouse%2520knockout%2520models%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2012%26volume%3D1821%26spage%3D198%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Niederreither, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraulob, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garnier, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolle, P.</span><span> </span><span class="NLM_article-title">Differential expression of retinoic acid-synthesizing (RALDH) enzymes during fetal development and organ differentiation in the mouse</span> <span class="citation_source-journal">Mech. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">165</span><span class="NLM_x">–</span> <span class="NLM_lpage">171</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2002&pages=165-171&author=K.+Niederreitherauthor=V.+Fraulobauthor=J.+M.+Garnierauthor=P.+Chambonauthor=P.+Dolle&title=Differential+expression+of+retinoic+acid-synthesizing+%28RALDH%29+enzymes+during+fetal+development+and+organ+differentiation+in+the+mouse"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNiederreither%26aufirst%3DK.%26aulast%3DFraulob%26aufirst%3DV.%26aulast%3DGarnier%26aufirst%3DJ.%2BM.%26aulast%3DChambon%26aufirst%3DP.%26aulast%3DDolle%26aufirst%3DP.%26atitle%3DDifferential%2520expression%2520of%2520retinoic%2520acid-synthesizing%2520%2528RALDH%2529%2520enzymes%2520during%2520fetal%2520development%2520and%2520organ%2520differentiation%2520in%2520the%2520mouse%26jtitle%3DMech.%2520Dev.%26date%3D2002%26volume%3D110%26spage%3D165%26epage%3D171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Koppaka, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellermann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolaou, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juvonen, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deitrich, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span> </span><span class="NLM_article-title">Aldehyde dehydrogenase inhibitors: a comprehensive review of the pharmacology, mechanism of action, substrate specificity, and clinical application</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">520</span><span class="NLM_x">–</span> <span class="NLM_lpage">539</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1124%2Fpr.111.005538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=22544865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslaktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2012&pages=520-539&author=V.+Koppakaauthor=D.+C.+Thompsonauthor=Y.+Chenauthor=M.+Ellermannauthor=K.+C.+Nicolaouauthor=R.+O.+Juvonenauthor=D.+Petersenauthor=R.+A.+Deitrichauthor=T.+D.+Hurleyauthor=V.+Vasiliou&title=Aldehyde+dehydrogenase+inhibitors%3A+a+comprehensive+review+of+the+pharmacology%2C+mechanism+of+action%2C+substrate+specificity%2C+and+clinical+application"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenase inhibitors: a comprehensive review of the pharmacology, mechanism of action, substrate specificity, and clinical application</span></div><div class="casAuthors">Koppaka, Vindhya; Thompson, David C.; Chen, Ying; Ellermann, Manuel; Nicolaou, Kyriacos C.; Juvonen, Risto O.; Petersen, Dennis; Deitrich, Richard A.; Hurley, Thomas D.; Vasiliou, Vasilis</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">520-539</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Aldehyde dehydrogenases (ALDHs) belong to a superfamily of enzymes that play a key role in the metab. of aldehydes of both endogenous and exogenous derivation.  The human ALDH superfamily comprises 19 isoenzymes that possess important physiol. and toxicol. functions.  The ALDH1A subfamily plays a pivotal role in embryogenesis and development by mediating retinoic acid signaling.  ALDH2, as a key enzyme that oxidizes acetaldehyde, is crucial for alc. metab.  ALDH1A1 and ALDH3A1 are lens and corneal crystallins, which are essential elements of the cellular defense mechanism against UV radiation-induced damage in ocular tissues.  Many ALDH isoenzymes are important in oxidizing reactive aldehydes derived from lipid peroxidn. and thereby help maintain cellular homeostasis.  Increased expression and activity of ALDH isoenzymes have been reported in various human cancers and are assocd. with cancer relapse.  As a direct consequence of their significant physiol. and toxicol. roles, inhibitors of the ALDH enzymes have been developed to treat human diseases.  This review summarizes known ALDH inhibitors, their mechanisms of action, isoenzyme selectivity, potency, and clin. uses.  The purpose of this review is to 1) establish the current status of pharmacol. inhibition of the ALDHs, 2) provide a rationale for the continued development of ALDH isoenzyme-selective inhibitors, and 3) identify the challenges and potential therapeutic rewards assocd. with the creation of such agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOl6kdU4MnaLVg90H21EOLACvtfcHk0lhk-gJB2IHY2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslaktw%253D%253D&md5=1de904ae3778b8715a6ec10e8d0242db</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fpr.111.005538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.111.005538%26sid%3Dliteratum%253Aachs%26aulast%3DKoppaka%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DJuvonen%26aufirst%3DR.%2BO.%26aulast%3DPetersen%26aufirst%3DD.%26aulast%3DDeitrich%26aufirst%3DR.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DAldehyde%2520dehydrogenase%2520inhibitors%253A%2520a%2520comprehensive%2520review%2520of%2520the%2520pharmacology%252C%2520mechanism%2520of%2520action%252C%2520substrate%2520specificity%252C%2520and%2520clinical%2520application%26jtitle%3DPharmacol.%2520Rev.%26date%3D2012%26volume%3D64%26spage%3D520%26epage%3D539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Feldman, R. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiner, H.</span><span> </span><span class="NLM_article-title">Horse liver aldehyde dehydrogenase. II. Kinetics and mechanistic implications of the dehydrogenase and esterase activity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">247</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=1972&pages=267-272&author=R.+I.+Feldmanauthor=H.+Weiner&title=Horse+liver+aldehyde+dehydrogenase.+II.+Kinetics+and+mechanistic+implications+of+the+dehydrogenase+and+esterase+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFeldman%26aufirst%3DR.%2BI.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DHorse%2520liver%2520aldehyde%2520dehydrogenase.%2520II.%2520Kinetics%2520and%2520mechanistic%2520implications%2520of%2520the%2520dehydrogenase%2520and%2520esterase%2520activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1972%26volume%3D247%26spage%3D267%26epage%3D272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Sidhu, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blair, A. H.</span><span> </span><span class="NLM_article-title">Human liver aldehyde dehydrogenase. Esterase activity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1975</span><span class="NLM_x">, </span> <span class="NLM_volume">250</span><span class="NLM_x">, </span> <span class="NLM_fpage">7894</span><span class="NLM_x">–</span> <span class="NLM_lpage">7898</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=250&publication_year=1975&pages=7894-7898&author=R.+S.+Sidhuauthor=A.+H.+Blair&title=Human+liver+aldehyde+dehydrogenase.+Esterase+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSidhu%26aufirst%3DR.%2BS.%26aulast%3DBlair%26aufirst%3DA.%2BH.%26atitle%3DHuman%2520liver%2520aldehyde%2520dehydrogenase.%2520Esterase%2520activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1975%26volume%3D250%26spage%3D7894%26epage%3D7898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Wang, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, S. J.</span><span> </span><span class="NLM_article-title">Substrate specificity of human and yeast aldehyde dehydrogenases</span> <span class="citation_source-journal">Chem. Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">36</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2008.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=18983993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFOju78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2009&pages=36-39&author=M.+F.+Wangauthor=C.+L.+Hanauthor=S.+J.+Yin&title=Substrate+specificity+of+human+and+yeast+aldehyde+dehydrogenases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate specificity of human and yeast aldehyde dehydrogenases</span></div><div class="casAuthors">Wang, Ming-Fang; Han, Chih-Li; Yin, Shih-Jiun</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">36-39</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">The human aldehyde dehydrogenase (ALDH) family may contribute to the metab. of hydrocarbons, biogenic amines, retinoids, steroids, and lipid peroxidn.  The authors previously reported the kinetic properties of human cytosolic ALDH1 and mitochondrial ALDH2 toward the oxidn. of straight-chain and branched-chain aliph. aldehydes with various chain lengths.  Here, the authors present substrate specificities for arom. and heterocyclic aldehydes with purified human liver ALDH1 and ALDH2, and also with yeast mitochondrial ALDH2 for comparison.  The kinetic assay for the human ALDHs was performed in 50 mM HEPES, pH 7.5 and 25°, contg. 0.5 mM NAD, 1.7% acetonitrile (as a solvent carrier for aldehydes), and varied concns. of substrate, and for yeast ALDH2 the assay was detd. in the same reaction mixt. except contg. 3 mM NAD and the addn. of 200 mM KCl.  With respect to phenylacetaldehyde, 2-phenylpropionaldehyde, benzaldehyde, p-nitrobenzaldehyde, cinnamaldehyde, 2-furaldehyde and indole-3-acetaldehyde, human liver ALDH1 exhibited Km values in the range of 0.25-4.8 μM, Vmax values of 0.34-2.4 U/mg, and catalytic efficiency (Vmax/Km) values of 0.070-3.9 U/(mg μM).  Human ALDH2 exhibited Km values in the range of <0.15-0.74 μM, Vmax values of 0.039-0.51 U/mg, and Vmax/Km values of 0.15-1.0 U/(mg μM).  Human ALDH1 and ALDH2 exhibited substate Ki values for phenylacetaldehyde of 95 and 430 μM, resp.  Yeast ALDH2 exhibited Km values for straight-chain aliph. aldehydes (C1-C10) of 2.3-210 μM, and substrate Ki values (C2-C10) of 79-2900 μM, with a trend of having smaller Km and Ki values for longer chain lengths; and Km values for cinnamaldehyde, benzaldehyde, and 2-furaldehyde of 5.0, 79, and 1000 μM, resp.  Therefore, human ALDH1/ALDH2 and yeast ALDH2 can contribute to the detoxification or metab. of various exogenous/endogenous aliph. and arom. aldehydes.  The systematic changes in kinetic parameters for the oxidn. of structurally related aldehydes may reflect subtle functional topog. distinctions of the substrate pocket for human and yeast ALDHs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXbpkm6VyDdLVg90H21EOLACvtfcHk0lhk-gJB2IHY2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFOju78%253D&md5=f82b3d7e3693f4832c02fdcc18c19bc5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2008.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2008.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%2BF.%26aulast%3DHan%26aufirst%3DC.%2BL.%26aulast%3DYin%26aufirst%3DS.%2BJ.%26atitle%3DSubstrate%2520specificity%2520of%2520human%2520and%2520yeast%2520aldehyde%2520dehydrogenases%26jtitle%3DChem.%2520Biol.%2520Interact.%26date%3D2009%26volume%3D178%26spage%3D36%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budas, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churchill, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Disatnik, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochly-Rosen, D.</span><span> </span><span class="NLM_article-title">Activation of aldehyde dehydrogenase-2 reduces ischemic damage to the heart</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">321</span><span class="NLM_x">, </span> <span class="NLM_fpage">1493</span><span class="NLM_x">–</span> <span class="NLM_lpage">1495</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1126%2Fscience.1158554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=18787169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtV2jsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=1493-1495&author=C.+H.+Chenauthor=G.+R.+Budasauthor=E.+N.+Churchillauthor=M.+H.+Disatnikauthor=T.+D.+Hurleyauthor=D.+Mochly-Rosen&title=Activation+of+aldehyde+dehydrogenase-2+reduces+ischemic+damage+to+the+heart"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Aldehyde Dehydrogenase-2 Reduces Ischemic Damage to the Heart</span></div><div class="casAuthors">Chen, Che-Hong; Budas, Grant R.; Churchill, Eric N.; Disatnik, Marie-Helene; Hurley, Thomas D.; Mochly-Rosen, Daria</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">5895</span>),
    <span class="NLM_cas:pages">1493-1495</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">There is substantial interest in the development of drugs that limit the extent of ischemia-induced cardiac damage caused by myocardial infarction or by certain surgical procedures.  Here, using an unbiased proteomic search, we identified mitochondrial aldehyde dehydrogenase 2 (ALDH2) as an enzyme whose activation correlates with reduced ischemic heart damage in rodent models.  A high-throughput screen yielded a small-mol. activator of ALDH2 (Alda-1) that, when administered to rats before an ischemic event, reduced infarct size by 60%, most likely through its inhibitory effect on the formation of cytotoxic aldehydes.  In vitro, Alda-1 was a particularly effective activator of ALDH2*2, an inactive mutant form of the enzyme that is found in 40% of East Asian populations.  Thus, pharmacol. enhancement of ALDH2 activity may be useful for patients with wild-type or mutant ALDH2 who are subjected to cardiac ischemia, such as during coronary bypass surgery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXwr6e-l58-7Vg90H21EOLACvtfcHk0lhk-gJB2IHY2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtV2jsb7O&md5=b0d4c3a13131a86c2373bf1b138ce8cc</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscience.1158554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1158554%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DBudas%26aufirst%3DG.%2BR.%26aulast%3DChurchill%26aufirst%3DE.%2BN.%26aulast%3DDisatnik%26aufirst%3DM.%2BH.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DMochly-Rosen%26aufirst%3DD.%26atitle%3DActivation%2520of%2520aldehyde%2520dehydrogenase-2%2520reduces%2520ischemic%2520damage%2520to%2520the%2520heart%26jtitle%3DScience%26date%3D2008%26volume%3D321%26spage%3D1493%26epage%3D1495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimble-Hill, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khanna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivanova, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroueh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Discovery of novel regulators of aldehyde dehydrogenase isoenzymes</span> <span class="citation_source-journal">Chem. Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">191</span><span class="NLM_x">, </span> <span class="NLM_fpage">153</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2011.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=21349255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFajtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2011&pages=153-158&author=B.+Parajuliauthor=A.+C.+Kimble-Hillauthor=M.+Khannaauthor=Y.+Ivanovaauthor=S.+Merouehauthor=T.+D.+Hurley&title=Discovery+of+novel+regulators+of+aldehyde+dehydrogenase+isoenzymes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel regulators of aldehyde dehydrogenase isoenzymes</span></div><div class="casAuthors">Parajuli, Bibek; Kimble-Hill, Ann C.; Khanna, May; Ivanova, Yvelina; Meroueh, Samy; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">153-158</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Over the past three years we have been involved in high-throughput screening in an effort to discover novel small mol. modulators of aldehyde dehydrogenase (ALDH) activity.  In particular, we have been interested in both the activation and inhibition of the three commonly studied isoenzymes, ALDH1A1, ALDH2 and ALDH3A1, as their distinct, yet overlapping substrate specificities, present a particularly difficult challenge for inhibitor discovery and design.  Activation of ALDH2 has been shown to benefit cardiovascular outcome following periods of ischemia and renewed interest in specific inhibition of ALDH2 has application for alc. aversion therapy, and more recently, in cocaine addiction.  In contrast, inhibition of either ALDH1A1 or ALDH3A1 has application in cancer treatments where the isoenzymes are commonly over-expressed and serve as markers for cancer stem cells.  We are taking two distinct approaches for these screens: in vitro enzyme activity screens using chem. libraries and virtual computational screens using the structures of the target enzymes as filters for identifying potential inhibitors, followed by in vitro testing of their ability to inhibit their intended targets.  We have identified selective inhibitors of each of these three isoenzymes with inhibition consts. in the high nanomolar to low micromolar range from these screening procedures.  Together, these inhibitors provide proof for concept that selective inhibition of these broad specificity general detoxication enzymes through small mol. discovery and design is possible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcKIxSAscaUbVg90H21EOLACvtfcHk0liGTPGLwGMvWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFajtbc%253D&md5=f8c284f0a5ad00a61f68cd49e5d4bff1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2011.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2011.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DKimble-Hill%26aufirst%3DA.%2BC.%26aulast%3DKhanna%26aufirst%3DM.%26aulast%3DIvanova%26aufirst%3DY.%26aulast%3DMeroueh%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDiscovery%2520of%2520novel%2520regulators%2520of%2520aldehyde%2520dehydrogenase%2520isoenzymes%26jtitle%3DChem.%2520Biol.%2520Interact.%26date%3D2011%26volume%3D191%26spage%3D153%26epage%3D158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Khanna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimble-Hill, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Miller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baskaran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dria, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochly-Rosen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Discovery of a novel class of covalent inhibitor for aldehyde dehydrogenases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">43486</span><span class="NLM_x">–</span> <span class="NLM_lpage">43494</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=43486-43494&author=M.+Khannaauthor=C.+H.+Chenauthor=A.+Kimble-Hillauthor=B.+Parajuliauthor=S.+Perez-Millerauthor=S.+Baskaranauthor=J.+Kimauthor=K.+Driaauthor=V.+Vasiliouauthor=D.+Mochly-Rosenauthor=T.+D.+Hurley&title=Discovery+of+a+novel+class+of+covalent+inhibitor+for+aldehyde+dehydrogenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKhanna%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DKimble-Hill%26aufirst%3DA.%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DPerez-Miller%26aufirst%3DS.%26aulast%3DBaskaran%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DDria%26aufirst%3DK.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DMochly-Rosen%26aufirst%3DD.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520class%2520of%2520covalent%2520inhibitor%2520for%2520aldehyde%2520dehydrogenases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D43486%26epage%3D43494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Moreb, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ucar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amory, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldstein, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostmark, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, L. J.</span><span> </span><span class="NLM_article-title">The enzymatic activity of human aldehyde dehydrogenases 1A2 and 2 (ALDH1A2 and ALDH2) is detected by Aldefluor, inhibited by diethylaminobenzaldehyde and has significant effects on cell proliferation and drug resistance</span> <span class="citation_source-journal">Chem. Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">195</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2011.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=22079344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Oqs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2012&pages=52-60&author=J.+S.+Morebauthor=D.+Ucarauthor=S.+Hanauthor=J.+K.+Amoryauthor=A.+S.+Goldsteinauthor=B.+Ostmarkauthor=L.+J.+Chang&title=The+enzymatic+activity+of+human+aldehyde+dehydrogenases+1A2+and+2+%28ALDH1A2+and+ALDH2%29+is+detected+by+Aldefluor%2C+inhibited+by+diethylaminobenzaldehyde+and+has+significant+effects+on+cell+proliferation+and+drug+resistance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The enzymatic activity of human aldehyde dehydrogenases 1A2 and 2 (ALDH1A2 and ALDH2) is detected by Aldefluor, inhibited by diethylaminobenzaldehyde and has significant effects on cell proliferation and drug resistance</span></div><div class="casAuthors">Moreb, Jan S.; Ucar, Deniz; Han, Shuhong; Amory, John K.; Goldstein, Alex S.; Ostmark, Blanca; Chang, Lung-Ji</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-60</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">There has been a new interest in using aldehyde dehydrogenase (ALDH) activity as one marker for stem cells since the Aldefluor flow cytometry-based assay has become available.  Diethylaminobenzaldehyde (DEAB), used in the Aldeflour assay, has been considered a specific inhibitor for ALDH1A1 isoform.  In this study, we explore the effects of human ALDH isoenzymes, ALDH1A2 and ALDH2, on drug resistance and proliferation, and the specificity of DEAB as an inhibitor.  We also screened for the expression of 19 ALDH isoenzymes in K562 cells using TaqMan Low D. Array (TLDA).  We used lentiviral vectors contg. the full cDNA length of either ALDH2 or ALDH1A2 to over express the enzymes in K562 leukemia and H1299 lung cancer cell lines.  Successful expression was measured by activity assay, Western blot, RT-PCR, and Aldefluor assay.  Both cell lines, with either ALDH1A2 or ALDH2, exhibited higher cell proliferation rates, higher clonal efficiency, and increased drug resistance to 4-hydroperoxycyclophosphamide and doxorubicin.  In order to study the specificity of known ALDH activity inhibitors, DEAB and disulfiram, we incubated each cell line with either inhibitor and measured the remaining ALDH enzymic activity.  Both inhibitors reduced ALDH activity of both isoenzymes by 65-90%.  Furthermore, our TLDA results revealed that ALDH1, ALDH7, ALDH3 and ALDH8 are expressed in K562 cells.  We conclude that DEAB is not a specific inhibitor for ALDH1A1 and that Aldefluor assay is not specific for ALDH1A1 activity.  In addn., other ALDH isoenzymes seem to play a major role in the biol. and drug resistance of various malignant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5EFhYLiyR8rVg90H21EOLACvtfcHk0liGTPGLwGMvWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Oqs7fP&md5=ef48a5eba2e4884fb70e0cde6437f73b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2011.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2011.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DMoreb%26aufirst%3DJ.%2BS.%26aulast%3DUcar%26aufirst%3DD.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DAmory%26aufirst%3DJ.%2BK.%26aulast%3DGoldstein%26aufirst%3DA.%2BS.%26aulast%3DOstmark%26aufirst%3DB.%26aulast%3DChang%26aufirst%3DL.%2BJ.%26atitle%3DThe%2520enzymatic%2520activity%2520of%2520human%2520aldehyde%2520dehydrogenases%25201A2%2520and%25202%2520%2528ALDH1A2%2520and%2520ALDH2%2529%2520is%2520detected%2520by%2520Aldefluor%252C%2520inhibited%2520by%2520diethylaminobenzaldehyde%2520and%2520has%2520significant%2520effects%2520on%2520cell%2520proliferation%2520and%2520drug%2520resistance%26jtitle%3DChem.%2520Biol.%2520Interact.%26date%3D2012%26volume%3D195%26spage%3D52%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Morgan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dria, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title"><i>N</i>,<i>N</i>-Diethylaminobenzaldehyde (DEAB) as a substrate and mechanism-based inhibitor for human ALDH isoenzymes</span> <span class="citation_source-journal">Chem.–Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1016/j.cbi.2014.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2014.12.008" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=C.+A.+Morganauthor=B.+Parajuliauthor=C.+Buchmanauthor=K.+Driaauthor=T.+D.+Hurley&title=N%2CN-Diethylaminobenzaldehyde+%28DEAB%29+as+a+substrate+and+mechanism-based+inhibitor+for+human+ALDH+isoenzymes&doi=10.1016%2Fj.cbi.2014.12.008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DBuchman%26aufirst%3DC.%26aulast%3DDria%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DN%252CN-Diethylaminobenzaldehyde%2520%2528DEAB%2529%2520as%2520a%2520substrate%2520and%2520mechanism-based%2520inhibitor%2520for%2520human%2520ALDH%2520isoenzymes%26jtitle%3DChem.%25E2%2580%2593Biol.%2520Interact.%26date%3D2014%26doi%3D10.1016%2Fj.cbi.2014.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Morgan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Development of a high-throughput in vitro assay to identify selective inhibitors for human ALDH1A1</span> <span class="citation_source-journal">Chem.–Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1016/j.cbi.2014.10.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2014.10.028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=C.+A.+Morganauthor=T.+D.+Hurley&title=Development+of+a+high-throughput+in+vitro+assay+to+identify+selective+inhibitors+for+human+ALDH1A1&doi=10.1016%2Fj.cbi.2014.10.028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.10.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.10.028%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520a%2520high-throughput%2520in%2520vitro%2520assay%2520to%2520identify%2520selective%2520inhibitors%2520for%2520human%2520ALDH1A1%26jtitle%3DChem.%25E2%2580%2593Biol.%2520Interact.%26date%3D2014%26doi%3D10.1016%2Fj.cbi.2014.10.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Perez-Miller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Younus, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochly-Rosen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Alda-1 is an agonist and chemical chaperone for the common human aldehyde dehydrogenase 2 variant</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">164</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1038%2Fnsmb.1737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=20062057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsVeisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=159-164&author=S.+Perez-Millerauthor=H.+Younusauthor=R.+Vanamauthor=C.+H.+Chenauthor=D.+Mochly-Rosenauthor=T.+D.+Hurley&title=Alda-1+is+an+agonist+and+chemical+chaperone+for+the+common+human+aldehyde+dehydrogenase+2+variant"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Alda-1 is an agonist and chemical chaperone for the common human aldehyde dehydrogenase 2 variant</span></div><div class="casAuthors">Perez-Miller, Samantha; Younus, Hina; Vanam, Ram; Chen, Che-Hong; Mochly-Rosen, Daria; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">159-164</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In approx. one billion people, a point mutation inactivates a key detoxifying enzyme, aldehyde dehydrogenase (ALDH2).  This mitochondrial enzyme metabolizes toxic biogenic and environmental aldehydes, including the endogenously produced 4-hydroxynonenal (4HNE) and the environmental pollutant acrolein, and also bioactivates nitroglycerin.  ALDH2 is best known, however, for its role in ethanol metab.  The accumulation of acetaldehyde following the consumption of even a single alc. beverage leads to the Asian alc.-induced flushing syndrome in ALDH2*2 homozygotes.  The ALDH2*2 allele is semidominant, and heterozygotic individuals show a similar but less severe phenotype.  We recently identified a small mol., Alda-1, that activates wild-type ALDH2 and restores near-wild-type activity to ALDH2*2.  The structures of Alda-1 bound to ALDH2 and ALDH2*2 reveal how Alda-1 activates the wild-type enzyme and how it restores the activity of ALDH2*2 by acting as a structural chaperone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosPZft24xVerVg90H21EOLACvtfcHk0lh73DhuF69Wsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsVeisg%253D%253D&md5=d7e5592ee6a832b74c5470da9afe7e6b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1737%26sid%3Dliteratum%253Aachs%26aulast%3DPerez-Miller%26aufirst%3DS.%26aulast%3DYounus%26aufirst%3DH.%26aulast%3DVanam%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DMochly-Rosen%26aufirst%3DD.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DAlda-1%2520is%2520an%2520agonist%2520and%2520chemical%2520chaperone%2520for%2520the%2520common%2520human%2520aldehyde%2520dehydrogenase%25202%2520variant%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D159%26epage%3D164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Condello, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagdas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matei, D.</span><span> </span><span class="NLM_article-title">beta-Catenin-regulated ALDH1A1 is a target in ovarian cancer spheroids</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1038/onc.2014.178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1038%2Fonc.2014.178" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=S.+Condelloauthor=C.+A.+Morganauthor=S.+Nagdasauthor=L.+Caoauthor=J.+Turekauthor=T.+D.+Hurleyauthor=D.+Matei&title=beta-Catenin-regulated+ALDH1A1+is+a+target+in+ovarian+cancer+spheroids&doi=10.1038%2Fonc.2014.178"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.178%26sid%3Dliteratum%253Aachs%26aulast%3DCondello%26aufirst%3DS.%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DNagdas%26aufirst%3DS.%26aulast%3DCao%26aufirst%3DL.%26aulast%3DTurek%26aufirst%3DJ.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DMatei%26aufirst%3DD.%26atitle%3Dbeta-Catenin-regulated%2520ALDH1A1%2520is%2520a%2520target%2520in%2520ovarian%2520cancer%2520spheroids%26jtitle%3DOncogene%26date%3D2014%26doi%3D10.1038%2Fonc.2014.178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Ni, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiner, H.</span><span> </span><span class="NLM_article-title">Human liver mitochondrial aldehyde dehydrogenase: three-dimensional structure and the restoration of solubility and activity of chimeric forms</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2784</span><span class="NLM_x">–</span> <span class="NLM_lpage">2790</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1110%2Fps.8.12.2784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=10631996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtFyltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1999&pages=2784-2790&author=L.+Niauthor=J.+Zhouauthor=T.+D.+Hurleyauthor=H.+Weiner&title=Human+liver+mitochondrial+aldehyde+dehydrogenase%3A+three-dimensional+structure+and+the+restoration+of+solubility+and+activity+of+chimeric+forms"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Human liver mitochondrial aldehyde dehydrogenase: Three-dimensional structure and the restoration of solubility and activity of chimeric forms</span></div><div class="casAuthors">Ni, Li; Zhou, Jianzhong; Hurley, Thomas D.; Weiner, Henry</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2784-2790</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Human liver cytosolic and mitochondrial isoenzymes of aldehyde dehydrogenase share 70% sequence identity.  However, the first 21 residues are not conserved between the human isoenzymes (15% identity).  The three-dimensional structures of the beef mitochondrial and sheep cytosolic forms have virtually identical three-dimensional structures.  Here, we solved the structure of the human mitochondrial enzyme and found it to be identical to the beef enzyme.  The first 21 residues are found on the surface of the enzyme and make no contact with other subunits in the tetramer.  A pair of chimeric enzymes between the human isoenzymes was made.  Each chimera had the first 21 residues from one isoenzyme and the remaining 479 from the other.  When the first 21 residues were from the mitochondrial isoenzyme, an enzyme with cytosolic-like properties was produced.  The other was expressed but was insol.  It was possible to restore soly. and activity to the chimera that had the first 21 cytosolic residues fused to the balance of the mitochondrial enzyme by making point mutations to residues at the N-terminal end.  When residue 19 was changed from tyrosine to a cysteine, the residue found in the mitochondrial form, an active enzyme could be made, although the Km for NAD+ was 35 times higher than the native mitochondrial isoenzyme and the specific activity was reduced by 75%.  This residue interacts with residue 203, a nonconserved, nonactive site residue.  A mutation of residue 18, which also interacts with 203, restored soly., but not activity.  Mutation to residue 15, which interacts with 104, also restored soly. but not activity.  It appears that to have a sol. or active enzyme, a favorable interaction must occur between a residue in a surface loop and a residue elsewhere in the mol. even though neither make contact with the active site region of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreCoI5nn3RC7Vg90H21EOLACvtfcHk0lh73DhuF69Wsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtFyltA%253D%253D&md5=71f076c641f5b09dbcbb13818d20a909</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1110%2Fps.8.12.2784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.8.12.2784%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DHuman%2520liver%2520mitochondrial%2520aldehyde%2520dehydrogenase%253A%2520three-dimensional%2520structure%2520and%2520the%2520restoration%2520of%2520solubility%2520and%2520activity%2520of%2520chimeric%2520forms%26jtitle%3DProtein%2520Sci.%26date%3D1999%26volume%3D8%26spage%3D2784%26epage%3D2790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Srivastava, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watowich, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrash, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhatnagar, A.</span><span> </span><span class="NLM_article-title">Structural and kinetic determinants of aldehyde reduction by aldose reductase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">42</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi981794l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=42-54&author=S.+Srivastavaauthor=S.+J.+Watowichauthor=J.+M.+Petrashauthor=S.+K.+Srivastavaauthor=A.+Bhatnagar&title=Structural+and+kinetic+determinants+of+aldehyde+reduction+by+aldose+reductase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fbi981794l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi981794l%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DS.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26aulast%3DPetrash%26aufirst%3DJ.%2BM.%26aulast%3DSrivastava%26aufirst%3DS.%2BK.%26aulast%3DBhatnagar%26aufirst%3DA.%26atitle%3DStructural%2520and%2520kinetic%2520determinants%2520of%2520aldehyde%2520reduction%2520by%2520aldose%2520reductase%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D42%26epage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Moore, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blythe, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, E. N.</span><span> </span><span class="NLM_article-title">Sheep liver cytosolic aldehyde dehydrogenase: the structure reveals the basis for the retinal specificity of class 1 aldehyde dehydrogenases</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1541</span><span class="NLM_x">–</span> <span class="NLM_lpage">1551</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=1541-1551&author=S.+A.+Mooreauthor=H.+M.+Bakerauthor=T.+J.+Blytheauthor=K.+E.+Kitsonauthor=T.+M.+Kitsonauthor=E.+N.+Baker&title=Sheep+liver+cytosolic+aldehyde+dehydrogenase%3A+the+structure+reveals+the+basis+for+the+retinal+specificity+of+class+1+aldehyde+dehydrogenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DS.%2BA.%26aulast%3DBaker%26aufirst%3DH.%2BM.%26aulast%3DBlythe%26aufirst%3DT.%2BJ.%26aulast%3DKitson%26aufirst%3DK.%2BE.%26aulast%3DKitson%26aufirst%3DT.%2BM.%26aulast%3DBaker%26aufirst%3DE.%2BN.%26atitle%3DSheep%2520liver%2520cytosolic%2520aldehyde%2520dehydrogenase%253A%2520the%2520structure%2520reveals%2520the%2520basis%2520for%2520the%2520retinal%2520specificity%2520of%2520class%25201%2520aldehyde%2520dehydrogenases%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26spage%3D1541%26epage%3D1551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Lamb, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newcomer, M. E.</span><span> </span><span class="NLM_article-title">The structure of retinal dehydrogenase type II at 2.7 A resolution: implications for retinal specificity</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">6003</span><span class="NLM_x">–</span> <span class="NLM_lpage">6011</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi9900471" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=6003-6011&author=A.+L.+Lambauthor=M.+E.+Newcomer&title=The+structure+of+retinal+dehydrogenase+type+II+at+2.7+A+resolution%3A+implications+for+retinal+specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fbi9900471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9900471%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DA.%2BL.%26aulast%3DNewcomer%26aufirst%3DM.%2BE.%26atitle%3DThe%2520structure%2520of%2520retinal%2520dehydrogenase%2520type%2520II%2520at%25202.7%2520A%2520resolution%253A%2520implications%2520for%2520retinal%2520specificity%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D6003%26epage%3D6011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Keung, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vallee, B. L.</span><span> </span><span class="NLM_article-title">Daidzin: a potent, selective inhibitor of human mitochondrial aldehyde dehydrogenase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">1247</span><span class="NLM_x">–</span> <span class="NLM_lpage">1251</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=1247-1251&author=W.+M.+Keungauthor=B.+L.+Vallee&title=Daidzin%3A+a+potent%2C+selective+inhibitor+of+human+mitochondrial+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKeung%26aufirst%3DW.%2BM.%26aulast%3DVallee%26aufirst%3DB.%2BL.%26atitle%3DDaidzin%253A%2520a%2520potent%252C%2520selective%2520inhibitor%2520of%2520human%2520mitochondrial%2520aldehyde%2520dehydrogenase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1993%26volume%3D90%26spage%3D1247%26epage%3D1251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Lowe, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keung, W. M.</span><span> </span><span class="NLM_article-title">Structure of daidzin, a naturally occurring anti-alcohol-addiction agent, in complex with human mitochondrial aldehyde dehydrogenase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4482</span><span class="NLM_x">–</span> <span class="NLM_lpage">4487</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800488j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4482-4487&author=E.+D.+Loweauthor=G.+Y.+Gaoauthor=L.+N.+Johnsonauthor=W.+M.+Keung&title=Structure+of+daidzin%2C+a+naturally+occurring+anti-alcohol-addiction+agent%2C+in+complex+with+human+mitochondrial+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm800488j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800488j%26sid%3Dliteratum%253Aachs%26aulast%3DLowe%26aufirst%3DE.%2BD.%26aulast%3DGao%26aufirst%3DG.%2BY.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26aulast%3DKeung%26aufirst%3DW.%2BM.%26atitle%3DStructure%2520of%2520daidzin%252C%2520a%2520naturally%2520occurring%2520anti-alcohol-addiction%2520agent%252C%2520in%2520complex%2520with%2520human%2520mitochondrial%2520aldehyde%2520dehydrogenase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4482%26epage%3D4487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Vasiliou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandoval, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Backos, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reigan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanaspa, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koppaka, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, D. C.</span><span> </span><span class="NLM_article-title">ALDH16A1 is a novel non-catalytic enzyme that may be involved in the etiology of gout via protein–protein interactions with HPRT1</span> <span class="citation_source-journal">Chem.–Biol. Interact</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">202</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2012.12.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=23348497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFamsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=202&publication_year=2013&pages=22-31&author=V.+Vasiliouauthor=M.+Sandovalauthor=D.+S.+Backosauthor=B.+C.+Jacksonauthor=Y.+Chenauthor=P.+Reiganauthor=M.+A.+Lanaspaauthor=R.+J.+Johnsonauthor=V.+Koppakaauthor=D.+C.+Thompson&title=ALDH16A1+is+a+novel+non-catalytic+enzyme+that+may+be+involved+in+the+etiology+of+gout+via+protein%E2%80%93protein+interactions+with+HPRT1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH16A1 is a novel non-catalytic enzyme that may be involved in the etiology of gout via protein-protein interactions with HPRT1</span></div><div class="casAuthors">Vasiliou, Vasilis; Sandoval, Monica; Backos, Donald S.; Jackson, Brian C.; Chen, Ying; Reigan, Philip; Lanaspa, Miguel A.; Johnson, Richard J.; Koppaka, Vindhya; Thompson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">202</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">22-31</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Gout, a common form of inflammatory arthritis, is strongly assocd. with elevated uric acid concns. in the blood (hyperuricemia).  A recent study in Icelanders identified a rare missense single nucleotide polymorphism (SNP) in the ALDH16A1 gene, ALDH16A1*2, to be assocd. with gout and serum uric acid levels.  ALDH16A1 is a novel and rather unique member of the ALDH superfamily in relation to its gene and protein structures.  ALDH16 genes are present in fish, amphibians, protista, bacteria but absent from archaea, fungi and plants.  In most mammalian species, two ALDH16A1 spliced variants (ALDH16A1, long form and ALDH16A1v2, short form) have been identified and both are expressed in HepG-2, HK-2 and HK-293 human cell lines.  The ALDH16 proteins contain two ALDH domains (as opposed to one in the other members of the superfamily), four transmembrane and one coiled-coil domains.  The active site of ALDH16 proteins from bacterial, frog and lower animals contain the catalytically important cysteine residue (Cys-302); this residue is absent from the mammalian and fish orthologs.  Mol. modeling predicts that both the short and long forms of human ALDH16A1 protein would lack catalytic activity but may interact with the hypoxanthine-guanine phosphoribosyltransferase (HPRT1) protein, a key enzyme involved in uric acid metab. and gout.  Interestingly, such protein-protein interactions with HPRT1 are predicted to be impaired for the long or short forms of ALDH16A1*2.  These results lead to the intriguing possibility that assocn. between ALDH16A1 and HPRT1 may be required for optimal HPRT activity with disruption of this interaction possibly contributing to the hyperuricemia seen in ALDH16A1*2 carriers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPm2s2vvZfzbVg90H21EOLACvtfcHk0ljxihSV1926CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFamsLs%253D&md5=efe699e4b09fc9256d48e2d6880605e7</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2012.12.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2012.12.018%26sid%3Dliteratum%253Aachs%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DSandoval%26aufirst%3DM.%26aulast%3DBackos%26aufirst%3DD.%2BS.%26aulast%3DJackson%26aufirst%3DB.%2BC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DReigan%26aufirst%3DP.%26aulast%3DLanaspa%26aufirst%3DM.%2BA.%26aulast%3DJohnson%26aufirst%3DR.%2BJ.%26aulast%3DKoppaka%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26atitle%3DALDH16A1%2520is%2520a%2520novel%2520non-catalytic%2520enzyme%2520that%2520may%2520be%2520involved%2520in%2520the%2520etiology%2520of%2520gout%2520via%2520protein%25E2%2580%2593protein%2520interactions%2520with%2520HPRT1%26jtitle%3DChem.%25E2%2580%2593Biol.%2520Interact%26date%3D2013%26volume%3D202%26spage%3D22%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Levi, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yilmaz, O. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duester, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, S. J.</span><span> </span><span class="NLM_article-title">Aldehyde dehydrogenase 1a1 is dispensable for stem cell function in the mouse hematopoietic and nervous systems</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">1670</span><span class="NLM_x">–</span> <span class="NLM_lpage">1680</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=1670-1680&author=B.+P.+Leviauthor=O.+H.+Yilmazauthor=G.+Duesterauthor=S.+J.+Morrison&title=Aldehyde+dehydrogenase+1a1+is+dispensable+for+stem+cell+function+in+the+mouse+hematopoietic+and+nervous+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLevi%26aufirst%3DB.%2BP.%26aulast%3DYilmaz%26aufirst%3DO.%2BH.%26aulast%3DDuester%26aufirst%3DG.%26aulast%3DMorrison%26aufirst%3DS.%2BJ.%26atitle%3DAldehyde%2520dehydrogenase%25201a1%2520is%2520dispensable%2520for%2520stem%2520cell%2520function%2520in%2520the%2520mouse%2520hematopoietic%2520and%2520nervous%2520systems%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26spage%3D1670%26epage%3D1680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Ucar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cogle, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zucali, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostmark, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zori, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreb, J. S.</span><span> </span><span class="NLM_article-title">Aldehyde dehydrogenase activity as a functional marker for lung cancer</span> <span class="citation_source-journal">Chem.–Biol. Interact</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2008.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=18952074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFOju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2009&pages=48-55&author=D.+Ucarauthor=C.+R.+Cogleauthor=J.+R.+Zucaliauthor=B.+Ostmarkauthor=E.+W.+Scottauthor=R.+Zoriauthor=B.+A.+Grayauthor=J.+S.+Moreb&title=Aldehyde+dehydrogenase+activity+as+a+functional+marker+for+lung+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenase activity as a functional marker for lung cancer</span></div><div class="casAuthors">Ucar, Deniz; Cogle, Christopher R.; Zucali, James R.; Ostmark, Blanca; Scott, Edward W.; Zori, Robert; Gray, Brian A.; Moreb, Jan S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">48-55</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenase (ALDH) activity was implicated in multiple biol. and biochem. pathways and was used to identify potential cancer stem cells.  Our main hypothesis is that ALDH activity may be a lung cancer stem cell marker.  Using flow cytometry, we sorted cells with bright (ALDHbr) and dim (ALDHlo) ALDH activity found in H522 lung cancer cell line.  We used in vitro proliferation and colony assays as well as a xenograft animal model to test our hypothesis.  Cytogenetic anal. demonstrated that the ALDHbr cells are indeed a different clone, but when left in normal culture conditions will give rise to ALDHlo cells.  Furthermore, the ALDHbr cells grow slower, have low clonal efficiency, and give rise to morphol. distinct colonies.  The ability to form primary xenografts in NOD/SCID mice by ALDHbr and ALDHlo cells was tested by injecting single cell suspension under the skin in each flank of same animal.  Tumor size was calcd. weekly.  ALDH1A1 and ALDH3A1 immunohistochem. (IHC) was performed on excised tumors.  These tumors were also used to re-establish cell suspension, measure ALDH activity, and re-injection for secondary and tertiary transplants.  The results indicate that both cell types can form tumors but the ones from ALDHbr cells grew much slower in primary recipient mice.  Histol., there was no significant difference in the expression of ALDH in primary tumors originating from ALDHbr or ALDHlo cells.  Secondary and tertiary xenografts originating from ALDHbr grew faster and bigger than those formed by ALDHlo cells.  In conclusion, ALDHbr cells may have some of the traditional features of stem cells in terms of being mostly dormant and slow to divide, but require support of other cells (ALDHlo) to sustain tumor growth.  These observations and the known role of ALDH in drug resistance may have significant therapeutic implications in the treatment of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmiRq2oQb45rVg90H21EOLACvtfcHk0ljxihSV1926CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFOju70%253D&md5=13c50c224f3f14dec7316fe125513c9e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2008.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2008.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DUcar%26aufirst%3DD.%26aulast%3DCogle%26aufirst%3DC.%2BR.%26aulast%3DZucali%26aufirst%3DJ.%2BR.%26aulast%3DOstmark%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DE.%2BW.%26aulast%3DZori%26aufirst%3DR.%26aulast%3DGray%26aufirst%3DB.%2BA.%26aulast%3DMoreb%26aufirst%3DJ.%2BS.%26atitle%3DAldehyde%2520dehydrogenase%2520activity%2520as%2520a%2520functional%2520marker%2520for%2520lung%2520cancer%26jtitle%3DChem.%25E2%2580%2593Biol.%2520Interact%26date%3D2009%26volume%3D178%26spage%3D48%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Landen, C. N.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katre, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steg, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nick, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mejia, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jennings, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gershenson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bast, R. C.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Berestein, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sood, A. K.</span><span> </span><span class="NLM_article-title">Targeting aldehyde dehydrogenase cancer stem cells in ovarian cancer</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3186</span><span class="NLM_x">–</span> <span class="NLM_lpage">3199</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1158%2F1535-7163.MCT-10-0563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=20889728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=3186-3199&author=C.+N.+Landenauthor=B.+Goodmanauthor=A.+A.+Katreauthor=A.+D.+Stegauthor=A.+M.+Nickauthor=R.+L.+Stoneauthor=L.+D.+Millerauthor=P.+V.+Mejiaauthor=N.+B.+Jenningsauthor=D.+M.+Gershensonauthor=R.+C.+Bastauthor=R.+L.+Colemanauthor=G.+Lopez-Beresteinauthor=A.+K.+Sood&title=Targeting+aldehyde+dehydrogenase+cancer+stem+cells+in+ovarian+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Aldehyde Dehydrogenase Cancer Stem Cells in Ovarian Cancer</span></div><div class="casAuthors">Landen, Charles N., Jr.; Goodman, Blake; Katre, Ashwini A.; Steg, Adam D.; Nick, Alpa M.; Stone, Rebecca L.; Miller, Lance D.; Mejia, Pablo Vivas; Jennings, Nicolas B.; Gershenson, David M.; Bast, Robert C., Jr.; Coleman, Robert L.; Lopez-Berestein, Gabriel; Sood, Anil K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3186-3199</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenase-1A1 (ALDH1A1) expression characterizes a subpopulation of cells with tumor-initiating or cancer stem cell properties in several malignancies.  Our goal was to characterize the phenotype of ALDH1A1-pos. ovarian cancer cells and examine the biol. effects of ALDH1A1 gene silencing.  In our anal. of multiple ovarian cancer cell lines, we found that ALDH1A1 expression and activity was significantly higher in taxane- and platinum-resistant cell lines.  In patient samples, 72.9% of ovarian cancers had ALDH1A1 expression in which the percentage of ALDH1A1-pos. cells correlated neg. with progression-free survival (6.05 vs. 13.81 mo; P < 0.035).  Subpopulations of A2780cp20 cells with ALDH1A1 activity were isolated for orthotopic tumor-initiating studies, where tumorigenicity was approx. 50-fold higher with ALDH1A1-pos. cells.  Interestingly, tumors derived from ALDH1A1-pos. cells gave rise to both ALDH1A1-pos. and ALDH1A1-neg. populations, but ALDH1A1-neg. cells could not generate ALDH1A1-pos. cells.  In an in vivo orthotopic mouse model of ovarian cancer, ALDH1A1 silencing using nanoliposomal siRNA sensitized both taxane- and platinum-resistant cell lines to chemotherapy, significantly reducing tumor growth in mice compared with chemotherapy alone (a 74%-90% redn.; P < 0.015).  These data show that the ALDH1A1 subpopulation is assocd. with chemoresistance and outcome in ovarian cancer patients, and targeting ALDH1A1 sensitizes resistant cells to chemotherapy.  ALDH1A1-pos. cells have enhanced, but not abs., tumorigenicity but do have differentiation capacity lacking in ALDH1A1-neg. cells.  This enzyme may be important for identification and targeting of chemoresistant cell populations in ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeKpHNJYkfVLVg90H21EOLACvtfcHk0lgDuKXszCtIvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsL%252FP&md5=2ddb105fb819e82928e7c95994ee317e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-10-0563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-10-0563%26sid%3Dliteratum%253Aachs%26aulast%3DLanden%26aufirst%3DC.%2BN.%26aulast%3DGoodman%26aufirst%3DB.%26aulast%3DKatre%26aufirst%3DA.%2BA.%26aulast%3DSteg%26aufirst%3DA.%2BD.%26aulast%3DNick%26aufirst%3DA.%2BM.%26aulast%3DStone%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DL.%2BD.%26aulast%3DMejia%26aufirst%3DP.%2BV.%26aulast%3DJennings%26aufirst%3DN.%2BB.%26aulast%3DGershenson%26aufirst%3DD.%2BM.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DColeman%26aufirst%3DR.%2BL.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DSood%26aufirst%3DA.%2BK.%26atitle%3DTargeting%2520aldehyde%2520dehydrogenase%2520cancer%2520stem%2520cells%2520in%2520ovarian%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D3186%26epage%3D3199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mei, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stass, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, F.</span><span> </span><span class="NLM_article-title">ALDH1A1 is a marker for malignant prostate stem cells and predictor of prostate cancer patients’ outcome</span> <span class="citation_source-journal">Lab. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">234</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2010&pages=234-244&author=T.+Liauthor=Y.+Suauthor=Y.+Meiauthor=Q.+Lengauthor=B.+Lengauthor=Z.+Liuauthor=S.+A.+Stassauthor=F.+Jiang&title=ALDH1A1+is+a+marker+for+malignant+prostate+stem+cells+and+predictor+of+prostate+cancer+patients%E2%80%99+outcome"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DT.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DLeng%26aufirst%3DQ.%26aulast%3DLeng%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DStass%26aufirst%3DS.%2BA.%26aulast%3DJiang%26aufirst%3DF.%26atitle%3DALDH1A1%2520is%2520a%2520marker%2520for%2520malignant%2520prostate%2520stem%2520cells%2520and%2520predictor%2520of%2520prostate%2520cancer%2520patients%25E2%2580%2599%2520outcome%26jtitle%3DLab.%2520Invest.%26date%3D2010%26volume%3D90%26spage%3D234%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Sodek, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringuette, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, T. J.</span><span> </span><span class="NLM_article-title">Compact spheroid formation by ovarian cancer cells is associated with contractile behavior and an invasive phenotype</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">2060</span><span class="NLM_x">–</span> <span class="NLM_lpage">2070</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1002%2Fijc.24188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=19132753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktVWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2009&pages=2060-2070&author=K.+L.+Sodekauthor=M.+J.+Ringuetteauthor=T.+J.+Brown&title=Compact+spheroid+formation+by+ovarian+cancer+cells+is+associated+with+contractile+behavior+and+an+invasive+phenotype"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Compact spheroid formation by ovarian cancer cells is associated with contractile behavior and an invasive phenotype</span></div><div class="casAuthors">Sodek, Katharine L.; Ringuette, Maurice J.; Brown, Theodore J.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2060-2070</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Ovarian cancer cells are present in malignant ascites both as individual cells and as multicellular spheroid aggregates.  Although spheroid formation affords protection of cancer cells against some chemotherapeutic agents, it has not been established whether a relationship exists between invasive behavior and predisposition to spheroid formation.  Aspects of spheroid formation, including cell-matrix adhesion, remodeling and contractility are characteristic myofibroblast-like behaviors assocd. with fibrosis that contribute to tumor growth and dissemination.  We explored the possibility that cell behaviors that promote spheroid formation also facilitate invasion.  Our anal. of 6 human ovarian cancer cell lines indicated that ovarian cancer cells possessing myofibroblast-like properties formed compact spheroids and invaded 3D matrixes.  These cells readily contracted collagen I gels, possessed a spindle-like morphol., and had elevated expression of genes assocd. with the TGFβ-mediated fibrotic response and/or β1 integrin function, including fibronectin (FN), connective tissue growth factor (CTGF/CCN2), lysyl oxidase (LOX1), tissue transglutaminase 2 (TGM2) and urinary plasminogen activator receptor (uPAR).  Whereas cell aggregation was induced by TGFβ, and by β1-integrin overexpression and activation, these treatments did not stimulate the contractile activity required for spheroid compaction.  The pos. relationship found between compact spheroid formation and invasive behavior implies a preferential survival of an invasive subpopulation of ovarian cancer cells, as cells in spheroids are more resistant to several chemotherapeutics.  Preventing the formation of ovarian cancer spheroids may represent a novel strategy to improve the efficacy of existing therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcsLZtclB7u7Vg90H21EOLACvtfcHk0lgDuKXszCtIvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktVWntbc%253D&md5=a6d07729f906ffc92e2892a81df196f6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fijc.24188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.24188%26sid%3Dliteratum%253Aachs%26aulast%3DSodek%26aufirst%3DK.%2BL.%26aulast%3DRinguette%26aufirst%3DM.%2BJ.%26aulast%3DBrown%26aufirst%3DT.%2BJ.%26atitle%3DCompact%2520spheroid%2520formation%2520by%2520ovarian%2520cancer%2520cells%2520is%2520associated%2520with%2520contractile%2520behavior%2520and%2520an%2520invasive%2520phenotype%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2009%26volume%3D124%26spage%3D2060%26epage%3D2070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fishel, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Selective ALDH3A1 inhibition by benzimidazole analogues increase mafosfamide sensitivity in cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">449</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm401508p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=449-461&author=B.+Parajuliauthor=M.+L.+Fishelauthor=T.+D.+Hurley&title=Selective+ALDH3A1+inhibition+by+benzimidazole+analogues+increase+mafosfamide+sensitivity+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm401508p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401508p%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DSelective%2520ALDH3A1%2520inhibition%2520by%2520benzimidazole%2520analogues%2520increase%2520mafosfamide%2520sensitivity%2520in%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D449%26epage%3D461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiadis, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fishel, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Development of selective inhibitors for human aldehyde dehydrogenase 3A1 (ALDH3A1) for the enhancement of cyclophosphamide cytotoxicity</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=701-712&author=B.+Parajuliauthor=T.+M.+Georgiadisauthor=M.+L.+Fishelauthor=T.+D.+Hurley&title=Development+of+selective+inhibitors+for+human+aldehyde+dehydrogenase+3A1+%28ALDH3A1%29+for+the+enhancement+of+cyclophosphamide+cytotoxicity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DGeorgiadis%26aufirst%3DT.%2BM.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520selective%2520inhibitors%2520for%2520human%2520aldehyde%2520dehydrogenase%25203A1%2520%2528ALDH3A1%2529%2520for%2520the%2520enhancement%2520of%2520cyclophosphamide%2520cytotoxicity%26jtitle%3DChemBioChem%26date%3D2014%26volume%3D15%26spage%3D701%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Hammen, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallenga, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiner, H.</span><span> </span><span class="NLM_article-title">Multiple conformations of NAD and NADH when bound to human cytosolic and mitochondrial aldehyde dehydrogenase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">7156</span><span class="NLM_x">–</span> <span class="NLM_lpage">7168</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi012197t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=7156-7168&author=P.+K.+Hammenauthor=A.+Allali-Hassaniauthor=K.+Hallengaauthor=T.+D.+Hurleyauthor=H.+Weiner&title=Multiple+conformations+of+NAD+and+NADH+when+bound+to+human+cytosolic+and+mitochondrial+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fbi012197t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi012197t%26sid%3Dliteratum%253Aachs%26aulast%3DHammen%26aufirst%3DP.%2BK.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHallenga%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DMultiple%2520conformations%2520of%2520NAD%2520and%2520NADH%2520when%2520bound%2520to%2520human%2520cytosolic%2520and%2520mitochondrial%2520aldehyde%2520dehydrogenase%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D7156%26epage%3D7168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Zheng, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiner, H.</span><span> </span><span class="NLM_article-title">Cloning and expression of the full-length cDNAS encoding human liver class 1 and class 2 aldehyde dehydrogenase</span> <span class="citation_source-journal">Alcohol.: Clin. Exp. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">828</span><span class="NLM_x">–</span> <span class="NLM_lpage">831</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1111%2Fj.1530-0277.1993.tb00849.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=8214422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK2cXht1Kgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1993&pages=828-831&author=C.+F.+Zhengauthor=T.+T.+Wangauthor=H.+Weiner&title=Cloning+and+expression+of+the+full-length+cDNAS+encoding+human+liver+class+1+and+class+2+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Cloning and expression of the full-length cDNAs encoding human liver class 1 and class 2 aldehyde dehydrogenase</span></div><div class="casAuthors">Zheng, Chao Feng; Wang, Thomas T. Y.; Weiner, Henry</div><div class="citationInfo"><span class="NLM_cas:title">Alcoholism:  Clinical and Experimental Research</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">828-31</span>CODEN:
                <span class="NLM_cas:coden">ACRSDM</span>;
        ISSN:<span class="NLM_cas:issn">0145-6008</span>.
    </div><div class="casAbstract">The amino acid sequences of both human class 1 and 2 aldehyde dehydrogenase (ALDH) and the sequences of the genes coding for them are known.  Based on this sequence data, primers were designed and full-length cDNAs isolated encoding both isoenzymes from a human liver mRNA pool.  The cDNAs were subcloned in the plasmid pT7-7 and expressed in Escherichia coli with a yield of ∼3 mg ALDH protein/L of cell culture, although only one-third of the enzyme was sol.  The sol. recombinantly expressed ALDHs were purified to homogeneity using a hydroxyacetophenone-Sepharose affinity column.  The mitochondrial isoenzyme had a subunit mol. wt. of 55 kDa, an isoelec. point of 4.9, and a specific activity of 1.10 units/mg, which were in good agreement with that from the native enzyme.  The expressed cytosolic isoenzyme had the same subunit mol. wt. (55 kDa) and pI (5.4) as that reported for the native enzyme and had a specific activity of 0.26 units/mg.  The expressed mitochondrial isoenzyme could be recognized by antibodies raised against rat mitochondrial ALDH, whereas the cytosolic isoenzyme could be recognized by antibody raised against horse cytosolic ALDH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGhJDXkXENbrVg90H21EOLACvtfcHk0lhOmQ9ZFf2UaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXht1Kgtr8%253D&md5=009015297d0e380d0c981bc657666614</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1111%2Fj.1530-0277.1993.tb00849.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1530-0277.1993.tb00849.x%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DC.%2BF.%26aulast%3DWang%26aufirst%3DT.%2BT.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DCloning%2520and%2520expression%2520of%2520the%2520full-length%2520cDNAS%2520encoding%2520human%2520liver%2520class%25201%2520and%2520class%25202%2520aldehyde%2520dehydrogenase%26jtitle%3DAlcohol.%253A%2520Clin.%2520Exp.%2520Res.%26date%3D1993%26volume%3D17%26spage%3D828%26epage%3D831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Macromol. Crystallogr., Part A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhOmQ9ZFf2UaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMacromol.%2520Crystallogr.%252C%2520Part%2520A%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>.  <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=15299374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A280%3ADC%252BD2czpsV2gug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+CCP4+suite%3A+programs+for+protein+crystallography.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+1994%2C+50%2C+760%E2%80%93763."target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">The CCP4 suite: programs for protein crystallography</span></div><div class="casAuthors">Anonymous</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">760-3</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">The CCP4 (Collaborative Computational Project, number 4) program suite is a collection of programs and associated data and subroutine libraries which can be used for macromolecular structure determination by X-ray crystallography.  The suite is designed to be flexible, allowing users a number of methods of achieving their aims and so there may be more than one program to cover each function.  The programs are written mainly in standard Fortran77.  They are from a wide variety of sources but are connected by standard data file formats.  The package has been ported to all the major platforms under both Unix and VMS.  The suite is distributed by anonymous ftp from Daresbury Laboratory and is widely used throughout the world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ1Q-skJYIZ2IYPsCkJIw00fW6udTcc2eboxUus7W6Etbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czpsV2gug%253D%253D&md5=92b759927d6071ea3f20c08b78ad1eb2</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lges4xYDYHfFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Painter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merritt, E. A.</span><span> </span><span class="NLM_article-title">A molecular viewer for the analysis of TLS rigid-body motion in macromolecules</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">471</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=465-471&author=J.+Painterauthor=E.+A.+Merritt&title=A+molecular+viewer+for+the+analysis+of+TLS+rigid-body+motion+in+macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DA%2520molecular%2520viewer%2520for%2520the%2520analysis%2520of%2520TLS%2520rigid-body%2520motion%2520in%2520macromolecules%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D465%26epage%3D471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Painter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merritt, E. A.</span><span> </span><span class="NLM_article-title">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1107%2FS0907444906005270" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=16552146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=439-450&author=J.+Painterauthor=E.+A.+Merritt&title=Optimal+description+of+a+protein+structure+in+terms+of+multiple+groups+undergoing+TLS+motion"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-450</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A single protein crystal structure contains information about dynamic properties of the protein as well as providing a static view of one three-dimensional conformation.  This addnl. information is to be found in the distribution of obsd. electron d. about the mean position of each atom.  It is general practice to account for this by refining a sep. at. displacement parameter (ADP) for each at. center.  However, these same displacements are often described well by simpler models based on TLS (translation/libration/screw) rigid-body motion of large groups of atoms, for example interdomain hinge motion.  A procedure, TLSMD, has been developed that analyzes the distribution of ADPs in a previously refined protein crystal structure in order to generate optimal multi-group TLS descriptions of the constituent protein chains.  TLSMD is applicable to crystal structures at any resoln.  The models generated by TLSMD anal. can significantly improve the std. crystallog. residuals R and Rfree and can reveal intrinsic dynamic properties of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIR_NcneJBNbVg90H21EOLACvtfcHk0lges4xYDYHfFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D&md5=6289483d19fb6dd64fe9b032e9c45005</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0907444906005270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906005270%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DOptimal%2520description%2520of%2520a%2520protein%2520structure%2520in%2520terms%2520of%2520multiple%2520groups%2520undergoing%2520TLS%2520motion%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D439%26epage%3D450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 40 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Sebastiaan T. A. Koenders, Lukas S. Wijaya, Martje N. Erkelens, Alexander T. Bakker, Vera E. van der Noord, Eva J. van Rooden, Lindsey Burggraaff, Pasquale C. Putter, Else Botter, Kim Wals, Hans van den Elst, Hans den Dulk, Bogdan I. Florea, Bob van de Water, Gerard J. P. van Westen, Reina E. Mebius, Herman S. Overkleeft, Sylvia E. Le Dévédec, <span class="NLM_string-name hlFld-ContribAuthor">Mario van der Stelt</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Retinal-Based Probe for the Profiling of Retinaldehyde Dehydrogenases in Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2019,</strong> <em>5 </em>
                                    (12)
                                     , 1965-1974. <a href="https://doi.org/10.1021/acscentsci.9b01022" title="DOI URL">https://doi.org/10.1021/acscentsci.9b01022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.9b01022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.9b01022%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DDevelopment%252Bof%252Ba%252BRetinal-Based%252BProbe%252Bfor%252Bthe%252BProfiling%252Bof%252BRetinaldehyde%252BDehydrogenases%252Bin%252BCancer%252BCells%26aulast%3DKoenders%26aufirst%3DSebastiaan%2BT.%2BA.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07102019%26date%3D12122019%26volume%3D5%26issue%3D12%26spage%3D1965%26epage%3D1974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Brandt
C. Huddle, Edward Grimley, Cameron D. Buchman, Mikhail Chtcherbinine, Bikash Debnath, Pooja Mehta, Kun Yang, Cynthia A. Morgan, Siwei Li, Jeremy Felton, Duxin Sun, Geeta Mehta, Nouri Neamati, Ronald J. Buckanovich, Thomas D. Hurley, <span class="NLM_string-name hlFld-ContribAuthor">Scott D. Larsen</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Optimization of a Novel Class of Aldehyde Dehydrogenase 1A (ALDH1A) Subfamily-Selective Inhibitors as Potential Adjuncts to Ovarian Cancer Chemotherapy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (19)
                                     , 8754-8773. <a href="https://doi.org/10.1021/acs.jmedchem.8b00930" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00930</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00930%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BOptimization%252Bof%252Ba%252BNovel%252BClass%252Bof%252BAldehyde%252BDehydrogenase%252B1A%252B%252528ALDH1A%252529%252BSubfamily-Selective%252BInhibitors%252Bas%252BPotential%252BAdjuncts%252Bto%252BOvarian%252BCancer%252BChemotherapy%26aulast%3DHuddle%26aufirst%3DBrandt%2BC.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D11062018%26date%3D28092018%26date%3D17092018%26volume%3D61%26issue%3D19%26spage%3D8754%26epage%3D8773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shyh-Ming Yang, Natalia J. Martinez, Adam Yasgar, Carina Danchik, Catrine Johansson, Yuhong Wang, Bolormaa Baljinnyam, Amy Q. Wang, Xin Xu, Pranav Shah, Dorian Cheff, Xinran S. Wang, Jacob Roth, Madhu Lal-Nag, James E. Dunford, Udo Oppermann, Vasilis Vasiliou, Anton Simeonov, Ajit Jadhav, <span class="NLM_string-name hlFld-ContribAuthor">David J. Maloney</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Orally Bioavailable, Quinoline-Based Aldehyde Dehydrogenase 1A1 (ALDH1A1) Inhibitors with Potent Cellular Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (11)
                                     , 4883-4903. <a href="https://doi.org/10.1021/acs.jmedchem.8b00270" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00270</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00270%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BOrally%252BBioavailable%25252C%252BQuinoline-Based%252BAldehyde%252BDehydrogenase%252B1A1%252B%252528ALDH1A1%252529%252BInhibitors%252Bwith%252BPotent%252BCellular%252BActivity%26aulast%3DYang%26aufirst%3DShyh-Ming%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D17022018%26date%3D31052018%26date%3D16052018%26volume%3D61%26issue%3D11%26spage%3D4883%26epage%3D4903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yan Chen, Jin-Yi Zhu, Kwon Ho Hong, David C. Mikles, Gunda I. Georg, Alex S. Goldstein, John K. Amory, <span class="NLM_string-name hlFld-ContribAuthor">Ernst Schönbrunn</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (3)
                                     , 582-590. <a href="https://doi.org/10.1021/acschembio.7b00685" title="DOI URL">https://doi.org/10.1021/acschembio.7b00685</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.7b00685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.7b00685%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DStructural%252BBasis%252Bof%252BALDH1A2%252BInhibition%252Bby%252BIrreversible%252Band%252BReversible%252BSmall%252BMolecule%252BInhibitors%26aulast%3DChen%26aufirst%3DYan%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D09082017%26date%3D14122017%26date%3D03012018%26date%3D14122017%26volume%3D13%26issue%3D3%26spage%3D582%26epage%3D590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cameron D.  Buchman</span> and <span class="hlFld-ContribAuthor ">Thomas D.  Hurley</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibition of the Aldehyde Dehydrogenase 1/2 Family by Psoralen and Coumarin Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (6)
                                     , 2439-2455. <a href="https://doi.org/10.1021/acs.jmedchem.6b01825" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01825</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01825%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DInhibition%252Bof%252Bthe%252BAldehyde%252BDehydrogenase%252B1%25252F2%252BFamily%252Bby%252BPsoralen%252Band%252BCoumarin%252BDerivatives%26aulast%3DBuchman%26aufirst%3DCameron%2BD.%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D14122016%26date%3D06032017%26date%3D23032017%26date%3D20022017%26volume%3D60%26issue%3D6%26spage%3D2439%26epage%3D2455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Doohyun  Lee</span>, <span class="hlFld-ContribAuthor ">Seungyeon  Lee</span>, <span class="hlFld-ContribAuthor ">Kwang-Hyeon  Liu</span>, <span class="hlFld-ContribAuthor ">Jong-Sup  Bae</span>, <span class="hlFld-ContribAuthor ">Dong Jae  Baek</span>, and <span class="hlFld-ContribAuthor ">Taeho  Lee</span>  . </span><span class="cited-content_cbyCitation_article-title">Solid-Phase Synthesis of 1,3,7,8-Tetrasubstituted Xanthine Derivatives on Traceless Solid Support. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2016,</strong> <em>18 </em>
                                    (1)
                                     , 70-74. <a href="https://doi.org/10.1021/acscombsci.5b00148" title="DOI URL">https://doi.org/10.1021/acscombsci.5b00148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.5b00148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.5b00148%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DSolid-Phase%252BSynthesis%252Bof%252B1%25252C3%25252C7%25252C8-Tetrasubstituted%252BXanthine%252BDerivatives%252Bon%252BTraceless%252BSolid%252BSupport%26aulast%3DLee%26aufirst%3DDoohyun%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D23092015%26date%3D11112015%26date%3D14122015%26date%3D11012016%26date%3D30112015%26volume%3D18%26issue%3D1%26spage%3D70%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shyh-Ming  Yang</span>, <span class="hlFld-ContribAuthor ">Adam  Yasgar</span>, <span class="hlFld-ContribAuthor ">Bettina  Miller</span>, <span class="hlFld-ContribAuthor ">Madhu  Lal-Nag</span>, <span class="hlFld-ContribAuthor ">Kyle  Brimacombe</span>, <span class="hlFld-ContribAuthor ">Xin  Hu</span>, <span class="hlFld-ContribAuthor ">Hongmao  Sun</span>, <span class="hlFld-ContribAuthor ">Amy  Wang</span>, <span class="hlFld-ContribAuthor ">Xin  Xu</span>, <span class="hlFld-ContribAuthor ">Kimloan  Nguyen</span>, <span class="hlFld-ContribAuthor ">Udo  Oppermann</span>, <span class="hlFld-ContribAuthor ">Marc  Ferrer</span>, <span class="hlFld-ContribAuthor ">Vasilis  Vasiliou</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, and <span class="hlFld-ContribAuthor ">David J.  Maloney</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of NCT-501, a Potent and Selective Theophylline-Based Inhibitor of Aldehyde Dehydrogenase 1A1 (ALDH1A1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (15)
                                     , 5967-5978. <a href="https://doi.org/10.1021/acs.jmedchem.5b00577" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00577</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b00577%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNCT-501%25252C%252Ba%252BPotent%252Band%252BSelective%252BTheophylline-Based%252BInhibitor%252Bof%252BAldehyde%252BDehydrogenase%252B1A1%252B%252528ALDH1A1%252529%26aulast%3DYang%26aufirst%3DShyh-Ming%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D13042015%26date%3D24072015%26date%3D13082015%26volume%3D58%26issue%3D15%26spage%3D5967%26epage%3D5978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gera  Narendra</span>, <span class="hlFld-ContribAuthor ">Baddipadige  Raju</span>, <span class="hlFld-ContribAuthor ">Himanshu  Verma</span>, <span class="hlFld-ContribAuthor ">Bharti  Sapra</span>, <span class="hlFld-ContribAuthor ">Om  Silakari</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple machine learning models combined with virtual screening and molecular docking to identify selective human ALDH1A1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2021,</strong> <em>107 </em>, 107950. <a href="https://doi.org/10.1016/j.jmgm.2021.107950" title="DOI URL">https://doi.org/10.1016/j.jmgm.2021.107950</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2021.107950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2021.107950%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DMultiple%252Bmachine%252Blearning%252Bmodels%252Bcombined%252Bwith%252Bvirtual%252Bscreening%252Band%252Bmolecular%252Bdocking%252Bto%252Bidentify%252Bselective%252Bhuman%252BALDH1A1%252Binhibitors%26aulast%3DNarendra%26aufirst%3DGera%26date%3D2021%26volume%3D107%26spage%3D107950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mateusz  Winiarczyk</span>, <span class="hlFld-ContribAuthor ">Dagmara  Winiarczyk</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Michalak</span>, <span class="hlFld-ContribAuthor ">Kai  Kaarniranta</span>, <span class="hlFld-ContribAuthor ">Łukasz  Adaszek</span>, <span class="hlFld-ContribAuthor ">Stanisław  Winiarczyk</span>, <span class="hlFld-ContribAuthor ">Jerzy  Mackiewicz</span>. </span><span class="cited-content_cbyCitation_article-title">Dysregulated Tear Film Proteins in Macular Edema Due to the Neovascular Age-Related Macular Degeneration Are Involved in the Regulation of Protein Clearance, Inflammation, and Neovascularization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Clinical Medicine</span><span> <strong>2021,</strong> <em>10 </em>
                                    (14)
                                     , 3060. <a href="https://doi.org/10.3390/jcm10143060" title="DOI URL">https://doi.org/10.3390/jcm10143060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/jcm10143060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fjcm10143060%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Clinical%2520Medicine%26atitle%3DDysregulated%252BTear%252BFilm%252BProteins%252Bin%252BMacular%252BEdema%252BDue%252Bto%252Bthe%252BNeovascular%252BAge-Related%252BMacular%252BDegeneration%252BAre%252BInvolved%252Bin%252Bthe%252BRegulation%252Bof%252BProtein%252BClearance%25252C%252BInflammation%25252C%252Band%252BNeovascularization%26aulast%3DWiniarczyk%26aufirst%3DMateusz%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D14%26spage%3D3060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thi Bao Chau  Bui</span>, <span class="hlFld-ContribAuthor ">Shohei  Nosaki</span>, <span class="hlFld-ContribAuthor ">Mito  Kokawa</span>, <span class="hlFld-ContribAuthor ">Yuqun  Xu</span>, <span class="hlFld-ContribAuthor ">Yutaka  Kitamura</span>, <span class="hlFld-ContribAuthor ">Masaru  Tanokura</span>, <span class="hlFld-ContribAuthor ">Satoshi  Hachimura</span>, <span class="hlFld-ContribAuthor ">Takuya  Miyakawa</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of spice and herb as phyto-derived selective modulators of human retinaldehyde dehydrogenases using a simple
              in vitro
              method. </span><span class="cited-content_cbyCitation_journal-name">Bioscience Reports</span><span> <strong>2021,</strong> <em>41 </em>
                                    (5)
                                     <a href="https://doi.org/10.1042/BSR20210491" title="DOI URL">https://doi.org/10.1042/BSR20210491</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BSR20210491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBSR20210491%26sid%3Dliteratum%253Aachs%26jtitle%3DBioscience%2520Reports%26atitle%3DEvaluation%252Bof%252Bspice%252Band%252Bherb%252Bas%252Bphyto-derived%252Bselective%252Bmodulators%252Bof%252Bhuman%252Bretinaldehyde%252Bdehydrogenases%252Busing%252Ba%252Bsimple%252Bin%252Bvitro%252Bmethod%26aulast%3DBui%26aufirst%3DThi%25C2%25A0Bao%25C2%25A0Chau%26date%3D2021%26date%3D2021%26volume%3D41%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah H.  Gardner</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Reinhardt</span>, <span class="hlFld-ContribAuthor ">Jefferson  Chan</span>. </span><span class="cited-content_cbyCitation_article-title">Fortschritte bei aktivitätsbasierten Sonden für die isoformselektive Bildgebung enzymatischer Aktivität. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (10)
                                     , 5052-5062. <a href="https://doi.org/10.1002/ange.202003687" title="DOI URL">https://doi.org/10.1002/ange.202003687</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202003687&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202003687%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DFortschritte%252Bbei%252Baktivit%2525C3%2525A4tsbasierten%252BSonden%252Bf%2525C3%2525BCr%252Bdie%252Bisoformselektive%252BBildgebung%252Benzymatischer%252BAktivit%2525C3%2525A4t%26aulast%3DGardner%26aufirst%3DSarah%2BH.%26date%3D2021%26date%3D2020%26volume%3D133%26issue%3D10%26spage%3D5052%26epage%3D5062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah H.  Gardner</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Reinhardt</span>, <span class="hlFld-ContribAuthor ">Jefferson  Chan</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in Activity‐Based Sensing Probes for Isoform‐Selective Imaging of Enzymatic Activity. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (10)
                                     , 5000-5009. <a href="https://doi.org/10.1002/anie.202003687" title="DOI URL">https://doi.org/10.1002/anie.202003687</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202003687&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202003687%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DAdvances%252Bin%252BActivity%2525E2%252580%252590Based%252BSensing%252BProbes%252Bfor%252BIsoform%2525E2%252580%252590Selective%252BImaging%252Bof%252BEnzymatic%252BActivity%26aulast%3DGardner%26aufirst%3DSarah%2BH.%26date%3D2021%26date%3D2020%26volume%3D60%26issue%3D10%26spage%3D5000%26epage%3D5009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandt C.  Huddle</span>, <span class="hlFld-ContribAuthor ">Edward  Grimley</span>, <span class="hlFld-ContribAuthor ">Mikhail  Chtcherbinine</span>, <span class="hlFld-ContribAuthor ">Cameron D.  Buchman</span>, <span class="hlFld-ContribAuthor ">Cyrus  Takahashi</span>, <span class="hlFld-ContribAuthor ">Bikash  Debnath</span>, <span class="hlFld-ContribAuthor ">Stacy C.  McGonigal</span>, <span class="hlFld-ContribAuthor ">Shuai  Mao</span>, <span class="hlFld-ContribAuthor ">Siwei  Li</span>, <span class="hlFld-ContribAuthor ">Jeremy  Felton</span>, <span class="hlFld-ContribAuthor ">Shu  Pan</span>, <span class="hlFld-ContribAuthor ">Bo  Wen</span>, <span class="hlFld-ContribAuthor ">Duxin  Sun</span>, <span class="hlFld-ContribAuthor ">Nouri  Neamati</span>, <span class="hlFld-ContribAuthor ">Ronald J.  Buckanovich</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Hurley</span>, <span class="hlFld-ContribAuthor ">Scott D.  Larsen</span>. </span><span class="cited-content_cbyCitation_article-title">Development of 2,5-dihydro-4H-pyrazolo[3,4-d]pyrimidin-4-one inhibitors of aldehyde dehydrogenase 1A (ALDH1A) as potential adjuncts to ovarian cancer chemotherapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>211 </em>, 113060. <a href="https://doi.org/10.1016/j.ejmech.2020.113060" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113060%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252B2%25252C5-dihydro-4H-pyrazolo%25255B3%25252C4-d%25255Dpyrimidin-4-one%252Binhibitors%252Bof%252Baldehyde%252Bdehydrogenase%252B1A%252B%252528ALDH1A%252529%252Bas%252Bpotential%252Badjuncts%252Bto%252Bovarian%252Bcancer%252Bchemotherapy%26aulast%3DHuddle%26aufirst%3DBrandt%2BC.%26date%3D2021%26volume%3D211%26spage%3D113060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bingyan  Li</span>, <span class="hlFld-ContribAuthor ">Kang  Yang</span>, <span class="hlFld-ContribAuthor ">Dailin  Liang</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>, <span class="hlFld-ContribAuthor ">Zonghui  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and development of selective aldehyde dehydrogenase 1A1 (ALDH1A1) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112940. <a href="https://doi.org/10.1016/j.ejmech.2020.112940" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112940</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112940%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252Bdevelopment%252Bof%252Bselective%252Baldehyde%252Bdehydrogenase%252B1A1%252B%252528ALDH1A1%252529%252Binhibitors%26aulast%3DLi%26aufirst%3DBingyan%26date%3D2021%26volume%3D209%26spage%3D112940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thayssa Tavares  da Silva Cunha</span>, <span class="hlFld-ContribAuthor ">Felipe Rodrigues  de Souza</span>, <span class="hlFld-ContribAuthor ">Pedro  de Sena Murteira Pinheiro</span>, <span class="hlFld-ContribAuthor ">Carlos Maurício Rabello  de Sant’Anna</span>, <span class="hlFld-ContribAuthor ">François  Noël</span>, <span class="hlFld-ContribAuthor ">Tanos Celmar  Costa França</span>, <span class="hlFld-ContribAuthor ">Carlos Alberto  Manssour Fraga</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the Molecular Basis for the Selective Inhibition of Aldehyde Dehydrogenase 2 by the Isoflavonoid Daidzin. </span><span class="cited-content_cbyCitation_journal-name">CNS & Neurological Disorders - Drug Targets</span><span> <strong>2020,</strong> <em>19 </em>
                                    (6)
                                     , 437-447. <a href="https://doi.org/10.2174/1871527319999200817153150" title="DOI URL">https://doi.org/10.2174/1871527319999200817153150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1871527319999200817153150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1871527319999200817153150%26sid%3Dliteratum%253Aachs%26jtitle%3DCNS%2520%2526%2520Neurological%2520Disorders%2520-%2520Drug%2520Targets%26atitle%3DInvestigating%252Bthe%252BMolecular%252BBasis%252Bfor%252Bthe%252BSelective%252BInhibition%252Bof%252BAldehyde%252BDehydrogenase%252B2%252Bby%252Bthe%252BIsoflavonoid%252BDaidzin%26aulast%3Dda%2BSilva%2BCunha%26aufirst%3DThayssa%2BTavares%26date%3D2020%26volume%3D19%26issue%3D6%26spage%3D437%26epage%3D447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Himanshu  Verma</span>, <span class="hlFld-ContribAuthor ">Om  Silakari</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the Role of Missense SNPs on ALDH 1A1 mediated pharmacokinetic resistance to cyclophosphamide. </span><span class="cited-content_cbyCitation_journal-name">Computers in Biology and Medicine</span><span> <strong>2020,</strong> <em>125 </em>, 103979. <a href="https://doi.org/10.1016/j.compbiomed.2020.103979" title="DOI URL">https://doi.org/10.1016/j.compbiomed.2020.103979</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiomed.2020.103979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiomed.2020.103979%26sid%3Dliteratum%253Aachs%26jtitle%3DComputers%2520in%2520Biology%2520and%2520Medicine%26atitle%3DInvestigating%252Bthe%252BRole%252Bof%252BMissense%252BSNPs%252Bon%252BALDH%252B1A1%252Bmediated%252Bpharmacokinetic%252Bresistance%252Bto%252Bcyclophosphamide%26aulast%3DVerma%26aufirst%3DHimanshu%26date%3D2020%26volume%3D125%26spage%3D103979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caroline A.  Magee</span>, <span class="hlFld-ContribAuthor ">Larryn W.  Peterson</span>, <span class="hlFld-ContribAuthor ">Mauricio  Cafiero</span>, <span class="hlFld-ContribAuthor ">Emma F.  Selner</span>. </span><span class="cited-content_cbyCitation_article-title">The effects of ligand charge, orientation and size on the binding of potential inhibitors for aldehyde dehydrogenase. </span><span class="cited-content_cbyCitation_journal-name">Computational and Theoretical Chemistry</span><span> <strong>2020,</strong> <em>1185 </em>, 112868. <a href="https://doi.org/10.1016/j.comptc.2020.112868" title="DOI URL">https://doi.org/10.1016/j.comptc.2020.112868</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.comptc.2020.112868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.comptc.2020.112868%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Theoretical%2520Chemistry%26atitle%3DThe%252Beffects%252Bof%252Bligand%252Bcharge%25252C%252Borientation%252Band%252Bsize%252Bon%252Bthe%252Bbinding%252Bof%252Bpotential%252Binhibitors%252Bfor%252Baldehyde%252Bdehydrogenase%26aulast%3DMagee%26aufirst%3DCaroline%2BA.%26date%3D2020%26volume%3D1185%26spage%3D112868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zonghui  Ma</span>, <span class="hlFld-ContribAuthor ">Ling  Jiang</span>, <span class="hlFld-ContribAuthor ">Guoxin  Li</span>, <span class="hlFld-ContribAuthor ">Dailin  Liang</span>, <span class="hlFld-ContribAuthor ">Luanfeng  Li</span>, <span class="hlFld-ContribAuthor ">Li  Liu</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis of 1,3-dimethylpyrimidine-2,4-diones as potent and selective aldehyde dehydrogenase 1A1 (ALDH1A1) inhibitors with glucose consumption improving activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>101 </em>, 103971. <a href="https://doi.org/10.1016/j.bioorg.2020.103971" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103971</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103971%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Bof%252B1%25252C3-dimethylpyrimidine-2%25252C4-diones%252Bas%252Bpotent%252Band%252Bselective%252Baldehyde%252Bdehydrogenase%252B1A1%252B%252528ALDH1A1%252529%252Binhibitors%252Bwith%252Bglucose%252Bconsumption%252Bimproving%252Bactivity%26aulast%3DMa%26aufirst%3DZonghui%26date%3D2020%26volume%3D101%26spage%3D103971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Davide M.  Ferraris</span>, <span class="hlFld-ContribAuthor ">Edoardo L. M.  Gelardi</span>, <span class="hlFld-ContribAuthor ">Silvia  Garavaglia</span>, <span class="hlFld-ContribAuthor ">Riccardo  Miggiano</span>, <span class="hlFld-ContribAuthor ">Menico  Rizzi</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting NAD-dependent dehydrogenases in drug discovery against infectious diseases and cancer. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Society Transactions</span><span> <strong>2020,</strong> <em>48 </em>
                                    (2)
                                     , 693-707. <a href="https://doi.org/10.1042/BST20191261" title="DOI URL">https://doi.org/10.1042/BST20191261</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BST20191261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBST20191261%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Society%2520Transactions%26atitle%3DTargeting%252BNAD-dependent%252Bdehydrogenases%252Bin%252Bdrug%252Bdiscovery%252Bagainst%252Binfectious%252Bdiseases%252Band%252Bcancer%26aulast%3DFerraris%26aufirst%3DDavide%2BM.%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D2%26spage%3D693%26epage%3D707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vaishnavi  Muralikrishnan</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Hurley</span>, <span class="hlFld-ContribAuthor ">Kenneth P.  Nephew</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Aldehyde Dehydrogenases to Eliminate Cancer Stem Cells in Gynecologic Malignancies. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (4)
                                     , 961. <a href="https://doi.org/10.3390/cancers12040961" title="DOI URL">https://doi.org/10.3390/cancers12040961</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12040961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12040961%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DTargeting%252BAldehyde%252BDehydrogenases%252Bto%252BEliminate%252BCancer%252BStem%252BCells%252Bin%252BGynecologic%252BMalignancies%26aulast%3DMuralikrishnan%26aufirst%3DVaishnavi%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D4%26spage%3D961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raquel  Pequerul</span>, <span class="hlFld-ContribAuthor ">Javier  Vera</span>, <span class="hlFld-ContribAuthor ">Joan  Giménez-Dejoz</span>, <span class="hlFld-ContribAuthor ">Isidro  Crespo</span>, <span class="hlFld-ContribAuthor ">Joan  Coines</span>, <span class="hlFld-ContribAuthor ">Sergio  Porté</span>, <span class="hlFld-ContribAuthor ">Carme  Rovira</span>, <span class="hlFld-ContribAuthor ">Xavier  Parés</span>, <span class="hlFld-ContribAuthor ">Jaume  Farrés</span>. </span><span class="cited-content_cbyCitation_article-title">Structural and kinetic features of aldehyde dehydrogenase 1A (ALDH1A) subfamily members, cancer stem cell markers active in retinoic acid biosynthesis. </span><span class="cited-content_cbyCitation_journal-name">Archives of Biochemistry and Biophysics</span><span> <strong>2020,</strong> <em>681 </em>, 108256. <a href="https://doi.org/10.1016/j.abb.2020.108256" title="DOI URL">https://doi.org/10.1016/j.abb.2020.108256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.abb.2020.108256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.abb.2020.108256%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Biochemistry%2520and%2520Biophysics%26atitle%3DStructural%252Band%252Bkinetic%252Bfeatures%252Bof%252Baldehyde%252Bdehydrogenase%252B1A%252B%252528ALDH1A%252529%252Bsubfamily%252Bmembers%25252C%252Bcancer%252Bstem%252Bcell%252Bmarkers%252Bactive%252Bin%252Bretinoic%252Bacid%252Bbiosynthesis%26aulast%3DPequerul%26aufirst%3DRaquel%26date%3D2020%26volume%3D681%26spage%3D108256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea S.  Heimann</span>, <span class="hlFld-ContribAuthor ">A.C.  Giardini</span>, <span class="hlFld-ContribAuthor ">M.B.  Sant’Anna</span>, <span class="hlFld-ContribAuthor ">Nilton B.  Dos Santos</span>, <span class="hlFld-ContribAuthor ">Mayara C.F.  Gewehr</span>, <span class="hlFld-ContribAuthor ">Carolina D.  Munhoz</span>, <span class="hlFld-ContribAuthor ">Leandro M.  Castro</span>, <span class="hlFld-ContribAuthor ">Gisele  Picolo</span>, <span class="hlFld-ContribAuthor ">Ricardo A.  Remer</span>, <span class="hlFld-ContribAuthor ">Emer S.  Ferro</span>. </span><span class="cited-content_cbyCitation_article-title">NFKF is a synthetic fragment derived from rat hemopressin that protects mice from neurodegeneration. </span><span class="cited-content_cbyCitation_journal-name">Neuroscience Letters</span><span> <strong>2020,</strong> <em>721 </em>, 134765. <a href="https://doi.org/10.1016/j.neulet.2020.134765" title="DOI URL">https://doi.org/10.1016/j.neulet.2020.134765</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neulet.2020.134765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neulet.2020.134765%26sid%3Dliteratum%253Aachs%26jtitle%3DNeuroscience%2520Letters%26atitle%3DNFKF%252Bis%252Ba%252Bsynthetic%252Bfragment%252Bderived%252Bfrom%252Brat%252Bhemopressin%252Bthat%252Bprotects%252Bmice%252Bfrom%252Bneurodegeneration%26aulast%3DHeimann%26aufirst%3DAndrea%2BS.%26date%3D2020%26volume%3D721%26spage%3D134765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dailin  Liang</span>, <span class="hlFld-ContribAuthor ">Yazhou  Fan</span>, <span class="hlFld-ContribAuthor ">Zhou  Yang</span>, <span class="hlFld-ContribAuthor ">Zhenguo  Zhang</span>, <span class="hlFld-ContribAuthor ">Meiyang  Liu</span>, <span class="hlFld-ContribAuthor ">Li  Liu</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of coumarin-based selective aldehyde dehydrogenase 1A1 inhibitors with glucose metabolism improving activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111923. <a href="https://doi.org/10.1016/j.ejmech.2019.111923" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111923</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111923%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bcoumarin-based%252Bselective%252Baldehyde%252Bdehydrogenase%252B1A1%252Binhibitors%252Bwith%252Bglucose%252Bmetabolism%252Bimproving%252Bactivity%26aulast%3DLiang%26aufirst%3DDailin%26date%3D2020%26volume%3D187%26spage%3D111923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saketh S.  Dinavahi</span>, <span class="hlFld-ContribAuthor ">Raghavendra  Gowda</span>, <span class="hlFld-ContribAuthor ">Christopher G.  Bazewicz</span>, <span class="hlFld-ContribAuthor ">Madhu Babu  Battu</span>, <span class="hlFld-ContribAuthor ">Jyh Ming  Lin</span>, <span class="hlFld-ContribAuthor ">Robert J.  Chitren</span>, <span class="hlFld-ContribAuthor ">Manoj K.  Pandey</span>, <span class="hlFld-ContribAuthor ">Shantu  Amin</span>, <span class="hlFld-ContribAuthor ">Gavin P.  Robertson</span>, <span class="hlFld-ContribAuthor ">Krishne  Gowda</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis characterization and biological evaluation of novel multi-isoform ALDH inhibitors as potential anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111962. <a href="https://doi.org/10.1016/j.ejmech.2019.111962" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111962%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Bcharacterization%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bmulti-isoform%252BALDH%252Binhibitors%252Bas%252Bpotential%252Banticancer%252Bagents%26aulast%3DDinavahi%26aufirst%3DSaketh%2BS.%26date%3D2020%26volume%3D187%26spage%3D111962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jung-A  Choi</span>, <span class="hlFld-ContribAuthor ">Hyunja  Kwon</span>, <span class="hlFld-ContribAuthor ">Hanbyoul  Cho</span>, <span class="hlFld-ContribAuthor ">Joon-Yong  Chung</span>, <span class="hlFld-ContribAuthor ">Stephen M.  Hewitt</span>, <span class="hlFld-ContribAuthor ">Jae-Hoon  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">ALDH1A2 Is a Candidate Tumor Suppressor Gene in Ovarian Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (10)
                                     , 1553. <a href="https://doi.org/10.3390/cancers11101553" title="DOI URL">https://doi.org/10.3390/cancers11101553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11101553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11101553%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DALDH1A2%252BIs%252Ba%252BCandidate%252BTumor%252BSuppressor%252BGene%252Bin%252BOvarian%252BCancer%26aulast%3DChoi%26aufirst%3DJung-A%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D10%26spage%3D1553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saketh S.  Dinavahi</span>, <span class="hlFld-ContribAuthor ">Christopher G.  Bazewicz</span>, <span class="hlFld-ContribAuthor ">Raghavendra  Gowda</span>, <span class="hlFld-ContribAuthor ">Gavin P.  Robertson</span>. </span><span class="cited-content_cbyCitation_article-title">Aldehyde Dehydrogenase Inhibitors for Cancer Therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2019,</strong> <em>40 </em>
                                    (10)
                                     , 774-789. <a href="https://doi.org/10.1016/j.tips.2019.08.002" title="DOI URL">https://doi.org/10.1016/j.tips.2019.08.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2019.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2019.08.002%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DAldehyde%252BDehydrogenase%252BInhibitors%252Bfor%252BCancer%252BTherapeutics%26aulast%3DDinavahi%26aufirst%3DSaketh%2BS.%26date%3D2019%26volume%3D40%26issue%3D10%26spage%3D774%26epage%3D789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafael  Jiménez</span>, <span class="hlFld-ContribAuthor ">Raquel  Pequerul</span>, <span class="hlFld-ContribAuthor ">Adrián  Amor</span>, <span class="hlFld-ContribAuthor ">Julia  Lorenzo</span>, <span class="hlFld-ContribAuthor ">Kamel  Metwally</span>, <span class="hlFld-ContribAuthor ">Francesc Xavier  Avilés</span>, <span class="hlFld-ContribAuthor ">Xavier  Parés</span>, <span class="hlFld-ContribAuthor ">Jaume  Farrés</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of aldehyde dehydrogenases of the 1A subfamily as putative anticancer agents: Kinetic characterization and effect on human cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Chemico-Biological Interactions</span><span> <strong>2019,</strong> <em>306 </em>, 123-130. <a href="https://doi.org/10.1016/j.cbi.2019.04.004" title="DOI URL">https://doi.org/10.1016/j.cbi.2019.04.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbi.2019.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbi.2019.04.004%26sid%3Dliteratum%253Aachs%26jtitle%3DChemico-Biological%2520Interactions%26atitle%3DInhibitors%252Bof%252Baldehyde%252Bdehydrogenases%252Bof%252Bthe%252B1A%252Bsubfamily%252Bas%252Bputative%252Banticancer%252Bagents%25253A%252BKinetic%252Bcharacterization%252Band%252Beffect%252Bon%252Bhuman%252Bcancer%252Bcells%26aulast%3DJim%25C3%25A9nez%26aufirst%3DRafael%26date%3D2019%26volume%3D306%26spage%3D123%26epage%3D130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Himanshu  Verma</span>, <span class="hlFld-ContribAuthor ">Malkeet  Singh Bahia</span>, <span class="hlFld-ContribAuthor ">Shalki  Choudhary</span>, <span class="hlFld-ContribAuthor ">Pankaj  Kumar Singh</span>, <span class="hlFld-ContribAuthor ">Om  Silakari</span>. </span><span class="cited-content_cbyCitation_article-title">Drug metabolizing enzymes-associated chemo resistance and strategies to overcome it. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism Reviews</span><span> <strong>2019,</strong> <em>51 </em>
                                    (2)
                                     , 196-223. <a href="https://doi.org/10.1080/03602532.2019.1632886" title="DOI URL">https://doi.org/10.1080/03602532.2019.1632886</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/03602532.2019.1632886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F03602532.2019.1632886%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520Reviews%26atitle%3DDrug%252Bmetabolizing%252Benzymes-associated%252Bchemo%252Bresistance%252Band%252Bstrategies%252Bto%252Bovercome%252Bit%26aulast%3DVerma%26aufirst%3DHimanshu%26date%3D2019%26date%3D2019%26volume%3D51%26issue%3D2%26spage%3D196%26epage%3D223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher G.  Bazewicz</span>, <span class="hlFld-ContribAuthor ">Saketh S.  Dinavahi</span>, <span class="hlFld-ContribAuthor ">Todd D.  Schell</span>, <span class="hlFld-ContribAuthor ">Gavin P.  Robertson</span>. </span><span class="cited-content_cbyCitation_article-title">Aldehyde dehydrogenase in regulatory T-cell development, immunity and cancer. </span><span class="cited-content_cbyCitation_journal-name">Immunology</span><span> <strong>2019,</strong> <em>156 </em>
                                    (1)
                                     , 47-55. <a href="https://doi.org/10.1111/imm.13016" title="DOI URL">https://doi.org/10.1111/imm.13016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/imm.13016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fimm.13016%26sid%3Dliteratum%253Aachs%26jtitle%3DImmunology%26atitle%3DAldehyde%252Bdehydrogenase%252Bin%252Bregulatory%252BT-cell%252Bdevelopment%25252C%252Bimmunity%252Band%252Bcancer%26aulast%3DBazewicz%26aufirst%3DChristopher%2BG.%26date%3D2019%26date%3D2018%26volume%3D156%26issue%3D1%26spage%3D47%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angelica R.  Harper</span>, <span class="hlFld-ContribAuthor ">Anh T.  Le</span>, <span class="hlFld-ContribAuthor ">Timothy  Mather</span>, <span class="hlFld-ContribAuthor ">Anthony  Burgett</span>, <span class="hlFld-ContribAuthor ">William  Berry</span>, <span class="hlFld-ContribAuthor ">Jody A.  Summers</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and ex vivo evaluation of a selective inhibitor for retinaldehyde dehydrogenase enzymes. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (22)
                                     , 5766-5779. <a href="https://doi.org/10.1016/j.bmc.2018.10.009" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.10.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.10.009%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bex%252Bvivo%252Bevaluation%252Bof%252Ba%252Bselective%252Binhibitor%252Bfor%252Bretinaldehyde%252Bdehydrogenase%252Benzymes%26aulast%3DHarper%26aufirst%3DAngelica%2BR.%26date%3D2018%26volume%3D26%26issue%3D22%26spage%3D5766%26epage%3D5779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valerio  Ciccone</span>, <span class="hlFld-ContribAuthor ">Erika  Terzuoli</span>, <span class="hlFld-ContribAuthor ">Sandra  Donnini</span>, <span class="hlFld-ContribAuthor ">Antonio  Giachetti</span>, <span class="hlFld-ContribAuthor ">Lucia  Morbidelli</span>, <span class="hlFld-ContribAuthor ">Marina  Ziche</span>. </span><span class="cited-content_cbyCitation_article-title">Stemness marker ALDH1A1 promotes tumor angiogenesis via retinoic acid/HIF-1α/VEGF signalling in MCF-7 breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Experimental & Clinical Cancer Research</span><span> <strong>2018,</strong> <em>37 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13046-018-0975-0" title="DOI URL">https://doi.org/10.1186/s13046-018-0975-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13046-018-0975-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13046-018-0975-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Experimental%2520%2526%2520Clinical%2520Cancer%2520Research%26atitle%3DStemness%252Bmarker%252BALDH1A1%252Bpromotes%252Btumor%252Bangiogenesis%252Bvia%252Bretinoic%252Bacid%25252FHIF-1%2525CE%2525B1%25252FVEGF%252Bsignalling%252Bin%252BMCF-7%252Bbreast%252Bcancer%252Bcells%26aulast%3DCiccone%26aufirst%3DValerio%26date%3D2018%26date%3D2018%26volume%3D37%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Waheed  Abdul</span>, <span class="hlFld-ContribAuthor ">Sami R.  Aliyu</span>, <span class="hlFld-ContribAuthor ">Lili  Lin</span>, <span class="hlFld-ContribAuthor ">Malota  Sekete</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Chen</span>, <span class="hlFld-ContribAuthor ">Frankline J.  Otieno</span>, <span class="hlFld-ContribAuthor ">Tao  Yang</span>, <span class="hlFld-ContribAuthor ">Yahong  Lin</span>, <span class="hlFld-ContribAuthor ">Justice  Norvienyeku</span>, <span class="hlFld-ContribAuthor ">Zonghua  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Family-Four Aldehyde Dehydrogenases Play an Indispensable Role in the Pathogenesis of Magnaporthe oryzae. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Plant Science</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fpls.2018.00980" title="DOI URL">https://doi.org/10.3389/fpls.2018.00980</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fpls.2018.00980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffpls.2018.00980%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Plant%2520Science%26atitle%3DFamily-Four%252BAldehyde%252BDehydrogenases%252BPlay%252Ban%252BIndispensable%252BRole%252Bin%252Bthe%252BPathogenesis%252Bof%252BMagnaporthe%252Boryzae%26aulast%3DAbdul%26aufirst%3DWaheed%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">V.  Vasiliou</span>, <span class="hlFld-ContribAuthor ">D.C.  Thompson</span>, <span class="hlFld-ContribAuthor ">D.R.  Petersen</span>. </span><span class="cited-content_cbyCitation_article-title">Aldehyde Dehydrogenases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 146-163. <a href="https://doi.org/10.1016/B978-0-12-801238-3.99183-9" title="DOI URL">https://doi.org/10.1016/B978-0-12-801238-3.99183-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801238-3.99183-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801238-3.99183-9%26sid%3Dliteratum%253Aachs%26atitle%3DAldehyde%252BDehydrogenases%26aulast%3DVasiliou%26aufirst%3DV.%26date%3D2018%26spage%3D146%26epage%3D163%26pub%3DElsevier%26atitle%3DComprehensive%252BToxicology%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vaibhav A.  Dixit</span>, <span class="hlFld-ContribAuthor ">L. Arun  Lal</span>, <span class="hlFld-ContribAuthor ">Simran R.  Agrawal</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in the prediction of non‐
              CYP450
              ‐mediated drug metabolism. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2017,</strong> <em>7 </em>
                                    (6)
                                     <a href="https://doi.org/10.1002/wcms.1323" title="DOI URL">https://doi.org/10.1002/wcms.1323</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1323%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DRecent%252Badvances%252Bin%252Bthe%252Bprediction%252Bof%252Bnon%2525E2%252580%252590%252BCYP450%252B%2525E2%252580%252590mediated%252Bdrug%252Bmetabolism%26aulast%3DDixit%26aufirst%3DVaibhav%2BA.%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Yang</span>, <span class="hlFld-ContribAuthor ">Wei-hong  Li</span>, <span class="hlFld-ContribAuthor ">Rong  An</span>, <span class="hlFld-ContribAuthor ">Yi-li  Wang</span>, <span class="hlFld-ContribAuthor ">Yan  Xu</span>, <span class="hlFld-ContribAuthor ">Jie  Chen</span>, <span class="hlFld-ContribAuthor ">Xiao-fang  Wang</span>, <span class="hlFld-ContribAuthor ">Xiao-bo  Zhang</span>, <span class="hlFld-ContribAuthor ">Jing  Li</span>, <span class="hlFld-ContribAuthor ">Wei-jun  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Differentially expressed genes in heads and tails of Angelica sinensis diels: Focusing on ferulic acid metabolism. </span><span class="cited-content_cbyCitation_journal-name">Chinese Journal of Integrative Medicine</span><span> <strong>2017,</strong> <em>23 </em>
                                    (10)
                                     , 779-785. <a href="https://doi.org/10.1007/s11655-016-2603-1" title="DOI URL">https://doi.org/10.1007/s11655-016-2603-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11655-016-2603-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11655-016-2603-1%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Journal%2520of%2520Integrative%2520Medicine%26atitle%3DDifferentially%252Bexpressed%252Bgenes%252Bin%252Bheads%252Band%252Btails%252Bof%252BAngelica%252Bsinensis%252Bdiels%25253A%252BFocusing%252Bon%252Bferulic%252Bacid%252Bmetabolism%26aulast%3DYang%26aufirst%3DJie%26date%3D2017%26date%3D2016%26volume%3D23%26issue%3D10%26spage%3D779%26epage%3D785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Salwa Mansur  Ali</span>, <span class="hlFld-ContribAuthor ">Ruqaiyyah  Siddiqui</span>, <span class="hlFld-ContribAuthor ">Seng-Kai  Ong</span>, <span class="hlFld-ContribAuthor ">Muhammad Raza  Shah</span>, <span class="hlFld-ContribAuthor ">Ayaz  Anwar</span>, <span class="hlFld-ContribAuthor ">Peter J.  Heard</span>, <span class="hlFld-ContribAuthor ">Naveed Ahmed  Khan</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and characterization of antibacterial compound(s) of cockroaches (Periplaneta americana). </span><span class="cited-content_cbyCitation_journal-name">Applied Microbiology and Biotechnology</span><span> <strong>2017,</strong> <em>101 </em>
                                    (1)
                                     , 253-286. <a href="https://doi.org/10.1007/s00253-016-7872-2" title="DOI URL">https://doi.org/10.1007/s00253-016-7872-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00253-016-7872-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00253-016-7872-2%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520Microbiology%2520and%2520Biotechnology%26atitle%3DIdentification%252Band%252Bcharacterization%252Bof%252Bantibacterial%252Bcompound%252528s%252529%252Bof%252Bcockroaches%252B%252528Periplaneta%252Bamericana%252529%26aulast%3DAli%26aufirst%3DSalwa%2BMansur%26date%3D2017%26date%3D2016%26volume%3D101%26issue%3D1%26spage%3D253%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seung-Hye  Hong</span>, <span class="hlFld-ContribAuthor ">Ho-Phuong-Thuy  Ngo</span>, <span class="hlFld-ContribAuthor ">Hyun-Koo  Nam</span>, <span class="hlFld-ContribAuthor ">Kyoung-Rok  Kim</span>, <span class="hlFld-ContribAuthor ">Lin-Woo  Kang</span>, <span class="hlFld-ContribAuthor ">Deok-Kun  Oh</span>, . </span><span class="cited-content_cbyCitation_article-title">Alternative Biotransformation of Retinal to Retinoic Acid or Retinol by an Aldehyde Dehydrogenase from Bacillus cereus. </span><span class="cited-content_cbyCitation_journal-name">Applied and Environmental Microbiology</span><span> <strong>2016,</strong> <em>82 </em>
                                    (13)
                                     , 3940-3946. <a href="https://doi.org/10.1128/AEM.00848-16" title="DOI URL">https://doi.org/10.1128/AEM.00848-16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/AEM.00848-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FAEM.00848-16%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520and%2520Environmental%2520Microbiology%26atitle%3DAlternative%252BBiotransformation%252Bof%252BRetinal%252Bto%252BRetinoic%252BAcid%252Bor%252BRetinol%252Bby%252Ban%252BAldehyde%252BDehydrogenase%252Bfrom%252BBacillus%252Bcereus%26aulast%3DHong%26aufirst%3DSeung-Hye%26date%3D2016%26volume%3D82%26issue%3D13%26spage%3D3940%26epage%3D3946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. V.  Puchinskaya</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer stem cell markers and their prognostic value. </span><span class="cited-content_cbyCitation_journal-name">Arkhiv patologii</span><span> <strong>2016,</strong> <em>78 </em>
                                    (2)
                                     , 47. <a href="https://doi.org/10.17116/patol201678247-54" title="DOI URL">https://doi.org/10.17116/patol201678247-54</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.17116/patol201678247-54&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.17116%2Fpatol201678247-54%26sid%3Dliteratum%253Aachs%26jtitle%3DArkhiv%2520patologii%26atitle%3DCancer%252Bstem%252Bcell%252Bmarkers%252Band%252Btheir%252Bprognostic%252Bvalue%26aulast%3DPuchinskaya%26aufirst%3DM.%2BV.%26date%3D2016%26volume%3D78%26issue%3D2%26spage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maximilian F.  Koch</span>, <span class="hlFld-ContribAuthor ">Sabrina  Harteis</span>, <span class="hlFld-ContribAuthor ">Iris D.  Blank</span>, <span class="hlFld-ContribAuthor ">Galina  Pestel</span>, <span class="hlFld-ContribAuthor ">Lutz F.  Tietze</span>, <span class="hlFld-ContribAuthor ">Christian  Ochsenfeld</span>, <span class="hlFld-ContribAuthor ">Sabine  Schneider</span>, <span class="hlFld-ContribAuthor ">Stephan A.  Sieber</span>. </span><span class="cited-content_cbyCitation_article-title">Structural, Biochemical, and Computational Studies Reveal the Mechanism of Selective Aldehyde Dehydrogenase 1A1 Inhibition by Cytotoxic Duocarmycin Analogues. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2015,</strong> <em>127 </em>
                                    (46)
                                     , 13754-13758. <a href="https://doi.org/10.1002/ange.201505749" title="DOI URL">https://doi.org/10.1002/ange.201505749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201505749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201505749%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DStructural%25252C%252BBiochemical%25252C%252Band%252BComputational%252BStudies%252BReveal%252Bthe%252BMechanism%252Bof%252BSelective%252BAldehyde%252BDehydrogenase%252B1A1%252BInhibition%252Bby%252BCytotoxic%252BDuocarmycin%252BAnalogues%26aulast%3DKoch%26aufirst%3DMaximilian%2BF.%26date%3D2015%26date%3D2015%26volume%3D127%26issue%3D46%26spage%3D13754%26epage%3D13758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maximilian F.  Koch</span>, <span class="hlFld-ContribAuthor ">Sabrina  Harteis</span>, <span class="hlFld-ContribAuthor ">Iris D.  Blank</span>, <span class="hlFld-ContribAuthor ">Galina  Pestel</span>, <span class="hlFld-ContribAuthor ">Lutz F.  Tietze</span>, <span class="hlFld-ContribAuthor ">Christian  Ochsenfeld</span>, <span class="hlFld-ContribAuthor ">Sabine  Schneider</span>, <span class="hlFld-ContribAuthor ">Stephan A.  Sieber</span>. </span><span class="cited-content_cbyCitation_article-title">Structural, Biochemical, and Computational Studies Reveal the Mechanism of Selective Aldehyde Dehydrogenase 1A1 Inhibition by Cytotoxic Duocarmycin Analogues. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2015,</strong> <em>54 </em>
                                    (46)
                                     , 13550-13554. <a href="https://doi.org/10.1002/anie.201505749" title="DOI URL">https://doi.org/10.1002/anie.201505749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201505749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201505749%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DStructural%25252C%252BBiochemical%25252C%252Band%252BComputational%252BStudies%252BReveal%252Bthe%252BMechanism%252Bof%252BSelective%252BAldehyde%252BDehydrogenase%252B1A1%252BInhibition%252Bby%252BCytotoxic%252BDuocarmycin%252BAnalogues%26aulast%3DKoch%26aufirst%3DMaximilian%2BF.%26date%3D2015%26date%3D2015%26volume%3D54%26issue%3D46%26spage%3D13550%26epage%3D13554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of CM026, a selective inhibitor of ALDH1A1. (A) Structure of CM026. (B) Selectivity of 20 μM of CM026 with respect to nine ALDH isoenzymes. (C) IC<sub>50</sub> of CM026 with ALDH1A1.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (D) Lineweaver–Burk representation of noncompetitive inhibition for CM026 (0–4 μM) versus varied acetaldehyde (100–800 μM) at fixed concentration of NAD<sup>+</sup> (800 μM). (E) Lineweaver–Burk representation of uncompetitive inhibition for CM026 (0–3 μM) versus varied NAD<sup>+</sup> (25–250 μM) at fixed concentration of propionaldehyde (200 μM). The IC<sub>50</sub> curves and Lineweaver–Burk plots represent one of three experiments performed for each condition, with each point the mean/SEM of three data points at each concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characterization of CM037, a selective inhibitor of ALDH1A1. (A) The structure of CM037 with an atomic mass of 431.6 Da. (B) Selectivity of 20 μM of CM037 with respect to nine ALDH isoenzymes. (C) IC<sub>50</sub> of CM037 with ALDH1A1.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The IC<sub>50</sub> curve represents one of three experiments performed, with each point the mean/SEM of three data points at each concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of CM053. (A) IC<sub>50</sub> of CM053 with ALDH1A1. (B) Lineweaver–Burk representation of noncompetitive tight inhibition for CM053 (0–300 nM) versus varied acetaldehyde (100–800 μM) at fixed concentration of NAD<sup>+</sup> (1000 μM). Both curves represent one of three experiments performed, with each point the mean/SEM of three data points at each concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure of ALDH1A1 N121S with CM026 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>). (A) CM026 binds in the active site near Cys 303. Key residues are colored based on element, and Gly458 is shown in orange. (B) Two-dimensional representation of the key hydrogen bonds, illustrated with red dashed lines, and hydrophobic interactions, illustrated with black arcs, between ALDH1A1 and CM026. (C) Electron density maps of CM026, with the original <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in green contoured at 2.5 standard deviations and the final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in gray contoured at 1.0 standard deviations. A and C are generated in Pymol, while B is generated in ChemDraw.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure of human ALDH1A1 with CM053 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WPN">4WPN</a>). (A) Two dimensional representation of the key hydrogen bonds, illustrated with red dashed lines, and hydrophobic interactions, illustrated with black arcs, between ALDH1A1 and CM053. (B) Electron density maps of CM053 with the original <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in green contoured at 2.5 standard deviations and the final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in gray contoured at 1.0 standard deviations. A is generated in ChemDraw, and B is generated in Pymol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structure of ALDH1A1 N121S with CM037 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L">4X4L</a>). (A) CM037 binds in the active site near Cys 303. Key residues are colored based on element, and Gly458 is shown in orange. (B) Two-dimensional representation of the hydrophobic interactions, illustrated with black arcs, between ALDH1A1 and CM037. (C) Binding of CM037 induces structural changes in ALDH1A1 (in blue) compared to apo-ALDH1A1 (in gray), particularly at W178. NADH binding (in cyan) induces conformational changes at the cofactor binding site, as seen here with E269. (D) Electron density maps of CM037, with the original <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in green contoured at 2 standard deviations and the final 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map in gray contoured at 1.0 standard deviations. A, C, and D are generated in Pymol, while B is generated in ChemDraw.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structural basis of selectivity of CM026 for ALDH1A1. (A) Multiple sequence alignment in the region of ALDH1A1 Gly458 to the mature form of ALDH2. (B) Structure of ALDH1A1 with bound CM026 (blue) compared to ALDH2 (cyan) and ALDH3A1 (gray), indicating that a bulky amino acid such as the Asp of ALDH2 and Ile of ALDH3A1 would clash with CM026 and prevent the compound from inhibiting the enzyme. Sequence alignment was performed using NCBI delta-BLAST, while structural alignment was performed using least-square fit (LSQ) in Coot. B is generated in Pymol. PDB codes: ALDH1A1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>), ALDH2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VLE">2VLE</a>), and ALDH3A1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SZA">3SZA</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Selectivity of compounds for WT vs G458N mutant. For CM026 and its analogues, 100 μM of compound was used. For CM037 and CM302, 20 μM of compound was used. Each value is mean/SEM (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Comparison of the active site topography of human ALDH1A1, ALDH2, and ALDH3A1. The three isoenzymes were aligned using LSQ in Coot and the surface figures generated via Pymol. The active site cysteine is shown in red for all three isoenzymes. G458 in ALDH1A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WJ9">4WJ9</a>) and its equivalent residues D457 in ALDH2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=ICW3">ICW3</a>) and I391 in ALDH3A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SZA">3SZA</a>) are shown in yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/medium/jm-2014-01900s_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Comparison of the binding of CM026 to ALDH1A1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7">4WP7</a>) with the binding of daidzin to ALDH2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VLE">2VLE</a>). ALDH1A1 is shown in light blue with CM026 in dark blue. ALDH2 is shown in light green with daidzin in dark green. The active site cysteine is shown in red for both isoenzymes. Although both compounds bind in the same location, they bind in a different orientation enabling CM026 to inhibit ALDH1A1 but not ALDH2 due to steric hindrance of D474 (D457 in the mature sequence, in orange), while daidzin inhibits both isoenzymes. Figure generated in Pymol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-4/jm501900s/20150220/images/large/jm-2014-01900s_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm501900s&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i20">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35626" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35626" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 63 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Vasiliou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estey, T.</span><span> </span><span class="NLM_article-title">Role of human aldehyde dehydrogenases in endobiotic and xenobiotic metabolism</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">279</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1081%2FDMR-120034001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=15237855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVSmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2004&pages=279-299&author=V.+Vasiliouauthor=A.+Pappaauthor=T.+Estey&title=Role+of+human+aldehyde+dehydrogenases+in+endobiotic+and+xenobiotic+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Human Aldehyde Dehydrogenases in Endobiotic and Xenobiotic Metabolism</span></div><div class="casAuthors">Vasiliou, Vasilis; Pappa, Aglaia; Estey, Tia</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">279-299</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">A review.  The human genome contains at least 17 genes that are members of the aldehyde dehydrogenase (ALDH) superfamily.  These genes encode NAD(P)+-dependent enzymes that oxidize a wide range of aldehydes to their corresponding carboxylic acids.  Aldehydes are highly reactive mols. that are intermediates or products involved in a broad spectrum of physiol., biol., and pharmacol. processes.  Aldehydes are generated during retinoic acid biosynthesis and the metab. of amino acids, lipids, carbohydrates, and drugs.  Mutations in several ALDH genes are the mol. basis of inborn errors of metab. and contribute to environmentally induced diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtaDV8EfwFcbVg90H21EOLACvtfcHk0lgKUMI4w0R5xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVSmurg%253D&md5=5e453a7e29a0e350cd7d88b8ef31744e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1081%2FDMR-120034001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1081%252FDMR-120034001%26sid%3Dliteratum%253Aachs%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DPappa%26aufirst%3DA.%26aulast%3DEstey%26aufirst%3DT.%26atitle%3DRole%2520of%2520human%2520aldehyde%2520dehydrogenases%2520in%2520endobiotic%2520and%2520xenobiotic%2520metabolism%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2004%26volume%3D36%26spage%3D279%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Yoshida, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dave, V.</span><span> </span><span class="NLM_article-title">Retinal oxidation activity and biological role of human cytosolic aldehyde dehydrogenase</span> <span class="citation_source-journal">Enzyme</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=1292933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK3sXitVyhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1992&pages=239-244&author=A.+Yoshidaauthor=L.+C.+Hsuauthor=V.+Dave&title=Retinal+oxidation+activity+and+biological+role+of+human+cytosolic+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Retinal oxidation activity and biological role of human cytosolic aldehyde dehydrogenase</span></div><div class="casAuthors">Yoshida, Akira; Hsu, Lily C.; Dave, Vibha</div><div class="citationInfo"><span class="NLM_cas:title">Enzyme</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4-5</span>),
    <span class="NLM_cas:pages">239-44</span>CODEN:
                <span class="NLM_cas:coden">ENZYBT</span>;
        ISSN:<span class="NLM_cas:issn">0013-9432</span>.
    </div><div class="casAbstract">The major human liver cytosolic aldehyde dehydrogenase isoenzyme 1 (ALDH1) is shown to exhibit strong activity for the oxidn. of retinal to retinoic acid, whereas the major mitochondrial isoenzyme 2 (ALDH2) and stomach cytosolic isoenzyme 3 (ALDH3) had no such activity.  The Km of ALDH1 for retinal was ∼0.06 μM at pH 7.5, and the catalytic efficiency (Vmax/Km) for retinal was ∼600-fold higher than that for acetaldehyde.  Thus, ALDH1 can efficiently produce retinoic acid from retinal in tissues with low retinal concns. (<0.1 μM).  The gene for ALDH1 was shown to have hormone response elements.  These findings suggest that the major physiol. substrate of human ALDH1 is retinal, and that its primary biol. role is the generation of retinoic acid, which results in the modulation of cell differentiation, including hormone-mediated development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfdQhgkxbQMbVg90H21EOLACvtfcHk0lgKUMI4w0R5xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXitVyhsbw%253D&md5=9ad5dbcc971d0aa52489f3551b6f13b5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DA.%26aulast%3DHsu%26aufirst%3DL.%2BC.%26aulast%3DDave%26aufirst%3DV.%26atitle%3DRetinal%2520oxidation%2520activity%2520and%2520biological%2520role%2520of%2520human%2520cytosolic%2520aldehyde%2520dehydrogenase%26jtitle%3DEnzyme%26date%3D1992%26volume%3D46%26spage%3D239%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kurys, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambroziak, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietruszko, R.</span><span> </span><span class="NLM_article-title">Human aldehyde dehydrogenase. Purification and characterization of a third isozyme with low <i>K</i><sub>m</sub> for gamma-aminobutyraldehyde</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">264</span><span class="NLM_x">, </span> <span class="NLM_fpage">4715</span><span class="NLM_x">–</span> <span class="NLM_lpage">4721</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=4715-4721&author=G.+Kurysauthor=W.+Ambroziakauthor=R.+Pietruszko&title=Human+aldehyde+dehydrogenase.+Purification+and+characterization+of+a+third+isozyme+with+low+Km+for+gamma-aminobutyraldehyde"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKurys%26aufirst%3DG.%26aulast%3DAmbroziak%26aufirst%3DW.%26aulast%3DPietruszko%26aufirst%3DR.%26atitle%3DHuman%2520aldehyde%2520dehydrogenase.%2520Purification%2520and%2520characterization%2520of%2520a%2520third%2520isozyme%2520with%2520low%2520Km%2520for%2520gamma-aminobutyraldehyde%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D4715%26epage%3D4721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Esterbauer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaur, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zollner, H.</span><span> </span><span class="NLM_article-title">Chemistry and biochemistry of 4-hydroxynonenal, malonaldehyde and related aldehydes</span> <span class="citation_source-journal">Free Radical Biol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2F0891-5849%2891%2990192-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=1937131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK3MXmsFSnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=81-128&author=H.+Esterbauerauthor=R.+J.+Schaurauthor=H.+Zollner&title=Chemistry+and+biochemistry+of+4-hydroxynonenal%2C+malonaldehyde+and+related+aldehydes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry and biochemistry of 4-hydroxynonenal, malonaldehyde and related aldehydes</span></div><div class="casAuthors">Esterbauer, Hermann; Schaur, Rudolf Joerg; Zollner, Helmward</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-128</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    </div><div class="casAbstract">A review, with 400 refs.  The purpose of this review is to provide a comprehensive summary on the chem. properties of 4-hydroxynonenal, 4-hydroxyhexenal, and malonaldehyde, the mechanisms of their formation and their occurrence in biol. systems and methods for their detn.  The authors will also review the reactions of 4-hydroxyalkenals and malonaldehyde with bimol. (amino acids, proteins, nucleic acid bases), their metab. in isolated cells and excretion in whole animals, as well as many types of biol. activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2QNkF-AU-EbVg90H21EOLACvtfcHk0lgumN-b68KR3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXmsFSnsbo%253D&md5=fce66ade3615bb95318c987bc7d63639</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2F0891-5849%2891%2990192-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0891-5849%252891%252990192-6%26sid%3Dliteratum%253Aachs%26aulast%3DEsterbauer%26aufirst%3DH.%26aulast%3DSchaur%26aufirst%3DR.%2BJ.%26aulast%3DZollner%26aufirst%3DH.%26atitle%3DChemistry%2520and%2520biochemistry%2520of%25204-hydroxynonenal%252C%2520malonaldehyde%2520and%2520related%2520aldehydes%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D1991%26volume%3D11%26spage%3D81%26epage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Sayre, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, G.</span><span> </span><span class="NLM_article-title">Chemistry and biochemistry of oxidative stress in neurodegenerative disease</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">738</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.2174%2F0929867013372922" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=11375746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjsVChtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=721-738&author=L.+M.+Sayreauthor=M.+A.+Smithauthor=G.+Perry&title=Chemistry+and+biochemistry+of+oxidative+stress+in+neurodegenerative+disease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry and biochemistry of oxidative stress in neurodegenerative disease</span></div><div class="casAuthors">Sayre, Lawrence M.; Smith, Mark A.; Perry, George</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">721-738</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review with 234 refs.  The age-related neurodegenerative diseases exemplified by Alzheimer's disease (AD), Lewy body diseases such as Parkinson's disease (PD), amyotrophic lateral sclerosis (ALS), and Huntington's disease are characterized by the deposition of abnormal forms of specific proteins in the brain.  Although several factors appear to underlie the pathol. depositions, the cause of neuronal death in each disease appears to be multifactorial.  In this regard, evidence in each case for a role of oxidative stress is provided by the finding that the pathol. deposits are immunoreactive to antibodies recognizing protein side-chains modified either directly by reactive oxygen or nitrogen species, or by products of lipid peroxidn. or glycoxidn.  Although the source(s) of increased oxidative damage are not entirely clear, the findings of increased localization of redox-active transition metals in the brain regions most affected is consistent with their contribution to oxidative stress.  It is tempting to speculate that free radical oxygen chem. plays a pathogenetic role in all these neurodegenerative conditions, though it is as yet undetd. what types of oxidative damage occur early in pathogenesis, and what types are secondary manifestations of dying neurons.  Delineation of the profile of oxidative damage in each disease will provide clues to how the specific neuronal populations are differentially affected by the individual disease conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9DAa0j4fYwbVg90H21EOLACvtfcHk0lgumN-b68KR3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjsVChtbs%253D&md5=ff9d8a618f96b8fc7b6994e88e07191c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F0929867013372922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867013372922%26sid%3Dliteratum%253Aachs%26aulast%3DSayre%26aufirst%3DL.%2BM.%26aulast%3DSmith%26aufirst%3DM.%2BA.%26aulast%3DPerry%26aufirst%3DG.%26atitle%3DChemistry%2520and%2520biochemistry%2520of%2520oxidative%2520stress%2520in%2520neurodegenerative%2520disease%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2001%26volume%3D8%26spage%3D721%26epage%3D738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Chevion, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berenshtein, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stadtman, E. R.</span><span> </span><span class="NLM_article-title">Human studies related to protein oxidation: protein carbonyl content as a marker of damage</span> <span class="citation_source-journal">Free Radical Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl</span><span class="NLM_x">) </span> <span class="NLM_fpage">S99</span><span class="NLM_x">–</span> <span class="NLM_lpage">S108</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=S99-S108&issue=Suppl&author=M.+Chevionauthor=E.+Berenshteinauthor=E.+R.+Stadtman&title=Human+studies+related+to+protein+oxidation%3A+protein+carbonyl+content+as+a+marker+of+damage"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChevion%26aufirst%3DM.%26aulast%3DBerenshtein%26aufirst%3DE.%26aulast%3DStadtman%26aufirst%3DE.%2BR.%26atitle%3DHuman%2520studies%2520related%2520to%2520protein%2520oxidation%253A%2520protein%2520carbonyl%2520content%2520as%2520a%2520marker%2520of%2520damage%26jtitle%3DFree%2520Radical%2520Res.%26date%3D2000%26volume%3D33%26issue%3DSuppl%26spage%3DS99%26epage%3DS108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Marchitti, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brocker, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stagos, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span> </span><span class="NLM_article-title">Non-P450 aldehyde oxidizing enzymes: the aldehyde dehydrogenase superfamily</span> <span class="citation_source-journal">Expert Opin. Drug Metab. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">697</span><span class="NLM_x">–</span> <span class="NLM_lpage">720</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1517%2F17425255.4.6.697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=18611112" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1CgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=697-720&author=S.+A.+Marchittiauthor=C.+Brockerauthor=D.+Stagosauthor=V.+Vasiliou&title=Non-P450+aldehyde+oxidizing+enzymes%3A+the+aldehyde+dehydrogenase+superfamily"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Non-P450 aldehyde oxidizing enzymes: the aldehyde dehydrogenase superfamily</span></div><div class="casAuthors">Marchitti, Satori A.; Brocker, Chad; Stagos, Dimitrios; Vasiliou, Vasilis</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">697-720</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Aldehydes are highly reactive mols.  While several non-P 450 enzyme systems participate in their metab., one of the most important is the aldehyde dehydrogenase (ALDH) superfamily, composed of NAD(P)+-dependent enzymes that catalyze aldehyde oxidn.  This article presents a review of what is currently known about each member of the human ALDH superfamily including the pathophysiol. significance of these enzymes.  Relevant literature involving all members of the human ALDH family was extensively reviewed, with the primary focus on recent and novel findings.  To date, 19 ALDH genes have been identified in the human genome and mutations in these genes and subsequent inborn errors in aldehyde metab. are the mol. basis of several diseases, including Sjoegren-Larsson syndrome, type II hyperprolinemia, γ-hydroxybutyric aciduria and pyridoxine-dependent seizures.  ALDH enzymes also play important roles in embryogenesis and development, neurotransmission, oxidative stress and cancer.  Finally, ALDH enzymes display multiple catalytic and non-catalytic functions including ester hydrolysis, antioxidant properties, xenobiotic bioactivation and UV light absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6bSfChe5Ke7Vg90H21EOLACvtfcHk0lgcBTF0uNzwAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1CgsLc%253D&md5=d377b38fcde9fd18bd30c7dc6b48811a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1517%2F17425255.4.6.697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.4.6.697%26sid%3Dliteratum%253Aachs%26aulast%3DMarchitti%26aufirst%3DS.%2BA.%26aulast%3DBrocker%26aufirst%3DC.%26aulast%3DStagos%26aufirst%3DD.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DNon-P450%2520aldehyde%2520oxidizing%2520enzymes%253A%2520the%2520aldehyde%2520dehydrogenase%2520superfamily%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2008%26volume%3D4%26spage%3D697%26epage%3D720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">O’Brien, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siraki, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shangari, N.</span><span> </span><span class="NLM_article-title">Aldehyde sources, metabolism, molecular toxicity mechanisms, and possible effects on human health</span> <span class="citation_source-journal">Crit. Rev. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">662</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1080%2F10408440591002183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=16417045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1agtr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2005&pages=609-662&author=P.+J.+O%E2%80%99Brienauthor=A.+G.+Sirakiauthor=N.+Shangari&title=Aldehyde+sources%2C+metabolism%2C+molecular+toxicity+mechanisms%2C+and+possible+effects+on+human+health"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde sources, metabolism, molecular toxicity mechanisms, and possible effects on human health</span></div><div class="casAuthors">O'Brien, Peter; Siraki, Arno; Shangari, Nandita</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">609-662</span>CODEN:
                <span class="NLM_cas:coden">CRTXB2</span>;
        ISSN:<span class="NLM_cas:issn">1040-8444</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">A review.  Aldehydes are org. compds. that are widespread in nature.  They can be formed endogenously by lipid peroxidn. (LPO), carbohydrate or metab. ascorbate autoxidn., amine oxidases, cytochrome P-450s, or myeloperoxidase-catalyzed metabolic activation.  This review compares the reactivity of many aldehydes towards biomols. particularly macromols.  Furthermore, it includes not only aldehydes of environmental or occupational concerns but also dietary aldehydes and aldehydes formed endogenously by intermediary metab.  Drugs that are aldehydes or form reactive aldehyde metabolites that cause side-effect toxicity are also included.  The effects of these aldehydes on biol. function, their contribution to human diseases, and the role of nucleic acid and protein carbonylation/oxidn. in mutagenicity and cytotoxicity mechanisms, resp., as well as carbonyl signal transduction and gene expression, are reviewed.  Aldehyde metabolic activation and detoxication by metabolizing enzymes are also reviewed, as well as the toxicol. and anticancer therapeutic effects of metabolizing enzyme inhibitors.  The human health risks from clin. and animal research studies are reviewed, including aldehydes as haptens in allergenic hypersensitivity diseases, respiratory allergies, and idiosyncratic drug toxicity; the potential carcinogenic risks of the carbonyl body burden; and the toxic effects of aldehydes in liver disease, embryo toxicity/teratogenicity, diabetes/hypertension, sclerosing peritonitis, cerebral ischemia/neurodegenerative diseases, and other aging-assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw561xTk_YUrVg90H21EOLACvtfcHk0lgcBTF0uNzwAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1agtr3L&md5=b5be34c7f0253a8e988dd0e93427a399</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1080%2F10408440591002183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10408440591002183%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DP.%2BJ.%26aulast%3DSiraki%26aufirst%3DA.%2BG.%26aulast%3DShangari%26aufirst%3DN.%26atitle%3DAldehyde%2520sources%252C%2520metabolism%252C%2520molecular%2520toxicity%2520mechanisms%252C%2520and%2520possible%2520effects%2520on%2520human%2520health%26jtitle%3DCrit.%2520Rev.%2520Toxicol.%26date%3D2005%26volume%3D35%26spage%3D609%26epage%3D662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Klyosov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rashkovetsky, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keung, W. M.</span><span> </span><span class="NLM_article-title">Possible role of liver cytosolic and mitochondrial aldehyde dehydrogenases in acetaldehyde metabolism</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">4445</span><span class="NLM_x">–</span> <span class="NLM_lpage">4456</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi9521093" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1996&pages=4445-4456&author=A.+A.+Klyosovauthor=L.+G.+Rashkovetskyauthor=M.+K.+Tahirauthor=W.+M.+Keung&title=Possible+role+of+liver+cytosolic+and+mitochondrial+aldehyde+dehydrogenases+in+acetaldehyde+metabolism"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fbi9521093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9521093%26sid%3Dliteratum%253Aachs%26aulast%3DKlyosov%26aufirst%3DA.%2BA.%26aulast%3DRashkovetsky%26aufirst%3DL.%2BG.%26aulast%3DTahir%26aufirst%3DM.%2BK.%26aulast%3DKeung%26aufirst%3DW.%2BM.%26atitle%3DPossible%2520role%2520of%2520liver%2520cytosolic%2520and%2520mitochondrial%2520aldehyde%2520dehydrogenases%2520in%2520acetaldehyde%2520metabolism%26jtitle%3DBiochemistry%26date%3D1996%26volume%3D35%26spage%3D4445%26epage%3D4456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Yoshida, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, I. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikawa, M.</span><span> </span><span class="NLM_article-title">Molecular abnormality of an inactive aldehyde dehydrogenase variant commonly found in Orientals</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">258</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=258-261&author=A.+Yoshidaauthor=I.+Y.+Huangauthor=M.+Ikawa&title=Molecular+abnormality+of+an+inactive+aldehyde+dehydrogenase+variant+commonly+found+in+Orientals"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DI.%2BY.%26aulast%3DIkawa%26aufirst%3DM.%26atitle%3DMolecular%2520abnormality%2520of%2520an%2520inactive%2520aldehyde%2520dehydrogenase%2520variant%2520commonly%2520found%2520in%2520Orientals%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1984%26volume%3D81%26spage%3D258%26epage%3D261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Peng, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsao, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, S. J.</span><span> </span><span class="NLM_article-title">Pharmacokinetic and pharmacodynamic basis for partial protection against alcoholism in Asians, heterozygous for the variant ALDH2*2 gene allele</span> <span class="citation_source-journal">Pharmacogenet. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">845</span><span class="NLM_x">–</span> <span class="NLM_lpage">855</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1097%2FFPC.0b013e3282609e67" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=17885622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVKhsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=845-855&author=G.+S.+Pengauthor=Y.+C.+Chenauthor=T.+P.+Tsaoauthor=M.+F.+Wangauthor=S.+J.+Yin&title=Pharmacokinetic+and+pharmacodynamic+basis+for+partial+protection+against+alcoholism+in+Asians%2C+heterozygous+for+the+variant+ALDH2*2+gene+allele"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetic and pharmacodynamic basis for partial protection against alcoholism in Asians, heterozygous for the variant ALDH2*2 gene allele</span></div><div class="casAuthors">Peng, Giia-Sheun; Chen, Yi-Chyan; Tsao, Tien-Ping; Wang, Ming-Fang; Yin, Shih-Jiun</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacogenetics and Genomics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">845-855</span>CODEN:
                <span class="NLM_cas:coden">PGHEAI</span>;
        ISSN:<span class="NLM_cas:issn">1744-6872</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Objectives: It has been well documented that although homozygosity of the variant aldehyde dehydrogenase-2 (ALDH2) gene allele, ALDH2*2, in Asians almost fully protects against alcoholism, the heterozygosity only affords a partial protection to varying degrees.  The full protection against alcoholism has been ascribed to the low-amt. of alc. hypersensitivity accompanied by a prolonged and large accumulation of acetaldehyde in blood (Peng et al. Pharmacogenetics 1999; 9:463-476).  The physiol. basis for the partial protection, however, remains unclear.  Methods: To address this question, we recruited a total of 32 adult Han Chinese men, matched by age, body-mass index, and nutritional state from a population base of 404 men.  The subjects were divided into 3 combinatorial genotypic groups of alc. dehydrogenase (ADH) and ALDH, i.e., ALDH2*1/*1-ADH1B*1/*1-ADH1C*1/*1 (n = 8), ALDH2*1/*1-ADH1B*2/*2-ADH1C*1/*1 (n = 8), and ALDH2*1/*2-ADH1B*2/*2-ADH1C*1/*1 (n = 16).  Following a moderate dose of ethanol (0.5 g/kg body wt.), blood ethanol/acetaldehyde/acetate concns., cardiac and extracranial/intracranial arterial hemodynamic parameters, as well as self-rated subjective sensations, were measured for 130 min.  Results: Heterozygotic ALDH2*1/*2 subjects were found to be strikingly responsive to the moderate amt. of alc., as evidenced by the prominent cardiovascular effects as well as subjective perceptions of general discomfort for as long as 2 h following ingestion.  The ADH1B polymorphisms did not appear to correlate with the pharmacokinetic and pharmacodynamic effects.  Conclusions: These results indicate that acetaldehyde, rather than ethanol or acetate, is primarily responsible for the obsd. alc. sensitivity reactions and suggest that substantially lower accumulation of blood acetaldehyde in the heterozygotes significantly reduces the aversion reaction to low amts. of alc. that permits other biol. as well as environmental factors to facilitate drinking and the according risk for the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowSjexF008DrVg90H21EOLACvtfcHk0ljWq3PqLJ7V8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVKhsLfF&md5=534c77731b1f1b51c8890ad174ea3c8b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1097%2FFPC.0b013e3282609e67&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FFPC.0b013e3282609e67%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DG.%2BS.%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DTsao%26aufirst%3DT.%2BP.%26aulast%3DWang%26aufirst%3DM.%2BF.%26aulast%3DYin%26aufirst%3DS.%2BJ.%26atitle%3DPharmacokinetic%2520and%2520pharmacodynamic%2520basis%2520for%2520partial%2520protection%2520against%2520alcoholism%2520in%2520Asians%252C%2520heterozygous%2520for%2520the%2520variant%2520ALDH2%252A2%2520gene%2520allele%26jtitle%3DPharmacogenet.%2520Genomics%26date%3D2007%26volume%3D17%26spage%3D845%26epage%3D855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Marchitti, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deitrich, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span> </span><span class="NLM_article-title">Neurotoxicity and metabolism of the catecholamine-derived 3,4-dihydroxyphenylacetaldehyde and 3,4-dihydroxyphenylglycolaldehyde: the role of aldehyde dehydrogenase</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">150</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2007&pages=125-150&author=S.+A.+Marchittiauthor=R.+A.+Deitrichauthor=V.+Vasiliou&title=Neurotoxicity+and+metabolism+of+the+catecholamine-derived+3%2C4-dihydroxyphenylacetaldehyde+and+3%2C4-dihydroxyphenylglycolaldehyde%3A+the+role+of+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarchitti%26aufirst%3DS.%2BA.%26aulast%3DDeitrich%26aufirst%3DR.%2BA.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DNeurotoxicity%2520and%2520metabolism%2520of%2520the%2520catecholamine-derived%25203%252C4-dihydroxyphenylacetaldehyde%2520and%25203%252C4-dihydroxyphenylglycolaldehyde%253A%2520the%2520role%2520of%2520aldehyde%2520dehydrogenase%26jtitle%3DPharmacol.%2520Rev.%26date%3D2007%26volume%3D59%26spage%3D125%26epage%3D150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Yao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arolfo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olive, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zablocki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shryock, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackburn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diamond, I.</span><span> </span><span class="NLM_article-title">Inhibition of aldehyde dehydrogenase-2 suppresses cocaine seeking by generating THP, a cocaine use-dependent inhibitor of dopamine synthesis</span> <span class="citation_source-journal">Nature Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1024</span><span class="NLM_x">–</span> <span class="NLM_lpage">1028</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=1024-1028&author=L.+Yaoauthor=P.+Fanauthor=M.+Arolfoauthor=Z.+Jiangauthor=M.+F.+Oliveauthor=J.+Zablockiauthor=H.+L.+Sunauthor=N.+Chuauthor=J.+Leeauthor=H.+Y.+Kimauthor=K.+Leungauthor=J.+Shryockauthor=B.+Blackburnauthor=I.+Diamond&title=Inhibition+of+aldehyde+dehydrogenase-2+suppresses+cocaine+seeking+by+generating+THP%2C+a+cocaine+use-dependent+inhibitor+of+dopamine+synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DL.%26aulast%3DFan%26aufirst%3DP.%26aulast%3DArolfo%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DOlive%26aufirst%3DM.%2BF.%26aulast%3DZablocki%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DH.%2BL.%26aulast%3DChu%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%2BY.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DShryock%26aufirst%3DJ.%26aulast%3DBlackburn%26aufirst%3DB.%26aulast%3DDiamond%26aufirst%3DI.%26atitle%3DInhibition%2520of%2520aldehyde%2520dehydrogenase-2%2520suppresses%2520cocaine%2520seeking%2520by%2520generating%2520THP%252C%2520a%2520cocaine%2520use-dependent%2520inhibitor%2520of%2520dopamine%2520synthesis%26jtitle%3DNature%2520Med.%26date%3D2010%26volume%3D16%26spage%3D1024%26epage%3D1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Rizzo, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dammann, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craft, D. A.</span><span> </span><span class="NLM_article-title">Sjogren–Larsson syndrome. Impaired fatty alcohol oxidation in cultured fibroblasts due to deficient fatty alcohol:nicotinamide adenine dinucleotide oxidoreductase activity</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">738</span><span class="NLM_x">–</span> <span class="NLM_lpage">744</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1988&pages=738-744&author=W.+B.+Rizzoauthor=A.+L.+Dammannauthor=D.+A.+Craft&title=Sjogren%E2%80%93Larsson+syndrome.+Impaired+fatty+alcohol+oxidation+in+cultured+fibroblasts+due+to+deficient+fatty+alcohol%3Anicotinamide+adenine+dinucleotide+oxidoreductase+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRizzo%26aufirst%3DW.%2BB.%26aulast%3DDammann%26aufirst%3DA.%2BL.%26aulast%3DCraft%26aufirst%3DD.%2BA.%26atitle%3DSjogren%25E2%2580%2593Larsson%2520syndrome.%2520Impaired%2520fatty%2520alcohol%2520oxidation%2520in%2520cultured%2520fibroblasts%2520due%2520to%2520deficient%2520fatty%2520alcohol%253Anicotinamide%2520adenine%2520dinucleotide%2520oxidoreductase%2520activity%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1988%26volume%3D81%26spage%3D738%26epage%3D744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Rizzo, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, G.</span><span> </span><span class="NLM_article-title">Sjogren–Larsson syndrome: diversity of mutations and polymorphisms in the fatty aldehyde dehydrogenase gene (ALDH3A2)</span> <span class="citation_source-journal">Hum. Mutat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1002%2Fhumu.20181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=15931689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpslSksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1-10&author=W.+B.+Rizzoauthor=G.+Carney&title=Sjogren%E2%80%93Larsson+syndrome%3A+diversity+of+mutations+and+polymorphisms+in+the+fatty+aldehyde+dehydrogenase+gene+%28ALDH3A2%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Sjogren-Larsson syndrome: diversity of mutations and polymorphisms in the fatty aldehyde dehydrogenase gene (ALDH3A2)</span></div><div class="casAuthors">Rizzo, William B.; Carney, Gael</div><div class="citationInfo"><span class="NLM_cas:title">Human Mutation</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-10</span>CODEN:
                <span class="NLM_cas:coden">HUMUE3</span>;
        ISSN:<span class="NLM_cas:issn">1059-7794</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  Sjogren-Larsson syndrome (SLS) is an autosomal recessive disorder characterized by ichthyosis, mental retardation, and spastic diplegia or tetraplegia.  The disease is caused by mutations in the ALDH3A2 gene (also known as FALDH and ALDH10) on chromosome 17p11.2 that encodes fatty aldehyde dehydrogenase (FALDH), an enzyme that catalyzes the oxidn. of long-chain aldehydes derived from lipid metab.  In SLS patients, 72 mutations have been identified, with a distribution that is scattered throughout the ALDH3A2 gene.  Most mutations are private but several common mutations have been detected, which probably reflect founder effects or recurrent mutational events.  Missense mutations comprise the most abundant class (38%) and expression studies indicate that most of these result in a profound redn. in enzyme activity.  Deletions account for about 25% of the mutations and range from single nucleotides to entire exons.  Twelve splice-site mutations have been demonstrated to cause aberrant splicing in cultured fibroblasts.  To date, more than a dozen intragenic ALDH3A2 polymorphisms consisting of SNPs and one microsatellite marker have been characterized, although none of them alter the FALDH protein sequence.  The striking mutational diversity in SLS offers a challenge for DNA-based diagnosis, but promises to provide a wealth of information about enzyme structure-function correlations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS0A16LhWAU7Vg90H21EOLACvtfcHk0liFkPG771e43g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpslSksL0%253D&md5=c5969126d454ddd0c65579a0215367d5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fhumu.20181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhumu.20181%26sid%3Dliteratum%253Aachs%26aulast%3DRizzo%26aufirst%3DW.%2BB.%26aulast%3DCarney%26aufirst%3DG.%26atitle%3DSjogren%25E2%2580%2593Larsson%2520syndrome%253A%2520diversity%2520of%2520mutations%2520and%2520polymorphisms%2520in%2520the%2520fatty%2520aldehyde%2520dehydrogenase%2520gene%2520%2528ALDH3A2%2529%26jtitle%3DHum.%2520Mutat.%26date%3D2005%26volume%3D26%26spage%3D1%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Mills, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Struys, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakobs, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plecko, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumgartner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willemsen, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omran, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tacke, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uhlenberg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weschke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clayton, P. T.</span><span> </span><span class="NLM_article-title">Mutations in antiquitin in individuals with pyridoxine-dependent seizures</span> <span class="citation_source-journal">Nature Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">309</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=307-309&author=P.+B.+Millsauthor=E.+Struysauthor=C.+Jakobsauthor=B.+Pleckoauthor=P.+Baxterauthor=M.+Baumgartnerauthor=M.+A.+Willemsenauthor=H.+Omranauthor=U.+Tackeauthor=B.+Uhlenbergauthor=B.+Weschkeauthor=P.+T.+Clayton&title=Mutations+in+antiquitin+in+individuals+with+pyridoxine-dependent+seizures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMills%26aufirst%3DP.%2BB.%26aulast%3DStruys%26aufirst%3DE.%26aulast%3DJakobs%26aufirst%3DC.%26aulast%3DPlecko%26aufirst%3DB.%26aulast%3DBaxter%26aufirst%3DP.%26aulast%3DBaumgartner%26aufirst%3DM.%26aulast%3DWillemsen%26aufirst%3DM.%2BA.%26aulast%3DOmran%26aufirst%3DH.%26aulast%3DTacke%26aufirst%3DU.%26aulast%3DUhlenberg%26aufirst%3DB.%26aulast%3DWeschke%26aufirst%3DB.%26aulast%3DClayton%26aufirst%3DP.%2BT.%26atitle%3DMutations%2520in%2520antiquitin%2520in%2520individuals%2520with%2520pyridoxine-dependent%2520seizures%26jtitle%3DNature%2520Med.%26date%3D2006%26volume%3D12%26spage%3D307%26epage%3D309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Geraghty, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaughn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jimenez-Sanchez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, C. A.</span><span> </span><span class="NLM_article-title">Mutations in the Delta1-pyrroline 5-carboxylate dehydrogenase gene cause type II hyperprolinemia</span> <span class="citation_source-journal">Hum. Mol. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1411</span><span class="NLM_x">–</span> <span class="NLM_lpage">1415</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1093%2Fhmg%2F7.9.1411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=9700195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK1cXmtVSlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1998&pages=1411-1415&author=M.+T.+Geraghtyauthor=D.+Vaughnauthor=A.+J.+Nicholsonauthor=W.+W.+Linauthor=G.+Jimenez-Sanchezauthor=C.+Obieauthor=M.+P.+Flynnauthor=D.+Valleauthor=C.+A.+Hu&title=Mutations+in+the+Delta1-pyrroline+5-carboxylate+dehydrogenase+gene+cause+type+II+hyperprolinemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the Δ1-pyrroline 5-carboxylate dehydrogenase gene cause type II hyperprolinemia</span></div><div class="casAuthors">Geraghty, Michael T.; Vaughn, D.; Nicholson, A. J.; Lin, Wei-Wen; Jimenez-Sanchez, Gerardo; Obie, Cassandra; Flynn, M. P.; Valle, David; Hu, Chien-an A.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1411-1415</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We surveyed Δ1-pyrroline 5-carboxylate dehydrogenase genes from four patients with hyperprolinemia type II using RT-PCR amplification, genomic PCR amplification and direct sequencing.  We found four mutant alleles, two with frameshift mutations [A7fs(-1) and G521fs(+1)] and two with missense mutations (S352L and P16L).  To test the functional consequences of three of these, we expressed them in a P5CDh-deficient strain of Saccharomyces cerevisiae.  In contrast to wild-type human P5CDh, yeast expressing S352L and G521fs(+1) failed to grow on proline and had no detectable P5CDh activity.  The P16L allele, however, produced fully functional P5CDh and subsequent anal. suggests that it is polymorphic in the relevant (Spanish) population.  Interestingly, the G521fs(+1) allele segregates in the large Irish Traveler pedigree used to define the HPII phenotype.  To our knowledge, this is the first description of the mol. basis for this inborn error.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo6wZtdarVurVg90H21EOLACvtfcHk0liFkPG771e43g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtVSlsLg%253D&md5=5e068b38cf959de7c04dcc1bcea392d0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F7.9.1411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F7.9.1411%26sid%3Dliteratum%253Aachs%26aulast%3DGeraghty%26aufirst%3DM.%2BT.%26aulast%3DVaughn%26aufirst%3DD.%26aulast%3DNicholson%26aufirst%3DA.%2BJ.%26aulast%3DLin%26aufirst%3DW.%2BW.%26aulast%3DJimenez-Sanchez%26aufirst%3DG.%26aulast%3DObie%26aufirst%3DC.%26aulast%3DFlynn%26aufirst%3DM.%2BP.%26aulast%3DValle%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DC.%2BA.%26atitle%3DMutations%2520in%2520the%2520Delta1-pyrroline%25205-carboxylate%2520dehydrogenase%2520gene%2520cause%2520type%2520II%2520hyperprolinemia%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D1998%26volume%3D7%26spage%3D1411%26epage%3D1415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Farrant, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellor, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langley, G. J.</span><span> </span><span class="NLM_article-title">Pyridoxal phosphate deactivation by pyrroline-5-carboxylic acid. Increased risk of vitamin B6 deficiency and seizures in hyperprolinemia type II</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">15107</span><span class="NLM_x">–</span> <span class="NLM_lpage">15116</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=15107-15116&author=R.+D.+Farrantauthor=V.+Walkerauthor=G.+A.+Millsauthor=J.+M.+Mellorauthor=G.+J.+Langley&title=Pyridoxal+phosphate+deactivation+by+pyrroline-5-carboxylic+acid.+Increased+risk+of+vitamin+B6+deficiency+and+seizures+in+hyperprolinemia+type+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFarrant%26aufirst%3DR.%2BD.%26aulast%3DWalker%26aufirst%3DV.%26aulast%3DMills%26aufirst%3DG.%2BA.%26aulast%3DMellor%26aufirst%3DJ.%2BM.%26aulast%3DLangley%26aufirst%3DG.%2BJ.%26atitle%3DPyridoxal%2520phosphate%2520deactivation%2520by%2520pyrroline-5-carboxylic%2520acid.%2520Increased%2520risk%2520of%2520vitamin%2520B6%2520deficiency%2520and%2520seizures%2520in%2520hyperprolinemia%2520type%2520II%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D15107%26epage%3D15116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Marchitti, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlicky, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span> </span><span class="NLM_article-title">Expression and initial characterization of human ALDH3B1</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">356</span><span class="NLM_x">, </span> <span class="NLM_fpage">792</span><span class="NLM_x">–</span> <span class="NLM_lpage">798</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2007&pages=792-798&author=S.+A.+Marchittiauthor=D.+J.+Orlickyauthor=V.+Vasiliou&title=Expression+and+initial+characterization+of+human+ALDH3B1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarchitti%26aufirst%3DS.%2BA.%26aulast%3DOrlicky%26aufirst%3DD.%2BJ.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DExpression%2520and%2520initial%2520characterization%2520of%2520human%2520ALDH3B1%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D356%26spage%3D792%26epage%3D798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Ma, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allan, A. L.</span><span> </span><span class="NLM_article-title">The role of human aldehyde dehydrogenase in normal and cancer stem cells</span> <span class="citation_source-journal">Stem Cell Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">292</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1007%2Fs12015-010-9208-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=21103958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFWjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=292-306&author=I.+Maauthor=A.+L.+Allan&title=The+role+of+human+aldehyde+dehydrogenase+in+normal+and+cancer+stem+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The role of human aldehyde dehydrogenase in normal and cancer stem cells</span></div><div class="casAuthors">Ma, Irene; Allan, Alison L.</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reviews and Reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">292-306</span>CODEN:
                <span class="NLM_cas:coden">SCRREE</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Normal stem cells and cancer stem cells (CSCs) share similar properties, in that both have the capacity to self-renew and differentiate into multiple cell types.  In both the normal stem cell and cancer stem cell fields, there has been a great need for a universal marker that can effectively identify and isolate these rare populations of cells in order to characterize them and use this information for research and therapeutic purposes.  Currently, it would appear that certain isoenzymes of the aldehyde dehydrogenase (ALDH) superfamily may be able to fulfill this role as a marker for both normal and cancer stem cells.  ALDH has been identified as an important enzyme in the protection of normal hematopoietic stem cells, and is now also widely used as a marker to identify and isolate various types of normal stem cells and CSCs.  In addn., emerging evidence suggests that ALDH1 is not only a marker for stem cells, but may also play important functional roles related to self-protection, differentiation, and expansion.  This comprehensive review discusses the role that ALDH plays in normal stem cells and CSCs, with focus on the ALDH1 and ALDH3A1 isoenzymes.  Discrepancies in the functional themes between cell types and future perspectives for therapeutic applications are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog42zeNePwFLVg90H21EOLACvtfcHk0ljFTwDgE-gs0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFWjtbk%253D&md5=0fbede2e32f7ffcce91c8b6dba24711a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs12015-010-9208-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12015-010-9208-4%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DI.%26aulast%3DAllan%26aufirst%3DA.%2BL.%26atitle%3DThe%2520role%2520of%2520human%2520aldehyde%2520dehydrogenase%2520in%2520normal%2520and%2520cancer%2520stem%2520cells%26jtitle%3DStem%2520Cell%2520Rev.%26date%3D2011%26volume%3D7%26spage%3D292%26epage%3D306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Sladek, N. E.</span><span> </span><span class="NLM_article-title">Aldehyde dehydrogenase-mediated cellular relative insensitivity to the oxazaphosphorines</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">625</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=607-625&author=N.+E.+Sladek&title=Aldehyde+dehydrogenase-mediated+cellular+relative+insensitivity+to+the+oxazaphosphorines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSladek%26aufirst%3DN.%2BE.%26atitle%3DAldehyde%2520dehydrogenase-mediated%2520cellular%2520relative%2520insensitivity%2520to%2520the%2520oxazaphosphorines%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D1999%26volume%3D5%26spage%3D607%26epage%3D625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Agarwal, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Eitzen, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meier-Tackmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goedde, H. W.</span><span> </span><span class="NLM_article-title">Metabolism of cyclophosphamide by aldehyde dehydrogenases</span> <span class="citation_source-journal">Adv. Exp. Med. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">372</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=1995&pages=115-122&author=D.+P.+Agarwalauthor=U.+von+Eitzenauthor=D.+Meier-Tackmannauthor=H.+W.+Goedde&title=Metabolism+of+cyclophosphamide+by+aldehyde+dehydrogenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwal%26aufirst%3DD.%2BP.%26aulast%3Dvon%2BEitzen%26aufirst%3DU.%26aulast%3DMeier-Tackmann%26aufirst%3DD.%26aulast%3DGoedde%26aufirst%3DH.%2BW.%26atitle%3DMetabolism%2520of%2520cyclophosphamide%2520by%2520aldehyde%2520dehydrogenases%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D1995%26volume%3D372%26spage%3D115%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Yoon, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arakawa, H.</span><span> </span><span class="NLM_article-title">Identification of ALDH4 as a p53-inducible gene and its protective role in cellular stresses</span> <span class="citation_source-journal">J. Hum. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">134</span><span class="NLM_x">–</span> <span class="NLM_lpage">140</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2004&pages=134-140&author=K.+A.+Yoonauthor=Y.+Nakamuraauthor=H.+Arakawa&title=Identification+of+ALDH4+as+a+p53-inducible+gene+and+its+protective+role+in+cellular+stresses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoon%26aufirst%3DK.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DArakawa%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520ALDH4%2520as%2520a%2520p53-inducible%2520gene%2520and%2520its%2520protective%2520role%2520in%2520cellular%2520stresses%26jtitle%3DJ.%2520Hum.%2520Genet.%26date%3D2004%26volume%3D49%26spage%3D134%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapointe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaygusuz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ong, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Rijn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollack, J. R.</span><span> </span><span class="NLM_article-title">The retinoic acid synthesis gene ALDH1a2 is a candidate tumor suppressor in prostate cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">8118</span><span class="NLM_x">–</span> <span class="NLM_lpage">8124</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=8118-8124&author=H.+Kimauthor=J.+Lapointeauthor=G.+Kaygusuzauthor=D.+E.+Ongauthor=C.+Liauthor=M.+van+de+Rijnauthor=J.+D.+Brooksauthor=J.+R.+Pollack&title=The+retinoic+acid+synthesis+gene+ALDH1a2+is+a+candidate+tumor+suppressor+in+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.%26aulast%3DLapointe%26aufirst%3DJ.%26aulast%3DKaygusuz%26aufirst%3DG.%26aulast%3DOng%26aufirst%3DD.%2BE.%26aulast%3DLi%26aufirst%3DC.%26aulast%3Dvan%2Bde%2BRijn%26aufirst%3DM.%26aulast%3DBrooks%26aufirst%3DJ.%2BD.%26aulast%3DPollack%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520retinoic%2520acid%2520synthesis%2520gene%2520ALDH1a2%2520is%2520a%2520candidate%2520tumor%2520suppressor%2520in%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D8118%26epage%3D8124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Matsuda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabushita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanaly, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibutani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, A.</span><span> </span><span class="NLM_article-title">Increased DNA damage in ALDH2-deficient alcoholics</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1374</span><span class="NLM_x">–</span> <span class="NLM_lpage">1378</span></span><div class="citationLinks">[<a href="/doi/10.1021/tx060113h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVynsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2006&pages=1374-1378&author=T.+Matsudaauthor=H.+Yabushitaauthor=R.+A.+Kanalyauthor=S.+Shibutaniauthor=A.+Yokoyama&title=Increased+DNA+damage+in+ALDH2-deficient+alcoholics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Increased DNA Damage in ALDH2-Deficient Alcoholics</span></div><div class="casAuthors">Matsuda, Tomonari; Yabushita, Hisatoshi; Kanaly, Robert A.; Shibutani, Shinya; Yokoyama, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1374-1378</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drinking alc. is a risk factor for cancers of the oral cavity, pharynx, larynx, and esophagus.  Although many studies suggest that acetaldehyde, a major metabolite of orally ingested alc., plays a crucial role in cancer initiation, the link between the aldehyde dehydrogenase-2 (ALDH2) genotype and acetaldehyde-derived DNA damage has not yet been explored.  We have developed a sensitive and quant. method for detecting the acetaldehyde-derived DNA adducts, N2-ethyl-2'-deoxyguanosine (N2-Et-dG), α-S- and α-R-methyl-γ-hydroxy-1,N2-propano-2'-deoxyguanosine (α-S-Me-γ-OH-PdG and α-R-Me-γ-OH-PdG), and N2-(2,6-dimethyl-1,3-dioxan-4-yl)-deoxyguanosine (N2-Dio-dG), by using liq. chromatog. electrospray tandem mass spectrometry (LC/ESI-MS/MS) and stable-isotope internal stds.  We detd. the DNA adducts in 44 blood DNA samples from Japanese alc. patients.  The levels of three acetaldehyde-derived DNA adducts, N2-Et-dG, α-S-Me-γ-OH-PdG, and α-R-Me-γ-OH-PdG, were significantly higher in alcoholics with the ALDH2*1/2*2 genotype compared to those with the ALDH2*1/2*1 genotype.  N2-Dio-dG was not detected in any of the DNA samples analyzed.  These results provide mol. evidence that the ALDH2 genotype affects the genotoxic damage caused by acetaldehyde.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQSJneK9U1XrVg90H21EOLACvtfcHk0lizHXlOa-wRig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVynsLk%253D&md5=b7fe8c0a95ed1ee6564ff73a75409998</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Ftx060113h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx060113h%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuda%26aufirst%3DT.%26aulast%3DYabushita%26aufirst%3DH.%26aulast%3DKanaly%26aufirst%3DR.%2BA.%26aulast%3DShibutani%26aufirst%3DS.%26aulast%3DYokoyama%26aufirst%3DA.%26atitle%3DIncreased%2520DNA%2520damage%2520in%2520ALDH2-deficient%2520alcoholics%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2006%26volume%3D19%26spage%3D1374%26epage%3D1378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Yokoyama, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Omori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsushita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maruyama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishii, H.</span><span> </span><span class="NLM_article-title">Alcohol and aldehyde dehydrogenase gene polymorphisms and oropharyngolaryngeal, esophageal and stomach cancers in Japanese alcoholics</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">433</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=433-439&author=A.+Yokoyamaauthor=T.+Muramatsuauthor=T.+Omoriauthor=T.+Yokoyamaauthor=S.+Matsushitaauthor=S.+Higuchiauthor=K.+Maruyamaauthor=H.+Ishii&title=Alcohol+and+aldehyde+dehydrogenase+gene+polymorphisms+and+oropharyngolaryngeal%2C+esophageal+and+stomach+cancers+in+Japanese+alcoholics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DOmori%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DT.%26aulast%3DMatsushita%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DS.%26aulast%3DMaruyama%26aufirst%3DK.%26aulast%3DIshii%26aufirst%3DH.%26atitle%3DAlcohol%2520and%2520aldehyde%2520dehydrogenase%2520gene%2520polymorphisms%2520and%2520oropharyngolaryngeal%252C%2520esophageal%2520and%2520stomach%2520cancers%2520in%2520Japanese%2520alcoholics%26jtitle%3DCarcinogenesis%26date%3D2001%26volume%3D22%26spage%3D433%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Jones, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barber, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vala, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collector, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufmann, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludeman, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colvin, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilton, J.</span><span> </span><span class="NLM_article-title">Assessment of aldehyde dehydrogenase in viable cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">2742</span><span class="NLM_x">–</span> <span class="NLM_lpage">2746</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=1995&pages=2742-2746&author=R.+J.+Jonesauthor=J.+P.+Barberauthor=M.+S.+Valaauthor=M.+I.+Collectorauthor=S.+H.+Kaufmannauthor=S.+M.+Ludemanauthor=O.+M.+Colvinauthor=J.+Hilton&title=Assessment+of+aldehyde+dehydrogenase+in+viable+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DR.%2BJ.%26aulast%3DBarber%26aufirst%3DJ.%2BP.%26aulast%3DVala%26aufirst%3DM.%2BS.%26aulast%3DCollector%26aufirst%3DM.%2BI.%26aulast%3DKaufmann%26aufirst%3DS.%2BH.%26aulast%3DLudeman%26aufirst%3DS.%2BM.%26aulast%3DColvin%26aufirst%3DO.%2BM.%26aulast%3DHilton%26aufirst%3DJ.%26atitle%3DAssessment%2520of%2520aldehyde%2520dehydrogenase%2520in%2520viable%2520cells%26jtitle%3DBlood%26date%3D1995%26volume%3D85%26spage%3D2742%26epage%3D2746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Chambon, P.</span><span> </span><span class="NLM_article-title">A decade of molecular biology of retinoic acid receptors</span> <span class="citation_source-journal">FASEB J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">940</span><span class="NLM_x">–</span> <span class="NLM_lpage">954</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1096%2Ffasebj.10.9.8801176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=8801176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK28Xks1Wnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1996&pages=940-954&author=P.+Chambon&title=A+decade+of+molecular+biology+of+retinoic+acid+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of molecular biology of retinoic acid receptors</span></div><div class="casAuthors">Chambon, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">940-954</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A review, with 120 refs., on the signaling pathway, regulation, and structure-activity relations of retinoic acid receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKohrdFWv7e7Vg90H21EOLACvtfcHk0lhEjyXco7ORDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xks1Wnsb0%253D&md5=d20140a9229ec056a20a7d79b16418b8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1096%2Ffasebj.10.9.8801176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffasebj.10.9.8801176%26sid%3Dliteratum%253Aachs%26aulast%3DChambon%26aufirst%3DP.%26atitle%3DA%2520decade%2520of%2520molecular%2520biology%2520of%2520retinoic%2520acid%2520receptors%26jtitle%3DFASEB%2520J.%26date%3D1996%26volume%3D10%26spage%3D940%26epage%3D954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandell, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trainor, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koentgen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duester, G.</span><span> </span><span class="NLM_article-title">Alcohol and aldehyde dehydrogenases: retinoid metabolic effects in mouse knockout models</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">1821</span><span class="NLM_x">, </span> <span class="NLM_fpage">198</span><span class="NLM_x">–</span> <span class="NLM_lpage">205</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1821&publication_year=2012&pages=198-205&author=S.+Kumarauthor=L.+L.+Sandellauthor=P.+A.+Trainorauthor=F.+Koentgenauthor=G.+Duester&title=Alcohol+and+aldehyde+dehydrogenases%3A+retinoid+metabolic+effects+in+mouse+knockout+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DSandell%26aufirst%3DL.%2BL.%26aulast%3DTrainor%26aufirst%3DP.%2BA.%26aulast%3DKoentgen%26aufirst%3DF.%26aulast%3DDuester%26aufirst%3DG.%26atitle%3DAlcohol%2520and%2520aldehyde%2520dehydrogenases%253A%2520retinoid%2520metabolic%2520effects%2520in%2520mouse%2520knockout%2520models%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2012%26volume%3D1821%26spage%3D198%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Niederreither, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraulob, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garnier, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolle, P.</span><span> </span><span class="NLM_article-title">Differential expression of retinoic acid-synthesizing (RALDH) enzymes during fetal development and organ differentiation in the mouse</span> <span class="citation_source-journal">Mech. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">165</span><span class="NLM_x">–</span> <span class="NLM_lpage">171</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2002&pages=165-171&author=K.+Niederreitherauthor=V.+Fraulobauthor=J.+M.+Garnierauthor=P.+Chambonauthor=P.+Dolle&title=Differential+expression+of+retinoic+acid-synthesizing+%28RALDH%29+enzymes+during+fetal+development+and+organ+differentiation+in+the+mouse"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNiederreither%26aufirst%3DK.%26aulast%3DFraulob%26aufirst%3DV.%26aulast%3DGarnier%26aufirst%3DJ.%2BM.%26aulast%3DChambon%26aufirst%3DP.%26aulast%3DDolle%26aufirst%3DP.%26atitle%3DDifferential%2520expression%2520of%2520retinoic%2520acid-synthesizing%2520%2528RALDH%2529%2520enzymes%2520during%2520fetal%2520development%2520and%2520organ%2520differentiation%2520in%2520the%2520mouse%26jtitle%3DMech.%2520Dev.%26date%3D2002%26volume%3D110%26spage%3D165%26epage%3D171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Koppaka, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellermann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolaou, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juvonen, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deitrich, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span> </span><span class="NLM_article-title">Aldehyde dehydrogenase inhibitors: a comprehensive review of the pharmacology, mechanism of action, substrate specificity, and clinical application</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">520</span><span class="NLM_x">–</span> <span class="NLM_lpage">539</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1124%2Fpr.111.005538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=22544865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslaktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2012&pages=520-539&author=V.+Koppakaauthor=D.+C.+Thompsonauthor=Y.+Chenauthor=M.+Ellermannauthor=K.+C.+Nicolaouauthor=R.+O.+Juvonenauthor=D.+Petersenauthor=R.+A.+Deitrichauthor=T.+D.+Hurleyauthor=V.+Vasiliou&title=Aldehyde+dehydrogenase+inhibitors%3A+a+comprehensive+review+of+the+pharmacology%2C+mechanism+of+action%2C+substrate+specificity%2C+and+clinical+application"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenase inhibitors: a comprehensive review of the pharmacology, mechanism of action, substrate specificity, and clinical application</span></div><div class="casAuthors">Koppaka, Vindhya; Thompson, David C.; Chen, Ying; Ellermann, Manuel; Nicolaou, Kyriacos C.; Juvonen, Risto O.; Petersen, Dennis; Deitrich, Richard A.; Hurley, Thomas D.; Vasiliou, Vasilis</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">520-539</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Aldehyde dehydrogenases (ALDHs) belong to a superfamily of enzymes that play a key role in the metab. of aldehydes of both endogenous and exogenous derivation.  The human ALDH superfamily comprises 19 isoenzymes that possess important physiol. and toxicol. functions.  The ALDH1A subfamily plays a pivotal role in embryogenesis and development by mediating retinoic acid signaling.  ALDH2, as a key enzyme that oxidizes acetaldehyde, is crucial for alc. metab.  ALDH1A1 and ALDH3A1 are lens and corneal crystallins, which are essential elements of the cellular defense mechanism against UV radiation-induced damage in ocular tissues.  Many ALDH isoenzymes are important in oxidizing reactive aldehydes derived from lipid peroxidn. and thereby help maintain cellular homeostasis.  Increased expression and activity of ALDH isoenzymes have been reported in various human cancers and are assocd. with cancer relapse.  As a direct consequence of their significant physiol. and toxicol. roles, inhibitors of the ALDH enzymes have been developed to treat human diseases.  This review summarizes known ALDH inhibitors, their mechanisms of action, isoenzyme selectivity, potency, and clin. uses.  The purpose of this review is to 1) establish the current status of pharmacol. inhibition of the ALDHs, 2) provide a rationale for the continued development of ALDH isoenzyme-selective inhibitors, and 3) identify the challenges and potential therapeutic rewards assocd. with the creation of such agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOl6kdU4MnaLVg90H21EOLACvtfcHk0liY7hDw35_OYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslaktw%253D%253D&md5=1de904ae3778b8715a6ec10e8d0242db</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fpr.111.005538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.111.005538%26sid%3Dliteratum%253Aachs%26aulast%3DKoppaka%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DEllermann%26aufirst%3DM.%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DJuvonen%26aufirst%3DR.%2BO.%26aulast%3DPetersen%26aufirst%3DD.%26aulast%3DDeitrich%26aufirst%3DR.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DAldehyde%2520dehydrogenase%2520inhibitors%253A%2520a%2520comprehensive%2520review%2520of%2520the%2520pharmacology%252C%2520mechanism%2520of%2520action%252C%2520substrate%2520specificity%252C%2520and%2520clinical%2520application%26jtitle%3DPharmacol.%2520Rev.%26date%3D2012%26volume%3D64%26spage%3D520%26epage%3D539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Feldman, R. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiner, H.</span><span> </span><span class="NLM_article-title">Horse liver aldehyde dehydrogenase. II. Kinetics and mechanistic implications of the dehydrogenase and esterase activity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">247</span><span class="NLM_x">, </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=1972&pages=267-272&author=R.+I.+Feldmanauthor=H.+Weiner&title=Horse+liver+aldehyde+dehydrogenase.+II.+Kinetics+and+mechanistic+implications+of+the+dehydrogenase+and+esterase+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFeldman%26aufirst%3DR.%2BI.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DHorse%2520liver%2520aldehyde%2520dehydrogenase.%2520II.%2520Kinetics%2520and%2520mechanistic%2520implications%2520of%2520the%2520dehydrogenase%2520and%2520esterase%2520activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1972%26volume%3D247%26spage%3D267%26epage%3D272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Sidhu, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blair, A. H.</span><span> </span><span class="NLM_article-title">Human liver aldehyde dehydrogenase. Esterase activity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1975</span><span class="NLM_x">, </span> <span class="NLM_volume">250</span><span class="NLM_x">, </span> <span class="NLM_fpage">7894</span><span class="NLM_x">–</span> <span class="NLM_lpage">7898</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=250&publication_year=1975&pages=7894-7898&author=R.+S.+Sidhuauthor=A.+H.+Blair&title=Human+liver+aldehyde+dehydrogenase.+Esterase+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSidhu%26aufirst%3DR.%2BS.%26aulast%3DBlair%26aufirst%3DA.%2BH.%26atitle%3DHuman%2520liver%2520aldehyde%2520dehydrogenase.%2520Esterase%2520activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1975%26volume%3D250%26spage%3D7894%26epage%3D7898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Wang, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, S. J.</span><span> </span><span class="NLM_article-title">Substrate specificity of human and yeast aldehyde dehydrogenases</span> <span class="citation_source-journal">Chem. Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">36</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2008.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=18983993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFOju78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2009&pages=36-39&author=M.+F.+Wangauthor=C.+L.+Hanauthor=S.+J.+Yin&title=Substrate+specificity+of+human+and+yeast+aldehyde+dehydrogenases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate specificity of human and yeast aldehyde dehydrogenases</span></div><div class="casAuthors">Wang, Ming-Fang; Han, Chih-Li; Yin, Shih-Jiun</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">36-39</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">The human aldehyde dehydrogenase (ALDH) family may contribute to the metab. of hydrocarbons, biogenic amines, retinoids, steroids, and lipid peroxidn.  The authors previously reported the kinetic properties of human cytosolic ALDH1 and mitochondrial ALDH2 toward the oxidn. of straight-chain and branched-chain aliph. aldehydes with various chain lengths.  Here, the authors present substrate specificities for arom. and heterocyclic aldehydes with purified human liver ALDH1 and ALDH2, and also with yeast mitochondrial ALDH2 for comparison.  The kinetic assay for the human ALDHs was performed in 50 mM HEPES, pH 7.5 and 25°, contg. 0.5 mM NAD, 1.7% acetonitrile (as a solvent carrier for aldehydes), and varied concns. of substrate, and for yeast ALDH2 the assay was detd. in the same reaction mixt. except contg. 3 mM NAD and the addn. of 200 mM KCl.  With respect to phenylacetaldehyde, 2-phenylpropionaldehyde, benzaldehyde, p-nitrobenzaldehyde, cinnamaldehyde, 2-furaldehyde and indole-3-acetaldehyde, human liver ALDH1 exhibited Km values in the range of 0.25-4.8 μM, Vmax values of 0.34-2.4 U/mg, and catalytic efficiency (Vmax/Km) values of 0.070-3.9 U/(mg μM).  Human ALDH2 exhibited Km values in the range of <0.15-0.74 μM, Vmax values of 0.039-0.51 U/mg, and Vmax/Km values of 0.15-1.0 U/(mg μM).  Human ALDH1 and ALDH2 exhibited substate Ki values for phenylacetaldehyde of 95 and 430 μM, resp.  Yeast ALDH2 exhibited Km values for straight-chain aliph. aldehydes (C1-C10) of 2.3-210 μM, and substrate Ki values (C2-C10) of 79-2900 μM, with a trend of having smaller Km and Ki values for longer chain lengths; and Km values for cinnamaldehyde, benzaldehyde, and 2-furaldehyde of 5.0, 79, and 1000 μM, resp.  Therefore, human ALDH1/ALDH2 and yeast ALDH2 can contribute to the detoxification or metab. of various exogenous/endogenous aliph. and arom. aldehydes.  The systematic changes in kinetic parameters for the oxidn. of structurally related aldehydes may reflect subtle functional topog. distinctions of the substrate pocket for human and yeast ALDHs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXbpkm6VyDdLVg90H21EOLACvtfcHk0lgmJ8s2ua12HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFOju78%253D&md5=f82b3d7e3693f4832c02fdcc18c19bc5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2008.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2008.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%2BF.%26aulast%3DHan%26aufirst%3DC.%2BL.%26aulast%3DYin%26aufirst%3DS.%2BJ.%26atitle%3DSubstrate%2520specificity%2520of%2520human%2520and%2520yeast%2520aldehyde%2520dehydrogenases%26jtitle%3DChem.%2520Biol.%2520Interact.%26date%3D2009%26volume%3D178%26spage%3D36%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budas, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churchill, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Disatnik, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochly-Rosen, D.</span><span> </span><span class="NLM_article-title">Activation of aldehyde dehydrogenase-2 reduces ischemic damage to the heart</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">321</span><span class="NLM_x">, </span> <span class="NLM_fpage">1493</span><span class="NLM_x">–</span> <span class="NLM_lpage">1495</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1126%2Fscience.1158554" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=18787169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtV2jsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=1493-1495&author=C.+H.+Chenauthor=G.+R.+Budasauthor=E.+N.+Churchillauthor=M.+H.+Disatnikauthor=T.+D.+Hurleyauthor=D.+Mochly-Rosen&title=Activation+of+aldehyde+dehydrogenase-2+reduces+ischemic+damage+to+the+heart"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Aldehyde Dehydrogenase-2 Reduces Ischemic Damage to the Heart</span></div><div class="casAuthors">Chen, Che-Hong; Budas, Grant R.; Churchill, Eric N.; Disatnik, Marie-Helene; Hurley, Thomas D.; Mochly-Rosen, Daria</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">5895</span>),
    <span class="NLM_cas:pages">1493-1495</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">There is substantial interest in the development of drugs that limit the extent of ischemia-induced cardiac damage caused by myocardial infarction or by certain surgical procedures.  Here, using an unbiased proteomic search, we identified mitochondrial aldehyde dehydrogenase 2 (ALDH2) as an enzyme whose activation correlates with reduced ischemic heart damage in rodent models.  A high-throughput screen yielded a small-mol. activator of ALDH2 (Alda-1) that, when administered to rats before an ischemic event, reduced infarct size by 60%, most likely through its inhibitory effect on the formation of cytotoxic aldehydes.  In vitro, Alda-1 was a particularly effective activator of ALDH2*2, an inactive mutant form of the enzyme that is found in 40% of East Asian populations.  Thus, pharmacol. enhancement of ALDH2 activity may be useful for patients with wild-type or mutant ALDH2 who are subjected to cardiac ischemia, such as during coronary bypass surgery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXwr6e-l58-7Vg90H21EOLACvtfcHk0lgmJ8s2ua12HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtV2jsb7O&md5=b0d4c3a13131a86c2373bf1b138ce8cc</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscience.1158554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1158554%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DBudas%26aufirst%3DG.%2BR.%26aulast%3DChurchill%26aufirst%3DE.%2BN.%26aulast%3DDisatnik%26aufirst%3DM.%2BH.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DMochly-Rosen%26aufirst%3DD.%26atitle%3DActivation%2520of%2520aldehyde%2520dehydrogenase-2%2520reduces%2520ischemic%2520damage%2520to%2520the%2520heart%26jtitle%3DScience%26date%3D2008%26volume%3D321%26spage%3D1493%26epage%3D1495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimble-Hill, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khanna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivanova, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroueh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Discovery of novel regulators of aldehyde dehydrogenase isoenzymes</span> <span class="citation_source-journal">Chem. Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">191</span><span class="NLM_x">, </span> <span class="NLM_fpage">153</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2011.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=21349255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFajtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2011&pages=153-158&author=B.+Parajuliauthor=A.+C.+Kimble-Hillauthor=M.+Khannaauthor=Y.+Ivanovaauthor=S.+Merouehauthor=T.+D.+Hurley&title=Discovery+of+novel+regulators+of+aldehyde+dehydrogenase+isoenzymes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel regulators of aldehyde dehydrogenase isoenzymes</span></div><div class="casAuthors">Parajuli, Bibek; Kimble-Hill, Ann C.; Khanna, May; Ivanova, Yvelina; Meroueh, Samy; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">153-158</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Over the past three years we have been involved in high-throughput screening in an effort to discover novel small mol. modulators of aldehyde dehydrogenase (ALDH) activity.  In particular, we have been interested in both the activation and inhibition of the three commonly studied isoenzymes, ALDH1A1, ALDH2 and ALDH3A1, as their distinct, yet overlapping substrate specificities, present a particularly difficult challenge for inhibitor discovery and design.  Activation of ALDH2 has been shown to benefit cardiovascular outcome following periods of ischemia and renewed interest in specific inhibition of ALDH2 has application for alc. aversion therapy, and more recently, in cocaine addiction.  In contrast, inhibition of either ALDH1A1 or ALDH3A1 has application in cancer treatments where the isoenzymes are commonly over-expressed and serve as markers for cancer stem cells.  We are taking two distinct approaches for these screens: in vitro enzyme activity screens using chem. libraries and virtual computational screens using the structures of the target enzymes as filters for identifying potential inhibitors, followed by in vitro testing of their ability to inhibit their intended targets.  We have identified selective inhibitors of each of these three isoenzymes with inhibition consts. in the high nanomolar to low micromolar range from these screening procedures.  Together, these inhibitors provide proof for concept that selective inhibition of these broad specificity general detoxication enzymes through small mol. discovery and design is possible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcKIxSAscaUbVg90H21EOLACvtfcHk0ljOAiMvlYAemw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFajtbc%253D&md5=f8c284f0a5ad00a61f68cd49e5d4bff1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2011.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2011.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DKimble-Hill%26aufirst%3DA.%2BC.%26aulast%3DKhanna%26aufirst%3DM.%26aulast%3DIvanova%26aufirst%3DY.%26aulast%3DMeroueh%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDiscovery%2520of%2520novel%2520regulators%2520of%2520aldehyde%2520dehydrogenase%2520isoenzymes%26jtitle%3DChem.%2520Biol.%2520Interact.%26date%3D2011%26volume%3D191%26spage%3D153%26epage%3D158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Khanna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimble-Hill, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Miller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baskaran, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dria, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasiliou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochly-Rosen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Discovery of a novel class of covalent inhibitor for aldehyde dehydrogenases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">43486</span><span class="NLM_x">–</span> <span class="NLM_lpage">43494</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=43486-43494&author=M.+Khannaauthor=C.+H.+Chenauthor=A.+Kimble-Hillauthor=B.+Parajuliauthor=S.+Perez-Millerauthor=S.+Baskaranauthor=J.+Kimauthor=K.+Driaauthor=V.+Vasiliouauthor=D.+Mochly-Rosenauthor=T.+D.+Hurley&title=Discovery+of+a+novel+class+of+covalent+inhibitor+for+aldehyde+dehydrogenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKhanna%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DKimble-Hill%26aufirst%3DA.%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DPerez-Miller%26aufirst%3DS.%26aulast%3DBaskaran%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DDria%26aufirst%3DK.%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DMochly-Rosen%26aufirst%3DD.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520class%2520of%2520covalent%2520inhibitor%2520for%2520aldehyde%2520dehydrogenases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D43486%26epage%3D43494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Moreb, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ucar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amory, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldstein, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostmark, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, L. J.</span><span> </span><span class="NLM_article-title">The enzymatic activity of human aldehyde dehydrogenases 1A2 and 2 (ALDH1A2 and ALDH2) is detected by Aldefluor, inhibited by diethylaminobenzaldehyde and has significant effects on cell proliferation and drug resistance</span> <span class="citation_source-journal">Chem. Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">195</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2011.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=22079344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Oqs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2012&pages=52-60&author=J.+S.+Morebauthor=D.+Ucarauthor=S.+Hanauthor=J.+K.+Amoryauthor=A.+S.+Goldsteinauthor=B.+Ostmarkauthor=L.+J.+Chang&title=The+enzymatic+activity+of+human+aldehyde+dehydrogenases+1A2+and+2+%28ALDH1A2+and+ALDH2%29+is+detected+by+Aldefluor%2C+inhibited+by+diethylaminobenzaldehyde+and+has+significant+effects+on+cell+proliferation+and+drug+resistance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The enzymatic activity of human aldehyde dehydrogenases 1A2 and 2 (ALDH1A2 and ALDH2) is detected by Aldefluor, inhibited by diethylaminobenzaldehyde and has significant effects on cell proliferation and drug resistance</span></div><div class="casAuthors">Moreb, Jan S.; Ucar, Deniz; Han, Shuhong; Amory, John K.; Goldstein, Alex S.; Ostmark, Blanca; Chang, Lung-Ji</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-60</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">There has been a new interest in using aldehyde dehydrogenase (ALDH) activity as one marker for stem cells since the Aldefluor flow cytometry-based assay has become available.  Diethylaminobenzaldehyde (DEAB), used in the Aldeflour assay, has been considered a specific inhibitor for ALDH1A1 isoform.  In this study, we explore the effects of human ALDH isoenzymes, ALDH1A2 and ALDH2, on drug resistance and proliferation, and the specificity of DEAB as an inhibitor.  We also screened for the expression of 19 ALDH isoenzymes in K562 cells using TaqMan Low D. Array (TLDA).  We used lentiviral vectors contg. the full cDNA length of either ALDH2 or ALDH1A2 to over express the enzymes in K562 leukemia and H1299 lung cancer cell lines.  Successful expression was measured by activity assay, Western blot, RT-PCR, and Aldefluor assay.  Both cell lines, with either ALDH1A2 or ALDH2, exhibited higher cell proliferation rates, higher clonal efficiency, and increased drug resistance to 4-hydroperoxycyclophosphamide and doxorubicin.  In order to study the specificity of known ALDH activity inhibitors, DEAB and disulfiram, we incubated each cell line with either inhibitor and measured the remaining ALDH enzymic activity.  Both inhibitors reduced ALDH activity of both isoenzymes by 65-90%.  Furthermore, our TLDA results revealed that ALDH1, ALDH7, ALDH3 and ALDH8 are expressed in K562 cells.  We conclude that DEAB is not a specific inhibitor for ALDH1A1 and that Aldefluor assay is not specific for ALDH1A1 activity.  In addn., other ALDH isoenzymes seem to play a major role in the biol. and drug resistance of various malignant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5EFhYLiyR8rVg90H21EOLACvtfcHk0ljOAiMvlYAemw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Oqs7fP&md5=ef48a5eba2e4884fb70e0cde6437f73b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2011.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2011.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DMoreb%26aufirst%3DJ.%2BS.%26aulast%3DUcar%26aufirst%3DD.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DAmory%26aufirst%3DJ.%2BK.%26aulast%3DGoldstein%26aufirst%3DA.%2BS.%26aulast%3DOstmark%26aufirst%3DB.%26aulast%3DChang%26aufirst%3DL.%2BJ.%26atitle%3DThe%2520enzymatic%2520activity%2520of%2520human%2520aldehyde%2520dehydrogenases%25201A2%2520and%25202%2520%2528ALDH1A2%2520and%2520ALDH2%2529%2520is%2520detected%2520by%2520Aldefluor%252C%2520inhibited%2520by%2520diethylaminobenzaldehyde%2520and%2520has%2520significant%2520effects%2520on%2520cell%2520proliferation%2520and%2520drug%2520resistance%26jtitle%3DChem.%2520Biol.%2520Interact.%26date%3D2012%26volume%3D195%26spage%3D52%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Morgan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dria, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title"><i>N</i>,<i>N</i>-Diethylaminobenzaldehyde (DEAB) as a substrate and mechanism-based inhibitor for human ALDH isoenzymes</span> <span class="citation_source-journal">Chem.–Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1016/j.cbi.2014.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2014.12.008" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=C.+A.+Morganauthor=B.+Parajuliauthor=C.+Buchmanauthor=K.+Driaauthor=T.+D.+Hurley&title=N%2CN-Diethylaminobenzaldehyde+%28DEAB%29+as+a+substrate+and+mechanism-based+inhibitor+for+human+ALDH+isoenzymes&doi=10.1016%2Fj.cbi.2014.12.008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DBuchman%26aufirst%3DC.%26aulast%3DDria%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DN%252CN-Diethylaminobenzaldehyde%2520%2528DEAB%2529%2520as%2520a%2520substrate%2520and%2520mechanism-based%2520inhibitor%2520for%2520human%2520ALDH%2520isoenzymes%26jtitle%3DChem.%25E2%2580%2593Biol.%2520Interact.%26date%3D2014%26doi%3D10.1016%2Fj.cbi.2014.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Morgan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Development of a high-throughput in vitro assay to identify selective inhibitors for human ALDH1A1</span> <span class="citation_source-journal">Chem.–Biol. Interact.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1016/j.cbi.2014.10.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2014.10.028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=C.+A.+Morganauthor=T.+D.+Hurley&title=Development+of+a+high-throughput+in+vitro+assay+to+identify+selective+inhibitors+for+human+ALDH1A1&doi=10.1016%2Fj.cbi.2014.10.028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.10.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.10.028%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520a%2520high-throughput%2520in%2520vitro%2520assay%2520to%2520identify%2520selective%2520inhibitors%2520for%2520human%2520ALDH1A1%26jtitle%3DChem.%25E2%2580%2593Biol.%2520Interact.%26date%3D2014%26doi%3D10.1016%2Fj.cbi.2014.10.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Perez-Miller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Younus, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochly-Rosen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Alda-1 is an agonist and chemical chaperone for the common human aldehyde dehydrogenase 2 variant</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">164</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1038%2Fnsmb.1737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=20062057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsVeisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=159-164&author=S.+Perez-Millerauthor=H.+Younusauthor=R.+Vanamauthor=C.+H.+Chenauthor=D.+Mochly-Rosenauthor=T.+D.+Hurley&title=Alda-1+is+an+agonist+and+chemical+chaperone+for+the+common+human+aldehyde+dehydrogenase+2+variant"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Alda-1 is an agonist and chemical chaperone for the common human aldehyde dehydrogenase 2 variant</span></div><div class="casAuthors">Perez-Miller, Samantha; Younus, Hina; Vanam, Ram; Chen, Che-Hong; Mochly-Rosen, Daria; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">159-164</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In approx. one billion people, a point mutation inactivates a key detoxifying enzyme, aldehyde dehydrogenase (ALDH2).  This mitochondrial enzyme metabolizes toxic biogenic and environmental aldehydes, including the endogenously produced 4-hydroxynonenal (4HNE) and the environmental pollutant acrolein, and also bioactivates nitroglycerin.  ALDH2 is best known, however, for its role in ethanol metab.  The accumulation of acetaldehyde following the consumption of even a single alc. beverage leads to the Asian alc.-induced flushing syndrome in ALDH2*2 homozygotes.  The ALDH2*2 allele is semidominant, and heterozygotic individuals show a similar but less severe phenotype.  We recently identified a small mol., Alda-1, that activates wild-type ALDH2 and restores near-wild-type activity to ALDH2*2.  The structures of Alda-1 bound to ALDH2 and ALDH2*2 reveal how Alda-1 activates the wild-type enzyme and how it restores the activity of ALDH2*2 by acting as a structural chaperone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosPZft24xVerVg90H21EOLACvtfcHk0ljaeO1YoPq5oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsVeisg%253D%253D&md5=d7e5592ee6a832b74c5470da9afe7e6b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1737%26sid%3Dliteratum%253Aachs%26aulast%3DPerez-Miller%26aufirst%3DS.%26aulast%3DYounus%26aufirst%3DH.%26aulast%3DVanam%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DMochly-Rosen%26aufirst%3DD.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DAlda-1%2520is%2520an%2520agonist%2520and%2520chemical%2520chaperone%2520for%2520the%2520common%2520human%2520aldehyde%2520dehydrogenase%25202%2520variant%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D159%26epage%3D164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Condello, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagdas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matei, D.</span><span> </span><span class="NLM_article-title">beta-Catenin-regulated ALDH1A1 is a target in ovarian cancer spheroids</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1038/onc.2014.178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1038%2Fonc.2014.178" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=S.+Condelloauthor=C.+A.+Morganauthor=S.+Nagdasauthor=L.+Caoauthor=J.+Turekauthor=T.+D.+Hurleyauthor=D.+Matei&title=beta-Catenin-regulated+ALDH1A1+is+a+target+in+ovarian+cancer+spheroids&doi=10.1038%2Fonc.2014.178"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fonc.2014.178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2014.178%26sid%3Dliteratum%253Aachs%26aulast%3DCondello%26aufirst%3DS.%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DNagdas%26aufirst%3DS.%26aulast%3DCao%26aufirst%3DL.%26aulast%3DTurek%26aufirst%3DJ.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DMatei%26aufirst%3DD.%26atitle%3Dbeta-Catenin-regulated%2520ALDH1A1%2520is%2520a%2520target%2520in%2520ovarian%2520cancer%2520spheroids%26jtitle%3DOncogene%26date%3D2014%26doi%3D10.1038%2Fonc.2014.178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Ni, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiner, H.</span><span> </span><span class="NLM_article-title">Human liver mitochondrial aldehyde dehydrogenase: three-dimensional structure and the restoration of solubility and activity of chimeric forms</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2784</span><span class="NLM_x">–</span> <span class="NLM_lpage">2790</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1110%2Fps.8.12.2784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=10631996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtFyltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1999&pages=2784-2790&author=L.+Niauthor=J.+Zhouauthor=T.+D.+Hurleyauthor=H.+Weiner&title=Human+liver+mitochondrial+aldehyde+dehydrogenase%3A+three-dimensional+structure+and+the+restoration+of+solubility+and+activity+of+chimeric+forms"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Human liver mitochondrial aldehyde dehydrogenase: Three-dimensional structure and the restoration of solubility and activity of chimeric forms</span></div><div class="casAuthors">Ni, Li; Zhou, Jianzhong; Hurley, Thomas D.; Weiner, Henry</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2784-2790</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Human liver cytosolic and mitochondrial isoenzymes of aldehyde dehydrogenase share 70% sequence identity.  However, the first 21 residues are not conserved between the human isoenzymes (15% identity).  The three-dimensional structures of the beef mitochondrial and sheep cytosolic forms have virtually identical three-dimensional structures.  Here, we solved the structure of the human mitochondrial enzyme and found it to be identical to the beef enzyme.  The first 21 residues are found on the surface of the enzyme and make no contact with other subunits in the tetramer.  A pair of chimeric enzymes between the human isoenzymes was made.  Each chimera had the first 21 residues from one isoenzyme and the remaining 479 from the other.  When the first 21 residues were from the mitochondrial isoenzyme, an enzyme with cytosolic-like properties was produced.  The other was expressed but was insol.  It was possible to restore soly. and activity to the chimera that had the first 21 cytosolic residues fused to the balance of the mitochondrial enzyme by making point mutations to residues at the N-terminal end.  When residue 19 was changed from tyrosine to a cysteine, the residue found in the mitochondrial form, an active enzyme could be made, although the Km for NAD+ was 35 times higher than the native mitochondrial isoenzyme and the specific activity was reduced by 75%.  This residue interacts with residue 203, a nonconserved, nonactive site residue.  A mutation of residue 18, which also interacts with 203, restored soly., but not activity.  Mutation to residue 15, which interacts with 104, also restored soly. but not activity.  It appears that to have a sol. or active enzyme, a favorable interaction must occur between a residue in a surface loop and a residue elsewhere in the mol. even though neither make contact with the active site region of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreCoI5nn3RC7Vg90H21EOLACvtfcHk0ljaeO1YoPq5oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtFyltA%253D%253D&md5=71f076c641f5b09dbcbb13818d20a909</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1110%2Fps.8.12.2784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.8.12.2784%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DHuman%2520liver%2520mitochondrial%2520aldehyde%2520dehydrogenase%253A%2520three-dimensional%2520structure%2520and%2520the%2520restoration%2520of%2520solubility%2520and%2520activity%2520of%2520chimeric%2520forms%26jtitle%3DProtein%2520Sci.%26date%3D1999%26volume%3D8%26spage%3D2784%26epage%3D2790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Srivastava, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watowich, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrash, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhatnagar, A.</span><span> </span><span class="NLM_article-title">Structural and kinetic determinants of aldehyde reduction by aldose reductase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">42</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi981794l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=42-54&author=S.+Srivastavaauthor=S.+J.+Watowichauthor=J.+M.+Petrashauthor=S.+K.+Srivastavaauthor=A.+Bhatnagar&title=Structural+and+kinetic+determinants+of+aldehyde+reduction+by+aldose+reductase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fbi981794l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi981794l%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DS.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26aulast%3DPetrash%26aufirst%3DJ.%2BM.%26aulast%3DSrivastava%26aufirst%3DS.%2BK.%26aulast%3DBhatnagar%26aufirst%3DA.%26atitle%3DStructural%2520and%2520kinetic%2520determinants%2520of%2520aldehyde%2520reduction%2520by%2520aldose%2520reductase%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D42%26epage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Moore, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blythe, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, E. N.</span><span> </span><span class="NLM_article-title">Sheep liver cytosolic aldehyde dehydrogenase: the structure reveals the basis for the retinal specificity of class 1 aldehyde dehydrogenases</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1541</span><span class="NLM_x">–</span> <span class="NLM_lpage">1551</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=1541-1551&author=S.+A.+Mooreauthor=H.+M.+Bakerauthor=T.+J.+Blytheauthor=K.+E.+Kitsonauthor=T.+M.+Kitsonauthor=E.+N.+Baker&title=Sheep+liver+cytosolic+aldehyde+dehydrogenase%3A+the+structure+reveals+the+basis+for+the+retinal+specificity+of+class+1+aldehyde+dehydrogenases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DS.%2BA.%26aulast%3DBaker%26aufirst%3DH.%2BM.%26aulast%3DBlythe%26aufirst%3DT.%2BJ.%26aulast%3DKitson%26aufirst%3DK.%2BE.%26aulast%3DKitson%26aufirst%3DT.%2BM.%26aulast%3DBaker%26aufirst%3DE.%2BN.%26atitle%3DSheep%2520liver%2520cytosolic%2520aldehyde%2520dehydrogenase%253A%2520the%2520structure%2520reveals%2520the%2520basis%2520for%2520the%2520retinal%2520specificity%2520of%2520class%25201%2520aldehyde%2520dehydrogenases%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26spage%3D1541%26epage%3D1551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Lamb, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newcomer, M. E.</span><span> </span><span class="NLM_article-title">The structure of retinal dehydrogenase type II at 2.7 A resolution: implications for retinal specificity</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">6003</span><span class="NLM_x">–</span> <span class="NLM_lpage">6011</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi9900471" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=6003-6011&author=A.+L.+Lambauthor=M.+E.+Newcomer&title=The+structure+of+retinal+dehydrogenase+type+II+at+2.7+A+resolution%3A+implications+for+retinal+specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fbi9900471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9900471%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DA.%2BL.%26aulast%3DNewcomer%26aufirst%3DM.%2BE.%26atitle%3DThe%2520structure%2520of%2520retinal%2520dehydrogenase%2520type%2520II%2520at%25202.7%2520A%2520resolution%253A%2520implications%2520for%2520retinal%2520specificity%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D6003%26epage%3D6011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Keung, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vallee, B. L.</span><span> </span><span class="NLM_article-title">Daidzin: a potent, selective inhibitor of human mitochondrial aldehyde dehydrogenase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">1247</span><span class="NLM_x">–</span> <span class="NLM_lpage">1251</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=1247-1251&author=W.+M.+Keungauthor=B.+L.+Vallee&title=Daidzin%3A+a+potent%2C+selective+inhibitor+of+human+mitochondrial+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKeung%26aufirst%3DW.%2BM.%26aulast%3DVallee%26aufirst%3DB.%2BL.%26atitle%3DDaidzin%253A%2520a%2520potent%252C%2520selective%2520inhibitor%2520of%2520human%2520mitochondrial%2520aldehyde%2520dehydrogenase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1993%26volume%3D90%26spage%3D1247%26epage%3D1251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Lowe, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, G. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keung, W. M.</span><span> </span><span class="NLM_article-title">Structure of daidzin, a naturally occurring anti-alcohol-addiction agent, in complex with human mitochondrial aldehyde dehydrogenase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4482</span><span class="NLM_x">–</span> <span class="NLM_lpage">4487</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800488j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4482-4487&author=E.+D.+Loweauthor=G.+Y.+Gaoauthor=L.+N.+Johnsonauthor=W.+M.+Keung&title=Structure+of+daidzin%2C+a+naturally+occurring+anti-alcohol-addiction+agent%2C+in+complex+with+human+mitochondrial+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm800488j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800488j%26sid%3Dliteratum%253Aachs%26aulast%3DLowe%26aufirst%3DE.%2BD.%26aulast%3DGao%26aufirst%3DG.%2BY.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26aulast%3DKeung%26aufirst%3DW.%2BM.%26atitle%3DStructure%2520of%2520daidzin%252C%2520a%2520naturally%2520occurring%2520anti-alcohol-addiction%2520agent%252C%2520in%2520complex%2520with%2520human%2520mitochondrial%2520aldehyde%2520dehydrogenase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4482%26epage%3D4487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Vasiliou, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandoval, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Backos, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reigan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanaspa, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koppaka, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, D. C.</span><span> </span><span class="NLM_article-title">ALDH16A1 is a novel non-catalytic enzyme that may be involved in the etiology of gout via protein–protein interactions with HPRT1</span> <span class="citation_source-journal">Chem.–Biol. Interact</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">202</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2012.12.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=23348497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFamsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=202&publication_year=2013&pages=22-31&author=V.+Vasiliouauthor=M.+Sandovalauthor=D.+S.+Backosauthor=B.+C.+Jacksonauthor=Y.+Chenauthor=P.+Reiganauthor=M.+A.+Lanaspaauthor=R.+J.+Johnsonauthor=V.+Koppakaauthor=D.+C.+Thompson&title=ALDH16A1+is+a+novel+non-catalytic+enzyme+that+may+be+involved+in+the+etiology+of+gout+via+protein%E2%80%93protein+interactions+with+HPRT1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH16A1 is a novel non-catalytic enzyme that may be involved in the etiology of gout via protein-protein interactions with HPRT1</span></div><div class="casAuthors">Vasiliou, Vasilis; Sandoval, Monica; Backos, Donald S.; Jackson, Brian C.; Chen, Ying; Reigan, Philip; Lanaspa, Miguel A.; Johnson, Richard J.; Koppaka, Vindhya; Thompson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">202</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">22-31</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Gout, a common form of inflammatory arthritis, is strongly assocd. with elevated uric acid concns. in the blood (hyperuricemia).  A recent study in Icelanders identified a rare missense single nucleotide polymorphism (SNP) in the ALDH16A1 gene, ALDH16A1*2, to be assocd. with gout and serum uric acid levels.  ALDH16A1 is a novel and rather unique member of the ALDH superfamily in relation to its gene and protein structures.  ALDH16 genes are present in fish, amphibians, protista, bacteria but absent from archaea, fungi and plants.  In most mammalian species, two ALDH16A1 spliced variants (ALDH16A1, long form and ALDH16A1v2, short form) have been identified and both are expressed in HepG-2, HK-2 and HK-293 human cell lines.  The ALDH16 proteins contain two ALDH domains (as opposed to one in the other members of the superfamily), four transmembrane and one coiled-coil domains.  The active site of ALDH16 proteins from bacterial, frog and lower animals contain the catalytically important cysteine residue (Cys-302); this residue is absent from the mammalian and fish orthologs.  Mol. modeling predicts that both the short and long forms of human ALDH16A1 protein would lack catalytic activity but may interact with the hypoxanthine-guanine phosphoribosyltransferase (HPRT1) protein, a key enzyme involved in uric acid metab. and gout.  Interestingly, such protein-protein interactions with HPRT1 are predicted to be impaired for the long or short forms of ALDH16A1*2.  These results lead to the intriguing possibility that assocn. between ALDH16A1 and HPRT1 may be required for optimal HPRT activity with disruption of this interaction possibly contributing to the hyperuricemia seen in ALDH16A1*2 carriers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPm2s2vvZfzbVg90H21EOLACvtfcHk0lhiyB3Vxldt-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFamsLs%253D&md5=efe699e4b09fc9256d48e2d6880605e7</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2012.12.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2012.12.018%26sid%3Dliteratum%253Aachs%26aulast%3DVasiliou%26aufirst%3DV.%26aulast%3DSandoval%26aufirst%3DM.%26aulast%3DBackos%26aufirst%3DD.%2BS.%26aulast%3DJackson%26aufirst%3DB.%2BC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DReigan%26aufirst%3DP.%26aulast%3DLanaspa%26aufirst%3DM.%2BA.%26aulast%3DJohnson%26aufirst%3DR.%2BJ.%26aulast%3DKoppaka%26aufirst%3DV.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26atitle%3DALDH16A1%2520is%2520a%2520novel%2520non-catalytic%2520enzyme%2520that%2520may%2520be%2520involved%2520in%2520the%2520etiology%2520of%2520gout%2520via%2520protein%25E2%2580%2593protein%2520interactions%2520with%2520HPRT1%26jtitle%3DChem.%25E2%2580%2593Biol.%2520Interact%26date%3D2013%26volume%3D202%26spage%3D22%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Levi, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yilmaz, O. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duester, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, S. J.</span><span> </span><span class="NLM_article-title">Aldehyde dehydrogenase 1a1 is dispensable for stem cell function in the mouse hematopoietic and nervous systems</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">1670</span><span class="NLM_x">–</span> <span class="NLM_lpage">1680</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=1670-1680&author=B.+P.+Leviauthor=O.+H.+Yilmazauthor=G.+Duesterauthor=S.+J.+Morrison&title=Aldehyde+dehydrogenase+1a1+is+dispensable+for+stem+cell+function+in+the+mouse+hematopoietic+and+nervous+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLevi%26aufirst%3DB.%2BP.%26aulast%3DYilmaz%26aufirst%3DO.%2BH.%26aulast%3DDuester%26aufirst%3DG.%26aulast%3DMorrison%26aufirst%3DS.%2BJ.%26atitle%3DAldehyde%2520dehydrogenase%25201a1%2520is%2520dispensable%2520for%2520stem%2520cell%2520function%2520in%2520the%2520mouse%2520hematopoietic%2520and%2520nervous%2520systems%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26spage%3D1670%26epage%3D1680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Ucar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cogle, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zucali, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostmark, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zori, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreb, J. S.</span><span> </span><span class="NLM_article-title">Aldehyde dehydrogenase activity as a functional marker for lung cancer</span> <span class="citation_source-journal">Chem.–Biol. Interact</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2Fj.cbi.2008.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=18952074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFOju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2009&pages=48-55&author=D.+Ucarauthor=C.+R.+Cogleauthor=J.+R.+Zucaliauthor=B.+Ostmarkauthor=E.+W.+Scottauthor=R.+Zoriauthor=B.+A.+Grayauthor=J.+S.+Moreb&title=Aldehyde+dehydrogenase+activity+as+a+functional+marker+for+lung+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde dehydrogenase activity as a functional marker for lung cancer</span></div><div class="casAuthors">Ucar, Deniz; Cogle, Christopher R.; Zucali, James R.; Ostmark, Blanca; Scott, Edward W.; Zori, Robert; Gray, Brian A.; Moreb, Jan S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">48-55</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenase (ALDH) activity was implicated in multiple biol. and biochem. pathways and was used to identify potential cancer stem cells.  Our main hypothesis is that ALDH activity may be a lung cancer stem cell marker.  Using flow cytometry, we sorted cells with bright (ALDHbr) and dim (ALDHlo) ALDH activity found in H522 lung cancer cell line.  We used in vitro proliferation and colony assays as well as a xenograft animal model to test our hypothesis.  Cytogenetic anal. demonstrated that the ALDHbr cells are indeed a different clone, but when left in normal culture conditions will give rise to ALDHlo cells.  Furthermore, the ALDHbr cells grow slower, have low clonal efficiency, and give rise to morphol. distinct colonies.  The ability to form primary xenografts in NOD/SCID mice by ALDHbr and ALDHlo cells was tested by injecting single cell suspension under the skin in each flank of same animal.  Tumor size was calcd. weekly.  ALDH1A1 and ALDH3A1 immunohistochem. (IHC) was performed on excised tumors.  These tumors were also used to re-establish cell suspension, measure ALDH activity, and re-injection for secondary and tertiary transplants.  The results indicate that both cell types can form tumors but the ones from ALDHbr cells grew much slower in primary recipient mice.  Histol., there was no significant difference in the expression of ALDH in primary tumors originating from ALDHbr or ALDHlo cells.  Secondary and tertiary xenografts originating from ALDHbr grew faster and bigger than those formed by ALDHlo cells.  In conclusion, ALDHbr cells may have some of the traditional features of stem cells in terms of being mostly dormant and slow to divide, but require support of other cells (ALDHlo) to sustain tumor growth.  These observations and the known role of ALDH in drug resistance may have significant therapeutic implications in the treatment of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmiRq2oQb45rVg90H21EOLACvtfcHk0lhiyB3Vxldt-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFOju70%253D&md5=13c50c224f3f14dec7316fe125513c9e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2008.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2008.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DUcar%26aufirst%3DD.%26aulast%3DCogle%26aufirst%3DC.%2BR.%26aulast%3DZucali%26aufirst%3DJ.%2BR.%26aulast%3DOstmark%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DE.%2BW.%26aulast%3DZori%26aufirst%3DR.%26aulast%3DGray%26aufirst%3DB.%2BA.%26aulast%3DMoreb%26aufirst%3DJ.%2BS.%26atitle%3DAldehyde%2520dehydrogenase%2520activity%2520as%2520a%2520functional%2520marker%2520for%2520lung%2520cancer%26jtitle%3DChem.%25E2%2580%2593Biol.%2520Interact%26date%3D2009%26volume%3D178%26spage%3D48%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Landen, C. N.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katre, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steg, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nick, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mejia, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jennings, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gershenson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bast, R. C.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Berestein, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sood, A. K.</span><span> </span><span class="NLM_article-title">Targeting aldehyde dehydrogenase cancer stem cells in ovarian cancer</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3186</span><span class="NLM_x">–</span> <span class="NLM_lpage">3199</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1158%2F1535-7163.MCT-10-0563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=20889728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=3186-3199&author=C.+N.+Landenauthor=B.+Goodmanauthor=A.+A.+Katreauthor=A.+D.+Stegauthor=A.+M.+Nickauthor=R.+L.+Stoneauthor=L.+D.+Millerauthor=P.+V.+Mejiaauthor=N.+B.+Jenningsauthor=D.+M.+Gershensonauthor=R.+C.+Bastauthor=R.+L.+Colemanauthor=G.+Lopez-Beresteinauthor=A.+K.+Sood&title=Targeting+aldehyde+dehydrogenase+cancer+stem+cells+in+ovarian+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Aldehyde Dehydrogenase Cancer Stem Cells in Ovarian Cancer</span></div><div class="casAuthors">Landen, Charles N., Jr.; Goodman, Blake; Katre, Ashwini A.; Steg, Adam D.; Nick, Alpa M.; Stone, Rebecca L.; Miller, Lance D.; Mejia, Pablo Vivas; Jennings, Nicolas B.; Gershenson, David M.; Bast, Robert C., Jr.; Coleman, Robert L.; Lopez-Berestein, Gabriel; Sood, Anil K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3186-3199</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenase-1A1 (ALDH1A1) expression characterizes a subpopulation of cells with tumor-initiating or cancer stem cell properties in several malignancies.  Our goal was to characterize the phenotype of ALDH1A1-pos. ovarian cancer cells and examine the biol. effects of ALDH1A1 gene silencing.  In our anal. of multiple ovarian cancer cell lines, we found that ALDH1A1 expression and activity was significantly higher in taxane- and platinum-resistant cell lines.  In patient samples, 72.9% of ovarian cancers had ALDH1A1 expression in which the percentage of ALDH1A1-pos. cells correlated neg. with progression-free survival (6.05 vs. 13.81 mo; P < 0.035).  Subpopulations of A2780cp20 cells with ALDH1A1 activity were isolated for orthotopic tumor-initiating studies, where tumorigenicity was approx. 50-fold higher with ALDH1A1-pos. cells.  Interestingly, tumors derived from ALDH1A1-pos. cells gave rise to both ALDH1A1-pos. and ALDH1A1-neg. populations, but ALDH1A1-neg. cells could not generate ALDH1A1-pos. cells.  In an in vivo orthotopic mouse model of ovarian cancer, ALDH1A1 silencing using nanoliposomal siRNA sensitized both taxane- and platinum-resistant cell lines to chemotherapy, significantly reducing tumor growth in mice compared with chemotherapy alone (a 74%-90% redn.; P < 0.015).  These data show that the ALDH1A1 subpopulation is assocd. with chemoresistance and outcome in ovarian cancer patients, and targeting ALDH1A1 sensitizes resistant cells to chemotherapy.  ALDH1A1-pos. cells have enhanced, but not abs., tumorigenicity but do have differentiation capacity lacking in ALDH1A1-neg. cells.  This enzyme may be important for identification and targeting of chemoresistant cell populations in ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeKpHNJYkfVLVg90H21EOLACvtfcHk0liINHAnhBEXPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFGrsL%252FP&md5=2ddb105fb819e82928e7c95994ee317e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-10-0563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-10-0563%26sid%3Dliteratum%253Aachs%26aulast%3DLanden%26aufirst%3DC.%2BN.%26aulast%3DGoodman%26aufirst%3DB.%26aulast%3DKatre%26aufirst%3DA.%2BA.%26aulast%3DSteg%26aufirst%3DA.%2BD.%26aulast%3DNick%26aufirst%3DA.%2BM.%26aulast%3DStone%26aufirst%3DR.%2BL.%26aulast%3DMiller%26aufirst%3DL.%2BD.%26aulast%3DMejia%26aufirst%3DP.%2BV.%26aulast%3DJennings%26aufirst%3DN.%2BB.%26aulast%3DGershenson%26aufirst%3DD.%2BM.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DColeman%26aufirst%3DR.%2BL.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DSood%26aufirst%3DA.%2BK.%26atitle%3DTargeting%2520aldehyde%2520dehydrogenase%2520cancer%2520stem%2520cells%2520in%2520ovarian%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D3186%26epage%3D3199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mei, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stass, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, F.</span><span> </span><span class="NLM_article-title">ALDH1A1 is a marker for malignant prostate stem cells and predictor of prostate cancer patients’ outcome</span> <span class="citation_source-journal">Lab. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">234</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2010&pages=234-244&author=T.+Liauthor=Y.+Suauthor=Y.+Meiauthor=Q.+Lengauthor=B.+Lengauthor=Z.+Liuauthor=S.+A.+Stassauthor=F.+Jiang&title=ALDH1A1+is+a+marker+for+malignant+prostate+stem+cells+and+predictor+of+prostate+cancer+patients%E2%80%99+outcome"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DT.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DLeng%26aufirst%3DQ.%26aulast%3DLeng%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DStass%26aufirst%3DS.%2BA.%26aulast%3DJiang%26aufirst%3DF.%26atitle%3DALDH1A1%2520is%2520a%2520marker%2520for%2520malignant%2520prostate%2520stem%2520cells%2520and%2520predictor%2520of%2520prostate%2520cancer%2520patients%25E2%2580%2599%2520outcome%26jtitle%3DLab.%2520Invest.%26date%3D2010%26volume%3D90%26spage%3D234%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Sodek, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringuette, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, T. J.</span><span> </span><span class="NLM_article-title">Compact spheroid formation by ovarian cancer cells is associated with contractile behavior and an invasive phenotype</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">2060</span><span class="NLM_x">–</span> <span class="NLM_lpage">2070</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1002%2Fijc.24188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=19132753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktVWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2009&pages=2060-2070&author=K.+L.+Sodekauthor=M.+J.+Ringuetteauthor=T.+J.+Brown&title=Compact+spheroid+formation+by+ovarian+cancer+cells+is+associated+with+contractile+behavior+and+an+invasive+phenotype"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Compact spheroid formation by ovarian cancer cells is associated with contractile behavior and an invasive phenotype</span></div><div class="casAuthors">Sodek, Katharine L.; Ringuette, Maurice J.; Brown, Theodore J.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2060-2070</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Ovarian cancer cells are present in malignant ascites both as individual cells and as multicellular spheroid aggregates.  Although spheroid formation affords protection of cancer cells against some chemotherapeutic agents, it has not been established whether a relationship exists between invasive behavior and predisposition to spheroid formation.  Aspects of spheroid formation, including cell-matrix adhesion, remodeling and contractility are characteristic myofibroblast-like behaviors assocd. with fibrosis that contribute to tumor growth and dissemination.  We explored the possibility that cell behaviors that promote spheroid formation also facilitate invasion.  Our anal. of 6 human ovarian cancer cell lines indicated that ovarian cancer cells possessing myofibroblast-like properties formed compact spheroids and invaded 3D matrixes.  These cells readily contracted collagen I gels, possessed a spindle-like morphol., and had elevated expression of genes assocd. with the TGFβ-mediated fibrotic response and/or β1 integrin function, including fibronectin (FN), connective tissue growth factor (CTGF/CCN2), lysyl oxidase (LOX1), tissue transglutaminase 2 (TGM2) and urinary plasminogen activator receptor (uPAR).  Whereas cell aggregation was induced by TGFβ, and by β1-integrin overexpression and activation, these treatments did not stimulate the contractile activity required for spheroid compaction.  The pos. relationship found between compact spheroid formation and invasive behavior implies a preferential survival of an invasive subpopulation of ovarian cancer cells, as cells in spheroids are more resistant to several chemotherapeutics.  Preventing the formation of ovarian cancer spheroids may represent a novel strategy to improve the efficacy of existing therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcsLZtclB7u7Vg90H21EOLACvtfcHk0liINHAnhBEXPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktVWntbc%253D&md5=a6d07729f906ffc92e2892a81df196f6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fijc.24188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.24188%26sid%3Dliteratum%253Aachs%26aulast%3DSodek%26aufirst%3DK.%2BL.%26aulast%3DRinguette%26aufirst%3DM.%2BJ.%26aulast%3DBrown%26aufirst%3DT.%2BJ.%26atitle%3DCompact%2520spheroid%2520formation%2520by%2520ovarian%2520cancer%2520cells%2520is%2520associated%2520with%2520contractile%2520behavior%2520and%2520an%2520invasive%2520phenotype%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2009%26volume%3D124%26spage%3D2060%26epage%3D2070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fishel, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Selective ALDH3A1 inhibition by benzimidazole analogues increase mafosfamide sensitivity in cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">449</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm401508p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=449-461&author=B.+Parajuliauthor=M.+L.+Fishelauthor=T.+D.+Hurley&title=Selective+ALDH3A1+inhibition+by+benzimidazole+analogues+increase+mafosfamide+sensitivity+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm401508p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401508p%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DSelective%2520ALDH3A1%2520inhibition%2520by%2520benzimidazole%2520analogues%2520increase%2520mafosfamide%2520sensitivity%2520in%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D449%26epage%3D461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Parajuli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiadis, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fishel, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span> </span><span class="NLM_article-title">Development of selective inhibitors for human aldehyde dehydrogenase 3A1 (ALDH3A1) for the enhancement of cyclophosphamide cytotoxicity</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=701-712&author=B.+Parajuliauthor=T.+M.+Georgiadisauthor=M.+L.+Fishelauthor=T.+D.+Hurley&title=Development+of+selective+inhibitors+for+human+aldehyde+dehydrogenase+3A1+%28ALDH3A1%29+for+the+enhancement+of+cyclophosphamide+cytotoxicity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParajuli%26aufirst%3DB.%26aulast%3DGeorgiadis%26aufirst%3DT.%2BM.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520selective%2520inhibitors%2520for%2520human%2520aldehyde%2520dehydrogenase%25203A1%2520%2528ALDH3A1%2529%2520for%2520the%2520enhancement%2520of%2520cyclophosphamide%2520cytotoxicity%26jtitle%3DChemBioChem%26date%3D2014%26volume%3D15%26spage%3D701%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Hammen, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hallenga, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiner, H.</span><span> </span><span class="NLM_article-title">Multiple conformations of NAD and NADH when bound to human cytosolic and mitochondrial aldehyde dehydrogenase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">7156</span><span class="NLM_x">–</span> <span class="NLM_lpage">7168</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi012197t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=7156-7168&author=P.+K.+Hammenauthor=A.+Allali-Hassaniauthor=K.+Hallengaauthor=T.+D.+Hurleyauthor=H.+Weiner&title=Multiple+conformations+of+NAD+and+NADH+when+bound+to+human+cytosolic+and+mitochondrial+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fbi012197t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi012197t%26sid%3Dliteratum%253Aachs%26aulast%3DHammen%26aufirst%3DP.%2BK.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHallenga%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DMultiple%2520conformations%2520of%2520NAD%2520and%2520NADH%2520when%2520bound%2520to%2520human%2520cytosolic%2520and%2520mitochondrial%2520aldehyde%2520dehydrogenase%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D7156%26epage%3D7168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Zheng, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiner, H.</span><span> </span><span class="NLM_article-title">Cloning and expression of the full-length cDNAS encoding human liver class 1 and class 2 aldehyde dehydrogenase</span> <span class="citation_source-journal">Alcohol.: Clin. Exp. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">828</span><span class="NLM_x">–</span> <span class="NLM_lpage">831</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1111%2Fj.1530-0277.1993.tb00849.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=8214422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK2cXht1Kgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1993&pages=828-831&author=C.+F.+Zhengauthor=T.+T.+Wangauthor=H.+Weiner&title=Cloning+and+expression+of+the+full-length+cDNAS+encoding+human+liver+class+1+and+class+2+aldehyde+dehydrogenase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Cloning and expression of the full-length cDNAs encoding human liver class 1 and class 2 aldehyde dehydrogenase</span></div><div class="casAuthors">Zheng, Chao Feng; Wang, Thomas T. Y.; Weiner, Henry</div><div class="citationInfo"><span class="NLM_cas:title">Alcoholism:  Clinical and Experimental Research</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">828-31</span>CODEN:
                <span class="NLM_cas:coden">ACRSDM</span>;
        ISSN:<span class="NLM_cas:issn">0145-6008</span>.
    </div><div class="casAbstract">The amino acid sequences of both human class 1 and 2 aldehyde dehydrogenase (ALDH) and the sequences of the genes coding for them are known.  Based on this sequence data, primers were designed and full-length cDNAs isolated encoding both isoenzymes from a human liver mRNA pool.  The cDNAs were subcloned in the plasmid pT7-7 and expressed in Escherichia coli with a yield of ∼3 mg ALDH protein/L of cell culture, although only one-third of the enzyme was sol.  The sol. recombinantly expressed ALDHs were purified to homogeneity using a hydroxyacetophenone-Sepharose affinity column.  The mitochondrial isoenzyme had a subunit mol. wt. of 55 kDa, an isoelec. point of 4.9, and a specific activity of 1.10 units/mg, which were in good agreement with that from the native enzyme.  The expressed cytosolic isoenzyme had the same subunit mol. wt. (55 kDa) and pI (5.4) as that reported for the native enzyme and had a specific activity of 0.26 units/mg.  The expressed mitochondrial isoenzyme could be recognized by antibodies raised against rat mitochondrial ALDH, whereas the cytosolic isoenzyme could be recognized by antibody raised against horse cytosolic ALDH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGhJDXkXENbrVg90H21EOLACvtfcHk0ljFljPEVAPvPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXht1Kgtr8%253D&md5=009015297d0e380d0c981bc657666614</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1111%2Fj.1530-0277.1993.tb00849.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1530-0277.1993.tb00849.x%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DC.%2BF.%26aulast%3DWang%26aufirst%3DT.%2BT.%26aulast%3DWeiner%26aufirst%3DH.%26atitle%3DCloning%2520and%2520expression%2520of%2520the%2520full-length%2520cDNAS%2520encoding%2520human%2520liver%2520class%25201%2520and%2520class%25202%2520aldehyde%2520dehydrogenase%26jtitle%3DAlcohol.%253A%2520Clin.%2520Exp.%2520Res.%26date%3D1993%26volume%3D17%26spage%3D828%26epage%3D831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Macromol. Crystallogr., Part A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0ljFljPEVAPvPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMacromol.%2520Crystallogr.%252C%2520Part%2520A%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>.  <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=15299374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A280%3ADC%252BD2czpsV2gug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+CCP4+suite%3A+programs+for+protein+crystallography.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+1994%2C+50%2C+760%E2%80%93763."target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">The CCP4 suite: programs for protein crystallography</span></div><div class="casAuthors">Anonymous</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">760-3</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">The CCP4 (Collaborative Computational Project, number 4) program suite is a collection of programs and associated data and subroutine libraries which can be used for macromolecular structure determination by X-ray crystallography.  The suite is designed to be flexible, allowing users a number of methods of achieving their aims and so there may be more than one program to cover each function.  The programs are written mainly in standard Fortran77.  They are from a wide variety of sources but are connected by standard data file formats.  The package has been ported to all the major platforms under both Unix and VMS.  The suite is distributed by anonymous ftp from Daresbury Laboratory and is widely used throughout the world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ1Q-skJYIZ2IYPsCkJIw00fW6udTcc2eatsdzqkKbngbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czpsV2gug%253D%253D&md5=92b759927d6071ea3f20c08b78ad1eb2</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljvA-BwVhXV1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Painter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merritt, E. A.</span><span> </span><span class="NLM_article-title">A molecular viewer for the analysis of TLS rigid-body motion in macromolecules</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">471</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=465-471&author=J.+Painterauthor=E.+A.+Merritt&title=A+molecular+viewer+for+the+analysis+of+TLS+rigid-body+motion+in+macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DA%2520molecular%2520viewer%2520for%2520the%2520analysis%2520of%2520TLS%2520rigid-body%2520motion%2520in%2520macromolecules%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D465%26epage%3D471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Painter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merritt, E. A.</span><span> </span><span class="NLM_article-title">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Fjm501900s&amp;key=10.1107%2FS0907444906005270" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Fjm501900s&amp;key=16552146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjm501900s&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=439-450&author=J.+Painterauthor=E.+A.+Merritt&title=Optimal+description+of+a+protein+structure+in+terms+of+multiple+groups+undergoing+TLS+motion"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal description of a protein structure in terms of multiple groups undergoing TLS motion</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-450</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A single protein crystal structure contains information about dynamic properties of the protein as well as providing a static view of one three-dimensional conformation.  This addnl. information is to be found in the distribution of obsd. electron d. about the mean position of each atom.  It is general practice to account for this by refining a sep. at. displacement parameter (ADP) for each at. center.  However, these same displacements are often described well by simpler models based on TLS (translation/libration/screw) rigid-body motion of large groups of atoms, for example interdomain hinge motion.  A procedure, TLSMD, has been developed that analyzes the distribution of ADPs in a previously refined protein crystal structure in order to generate optimal multi-group TLS descriptions of the constituent protein chains.  TLSMD is applicable to crystal structures at any resoln.  The models generated by TLSMD anal. can significantly improve the std. crystallog. residuals R and Rfree and can reveal intrinsic dynamic properties of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIR_NcneJBNbVg90H21EOLACvtfcHk0ljvA-BwVhXV1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1Gkt74%253D&md5=6289483d19fb6dd64fe9b032e9c45005</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0907444906005270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906005270%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DOptimal%2520description%2520of%2520a%2520protein%2520structure%2520in%2520terms%2520of%2520multiple%2520groups%2520undergoing%2520TLS%2520motion%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D439%26epage%3D450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CW3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CW3','PDB','1CW3'); return false;">PDB: 1CW3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SZA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SZA','PDB','3SZA'); return false;">PDB: 3SZA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V9G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V9G','PDB','3V9G'); return false;">PDB: 3V9G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BXS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BXS','PDB','1BXS'); return false;">PDB: 1BXS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BI9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BI9','PDB','1BI9'); return false;">PDB: 1BI9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WP7','PDB','4WP7'); return false;">PDB: 4WP7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WPN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WPN','PDB','4WPN'); return false;">PDB: 4WPN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X4L','PDB','4X4L'); return false;">PDB: 4X4L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VLE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VLE','PDB','2VLE'); return false;">PDB: 2VLE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WJ9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WJ9','PDB','4WJ9'); return false;">PDB: 4WJ9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=ICW3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=ICW3','PDB','ICW3'); return false;">PDB: ICW3</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm501900s&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm501900s%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-4" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm501900s" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d36698383c70","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
