
<html lang="en"     class="pb-page"  data-request-id="5a123dd5-f289-4090-a80d-a95485e2a33e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2017.60.issue-12;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.6b01796;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction" /></meta><meta name="dc.Creator" content="Hacer  Karatas" /></meta><meta name="dc.Creator" content="Yangbing  Li" /></meta><meta name="dc.Creator" content="Liu  Liu" /></meta><meta name="dc.Creator" content="Jiao  Ji" /></meta><meta name="dc.Creator" content="Shirley  Lee" /></meta><meta name="dc.Creator" content="Yong  Chen" /></meta><meta name="dc.Creator" content="Jiuling  Yang" /></meta><meta name="dc.Creator" content="Liyue  Huang" /></meta><meta name="dc.Creator" content="Denzil  Bernard" /></meta><meta name="dc.Creator" content="Jing  Xu" /></meta><meta name="dc.Creator" content="Elizabeth C.  Townsend" /></meta><meta name="dc.Creator" content="Fang  Cao" /></meta><meta name="dc.Creator" content="Xu  Ran" /></meta><meta name="dc.Creator" content="Xiaoqin  Li" /></meta><meta name="dc.Creator" content="Bo  Wen" /></meta><meta name="dc.Creator" content="Duxin  Sun" /></meta><meta name="dc.Creator" content="Jeanne A  Stuckey" /></meta><meta name="dc.Creator" content="Ming  Lei" /></meta><meta name="dc.Creator" content="Yali  Dou" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Description" content="We report herein the design, synthesis, and evaluation of macrocyclic peptidomimetics that bind to WD repeat domain 5 (WDR5) and block the WDR5–mixed lineage leukemia (MLL) protein–protein interact..." /></meta><meta name="Description" content="We report herein the design, synthesis, and evaluation of macrocyclic peptidomimetics that bind to WD repeat domain 5 (WDR5) and block the WDR5–mixed lineage leukemia (MLL) protein–protein interact..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 12, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01796" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01796" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01796" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01796" /></link>
        
    
    

<title>Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01796" /></meta><meta property="og:title" content="Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0001.jpeg" /></meta><meta property="og:description" content="We report herein the design, synthesis, and evaluation of macrocyclic peptidomimetics that bind to WD repeat domain 5 (WDR5) and block the WDR5–mixed lineage leukemia (MLL) protein–protein interaction. Compound 18 (MM-589) binds to WDR5 with an IC50 value of 0.90 nM (Ki value &lt;1 nM) and inhibits the MLL H3K4 methyltransferase (HMT) activity with an IC50 value of 12.7 nM. Compound 18 potently and selectively inhibits cell growth in human leukemia cell lines harboring MLL translocations and is &gt;40 times better than the previously reported compound MM-401. Cocrystal structures of 16 and 18 complexed with WDR5 provide structural basis for their high affinity binding to WDR5. Additionally, we have developed and optimized a new AlphaLISA-based MLL HMT functional assay to facilitate the functional evaluation of these designed compounds. Compound 18 represents the most potent inhibitor of the WDR5–MLL interaction reported to date, and further optimization of 18 may yield a new therapy for acute leukemia." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01796"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01796">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01796&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01796&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01796&amp;href=/doi/10.1021/acs.jmedchem.6b01796" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 4818-4839</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01336" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00008" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hacer++Karatas">Hacer Karatas</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol "><sup>Δ</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yangbing++Li">Yangbing Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol "><sup>Δ</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Liu++Liu">Liu Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol "><sup>Δ</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiao++Ji">Jiao Ji</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol "><sup>Δ</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shirley++Lee">Shirley Lee</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol "><sup>Δ</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Chen">Yong Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiuling++Yang">Jiuling Yang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Liyue++Huang">Liyue Huang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Denzil++Bernard">Denzil Bernard</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Xu">Jing Xu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth+C.++Townsend">Elizabeth C. Townsend</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fang++Cao">Fang Cao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xu++Ran">Xu Ran</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoqin++Li">Xiaoqin Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bo++Wen">Bo Wen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Duxin++Sun">Duxin Sun</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeanne+A++Stuckey">Jeanne A Stuckey</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>¶</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ming++Lei">Ming Lei</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yali++Dou">Yali Dou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>○</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Department of Medicinal Chemistry, <sup>‡</sup>Department of Internal Medicine, <sup>§</sup>Comprehensive Cancer Center, <sup>∥</sup>Department of Pathology, <sup>⊥</sup>Howard Hughes Medical Institute, <sup>#</sup>Department of Biological Chemistry, <sup>∇</sup>Department of Pharmaceutical Sciences, <sup>○</sup>Department of Pharmacology, and <sup>¶</sup>Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div><div class="corresp-info"><strong>*</strong>Tel: 1-734-615-0362. Fax: 1-734-647-9647. E-mail: <a href="/cdn-cgi/l/email-protection#f3809b929c9e969d94b3869e9a909bdd969786"><span class="__cf_email__" data-cfemail="7d0e151c121018131a3d0810141e1553181908">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01796&amp;href=/doi/10.1021%2Facs.jmedchem.6b01796" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 4818–4839</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 12, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 December 2016</li><li><span class="item_label"><b>Published</b> online</span>12 June 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 June 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01796" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01796</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4818%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DHacer%2BKaratas%252C%2BYangbing%2BLi%252C%2BLiu%2BLiu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D12%26contentID%3Dacs.jmedchem.6b01796%26title%3DDiscovery%2Bof%2Ba%2BHighly%2BPotent%252C%2BCell-Permeable%2BMacrocyclic%2BPeptidomimetic%2B%2528MM-589%2529%2BTargeting%2Bthe%2BWD%2BRepeat%2BDomain%2B5%2BProtein%2B%2528WDR5%2529%25E2%2580%2593Mixed%2BLineage%2BLeukemia%2B%2528MLL%2529%2BProtein%25E2%2580%2593Protein%2BInteraction%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4839%26publicationDate%3DJune%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01796"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3316</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">38</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01796" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Hacer&quot;,&quot;last_name&quot;:&quot;Karatas&quot;},{&quot;first_name&quot;:&quot;Yangbing&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Liu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jiao&quot;,&quot;last_name&quot;:&quot;Ji&quot;},{&quot;first_name&quot;:&quot;Shirley&quot;,&quot;last_name&quot;:&quot;Lee&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jiuling&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Liyue&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Denzil&quot;,&quot;last_name&quot;:&quot;Bernard&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;C. Townsend&quot;},{&quot;first_name&quot;:&quot;Fang&quot;,&quot;last_name&quot;:&quot;Cao&quot;},{&quot;first_name&quot;:&quot;Xu&quot;,&quot;last_name&quot;:&quot;Ran&quot;},{&quot;first_name&quot;:&quot;Xiaoqin&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Bo&quot;,&quot;last_name&quot;:&quot;Wen&quot;},{&quot;first_name&quot;:&quot;Duxin&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Jeanne&quot;,&quot;last_name&quot;:&quot;A Stuckey&quot;},{&quot;first_name&quot;:&quot;Ming&quot;,&quot;last_name&quot;:&quot;Lei&quot;},{&quot;first_name&quot;:&quot;Yali&quot;,&quot;last_name&quot;:&quot;Dou&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;4818-4839&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01796&quot;},&quot;abstract&quot;:&quot;We report herein the design, synthesis, and evaluation of macrocyclic peptidomimetics that bind to WD repeat domain 5 (WDR5) and block the WDR5–mixed lineage leukemia (MLL) protein–protein interaction. Compound 18 (MM-589) binds to WDR5 with an IC50 value of 0.90 nM (Ki value &lt;1 nM) and inhibits the MLL H3K4 methyltransferase (HMT) activity with an IC50 value of 12.7 nM. Compound 18 potently and selectively inhibits cell growth in human leukemia cell lines harboring MLL translocations and is &gt;40 times better than the previously reported compound MM-401. Cocrystal structures of 16 and 18 complexed with WDR5 provide structural basis for their high affinity binding to WDR5. Additionally, we have developed and optimized a new AlphaLISA-based MLL HMT functional assay to facilitate the functional evaluation of these designed compounds. Compound 18 represents the most potent inhibitor of the WDR5–MLL interaction reported to date, and further optimization of 18 may yield a new therapy for acute leukemia.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01796&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01796" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01796&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01796" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01796&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01796" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01796&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01796&amp;href=/doi/10.1021/acs.jmedchem.6b01796" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01796" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01796" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01796%26sid%3Dliteratum%253Aachs%26pmid%3D28603984%26genre%3Darticle%26aulast%3DKaratas%26date%3D2017%26atitle%3DDiscovery%2Bof%2Ba%2BHighly%2BPotent%252C%2BCell-Permeable%2BMacrocyclic%2BPeptidomimetic%2B%2528MM-589%2529%2BTargeting%2Bthe%2BWD%2BRepeat%2BDomain%2B5%2BProtein%2B%2528WDR5%2529%25E2%2580%2593Mixed%2BLineage%2BLeukemia%2B%2528MLL%2529%2BProtein%25E2%2580%2593Protein%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D12%26spage%3D4818%26epage%3D4839%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290914" title="Peptidomimetics">Peptidomimetics</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291774" title="Nuclear magnetic resonance spectroscopy">Nuclear magnetic resonance spectroscopy</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/jmcmar.2017.60.issue-12/20170622/jmcmar.2017.60.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We report herein the design, synthesis, and evaluation of macrocyclic peptidomimetics that bind to WD repeat domain 5 (WDR5) and block the WDR5–mixed lineage leukemia (MLL) protein–protein interaction. Compound <b>18</b> (MM-589) binds to WDR5 with an IC<sub>50</sub> value of 0.90 nM (<i>K</i><sub>i</sub> value <1 nM) and inhibits the MLL H3K4 methyltransferase (HMT) activity with an IC<sub>50</sub> value of 12.7 nM. Compound <b>18</b> potently and selectively inhibits cell growth in human leukemia cell lines harboring MLL translocations and is >40 times better than the previously reported compound MM-401. Cocrystal structures of <b>16</b> and <b>18</b> complexed with WDR5 provide structural basis for their high affinity binding to WDR5. Additionally, we have developed and optimized a new AlphaLISA-based MLL HMT functional assay to facilitate the functional evaluation of these designed compounds. Compound <b>18</b> represents the most potent inhibitor of the WDR5–MLL interaction reported to date, and further optimization of <b>18</b> may yield a new therapy for acute leukemia.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50607" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50607" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Mixed lineage leukemia (MLL) protein, also known as MLL1 to distinguish it from other MLL family members (MLL2–MLL4), is a histone H3 lysine 4 (H3K4) methyltransferase. Although MLL has a role in normal hematopoiesis, it is frequently misregulated in a subset of human acute leukemias.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Chromosomal translocations at 11q23 lead to more than 70 MLL fusion proteins, which result in aggressive leukemia with very poor prognosis and treatment outcome.<a onclick="showRef(event, 'ref1 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref3">(1, 3)</a> MLL fusion occurs in nearly 70% of acute lymphoid leukemia (ALL) in infants and 5–10% of acute myeloid leukemia (AML) in adults.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4-6)</a></div><div class="NLM_p">Interestingly, MLL translocations invariably occur on only one MLL allele, and the MLL fusion proteins paradoxically display loss of the H3K4 methyltransferase activity. However, the remaining wild-type MLL allele, which retains the H3K4 methyltransferase activity, has been shown to be required for MLL-AF9-induced leukemogenesis and maintenance of MLL-AF9-transformed cells.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> Therefore, we, and others, have proposed that inhibition of the H3K4 methyltransferase activity of wild-type MLL protein may represent a new therapeutic strategy for the treatment of aggressive leukemia harboring MLL fusion proteins.<a onclick="showRef(event, 'ref10 ref20 ref18'); return false;" href="javascript:void(0);" class="ref ref10 ref20 ref18">(9-11)</a></div><div class="NLM_p">Although MLL protein contains a catalytic SET domain for its H3K4 methyltransferase activity (HMT), the enzymatic activity is very low when MLL protein is present in isolation. However, the enzymatic activity of MLL is dramatically enhanced in a core complex consisting of MLL, WDR5 (WD repeat domain 5 protein), ASH2L (Absent, Small, or Homeotic-2-Like), and RbBP5 (Retinoblastoma Binding Protein 5).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(12)</a> Hence, targeting protein–protein interactions (PPIs) in the MLL core complex could be an effective strategy to inhibit the H3K4 HMT activity of MLL.</div><div class="NLM_p">Among several PPIs within the MLL core complex, the WDR5–MLL interaction is particularly attractive for the design of small-molecule inhibitors to inhibit the MLL HMT activity. First, the WDR5–MLL interaction is essential for the formation of the MLL core complex and for the HMT activity of the MLL complex. Second, the interaction of MLL with WDR5 has been mapped to a 12-residue WIN (WDR5 interacting) motif (residues 3762–3773), which provided a good starting point for the design of small-molecule inhibitors.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(9, 13)</a> Third, the cocrystal structure of WDR5 protein in complex with the 12-residue MLL WIN peptide has shown that MLL binds to a well-defined pocket in WDR5, which appears to be suitable for the design of drug-like, small-molecule inhibitors.</div><div class="NLM_p">Our group has reported the first-in-class inhibitors of WDR5–MLL interaction.<a onclick="showRef(event, 'ref20 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref20 ref18 ref19">(10, 11, 14)</a> In our strategy, we have pursued the design of peptidomimetics to block the WDR5–MLL interaction.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(11, 14)</a> Recently, several nonpeptide, small molecule inhibitors were identified to disrupt WDR5–MLL interaction by high throughput screening on compound libraries and following optimization based on the cocrystal structures of the inhibitors with WDR5 protein.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16">(15-19)</a> For example, OICR-9429<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(16)</a> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) was discovered as a fairly potent inhibitor of WDR5–MLL interaction and was used to explore the mechanism of p30-dependent transformation, leading to the discovery that WDR5 is a therapeutic target in CEBPA mutant AML.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(20)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of three previously reported WDR5–MLL interaction inhibitors including a linear peptidomimetic, a cyclic peptidomimetic, and a nonpeptide inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Previously, we determined that a 3-residue peptide (Ac-ARA-NH<sub>2</sub>) binds to WDR5 with a <i>K</i><sub>i</sub> value of 120 nM, essentially the same as the 12-residue WIN peptide (<i>K</i><sub>i</sub> = 160 nM to WDR5 in the same assay).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(10)</a> Further extensive modifications of the Ac-ARA-NH<sub>2</sub> peptide using both natural and unnatural amino acids have yielded a number of linear peptidomimetics with very high affinities to WDR5 with <b>1</b> (MM-101) being one of the best compounds (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(11)</a> Because these compounds mimic MLL WIN peptide for binding to WDR5, we also refer to them as MLL mimetics (MM).</div><div class="NLM_p last">Compound <b>1</b> binds to WDR5 with <i>K</i><sub>i</sub> value of <1 nM and potently and dose-dependently inhibits the MLL HMT activity in a fully reconstituted <i>in vitro</i> H3K4 methyltransferase assay.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(11)</a> However, linear peptidomimetic <b>1</b> has two major shortcomings: it has very modest cellular potency and poor metabolic stability in human microsomes. To improve cellular potency and microsomal stability, we have performed structure-based design, synthesis, and evaluation of conformationally constrained, macrocyclic peptidomimetics based upon the linear peptidomimetic <b>1</b>. In the previous study,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(14)</a> we have disclosed the design of one such cyclic peptidomimetic (compound <b>2</b> in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) and its extensive characterization for activity, specificity, and mechanism of action. In present study, we describe further design, synthesis, and biochemical and biological testing of a series of macrocyclic peptidomimetics. The present study has yielded a number of cyclic peptidomimetics that bind to WDR5 with very high affinities (<i>K</i><sub>i</sub> ≪ 1 nM). In particular, we have identified a new cyclic peptidomimetic, <b>18</b>, which is >40-times more potent than the reported cyclic peptidomimetic <b>2</b> in inhibition of cell growth in human leukemia cells harboring MLL translocations. Determination of the cocrystal structure of compound <b>18</b> in complex with WDR5 provides further structural insights for its high binding affinity to WDR5.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31163" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31163" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Modification of Cyclic Peptidomimetic <b>2</b></h3><div class="NLM_p">Based upon the cocrystal structure of <b>2</b> in complex with WDR5, we designed and synthesized a series of new compounds to further determine the structure–activity relationships for this class of macrocyclic peptidomimetics. The results are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Chemical Structures, Binding Affinities to WDR5 and MLL HMT Inhibition Activities of Compound <b>2</b> and Its Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0014.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">binding affinity to WDR5</th><th class="rowsep1 colsep0" align="center" char="±">inhibition of MLL HMT activity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">R<sub>3</sub></th><th class="colsep0 rowsep0" align="center">R<sub>4</sub></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> ± SD (nM)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>i</sub> ± SD (nM)</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> ± SD (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">2.9 ± 1.4</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">578 ± 190</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="char" char=".">0.9 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">373 ± 44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="char" char=".">1.6 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">190 ± 33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">-F</td><td class="colsep0 rowsep0" align="left">-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="char" char=".">1.6 ± 1.0</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">138 ± 39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">-Cl</td><td class="colsep0 rowsep0" align="left">-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="char" char=".">1.5 ± 0.9</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">124 ± 28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="char" char=".">1.1 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">452 ± 55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">2.4 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">1259 ± 236</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">>10 000</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="±">>10 000</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Not applicable.</p></div></div></div><div class="NLM_p">The cocrystal structure of <b>2</b> in complex with WDR5 showed that the amino group connecting to the alkyl linker in <b>2</b> is exposed to solvent (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(14)</a> Computational modeling indicated that methylation of this amino group would not distort the bound conformation. Since methylation of the amino group reduces the polar surface, it may improve cellular permeability and hence cellular activity. Accordingly, we have designed and synthesized compound <b>3</b>. Our previous study showed that <i>p</i>-F or <i>p</i>-Cl substitution on the phenyl group in linear peptidomimetics can enhance their cellular potency,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(11)</a> and compounds <b>4</b> and <b>5</b> with <i>p</i>-F or <i>p</i>-Cl substitution, respectively, on the phenyl group of <b>3</b> were therefore synthesized. The positively charged guanidine group in the arginine side chain was shown to be critical for binding of linear peptidomimetics to WDR5.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(11)</a> However, modeling studies suggested that guanidine group can tolerate both monomethylation and dimethylation maintaining strong interactions with WDR5. We, therefore, synthesized compounds <b>6</b> and <b>7</b> with monomethylation and symmetric dimethylation, respectively, on compound <b>2</b>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Cyclic peptidomimetic <b>2</b> in complex with WDR5 [PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM9">4GM9</a>]. Compound <b>2</b> is shown in stick, and WDR5 is shown with surface model. (B) Superposition of the docked model of <b>13</b> (cyan) onto the cocrystal structure of compound <b>2</b> complexed with WDR5. Carbon atoms are shown in yellow for compound <b>2</b> or cyan for compound <b>13</b>, oxygen atoms are shown in red and nitrogen atoms are shown in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Binding Affinities of Cyclic Peptidomimetics to WDR5</h3><div class="NLM_p last">We evaluated the binding affinities of these cyclic peptidomimetics to WDR5 using an optimized, FP-based competitive binding assay.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(10)</a> The binding data showed that cyclic peptidomimetics <b>3</b>–<b>7</b> bind to WDR5 with IC<sub>50</sub> values of 1.1–2.4 nM with estimated <i>K</i><sub>i</sub> values of <1 nM (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Of note, because the affinities of these cyclic peptidomimetics exceed the lower assay limit of the FP-based competitive binding assay, their exact <i>K</i><sub>i</sub> values cannot be accurately determined. Nevertheless, the binding data showed that all these cyclic peptidomimetics have very high binding affinities to WDR5. In comparison, compound <b>8</b>, the enantiomer of <b>2</b>, has no appreciable binding to WDR5 at concentrations as high as 10 μM, indicating that the binding of compound <b>2</b> to WDR5 is stereospecific.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Development and Optimization of a New AlphaLISA based MLL HMT Functional Assay and Inhibition of the MLL HMT Activity by Cyclic Peptidomimetics</h3><div class="NLM_p">Because the WDR5–MLL interaction is required for the MLL complex to achieve robust H3K4 HMT activity, compounds that target this interaction are predicted to effectively inhibit the MLL H3K4 HMT activity. Since our previous MLL HMT functional assay utilizes radioactive <sup>3</sup>H labeled <i>S</i>-adenosyl methionine (SAM), involves multiple wash and transfer steps and is, therefore, quite time-consuming,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(11)</a> we have developed and optimized a new, amplified luminescent proximity homogeneous assay (AlphaLISA) to evaluate the MLL HMT inhibitory activities of the newly designed cyclic peptidomimetics. The assay flow is shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A. In this assay, recombinant nucleosomes were initially treated with the MLL complex for methylation in the presence of <i>S</i>-adenosyl methionine (SAM) as cofactor before the reaction was stopped with high salt buffer. Then, anti-H3K4Me1/2 antibody, which is covalently linked to acceptor beads, and biotinylated anti-H3 (C-terminus) antibody were added. Lastly, streptavidin labeled donor beads were added, and the assay plate was imaged with a microplate reader using excitation wavelength of 680 nm and emission wavelength of 615 nm.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Development and optimization of a homogeneous AlphaLISA MLL functional assay. (A) Illustration of the homogeneous AlphaLISA MLL HMT functional assay. (B, C) Enzymatic activity of MLL complex, represented by the fluorescence intensity in relative fluorescence units (RFU) from acceptor beads detected at 615 nm at different concentrations and different reaction time points with a low (300 nM, B) or a high (3 μM, C) SAM concentration. (D) Representative inhibitory curves of selected WDR5 inhibitors. (E) Comparison of potencies of compounds determined from the competitive WDR5 binding assay and the AlphaLISA MLL functional assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Different from widely used homogeneous time resolved fluorescence (HTRF) assays, in which energy transfer can only take place between donor–acceptor fluorophores located within approximate 10 nm, short-lived singlet oxygen (<sup>1</sup>O<sub>2</sub>) generated from AlphaLISA donor beads can reach the acceptor beads as far as 200 nm away. This extra-long effective range significantly enhances the versatility of Alpha assays, in which multiple labeled antibodies can be utilized. Such feature is also particularly useful for HMT functional assays involving large-size components, such as nucleosomes as substrates, which are 11 nm in size (e.g., mononucleosomes) or much larger (e.g., oligonucleosomes). Histone methylation assays that use nucleosomes as substrates more closely mimic the MLL HMT reaction in cells, compared with those that utilize histone peptide or proteins as substrates. Therefore, a functional AlphaLISA MLL HMT assay using nucleosomes as substrates would be highly valuable to evaluate these WDR5 inhibitors.</div><div class="NLM_p">Although no robust MLL HMT AlphaLISA assay has been reported yet, the acceptor bead conjugated with antibodies that can recognize methylated H3K4 is commercially available and was used here to develop the MLL HMT AlphaLISA assay (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). As indicated by the manufacturer and confirmed in our own experiments, this antibody can only recognize mono- and dimethylated H3K4. Therefore, we optimized assay conditions, described below, and paid particular attention to minimize trimethylation of nucleosomes, which would greatly compromise the interactions of acceptor beads with mono- and dimethylated nucleosomes and thereby significantly reduce fluorescence intensity detected in the assay yielding a low dynamic range.</div><div class="NLM_p">We investigated the influence of reaction time and concentration of the MLL complex on fluorescence intensity using two different SAM concentrations (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B,C). At low SAM concentration (300 nM), AlphaLISA fluorescence signals increased with longer reaction time when MLL complex concentration was 1–5 nM, were steady with 10 nM MLL complex, but decreased as early as 30 min with 20 nM of the MLL complex. Furthermore, when SAM concentration was 10-fold higher (3 μM) with which reaction was even faster, only a slight signal increase was observed when using 5 nM of the MLL complex and significant signal decrease started at 30 min with both 10 and 20 nM of the MLL complex. Consistent with the expectation, AlphaLISA signal decrease was observed with higher MLL complex concentrations, longer reaction time, and even more severely with higher SAM concentrations, which is most likely attributed to generation of more trimethylated nucleosomes.</div><div class="NLM_p">Representative inhibitory curves of selected WDR5 inhibitors using the optimized AlphaLISA MLL HMT functional assay are shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D. This new functional assay was able to discriminate compounds with very high affinities to WDR5 as determined in the FP-based competitive binding assay. We thus compared the IC<sub>50</sub> values obtained from the FP-based competitive binding assay with those obtained from the AlphaLISA functional assay for all the compounds included in the present study (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>E). The data showed that there is a linear correlation between the IC<sub>50</sub> values obtained from WDR5 competitive binding assay (<i>y</i>-axis) and MLL HMT functional assay (<i>x</i>-axis) as shown with blue dashed line. However, compounds with IC<sub>50</sub> values of 1–2 nM in the WDR5 competitive binding assay deviate from this linear correlation and have IC<sub>50</sub> values ranging between 10 and 500 nM in the functional assay. These data indicate that this new MLL HMT functional assay can nicely discriminate highly potent WDR5 inhibitors, which is not possible with the FP-based competitive binding assay. IC<sub>50</sub> values obtained for compounds <b>1</b>–<b>8</b> in this MLL functional assay are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</div><div class="NLM_p last">Functional data showed that linear peptidomimetic <b>1</b> effectively inhibits the MLL HMT activity with an IC<sub>50</sub> value of 578 nM, and cyclic peptidomimetic <b>2</b> has an IC<sub>50</sub> value of 373 nM. New compounds <b>3</b>–<b>5</b> achieve IC<sub>50</sub> values of 124–190 nM in the HMT functional assay and are thus 2–3 times more potent than compound <b>2</b>. Compound <b>6</b> with a monomethylated guanidine group has an IC<sub>50</sub> value of 452 nM, which is slightly less potent than <b>2</b>, and compound <b>7</b> with a dimethylated guanidine group is 3-times less potent than <b>2</b>. Consistent with its weak binding affinity to WDR5, the control compound <b>8</b> fails to inhibit the MLL HMT activity at the highest concentration (10 μM) tested. It should be noted that potencies determined in the MLL MHT assay for all the compounds reported in the present study are the based on their ability to inhibit mono- and dimethylation activities of the MLL complex.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Activity and Selectivity of Cyclic Peptidomimetics To Inhibit Cell Growth of Human Leukemia Cell Lines</h3><div class="NLM_p">We next evaluated representative cyclic peptidomimetics <b>2</b>, <b>4</b>, <b>6</b>, <b>7</b>, and <b>8</b> in three human acute leukemia cell lines for their activity and selectivity in inhibition of cell growth. The data are summarized in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. In the MV4-11 cell line harboring MLL-AF4 fusion protein and the MOLM-13 cell line harboring MLL-AF9 fusion protein, compound <b>2</b> has IC<sub>50</sub> values of 9.8 and 18.4 μM, respectively, similar to those values reported in the previous study.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(14)</a> Compound <b>4</b> has comparable potencies as compound <b>2</b> in inhibition of cell growth in both the MV4-11 and MOLM-13 cell lines. Compound <b>7</b> is slightly less potent than <b>2</b> in both the MV4-11 and MOLM-13 cell lines. However, compound <b>6</b> has IC<sub>50</sub> values of 3.6 (MOLM-13) and 4.3 μM (MV4-11) and is 2–5 times more potent than <b>2</b>. Compound <b>8</b>, which has no appreciable binding to WDR5 and does not inhibit the MLL HMT activity at 10 μM, has IC<sub>50</sub> > 100 μM in both the MV4-11 and MOLM-13 cell lines in the same cell growth inhibition assay, suggesting that cellular activity for the cyclic peptidomimetics in these two leukemia cell lines is specific and on-target.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibition of Cell Growth (IC<sub>50</sub>, μM) with Representative Cyclic Peptidomimetics</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">MOLM-13</th><th class="colsep0 rowsep0" align="center" char=".">MV4-11</th><th class="colsep0 rowsep0" align="center" char=".">HL-60</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">18.4 ± 6.6</td><td class="colsep0 rowsep0" align="char" char=".">9.8 ± 8.0</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">16.2 ± 6.1</td><td class="colsep0 rowsep0" align="char" char=".">5.2 ± 1.1</td><td class="colsep0 rowsep0" align="char" char=".">60.0 ± 19.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">3.6 ± 0.8</td><td class="colsep0 rowsep0" align="char" char=".">4.3 ± 1.5</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">31.5 ± 9.8</td><td class="colsep0 rowsep0" align="char" char=".">15.2 ± 2.9</td><td class="colsep0 rowsep0" align="char" char=".">84.8 ± 13.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr></tbody></table></div></div><div class="NLM_p last">Next, we assessed specificity of cyclic peptidomimetics to inhibit cell growth in the HL-60 leukemia cell line lacking MLL translocation. Compounds <b>2</b>, <b>6</b>, and <b>8</b> have IC<sub>50</sub> values of >100 μM, and <b>4</b> and <b>7</b> have modest activity with IC<sub>50</sub> values of 60.0 and 84.8 μM, respectively. Hence, these cyclic peptidomimetics display cellular selectivity against the MV4-11 and MOLM-13 leukemia cell lines harboring MLL fusion proteins over the HL-60 cell line lacking MLL translocation.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Optimization of the Ring Size in These Cyclic Peptidomimetics</h3><div class="NLM_p">Since the length of the linker in the designed cyclic peptidomimetics plays a key role for their conformations, we have modeled compounds with different linker lengths. Interestingly, modeling suggested that cyclic peptidomimetics with a linker as short as 2-carbon appear to be able to adopt low-energy conformations that closely resemble the bound conformation of compound <b>2</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B).</div><div class="NLM_p">Since cyclic peptidomimetic <b>2</b> has a binding affinity exceeding the lower assay limit in the FP-based competitive binding assay, we sought to use a less potent cyclic peptidomimetic as the template to better understand the effect of the linker length on binding affinity to WDR5. For linear peptidomimetics, we have shown that the methyl group attached to the bridging carbon atom contributes significantly to their affinity to WDR5.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(11)</a> Indeed, removal of this methyl group in <b>2</b> yielded <b>12</b>, which has a <i>K</i><sub>i</sub> value of 60.3 nM, >50-times less potent than <b>2</b>. Compound <b>12</b> was thus used as the template for further investigation of the linker. The results are summarized in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>. Compound <b>9</b> with a one-carbon linker has <i>K</i><sub>i</sub> value of 242 nM to WDR5 and is thus 4-times less potent than <b>12</b> with a 4-carbon linker. However, compound <b>10</b> with a 2-carbon linker has <i>K</i><sub>i</sub> value of 0.8 nM to WDR5 and is >70-times more potent than <b>12</b>. Compound <b>11</b> with a 3-carbon linker has <i>K</i><sub>i</sub> value of 32.9 nM to WDR5 and is 2-times more potent than <b>12</b>. Hence, further investigation of the linker length showed that compound <b>10</b> with a 2-carbon linker achieves the highest binding affinity among this series of analogues.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Chemical Structures, Binding Affinities to WDR5, and MLL HMT Inhibition Activities of Demethylated Analogues of Compound <b>2</b> with Different Linker Sizes</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0015.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">binding affinity to WDR5</th><th class="rowsep1 colsep0" align="center" char=".">inhibition of MLL HMT activity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>n</i></th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> ± SD (nM)</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>i</sub> ± SD (nM)</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> ± SD (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char="±">1190 ± 160</td><td class="colsep0 rowsep0" align="char" char="±">242 ± 33</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char="±">6.2 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">0.8 ± 0.1</td><td class="colsep0 rowsep0" align="char" char=".">1.8 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char="±">164 ± 26</td><td class="colsep0 rowsep0" align="char" char="±">32.9 ± 5.3</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char="±">298 ± 58</td><td class="colsep0 rowsep0" align="char" char="±">60.3 ± 11.9</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr></tbody></table></div></div><div class="NLM_p last">We next tested these compounds (<b>9</b>–<b>12</b>) for their inhibitory activity in the newly established MLL HMT functional assay. Functional data showed that their potencies in inhibition of the MLL HMT activity correlate nicely with their binding affinities to WDR5. Accordingly, compound <b>10</b> has the best potency to inhibit the MLL HMT activity (IC<sub>50</sub> = 1.8 μM), while IC<sub>50</sub> values of other compounds are higher than 10 μM (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). These data further show that the newly established HMT functional assay can clearly distinguish the potency differences even among compounds with very high binding affinities to WDR5.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Design and Synthesis of Compound <b>2</b> Analogues with Different Ring Sizes</h3><div class="NLM_p">Based upon the promising data for compound <b>10</b>, we synthesized and evaluated a series of analogues of compound <b>2</b> with different ring sizes with the results summarized in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Chemical Structures, Binding Affinities to WDR5, and MLL HMT Inhibition Activities of Compound <b>2</b> Analogues with Different Linker Length</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0016.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">binding affinity to WDR5</th><th class="rowsep1 colsep0" align="center" char="±">inhibition of MLL HMT activity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>n</i></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> ± SD (nM)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>i</sub> ± SD (nM)</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> ± SD (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char=".">0.9 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">373 ± 44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">1.08 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">12.6 ± 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="char" char=".">0.97 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">84.8 ± 23.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="char" char=".">2.6 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">1527 ± 218</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="char" char=".">1.0 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">216 ± 88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="char" char=".">1.3 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">744 ± 79</td></tr></tbody></table></div></div><div class="NLM_p">Compounds <b>13</b>, <b>14</b>, <b>2</b>, <b>15</b>, <b>16</b>, and <b>17</b> with linker lengths of 2, 3, 4, 5, 6, and 8 carbons, respectively, all bind to WDR5 with very high affinities (IC<sub>50</sub> = 0.9–2.6 nM, <i>K</i><sub>i</sub> < 1 nM). Since the binding affinities of these compounds exceed the lower assay limit, the potential differences in their binding affinities to WDR5 cannot be discerned. However, the functional assay showed that while all of them can effectively inhibit MLL HMT activity, their potencies are not the same. Among these analogs, compound <b>13</b> has the best inhibitory activity with IC<sub>50</sub> value of 12.6 nM, which is about 30-times more potent than <b>2</b>. Compound <b>14</b> with a 3-carbon linker is the second most potent inhibitor of the MLL HMT activity with an IC<sub>50</sub> value of 84.8 nM.</div><div class="NLM_p">We next evaluated these compounds for their cell growth inhibitory activity in MOLM-13, MV4-11, and HL-60 cell lines (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). Compound <b>10</b> displays modest cellular activities with IC<sub>50</sub> values of 42.8 μM and 43.4 μM against MOLM-13 and MV4-11 cell lines, respectively, and compounds <b>11</b> and <b>12</b> have minimal activity at 100 μM in both cell lines. Compound <b>14</b> shows fairly potent cellular activity with IC<sub>50</sub> values of 1.28 and 1.29 μM in the MOLM-13 and MV4-11 cell lines, respectively. Compound <b>13</b> (MM-581) is the most potent compound in this series and has IC<sub>50</sub> values of 0.28 and 0.46 μM, respectively, in inhibition of cell growth in the MOLM-13 and MV4-11 cell lines. Significantly, compounds <b>13</b> and <b>14</b> have much weaker activity in the HL-60 cell line lacking MLL translocation and display more than 30-fold cellular selectivity.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Inhibition of Cell Growth (IC<sub>50</sub>, μM) with Representative Cyclic Peptidomimetics</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">MOLM-13</th><th class="colsep0 rowsep0" align="center" char=".">MV4-11</th><th class="colsep0 rowsep0" align="center" char=".">HL-60</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">18.4 ± 6.6</td><td class="colsep0 rowsep0" align="char" char=".">9.8 ± 8.0</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">42.8</td><td class="colsep0 rowsep0" align="char" char=".">43.4</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">0.28 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">0.46 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">17.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">1.28 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">1.29</td><td class="colsep0 rowsep0" align="char" char=".">46.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">37.6 ± 12.1</td><td class="colsep0 rowsep0" align="char" char=".">15.1 ± 11.8</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char=".">5.6 ± 0.7</td><td class="colsep0 rowsep0" align="char" char=".">3.2 ± 0.4</td><td class="colsep0 rowsep0" align="char" char=".">29.7 ± 11.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">21.6 ± 4.8</td><td class="colsep0 rowsep0" align="char" char=".">7.5 ± 3.0</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">0.21 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">0.25 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">8.56 ± 1.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">14.3 ± 0.9</td><td class="colsep0 rowsep0" align="char" char=".">35.0 ± 4.4</td><td class="colsep0 rowsep0" align="char" char="."><a class="ref internalNav" href="#tbl5-fn1" aria-label="a">a</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl5-fn1"><div class="footnote" id="tbl5-fn1"><sup>a</sup><p class="last">Not tested.</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Further Modifications of Compound <b>13</b></h3><div class="NLM_p">Monomethylation of the guanidine group in compound <b>2</b>, which yielded compound <b>6</b>, improves cellular potency against MV4-11 and MOLM-13 cell lines (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). We therefore monomethylated the guanidine group in compound <b>13</b>, which yielded compound <b>18</b> (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). Similar to compound <b>13</b>, compound <b>18</b> binds to WDR5 with high affinity (IC<sub>50</sub> = 0.90 nM, <i>K</i><sub>i</sub> value of <1 nM) and potently inhibits the MLL HMT activity (IC<sub>50</sub> = 12.7 nM).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Chemical Structures, Binding Affinities to WDR5, and MLL HMT Inhibition Activities of Compounds <b>13</b>, <b>18</b>, and <b>19</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0017.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">binding affinity to WDR5</th><th class="rowsep1 colsep0" align="center" char="±">inhibition of MLL HMT activity</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> ± SD (nM)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>i</sub> ± SD (nM)</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> ± SD (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">1.08 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">12.6 ± 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">0.90 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">12.7 ± 1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">2.0 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="±">477 ± 51</td></tr></tbody></table></div></div><div class="NLM_p">The specificity of compound <b>18</b> was tested against MLL and other SET1 family members (MLL2, MLL3, MLL4, SET1a and SET1b) using the previously published HMT assays.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(14)</a> The data showed that while <b>18</b> effectively inhibits the MLL HMT activity (IC<sub>50</sub>= 12.7 nM) in our new HMT assay, it has no or a minimal effect up to 100 μM in inhibition of the HMT activity of other SET1 family members (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_001.pdf" class="ext-link">Figure S1</a>).</div><div class="NLM_p">Compound <b>18</b> was next evaluated for its activity and specificity to inhibit cell growth of acute leukemia cancer cell lines (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). Compound <b>18</b> achieves potent cell growth inhibitory activity in both the MV4-11 and MOLM-13 cell lines with IC<sub>50</sub> values of 0.25 and 0.21 μM, respectively. Compound <b>18</b> has much weaker activity in the inhibition of cell growth of the HL-60 cell line with an IC<sub>50</sub> value of 8.6 μM, hence displaying >30-fold selectivity for the MV4-11 and MOLM-13 cell lines harboring MLL translocations over the HL-60 cell line lacking MLL translocation.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A,B) Cell growth inhibition curves of representative cyclic peptidomimetics in MOLM-13 and MV4-11 human leukemia cell lines harboring MLL translocation. (C) Correlation between inhibition potencies of cell growth and MLL HMT activity of representative cyclic peptidomimetics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To test the stereospecificity of <b>18</b>, we synthesized compound <b>19</b> in which the chiral center in the phenylglycine residue is changed from <span class="smallcaps smallerCapital">d</span>- to <span class="smallcaps smallerCapital">l</span>-configuration (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). Compound <b>19</b> still binds to WDR5 with a high affinity (IC<sub>50</sub> = 2.0 nM, <i>K</i><sub>i</sub> < 1 nM) but is 40-times less potent (IC<sub>50</sub> = 477 nM) than compound <b>18</b> (IC<sub>50</sub> = 12.7 nM) in inhibition of the MLL HMT activity (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). Consistently, compound <b>19</b> has IC<sub>50</sub> values of 14.3 and 35.0 μM in inhibition of cell growth in MOLM-13 and MV4-11 cell lines, respectively (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>), and is thus >50–100 times less potent than compound <b>18</b>.</div><div class="NLM_p">We next investigated whether there is a correlation between the potencies of cyclic peptidomimetics in inhibition of the MLL HMT activity and inhibition of cell growth of both MV4-11 and MOLM-13 cell lines. The analysis revealed that there is a fairly good correlation between the IC<sub>50</sub> values in inhibition of the MLL HMT activity (<i>x</i>-axis) and the IC<sub>50</sub> values of cell growth inhibition (<i>y</i>-axis) in both cell lines (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C).</div><div class="NLM_p">Small molecule inhibitors targeting HMT activity, such as inhibitors of the EZH2 H3K27 activity<a onclick="showRef(event, 'ref500'); return false;" href="javascript:void(0);" class="ref ref500">(21)</a> and the DOT1L H3K79 activity,<a onclick="showRef(event, 'ref600'); return false;" href="javascript:void(0);" class="ref ref600">(22)</a> were shown to have very slow kinetics in inhibition of cell growth. We therefore evaluated compound <b>18</b> for its inhibitory activity of cell growth in MV4-11 and MOLM-13 leukemia cell lines with either 4- or 7-day treatment. Although the IC<sub>50</sub> values for <b>18</b> improved 2–3 times from 4-day treatment to 7-day treatment in both cell lines, the maximum inhibition has changed significantly, from partial inhibition to essentially complete inhibition (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). While approximately 75% maximum growth inhibition was achieved with <b>18</b> upon 4-day treatment in MV4-11 and MOLM-13 cell lines, > 95% maximum inhibition was achieved upon 7-day treatment in both cell lines. Of note, in this experiment, we seeded 100 000 cells in each well in 24-well plates, whereas the cell growth inhibition data obtained for compound <b>2</b> in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a> was obtained by seeding 10 000 cells in 96-well plates. Additionally, we have tested the stability of compound <b>18</b> in cell culture media up to 7 days and found that the compound is stable (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_001.pdf" class="ext-link">Figure S3</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cell growth inhibition of compound <b>18</b> upon 4-day or 7-day treatment time in MV4-11 and MOLM-13 acute leukemia cell lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Determination of Cocrystal Structures of Compounds <b>16</b> and <b>18</b> in Complex with WDR5</h3><div class="NLM_p">We have previously reported a cocrystal structure of cyclic peptidomimetic <b>2</b> in complex with WDR5.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(14)</a> To gain further structural insights into the high-affinity interactions of cyclic peptidomimetics with WDR5, we have obtained additional cocrystal structures. To date, we have successfully determined a cocrystal structure for compound <b>16</b> at a resolution of 2.8 Å (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB">4GMB</a>) and a cocrystal structure for <b>18</b> in complex with WDR5 at a resolution of 1.64 Å (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC">5VFC</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0007.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A–C) Cocrystal structure of compound <b>16</b> in complex with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB">4GMB</a>) and (D) in comparison with cocrystal structure of <b>2</b> in complex with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM9">4GM9</a>). Carbon atoms for compound <b>16</b> and compound <b>2</b> are shown in cyan and yellow, respectively. Nitrogen atoms are shown in blue and oxygen atoms are shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0008.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Cocrystal structure of cyclic peptidomimetic <b>18</b> in complex with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC">5VFC</a>). (A) The unbiased difference electron density map for <b>18</b> contoured at 3σ is shown as a gray grid. (B) Structural alignment of WDR5–<b>18</b> with the cocrystal structures of compounds <b>2</b> and <b>16</b> in complex with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM9">4GM9</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB">4GMB</a>, respectively). (C) The hydrogen bonding and (D) hydrophobic interactions of WDR5 and <b>18</b>. Dashed lines represent hydrogen bonds (C). Carbon atoms for compounds <b>18</b>, <b>16</b>, and <b>2</b> are shown in yellow, cyan and white, respectively. Nitrogen atoms are shown in blue and oxygen atoms are shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The WDR5 component in the cocrystal structures of <b>16</b> and <b>18</b> adopts the same β-propeller configuration as the apo-WDR5 structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2H14">2H14</a>), with RMSDs (root-main-square deviation) of 0.505 and 0.511 Å, respectively. The compounds were unambiguously placed into the central channel of WDR5 through the guidance of difference electron density maps. (<a class="ref internalNav" href="#fig6" aria-label="Figures 6">Figures 6</a>A and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>A). Compounds <b>16</b> and <b>18</b> bind to the central channel of WD40 propeller in WDR5 through the conserved interaction network as observed in WDR5/MLL and WDR5/<b>2</b> cocrystal complex structures.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(14)</a> In agreement with the previous WDR5/<b>2</b> cocrystal structure, the arginine moiety in <b>16</b> and <b>18</b>, which is sandwiched between two phenyl rings from Phe133 and Phe263, engages in a complex array of hydrogen bonding with WDR5. The guanidinium moiety in <b>16</b> and <b>18</b> takes part in direct hydrogen bonds with Cys261, Phe133, and Ser91 and water-mediated hydrogen bonds with Ser218. The guanidinium moiety in <b>16</b> also forms a water-mediated hydrogen bond with Ser175. The N-terminal amide group of arginine in <b>16</b> and <b>18</b> forms a hydrogen bond with hydroxyl group of Ser91, and C-terminal carbonyl group of arginine engages in a water-mediated hydrogen bond with amide group of Cys261. Compounds <b>16</b> and <b>18</b> adopt a compact bound conformation mediated by 5 direct and 5 or 4 water mediated hydrogen bonds to WDR5 (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C).</div><div class="NLM_p">The methyl group next to the guanidinium moiety in <b>18</b> gained hydrophobic interactions with Cys134 and Phe263 (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>D). Complementary to these hydrogen bonds, the aliphatic carbon linker and side chains (methyl, isopropyl, ethyl, and phenyl groups) of both <b>16</b> and <b>18</b> make extensive hydrophobic packing against the hydrophobic surfaces surrounded by the side chains of Tyr131, Phe133, Tyr191, Tyr260, Leu321, Ile305, Ala47, and Ala65 from WDR5 (<a class="ref internalNav" href="#fig6" aria-label="Figures 6">Figures 6</a>C and <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>D). The different linker lengths in compounds <b>2</b>, <b>16</b>, and <b>18</b> lead to different orientations of the phenyl group in these compounds (<a class="ref internalNav" href="#fig6" aria-label="Figures 6">Figures 6</a>D and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B). In compounds <b>2</b> and <b>18</b>, the phenyl group stacks nearly parallel to the phenyl group of WDR5 Tyr260. In contrast, the phenyl group in <b>16</b> is oriented nearly perpendicular to the phenyl group of WDR5 Tyr260, comprising an incomplete aromatic cage. Interestingly, the side chain of WDR5 Lys259 is surrounded by an aromatic cage through cation−π stacking interactions (<a class="ref internalNav" href="#fig6" aria-label="Figures 6">Figures 6</a>D and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>D), which is reminiscent of the interaction mode observed in chromodomain recognition of methyl lysine.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(23)</a> Comparing with <b>2</b> and <b>16</b>, the smaller ring size of <b>18</b> forms a more compact structure, which facilitates the formation of several optimal intramolecular hydrogen bonds and reduces its conformational flexibility, allowing this compound to achieve a very high affinity to WDR5 and consequently high potency to inhibit the MLL HMT activity.</div><div class="NLM_p last">Overall, although cyclic peptidomimetics <b>2</b>, <b>16</b>, and <b>18</b> bind to WDR5 with similar binding modes, there are some differences observed in their structures and interactions with WDR5.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Microsomal Stability of Macrocyclic Peptidomimetics</h3><div class="NLM_p">One significant shortcoming we observed for linear peptidomimetic <b>1</b> was that it has poor stability in human microsomes (<i>T</i><sub>1/2</sub> = 4 min) and moderate stability in mouse and rat microsomes (<i>T</i><sub>1/2</sub> = 44 min and <i>T</i><sub>1/2</sub> = 34 min, respectively). To determine whether cyclization has improved the microsomal stability, we evaluated a number of cyclic peptidomimetics for their stability in human, mouse, and rat microsomes, and the data are summarized in <a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Microsomal Stabilities (<i>T</i><sub>1/2</sub>, min) of Linear Peptidomimetic <b>1</b> and Selected Cyclic Peptidomimetics</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">human</th><th class="colsep0 rowsep0" align="center" char=".">rat</th><th class="colsep0 rowsep0" align="center" char=".">mouse</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char=".">44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td><td class="colsep0 rowsep0" align="char" char=".">>60</td></tr></tbody></table></div></div><div class="NLM_p">The microsomal stability data showed that <b>2</b>, <b>18</b>, and a number of other macrocyclic peptidomimetics all have excellent microsomal stability in human, mouse, and rat microsomes with half-life greater than 60 min, except for compound <b>16</b> with shorter half-life in human microsomes (<i>T</i><sub>1/2</sub> = 21 min).</div><div class="NLM_p last">Overall, the microsomal stability data show that <b>2</b>, <b>18</b>, and a number of other cyclic peptidomimetics have excellent microsomal stability in human, mouse, and rat microsomes, and their stability significantly improved over linear peptidomimetic <b>1</b>.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Chemical Synthesis</h3><div class="NLM_p">Synthesis of common intermediates for cyclic peptidomimetics is shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. N-Termini of unnatural amino acids <b>20</b>, <b>22a</b>, and <b>22b</b> were protected with Fmoc group to give intermediates <b>21</b>, <b>23a</b>, and <b>23b</b>, respectively (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Fmoc Protection of Unnatural Amino Acids<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Fmoc-OSu, DIPEA, dioxane–H<sub>2</sub>O (2:1), rt, 5 h.</p></p></figure><div class="NLM_p">Intermediates <b>25</b>, <b>27</b>, and <b>29</b> were prepared on the 2-chlorotrityl chloride resin (<b>24</b>) using solid phase peptide synthesis with Fmoc chemistry and cleaved from the resin to yield carboxylic acids <b>26</b>, <b>28</b>, and <b>30</b>, respectively (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Solid Phase Synthesis of Carboxylic Acid Intermediates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Solid phase peptide synthesis; (b) 1% CF<sub>3</sub>COOH in CH<sub>2</sub>Cl<sub>2</sub>, rt, 30 min.</p></p></figure><div class="NLM_p">Synthesis of the designed cyclic peptidomimetics <b>2</b>–<b>7</b> and <b>15</b>–<b>17</b> is shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. An alkeneamine (<b>31a</b>–<b>31e</b>) was first attached to an Fmoc-phenylglycine (<b>23a</b>–<b>23c</b>) yielding <b>32a</b>–<b>32g</b>. Fmoc protecting group from <b>32a</b> and <b>32e</b> was then removed with diethylamine treatment, and the remaining amine was coupled to Fmoc-2-Abu-OH yielding <b>33a</b> and <b>33e</b>, respectively. A similar method used to make <b>33a</b> and <b>33e</b> was applied for the synthesis of <b>34a</b>, <b>34e</b>, <b>34h</b>, <b>35a</b>, <b>35e</b>, and <b>35h</b>. Intermediates <b>35f</b> and <b>35g</b> were prepared upon Fmoc deprotection of <b>32f</b> and <b>32g</b> with diethylamine followed by amide coupling with intermediate <b>28</b> synthesized in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Fmoc deprotection of <b>35a</b> and <b>35e</b>–<b>35h</b> followed by N-terminal capping with isobutiryl chloride afforded <b>36a</b> and <b>36e</b>–<b>36h</b>. Fmoc deprotection of <b>32b</b>–<b>32d</b> followed by amide coupling with intermediate <b>26</b> afforded <b>36b</b>–<b>36d</b>. The same procedure was applied to achieve <b>36i</b> starting with <b>32a</b> and carboxylic acid <b>30</b>. RCM cyclization of <b>36a</b>–<b>36h</b> followed by catalytic hydrogenation and removal of the Pbf protecting group from arginine side chain yielded the cyclic peptidomimetics <b>2</b>–<b>6</b> and <b>15</b>–<b>17</b> as trifluoroacetic acid salts. For the synthesis of <b>7</b>, <b>36i</b> was subjected to RCM cyclization, followed by catalytic hydrogenation and treatment with acid to obtain <b>7</b> as trifluoroacetic acid salt.</div><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Designed Cyclic Peptidomimetics <b>2–7</b> and <b>15</b>–<b>17</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 2–3 h; (b) DEA, acetonitrile, 2 h; (c) Fmoc-2-Abu-OH, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (d) Fmoc-Arg(Pbf)-OH (for intermediate <b>34a</b> and <b>34e</b>) or Fmoc-Arg(Me)(Pbf)-OH (for intermediate <b>34h</b>), EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (e) <b>21</b>, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (f) <b>28</b>, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (g) <b>26</b>, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (h) isobutiryl chloride, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, 2 h, rt; (i) <b>30</b>, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (j) Hoveyda–Grubb’s second generation catalyst, CH<sub>2</sub>Cl<sub>2</sub>, rt, overnight; (k) H<sub>2</sub>/Pd.C, MeOH, rt, 2 h; (l) CH<sub>2</sub>Cl<sub>2</sub>/CF<sub>3</sub>COOH/H<sub>2</sub>O (20:10:0.5), reflux, 2 h; (m) CH<sub>2</sub>Cl<sub>2</sub>/CF<sub>3</sub>COOH (1:1), rt, 0.5 h.</p></p></figure><div class="NLM_p">Synthesis of cyclic peptidomimetics <b>9</b>–<b>14</b>, <b>18</b>, and <b>19</b> is shown in <a class="ref internalNav" href="#sch4" aria-label="Schemes 4">Schemes 4</a> and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>. Intermediates <b>37a</b>–<b>37e</b>, <b>41a</b>, and <b>41b</b> were prepared on the 2-chlorotrityl chloride resin (<b>24</b>) using solid phase peptide synthesis with Fmoc chemistry. The intermediate peptides were cleaved from the resin followed by Boc or Benzyl protecting group removal with 10% trifluoroacetic acid in dichloromethane or hydrogenation with Pd/C in ethanol, respectively, to yield peptides <b>38a</b>–<b>38e</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) and <b>42a</b>–<b>42c</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). Intramolecular amide coupling of <b>38a</b>–<b>38e</b> and <b>42a</b>–<b>42c</b> followed by removal of the Pbf protecting group from the arginine side chain yielded cyclic peptidomimetics <b>9</b>–<b>14</b>, <b>18</b>, and <b>19</b> as trifluoroacetic acid salts.</div><figure id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Designed Cyclic Peptidomimetics <b>9–12</b> and <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Solid phase peptide synthesis; (b) 1% CF<sub>3</sub>COOH in CH<sub>2</sub>Cl<sub>2</sub>, rt, 30 min then 10% CF<sub>3</sub>COOH in CH<sub>2</sub>Cl<sub>2</sub> 1 h; (c) HATU, DIPEA, DMF, rt, 1 h; (d) 95% CF<sub>3</sub>COOH in water, rt, 2 h.</p></p></figure><figure id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0013.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Designed Cyclic Peptidomimetics <b>13</b>, <b>18</b>, and <b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) OsO<sub>4</sub>, NaIO<sub>4,</sub> H<sub>2</sub>O, THF, rt, 4 h; (b) dibenzylamine, NaBH(OAc)<sub>3</sub>, ClCH<sub>2</sub>CH<sub>2</sub>Cl, rt, 10 h; (c) solid phase peptide synthesis; (d) 1% CF<sub>3</sub>COOH in CH<sub>2</sub>Cl<sub>2</sub>, rt, 30 min then H<sub>2</sub>, Pd/C, ethanol, 50 °C, 12 h; (e) HATU, DIPEA, DMF, rt, 1 h; (f) 95% CF<sub>3</sub>COOH in water, rt, 2 h.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Summary</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30413" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30413" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In the present study, we describe the design, synthesis, and evaluation of macrocyclic peptidomimetics that bind to the MLL binding site in WDR5. This study has determined the optimal linker length in these macrocyclic peptidomimetics and discovered a number of potent and promising macrocyclic peptidomimetics with <b>18</b> as the best compound. Compound <b>18</b> binds to WDR5 with an IC<sub>50</sub> value of 0.90 nM (<i>K</i><sub>i</sub> value of <1 nM) and inhibits the MLL HMT activity with an IC<sub>50</sub> value of 12.7 nM. Significantly, <b>18</b> achieves IC<sub>50</sub> values of 0.21 μM and 0.25 μM, respectively, in inhibition of cell growth in the MOLM-13 and MV4-11 human AML cell lines harboring MLL translocation. In comparison, <b>18</b> is >40-times more potent than the previously published best compound <b>2</b> in inhibition of the MLL HMT activity and growth of MOLM-13 and MV4-11 cell lines. Furthermore, <b>18</b> displays >30-fold selectivity for MOLM-13 and MV4-11 cell lines carrying MLL translocation over HL-60 leukemia cell line lacking MLL translocation. Compound <b>18</b> displays excellent metabolic stability in human, mouse, and rat microsomes (<i>T</i><sub>1/2</sub> > 60 min). Cocrystal structures of compounds <b>16</b> and <b>18</b> in complex with WDR5 provide structural basis for their high binding affinity to WDR5. Compound <b>18</b> represents the most potent inhibitor of the WDR5–MLL interaction reported to date. Further optimization of <b>18</b> may ultimately yield a new therapy for the treatment of human acute leukemia carrying MLL translocation and potentially other human diseases and conditions that depend upon the MLL HTM activity, the WDR5–MLL interaction, or WDR5.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57485" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57485" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General</h3><div class="NLM_p last">All the final compounds were characterized with <sup>1</sup>H NMR, <sup>13</sup>C NMR (300 or 400 MHz, Bruker), and HRMS (ESI+) (Agilent Q-TOF Electrospray). The intermediates were characterized with <sup>1</sup>H NMR, <sup>13</sup>C NMR (300 or 400 MHz, Bruker), and MS (ESI+) (Thermo Scientific LCQ Fleet). Chemical shifts were reported in ppm relative to TMS. D<sub>2</sub>O (4.79 ppm), CD<sub>3</sub>OD (3.31 ppm), CD<sub>3</sub>CN (1.94 ppm), or DMSO-<i>d</i><sub>6</sub> (2.50 ppm) was used as the internal standard for <sup>1</sup>H NMR spectra. D<sub>2</sub>O (1,4-dioxane, 66.7 ppm), CD<sub>3</sub>OD (49.2 ppm), CD<sub>3</sub>CN (1.4 ppm), or DMSO-<i>d</i><sub>6</sub> (39.5 ppm) was used as internal standard for <sup>13</sup>C NMR spectra. The final products were purified on a preparative HPLC (Waters 2545, Quaternary Gradient Module) with a SunFire Prep C18 OBD 5 μm, 50 mm × 100 mm reverse-phase column. The mobile phase was a gradient of solvent A (0.1% trifluoroacetic acid in water) and solvent B (0.1% trifluoroacetic acid in CH<sub>3</sub>CN) at a flow rate of 60 mL/min and 1%/min increase of solvent B. All final compounds have purity ≥95% as determined by Waters ACQUITY UPLC using reverse-phase column (SunFire, C18, 5 μm, 4.6 mm × 150 mm) and a solvent gradient of A (0.1% of trifluoroacetic acid in water) and solvent B (0.1% of trifluoroacetic acid in CH<sub>3</sub>CN).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Competitive Binding Experiments to WDR5 Protein</h3><div class="NLM_p last">The binding affinities of these synthesized compounds were tested using a fluorescence polarization (FP) based competitive binding assay described earlier.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(10)</a> Briefly, to a 5 μL solution of the tested compound in DMSO, 120 μL of preincubated complex solution (N-terminal His-tagged WDR5 protein (residues 24–334), named WDR5Δ23, and 5-FAM labeled tracer) in assay buffer (0.1 M phosphate, 25 mM KCl, 0.01% Triton, pH 6.5) was added, giving final concentrations of WDR5Δ23 and the tracer of 4 nM and 0.6 nM, respectively. The plates were incubated at room temperature on a shaker for 3 h, and then the mP values were measured using the Tecan Infinite M-1000 plate reader (Tecan U.S., Research Triangle Park, NC). <i>K</i><sub>i</sub> values were calculated using the equation described previously.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(10)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>In Vitro</i> Cell-Free MLL HMT Functional Assay</h3><div class="NLM_p">The recombinant MLL complex containing human MLL (MLL1) protein (3735–3973) with N-terminal GST tag and MW = 53.7 kDa, full length human WDR5 with N-terminal 6×His tag and MW = 35 kDa, full length human ASH2L with N-terminal 6×His tag and MW = 61 kDa, full length human RbBP5 with N-terminal 6×His-tag and MW = 60 kDa, and full length human DPY30 with N-terminal 6×His-tag and MW = 12 kDa, and recombinant nucleosomes were obtained from Activmotif (Carlsbad, CA). Anti-histone H3 lysine 4 (H3K4me1–2) AlphaLISA acceptor beads, AlphaScreen streptavidin donor beads and biotinylated anti-H3 (C-terminus) antibody were obtained from PerkinElmer Life Sciences (Waltham, MA). Compound serial dilutions (2.5 μL) in assay buffer with 4% DMSO and 5 μL of pentameric MLL complex solution were added into a white low volume 384 well microtiter plate, which was incubated for 30 min with gentle shaking at room temperature, followed by addition of 2.5 μL of SAM/Nucleosome mixture. The methylation reaction was performed in 50 mM Tris, pH 8.5, with 1 mM DTT and 0.01% Tween-20 added right before the assay. Final concentrations of MLL complex, SAM, and nucleosomes were 5, 200, and 3 nM, respectively. Final DMSO in the reaction mixture was 1%. The reaction was allowed to proceed for 120 min in the dark with gentle shaking at room temperature. Concentrations of reaction components and times were adjusted accordingly for assay development experiments. Five microliters of high salt stopping solution (50 mM Tris, pH 7.4, with 1 M NaCl, 0.1% Tween-20, and 0.3% poly(<span class="smallcaps smallerCapital">l</span>-lysine)) was added to stop the methylation reactions for 15 min. Five microliters of 5× acceptor beads/biotinylated anti-H3 antibody mixture in detection buffer (50 mM Tris, pH 7.4, with 0.3 M NaCl, 0.1% Tween-20, and 0.001% poly(<span class="smallcaps smallerCapital">l</span>-lysine)) was added, followed by 1 h incubation at room temperature to allow full interaction between antibodies and methylated nucleosomes. Then 5 μL of 5× streptavidin donor beads was added in detection and incubated for 30 min.</div><div class="NLM_p last">Plates were read on a BMG CLARIOstar microplate reader with an excitation wavelength of 680 nm and emission wavelength of 615 nm. IC<sub>50</sub> values of compounds were obtained by fitting the fluorescence intensities detected at 615 nm versus compound concentrations in a sigmoidal dose–response curve (variable slope) with a nonlinear regression, using Graphpad Prism 6.0 software (Graphpad Software, San Diego, CA).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Purification of WDR5 Protein</h3><div class="NLM_p last">WDR5 (residues 24–334) was cloned into a His<sub>6</sub>-SUMO-vector. The protein was expressed in <i>Escherichia coli</i> Rosetta 2 (DE3) cells using Luria Broth media. Cells were subsequently sonicated in 25 mM Tris, pH 8.0, 500 mM NaCl, 5% glycerol, 1 mM benzamidine, 0.1% NP40, and 0.1% βME with protease inhibitors. The cellular debris was pelleted at 17 000 rpm for 45 min, and the supernatant was loaded onto a Ni-NTA resin (Qiagen) pre-equilibrated with 25 mM Tris, pH 8.0, 500 mM NaCl, 20 mM imidazole, and 5% glycerol. The column was then washed with 25 mM Tris, pH 8.0, 1 M NaCl, and 5% glycerol to remove contaminants, and the protein was eluted with 25 mM Tris, pH 8.0, 150 mM NaCl, 250 mM imidazole, and 5% glycerol. The eluate was incubated with His<sub>6</sub>-Ulp1 and dialyzed against 25 mM Tris, pH 8.0, and 150 mM NaCl overnight at 4 °C, then applied to fresh Ni-NTA to remove the cleaved tag and protease. The flow through of the Ni-NTA column was loaded onto a Source S column (GE Healthcare) pre-equilibrated with 25 mM Tris, pH 8.0, and 5% glycerol. The protein was eluted with a 0–500 mM NaCl gradient. For crystallographic studies, the protein was then dialyzed against 25 mM Tris, pH 8.0, and 150 mM NaCl overnight at 4 °C, concentrated to 20–30 mg/mL, and stored at −80 °C.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Crystallization and Structural Determination</h3><div class="NLM_p">WDR5/compound <b>16</b> binary complex was obtained by mixing WDR5 and compound <b>16</b> at molar ratio 1:2 before crystallization. The complex was crystallized at 25% <i>tert</i>-butanol, 0.1 M Tris-HCl, pH 7.5, at 293 K. The crystals were harvested in the same buffer with 20% glycerol. The 2.8 Å data set was collected at Advanced Photon Source beamline 21ID-G and was processed by HKL2000.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(24)</a> The crystals belong to <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> space group. The structure was solved by molecular replacement with Phaser<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(25)</a> using the previously published WDR5 structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2H14">2H14</a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(26)</a> There is one WDR5 molecule in one asymmetric unit. The structure was refined in Phenix<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(27)</a> with manual model building in Coot.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(28)</a> The final model has good stereochemistry with an <i>R</i>-value of 19.0% and an <i>R</i><sub>free</sub> of 24.7%.</div><div class="NLM_p last">The WDR5–<b>18</b> complex was formed by diluting the protein to 7 mg/mL with 50 mM Bis-Tris, pH 6.5, and 150 mM NaCl, then incubating with <b>18</b> in a 1:1.1 molar ratio for 1 h at 4 °C. Crystals formed at 20 °C in drops containing equal volumes of protein solution and precipitant (0.1 M Na Bis-Tris, pH 6.5, 26% PEG 8000, and 0.1 M ammonium sulfate). The crystals were cryoprotected with well solution containing 20% ethylene glycol. Diffraction data were collected at 0.9787 Å wavelength on a Rayonix – MX300 detector at LS-CAT 21-ID-F beamline at the Advanced Photon Source, then processed with HKL2000.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(24)</a> WDR5 in complex with compound <b>18</b> crystallized in <i>C</i>2 space group with 1 molecule of WDR5 per asymmetric unit. The structure was solved to 1.64 Å via molecular replacement<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(29)</a> with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR">3SMR</a>) as the search model. Iterative rounds of electron density fitting and refinement were completed using Coot<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(28)</a> and BUSTER,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(30)</a> respectively. The coordinates and geometric restraints for each compound were created from smiles using grade<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(30)</a> with the qm+mogul option. The first seven residues are disordered in all the structures. Data collection and refinement statistics are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_001.pdf" class="ext-link">Table S1</a>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Cell Viability Assay of Leukemia Cells</h3><div class="NLM_p">To assess cell viability (<a class="ref internalNav" href="#tbl2" aria-label="Tables 2">Tables 2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>), MOLM-13 leukemia cells carrying MLL-AF9 fusion, MV4-11 leukemia cells carrying MLL-AF4 fusion, or HL-60 leukemia cells carrying no MLL fusion were seeded 1 × 10<sup>4</sup> cells/well in 96-well plates and treated with an inhibitor for 4 days at different concentrations, in culture media containing 0.2% DMSO as the final concentration. Cell viability was determined using the WST-8 cell proliferation assay kit (Dojindo Molecular Technologies) according to manufacturer’s instructions. Three independent experiments in triplicate were performed. Data were analyzed using Prism software to determine 50% of cell growth inhibition (IC<sub>50</sub>) values versus DMSO control.</div><div class="NLM_p last">To assess the effect of long-term treatment of compound <b>18</b> on leukemia cells (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>), MOLM-13 and MV4-11 cell lines were plated at a density of 5 × 10<sup>4</sup> cell/mL in 24-well plates (2 mL/well) and treated with the relevant concentrations. On day 4, cell viability for each treatment was measured using the WST-8 cell count kit. Then 10% of viable cells from each well were transferred to freshly prepared medium containing corresponding concentrations of compound <b>18</b> and cultured for additional 3 days. On day 7, cell viability for each treatment was determined.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Computational Docking</h3><div class="NLM_p last">Docking studies were performed using the previously reported structure of WDR5 in complex with <b>1</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM3">4GM3</a>) with the GOLD<a onclick="showRef(event, 'ref700 ref800'); return false;" href="javascript:void(0);" class="ref ref700 ref800">(31, 32)</a> program (version 5.2) with the Arg moiety of compound <b>1</b> serving as a scaffold for constrained docking. For each genetic algorithm (GA) run, a maximum number of 200 000 operations were performed on a population of five islands of 100 individuals. Operator weights for crossover, mutation, and migration were set to 95, 95, and 10, respectively. The docking simulations were terminated after 10 runs for each ligand, using the Goldscore fitness function to evaluate the docked conformations. The top 10 conformations were saved for analysis of the predicted docking modes.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Chemical Synthesis Procedures, NMR Spectra, and HRMS Data of the Representative Compounds</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Method A</h4><div class="NLM_p last">The Fmoc protected intermediate was treated with diethylamine (20 equiv) in CH<sub>3</sub>CN for 2 h at room temperature followed by removal of the solvent and diethylamine in vacuo. The resulting crude was further dried under vacuum, then taken into CH<sub>2</sub>Cl<sub>2</sub> and mixed with the corresponding Fmoc-amino acid or peptide carboxylic acid (1.5 equiv), EDCI (1.5 equiv), HOAt (1.5 equiv), and diisopropylethylamine (1.5 equiv). The reaction mixture was stirred at room temperature for 2–3 h, quenched with H<sub>2</sub>O and extracted to CH<sub>2</sub>Cl<sub>2</sub>. The organic layers were collected and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, evaporated, and purified over flash chromatography.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Method B</h4><div class="NLM_p last">To a solution of intermediate <b>36a</b>–<b>36h</b> in CH<sub>2</sub>Cl<sub>2</sub>, Hoveyda–Grubbs second generation catalyst (0.5 equiv) was added under N<sub>2</sub> atmosphere, and the reaction mixture was stirred at 45 °C overnight under N<sub>2</sub> atmosphere unless stated otherwise. With intermediates <b>36e</b>–<b>36g</b>, another portion of the catalyst (0.5 equiv) was added and stirred further overnight under the same conditions before filtering through Celite and concentrating in vacuo. The remaining crude was purified over flash chromatography using CH<sub>2</sub>Cl<sub>2</sub>/MeOH. The cyclic product was taken up in MeOH, and the double bond was reduced using 10% Pd/C under 1 atm of H<sub>2</sub>. The reaction mixture was filtered through Celite and concentrated. The remaining crude was refluxed in CH<sub>2</sub>Cl<sub>2</sub>/trifluoroacetic acid/H<sub>2</sub>O (20:10:0.5) for 2 h in order to remove the Pbf group from arginine guanidine and evaporated. The crude product was purified with preparative HPLC using the C18 reverse phase column (Waters, Sunfire Prep C<sub>18</sub> OBD, 5 μm, 50 mm × 100 mm). The final compound then dissolved in CH<sub>3</sub>CN/H<sub>2</sub>O (1:1) and lyophilized.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-12-methyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclohexadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>2</b>)</h4><div class="NLM_p last">Compound <b>2</b> was prepared according Method B starting from <b>36a</b>. The RCM cyclization was achieved at room temperature overnight. White solid (53% yield over 3 steps). HRMS (ESI+): <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>47</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 587.3664, found 587.3664. <sup>1</sup>H NMR (300 MHz, MeOD): δ 7.42–7.30 (m, 5H), 5.25 (s, 1H), 4.30 (dd, <i>J</i> = 4.2, 9.8 Hz, 1H), 4.16 (dd, <i>J</i> = 6.2, 7.7 Hz, 1H), 3.51–3.42 (m, 1H), 3.19 (t, <i>J</i> = 6.9 Hz, 2H), 3.12–2.99 (m, 1H), 2.59–2.48 (m, 1H), 1.94–1.26 (m, 15H), 1.15–1.08 (m, 6H), 0.92 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, MeOD-<i>d</i><sub>4</sub>): δ 179.86, 176.83, 174.99, 173.64, 172.97, 158.86, 137.93, 130.06, 129.69, 129.32, 61.24, 60.70, 56.67, 55.27, 41.91, 40.34, 39.25, 36.12, 29.99, 26.39, 25.94, 22.81, 21.83, 20.06, 19.91, 10.84.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-1,12-dimethyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclohexadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>3</b>)</h4><div class="NLM_p last">The compound was prepared according Method B starting from <b>36e</b>. White solid (18% yield over 3 steps). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>49</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 601.3820, found 601.3819. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, rotamers) δ: 7.41–7.21 (m, 5H), 5.91 (s, 1H), 4.28–4.20 (m, 1H), 4.10 (dd, <i>J</i> = 4.7, 10.1 Hz, 1H), 3.21 (t, <i>J</i> = 7.1 Hz, 2H), 3.01 (s, 2.6H),2.95 (s, 0.4H), 2.84–2.72 (m, 1H), 2.62–2.51 (m, 1H), 2.07–1.21 (m, 16H), 1.18–1.08 (m, 6H), 1.00 (t, <i>J</i> = 7.3 Hz, 2.4H), 0.91 (t, <i>J</i> = 7.3 Hz, 0.6H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 180.14, 176.79, 174.99, 173.94, 172.30, 158.88, 138.57, 129.49, 129.28, 128.95, 60.76, 57.29, 56.26, 56.00, 47.38, 42.01, 40.82, 36.21, 35.54, 29.60, 27.91, 26.38, 25.61, 25.50, 21.67, 20.58, 19.72, 17.73.</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-3-(4-fluorophenyl)-9-(3-guanidinopropyl)-1,12-dimethyl-2,5,8,11-tetraoxo-1,4,7,10-tetraazacyclohexadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>4</b>)</h4><div class="NLM_p last">The compound was synthesized using Method B starting from <b>36f</b>. White solid (11% yield over 3 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>30</sub>H<sub>48</sub>FN<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 619.3726, found 619.3731. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 7.44–7.36 (m, 2H), 7.16–7.00 (m, 2H), 5.94 (s, 1H), 4.24–4.16 (m, 1H), 4.07 (dd, <i>J</i> = 4.2, 10.5 Hz, 1H), 3.20 (t, <i>J</i> = 7.1 Hz, 2H), 2.99 (s, 2.5H), 2.93 (s, 0.5 Hz), 2.78–2.68 (m, 1H), 2.63–2.49 (m, 1H), 2.14–1.18 (m, 16H), 1.15 (d, <i>J</i> = 6.8 Hz, 3H), 1.11 (d, <i>J</i> = 6.7 Hz, 3H), 1.01 (t, <i>J</i> = 7.3 Hz, 2.5H), 0.89 (t, <i>J</i> = 7.4 Hz, 0.5H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.14, 177.04, 175.19, 174.05, 172.07, 158.88, 134.96, 134.93, 131.36, 131.26, 116.12, 115.83, 60.69, 57.55, 56.61, 55.15, 46.99, 42.01, 40.67, 36.21, 35.29, 29.36, 27.51, 26.43, 26.16, 25.44, 21.54, 20.72, 19.63, 11.89.</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-3-(4-Chlorophenyl)-6-ethyl-9-(3-guanidinopropyl)-1,12-dimethyl-2,5,8,11-tetraoxo-1,4,7,10-tetraazacyclohexadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>5</b>)</h4><div class="NLM_p last">The compound was synthesized using Method B starting from <b>36g</b>. Ph<sub>2</sub>S was added during catalytic hydrogenation in order to reduce the activity of Pd/C. White solid (11% yield over 3 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>30</sub>H<sub>48</sub>ClN<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 635.3431, found 635.3428. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, rotamers) δ: 7.41–7.29 (m, 4H), 5.95 (s, 1H), 4.20 (t, 1H, <i>J</i> = 6.6 Hz), 4.07 (dd, 1H, <i>J</i> = 4.2, 10.4 Hz), 3.25–3.17 (m, 2H), 2.99 (s, 2.4H), 2.94 (s, 0.4 H), 2.90 (s, 0.2H), 2.77–2.68 (m, 1H), 2.62–2.51 (m, 1H), 2.15–1.59 (m, 9H), 1.55–1.18 (m, 7H), 1.16 (d, 3H, <i>J</i> = 6.8 Hz), 1.12 (d, 3H, <i>J</i> = 6.7 Hz), 1.02 (t, <i>J</i> = 7.4 Hz, 2.3 H), 0.90 (t, <i>J</i> = 7.5 Hz, 0.7 H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.13, 177.03, 175.22, 174.10, 171.85, 158.89, 137.86, 134.64, 131.07, 129.38, 60.68, 57.64, 56.59, 55.23, 46.88, 42.01, 40.69, 36.23, 35.25, 29.35, 27.43, 26.43, 26.29, 25.41, 21.51, 20.75, 19.61, 11.92.</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-12-methyl-9-(3-(3-methylguanidino)propyl)-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclohexadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>6</b>)</h4><div class="NLM_p last">The compound was synthesized using Method B starting from <b>36h</b>. The RCM cyclization was achieved at room temperature overnight. White solid (75% yield over 3 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>30</sub>H<sub>49</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 601.3820, found 601.3829. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.42–7.33 (m, 5H), 5.24 (s, 1H), 4.30 (dd, <i>J</i> = 4.2, 9.8 Hz, 1H), 4.17 (dd, <i>J</i> = 6.0, 7.8 Hz, 1H), 3.52–3.43 (m, 1H), 3.18 (t, <i>J</i> = 6.9 Hz, 2H), 3.11–2.99 (m, 1H), 2.85 (s, 3H), 2.59–2.48 (m, 1H), 1.96–1.24 (m, 15H), 1.18–1.06 (m, 6H), 0.92 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 179.84, 176.79, 175.01, 173.94, 172.98, 158.42, 137.90, 130.06, 129.70, 129.33, 61.29, 60.68, 56.62, 55.22, 41.93, 40.39, 39.28, 36.11, 30.06, 28.54, 26.40, 25.98, 22.79, 21.86, 20.08, 19.90, 10.81.</div></div><div id="sec4_8_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-9-(3-((<i>E</i>)-2,3-dimethylguanidino)propyl)-6-ethyl-12-methyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclohexadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>7</b>)</h4><div class="NLM_p last">The compound was synthesized according to Method B starting from <b>36i</b> with the following modification. The RCM cyclization was achieved at room temperature overnight. After the catalytic hydrogenation step, the crude was taken into CH<sub>2</sub>Cl<sub>2</sub>/trifluoroacetic acid (1:1) and stirred for 0.5 h at room temperature in order to form the trifluoroacetic acid salt. This mixture was evaporated and purified. White solid (27% yield over 3 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>31</sub>H<sub>51</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 615.3977, found 615.3975. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.42–7.33 (m, 5H), 5.24 (s, 1H), 4.36–4.28 (m, 1H), 4.18 (dd, <i>J</i> = 6.0, 7.8 Hz, 1H), 3.53–3.43 (m, 1H), 3.20 (t, <i>J</i> = 7.0 Hz, 2H), 3.10–2.98 (m, 1H), 2.85 (s, 6H), 2.59–2.48 (m, 1H), 1.96–1.23 (m, 15H), 1.15–1.07 (m, 6H), 0.92 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 179.79, 176.72, 175.06, 173.95, 172.97, 157.53, 137.88, 130.06, 129.71, 129.32, 61.34, 60.66, 56.56, 55.17, 41.93, 40.41, 39.29, 36.10, 30.06, 28.52, 26.34, 26.01, 22.89, 21.91, 20.07, 19.91, 10.81.</div></div><div id="sec4_8_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>N</i>-((3<i>S</i>,6<i>R</i>,9<i>R</i>,12<i>S</i>)-6-Ethyl-9-(3-guanidinopropyl)-12-methyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclohexadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>8</b>, MM-NC-401)</h4><div class="NLM_p last">The procedure, used to make <b>2</b>, was applied for making <b>8</b>. Identical NMR spectrum and ESI-MS data were observed.</div></div><div id="sec4_8_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclotridecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>9</b>)</h4><div class="NLM_p last">A method applied for <b>12</b> was used to make <b>9</b> starting from <b>24</b>. White solid (24.5% yield over 4 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>25</sub>H<sub>39</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 531.3038, found 531.3040. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.47–7.12 (m, 5H), 5.29 (s, 1H), 4.36 (t, <i>J</i> = 4.4 Hz, 1H), 4.10 (t, <i>J</i> = 7.8 Hz, 1H), 4.04 (t, <i>J</i> = 7.5 Hz, 1H), 3.58 (dd, <i>J</i> = 13.8, 4.4 Hz, 1H), 3.34 (dd, <i>J</i> = 13.6, 4.7 Hz, 1H), 3.12–2.93 (m, 2H), 2.48–2.41 (m, 1H), 1.80–1.29 (m, 6H), 1.03 (d, <i>J</i> = 2.6 Hz, 3H), 1.01 (d, <i>J</i> = 2.7 Hz, 3H), 0.77 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, DMSO) δ 176.36, 172.67, 171.06, 170.69, 170.63, 156.74, 137.19, 128.18, 127.59, 127.55, 57.49, 55.18, 54.67, 53.39, 40.78, 34.24, 27.40, 25.31, 23.21, 19.48, 19.09, 10.54.</div></div><div id="sec4_8_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclotetradecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>10</b>)</h4><div class="NLM_p last">A method applied for <b>12</b> was used to make <b>10</b> starting from <b>24</b>. White solid (21.7% yield over 4 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>26</sub>H<sub>41</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 545.3200, found 545.3197. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.44–7.36 (m, 2H), 7.36–7.26 (m, 3H), 5.44 (s, 1H), 4.33–4.21 (m, 2H), 4.18 (dd, <i>J</i> = 7.7, 6.2 Hz, 1H), 3.68 (dt, <i>J</i> = 14.4, 3.7 Hz, 1H), 3.26–3.10 (m, 2H), 2.88 (dd, <i>J</i> = 14.3, 11.8 Hz, 1H), 2.65 (hept, <i>J</i> = 6.6 Hz, 1H), 2.55–2.39 (m, 1H), 2.22–2.03 (m, 1H), 1.89–1.59 (m, 6H), 1.19 (dd, <i>J</i> = 13.8, 6.8 Hz, 6H), 0.94 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 181.84, 175.27, 175.05, 172.50, 172.27, 158.65, 139.13, 129.49, 129.39, 128.96, 58.55, 57.07, 55.76, 54.13, 41.74, 36.09, 35.68, 30.21, 28.78, 26.53, 24.52, 20.38, 19.42, 11.07.</div></div><div id="sec4_8_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclopentadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>11</b>)</h4><div class="NLM_p last">A method applied for <b>12</b> was used to make <b>11</b> starting from <b>24</b>. White solid (21.5% yield over 4 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>27</sub>H<sub>43</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 559.3356, found 559.3351. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.47–7.25 (m, 5H), 5.27 (s, 1H), 4.31 (dd, <i>J</i> = 10.5, 5.0 Hz, 1H), 4.20 (dd, <i>J</i> = 7.9, 6.4 Hz, 1H), 3.96 (dd, <i>J</i> = 9.5, 3.7 Hz, 1H), 3.46–3.37 (m, 1H), 3.24–3.01 (m, 3H), 2.65–2.49 (m, 1H), 1.93–1.60 (m, 10H), 1.15 (dd, <i>J</i> = 6.9, 5.9 Hz, 6H), 0.93 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 180.68, 175.30, 174.76, 173.75, 172.98, 158.61, 137.32, 129.90, 129.58, 129.22, 60.94, 56.96, 56.23, 55.33, 41.56, 39.56, 35.76, 29.84, 28.93, 26.58, 26.15, 19.96, 19.63, 10.67.</div></div><div id="sec4_8_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclohexadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>12</b>)</h4><div class="NLM_p">Fmoc-D-Phg-OH (0.5 mmol, 0.17 g) was loaded on the 0.1 mmol 2-chlorotrityl chloride <b>(24</b>) resin (ChemPep) (1 mmol/g) overnight in CH<sub>2</sub>Cl<sub>2</sub> and in the presence of 2,4,6-collidine (3 mmol, 0.4 mL). Then, the resin was washed with DMF, MeOH, and CH<sub>2</sub>Cl<sub>2</sub>, successively, mixed with DIPEA (0.29 mmol, 0.5 mL) in MeOH/CH<sub>2</sub>Cl<sub>2</sub> (1:5) and shaken for 30 min to end-cap unreacted 2-chlorotrityl groups on the resin. Next, classical chain elongation was carried out with Fmoc chemistry. The peptide intermediate (<b>38d</b>) was cleaved from the resin by treatment of <b>37d</b> with 4 mL of 1% trifluoroacetic acid in CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 min). The filtrate was evaporated, followed by treatment with 10% trifluoroacetic acid in CH<sub>2</sub>Cl<sub>2</sub> for 30 min. Then the solvent was evaporated, and the remaining crude was purified with preparative HPLC using the C18 reverse phase column (Waters, Sunfire Prep C<sub>18</sub> OBD, 5 μm, 50 mm × 100 mm) to yield <b>38d</b>. White powder, MS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>41</sub>H<sub>63</sub>N<sub>8</sub>O<sub>9</sub>S [M + H]<sup>+</sup> 843.44, found 843.38.</div><div class="NLM_p">Intermediate <b>38d</b> (50 mg, 0.05 mmol) dissolved in 5 mL of DMF was slowly added to a solution of HATU (38 mg, 0.1 mmol) and DIPEA (0.05 mL, 0.25 mmol) in 5 mL of DMF during 30 min. The reaction was stirred for another 30 min, and then the solvent was evaporated. The remaining crude was purified with preparative HPLC using the C18 reverse phase column (Waters, Sunfire Prep C<sub>18</sub> OBD, 5 μm, 50 mm × 100 mm) to yield <b>39d</b>. White powder, MS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>41</sub>H<sub>61</sub>N<sub>8</sub>O<sub>8</sub>S [M + H]<sup>+</sup> 825.43, found 825.36.</div><div class="NLM_p last">The cyclic product <b>39d</b> was then dissolved in trifluoroacetic acid/H<sub>2</sub>O (95:5) and stirred at room temperature for 2 h in order to remove the Pbf group from arginine guanidine. Then the solvent was evaporated, and the crude product was purified with preparative HPLC using the C18 reverse phase column (Waters, Sunfire Prep C<sub>18</sub> OBD, 5 μm, 50 mm × 100 mm). The final compound <b>12</b> then was dissolved in CH<sub>3</sub>CN/H<sub>2</sub>O (1:1) and lyophilized. White solid (12.7% yield over 4 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>28</sub>H<sub>45</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 573.3513, found 573.3507. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.50–7.28 (m, 5H), 5.14 (s, 1H), 4.28 (t, <i>J</i> = 6.8 Hz, 1H), 4.13 (dd, <i>J</i> = 10.4, 5.2 Hz, 1H), 4.08–3.99 (m, 1H), 3.54–3.41 (m, 1H), 3.18 (td, <i>J</i> = 7.1, 2.5 Hz, 2H), 3.04–2.97 (m, 1H), 2.62–2.51 (m, 1H), 1.94–1.54 (m, 9H), 1.47–1.24 (m, 3H), 1.13 (dd, <i>J</i> = 6.9, 3.3 Hz, 6H), 0.90 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 180.51, 175.63, 174.34, 173.64, 172.92, 158.63, 137.30, 129.92, 129.67, 129.31, 61.54, 57.09, 56.08, 55.83, 41.61, 39.68, 35.72, 32.14, 29.95, 29.24, 26.68, 26.53, 23.37, 19.93, 19.74, 10.25.</div></div><div id="sec4_8_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-12-methyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclotetradecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>13</b>)</h4><div class="NLM_p">Fmoc-D-Phg-OH (0.5 mmol, 0.17 g) was loaded on the 0.1 mmol 2-chlorotrityl chloride <b>(24</b>) resin (ChemPep) (1 mmol/g) overnight in CH<sub>2</sub>Cl<sub>2</sub> and in the presence of 2,4,6-collidine (3 mmol, 0.4 mL). Then, the resin was washed with DMF, MeOH, and CH<sub>2</sub>Cl<sub>2</sub>, successively, mixed with DIPEA (0.29 mmol, 0.5 mL) in MeOH/CH<sub>2</sub>Cl<sub>2</sub> (1:5), and shaken for 30 min to end-cap unreacted 2-chlorotrityl group on the resin. Next, classical chain elongation was carried out with Fmoc chemistry. The peptide intermediate was cleaved from the resin by treatment of <b>41a</b> with 4 mL of 1% trifluoroacetic acid in CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 min). The filtrate was evaporated, and the residue was dissolved in anhydrous ethanol. Pd/C (20 mg) was added to the flask, and the reaction was stirred at H<sub>2</sub> atmosphere for 12 h at 50 °C. Then the reaction was filtered, the filtrate was evaporated, and the remaining crude was purified with preparative HPLC using the C18 reverse phase column (Waters, Sunfire Prep C<sub>18</sub> OBD, 5 μm, 50 mm × 100 mm) to yield <b>42a</b>. White powder, MS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>40</sub>H<sub>61</sub>N<sub>8</sub>O<sub>9</sub>S [M + H]<sup>+</sup> 829.43, found 829.25.</div><div class="NLM_p last">Then the method applied for <b>12</b> starting from <b>38d</b> was used to make <b>13</b> starting from <b>42a</b>. White solid (8.7% yield over 4 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>27</sub>H<sub>43</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 559.3352, found 559.3351. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.49–7.37 (m, 2H), 7.37–7.20 (m, 3H), 5.46 (d, <i>J</i> = 9.2 Hz, 1H), 4.38 (dd, <i>J</i> = 10.5, 3.8 Hz, 1H), 4.24–4.06 (m, 1H), 3.85–3.63 (m, 1H), 3.26–3.16 (m, 2H), 2.76–2.56 (m, 3H), 2.24–2.07 (m, 1H), 1.92–1.59 (m, 5H), 1.53 (s, 3H), 1.40–1.32 (m, 1H), 1.23 (d, <i>J</i> = 6.9 Hz, 3H), 1.16 (d, <i>J</i> = 6.7 Hz, 3H), 0.96 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, MeOD) δ 182.07, 177.62, 175.47, 172.28, 172.11, 158.66, 139.36, 129.48, 129.42, 128.86, 59.98, 58.21, 57.51, 55.53, 41.72, 38.63, 36.40, 35.98, 28.83, 26.67, 24.75, 21.17, 18.90, 11.06.</div></div><div id="sec4_8_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-12-methyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclopentadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>14</b>)</h4><div class="NLM_p last">A method applied for <b>12</b> was used to make <b>14</b> starting from <b>24</b>. White solid (19.4% yield over 4 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>28</sub>H<sub>45</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 573.3513, found 573.3506. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.55–7.23 (m, 5H), 5.35 (s, 1H), 4.40 (dd, <i>J</i> = 10.0, 4.7 Hz, 1H), 4.16 (dd, <i>J</i> = 8.6, 6.3 Hz, 1H), 3.41–3.33 (m, 1H), 3.24–3.12 (m, 3H), 2.65–2.37 (m, 1H), 2.00–1.48 (m, 10H), 1.46 (s, 3H), 1.12 (dd, <i>J</i> = 6.8, 2.5 Hz, 6H), 0.94 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 179.91, 176.79, 174.82, 174.33, 172.75, 158.62, 138.00, 129.82, 129.37, 128.93, 60.47, 60.37, 56.91, 55.12, 41.74, 40.12, 37.57, 35.81, 28.76, 26.24, 25.14, 23.68, 21.04, 19.97, 19.36, 11.00.</div></div><div id="sec4_8_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-12-methyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacycloheptadecan-12-yl)isobutyramideCF<sub>3</sub>COOH Salt (<b>15</b>)</h4><div class="NLM_p last">Compound <b>15</b> was prepared according Method B starting from <b>36b</b>. The RCM cyclization was achieved at room temperature overnight. White solid (28% yield over 3 steps). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>49</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 601.3820, found 601.3827. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.45–7.32 (m, 5H), 5.22 (s, 1H), 4.45–4.31 (m, 2H), 3.28–3.13 (m, 4H), 2.59–2.49 (m, 1H), 1.97–1.05 (m, 23H), 0.89 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 179.80, 176.66, 174.54, 173.62, 173.32, 158.81, 137.54, 130.08, 129.79, 129.44, 61.04, 60.91, 55.74, 54.59, 41.83, 41.69, 39.95, 36.05, 30.64, 29.75, 27.18, 27.10, 26.14, 25.08, 21.12, 20.33, 19.48, 10.18.</div></div><div id="sec4_8_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-12-methyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclooctadecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>16</b>)</h4><div class="NLM_p last">Compound <b>16</b> was prepared according Method B starting from <b>36c</b>. The RCM cyclization was achieved at room temperature overnight. White solid (46% yield over 3 steps). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>31</sub>H<sub>51</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 615.3977, found 615.3976. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.45–7.29 (m, 5H), 5.40 (s, 1H), 4.29–4.19 (m, 2H), 3.59–3.47 (m, 1H), 3.19 (t, <i>J</i> = 6.7 Hz, 2H), 3.09–2.96 (m, 1H), 2.63–2.52 (m, 1H), 2.02–1.59 (m, 8H), 1.58–1.27 (m, 11H), 1.14 (d, <i>J</i> = 6.5 Hz, 3H), 1.12 (d, <i>J</i> = 6.5 Hz, 3H), 0.89 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.53, 177.59, 174.48, 173.99, 172.88, 158.86, 138.38, 129.97, 129.57, 60.67, 59.87, 56.51, 55.22, 42.04, 39.80, 37.53, 36.09, 29.98, 28.42, 28.38, 26.42, 25.37, 22.72, 22.47, 20.08, 19.94, 10.69.</div></div><div id="sec4_8_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-9-(3-guanidinopropyl)-12-methyl-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacycloicosan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>17</b>)</h4><div class="NLM_p last">Compound <b>17</b> was prepared according Method B starting from <b>36d</b>. The RCM cyclization was achieved at room temperature overnight. White solid (27% yield over 3 steps). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>55</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 643.4290, found 643.4292. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.41–7.30 (m, 5H), 5.40 (s, 1H), 4.34–4.28 (m, 1H), 4.18 (t, <i>J</i> = 7.1 Hz, 1H), 3.55–3.44 (m, 1H), 3.17 (t, <i>J</i> = 6.9 Hz, 2H), 3.07–2.95 (m, 1H), 2.61–2.50 (m, 1H), 2.00–1.28 (m, 23H), 1.16–1.07 (m, 6H), 0.92 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.22, 176.92, 174.53, 174.30, 172.69, 158.81, 138.70, 130.03, 129.55, 129.39, 60.72, 59.51, 57.11, 54.55, 42.14, 40.03, 38.56, 36.13, 30.28, 29.31, 28.92, 28.63, 28.04, 26.56, 26.16, 26.10, 23.50, 22.76, 20.05, 10.81.</div></div><div id="sec4_8_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-((3<i>R</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-12-methyl-9-(3-(3-methylguanidino)propyl)-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclotetradecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>18</b>)</h4><div class="NLM_p last">A method applied for <b>13</b> was used to make <b>18</b> starting from <b>24</b>. White solid (15.4% yield over 4 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>28</sub>H<sub>45</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 573.3507, found 573.3511. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.44–7.37 (m, 2H), 7.37–7.26 (m, 3H), 5.48–5.44 (m, 1H), 4.38 (dd, <i>J</i> = 10.5, 3.7 Hz, 1H), 4.17 (td, <i>J</i> = 7.0, 3.2 Hz, 1H), 3.80–3.72 (m, 1H), 3.28–3.16 (m, 2H), 2.83 (s, 3H), 2.75–2.57 (m, 3H), 2.25–2.11 (m, 1H), 1.96–1.69 (m, 4H), 1.53 (s, 3H), 1.41–1.32 (m, 1H), 1.22 (d, <i>J</i> = 6.9 Hz, 3H), 1.16 (d, <i>J</i> = 6.7 Hz, 3H), 0.95 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, MeOD) δ 182.17, 177.63, 175.52, 172.28, 172.10, 158.21, 139.37, 129.49, 129.41, 128.86, 60.07, 58.20, 57.47, 55.52, 41.75, 38.63, 36.41, 36.02, 28.88, 28.31, 26.64, 24.75, 21.18, 18.90, 11.06.</div></div><div id="sec4_8_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-((3<i>S</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-6-Ethyl-12-methyl-9-(3-(3-methylguanidino)propyl)-2,5,8,11-tetraoxo-3-phenyl-1,4,7,10-tetraazacyclotetradecan-12-yl)isobutyramide CF<sub>3</sub>COOH Salt (<b>19</b>)</h4><div class="NLM_p last">Compound <b>19</b> was obtained in the synthesis of <b>18</b> starting from <b>24</b>, as <span class="smallcaps smallerCapital">d</span>-α-phenylglycine was isomerized to <span class="smallcaps smallerCapital">l</span>-α-phenylglycine during the solid phase peptide synthesis. White solid (7% yield over 4 steps). HRMS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>28</sub>H<sub>45</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup> 573.3507, found 573.3504. <sup>1</sup>H NMR (300 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.36–7.19 (m, 5H), 5.21 (s, 1H), 4.27 (dd, <i>J</i> = 8.8, 6.3 Hz, 1H), 4.00 (dd, <i>J</i> = 8.5, 6.0 Hz, 1H), 3.38 (dt, <i>J</i> = 14.6, 3.9 Hz, 1H), 3.28–3.14 (m, 2H), 3.04 (t, <i>J</i> = 13.5 Hz, 1H), 2.84 (s, 3H), 2.70- 2.53 (m, 2H), 2.00–1.60 (m, 7H), 1.55 (s, 3H), 1.20 (d, J = 5.5 Hz, 3H), 1.18 (d, J = 5.3 Hz, 3H), 0.91 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, MeOD) δ 181.09, 177.37, 174.97, 173.54, 171.64, 158.20, 138.80, 129.24, 128.74, 128.60, 60.17, 59.82, 57.73, 57.65, 41.83, 37.48, 37.15, 36.12, 28.51, 28.30, 26.74, 26.11, 21.12, 19.18, 10.88.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Fmoc Protection of Unnatural Amino Acids (<b>21</b>, <b>23a</b>, <b>23b</b>): (<i>R</i>)-2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-2-methylpent-4-enoic Acid (<b>21</b>)</h4><div class="NLM_p">Compound <b>20</b> (<i>R</i>-α-allylalanine·H<sub>2</sub>O (Nagase), 750 mg, 5.1 mmol) was mixed with Fmoc-OSu (7.7 mmol, 2.6 g) and DIPEA (7.7 mmol, 1.3 mL) in 60 mL of 1,4-dioxane/H<sub>2</sub>O (2:1). The reaction was stirred at room temperature overnight. 1,4-Dioxane was evaporated; the remaining mixture was neutralized with 1 N HCl and extracted to ethyl acetate. The organic layers were collected and dried over anhydrous sodium sulfate, filtered, evaporated, and purified over flash chromatography using CH<sub>2</sub>Cl<sub>2</sub>/MeOH (100:0.5) yielding 1.42 g of <b>21</b> as white solid (80% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 352.15, found 352.40. <sup>1</sup>H NMR (300 MHz, CD3OD) δ: 7.79 (d, <i>J</i> = 7.4 Hz, 2H), 7.66 (d, <i>J</i> = 7.4 Hz, 2H), 7.39 (t, <i>J</i> = 7.4 Hz, 2H), 7.31 (dt, <i>J</i> = 1.1 Hz, 7.4 Hz, 2H), 5.81–5.63 (m, 1H), 5.15–5.01 (m, 2H), 4.39–4.27 (m, 2H), 4.21 (t, <i>J</i> = 6.6 Hz, 1H), 2.78–2.53 (m, 2H), 1.44 (s, 3H).</div><div class="NLM_p last">A procedure similar to that applied for intermediate <b>21</b> was used for <b>23a</b> and <b>23b</b> starting from <b>22a</b> and <b>22b</b>, respectively. Quantitative yields were obtained for both intermediates.</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>S</i>)-2-((<i>S</i>)-2-((<i>R</i>)-2-Isobutyramido-2-methylpent-4-enamido)-5-(3-((2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl)guanidino)pentanamido)butanoic Acid (<b>26</b>)</h4><div class="NLM_p last">Fmoc-2-Abu-OH (8.1 mmol, 2.6 g) was loaded on the 2.7 mmol 2-chlorotrityl chloride (<b>24</b>) resin (ChemPep) (1.2 mmol/g) overnight in CH<sub>2</sub>Cl<sub>2</sub> and in the presence of DIPEA (8.1 mmol, 1.4 mL). Then, the resin was washed with DMF, MeOH, and CH<sub>2</sub>Cl<sub>2</sub>, successively, mixed with DIPEA (0.29 mmol, 0.5 mL) in MeOH/CH<sub>2</sub>Cl<sub>2</sub> (1:5), and shaken for 30 min to end-cap unreacted 2-chlorotrityl groups on the resin. Next, classical chain elongation was carried out with Fmoc chemistry. The carboxylic acid intermediate (<b>26</b>) was cleaved from the resin by treatment of <b>25</b> with 4 mL of 1% trifluoroacetic acid in CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 min). The filtrate was evaporated, and the remaining crude was purified with preparative HPLC using the C18 reverse phase column (Waters, Sunfire Prep C<sub>18</sub> OBD, 5 μm, 50 mm × 100 mm), yielding 0.9 g of white powder (70% yield). MS (ESI): <i>m</i>/<i>z</i> calculated for [M + H]<sup>+</sup> 693.37, found 693.42. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 5.79–5.63 (m, 1H), 5.12–5.03 (m, 2H), 4.45–4.36 (m, 1H), 4.24 (dd, <i>J</i> = 5.2, 8.4 Hz, 1H), 3.24–3.15 (m, 2H), 3.00 (s, 2H), 2.79–2.67 (m, 1H), 2.60–2.43 (m, 8H), 2.08 (s, 3H), 1.99–1.51 (m, 6H), 1.45 (s, 6H), 1.40 (s, 3H), 1.07 (d, <i>J</i> = 6.8 Hz, 6H), 0.97 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 179.89, 176.19, 175.20, 174.09, 133.74, 126.49, 119.96, 118.86, 88.10, 60.30, 60.21, 55.49, 54.00, 44.04, 41.91, 36.07, 30.31, 28.89, 25.75, 23.06, 20.17, 19.91, 19.77, 18.48, 12.71, 11.03.</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (5<i>R</i>,8<i>S</i>,11<i>S</i>)-5-Allyl-11-ethyl-1-(9<i>H</i>-fluoren-9-yl)-5-methyl-3,6,9-trioxo-8-(3-(3-((2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl)guanidino)propyl)-2-oxa-4,7,10-triazadodecan-12-oic Acid (<b>28</b>)</h4><div class="NLM_p last">Intermediate <b>28</b> was synthesized using the procedure applied for <b>26</b>. (70% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 845.52, found 845.39. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.79 (d, <i>J</i> = 7.4 Hz, 2H), 7.64 (d, <i>J</i> = 7.4 Hz, 2H), 7.38 (t, <i>J</i> = 7.4 Hz, 2H), 7.30 (ddt, <i>J</i> = 1.2, 2.9, 7.4 Hz, 2H), 5.76–5.59 (m, 1H), 5.11–4.98 (m, 2H), 4.49–4.28 (m, 3H), 4.28–4.13 (m, 2H), 3.15 (t, <i>J</i> = 6.6 Hz, 2H), 2.96 (s, 2H), 2.70–2.43 (m, 8H), 2.06 (s, 3H), 1.97–1.47 (m, 6H), 1.42 (s, 6H), 1.36 (s, 3H), 0.95 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 176.58, 175.20, 174.16, 157.67, 145.42, 145.39, 142.80, 133.80, 128.98, 128.36, 128.34, 126.36, 126.30, 121.13, 120.00, 67.87, 60.57, 55.51, 44.03, 42.30, 30.06, 28.85, 25.87, 23.35, 19.77, 18.46, 12.69, 11.00.</div></div><div id="sec4_9_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>S</i>)-2-((<i>S</i>)-5-((<i>E</i>)-2,3-Dimethylguanidino)-2-((<i>R</i>)-2-isobutyramido-2-methylpent-4-enamido)pentanamido)butanoic Acid (<b>30</b>)</h4><div class="NLM_p last">Intermediate <b>30</b> was synthesized using the procedure applied for <b>26</b> (38% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 469.31, found 469.50. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 5.81–5.66 (m, 1H), 5.15–5.05 (m, 2H), 4.47–4.41 (m, 1H), 4.26 (dd, <i>J</i> = 5.1, 8.4 Hz, 1H), 3.27–3.13 (m, 2H), 2.85 (s, 6H), 2.71 (dd, <i>J</i> = 7.6, 13.8 Hz, 1H), 2.58–2.47(m, 2H), 2.00–1.58 (m, 6H), 1.42 (s, 3H), 1.13–1.06 (m, 6H), 0.99 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.00, 176.21, 175.23, 174.05, 157.46, 133.70, 119.95, 60.31, 55.50, 53.78, 42.40, 42.13, 36.08, 30.55, 28.49, 25.99, 25.83, 22.86, 20.13, 19.89, 10.97.</div></div><div id="sec4_9_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (9<i>H</i>-Fluoren-9-yl)methyl (<i>R</i>)-2-((2-(Allylamino)-2-oxo-1-phenylethyl)amino)-2-oxoacetate (<b>32a</b>)</h4><div class="NLM_p last">Fmoc-<i>R</i>-Phenylglycine (<b>23c</b>; 2.7 mmol, 1 g) was mixed with allylamine hydrochloride (<b>31a</b>) (4.1 mmol, 0.38 g), EDCI (4.1 mmol, 0.79 g), HOAt (4.1 mmol, 0.55 g), and 3 equiv of DIPEA (8.1 mmol, 1.4 mL) in 100 mL of CH<sub>2</sub>Cl<sub>2</sub> and stirred at room temperature for 3 h. The reaction mixture was quenched with H<sub>2</sub>O and extracted to CH<sub>2</sub>Cl<sub>2</sub>. The organic layer together with a white precipitate was collected and concentrated in vacuo. The remaining white solid was washed with CH<sub>2</sub>Cl<sub>2</sub> yielding 0.96 g of intermediate (<b>32a</b>) as white powder (86% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>25</sub>N<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup> 413.1860, found 413.1862. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>CN) δ: 7.85 (d, <i>J</i> = 7.5 Hz, 2H), 7.68 (d, <i>J</i> = 7.0 Hz, 2H), 7.47–7.29 (m, 9H), 6.83 (brs, 1H), 6.47 (brs, 1H), 5.85–5.71 (m, 1H), 5.16 (d, <i>J</i> = 6.6 Hz, 1H), 5.07–4.96 (m, 2H), 4.35 (d, <i>J</i> = 6.8 Hz, 2H), 4.24 (t, <i>J</i> = 6.9 Hz, 1H), 3.81–3.74 (m, 2H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>CN): δ 170.75, 145.28, 145.18, 142.26, 135.61, 129.81, 129.24, 128.81, 128.42, 128.24, 126.30, 121.09, 115.87, 67.53, 59.96, 48.18, 42.31.</div></div><div id="sec4_9_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>R</i>)-(9<i>H</i>-Fluoren-9-yl)methyl 2-((2-(but-3-en-1-ylamino)-2-oxo-1-phenylethyl)amino)-2-oxoacetate (<b>32b</b>)</h4><div class="NLM_p last">Fmoc-<i>R</i>-Phenylglycine (<b>23c</b>; 1 mmol, 0.37 g) was mixed with 3-butenyl amine hydrochloride (<b>31b</b>, 1.5 mmol, 0.16 g), EDCI (1.5 mmol, 0.29 g), HOAt (1.5 mmol, 0.21 g), and DIPEA (3 mmol, 0.52 mL) in 40 mL of CH<sub>2</sub>Cl<sub>2</sub> and stirred at room temperature for 4 h. The reaction mixture was quenched with H<sub>2</sub>O and extracted to CH<sub>2</sub>Cl<sub>2</sub>. The organic layers were collected and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, evaporated and purified over flash chromatography using CH<sub>2</sub>Cl<sub>2</sub>:EtOAc (1:1) to afford 0.41 g of <b>32b</b> as white solid (96% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 427.20, found 427.25. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ: 7.75 (d, <i>J</i> = 7.5 Hz, 2H), 7.57 (brs, 2H), 7.45–7.23 (m, 9H), 6.32–6.21 (m, 1H), 5.77 (brs, 1H),5.70–5.53 (m, 1H), 5.18 (d, <i>J</i> = 5.3 Hz, 1H), 5.01–4.83 (m, 2H), 4.35(d, <i>J</i> = 6.5 Hz, 2H), 4.19 (t, <i>J</i> = 6.9 Hz, 1H), 3.41–3.18 (m, 2H), 2.24–2.09 (m, 2H).<sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ: 169.76, 155.88, 144.02, 143.96, 141.45, 134.77, 129.30, 128.71, 127.88, 127.41, 127.27, 125.28, 120.16, 117.76, 67.32, 59.09, 47.30, 38.87, 33.71.</div></div><div id="sec4_9_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>R</i>)-(9<i>H</i>-Fluoren-9-yl)methyl 2-oxo-2-((2-oxo-2-(pent-4-en-1-ylamino)-1-phenylethyl)amino)acetate (<b>32c</b>)</h4><div class="NLM_p last">Compound <b>32c</b> was prepared according to the procedure used for <b>32b</b> starting from <b>23c</b> and <b>31c</b>. The product was purified over flash chromatography using hexanes/EtOAc (3:1) to afford <b>32c</b> as white solid (84% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 441.22, found 441.50. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.79 (d, <i>J</i> = 7.5 Hz, 2H), 7.65 (d, <i>J</i> = 7.2 Hz, 2H), 7.44–7.24 (m, 9H), 5.83–5.67 (m, 1H), 5.19 (s, 1H), 4.99–4.88 (m, 2H), 4.37 (d, <i>J</i> = 6.6 Hz, 2H), 4.22 (t, <i>J</i> = 6.6 Hz, 1H), 3.27–3.10 (m, 2H), 2.03–1.92 (m, 2H), 1.60–1.48 (m, 2H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 172.95, 145.38, 142.78, 139.21, 129.95, 129.46, 128.99, 128.67, 128.38, 126.41, 121.12, 115.67, 68.31, 60.64, 40.23, 32.18, 29.77.</div></div><div id="sec4_9_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>R</i>)-(9<i>H</i>-Fluoren-9-yl)methyl 2-((2-(hept-6-en-1-ylamino)-2-oxo-1-phenylethyl)amino)-2-oxoacetate (<b>32d</b>)</h4><div class="NLM_p last">Compound <b>32d</b> was prepared according to the procedure used for <b>32b</b> starting from <b>23c</b> and <b>31d</b>. The product was purified over flash chromatography using CH<sub>2</sub>Cl<sub>2</sub>/EtOAc (3:1), and the product was further washed with <i>n</i>-hexanes to afford <b>32d</b> as white solid (80% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 469.25, found 469.00. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.73 (d, <i>J</i> = 7.5 Hz, 2H), 7.58 (d, <i>J</i> = 7.0 Hz, 2H), 7.41–7.21 (m, 9H), 5.79–5.63 (m, 1H), 5.17 (s, 1H), 4.97–4.83 (m, 2H), 4.35 (d, <i>J</i> = 6.7 Hz, 2H), 4.18 (t, <i>J</i> = 6.7 Hz, 1H), 3.25–3.06 (m, 2H), 2.01–1.89 (m, 2H), 1.51–1.13 (m, 6H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 171.72, 157.16, 144.60, 144.51, 142.06, 139.40, 129.49, 128.96, 128.45, 127.83, 127.77, 125.78, 120.63, 114.88, 67.82, 59.48, 47.87, 40.29, 34.32, 29.66, 29.20, 26.91.</div></div><div id="sec4_9_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (<i>R</i>)-(9<i>H</i>-Fluoren-9-yl)methyl (2-(allyl(methyl)amino)-2-oxo-1-phenylethyl)carbamate (<b>32e</b>)</h4><div class="NLM_p last">Compound <b>32e</b> was prepared according to the procedure used for <b>32b</b> starting from <b>23c</b> and <b>31e</b>. The product was purified over flash chromatography using hexanes/EtOAc (1:1). White solid (84% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 427.20, found 427.08. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, rotamers) δ: 7.85–7.70 (m, 2H), 7.62–7.47 (m, 2H), 7.46–7.16 (m, 9H), 5.76–5.41 (m, 2H), 5.13–4.88 (m, 2H), 4.36–3.65 (m, 5H), 2.88 (s, 1.3H), 2.84 (s, 1.5H), 2.76 (s, 0.2H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 172.32, 172.10, 158.01, 145.46, 145.29, 142.68, 138.58, 138.07, 133.55, 133.46, 130.24, 129.73, 129.41, 129.31, 128.93, 128.33, 126.46, 126.42, 121.07, 118.31, 117.84, 68.31, 57.61, 57.52, 52.99, 51.75, 48.50, 35.35, 34.36.</div></div><div id="sec4_9_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> (<i>R</i>)-(9<i>H</i>-fluoren-9-yl)methyl(2-(allyl(methyl)amino)-1-(4-fluorophenyl)-2-oxoethyl)carbamate (<b>32f</b>)</h4><div class="NLM_p last">Intermediate <b>32f</b> was prepared according to the procedure used for <b>32b</b> starting from <b>23a</b> and <b>31e</b>. The product was purified over flash chromatography using hexane/ethyl acetate (3:1). White solid (75% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 445.19, found 444.96. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, rotamers) δ: 7.88–7.72 (m, 2H), 7.65–7.50 (m, 2H), 7.47–7.19 (m, 6H), 7.14–6.78 (m, 2H), 5.80–5.51 (m, 2H), 5.17–5.03 (m, 2H), 4.42–4.19 (m, 3H), 4.11–3.88 (m, 2H), 2.90 (s, 1.2H), 2.87 (s, 1.5H), 2.77 (s, 0.3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 172.25, 172.00, 165.91, 162.64, 158.03, 157.98, 145.48, 145.31, 142.74, 134.73, 134.69, 134.21, 134.16, 133.54, 133.49, 131.54, 131.44, 131.35, 128.96, 128.34, 128.32, 126.43, 126.41, 121.09, 118.25, 117.90, 117.06, 116.77, 68.29, 56.80, 56.68, 53.03, 51.80, 48.56, 35.37, 34.44.</div></div><div id="sec4_9_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (<i>R</i>)-(9<i>H</i>-Fluoren-9-yl)methyl(2-(allyl(methyl)amino)-1-(4-chlorophenyl)-2-oxoethyl)carbamate (<b>32g</b>)</h4><div class="NLM_p last">Intermediate <b>32g</b> was prepared according to the procedure used for <b>32b</b> starting from <b>23b</b> and <b>31e</b>. The product was purified over flash chromatography using hexane/ethyl acetate (3:1). White solid (75% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 461.16, found 461.19. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, rotamers) δ: 7.88–7.72 (m, 2H), 7.65–7.50 (m, 2H), 7.47–7.20 (m, 6H), 7.14–7.04 (m, 2H), 5.78–5.47 (m, 2H), 5.15–5.01 (m, 2H), 4.38–4.25 (m, 2H), 4.23–4.13 (m, 1H), 4.08–3.85 (m, 2H), 2.90 (s, 1.2H), 2.87 (s, 1.5H),2.77 (s, 0.3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 171.86, 171.59, 157.87, 145.39, 145.21, 142.65, 137.45, 136.95, 135.51, 133.45, 133.40, 131.00, 130.93, 130.24, 128.92, 128.30, 126.37, 121.07, 118.28, 117.95, 68.23, 56.74, 56.59, 52.98, 51.74, 48.48, 35.36, 34.44.</div></div><div id="sec4_9_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (9<i>H</i>-Fluoren-9-yl)methyl ((<i>S</i>)-1-(((<i>R</i>)-2-(Allylamino)-2-oxo-1-phenylethyl)amino)-1-oxobutan-2-yl)carbamate (<b>33a</b>)</h4><div class="NLM_p last">Intermediate <b>32a</b> (1.65 mmol, 0.68 g) was treated with diethylamine (33 mmol, 3.5 mL) in 100 mL of CH<sub>3</sub>CN at 40 °C for 4 h followed by removal of the solvent and diethylamine in vacuo. The resulting crude was further dried under vacuum then taken into CH<sub>2</sub>Cl<sub>2</sub> and mixed with Fmoc-2-Abu-OH (2.5 mmol, 0.81 g), EDCI (2.5 mmol, 0.48 g), HOAt (2.5 mmol, 0.34 g), and DIPEA (2.5 mmol, 0.44 mL) at room temperature for 4 h. The reaction mixture was quenched with H<sub>2</sub>O and extracted to CH<sub>2</sub>Cl<sub>2</sub>. The organic layer together with a white precipitate was collected and concentrated in vacuo. The remaining white solid was washed from CH<sub>2</sub>Cl<sub>2</sub> yielding 0.57 g of intermediate <b>33a</b> as white powder (70% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>32</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup> 498.2387, found 498.2392. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>SO) δ: 8.56 (d, <i>J</i> = 8.0 Hz, 1H), 8.44 (t, <i>J</i> = 5.5 Hz, 1H), 7.89 (d, <i>J</i> = 7.5 Hz, 2H), 7.76–7.70 (m, 2H), 7.55 (d, <i>J</i> = 8.1 Hz, 1H), 7.45–7.38 (m, 4H), 7.36–7.22 (m, 4H), 5.80–5.67 (m, 1H), 5.48 (d, <i>J</i> = 8.0 Hz, 1H), 5.05–4.93 (m, 2H), 4.30–4.03 (m, 4H), 3.73–3.64 (m, 2H), 1.69–1.43 (m, 2H), 0.81 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>SO) δ: 171.52, 169.41, 156.07, 143.86, 143.75, 140.68, 138.99, 134.70, 128.22, 127.61, 127.45, 127.07, 126.88, 125.25, 120.07, 115.14, 65.67, 56.02, 55.91, 46.65, 40.81, 25.17, 10.35.</div></div><div id="sec4_9_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (9<i>H</i>-fluoren-9-yl)methyl ((<i>S</i>)-1-(((<i>R</i>)-2-(Allyl(methyl)amino)-2-oxo-1-phenylethyl)amino)-1-oxobutan-2-yl)carbamate (<b>33e</b>)</h4><div class="NLM_p last">Compound <b>33e</b> was prepared according to Method A starting from <b>32e</b> (0.7 g 1.64 mmol). Fmoc-2-Abu-OH used as the amino acid. The compound was purified over flash chromatography using hexanes/EtOAc (1:1) affording 0.68 g of intermediate <b>33e</b> as white solid (81% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 512.25, found 512.58. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 7.78 (d, <i>J</i> = 7.5 Hz, 2H), 7.64 (d, <i>J</i> = 7.3 Hz, 2H), 7.42–7.24 (m, 9H), 5.85 (s, 0.55H), 5.83 (s, 0.45H), 5.77–5.42 (m, 1H), 5.14–4.97 (m, 2H), 4.39–4.24 (m, 2H), 4.22–4.00 (m, 3H), 3.96–3.82 (m, 1H), 2.89 (s, 3H), 1.84–1.51 (m, 2H), 0.86 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 174.01, 173.94, 171.69, 171.46, 158.59, 145.56, 145.35, 142.77, 138.50, 138.01, 133.54, 133.49, 130.25, 129.77, 129.37, 129.29, 128.96, 128.39, 128.35, 126.43, 126.36, 121.11, 118.37, 117.89, 68.21, 58.03, 55.84, 55.71, 53.07, 51.74, 35.37, 34.31, 26.73, 10.76.</div></div><div id="sec4_9_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (9<i>H</i>-Fluoren-9-yl)methyl ((6<i>S</i>,9<i>S</i>,12<i>R</i>)-9-Ethyl-1-imino-7,10,13-trioxo-1-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-12-phenyl-2,8,11,14-tetraazaheptadec-16-en-6-yl)carbamate (<b>34a</b>)</h4><div class="NLM_p last">A method applied for <b>33a</b> was used to make <b>34a</b> starting from <b>33a</b>. White solid (83% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>49</sub>H<sub>60</sub>N<sub>7</sub>O<sub>8</sub>S [M + H]<sup>+</sup> 906.4219, found 906.4222. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>SO) δ: 8.59 (d, <i>J</i> = 8.1 Hz, 1H), 8.45 (t, <i>J</i> = 5.7 Hz, 1H), 7.96 (d, <i>J</i> = 7.6 Hz, 1H), 7.89 (d, <i>J</i> = 7.4 Hz, 2H), 7.75–7.67 (m, 2H), 7.53 (d, <i>J</i> = 8.2 Hz, 1H), 7.44–7.37 (m, 4H), 7.35–7.22 (m, 5H), 5.80–5.66 (m, 1H), 5.48 (d, <i>J</i> = 8.1 Hz, 1H), 5.04–4.94 (m, 2H), 4.39–4.17 (m, 4H), 4.08–3.96 (m, 1H), 3.72–3.64 (m, 2H), 3.07–2.98 (m, 2H), 2.93 (s, 2H), 2.49 (s, 3H), 2.43 (s, 3H), 1.99 (s, 3H), 1.72–1.35 (m, 12H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>SO): δ 171.77, 170.96, 169.40, 157.44, 156.05, 155.91, 143.85, 143.72, 140.69, 138.93, 137.26, 134.70, 134.20, 131.43, 128.22, 127.61, 127.47, 127.06, 126.93, 125.26, 124.30, 120.06, 116.25, 115.13, 86.25, 65.62, 56.03, 53.68, 46.67, 42.45, 40.84, 28.25, 25.33, 18.92, 17.58, 12.23, 10.02.</div></div><div id="sec4_9_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (9<i>H</i>-Fluoren-9-yl)methyl ((6<i>S</i>,9<i>S</i>,12<i>R</i>)-9-Ethyl-1-imino-14-methyl-7,10,13-trioxo-1-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-12-phenyl-2,8,11,14-tetraazaheptadec-16-en-6-yl)carbamate (<b>34e</b>)</h4><div class="NLM_p last">Compound <b>34e</b> was prepared according to Method A starting from <b>33e</b> (0.66 g 1.29 mmol). Fmoc-Arg(Pbf)-OH used as the amino acid. The compound was purified over flash chromatography using hexanes/EtOAc (1:1) affording 1.0 g (1.09 mmol) of intermediate <b>34e</b> as white solid (84% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 920.44, found 920.25. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 7.79 (d, <i>J</i> = 7.5 Hz, 2H), 7.66 (t, <i>J</i> = 7.2 Hz, 2H), 7.42–7.26 (m, 9H), 5.81 (s, 0.5H), 5.79 (s, 0.5H), 5.74–5.58 (m, 1H), 5.11–4.97 (m, 2H), 4.40–4.29 (m, 3H), 4.21 (t, <i>J</i> = 6.5 Hz, 1H), 4.11–3.98 (m, 2H), 3.91–3.80 (m, 1H), 3.19–3.10 (m, 2H), 2.96 (s, 2H), 2.86 (s, 3H), 2.58 (s, 3H), 2.51 (s, 3H), 2.06 (s, 3H), 1.87–1.46 (m, 6H), 1.42 (s, 6H), 0.85 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ:174.86, 173.27, 171.71, 171.54, 160.07, 158.70, 158.33, 145.65, 145.33, 142.80, 139.62, 137.70, 134.63, 133.73, 133.55, 133.46, 130.28, 129.83, 129.52, 129.42, 129.00, 128.38, 126.44, 126.23, 121.13, 118.65, 118.42, 117.82, 87.85, 68.21, 56.42, 56.07, 56.00, 53.08, 51.78, 44.17, 41.77, 35.41, 34.37, 30.6̀3, 28.89, 26.94, 26.41, 19.79, 18.60, 12.71, 10.76.</div></div><div id="sec4_9_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (9<i>H</i>-Fluoren-9-yl)methyl ((8<i>S</i>,11<i>S</i>,14<i>R</i>,<i>E</i>)-11-Ethyl-9,12,15-trioxo-3-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-14-phenyl-2,4,10,13,16-pentaazanonadeca-2,18-dien-8-yl)carbamate (<b>34h</b>)</h4><div class="NLM_p last">A method similar that applied for <b>33a</b> was used for the synthesis of <b>34h</b> starting from <b>33a</b> (0.5 g, 1 mmol). Fmoc-Arg(Me) (Pbf)-OH was used as the amino acid. Intermediate <b>34h</b> (0.36 g) was obtained as white solid (65% yield) and used for the next step without further purification. MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 920.44, found 920.25.</div></div><div id="sec4_9_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (9<i>H</i>-Fluoren-9-yl)methyl ((6<i>R</i>,9<i>S</i>,12<i>S</i>,15<i>R</i>)-9-Ethyl-15-methyl-5,8,11,14-tetraoxo-12-(3-(3-((2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl)guanidino)propyl)-6-phenyl-4,7,10,13-tetraazaoctadeca-1,17-dien-15-yl)carbamate (<b>35a</b>)</h4><div class="NLM_p last">Intermediate <b>34a</b> (1.18 g, 1.3 mmol) was treated with diethylamine (26 mmol, 2.7 mL) in 50 mL of CH<sub>3</sub>CN at 40 °C for 2 h followed by removal of the solvent and diethylamine in vacuo. The resulting crude was taken into CH<sub>2</sub>Cl<sub>2</sub> and mixed with <b>21</b> (2 mmol, 0.7 g), EDCI (2 mmol, 0.38 g), HOAt (2 mmol, 0.27 g), and DIPEA (2 mmol, 0.35 mL) at room temperature for 4 h. The reaction mixture was quenched with H<sub>2</sub>O and extracted to CH<sub>2</sub>Cl<sub>2</sub>. The organic layers were collected and dried over anh. sodium sulfate, filtered, evaporated, and purified over flash chromatography using EtAc/MeOH (50:0.7) yielding 0.8 g of intermediate <b>35a</b> as white solid (60% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>55</sub>H<sub>69</sub>N<sub>8</sub>O<sub>9</sub>S [M + H]<sup>+</sup> 1017.4903, found 1017.4899. <sup>1</sup>H NMR (300 MHz, MeOD) δ: 7.79 (d, <i>J</i> = 7.4 Hz, 2H), 7.65 (d, <i>J</i> = 7.4 Hz, 2H), 7.43–7.25 (m, 9H), 5.77–5.60 (m, 2H), 5.40 (s, 1H), 5.11–4.92 (m, 4H), 4.52–4.43 (m, 1H), 4.36–4.15 (m, 4H), 3.84–3.64 (m, 2H), 3.17–3.09 (m, 2H), 2.95 (s, 2H), 2.67–2.39 (m, 8H), 2.05 (s, 3H), 1.96–1.45 (m, 6H), 1.41 (s, 6H), 1.34 (s, 3H), 0.89 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD): δ 171.77, 170.96, 169.40, 157.44, 156.05, 155.91, 143.85, 143.72, 140.69, 138.93, 137.26, 134.70, 134.20, 131.43, 128.22, 127.61, 127.47, 127.06, 126.93, 125.26, 124.30, 120.06, 116.25, 115.13, 86.25, 65.62, 56.03, 53.68, 46.67, 42.45, 40.84, 28.25, 25.33, 18.92, 17.58, 12.23, 10.02.</div></div><div id="sec4_9_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (9<i>H</i>-Fluoren-9-yl)methyl ((6<i>R</i>,9<i>S</i>,12<i>S</i>,15<i>R</i>)-9-Ethyl-4,15-dimethyl-5,8,11,14-tetraoxo-12-(3-(3-((2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl)guanidino)propyl)-6-phenyl-4,7,10,13-tetraazaoctadeca-1,17-dien-15-yl)carbamate (<b>35e</b>)</h4><div class="NLM_p last">Compound <b>35e</b> was prepared according to Method A starting from <b>34e</b> (0.9 g 1.0 mmol). Intermediate <b>21</b> was used as the amino acid. The compound was purified over flash chromatography using hexanes/EtOAc (1:1) affording 0.83 g (0.8 mmol) intermediate <b>35e</b> as white solid (80% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 1031.51, found 1031.75. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, rotamers) δ: 7.79 (d, <i>J</i> = 7.4 Hz, 2H), 7.68–7.62 (m, 2H), 7–44–7.24 (m, 9H), 5.78–5.76 (m, 1H), 5.74–5.43 (m, 2H), 5.12–4.96 (m, 4H), 4.59–4.48 (m, 1H), 4.35–4.15 (m, 4H), 4.07–3.77 (m, 2H), 3.17–3.08 (m, 2H), 2.95 (s, 2H), 2.87 (s, 0.2H), 2.85 (s, 1.6H), 2.83 (s, 1.2H),2.64–2.36 (m, 8H), 2.05 (s, 3H), 1.94–1.45 (m, 6H), 1.42 (s, 6H), 1.33 (s, 3H), 0.87 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ:176.90, 174.37, 173.49, 171.57, 171.37, 160.05, 158.29, 157.88, 145.51, 145.44, 142.87, 142.85, 139.60, 137.69, 134.65, 134.06, 133.71, 133.62, 130.19, 129.76, 129.56, 129.46, 129.01, 128.39, 126.46, 126.34, 126.21, 121.15, 119.98, 118.63, 118.37, 117.79, 87.84, 67.99, 60.45,56.82, 56.09, 55.89, 54.88, 53.09, 51.75, 44.17, 41.80, 35.37, 34.33, 29.85, 28.90, 27.04, 26.34, 23.51, 19.77, 18.58, 12.71, 11.13.</div></div><div id="sec4_9_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> (9<i>H</i>-Fluoren-9-yl)methyl ((6<i>R</i>,9<i>S</i>,12<i>S</i>,15<i>R</i>)-9-Ethyl-6-(4-fluorophenyl)-4,15-dimethyl-5,8,11,14-tetraoxo-12-(3-(3-((2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl)guanidino)propyl)-4,7,10,13-tetraazaoctadeca-1,17-dien-15-yl)carbamate (<b>35f</b>)</h4><div class="NLM_p last">Intermediate <b>35f</b> was prepared according to Method A starting from <b>32f</b>. Intermediate <b>28</b> was used as the peptide carboxylic acid. The crude was purified over flash chromatography using ethyl acetate. White solid (45% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 1049.26, found 1049.66. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 7.80 (d, <i>J</i> = 7.5 Hz, 2H), 7.71–7.61 (m, 2H), 7.46–7.26 (m, 6H), 7.10–6.99 (m, 2H), 5.78 (s, 1H), 5.75–5.49 (m, 2H), 5.11–4.96 (m, 4H), 4.61–4.49 (m, 1H), 4.37–4.26 (m, 1H), 4.26–4.13 (m, 3H), 4.05–3.75 (m, 2H), 3.17–3.10 (m, 2H), 2.96 (s, 2H), 2.87 (s, 1.6H), 2.83 (s, 1.4H), 2.63–2.35 (m, 8H), 2.06 (s, 3H), 1.93–1.46 (m, 6H), 1.42 (s, 6H), 1.32 (s, 3H), 0.87 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 177.06, 176.97, 174.47, 173.57, 173.51, 171.40, 171.17, 165.83, 162.55, 160.02, 158.25, 157.92, 145.46, 145.40, 142.83, 142.81, 139.56, 134.63, 134.03, 133.68, 133.62, 133.57, 131.69, 131.58, 131.49, 129.01, 128.37, 126.43, 126.31, 126.19, 121.16, 120.01, 118.61, 118.30, 117.86, 117.00, 116.94, 116.71, 116.65, 87.81, 68.01, 60.38, 56.85, 55.26, 55.00, 53.06, 51.75, 44.14, 41.68, 35.36, 34.40, 29.73, 28.89, 27.15, 26.29, 26.23, 23.58, 19.80, 18.60, 12.72, 11.21.</div></div><div id="sec4_9_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> (9<i>H</i>-Fluoren-9-yl)methyl ((6<i>R</i>,9<i>S</i>,12<i>S</i>,15<i>R</i>)-6-(4-Chlorophenyl)-9-ethyl-4,15-dimethyl-5,8,11,14-tetraoxo-12-(3-(3-((2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl)guanidino)propyl)-4,7,10,13-tetraazaoctadeca-1,17-dien-15-yl)carbamate (<b>35g</b>)</h4><div class="NLM_p last">Intermediate <b>35g</b> was prepared according to Method A starting from <b>32g</b>. Intermediate <b>28</b> was used as the peptide carboxylic acid. The product was purified over flash chromatography using ethyl acetate. White solid (44% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 1065.47, found 1065.08. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 7.80 (d, <i>J</i> = 7.5 Hz, 2H), 7.69–7.62 (m, 2H), 7.43–7.25 (m, 8H), 5.78 (s, 1H), 5.73–5.50 (m, 2H), 5.12–4.97 (m, 4H), 4.60–4.51 (m, 1H), 4.35–4.27 (m, 1H), 4.25–4.14 (m, 3H), 4.04–3.78 (m, 2H), 3.19–3.09 (m, 2H), 2.96 (s, 2H), 2.90–2.81 (m, 3H), 2.62–2.36 (m, 8H), 2.06 (s, 3H), 1.94–1.46 (m, 6H), 1.42 (s, 6H), 1.32 (s, 3H), 0.88 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 177.03, 174.50, 173.64, 173.59, 171.18, 170.94, 160.04, 158.27, 157.96, 145.47, 145.42, 142.84, 139.58, 137.09, 136.55, 135.55, 135.46, 134.63, 134.06, 134.04, 133.69, 133.65, 133.57, 131.24, 131.16, 130.22, 130.15, 129.02, 128.38, 126.45, 126.31, 126.20, 121.17, 120.03, 118.62, 118.32, 117.92, 87.82, 68.02, 60.38, 56.91, 55.31, 55.03, 53.09, 51.78, 44.15, 41.66, 35.38, 34.43, 29.71, 28.90, 27.16, 26.28, 26.22, 23.58, 19.80, 18.61, 12.73, 11.24.</div></div><div id="sec4_9_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> (9<i>H</i>-Fluoren-9-yl)methyl ((6<i>R</i>,9<i>S</i>,12<i>S</i>,15<i>R</i>)-9-Ethyl-15-methyl-12-(3-((<i>E</i>)-2-methyl-3-((2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl)guanidino)propyl)-5,8,11,14-tetraoxo-6-phenyl-4,7,10,13-tetraazaoctadeca-1,17-dien-15-yl)carbamate (<b>35h</b>)</h4><div class="NLM_p last">Intermediate <b>35h</b> was prepared according to Method A starting from <b>34h</b> (0.33 g 0.35 mmol). Intermediate <b>21</b> was used as the amino acid. The compound was purified over flash chromatography using EtOAc/MeOH (10:0.1) affording 0.29 g (0.28 mmol) of intermediate <b>35h</b> as white solid (85% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 1031.51, found 1031.08. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ:7.79 (d, <i>J</i> = 7.4 Hz, 2H), 7.65 (d, <i>J</i> = 7.4 Hz, 2H), 7.42–7.25 (m, 9H), 5.78–5.60 (m, 2H), 5.40 (s, 1H), 5.12–4.91 (m, 4H), 4.52–4.43 (m, 1H), 4.36–4.15 (m, 4H), 3.84–3.64 (m, 2H), 3.20–3.09 (m, 2H), 2.95 (s, 2H), 2.73 (s, 3H), 2.67–2.40 (m, 8H), 2.05 (s, 3H), 1.95–1.46 (m, 6H), 1.42 (s, 6H), 1.35 (s, 3H), 0.89 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 174.56, 174.06, 172.31, 160.00, 158.02, 157.57, 145.42, 145.38, 142.85, 139.46, 138.78, 135.33, 134.83, 134.02, 133.58, 129.94, 129.51, 129.08, 129.04, 128.39, 126.40, 126.35, 126.22, 121.18, 120.04, 118.65, 116.32, 87.83, 68.08, 60.42, 59.35, 57.02, 44.16, 42.91, 42.10, 41.87, 28.91, 28.70, 25.97, 23.58, 19.86, 18.66, 12.72, 11.16.</div></div><div id="sec4_9_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> (<i>R</i>)-<i>N</i>-((6<i>S</i>,9<i>S</i>,12<i>R</i>)-9-Ethyl-1-imino-7,10,13-trioxo-1-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-12-phenyl-2,8,11,14-tetraazaheptadec-16-en-6-yl)-2-isobutyramido-2-methylpent-4-enamide (<b>36a</b>)</h4><div class="NLM_p last">Intermediate <b>35a</b> (0.43 mmol, 0.44 g) was dissolved in 50 mL of CH<sub>3</sub>CN and treated with 20 equiv of diethylamine (8.6 mmol, 0.63 mL) at room temperature for 2 h. The solvent and diethylamine were removed in vacuo, and the resulting crude was mixed with isobutyryl chloride (1.9 mmol, 0.2 mL) and DIPEA (1.9 mmol, 0.33 mL) in CH<sub>2</sub>Cl<sub>2</sub> at room temperature for 2 h. The reaction mixture was evaporated, and the remaining crude was purified over flash chromatography using EtAc/MeOH (50:0.7) yielding 0.23 g of intermediate (<b>36a</b>) as white solid (62% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>44</sub>H<sub>65</sub>N<sub>8</sub>O<sub>8</sub>S [M + H]<sup>+</sup> 865.4641, found 865.4642. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.45–7.28 (m, 5H), 5.84–5.64 (m, 2H), 5.42 (s, 1H), 5.14–4.99 (m, 4H), 4.26–4.19 (m, 2H), 3.89–3.69 (m, 2H), 3.15 (t, <i>J</i> = 6.7 Hz, 2H), 2.99 (s, 2H), 2.74–2.65 (m, 1H), 2.60–2.40 (m, 8H), 2.07 (s, 3H), 1.98–1.48 (m, 6H), 1.45 (s, 6H), 1.37 (s, 3H), 1.13–1.06 (m, 6H), 0.92 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD): δ 180.33, 176.96, 174.56, 174.22, 172.49, 160.08, 158.33, 139.58, 138.59, 135.36, 134.61, 134.16, 133.72, 129.93, 129.55, 129.21, 126.22, 119.95, 118.64, 116.31, 87.87, 59.99, 59.54, 56.93, 55.03, 44.19, 42.92, 41.24, 36.15, 30.01, 28.91, 25.96, 23.42, 20.43, 20.03, 19.84, 19.75, 18.57, 12.69, 11.25.</div></div><div id="sec4_9_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> (<i>R</i>)-<i>N</i>-((6<i>S</i>,9<i>S</i>,12<i>R</i>)-9-Ethyl-1-imino-7,10,13-trioxo-1-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-12-phenyl-2,8,11,14-tetraazaoctadec-17-en-6-yl)-2-isobutyramido-2-methylpent-4-enamide (<b>36b</b>)</h4><div class="NLM_p last">Intermediate <b>36b</b> was prepared according to Method A starting from <b>32b</b>. Intermediate <b>26</b> was used as the peptide carboxylic acid. The crude was purified over flash chromatography using EtOAc/MeOH (30:1). White solid (72% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 879.48, found 879.42. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.44–7.23 (m, 5H), 5.80–5.63 (m, 2H), 5.40 (s, 1H), 5.15–4.91 (m, 4H), 4.27–4.17 (m, 2H), 3.28–3.09 (m, 4H), 2.99 (s, 2H), 2.76–2.65 (m, 1H), 2.61–2.40 (m, 8H), 2.21(q, <i>J</i> = 6.9 Hz, 2H), 2.07 (s, 3H), 1.97–1.48 (m, 6H), 1.44 (s, 6H), 1.37 (s, 3H), 1.15–1.04 (m, 6H),0.92 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.29, 176.98, 174.53, 174.16, 172.47, 160.03, 158.24, 139.55, 138.53, 136.56, 134.55, 134.14, 133.67, 129.86, 129.48, 129.21, 126.18, 119.99, 118.60, 117.44, 87.83, 59.89, 59.47, 56.90, 55.09, 44.16, 41.66, 41.17, 40.27, 36.08,34.86, 29.88, 28.94, 27.12, 25.95, 23.42, 20.51, 19.83, 18.64, 12.76, 11.32.</div></div><div id="sec4_9_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (9<i>H</i>-Fluoren-9-yl)methyl ((4<i>R</i>,7<i>S</i>,10<i>S</i>,13<i>R</i>)-10-Ethyl-4-methyl-5,8,11,14-tetraoxo-7-(3-(3-((2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-yl)sulfonyl)guanidino)propyl)-13-phenyl-6,9,12,15-tetraazaicosa-1,19-dien-4-yl)carbamate (<b>36c</b>)</h4><div class="NLM_p last">Intermediate <b>36c</b> was prepared according to Method A starting from <b>32c</b>. Intermediate <b>26</b> was used as the peptide carboxylic acid. The crude was purified over flash chromatography using EtOAc/MeOH (25:4). White solid (80% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 893.50, found 893.62. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.43–7.26 (m, 5H), 5.84–5.64 (m, 2H), 5.38 (s, 1H), 5.14–4.91 (m, 4H), 4.26–4.18 (m, 2H), 3.28–3.06 (m, 4H), 2.99 (s, 2H), 2.75–2.64 (m, 1H), 2.61–2.41 (m, 8H), 2.08 (s, 3H), 2.05–1.48 (m, 10H), 1.45 (s, 6H), 1.37 (s, 3H), 1.14–1.06 (m, 6H),0.92 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.31, 176.94, 174.56, 174.18, 172.50, 160.08, 158.33, 139.59, 139.24, 138.69, 134.61, 134.14, 133.72, 129.91, 129.51, 129.18, 126.22, 119.96, 118.64, 115.72, 87.87, 59.99, 59.51, 56.96, 50.55, 44.20, 41.28, 40.34, 36.15, 32.24, 30.00, 29.92, 28.91, 27.04, 25.97, 23.40, 20.45, 19.85, 19.76, 18.58, 12.69, 11.26.</div></div><div id="sec4_9_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> (<i>R</i>)-<i>N</i>-((6<i>S</i>,9<i>S</i>,12<i>R</i>)-9-Ethyl-1-imino-7,10,13-trioxo-1-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-12-phenyl-2,8,11,14-tetraazahenicos-20-en-6-yl)-2-isobutyramido-2-methylpent-4-enamide (<b>36d</b>)</h4><div class="NLM_p last">Intermediate <b>36d</b> was prepared according to Method A starting from <b>32d</b>. Intermediate <b>26</b> was used as the peptide carboxylic acid. The crude was purified over flash chromatography using EtOAc/MeOH (40:1) to afford <b>36d</b> as white solid (71% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 921.53, found 921.42. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.43–7.28 (m, 5H), 5.83–5.64 (m, 2H), 5.39 (s, 1H), 5.14–4.90 (m, 4H), 4.25–4.19 (m, 2H), 3.26–3.07 (m, 4H), 2.99 (s, 2H), 2.74–2.65 (m, 1H), 2.58–2.41 (m, 8H), 2.07 (s, 3H), 2.04–1.46 (m, 10H), 1.45 (s, 6H), 1.37 (s, 3H), 1.35–1.21 (m, 4H), 1.13–1.08 (m, 6H), 0.92 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.32, 177.00, 174.56, 174.15, 172.45, 160.06, 158.29, 140.09, 139.57, 138.66, 134.57, 134.16, 133.70, 129.89, 129.49, 129.18, 126.20, 119.98, 118.62, 115.10, 87.85, 59.93, 59.50, 56.90, 50.10, 44.18, 41.70, 41.22, 40.73, 36.12, 34.97, 30.46, 29.94, 28.92, 27.54, 27.08, 25.97, 23.40, 20.49, 19.83, 19.79, 18.61, 12.72, 11.30.</div></div><div id="sec4_9_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> (<i>R</i>)-<i>N</i>-((6<i>S</i>,9<i>S</i>,12<i>R</i>)-9-Ethyl-1-imino-14-methyl-7,10,13-trioxo-1-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-12-phenyl-2,8,11,14-tetraazaheptadec-16-en-6-yl)-2-isobutyramido-2-methylpent-4-enamide (<b>36e</b>)</h4><div class="NLM_p last">Intermediate <b>36e</b> was prepared starting from <b>35e</b> according to the procedure used for <b>36a</b>. The reaction mixture was concentrated under vacuum, and the remaining crude was purified over flash chromatography using EtAc/MeOH (20:1) yielding 0.19 g intermediate <b>36e</b> as white solid (45% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 879.48, found 879.78. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 7.44–7.27 (m, 5H), 5.80 (s, 1H), 5.76–5.47 (m, 2H), 5.16–4.99 (m, 4H), 4.31–4.18 (m, 2H), 4.07–3.81 (m, 2H), 3.19–3.10 (m, 2H), 3.00 (s, 2H), 2.90 (s, 1.6H), 2.87 (s, 1.4H),2.69–2.38 (m, 9H), 2.08 (s, 3H), 1.92–1.47 (m, 6H) 1.45 (s, 6H), 1.36 (s, 3H), 1.09 (d, <i>J</i> = 6.8 Hz, 6H), 0.90 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 180.00, 176.66, 176.55, 174.35, 174.29, 173.66, 171.65, 171.46, 160.06, 158.31, 139.59, 138.21, 137.63, 134.65, 134.19, 133.72, 133.64, 130.18, 130.14, 129.79, 129.70, 129.62, 129.52, 126.20, 119.87, 118.62, 118.39, 117.78, 87.86, 60.06, 56.84, 56.16, 55.93, 54.84, 54.73, 53.15, 51.81, 44.19, 41.76, 41.30, 36.15, 35.42, 34.37, 30.20, 28.91, 26.81, 26.36, 26.27, 23.36, 20.41, 20.38, 19.85, 19.77, 18.59, 12.70, 11.24.</div></div><div id="sec4_9_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> (<i>R</i>)-<i>N</i>-((6<i>S</i>,9<i>S</i>,12<i>R</i>)-9-Ethyl-12-(4-fluorophenyl)-1-imino-14-methyl-7,10,13-trioxo-1-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-2,8,11,14-tetraazaheptadec-16-en-6-yl)-2-isobutyramido-2-methylpent-4-enamide (<b>36f</b>)</h4><div class="NLM_p last">Intermediate <b>36f</b> was prepared starting from <b>35f</b> according to the procedure used for <b>36a</b>. The compound was purified over flash chromatography using ethyl acetate/MeOH (15:1). White solid (58% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 897.47, found 897.76. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 7.47–7.38 (m, 2H), 7.11–7.01 (m, 2H), 5.81 (s, 1H), 5.77–5.61 (m, 2H), 5.16–5.01 (m, 4H), 4.27–4.17 (m, 2H), 4.09–3.81 (m, 2H), 3.21–3.11 (m, 2H), 3.00 (s, 2H), 2.92 (s, 1.6H), 2.87 (s, 1.4H), 2.68–2.37 (m, 9H), 2.08 (s, 3H), 1.93–1.43 (m, 12H), 1.35 (s, 3H), 1.09 (d, <i>J</i> = 6.8 Hz, 6H), 0.90 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 180.05, 176.88, 176.73, 174.45, 174.37, 173.74, 171.50, 171.30, 165.89, 162.62, 160.06, 158.31, 139.57, 134.62, 134.23, 134.18, 133.83, 133.70, 133.62, 131.78, 131.68, 131.57, 126.20, 119.89, 118.62, 118.35, 117.85, 116.99, 116.89, 116.70, 116.60, 87.85, 59.98, 59.96, 56.96, 56.92, 55.35, 55.06, 53.16, 51.83, 44.18, 41.73, 41.10, 36.12, 35.41, 34.43, 30.04, 28.91, 26.94, 26.33, 26.22, 23.42, 20.48, 20.45, 19.82, 19.77, 18.59, 12.70, 11.35, 11.28.</div></div><div id="sec4_9_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> (<i>R</i>)-<i>N</i>-((6<i>S</i>,9<i>S</i>,12<i>R</i>)-12-(4-Chlorophenyl)-9-ethyl-1-imino-14-methyl-7,10,13-trioxo-1-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-2,8,11,14-tetraazaheptadec-16-en-6-yl)-2-isobutyramido-2-methylpent-4-enamide (<b>36g</b>)</h4><div class="NLM_p last">Intermediate <b>36g</b> was prepared starting from <b>35g</b> according to the procedure used for <b>36a</b>. The compound was purified over flash chromatography using ethyl acetate/MeOH (70:1). White solid (76% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 913.44, found 913.42. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD, 2-rotamers) δ: 7.45–7.29 (m, 4H), 5.81 (s, 1H), 5.77–5.62 (m, 2H), 5.17–5.02 (m, 4H), 4.27–4.16 (m, 2H), 4.09–3.87 (m, 2H), 3.19–3.12 (m, 2H), 3.00 (s, 2H), 2.92 (s, 1.7H), 2.87 (s, 1.3H), 2.67–2.38 (m, 9H), 2.08 (s, 3H), 1.92–1.43 (m, 12H), 1.35 (m, 3H), 1.09 (d, <i>J</i> = 6.9 Hz, 6H), 0.91 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD, rotamers) δ: 180.05, 176.98, 176.79, 174.52, 174.43, 173.77, 171.25, 171.04, 160.03, 158.26, 139.54, 137.07, 136.47, 135.53, 135.40, 134.60, 134.20, 134.15, 133.82, 133.67, 133.57, 131.30, 131.22, 130.18, 130.08, 126.18, 119.94, 118.60, 118.40, 117.94, 87.83, 59.93, 57.00, 55.38, 55.23, 55.05, 53.17, 51.84, 44.17, 41.70, 41.04, 40.92, 36.08, 35.43, 34.46, 29.93, 29.86, 28.93, 27.00, 26.27, 26.14, 23.43, 20.52, 20.47, 19.80, 18.61, 12.74, 11.42, 11.34.</div></div><div id="sec4_9_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (<i>R</i>)-<i>N</i>-((8<i>S</i>,11<i>S</i>,14<i>R</i>,<i>E</i>)-11-Ethyl-9,12,15-trioxo-3-(2,2,4,6,7-pentamethyl-2,3-dihydrobenzofuran-5-sulfonamido)-14-phenyl-2,4,10,13,16-pentaazanonadeca-2,18-dien-8-yl)-2-isobutyramido-2-methylpent-4-enamide (<b>36h</b>)</h4><div class="NLM_p last">Intermediate <b>36h</b> was prepared starting from <b>35h</b> according to the procedure used for <b>36a</b>. The compound was purified over flash chromatography using EtOAc/MeOH (250:4). White solid (59% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 879.48, found 879.50. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.45–7.40 (m, 2H), 7.37–7.25 (m, 3H), 5.85–5.64 (m, 2H), 5.45 (s, 1H), 5.14–4.99 (m, 4H), 4.27–4.20 (m, 2H), 3.90–3.69 (m, 2H), 3.21–3.12 (m, 2H), 2.98 (s, 2H), 2.75 (s, 3H), 2.73–2.2.65 (m, 1H), 2.58 (s, 3H), 2.56–2.41 (m, 5H), 2.07 (s, 3H), 2.00–1.49 (m, 6H), 1.44 (s, 6H), 1.37 (s, 3H), 1.14–1.06 (m, 6H), 0.93 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ:180.28, 176.97, 174.49, 174.12, 172.41, 159.96, 157.51, 139.39, 138.53, 135.33, 134.83, 134.15, 133.52, 129.88, 129.49, 129.17, 126.17, 119.95, 118.60, 116.28, 87.81, 59.89, 59.50, 56.89, 55.01, 44.15, 42.86, 42.07, 41.14, 36.07, 29.88, 28.94, 28.72, 27.11, 25.93, 23.46, 20.49, 19.87, 19.82, 18.67, 12.74, 11.30.</div></div><div id="sec4_9_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> (<i>R</i>)-<i>N</i>-((8<i>S</i>,11<i>S</i>,14<i>R</i>,<i>Z</i>)-11-ethyl-3-(methylamino)-9,12,15-trioxo-14-phenyl-2,4,10,13,16-pentaazanonadeca-2,18-dien-8-yl)-2-isobutyramido-2-methylpent-4-enamide (<b>36i</b>)</h4><div class="NLM_p last">Intermediate <b>36i</b> was prepared according to Method A starting from <b>32a</b>. Intermediate <b>30</b> was used as the peptide carboxylic acid. The crude was purified with preparative HPLC using the C18 reverse phase column (Waters, Sunfire Prep C<sub>18</sub> OBD, 5 μm, 50 mm × 100 mm). White solid (44% yield). MS (ESI): <i>m</i>/<i>z</i> calcd for [M + H]<sup>+</sup> 641.41, found 641.42. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ: 7.45–7.28 (m, 5H), 5.87–5.65 (m, 2H), 5.42 (s, 1H), 5.16–5.01 (m, 4H), 4.27–4.16 (m, 2H), 4.32 (dd, <i>J</i> = 5.0, 7.6 Hz, 1H), 4.25 (dd, <i>J</i> = 5.7, 8.8 Hz, 1H), 3.90–3.72 (m, 2H), 3.20 (t, <i>J</i> = 7.0 Hz, 2H), 2.83 (s, 6H), 2.74–2.64 (m, 1H), 2.60–2.41 (m, 2H), 1.99–1.57 (m, 6H), 1.39 (m, 3H), 1.15–1.07 (m, 6H), 0.93 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD) δ: 180.33, 176.81, 174.35, 174.13, 172.46, 157.45, 138.61, 135.35, 133.98, 129.94, 129.58, 129.15, 119.97, 116.33, 60.05, 59.51, 56.86, 54.42, 42.92, 42.13, 41.69, 36.07, 30.19, 28.50, 26.17, 26.04, 23.19, 20.38, 19.79, 11.12.</div></div><div id="sec4_9_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> (<i>R</i>)-2-((((9<i>H</i>-fluoren-9-yl)methoxy)carbonyl)amino)-4-(dibenzylamino)-2-methylbutanoic Acid CF<sub>3</sub>COOH Salt (<b>40</b>)</h4><div class="NLM_p last">(<i>R</i>)-2-Fmoc-NH-2-methylpent-4-enoic acid (1 mmol, 350 mg) was dissolved in THF/H<sub>2</sub>O (2.6 mL/1 mL) in a flask. The hood lights were turned off, and the flask was covered with aluminum foil. Osmium tetroxide (0.05 mmol in 0.32 mL of H<sub>2</sub>O) was added to the flask. After 5 min, NaIO<sub>4</sub> (2.5 mmol, 535 mg) was added in small portions over a 15 min period. The reaction was kept at room temperature for 4 h before it was filtered, and the THF was evaporated off. The residue was dissolved in EtOAc, and saturated NH<sub>4</sub>Cl aqueous solution was added. The aqueous phase was extracted with EtOAc (2 × 30 mL) and the organic phase was combined, washed with brine, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solution was concentrated in vacuo, and the crude product was dissolved in 1,2-dichloroethane (3 mL) in a flask. Then dibenzylamine (1.5 mmol, 0.29 mL) and sodium triacetoxyborohydride (3 mmol, 636 mg) were added to the flask. The reaction was stirred at room temperature until the starting material disappeared on TLC. The solvent was evaporated, and the remaining crude was purified with C-18 reverse phase flash column to yield <b>40</b>. White solid (75% yield over 2 steps). MS (ESI): <i>m</i>/<i>z</i> calculated for C<sub>34</sub>H<sub>35</sub>N<sub>2</sub>O<sub>4</sub> [M + H]<sup>+</sup> 535.26, found 535.23. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.80 (d, <i>J</i> = 7.6 Hz, 2H), 7.62 (dd, <i>J</i> = 15.7, 7.5 Hz, 2H), 7.48–7.35 (m, 12H), 7.33–7.25 (m, 2H), 4.43–4.05 (m, 7H), 3.25–3.03 (m, 2H), 2.55 (q, <i>J</i> = 8.3, 6.5 Hz, 1H), 2.46–2.20 (m, 1H), 1.35 (s, 3H). <sup>13</sup>C NMR (101 MHz, MeOD) δ 176.68, 157.48, 145.26, 145.07, 142.58, 142.56, 132.20, 131.24, 130.45, 130.44, 128.86, 128.19, 126.29, 126.14, 120.98, 67.91, 58.81, 58.40, 48.28, 31.28, 24.16.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i86"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01796">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27181" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27181" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01796" class="ext-link">10.1021/acs.jmedchem.6b01796</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Crystallography data collection and refinement statistics, inhibitory activity of compound <b>18</b> against SET1 family histone methyltransferases, stability test of compound <b>18</b> in cell culture, and experimental methods for HMT inhibition assay of compound <b>18</b> for other SET1 family members (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Binding model of WDR5 with compound <b>13</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_001.pdf">jm6b01796_si_001.pdf (466.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_002.pdb">jm6b01796_si_002.pdb (187.3 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_003.csv">jm6b01796_si_003.csv (2.81 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the cocrystal structure for compounds <b>16</b> and <b>18</b> in complex with WDR5 have been deposited into PDB (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB">4GMB</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC">5VFC</a>).</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01796" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87676" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87676" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9fecf7fef0f2faf1f8dfeaf2f6fcf7b1fafbea"><span class="__cf_email__" data-cfemail="b7c4dfd6d8dad2d9d0f7c2daded4df99d2d3c2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hacer Karatas</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yangbing Li</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liu Liu</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiao Ji</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shirley Lee</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Chen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiuling Yang</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liyue Huang</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Denzil Bernard</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Xu</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth C. Townsend</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fang Cao</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xu Ran</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoqin Li</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bo Wen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duxin Sun</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeanne A Stuckey</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ming Lei</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yali Dou</span> - <span class="hlFld-Affiliation affiliation">†Department
of Medicinal Chemistry, ‡Department of Internal Medicine, §Comprehensive Cancer
Center, ∥Department of Pathology, ⊥Howard Hughes Medical Institute, #Department of Biological Chemistry, ∇Department of Pharmaceutical
Sciences, ○Department of Pharmacology, and ¶Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>H.K., Y.L., L.L., J.J., and S.L. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Multiple patents have been filed by University of Michigan on these MLL-WDR5 inhibitors, which have been licensed by Ascentage Pharma Group. Shaomeng Wang, Hacer Karatas, Yali Dou, Yangbing Li, Liu Liu, and Denzil Bernard are inventors of these MLL-WDR5 inhibitors reported in this manuscript. Shaomeng Wang is a co-founder of Ascentage, owns stock in Ascentage, and serves as a consultant for Ascentage. The University of Michigan and Shaomeng Wang have also received a research contract from Ascentage.<br /></br></div></li></ul></div><div class="ack" id="ACK-d226e7644-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i89">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78253" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78253" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful for the financial support from the National Cancer Institute, National Institutes of Health (NIH) (Grant R01 CA177307 to S. Wang and Y. Dou), Leukemia and Lymphoma Society (to S. Wang), Leukemia and Lymphoma Society Scholar grant (to Y. Dou), and Ascentage Pharma Group (to S. Wang). Use of the Advanced Photon Source, an Office of Science User Facility operated for the U.S. Department of Energy (DOE) Office of Science by Argonne National Laboratory, was supported by the U.S. DOE under Contract No. DE-AC02-06CH11357. Use of the LS-CAT Sector 21 was supported by the Michigan Economic Development Corporation and the Michigan Technology Tri-Corridor (Grant 085P1000817).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i90" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i90"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i91" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i91"> Abbreviations</h2><tr><td class="NLM_term">WDR5</td><td class="NLM_def"><p class="first last">WD repeat domain 5 protein</p></td></tr><tr><td class="NLM_term">MLL</td><td class="NLM_def"><p class="first last">mixed lineage leukemia</p></td></tr><tr><td class="NLM_term">MLL1–4</td><td class="NLM_def"><p class="first last">mixed lineage leukemia 1–4</p></td></tr><tr><td class="NLM_term">H3K4</td><td class="NLM_def"><p class="first last">histone 3 lysine 4</p></td></tr><tr><td class="NLM_term">ALL</td><td class="NLM_def"><p class="first last">acute lymphoid leukemia</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">HMT</td><td class="NLM_def"><p class="first last">histone methyltransferase</p></td></tr><tr><td class="NLM_term">Fmoc-OSu</td><td class="NLM_def"><p class="first last">9-fluorenylmethyl <i>N</i>-succinimidyl carbonate</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">Pbf</td><td class="NLM_def"><p class="first last">2,2,4,6,7-pentamethyldihydrobenzofuran-5-sulfonyl</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-(dimethylamino)propyl)carbodiimide</p></td></tr><tr><td class="NLM_term">HOAt</td><td class="NLM_def"><p class="first last">1-hydroxy-7-azabenzotriazole</p></td></tr><tr><td class="NLM_term">DEA</td><td class="NLM_def"><p class="first last">diethylamine</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">TMS</td><td class="NLM_def"><p class="first last">tetramethylsilane</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42437" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42437" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 32 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Hess, J. L.</span><span> </span><span class="NLM_article-title">MLL: A histone methyltransferase disrupted in leukemia</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">500</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.molmed.2004.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.molmed.2004.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=15464450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotF2is7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=500-507&author=J.+L.+Hess&title=MLL%3A+A+histone+methyltransferase+disrupted+in+leukemia&doi=10.1016%2Fj.molmed.2004.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MLL: a histone methyltransferase disrupted in leukemia</span></div><div class="casAuthors">Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">500-507</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Rearrangements of the MLL gene, which is located at chromosome 11q23, are assocd. with aggressive acute leukemias in both children and adults.  MLL regulates Hox gene expression through direct promoter binding and histone modification.  MLL rearrangements occurring in leukemia include MLL fusion genes, partial tandem duplications of MLL and MLL amplification.  MLL fusions and amplification upregulate Hox expression, apparently resulting in a block of hematopoietic differentiation.  Future therapies for MLL-assocd. leukemia might involve blocking Hox gene upregulation by using fusion proteins or inhibiting the activity of Hox proteins themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom2pQrErnZf7Vg90H21EOLACvtfcHk0lg8b8qhiy_efA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotF2is7o%253D&md5=ae9d59e5e59cc017fd0fcf479a4a618d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2004.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2004.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%253A%2520A%2520histone%2520methyltransferase%2520disrupted%2520in%2520leukemia%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2004%26volume%3D10%26spage%3D500%26epage%3D507%26doi%3D10.1016%2Fj.molmed.2004.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Jude, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Climer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artinger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, P.</span><span> </span><span class="NLM_article-title">Unique and independent roles for MLL in adult hematopoietic stem cells and progenitors</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">324</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.1016/j.stem.2007.05.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.stem.2007.05.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=18371366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagu7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=324-337&author=C.+D.+Judeauthor=L.+Climerauthor=D.+Xuauthor=E.+Artingerauthor=J.+K.+Fisherauthor=P.+Ernst&title=Unique+and+independent+roles+for+MLL+in+adult+hematopoietic+stem+cells+and+progenitors&doi=10.1016%2Fj.stem.2007.05.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Unique and independent roles for MLL in adult hematopoietic stem cells and progenitors</span></div><div class="casAuthors">Jude, Craig D.; Climer, Leslie; Xu, Diyong; Artinger, Erika; Fisher, Jill K.; Ernst, Patricia</div><div class="citationInfo"><span class="NLM_cas:title">Cell Stem Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">324-337</span>CODEN:
                <span class="NLM_cas:coden">CSCEC4</span>;
        ISSN:<span class="NLM_cas:issn">1934-5909</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Mixed Lineage Leukemia (MLL) gene is essential for embryonic hematopoietic stem cell (HSC) development, but its role during adult hematopoiesis is unknown.  Using an inducible knockout model, we demonstrate that Mll is essential for the maintenance of adult HSCs and progenitors, with fatal bone marrow failure occurring within 3 wk of Mll deletion.  Mll-deficient cells are selectively lost from mixed bone marrow chimeras, demonstrating their failure to self-renew even in an intact bone marrow environment.  Surprisingly, HSCs lacking Mll exhibit ectopic cell-cycle entry, resulting in the depletion of quiescent HSCs.  In contrast, Mll deletion in myelo-erythroid progenitors results in reduced proliferation and reduced response to cytokine-induced cell-cycle entry.  Committed lymphoid and myeloid cells no longer require Mll, defining the early multi-potent stages of hematopoiesis as Mll dependent.  These studies demonstrate that Mll plays selective and independent roles within the hematopoietic system, maintaining quiescence in HSCs and promoting proliferation in progenitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqiXPklTPaJLVg90H21EOLACvtfcHk0lg8b8qhiy_efA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagu7vO&md5=e8ce8a59ef3ecda7ac594d2fdd6eb41e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.stem.2007.05.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stem.2007.05.019%26sid%3Dliteratum%253Aachs%26aulast%3DJude%26aufirst%3DC.%2BD.%26aulast%3DClimer%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DArtinger%26aufirst%3DE.%26aulast%3DFisher%26aufirst%3DJ.%2BK.%26aulast%3DErnst%26aufirst%3DP.%26atitle%3DUnique%2520and%2520independent%2520roles%2520for%2520MLL%2520in%2520adult%2520hematopoietic%2520stem%2520cells%2520and%2520progenitors%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2007%26volume%3D1%26spage%3D324%26epage%3D337%26doi%3D10.1016%2Fj.stem.2007.05.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Meyer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burmeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gröger, D.</span><span class="NLM_etal"></span><span> </span><span class="NLM_article-title">The MLL recombinome of acute leukemias in 2013</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">2165</span><span class="NLM_x">–</span> <span class="NLM_lpage">2176</span><span class="refDoi"> DOI: 10.1038/leu.2013.135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fleu.2013.135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=23628958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslaqs73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=2165-2176&author=C.+Meyerauthor=J.+Hofmannauthor=T.+Burmeisterauthor=D.+Gr%C3%B6ger&title=The+MLL+recombinome+of+acute+leukemias+in+2013&doi=10.1038%2Fleu.2013.135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The MLL recombinome of acute leukemias in 2013</span></div><div class="casAuthors">Meyer, C.; Hofmann, J.; Burmeister, T.; Groeger, D.; Park, T. S.; Emerenciano, M.; Pombo de Oliveira, M.; Renneville, A.; Villarese, P.; Macintyre, E.; Cave, H.; Clappier, E.; Mass-Malo, K.; Zuna, J.; Trka, J.; De Braekeleer, E.; De Braekeleer, M.; Oh, S. H.; Tsaur, G.; Fechina, L.; van der Velden, V. H. J.; van Dongen, J. J. M.; Delabesse, E.; Binato, R.; Silva, M. L. M.; Kustanovich, A.; Aleinikova, O.; Harris, M. H.; Lund-Aho, T.; Juvonen, V.; Heidenreich, O.; Vormoor, J.; Choi, W. W. L.; Jarosova, M.; Kolenova, A.; Bueno, C.; Menendez, P.; Wehner, S.; Eckert, C.; Talmant, P.; Tondeur, S.; Lippert, E.; Launay, E.; Henry, C.; Ballerini, P.; Lapillone, H.; Callanan, M. B.; Cayuela, J. M.; Herbaux, C.; Cazzaniga, G.; Kakadiya, P. M.; Bohlander, S.; Ahlmann, M.; Choi, J. R.; Gameiro, P.; Lee, D. S.; Krauter, J.; Cornillet-Lefebvre, P.; Te Kronnie, G.; Schaefer, B. W.; Kubetzko, S.; Alonso, C. N.; zur Stadt, U.; Sutton, R.; Venn, N. C.; Izraeli, S.; Trakhtenbrot, L.; Madsen, H. O.; Archer, P.; Hancock, J.; Cerveira, N.; Teixeira, M. R.; Lo Nigro, L.; Moericke, A.; Stanulla, M.; Schrappe, M.; Sedek, L.; Szczepanski, T.; Zwaan, C. M.; Coenen, E. A.; van den Heuvel-Eibrink, M. M.; Strehl, S.; Dworzak, M.; Panzer-Gruemayer, R.; Dingermann, T.; Klingebiel, T.; Marschalek, R.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2165-2176</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chromosomal rearrangements of the human MLL (mixed lineage leukemia) gene are assocd. with high-risk infant, pediatric, adult and therapy-induced acute leukemias.  We used long-distance inverse-polymerase chain reaction to characterize the chromosomal rearrangement of individual acute leukemia patients.  We present data of the mol. characterization of 1590 MLL-rearranged biopsy samples obtained from acute leukemia patients.  The precise localization of genomic breakpoints within the MLL gene and the involved translocation partner genes (TPGs) were detd. and novel TPGs identified.  All patients were classified according to their gender (852 females and 745 males), age at diagnosis (558 infant, 416 pediatric and 616 adult leukemia patients) and other clin. criteria.  Combined data of our study and recently published data revealed a total of 121 different MLL rearrangements, of which 79 TPGs are now characterized at the mol. level.  However, only seven rearrangements seem to be predominantly assocd. with illegitimate recombinations of the MLL gene (∼90%): AFF1/AF4, MLLT3/AF9, MLLT1/ENL, MLLT10/AF10, ELL, partial tandem duplications (MLL PTDs) and MLLT4/AF6, resp.  The MLL breakpoint distributions for all clin. relevant subtypes (gender, disease type, age at diagnosis, reciprocal, complex and therapy-induced translocations) are presented.  Finally, we present the extending network of reciprocal MLL fusions deriving from complex rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoepBrGUCQUZrVg90H21EOLACvtfcHk0ljmgVrh32BW-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslaqs73M&md5=a82f47327e5932e0edc20a098a5d1a1d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fleu.2013.135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2013.135%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DHofmann%26aufirst%3DJ.%26aulast%3DBurmeister%26aufirst%3DT.%26aulast%3DGr%25C3%25B6ger%26aufirst%3DD.%26atitle%3DThe%2520MLL%2520recombinome%2520of%2520acute%2520leukemias%2520in%25202013%26jtitle%3DLeukemia%26date%3D2013%26volume%3D27%26spage%3D2165%26epage%3D2176%26doi%3D10.1038%2Fleu.2013.135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Borkhardt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wuchter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viehmann, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pils, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teigler-Schlegel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanulla, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmermann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludwig, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janka-Schaub, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrappe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harbott, J.</span><span> </span><span class="NLM_article-title">Infant acute lymphoblastic leukemia - combined cytogenetic, immunophenotypical and molecular analysis of 77 cases</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1685</span><span class="NLM_x">–</span> <span class="NLM_lpage">1690</span><span class="refDoi"> DOI: 10.1038/sj.leu.2402595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fsj.leu.2402595" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1685-1690&author=A.+Borkhardtauthor=C.+Wuchterauthor=S.+Viehmannauthor=S.+Pilsauthor=A.+Teigler-Schlegelauthor=M.+Stanullaauthor=M.+Zimmermannauthor=W.+D.+Ludwigauthor=G.+Janka-Schaubauthor=M.+Schrappeauthor=J.+Harbott&title=Infant+acute+lymphoblastic+leukemia+-+combined+cytogenetic%2C+immunophenotypical+and+molecular+analysis+of+77+cases&doi=10.1038%2Fsj.leu.2402595"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2402595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2402595%26sid%3Dliteratum%253Aachs%26aulast%3DBorkhardt%26aufirst%3DA.%26aulast%3DWuchter%26aufirst%3DC.%26aulast%3DViehmann%26aufirst%3DS.%26aulast%3DPils%26aufirst%3DS.%26aulast%3DTeigler-Schlegel%26aufirst%3DA.%26aulast%3DStanulla%26aufirst%3DM.%26aulast%3DZimmermann%26aufirst%3DM.%26aulast%3DLudwig%26aufirst%3DW.%2BD.%26aulast%3DJanka-Schaub%26aufirst%3DG.%26aulast%3DSchrappe%26aufirst%3DM.%26aulast%3DHarbott%26aufirst%3DJ.%26atitle%3DInfant%2520acute%2520lymphoblastic%2520leukemia%2520-%2520combined%2520cytogenetic%252C%2520immunophenotypical%2520and%2520molecular%2520analysis%2520of%252077%2520cases%26jtitle%3DLeukemia%26date%3D2002%26volume%3D16%26spage%3D1685%26epage%3D1690%26doi%3D10.1038%2Fsj.leu.2402595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Felix, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessells, J. M.</span><span> </span><span class="NLM_article-title">Pediatric Acute Lymphoblastic Leukemia: Challenges and Controversies in 2000</span> <span class="citation_source-journal">Hematology Am. Soc. Hematol. Educ. Program</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">2000</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1182/asheducation-2000.1.285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1182%2Fasheducation-2000.1.285" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2000&publication_year=2000&pages=285-302&author=C.+A.+Felixauthor=B.+J.+Langeauthor=J.+M.+Chessells&title=Pediatric+Acute+Lymphoblastic+Leukemia%3A+Challenges+and+Controversies+in+2000&doi=10.1182%2Fasheducation-2000.1.285"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1182%2Fasheducation-2000.1.285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fasheducation-2000.1.285%26sid%3Dliteratum%253Aachs%26aulast%3DFelix%26aufirst%3DC.%2BA.%26aulast%3DLange%26aufirst%3DB.%2BJ.%26aulast%3DChessells%26aufirst%3DJ.%2BM.%26atitle%3DPediatric%2520Acute%2520Lymphoblastic%2520Leukemia%253A%2520Challenges%2520and%2520Controversies%2520in%25202000%26jtitle%3DHematology%2520Am.%2520Soc.%2520Hematol.%2520Educ.%2520Program%26date%3D2000%26volume%3D2000%26spage%3D285%26epage%3D302%26doi%3D10.1182%2Fasheducation-2000.1.285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Krivtsov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span> </span><span class="NLM_article-title">MLL translocations, histone modifications and leukaemia stem-cell development</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">823</span><span class="NLM_x">–</span> <span class="NLM_lpage">833</span><span class="refDoi"> DOI: 10.1038/nrc2253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnrc2253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=17957188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=823-833&author=A.+V.+Krivtsovauthor=S.+A.+Armstrong&title=MLL+translocations%2C+histone+modifications+and+leukaemia+stem-cell+development&doi=10.1038%2Fnrc2253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">MLL translocations, histone modifications and leukaemia stem-cell development</span></div><div class="casAuthors">Krivtsov, Andrei V.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia (MLL) has histone methyltransferase activity and regulates the expression of genes such as Hox genes.  This activity is lost in MLL fusion proteins resulting from inter-chromosomal translocations, which are leukemogenic.  How do MLL fusions function and what is their role in leukemia stem cells.  Translocations that involve the mixed lineage leukemia (MLL) gene identify a unique group of acute leukemias, and often predict a poor prognosis.  The MLL gene encodes a DNA-binding protein that methylates histone H3 lysine 4 (H3K4), and pos. regulates gene expression including multiple Hox genes.  Leukaemogenic MLL translocations encode MLL fusion proteins that have lost H3K4 methyltransferase activity.  A key feature of MLL fusion proteins is their ability to efficiently transform haematopoietic cells into leukemia stem cells.  The link between a chromatin modulator and leukemia stem cells provides support for epigenetic landscapes as an important part of leukemia and normal stem-cell development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9NXrcNtrd7Vg90H21EOLACvtfcHk0ljmgVrh32BW-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ&md5=56efc9bbbb9f24fb4233e40ed86628fd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc2253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2253%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DMLL%2520translocations%252C%2520histone%2520modifications%2520and%2520leukaemia%2520stem-cell%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D823%26epage%3D833%26doi%3D10.1038%2Fnrc2253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Thiel, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blessington, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feather, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koretzky, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, X.</span><span> </span><span class="NLM_article-title">MLL-AF9-induced leukemogenesis requires coexpression of the wild-type Mll allele</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1016/j.ccr.2009.12.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.ccr.2009.12.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=20159607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Kls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=148-159&author=A.+T.+Thielauthor=P.+Blessingtonauthor=T.+Zouauthor=D.+Featherauthor=X.+Wuauthor=J.+Yanauthor=H.+Zhangauthor=Z.+Liuauthor=P.+Ernstauthor=G.+A.+Koretzkyauthor=X.+Hua&title=MLL-AF9-induced+leukemogenesis+requires+coexpression+of+the+wild-type+Mll+allele&doi=10.1016%2Fj.ccr.2009.12.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MLL-AF9-induced leukemogenesis requires coexpression of the wild-type Mll allele</span></div><div class="casAuthors">Thiel, Austin T.; Blessington, Peter; Zou, Tao; Feather, Danielle; Wu, Xinjiang; Yan, Jizhou; Zhang, Hui; Liu, Zuguo; Ernst, Patricia; Koretzky, Gary A.; Hua, Xianxin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-159</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Oncogenic fusion proteins are capable of initiating tumorigenesis, but the role of their wild-type counterparts in this process is poorly understood.  The mixed lineage leukemia (MLL) gene undergoes chromosomal translocations, resulting in the formation of oncogenic MLL fusion proteins (MLL-FPs).  Here, we show that menin recruits both wild-type MLL and oncogenic MLL-AF9 fusion protein to the loci of HOX genes to activate their transcription.  Wild-type MLL not only catalyzes histone methylation at key target genes but also controls distinct MLL-AF9-induced histone methylation.  Notably, the wild-type Mll allele is required for MLL-AF9-induced leukemogenesis and maintenance of MLL-AF9-transformed cells.  These findings suggest an essential cooperation between an oncogene and its wild-type counterpart in MLL-AF9-induced leukemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUZ8jLUM0JCrVg90H21EOLACvtfcHk0li9_G6MHfJdtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Kls7s%253D&md5=ddcce857a9f60c6a94eb17440ccaa77d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.12.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.12.034%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DA.%2BT.%26aulast%3DBlessington%26aufirst%3DP.%26aulast%3DZou%26aufirst%3DT.%26aulast%3DFeather%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DErnst%26aufirst%3DP.%26aulast%3DKoretzky%26aufirst%3DG.%2BA.%26aulast%3DHua%26aufirst%3DX.%26atitle%3DMLL-AF9-induced%2520leukemogenesis%2520requires%2520coexpression%2520of%2520the%2520wild-type%2520Mll%2520allele%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26spage%3D148%26epage%3D159%26doi%3D10.1016%2Fj.ccr.2009.12.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stadler, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basrur, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitcomb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elenitoba-Johnson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Multiple interactions recruit MLL1 and MLL1 fusion proteins to the HOXA9 locus in leukemogenesis</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">853</span><span class="NLM_x">–</span> <span class="NLM_lpage">863</span><span class="refDoi"> DOI: 10.1016/j.molcel.2010.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.molcel.2010.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=20541448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVWmtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=853-863&author=T.+A.+Milneauthor=J.+Kimauthor=G.+G.+Wangauthor=S.+C.+Stadlerauthor=V.+Basrurauthor=S.+J.+Whitcombauthor=Z.+Wangauthor=A.+J.+Ruthenburgauthor=K.+S.+Elenitoba-Johnsonauthor=R.+G.+Roederauthor=C.+D.+Allis&title=Multiple+interactions+recruit+MLL1+and+MLL1+fusion+proteins+to+the+HOXA9+locus+in+leukemogenesis&doi=10.1016%2Fj.molcel.2010.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple interactions recruit MLL1 and MLL1 fusion proteins to the HOXA9 locus in leukemogenesis</span></div><div class="casAuthors">Milne, Thomas A.; Kim, Jaehoon; Wang, Gang G.; Stadler, Sonja C.; Basrur, Venkatesha; Whitcomb, Sarah J.; Wang, Zhanxin; Ruthenburg, Alexander J.; Elenitoba-Johnson, Kojo S. J.; Roeder, Robert G.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">853-863</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">MLL1 fusion proteins activate HoxA9 gene expression and cause aggressive leukemias that respond poorly to treatment, but how they recognize and stably bind to HoxA9 is not clearly understood.  In a systematic anal. of MLL1 domain recruitment activity, we identified an essential MLL1 recruitment domain that includes the CXXC domain and PHD fingers and is controlled by direct interactions with the PAF elongation complex and H3K4Me2/3.  MLL1 fusion proteins lack the PHD fingers and require prebinding of a wild-type MLL1 complex and CXXC domain recognition of DNA for stable HoxA9 assocn.  Together, these results suggest that specific recruitment of MLL1 requires multiple interactions and is a precondition for stable recruitment of MLL1 fusion proteins to HoxA9 in leukemogenesis.  Since wild-type MLL1 and oncogenic MLL1 fusion proteins have overlapping yet distinct recruitment mechanisms, this creates a window of opportunity that could be exploited for the development of targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSrmMWM1988rVg90H21EOLACvtfcHk0li9_G6MHfJdtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVWmtbs%253D&md5=bb5a19985cad898e15c44851d5b3b839</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DStadler%26aufirst%3DS.%2BC.%26aulast%3DBasrur%26aufirst%3DV.%26aulast%3DWhitcomb%26aufirst%3DS.%2BJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DMultiple%2520interactions%2520recruit%2520MLL1%2520and%2520MLL1%2520fusion%2520proteins%2520to%2520the%2520HOXA9%2520locus%2520in%2520leukemogenesis%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D38%26spage%3D853%26epage%3D863%26doi%3D10.1016%2Fj.molcel.2010.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vought, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">A conserved arginine-containing motif crucial for the assembly and enzymatic activity of the mixed lineage leukemia protein-1 core complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32162</span><span class="NLM_x">–</span> <span class="NLM_lpage">32175</span><span class="refDoi"> DOI: 10.1074/jbc.M806317200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+conserved+arginine-containing+motif+crucial+for+the+assembly+and+enzymatic+activity+of+the+mixed+lineage+leukemia+protein-1+core+complex&doi=10.1074%2Fjbc.M806317200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0liKBh11sAlPmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520conserved%2520arginine-containing%2520motif%2520crucial%2520for%2520the%2520assembly%2520and%2520enzymatic%2520activity%2520of%2520the%2520mixed%2520lineage%2520leukemia%2520protein-1%2520core%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175%26doi%3D10.1074%2Fjbc.M806317200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Analysis of the binding of mixed lineage leukemia 1 (MLL1) and histone 3 peptides to WD repeat domain 5 (WDR5) for the design of inhibitors of the MLL1-WDR5 interaction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5179</span><span class="NLM_x">–</span> <span class="NLM_lpage">5185</span><span class="refDoi"> DOI: 10.1021/jm100139b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100139b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVKqu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5179-5185&author=H.+Karatasauthor=E.+C.+Townsendauthor=D.+Bernardauthor=Y.+Douauthor=S.+Wang&title=Analysis+of+the+binding+of+mixed+lineage+leukemia+1+%28MLL1%29+and+histone+3+peptides+to+WD+repeat+domain+5+%28WDR5%29+for+the+design+of+inhibitors+of+the+MLL1-WDR5+interaction&doi=10.1021%2Fjm100139b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of the Binding of Mixed Lineage Leukemia 1 (MLL1) and Histone 3 Peptides to WD Repeat Domain 5 (WDR5) for the Design of Inhibitors of the MLL1-WDR5 Interaction</span></div><div class="casAuthors">Karatas, Hacer; Townsend, Elizabeth C.; Bernard, Denzil; Dou, Yali; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5179-5185</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MLL1 is a histone 3 lysine 4 (H3K4) methyltransferase and a promising new cancer therapeutic target.  The catalytic activity of MLL1 is regulated by the formation of a core complex consisting of MLL1, WDR5, RbBP5, and Ash2L.  The interaction between WDR5 and MLL1 plays an essential role in regulation of the H3K4 methyltransferase activity of MLL1 and targeting this interaction using small mols. may represent an attractive therapeutic strategy.  In this study, we have defined the essential elements in MLL1 required for its high-affinity binding to WDR5.  Our data showed that the minimal elements crucial for high-affinity binding of MLL1 to WDR5 are -CO-ARA-NH- motif and two intramol. hydrogen bonds that stabilize the conformation of this motif.  Two 3-mer peptides, Ac-ARA-NH2 and Ac-ART-NH2, were designed based upon MLL1 and H3 sequences and achieved Ki values of 120 and 20 nM to WDR5, resp.  Our study provides a concrete basis for the design of potent peptidomimetics and nonpeptidic compds. to inhibit MLL1 activity by targeting the MLL1 and WDR5 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxEb9S0TLVzrVg90H21EOLACvtfcHk0liKBh11sAlPmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVKqu74%253D&md5=b13aa34ad30e55a0d4742294a46de8ff</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm100139b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100139b%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520the%2520binding%2520of%2520mixed%2520lineage%2520leukemia%25201%2520%2528MLL1%2529%2520and%2520histone%25203%2520peptides%2520to%2520WD%2520repeat%2520domain%25205%2520%2528WDR5%2529%2520for%2520the%2520design%2520of%2520inhibitors%2520of%2520the%2520MLL1-WDR5%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5179%26epage%3D5185%26doi%3D10.1021%2Fjm100139b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">High-affinity, small-molecule peptidomimetic inhibitors of MLL1/WDR5 protein-protein interaction</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">682</span><span class="refDoi"> DOI: 10.1021/ja306028q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja306028q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=669-682&author=H.+Karatasauthor=E.+C.+Townsendauthor=F.+Caoauthor=Y.+Chenauthor=D.+Bernardauthor=L.+Liuauthor=M.+Leiauthor=Y.+Douauthor=S.+Wang&title=High-affinity%2C+small-molecule+peptidomimetic+inhibitors+of+MLL1%2FWDR5+protein-protein+interaction&doi=10.1021%2Fja306028q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fja306028q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja306028q%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DHigh-affinity%252C%2520small-molecule%2520peptidomimetic%2520inhibitors%2520of%2520MLL1%252FWDR5%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D669%26epage%3D682%26doi%3D10.1021%2Fja306028q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span> </span><span class="NLM_article-title">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">713</span><span class="NLM_x">–</span> <span class="NLM_lpage">719</span><span class="refDoi"> DOI: 10.1038/nsmb1128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnsmb1128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=16878130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=713-719&author=Y.+Douauthor=T.+A.+Milneauthor=A.+J.+Ruthenburgauthor=S.+Leeauthor=J.+W.+Leeauthor=G.+L.+Verdineauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Regulation+of+MLL1+H3K4+methyltransferase+activity+by+its+core+components&doi=10.1038%2Fnsmb1128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span></div><div class="casAuthors">Dou, Yali; Milne, Thomas A.; Ruthenburg, Alexander J.; Lee, Seunghee; Lee, Jae Woon; Verdine, Gregory L.; Allis, C. David; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">713-719</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 Lys4 (H3K4) methylation is a prevalent mark assocd. with transcription activation.  A common feature of several H3K4 methyltransferase complexes is the presence of three structural components (RbBP5, Ash2L and WDR5) and a catalytic subunit contg. a SET domain.  Here we report the first biochem. reconstitution of a functional four-component mixed-lineage leukemia protein-1 (MLL1) core complex.  This reconstitution, combined with in vivo assays, allows direct anal. of the contribution of each component to MLL1 enzymic activity and their roles in transcriptional regulation.  Moreover, taking clues from a crystal structure anal., we demonstrate that WDR5 mediates interactions of the MLL1 catalytic unit both with the common structural platform and with the histone substrate.  Mechanistic insights gained from this study can be generalized to the whole family of SET1-like histone methyltransferases in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp65Iy--Is42bVg90H21EOLACvtfcHk0lgSbcoOQ1qg1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D&md5=d6fa704fa0a2fa69afddb540fbff28ae</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1128%26sid%3Dliteratum%253Aachs%26aulast%3DDou%26aufirst%3DY.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DRegulation%2520of%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520by%2520its%2520core%2520components%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D713%26epage%3D719%26doi%3D10.1038%2Fnsmb1128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">Structure of WDR5 bound to mixed lineage leukemia protein-1 peptide</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32158</span><span class="NLM_x">–</span> <span class="NLM_lpage">32161</span><span class="refDoi"> DOI: 10.1074/jbc.C800164200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1074%2Fjbc.C800164200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=18829459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32158-32161&author=A.+Patelauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=Structure+of+WDR5+bound+to+mixed+lineage+leukemia+protein-1+peptide&doi=10.1074%2Fjbc.C800164200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32158-32161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) catalyzes histone H3 lysine 4 methylation and is regulated by interaction with WDR5 (WD-repeat protein-5), RbBP5 (retinoblastoma-binding protein-5), and the Ash2L (absent, small, homeotic disks-2-like) oncoprotein.  In the accompanying investigation, we describe the identification of a conserved arginine contg. motif, called the "Win" or WDR5 interaction motif, that is essential for the assembly and H3K4 dimethylation activity of the MLL1 core complex.  Here we present a 1.7-Å crystal structure of WDR5 bound to a peptide derived from the MLL1 Win motif.  Our results show that Arg-3765 of MLL1 is bound in the same arginine binding pocket on WDR5 that was previously suggested to bind histone H3.  Thermodn. binding expts. show that the MLL1 Win peptide is preferentially recognized by WDR5.  These results are consistent with a model in which WDR5 recognizes Arg-3765 of MLL1, which is essential for the assembly and enzymic activity of the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2hQy6dZIN-LVg90H21EOLACvtfcHk0lgSbcoOQ1qg1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M&md5=770e511a13cf8ada19b37e0cb7115eaf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C800164200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C800164200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DStructure%2520of%2520WDR5%2520bound%2520to%2520mixed%2520lineage%2520leukemia%2520protein-1%2520peptide%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32158%26epage%3D32161%26doi%3D10.1074%2Fjbc.C800164200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Cao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouillette, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span> </span><span class="NLM_article-title">Targeting MLL1 H3K4 methyltransferase activity in mixed-lineage leukemia</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1016/j.molcel.2013.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.molcel.2013.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=24389101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=247-61&author=F.+Caoauthor=E.+C.+Townsendauthor=H.+Karatasauthor=J.+Xuauthor=L.+Liauthor=S.+Leeauthor=L.+Liuauthor=Y.+Chenauthor=P.+Ouilletteauthor=J.+Zhuauthor=J.+L.+Hessauthor=P.+Atadjaauthor=M.+Leiauthor=Z.+S.+Qinauthor=S.+Malekauthor=S.+Wangauthor=Y.+Dou&title=Targeting+MLL1+H3K4+methyltransferase+activity+in+mixed-lineage+leukemia&doi=10.1016%2Fj.molcel.2013.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MLL1 H3K4 Methyltransferase Activity in Mixed-Lineage Leukemia</span></div><div class="casAuthors">Cao, Fang; Townsend, Elizabeth C.; Karatas, Hacer; Xu, Jing; Li, Li; Lee, Shirley; Liu, Liu; Chen, Yong; Ouillette, Peter; Zhu, Jidong; Hess, Jay L.; Atadja, Peter; Lei, Ming; Qin, Zhaohui S.; Malek, Sami; Wang, Shaomeng; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-261</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Here we report a comprehensive characterization of our recently developed inhibitor MM-401 that targets the MLL1 H3K4 methyltransferase activity.  MM-401 is able to specifically inhibit MLL1 activity by blocking MLL1-WDR5 interaction and thus the complex assembly.  This targeting strategy does not affect other mixed-lineage leukemia (MLL) family histone methyltransferases (HMTs), revealing a unique regulatory feature for the MLL1 complex.  Using MM-401 and its enantiomer control MM-NC-401, we show that inhibiting MLL1 methyltransferase activity specifically blocks proliferation of MLL cells by inducing cell-cycle arrest, apoptosis, and myeloid differentiation without general toxicity to normal bone marrow cells or non-MLL cells.  More importantly, transcriptome analyses show that MM-401 induces changes in gene expression similar to those of MLL1 deletion, supporting a predominant role of MLL1 activity in regulating MLL1-dependent leukemia transcription program.  We envision broad applications for MM-401 in basic and translational research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov9sdJIz15pbVg90H21EOLACvtfcHk0lgvL6mCdAkWmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D&md5=f3e482ba3474b39e1bbdde0d10407d1f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DF.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuillette%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DQin%26aufirst%3DZ.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DTargeting%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520in%2520mixed-lineage%2520leukemia%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D53%26spage%3D247%26epage%3D61%26doi%3D10.1016%2Fj.molcel.2013.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seitova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Small-molecule inhibition of MLL activity by disruption of its interaction with WDR5</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1042/BJ20121280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1042%2FBJ20121280" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=151-159&author=G.+Senisterraauthor=H.+Wuauthor=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=D.+Barsyte-Lovejoyauthor=L.+Dombrovskiauthor=A.+Dongauthor=K.+T.+Nguyenauthor=D.+Smilauthor=Y.+Bolshanauthor=T.+Hajianauthor=H.+Heauthor=A.+Seitovaauthor=I.+Chauauthor=F.+Liauthor=G.+Podaauthor=J.+F.+Coutureauthor=P.+J.+Brownauthor=R.+Al-Awarauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Small-molecule+inhibition+of+MLL+activity+by+disruption+of+its+interaction+with+WDR5&doi=10.1042%2FBJ20121280"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2FBJ20121280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121280%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DSeitova%26aufirst%3DA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520MLL%2520activity%2520by%2520disruption%2520of%2520its%2520interaction%2520with%2520WDR5%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D151%26epage%3D159%26doi%3D10.1042%2FBJ20121280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zepeda-Velazquez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiyota, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span> </span><span class="NLM_article-title">Structure-based optimization of a small molecule antagonist of the interaction between WD repeat-containing protein 5 (WDR5) and mixed-lineage leukemia 1 (MLL1)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">2478</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01630</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01630" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2478-96&author=M.+Getlikauthor=D.+Smilauthor=C.+Zepeda-Velazquezauthor=Y.+Bolshanauthor=G.+Podaauthor=H.+Wuauthor=A.+Dongauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=G.+Senisterraauthor=L.+Dombrovskiauthor=T.+Hajianauthor=T.+Kiyotaauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=R.+Al-Awar&title=Structure-based+optimization+of+a+small+molecule+antagonist+of+the+interaction+between+WD+repeat-containing+protein+5+%28WDR5%29+and+mixed-lineage+leukemia+1+%28MLL1%29&doi=10.1021%2Facs.jmedchem.5b01630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span></div><div class="casAuthors">Getlik, Matthaus; Smil, David; Zepeda-Velazquez, Carlos; Bolshan, Yuri; Poda, Gennady; Wu, Hong; Dong, Aiping; Kuznetsova, Ekaterina; Marcellus, Richard; Senisterra, Guillermo; Dombrovski, Ludmila; Hajian, Taraneh; Kiyota, Taira; Schapira, Matthieu; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Al-awar, Rima</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2478-2496</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">WD repeat-contg. protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1).  Rearrangement of the MLL1 gene is assocd. with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics.  Our previous work showed that binding of small mol. ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity.  Initial structure-activity relationship studies identified N-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein-protein interaction.  Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents.  This allowed us to develop the first highly potent (Kdisp < 100 nM) small mol. antagonists of the WDR5-MLL1 interaction and demonstrate that N-(4-(4-methylpiperazin-1-yl)-3'-(morpholinomethyl)-[1,1'-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide 16d (OICR-9429) is a potent and selective chem. probe suitable to help dissect the biol. role of WDR5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60Phwp2ghZLVg90H21EOLACvtfcHk0lgvL6mCdAkWmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D&md5=f3a3a529dcf114a459a8b5d6f7270154</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01630%26sid%3Dliteratum%253Aachs%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DKiyota%26aufirst%3DT.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAl-Awar%26aufirst%3DR.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520small%2520molecule%2520antagonist%2520of%2520the%2520interaction%2520between%2520WD%2520repeat-containing%2520protein%25205%2520%2528WDR5%2529%2520and%2520mixed-lineage%2520leukemia%25201%2520%2528MLL1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2478%26epage%3D96%26doi%3D10.1021%2Facs.jmedchem.5b01630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Li, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Z. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, X. K.</span><span> </span><span class="NLM_article-title">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">480</span><span class="NLM_x">–</span> <span class="NLM_lpage">489</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2016.08.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.ejmech.2016.08.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=27598236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2016&pages=480-489&author=D.+D.+Liauthor=W.+L.+Chenauthor=Z.+H.+Wangauthor=Y.+Y.+Xieauthor=X.+L.+Xuauthor=Z.+Y.+Jiangauthor=X.+J.+Zhangauthor=Q.+D.+Youauthor=X.+K.+Guo&title=High-affinity+small+molecular+blockers+of+mixed+lineage+leukemia+1+%28MLL1%29-WDR5+interaction+inhibit+MLL1+complex+H3K4+methyltransferase+activity&doi=10.1016%2Fj.ejmech.2016.08.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Wang, Zhi-Hui; Xie, Yi-Yue; Xu, Xiao-Li; Jiang, Zheng-Yu; Zhang, Xiao-Jin; You, Qi-Dong; Guo, Xiao-Ke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">480-489</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1-WDR5 protein-protein interaction is essential for MLL1 H3K4 methyltransferase activity.  Targeting MLL1 enzymic activity to regulate expression level of MLL-dependent genes represents a therapeutic strategy for acute leukemia harboring MLL fusion proteins.  Herein we reported a series of biphenyl compds. disturbed MLL1-WDR5 interaction.  These compds. effectively inhibited MLL1 histone methyltransferase (HMT) activity in vitro and in MV4-11 cell line.  The representative compd. 30 (DDO-2084) inhibited proliferation and induced apoptosis of MV4-11 cells through deregulating expression level of Hoxa9 and Meis-1 genes, which emphasized our compds. were on-target.  Optimization of compd. 30 led to high-affinity inhibitors.  Esp., compd. 42 (DDO-2117, IC50 = 7.6 nM) bearing an amino and a 4-aminobutanamido group was the most potent inhibitor reported to-date, and showed the most potent inhibitory activity (IC50 = 0.19 μM) in HMT assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIPjH4_g5XrVg90H21EOLACvtfcHk0lgOO-Hb_5Hxkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ&md5=987a0be9098e067213e4bccf726e3dd3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.08.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.08.036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%2BD.%26aulast%3DChen%26aufirst%3DW.%2BL.%26aulast%3DWang%26aufirst%3DZ.%2BH.%26aulast%3DXie%26aufirst%3DY.%2BY.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26atitle%3DHigh-affinity%2520small%2520molecular%2520blockers%2520of%2520mixed%2520lineage%2520leukemia%25201%2520%2528MLL1%2529-WDR5%2520interaction%2520inhibit%2520MLL1%2520complex%2520H3K4%2520methyltransferase%2520activity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D124%26spage%3D480%26epage%3D489%26doi%3D10.1016%2Fj.ejmech.2016.08.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Li, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, X. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H. P.</span><span> </span><span class="NLM_article-title">Structure-based design and synthesis of small molecular inhibitors disturbing the interaction of MLL1-WDR5</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2016.04.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.ejmech.2016.04.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=27116709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVCksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2016&pages=1-8&author=D.+D.+Liauthor=W.+L.+Chenauthor=X.+L.+Xuauthor=F.+Jiangauthor=L.+Wangauthor=Y.+Y.+Xieauthor=X.+J.+Zhangauthor=X.+K.+Guoauthor=Q.+D.+Youauthor=H.+P.+Sun&title=Structure-based+design+and+synthesis+of+small+molecular+inhibitors+disturbing+the+interaction+of+MLL1-WDR5&doi=10.1016%2Fj.ejmech.2016.04.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design and synthesis of small molecular inhibitors disturbing the interaction of MLL1-WDR5</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Xu, Xiao-Li; Jiang, Fen; Wang, Lei; Xie, Yi-Yue; Zhang, Xiao-Jin; Guo, Xiao-Ke; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1 complex catalyzes the methylation of H3K4, and plays important roles in the development of acute leukemia harboring MLL fusion proteins.  Targeting MLL1-WDR5 protein-protein interaction (PPI) to inhibit the activity of histone methyltransferase of MLL1 complex is a novel strategy for treating of acute leukemia.  I (IC50 = 0.3 μM) was defined as a potent small mol. to disturb the interaction of MLL1-WDR5.  Here, the authors described structure-based design and synthesis of small mol. inhibitors to block MLL1-WDR5 PPI.  Esp., compd. II (IC50 = 104 nM) was the most potent small mol., and about 3-times more potent than I.  The authors also discussed the SAR of these series of compds. with docking study, which may stimulate more potent compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNsAbY0it5GLVg90H21EOLACvtfcHk0lgOO-Hb_5Hxkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVCksLw%253D&md5=b18d482f34731dd1558e9b78b770c2c1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.04.032%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%2BD.%26aulast%3DChen%26aufirst%3DW.%2BL.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DY.%2BY.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DSun%26aufirst%3DH.%2BP.%26atitle%3DStructure-based%2520design%2520and%2520synthesis%2520of%2520small%2520molecular%2520inhibitors%2520disturbing%2520the%2520interaction%2520of%2520MLL1-WDR5%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D118%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ejmech.2016.04.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Li, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, X. K.</span><span> </span><span class="NLM_article-title">Structure-based design of ester compounds to inhibit MLL complex catalytic activity by targeting mixed lineage leukemia 1 (MLL1)-WDR5 interaction</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">6109</span><span class="NLM_x">–</span> <span class="NLM_lpage">6118</span><span class="refDoi"> DOI: 10.1016/j.bmc.2016.09.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.bmc.2016.09.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=27720555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GnsL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=6109-6118&author=D.+D.+Liauthor=Z.+H.+Wangauthor=W.+L.+Chenauthor=Y.+Y.+Xieauthor=Q.+D.+Youauthor=X.+K.+Guo&title=Structure-based+design+of+ester+compounds+to+inhibit+MLL+complex+catalytic+activity+by+targeting+mixed+lineage+leukemia+1+%28MLL1%29-WDR5+interaction&doi=10.1016%2Fj.bmc.2016.09.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of ester compounds to inhibit MLL complex catalytic activity by targeting mixed lineage leukemia 1 (MLL1)-WDR5 interaction</span></div><div class="casAuthors">Li, Dong-Dong; Wang, Zhi-Hui; Chen, Wei-Lin; Xie, Yi-Yue; You, Qi-Dong; Guo, Xiao-Ke</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6109-6118</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">WDR5 is an essential protein for enzymic activity of MLL1.  Targeting the protein-protein interaction (PPI) between MLL1 and WDR5 represents a new potential therapeutic strategy for MLL leukemia.  Based on the structure of reported inhibitor WDR5-0103, a class of ester compds. were designed and synthesized to disturb MLL1-WDR5 PPI.  These inhibitors efficiently inhibited the histone methyltransferase activity in vitro.  Esp., WL-15 was one of the most potent inhibitors, blocking the interaction of MLL1-WDR5 with IC50 value of 26.4 nM in competitive binding assay and inhibiting the catalytic activity of MLL1 complex with IC50 value of 5.4 μM.  Docking model indicated that ester compds. suitably occupied the central cavity of WDR5 protein and recapitulated the interactions of WDR5-0103 and the hydrophobic groups and key amino greatly increased the activity in blocking MLL1-WDR5 PPI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogEjIflnBk3LVg90H21EOLACvtfcHk0lgAAbTg87nbmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GnsL7F&md5=b5daa8ba82b039d050cc9ceb1a844a95</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.09.073%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%2BD.%26aulast%3DWang%26aufirst%3DZ.%2BH.%26aulast%3DChen%26aufirst%3DW.%2BL.%26aulast%3DXie%26aufirst%3DY.%2BY.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520ester%2520compounds%2520to%2520inhibit%2520MLL%2520complex%2520catalytic%2520activity%2520by%2520targeting%2520mixed%2520lineage%2520leukemia%25201%2520%2528MLL1%2529-WDR5%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D6109%26epage%3D6118%26doi%3D10.1016%2Fj.bmc.2016.09.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Grebien, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giambruno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grover, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avellino, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skucha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vittori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schonegger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilban, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bock, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerlov, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPalpha N-terminal leukemia</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">578</span><span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+Huberauthor=A.+Schoneggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-Awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+targeting+of+the+Wdr5-MLL+interaction+in+C%2FEBPalpha+N-terminal+leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0lgAAbTg87nbmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DSchonegger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520targeting%2520of%2520the%2520Wdr5-MLL%2520interaction%2520in%2520C%252FEBPalpha%2520N-terminal%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref500"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref500'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit500"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1038/nchembio.1084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref500/cit500&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref500/cit500&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref500/cit500&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells&doi=10.1038%2Fnchembio.1084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit500R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lgmhpsKRK5vFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit500&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896%26doi%3D10.1038%2Fnchembio.1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref600"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref600'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit600"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1016/j.ccr.2011.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref600/cit600&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref600/cit600&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref600/cit600&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor&doi=10.1016%2Fj.ccr.2011.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit600R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0lgmhpsKRK5vFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit600&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65%26doi%3D10.1016%2Fj.ccr.2011.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Nielsen, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nietlispach, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bannister, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murzin, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murzina, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laue, E. D.</span><span> </span><span class="NLM_article-title">Structure of the HP1 chromodomain bound to histone H3 methylated at lysine 9</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">416</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span><span class="refDoi"> DOI: 10.1038/nature722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnature722" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=11882902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD38Xitlaisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=416&publication_year=2002&pages=103-107&author=P.+R.+Nielsenauthor=D.+Nietlispachauthor=H.+R.+Mottauthor=J.+Callaghanauthor=A.+Bannisterauthor=T.+Kouzaridesauthor=A.+G.+Murzinauthor=N.+V.+Murzinaauthor=E.+D.+Laue&title=Structure+of+the+HP1+chromodomain+bound+to+histone+H3+methylated+at+lysine+9&doi=10.1038%2Fnature722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the HP1 chromodomain bound to histone H3 methylated at lysine 9</span></div><div class="casAuthors">Nielsen, Peter R.; Nietlispach, Daniel; Mott, Helen R.; Callaghan, Juliana; Bannister, Andrew; Kouzarides, Tony; Murzin, Alexey G.; Murzina, Natalia V.; Laue, Ernest D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">416</span>
        (<span class="NLM_cas:issue">6876</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Specific modifications to histones are essential epigenetic markers-heritable changes in gene expression that do not affect the DNA sequence.  Methylation of lysine 9 in histone H3 is recognized by heterochromatin protein 1 (HP1), which directs the binding of other proteins to control chromatin structure and gene expression.  Here we show that HP1 uses an induced-fit mechanism for recognition of this modification, as revealed by the structure of its chromodomain bound to a histone H3 peptide dimethylated at Nζ of lysine 9.  The binding pocket for the N-Me groups is provided by three arom. side chains, Tyr21, Trp42 and Phe45, which reside in two regions that become ordered on binding of the peptide.  The side chain of Lys 9 is almost fully extended and surrounded by residues that are conserved in many other chromodomains.  The QTAR peptide sequence preceding Lys9 makes most of the addnl. interactions with the chromodomain, with HP1 residues Val23, Leu40, Trp42, Leu58 and Cys60 appearing to be a major determinant of specificity by binding the key buried Ala7.  These findings predict which other chromodomains will bind methylated proteins and suggest a motif that they recognize.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokEEsxGbMfvbVg90H21EOLACvtfcHk0lgmhpsKRK5vFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xitlaisbg%253D&md5=b391888613b423cee326057d236ae6ae</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature722%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DP.%2BR.%26aulast%3DNietlispach%26aufirst%3DD.%26aulast%3DMott%26aufirst%3DH.%2BR.%26aulast%3DCallaghan%26aufirst%3DJ.%26aulast%3DBannister%26aufirst%3DA.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMurzin%26aufirst%3DA.%2BG.%26aulast%3DMurzina%26aufirst%3DN.%2BV.%26aulast%3DLaue%26aufirst%3DE.%2BD.%26atitle%3DStructure%2520of%2520the%2520HP1%2520chromodomain%2520bound%2520to%2520histone%2520H3%2520methylated%2520at%2520lysine%25209%26jtitle%3DNature%26date%3D2002%26volume%3D416%26spage%3D103%26epage%3D107%26doi%3D10.1038%2Fnature722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of x-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span><span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+x-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0li6Gof4-joc5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520x-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0li6Gof4-joc5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Molecular recognition of histone H3 by the WD40 protein WDR5</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">703</span><span class="refDoi"> DOI: 10.1038/nsmb1116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnsmb1116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=16829960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=698-703&author=J.+F.+Coutureauthor=E.+Collazoauthor=R.+C.+Trievel&title=Molecular+recognition+of+histone+H3+by+the+WD40+protein+WDR5&doi=10.1038%2Fnsmb1116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of histone H3 by the WD40 protein WDR5</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">698-703</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The WD40-repeat protein WDR5 is a conserved subunit of Trithorax (TRX) histone methyltransferase complexes.  WDR5 has been reported to selectively bind dimethylated Lys-4 (K4me2) in histone H3 to promote K4 trimethylation by TRX.  To elucidate the basis of this binding specificity, we have detd. the crystal structure of WDR5 bound to a histone H3 peptide bearing K4me2.  The structure reveals that the N terminus of histone H3 binds as a 310-helix in the central depression formed by the WD40 repeats.  R2 in histone H3 is bound in the acidic channel in the protein's core, whereas K4me2 is solvent exposed and does not engage in direct interactions with WDR5.  Functional studies confirm that WDR5 recognizes A1, R2 and T3 in histone H3 but has virtually identical affinities for the unmodified and mono-, di- and trimethylated forms of K4, demonstrating that it does not discriminate among different degrees of methylation of this residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNU0kVOM0Y3rVg90H21EOLACvtfcHk0ljCjh2k4Il_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrY%253D&md5=8c77c52b072810965f8c68fb6358a69c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1116%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520recognition%2520of%2520histone%2520H3%2520by%2520the%2520WD40%2520protein%2520WDR5%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D698%26epage%3D703%26doi%3D10.1038%2Fnsmb1116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ioerger, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacchettini, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span> </span><span class="NLM_article-title">PHENIX: Building new software for automated crystallographic structure determination</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1948</span><span class="NLM_x">–</span> <span class="NLM_lpage">1954</span><span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.+W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+Building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0ljCjh2k4Il_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520Building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: Model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljCjh2k4Il_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: An automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span><span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+An+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lhncZ10Cjc8XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520An%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Bricogne, G.; Blanc, E.; Brandl, M.; Flensburg, C.; Keller, P.; Paciorek, W.; Roversi, P.; Sharff, A.; Smart, O. S.; Vonrhein, C.; Womack, T. O.</span> BUSTER, 2.10.0., <span class="NLM_publisher-name">Global Phasing Ltd.</span>, <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bricogne%2C+G.%3B+Blanc%2C+E.%3B+Brandl%2C+M.%3B+Flensburg%2C+C.%3B+Keller%2C+P.%3B+Paciorek%2C+W.%3B+Roversi%2C+P.%3B+Sharff%2C+A.%3B+Smart%2C+O.+S.%3B+Vonrhein%2C+C.%3B+Womack%2C+T.+O.+BUSTER%2C+2.10.0.%2C+Global+Phasing+Ltd.%2C+Cambridge%2C+United+Kingdom%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref700"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref700'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit700"><span><span class="NLM_contrib-group">Jones, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">748</span><span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref700/cit700&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref700/cit700&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref700/cit700&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit700R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lhncZ10Cjc8XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit700&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref800"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref800'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit800"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span> </span><span class="NLM_article-title">Improved protein - ligand docking using GOLD</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">623</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+protein+-+ligand+docking+using+GOLD"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit800&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520protein%2520-%2520ligand%2520docking%2520using%2520GOLD%26jtitle%3DProteins%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref10':['cit10'],'ref20':['cit20'],'ref18':['cit18'],'ref9':['cit9'],'ref11':['cit11'],'ref19':['cit19'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref500':['cit500'],'ref600':['cit600'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref700':['cit700'],'ref800':['cit800']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 38 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Weilin Chen, Xin Chen, Dongdong Li, Jianrui Zhou, Zhengyu Jiang, Qidong You, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoke Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of DDO-2213 as a Potent and Orally Bioavailable Inhibitor of the WDR5–Mixed Lineage Leukemia 1 Protein–Protein Interaction for the Treatment of MLL Fusion Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8221-8245. <a href="https://doi.org/10.1021/acs.jmedchem.1c00091" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00091%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BDDO-2213%252Bas%252Ba%252BPotent%252Band%252BOrally%252BBioavailable%252BInhibitor%252Bof%252Bthe%252BWDR5%2525E2%252580%252593Mixed%252BLineage%252BLeukemia%252B1%252BProtein%2525E2%252580%252593Protein%252BInteraction%252Bfor%252Bthe%252BTreatment%252Bof%252BMLL%252BFusion%252BLeukemia%26aulast%3DChen%26aufirst%3DWeilin%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D19012021%26date%3D09062021%26volume%3D64%26issue%3D12%26spage%3D8221%26epage%3D8245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiang Luo, Qianzhong Zhang, Yi Jiang, Chengxin Wang, Xianheng Song, Jianheng Li, Qinfang Yan, Albert S. C. Chan, <span class="NLM_string-name hlFld-ContribAuthor">Yong Zou</span>. </span><span class="cited-content_cbyCitation_article-title">Copper-Catalyzed Lactamization of (E)-2-(2-Bromophenyl)-3-arylacrylamides for the Synthesis of (E)-3-Arylideneindolin-2-ones. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>86 </em>
                                    (9)
                                     , 6698-6710. <a href="https://doi.org/10.1021/acs.joc.1c00452" title="DOI URL">https://doi.org/10.1021/acs.joc.1c00452</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.1c00452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.1c00452%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DCopper-Catalyzed%252BLactamization%252Bof%252B%252528E%252529-2-%2525282-Bromophenyl%252529-3-arylacrylamides%252Bfor%252Bthe%252BSynthesis%252Bof%252B%252528E%252529-3-Arylideneindolin-2-ones%26aulast%3DLuo%26aufirst%3DXiang%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D23022021%26date%3D21042021%26volume%3D86%26issue%3D9%26spage%3D6698%26epage%3D6710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ting-Rong Chern, Liu Liu, Elyse Petrunak, Jeanne A. Stuckey, Mi Wang, Denzil Bernard, Haibin Zhou, Shirley Lee, Yali Dou, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent Small-Molecule Inhibitors of MLL Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (6)
                                     , 1348-1352. <a href="https://doi.org/10.1021/acsmedchemlett.0c00229" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00229</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00229%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BPotent%252BSmall-Molecule%252BInhibitors%252Bof%252BMLL%252BMethyltransferase%26aulast%3DChern%26aufirst%3DTing-Rong%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30042020%26date%3D14052020%26date%3D18052020%26date%3D14052020%26volume%3D11%26issue%3D6%26spage%3D1348%26epage%3D1352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianhua Tian, Kevin B. Teuscher, Erin R. Aho, Joseph R. Alvarado, Jonathan J. Mills, Kenneth M. Meyers, Rocco D. Gogliotti, Changho Han, Jonathan D. Macdonald, Jiqing Sai, J. Grace Shaw, John L. Sensintaffar, Bin Zhao, Tyson A. Rietz, Lance R. Thomas, William G. Payne, William J. Moore, Gordon M. Stott, Jumpei Kondo, Masahiro Inoue, Robert J. Coffey, William P. Tansey, Shaun R. Stauffer, Taekyu Lee, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Structure-Based Optimization of Potent and Selective WD Repeat Domain 5 (WDR5) Inhibitors Containing a Dihydroisoquinolinone Bicyclic Core. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 656-675. <a href="https://doi.org/10.1021/acs.jmedchem.9b01608" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01608</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01608%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BStructure-Based%252BOptimization%252Bof%252BPotent%252Band%252BSelective%252BWD%252BRepeat%252BDomain%252B5%252B%252528WDR5%252529%252BInhibitors%252BContaining%252Ba%252BDihydroisoquinolinone%252BBicyclic%252BCore%26aulast%3DTian%26aufirst%3DJianhua%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D27092019%26date%3D07012020%26date%3D20122019%26volume%3D63%26issue%3D2%26spage%3D656%26epage%3D675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan D. Macdonald, Selena Chacón Simon, Changho Han, Feng Wang, J. Grace Shaw, Jennifer E. Howes, Jiqing Sai, Joannes P. Yuh, Demarco Camper, Bethany M. Alicie, Joseph Alvarado, Sameer Nikhar, William Payne, Erin R. Aho, Joshua A. Bauer, Bin Zhao, Jason Phan, Lance R. Thomas, Olivia W. Rossanese, William P. Tansey, Alex G. Waterson, Shaun R. Stauffer, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5–MYC Protein–Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (24)
                                     , 11232-11259. <a href="https://doi.org/10.1021/acs.jmedchem.9b01411" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01411</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01411%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252BSalicylic%252BAcid-Derived%252BSulfonamide%252BInhibitors%252Bof%252Bthe%252BWD%252BRepeat-Containing%252BProtein%252B5%2525E2%252580%252593MYC%252BProtein%2525E2%252580%252593Protein%252BInteraction%26aulast%3DMacdonald%26aufirst%3DJonathan%2BD.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26082019%26date%3D05122019%26date%3D14112019%26volume%3D62%26issue%3D24%26spage%3D11232%26epage%3D11259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alpeshkumar
K. Malde, Timothy A. Hill, Abishek Iyer, <span class="NLM_string-name hlFld-ContribAuthor">David P. Fairlie</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal Structures of Protein-Bound Cyclic Peptides. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (17)
                                     , 9861-9914. <a href="https://doi.org/10.1021/acs.chemrev.8b00807" title="DOI URL">https://doi.org/10.1021/acs.chemrev.8b00807</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.8b00807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.8b00807%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DCrystal%252BStructures%252Bof%252BProtein-Bound%252BCyclic%252BPeptides%26aulast%3DMalde%26aufirst%3DAlpeshkumar%2BK.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D31122018%26date%3D02052019%26volume%3D119%26issue%3D17%26spage%3D9861%26epage%3D9914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Feng Wang, Kyu Ok Jeon, James M. Salovich, Jonathan D. Macdonald, Joseph Alvarado, Rocco D. Gogliotti, Jason Phan, Edward T. Olejniczak, Qi Sun, Shidong Wang, DeMarco Camper, Joannes P. Yuh, J. Grace Shaw, Jiqing Sai, Olivia W. Rossanese, William P. Tansey, Shaun R. Stauffer, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent 2-Aryl-6,7-dihydro-5H-pyrrolo[1,2-a]imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (13)
                                     , 5623-5642. <a href="https://doi.org/10.1021/acs.jmedchem.8b00375" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00375%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%252B2-Aryl-6%25252C7-dihydro-5H-pyrrolo%25255B1%25252C2-a%25255Dimidazoles%252Bas%252BWDR5-WIN-Site%252BInhibitors%252BUsing%252BFragment-Based%252BMethods%252Band%252BStructure-Based%252BDesign%26aulast%3DWang%26aufirst%3DFeng%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D08032018%26date%3D29062018%26date%3D11062018%26volume%3D61%26issue%3D13%26spage%3D5623%26epage%3D5642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Structure, function and inhibition of critical protein–protein interactions involving mixed lineage leukemia 1 and its fusion oncoproteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01057-7" title="DOI URL">https://doi.org/10.1186/s13045-021-01057-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01057-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01057-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DStructure%25252C%252Bfunction%252Band%252Binhibition%252Bof%252Bcritical%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Binvolving%252Bmixed%252Blineage%252Bleukemia%252B1%252Band%252Bits%252Bfusion%252Boncoproteins%26aulast%3DLi%26aufirst%3DXin%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weilin  Chen</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Dongdong  Li</span>, <span class="hlFld-ContribAuthor ">Xianghan  Wang</span>, <span class="hlFld-ContribAuthor ">Guanlu  Long</span>, <span class="hlFld-ContribAuthor ">Zhengyu  Jiang</span>, <span class="hlFld-ContribAuthor ">Qidong  You</span>, <span class="hlFld-ContribAuthor ">Xiaoke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent MLL1 and WDR5 protein-protein interaction inhibitor with in vivo antitumor activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>223 </em>, 113677. <a href="https://doi.org/10.1016/j.ejmech.2021.113677" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113677%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252BMLL1%252Band%252BWDR5%252Bprotein-protein%252Binteraction%252Binhibitor%252Bwith%252Bin%2525C2%2525A0vivo%252Bantitumor%252Bactivity%26aulast%3DChen%26aufirst%3DWeilin%26date%3D2021%26volume%3D223%26spage%3D113677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ali  Imran</span>, <span class="hlFld-ContribAuthor ">Brandon S.  Moyer</span>, <span class="hlFld-ContribAuthor ">Ashley J.  Canning</span>, <span class="hlFld-ContribAuthor ">Dan  Kalina</span>, <span class="hlFld-ContribAuthor ">Thomas M.  Duncan</span>, <span class="hlFld-ContribAuthor ">Kelsey J.  Moody</span>, <span class="hlFld-ContribAuthor ">Aaron J.  Wolfe</span>, <span class="hlFld-ContribAuthor ">Michael S.  Cosgrove</span>, <span class="hlFld-ContribAuthor ">Liviu  Movileanu</span>. </span><span class="cited-content_cbyCitation_article-title">Kinetics of the multitasking high-affinity Win binding site of WDR5 in restricted and unrestricted conditions. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2021,</strong> <em>478 </em>
                                    (11)
                                     , 2145-2161. <a href="https://doi.org/10.1042/BCJ20210253" title="DOI URL">https://doi.org/10.1042/BCJ20210253</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20210253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20210253%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DKinetics%252Bof%252Bthe%252Bmultitasking%252Bhigh-affinity%252BWin%252Bbinding%252Bsite%252Bof%252BWDR5%252Bin%252Brestricted%252Band%252Bunrestricted%252Bconditions%26aulast%3DImran%26aufirst%3DAli%26date%3D2021%26date%3D2021%26volume%3D478%26issue%3D11%26spage%3D2145%26epage%3D2161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqing  Ye</span>, <span class="hlFld-ContribAuthor ">Gang  Chen</span>, <span class="hlFld-ContribAuthor ">Jia  Jin</span>, <span class="hlFld-ContribAuthor ">Binzhong  Zhang</span>, <span class="hlFld-ContribAuthor ">Yinda  Wang</span>, <span class="hlFld-ContribAuthor ">Zhenhai  Cai</span>, <span class="hlFld-ContribAuthor ">Fei  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">The Development of Inhibitors Targeting the Mixed Lineage Leukemia 1 (MLL1)-WD Repeat Domain 5 Protein (WDR5) Protein- Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (33)
                                     , 5530-5542. <a href="https://doi.org/10.2174/0929867326666190528080514" title="DOI URL">https://doi.org/10.2174/0929867326666190528080514</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867326666190528080514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867326666190528080514%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DThe%252BDevelopment%252Bof%252BInhibitors%252BTargeting%252Bthe%252BMixed%252BLineage%252BLeukemia%252B1%252B%252528MLL1%252529-WD%252BRepeat%252BDomain%252B5%252BProtein%252B%252528WDR5%252529%252BProtein-%252BProtein%252BInteraction%26aulast%3DYe%26aufirst%3DXiaoqing%26date%3D2020%26volume%3D27%26issue%3D33%26spage%3D5530%26epage%3D5542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">The complex activities of the SET1/MLL complex core subunits in development and disease. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</span><span> <strong>2020,</strong> <em>1863 </em>
                                    (7)
                                     , 194560. <a href="https://doi.org/10.1016/j.bbagrm.2020.194560" title="DOI URL">https://doi.org/10.1016/j.bbagrm.2020.194560</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagrm.2020.194560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagrm.2020.194560%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Gene%2520Regulatory%2520Mechanisms%26atitle%3DThe%252Bcomplex%252Bactivities%252Bof%252Bthe%252BSET1%25252FMLL%252Bcomplex%252Bcore%252Bsubunits%252Bin%252Bdevelopment%252Band%252Bdisease%26aulast%3DJiang%26aufirst%3DHao%26date%3D2020%26volume%3D1863%26issue%3D7%26spage%3D194560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brian M  Linhares</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic protein–protein interactions with small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (14)
                                     , 1305-1326. <a href="https://doi.org/10.4155/fmc-2020-0082" title="DOI URL">https://doi.org/10.4155/fmc-2020-0082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0082%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bepigenetic%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Bwith%252Bsmall-molecule%252Binhibitors%26aulast%3DLinhares%26aufirst%3DBrian%2BM%26date%3D2020%26volume%3D12%26issue%3D14%26spage%3D1305%26epage%3D1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke  Liu</span>, <span class="hlFld-ContribAuthor ">Jinrong  Min</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Basis for the Recognition of Non-methylated DNA by the CXXC Domain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2020,</strong> <em>432 </em>
                                    (6)
                                     , 1674-1686. <a href="https://doi.org/10.1016/j.jmb.2019.09.025" title="DOI URL">https://doi.org/10.1016/j.jmb.2019.09.025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2019.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2019.09.025%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DStructural%252BBasis%252Bfor%252Bthe%252BRecognition%252Bof%252BNon-methylated%252BDNA%252Bby%252Bthe%252BCXXC%252BDomain%26aulast%3DLiu%26aufirst%3DKe%26date%3D2020%26volume%3D432%26issue%3D6%26spage%3D1674%26epage%3D1686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haley I.  Merritt</span>, <span class="hlFld-ContribAuthor ">Nicholas  Sawyer</span>, <span class="hlFld-ContribAuthor ">Paramjit S.  Arora</span>. </span><span class="cited-content_cbyCitation_article-title">Bent into shape: Folded peptides to mimic protein structure and modulate protein function. </span><span class="cited-content_cbyCitation_journal-name">Peptide Science</span><span> <strong>2020,</strong> <em>112 </em>
                                    (1)
                                     <a href="https://doi.org/10.1002/pep2.24145" title="DOI URL">https://doi.org/10.1002/pep2.24145</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pep2.24145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpep2.24145%26sid%3Dliteratum%253Aachs%26jtitle%3DPeptide%2520Science%26atitle%3DBent%252Binto%252Bshape%25253A%252BFolded%252Bpeptides%252Bto%252Bmimic%252Bprotein%252Bstructure%252Band%252Bmodulate%252Bprotein%252Bfunction%26aulast%3DMerritt%26aufirst%3DHaley%2BI.%26date%3D2020%26date%3D2020%26volume%3D112%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Di  Huang</span>, <span class="hlFld-ContribAuthor ">Xue  Chen</span>, <span class="hlFld-ContribAuthor ">Xuan  Chen</span>, <span class="hlFld-ContribAuthor ">Yan  Qu</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Yafei  Yang</span>, <span class="hlFld-ContribAuthor ">Yufeng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title"><p>WDR5 Promotes Proliferation and Correlates with Poor Prognosis in Oesophageal Squamous Cell Carcinoma</p>. </span><span class="cited-content_cbyCitation_journal-name">OncoTargets and Therapy</span><span> <strong>2020,</strong> <em>Volume 13 </em>, 10525-10534. <a href="https://doi.org/10.2147/OTT.S234773" title="DOI URL">https://doi.org/10.2147/OTT.S234773</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2147/OTT.S234773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2147%2FOTT.S234773%26sid%3Dliteratum%253Aachs%26jtitle%3DOncoTargets%2520and%2520Therapy%26atitle%3D%25253Cp%25253EWDR5%252BPromotes%252BProliferation%252Band%252BCorrelates%252Bwith%252BPoor%252BPrognosis%252Bin%252BOesophageal%252BSquamous%252BCell%252BCarcinoma%25253C%25252Fp%25253E%26aulast%3DHuang%26aufirst%3DDi%26date%3D2020%26date%3D2020%26volume%3DVolume%252013%26spage%3D10525%26epage%3D10534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark P.  Waterhouse</span>, <span class="hlFld-ContribAuthor ">Rosie  Ugur</span>, <span class="hlFld-ContribAuthor ">Walid T.  Khaled</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic and Mechanistic Perspectives of Protein Complexes in Breast Cancer. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fcell.2019.00335" title="DOI URL">https://doi.org/10.3389/fcell.2019.00335</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2019.00335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2019.00335%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DTherapeutic%252Band%252BMechanistic%252BPerspectives%252Bof%252BProtein%252BComplexes%252Bin%252BBreast%252BCancer%26aulast%3DWaterhouse%26aufirst%3DMark%2BP.%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Hongbo  Wang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic machinery: Emerging novel allosteric inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2019,</strong> <em>204 </em>, 107406. <a href="https://doi.org/10.1016/j.pharmthera.2019.107406" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2019.107406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2019.107406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2019.107406%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DTargeting%252Bepigenetic%252Bmachinery%25253A%252BEmerging%252Bnovel%252Ballosteric%252Binhibitors%26aulast%3DYe%26aufirst%3DFei%26date%3D2019%26volume%3D204%26spage%3D107406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ji  Yu</span>, <span class="hlFld-ContribAuthor ">Zhongxia  Zhou</span>, <span class="hlFld-ContribAuthor ">Jacob  Kongsted</span>, <span class="hlFld-ContribAuthor ">Yuning  Song</span>, <span class="hlFld-ContribAuthor ">Christophe  Pannecouque</span>, <span class="hlFld-ContribAuthor ">Erik  De Clercq</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design opportunities presented by solvent‐exposed regions of target proteins. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2019,</strong> <em>39 </em>
                                    (6)
                                     , 2194-2238. <a href="https://doi.org/10.1002/med.21581" title="DOI URL">https://doi.org/10.1002/med.21581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21581%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DMolecular%252Bdesign%252Bopportunities%252Bpresented%252Bby%252Bsolvent%2525E2%252580%252590exposed%252Bregions%252Bof%252Btarget%252Bproteins%26aulast%3DJiang%26aufirst%3DXiangyi%26date%3D2019%26date%3D2019%26volume%3D39%26issue%3D6%26spage%3D2194%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew L.  Dennis</span>, <span class="hlFld-ContribAuthor ">Benjamin J.  Morrow</span>, <span class="hlFld-ContribAuthor ">Olan  Dolezal</span>, <span class="hlFld-ContribAuthor ">Anthony N.  Cuzzupe</span>, <span class="hlFld-ContribAuthor ">Alexandra E.  Stupple</span>, <span class="hlFld-ContribAuthor ">Janet  Newman</span>, <span class="hlFld-ContribAuthor ">John  Bentley</span>, <span class="hlFld-ContribAuthor ">Meghan  Hattarki</span>, <span class="hlFld-ContribAuthor ">Stewart D.  Nuttall</span>, <span class="hlFld-ContribAuthor ">Richard C.  Foitzik</span>, <span class="hlFld-ContribAuthor ">Ian P.  Street</span>, <span class="hlFld-ContribAuthor ">Paul A.  Stupple</span>, <span class="hlFld-ContribAuthor ">Brendon J.  Monahan</span>, <span class="hlFld-ContribAuthor ">Thomas. S.  Peat</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment screening for a protein-protein interaction inhibitor to WDR5. </span><span class="cited-content_cbyCitation_journal-name">Structural Dynamics</span><span> <strong>2019,</strong> <em>6 </em>
                                    (6)
                                     , 064701. <a href="https://doi.org/10.1063/1.5122849" title="DOI URL">https://doi.org/10.1063/1.5122849</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/1.5122849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F1.5122849%26sid%3Dliteratum%253Aachs%26jtitle%3DStructural%2520Dynamics%26atitle%3DFragment%252Bscreening%252Bfor%252Ba%252Bprotein-protein%252Binteraction%252Binhibitor%252Bto%252BWDR5%26aulast%3DDennis%26aufirst%3DMatthew%2BL.%26date%3D2019%26volume%3D6%26issue%3D6%26spage%3D064701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting non-bromodomain chromatin readers. </span><span class="cited-content_cbyCitation_journal-name">Nature Structural & Molecular Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (10)
                                     , 863-869. <a href="https://doi.org/10.1038/s41594-019-0290-2" title="DOI URL">https://doi.org/10.1038/s41594-019-0290-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41594-019-0290-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41594-019-0290-2%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Structural%2520%2526%2520Molecular%2520Biology%26atitle%3DTargeting%252Bnon-bromodomain%252Bchromatin%252Breaders%26aulast%3DArrowsmith%26aufirst%3DCheryl%2BH.%26date%3D2019%26date%3D2019%26volume%3D26%26issue%3D10%26spage%3D863%26epage%3D869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kushani K.  Shah</span>, <span class="hlFld-ContribAuthor ">Robert H.  Whitaker</span>, <span class="hlFld-ContribAuthor ">Theodore  Busby</span>, <span class="hlFld-ContribAuthor ">Jing  Hu</span>, <span class="hlFld-ContribAuthor ">Bi  Shi</span>, <span class="hlFld-ContribAuthor ">Zhenjia  Wang</span>, <span class="hlFld-ContribAuthor ">Chongzhi  Zang</span>, <span class="hlFld-ContribAuthor ">William J.  Placzek</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Specific inhibition of DPY30 activity by ASH2L-derived peptides suppresses blood cancer cell growth. </span><span class="cited-content_cbyCitation_journal-name">Experimental Cell Research</span><span> <strong>2019,</strong> <em>382 </em>
                                    (2)
                                     , 111485. <a href="https://doi.org/10.1016/j.yexcr.2019.06.030" title="DOI URL">https://doi.org/10.1016/j.yexcr.2019.06.030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.yexcr.2019.06.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.yexcr.2019.06.030%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520Cell%2520Research%26atitle%3DSpecific%252Binhibition%252Bof%252BDPY30%252Bactivity%252Bby%252BASH2L-derived%252Bpeptides%252Bsuppresses%252Bblood%252Bcancer%252Bcell%252Bgrowth%26aulast%3DShah%26aufirst%3DKushani%2BK.%26date%3D2019%26volume%3D382%26issue%3D2%26spage%3D111485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lea  Albert</span>, <span class="hlFld-ContribAuthor ">Alberto  Peñalver</span>, <span class="hlFld-ContribAuthor ">Nemanja  Djokovic</span>, <span class="hlFld-ContribAuthor ">Laura  Werel</span>, <span class="hlFld-ContribAuthor ">Malte  Hoffarth</span>, <span class="hlFld-ContribAuthor ">Dusan  Ruzic</span>, <span class="hlFld-ContribAuthor ">Jing  Xu</span>, <span class="hlFld-ContribAuthor ">Lars‐Oliver  Essen</span>, <span class="hlFld-ContribAuthor ">Katarina  Nikolic</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>, <span class="hlFld-ContribAuthor ">Olalla  Vázquez</span>. </span><span class="cited-content_cbyCitation_article-title">Modulating Protein–Protein Interactions with Visible‐Light‐Responsive Peptide Backbone Photoswitches. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2019,</strong> <em>20 </em>
                                    (11)
                                     , 1417-1429. <a href="https://doi.org/10.1002/cbic.201800737" title="DOI URL">https://doi.org/10.1002/cbic.201800737</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201800737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201800737%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DModulating%252BProtein%2525E2%252580%252593Protein%252BInteractions%252Bwith%252BVisible%2525E2%252580%252590Light%2525E2%252580%252590Responsive%252BPeptide%252BBackbone%252BPhotoswitches%26aulast%3DAlbert%26aufirst%3DLea%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D11%26spage%3D1417%26epage%3D1429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kazunori  Hirayama</span>, <span class="hlFld-ContribAuthor ">Yuuki  Fujiwara</span>, <span class="hlFld-ContribAuthor ">Tohru  Terada</span>, <span class="hlFld-ContribAuthor ">Kentaro  Shimizu</span>, <span class="hlFld-ContribAuthor ">Keiji  Wada</span>, <span class="hlFld-ContribAuthor ">Tomohiro  Kabuta</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual screening identification of novel chemical inhibitors for aberrant interactions between pathogenic mutant SOD1 and tubulin. </span><span class="cited-content_cbyCitation_journal-name">Neurochemistry International</span><span> <strong>2019,</strong> <em>126 </em>, 19-26. <a href="https://doi.org/10.1016/j.neuint.2019.02.020" title="DOI URL">https://doi.org/10.1016/j.neuint.2019.02.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neuint.2019.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neuint.2019.02.020%26sid%3Dliteratum%253Aachs%26jtitle%3DNeurochemistry%2520International%26atitle%3DVirtual%252Bscreening%252Bidentification%252Bof%252Bnovel%252Bchemical%252Binhibitors%252Bfor%252Baberrant%252Binteractions%252Bbetween%252Bpathogenic%252Bmutant%252BSOD1%252Band%252Btubulin%26aulast%3DHirayama%26aufirst%3DKazunori%26date%3D2019%26volume%3D126%26spage%3D19%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lambert</span>, <span class="hlFld-ContribAuthor ">Alioui</span>, <span class="hlFld-ContribAuthor ">Jambon</span>, <span class="hlFld-ContribAuthor ">Depauw</span>, <span class="hlFld-ContribAuthor ">Van Seuningen</span>, <span class="hlFld-ContribAuthor ">David-Cordonnier</span>. </span><span class="cited-content_cbyCitation_article-title">Direct and Indirect Targeting of HOXA9 Transcription Factor in Acute Myeloid Leukemia.. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (6)
                                     , 837. <a href="https://doi.org/10.3390/cancers11060837" title="DOI URL">https://doi.org/10.3390/cancers11060837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11060837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11060837%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DDirect%252Band%252BIndirect%252BTargeting%252Bof%252BHOXA9%252BTranscription%252BFactor%252Bin%252BAcute%252BMyeloid%252BLeukemia.%26aulast%3DLambert%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D6%26spage%3D837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqing  Ye</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Fulin  Lian</span>, <span class="hlFld-ContribAuthor ">Weiyao  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Jie  Han</span>, <span class="hlFld-ContribAuthor ">Naixia  Zhang</span>, <span class="hlFld-ContribAuthor ">Jia  Jin</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">The identification of novel small-molecule inhibitors targeting WDR5-MLL1 interaction through fluorescence polarization based high-throughput screening. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (4)
                                     , 638-645. <a href="https://doi.org/10.1016/j.bmcl.2018.12.035" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.12.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.12.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.12.035%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DThe%252Bidentification%252Bof%252Bnovel%252Bsmall-molecule%252Binhibitors%252Btargeting%252BWDR5-MLL1%252Binteraction%252Bthrough%252Bfluorescence%252Bpolarization%252Bbased%252Bhigh-throughput%252Bscreening%26aulast%3DYe%26aufirst%3DXiaoqing%26date%3D2019%26volume%3D29%26issue%3D4%26spage%3D638%26epage%3D645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Methyltransferases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 123-157. <a href="https://doi.org/10.1007/7355_2019_72" title="DOI URL">https://doi.org/10.1007/7355_2019_72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_72%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BMethyltransferases%252Band%252BTheir%252BInhibitors%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2019%26date%3D2019%26spage%3D123%26epage%3D157%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Methyl-Readers and Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 339-399. <a href="https://doi.org/10.1007/7355_2019_78" title="DOI URL">https://doi.org/10.1007/7355_2019_78</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_78%26sid%3Dliteratum%253Aachs%26atitle%3DMethyl-Readers%252Band%252BInhibitors%26aulast%3DSbardella%26aufirst%3DGianluca%26date%3D2019%26date%3D2019%26spage%3D339%26epage%3D399%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kebin  Lu</span>, <span class="hlFld-ContribAuthor ">He  Tao</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Si</span>, <span class="hlFld-ContribAuthor ">Qingjuan  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">The Histone H3 Lysine 4 Presenter WDR5 as an Oncogenic Protein and Novel Epigenetic Target in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2018,</strong> <em>8 </em><a href="https://doi.org/10.3389/fonc.2018.00502" title="DOI URL">https://doi.org/10.3389/fonc.2018.00502</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2018.00502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2018.00502%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DThe%252BHistone%252BH3%252BLysine%252B4%252BPresenter%252BWDR5%252Bas%252Ban%252BOncogenic%252BProtein%252Band%252BNovel%252BEpigenetic%252BTarget%252Bin%252BCancer%26aulast%3DLu%26aufirst%3DKebin%26date%3D2018%26date%3D2018%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xu  Ran</span>, <span class="hlFld-ContribAuthor ">Eileen T.  Burchfiel</span>, <span class="hlFld-ContribAuthor ">Bushu  Dong</span>, <span class="hlFld-ContribAuthor ">Nicholas J.  Rettko</span>, <span class="hlFld-ContribAuthor ">Bryan M.  Dunyak</span>, <span class="hlFld-ContribAuthor ">Hao  Shao</span>, <span class="hlFld-ContribAuthor ">Dennis J.  Thiele</span>, <span class="hlFld-ContribAuthor ">Jason E.  Gestwicki</span>. </span><span class="cited-content_cbyCitation_article-title">Rational design and screening of peptide-based inhibitors of heat shock factor 1 (HSF1). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (19)
                                     , 5299-5306. <a href="https://doi.org/10.1016/j.bmc.2018.04.018" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.04.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.04.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.04.018%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DRational%252Bdesign%252Band%252Bscreening%252Bof%252Bpeptide-based%252Binhibitors%252Bof%252Bheat%252Bshock%252Bfactor%252B1%252B%252528HSF1%252529%26aulast%3DRan%26aufirst%3DXu%26date%3D2018%26volume%3D26%26issue%3D19%26spage%3D5299%26epage%3D5306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas C.  Stephens</span>, <span class="hlFld-ContribAuthor ">Aggie  Lawer</span>, <span class="hlFld-ContribAuthor ">Thomas  French</span>, <span class="hlFld-ContribAuthor ">William P.  Unsworth</span>. </span><span class="cited-content_cbyCitation_article-title">Iterative Assembly of Macrocyclic Lactones using Successive Ring Expansion Reactions. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2018,</strong> <em>24 </em>
                                    (52)
                                     , 13947-13953. <a href="https://doi.org/10.1002/chem.201803064" title="DOI URL">https://doi.org/10.1002/chem.201803064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201803064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201803064%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DIterative%252BAssembly%252Bof%252BMacrocyclic%252BLactones%252Busing%252BSuccessive%252BRing%252BExpansion%252BReactions%26aulast%3DStephens%26aufirst%3DThomas%2BC.%26date%3D2018%26date%3D2018%26volume%3D24%26issue%3D52%26spage%3D13947%26epage%3D13953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiong  Zhang</span>, <span class="hlFld-ContribAuthor ">Xingling  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong  Yang</span>, <span class="hlFld-ContribAuthor ">Juan  Yan</span>, <span class="hlFld-ContribAuthor ">Xuhong  Fu</span>, <span class="hlFld-ContribAuthor ">Rongrui  Wei</span>, <span class="hlFld-ContribAuthor ">Xiaowei  Xu</span>, <span class="hlFld-ContribAuthor ">Zhuqing  Zhang</span>, <span class="hlFld-ContribAuthor ">Aisong  Yu</span>, <span class="hlFld-ContribAuthor ">Kaixin  Zhou</span>, <span class="hlFld-ContribAuthor ">Jian  Ding</span>, <span class="hlFld-ContribAuthor ">Meiyu  Geng</span>, <span class="hlFld-ContribAuthor ">Xun  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Piribedil disrupts the MLL1-WDR5 interaction and sensitizes MLL-rearranged acute myeloid leukemia (AML) to doxorubicin-induced apoptosis. </span><span class="cited-content_cbyCitation_journal-name">Cancer Letters</span><span> <strong>2018,</strong> <em>431 </em>, 150-160. <a href="https://doi.org/10.1016/j.canlet.2018.05.034" title="DOI URL">https://doi.org/10.1016/j.canlet.2018.05.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.canlet.2018.05.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.canlet.2018.05.034%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Letters%26atitle%3DPiribedil%252Bdisrupts%252Bthe%252BMLL1-WDR5%252Binteraction%252Band%252Bsensitizes%252BMLL-rearranged%252Bacute%252Bmyeloid%252Bleukemia%252B%252528AML%252529%252Bto%252Bdoxorubicin-induced%252Bapoptosis%26aulast%3DZhang%26aufirst%3DXiong%26date%3D2018%26volume%3D431%26spage%3D150%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Schuette</span>, <span class="hlFld-ContribAuthor ">Lauren M.  Moore</span>, <span class="hlFld-ContribAuthor ">Marie E.  Robert</span>, <span class="hlFld-ContribAuthor ">Tamar H.  Taddei</span>, <span class="hlFld-ContribAuthor ">Barbara E.  Ehrlich</span>. </span><span class="cited-content_cbyCitation_article-title">Hepatocellular Carcinoma Outcome Is Predicted by Expression of Neuronal Calcium Sensor 1. </span><span class="cited-content_cbyCitation_journal-name">Cancer Epidemiology Biomarkers & Prevention</span><span> <strong>2018,</strong> <em>27 </em>
                                    (9)
                                     , 1091-1100. <a href="https://doi.org/10.1158/1055-9965.EPI-18-0167" title="DOI URL">https://doi.org/10.1158/1055-9965.EPI-18-0167</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1055-9965.EPI-18-0167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1055-9965.EPI-18-0167%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Epidemiology%2520Biomarkers%2520%2526%2520Prevention%26atitle%3DHepatocellular%252BCarcinoma%252BOutcome%252BIs%252BPredicted%252Bby%252BExpression%252Bof%252BNeuronal%252BCalcium%252BSensor%252B1%26aulast%3DSchuette%26aufirst%3DDaniel%26date%3D2018%26date%3D2018%26volume%3D27%26issue%3D9%26spage%3D1091%26epage%3D1100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mélanie  Lambert</span>, <span class="hlFld-ContribAuthor ">Samy  Jambon</span>, <span class="hlFld-ContribAuthor ">Sabine  Depauw</span>, <span class="hlFld-ContribAuthor ">Marie-Hélène  David-Cordonnier</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Transcription Factors for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (6)
                                     , 1479. <a href="https://doi.org/10.3390/molecules23061479" title="DOI URL">https://doi.org/10.3390/molecules23061479</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23061479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23061479%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DTargeting%252BTranscription%252BFactors%252Bfor%252BCancer%252BTreatment%26aulast%3DLambert%26aufirst%3DM%25C3%25A9lanie%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D6%26spage%3D1479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xu  Ran</span>, <span class="hlFld-ContribAuthor ">Jason E  Gestwicki</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of protein–protein interactions (PPIs): an analysis of scaffold choices and buried surface area. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>44 </em>, 75-86. <a href="https://doi.org/10.1016/j.cbpa.2018.06.004" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.06.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.06.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DInhibitors%252Bof%252Bprotein%2525E2%252580%252593protein%252Binteractions%252B%252528PPIs%252529%25253A%252Ban%252Banalysis%252Bof%252Bscaffold%252Bchoices%252Band%252Bburied%252Bsurface%252Barea%26aulast%3DRan%26aufirst%3DXu%26date%3D2018%26volume%3D44%26spage%3D75%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sulejman  Alihodžić</span>, <span class="hlFld-ContribAuthor ">Mirjana  Bukvić</span>, <span class="hlFld-ContribAuthor ">Ivaylo J.  Elenkov</span>, <span class="hlFld-ContribAuthor ">Antun  Hutinec</span>, <span class="hlFld-ContribAuthor ">Sanja  Koštrun</span>, <span class="hlFld-ContribAuthor ">Dijana  Pešić</span>, <span class="hlFld-ContribAuthor ">Gordon  Saxty</span>, <span class="hlFld-ContribAuthor ">Linda  Tomašković</span>, <span class="hlFld-ContribAuthor ">Dinko  Žiher</span>. </span><span class="cited-content_cbyCitation_article-title">Current Trends in Macrocyclic Drug Discovery and beyond -Ro5. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 113-233. <a href="https://doi.org/10.1016/bs.pmch.2018.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2018.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2018.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2018.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252BTrends%252Bin%252BMacrocyclic%252BDrug%252BDiscovery%252Band%252Bbeyond%252B-Ro5%26aulast%3DAlihod%25C5%25BEi%25C4%2587%26aufirst%3DSulejman%26date%3D2018%26spage%3D113%26epage%3D233%26pub%3DElsevier%26date%3D2018%26volume%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein-protein interaction between MLL1 and reciprocal proteins for leukemia therapy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (2)
                                     , 356-365. <a href="https://doi.org/10.1016/j.bmc.2017.11.045" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.11.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.11.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.11.045%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bprotein-protein%252Binteraction%252Bbetween%252BMLL1%252Band%252Breciprocal%252Bproteins%252Bfor%252Bleukemia%252Btherapy%26aulast%3DWang%26aufirst%3DZhi-Hui%26date%3D2018%26volume%3D26%26issue%3D2%26spage%3D356%26epage%3D365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas C.  Stephens</span>, <span class="hlFld-ContribAuthor ">Mahendar  Lodi</span>, <span class="hlFld-ContribAuthor ">Andrew M.  Steer</span>, <span class="hlFld-ContribAuthor ">Yun  Lin</span>, <span class="hlFld-ContribAuthor ">Matthew T.  Gill</span>, <span class="hlFld-ContribAuthor ">William P.  Unsworth</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Cyclic Peptide Mimetics by the Successive Ring Expansion of Lactams. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2017,</strong> <em>23 </em>
                                    (54)
                                     , 13314-13318. <a href="https://doi.org/10.1002/chem.201703316" title="DOI URL">https://doi.org/10.1002/chem.201703316</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201703316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201703316%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DSynthesis%252Bof%252BCyclic%252BPeptide%252BMimetics%252Bby%252Bthe%252BSuccessive%252BRing%252BExpansion%252Bof%252BLactams%26aulast%3DStephens%26aufirst%3DThomas%2BC.%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D54%26spage%3D13314%26epage%3D13318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of three previously reported WDR5–MLL interaction inhibitors including a linear peptidomimetic, a cyclic peptidomimetic, and a nonpeptide inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Cyclic peptidomimetic <b>2</b> in complex with WDR5 [PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM9">4GM9</a>]. Compound <b>2</b> is shown in stick, and WDR5 is shown with surface model. (B) Superposition of the docked model of <b>13</b> (cyan) onto the cocrystal structure of compound <b>2</b> complexed with WDR5. Carbon atoms are shown in yellow for compound <b>2</b> or cyan for compound <b>13</b>, oxygen atoms are shown in red and nitrogen atoms are shown in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Development and optimization of a homogeneous AlphaLISA MLL functional assay. (A) Illustration of the homogeneous AlphaLISA MLL HMT functional assay. (B, C) Enzymatic activity of MLL complex, represented by the fluorescence intensity in relative fluorescence units (RFU) from acceptor beads detected at 615 nm at different concentrations and different reaction time points with a low (300 nM, B) or a high (3 μM, C) SAM concentration. (D) Representative inhibitory curves of selected WDR5 inhibitors. (E) Comparison of potencies of compounds determined from the competitive WDR5 binding assay and the AlphaLISA MLL functional assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A,B) Cell growth inhibition curves of representative cyclic peptidomimetics in MOLM-13 and MV4-11 human leukemia cell lines harboring MLL translocation. (C) Correlation between inhibition potencies of cell growth and MLL HMT activity of representative cyclic peptidomimetics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cell growth inhibition of compound <b>18</b> upon 4-day or 7-day treatment time in MV4-11 and MOLM-13 acute leukemia cell lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0007.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A–C) Cocrystal structure of compound <b>16</b> in complex with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB">4GMB</a>) and (D) in comparison with cocrystal structure of <b>2</b> in complex with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM9">4GM9</a>). Carbon atoms for compound <b>16</b> and compound <b>2</b> are shown in cyan and yellow, respectively. Nitrogen atoms are shown in blue and oxygen atoms are shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0008.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Cocrystal structure of cyclic peptidomimetic <b>18</b> in complex with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC">5VFC</a>). (A) The unbiased difference electron density map for <b>18</b> contoured at 3σ is shown as a gray grid. (B) Structural alignment of WDR5–<b>18</b> with the cocrystal structures of compounds <b>2</b> and <b>16</b> in complex with WDR5 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM9">4GM9</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB">4GMB</a>, respectively). (C) The hydrogen bonding and (D) hydrophobic interactions of WDR5 and <b>18</b>. Dashed lines represent hydrogen bonds (C). Carbon atoms for compounds <b>18</b>, <b>16</b>, and <b>2</b> are shown in yellow, cyan and white, respectively. Nitrogen atoms are shown in blue and oxygen atoms are shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Fmoc Protection of Unnatural Amino Acids<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Fmoc-OSu, DIPEA, dioxane–H<sub>2</sub>O (2:1), rt, 5 h.</p></p></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Solid Phase Synthesis of Carboxylic Acid Intermediates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Solid phase peptide synthesis; (b) 1% CF<sub>3</sub>COOH in CH<sub>2</sub>Cl<sub>2</sub>, rt, 30 min.</p></p></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Designed Cyclic Peptidomimetics <b>2–7</b> and <b>15</b>–<b>17</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 2–3 h; (b) DEA, acetonitrile, 2 h; (c) Fmoc-2-Abu-OH, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (d) Fmoc-Arg(Pbf)-OH (for intermediate <b>34a</b> and <b>34e</b>) or Fmoc-Arg(Me)(Pbf)-OH (for intermediate <b>34h</b>), EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (e) <b>21</b>, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (f) <b>28</b>, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (g) <b>26</b>, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (h) isobutiryl chloride, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, 2 h, rt; (i) <b>30</b>, EDCI, HOAt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h; (j) Hoveyda–Grubb’s second generation catalyst, CH<sub>2</sub>Cl<sub>2</sub>, rt, overnight; (k) H<sub>2</sub>/Pd.C, MeOH, rt, 2 h; (l) CH<sub>2</sub>Cl<sub>2</sub>/CF<sub>3</sub>COOH/H<sub>2</sub>O (20:10:0.5), reflux, 2 h; (m) CH<sub>2</sub>Cl<sub>2</sub>/CF<sub>3</sub>COOH (1:1), rt, 0.5 h.</p></p></figure><figure data-id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Designed Cyclic Peptidomimetics <b>9–12</b> and <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Solid phase peptide synthesis; (b) 1% CF<sub>3</sub>COOH in CH<sub>2</sub>Cl<sub>2</sub>, rt, 30 min then 10% CF<sub>3</sub>COOH in CH<sub>2</sub>Cl<sub>2</sub> 1 h; (c) HATU, DIPEA, DMF, rt, 1 h; (d) 95% CF<sub>3</sub>COOH in water, rt, 2 h.</p></p></figure><figure data-id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/medium/jm-2016-017969_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0013.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Designed Cyclic Peptidomimetics <b>13</b>, <b>18</b>, and <b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01796/20170616/images/large/jm-2016-017969_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01796&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) OsO<sub>4</sub>, NaIO<sub>4,</sub> H<sub>2</sub>O, THF, rt, 4 h; (b) dibenzylamine, NaBH(OAc)<sub>3</sub>, ClCH<sub>2</sub>CH<sub>2</sub>Cl, rt, 10 h; (c) solid phase peptide synthesis; (d) 1% CF<sub>3</sub>COOH in CH<sub>2</sub>Cl<sub>2</sub>, rt, 30 min then H<sub>2</sub>, Pd/C, ethanol, 50 °C, 12 h; (e) HATU, DIPEA, DMF, rt, 1 h; (f) 95% CF<sub>3</sub>COOH in water, rt, 2 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i92">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70861" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70861" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 32 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Hess, J. L.</span><span> </span><span class="NLM_article-title">MLL: A histone methyltransferase disrupted in leukemia</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">500</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.molmed.2004.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.molmed.2004.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=15464450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotF2is7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=500-507&author=J.+L.+Hess&title=MLL%3A+A+histone+methyltransferase+disrupted+in+leukemia&doi=10.1016%2Fj.molmed.2004.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MLL: a histone methyltransferase disrupted in leukemia</span></div><div class="casAuthors">Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">500-507</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Rearrangements of the MLL gene, which is located at chromosome 11q23, are assocd. with aggressive acute leukemias in both children and adults.  MLL regulates Hox gene expression through direct promoter binding and histone modification.  MLL rearrangements occurring in leukemia include MLL fusion genes, partial tandem duplications of MLL and MLL amplification.  MLL fusions and amplification upregulate Hox expression, apparently resulting in a block of hematopoietic differentiation.  Future therapies for MLL-assocd. leukemia might involve blocking Hox gene upregulation by using fusion proteins or inhibiting the activity of Hox proteins themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom2pQrErnZf7Vg90H21EOLACvtfcHk0lg_qWu3zHOQCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotF2is7o%253D&md5=ae9d59e5e59cc017fd0fcf479a4a618d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2004.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2004.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%253A%2520A%2520histone%2520methyltransferase%2520disrupted%2520in%2520leukemia%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2004%26volume%3D10%26spage%3D500%26epage%3D507%26doi%3D10.1016%2Fj.molmed.2004.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Jude, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Climer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artinger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, P.</span><span> </span><span class="NLM_article-title">Unique and independent roles for MLL in adult hematopoietic stem cells and progenitors</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">324</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.1016/j.stem.2007.05.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.stem.2007.05.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=18371366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagu7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=324-337&author=C.+D.+Judeauthor=L.+Climerauthor=D.+Xuauthor=E.+Artingerauthor=J.+K.+Fisherauthor=P.+Ernst&title=Unique+and+independent+roles+for+MLL+in+adult+hematopoietic+stem+cells+and+progenitors&doi=10.1016%2Fj.stem.2007.05.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Unique and independent roles for MLL in adult hematopoietic stem cells and progenitors</span></div><div class="casAuthors">Jude, Craig D.; Climer, Leslie; Xu, Diyong; Artinger, Erika; Fisher, Jill K.; Ernst, Patricia</div><div class="citationInfo"><span class="NLM_cas:title">Cell Stem Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">324-337</span>CODEN:
                <span class="NLM_cas:coden">CSCEC4</span>;
        ISSN:<span class="NLM_cas:issn">1934-5909</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Mixed Lineage Leukemia (MLL) gene is essential for embryonic hematopoietic stem cell (HSC) development, but its role during adult hematopoiesis is unknown.  Using an inducible knockout model, we demonstrate that Mll is essential for the maintenance of adult HSCs and progenitors, with fatal bone marrow failure occurring within 3 wk of Mll deletion.  Mll-deficient cells are selectively lost from mixed bone marrow chimeras, demonstrating their failure to self-renew even in an intact bone marrow environment.  Surprisingly, HSCs lacking Mll exhibit ectopic cell-cycle entry, resulting in the depletion of quiescent HSCs.  In contrast, Mll deletion in myelo-erythroid progenitors results in reduced proliferation and reduced response to cytokine-induced cell-cycle entry.  Committed lymphoid and myeloid cells no longer require Mll, defining the early multi-potent stages of hematopoiesis as Mll dependent.  These studies demonstrate that Mll plays selective and independent roles within the hematopoietic system, maintaining quiescence in HSCs and promoting proliferation in progenitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqiXPklTPaJLVg90H21EOLACvtfcHk0lipwDGvBT9qAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagu7vO&md5=e8ce8a59ef3ecda7ac594d2fdd6eb41e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.stem.2007.05.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stem.2007.05.019%26sid%3Dliteratum%253Aachs%26aulast%3DJude%26aufirst%3DC.%2BD.%26aulast%3DClimer%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DArtinger%26aufirst%3DE.%26aulast%3DFisher%26aufirst%3DJ.%2BK.%26aulast%3DErnst%26aufirst%3DP.%26atitle%3DUnique%2520and%2520independent%2520roles%2520for%2520MLL%2520in%2520adult%2520hematopoietic%2520stem%2520cells%2520and%2520progenitors%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2007%26volume%3D1%26spage%3D324%26epage%3D337%26doi%3D10.1016%2Fj.stem.2007.05.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Meyer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burmeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gröger, D.</span><span class="NLM_etal"></span><span> </span><span class="NLM_article-title">The MLL recombinome of acute leukemias in 2013</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">2165</span><span class="NLM_x">–</span> <span class="NLM_lpage">2176</span><span class="refDoi"> DOI: 10.1038/leu.2013.135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fleu.2013.135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=23628958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslaqs73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=2165-2176&author=C.+Meyerauthor=J.+Hofmannauthor=T.+Burmeisterauthor=D.+Gr%C3%B6ger&title=The+MLL+recombinome+of+acute+leukemias+in+2013&doi=10.1038%2Fleu.2013.135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The MLL recombinome of acute leukemias in 2013</span></div><div class="casAuthors">Meyer, C.; Hofmann, J.; Burmeister, T.; Groeger, D.; Park, T. S.; Emerenciano, M.; Pombo de Oliveira, M.; Renneville, A.; Villarese, P.; Macintyre, E.; Cave, H.; Clappier, E.; Mass-Malo, K.; Zuna, J.; Trka, J.; De Braekeleer, E.; De Braekeleer, M.; Oh, S. H.; Tsaur, G.; Fechina, L.; van der Velden, V. H. J.; van Dongen, J. J. M.; Delabesse, E.; Binato, R.; Silva, M. L. M.; Kustanovich, A.; Aleinikova, O.; Harris, M. H.; Lund-Aho, T.; Juvonen, V.; Heidenreich, O.; Vormoor, J.; Choi, W. W. L.; Jarosova, M.; Kolenova, A.; Bueno, C.; Menendez, P.; Wehner, S.; Eckert, C.; Talmant, P.; Tondeur, S.; Lippert, E.; Launay, E.; Henry, C.; Ballerini, P.; Lapillone, H.; Callanan, M. B.; Cayuela, J. M.; Herbaux, C.; Cazzaniga, G.; Kakadiya, P. M.; Bohlander, S.; Ahlmann, M.; Choi, J. R.; Gameiro, P.; Lee, D. S.; Krauter, J.; Cornillet-Lefebvre, P.; Te Kronnie, G.; Schaefer, B. W.; Kubetzko, S.; Alonso, C. N.; zur Stadt, U.; Sutton, R.; Venn, N. C.; Izraeli, S.; Trakhtenbrot, L.; Madsen, H. O.; Archer, P.; Hancock, J.; Cerveira, N.; Teixeira, M. R.; Lo Nigro, L.; Moericke, A.; Stanulla, M.; Schrappe, M.; Sedek, L.; Szczepanski, T.; Zwaan, C. M.; Coenen, E. A.; van den Heuvel-Eibrink, M. M.; Strehl, S.; Dworzak, M.; Panzer-Gruemayer, R.; Dingermann, T.; Klingebiel, T.; Marschalek, R.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2165-2176</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chromosomal rearrangements of the human MLL (mixed lineage leukemia) gene are assocd. with high-risk infant, pediatric, adult and therapy-induced acute leukemias.  We used long-distance inverse-polymerase chain reaction to characterize the chromosomal rearrangement of individual acute leukemia patients.  We present data of the mol. characterization of 1590 MLL-rearranged biopsy samples obtained from acute leukemia patients.  The precise localization of genomic breakpoints within the MLL gene and the involved translocation partner genes (TPGs) were detd. and novel TPGs identified.  All patients were classified according to their gender (852 females and 745 males), age at diagnosis (558 infant, 416 pediatric and 616 adult leukemia patients) and other clin. criteria.  Combined data of our study and recently published data revealed a total of 121 different MLL rearrangements, of which 79 TPGs are now characterized at the mol. level.  However, only seven rearrangements seem to be predominantly assocd. with illegitimate recombinations of the MLL gene (∼90%): AFF1/AF4, MLLT3/AF9, MLLT1/ENL, MLLT10/AF10, ELL, partial tandem duplications (MLL PTDs) and MLLT4/AF6, resp.  The MLL breakpoint distributions for all clin. relevant subtypes (gender, disease type, age at diagnosis, reciprocal, complex and therapy-induced translocations) are presented.  Finally, we present the extending network of reciprocal MLL fusions deriving from complex rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoepBrGUCQUZrVg90H21EOLACvtfcHk0lipwDGvBT9qAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslaqs73M&md5=a82f47327e5932e0edc20a098a5d1a1d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fleu.2013.135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2013.135%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DHofmann%26aufirst%3DJ.%26aulast%3DBurmeister%26aufirst%3DT.%26aulast%3DGr%25C3%25B6ger%26aufirst%3DD.%26atitle%3DThe%2520MLL%2520recombinome%2520of%2520acute%2520leukemias%2520in%25202013%26jtitle%3DLeukemia%26date%3D2013%26volume%3D27%26spage%3D2165%26epage%3D2176%26doi%3D10.1038%2Fleu.2013.135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Borkhardt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wuchter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viehmann, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pils, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teigler-Schlegel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanulla, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmermann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludwig, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janka-Schaub, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrappe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harbott, J.</span><span> </span><span class="NLM_article-title">Infant acute lymphoblastic leukemia - combined cytogenetic, immunophenotypical and molecular analysis of 77 cases</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1685</span><span class="NLM_x">–</span> <span class="NLM_lpage">1690</span><span class="refDoi"> DOI: 10.1038/sj.leu.2402595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fsj.leu.2402595" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1685-1690&author=A.+Borkhardtauthor=C.+Wuchterauthor=S.+Viehmannauthor=S.+Pilsauthor=A.+Teigler-Schlegelauthor=M.+Stanullaauthor=M.+Zimmermannauthor=W.+D.+Ludwigauthor=G.+Janka-Schaubauthor=M.+Schrappeauthor=J.+Harbott&title=Infant+acute+lymphoblastic+leukemia+-+combined+cytogenetic%2C+immunophenotypical+and+molecular+analysis+of+77+cases&doi=10.1038%2Fsj.leu.2402595"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2402595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2402595%26sid%3Dliteratum%253Aachs%26aulast%3DBorkhardt%26aufirst%3DA.%26aulast%3DWuchter%26aufirst%3DC.%26aulast%3DViehmann%26aufirst%3DS.%26aulast%3DPils%26aufirst%3DS.%26aulast%3DTeigler-Schlegel%26aufirst%3DA.%26aulast%3DStanulla%26aufirst%3DM.%26aulast%3DZimmermann%26aufirst%3DM.%26aulast%3DLudwig%26aufirst%3DW.%2BD.%26aulast%3DJanka-Schaub%26aufirst%3DG.%26aulast%3DSchrappe%26aufirst%3DM.%26aulast%3DHarbott%26aufirst%3DJ.%26atitle%3DInfant%2520acute%2520lymphoblastic%2520leukemia%2520-%2520combined%2520cytogenetic%252C%2520immunophenotypical%2520and%2520molecular%2520analysis%2520of%252077%2520cases%26jtitle%3DLeukemia%26date%3D2002%26volume%3D16%26spage%3D1685%26epage%3D1690%26doi%3D10.1038%2Fsj.leu.2402595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Felix, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessells, J. M.</span><span> </span><span class="NLM_article-title">Pediatric Acute Lymphoblastic Leukemia: Challenges and Controversies in 2000</span> <span class="citation_source-journal">Hematology Am. Soc. Hematol. Educ. Program</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">2000</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1182/asheducation-2000.1.285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1182%2Fasheducation-2000.1.285" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2000&publication_year=2000&pages=285-302&author=C.+A.+Felixauthor=B.+J.+Langeauthor=J.+M.+Chessells&title=Pediatric+Acute+Lymphoblastic+Leukemia%3A+Challenges+and+Controversies+in+2000&doi=10.1182%2Fasheducation-2000.1.285"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1182%2Fasheducation-2000.1.285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fasheducation-2000.1.285%26sid%3Dliteratum%253Aachs%26aulast%3DFelix%26aufirst%3DC.%2BA.%26aulast%3DLange%26aufirst%3DB.%2BJ.%26aulast%3DChessells%26aufirst%3DJ.%2BM.%26atitle%3DPediatric%2520Acute%2520Lymphoblastic%2520Leukemia%253A%2520Challenges%2520and%2520Controversies%2520in%25202000%26jtitle%3DHematology%2520Am.%2520Soc.%2520Hematol.%2520Educ.%2520Program%26date%3D2000%26volume%3D2000%26spage%3D285%26epage%3D302%26doi%3D10.1182%2Fasheducation-2000.1.285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Krivtsov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span> </span><span class="NLM_article-title">MLL translocations, histone modifications and leukaemia stem-cell development</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">823</span><span class="NLM_x">–</span> <span class="NLM_lpage">833</span><span class="refDoi"> DOI: 10.1038/nrc2253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnrc2253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=17957188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=823-833&author=A.+V.+Krivtsovauthor=S.+A.+Armstrong&title=MLL+translocations%2C+histone+modifications+and+leukaemia+stem-cell+development&doi=10.1038%2Fnrc2253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">MLL translocations, histone modifications and leukaemia stem-cell development</span></div><div class="casAuthors">Krivtsov, Andrei V.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia (MLL) has histone methyltransferase activity and regulates the expression of genes such as Hox genes.  This activity is lost in MLL fusion proteins resulting from inter-chromosomal translocations, which are leukemogenic.  How do MLL fusions function and what is their role in leukemia stem cells.  Translocations that involve the mixed lineage leukemia (MLL) gene identify a unique group of acute leukemias, and often predict a poor prognosis.  The MLL gene encodes a DNA-binding protein that methylates histone H3 lysine 4 (H3K4), and pos. regulates gene expression including multiple Hox genes.  Leukaemogenic MLL translocations encode MLL fusion proteins that have lost H3K4 methyltransferase activity.  A key feature of MLL fusion proteins is their ability to efficiently transform haematopoietic cells into leukemia stem cells.  The link between a chromatin modulator and leukemia stem cells provides support for epigenetic landscapes as an important part of leukemia and normal stem-cell development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9NXrcNtrd7Vg90H21EOLACvtfcHk0lhxZ83V1SUwag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ&md5=56efc9bbbb9f24fb4233e40ed86628fd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc2253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2253%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DMLL%2520translocations%252C%2520histone%2520modifications%2520and%2520leukaemia%2520stem-cell%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D823%26epage%3D833%26doi%3D10.1038%2Fnrc2253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Thiel, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blessington, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feather, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koretzky, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, X.</span><span> </span><span class="NLM_article-title">MLL-AF9-induced leukemogenesis requires coexpression of the wild-type Mll allele</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1016/j.ccr.2009.12.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.ccr.2009.12.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=20159607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Kls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=148-159&author=A.+T.+Thielauthor=P.+Blessingtonauthor=T.+Zouauthor=D.+Featherauthor=X.+Wuauthor=J.+Yanauthor=H.+Zhangauthor=Z.+Liuauthor=P.+Ernstauthor=G.+A.+Koretzkyauthor=X.+Hua&title=MLL-AF9-induced+leukemogenesis+requires+coexpression+of+the+wild-type+Mll+allele&doi=10.1016%2Fj.ccr.2009.12.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MLL-AF9-induced leukemogenesis requires coexpression of the wild-type Mll allele</span></div><div class="casAuthors">Thiel, Austin T.; Blessington, Peter; Zou, Tao; Feather, Danielle; Wu, Xinjiang; Yan, Jizhou; Zhang, Hui; Liu, Zuguo; Ernst, Patricia; Koretzky, Gary A.; Hua, Xianxin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-159</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Oncogenic fusion proteins are capable of initiating tumorigenesis, but the role of their wild-type counterparts in this process is poorly understood.  The mixed lineage leukemia (MLL) gene undergoes chromosomal translocations, resulting in the formation of oncogenic MLL fusion proteins (MLL-FPs).  Here, we show that menin recruits both wild-type MLL and oncogenic MLL-AF9 fusion protein to the loci of HOX genes to activate their transcription.  Wild-type MLL not only catalyzes histone methylation at key target genes but also controls distinct MLL-AF9-induced histone methylation.  Notably, the wild-type Mll allele is required for MLL-AF9-induced leukemogenesis and maintenance of MLL-AF9-transformed cells.  These findings suggest an essential cooperation between an oncogene and its wild-type counterpart in MLL-AF9-induced leukemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUZ8jLUM0JCrVg90H21EOLACvtfcHk0li2ApdMecMK1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Kls7s%253D&md5=ddcce857a9f60c6a94eb17440ccaa77d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.12.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.12.034%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DA.%2BT.%26aulast%3DBlessington%26aufirst%3DP.%26aulast%3DZou%26aufirst%3DT.%26aulast%3DFeather%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DErnst%26aufirst%3DP.%26aulast%3DKoretzky%26aufirst%3DG.%2BA.%26aulast%3DHua%26aufirst%3DX.%26atitle%3DMLL-AF9-induced%2520leukemogenesis%2520requires%2520coexpression%2520of%2520the%2520wild-type%2520Mll%2520allele%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26spage%3D148%26epage%3D159%26doi%3D10.1016%2Fj.ccr.2009.12.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stadler, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basrur, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitcomb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elenitoba-Johnson, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Multiple interactions recruit MLL1 and MLL1 fusion proteins to the HOXA9 locus in leukemogenesis</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">853</span><span class="NLM_x">–</span> <span class="NLM_lpage">863</span><span class="refDoi"> DOI: 10.1016/j.molcel.2010.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.molcel.2010.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=20541448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVWmtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=853-863&author=T.+A.+Milneauthor=J.+Kimauthor=G.+G.+Wangauthor=S.+C.+Stadlerauthor=V.+Basrurauthor=S.+J.+Whitcombauthor=Z.+Wangauthor=A.+J.+Ruthenburgauthor=K.+S.+Elenitoba-Johnsonauthor=R.+G.+Roederauthor=C.+D.+Allis&title=Multiple+interactions+recruit+MLL1+and+MLL1+fusion+proteins+to+the+HOXA9+locus+in+leukemogenesis&doi=10.1016%2Fj.molcel.2010.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple interactions recruit MLL1 and MLL1 fusion proteins to the HOXA9 locus in leukemogenesis</span></div><div class="casAuthors">Milne, Thomas A.; Kim, Jaehoon; Wang, Gang G.; Stadler, Sonja C.; Basrur, Venkatesha; Whitcomb, Sarah J.; Wang, Zhanxin; Ruthenburg, Alexander J.; Elenitoba-Johnson, Kojo S. J.; Roeder, Robert G.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">853-863</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">MLL1 fusion proteins activate HoxA9 gene expression and cause aggressive leukemias that respond poorly to treatment, but how they recognize and stably bind to HoxA9 is not clearly understood.  In a systematic anal. of MLL1 domain recruitment activity, we identified an essential MLL1 recruitment domain that includes the CXXC domain and PHD fingers and is controlled by direct interactions with the PAF elongation complex and H3K4Me2/3.  MLL1 fusion proteins lack the PHD fingers and require prebinding of a wild-type MLL1 complex and CXXC domain recognition of DNA for stable HoxA9 assocn.  Together, these results suggest that specific recruitment of MLL1 requires multiple interactions and is a precondition for stable recruitment of MLL1 fusion proteins to HoxA9 in leukemogenesis.  Since wild-type MLL1 and oncogenic MLL1 fusion proteins have overlapping yet distinct recruitment mechanisms, this creates a window of opportunity that could be exploited for the development of targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSrmMWM1988rVg90H21EOLACvtfcHk0li2ApdMecMK1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVWmtbs%253D&md5=bb5a19985cad898e15c44851d5b3b839</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DStadler%26aufirst%3DS.%2BC.%26aulast%3DBasrur%26aufirst%3DV.%26aulast%3DWhitcomb%26aufirst%3DS.%2BJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DMultiple%2520interactions%2520recruit%2520MLL1%2520and%2520MLL1%2520fusion%2520proteins%2520to%2520the%2520HOXA9%2520locus%2520in%2520leukemogenesis%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D38%26spage%3D853%26epage%3D863%26doi%3D10.1016%2Fj.molcel.2010.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vought, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">A conserved arginine-containing motif crucial for the assembly and enzymatic activity of the mixed lineage leukemia protein-1 core complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32162</span><span class="NLM_x">–</span> <span class="NLM_lpage">32175</span><span class="refDoi"> DOI: 10.1074/jbc.M806317200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+conserved+arginine-containing+motif+crucial+for+the+assembly+and+enzymatic+activity+of+the+mixed+lineage+leukemia+protein-1+core+complex&doi=10.1074%2Fjbc.M806317200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0liz0Uv7N_Tf1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520conserved%2520arginine-containing%2520motif%2520crucial%2520for%2520the%2520assembly%2520and%2520enzymatic%2520activity%2520of%2520the%2520mixed%2520lineage%2520leukemia%2520protein-1%2520core%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175%26doi%3D10.1074%2Fjbc.M806317200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Analysis of the binding of mixed lineage leukemia 1 (MLL1) and histone 3 peptides to WD repeat domain 5 (WDR5) for the design of inhibitors of the MLL1-WDR5 interaction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5179</span><span class="NLM_x">–</span> <span class="NLM_lpage">5185</span><span class="refDoi"> DOI: 10.1021/jm100139b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100139b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVKqu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5179-5185&author=H.+Karatasauthor=E.+C.+Townsendauthor=D.+Bernardauthor=Y.+Douauthor=S.+Wang&title=Analysis+of+the+binding+of+mixed+lineage+leukemia+1+%28MLL1%29+and+histone+3+peptides+to+WD+repeat+domain+5+%28WDR5%29+for+the+design+of+inhibitors+of+the+MLL1-WDR5+interaction&doi=10.1021%2Fjm100139b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of the Binding of Mixed Lineage Leukemia 1 (MLL1) and Histone 3 Peptides to WD Repeat Domain 5 (WDR5) for the Design of Inhibitors of the MLL1-WDR5 Interaction</span></div><div class="casAuthors">Karatas, Hacer; Townsend, Elizabeth C.; Bernard, Denzil; Dou, Yali; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5179-5185</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MLL1 is a histone 3 lysine 4 (H3K4) methyltransferase and a promising new cancer therapeutic target.  The catalytic activity of MLL1 is regulated by the formation of a core complex consisting of MLL1, WDR5, RbBP5, and Ash2L.  The interaction between WDR5 and MLL1 plays an essential role in regulation of the H3K4 methyltransferase activity of MLL1 and targeting this interaction using small mols. may represent an attractive therapeutic strategy.  In this study, we have defined the essential elements in MLL1 required for its high-affinity binding to WDR5.  Our data showed that the minimal elements crucial for high-affinity binding of MLL1 to WDR5 are -CO-ARA-NH- motif and two intramol. hydrogen bonds that stabilize the conformation of this motif.  Two 3-mer peptides, Ac-ARA-NH2 and Ac-ART-NH2, were designed based upon MLL1 and H3 sequences and achieved Ki values of 120 and 20 nM to WDR5, resp.  Our study provides a concrete basis for the design of potent peptidomimetics and nonpeptidic compds. to inhibit MLL1 activity by targeting the MLL1 and WDR5 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxEb9S0TLVzrVg90H21EOLACvtfcHk0liz0Uv7N_Tf1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVKqu74%253D&md5=b13aa34ad30e55a0d4742294a46de8ff</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm100139b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100139b%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520the%2520binding%2520of%2520mixed%2520lineage%2520leukemia%25201%2520%2528MLL1%2529%2520and%2520histone%25203%2520peptides%2520to%2520WD%2520repeat%2520domain%25205%2520%2528WDR5%2529%2520for%2520the%2520design%2520of%2520inhibitors%2520of%2520the%2520MLL1-WDR5%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5179%26epage%3D5185%26doi%3D10.1021%2Fjm100139b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">High-affinity, small-molecule peptidomimetic inhibitors of MLL1/WDR5 protein-protein interaction</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">682</span><span class="refDoi"> DOI: 10.1021/ja306028q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja306028q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=669-682&author=H.+Karatasauthor=E.+C.+Townsendauthor=F.+Caoauthor=Y.+Chenauthor=D.+Bernardauthor=L.+Liuauthor=M.+Leiauthor=Y.+Douauthor=S.+Wang&title=High-affinity%2C+small-molecule+peptidomimetic+inhibitors+of+MLL1%2FWDR5+protein-protein+interaction&doi=10.1021%2Fja306028q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fja306028q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja306028q%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DHigh-affinity%252C%2520small-molecule%2520peptidomimetic%2520inhibitors%2520of%2520MLL1%252FWDR5%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D669%26epage%3D682%26doi%3D10.1021%2Fja306028q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span> </span><span class="NLM_article-title">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">713</span><span class="NLM_x">–</span> <span class="NLM_lpage">719</span><span class="refDoi"> DOI: 10.1038/nsmb1128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnsmb1128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=16878130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=713-719&author=Y.+Douauthor=T.+A.+Milneauthor=A.+J.+Ruthenburgauthor=S.+Leeauthor=J.+W.+Leeauthor=G.+L.+Verdineauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Regulation+of+MLL1+H3K4+methyltransferase+activity+by+its+core+components&doi=10.1038%2Fnsmb1128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span></div><div class="casAuthors">Dou, Yali; Milne, Thomas A.; Ruthenburg, Alexander J.; Lee, Seunghee; Lee, Jae Woon; Verdine, Gregory L.; Allis, C. David; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">713-719</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 Lys4 (H3K4) methylation is a prevalent mark assocd. with transcription activation.  A common feature of several H3K4 methyltransferase complexes is the presence of three structural components (RbBP5, Ash2L and WDR5) and a catalytic subunit contg. a SET domain.  Here we report the first biochem. reconstitution of a functional four-component mixed-lineage leukemia protein-1 (MLL1) core complex.  This reconstitution, combined with in vivo assays, allows direct anal. of the contribution of each component to MLL1 enzymic activity and their roles in transcriptional regulation.  Moreover, taking clues from a crystal structure anal., we demonstrate that WDR5 mediates interactions of the MLL1 catalytic unit both with the common structural platform and with the histone substrate.  Mechanistic insights gained from this study can be generalized to the whole family of SET1-like histone methyltransferases in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp65Iy--Is42bVg90H21EOLACvtfcHk0liCCnF_h6FAcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D&md5=d6fa704fa0a2fa69afddb540fbff28ae</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1128%26sid%3Dliteratum%253Aachs%26aulast%3DDou%26aufirst%3DY.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DRegulation%2520of%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520by%2520its%2520core%2520components%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D713%26epage%3D719%26doi%3D10.1038%2Fnsmb1128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">Structure of WDR5 bound to mixed lineage leukemia protein-1 peptide</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32158</span><span class="NLM_x">–</span> <span class="NLM_lpage">32161</span><span class="refDoi"> DOI: 10.1074/jbc.C800164200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1074%2Fjbc.C800164200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=18829459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32158-32161&author=A.+Patelauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=Structure+of+WDR5+bound+to+mixed+lineage+leukemia+protein-1+peptide&doi=10.1074%2Fjbc.C800164200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32158-32161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) catalyzes histone H3 lysine 4 methylation and is regulated by interaction with WDR5 (WD-repeat protein-5), RbBP5 (retinoblastoma-binding protein-5), and the Ash2L (absent, small, homeotic disks-2-like) oncoprotein.  In the accompanying investigation, we describe the identification of a conserved arginine contg. motif, called the "Win" or WDR5 interaction motif, that is essential for the assembly and H3K4 dimethylation activity of the MLL1 core complex.  Here we present a 1.7-Å crystal structure of WDR5 bound to a peptide derived from the MLL1 Win motif.  Our results show that Arg-3765 of MLL1 is bound in the same arginine binding pocket on WDR5 that was previously suggested to bind histone H3.  Thermodn. binding expts. show that the MLL1 Win peptide is preferentially recognized by WDR5.  These results are consistent with a model in which WDR5 recognizes Arg-3765 of MLL1, which is essential for the assembly and enzymic activity of the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2hQy6dZIN-LVg90H21EOLACvtfcHk0liCCnF_h6FAcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M&md5=770e511a13cf8ada19b37e0cb7115eaf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C800164200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C800164200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DStructure%2520of%2520WDR5%2520bound%2520to%2520mixed%2520lineage%2520leukemia%2520protein-1%2520peptide%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32158%26epage%3D32161%26doi%3D10.1074%2Fjbc.C800164200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Cao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouillette, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span> </span><span class="NLM_article-title">Targeting MLL1 H3K4 methyltransferase activity in mixed-lineage leukemia</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1016/j.molcel.2013.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.molcel.2013.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=24389101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=247-61&author=F.+Caoauthor=E.+C.+Townsendauthor=H.+Karatasauthor=J.+Xuauthor=L.+Liauthor=S.+Leeauthor=L.+Liuauthor=Y.+Chenauthor=P.+Ouilletteauthor=J.+Zhuauthor=J.+L.+Hessauthor=P.+Atadjaauthor=M.+Leiauthor=Z.+S.+Qinauthor=S.+Malekauthor=S.+Wangauthor=Y.+Dou&title=Targeting+MLL1+H3K4+methyltransferase+activity+in+mixed-lineage+leukemia&doi=10.1016%2Fj.molcel.2013.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MLL1 H3K4 Methyltransferase Activity in Mixed-Lineage Leukemia</span></div><div class="casAuthors">Cao, Fang; Townsend, Elizabeth C.; Karatas, Hacer; Xu, Jing; Li, Li; Lee, Shirley; Liu, Liu; Chen, Yong; Ouillette, Peter; Zhu, Jidong; Hess, Jay L.; Atadja, Peter; Lei, Ming; Qin, Zhaohui S.; Malek, Sami; Wang, Shaomeng; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-261</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Here we report a comprehensive characterization of our recently developed inhibitor MM-401 that targets the MLL1 H3K4 methyltransferase activity.  MM-401 is able to specifically inhibit MLL1 activity by blocking MLL1-WDR5 interaction and thus the complex assembly.  This targeting strategy does not affect other mixed-lineage leukemia (MLL) family histone methyltransferases (HMTs), revealing a unique regulatory feature for the MLL1 complex.  Using MM-401 and its enantiomer control MM-NC-401, we show that inhibiting MLL1 methyltransferase activity specifically blocks proliferation of MLL cells by inducing cell-cycle arrest, apoptosis, and myeloid differentiation without general toxicity to normal bone marrow cells or non-MLL cells.  More importantly, transcriptome analyses show that MM-401 induces changes in gene expression similar to those of MLL1 deletion, supporting a predominant role of MLL1 activity in regulating MLL1-dependent leukemia transcription program.  We envision broad applications for MM-401 in basic and translational research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov9sdJIz15pbVg90H21EOLACvtfcHk0lgEWISqL1tv-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D&md5=f3e482ba3474b39e1bbdde0d10407d1f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DF.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuillette%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DQin%26aufirst%3DZ.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DTargeting%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520in%2520mixed-lineage%2520leukemia%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D53%26spage%3D247%26epage%3D61%26doi%3D10.1016%2Fj.molcel.2013.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seitova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Small-molecule inhibition of MLL activity by disruption of its interaction with WDR5</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1042/BJ20121280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1042%2FBJ20121280" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=151-159&author=G.+Senisterraauthor=H.+Wuauthor=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=D.+Barsyte-Lovejoyauthor=L.+Dombrovskiauthor=A.+Dongauthor=K.+T.+Nguyenauthor=D.+Smilauthor=Y.+Bolshanauthor=T.+Hajianauthor=H.+Heauthor=A.+Seitovaauthor=I.+Chauauthor=F.+Liauthor=G.+Podaauthor=J.+F.+Coutureauthor=P.+J.+Brownauthor=R.+Al-Awarauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Small-molecule+inhibition+of+MLL+activity+by+disruption+of+its+interaction+with+WDR5&doi=10.1042%2FBJ20121280"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2FBJ20121280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121280%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DSeitova%26aufirst%3DA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520MLL%2520activity%2520by%2520disruption%2520of%2520its%2520interaction%2520with%2520WDR5%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D151%26epage%3D159%26doi%3D10.1042%2FBJ20121280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zepeda-Velazquez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiyota, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span> </span><span class="NLM_article-title">Structure-based optimization of a small molecule antagonist of the interaction between WD repeat-containing protein 5 (WDR5) and mixed-lineage leukemia 1 (MLL1)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">2478</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01630</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01630" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2478-96&author=M.+Getlikauthor=D.+Smilauthor=C.+Zepeda-Velazquezauthor=Y.+Bolshanauthor=G.+Podaauthor=H.+Wuauthor=A.+Dongauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=G.+Senisterraauthor=L.+Dombrovskiauthor=T.+Hajianauthor=T.+Kiyotaauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=R.+Al-Awar&title=Structure-based+optimization+of+a+small+molecule+antagonist+of+the+interaction+between+WD+repeat-containing+protein+5+%28WDR5%29+and+mixed-lineage+leukemia+1+%28MLL1%29&doi=10.1021%2Facs.jmedchem.5b01630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span></div><div class="casAuthors">Getlik, Matthaus; Smil, David; Zepeda-Velazquez, Carlos; Bolshan, Yuri; Poda, Gennady; Wu, Hong; Dong, Aiping; Kuznetsova, Ekaterina; Marcellus, Richard; Senisterra, Guillermo; Dombrovski, Ludmila; Hajian, Taraneh; Kiyota, Taira; Schapira, Matthieu; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Al-awar, Rima</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2478-2496</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">WD repeat-contg. protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1).  Rearrangement of the MLL1 gene is assocd. with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics.  Our previous work showed that binding of small mol. ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity.  Initial structure-activity relationship studies identified N-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein-protein interaction.  Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents.  This allowed us to develop the first highly potent (Kdisp < 100 nM) small mol. antagonists of the WDR5-MLL1 interaction and demonstrate that N-(4-(4-methylpiperazin-1-yl)-3'-(morpholinomethyl)-[1,1'-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide 16d (OICR-9429) is a potent and selective chem. probe suitable to help dissect the biol. role of WDR5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo60Phwp2ghZLVg90H21EOLACvtfcHk0ljGjQGk-aHSGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVSjsr0%253D&md5=f3a3a529dcf114a459a8b5d6f7270154</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01630%26sid%3Dliteratum%253Aachs%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DKiyota%26aufirst%3DT.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAl-Awar%26aufirst%3DR.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520small%2520molecule%2520antagonist%2520of%2520the%2520interaction%2520between%2520WD%2520repeat-containing%2520protein%25205%2520%2528WDR5%2529%2520and%2520mixed-lineage%2520leukemia%25201%2520%2528MLL1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2478%26epage%3D96%26doi%3D10.1021%2Facs.jmedchem.5b01630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Li, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Z. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, X. K.</span><span> </span><span class="NLM_article-title">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">480</span><span class="NLM_x">–</span> <span class="NLM_lpage">489</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2016.08.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.ejmech.2016.08.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=27598236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2016&pages=480-489&author=D.+D.+Liauthor=W.+L.+Chenauthor=Z.+H.+Wangauthor=Y.+Y.+Xieauthor=X.+L.+Xuauthor=Z.+Y.+Jiangauthor=X.+J.+Zhangauthor=Q.+D.+Youauthor=X.+K.+Guo&title=High-affinity+small+molecular+blockers+of+mixed+lineage+leukemia+1+%28MLL1%29-WDR5+interaction+inhibit+MLL1+complex+H3K4+methyltransferase+activity&doi=10.1016%2Fj.ejmech.2016.08.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Wang, Zhi-Hui; Xie, Yi-Yue; Xu, Xiao-Li; Jiang, Zheng-Yu; Zhang, Xiao-Jin; You, Qi-Dong; Guo, Xiao-Ke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">480-489</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1-WDR5 protein-protein interaction is essential for MLL1 H3K4 methyltransferase activity.  Targeting MLL1 enzymic activity to regulate expression level of MLL-dependent genes represents a therapeutic strategy for acute leukemia harboring MLL fusion proteins.  Herein we reported a series of biphenyl compds. disturbed MLL1-WDR5 interaction.  These compds. effectively inhibited MLL1 histone methyltransferase (HMT) activity in vitro and in MV4-11 cell line.  The representative compd. 30 (DDO-2084) inhibited proliferation and induced apoptosis of MV4-11 cells through deregulating expression level of Hoxa9 and Meis-1 genes, which emphasized our compds. were on-target.  Optimization of compd. 30 led to high-affinity inhibitors.  Esp., compd. 42 (DDO-2117, IC50 = 7.6 nM) bearing an amino and a 4-aminobutanamido group was the most potent inhibitor reported to-date, and showed the most potent inhibitory activity (IC50 = 0.19 μM) in HMT assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIPjH4_g5XrVg90H21EOLACvtfcHk0ljGjQGk-aHSGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWisLzJ&md5=987a0be9098e067213e4bccf726e3dd3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.08.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.08.036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%2BD.%26aulast%3DChen%26aufirst%3DW.%2BL.%26aulast%3DWang%26aufirst%3DZ.%2BH.%26aulast%3DXie%26aufirst%3DY.%2BY.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DJiang%26aufirst%3DZ.%2BY.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26atitle%3DHigh-affinity%2520small%2520molecular%2520blockers%2520of%2520mixed%2520lineage%2520leukemia%25201%2520%2528MLL1%2529-WDR5%2520interaction%2520inhibit%2520MLL1%2520complex%2520H3K4%2520methyltransferase%2520activity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D124%26spage%3D480%26epage%3D489%26doi%3D10.1016%2Fj.ejmech.2016.08.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Li, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, X. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H. P.</span><span> </span><span class="NLM_article-title">Structure-based design and synthesis of small molecular inhibitors disturbing the interaction of MLL1-WDR5</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2016.04.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.ejmech.2016.04.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=27116709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVCksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2016&pages=1-8&author=D.+D.+Liauthor=W.+L.+Chenauthor=X.+L.+Xuauthor=F.+Jiangauthor=L.+Wangauthor=Y.+Y.+Xieauthor=X.+J.+Zhangauthor=X.+K.+Guoauthor=Q.+D.+Youauthor=H.+P.+Sun&title=Structure-based+design+and+synthesis+of+small+molecular+inhibitors+disturbing+the+interaction+of+MLL1-WDR5&doi=10.1016%2Fj.ejmech.2016.04.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design and synthesis of small molecular inhibitors disturbing the interaction of MLL1-WDR5</span></div><div class="casAuthors">Li, Dong-Dong; Chen, Wei-Lin; Xu, Xiao-Li; Jiang, Fen; Wang, Lei; Xie, Yi-Yue; Zhang, Xiao-Jin; Guo, Xiao-Ke; You, Qi-Dong; Sun, Hao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MLL1 complex catalyzes the methylation of H3K4, and plays important roles in the development of acute leukemia harboring MLL fusion proteins.  Targeting MLL1-WDR5 protein-protein interaction (PPI) to inhibit the activity of histone methyltransferase of MLL1 complex is a novel strategy for treating of acute leukemia.  I (IC50 = 0.3 μM) was defined as a potent small mol. to disturb the interaction of MLL1-WDR5.  Here, the authors described structure-based design and synthesis of small mol. inhibitors to block MLL1-WDR5 PPI.  Esp., compd. II (IC50 = 104 nM) was the most potent small mol., and about 3-times more potent than I.  The authors also discussed the SAR of these series of compds. with docking study, which may stimulate more potent compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNsAbY0it5GLVg90H21EOLACvtfcHk0lgRywkj7RMI7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVCksLw%253D&md5=b18d482f34731dd1558e9b78b770c2c1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.04.032%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%2BD.%26aulast%3DChen%26aufirst%3DW.%2BL.%26aulast%3DXu%26aufirst%3DX.%2BL.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DY.%2BY.%26aulast%3DZhang%26aufirst%3DX.%2BJ.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DSun%26aufirst%3DH.%2BP.%26atitle%3DStructure-based%2520design%2520and%2520synthesis%2520of%2520small%2520molecular%2520inhibitors%2520disturbing%2520the%2520interaction%2520of%2520MLL1-WDR5%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D118%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ejmech.2016.04.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Li, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, X. K.</span><span> </span><span class="NLM_article-title">Structure-based design of ester compounds to inhibit MLL complex catalytic activity by targeting mixed lineage leukemia 1 (MLL1)-WDR5 interaction</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">6109</span><span class="NLM_x">–</span> <span class="NLM_lpage">6118</span><span class="refDoi"> DOI: 10.1016/j.bmc.2016.09.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.bmc.2016.09.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=27720555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GnsL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=6109-6118&author=D.+D.+Liauthor=Z.+H.+Wangauthor=W.+L.+Chenauthor=Y.+Y.+Xieauthor=Q.+D.+Youauthor=X.+K.+Guo&title=Structure-based+design+of+ester+compounds+to+inhibit+MLL+complex+catalytic+activity+by+targeting+mixed+lineage+leukemia+1+%28MLL1%29-WDR5+interaction&doi=10.1016%2Fj.bmc.2016.09.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of ester compounds to inhibit MLL complex catalytic activity by targeting mixed lineage leukemia 1 (MLL1)-WDR5 interaction</span></div><div class="casAuthors">Li, Dong-Dong; Wang, Zhi-Hui; Chen, Wei-Lin; Xie, Yi-Yue; You, Qi-Dong; Guo, Xiao-Ke</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6109-6118</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">WDR5 is an essential protein for enzymic activity of MLL1.  Targeting the protein-protein interaction (PPI) between MLL1 and WDR5 represents a new potential therapeutic strategy for MLL leukemia.  Based on the structure of reported inhibitor WDR5-0103, a class of ester compds. were designed and synthesized to disturb MLL1-WDR5 PPI.  These inhibitors efficiently inhibited the histone methyltransferase activity in vitro.  Esp., WL-15 was one of the most potent inhibitors, blocking the interaction of MLL1-WDR5 with IC50 value of 26.4 nM in competitive binding assay and inhibiting the catalytic activity of MLL1 complex with IC50 value of 5.4 μM.  Docking model indicated that ester compds. suitably occupied the central cavity of WDR5 protein and recapitulated the interactions of WDR5-0103 and the hydrophobic groups and key amino greatly increased the activity in blocking MLL1-WDR5 PPI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogEjIflnBk3LVg90H21EOLACvtfcHk0lgRywkj7RMI7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GnsL7F&md5=b5daa8ba82b039d050cc9ceb1a844a95</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.09.073%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%2BD.%26aulast%3DWang%26aufirst%3DZ.%2BH.%26aulast%3DChen%26aufirst%3DW.%2BL.%26aulast%3DXie%26aufirst%3DY.%2BY.%26aulast%3DYou%26aufirst%3DQ.%2BD.%26aulast%3DGuo%26aufirst%3DX.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520ester%2520compounds%2520to%2520inhibit%2520MLL%2520complex%2520catalytic%2520activity%2520by%2520targeting%2520mixed%2520lineage%2520leukemia%25201%2520%2528MLL1%2529-WDR5%2520interaction%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D6109%26epage%3D6118%26doi%3D10.1016%2Fj.bmc.2016.09.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Grebien, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giambruno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grover, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avellino, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skucha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vittori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schonegger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilban, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bock, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerlov, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPalpha N-terminal leukemia</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">578</span><span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+Huberauthor=A.+Schoneggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-Awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+targeting+of+the+Wdr5-MLL+interaction+in+C%2FEBPalpha+N-terminal+leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0liUPDcTFHTkGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DSchonegger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520targeting%2520of%2520the%2520Wdr5-MLL%2520interaction%2520in%2520C%252FEBPalpha%2520N-terminal%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref500"><div class="reference"><strong class="refLabel"><a href="#ref500" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit500"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1038/nchembio.1084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref500/cit500&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref500/cit500&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref500/cit500&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells&doi=10.1038%2Fnchembio.1084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit500R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0liUPDcTFHTkGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit500&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896%26doi%3D10.1038%2Fnchembio.1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref600"><div class="reference"><strong class="refLabel"><a href="#ref600" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit600"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1016/j.ccr.2011.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref600/cit600&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref600/cit600&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref600/cit600&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor&doi=10.1016%2Fj.ccr.2011.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit600R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0ljYsd8OnemquA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit600&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65%26doi%3D10.1016%2Fj.ccr.2011.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Nielsen, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nietlispach, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bannister, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murzin, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murzina, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laue, E. D.</span><span> </span><span class="NLM_article-title">Structure of the HP1 chromodomain bound to histone H3 methylated at lysine 9</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">416</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span><span class="refDoi"> DOI: 10.1038/nature722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnature722" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=11882902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD38Xitlaisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=416&publication_year=2002&pages=103-107&author=P.+R.+Nielsenauthor=D.+Nietlispachauthor=H.+R.+Mottauthor=J.+Callaghanauthor=A.+Bannisterauthor=T.+Kouzaridesauthor=A.+G.+Murzinauthor=N.+V.+Murzinaauthor=E.+D.+Laue&title=Structure+of+the+HP1+chromodomain+bound+to+histone+H3+methylated+at+lysine+9&doi=10.1038%2Fnature722"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the HP1 chromodomain bound to histone H3 methylated at lysine 9</span></div><div class="casAuthors">Nielsen, Peter R.; Nietlispach, Daniel; Mott, Helen R.; Callaghan, Juliana; Bannister, Andrew; Kouzarides, Tony; Murzin, Alexey G.; Murzina, Natalia V.; Laue, Ernest D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">416</span>
        (<span class="NLM_cas:issue">6876</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Specific modifications to histones are essential epigenetic markers-heritable changes in gene expression that do not affect the DNA sequence.  Methylation of lysine 9 in histone H3 is recognized by heterochromatin protein 1 (HP1), which directs the binding of other proteins to control chromatin structure and gene expression.  Here we show that HP1 uses an induced-fit mechanism for recognition of this modification, as revealed by the structure of its chromodomain bound to a histone H3 peptide dimethylated at Nζ of lysine 9.  The binding pocket for the N-Me groups is provided by three arom. side chains, Tyr21, Trp42 and Phe45, which reside in two regions that become ordered on binding of the peptide.  The side chain of Lys 9 is almost fully extended and surrounded by residues that are conserved in many other chromodomains.  The QTAR peptide sequence preceding Lys9 makes most of the addnl. interactions with the chromodomain, with HP1 residues Val23, Leu40, Trp42, Leu58 and Cys60 appearing to be a major determinant of specificity by binding the key buried Ala7.  These findings predict which other chromodomains will bind methylated proteins and suggest a motif that they recognize.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokEEsxGbMfvbVg90H21EOLACvtfcHk0ljYsd8OnemquA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xitlaisbg%253D&md5=b391888613b423cee326057d236ae6ae</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature722%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DP.%2BR.%26aulast%3DNietlispach%26aufirst%3DD.%26aulast%3DMott%26aufirst%3DH.%2BR.%26aulast%3DCallaghan%26aufirst%3DJ.%26aulast%3DBannister%26aufirst%3DA.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMurzin%26aufirst%3DA.%2BG.%26aulast%3DMurzina%26aufirst%3DN.%2BV.%26aulast%3DLaue%26aufirst%3DE.%2BD.%26atitle%3DStructure%2520of%2520the%2520HP1%2520chromodomain%2520bound%2520to%2520histone%2520H3%2520methylated%2520at%2520lysine%25209%26jtitle%3DNature%26date%3D2002%26volume%3D416%26spage%3D103%26epage%3D107%26doi%3D10.1038%2Fnature722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of x-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span><span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+x-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0ljYsd8OnemquA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520x-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljhLbyIPNs9Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Molecular recognition of histone H3 by the WD40 protein WDR5</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">703</span><span class="refDoi"> DOI: 10.1038/nsmb1116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1038%2Fnsmb1116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=16829960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=698-703&author=J.+F.+Coutureauthor=E.+Collazoauthor=R.+C.+Trievel&title=Molecular+recognition+of+histone+H3+by+the+WD40+protein+WDR5&doi=10.1038%2Fnsmb1116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of histone H3 by the WD40 protein WDR5</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">698-703</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The WD40-repeat protein WDR5 is a conserved subunit of Trithorax (TRX) histone methyltransferase complexes.  WDR5 has been reported to selectively bind dimethylated Lys-4 (K4me2) in histone H3 to promote K4 trimethylation by TRX.  To elucidate the basis of this binding specificity, we have detd. the crystal structure of WDR5 bound to a histone H3 peptide bearing K4me2.  The structure reveals that the N terminus of histone H3 binds as a 310-helix in the central depression formed by the WD40 repeats.  R2 in histone H3 is bound in the acidic channel in the protein's core, whereas K4me2 is solvent exposed and does not engage in direct interactions with WDR5.  Functional studies confirm that WDR5 recognizes A1, R2 and T3 in histone H3 but has virtually identical affinities for the unmodified and mono-, di- and trimethylated forms of K4, demonstrating that it does not discriminate among different degrees of methylation of this residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNU0kVOM0Y3rVg90H21EOLACvtfcHk0ljhLbyIPNs9Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrY%253D&md5=8c77c52b072810965f8c68fb6358a69c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1116%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520recognition%2520of%2520histone%2520H3%2520by%2520the%2520WD40%2520protein%2520WDR5%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D698%26epage%3D703%26doi%3D10.1038%2Fnsmb1116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ioerger, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacchettini, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span> </span><span class="NLM_article-title">PHENIX: Building new software for automated crystallographic structure determination</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1948</span><span class="NLM_x">–</span> <span class="NLM_lpage">1954</span><span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.+W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+Building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0ljhLbyIPNs9Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520Building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: Model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhNmhIEGGAvYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: An automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span><span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+An+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lhNmhIEGGAvYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520An%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Bricogne, G.; Blanc, E.; Brandl, M.; Flensburg, C.; Keller, P.; Paciorek, W.; Roversi, P.; Sharff, A.; Smart, O. S.; Vonrhein, C.; Womack, T. O.</span> BUSTER, 2.10.0., <span class="NLM_publisher-name">Global Phasing Ltd.</span>, <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bricogne%2C+G.%3B+Blanc%2C+E.%3B+Brandl%2C+M.%3B+Flensburg%2C+C.%3B+Keller%2C+P.%3B+Paciorek%2C+W.%3B+Roversi%2C+P.%3B+Sharff%2C+A.%3B+Smart%2C+O.+S.%3B+Vonrhein%2C+C.%3B+Womack%2C+T.+O.+BUSTER%2C+2.10.0.%2C+Global+Phasing+Ltd.%2C+Cambridge%2C+United+Kingdom%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref700"><div class="reference"><strong class="refLabel"><a href="#ref700" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit700"><span><span class="NLM_contrib-group">Jones, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">748</span><span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref700/cit700&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref700/cit700&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref700/cit700&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit700R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lhsfVHExIwSlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit700&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref800"><div class="reference"><strong class="refLabel"><a href="#ref800" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit800"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span> </span><span class="NLM_article-title">Improved protein - ligand docking using GOLD</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">623</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+protein+-+ligand+docking+using+GOLD"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit800&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520protein%2520-%2520ligand%2520docking%2520using%2520GOLD%26jtitle%3DProteins%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM9','PDB','4GM9'); return false;">PDB: 4GM9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB','PDB','4GMB'); return false;">PDB: 4GMB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC','PDB','5VFC'); return false;">PDB: 5VFC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2H14" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2H14','PDB','2H14'); return false;">PDB: 2H14</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR','PDB','3SMR'); return false;">PDB: 3SMR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GM3','PDB','4GM3'); return false;">PDB: 4GM3</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i86"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01796">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_25564"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01796">10.1021/acs.jmedchem.6b01796</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Crystallography data collection and refinement statistics, inhibitory activity of compound <b>18</b> against SET1 family histone methyltransferases, stability test of compound <b>18</b> in cell culture, and experimental methods for HMT inhibition assay of compound <b>18</b> for other SET1 family members (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Binding model of WDR5 with compound <b>13</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_001.pdf">jm6b01796_si_001.pdf (466.12 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_002.pdb">jm6b01796_si_002.pdb (187.3 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01796/suppl_file/jm6b01796_si_003.csv">jm6b01796_si_003.csv (2.81 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the cocrystal structure for compounds <b>16</b> and <b>18</b> in complex with WDR5 have been deposited into PDB (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GMB">4GMB</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC">5VFC</a>).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01796&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-12%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01796%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01796" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991e012c72d1c3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
